

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/311/2949522","@type":"ScholarlyArticle","name":"Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine","datePublished":"2017-01-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw061","keywords":["gene co-expression network","cis-regulatory element","grapevine","promoter","transcriptional regulation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/311/2949522","author":[{"name":"Wong, Darren Chern Jan","affiliation":"1 Wine Research Centre, University of British Columbia, Vancouver, BC, Canada","@type":"Person"},{"name":"Lopez Gutierrez, Rodrigo","affiliation":"1 Wine Research Centre, University of British Columbia, Vancouver, BC, Canada","@type":"Person"},{"name":"Gambetta, Gregory Alan","affiliation":"2 Ecophysiologie et Génomique Fonctionnelle de la Vigne, Bordeaux Sciences Agro, INRA, Université de Bordeaux, 33140 Villenave d’Ornon, France","@type":"Person"},{"name":"Castellarin, Simone Diego","affiliation":"1 Wine Research Centre, University of British Columbia, Vancouver, BC, Canada","@type":"Person"}],"description":"Abstract.  Coordinated transcriptional and metabolic reprogramming ensures a plant’s continued growth and survival under adverse environmental conditions. Trans","pageStart":"311","pageEnd":"326","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine" />
    <meta property="og:description" content="Abstract.  Coordinated transcriptional and metabolic reprogramming ensures a plant’s continued growth and survival under adverse environmental conditions. Trans" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/311/2949522" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wong, Darren Chern Jan" /><meta name="citation_author_institution" content="Wine Research Centre, University of British Columbia, Vancouver, BC, Canada" /><meta name="citation_author" content="Lopez Gutierrez, Rodrigo" /><meta name="citation_author_institution" content="Wine Research Centre, University of British Columbia, Vancouver, BC, Canada" /><meta name="citation_author" content="Gambetta, Gregory Alan" /><meta name="citation_author_institution" content="Ecophysiologie et Génomique Fonctionnelle de la Vigne, Bordeaux Sciences Agro, INRA, Université de Bordeaux, 33140 Villenave d’Ornon, France" /><meta name="citation_author" content="Castellarin, Simone Diego" /><meta name="citation_author_institution" content="Wine Research Centre, University of British Columbia, Vancouver, BC, Canada" /><meta name="citation_title" content="Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine" /><meta name="citation_firstpage" content="311" /><meta name="citation_lastpage" content="326" /><meta name="citation_doi" content="10.1093/dnares/dsw061" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=1182-7; " /><meta name="citation_reference" content="citation_title=A genomic perspective on plant transcription factors;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2000;  citation_volume=3;  citation_pages=423-34; " /><meta name="citation_reference" content="citation_title=Plant cis-acting regulatory DNA elements (PLACE) database: 1999;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=297-300; " /><meta name="citation_reference" content="citation_title=AGRIS: the arabidopsis gene regulatory information server, an update;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D1118-22; " /><meta name="citation_reference" content="citation_title=DNA-binding specificities of plant transcription factors and their potential to define target genes;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2014;  citation_volume=111;  citation_pages=2367-72; " /><meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in cellular responses and tolerance to dehydration and cold stresses;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2006;  citation_volume=57;  citation_pages=781-803; " /><meta name="citation_reference" content="citation_title=A hitchhiker’s guide to the MADS world of plants;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=214" /><meta name="citation_reference" content="citation_title=Auxin response factors;  citation_journal_title=Plant. Cell Environ;  citation_year=2016;  citation_volume=39;  citation_pages=1014-28; " /><meta name="citation_reference" content="citation_title=Genome-wide identification of potential plant E2F target genes;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=139;  citation_pages=316-28; " /><meta name="citation_reference" content="citation_title=The interaction between MYB proteins and their target DNA binding sites;  citation_journal_title=Biochim. Biophys. Acta – Gene Regul. Mech;  citation_year=2012;  citation_volume=1819;  citation_pages=67-77; " /><meta name="citation_reference" content="citation_title=Structural basis for DNA binding specificity by the auxin-dependent ARF transcription factors;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=156;  citation_pages=577-89; " /><meta name="citation_reference" content="citation_title=Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=14992-7; " /><meta name="citation_reference" content="citation_title=Incorporating motif analysis into gene co-expression networks reveals novel modular expression pattern and new signaling pathways;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003840" /><meta name="citation_reference" content="citation_title=Unraveling transcriptional control in Arabidopsis using cis-regulatory elements and coexpression networks;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=150;  citation_pages=535-46; " /><meta name="citation_reference" content="citation_title=Identification of cis-acting promoter elements in cold- and dehydration-induced transcriptional pathways in Arabidopsis, Rice, and Soybean;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=37-49; " /><meta name="citation_reference" content="citation_title=Thousands of cis-regulatory sequence combinations are shared by Arabidopsis and Poplar;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=158;  citation_pages=145-55; " /><meta name="citation_reference" content="citation_title=Identification of plant promoter constituents by analysis of local distribution of short sequences;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=67" /><meta name="citation_reference" content="citation_title=Clustering of DNA sequences in human promoters;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=1562-74; " /><meta name="citation_reference" content="citation_title=ATTED-II updates: condition-specific gene coexpression to extend coexpression analyses and applications to a broad range of flowering plants. Plant;  citation_journal_title=Cell Physiol;  citation_year=2011;  citation_volume=52;  citation_pages=213-9; " /><meta name="citation_reference" content="citation_title=ATTED-II in 2016: a plant coexpression database towards lineage-specific coexpression;  citation_journal_title=Plant Cell Physiol;  citation_year=2016;  citation_volume=57;  citation_pages=e5" /><meta name="citation_reference" content="citation_title=PlaNet: combined sequence and expression comparisons across plant networks derived from seven species;  citation_journal_title=Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=895-910; " /><meta name="citation_reference" content="citation_title=Network analysis for gene discovery in plant-specialized metabolism;  citation_journal_title=Plant. Cell Environ;  citation_year=2013;  citation_volume=36;  citation_pages=1597-606; " /><meta name="citation_reference" content="citation_title=Network-assisted crop systems genetics: network inference and integrative analysis;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2015;  citation_volume=24;  citation_pages=61-70; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The grapevine expression atlas reveals a deep transcriptome shift driving the entire plant into a maturation program;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=3489-505; " /><meta name="citation_reference" content="citation_title=Integrated network analysis identifies fight-club nodes as a class of hubs encompassing key putative switch genes that induce major transcriptome reprogramming during grapevine development;  citation_journal_title=Plant Cell Online;  citation_year=2014;  citation_volume=26;  citation_pages=4617-35; " /><meta name="citation_reference" content="citation_title=The plasticity of the grapevine berry transcriptome;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=r54" /><meta name="citation_reference" content="citation_title=The grapevine VvibZIPC22 transcription factor is involved in the regulation of flavonoid biosynthesis;  citation_journal_title=J. Exp. Bot;  citation_year=2016;  citation_volume=67;  citation_pages=3509-22; " /><meta name="citation_reference" content="citation_title=The photomorphogenic factors UV-B RECEPTOR 1, ELONGATED HYPOCOTYL 5, and HY5 HOMOLOGUE are part of the UV-B signalling pathway in grapevine and mediate flavonol accumulation in response to the environment;  citation_journal_title=J. Exp. Bot;  citation_year=2016;  citation_volume=67;  citation_pages=5429-5445; " /><meta name="citation_reference" content="citation_title=A systems-oriented analysis of the grapevine R2R3-MYB transcription factor family uncovers new insights into the regulation of stilbene accumulation;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=451-466; " /><meta name="citation_reference" content="citation_title=Transcriptome and metabolite profiling reveals that prolonged drought modulates the phenylpropanoid and terpenoid pathway in white grapes (Vitis vinifera L.);  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=67" /><meta name="citation_reference" content="citation_title=Combined physiological, transcriptome, and cis-regulatory element analyses indicate that key aspects of ripening, metabolism, and transcriptional program in grapes (Vitis vinifera L.) are differentially modulated accordingly to fruit size;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=416" /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=HTSeq a python framework to work with high-throughput sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=31;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=550" /><meta name="citation_reference" content="citation_title=Limma powers differential expression analyses for RNA-sequencing and microarray studies;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=e47" /><meta name="citation_reference" content="citation_title=Finding and evaluating community structure in networks;  citation_journal_title=Phys. Rev. E – Stat. Nonlinear, Soft Matter Phys;  citation_year=2004;  citation_volume=69;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes;  citation_journal_title=Plant J;  citation_year=2004;  citation_volume=37;  citation_pages=914-39; " /><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2498-504; " /><meta name="citation_reference" content="citation_title=DNA free energy-based promoter prediction and comparative analysis of Arabidopsis and Rice genomes;  citation_journal_title=Plant Physiol;  citation_year=2011;  citation_volume=156;  citation_pages=1300-15; " /><meta name="citation_reference" content="citation_title=ARF1, a transcription factor that binds to auxin response elements;  citation_journal_title=Science;  citation_year=1997;  citation_volume=276;  citation_pages=1865-8; " /><meta name="citation_reference" content="citation_title=APETALA2/Ethylene Responsive Factor (AP2/ERF) transcription factors: mediators of stress responses and developmental programs;  citation_journal_title=New Phytol;  citation_year=2013;  citation_volume=199;  citation_pages=639-49; " /><meta name="citation_reference" content="citation_title=bZIP transcription factors in Arabidopsis;  citation_journal_title=Trends Plant Sci;  citation_year=2002;  citation_volume=7;  citation_pages=106-11; " /><meta name="citation_reference" content="citation_title=PIFs: systems integrators in plant development;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=56-78; " /><meta name="citation_reference" content="citation_title=Genome-wide classification and evolutionary analysis of the bHLH family of transcription factors in Arabidopsis, poplar, rice, moss, and algae;  citation_journal_title=Plant Physiol;  citation_year=2010;  citation_volume=153;  citation_pages=1398-412; " /><meta name="citation_reference" content="citation_title=A calmodulin-binding/CGCG box DNA-binding protein family involved in multiple signaling pathways in plants;  citation_journal_title=J. Biol. Chem;  citation_year=2002;  citation_volume=277;  citation_pages=45049-58; " /><meta name="citation_reference" content="citation_title=Genome-wide identification of CCA1 targets uncovers an expanded clock network in Arabidopsis;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2015;  citation_volume=112;  citation_pages=E4802-10; " /><meta name="citation_reference" content="citation_title=The choice between MapMan and Gene ontology for automated gene function prediction in plant science;  citation_journal_title=Front. Genet;  citation_year=2012;  citation_volume=3;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Grape berry ripening delay induced by a pre-véraison NAA treatment is paralleled by a shift in the expression pattern of auxin- and ethylene-related genes;  citation_journal_title=BMC Plant Biol;  citation_year=2012;  citation_volume=12;  citation_pages=185" /><meta name="citation_reference" content="citation_title=Light-regulated transcriptional networks in higher plants;  citation_journal_title=Nat. Rev. Genet;  citation_year=2007;  citation_volume=8;  citation_pages=217-30; " /><meta name="citation_reference" content="citation_title=UV-B photoreceptor-mediated signalling in plants;  citation_journal_title=Trends Plant Sci;  citation_year=2012;  citation_volume=17;  citation_pages=230-7; " /><meta name="citation_reference" content="citation_title=The basic leucine zipper transcription factor ABSCISIC ACID RESPONSE ELEMENT-BINDING FACTOR2 is an important transcriptional regulator of abscisic acid-dependent grape berry ripening processes;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=164;  citation_pages=365-83; " /><meta name="citation_reference" content="citation_title=The basic helix-loop-helix transcription factor MYC1 is involved in the regulation of the flavonoid biosynthesis pathway in grapevine;  citation_journal_title=Mol. Plant;  citation_year=2010;  citation_volume=3;  citation_pages=509-23; " /><meta name="citation_reference" content="citation_title=Transcriptional networks in plant immunity;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=206;  citation_pages=932-47; " /><meta name="citation_reference" content="citation_title=Integrating circadian dynamics with physiological processes in plants;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=598-610; " /><meta name="citation_reference" content="citation_title=Network analysis of the MVA and MEP pathways for isoprenoid synthesis;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2013;  citation_volume=64;  citation_pages=665-700; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of the grapevine stilbene synthase multigenic family: genomic organization and expression profiles upon biotic and abiotic stresses;  citation_journal_title=BMC Plant Biol;  citation_year=2012;  citation_volume=12;  citation_pages=130" /><meta name="citation_reference" content="citation_title=Functional annotation, genome organization and phylogeny of the grapevine (Vitis vinifera) terpene synthase gene family based on genome assembly, FLcDNA cloning, and enzyme assays;  citation_journal_title=BMC Plant Biol;  citation_year=2010;  citation_volume=10;  citation_pages=226" /><meta name="citation_reference" content="citation_title=Circadian oscillatory transcriptional programs in grapevine ripening fruits;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=78" /><meta name="citation_reference" content="citation_title=VTCdb: a gene co-expression database for the crop species Vitis vinifera (grapevine);  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=882" /><meta name="citation_reference" content="citation_title=AREB1, AREB2, and ABF3 are master transcription factors that cooperatively regulate ABRE-dependent ABA signaling involved in drought stress tolerance and require ABA for full activation;  citation_journal_title=Plant J;  citation_year=2010;  citation_volume=61;  citation_pages=672-85; " /><meta name="citation_reference" content="citation_title=Abscisic acid: emergence of a core signaling network;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2010;  citation_volume=61;  citation_pages=651-79; " /><meta name="citation_reference" content="citation_title=The homeobox genes ATHB12 and ATHB7 encode potential regulators of growth in response to water deficit in Arabidopsis;  citation_journal_title=Plant Mol. Biol;  citation_year=2004;  citation_volume=55;  citation_pages=663-77; " /><meta name="citation_reference" content="citation_title=ABA-mediated responses to water deficit separate grapevine genotypes by their genetic background;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=91" /><meta name="citation_reference" content="citation_title=Abscisic acid transcriptomic signaling varies with grapevine organ;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=72" /><meta name="citation_reference" content="citation_title=Transcriptomic and metabolomic setworks in the grape berry illustrate that it takes more than flavonoids to fight against ultraviolet radiation;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7;  citation_pages=1337" /><meta name="citation_reference" content="citation_title=Functional differentiation of the glycosyltransferases that contribute to the chemical diversity of bioactive flavonol glycosides in grapevines (Vitis vinifera);  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=2856-71; " /><meta name="citation_reference" content="citation_title=MYBD employed by HY5 increases anthocyanin accumulation via repression of MYBL2 in Arabidopsis;  citation_journal_title=Plant J;  citation_year=2015;  citation_volume=84;  citation_pages=1192-205; " /><meta name="citation_reference" content="citation_title=Transcriptomic analysis of the late stages of grapevine (Vitis vinifera cv. Cabernet Sauvignon) berry ripening reveals significant induction of ethylene signaling and flavor pathways in the skin;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_issue=1;  citation_pages=21" /><meta name="citation_reference" content="citation_title=Grapevine pathogenic microorganisms: understanding infection strategies and host response scenarios;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7;  citation_pages=382" /><meta name="citation_reference" content="citation_title=General detoxification and stress responses are mediated by oxidized lipids through TGA transcription factors in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=768-85; " /><meta name="citation_reference" content="citation_title=The membrane-anchored BOTRYTIS-INDUCED KINASE1 plays distinct roles in Arabidopsis resistance to necrotrophic and biotrophic pathogens;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=257-73; " /><meta name="citation_reference" content="citation_title=The R2R3-MYB transcription factors MYB14 and MYB15 regulate stilbene biosynthesis in Vitis vinifera;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=4135-49; " /><meta name="citation_reference" content="citation_title=Arabidopsis WRKY33 is a key transcriptional regulator of hormonal and metabolic responses toward Botrytis cinerea infection;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=159;  citation_pages=266-85; " /><meta name="citation_reference" content="citation_title=Physical and functional interactions between pathogen-induced Arabidopsis WRKY18, WRKY40, and WRKY60 transcription factors;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1310-26; " /><meta name="citation_reference" content="citation_title=Dual function of Arabidopsis ATAF1 in abiotic and biotic stress responses;  citation_journal_title=Cell Res;  citation_year=2009;  citation_volume=19;  citation_pages=1279-90; " /><meta name="citation_reference" content="citation_title=Hormonal modulation of plant immunity;  citation_journal_title=Annu. Rev. Cell Dev. Biol;  citation_year=2012;  citation_volume=28;  citation_pages=489-521; " /><meta name="citation_reference" content="citation_title=Host cell entry of powdery mildew is correlated with endosomal transport of antagonistically acting VvPEN1 and VvMLO to the papilla;  citation_journal_title=Mol. Plant. Microbe. Interact;  citation_year=2013;  citation_volume=26;  citation_pages=1138-50; " /><meta name="citation_reference" content="citation_title=Asymmetric leaves1 mediates leaf patterning and stem cell function in Arabidopsis;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=967-71; " /><meta name="citation_reference" content="citation_title=SPL8 and miR156-targeted SPL genes redundantly regulate Arabidopsis gynoecium differential patterning;  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=75;  citation_pages=566-77; " /><meta name="citation_reference" content="citation_title=MYB83 is a direct target of SND1 and acts redundantly with MYB46 in the regulation of secondary cell wall biosynthesis in arabidopsis;  citation_journal_title=Plant Cell Physiol;  citation_year=2009;  citation_volume=50;  citation_pages=1950-64; " /><meta name="citation_reference" content="citation_title=Secondary cell walls: biosynthesis, patterned deposition and transcriptional regulation;  citation_journal_title=Plant Cell Physiol;  citation_year=2015;  citation_volume=56;  citation_pages=195-214; " /><meta name="citation_reference" content="citation_title=MYB46 and MYB83 bind to the SMRE sites and directly activate a suite of transcription factors and secondary wall biosynthetic genes;  citation_journal_title=Plant Cell Physiol;  citation_year=2012;  citation_volume=53;  citation_pages=368-80; " /><meta name="citation_reference" content="citation_title=A DNA-binding-site landscape and regulatory network analysis for NAC transcription factors in Arabidopsis thaliana;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=7681-93; " /><meta name="citation_reference" content="citation_title=Biogenesis and homeostasis of chloroplasts and other plastids;  citation_journal_title=Nat. Rev. Mol. Cell Biol;  citation_year=2013;  citation_volume=14;  citation_pages=787-802; " /><meta name="citation_reference" content="citation_title=Tomato GOLDEN2-LIKE transcription factors reveal molecular gradients that function during fruit development and ripening;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=585-601; " /><meta name="citation_reference" content="citation_title=Uniform ripening encodes a Golden 2-like transcription factor regulating tomato fruit chloroplast development;  citation_journal_title=Science;  citation_year=2012;  citation_volume=336;  citation_pages=1711-5; " /><meta name="citation_reference" content="citation_title=Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=215-24; " /><meta name="citation_reference" content="citation_title=Survey of genome sequences in a wild sweet potato, Ipomoea trifida (H. B. K.) G. Don;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=171-9; " /><meta name="citation_reference" content="citation_title=Improvement of barley genome annotations by deciphering the Haruna Nijo genome;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=23;  citation_pages=21-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/311/17743927/dsw061.pdf" /><meta name="description" content="Abstract.  Coordinated transcriptional and metabolic reprogramming ensures a plant’s continued growth and survival under adverse environmental conditions. Trans" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/311/2949522" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/311/2949522" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/311/2949522");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw061';
    </script>
<title>
	Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2949522" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYnOATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV29uZywgRGFycmVuIENoZXJuIEphbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldpbmUgUmVzZWFyY2ggQ2VudHJlLCBVbml2ZXJzaXR5IG9mIEJyaXRpc2ggQ29sdW1iaWEsIFZhbmNvdXZlciwgQkMsIENhbmFkYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxvcGV6IEd1dGllcnJleiwgUm9kcmlnbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldpbmUgUmVzZWFyY2ggQ2VudHJlLCBVbml2ZXJzaXR5IG9mIEJyaXRpc2ggQ29sdW1iaWEsIFZhbmNvdXZlciwgQkMsIENhbmFkYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdhbWJldHRhLCBHcmVnb3J5IEFsYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFY29waHlzaW9sb2dpZSBldCBHw6lub21pcXVlIEZvbmN0aW9ubmVsbGUgZGUgbGEgVmlnbmUsIEJvcmRlYXV4IFNjaWVuY2VzIEFncm8sIElOUkEsIFVuaXZlcnNpdMOpIGRlIEJvcmRlYXV4LCAzMzE0MCBWaWxsZW5hdmUgZOKAmU9ybm9uLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYXN0ZWxsYXJpbiwgU2ltb25lIERpZWdvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2luZSBSZXNlYXJjaCBDZW50cmUsIFVuaXZlcnNpdHkgb2YgQnJpdGlzaCBDb2x1bWJpYSwgVmFuY291dmVyLCBCQywgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBjaXMtcmVndWxhdG9yeSBlbGVtZW50IHN0cnVjdHVyZSBhbmQgZGlzY292ZXJ5IG9mIG1vdGlmLWRyaXZlbiBnZW5lIGNvLWV4cHJlc3Npb24gbmV0d29ya3MgaW4gZ3JhcGV2aW5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50VEZEQiAzLjA6IGEgcG9ydGFsIGZvciB0aGUgZnVuY3Rpb25hbCBhbmQgZXZvbHV0aW9uYXJ5IHN0dWR5IG9mIHBsYW50IHRyYW5zY3JpcHRpb24gZmFjdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMTgyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21pYyBwZXJzcGVjdGl2ZSBvbiBwbGFudCB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQyMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgY2lzLWFjdGluZyByZWd1bGF0b3J5IEROQSBlbGVtZW50cyAoUExBQ0UpIGRhdGFiYXNlOiAxOTk5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5Ny0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFHUklTOiB0aGUgYXJhYmlkb3BzaXMgZ2VuZSByZWd1bGF0b3J5IGluZm9ybWF0aW9uIHNlcnZlciwgYW4gdXBkYXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPUQxMTE4LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEtYmluZGluZyBzcGVjaWZpY2l0aWVzIG9mIHBsYW50IHRyYW5zY3JpcHRpb24gZmFjdG9ycyBhbmQgdGhlaXIgcG90ZW50aWFsIHRvIGRlZmluZSB0YXJnZXQgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0yMzY3LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBuZXR3b3JrcyBpbiBjZWxsdWxhciByZXNwb25zZXMgYW5kIHRvbGVyYW5jZSB0byBkZWh5ZHJhdGlvbiBhbmQgY29sZCBzdHJlc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9NzgxLTgwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaXRjaGhpa2Vy4oCZcyBndWlkZSB0byB0aGUgTUFEUyB3b3JsZCBvZiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV4aW4gcmVzcG9uc2UgZmFjdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQuIENlbGwgRW52aXJvbjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTQtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGlkZW50aWZpY2F0aW9uIG9mIHBvdGVudGlhbCBwbGFudCBFMkYgdGFyZ2V0IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzOTsgIGNpdGF0aW9uX3BhZ2VzPTMxNi0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGludGVyYWN0aW9uIGJldHdlZW4gTVlCIHByb3RlaW5zIGFuZCB0aGVpciB0YXJnZXQgRE5BIGJpbmRpbmcgc2l0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0uIEJpb3BoeXMuIEFjdGEg4oCTIEdlbmUgUmVndWwuIE1lY2g7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTgxOTsgIGNpdGF0aW9uX3BhZ2VzPTY3LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGJhc2lzIGZvciBETkEgYmluZGluZyBzcGVjaWZpY2l0eSBieSB0aGUgYXV4aW4tZGVwZW5kZW50IEFSRiB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9NTc3LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXMtcmVndWxhdG9yeSBjb2RlIG9mIHN0cmVzcy1yZXNwb25zaXZlIHRyYW5zY3JpcHRpb24gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xNDk5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNvcnBvcmF0aW5nIG1vdGlmIGFuYWx5c2lzIGludG8gZ2VuZSBjby1leHByZXNzaW9uIG5ldHdvcmtzIHJldmVhbHMgbm92ZWwgbW9kdWxhciBleHByZXNzaW9uIHBhdHRlcm4gYW5kIG5ldyBzaWduYWxpbmcgcGF0aHdheXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzODQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5yYXZlbGluZyB0cmFuc2NyaXB0aW9uYWwgY29udHJvbCBpbiBBcmFiaWRvcHNpcyB1c2luZyBjaXMtcmVndWxhdG9yeSBlbGVtZW50cyBhbmQgY29leHByZXNzaW9uIG5ldHdvcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTUzNS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY2lzLWFjdGluZyBwcm9tb3RlciBlbGVtZW50cyBpbiBjb2xkLSBhbmQgZGVoeWRyYXRpb24taW5kdWNlZCB0cmFuc2NyaXB0aW9uYWwgcGF0aHdheXMgaW4gQXJhYmlkb3BzaXMsIFJpY2UsIGFuZCBTb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzctNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRob3VzYW5kcyBvZiBjaXMtcmVndWxhdG9yeSBzZXF1ZW5jZSBjb21iaW5hdGlvbnMgYXJlIHNoYXJlZCBieSBBcmFiaWRvcHNpcyBhbmQgUG9wbGFyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE1ODsgIGNpdGF0aW9uX3BhZ2VzPTE0NS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcGxhbnQgcHJvbW90ZXIgY29uc3RpdHVlbnRzIGJ5IGFuYWx5c2lzIG9mIGxvY2FsIGRpc3RyaWJ1dGlvbiBvZiBzaG9ydCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyaW5nIG9mIEROQSBzZXF1ZW5jZXMgaW4gaHVtYW4gcHJvbW90ZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTU2Mi03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVRURUQtSUkgdXBkYXRlczogY29uZGl0aW9uLXNwZWNpZmljIGdlbmUgY29leHByZXNzaW9uIHRvIGV4dGVuZCBjb2V4cHJlc3Npb24gYW5hbHlzZXMgYW5kIGFwcGxpY2F0aW9ucyB0byBhIGJyb2FkIHJhbmdlIG9mIGZsb3dlcmluZyBwbGFudHMuIFBsYW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0yMTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVRURUQtSUkgaW4gMjAxNjogYSBwbGFudCBjb2V4cHJlc3Npb24gZGF0YWJhc2UgdG93YXJkcyBsaW5lYWdlLXNwZWNpZmljIGNvZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9ZTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFOZXQ6IGNvbWJpbmVkIHNlcXVlbmNlIGFuZCBleHByZXNzaW9uIGNvbXBhcmlzb25zIGFjcm9zcyBwbGFudCBuZXR3b3JrcyBkZXJpdmVkIGZyb20gc2V2ZW4gc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTg5NS05MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldHdvcmsgYW5hbHlzaXMgZm9yIGdlbmUgZGlzY292ZXJ5IGluIHBsYW50LXNwZWNpYWxpemVkIG1ldGFib2xpc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50LiBDZWxsIEVudmlyb247ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0xNTk3LTYwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV0d29yay1hc3Npc3RlZCBjcm9wIHN5c3RlbXMgZ2VuZXRpY3M6IG5ldHdvcmsgaW5mZXJlbmNlIGFuZCBpbnRlZ3JhdGl2ZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTYxLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3JhcGV2aW5lIGdlbm9tZSBzZXF1ZW5jZSBzdWdnZXN0cyBhbmNlc3RyYWwgaGV4YXBsb2lkaXphdGlvbiBpbiBtYWpvciBhbmdpb3NwZXJtIHBoeWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDQ5OyAgY2l0YXRpb25fcGFnZXM9NDYzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBncmFwZXZpbmUgZXhwcmVzc2lvbiBhdGxhcyByZXZlYWxzIGEgZGVlcCB0cmFuc2NyaXB0b21lIHNoaWZ0IGRyaXZpbmcgdGhlIGVudGlyZSBwbGFudCBpbnRvIGEgbWF0dXJhdGlvbiBwcm9ncmFtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MzQ4OS01MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0ZWQgbmV0d29yayBhbmFseXNpcyBpZGVudGlmaWVzIGZpZ2h0LWNsdWIgbm9kZXMgYXMgYSBjbGFzcyBvZiBodWJzIGVuY29tcGFzc2luZyBrZXkgcHV0YXRpdmUgc3dpdGNoIGdlbmVzIHRoYXQgaW5kdWNlIG1ham9yIHRyYW5zY3JpcHRvbWUgcmVwcm9ncmFtbWluZyBkdXJpbmcgZ3JhcGV2aW5lIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIE9ubGluZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQ2MTctMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwbGFzdGljaXR5IG9mIHRoZSBncmFwZXZpbmUgYmVycnkgdHJhbnNjcmlwdG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1yNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3JhcGV2aW5lIFZ2aWJaSVBDMjIgdHJhbnNjcmlwdGlvbiBmYWN0b3IgaXMgaW52b2x2ZWQgaW4gdGhlIHJlZ3VsYXRpb24gb2YgZmxhdm9ub2lkIGJpb3N5bnRoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0zNTA5LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGhvdG9tb3JwaG9nZW5pYyBmYWN0b3JzIFVWLUIgUkVDRVBUT1IgMSwgRUxPTkdBVEVEIEhZUE9DT1RZTCA1LCBhbmQgSFk1IEhPTU9MT0dVRSBhcmUgcGFydCBvZiB0aGUgVVYtQiBzaWduYWxsaW5nIHBhdGh3YXkgaW4gZ3JhcGV2aW5lIGFuZCBtZWRpYXRlIGZsYXZvbm9sIGFjY3VtdWxhdGlvbiBpbiByZXNwb25zZSB0byB0aGUgZW52aXJvbm1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9NTQyOS01NDQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN5c3RlbXMtb3JpZW50ZWQgYW5hbHlzaXMgb2YgdGhlIGdyYXBldmluZSBSMlIzLU1ZQiB0cmFuc2NyaXB0aW9uIGZhY3RvciBmYW1pbHkgdW5jb3ZlcnMgbmV3IGluc2lnaHRzIGludG8gdGhlIHJlZ3VsYXRpb24gb2Ygc3RpbGJlbmUgYWNjdW11bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDUxLTQ2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmQgbWV0YWJvbGl0ZSBwcm9maWxpbmcgcmV2ZWFscyB0aGF0IHByb2xvbmdlZCBkcm91Z2h0IG1vZHVsYXRlcyB0aGUgcGhlbnlscHJvcGFub2lkIGFuZCB0ZXJwZW5vaWQgcGF0aHdheSBpbiB3aGl0ZSBncmFwZXMgKFZpdGlzIHZpbmlmZXJhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz02NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbWJpbmVkIHBoeXNpb2xvZ2ljYWwsIHRyYW5zY3JpcHRvbWUsIGFuZCBjaXMtcmVndWxhdG9yeSBlbGVtZW50IGFuYWx5c2VzIGluZGljYXRlIHRoYXQga2V5IGFzcGVjdHMgb2YgcmlwZW5pbmcsIG1ldGFib2xpc20sIGFuZCB0cmFuc2NyaXB0aW9uYWwgcHJvZ3JhbSBpbiBncmFwZXMgKFZpdGlzIHZpbmlmZXJhIEwuKSBhcmUgZGlmZmVyZW50aWFsbHkgbW9kdWxhdGVkIGFjY29yZGluZ2x5IHRvIGZydWl0IHNpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyaW1tb21hdGljOiBhIGZsZXhpYmxlIHRyaW1tZXIgZm9yIGlsbHVtaW5hIHNlcXVlbmNlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjExNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SFRTZXEgYSBweXRob24gZnJhbWV3b3JrIHRvIHdvcmsgd2l0aCBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlcmF0ZWQgZXN0aW1hdGlvbiBvZiBmb2xkIGNoYW5nZSBhbmQgZGlzcGVyc2lvbiBmb3IgUk5BLXNlcSBkYXRhIHdpdGggREVTZXEyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbW1hIHBvd2VycyBkaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBhbmFseXNlcyBmb3IgUk5BLXNlcXVlbmNpbmcgYW5kIG1pY3JvYXJyYXkgc3R1ZGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1lNDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5kaW5nIGFuZCBldmFsdWF0aW5nIGNvbW11bml0eSBzdHJ1Y3R1cmUgaW4gbmV0d29ya3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXMuIFJldi4gRSDigJMgU3RhdC4gTm9ubGluZWFyLCBTb2Z0IE1hdHRlciBQaHlzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFQTUFOOiBhIHVzZXItZHJpdmVuIHRvb2wgdG8gZGlzcGxheSBnZW5vbWljcyBkYXRhIHNldHMgb250byBkaWFncmFtcyBvZiBtZXRhYm9saWMgcGF0aHdheXMgYW5kIG90aGVyIGJpb2xvZ2ljYWwgcHJvY2Vzc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9OTE0LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvc2NhcGU6IGEgc29mdHdhcmUgZW52aXJvbm1lbnQgZm9yIGludGVncmF0ZWQgbW9kZWxzIG9mIGJpb21vbGVjdWxhciBpbnRlcmFjdGlvbiBuZXR3b3JrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0OTgtNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgZnJlZSBlbmVyZ3ktYmFzZWQgcHJvbW90ZXIgcHJlZGljdGlvbiBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgQXJhYmlkb3BzaXMgYW5kIFJpY2UgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0xMzAwLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUkYxLCBhIHRyYW5zY3JpcHRpb24gZmFjdG9yIHRoYXQgYmluZHMgdG8gYXV4aW4gcmVzcG9uc2UgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mjc2OyAgY2l0YXRpb25fcGFnZXM9MTg2NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUEVUQUxBMi9FdGh5bGVuZSBSZXNwb25zaXZlIEZhY3RvciAoQVAyL0VSRikgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOiBtZWRpYXRvcnMgb2Ygc3RyZXNzIHJlc3BvbnNlcyBhbmQgZGV2ZWxvcG1lbnRhbCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTk7ICBjaXRhdGlvbl9wYWdlcz02MzktNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWJaSVAgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBJRnM6IHN5c3RlbXMgaW50ZWdyYXRvcnMgaW4gcGxhbnQgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01Ni03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY2xhc3NpZmljYXRpb24gYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiB0aGUgYkhMSCBmYW1pbHkgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGluIEFyYWJpZG9wc2lzLCBwb3BsYXIsIHJpY2UsIG1vc3MsIGFuZCBhbGdhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTM7ICBjaXRhdGlvbl9wYWdlcz0xMzk4LTQxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjYWxtb2R1bGluLWJpbmRpbmcvQ0dDRyBib3ggRE5BLWJpbmRpbmcgcHJvdGVpbiBmYW1pbHkgaW52b2x2ZWQgaW4gbXVsdGlwbGUgc2lnbmFsaW5nIHBhdGh3YXlzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc7ICBjaXRhdGlvbl9wYWdlcz00NTA0OS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaWRlbnRpZmljYXRpb24gb2YgQ0NBMSB0YXJnZXRzIHVuY292ZXJzIGFuIGV4cGFuZGVkIGNsb2NrIG5ldHdvcmsgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz1FNDgwMi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNob2ljZSBiZXR3ZWVuIE1hcE1hbiBhbmQgR2VuZSBvbnRvbG9neSBmb3IgYXV0b21hdGVkIGdlbmUgZnVuY3Rpb24gcHJlZGljdGlvbiBpbiBwbGFudCBzY2llbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYXBlIGJlcnJ5IHJpcGVuaW5nIGRlbGF5IGluZHVjZWQgYnkgYSBwcmUtdsOpcmFpc29uIE5BQSB0cmVhdG1lbnQgaXMgcGFyYWxsZWxlZCBieSBhIHNoaWZ0IGluIHRoZSBleHByZXNzaW9uIHBhdHRlcm4gb2YgYXV4aW4tIGFuZCBldGh5bGVuZS1yZWxhdGVkIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZ2h0LXJlZ3VsYXRlZCB0cmFuc2NyaXB0aW9uYWwgbmV0d29ya3MgaW4gaGlnaGVyIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMTctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVWLUIgcGhvdG9yZWNlcHRvci1tZWRpYXRlZCBzaWduYWxsaW5nIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTIzMC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFzaWMgbGV1Y2luZSB6aXBwZXIgdHJhbnNjcmlwdGlvbiBmYWN0b3IgQUJTQ0lTSUMgQUNJRCBSRVNQT05TRSBFTEVNRU5ULUJJTkRJTkcgRkFDVE9SMiBpcyBhbiBpbXBvcnRhbnQgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvciBvZiBhYnNjaXNpYyBhY2lkLWRlcGVuZGVudCBncmFwZSBiZXJyeSByaXBlbmluZyBwcm9jZXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTY0OyAgY2l0YXRpb25fcGFnZXM9MzY1LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFzaWMgaGVsaXgtbG9vcC1oZWxpeCB0cmFuc2NyaXB0aW9uIGZhY3RvciBNWUMxIGlzIGludm9sdmVkIGluIHRoZSByZWd1bGF0aW9uIG9mIHRoZSBmbGF2b25vaWQgYmlvc3ludGhlc2lzIHBhdGh3YXkgaW4gZ3JhcGV2aW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01MDktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBuZXR3b3JrcyBpbiBwbGFudCBpbW11bml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDY7ICBjaXRhdGlvbl9wYWdlcz05MzItNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW5nIGNpcmNhZGlhbiBkeW5hbWljcyB3aXRoIHBoeXNpb2xvZ2ljYWwgcHJvY2Vzc2VzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTk4LTYxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV0d29yayBhbmFseXNpcyBvZiB0aGUgTVZBIGFuZCBNRVAgcGF0aHdheXMgZm9yIGlzb3ByZW5vaWQgc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz02NjUtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiB0aGUgZ3JhcGV2aW5lIHN0aWxiZW5lIHN5bnRoYXNlIG11bHRpZ2VuaWMgZmFtaWx5OiBnZW5vbWljIG9yZ2FuaXphdGlvbiBhbmQgZXhwcmVzc2lvbiBwcm9maWxlcyB1cG9uIGJpb3RpYyBhbmQgYWJpb3RpYyBzdHJlc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGFubm90YXRpb24sIGdlbm9tZSBvcmdhbml6YXRpb24gYW5kIHBoeWxvZ2VueSBvZiB0aGUgZ3JhcGV2aW5lIChWaXRpcyB2aW5pZmVyYSkgdGVycGVuZSBzeW50aGFzZSBnZW5lIGZhbWlseSBiYXNlZCBvbiBnZW5vbWUgYXNzZW1ibHksIEZMY0ROQSBjbG9uaW5nLCBhbmQgZW56eW1lIGFzc2F5czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yMjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjYWRpYW4gb3NjaWxsYXRvcnkgdHJhbnNjcmlwdGlvbmFsIHByb2dyYW1zIGluIGdyYXBldmluZSByaXBlbmluZyBmcnVpdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WVENkYjogYSBnZW5lIGNvLWV4cHJlc3Npb24gZGF0YWJhc2UgZm9yIHRoZSBjcm9wIHNwZWNpZXMgVml0aXMgdmluaWZlcmEgKGdyYXBldmluZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTg4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFSRUIxLCBBUkVCMiwgYW5kIEFCRjMgYXJlIG1hc3RlciB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgdGhhdCBjb29wZXJhdGl2ZWx5IHJlZ3VsYXRlIEFCUkUtZGVwZW5kZW50IEFCQSBzaWduYWxpbmcgaW52b2x2ZWQgaW4gZHJvdWdodCBzdHJlc3MgdG9sZXJhbmNlIGFuZCByZXF1aXJlIEFCQSBmb3IgZnVsbCBhY3RpdmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NjcyLTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNjaXNpYyBhY2lkOiBlbWVyZ2VuY2Ugb2YgYSBjb3JlIHNpZ25hbGluZyBuZXR3b3JrOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz02NTEtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBob21lb2JveCBnZW5lcyBBVEhCMTIgYW5kIEFUSEI3IGVuY29kZSBwb3RlbnRpYWwgcmVndWxhdG9ycyBvZiBncm93dGggaW4gcmVzcG9uc2UgdG8gd2F0ZXIgZGVmaWNpdCBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9NjYzLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BQkEtbWVkaWF0ZWQgcmVzcG9uc2VzIHRvIHdhdGVyIGRlZmljaXQgc2VwYXJhdGUgZ3JhcGV2aW5lIGdlbm90eXBlcyBieSB0aGVpciBnZW5ldGljIGJhY2tncm91bmQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9OTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNjaXNpYyBhY2lkIHRyYW5zY3JpcHRvbWljIHNpZ25hbGluZyB2YXJpZXMgd2l0aCBncmFwZXZpbmUgb3JnYW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pYyBhbmQgbWV0YWJvbG9taWMgc2V0d29ya3MgaW4gdGhlIGdyYXBlIGJlcnJ5IGlsbHVzdHJhdGUgdGhhdCBpdCB0YWtlcyBtb3JlIHRoYW4gZmxhdm9ub2lkcyB0byBmaWdodCBhZ2FpbnN0IHVsdHJhdmlvbGV0IHJhZGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTMzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgZGlmZmVyZW50aWF0aW9uIG9mIHRoZSBnbHljb3N5bHRyYW5zZmVyYXNlcyB0aGF0IGNvbnRyaWJ1dGUgdG8gdGhlIGNoZW1pY2FsIGRpdmVyc2l0eSBvZiBiaW9hY3RpdmUgZmxhdm9ub2wgZ2x5Y29zaWRlcyBpbiBncmFwZXZpbmVzIChWaXRpcyB2aW5pZmVyYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yODU2LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NWUJEIGVtcGxveWVkIGJ5IEhZNSBpbmNyZWFzZXMgYW50aG9jeWFuaW4gYWNjdW11bGF0aW9uIHZpYSByZXByZXNzaW9uIG9mIE1ZQkwyIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTE5Mi0yMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWljIGFuYWx5c2lzIG9mIHRoZSBsYXRlIHN0YWdlcyBvZiBncmFwZXZpbmUgKFZpdGlzIHZpbmlmZXJhIGN2LiBDYWJlcm5ldCBTYXV2aWdub24pIGJlcnJ5IHJpcGVuaW5nIHJldmVhbHMgc2lnbmlmaWNhbnQgaW5kdWN0aW9uIG9mIGV0aHlsZW5lIHNpZ25hbGluZyBhbmQgZmxhdm9yIHBhdGh3YXlzIGluIHRoZSBza2luOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX2lzc3VlPTE7ICBjaXRhdGlvbl9wYWdlcz0yMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYXBldmluZSBwYXRob2dlbmljIG1pY3Jvb3JnYW5pc21zOiB1bmRlcnN0YW5kaW5nIGluZmVjdGlvbiBzdHJhdGVnaWVzIGFuZCBob3N0IHJlc3BvbnNlIHNjZW5hcmlvczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhbCBkZXRveGlmaWNhdGlvbiBhbmQgc3RyZXNzIHJlc3BvbnNlcyBhcmUgbWVkaWF0ZWQgYnkgb3hpZGl6ZWQgbGlwaWRzIHRocm91Z2ggVEdBIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTc2OC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1lbWJyYW5lLWFuY2hvcmVkIEJPVFJZVElTLUlORFVDRUQgS0lOQVNFMSBwbGF5cyBkaXN0aW5jdCByb2xlcyBpbiBBcmFiaWRvcHNpcyByZXNpc3RhbmNlIHRvIG5lY3JvdHJvcGhpYyBhbmQgYmlvdHJvcGhpYyBwYXRob2dlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yNTctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSMlIzLU1ZQiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgTVlCMTQgYW5kIE1ZQjE1IHJlZ3VsYXRlIHN0aWxiZW5lIGJpb3N5bnRoZXNpcyBpbiBWaXRpcyB2aW5pZmVyYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTQxMzUtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIFdSS1kzMyBpcyBhIGtleSB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yIG9mIGhvcm1vbmFsIGFuZCBtZXRhYm9saWMgcmVzcG9uc2VzIHRvd2FyZCBCb3RyeXRpcyBjaW5lcmVhIGluZmVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz0yNjYtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeXNpY2FsIGFuZCBmdW5jdGlvbmFsIGludGVyYWN0aW9ucyBiZXR3ZWVuIHBhdGhvZ2VuLWluZHVjZWQgQXJhYmlkb3BzaXMgV1JLWTE4LCBXUktZNDAsIGFuZCBXUktZNjAgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTMxMC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHVhbCBmdW5jdGlvbiBvZiBBcmFiaWRvcHNpcyBBVEFGMSBpbiBhYmlvdGljIGFuZCBiaW90aWMgc3RyZXNzIHJlc3BvbnNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMjc5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3Jtb25hbCBtb2R1bGF0aW9uIG9mIHBsYW50IGltbXVuaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIENlbGwgRGV2LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NDg5LTUyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9zdCBjZWxsIGVudHJ5IG9mIHBvd2RlcnkgbWlsZGV3IGlzIGNvcnJlbGF0ZWQgd2l0aCBlbmRvc29tYWwgdHJhbnNwb3J0IG9mIGFudGFnb25pc3RpY2FsbHkgYWN0aW5nIFZ2UEVOMSBhbmQgVnZNTE8gdG8gdGhlIHBhcGlsbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQuIE1pY3JvYmUuIEludGVyYWN0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTEzOC01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpYyBsZWF2ZXMxIG1lZGlhdGVzIGxlYWYgcGF0dGVybmluZyBhbmQgc3RlbSBjZWxsIGZ1bmN0aW9uIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9OTY3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TUEw4IGFuZCBtaVIxNTYtdGFyZ2V0ZWQgU1BMIGdlbmVzIHJlZHVuZGFudGx5IHJlZ3VsYXRlIEFyYWJpZG9wc2lzIGd5bm9lY2l1bSBkaWZmZXJlbnRpYWwgcGF0dGVybmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTU2Ni03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVlCODMgaXMgYSBkaXJlY3QgdGFyZ2V0IG9mIFNORDEgYW5kIGFjdHMgcmVkdW5kYW50bHkgd2l0aCBNWUI0NiBpbiB0aGUgcmVndWxhdGlvbiBvZiBzZWNvbmRhcnkgY2VsbCB3YWxsIGJpb3N5bnRoZXNpcyBpbiBhcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MTk1MC02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2Vjb25kYXJ5IGNlbGwgd2FsbHM6IGJpb3N5bnRoZXNpcywgcGF0dGVybmVkIGRlcG9zaXRpb24gYW5kIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xOTUtMjE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NWUI0NiBhbmQgTVlCODMgYmluZCB0byB0aGUgU01SRSBzaXRlcyBhbmQgZGlyZWN0bHkgYWN0aXZhdGUgYSBzdWl0ZSBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgYW5kIHNlY29uZGFyeSB3YWxsIGJpb3N5bnRoZXRpYyBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MzY4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEROQS1iaW5kaW5nLXNpdGUgbGFuZHNjYXBlIGFuZCByZWd1bGF0b3J5IG5ldHdvcmsgYW5hbHlzaXMgZm9yIE5BQyB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9NzY4MS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvZ2VuZXNpcyBhbmQgaG9tZW9zdGFzaXMgb2YgY2hsb3JvcGxhc3RzIGFuZCBvdGhlciBwbGFzdGlkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1vbC4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9Nzg3LTgwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9tYXRvIEdPTERFTjItTElLRSB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgcmV2ZWFsIG1vbGVjdWxhciBncmFkaWVudHMgdGhhdCBmdW5jdGlvbiBkdXJpbmcgZnJ1aXQgZGV2ZWxvcG1lbnQgYW5kIHJpcGVuaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg1LTYwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pZm9ybSByaXBlbmluZyBlbmNvZGVzIGEgR29sZGVuIDItbGlrZSB0cmFuc2NyaXB0aW9uIGZhY3RvciByZWd1bGF0aW5nIHRvbWF0byBmcnVpdCBjaGxvcm9wbGFzdCBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzY7ICBjaXRhdGlvbl9wYWdlcz0xNzExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2VtYmx5IG9mIHRoZSBkcmFmdCBnZW5vbWUgb2YgYnVja3doZWF0IGFuZCBpdHMgYXBwbGljYXRpb25zIGluIGlkZW50aWZ5aW5nIGFncm9ub21pY2FsbHkgdXNlZnVsIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjE1LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXJ2ZXkgb2YgZ2Vub21lIHNlcXVlbmNlcyBpbiBhIHdpbGQgc3dlZXQgcG90YXRvLCBJcG9tb2VhIHRyaWZpZGEgKEguIEIuIEsuKSBHLiBEb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZW1lbnQgb2YgYmFybGV5IGdlbm9tZSBhbm5vdGF0aW9ucyBieSBkZWNpcGhlcmluZyB0aGUgSGFydW5hIE5pam8gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzMvMzExLzE3NzQzOTI3L2RzdzA2MS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ29vcmRpbmF0ZWQgdHJhbnNjcmlwdGlvbmFsIGFuZCBtZXRhYm9saWMgcmVwcm9ncmFtbWluZyBlbnN1cmVzIGEgcGxhbnTigJlzIGNvbnRpbnVlZCBncm93dGggYW5kIHN1cnZpdmFsIHVuZGVyIGFkdmVyc2UgZW52aXJvbm1lbnRhbCBjb25kaXRpb25zLiBUcmFucyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjk0OTUyMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSq4FuFZUTNsGd7yYHcWBRjDpq1OA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f311%2f2949522" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f311%2f2949522" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473200">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473200">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473202">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473202">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473207">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473207">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473218">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473218">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473262">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473262">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473264">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473264">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473272">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473272">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/303/2972686">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/327/3737833">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Darren Chern Jan Wong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Wine Research Centre, University of British Columbia, Vancouver, BC, Canada</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this study.</p>
                </div>
                <div class="info-card-note">
                    <a reveal-id="dsw061-cor1" data-open="dsw061-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Darren+Chern+Jan+Wong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wong D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wong D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Darren Chern Jan Wong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rodrigo Lopez Gutierrez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Wine Research Centre, University of British Columbia, Vancouver, BC, Canada</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this study.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rodrigo+Lopez+Gutierrez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lopez Gutierrez R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lopez Gutierrez R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rodrigo Lopez Gutierrez</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gregory Alan Gambetta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Ecophysiologie et Génomique Fonctionnelle de la Vigne, Bordeaux Sciences Agro, INRA, Université de Bordeaux, 33140 Villenave d’Ornon, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gregory+Alan+Gambetta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gambetta G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gambetta G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gregory Alan Gambetta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simone Diego Castellarin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Wine Research Centre, University of British Columbia, Vancouver, BC, Canada</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw061-cor1" data-open="dsw061-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 604 827 2667. Email: <a href="mailto:wongdcj@gmail.com" target="_blank">wongdcj@gmail.com</a> (D.C.J.W.); <a href="mailto:simone.castellarin@ubc.ca" target="_blank">simone.castellarin@ubc.ca</a> (S.D.C.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Simone+Diego+Castellarin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Castellarin S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Castellarin S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Simone Diego Castellarin</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw061-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this study.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw061-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 311–326, <a href='https://doi.org/10.1093/dnares/dsw061'>https://doi.org/10.1093/dnares/dsw061</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 October 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2949522" href="/dnaresearch/article-pdf/24/3/311/17743927/dsw061.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Darren Chern Jan Wong, Rodrigo Lopez Gutierrez, Gregory Alan Gambetta, Simone Diego Castellarin;  Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 311–326, <a href='https://doi.org/10.1093/dnares/dsw061'>https://doi.org/10.1093/dnares/dsw061</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2949522?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20analysis%20of%20cis-regulatory%20element%20structure%20and%20discovery%20of%20motif-driven%20gene%20co-expression%20networks%20in%20grapevine&amp;publicationDate=2017-01-24&amp;volumeNum=24&amp;issueNum=3&amp;author=Wong%2C%20Darren%20Chern%20Jan%3B%20Lopez%20Gutierrez%2C%20Rodrigo&amp;startPage=311&amp;endPage=326&amp;contentId=10.1093%2Fdnares%2Fdsw061&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f311%2f2949522" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473200 id="88473200" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Coordinated transcriptional and metabolic reprogramming ensures a plant’s continued growth and survival under adverse environmental conditions. Transcription factors (TFs) act to modulate gene expression through complex cis-regulatory element (CRE) interactions. Genome-wide analysis of known plant CREs was performed for all currently predicted protein-coding gene promoters in grapevine (<em>Vitis vinifera</em> L.). Many CREs such as abscisic acid (ABA)-responsive, drought-responsive, auxin-responsive, and evening elements, exhibit <em>bona fide</em> CRE properties such as strong position bias towards the transcription start site (TSS) and over-representation when compared with random promoters. Genes containing these CREs are enriched in a large repertoire of plant biological pathways. Large-scale transcriptome analyses also show that these CREs are highly implicated in grapevine development and stress response. Numerous CRE-driven modules in condition-specific gene co-expression networks (GCNs) were identified and many of these modules were highly enriched for plant biological functions. Several modules corroborate known roles of CREs in drought response, pathogen defense, cell wall metabolism, and fruit ripening, whereas others reveal novel functions in plants. Comparisons with Arabidopsis suggest a general conservation in promoter architecture, gene expression dynamics, and GCN structure across species. Systems analyses of CREs provide insights into the grapevine cis-regulatory code and establish a foundation for future genomic studies in grapevine.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene co-expression network&quot;">gene co-expression network</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cis-regulatory element&quot;">cis-regulatory element</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="grapevine">grapevine</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="promoter">promoter</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptional regulation&quot;">transcriptional regulation</a></div>                            <h2 scrollto-destination=88473202 id="88473202" class="section-title" >1. Introduction</h2>
<p>Transcription factors (TFs) regulate the transcription of target genes both spatially and temporally through the specific binding of cis-regulatory elements (CREs or ‘motifs’) present in their promoters. On average 5% of the protein-coding genes encode TFs in monocotyledon and dicotyledonous plants, which can be divided into super families, and further into families, based on their DNA-binding domains.<a reveal-id="dsw061-B1" data-open="dsw061-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> To date, 58 families have been assigned for plants,<a reveal-id="dsw061-B1" data-open="dsw061-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> some of them are grouped in six TF super families. Many of these TF are essential for plant cell division, development, growth, and responses to the environment.<a reveal-id="dsw061-B2" data-open="dsw061-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In the last decade, ongoing efforts in curating reported plant CREs<a reveal-id="dsw061-B3" data-open="dsw061-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw061-B4" data-open="dsw061-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> have been fruitful, providing useful clues into the role of TFs in regulating specific promoter sequences of interest. There have been recent efforts to elucidate the TF-binding specificity in Arabidopsis at a genome-wide scale using <em>in vitro</em> techniques coupled to high-throughput platforms such as protein binding microarrays.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> This strategy provided valuable CRE information for a large number of TFs (63 in total) covering 25 TF families. The most widely studied CREs in plants include the ABRE (abscisic acid (ABA)-responsive element) and DRE (dehydration-responsive element)/CRT (C-repeat) elements which are largely implicated in stress responses,<a reveal-id="dsw061-B6" data-open="dsw061-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> CArG-box,<a reveal-id="dsw061-B7" data-open="dsw061-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> AuxRE,<a reveal-id="dsw061-B8" data-open="dsw061-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and E2F-binding elements<a reveal-id="dsw061-B9" data-open="dsw061-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> in plant development, and MBS elements implicated in a myriad of development processes and stress responses.<a reveal-id="dsw061-B10" data-open="dsw061-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Others CREs that belong to lesser known TF families (e.g. Golden2-like, GLK)<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> or that are critical for the binding of well characterized TFs (e.g. AUXIN RESPONSE FACTOR, ARF)<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B11" data-open="dsw061-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> have been recently characterized as well.</p><p>Knowledge pertaining to TF and CRE repertoire has been instrumental in developing regulatory models explaining changes in gene expression patterns across organs, tissues, or in response to stress in Arabidopsis<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B12 dsw061-B13 dsw061-B14" data-open="dsw061-B12 dsw061-B13 dsw061-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> and some crop species.<a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw061-B16" data-open="dsw061-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Approaches that consider the absence/presence of CREs, the distribution of CRE positions, or their over-representation (ratio of observed/random expectation) have all been used to establish transcriptional regulatory models in plants<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw061-B17" data-open="dsw061-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and mammals.<a reveal-id="dsw061-B18" data-open="dsw061-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Many of these studies have highlighted that CREs having a position bias towards the transcription start site (TSS) are more likely to be <em>bona fide</em> CREs functioning in important biological roles. Other sophisticated models that consider CRE combinatorial rules<a reveal-id="dsw061-B12" data-open="dsw061-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw061-B16" data-open="dsw061-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> or its integration with gene co-expression networks (GCNs)<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B12 dsw061-B13 dsw061-B14" data-open="dsw061-B12 dsw061-B13 dsw061-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> have also been favored. In GCNs, genes are represented as nodes, and edges connecting any two nodes depict similarity in expression profile across different organs/tissues, developmental stages, and/or stress conditions. GCNs can be further classified into ‘condition-independent’ or ‘condition-specific’ GCNs, representing relationships inferred from multiple experimental conditions or in determined conditions (e.g. abiotic stress, biotic stress, or tissue-specificity), respectively.<a reveal-id="dsw061-B19" data-open="dsw061-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Lineage-specific GCNs have also been explored.<a reveal-id="dsw061-B20" data-open="dsw061-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw061-B21" data-open="dsw061-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Systems-based approaches, such as the integration of CREs, promoter architecture, and GCNs analysis, are promising methodologies to infer gene function,<a reveal-id="dsw061-B22" data-open="dsw061-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and are complementary to other network inference methods such as associologs—orthologous gene associations—and genome context similarities.<a reveal-id="dsw061-B23" data-open="dsw061-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>Grapes (<em>Vitis vinifera</em> L.) are amongst the world’s major fruit crops and are highly valued for fresh fruit consumption and winemaking. The grapevine genome<a reveal-id="dsw061-B24" data-open="dsw061-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> provides a strong foundation for elucidating the molecular genetics involved in grapevine physiology and response to environment. Specifically, seminal studies involving large-scale gene expression profiling have revealed important mechanisms and biomarkers critical for grapevine development across organs/tissues<a reveal-id="dsw061-B25" data-open="dsw061-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw061-B26" data-open="dsw061-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and berry phenotypic plasticity (e.g. genotype × environment interactions).<a reveal-id="dsw061-B27" data-open="dsw061-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The availability of the grapevine genome and the rapid accumulation of publicly available grapevine whole-genome microarray and RNA-seq datasets provide the raw material necessary for GCN and CRE analyses. To date, several studies have utilized GCN and CRE analyses in revealing biologically relevant target genes of grapevine TFs<a reveal-id="dsw061-B28 dsw061-B29 dsw061-B30" data-open="dsw061-B28 dsw061-B29 dsw061-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> and/or providing clues into the regulatory relationships of TFs in key metabolic and ripening pathways.<a reveal-id="dsw061-B31" data-open="dsw061-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw061-B32" data-open="dsw061-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> However, to date no study has characterized grapevine promoter architecture and identified putative CRE regulatory modules at a genome-wide scale.</p><p>In this study, the distribution and organization of promoter CREs and their properties were analysed for all currently predicted protein-coding genes in grapevine. Further insights into the biological role(s) of genes containing these CREs were achieved through analysis of enriched plant biological pathways. Meta-analysis of next-generation sequencing (RNA-seq) datasets and the grapevine tissue developmental atlas revealed the importance of many CREs in stress and developmental regulation. A <em>bottom-up</em> strategy was adopted to identify regulatory modules potentially driven by CREs in both stress- and development-specific GCNs. We apply a robust approach by integrating different layers of information—promoter architecture, gene expression, and GCNs—to identify the cis-regulatory code during environment stress, growth, and/or development in grapevine. The insights of this study will allow the prioritization of CREs and gene candidates for functional characterization in grapevines.</p>                            <h2 scrollto-destination=88473207 id="88473207" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473208 id="88473208" class="section-title" >2.1. Discovery of known plant CREs in grapevine promoters</h3>
<p>Candidate plant CREs, were compiled from two plant promoter databases, namely AGRIS<a reveal-id="dsw061-B4" data-open="dsw061-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and PLACE,<a reveal-id="dsw061-B3" data-open="dsw061-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and from protein-binding microarray inferred CREs of 25 Arabidopsis TF families.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> A total of 559 non-redundant CREs were retained for subsequent analysis. The promoters of the grapevine genes were downloaded from EnsemblPlants using Biomart (<a class="link link-uri" href="http://plants.ensembl.org/index.html" target="_blank">http://plants.ensembl.org/index.html</a>; 1 March 2016, date last accessed). Sequences between -1 to -1,000 bp upstream of each gene TSS containing &lt; 10% N sequences were deemed as candidate promoter sequence of each gene. All 559 non-redundant CREs were scanned for exact pattern match (no mismatch) across 29,839 promoters on both + and – strands, and indexed for the number of occurrence, strand, and position information.</p>                            <h3 scrollto-destination=88473210 id="88473210" class="section-title" >2.2. Positional bias and fold-change enrichment analysis of CREs in grapevine promoters</h3>
<p>Each CRE was evaluated for positional bias using two popular approaches; the clustering factor (<em>CF</em>),<a reveal-id="dsw061-B18" data-open="dsw061-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and a recently proposed index, the <em>Z-</em><em>score</em>.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Another approach based on the fold-change/enrichment between observed/expected frequencies with resampling was considered.<a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> For the <em>CF</em> calculation, the number of occurrences of each CRE in each bin (50 bases, 20 bins total) was calculated. Based on the abundance of the CRE in the 20 bins, the mean (<em>x̄</em>) and standard deviation (<em>σ</em>) were calculated. A second mean (<em>x̄</em> ʹ) and a second standard deviation (<em>σ</em>ʹ) were calculated based only on bin values that were &lt; 2<em>σ</em> above <em>x̄</em>. Then the <em>CF</em> is calculated with the equation: <em>CF</em> = (<em>x</em><sub>max</sub> - <em>x̄</em> ′)/(<em>σ</em>′). The <em>Z-</em><em>score</em>,<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> for a given CRE was calculated considering every gene in the mutual rank (MR) network(s). The <em>Z-score</em> for each CRE is calculated with the equation: <em>Z-score</em><sub> </sub>= (<em>L</em>/2 + <em>p</em>)/√[((<em>L</em> - <em>l </em>+<em> </em>1) ^2 − 1)/<em>n</em>], where <em>L</em> corresponds to the length of the promoter, <em>l</em> corresponds to the length of the motif, <em>n</em> corresponds the number of instances that the CRE is found in the promoters of the genes, and <em>p</em> corresponds to the mean position from all the instances the CRE was found. For the fold-change/enrichment (<em>FE</em>) calculation, the observed frequency is calculated as in the <em>Z-score</em>, based on the number of instances that the CRE was found in the promoters, and the expected frequency was calculated by performing 1,000 replicates of random sampling with equal sizes (number of genes containing the selected CRE).</p>                            <h3 scrollto-destination=88473212 id="88473212" class="section-title" >2.3. Compilation of transcriptome datasets, differential expression (DE), and comparative gene expression analysis</h3>
<p>Two different compilations of databases were made: a stress-related compendia, which was mainly derived from next-generation sequencing (RNA-seq) datasets publicly available from NCBI Sequence Read Archive (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra" target="_blank">http://www.ncbi.nlm.nih.gov/sra</a>; last accessed 01/03/2016), and a grapevine tissue developmental series (atlas), based on the NimbleGen Grape Whole-genome microarray platform. For RNA-seq datasets, quality control of raw single- and pair-end FASTQ reads were performed with <em>Trimmomatic</em> v0.36,<a reveal-id="dsw061-B33" data-open="dsw061-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> ensuring all reads maintained a minimum <em>Q</em> score of 20 and a length of 40. <em>Bowtie2</em> v2.2.7<a reveal-id="dsw061-B34" data-open="dsw061-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was used with default parameters for read alignments using the 12× grapevine reference genome PN40024.<a reveal-id="dsw061-B24" data-open="dsw061-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Read summarization was performed with htseq-count v0.6.1 using the 12x V1 annotation with default settings.<a reveal-id="dsw061-B35" data-open="dsw061-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><em>DESeq2</em><a reveal-id="dsw061-B36" data-open="dsw061-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> was used for the DE analysis of the stress (RNA-seq) datasets, comparing the experiments’ treatments against controls. For the grapevine <em>atlas</em> dataset, DE analysis was performed using <em>limma</em><a reveal-id="dsw061-B37" data-open="dsw061-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> comparing the development progression from young to mature of each organ, except flower-, seedling-, and root-specific tissues. A threshold of false discovery rate (FDR) &lt; 0.05 and absolute log2FC &gt; 1 represents significant DE genes identified through <em>DESeq2</em> and <em>limma</em> in each comparison. Estimation of RNA-seq derived transcript abundance was performed with <em>DESeq2</em> by applying variance-stabilizing transformation (VST) while pre-normalized probe intensities (using Robust Multi-Array Average), were gathered from.<a reveal-id="dsw061-B25" data-open="dsw061-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> All predicted grapevine-Arabidopsis orthologues were downloaded from EnsemblPlants (<a class="link link-uri" href="http://plants.ensembl.org/index.html" target="_blank">http://plants.ensembl.org/index.html</a>; 1 March 2016, date last accessed). Expression analysis of Arabidopsis genes was conducted using the AtGenExpress compendium for the ‘stress’ and ‘pathogen’ series via the Bio-Analytic Resource for Plant biology database (<a class="link link-uri" href="http://bar.utoronto.ca/" target="_blank">http://bar.utoronto.ca/</a>; 1 December 2016, date last accessed) expression browser tool.</p>                            <h3 scrollto-destination=88473214 id="88473214" class="section-title" >2.4. Construction of GCNs</h3>
<p>Prior to network construction, genes that are not DE in at least one comparison in each compendium were filtered. A final non-redundant list of DE genes in the stress-responsive and atlas compendia were obtained and used for GCN construction. Principal component analysis was performed on the RNA-seq VST dataset using R (<a class="link link-uri" href="http://www.r-project.org" target="_blank">http://www.r-project.org</a>). Pearson’s correlation coefficient (PCC) values were used as a measurement of expression similarity between gene pairs. Ranks were assigned to gene pairs based on the PCC values of their genes. The formula for geometric mean of the ranks, MR<sub>(AB)</sub> = √(Rank(A→B) × Rank(B→A)), was used to calculate the MR value for each pair-wise genes.<a reveal-id="dsw061-B19" data-open="dsw061-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsw061-B20" data-open="dsw061-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Rank(A→B) corresponds to the rank assigned to gene B given the list of co-expression genes from gene A, while vice versa for Rank(B→A). For each individual network (stress and atlas), the significance of MR values was estimated based on 1,000 permutations as per.<a reveal-id="dsw061-B21" data-open="dsw061-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p>                            <h3 scrollto-destination=88473216 id="88473216" class="section-title" >2.5. Network-based CRE subnetwork selection and functional enrichment analysis</h3>
<p>For each gene and all immediate connected genes (nodes) forming a group, enrichment <em>P</em>-values for CREs in their promoters were based on hypergeometric distribution.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> From each GCN with significant MR values, genes having a given CRE enriched at <em>P</em> &lt; 0.01 were then selected as ‘seed’ or ‘guide’ genes to generate the relevant CRE network. For the construction of Arabidopsis CRE network (<em>P</em> &lt; 0.01), the Arabidopsis graphical Gaussian model gene network with a partial correlation threshold &gt; 0.05 was used.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> To further identify densely connected regions (subnetworks) within each CRE network, the Girvan–Newman fast greedy algorithm was applied.<a reveal-id="dsw061-B38" data-open="dsw061-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Functional enrichment of MapMan<a reveal-id="dsw061-B39" data-open="dsw061-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> ontology terms for grapevine and Arabidopsis was performed in R and terms with an FDR &lt; 0.05 were considered significantly enriched. In each CRE subnetwork, genes within selected ‘seed’ or ‘guide’ gene(s) node vicinity of up to two steps (<em>k </em>=<em> </em>2) were extracted for further analyses. All visualization of various CRE subnetworks was performed with Cytoscape v3.4.<a reveal-id="dsw061-B40" data-open="dsw061-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h2 scrollto-destination=88473218 id="88473218" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=88473219 id="88473219" class="section-title" >3.1. Distribution of CREs in grapevine promoters</h3>
<p>An evaluation of base composition is often the first step in characterizing promoter sequences of an organism. In this study, analysis of promoter sequences (1,000 bp upstream of the TSS) revealed an AT-rich ratio, where A:C:G:T composition are 0.33:0.16:0.16:0.34, respectively. Promoter sequences of Arabidopsis, soybean, and rice are also AT-rich<a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which is a strong indicator of <em>bona fide</em> plant promoter regions.<a reveal-id="dsw061-B41" data-open="dsw061-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Therefore, promoter sequences extracted here likely reflect actual promoter regions and not random genomic regions. Rigorous statistical and DNA-binding specificity evaluation have shown that CREs between 6- and 8-mer provide an optimal threshold for capturing most biologically relevant CREs.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B16" data-open="dsw061-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw061-B17" data-open="dsw061-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Indeed, the occurrence of CREs, shorter than 6-mer or larger than 8-mer, in the promoters of the grapevine genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File 1</a></span>) might result in a large or small number of hits, respectively, complicating the statistical evaluation for the identification of biologically relevant CREs (<a reveal-id="dsw061-F1" data-open="dsw061-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Focussing on promoter sequences containing CREs between 6- and 8-mer, three separate approaches were considered to identify biologically relevant CREs, fulfilling a positional bias towards the TSS (using <em>CF</em> and <em>Z-score</em>) and/or having high fold enrichment when compared with random occurrences in shuffled promoter sequences (using <em>FE</em>). CREs were seen to score differently according to each of the three approaches (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                        <a id="88473221" scrollto-destination="88473221"></a>
<div id="dsw061-F1" data-id="dsw061-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f1.png?Expires=2147483647&Signature=kKCG01hGZy8W4biiYj5Y1QK5mhYKyjnSYRlRfvF-2EY92GpGk-biEXHXK4uqp7RC~DBCbo9Hi~Pdg~YuI7XGftXngEc9iE0M7tipe8JsaJxh1PZCYpgKj~Ee2XcbQtXxH8DEoubiyhaOA9Qqn3XjluoRcS69HufP6mo2HMmv4xUoj2ijxGvWSt6n-9t6vwjkwa~zBXiRDXKn7ebE0apf4OL49TFZ6p2kTfWxDjXTMZ30EB0qJjSafjlcRCrtcbdqB2hYO2IkgpycLaE~8fndBdknLFuhXA5z3PrYbVZyY~Kam8YVnxvZvUM2rij6eRnP0OeL7IeRP3ocew-WZWHDGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of selected CREs in grapevine promoter sequences. CREs of each size category (4–12-mer) were ranked according to their Z-score values in descending order. The frequencies of each CRE are binned into 50 bp intervals from the TSS (0 bp) to –1,000 bp. High and low frequencies of each CRE are depicted as red/light gray and blue/dark gray color shades, respectively. Examples of CREs and their commonly associated names are indicated in brackets. TSS, transcription start site." path-from-xml="dsw061f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473221/dsw061f1.tif" path-from-xml="dsw061f1.tif" target="_blank">View large</a><a section="88473221" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f1.png?Expires=2147483647&Signature=P8buqVcrJ6Mo3epdGV-l8C5NbyYYXFOOVD8ZUVzvo0dTI9Ajh0TC764o5mXEGhtVB71i-mzHDOLpvQOENpt0p4cLd~m6C5P53Lo9slNcO-UmooNDRFNOZhAblOr6WTXTUhwWT22WCULyXi5oXqGHs5u4AHDJ38OKA-i3sYuO1~pNgJ6BhEYXsDHwYYiVBEGX0luT3OGnc4fLN7~Q23opT-k8dzt3zW-bh~owa16WLdSNDtOJbJ8Jz5PYiJO7b6CIevw7TlL9iGsL~V9RghOIm7lkZZaTWSQdeELI7U4MYG9MY3KjD6GLeAIshfOOkOPpcyj0adOBYSwaFOWj41sOnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473221&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of selected CREs in grapevine promoter sequences. CREs of each size category (4–12-mer) were ranked according to their <em>Z-score</em> values in descending order. The frequencies of each CRE are binned into 50 bp intervals from the TSS (0 bp) to –1,000 bp. High and low frequencies of each CRE are depicted as red/light gray and blue/dark gray color shades, respectively. Examples of CREs and their commonly associated names are indicated in brackets. TSS, transcription start site.</p></div></div><div content-id="dsw061-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f1.png?Expires=2147483647&Signature=kKCG01hGZy8W4biiYj5Y1QK5mhYKyjnSYRlRfvF-2EY92GpGk-biEXHXK4uqp7RC~DBCbo9Hi~Pdg~YuI7XGftXngEc9iE0M7tipe8JsaJxh1PZCYpgKj~Ee2XcbQtXxH8DEoubiyhaOA9Qqn3XjluoRcS69HufP6mo2HMmv4xUoj2ijxGvWSt6n-9t6vwjkwa~zBXiRDXKn7ebE0apf4OL49TFZ6p2kTfWxDjXTMZ30EB0qJjSafjlcRCrtcbdqB2hYO2IkgpycLaE~8fndBdknLFuhXA5z3PrYbVZyY~Kam8YVnxvZvUM2rij6eRnP0OeL7IeRP3ocew-WZWHDGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Distribution of selected CREs in grapevine promoter sequences. CREs of each size category (4–12-mer) were ranked according to their Z-score values in descending order. The frequencies of each CRE are binned into 50 bp intervals from the TSS (0 bp) to –1,000 bp. High and low frequencies of each CRE are depicted as red/light gray and blue/dark gray color shades, respectively. Examples of CREs and their commonly associated names are indicated in brackets. TSS, transcription start site." path-from-xml="dsw061f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473221/dsw061f1.tif" path-from-xml="dsw061f1.tif" target="_blank">View large</a><a section="88473221" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f1.png?Expires=2147483647&Signature=P8buqVcrJ6Mo3epdGV-l8C5NbyYYXFOOVD8ZUVzvo0dTI9Ajh0TC764o5mXEGhtVB71i-mzHDOLpvQOENpt0p4cLd~m6C5P53Lo9slNcO-UmooNDRFNOZhAblOr6WTXTUhwWT22WCULyXi5oXqGHs5u4AHDJ38OKA-i3sYuO1~pNgJ6BhEYXsDHwYYiVBEGX0luT3OGnc4fLN7~Q23opT-k8dzt3zW-bh~owa16WLdSNDtOJbJ8Jz5PYiJO7b6CIevw7TlL9iGsL~V9RghOIm7lkZZaTWSQdeELI7U4MYG9MY3KjD6GLeAIshfOOkOPpcyj0adOBYSwaFOWj41sOnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473221&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Distribution of selected CREs in grapevine promoter sequences. CREs of each size category (4–12-mer) were ranked according to their <em>Z-score</em> values in descending order. The frequencies of each CRE are binned into 50 bp intervals from the TSS (0 bp) to –1,000 bp. High and low frequencies of each CRE are depicted as red/light gray and blue/dark gray color shades, respectively. Examples of CREs and their commonly associated names are indicated in brackets. TSS, transcription start site.</p></div></div><p>Several examples 6-mer CREs that are canonical TF binding sites for auxin responsive factor (AuxRE<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B11" data-open="dsw061-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw061-B42" data-open="dsw061-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>: TGTCTC and TGTCGG), AP2/ERF superfamily (DRE<a reveal-id="dsw061-B43" data-open="dsw061-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a>: RCCGAC and GCCGAC; and GCC-box<a reveal-id="dsw061-B43" data-open="dsw061-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a>: GCCGCC), bZIP<a reveal-id="dsw061-B44" data-open="dsw061-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> (G-box: CACGTG; and C-box: GACGTC), bHLH<a reveal-id="dsw061-B45" data-open="dsw061-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsw061-B46" data-open="dsw061-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> (G-box), and CAMTA (CGCGBOXAT<a reveal-id="dsw061-B47" data-open="dsw061-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>: VCGCGB) TFs showed strong position bias towards the TSS and/or were strongly over-represented. The AuxRE TGTCTC CRE had a <em>CF</em> (highest) and <em>Z-score</em> score of 5.8, was present in 9,111 promoters (<em>n</em> = 11,621 occurrences) with a peak position between 0 and 50 bp from the TSS, and had a <em>FE</em> value showing 3.3-fold more observed occurrences compared with occurrences in random promoters (<a reveal-id="dsw061-F1" data-open="dsw061-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). A novel AuxRE, TGTCGG, which was recently shown to be the preferred binding site for ARFs compared with TGTCTC<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B11" data-open="dsw061-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> has a slightly different distribution profile. TGTCGG is present in fewer (4,411) promoters, with a different peak position (50–100 bp upstream of TSS), lower <em>CF</em> and <em>Z-score</em>, but with higher <em>FE</em> values (6.8-fold more occurrences than in random promoters). Commonly associated bZIP and bHLH binding site motif, G-box (CACGTG), was present in 6,200 promoters (<em>n</em> =7,956 occurrences) with a peak position between 50 and 100 bp from the TSS, and <em>CF</em>, <em>Z-score</em>, and <em>FE</em>, of 2.9, 6.1, and 4.8 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Meanwhile, the GCCGAC CRE, a component of DRE, was among the CREs with the highest <em>FE</em> (12.9-fold more occurrences than in random promoters), was present in 2,312 promoters (<em>n </em>=<em> </em>2,529 occurrences), with the same peak position as G-box, with <em>CF</em> and <em>Z-score</em> of 2.9 and 3.9, respectively.</p><p>Considering 8-mer CREs, several CREs potentially targeted by E2F TF<a reveal-id="dsw061-B9" data-open="dsw061-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> including the GCGGGAAA (E2F1OSPCNA) and TTTCCCGC (E2FANTRNR), present in 347 (<em>n</em>= 360 occurrences) and 410 (<em>n</em>= 418 occurrences) promoters were ranked highly considering <em>CF</em>, <em>Z-scores</em>, and <em>FE</em> criterions. In addition, several longer instances of G-box (e.g. CACGTGGC and CCACGTGG), and other instances of bZIP TFs<a reveal-id="dsw061-B44" data-open="dsw061-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> (e.g. TACGTGTC) were also highly ranked. Among the top-scoring 8-mer CREs for <em>CF</em> is the AGATATTT sequence, a circadian clock associated CRE, recognized by Arabidopsis MYB-related TF, CCA1, and RVE1.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B48" data-open="dsw061-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> There were 4,120 promoters containing AGATATTT (<em>n </em>=<em> </em>4,567 occurrences) with a peak position between 0 and 50 bp from the TSS, and a <em>FE</em> value of 7.2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p><p>Top-scoring CREs in grapevine promoters, such as the G-box, CGCGBOXAT, DRE, GCC-box, E2F1OSPCNA, and AGATATTT, share similar promoter distribution profiles as in the promoters of other species; having a peak position within 200 bp upstream of TSS,<a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw061-B17" data-open="dsw061-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> a high <em>Z-score</em> (&gt;3)<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and/or <em>CF</em>, or are highly over-represented compared with random promoters<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> reaffirming some extent of conservation in plant promoter architecture and CRE distribution.<a reveal-id="dsw061-B14" data-open="dsw061-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw061-B16" data-open="dsw061-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw061-B17" data-open="dsw061-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Several exceptions exist where well-characterized plant CREs, such as CArG box (CC[A/T]<sub>6</sub>GG) and A-box (TACGTA) recognized by MADS TFs<a reveal-id="dsw061-B7" data-open="dsw061-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and bZIP<a reveal-id="dsw061-B44" data-open="dsw061-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> TFs, respectively (<a reveal-id="dsw061-F1" data-open="dsw061-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), do not possess authentic CRE properties. This observation does not discount these CREs’ involvement in the regulation of plant biological functions, showing the need for additional scanning approaches or other statistical models<a reveal-id="dsw061-B12" data-open="dsw061-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> in order to capture the full spectrum of functional CREs in grapevine.</p>                            <h3 scrollto-destination=88473225 id="88473225" class="section-title" >3.2. Enriched biological pathways support the relevance of CREs in grapevine promoters</h3>
<p>To ascertain the biological roles of genes whose promoters contain these CREs, functional enrichment analysis has been widely adopted.<a reveal-id="dsw061-B14" data-open="dsw061-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw061-B16" data-open="dsw061-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> In this study, the MapMan ontology,<a reveal-id="dsw061-B39" data-open="dsw061-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> tailored specifically for plants, was adopted for the identification of enriched biological pathways due to its improve gene function prediction performance compared with gene ontology.<a reveal-id="dsw061-B49" data-open="dsw061-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The evaluation of all compiled CREs was performed and summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. Emphasis is given to those which scored highly across the <em>CF</em>, <em>Z-score</em>, or <em>FE</em> (<a reveal-id="dsw061-F2" data-open="dsw061-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).</p>                        <a id="88473227" scrollto-destination="88473227"></a>
<div id="dsw061-F2" data-id="dsw061-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f2.png?Expires=2147483647&Signature=CgPYuAzEn6J2r5VjPxyDo54m0q2cdhRZGe4VgPqNUZXLLEQp0v~vwRj3tiUXQvXG4GzjDgZRYA484TYQfjVluah3I5lZCoBuq8OtmljUZka0HtH61yhOIgevbNqfXi6Bx54VgDrYDDU1t00n-HjgrSAN3uJ7eD-rOyel5k3B~hbsf~zXLmftdstizb7I08F2WDG8Ptmq4~vJyYs4EqmJNr9UTxcTtkrJv7x7InhJaX4YkR5end5lx8tZD4MsF1ajYZcUqmNBeaepFIzf~3DFCyPtlba-F6jbnmMpObSCwv1YuwMyZ5ySKjqqBV0vMivGBsDKIXjyM8QQHqM5IUmP4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Enriched functional categories of genes containing selected 6-mer and 8-mer CREs in their promoter sequences. The opacity of the dot corresponds to –log10 FDR functional category enrichment values. The size of the dots corresponds to the number of genes (log10 scaled) whose promoters contain the selected CRE. Commonly associated names for CREs are indicated in brackets." path-from-xml="dsw061f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473227/dsw061f2.tif" path-from-xml="dsw061f2.tif" target="_blank">View large</a><a section="88473227" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f2.png?Expires=2147483647&Signature=cy4on4jcvD4FBGoRv0vz854~ZUUtBUEzloehnADSuG2PzD8ULeO51w52sqpJMVV13bbOj9WvfE5x05WN7MTzzw4iOwV8CJhYGq30VduZQgypkf3jJ3KMlN5~CkxAAgNsdQx79FYQIjGWNMXq26N3ozIDoodMRJmLMGoWKWbwB-8rh4RhBut-AwYBO1j3ObDgp-Z6r275Xo9SKCrUfvsOPRsAEr4pPlZZXDyYIdliaes5vJF1HuaFeDstcXaEQX1jCNDwHLfp7mQGEFa3iPn3BJDmiCj0HpRjl6~ekWqjrcztQD2~h21-4wj5m46Ia7T9sjYEO7lYrwPs80tjLR1r~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473227&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Enriched functional categories of genes containing selected 6-mer and 8-mer CREs in their promoter sequences. The opacity of the dot corresponds to –log10 FDR functional category enrichment values. The size of the dots corresponds to the number of genes (log10 scaled) whose promoters contain the selected CRE. Commonly associated names for CREs are indicated in brackets.</p></div></div><div content-id="dsw061-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f2.png?Expires=2147483647&Signature=CgPYuAzEn6J2r5VjPxyDo54m0q2cdhRZGe4VgPqNUZXLLEQp0v~vwRj3tiUXQvXG4GzjDgZRYA484TYQfjVluah3I5lZCoBuq8OtmljUZka0HtH61yhOIgevbNqfXi6Bx54VgDrYDDU1t00n-HjgrSAN3uJ7eD-rOyel5k3B~hbsf~zXLmftdstizb7I08F2WDG8Ptmq4~vJyYs4EqmJNr9UTxcTtkrJv7x7InhJaX4YkR5end5lx8tZD4MsF1ajYZcUqmNBeaepFIzf~3DFCyPtlba-F6jbnmMpObSCwv1YuwMyZ5ySKjqqBV0vMivGBsDKIXjyM8QQHqM5IUmP4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Enriched functional categories of genes containing selected 6-mer and 8-mer CREs in their promoter sequences. The opacity of the dot corresponds to –log10 FDR functional category enrichment values. The size of the dots corresponds to the number of genes (log10 scaled) whose promoters contain the selected CRE. Commonly associated names for CREs are indicated in brackets." path-from-xml="dsw061f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473227/dsw061f2.tif" path-from-xml="dsw061f2.tif" target="_blank">View large</a><a section="88473227" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f2.png?Expires=2147483647&Signature=cy4on4jcvD4FBGoRv0vz854~ZUUtBUEzloehnADSuG2PzD8ULeO51w52sqpJMVV13bbOj9WvfE5x05WN7MTzzw4iOwV8CJhYGq30VduZQgypkf3jJ3KMlN5~CkxAAgNsdQx79FYQIjGWNMXq26N3ozIDoodMRJmLMGoWKWbwB-8rh4RhBut-AwYBO1j3ObDgp-Z6r275Xo9SKCrUfvsOPRsAEr4pPlZZXDyYIdliaes5vJF1HuaFeDstcXaEQX1jCNDwHLfp7mQGEFa3iPn3BJDmiCj0HpRjl6~ekWqjrcztQD2~h21-4wj5m46Ia7T9sjYEO7lYrwPs80tjLR1r~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473227&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Enriched functional categories of genes containing selected 6-mer and 8-mer CREs in their promoter sequences. The opacity of the dot corresponds to –log10 FDR functional category enrichment values. The size of the dots corresponds to the number of genes (log10 scaled) whose promoters contain the selected CRE. Commonly associated names for CREs are indicated in brackets.</p></div></div><p>The AuxRE, TGTCTC, was enriched (FDR &lt; 0.05) in protein targeting, regulation of transcription, and secondary metabolism with 64,724, and 251 genes, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Conversely, TGTCGG-containing promoters are highly enriched in hormone metabolism (168 genes, FDR &lt; 5.5E-04) pathways, specifically auxin (49 genes) and ethylene (38 genes) biosynthesis and regulation. Cell wall metabolism (105 genes) and post-translational modification (160 genes) pathways were also significantly enriched. Enrichment of hormone homeostasis and post-translational modification pathways is not surprising as these mechanisms regulate plant ARF function and ARFs are known to facilitate feedback loops within the auxin signalling pathway.<a reveal-id="dsw061-B8" data-open="dsw061-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Although both TGTCTC’s and TGTCGG’s functional enrichment profiles suggest that various pathways are potentially targeted by ARFs in grapevine, TGTCGG-containing promoters show higher enrichment profiles coinciding with a higher affinity presence of ARFs.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B11" data-open="dsw061-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Nonetheless, treatment of the auxin analogue (naphtalenacetic acid) in grape berries at the onset of ripening and harvest significantly modulated &gt; 1,000 genes at each stage, many of which were auxin- and ethylene-related pathway genes,<a reveal-id="dsw061-B50" data-open="dsw061-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> consistent with the enrichment profiles of TGTCGG-containing genes reported here. As no grapevine ARFs have been functionally characterized, the specificity of each member (19 in total<a reveal-id="dsw061-B8" data-open="dsw061-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a>) in delineating real targets from the vast number of AuxRE-containing promoters, and the exact mode of their regulation (activation/repression), remains to be determined.</p><p>The CACGTG (G-box) containing promoters were enriched (FDR &lt; 0.05) in a plethora of pathways including regulation of transcription (552 genes), abiotic and biotic stress (230 genes), development (185 genes), flavonoid metabolism (85 genes), sugar metabolism (43 genes), and many more (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). This observation is consistent with the role of plant bZIP<a reveal-id="dsw061-B44" data-open="dsw061-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and bHLH<a reveal-id="dsw061-B46" data-open="dsw061-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> TFs; especially a subset of those who regulate a large repertoire of plant development processes and stress responses (e.g. bZIP-HY5<a reveal-id="dsw061-B51" data-open="dsw061-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsw061-B52" data-open="dsw061-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and bHLH-PIFs<a reveal-id="dsw061-B45" data-open="dsw061-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a>). In grapevine, recent studies demonstrate that bZIP/bHLH TFs have a role in regulating fruit ripening (<em>VviABF2</em><a reveal-id="dsw061-B53" data-open="dsw061-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a>) and the flavonoid pathway (<em>VvibZIPC22</em>,<a reveal-id="dsw061-B28" data-open="dsw061-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><em>VviHY5</em>,<a reveal-id="dsw061-B29" data-open="dsw061-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and <em>VviMYC1</em><a reveal-id="dsw061-B54" data-open="dsw061-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a>). The high enrichment of CACGTG-containing genes (122 genes, FDR &lt; 2.1E-05) in biotic stress pathways is of interest as limited information is known on the roles of bZIP and bHLH TFs in plant immunity regulatory networks.<a reveal-id="dsw061-B55" data-open="dsw061-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p><p>Promoters of genes containing AGATATTT, a circadian clock associated CRE, were highly enriched in secondary metabolism (197 genes, FDR &lt; 4.6E-06; especially flavonoids and isoprenoids), hormone metabolism (159 genes, FDR &lt; 1.1E-04), and abiotic stress (102 genes, FDR &lt; 1.5E-03) pathways. The ubiquity of the circadian clock in regulating many aspects of secondary metabolism (e.g. flavonoid and isoprenoid), hormone biosynthesis and signaling (e.g. jasmonic acid (JA), salicylic acid (SA), auxin, and ABA), and abiotic stress (e.g. cold, drought, and nutrient availability) is well documented.<a reveal-id="dsw061-B56" data-open="dsw061-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsw061-B57" data-open="dsw061-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Interestingly, many chalcone (10 genes), stilbene (20 genes),<a reveal-id="dsw061-B58" data-open="dsw061-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> and terpene (35 genes)<a reveal-id="dsw061-B59" data-open="dsw061-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> synthases that contain the AGATATTT CRE within their promoters, exist in grapevines as highly duplicated gene families compared with other plants.<a reveal-id="dsw061-B24" data-open="dsw061-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Many of these stilbene and terpene synthases may belong to a novel ‘expanded’ clock regulatory network in grapes, especially for stilbene synthases which are exclusively found in stilbene-producing plants, such as grapevines.<a reveal-id="dsw061-B58" data-open="dsw061-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> These insights reveal both potential and novel regulatory repertoires of grapevine circadian clock homologues, <em>VviLHY</em> and <em>VviRVE1</em>.<a reveal-id="dsw061-B60" data-open="dsw061-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a></p><p>Collectively, these examples and many others (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) characterize potential targets of grapevine TFs and provide insight into the collective roles of genes containing certain CREs. Nonetheless, we further explored the possibility of obtaining more information on the grapevine CRE cis-regulatory code by integrating large-scale gene expression meta-analysis and GCN. Compared with approaches that solely rely on the distribution of CREs alone, integrated approaches provides a deeper understanding of the contribution of CREs in determining global gene expression responses<a reveal-id="dsw061-B12" data-open="dsw061-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and the coordinated regulation among genes in a GCN based on their network connectivity and promoter CRE structure.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p>                            <h3 scrollto-destination=88473232 id="88473232" class="section-title" >3.3. Numerous CREs are implicated in grapevine stress-responsive and development-dynamic transcription</h3>
<p>To determine the contribution of CREs in determining stress-responsive and development-dynamic gene transcription, meta-analysis of publicly available next-generation sequencing (RNA-seq) and the grapevine tissue developmental atlas was first performed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The survey for stress-related RNA-seq datasets yielded a total of 87 conditions (260 non-averaged) from 10 experiments that considered various abiotic (e.g. drought and shading) and biotic stress (e.g. <em>Botrytis cinerea</em>, <em>Erysiphe</em><em>necator</em>, and <em>Plasmopara</em><em>viticola</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). All these conditions also correspond to three different organs: leaf, berry, and flower. Given the incompleteness of sequenced mRNA across tissues and associated developmental stages, the grapevine expression atlas was considered.<a reveal-id="dsw061-B25" data-open="dsw061-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Based on the NimbleGen microarray platform capable of profiling &gt; 95% of the predicted grapevine transcriptome. The grapevine atlas represents the most comprehensive (54 tissues) compendia of tissue <em>x</em> developmental gene expression series to date.</p><p>In the stress-related compendia, 17,623 genes (59% of transcriptome) were collectively modulated (|log2FC|&gt;1, FDR &lt; 0.05) across all the comparisons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Abiotic stress treatments (drought and shading) modulated more than 5,000 genes in leaves and flowers, while biotic stress treatments (<em>B. cinerea</em>, <em>E. necator</em>, <em>P. viticola</em>, and <em>Tetranychus</em><em>urticae</em>) modulated more than 4,000 genes collectively in berries and leaves. Previous work by Fasoli et al.<a reveal-id="dsw061-B25" data-open="dsw061-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> revealed the grapevine atlas transcriptome relationships to be highly clustered according to their developmental indices, but did not identify the genes that are developmentally modulated in each tissue/organ. Meanwhile, reanalysis of grapevine atlas was performed to identify genes that are modulated during development and maturity progression. As expected, more than 85% (25,947 genes) of the predicted transcriptome is significantly modulated (|log2FC|&gt;1, FDR &lt; 0.05) across the whole atlas (excluding carpel, stamen, petal, pollen, seedling, and root). In organs such as berry (flesh, pericarp, and seed), tendril, stem, and buds more than 10,000 genes were modulated as development progresses while berry skin, rachis, leaf, and inflorescence had lesser modulated genes.</p><p>Several studies have shown that filtering for highly modulated genes, often within top 100 most modulated genes<a reveal-id="dsw061-B15" data-open="dsw061-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> or having large (&gt; 6-fold) fold changes,<a reveal-id="dsw061-B12" data-open="dsw061-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> may be better suited for identifying major CREs driving expression under a particular condition. We empirically determined that the top 300 most up- and down-regulated genes would provide a reasonable threshold closely satisfying the large fold change requirements (<a reveal-id="dsw061-F3" data-open="dsw061-F3" class="link link-reveal link-table xref-fig">Fig. 3A;</a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). These genes typically have a median fold change greater than 4- and 8-fold in the stress and development datasets, respectively. In addition, 10,847 and 8,224 unique genes were within top 300 up- and down-regulated genes in at least one stress and developmental condition, respectively, with an overlap of 5,601 genes that were within the 300 up- and down-regulated genes in both stress-responsive and developmental-dynamic condition.</p>                        <a id="88473236" scrollto-destination="88473236"></a>
<div id="dsw061-F3" data-id="dsw061-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f3.png?Expires=2147483647&Signature=G5Fdrz6nETf4v5vXmkRf9cOEQxa6-WBIxVhM1~s9pT1hcrue0G55ggbwe7YzGeb1W9k-B-Ev2m1nhOe9zmX7HTrGX7Q3PNtnDkJSvo0PnCBLdIRztB56EN23VaiAJsOUZELNciUkE5t1f0PtSbKo0Lxr8tkphoCMiECQQKkxIkNbRgUnR2VM~bvx6XxgV~7-RESo-jDFc399LjaUG5VP~XGZVWaEPgzV8xRKlDQURzKLh3ELiWyaDWeOjHnKF45iNBk-Kf07SGFGyS4pjcLXKkY9BooARkq2~ThTW37OYHlaWqKG8PajeuSbzrJIOWV5QaNfEb3xCyOVkpy42gCVBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of differential expression analysis and distribution patterns of CRE presence under stress and development conditions. (A) Boxplots depict the fold change distribution of all top 300 most differentially modulated (up-regulated and down-regulated) genes in S and D compendia. S, stress; D, development. (B) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to either stress-responsive and/or development-dynamic (orange) or were non-responsive to the tested conditions (purple), for all 6-mer and 8-mer CRE. (C) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to only stress-responsive (red), development-dynamic (blue), and both stress-responsive and development-dynamic (green)." path-from-xml="dsw061f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473236/dsw061f3.tif" path-from-xml="dsw061f3.tif" target="_blank">View large</a><a section="88473236" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f3.png?Expires=2147483647&Signature=iAmQAF-Uz-390HWS2mo1u0zN2wjS5XCyGoneYhdTAGljoai6WntGRNMUhv1dzZwMzKg9tpDI7Z9Om~tGce5f5uFtmk3DjkTgWJGvotNotcxCOZgFDiKALco7pMJgzl7kcs5zeukQDnyAkkYGIFPI8RZmaaFxHTDsCNcdw6jewtAGbti6s6BymWhdw-PVdL7MpsX9zqYCGRs7Ch9ytT5Xgshoko6YyuR3AiiA8p4Y-CBPUMCmasnYsu4KhG5C5HnI1u1yUm7ru2m7w28xDqg9OuDkHvYEZOMZUq5Gg4s4BB1vIZe4RtvbUNWSVtE-lMk0-zIKMM7u8h~FGfUlbaKbZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473236&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of differential expression analysis and distribution patterns of CRE presence under stress and development conditions. (A) Boxplots depict the fold change distribution of all top 300 most differentially modulated (up-regulated and down-regulated) genes in S and D compendia. S, stress; D, development. (B) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to either stress-responsive and/or development-dynamic (orange) or were non-responsive to the tested conditions (purple), for all 6-mer and 8-mer CRE. (C) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to only stress-responsive (red), development-dynamic (blue), and both stress-responsive and development-dynamic (green).</p></div></div><div content-id="dsw061-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f3.png?Expires=2147483647&Signature=G5Fdrz6nETf4v5vXmkRf9cOEQxa6-WBIxVhM1~s9pT1hcrue0G55ggbwe7YzGeb1W9k-B-Ev2m1nhOe9zmX7HTrGX7Q3PNtnDkJSvo0PnCBLdIRztB56EN23VaiAJsOUZELNciUkE5t1f0PtSbKo0Lxr8tkphoCMiECQQKkxIkNbRgUnR2VM~bvx6XxgV~7-RESo-jDFc399LjaUG5VP~XGZVWaEPgzV8xRKlDQURzKLh3ELiWyaDWeOjHnKF45iNBk-Kf07SGFGyS4pjcLXKkY9BooARkq2~ThTW37OYHlaWqKG8PajeuSbzrJIOWV5QaNfEb3xCyOVkpy42gCVBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of differential expression analysis and distribution patterns of CRE presence under stress and development conditions. (A) Boxplots depict the fold change distribution of all top 300 most differentially modulated (up-regulated and down-regulated) genes in S and D compendia. S, stress; D, development. (B) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to either stress-responsive and/or development-dynamic (orange) or were non-responsive to the tested conditions (purple), for all 6-mer and 8-mer CRE. (C) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to only stress-responsive (red), development-dynamic (blue), and both stress-responsive and development-dynamic (green)." path-from-xml="dsw061f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473236/dsw061f3.tif" path-from-xml="dsw061f3.tif" target="_blank">View large</a><a section="88473236" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f3.png?Expires=2147483647&Signature=iAmQAF-Uz-390HWS2mo1u0zN2wjS5XCyGoneYhdTAGljoai6WntGRNMUhv1dzZwMzKg9tpDI7Z9Om~tGce5f5uFtmk3DjkTgWJGvotNotcxCOZgFDiKALco7pMJgzl7kcs5zeukQDnyAkkYGIFPI8RZmaaFxHTDsCNcdw6jewtAGbti6s6BymWhdw-PVdL7MpsX9zqYCGRs7Ch9ytT5Xgshoko6YyuR3AiiA8p4Y-CBPUMCmasnYsu4KhG5C5HnI1u1yUm7ru2m7w28xDqg9OuDkHvYEZOMZUq5Gg4s4BB1vIZe4RtvbUNWSVtE-lMk0-zIKMM7u8h~FGfUlbaKbZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473236&ar=2949522&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw061f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of differential expression analysis and distribution patterns of CRE presence under stress and development conditions. (A) Boxplots depict the fold change distribution of all top 300 most differentially modulated (up-regulated and down-regulated) genes in S and D compendia. S, stress; D, development. (B) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to either stress-responsive and/or development-dynamic (orange) or were non-responsive to the tested conditions (purple), for all 6-mer and 8-mer CRE. (C) Boxplots depict the proportions of the genes (containing the respective CRE) that were highly responsive to only stress-responsive (red), development-dynamic (blue), and both stress-responsive and development-dynamic (green).</p></div></div><p>For each 6-mer CRE, the proportion of the genes (containing the respective CRE) that were highly responsive to either stress-responsive and/or development-dynamic was analysed in detail. Considering all 6-mer CREs, their proportions had a median distribution of 46% (<a reveal-id="dsw061-F3" data-open="dsw061-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>); more specifically, a median distribution of 37% was considered only stress-responsive, 28% was only development-dynamic, while 19% was considered both stress-responsive and development-dynamic (<a reveal-id="dsw061-F3" data-open="dsw061-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). Similar proportions were also observed for 8-mer CREs (<a reveal-id="dsw061-F3" data-open="dsw061-F3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>), albeit some outlier CREs show that genes containing the latter CREs were neither highly stress-responsive nor development-dynamic. Therefore, this suggests that the distribution of 6-mer to 8-mer CREs in gene promoters likely contributes to approximately half of the DE across the various abiotic, biotic stresses, and development comparisons evaluated. These analyses provided a basis for selecting genes for the construction of condition-specific GCNs in order to discover condition-specific 6–8-mer CRE-driven regulatory modules.</p>                            <h3 scrollto-destination=88473238 id="88473238" class="section-title" >3.4. CRE-driven modules are biologically relevant in condition-specific GCNs</h3>
<p>Two GCNs (one stress- and one development-specific) were constructed with gene expression matrices using the MR approach. Co-expression studies in many plants<a reveal-id="dsw061-B23" data-open="dsw061-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and in grapevine<a reveal-id="dsw061-B28 dsw061-B29 dsw061-B30" data-open="dsw061-B28 dsw061-B29 dsw061-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a><sup>,</sup><a reveal-id="dsw061-B61" data-open="dsw061-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> have shown that direct correlations (e.g. PCC) and rank-based co-expression (e.g. MR) can be used to infer co-expression relationships, albeit rank-based co-expression is generally more robust and is the most popular approach among crop GCNs.<a reveal-id="dsw061-B23" data-open="dsw061-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> In addition, establishment of statistically significant reciprocal ranks, MR in this case, may further improve the robustness of the resulting GCN by retaining strong and significant co-expression.<a reveal-id="dsw061-B21" data-open="dsw061-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> In the two datasets, we empirically chose a MR cut-off of 35 based on 1,000 permutations of each expression matrices, which corresponds to a statistical confidence of <em>P</em> &lt; 0.003 and 0.005 for the surviving MR in the stress- and development-specific GCN, respectively. These network criteria are highly comparable to the parameters applied in other studies that define strong and significant co-expression. For example, GCNs constructed with MR &lt; 50,<a reveal-id="dsw061-B19" data-open="dsw061-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> statistical significance of reciprocal ranks with <em>P</em> &lt; 0.01,<a reveal-id="dsw061-B21" data-open="dsw061-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> or by retaining top 100 strongest connections.<a reveal-id="dsw061-B20" data-open="dsw061-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The final stress-specific GCN results in 10,800 nodes and 134,100 edges, while the final development-specific GCN consists of 8,224 nodes, 113,790 edges (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5 and S6</a></span>). The number of connections for each node in the stress-specific GCN network ranges between 1 and 84, with a median of 21 connections, while the development-specific GCN ranged between 2 and 93, with a median of 24 connections.</p><p>Next, we adopted a recently proposed ‘bottom-up’ approach by Ma and co-workers<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> to identify gene expression modules potentially driven by CREs of co-expressed genes. Using this approach, 245 and 289 CRE-driven modules (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7 and S8</a></span>) in stress- and development-specific GCNs, respectively, were enriched (FDR &lt; 0.05) with at least one MapMan BIN category (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Although modular structures self-manifested in most CRE-driven modules,<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> we applied community clustering to ensure robust final submodules.<a reveal-id="dsw061-B38" data-open="dsw061-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> In total, 880 and 1,095 CRE-driven submodules were enriched (FDR &lt; 0.05) in the stress and development-specific GCNs, respectively, with at least one MapMan BIN category (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Interestingly, these enriched submodules corresponded to more ‘global’ CRE-driven modules, 293 and 314 for stress and development, respectively, demonstrating that CRE-driven submodules have higher predictive performance compared with interpreting ‘global’ CRE-driven modules as a whole. The biological relevance for some of these condition-specific CRE-driven submodules will be discussed in greater detail.</p>                            <h3 scrollto-destination=88473241 id="88473241" class="section-title" >3.5. CACGTG enriched co-expression network is involved in ABA metabolism and light-related functions</h3>
<p>The G-box (CACGTG) was found to be highly enriched for stress-responsive genes having scored a <em>CF</em>, <em>Z-score</em>, and <em>FE</em> of 2.7, 3.7, and 4.5, respectively. To investigate whether the enrichment observed has a biological relevance, a gene network was constructed for the stress dataset. A total of 842 genes composed a network comprising a total of 20 community clusters. Some of these clusters were significantly enriched with different MapMan BIN categories (<a reveal-id="dsw061-T1" data-open="dsw061-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Focus is given to the core modules with bZIP and bHLH TFs since previous research has shown that the G-box CRE has been associated with the binding of these TFs.<a reveal-id="dsw061-B44" data-open="dsw061-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw061-B46" data-open="dsw061-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The presence of these TFs within the modules of the CACGTG network would suggest that the TFs contribute to the regulation of genes and processes pertaining to the selected module. <div content-id="dsw061-T1" class="table-modal"><div class="table-wrap"><div id="dsw061-T1" data-id="dsw061-T1" class="table-wrap-title"><div class="label -label">Table <strong>1</strong></div><div class="caption"><p>MapMan enrichment of selected G-box and GCC-core submodules in the stress dataset</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>MapMan term </th></tr></thead><tbody><tr><td rowspan="16">CACGTG (G-box) </td><td>I </td><td>7 </td><td>189 </td><td>132 </td><td>2.19E-05 </td><td>1.48E-02 </td><td>Hormone metabolism.abscisic acid </td></tr><tr><td rowspan="3">II </td><td>5 </td><td>55 </td><td>74 </td><td>2.71E-07 </td><td>3.68E-04 </td><td>Cell wall.pectin*esterases </td></tr><tr><td>5 </td><td>55 </td><td>114 </td><td>2.31E-06 </td><td>1.57E-03 </td><td>Misc.glutathione S transferases </td></tr><tr><td>6 </td><td>55 </td><td>268 </td><td>1.01E-05 </td><td>3.43E-03 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td>III </td><td>9 </td><td>67 </td><td>549 </td><td>3.75E-06 </td><td>5.09E-03 </td><td>Cell wall </td></tr><tr><td>V </td><td>5 </td><td>57 </td><td>181 </td><td>2.55E-05 </td><td>1.16E-02 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>VI </td><td>7 </td><td>163 </td><td>145 </td><td>1.56E-05 </td><td>1.35E-02 </td><td>Hormone metabolism.ethylene </td></tr><tr><td>VII </td><td>3 </td><td>7 </td><td>389 </td><td>7.61E-05 </td><td>3.45E-02 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>VIII </td><td>3 </td><td>5 </td><td>498 </td><td>4.66E-05 </td><td>3.16E-02 </td><td>Stress.abiotic </td></tr><tr><td rowspan="3">IX </td><td>11 </td><td>25 </td><td>288 </td><td>2.55E-16 </td><td>6.92E-14 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>16 </td><td>25 </td><td>1,448 </td><td>1.43E-15 </td><td>3.23E-13 </td><td>Signalling.receptor kinases </td></tr><tr><td>8 </td><td>25 </td><td>1,111 </td><td>2.28E-06 </td><td>3.09E-04 </td><td>Stress.biotic </td></tr><tr><td rowspan="2">X </td><td>6 </td><td>143 </td><td>8 </td><td>3.29E-13 </td><td>4.47E-10 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td>3 </td><td>143 </td><td>18 </td><td>8.54E-05 </td><td>3.87E-02 </td><td>Development.late embryogenesis abundant </td></tr><tr><td rowspan="2">XII </td><td>32 </td><td>73 </td><td>127 </td><td>1.32E-57 </td><td>8.99E-55 </td><td>PS.lightreaction </td></tr><tr><td>6 </td><td>73 </td><td>36 </td><td>3.44E-10 </td><td>5.83E-08 </td><td>PS.calvin cycle </td></tr><tr><td rowspan="7">GCC-core </td><td>I </td><td>6 </td><td>101 </td><td>145 </td><td>1.04E-05 </td><td>5.19E-03 </td><td>Hormone metabolism.ethylene </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>94 </td><td>61 </td><td>1.50E-06 </td><td>2.15E-03 </td><td>RNA.regulation of transcription.WRKY </td></tr><tr><td>8 </td><td>94 </td><td>309 </td><td>6.20E-06 </td><td>4.44E-03 </td><td>RNA.regulation of transcription.AP2/ERF </td></tr><tr><td rowspan="2">III </td><td>33 </td><td>71 </td><td>304 </td><td>6.49E-47 </td><td>3.10E-44 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>9 </td><td>71 </td><td>174 </td><td>3.77E-10 </td><td>7.71E-08 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>V </td><td>3 </td><td>30 </td><td>11 </td><td>1.57E-07 </td><td>5.61E-05 </td><td>Mitochondrial electron transport/ATP synthesis.cytochrome c </td></tr><tr><td>VI </td><td>8 </td><td>27 </td><td>1,016 </td><td>2.25E-06 </td><td>1.61E-03 </td><td>Signalling.receptor kinases </td></tr></tbody></table></div><div content-id="dsw061-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>MapMan term </th></tr></thead><tbody><tr><td rowspan="16">CACGTG (G-box) </td><td>I </td><td>7 </td><td>189 </td><td>132 </td><td>2.19E-05 </td><td>1.48E-02 </td><td>Hormone metabolism.abscisic acid </td></tr><tr><td rowspan="3">II </td><td>5 </td><td>55 </td><td>74 </td><td>2.71E-07 </td><td>3.68E-04 </td><td>Cell wall.pectin*esterases </td></tr><tr><td>5 </td><td>55 </td><td>114 </td><td>2.31E-06 </td><td>1.57E-03 </td><td>Misc.glutathione S transferases </td></tr><tr><td>6 </td><td>55 </td><td>268 </td><td>1.01E-05 </td><td>3.43E-03 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td>III </td><td>9 </td><td>67 </td><td>549 </td><td>3.75E-06 </td><td>5.09E-03 </td><td>Cell wall </td></tr><tr><td>V </td><td>5 </td><td>57 </td><td>181 </td><td>2.55E-05 </td><td>1.16E-02 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>VI </td><td>7 </td><td>163 </td><td>145 </td><td>1.56E-05 </td><td>1.35E-02 </td><td>Hormone metabolism.ethylene </td></tr><tr><td>VII </td><td>3 </td><td>7 </td><td>389 </td><td>7.61E-05 </td><td>3.45E-02 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>VIII </td><td>3 </td><td>5 </td><td>498 </td><td>4.66E-05 </td><td>3.16E-02 </td><td>Stress.abiotic </td></tr><tr><td rowspan="3">IX </td><td>11 </td><td>25 </td><td>288 </td><td>2.55E-16 </td><td>6.92E-14 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>16 </td><td>25 </td><td>1,448 </td><td>1.43E-15 </td><td>3.23E-13 </td><td>Signalling.receptor kinases </td></tr><tr><td>8 </td><td>25 </td><td>1,111 </td><td>2.28E-06 </td><td>3.09E-04 </td><td>Stress.biotic </td></tr><tr><td rowspan="2">X </td><td>6 </td><td>143 </td><td>8 </td><td>3.29E-13 </td><td>4.47E-10 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td>3 </td><td>143 </td><td>18 </td><td>8.54E-05 </td><td>3.87E-02 </td><td>Development.late embryogenesis abundant </td></tr><tr><td rowspan="2">XII </td><td>32 </td><td>73 </td><td>127 </td><td>1.32E-57 </td><td>8.99E-55 </td><td>PS.lightreaction </td></tr><tr><td>6 </td><td>73 </td><td>36 </td><td>3.44E-10 </td><td>5.83E-08 </td><td>PS.calvin cycle </td></tr><tr><td rowspan="7">GCC-core </td><td>I </td><td>6 </td><td>101 </td><td>145 </td><td>1.04E-05 </td><td>5.19E-03 </td><td>Hormone metabolism.ethylene </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>94 </td><td>61 </td><td>1.50E-06 </td><td>2.15E-03 </td><td>RNA.regulation of transcription.WRKY </td></tr><tr><td>8 </td><td>94 </td><td>309 </td><td>6.20E-06 </td><td>4.44E-03 </td><td>RNA.regulation of transcription.AP2/ERF </td></tr><tr><td rowspan="2">III </td><td>33 </td><td>71 </td><td>304 </td><td>6.49E-47 </td><td>3.10E-44 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>9 </td><td>71 </td><td>174 </td><td>3.77E-10 </td><td>7.71E-08 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>V </td><td>3 </td><td>30 </td><td>11 </td><td>1.57E-07 </td><td>5.61E-05 </td><td>Mitochondrial electron transport/ATP synthesis.cytochrome c </td></tr><tr><td>VI </td><td>8 </td><td>27 </td><td>1,016 </td><td>2.25E-06 </td><td>1.61E-03 </td><td>Signalling.receptor kinases </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>x</em>, overlap size; <em>n</em>, query size; and <em>k</em>, term size represent the contingency tables for MapMan enrichment calculations.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw061-T1" data-id="dsw061-T1" class="table-wrap-title"><div class="label -label">Table <strong>1</strong></div><div class="caption"><p>MapMan enrichment of selected G-box and GCC-core submodules in the stress dataset</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>MapMan term </th></tr></thead><tbody><tr><td rowspan="16">CACGTG (G-box) </td><td>I </td><td>7 </td><td>189 </td><td>132 </td><td>2.19E-05 </td><td>1.48E-02 </td><td>Hormone metabolism.abscisic acid </td></tr><tr><td rowspan="3">II </td><td>5 </td><td>55 </td><td>74 </td><td>2.71E-07 </td><td>3.68E-04 </td><td>Cell wall.pectin*esterases </td></tr><tr><td>5 </td><td>55 </td><td>114 </td><td>2.31E-06 </td><td>1.57E-03 </td><td>Misc.glutathione S transferases </td></tr><tr><td>6 </td><td>55 </td><td>268 </td><td>1.01E-05 </td><td>3.43E-03 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td>III </td><td>9 </td><td>67 </td><td>549 </td><td>3.75E-06 </td><td>5.09E-03 </td><td>Cell wall </td></tr><tr><td>V </td><td>5 </td><td>57 </td><td>181 </td><td>2.55E-05 </td><td>1.16E-02 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>VI </td><td>7 </td><td>163 </td><td>145 </td><td>1.56E-05 </td><td>1.35E-02 </td><td>Hormone metabolism.ethylene </td></tr><tr><td>VII </td><td>3 </td><td>7 </td><td>389 </td><td>7.61E-05 </td><td>3.45E-02 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>VIII </td><td>3 </td><td>5 </td><td>498 </td><td>4.66E-05 </td><td>3.16E-02 </td><td>Stress.abiotic </td></tr><tr><td rowspan="3">IX </td><td>11 </td><td>25 </td><td>288 </td><td>2.55E-16 </td><td>6.92E-14 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>16 </td><td>25 </td><td>1,448 </td><td>1.43E-15 </td><td>3.23E-13 </td><td>Signalling.receptor kinases </td></tr><tr><td>8 </td><td>25 </td><td>1,111 </td><td>2.28E-06 </td><td>3.09E-04 </td><td>Stress.biotic </td></tr><tr><td rowspan="2">X </td><td>6 </td><td>143 </td><td>8 </td><td>3.29E-13 </td><td>4.47E-10 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td>3 </td><td>143 </td><td>18 </td><td>8.54E-05 </td><td>3.87E-02 </td><td>Development.late embryogenesis abundant </td></tr><tr><td rowspan="2">XII </td><td>32 </td><td>73 </td><td>127 </td><td>1.32E-57 </td><td>8.99E-55 </td><td>PS.lightreaction </td></tr><tr><td>6 </td><td>73 </td><td>36 </td><td>3.44E-10 </td><td>5.83E-08 </td><td>PS.calvin cycle </td></tr><tr><td rowspan="7">GCC-core </td><td>I </td><td>6 </td><td>101 </td><td>145 </td><td>1.04E-05 </td><td>5.19E-03 </td><td>Hormone metabolism.ethylene </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>94 </td><td>61 </td><td>1.50E-06 </td><td>2.15E-03 </td><td>RNA.regulation of transcription.WRKY </td></tr><tr><td>8 </td><td>94 </td><td>309 </td><td>6.20E-06 </td><td>4.44E-03 </td><td>RNA.regulation of transcription.AP2/ERF </td></tr><tr><td rowspan="2">III </td><td>33 </td><td>71 </td><td>304 </td><td>6.49E-47 </td><td>3.10E-44 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>9 </td><td>71 </td><td>174 </td><td>3.77E-10 </td><td>7.71E-08 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>V </td><td>3 </td><td>30 </td><td>11 </td><td>1.57E-07 </td><td>5.61E-05 </td><td>Mitochondrial electron transport/ATP synthesis.cytochrome c </td></tr><tr><td>VI </td><td>8 </td><td>27 </td><td>1,016 </td><td>2.25E-06 </td><td>1.61E-03 </td><td>Signalling.receptor kinases </td></tr></tbody></table></div><div content-id="dsw061-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>MapMan term </th></tr></thead><tbody><tr><td rowspan="16">CACGTG (G-box) </td><td>I </td><td>7 </td><td>189 </td><td>132 </td><td>2.19E-05 </td><td>1.48E-02 </td><td>Hormone metabolism.abscisic acid </td></tr><tr><td rowspan="3">II </td><td>5 </td><td>55 </td><td>74 </td><td>2.71E-07 </td><td>3.68E-04 </td><td>Cell wall.pectin*esterases </td></tr><tr><td>5 </td><td>55 </td><td>114 </td><td>2.31E-06 </td><td>1.57E-03 </td><td>Misc.glutathione S transferases </td></tr><tr><td>6 </td><td>55 </td><td>268 </td><td>1.01E-05 </td><td>3.43E-03 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td>III </td><td>9 </td><td>67 </td><td>549 </td><td>3.75E-06 </td><td>5.09E-03 </td><td>Cell wall </td></tr><tr><td>V </td><td>5 </td><td>57 </td><td>181 </td><td>2.55E-05 </td><td>1.16E-02 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>VI </td><td>7 </td><td>163 </td><td>145 </td><td>1.56E-05 </td><td>1.35E-02 </td><td>Hormone metabolism.ethylene </td></tr><tr><td>VII </td><td>3 </td><td>7 </td><td>389 </td><td>7.61E-05 </td><td>3.45E-02 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>VIII </td><td>3 </td><td>5 </td><td>498 </td><td>4.66E-05 </td><td>3.16E-02 </td><td>Stress.abiotic </td></tr><tr><td rowspan="3">IX </td><td>11 </td><td>25 </td><td>288 </td><td>2.55E-16 </td><td>6.92E-14 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>16 </td><td>25 </td><td>1,448 </td><td>1.43E-15 </td><td>3.23E-13 </td><td>Signalling.receptor kinases </td></tr><tr><td>8 </td><td>25 </td><td>1,111 </td><td>2.28E-06 </td><td>3.09E-04 </td><td>Stress.biotic </td></tr><tr><td rowspan="2">X </td><td>6 </td><td>143 </td><td>8 </td><td>3.29E-13 </td><td>4.47E-10 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td>3 </td><td>143 </td><td>18 </td><td>8.54E-05 </td><td>3.87E-02 </td><td>Development.late embryogenesis abundant </td></tr><tr><td rowspan="2">XII </td><td>32 </td><td>73 </td><td>127 </td><td>1.32E-57 </td><td>8.99E-55 </td><td>PS.lightreaction </td></tr><tr><td>6 </td><td>73 </td><td>36 </td><td>3.44E-10 </td><td>5.83E-08 </td><td>PS.calvin cycle </td></tr><tr><td rowspan="7">GCC-core </td><td>I </td><td>6 </td><td>101 </td><td>145 </td><td>1.04E-05 </td><td>5.19E-03 </td><td>Hormone metabolism.ethylene </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>94 </td><td>61 </td><td>1.50E-06 </td><td>2.15E-03 </td><td>RNA.regulation of transcription.WRKY </td></tr><tr><td>8 </td><td>94 </td><td>309 </td><td>6.20E-06 </td><td>4.44E-03 </td><td>RNA.regulation of transcription.AP2/ERF </td></tr><tr><td rowspan="2">III </td><td>33 </td><td>71 </td><td>304 </td><td>6.49E-47 </td><td>3.10E-44 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>9 </td><td>71 </td><td>174 </td><td>3.77E-10 </td><td>7.71E-08 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>V </td><td>3 </td><td>30 </td><td>11 </td><td>1.57E-07 </td><td>5.61E-05 </td><td>Mitochondrial electron transport/ATP synthesis.cytochrome c </td></tr><tr><td>VI </td><td>8 </td><td>27 </td><td>1,016 </td><td>2.25E-06 </td><td>1.61E-03 </td><td>Signalling.receptor kinases </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>x</em>, overlap size; <em>n</em>, query size; and <em>k</em>, term size represent the contingency tables for MapMan enrichment calculations.</p></div></div></div></div></p><p>Module I (<a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>) had a MapMan ontology enrichment pertaining to ABA metabolism. In this module six bZIPs and one bHLH were identified, of which one bZIP encodes the grapevine ABSCISIC ACID RESPONSE ELEMENT-BINDING FACTOR2 (<em>VviABF2</em>, VIT_18s0001g10450) involved in regulating berry ripening in an ABA-dependent manner.<a reveal-id="dsw061-B53" data-open="dsw061-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> However, direct implication of <em>VviABF2</em> in regulating stress response networks remains to be established. In Arabidopsis, previous research has shown that this homologous bZIP is involved in ABA mediated stress responses including drought.<a reveal-id="dsw061-B62" data-open="dsw061-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> In the <em>VviABF2</em> subnetwork (<em>k </em>=<em> </em>2) of Module I (<a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), members of the ABA signaling pathway were immediately connected with <em>VviABF2</em> including a homolog (<em>VviABI1</em>, VIT_11s0016g03180) of an Arabidopsis protein phosphatase 2C, <em>ABA insensitive 1</em>, an essential component of the ABA signaling network,<a reveal-id="dsw061-B63" data-open="dsw061-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> and two homeodomain-leucine zipper TFs (<em>VviHB7</em>, VIT_15s0048g02870 and <em>VviHB12</em>, VIT_02s0025g02590), involved in the signal transduction pathways mediating growth in response to drought.<a reveal-id="dsw061-B64" data-open="dsw061-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> In addition, <em>VviABF2</em> might be involved indirectly (<em>k </em>=<em> </em>2) in the regulation of a 9-cis-epoxycarotenoid dioxygenase 2 (<em>VviNCED3</em>, VIT_19s0093g00550), encoding the putative rate-limiting step of ABA biosynthesis, and a protein phosphatase 2C gene (<em>VviPP2C9</em>, VIT_06s0004g05460) involved in ABA signalling.<a reveal-id="dsw061-B63" data-open="dsw061-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> To determine the involvement of this subnetwork in grapevine stress responses, meta-analysis of stress-related RNA-seq datasets described above was performed. Water deficit resulted in higher gene expression and greater up-regulation of module genes especially in leaves compared with berries (<a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). This observation is in agreement with the role of ABA metabolism and signaling in response to drought,<a reveal-id="dsw061-B63" data-open="dsw061-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> while highlighting differences in ABA metabolism and signaling pathways that arise between organs.<a reveal-id="dsw061-B65" data-open="dsw061-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsw061-B66" data-open="dsw061-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a></p>                        <a id="88473244" scrollto-destination="88473244"></a>
<div id="dsw061-F4" data-id="dsw061-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f4.png?Expires=2147483647&amp;Signature=tODsvxjmMQCxRxQBRsbAKSoSD0NduIbBUrBk4u0r1JdJDDQ8YCn~mpMYBBBAJfcMEncPG15Z8nMBMtG8cOYSKw6lIFq1pKUZVZTq1eED8SyutRYB7YTc6QLClDtA28s670s~xnSA0q-R-yMHGZQTtCILCM3nzr8aaqHa0mqyZrMwaExJBKYY-BlH53g5R7o3IPsqXgdOQ3OheYvy8nhxu3KUK0l0ftXowGQOyyKOPPPXgMwpuDsbsknUxKygtnv9j5qzfQ6~SEPed3IaqCxSojGvDCyzmiaoxTnopdqhED0~Yw-BrS82aKEuuQRmOOyVS4JSDfXC0GdtzlkYPxPj9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A stress-specific G-box (CACGTG) submodule and the expression profile of submodule genes in different stress conditions and tissues. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The relative position of each submodule within stress-specific G-box submodule I (upper right of each subnetwork) is shown by the black circle. Submodule of (A) VviABF2 and (B) VviHYH as the ‘seed’ or ‘guide’ gene (k = 2). Circle and square nodes depict co-expressed gene and transcription factors, respectively. The red/gray-circle subnode depicts the presence of CRE in gene promoter. Node color depict assigned functions of genes involved in hormone signaling, hormone metabolism, DNA repair, electron transport/oxidative stress, chloroplast-related function, oxidative stress, phenylalanine and flavonoid metabolism, plastid import, and signaling. Edges indicate statically significant relationships (mutual rank &lt; 35) between any two nodes. (C) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (See Supplementary Fig. S1A for descriptions)." path-from-xml="dsw061f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473244/dsw061f4.tif" path-from-xml="dsw061f4.tif" target="_blank">View large</a><a section="88473244" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f4.png?Expires=2147483647&amp;Signature=UKG1G2Ll-jb77YXK73w0iJVg3Sh9XJajZNox8UnQiT3ndq3U1GLYF56axrtY5e93yq91UUWwZpSNMnLDKUtsZLmtGspfOW2lETX416uylUXW~Lubfn3PfViwqhsF9jHlfKbjynz8ga0pSpWknibPgyWStCSBBFlokX3RanFlt06xAf7PdqodE80-SkDbQ85vYlOHM4mSEB343-u6OU6nQT9bMN4rmoX6aPnYbV8SmEi3Q2Gerzsp8hbb193T1mFc8pP2Fzw-aHNuBwLUK7RhWS6p5oseNh9umIDUdT~XKl47pkQ5sYpJ1AO~ckqjTvCRzwaMr17by0A-fJj~HfXtyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473244&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A stress-specific G-box (CACGTG) submodule and the expression profile of submodule genes in different stress conditions and tissues. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The relative position of each submodule within stress-specific G-box submodule I (upper right of each subnetwork) is shown by the black circle. Submodule of (A) <em>VviABF2</em> and (B) <em>VviHYH</em> as the ‘seed’ or ‘guide’ gene (<em>k </em>=<em> </em>2). Circle and square nodes depict co-expressed gene and transcription factors, respectively. The red/gray-circle subnode depicts the presence of CRE in gene promoter. Node color depict assigned functions of genes involved in hormone signaling, hormone metabolism, DNA repair, electron transport/oxidative stress, chloroplast-related function, oxidative stress, phenylalanine and flavonoid metabolism, plastid import, and signaling. Edges indicate statically significant relationships (mutual rank &lt; 35) between any two nodes. (C) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>A for descriptions).</p></div></div><div content-id="dsw061-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f4.png?Expires=2147483647&amp;Signature=tODsvxjmMQCxRxQBRsbAKSoSD0NduIbBUrBk4u0r1JdJDDQ8YCn~mpMYBBBAJfcMEncPG15Z8nMBMtG8cOYSKw6lIFq1pKUZVZTq1eED8SyutRYB7YTc6QLClDtA28s670s~xnSA0q-R-yMHGZQTtCILCM3nzr8aaqHa0mqyZrMwaExJBKYY-BlH53g5R7o3IPsqXgdOQ3OheYvy8nhxu3KUK0l0ftXowGQOyyKOPPPXgMwpuDsbsknUxKygtnv9j5qzfQ6~SEPed3IaqCxSojGvDCyzmiaoxTnopdqhED0~Yw-BrS82aKEuuQRmOOyVS4JSDfXC0GdtzlkYPxPj9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A stress-specific G-box (CACGTG) submodule and the expression profile of submodule genes in different stress conditions and tissues. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The relative position of each submodule within stress-specific G-box submodule I (upper right of each subnetwork) is shown by the black circle. Submodule of (A) VviABF2 and (B) VviHYH as the ‘seed’ or ‘guide’ gene (k = 2). Circle and square nodes depict co-expressed gene and transcription factors, respectively. The red/gray-circle subnode depicts the presence of CRE in gene promoter. Node color depict assigned functions of genes involved in hormone signaling, hormone metabolism, DNA repair, electron transport/oxidative stress, chloroplast-related function, oxidative stress, phenylalanine and flavonoid metabolism, plastid import, and signaling. Edges indicate statically significant relationships (mutual rank &lt; 35) between any two nodes. (C) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (See Supplementary Fig. S1A for descriptions)." path-from-xml="dsw061f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473244/dsw061f4.tif" path-from-xml="dsw061f4.tif" target="_blank">View large</a><a section="88473244" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f4.png?Expires=2147483647&amp;Signature=UKG1G2Ll-jb77YXK73w0iJVg3Sh9XJajZNox8UnQiT3ndq3U1GLYF56axrtY5e93yq91UUWwZpSNMnLDKUtsZLmtGspfOW2lETX416uylUXW~Lubfn3PfViwqhsF9jHlfKbjynz8ga0pSpWknibPgyWStCSBBFlokX3RanFlt06xAf7PdqodE80-SkDbQ85vYlOHM4mSEB343-u6OU6nQT9bMN4rmoX6aPnYbV8SmEi3Q2Gerzsp8hbb193T1mFc8pP2Fzw-aHNuBwLUK7RhWS6p5oseNh9umIDUdT~XKl47pkQ5sYpJ1AO~ckqjTvCRzwaMr17by0A-fJj~HfXtyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473244&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A stress-specific G-box (CACGTG) submodule and the expression profile of submodule genes in different stress conditions and tissues. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The relative position of each submodule within stress-specific G-box submodule I (upper right of each subnetwork) is shown by the black circle. Submodule of (A) <em>VviABF2</em> and (B) <em>VviHYH</em> as the ‘seed’ or ‘guide’ gene (<em>k </em>=<em> </em>2). Circle and square nodes depict co-expressed gene and transcription factors, respectively. The red/gray-circle subnode depicts the presence of CRE in gene promoter. Node color depict assigned functions of genes involved in hormone signaling, hormone metabolism, DNA repair, electron transport/oxidative stress, chloroplast-related function, oxidative stress, phenylalanine and flavonoid metabolism, plastid import, and signaling. Edges indicate statically significant relationships (mutual rank &lt; 35) between any two nodes. (C) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>A for descriptions).</p></div></div><p>Another bZIP gene in this module, encodes a grapevine HY5 HOMOLOGUE (<em>VviHYH</em>, VIT_05s0020g01090). Our recent work proposed that <em>VviHYH</em> may fulfil both co-operative and partially redundant roles with <em>VviHY5</em> during both grapevine photomorphogenic growth, berry development, and under stress.<a reveal-id="dsw061-B29" data-open="dsw061-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw061-B67" data-open="dsw061-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Supporting this, many genes involved in light- and chloroplast-related functions were densely connected the <em>VviHYH</em> subnetwork (<em>k </em>=<em> </em>2) of Module I (<a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). This includes electron transport pathway genes (e.g. <em>NDHT</em>, VIT_18s0117g00260 and <em>PGR5</em>, VIT_05s0049g01760), chlorophyll breakdown (e.g. <em>ACD1/PAO</em>, VIT_06s0004g00610), chloroplast-targeting (<em>TIC55</em>, VIT_04s0008g07020 and <em>ANKYRIN REPEAT PROTEIN</em>, VIT_04s0008g03840), DNA-damage repair (e.g. <em>PHOTOLYASE 1/PHR1/UVR2</em>, VIT_02s0241g00040 and <em>CRY3</em>, VIT_04s0008g02670), and light signaling (<em>SUPPRESSOR OF PHYA 3/SPA3</em>, VIT_12s0028g03570). In addition, genes involved in the glycosylation of flavonols—essential for enhancing their solubility in water and accumulation in plant cells<a reveal-id="dsw061-B68" data-open="dsw061-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a>—namely <em>VviGT5</em> (VIT_11s0052g01630) and <em>VviGT6</em> (VIT_11s0052g01600), were connected with <em>VviHYH</em> within one and two steps, respectively. An Arabidopsis MYB-like domain TF homolog (<em>VviMYBD</em>, VIT_01s0026g01050), involved in the regulation of anthocyanin biosynthesis,<a reveal-id="dsw061-B69" data-open="dsw061-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> was also co-regulated with <em>VviHYH</em> (<em>k </em>=<em> </em>2). These observations reaffirm the involvement of <em>VviHYH</em> submodule genes in light-regulated transcriptional and secondary metabolism networks in a stress-related context. The strong presence of the G-box CRE may imply a co-operative role of other bZIP TFs in its regulation, such as HY5.<a reveal-id="dsw061-B51" data-open="dsw061-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> This has been shown for <em>VviGT5</em>, which is up-regulated in a grape HY5 transient expression assay,<a reveal-id="dsw061-B29" data-open="dsw061-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> although <em>VviHYH</em> might also be involved in <em>VviGT5</em> regulation. Expression of module genes was largely down-regulated during the early stages (12–24 hours post-inoculation, hpi) of leaves infected with <em>P. viticola</em> and <em>E. necator</em>, spider-mites, as well as berries infected with <em>B. cinerea</em> at the late stages of berry development (<a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). This strong and rapid negative response to pathogen and pest attack supports the growing evidence of the involvement of light-regulated transcriptional networks, such as the <em>VviHYH</em> subnetwork, in plant–herbivore and plant–pathogen interactions.<a reveal-id="dsw061-B52" data-open="dsw061-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a></p>                            <h3 scrollto-destination=88473246 id="88473246" class="section-title" >3.6. GCC-box network highlights pathogen response and signalling in stress dataset</h3>
<p>Although individual CRE modules may reveal important biological insights, many CRE modules often overlap<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> as underlying genes are shared between similar as well as distinct CRE modules. In the case of similar CREs, the canonical GCC-box sequence (GCCGCC), a well-known binding site for DREB and ERF subfamilies of AP2/ERFs TFs, mediates ethylene and jasmonate signaling in response to pathogens.<a reveal-id="dsw061-B43" data-open="dsw061-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsw061-B55" data-open="dsw061-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Recent studies now show that GCC-related elements (GCCGTC and GCCGGC) are also CREs targeted by DREB and ERF TFs.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In this study, GCC-box, and GCC-related CREs (GCCGTC and GCCGGC) were found to be enriched. GCNs were built for these CREs. Each of three CREs’ GCNs contained at least one module with an AP2/ERF TF: Modules I and VI for GCCGCC, Module II for GCCGTC, and Modules I, III, and IV for GCCGGC (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Comparing those modules, the expression profiles of their genes were highly similar (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) and were enriched with genes pertaining to ethylene and jasmonate signaling. Furthermore, when comparing the genes involved in their networks, GCCGCC contained 82.9% unique genes, GCCGTC had 42.2% unique genes, and GCCGGC had 37.8% unique genes (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Taking note of the similarity between these networks, a GCC core network (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>) comprised of 428 genes was formed by merging the three mentioned networks. From the modules generated, two of them, Module I (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>) and II (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>) contained various combination of AP2/ERF CREs.</p>                        <a id="88473248" scrollto-destination="88473248"></a>
<div id="dsw061-F5" data-id="dsw061-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f5.png?Expires=2147483647&amp;Signature=rWHtsGmYMGdTSJKpxs-2H84Uagg4Jz-e4yKdCob5co~fGf41EgGd32xolUzdnocpidggAmh4IcY0eLH7B3zWDyPdGyhQ4As1CBQswQQnXCgwnaNB5z2OyKl57sV8dONRLqyM9AtKG-rIBvZjRtNnYuA7DQJt-C~QCmE4oDJo7uFEOp7K8bqMnQofIZMRPm8g5cjHB5xcWCTsu~gLcn2hTEPDr4tfiiEQx5~AeJ~oH7ExN4DNDa~10rsngICF-Yyi-TWv8p7zzbB2rmm8jqE4TRQoiCRGyyEwg2q0HSb-hlSCtp4zrABqBEmUZ0hQD5742wKAfbbjwbn1KJ7Sd10cCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Combined stress-specific GCC-box and related submodules and the expression profile of submodule genes in different stress conditions. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding module (lower or upper right of each subnetwork) is shown by the black circle. (A) Venn diagram representing the number of unique and shared genes between the GCCGCC, GCCGTC, and GCCGGC modules. (B) Visualization of the stress-specific GCC-core module, constructed through the integration of GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) modules. Submodule (C) I and (D) II of the stress-specific GCC-core module constructed using AP2/ERFs as the ‘seed’ or ‘guide’ genes. Circle and square nodes depict co-expressed gene and transcription factors, respectively. Circle subnode consisting of different blue/gray shades depicts the presence of the GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) in gene promoter. Node color depict assigned functions for genes involved in hormone signaling, pathogen signaling, pathogenesis related, signaling, detoxification, hormone metabolism, and phenylalanine and flavonoid metabolism. (E) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (see Supplementary Fig. S1A for descriptions)." path-from-xml="dsw061f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473248/dsw061f5.tif" path-from-xml="dsw061f5.tif" target="_blank">View large</a><a section="88473248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f5.png?Expires=2147483647&amp;Signature=koUX6dHMSdnd2KLsbmcesZMHsFtHczuvKzJzjtPYTbfZnIOMyZDhuf42R5gpG4zM7qsgQ-iJNXkRwdcb5Yb2Z6inELwpnSjLcu9vUJcBaEYvHHJzkp~sMC-8EGeQM9TBOJ2ZSn1lYLShwyQ-Wi3JWBCTbQQgfc9gfOlhboNknMFLN7sgleO1NjAIuGrDrxvV5DXtR1IosL0vec~NCS-zfVpu9qUN1QzAYUcmufXbn5a3vqc3dCp48xAPt8ENlEvjeZQW8iye3FlYTify02lbuxZfgmVbzFhN4HPr38NFFqwQwxPhIEw4ZVBF5~eC1f8~5~2tlX2uQ3QCVkK491Fw-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473248&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Combined stress-specific GCC-box and related submodules and the expression profile of submodule genes in different stress conditions. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding module (lower or upper right of each subnetwork) is shown by the black circle. (A) Venn diagram representing the number of unique and shared genes between the GCCGCC, GCCGTC, and GCCGGC modules. (B) Visualization of the stress-specific GCC-core module, constructed through the integration of GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) modules. Submodule (C) I and (D) II of the stress-specific GCC-core module constructed using AP2/ERFs as the ‘seed’ or ‘guide’ genes. Circle and square nodes depict co-expressed gene and transcription factors, respectively. Circle subnode consisting of different blue/gray shades depicts the presence of the GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) in gene promoter. Node color depict assigned functions for genes involved in hormone signaling, pathogen signaling, pathogenesis related, signaling, detoxification, hormone metabolism, and phenylalanine and flavonoid metabolism. (E) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span> for descriptions).</p></div></div><div content-id="dsw061-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f5.png?Expires=2147483647&amp;Signature=rWHtsGmYMGdTSJKpxs-2H84Uagg4Jz-e4yKdCob5co~fGf41EgGd32xolUzdnocpidggAmh4IcY0eLH7B3zWDyPdGyhQ4As1CBQswQQnXCgwnaNB5z2OyKl57sV8dONRLqyM9AtKG-rIBvZjRtNnYuA7DQJt-C~QCmE4oDJo7uFEOp7K8bqMnQofIZMRPm8g5cjHB5xcWCTsu~gLcn2hTEPDr4tfiiEQx5~AeJ~oH7ExN4DNDa~10rsngICF-Yyi-TWv8p7zzbB2rmm8jqE4TRQoiCRGyyEwg2q0HSb-hlSCtp4zrABqBEmUZ0hQD5742wKAfbbjwbn1KJ7Sd10cCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Combined stress-specific GCC-box and related submodules and the expression profile of submodule genes in different stress conditions. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding module (lower or upper right of each subnetwork) is shown by the black circle. (A) Venn diagram representing the number of unique and shared genes between the GCCGCC, GCCGTC, and GCCGGC modules. (B) Visualization of the stress-specific GCC-core module, constructed through the integration of GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) modules. Submodule (C) I and (D) II of the stress-specific GCC-core module constructed using AP2/ERFs as the ‘seed’ or ‘guide’ genes. Circle and square nodes depict co-expressed gene and transcription factors, respectively. Circle subnode consisting of different blue/gray shades depicts the presence of the GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) in gene promoter. Node color depict assigned functions for genes involved in hormone signaling, pathogen signaling, pathogenesis related, signaling, detoxification, hormone metabolism, and phenylalanine and flavonoid metabolism. (E) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (see Supplementary Fig. S1A for descriptions)." path-from-xml="dsw061f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473248/dsw061f5.tif" path-from-xml="dsw061f5.tif" target="_blank">View large</a><a section="88473248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f5.png?Expires=2147483647&amp;Signature=koUX6dHMSdnd2KLsbmcesZMHsFtHczuvKzJzjtPYTbfZnIOMyZDhuf42R5gpG4zM7qsgQ-iJNXkRwdcb5Yb2Z6inELwpnSjLcu9vUJcBaEYvHHJzkp~sMC-8EGeQM9TBOJ2ZSn1lYLShwyQ-Wi3JWBCTbQQgfc9gfOlhboNknMFLN7sgleO1NjAIuGrDrxvV5DXtR1IosL0vec~NCS-zfVpu9qUN1QzAYUcmufXbn5a3vqc3dCp48xAPt8ENlEvjeZQW8iye3FlYTify02lbuxZfgmVbzFhN4HPr38NFFqwQwxPhIEw4ZVBF5~eC1f8~5~2tlX2uQ3QCVkK491Fw-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473248&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Combined stress-specific GCC-box and related submodules and the expression profile of submodule genes in different stress conditions. The color of the box in the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding module (lower or upper right of each subnetwork) is shown by the black circle. (A) Venn diagram representing the number of unique and shared genes between the GCCGCC, GCCGTC, and GCCGGC modules. (B) Visualization of the stress-specific GCC-core module, constructed through the integration of GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) modules. Submodule (C) I and (D) II of the stress-specific GCC-core module constructed using AP2/ERFs as the ‘seed’ or ‘guide’ genes. Circle and square nodes depict co-expressed gene and transcription factors, respectively. Circle subnode consisting of different blue/gray shades depicts the presence of the GCC-box and related CREs (GCCGCC, GCCGTC, and GCCGGC) in gene promoter. Node color depict assigned functions for genes involved in hormone signaling, pathogen signaling, pathogenesis related, signaling, detoxification, hormone metabolism, and phenylalanine and flavonoid metabolism. (E) Heat map showing significant and strong differential expression (up-regulation, red/light gray; down-regulation, blue/dark gray) of submodule genes in several stress conditions considering different tissues (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span> for descriptions).</p></div></div><p>In Module I, five predicted grapevine AP2/ERFs (<em>VviERF106</em>, VIT_15s0021g01630; <em>VviERF107</em>, VIT_16s0013g00890; <em>VviERF127</em>, VIT_07s0005g03220; <em>VviERF132</em>, VIT_14s0081g00730; <em>VviERF1L2</em>, VIT_07s0005g03230) were found (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>), all of which are homologous to group IX (B-3) AP2/ERFs in Arabidopsis.<a reveal-id="dsw061-B70" data-open="dsw061-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Members of group IX (B-3) AP2/ERFs are widely implicated in pathogen responses in Arabidopsis.<a reveal-id="dsw061-B43" data-open="dsw061-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> In this module, <em>VviERF1L2</em> is directly co-expressed with several genes containing GCC-related CREs including one class I chitinase (VIT_07s0005g02560) which may fulfil basic defense-related roles in grapevine,<a reveal-id="dsw061-B71" data-open="dsw061-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> an Arabidopsis homolog of 12-oxophytodienoate reductase (<em>VviOPR1</em>, VIT_18s0041g02070) involved in the detoxification of harmful oxylipins accumulated during pathogen attack or wounding and potentially JA biosynthesis,<a reveal-id="dsw061-B72" data-open="dsw061-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> and a protein kinase gene (VIT_13s0019g01080) closely related to Arabidopsis <em>BOTRYTIS-INDUCED KINASE1</em><a reveal-id="dsw061-B73" data-open="dsw061-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> involved in the signaling cascade for defense response against pathogens. Other basic defense-related genes such as a pathogenesis-related protein (VIT_05s0077g01690) containing a GCC-box CRE is connected with <em>VviERF132</em>. Phenylpropanoid pathway genes encoding stilbene synthases (VIT_10s0042g00850 and VIT_10s0042g00860) and a cinnamoyl-CoA reductase (VIT_14s0066g01150) involved in the biosynthesis of stilbene and lignin compounds implicated in grapevine biotic stress response,<a reveal-id="dsw061-B58" data-open="dsw061-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsw061-B74" data-open="dsw061-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> were directly connected with <em>VviERF106</em> and <em>VviERF127</em>. One ethylene biosynthesis gene encoding 1-aminocyclopropane-1-carboxylic acid oxidase (VIT_12s0059g01380) was directly co-expressed with <em>VviERF107</em>. The gene expression profile of module genes involved in hormone biosynthesis (ethylene and JA), signalling, secondary metabolism, and transcriptional regulation, were induced by <em>B. cinerea</em> infection in berries (24 hpi), leaves upon herbivory attack by spider mite, and leaves with powdery mildew (5 days post-infection) in the susceptible genotype Carignan (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). This observation supports the participation of module genes and multiple GCC-related CREs in grapevine response to pathogens (necrotrophic and biotrophic) and insect herbivory, whereby activation of inducible defense mechanism via ethylene and JA pathways might confer basal defense against these biotic stressors.</p><p>In Module II, three AP2/ERF genes were present namely <em>VviERF2</em> (VIT_02s0234g00130), <em>VviERF017</em> (VIT_11s0016g00660), and <em>VviRRTF1</em> (VIT_03s0063g00460), of group IX (B-3), II (A-5), and X (B-4), respectively.<a reveal-id="dsw061-B70" data-open="dsw061-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Compared with the previous module, this module had an enrichment pertaining to transcription regulation and signaling with respect to AP2/ERFs and WRKYs (<a reveal-id="dsw061-T1" data-open="dsw061-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Three ERFs were selected as ‘seed’ or ‘guide’ ERFs to create a subnetwork, where <em>VviERF2</em> is central, connecting all nodes within this subnetwork (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). The homologs of Arabidopsis <em>WRKY33</em> (<em>VviWRKY33</em>, VIT_06s0004g07500) and <em>WRKY40</em> (<em>VviWRKY40</em>, VIT_09s0018g00240) were directly connected with <em>VviERF2</em>. In Arabidopsis, <em>WRKY33</em> modulates many hormonal and metabolic pathway genes, including <em>AtERF2</em>, and confers defense against <em>B.</em><em>cinerea</em>.<a reveal-id="dsw061-B75" data-open="dsw061-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> The presence of the GCC-box CRE in promoters of <em>VviWRKY33</em> and direct connectivity with <em>VviERF2</em> highlights a potential conserved regulation with that of Arabidopsis, where <em>VviWRKY33</em> might be a target of <em>VviERF2</em> in grapevines. Arabidopsis <em>WRKY40</em> also contributes to broad resistance against necrotrophs, biotrophs, and hemibiotrophs, through the modulation of JA and SA crosstalk when facing different pathogen lifestyles.<a reveal-id="dsw061-B76" data-open="dsw061-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> In addition, a NAC TF gene homologous to Arabidopsis <em>ATAF1</em> (<em>ATAF1</em>, VIT_07s0031g02610), was directly connected with <em>VviERF2</em>. Arabidopsis <em>ATAF1</em> is involved in both JA and ABA crosstalk and has dual roles in abiotic and biotic stress.<a reveal-id="dsw061-B77" data-open="dsw061-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> Together, the co-expression network surrounding <em>VviERF2</em>, <em>VviERF017</em>, and <em>VviRRTF1</em>, may be implicated in hormonal crosstalk during plant immune response and abiotic stress through cooperative action with WRKY and NAC TFs. Both abiotic and biotic transcriptional networks are known to be mediated by AP2/ERF, WRKY, and NAC TFs regulons in plants.<a reveal-id="dsw061-B6" data-open="dsw061-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw061-B78" data-open="dsw061-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> One pathogen-related secretory pathway gene encoding grapevine PENETRATION1 (<em>VviPEN1</em>, VIT_08s0032g01150) was directly co-expressed with <em>VviERF2</em> and <em>VviRRTF1</em>. The PEN1-mediated secretory pathway is an important defense strategy against powdery mildew, mediating penetration resistance.<a reveal-id="dsw061-B79" data-open="dsw061-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> The gene expression profile of the GCC module II subnetwork is similar to module I, concordant with the role of module genes involved in immunity against necrotrophs as observed in Arabidopsis (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). Striking differences were observed especially in grapevine leaves infected with <em>P. viticola</em>, as many module genes were down-regulated, potentially highlighting antagonism between JA and SA signaling pathway. Down-regulation of JA signaling might alleviate repression of SA signaling, and activate early defense-related pathways during <em>P. viticola</em> interaction.<a reveal-id="dsw061-B71" data-open="dsw061-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> In addition, module genes may play a role in the regulation of signaling pathways for flower abscission, as module genes were largely down-regulated by gibberellic acid treated flowers but up-regulated in shaded ones. Therefore, comparison of module I and II of the GCC network may reveal different aspects of transcriptional networks targeted by GCC-box and related sequences, such as the activation of defense mechanisms or the modulation of hormone signaling.</p>                            <h3 scrollto-destination=88473251 id="88473251" class="section-title" >3.7. Different cell wall remodelling genes are enriched in subnetworks of the CCWACC development-specific network</h3>
<p>Differences in biological function enrichments were also observed for many CRE submodules constructed from the developmental and stress-specific GCNs. For example, CCWACC commonly referred to as an AC-element, is one of the three main R2R3-MYB binding sites.<a reveal-id="dsw061-B10" data-open="dsw061-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> It showed <em>bona fide</em> CRE properties (<em>CF</em> 1.90, <em>Z-score</em> 3.61, and <em>FE</em> 1.86 in stress-responsive, and <em>CF</em> 1.87, <em>Z-score</em> 3.10, and <em>FE</em> 1.87 in development-dynamic genes), and was highly enriched for biological functions in both sets of genes. However, in the developmental dataset greater enrichment for biological pathways, especially cell wall metabolism, was observed compared with stress. Therefore, focus will be given to the development-specific CCWACC network.</p><p>Two modules in the CCWACC network were enriched with distinct cell wall remodeling gene families (<a reveal-id="dsw061-T2" data-open="dsw061-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The Module I (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>) contained genes related to primary cell wall-related functions, while the Module VI (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>) contained genes related to secondary cell wall metabolism. One MYB gene in module I (<em>VviMYB91A</em>, VIT_08s0007g00410) encodes an orthologue of Arabidopsis <em>MYB91/ASYMETRIC LEAVES 1</em>.<a reveal-id="dsw061-B80" data-open="dsw061-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> In Arabidopsis, <em>MYB91/ASYMETRIC LEAVES 1</em> regulates the development of several vegetative and reproductive organs including leaves and flowers.<a reveal-id="dsw061-B80" data-open="dsw061-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> Additionally, a <em>SQUAMOSA PROMOTER-BINDING PROTEIN 8</em> gene (<em>VviSPL8</em>, VIT_15s0021g02300), shown to regulate the floral and seed development in Arabidopsis,<a reveal-id="dsw061-B81" data-open="dsw061-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> was directly connected to <em>VviMYB91A.</em> Many cell wall remodelling genes involved in degradation and modification containing CCWACC CREs in their promoter were also present, indirectly and directly connected to <em>VviMYB91A</em> and <em>VviSPL8</em>, respectively. Examples include one gene encoding expansin B (<em>VviEXPB2</em>, VIT_12s0059g00190), pectin methylesterase (<em>VviPME3</em>, VIT_03s0017g01950), two predicted polygalacturonases (<em>VviPG</em>, VIT_05s0020g00420 and VIT_15s0046g02000), and orthologues of Arabidopsis Beta-<span class="small-caps">d</span>-xylosidase 4 (<em>VviXYL4</em>, VIT_12s0055g01180 and VIT_12s0121g00230). Module genes were highly expressed during the early growth stages and decreased during development of various vegetative (e.g. rachis, tendril, leaf, stem, and seedling) and in reproductive (e.g. inflorescence, flowering, berry skin, and seed) organs (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). This observation reveals a strong coordinated regulation of module genes and TFs (e.g. <em>VviMYB91A</em> and <em>VviSPL8</em>) which might be critical for precise growth control during early stages of development.</p>                        <a id="88473254" scrollto-destination="88473254"></a>
<div id="dsw061-F6" data-id="dsw061-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f6.png?Expires=2147483647&amp;Signature=hcinZFV1-gYtwkwBUD4hRvPoPYvpCmdsJadCt336ucwzdl7wk8-A9wiz5davs4FTGE9QNKmrsZcmKbvbsYIRv0BfKyrGEv-HiESixcgdLWyQSBVJdBFdrXcPPBM1lpS3VOBG8i-7zf0T2oNd6wYZLjJ~PveydgLssAAjQy9vThxfwcfmxENB~~zjeRTOqcgNh8Ra8sPTPfcjON-xHT8AdNxmpJBZw-9r5G~Don0iT3XQeO3NHMj-arvTNhAWhwQ8KjSz-CsXvdKwAsFou7ry5mAdl5uj~GMuG-Dau-ikSnOPd1upsUDcIdrs0KgGnpTb9~0HP3orKDHHvg-xwLpCuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Development-specific submodules extracted from AC-element (CCWACC) and AGATTCT CRE modules, and the expression profile of submodule genes in different tissues and developmental stages. The color from the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding CRE’s network module (upper right of each subnetwork) is shown by the black circle. Development-specific (A) G-box, (B) AC-element, and (C) GLK1 motif submodule extracted using bZIP, R2R3 MYB, and GLK1 genes as ‘seed’ or ‘guide’ genes, respectively. Descriptions for nodes (circle and square), circle subnodes, and edges are the same with Fig. 4A and B. Node color depict assigned functions for genes involved in general cell wall metabolism, cell wall signaling, secondary cell wall-specific metabolism, and photosynthesis. (D) Heat map depicts the gene expression (row Z-score normalized) of submodule genes in various tissues and across multiple developmental stages (see Supplementary Fig. S1B for descriptions)." path-from-xml="dsw061f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473254/dsw061f6.tif" path-from-xml="dsw061f6.tif" target="_blank">View large</a><a section="88473254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f6.png?Expires=2147483647&amp;Signature=suB-xmHXHG8D8oXZt7Kay5uM22g8K7wJKSD8DsglKW8Rx4EB4q1XEs58WSCRqtOUUEvf1me5ByPvGjtp8q3VBQGfY99icaOKAXOm35Hw8Hv-iaaR~ykXWxRvoE0AYqXOwtEEYaZHxL0hVqWMBJzW80AuQruO6Y1vNxPxhHh13cq~ZsWvobt5oqi0Unz-6b3RX7DAwgs7OdTO~mOWu6werSneQxPpDdh9g6NZ40jd3ZbtLPu~PPGmJIOLQ1lHWRNsBZEav~sUEmSMsGucgXRgQRvbkSMMvsoPKxMoPakzhPlpGn~q7UyCR0~WaiLSxFNQGM6TceAZspAKtzkYwgoDqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473254&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Development-specific submodules extracted from AC-element (CCWACC) and AGATTCT CRE modules, and the expression profile of submodule genes in different tissues and developmental stages. The color from the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding CRE’s network module (upper right of each subnetwork) is shown by the black circle. Development-specific (A) G-box, (B) AC-element, and (C) GLK1 motif submodule extracted using bZIP, R2R3 MYB, and GLK1 genes as ‘seed’ or ‘guide’ genes, respectively. Descriptions for nodes (circle and square), circle subnodes, and edges are the same with <a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>. Node color depict assigned functions for genes involved in general cell wall metabolism, cell wall signaling, secondary cell wall-specific metabolism, and photosynthesis. (D) Heat map depicts the gene expression (row <em>Z-score</em> normalized) of submodule genes in various tissues and across multiple developmental stages (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>B for descriptions).</p></div></div><div content-id="dsw061-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/m_dsw061f6.png?Expires=2147483647&amp;Signature=hcinZFV1-gYtwkwBUD4hRvPoPYvpCmdsJadCt336ucwzdl7wk8-A9wiz5davs4FTGE9QNKmrsZcmKbvbsYIRv0BfKyrGEv-HiESixcgdLWyQSBVJdBFdrXcPPBM1lpS3VOBG8i-7zf0T2oNd6wYZLjJ~PveydgLssAAjQy9vThxfwcfmxENB~~zjeRTOqcgNh8Ra8sPTPfcjON-xHT8AdNxmpJBZw-9r5G~Don0iT3XQeO3NHMj-arvTNhAWhwQ8KjSz-CsXvdKwAsFou7ry5mAdl5uj~GMuG-Dau-ikSnOPd1upsUDcIdrs0KgGnpTb9~0HP3orKDHHvg-xwLpCuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Development-specific submodules extracted from AC-element (CCWACC) and AGATTCT CRE modules, and the expression profile of submodule genes in different tissues and developmental stages. The color from the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding CRE’s network module (upper right of each subnetwork) is shown by the black circle. Development-specific (A) G-box, (B) AC-element, and (C) GLK1 motif submodule extracted using bZIP, R2R3 MYB, and GLK1 genes as ‘seed’ or ‘guide’ genes, respectively. Descriptions for nodes (circle and square), circle subnodes, and edges are the same with Fig. 4A and B. Node color depict assigned functions for genes involved in general cell wall metabolism, cell wall signaling, secondary cell wall-specific metabolism, and photosynthesis. (D) Heat map depicts the gene expression (row Z-score normalized) of submodule genes in various tissues and across multiple developmental stages (see Supplementary Fig. S1B for descriptions)." path-from-xml="dsw061f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473254/dsw061f6.tif" path-from-xml="dsw061f6.tif" target="_blank">View large</a><a section="88473254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061f6.png?Expires=2147483647&amp;Signature=suB-xmHXHG8D8oXZt7Kay5uM22g8K7wJKSD8DsglKW8Rx4EB4q1XEs58WSCRqtOUUEvf1me5ByPvGjtp8q3VBQGfY99icaOKAXOm35Hw8Hv-iaaR~ykXWxRvoE0AYqXOwtEEYaZHxL0hVqWMBJzW80AuQruO6Y1vNxPxhHh13cq~ZsWvobt5oqi0Unz-6b3RX7DAwgs7OdTO~mOWu6werSneQxPpDdh9g6NZ40jd3ZbtLPu~PPGmJIOLQ1lHWRNsBZEav~sUEmSMsGucgXRgQRvbkSMMvsoPKxMoPakzhPlpGn~q7UyCR0~WaiLSxFNQGM6TceAZspAKtzkYwgoDqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=88473254&amp;ar=2949522&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw061f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Development-specific submodules extracted from AC-element (CCWACC) and AGATTCT CRE modules, and the expression profile of submodule genes in different tissues and developmental stages. The color from the submodule name corresponds to the color on the left of the gene expression heat map. The position of each submodule within their corresponding CRE’s network module (upper right of each subnetwork) is shown by the black circle. Development-specific (A) G-box, (B) AC-element, and (C) GLK1 motif submodule extracted using bZIP, R2R3 MYB, and GLK1 genes as ‘seed’ or ‘guide’ genes, respectively. Descriptions for nodes (circle and square), circle subnodes, and edges are the same with <a reveal-id="dsw061-F4" data-open="dsw061-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>. Node color depict assigned functions for genes involved in general cell wall metabolism, cell wall signaling, secondary cell wall-specific metabolism, and photosynthesis. (D) Heat map depicts the gene expression (row <em>Z-score</em> normalized) of submodule genes in various tissues and across multiple developmental stages (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>B for descriptions).</p></div></div><p><div content-id="dsw061-T2" class="table-modal"><div class="table-wrap"><div id="dsw061-T2" data-id="dsw061-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>MapMan enrichment of selected MYB and GLK submodules in the development dataset</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>Mapman term </th></tr></thead><tbody><tr><td rowspan="13">CCWACC (MYB) </td><td>I </td><td>6 </td><td>87 </td><td>170 </td><td>1.09E-05 </td><td>3.71E-03 </td><td>Cell wall.degradation </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>17 </td><td>288 </td><td>4.78E-07 </td><td>1.30E-04 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>7 </td><td>17 </td><td>1,448 </td><td>8.11E-06 </td><td>1.57E-03 </td><td>Signalling.receptor kinases </td></tr><tr><td>V </td><td>2 </td><td>34 </td><td>6 </td><td>1.94E-05 </td><td>2.63E-02 </td><td>Hormone metabolism.jasmonate </td></tr><tr><td rowspan="3">VI </td><td>6 </td><td>43 </td><td>20 </td><td>2.58E-13 </td><td>1.17E-10 </td><td>Cell wall.hemicellulose synthesis </td></tr><tr><td>5 </td><td>43 </td><td>98 </td><td>3.15E-07 </td><td>1.07E-04 </td><td>Cell wall.cellulose synthesis </td></tr><tr><td>3 </td><td>43 </td><td>47 </td><td>4.44E-05 </td><td>1.21E-02 </td><td>Cell wall.cell wall proteins </td></tr><tr><td>VII </td><td>7 </td><td>15 </td><td>498 </td><td>2.15E-09 </td><td>1.46E-06 </td><td>Stress.abiotic </td></tr><tr><td>VIII </td><td>9 </td><td>9 </td><td>268 </td><td>3.77E-19 </td><td>5.12E-16 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td rowspan="2">IX </td><td>30 </td><td>78 </td><td>389 </td><td>2.66E-36 </td><td>9.04E-34 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>21 </td><td>78 </td><td>181 </td><td>4.47E-29 </td><td>8.67E-27 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>X </td><td>3 </td><td>6 </td><td>512 </td><td>9.94E-05 </td><td>4.50E-02 </td><td>DNA </td></tr><tr><td>XV </td><td>2 </td><td>5 </td><td>8 </td><td>6.46E-07 </td><td>8.78E-04 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td rowspan="2">AGATTCT </td><td>I </td><td>7 </td><td>15 </td><td>127 </td><td>1.47E-13 </td><td>9.98E-11 </td><td>PS.lightreaction </td></tr><tr><td>II </td><td>5 </td><td>21 </td><td>127 </td><td>2.63E-08 </td><td>1.79E-05 </td><td>PS.lightreaction </td></tr></tbody></table></div><div content-id="dsw061-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>Mapman term </th></tr></thead><tbody><tr><td rowspan="13">CCWACC (MYB) </td><td>I </td><td>6 </td><td>87 </td><td>170 </td><td>1.09E-05 </td><td>3.71E-03 </td><td>Cell wall.degradation </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>17 </td><td>288 </td><td>4.78E-07 </td><td>1.30E-04 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>7 </td><td>17 </td><td>1,448 </td><td>8.11E-06 </td><td>1.57E-03 </td><td>Signalling.receptor kinases </td></tr><tr><td>V </td><td>2 </td><td>34 </td><td>6 </td><td>1.94E-05 </td><td>2.63E-02 </td><td>Hormone metabolism.jasmonate </td></tr><tr><td rowspan="3">VI </td><td>6 </td><td>43 </td><td>20 </td><td>2.58E-13 </td><td>1.17E-10 </td><td>Cell wall.hemicellulose synthesis </td></tr><tr><td>5 </td><td>43 </td><td>98 </td><td>3.15E-07 </td><td>1.07E-04 </td><td>Cell wall.cellulose synthesis </td></tr><tr><td>3 </td><td>43 </td><td>47 </td><td>4.44E-05 </td><td>1.21E-02 </td><td>Cell wall.cell wall proteins </td></tr><tr><td>VII </td><td>7 </td><td>15 </td><td>498 </td><td>2.15E-09 </td><td>1.46E-06 </td><td>Stress.abiotic </td></tr><tr><td>VIII </td><td>9 </td><td>9 </td><td>268 </td><td>3.77E-19 </td><td>5.12E-16 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td rowspan="2">IX </td><td>30 </td><td>78 </td><td>389 </td><td>2.66E-36 </td><td>9.04E-34 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>21 </td><td>78 </td><td>181 </td><td>4.47E-29 </td><td>8.67E-27 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>X </td><td>3 </td><td>6 </td><td>512 </td><td>9.94E-05 </td><td>4.50E-02 </td><td>DNA </td></tr><tr><td>XV </td><td>2 </td><td>5 </td><td>8 </td><td>6.46E-07 </td><td>8.78E-04 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td rowspan="2">AGATTCT </td><td>I </td><td>7 </td><td>15 </td><td>127 </td><td>1.47E-13 </td><td>9.98E-11 </td><td>PS.lightreaction </td></tr><tr><td>II </td><td>5 </td><td>21 </td><td>127 </td><td>2.63E-08 </td><td>1.79E-05 </td><td>PS.lightreaction </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><em>x</em>, overlap size; <em>n</em>, query size; and <em>k</em>, term size represent the contingency tables for MapMan enrichment calculations.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw061-T2" data-id="dsw061-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>MapMan enrichment of selected MYB and GLK submodules in the development dataset</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>Mapman term </th></tr></thead><tbody><tr><td rowspan="13">CCWACC (MYB) </td><td>I </td><td>6 </td><td>87 </td><td>170 </td><td>1.09E-05 </td><td>3.71E-03 </td><td>Cell wall.degradation </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>17 </td><td>288 </td><td>4.78E-07 </td><td>1.30E-04 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>7 </td><td>17 </td><td>1,448 </td><td>8.11E-06 </td><td>1.57E-03 </td><td>Signalling.receptor kinases </td></tr><tr><td>V </td><td>2 </td><td>34 </td><td>6 </td><td>1.94E-05 </td><td>2.63E-02 </td><td>Hormone metabolism.jasmonate </td></tr><tr><td rowspan="3">VI </td><td>6 </td><td>43 </td><td>20 </td><td>2.58E-13 </td><td>1.17E-10 </td><td>Cell wall.hemicellulose synthesis </td></tr><tr><td>5 </td><td>43 </td><td>98 </td><td>3.15E-07 </td><td>1.07E-04 </td><td>Cell wall.cellulose synthesis </td></tr><tr><td>3 </td><td>43 </td><td>47 </td><td>4.44E-05 </td><td>1.21E-02 </td><td>Cell wall.cell wall proteins </td></tr><tr><td>VII </td><td>7 </td><td>15 </td><td>498 </td><td>2.15E-09 </td><td>1.46E-06 </td><td>Stress.abiotic </td></tr><tr><td>VIII </td><td>9 </td><td>9 </td><td>268 </td><td>3.77E-19 </td><td>5.12E-16 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td rowspan="2">IX </td><td>30 </td><td>78 </td><td>389 </td><td>2.66E-36 </td><td>9.04E-34 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>21 </td><td>78 </td><td>181 </td><td>4.47E-29 </td><td>8.67E-27 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>X </td><td>3 </td><td>6 </td><td>512 </td><td>9.94E-05 </td><td>4.50E-02 </td><td>DNA </td></tr><tr><td>XV </td><td>2 </td><td>5 </td><td>8 </td><td>6.46E-07 </td><td>8.78E-04 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td rowspan="2">AGATTCT </td><td>I </td><td>7 </td><td>15 </td><td>127 </td><td>1.47E-13 </td><td>9.98E-11 </td><td>PS.lightreaction </td></tr><tr><td>II </td><td>5 </td><td>21 </td><td>127 </td><td>2.63E-08 </td><td>1.79E-05 </td><td>PS.lightreaction </td></tr></tbody></table></div><div content-id="dsw061-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>CRE </th><th>Module </th><th><em>x</em> </th><th><em>n</em> </th><th><em>k</em> </th><th><em>P</em> value </th><th>FDR </th><th>Mapman term </th></tr></thead><tbody><tr><td rowspan="13">CCWACC (MYB) </td><td>I </td><td>6 </td><td>87 </td><td>170 </td><td>1.09E-05 </td><td>3.71E-03 </td><td>Cell wall.degradation </td></tr><tr><td rowspan="2">II </td><td>5 </td><td>17 </td><td>288 </td><td>4.78E-07 </td><td>1.30E-04 </td><td>Lipid metabolism.lipid degradation </td></tr><tr><td>7 </td><td>17 </td><td>1,448 </td><td>8.11E-06 </td><td>1.57E-03 </td><td>Signalling.receptor kinases </td></tr><tr><td>V </td><td>2 </td><td>34 </td><td>6 </td><td>1.94E-05 </td><td>2.63E-02 </td><td>Hormone metabolism.jasmonate </td></tr><tr><td rowspan="3">VI </td><td>6 </td><td>43 </td><td>20 </td><td>2.58E-13 </td><td>1.17E-10 </td><td>Cell wall.hemicellulose synthesis </td></tr><tr><td>5 </td><td>43 </td><td>98 </td><td>3.15E-07 </td><td>1.07E-04 </td><td>Cell wall.cellulose synthesis </td></tr><tr><td>3 </td><td>43 </td><td>47 </td><td>4.44E-05 </td><td>1.21E-02 </td><td>Cell wall.cell wall proteins </td></tr><tr><td>VII </td><td>7 </td><td>15 </td><td>498 </td><td>2.15E-09 </td><td>1.46E-06 </td><td>Stress.abiotic </td></tr><tr><td>VIII </td><td>9 </td><td>9 </td><td>268 </td><td>3.77E-19 </td><td>5.12E-16 </td><td>Secondary metabolism.isoprenoids </td></tr><tr><td rowspan="2">IX </td><td>30 </td><td>78 </td><td>389 </td><td>2.66E-36 </td><td>9.04E-34 </td><td>Secondary metabolism.flavonoids </td></tr><tr><td>21 </td><td>78 </td><td>181 </td><td>4.47E-29 </td><td>8.67E-27 </td><td>Secondary metabolism.phenylpropanoids </td></tr><tr><td>X </td><td>3 </td><td>6 </td><td>512 </td><td>9.94E-05 </td><td>4.50E-02 </td><td>DNA </td></tr><tr><td>XV </td><td>2 </td><td>5 </td><td>8 </td><td>6.46E-07 </td><td>8.78E-04 </td><td>Lipid metabolism.TAG synthesis </td></tr><tr><td rowspan="2">AGATTCT </td><td>I </td><td>7 </td><td>15 </td><td>127 </td><td>1.47E-13 </td><td>9.98E-11 </td><td>PS.lightreaction </td></tr><tr><td>II </td><td>5 </td><td>21 </td><td>127 </td><td>2.63E-08 </td><td>1.79E-05 </td><td>PS.lightreaction </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><em>x</em>, overlap size; <em>n</em>, query size; and <em>k</em>, term size represent the contingency tables for MapMan enrichment calculations.</p></div></div></div></div></p><p>Module VI contained a homolog of Arabidopsis <em>MYB83</em> (<em>VviMYB185</em>, VIT_06s0004g02110), which is widely implicated in the regulation of secondary cell wall biosynthesis components.<a reveal-id="dsw061-B82" data-open="dsw061-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> Accordingly, genes directly connected to <em>VviMYB185</em> are widely implicated in secondary cell wall biosynthesis and deposition. These include a fasciclin arabinogalactan-protein (<em>VviFLA11</em>, VIT_06s0004g03050), a glucuronyltransferase (<em>VviGUX2</em>, VIT_18s0001g10580), a galacturonosyltransferase (<em>VviGAUT12</em>, VIT_19s0014g05130), a COBRA protein (<em>VviCOBL4</em>, VIT_14s0083g01150), one cellulose synthase (<em>VviCESA8</em>, VIT_10s0003g01560), and two laccase (<em>VviLAC17</em>, VIT_13s0019g01920, VIT_13s0019g02160) genes. Interestingly, one <em>NAC</em> TF (<em>VviNST1</em>, VIT_02s0025g02710), homolog to Arabidopsis <em>NST1</em>, was directly connected with <em>VviMYB185</em>. In Arabidopsis, both <em>NST1</em> and <em>MYB83</em> are master switches for secondary cell wall development, albeit <em>NST1</em> function as the primary master switch while <em>MYB83</em> is downstream in the hierarchy.<a reveal-id="dsw061-B83" data-open="dsw061-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> Meanwhile, an additional battery of secondary cell wall-related genes indirectly connected with <em>VviMYB185</em> encoding a cellulose synthase (<em>VviCESA7</em>, VIT_11s0037g00530), a laccase (<em>VviLAC4</em>, VIT_06s0004g06090), a galacturonosyltransferase-like protein (<em>VviGATL1</em>, VIT_04s0023g01120), a beta-xylosidase (<em>VviBXL1</em>, VIT_05s0077g01280), and two xylan synthases (<em>VviIRX9</em>, VIT_08s0040g02340) contain CCWACC CRE in their promoters. The latter genes may also be potential regulatory targets of <em>VviMYB185</em> or of other R2R3-MYBs. Supporting this idea, many secondary cell wall genes in Arabidopsis that are present in the <em>VviMYB185</em> network are known to be regulated by R2R3-MYB TFs (including MYB83) through the AC-element and related sequences.<a reveal-id="dsw061-B83" data-open="dsw061-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a><sup>,</sup><a reveal-id="dsw061-B84" data-open="dsw061-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> Compared with Module I of the developmental CCWACC network, this module’s activation was mainly present at the early development stages of the seed, rachis, and stem, and in the late stages of the inflorescence and tendrils (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). The difference in cell wall metabolism pathway enrichment is not limited to the CCWACC network, but to other developmental submodules that are potentially regulated by ARF (TGTCGG and TGTCGA),<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw061-B11" data-open="dsw061-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> NAC (TTGCGT and TTCCTT),<a reveal-id="dsw061-B85" data-open="dsw061-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> and AP2/ERF (ACCGACA).<a reveal-id="dsw061-B6" data-open="dsw061-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw061-B43" data-open="dsw061-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p>                            <h3 scrollto-destination=88473257 id="88473257" class="section-title" >3.8. AGATTCT developmental network module is specifically enriched in photosynthesis-related genes</h3>
<p>Although CREs and CRE-driven subnetworks, such as those previously discussed, show that strong positional bias towards the TSS and <em>FE</em> scores provide reasonable proxies for identifying <em>bona fide</em> CREs and biologically relevant modules, some CREs and CRE-driven subnetworks lack these properties. For example, the 7-mer CRE AGATTCT (<em>CF</em> 2.69, <em>Z-score</em> 0.13, and <em>FE</em> 6.91 in the developmental dataset), whose network contains a G2-like TF. DNA-binding specificities of lesser known plant TF families such as G2-like, a subclass of GARP proteins, have been recently characterized.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The G2-like TF, GLK, is implicated in photosynthesis and chloroplast development in many plants<a reveal-id="dsw061-B86" data-open="dsw061-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> and is a key factor regulating the latitudinal gradient of ripening<a reveal-id="dsw061-B87" data-open="dsw061-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> and various quality traits in fruits.<a reveal-id="dsw061-B88" data-open="dsw061-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> Arabidopsis GLK1 recognizes the AGATTCT CRE sequence.<a reveal-id="dsw061-B5" data-open="dsw061-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The AGATTCT network in the developmental dataset contained 113 genes and was highly enriched for many photosynthesis-related genes, especially in modules I and II (<a reveal-id="dsw061-T2" data-open="dsw061-T2" class="link link-reveal link-table xref-fig">Table 2</a>) suggesting that GLK-regulated gene networks are highly conserved across plants including grapevine. Present in module I (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>) are <em>VviGLK1</em>, the homolog of Arabidopsis GLK1 (<em>VviGLK1</em>, VIT_12s0028g03100), and its likely target genes which encode for photosystem I polypeptide subunits (<em>VviPSAN</em>, VIT_04s0044g01410; <em>VviPSAF</em>, VIT_00s0125g00280; <em>VviPSAD-2</em>, VIT_05s0020g03180) and light harvesting complexes (<em>VviCAB1-1</em>, VIT_10s0003g02890; <em>VviCAB1-2</em>, VIT_10s0003g02900; <em>VviLHB6</em>, VIT_12s0055g01110). One gene involved in cellular redox homeostasis (<em>VviCDSP32</em>, VIT_18s0001g00820) was also implicated in this module. The expression survey across the developmental dataset revealed that module I genes are highly expressed in immature organs of the berry pericarp, flesh, skin, and seed, as well as in rachis, buds, and leaves, and gradually decrease their expression as development of each organ progresses (<a reveal-id="dsw061-F6" data-open="dsw061-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). Therefore, the consistent down-regulation of module genes may reflect an active role of photosynthetic genes and <em>VviGLK1</em> in the immature-to-mature phase transition of grapevine organs, similar to the observations in other plants.<a reveal-id="dsw061-B86" data-open="dsw061-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> Additionally in grape berries, these module genes may be critical for the establishment of differing ripening gradients during the onset of ripening, as seen with the role of GLK1 in tomatoes.<a reveal-id="dsw061-B87" data-open="dsw061-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a><sup>,</sup><a reveal-id="dsw061-B88" data-open="dsw061-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a></p>                            <h3 scrollto-destination=88473259 id="88473259" class="section-title" >3.9. Comparison between grapevine and Arabidopsis motif-driven GCNs</h3>
<p>Compared with other plants, CRE-driven GCNs have only been elucidated in Arabidopsis; several of which include the G-box, MYB (CCWACC), WRKY (KTTGACY), and the site II element (TGGGCY) CRE modules.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Based on the conserved G-box and CCWACC CRE, which are also evaluated in this study, many enriched (GO enrichment) biological pathways in Arabidopsis are shared with those identified in this study. For example, highly enriched terms related to photosynthesis, hormone (ethylene and ABA) metabolism, and stress responses were common in G-box CRE modules between grapevine (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) and Arabidopsis.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Similarly, secondary metabolic pathways, especially related to flavonoid and phenylpropanoid metabolism, cell wall, and stress responses, were common for the CCWACC (MYB) CRE module. To determine if other grapevine CRE modules harbor unique and/or common features at a larger scale, we expanded the search in Arabidopsis by constructing CRE modules (<em>P</em> &lt; 0.01) using the Arabidopsis graphical Gaussian model gene network with a partial correlation threshold &gt; 0.05 (Ma et al.<a reveal-id="dsw061-B13" data-open="dsw061-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). We then determined their biological pathway (Mapman) enrichment profiles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>) and performed the comparison with CRE modules from grapevine. In addition to the G-box and CCWACC CRE, some grapevine CRE modules share a high degree of similar gene enrichments with Arabidopsis CRE modules which includes ACGTGKC (ABRE), TGTCGG, TTTCCCGC, and YAACKG (MYB) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>), suggesting similar properties of gene expression and gene networks between grapevine and Arabidopsis. However, others share a small proportion of enriched pathways and/or have entirely unique enriched terms such as GCCGCC, RCCGAC, TGACGTG (bZIP), TTGACC (W-box), TGTCGA, TGTCTC, TTGCGT (NAC), and TTCCTT (NAC) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). For example, enriched terms in the Arabidopsis GCC-box CRE modules include biotic stress and ATP synthesis. In comparison, the grapevine stress-dependent GCC-box CRE modules share enrichment for ATP synthesis but have unique enriched terms related to hormone signaling—jasmonate and ethylene, TF regulation, and flavonoid and phenylpropanoid metabolism. Similarly, CRE modules such as DRE (A/GCCGAC) conserved between Arabidopsis and grapevine share enrichment profiles for galactinol/raffinose metabolism but differ in specific cell wall metabolic processes, hormone metabolism, and stress responses. Unique enrichment profiles for the grapevine DRE module include lipid metabolism and photosynthesis. These differences might indicate potential diversification of utilized CREs between grapevine and Arabidopsis that maintain the enriched CRE modules but with differing biological functions. Nonetheless, parameters used to construct the respective CRE modules which include different underlying experimental conditions, correlation metrics, and CRE <em>P</em>-value threshold may also introduce differences.</p><p>Comparative expression analysis of grapevine genes (assigned to CRE modules) with corresponding Arabidopsis orthologues may also reveal potential similarities and differences in expression dynamics between species. Genes of selected CRE modules such as G-box and GCC-core modules were further evaluated considering the responses of their putative Arabidopsis orthologues under similar and additional treatments (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). For example, 123 (of the 189) genes in the grapevine stress-specific G-box module I have predicted Arabidopsis orthologues. Arabidopsis orthologues for these genes are largely induced in various abiotic stress conditions especially cold, osmotic stress, and salt, but also to UV-B, drought, and heat stress albeit at a lesser extent (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>A). Biotic stress pathways are also highly modulated in the G-box module I of Arabidopsis and a greater proportion of orthologues are down-regulated against different biotic stress including <em>B. cinerea</em> and powdery mildew, as seen in grapevine (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>B). In addition, in module I (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>C) and II (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>D) of the stress-specific GCC-core module, there are 47 (of 102) and 66 (of 94) grapevine genes which have Arabidopsis orthologues, respectively. Like grapevine, Arabidopsis orthologues for these genes were highly induced upon a wide variety elicitors and plant–pathogen treatments. Although similar gene expression responses were observed for <em>B. cinerea</em> and powdery mildew infection for grapevine and Arabidopsis, an opposite response (induction) was observed for Arabidopsis infection with <em>H. arabidopsidis</em>, the causal agent of downy mildew (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>C and D), compared with grapevine (<a reveal-id="dsw061-F5" data-open="dsw061-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). This highlights potential differences in the transcriptional networks of plant immunity employed by the two species that may involve differing CRE regulation. Nonetheless, the strong and consistent modulation of gene orthologues of CRE modules between grapevine and Arabidopsis suggest a general conservation in promoter architecture, gene expression dynamics, and gene regulatory networks.</p>                            <h2 scrollto-destination=88473262 id="88473262" class="section-title" >4. Conclusion</h2>
<p>In this study, we have elucidated the properties of a large selection of known plant CREs in grapevine promoters at the genome-scale. We observed that many CREs fulfill the widely-adopted criterions defining authentic plant CREs. We corroborated these findings through extensive biological pathway enrichment analysis suggesting the characterized CRE structure has strong biological relevance. Together, these assessments have elucidated many potential targets of a large repertoire of grapevine TFs for the first time. Large-scale transcriptome analysis using the developmental atlas dataset and a selection of stress-related RNA-seq datasets demonstrated that the distribution of CREs in gene promoters explains close to half of the DE across the various abiotic and biotic stresses, and developmental comparisons evaluated. However, several exceptions exist where well-characterized plant CREs does not possess authentic CRE properties and/or strong enrichment, highlighting the potential diversification of CREs in grapevine promoters and divergence of transcriptional regulatory pathways. Therefore, additional scanning approaches or other statistical models, such as scans using position weight matrices or all possible <em>k</em>-mer nucleotide combinations, might be necessary to capture the full spectrum of grapevine CREs and their biological functions. Nonetheless, by integrating condition-specific GCN, CRE enrichment, and gene expression profiles, numerous condition-specific CRE-driven modules (and submodules) were discovered, providing additional information on the coordinated regulation of submodule genes. Although some of these genes may not contain the given CRE, they are implicated in similar functions, specificity of stress responses, and/or gene expression pattern in tissues/organs within and between CRE-driven submodules. This integrated approach uncovered biologically meaningful submodules in a more robust fashion than just utilizing the distribution of CREs alone. This is exemplified by the modules that corroborated known roles in drought response, pathogen defense, cell wall metabolism, and fruit ripening in grapevine and in other plant species. Some modules reveal novel regulatory functions, although further functional validation will be necessary to confirm the transcriptional control exerted by the large TF repertoire in grapevines. Application of these approaches is not just limited to grapevines but to other sequenced plant genomes, especially crop species,<a reveal-id="dsw061-B89 dsw061-B90 dsw061-B91" data-open="dsw061-B89 dsw061-B90 dsw061-B91" class="link link-ref link-reveal xref-bibr"><sup>89–91</sup></a> permitting a greater understanding of the evolution and conservation of plant CREs and transcriptional regulatory pathways.</p>                            <h2 scrollto-destination=88473264 id="88473264" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the grapevine research community for making various RNA-seq and microarray datasets publicly available.</p>                            <h3 scrollto-destination=88473266 id="88473266" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=88473268 id="88473268" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=88473270 id="88473270" class="section-title" >Funding</h3>
<p>This study was funded by the University of British Columbia (10R18459), Genome British Columbia (10R21188), and the Natural Sciences and Engineering Research Council of Canada (10R23082).</p>                            <h2 scrollto-destination=88473272 id="88473272" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw061-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">1182</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantTFDB%203.0%3A%20a%20portal%20for%20the%20functional%20and%20evolutionary%20study%20of%20plant%20transcription%20factors&amp;author=J.%20Jin&amp;author=H.%20Zhang&amp;author=L.%20Kong&amp;author=G.%20Gao&amp;author=J.%20Luo&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=1182-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1016"> </span></div> </div></div></div></div></div><div content-id="dsw061-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Riechmann</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Ratcliffe</div> <div class="given-names">O. J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">A genomic perspective on plant transcription factors</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">3</div>, <div class="fpage">423</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20perspective%20on%20plant%20transcription%20factors&amp;author=J.%20L.%20Riechmann&amp;author=O.%20J.%20Ratcliffe&amp;publication_year=2000&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=3&amp;pages=423-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(00)00107-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(00)00107-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(00)00107-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11019812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higo</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ugawa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Iwamoto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Korenaga</div> <div class="given-names">T.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Plant cis-acting regulatory DNA elements (PLACE) database: 1999</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">297</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20cis-acting%20regulatory%20DNA%20elements%20(PLACE)%20database%3A%201999&amp;author=K.%20Higo&amp;author=Y.%20Ugawa&amp;author=M.%20Iwamoto&amp;author=T.%20Korenaga&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=297-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yilmaz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mejia-Guerra</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Kurz</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Welch</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Grotewold</div> <div class="given-names">E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">AGRIS: the arabidopsis gene regulatory information server, an update</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D1118</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AGRIS%3A%20the%20arabidopsis%20gene%20regulatory%20information%20server%2C%20an%20update&amp;author=A.%20Yilmaz&amp;author=M.%20K.%20Mejia-Guerra&amp;author=K.%20Kurz&amp;author=X.%20Liang&amp;author=L.%20Welch&amp;author=E.%20Grotewold&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D1118-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21059685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Franco-Zorrilla</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">López-Vidriero</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Carrasco</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Godoy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Vera</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Solano</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">DNA-binding specificities of plant transcription factors and their potential to define target genes</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">111</div>, <div class="fpage">2367</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-binding%20specificities%20of%20plant%20transcription%20factors%20and%20their%20potential%20to%20define%20target%20genes&amp;author=J.%20M.%20Franco-Zorrilla&amp;author=I.%20L%C3%B3pez-Vidriero&amp;author=J.%20L.%20Carrasco&amp;author=M.%20Godoy&amp;author=P.%20Vera&amp;author=R.%20Solano&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=111&amp;pages=2367-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1316278111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1316278111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1316278111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24477691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Transcriptional regulatory networks in cellular responses and tolerance to dehydration and cold stresses</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">57</div>, <div class="fpage">781</div>–<div class="lpage">803</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulatory%20networks%20in%20cellular%20responses%20and%20tolerance%20to%20dehydration%20and%20cold%20stresses&amp;author=K.%20Yamaguchi-Shinozaki&amp;author=K.%20Shinozaki&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=57&amp;pages=781-803" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gramzow</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Theissen</div> <div class="given-names">G.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A hitchhiker’s guide to the MADS world of plants</div>. <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hitchhiker%E2%80%99s%20guide%20to%20the%20MADS%20world%20of%20plants&amp;author=L.%20Gramzow&amp;author=G.%20Theissen&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-6-214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-6-214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-6-214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20587009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chandler</div> <div class="given-names">J. W.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Auxin response factors</div>. <div class="source">Plant. Cell Environ</div> ., <div class="volume">39</div>, <div class="fpage">1014</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Auxin%20response%20factors&amp;author=J.%20W.%20Chandler&amp;publication_year=2016&amp;journal=Plant.%20Cell%20Environ&amp;volume=39&amp;pages=1014-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.12662" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.12662" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.12662"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26487015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vandepoele</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Vlieghe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Florquin</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Genome-wide identification of potential plant E2F target genes</div>. <div class="source">Plant Physiol</div> ., <div class="volume">139</div>, <div class="fpage">316</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20potential%20plant%20E2F%20target%20genes&amp;author=K.%20Vandepoele&amp;author=K.%20Vlieghe&amp;author=K.%20Florquin&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=139&amp;pages=316-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.066290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.066290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.066290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16126853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Prouse</div> <div class="given-names">M. B.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">M. M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The interaction between MYB proteins and their target DNA binding sites</div>. <div class="source">Biochim. Biophys. Acta – Gene Regul. Mech</div> ., <div class="volume">1819</div>, <div class="fpage">67</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20interaction%20between%20MYB%20proteins%20and%20their%20target%20DNA%20binding%20sites&amp;author=M.%20B.%20Prouse&amp;author=M.%20M.%20Campbell&amp;publication_year=2012&amp;journal=Biochim.%20Biophys.%20Acta%20%E2%80%93%20Gene%20Regul.%20Mech&amp;volume=1819&amp;pages=67-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagrm.2011.10.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2011.10.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagrm.2011.10.010"> </span></div> </div></div></div></div></div><div content-id="dsw061-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boer</div> <div class="given-names">D. R.</div></div>, <div class="name string-name"><div class="surname">Freire-Rios</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Van Den Berg</div> <div class="given-names">W. A. M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Structural basis for DNA binding specificity by the auxin-dependent ARF transcription factors</div>. <div class="source">Cell</div> , <div class="volume">156</div>, <div class="fpage">577</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20DNA%20binding%20specificity%20by%20the%20auxin-dependent%20ARF%20transcription%20factors&amp;author=D.%20R.%20Boer&amp;author=A.%20Freire-Rios&amp;author=W.%20A.%20M.%20Van%20Den%20Berg&amp;publication_year=2014&amp;journal=Cell&amp;volume=156&amp;pages=577-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.12.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.12.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.12.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24485461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zou</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mackaluso</div> <div class="given-names">J. D.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">14992</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cis-regulatory%20code%20of%20stress-responsive%20transcription%20in%20Arabidopsis%20thaliana&amp;author=C.%20Zou&amp;author=K.%20Sun&amp;author=J.%20D.%20Mackaluso&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=14992-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1103202108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1103202108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1103202108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21849619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shah</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bohnert</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Snyder</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dinesh-Kumar</div> <div class="given-names">S. P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Incorporating motif analysis into gene co-expression networks reveals novel modular expression pattern and new signaling pathways</div>. <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003840</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incorporating%20motif%20analysis%20into%20gene%20co-expression%20networks%20reveals%20novel%20modular%20expression%20pattern%20and%20new%20signaling%20pathways&amp;author=S.%20Ma&amp;author=S.%20Shah&amp;author=H.%20J.%20Bohnert&amp;author=M.%20Snyder&amp;author=S.%20P.%20Dinesh-Kumar&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003840" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24098147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vandepoele</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Quimbaya</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Casneuf</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">De Veylder</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Unraveling transcriptional control in Arabidopsis using cis-regulatory elements and coexpression networks</div>. <div class="source">Plant Physiol</div> ., <div class="volume">150</div>, <div class="fpage">535</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unraveling%20transcriptional%20control%20in%20Arabidopsis%20using%20cis-regulatory%20elements%20and%20coexpression%20networks&amp;author=K.%20Vandepoele&amp;author=M.%20Quimbaya&amp;author=T.%20Casneuf&amp;author=L.%20De%20Veylder&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=150&amp;pages=535-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.136028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.136028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.136028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19357200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Todaka</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mizoi</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Identification of cis-acting promoter elements in cold- and dehydration-induced transcriptional pathways in Arabidopsis, Rice, and Soybean</div>. <div class="source">DNA Res</div> ., <div class="volume">19</div>, <div class="fpage">37</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20cis-acting%20promoter%20elements%20in%20cold-%20and%20dehydration-induced%20transcriptional%20pathways%20in%20Arabidopsis%2C%20Rice%2C%20and%20Soybean&amp;author=K.%20Maruyama&amp;author=D.%20Todaka&amp;author=J.%20Mizoi&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=37-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22184637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ding</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Thousands of cis-regulatory sequence combinations are shared by Arabidopsis and Poplar</div>. <div class="source">Plant Physiol</div> ., <div class="volume">158</div>, <div class="fpage">145</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Thousands%20of%20cis-regulatory%20sequence%20combinations%20are%20shared%20by%20Arabidopsis%20and%20Poplar&amp;author=J.%20Ding&amp;author=H.%20Hu&amp;author=X.%20Li&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=158&amp;pages=145-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.186080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.186080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.186080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22058225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">Y. Y.</div></div>, <div class="name string-name"><div class="surname">Ichida</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsui</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Identification of plant promoter constituents by analysis of local distribution of short sequences</div>. <div class="source">BMC Genomics</div> , <div class="volume">8</div>, <div class="fpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20plant%20promoter%20constituents%20by%20analysis%20of%20local%20distribution%20of%20short%20sequences&amp;author=Y.%20Y.%20Yamamoto&amp;author=H.%20Ichida&amp;author=M.%20Matsui&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-67"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17346352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">FitzGerald</div> <div class="given-names">P. C.</div></div>, <div class="name string-name"><div class="surname">Shlyakhtenko</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mir</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Vinson</div> <div class="given-names">C.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Clustering of DNA sequences in human promoters</div>. <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">1562</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustering%20of%20DNA%20sequences%20in%20human%20promoters&amp;author=P.%20C.%20FitzGerald&amp;author=A.%20Shlyakhtenko&amp;author=A.%20A.%20Mir&amp;author=C.%20Vinson&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=1562-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1953904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1953904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1953904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15256515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nishida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kasahara</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">ATTED-II updates: condition-specific gene coexpression to extend coexpression analyses and applications to a broad range of flowering plants. <em>Plant</em></div><div class="source">Cell Physiol</div> ., <div class="volume">52</div>, <div class="fpage">213</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ATTED-II%20updates%3A%20condition-specific%20gene%20coexpression%20to%20extend%20coexpression%20analyses%20and%20applications%20to%20a%20broad%20range%20of%20flowering%20plants.%20Plant&amp;author=T.%20Obayashi&amp;author=K.%20Nishida&amp;author=K.%20Kasahara&amp;author=K.%20Kinoshita&amp;publication_year=2011&amp;journal=Cell%20Physiol&amp;volume=52&amp;pages=213-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcq203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcq203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcq203"> </span></div> </div></div></div></div></div><div content-id="dsw061-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aoki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Okamura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tadaka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">ATTED-II in 2016: a plant coexpression database towards lineage-specific coexpression</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">57</div>, <div class="fpage">e5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ATTED-II%20in%202016%3A%20a%20plant%20coexpression%20database%20towards%20lineage-specific%20coexpression&amp;author=Y.%20Aoki&amp;author=Y.%20Okamura&amp;author=S.%20Tadaka&amp;author=K.%20Kinoshita&amp;author=T.%20Obayashi&amp;publication_year=2016&amp;journal=Plant%20Cell%20Physiol&amp;volume=57&amp;pages=e5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcv165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcv165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcv165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26546318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mutwil</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Klie</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tohge</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">PlaNet: combined sequence and expression comparisons across plant networks derived from seven species</div>. <div class="source">Plant Cell</div> , <div class="volume">23</div>, <div class="fpage">895</div>–<div class="lpage">910</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlaNet%3A%20combined%20sequence%20and%20expression%20comparisons%20across%20plant%20networks%20derived%20from%20seven%20species&amp;author=M.%20Mutwil&amp;author=S.%20Klie&amp;author=T.%20Tohge&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=895-910" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.083667" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.083667" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.083667"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21441431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higashi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Network analysis for gene discovery in plant-specialized metabolism</div>. <div class="source">Plant. Cell Environ</div> ., <div class="volume">36</div>, <div class="fpage">1597</div>–<div class="lpage">606</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Network%20analysis%20for%20gene%20discovery%20in%20plant-specialized%20metabolism&amp;author=Y.%20Higashi&amp;author=K.%20Saito&amp;publication_year=2013&amp;journal=Plant.%20Cell%20Environ&amp;volume=36&amp;pages=1597-606" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2013.36.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2013.36.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2013.36.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23336321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">I.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Network-assisted crop systems genetics: network inference and integrative analysis</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">24</div>, <div class="fpage">61</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Network-assisted%20crop%20systems%20genetics%3A%20network%20inference%20and%20integrative%20analysis&amp;author=T.%20Lee&amp;author=H.%20Kim&amp;author=I.%20Lee&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=24&amp;pages=61-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2015.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2015.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2015.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25698380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J.-M.</div></div>, <div class="name string-name"><div class="surname">Noel</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>. <div class="source">Nature</div> , <div class="volume">449</div>, <div class="fpage">463</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.-M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fasoli</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dal Santo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The grapevine expression atlas reveals a deep transcriptome shift driving the entire plant into a maturation program</div>. <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">3489</div>–<div class="lpage">505</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20expression%20atlas%20reveals%20a%20deep%20transcriptome%20shift%20driving%20the%20entire%20plant%20into%20a%20maturation%20program&amp;author=M.%20Fasoli&amp;author=S.%20Dal%20Santo&amp;author=S.%20Zenoni&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=3489-505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.100230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.100230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.100230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22948079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palumbo</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fasoli</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Integrated network analysis identifies fight-club nodes as a class of hubs encompassing key putative switch genes that induce major transcriptome reprogramming during grapevine development</div>. <div class="source">Plant Cell Online</div> , <div class="volume">26</div>, <div class="fpage">4617</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20network%20analysis%20identifies%20fight-club%20nodes%20as%20a%20class%20of%20hubs%20encompassing%20key%20putative%20switch%20genes%20that%20induce%20major%20transcriptome%20reprogramming%20during%20grapevine%20development&amp;author=M.%20C.%20Palumbo&amp;author=S.%20Zenoni&amp;author=M.%20Fasoli&amp;publication_year=2014&amp;journal=Plant%20Cell%20Online&amp;volume=26&amp;pages=4617-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.133710" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.133710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.133710"> </span></div> </div></div></div></div></div><div content-id="dsw061-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dal Santo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tornielli</div> <div class="given-names">G. B.</div></div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The plasticity of the grapevine berry transcriptome</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">r54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plasticity%20of%20the%20grapevine%20berry%20transcriptome&amp;author=S.%20Dal%20Santo&amp;author=G.%20B.%20Tornielli&amp;author=S.%20Zenoni&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=r54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23759170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Malacarne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Coller</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Czemmel</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The grapevine VvibZIPC22 transcription factor is involved in the regulation of flavonoid biosynthesis</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">67</div>, <div class="fpage">3509</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20VvibZIPC22%20transcription%20factor%20is%20involved%20in%20the%20regulation%20of%20flavonoid%20biosynthesis&amp;author=G.%20Malacarne&amp;author=E.%20Coller&amp;author=S.%20Czemmel&amp;publication_year=2016&amp;journal=J.%20Exp.%20Bot&amp;volume=67&amp;pages=3509-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erw181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferw181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferw181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27194742" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loyola</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Herrera</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mas</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The photomorphogenic factors UV-B RECEPTOR 1, ELONGATED HYPOCOTYL 5, and HY5 HOMOLOGUE are part of the UV-B signalling pathway in grapevine and mediate flavonol accumulation in response to the environment</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">67</div>, <div class="fpage">5429</div>–<div class="lpage">5445</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20photomorphogenic%20factors%20UV-B%20RECEPTOR%201%2C%20ELONGATED%20HYPOCOTYL%205%2C%20and%20HY5%20HOMOLOGUE%20are%20part%20of%20the%20UV-B%20signalling%20pathway%20in%20grapevine%20and%20mediate%20flavonol%20accumulation%20in%20response%20to%20the%20environment&amp;author=R.%20Loyola&amp;author=D.%20Herrera&amp;author=A.%20Mas&amp;publication_year=2016&amp;journal=J.%20Exp.%20Bot&amp;volume=67&amp;pages=5429-5445" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erw307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferw307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferw307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27543604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wong</div> <div class="given-names">D. C. J.</div></div>, <div class="name string-name"><div class="surname">Schlechter</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Vannozzi</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A systems-oriented analysis of the grapevine R2R3-MYB transcription factor family uncovers new insights into the regulation of stilbene accumulation</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">451</div>–<div class="lpage">466</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20systems-oriented%20analysis%20of%20the%20grapevine%20R2R3-MYB%20transcription%20factor%20family%20uncovers%20new%20insights%20into%20the%20regulation%20of%20stilbene%20accumulation&amp;author=D.%20C.%20J.%20Wong&amp;author=R.%20Schlechter&amp;author=A.%20Vannozzi&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=451-466" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw028"> </span></div> </div></div></div></div></div><div content-id="dsw061-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Savoi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">D. C. J.</div></div>, <div class="name string-name"><div class="surname">Arapitsas</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Transcriptome and metabolite profiling reveals that prolonged drought modulates the phenylpropanoid and terpenoid pathway in white grapes (<em>Vitis vinifera</em> L.)</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20and%20metabolite%20profiling%20reveals%20that%20prolonged%20drought%20modulates%20the%20phenylpropanoid%20and%20terpenoid%20pathway%20in%20white%20grapes%20(Vitis%20vinifera%20L.)&amp;author=S.%20Savoi&amp;author=D.%20C.%20J.%20Wong&amp;author=P.%20Arapitsas&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0760-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0760-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0760-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27001212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wong</div> <div class="given-names">D. C. J.</div></div>, <div class="name string-name"><div class="surname">Lopez Gutierrez</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Dimopoulos</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Gambetta</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Castellarin</div> <div class="given-names">S. D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Combined physiological, transcriptome, and cis-regulatory element analyses indicate that key aspects of ripening, metabolism, and transcriptional program in grapes (<em>Vitis vinifera</em> L.) are differentially modulated accordingly to fruit size</div>. <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">416</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20physiological%2C%20transcriptome%2C%20and%20cis-regulatory%20element%20analyses%20indicate%20that%20key%20aspects%20of%20ripening%2C%20metabolism%2C%20and%20transcriptional%20program%20in%20grapes%20(Vitis%20vinifera%20L.)%20are%20differentially%20modulated%20accordingly%20to%20fruit%20size&amp;author=D.%20C.%20J.%20Wong&amp;author=R.%20Lopez%20Gutierrez&amp;author=N.%20Dimopoulos&amp;author=G.%20A.%20Gambetta&amp;author=S.%20D.%20Castellarin&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2660-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2660-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2660-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27245662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for illumina sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pyl</div> <div class="given-names">P. T.</div></div>, <div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">HTSeq a python framework to work with high-throughput sequencing data</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">166</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HTSeq%20a%20python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&amp;author=S.%20Anders&amp;author=P.%20T.%20Pyl&amp;author=W.%20Huber&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=31&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu638" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu638" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu638"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25260700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Love</div> <div class="given-names">M. I.</div></div>, <div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</div>. <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">550</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=M.%20I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=550" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0550-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0550-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0550-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ritchie</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Phipson</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Limma powers differential expression analyses for RNA-sequencing and microarray studies</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">e47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;author=M.%20E.%20Ritchie&amp;author=B.%20Phipson&amp;author=D.%20Wu&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=e47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Newman</div> <div class="given-names">M. E. J.</div></div>, <div class="name string-name"><div class="surname">Girvan</div> <div class="given-names">M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Finding and evaluating community structure in networks</div>. <div class="source">Phys. Rev. E – Stat. Nonlinear, Soft Matter Phys</div> ., <div class="volume">69</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.</div></div></div></div><div content-id="dsw061-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thimm</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Bläsing</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Gibon</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">MAPMAN: a user-driven tool to display genomics data sets onto diagrams of metabolic pathways and other biological processes</div>. <div class="source">Plant J</div> ., <div class="volume">37</div>, <div class="fpage">914</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAPMAN%3A%20a%20user-driven%20tool%20to%20display%20genomics%20data%20sets%20onto%20diagrams%20of%20metabolic%20pathways%20and%20other%20biological%20processes&amp;author=O.%20Thimm&amp;author=O.%20Bl%C3%A4sing&amp;author=Y.%20Gibon&amp;publication_year=2004&amp;journal=Plant%20J&amp;volume=37&amp;pages=914-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.37.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.37.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.37.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14996223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shannon</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Markiel</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ozier</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</div>. <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2498</div>–<div class="lpage">504</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoscape%3A%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P.%20Shannon&amp;author=A.%20Markiel&amp;author=O.%20Ozier&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2498-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1239303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1239303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1239303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morey</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Mookherjee</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rajasekaran</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bansal</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">DNA free energy-based promoter prediction and comparative analysis of Arabidopsis and Rice genomes</div>. <div class="source">Plant Physiol</div> ., <div class="volume">156</div>, <div class="fpage">1300</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20free%20energy-based%20promoter%20prediction%20and%20comparative%20analysis%20of%20Arabidopsis%20and%20Rice%20genomes&amp;author=C.%20Morey&amp;author=S.%20Mookherjee&amp;author=G.%20Rajasekaran&amp;author=M.%20Bansal&amp;publication_year=2011&amp;journal=Plant%20Physiol&amp;volume=156&amp;pages=1300-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.167809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.167809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.167809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21531900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ulmasov</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hagen</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Guilfoyle</div> <div class="given-names">T. J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">ARF1, a transcription factor that binds to auxin response elements</div>. <div class="source">Science</div> , <div class="volume">276</div>, <div class="fpage">1865</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARF1%2C%20a%20transcription%20factor%20that%20binds%20to%20auxin%20response%20elements&amp;author=T.%20Ulmasov&amp;author=G.%20Hagen&amp;author=T.%20J.%20Guilfoyle&amp;publication_year=1997&amp;journal=Science&amp;volume=276&amp;pages=1865-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.276.5320.1865" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.276.5320.1865" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.276.5320.1865"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9188533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Licausi</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Ohme-Takagi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Perata</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">APETALA2/Ethylene Responsive Factor (AP2/ERF) transcription factors: mediators of stress responses and developmental programs</div>. <div class="source">New Phytol</div> ., <div class="volume">199</div>, <div class="fpage">639</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=APETALA2%2FEthylene%20Responsive%20Factor%20(AP2%2FERF)%20transcription%20factors%3A%20mediators%20of%20stress%20responses%20and%20developmental%20programs&amp;author=F.%20Licausi&amp;author=M.%20Ohme-Takagi&amp;author=P.%20Perata&amp;publication_year=2013&amp;journal=New%20Phytol&amp;volume=199&amp;pages=639-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24010138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jakoby</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dröge-Laser</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">bZIP transcription factors in Arabidopsis</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">106</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=bZIP%20transcription%20factors%20in%20Arabidopsis&amp;author=M.%20Jakoby&amp;author=B.%20Weisshaar&amp;author=W.%20Dr%C3%B6ge-Laser&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci&amp;volume=7&amp;pages=106-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(01)02223-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(01)02223-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(01)02223-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11906833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leivar</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Monte</div> <div class="given-names">E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">PIFs: systems integrators in plant development</div>. <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">56</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PIFs%3A%20systems%20integrators%20in%20plant%20development&amp;author=P.%20Leivar&amp;author=E.%20Monte&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=56-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.120857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.120857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.120857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24481072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carretero-Paulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Galstyan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Roig-Villanova</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Martínez-García</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Bilbao-Castro</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Robertson</div> <div class="given-names">D. L.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Genome-wide classification and evolutionary analysis of the bHLH family of transcription factors in Arabidopsis, poplar, rice, moss, and algae</div>. <div class="source">Plant Physiol</div> ., <div class="volume">153</div>, <div class="fpage">1398</div>–<div class="lpage">412</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20classification%20and%20evolutionary%20analysis%20of%20the%20bHLH%20family%20of%20transcription%20factors%20in%20Arabidopsis%2C%20poplar%2C%20rice%2C%20moss%2C%20and%20algae&amp;author=L.%20Carretero-Paulet&amp;author=A.%20Galstyan&amp;author=I.%20Roig-Villanova&amp;author=J.%20F.%20Mart%C3%ADnez-Garc%C3%ADa&amp;author=J.%20R.%20Bilbao-Castro&amp;author=D.%20L.%20Robertson&amp;publication_year=2010&amp;journal=Plant%20Physiol&amp;volume=153&amp;pages=1398-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.153593" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.153593" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.153593"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20472752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Poovaiah</div> <div class="given-names">B. W.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">A calmodulin-binding/CGCG box DNA-binding protein family involved in multiple signaling pathways in plants</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">277</div>, <div class="fpage">45049</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20calmodulin-binding%2FCGCG%20box%20DNA-binding%20protein%20family%20involved%20in%20multiple%20signaling%20pathways%20in%20plants&amp;author=T.%20Yang&amp;author=B.%20W.%20Poovaiah&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem&amp;volume=277&amp;pages=45049-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M207941200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M207941200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M207941200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12218065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagel</div> <div class="given-names">D. H.</div></div>, <div class="name string-name"><div class="surname">Doherty</div> <div class="given-names">C. J.</div></div>, <div class="name string-name"><div class="surname">Pruneda-Paz</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Ecker</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Kay</div> <div class="given-names">S. A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide identification of CCA1 targets uncovers an expanded clock network in Arabidopsis</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">112</div>, <div class="fpage">E4802</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20CCA1%20targets%20uncovers%20an%20expanded%20clock%20network%20in%20Arabidopsis&amp;author=D.%20H.%20Nagel&amp;author=C.%20J.%20Doherty&amp;author=J.%20L.%20Pruneda-Paz&amp;author=R.%20J.%20Schmitz&amp;author=J.%20R.%20Ecker&amp;author=S.%20A.%20Kay&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=112&amp;pages=E4802-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1513609112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1513609112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1513609112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26261339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Klie</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nikoloski</div> <div class="given-names">Z.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The choice between MapMan and Gene ontology for automated gene function prediction in plant science</div>. <div class="source">Front. Genet</div> ., <div class="volume">3</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20choice%20between%20MapMan%20and%20Gene%20ontology%20for%20automated%20gene%20function%20prediction%20in%20plant%20science&amp;author=S.%20Klie&amp;author=Z.%20Nikoloski&amp;publication_year=2012&amp;journal=Front.%20Genet&amp;volume=3&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2012.00115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2012.00115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2012.00115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22303408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ziliotto</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Corso</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rizzini</div> <div class="given-names">F. M.</div></div>, <div class="name string-name"><div class="surname">Rasori</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Botton</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bonghi</div> <div class="given-names">C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Grape berry ripening delay induced by a pre-véraison NAA treatment is paralleled by a shift in the expression pattern of auxin- and ethylene-related genes</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">12</div>, <div class="fpage">185</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grape%20berry%20ripening%20delay%20induced%20by%20a%20pre-v%C3%A9raison%20NAA%20treatment%20is%20paralleled%20by%20a%20shift%20in%20the%20expression%20pattern%20of%20auxin-%20and%20ethylene-related%20genes&amp;author=F.%20Ziliotto&amp;author=M.%20Corso&amp;author=F.%20M.%20Rizzini&amp;author=A.%20Rasori&amp;author=A.%20Botton&amp;author=C.%20Bonghi&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol&amp;volume=12&amp;pages=185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23046684" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jiao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lau</div> <div class="given-names">O. S.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X. W.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Light-regulated transcriptional networks in higher plants</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">8</div>, <div class="fpage">217</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Light-regulated%20transcriptional%20networks%20in%20higher%20plants&amp;author=Y.%20Jiao&amp;author=O.%20S.%20Lau&amp;author=X.%20W.%20Deng&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet&amp;volume=8&amp;pages=217-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17304247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heijde</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ulm</div> <div class="given-names">R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">UV-B photoreceptor-mediated signalling in plants</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">17</div>, <div class="fpage">230</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UV-B%20photoreceptor-mediated%20signalling%20in%20plants&amp;author=M.%20Heijde&amp;author=R.%20Ulm&amp;publication_year=2012&amp;journal=Trends%20Plant%20Sci&amp;volume=17&amp;pages=230-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2012.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2012.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2012.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22326562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nicolas</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Lecourieux</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kappel</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The basic leucine zipper transcription factor ABSCISIC ACID RESPONSE ELEMENT-BINDING FACTOR2 is an important transcriptional regulator of abscisic acid-dependent grape berry ripening processes</div>. <div class="source">Plant Physiol</div> ., <div class="volume">164</div>, <div class="fpage">365</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20basic%20leucine%20zipper%20transcription%20factor%20ABSCISIC%20ACID%20RESPONSE%20ELEMENT-BINDING%20FACTOR2%20is%20an%20important%20transcriptional%20regulator%20of%20abscisic%20acid-dependent%20grape%20berry%20ripening%20processes&amp;author=P.%20Nicolas&amp;author=D.%20Lecourieux&amp;author=C.%20Kappel&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=164&amp;pages=365-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.231977" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.231977" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.231977"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24276949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hichri</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Heppel</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Pillet</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The basic helix-loop-helix transcription factor MYC1 is involved in the regulation of the flavonoid biosynthesis pathway in grapevine</div>. <div class="source">Mol. Plant</div> , <div class="volume">3</div>, <div class="fpage">509</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20basic%20helix-loop-helix%20transcription%20factor%20MYC1%20is%20involved%20in%20the%20regulation%20of%20the%20flavonoid%20biosynthesis%20pathway%20in%20grapevine&amp;author=I.%20Hichri&amp;author=S.%20C.%20Heppel&amp;author=J.%20Pillet&amp;publication_year=2010&amp;journal=Mol.%20Plant&amp;volume=3&amp;pages=509-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/ssp118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssp118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fssp118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20118183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Somssich</div> <div class="given-names">I. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Transcriptional networks in plant immunity</div>. <div class="source">New Phytol</div> ., <div class="volume">206</div>, <div class="fpage">932</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20networks%20in%20plant%20immunity&amp;author=K.%20Tsuda&amp;author=I.%20E.%20Somssich&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=206&amp;pages=932-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2015.206.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2015.206.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2015.206.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25623163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greenham</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">McClung</div> <div class="given-names">C. R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Integrating circadian dynamics with physiological processes in plants</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">16</div>, <div class="fpage">598</div>–<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrating%20circadian%20dynamics%20with%20physiological%20processes%20in%20plants&amp;author=K.%20Greenham&amp;author=C.%20R.%20McClung&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=598-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26370901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vranová</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Coman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gruissem</div> <div class="given-names">W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Network analysis of the MVA and MEP pathways for isoprenoid synthesis</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">64</div>, <div class="fpage">665</div>–<div class="lpage">700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Network%20analysis%20of%20the%20MVA%20and%20MEP%20pathways%20for%20isoprenoid%20synthesis&amp;author=E.%20Vranov%C3%A1&amp;author=D.%20Coman&amp;author=W.%20Gruissem&amp;publication_year=2013&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=64&amp;pages=665-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050312-120116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050312-120116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050312-120116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23451776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vannozzi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dry</div> <div class="given-names">I. B.</div></div>, <div class="name string-name"><div class="surname">Fasoli</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lucchin</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genome-wide analysis of the grapevine stilbene synthase multigenic family: genomic organization and expression profiles upon biotic and abiotic stresses</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">12</div>, <div class="fpage">130</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20the%20grapevine%20stilbene%20synthase%20multigenic%20family%3A%20genomic%20organization%20and%20expression%20profiles%20upon%20biotic%20and%20abiotic%20stresses&amp;author=A.%20Vannozzi&amp;author=I.%20B.%20Dry&amp;author=M.%20Fasoli&amp;author=S.%20Zenoni&amp;author=M.%20Lucchin&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol&amp;volume=12&amp;pages=130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22863370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Aubourg</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schouwey</div> <div class="given-names">M. B.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Functional annotation, genome organization and phylogeny of the grapevine (<em>Vitis vinifera</em>) terpene synthase gene family based on genome assembly, FLcDNA cloning, and enzyme assays</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">10</div>, <div class="fpage">226</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%2C%20genome%20organization%20and%20phylogeny%20of%20the%20grapevine%20(Vitis%20vinifera)%20terpene%20synthase%20gene%20family%20based%20on%20genome%20assembly%2C%20FLcDNA%20cloning%2C%20and%20enzyme%20assays&amp;author=D.%20M.%20Martin&amp;author=S.%20Aubourg&amp;author=M.%20B.%20Schouwey&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol&amp;volume=10&amp;pages=226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20964856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carbonell-Bejerano</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rodríguez</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Royo</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Circadian oscillatory transcriptional programs in grapevine ripening fruits</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="fpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circadian%20oscillatory%20transcriptional%20programs%20in%20grapevine%20ripening%20fruits&amp;author=P.%20Carbonell-Bejerano&amp;author=V.%20Rodr%C3%ADguez&amp;author=C.%20Royo&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-78" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-78" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-78"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24666982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wong</div> <div class="given-names">D. C. J.</div></div>, <div class="name string-name"><div class="surname">Sweetman</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Drew</div> <div class="given-names">D. P.</div></div>, <div class="name string-name"><div class="surname">Ford</div> <div class="given-names">C. M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">VTCdb: a gene co-expression database for the crop species <em>Vitis vinifera</em> (grapevine)</div>. <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">882</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VTCdb%3A%20a%20gene%20co-expression%20database%20for%20the%20crop%20species%20Vitis%20vinifera%20(grapevine)&amp;author=D.%20C.%20J.%20Wong&amp;author=C.%20Sweetman&amp;author=D.%20P.%20Drew&amp;author=C.%20M.%20Ford&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=882" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-882" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-882" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-882"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24341535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sayama</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">AREB1, AREB2, and ABF3 are master transcription factors that cooperatively regulate ABRE-dependent ABA signaling involved in drought stress tolerance and require ABA for full activation</div>. <div class="source">Plant J</div> ., <div class="volume">61</div>, <div class="fpage">672</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AREB1%2C%20AREB2%2C%20and%20ABF3%20are%20master%20transcription%20factors%20that%20cooperatively%20regulate%20ABRE-dependent%20ABA%20signaling%20involved%20in%20drought%20stress%20tolerance%20and%20require%20ABA%20for%20full%20activation&amp;author=T.%20Yoshida&amp;author=Y.%20Fujita&amp;author=H.%20Sayama&amp;publication_year=2010&amp;journal=Plant%20J&amp;volume=61&amp;pages=672-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.61.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.61.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.61.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19947981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cutler</div> <div class="given-names">S. R.</div></div>, <div class="name string-name"><div class="surname">Rodriguez</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Finkelstein</div> <div class="given-names">R. R.</div></div>, <div class="name string-name"><div class="surname">Abrams</div> <div class="given-names">S. R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Abscisic acid: emergence of a core signaling network</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">61</div>, <div class="fpage">651</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abscisic%20acid%3A%20emergence%20of%20a%20core%20signaling%20network&amp;author=S.%20R.%20Cutler&amp;author=P.%20L.%20Rodriguez&amp;author=R.%20R.%20Finkelstein&amp;author=S.%20R.%20Abrams&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=61&amp;pages=651-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-042809-112122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-042809-112122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-042809-112122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20192755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Olsson</div> <div class="given-names">A. S. B.</div></div>, <div class="name string-name"><div class="surname">Engström</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Söderman</div> <div class="given-names">E.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The homeobox genes ATHB12 and ATHB7 encode potential regulators of growth in response to water deficit in Arabidopsis</div>. <div class="source">Plant Mol. Biol</div> ., <div class="volume">55</div>, <div class="fpage">663</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20homeobox%20genes%20ATHB12%20and%20ATHB7%20encode%20potential%20regulators%20of%20growth%20in%20response%20to%20water%20deficit%20in%20Arabidopsis&amp;author=A.%20S.%20B.%20Olsson&amp;author=P.%20Engstr%C3%B6m&amp;author=E.%20S%C3%B6derman&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol&amp;volume=55&amp;pages=663-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-004-1581-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-004-1581-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-004-1581-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15604708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rossdeutsch</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Cookson</div> <div class="given-names">S. J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">ABA-mediated responses to water deficit separate grapevine genotypes by their genetic background</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABA-mediated%20responses%20to%20water%20deficit%20separate%20grapevine%20genotypes%20by%20their%20genetic%20background&amp;author=L.%20Rossdeutsch&amp;author=E.%20Edwards&amp;author=S.%20J.%20Cookson&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0778-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0778-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0778-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27091220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rattanakon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ghan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Gambetta</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Deluc</div> <div class="given-names">L. G.</div></div>, <div class="name string-name"><div class="surname">Schlauch</div> <div class="given-names">K. A.</div></div>, <div class="name string-name"><div class="surname">Cramer</div> <div class="given-names">G. R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Abscisic acid transcriptomic signaling varies with grapevine organ</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abscisic%20acid%20transcriptomic%20signaling%20varies%20with%20grapevine%20organ&amp;author=S.%20Rattanakon&amp;author=R.%20Ghan&amp;author=G.%20A.%20Gambetta&amp;author=L.%20G.%20Deluc&amp;author=K.%20A.%20Schlauch&amp;author=G.%20R.%20Cramer&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0763-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0763-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0763-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27001301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matus</div> <div class="given-names">J. T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Transcriptomic and metabolomic setworks in the grape berry illustrate that it takes more than flavonoids to fight against ultraviolet radiation</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">1337</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20and%20metabolomic%20setworks%20in%20the%20grape%20berry%20illustrate%20that%20it%20takes%20more%20than%20flavonoids%20to%20fight%20against%20ultraviolet%20radiation&amp;author=J.%20T.%20Matus&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=7&amp;pages=1337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2016.01337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2016.01337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2016.01337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27625679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ono</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Homma</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Horikawa</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Functional differentiation of the glycosyltransferases that contribute to the chemical diversity of bioactive flavonol glycosides in grapevines (<em>Vitis vinifera</em>)</div>. <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">2856</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20differentiation%20of%20the%20glycosyltransferases%20that%20contribute%20to%20the%20chemical%20diversity%20of%20bioactive%20flavonol%20glycosides%20in%20grapevines%20(Vitis%20vinifera)&amp;author=E.%20Ono&amp;author=Y.%20Homma&amp;author=M.%20Horikawa&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=2856-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.074625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.074625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.074625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20693356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">N. H.</div></div>, <div class="name string-name"><div class="surname">Jeong</div> <div class="given-names">C. Y.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">G. H.</div></div>, <div class="name string-name"><div class="surname">Yoo</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">MYBD employed by HY5 increases anthocyanin accumulation via repression of MYBL2 in Arabidopsis</div>. <div class="source">Plant J</div> ., <div class="volume">84</div>, <div class="fpage">1192</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYBD%20employed%20by%20HY5%20increases%20anthocyanin%20accumulation%20via%20repression%20of%20MYBL2%20in%20Arabidopsis&amp;author=N.%20H.%20Nguyen&amp;author=C.%20Y.%20Jeong&amp;author=G.%20H.%20Kang&amp;author=S.%20D.%20Yoo&amp;author=S.%20W.%20Hong&amp;author=H.%20Lee&amp;publication_year=2015&amp;journal=Plant%20J&amp;volume=84&amp;pages=1192-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.13077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.13077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26576746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cramer</div> <div class="given-names">G. R.</div></div>, <div class="name string-name"><div class="surname">Ghan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Schlauch</div> <div class="given-names">K. A.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptomic analysis of the late stages of grapevine (<em>Vitis vinifera</em> cv. Cabernet Sauvignon) berry ripening reveals significant induction of ethylene signaling and flavor pathways in the skin</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="issue">1</div>–<div class="fpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20of%20the%20late%20stages%20of%20grapevine%20(Vitis%20vinifera%20cv.%20Cabernet%20Sauvignon)%20berry%20ripening%20reveals%20significant%20induction%20of%20ethylene%20signaling%20and%20flavor%20pathways%20in%20the%20skin&amp;author=G.%20R.%20Cramer&amp;author=R.%20Ghan&amp;author=K.%20A.%20Schlauch&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0370-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0370-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0370-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24410747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Armijo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schlechter</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Agurto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Muñoz</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Nuñez</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Arce-Johnson</div> <div class="given-names">P.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Grapevine pathogenic microorganisms: understanding infection strategies and host response scenarios</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">382</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grapevine%20pathogenic%20microorganisms%3A%20understanding%20infection%20strategies%20and%20host%20response%20scenarios&amp;author=G.%20Armijo&amp;author=R.%20Schlechter&amp;author=M.%20Agurto&amp;author=D.%20Mu%C3%B1oz&amp;author=C.%20Nu%C3%B1ez&amp;author=P.%20Arce-Johnson&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=7&amp;pages=382" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2016.00382" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2016.00382" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2016.00382"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27066032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mueller</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hilbert</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dueckershoff</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">General detoxification and stress responses are mediated by oxidized lipids through TGA transcription factors in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">20</div>, <div class="fpage">768</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20detoxification%20and%20stress%20responses%20are%20mediated%20by%20oxidized%20lipids%20through%20TGA%20transcription%20factors%20in%20Arabidopsis&amp;author=S.%20Mueller&amp;author=B.%20Hilbert&amp;author=K.%20Dueckershoff&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=768-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.107.054809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.107.054809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.107.054809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18334669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Veronese</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Nakagami</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bluhm</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">The membrane-anchored BOTRYTIS-INDUCED KINASE1 plays distinct roles in Arabidopsis resistance to necrotrophic and biotrophic pathogens</div>. <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">257</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20membrane-anchored%20BOTRYTIS-INDUCED%20KINASE1%20plays%20distinct%20roles%20in%20Arabidopsis%20resistance%20to%20necrotrophic%20and%20biotrophic%20pathogens&amp;author=P.%20Veronese&amp;author=H.%20Nakagami&amp;author=B.%20Bluhm&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=257-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.035576" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.035576" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.035576"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Höll</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Vannozzi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Czemmel</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The R2R3-MYB transcription factors MYB14 and MYB15 regulate stilbene biosynthesis in <em>Vitis vinifera</em></div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">4135</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20R2R3-MYB%20transcription%20factors%20MYB14%20and%20MYB15%20regulate%20stilbene%20biosynthesis%20in%20Vitis%20vinifera&amp;author=J.%20H%C3%B6ll&amp;author=A.%20Vannozzi&amp;author=S.%20Czemmel&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=4135-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.117127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.117127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.117127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24151295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Birkenbihl</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Diezel</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Somssich</div> <div class="given-names">I. E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Arabidopsis WRKY33 is a key transcriptional regulator of hormonal and metabolic responses toward <em>Botrytis cinerea</em> infection</div>. <div class="source">Plant Physiol</div> ., <div class="volume">159</div>, <div class="fpage">266</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20WRKY33%20is%20a%20key%20transcriptional%20regulator%20of%20hormonal%20and%20metabolic%20responses%20toward%20Botrytis%20cinerea%20infection&amp;author=R.%20P.%20Birkenbihl&amp;author=C.%20Diezel&amp;author=I.%20E.%20Somssich&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=159&amp;pages=266-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.192641" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.192641" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.192641"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22392279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Physical and functional interactions between pathogen-induced Arabidopsis WRKY18, WRKY40, and WRKY60 transcription factors</div>. <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">1310</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20and%20functional%20interactions%20between%20pathogen-induced%20Arabidopsis%20WRKY18%2C%20WRKY40%2C%20and%20WRKY60%20transcription%20factors&amp;author=X.%20Xu&amp;author=C.%20Chen&amp;author=B.%20Fan&amp;author=Z.%20Chen&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1310-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.037523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.037523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.037523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16603654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Lai</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Dual function of Arabidopsis ATAF1 in abiotic and biotic stress responses</div>. <div class="source">Cell Res</div> ., <div class="volume">19</div>, <div class="fpage">1279</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20function%20of%20Arabidopsis%20ATAF1%20in%20abiotic%20and%20biotic%20stress%20responses&amp;author=Y.%20Wu&amp;author=Z.%20Deng&amp;author=J.%20Lai&amp;publication_year=2009&amp;journal=Cell%20Res&amp;volume=19&amp;pages=1279-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cr.2009.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcr.2009.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcr.2009.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19752887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pieterse</div> <div class="given-names">C. M. J.</div></div>, <div class="name string-name"><div class="surname">Van der Does</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zamioudis</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Leon-Reyes</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Van Wees</div> <div class="given-names">S. C. M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Hormonal modulation of plant immunity</div>. <div class="source">Annu. Rev. Cell Dev. Biol</div> ., <div class="volume">28</div>, <div class="fpage">489</div>–<div class="lpage">521</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hormonal%20modulation%20of%20plant%20immunity&amp;author=C.%20M.%20J.%20Pieterse&amp;author=D.%20Van%20der%20Does&amp;author=C.%20Zamioudis&amp;author=A.%20Leon-Reyes&amp;author=S.%20C.%20M.%20Van%20Wees&amp;publication_year=2012&amp;journal=Annu.%20Rev.%20Cell%20Dev.%20Biol&amp;volume=28&amp;pages=489-521" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-cellbio-092910-154055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-cellbio-092910-154055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-cellbio-092910-154055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22559264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feechan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jermakow</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Ivancevic</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Host cell entry of powdery mildew is correlated with endosomal transport of antagonistically acting VvPEN1 and VvMLO to the papilla</div>. <div class="source">Mol. Plant. Microbe. Interact</div> ., <div class="volume">26</div>, <div class="fpage">1138</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host%20cell%20entry%20of%20powdery%20mildew%20is%20correlated%20with%20endosomal%20transport%20of%20antagonistically%20acting%20VvPEN1%20and%20VvMLO%20to%20the%20papilla&amp;author=A.%20Feechan&amp;author=A.%20M.%20Jermakow&amp;author=A.%20Ivancevic&amp;publication_year=2013&amp;journal=Mol.%20Plant.%20Microbe.%20Interact&amp;volume=26&amp;pages=1138-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-04-13-0091-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-04-13-0091-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-04-13-0091-R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23819806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Barley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Curtis</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2000</div>, 
<div class="article-title">Asymmetric leaves1 mediates leaf patterning and stem cell function in Arabidopsis</div>. <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">967</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20leaves1%20mediates%20leaf%20patterning%20and%20stem%20cell%20function%20in%20Arabidopsis&amp;author=M.%20E.%20Byrne&amp;author=R.%20Barley&amp;author=M.%20Curtis&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=967-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35050091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35050091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35050091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11140682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xing</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Salinas</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Garcia-Molina</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Höhmann</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Berndtgen</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Huijser</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">SPL8 and miR156-targeted SPL genes redundantly regulate Arabidopsis gynoecium differential patterning</div>. <div class="source">Plant J</div> ., <div class="volume">75</div>, <div class="fpage">566</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPL8%20and%20miR156-targeted%20SPL%20genes%20redundantly%20regulate%20Arabidopsis%20gynoecium%20differential%20patterning&amp;author=S.%20Xing&amp;author=M.%20Salinas&amp;author=A.%20Garcia-Molina&amp;author=S.%20H%C3%B6hmann&amp;author=R.%20Berndtgen&amp;author=P.%20Huijser&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=75&amp;pages=566-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.75.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.75.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.75.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23621152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McCarthy</div> <div class="given-names">R. L.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z. H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">MYB83 is a direct target of SND1 and acts redundantly with MYB46 in the regulation of secondary cell wall biosynthesis in arabidopsis</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">50</div>, <div class="fpage">1950</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB83%20is%20a%20direct%20target%20of%20SND1%20and%20acts%20redundantly%20with%20MYB46%20in%20the%20regulation%20of%20secondary%20cell%20wall%20biosynthesis%20in%20arabidopsis&amp;author=R.%20L.%20McCarthy&amp;author=R.%20Zhong&amp;author=Z.%20H.%20Ye&amp;publication_year=2009&amp;journal=Plant%20Cell%20Physiol&amp;volume=50&amp;pages=1950-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcp139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcp139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcp139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19808805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z. H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Secondary cell walls: biosynthesis, patterned deposition and transcriptional regulation</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">56</div>, <div class="fpage">195</div>–<div class="lpage">214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Secondary%20cell%20walls%3A%20biosynthesis%2C%20patterned%20deposition%20and%20transcriptional%20regulation&amp;author=R.%20Zhong&amp;author=Z.%20H.%20Ye&amp;publication_year=2015&amp;journal=Plant%20Cell%20Physiol&amp;volume=56&amp;pages=195-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcu140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcu140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcu140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25294860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z.-H.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">MYB46 and MYB83 bind to the SMRE sites and directly activate a suite of transcription factors and secondary wall biosynthetic genes</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">53</div>, <div class="fpage">368</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB46%20and%20MYB83%20bind%20to%20the%20SMRE%20sites%20and%20directly%20activate%20a%20suite%20of%20transcription%20factors%20and%20secondary%20wall%20biosynthetic%20genes&amp;author=R.%20Zhong&amp;author=Z.-H.%20Ye&amp;publication_year=2012&amp;journal=Plant%20Cell%20Physiol&amp;volume=53&amp;pages=368-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22197883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lindemose</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">De Velde</div> <div class="given-names">J.</div></div></div> Van, et al.   <div class="year">2014</div>, 
<div class="article-title">A DNA-binding-site landscape and regulatory network analysis for NAC transcription factors in Arabidopsis thaliana</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">7681</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20DNA-binding-site%20landscape%20and%20regulatory%20network%20analysis%20for%20NAC%20transcription%20factors%20in%20Arabidopsis%20thaliana&amp;author=S.%20Lindemose&amp;author=M.%20K.%20Jensen&amp;author=J.%20De%20Velde&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=7681-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24914054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jarvis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">López-Juez</div> <div class="given-names">E.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Biogenesis and homeostasis of chloroplasts and other plastids</div>. <div class="source">Nat. Rev. Mol. Cell Biol</div> ., <div class="volume">14</div>, <div class="fpage">787</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biogenesis%20and%20homeostasis%20of%20chloroplasts%20and%20other%20plastids&amp;author=P.%20Jarvis&amp;author=E.%20L%C3%B3pez-Juez&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;volume=14&amp;pages=787-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm3702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm3702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm3702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24263360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">C. V</div></div>, <div class="name string-name"><div class="surname">Vrebalov</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Gapper</div> <div class="given-names">N. E.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Tomato GOLDEN2-LIKE transcription factors reveal molecular gradients that function during fruit development and ripening</div>. <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">585</div>–<div class="lpage">601</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20GOLDEN2-LIKE%20transcription%20factors%20reveal%20molecular%20gradients%20that%20function%20during%20fruit%20development%20and%20ripening&amp;author=C.%20V%20Nguyen&amp;author=J.%20T.%20Vrebalov&amp;author=N.%20E.%20Gapper&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=585-601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.118794" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.118794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.118794"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24510723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Powell</div> <div class="given-names">A. L. T.</div></div>, <div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">C. V.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Uniform ripening encodes a Golden 2-like transcription factor regulating tomato fruit chloroplast development</div>. <div class="source">Science</div> , <div class="volume">336</div>, <div class="fpage">1711</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uniform%20ripening%20encodes%20a%20Golden%202-like%20transcription%20factor%20regulating%20tomato%20fruit%20chloroplast%20development&amp;author=A.%20L.%20T.%20Powell&amp;author=C.%20V.%20Nguyen&amp;author=T.%20Hill&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=1711-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1222218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1222218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1222218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yasui</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ueno</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">215</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20of%20the%20draft%20genome%20of%20buckwheat%20and%20its%20applications%20in%20identifying%20agronomically%20useful%20genes&amp;author=Y.%20Yasui&amp;author=H.%20Hirakawa&amp;author=M.%20Ueno&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=215-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27037832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Okada</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tabuchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Survey of genome sequences in a wild sweet potato, Ipomoea trifida (H. B. K.) G. Don</div>. <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">171</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20of%20genome%20sequences%20in%20a%20wild%20sweet%20potato%2C%20Ipomoea%20trifida%20(H.%20B.%20K.)%20G.%20Don&amp;author=H.%20Hirakawa&amp;author=Y.%20Okada&amp;author=H.%20Tabuchi&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=171-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25805887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw061-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw061-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shigenobu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Motoi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Improvement of barley genome annotations by deciphering the Haruna Nijo genome</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">21</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20barley%20genome%20annotations%20by%20deciphering%20the%20Haruna%20Nijo%20genome&amp;author=K.%20Sato&amp;author=T.%20Tanaka&amp;author=S.%20Shigenobu&amp;author=Y.%20Motoi&amp;author=J.%20Wu&amp;author=T.%20Itoh&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=23&amp;pages=21-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26622062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2949522&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw061/1/dsw061_Supp.zip?Expires=2147483647&amp;Signature=3pCS4r7DmZIGWAK1ig0Gz6AWOSDkUcrpR6jyeXvh97pkbctsk9IWkIzMklGHJ8~XJV0IvWLJzm0LOZgbZmK4GEIn4fLSR4ft7HC121Qxev4fT~w2rtseNp6EIMdbjhLpZcuTgBXuqcqbZcn4T0mOp~ywMnSO8jJF3T2fUG33xbj9UCWFiub4WhUF0nXLk18nJ9qShkfgJ21Hs~Vt8-PSeT16TcRQR8F8ukE5X69yZLodTKW80-PZnIdZ6ak4TRWsvrA6YDeqreBbaKL7cgGrNxBiFjdWoa~FiFiPyYe2vBe6wsrRQC846wfoD~iM8TsnP0pw2BrR8xkFVwuOsKrN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,294</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw061"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2949522"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f311%2f2949522">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F311%2F2949522" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2949522'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide analysis of cis-regulatory element structure and discovery of motif-driven gene co-expression networks in grapevine" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw061");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f311%2f2949522">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

