

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2059,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/1/4107259","@type":"ScholarlyArticle","name":"A re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in  Cucumis melo ","datePublished":"2017-09-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx033","keywords":["ultra-dense genetic map","re-sequencing","gummy stem blight","<italic>Cucumis melo</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/1/4107259","author":[{"name":"Hu, Zhongyuan","affiliation":"Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China","@type":"Person"},{"name":"Deng, Guancong","affiliation":"Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China","@type":"Person"},{"name":"Mou, Haipeng","affiliation":"Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China","@type":"Person"},{"name":"Xu, Yuhui","affiliation":"Biomarker Technologies Corporation, Beijing 101300, China","@type":"Person"},{"name":"Chen, Li","affiliation":"Biomarker Technologies Corporation, Beijing 101300, China","@type":"Person"},{"name":"Yang, Jinghua","affiliation":"Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China  Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China  Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China","@type":"Person"},{"name":"Zhang, Mingfang","affiliation":"Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China  Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China  Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China","@type":"Person"}],"description":"Abstract.  The melon (Cucumis melo) genome and genetic maps with hundreds to thousands of single nucleotide polymorphism markers were recently released. However","pageStart":"1","pageEnd":"10","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in  Cucumis melo ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in  Cucumis melo " />
    <meta property="og:description" content="Abstract.  The melon (Cucumis melo) genome and genetic maps with hundreds to thousands of single nucleotide polymorphism markers were recently released. However" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/1/4107259" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hu, Zhongyuan" /><meta name="citation_author_institution" content="Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author" content="Deng, Guancong" /><meta name="citation_author_institution" content="Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author" content="Mou, Haipeng" /><meta name="citation_author_institution" content="Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author" content="Xu, Yuhui" /><meta name="citation_author_institution" content="Biomarker Technologies Corporation, Beijing 101300, China" /><meta name="citation_author" content="Chen, Li" /><meta name="citation_author_institution" content="Biomarker Technologies Corporation, Beijing 101300, China" /><meta name="citation_author" content="Yang, Jinghua" /><meta name="citation_author_institution" content="Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China" /><meta name="citation_author_institution" content="Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China" /><meta name="citation_author" content="Zhang, Mingfang" /><meta name="citation_author_institution" content="Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China" /><meta name="citation_author_institution" content="Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China" /><meta name="citation_title" content="A re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in Cucumis melo" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="10" /><meta name="citation_doi" content="10.1093/dnares/dsx033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=From native plants in Central Europe to cultivated crops worldwide: the emergence of Didymella bryoniae as a Cucurbit Pathogen;  citation_journal_title=Hortscience;  citation_year=2011;  citation_volume=46;  citation_pages=532-5; " /><meta name="citation_reference" content="citation_title=Breeding for disease resistance in Cucurbits;  citation_journal_title=Annu. Rev. Phytopathol;  citation_year=1972;  citation_volume=10;  citation_pages=471-90; " /><meta name="citation_reference" content="citation_title=Five independent loci each control monogenic resistance to gummy stem blight in melon (Cucumis melo L.);  citation_journal_title=Theor. Appl. Genet;  citation_year=2004;  citation_volume=108;  citation_pages=1033-8; " /><meta name="citation_reference" content="citation_title=Identification of amplified fragment length polymorphism markers linked to gummy stem blight (Didymella bryoniae) resistance in Melon (Cucumis melo L.) PI 420145;  citation_journal_title=Hortscience;  citation_year=2009;  citation_volume=44;  citation_pages=32-4; " /><meta name="citation_reference" content="citation_title=Pryamiding disease resistance genes and variety improvment by molecular marker-assisted selection in melon (Cucumis melon L.);  citation_journal_title=Sci. Agric. Sin;  citation_year=2015;  citation_volume=48;  citation_pages=523-33; " /><meta name="citation_reference" content="citation_title=A consensus linkage map for molecular markers and quantitative trait loci associated with economically important traits in melon (Cucumis melo L.);  citation_journal_title=BMC Plant Biol;  citation_year=2011;  citation_volume=11;  citation_pages=111" /><meta name="citation_reference" content="citation_title=A genetic map of melon highly enriched with fruit quality QTLs and EST markers, including sugar and carotenoid metabolism genes;  citation_journal_title=Theor. Appl. Genet;  citation_year=2010;  citation_volume=121;  citation_pages=511-33; " /><meta name="citation_reference" content="citation_title=Genetic linkage map of melon (Cucumis melo L.) and localization of a major QTL for powdery mildew resistance;  citation_journal_title=Mol. Breeding;  citation_year=2011;  citation_volume=27;  citation_pages=181-92; " /><meta name="citation_reference" content="citation_title=Genome-wide single nucleotide polymorphism discovery and the construction of a high-density genetic map for melon (Cucumis melo L.) using genotyping-by-sequencing;  citation_journal_title=Front. Plant Sci;  citation_year=2017;  citation_volume=8;  citation_pages=125" /><meta name="citation_reference" content="citation_title=The genome of melon (Cucumis melo L.);  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2012;  citation_volume=109;  citation_pages=11872-7; " /><meta name="citation_reference" content="citation_title=Anchoring the consensus ICuGI genetic map to the melon (Cucumis melo L.) genome;  citation_journal_title=Mol. Breeding;  citation_year=2015;  citation_volume=35;  citation_pages=188" /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping by whole-genome resequencing;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1068-76; " /><meta name="citation_reference" content="citation_title=Pinpointing genes underlying the quantitative trait loci for root-knot nematode resistance in palaeopolyploid soybean by whole genome resequencing;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2013;  citation_volume=110;  citation_pages=13469-74; " /><meta name="citation_reference" content="citation_title=Sequence-based ultra-dense genetic and physical maps reveal structural variations of allopolyploid cotton genomes;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=108" /><meta name="citation_reference" content="citation_title=Construction of a reference genetic map of Raphanus sativus based on genotyping by whole-genome resequencing;  citation_journal_title=Theor. Appl. Genet;  citation_year=2015;  citation_volume=128;  citation_pages=259-72; " /><meta name="citation_reference" content="citation_title=Getting started in mapping-by-sequencing;  citation_journal_title=J. Integr. Plant Biol;  citation_year=2015;  citation_volume=57;  citation_pages=606-12; " /><meta name="citation_reference" content="citation_title=Using next-generation sequencing to isolate mutant genes from forward genetic screens;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=662-76; " /><meta name="citation_reference" content="citation_title=Host-microbe interactions: shaping the evolution of the plant immune response;  citation_journal_title=Cell;  citation_year=2006;  citation_volume=124;  citation_pages=803-14; " /><meta name="citation_reference" content="citation_title=Plant immunity: towards an integrated view of plant-pathogen interactions;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=539-48; " /><meta name="citation_reference" content="citation_title=The plant immune system;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=323-9; " /><meta name="citation_reference" content="citation_title=Comparing signaling mechanisms engaged in pattern-triggered and effector-triggered immunity;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2010;  citation_volume=13;  citation_pages=459-65; " /><meta name="citation_reference" content="citation_title=Standardization of a mass production technique for pycnidiospores of Didymella bryoniae, gummy stem blight fungus of Cucurbits;  citation_journal_title=Korean J. Plant Pathol;  citation_year=1997;  citation_volume=13;  citation_pages=105-12; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=Construction and analysis of high-density linkage map using high-throughput sequencing data;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e98855" /><meta name="citation_reference" content="citation_title=ALLMAPS: robust scaffold ordering based on multiple maps;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=3" /><meta name="citation_reference" content="citation_title=High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (Prunus mume Sieb. et Zucc);  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=183-91; " /><meta name="citation_reference" content="citation_publisher=Springer, New York;  citation_title=A guide to QTL mapping with Q/qtl;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Use of targeted SNP selection for an improved anchoring of the melon (Cucumis melo L.) scaffold genome assembly;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing for plant breeding and genetics;  citation_journal_title=Plant Genome-Us;  citation_year=2012;  citation_volume=5;  citation_pages=92-102; " /><meta name="citation_reference" content="citation_title=High-density linkage map construction and mapping of seed trait QTLs in chickpea (Cicer arietinum L.) using genotyping-by-sequencing (GBS);  citation_journal_title=Sci. Rep.-Uk;  citation_year=2015;  citation_volume=5;  citation_pages=17512" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (Brassica oleracea L.);  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=29-41; " /><meta name="citation_reference" content="citation_title=Crop genome sequencing: lessons and rationales;  citation_journal_title=Trends Plant Sci;  citation_year=2011;  citation_volume=16;  citation_pages=77-88; " /><meta name="citation_reference" content="citation_title=The draft genome of watermelon (Citrullus lanatus) and resequencing of 20 diverse accessions;  citation_journal_title=Nat Genet;  citation_year=2013;  citation_volume=45;  citation_pages=51-82; " /><meta name="citation_reference" content="citation_title=The genome of the cucumber, Cucumis sativus L;  citation_journal_title=Nat. Genet;  citation_year=2009;  citation_volume=41;  citation_pages=1275-U1229; " /><meta name="citation_reference" content="citation_title=Helping hands for budding prospects: ENTH/ANTH/VHS accessory proteins in endocytosis, vacuolar transport, and secretion;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=4232-44; " /><meta name="citation_reference" content="citation_title=Assessing the function of the plant nuclear pore complex and the search for specificity;  citation_journal_title=J. Exp. Bot;  citation_year=2013;  citation_volume=64;  citation_pages=833-45; " /><meta name="citation_reference" content="citation_title=Retrotransposon replication in plants;  citation_journal_title=Curr. Opin. Virol;  citation_year=2013;  citation_volume=3;  citation_pages=604-14; " /><meta name="citation_reference" content="citation_title=A widespread class of reverse transcriptase-related cellular genes;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2011;  citation_volume=108;  citation_pages=20311-6; " /><meta name="citation_reference" content="citation_title=Comprehensive phylogenetic analysis of bacterial reverse transcriptases;  citation_journal_title=Plos One;  citation_year=2014;  citation_volume=9;  citation_pages=e114083" /><meta name="citation_reference" content="citation_title=The GATA family of transcription factors in Arabidopsis and rice;  citation_journal_title=Plant Physiol;  citation_year=2004;  citation_volume=134;  citation_pages=1718-32; " /><meta name="citation_reference" content="citation_title=Conservation, convergence, and divergence of light-responsive, circadian-regulated, and tissue-specific expression patterns during evolution of the Arabidopsis GATA gene family;  citation_journal_title=Plant Physiol;  citation_year=2007;  citation_volume=143;  citation_pages=941-58; " /><meta name="citation_reference" content="citation_title=Fungal effector proteins;  citation_journal_title=Annu. Rev. Phytopathol;  citation_year=2009;  citation_volume=47;  citation_pages=233-63; " /><meta name="citation_reference" content="citation_title=Plant response to bacterial pathogens. Overlap between innate and gene-for-gene defense response;  citation_journal_title=Plant Physiol;  citation_year=2006;  citation_volume=142;  citation_pages=809-11; " /><meta name="citation_reference" content="citation_title=Functional analysis of Avr9/Cf-9 rapidly elicited genes identifies a protein kinase, ACIK1, that is essential for full Cf-9-dependent disease resistance in tomato;  citation_journal_title=Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=295-310; " /><meta name="citation_reference" content="citation_title=CITRX thioredoxin is a putative adaptor protein connecting Cf-9 and the ACIK1 protein kinase during the Cf-9/Avr9-induced defence response;  citation_journal_title=FEBS Lett;  citation_year=2006;  citation_volume=580;  citation_pages=4236-41; " /><meta name="citation_reference" content="citation_title=The F-box protein ACRE189/ACIF1 regulates cell death and defense responses activated during pathogen recognition in tobacco and tomato;  citation_journal_title=Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=697-719; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/1/24151401/dsx033.pdf" /><meta name="description" content="Abstract.  The melon (Cucumis melo) genome and genetic maps with hundreds to thousands of single nucleotide polymorphism markers were recently released. However" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/1/4107259" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/1/4107259" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/1/4107259");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx033';
    </script>
<title>
	re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in Cucumis melo | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4107259" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbJ9PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdSwgWmhvbmd5dWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBHZXJtcGxhc20gSW5ub3ZhdGlvbiBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBJbnN0aXR1dGUgb2YgVmVnZXRhYmxlIFNjaWVuY2UsIFpoZWppYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEZW5nLCBHdWFuY29uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2VybXBsYXNtIElubm92YXRpb24gYW5kIE1vbGVjdWxhciBCcmVlZGluZywgSW5zdGl0dXRlIG9mIFZlZ2V0YWJsZSBTY2llbmNlLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBIYW5nemhvdSAzMTAwNTgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW91LCBIYWlwZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBHZXJtcGxhc20gSW5ub3ZhdGlvbiBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBJbnN0aXR1dGUgb2YgVmVnZXRhYmxlIFNjaWVuY2UsIFpoZWppYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJYdSwgWXVodWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9tYXJrZXIgVGVjaG5vbG9naWVzIENvcnBvcmF0aW9uLCBCZWlqaW5nIDEwMTMwMCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBMaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb21hcmtlciBUZWNobm9sb2dpZXMgQ29ycG9yYXRpb24sIEJlaWppbmcgMTAxMzAwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIEppbmdodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEdlcm1wbGFzbSBJbm5vdmF0aW9uIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIEluc3RpdHV0ZSBvZiBWZWdldGFibGUgU2NpZW5jZSwgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgR3Jvd3RoLCBEZXZlbG9wbWVudCAmYW1wOyBRdWFsaXR5IEltcHJvdmVtZW50LCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlpoZWppYW5nIFByb3ZpbmNpYWwgS2V5IExhYm9yYXRvcnkgb2YgSG9ydGljdWx0dXJhbCBQbGFudCBJbnRlZ3JhdGl2ZSBCaW9sb2d5LCBIYW5nemhvdSAzMTAwNTgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIE1pbmdmYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBHZXJtcGxhc20gSW5ub3ZhdGlvbiBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBJbnN0aXR1dGUgb2YgVmVnZXRhYmxlIFNjaWVuY2UsIFpoZWppYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBIb3J0aWN1bHR1cmFsIFBsYW50IEdyb3d0aCwgRGV2ZWxvcG1lbnQgJmFtcDsgUXVhbGl0eSBJbXByb3ZlbWVudCwgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJaaGVqaWFuZyBQcm92aW5jaWFsIEtleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgSW50ZWdyYXRpdmUgQmlvbG9neSwgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSByZS1zZXF1ZW5jaW5nLWJhc2VkIHVsdHJhLWRlbnNlIGdlbmV0aWMgbWFwIHJldmVhbHMgYSBndW1teSBzdGVtIGJsaWdodCByZXNpc3RhbmNlLWFzc29jaWF0ZWQgZ2VuZSBpbiBDdWN1bWlzIG1lbG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSBuYXRpdmUgcGxhbnRzIGluIENlbnRyYWwgRXVyb3BlIHRvIGN1bHRpdmF0ZWQgY3JvcHMgd29ybGR3aWRlOiB0aGUgZW1lcmdlbmNlIG9mIERpZHltZWxsYSBicnlvbmlhZSBhcyBhIEN1Y3VyYml0IFBhdGhvZ2VuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ib3J0c2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTUzMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmVlZGluZyBmb3IgZGlzZWFzZSByZXNpc3RhbmNlIGluIEN1Y3VyYml0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQaHl0b3BhdGhvbDsgIGNpdGF0aW9uX3llYXI9MTk3MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ3MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rml2ZSBpbmRlcGVuZGVudCBsb2NpIGVhY2ggY29udHJvbCBtb25vZ2VuaWMgcmVzaXN0YW5jZSB0byBndW1teSBzdGVtIGJsaWdodCBpbiBtZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTEwMzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYW1wbGlmaWVkIGZyYWdtZW50IGxlbmd0aCBwb2x5bW9ycGhpc20gbWFya2VycyBsaW5rZWQgdG8gZ3VtbXkgc3RlbSBibGlnaHQgKERpZHltZWxsYSBicnlvbmlhZSkgcmVzaXN0YW5jZSBpbiBNZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKSBQSSA0MjAxNDU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhvcnRzY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MzItNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJ5YW1pZGluZyBkaXNlYXNlIHJlc2lzdGFuY2UgZ2VuZXMgYW5kIHZhcmlldHkgaW1wcm92bWVudCBieSBtb2xlY3VsYXIgbWFya2VyLWFzc2lzdGVkIHNlbGVjdGlvbiBpbiBtZWxvbiAoQ3VjdW1pcyBtZWxvbiBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gQWdyaWMuIFNpbjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTUyMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbGlua2FnZSBtYXAgZm9yIG1vbGVjdWxhciBtYXJrZXJzIGFuZCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBhc3NvY2lhdGVkIHdpdGggZWNvbm9taWNhbGx5IGltcG9ydGFudCB0cmFpdHMgaW4gbWVsb24gKEN1Y3VtaXMgbWVsbyBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5ldGljIG1hcCBvZiBtZWxvbiBoaWdobHkgZW5yaWNoZWQgd2l0aCBmcnVpdCBxdWFsaXR5IFFUTHMgYW5kIEVTVCBtYXJrZXJzLCBpbmNsdWRpbmcgc3VnYXIgYW5kIGNhcm90ZW5vaWQgbWV0YWJvbGlzbSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTUxMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBtZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKSBhbmQgbG9jYWxpemF0aW9uIG9mIGEgbWFqb3IgUVRMIGZvciBwb3dkZXJ5IG1pbGRldyByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTgxLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGFuZCB0aGUgY29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwIGZvciBtZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKSB1c2luZyBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgbWVsb24gKEN1Y3VtaXMgbWVsbyBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTE4NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5jaG9yaW5nIHRoZSBjb25zZW5zdXMgSUN1R0kgZ2VuZXRpYyBtYXAgdG8gdGhlIG1lbG9uIChDdWN1bWlzIG1lbG8gTC4pIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTE4OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGdlbmV0aWMgbWFya2VyIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDk5LTUxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGdlbm90eXBpbmcgYnkgd2hvbGUtZ2Vub21lIHJlc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTEwNjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpbnBvaW50aW5nIGdlbmVzIHVuZGVybHlpbmcgdGhlIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGZvciByb290LWtub3QgbmVtYXRvZGUgcmVzaXN0YW5jZSBpbiBwYWxhZW9wb2x5cGxvaWQgc295YmVhbiBieSB3aG9sZSBnZW5vbWUgcmVzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTEzNDY5LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1iYXNlZCB1bHRyYS1kZW5zZSBnZW5ldGljIGFuZCBwaHlzaWNhbCBtYXBzIHJldmVhbCBzdHJ1Y3R1cmFsIHZhcmlhdGlvbnMgb2YgYWxsb3BvbHlwbG9pZCBjb3R0b24gZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSByZWZlcmVuY2UgZ2VuZXRpYyBtYXAgb2YgUmFwaGFudXMgc2F0aXZ1cyBiYXNlZCBvbiBnZW5vdHlwaW5nIGJ5IHdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0yNTktNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdldHRpbmcgc3RhcnRlZCBpbiBtYXBwaW5nLWJ5LXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEludGVnci4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTYwNi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdG8gaXNvbGF0ZSBtdXRhbnQgZ2VuZXMgZnJvbSBmb3J3YXJkIGdlbmV0aWMgc2NyZWVuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NjYyLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3N0LW1pY3JvYmUgaW50ZXJhY3Rpb25zOiBzaGFwaW5nIHRoZSBldm9sdXRpb24gb2YgdGhlIHBsYW50IGltbXVuZSByZXNwb25zZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz04MDMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IGltbXVuaXR5OiB0b3dhcmRzIGFuIGludGVncmF0ZWQgdmlldyBvZiBwbGFudC1wYXRob2dlbiBpbnRlcmFjdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTUzOS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBsYW50IGltbXVuZSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz0zMjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaW5nIHNpZ25hbGluZyBtZWNoYW5pc21zIGVuZ2FnZWQgaW4gcGF0dGVybi10cmlnZ2VyZWQgYW5kIGVmZmVjdG9yLXRyaWdnZXJlZCBpbW11bml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhbmRhcmRpemF0aW9uIG9mIGEgbWFzcyBwcm9kdWN0aW9uIHRlY2huaXF1ZSBmb3IgcHljbmlkaW9zcG9yZXMgb2YgRGlkeW1lbGxhIGJyeW9uaWFlLCBndW1teSBzdGVtIGJsaWdodCBmdW5ndXMgb2YgQ3VjdXJiaXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Lb3JlYW4gSi4gUGxhbnQgUGF0aG9sOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTA1LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZnJhbWV3b3JrIGZvciB2YXJpYXRpb24gZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NDkxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgYW5hbHlzaXMgb2YgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIHVzaW5nIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lOTg4NTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BTExNQVBTOiByb2J1c3Qgc2NhZmZvbGQgb3JkZXJpbmcgYmFzZWQgb24gbXVsdGlwbGUgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwIGNvbnN0cnVjdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2YgYSBsb2N1cyBjb250cm9sbGluZyB3ZWVwaW5nIHRyYWl0IGluIGFuIG9ybmFtZW50YWwgd29vZHkgcGxhbnQgKFBydW51cyBtdW1lIFNpZWIuIGV0IFp1Y2MpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTgzLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9QSBndWlkZSB0byBRVEwgbWFwcGluZyB3aXRoIFEvcXRsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgdGFyZ2V0ZWQgU05QIHNlbGVjdGlvbiBmb3IgYW4gaW1wcm92ZWQgYW5jaG9yaW5nIG9mIHRoZSBtZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKSBzY2FmZm9sZCBnZW5vbWUgYXNzZW1ibHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgZm9yIHBsYW50IGJyZWVkaW5nIGFuZCBnZW5ldGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lLVVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05Mi0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtZGVuc2l0eSBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gYW5kIG1hcHBpbmcgb2Ygc2VlZCB0cmFpdCBRVExzIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pIHVzaW5nIGdlbm90eXBpbmctYnktc2VxdWVuY2luZyAoR0JTKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXAuLVVrOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNzUxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmctYnktc2VxdWVuY2luZyBtYXAgcGVybWl0cyBpZGVudGlmaWNhdGlvbiBvZiBjbHVicm9vdCByZXNpc3RhbmNlIFFUTHMgYW5kIHJldmlzaW9uIG9mIHRoZSByZWZlcmVuY2UgZ2Vub21lIGFzc2VtYmx5IGluIGNhYmJhZ2UgKEJyYXNzaWNhIG9sZXJhY2VhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dcm9wIGdlbm9tZSBzZXF1ZW5jaW5nOiBsZXNzb25zIGFuZCByYXRpb25hbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NzctODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2Ygd2F0ZXJtZWxvbiAoQ2l0cnVsbHVzIGxhbmF0dXMpIGFuZCByZXNlcXVlbmNpbmcgb2YgMjAgZGl2ZXJzZSBhY2Nlc3Npb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgY3VjdW1iZXIsIEN1Y3VtaXMgc2F0aXZ1cyBMOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTI3NS1VMTIyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVscGluZyBoYW5kcyBmb3IgYnVkZGluZyBwcm9zcGVjdHM6IEVOVEgvQU5USC9WSFMgYWNjZXNzb3J5IHByb3RlaW5zIGluIGVuZG9jeXRvc2lzLCB2YWN1b2xhciB0cmFuc3BvcnQsIGFuZCBzZWNyZXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MjMyLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgdGhlIGZ1bmN0aW9uIG9mIHRoZSBwbGFudCBudWNsZWFyIHBvcmUgY29tcGxleCBhbmQgdGhlIHNlYXJjaCBmb3Igc3BlY2lmaWNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9ODMzLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXRyb3RyYW5zcG9zb24gcmVwbGljYXRpb24gaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBWaXJvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NjA0LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHdpZGVzcHJlYWQgY2xhc3Mgb2YgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlLXJlbGF0ZWQgY2VsbHVsYXIgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MjAzMTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgb2YgYmFjdGVyaWFsIHJldmVyc2UgdHJhbnNjcmlwdGFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb3MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTE0MDgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdBVEEgZmFtaWx5IG9mIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiBBcmFiaWRvcHNpcyBhbmQgcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz0xNzE4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24sIGNvbnZlcmdlbmNlLCBhbmQgZGl2ZXJnZW5jZSBvZiBsaWdodC1yZXNwb25zaXZlLCBjaXJjYWRpYW4tcmVndWxhdGVkLCBhbmQgdGlzc3VlLXNwZWNpZmljIGV4cHJlc3Npb24gcGF0dGVybnMgZHVyaW5nIGV2b2x1dGlvbiBvZiB0aGUgQXJhYmlkb3BzaXMgR0FUQSBnZW5lIGZhbWlseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDM7ICBjaXRhdGlvbl9wYWdlcz05NDEtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmdhbCBlZmZlY3RvciBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQaHl0b3BhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTIzMy02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgcmVzcG9uc2UgdG8gYmFjdGVyaWFsIHBhdGhvZ2Vucy4gT3ZlcmxhcCBiZXR3ZWVuIGlubmF0ZSBhbmQgZ2VuZS1mb3ItZ2VuZSBkZWZlbnNlIHJlc3BvbnNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTgwOS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbmFseXNpcyBvZiBBdnI5L0NmLTkgcmFwaWRseSBlbGljaXRlZCBnZW5lcyBpZGVudGlmaWVzIGEgcHJvdGVpbiBraW5hc2UsIEFDSUsxLCB0aGF0IGlzIGVzc2VudGlhbCBmb3IgZnVsbCBDZi05LWRlcGVuZGVudCBkaXNlYXNlIHJlc2lzdGFuY2UgaW4gdG9tYXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9Mjk1LTMxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0lUUlggdGhpb3JlZG94aW4gaXMgYSBwdXRhdGl2ZSBhZGFwdG9yIHByb3RlaW4gY29ubmVjdGluZyBDZi05IGFuZCB0aGUgQUNJSzEgcHJvdGVpbiBraW5hc2UgZHVyaW5nIHRoZSBDZi05L0F2cjktaW5kdWNlZCBkZWZlbmNlIHJlc3BvbnNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTgwOyAgY2l0YXRpb25fcGFnZXM9NDIzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEYtYm94IHByb3RlaW4gQUNSRTE4OS9BQ0lGMSByZWd1bGF0ZXMgY2VsbCBkZWF0aCBhbmQgZGVmZW5zZSByZXNwb25zZXMgYWN0aXZhdGVkIGR1cmluZyBwYXRob2dlbiByZWNvZ25pdGlvbiBpbiB0b2JhY2NvIGFuZCB0b21hdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz02OTctNzE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMS8xLzI0MTUxNDAxL2RzeDAzMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIG1lbG9uIChDdWN1bWlzIG1lbG8pIGdlbm9tZSBhbmQgZ2VuZXRpYyBtYXBzIHdpdGggaHVuZHJlZHMgdG8gdGhvdXNhbmRzIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIHdlcmUgcmVjZW50bHkgcmVsZWFzZWQuIEhvd2V2ZXIiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQxMDcyNTkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkY506lOMgByzlhXXl0Y7lbPNoI44=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f1%2f4107259" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f1%2f4107259" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112481435">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481435">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481437">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481437">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481442">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481442">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481462">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481462">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481484">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481484">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481490">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481490">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481492">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481500">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481500">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/11/4106313">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in <em>Cucumis melo</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhongyuan Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhongyuan+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhongyuan Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guancong Deng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guancong+Deng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Deng G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Deng G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guancong Deng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haipeng Mou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haipeng+Mou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mou H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mou H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haipeng Mou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuhui Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Biomarker Technologies Corporation, Beijing 101300, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuhui+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuhui Xu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Li Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Biomarker Technologies Corporation, Beijing 101300, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Li+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Li Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinghua Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China</div><div class="aff">Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China</div><div class="aff">Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx033-cor1" data-open="dsx033-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 57188982123. Fax. +86 57188982123. Email: <a href="mailto:yangjinghua@zju.edu.cn" target="_blank">yangjinghua@zju.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jinghua+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jinghua Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mingfang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratory of Germplasm Innovation and Molecular Breeding, Institute of Vegetable Science, Zhejiang University, Hangzhou 310058, China</div><div class="aff">Key Laboratory of Horticultural Plant Growth, Development &amp; Quality Improvement, Ministry of Agriculture, Hangzhou 310058, China</div><div class="aff">Zhejiang Provincial Key Laboratory of Horticultural Plant Integrative Biology, Hangzhou 310058, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mingfang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mingfang Zhang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx033-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Doil Choi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 1–10, <a href='https://doi.org/10.1093/dnares/dsx033'>https://doi.org/10.1093/dnares/dsx033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 May 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4107259" href="/dnaresearch/article-pdf/25/1/1/24151401/dsx033.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Zhongyuan Hu, Guancong Deng, Haipeng Mou, Yuhui Xu, Li Chen, Jinghua Yang, Mingfang Zhang;  A re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in <em>Cucumis melo</em>, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 1–10, <a href='https://doi.org/10.1093/dnares/dsx033'>https://doi.org/10.1093/dnares/dsx033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107259?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20re-sequencing-based%20ultra-dense%20genetic%20map%20reveals%20a%20gummy%20stem%20blight%20resistance-associated%20gene%20in%20%26lt%3Bem%26gt%3BCucumis%20melo%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-09-07&amp;volumeNum=25&amp;issueNum=1&amp;author=Hu%2C%20Zhongyuan%3B%20Deng%2C%20Guancong&amp;startPage=1&amp;endPage=10&amp;contentId=10.1093%2Fdnares%2Fdsx033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f1%2f4107259" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112481435 id="112481435" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The melon (<em>Cucumis melo</em>) genome and genetic maps with hundreds to thousands of single nucleotide polymorphism markers were recently released. However, a high-resolution genetic map was lacking. Gummy stem blight (Gsb) is a destructive disease responsible for considerable economic losses during melon production. We herein describe the development of an ultra-dense genetic map consisting of 12,932 recombination bin markers covering 1,818 cM, with an average distance of 0.17 cM between adjacent tags. A comparison of the genetic maps for melon, watermelon, and cucumber revealed chromosome-level syntenic relationships and recombination events among the three <em>Cucurbitaceae</em> species. Our genetic map was useful for re-anchoring the genome scaffolds of melon. More than 92% assembly was anchored to 12 pseudo-chromosomes and 90% of them were oriented. Furthermore, 1,135 recombination hotspots revealed an unbalanced recombination rate across the melon genome. Genetic analyses of the Gsb-resistant and -susceptible lines indicated the resistance phenotype is mediated by a single dominant gene. We identified Gsb-resistance gene candidates in a 108-kb region on pseudo-chromosome 4. Our findings verify the utility of an ultra-dense genetic map for mapping a gene of interest, and for identifying new disease resistant genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ultra-dense genetic map&quot;">ultra-dense genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="re-sequencing">re-sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gummy stem blight&quot;">gummy stem blight</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cucumis melo&quot;"><em>Cucumis melo</em></a></div>                            <h2 scrollto-destination=112481437 id="112481437" class="section-title" >1. Introduction</h2>
<p>Gummy stem blight (Gsb) is a destructive disease caused by the ascomycetous fungus <em>Didymella bryoniae</em> (Auersw.) Rehm, which can infect many cucurbit species.<a reveal-id="dsx033-B1" data-open="dsx033-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Because of the agricultural importance of this disease, cucurbit sources of genetic resistance were selected several decades ago.<a reveal-id="dsx033-B2" data-open="dsx033-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Gsb of melon has increased in importance because it is responsible for considerable economic losses, especially in humid tropical and sub-tropical regions. Many Gsb-resistant melon varieties have been identified, and genetic analyses revealed several independent Gsb-resistance loci, including five monogenic dominant resistance loci (<em>Gsb-1</em>, <em>Gsb-2</em>, <em>Gsb-3</em>, <em>Gsb-4</em>, and <em>Gsb-6</em>) and one monogenic recessive resistance locus (<em>gsb-5</em>) from six resistant PIs (PI140471, PI157082, PI511890, PI482398, PI420145, and PI482399), respectively.<a reveal-id="dsx033-B3" data-open="dsx033-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx033-B4" data-open="dsx033-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Different molecular markers linked to Gsb resistance have been used for marker-assisted selection of Gsb-resistant germplasms.<a reveal-id="dsx033-B5" data-open="dsx033-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> However, a functional Gsb-resistance gene has not been identified.</p><p>Several genetic linkage maps from different populations have been constructed based on relatively few molecular markers, which were used to map quantitative trait loci (QTLs) for agronomic and disease-resistance traits in melon.<a reveal-id="dsx033-B6 dsx033-B7 dsx033-B8 dsx033-B9" data-open="dsx033-B6 dsx033-B7 dsx033-B8 dsx033-B9" class="link link-ref link-reveal xref-bibr"><sup>6–9</sup></a> After the whole melon (<em>Cucumis melo</em>) genome sequence was published,<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> those available simple sequence repeat and single nucleotide polymorphism (SNP) markers were integrated to build preferably genetic maps useful for anchoring contigs/scaffolds or locating QTLs.<a reveal-id="dsx033-B11" data-open="dsx033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> High-throughput re-sequencing has been proposed as a viable option for detecting markers, genotyping and constructing genetic maps.<a reveal-id="dsx033-B12" data-open="dsx033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> It has been used to construct ultra-dense genetic maps based on the availability of advanced recombinant inbred lines or an F<sub>2</sub> population.<a reveal-id="dsx033-B13 dsx033-B14 dsx033-B15 dsx033-B16" data-open="dsx033-B13 dsx033-B14 dsx033-B15 dsx033-B16" class="link link-ref link-reveal xref-bibr"><sup>13–16</sup></a> Many important QTLs or genes have been identified using mapping-by-sequencing methods, which have accelerated forward-genetics research even in non-model species.<a reveal-id="dsx033-B17" data-open="dsx033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx033-B18" data-open="dsx033-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p><p>During the co-evolution of plants and microbes, host-adapted pathogens evolved effectors to suppress pathogen-associated molecular patterns-triggered immunity (PTI), which is the first basal defence against infections by non-adapted pathogens. Effector-triggered immunity (ETI) conferred by plant resistance (R) genes contributes to a highly effective defence system, which is deployed by plants as a second, largely intracellular, layer of immunity.<a reveal-id="dsx033-B19 dsx033-B20 dsx033-B21 dsx033-B22" data-open="dsx033-B19 dsx033-B20 dsx033-B21 dsx033-B22" class="link link-ref link-reveal xref-bibr"><sup>19–22</sup></a> A coordinated relay of information between the cytoplasm and nucleus by intracellular signalling systems is required for both PTI and ETI to ensure defence-related transcripts are transferred from nucleus to the cytoplasmic protein synthesis machinery.</p><p>In this study, we constructed an ultra-dense genetic map with 12,932 recombination bin markers comprising 1,188,159 SNPs. This map may be useful for updating the melon genome and for mapping <em>C. melo</em> genes. For example, we mapped a 108-kb region associated with Gsb resistance on pseudo-chromosome 4. This ultra-dense genetic map may be useful for future molecular analyses of melon.</p>                            <h2 scrollto-destination=112481442 id="112481442" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112481443 id="112481443" class="section-title" >2.1. Plant materials and <em>Didymella bryoniae</em> inoculation</h3>
<p>An inbred line of <em>Cucumis melo</em> spp. <em>conomon</em> (var. <em>conomon</em>) (HS) and an inbred line of <em>Cucumis melo</em> spp. <em>melo</em> (var. <em>inodorus</em>) (XH) were crossed to generate F<sub>1</sub>, F<sub>2</sub>, and BC<sub>1</sub> populations. The HS and XH lines as well as their F<sub>1</sub>, F<sub>2</sub>, and BC<sub>1</sub> populations were used for genetic analyses. The F<sub>2</sub> population was used to construct a genetic map and map the candidate Gsb<em>-</em>resistance gene.</p><p>Plants were inoculated with <em>D. bryoniae</em> using a modified version of a published method.<a reveal-id="dsx033-B23" data-open="dsx033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The <em>D. bryoniae</em> spores collected from infected melon plants were cultured on potato dextrose agar medium in Petri dishes using a mycelial plug inoculation. The Petri dishes were incubated at 25 °C in the darkness for 7 days and then under ultraviolet irradiation conditions (40 w, 12 h light/12 h dark) for 4 days. Inocula were prepared by flooding the Petri dishes with 5–10 ml acidified distilled water. The resulting solution was acidified to pH 4.0 using lactic acid, and Tween-20 was added as a surfactant (20 drops/l). The spore suspension was adjusted to 5 × 10<sup>5</sup> spores/ml using a haemocytometer, and then applied to 3- to 4-week-old plants in a foliar spray. Inoculated plants were covered by a plastic tent to maintain a relative humidity of 92%. The plastic wrap was removed 3 days later. Plants were re-inoculated to ensure there were no escapees or false-positives.</p>                            <h3 scrollto-destination=112481446 id="112481446" class="section-title" >2.2. Sequencing library construction and high-throughput sequencing</h3>
<p>Genomic DNAs was extracted from the leaves of the parental lines (HS and XH) and 150 individual lines from the F<sub>2</sub> population according to the CTAB method. The genomic DNAs were sheared into ∼500-bp fragments using the S2/E210 Ultrasonicator (Covaris, USA) for the subsequent paired-end (PE125) sequencing using the HiSeq2500 system (Illunima, USA).</p>                            <h3 scrollto-destination=112481448 id="112481448" class="section-title" >2.3. SNP identification and genotyping</h3>
<p>Raw reads were filtered to generate clean reads, which were mapped to the genome to estimate the insert sizes in the sequencing library, calculate the reads depth, and determine the distribution of reads in the melon genome (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/assembly/GCF_000313045.1/" target="_blank">http://www.ncbi.nlm.nih.gov/assembly/GCF_000313045.1/</a> (23 August 2017, date last accessed))<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> using the Burrows–Wheeler Aligner programme.<a reveal-id="dsx033-B24" data-open="dsx033-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Duplicated reads were removed using SAMtools.<a reveal-id="dsx033-B25" data-open="dsx033-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Potential SNPs between all lines and the genome were detected using the GATK toolkit.<a reveal-id="dsx033-B26" data-open="dsx033-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The SNPs identified between the parents were considered as polymorphic for a subsequent bin calling. The F<sub>2</sub> genotypes were identified based on the SNP positions. The ‘pileup’ function of the SAMtools programme was used to merge the SNP datasets, in which only biallelic SNPs were kept. Short reads that were matched to multiple locations in either genome, or that did not match perfectly with at least one of the parental genomes were discarded. The VCF format document was used for genotyping according to the parental SNPs, and only the aa × bb genotype was kept. The SNPs with less than 4× coverage in either parent or with fewer than 15 SNPs in a single scaffold were eliminated. Additionally, the SNPs with an extreme segregation distortion (<em>P</em> &lt; 0.01) were initially excluded by the <em>χ</em><sup>2</sup> test for a subsequent bin calling. </p>                            <h3 scrollto-destination=112481450 id="112481450" class="section-title" >2.4. Bin map and linkage map construction</h3>
<p>A modified slide window method was adopted for bin calling.<a reveal-id="dsx033-B13" data-open="dsx033-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The genotype of each window was called with a window size of 15 SNPs and a step size of 1 SNP. Windows containing more than 13 ‘aa’ or ‘bb’ types were genotyped as ‘aa’ or ‘bb’, respectively. Fifteen adjacent SNP intervals with the same genotype across the entire F<sub>2</sub> population were combined into a recombination bin. A final set of bin markers was used to construct linkage groups (LGs) with the HighMap programme.<a reveal-id="dsx033-B27" data-open="dsx033-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Marker loci were partitioned primarily into LGs based on their locations in the <em>C. melo</em> genome. Next, the modified logarithm of odds (MLOD) scores between markers were calculated to verify the robustness of the markers in each LG. Markers with MLOD scores &lt;5 were filtered prior to ordering. The error correction strategy of SMOOTH was then used according to the parental contributions to genotypes, and a k-nearest neighbour algorithm was applied to impute missing genotypes. Skewed markers were then added into the map by applying a multipoint maximum likelihood mapping. Map distances were estimated using the Kosambi mapping function. We then used the ALLMAPS programme to construct melon pseudo-chromosomes based on a comparison between the constructed genetic map and the map of the melon genome.<a reveal-id="dsx033-B28" data-open="dsx033-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Finally, we constructed pseudo-chromosomes referred to the published genome of melon.</p>                            <h3 scrollto-destination=112481452 id="112481452" class="section-title" >2.5. Analysis of co-linearity and recombination hotspots</h3>
<p>The sequences of the bin markers included in the linkage map were aligned to the physical sequences of <em>C. lanatus</em> (<a class="link link-uri" href="http://ftp://www.icugi.org/pub/genome/watermelon/97103/v1/" target="_blank">ftp://www.icugi.org/pub/genome/watermelon/97103/v1/</a> (23 August 2017, date last accessed)) and <em>C. sativus</em> (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genome/1639?genome_assembly_id=228904" target="_blank">http://www.ncbi.nlm.nih.gov/genome/1639?genome_assembly_id=228904</a> (23 August 2017, date last accessed)) using the BLAT programme to confirm their physical positions in the genome. Spearman’s rank-order correlation coefficient was calculated to assess the co-linearity between <em>C. melo</em> and <em>C. lanatus</em>, and between <em>C. melo</em> and <em>C. sativus</em>. Recombination hotspots (RHs) were predicted based on the recombination rate of markers and the anchored <em>C. melo</em> genome. If the value that the genetic distance between adjacent markers was divided by a value &gt;20 cM/Mb, the region between the two adjacent markers was considered to be an RH.<a reveal-id="dsx033-B29" data-open="dsx033-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a></p>                            <h3 scrollto-destination=112481454 id="112481454" class="section-title" >2.6. Re-anchoring scaffolds</h3>
<p>Filtered markers were mapped to genome assembly scaffolds, and the markers on scaffolds that were mapped to pseudo-chromosomes were counted. If the value of the following equation was &gt;0.5, we kept the scaffold mapping results, otherwise, they were eliminated: [number of markers in the largest pseudo-chromosome − number of markers in the second largest pseudo-chromosome]/number of markers in the largest pseudo-chromosome. We then determined the position of each marker relative to each other. If the relative position in the scaffold was identical to the relative position in the genetic map, the scaffold was considered to be oriented in the forward direction. Additionally, if more markers were in the forward position than in the reverse position, the scaffold was considered to be facing the forward direction. Finally, we selected one clearly oriented scaffold as the initial scaffold, and iteratively anchored other scaffolds with higher scores.</p>                            <h3 scrollto-destination=112481456 id="112481456" class="section-title" >2.7. Mapping of candidate Gsb-resistance genes</h3>
<p>The rQTL package was used to map the candidate Gsb-resistance genes. The IM method of the rQTL package was applied for qualitative trait mapping.<a reveal-id="dsx033-B30" data-open="dsx033-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The significance thresholds were determined using 1,000 permutations, in which the threshold value was set as 2.5. The hk algorithm was selected as the me parameter, while 1 and 3 were used as the win and cofactor parameters, respectively.</p>                            <h3 scrollto-destination=112481458 id="112481458" class="section-title" >2.8. RNA isolation, cDNA preparation, and quantitative real-time PCR</h3>
<p>Leaves from HS and XH plants after inoculation (0, 0.5, 1, 2, 3, 5, and 7 days) were sampled and frozen in liquid nitrogen. Total RNA was isolated from leaves using mirVanaTM miRNA Isolation Kit (Ambion) according to the manufacturer’s instructions. RNA concentration and quality were assessed using a Thermo 2000 Bioanalyzer with an RNA NanoDrop (Thermo Scientific, USA; <a class="link link-uri" href="http://www.thermo.com" target="_blank">http://www.thermo.com</a> (23 August 2017, date last accessed)). Samples showing A260/A230 ratio of 2.0–2.2 and A260/A280 ratio of 1.8–2.0 were used for further analysis. For quantitative real-time PCR, 1 µg of total RNA was reverse transcribed to first-strand cDNA in a final reaction volume of 20 µl using miScript II RT Kit (Qiagen). The transcriptional patterns of the candidate Gsb-resistance genes were analysed using a quantitative real-time PCR (RT-qPCR). Details regarding the primers used in this study are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>.</p>                            <h3 scrollto-destination=112481460 id="112481460" class="section-title" >2.9. Genotyping analysis of <em>MELO03C012987</em> gene</h3>
<p>Full-length sequence of <em>MELO03C012987</em> gene (except for stop codon) was amplified by PCR from leaves of six resistant inbred lines (HP9818, HB6, HB11, HB13, CNZ, and QDH) and five susceptible inbred lines (RM, MW, DZX, M14086, and M15019) to analyse the SNP further. The primers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. The resulting PCR products were cloned into a <em>pEASY</em>-Blunt Zero Cloning Vector according to the manufacturer’s instructions (TRANs, Beijing, China; <a class="link link-uri" href="http://www.transgen.com.cn" target="_blank">www.transgen.com.cn</a> (23 August 2017, date last accessed)) and amplified in <em>E coil</em> overnight. Subsequently, eight clones of each line were sequenced by ABI3730xl sequencer. Few clones with sequence that did not match perfectly with reference full length were discarded.</p>                            <h2 scrollto-destination=112481462 id="112481462" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=112481463 id="112481463" class="section-title" >3.1. Gsb resistance in <em>C. melo</em> is controlled by a single dominant gene</h3>
<p>The HS and XH inbred lines were used to construct a genetic map and evaluate Gsb resistance (<a reveal-id="dsx033-F1" data-open="dsx033-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Plants growth and the development of disease symptoms in seedlings inoculated with <em>D. bryoniae</em> were used to assess Gsb resistance or susceptibility. Accordingly, the HS and XH inbred lines were resistant and susceptible to Gsb, respectively (<a reveal-id="dsx033-F1" data-open="dsx033-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and <a reveal-id="dsx033-T1" data-open="dsx033-T1" class="link link-reveal link-table xref-fig">Table 1</a>). A phenotypic analysis revealed that the F<sub>1</sub> population from the XH × HS cross was resistant to Gsb, which is suggestive of a dominant resistance trait (<a reveal-id="dsx033-F1" data-open="dsx033-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a> and <a reveal-id="dsx033-T1" data-open="dsx033-T1" class="link link-reveal link-table xref-fig">Table 1</a>). A phenotypic analysis of the F<sub>2</sub> population derived from the XH × HS reciprocal cross revealed a 1:3 segregation ratio for the F<sub>2</sub> population and a 1:1 segregation ratio for the BC<sub>1</sub> population, which is consistent with a monogenic resistance trait (<a reveal-id="dsx033-T1" data-open="dsx033-T1" class="link link-reveal link-table xref-fig">Table 1</a>). <div content-id="dsx033-T1" class="table-modal"><div class="table-wrap"><div id="dsx033-T1" data-id="dsx033-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Inheritance of Gsb resistance in <em>C. melo</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Crosses </th><th>Generation </th><th>Resistant </th><th>Susceptible </th><th>Expected ratio (R:S) </th><th>Chi-square </th><th><strong><em>P</em></strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td>HS </td><td>P1 </td><td>20 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH </td><td>P2 </td><td>0 </td><td>20 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>1</sub> </td><td>23 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>2</sub> </td><td>165 </td><td>54 </td><td>3:1 </td><td>0.013 </td><td>0.91 </td></tr><tr><td>HS×XH </td><td>F<sub>2</sub> </td><td>81 </td><td>32 </td><td>3:1 </td><td>0.663 </td><td>0.42 </td></tr><tr><td>(XH×HS)×XH </td><td>BC<sub>1</sub> </td><td>47 </td><td>73 </td><td>1:1 </td><td>5.663 </td><td>0.017 </td></tr></tbody></table></div><div content-id="dsx033-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Crosses </th><th>Generation </th><th>Resistant </th><th>Susceptible </th><th>Expected ratio (R:S) </th><th>Chi-square </th><th><strong><em>P</em></strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td>HS </td><td>P1 </td><td>20 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH </td><td>P2 </td><td>0 </td><td>20 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>1</sub> </td><td>23 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>2</sub> </td><td>165 </td><td>54 </td><td>3:1 </td><td>0.013 </td><td>0.91 </td></tr><tr><td>HS×XH </td><td>F<sub>2</sub> </td><td>81 </td><td>32 </td><td>3:1 </td><td>0.663 </td><td>0.42 </td></tr><tr><td>(XH×HS)×XH </td><td>BC<sub>1</sub> </td><td>47 </td><td>73 </td><td>1:1 </td><td>5.663 </td><td>0.017 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Phenotypic observations for the whole plant 21 days post-inoculation were used to determine resistance/susceptible to Gsb.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">a</div><p>Observed segregation ratios are statistically consistent with the expected ratios (<em>χ</em><sup>2</sup> test, <em>P</em> &lt; 0.05).</p></div></div></div></div></div><div class="table-wrap"><div id="dsx033-T1" data-id="dsx033-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Inheritance of Gsb resistance in <em>C. melo</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Crosses </th><th>Generation </th><th>Resistant </th><th>Susceptible </th><th>Expected ratio (R:S) </th><th>Chi-square </th><th><strong><em>P</em></strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td>HS </td><td>P1 </td><td>20 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH </td><td>P2 </td><td>0 </td><td>20 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>1</sub> </td><td>23 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>2</sub> </td><td>165 </td><td>54 </td><td>3:1 </td><td>0.013 </td><td>0.91 </td></tr><tr><td>HS×XH </td><td>F<sub>2</sub> </td><td>81 </td><td>32 </td><td>3:1 </td><td>0.663 </td><td>0.42 </td></tr><tr><td>(XH×HS)×XH </td><td>BC<sub>1</sub> </td><td>47 </td><td>73 </td><td>1:1 </td><td>5.663 </td><td>0.017 </td></tr></tbody></table></div><div content-id="dsx033-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Crosses </th><th>Generation </th><th>Resistant </th><th>Susceptible </th><th>Expected ratio (R:S) </th><th>Chi-square </th><th><strong><em>P</em></strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td>HS </td><td>P1 </td><td>20 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH </td><td>P2 </td><td>0 </td><td>20 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>1</sub> </td><td>23 </td><td>0 </td><td> </td><td> </td><td> </td></tr><tr><td>XH×HS </td><td>F<sub>2</sub> </td><td>165 </td><td>54 </td><td>3:1 </td><td>0.013 </td><td>0.91 </td></tr><tr><td>HS×XH </td><td>F<sub>2</sub> </td><td>81 </td><td>32 </td><td>3:1 </td><td>0.663 </td><td>0.42 </td></tr><tr><td>(XH×HS)×XH </td><td>BC<sub>1</sub> </td><td>47 </td><td>73 </td><td>1:1 </td><td>5.663 </td><td>0.017 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Phenotypic observations for the whole plant 21 days post-inoculation were used to determine resistance/susceptible to Gsb.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">a</div><p>Observed segregation ratios are statistically consistent with the expected ratios (<em>χ</em><sup>2</sup> test, <em>P</em> &lt; 0.05).</p></div></div></div></div></p>                        <a id="112481465" scrollto-destination="112481465"></a>
<div id="dsx033-F1" data-id="dsx033-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f1.png?Expires=2147483647&Signature=RSQXS-I5ydO56Y-OPMhqWEMKZtRuhWNjGeXe6n3xPZbYqyWQN1ClV~ALIZnldm-SyOfT6ylWJBHndzcz8k31GGLTHE~ZVjR~YIN6Fqx-yBtbN7pCNb54uugiMS30IQvBb58H0JCV7maNjGFCkMfkvijOOx5f5cyCvSlZG6~wUinwN6GouK3rKRyfJ9CgtU710eys55SEfp1p-KVhUD5e3mi-S4EcFr0cPF1U5~t45Qr6Lwn2qAzWd4ufgapkMGIvaLBUHcTiq1WjEkyBw38l6NC~8hACU3WwH9pijEoKmtsNvfDwWI2-7WEGNfJUHkxRhXztu6W8Wv35CGdLUdALmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotypic analysis of the two inbred lines used to construct a genetic map and for mapping the Gsb-resistance gene. (A) Phenotypes of C. melo spp. conomon (HS) and C. melo spp. melo (XH) under normal conditions. (B) Phenotypes of the two inbred lines and their F1 generation after being inoculated with Didymella bryoniae." path-from-xml="dsx033f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481465/dsx033f1.tif" path-from-xml="dsx033f1.tif" target="_blank">View large</a><a section="112481465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f1.png?Expires=2147483647&Signature=qj39cu-d~9C8ktCxQ3w6dR7Hx70izTMHZA9bpo2tECZZhwYlQPSSS1G1SPgu4qr0k0SAH2HFwi09V3wfaL1Gqzq4SFi8SNiv2GLm34mhYzAUXp7CFMOyRMvAzCRIOtoP7-Kkmld5ylRyqQwzHJudCt2FVmUYYOEbGFHqh7fchYdJ06Sa8dFRv-RCg8ZgYGs2ZIbV8a~gXWocuDJNSxuD9GC7ofnY24Bqeg8gWcl2hPZeb6R9rtRnoEhTSWTf0FQqQmz7S0vHiuQXjko4rlwK0ASxK-qVHZK2ZVCZpLbWc6bPsSb2lAMO53a5XaszIr0hHIKBpcBua6s-iqVInXuZJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481465&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypic analysis of the two inbred lines used to construct a genetic map and for mapping the Gsb-resistance gene. <strong>(A)</strong> Phenotypes of <em>C. melo</em> spp. <em>conomon</em> (HS) and <em>C. melo</em> spp. <em>melo</em> (XH) under normal conditions. <strong>(B)</strong> Phenotypes of the two inbred lines and their F<sub>1</sub> generation after being inoculated with <em>Didymella bryoniae</em>.</p></div></div><div content-id="dsx033-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f1.png?Expires=2147483647&Signature=RSQXS-I5ydO56Y-OPMhqWEMKZtRuhWNjGeXe6n3xPZbYqyWQN1ClV~ALIZnldm-SyOfT6ylWJBHndzcz8k31GGLTHE~ZVjR~YIN6Fqx-yBtbN7pCNb54uugiMS30IQvBb58H0JCV7maNjGFCkMfkvijOOx5f5cyCvSlZG6~wUinwN6GouK3rKRyfJ9CgtU710eys55SEfp1p-KVhUD5e3mi-S4EcFr0cPF1U5~t45Qr6Lwn2qAzWd4ufgapkMGIvaLBUHcTiq1WjEkyBw38l6NC~8hACU3WwH9pijEoKmtsNvfDwWI2-7WEGNfJUHkxRhXztu6W8Wv35CGdLUdALmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenotypic analysis of the two inbred lines used to construct a genetic map and for mapping the Gsb-resistance gene. (A) Phenotypes of C. melo spp. conomon (HS) and C. melo spp. melo (XH) under normal conditions. (B) Phenotypes of the two inbred lines and their F1 generation after being inoculated with Didymella bryoniae." path-from-xml="dsx033f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481465/dsx033f1.tif" path-from-xml="dsx033f1.tif" target="_blank">View large</a><a section="112481465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f1.png?Expires=2147483647&Signature=qj39cu-d~9C8ktCxQ3w6dR7Hx70izTMHZA9bpo2tECZZhwYlQPSSS1G1SPgu4qr0k0SAH2HFwi09V3wfaL1Gqzq4SFi8SNiv2GLm34mhYzAUXp7CFMOyRMvAzCRIOtoP7-Kkmld5ylRyqQwzHJudCt2FVmUYYOEbGFHqh7fchYdJ06Sa8dFRv-RCg8ZgYGs2ZIbV8a~gXWocuDJNSxuD9GC7ofnY24Bqeg8gWcl2hPZeb6R9rtRnoEhTSWTf0FQqQmz7S0vHiuQXjko4rlwK0ASxK-qVHZK2ZVCZpLbWc6bPsSb2lAMO53a5XaszIr0hHIKBpcBua6s-iqVInXuZJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481465&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phenotypic analysis of the two inbred lines used to construct a genetic map and for mapping the Gsb-resistance gene. <strong>(A)</strong> Phenotypes of <em>C. melo</em> spp. <em>conomon</em> (HS) and <em>C. melo</em> spp. <em>melo</em> (XH) under normal conditions. <strong>(B)</strong> Phenotypes of the two inbred lines and their F<sub>1</sub> generation after being inoculated with <em>Didymella bryoniae</em>.</p></div></div>                            <h3 scrollto-destination=112481466 id="112481466" class="section-title" >3.2. Construction of a sequence-based ultra-dense <em>C. melo</em> genetic map</h3>
<p>In total, 12.94 and 13.14 Gb clean reads were generated for the XH and HS inbred lines (26× genome coverage), respectively. Meanwhile, 217.7 Gb clean reads were generated for the 150 F<sub>2</sub> population individuals (2× genome coverage), with more than 85% of the bases higher than Q30 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The well-distributed clean reads coverage on assembly scaffolds as well as the distribution of SNP mutation and SNP quality were indicative of the stochastic nature of resequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Figs S1 and S2</a></span>). The co-segregating SNPs were clustered in recombination bins (marked as Block), which were used to construct genetic linkage map. We constructed an ultra-dense genetic map consisting of 12 pseudo-chromosomes based on 12,932 recombination bin markers comprising 1,188,159 <em>C. melo</em> SNPs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>), which covered 1,818 cM, with an average distance of 0.17 cM between adjacent bin markers (<a reveal-id="dsx033-T2" data-open="dsx033-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). <div content-id="dsx033-T2" class="table-modal"><div class="table-wrap"><div id="dsx033-T2" data-id="dsx033-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Construction of an ultra-dense <em>C. melo</em> genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Total maker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap (cM) </th></tr></thead><tbody><tr><td>Chr1 </td><td>969 </td><td>167.29 </td><td>0.17 </td><td>2.402 </td></tr><tr><td>Chr2 </td><td>1,041 </td><td>148.75 </td><td>0.14 </td><td>3.74 </td></tr><tr><td>Chr3 </td><td>1,597 </td><td>179.22 </td><td>0.11 </td><td>8.07 </td></tr><tr><td>Chr4 </td><td>1,073 </td><td>226.08 </td><td>0.21 </td><td>12.461 </td></tr><tr><td>Chr5 </td><td>632 </td><td>79.69 </td><td>0.13 </td><td>7.933 </td></tr><tr><td>Chr6 </td><td>1,031 </td><td>164.23 </td><td>0.16 </td><td>4.209 </td></tr><tr><td>Chr7 </td><td>1,408 </td><td>170.29 </td><td>0.12 </td><td>2.736 </td></tr><tr><td>Chr8 </td><td>1,469 </td><td>122.15 </td><td>0.08 </td><td>3.138 </td></tr><tr><td>Chr9 </td><td>950 </td><td>153.95 </td><td>0.16 </td><td>2.402 </td></tr><tr><td>Chr10 </td><td>170 </td><td>79.69 </td><td>0.47 </td><td>20.059 </td></tr><tr><td>Chr11 </td><td>1,562 </td><td>169.69 </td><td>0.11 </td><td>3.807 </td></tr><tr><td>Chr12 </td><td>1,030 </td><td>157.15 </td><td>0.15 </td><td>3.071 </td></tr><tr><td>Total/average </td><td>12,932 </td><td>1818.18 </td><td>0.17 </td><td>20.059 </td></tr></tbody></table></div><div content-id="dsx033-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Total maker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap (cM) </th></tr></thead><tbody><tr><td>Chr1 </td><td>969 </td><td>167.29 </td><td>0.17 </td><td>2.402 </td></tr><tr><td>Chr2 </td><td>1,041 </td><td>148.75 </td><td>0.14 </td><td>3.74 </td></tr><tr><td>Chr3 </td><td>1,597 </td><td>179.22 </td><td>0.11 </td><td>8.07 </td></tr><tr><td>Chr4 </td><td>1,073 </td><td>226.08 </td><td>0.21 </td><td>12.461 </td></tr><tr><td>Chr5 </td><td>632 </td><td>79.69 </td><td>0.13 </td><td>7.933 </td></tr><tr><td>Chr6 </td><td>1,031 </td><td>164.23 </td><td>0.16 </td><td>4.209 </td></tr><tr><td>Chr7 </td><td>1,408 </td><td>170.29 </td><td>0.12 </td><td>2.736 </td></tr><tr><td>Chr8 </td><td>1,469 </td><td>122.15 </td><td>0.08 </td><td>3.138 </td></tr><tr><td>Chr9 </td><td>950 </td><td>153.95 </td><td>0.16 </td><td>2.402 </td></tr><tr><td>Chr10 </td><td>170 </td><td>79.69 </td><td>0.47 </td><td>20.059 </td></tr><tr><td>Chr11 </td><td>1,562 </td><td>169.69 </td><td>0.11 </td><td>3.807 </td></tr><tr><td>Chr12 </td><td>1,030 </td><td>157.15 </td><td>0.15 </td><td>3.071 </td></tr><tr><td>Total/average </td><td>12,932 </td><td>1818.18 </td><td>0.17 </td><td>20.059 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx033-T2" data-id="dsx033-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Construction of an ultra-dense <em>C. melo</em> genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Total maker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap (cM) </th></tr></thead><tbody><tr><td>Chr1 </td><td>969 </td><td>167.29 </td><td>0.17 </td><td>2.402 </td></tr><tr><td>Chr2 </td><td>1,041 </td><td>148.75 </td><td>0.14 </td><td>3.74 </td></tr><tr><td>Chr3 </td><td>1,597 </td><td>179.22 </td><td>0.11 </td><td>8.07 </td></tr><tr><td>Chr4 </td><td>1,073 </td><td>226.08 </td><td>0.21 </td><td>12.461 </td></tr><tr><td>Chr5 </td><td>632 </td><td>79.69 </td><td>0.13 </td><td>7.933 </td></tr><tr><td>Chr6 </td><td>1,031 </td><td>164.23 </td><td>0.16 </td><td>4.209 </td></tr><tr><td>Chr7 </td><td>1,408 </td><td>170.29 </td><td>0.12 </td><td>2.736 </td></tr><tr><td>Chr8 </td><td>1,469 </td><td>122.15 </td><td>0.08 </td><td>3.138 </td></tr><tr><td>Chr9 </td><td>950 </td><td>153.95 </td><td>0.16 </td><td>2.402 </td></tr><tr><td>Chr10 </td><td>170 </td><td>79.69 </td><td>0.47 </td><td>20.059 </td></tr><tr><td>Chr11 </td><td>1,562 </td><td>169.69 </td><td>0.11 </td><td>3.807 </td></tr><tr><td>Chr12 </td><td>1,030 </td><td>157.15 </td><td>0.15 </td><td>3.071 </td></tr><tr><td>Total/average </td><td>12,932 </td><td>1818.18 </td><td>0.17 </td><td>20.059 </td></tr></tbody></table></div><div content-id="dsx033-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Pseudo- chromosome </th><th>Total maker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap (cM) </th></tr></thead><tbody><tr><td>Chr1 </td><td>969 </td><td>167.29 </td><td>0.17 </td><td>2.402 </td></tr><tr><td>Chr2 </td><td>1,041 </td><td>148.75 </td><td>0.14 </td><td>3.74 </td></tr><tr><td>Chr3 </td><td>1,597 </td><td>179.22 </td><td>0.11 </td><td>8.07 </td></tr><tr><td>Chr4 </td><td>1,073 </td><td>226.08 </td><td>0.21 </td><td>12.461 </td></tr><tr><td>Chr5 </td><td>632 </td><td>79.69 </td><td>0.13 </td><td>7.933 </td></tr><tr><td>Chr6 </td><td>1,031 </td><td>164.23 </td><td>0.16 </td><td>4.209 </td></tr><tr><td>Chr7 </td><td>1,408 </td><td>170.29 </td><td>0.12 </td><td>2.736 </td></tr><tr><td>Chr8 </td><td>1,469 </td><td>122.15 </td><td>0.08 </td><td>3.138 </td></tr><tr><td>Chr9 </td><td>950 </td><td>153.95 </td><td>0.16 </td><td>2.402 </td></tr><tr><td>Chr10 </td><td>170 </td><td>79.69 </td><td>0.47 </td><td>20.059 </td></tr><tr><td>Chr11 </td><td>1,562 </td><td>169.69 </td><td>0.11 </td><td>3.807 </td></tr><tr><td>Chr12 </td><td>1,030 </td><td>157.15 </td><td>0.15 </td><td>3.071 </td></tr><tr><td>Total/average </td><td>12,932 </td><td>1818.18 </td><td>0.17 </td><td>20.059 </td></tr></tbody></table></div></div></p><p>Haplotype maps and linkage relationships heat maps were used to evaluate the quality of the genetic map. Haplotype maps, which indicate genotyping errors, were generated for each of the 150 lines in the F<sub>2</sub> population as well as the parental lines. These maps presented the recombination events (pseudo-chromosome 10 haplotype map in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> and all haplotype maps for the 12 pseudo-chromosomes in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). Almost all the recombination blocks were completely defined, with only pseudo-chromosome 7 missing 0.01% of its data. The linkage relationships heat maps, reflected the relationship between recombination markers in one pseudo-chromosome, and were used for ordering errors. These maps were created based on pair-wise recombination values for the 12,932 bin markers (pseudo-chromosome 10 linkage relationship heat map in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> and all linkage relationship heat maps for the 12 pseudo-chromosomes in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>). The constructed pseudo-chromosomes generally performed well according to the generated the heat maps. </p><p>We then compared our genetic map (HS × XH) with a published melon genetic map (PS × SC).<a reveal-id="dsx033-B31" data-open="dsx033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Of the 580 SNPs in the PS × SC F<sub>2</sub> genetic map, 558 were mapped to the HS × XH F<sub>2</sub> genetic map. Most of the pseudo-chromosomes were consistent with co-linearity (<a reveal-id="dsx033-F2" data-open="dsx033-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). We likewise compared the HS × XH F<sub>2</sub> genetic map with the melon genetic map compiled by the International Cucurbit Genomics Initiative.<a reveal-id="dsx033-B6" data-open="dsx033-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> This comparison uncovered similar co-linear relationships among pseudo-chromosomes. </p>                        <a id="112481470" scrollto-destination="112481470"></a>
<div id="dsx033-F2" data-id="dsx033-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f2.png?Expires=2147483647&Signature=IXuy6rW~QfF1Cjb-uXrtNOPFbnCcUNe66R19AueRRsuyww3a~1Be8tJT7~p97h6CNQ1yj0l2FTGR7k~ruJ6bcMdogkCejETsP7PqMee8GQwHLjV9blMNrztGu-E6YZwMXu4w~jGfDrzgzjvYUmHjarIKluYL0fBgH383PVcfTUxh-KZGziz~CDRBUNqcQ8Uc09FnPMS-rmf2UV93ywfoHd~SQ8jgASviYWPagwiyLf~91F-gu8sAc-7~WaS2O~b8k-rzRH-9IP~u64ZK-4ijy39wgJBvWQi3GyXnqPN6-FxUCv6yBKG1WtkarJBW5UYr1GAlDXaJnkWXcKdxD-9IfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Constructed pseudo-chromosome (Chr9) and the co-linearity between the genetic map constructed from the HS × XH population in this study and a published genetic map for the PS × SC-9 population." path-from-xml="dsx033f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481470/dsx033f2.tif" path-from-xml="dsx033f2.tif" target="_blank">View large</a><a section="112481470" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f2.png?Expires=2147483647&Signature=EkRHZSmkTc12v1OAEH0vZRsDFK36MLM34DBNJtJtfFdx6YewwXZ3wzIbM3RGexopPaMIVM1L2Bqe2lnwUiRyi9xM5zDoP2bZv7rxWJ7X8X9n5fAElI-6k0CPffsMg8jI5kXudasIokwZFI3azN5R9mmTW8J-~v4zRnRXttl5s4hxj0eaUv9Mfzmvq7yjga7CTReCk1lGBbdnOck26WFmOovjJQ7dWcegt9M81nl~mCc3FbmghkiIWQxuip7jbTNO7ddrGdzX0JI8KLxmu8YQEgzha1S-wjHfgLUi9mTncF2qlOAcWwuj6kvPKivFEHtvc8Ncp0kIqxvFetrkHyE1oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481470&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Constructed pseudo-chromosome (Chr9) and the co-linearity between the genetic map constructed from the HS × XH population in this study and a published genetic map for the PS × SC-9 population.</p></div></div><div content-id="dsx033-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f2.png?Expires=2147483647&Signature=IXuy6rW~QfF1Cjb-uXrtNOPFbnCcUNe66R19AueRRsuyww3a~1Be8tJT7~p97h6CNQ1yj0l2FTGR7k~ruJ6bcMdogkCejETsP7PqMee8GQwHLjV9blMNrztGu-E6YZwMXu4w~jGfDrzgzjvYUmHjarIKluYL0fBgH383PVcfTUxh-KZGziz~CDRBUNqcQ8Uc09FnPMS-rmf2UV93ywfoHd~SQ8jgASviYWPagwiyLf~91F-gu8sAc-7~WaS2O~b8k-rzRH-9IP~u64ZK-4ijy39wgJBvWQi3GyXnqPN6-FxUCv6yBKG1WtkarJBW5UYr1GAlDXaJnkWXcKdxD-9IfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Constructed pseudo-chromosome (Chr9) and the co-linearity between the genetic map constructed from the HS × XH population in this study and a published genetic map for the PS × SC-9 population." path-from-xml="dsx033f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481470/dsx033f2.tif" path-from-xml="dsx033f2.tif" target="_blank">View large</a><a section="112481470" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f2.png?Expires=2147483647&Signature=EkRHZSmkTc12v1OAEH0vZRsDFK36MLM34DBNJtJtfFdx6YewwXZ3wzIbM3RGexopPaMIVM1L2Bqe2lnwUiRyi9xM5zDoP2bZv7rxWJ7X8X9n5fAElI-6k0CPffsMg8jI5kXudasIokwZFI3azN5R9mmTW8J-~v4zRnRXttl5s4hxj0eaUv9Mfzmvq7yjga7CTReCk1lGBbdnOck26WFmOovjJQ7dWcegt9M81nl~mCc3FbmghkiIWQxuip7jbTNO7ddrGdzX0JI8KLxmu8YQEgzha1S-wjHfgLUi9mTncF2qlOAcWwuj6kvPKivFEHtvc8Ncp0kIqxvFetrkHyE1oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481470&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Constructed pseudo-chromosome (Chr9) and the co-linearity between the genetic map constructed from the HS × XH population in this study and a published genetic map for the PS × SC-9 population.</p></div></div>                            <h3 scrollto-destination=112481471 id="112481471" class="section-title" >3.3. Genome re-anchoring</h3>
<p>The HS × XH F<sub>2</sub> genetic map was used to re-anchor the genome scaffolds to the 12 pseudo-chromosomes. We obtained all assembled scaffolds from the melon reference genome.<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Based on the genetic map, we anchored 92% of the scaffolds assembly (344.9 Mb of 374.8 Mb; 153 scaffolds) onto the 12 pseudo-chromosomes (<a reveal-id="dsx033-F3" data-open="dsx033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). We determined the orientation of 97.75% (336 Mb) of the anchored scaffolds, and detected chimeric scaffolds, with each anchored to different pseudo-chromosomes in the genome. For example, in scaffold NW_007546289.1, 264 Blocks were mapped onto pseudo-chromosome 5, and 72 Blocks were anchored onto pseudo-chromosome 8. In addition, in scaffold NW_007546312.1, 79 Blocks were mapped to pseudo-chromosome 8, and 41 Blocks were anchored to pseudo-chromosome 11. These results indicate that this ultra-dense genetic map may be used to improve the published melon genome assembly. </p>                        <a id="112481473" scrollto-destination="112481473"></a>
<div id="dsx033-F3" data-id="dsx033-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f3.png?Expires=2147483647&Signature=moIXKFOflyDXKsaQG6YhQO675pImgm81B9ZJtF8BuUdfk~sxPn2jQ~-VPs8ID6sqiMjBGVb0G4M1bNdTkgpqZvHE7tJhA~0bRpvDuAEiTACMxDh2gA8dQz6Q5c3euUkFnH2BmRhrmVQum0-b65w-9oro13dbp0L9JJreNLuZBOT40qDyw40vuhNbBKgaZWR2RYbMydyFKEcYdyAk1d8kVY1TGUZ4hAnNB~2noDOl7WTTe4V9Z0aGS7UkPOPMsV~VglfqzKocEGqGDls~cgbPp0yPASBC~jjNApwflSBuRceCvU1W3R1prOttaeyjhf5brGZmHFQV8EGLuxr-B0FWvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Re-anchored scaffolds based on the ultra-dense genetic map constructed in this study. Yellow columns represent the 12 pseudo-chromosomes of melon. Bin markers are located according to genetic distance (cM). Scaffolds were displayed in blue and grey lines represent corresponding genetic markers between each pseudo-chromosome and scaffolds." path-from-xml="dsx033f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481473/dsx033f3.tif" path-from-xml="dsx033f3.tif" target="_blank">View large</a><a section="112481473" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f3.png?Expires=2147483647&Signature=N4ar8yDpJnlKgWQfRADoc0wLKpfi4K7r9xElARsqhq0WiUHtbPzzBiDtmYGE5sDmDD7LHi-nNqVDIA~R9LZR7xkZjZw8f4rphfkraeuYeZ2qgL7yNKr8fwaAajAeKuraxYUSLoWhOppY9GlcVnh44kEYUoXYNakZhq9FdXC37yISWZuhOhyXPsBTYiAFENfGfpS5bRObSDUOw0nsEIuVueh3JtQvpaWF4Ax1w4yO5KAZ8ykTmNQ5Eq-EI17Tvr01VxoeCGwoze2oZO1mZrnLzS19yp95o9SEH4jnDRH7zinQgOdc46tsOxeoUw81qFKIoLiQGkqniew4XF-pQlCnVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481473&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Re-anchored scaffolds based on the ultra-dense genetic map constructed in this study. Yellow columns represent the 12 pseudo-chromosomes of melon. Bin markers are located according to genetic distance (cM). Scaffolds were displayed in blue and grey lines represent corresponding genetic markers between each pseudo-chromosome and scaffolds.</p></div></div><div content-id="dsx033-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f3.png?Expires=2147483647&Signature=moIXKFOflyDXKsaQG6YhQO675pImgm81B9ZJtF8BuUdfk~sxPn2jQ~-VPs8ID6sqiMjBGVb0G4M1bNdTkgpqZvHE7tJhA~0bRpvDuAEiTACMxDh2gA8dQz6Q5c3euUkFnH2BmRhrmVQum0-b65w-9oro13dbp0L9JJreNLuZBOT40qDyw40vuhNbBKgaZWR2RYbMydyFKEcYdyAk1d8kVY1TGUZ4hAnNB~2noDOl7WTTe4V9Z0aGS7UkPOPMsV~VglfqzKocEGqGDls~cgbPp0yPASBC~jjNApwflSBuRceCvU1W3R1prOttaeyjhf5brGZmHFQV8EGLuxr-B0FWvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Re-anchored scaffolds based on the ultra-dense genetic map constructed in this study. Yellow columns represent the 12 pseudo-chromosomes of melon. Bin markers are located according to genetic distance (cM). Scaffolds were displayed in blue and grey lines represent corresponding genetic markers between each pseudo-chromosome and scaffolds." path-from-xml="dsx033f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481473/dsx033f3.tif" path-from-xml="dsx033f3.tif" target="_blank">View large</a><a section="112481473" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f3.png?Expires=2147483647&Signature=N4ar8yDpJnlKgWQfRADoc0wLKpfi4K7r9xElARsqhq0WiUHtbPzzBiDtmYGE5sDmDD7LHi-nNqVDIA~R9LZR7xkZjZw8f4rphfkraeuYeZ2qgL7yNKr8fwaAajAeKuraxYUSLoWhOppY9GlcVnh44kEYUoXYNakZhq9FdXC37yISWZuhOhyXPsBTYiAFENfGfpS5bRObSDUOw0nsEIuVueh3JtQvpaWF4Ax1w4yO5KAZ8ykTmNQ5Eq-EI17Tvr01VxoeCGwoze2oZO1mZrnLzS19yp95o9SEH4jnDRH7zinQgOdc46tsOxeoUw81qFKIoLiQGkqniew4XF-pQlCnVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481473&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Re-anchored scaffolds based on the ultra-dense genetic map constructed in this study. Yellow columns represent the 12 pseudo-chromosomes of melon. Bin markers are located according to genetic distance (cM). Scaffolds were displayed in blue and grey lines represent corresponding genetic markers between each pseudo-chromosome and scaffolds.</p></div></div>                            <h3 scrollto-destination=112481474 id="112481474" class="section-title" >3.4. Analysis of RHs</h3>
<p>We identified 1,135 RHs, which were unequally distributed to all 12 pseudo-chromosomes. pseudo-chromosome 9 had the most RHs (153), while pseudo-chromosome 10 had the fewest (30) (<a reveal-id="dsx033-F4" data-open="dsx033-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Most RHs were located at the telomeres of pseudo-chromosome, which suggested the pericentromeric regions lacked recombination events (<a reveal-id="dsx033-F4" data-open="dsx033-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The fact that almost no RHs were detected on one arm of pseudo-chromosome 10, may have been because very few markers were distributed in this region (<a reveal-id="dsx033-F4" data-open="dsx033-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), or might be related to the acrocentric morphology of this chromosome.<a reveal-id="dsx033-B31" data-open="dsx033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                        <a id="112481476" scrollto-destination="112481476"></a>
<div id="dsx033-F4" data-id="dsx033-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f4.png?Expires=2147483647&Signature=sQpGJ9f7gjPO2qHL8mhXPy0qXt5ILbYQoYlqXnfuDLqpz55a2tUxN7ipja~ibGEG0YfAovXHy3XW1S2EEyuDtdnaK3MVBxst44HFRgrZCGehYndWOARS54ZlDFoYAzlT35Q8x7uIR9jYM3krY3dUnS2zYiKPxKmL5oNMRx6mYYUa5o-YnxqRaeZ9zVr9BmruLHXuqsJkPM8WGpoLu2JgM~luTxsW4ZzvnEQQNd27bz50pI-Hg8vzv-CCF7PUYQkJuIieQTxV~i7IudSDoDUYrDjwtZEduhqWEyvPZb5zj6ax1u9kw7sJS7Q2ntnTcwDLHgtABOCbtvOjUYTDtIO12w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic positions of the recombination hotspots in the 12 melon pseudo-chromosome." path-from-xml="dsx033f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481476/dsx033f4.tif" path-from-xml="dsx033f4.tif" target="_blank">View large</a><a section="112481476" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f4.png?Expires=2147483647&Signature=kDU2THAj1hiwC4x0K2L3IZ5jAQxJMLLXDrWPfwbvbI3iy7NF2S7x5VVnqaWunNtUJtQWlcfwyH4esxr478sU-xD702ePjwGWKQxD~ps5gT1Jk8p4HgWnSmEijxpjhL9Ph1JC0l1cPXdzCrB0hYveT-9JRtnjyiXWek9p3MEMotgc5o8LQi-fqUB6F8h~p65iVlQfgDr4pHBZZnqJ3cQ6NHyz~KIcJNxb5rwlsvKLFOGfl-gBKan2D0kYQjjbrNvbL0N2N5xNb30nlRcOY5WG5B9GEO7D5rx1lNgeKc~oKzK9Z-0bh9u0apAz4KqcAZofrpcJCtnLArhVjAYKmcPQkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481476&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic positions of the recombination hotspots in the 12 melon pseudo-chromosome.</p></div></div><div content-id="dsx033-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f4.png?Expires=2147483647&Signature=sQpGJ9f7gjPO2qHL8mhXPy0qXt5ILbYQoYlqXnfuDLqpz55a2tUxN7ipja~ibGEG0YfAovXHy3XW1S2EEyuDtdnaK3MVBxst44HFRgrZCGehYndWOARS54ZlDFoYAzlT35Q8x7uIR9jYM3krY3dUnS2zYiKPxKmL5oNMRx6mYYUa5o-YnxqRaeZ9zVr9BmruLHXuqsJkPM8WGpoLu2JgM~luTxsW4ZzvnEQQNd27bz50pI-Hg8vzv-CCF7PUYQkJuIieQTxV~i7IudSDoDUYrDjwtZEduhqWEyvPZb5zj6ax1u9kw7sJS7Q2ntnTcwDLHgtABOCbtvOjUYTDtIO12w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic positions of the recombination hotspots in the 12 melon pseudo-chromosome." path-from-xml="dsx033f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481476/dsx033f4.tif" path-from-xml="dsx033f4.tif" target="_blank">View large</a><a section="112481476" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f4.png?Expires=2147483647&Signature=kDU2THAj1hiwC4x0K2L3IZ5jAQxJMLLXDrWPfwbvbI3iy7NF2S7x5VVnqaWunNtUJtQWlcfwyH4esxr478sU-xD702ePjwGWKQxD~ps5gT1Jk8p4HgWnSmEijxpjhL9Ph1JC0l1cPXdzCrB0hYveT-9JRtnjyiXWek9p3MEMotgc5o8LQi-fqUB6F8h~p65iVlQfgDr4pHBZZnqJ3cQ6NHyz~KIcJNxb5rwlsvKLFOGfl-gBKan2D0kYQjjbrNvbL0N2N5xNb30nlRcOY5WG5B9GEO7D5rx1lNgeKc~oKzK9Z-0bh9u0apAz4KqcAZofrpcJCtnLArhVjAYKmcPQkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481476&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic positions of the recombination hotspots in the 12 melon pseudo-chromosome.</p></div></div>                            <h3 scrollto-destination=112481477 id="112481477" class="section-title" >3.5. Comparison of the <em>Cucurbitaceae</em> pseudo-chromosomes</h3>
<p>A comparison between our genetic map of melon and the genetic maps of other <em>Cucurbitaceae</em> species (i.e., watermelon and cucumber) revealed a relatively weak syntenic relationship between the genomes (<a reveal-id="dsx033-F5" data-open="dsx033-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S6</a></span>). In a comparison with cucumber chromosomes, Spearman’s rank-order correlation coefficients for 12 melon pseudo-chromosomes varied from 0.63 to 0.07. Additionally, in a comparison with watermelon chromosomes, Spearman’s rank-order correlation coefficients for 12 melon pseudo-chromosomes ranged from 0.67 to 0.01 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Of the 12 melon pseudo-chromosomes, pseudo-chromosome 10 exhibited a closer syntenic relationship with watermelon Chr 5 and cucumber Chr 5 than the other pseudo-chromosomes (<a reveal-id="dsx033-F5" data-open="dsx033-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). These results imply that the genomic structures of pseudo-chromosome 10 from melon, Chr 5 from watermelon and cucumber are relatively conserved. </p>                        <a id="112481479" scrollto-destination="112481479"></a>
<div id="dsx033-F5" data-id="dsx033-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f5.png?Expires=2147483647&Signature=EzkRFncbEMo4r3Tdlrbzw5-f5Jj4SCV-czlPbRcHVFq03XKRBX9RfI4WzTune2dY7AptvqQHPv5FvsuB~ZpdhjggSxlHar7-oDSYD7GAc2JO5Ulyrol3vEC4DVxZc1la8jKflkW--41ZTLKdZ1rK-plX9CqLQiTwaxuoYTjyntmmi9BdPMn0oBngAecvtoNEXsXIwzHufivVyYa3RxirwJ9~lVKR-g9dM7kNV1VD5dHUayIDDV1g3Sw-mQL62--WdQGGPEp8wTUehqGKaH5dHBCFhWwJtrQAvOIhB8WWYq9wpZv-1SpjQxTIgYYhCCktgJNFw3~Rf29XLBIydO6LoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the syntenic relationships among the pseudo-chromosomes of C. melo, and chromosomes of C. sativus and C. lanatus. (A) Syntenic relationships between C. melo and C. sativus. (B) Syntenic relationships between C. melo and C. lanatus." path-from-xml="dsx033f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481479/dsx033f5.tif" path-from-xml="dsx033f5.tif" target="_blank">View large</a><a section="112481479" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f5.png?Expires=2147483647&Signature=vY478luo0ydd8H36tPXyJUmYWWlMwLzMyS5N2KaVFeJajE8XZeqKCJ4uFUfk6t5JZhhDzJIiaEPEpHxMQS2dAtOiF0sXZT95sYO3vfkjVdQ9xf-O9pc2hqywhFZYxix5fOcBZ5WY6rWAWoDNsh1Fttuq3HTTlOWSf9WwI-13QN4g~5gkgWKYz2ifCUEVLU9~KuYu-gqMKJ8jds-wQKI6usB~MFwZmdinMUgamqaZpKIB~PX9Tv38RfuSvu8Qm1nd~~5sA3U9WSruwv~AxfGtrn43qbi0m~aa3eAXZqGJ6tlK9YdEUgFDMf1skayiXl5jmsDzCcKRW6Sk-yabpKkLzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481479&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the syntenic relationships among the pseudo-chromosomes of <em>C. melo</em>, and chromosomes of <em>C. sativus</em> and <em>C. lanatus</em>. (<strong>A</strong>) Syntenic relationships between <em>C. melo</em> and <em>C. sativus</em>. (<strong>B</strong>) Syntenic relationships between <em>C. melo</em> and <em>C. lanatus</em>.</p></div></div><div content-id="dsx033-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f5.png?Expires=2147483647&Signature=EzkRFncbEMo4r3Tdlrbzw5-f5Jj4SCV-czlPbRcHVFq03XKRBX9RfI4WzTune2dY7AptvqQHPv5FvsuB~ZpdhjggSxlHar7-oDSYD7GAc2JO5Ulyrol3vEC4DVxZc1la8jKflkW--41ZTLKdZ1rK-plX9CqLQiTwaxuoYTjyntmmi9BdPMn0oBngAecvtoNEXsXIwzHufivVyYa3RxirwJ9~lVKR-g9dM7kNV1VD5dHUayIDDV1g3Sw-mQL62--WdQGGPEp8wTUehqGKaH5dHBCFhWwJtrQAvOIhB8WWYq9wpZv-1SpjQxTIgYYhCCktgJNFw3~Rf29XLBIydO6LoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the syntenic relationships among the pseudo-chromosomes of C. melo, and chromosomes of C. sativus and C. lanatus. (A) Syntenic relationships between C. melo and C. sativus. (B) Syntenic relationships between C. melo and C. lanatus." path-from-xml="dsx033f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481479/dsx033f5.tif" path-from-xml="dsx033f5.tif" target="_blank">View large</a><a section="112481479" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f5.png?Expires=2147483647&Signature=vY478luo0ydd8H36tPXyJUmYWWlMwLzMyS5N2KaVFeJajE8XZeqKCJ4uFUfk6t5JZhhDzJIiaEPEpHxMQS2dAtOiF0sXZT95sYO3vfkjVdQ9xf-O9pc2hqywhFZYxix5fOcBZ5WY6rWAWoDNsh1Fttuq3HTTlOWSf9WwI-13QN4g~5gkgWKYz2ifCUEVLU9~KuYu-gqMKJ8jds-wQKI6usB~MFwZmdinMUgamqaZpKIB~PX9Tv38RfuSvu8Qm1nd~~5sA3U9WSruwv~AxfGtrn43qbi0m~aa3eAXZqGJ6tlK9YdEUgFDMf1skayiXl5jmsDzCcKRW6Sk-yabpKkLzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481479&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the syntenic relationships among the pseudo-chromosomes of <em>C. melo</em>, and chromosomes of <em>C. sativus</em> and <em>C. lanatus</em>. (<strong>A</strong>) Syntenic relationships between <em>C. melo</em> and <em>C. sativus</em>. (<strong>B</strong>) Syntenic relationships between <em>C. melo</em> and <em>C. lanatus</em>.</p></div></div>                            <h3 scrollto-destination=112481480 id="112481480" class="section-title" >3.6. Mapping of <em>C. melo</em> Gsb-resistance gene candidates</h3>
<p>The 12 pseudo-chromosomes as well as the genotyping and phenotyping data related to Gsb resistance were used to map the candidate Gsb-resistance gene (<em>GsbR</em>). We identified a 0.667-cM region on pseudo-chromosome 4 with two bin markers (Block13188 and Block13179) that satisfied the LOD threshold of 2.5, implying the region is linked to Gsb resistance. The LOD values of Block13188 and Block13179 were 2.528 and 2.737, respectively, while the contribution rates were 9.36 and 10.72, respectively (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The additive effects and dominant effects were 0.177 and 0.12, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Furthermore, a scaffold (CM3.5_scaffold00018) with 71 well-distributed SNPs was identified during the genotyping of the HS (Gsb resistant) and XH (Gsb susceptible) parent lines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Eight candidate genes were annotated in this region based on the melon reference genome, GO, NR, Swiss-Prot, COG, and KEGG databases (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>, <a reveal-id="dsx033-T3" data-open="dsx033-T3" class="link link-reveal link-table xref-fig">Table 3</a>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). <div content-id="dsx033-T3" class="table-modal"><div class="table-wrap"><div id="dsx033-T3" data-id="dsx033-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Annotation of Gsb<em>-</em>resistance candidate genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Location </th><th>Direction </th><th>Annotation </th><th>Expression </th></tr></thead><tbody><tr><td><em>MELO3C012986</em> </td><td>3934220–3943507 </td><td>+ </td><td>Similar to VHS domain-containing protein </td><td>Expressed </td></tr><tr><td><em>MELO3C012987</em> </td><td>3957629–3958543 </td><td>− </td><td>Similar to uncharacterized <em>Avr9/Cf-9</em> rapidly elicited protein 146 </td><td>Expressed </td></tr><tr><td><em>MELO3C012988</em> </td><td>3960154–3961860 </td><td>− </td><td>Similar to putative ribonuclease H protein </td><td>ND </td></tr><tr><td><em>MELO3C012989</em> </td><td>3962492–3963050 </td><td>− </td><td><em>LINE-1</em> retrotransposable element ORF2 protein </td><td>ND </td></tr><tr><td><em>MELO3C012990</em> </td><td>3963801–3965186 </td><td>− </td><td>uncharacterized protein </td><td>ND </td></tr><tr><td><em>MELO3C012991</em> </td><td>3967157–3970154 </td><td>+ </td><td>Similar to Protein <em>SEC13</em> homolog </td><td>Expressed </td></tr><tr><td><em>MELO3C012992</em> </td><td>3979703–3980781 </td><td>+ </td><td>Similar to GATA transcription factor 9 </td><td>Expressed </td></tr><tr><td><em>MELO3C012993</em> </td><td>4015998–4027273 </td><td>+ </td><td>Similar to Biotin–protein ligase </td><td>Expressed </td></tr></tbody></table></div><div content-id="dsx033-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Location </th><th>Direction </th><th>Annotation </th><th>Expression </th></tr></thead><tbody><tr><td><em>MELO3C012986</em> </td><td>3934220–3943507 </td><td>+ </td><td>Similar to VHS domain-containing protein </td><td>Expressed </td></tr><tr><td><em>MELO3C012987</em> </td><td>3957629–3958543 </td><td>− </td><td>Similar to uncharacterized <em>Avr9/Cf-9</em> rapidly elicited protein 146 </td><td>Expressed </td></tr><tr><td><em>MELO3C012988</em> </td><td>3960154–3961860 </td><td>− </td><td>Similar to putative ribonuclease H protein </td><td>ND </td></tr><tr><td><em>MELO3C012989</em> </td><td>3962492–3963050 </td><td>− </td><td><em>LINE-1</em> retrotransposable element ORF2 protein </td><td>ND </td></tr><tr><td><em>MELO3C012990</em> </td><td>3963801–3965186 </td><td>− </td><td>uncharacterized protein </td><td>ND </td></tr><tr><td><em>MELO3C012991</em> </td><td>3967157–3970154 </td><td>+ </td><td>Similar to Protein <em>SEC13</em> homolog </td><td>Expressed </td></tr><tr><td><em>MELO3C012992</em> </td><td>3979703–3980781 </td><td>+ </td><td>Similar to GATA transcription factor 9 </td><td>Expressed </td></tr><tr><td><em>MELO3C012993</em> </td><td>4015998–4027273 </td><td>+ </td><td>Similar to Biotin–protein ligase </td><td>Expressed </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx033-T3" data-id="dsx033-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Annotation of Gsb<em>-</em>resistance candidate genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Location </th><th>Direction </th><th>Annotation </th><th>Expression </th></tr></thead><tbody><tr><td><em>MELO3C012986</em> </td><td>3934220–3943507 </td><td>+ </td><td>Similar to VHS domain-containing protein </td><td>Expressed </td></tr><tr><td><em>MELO3C012987</em> </td><td>3957629–3958543 </td><td>− </td><td>Similar to uncharacterized <em>Avr9/Cf-9</em> rapidly elicited protein 146 </td><td>Expressed </td></tr><tr><td><em>MELO3C012988</em> </td><td>3960154–3961860 </td><td>− </td><td>Similar to putative ribonuclease H protein </td><td>ND </td></tr><tr><td><em>MELO3C012989</em> </td><td>3962492–3963050 </td><td>− </td><td><em>LINE-1</em> retrotransposable element ORF2 protein </td><td>ND </td></tr><tr><td><em>MELO3C012990</em> </td><td>3963801–3965186 </td><td>− </td><td>uncharacterized protein </td><td>ND </td></tr><tr><td><em>MELO3C012991</em> </td><td>3967157–3970154 </td><td>+ </td><td>Similar to Protein <em>SEC13</em> homolog </td><td>Expressed </td></tr><tr><td><em>MELO3C012992</em> </td><td>3979703–3980781 </td><td>+ </td><td>Similar to GATA transcription factor 9 </td><td>Expressed </td></tr><tr><td><em>MELO3C012993</em> </td><td>4015998–4027273 </td><td>+ </td><td>Similar to Biotin–protein ligase </td><td>Expressed </td></tr></tbody></table></div><div content-id="dsx033-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Location </th><th>Direction </th><th>Annotation </th><th>Expression </th></tr></thead><tbody><tr><td><em>MELO3C012986</em> </td><td>3934220–3943507 </td><td>+ </td><td>Similar to VHS domain-containing protein </td><td>Expressed </td></tr><tr><td><em>MELO3C012987</em> </td><td>3957629–3958543 </td><td>− </td><td>Similar to uncharacterized <em>Avr9/Cf-9</em> rapidly elicited protein 146 </td><td>Expressed </td></tr><tr><td><em>MELO3C012988</em> </td><td>3960154–3961860 </td><td>− </td><td>Similar to putative ribonuclease H protein </td><td>ND </td></tr><tr><td><em>MELO3C012989</em> </td><td>3962492–3963050 </td><td>− </td><td><em>LINE-1</em> retrotransposable element ORF2 protein </td><td>ND </td></tr><tr><td><em>MELO3C012990</em> </td><td>3963801–3965186 </td><td>− </td><td>uncharacterized protein </td><td>ND </td></tr><tr><td><em>MELO3C012991</em> </td><td>3967157–3970154 </td><td>+ </td><td>Similar to Protein <em>SEC13</em> homolog </td><td>Expressed </td></tr><tr><td><em>MELO3C012992</em> </td><td>3979703–3980781 </td><td>+ </td><td>Similar to GATA transcription factor 9 </td><td>Expressed </td></tr><tr><td><em>MELO3C012993</em> </td><td>4015998–4027273 </td><td>+ </td><td>Similar to Biotin–protein ligase </td><td>Expressed </td></tr></tbody></table></div></div></p>                        <a id="112481482" scrollto-destination="112481482"></a>
<div id="dsx033-F6" data-id="dsx033-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f6.png?Expires=2147483647&Signature=ybDTdrDQkQ95El6wwL~58WQGRaizUvcIVNjvcdR3ELLCZjyhS5xaBGQvrkE6I21UbrlqgOwQm2ZpXeNtEB8-vYKdU7yuzlH6I7W6EWOHKV5RLpTpS5BMv4MO8kFs2s8x7i6nbz45vRdpYim8eQZDXg-2DHoa~XAJRJdKovtzn4R2hh7kYMm9nplCUK4EqH7B7Uq1sM4FSWXigxj-AXl--C9M66Z2Tb2AFVp3JMEl55mc8BtJJp~e4SaKPjoERz~Vrlx-lJ741YCET98n7B5CCWaW5rM9kkkNvNt8Iokz10jDKbbed9OXSDX2h2qjos-JcuSZFQ2iRtcs9xEkHeHKYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mapping of C. melo Gsb-resistance candidate genes. (A) Mapping of Gsb-resistance candidate genes. (B) Analysis of Gsb-resistance candidate gene expression levels in plants inoculated with Didymella bryoniae. (C) DUF761 domain in MEL03C012987 gene. (D) A non-synonymous SNP in the MELO03C012987 gene. (E) Genotyping of the non-synonymous substitution of MEL03C012987 in resistant and susceptible lines of C. melo." path-from-xml="dsx033f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481482/dsx033f6.tif" path-from-xml="dsx033f6.tif" target="_blank">View large</a><a section="112481482" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f6.png?Expires=2147483647&Signature=3N~LBlNbcgXW-zaWdx0W72YlVrLRRCyWpCM7wrYxATgLzKtfDMJlvckVscGN1jQmH0ZNprMEEOIQgXPvGd7TE0pxTMPJGffuMtu4MS7JdVv10zU53ivPNpnOwlrYUUkIxhMEaKhpCqr6kpRtO9UKgm60Md9Eaam1twi8DHSR5XAuO6VMUZx8NVvG4mVyQLbisyrDtJVt0qgmHw0hlPqm2JqWg4h9ZjBLDj4B0f8ZSeVE~OZnmA72fRKd4b3Bhv7vCEIJSZqDPfcYbFIjx10SnQot14Lxp-uTlndqHBJD2h~X2HPI8GQnOA3L7bunwIHA85u8Iyxl5LHbQidrGXwXUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481482&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mapping of <em>C. melo</em> Gsb-resistance candidate genes. (<strong>A</strong>) Mapping of Gsb-resistance candidate genes. (<strong>B</strong>) Analysis of Gsb-resistance candidate gene expression levels in plants inoculated with <em>Didymella bryoniae</em>. (<strong>C</strong>) DUF761 domain in <em>MEL03C012987</em> gene. (<strong>D</strong>) A non-synonymous SNP in the <em>MELO03C012987</em> gene. (<strong>E</strong>) Genotyping of the non-synonymous substitution of <em>MEL03C012987</em> in resistant and susceptible lines of <em>C. melo</em>.</p></div></div><div content-id="dsx033-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/m_dsx033f6.png?Expires=2147483647&Signature=ybDTdrDQkQ95El6wwL~58WQGRaizUvcIVNjvcdR3ELLCZjyhS5xaBGQvrkE6I21UbrlqgOwQm2ZpXeNtEB8-vYKdU7yuzlH6I7W6EWOHKV5RLpTpS5BMv4MO8kFs2s8x7i6nbz45vRdpYim8eQZDXg-2DHoa~XAJRJdKovtzn4R2hh7kYMm9nplCUK4EqH7B7Uq1sM4FSWXigxj-AXl--C9M66Z2Tb2AFVp3JMEl55mc8BtJJp~e4SaKPjoERz~Vrlx-lJ741YCET98n7B5CCWaW5rM9kkkNvNt8Iokz10jDKbbed9OXSDX2h2qjos-JcuSZFQ2iRtcs9xEkHeHKYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mapping of C. melo Gsb-resistance candidate genes. (A) Mapping of Gsb-resistance candidate genes. (B) Analysis of Gsb-resistance candidate gene expression levels in plants inoculated with Didymella bryoniae. (C) DUF761 domain in MEL03C012987 gene. (D) A non-synonymous SNP in the MELO03C012987 gene. (E) Genotyping of the non-synonymous substitution of MEL03C012987 in resistant and susceptible lines of C. melo." path-from-xml="dsx033f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481482/dsx033f6.tif" path-from-xml="dsx033f6.tif" target="_blank">View large</a><a section="112481482" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033f6.png?Expires=2147483647&Signature=3N~LBlNbcgXW-zaWdx0W72YlVrLRRCyWpCM7wrYxATgLzKtfDMJlvckVscGN1jQmH0ZNprMEEOIQgXPvGd7TE0pxTMPJGffuMtu4MS7JdVv10zU53ivPNpnOwlrYUUkIxhMEaKhpCqr6kpRtO9UKgm60Md9Eaam1twi8DHSR5XAuO6VMUZx8NVvG4mVyQLbisyrDtJVt0qgmHw0hlPqm2JqWg4h9ZjBLDj4B0f8ZSeVE~OZnmA72fRKd4b3Bhv7vCEIJSZqDPfcYbFIjx10SnQot14Lxp-uTlndqHBJD2h~X2HPI8GQnOA3L7bunwIHA85u8Iyxl5LHbQidrGXwXUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481482&ar=4107259&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx033f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mapping of <em>C. melo</em> Gsb-resistance candidate genes. (<strong>A</strong>) Mapping of Gsb-resistance candidate genes. (<strong>B</strong>) Analysis of Gsb-resistance candidate gene expression levels in plants inoculated with <em>Didymella bryoniae</em>. (<strong>C</strong>) DUF761 domain in <em>MEL03C012987</em> gene. (<strong>D</strong>) A non-synonymous SNP in the <em>MELO03C012987</em> gene. (<strong>E</strong>) Genotyping of the non-synonymous substitution of <em>MEL03C012987</em> in resistant and susceptible lines of <em>C. melo</em>.</p></div></div><p>We analysed the expression patterns of these candidate genes in HS and XH plants inoculated with <em>D. bryoniae</em>. We were unable to detect the expression of <em>MELO3C012988</em>, <em>MELO3C012989</em>, and <em>MELO3C012990</em> in control and inoculated plants, implying these genes may not directly influence Gsb-resistance. In contrast, <em>MELO03C012987</em> expression was considerably upregulated in HS and XH plants inoculated with <em>D. bryoniae</em> (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). The expression levels of the remaining candidate genes (i.e., <em>MELO3C012986</em>, <em>MELO3C012991</em>, <em>MELO3C012992</em>, and <em>MELO3C012993</em>) were not induced by <em>D. bryoniae</em> inoculation (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). These results indicated that <em>MELO03C012987</em> is most likely the gene responsible for Gsb-resistance in melon. A Pfam BLAST analysis indicated that MELO03C012987 includes a DUF761 domain (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>). We then compared the mapped region between the HS and XH lines. A non-synonymous SNP was identified in the <em>MELO03C012987</em> gene, which resulted in a Lysine being replaced by a glutamic acid in the encoded protein (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). Genotyping <em>MELO03C012987</em> from six resistant and five susceptible inbred lines of <em>C. melo</em> revealed that all five susceptible lines (RM, MW, DZX, M14086, and M15019) exhibited Adenine at the non-synonymous SNP, which is same as that found in XH plant. Of the six resistant lines genotyped, five (HB6, HB11, HB13, CNZ, and QDH) had the Guanine at this non-synonymous substitution as that in HS plant. Whereas, HP9818 exhibited either Guanine or Adenine, representing a heterozygous genotype (<a reveal-id="dsx033-F6" data-open="dsx033-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>).</p>                            <h2 scrollto-destination=112481484 id="112481484" class="section-title" >4. Discussion</h2>
<p>The melon genome sequence<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and several genetic maps<a reveal-id="dsx033-B7" data-open="dsx033-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx033-B8" data-open="dsx033-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx033-B11" data-open="dsx033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx033-B31" data-open="dsx033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> have recently been published. However, functional genomics studies involving melon are still restricted by the fact only a few hundred or thousand markers have been explored. Based on a F<sub>2</sub> population of double-haploid DHL92 used for melon genome sequence,<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> a genetic map was developed composing of 580 SNPs, which anchored the scaffolds into a chromosome-scale pseudomolecules.<a reveal-id="dsx033-B31" data-open="dsx033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> This work not only provided a significantly updated melon genome, but also indicated that high-resolution genetic map could be an efficient approach for the improvement of genome draft sequence. Rapid advances in high-throughput sequencing have revolutionized SNP marker discovery and genotyping analyses. Additionally, high-throughput sequencing-based approaches are now routinely used to construct genetic maps.<a reveal-id="dsx033-B13" data-open="dsx033-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx033-B15" data-open="dsx033-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx033-B16" data-open="dsx033-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> An important strength of genotyping-by-sequencing (GBS) is that the detection and genotyping of genome-wide SNP markers occur simultaneously. Moreover, a GBS approach does not require a reference genome for SNP calling and genotyping. However, an available reference genome is useful because it enables the proper alignment and ordering of sequenced tags and helps to impute low-coverage data.<a reveal-id="dsx033-B32" data-open="dsx033-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> These two sub-species are commonly used in breeding programmes because they are resistant to multiple diseases and exhibit desirable traits regarding sugar content and flavour. After a genotyping analysis, we obtained 12,932 bin markers, which included more than one million SNPs. Using these bin markers, we constructed the first ultra-dense genetic map that can be used as a reference in future studies of melon. Recently, 354.98 Mb of melon scaffolds were anchored and 325.4 Mb of sequences were oriented by a new set of targeted SNP markers with better distribution in the parents of the DHL92 melon reference genome.<a reveal-id="dsx033-B31" data-open="dsx033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> In our study, several missed scaffolds were assembled into melon pseudo-chromosome. Furthermore, more sequence (336.05 Mb) were oriented in our new scaffold genome, suggesting that our ultra-dense genetic map could further improve the anchoring and orienting of melon scaffold genome assembly.</p><p>A GBS-based high-resolution genetic map can be efficiently used to identify genetic regions and candidate genes underlying agronomically important traits.<a reveal-id="dsx033-B17" data-open="dsx033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx033-B33" data-open="dsx033-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx033-B34" data-open="dsx033-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> We mapped a 0.667 cM region on pseudo-chromosome 4, and identified eight candidate genes for Gsb resistance in this region. Thus, an ultra-dense genetic map can be constructed and candidate genes underlying Gsb resistance can be mapped using a GBS approach. Genetic map-based high-quality reference genome sequences provide useful information not only for identifying genes and regulatory elements, but also for characterizing genomic diversity.<a reveal-id="dsx033-B35" data-open="dsx033-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Our ultra-dense genetic map improved the re-anchoring of scaffolds to the reference genome. It may also contribute fundamental information for the <em>de novo</em> assembly of a high-quality oriental melon genome. Moreover, the improved anchoring of scaffolds may help researchers analyse chromosomal structures to improve the accuracy of gene mapping. Comparisons of genetic maps can reveal the genetic basis for conserved and variable sequences within species or among related species. We observed a strong co-linearity, but also rearrangements between our HS × XH F<sub>2</sub> genetic map and the genetic maps of whole-genome sequenced melons, especially in pseudo-chromosomes 3–5. A chromosome-level comparison between the <em>C. melo</em> genetic map and the <em>C. sativus</em> and <em>C. lanatus</em> genomes revealed syntenic relationships among these three <em>Cucurbitaceae</em> species. The complex syntenic patterns presented as segmented chromosome-to-chromosome orthologous relationships were consistent with the notion that the chromosomal evolution and rearrangement among these three <em>Cucurbitaceae</em> species is very complex.<a reveal-id="dsx033-B10" data-open="dsx033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx033-B36" data-open="dsx033-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsx033-B37" data-open="dsx033-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> However, it remains possible that such differences are derived from different mapping populations being used for various genotypes associated with a chromosome-level reorganization.</p><p>Among the detected Gsb-resistance candidate genes, <em>MELO3C012986</em> encodes a protein similar to a VHS domain-containing protein, which is involved in clathrin-related endomembrane trafficking in plants.<a reveal-id="dsx033-B38" data-open="dsx033-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><em>MELO3C012991</em> encodes a Sec13, which is part of a nuclear pore complex with subunits that are involved in the responses to pathogens, cold stress, auxin, and ethylene.<a reveal-id="dsx033-B39" data-open="dsx033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> We identified an <em>RT</em>-like gene (i.e., encoding a reverse transcriptase) upstream of <em>Sec13.</em> An RT gene is usually associated with a mobile element, such as a retrotransposon, retrovirus, group II intron, retron, or retroplasmid, as well as an occasional retro (pseudo) gene.<a reveal-id="dsx033-B40 dsx033-B41 dsx033-B42" data-open="dsx033-B40 dsx033-B41 dsx033-B42" class="link link-ref link-reveal xref-bibr"><sup>40–42</sup></a><sup>,</sup><em>MELO3C012992</em> encodes a GATA zinc finger transcription factor family gene (GATA TF4-like), which has been implicated in light-responsive, circadian-regulated, and nitrate-dependent control of transcription.<a reveal-id="dsx033-B43" data-open="dsx033-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx033-B44" data-open="dsx033-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p><p>Plants are often attacked by pathogens and have evolved a multi-layered defence system. An essential defence mechanism involves the expression of resistance (<em>R</em>) genes, which enables plants to detect pathogens carrying the corresponding avirulence (<em>Avr</em>) genes.<a reveal-id="dsx033-B21" data-open="dsx033-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The R protein-mediated pathogen recognition can be direct or indirect via the action of effectors (Avr proteins) on host targets. The <em>R</em> gene-based resistance is usually considered to be related to gene-for-gene resistance, which often involves a hypersensitive response (HR). <em>MELO3C012987</em> encodes a protein that is similar to the uncharacterized Avr9/Cf-9 rapidly elicited (ACRE) protein 146, which is related to the gene-for-gene dominant resistance to fungal pathogens.<a reveal-id="dsx033-B45" data-open="dsx033-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx033-B46" data-open="dsx033-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Many ACRE genes encode putative signalling components, suggesting they are essential for early defence responses.<a reveal-id="dsx033-B45" data-open="dsx033-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Four genes (i.e., <em>ACRE74</em>, <em>ACRE189</em>, <em>ACRE264</em>, and <em>ACRE276</em>) have been confirmed as essential for both Cf-9- and Cf-4-mediated HR. Previous studies determined that <em>ACRE74</em>, <em>ACRE264</em>, and <em>ACRE276</em> are critical for the Cf-9-mediated resistance to <em>C. fulvum</em>.<a reveal-id="dsx033-B47 dsx033-B48 dsx033-B49" data-open="dsx033-B47 dsx033-B48 dsx033-B49" class="link link-ref link-reveal xref-bibr"><sup>47–49</sup></a> Our data suggested that <em>MELO3C012987</em>, which encodes a protein similar to ACRE146, might be the Gsb-resistance gene in melon.</p><p>In conclusion, our ultra-dense genetic map may be useful for future studies aimed at identifying the genes regulating agronomically important melon traits. Additionally, functionally characterizing <em>MELO3C01</em>2987 may contribute to the development of new strategies to control Gsb in <em>Cucurbitaceae</em> species. However, gene-based disease resistance sometimes fails because of developmental and/or environmental changes. Incorporating Gsb resistance into field-grown crops will require the development of sustainable broad-spectrum disease resistance.</p>                            <h2 scrollto-destination=112481490 id="112481490" class="section-title" >Availability</h2>
<p>The re-sequence data can be available at NCBI under accessions SRP114921.</p>                            <h2 scrollto-destination=112481492 id="112481492" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Prof Fengming Song from Institute of Biotechnology, Zhejiang University, to offer the Gsb isolate.</p>                            <h3 scrollto-destination=112481494 id="112481494" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=112481496 id="112481496" class="section-title" >Funding</h3>
<p>This study was supported by grants from the earmarked fund for Modern Agro-Industry Technology Research System of China (CARS-26-17), the Science and Technology Program of Zhejiang Province (2017C32041), and the Breeding Alliance of Rice and Major Economic Crops of Zhejiang University.</p>                            <h3 scrollto-destination=112481498 id="112481498" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=112481500 id="112481500" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx033-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keinath</div> <div class="given-names">A.P.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">From native plants in Central Europe to cultivated crops worldwide: the emergence of Didymella bryoniae as a Cucurbit Pathogen</div>. <div class="source">Hortscience</div> , <div class="volume">46</div>, <div class="fpage">532</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="dsx033-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sitterly</div> <div class="given-names">W.R.</div></div></div> <div class="year">1972</div>, 
<div class="article-title">Breeding for disease resistance in Cucurbits</div>. <div class="source">Annu. Rev. Phytopathol</div> ., <div class="volume">10</div>, <div class="fpage">471</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Breeding%20for%20disease%20resistance%20in%20Cucurbits&amp;author=W.R.%20Sitterly&amp;publication_year=1972&amp;journal=Annu.%20Rev.%20Phytopathol&amp;volume=10&amp;pages=471-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.py.10.090172.002351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.py.10.090172.002351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.py.10.090172.002351"> </span></div> </div></div></div></div></div><div content-id="dsx033-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Frantz</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Jahn</div> <div class="given-names">M.M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Five independent loci each control monogenic resistance to gummy stem blight in melon (<em>Cucumis melo</em> L.)</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">108</div>, <div class="fpage">1033</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Five%20independent%20loci%20each%20control%20monogenic%20resistance%20to%20gummy%20stem%20blight%20in%20melon%20(Cucumis%20melo%20L.)&amp;author=J.D.%20Frantz&amp;author=M.M.%20Jahn&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet&amp;volume=108&amp;pages=1033-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1519-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1519-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1519-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolukau</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">X.H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.F.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Identification of amplified fragment length polymorphism markers linked to gummy stem blight (Didymella bryoniae) resistance in Melon (<em>Cucumis melo</em> L.) PI 420145</div>. <div class="source">Hortscience</div> , <div class="volume">44</div>, <div class="fpage">32</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="dsx033-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Qi</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Pryamiding disease resistance genes and variety improvment by molecular marker-assisted selection in melon (<em>Cucumis melon</em> L.)</div>. <div class="source">Sci. Agric. Sin</div> ., <div class="volume">48</div>, <div class="fpage">523</div>–<div class="lpage">33</div>.</div></div></div></div><div content-id="dsx033-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diaz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fergany</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Formisano</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A consensus linkage map for molecular markers and quantitative trait loci associated with economically important traits in melon (<em>Cucumis melo</em> L.)</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">11</div>, <div class="fpage">111</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20for%20molecular%20markers%20and%20quantitative%20trait%20loci%20associated%20with%20economically%20important%20traits%20in%20melon%20(Cucumis%20melo%20L.)&amp;author=A.%20Diaz&amp;author=M.%20Fergany&amp;author=G.%20Formisano&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol&amp;volume=11&amp;pages=111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21797998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harel-Beja</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Tzuri</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Portnoy</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A genetic map of melon highly enriched with fruit quality QTLs and EST markers, including sugar and carotenoid metabolism genes</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">121</div>, <div class="fpage">511</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20map%20of%20melon%20highly%20enriched%20with%20fruit%20quality%20QTLs%20and%20EST%20markers%2C%20including%20sugar%20and%20carotenoid%20metabolism%20genes&amp;author=R.%20Harel-Beja&amp;author=G.%20Tzuri&amp;author=V.%20Portnoy&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet&amp;volume=121&amp;pages=511-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1327-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1327-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1327-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20401460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yuste-Lisbona</div> <div class="given-names">F.J.</div></div>, <div class="name string-name"><div class="surname">Capel</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sarria</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Genetic linkage map of melon (<em>Cucumis melo</em> L.) and localization of a major QTL for powdery mildew resistance</div>. <div class="source">Mol. Breeding</div> , <div class="volume">27</div>, <div class="fpage">181</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20of%20melon%20(Cucumis%20melo%20L.)%20and%20localization%20of%20a%20major%20QTL%20for%20powdery%20mildew%20resistance&amp;author=F.J.%20Yuste-Lisbona&amp;author=C.%20Capel&amp;author=E.%20Sarria&amp;publication_year=2011&amp;journal=Mol.%20Breeding&amp;volume=27&amp;pages=181-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9421-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9421-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9421-5"> </span></div> </div></div></div></div></div><div content-id="dsx033-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chang</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.H.</div></div>, <div class="name string-name"><div class="surname">Tung</div> <div class="given-names">C.W.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Genome-wide single nucleotide polymorphism discovery and the construction of a high-density genetic map for melon (<em>Cucumis melo</em> L.) using genotyping-by-sequencing</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">8</div>, <div class="fpage">125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20single%20nucleotide%20polymorphism%20discovery%20and%20the%20construction%20of%20a%20high-density%20genetic%20map%20for%20melon%20(Cucumis%20melo%20L.)%20using%20genotyping-by-sequencing&amp;author=C.W.%20Chang&amp;author=Y.H.%20Wang&amp;author=C.W.%20Tung&amp;publication_year=2017&amp;journal=Front.%20Plant%20Sci&amp;volume=8&amp;pages=125" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28220139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Benjak</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Sanseverino</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The genome of melon (<em>Cucumis melo</em> L.)</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">109</div>, <div class="fpage">11872</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20melon%20(Cucumis%20melo%20L.)&amp;author=J.%20Garcia-Mas&amp;author=A.%20Benjak&amp;author=W.%20Sanseverino&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=109&amp;pages=11872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1205415109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1205415109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1205415109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22753475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Diaz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Forment</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Argyris</div> <div class="given-names">J.M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Anchoring the consensus ICuGI genetic map to the melon (<em>Cucumis melo</em> L.) genome</div>. <div class="source">Mol. Breeding</div> , <div class="volume">35</div>, <div class="fpage">188</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anchoring%20the%20consensus%20ICuGI%20genetic%20map%20to%20the%20melon%20(Cucumis%20melo%20L.)%20genome&amp;author=A.%20Diaz&amp;author=J.%20Forment&amp;author=J.M.%20Argyris&amp;publication_year=2015&amp;journal=Mol.%20Breeding&amp;volume=35&amp;pages=188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-015-0381-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-015-0381-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-015-0381-7"> </span></div> </div></div></div></div></div><div content-id="dsx033-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davey</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Boone</div> <div class="given-names">J.Q.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">499</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.W.%20Davey&amp;author=P.A.%20Hohenlohe&amp;author=P.D.%20Etter&amp;author=J.Q.%20Boone&amp;author=J.M.%20Catchen&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.H.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">High-throughput genotyping by whole-genome resequencing</div>. <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1068</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20by%20whole-genome%20resequencing&amp;author=X.H.%20Huang&amp;author=Q.%20Feng&amp;author=Q.%20Qian&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1068-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089516.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089516.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089516.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xua</div> <div class="given-names">X.Y.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Pinpointing genes underlying the quantitative trait loci for root-knot nematode resistance in palaeopolyploid soybean by whole genome resequencing</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">110</div>, <div class="fpage">13469</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pinpointing%20genes%20underlying%20the%20quantitative%20trait%20loci%20for%20root-knot%20nematode%20resistance%20in%20palaeopolyploid%20soybean%20by%20whole%20genome%20resequencing&amp;author=X.Y.%20Xua&amp;author=L.%20Zeng&amp;author=Y.%20Tao&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=110&amp;pages=13469-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1222368110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1222368110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1222368110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23898176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Sequence-based ultra-dense genetic and physical maps reveal structural variations of allopolyploid cotton genomes</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20ultra-dense%20genetic%20and%20physical%20maps%20reveal%20structural%20variations%20of%20allopolyploid%20cotton%20genomes&amp;author=S.%20Wang&amp;author=J.%20Chen&amp;author=W.%20Zhang&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0678-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0678-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0678-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26003111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mun</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Chung</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chung</div> <div class="given-names">W.H.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Construction of a reference genetic map of <em>Raphanus sativus</em> based on genotyping by whole-genome resequencing</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">128</div>, <div class="fpage">259</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20reference%20genetic%20map%20of%20Raphanus%20sativus%20based%20on%20genotyping%20by%20whole-genome%20resequencing&amp;author=J.H.%20Mun&amp;author=H.%20Chung&amp;author=W.H.%20Chung&amp;publication_year=2015&amp;journal=Theor.%20Appl.%20Genet&amp;volume=128&amp;pages=259-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-014-2426-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-014-2426-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-014-2426-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25403353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Candela</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Casanova-Saez</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Micol</div> <div class="given-names">J.L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Getting started in mapping-by-sequencing</div>. <div class="source">J. Integr. Plant Biol</div> ., <div class="volume">57</div>, <div class="fpage">606</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Getting%20started%20in%20mapping-by-sequencing&amp;author=H.%20Candela&amp;author=R.%20Casanova-Saez&amp;author=J.L.%20Micol&amp;publication_year=2015&amp;journal=J.%20Integr.%20Plant%20Biol&amp;volume=57&amp;pages=606-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.v57.7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.v57.7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.v57.7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25359627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Using next-generation sequencing to isolate mutant genes from forward genetic screens</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">662</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20next-generation%20sequencing%20to%20isolate%20mutant%20genes%20from%20forward%20genetic%20screens&amp;author=K.%20Schneeberger&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=662-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25139187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chisholm</div> <div class="given-names">S.T.</div></div>, <div class="name string-name"><div class="surname">Coaker</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Day</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Staskawicz</div> <div class="given-names">B.J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Host-microbe interactions: shaping the evolution of the plant immune response</div>. <div class="source">Cell</div> , <div class="volume">124</div>, <div class="fpage">803</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host-microbe%20interactions%3A%20shaping%20the%20evolution%20of%20the%20plant%20immune%20response&amp;author=S.T.%20Chisholm&amp;author=G.%20Coaker&amp;author=B.%20Day&amp;author=B.J.%20Staskawicz&amp;publication_year=2006&amp;journal=Cell&amp;volume=124&amp;pages=803-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.02.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.02.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.02.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16497589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dodds</div> <div class="given-names">P.N.</div></div>, <div class="name string-name"><div class="surname">Rathjen</div> <div class="given-names">J.P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Plant immunity: towards an integrated view of plant-pathogen interactions</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">539</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20immunity%3A%20towards%20an%20integrated%20view%20of%20plant-pathogen%20interactions&amp;author=P.N.%20Dodds&amp;author=J.P.%20Rathjen&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=539-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20585331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J.D.G.</div></div>, <div class="name string-name"><div class="surname">Dangl</div> <div class="given-names">J.L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The plant immune system</div>. <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">323</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plant%20immune%20system&amp;author=J.D.G.%20Jones&amp;author=J.L.%20Dangl&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=323-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05286"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17108957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Katagiri</div> <div class="given-names">F.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Comparing signaling mechanisms engaged in pattern-triggered and effector-triggered immunity</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">13</div>, <div class="fpage">459</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparing%20signaling%20mechanisms%20engaged%20in%20pattern-triggered%20and%20effector-triggered%20immunity&amp;author=K.%20Tsuda&amp;author=F.%20Katagiri&amp;publication_year=2010&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=13&amp;pages=459-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2010.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2010.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2010.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20471306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kwon</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Sung</div> <div class="given-names">K.Y.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">B.H.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">K.C.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Standardization of a mass production technique for pycnidiospores of <em>Didymella bryoniae</em>, gummy stem blight fungus of Cucurbits</div>. <div class="source">Korean J. Plant Pathol</div> ., <div class="volume">13</div>, <div class="fpage">105</div>–<div class="lpage">12</div>.</div></div></div></div><div content-id="dsx033-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DePristo</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.%20A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.Y.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">C.X.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">W.G.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Construction and analysis of high-density linkage map using high-throughput sequencing data</div>. <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e98855</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20analysis%20of%20high-density%20linkage%20map%20using%20high-throughput%20sequencing%20data&amp;author=D.Y.%20Liu&amp;author=C.X.%20Ma&amp;author=W.G.%20Hong&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e98855" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0098855" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0098855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0098855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24905985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.T.</div></div>, <div class="name string-name"><div class="surname">Miao</div> <div class="given-names">C.Y.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">ALLMAPS: robust scaffold ordering based on multiple maps</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ALLMAPS%3A%20robust%20scaffold%20ordering%20based%20on%20multiple%20maps&amp;author=H.B.%20Tang&amp;author=X.T.%20Zhang&amp;author=C.Y.%20Miao&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0573-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0573-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0573-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.X.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">T.R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (<em>Prunus mume</em> Sieb. et Zucc)</div>. <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">183</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20genetic%20map%20construction%20and%20identification%20of%20a%20locus%20controlling%20weeping%20trait%20in%20an%20ornamental%20woody%20plant%20(Prunus%20mume%20Sieb.%20et%20Zucc)&amp;author=J.%20Zhang&amp;author=Q.X.%20Zhang&amp;author=T.R.%20Cheng&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=183-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25776277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Broman</div> <div class="given-names">K.W.</div></div>, <div class="name string-name"><div class="surname">Sen</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, <div class="source">A guide to QTL mapping with Q/qtl</div> . 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">New York</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20guide%20to%20QTL%20mapping%20with%20Q%2Fqtl&amp;author=K.W.%20Broman&amp;author=S.%20Sen&amp;publication_year=2009&amp;book=A%20guide%20to%20QTL%20mapping%20with%20Q%2Fqtl" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-0-387-92125-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-0-387-92125-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-0-387-92125-9"> </span></div> </div></div></div></div></div><div content-id="dsx033-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Argyris</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Ruiz-Herrera</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Madriz-Masis</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Use of targeted SNP selection for an improved anchoring of the melon (<em>Cucumis melo</em> L.) scaffold genome assembly</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20targeted%20SNP%20selection%20for%20an%20improved%20anchoring%20of%20the%20melon%20(Cucumis%20melo%20L.)%20scaffold%20genome%20assembly&amp;author=J.M.%20Argyris&amp;author=A.%20Ruiz-Herrera&amp;author=P.%20Madriz-Masis&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-014-1196-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-014-1196-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-014-1196-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25612459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Poland</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Rife</div> <div class="given-names">T.W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genotyping-by-sequencing for plant breeding and genetics</div>. <div class="source">Plant Genome-Us</div> , <div class="volume">5</div>, <div class="fpage">92</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20for%20plant%20breeding%20and%20genetics&amp;author=J.A.%20Poland&amp;author=T.W.%20Rife&amp;publication_year=2012&amp;journal=Plant%20Genome-Us&amp;volume=5&amp;pages=92-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2012.05.0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2012.05.0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2012.05.0005"> </span></div> </div></div></div></div></div><div content-id="dsx033-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bandhiwal</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bharadwaj</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bhatia</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">High-density linkage map construction and mapping of seed trait QTLs in chickpea (<em>Cicer arietinum</em> L.) using genotyping-by-sequencing (GBS)</div>. <div class="source">Sci. Rep.-Uk</div> , <div class="volume">5</div>, <div class="fpage">17512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20linkage%20map%20construction%20and%20mapping%20of%20seed%20trait%20QTLs%20in%20chickpea%20(Cicer%20arietinum%20L.)%20using%20genotyping-by-sequencing%20(GBS)&amp;author=S.%20Verma&amp;author=S.%20Gupta&amp;author=N.%20Bandhiwal&amp;author=T.%20Kumar&amp;author=C.%20Bharadwaj&amp;author=S.%20Bhatia&amp;publication_year=2015&amp;journal=Sci.%20Rep.-Uk&amp;volume=5&amp;pages=17512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep17512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep17512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep17512"> </span></div> </div></div></div></div></div><div content-id="dsx033-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Izzah</div> <div class="given-names">N.K.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">B.S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (<em>Brassica oleracea</em> L.)</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">29</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20map%20permits%20identification%20of%20clubroot%20resistance%20QTLs%20and%20revision%20of%20the%20reference%20genome%20assembly%20in%20cabbage%20(Brassica%20oleracea%20L.)&amp;author=J.%20Lee&amp;author=N.K.%20Izzah&amp;author=B.S.%20Choi&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=29-41" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26622061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feuillet</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Leach</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Rogers</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Eversole</div> <div class="given-names">K.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Crop genome sequencing: lessons and rationales</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">16</div>, <div class="fpage">77</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20genome%20sequencing%3A%20lessons%20and%20rationales&amp;author=C.%20Feuillet&amp;author=J.E.%20Leach&amp;author=J.%20Rogers&amp;author=P.S.%20Schnable&amp;author=K.%20Eversole&amp;publication_year=2011&amp;journal=Trends%20Plant%20Sci&amp;volume=16&amp;pages=77-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">H.H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The draft genome of watermelon (<em>Citrullus lanatus</em>) and resequencing of 20 diverse accessions</div>. <div class="source">Nat Genet</div> ., <div class="volume">45</div>, <div class="fpage">51</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20watermelon%20(Citrullus%20lanatus)%20and%20resequencing%20of%2020%20diverse%20accessions&amp;author=S.G.%20Guo&amp;author=J.G.%20Zhang&amp;author=H.H.%20Sun&amp;publication_year=2013&amp;journal=Nat%20Genet&amp;volume=45&amp;pages=51-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2470" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2470" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2470"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23179023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.Q.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.H.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The genome of the cucumber, <em>Cucumis sativus</em> L</div>. <div class="source">Nat. Genet</div> ., <div class="volume">41</div>, <div class="fpage">1275</div>–<div class="lpage">U1229</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20cucumber%2C%20Cucumis%20sativus%20L&amp;author=S.W.%20Huang&amp;author=R.Q.%20Li&amp;author=Z.H.%20Zhang&amp;publication_year=2009&amp;journal=Nat.%20Genet&amp;volume=41&amp;pages=1275-U1229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zouhar</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sauer</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Helping hands for budding prospects: ENTH/ANTH/VHS accessory proteins in endocytosis, vacuolar transport, and secretion</div>. <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">4232</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helping%20hands%20for%20budding%20prospects%3A%20ENTH%2FANTH%2FVHS%20accessory%20proteins%20in%20endocytosis%2C%20vacuolar%20transport%2C%20and%20secretion&amp;author=J.%20Zouhar&amp;author=M.%20Sauer&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=4232-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.131680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.131680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.131680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25415979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parry</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Assessing the function of the plant nuclear pore complex and the search for specificity</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">64</div>, <div class="fpage">833</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20function%20of%20the%20plant%20nuclear%20pore%20complex%20and%20the%20search%20for%20specificity&amp;author=G.%20Parry&amp;publication_year=2013&amp;journal=J.%20Exp.%20Bot&amp;volume=64&amp;pages=833-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/ers289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fers289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Fers289"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23077202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schulman</div> <div class="given-names">A.H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Retrotransposon replication in plants</div>. <div class="source">Curr. Opin. Virol</div> ., <div class="volume">3</div>, <div class="fpage">604</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposon%20replication%20in%20plants&amp;author=A.H.%20Schulman&amp;publication_year=2013&amp;journal=Curr.%20Opin.%20Virol&amp;volume=3&amp;pages=604-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.coviro.2013.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.coviro.2013.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.coviro.2013.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24035277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gladyshev</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Arkhipova</div> <div class="given-names">I.R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A widespread class of reverse transcriptase-related cellular genes</div>. <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">108</div>, <div class="fpage">20311</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20widespread%20class%20of%20reverse%20transcriptase-related%20cellular%20genes&amp;author=E.A.%20Gladyshev&amp;author=I.R.%20Arkhipova&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=108&amp;pages=20311-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1100266108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1100266108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1100266108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21876125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Toro</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Nisa-Martinez</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comprehensive phylogenetic analysis of bacterial reverse transcriptases</div>. <div class="source">Plos One</div> , <div class="volume">9</div>, <div class="fpage">e114083</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20phylogenetic%20analysis%20of%20bacterial%20reverse%20transcriptases&amp;author=N.%20Toro&amp;author=R.%20Nisa-Martinez&amp;publication_year=2014&amp;journal=Plos%20One&amp;volume=9&amp;pages=e114083" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0114083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0114083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0114083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25423096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reyes</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Muro-Pastor</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Florencio</div> <div class="given-names">F.J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The GATA family of transcription factors in Arabidopsis and rice</div>. <div class="source">Plant Physiol</div> ., <div class="volume">134</div>, <div class="fpage">1718</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GATA%20family%20of%20transcription%20factors%20in%20Arabidopsis%20and%20rice&amp;author=J.C.%20Reyes&amp;author=M.I.%20Muro-Pastor&amp;author=F.J.%20Florencio&amp;publication_year=2004&amp;journal=Plant%20Physiol&amp;volume=134&amp;pages=1718-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.037788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.037788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.037788"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15084732" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Manfield</div> <div class="given-names">I.W.</div></div>, <div class="name string-name"><div class="surname">Devlin</div> <div class="given-names">P.F.</div></div>, <div class="name string-name"><div class="surname">Jen</div> <div class="given-names">C.H.</div></div>, <div class="name string-name"><div class="surname">Westhead</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Gilmartin</div> <div class="given-names">P.M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Conservation, convergence, and divergence of light-responsive, circadian-regulated, and tissue-specific expression patterns during evolution of the Arabidopsis GATA gene family</div>. <div class="source">Plant Physiol</div> ., <div class="volume">143</div>, <div class="fpage">941</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%2C%20convergence%2C%20and%20divergence%20of%20light-responsive%2C%20circadian-regulated%2C%20and%20tissue-specific%20expression%20patterns%20during%20evolution%20of%20the%20Arabidopsis%20GATA%20gene%20family&amp;author=I.W.%20Manfield&amp;author=P.F.%20Devlin&amp;author=C.H.%20Jen&amp;author=D.R.%20Westhead&amp;author=P.M.%20Gilmartin&amp;publication_year=2007&amp;journal=Plant%20Physiol&amp;volume=143&amp;pages=941-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.090761" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.090761" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.090761"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17208962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stergiopoulos</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">de Wit</div> <div class="given-names">P.J.G.M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fungal effector proteins</div>. <div class="source">Annu. Rev. Phytopathol</div> ., <div class="volume">47</div>, <div class="fpage">233</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20effector%20proteins&amp;author=I.%20Stergiopoulos&amp;author=P.J.G.M.%20de%20Wit&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Phytopathol&amp;volume=47&amp;pages=233-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.phyto.112408.132637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.phyto.112408.132637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.phyto.112408.132637"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19400631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grennan</div> <div class="given-names">A.K.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Plant response to bacterial pathogens. Overlap between innate and gene-for-gene defense response</div>. <div class="source">Plant Physiol</div> ., <div class="volume">142</div>, <div class="fpage">809</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20response%20to%20bacterial%20pathogens.%20Overlap%20between%20innate%20and%20gene-for-gene%20defense%20response&amp;author=A.K.%20Grennan&amp;publication_year=2006&amp;journal=Plant%20Physiol&amp;volume=142&amp;pages=809-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.900207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.900207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.900207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17093133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rowland</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ludwig</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Merrick</div> <div class="given-names">C.J.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Functional analysis of Avr9/Cf-9 rapidly elicited genes identifies a protein kinase, ACIK1, that is essential for full Cf-9-dependent disease resistance in tomato</div>. <div class="source">Plant Cell</div> , <div class="volume">17</div>, <div class="fpage">295</div>–<div class="lpage">310</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20analysis%20of%20Avr9%2FCf-9%20rapidly%20elicited%20genes%20identifies%20a%20protein%20kinase%2C%20ACIK1%2C%20that%20is%20essential%20for%20full%20Cf-9-dependent%20disease%20resistance%20in%20tomato&amp;author=O.%20Rowland&amp;author=A.A.%20Ludwig&amp;author=C.J.%20Merrick&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=295-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.026013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.026013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.026013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15598806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nekrasov</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Ludwig</div> <div class="given-names">A. A.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J. D. G.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">CITRX thioredoxin is a putative adaptor protein connecting Cf-9 and the ACIK1 protein kinase during the Cf-9/Avr9-induced defence response</div>. <div class="source">FEBS Lett</div> ., <div class="volume">580</div>, <div class="fpage">4236</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CITRX%20thioredoxin%20is%20a%20putative%20adaptor%20protein%20connecting%20Cf-9%20and%20the%20ACIK1%20protein%20kinase%20during%20the%20Cf-9%2FAvr9-induced%20defence%20response&amp;author=V.%20Nekrasov&amp;author=A.%20A.%20Ludwig&amp;author=J.%20D.%20G.%20Jones&amp;publication_year=2006&amp;journal=FEBS%20Lett&amp;volume=580&amp;pages=4236-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2006.06.077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2006.06.077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2006.06.077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16831430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx033-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx033-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van den Burg</div> <div class="given-names">H.A.</div></div>, <div class="name string-name"><div class="surname">Tsitsigiannis</div> <div class="given-names">D.I.</div></div>, <div class="name string-name"><div class="surname">Rowland</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">The F-box protein ACRE189/ACIF1 regulates cell death and defense responses activated during pathogen recognition in tobacco and tomato</div>. <div class="source">Plant Cell</div> , <div class="volume">20</div>, <div class="fpage">697</div>–<div class="lpage">719</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20F-box%20protein%20ACRE189%2FACIF1%20regulates%20cell%20death%20and%20defense%20responses%20activated%20during%20pathogen%20recognition%20in%20tobacco%20and%20tomato&amp;author=H.A.%20van%20den%20Burg&amp;author=D.I.%20Tsitsigiannis&amp;author=O.%20Rowland&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=697-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.107.056978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.107.056978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.107.056978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4107259&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx033/3/dsx033_suppl_data.zip?Expires=2147483647&amp;Signature=NdNNhSvmiUAZSdipyxi4KLP1f9gbLpTOk-O3z7ZoT0O2XYzoR7jS1sIna20iVGDA8yNEPxp8Pv5D1JGIVirw~LffwC7K9oOmfSy2V~D5zOYIEWFhMNFej6UCbzY~Arf7pAlnHZPzu0nIYGTcJnprm1SFGytIz9B3YprPTnhqHzUX8yjGruxl5eq8cAH7vr99EFFtcujNcSeIlW5kVRCKvL3q4mRJH60uxeF~0lIz6xpUNpMUa4RMWM~rMnd8f4nes9Y0AdexlGOv6nCygPSeGfxTUqVCpZvmsmS8mfsX4mfxO4zNYPpA5niImFzgUgPwQqd1rQKLTg2B8etaGLy66Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,768</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx033"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4107259"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f1%2f4107259">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F1%2F4107259" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4107259'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="re-sequencing-based ultra-dense genetic map reveals a gummy stem blight resistance-associated gene in Cucumis melo" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx033");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f1%2f4107259">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

