

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/1/53/377009","@type":"ScholarlyArticle","name":" De Novo  Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification","datePublished":"2011-01-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2011-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq028","keywords":["<italic>De novo</italic> assembly","chickpea","next generation sequencing","transcriptome","short read"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/1/53/377009","author":[{"name":"Garg, Rohini","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110 067 ,  India","@type":"Person"},{"name":"Patel, Ravi K.","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110 067 ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110 067 ,  India","@type":"Person"},{"name":"Jain, Mukesh","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg ,  New Delhi 110 067 ,  India","@type":"Person"}],"description":"Abstract.  Chickpea ranks third among the food legume crops production in the world. However, the genomic resources available for chickpea are still very limite","pageStart":"53","pageEnd":"63","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" De Novo  Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" De Novo  Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification" />
    <meta property="og:description" content="Abstract.  Chickpea ranks third among the food legume crops production in the world. However, the genomic resources available for chickpea are still very limite" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/1/53/377009" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-01-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Garg, Rohini" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Patel, Ravi K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Jain, Mukesh" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_title" content="De Novo Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification" /><meta name="citation_firstpage" content="53" /><meta name="citation_lastpage" content="63" /><meta name="citation_doi" content="10.1093/dnares/dsq028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2011/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Legumes: importance and constraints to greater use; Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_pages=872-7; " /><meta name="citation_reference" content="citation_title=Chickpea molecular breeding: new tools and concepts; Euphytica;  citation_year=2006;  citation_volume=147;  citation_pages=81-103; " /><meta name="citation_reference" content="citation_title=Genomics-assisted breeding for crop improvement; Trends Plant Sci.;  citation_year=2005;  citation_volume=10;  citation_pages=621-30; " /><meta name="citation_reference" content="citation_title=Advances in cereal genomics and applications in crop breeding; Trends Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=490-9; " /><meta name="citation_reference" content="citation_title=Development of ESTs from chickpea roots and their use in diversity analysis of the Cicer genus; BMC Plant Biol.;  citation_year=2005;  citation_volume=5;  citation_pages=16" /><meta name="citation_reference" content="citation_title=Comparative analysis of ESTs in response to drought stress in chickpea (C. arietinum L.); Biochem. Biophys. Res. Commun.;  citation_year=2008;  citation_volume=376;  citation_pages=578-83; " /><meta name="citation_reference" content="citation_title=Comparative analyses of genotype dependent expressed sequence tags and stress-responsive transcriptome of chickpea wilt illustrate predicted and unexpected genes and novel regulators of plant immunity; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=415" /><meta name="citation_reference" content="citation_title=A comprehensive resource of drought- and salinity-responsive ESTs for gene discovery and marker development in chickpea (Cicer arietinum L.); BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=523" /><meta name="citation_reference" content="citation_title=Analysis of gene expression in response to water deficit of chickpea (Cicer arietinum L.) varieties differing in drought tolerance; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=24" /><meta name="citation_reference" content="citation_title=Transcriptional profiling of chickpea genes differentially regulated in response to high-salinity, cold and drought; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=303" /><meta name="citation_reference" content="citation_title=SuperSAGE: the drought stress-responsive transcriptome of chickpea roots; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=553" /><meta name="citation_reference" content="citation_title=Expression of CAP2, an APETALA2-family transcription factor from chickpea, enhances growth and tolerance to dehydration and salt stress in transgenic tobacco; Plant Physiol.;  citation_year=2006;  citation_volume=142;  citation_pages=113-23; " /><meta name="citation_reference" content="citation_title=CIPK6, a CBL-interacting protein kinase is required for development and salt tolerance in plants; Plant J.;  citation_year=2009;  citation_volume=58;  citation_pages=778-90; " /><meta name="citation_reference" content="citation_title=Applications of new sequencing technologies for transcriptome analysis; Annu. Rev. Genomics Hum. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=135-51; " /><meta name="citation_reference" content="citation_title=Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing; Mol. Ecol.;  citation_year=2008;  citation_volume=17;  citation_pages=1636-47; " /><meta name="citation_reference" content="citation_title=Development of a EST dataset and characterization of EST-SSRs in a traditional Chinese medicinal plant, Epimedium sagittatum (Sieb. Et Zucc.) Maxim; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=94" /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=ABySS: a parallel assembler for short read sequence data; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1117-23; " /><meta name="citation_reference" content="citation_title=Parallel short sequence assembly of transcriptomes; BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_issue=Suppl 1;  citation_pages=S14" /><meta name="citation_reference" content="citation_title=De novo transcriptome assembly with ABySS; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2872-7; " /><meta name="citation_reference" content="citation_title=Validation of internal control genes for quantitative gene expression studies in chickpea (Cicer arietinum L.); Biochem. Biophys. Res. Commun.;  citation_year=2010;  citation_volume=396;  citation_pages=283-8; " /><meta name="citation_reference" content="citation_title=Two classes of genes in plants; Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1819-25; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-5; " /><meta name="citation_reference" content="citation_title=Legume transcription factor genes: what makes legumes so special; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=991-1001; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=Gene expression analysis goes digital; Nat. Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=878-80; " /><meta name="citation_reference" content="citation_title=RNA-Seq: a revolutionary tool for transcriptomics; Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=57-63; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing transforms today's biology; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=16-8; " /><meta name="citation_reference" content="citation_title=Benchmarking next-generation transcriptome sequencing for functional and evolutionary genomics; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=2731-44; " /><meta name="citation_reference" content="citation_title=Sequencing Medicago truncatula expressed sequenced tags using 454 Life Sciences technology; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=272" /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Three sequenced legume genomes and many crop species: rich opportunities for translational genomics; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=970-7; " /><meta name="citation_reference" content="citation_title=Compositional properties of homologous coding sequences from plants; J. Mol. Evol.;  citation_year=1998;  citation_volume=46;  citation_pages=45-53; " /><meta name="citation_reference" content="citation_title=DNA helix: the importance of being GC-rich; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=1838-44; " /><meta name="citation_reference" content="citation_title=GC/AT-content spikes as genomic punctuation marks; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=16855-60; " /><meta name="citation_reference" content="citation_title=Generation of expressed sequence tags (ESTs) for gene discovery and marker development in cultivated peanut; Crop Sci.;  citation_year=2005;  citation_volume=45;  citation_pages=346-53; " /><meta name="citation_reference" content="citation_title=The first set of EST resource for gene discovery and marker development in pigeonpea (Cajanus cajan L.); BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Computational identification and characterization of novel genes from legumes; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1179-97; " /><meta name="citation_reference" content="citation_title=Identification and characterization of lineage-specific genes within the Poaceae; Plant Physiol.;  citation_year=2007;  citation_volume=145;  citation_pages=1311-22; " /><meta name="citation_reference" content="citation_title=Comparative analyses reveal distinct sets of lineage-specific genes within Arabidopsis thaliana; BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=41" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/1/53/1071414/dsq028.pdf" /><meta name="description" content="Abstract.  Chickpea ranks third among the food legume crops production in the world. However, the genomic resources available for chickpea are still very limite" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/1/53/377009" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/1/53/377009" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/1/53/377009");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq028';
    </script>
<title>
	De Novo Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./377009" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf1UPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYXJnLCBSb2hpbmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhdGVsLCBSYXZpIEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUeWFnaSwgQWtoaWxlc2ggSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkphaW4sIE11a2VzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZSBOb3ZvIEFzc2VtYmx5IG9mIENoaWNrcGVhIFRyYW5zY3JpcHRvbWUgVXNpbmcgU2hvcnQgUmVhZHMgZm9yIEdlbmUgRGlzY292ZXJ5IGFuZCBNYXJrZXIgSWRlbnRpZmljYXRpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3EwMjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZXM6IGltcG9ydGFuY2UgYW5kIGNvbnN0cmFpbnRzIHRvIGdyZWF0ZXIgdXNlOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz04NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hpY2twZWEgbW9sZWN1bGFyIGJyZWVkaW5nOiBuZXcgdG9vbHMgYW5kIGNvbmNlcHRzOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9ODEtMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcy1hc3Npc3RlZCBicmVlZGluZyBmb3IgY3JvcCBpbXByb3ZlbWVudDsgVHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIGNlcmVhbCBnZW5vbWljcyBhbmQgYXBwbGljYXRpb25zIGluIGNyb3AgYnJlZWRpbmc7IFRyZW5kcyBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ5MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBFU1RzIGZyb20gY2hpY2twZWEgcm9vdHMgYW5kIHRoZWlyIHVzZSBpbiBkaXZlcnNpdHkgYW5hbHlzaXMgb2YgdGhlIENpY2VyIGdlbnVzOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgRVNUcyBpbiByZXNwb25zZSB0byBkcm91Z2h0IHN0cmVzcyBpbiBjaGlja3BlYSAoQy4gYXJpZXRpbnVtIEwuKTsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM3NjsgIGNpdGF0aW9uX3BhZ2VzPTU3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzZXMgb2YgZ2Vub3R5cGUgZGVwZW5kZW50IGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIGFuZCBzdHJlc3MtcmVzcG9uc2l2ZSB0cmFuc2NyaXB0b21lIG9mIGNoaWNrcGVhIHdpbHQgaWxsdXN0cmF0ZSBwcmVkaWN0ZWQgYW5kIHVuZXhwZWN0ZWQgZ2VuZXMgYW5kIG5vdmVsIHJlZ3VsYXRvcnMgb2YgcGxhbnQgaW1tdW5pdHk7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSByZXNvdXJjZSBvZiBkcm91Z2h0LSBhbmQgc2FsaW5pdHktcmVzcG9uc2l2ZSBFU1RzIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGRldmVsb3BtZW50IGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBnZW5lIGV4cHJlc3Npb24gaW4gcmVzcG9uc2UgdG8gd2F0ZXIgZGVmaWNpdCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSB2YXJpZXRpZXMgZGlmZmVyaW5nIGluIGRyb3VnaHQgdG9sZXJhbmNlOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBwcm9maWxpbmcgb2YgY2hpY2twZWEgZ2VuZXMgZGlmZmVyZW50aWFsbHkgcmVndWxhdGVkIGluIHJlc3BvbnNlIHRvIGhpZ2gtc2FsaW5pdHksIGNvbGQgYW5kIGRyb3VnaHQ7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MzAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwZXJTQUdFOiB0aGUgZHJvdWdodCBzdHJlc3MtcmVzcG9uc2l2ZSB0cmFuc2NyaXB0b21lIG9mIGNoaWNrcGVhIHJvb3RzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgQ0FQMiwgYW4gQVBFVEFMQTItZmFtaWx5IHRyYW5zY3JpcHRpb24gZmFjdG9yIGZyb20gY2hpY2twZWEsIGVuaGFuY2VzIGdyb3d0aCBhbmQgdG9sZXJhbmNlIHRvIGRlaHlkcmF0aW9uIGFuZCBzYWx0IHN0cmVzcyBpbiB0cmFuc2dlbmljIHRvYmFjY287IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0lQSzYsIGEgQ0JMLWludGVyYWN0aW5nIHByb3RlaW4ga2luYXNlIGlzIHJlcXVpcmVkIGZvciBkZXZlbG9wbWVudCBhbmQgc2FsdCB0b2xlcmFuY2UgaW4gcGxhbnRzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTc3OC05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb25zIG9mIG5ldyBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llcyBmb3IgdHJhbnNjcmlwdG9tZSBhbmFseXNpczsgQW5udS4gUmV2LiBHZW5vbWljcyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEzNS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgdHJhbnNjcmlwdG9tZSBjaGFyYWN0ZXJpemF0aW9uIGZvciBhIG5vbm1vZGVsIG9yZ2FuaXNtIHVzaW5nIDQ1NCBweXJvc2VxdWVuY2luZzsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE2MzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgRVNUIGRhdGFzZXQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgRVNULVNTUnMgaW4gYSB0cmFkaXRpb25hbCBDaGluZXNlIG1lZGljaW5hbCBwbGFudCwgRXBpbWVkaXVtIHNhZ2l0dGF0dW0gKFNpZWIuIEV0IFp1Y2MuKSBNYXhpbTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZWx2ZXQ6IGFsZ29yaXRobXMgZm9yIGRlIG5vdm8gc2hvcnQgcmVhZCBhc3NlbWJseSB1c2luZyBkZSBCcnVpam4gZ3JhcGhzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BQnlTUzogYSBwYXJhbGxlbCBhc3NlbWJsZXIgZm9yIHNob3J0IHJlYWQgc2VxdWVuY2UgZGF0YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbCBzaG9ydCBzZXF1ZW5jZSBhc3NlbWJseSBvZiB0cmFuc2NyaXB0b21lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25faXNzdWU9U3VwcGwgMTsgIGNpdGF0aW9uX3BhZ2VzPVMxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gdHJhbnNjcmlwdG9tZSBhc3NlbWJseSB3aXRoIEFCeVNTOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI4NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFsaWRhdGlvbiBvZiBpbnRlcm5hbCBjb250cm9sIGdlbmVzIGZvciBxdWFudGl0YXRpdmUgZ2VuZSBleHByZXNzaW9uIHN0dWRpZXMgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zOTY7ICBjaXRhdGlvbl9wYWdlcz0yODMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIGNsYXNzZXMgb2YgZ2VuZXMgaW4gcGxhbnRzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTQ7ICBjaXRhdGlvbl9wYWdlcz0xODE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBsYW50czogZmVhdHVyZXMgYW5kIGFwcGxpY2F0aW9uczsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDgtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVndW1lIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmVzOiB3aGF0IG1ha2VzIGxlZ3VtZXMgc28gc3BlY2lhbDsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9OTkxLTEwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcGFsYWVvcG9seXBsb2lkIHNveWJlYW47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyBnb2VzIGRpZ2l0YWw7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz04NzgtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQS1TZXE6IGEgcmV2b2x1dGlvbmFyeSB0b29sIGZvciB0cmFuc2NyaXB0b21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01Ny02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdHJhbnNmb3JtcyB0b2RheSdzIGJpb2xvZ3k7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmVuY2htYXJraW5nIG5leHQtZ2VuZXJhdGlvbiB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgZm9yIGZ1bmN0aW9uYWwgYW5kIGV2b2x1dGlvbmFyeSBnZW5vbWljczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI3MzEtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgTWVkaWNhZ28gdHJ1bmNhdHVsYSBleHByZXNzZWQgc2VxdWVuY2VkIHRhZ3MgdXNpbmcgNDU0IExpZmUgU2NpZW5jZXMgdGVjaG5vbG9neTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBsZWd1bWUsIExvdHVzIGphcG9uaWN1czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRocmVlIHNlcXVlbmNlZCBsZWd1bWUgZ2Vub21lcyBhbmQgbWFueSBjcm9wIHNwZWNpZXM6IHJpY2ggb3Bwb3J0dW5pdGllcyBmb3IgdHJhbnNsYXRpb25hbCBnZW5vbWljczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9OTcwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgcHJvcGVydGllcyBvZiBob21vbG9nb3VzIGNvZGluZyBzZXF1ZW5jZXMgZnJvbSBwbGFudHM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz00NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGhlbGl4OiB0aGUgaW1wb3J0YW5jZSBvZiBiZWluZyBHQy1yaWNoOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xODM4LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HQy9BVC1jb250ZW50IHNwaWtlcyBhcyBnZW5vbWljIHB1bmN0dWF0aW9uIG1hcmtzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xNjg1NS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBleHByZXNzZWQgc2VxdWVuY2UgdGFncyAoRVNUcykgZm9yIGdlbmUgZGlzY292ZXJ5IGFuZCBtYXJrZXIgZGV2ZWxvcG1lbnQgaW4gY3VsdGl2YXRlZCBwZWFudXQ7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTM0Ni01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZpcnN0IHNldCBvZiBFU1QgcmVzb3VyY2UgZm9yIGdlbmUgZGlzY292ZXJ5IGFuZCBtYXJrZXIgZGV2ZWxvcG1lbnQgaW4gcGlnZW9ucGVhIChDYWphbnVzIGNhamFuIEwuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG5vdmVsIGdlbmVzIGZyb20gbGVndW1lczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM1OyAgY2l0YXRpb25fcGFnZXM9MTE3OS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgbGluZWFnZS1zcGVjaWZpYyBnZW5lcyB3aXRoaW4gdGhlIFBvYWNlYWU7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTEzMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuYWx5c2VzIHJldmVhbCBkaXN0aW5jdCBzZXRzIG9mIGxpbmVhZ2Utc3BlY2lmaWMgZ2VuZXMgd2l0aGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz00MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOC8xLzUzLzEwNzE0MTQvZHNxMDI4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBDaGlja3BlYSByYW5rcyB0aGlyZCBhbW9uZyB0aGUgZm9vZCBsZWd1bWUgY3JvcHMgcHJvZHVjdGlvbiBpbiB0aGUgd29ybGQuIEhvd2V2ZXIsIHRoZSBnZW5vbWljIHJlc291cmNlcyBhdmFpbGFibGUgZm9yIGNoaWNrcGVhIGFyZSBzdGlsbCB2ZXJ5IGxpbWl0ZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzc3MDA5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFGH/Uvsi7x7JRKADx5qv9nAUy8D" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f53%2f377009" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f53%2f377009" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14695" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&amp;Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/1">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5495224">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495224">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495228">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495228">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495238">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495238">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495276">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495357">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495357">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495371">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495371">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495374">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495374">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5495376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5495376">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/1/39/378044">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/1/65/377199">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>De Novo</em> Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rohini Garg</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rohini+Garg">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Garg R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Garg R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rohini Garg</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ravi K. Patel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ravi+K.+Patel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Patel R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Patel R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ravi K. Patel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tyagi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mukesh Jain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg</div>, <div class="addr-line">New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91-11-26735182; Fax. +91-11-26741658. E-mail: <a href="mailto:mjain@nipgr.res.in" target="_blank">mjain@nipgr.res.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mukesh+Jain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jain M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jain M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mukesh Jain</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 53–63, <a href='https://doi.org/10.1093/dnares/dsq028'>https://doi.org/10.1093/dnares/dsq028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 January 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 September 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 November 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="377009" href="/dnaresearch/article-pdf/18/1/53/1071414/dsq028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rohini Garg, Ravi K. Patel, Akhilesh K. Tyagi, Mukesh Jain;  <em>De Novo</em> Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification, <em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 53–63, <a href='https://doi.org/10.1093/dnares/dsq028'>https://doi.org/10.1093/dnares/dsq028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/377009?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/377009?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BDe%20Novo%26lt%3B%2Fem%26gt%3B%20Assembly%20of%20Chickpea%20Transcriptome%20Using%20Short%20Reads%20for%20Gene%20Discovery%20and%20Marker%20Identification&amp;publicationDate=2011-01-07&amp;volumeNum=18&amp;issueNum=1&amp;author=Garg%2C%20Rohini%3B%20Patel%2C%20Ravi%20K.&amp;startPage=53&amp;endPage=63&amp;contentId=10.1093%2Fdnares%2Fdsq028&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f53%2f377009" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5495224 id="5495224" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Chickpea ranks third among the food legume crops production in the world. However, the genomic resources available for chickpea are still very limited. In the present study, the transcriptome of chickpea was sequenced with short reads on Illumina Genome Analyzer platform. We have assessed the effect of sequence quality, various assembly parameters and assembly programs on the final assembly output. We assembled ∼107million high-quality trimmed reads using Velvet followed by Oases with optimal parameters into a non-redundant set of 53 409 transcripts (≥100 bp), representing about 28 Mb of unique transcriptome sequence. The average length of transcripts was 523 bp and N50 length of 900 bp with coverage of 25.7 rpkm (reads per kilobase per million). At the protein level, a total of 45 636 (85.5%) chickpea transcripts showed significant similarity with unigenes/predicted proteins from other legumes or sequenced plant genomes. Functional categorization revealed the conservation of genes involved in various biological processes in chickpea. In addition, we identified simple sequence repeat motifs in transcripts. The chickpea transcripts set generated here provides a resource for gene discovery and development of functional molecular markers. In addition, the strategy for <em>de novo</em> assembly of transcriptome data presented here will be helpful in other similar transcriptome studies.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;De novo assembly&quot;"><em>De novo</em> assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chickpea">chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;short read&quot;">short read</a></div>                            <h2 scrollto-destination=5495228 id="5495228" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Chickpea (<em>Cicer arietinum</em> L.) is the third most consumed legume crop, which is cultivated in arid and semi-arid areas around the world.<sup><a reveal-id="DSQ028C1" data-open="DSQ028C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Chickpea is a self-pollinated, diploid (2<em>n</em> = 2<em>x</em> = 16) and annual plant with a moderate genome size of about 740 Mb. Despite growing demand and high-yield potential, chickpea productivity is very low. Several biotic such as <em>Ascochyta</em> blight, dry root rot, <em>Fusarium</em> wilt and pod borer, and abiotic such as drought, salinity and low temperature, constraints are major factors for lower chickpea production. Modern breeding technologies with biotechnological techniques are required to increase the productivity.<sup><a reveal-id="DSQ028C2" data-open="DSQ028C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Unfortunately, very limited genomic information is available for chickpea.</p><p>Various genomic tools have facilitated greatly the development of improved genotypes/varieties in several crop species.<sup><a reveal-id="DSQ028C3" data-open="DSQ028C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ028C4" data-open="DSQ028C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Although a few expressed sequence tags (ESTs) have been generated and gene-based markers have been developed, the functional genomics studies in chickpea is still in its infancy. Most of the ESTs have been generated with the aim to identify the candidate genes involved in various abiotic and biotic stress responses and development of molecular markers.<sup><a reveal-id="DSQ028C5  DSQ028C9" data-open="DSQ028C5  DSQ028C9" class="link link-ref link-reveal xref-bibr">5–9</a></sup> In addition, other microarray and SAGE technologies have also been used to identify the stress-responsive transcriptome in chickpea.<sup><a reveal-id="DSQ028C10" data-open="DSQ028C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSQ028C11" data-open="DSQ028C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The efforts have been made to clone genes of interest via candidate gene approach.<sup><a reveal-id="DSQ028C2" data-open="DSQ028C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The function of a few genes in stress responses has also been demonstrated using transgenic approach.<sup><a reveal-id="DSQ028C12" data-open="DSQ028C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSQ028C13" data-open="DSQ028C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>The generation of large-scale ESTs is a very useful approach to accelerate the research on non-model species. Although ESTs and other cDNA sequences are among the most reliable evidences for the identification of gene-rich regions in a genome, gene identification and genome annotation, very less effort has been made for chickpea in this direction when compared with other crop plants. This is reflected by a very small number of ESTs (34 587) present in the dbEST database at NCBI (release 100110; 1 October 2010) for chickpea. The next generation sequencing technologies provide a cost-effective means of sequencing the transcriptome of an organism.<sup><a reveal-id="DSQ028C14" data-open="DSQ028C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Several studies have reported the transcriptome sequencing of various model and non-model species using these technologies. However, most of these studies are based on the long-read sequence data using 454 pyrosequencing or employing hybrid approach.<sup><a reveal-id="DSQ028C15" data-open="DSQ028C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSQ028C16" data-open="DSQ028C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Although the efforts have been made to develop tools for <em>de novo</em> assembly using short-read sequence data,<sup><a reveal-id="DSQ028C17  DSQ028C19" data-open="DSQ028C17  DSQ028C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> their use in transcriptome assembly has not been well demonstrated yet. Recently, the <em>de novo</em> assembly of human transcriptome has been reported using ABySS program.<sup><a reveal-id="DSQ028C20" data-open="DSQ028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Here, we present a <em>de novo</em> assembly approach for transcriptome of a plant species using only short-read sequence data.</p><p>The present study has two major goals. First, we report a strategy for <em>de novo</em> assembly of transcriptome using short-read sequence data and effect of sequence quality and various parameters. Secondly, we report for the first time the complete transcriptome of chickpea, the legume crop plant. We have generated millions of sequence reads from chickpea transcriptome sequencing. A non-redundant set of transcripts have been generated and various analyses, including GC content analysis, sequence similarity/conservation with other plant species, functional categorization and identification of simple sequence repeats (SSRs) have been done. Our data provide a very useful genomic resource for future studies in chickpea.</p>                            <h2 scrollto-destination=5495238 id="5495238" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5495241 id="5495241" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>Chickpea (<em>C. arietinum</em> L. genotype ICC4958) seeds procured from ICRISAT, Hyderabad, India, were grown as described.<sup><a reveal-id="DSQ028C21" data-open="DSQ028C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Root and shoot tissue samples were collected from the 15-day-old seedlings grown in autoclaved mixture (1:1) of agropeat and vermiculite in 3in. plastic pots at 22 ± 1°C in a culture room with a photoperiod of 14 h. The mature leaves and flower buds were collected from plants grown in the field. At least three independent biological replicates of each tissue sample were harvested and immediately frozen in liquid nitrogen.</p>                            <h3 scrollto-destination=5495245 id="5495245" class="section-title" data-legacy-id=s2b>2.2. RNA isolation and quality controls</h3>
<p>Total RNA was extracted from all the tissue samples using TRI Reagent (Sigma Life Science, USA) according to manufacturer's instructions. The quality and quantity of each RNA sample was assessed as described previously.<sup><a reveal-id="DSQ028C21" data-open="DSQ028C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Only the RNA samples with 260 of 280 ratio from 1.9 to 2.1, 260 of 230 ratio from 2.0 to 2.5 and RIN (RNA integrity number) more than 8.0, were used for the analysis.</p>                            <h3 scrollto-destination=5495247 id="5495247" class="section-title" data-legacy-id=s2c>2.3. Illumina sequencing and quality controls</h3>
<p>Three cDNA libraries were generated using mRNA-Seq assay for transcriptome sequencing on Illumina Genome Analyzer II platform. One paired-end (PE) cDNA library was generated from the pooled total RNA of shoot, root, mature leaf and flower buds in equal quantity and sequencing was done in one lane to generate 72 bp PE reads. Two cDNA libraries were generated one each from total RNA of root and shoot tissues and sequencing was done in one lane each to generate 51 bp single-end (SE) reads. The library construction and sequencing was performed by commercial service providers (PE, Genotypic Technology, Bangalore, India; SE, BC Cancer Agency Genome Sciences Centre, Vancouver, Canada). The sequence data generated in this study have been deposited at NCBI in the Short Read Archive database under the accession number SRA023503 (experiment accession numbers SRX025413 and SRX025414 for PE and SE read sequencing, respectively). Various quality controls, including filtering of high-quality reads based on the score value given in fastq files, removal of reads containing primer/adaptor sequences and trimming of read length were done using in-house tool kit (Patel and Jain, unpublished). We evaluated the effect of sequence quality on the <em>de novo</em> assembly of sequence reads.</p>                            <h3 scrollto-destination=5495252 id="5495252" class="section-title" data-legacy-id=s2d>2.4. <em>De novo</em> assembly</h3>
<p>All the assemblies were performed on a server with 48cores and 128GB random access memory. We used various programs for <em>de novo</em> assembly of the PE and SE sequence reads to generate a non-redundant set of transcripts. Among the various programs available, we validated publicly available program, Velvet (version 0.7.62; <a class="link link-uri" href="http://www.ebi.ac.uk/~zerbino/velvet/">http://www.ebi.ac.uk/~zerbino/velvet/</a>), which have been developed for assembly of short reads using de Bruijn graph algorithm.<sup><a reveal-id="DSQ028C17" data-open="DSQ028C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Various assembly parameters were also optimized for best results. In addition, we used other publicly available programs, including Oases (version 0.1.8; <a class="link link-uri" href="http://www.ebi.ac.uk/~zerbino/oases/">http://www.ebi.ac.uk/~zerbino/oases/</a>), ABySS (version 1.1.2; <a class="link link-uri" href="http://www.bcgsc.ca/platform/bioinfo/software/abyss)">http://www.bcgsc.ca/platform/bioinfo/software/abyss)</a><sup><a reveal-id="DSQ028C18" data-open="DSQ028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and SOAPdenovo (version 1.04; <a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>), and commercially available CLC Genomics workbench (version 3.7.1), which have also been developed for <em>de novo</em> assembly of short reads, to obtain best assembly results with our data set.</p>                            <h3 scrollto-destination=5495256 id="5495256" class="section-title" data-legacy-id=s2e>2.5. Similarity search and functional annotation</h3>
<p>The proteome data sets for all the completely sequenced plant genomes so far were downloaded from their respective genome project websites. For generating non-redundant unigene data sets from various legume species, including <em>Glycine max</em> (soybean), <em>Medicago truncatula</em>, <em>Lotus japonicus</em>, <em>Vigna unguiculata</em> and <em>Pisum sativum</em>, all the available EST and mRNA sequences were downloaded from Genbank and assembled using TGI Clustering Tool (TGICL) after removing/trimming contaminating vector sequences and short reads (&lt;100 bp) using SeqClean. The parameters used for assembly were more than 95% identity over a minimum of 40 bases with maximum of 20 bases of unmatched overhangs at sequence end. The chickpea transcripts were searched against proteome sequences and legume unigenes sets using BLASTX and TBLASTX searches, respectively, with an expect (E)-value cut-off of ≤1E−05 to reveal sequence conservation.</p><p>To deduce the putative function, chickpea non-redundant transcript data set was subjected to BLASTX analysis against the non-redundant protein database of UniProt and all annotated protein sequences of <em>Arabidopsis</em> (available at The Arabidopsis Information Resource). The results of only the best hit were extracted and the hits with an <em>E</em>-value ≤ 1E−05 were considered to be significant. The GOSlim terms for molecular function, biological process, and cellular component categories associated with the best BLASTX hit with <em>Arabidopsis</em> protein were assigned to the corresponding chickpea transcript. For the identification of transcription factor families represented in chickpea transcriptome, the chickpea transcripts were searched against all the transcription factor protein sequences at Plant transcription factor database (PlnTFDB; <a class="link link-uri" href="http://plntfdb.bio.uni-potsdam.de/v3.0/downloads.php">http://plntfdb.bio.uni-potsdam.de/v3.0/downloads.php</a>) using BLASTX with an <em>E</em>-value cut-off of ≤1E−05.</p>                            <h3 scrollto-destination=5495263 id="5495263" class="section-title" data-legacy-id=s2f>2.6. GC content analysis and SSRs identification</h3>
<p>GC content analysis was done using in-house perl scripts. The perl script program MISA (MIcroSAtellite; <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>) was used for identification of SSRs. The repeats of mono-nucleotide more than 10 times, di-nucleotides repeats more than 6 times, tri-, tetra-, penta- and hexa-nucleotide repeats more than 5 times were considered as search criteria in MISA script.</p>                            <h3 scrollto-destination=5495270 id="5495270" class="section-title" data-legacy-id=s2g>2.7. Mapping of sequence reads onto chickpea transcripts</h3>
<p>We mapped all the reads from three experiments onto the non-redundant set of transcripts to quantify the abundance of transcripts assembled, using CLC Genomics Workbench and Maq (v0.7.1; <a class="link link-uri" href="http://maq.sourceforge.net/index.shtml">http://maq.sourceforge.net/index.shtml</a>) softwares and the number of reads and reads per million (rpm) corresponding to each transcript were determined. In addition, the coverage of each transcript was determined in terms of number of reads per kilobase per million (rpkm).</p>                            <h2 scrollto-destination=5495276 id="5495276" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5495278 id="5495278" class="section-title" data-legacy-id=s3a>3.1. Sequencing of chickpea transcriptome</h3>
<p>We generated a total of 134 954 354 sequence reads, including 65 900 072 PE sequence reads (32 950 036 from each end) each 72 bp in length and 69 054 282 SE sequence reads each 51 bp in length, encompassing about 21GB of sequence data in fastq format (Table <a reveal-id="DSQ028TB1" data-open="DSQ028TB1" class="link link-reveal link-table xref-fig">1</a>). We filtered the sequence data for low-quality reads at high stringency (reads with more than 30% of bases with Phred quality score of ≤20) and reads containing primer/adaptor sequence. This resulted in a total of 106 660 317 (79%) high-quality sequence reads, including 50 523 492 (76.7%) PE sequence reads each 72 bp in length and 56 136 825 (81.3%) SE sequence reads each 51 bp in length (Table <a reveal-id="DSQ028TB1" data-open="DSQ028TB1" class="link link-reveal link-table xref-fig">1</a>). After filtering, the average quality score increased significantly at each base position of the sequence reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The average quality score was more than 30 at each base position in all the three data sets except for last seven bases at the 3′ end in PE data set. The final data set comprising ∼107million very high-quality reads was used for optimization of <em>de novo</em> assembly and analysis of chickpea transcriptome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). </p>                        <a id="5495280" scrollto-destination="5495280"></a>
<div content-id="DSQ028TB1" class="table-modal"><div class="table-wrap"><div id="DSQ028TB1" class="table-wrap-title" data-id="DSQ028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of data generated for chickpea transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library </th><th>Total no. of reads </th><th>Fastq file size (GB) </th><th>No. of reads after filtering low-quality reads </th><th>No. of reads after removing primer/adapter containing reads </th></tr></thead><tbody><tr><td>Pooled (72 bp PE) </td><td>65 900 072 </td><td>11.44 </td><td>52 430 156 </td><td>50 523 492 </td></tr><tr><td>Root (51 bp SE) </td><td>31 028 774 </td><td>4.06 </td><td>24 671 859 </td><td>24 670 440 </td></tr><tr><td>Shoot (51 bp SE) </td><td>38 025 508 </td><td>5.17 </td><td>31 468 818 </td><td>31 466 385 </td></tr><tr><td>Total </td><td>134 954 354 </td><td>20.67 </td><td>108 570 833 </td><td>106 660 317 </td></tr></tbody></table></div><div content-id="DSQ028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library </th><th>Total no. of reads </th><th>Fastq file size (GB) </th><th>No. of reads after filtering low-quality reads </th><th>No. of reads after removing primer/adapter containing reads </th></tr></thead><tbody><tr><td>Pooled (72 bp PE) </td><td>65 900 072 </td><td>11.44 </td><td>52 430 156 </td><td>50 523 492 </td></tr><tr><td>Root (51 bp SE) </td><td>31 028 774 </td><td>4.06 </td><td>24 671 859 </td><td>24 670 440 </td></tr><tr><td>Shoot (51 bp SE) </td><td>38 025 508 </td><td>5.17 </td><td>31 468 818 </td><td>31 466 385 </td></tr><tr><td>Total </td><td>134 954 354 </td><td>20.67 </td><td>108 570 833 </td><td>106 660 317 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495280" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ028TB1" class="table-wrap-title" data-id="DSQ028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of data generated for chickpea transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library </th><th>Total no. of reads </th><th>Fastq file size (GB) </th><th>No. of reads after filtering low-quality reads </th><th>No. of reads after removing primer/adapter containing reads </th></tr></thead><tbody><tr><td>Pooled (72 bp PE) </td><td>65 900 072 </td><td>11.44 </td><td>52 430 156 </td><td>50 523 492 </td></tr><tr><td>Root (51 bp SE) </td><td>31 028 774 </td><td>4.06 </td><td>24 671 859 </td><td>24 670 440 </td></tr><tr><td>Shoot (51 bp SE) </td><td>38 025 508 </td><td>5.17 </td><td>31 468 818 </td><td>31 466 385 </td></tr><tr><td>Total </td><td>134 954 354 </td><td>20.67 </td><td>108 570 833 </td><td>106 660 317 </td></tr></tbody></table></div><div content-id="DSQ028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library </th><th>Total no. of reads </th><th>Fastq file size (GB) </th><th>No. of reads after filtering low-quality reads </th><th>No. of reads after removing primer/adapter containing reads </th></tr></thead><tbody><tr><td>Pooled (72 bp PE) </td><td>65 900 072 </td><td>11.44 </td><td>52 430 156 </td><td>50 523 492 </td></tr><tr><td>Root (51 bp SE) </td><td>31 028 774 </td><td>4.06 </td><td>24 671 859 </td><td>24 670 440 </td></tr><tr><td>Shoot (51 bp SE) </td><td>38 025 508 </td><td>5.17 </td><td>31 468 818 </td><td>31 466 385 </td></tr><tr><td>Total </td><td>134 954 354 </td><td>20.67 </td><td>108 570 833 </td><td>106 660 317 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495280" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5495281 id="5495281" class="section-title" data-legacy-id=s3b>3.2. <em>De novo</em> assembly</h3>
<p>The <em>de novo</em> assembly of chickpea transcriptome was optimized after assessing the effect of various assembly parameters, trimming bases at sequence read ends and different assembly programs as described below.</p>                            <h4 scrollto-destination=5495283 id="5495283" class="section-title" data-legacy-id=s3b1>3.2.1. Assessment of the effect of assembly parameters</h4>
<p>The untrimmed high-quality sequence reads were assembled using Velvet program at different <em>k</em>-mer length of 21, 27, 31, 37, 41, 47, 51 and 57. We analysed various output parameters like number of used reads, nodes, total number of contigs, contigs longer than 100 bp, N50 length, longest contig length and average contig length as a function of <em>k</em>-mer length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1A</a>; Fig. <a reveal-id="DSQ028F1" data-open="DSQ028F1" class="link link-reveal link-table xref-fig">1</a>). The results suggested that <em>k</em>-mer length affects inversely to the number of contigs. We found the best assembly to be that for <em>k</em> = 47, as it resulted in highest N50 length of 675 bp, largest contig length of 7827 bp and largest average contig length of 432 bp (Fig. <a reveal-id="DSQ028F1" data-open="DSQ028F1" class="link link-reveal link-table xref-fig">1</a>A). The assembly resulted in a total of 74 651 contigs of at least 100 bp length. The total number of reads used for the assembly was also highest (73.6%) for <em>k</em> = 47 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). We assessed the effect of other parameters such as insert length and expected coverage on Velvet assembly at different <em>k</em>-mer length and did not find any significant effect (data not shown). </p>                        <a id="5495285" scrollto-destination="5495285"></a>
<div id="DSQ028F1" data-id="DSQ028F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02801.jpeg?Expires=2147483647&Signature=k~CC~bhQ9ui5mcK~qcBMqMh6kaWf17mp9nSMiFGDpDQINL4NRC0NvgIm7C7xUVI3Kort2OExk1nUeWfUm2xYuKO7RI4F-Mu4D6MQtvAMBwBAbp1K84MMuP05DZlxCY1WkneBijdvQi8Iw0M-93Gn3-53KGaRqGEZzKobmREuZ35blsKl4MEDKJksXX1dkVETJ49DnQcAdyQKghdtiU3ECeXH0cM3KNAz5Eh3moOnPYvjwwgOYQ8ZuynzuSsqqO14UVc4AO1DoJBUWQtt2OL47gj5MZwyczpHbd1~c8~2UkZJ~EKxjgvVoxFar6VFTftaRzY7bZZi3Ngf9II1DqYKXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of de novo assembly of three data sets using Velvet program as a function of k-mer length. The three data sets include (A) untrimmed data set (72 bp PE and 51 bp SE reads), (B) trimmed data set 1 (70 bp PE and 50 bp SE reads) and (C) trimmed data set 2 (65 bp PE and 50 bp SE reads). The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis)." path-from-xml="dsq02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495285/dsq02801.jpeg" path-from-xml="dsq02801.jpeg" target="_blank">View large</a><a section="5495285" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02801.jpeg?Expires=2147483647&Signature=3vXWz5SK1j6cBXv6XLhd2dbpDxYiu0-gICaK-~PKF9IJw4t7gVhEDGGSqzPyzW09iACmvb3ODnJBKNwLp1RKF~qc1lfwBMJ-5-W3Y5IeIuoM4YtAPdVix4vF3XWjiOeV0MAk-DwhegS8Q8vCi4gG80LqEE6~c9RI6EyQx09XgkSNlv1BnBNx8DWPYGFCSYVLFB0RbLifiA9DIiNFOVhTrYfcpZvn1V1~uQdWM-1DEV5w-tGspDWnPcpkhsW1Il~YuhcJqu9Xe2qXD3Bys5KczYMTl5sDXtr0Ka6gnyBlw2t~jQy8-KIB5R15l0yYajguwyOIiQ5SVsdInEW3vUv8rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495285&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of <em>de novo</em> assembly of three data sets using Velvet program as a function of <em>k</em>-mer length. The three data sets include (A) untrimmed data set (72 bp PE and 51 bp SE reads), (B) trimmed data set 1 (70 bp PE and 50 bp SE reads) and (C) trimmed data set 2 (65 bp PE and 50 bp SE reads). The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis).</p></div></div><div content-id="DSQ028F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02801.jpeg?Expires=2147483647&Signature=k~CC~bhQ9ui5mcK~qcBMqMh6kaWf17mp9nSMiFGDpDQINL4NRC0NvgIm7C7xUVI3Kort2OExk1nUeWfUm2xYuKO7RI4F-Mu4D6MQtvAMBwBAbp1K84MMuP05DZlxCY1WkneBijdvQi8Iw0M-93Gn3-53KGaRqGEZzKobmREuZ35blsKl4MEDKJksXX1dkVETJ49DnQcAdyQKghdtiU3ECeXH0cM3KNAz5Eh3moOnPYvjwwgOYQ8ZuynzuSsqqO14UVc4AO1DoJBUWQtt2OL47gj5MZwyczpHbd1~c8~2UkZJ~EKxjgvVoxFar6VFTftaRzY7bZZi3Ngf9II1DqYKXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of de novo assembly of three data sets using Velvet program as a function of k-mer length. The three data sets include (A) untrimmed data set (72 bp PE and 51 bp SE reads), (B) trimmed data set 1 (70 bp PE and 50 bp SE reads) and (C) trimmed data set 2 (65 bp PE and 50 bp SE reads). The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis)." path-from-xml="dsq02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495285/dsq02801.jpeg" path-from-xml="dsq02801.jpeg" target="_blank">View large</a><a section="5495285" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02801.jpeg?Expires=2147483647&Signature=3vXWz5SK1j6cBXv6XLhd2dbpDxYiu0-gICaK-~PKF9IJw4t7gVhEDGGSqzPyzW09iACmvb3ODnJBKNwLp1RKF~qc1lfwBMJ-5-W3Y5IeIuoM4YtAPdVix4vF3XWjiOeV0MAk-DwhegS8Q8vCi4gG80LqEE6~c9RI6EyQx09XgkSNlv1BnBNx8DWPYGFCSYVLFB0RbLifiA9DIiNFOVhTrYfcpZvn1V1~uQdWM-1DEV5w-tGspDWnPcpkhsW1Il~YuhcJqu9Xe2qXD3Bys5KczYMTl5sDXtr0Ka6gnyBlw2t~jQy8-KIB5R15l0yYajguwyOIiQ5SVsdInEW3vUv8rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495285&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of <em>de novo</em> assembly of three data sets using Velvet program as a function of <em>k</em>-mer length. The three data sets include (A) untrimmed data set (72 bp PE and 51 bp SE reads), (B) trimmed data set 1 (70 bp PE and 50 bp SE reads) and (C) trimmed data set 2 (65 bp PE and 50 bp SE reads). The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis).</p></div></div>                            <h4 scrollto-destination=5495286 id="5495286" class="section-title" data-legacy-id=s3b2>3.2.2. Assessment of the effect of trimming</h4>
<p>To improve the quality of assembly and observe the effect of trimming of low-quality bases at the end of reads, we generated two trimmed data sets; first data set containing 70 bp PE (2 bp trimmed from 3′ end) and 50 bp SE (1 bp trimmed from 3′ end) sequence reads and second data set containing 65 bp PE (7 bp trimmed from 3′ end) and 50 bp SE (1 bp trimmed from 3′ end) sequence reads. We then assembled both these data sets also with different <em>k</em>-mer length using Velvet and noted various output parameters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1B and C</a>; Fig. <a reveal-id="DSQ028F1" data-open="DSQ028F1" class="link link-reveal link-table xref-fig">1</a>B and C). For first trimmed data set, the best assembly was for <em>k</em> = 47 with improved highest N50 length of 683 bp, largest contig length of 7666 bp and average contig length of 437 bp. The total number of contigs generated with at least 100 bp in length was 71 217 using 75% of the total reads. For the second trimmed data set, although the largest contig length was highest for <em>k</em> = 41 (10 616 bp), N50 length of 730 bp and average contig length of 453 bp were better for <em>k</em> = 47. The total number of contigs generated with at least 100 bp in length was 63 365 at <em>k</em> = 47 using highest number of total reads (77.8%). Taken together, we considered the assembly results of second trimmed data set for <em>k</em> = 47 as the best. The results show that the trimming of low-quality bases at the sequence read ends improved the assembly significantly.</p>                            <h4 scrollto-destination=5495288 id="5495288" class="section-title" data-legacy-id=s3b3>3.2.3. Assessment of assembly programs</h4>
<p>It has been suggested that assembly of Velvet followed by Oases yields better contigs/transcripts. The Oases program has been developed specifically for the <em>de novo</em> assembly of transcriptomes using short reads, which takes the assembly generated by Velvet as input and exploits the read sequence and pairing information to produce transcript isoforms. We performed assembly of contigs generated by Velvet for the second trimmed data set (<em>k</em> = 47) into transcripts using Oases with default parameters. This resulted in a total number of 59 178 transcript isoforms (≥100 bp in length). The number of reads used increased by 5% in Oases (82.8%) when compared with Velvet (77.8%). Among these, 4232 transcripts were represented by 10 001 transcript isoforms, which might represent alternative splicing events. From the 59 178 transcript isoforms obtained by Oases assembly, a set of 53 409 non-redundant transcripts (including only the largest transcript isoform) was obtained with N50 length of 900 bp, largest contig length of 8173 bp and average contig length of 523 bp (Fig. <a reveal-id="DSQ028F2" data-open="DSQ028F2" class="link link-reveal link-table xref-fig">2</a>A). In addition, we also performed assembly of above three data sets (untrimmed data set, trimmed data set 1 and trimmed data set 2) using CLC Genomics workbench. We obtained the best assembly results with the second trimmed data set with a total number of 113 893 contigs (≥100 bp in length) with N50 length of 1151 bp, largest contig length of 15 684 bp and average contig length of 428 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>; Fig. <a reveal-id="DSQ028F2" data-open="DSQ028F2" class="link link-reveal link-table xref-fig">2</a>A). Although N50 length was better in CLC Genomics Workbench, the average contig length was lower than that of Oases and total number of contigs generated was also much higher than expected (Fig. <a reveal-id="DSQ028F2" data-open="DSQ028F2" class="link link-reveal link-table xref-fig">2</a>A). Further, the assembly of second trimmed data set using ABySS (<em>k</em> = 47) and SOAPdenovo (<em>k</em> = 31) programs generated 48 185 and 124 160 contigs, respectively, of at least 100 bp length. The N50 and average lengths for contigs generated by ABySS were 1192 and 613 bp, respectively, and that of contigs generated by SOAPdenovo were 525 and 340 bp, respectively (Fig. <a reveal-id="DSQ028F2" data-open="DSQ028F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="5495290" scrollto-destination="5495290"></a>
<div id="DSQ028F2" data-id="DSQ028F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02802.jpeg?Expires=2147483647&Signature=YK4VEKtY2Idug8pkH~lwRFcuPta1FLsi05Lc1PhJFeKV197-KmlrJ2ndMFe2-YjQyNFQv3NwuD8P9Sp3l7L1Y6SmaD~v6FBBO1P~XQLo~JQUr97m13TGGXuC1ARZDKDGQ7H~LgxbaeJXcELvCAe6ul7gP9Lpp~9Sjnr3RzOEMv-GAD8PEs1IgidwSF-n~wqzr5tb0U9lls~0ebYxKJSFAUFisWSEDLWXltrFll8pWcctPuzotOHB5D78D0DGb2jtnoOCE45hZ5dfx9JfZ8VKbONodJ9PByUz789NlUWYNVkvyfxr8kkzyie1g6Ynl-cFBQHSWhBvGj4xMpcZzUZCuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison and validation of de novo assembly using various assembly programs. (A) Comparison of de novo assembly of trimmed data set 2 (65 bp PE and 50 bp SE reads) using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo. The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis). (B and C) Percentage of contigs generated using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo showing significant hits with Arabidopsis (B) and soybean (C) proteins at different E-value cut-offs." path-from-xml="dsq02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495290/dsq02802.jpeg" path-from-xml="dsq02802.jpeg" target="_blank">View large</a><a section="5495290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02802.jpeg?Expires=2147483647&Signature=jsA8DNKxBuLk-d9ATYutLeK89QWk2NhCMcvKuWx498pH4s0uxFjIpD~MFpLD60-wLmB1b4KVh6ASZ1wgkd~LAJuacraCIqW~JbWKN~~kgiPkoJmJ-Bq1cpV-R8FPIT3EUm1b162zEvSDIIFwfrrkb6RjBukXt3xK6TCPOTPIGNryXSdWP5P7CXqY3i46fweN~7y9K5gdtvGhRPsJgM4Hj-RCPQgswJ4zWI2Iy4qjZZvAHVsLHNjMKK9U1GXHfUhTm-uJTk25ct~YteQMRBiaNxrAlXwdZvc~71RERU-~BTdOIvHyE6W8B0MNuLuM~5BzoDQwQKVBak2kvWFBgoBZcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495290&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison and validation of <em>de novo</em> assembly using various assembly programs. (A) Comparison of <em>de novo</em> assembly of trimmed data set 2 (65 bp PE and 50 bp SE reads) using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo. The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis). (B and C) Percentage of contigs generated using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo showing significant hits with <em>Arabidopsis</em> (B) and soybean (C) proteins at different <em>E</em>-value cut-offs.</p></div></div><div content-id="DSQ028F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02802.jpeg?Expires=2147483647&Signature=YK4VEKtY2Idug8pkH~lwRFcuPta1FLsi05Lc1PhJFeKV197-KmlrJ2ndMFe2-YjQyNFQv3NwuD8P9Sp3l7L1Y6SmaD~v6FBBO1P~XQLo~JQUr97m13TGGXuC1ARZDKDGQ7H~LgxbaeJXcELvCAe6ul7gP9Lpp~9Sjnr3RzOEMv-GAD8PEs1IgidwSF-n~wqzr5tb0U9lls~0ebYxKJSFAUFisWSEDLWXltrFll8pWcctPuzotOHB5D78D0DGb2jtnoOCE45hZ5dfx9JfZ8VKbONodJ9PByUz789NlUWYNVkvyfxr8kkzyie1g6Ynl-cFBQHSWhBvGj4xMpcZzUZCuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison and validation of de novo assembly using various assembly programs. (A) Comparison of de novo assembly of trimmed data set 2 (65 bp PE and 50 bp SE reads) using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo. The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis). (B and C) Percentage of contigs generated using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo showing significant hits with Arabidopsis (B) and soybean (C) proteins at different E-value cut-offs." path-from-xml="dsq02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495290/dsq02802.jpeg" path-from-xml="dsq02802.jpeg" target="_blank">View large</a><a section="5495290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02802.jpeg?Expires=2147483647&Signature=jsA8DNKxBuLk-d9ATYutLeK89QWk2NhCMcvKuWx498pH4s0uxFjIpD~MFpLD60-wLmB1b4KVh6ASZ1wgkd~LAJuacraCIqW~JbWKN~~kgiPkoJmJ-Bq1cpV-R8FPIT3EUm1b162zEvSDIIFwfrrkb6RjBukXt3xK6TCPOTPIGNryXSdWP5P7CXqY3i46fweN~7y9K5gdtvGhRPsJgM4Hj-RCPQgswJ4zWI2Iy4qjZZvAHVsLHNjMKK9U1GXHfUhTm-uJTk25ct~YteQMRBiaNxrAlXwdZvc~71RERU-~BTdOIvHyE6W8B0MNuLuM~5BzoDQwQKVBak2kvWFBgoBZcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495290&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison and validation of <em>de novo</em> assembly using various assembly programs. (A) Comparison of <em>de novo</em> assembly of trimmed data set 2 (65 bp PE and 50 bp SE reads) using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo. The bars indicate number of contigs 100 bp or longer (left axis). The lines indicate N50 length (rectangles) and average contig length (triangles) in bp (right axis). (B and C) Percentage of contigs generated using Velvet, Oases, CLC Genomics Workbench, ABySS and SOAPdenovo showing significant hits with <em>Arabidopsis</em> (B) and soybean (C) proteins at different <em>E</em>-value cut-offs.</p></div></div><p>Further, the validation of assembly output was done by BLASTX search of contigs/transcripts generated by various programs against <em>Arabidopsis</em> and soybean annotated proteomes. The significant hits were identified at different <em>E</em>-value cut-offs. The largest number of transcripts generated by Oases assembly showed significant similarity with <em>Arabidopsis</em> and soybean proteins at all <em>E</em>-value cut-offs when compared with the contigs generated by Velvet, CLC Genomics Workbench, ABySS and SOAPdenovo programs (Fig. <a reveal-id="DSQ028F2" data-open="DSQ028F2" class="link link-reveal link-table xref-fig">2</a>B and C). Although the contigs generated by ABySS showed higher N50 and average lengths, lesser number and percentage of contigs showed significant similarity with <em>Arabidopsis</em> and soybean proteins. Overall, taken together, the assembly of second trimmed short-read data set obtained by Oases was found better than others and this non-redundant transcript data set (53 409) was analysed further. The statistics of the final assembly obtained from Oases program are given in Table <a reveal-id="DSQ028TB2" data-open="DSQ028TB2" class="link link-reveal link-table xref-fig">2</a>. The comparative analysis of all (32 747) the ESTs and ESTs reported in various individual transcriptome studies,<sup><a reveal-id="DSQ028C6" data-open="DSQ028C6" class="link link-ref link-reveal xref-bibr">6</a>–<a reveal-id="DSQ028C8" data-open="DSQ028C8" class="link link-ref link-reveal xref-bibr">8</a></sup> available at NCBI [after removing/trimming contaminating vector sequences and short reads (&lt;100 bp) using SeqClean] with the above non-redundant transcript data set of chickpea showed that most (79% of the total and 74–88% of the individual EST data set reported in previous studies) of the ESTs are represented in our data set showing ≥90% identity over a length of ≥100 bp. </p>                        <a id="5495296" scrollto-destination="5495296"></a>
<div content-id="DSQ028TB2" class="table-modal"><div class="table-wrap"><div id="DSQ028TB2" class="table-wrap-title" data-id="DSQ028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of non-redundant set of chickpea transcripts obtained from Oases assembly</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total number of reads </td><td>106 660 317 </td></tr><tr><td>Number of used reads </td><td>88 337 267 </td></tr><tr><td>Number of unused reads </td><td>18 323 050 </td></tr><tr><td>Number of transcript isoforms (≥100 bp) </td><td>59 178 </td></tr><tr><td>Number of non-redundant transcripts (≥100 bp) </td><td>53 409 </td></tr><tr><td>Total size of transcriptome (bp) </td><td>27 950 383 </td></tr><tr><td>N50 length (bp) </td><td>900 </td></tr><tr><td>Average contig length (bp) </td><td>523 </td></tr><tr><td>Largest contig length (bp) </td><td>8173 </td></tr><tr><td>Average coverage (rpkm) </td><td>25.7 </td></tr><tr><td>Average number of reads per transcript </td><td>1616.7 </td></tr></tbody></table></div><div content-id="DSQ028TB2" class="table-modal"><table><tbody><tr><td>Total number of reads </td><td>106 660 317 </td></tr><tr><td>Number of used reads </td><td>88 337 267 </td></tr><tr><td>Number of unused reads </td><td>18 323 050 </td></tr><tr><td>Number of transcript isoforms (≥100 bp) </td><td>59 178 </td></tr><tr><td>Number of non-redundant transcripts (≥100 bp) </td><td>53 409 </td></tr><tr><td>Total size of transcriptome (bp) </td><td>27 950 383 </td></tr><tr><td>N50 length (bp) </td><td>900 </td></tr><tr><td>Average contig length (bp) </td><td>523 </td></tr><tr><td>Largest contig length (bp) </td><td>8173 </td></tr><tr><td>Average coverage (rpkm) </td><td>25.7 </td></tr><tr><td>Average number of reads per transcript </td><td>1616.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>bp, base pair; rpkm, reads per kilo base per million.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495296" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ028TB2" class="table-wrap-title" data-id="DSQ028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of non-redundant set of chickpea transcripts obtained from Oases assembly</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total number of reads </td><td>106 660 317 </td></tr><tr><td>Number of used reads </td><td>88 337 267 </td></tr><tr><td>Number of unused reads </td><td>18 323 050 </td></tr><tr><td>Number of transcript isoforms (≥100 bp) </td><td>59 178 </td></tr><tr><td>Number of non-redundant transcripts (≥100 bp) </td><td>53 409 </td></tr><tr><td>Total size of transcriptome (bp) </td><td>27 950 383 </td></tr><tr><td>N50 length (bp) </td><td>900 </td></tr><tr><td>Average contig length (bp) </td><td>523 </td></tr><tr><td>Largest contig length (bp) </td><td>8173 </td></tr><tr><td>Average coverage (rpkm) </td><td>25.7 </td></tr><tr><td>Average number of reads per transcript </td><td>1616.7 </td></tr></tbody></table></div><div content-id="DSQ028TB2" class="table-modal"><table><tbody><tr><td>Total number of reads </td><td>106 660 317 </td></tr><tr><td>Number of used reads </td><td>88 337 267 </td></tr><tr><td>Number of unused reads </td><td>18 323 050 </td></tr><tr><td>Number of transcript isoforms (≥100 bp) </td><td>59 178 </td></tr><tr><td>Number of non-redundant transcripts (≥100 bp) </td><td>53 409 </td></tr><tr><td>Total size of transcriptome (bp) </td><td>27 950 383 </td></tr><tr><td>N50 length (bp) </td><td>900 </td></tr><tr><td>Average contig length (bp) </td><td>523 </td></tr><tr><td>Largest contig length (bp) </td><td>8173 </td></tr><tr><td>Average coverage (rpkm) </td><td>25.7 </td></tr><tr><td>Average number of reads per transcript </td><td>1616.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>bp, base pair; rpkm, reads per kilo base per million.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495296" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5495298 id="5495298" class="section-title" data-legacy-id=s3c>3.3. GC content analysis of chickpea transcriptome</h3>
<p>GC content (ratio of guanine and cytosine) of all the chickpea transcripts along with soybean (legume reference), <em>Arabidopsis</em> (dicot reference) and rice (monocot reference) was determined. The average GC content of chickpea transcripts (40.3%) and soybean unigenes (40.9%) was little lower than that of <em>Arabidopsis</em> (42.5%). The average GC content in rice was much higher (55%) as reported previously as well.<sup><a reveal-id="DSQ028C22" data-open="DSQ028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Although the average GC contents of chickpea and <em>Arabidopsis</em> were comparable, chickpea has a higher proportion of transcripts with GC content in range of 35–40% but lower proportion of transcripts with high GC content in range of 40–45% (Fig. <a reveal-id="DSQ028F3" data-open="DSQ028F3" class="link link-reveal link-table xref-fig">3</a>). In addition, the range of GC content was broader in chickpea and soybean when compared with <em>Arabidopsis</em> (Fig. <a reveal-id="DSQ028F3" data-open="DSQ028F3" class="link link-reveal link-table xref-fig">3</a>). A similar observation was found for the unigene sets from other legume species as well (data not shown). </p>                        <a id="5495302" scrollto-destination="5495302"></a>
<div id="DSQ028F3" data-id="DSQ028F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02803.jpeg?Expires=2147483647&Signature=F3v4nz~p1dMKPp6mYczhYEOQgNYw87um9Z9fY52gCUavdPBsKRhMgs7t-dQ9wfO~KmRIeUFk3tml0Rtr2njvhXT049aasc4vbTHDXsqdpR68I4RN1xa1yb1a5LkC2uAvfQUCZkIpP1Hx8RV5l6tJU1xBG~Oxw3W3LQQuwKDLdxZvSnxjIWZmgDK1fgQn0afht0VJ4ZaESM0nxrjUNa166kMvqRdTeceVlS0-EL8Te9Dsudr-W8WXvPArXH0IVlAxkW8uR7VYjjOZ4lS0nlKIyyZvYiu8OPA3sm0IXulXTEH3lPr9nuNGltAIpBt1-kp~zPuUR3E~5zrdXpk-~Ndrkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content analysis of chickpea transcripts. The average GC content of each transcript for chickpea, soybean, Arabidopsis and rice was calculated and percentage of transcripts with GC content within a range are represented." path-from-xml="dsq02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495302/dsq02803.jpeg" path-from-xml="dsq02803.jpeg" target="_blank">View large</a><a section="5495302" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02803.jpeg?Expires=2147483647&Signature=j6b-s7Kn-edhLyFJcKacNxooULmYn4XmnXVcF54M1MbDhX0JsH2Cr2eQz8DdRihEhBoJzzN~jhqLZ2BfVjs9eqNTSMUxFMJ71215MCb3P6OH2fCwURbdJPAv654pnjj~mGvmJ1LEDu29B2l1yU5Kq6uqUbbo6ngISNkA-zF7EdghZh8GgZAF2vu1071tvqA550r~zlcM~rGQ51B7IF9~vsRNvu9xuimnqLQ61cmX~DmGQq~SEAgArmxcZD4bGsMwR8K6ceCSqwWBnwI~~SzkFt4I22IG7beoJTTn2xXNJTlw4Ub~w1sIbY5UOTS~E4iUHG-LFFxcizuGjVCT1iZXAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495302&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content analysis of chickpea transcripts. The average GC content of each transcript for chickpea, soybean, <em>Arabidopsis</em> and rice was calculated and percentage of transcripts with GC content within a range are represented.</p></div></div><div content-id="DSQ028F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02803.jpeg?Expires=2147483647&Signature=F3v4nz~p1dMKPp6mYczhYEOQgNYw87um9Z9fY52gCUavdPBsKRhMgs7t-dQ9wfO~KmRIeUFk3tml0Rtr2njvhXT049aasc4vbTHDXsqdpR68I4RN1xa1yb1a5LkC2uAvfQUCZkIpP1Hx8RV5l6tJU1xBG~Oxw3W3LQQuwKDLdxZvSnxjIWZmgDK1fgQn0afht0VJ4ZaESM0nxrjUNa166kMvqRdTeceVlS0-EL8Te9Dsudr-W8WXvPArXH0IVlAxkW8uR7VYjjOZ4lS0nlKIyyZvYiu8OPA3sm0IXulXTEH3lPr9nuNGltAIpBt1-kp~zPuUR3E~5zrdXpk-~Ndrkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content analysis of chickpea transcripts. The average GC content of each transcript for chickpea, soybean, Arabidopsis and rice was calculated and percentage of transcripts with GC content within a range are represented." path-from-xml="dsq02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495302/dsq02803.jpeg" path-from-xml="dsq02803.jpeg" target="_blank">View large</a><a section="5495302" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02803.jpeg?Expires=2147483647&Signature=j6b-s7Kn-edhLyFJcKacNxooULmYn4XmnXVcF54M1MbDhX0JsH2Cr2eQz8DdRihEhBoJzzN~jhqLZ2BfVjs9eqNTSMUxFMJ71215MCb3P6OH2fCwURbdJPAv654pnjj~mGvmJ1LEDu29B2l1yU5Kq6uqUbbo6ngISNkA-zF7EdghZh8GgZAF2vu1071tvqA550r~zlcM~rGQ51B7IF9~vsRNvu9xuimnqLQ61cmX~DmGQq~SEAgArmxcZD4bGsMwR8K6ceCSqwWBnwI~~SzkFt4I22IG7beoJTTn2xXNJTlw4Ub~w1sIbY5UOTS~E4iUHG-LFFxcizuGjVCT1iZXAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495302&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content analysis of chickpea transcripts. The average GC content of each transcript for chickpea, soybean, <em>Arabidopsis</em> and rice was calculated and percentage of transcripts with GC content within a range are represented.</p></div></div>                            <h3 scrollto-destination=5495307 id="5495307" class="section-title" data-legacy-id=s3d>3.4. Identification of SSRs</h3>
<p>The transcript/EST-based markers are important resource for determining functional genetic variation.<sup><a reveal-id="DSQ028C23" data-open="DSQ028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Among the various molecular markers, SSRs are highly polymorphic, easier to develop and serve as rich resource of diversity. For identification of SSRs, all the chickpea transcripts were searched with perl script MISA. We identified a total of 4816 SSRs in 4180 (7.8%) transcripts of chickpea with frequency of one SSR per 5.80 kb of the sequence (Table <a reveal-id="DSQ028TB3" data-open="DSQ028TB3" class="link link-reveal link-table xref-fig">3</a>). The mono-nucleotide SSRs represented the largest fraction (41.9%) of SSRs identified followed by tri-nucleotide (36.1%) and di-nucleotide (19.3%) SSRs. Although only a small fraction of tetra- (50), penta- (22) and hexa-nucleotide (58) SSRs were identified in chickpea transcripts, the number is quite significant. </p>                        <a id="5495312" scrollto-destination="5495312"></a>
<div content-id="DSQ028TB3" class="table-modal"><div class="table-wrap"><div id="DSQ028TB3" class="table-wrap-title" data-id="DSQ028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of SSRs identified in chickpea transcripts</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="2">SSR mining </td></tr><tr><td> Total number of sequences examined </td><td>53 409 </td></tr><tr><td> Total size of examined sequences (bp) </td><td>27 950 383 </td></tr><tr><td> Total number of identified SSRs </td><td>4816 </td></tr><tr><td> Number of SSR containing sequences </td><td>4180 (7.8%) </td></tr><tr><td> Number of sequences containing more than one SSR </td><td>534 </td></tr><tr><td> Number of SSRs present in compound formation </td><td>282 </td></tr><tr><td> Frequency of SSRs </td><td>One per 5.80 kb </td></tr><tr><td colspan="2">Distribution of SSRs in different repeat types </td></tr><tr><td> Mono-nucleotide </td><td>2020 (41.9%) </td></tr><tr><td> Di-nucleotide </td><td>930 (19.3%) </td></tr><tr><td> Tri-nucleotide </td><td>1736 (36.1%) </td></tr><tr><td> Tetra-nucleotide </td><td>50 (0.010%) </td></tr><tr><td> Penta-nucleotide </td><td>22 (0.005%) </td></tr><tr><td> Hexa-nucleotide </td><td>58 (0.012%) </td></tr></tbody></table></div><div content-id="DSQ028TB3" class="table-modal"><table><tbody><tr><td colspan="2">SSR mining </td></tr><tr><td> Total number of sequences examined </td><td>53 409 </td></tr><tr><td> Total size of examined sequences (bp) </td><td>27 950 383 </td></tr><tr><td> Total number of identified SSRs </td><td>4816 </td></tr><tr><td> Number of SSR containing sequences </td><td>4180 (7.8%) </td></tr><tr><td> Number of sequences containing more than one SSR </td><td>534 </td></tr><tr><td> Number of SSRs present in compound formation </td><td>282 </td></tr><tr><td> Frequency of SSRs </td><td>One per 5.80 kb </td></tr><tr><td colspan="2">Distribution of SSRs in different repeat types </td></tr><tr><td> Mono-nucleotide </td><td>2020 (41.9%) </td></tr><tr><td> Di-nucleotide </td><td>930 (19.3%) </td></tr><tr><td> Tri-nucleotide </td><td>1736 (36.1%) </td></tr><tr><td> Tetra-nucleotide </td><td>50 (0.010%) </td></tr><tr><td> Penta-nucleotide </td><td>22 (0.005%) </td></tr><tr><td> Hexa-nucleotide </td><td>58 (0.012%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495312" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ028TB3" class="table-wrap-title" data-id="DSQ028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of SSRs identified in chickpea transcripts</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="2">SSR mining </td></tr><tr><td> Total number of sequences examined </td><td>53 409 </td></tr><tr><td> Total size of examined sequences (bp) </td><td>27 950 383 </td></tr><tr><td> Total number of identified SSRs </td><td>4816 </td></tr><tr><td> Number of SSR containing sequences </td><td>4180 (7.8%) </td></tr><tr><td> Number of sequences containing more than one SSR </td><td>534 </td></tr><tr><td> Number of SSRs present in compound formation </td><td>282 </td></tr><tr><td> Frequency of SSRs </td><td>One per 5.80 kb </td></tr><tr><td colspan="2">Distribution of SSRs in different repeat types </td></tr><tr><td> Mono-nucleotide </td><td>2020 (41.9%) </td></tr><tr><td> Di-nucleotide </td><td>930 (19.3%) </td></tr><tr><td> Tri-nucleotide </td><td>1736 (36.1%) </td></tr><tr><td> Tetra-nucleotide </td><td>50 (0.010%) </td></tr><tr><td> Penta-nucleotide </td><td>22 (0.005%) </td></tr><tr><td> Hexa-nucleotide </td><td>58 (0.012%) </td></tr></tbody></table></div><div content-id="DSQ028TB3" class="table-modal"><table><tbody><tr><td colspan="2">SSR mining </td></tr><tr><td> Total number of sequences examined </td><td>53 409 </td></tr><tr><td> Total size of examined sequences (bp) </td><td>27 950 383 </td></tr><tr><td> Total number of identified SSRs </td><td>4816 </td></tr><tr><td> Number of SSR containing sequences </td><td>4180 (7.8%) </td></tr><tr><td> Number of sequences containing more than one SSR </td><td>534 </td></tr><tr><td> Number of SSRs present in compound formation </td><td>282 </td></tr><tr><td> Frequency of SSRs </td><td>One per 5.80 kb </td></tr><tr><td colspan="2">Distribution of SSRs in different repeat types </td></tr><tr><td> Mono-nucleotide </td><td>2020 (41.9%) </td></tr><tr><td> Di-nucleotide </td><td>930 (19.3%) </td></tr><tr><td> Tri-nucleotide </td><td>1736 (36.1%) </td></tr><tr><td> Tetra-nucleotide </td><td>50 (0.010%) </td></tr><tr><td> Penta-nucleotide </td><td>22 (0.005%) </td></tr><tr><td> Hexa-nucleotide </td><td>58 (0.012%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5495312" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5495314 id="5495314" class="section-title" data-legacy-id=s3e>3.5. Sequence similarity of chickpea transcripts with other plants</h3>
<p>The transcript set of chickpea was analysed for similarity/sequence conservation against the unigene data sets of various legumes species namely soybean, <em>Medicago</em>, <em>Lotus</em>, <em>Vigna</em> and <em>Pisum</em> using TBLASTX search. An <em>E</em>-value cut-off threshold of ≤1E−05 was considered to define a significant hit. The largest number (72.4%) of chickpea transcripts showed significant similarity with soybean unigenes followed by <em>Medicago</em> (69.5%), <em>Lotus</em> (65.5%), <em>Vigna</em> (60.1%) and the least similarity with <em>Pisum</em> (39.3%; Fig. <a reveal-id="DSQ028F4" data-open="DSQ028F4" class="link link-reveal link-table xref-fig">4</a>A). Overall, a total of 43 516 (81.5%) of the chickpea transcripts showed significant similarity with at least one of the other legume unigenes. Likewise, we analysed the sequence conservation of chickpea transcripts with proteomes of all sequenced plant species. A total of 42 012 (79%) transcripts exhibited significant similarity with at least one of the predicted protein from sequenced plants. The largest number (75%) of chickpea transcripts showed significant similarity with soybean followed by <em>Medicago</em> (69%) and least with <em>Physcomitrella</em> (50%; Fig. <a reveal-id="DSQ028F4" data-open="DSQ028F4" class="link link-reveal link-table xref-fig">4</a>B). As expected, lesser number of chickpea transcripts showed significant similarity with monocots (52–61%) when compared with dicots (65–75%). Although a large number of the chickpea transcripts showed significant similarity with predicted proteins from legumes, the extent of coverage of the coding region was quite less than expected. Only 28.4 and 29.5% of the transcripts which showed significant similarity covered ≥50% of the coding region of the predicted proteins from soybean and <em>Medicago</em>, respectively. Considering a high degree of conservation among legumes, it may be assumed that the assembly of chickpea transcriptome may further be improved as more sequence data become available. </p>                        <a id="5495320" scrollto-destination="5495320"></a>
<div id="DSQ028F4" data-id="DSQ028F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02804.jpeg?Expires=2147483647&Signature=bP-WZulBY2u8tT2-HFZMtWheOXKp3ezkeC5Cgecw~cO5ULWVzPcZnypcE7MtnAfvHEqUcHucbFMM2L~3VjiY1AbFsWemhqPmm2fzEgOTTk2iEoBJe5Rqb8I1RXbW-zyughktA1VEM7J0jPgGKFfb7neWu1Sja0ptnsAztfEwvFG3PuVpsRafnsanZMGETQJ7LL7nL9g5bF-oRTOrqC3xkm79renEUh7fkLW9aJfrSK2Epav7rAdWGhewKYswnCdSrf0-9aTWekNVT19JF-6nRW0dbasvHcRqUlRotV42sk~ydEWoqztoIjfxlGWLi0AbO7RwMKtTyXabHOkG3OcxyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence conservation of chickpea transcripts with other plant species. (A) Sequence conservation of chickpea transcripts with putative transcript consensus (TC) sequences of various legume species. (B) Sequence conservation of chickpea transcripts with annotated proteins of completely sequenced plant species. The percentage of transcripts showing significant similarity (E-value ≤ 1E−05) in TBLASTX (A) and BLASTX (B) searches are shown." path-from-xml="dsq02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495320/dsq02804.jpeg" path-from-xml="dsq02804.jpeg" target="_blank">View large</a><a section="5495320" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02804.jpeg?Expires=2147483647&Signature=HQNKUrk~soRuMmt3zIl4MZcjDyNmxbKhduDhELkYsmk7cbrJ4OOh4bkLrPsGOYnZNsMjwcdhAqHQ0ZxaVgHskoXwfnK4b3iBTQdrLr6DK1Bm1BmEOSgfJW1jR1UAhYr332iVqwW-XSk8~Skzoan3ISKIlcAF56NHLcdwQpbt6cg~AfpjbbD~OOCvlEptQkSUINnLq6ZessONlYIFA~1OaurLCTxh1nnxVZfq8kFnReRhA3oFyLyfg8MERR~VlFEfJ4RdCa1THmL9FC-oHOxMoJABfNc95kTdHIr0NSBWQR9zwWl8jXmbZ3RhO8ps~9jpjLumpPrqv3xh37-jOepo2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495320&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence conservation of chickpea transcripts with other plant species. (A) Sequence conservation of chickpea transcripts with putative transcript consensus (TC) sequences of various legume species. (B) Sequence conservation of chickpea transcripts with annotated proteins of completely sequenced plant species. The percentage of transcripts showing significant similarity (<em>E</em>-value ≤ 1E−05) in TBLASTX (A) and BLASTX (B) searches are shown.</p></div></div><div content-id="DSQ028F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02804.jpeg?Expires=2147483647&Signature=bP-WZulBY2u8tT2-HFZMtWheOXKp3ezkeC5Cgecw~cO5ULWVzPcZnypcE7MtnAfvHEqUcHucbFMM2L~3VjiY1AbFsWemhqPmm2fzEgOTTk2iEoBJe5Rqb8I1RXbW-zyughktA1VEM7J0jPgGKFfb7neWu1Sja0ptnsAztfEwvFG3PuVpsRafnsanZMGETQJ7LL7nL9g5bF-oRTOrqC3xkm79renEUh7fkLW9aJfrSK2Epav7rAdWGhewKYswnCdSrf0-9aTWekNVT19JF-6nRW0dbasvHcRqUlRotV42sk~ydEWoqztoIjfxlGWLi0AbO7RwMKtTyXabHOkG3OcxyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence conservation of chickpea transcripts with other plant species. (A) Sequence conservation of chickpea transcripts with putative transcript consensus (TC) sequences of various legume species. (B) Sequence conservation of chickpea transcripts with annotated proteins of completely sequenced plant species. The percentage of transcripts showing significant similarity (E-value ≤ 1E−05) in TBLASTX (A) and BLASTX (B) searches are shown." path-from-xml="dsq02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495320/dsq02804.jpeg" path-from-xml="dsq02804.jpeg" target="_blank">View large</a><a section="5495320" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02804.jpeg?Expires=2147483647&Signature=HQNKUrk~soRuMmt3zIl4MZcjDyNmxbKhduDhELkYsmk7cbrJ4OOh4bkLrPsGOYnZNsMjwcdhAqHQ0ZxaVgHskoXwfnK4b3iBTQdrLr6DK1Bm1BmEOSgfJW1jR1UAhYr332iVqwW-XSk8~Skzoan3ISKIlcAF56NHLcdwQpbt6cg~AfpjbbD~OOCvlEptQkSUINnLq6ZessONlYIFA~1OaurLCTxh1nnxVZfq8kFnReRhA3oFyLyfg8MERR~VlFEfJ4RdCa1THmL9FC-oHOxMoJABfNc95kTdHIr0NSBWQR9zwWl8jXmbZ3RhO8ps~9jpjLumpPrqv3xh37-jOepo2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495320&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence conservation of chickpea transcripts with other plant species. (A) Sequence conservation of chickpea transcripts with putative transcript consensus (TC) sequences of various legume species. (B) Sequence conservation of chickpea transcripts with annotated proteins of completely sequenced plant species. The percentage of transcripts showing significant similarity (<em>E</em>-value ≤ 1E−05) in TBLASTX (A) and BLASTX (B) searches are shown.</p></div></div><p>Further, the combined analysis of BLAST results with legume unigenes and plant proteomes revealed that a total of 45 636 (85.5%) transcripts were conserved in chickpea showing significant similarity with at least one sequence. Among these, 3624 (6.8%) transcripts were conserved only in legumes representing legume-specific genes. However, 7773 (14.5%) transcripts did not show significant similarity with any of the data set analysed, and may represent chickpea-specific genes.</p>                            <h3 scrollto-destination=5495329 id="5495329" class="section-title" data-legacy-id=s3f>3.6. Functional annotation and characterization of chickpea transcripts</h3>
<p>To identify the putative function of chickpea transcripts, they were compared against the non-redundant protein sequences available at UniProt database and <em>Arabidopsis</em> proteins using BLASTX search. A total of 23 864 (44.7%) and 34 993 (65.5%) chickpea transcripts showed significant hit with UniProt and <em>Arabidopsis</em> proteins, respectively. Together, 35 279 (66.1%) transcripts showed significant hit with at least one UniProt or <em>Arabidopsis</em> protein. Broadly, the putative orthologs of genes involved in various pathways and cellular processes were found to be conserved in chickpea. In addition, many chickpea transcripts showed homology to uncharacterized proteins annotated as unknown, hypothetical and expressed proteins as well. Further, GO terms were assigned to chickpea transcripts, which showed significant similarity with <em>Arabidopsis</em> proteins annotated with GO terms. A total of 34 676 (64.9%) transcripts were assigned at least one GO term, among which 31 250 were assigned at least one GO term in biological process category, 31 598 in molecular function category and 30 264 in cellular component category. Among the various biological processes, ignoring unknown and other biological process categories, protein metabolism (19.5%) and developmental processes (15.6%) were most highly represented (Fig. <a reveal-id="DSQ028F5" data-open="DSQ028F5" class="link link-reveal link-table xref-fig">5</a>). The genes involved in other important biological processes such as response to abiotic and biotic stimulus/stress, transport, transcription and signal transduction, were also identified through GO annotations. Similarly, transferase activity and hydrolase activity were most represented among the various molecular functions, and chloroplast and plasma membrane were most represented among the cellular component categories (Fig. <a reveal-id="DSQ028F5" data-open="DSQ028F5" class="link link-reveal link-table xref-fig">5</a>). Further, we identified transcription factor encoding transcripts by sequence comparison to known transcription factor gene families. In total, 6577 putative chickpea transcription factor genes, distributed in at least 57 families, were identified representing 12.3% of chickpea transcripts (Fig. <a reveal-id="DSQ028F6" data-open="DSQ028F6" class="link link-reveal link-table xref-fig">6</a>). The overall distribution of transcription factor encoding transcripts among the various known protein families is very similar with that of soybean and other legumes as predicted earlier.<sup><a reveal-id="DSQ028C24" data-open="DSQ028C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ028C25" data-open="DSQ028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> However, a few families showed the events of expansion (for example, Aux/IAA-ARF, bHLH, C3H, MADS, NAC, PHD and RWP-RK etc.) and contraction (for example, C2C2 zinc finger, CCAAT, LIM and MYB etc.) indicating their evolutionary significance (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). In fact, the number of predicted transcription factor encoding genes in <em>Medicago</em> and <em>Lotus</em> are very less when compared with soybean.<sup><a reveal-id="DSQ028C24" data-open="DSQ028C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ028C25" data-open="DSQ028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> As the complete genome sequence is available only for soybean as of now, the complete picture about evolution of various transcription factor families will emerge once the complete genome sequences and analyses thereof will be available for other legume species too. </p>                        <a id="5495338" scrollto-destination="5495338"></a>
<div id="DSQ028F5" data-id="DSQ028F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02805.jpeg?Expires=2147483647&Signature=0T4EBEeIjueOZB5abocwysi4l89zehgk9km7pzeZCMFAWTkXVLf94I1E-OmfXPkRDQFoq0RFIH44km7nHAaboI4hslSpDQUxG4ERcuz15WRAaAp-J2hFRcBi0FP39U1KfOuQHinRNsYpiQfyANWWIExJZF3H~EXQVUH8V9DdlENMiUWZ1UhDa8Ci1oOVL2ykKDGiBRrabRxkalvNFuzFRWEyHi7HM3pHMItNkz22EI6M3DxBNUWytR0WUgaDf6BW5rQ4d5dErjNDtzMGLS0EDxlTGM0bxVzY8dSgIZp4KwL6Bp~ymDic6mxEmMaXpqW38CagYdHOlOHqUUjITiitDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional annotation of chickpea transcripts. GOSlim term assignment to the chickpea transcripts in different categories of biological process, molecular function and cellular component." path-from-xml="dsq02805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495338/dsq02805.jpeg" path-from-xml="dsq02805.jpeg" target="_blank">View large</a><a section="5495338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02805.jpeg?Expires=2147483647&Signature=TxnKl8BUnEJDf0wfR6CJNiEN4KXzz-~IXexDW1ItwV4QcDTRj8bJdrQ56OdueCkIXnOdsMTEulDJgo0l8dIESeSSQZsaJtrqSqlry4JSfU6Fcsv-Mizio04Fk9yzqEAavPP6VnEIn1gelOl6rhoeqJJ7XA4jfzXDDhYs8GiUOe3V40PQqZdPHkBgODboQYZdrtiFBw-vUMx~Tm0v0jWPybFHh3rDoR5EoAQj8s0M8T9dum1C-qJHfMKh45tNzqEgyapEXwcJLKZlUTaMtTiPw6HDBRmh5GdoWaP8h4tQNKYYdJjTRDNTHAjoNkFTz-oPTOeLPrqGs16itO2l~SIyuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495338&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional annotation of chickpea transcripts. GOSlim term assignment to the chickpea transcripts in different categories of biological process, molecular function and cellular component.</p></div></div><div content-id="DSQ028F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02805.jpeg?Expires=2147483647&Signature=0T4EBEeIjueOZB5abocwysi4l89zehgk9km7pzeZCMFAWTkXVLf94I1E-OmfXPkRDQFoq0RFIH44km7nHAaboI4hslSpDQUxG4ERcuz15WRAaAp-J2hFRcBi0FP39U1KfOuQHinRNsYpiQfyANWWIExJZF3H~EXQVUH8V9DdlENMiUWZ1UhDa8Ci1oOVL2ykKDGiBRrabRxkalvNFuzFRWEyHi7HM3pHMItNkz22EI6M3DxBNUWytR0WUgaDf6BW5rQ4d5dErjNDtzMGLS0EDxlTGM0bxVzY8dSgIZp4KwL6Bp~ymDic6mxEmMaXpqW38CagYdHOlOHqUUjITiitDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional annotation of chickpea transcripts. GOSlim term assignment to the chickpea transcripts in different categories of biological process, molecular function and cellular component." path-from-xml="dsq02805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495338/dsq02805.jpeg" path-from-xml="dsq02805.jpeg" target="_blank">View large</a><a section="5495338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02805.jpeg?Expires=2147483647&Signature=TxnKl8BUnEJDf0wfR6CJNiEN4KXzz-~IXexDW1ItwV4QcDTRj8bJdrQ56OdueCkIXnOdsMTEulDJgo0l8dIESeSSQZsaJtrqSqlry4JSfU6Fcsv-Mizio04Fk9yzqEAavPP6VnEIn1gelOl6rhoeqJJ7XA4jfzXDDhYs8GiUOe3V40PQqZdPHkBgODboQYZdrtiFBw-vUMx~Tm0v0jWPybFHh3rDoR5EoAQj8s0M8T9dum1C-qJHfMKh45tNzqEgyapEXwcJLKZlUTaMtTiPw6HDBRmh5GdoWaP8h4tQNKYYdJjTRDNTHAjoNkFTz-oPTOeLPrqGs16itO2l~SIyuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495338&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional annotation of chickpea transcripts. GOSlim term assignment to the chickpea transcripts in different categories of biological process, molecular function and cellular component.</p></div></div>                        <a id="5495343" scrollto-destination="5495343"></a>
<div id="DSQ028F6" data-id="DSQ028F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02806.jpeg?Expires=2147483647&Signature=NzBxFdleXmI-iJ5yrmcV0uYJ6TbVrsTQGMQBMjKWyuQ2SdLtkOW6fn0eTyLmg04MV9ud1iJcp8~21trp02-rkoMz31jbpxxJbgr15vNJC2nMOZq4QXJ13JlfFJf~2vGTPri7JVuJAaV1pJ4SdQEqloczpLM43IFCHueiFfMYk7onoyVqMWR1mtOjT3IHQSDkBX4TnSPl-eMi0AtDvrjFDtLD3U66Tx3kY07YwSVhRyyoEnfShUAhMrPTqSuEAY45jiVaGERmLCKT0XMaQC7ov3-7MNc3PCxDenbWZrGKd~a-H1OhzIy9UdCHZPVTRFnWL17yfk4TJTXO5-5bacxwtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of chickpea transcripts in different transcription factor families." path-from-xml="dsq02806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495343/dsq02806.jpeg" path-from-xml="dsq02806.jpeg" target="_blank">View large</a><a section="5495343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02806.jpeg?Expires=2147483647&Signature=3oIM7legfOsLEwuV1Ui2NEMALLA8MBiC9fyZNy7JKXuw2al5fTBfv4C4Aj4KzM~0yC9its3d-ydW~ynq7-laiKEd0iA6OPjOK7dEaoGzip7ok4EvnOn74wbyLAzfT6DduOYD35Y2OHud91rKdwn6-vRS2Cp0oxqxzTRvrjaLHeVlYBNq33awWrigRI4Z6GMgzN8AGF9FksJs~f-xPWayBey6fPY~9P2dSNehb~hpRAqPavs~XB4CFwhwLCs10mKHyMDE16Bv8CiYuAE3uGmT9dsUA7YCgSqVHJJKe-Mmv6Otad2vr~9omM3Er2aTglmAwjJwpvPEao~DD5v8NrXqFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495343&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of chickpea transcripts in different transcription factor families.</p></div></div><div content-id="DSQ028F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/m_dsq02806.jpeg?Expires=2147483647&Signature=NzBxFdleXmI-iJ5yrmcV0uYJ6TbVrsTQGMQBMjKWyuQ2SdLtkOW6fn0eTyLmg04MV9ud1iJcp8~21trp02-rkoMz31jbpxxJbgr15vNJC2nMOZq4QXJ13JlfFJf~2vGTPri7JVuJAaV1pJ4SdQEqloczpLM43IFCHueiFfMYk7onoyVqMWR1mtOjT3IHQSDkBX4TnSPl-eMi0AtDvrjFDtLD3U66Tx3kY07YwSVhRyyoEnfShUAhMrPTqSuEAY45jiVaGERmLCKT0XMaQC7ov3-7MNc3PCxDenbWZrGKd~a-H1OhzIy9UdCHZPVTRFnWL17yfk4TJTXO5-5bacxwtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of chickpea transcripts in different transcription factor families." path-from-xml="dsq02806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5495343/dsq02806.jpeg" path-from-xml="dsq02806.jpeg" target="_blank">View large</a><a section="5495343" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq02806.jpeg?Expires=2147483647&Signature=3oIM7legfOsLEwuV1Ui2NEMALLA8MBiC9fyZNy7JKXuw2al5fTBfv4C4Aj4KzM~0yC9its3d-ydW~ynq7-laiKEd0iA6OPjOK7dEaoGzip7ok4EvnOn74wbyLAzfT6DduOYD35Y2OHud91rKdwn6-vRS2Cp0oxqxzTRvrjaLHeVlYBNq33awWrigRI4Z6GMgzN8AGF9FksJs~f-xPWayBey6fPY~9P2dSNehb~hpRAqPavs~XB4CFwhwLCs10mKHyMDE16Bv8CiYuAE3uGmT9dsUA7YCgSqVHJJKe-Mmv6Otad2vr~9omM3Er2aTglmAwjJwpvPEao~DD5v8NrXqFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5495343&ar=377009&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of chickpea transcripts in different transcription factor families.</p></div></div>                            <h3 scrollto-destination=5495349 id="5495349" class="section-title" data-legacy-id=s3g>3.7. Quantification of chickpea transcripts</h3>
<p>The digital expression profiling, also called RNA-Seq, is a powerful and efficient approach for gene expression analysis.<sup><a reveal-id="DSQ028C26" data-open="DSQ028C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSQ028C27" data-open="DSQ028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The mapping of all the reads onto the non-redundant set of chickpea transcripts revealed that the number of reads corresponding to each transcript ranged from 14 (0.16 rpm) to 270 894 (3,137.3 rpm) with an average of 1617 reads (18.7 rpm) per transcript, indicating a very wide range of expression levels of chickpea transcripts (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). It also indicates that very low expressed chickpea transcripts are also represented in our assembly. The minimum coverage (rpkm) of a chickpea transcript was 1.2 and maximum of 9015.1 with an average of 25.7 (Table <a reveal-id="DSQ028TB2" data-open="DSQ028TB2" class="link link-reveal link-table xref-fig">2</a>). Further, we analysed the expression of chickpea transcripts in root, shoot and pooled (root, shoot, mature leaf and flower bud) tissue samples. The mapping of reads on the chickpea transcript data set revealed that a total of 1974 and 1174 transcripts are not represented in the root and shoot tissue sequence data sets, respectively. Among these, 420 transcripts have no read mapped from root and shoot data sets, indicating their expression in mature leaf and/or flower buds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Further, another 250 transcripts were found to have root-specific expression, as they have no read mapped from shoot data set and at least 3 rpm mapped from root data set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Likewise, another 217 transcripts were found to have shoot-specific expression, as they have no read mapped from root data set and at least 3 rpm mapped from shoot data set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). However, their expression in other chickpea tissues not analysed in this study is not ruled out.</p>                            <h2 scrollto-destination=5495357 id="5495357" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The transcriptome sequencing enables various functional genomic studies for an organism. Although several high throughput technologies have been developed for rapid sequencing and characterization of transcriptomes, expressed sequence data are still not available for many organisms, including crop plants. The next generation sequencing technologies provide a low cost, labour saving and rapid means of transcriptome sequencing and characterization.<sup><a reveal-id="DSQ028C14" data-open="DSQ028C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Similar to sequencing technologies, many bioinformatics tools have also been developed for the short-read sequence data assembly and analysis,<sup><a reveal-id="DSQ028C17" data-open="DSQ028C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSQ028C18" data-open="DSQ028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> but the requisite knowledge is very limited. The <em>de novo</em> assembly of short reads without a known reference is considered difficult.<sup><a reveal-id="DSQ028C28" data-open="DSQ028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Therefore, the use of more expensive 454 Life Sciences (Roche) technology is used for non-model organisms, which produces longer sequence reads.<sup><a reveal-id="DSQ028C15" data-open="DSQ028C15" class="link link-ref link-reveal xref-bibr">15</a></sup> However, the <em>de novo</em> assembly of transcriptomes using short reads has also received attention.<sup><a reveal-id="DSQ028C20" data-open="DSQ028C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSQ028C29" data-open="DSQ028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In this study, we demonstrate a strategy for <em>de novo</em> assembly of transcriptome using short reads for a non-model crop plant, chickpea, for which sequence data is very limited so far in the public databases. We showed that there is significant effect of the assembly program parameters and sequence quality on the assembly output. A larger N50 length and average length are considered indicative of better assembly. Our results show that N50 length of the contigs generated using Velvet assembly program varied greatly as a function of <em>k</em>-mer length and increased from 675 bp for untrimmed data set to 730 bp after trimming low-quality bases. Hence, we suggest that the optimization of program parameters and trimming of low-quality bases at the ends of sequence reads might improve the assembly output significantly. In addition, the validation of different assembly programs is also required to get the optimum results. We found the assembly of Velvet followed by Oases program better than that of Velvet alone, CLC Genomics Workbench, ABySS and SOAPdenovo programs based on various assessment parameters such as N50 length, average contig length and sequence similarity with closely related species.</p><p>Chickpea is one of the most important legume crop plants rich in proteins, carbohydrates and other nutrients, which makes it very important target for genomic studies.<sup><a reveal-id="DSQ028C1" data-open="DSQ028C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ028C2" data-open="DSQ028C2" class="link link-ref link-reveal xref-bibr">2</a></sup> For the model legumes such as soybean, <em>Medicago</em> and <em>Lotus</em>, genome sequencing has been nearly completed and a vast collection of ESTs are available for functional genomic studies.<sup><a reveal-id="DSQ028C25" data-open="DSQ028C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSQ028C30  DSQ028C32" data-open="DSQ028C30  DSQ028C32" class="link link-ref link-reveal xref-bibr">30–32</a></sup> However, very few genomic resources, including genome sequence, EST sequences and molecular markers are available for chickpea so far when compared with other legumes. We have generated more than 100million sequence reads for chickpea and report a non-redundant set of 53 409 transcripts representing about 28 Mb sequence and 3.8% of the chickpea genome. The coverage of chickpea transcripts was quite high (average of 25.7 rpkm), which is very crucial for quality and length of transcripts obtained. The GC content analysis revealed that chickpea transcripts have a low GC content similar to other dicots. However, the GC content range was broader when compared with other dicots which cover a narrow GC range.<sup><a reveal-id="DSQ028C33" data-open="DSQ028C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The GC content analysis provide insights into various aspects related to genome of an organism, including evolution, gene structure (intron size and number), thermostability and gene regulation.<sup><a reveal-id="DSQ028C22" data-open="DSQ028C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSQ028C34" data-open="DSQ028C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSQ028C35" data-open="DSQ028C35" class="link link-ref link-reveal xref-bibr">35</a></sup> We identified a large number of SSRs in the chickpea transcripts. The number of tri-nucleotide SSRs was much higher than di-nucleotide SSRs. Earlier studies have also reported the higher number of tri-nucleotide SSRs when compared with di-nucleotide SSRs in ESTs than genomic sequences.<sup><a reveal-id="DSQ028C23" data-open="DSQ028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSQ028C36" data-open="DSQ028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> However, recently the larger number of di-nucleotide SSRs than tri-nucleotide SSRs has been reported in pigeonpea ESTs.<sup><a reveal-id="DSQ028C37" data-open="DSQ028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The frequency and distribution of SSRs have been proposed to be dependent on various factors such as size of data set, tools and criteria used.<sup><a reveal-id="DSQ028C23" data-open="DSQ028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The identification of SSRs provides a very cost-effective option to develop functional markers for various marker-assisted breeding purposes.</p><p>The analysis of sequence conservation helps in transfer of knowledge from model plants to chickpea for functional genomic studies. A large number (60–72%) of chickpea transcripts showed significant similarity with legumes at protein level as expected except for <em>Pisum</em>, indicating that their function might also be conserved. The low similarity with <em>Pisum</em> unigenes may be due to the availability of lesser number of sequences. The low similarity of chickpea transcripts with monocot proteomes when compared with dicot proteomes is also not unexpected due to the phylogenetic divergence of monocots and dicots during evolution. A significant number of transcripts were found to be conserved only in legumes, which may perform legume-specific functions. Interestingly, about 15% of the transcripts did not show significant homology with any other sequences, which may be novel and perform species-specific functions. The lineage- and species-specific genes have been identified in other plant species, including legumes as well.<sup><a reveal-id="DSQ028C38  DSQ028C40" data-open="DSQ028C38  DSQ028C40" class="link link-ref link-reveal xref-bibr">38–40</a></sup> The study of these genes will be very important to dissect the lineage- or species-specific cellular processes and study evolutionary processes such as speciation and adaptation. Further, the chickpea transcripts were found to belong to various functional categories conserved in other plants also. However, overrepresentation of few functional categories might provide a clue towards specific functions/pathways operative in legumes or chickpea. Likewise, the conservation of transcription factor families indicates the presence of conserved gene regulatory machinery in chickpea. However, the lineage- and/or species-specific evolutionary expansion and contraction along with unique gene expression patterns of some of transcription factors may contribute to the legume-specific traits.</p><p>In conclusion, we have demonstrated the use of short-read sequence data to rapidly characterize a draft transcriptome of an organism. The strategy of <em>de novo</em> assembly described here can be potentially used for any species. In addition, our study contributes a significant non-redundant set of 53 409 transcripts in chickpea. The detailed analyses of the data set has provided several important features of chickpea transcriptome such as GC content, conserved genes across legumes and other plant species, assignment of functional categories and identification of SSRs. It is anticipated that this study is a significant contribution towards development of genomic resources for chickpea and will accelerate functional genomic studies and breeding programmes.</p>                            <h2 scrollto-destination=5495371 id="5495371" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5495374 id="5495374" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported financially by the Department of Biotechnology, Government of India, New Delhi, under the Next Generation Challenge Programme on Chickpea Genomics.</p>                            <h2 scrollto-destination=5495376 id="5495376" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graham</div> <div class="given-names">P.H.</div></div>,  <div class="name"><div class="surname">Vance</div> <div class="given-names">C.P.</div></div>. </div><div class="article-title">Legumes: importance and constraints to greater use</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">131</div> (pg. <div class="fpage">872</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.017004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.017004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%3A%20importance%20and%20constraints%20to%20greater%20use&amp;author=P.H.%20Graham&amp;author=C.P.%20Vance&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.017004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.017004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.017004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Millan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Clarke</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Siddique</div> <div class="given-names">K.H.M.</div></div>, et al. </div><div class="article-title">Chickpea molecular breeding: new tools and concepts</div>, <div class="source">Euphytica</div> , <div class="year">2006</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">81</div>-<div class="lpage">103</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10681-006-4261-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10681-006-4261-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chickpea%20molecular%20breeding%3A%20new%20tools%20and%20concepts&amp;author=T.%20Millan&amp;author=H.J.%20Clarke&amp;author=K.H.M.%20Siddique&amp;publication_year=2006&amp;journal=Euphytica&amp;volume=147&amp;pages=81-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-006-4261-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-006-4261-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-006-4261-4"> </span></div> </div></div></div></div></div><div content-id="DSQ028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genomics-assisted breeding for crop improvement</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">621</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tplants.2005.10.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tplants.2005.10.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics-assisted%20breeding%20for%20crop%20improvement&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Plant%20Sci.&amp;volume=10&amp;pages=621-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2005.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2005.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2005.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16290213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Advances in cereal genomics and applications in crop breeding</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">490</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tibtech.2006.08.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tibtech.2006.08.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20cereal%20genomics%20and%20applications%20in%20crop%20breeding&amp;author=R.K.%20Varshney&amp;author=D.A.%20Hoisington&amp;author=A.K.%20Tyagi&amp;publication_year=2006&amp;journal=Trends%20Biotechnol.&amp;volume=24&amp;pages=490-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2006.08.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2006.08.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2006.08.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16956681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buhariwalla</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Jayashree</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Eshwar</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">Development of ESTs from chickpea roots and their use in diversity analysis of the <em>Cicer</em> genus</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> pg. <div class="fpage">16</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-5-16" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-5-16</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20ESTs%20from%20chickpea%20roots%20and%20their%20use%20in%20diversity%20analysis%20of%20the%20Cicer%20genus&amp;author=H.K.%20Buhariwalla&amp;author=B.%20Jayashree&amp;author=K.%20Eshwar&amp;author=J.H.%20Crouch&amp;publication_year=2005&amp;journal=BMC%20Plant%20Biol.&amp;volume=5&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-5-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-5-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-5-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16107212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Q.Y.</div></div>, et al. </div><div class="article-title">Comparative analysis of ESTs in response to drought stress in chickpea (<em>C. arietinum</em> L.)</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2008</div>, vol. <div class="volume">376</div> (pg. <div class="fpage">578</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2008.09.030" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbrc.2008.09.030</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20ESTs%20in%20response%20to%20drought%20stress%20in%20chickpea%20(C.%20arietinum%20L.)&amp;author=W.R.%20Gao&amp;author=X.S.%20Wang&amp;author=Q.Y.%20Liu&amp;publication_year=2008&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=376&amp;pages=578-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2008.09.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2008.09.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2008.09.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18804093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashraf</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ghai</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Barman</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Comparative analyses of genotype dependent expressed sequence tags and stress-responsive transcriptome of chickpea wilt illustrate predicted and unexpected genes and novel regulators of plant immunity</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">415</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-415" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-415</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20of%20genotype%20dependent%20expressed%20sequence%20tags%20and%20stress-responsive%20transcriptome%20of%20chickpea%20wilt%20illustrate%20predicted%20and%20unexpected%20genes%20and%20novel%20regulators%20of%20plant%20immunity&amp;author=N.%20Ashraf&amp;author=D.%20Ghai&amp;author=P.%20Barman&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=415" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-415" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-415" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-415"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19732460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Hiremath</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Lekha</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A comprehensive resource of drought- and salinity-responsive ESTs for gene discovery and marker development in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">523</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-523" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-523</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20resource%20of%20drought-%20and%20salinity-responsive%20ESTs%20for%20gene%20discovery%20and%20marker%20development%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.K.%20Varshney&amp;author=P.J.%20Hiremath&amp;author=P.%20Lekha&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19912666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chattopadhyay</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Analysis of gene expression in response to water deficit of chickpea (<em>Cicer arietinum</em> L.) varieties differing in drought tolerance</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">24</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-24" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-24</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene%20expression%20in%20response%20to%20water%20deficit%20of%20chickpea%20(Cicer%20arietinum%20L.)%20varieties%20differing%20in%20drought%20tolerance&amp;author=D.%20Jain&amp;author=D.%20Chattopadhyay&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20144227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mantri</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Ford</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Coram</div> <div class="given-names">T.E.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">E.C.</div></div>. </div><div class="article-title">Transcriptional profiling of chickpea genes differentially regulated in response to high-salinity, cold and drought</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">303</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-8-303" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-8-303</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20of%20chickpea%20genes%20differentially%20regulated%20in%20response%20to%20high-salinity%2C%20cold%20and%20drought&amp;author=N.L.%20Mantri&amp;author=R.%20Ford&amp;author=T.E.%20Coram&amp;author=E.C.%20Pang&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17764573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Molina</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rotter</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Horres</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">SuperSAGE: the drought stress-responsive transcriptome of chickpea roots</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">553</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-553" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-553</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SuperSAGE%3A%20the%20drought%20stress-responsive%20transcriptome%20of%20chickpea%20roots&amp;author=C.%20Molina&amp;author=B.%20Rotter&amp;author=R.%20Horres&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=553" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19025623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shukla</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Raha</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tripathi</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Chattopadhyay</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Expression of CAP2, an APETALA2-family transcription factor from chickpea, enhances growth and tolerance to dehydration and salt stress in transgenic tobacco</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">113</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.106.081752" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.106.081752</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20CAP2%2C%20an%20APETALA2-family%20transcription%20factor%20from%20chickpea%2C%20enhances%20growth%20and%20tolerance%20to%20dehydration%20and%20salt%20stress%20in%20transgenic%20tobacco&amp;author=R.K.%20Shukla&amp;author=S.%20Raha&amp;author=V.%20Tripathi&amp;author=D.%20Chattopadhyay&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=142&amp;pages=113-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.081752" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.081752" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.081752"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16844836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tripathi</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Parasuraman</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Laxmi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chattopadhyay</div> <div class="given-names">D.</div></div>. </div><div class="article-title">CIPK6, a CBL-interacting protein kinase is required for development and salt tolerance in plants</div>, <div class="source">Plant J.</div> , <div class="year">2009</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">778</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2009.03812.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2009.03812.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CIPK6%2C%20a%20CBL-interacting%20protein%20kinase%20is%20required%20for%20development%20and%20salt%20tolerance%20in%20plants&amp;author=V.%20Tripathi&amp;author=B.%20Parasuraman&amp;author=A.%20Laxmi&amp;author=D.%20Chattopadhyay&amp;publication_year=2009&amp;journal=Plant%20J.&amp;volume=58&amp;pages=778-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.58.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.58.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.58.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19187042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morozova</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Hirst</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Marra</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Applications of new sequencing technologies for transcriptome analysis</div>, <div class="source">Annu. Rev. Genomics Hum. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">135</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev-genom-082908-145957" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev-genom-082908-145957</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20new%20sequencing%20technologies%20for%20transcriptome%20analysis&amp;author=O.%20Morozova&amp;author=M.%20Hirst&amp;author=M.A.%20Marra&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&amp;volume=10&amp;pages=135-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genom-082908-145957" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genom-082908-145957" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genom-082908-145957"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19715439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vera</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Wheat</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Fescemyer</div> <div class="given-names">H.W.</div></div>, et al. </div><div class="article-title">Rapid transcriptome characterization for a nonmodel organism using 454 pyrosequencing</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1636</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-294X.2008.03666.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-294X.2008.03666.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20transcriptome%20characterization%20for%20a%20nonmodel%20organism%20using%20454%20pyrosequencing&amp;author=J.C.%20Vera&amp;author=C.W.%20Wheat&amp;author=H.W.%20Fescemyer&amp;publication_year=2008&amp;journal=Mol.%20Ecol.&amp;volume=17&amp;pages=1636-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2008.17.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2008.17.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2008.17.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Development of a EST dataset and characterization of EST-SSRs in a traditional Chinese medicinal plant, <em>Epimedium sagittatum</em> (Sieb. Et Zucc.) Maxim</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">94</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-94" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-94</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20EST%20dataset%20and%20characterization%20of%20EST-SSRs%20in%20a%20traditional%20Chinese%20medicinal%20plant%2C%20Epimedium%20sagittatum%20(Sieb.%20Et%20Zucc.)%20Maxim&amp;author=S.%20Zeng&amp;author=G.%20Xiao&amp;author=J.%20Guo&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-94" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-94" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-94"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20141623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.074492.107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.074492.107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jackman</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>. </div><div class="article-title">ABySS: a parallel assembler for short read sequence data</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1117</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.089532.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.089532.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABySS%3A%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&amp;author=J.T.%20Simpson&amp;author=K.%20Wong&amp;author=S.D.%20Jackman&amp;author=J.E.%20Schein&amp;author=S.J.%20Jones&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089532.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089532.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089532.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Aluru</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Parallel short sequence assembly of transcriptomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> <div class="issue">Suppl 1</div>pg. <div class="fpage">S14</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-10-S1-S14" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-10-S1-S14</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallel%20short%20sequence%20assembly%20of%20transcriptomes&amp;author=B.G.%20Jackson&amp;author=P.S.%20Schnable&amp;author=S.%20Aluru&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=S14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-S1-S14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-S1-S14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-S1-S14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19208113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Jackman</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">C.B.</div></div>, et al. </div><div class="article-title"><em>De novo</em> transcriptome assembly with ABySS</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2872</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp367" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp367</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20assembly%20with%20ABySS&amp;author=I.%20Birol&amp;author=S.D.%20Jackman&amp;author=C.B.%20Nielsen&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19528083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Validation of internal control genes for quantitative gene expression studies in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2010</div>, vol. <div class="volume">396</div> (pg. <div class="fpage">283</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2010.04.079" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbrc.2010.04.079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20internal%20control%20genes%20for%20quantitative%20gene%20expression%20studies%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Garg&amp;author=A.%20Sahoo&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2010&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=396&amp;pages=283-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2010.04.079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2010.04.079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2010.04.079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20399753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carels</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Two classes of genes in plants</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">1819</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/" class="link link-pub-id-doi" target="_blank">https://doi.org/</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20classes%20of%20genes%20in%20plants&amp;author=N.%20Carels&amp;author=G.%20Bernardi&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1819-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tibtech.2004.11.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tibtech.2004.11.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Libault</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Joshi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Benedito</div> <div class="given-names">V.A.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Udvardi</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Stacy</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Legume transcription factor genes: what makes legumes so special</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">991</div>-<div class="lpage">1001</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.144105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.144105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20transcription%20factor%20genes%3A%20what%20makes%20legumes%20so%20special&amp;author=M.%20Libault&amp;author=T.%20Joshi&amp;author=V.A.%20Benedito&amp;author=D.%20Xu&amp;author=M.K.%20Udvardi&amp;author=G.%20Stacy&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=991-1001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.144105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.144105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.144105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19726573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08670" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08670</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velculescu</div> <div class="given-names">V.E.</div></div>,  <div class="name"><div class="surname">Kinzler</div> <div class="given-names">K.W.</div></div>. </div><div class="article-title">Gene expression analysis goes digital</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">878</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt0807-878" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt0807-878</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20analysis%20goes%20digital&amp;author=V.E.%20Velculescu&amp;author=K.W.%20Kinzler&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=878-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt0807-878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt0807-878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt0807-878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17687366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Snyder</div> <div class="given-names">M.</div></div>. </div><div class="article-title">RNA-Seq: a revolutionary tool for transcriptomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">57</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2484" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg2484</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics&amp;author=Z.%20Wang&amp;author=M.%20Gerstein&amp;author=M.%20Snyder&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=57-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuster</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Next-generation sequencing transforms today's biology</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">16</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth1156" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth1156</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20transforms%20today's%20biology&amp;author=S.C.%20Schuster&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=16-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18165802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibbons</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Janson</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Johnston</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abbot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rokas</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Benchmarking next-generation transcriptome sequencing for functional and evolutionary genomics</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2731</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msp188" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msp188</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Benchmarking%20next-generation%20transcriptome%20sequencing%20for%20functional%20and%20evolutionary%20genomics&amp;author=J.G.%20Gibbons&amp;author=E.M.%20Janson&amp;author=C.T.%20Hittinger&amp;author=M.%20Johnston&amp;author=P.%20Abbot&amp;author=A.%20Rokas&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=26&amp;pages=2731-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19706727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Goldberg</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Town</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">Sequencing <em>Medicago truncatula</em> expressed sequenced tags using 454 Life Sciences technology</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">272</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-7-272" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-7-272</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20Medicago%20truncatula%20expressed%20sequenced%20tags%20using%20454%20Life%20Sciences%20technology&amp;author=F.%20Cheung&amp;author=B.J.%20Haas&amp;author=S.M.%20Goldberg&amp;author=G.D.%20May&amp;author=Y.%20Xiao&amp;author=C.D.%20Town&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=272" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">227</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Three sequenced legume genomes and many crop species: rich opportunities for translational genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">970</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.144659" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.144659</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20sequenced%20legume%20genomes%20and%20many%20crop%20species%3A%20rich%20opportunities%20for%20translational%20genomics&amp;author=S.B.%20Cannon&amp;author=G.D.%20May&amp;author=S.A.%20Jackson&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=970-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.144659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.144659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.144659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19759344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carels</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hatey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jabbari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Compositional properties of homologous coding sequences from plants</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">45</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/PL00006282" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/PL00006282</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20properties%20of%20homologous%20coding%20sequences%20from%20plants&amp;author=N.%20Carels&amp;author=P.%20Hatey&amp;author=K.%20Jabbari&amp;author=G.%20Bernardi&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol.&amp;volume=46&amp;pages=45-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9419224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vinogradov</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">DNA helix: the importance of being GC-rich</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1838</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkg296" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkg296</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20helix%3A%20the%20importance%20of%20being%20GC-rich&amp;author=A.E.%20Vinogradov&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=1838-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cantor</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Broude</div> <div class="given-names">N.E.</div></div>. </div><div class="article-title">GC/AT-content spikes as genomic punctuation marks</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">16855</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0407821101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0407821101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GC%2FAT-content%20spikes%20as%20genomic%20punctuation%20marks&amp;author=L.%20Zhang&amp;author=S.%20Kasif&amp;author=C.R.%20Cantor&amp;author=N.E.%20Broude&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=16855-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407821101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407821101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407821101"> </span></div> </div></div></div></div></div><div content-id="DSQ028C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dang</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">B.Z.</div></div>, et al. </div><div class="article-title">Generation of expressed sequence tags (ESTs) for gene discovery and marker development in cultivated peanut</div>, <div class="source">Crop Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">346</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2135/cropsci2005.0346" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2135/cropsci2005.0346</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20expressed%20sequence%20tags%20(ESTs)%20for%20gene%20discovery%20and%20marker%20development%20in%20cultivated%20peanut&amp;author=M.%20Luo&amp;author=P.%20Dang&amp;author=B.Z.%20Guo&amp;publication_year=2005&amp;journal=Crop%20Sci.&amp;volume=45&amp;pages=346-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2005.0346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2005.0346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2005.0346"> </span></div> </div></div></div></div></div><div content-id="DSQ028C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raju</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Gnanesh</div> <div class="given-names">B.N.</div></div>,  <div class="name"><div class="surname">Lekha</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The first set of EST resource for gene discovery and marker development in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-45" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-45</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20set%20of%20EST%20resource%20for%20gene%20discovery%20and%20marker%20development%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=N.L.%20Raju&amp;author=B.N.%20Gnanesh&amp;author=P.%20Lekha&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20222972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graham</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Silverstein</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">VandenBosch</div> <div class="given-names">K.A.</div></div>. </div><div class="article-title">Computational identification and characterization of novel genes from legumes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1179</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.037531" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.037531</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20identification%20and%20characterization%20of%20novel%20genes%20from%20legumes&amp;author=M.A.%20Graham&amp;author=K.A.%20Silverstein&amp;author=S.B.%20Cannon&amp;author=K.A.%20VandenBosch&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1179-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.037531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.037531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.037531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15266052" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Identification and characterization of lineage-specific genes within the Poaceae</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">1311</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.107.104513" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.107.104513</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20lineage-specific%20genes%20within%20the%20Poaceae&amp;author=M.A.%20Campbell&amp;author=W.%20Zhu&amp;author=N.%20Jiang&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=145&amp;pages=1311-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.104513" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.104513" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.104513"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17951464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ028C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ028C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moghe</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ouyang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Comparative analyses reveal distinct sets of lineage-specific genes within <em>Arabidopsis thaliana</em></div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">41</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-10-41" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2148-10-41</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20reveal%20distinct%20sets%20of%20lineage-specific%20genes%20within%20Arabidopsis%20thaliana&amp;author=H.%20Lin&amp;author=G.%20Moghe&amp;author=S.%20Ouyang&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20152032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=377009&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_table4.xls?Expires=2147483647&amp;Signature=kddDsZqjpOPBp2pm-UUVe~3DBnvBfm6isyQx6XnHqHmAGjpwaHFhvFGYntTL1i8kbukGPs95kmbHYLDqMcYIcAXfH-uKUkCcEqkJ3s946yIf9Fa3oK3wLlZApl0CLPKjL1T9eGkoN3iy6dfHFjdzIas54ktSlURa39aDf2jMixYSF2C5vnCObRt6eFQk57dYeaoF3XL~B45VDwqIxeXMJBrUM6~kk6eTPTKaD0zwVz6sgUleCpgwsfbJ-pq7fF-L04CnPcN00gT32x0IwHW8JDNVMo7UPCI56vhFGTXTA0Unen529jrfmW0kQJG3xs6EvjDtTLuFzI~TFG7FC4XxQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_table3.pdf?Expires=2147483647&amp;Signature=ONht6lnMTwW0SEoioD5M7A227eeXCcXUOQeMO8XjdzsVJdkNzVBz5z85NqYD26sIYHRJHwfihkqk1w7vrUO0Qz6-q5s0tMYBLClS0Us96v7lnVrCYMzp7oxry6ZHhA9ITEtLxjPO46FVWFZk0am4H6lnzXV2HO~PVER1dlAVrJmOsrMH5iDx-LuWyv3yTRVolp4s0Rm9zwXbnVgOdaokHc-h7FPazwQCwrg3ZTflEXraAZZKasEZLS6SVcjDzO-rCNX~mdR-4dszx6CozHly4FMj0iSlFDRguO5aGZ~C5tyXkrf8t7bdLpEtYTPHRajJdsRpAMu5~cVvXo6rHzJH4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_table2.pdf?Expires=2147483647&amp;Signature=OaltKvkAjNCWSkoZ0TqOluivkBKxGiNawR-xLiXY4FJr4j8lIIlwGQqBn0DRrcWuDBc~BFXN9keqIFNH3QJ5oE0yMgpaeXUKjAyf3Cye94XETzLEAuPbWl78NJQsGcV86916cLran6Mcse5cM7DdsCxGoE3~Y~tio4fYiC3Gw3-mTeJ2mywGmlXV5t8NiwNCxKS31amfgmcmjdq994HRUGi6TrUKTRh3rRKeWVa1XXrhDN2aKUcEF7dFOO21EoMQxwJMderJs-gSeYIPbCPMpEf6IS5NV2B08F2HXIpykAzNCKiLTxn2NXcopLNH15IXof4MlSj7A7Zgb8WvmQPyJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_table1.pdf?Expires=2147483647&amp;Signature=SnNA8Xbz-ZWubEzu-zRmUwbqjWeQc4Ql~b~Vg4HQYUZEmb8kvHyL7nIo8zcEbQIH-HLPzx4P8KGTUDZderUuSIkL~m8iosqL6B2cXMjE9G2mN3qwgOlAApTIgPTpP2KciYpMmZW7IOGzPtAO1jK~l0U8YPzcIXD-GAMJGa51vadqbLHMP2xfRUH2ZzGe1y6W9lae~acq6As39U8pnOsr2tWiy~bJQDmDWrBaIi3OeiJy~iMkF72cePv61Pw7t3rPEA0qQVXSkvoUG8WgVlD5kzyZTZNnn76CkbNlzLFNlm-WhEY0X3t0hBTgWMRtqjmPHLZEGqRlAn6Lyo2ANZGz4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_fig2.ppt?Expires=2147483647&amp;Signature=DIauzRxKsM1JtLDE-b0H5xSq3lQiQymgNQgGTvq3cXCL~-0hOYrpYdnnYY5IAi1C6z~CN6TIRag7gpFIaOkFg0I8IUu55f-bbFbqyCPsPTj0p~ebgDav5UAJn~QiIqEPsjwIwBUEzlOJVIBzDydpsBHhnKEdUxtzlIrI09G4rvezPtloq72Sdn9COnyL0zrFL76mBQsUW-TOOa19ByWr~~6NB2GYXbiw8g-G2vq1bO4juWOfEcj3~4IAfF2LgacB-Dyd4~bXOu1HIsCI5IjEgDgD0w7IANdRZlO1DRaBKwfEDJfR9-WfYiZPjamrbDO2lEfSfmTntv9epKEChK0KGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq028/2/dsq028-dsq028supp_fig1.ppt?Expires=2147483647&amp;Signature=4k400VEU5yk7dzxoEsJvrwTZTakJ8PyxJ--UE-wEohvQHdjQYuXhkQEwyA1OiZi2V4fS02gNhh3SgTaoTmMEiqA2GrbG~Dx7AHW4P3yS8if5ysvt~fqBQBeGlLvRW01DOXHTVXlZb9HWnomvQQv5JizJNPuixw7MLHY~7dxV6ISn6icXAeNyhKqhzsFW2QdoFe8AIt9G-lGwVAeZxw~eAgEGjJTBxaTuV-f-SeecQufNxYgbHSSGC0BE7ZQPbuFXdQ4I0UPyLZgXzRkD9rchcZ-beKslROnsTKHT4Wq5wEsS7N6um6Phy1dtRmKQnvXcl2E9suq0K-HJhuMJ2HJYWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,715</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">266</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="377009"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f1%2f53%2f377009">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000288188700005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (266)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F1%2F53%2F377009" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/377009'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="De Novo Assembly of Chickpea Transcriptome Using Short Reads for Gene Discovery and Marker Identification" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f53%2f377009">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

