

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/6/613/551332","@type":"ScholarlyArticle","name":"Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in  Brassica napus ","datePublished":"2014-07-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2014-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu024","keywords":["associative transcriptomics","SNP","GEM","glucosinolate"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/6/613/551332","author":[{"name":"Lu, Guangyuan","affiliation":"1  Centre for Novel Agricultural Products, Department of Biology ,  University of York ,  Heslington, York YO10 5DD ,  UK    2  Oil Crops Research Institute, CAAS ,  Wuhan 430062, Hubei ,  China","@type":"Person"},{"name":"Harper, Andrea L.","affiliation":"1  Centre for Novel Agricultural Products, Department of Biology ,  University of York ,  Heslington, York YO10 5DD ,  UK","@type":"Person"},{"name":"Trick, Martin","affiliation":"3  John Innes Centre, Norwich Research Park ,  Norwich, Norfolk NR4 7UH ,  UK","@type":"Person"},{"name":"Morgan, Colin","affiliation":"3  John Innes Centre, Norwich Research Park ,  Norwich, Norfolk NR4 7UH ,  UK","@type":"Person"},{"name":"Fraser, Fiona","affiliation":"3  John Innes Centre, Norwich Research Park ,  Norwich, Norfolk NR4 7UH ,  UK","@type":"Person"},{"name":"O'Neill, Carmel","affiliation":"3  John Innes Centre, Norwich Research Park ,  Norwich, Norfolk NR4 7UH ,  UK","@type":"Person"},{"name":"Bancroft, Ian","affiliation":"1  Centre for Novel Agricultural Products, Department of Biology ,  University of York ,  Heslington, York YO10 5DD ,  UK","@type":"Person"}],"description":"Abstract.  Breeding new varieties with low seed glucosinolate (GS) concentrations has long been a prime target in Brassica napus. In this study, a novel associa","pageStart":"613","pageEnd":"625","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in  Brassica napus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in  Brassica napus " />
    <meta property="og:description" content="Abstract.  Breeding new varieties with low seed glucosinolate (GS) concentrations has long been a prime target in Brassica napus. In this study, a novel associa" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/6/613/551332" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-07-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Lu, Guangyuan" /><meta name="citation_author_institution" content="Centre for Novel Agricultural Products, Department of Biology, University of York, Heslington, York YO10 5DD, UK" /><meta name="citation_author_institution" content="Oil Crops Research Institute, CAAS, Wuhan 430062, Hubei, China" /><meta name="citation_author" content="Harper, Andrea L." /><meta name="citation_author_institution" content="Centre for Novel Agricultural Products, Department of Biology, University of York, Heslington, York YO10 5DD, UK" /><meta name="citation_author" content="Trick, Martin" /><meta name="citation_author_institution" content="John Innes Centre, Norwich Research Park, Norwich, Norfolk NR4 7UH, UK" /><meta name="citation_author" content="Morgan, Colin" /><meta name="citation_author_institution" content="John Innes Centre, Norwich Research Park, Norwich, Norfolk NR4 7UH, UK" /><meta name="citation_author" content="Fraser, Fiona" /><meta name="citation_author_institution" content="John Innes Centre, Norwich Research Park, Norwich, Norfolk NR4 7UH, UK" /><meta name="citation_author" content="O'Neill, Carmel" /><meta name="citation_author_institution" content="John Innes Centre, Norwich Research Park, Norwich, Norfolk NR4 7UH, UK" /><meta name="citation_author" content="Bancroft, Ian" /><meta name="citation_author_institution" content="Centre for Novel Agricultural Products, Department of Biology, University of York, Heslington, York YO10 5DD, UK" /><meta name="citation_title" content="Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in Brassica napus" /><meta name="citation_firstpage" content="613" /><meta name="citation_lastpage" content="625" /><meta name="citation_doi" content="10.1093/dnares/dsu024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2014/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The chemical diversity and distribution of glucosinolates and isothiocyanates among plants; Phytochemistry;  citation_year=2001;  citation_volume=56;  citation_pages=5-51; " /><meta name="citation_reference" content="citation_title=Glucosinolate research in the Arabidopsis era; Trends Plant Sci.;  citation_year=2002;  citation_volume=7;  citation_pages=263-70; " /><meta name="citation_reference" content="citation_title=Glucosinolates—biochemistry, genetics and biological activity; Plant Growth Regul.;  citation_year=2001;  citation_volume=34;  citation_pages=91-103; " /><meta name="citation_reference" content="citation_title=RFLP mapping of quantitative trait loci controlling seed aliphatic glucosinolate content in oilseed rape (Brassica napus L.); Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=802-8; " /><meta name="citation_reference" content="citation_title=Leaf glucosinolate profiles and their relationship to pest and disease resistance in oilseed rape; Euphytica;  citation_year=1992;  citation_volume=63;  citation_pages=71-83; " /><meta name="citation_reference" content="citation_title=Variation in the glucosinolate content of vegetative tissues of Chinese lines of Brassica napus L.; Ann. Appl. Biol.;  citation_year=1999;  citation_volume=134;  citation_pages=131-6; " /><meta name="citation_reference" content="citation_title=Allylglucosinolate and herbivorous caterpillars: a contrast in toxicity and tolerance; Science;  citation_year=1978;  citation_volume=200;  citation_pages=1296-8; " /><meta name="citation_reference" content="citation_title=Comparative analysis of insect resistance QTL and QTL controlling the myrosinase/glucosinolate system in Arabidopsis thaliana; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=325-32; " /><meta name="citation_reference" content="citation_title=The association of Sclerotinia sclerotiorum resistance with glucosinolates in Brassica napus double-low DH population; J. Plant Pathol.;  citation_year=2008;  citation_volume=90;  citation_pages=43-8; " /><meta name="citation_reference" content="citation_title=The biosynthesis of glucosinolates; Trends Plant Sci.;  citation_year=1997;  citation_volume=2;  citation_pages=425-31; " /><meta name="citation_reference" content="citation_title=Biology and biochemistry of glucosinolates; Annu. Rev. Plant Biol.;  citation_year=2006;  citation_volume=57;  citation_pages=303-33; " /><meta name="citation_reference" content="citation_title=Arabidopsis cytochrome P450 s that catalyze the first step of tryptophan-dependent indole-3-acetic acid biosynthesis; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=2379-84; " /><meta name="citation_reference" content="citation_title=Genetic analysis, expression and molecular characterization of BoGSL-ELONG, a major gene involved in the aliphatic glucosinolate pathway of Brassica species; Genetics;  citation_year=2002;  citation_volume=162;  citation_pages=1937-43; " /><meta name="citation_reference" content="citation_title=In planta side-chain glucosinolate modification in Arabidopsis by introduction of dioxygenase Brassica homolog BoGSL-ALK; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=1116-21; " /><meta name="citation_reference" content="citation_title=CYP79F1 and CYP79F2 have distinct functions in the biosynthesis of aliphatic glucosinolates in Arabidopsis; Plant J.;  citation_year=2003;  citation_volume=33;  citation_pages=923-37; " /><meta name="citation_reference" content="citation_title=Arabidopsis glucosyltransferase UGT74B1 functions in glucosinolate biosynthesis and auxin homeostasis; Plant J.;  citation_year=2004;  citation_volume=40;  citation_pages=893-908; " /><meta name="citation_reference" content="citation_title=Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=6478-83; " /><meta name="citation_reference" content="citation_title=A redox active isopropylmalate dehydrogenase functions in the biosynthesis of glucosinolates and leucine in Arabidopsis; Plant J.;  citation_year=2009;  citation_volume=60;  citation_pages=679-90; " /><meta name="citation_reference" content="citation_title=Omics-based approaches to methionine side chain elongation in Arabidopsis: characterization of the genes encoding methylthioalkylmalate isomerase and methylthioalkylmalate dehydrogenase; Plant Cell Physiol.;  citation_year=2009;  citation_volume=50;  citation_pages=1181-90; " /><meta name="citation_reference" content="citation_title=Bioinformatic analysis of molecular network of glucosinolate biosynthesis; Comput. Biol. Chem.;  citation_year=2011;  citation_volume=35;  citation_pages=10-8; " /><meta name="citation_reference" content="citation_title=Structural and functional evolution of isopropylmalate dehydrogenases in leucine and glucosinolate pathways of Arabidopsis thaliana; J. Biol. Chem.;  citation_year=2011;  citation_volume=286;  citation_pages=28794-801; " /><meta name="citation_reference" content="citation_title=Functional characterization of Arabidopsis isopropylmalate dehydrogenases reveals their important roles in gametophyte development; New Phytol.;  citation_year=2011;  citation_volume=189;  citation_pages=160-75; " /><meta name="citation_reference" content="citation_title=Novel insights into the function of Arabidopsis R2R3-MYB transcription factors regulating aliphatic glucosinolate biosynthesis; Plant Cell Physiol.;  citation_year=2013;  citation_volume=54;  citation_pages=1335-44; " /><meta name="citation_reference" content="citation_title=Evolution of flux control in the glucosinolate pathway in Arabidopsis thaliana; Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=14-23; " /><meta name="citation_reference" content="citation_title=Glucosinolate metabolism and its control; Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=89-100; " /><meta name="citation_reference" content="citation_title=Biosynthesis of glucosinolates-gene discovery and beyond; Trends Plant Sci.;  citation_year=2010;  citation_volume=15;  citation_pages=283-90; " /><meta name="citation_reference" content="citation_title=Molecular genetics of glucosinolate biosynthesis in Brassicas: genetic manipulation and application aspect.; citation_text=ISBN: 978-953-51-0527-5 pp. 189–216; citation_publisher=InTech: Croatia, ; Crop Plant;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Biosynthesis of glucosinolates in the developing silique walls; Phytochemistry;  citation_year=1998;  citation_volume=48;  citation_pages=1145-50; " /><meta name="citation_reference" content="citation_title=Characterization of glucosinolate uptake by leaf protoplasts of Brassica napus; J. Biol. Chem.;  citation_year=2000;  citation_volume=275;  citation_pages=22955-60; " /><meta name="citation_reference" content="citation_title=Long-distance phloem transport of glucosinolates in Arabidopsis; Plant Physiol.;  citation_year=2001;  citation_volume=127;  citation_pages=194-201; " /><meta name="citation_reference" content="citation_title=Update on glucosinolate metabolism and transport; Plant Physiol. Biochem.;  citation_year=2001;  citation_volume=39;  citation_pages=743-58; " /><meta name="citation_reference" content="citation_title=Genetic control of natural variation in Arabidopsis glucosinolate accumulation; Plant Physiol.;  citation_year=2001;  citation_volume=126;  citation_pages=811-25; " /><meta name="citation_reference" content="citation_title=NRT/PTR transporters are essential for translocation of glucosinolate defence compounds to seeds; Nature;  citation_year=2012;  citation_volume=488;  citation_pages=531-4; " /><meta name="citation_reference" content="citation_title=Naturally occurring genetic variation in Arabidopsis thaliana; Annu. Rev. Plant Biol.;  citation_year=2004;  citation_volume=55;  citation_pages=141-72; " /><meta name="citation_reference" content="citation_title=Mapping the genome of rapeseed (Brassica napus L.).1. Construction of an RFLP linkage map and localization of QTLs for seed glucosinolate content; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=90;  citation_pages=194-204; " /><meta name="citation_reference" content="citation_title=Homoeologue loci control the accumulation of seed glucosinolates in oilseed rape (Brassica napus); Genome;  citation_year=2003;  citation_volume=46;  citation_pages=454-60; " /><meta name="citation_reference" content="citation_title=Detection of loci controlling seed glucosinolate content and their association with Sclerotinia resistance in Brassica napus; Plant Breed.;  citation_year=2003;  citation_volume=122;  citation_pages=19-23; " /><meta name="citation_reference" content="citation_title=Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (Brassica napus L.): 1. Identification of genomic regions from winter germplasm; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=549-61; " /><meta name="citation_reference" content="citation_title=Characterization of metabolite quantitative trait loci and metabolic networks that control glucosinolate concentration in the seeds and leaves of Brassica napus; New Phytol.;  citation_year=2011;  citation_volume=193;  citation_pages=96-108; " /><meta name="citation_reference" content="citation_title=Status and prospects of association mapping in plants; Plant Genome;  citation_year=2008;  citation_volume=1;  citation_pages=5-20; " /><meta name="citation_reference" content="citation_title=Association mapping for enhancing maize (Zea mays L.) genetic improvement; Crop Sci.;  citation_year=2011;  citation_volume=51;  citation_pages=433-49; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies identified three independent polymorphisms associated with α-Tocopherol content in maize kernels; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e36807" /><meta name="citation_reference" content="citation_title=Rare genetic variation at Zea mays crtRB1 increases β-carotene in maize grain; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=322-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels; Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=43-50; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm; Nat. Genet.;  citation_year=2011;  citation_volume=44;  citation_pages=32-9; " /><meta name="citation_reference" content="citation_title=Extent and structure of linkage disequilibrium in canola quality winter rapeseed (Brassica napus L.); Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=921-31; " /><meta name="citation_reference" content="citation_title=Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (Brassica napus ssp. napus); Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=1035-49; " /><meta name="citation_reference" content="citation_title=Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (Brassica napus L.); DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=mRNA-Seq whole-transcriptome analysis of a single cell; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=377-82; " /><meta name="citation_reference" content="citation_title=Genome dissection in the polyploid crop oilseed rape by transcriptome sequencing; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=762-6; " /><meta name="citation_reference" content="citation_title=Associative transcriptomics of traits in the polyploid crop species Brassica napus; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=798-802; " /><meta name="citation_reference" content="citation_title=The identification and mapping of candidate genes and QTL involved in the fatty acid desaturation pathway in Brassica napus; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1075-90; " /><meta name="citation_reference" content="citation_title=Use of mRNA-Seq to discriminate contributions to the transcriptome from the constituent genomes of the polyploid crop species Brassica napus; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=247-60; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism (SNP) discovery in the polyploid Brassica napus using Solexa transcriptome sequencing; Plant Biotechnol. J.;  citation_year=2009;  citation_volume=7;  citation_pages=334-46; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study; Mol. Ecol.;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=A unified mixed model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=WGCNA: an R package for weighted correlation network analysis; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=559-71; " /><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2498-504; " /><meta name="citation_reference" content="citation_title=agriGO: a GO analysis toolkit for the agricultural community; Nucleic. Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=64-70; " /><meta name="citation_reference" content="citation_title=A gene controlling variation in Arabidopsis glucosinolate composition is part of the methionine chain elongation pathway; Plant Physiol.;  citation_year=2001;  citation_volume=127;  citation_pages=1077-88; " /><meta name="citation_reference" content="citation_title=The plastidic bile acid transporter 5 is required for the biosynthesis of methionine-derived glucosinolates in Arabidopsis thaliana; Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=1813-29; " /><meta name="citation_reference" content="citation_title=Arabidopsis branched-chain aminotransferase 3 functions in both amino acid and glucosinolate biosynthesis; Plant Physiol.;  citation_year=2008;  citation_volume=146;  citation_pages=1028-39; " /><meta name="citation_reference" content="citation_title=A complex interplay of three R2R3 MYB transcription factors determines the profile of aliphatic glucosinolates in Arabidopsis; Plant Physiol.;  citation_year=2010;  citation_volume=153;  citation_pages=348-63; " /><meta name="citation_reference" content="citation_title=Genome analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; Japan J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=An association transcriptomics approach to the prediction of hybrid performance; Mol. Breed.;  citation_year=2010;  citation_volume=26;  citation_pages=91-106; " /><meta name="citation_reference" content="citation_title=Can metals defend plants against biotic stress?; Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=288-95; " /><meta name="citation_reference" content="citation_title=Interactions between metal ion toxicity and defences against biotic stress: glucosinolates and benzoxazinoids as case studies; For. Snow Landsc. Res.;  citation_year=2006;  citation_volume=80;  citation_pages=149-60; " /><meta name="citation_reference" content="citation_title=Arabidopsis HMA2, a divalent heavy metal transporting PIB type ATPase, is involved in cytoplasmic Zn2+ homeostasis; Plant Physiol.;  citation_year=2004;  citation_volume=136;  citation_pages=3712-23; " /><meta name="citation_reference" content="citation_title=Influence of zinc hyperaccumulation on glucosinolates in Thlaspi caerulescens; New Phytol.;  citation_year=2001;  citation_volume=151;  citation_pages=621-6; " /><meta name="citation_reference" content="citation_title=Zinc availability in hydroponic culture influences glucosinolate concentrations in Brassica rapa; Hortscience;  citation_year=2004;  citation_volume=39;  citation_pages=84-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/6/613/1462385/dsu024.pdf" /><meta name="description" content="Abstract.  Breeding new varieties with low seed glucosinolate (GS) concentrations has long been a prime target in Brassica napus. In this study, a novel associa" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/6/613/551332" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/6/613/551332" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/6/613/551332");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu024';
    </script>
<title>
	Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in Brassica napus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./551332" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdWcATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHUsIEd1YW5neXVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBmb3IgTm92ZWwgQWdyaWN1bHR1cmFsIFByb2R1Y3RzLCBEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgWW9yaywgSGVzbGluZ3RvbiwgWW9yayBZTzEwIDVERCwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlLCBDQUFTLCBXdWhhbiA0MzAwNjIsIEh1YmVpLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhcnBlciwgQW5kcmVhIEwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBOb3ZlbCBBZ3JpY3VsdHVyYWwgUHJvZHVjdHMsIERlcGFydG1lbnQgb2YgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBZb3JrLCBIZXNsaW5ndG9uLCBZb3JrIFlPMTAgNURELCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRyaWNrLCBNYXJ0aW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKb2huIElubmVzIENlbnRyZSwgTm9yd2ljaCBSZXNlYXJjaCBQYXJrLCBOb3J3aWNoLCBOb3Jmb2xrIE5SNCA3VUgsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yZ2FuLCBDb2xpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkpvaG4gSW5uZXMgQ2VudHJlLCBOb3J3aWNoIFJlc2VhcmNoIFBhcmssIE5vcndpY2gsIE5vcmZvbGsgTlI0IDdVSCwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGcmFzZXIsIEZpb25hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSm9obiBJbm5lcyBDZW50cmUsIE5vcndpY2ggUmVzZWFyY2ggUGFyaywgTm9yd2ljaCwgTm9yZm9sayBOUjQgN1VILCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik8nTmVpbGwsIENhcm1lbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkpvaG4gSW5uZXMgQ2VudHJlLCBOb3J3aWNoIFJlc2VhcmNoIFBhcmssIE5vcndpY2gsIE5vcmZvbGsgTlI0IDdVSCwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW5jcm9mdCwgSWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBOb3ZlbCBBZ3JpY3VsdHVyYWwgUHJvZHVjdHMsIERlcGFydG1lbnQgb2YgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBZb3JrLCBIZXNsaW5ndG9uLCBZb3JrIFlPMTAgNURELCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQXNzb2NpYXRpdmUgVHJhbnNjcmlwdG9taWNzIFN0dWR5IERpc3NlY3RzIHRoZSBHZW5ldGljIEFyY2hpdGVjdHVyZSBvZiBTZWVkIEdsdWNvc2lub2xhdGUgQ29udGVudCBpbiBCcmFzc2ljYSBuYXB1cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjYxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2hlbWljYWwgZGl2ZXJzaXR5IGFuZCBkaXN0cmlidXRpb24gb2YgZ2x1Y29zaW5vbGF0ZXMgYW5kIGlzb3RoaW9jeWFuYXRlcyBhbW9uZyBwbGFudHM7IFBoeXRvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2x1Y29zaW5vbGF0ZSByZXNlYXJjaCBpbiB0aGUgQXJhYmlkb3BzaXMgZXJhOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjYzLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbHVjb3Npbm9sYXRlc+KAlGJpb2NoZW1pc3RyeSwgZ2VuZXRpY3MgYW5kIGJpb2xvZ2ljYWwgYWN0aXZpdHk7IFBsYW50IEdyb3d0aCBSZWd1bC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz05MS0xMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJGTFAgbWFwcGluZyBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBjb250cm9sbGluZyBzZWVkIGFsaXBoYXRpYyBnbHVjb3Npbm9sYXRlIGNvbnRlbnQgaW4gb2lsc2VlZCByYXBlIChCcmFzc2ljYSBuYXB1cyBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz04MDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVhZiBnbHVjb3Npbm9sYXRlIHByb2ZpbGVzIGFuZCB0aGVpciByZWxhdGlvbnNoaXAgdG8gcGVzdCBhbmQgZGlzZWFzZSByZXNpc3RhbmNlIGluIG9pbHNlZWQgcmFwZTsgRXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9NzEtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiB0aGUgZ2x1Y29zaW5vbGF0ZSBjb250ZW50IG9mIHZlZ2V0YXRpdmUgdGlzc3VlcyBvZiBDaGluZXNlIGxpbmVzIG9mIEJyYXNzaWNhIG5hcHVzIEwuOyBBbm4uIEFwcGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTEzNDsgIGNpdGF0aW9uX3BhZ2VzPTEzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGx5bGdsdWNvc2lub2xhdGUgYW5kIGhlcmJpdm9yb3VzIGNhdGVycGlsbGFyczogYSBjb250cmFzdCBpbiB0b3hpY2l0eSBhbmQgdG9sZXJhbmNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTc4OyAgY2l0YXRpb25fdm9sdW1lPTIwMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOTYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgaW5zZWN0IHJlc2lzdGFuY2UgUVRMIGFuZCBRVEwgY29udHJvbGxpbmcgdGhlIG15cm9zaW5hc2UvZ2x1Y29zaW5vbGF0ZSBzeXN0ZW0gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFzc29jaWF0aW9uIG9mIFNjbGVyb3RpbmlhIHNjbGVyb3Rpb3J1bSByZXNpc3RhbmNlIHdpdGggZ2x1Y29zaW5vbGF0ZXMgaW4gQnJhc3NpY2EgbmFwdXMgZG91YmxlLWxvdyBESCBwb3B1bGF0aW9uOyBKLiBQbGFudCBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9NDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJpb3N5bnRoZXNpcyBvZiBnbHVjb3Npbm9sYXRlczsgVHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTQyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvbG9neSBhbmQgYmlvY2hlbWlzdHJ5IG9mIGdsdWNvc2lub2xhdGVzOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MzAzLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBjeXRvY2hyb21lIFA0NTAgcyB0aGF0IGNhdGFseXplIHRoZSBmaXJzdCBzdGVwIG9mIHRyeXB0b3BoYW4tZGVwZW5kZW50IGluZG9sZS0zLWFjZXRpYyBhY2lkIGJpb3N5bnRoZXNpczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0yMzc5LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzLCBleHByZXNzaW9uIGFuZCBtb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbiBvZiBCb0dTTC1FTE9ORywgYSBtYWpvciBnZW5lIGludm9sdmVkIGluIHRoZSBhbGlwaGF0aWMgZ2x1Y29zaW5vbGF0ZSBwYXRod2F5IG9mIEJyYXNzaWNhIHNwZWNpZXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTE5MzctNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHBsYW50YSBzaWRlLWNoYWluIGdsdWNvc2lub2xhdGUgbW9kaWZpY2F0aW9uIGluIEFyYWJpZG9wc2lzIGJ5IGludHJvZHVjdGlvbiBvZiBkaW94eWdlbmFzZSBCcmFzc2ljYSBob21vbG9nIEJvR1NMLUFMSzsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xMTE2LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DWVA3OUYxIGFuZCBDWVA3OUYyIGhhdmUgZGlzdGluY3QgZnVuY3Rpb25zIGluIHRoZSBiaW9zeW50aGVzaXMgb2YgYWxpcGhhdGljIGdsdWNvc2lub2xhdGVzIGluIEFyYWJpZG9wc2lzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTkyMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJhYmlkb3BzaXMgZ2x1Y29zeWx0cmFuc2ZlcmFzZSBVR1Q3NEIxIGZ1bmN0aW9ucyBpbiBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpcyBhbmQgYXV4aW4gaG9tZW9zdGFzaXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ODkzLTkwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T21pY3MtYmFzZWQgaWRlbnRpZmljYXRpb24gb2YgQXJhYmlkb3BzaXMgTXliIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZWd1bGF0aW5nIGFsaXBoYXRpYyBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NjQ3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWRveCBhY3RpdmUgaXNvcHJvcHlsbWFsYXRlIGRlaHlkcm9nZW5hc2UgZnVuY3Rpb25zIGluIHRoZSBiaW9zeW50aGVzaXMgb2YgZ2x1Y29zaW5vbGF0ZXMgYW5kIGxldWNpbmUgaW4gQXJhYmlkb3BzaXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9Njc5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbWljcy1iYXNlZCBhcHByb2FjaGVzIHRvIG1ldGhpb25pbmUgc2lkZSBjaGFpbiBlbG9uZ2F0aW9uIGluIEFyYWJpZG9wc2lzOiBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBnZW5lcyBlbmNvZGluZyBtZXRoeWx0aGlvYWxreWxtYWxhdGUgaXNvbWVyYXNlIGFuZCBtZXRoeWx0aGlvYWxreWxtYWxhdGUgZGVoeWRyb2dlbmFzZTsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTExODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWMgYW5hbHlzaXMgb2YgbW9sZWN1bGFyIG5ldHdvcmsgb2YgZ2x1Y29zaW5vbGF0ZSBiaW9zeW50aGVzaXM7IENvbXB1dC4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuZCBmdW5jdGlvbmFsIGV2b2x1dGlvbiBvZiBpc29wcm9weWxtYWxhdGUgZGVoeWRyb2dlbmFzZXMgaW4gbGV1Y2luZSBhbmQgZ2x1Y29zaW5vbGF0ZSBwYXRod2F5cyBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg2OyAgY2l0YXRpb25fcGFnZXM9Mjg3OTQtODAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGNoYXJhY3Rlcml6YXRpb24gb2YgQXJhYmlkb3BzaXMgaXNvcHJvcHlsbWFsYXRlIGRlaHlkcm9nZW5hc2VzIHJldmVhbHMgdGhlaXIgaW1wb3J0YW50IHJvbGVzIGluIGdhbWV0b3BoeXRlIGRldmVsb3BtZW50OyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODk7ICBjaXRhdGlvbl9wYWdlcz0xNjAtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIGluc2lnaHRzIGludG8gdGhlIGZ1bmN0aW9uIG9mIEFyYWJpZG9wc2lzIFIyUjMtTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZWd1bGF0aW5nIGFsaXBoYXRpYyBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpczsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTEzMzUtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBmbHV4IGNvbnRyb2wgaW4gdGhlIGdsdWNvc2lub2xhdGUgcGF0aHdheSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE0LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbHVjb3Npbm9sYXRlIG1ldGFib2xpc20gYW5kIGl0cyBjb250cm9sOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTg5LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlvc3ludGhlc2lzIG9mIGdsdWNvc2lub2xhdGVzLWdlbmUgZGlzY292ZXJ5IGFuZCBiZXlvbmQ7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjgzLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZ2VuZXRpY3Mgb2YgZ2x1Y29zaW5vbGF0ZSBiaW9zeW50aGVzaXMgaW4gQnJhc3NpY2FzOiBnZW5ldGljIG1hbmlwdWxhdGlvbiBhbmQgYXBwbGljYXRpb24gYXNwZWN0LjsgY2l0YXRpb25fdGV4dD1JU0JOOiA5NzgtOTUzLTUxLTA1MjctNSBwcC4gMTg54oCTMjE2OyBjaXRhdGlvbl9wdWJsaXNoZXI9SW5UZWNoOiBDcm9hdGlhLCA7IENyb3AgUGxhbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb3N5bnRoZXNpcyBvZiBnbHVjb3Npbm9sYXRlcyBpbiB0aGUgZGV2ZWxvcGluZyBzaWxpcXVlIHdhbGxzOyBQaHl0b2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTExNDUtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgZ2x1Y29zaW5vbGF0ZSB1cHRha2UgYnkgbGVhZiBwcm90b3BsYXN0cyBvZiBCcmFzc2ljYSBuYXB1czsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjc1OyAgY2l0YXRpb25fcGFnZXM9MjI5NTUtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmctZGlzdGFuY2UgcGhsb2VtIHRyYW5zcG9ydCBvZiBnbHVjb3Npbm9sYXRlcyBpbiBBcmFiaWRvcHNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9MTk0LTIwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBkYXRlIG9uIGdsdWNvc2lub2xhdGUgbWV0YWJvbGlzbSBhbmQgdHJhbnNwb3J0OyBQbGFudCBQaHlzaW9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTc0My01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjb250cm9sIG9mIG5hdHVyYWwgdmFyaWF0aW9uIGluIEFyYWJpZG9wc2lzIGdsdWNvc2lub2xhdGUgYWNjdW11bGF0aW9uOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz04MTEtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5SVC9QVFIgdHJhbnNwb3J0ZXJzIGFyZSBlc3NlbnRpYWwgZm9yIHRyYW5zbG9jYXRpb24gb2YgZ2x1Y29zaW5vbGF0ZSBkZWZlbmNlIGNvbXBvdW5kcyB0byBzZWVkczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4ODsgIGNpdGF0aW9uX3BhZ2VzPTUzMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsbHkgb2NjdXJyaW5nIGdlbmV0aWMgdmFyaWF0aW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTQxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHRoZSBnZW5vbWUgb2YgcmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKS4xLiBDb25zdHJ1Y3Rpb24gb2YgYW4gUkZMUCBsaW5rYWdlIG1hcCBhbmQgbG9jYWxpemF0aW9uIG9mIFFUTHMgZm9yIHNlZWQgZ2x1Y29zaW5vbGF0ZSBjb250ZW50OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MTk0LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9tb2VvbG9ndWUgbG9jaSBjb250cm9sIHRoZSBhY2N1bXVsYXRpb24gb2Ygc2VlZCBnbHVjb3Npbm9sYXRlcyBpbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzKTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NDU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgbG9jaSBjb250cm9sbGluZyBzZWVkIGdsdWNvc2lub2xhdGUgY29udGVudCBhbmQgdGhlaXIgYXNzb2NpYXRpb24gd2l0aCBTY2xlcm90aW5pYSByZXNpc3RhbmNlIGluIEJyYXNzaWNhIG5hcHVzOyBQbGFudCBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9MTktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSB0cmFpdCBhbmFseXNpcyBvZiBzZWVkIHlpZWxkIGFuZCBvdGhlciBjb21wbGV4IHRyYWl0cyBpbiBoeWJyaWQgc3ByaW5nIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik6IDEuIElkZW50aWZpY2F0aW9uIG9mIGdlbm9taWMgcmVnaW9ucyBmcm9tIHdpbnRlciBnZXJtcGxhc207IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NTQ5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIG1ldGFib2xpdGUgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgYW5kIG1ldGFib2xpYyBuZXR3b3JrcyB0aGF0IGNvbnRyb2wgZ2x1Y29zaW5vbGF0ZSBjb25jZW50cmF0aW9uIGluIHRoZSBzZWVkcyBhbmQgbGVhdmVzIG9mIEJyYXNzaWNhIG5hcHVzOyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz05Ni0xMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXR1cyBhbmQgcHJvc3BlY3RzIG9mIGFzc29jaWF0aW9uIG1hcHBpbmcgaW4gcGxhbnRzOyBQbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTUtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG1hcHBpbmcgZm9yIGVuaGFuY2luZyBtYWl6ZSAoWmVhIG1heXMgTC4pIGdlbmV0aWMgaW1wcm92ZW1lbnQ7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTQzMy00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBpZGVudGlmaWVkIHRocmVlIGluZGVwZW5kZW50IHBvbHltb3JwaGlzbXMgYXNzb2NpYXRlZCB3aXRoIM6xLVRvY29waGVyb2wgY29udGVudCBpbiBtYWl6ZSBrZXJuZWxzOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTM2ODA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFyZSBnZW5ldGljIHZhcmlhdGlvbiBhdCBaZWEgbWF5cyBjcnRSQjEgaW5jcmVhc2VzIM6yLWNhcm90ZW5lIGluIG1haXplIGdyYWluOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTMyMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSBkaXNzZWN0cyB0aGUgZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2Ygb2lsIGJpb3N5bnRoZXNpcyBpbiBtYWl6ZSBrZXJuZWxzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQzLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIG9mIDE0IGFncm9ub21pYyB0cmFpdHMgaW4gcmljZSBsYW5kcmFjZXM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIGZsb3dlcmluZyB0aW1lIGFuZCBncmFpbiB5aWVsZCB0cmFpdHMgaW4gYSB3b3JsZHdpZGUgY29sbGVjdGlvbiBvZiByaWNlIGdlcm1wbGFzbTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0zMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnQgYW5kIHN0cnVjdHVyZSBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIGNhbm9sYSBxdWFsaXR5IHdpbnRlciByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTkyMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gb2YgZ2VuZS1saW5rZWQgU1NSIG1hcmtlcnMgdG8gc2VlZCBnbHVjb3Npbm9sYXRlIGNvbnRlbnQgaW4gb2lsc2VlZCByYXBlIChCcmFzc2ljYSBuYXB1cyBzc3AuIG5hcHVzKTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz0xMDM1LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSBkaXNzZWN0cyB0aGUgZ2VuZXRpYyBhcmNoaXRlY3R1cmUgb2Ygc2VlZCB3ZWlnaHQgYW5kIHNlZWQgcXVhbGl0eSBpbiByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bVJOQS1TZXEgd2hvbGUtdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiBhIHNpbmdsZSBjZWxsOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTM3Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGRpc3NlY3Rpb24gaW4gdGhlIHBvbHlwbG9pZCBjcm9wIG9pbHNlZWQgcmFwZSBieSB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmc7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz03NjItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpdmUgdHJhbnNjcmlwdG9taWNzIG9mIHRyYWl0cyBpbiB0aGUgcG9seXBsb2lkIGNyb3Agc3BlY2llcyBCcmFzc2ljYSBuYXB1czsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTc5OC04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpZGVudGlmaWNhdGlvbiBhbmQgbWFwcGluZyBvZiBjYW5kaWRhdGUgZ2VuZXMgYW5kIFFUTCBpbnZvbHZlZCBpbiB0aGUgZmF0dHkgYWNpZCBkZXNhdHVyYXRpb24gcGF0aHdheSBpbiBCcmFzc2ljYSBuYXB1czsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0xMDc1LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgbVJOQS1TZXEgdG8gZGlzY3JpbWluYXRlIGNvbnRyaWJ1dGlvbnMgdG8gdGhlIHRyYW5zY3JpcHRvbWUgZnJvbSB0aGUgY29uc3RpdHVlbnQgZ2Vub21lcyBvZiB0aGUgcG9seXBsb2lkIGNyb3Agc3BlY2llcyBCcmFzc2ljYSBuYXB1czsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjQ3LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gKFNOUCkgZGlzY292ZXJ5IGluIHRoZSBwb2x5cGxvaWQgQnJhc3NpY2EgbmFwdXMgdXNpbmcgU29sZXhhIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTMzNC00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJlbmNlIG9mIHBvcHVsYXRpb24gc3RydWN0dXJlIHVzaW5nIG11bHRpbG9jdXMgZ2Vub3R5cGUgZGF0YTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3RpbmcgdGhlIG51bWJlciBvZiBjbHVzdGVycyBvZiBpbmRpdmlkdWFscyB1c2luZyB0aGUgc29mdHdhcmUgU1RSVUNUVVJFOiBhIHNpbXVsYXRpb24gc3R1ZHk7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNjExLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaWZpZWQgbWl4ZWQgbW9kZWwgbWV0aG9kIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIHRoYXQgYWNjb3VudHMgZm9yIG11bHRpcGxlIGxldmVscyBvZiByZWxhdGVkbmVzczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V0dDTkE6IGFuIFIgcGFja2FnZSBmb3Igd2VpZ2h0ZWQgY29ycmVsYXRpb24gbmV0d29yayBhbmFseXNpczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01NTktNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9zY2FwZTogYSBzb2Z0d2FyZSBlbnZpcm9ubWVudCBmb3IgaW50ZWdyYXRlZCBtb2RlbHMgb2YgYmlvbW9sZWN1bGFyIGludGVyYWN0aW9uIG5ldHdvcmtzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0OTgtNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1hZ3JpR086IGEgR08gYW5hbHlzaXMgdG9vbGtpdCBmb3IgdGhlIGFncmljdWx0dXJhbCBjb21tdW5pdHk7IE51Y2xlaWMuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz02NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lIGNvbnRyb2xsaW5nIHZhcmlhdGlvbiBpbiBBcmFiaWRvcHNpcyBnbHVjb3Npbm9sYXRlIGNvbXBvc2l0aW9uIGlzIHBhcnQgb2YgdGhlIG1ldGhpb25pbmUgY2hhaW4gZWxvbmdhdGlvbiBwYXRod2F5OyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGxhc3RpZGljIGJpbGUgYWNpZCB0cmFuc3BvcnRlciA1IGlzIHJlcXVpcmVkIGZvciB0aGUgYmlvc3ludGhlc2lzIG9mIG1ldGhpb25pbmUtZGVyaXZlZCBnbHVjb3Npbm9sYXRlcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4MTMtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIGJyYW5jaGVkLWNoYWluIGFtaW5vdHJhbnNmZXJhc2UgMyBmdW5jdGlvbnMgaW4gYm90aCBhbWlubyBhY2lkIGFuZCBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ2OyAgY2l0YXRpb25fcGFnZXM9MTAyOC0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wbGV4IGludGVycGxheSBvZiB0aHJlZSBSMlIzIE1ZQiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgZGV0ZXJtaW5lcyB0aGUgcHJvZmlsZSBvZiBhbGlwaGF0aWMgZ2x1Y29zaW5vbGF0ZXMgaW4gQXJhYmlkb3BzaXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTM0OC02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuYWx5c2lzIGluIEJyYXNzaWNhIHdpdGggc3BlY2lhbCByZWZlcmVuY2UgdG8gdGhlIGV4cGVyaW1lbnRhbCBmb3JtYXRpb24gb2YgQi4gbmFwdXMgYW5kIHBlY3VsaWFyIG1vZGUgb2YgZmVydGlsaXphdGlvbjsgSmFwYW4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTkzNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQ1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXNzb2NpYXRpb24gdHJhbnNjcmlwdG9taWNzIGFwcHJvYWNoIHRvIHRoZSBwcmVkaWN0aW9uIG9mIGh5YnJpZCBwZXJmb3JtYW5jZTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz05MS0xMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbiBtZXRhbHMgZGVmZW5kIHBsYW50cyBhZ2FpbnN0IGJpb3RpYyBzdHJlc3M/OyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI4OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJhY3Rpb25zIGJldHdlZW4gbWV0YWwgaW9uIHRveGljaXR5IGFuZCBkZWZlbmNlcyBhZ2FpbnN0IGJpb3RpYyBzdHJlc3M6IGdsdWNvc2lub2xhdGVzIGFuZCBiZW56b3hhemlub2lkcyBhcyBjYXNlIHN0dWRpZXM7IEZvci4gU25vdyBMYW5kc2MuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIEhNQTIsIGEgZGl2YWxlbnQgaGVhdnkgbWV0YWwgdHJhbnNwb3J0aW5nIFBJQiB0eXBlIEFUUGFzZSwgaXMgaW52b2x2ZWQgaW4gY3l0b3BsYXNtaWMgWm4yKyBob21lb3N0YXNpczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9MzcxMi0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mbHVlbmNlIG9mIHppbmMgaHlwZXJhY2N1bXVsYXRpb24gb24gZ2x1Y29zaW5vbGF0ZXMgaW4gVGhsYXNwaSBjYWVydWxlc2NlbnM7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTYyMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1aaW5jIGF2YWlsYWJpbGl0eSBpbiBoeWRyb3BvbmljIGN1bHR1cmUgaW5mbHVlbmNlcyBnbHVjb3Npbm9sYXRlIGNvbmNlbnRyYXRpb25zIGluIEJyYXNzaWNhIHJhcGE7IEhvcnRzY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9ODQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzYvNjEzLzE0NjIzODUvZHN1MDI0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBCcmVlZGluZyBuZXcgdmFyaWV0aWVzIHdpdGggbG93IHNlZWQgZ2x1Y29zaW5vbGF0ZSAoR1MpIGNvbmNlbnRyYXRpb25zIGhhcyBsb25nIGJlZW4gYSBwcmltZSB0YXJnZXQgaW4gQnJhc3NpY2EgbmFwdXMuIEluIHRoaXMgc3R1ZHksIGEgbm92ZWwgYXNzb2NpYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTUxMzMyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZI9vBnet1TB+kTFKbfDDvEcgNlEO" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f613%2f551332" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f613%2f551332" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=22047" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&amp;Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/6">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="8721351">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721351">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721353">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721353">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721360">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721360">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721381">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721381">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721412">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721412">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721426">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721426">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721428">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721428">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721430">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721430">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="8721432">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#8721432">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/6/603/2754543">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/6/627/2754545">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in <em>Brassica napus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guangyuan Lu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre for Novel Agricultural Products, Department of Biology</div>, <div class="insititution">University of York</div>, <div class="addr-line">Heslington, York YO10 5DD</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Oil Crops Research Institute, CAAS</div>, <div class="addr-line">Wuhan 430062, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guangyuan+Lu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lu G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guangyuan Lu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea L. Harper</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre for Novel Agricultural Products, Department of Biology</div>, <div class="insititution">University of York</div>, <div class="addr-line">Heslington, York YO10 5DD</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrea+L.+Harper">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harper A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harper A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrea L. Harper</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Trick</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">John Innes Centre, Norwich Research Park</div>, <div class="addr-line">Norwich, Norfolk NR4 7UH</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Trick">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Trick M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Trick M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin Trick</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Colin Morgan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">John Innes Centre, Norwich Research Park</div>, <div class="addr-line">Norwich, Norfolk NR4 7UH</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Colin+Morgan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morgan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morgan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Colin Morgan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fiona Fraser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">John Innes Centre, Norwich Research Park</div>, <div class="addr-line">Norwich, Norfolk NR4 7UH</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fiona+Fraser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fraser F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fraser F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fiona Fraser</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carmel O&#39;Neill</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">John Innes Centre, Norwich Research Park</div>, <div class="addr-line">Norwich, Norfolk NR4 7UH</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carmel+O%27Neill">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=O&#39;Neill C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;O&#39;Neill C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Carmel O'Neill</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ian Bancroft</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre for Novel Agricultural Products, Department of Biology</div>, <div class="insititution">University of York</div>, <div class="addr-line">Heslington, York YO10 5DD</div>, <div class="country">UK</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +44 01904-328778. Fax. +44 01 904-328762. Email: <a href="mailto:ian.bancroft@york.ac.uk" target="_blank">ian.bancroft@york.ac.uk</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ian+Bancroft">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bancroft I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bancroft I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ian Bancroft</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 613–625, <a href='https://doi.org/10.1093/dnares/dsu024'>https://doi.org/10.1093/dnares/dsu024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 July 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 March 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 June 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="551332" href="/dnaresearch/article-pdf/21/6/613/1462385/dsu024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Guangyuan Lu, Andrea L. Harper, Martin Trick, Colin Morgan, Fiona Fraser, Carmel O'Neill, Ian Bancroft;  Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in <em>Brassica napus</em>, <em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 613–625, <a href='https://doi.org/10.1093/dnares/dsu024'>https://doi.org/10.1093/dnares/dsu024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/551332?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/551332?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Associative%20Transcriptomics%20Study%20Dissects%20the%20Genetic%20Architecture%20of%20Seed%20Glucosinolate%20Content%20in%20%26lt%3Bem%26gt%3BBrassica%20napus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2014-07-15&amp;volumeNum=21&amp;issueNum=6&amp;author=Lu%2C%20Guangyuan%3B%20Harper%2C%20Andrea%20L.&amp;startPage=613&amp;endPage=625&amp;contentId=10.1093%2Fdnares%2Fdsu024&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f613%2f551332" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=8721351 id="8721351" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Breeding new varieties with low seed glucosinolate (GS) concentrations has long been a prime target in <em>Brassica napus</em>. In this study, a novel association mapping methodology termed ‘associative transcriptomics’ (AT) was applied to a panel of 101 <em>B. napus</em> lines to define genetic regions and also candidate genes controlling total seed GS contents. Over 100,000 informative single-nucleotide polymorphisms (SNPs) and gene expression markers (GEMs) were developed for AT analysis, which led to the identification of 10 SNP and 7 GEM association peaks. Within these peaks, 26 genes were inferred to be involved in GS biosynthesis. A weighted gene co-expression network analysis provided additional 40 candidate genes. The transcript abundance in leaves of two candidate genes, <em>BnaA.GTR2a</em> located on chromosome A2 and <em>BnaC.HAG3b</em> on C9, was correlated with seed GS content, explaining 18.8 and 16.8% of phenotypic variation, respectively. Resequencing of genomic regions revealed six new SNPs in <em>BnaA.GTR2a</em> and four insertions or deletions in <em>BnaC.HAG3b</em>. These deletion polymorphisms were then successfully converted into polymerase chain reaction–based diagnostic markers that can, due to high linkage disequilibrium observed in these regions of the genome, be used for marker-assisted breeding for low seed GS lines.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;associative transcriptomics&quot;">associative transcriptomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="GEM">GEM</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="glucosinolate">glucosinolate</a></div>                            <h2 scrollto-destination=8721353 id="8721353" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Glucosinolates (GSs) are secondary metabolites mainly found in the family of Brassicaceae<sup><a reveal-id="DSU024C1" data-open="DSU024C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU024C2" data-open="DSU024C2" class="link link-ref link-reveal xref-bibr">2</a></sup> which includes rapeseed (<em>Brassica napus</em> L.), the globally important oil crop. Some breakdown products of GS have an anti-nutritional value for livestock,<sup><a reveal-id="DSU024C3" data-open="DSU024C3" class="link link-ref link-reveal xref-bibr">3</a></sup> thus making it necessary to breed for rapeseed varieties with low GS (&lt;30 mol g<sup>−1</sup>) in seeds.<sup><a reveal-id="DSU024C4" data-open="DSU024C4" class="link link-ref link-reveal xref-bibr">4</a></sup> However, modern varieties with low GS in seeds tend to be associated with a concomitant reduction of the GS content in leaves,<sup><a reveal-id="DSU024C5" data-open="DSU024C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU024C6" data-open="DSU024C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and thus they are more susceptible to insects<sup><a reveal-id="DSU024C7" data-open="DSU024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU024C8" data-open="DSU024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and diseases such as <em>Sclerotinia sclerotiorum</em>.<sup><a reveal-id="DSU024C9" data-open="DSU024C9" class="link link-ref link-reveal xref-bibr">9</a></sup> For this reason, it is desirable to reduce the GS contents within seeds so that the cake is suitable for fodder and yet maintain the disease-protective effects of high GS contents in other organs. As a prerequisite for this aim, candidate genes for GS biosynthesis and transportation in rapeseed must be identified.</p><p>The chemical structure of GS comprises a thioglucose moiety, a sulphonated oxime, and a side chain derived from aliphatic or aromatic amino acids, or tryptophan.<sup><a reveal-id="DSU024C3" data-open="DSU024C3" class="link link-ref link-reveal xref-bibr">3</a></sup> There are three basic steps for GS biosynthesis in plants, i.e. amino acid chain elongation, GS skeleton formation, and side-chain modification.<sup><a reveal-id="DSU024C10" data-open="DSU024C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU024C11" data-open="DSU024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> To date, nearly all genes responsible for biosynthetic steps have been identified,<sup><a reveal-id="DSU024C12  DSU024C24" data-open="DSU024C12  DSU024C24" class="link link-ref link-reveal xref-bibr">12–24</a></sup> leading to the clarification of the core pathway of GS biosynthesis in Brassicaceae (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).<sup><a reveal-id="DSU024C25  DSU024C27" data-open="DSU024C25  DSU024C27" class="link link-ref link-reveal xref-bibr">25–27</a></sup> The GSs are believed to be synthesized mainly in rosette and silique walls and then relocated actively to embryos through phloem by specific transporters.<sup><a reveal-id="DSU024C28  DSU024C32" data-open="DSU024C28  DSU024C32" class="link link-ref link-reveal xref-bibr">28–32</a></sup> Blocking the reallocation of GS from vegetative organs to embryos could be an effective way of reducing GS concentrations in seeds without affecting other tissues. This concept has been supported by the most recent identification of <em>GTR1</em> and <em>GTR2</em> that encode a GS transporter in <em>Arabidopsis</em>. In the <em>gtr1 gtr2</em> double-mutant plants, the GS content was found to be reduced by 100% in seeds but with a 10-fold increase in rosette.<sup><a reveal-id="DSU024C33" data-open="DSU024C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p><p>Quantitative trait locus (QTL) analysis is a powerful method to study the genetics underpinning quantitative variation in GS profiles.<sup><a reveal-id="DSU024C34" data-open="DSU024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> For total GS accumulation in the seeds, seven QTLs have been identified on several linkage groups in rapeseed.<sup><a reveal-id="DSU024C35  DSU024C38" data-open="DSU024C35  DSU024C38" class="link link-ref link-reveal xref-bibr">35–38</a></sup> More recently, Feng <em>et al.</em><sup><a reveal-id="DSU024C39" data-open="DSU024C39" class="link link-ref link-reveal xref-bibr">39</a></sup> identified 105 metabolite QTLs that had an effect on the GS concentration and constructed an advanced metabolic network for the GS composition in both leaves and seeds of rapeseed. Genome-wide association study (GWAS) is another powerful tool of identifying genes associated with complex traits, which has several advantages over bi-parental QTL mapping.<sup><a reveal-id="DSU024C40" data-open="DSU024C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSU024C41" data-open="DSU024C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The number of GWASs conducted is rapidly increasing, and it has resulted in the discovery of genes for tocopherol, carotenoid, and oil content in maize,<sup><a reveal-id="DSU024C42  DSU024C44" data-open="DSU024C42  DSU024C44" class="link link-ref link-reveal xref-bibr">42–44</a></sup> and genes underlying important traits such as flowering time and grain yield in rice.<sup><a reveal-id="DSU024C45" data-open="DSU024C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSU024C46" data-open="DSU024C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Studies regarding GWAS in rapeseed have gained attention in recent years. The overall level of linkage disequilibrium (LD) in 85 winter rapeseed genotypes was found to be very low, with a mean <em>r</em><sup>2</sup> of 0.027.<sup><a reveal-id="DSU024C47" data-open="DSU024C47" class="link link-ref link-reveal xref-bibr">47</a></sup> A structure-based association study using gene-linked simple-sequence repeat markers revealed that four genes were associated with the total GS content in seeds.<sup><a reveal-id="DSU024C48" data-open="DSU024C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Most recently, a panel of 472 rapeseed lines were further applied to a GWAS of seed weight and seed quality traits, leading to the identification of four clusters of single-nucleotide polymorphisms (SNPs) highly associated with the GS content.<sup><a reveal-id="DSU024C49" data-open="DSU024C49" class="link link-ref link-reveal xref-bibr">49</a></sup></p><p>With the rapid development of the new biotechnology, especially the emergence and application of next-generation sequencing technologies such as Illumina (Solexa) sequencing, a considerable progress in the accumulation and distribution of <em>Brassica</em> genome data has been made in recent years. These endeavours have resulted in a 95-k unigene set and 41,211 SNPs publicly available for <em>Brassica</em> community (<em>Brassica</em> genome gateway; <a class="link link-uri" href="http://brassica.nbi.ac.uk/">http://brassica.nbi.ac.uk/</a>). More recently, the massively parallel RNA sequencing (mRNA-Seq)<sup><a reveal-id="DSU024C50" data-open="DSU024C50" class="link link-ref link-reveal xref-bibr">50</a></sup> was also applied to dissect the genome of <em>B. napus</em> at transcriptome level, which led to the development of some 23,000 SNP markers and the construction of an ultradensity linkage map in <em>B. napus.</em><sup><a reveal-id="DSU024C51" data-open="DSU024C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Moreover, gene expression variation (i.e. gene expression marker, GEM) for each unigene can also be inferred from the same set of mRNA-Seq data, providing additional <em>ca.</em> 189,000 GEMs for both A and C genome. With the huge amount of markers (SNPs and GEMs) derived from mRNA-Seq, an improved GWAS method termed ‘associative transcriptomics’ (AT) was proposed to overcome the difficulties (e.g. the complexity of polyploidy and the lack of reference genome sequences to order SNPs) that hinder GWAS in <em>B. napus</em>.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The novel AT methodology has been proved in a relatively small panel, and the orthologs of <em>HAG1</em> (also known as <em>MYB28</em>), the key player in the regulation of aliphatic GS biosynthesis in <em>Arabidopsis</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 1</a>), were discovered to be vital for GS variation in <em>B. napus</em>.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> However, due to the small number of lines used, the power of AT has not been fully exploited, and many other candidate genes for GS as well as their allelic variations in the germplasm have yet to be characterized. </p>                        <a id="8721358" scrollto-destination="8721358"></a>
<div id="DSU024F1" data-id="DSU024F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02401.png?Expires=2147483647&Signature=11qLKdtwCcSuqkaSXV9M4OptKCsvDUBGI4WAkSvIyYu1z8R-jHVFPZxxJHT93MiFx3tlkObJB9vC~VMpnvqeTVafEu8x4p~RUp7Qo3aFX0jg5FiQnZwkBhgGq5YyEIgaD6UMXS3NCEq7qhxc~sZRvGOrA3ilBQbLCeyPu7StUuvBKFoZi3KucG-kkEPCe9Dd6-ZWsFoISZw0JOnHVwJz5fDBrtZrefmQ-KEFbZPj2QbxGFflviz2pKQydIPcazRxSqOB8-fi5XrwfMYnAYM8m7A8ua~I9~eU-CNpEX2JfVU1u3ZpLkeNlYITzy3-aDZ6yBWEhvpG~NDL7iKtmOXkIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of total seed glucosinolate concentrations in the diversity panel." path-from-xml="dsu02401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721358/dsu02401.tiff" path-from-xml="dsu02401.tiff" target="_blank">View large</a><a section="8721358" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02401.png?Expires=2147483647&Signature=dZGk2UUl5a370Ceeu2URhB8LE6PCM3SFxoAW9ScNImBI8hu6CNjtOiF5tkFR~sX7Xqp~Lc7nO0qIvVy5F~vmg65-QFDPoWBhsPG4-0yLaeHBZVigV81tSiPAxcOKRGKuudanr~QPt2dnWFLwEqzzU0FE6NNBNhsbYLlBofllEDq8N1DBKVVmoQivjte09CR0xjGnTlFR5xARHxCxrIr28o5X0mqpdyVNXRUaRnV~5Evo9qROL-au9DCVY8LJvNGy0U28k3nOomLEp~~3PQeoCWzn0lT9XsfE3b3jSBe60wxllKATVPyATwsyrQapR1govNnXFoDzhnUwCujvdGWo0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721358&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of total seed glucosinolate concentrations in the diversity panel.</p></div></div><div content-id="DSU024F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02401.png?Expires=2147483647&Signature=11qLKdtwCcSuqkaSXV9M4OptKCsvDUBGI4WAkSvIyYu1z8R-jHVFPZxxJHT93MiFx3tlkObJB9vC~VMpnvqeTVafEu8x4p~RUp7Qo3aFX0jg5FiQnZwkBhgGq5YyEIgaD6UMXS3NCEq7qhxc~sZRvGOrA3ilBQbLCeyPu7StUuvBKFoZi3KucG-kkEPCe9Dd6-ZWsFoISZw0JOnHVwJz5fDBrtZrefmQ-KEFbZPj2QbxGFflviz2pKQydIPcazRxSqOB8-fi5XrwfMYnAYM8m7A8ua~I9~eU-CNpEX2JfVU1u3ZpLkeNlYITzy3-aDZ6yBWEhvpG~NDL7iKtmOXkIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of total seed glucosinolate concentrations in the diversity panel." path-from-xml="dsu02401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721358/dsu02401.tiff" path-from-xml="dsu02401.tiff" target="_blank">View large</a><a section="8721358" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02401.png?Expires=2147483647&Signature=dZGk2UUl5a370Ceeu2URhB8LE6PCM3SFxoAW9ScNImBI8hu6CNjtOiF5tkFR~sX7Xqp~Lc7nO0qIvVy5F~vmg65-QFDPoWBhsPG4-0yLaeHBZVigV81tSiPAxcOKRGKuudanr~QPt2dnWFLwEqzzU0FE6NNBNhsbYLlBofllEDq8N1DBKVVmoQivjte09CR0xjGnTlFR5xARHxCxrIr28o5X0mqpdyVNXRUaRnV~5Evo9qROL-au9DCVY8LJvNGy0U28k3nOomLEp~~3PQeoCWzn0lT9XsfE3b3jSBe60wxllKATVPyATwsyrQapR1govNnXFoDzhnUwCujvdGWo0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721358&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of total seed glucosinolate concentrations in the diversity panel.</p></div></div><p>The aim of this study is to address the genetic control of GS natural variation in <em>B. napus</em> using AT. This has been achieved by genotyping a panel of 101 lines by mRNA-Seq and phenotyping the total GS content in seeds. Four consensus association peaks and also many candidate genes involved in the GS pathway were identified. The identification of candidate genes not only furthers our understanding of the gene network for GS biosynthesis but also provides markers for the breeding of low GS rapeseed varieties.</p>                            <h2 scrollto-destination=8721360 id="8721360" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=8721361 id="8721361" class="section-title" data-legacy-id=s2a>2.1. Plant materials and GS measurements</h3>
<p>A diversity panel comprising 101 <em>B. napus</em> lines was used for association mapping study. Within this panel, there are 54 winter, 17 spring, and 5 semi-winter rapeseed type lines, which are mainly collected from Europe, Canada, and China, respectively. To maximize allelic variations for GS-related genes, <em>B. napus</em> lines of five kale types, nine swede types, nine fodder types, one synthetic type, and one vegetable type were also included (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>Seeds of all lines were sown into 9-cm pots containing Scotts Levington F1 compost (Scotts), germinated, and grown in long-day (16-h photoperiod) glasshouse conditions at 15°C for 21 days, as described previously.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The experiment was arranged into a four-block, one-way randomized design with one plant of each of the lines per block and randomized within each block. The first true leaf of each plant (21 days after sowing) was excised and pooled according to line and frozen in liquid nitrogen, giving a final harvest of four pooled leaf samples per line. These samples will be used for DNA and RNA extractions.</p><p>To phenotype the GS content, seeds of all lines were sown in January 2012 as described by Smooker <em>et al.</em><sup><a reveal-id="DSU024C53" data-open="DSU024C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Then, the vernalized seedlings were transplanted into the field at John Innes Centre, Norwich, UK (1.297°E, 52.628°N) by the end of April 2012 in four randomized blocks at an average density of three plants per square meter. Before the flowers opened, racemes were covered by bread bags to obtain selfed seeds for GS measurement. Mature seeds were harvested from each plot and measured for the total GS content using near infra-red spectroscopy (NIRS) at KWS-UK (Foss NIRS Systems 5000). The GS content for each line was presented as a mean value of four replicates.</p>                            <h3 scrollto-destination=8721365 id="8721365" class="section-title" data-legacy-id=s2b>2.2. Transcriptome sequencing and SNP calling</h3>
<p>RNA was prepared by grinding juvenile leaves in liquid nitrogen and extracting the RNA using the E.Z.N.A. Plant RNA Kit (Omega Bio-Tek) according to the manufacturer’'s protocol. After RNA samples had been isolated and dried, they were dissolved in diethylpyrocarbonate-treated H<sub>2</sub>O, and a NanoDrop spectrophotometer (model ND-1000) was used to determine the RNA concentration. RNA quality was assessed by running 1 μl of each RNA sample on an Agilent RNA 6000 Nano LabChip (Agilent Technology 2100 Bioanalyzer).</p><p>Illumina sequencing, quality checking, and processing were conducted as described previously.<sup><a reveal-id="DSU024C51" data-open="DSU024C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Briefly, the sequencing libraries for all lines were prepared separately using the Illumina mRNA-Seq kit (RS-100–0801, Illumina Inc.), and run on a single lane for 80 cycles on the Illumina Genome Analyzer GAIIx. Illumina base calling files were processed using GERALD to produce a sequence file containing 80 base reads for each sample.</p><p>SNPs were called by the meta-analysis of alignments of Illumina reads obtained from each of 101 <em>B. napus</em> against a <em>Brassica</em> A and C genome-cured unigene reference sequence, as described previously.<sup><a reveal-id="DSU024C54" data-open="DSU024C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSU024C55" data-open="DSU024C55" class="link link-ref link-reveal xref-bibr">55</a></sup> SNP positions were excluded from further analysis if more than two alleles were detected across the accessions, and a noise threshold of 0.15 was employed to reduce false SNP calls due to sequencing errors.</p><p>Using methods and scripts also described before,<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSU024C54" data-open="DSU024C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSU024C55" data-open="DSU024C55" class="link link-ref link-reveal xref-bibr">55</a></sup> SNPs identified within the superset assembled over all the 101 <em>B. napus</em> lines were then assigned to the A or C genomes through the computational detection of <em>cis</em>-linkage within sequenced Illumina reads to independently identify inter-homologue polymorphisms (IHPs). Only simple cases with two classes of linkage (each of the bases constituting the IHP linked exclusively to only one of the bases constituting the SNP) were accepted. SNPs were traversed to make genome assignments based on the IHP-mediated and other lines of evidence.</p>                            <h3 scrollto-destination=8721370 id="8721370" class="section-title" data-legacy-id=s2c>2.3. GEM calling</h3>
<p>Using methods and scripts described before,<sup><a reveal-id="DSU024C54" data-open="DSU024C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSU024C55" data-open="DSU024C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Illumina reads from the 101 <em>B. napus</em> lines were realigned against the ‘cured’ reference comprising the A and C genome versions of 94,558 unigenes (189,116 in total) using MAQ version 0.7.1 (<a class="link link-uri" href="http://maq.sourceforge.net/index.shtml">http://maq.sourceforge.net/index.shtml</a>) and assigned to the A and C genome. When a read maps equally well to multiple IHP positions, MAQ will randomly pick one position, thereby distributing reads evenly between the A and C genome versions of the unigene where the sequence is identical. MAQ pile-up text files were generated from the MAQ binary map files. The Perl script tagcounter.pl<sup><a reveal-id="DSU024C54" data-open="DSU024C54" class="link link-ref link-reveal xref-bibr">54</a></sup> was used to count the number of reads aligning to the A and C genome version of each unigene by accessing the pile-up files, outputting a count and calculated reads per kb (of unigene) per million aligned reads (RPKM) value for each unigene.</p>                            <h3 scrollto-destination=8721372 id="8721372" class="section-title" data-legacy-id=s2d>2.4. SNP association analysis</h3>
<p>The SNP data set for the 101 lines was entered into the program STRUCTURE 2.3.4.<sup><a reveal-id="DSU024C56" data-open="DSU024C56" class="link link-ref link-reveal xref-bibr">56</a></sup> An admixture model with independent allele frequencies was used, and the <em>K</em> value best representing the data set was determined according to the method of Evanno <em>et al.</em><sup><a reveal-id="DSU024C57" data-open="DSU024C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Once the optimal number of <em>K</em> populations was established, a Q matrix score for each individual line could be used as a fixed effect in the subsequent association analysis. The GS trait data, Q matrix, and SNP data for all lines were entered into the program TASSEL 3.0.<sup><a reveal-id="DSU024C58" data-open="DSU024C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Minor allele states below 0.05 were removed from the SNP data set, and a kinship (<em>K</em>) matrix was calculated to estimate the pairwise relatedness between individuals. These data sets were entered into a mixed linear model (MLM) with optimum compression and P3D variance component estimation to decrease the computing time for the large data set. The significant value and also the marker effect for each SNP were exported, and a Manhattan plot was generated in R package (<a class="link link-uri" href="http://cran.r-project.org">http://cran.r-project.org</a>).</p>                            <h3 scrollto-destination=8721374 id="8721374" class="section-title" data-legacy-id=s2e>2.5. GEM association analysis</h3>
<p>The relationship between gene expression and GS content of seeds was determined by linear regression using R package.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> For each unigene, RPKM values were regressed as the dependent variable and the GS content as the independent variable, and <em>R</em><sup>2</sup> and significance values (<em>P</em>) were calculated for each unigene. The <em>P</em>-value for each unigene was converted into −log<sub>10</sub><em>P</em> and plotted against its physical position in the ‘pseudomolecules’ to generate a Manhattan plot.</p>                            <h3 scrollto-destination=8721376 id="8721376" class="section-title" data-legacy-id=s2f>2.6. Co-expression analysis</h3>
<p>Weighted gene co-expression network analysis (WGCNA)<sup><a reveal-id="DSU024C59" data-open="DSU024C59" class="link link-ref link-reveal xref-bibr">59</a></sup> was performed as described previously.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Briefly, 101 <em>B. napus</em> lines were clustered according to the RPKM values of unigenes. Then, a soft-thresholding power (<em>β</em> = 5) to approximate scale-free topology within the network was determined by plotting the scale-free topology fitting index against soft threshold (power). Genes were then clustered using dissimilarity based on the topological overlap calculated between all genes. A value of 0.2 was selected to cut the branches of the dendrogram, resulting in a network containing 122 modules, each represented by a colour. Each module was summarized by the first principal component of the scaled (standardized) module expression profiles. Thus, the module eigengene explains the maximum amount of variation of the module expression levels. Network construction was performed using the ‘blockwiseModules’ function in the software package, which allows the network construction for the entire data set. The summary profile (eigengene) for each module was then correlated with external traits. This analysis identifies several significant module trait associations; the most interesting is the relationship between the ‘lightblue4’ module and total GS content in seeds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The programme Cytoscape<sup><a reveal-id="DSU024C60" data-open="DSU024C60" class="link link-ref link-reveal xref-bibr">60</a></sup> was used to draw the network with significant connected genes.</p><p>To analyze gene ontology (GO), all unigenes from ‘lightblue4’ module were submitted to the online toolkit, agriGO (<a class="link link-uri" href="http://bioinfo.cau.edu.cn/agriGO/">http://bioinfo.cau.edu.cn/agriGO/</a>), to generate a GO network using the Singular Enrichment Analysis tool.<sup><a reveal-id="DSU024C61" data-open="DSU024C61" class="link link-ref link-reveal xref-bibr">61</a></sup></p>                            <h3 scrollto-destination=8721379 id="8721379" class="section-title" data-legacy-id=s2g>2.7. Polymerase chain reaction amplification and resequencing</h3>
<p>The nucleotide sequences of two unigenes, JCVI_13343 (<em>BnaA</em>.<em>GTR2a</em>) and EX043693 (<em>BnaC.HAG3b</em>), were retrieved from database (<a class="link link-uri" href="http://brassica.nbi.ac.uk/">http://brassica.nbi.ac.uk/</a>) and used as templates to design specific polymerase chain reaction (PCR) primers. <em>BnaA</em>.<em>GTR2a</em> was amplified from genomic DNA using oligonucleotide primers GTR2F (GGGATTTTCTTCGCCTTT) and GTR2R (GTCCAAAGAGTTGTAAATGGT), and <em>BnaC.HAG3b</em> was amplified with HAG3F (TGGAGTGTACGAGAAAAAC) and HAG3R (TTCATACATCAAATACCAAAC). Following PCR amplification, the products were resolved on a ABI 3730XL capillary sequencer by GATC Biotech Ltd. (London, UK) as a commercial service and analysed using Sequence Scanner V1 (<a class="link link-uri" href="http://www.appliedbiosystems.com">http://www.appliedbiosystems.com</a>) to determine the presence or absence of each DNA polymorphism.</p>                            <h2 scrollto-destination=8721381 id="8721381" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=8721382 id="8721382" class="section-title" data-legacy-id=s3a>3.1. Phenotypic variation of total GS concentration in seeds</h3>
<p>Seeds of 101 <em>B. napus</em> lines were harvested and measured for total GS concentration using NIRS (Fig. <a reveal-id="DSU024F1" data-open="DSU024F1" class="link link-reveal link-table xref-fig">1</a>). The GS content for each line ranged from 8.0 to 195.6 μmol g<sup>−1</sup>, with an average of 65.3 μmol g<sup>−1</sup>. The coefficient of variation was estimated as high as 73.2%, a sign of wide variation for GS concentrations in this panel. Thus, it is suitable for association mapping. Moreover, 41% of lines were classified as low GS (&lt;30 μmol g<sup>−1</sup>), one of the two key characteristics for the canola quality in rapeseed. The GS concentrations for Tapidor and Ningyou7 (two parents of the widely used mapping population TNDH) were 19.0 and 85.7 μmol g<sup>−1</sup>, respectively, which were very close to previous results (19.9 and 78.6 μmol g<sup>−1</sup>).<sup><a reveal-id="DSU024C39" data-open="DSU024C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=8721384 id="8721384" class="section-title" data-legacy-id=s3b>3.2. Genotyping of <em>B. napus</em> lines</h3>
<p>RNA was isolated and sequenced from each of 101 <em>B. napus</em> lines, providing a total of &gt;200 Gb of 80-bp sequence reads (under accession numbers: ERA122949, ERA036824, and ERA063602). By mapping sequence reads to a reference sequence comprising the <em>Brassica</em> unigene set, a total of 225,011 SNP markers were called (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Among these markers, 80,880 with minor allele frequencies (MAF) &lt;0.05 were removed, and the remaining 144,131 SNP markers were further mapped onto specific genomes. As expected, only a few of these markers could be mapped onto either A (7580) or C (7673) genome, which are very useful for anchoring genetic loci to a specific genome location; the majority (89.4%) was mapped onto both genomes, because A and C genome sequences are highly similar.<sup><a reveal-id="DSU024C51" data-open="DSU024C51" class="link link-ref link-reveal xref-bibr">51</a></sup></p><p>The GEM, expressed as RPKM value of a unigene, can also be inferred from the mRNA-Seq data. This exercise provided a total of 189,116 GEMs, with 94,558 on A and C genomes. Among these GEMs, 49,599 and 50,935 were shown to be informative (i.e. RPKM &gt; 0 for at least some lines) on respective A and C genomes; thus they were also used for marker-trait association study.</p>                            <h3 scrollto-destination=8721387 id="8721387" class="section-title" data-legacy-id=s3c>3.3. Genetic structure and LD</h3>
<p>The 144,131 informative SNPs were first used for genetic structure analysis. Again, all <em>B. napus</em> lines can be largely divided into two clusters by the Bayesian clustering algorithms implemented in STRUCTURE, one of which consists of most winter-type lines and the other of spring-type, swede, and kale lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3 and Table S1</a>).</p><p>To facilitate comparison, the 15,253 mapped SNP markers that have been previously used in a smaller panel<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> were again used for LD analysis in 101 lines. The extent of LD was gauged by calculating pairwise <em>r</em><sup>2</sup> for the mapped SNPs using an LD window of 500 (providing &gt;30,000 pairwise values of <em>r</em><sup>2</sup>). The mean LD across the whole genome was 0.0209, which is close to the previous estimates of 0.0246 in a subset of these lines<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and confirms the low overall level of LD in <em>B. napus</em>.</p><p>The strength of LD was also measured across each chromosome pseudomolecule. As an example, there were several small but strong LD blocks (<em>r</em><sup>2</sup> &gt; 0.2) in A2 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4a</a>). Meanwhile, two large LD blocks sit on both ends of C9 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4b</a>).</p>                            <h3 scrollto-destination=8721391 id="8721391" class="section-title" data-legacy-id=s3d>3.4. Loci associated with the GS concentration</h3>
<p>SNPs and GEMs were separately used for AT study on total GS concentration in seeds. Firstly, 144,131 informative SNPs were regressed with the GS trait using a MLM implemented in TASSEL, leading to the identification of 10 association peaks at a Bonferroni threshold of <em>P</em> &lt; 6.9 × 10<sup>−6</sup> (i.e. <em>P</em> = 1/144,131; −log<sub>10</sub><em>P</em> = 5.2) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5a</a>). These peaks were located on A2, A3, A6, A9, C2, C3, C4, C7, and C9 (Table <a reveal-id="DSU024TB1" data-open="DSU024TB1" class="link link-reveal link-table xref-fig">1</a>). It is not surprising that some peaks on the A genome are very similar to those on the C genome, owing to the fact that most SNPs were mapped onto both A and C genomes. The well-defined peaks were found on A2, A3, A6, A9, C2, C3, and C9, within which there were foci to identify candidate genes for GS. However, <em>P-</em>values erroneously fail to be significant for markers in multiple comparison tests when analysing a large number of SNPs.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Therefore, we used an <em>ad hoc</em> threshold of 10<sup>−4</sup> to assess genomic regions underlying association peaks for the presence of candidate genes. In all, 255 SNPs were found to be highly associated with GS, which were derived from 110 unigenes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Of the 110 unigenes, only 2 were directly implicated in the GS metabolism pathway (Table <a reveal-id="DSU024TB2" data-open="DSU024TB2" class="link link-reveal link-table xref-fig">2</a>), and the proteins encoded by the remaining 108 genes were classified as transcription factors, factors responding to stimulus or involved in cellular process, catalytic activity, or with unknown functions (data not shown). </p>                        <a id="8721393" scrollto-destination="8721393"></a>
<div content-id="DSU024TB1" class="table-modal"><div class="table-wrap"><div id="DSU024TB1" class="table-wrap-title" data-id="DSU024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of association peaks for the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="2">Peak interval (Mb)<sup>a</sup><hr /> </th><th colspan="2"><em>P</em>-value<sup>b</sup><hr /> </th></tr><tr><th>SNP </th><th>GEM </th><th>SNP </th><th>GEM </th></tr></thead><tbody><tr><td>A2 </td><td>24.7–25.0 </td><td>24.4–25.8 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.5 × 10<sup>−9</sup> </td></tr><tr><td>A3 </td><td>21.3–21.4 </td><td> </td><td>4.4 × 10<sup>−6</sup> </td><td> </td></tr><tr><td>A4 </td><td> </td><td>5.1–5.4 </td><td> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td rowspan="2">A6 </td><td>15.1–15.2 </td><td> </td><td>3.5 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>19.9–20.3 </td><td> </td><td>1.8 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>A9 </td><td>1.6–3.7 </td><td>1.6–3.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td>C2 </td><td>48.5–50.0 </td><td>49.1–50.5 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.4 × 10<sup>−11</sup> </td></tr><tr><td>C3 </td><td>41.4–41.8 </td><td> </td><td>3.4 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>C4 </td><td>47.7–47.8 </td><td>22.0–22.1 </td><td>1.3 × 10<sup>−6</sup> </td><td>4.9 × 10<sup>−11</sup> </td></tr><tr><td>C7 </td><td>39.8–40.7 </td><td>39.7–40.7 </td><td>4.3 × 10<sup>−6</sup> </td><td>3.7 × 10<sup>−7</sup> </td></tr><tr><td>C9 </td><td>2.0–5.2 </td><td>0.8–5.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>5.4 × 10<sup>−9</sup> </td></tr></tbody></table></div><div content-id="DSU024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="2">Peak interval (Mb)<sup>a</sup><hr /> </th><th colspan="2"><em>P</em>-value<sup>b</sup><hr /> </th></tr><tr><th>SNP </th><th>GEM </th><th>SNP </th><th>GEM </th></tr></thead><tbody><tr><td>A2 </td><td>24.7–25.0 </td><td>24.4–25.8 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.5 × 10<sup>−9</sup> </td></tr><tr><td>A3 </td><td>21.3–21.4 </td><td> </td><td>4.4 × 10<sup>−6</sup> </td><td> </td></tr><tr><td>A4 </td><td> </td><td>5.1–5.4 </td><td> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td rowspan="2">A6 </td><td>15.1–15.2 </td><td> </td><td>3.5 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>19.9–20.3 </td><td> </td><td>1.8 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>A9 </td><td>1.6–3.7 </td><td>1.6–3.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td>C2 </td><td>48.5–50.0 </td><td>49.1–50.5 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.4 × 10<sup>−11</sup> </td></tr><tr><td>C3 </td><td>41.4–41.8 </td><td> </td><td>3.4 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>C4 </td><td>47.7–47.8 </td><td>22.0–22.1 </td><td>1.3 × 10<sup>−6</sup> </td><td>4.9 × 10<sup>−11</sup> </td></tr><tr><td>C7 </td><td>39.8–40.7 </td><td>39.7–40.7 </td><td>4.3 × 10<sup>−6</sup> </td><td>3.7 × 10<sup>−7</sup> </td></tr><tr><td>C9 </td><td>2.0–5.2 </td><td>0.8–5.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>5.4 × 10<sup>−9</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The physical position is inferred from the chromosome pseudomolecules in <em>Brassica napus</em> (Harper <em>et al.</em> 2012).</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>The <em>P</em>-value is calculated for the lead (most significant) marker within each peak only.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721393" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU024TB1" class="table-wrap-title" data-id="DSU024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of association peaks for the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="2">Peak interval (Mb)<sup>a</sup><hr /> </th><th colspan="2"><em>P</em>-value<sup>b</sup><hr /> </th></tr><tr><th>SNP </th><th>GEM </th><th>SNP </th><th>GEM </th></tr></thead><tbody><tr><td>A2 </td><td>24.7–25.0 </td><td>24.4–25.8 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.5 × 10<sup>−9</sup> </td></tr><tr><td>A3 </td><td>21.3–21.4 </td><td> </td><td>4.4 × 10<sup>−6</sup> </td><td> </td></tr><tr><td>A4 </td><td> </td><td>5.1–5.4 </td><td> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td rowspan="2">A6 </td><td>15.1–15.2 </td><td> </td><td>3.5 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>19.9–20.3 </td><td> </td><td>1.8 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>A9 </td><td>1.6–3.7 </td><td>1.6–3.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td>C2 </td><td>48.5–50.0 </td><td>49.1–50.5 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.4 × 10<sup>−11</sup> </td></tr><tr><td>C3 </td><td>41.4–41.8 </td><td> </td><td>3.4 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>C4 </td><td>47.7–47.8 </td><td>22.0–22.1 </td><td>1.3 × 10<sup>−6</sup> </td><td>4.9 × 10<sup>−11</sup> </td></tr><tr><td>C7 </td><td>39.8–40.7 </td><td>39.7–40.7 </td><td>4.3 × 10<sup>−6</sup> </td><td>3.7 × 10<sup>−7</sup> </td></tr><tr><td>C9 </td><td>2.0–5.2 </td><td>0.8–5.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>5.4 × 10<sup>−9</sup> </td></tr></tbody></table></div><div content-id="DSU024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="2">Peak interval (Mb)<sup>a</sup><hr /> </th><th colspan="2"><em>P</em>-value<sup>b</sup><hr /> </th></tr><tr><th>SNP </th><th>GEM </th><th>SNP </th><th>GEM </th></tr></thead><tbody><tr><td>A2 </td><td>24.7–25.0 </td><td>24.4–25.8 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.5 × 10<sup>−9</sup> </td></tr><tr><td>A3 </td><td>21.3–21.4 </td><td> </td><td>4.4 × 10<sup>−6</sup> </td><td> </td></tr><tr><td>A4 </td><td> </td><td>5.1–5.4 </td><td> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td rowspan="2">A6 </td><td>15.1–15.2 </td><td> </td><td>3.5 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>19.9–20.3 </td><td> </td><td>1.8 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>A9 </td><td>1.6–3.7 </td><td>1.6–3.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>4.0 × 10<sup>−9</sup> </td></tr><tr><td>C2 </td><td>48.5–50.0 </td><td>49.1–50.5 </td><td>2.5 × 10<sup>−7</sup> </td><td>5.4 × 10<sup>−11</sup> </td></tr><tr><td>C3 </td><td>41.4–41.8 </td><td> </td><td>3.4 × 10<sup>−7</sup> </td><td> </td></tr><tr><td>C4 </td><td>47.7–47.8 </td><td>22.0–22.1 </td><td>1.3 × 10<sup>−6</sup> </td><td>4.9 × 10<sup>−11</sup> </td></tr><tr><td>C7 </td><td>39.8–40.7 </td><td>39.7–40.7 </td><td>4.3 × 10<sup>−6</sup> </td><td>3.7 × 10<sup>−7</sup> </td></tr><tr><td>C9 </td><td>2.0–5.2 </td><td>0.8–5.8 </td><td>1.2 × 10<sup>−9</sup> </td><td>5.4 × 10<sup>−9</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The physical position is inferred from the chromosome pseudomolecules in <em>Brassica napus</em> (Harper <em>et al.</em> 2012).</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>The <em>P</em>-value is calculated for the lead (most significant) marker within each peak only.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721393" target="_blank">View Large</a></div></div>                        <a id="8721394" scrollto-destination="8721394"></a>
<div content-id="DSU024TB2" class="table-modal"><div class="table-wrap"><div id="DSU024TB2" class="table-wrap-title" data-id="DSU024TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SNPs and candidate genes significantly associated with the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome<sup>a</sup> </th><th>Position (bp)<sup>b</sup> </th><th>SNP<sup>c</sup> </th><th>Allele<sup>d</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>GSH2</em> </td><td>A3/C3 </td><td>4,909,494/6,340,070 </td><td>JCVI_3734:475 </td><td>A,<span class="underline">R</span> </td><td>3.60 × 10<sup>−7</sup> </td><td>Glutathione synthase </td></tr><tr><td><em>HAG1</em> </td><td>A2/C2 </td><td>24,774,040/49,619,780 </td><td>EX092364:579 </td><td><span class="underline">G</span>,R </td><td>1.90 × 10<sup>−6</sup> </td><td>Transcription factor for high aliphatic glucosinolate </td></tr></tbody></table></div><div content-id="DSU024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome<sup>a</sup> </th><th>Position (bp)<sup>b</sup> </th><th>SNP<sup>c</sup> </th><th>Allele<sup>d</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>GSH2</em> </td><td>A3/C3 </td><td>4,909,494/6,340,070 </td><td>JCVI_3734:475 </td><td>A,<span class="underline">R</span> </td><td>3.60 × 10<sup>−7</sup> </td><td>Glutathione synthase </td></tr><tr><td><em>HAG1</em> </td><td>A2/C2 </td><td>24,774,040/49,619,780 </td><td>EX092364:579 </td><td><span class="underline">G</span>,R </td><td>1.90 × 10<sup>−6</sup> </td><td>Transcription factor for high aliphatic glucosinolate </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p><sup>a</sup>The candidate gene has homologues in both A and C genome.</p></div></div><div content-id="TN4"><div class="fn"><p><sup>b</sup>The physical position is based on the respective chromosome pseudomolecules in <em>Brassica napus</em>.</p></div></div><div content-id="TN5"><div class="fn"><p><sup>c</sup>For each gene, only the lead SNP (most significant) is listed if there is more than one.</p></div></div><div content-id="TN6"><div class="fn"><p><sup>d</sup>The favourable allele (leads to lower glucosinolate content) is underlined. International Union of Biochemistry ambiguity codes: R = A or G.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721394" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU024TB2" class="table-wrap-title" data-id="DSU024TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SNPs and candidate genes significantly associated with the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome<sup>a</sup> </th><th>Position (bp)<sup>b</sup> </th><th>SNP<sup>c</sup> </th><th>Allele<sup>d</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>GSH2</em> </td><td>A3/C3 </td><td>4,909,494/6,340,070 </td><td>JCVI_3734:475 </td><td>A,<span class="underline">R</span> </td><td>3.60 × 10<sup>−7</sup> </td><td>Glutathione synthase </td></tr><tr><td><em>HAG1</em> </td><td>A2/C2 </td><td>24,774,040/49,619,780 </td><td>EX092364:579 </td><td><span class="underline">G</span>,R </td><td>1.90 × 10<sup>−6</sup> </td><td>Transcription factor for high aliphatic glucosinolate </td></tr></tbody></table></div><div content-id="DSU024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome<sup>a</sup> </th><th>Position (bp)<sup>b</sup> </th><th>SNP<sup>c</sup> </th><th>Allele<sup>d</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>GSH2</em> </td><td>A3/C3 </td><td>4,909,494/6,340,070 </td><td>JCVI_3734:475 </td><td>A,<span class="underline">R</span> </td><td>3.60 × 10<sup>−7</sup> </td><td>Glutathione synthase </td></tr><tr><td><em>HAG1</em> </td><td>A2/C2 </td><td>24,774,040/49,619,780 </td><td>EX092364:579 </td><td><span class="underline">G</span>,R </td><td>1.90 × 10<sup>−6</sup> </td><td>Transcription factor for high aliphatic glucosinolate </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p><sup>a</sup>The candidate gene has homologues in both A and C genome.</p></div></div><div content-id="TN4"><div class="fn"><p><sup>b</sup>The physical position is based on the respective chromosome pseudomolecules in <em>Brassica napus</em>.</p></div></div><div content-id="TN5"><div class="fn"><p><sup>c</sup>For each gene, only the lead SNP (most significant) is listed if there is more than one.</p></div></div><div content-id="TN6"><div class="fn"><p><sup>d</sup>The favourable allele (leads to lower glucosinolate content) is underlined. International Union of Biochemistry ambiguity codes: R = A or G.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721394" target="_blank">View Large</a></div></div><p>Secondly, 100,534 GEMs were then regressed (simple linear regression) with GS concentrations, which resulted in the identification of seven association peaks located on A2, A4, A9, C2, C4, C7, and C9 at <em>P</em> &lt; 9.9 × 10<sup>−6</sup> (i.e. <em>P</em> = 1/100,534; −log<sub>10</sub><em>P</em> = 5.0) (Table <a reveal-id="DSU024TB1" data-open="DSU024TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5b</a>). A total of 352 GEMs (262 unigenes) were screened within these peaks at <em>P</em> &lt; 10<sup>−4</sup>. Of these unigenes, 22 have also been identified by SNPs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Moreover, 24 unigenes detected by GEMs were shown to be directly involved in GS metabolism (Table <a reveal-id="DSU024TB3" data-open="DSU024TB3" class="link link-reveal link-table xref-fig">3</a>). This number is 12 times of those captured by SNPs, demonstrating that the GEM analysis is more powerful than the SNP analysis in terms of capacity of screening candidate genes for GS. </p>                        <a id="8721396" scrollto-destination="8721396"></a>
<div content-id="DSU024TB3" class="table-modal"><div class="table-wrap"><div id="DSU024TB3" class="table-wrap-title" data-id="DSU024TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of GEMs and candidate genes significantly associated with the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome </th><th>Position (bp)<sup>a</sup> </th><th>GEM<sup>b</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>HAG1</em> </td><td>A9 </td><td>3,425,844 </td><td>A_JCVI_40613 </td><td>8.28 × 10<sup>−10</sup> </td><td>MYB transcription factor family </td></tr><tr><td><em>CYP83A1</em> </td><td>C4 </td><td>32,842,478 </td><td>C_JCVI_26799 </td><td>3.97 × 10<sup>−9</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>MAM1</em> </td><td>A2 </td><td>24,487,376 </td><td>A_JCVI_30455 </td><td>1.91 × 10<sup>−8</sup> </td><td>Methylthiolkylmalate synthase </td></tr><tr><td><em>AOP2</em> </td><td>A9 </td><td>1,155,687 </td><td>A_JCVI_33047 </td><td>2.79 × 10<sup>−8</sup> </td><td>2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>AT2G31790.1</em> </td><td>A5 </td><td>6,660,619 </td><td>A_JCVI_6771 </td><td>3.63 × 10<sup>−8</sup> </td><td>UDP-Glycosyltransferase superfamily protein </td></tr><tr><td><em>ATGSTF11</em> </td><td>A5 </td><td>23,602,455 </td><td>A_JCVI_6891 </td><td>3.36 × 10<sup>−7</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>CYP79F1</em> </td><td>C5 </td><td>7,734,381 </td><td>C_JCVI_17335 </td><td>6.68 × 10<sup>−7</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>SAM-2</em> </td><td>C9 </td><td>805,498 </td><td>C_JCVI_12068 </td><td>8.02 × 10<sup>−7</sup> </td><td>S-adenosylmethionine synthetase 2 (SAM-2) </td></tr><tr><td><em>SUR1</em> </td><td>C7 </td><td>200,177 </td><td>C_JCVI_531 </td><td>1.11 × 10<sup>−6</sup> </td><td>C-S lyase involved in converting S-alkylthiohydroximate to thiohydroximate </td></tr><tr><td><em>BCAT4</em> </td><td>A5 </td><td>16,822,435 </td><td>A_JCVI_34763 </td><td>1.18 × 10<sup>−6</sup> </td><td>Methionine-oxo-acid transaminase </td></tr><tr><td><em>AK3</em> </td><td>A2 </td><td>6,569,882 </td><td>A_EX056141 </td><td>1.25 × 10<sup>−6</sup> </td><td>Encodes a monofunctional aspartate kinase </td></tr><tr><td><em>BAT5</em> </td><td>C9 </td><td>28,810,650 </td><td>C_JCVI_16890 </td><td>2.92 × 10<sup>−6</sup> </td><td>Transporter of 2-keto acids between chloroplasts and the cytosol </td></tr><tr><td><em>SOT18</em> </td><td>A7 </td><td>23,340,811 </td><td>A_JCVI_17243 </td><td>3.56 × 10<sup>−6</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>GSTU23</em> </td><td>A7 </td><td>17,429,665 </td><td>A_EE467545 </td><td>1.47 × 10<sup>−5</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>UGT74B1</em> </td><td>A9 </td><td>27,647,388 </td><td>A_JCVI_31290 </td><td>1.68 × 10<sup>−5</sup> </td><td>UDP-glucose: thiohydroximate S-glucosyltransferase </td></tr><tr><td><em>MTO1</em> </td><td>A5 </td><td>23,899,195 </td><td>A_EV191151 </td><td>1.95 × 10<sup>−5</sup> </td><td>Cystathionine gamma-synthase </td></tr><tr><td><em>GTR2</em> </td><td>A2 </td><td>25,155,291 </td><td>A_JCVI_13343 </td><td>2.69 × 10<sup>−5</sup> </td><td>High-affinity, proton-dependent glucosinolate-specific transporter </td></tr><tr><td><em>AOP1</em> </td><td>A3 </td><td>14,140,138 </td><td>A_JCVI_31233 </td><td>4.41 × 10<sup>−5</sup> </td><td>Encodes a possible 2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>APS1</em> </td><td>A3 </td><td>19,165,411 </td><td>A_EV196558 </td><td>4.72 × 10<sup>−5</sup> </td><td>ATP sulfurylase </td></tr><tr><td><em>SOT17</em> </td><td>A6 </td><td>7,244,176 </td><td>A_JCVI_37729 </td><td>5.31 × 10<sup>−5</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>CBL</em> </td><td>C4 </td><td>29,183,950 </td><td>C_JCVI_35734 </td><td>5.38 × 10<sup>−5</sup> </td><td>Second enzyme in the methionine biosynthetic pathway </td></tr><tr><td><em>HAG3</em> </td><td>C9 </td><td>3,426,787 </td><td>C_EX043693 </td><td>6.86 × 10<sup>−5</sup> </td><td>MYB domain containing protein 29 </td></tr><tr><td><em>XT2</em> </td><td>A9 </td><td>987,590 </td><td>C_EE527736 </td><td>6.87 × 10<sup>−5</sup> </td><td>Protein with xylosyltransferase activity </td></tr><tr><td><em>FMO GS-OX2</em> </td><td>A9 </td><td>8,429,062 </td><td>A_JCVI_5227 </td><td>9.99 × 10<sup>−5</sup> </td><td>Glucosinolate S-oxygenase </td></tr></tbody></table></div><div content-id="DSU024TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome </th><th>Position (bp)<sup>a</sup> </th><th>GEM<sup>b</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>HAG1</em> </td><td>A9 </td><td>3,425,844 </td><td>A_JCVI_40613 </td><td>8.28 × 10<sup>−10</sup> </td><td>MYB transcription factor family </td></tr><tr><td><em>CYP83A1</em> </td><td>C4 </td><td>32,842,478 </td><td>C_JCVI_26799 </td><td>3.97 × 10<sup>−9</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>MAM1</em> </td><td>A2 </td><td>24,487,376 </td><td>A_JCVI_30455 </td><td>1.91 × 10<sup>−8</sup> </td><td>Methylthiolkylmalate synthase </td></tr><tr><td><em>AOP2</em> </td><td>A9 </td><td>1,155,687 </td><td>A_JCVI_33047 </td><td>2.79 × 10<sup>−8</sup> </td><td>2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>AT2G31790.1</em> </td><td>A5 </td><td>6,660,619 </td><td>A_JCVI_6771 </td><td>3.63 × 10<sup>−8</sup> </td><td>UDP-Glycosyltransferase superfamily protein </td></tr><tr><td><em>ATGSTF11</em> </td><td>A5 </td><td>23,602,455 </td><td>A_JCVI_6891 </td><td>3.36 × 10<sup>−7</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>CYP79F1</em> </td><td>C5 </td><td>7,734,381 </td><td>C_JCVI_17335 </td><td>6.68 × 10<sup>−7</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>SAM-2</em> </td><td>C9 </td><td>805,498 </td><td>C_JCVI_12068 </td><td>8.02 × 10<sup>−7</sup> </td><td>S-adenosylmethionine synthetase 2 (SAM-2) </td></tr><tr><td><em>SUR1</em> </td><td>C7 </td><td>200,177 </td><td>C_JCVI_531 </td><td>1.11 × 10<sup>−6</sup> </td><td>C-S lyase involved in converting S-alkylthiohydroximate to thiohydroximate </td></tr><tr><td><em>BCAT4</em> </td><td>A5 </td><td>16,822,435 </td><td>A_JCVI_34763 </td><td>1.18 × 10<sup>−6</sup> </td><td>Methionine-oxo-acid transaminase </td></tr><tr><td><em>AK3</em> </td><td>A2 </td><td>6,569,882 </td><td>A_EX056141 </td><td>1.25 × 10<sup>−6</sup> </td><td>Encodes a monofunctional aspartate kinase </td></tr><tr><td><em>BAT5</em> </td><td>C9 </td><td>28,810,650 </td><td>C_JCVI_16890 </td><td>2.92 × 10<sup>−6</sup> </td><td>Transporter of 2-keto acids between chloroplasts and the cytosol </td></tr><tr><td><em>SOT18</em> </td><td>A7 </td><td>23,340,811 </td><td>A_JCVI_17243 </td><td>3.56 × 10<sup>−6</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>GSTU23</em> </td><td>A7 </td><td>17,429,665 </td><td>A_EE467545 </td><td>1.47 × 10<sup>−5</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>UGT74B1</em> </td><td>A9 </td><td>27,647,388 </td><td>A_JCVI_31290 </td><td>1.68 × 10<sup>−5</sup> </td><td>UDP-glucose: thiohydroximate S-glucosyltransferase </td></tr><tr><td><em>MTO1</em> </td><td>A5 </td><td>23,899,195 </td><td>A_EV191151 </td><td>1.95 × 10<sup>−5</sup> </td><td>Cystathionine gamma-synthase </td></tr><tr><td><em>GTR2</em> </td><td>A2 </td><td>25,155,291 </td><td>A_JCVI_13343 </td><td>2.69 × 10<sup>−5</sup> </td><td>High-affinity, proton-dependent glucosinolate-specific transporter </td></tr><tr><td><em>AOP1</em> </td><td>A3 </td><td>14,140,138 </td><td>A_JCVI_31233 </td><td>4.41 × 10<sup>−5</sup> </td><td>Encodes a possible 2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>APS1</em> </td><td>A3 </td><td>19,165,411 </td><td>A_EV196558 </td><td>4.72 × 10<sup>−5</sup> </td><td>ATP sulfurylase </td></tr><tr><td><em>SOT17</em> </td><td>A6 </td><td>7,244,176 </td><td>A_JCVI_37729 </td><td>5.31 × 10<sup>−5</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>CBL</em> </td><td>C4 </td><td>29,183,950 </td><td>C_JCVI_35734 </td><td>5.38 × 10<sup>−5</sup> </td><td>Second enzyme in the methionine biosynthetic pathway </td></tr><tr><td><em>HAG3</em> </td><td>C9 </td><td>3,426,787 </td><td>C_EX043693 </td><td>6.86 × 10<sup>−5</sup> </td><td>MYB domain containing protein 29 </td></tr><tr><td><em>XT2</em> </td><td>A9 </td><td>987,590 </td><td>C_EE527736 </td><td>6.87 × 10<sup>−5</sup> </td><td>Protein with xylosyltransferase activity </td></tr><tr><td><em>FMO GS-OX2</em> </td><td>A9 </td><td>8,429,062 </td><td>A_JCVI_5227 </td><td>9.99 × 10<sup>−5</sup> </td><td>Glucosinolate S-oxygenase </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN7"><div class="fn"><p><sup>a</sup>The physical position on chromosome pseudomolecules.</p></div></div><div content-id="TN8"><div class="fn"><p><sup>b</sup>Only the lead GEM (most significant) was listed if there is more than one for a candidate gene.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721396" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU024TB3" class="table-wrap-title" data-id="DSU024TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of GEMs and candidate genes significantly associated with the seed glucosinolate content</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome </th><th>Position (bp)<sup>a</sup> </th><th>GEM<sup>b</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>HAG1</em> </td><td>A9 </td><td>3,425,844 </td><td>A_JCVI_40613 </td><td>8.28 × 10<sup>−10</sup> </td><td>MYB transcription factor family </td></tr><tr><td><em>CYP83A1</em> </td><td>C4 </td><td>32,842,478 </td><td>C_JCVI_26799 </td><td>3.97 × 10<sup>−9</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>MAM1</em> </td><td>A2 </td><td>24,487,376 </td><td>A_JCVI_30455 </td><td>1.91 × 10<sup>−8</sup> </td><td>Methylthiolkylmalate synthase </td></tr><tr><td><em>AOP2</em> </td><td>A9 </td><td>1,155,687 </td><td>A_JCVI_33047 </td><td>2.79 × 10<sup>−8</sup> </td><td>2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>AT2G31790.1</em> </td><td>A5 </td><td>6,660,619 </td><td>A_JCVI_6771 </td><td>3.63 × 10<sup>−8</sup> </td><td>UDP-Glycosyltransferase superfamily protein </td></tr><tr><td><em>ATGSTF11</em> </td><td>A5 </td><td>23,602,455 </td><td>A_JCVI_6891 </td><td>3.36 × 10<sup>−7</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>CYP79F1</em> </td><td>C5 </td><td>7,734,381 </td><td>C_JCVI_17335 </td><td>6.68 × 10<sup>−7</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>SAM-2</em> </td><td>C9 </td><td>805,498 </td><td>C_JCVI_12068 </td><td>8.02 × 10<sup>−7</sup> </td><td>S-adenosylmethionine synthetase 2 (SAM-2) </td></tr><tr><td><em>SUR1</em> </td><td>C7 </td><td>200,177 </td><td>C_JCVI_531 </td><td>1.11 × 10<sup>−6</sup> </td><td>C-S lyase involved in converting S-alkylthiohydroximate to thiohydroximate </td></tr><tr><td><em>BCAT4</em> </td><td>A5 </td><td>16,822,435 </td><td>A_JCVI_34763 </td><td>1.18 × 10<sup>−6</sup> </td><td>Methionine-oxo-acid transaminase </td></tr><tr><td><em>AK3</em> </td><td>A2 </td><td>6,569,882 </td><td>A_EX056141 </td><td>1.25 × 10<sup>−6</sup> </td><td>Encodes a monofunctional aspartate kinase </td></tr><tr><td><em>BAT5</em> </td><td>C9 </td><td>28,810,650 </td><td>C_JCVI_16890 </td><td>2.92 × 10<sup>−6</sup> </td><td>Transporter of 2-keto acids between chloroplasts and the cytosol </td></tr><tr><td><em>SOT18</em> </td><td>A7 </td><td>23,340,811 </td><td>A_JCVI_17243 </td><td>3.56 × 10<sup>−6</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>GSTU23</em> </td><td>A7 </td><td>17,429,665 </td><td>A_EE467545 </td><td>1.47 × 10<sup>−5</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>UGT74B1</em> </td><td>A9 </td><td>27,647,388 </td><td>A_JCVI_31290 </td><td>1.68 × 10<sup>−5</sup> </td><td>UDP-glucose: thiohydroximate S-glucosyltransferase </td></tr><tr><td><em>MTO1</em> </td><td>A5 </td><td>23,899,195 </td><td>A_EV191151 </td><td>1.95 × 10<sup>−5</sup> </td><td>Cystathionine gamma-synthase </td></tr><tr><td><em>GTR2</em> </td><td>A2 </td><td>25,155,291 </td><td>A_JCVI_13343 </td><td>2.69 × 10<sup>−5</sup> </td><td>High-affinity, proton-dependent glucosinolate-specific transporter </td></tr><tr><td><em>AOP1</em> </td><td>A3 </td><td>14,140,138 </td><td>A_JCVI_31233 </td><td>4.41 × 10<sup>−5</sup> </td><td>Encodes a possible 2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>APS1</em> </td><td>A3 </td><td>19,165,411 </td><td>A_EV196558 </td><td>4.72 × 10<sup>−5</sup> </td><td>ATP sulfurylase </td></tr><tr><td><em>SOT17</em> </td><td>A6 </td><td>7,244,176 </td><td>A_JCVI_37729 </td><td>5.31 × 10<sup>−5</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>CBL</em> </td><td>C4 </td><td>29,183,950 </td><td>C_JCVI_35734 </td><td>5.38 × 10<sup>−5</sup> </td><td>Second enzyme in the methionine biosynthetic pathway </td></tr><tr><td><em>HAG3</em> </td><td>C9 </td><td>3,426,787 </td><td>C_EX043693 </td><td>6.86 × 10<sup>−5</sup> </td><td>MYB domain containing protein 29 </td></tr><tr><td><em>XT2</em> </td><td>A9 </td><td>987,590 </td><td>C_EE527736 </td><td>6.87 × 10<sup>−5</sup> </td><td>Protein with xylosyltransferase activity </td></tr><tr><td><em>FMO GS-OX2</em> </td><td>A9 </td><td>8,429,062 </td><td>A_JCVI_5227 </td><td>9.99 × 10<sup>−5</sup> </td><td>Glucosinolate S-oxygenase </td></tr></tbody></table></div><div content-id="DSU024TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Candidate gene </th><th>Chromosome </th><th>Position (bp)<sup>a</sup> </th><th>GEM<sup>b</sup> </th><th><em>P</em>-value </th><th>Annotation </th></tr></thead><tbody><tr><td><em>HAG1</em> </td><td>A9 </td><td>3,425,844 </td><td>A_JCVI_40613 </td><td>8.28 × 10<sup>−10</sup> </td><td>MYB transcription factor family </td></tr><tr><td><em>CYP83A1</em> </td><td>C4 </td><td>32,842,478 </td><td>C_JCVI_26799 </td><td>3.97 × 10<sup>−9</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>MAM1</em> </td><td>A2 </td><td>24,487,376 </td><td>A_JCVI_30455 </td><td>1.91 × 10<sup>−8</sup> </td><td>Methylthiolkylmalate synthase </td></tr><tr><td><em>AOP2</em> </td><td>A9 </td><td>1,155,687 </td><td>A_JCVI_33047 </td><td>2.79 × 10<sup>−8</sup> </td><td>2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>AT2G31790.1</em> </td><td>A5 </td><td>6,660,619 </td><td>A_JCVI_6771 </td><td>3.63 × 10<sup>−8</sup> </td><td>UDP-Glycosyltransferase superfamily protein </td></tr><tr><td><em>ATGSTF11</em> </td><td>A5 </td><td>23,602,455 </td><td>A_JCVI_6891 </td><td>3.36 × 10<sup>−7</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>CYP79F1</em> </td><td>C5 </td><td>7,734,381 </td><td>C_JCVI_17335 </td><td>6.68 × 10<sup>−7</sup> </td><td>Cytochrome p450 enzyme </td></tr><tr><td><em>SAM-2</em> </td><td>C9 </td><td>805,498 </td><td>C_JCVI_12068 </td><td>8.02 × 10<sup>−7</sup> </td><td>S-adenosylmethionine synthetase 2 (SAM-2) </td></tr><tr><td><em>SUR1</em> </td><td>C7 </td><td>200,177 </td><td>C_JCVI_531 </td><td>1.11 × 10<sup>−6</sup> </td><td>C-S lyase involved in converting S-alkylthiohydroximate to thiohydroximate </td></tr><tr><td><em>BCAT4</em> </td><td>A5 </td><td>16,822,435 </td><td>A_JCVI_34763 </td><td>1.18 × 10<sup>−6</sup> </td><td>Methionine-oxo-acid transaminase </td></tr><tr><td><em>AK3</em> </td><td>A2 </td><td>6,569,882 </td><td>A_EX056141 </td><td>1.25 × 10<sup>−6</sup> </td><td>Encodes a monofunctional aspartate kinase </td></tr><tr><td><em>BAT5</em> </td><td>C9 </td><td>28,810,650 </td><td>C_JCVI_16890 </td><td>2.92 × 10<sup>−6</sup> </td><td>Transporter of 2-keto acids between chloroplasts and the cytosol </td></tr><tr><td><em>SOT18</em> </td><td>A7 </td><td>23,340,811 </td><td>A_JCVI_17243 </td><td>3.56 × 10<sup>−6</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>GSTU23</em> </td><td>A7 </td><td>17,429,665 </td><td>A_EE467545 </td><td>1.47 × 10<sup>−5</sup> </td><td>Glutathione transferase </td></tr><tr><td><em>UGT74B1</em> </td><td>A9 </td><td>27,647,388 </td><td>A_JCVI_31290 </td><td>1.68 × 10<sup>−5</sup> </td><td>UDP-glucose: thiohydroximate S-glucosyltransferase </td></tr><tr><td><em>MTO1</em> </td><td>A5 </td><td>23,899,195 </td><td>A_EV191151 </td><td>1.95 × 10<sup>−5</sup> </td><td>Cystathionine gamma-synthase </td></tr><tr><td><em>GTR2</em> </td><td>A2 </td><td>25,155,291 </td><td>A_JCVI_13343 </td><td>2.69 × 10<sup>−5</sup> </td><td>High-affinity, proton-dependent glucosinolate-specific transporter </td></tr><tr><td><em>AOP1</em> </td><td>A3 </td><td>14,140,138 </td><td>A_JCVI_31233 </td><td>4.41 × 10<sup>−5</sup> </td><td>Encodes a possible 2-oxoglutarate-dependent dioxygenase </td></tr><tr><td><em>APS1</em> </td><td>A3 </td><td>19,165,411 </td><td>A_EV196558 </td><td>4.72 × 10<sup>−5</sup> </td><td>ATP sulfurylase </td></tr><tr><td><em>SOT17</em> </td><td>A6 </td><td>7,244,176 </td><td>A_JCVI_37729 </td><td>5.31 × 10<sup>−5</sup> </td><td>Desulfoglucosinolate sulfotransferase </td></tr><tr><td><em>CBL</em> </td><td>C4 </td><td>29,183,950 </td><td>C_JCVI_35734 </td><td>5.38 × 10<sup>−5</sup> </td><td>Second enzyme in the methionine biosynthetic pathway </td></tr><tr><td><em>HAG3</em> </td><td>C9 </td><td>3,426,787 </td><td>C_EX043693 </td><td>6.86 × 10<sup>−5</sup> </td><td>MYB domain containing protein 29 </td></tr><tr><td><em>XT2</em> </td><td>A9 </td><td>987,590 </td><td>C_EE527736 </td><td>6.87 × 10<sup>−5</sup> </td><td>Protein with xylosyltransferase activity </td></tr><tr><td><em>FMO GS-OX2</em> </td><td>A9 </td><td>8,429,062 </td><td>A_JCVI_5227 </td><td>9.99 × 10<sup>−5</sup> </td><td>Glucosinolate S-oxygenase </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN7"><div class="fn"><p><sup>a</sup>The physical position on chromosome pseudomolecules.</p></div></div><div content-id="TN8"><div class="fn"><p><sup>b</sup>Only the lead GEM (most significant) was listed if there is more than one for a candidate gene.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/8721396" target="_blank">View Large</a></div></div><p>Collectively, a total of 607 marker loci and 17 association peaks were found to be associated with GS in respective SNP and GEM analyses. Among these peaks, five co-located on A2, A9, C2, C7, and C9 (Table <a reveal-id="DSU024TB1" data-open="DSU024TB1" class="link link-reveal link-table xref-fig">1</a>). Interestingly, four out of the five common peaks locate in the same intervals of previous QTLs for total GS contents<sup><a reveal-id="DSU024C39" data-open="DSU024C39" class="link link-ref link-reveal xref-bibr">39</a></sup> (Fig. <a reveal-id="DSU024F2" data-open="DSU024F2" class="link link-reveal link-table xref-fig">2</a>). Within these peaks, the previously characterized orthologs of <em>HAG1</em>,<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> a gene encoding transcription factor for GS biosynthesis, were re-identified by both SNP and GEMs (Tables <a reveal-id="DSU024TB2" data-open="DSU024TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSU024TB3" data-open="DSU024TB3" class="link link-reveal link-table xref-fig">3</a>), indicating the robustness of AT results. </p>                        <a id="8721398" scrollto-destination="8721398"></a>
<div id="DSU024F2" data-id="DSU024F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02402.png?Expires=2147483647&Signature=AmQeGACLGewleMpH638gZ2LblnhN3HMZb3fMNbj0eB4g3jYssPVD4tK1RXMd8X7z7aVA4xkeTFa7DeJCkcQIWUg-y1nVr7tgFQfcFLMCDtLYLdI4jxvbKUb7akSOEDx7fkzxOkXygqiIqlDjdAE3N3nyjQcg3ncFhO6LMzCWZZ5NNAAIntz-rXIt3DWRMlWWo~JMzluDfz1qD2PdGy9qroIKgTA~ij57~clHrj2LxuUlHPei~2383vFS~9Pnh3An~FWC4Gw81bRPxgoE8b94SQvEVT5pOzZZJNVJq8e0uGNX1krhchA1HWG0Ab5xFbavPVHQpeJ~EUxdYudEAv4LYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Associative transcriptomics for seed glucosinolate content. These plots are based on the association results in 101 lines using either 144,131 SNPs or 100,534 GEMs. Each dot represents a SNP (black) or a GEM (red). Blue bar beneath chromosome pseudomolecule indicates the confident interval of a QTL for total seed glucosinolate content reported.39" path-from-xml="dsu02402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721398/dsu02402.tiff" path-from-xml="dsu02402.tiff" target="_blank">View large</a><a section="8721398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02402.png?Expires=2147483647&Signature=5L9bNNEIRBGI15h~wE42eQ5icVM5qvqEJpGhAYfoH30qejQa5rBRBrbtIVb8AqXTKz-vsAOZi8-~ZpYSy0FYbMcOV9odrz2DWT3dzy9RhZ~QSH-3Rl32N8aKU4xporjZDTvnd8LRZ42AAArhYOcnP30GbTFTJ-FiLb8ZzUkCORC2usarN~71gzjc~RauOoc7cnxZ~xnXM4jD2UsREpObZ2BxqtTiW~IYcZe-PWWGnK3pWve~PWezf9KUa-4hWn9NzjjuyVFZEFzXFUd6w5DTMp-6rWA0Ffw2AElDBCBDEr8D-ryCVyJus0RQuvj5OiCQZ4voX9kYWknwm1zUBIS04A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721398&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Associative transcriptomics for seed glucosinolate content. These plots are based on the association results in 101 lines using either 144,131 SNPs or 100,534 GEMs. Each dot represents a SNP (black) or a GEM (red). Blue bar beneath chromosome pseudomolecule indicates the confident interval of a QTL for total seed glucosinolate content reported.<sup><a reveal-id="DSU024C39" data-open="DSU024C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p></div></div><div content-id="DSU024F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02402.png?Expires=2147483647&Signature=AmQeGACLGewleMpH638gZ2LblnhN3HMZb3fMNbj0eB4g3jYssPVD4tK1RXMd8X7z7aVA4xkeTFa7DeJCkcQIWUg-y1nVr7tgFQfcFLMCDtLYLdI4jxvbKUb7akSOEDx7fkzxOkXygqiIqlDjdAE3N3nyjQcg3ncFhO6LMzCWZZ5NNAAIntz-rXIt3DWRMlWWo~JMzluDfz1qD2PdGy9qroIKgTA~ij57~clHrj2LxuUlHPei~2383vFS~9Pnh3An~FWC4Gw81bRPxgoE8b94SQvEVT5pOzZZJNVJq8e0uGNX1krhchA1HWG0Ab5xFbavPVHQpeJ~EUxdYudEAv4LYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Associative transcriptomics for seed glucosinolate content. These plots are based on the association results in 101 lines using either 144,131 SNPs or 100,534 GEMs. Each dot represents a SNP (black) or a GEM (red). Blue bar beneath chromosome pseudomolecule indicates the confident interval of a QTL for total seed glucosinolate content reported.39" path-from-xml="dsu02402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721398/dsu02402.tiff" path-from-xml="dsu02402.tiff" target="_blank">View large</a><a section="8721398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02402.png?Expires=2147483647&Signature=5L9bNNEIRBGI15h~wE42eQ5icVM5qvqEJpGhAYfoH30qejQa5rBRBrbtIVb8AqXTKz-vsAOZi8-~ZpYSy0FYbMcOV9odrz2DWT3dzy9RhZ~QSH-3Rl32N8aKU4xporjZDTvnd8LRZ42AAArhYOcnP30GbTFTJ-FiLb8ZzUkCORC2usarN~71gzjc~RauOoc7cnxZ~xnXM4jD2UsREpObZ2BxqtTiW~IYcZe-PWWGnK3pWve~PWezf9KUa-4hWn9NzjjuyVFZEFzXFUd6w5DTMp-6rWA0Ffw2AElDBCBDEr8D-ryCVyJus0RQuvj5OiCQZ4voX9kYWknwm1zUBIS04A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721398&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Associative transcriptomics for seed glucosinolate content. These plots are based on the association results in 101 lines using either 144,131 SNPs or 100,534 GEMs. Each dot represents a SNP (black) or a GEM (red). Blue bar beneath chromosome pseudomolecule indicates the confident interval of a QTL for total seed glucosinolate content reported.<sup><a reveal-id="DSU024C39" data-open="DSU024C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p></div></div>                            <h3 scrollto-destination=8721399 id="8721399" class="section-title" data-legacy-id=s3e>3.5. Distribution pattern of GEMs</h3>
<p>There was a well-defined association peak on A2. Interestingly, GEMs within this peak showed a distinct clustering pattern, as reflected by the existence of a gap (i.e. some GEMs clustered at –log<sub>10</sub><em>P</em> &gt; 6.2 while the others at &lt;4.9) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5b</a>). Actually, the eight GEMs clustering at the top of this peak were highly correlated with each other (<em>P</em> &lt; 0.01), and two of them have similar functions (encode methylthioalkylmalate synthase, <em>MAM</em>)<sup><a reveal-id="DSU024C62" data-open="DSU024C62" class="link link-ref link-reveal xref-bibr">62</a></sup> in GS biosynthesis, whereas the other six encode proteins with unknown functions. This observation indicated that genes involved in the GS pathway may tend to be located in close proximity to each other and have similar expression patterns, resulting in this clustering of GEMs.</p>                            <h3 scrollto-destination=8721401 id="8721401" class="section-title" data-legacy-id=s3f>3.6. Gene co-expression analysis for GS</h3>
<p>GEM analysis has the additional advantage that it can be used for WGCNA to dissect the biological process underlying traits. This approach was used to construct a co-expression network for the GS concentration that contained 122 modules (co-expressed genes), with between 31 and 16,989 unigenes in each module. The ‘lightblue4’ module was highly correlated with the total GS content of seeds (<em>r</em> = 0.55; <em>P</em> = 2.0 × 10<sup>−8</sup>) (see also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). This module comprises 91 unigenes (114 GEMs), of which 40 were implied to be involved in GS biosynthesis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>Then, a co-expression network was constructed with probes (genes) from ‘lightblue4’ module to identify the relationships between genes highly associated with GS metabolism. It was found that the unigene JCVI-16890, the <em>Arabidopsis</em> ortholog of which encodes a plastidic bile acid transporter (<em>BAT5</em>),<sup><a reveal-id="DSU024C63" data-open="DSU024C63" class="link link-ref link-reveal xref-bibr">63</a></sup> is in the central node in this network. Other hub genes in this network included JCVI_12709 and JCVI_30455, the orthologs of which encode branched-chain amino acid aminotransferase (<em>BCAT/MAA</em>T)<sup><a reveal-id="DSU024C64" data-open="DSU024C64" class="link link-ref link-reveal xref-bibr">64</a></sup> and <em>MAM1</em>, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). All these genes have been shown to play key roles in the GS biosynthesis pathway in <em>Arabidopsis</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>GO analysis was further performed with unigenes from the ‘lightblue4’ module to construct a biological metabolism network. As a result, most of the genes are enriched in the biological process related to GS synthesis, such as carbohydrate metabolic process and cellular nitrogen compound metabolic process in the initial stage and later in the GS biosynthetic process and sulphur amino acid biosynthetic process. Meanwhile, a few genes encode proteins for cellular components responding to external stimulus (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>).</p>                            <h3 scrollto-destination=8721405 id="8721405" class="section-title" data-legacy-id=s3g>3.7. Identification of DNA polymorphism by re-sequencing</h3>
<p>Approximately 66 candidate genes involved in the GS metabolic pathway have been identified by either AT or WGCNA (Tables <a reveal-id="DSU024TB2" data-open="DSU024TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSU024TB3" data-open="DSU024TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Some genes of interest were further selected to investigate and verify the potential associations of allelic variation of genes with phenotypic variation in the association panel. This was achieved by amplifying and re-sequencing PCR products using lines with various GS contents in seeds.</p><p>The first unigene of interest is JCVI_13343 (<em>BnaA.GTR2a</em>); the <em>Arabidopsis</em> ortholog encodes proteins for transporting GS compounds from leaf to seed.<sup><a reveal-id="DSU024C33" data-open="DSU024C33" class="link link-ref link-reveal xref-bibr">33</a></sup><em>BnaA.GTR2a</em> is located in a genomic region within the peak on A2 (Fig. <a reveal-id="DSU024F3" data-open="DSU024F3" class="link link-reveal link-table xref-fig">3</a>a), and its expression was positively correlated with the accumulation of GS in seeds (<em>r</em> = 0.43, <em>P</em> &lt; 10<sup>−4</sup>), accounting for 18.8% of trait variation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10a</a>). Although there were nine known SNPs (at positions 469, 625, 655, 667, 688, 775, 860, 861, and 952) within this unigene, none of them was strongly associated with GS. To detect the potential new sequence variation (i.e. insertion or deletion, InDel) within this locus, specific primers were designed and used to amplify the complete unigene (<em>ca.</em> 600 bp) from 42 lines. By sequences alignment, all nine known SNPs were re-identified although some of them were not polymorphic (as only a subset of lines were analysed), and six were previously unidentified SNPs (at positions 538, 565, 571, 640, 727, and 748) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). These results confirmed the robustness and efficiency of SNP development via mRNA-Seq in <em>B. napus</em>. However, none of the 10 polymorphic SNPs (at positions 469, 538, 565, 571, 640, 655, 727, 748, 860, and 861) were correlated with the GS content (<em>r</em> &lt; 0.11, <em>P</em> &gt; 0.500) and thus were not likely to be causative for trait variation. </p>                        <a id="8721408" scrollto-destination="8721408"></a>
<div id="DSU024F3" data-id="DSU024F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02403.png?Expires=2147483647&Signature=lQROY9ezj~eop4e0mCsff~31OIQRGCcFwLncFh7tPUzn6KhsPlNFmhvnVbXe0kBbm8VUxzJRKQr6CFC7J10TEoVhLvPbFvWvXUXJZ0K0Mqny4cu-1n~5Ju9CUI-5i5DdUlqaNjWmqe0S7a6Kid~hnuBOmf~xSrQ2lHaNRXvlE4mGgvL8ihSuPB~RBBn43hpvvNQplh9MqwhDKBTq5xHAP-aVyQF4xX9MHUHLzTikkutft1B1ugBgoUok5h9svkIiX4Nr~WBnTBLz9zVVsBYSvn3XhgthdQicqfC34zTKs7CTnNrL-pAc1moof0cdjS9AeHCdv4yic8cjoaSdhp0-cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Associations and genomic locations of two candidate genes for the seed glucosinolate content. Top, marker association scans are illustrated, for both SNP (black dot) and GEM (red dot) markers, with significance of association (as −log10P values) plotted against positions within a specific chromosome pseudomolecule. Bottom, a representation of the pairwise r2 (a measure of LD) among the mapped SNPs surrounding the peak, where the colour of each box corresponds to the r2 value according to the legend. The positions of the candidate genes are indicated by arrows. (a) A locus identified on A2, and (b) A locus identified on C9." path-from-xml="dsu02403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721408/dsu02403.tiff" path-from-xml="dsu02403.tiff" target="_blank">View large</a><a section="8721408" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02403.png?Expires=2147483647&Signature=4yPTIDDfnTg7yIf1TzmhqEV9KCdmQ4nfU1Bcoz2wp7s02OdJiYZFEYIYSrH-uYgWnHUEJBh7ZFGbWVXtEjsFDE8eROlnQdA15LZFyCUuMnDUxZkpQkxoYF2w1eQtnBgtBvGO6MMj2r6J-cxm00cASE-hNgLB9nZuFaZG2c1fPQvh2gDPSK3Ik8IfMif86hQFnofFhiv9TVpF6UWpOLomnd0gYlrAVqACyfglV-19CqEaYk187sSBepZrTcH8sgm63Y9q3E~Nq2xsFe90TlLai-P5YzWVuFqmVzDFFp8G0ZJqqb-BPkWI9TDhechGZ056BECo54EbA0ap7qXWURqhKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721408&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Associations and genomic locations of two candidate genes for the seed glucosinolate content. Top, marker association scans are illustrated, for both SNP (black dot) and GEM (red dot) markers, with significance of association (as −log<sub>10</sub><em>P</em> values) plotted against positions within a specific chromosome pseudomolecule. Bottom, a representation of the pairwise <em>r</em><sup>2</sup> (a measure of LD) among the mapped SNPs surrounding the peak, where the colour of each box corresponds to the <em>r</em><sup>2</sup> value according to the legend. The positions of the candidate genes are indicated by arrows. (a) A locus identified on A2, and (b) A locus identified on C9.</p></div></div><div content-id="DSU024F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02403.png?Expires=2147483647&Signature=lQROY9ezj~eop4e0mCsff~31OIQRGCcFwLncFh7tPUzn6KhsPlNFmhvnVbXe0kBbm8VUxzJRKQr6CFC7J10TEoVhLvPbFvWvXUXJZ0K0Mqny4cu-1n~5Ju9CUI-5i5DdUlqaNjWmqe0S7a6Kid~hnuBOmf~xSrQ2lHaNRXvlE4mGgvL8ihSuPB~RBBn43hpvvNQplh9MqwhDKBTq5xHAP-aVyQF4xX9MHUHLzTikkutft1B1ugBgoUok5h9svkIiX4Nr~WBnTBLz9zVVsBYSvn3XhgthdQicqfC34zTKs7CTnNrL-pAc1moof0cdjS9AeHCdv4yic8cjoaSdhp0-cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Associations and genomic locations of two candidate genes for the seed glucosinolate content. Top, marker association scans are illustrated, for both SNP (black dot) and GEM (red dot) markers, with significance of association (as −log10P values) plotted against positions within a specific chromosome pseudomolecule. Bottom, a representation of the pairwise r2 (a measure of LD) among the mapped SNPs surrounding the peak, where the colour of each box corresponds to the r2 value according to the legend. The positions of the candidate genes are indicated by arrows. (a) A locus identified on A2, and (b) A locus identified on C9." path-from-xml="dsu02403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721408/dsu02403.tiff" path-from-xml="dsu02403.tiff" target="_blank">View large</a><a section="8721408" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02403.png?Expires=2147483647&Signature=4yPTIDDfnTg7yIf1TzmhqEV9KCdmQ4nfU1Bcoz2wp7s02OdJiYZFEYIYSrH-uYgWnHUEJBh7ZFGbWVXtEjsFDE8eROlnQdA15LZFyCUuMnDUxZkpQkxoYF2w1eQtnBgtBvGO6MMj2r6J-cxm00cASE-hNgLB9nZuFaZG2c1fPQvh2gDPSK3Ik8IfMif86hQFnofFhiv9TVpF6UWpOLomnd0gYlrAVqACyfglV-19CqEaYk187sSBepZrTcH8sgm63Y9q3E~Nq2xsFe90TlLai-P5YzWVuFqmVzDFFp8G0ZJqqb-BPkWI9TDhechGZ056BECo54EbA0ap7qXWURqhKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721408&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Associations and genomic locations of two candidate genes for the seed glucosinolate content. Top, marker association scans are illustrated, for both SNP (black dot) and GEM (red dot) markers, with significance of association (as −log<sub>10</sub><em>P</em> values) plotted against positions within a specific chromosome pseudomolecule. Bottom, a representation of the pairwise <em>r</em><sup>2</sup> (a measure of LD) among the mapped SNPs surrounding the peak, where the colour of each box corresponds to the <em>r</em><sup>2</sup> value according to the legend. The positions of the candidate genes are indicated by arrows. (a) A locus identified on A2, and (b) A locus identified on C9.</p></div></div><p>Another unigene of interest is EX043693 (<em>BnaC.HAG3b</em>), which is located within the peak on C9 which coincides with a region of strong and extensive LD (Fig. <a reveal-id="DSU024F3" data-open="DSU024F3" class="link link-reveal link-table xref-fig">3</a>b). In <em>Arabidopsis</em>, <em>HAG3</em> (<em>MYB29</em>) is a transcription factor modulating many genes in the GS pathway.<sup><a reveal-id="DSU024C65" data-open="DSU024C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Expression of <em>BnaC.HAG3b</em> is highly associated with GS accumulation (<em>r</em> = 0.41, <em>P</em> &lt; 10<sup>−8</sup>) and accounts for 16.8% of trait variation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10b</a>). However, no SNPs have been detected by mRNA-Seq within this gene. Therefore, genomic regions covering the length of EX043693 were amplified from 70 lines and then sequenced to detect potential DNA sequence variations. By comparing sequences of these DNA fragments, four InDels, i.e. InDel3–1 (3-bp insertion), InDel3–2 (3-bp deletion), InDel7 (7-bp deletion), and InDel1 (1-bp deletion), were detected. These InDels formed two haplotypes: haplotype I includes the 3-bp insertion and 11-bp (i.e. 3 plus 7 plus 1) of deletions, while haplotype II has the sequence identical to the reference unigene EX043693 (Fig. <a reveal-id="DSU024F4" data-open="DSU024F4" class="link link-reveal link-table xref-fig">4</a>). A total number of 60 lines were determined as haplotype I at these loci and 10 lines as haplotype II. The average GS concentration of haplotype I (56.3 µmol g<sup>−1</sup>) was only 55% of that of haplotype II (the wild type), consistent with the net 8-bp (frame-shift) deletion reducing the functional properties of the encoded protein. To facilitate germplasm screening and marker-assisted selection for the low GS content, a pair of specific PCR primers was designed, which only captures the 11-bp deletions so that the polymorphism can be more easily resolved by agarose gel (Fig. <a reveal-id="DSU024F4" data-open="DSU024F4" class="link link-reveal link-table xref-fig">4</a>). An example of PCR amplification is given in Fig. <a reveal-id="DSU024F5" data-open="DSU024F5" class="link link-reveal link-table xref-fig">5</a>; the haplotype I and II lines can be clearly distinguished by the presence of a 226-bp- and 237-bp-specific fragment, respectively. Thus, the polymorphism at <em>BnaC.HAG3b</em> locus has been successfully converted into a PCR-based marker. </p>                        <a id="8721410" scrollto-destination="8721410"></a>
<div id="DSU024F4" data-id="DSU024F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02404.png?Expires=2147483647&Signature=lYQm8g421Ix9-5jK8fP9Pnt~49v1J6AdDtWw2ryh3AkOQwowNSot6JmQ-DbJMZoyBrBSPbMA1LrA3dhcHMSLO4~XZoTxgiKg83b2gkvmfAbX0GRxL6WGBT9YmJlARAog5vWapJuDNx2R2ZNrO1-uy~t-P8vOhuFFPSzFCVJgLS-9vQqmEK6WKThrMA79EmrE8ckt9f9xVtsPC6YxkeCHp3jt~RRg0xLbgtO4qYg110gQJA9yPYHyUDiPW0-CHTSRdSwKNz~7i0p9uWv7c~OPffik-A6dtyAslRzoFaGwFw67rHrhpjZaIORY3x4O0oB1wE-394IX~sZKZM4oCmOrlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of significant polymorphisms at BnaC.HAG3b locus. The locations of DNA sequence polymorphisms (in bp) are based on unigene EX043693. All four polymorphisms were combined into two haplotypes. The number of lines sharing each haplotype, as well as the glucosinolate content (mean±standard error) was given at the right. **indicates the statistical difference at P &lt; 0.01 in t-test. Arrows indicate the positions of primers (260F: TTGTAATAGAGTTCATATATATCG; 490R: TTCATACATCAAATACCAAAC) for the converted PCR marker." path-from-xml="dsu02404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721410/dsu02404.tiff" path-from-xml="dsu02404.tiff" target="_blank">View large</a><a section="8721410" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02404.png?Expires=2147483647&Signature=oh3iMorxnDSWi-TwN6vAaZy5Ic-c5P2UjRyH9UEPaJNfoPl2~sBU3EJL88QIotz-cIN6z0ltBKgCO8nla1uneR1con-AHPi8tubo1XrTVUnx1EsCYR3QKiHvZQmq1Txd~q1CM4q~KCZVOcH8W88~GkhF2GKMezS95bWxkjd-0kdo6zrT0QWebuh1fqpNYiEjS3jCwVKHSWv8sg76-bRD2BGeXUZIXExmOkBCfvYpvVcuujDZllLHJZ9tDu7DpcgwOY2eQypV8lqYi1oM3QNEWQ4bGCCJTZW2OdSQOD3Odc-uP3s-Lk44dPAM7mZPX-PV0a35YO-dxc8SgHAbO90i1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721410&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of significant polymorphisms at <em>BnaC.HAG3b</em> locus. The locations of DNA sequence polymorphisms (in bp) are based on unigene EX043693. All four polymorphisms were combined into two haplotypes. The number of lines sharing each haplotype, as well as the glucosinolate content (mean±standard error) was given at the right. **indicates the statistical difference at <em>P</em> &lt; 0.01 in <em>t</em>-test. Arrows indicate the positions of primers (260F: TTGTAATAGAGTTCATATATATCG; 490R: TTCATACATCAAATACCAAAC) for the converted PCR marker.</p></div></div><div content-id="DSU024F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02404.png?Expires=2147483647&Signature=lYQm8g421Ix9-5jK8fP9Pnt~49v1J6AdDtWw2ryh3AkOQwowNSot6JmQ-DbJMZoyBrBSPbMA1LrA3dhcHMSLO4~XZoTxgiKg83b2gkvmfAbX0GRxL6WGBT9YmJlARAog5vWapJuDNx2R2ZNrO1-uy~t-P8vOhuFFPSzFCVJgLS-9vQqmEK6WKThrMA79EmrE8ckt9f9xVtsPC6YxkeCHp3jt~RRg0xLbgtO4qYg110gQJA9yPYHyUDiPW0-CHTSRdSwKNz~7i0p9uWv7c~OPffik-A6dtyAslRzoFaGwFw67rHrhpjZaIORY3x4O0oB1wE-394IX~sZKZM4oCmOrlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of significant polymorphisms at BnaC.HAG3b locus. The locations of DNA sequence polymorphisms (in bp) are based on unigene EX043693. All four polymorphisms were combined into two haplotypes. The number of lines sharing each haplotype, as well as the glucosinolate content (mean±standard error) was given at the right. **indicates the statistical difference at P &lt; 0.01 in t-test. Arrows indicate the positions of primers (260F: TTGTAATAGAGTTCATATATATCG; 490R: TTCATACATCAAATACCAAAC) for the converted PCR marker." path-from-xml="dsu02404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721410/dsu02404.tiff" path-from-xml="dsu02404.tiff" target="_blank">View large</a><a section="8721410" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02404.png?Expires=2147483647&Signature=oh3iMorxnDSWi-TwN6vAaZy5Ic-c5P2UjRyH9UEPaJNfoPl2~sBU3EJL88QIotz-cIN6z0ltBKgCO8nla1uneR1con-AHPi8tubo1XrTVUnx1EsCYR3QKiHvZQmq1Txd~q1CM4q~KCZVOcH8W88~GkhF2GKMezS95bWxkjd-0kdo6zrT0QWebuh1fqpNYiEjS3jCwVKHSWv8sg76-bRD2BGeXUZIXExmOkBCfvYpvVcuujDZllLHJZ9tDu7DpcgwOY2eQypV8lqYi1oM3QNEWQ4bGCCJTZW2OdSQOD3Odc-uP3s-Lk44dPAM7mZPX-PV0a35YO-dxc8SgHAbO90i1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721410&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of significant polymorphisms at <em>BnaC.HAG3b</em> locus. The locations of DNA sequence polymorphisms (in bp) are based on unigene EX043693. All four polymorphisms were combined into two haplotypes. The number of lines sharing each haplotype, as well as the glucosinolate content (mean±standard error) was given at the right. **indicates the statistical difference at <em>P</em> &lt; 0.01 in <em>t</em>-test. Arrows indicate the positions of primers (260F: TTGTAATAGAGTTCATATATATCG; 490R: TTCATACATCAAATACCAAAC) for the converted PCR marker.</p></div></div>                        <a id="8721411" scrollto-destination="8721411"></a>
<div id="DSU024F5" data-id="DSU024F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02405.png?Expires=2147483647&Signature=gDqPpX4l1~vSqCdQR105dd0kUN1OmxE-mACE45yPNe6ezB5qUxqsld80SvYIWLmxYJCcThXb8ZT7fsSwelgJ7wztkyJlUTUzUOjA7zr1u8-Wsg4Ow6KIAjp0129lavCCI5qvgybf1PMrvWt7O~xARZKbJUv4PNnwxsUQOCcTF0tx0rheauewDDG57zpvgBkqKomhRJBtjrseJeROOTg3QhXI0x9q4MO6Shh-zh6lZNEZ38~6tZ5VZizncXsg49dnfR3TJwLXa4q9WGBFASQIHYrp7t0~VWwsPjWR~Jspiq-Xd2GJPyJS5HMXYQA1SYfn-Dg1IM1j1vkzPH0HmgIptg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCR assay for the 11-bp deletions at BnaC.HAG3b locus. PCR primer combination 260F/490R was used, which produced a 226-bp (haplotype I) or 237-bp (haplotype II) band. Numbers on the left and right sides are fragment length in base pair. M, 100-bp DNA ladder." path-from-xml="dsu02405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721411/dsu02405.tiff" path-from-xml="dsu02405.tiff" target="_blank">View large</a><a section="8721411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02405.png?Expires=2147483647&Signature=nlIvDsbdj0TLtlfJjG-SSua~kA40z7X7PruN5jKHQHXKFrkTIjbpm1U-UkLsdnesfYKJIGlv3KiGR36E5UwfgW7eY77fSJtP8feGO4ezPFwowDteExCQ-dh8dfwXUOReScP9z2sEHJKx9s0hW5GG8-RAIRhdf1Y3LbuRWn31oNASFDtb1NiJb75HEKCmJFj0BHh8RkPztVeit8Ww0bQKBLefwvprQfZoiSZHJH~IOmBX6G7Z4dlEA74YkiDtFu0ycYLX19O9oBwx2FhlYTGJovd-lmKRNrihiK-38Jd4xmyh8LtQmMrXajdr9g4DTq2TrmBu3GWFFo69GX6CShIcbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721411&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCR assay for the 11-bp deletions at <em>BnaC.HAG3b</em> locus. PCR primer combination 260F/490R was used, which produced a 226-bp (haplotype I) or 237-bp (haplotype II) band. Numbers on the left and right sides are fragment length in base pair. M, 100-bp DNA ladder.</p></div></div><div content-id="DSU024F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/m_dsu02405.png?Expires=2147483647&Signature=gDqPpX4l1~vSqCdQR105dd0kUN1OmxE-mACE45yPNe6ezB5qUxqsld80SvYIWLmxYJCcThXb8ZT7fsSwelgJ7wztkyJlUTUzUOjA7zr1u8-Wsg4Ow6KIAjp0129lavCCI5qvgybf1PMrvWt7O~xARZKbJUv4PNnwxsUQOCcTF0tx0rheauewDDG57zpvgBkqKomhRJBtjrseJeROOTg3QhXI0x9q4MO6Shh-zh6lZNEZ38~6tZ5VZizncXsg49dnfR3TJwLXa4q9WGBFASQIHYrp7t0~VWwsPjWR~Jspiq-Xd2GJPyJS5HMXYQA1SYfn-Dg1IM1j1vkzPH0HmgIptg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCR assay for the 11-bp deletions at BnaC.HAG3b locus. PCR primer combination 260F/490R was used, which produced a 226-bp (haplotype I) or 237-bp (haplotype II) band. Numbers on the left and right sides are fragment length in base pair. M, 100-bp DNA ladder." path-from-xml="dsu02405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/8721411/dsu02405.tiff" path-from-xml="dsu02405.tiff" target="_blank">View large</a><a section="8721411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu02405.png?Expires=2147483647&Signature=nlIvDsbdj0TLtlfJjG-SSua~kA40z7X7PruN5jKHQHXKFrkTIjbpm1U-UkLsdnesfYKJIGlv3KiGR36E5UwfgW7eY77fSJtP8feGO4ezPFwowDteExCQ-dh8dfwXUOReScP9z2sEHJKx9s0hW5GG8-RAIRhdf1Y3LbuRWn31oNASFDtb1NiJb75HEKCmJFj0BHh8RkPztVeit8Ww0bQKBLefwvprQfZoiSZHJH~IOmBX6G7Z4dlEA74YkiDtFu0ycYLX19O9oBwx2FhlYTGJovd-lmKRNrihiK-38Jd4xmyh8LtQmMrXajdr9g4DTq2TrmBu3GWFFo69GX6CShIcbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=8721411&ar=551332&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCR assay for the 11-bp deletions at <em>BnaC.HAG3b</em> locus. PCR primer combination 260F/490R was used, which produced a 226-bp (haplotype I) or 237-bp (haplotype II) band. Numbers on the left and right sides are fragment length in base pair. M, 100-bp DNA ladder.</p></div></div>                            <h2 scrollto-destination=8721412 id="8721412" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=8721413 id="8721413" class="section-title" data-legacy-id=s4a>4.1. Scenario of AT for GS</h3>
<p>Association mapping is a powerful tool for the identification of genes underlying complex traits. However, it is not so straightforward to perform GWAS in polyploid crops such as rapeseed, mainly due to the complexity of genome constitution and the lack of complete genome reference sequences. As an allopolyploid species, rapeseed is formed by the hybridization of progenitor species <em>B. rapa</em> (which contributed the A genome) and <em>B. oleracea</em> (which contributed the C genome).<sup><a reveal-id="DSU024C66" data-open="DSU024C66" class="link link-ref link-reveal xref-bibr">66</a></sup> The constituent A and C genomes within <em>B. napus</em> are highly similar to their ancestors, with only ∼15% difference at a nucleotide level and only 3% at a transcript level,<sup><a reveal-id="DSU024C51" data-open="DSU024C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSU024C54" data-open="DSU024C54" class="link link-ref link-reveal xref-bibr">54</a></sup> which thus hinder the development of tens of thousands of SNPs for GWAS. To address this challenge, our study focused on the development of molecular markers by mRNA-Seq, which can not only detect the sequence variation (i.e. SNPs) but also transcript abundance (i.e. GEMs). Juvenile leaf was used as a tissue for mRNA extraction, because it has a large number of expressing genes and so serves as a good gene compartment. In our AT study, SNPs were successfully associated with GS concentration in seeds. Interestingly, such association could also be achieved by using GEMs as independent variants to regress with GS contents as dependent variants. The scenario of AT for GS is that the juvenile leaf, where expressing genes have been captured by mRNA-Seq, is a major organ for GS synthesis at the vegetative stage. The GS compounds are then stored in these organs and subsequently transported to the embryos at a later developmental stage, as exhibited in <em>Arabidopsis</em>.<sup><a reveal-id="DSU024C30  DSU024C33" data-open="DSU024C30  DSU024C33" class="link link-ref link-reveal xref-bibr">30–33</a></sup> Thus, GS accumulation in seeds is biologically connected with gene expression in leaves. Indeed, the expression of many genes, including those already known to be involved in the GS pathway, was found to be highly associated with the GS content and formed several peaks on the genome (Fig. <a reveal-id="DSU024F2" data-open="DSU024F2" class="link link-reveal link-table xref-fig">2</a>). More generally, the high association of gene expression at the early developmental stage (e.g. juvenile leaf) with a target trait at a later development stage (e.g. in seeds) is due to allelic, <em>cis</em>-acting variation rather than being a read-out of a transcription network, as hypothesized previously for the association in maize hybrids of transcript abundance variation in leaves with grain yield.<sup><a reveal-id="DSU024C67" data-open="DSU024C67" class="link link-ref link-reveal xref-bibr">67</a></sup> With this concept in mind, many association peaks were also identified in wheat for straw biomass traits such as height, weight, and width by AT (Harper <em>et al.</em>, submitted). Therefore, it seems that AT can be widely applied to many crops, even including those with complex genomes like rapeseed and wheat. Moreover, candidate loci can also be successfully identified using the transcriptome of a single tissue that provides a suitable genome compartment, such as juvenile leaf. This has the added benefit that multiple trait types can be mapped using a single mRNA-Seq data set.</p>                            <h3 scrollto-destination=8721415 id="8721415" class="section-title" data-legacy-id=s4b>4.2. Improving the resolution of AT</h3>
<p>Compared with the previous AT study for the GS content in <em>B. napus</em> using 53 lines as a proof of concept,<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> a greatly enhanced resolution was achieved by using an extended panel comprising 101 lines in this study. With this new diversity panel, the total number of SNP markers was found to increase from the previous 101,644 to the present 225,011. Even after removing markers with MAF &lt; 0.05, the informative SNP markers (144,131) that can be used for the AT study were also twice as high as the previous report, leading to the detection of 10 association peaks at a significance level of <em>P</em> &lt; 6.9 × 10<sup>−6</sup>. In comparison, only four peaks were detected in the previous study, even at a lower threshold (<em>P</em> &lt; 10<sup>−4</sup>).<sup><a reveal-id="DSU024C36" data-open="DSU024C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Likewise, three more peaks were detected using a similar number of GEMs in this study, leading to the identification of many more candidate genes for GS. Given that many more association peaks and candidate genes were identified, the resolution of AT appears to be markedly improved by using an enlarged panel. It is anticipated that the resolution can be further improved by using an even larger diversity panel.</p><p>Most recently, Li <em>et al.</em><sup><a reveal-id="DSU024C49" data-open="DSU024C49" class="link link-ref link-reveal xref-bibr">49</a></sup> also carried out a GWAS and identified four association peaks on A9, C2, C7, and C9 for GS content, which were reconfirmed in our study. It is worth noting that the number of association peaks in our analysis is still superior to theirs (i.e. 10 versus 7) although they have used a larger panel (472 lines), possibly due to a much larger number of SNP markers employed (144,131 versus 24,256). Another unique feature of our study is that additional GEMs can be developed for marker-trait association and gene co-expression analysis, which in turn allowed for the identification of many more candidate genes (Table <a reveal-id="DSU024TB3" data-open="DSU024TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                            <h3 scrollto-destination=8721418 id="8721418" class="section-title" data-legacy-id=s4c>4.3. Possible function of new genes inferred from gene co-expression analysis</h3>
<p>In our study, a gene network for GS has been inferred from WGCNA (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>), which was further confirmed by GO analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). As for GO analysis, most genes are enriched in GS or sulphate amino acid (a precursor for GS) synthesis. The unigene JCVI_16890 (<em>BnaC.BAT5</em>) was found in the hub of the core network (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). In <em>Arabidopsis</em>, <em>BAT5</em> is a member of the putative bile acid transporter family and the target of the aliphatic GS regulators, <em>HAG1</em> and <em>HAG3</em> (<em>MYB29</em>). Moreover, <em>BAT5</em> mediates the transport of 4-methylthio-2-oxobutanoate and of long-chain 2-keto acids across the chloroplast envelope membrane before, during, and after side-chain elongation of 2-keto acids and is thus a key player in the aliphatic GS biosynthetic pathway.<sup><a reveal-id="DSU024C63" data-open="DSU024C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Other genes in the present core network included <em>MAM1</em>, <em>BCAT4</em>, and <em>AOP2</em>, and those genes encoding proteins for amino acid metabolites (<em>AK3</em> and <em>IMD1</em>). These genes function in nearly all key steps in the GS pathway (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), and all were connected to the some extent, with <em>BAT5</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). Thus, <em>BnaC.BAT5</em> also seems to have a key role for GS biosynthesis in <em>B. napus</em>, which was underlined by the fact that the transcript abundance of <em>BnaC.BAT5</em> is positively correlated with GS accumulation in seeds (<em>r</em> = 0.493, <em>P</em> = 1.2 × 10<sup>−6</sup>).</p><p>Another unigene of interest was JCVI_9761 (ortholog of AT5G14910), which was the only one found in common between the WGCNA, SNP, and GEM analyses. This gene encodes a putative heavy metal transport/detoxification containing domain protein in <em>Arabidopsis</em>. It is located in the chloroplast thylakoid membrane, chloroplast stroma, or chloroplast, and is involved in heavy metal ion transport (<a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>). Although evidence is still lacking for the direct connection of AT5G14910 with GS biosynthesis, some clues exist for such a connection. For instance, a complex interaction between metals and GS levels was observed,<sup><a reveal-id="DSU024C68" data-open="DSU024C68" class="link link-ref link-reveal xref-bibr">68</a></sup> which underlines a mechanism for plant defence against herbivores or pathogens.<sup><a reveal-id="DSU024C69" data-open="DSU024C69" class="link link-ref link-reveal xref-bibr">69</a></sup> Zinc can be taken up and compartmented by specific transporters<sup><a reveal-id="DSU024C70" data-open="DSU024C70" class="link link-ref link-reveal xref-bibr">70</a></sup> and clearly had a distinctive effect on the specific group of indolyl GS in <em>Thlaspi caerulescens</em>. Within both roots and shoots, the levels of these compounds were drastically reduced by zinc.<sup><a reveal-id="DSU024C71" data-open="DSU024C71" class="link link-ref link-reveal xref-bibr">71</a></sup> In <em>B. rapa</em>, higher zinc concentration in hydroponic solution markedly decreased the accumulation of aliphatic GS but increased the indole and aromatic GS in shoots.<sup><a reveal-id="DSU024C72" data-open="DSU024C72" class="link link-ref link-reveal xref-bibr">72</a></sup> Thus, the ortholog of AT5G14910 in <em>B. napus</em> (i.e. JCVI_9761) seems to be a potential regulator of the GS biosynthesis although it still needs to be fully elucidated in future.</p>                            <h3 scrollto-destination=8721421 id="8721421" class="section-title" data-legacy-id=s4d>4.4. Sequence variation of GTR2 and HAG3, and the development of markers for breeding selection</h3>
<p>mRNA-Seq is a powerful tool to develop SNPs but has some limitations for detecting more extensive sequence variation. It is therefore necessary to confirm or detect new causative polymorphisms by re-sequencing PCR fragments amplified from genomic DNA for two main reasons. Firstly, during the discovery of SNPs by aligning 80-bp mRNA-Seq reads to the reference sequences, very restrictive criteria were empirically applied (allowing only 1-bp mismatch) to avoid false discovery incurred from sequence errors,<sup><a reveal-id="DSU024C55" data-open="DSU024C55" class="link link-ref link-reveal xref-bibr">55</a></sup> but this process also removes all sequence variation ≥2-bp. Secondly, sequence variation at noncoding regions within a gene such as promoter, terminator, or intron cannot be detected by mRNA-Seq.</p><p>Previously, orthologs of <em>HAG1</em> on A9 and C2 have been identified as key regulators for GS synthesis in rapeseed.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In this study, another two genes, <em>BnaA.GTR2a</em> and <em>BnaC.HAG3b</em> within the respective peaks on A2 and C9, were of particular interest in that they can jointly explain 25.8% of trait variation by regression analysis. Unfortunately, no causative SNPs were found in the mRNA-Seq data for either <em>BnaA.GTR2a</em> or <em>BnaC.HAG3b</em>. Therefore, specific primers were designed and used to amplify the corresponding genomic regions. Re-sequencing of the PCR products failed to detect any causative polymorphism within JCVI_13343 locus, which only covers 21% of <em>GTR2</em> mRNA (2.8 kb) in <em>Arabidopsis</em>. In future, sequencing the whole length of <em>GTR2</em> in <em>B. napus</em> is needed to identify more sequence variations responsible for GS, because InDels may also exist in the promoter, terminator, or intron regions.<sup><a reveal-id="DSU024C42" data-open="DSU024C42" class="link link-ref link-reveal xref-bibr">42</a></sup> An alternative explanation of the lack of causative polymorphisms in JCVI_13343 is that it is <em>trans-</em>regulated, i.e. its expression level (represented as the RPKM value) is modulated by other gene(s), not by its own sequence variation.</p><p>As for unigene EX043693 (<em>BnaC.HAG3b</em>), the 3-bp insertion and 11-bp deletion are likely to be important for its function. In fact, insertion and deletion in the genome are very common in crops and are an important mechanism underlying trait variation. For example, two copies of <em>HAG1</em> (<em>BnaC.HAG1a</em> and <em>BnaA.HAG1c</em>) were shown to have been deleted from both C2 and A9 in low GS <em>B. napus</em> lines.<sup><a reveal-id="DSU024C52" data-open="DSU024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In maize, a 117-bp insertion in the promoter and a 35-bp deletion in the intron of <em>ZmVTE4</em> resulted in a significantly higher level of tocopherol content.<sup><a reveal-id="DSU024C42" data-open="DSU024C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p><p>Finally, we have successfully developed a PCR-based marker to detect the 11-bp deletions in <em>BnaC.HAG3b</em>. The genetic effect of this PCR marker on GS has been verified in the diversity panel and thus can be used in germplasm screening and breeding selection of low GS lines.</p>                            <h2 scrollto-destination=8721426 id="8721426" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=8721428 id="8721428" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by UK Biotechnology and Biological Sciences Research Council (BBSRC BB/H004351/1 (IBTI Club), BB/E017363/1, ERAPG08.008) and UK Department for Environment, Food and Rural Affairs (Defra IF0144) as well as the National Natural Science Foundation of China (31371663). Funding to pay the Open Access publication charges for this article was provided by Research Councils UK.</p>                            <h2 scrollto-destination=8721430 id="8721430" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank The Genome Analysis Centre (TGAC) for generating Illumina sequence data. We would also like to thank Rod Snowdon, Jackie Barker, and Graham Teakle for providing germplasm and Debra Manning and Margaret Turner of KWS-UK and Peter Tillmann of VDLUFA Qualitatssicherung NIRS/NIT, Kassel, Germany for their assistance with NIRS measurements.</p>                            <h2 scrollto-destination=8721432 id="8721432" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU024C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fahey</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Zalcmann</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Talalay</div> <div class="given-names">P.</div></div>. </div><div class="article-title">The chemical diversity and distribution of glucosinolates and isothiocyanates among plants</div>, <div class="source">Phytochemistry</div> , <div class="year">2001</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">5</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chemical%20diversity%20and%20distribution%20of%20glucosinolates%20and%20isothiocyanates%20among%20plants&amp;author=J.W.%20Fahey&amp;author=A.T.%20Zalcmann&amp;author=P.%20Talalay&amp;publication_year=2001&amp;journal=Phytochemistry&amp;volume=56&amp;pages=5-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0031-9422(00)00316-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0031-9422(00)00316-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0031-9422(00)00316-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11198818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wittstock</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Glucosinolate research in the Arabidopsis era</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">263</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glucosinolate%20research%20in%20the%20Arabidopsis%20era&amp;author=U.%20Wittstock&amp;author=B.A.%20Halkier&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci.&amp;volume=7&amp;pages=263-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02273-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02273-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02273-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12049923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mithen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Glucosinolates—biochemistry, genetics and biological activity</div>, <div class="source">Plant Growth Regul.</div> , <div class="year">2001</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">91</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glucosinolates%E2%80%94biochemistry%2C%20genetics%20and%20biological%20activity&amp;author=R.%20Mithen&amp;publication_year=2001&amp;journal=Plant%20Growth%20Regul.&amp;volume=34&amp;pages=91-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1013330819778" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1013330819778" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1013330819778"> </span></div> </div></div></div></div></div><div content-id="DSU024C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toroser</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Thormann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mithen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">RFLP mapping of quantitative trait loci controlling seed aliphatic glucosinolate content in oilseed rape (<em>Brassica napus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">802</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP%20mapping%20of%20quantitative%20trait%20loci%20controlling%20seed%20aliphatic%20glucosinolate%20content%20in%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=D.%20Toroser&amp;author=C.%20Thormann&amp;author=T.%20Osborn&amp;author=R.%20Mithen&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=802-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00220963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00220963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00220963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mithen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Leaf glucosinolate profiles and their relationship to pest and disease resistance in oilseed rape</div>, <div class="source">Euphytica</div> , <div class="year">1992</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">71</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Leaf%20glucosinolate%20profiles%20and%20their%20relationship%20to%20pest%20and%20disease%20resistance%20in%20oilseed%20rape&amp;author=R.%20Mithen&amp;publication_year=1992&amp;journal=Euphytica&amp;volume=63&amp;pages=71-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00023913" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00023913" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00023913"> </span></div> </div></div></div></div></div><div content-id="DSU024C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kiddle</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Bennet</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Doughty</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wallsgrove</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Variation in the glucosinolate content of vegetative tissues of Chinese lines of <em>Brassica napus</em> L.</div>, <div class="source"><em>Ann. Appl. Biol</em>.</div> , <div class="year">1999</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">131</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20glucosinolate%20content%20of%20vegetative%20tissues%20of%20Chinese%20lines%20of%20Brassica%20napus%20L.&amp;author=Y.%20Li&amp;author=G.%20Kiddle&amp;author=R.%20Bennet&amp;author=K.%20Doughty&amp;author=R.%20Wallsgrove&amp;publication_year=1999&amp;journal=Ann.%20Appl.%20Biol.&amp;volume=134&amp;pages=131-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/aab.1999.134.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Faab.1999.134.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Faab.1999.134.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSU024C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blau</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Feeny</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Contardo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Robson</div> <div class="given-names">D.S.</div></div>. </div><div class="article-title">Allylglucosinolate and herbivorous caterpillars: a contrast in toxicity and tolerance</div>, <div class="source">Science</div> , <div class="year">1978</div>, vol. <div class="volume">200</div> (pg. <div class="fpage">1296</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allylglucosinolate%20and%20herbivorous%20caterpillars%3A%20a%20contrast%20in%20toxicity%20and%20tolerance&amp;author=P.A.%20Blau&amp;author=P.%20Feeny&amp;author=L.%20Contardo&amp;author=D.S.%20Robson&amp;publication_year=1978&amp;journal=Science&amp;volume=200&amp;pages=1296-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.200.4347.1296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.200.4347.1296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.200.4347.1296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17738725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kliebenstein</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Pedersen</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Comparative analysis of insect resistance QTL and QTL controlling the myrosinase/glucosinolate system in Arabidopsis thaliana</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">325</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20insect%20resistance%20QTL%20and%20QTL%20controlling%20the%20myrosinase%2Fglucosinolate%20system%20in%20Arabidopsis%20thaliana&amp;author=D.J.%20Kliebenstein&amp;author=D.%20Pedersen&amp;author=T.%20Mitchell-Olds&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=325-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12019246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">Z.X.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">W.X.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.L.</div></div>, et al. </div><div class="article-title">The association of <em>Sclerotinia sclerotiorum</em> resistance with glucosinolates in <em>Brassica napus</em> double-low DH population</div>, <div class="source">J. Plant Pathol.</div> , <div class="year">2008</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">43</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSU024C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The biosynthesis of glucosinolates</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">1997</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">425</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biosynthesis%20of%20glucosinolates&amp;author=B.A.%20Halkier&amp;author=L.%20Du&amp;publication_year=1997&amp;journal=Trends%20Plant%20Sci.&amp;volume=2&amp;pages=425-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(97)90026-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(97)90026-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(97)90026-1"> </span></div> </div></div></div></div></div><div content-id="DSU024C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Gershenzon</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Biology and biochemistry of glucosinolates</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">303</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20and%20biochemistry%20of%20glucosinolates&amp;author=B.A.%20Halkier&amp;author=J.%20Gershenzon&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=57&amp;pages=303-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hull</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Vij</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Celenza</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Arabidopsis cytochrome P450 s that catalyze the first step of tryptophan-dependent indole-3-acetic acid biosynthesis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">2379</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20cytochrome%20P450%20s%20that%20catalyze%20the%20first%20step%20of%20tryptophan-dependent%20indole-3-acetic%20acid%20biosynthesis&amp;author=A.K.%20Hull&amp;author=R.%20Vij&amp;author=J.L.%20Celenza&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=2379-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.040569997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.040569997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.040569997"> </span></div> </div></div></div></div></div><div content-id="DSU024C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Quiros</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">Genetic analysis, expression and molecular characterization of BoGSL-ELONG, a major gene involved in the aliphatic glucosinolate pathway of <em>Brassica</em> species</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">1937</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%2C%20expression%20and%20molecular%20characterization%20of%20BoGSL-ELONG%2C%20a%20major%20gene%20involved%20in%20the%20aliphatic%20glucosinolate%20pathway%20of%20Brassica%20species&amp;author=G.%20Li&amp;author=C.F.%20Quiros&amp;publication_year=2002&amp;journal=Genetics&amp;volume=162&amp;pages=1937-43" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12524361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Quiros</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">In planta side-chain glucosinolate modification in Arabidopsis by introduction of dioxygenase <em>Brassica</em> homolog <em>BoGSL-ALK</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1116</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20planta%20side-chain%20glucosinolate%20modification%20in%20Arabidopsis%20by%20introduction%20of%20dioxygenase%20Brassica%20homolog%20BoGSL-ALK&amp;author=G.%20Li&amp;author=C.F.%20Quiros&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=1116-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Glawishnig</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jorgensen</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title"><em>CYP79F1</em> and <em>CYP79F2</em> have distinct functions in the biosynthesis of aliphatic glucosinolates in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">923</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CYP79F1%20and%20CYP79F2%20have%20distinct%20functions%20in%20the%20biosynthesis%20of%20aliphatic%20glucosinolates%20in%20Arabidopsis&amp;author=S.%20Chen&amp;author=E.%20Glawishnig&amp;author=K.%20Jorgensen&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=33&amp;pages=923-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01679.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01679.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01679.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12609033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grubb</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Zipp</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Ludwig-Müller</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Masuno</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Molinski</div> <div class="given-names">T.F.</div></div>,  <div class="name"><div class="surname">Abel</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Arabidopsis glucosyltransferase <em>UGT74B1</em> functions in glucosinolate biosynthesis and auxin homeostasis</div>, <div class="source">Plant J.</div> , <div class="year">2004</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">893</div>-<div class="lpage">908</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20glucosyltransferase%20UGT74B1%20functions%20in%20glucosinolate%20biosynthesis%20and%20auxin%20homeostasis&amp;author=C.D.%20Grubb&amp;author=B.J.%20Zipp&amp;author=J.%20Ludwig-M%C3%BCller&amp;author=M.N.%20Masuno&amp;author=T.F.%20Molinski&amp;author=S.%20Abel&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=40&amp;pages=893-908" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.40.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.40.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.40.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15584955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirai</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Sugiyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">6478</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omics-based%20identification%20of%20Arabidopsis%20Myb%20transcription%20factors%20regulating%20aliphatic%20glucosinolate%20biosynthesis&amp;author=M.Y.%20Hirai&amp;author=K.%20Sugiyama&amp;author=Y.%20Sawada&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=6478-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611629104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611629104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611629104"> </span></div> </div></div></div></div></div><div content-id="DSU024C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">He</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mawhinney</div> <div class="given-names">T.P.</div></div>,  <div class="name"><div class="surname">Preuss</div> <div class="given-names">M.L.</div></div>, et al. </div><div class="article-title">A redox active isopropylmalate dehydrogenase functions in the biosynthesis of glucosinolates and leucine in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">679</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20redox%20active%20isopropylmalate%20dehydrogenase%20functions%20in%20the%20biosynthesis%20of%20glucosinolates%20and%20leucine%20in%20Arabidopsis&amp;author=Y.%20He&amp;author=T.P.%20Mawhinney&amp;author=M.L.%20Preuss&amp;publication_year=2009&amp;journal=Plant%20J.&amp;volume=60&amp;pages=679-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.60.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.60.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.60.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19674406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kuwahara</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nagano</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Omics-based approaches to methionine side chain elongation in Arabidopsis: characterization of the genes encoding methylthioalkylmalate isomerase and methylthioalkylmalate dehydrogenase</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">1181</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omics-based%20approaches%20to%20methionine%20side%20chain%20elongation%20in%20Arabidopsis%3A%20characterization%20of%20the%20genes%20encoding%20methylthioalkylmalate%20isomerase%20and%20methylthioalkylmalate%20dehydrogenase&amp;author=Y.%20Sawada&amp;author=A.%20Kuwahara&amp;author=M.%20Nagano&amp;publication_year=2009&amp;journal=Plant%20Cell%20Physiol.&amp;volume=50&amp;pages=1181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcp079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcp079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcp079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19493961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Bioinformatic analysis of molecular network of glucosinolate biosynthesis</div>, <div class="source">Comput. Biol. Chem.</div> , <div class="year">2011</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">10</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20analysis%20of%20molecular%20network%20of%20glucosinolate%20biosynthesis&amp;author=Y.%20Chen&amp;author=X.%20Yan&amp;author=S.%20Chen&amp;publication_year=2011&amp;journal=Comput.%20Biol.%20Chem.&amp;volume=35&amp;pages=10-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.compbiolchem.2010.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.compbiolchem.2010.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.compbiolchem.2010.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21247808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">He</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Galant</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">Structural and functional evolution of isopropylmalate dehydrogenases in leucine and glucosinolate pathways of Arabidopsis thaliana</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2011</div>, vol. <div class="volume">286</div> (pg. <div class="fpage">28794</div>-<div class="lpage">801</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20evolution%20of%20isopropylmalate%20dehydrogenases%20in%20leucine%20and%20glucosinolate%20pathways%20of%20Arabidopsis%20thaliana&amp;author=Y.%20He&amp;author=A.%20Galant&amp;author=Q.%20Pang&amp;publication_year=2011&amp;journal=J.%20Biol.%20Chem.&amp;volume=286&amp;pages=28794-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M111.262519" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M111.262519" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M111.262519"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21697089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">He</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Functional characterization of Arabidopsis isopropylmalate dehydrogenases reveals their important roles in gametophyte development</div>, <div class="source">New Phytol.</div> , <div class="year">2011</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">160</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20Arabidopsis%20isopropylmalate%20dehydrogenases%20reveals%20their%20important%20roles%20in%20gametophyte%20development&amp;author=Y.%20He&amp;author=L.%20Chen&amp;author=Y.%20Zhou&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=189&amp;pages=160-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03460.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03460.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03460.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20840499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Novel insights into the function of Arabidopsis R2R3-MYB transcription factors regulating aliphatic glucosinolate biosynthesis</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2013</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">1335</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20insights%20into%20the%20function%20of%20Arabidopsis%20R2R3-MYB%20transcription%20factors%20regulating%20aliphatic%20glucosinolate%20biosynthesis&amp;author=Y.%20Li&amp;author=Y.%20Sawada&amp;author=A.%20Hirai&amp;publication_year=2013&amp;journal=Plant%20Cell%20Physiol.&amp;volume=54&amp;pages=1335-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pct085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpct085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpct085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23792303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olson-Manning</div> <div class="given-names">C.F.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Rausher</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Evolution of flux control in the glucosinolate pathway in <em>Arabidopsis thaliana</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2013</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">14</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20flux%20control%20in%20the%20glucosinolate%20pathway%20in%20Arabidopsis%20thaliana&amp;author=C.F.%20Olson-Manning&amp;author=C.R.%20Lee&amp;author=M.D.%20Rausher&amp;author=T.%20Mitchell-Olds&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=14-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grubb</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Abel</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Glucosinolate metabolism and its control</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">89</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glucosinolate%20metabolism%20and%20its%20control&amp;author=C.D.%20Grubb&amp;author=S.%20Abel&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=89-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2005.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2005.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2005.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16406306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sønderby</div> <div class="given-names">I.E.</div></div>,  <div class="name"><div class="surname">Geu-Flores</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Biosynthesis of glucosinolates-gene discovery and beyond</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">283</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20glucosinolates-gene%20discovery%20and%20beyond&amp;author=I.E.%20S%C3%B8nderby&amp;author=F.%20Geu-Flores&amp;author=B.A.%20Halkier&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci.&amp;volume=15&amp;pages=283-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.02.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20303821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirani</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zelmer</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">McVetty</div> <div class="given-names">P.B.E.</div></div>,  <div class="name"><div class="surname">Asif</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goyal</div> <div class="given-names">A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Goyal</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Molecular genetics of glucosinolate biosynthesis in Brassicas: genetic manipulation and application aspect.</div>, <div class="source">Crop Plant</div> , <div class="year">2012</div><div class="publisher-name">InTech: Croatia</div> <div class="comment">ISBN: 978-953-51-0527-5 pp. 189–216</div></div></div></div></div><div content-id="DSU024C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Biosynthesis of glucosinolates in the developing silique walls</div>, <div class="source">Phytochemistry</div> , <div class="year">1998</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1145</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20glucosinolates%20in%20the%20developing%20silique%20walls&amp;author=L.%20Du&amp;author=B.A.%20Halkier&amp;publication_year=1998&amp;journal=Phytochemistry&amp;volume=48&amp;pages=1145-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0031-9422(97)00990-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0031-9422(97)00990-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0031-9422(97)00990-4"> </span></div> </div></div></div></div></div><div content-id="DSU024C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Characterization of glucosinolate uptake by leaf protoplasts of <em>Brassica napus</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2000</div>, vol. <div class="volume">275</div> (pg. <div class="fpage">22955</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20glucosinolate%20uptake%20by%20leaf%20protoplasts%20of%20Brassica%20napus&amp;author=S.%20Chen&amp;author=B.A.%20Halkier&amp;publication_year=2000&amp;journal=J.%20Biol.%20Chem.&amp;volume=275&amp;pages=22955-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M002768200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M002768200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M002768200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10816580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Schulz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">Long-distance phloem transport of glucosinolates in Arabidopsis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">194</div>-<div class="lpage">201</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-distance%20phloem%20transport%20of%20glucosinolates%20in%20Arabidopsis&amp;author=S.%20Chen&amp;author=B.L.%20Petersen&amp;author=C.E.%20Olsen&amp;author=A.%20Schulz&amp;author=B.A.%20Halkier&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=127&amp;pages=194-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.127.1.194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.127.1.194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.127.1.194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11553747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Andereason</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Update on glucosinolate metabolism and transport</div>, <div class="source">Plant Physiol. Biochem.</div> , <div class="year">2001</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">743</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Update%20on%20glucosinolate%20metabolism%20and%20transport&amp;author=S.%20Chen&amp;author=E.%20Andereason&amp;publication_year=2001&amp;journal=Plant%20Physiol.%20Biochem.&amp;volume=39&amp;pages=743-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0981-9428(01)01301-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0981-9428(01)01301-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0981-9428(01)01301-8"> </span></div> </div></div></div></div></div><div content-id="DSU024C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kliebenstein</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Kroymann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Genetic control of natural variation in Arabidopsis glucosinolate accumulation</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">811</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20natural%20variation%20in%20Arabidopsis%20glucosinolate%20accumulation&amp;author=D.J.%20Kliebenstein&amp;author=J.%20Kroymann&amp;author=P.%20Brown&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=126&amp;pages=811-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.126.2.811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.126.2.811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.126.2.811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11402209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nour-Eldin</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Andersen</div> <div class="given-names">T.G.</div></div>,  <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">NRT/PTR transporters are essential for translocation of glucosinolate defence compounds to seeds</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">488</div> (pg. <div class="fpage">531</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NRT%2FPTR%20transporters%20are%20essential%20for%20translocation%20of%20glucosinolate%20defence%20compounds%20to%20seeds&amp;author=H.H.%20Nour-Eldin&amp;author=T.G.%20Andersen&amp;author=M.%20Burow&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=531-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koornneef</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Alonso-Blanco</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Vreugdenhil</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Naturally occurring genetic variation in Arabidopsis thaliana</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">141</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Naturally%20occurring%20genetic%20variation%20in%20Arabidopsis%20thaliana&amp;author=M.%20Koornneef&amp;author=C.%20Alonso-Blanco&amp;author=D.%20Vreugdenhil&amp;publication_year=2004&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=55&amp;pages=141-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.55.031903.141605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.55.031903.141605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.55.031903.141605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15377217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uzunova</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weissleder</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Robbelen</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Mapping the genome of rapeseed (<em>Brassica napus</em> L.).1. Construction of an RFLP linkage map and localization of QTLs for seed glucosinolate content</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">194</div>-<div class="lpage">204</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20the%20genome%20of%20rapeseed%20(Brassica%20napus%20L.).1.%20Construction%20of%20an%20RFLP%20linkage%20map%20and%20localization%20of%20QTLs%20for%20seed%20glucosinolate%20content&amp;author=M.%20Uzunova&amp;author=W.%20Ecke&amp;author=K.%20Weissleder&amp;author=G.%20Robbelen&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=90&amp;pages=194-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24173891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Howell</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Homoeologue loci control the accumulation of seed glucosinolates in oilseed rape (<em>Brassica napus</em>)</div>, <div class="source">Genome</div> , <div class="year">2003</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">454</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homoeologue%20loci%20control%20the%20accumulation%20of%20seed%20glucosinolates%20in%20oilseed%20rape%20(Brassica%20napus)&amp;author=P.M.%20Howell&amp;author=A.G.%20Sharpe&amp;author=D.J.%20Lydiate&amp;publication_year=2003&amp;journal=Genome&amp;volume=46&amp;pages=454-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12834062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Detection of loci controlling seed glucosinolate content and their association with Sclerotinia resistance in <em>Brassica napus</em></div>, <div class="source">Plant Breed.</div> , <div class="year">2003</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">19</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20loci%20controlling%20seed%20glucosinolate%20content%20and%20their%20association%20with%20Sclerotinia%20resistance%20in%20Brassica%20napus&amp;author=J.%20Zhao&amp;author=J.%20Meng&amp;publication_year=2003&amp;journal=Plant%20Breed.&amp;volume=122&amp;pages=19-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.2003.00784.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.2003.00784.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.2003.00784.x"> </span></div> </div></div></div></div></div><div content-id="DSU024C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quijada</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (<em>Brassica napus</em> L.): 1. Identification of genomic regions from winter germplasm</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">549</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20analysis%20of%20seed%20yield%20and%20other%20complex%20traits%20in%20hybrid%20spring%20rapeseed%20(Brassica%20napus%20L.)%3A%201.%20Identification%20of%20genomic%20regions%20from%20winter%20germplasm&amp;author=P.A.%20Quijada&amp;author=J.A.%20Udall&amp;author=B.%20Lambert&amp;author=T.C.%20Osborn&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=549-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0323-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0323-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0323-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16767447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Characterization of metabolite quantitative trait loci and metabolic networks that control glucosinolate concentration in the seeds and leaves of <em>Brassica napus</em></div>, <div class="source">New Phytol.</div> , <div class="year">2011</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">96</div>-<div class="lpage">108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20metabolite%20quantitative%20trait%20loci%20and%20metabolic%20networks%20that%20control%20glucosinolate%20concentration%20in%20the%20seeds%20and%20leaves%20of%20Brassica%20napus&amp;author=J.%20Feng&amp;author=Y.%20Long&amp;author=L.%20Shi&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=193&amp;pages=96-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2011.03890.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03890.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2011.03890.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21973035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gore</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Status and prospects of association mapping in plants</div>, <div class="source">Plant Genome</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">5</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Status%20and%20prospects%20of%20association%20mapping%20in%20plants&amp;author=C.%20Zhu&amp;author=M.%20Gore&amp;author=E.%20Buckler&amp;author=J.%20Yu&amp;publication_year=2008&amp;journal=Plant%20Genome&amp;volume=1&amp;pages=5-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2008.02.0089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2008.02.0089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2008.02.0089"> </span></div> </div></div></div></div></div><div content-id="DSU024C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Warburton</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Association mapping for enhancing maize (<em>Zea mays</em> L.) genetic improvement</div>, <div class="source">Crop Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">433</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20for%20enhancing%20maize%20(Zea%20mays%20L.)%20genetic%20improvement&amp;author=J.%20Yan&amp;author=M.L.%20Warburton&amp;author=J.%20Crouch&amp;publication_year=2011&amp;journal=Crop%20Sci.&amp;volume=51&amp;pages=433-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2010.04.0233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2010.04.0233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2010.04.0233"> </span></div> </div></div></div></div></div><div content-id="DSU024C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome-wide association studies identified three independent polymorphisms associated with α-Tocopherol content in maize kernels</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e36807</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20identified%20three%20independent%20polymorphisms%20associated%20with%20%CE%B1-Tocopherol%20content%20in%20maize%20kernels&amp;author=Q.%20Li&amp;author=X.%20Yang&amp;author=S.%20Xu&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e36807" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0036807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0036807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0036807"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22615816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kandianis</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Harjes</div> <div class="given-names">C.E.</div></div>, et al. </div><div class="article-title">Rare genetic variation at <em>Zea mays</em> crtRB1 increases β-carotene in maize grain</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">322</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rare%20genetic%20variation%20at%20Zea%20mays%20crtRB1%20increases%20%CE%B2-carotene%20in%20maize%20grain&amp;author=J.%20Yan&amp;author=C.%20Kandianis&amp;author=C.E.%20Harjes&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=322-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.551" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20305664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.H.</div></div>, et al. </div><div class="article-title">Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels</div>, <div class="source">Nat. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">43</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20dissects%20the%20genetic%20architecture%20of%20oil%20biosynthesis%20in%20maize%20kernels&amp;author=H.%20Li&amp;author=Z.Y.%20Peng&amp;author=X.H.%20Yang&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=43-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23242369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">961</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">32</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20flowering%20time%20and%20grain%20yield%20traits%20in%20a%20worldwide%20collection%20of%20rice%20germplasm&amp;author=X.%20Huang&amp;author=Y.%20Zhao&amp;author=X.%20Wei&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=32-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22138690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Clemens</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Honsdorf</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.C.</div></div>. </div><div class="article-title">Extent and structure of linkage disequilibrium in canola quality winter rapeseed (<em>Brassica napus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">921</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extent%20and%20structure%20of%20linkage%20disequilibrium%20in%20canola%20quality%20winter%20rapeseed%20(Brassica%20napus%20L.)&amp;author=W.%20Ecke&amp;author=R.%20Clemens&amp;author=N.%20Honsdorf&amp;author=H.C.%20Becker&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=921-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1221-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1221-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1221-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19953219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Friedt</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Pons-Kuhnemann</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (<em>Brassica napus</em> ssp. napus)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">1035</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20gene-linked%20SSR%20markers%20to%20seed%20glucosinolate%20content%20in%20oilseed%20rape%20(Brassica%20napus%20ssp.%20napus)&amp;author=M.%20Hasan&amp;author=W.%20Friedt&amp;author=J.%20Pons-Kuhnemann&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=1035-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0733-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0733-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0733-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18322671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">B.Y.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genome-wide association study dissects the genetic architecture of seed weight and seed quality in rapeseed (<em>Brassica napus</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2014</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">355</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20dissects%20the%20genetic%20architecture%20of%20seed%20weight%20and%20seed%20quality%20in%20rapeseed%20(Brassica%20napus%20L.)&amp;author=F.%20Li&amp;author=B.Y.%20Chen&amp;author=K.%20Xu&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24510440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Barbacioru</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">mRNA-Seq whole-transcriptome analysis of a single cell</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">377</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNA-Seq%20whole-transcriptome%20analysis%20of%20a%20single%20cell&amp;author=F.%20Tang&amp;author=C.%20Barbacioru&amp;author=Y.%20Wang&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=377-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19349980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Genome dissection in the polyploid crop oilseed rape by transcriptome sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">762</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20dissection%20in%20the%20polyploid%20crop%20oilseed%20rape%20by%20transcriptome%20sequencing&amp;author=I.%20Bancroft&amp;author=C.%20Morgan&amp;author=F.%20Fraser&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=762-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21804563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harper</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Associative transcriptomics of traits in the polyploid crop species <em>Brassica napus</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">798</div>-<div class="lpage">802</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associative%20transcriptomics%20of%20traits%20in%20the%20polyploid%20crop%20species%20Brassica%20napus&amp;author=A.L.%20Harper&amp;author=M.%20Trick&amp;author=J.%20Higgins&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=798-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22820317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smooker</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Wells</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">The identification and mapping of candidate genes and QTL involved in the fatty acid desaturation pathway in <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">1075</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20identification%20and%20mapping%20of%20candidate%20genes%20and%20QTL%20involved%20in%20the%20fatty%20acid%20desaturation%20pathway%20in%20Brassica%20napus&amp;author=A.M.%20Smooker&amp;author=R.%20Wells&amp;author=C.%20Morgan&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1075-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1512-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1512-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1512-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21184048" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Magusin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Use of mRNA-Seq to discriminate contributions to the transcriptome from the constituent genomes of the polyploid crop species <em>Brassica napus</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">247</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20mRNA-Seq%20to%20discriminate%20contributions%20to%20the%20transcriptome%20from%20the%20constituent%20genomes%20of%20the%20polyploid%20crop%20species%20Brassica%20napus&amp;author=J.%20Higgins&amp;author=A.%20Magusin&amp;author=M.%20Trick&amp;author=F.%20Fraser&amp;author=I.%20Bancroft&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=247-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Single nucleotide polymorphism (SNP) discovery in the polyploid <em>Brassica napus</em> using Solexa transcriptome sequencing</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">334</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20(SNP)%20discovery%20in%20the%20polyploid%20Brassica%20napus%20using%20Solexa%20transcriptome%20sequencing&amp;author=M.%20Trick&amp;author=Y.%20Long&amp;author=J.%20Meng&amp;author=I.%20Bancroft&amp;publication_year=2009&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=7&amp;pages=334-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2009.7.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2009.7.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2009.7.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19207216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">945</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evanno</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2611</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol.&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W.H.</div></div>, et al. </div><div class="article-title">A unified mixed model method for association mapping that accounts for multiple levels of relatedness</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">203</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed%20model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langfelder</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Horvath</div> <div class="given-names">S.</div></div>. </div><div class="article-title">WGCNA: an R package for weighted correlation network analysis</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">559</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WGCNA%3A%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&amp;author=P.%20Langfelder&amp;author=S.%20Horvath&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=559-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-559" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-559"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19114008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shannon</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Markiel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ozier</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2498</div>-<div class="lpage">504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoscape%3A%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P.%20Shannon&amp;author=A.%20Markiel&amp;author=O.%20Ozier&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2498-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1239303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1239303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1239303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ling</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">agriGO: a GO analysis toolkit for the agricultural community</div>, <div class="source">Nucleic. Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">64</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=agriGO%3A%20a%20GO%20analysis%20toolkit%20for%20the%20agricultural%20community&amp;author=Z.%20Du&amp;author=X.%20Zhou&amp;author=Y.%20Ling&amp;author=Z.H.%20Zhang&amp;author=Z.%20Su&amp;publication_year=2010&amp;journal=Nucleic.%20Acids%20Res.&amp;volume=38&amp;pages=64-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq310"> </span></div> </div></div></div></div></div><div content-id="DSU024C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kroymann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Textor</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tokuhisa</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Falk</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Bartram</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A gene controlling variation in Arabidopsis glucosinolate composition is part of the methionine chain elongation pathway</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">1077</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene%20controlling%20variation%20in%20Arabidopsis%20glucosinolate%20composition%20is%20part%20of%20the%20methionine%20chain%20elongation%20pathway&amp;author=J.%20Kroymann&amp;author=S.%20Textor&amp;author=J.G.%20Tokuhisa&amp;author=K.L.%20Falk&amp;author=S.%20Bartram&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=127&amp;pages=1077-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.010416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.010416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.010416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11706188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gigolashvili</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yatusevich</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rollwitz</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">The plastidic bile acid transporter 5 is required for the biosynthesis of methionine-derived glucosinolates in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell</div> , <div class="year">2009</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1813</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plastidic%20bile%20acid%20transporter%205%20is%20required%20for%20the%20biosynthesis%20of%20methionine-derived%20glucosinolates%20in%20Arabidopsis%20thaliana&amp;author=T.%20Gigolashvili&amp;author=R.%20Yatusevich&amp;author=I.%20Rollwitz&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=1813-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.066399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.066399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.066399"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19542295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knill</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Schuster</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Reichelt</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gershenzon</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Binder</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Arabidopsis branched-chain aminotransferase 3 functions in both amino acid and glucosinolate biosynthesis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">1028</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20branched-chain%20aminotransferase%203%20functions%20in%20both%20amino%20acid%20and%20glucosinolate%20biosynthesis&amp;author=T.%20Knill&amp;author=J.%20Schuster&amp;author=M.%20Reichelt&amp;author=J.%20Gershenzon&amp;author=S.%20Binder&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=146&amp;pages=1028-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.111609" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.111609" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.111609"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18162591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sønderby</div> <div class="given-names">I.E.</div></div>,  <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rowe</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Kliebenstein</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div>. </div><div class="article-title">A complex interplay of three R2R3 MYB transcription factors determines the profile of aliphatic glucosinolates in Arabidopsis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2010</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">348</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complex%20interplay%20of%20three%20R2R3%20MYB%20transcription%20factors%20determines%20the%20profile%20of%20aliphatic%20glucosinolates%20in%20Arabidopsis&amp;author=I.E.%20S%C3%B8nderby&amp;author=M.%20Burow&amp;author=H.C.%20Rowe&amp;author=D.J.%20Kliebenstein&amp;author=B.A.%20Halkier&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=153&amp;pages=348-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.149286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.149286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.149286"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20348214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">U</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Genome analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>, <div class="source">Japan J. Bot.</div> , <div class="year">1935</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="DSU024C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stokes</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">An association transcriptomics approach to the prediction of hybrid performance</div>, <div class="source">Mol. Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">91</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20association%20transcriptomics%20approach%20to%20the%20prediction%20of%20hybrid%20performance&amp;author=D.%20Stokes&amp;author=F.%20Fraser&amp;author=C.%20Morgan&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=26&amp;pages=91-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9379-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9379-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9379-3"> </span></div> </div></div></div></div></div><div content-id="DSU024C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poschenrieder</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tolrà</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Barceló</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Can metals defend plants against biotic stress?</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">288</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Can%20metals%20defend%20plants%20against%20biotic%20stress%3F&amp;author=C.%20Poschenrieder&amp;author=R.%20Tolr%C3%A0&amp;author=J.%20Barcel%C3%B3&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=288-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.04.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.04.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.04.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16697693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poschenrieder</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tolrà</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Barceló</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Interactions between metal ion toxicity and defences against biotic stress: glucosinolates and benzoxazinoids as case studies</div>, <div class="source">For. Snow Landsc. Res.</div> , <div class="year">2006</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">149</div>-<div class="lpage">60</div>)</div></div></div></div><div content-id="DSU024C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eren</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Argüello</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Arabidopsis HMA2, a divalent heavy metal transporting PIB type ATPase, is involved in cytoplasmic Zn<sup>2+</sup> homeostasis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">3712</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20HMA2%2C%20a%20divalent%20heavy%20metal%20transporting%20PIB%20type%20ATPase%2C%20is%20involved%20in%20cytoplasmic%20Zn2%2B%20homeostasis&amp;author=E.%20Eren&amp;author=J.M.%20Arg%C3%BCello&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=136&amp;pages=3712-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.046292" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.046292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.046292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15475410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU024C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tolrà</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Poschenrieder</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Alonso</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Barceló</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Barceló</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Influence of zinc hyperaccumulation on glucosinolates in <em>Thlaspi caerulescens</em></div>, <div class="source">New Phytol.</div> , <div class="year">2001</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">621</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Influence%20of%20zinc%20hyperaccumulation%20on%20glucosinolates%20in%20Thlaspi%20caerulescens&amp;author=R.%20Tolr%C3%A0&amp;author=C.%20Poschenrieder&amp;author=R.%20Alonso&amp;author=D.%20Barcel%C3%B3&amp;author=J.%20Barcel%C3%B3&amp;publication_year=2001&amp;journal=New%20Phytol.&amp;volume=151&amp;pages=621-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.0028-646x.2001.00221.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.0028-646x.2001.00221.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.0028-646x.2001.00221.x"> </span></div> </div></div></div></div></div><div content-id="DSU024C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU024C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coolong</div> <div class="given-names">T.W.</div></div>,  <div class="name"><div class="surname">Randle</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Toler</div> <div class="given-names">H.D.</div></div>. </div><div class="article-title">Zinc availability in hydroponic culture influences glucosinolate concentrations in <em>Brassica rapa</em></div>, <div class="source">Hortscience</div> , <div class="year">2004</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">84</div>-<div class="lpage">6</div>)</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=551332&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu024-dsu024supp_table4.xlsx?Expires=2147483647&amp;Signature=yITZlccoF1QQlPMdI8A-QmOcCK1UWLUx6UT2YZWqf51b~oMf1xhBfkBomHf--urKUDc68D9Wxv1n57w2sRq8zDPOA91WZujnLSOlaCUJxklHttz8Fr~5PvT2D-s-NjftzH5jf2IjEi7RpI9KdFvbjzLuLw2YJUFkWg7rVFjGqg9hhDvzLuNnmKaPtSXJHgCWfm9D5JnMOkk171mBrmGAmAxGmETxuIxDocIPJ-QrvXZeaky7l-fyxmMqYahdU6xVcBS5OB2caKxXrVJ9SOeuL--3QEw~L8MdovW3R7dPxjl~wuqEc-OwK55s4O7Sq0-eOSbdDs~kTnZ8PZxL4RhwoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu024-dsu024supp_table3.xlsx?Expires=2147483647&amp;Signature=2kUr66sjOTguBUZwDiHJctBLFzk7ps3hx42lF9HVaFs1wnkDqxCAqkdJfdyjwJPCw~ABWOX6JSyn1UTFTZKyz5nPmTazqpbTR~b5ITUCzk90wLS6J9uEbFWRNl9PpllG2aEGn9r-IxDBo-w69pYbfxnZGycwTLNkj4u0rtuzh9sQAmn6NQ3pYmrgr6m3i8M9YXikaEh6r8-dnxfvILmRSCN064KYrvM80a7J1oRTIEic-3OKiWry2sipkJEjbCagYuIWVAHG83VGc1NRLSao429ysYpJM6qx1LKpbiIKME2hopidMnVYw1Qro2GOAUya8fv5Mx3VABMVNod1sSytFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu024-dsu024supp_table2.xlsx?Expires=2147483647&amp;Signature=GIUQ9hrJ-NbR~HLFjIw0rdX-IJ9o6UJAkGE-vlfZlqSPeCIN~K8wSJz9NeM4c1tZzs2aZb~N~hrcV9Yptpax7LeKw0kjI-XME4-mmAbiSh8IwA5IcMc3Fvv4yv2vUE~EI5F8ULMYe9W81XIYiTn3dxiZZYMrM3vkoV70CVUBwD6cQ1PF~nJkcfTZlcnIp~HrPKQhwVZaWWFSS3H5JmwHbVnflF~XnuR7tIg3PIHvD~wBKfRe0CGeb~YWZAlTOPDFq~Sl3U5~Eo6zhyRuHYQurzNqpnOgOMCPKigP3ICUA1STn47EUXih5f9RfbQ-wO2kJU9rDD0Iko5S2H3nfG0ymg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu024-dsu024supp_table1.xlsx?Expires=2147483647&amp;Signature=wZppNLxF8trfCC6668dqeMo8Nvnq0GQB5MGm~OJiGKhpAl-QO3pWv25Lx2q0nctFtXXWBvEAMiNT0OwN~T84vBDEsT-I7qd9iMlMlKRg7MnHVGtSX79geq~IKaZxzT8wzLdI3pnvsGf1NbRqE1K5oZDBDJd9LbMVCuCbMfvc~pcQaZygfPFnpOKkvdbG71oxdXl1Q2Wc7wSXcWmHEUYE-FW2PSBD1PULiN1pniL6leUxf7HkAerQRGtzUn1FVIyqqxX4FVSg~Ai~28P6rzCcg~hEFe9TwwI~i9YgaPqWGpxLICqVMcaftRJrqgkRIzQSE68pLeHN7p2G8RyhXes73g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093/dnares/dsu024/2/dsu024-dsu024supp.doc?Expires=2147483647&amp;Signature=wHNoApz1hKmHGRGOSo4nug-AJ133Oh~yXkzpyWsxvvS5Vrom65lg2OyseqBNcFwtLrxd6pHFeFfmG2ifJ1g8bYXDQV-Ir23I7yq-nOtC2~R~RXVin4cNH93IEbqADU3vgzadwoilVrnQzCO2RSa0NhQlz3quc8dsj3CodQWewGbsXYPColx0YLmmgiwJAPH1yhOjte39XvgaF2We431t1~4wxtrozaqGjJ8com1wL83DQY9-OvgG6qt8STwMARw0OrymzmN4SAfjn0UXUOgG~1-O9fWvjZvDlUklQz3LRvh0LD3amY2skXfVRL3VpnJT4J2bxe4Plx~hvsHx1PFEYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">633</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">18</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="551332"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f6%2f613%2f551332">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000347101100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (18)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F6%2F613%2F551332" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/551332'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Associative Transcriptomics Study Dissects the Genetic Architecture of Seed Glucosinolate Content in Brassica napus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f613%2f551332">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

