

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/51/485713","@type":"ScholarlyArticle","name":"Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed ( Brassica napus  L.)","datePublished":"2011-12-22","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr041","keywords":["Diversity Array Technology","genetic diversity","genetic linkage mapping","<italic>Brassica</italic> species","rapeseed"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/51/485713","author":[{"name":"Raman, Harsh","affiliation":"1  EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries) ,  Wagga Wagga Agricultural Institute ,  Wagga Wagga, NSW 2650 ,  Australia","@type":"Person"},{"name":"Raman, Rosy","affiliation":"1  EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries) ,  Wagga Wagga Agricultural Institute ,  Wagga Wagga, NSW 2650 ,  Australia","@type":"Person"},{"name":"Nelson, Matthew N.","affiliation":"2  School of Plant Biology ,  The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia    3  Canola Breeders Western Australia Pty Ltd ,  Locked Bag 888, Como, WA 6952 ,  Australia","@type":"Person"},{"name":"Aslam, M.N.","affiliation":"2  School of Plant Biology ,  The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia","@type":"Person"},{"name":"Rajasekaran, Ravikesavan","affiliation":"2  School of Plant Biology ,  The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia    4  Maize Research Station ,  Tamil Nadu Agricultural University ,  Vagarai 624 613 ,  India","@type":"Person"},{"name":"Wratten, Neil","affiliation":"1  EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries) ,  Wagga Wagga Agricultural Institute ,  Wagga Wagga, NSW 2650 ,  Australia","@type":"Person"},{"name":"Cowling, Wallace A.","affiliation":"3  Canola Breeders Western Australia Pty Ltd ,  Locked Bag 888, Como, WA 6952 ,  Australia    5  The UWA Institute of Agriculture ,  The University of Western Australia ,  35 Stirling Highway, Crawley, WA 6009 ,  Australia","@type":"Person"},{"name":"Kilian, A.","affiliation":"6  Diversity Array Technology Pty Ltd ,  1 Wilf Crane, Yarralumla, ACT 2600 ,  Australia","@type":"Person"},{"name":"Sharpe, Andrew G.","affiliation":"7  Agriculture and AgriFood Canada ,  Saskatoon Research Centre ,  107 Science Place, Saskatoon, SK ,  Canada ,  S7N 0X2","@type":"Person"},{"name":"Schondelmaier, Joerg","affiliation":"8  Saaten-Union Biotec GmbH ,  Hovedisser Str.92, 33818 Leopoldshoehe ,  Germany","@type":"Person"}],"description":"Abstract.  We developed Diversity Array Technology (DArT) markers for application in genetic studies of Brassica napus and other Brassica species with A or C ge","pageStart":"51","pageEnd":"65","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed ( Brassica napus  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed ( Brassica napus  L.)" />
    <meta property="og:description" content="Abstract.  We developed Diversity Array Technology (DArT) markers for application in genetic studies of Brassica napus and other Brassica species with A or C ge" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/51/485713" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-12-22" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Raman, Harsh" /><meta name="citation_author_institution" content="EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries), Wagga Wagga Agricultural Institute, Wagga Wagga, NSW 2650, Australia" /><meta name="citation_author" content="Raman, Rosy" /><meta name="citation_author_institution" content="EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries), Wagga Wagga Agricultural Institute, Wagga Wagga, NSW 2650, Australia" /><meta name="citation_author" content="Nelson, Matthew N." /><meta name="citation_author_institution" content="School of Plant Biology, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author_institution" content="Canola Breeders Western Australia Pty Ltd, Locked Bag 888, Como, WA 6952, Australia" /><meta name="citation_author" content="Aslam, M.N." /><meta name="citation_author_institution" content="School of Plant Biology, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author" content="Rajasekaran, Ravikesavan" /><meta name="citation_author_institution" content="School of Plant Biology, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author_institution" content="Maize Research Station, Tamil Nadu Agricultural University, Vagarai 624 613, India" /><meta name="citation_author" content="Wratten, Neil" /><meta name="citation_author_institution" content="EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries), Wagga Wagga Agricultural Institute, Wagga Wagga, NSW 2650, Australia" /><meta name="citation_author" content="Cowling, Wallace A." /><meta name="citation_author_institution" content="Canola Breeders Western Australia Pty Ltd, Locked Bag 888, Como, WA 6952, Australia" /><meta name="citation_author_institution" content="The UWA Institute of Agriculture, The University of Western Australia, 35 Stirling Highway, Crawley, WA 6009, Australia" /><meta name="citation_author" content="Kilian, A." /><meta name="citation_author_institution" content="Diversity Array Technology Pty Ltd, 1 Wilf Crane, Yarralumla, ACT 2600, Australia" /><meta name="citation_author" content="Sharpe, Andrew G." /><meta name="citation_author_institution" content="Agriculture and AgriFood Canada, Saskatoon Research Centre, 107 Science Place, Saskatoon, SK, Canada, S7N 0X2" /><meta name="citation_author" content="Schondelmaier, Joerg" /><meta name="citation_author_institution" content="Saaten-Union Biotec GmbH, Hovedisser Str.92, 33818 Leopoldshoehe, Germany" /><meta name="citation_title" content="Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed (Brassica napus L.)" /><meta name="citation_firstpage" content="51" /><meta name="citation_lastpage" content="65" /><meta name="citation_doi" content="10.1093/dnares/dsr041" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomic analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilisation; Jpn. J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Characterisation of flowering time and SSR marker analysis of spring and winter type Brassica napus L. germplasm; Euphytica;  citation_year=2007;  citation_volume=153;  citation_pages=43-57; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; Am. J. Hum. Genet.;  citation_year=1980;  citation_volume=32;  citation_pages=314-31; " /><meta name="citation_reference" content="citation_title=Abundant class of human DNA polymorphisms which can be typed using the polymerase chain reaction; Am. J. Hum. Genet.;  citation_year=1989;  citation_volume=44;  citation_pages=388-96; " /><meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-14; " /><meta name="citation_reference" content="citation_title=DNA polymorphisms amplified by arbitrary primers are useful as genetic markers; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6531-5; " /><meta name="citation_reference" content="citation_title=Accessing genetic information with high-density DNA arrays; Science;  citation_year=1996;  citation_volume=274;  citation_pages=610-4; " /><meta name="citation_reference" content="citation_title=Genetic diversity of oilseed Brassica napus germplasm based on restriction fragment length polymorphisms; Theor. Appl. Genet.;  citation_year=1994;  citation_volume=88;  citation_pages=662-8; " /><meta name="citation_reference" content="citation_title=Genetic relationships and fingerprinting of rapeseed cultivars by AFLP: consequences for varietal registration; Crop Sci.;  citation_year=2000;  citation_volume=40;  citation_pages=1417-25; " /><meta name="citation_reference" content="citation_title=Genetic diversity amongst Australian canola cultivars determined by randomly amplified polymorphic DNA; Aust. J. Exp. Agric.;  citation_year=1997;  citation_volume=37;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=Analysis of genetic diversity in the Brassica napus L. gene pool using SSR markers; Genet. Res. Crop Evol.;  citation_year=2006;  citation_volume=53;  citation_pages=793-802; " /><meta name="citation_reference" content="citation_title=Assessment of genetic diversity in Australian canola (Brassica napus L.) cultivars using SSR markers; Crop Pasture Sci.;  citation_year=2009;  citation_volume=60;  citation_pages=1193-201; " /><meta name="citation_reference" content="citation_title=Molecular markers in Brassica oilseed breeding: current status and future possibilities; Plant Breed.;  citation_year=2004;  citation_volume=123;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=The development of multiplex simple sequence repeat (SSR) markers to complement distinctness, uniformity and stability testing of rape (Brassica napus L.) varieties; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=1091-101; " /><meta name="citation_reference" content="citation_title=Genetic diversity of Brassica napus accessions mainly from China detected with RFLP markers; Chin. J. Genet.;  citation_year=1997;  citation_volume=23;  citation_pages=221-32; " /><meta name="citation_reference" content="citation_title=DArT for high-throughput genotyping of Cassava (Manihot esculenta) and its wild relatives; Theor. Appl.Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1092-8; " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=Comparison of identity by descent and identity by state for detecting genetic regions under selection in a sorghum pedigree breeding program; Mol. Breed.;  citation_year=2004;  citation_volume=14;  citation_pages=441-54; " /><meta name="citation_reference" content="citation_title=Mapping As You Go: An effective approach for marker-assisted selection of complex traits; Crop Sci.;  citation_year=2004;  citation_volume=44;  citation_pages=1560-71; " /><meta name="citation_reference" content="citation_title=Genomic selection in plant breeding: from theory to practice; Brief. Funct. Genomics;  citation_year=2010;  citation_volume=9;  citation_pages=166-77; " /><meta name="citation_reference" content="citation_title=Automation in genotyping of single nucleotide polymorphisms; Hum. Mutat.;  citation_year=2001;  citation_volume=17;  citation_pages=475-92; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism (SNP) discovery in the polyploid Brassica napus using Solexa transcriptome sequencing; Plant Biotechnol. J.;  citation_year=2009;  citation_volume=7;  citation_pages=334-46; " /><meta name="citation_reference" content="citation_title=SNP discovery by amplicon sequencing and multiplex SNP genotyping in the allopolyploid species Brassica napus; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=948-56; " /><meta name="citation_reference" content="citation_title=The evolution of Brassica napus FLOWERING LOCUS T paralogues in the context of inverted chromosomal duplication blocks; BMC Evol. Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=271" /><meta name="citation_reference" content="citation_title=Control of flowering time by FLC orthologues in Brassica napus; Plant J.;  citation_year=2001;  citation_volume=28;  citation_pages=545-53; " /><meta name="citation_reference" content="citation_title=Towards unambiguous transcript mapping in the allotetraploid Brassica napus; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=929-38; " /><meta name="citation_reference" content="citation_title=Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing; Nat. Biotech.;  citation_year=2011;  citation_volume=29;  citation_pages=762-6; " /><meta name="citation_reference" content="citation_title=A robust, simple Genotyping-by-Sequencing (GBS) approach for high diversity species; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers; PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=Sequence analysis of the canola genome; citation_publisher=Prague, Czech Republic;  citation_year=2011;  citation_pages=9" /><meta name="citation_reference" content="citation_title=Diversity arrays: a solid state technology for sequence information independent genotyping; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=e25" /><meta name="citation_reference" content="citation_title=Diversity arrays technology (DArT) for whole-genome profiling of barley; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=9915-20; " /><meta name="citation_reference" content="citation_title=An integrated DArT-SSR linkage map of durum wheat; Mol. Breed.;  citation_year=2008;  citation_volume=22;  citation_pages=629-48; " /><meta name="citation_reference" content="citation_title=Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=1409-20; " /><meta name="citation_reference" content="citation_title=Comparison of genetic and cytogenetic maps of hexaploid wheat (Triticum aestivum L.) using SSR and DArT markers; Mol. Genet. Genomics;  citation_year=2009;  citation_volume=281;  citation_pages=181-91; " /><meta name="citation_reference" content="citation_title=DArT markers: diversity analyses and mapping in Sorghum bicolor; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=26" /><meta name="citation_reference" content="citation_title=A high-density consensus map of barley linking DArT markers to SSR, RFLP and STS loci and agricultural traits; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=206" /><meta name="citation_reference" content="citation_title=Localisation of quantitative trait loci for quality attributes in a doubled haploid population of wheat (Triticum aestivum L); Genome;  citation_year=2009;  citation_volume=52;  citation_pages=701-15; " /><meta name="citation_reference" content="citation_title=A DArT platform for quantitative bulked segregant analysis; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=196" /><meta name="citation_reference" content="citation_title=A second mechanism for aluminum resistance in wheat relies on the constitutive efflux of citrate from roots; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=340-51; " /><meta name="citation_reference" content="citation_title=Molecular diversity and genetic structure of modern and landrace cultivars of wheat (Triticum aestivum L.); Crop Pasture Sci.;  citation_year=2010;  citation_volume=61;  citation_pages=222-9; " /><meta name="citation_reference" content="citation_title=Assessment of molecular diversity in landraces of bread wheat (Triticum aestivum L.) held in an ex situ collection with Diversity Arrays Technology (DArT); Aust. J. Agric. Res.;  citation_year=2007;  citation_volume=58;  citation_pages=1174-82; " /><meta name="citation_reference" content="citation_title=Genome wide association analyses of common wheat (Triticum aestivum L) germplasm identifies multiple loci for aluminium resistance; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=957-66; " /><meta name="citation_reference" content="citation_title=Inheritance of molecular markers for high oleic and low linolenic acids in canola (Brassica napus L.) and impact of genotype and environment on these traits; citation_text=Thesis submitted to the School of Plant Biology; citation_publisher=The University of Western Australia, Perth;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Twinned microspore-derived embryos of canola (Brassica napus L.) are genetically identical; Plant Cell Rep.;  citation_year=2009;  citation_volume=28;  citation_pages=831-5; " /><meta name="citation_reference" content="citation_title=Variation in fatty acid composition among genetically homogeneous seeds of canola (Brassica napus), and implications for genotypic selection based on single seeds; Aust. J. Agric. Res.;  citation_year=2008;  citation_volume=59;  citation_pages=926-32; " /><meta name="citation_reference" content="citation_title=Diversity Arrays Technology (DArT)—a generic genome profiling technology on open platforms; citation_publisher=New York, USA: Humana Press, ; Data Production and Analysis in Population Genomics. Methods in Molecular Biology Series;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Optimizing parental selection for genetic linkage maps; Genome;  citation_year=1993;  citation_volume=36;  citation_pages=181-6; " /><meta name="citation_reference" content="citation_title=Nouvelles recherches sur la distribution florale; Bull. Soc. Vaudoise Sci. Nat.;  citation_year=1908;  citation_volume=44;  citation_pages=223-70; " /><meta name="citation_reference" content="citation_title=Phylogeny reconstruction; citation_publisher=Sinauer Associates, S., MA; Molecular Systematics;  citation_year=1990;  citation_pages=411-501; " /><meta name="citation_reference" content="citation_publisher=Freeman, San Francisco; Numerical Taxonomy;  citation_year=1973;  citation_pages=573" /><meta name="citation_reference" content="citation_publisher=Exeter Software, Setauket, NY; NTSYS-PC: Numerical Taxonomy and Multivariate Analysis System, Ver. 2.10z;  citation_year=2002; " /><meta name="citation_reference" content="citation_title=Efficient large-scale development of microsatellites for marker and mapping applications in Brassica crop species; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1103-12; " /><meta name="citation_reference" content="citation_title=Integration of linkage maps for the amphidiploid Brassica napus and comparative mapping with Arabidopsis and Brassica rapa; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=101" /><meta name="citation_reference" content="citation_title=Microspore culture preferentially selects unreduced (2n) gametes from an interspecific hybrid of Brassica napus L. × Brassica carinata Braun.; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=497-505; " /><meta name="citation_reference" content="citation_title=Comparative mapping of Brassica juncea and Arabidopsis thaliana using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=113" /><meta name="citation_reference" content="citation_title=Constructing large-scale genetic maps using an evolutionary strategy algorithm; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=2269-82; " /><meta name="citation_reference" content="citation_title=Aligning a new reference genetic map of Lupinus angustifolius with the genome sequence of the model legume, Lotus japonicus; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=73-83; " /><meta name="citation_reference" content="citation_title=Genetic diversity in Australian canola and implications for crop breeding for changing future environments; Field Crop. Res.;  citation_year=2007;  citation_volume=104;  citation_pages=103-11; " /><meta name="citation_reference" content="citation_title=Divergent patterns of allelic diversity from similar origins: the case of oilseed rape (Brassica napus L.) in China and Australia; Genome;  citation_year=2008;  citation_volume=51;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Biased distribution of microsatellites in rice genome; Mol. Genet. Genomics;  citation_year=2007;  citation_volume=277;  citation_pages=469-80; " /><meta name="citation_reference" content="citation_title=Brassica napus breeding. Canola in Australia: the First 30 Years; citation_publisher=Australia: Canberra, ; Organising Committee of the 10th International Rapeseed Congress;  citation_year=1999;  citation_pages=29-35; " /><meta name="citation_reference" content="citation_title=Einige experimentaelle Ergebnisse zur Fremdbefurchtungsrate bei Winterrap (Brassica napus oleifera) in Abhangigkeit von Sorte and Abstand.; Z. Pflanzenzuecht;  citation_year=1979;  citation_volume=83;  citation_pages=289-307; " /><meta name="citation_reference" content="citation_title=Outcrossing in rape and mustard under Saskatchewan prairie conditions; Can. J. Plant Sci.;  citation_year=1987;  citation_volume=67;  citation_pages=147-51; " /><meta name="citation_reference" content="citation_title=Comparison of rapeseed cultivars and resynthesized lines based on allozyme and RFLP markers; Theor. Appl. Genet;  citation_year=1995;  citation_volume=91;  citation_pages=62-7; " /><meta name="citation_reference" content="citation_title=Genetic dissection of natural variation for flowering time in rapeseed; citation_publisher=Prague, Czech Republic;  citation_year=2011;  citation_pages=139" /><meta name="citation_reference" content="citation_title=Mapping loci controlling vernalisation requirement and flowering time in Brassica napus; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=90;  citation_pages=727-32; " /><meta name="citation_reference" content="citation_title=Construction of an oilseed rape (Brassica napus L.) genetic map with SSR markers; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1514-23; " /><meta name="citation_reference" content="citation_title=A consensus linkage map for rapeseed (Brassica napus L.): construction and integration of three individual maps from DH populations; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=491-507; " /><meta name="citation_reference" content="citation_title=Genetic map construction and QTL mapping of resistance to blackleg (Leptosphaeria maculans) disease in Australian canola (Brassica napus L.) cultivars; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=120;  citation_pages=71-83; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/51/1312933/dsr041.pdf" /><meta name="description" content="Abstract.  We developed Diversity Array Technology (DArT) markers for application in genetic studies of Brassica napus and other Brassica species with A or C ge" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/51/485713" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/51/485713" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/1/51/485713");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr041';
    </script>
<title>
	Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed (Brassica napus L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./485713" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZyjATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFtYW4sIEhhcnNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRUggR3JhaGFtIENlbnRyZSBmb3IgQWdyaWN1bHR1cmFsIElubm92YXRpb24gKGFuIGFsbGlhbmNlIGJldHdlZW4gdGhlIENoYXJsZXMgU3R1cnQgVW5pdmVyc2l0eSBhbmQgTlNXIERlcGFydG1lbnQgb2YgUHJpbWFyeSBJbmR1c3RyaWVzKSwgV2FnZ2EgV2FnZ2EgQWdyaWN1bHR1cmFsIEluc3RpdHV0ZSwgV2FnZ2EgV2FnZ2EsIE5TVyAyNjUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYW1hbiwgUm9zeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVIIEdyYWhhbSBDZW50cmUgZm9yIEFncmljdWx0dXJhbCBJbm5vdmF0aW9uIChhbiBhbGxpYW5jZSBiZXR3ZWVuIHRoZSBDaGFybGVzIFN0dXJ0IFVuaXZlcnNpdHkgYW5kIE5TVyBEZXBhcnRtZW50IG9mIFByaW1hcnkgSW5kdXN0cmllcyksIFdhZ2dhIFdhZ2dhIEFncmljdWx0dXJhbCBJbnN0aXR1dGUsIFdhZ2dhIFdhZ2dhLCBOU1cgMjY1MCwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmVsc29uLCBNYXR0aGV3IE4uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIFBsYW50IEJpb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCAzNSBTdGlybGluZyBIaWdod2F5LCBDcmF3bGV5LCBXQSA2MDA5LCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDYW5vbGEgQnJlZWRlcnMgV2VzdGVybiBBdXN0cmFsaWEgUHR5IEx0ZCwgTG9ja2VkIEJhZyA4ODgsIENvbW8sIFdBIDY5NTIsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFzbGFtLCBNLk4uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIFBsYW50IEJpb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCAzNSBTdGlybGluZyBIaWdod2F5LCBDcmF3bGV5LCBXQSA2MDA5LCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWphc2VrYXJhbiwgUmF2aWtlc2F2YW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgUGxhbnQgQmlvbG9neSwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIDM1IFN0aXJsaW5nIEhpZ2h3YXksIENyYXdsZXksIFdBIDYwMDksIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1haXplIFJlc2VhcmNoIFN0YXRpb24sIFRhbWlsIE5hZHUgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFZhZ2FyYWkgNjI0IDYxMywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXcmF0dGVuLCBOZWlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRUggR3JhaGFtIENlbnRyZSBmb3IgQWdyaWN1bHR1cmFsIElubm92YXRpb24gKGFuIGFsbGlhbmNlIGJldHdlZW4gdGhlIENoYXJsZXMgU3R1cnQgVW5pdmVyc2l0eSBhbmQgTlNXIERlcGFydG1lbnQgb2YgUHJpbWFyeSBJbmR1c3RyaWVzKSwgV2FnZ2EgV2FnZ2EgQWdyaWN1bHR1cmFsIEluc3RpdHV0ZSwgV2FnZ2EgV2FnZ2EsIE5TVyAyNjUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb3dsaW5nLCBXYWxsYWNlIEEuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2Fub2xhIEJyZWVkZXJzIFdlc3Rlcm4gQXVzdHJhbGlhIFB0eSBMdGQsIExvY2tlZCBCYWcgODg4LCBDb21vLCBXQSA2OTUyLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgVVdBIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyZSwgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIDM1IFN0aXJsaW5nIEhpZ2h3YXksIENyYXdsZXksIFdBIDYwMDksIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbGlhbiwgQS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZlcnNpdHkgQXJyYXkgVGVjaG5vbG9neSBQdHkgTHRkLCAxIFdpbGYgQ3JhbmUsIFlhcnJhbHVtbGEsIEFDVCAyNjAwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGFycGUsIEFuZHJldyBHLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFncmljdWx0dXJlIGFuZCBBZ3JpRm9vZCBDYW5hZGEsIFNhc2thdG9vbiBSZXNlYXJjaCBDZW50cmUsIDEwNyBTY2llbmNlIFBsYWNlLCBTYXNrYXRvb24sIFNLLCBDYW5hZGEsIFM3TiAwWDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hvbmRlbG1haWVyLCBKb2VyZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNhYXRlbi1VbmlvbiBCaW90ZWMgR21iSCwgSG92ZWRpc3NlciBTdHIuOTIsIDMzODE4IExlb3BvbGRzaG9laGUsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpdmVyc2l0eSBBcnJheSBUZWNobm9sb2d5IE1hcmtlcnM6IEdlbmV0aWMgRGl2ZXJzaXR5IEFuYWx5c2VzIGFuZCBMaW5rYWdlIE1hcCBDb25zdHJ1Y3Rpb24gaW4gUmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjA0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBpbiBCcmFzc2ljYSB3aXRoIHNwZWNpYWwgcmVmZXJlbmNlIHRvIHRoZSBleHBlcmltZW50YWwgZm9ybWF0aW9uIG9mIEIuIG5hcHVzIGFuZCBwZWN1bGlhciBtb2RlIG9mIGZlcnRpbGlzYXRpb247IEpwbi4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTkzNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQ1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXNhdGlvbiBvZiBmbG93ZXJpbmcgdGltZSBhbmQgU1NSIG1hcmtlciBhbmFseXNpcyBvZiBzcHJpbmcgYW5kIHdpbnRlciB0eXBlIEJyYXNzaWNhIG5hcHVzIEwuIGdlcm1wbGFzbTsgRXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTQzLTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBnZW5ldGljIGxpbmthZ2UgbWFwIGluIG1hbiB1c2luZyByZXN0cmljdGlvbiBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtczsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzE0LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnVuZGFudCBjbGFzcyBvZiBodW1hbiBETkEgcG9seW1vcnBoaXNtcyB3aGljaCBjYW4gYmUgdHlwZWQgdXNpbmcgdGhlIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb247IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTM4OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUZMUDogYSBuZXcgdGVjaG5pcXVlIGZvciBETkEgZmluZ2VycHJpbnRpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ0MDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBwb2x5bW9ycGhpc21zIGFtcGxpZmllZCBieSBhcmJpdHJhcnkgcHJpbWVycyBhcmUgdXNlZnVsIGFzIGdlbmV0aWMgbWFya2VyczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjUzMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2Nlc3NpbmcgZ2VuZXRpYyBpbmZvcm1hdGlvbiB3aXRoIGhpZ2gtZGVuc2l0eSBETkEgYXJyYXlzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI3NDsgIGNpdGF0aW9uX3BhZ2VzPTYxMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBvZiBvaWxzZWVkIEJyYXNzaWNhIG5hcHVzIGdlcm1wbGFzbSBiYXNlZCBvbiByZXN0cmljdGlvbiBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTY2Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlbGF0aW9uc2hpcHMgYW5kIGZpbmdlcnByaW50aW5nIG9mIHJhcGVzZWVkIGN1bHRpdmFycyBieSBBRkxQOiBjb25zZXF1ZW5jZXMgZm9yIHZhcmlldGFsIHJlZ2lzdHJhdGlvbjsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTQxNy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgYW1vbmdzdCBBdXN0cmFsaWFuIGNhbm9sYSBjdWx0aXZhcnMgZGV0ZXJtaW5lZCBieSByYW5kb21seSBhbXBsaWZpZWQgcG9seW1vcnBoaWMgRE5BOyBBdXN0LiBKLiBFeHAuIEFncmljLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTc5My04MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGdlbmV0aWMgZGl2ZXJzaXR5IGluIHRoZSBCcmFzc2ljYSBuYXB1cyBMLiBnZW5lIHBvb2wgdXNpbmcgU1NSIG1hcmtlcnM7IEdlbmV0LiBSZXMuIENyb3AgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz03OTMtODAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIGdlbmV0aWMgZGl2ZXJzaXR5IGluIEF1c3RyYWxpYW4gY2Fub2xhIChCcmFzc2ljYSBuYXB1cyBMLikgY3VsdGl2YXJzIHVzaW5nIFNTUiBtYXJrZXJzOyBDcm9wIFBhc3R1cmUgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTExOTMtMjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWFya2VycyBpbiBCcmFzc2ljYSBvaWxzZWVkIGJyZWVkaW5nOiBjdXJyZW50IHN0YXR1cyBhbmQgZnV0dXJlIHBvc3NpYmlsaXRpZXM7IFBsYW50IEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkZXZlbG9wbWVudCBvZiBtdWx0aXBsZXggc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCAoU1NSKSBtYXJrZXJzIHRvIGNvbXBsZW1lbnQgZGlzdGluY3RuZXNzLCB1bmlmb3JtaXR5IGFuZCBzdGFiaWxpdHkgdGVzdGluZyBvZiByYXBlIChCcmFzc2ljYSBuYXB1cyBMLikgdmFyaWV0aWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTEwOTEtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBvZiBCcmFzc2ljYSBuYXB1cyBhY2Nlc3Npb25zIG1haW5seSBmcm9tIENoaW5hIGRldGVjdGVkIHdpdGggUkZMUCBtYXJrZXJzOyBDaGluLiBKLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURBclQgZm9yIGhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIG9mIENhc3NhdmEgKE1hbmlob3QgZXNjdWxlbnRhKSBhbmQgaXRzIHdpbGQgcmVsYXRpdmVzOyBUaGVvci4gQXBwbC5HZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTA5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaWZpZWQgbWl4ZWQtbW9kZWwgbWV0aG9kIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIHRoYXQgYWNjb3VudHMgZm9yIG11bHRpcGxlIGxldmVscyBvZiByZWxhdGVkbmVzczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBpZGVudGl0eSBieSBkZXNjZW50IGFuZCBpZGVudGl0eSBieSBzdGF0ZSBmb3IgZGV0ZWN0aW5nIGdlbmV0aWMgcmVnaW9ucyB1bmRlciBzZWxlY3Rpb24gaW4gYSBzb3JnaHVtIHBlZGlncmVlIGJyZWVkaW5nIHByb2dyYW07IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NDQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIEFzIFlvdSBHbzogQW4gZWZmZWN0aXZlIGFwcHJvYWNoIGZvciBtYXJrZXItYXNzaXN0ZWQgc2VsZWN0aW9uIG9mIGNvbXBsZXggdHJhaXRzOyBDcm9wIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xNTYwLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHNlbGVjdGlvbiBpbiBwbGFudCBicmVlZGluZzogZnJvbSB0aGVvcnkgdG8gcHJhY3RpY2U7IEJyaWVmLiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGlvbiBpbiBnZW5vdHlwaW5nIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXM7IEh1bS4gTXV0YXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NDc1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gKFNOUCkgZGlzY292ZXJ5IGluIHRoZSBwb2x5cGxvaWQgQnJhc3NpY2EgbmFwdXMgdXNpbmcgU29sZXhhIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTMzNC00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBieSBhbXBsaWNvbiBzZXF1ZW5jaW5nIGFuZCBtdWx0aXBsZXggU05QIGdlbm90eXBpbmcgaW4gdGhlIGFsbG9wb2x5cGxvaWQgc3BlY2llcyBCcmFzc2ljYSBuYXB1czsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9OTQ4LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIEJyYXNzaWNhIG5hcHVzIEZMT1dFUklORyBMT0NVUyBUIHBhcmFsb2d1ZXMgaW4gdGhlIGNvbnRleHQgb2YgaW52ZXJ0ZWQgY2hyb21vc29tYWwgZHVwbGljYXRpb24gYmxvY2tzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2wgb2YgZmxvd2VyaW5nIHRpbWUgYnkgRkxDIG9ydGhvbG9ndWVzIGluIEJyYXNzaWNhIG5hcHVzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTU0NS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkcyB1bmFtYmlndW91cyB0cmFuc2NyaXB0IG1hcHBpbmcgaW4gdGhlIGFsbG90ZXRyYXBsb2lkIEJyYXNzaWNhIG5hcHVzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz05MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3RpbmcgdGhlIGdlbm9tZSBvZiB0aGUgcG9seXBsb2lkIGNyb3Agb2lsc2VlZCByYXBlIGJ5IHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgTmF0LiBCaW90ZWNoLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTc2Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJvYnVzdCwgc2ltcGxlIEdlbm90eXBpbmctYnktU2VxdWVuY2luZyAoR0JTKSBhcHByb2FjaCBmb3IgaGlnaCBkaXZlcnNpdHkgc3BlY2llczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxOTM3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuYWx5c2lzIG9mIHRoZSBjYW5vbGEgZ2Vub21lOyBjaXRhdGlvbl9wdWJsaXNoZXI9UHJhZ3VlLCBDemVjaCBSZXB1YmxpYzsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3BhZ2VzPTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgYXJyYXlzOiBhIHNvbGlkIHN0YXRlIHRlY2hub2xvZ3kgZm9yIHNlcXVlbmNlIGluZm9ybWF0aW9uIGluZGVwZW5kZW50IGdlbm90eXBpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPWUyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBhcnJheXMgdGVjaG5vbG9neSAoREFyVCkgZm9yIHdob2xlLWdlbm9tZSBwcm9maWxpbmcgb2YgYmFybGV5OyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz05OTE1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIERBclQtU1NSIGxpbmthZ2UgbWFwIG9mIGR1cnVtIHdoZWF0OyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTYyOS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IGFycmF5cyB0ZWNobm9sb2d5IChEQXJUKSBmb3IgaGlnaC10aHJvdWdocHV0IHByb2ZpbGluZyBvZiB0aGUgaGV4YXBsb2lkIHdoZWF0IGdlbm9tZTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0xNDA5LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGdlbmV0aWMgYW5kIGN5dG9nZW5ldGljIG1hcHMgb2YgaGV4YXBsb2lkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLikgdXNpbmcgU1NSIGFuZCBEQXJUIG1hcmtlcnM7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4MTsgIGNpdGF0aW9uX3BhZ2VzPTE4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9REFyVCBtYXJrZXJzOiBkaXZlcnNpdHkgYW5hbHlzZXMgYW5kIG1hcHBpbmcgaW4gU29yZ2h1bSBiaWNvbG9yOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgY29uc2Vuc3VzIG1hcCBvZiBiYXJsZXkgbGlua2luZyBEQXJUIG1hcmtlcnMgdG8gU1NSLCBSRkxQIGFuZCBTVFMgbG9jaSBhbmQgYWdyaWN1bHR1cmFsIHRyYWl0czsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb2NhbGlzYXRpb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIHF1YWxpdHkgYXR0cmlidXRlcyBpbiBhIGRvdWJsZWQgaGFwbG9pZCBwb3B1bGF0aW9uIG9mIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMKTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NzAxLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIERBclQgcGxhdGZvcm0gZm9yIHF1YW50aXRhdGl2ZSBidWxrZWQgc2VncmVnYW50IGFuYWx5c2lzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE5NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2Vjb25kIG1lY2hhbmlzbSBmb3IgYWx1bWludW0gcmVzaXN0YW5jZSBpbiB3aGVhdCByZWxpZXMgb24gdGhlIGNvbnN0aXR1dGl2ZSBlZmZsdXggb2YgY2l0cmF0ZSBmcm9tIHJvb3RzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0zNDAtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBkaXZlcnNpdHkgYW5kIGdlbmV0aWMgc3RydWN0dXJlIG9mIG1vZGVybiBhbmQgbGFuZHJhY2UgY3VsdGl2YXJzIG9mIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7IENyb3AgUGFzdHVyZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MjIyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgbW9sZWN1bGFyIGRpdmVyc2l0eSBpbiBsYW5kcmFjZXMgb2YgYnJlYWQgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKSBoZWxkIGluIGFuIGV4IHNpdHUgY29sbGVjdGlvbiB3aXRoIERpdmVyc2l0eSBBcnJheXMgVGVjaG5vbG9neSAoREFyVCk7IEF1c3QuIEouIEFncmljLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTE3NC04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHdpZGUgYXNzb2NpYXRpb24gYW5hbHlzZXMgb2YgY29tbW9uIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMKSBnZXJtcGxhc20gaWRlbnRpZmllcyBtdWx0aXBsZSBsb2NpIGZvciBhbHVtaW5pdW0gcmVzaXN0YW5jZTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9OTU3LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbmhlcml0YW5jZSBvZiBtb2xlY3VsYXIgbWFya2VycyBmb3IgaGlnaCBvbGVpYyBhbmQgbG93IGxpbm9sZW5pYyBhY2lkcyBpbiBjYW5vbGEgKEJyYXNzaWNhIG5hcHVzIEwuKSBhbmQgaW1wYWN0IG9mIGdlbm90eXBlIGFuZCBlbnZpcm9ubWVudCBvbiB0aGVzZSB0cmFpdHM7IGNpdGF0aW9uX3RleHQ9VGhlc2lzIHN1Ym1pdHRlZCB0byB0aGUgU2Nob29sIG9mIFBsYW50IEJpb2xvZ3k7IGNpdGF0aW9uX3B1Ymxpc2hlcj1UaGUgVW5pdmVyc2l0eSBvZiBXZXN0ZXJuIEF1c3RyYWxpYSwgUGVydGg7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3aW5uZWQgbWljcm9zcG9yZS1kZXJpdmVkIGVtYnJ5b3Mgb2YgY2Fub2xhIChCcmFzc2ljYSBuYXB1cyBMLikgYXJlIGdlbmV0aWNhbGx5IGlkZW50aWNhbDsgUGxhbnQgQ2VsbCBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9ODMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiBmYXR0eSBhY2lkIGNvbXBvc2l0aW9uIGFtb25nIGdlbmV0aWNhbGx5IGhvbW9nZW5lb3VzIHNlZWRzIG9mIGNhbm9sYSAoQnJhc3NpY2EgbmFwdXMpLCBhbmQgaW1wbGljYXRpb25zIGZvciBnZW5vdHlwaWMgc2VsZWN0aW9uIGJhc2VkIG9uIHNpbmdsZSBzZWVkczsgQXVzdC4gSi4gQWdyaWMuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz05MjYtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBBcnJheXMgVGVjaG5vbG9neSAoREFyVCnigJRhIGdlbmVyaWMgZ2Vub21lIHByb2ZpbGluZyB0ZWNobm9sb2d5IG9uIG9wZW4gcGxhdGZvcm1zOyBjaXRhdGlvbl9wdWJsaXNoZXI9TmV3IFlvcmssIFVTQTogSHVtYW5hIFByZXNzLCA7IERhdGEgUHJvZHVjdGlvbiBhbmQgQW5hbHlzaXMgaW4gUG9wdWxhdGlvbiBHZW5vbWljcy4gTWV0aG9kcyBpbiBNb2xlY3VsYXIgQmlvbG9neSBTZXJpZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltaXppbmcgcGFyZW50YWwgc2VsZWN0aW9uIGZvciBnZW5ldGljIGxpbmthZ2UgbWFwczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdXZlbGxlcyByZWNoZXJjaGVzIHN1ciBsYSBkaXN0cmlidXRpb24gZmxvcmFsZTsgQnVsbC4gU29jLiBWYXVkb2lzZSBTY2kuIE5hdC47ICBjaXRhdGlvbl95ZWFyPTE5MDg7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0yMjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VueSByZWNvbnN0cnVjdGlvbjsgY2l0YXRpb25fcHVibGlzaGVyPVNpbmF1ZXIgQXNzb2NpYXRlcywgUy4sIE1BOyBNb2xlY3VsYXIgU3lzdGVtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl9wYWdlcz00MTEtNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RnJlZW1hbiwgU2FuIEZyYW5jaXNjbzsgTnVtZXJpY2FsIFRheG9ub215OyAgY2l0YXRpb25feWVhcj0xOTczOyAgY2l0YXRpb25fcGFnZXM9NTczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUV4ZXRlciBTb2Z0d2FyZSwgU2V0YXVrZXQsIE5ZOyBOVFNZUy1QQzogTnVtZXJpY2FsIFRheG9ub215IGFuZCBNdWx0aXZhcmlhdGUgQW5hbHlzaXMgU3lzdGVtLCBWZXIuIDIuMTB6OyAgY2l0YXRpb25feWVhcj0yMDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgbGFyZ2Utc2NhbGUgZGV2ZWxvcG1lbnQgb2YgbWljcm9zYXRlbGxpdGVzIGZvciBtYXJrZXIgYW5kIG1hcHBpbmcgYXBwbGljYXRpb25zIGluIEJyYXNzaWNhIGNyb3Agc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMTAzLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGlvbiBvZiBsaW5rYWdlIG1hcHMgZm9yIHRoZSBhbXBoaWRpcGxvaWQgQnJhc3NpY2EgbmFwdXMgYW5kIGNvbXBhcmF0aXZlIG1hcHBpbmcgd2l0aCBBcmFiaWRvcHNpcyBhbmQgQnJhc3NpY2EgcmFwYTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zcG9yZSBjdWx0dXJlIHByZWZlcmVudGlhbGx5IHNlbGVjdHMgdW5yZWR1Y2VkICgybikgZ2FtZXRlcyBmcm9tIGFuIGludGVyc3BlY2lmaWMgaHlicmlkIG9mIEJyYXNzaWNhIG5hcHVzIEwuIMOXIEJyYXNzaWNhIGNhcmluYXRhIEJyYXVuLjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBtYXBwaW5nIG9mIEJyYXNzaWNhIGp1bmNlYSBhbmQgQXJhYmlkb3BzaXMgdGhhbGlhbmEgdXNpbmcgSW50cm9uIFBvbHltb3JwaGlzbSAoSVApIG1hcmtlcnM6IGhvbW9lb2xvZ291cyByZWxhdGlvbnNoaXBzLCBkaXZlcnNpZmljYXRpb24gYW5kIGV2b2x1dGlvbiBvZiB0aGUgQSwgQiBhbmQgQyBCcmFzc2ljYSBnZW5vbWVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTExMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGluZyBsYXJnZS1zY2FsZSBnZW5ldGljIG1hcHMgdXNpbmcgYW4gZXZvbHV0aW9uYXJ5IHN0cmF0ZWd5IGFsZ29yaXRobTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MjI2OS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxpZ25pbmcgYSBuZXcgcmVmZXJlbmNlIGdlbmV0aWMgbWFwIG9mIEx1cGludXMgYW5ndXN0aWZvbGl1cyB3aXRoIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG1vZGVsIGxlZ3VtZSwgTG90dXMgamFwb25pY3VzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBpbiBBdXN0cmFsaWFuIGNhbm9sYSBhbmQgaW1wbGljYXRpb25zIGZvciBjcm9wIGJyZWVkaW5nIGZvciBjaGFuZ2luZyBmdXR1cmUgZW52aXJvbm1lbnRzOyBGaWVsZCBDcm9wLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJnZW50IHBhdHRlcm5zIG9mIGFsbGVsaWMgZGl2ZXJzaXR5IGZyb20gc2ltaWxhciBvcmlnaW5zOiB0aGUgY2FzZSBvZiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKSBpbiBDaGluYSBhbmQgQXVzdHJhbGlhOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0xLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzZWQgZGlzdHJpYnV0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiByaWNlIGdlbm9tZTsgTW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9NDY5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmFzc2ljYSBuYXB1cyBicmVlZGluZy4gQ2Fub2xhIGluIEF1c3RyYWxpYTogdGhlIEZpcnN0IDMwIFllYXJzOyBjaXRhdGlvbl9wdWJsaXNoZXI9QXVzdHJhbGlhOiBDYW5iZXJyYSwgOyBPcmdhbmlzaW5nIENvbW1pdHRlZSBvZiB0aGUgMTB0aCBJbnRlcm5hdGlvbmFsIFJhcGVzZWVkIENvbmdyZXNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fcGFnZXM9MjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVpbmlnZSBleHBlcmltZW50YWVsbGUgRXJnZWJuaXNzZSB6dXIgRnJlbWRiZWZ1cmNodHVuZ3NyYXRlIGJlaSBXaW50ZXJyYXAgKEJyYXNzaWNhIG5hcHVzIG9sZWlmZXJhKSBpbiBBYmhhbmdpZ2tlaXQgdm9uIFNvcnRlIGFuZCBBYnN0YW5kLjsgWi4gUGZsYW56ZW56dWVjaHQ7ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl92b2x1bWU9ODM7ICBjaXRhdGlvbl9wYWdlcz0yODktMzA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdXRjcm9zc2luZyBpbiByYXBlIGFuZCBtdXN0YXJkIHVuZGVyIFNhc2thdGNoZXdhbiBwcmFpcmllIGNvbmRpdGlvbnM7IENhbi4gSi4gUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTE0Ny01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiByYXBlc2VlZCBjdWx0aXZhcnMgYW5kIHJlc3ludGhlc2l6ZWQgbGluZXMgYmFzZWQgb24gYWxsb3p5bWUgYW5kIFJGTFAgbWFya2VyczsgVGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXNzZWN0aW9uIG9mIG5hdHVyYWwgdmFyaWF0aW9uIGZvciBmbG93ZXJpbmcgdGltZSBpbiByYXBlc2VlZDsgY2l0YXRpb25fcHVibGlzaGVyPVByYWd1ZSwgQ3plY2ggUmVwdWJsaWM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl9wYWdlcz0xMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGxvY2kgY29udHJvbGxpbmcgdmVybmFsaXNhdGlvbiByZXF1aXJlbWVudCBhbmQgZmxvd2VyaW5nIHRpbWUgaW4gQnJhc3NpY2EgbmFwdXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz03MjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKSBnZW5ldGljIG1hcCB3aXRoIFNTUiBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29uc2Vuc3VzIGxpbmthZ2UgbWFwIGZvciByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pOiBjb25zdHJ1Y3Rpb24gYW5kIGludGVncmF0aW9uIG9mIHRocmVlIGluZGl2aWR1YWwgbWFwcyBmcm9tIERIIHBvcHVsYXRpb25zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTQ5MS01MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwIGNvbnN0cnVjdGlvbiBhbmQgUVRMIG1hcHBpbmcgb2YgcmVzaXN0YW5jZSB0byBibGFja2xlZyAoTGVwdG9zcGhhZXJpYSBtYWN1bGFucykgZGlzZWFzZSBpbiBBdXN0cmFsaWFuIGNhbm9sYSAoQnJhc3NpY2EgbmFwdXMgTC4pIGN1bHRpdmFyczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz03MS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzEvNTEvMTMxMjkzMy9kc3IwNDEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIGRldmVsb3BlZCBEaXZlcnNpdHkgQXJyYXkgVGVjaG5vbG9neSAoREFyVCkgbWFya2VycyBmb3IgYXBwbGljYXRpb24gaW4gZ2VuZXRpYyBzdHVkaWVzIG9mIEJyYXNzaWNhIG5hcHVzIGFuZCBvdGhlciBCcmFzc2ljYSBzcGVjaWVzIHdpdGggQSBvciBDIGdlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODU3MTMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkJ++ccQ8ieq6c2FP45PAvMIpuc+0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f51%2f485713" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f51%2f485713" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19070" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&amp;Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7387260">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387260">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387262">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387262">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387269">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387269">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387293">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387293">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387308">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387308">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387318">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387318">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387320">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387320">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387322">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387322">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387324">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387324">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7387326">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7387326">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/1/37/485401">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/67/485988">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed (<em>Brassica napus</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Harsh Raman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries)</div>, <div class="insititution">Wagga Wagga Agricultural Institute</div>, <div class="addr-line">Wagga Wagga, NSW 2650</div>, <div class="country">Australia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +61 2-6938-1925. Fax. +61 2-6938-1809. E-mail: <a href="mailto:harsh.raman@industry.nsw.gov.au" target="_blank">harsh.raman@industry.nsw.gov.au</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Harsh+Raman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Raman H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Raman H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Harsh Raman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rosy Raman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries)</div>, <div class="insititution">Wagga Wagga Agricultural Institute</div>, <div class="addr-line">Wagga Wagga, NSW 2650</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rosy+Raman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Raman R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Raman R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rosy Raman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew N. Nelson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Plant Biology</div>, <div class="insititution">The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Canola Breeders Western Australia Pty Ltd</div>, <div class="addr-line">Locked Bag 888, Como, WA 6952</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+N.+Nelson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nelson M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nelson M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthew N. Nelson</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M.N. Aslam</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Plant Biology</div>, <div class="insititution">The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.N.+Aslam">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aslam M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aslam M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">M.N. Aslam</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ravikesavan Rajasekaran</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Plant Biology</div>, <div class="insititution">The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Maize Research Station</div>, <div class="insititution">Tamil Nadu Agricultural University</div>, <div class="addr-line">Vagarai 624 613</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ravikesavan+Rajasekaran">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rajasekaran R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rajasekaran R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ravikesavan Rajasekaran</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Neil Wratten</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">EH Graham Centre for Agricultural Innovation (an alliance between the Charles Sturt University and NSW Department of Primary Industries)</div>, <div class="insititution">Wagga Wagga Agricultural Institute</div>, <div class="addr-line">Wagga Wagga, NSW 2650</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Neil+Wratten">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wratten N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wratten N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Neil Wratten</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wallace A. Cowling</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Canola Breeders Western Australia Pty Ltd</div>, <div class="addr-line">Locked Bag 888, Como, WA 6952</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">The UWA Institute of Agriculture</div>, <div class="insititution">The University of Western Australia</div>, <div class="addr-line">35 Stirling Highway, Crawley, WA 6009</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wallace+A.+Cowling">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cowling W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cowling W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wallace A. Cowling</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A. Kilian</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Diversity Array Technology Pty Ltd</div>, <div class="addr-line">1 Wilf Crane, Yarralumla, ACT 2600</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A.+Kilian">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kilian A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kilian A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">A. Kilian</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew G. Sharpe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Agriculture and AgriFood Canada</div>, <div class="insititution">Saskatoon Research Centre</div>, <div class="addr-line">107 Science Place, Saskatoon, SK</div>, <div class="country">Canada</div>, <div class="addr-line">S7N 0X2</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+G.+Sharpe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sharpe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sharpe A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew G. Sharpe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Joerg Schondelmaier</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">Saaten-Union Biotec GmbH</div>, <div class="addr-line">Hovedisser Str.92, 33818 Leopoldshoehe</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Joerg+Schondelmaier">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schondelmaier J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schondelmaier J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Joerg Schondelmaier</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: NRC Plant Biotechnology Institute, 110 Gymnasium Place, Saskatoon, SK, Canada S7N 0W9.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 51–65, <a href='https://doi.org/10.1093/dnares/dsr041'>https://doi.org/10.1093/dnares/dsr041</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">22 December 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 September 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 November 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="485713" href="/dnaresearch/article-pdf/19/1/51/1312933/dsr041.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Harsh Raman, Rosy Raman, Matthew N. Nelson, M.N. Aslam, Ravikesavan Rajasekaran, Neil Wratten, Wallace A. Cowling, A. Kilian, Andrew G. Sharpe, Joerg Schondelmaier;  Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed (<em>Brassica napus</em> L.), <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 51–65, <a href='https://doi.org/10.1093/dnares/dsr041'>https://doi.org/10.1093/dnares/dsr041</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/485713?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/485713?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Diversity%20Array%20Technology%20Markers%3A%20Genetic%20Diversity%20Analyses%20and%20Linkage%20Map%20Construction%20in%20Rapeseed%20(%26lt%3Bem%26gt%3BBrassica%20napus%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2011-12-22&amp;volumeNum=19&amp;issueNum=1&amp;author=Raman%2C%20Harsh%3B%20Raman%2C%20Rosy&amp;startPage=51&amp;endPage=65&amp;contentId=10.1093%2Fdnares%2Fdsr041&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f51%2f485713" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7387260 id="7387260" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We developed Diversity Array Technology (DArT) markers for application in genetic studies of <em>Brassica napus</em> and other <em>Brassica</em> species with A or C genomes. Genomic representation from 107 diverse genotypes of <em>B. napus</em> L. var. <em>oleifera</em> (rapeseed, AACC genomes) and <em>B. rapa</em> (AA genome) was used to develop a DArT array comprising 11 520 clones generated using <em>Pst</em>I<em>/Ban</em>II and <em>Pst</em>I<em>/Bst</em>N1 complexity reduction methods. In total, 1547 polymorphic DArT markers of high technical quality were identified and used to assess molecular diversity among 89 accessions of <em>B. napus</em>, <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. carinata</em> collected from different parts of the world. Hierarchical cluster and principal component analyses based on genetic distance matrices identified distinct populations clustering mainly according to their origin/pedigrees. DArT markers were also mapped in a new doubled haploid population comprising 131 lines from a cross between spring rapeseed lines ‘Lynx-037DH’ and ‘Monty-028DH’. Linkage groups were assigned on the basis of previously mapped simple sequence repeat (SSRs), intron polymorphism (IP), and gene-based markers. The map consisted of 437 DArT, 135 SSR, 6 IP, and 6 gene-based markers and spanned 2288 cM. Our results demonstrate that DArT markers are suitable for genetic diversity analysis and linkage map construction in rapeseed.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Diversity Array Technology&quot;">Diversity Array Technology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic diversity&quot;">genetic diversity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage mapping&quot;">genetic linkage mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Brassica species&quot;"><em>Brassica</em> species</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rapeseed">rapeseed</a></div>                            <h2 scrollto-destination=7387262 id="7387262" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The Brassiceae tribe consists some of the world's most important oilseed and vegetable crops, such as <em>Brassica napus</em> L., (rapeseed, genomes = AACC, 2<em>n</em> = 4<em>x</em> = 38), <em>Brassica rapa</em> L. (Indian mustard, genome = AA, 2<em>n</em> = 2<em>x</em> = 20), <em>Brassica juncea</em> L. (genomes = AABB, 2<em>n</em> = 4<em>x</em> = 36), and <em>Brassica oleracea</em> L. (cauliflower, broccoli, Brussels sprout, cabbage, and kale, genome = CC, 2<em>n</em> = 2<em>x</em> = 18). <em>Brassica napus</em> originated as a result from spontaneous hybridization between <em>B. rapa</em> and <em>B. oleracea</em><sup><a reveal-id="DSR041C1" data-open="DSR041C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and is believed to be originated in the Mediterranean region of south-western Europe where native <em>B. rapa</em> and <em>B. oleracea</em> overlap,<sup><a reveal-id="DSR041C2" data-open="DSR041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> although no wild populations exist. Today, rapeseed is one of the leading sources of vegetable oil, oil-meal, fodder, and serves as the raw material for a broad range of industrial products including bio-fuel, especially in European countries.</p><p>Various molecular marker system based upon restriction fragment length polymorphism (RFLP), randomly amplified polymorphic DNA, amplified fragment length polymorphism (AFLP<sup>®</sup>), simple sequence repeats (SSRs), sequence related amplified polymorphisms, and single nucleotide polymorphism (SNP) have been developed<sup><a reveal-id="DSR041C3  DSR041C7" data-open="DSR041C3  DSR041C7" class="link link-ref link-reveal xref-bibr">3–7</a></sup> and further applied in rapeseed genetics and breeding research.<sup><a reveal-id="DSR041C2" data-open="DSR041C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR041C8  DSR041C15" data-open="DSR041C8  DSR041C15" class="link link-ref link-reveal xref-bibr">8–15</a></sup> Most of these markers are assayed on low-throughput agarose or polyacrylamide gel system, although markers based upon SSR and AFLP can be assayed on highly parallel genotyping platforms such as capillary electrophoresis systems, yet they are expensive to assay per data point.<sup><a reveal-id="DSR041C16" data-open="DSR041C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>Rapeseed breeding programmes require an efficient, cost-efficient and reproducible marker platform that is amenable for whole genomic analysis especially for pedigree and association analysis,<sup><a reveal-id="DSR041C17" data-open="DSR041C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR041C18" data-open="DSR041C18" class="link link-ref link-reveal xref-bibr">18</a></sup> mapping-as-you-go,<sup><a reveal-id="DSR041C19" data-open="DSR041C19" class="link link-ref link-reveal xref-bibr">19</a></sup> large-scale molecular evaluation of germplasm collections and for genome-wide selection of desirable alleles.<sup><a reveal-id="DSR041C20" data-open="DSR041C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Current polymerase chain reaction (PCR)-based marker technologies are not practicable in terms of consumable and labour costs for such applications.</p><p>DNA hybridization-based technologies such as some SNP technologies and Diversity Arrays Technology (DArT) are suitable for such applications. SNP markers are recognized as ‘markers of choice’ due to their abundance and distribution in the genomes and the ability to screen populations at relatively low cost.<sup><a reveal-id="DSR041C21" data-open="DSR041C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In the recent years, a large number of SNP markers has been identified in <em>B. napus</em> and related species.<sup><a reveal-id="DSR041C22" data-open="DSR041C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSR041C23" data-open="DSR041C23" class="link link-ref link-reveal xref-bibr">23</a></sup> However, the identification of SNPs and validation in the relevant germplasm especially those identified by sequencing candidate genes remains costly to implement for the routine marker-assisted selection, as it requires high-quality sequence information. This limits the broad application of SNP markers for rapeseed improvement using these strategies. Many genes of agronomic importance such as flowering time genes; <em>Flowering Locus C</em> (<em>FLC</em>) and <em>Flowering Locus T</em> (<em>FT</em>) occur in multiple copies in <em>Brassica</em> genomes.<sup><a reveal-id="DSR041C24" data-open="DSR041C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR041C25" data-open="DSR041C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The presence of multiple copies of genes in amphidiploid rapeseed poses a great challenge to develop allele-specific SNP markers. To date, only a few genome-specific SNP markers are currently available for marker-assisted selection although the ability to resolve allelic variation in members of gene families has been demonstrated.<sup><a reveal-id="DSR041C26" data-open="DSR041C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR041C27" data-open="DSR041C27" class="link link-ref link-reveal xref-bibr">27</a></sup> In addition, a large number of SSR and SNP markers have been developed in private consortia and are not readily accessible to the rapeseed breeding programmes. The development of ‘genotyping by sequencing’ and multiplex sequencing strategies using next generation DNA-sequencing technologies offer to generate massive amount of genetic data for various applications.<sup><a reveal-id="DSR041C28" data-open="DSR041C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR041C29" data-open="DSR041C29" class="link link-ref link-reveal xref-bibr">29</a></sup> However, these technologies are currently in the development stage and bioinformatics tools to handle such massive data in polyploid <em>Brassica</em> species are still being developed.<sup><a reveal-id="DSR041C30" data-open="DSR041C30" class="link link-ref link-reveal xref-bibr">30</a></sup> These limitations currently restrict the capacity of breeding programmes to routinely conduct genome-wide marker surveys.</p><p>DArT performs well in many polyploid species and does not require any DNA-sequence information. DArT marker analysis is a sequence-independent microarray-based genotyping platform, and enables high multiplexing; simultaneous typing of several hundred to several thousands of polymorphic loci spread over the genome.<sup><a reveal-id="DSR041C31" data-open="DSR041C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR041C32" data-open="DSR041C32" class="link link-ref link-reveal xref-bibr">32</a></sup> DArT polymorphisms results from nucleotide polymorphisms within restriction enzyme (RE) recognition sites and indels, and the high fidelity of the RE can provide better reproducibility compared with PCR-based assays that are based on lower fidelity selective primer annealing. DArT markers have been developed in more than 40 plant species including in wheat, durum, barley, oats, lupin (<a class="link link-uri" href="http://www.diversityarray.com">www.diversityarray.com</a>) and have been employed extensively for construction of molecular maps,<sup><a reveal-id="DSR041C33  DSR041C36" data-open="DSR041C33  DSR041C36" class="link link-ref link-reveal xref-bibr">33–36</a></sup> identifying trait-marker associations,<sup><a reveal-id="DSR041C37  DSR041C40" data-open="DSR041C37  DSR041C40" class="link link-ref link-reveal xref-bibr">37–40</a></sup> assessment of genetic diversity,<sup><a reveal-id="DSR041C34" data-open="DSR041C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSR041C41" data-open="DSR041C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSR041C42" data-open="DSR041C42" class="link link-ref link-reveal xref-bibr">42</a></sup> association mapping,<sup><a reveal-id="DSR041C43" data-open="DSR041C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and routine genotyping in various crops for varietal identification.</p><p>In this study, we report on (i) the development of a robust, cost-effective, and high-throughput DArT marker platform for rapeseed, and (ii) demonstrate the usefulness of DArT markers in the assessment of genetic diversity and linkage mapping.</p>                            <h2 scrollto-destination=7387269 id="7387269" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7387270 id="7387270" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>Eighty-nine accessions of <em>B. napus</em>, <em>B. juncea</em>, <em>B. rapa</em> and <em>B. carinata</em> comprising contemporary rapeseed cultivars and elite lines from Australian breeding programmes were chosen for molecular diversity analyses along with accessions from China, Germany, Japan, India, France, and Ethiopia (Table <a reveal-id="DSR041TB1" data-open="DSR041TB1" class="link link-reveal link-table xref-fig">1</a>). Seed samples were procured from the Australian National Brassica Germplasm Improvement Programs (Wagga Wagga and Horsham) and Australian Temperate Field Crops Collection, Horsham. </p>                        <a id="7387272" scrollto-destination="7387272"></a>
<div content-id="DSR041TB1" class="table-modal"><div class="table-wrap"><div id="DSR041TB1" class="table-wrap-title" data-id="DSR041TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of genotypes, their country of origin, and species used for DArT analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotype ID </th><th>Country of origin </th><th>Species </th><th>Breeding programme<sup>a</sup> </th><th>Genetic status </th></tr></thead><tbody><tr><td>03-p74-6 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>04-p34 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>44C73 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>44Y06<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C05 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C75<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45Y77<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C04 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C76<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46Y78 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>A-19890<sup>a</sup> </td><td>Unknown </td><td><em>B. napus</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>Ag-Comet </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Emblem </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Muster<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Outback<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Spectrum<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATC93184-1<sup>a</sup> </td><td>Unknown </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATC94044-1<sup>a</sup> </td><td>Ethiopia </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATR409 </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Barra </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Beacon </td><td>Australia </td><td><em>B.napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>ATR-Cobbler </td><td>Australia </td><td><em>B.napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Hyden </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Marlin </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Signal </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Stubby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Summitt </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Jade </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Opal </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Ruby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Sapphire </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Barossa<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>BravoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>Carousel-10<sup>a,b</sup> </td><td>Europe </td><td><em>B. napus</em> </td><td>Unknown </td><td>Selection </td></tr><tr><td>Charlton </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Colt<sup>a</sup> </td><td>Unknown </td><td><em>B. rapa</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Drakkar </td><td>France </td><td><em>B. napus</em> </td><td>INRA </td><td>Cultivar </td></tr><tr><td>Dunkeld </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Eureka<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Expander<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>BGRC </td><td>Cultivar </td></tr><tr><td>Fan023 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan028 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan168 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>FlindersTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Georgie </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Grouse </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>HurricaneTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola50 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola60 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola61 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Iwao-natane<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Karoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Lantern </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Lisora<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>DSV </td><td>Cultivar </td></tr><tr><td>Major<sup>a,b</sup> </td><td>France </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Maluka<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Marnoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Monty </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Mutu98-1<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>NSWA </td><td>Breeding line </td></tr><tr><td>Mystic </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Nindoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Norin22<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>OasisCL<sup>a</sup> </td><td>Australia </td><td><em>B. juncea</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Oscar </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>P624 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Purler </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Qu1104 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Rainbow<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Range </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag Seed </td><td>Cultivar </td></tr><tr><td>Ripper </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Rivette </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>RocketCL </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>RottnestTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>RSO94-67 (98-18) </td><td>Unknown </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Scoop </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Seetha<sup>a</sup> </td><td>India </td><td><em>B. juncea</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Shiralee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Skipton </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>StormTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific seeds </td><td>Cultivar </td></tr><tr><td>Surpass400<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass402CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass404CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass501TT<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass603CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tarcoola </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Tatyoon </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>TawrifficTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>TERI(OO)R9903<sup>a</sup> </td><td>India </td><td><em>B. napus</em> </td><td>TERI </td><td>Breeding line </td></tr><tr><td>ThunderTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>TornadoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tranby </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>WarriorCL </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Wesbarker<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesreo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesroona<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Yickadee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Yu 178 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Zhongyou 821 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr></tbody></table></div><div content-id="DSR041TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotype ID </th><th>Country of origin </th><th>Species </th><th>Breeding programme<sup>a</sup> </th><th>Genetic status </th></tr></thead><tbody><tr><td>03-p74-6 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>04-p34 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>44C73 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>44Y06<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C05 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C75<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45Y77<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C04 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C76<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46Y78 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>A-19890<sup>a</sup> </td><td>Unknown </td><td><em>B. napus</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>Ag-Comet </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Emblem </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Muster<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Outback<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Spectrum<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATC93184-1<sup>a</sup> </td><td>Unknown </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATC94044-1<sup>a</sup> </td><td>Ethiopia </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATR409 </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Barra </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Beacon </td><td>Australia </td><td><em>B.napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>ATR-Cobbler </td><td>Australia </td><td><em>B.napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Hyden </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Marlin </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Signal </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Stubby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Summitt </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Jade </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Opal </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Ruby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Sapphire </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Barossa<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>BravoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>Carousel-10<sup>a,b</sup> </td><td>Europe </td><td><em>B. napus</em> </td><td>Unknown </td><td>Selection </td></tr><tr><td>Charlton </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Colt<sup>a</sup> </td><td>Unknown </td><td><em>B. rapa</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Drakkar </td><td>France </td><td><em>B. napus</em> </td><td>INRA </td><td>Cultivar </td></tr><tr><td>Dunkeld </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Eureka<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Expander<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>BGRC </td><td>Cultivar </td></tr><tr><td>Fan023 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan028 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan168 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>FlindersTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Georgie </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Grouse </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>HurricaneTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola50 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola60 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola61 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Iwao-natane<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Karoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Lantern </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Lisora<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>DSV </td><td>Cultivar </td></tr><tr><td>Major<sup>a,b</sup> </td><td>France </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Maluka<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Marnoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Monty </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Mutu98-1<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>NSWA </td><td>Breeding line </td></tr><tr><td>Mystic </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Nindoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Norin22<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>OasisCL<sup>a</sup> </td><td>Australia </td><td><em>B. juncea</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Oscar </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>P624 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Purler </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Qu1104 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Rainbow<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Range </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag Seed </td><td>Cultivar </td></tr><tr><td>Ripper </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Rivette </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>RocketCL </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>RottnestTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>RSO94-67 (98-18) </td><td>Unknown </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Scoop </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Seetha<sup>a</sup> </td><td>India </td><td><em>B. juncea</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Shiralee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Skipton </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>StormTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific seeds </td><td>Cultivar </td></tr><tr><td>Surpass400<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass402CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass404CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass501TT<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass603CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tarcoola </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Tatyoon </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>TawrifficTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>TERI(OO)R9903<sup>a</sup> </td><td>India </td><td><em>B. napus</em> </td><td>TERI </td><td>Breeding line </td></tr><tr><td>ThunderTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>TornadoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tranby </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>WarriorCL </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Wesbarker<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesreo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesroona<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Yickadee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Yu 178 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Zhongyou 821 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>BGRC, Institut fur Pflanzenbau und Pflanzenzuchtung, Braunschweig (Germany); DSV, Deutsche Saatveredelung (Germany); AgWA, Department of Agriculture, Western Australia; NSWA, NSW Department of Agriculture (now NSW Department of Primary Industries - NSWDPI); USDA, United State Department of Agriculture; AgVic, Victorian Department of Agriculture.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genotypes were also used for testing suitability of DArT in differentiation of different species.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Genotypes are winter/semi-spring types and unmarked are spring types.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7387272" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR041TB1" class="table-wrap-title" data-id="DSR041TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of genotypes, their country of origin, and species used for DArT analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotype ID </th><th>Country of origin </th><th>Species </th><th>Breeding programme<sup>a</sup> </th><th>Genetic status </th></tr></thead><tbody><tr><td>03-p74-6 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>04-p34 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>44C73 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>44Y06<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C05 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C75<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45Y77<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C04 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C76<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46Y78 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>A-19890<sup>a</sup> </td><td>Unknown </td><td><em>B. napus</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>Ag-Comet </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Emblem </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Muster<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Outback<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Spectrum<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATC93184-1<sup>a</sup> </td><td>Unknown </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATC94044-1<sup>a</sup> </td><td>Ethiopia </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATR409 </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Barra </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Beacon </td><td>Australia </td><td><em>B.napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>ATR-Cobbler </td><td>Australia </td><td><em>B.napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Hyden </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Marlin </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Signal </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Stubby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Summitt </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Jade </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Opal </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Ruby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Sapphire </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Barossa<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>BravoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>Carousel-10<sup>a,b</sup> </td><td>Europe </td><td><em>B. napus</em> </td><td>Unknown </td><td>Selection </td></tr><tr><td>Charlton </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Colt<sup>a</sup> </td><td>Unknown </td><td><em>B. rapa</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Drakkar </td><td>France </td><td><em>B. napus</em> </td><td>INRA </td><td>Cultivar </td></tr><tr><td>Dunkeld </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Eureka<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Expander<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>BGRC </td><td>Cultivar </td></tr><tr><td>Fan023 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan028 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan168 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>FlindersTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Georgie </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Grouse </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>HurricaneTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola50 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola60 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola61 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Iwao-natane<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Karoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Lantern </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Lisora<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>DSV </td><td>Cultivar </td></tr><tr><td>Major<sup>a,b</sup> </td><td>France </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Maluka<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Marnoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Monty </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Mutu98-1<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>NSWA </td><td>Breeding line </td></tr><tr><td>Mystic </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Nindoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Norin22<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>OasisCL<sup>a</sup> </td><td>Australia </td><td><em>B. juncea</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Oscar </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>P624 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Purler </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Qu1104 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Rainbow<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Range </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag Seed </td><td>Cultivar </td></tr><tr><td>Ripper </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Rivette </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>RocketCL </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>RottnestTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>RSO94-67 (98-18) </td><td>Unknown </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Scoop </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Seetha<sup>a</sup> </td><td>India </td><td><em>B. juncea</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Shiralee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Skipton </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>StormTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific seeds </td><td>Cultivar </td></tr><tr><td>Surpass400<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass402CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass404CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass501TT<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass603CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tarcoola </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Tatyoon </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>TawrifficTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>TERI(OO)R9903<sup>a</sup> </td><td>India </td><td><em>B. napus</em> </td><td>TERI </td><td>Breeding line </td></tr><tr><td>ThunderTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>TornadoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tranby </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>WarriorCL </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Wesbarker<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesreo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesroona<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Yickadee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Yu 178 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Zhongyou 821 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr></tbody></table></div><div content-id="DSR041TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotype ID </th><th>Country of origin </th><th>Species </th><th>Breeding programme<sup>a</sup> </th><th>Genetic status </th></tr></thead><tbody><tr><td>03-p74-6 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>04-p34 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>44C73 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>44Y06<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C05 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45C75<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>45Y77<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C04 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46C76<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>46Y78 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pioneer </td><td>Cultivar </td></tr><tr><td>A-19890<sup>a</sup> </td><td>Unknown </td><td><em>B. napus</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>Ag-Comet </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Emblem </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Muster<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Outback<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Ag-Spectrum<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATC93184-1<sup>a</sup> </td><td>Unknown </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATC94044-1<sup>a</sup> </td><td>Ethiopia </td><td><em>B. carinata</em> </td><td>USDA </td><td>Breeding line </td></tr><tr><td>ATR409 </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Barra </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Beacon </td><td>Australia </td><td><em>B.napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>ATR-Cobbler </td><td>Australia </td><td><em>B.napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Hyden </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Marlin </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Signal </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>ATR-Stubby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>ATR-Summitt </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Jade </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Opal </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Ruby </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>AV-Sapphire </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Barossa<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>BravoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>Carousel-10<sup>a,b</sup> </td><td>Europe </td><td><em>B. napus</em> </td><td>Unknown </td><td>Selection </td></tr><tr><td>Charlton </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Colt<sup>a</sup> </td><td>Unknown </td><td><em>B. rapa</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Drakkar </td><td>France </td><td><em>B. napus</em> </td><td>INRA </td><td>Cultivar </td></tr><tr><td>Dunkeld </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Eureka<sup>a</sup> </td><td>Australia </td><td><em>B.napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Expander<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>BGRC </td><td>Cultivar </td></tr><tr><td>Fan023 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan028 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Fan168 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>FlindersTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag-Seed </td><td>Cultivar </td></tr><tr><td>Georgie </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Grouse </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>HurricaneTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola50 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola60 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Hyola61 </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Iwao-natane<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Karoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Lantern </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Lisora<sup>b</sup> </td><td>Germany </td><td><em>B. napus</em> </td><td>DSV </td><td>Cultivar </td></tr><tr><td>Major<sup>a,b</sup> </td><td>France </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Maluka<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Marnoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Monty </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Mutu98-1<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>NSWA </td><td>Breeding line </td></tr><tr><td>Mystic </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Nindoo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Norin22<sup>a,b</sup> </td><td>Japan </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>OasisCL<sup>a</sup> </td><td>Australia </td><td><em>B. juncea</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Oscar </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>P624 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Purler </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Qu1104 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Rainbow<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>Range </td><td>Australia </td><td><em>B. napus</em> </td><td>Ag Seed </td><td>Cultivar </td></tr><tr><td>Ripper </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Rivette </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>RocketCL </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>RottnestTTC </td><td>Australia </td><td><em>B. napus</em> </td><td>Nuseed </td><td>Cultivar </td></tr><tr><td>RSO94-67 (98-18) </td><td>Unknown </td><td><em>B. napus</em> </td><td>Unknown </td><td>Breeding line </td></tr><tr><td>Scoop </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Seetha<sup>a</sup> </td><td>India </td><td><em>B. juncea</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Shiralee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Skipton </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>StormTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific seeds </td><td>Cultivar </td></tr><tr><td>Surpass400<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass402CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass404CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass501TT<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Surpass603CL<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tarcoola </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Tatyoon </td><td>Australia </td><td><em>B. napus</em> </td><td>AgVic </td><td>Cultivar </td></tr><tr><td>TawrifficTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Nugrain </td><td>Cultivar </td></tr><tr><td>TERI(OO)R9903<sup>a</sup> </td><td>India </td><td><em>B. napus</em> </td><td>TERI </td><td>Breeding line </td></tr><tr><td>ThunderTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>TornadoTT </td><td>Australia </td><td><em>B. napus</em> </td><td>Pacific Seeds </td><td>Cultivar </td></tr><tr><td>Tranby </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>WarriorCL </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWDPI </td><td>Cultivar </td></tr><tr><td>Wesbarker<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesreo </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Wesroona<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>AgWA </td><td>Cultivar </td></tr><tr><td>Yickadee<sup>a</sup> </td><td>Australia </td><td><em>B. napus</em> </td><td>NSWA </td><td>Cultivar </td></tr><tr><td>Yu 178 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr><tr><td>Zhongyou 821 </td><td>China </td><td><em>B. napus</em> </td><td>Unknown </td><td>Cultivar </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>BGRC, Institut fur Pflanzenbau und Pflanzenzuchtung, Braunschweig (Germany); DSV, Deutsche Saatveredelung (Germany); AgWA, Department of Agriculture, Western Australia; NSWA, NSW Department of Agriculture (now NSW Department of Primary Industries - NSWDPI); USDA, United State Department of Agriculture; AgVic, Victorian Department of Agriculture.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Genotypes were also used for testing suitability of DArT in differentiation of different species.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Genotypes are winter/semi-spring types and unmarked are spring types.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7387272" target="_blank">View Large</a></div></div><p>DArT markers were tested for their performance in linkage mapping using a new <em>B. napus</em> doubled haploid (DH) mapping population named ‘BnaLMDH’.<sup><a reveal-id="DSR041C44" data-open="DSR041C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The founding parents of this population were ‘Lynx-037DH’ and ‘Monty-028DH’; these DH lines were derived from the open-pollinated spring <em>B. napus</em> varieties ‘Lynx’ and ‘Monty’, respectively, using the microspore culture methodology described by Cousin and Nelson.<sup><a reveal-id="DSR041C45" data-open="DSR041C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Lynx is a high oleic, low linolenic, and European spring variety; seed was provided by O. Sass (Norddeutsche Pflanzenzucht Hans-Georg Lembke KG, Hohenlieth, Germany). Monty is an Australian spring variety with typical canola quality seed oil characteristics;<sup><a reveal-id="DSR041C46" data-open="DSR041C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Monty-028DH seed was provided by Canola Breeders Western Australia Pty Ltd (Perth, Australia). The DH parental lines Lynx-037DH and Monty-028DH were crossed reciprocally to produce F<sub>1</sub> seeds. A single F<sub>1</sub> plant from each reciprocal cross was used as microspore donor for microspore culture using the same methods as used for the parental DH lines production. Self-seed of the primary DH progeny were multiplied either during the summer of 2004–05 at Manjimup Horticultural Research Centre (Manjimup, Australia) or during the winter of 2005 at The University of Western Australia's Shenton Park field station (Perth, Australia).</p>                            <h3 scrollto-destination=7387274 id="7387274" class="section-title" data-legacy-id=s2b>2.2. DNA isolation</h3>
<p>DNA was obtained from 131 ‘BnaLMDH’ lines along with parental and F<sub>1</sub> controls using a standard CTAB method<sup><a reveal-id="DSR041C44" data-open="DSR041C44" class="link link-ref link-reveal xref-bibr">44</a></sup> or using Illustra Nucleon Phytopure Genomic DNA Extraction Kits (GE Healthcare).</p>                            <h3 scrollto-destination=7387276 id="7387276" class="section-title" data-legacy-id=s2c>2.3. Development of <em>B. napus</em> DArT array</h3>
<p>Initially, the most frequently used seven methods of complexity reduction (all based on a combination of <em>Pst</em>I RE and a single ‘frequently cutting’ RE) were tested in several <em>Brassica</em> accessions by resolving products of representation amplification on 1.2% agarose gel. After this initial test, two methods with the most heterodispersed (no observable banding) smears of PCR products were selected: <em>Pst</em>I<em>/Bst</em>NI and <em>Pst</em>I<em>/Ban</em>II (data not reported). The DArT markers were designated with the prefix ‘<em>BrPb</em>’ where ‘<em>Br</em>’ indicates for Brassica, ‘<em>P</em>’ for <em>Pst</em>I (primary enzyme used) and ‘<em>b</em>’ for <em>Ban</em>II (secondary RE used) followed by a number corresponding to their unique clone ID.</p>                            <h3 scrollto-destination=7387278 id="7387278" class="section-title" data-legacy-id=s2d>2.4. Preparation of genomic representations</h3>
<p>The <em>Pst</em>I/<em>Bst</em>NI genomic representations were prepared as described before for barley,<sup><a reveal-id="DSR041C32" data-open="DSR041C32" class="link link-ref link-reveal xref-bibr">32</a></sup> while <em>Pst</em>I/<em>Ban</em>II representations were prepared similarly as for sorghum according to a previous report.<sup><a reveal-id="DSR041C36" data-open="DSR041C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Briefly, ∼50 ng of genomic DNA was digested with either <em>Pst</em>I and <em>Bst</em>NI or <em>Pst</em>I and <em>Ban</em>II RE combinations and resulting fragments ligated to a <em>Pst</em>I overhang compatible oligonucleotide adapter. A primer annealing to this adapter was used in PCR reaction to amplify complexity reduced representation of a sample. Amplification products were either used for cloning in marker development process or labelled with fluorescent dyes and hybridized to DArT array in genotyping process.</p>                            <h3 scrollto-destination=7387280 id="7387280" class="section-title" data-legacy-id=s2e>2.5. Library construction and array printing</h3>
<p>For library construction, we used 107 <em>Brassica</em> genotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Amplified <em>Pst</em>I restriction fragments from all accessions were cloned into pCR2.1-TOPO vector (Invitrogen, Australia) and 10 libraries were generated (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) as described by Jaccoud <em>et al.</em><sup><a reveal-id="DSR041C31" data-open="DSR041C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The white colonies containing genomic fragments inserted into pCR2.1-TOPO vector were picked into individual wells of 384-well microtitre plates filled with ampicillin/kanamycin-supplemented freezing medium. There were 7680 clones in <em>Pst</em>I<em>/Bst</em>NI and 3840 clones in <em>Pst</em>I<em>/Ban</em>II libraries (a total of 11 520 clones). Inserts from these clones were amplified using M13F and M13R primers in 384 plate format, a subset of PCR products were assessed for quality (10% of 25 µl PCR reaction) through gel electrophoresis and all remaining PCR products dried, washed and dissolved in a spotting buffer as described previously.<sup><a reveal-id="DSR041C47" data-open="DSR041C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Microarrays were printed with spot duplication on SuperChip poly-<span class="small-caps">l</span>-lysine slides (Thermo Scientific, Australia) using a MicroGrid arrayer (Genomics Solutions, UK).</p>                            <h3 scrollto-destination=7387282 id="7387282" class="section-title" data-legacy-id=s2f>2.6. DArT genotyping</h3>
<p>Each sample was assayed using methods described above for library construction. Genomic representations were assessed for quality through gel electrophoresis in 1.3% agarose and labelled with fluorescent dyes (Cy3 and Cy5) as described previously.<sup><a reveal-id="DSR041C32" data-open="DSR041C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Labelled targets were than hybridized to printed DArT arrays for 16 h at 62°C in a water bath. Slides were washed as described by Kilian <em>et al</em>.,<sup><a reveal-id="DSR041C47" data-open="DSR041C47" class="link link-ref link-reveal xref-bibr">47</a></sup> dried initially by centrifugation at 500 × <em>g</em> for 7 min and later in a desiccator under vacuum for 30 min. The slides were scanned using Tecan LS300 scanner generating three images per array: one image scanned at 488 nm for reference signal measures the amount of DNA within the spot based on hybridization signal of FAM-labelled fragment of a TOPO vector multiple cloning site fragment and two images for ‘target’ signal measurement: one scanned at 543 nM (for Cy3-labelled targets) and one at 633 nm (for Cy-5-labelled targets).</p>                            <h3 scrollto-destination=7387284 id="7387284" class="section-title" data-legacy-id=s2g>2.7. Array image processing and polymorphism scoring</h3>
<p>All the images were analysed with the DArTsoft v. 7.4.7 (DArT P/L, Canberra, Australia) software. The same software was used to score polymorphic markers in a binary manner (for the presence of marker in the representation as ‘1’ and for the absence as ‘0’) as described previously.<sup><a reveal-id="DSR041C32" data-open="DSR041C32" class="link link-ref link-reveal xref-bibr">32</a></sup> For quality control, 30% of genotypes were genotyped in full technical replication. Clones with <em>P</em> &gt; 77%, a call rate &gt;97%, and 100% allele-calling consistency across the replicates were selected as markers. <em>P</em>-value represents the allelic-states variance of the relative target hybridization intensity as a percentage of the total variance.<sup><a reveal-id="DSR041C34" data-open="DSR041C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The informativeness of the DArT markers was determined by calculating the polymorphism information content (PIC), within the panel of diverse accessions (Table <a reveal-id="DSR041TB1" data-open="DSR041TB1" class="link link-reveal link-table xref-fig">1</a>) according to Anderson <em>et al.</em><sup><a reveal-id="DSR041C48" data-open="DSR041C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p>                            <h3 scrollto-destination=7387286 id="7387286" class="section-title" data-legacy-id=s2h>2.8. Genetic diversity analysis</h3>
<p>Pair-wise genetic similarity matrix was calculated from binary DArT matrices using the Jaccard's coefficient (<em>J</em>) where <em>J</em> = <em>a/(n – d</em>) and ‘<em>a</em>’ is the number of fragments in common, ‘<em>d</em>’ is the number of fragments absent, and ‘<em>n</em>’ is the total number of DArT loci scored.<sup><a reveal-id="DSR041C49" data-open="DSR041C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Similarities matrices were converted to genetic distances according to Swofford and Olson.<sup><a reveal-id="DSR041C50" data-open="DSR041C50" class="link link-ref link-reveal xref-bibr">50</a></sup> The dendrogram was generated based on Jaccard's coefficient with Unweighted Pair-Group Method using Arithmetic average (UPGMA) method in Sequential, Agglomerative, Hierarchical and Nested Clustering module<sup><a reveal-id="DSR041C51" data-open="DSR041C51" class="link link-ref link-reveal xref-bibr">51</a></sup> of Numerical Taxonomy and Multivariate Analysis System for personal computers software (NTSYS-pc), version 2.21.<sup><a reveal-id="DSR041C52" data-open="DSR041C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The marker data were processed using the SIMINT module in NTSYS to compute a correlation among the columns for the 1000 bootstrap samples. The dendrogram was visualized using the program TREEVIEW implemented in NTSYSpc. The MXCOMP subroutine was used to calculate a cophenetic correlation matrix between the genetic similarity matrix and the tree matrix to measure goodness-of-fit implemented in the NTSYSpc package. Multivariate principle component analysis (PCA) based on genetic similarity matrices were used to resolve the genetic relationship and overall diversity among different accessions.</p>                            <h3 scrollto-destination=7387288 id="7387288" class="section-title" data-legacy-id=s2i>2.9. Linkage map construction</h3>
<p>The ‘BnaLMDH’ mapping population of 131 DH lines along with parental and F<sub>1</sub> controls were genotyped with the initial marker discovery DArT array (Version 1.0). Highly polymorphic DArT clones in the set of diverse rapeseed germplasm (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were selected and arrayed for routine DArT analysis (Version 2.0). A subset of 91 DH lines was also genotyped with Version 2.0 DArT array as described previously.</p><p>SSR primer sequences were obtained from Lowe <em>et al</em>.<sup><a reveal-id="DSR041C53" data-open="DSR041C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and from the Agriculture and AgriFood Canada Brassica Microsatellite Consortium (for more information, see <a class="link link-uri" href="http://brassica.agr.gc.ca">http://brassica.agr.gc.ca</a>). Genetic map locations for the majority of these markers were reported previously.<sup><a reveal-id="DSR041C54" data-open="DSR041C54" class="link link-ref link-reveal xref-bibr">54</a></sup> PCR and fragment analysis methods were as described by Nelson <em>et al</em>.<sup><a reveal-id="DSR041C55" data-open="DSR041C55" class="link link-ref link-reveal xref-bibr">55</a></sup> In addition, intron polymorphism (IP) primer sequences and methods were reported by Panjabi <em>et al.</em><sup><a reveal-id="DSR041C56" data-open="DSR041C56" class="link link-ref link-reveal xref-bibr">56</a></sup> New gene-based markers were developed based on publicly available cDNA sequences for fatty acid desaturase and FLC genes.<sup><a reveal-id="DSR041C44" data-open="DSR041C44" class="link link-ref link-reveal xref-bibr">44</a></sup> All polymorphic markers were scored using genotype codes ‘A’ (Lynx-037DH allele), ‘B’ (Monty-028DH allele), and ‘–’ (missing value).</p><p>Linkage mapping was conducted with the aid of MultiPoint 2.1 (MultiQTL Ltd, Haifa, Israel), which uses the ‘evolutionary optimization strategy’.<sup><a reveal-id="DSR041C57" data-open="DSR041C57" class="link link-ref link-reveal xref-bibr">57</a></sup> We followed the general approach described by Nelson <em>et al.</em><sup><a reveal-id="DSR041C58" data-open="DSR041C58" class="link link-ref link-reveal xref-bibr">58</a></sup> with minor modifications. Markers showing significant allele segregation distortion (i.e. diverging from the Mendelian expectation of 1A:1B) were excluded from the analysis with DArT markers (which were scored in a dominant manner) and treated more strictly than the other markers types (which were codominant). DArT markers with very severe segregation distortion (χ<sup>2</sup>, <em>P</em> &lt; 0.0001) were discarded before commencing linkage mapping on the basis that these may represent &gt;1 locus, while DArT markers with less extreme segregation distortion (χ<sup>2</sup>, <em>P</em> &lt; 0.001) were included in the analysis but only as ‘attached’ markers (see explanation of this term below). Some codominant SSR, IP, and gene-based markers with very significant segregation distortion (χ<sup>2</sup>, <em>P</em> &lt; 0.0001) were included in the linkage analysis but were only permitted to act as ‘attached’ markers.</p><p>Iterative clustering analysis began at recombination frequency: rf = 0.14 and was increased at 0.02 increments to a maximum of 0.30. Linkage groups were assigned chromosome names and orientated using microsatellite markers mapped in reference populations.<sup><a reveal-id="DSR041C54" data-open="DSR041C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Recombination frequencies between high-quality ‘framework’ markers were transformed to Kosambi distances in centiMorgans. ‘Redundant’ markers (those with identical map positions to their respective framework markers) and lower quality ‘attached’ markers (those with less well-supported map positions) were integrated into the most appropriate positions in framework marker map. As a quality control measure, marker genotyping scores that introduced singletons into the genotyping matrix of framework markers were re-checked to distinguish apparently true double crossover events from scoring errors. Care was taken not to over-correct the data such that apparently true double crossover events were retained in the scoring matrix. After correcting clear marker scoring errors, genetic interval sizes were recalculated.</p>                            <h2 scrollto-destination=7387293 id="7387293" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7387294 id="7387294" class="section-title" data-legacy-id=s3a>3.1. Assessment of genetic diversity</h3>
<p>A total of 1547 high-quality DArT markers were polymorphic in a set of 89 contemporary cultivars and elite rapeseed lines from breeding programmes. The call rate ranged from 78.9 to 100% with an average of 96.7% and scoring reproducibility was 100% for all selected markers. The PIC values of individual DArT markers ranged from 0.02 to 0.5 (original data not shown), with an average of 0.3. Similarity coefficients (<em>J</em>) based on the binary matrix between individual lines ranged from 0.50 to 0.93.</p><p>The hierarchical cluster analysis discriminated all 89 genotypes of rapeseed into three major clusters (I, II, and III; Fig. <a reveal-id="DSR041F1" data-open="DSR041F1" class="link link-reveal link-table xref-fig">1</a>), but clear differences were observed within these clusters, which were generally consistent with their phenology and genetic lineage. The cluster I consisted of a large number of accessions requiring vernalization for flowering such as ‘Fan023’, ‘Fan 028’, ‘Fan168’, ‘Qu1104’, ‘Yu178’, ‘Zhongyou821’, ‘P624’, ‘03-p74-6’, ‘04-p34’, ‘Carousel-10’, ‘Major’, ‘Expander’, ‘Drakker’, and ‘Mutu 98-1’ (Fig. <a reveal-id="DSR041F1" data-open="DSR041F1" class="link link-reveal link-table xref-fig">1</a>). These varieties originated from China, France, and Japan (Table <a reveal-id="DSR041TB1" data-open="DSR041TB1" class="link link-reveal link-table xref-fig">1</a>). Clusters II and III consisted of the Australian cultivars or their derivatives. In Cluster III, ‘RocketCL’, ‘Surpass400’, ‘Surpass402CL’, ‘Surpass404CL’, ‘Surpass501TT’, ‘Surpass603CL’, ‘Hyola 50’, and ‘Hyola60’ (bred by Pacific Seeds); ‘Marnoo’ and ‘Tatyoon’ (bred by Department of Primary Industries, Victoria); and ‘TawrifficTT’ (bred by Nuseed) showed strong grouping. Several subclades were evident in cluster II, and comprised a range of rapeseed cultivars bred by different Australian breeding programmes (Table <a reveal-id="DSR041TB1" data-open="DSR041TB1" class="link link-reveal link-table xref-fig">1</a>), as many of them share the same parents in their pedigrees.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> The cultivars released by Pioneer: ‘46C40’, ‘45C05’, ‘46C76’, ‘44C73’, ‘46Y78’, ‘45Y77’, and ‘44Y06’ showed tight genetic relationship with ‘Ag-Spectrum’ and ‘Rainbow’ and grouped together in subclade in cluster II. Likewise, a discrete grouping of triazine tolerant cultivars (‘ATR’- and ‘TT’) was also evident in cluster II. These groupings were strongly related to geographic origin, parentage and selection history (http:/<a class="link link-uri" href="http://www.ipaustralia.gov.au/pbr/">www.ipaustralia.gov.au/pbr/</a>). For example, ‘45C05’ and ‘46C76’ had Rainbow and Dunkeld in their pedigrees, respectively. Despite being of the same parentage ‘Major’ and ‘Wesreo’ (Major/Oro) varieties did not show tighter grouping, and may be heterogeneous or heterozygous. </p>                        <a id="7387297" scrollto-destination="7387297"></a>
<div id="DSR041F1" data-id="DSR041F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04101.jpeg?Expires=2147483647&Signature=h-yXmn9NuEzNCmIZoHKGeQTbeMZt0LnYMWp4B9UyB~Ao5dAlDtMxhIDPMjO9~bjz6DT4NFwxVyNoEC9IzYS~CMT4t~2iPShIoRDP9nqaY6LGlTaIR17vUVyaKQkZh0dvsyNbroTGM6kL~lXkT3IL4xtZw2M5QkGyyt5EZIUkjUrLs-6pStaEPETVt-BaUvhjNkFQRjApUm2L2Z7xeLRYmJcoNsLhpusLX2jEyDG4NqpVwHUFZYmOub0PEwh0Yms7IxUHMCmEWHLUg6bPeWVk2l4elpzhdBIT0kwE8bhIJoOcat4T5Kkn5a3GWSNgP0KjAHDnd2UJ8u0DdtjImfQkbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendrogram showing genetic similarity among 89 accessions of oilseed rape based upon the binary matrix obtained from 1547 DArT polymorphic loci, generated using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient in SAHN module of the NTSYSpc version 2.21h." path-from-xml="dsr04101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387297/dsr04101.jpeg" path-from-xml="dsr04101.jpeg" target="_blank">View large</a><a section="7387297" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04101.jpeg?Expires=2147483647&Signature=IEeHBycCLDVw9weWLdKODQA9r4~2z~8nCYZkuT--vF52FCRzee5wJMbDx~XqRC77sDJS4BGdgiZD3P5rbAywY5EqalmEJ5cGSVZmvkF3SYIDs9TV4siOxZpyZzmdcHBcV0hntvADn6oa0wP2NmBgvnKbQYZDkHAMJ9s0a~Ix97N9hUHw~ZSS7rx19Xtt7wm0rpayvwBlrcvTx3lljDkNJyeY525VwIF4cLtsmsBkRzi5pIBd4Ja2ney6jz0cGsHy6OVV40a4YGPHa3WEJyr0cPtiGVhEW-MXOpxuAHhIOllH8PMbAHgVJsOoLaf6U6jFNKZllsDfFV3J8MVZPZotPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7387297&ar=485713&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dendrogram showing genetic similarity among 89 accessions of oilseed rape based upon the binary matrix obtained from 1547 DArT polymorphic loci, generated using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient in SAHN module of the NTSYSpc version 2.21h.</p></div></div><div content-id="DSR041F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04101.jpeg?Expires=2147483647&Signature=h-yXmn9NuEzNCmIZoHKGeQTbeMZt0LnYMWp4B9UyB~Ao5dAlDtMxhIDPMjO9~bjz6DT4NFwxVyNoEC9IzYS~CMT4t~2iPShIoRDP9nqaY6LGlTaIR17vUVyaKQkZh0dvsyNbroTGM6kL~lXkT3IL4xtZw2M5QkGyyt5EZIUkjUrLs-6pStaEPETVt-BaUvhjNkFQRjApUm2L2Z7xeLRYmJcoNsLhpusLX2jEyDG4NqpVwHUFZYmOub0PEwh0Yms7IxUHMCmEWHLUg6bPeWVk2l4elpzhdBIT0kwE8bhIJoOcat4T5Kkn5a3GWSNgP0KjAHDnd2UJ8u0DdtjImfQkbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendrogram showing genetic similarity among 89 accessions of oilseed rape based upon the binary matrix obtained from 1547 DArT polymorphic loci, generated using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient in SAHN module of the NTSYSpc version 2.21h." path-from-xml="dsr04101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387297/dsr04101.jpeg" path-from-xml="dsr04101.jpeg" target="_blank">View large</a><a section="7387297" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04101.jpeg?Expires=2147483647&Signature=IEeHBycCLDVw9weWLdKODQA9r4~2z~8nCYZkuT--vF52FCRzee5wJMbDx~XqRC77sDJS4BGdgiZD3P5rbAywY5EqalmEJ5cGSVZmvkF3SYIDs9TV4siOxZpyZzmdcHBcV0hntvADn6oa0wP2NmBgvnKbQYZDkHAMJ9s0a~Ix97N9hUHw~ZSS7rx19Xtt7wm0rpayvwBlrcvTx3lljDkNJyeY525VwIF4cLtsmsBkRzi5pIBd4Ja2ney6jz0cGsHy6OVV40a4YGPHa3WEJyr0cPtiGVhEW-MXOpxuAHhIOllH8PMbAHgVJsOoLaf6U6jFNKZllsDfFV3J8MVZPZotPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7387297&ar=485713&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dendrogram showing genetic similarity among 89 accessions of oilseed rape based upon the binary matrix obtained from 1547 DArT polymorphic loci, generated using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient in SAHN module of the NTSYSpc version 2.21h.</p></div></div><p>Ultrametric cophenetic and genetic distance matrices that were used to generate the phylogenetic tree using the hierarchical clustering method, showed a ‘good’ fit (<em>Z</em>: cophenetic correlation coefficient = 0.8), indicating that the cophenetic distance is in congruence with the distance matrices obtained from the DArT marker data (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Three principal components (first three axes), explaining 25% of variation, were sufficient to represent most of the structured information generated with multivariate analysis and reconfirmed the groupings of different accessions resultant from the UPGMA-based phenogram (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p><p>In order to test if the current DArT array is suitable for genetic analysis of other <em>Brassica</em> species, we analysed a subset of 32 genotypes representing <em>B. rapa</em>, <em>B. juncea</em>, <em>B. carinata</em> and investigated their genetic relationships in relation to selected <em>B. napus</em> cultivars representing ‘spring’ and ‘winter’ types from different breeding programmes (Table <a reveal-id="DSR041TB1" data-open="DSR041TB1" class="link link-reveal link-table xref-fig">1</a>). Both cluster and PCA analyses differentiated different species of <em>Brassica</em>, as expected (Fig. <a reveal-id="DSR041F2" data-open="DSR041F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). For example, cultivars/breeding lines of <em>B. napus</em>, <em>B. rapa</em>, <em>B.</em><em>carinata</em>, and <em>B. juncea</em> clustered into distinct I–III groups. Within the <em>B. napus</em> cluster, two distinct subclades representing ‘winter and semi-spring cultivars’ (‘Carousel-10’, ‘Major’, ‘Norin 22’, ‘Iwao natame’, and ‘Mutu’) and ‘spring cultivars’ were clearly evident. Clustering of genotypes was consistent with their pedigrees and/or their origin (breeding programmes). Likewise, ‘Ag-Outback’ and ‘Ag-Muster’, and cultivars with suffix ‘Surpass’ share ancestry among themselves. For a general overview of Australian canola variety pedigrees, see Cowling.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> PCA of DArT data of 32 accessions also supported the grouping identified by cluster analysis. The top three principal components (dimensions) explained 58.6% of the genetic variation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). </p>                        <a id="7387300" scrollto-destination="7387300"></a>
<div id="DSR041F2" data-id="DSR041F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04102.jpeg?Expires=2147483647&Signature=R3rG5G4X0e9N4qitSEBipVEEIRsJ4xfXhyPL-lxzFhrR6T~K7V5jh5bj-6DekDLa2NOyyEnMpKVIZ65AqKIqkWHrJgrocgxMiSf0LX~44-sFl0tE-bbxrKHYjt7oP-~xO4s0f7hXNNZ-4FJo6AB2JbxbhEx82ZXq6R4pDvib5hjFw~eDE9cfFydtSUqD7RhcH3gH1qmhfTZuqIWLxl4xlaEkmQxLlmbWkMbHcg4dm8o4TAsJwd77WqCSIVW7EA8QNMW23SAl378xEZnTFrzX2TtG4nb~6-5A2VcXl~hr3nRZGl6adyZ6sDCLEdICbefpDs8a18K492SMijBf1BP6vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendrogram showing the genetic relationship among 32 accessions of B. napus, B. rapa, B. juncea, and B. carinata. Binary matrix obtained from DArT polymorphisms were used to generate dendrogram using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient." path-from-xml="dsr04102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387300/dsr04102.jpeg" path-from-xml="dsr04102.jpeg" target="_blank">View large</a><a section="7387300" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04102.jpeg?Expires=2147483647&Signature=eV26wxBajiGItbHcDaGDNIeC0JltI8HGtheGE9cdiGyxTTQUf1BdngWXTD2XYNW0fPDMA3qTLA8xEsSvJn1TGWOCmmPzktvWywbJxZD-Onjk~ESb9uXYtCco7EbfchBTwfG1Owz6WakRj~Q~Axovz7mg6y60l5GFvPYeVuxqVQrjGyZOkYlv2X8cw1CVJrUUkzRILInBJODCGfnr7KC1L5tjLgpb~42ZE7nxHwUtlafLzwY6AXBLonzyy6y6XosuYywsJRjbro0XVIADYklgKq-XOg3AKnuAC~sGTGGI2wkeYn9hyDaIg9yy-MKqzt3u1K14AAAOp7T9EERX3eqIzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7387300&ar=485713&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dendrogram showing the genetic relationship among 32 accessions of <em>B. napus</em>, <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. carinata.</em> Binary matrix obtained from DArT polymorphisms were used to generate dendrogram using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient.</p></div></div><div content-id="DSR041F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04102.jpeg?Expires=2147483647&Signature=R3rG5G4X0e9N4qitSEBipVEEIRsJ4xfXhyPL-lxzFhrR6T~K7V5jh5bj-6DekDLa2NOyyEnMpKVIZ65AqKIqkWHrJgrocgxMiSf0LX~44-sFl0tE-bbxrKHYjt7oP-~xO4s0f7hXNNZ-4FJo6AB2JbxbhEx82ZXq6R4pDvib5hjFw~eDE9cfFydtSUqD7RhcH3gH1qmhfTZuqIWLxl4xlaEkmQxLlmbWkMbHcg4dm8o4TAsJwd77WqCSIVW7EA8QNMW23SAl378xEZnTFrzX2TtG4nb~6-5A2VcXl~hr3nRZGl6adyZ6sDCLEdICbefpDs8a18K492SMijBf1BP6vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendrogram showing the genetic relationship among 32 accessions of B. napus, B. rapa, B. juncea, and B. carinata. Binary matrix obtained from DArT polymorphisms were used to generate dendrogram using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient." path-from-xml="dsr04102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387300/dsr04102.jpeg" path-from-xml="dsr04102.jpeg" target="_blank">View large</a><a section="7387300" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04102.jpeg?Expires=2147483647&Signature=eV26wxBajiGItbHcDaGDNIeC0JltI8HGtheGE9cdiGyxTTQUf1BdngWXTD2XYNW0fPDMA3qTLA8xEsSvJn1TGWOCmmPzktvWywbJxZD-Onjk~ESb9uXYtCco7EbfchBTwfG1Owz6WakRj~Q~Axovz7mg6y60l5GFvPYeVuxqVQrjGyZOkYlv2X8cw1CVJrUUkzRILInBJODCGfnr7KC1L5tjLgpb~42ZE7nxHwUtlafLzwY6AXBLonzyy6y6XosuYywsJRjbro0XVIADYklgKq-XOg3AKnuAC~sGTGGI2wkeYn9hyDaIg9yy-MKqzt3u1K14AAAOp7T9EERX3eqIzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7387300&ar=485713&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr04102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dendrogram showing the genetic relationship among 32 accessions of <em>B. napus</em>, <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. carinata.</em> Binary matrix obtained from DArT polymorphisms were used to generate dendrogram using the unweighted pair-group method with arithmetic mean analysis and Jaccard's coefficient.</p></div></div>                            <h3 scrollto-destination=7387301 id="7387301" class="section-title" data-legacy-id=s3b>3.2. Identifying single-copy DArT markers for linkage mapping</h3>
<p>In the 131 DH lines screened with the DArT array Version 1, 1171 DArT markers passed the standard DArT quality criteria. These criteria include minimal call rate for markers (&gt;80%) and high reproducibility of technical replicates (average scoring consensus &gt;99.7%). Given that <em>B. napus</em> is an allotetraploid species with strong residual homoeologous relationships among chromosomes, it was considered likely that some DArT markers would detect pairs of homoeologous loci rather than single loci. Therefore, χ<sup>2</sup> tests were used to test each DArT marker for goodness-of-fit for a one-locus model (1:1 segregation ratio of parental alleles) and for two-locus models (1:3 or 3:1 ratio of parental alleles). Only those fitting the single-locus model were retained for linkage mapping purposes. Of 1171 DArT markers scored, 644 were monomorphic, 415 had allelic ratios consistent with the one-locus model (1:1; <em>P</em> &lt; 0.001), and 112 had allelic ratios consistent with the two-locus model (3:1 or 1:3; <em>P</em> &lt; 0.001). In the 91 DH lines screening the Version 2 array, 1020 DArT markers passed the standard DArT quality criteria (as above). Of these, 552 were monomorphic, 351 had allelic ratios consistent with the one-locus model (1:1; <em>P</em> &lt; 0.001) and 117 had allelic ratios consistent with the two-locus model (3:1 or 1:3; <em>P</em> &lt; 0.001). Markers that were putatively single-locus were combined from both DArT analyses to give 444 DArT loci that were used for subsequent linkage mapping. There were few inconsistencies between independent DArT analyses, which were entered as missing values in the combined DArT marker set.</p>                            <h3 scrollto-destination=7387303 id="7387303" class="section-title" data-legacy-id=s3c>3.3. Linkage mapping</h3>
<p>In total, 584 marker loci (comprising 437 DArT, 135 SSR, 6 IP, and 6 gene-based markers) were used to generate a <em>B. napus</em> linkage map based on the BnaLMDH mapping population (Fig. <a reveal-id="DSR041F3" data-open="DSR041F3" class="link link-reveal link-table xref-fig">3</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>; all scoring data provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>, and summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Of these 584 loci, 329 were high-quality, non-redundant ‘framework’ markers, 209 were ‘redundant’ markers (i.e. co-segregated with respective framework markers), and 46 were lower quality ‘attached’ markers with less well-defined map positions. In consistent with the haploid chromosome number for <em>B. napus</em> (<em>n</em> = 19), there were 19 linkage groups with the shortest linkage group (A08) being 61.3 cM, the longest (C4) being 186.2 cM, and a total map length of 2288 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). There were also two small clusters (Cluster-1 and Cluster-2) comprising four and three loci, respectively. There were also four markers (brPb-661033, brPb-662131, brPb-809033, and sN2675) that remained unlinked. </p>                        <a id="7387305" scrollto-destination="7387305"></a>
<div id="DSR041F3" data-id="DSR041F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04103a.jpeg?Expires=2147483647&amp;Signature=J5MCDP3KVcX3wH7jLL9AZoW6TizAwAx70jUgp1XHr60DiUbAbbh70N78K2daWO6djUfMjUKN8gA6U92sa~D5ly3UCZQBmcELO3raCYYjXNSWk3E7T~91A-HiAP3m5Ho3iHcV6AVhBVZgId6rT9cAuzEoVIlI8hKNx2cVISGsYvT0NvSKIY4LuJAWHx7jIfhXGpStGcXI0u2pfSt1X2Yo5l4tcJD2zcBgKMpPha~W-YkJKaNqZ9Kpn0oDdLEhmT7LbBvCdj-aRxFOSYzTSOaozP25C2d0Dfsu-lFX-VMvqU0lV0XmyMj1g9OJj6mxc9V-vFY55nxK2-MaWR9OxY9B8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups." path-from-xml="dsr04103a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387305/dsr04103a.jpeg" path-from-xml="dsr04103a.jpeg" target="_blank">View large</a><a section="7387305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04103a.jpeg?Expires=2147483647&amp;Signature=RZbCCh2XxBECI2DcB6b6qPTAdFB36Sr8VV6EGAtjH0U-S82~MZV4TwxdXPktv5FMXGY-brfW8Rn1pxEikZDJoOLeg1k1M7E4e6MKK94ubg9HtSGN~7vngDj9q~h1ABTfnmBbXJAw8Ib8j6CHOIVg6spKKQR7xKu5vcexieT0Rqy7tC7EHlq1v6V1zNKd0mKij40K2yeLOmvfJRILzUiK8R~R4kg5qjYtPTeLQ26Cw3kyqsAOJiy9-1yNQr1hO4yyB5KFy1gipJNYtY7gHuBHojTas~8w~u2B9RbMyYNHYGXOCGubDs-xjuhkso2ETY8H-pTzHeM5dmRtVYFJUswWBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7387305&amp;ar=485713&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr04103a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04103b.jpeg?Expires=2147483647&amp;Signature=gqon-b9SfNJLRCpEWzRG0zOzLtrHuf-Qj-ZC4ujL~YEsKIHzrQAds63WNDsN6Eb~7Wu1Ub3~f8emR3rd7XVQoO7z9jPbhGaQbxOdTXQpbFl70nmrzxSM6McM-ijgXoP2ZlqO7SgKIskWdWiXgn8BzXs9AWNpyQCxyVA3VXOe3~PCEF-lRqy-xLaVWmnv2FO5jkklAyttzkvNZMl4wbW-DGA5G8Hbly5PA~F1b05HEpPhPGI3PczkaETxCTGzJd5lCyslX0pG-RceG1VW89ZdcGLhqV-gpqq3Gmk2nOMeTpScSlN2r4Lm7m6TOa2z3gMWhxBICob5sfcu1nOVueOO0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups." path-from-xml="dsr04103b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387305/dsr04103b.jpeg" path-from-xml="dsr04103b.jpeg" target="_blank">View large</a><a section="7387305" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04103b.jpeg?Expires=2147483647&amp;Signature=liXexd4rfmRt3iW3wV0PYwePbdV7ke1U5LUEH2govRylcjHHM2GazMFORCaN5hdZSvFEb44YVrDEJKtOR0V2v-pY1Px~~5bSPWtsLpcr0Wu9jbFe0efp-Ic8bv9pr-fSXDiIBeSzc6~M0d1MHbLn9vOAeOvP5b-Tw3rRaMS4pWeWGAEOMj6-yBE8biI0Mt8sRErV-61WA5WcCaYZppEjsZAeHyWpzes2Ajiuloh9nc6fJdeVHhOaSrHeHKEh4ArI~-1APrx~H~5AS0vX7XSKlDiUNbJmuMn85n6Y7RUZG10kqEZHzeuTl~Vl6xPRryNpyiiYepe6lCWOXtNkkOYV8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7387305&amp;ar=485713&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr04103b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups.</p></div></div><div content-id="DSR041F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04103a.jpeg?Expires=2147483647&amp;Signature=J5MCDP3KVcX3wH7jLL9AZoW6TizAwAx70jUgp1XHr60DiUbAbbh70N78K2daWO6djUfMjUKN8gA6U92sa~D5ly3UCZQBmcELO3raCYYjXNSWk3E7T~91A-HiAP3m5Ho3iHcV6AVhBVZgId6rT9cAuzEoVIlI8hKNx2cVISGsYvT0NvSKIY4LuJAWHx7jIfhXGpStGcXI0u2pfSt1X2Yo5l4tcJD2zcBgKMpPha~W-YkJKaNqZ9Kpn0oDdLEhmT7LbBvCdj-aRxFOSYzTSOaozP25C2d0Dfsu-lFX-VMvqU0lV0XmyMj1g9OJj6mxc9V-vFY55nxK2-MaWR9OxY9B8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups." path-from-xml="dsr04103a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387305/dsr04103a.jpeg" path-from-xml="dsr04103a.jpeg" target="_blank">View large</a><a section="7387305" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04103a.jpeg?Expires=2147483647&amp;Signature=RZbCCh2XxBECI2DcB6b6qPTAdFB36Sr8VV6EGAtjH0U-S82~MZV4TwxdXPktv5FMXGY-brfW8Rn1pxEikZDJoOLeg1k1M7E4e6MKK94ubg9HtSGN~7vngDj9q~h1ABTfnmBbXJAw8Ib8j6CHOIVg6spKKQR7xKu5vcexieT0Rqy7tC7EHlq1v6V1zNKd0mKij40K2yeLOmvfJRILzUiK8R~R4kg5qjYtPTeLQ26Cw3kyqsAOJiy9-1yNQr1hO4yyB5KFy1gipJNYtY7gHuBHojTas~8w~u2B9RbMyYNHYGXOCGubDs-xjuhkso2ETY8H-pTzHeM5dmRtVYFJUswWBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7387305&amp;ar=485713&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr04103a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/m_dsr04103b.jpeg?Expires=2147483647&amp;Signature=gqon-b9SfNJLRCpEWzRG0zOzLtrHuf-Qj-ZC4ujL~YEsKIHzrQAds63WNDsN6Eb~7Wu1Ub3~f8emR3rd7XVQoO7z9jPbhGaQbxOdTXQpbFl70nmrzxSM6McM-ijgXoP2ZlqO7SgKIskWdWiXgn8BzXs9AWNpyQCxyVA3VXOe3~PCEF-lRqy-xLaVWmnv2FO5jkklAyttzkvNZMl4wbW-DGA5G8Hbly5PA~F1b05HEpPhPGI3PczkaETxCTGzJd5lCyslX0pG-RceG1VW89ZdcGLhqV-gpqq3Gmk2nOMeTpScSlN2r4Lm7m6TOa2z3gMWhxBICob5sfcu1nOVueOO0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups." path-from-xml="dsr04103b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7387305/dsr04103b.jpeg" path-from-xml="dsr04103b.jpeg" target="_blank">View large</a><a section="7387305" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr04103b.jpeg?Expires=2147483647&amp;Signature=liXexd4rfmRt3iW3wV0PYwePbdV7ke1U5LUEH2govRylcjHHM2GazMFORCaN5hdZSvFEb44YVrDEJKtOR0V2v-pY1Px~~5bSPWtsLpcr0Wu9jbFe0efp-Ic8bv9pr-fSXDiIBeSzc6~M0d1MHbLn9vOAeOvP5b-Tw3rRaMS4pWeWGAEOMj6-yBE8biI0Mt8sRErV-61WA5WcCaYZppEjsZAeHyWpzes2Ajiuloh9nc6fJdeVHhOaSrHeHKEh4ArI~-1APrx~H~5AS0vX7XSKlDiUNbJmuMn85n6Y7RUZG10kqEZHzeuTl~Vl6xPRryNpyiiYepe6lCWOXtNkkOYV8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7387305&amp;ar=485713&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr04103b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical representation of the genetic linkage map constructed in the DH population from Lynx/Monty. Map distances are given in centiMorgan (cM) on the left of sample linkage groups.</p></div></div><p>Using SSR markers with known map locations, all linkage groups were unambiguously assigned chromosome names and orientated relative to the reference map.<sup><a reveal-id="DSR041C54" data-open="DSR041C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Alignment to the reference map revealed that this map encompasses ∼90% of the known <em>B. napus</em> genome with notably incomplete coverage of chromosomes A07 (bottom half) and A08 (top half). DArT marker were unevenly distributed, with the proportion of DArT loci ranging from 31% (chromosome C5) to 90% (chromosome A03), with an average of 70% across all chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>Several regions of the linkage map comprised markers with allelic segregation ratios diverging from the Mendelian expectation (1:1 for a DH population; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The most significantly skewed deviations (χ<sup>2</sup>, <em>P</em> &lt; 0.001) were on linkage groups A01, A03, and A06 and encompassed both DArT and SSR markers. We found evidence of high frequency of homoeologous recombination between chromosomes A07 and C6, as shown by the tendency of A07 and C6 to cluster together during the linkage mapping process, which had to be manually separated.</p>                            <h2 scrollto-destination=7387308 id="7387308" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We developed and applied a high-throughput DArT marker array for the first time in rapeseed. We demonstrated that this whole-genome profiling technology is useful for establishing phylogenetic relationships among varieties and elite breeding <em>Brassica</em> lines and for construction of linkage map that has extensive genome coverage of the rapeseed genome.</p>                            <h3 scrollto-destination=7387310 id="7387310" class="section-title" data-legacy-id=s4a>4.1. DArT markers for genetic diversity assessment</h3>
<p>In this study, we utilized up to 1547 DArT markers for the assessment of genetic diversity among accessions of rapeseed and related species. A number of these markers were distributed across the genome in the ‘BnaLMDH’ population (Fig. <a reveal-id="DSR041F3" data-open="DSR041F3" class="link link-reveal link-table xref-fig">3</a>). Previously, only a limited number of markers were used to analyse genetic diversity in rapeseed. For example, Wang <em>et al.</em><sup><a reveal-id="DSR041C12" data-open="DSR041C12" class="link link-ref link-reveal xref-bibr">12</a></sup> used 18 SSR primer pairs to generate 112 polymorphic features for genetic diversity analysis of 48 Australian rapeseed cultivars while Chen <em>et al.</em><sup><a reveal-id="DSR041C60" data-open="DSR041C60" class="link link-ref link-reveal xref-bibr">60</a></sup> used 55 SSR primer pairs to generate 287 polymorphic features for genetic diversity analysis of 72 cultivars. DArT markers with extensive genome coverage, developed in this study, will provide better estimate of the extent of genetic diversity. DArT markers have the additional advantage of being based largely on SNP variation, which more closely reflects normal genome evolutionary behaviour compared with SSRs that have high rates of mutation.<sup><a reveal-id="DSR041C61" data-open="DSR041C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Therefore, this DArT resource constitutes a significant improvement in marker density, and possibly quality, compared with that possible with marker technology previously available for rapeseed.</p><p>We employed both cluster and PCA analyses for studying genetic diversity and population structure. Results suggested that genetic diversity exists within the rapeseed germplasm. Most of the Australian varieties were grouped into distinct clusters according their common ancestry.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSR041C62" data-open="DSR041C62" class="link link-ref link-reveal xref-bibr">62</a></sup> For example, ‘Yickadee’, ‘Maluka’, ‘Eureka’, ‘Shiralee’, ‘Rainbow’, ‘45C75’, ‘44Y06’, ‘Ag-Spectrum’, ‘46C76’, and ‘45Y77’ share pedigrees with each other<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> (Patel personal communication. <a class="link link-uri" href="http://www.ipaustralia.gov.au/pbr/index.shtml">http://www.ipaustralia.gov.au/pbr/index.shtml</a>). This confirms that gene flow into Australian breeding programmes was restricted before the time of release of these varieties.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Furthermore, all Australian varieties up to the year 2000 were developed from the same set of donor sources collected from Japan, France, Poland, Germany, Sweden, and Canada.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> However, some accessions overlapped or distributed across well-defined clusters, for example ‘Karoo’ (Fig <a reveal-id="DSR041F1" data-open="DSR041F1" class="link link-reveal link-table xref-fig">1</a>). This could be due to the fact that rapeseed (syn canola) is a predominantly self-pollinated crop; however, outcrossing may occur, ranging from 5 to 36% under field conditions.<sup><a reveal-id="DSR041C63" data-open="DSR041C63" class="link link-ref link-reveal xref-bibr">63</a>,<a reveal-id="DSR041C64" data-open="DSR041C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Therefore, heterogeneity and heterozygosity in some varieties, such as ‘Karoo’, is very likely. This phenomena has been reported in previous studies,<sup><a reveal-id="DSR041C11" data-open="DSR041C11" class="link link-ref link-reveal xref-bibr">11</a></sup> especially in the Australian cultivars which were developed using open-pollinated pedigree selection without ‘intentional selfing’.<sup><a reveal-id="DSR041C62" data-open="DSR041C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Furthermore, many breeding programmes have intentionally released ‘genetically mixed’ varieties to cope with environmental variation and biotic stresses.<sup><a reveal-id="DSR041C63" data-open="DSR041C63" class="link link-ref link-reveal xref-bibr">63</a>,<a reveal-id="DSR041C64" data-open="DSR041C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Phenograms also clearly differentiated ‘winter’ and ‘spring’ types, which represent to different gene pools as described previously using RFLP markers.<sup><a reveal-id="DSR041C8" data-open="DSR041C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR041C65" data-open="DSR041C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Our results are consistent with previous studies which showed that rapeseed accessions can be grouped on the basis of flowering habit (winter and spring type), and on the basis of origin by geographical region and breeding organizations.<sup><a reveal-id="DSR041C8" data-open="DSR041C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR041C9" data-open="DSR041C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSR041C12" data-open="DSR041C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR041C65  DSR041C67" data-open="DSR041C65  DSR041C67" class="link link-ref link-reveal xref-bibr">65–67</a></sup></p><p>DArT markers also enabled us to assess genetic diversity among four agricultural <em>Brassica</em> species that are extensively being used in rapeseed improvement programmes. Although we have not sampled extensively these species, DArT markers allowed us to differentiate <em>B. rapa</em> (AA), <em>B. napus</em> (AACC), <em>B. juncea</em> (AABB), and <em>B. carinata</em> (BBCC) on their polymorphism patterns. Therefore, the current DArT platform is suitable for genotyping of different <em>Brassica</em> species especially those having ‘A’ and ‘C’ genomes and may facilitate alien gene introgression from related species.</p>                            <h3 scrollto-destination=7387314 id="7387314" class="section-title" data-legacy-id=s4b>4.2. Implication of diversity</h3>
<p>This DArT marker analysis supports previous studies that show that rapeseed accessions from Australia, Europe, and China are genetically differentiated.<sup><a reveal-id="DSR041C60" data-open="DSR041C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Crossing between these differentiated sources can be used to enlarge genetic diversity in rapeseed breeding programmes.<sup><a reveal-id="DSR041C59" data-open="DSR041C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Genetic distance estimates would assist breeding programmes to maximize the level of variation in their germplasm and to exploit high heterosis in hybrid varieties. DArT allelic profiles produced in this study will also be useful for fast and accurate DNA fingerprinting of elite Australian rapeseed genotypes and identifying loci associated with traits of agronomic importance using association mapping strategy.</p>                            <h3 scrollto-destination=7387316 id="7387316" class="section-title" data-legacy-id=s4c>4.3. Genome-wide coverage of DArT markers</h3>
<p>A genetic map of ‘BnaLMDH’ was constructed using 437 DArT, 135 SSR, 6 IP, and 6 gene-based markers. These markers provided good coverage across all 19 linkage groups, corresponding to all 10 chromosomes of A and 9 chromosomes of C genome of <em>B. napus</em>. Good genome coverage could be partly attributed to the enrichment of the genic regions with the use of <em>Pst</em>I. The map of ‘BnaLMDH’ population covering 2288 cM is comparable with the previous genetic linkage maps of <em>B. napus</em>, spanning distances of 1173–2619 cM.<sup><a reveal-id="DSR041C68  DSR041C70" data-open="DSR041C68  DSR041C70" class="link link-ref link-reveal xref-bibr">68–70</a></sup> Approximately 10% of the DArT loci were found to be duplicated in this study. Therefore, a limited effect of homoplasmy is anticipated on the usage of DArT in diversity analysis in allotetraploid <em>B. napus</em>.</p>                            <h2 scrollto-destination=7387318 id="7387318" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>We developed the DArT platform for genetic analysis of <em>Brassica</em> genomes and demonstrated its usefulness in genetic diversity assessment and genetic map construction. Our results have proved that DArT technology is amenable for various downstream applications such as cluster analysis and map construction, including for trait-marker association analyses.<sup><a reveal-id="DSR041C66" data-open="DSR041C66" class="link link-ref link-reveal xref-bibr">66</a></sup> It can supplement marker systems that cannot be analysed in a highly parallel genotyping format, such as SSR markers. SSRs are suitable to distinguish heterozygotes due to their, in general, co-dominance. Besides, excessive stutter bands and overlapping bands due to coamplification of loci also limit the usefulness of SSR markers in rapeseed. In light of the current paucity of high-density SNP assay formats for rapeseed and the embryonic nature of genotyping by sequencing technologies, the development of the DArT platform for rapeseed is a significant development for identifying trait-marker association, genetic diversity, and population genetics studies and whole-genome selection in this important crop species.</p>                            <h2 scrollto-destination=7387320 id="7387320" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7387322 id="7387322" class="section-title" data-legacy-id=s7>Funding</h2>
<p>Authors are thankful to the NSW Agricultural Genomic Centre funded under BioFirst Initiative of NSW Government, and various organizations for supporting research under the DArT International Rapeseed Consortium led by Dr Kilian. M.N.A. was supported by Australian Research Council project LP0210571, with industry partners Council of Grain Grower Organizations Ltd (Australia) and Norddeutsche Pflanzenzucht Hans-Georg Lembke KG (Germany).</p>                            <h2 scrollto-destination=7387324 id="7387324" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to Ms Belinda Taylor for technical assistance in DNA isolation and Ms Christine Sidebottom for technical support of SSR analysis.</p>                            <h2 scrollto-destination=7387326 id="7387326" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR041C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagaharu</div> <div class="given-names">U</div></div>. </div><div class="article-title">Genomic analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilisation</div>, <div class="source">Jpn. J. Bot.</div> , <div class="year">1935</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="DSR041C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cruz</div> <div class="given-names">V.M.V.</div></div>,  <div class="name"><div class="surname">Luhman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Marek</div> <div class="given-names">L.F.</div></div>, et al. </div><div class="article-title">Characterisation of flowering time and SSR marker analysis of spring and winter type <em>Brassica napus</em> L. germplasm</div>, <div class="source">Euphytica</div> , <div class="year">2007</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">43</div>-<div class="lpage">57</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10681-006-9233-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10681-006-9233-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterisation%20of%20flowering%20time%20and%20SSR%20marker%20analysis%20of%20spring%20and%20winter%20type%20Brassica%20napus%20L.%20germplasm&amp;author=V.M.V.%20Cruz&amp;author=R.%20Luhman&amp;author=L.F.%20Marek&amp;publication_year=2007&amp;journal=Euphytica&amp;volume=153&amp;pages=43-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-006-9233-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-006-9233-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-006-9233-1"> </span></div> </div></div></div></div></div><div content-id="DSR041C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">314</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=R.%20White&amp;author=M.%20Skolnick&amp;author=R.%20Davis&amp;publication_year=1980&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=32&amp;pages=314-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6247908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Abundant class of human DNA polymorphisms which can be typed using the polymerase chain reaction</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1989</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">388</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundant%20class%20of%20human%20DNA%20polymorphisms%20which%20can%20be%20typed%20using%20the%20polymerase%20chain%20reaction&amp;author=J.%20Weber&amp;author=P.%20May&amp;publication_year=1989&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=44&amp;pages=388-96" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2916582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">AFLP: a new technique for DNA fingerprinting</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4407</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/23.21.4407" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/23.21.4407</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=4407-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Williams</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kubelik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Livak</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tingey</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DNA polymorphisms amplified by arbitrary primers are useful as genetic markers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6531</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/18.22.6531" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/18.22.6531</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20polymorphisms%20amplified%20by%20arbitrary%20primers%20are%20useful%20as%20genetic%20markers&amp;author=J.%20Williams&amp;author=A.%20Kubelik&amp;author=K.%20Livak&amp;author=J.%20Rafalski&amp;author=S.%20Tingey&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6531-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1979162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chee</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hubbell</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Accessing genetic information with high-density DNA arrays</div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">274</div> (pg. <div class="fpage">610</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.274.5287.610" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.274.5287.610</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accessing%20genetic%20information%20with%20high-density%20DNA%20arrays&amp;author=M.%20Chee&amp;author=R.%20Yang&amp;author=E.%20Hubbell&amp;publication_year=1996&amp;journal=Science&amp;volume=274&amp;pages=610-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.274.5287.610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.274.5287.610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.274.5287.610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8849452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diers</div> <div class="given-names">B. W.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Genetic diversity of oilseed <em>Brassica napus</em> germplasm based on restriction fragment length polymorphisms</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">662</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01253968" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01253968</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20oilseed%20Brassica%20napus%20germplasm%20based%20on%20restriction%20fragment%20length%20polymorphisms&amp;author=B.%20W.%20Diers&amp;author=T.%20C.%20Osborn&amp;publication_year=1994&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=88&amp;pages=662-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01253968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01253968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01253968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24186160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Baril</div> <div class="given-names">C. P.</div></div>,  <div class="name"><div class="surname">Dubreuil</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Blouet</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Genetic relationships and fingerprinting of rapeseed cultivars by AFLP: consequences for varietal registration</div>, <div class="source">Crop Sci.</div> , <div class="year">2000</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1417</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2135/cropsci2000.4051417x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2135/cropsci2000.4051417x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20relationships%20and%20fingerprinting%20of%20rapeseed%20cultivars%20by%20AFLP%3A%20consequences%20for%20varietal%20registration&amp;author=V.%20Lombard&amp;author=C.%20P.%20Baril&amp;author=P.%20Dubreuil&amp;author=F.%20Blouet&amp;author=D.%20Zhang&amp;publication_year=2000&amp;journal=Crop%20Sci.&amp;volume=40&amp;pages=1417-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2000.4051417x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2000.4051417x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2000.4051417x"> </span></div> </div></div></div></div></div><div content-id="DSR041C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mailer</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Wratten</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Vonarx</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic diversity amongst Australian canola cultivars determined by randomly amplified polymorphic DNA</div>, <div class="source">Aust. J. Exp. Agric.</div> , <div class="year">1997</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/EA97059" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/EA97059</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20amongst%20Australian%20canola%20cultivars%20determined%20by%20randomly%20amplified%20polymorphic%20DNA&amp;author=R.%20J.%20Mailer&amp;author=N.%20Wratten&amp;author=M.%20Vonarx&amp;publication_year=1997&amp;journal=Aust.%20J.%20Exp.%20Agric.&amp;volume=37&amp;pages=793-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/EA97059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FEA97059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FEA97059"> </span></div> </div></div></div></div></div><div content-id="DSR041C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seyis</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Badani</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Analysis of genetic diversity in the <em>Brassica napus</em> L. gene pool using SSR markers</div>, <div class="source">Genet. Res. Crop Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">793</div>-<div class="lpage">802</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10722-004-5541-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10722-004-5541-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20in%20the%20Brassica%20napus%20L.%20gene%20pool%20using%20SSR%20markers&amp;author=M.%20Hasan&amp;author=F.%20Seyis&amp;author=A.%20Badani&amp;publication_year=2006&amp;journal=Genet.%20Res.%20Crop%20Evol.&amp;volume=53&amp;pages=793-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-004-5541-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-004-5541-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-004-5541-2"> </span></div> </div></div></div></div></div><div content-id="DSR041C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kaur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Assessment of genetic diversity in Australian canola (<em>Brassica napus</em> L.) cultivars using SSR markers</div>, <div class="source">Crop Pasture Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1193</div>-<div class="lpage">201</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/CP09165" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/CP09165</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20Australian%20canola%20(Brassica%20napus%20L.)%20cultivars%20using%20SSR%20markers&amp;author=J.%20Wang&amp;author=S.%20Kaur&amp;author=N.%20Cogan&amp;publication_year=2009&amp;journal=Crop%20Pasture%20Sci.&amp;volume=60&amp;pages=1193-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/CP09165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FCP09165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FCP09165"> </span></div> </div></div></div></div></div><div content-id="DSR041C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snowdon</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Friedt</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Molecular markers in Brassica oilseed breeding: current status and future possibilities</div>, <div class="source">Plant Breed.</div> , <div class="year">2004</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1439-0523.2003.00968.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1439-0523.2003.00968.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20markers%20in%20Brassica%20oilseed%20breeding%3A%20current%20status%20and%20future%20possibilities&amp;author=R.%20J.%20Snowdon&amp;author=W.%20Friedt&amp;publication_year=2004&amp;journal=Plant%20Breed.&amp;volume=123&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2004.123.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2004.123.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2004.123.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSR041C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tommasini</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Batley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Arnold</div> <div class="given-names">G. M.</div></div>, et al. </div><div class="article-title">The development of multiplex simple sequence repeat (SSR) markers to complement distinctness, uniformity and stability testing of rape (<em>Brassica napus</em> L.) varieties</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1091</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20multiplex%20simple%20sequence%20repeat%20(SSR)%20markers%20to%20complement%20distinctness%2C%20uniformity%20and%20stability%20testing%20of%20rape%20(Brassica%20napus%20L.)%20varieties&amp;author=L.%20Tommasini&amp;author=J.%20Batley&amp;author=G.%20M.%20Arnold&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=1091-101" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bowman</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Qian</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Genetic diversity of <em>Brassica napus</em> accessions mainly from China detected with RFLP markers</div>, <div class="source">Chin. J. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">221</div>-<div class="lpage">32</div>)</div></div></div></div><div content-id="DSR041C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">DArT for high-throughput genotyping of Cassava (<em>Manihot esculenta</em>) and its wild relatives</div>, <div class="source">Theor. Appl.Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1092</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-005-1937-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-005-1937-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DArT%20for%20high-throughput%20genotyping%20of%20Cassava%20(Manihot%20esculenta)%20and%20its%20wild%20relatives&amp;author=L.%20Xia&amp;author=K.%20Peng&amp;author=S.%20Yang&amp;publication_year=2005&amp;journal=Theor.%20Appl.Genet.&amp;volume=110&amp;pages=1092-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1937-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1937-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1937-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15742202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W. H.</div></div>, et al. </div><div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">203</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng1702" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng1702</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.%20H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jordan</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Tao</div> <div class="given-names">Y. Z.</div></div>,  <div class="name"><div class="surname">Godwin</div> <div class="given-names">I. D.</div></div>,  <div class="name"><div class="surname">Henzell</div> <div class="given-names">R. G.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McIntyre</div> <div class="given-names">C. L.</div></div>. </div><div class="article-title">Comparison of identity by descent and identity by state for detecting genetic regions under selection in a sorghum pedigree breeding program</div>, <div class="source">Mol. Breed.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">441</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-005-0901-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-005-0901-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20identity%20by%20descent%20and%20identity%20by%20state%20for%20detecting%20genetic%20regions%20under%20selection%20in%20a%20sorghum%20pedigree%20breeding%20program&amp;author=D.%20R.%20Jordan&amp;author=Y.%20Z.%20Tao&amp;author=I.%20D.%20Godwin&amp;author=R.%20G.%20Henzell&amp;author=M.%20Cooper&amp;author=C.%20L.%20McIntyre&amp;publication_year=2004&amp;journal=Mol.%20Breed.&amp;volume=14&amp;pages=441-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-005-0901-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-005-0901-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-005-0901-y"> </span></div> </div></div></div></div></div><div content-id="DSR041C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Podlich</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Winkler</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mapping As You Go: An effective approach for marker-assisted selection of complex traits</div>, <div class="source">Crop Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">1560</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2135/cropsci2004.1560" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2135/cropsci2004.1560</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20As%20You%20Go%3A%20An%20effective%20approach%20for%20marker-assisted%20selection%20of%20complex%20traits&amp;author=D.%20Podlich&amp;author=C.%20Winkler&amp;author=M.%20Cooper&amp;publication_year=2004&amp;journal=Crop%20Sci.&amp;volume=44&amp;pages=1560-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2004.1560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2004.1560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2004.1560"> </span></div> </div></div></div></div></div><div content-id="DSR041C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jannink</div> <div class="given-names">J.-L.</div></div>,  <div class="name"><div class="surname">Lorenz</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genomic selection in plant breeding: from theory to practice</div>, <div class="source">Brief. Funct. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">166</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bfgp/elq001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bfgp/elq001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20in%20plant%20breeding%3A%20from%20theory%20to%20practice&amp;author=J.-L.%20Jannink&amp;author=A.%20J.%20Lorenz&amp;author=H.%20Iwata&amp;publication_year=2010&amp;journal=Brief.%20Funct.%20Genomics&amp;volume=9&amp;pages=166-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elq001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felq001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felq001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20156985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gut</div> <div class="given-names">I. G.</div></div>. </div><div class="article-title">Automation in genotyping of single nucleotide polymorphisms</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">475</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.1131" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.1131</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automation%20in%20genotyping%20of%20single%20nucleotide%20polymorphisms&amp;author=I.%20G.%20Gut&amp;publication_year=2001&amp;journal=Hum.%20Mutat.&amp;volume=17&amp;pages=475-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11385706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Single nucleotide polymorphism (SNP) discovery in the polyploid <em>Brassica napus</em> using Solexa transcriptome sequencing</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">334</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2008.00396.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1467-7652.2008.00396.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20(SNP)%20discovery%20in%20the%20polyploid%20Brassica%20napus%20using%20Solexa%20transcriptome%20sequencing&amp;author=M.%20Trick&amp;author=Y.%20Long&amp;author=J.%20Meng&amp;author=I.%20Bancroft&amp;publication_year=2009&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=7&amp;pages=334-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2009.7.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2009.7.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2009.7.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19207216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Polley</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">SNP discovery by amplicon sequencing and multiplex SNP genotyping in the allopolyploid species <em>Brassica napus</em></div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">948</div>-<div class="lpage">56</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G10-079" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G10-079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20by%20amplicon%20sequencing%20and%20multiplex%20SNP%20genotyping%20in%20the%20allopolyploid%20species%20Brassica%20napus&amp;author=G.%20Durstewitz&amp;author=A.%20Polley&amp;author=J.%20Plieske&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=948-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">B. D.</div></div>, et al. </div><div class="article-title">The evolution of <em>Brassica napus FLOWERING LOCUS T</em> paralogues in the context of inverted chromosomal duplication blocks</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">271</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-9-271" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2148-9-271</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20Brassica%20napus%20FLOWERING%20LOCUS%20T%20paralogues%20in%20the%20context%20of%20inverted%20chromosomal%20duplication%20blocks&amp;author=J.%20Wang&amp;author=Y.%20Long&amp;author=B.%20D.%20Wu&amp;publication_year=2009&amp;journal=BMC%20Evol.%20Biol.&amp;volume=9&amp;pages=271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-9-271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-9-271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-9-271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19939256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tadege</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sheldon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Helliwell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Stoutjesdijk</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Dennis</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Peacock</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Control of flowering time by <em>FLC</em> orthologues in <em>Brassica napus</em></div>, <div class="source">Plant J.</div> , <div class="year">2001</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">545</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.2001.01182.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-313X.2001.01182.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20flowering%20time%20by%20FLC%20orthologues%20in%20Brassica%20napus&amp;author=M.%20Tadege&amp;author=C.%20Sheldon&amp;author=C.%20Helliwell&amp;author=P.%20Stoutjesdijk&amp;author=E.%20Dennis&amp;author=W.%20Peacock&amp;publication_year=2001&amp;journal=Plant%20J.&amp;volume=28&amp;pages=545-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2001.01182.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2001.01182.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2001.01182.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11849594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkin</div> <div class="given-names">I. A. P.</div></div>,  <div class="name"><div class="surname">Clarke</div> <div class="given-names">W. E.</div></div>,  <div class="name"><div class="surname">Sidebottom</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Towards unambiguous transcript mapping in the allotetraploid <em>Brassica napus</em></div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">929</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G10-053" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G10-053</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20unambiguous%20transcript%20mapping%20in%20the%20allotetraploid%20Brassica%20napus&amp;author=I.%20A.%20P.%20Parkin&amp;author=W.%20E.%20Clarke&amp;author=C.%20Sidebottom&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=929-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing</div>, <div class="source">Nat. Biotech.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">762</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.1926" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.1926</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20the%20genome%20of%20the%20polyploid%20crop%20oilseed%20rape%20by%20transcriptome%20sequencing&amp;author=I.%20Bancroft&amp;author=C.%20Morgan&amp;author=F.%20Fraser&amp;publication_year=2011&amp;journal=Nat.%20Biotech.&amp;volume=29&amp;pages=762-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1926"> </span></div> </div></div></div></div></div><div content-id="DSR041C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elshire</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Glaubitz</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">A robust, simple Genotyping-by-Sequencing (GBS) approach for high diversity species</div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e19379</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0019379" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0019379</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20Genotyping-by-Sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.%20J.%20Elshire&amp;author=J.%20C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baird</div> <div class="given-names">N. A.</div></div>,  <div class="name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>,  <div class="name"><div class="surname">Atwood</div> <div class="given-names">T. S.</div></div>, et al. </div><div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PLoS ONE</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e3376</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0003376" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0003376</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Batley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lorenc</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Sequence analysis of the canola genome</div>, <div class="year">2011</div><div class="conf-name">Proceedings of the International Rapeseed Conference (Abstract Book)</div><div class="publisher-loc">Prague, Czech Republic</div>pg. <div class="fpage">9</div> </div></div></div></div><div content-id="DSR041C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaccoud</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Feinstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kilian</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Diversity arrays: a solid state technology for sequence information independent genotyping</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> pg. <div class="fpage">e25</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/29.4.e25" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/29.4.e25</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20arrays%3A%20a%20solid%20state%20technology%20for%20sequence%20information%20independent%20genotyping&amp;author=D.%20Jaccoud&amp;author=K.%20Peng&amp;author=D.%20Feinstein&amp;author=A.%20Kilian&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=e25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.4.e25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.4.e25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.4.e25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11160945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wenzl</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Carling</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kudrna</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Diversity arrays technology (DArT) for whole-genome profiling of barley</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">9915</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0401076101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0401076101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20arrays%20technology%20(DArT)%20for%20whole-genome%20profiling%20of%20barley&amp;author=P.%20Wenzl&amp;author=J.%20Carling&amp;author=D.%20Kudrna&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=9915-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0401076101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0401076101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0401076101"> </span></div> </div></div></div></div></div><div content-id="DSR041C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mantovani</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Maccaferri</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sanguineti</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">An integrated DArT-SSR linkage map of durum wheat</div>, <div class="source">Mol. Breed.</div> , <div class="year">2008</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">629</div>-<div class="lpage">48</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-008-9205-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-008-9205-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20DArT-SSR%20linkage%20map%20of%20durum%20wheat&amp;author=P.%20Mantovani&amp;author=M.%20Maccaferri&amp;author=M.%20Sanguineti&amp;publication_year=2008&amp;journal=Mol.%20Breed.&amp;volume=22&amp;pages=629-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-008-9205-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-008-9205-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-008-9205-3"> </span></div> </div></div></div></div></div><div content-id="DSR041C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akbari</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wenzl</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Caig</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">1409</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0365-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0365-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20arrays%20technology%20(DArT)%20for%20high-throughput%20profiling%20of%20the%20hexaploid%20wheat%20genome&amp;author=M.%20Akbari&amp;author=P.%20Wenzl&amp;author=V.%20Caig&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=1409-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0365-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0365-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0365-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Francki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Walker</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Crawford</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparison of genetic and cytogenetic maps of hexaploid wheat (<em>Triticum aestivum</em> L.) using SSR and DArT markers</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">181</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00438-008-0403-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00438-008-0403-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20genetic%20and%20cytogenetic%20maps%20of%20hexaploid%20wheat%20(Triticum%20aestivum%20L.)%20using%20SSR%20and%20DArT%20markers&amp;author=M.%20Francki&amp;author=E.%20Walker&amp;author=A.%20Crawford&amp;publication_year=2009&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=281&amp;pages=181-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-008-0403-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-008-0403-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-008-0403-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19020902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mace</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jordan</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">DArT markers: diversity analyses and mapping in <em>Sorghum bicolor</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">26</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-26" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-26</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DArT%20markers%3A%20diversity%20analyses%20and%20mapping%20in%20Sorghum%20bicolor&amp;author=E.%20Mace&amp;author=L.%20Xia&amp;author=D.%20Jordan&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18208620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wenzl</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Carling</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A high-density consensus map of barley linking DArT markers to SSR, RFLP and STS loci and agricultural traits</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">206</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-7-206" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-7-206</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20consensus%20map%20of%20barley%20linking%20DArT%20markers%20to%20SSR%2C%20RFLP%20and%20STS%20loci%20and%20agricultural%20traits&amp;author=P.%20Wenzl&amp;author=H.%20Li&amp;author=J.%20Carling&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16904008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Allen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Diffey</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">McKelvie</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Localisation of quantitative trait loci for quality attributes in a doubled haploid population of wheat (<em>Triticum aestivum</em> L)</div>, <div class="source">Genome</div> , <div class="year">2009</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">701</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G09-045" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G09-045</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localisation%20of%20quantitative%20trait%20loci%20for%20quality%20attributes%20in%20a%20doubled%20haploid%20population%20of%20wheat%20(Triticum%20aestivum%20L)&amp;author=R.%20Raman&amp;author=H.%20Allen&amp;author=S.%20Diffey&amp;author=H.%20Raman&amp;author=P.%20Martin&amp;author=K.%20McKelvie&amp;publication_year=2009&amp;journal=Genome&amp;volume=52&amp;pages=701-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19767900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wenzl</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Huttner</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kilian</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A DArT platform for quantitative bulked segregant analysis</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">196</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-8-196" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-8-196</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20DArT%20platform%20for%20quantitative%20bulked%20segregant%20analysis&amp;author=P.%20Wenzl&amp;author=H.%20Raman&amp;author=J.%20Wang&amp;author=M.%20Zhou&amp;author=E.%20Huttner&amp;author=A.%20Kilian&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=196" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17597543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ryan</div> <div class="given-names">P. R.</div></div>,  <div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Horst</div> <div class="given-names">W. J.</div></div>,  <div class="name"><div class="surname">Delhaize</div> <div class="given-names">E.</div></div>. </div><div class="article-title">A second mechanism for aluminum resistance in wheat relies on the constitutive efflux of citrate from roots</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">340</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.129155" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.108.129155</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20mechanism%20for%20aluminum%20resistance%20in%20wheat%20relies%20on%20the%20constitutive%20efflux%20of%20citrate%20from%20roots&amp;author=P.%20R.%20Ryan&amp;author=H.%20Raman&amp;author=S.%20Gupta&amp;author=W.%20J.%20Horst&amp;author=E.%20Delhaize&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=340-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.129155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.129155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.129155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19005085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stodart</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Cavanagh</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Molecular diversity and genetic structure of modern and landrace cultivars of wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Crop Pasture Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">222</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/CP09093" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/CP09093</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20diversity%20and%20genetic%20structure%20of%20modern%20and%20landrace%20cultivars%20of%20wheat%20(Triticum%20aestivum%20L.)&amp;author=H.%20Raman&amp;author=B.%20Stodart&amp;author=C.%20Cavanagh&amp;publication_year=2010&amp;journal=Crop%20Pasture%20Sci.&amp;volume=61&amp;pages=222-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/CP09093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FCP09093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FCP09093"> </span></div> </div></div></div></div></div><div content-id="DSR041C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stodart</div> <div class="given-names">B. J.</div></div>,  <div class="name"><div class="surname">Mackay</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Assessment of molecular diversity in landraces of bread wheat (<em>Triticum aestivum</em> L.) held in an ex situ collection with Diversity Arrays Technology (DArT)</div>, <div class="source">Aust. J. Agric. Res.</div> , <div class="year">2007</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">1174</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/AR07010" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/AR07010</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20molecular%20diversity%20in%20landraces%20of%20bread%20wheat%20(Triticum%20aestivum%20L.)%20held%20in%20an%20ex%20situ%20collection%20with%20Diversity%20Arrays%20Technology%20(DArT)&amp;author=B.%20J.%20Stodart&amp;author=M.%20C.%20Mackay&amp;author=H.%20Raman&amp;publication_year=2007&amp;journal=Aust.%20J.%20Agric.%20Res.&amp;volume=58&amp;pages=1174-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/AR07010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FAR07010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FAR07010"> </span></div> </div></div></div></div></div><div content-id="DSR041C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stodart</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ryan</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Genome wide association analyses of common wheat (<em>Triticum aestivum</em> L) germplasm identifies multiple loci for aluminium resistance</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">957</div>-<div class="lpage">66</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G10-058" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G10-058</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20wide%20association%20analyses%20of%20common%20wheat%20(Triticum%20aestivum%20L)%20germplasm%20identifies%20multiple%20loci%20for%20aluminium%20resistance&amp;author=H.%20Raman&amp;author=B.%20Stodart&amp;author=P.%20Ryan&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=957-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aslam</div> <div class="given-names">M. N.</div></div>. </div><div class="article-title">Inheritance of molecular markers for high oleic and low linolenic acids in canola (<em>Brassica napus L.</em>) and impact of genotype and environment on these traits</div>, <div class="year">2007</div><div class="publisher-loc">Perth</div><div class="publisher-name">The University of Western Australia</div> <div class="comment"><em>Thesis submitted to the School of Plant Biology</em></div></div></div></div></div><div content-id="DSR041C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cousin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Twinned microspore-derived embryos of canola (<em>Brassica napus</em> L.) are genetically identical</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2009</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">831</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00299-009-0677-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00299-009-0677-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twinned%20microspore-derived%20embryos%20of%20canola%20(Brassica%20napus%20L.)%20are%20genetically%20identical&amp;author=A.%20Cousin&amp;author=M.%20Nelson&amp;publication_year=2009&amp;journal=Plant%20Cell%20Rep.&amp;volume=28&amp;pages=831-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-009-0677-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-009-0677-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-009-0677-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19219609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aslam</div> <div class="given-names">M. N.</div></div>,  <div class="name"><div class="surname">Kailis</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">M. N.</div></div>,  <div class="name"><div class="surname">Bayliss</div> <div class="given-names">K. L.</div></div>,  <div class="name"><div class="surname">Cowling</div> <div class="given-names">W. A.</div></div>. </div><div class="article-title">Variation in fatty acid composition among genetically homogeneous seeds of canola (<em>Brassica napus</em>), and implications for genotypic selection based on single seeds</div>, <div class="source">Aust. J. Agric. Res.</div> , <div class="year">2008</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">926</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1071/AR08153" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1071/AR08153</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20fatty%20acid%20composition%20among%20genetically%20homogeneous%20seeds%20of%20canola%20(Brassica%20napus)%2C%20and%20implications%20for%20genotypic%20selection%20based%20on%20single%20seeds&amp;author=M.%20N.%20Aslam&amp;author=S.%20Kailis&amp;author=M.%20N.%20Nelson&amp;author=K.%20L.%20Bayliss&amp;author=W.%20A.%20Cowling&amp;publication_year=2008&amp;journal=Aust.%20J.%20Agric.%20Res.&amp;volume=59&amp;pages=926-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/AR08153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FAR08153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FAR08153"> </span></div> </div></div></div></div></div><div content-id="DSR041C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kilian</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wenzl</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Huttner</div> <div class="given-names">E.</div></div>, et al. </div><div class="person-group"><div class="name"><div class="surname">Bonin</div> <div class="given-names">A, P. F.</div></div>. </div><div class="article-title">Diversity Arrays Technology (DArT)—a generic genome profiling technology on open platforms</div>, <div class="source">Data Production and Analysis in Population Genomics. Methods in Molecular Biology Series</div> , <div class="year">2011</div><div class="publisher-name">New York, USA: Humana Press</div></div></div></div></div><div content-id="DSR041C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anderson</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Churchill</div> <div class="given-names">G. A.</div></div>,  <div class="name"><div class="surname">Autrique</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M. E.</div></div>. </div><div class="article-title">Optimizing parental selection for genetic linkage maps</div>, <div class="source">Genome</div> , <div class="year">1993</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">181</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/g93-024" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/g93-024</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20parental%20selection%20for%20genetic%20linkage%20maps&amp;author=J.%20A.%20Anderson&amp;author=G.%20A.%20Churchill&amp;author=J.%20E.%20Autrique&amp;author=S.%20D.%20Tanksley&amp;author=M.%20E.%20Sorrells&amp;publication_year=1993&amp;journal=Genome&amp;volume=36&amp;pages=181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g93-024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg93-024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg93-024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaccard</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Nouvelles recherches sur la distribution florale</div>, <div class="source">Bull. Soc. Vaudoise Sci. Nat.</div> , <div class="year">1908</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">223</div>-<div class="lpage">70</div>)</div></div></div></div><div content-id="DSR041C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Olson</div> <div class="given-names">G. J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Mortiz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Phylogeny reconstruction</div>, <div class="source">Molecular Systematics</div> , <div class="year">1990</div><div class="publisher-loc">MA</div><div class="publisher-name">Sinauer Associates, S.</div>(pg. <div class="fpage">411</div>-<div class="lpage">501</div>)</div></div></div></div><div content-id="DSR041C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sneath</div> <div class="given-names">P. H.</div></div>,  <div class="name"><div class="surname">Sokal</div> <div class="given-names">R. R.</div></div>. </div>, <div class="source">Numerical Taxonomy</div> , <div class="year">1973</div><div class="publisher-loc">San Francisco</div><div class="publisher-name">Freeman</div>pg. <div class="fpage">573</div> </div></div></div></div><div content-id="DSR041C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rohlf</div> <div class="given-names">F. J.</div></div>. </div>, <div class="source">NTSYS-PC: Numerical Taxonomy and Multivariate Analysis System, Ver. 2.10z</div> , <div class="year">2002</div><div class="publisher-loc">Setauket, NY</div><div class="publisher-name">Exeter Software</div></div></div></div></div><div content-id="DSR041C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Moule</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Efficient large-scale development of microsatellites for marker and mapping applications in Brassica crop species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1103</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-003-1522-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-003-1522-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20large-scale%20development%20of%20microsatellites%20for%20marker%20and%20mapping%20applications%20in%20Brassica%20crop%20species&amp;author=A.%20Lowe&amp;author=C.%20Moule&amp;author=M.%20Trick&amp;author=K.%20Edwards&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1103-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1522-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1522-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1522-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Parkin</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Integration of linkage maps for the amphidiploid <em>Brassica napus</em> and comparative mapping with Arabidopsis and <em>Brassica rapa</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">101</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-12-101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-12-101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20linkage%20maps%20for%20the%20amphidiploid%20Brassica%20napus%20and%20comparative%20mapping%20with%20Arabidopsis%20and%20Brassica%20rapa&amp;author=J.%20Wang&amp;author=D.%20Lydiate&amp;author=I.%20Parkin&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21306613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mason</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Castello</div> <div class="given-names">M.-C.</div></div>,  <div class="name"><div class="surname">Thomson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Cowling</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Microspore culture preferentially selects unreduced (2n) gametes from an interspecific hybrid of <em>Brassica napus</em> L. × <em>Brassica carinata</em> Braun.</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">497</div>-<div class="lpage">505</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-009-1056-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-009-1056-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microspore%20culture%20preferentially%20selects%20unreduced%20(2n)%20gametes%20from%20an%20interspecific%20hybrid%20of%20Brassica%20napus%20L.%20%C3%97%20Brassica%20carinata%20Braun.&amp;author=M.%20Nelson&amp;author=A.%20Mason&amp;author=M.-C.%20Castello&amp;author=L.%20Thomson&amp;author=G.%20Yan&amp;author=W.%20Cowling&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=119&amp;pages=497-505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1056-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1056-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1056-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19436985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panjabi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jagannath</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bisht</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Comparative mapping of <em>Brassica juncea</em> and <em>Arabidopsis thaliana</em> using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">113</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-113" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-113</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20of%20Brassica%20juncea%20and%20Arabidopsis%20thaliana%20using%20Intron%20Polymorphism%20(IP)%20markers%3A%20homoeologous%20relationships%2C%20diversification%20and%20evolution%20of%20the%20A%2C%20B%20and%20C%20Brassica%20genomes&amp;author=P.%20Panjabi&amp;author=A.%20Jagannath&amp;author=N.%20Bisht&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18315867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ronin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Minkov</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Constructing large-scale genetic maps using an evolutionary strategy algorithm</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">2269</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constructing%20large-scale%20genetic%20maps%20using%20an%20evolutionary%20strategy%20algorithm&amp;author=D.%20Mester&amp;author=Y.%20Ronin&amp;author=D.%20Minkov&amp;author=E.%20Nevo&amp;author=A.%20Korol&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=2269-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Moolhuijzen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Boersma</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Aligning a new reference genetic map of <em>Lupinus angustifolius</em> with the genome sequence of the model legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">73</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aligning%20a%20new%20reference%20genetic%20map%20of%20Lupinus%20angustifolius%20with%20the%20genome%20sequence%20of%20the%20model%20legume%2C%20Lotus%20japonicus&amp;author=M.%20Nelson&amp;author=P.%20Moolhuijzen&amp;author=J.%20Boersma&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=73-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20133394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cowling</div> <div class="given-names">W. A.</div></div>. </div><div class="article-title">Genetic diversity in Australian canola and implications for crop breeding for changing future environments</div>, <div class="source">Field Crop. Res.</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">103</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fcr.2006.12.014" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fcr.2006.12.014</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20in%20Australian%20canola%20and%20implications%20for%20crop%20breeding%20for%20changing%20future%20environments&amp;author=W.%20A.%20Cowling&amp;publication_year=2007&amp;journal=Field%20Crop.%20Res.&amp;volume=104&amp;pages=103-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fcr.2006.12.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fcr.2006.12.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fcr.2006.12.014"> </span></div> </div></div></div></div></div><div content-id="DSR041C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">M. N.</div></div>,  <div class="name"><div class="surname">Ghamkhar</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cowling</div> <div class="given-names">W. A.</div></div>. </div><div class="article-title">Divergent patterns of allelic diversity from similar origins: the case of oilseed rape (<em>Brassica napus</em> L.) in China and Australia</div>, <div class="source">Genome</div> , <div class="year">2008</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G07-095" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G07-095</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20patterns%20of%20allelic%20diversity%20from%20similar%20origins%3A%20the%20case%20of%20oilseed%20rape%20(Brassica%20napus%20L.)%20in%20China%20and%20Australia&amp;author=S.%20Chen&amp;author=M.%20N.%20Nelson&amp;author=K.%20Ghamkhar&amp;author=T.%20Fu&amp;author=W.%20A.%20Cowling&amp;publication_year=2008&amp;journal=Genome&amp;volume=51&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G07-095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG07-095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG07-095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18356934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grover</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aishwarya</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">P. C.</div></div>. </div><div class="article-title">Biased distribution of microsatellites in rice genome</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">469</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00438-006-0204-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00438-006-0204-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20distribution%20of%20microsatellites%20in%20rice%20genome&amp;author=A.%20Grover&amp;author=V.%20Aishwarya&amp;author=P.%20C.%20Sharma&amp;publication_year=2007&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=277&amp;pages=469-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-006-0204-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-006-0204-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-006-0204-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salisbury</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Wratten</div> <div class="given-names">N.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Salisbury</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Potter</div> <div class="given-names">T. D.</div></div>,  <div class="name"><div class="surname">McDonald</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">A. G.</div></div>. </div><div class="article-title"><em>Brassica napus</em> breeding. Canola in Australia: the First 30 Years</div>, <div class="source">Organising Committee of the 10th International Rapeseed Congress</div> , <div class="year">1999</div><div class="publisher-name">Australia: Canberra</div>(pg. <div class="fpage">29</div>-<div class="lpage">35</div>)</div></div></div></div><div content-id="DSR041C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huhn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rakow</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Einige experimentaelle Ergebnisse zur Fremdbefurchtungsrate bei Winterrap (<em>Brassica napus oleifera</em>) in Abhangigkeit von Sorte and Abstand.</div>, <div class="source">Z. Pflanzenzuecht</div> , <div class="year">1979</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">289</div>-<div class="lpage">307</div>)</div></div></div></div><div content-id="DSR041C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rakow</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Woods</div> <div class="given-names">D. L.</div></div>. </div><div class="article-title">Outcrossing in rape and mustard under Saskatchewan prairie conditions</div>, <div class="source">Can. J. Plant Sci.</div> , <div class="year">1987</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">147</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.4141/cjps87-017" class="link link-pub-id-doi" target="_blank">https://doi.org/10.4141/cjps87-017</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Outcrossing%20in%20rape%20and%20mustard%20under%20Saskatchewan%20prairie%20conditions&amp;author=G.%20Rakow&amp;author=D.%20L.%20Woods&amp;publication_year=1987&amp;journal=Can.%20J.%20Plant%20Sci.&amp;volume=67&amp;pages=147-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4141/cjps87-017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4141%2Fcjps87-017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4141%2Fcjps87-017"> </span></div> </div></div></div></div></div><div content-id="DSR041C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">H. C.</div></div>,  <div class="name"><div class="surname">Engqvist</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Karlsson</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Comparison of rapeseed cultivars and resynthesized lines based on allozyme and RFLP markers</div>, <div class="source">Theor. Appl. Genet</div> , <div class="year">1995</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">62</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20rapeseed%20cultivars%20and%20resynthesized%20lines%20based%20on%20allozyme%20and%20RFLP%20markers&amp;author=H.%20C.%20Becker&amp;author=G.%20M.%20Engqvist&amp;author=B.%20Karlsson&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet&amp;volume=91&amp;pages=62-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00220859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00220859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00220859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Raman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Prangnell</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Genetic dissection of natural variation for flowering time in rapeseed</div>, <div class="year">2011</div><div class="conf-name">Proceedings of the International Rapeseed Congress (Abstract Book)</div><div class="publisher-loc">Prague, Czech Republic</div>pg. <div class="fpage">139</div> </div></div></div></div><div content-id="DSR041C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferreira</div> <div class="given-names">M. E.</div></div>,  <div class="name"><div class="surname">Satagopan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yandell</div> <div class="given-names">B. S.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">P. H.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Mapping loci controlling vernalisation requirement and flowering time in <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">727</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20loci%20controlling%20vernalisation%20requirement%20and%20flowering%20time%20in%20Brassica%20napus&amp;author=M.%20E.%20Ferreira&amp;author=J.%20Satagopan&amp;author=B.%20S.%20Yandell&amp;author=P.%20H.%20Williams&amp;author=T.%20C.%20Osborn&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=90&amp;pages=727-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24174034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piquemal</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cinquin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Couton</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Construction of an oilseed rape (<em>Brassica napus</em> L.) genetic map with SSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">1514</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-005-0080-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-005-0080-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20oilseed%20rape%20(Brassica%20napus%20L.)%20genetic%20map%20with%20SSR%20markers&amp;author=J.%20Piquemal&amp;author=E.%20Cinquin&amp;author=F.%20Couton&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1514-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0080-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0080-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0080-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16187118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR041C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Delourme</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A consensus linkage map for rapeseed (<em>Brassica napus</em> L.): construction and integration of three individual maps from DH populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">491</div>-<div class="lpage">507</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s001220100560" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s001220100560</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20for%20rapeseed%20(Brassica%20napus%20L.)%3A%20construction%20and%20integration%20of%20three%20individual%20maps%20from%20DH%20populations&amp;author=V.%20Lombard&amp;author=R.%20Delourme&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=491-507" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100560"> </span></div> </div></div></div></div></div><div content-id="DSR041C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR041C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N. O. I.</div></div>,  <div class="name"><div class="surname">Ye</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Genetic map construction and QTL mapping of resistance to blackleg (<em>Leptosphaeria maculans</em>) disease in Australian canola (<em>Brassica napus</em> L.) cultivars</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">71</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-009-1160-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-009-1160-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20map%20construction%20and%20QTL%20mapping%20of%20resistance%20to%20blackleg%20(Leptosphaeria%20maculans)%20disease%20in%20Australian%20canola%20(Brassica%20napus%20L.)%20cultivars&amp;author=S.%20Kaur&amp;author=N.%20O.%20I.%20Cogan&amp;author=G.%20Ye&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=71-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1160-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1160-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1160-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19821065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=485713&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_table4.pdf?Expires=2147483647&amp;Signature=c0k0Q4R~awn~0kczdTG2ZXuf3qrMH5qtPnm2wqNMwQnfIsWFIKt5HUe06-sxCUSv4Uiq~dAgCs6fPVJmBXZlIJkMGxu4tw40NFVesj~CJIwi1LwEoLZjFVKcQVzlZ6f2-FrM5N~TVq0yhMz~YRzwencAd8u5LAAwEMGJCIDC94KOE8ZcjVRZQk0hI7KLizZ21Dp~3kSgvM207wHnKp8KBdP8c7WxKzvDFM007~blsxTQQuzB-OQkQy5ykaJ3z9zitXGMq2qDmA0dvT7YDLrg5AR7pjhaOgUaO9Il49jBFsqPbyUGj0ImNdaOPWca-E4KZrz6tKQSXmyTJl-T-6Hj3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_table3.pdf?Expires=2147483647&amp;Signature=m6p0N8FdFXUhGkO-bTpodHZ4wCVMPHOBWD2V8UtDNEHqOa8-cX6C3Z4y2fmEcD5wDK7oC2HxPyYsBcPlkd~L8sns~VRBbJyqAcmei0jaEIn6hxCBcw9rnT2AJe~2bEIFXpDZ5cc6MFy1uNo8m8CxB650u1JdtzOw~mXjYGLVlMh03ZHcTKPdBjxYBWsu1pfTJSaOiWpQm7WW0~jTBo0tIACby14sBgpniYcoOCqRncCEFDtY-yux7QHlb7Dd1zXjRnklI2WPG52j86W-22W-FAOh~Ecfc1QLxpEXOll1w93lz7grU~eUo~Be1RdLS4P0x9oix9F5m2y65YzahScSeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_table2.xlsx?Expires=2147483647&amp;Signature=JBy~rAGbNA3OjZ1NLxVkpUFJMXM98I~4r1~acZa~Eii2WeQnNQmyt7Y9GJlibrsVnkvUQs~9mgqPkSMf-kQ64KysrP6OE2tssR-vFNqaSWc9y33rB2EP9R9fcVJoL6W-fPboYara-GIqrnKelDP39h6JN9HOau-SPwVer0M1sqT2i3RN1QcOd-jVMwDRx7c7T0XY5ybjnXH3shjmBa-On9VFr4M9-EqEFPhOKWyLMNaR9DDfObBCnCFoXpAM5N8b9Dnclz8HTz3fciienxO7E-DZWtQW8D~iEjSGUQmXnnkb1~EZgTURNtu1CfFraF0dQ6vJxwc4qlQzo8bSdVdF9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_table1.xlsx?Expires=2147483647&amp;Signature=3XQRiD0I4T4YAfMHzM35CsatzJUybQrIMN~EuaLy9w89R8FHnu9Zri5w7rZ6dhGKXTkYRoQN9KTIaSemY-qh~NTz8gURI8H91yOvcAQokdgWoeSDY9DAaK1nu4BwFmyUFYWH8Lh7U9i8DVjHKvmBXxZ3XqQRkVUzatgAAoT219x5TqWn-rctf59vNDyHxhggLfno6ZkHwxRCxGgGrdjihZWC--~mzJISxvQtVBEg9dST1ef9gidAFmo~mH3AHK4R22s8Wn7Z8EOPw~fNmQJZ-ZXRcChwKeK1rIaQTAEz8sxBhUIyF385rmi3lIFFfp4~-DlzRAys7NqarzODYDervA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_fig2.pdf?Expires=2147483647&amp;Signature=JPVKXcA-~Lg-DTB3tOG1j88crF-PpophTrh5eBhXQdlHijUMtwsJRauN3GVPBapc2XkEd-FKTwvrcAlTHAvWJZUlA0H12a1cEBbOWH6pF1I1vrhDuqp~JTqGlChufLqM1jUVyEjFljhGgTPqGsUtdLQiQFpOIXZztLMYef9oIWu~irqRz0FXvZWnI~pTg5eOvjIWcuZU6tCdwrkh3BCr02hzhwvgD-mYgOF4Huu9ZrK9fa8lZjjE9zR1hQr99-o~qVF16gazrdt42cM~QghuFGM1RWXrlZsMOALhlzsWpWqDyDUOGkZmdZeeR107i9zGBzHLwcpBJR2PbGl5DoO3uA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr041/2/dsr041-dsr041supp_fig1.ppt?Expires=2147483647&amp;Signature=GpdW2eYEcBglW3OHa4QEgoAVi0-DYtiQ5IOnZ1CN2Iu5xEakYPf67iTjYmw2cf6zfEUBqDBaEM8S7xBa0nSVJVN~Xs1U74qeHrJ0R2QYvra~xUDhcTMSdWhBNGa4nPp-24Us6G6IUwpuZkrmn2wIqfIEsnllGDPN7fijLpEsTWN6Pcj89UVGOB~dQrIh-MEWAmSqpNZ~jpiPesuYgEnR2APG3HvMYqiQeg9hxr0ifE720yjJ3cUR8G0uYWISxalA6~obT0otxn54r0Q3wt3PgrcmISaKjkoHpuNSs1P9XrSo-UKDVSM4wggTK6Xpx-ZC3RqrJ6tgJ6GyE-DCUEh3MQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">452</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">29</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr041"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="485713"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f51%2f485713">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000300760000005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (29)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F51%2F485713" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/485713'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Diversity Array Technology Markers: Genetic Diversity Analyses and Linkage Map Construction in Rapeseed (Brassica napus L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr041");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f51%2f485713">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

