

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1955,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/355/445136","@type":"ScholarlyArticle","name":"Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed ( Brassica napus  L.)","datePublished":"2014-02-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu002","keywords":["<italic>Brassica napus</italic>","association mapping","SNP","seed quality","seed weight"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/355/445136","author":[{"name":"Li, Feng","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Chen, Biyun","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Xu, Kun","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Wu, Jinfeng","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Song, Weilin","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Bancroft, Ian","affiliation":"2  Department of Biology ,  University of York ,  York ,  UK","@type":"Person"},{"name":"Harper, Andrea L.","affiliation":"2  Department of Biology ,  University of York ,  York ,  UK","@type":"Person"},{"name":"Trick, Martin","affiliation":"3  John Innes Centre ,  Norwich Research Park, Norwich NR4 7UH ,  UK","@type":"Person"},{"name":"Liu, Shengyi","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Gao, Guizhen","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Wang, Nian","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Yan, Guixin","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Qiao, Jiangwei","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Li, Jun","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Li, Hao","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Xiao, Xin","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Zhang, Tianyao","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"},{"name":"Wu, Xiaoming","affiliation":"1  Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops ,  Ministry of Agriculture ,  No. 2 Xudong Second Road, Hubei Province, Wuhan 430062 ,  China","@type":"Person"}],"description":"Abstract.  Association mapping can quickly and efficiently dissect complex agronomic traits. Rapeseed is one of the most economically important polyploid oil cr","pageStart":"355","pageEnd":"367","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed ( Brassica napus  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed ( Brassica napus  L.)" />
    <meta property="og:description" content="Abstract.  Association mapping can quickly and efficiently dissect complex agronomic traits. Rapeseed is one of the most economically important polyploid oil cr" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/355/445136" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-02-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Feng" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Chen, Biyun" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Xu, Kun" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Wu, Jinfeng" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Song, Weilin" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Bancroft, Ian" /><meta name="citation_author_institution" content="Department of Biology, University of York, York, UK" /><meta name="citation_author" content="Harper, Andrea L." /><meta name="citation_author_institution" content="Department of Biology, University of York, York, UK" /><meta name="citation_author" content="Trick, Martin" /><meta name="citation_author_institution" content="John Innes Centre, Norwich Research Park, Norwich NR4 7UH, UK" /><meta name="citation_author" content="Liu, Shengyi" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Gao, Guizhen" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Wang, Nian" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Yan, Guixin" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Qiao, Jiangwei" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Li, Jun" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Li, Hao" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Xiao, Xin" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Zhang, Tianyao" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_author" content="Wu, Xiaoming" /><meta name="citation_author_institution" content="Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops, Ministry of Agriculture, No. 2 Xudong Second Road, Hubei Province, Wuhan 430062, China" /><meta name="citation_title" content="Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed (Brassica napus L.)" /><meta name="citation_firstpage" content="355" /><meta name="citation_lastpage" content="367" /><meta name="citation_doi" content="10.1093/dnares/dsu002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Conservation of the microstructure of genome segments in Brassica napus and its diploid relatives; Plant J.;  citation_year=2004;  citation_volume=40;  citation_pages=725-33; " /><meta name="citation_reference" content="citation_title=Genome-analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; Japan J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Origin and domestication; citation_publisher=Elsevier, Amsterdam; Biology of Brassica coenospecies;  citation_year=1999;  citation_pages=33-58; " /><meta name="citation_reference" content="citation_title=History, evolution, and domestication of Brassica crops; citation_publisher=John Wiley &amp; Sons, Inc., New Jersey; Plant Breeding Reviews;  citation_year=2011;  citation_volume=vol. 35;  citation_pages=19-84; " /><meta name="citation_reference" content="citation_publisher=Proc Eucarpia Conference on the breeding of Cruciferous crops, Wageningen; The domestication of Brassica crops;  citation_year=1979;  citation_pages=47-56; " /><meta name="citation_reference" content="citation_title=Conditional QTL mapping of oil content in rapeseed with respect to protein content and traits related to plant development and grain yield; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=33-8; " /><meta name="citation_reference" content="citation_title=Mapping the genome of rapeseed (Brassica napus L.). I. Construction of an RFLP linkage map and localization of QTLs for seed glucosinolate content; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=90;  citation_pages=194-204; " /><meta name="citation_reference" content="citation_title=Mapping QTL controlling fatty acid composition in a doubled haploid rapeseed population segregating for oil content; Mol. Breed;  citation_year=2008;  citation_volume=21;  citation_pages=115-25; " /><meta name="citation_reference" content="citation_title=Flowering time quantitative trait loci analysis of oilseed Brassica in multiple environments and genomewide alignment with Arabidopsis; Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=2433-44; " /><meta name="citation_reference" content="citation_title=Unraveling the complex trait of crop yield with quantitative trait loci mapping in Brassica napus; Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=851-61; " /><meta name="citation_reference" content="citation_title=Genetic analysis of heterosis for yield and yield components in rapeseed (Brassica napus L.) by quantitative trait locus mapping; Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=1547-58; " /><meta name="citation_reference" content="citation_title=Identification of a major QTL for silique length and seed weight in oilseed rape (Brassica napus L.); Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=285-96; " /><meta name="citation_reference" content="citation_title=Identification and mapping of a major dominant quantitative trait locus controlling seeds per silique as a single Mendelian factor in Brassica napus L.; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=695-705; " /><meta name="citation_reference" content="citation_title=Next-generation genetics in plants; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=720-3; " /><meta name="citation_reference" content="citation_title=Linkage disequilibrium: what history has to tell us; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=83-90; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa; Nat. Commun.;  citation_year=2011;  citation_volume=2;  citation_pages=467" /><meta name="citation_reference" content="citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=159-62; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=32-9; " /><meta name="citation_reference" content="citation_title=Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels; Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=43-50; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines; Nature;  citation_year=2010;  citation_volume=465;  citation_pages=627-31; " /><meta name="citation_reference" content="citation_title=Chromosomal phylogeny and karyotype evolution in x=7 crucifer species (Brassicaceae); Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=2559-70; " /><meta name="citation_reference" content="citation_title=Segmental structure of the Brassica napus genome based on comparative analysis with Arabidopsis thaliana; Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=765-81; " /><meta name="citation_reference" content="citation_title=Patterns of molecular variation in a species-wide germplasm set of Brassica napus; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=1413-23; " /><meta name="citation_reference" content="citation_title=Extent and structure of linkage disequilibrium in canola quality winter rapeseed (Brassica napus L.); Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=921-31; " /><meta name="citation_reference" content="citation_title=Genetic structure and linkage disequilibrium pattern of a rapeseed (Brassica napus L.) association mapping panel revealed by microsatellites; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=Association mapping of seed oil content in Brassica napus and comparison with quantitative trait loci identified from linkage mapping; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=908-16; " /><meta name="citation_reference" content="citation_title=Association mapping for phenological, morphological, and quality traits in canola quality winter rapeseed (Brassica napus L.); Genome;  citation_year=2010;  citation_volume=53;  citation_pages=899-907; " /><meta name="citation_reference" content="citation_title=Regional association analysis delineates a sequenced chromosome region influencing antinutritive seed meal compounds in oilseed rape; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=917-28; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism (SNP) discovery in the polyploid Brassica napus using Solexa transcriptome sequencing; Plant Biotechnol. J;  citation_year=2009;  citation_volume=7;  citation_pages=334-46; " /><meta name="citation_reference" content="citation_title=Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=762-8; " /><meta name="citation_reference" content="citation_title=High-density SNP-based genetic map development and linkage disequilibrium assessment in Brassica napus L.; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=120" /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=Associative transcriptomics of traits in the polyploid crop species Brassica napus; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=798-802; " /><meta name="citation_reference" content="citation_title=Schnelle und einfache Analysen der Fettsäurezusammensetzung in einzelnen Rapskotyledonen I. Gaschromatographische und papierchromatographische Methoden; Z Pflanzenzücht;  citation_year=1971;  citation_volume=65;  citation_pages=181-202; " /><meta name="citation_reference" content="citation_title=Trait diversity and potential for selection indices based on variation among regionally adapted processing tomato germplasm; J. Am. Soc. Hortic. Sci.;  citation_year=2012;  citation_volume=137;  citation_pages=427-37; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic marker analysis; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2128-9; " /><meta name="citation_reference" content="citation_title=GCTA: a tool for genome-wide complex trait analysis; Am. J. Hum. Genet.;  citation_year=2011;  citation_volume=88;  citation_pages=76-82; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=Detecting the number of clusters of individuals using the software structure: a simulation study; Mol. Ecol.;  citation_year=2005;  citation_volume=14;  citation_pages=2611-20; " /><meta name="citation_reference" content="citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1801-6; " /><meta name="citation_reference" content="citation_title=DISTRUCT: a program for the graphical display of population structure; Mol. Ecol. Notes;  citation_year=2004;  citation_volume=4;  citation_pages=137-8; " /><meta name="citation_reference" content="citation_title=Estimation of genetic distances and phylogenetic trees from DNA analysis; citation_publisher=University of Guelph, Guelph, Ontario, Canada;  citation_year=1983;  citation_pages=405-12; " /><meta name="citation_reference" content="citation_title=SPAGeDi: a versatile computer program to analyse spatial genetic structure at the individual or population levels; Mol. Ecol. Notes;  citation_year=2002;  citation_volume=2;  citation_pages=618-20; " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-5; " /><meta name="citation_reference" content="citation_title=Genetic analysis and characterization of a new maize association mapping panel for quantitative trait loci dissection; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=417-31; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; J. R. Stat. Soc. B;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping in Norwegian Red cattle identifies quantitative trait loci for fertility and milk production on BTA12; Anim. Genet.;  citation_year=2011;  citation_volume=42;  citation_pages=466-74; " /><meta name="citation_reference" content="citation_title=Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species; Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=125-38; " /><meta name="citation_reference" content="citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e28334" /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; Am. J. Hum. Genet.;  citation_year=1980;  citation_volume=32;  citation_pages=314-31; " /><meta name="citation_reference" content="citation_title=Introgression of genomic components from Chinese Brassica rapa contributes to widening the genetic diversity in rapeseed (B. napus L.), with emphasis on the evolution of Chinese rapeseed; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=49-54; " /><meta name="citation_reference" content="citation_title=Rape breeding by interspecific crossing between Brassica napus and Brassica campestris in Japan; Japan Agric. Res. Quart.;  citation_year=1970;  citation_volume=5;  citation_pages=5-10; " /><meta name="citation_reference" content="citation_title=Exploitation of the late flowering species Brassica oleracea L. for the improvement of earliness in B. napus L.: an untraditional approach; Euphytica;  citation_year=2011;  citation_volume=177;  citation_pages=365-74; " /><meta name="citation_reference" content="citation_title=Structure of linkage disequilibrium in plants; Annu. Rev. Plant Biol.;  citation_year=2003;  citation_volume=54;  citation_pages=357-74; " /><meta name="citation_reference" content="citation_title=Genetic diversity of oilseed Brassica napus germplasm based on restriction fragment length polymorphisms; Theor. Appl. Genet.;  citation_year=1994;  citation_volume=88;  citation_pages=662-8; " /><meta name="citation_reference" content="citation_title=Analysis of genetic diversity in the Brassica napus L. Gene pool using SSR markers; Genet. Res. Crop Evol.;  citation_year=2006;  citation_volume=53;  citation_pages=793-802; " /><meta name="citation_reference" content="citation_title=Intersubgenomic heterosis in seed yield potential observed in a new type of Brassica napus introgressed with partial Brassica rapa genome; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1187-94; " /><meta name="citation_reference" content="citation_title=Genetic neighbourhood of clone structures in eelgrass meadows quantified by spatial autocorrelation of microsatellite markers; Heredity (Edinb);  citation_year=2003;  citation_volume=91;  citation_pages=448-55; " /><meta name="citation_reference" content="citation_title=Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (Brassica napus ssp. napus); Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=1035-49; " /><meta name="citation_reference" content="citation_title=Principal components analysis corrects for stratification in genome-wide association studies; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=904-9; " /><meta name="citation_reference" content="citation_title=Characterization of a global germplasm collection and its potential utilization for analysis of complex quantitative traits in maize; Mol. Breed.;  citation_year=2011;  citation_volume=28;  citation_pages=511-26; " /><meta name="citation_reference" content="citation_title=Association mapping of maturity and plant height using SNP markers with the sorghum mini core collection; Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=2003-15; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies for agronomical traits in a world wide spring barley collection; BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=16" /><meta name="citation_reference" content="citation_title=An Arabidopsis example of association mapping in structured samples; PLoS Genet.;  citation_year=2007;  citation_volume=3;  citation_pages=e4" /><meta name="citation_reference" content="citation_title=A functional genomics resource for Brassica napus: development of an EMS mutagenized population and discovery of FAE1 point mutations by TILLING; New Phytol.;  citation_year=2008;  citation_volume=180;  citation_pages=751-65; " /><meta name="citation_reference" content="citation_title=Zero erucic acid trait of rapeseed (Brassica napus L.) results from a deletion of four base pairs in the fatty acid elongase 1 gene; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=491-9; " /><meta name="citation_reference" content="citation_title=Homoeologous loci control the accumulation of seed glucosinolates in oilseed rape (Brassica napus); Genome;  citation_year=2003;  citation_volume=46;  citation_pages=454-60; " /><meta name="citation_reference" content="citation_title=Detection of loci controlling seed glucosinolate content and their association with Sclerotinia resistance in Brassica napus; Plant Breed.;  citation_year=2003;  citation_volume=122;  citation_pages=19-23; " /><meta name="citation_reference" content="citation_title=Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=6478-83; " /><meta name="citation_reference" content="citation_title=Targeted silencing of BjMYB28 transcription factor gene directs development of low glucosinolate lines in oilseed Brassica juncea; Plant Biotechnol. J;  citation_year=2013;  citation_volume=11;  citation_pages=855-66; " /><meta name="citation_reference" content="citation_title=QTL analysis of an intervarietal set of substitution lines in Brassica napus: (i) seed oil content and fatty acid composition; Heredity (Edinb);  citation_year=2003;  citation_volume=90;  citation_pages=39-48; " /><meta name="citation_reference" content="citation_title=Mapping the genome of rapeseed (Brassica napus L.). II. Localization of genes controlling erucic acid synthesis and seed oil content; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=972-7; " /><meta name="citation_reference" content="citation_title=A comparative linkage map of oilseed rape and its use for QTL analysis of seed oil and erucic acid content; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=114;  citation_pages=67-80; " /><meta name="citation_reference" content="citation_title=Comparative mapping of quantitative trait loci involved in heterosis for seedling and yield traits in oilseed rape (Brassica napus L.); Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=271-81; " /><meta name="citation_reference" content="citation_title=Mapping of quantitative trait loci and development of allele-specific markers for seed weight in Brassica napus; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=1289-301; " /><meta name="citation_reference" content="citation_title=Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (Brassica napus L.): 1. Identification of genomic regions from winter germplasm; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=549-61; " /><meta name="citation_reference" content="citation_title=Genetic and correlation analysis of silique-traits in Brassica napus L. by quantitative trait locus mapping; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=21-31; " /><meta name="citation_reference" content="citation_title=Genome-wide association study identifies candidate genes that affect plant height in Chinese elite maize (Zea mays L.) inbred lines; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e29229" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/355/1217863/dsu002.pdf" /><meta name="description" content="Abstract.  Association mapping can quickly and efficiently dissect complex agronomic traits. Rapeseed is one of the most economically important polyploid oil cr" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/355/445136" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/355/445136" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/355/445136");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu002';
    </script>
<title>
	Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed (Brassica napus L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./445136" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdzRATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIEZlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcCBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEtleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgT2lsIENyb3BzLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgTm8uIDIgWHVkb25nIFNlY29uZCBSb2FkLCBIdWJlaSBQcm92aW5jZSwgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIEJpeXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2lsIENyb3AgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE9pbCBDcm9wcywgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE5vLiAyIFh1ZG9uZyBTZWNvbmQgUm9hZCwgSHViZWkgUHJvdmluY2UsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJYdSwgS3VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2lsIENyb3AgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE9pbCBDcm9wcywgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE5vLiAyIFh1ZG9uZyBTZWNvbmQgUm9hZCwgSHViZWkgUHJvdmluY2UsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXdSwgSmluZmVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik9pbCBDcm9wIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBPaWwgQ3JvcHMsIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBOby4gMiBYdWRvbmcgU2Vjb25kIFJvYWQsIEh1YmVpIFByb3ZpbmNlLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU29uZywgV2VpbGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2lsIENyb3AgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE9pbCBDcm9wcywgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE5vLiAyIFh1ZG9uZyBTZWNvbmQgUm9hZCwgSHViZWkgUHJvdmluY2UsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW5jcm9mdCwgSWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIFlvcmssIFlvcmssIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFycGVyLCBBbmRyZWEgTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgWW9yaywgWW9yaywgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUcmljaywgTWFydGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSm9obiBJbm5lcyBDZW50cmUsIE5vcndpY2ggUmVzZWFyY2ggUGFyaywgTm9yd2ljaCBOUjQgN1VILCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgU2hlbmd5aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik9pbCBDcm9wIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBPaWwgQ3JvcHMsIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBOby4gMiBYdWRvbmcgU2Vjb25kIFJvYWQsIEh1YmVpIFByb3ZpbmNlLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2FvLCBHdWl6aGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2lsIENyb3AgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE9pbCBDcm9wcywgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE5vLiAyIFh1ZG9uZyBTZWNvbmQgUm9hZCwgSHViZWkgUHJvdmluY2UsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBOaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iT2lsIENyb3AgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBLZXkgTGFib3JhdG9yeSBvZiBCaW9sb2d5IGFuZCBHZW5ldGljIEltcHJvdmVtZW50IG9mIE9pbCBDcm9wcywgTWluaXN0cnkgb2YgQWdyaWN1bHR1cmUsIE5vLiAyIFh1ZG9uZyBTZWNvbmQgUm9hZCwgSHViZWkgUHJvdmluY2UsIFd1aGFuIDQzMDA2MiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW4sIEd1aXhpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik9pbCBDcm9wIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBPaWwgQ3JvcHMsIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBOby4gMiBYdWRvbmcgU2Vjb25kIFJvYWQsIEh1YmVpIFByb3ZpbmNlLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUWlhbywgSmlhbmd3ZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcCBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEtleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgT2lsIENyb3BzLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgTm8uIDIgWHVkb25nIFNlY29uZCBSb2FkLCBIdWJlaSBQcm92aW5jZSwgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBKdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcCBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEtleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgT2lsIENyb3BzLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgTm8uIDIgWHVkb25nIFNlY29uZCBSb2FkLCBIdWJlaSBQcm92aW5jZSwgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBIYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcCBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEtleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgT2lsIENyb3BzLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgTm8uIDIgWHVkb25nIFNlY29uZCBSb2FkLCBIdWJlaSBQcm92aW5jZSwgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlhpYW8sIFhpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik9pbCBDcm9wIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBPaWwgQ3JvcHMsIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBOby4gMiBYdWRvbmcgU2Vjb25kIFJvYWQsIEh1YmVpIFByb3ZpbmNlLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIFRpYW55YW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJPaWwgQ3JvcCBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEtleSBMYWJvcmF0b3J5IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWMgSW1wcm92ZW1lbnQgb2YgT2lsIENyb3BzLCBNaW5pc3RyeSBvZiBBZ3JpY3VsdHVyZSwgTm8uIDIgWHVkb25nIFNlY29uZCBSb2FkLCBIdWJlaSBQcm92aW5jZSwgV3VoYW4gNDMwMDYyLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild1LCBYaWFvbWluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik9pbCBDcm9wIFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgS2V5IExhYm9yYXRvcnkgb2YgQmlvbG9neSBhbmQgR2VuZXRpYyBJbXByb3ZlbWVudCBvZiBPaWwgQ3JvcHMsIE1pbmlzdHJ5IG9mIEFncmljdWx0dXJlLCBOby4gMiBYdWRvbmcgU2Vjb25kIFJvYWQsIEh1YmVpIFByb3ZpbmNlLCBXdWhhbiA0MzAwNjIsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtV2lkZSBBc3NvY2lhdGlvbiBTdHVkeSBEaXNzZWN0cyB0aGUgR2VuZXRpYyBBcmNoaXRlY3R1cmUgb2YgU2VlZCBXZWlnaHQgYW5kIFNlZWQgUXVhbGl0eSBpbiBSYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBvZiB0aGUgbWljcm9zdHJ1Y3R1cmUgb2YgZ2Vub21lIHNlZ21lbnRzIGluIEJyYXNzaWNhIG5hcHVzIGFuZCBpdHMgZGlwbG9pZCByZWxhdGl2ZXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NzI1LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtYW5hbHlzaXMgaW4gQnJhc3NpY2Egd2l0aCBzcGVjaWFsIHJlZmVyZW5jZSB0byB0aGUgZXhwZXJpbWVudGFsIGZvcm1hdGlvbiBvZiBCLiBuYXB1cyBhbmQgcGVjdWxpYXIgbW9kZSBvZiBmZXJ0aWxpemF0aW9uOyBKYXBhbiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTM1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zODktNDUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIGRvbWVzdGljYXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1FbHNldmllciwgQW1zdGVyZGFtOyBCaW9sb2d5IG9mIEJyYXNzaWNhIGNvZW5vc3BlY2llczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3BhZ2VzPTMzLTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b3J5LCBldm9sdXRpb24sIGFuZCBkb21lc3RpY2F0aW9uIG9mIEJyYXNzaWNhIGNyb3BzOyBjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obiBXaWxleSAmYW1wOyBTb25zLCBJbmMuLCBOZXcgSmVyc2V5OyBQbGFudCBCcmVlZGluZyBSZXZpZXdzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPXZvbC4gMzU7ICBjaXRhdGlvbl9wYWdlcz0xOS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVByb2MgRXVjYXJwaWEgQ29uZmVyZW5jZSBvbiB0aGUgYnJlZWRpbmcgb2YgQ3J1Y2lmZXJvdXMgY3JvcHMsIFdhZ2VuaW5nZW47IFRoZSBkb21lc3RpY2F0aW9uIG9mIEJyYXNzaWNhIGNyb3BzOyAgY2l0YXRpb25feWVhcj0xOTc5OyAgY2l0YXRpb25fcGFnZXM9NDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbmRpdGlvbmFsIFFUTCBtYXBwaW5nIG9mIG9pbCBjb250ZW50IGluIHJhcGVzZWVkIHdpdGggcmVzcGVjdCB0byBwcm90ZWluIGNvbnRlbnQgYW5kIHRyYWl0cyByZWxhdGVkIHRvIHBsYW50IGRldmVsb3BtZW50IGFuZCBncmFpbiB5aWVsZDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0zMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHRoZSBnZW5vbWUgb2YgcmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKS4gSS4gQ29uc3RydWN0aW9uIG9mIGFuIFJGTFAgbGlua2FnZSBtYXAgYW5kIGxvY2FsaXphdGlvbiBvZiBRVExzIGZvciBzZWVkIGdsdWNvc2lub2xhdGUgY29udGVudDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTE5NC0yMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgUVRMIGNvbnRyb2xsaW5nIGZhdHR5IGFjaWQgY29tcG9zaXRpb24gaW4gYSBkb3VibGVkIGhhcGxvaWQgcmFwZXNlZWQgcG9wdWxhdGlvbiBzZWdyZWdhdGluZyBmb3Igb2lsIGNvbnRlbnQ7IE1vbC4gQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMTUtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsb3dlcmluZyB0aW1lIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGFuYWx5c2lzIG9mIG9pbHNlZWQgQnJhc3NpY2EgaW4gbXVsdGlwbGUgZW52aXJvbm1lbnRzIGFuZCBnZW5vbWV3aWRlIGFsaWdubWVudCB3aXRoIEFyYWJpZG9wc2lzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzc7ICBjaXRhdGlvbl9wYWdlcz0yNDMzLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnJhdmVsaW5nIHRoZSBjb21wbGV4IHRyYWl0IG9mIGNyb3AgeWllbGQgd2l0aCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBtYXBwaW5nIGluIEJyYXNzaWNhIG5hcHVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODI7ICBjaXRhdGlvbl9wYWdlcz04NTEtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgb2YgaGV0ZXJvc2lzIGZvciB5aWVsZCBhbmQgeWllbGQgY29tcG9uZW50cyBpbiByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pIGJ5IHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBtYXBwaW5nOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzk7ICBjaXRhdGlvbl9wYWdlcz0xNTQ3LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhIG1ham9yIFFUTCBmb3Igc2lsaXF1ZSBsZW5ndGggYW5kIHNlZWQgd2VpZ2h0IGluIG9pbHNlZWQgcmFwZSAoQnJhc3NpY2EgbmFwdXMgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTI4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIG1hcHBpbmcgb2YgYSBtYWpvciBkb21pbmFudCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgY29udHJvbGxpbmcgc2VlZHMgcGVyIHNpbGlxdWUgYXMgYSBzaW5nbGUgTWVuZGVsaWFuIGZhY3RvciBpbiBCcmFzc2ljYSBuYXB1cyBMLjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gZ2VuZXRpY3MgaW4gcGxhbnRzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU2OyAgY2l0YXRpb25fcGFnZXM9NzIwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2UgZGlzZXF1aWxpYnJpdW06IHdoYXQgaGlzdG9yeSBoYXMgdG8gdGVsbCB1czsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgzLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIHJldmVhbHMgYSByaWNoIGdlbmV0aWMgYXJjaGl0ZWN0dXJlIG9mIGNvbXBsZXggdHJhaXRzIGluIE9yeXphIHNhdGl2YTsgTmF0LiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeSBvZiBsZWFmIGFyY2hpdGVjdHVyZSBpbiB0aGUgbWFpemUgbmVzdGVkIGFzc29jaWF0aW9uIG1hcHBpbmcgcG9wdWxhdGlvbjsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xNTktNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXMgb2YgMTQgYWdyb25vbWljIHRyYWl0cyBpbiByaWNlIGxhbmRyYWNlczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz05NjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgb2YgZmxvd2VyaW5nIHRpbWUgYW5kIGdyYWluIHlpZWxkIHRyYWl0cyBpbiBhIHdvcmxkd2lkZSBjb2xsZWN0aW9uIG9mIHJpY2UgZ2VybXBsYXNtOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTMyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IGRpc3NlY3RzIHRoZSBnZW5ldGljIGFyY2hpdGVjdHVyZSBvZiBvaWwgYmlvc3ludGhlc2lzIGluIG1haXplIGtlcm5lbHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NDMtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIDEwNyBwaGVub3R5cGVzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGluYnJlZCBsaW5lczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NTsgIGNpdGF0aW9uX3BhZ2VzPTYyNy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgcGh5bG9nZW55IGFuZCBrYXJ5b3R5cGUgZXZvbHV0aW9uIGluIHg9NyBjcnVjaWZlciBzcGVjaWVzIChCcmFzc2ljYWNlYWUpOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjU1OS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VnbWVudGFsIHN0cnVjdHVyZSBvZiB0aGUgQnJhc3NpY2EgbmFwdXMgZ2Vub21lIGJhc2VkIG9uIGNvbXBhcmF0aXZlIGFuYWx5c2lzIHdpdGggQXJhYmlkb3BzaXMgdGhhbGlhbmE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTc2NS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgbW9sZWN1bGFyIHZhcmlhdGlvbiBpbiBhIHNwZWNpZXMtd2lkZSBnZXJtcGxhc20gc2V0IG9mIEJyYXNzaWNhIG5hcHVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTE0MTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVudCBhbmQgc3RydWN0dXJlIG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gY2Fub2xhIHF1YWxpdHkgd2ludGVyIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9OTIxLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHN0cnVjdHVyZSBhbmQgbGlua2FnZSBkaXNlcXVpbGlicml1bSBwYXR0ZXJuIG9mIGEgcmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKSBhc3NvY2lhdGlvbiBtYXBwaW5nIHBhbmVsIHJldmVhbGVkIGJ5IG1pY3Jvc2F0ZWxsaXRlczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz00MzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG1hcHBpbmcgb2Ygc2VlZCBvaWwgY29udGVudCBpbiBCcmFzc2ljYSBuYXB1cyBhbmQgY29tcGFyaXNvbiB3aXRoIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGlkZW50aWZpZWQgZnJvbSBsaW5rYWdlIG1hcHBpbmc7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTkwOC0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gbWFwcGluZyBmb3IgcGhlbm9sb2dpY2FsLCBtb3JwaG9sb2dpY2FsLCBhbmQgcXVhbGl0eSB0cmFpdHMgaW4gY2Fub2xhIHF1YWxpdHkgd2ludGVyIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTg5OS05MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ2lvbmFsIGFzc29jaWF0aW9uIGFuYWx5c2lzIGRlbGluZWF0ZXMgYSBzZXF1ZW5jZWQgY2hyb21vc29tZSByZWdpb24gaW5mbHVlbmNpbmcgYW50aW51dHJpdGl2ZSBzZWVkIG1lYWwgY29tcG91bmRzIGluIG9pbHNlZWQgcmFwZTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9OTE3LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gKFNOUCkgZGlzY292ZXJ5IGluIHRoZSBwb2x5cGxvaWQgQnJhc3NpY2EgbmFwdXMgdXNpbmcgU29sZXhhIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgUGxhbnQgQmlvdGVjaG5vbC4gSjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzM0LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNzZWN0aW5nIHRoZSBnZW5vbWUgb2YgdGhlIHBvbHlwbG9pZCBjcm9wIG9pbHNlZWQgcmFwZSBieSB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmc7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz03NjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IFNOUC1iYXNlZCBnZW5ldGljIG1hcCBkZXZlbG9wbWVudCBhbmQgbGlua2FnZSBkaXNlcXVpbGlicml1bSBhc3Nlc3NtZW50IGluIEJyYXNzaWNhIG5hcHVzIEwuOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBtZXNvcG9seXBsb2lkIGNyb3Agc3BlY2llcyBCcmFzc2ljYSByYXBhOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpdmUgdHJhbnNjcmlwdG9taWNzIG9mIHRyYWl0cyBpbiB0aGUgcG9seXBsb2lkIGNyb3Agc3BlY2llcyBCcmFzc2ljYSBuYXB1czsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTc5OC04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjaG5lbGxlIHVuZCBlaW5mYWNoZSBBbmFseXNlbiBkZXIgRmV0dHPDpHVyZXp1c2FtbWVuc2V0enVuZyBpbiBlaW56ZWxuZW4gUmFwc2tvdHlsZWRvbmVuIEkuIEdhc2Nocm9tYXRvZ3JhcGhpc2NoZSB1bmQgcGFwaWVyY2hyb21hdG9ncmFwaGlzY2hlIE1ldGhvZGVuOyBaIFBmbGFuemVuesO8Y2h0OyAgY2l0YXRpb25feWVhcj0xOTcxOyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MTgxLTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhaXQgZGl2ZXJzaXR5IGFuZCBwb3RlbnRpYWwgZm9yIHNlbGVjdGlvbiBpbmRpY2VzIGJhc2VkIG9uIHZhcmlhdGlvbiBhbW9uZyByZWdpb25hbGx5IGFkYXB0ZWQgcHJvY2Vzc2luZyB0b21hdG8gZ2VybXBsYXNtOyBKLiBBbS4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9NDI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3dlck1hcmtlcjogYW4gaW50ZWdyYXRlZCBhbmFseXNpcyBlbnZpcm9ubWVudCBmb3IgZ2VuZXRpYyBtYXJrZXIgYW5hbHlzaXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjEyOC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HQ1RBOiBhIHRvb2wgZm9yIGdlbm9tZS13aWRlIGNvbXBsZXggdHJhaXQgYW5hbHlzaXM7IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz05NDUtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGluZyB0aGUgbnVtYmVyIG9mIGNsdXN0ZXJzIG9mIGluZGl2aWR1YWxzIHVzaW5nIHRoZSBzb2Z0d2FyZSBzdHJ1Y3R1cmU6IGEgc2ltdWxhdGlvbiBzdHVkeTsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI2MTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVU1QUDogYSBjbHVzdGVyIG1hdGNoaW5nIGFuZCBwZXJtdXRhdGlvbiBwcm9ncmFtIGZvciBkZWFsaW5nIHdpdGggbGFiZWwgc3dpdGNoaW5nIGFuZCBtdWx0aW1vZGFsaXR5IGluIGFuYWx5c2lzIG9mIHBvcHVsYXRpb24gc3RydWN0dXJlOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE4MDEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RElTVFJVQ1Q6IGEgcHJvZ3JhbSBmb3IgdGhlIGdyYXBoaWNhbCBkaXNwbGF5IG9mIHBvcHVsYXRpb24gc3RydWN0dXJlOyBNb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBnZW5ldGljIGRpc3RhbmNlcyBhbmQgcGh5bG9nZW5ldGljIHRyZWVzIGZyb20gRE5BIGFuYWx5c2lzOyBjaXRhdGlvbl9wdWJsaXNoZXI9VW5pdmVyc2l0eSBvZiBHdWVscGgsIEd1ZWxwaCwgT250YXJpbywgQ2FuYWRhOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fcGFnZXM9NDA1LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TUEFHZURpOiBhIHZlcnNhdGlsZSBjb21wdXRlciBwcm9ncmFtIHRvIGFuYWx5c2Ugc3BhdGlhbCBnZW5ldGljIHN0cnVjdHVyZSBhdCB0aGUgaW5kaXZpZHVhbCBvciBwb3B1bGF0aW9uIGxldmVsczsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NjE4LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHVuaWZpZWQgbWl4ZWQtbW9kZWwgbWV0aG9kIGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nIHRoYXQgYWNjb3VudHMgZm9yIG11bHRpcGxlIGxldmVscyBvZiByZWxhdGVkbmVzczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VEFTU0VMOiBzb2Z0d2FyZSBmb3IgYXNzb2NpYXRpb24gbWFwcGluZyBvZiBjb21wbGV4IHRyYWl0cyBpbiBkaXZlcnNlIHNhbXBsZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjYzMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbmV3IG1haXplIGFzc29jaWF0aW9uIG1hcHBpbmcgcGFuZWwgZm9yIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGRpc3NlY3Rpb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9NDE3LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGU6IGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyBKLiBSLiBTdGF0LiBTb2MuIEI7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yODktMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIGluIE5vcndlZ2lhbiBSZWQgY2F0dGxlIGlkZW50aWZpZXMgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIGZlcnRpbGl0eSBhbmQgbWlsayBwcm9kdWN0aW9uIG9uIEJUQTEyOyBBbmltLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz00NjYtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uLCBjaGFyYWN0ZXJpemF0aW9uIGFuZCBpbnRlcnByZXRhdGlvbiBvZiBzaW5nbGUtbnVjbGVvdGlkZSBzZXF1ZW5jZSB2YXJpYXRpb24gaW4gYWxsb3BvbHlwbG9pZCBjcm9wIHNwZWNpZXM7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTI1LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxhcmdlIG1haXplIChaZWEgbWF5cyBMLikgU05QIGdlbm90eXBpbmcgYXJyYXk6IGRldmVsb3BtZW50IGFuZCBnZXJtcGxhc20gZ2Vub3R5cGluZywgYW5kIGdlbmV0aWMgbWFwcGluZyB0byBjb21wYXJlIHdpdGggdGhlIEI3MyByZWZlcmVuY2UgZ2Vub21lOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI4MzM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBpbiBtYW4gdXNpbmcgcmVzdHJpY3Rpb24gZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGlzbXM7IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTMxNC0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9ncmVzc2lvbiBvZiBnZW5vbWljIGNvbXBvbmVudHMgZnJvbSBDaGluZXNlIEJyYXNzaWNhIHJhcGEgY29udHJpYnV0ZXMgdG8gd2lkZW5pbmcgdGhlIGdlbmV0aWMgZGl2ZXJzaXR5IGluIHJhcGVzZWVkIChCLiBuYXB1cyBMLiksIHdpdGggZW1waGFzaXMgb24gdGhlIGV2b2x1dGlvbiBvZiBDaGluZXNlIHJhcGVzZWVkOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBlIGJyZWVkaW5nIGJ5IGludGVyc3BlY2lmaWMgY3Jvc3NpbmcgYmV0d2VlbiBCcmFzc2ljYSBuYXB1cyBhbmQgQnJhc3NpY2EgY2FtcGVzdHJpcyBpbiBKYXBhbjsgSmFwYW4gQWdyaWMuIFJlcy4gUXVhcnQuOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb2l0YXRpb24gb2YgdGhlIGxhdGUgZmxvd2VyaW5nIHNwZWNpZXMgQnJhc3NpY2Egb2xlcmFjZWEgTC4gZm9yIHRoZSBpbXByb3ZlbWVudCBvZiBlYXJsaW5lc3MgaW4gQi4gbmFwdXMgTC46IGFuIHVudHJhZGl0aW9uYWwgYXBwcm9hY2g7IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzc7ICBjaXRhdGlvbl9wYWdlcz0zNjUtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZSBvZiBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIGluIHBsYW50czsgQW5udS4gUmV2LiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgb2Ygb2lsc2VlZCBCcmFzc2ljYSBuYXB1cyBnZXJtcGxhc20gYmFzZWQgb24gcmVzdHJpY3Rpb24gZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGlzbXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz02NjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgZ2VuZXRpYyBkaXZlcnNpdHkgaW4gdGhlIEJyYXNzaWNhIG5hcHVzIEwuIEdlbmUgcG9vbCB1c2luZyBTU1IgbWFya2VyczsgR2VuZXQuIFJlcy4gQ3JvcCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTc5My04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyc3ViZ2Vub21pYyBoZXRlcm9zaXMgaW4gc2VlZCB5aWVsZCBwb3RlbnRpYWwgb2JzZXJ2ZWQgaW4gYSBuZXcgdHlwZSBvZiBCcmFzc2ljYSBuYXB1cyBpbnRyb2dyZXNzZWQgd2l0aCBwYXJ0aWFsIEJyYXNzaWNhIHJhcGEgZ2Vub21lOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTExODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbmVpZ2hib3VyaG9vZCBvZiBjbG9uZSBzdHJ1Y3R1cmVzIGluIGVlbGdyYXNzIG1lYWRvd3MgcXVhbnRpZmllZCBieSBzcGF0aWFsIGF1dG9jb3JyZWxhdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzOyBIZXJlZGl0eSAoRWRpbmIpOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NDQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBvZiBnZW5lLWxpbmtlZCBTU1IgbWFya2VycyB0byBzZWVkIGdsdWNvc2lub2xhdGUgY29udGVudCBpbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIHNzcC4gbmFwdXMpOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTEwMzUtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW5jaXBhbCBjb21wb25lbnRzIGFuYWx5c2lzIGNvcnJlY3RzIGZvciBzdHJhdGlmaWNhdGlvbiBpbiBnZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTkwNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2xvYmFsIGdlcm1wbGFzbSBjb2xsZWN0aW9uIGFuZCBpdHMgcG90ZW50aWFsIHV0aWxpemF0aW9uIGZvciBhbmFseXNpcyBvZiBjb21wbGV4IHF1YW50aXRhdGl2ZSB0cmFpdHMgaW4gbWFpemU7IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NTExLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBtYXBwaW5nIG9mIG1hdHVyaXR5IGFuZCBwbGFudCBoZWlnaHQgdXNpbmcgU05QIG1hcmtlcnMgd2l0aCB0aGUgc29yZ2h1bSBtaW5pIGNvcmUgY29sbGVjdGlvbjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz0yMDAzLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIGZvciBhZ3Jvbm9taWNhbCB0cmFpdHMgaW4gYSB3b3JsZCB3aWRlIHNwcmluZyBiYXJsZXkgY29sbGVjdGlvbjsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBBcmFiaWRvcHNpcyBleGFtcGxlIG9mIGFzc29jaWF0aW9uIG1hcHBpbmcgaW4gc3RydWN0dXJlZCBzYW1wbGVzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzb3VyY2UgZm9yIEJyYXNzaWNhIG5hcHVzOiBkZXZlbG9wbWVudCBvZiBhbiBFTVMgbXV0YWdlbml6ZWQgcG9wdWxhdGlvbiBhbmQgZGlzY292ZXJ5IG9mIEZBRTEgcG9pbnQgbXV0YXRpb25zIGJ5IFRJTExJTkc7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTc1MS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WmVybyBlcnVjaWMgYWNpZCB0cmFpdCBvZiByYXBlc2VlZCAoQnJhc3NpY2EgbmFwdXMgTC4pIHJlc3VsdHMgZnJvbSBhIGRlbGV0aW9uIG9mIGZvdXIgYmFzZSBwYWlycyBpbiB0aGUgZmF0dHkgYWNpZCBlbG9uZ2FzZSAxIGdlbmU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9NDkxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9lb2xvZ291cyBsb2NpIGNvbnRyb2wgdGhlIGFjY3VtdWxhdGlvbiBvZiBzZWVkIGdsdWNvc2lub2xhdGVzIGluIG9pbHNlZWQgcmFwZSAoQnJhc3NpY2EgbmFwdXMpOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz00NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBvZiBsb2NpIGNvbnRyb2xsaW5nIHNlZWQgZ2x1Y29zaW5vbGF0ZSBjb250ZW50IGFuZCB0aGVpciBhc3NvY2lhdGlvbiB3aXRoIFNjbGVyb3RpbmlhIHJlc2lzdGFuY2UgaW4gQnJhc3NpY2EgbmFwdXM7IFBsYW50IEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0xOS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T21pY3MtYmFzZWQgaWRlbnRpZmljYXRpb24gb2YgQXJhYmlkb3BzaXMgTXliIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZWd1bGF0aW5nIGFsaXBoYXRpYyBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NjQ3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgc2lsZW5jaW5nIG9mIEJqTVlCMjggdHJhbnNjcmlwdGlvbiBmYWN0b3IgZ2VuZSBkaXJlY3RzIGRldmVsb3BtZW50IG9mIGxvdyBnbHVjb3Npbm9sYXRlIGxpbmVzIGluIG9pbHNlZWQgQnJhc3NpY2EganVuY2VhOyBQbGFudCBCaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODU1LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgYW5hbHlzaXMgb2YgYW4gaW50ZXJ2YXJpZXRhbCBzZXQgb2Ygc3Vic3RpdHV0aW9uIGxpbmVzIGluIEJyYXNzaWNhIG5hcHVzOiAoaSkgc2VlZCBvaWwgY29udGVudCBhbmQgZmF0dHkgYWNpZCBjb21wb3NpdGlvbjsgSGVyZWRpdHkgKEVkaW5iKTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTM5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHRoZSBnZW5vbWUgb2YgcmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKS4gSUkuIExvY2FsaXphdGlvbiBvZiBnZW5lcyBjb250cm9sbGluZyBlcnVjaWMgYWNpZCBzeW50aGVzaXMgYW5kIHNlZWQgb2lsIGNvbnRlbnQ7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz05NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJhdGl2ZSBsaW5rYWdlIG1hcCBvZiBvaWxzZWVkIHJhcGUgYW5kIGl0cyB1c2UgZm9yIFFUTCBhbmFseXNpcyBvZiBzZWVkIG9pbCBhbmQgZXJ1Y2ljIGFjaWQgY29udGVudDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz02Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWFwcGluZyBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbnZvbHZlZCBpbiBoZXRlcm9zaXMgZm9yIHNlZWRsaW5nIGFuZCB5aWVsZCB0cmFpdHMgaW4gb2lsc2VlZCByYXBlIChCcmFzc2ljYSBuYXB1cyBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MjcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIG9mIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGFuZCBkZXZlbG9wbWVudCBvZiBhbGxlbGUtc3BlY2lmaWMgbWFya2VycyBmb3Igc2VlZCB3ZWlnaHQgaW4gQnJhc3NpY2EgbmFwdXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9MTI4OS0zMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSB0cmFpdCBhbmFseXNpcyBvZiBzZWVkIHlpZWxkIGFuZCBvdGhlciBjb21wbGV4IHRyYWl0cyBpbiBoeWJyaWQgc3ByaW5nIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik6IDEuIElkZW50aWZpY2F0aW9uIG9mIGdlbm9taWMgcmVnaW9ucyBmcm9tIHdpbnRlciBnZXJtcGxhc207IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NTQ5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBjb3JyZWxhdGlvbiBhbmFseXNpcyBvZiBzaWxpcXVlLXRyYWl0cyBpbiBCcmFzc2ljYSBuYXB1cyBMLiBieSBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgbWFwcGluZzsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0yMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgaWRlbnRpZmllcyBjYW5kaWRhdGUgZ2VuZXMgdGhhdCBhZmZlY3QgcGxhbnQgaGVpZ2h0IGluIENoaW5lc2UgZWxpdGUgbWFpemUgKFplYSBtYXlzIEwuKSBpbmJyZWQgbGluZXM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjkyMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNC8zNTUvMTIxNzg2My9kc3UwMDIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFzc29jaWF0aW9uIG1hcHBpbmcgY2FuIHF1aWNrbHkgYW5kIGVmZmljaWVudGx5IGRpc3NlY3QgY29tcGxleCBhZ3Jvbm9taWMgdHJhaXRzLiBSYXBlc2VlZCBpcyBvbmUgb2YgdGhlIG1vc3QgZWNvbm9taWNhbGx5IGltcG9ydGFudCBwb2x5cGxvaWQgb2lsIGNyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDUxMzYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk/pbOqxAtQM8bPl2XeS0Jxz5BEuc=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f355%2f445136" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f355%2f445136" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6655753">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655753">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655755">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655755">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655761">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655761">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655776">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655776">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655796">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655796">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655805">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655807">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655807">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6655809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6655809">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/4/341/444964">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/4/369/445591">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed (<em>Brassica napus</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Feng Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Biyun Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Biyun+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Biyun Chen</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kun Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kun+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kun Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinfeng Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jinfeng+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jinfeng Wu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weilin Song</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weilin+Song">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Song W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Song W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Weilin Song</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ian Bancroft</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biology</div>, <div class="insititution">University of York</div>, <div class="addr-line">York</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ian+Bancroft">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bancroft I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bancroft I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ian Bancroft</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea L. Harper</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biology</div>, <div class="insititution">University of York</div>, <div class="addr-line">York</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrea+L.+Harper">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harper A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harper A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Andrea L. Harper</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Trick</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">John Innes Centre</div>, <div class="addr-line">Norwich Research Park, Norwich NR4 7UH</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Trick">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Trick M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Trick M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Martin Trick</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shengyi Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shengyi+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shengyi Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guizhen Gao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guizhen+Gao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gao G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gao G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Guizhen Gao</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nian Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nian+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nian Wang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guixin Yan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guixin+Yan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yan G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yan G&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Guixin Yan</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiangwei Qiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiangwei+Qiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qiao J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qiao J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jiangwei Qiao</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jun Li</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hao Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hao+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hao Li</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xin Xiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xin+Xiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xiao X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xiao X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xin Xiao</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tianyao Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tianyao+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tianyao Zhang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoming Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Oil Crop Research Institute, Chinese Academy of Agricultural Sciences, Key Laboratory of Biology and Genetic Improvement of Oil Crops</div>, <div class="insititution">Ministry of Agriculture</div>, <div class="addr-line">No. 2 Xudong Second Road, Hubei Province, Wuhan 430062</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-8681-2960. Fax. +86 27-8681-2960. E-mail: <a href="mailto:wuxm@oilcrops.cn" target="_blank">wuxm@oilcrops.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaoming+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xiaoming Wu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 355–367, <a href='https://doi.org/10.1093/dnares/dsu002'>https://doi.org/10.1093/dnares/dsu002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 February 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 October 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 January 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="445136" href="/dnaresearch/article-pdf/21/4/355/1217863/dsu002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Feng Li, Biyun Chen, Kun Xu, Jinfeng Wu, Weilin Song, Ian Bancroft, Andrea L. Harper, Martin Trick, Shengyi Liu, Guizhen Gao, Nian Wang, Guixin Yan, Jiangwei Qiao, Jun Li, Hao Li, Xin Xiao, Tianyao Zhang, Xiaoming Wu;  Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed (<em>Brassica napus</em> L.), <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 355–367, <a href='https://doi.org/10.1093/dnares/dsu002'>https://doi.org/10.1093/dnares/dsu002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/445136?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/445136?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Association%20Study%20Dissects%20the%20Genetic%20Architecture%20of%20Seed%20Weight%20and%20Seed%20Quality%20in%20Rapeseed%20(%26lt%3Bem%26gt%3BBrassica%20napus%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2014-02-07&amp;volumeNum=21&amp;issueNum=4&amp;author=Li%2C%20Feng%3B%20Chen%2C%20Biyun&amp;startPage=355&amp;endPage=367&amp;contentId=10.1093%2Fdnares%2Fdsu002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f355%2f445136" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6655753 id="6655753" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Association mapping can quickly and efficiently dissect complex agronomic traits. Rapeseed is one of the most economically important polyploid oil crops, although its genome sequence is not yet published. In this study, a recently developed 60K <em>Brassica</em> Infinium<sup>®</sup> SNP array was used to analyse an association panel with 472 accessions. The single-nucleotide polymorphisms (SNPs) of the array were <em>in silico</em> mapped using ‘pseudomolecules’ representative of the genome of rapeseed to establish their hypothetical order and to perform association mapping of seed weight and seed quality. As a result, two significant associations on A8 and C3 of <em>Brassica napus</em> were detected for erucic acid content, and the peak SNPs were found to be only 233 and 128 kb away from the key genes <em>BnaA.FAE1</em> and <em>BnaC.FAE1. BnaA.FAE1</em> was also identified to be significantly associated with the oil content. Orthologues of <em>Arabidopsis thaliana HAG1</em> were identified close to four clusters of SNPs associated with glucosinolate content on A9, C2, C7 and C9. For seed weight, we detected two association signals on A7 and A9, which were consistent with previous studies of quantitative trait loci mapping<em>.</em> The results indicate that our association mapping approach is suitable for fine mapping of the complex traits in rapeseed.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Brassica napus&quot;"><em>Brassica napus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;association mapping&quot;">association mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;seed quality&quot;">seed quality</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;seed weight&quot;">seed weight</a></div>                            <h2 scrollto-destination=6655755 id="6655755" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Rapeseed (<em>Brassica napus</em> L.; AACC, 2<em>n</em> = 38) is the one of the most important oil crops worldwide. <em>Brassica napus</em> is a recent allopolyploid species derived from natural interspecies hybridization between two phylogenetically close species, <em>B. rapa</em> (AA, 2<em>n</em> = 20) and <em>B. oleracea</em> (CC, 2<em>n</em> = 18) &lt;10 000 years ago.<sup><a reveal-id="DSU002C1" data-open="DSU002C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU002C2" data-open="DSU002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Rapeseed has a short domestication history of only ∼400–500 years.<sup><a reveal-id="DSU002C3  DSU002C5" data-open="DSU002C3  DSU002C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> During the breeding history of rapeseed, the most outstanding event is the introduction of two traits, zero seed erucic acid and low seed glucosinolate content (so called double-low, canola quality 00). However, many traits such as oil content, seed yield, disease resistance, etc. urgently need to be further improved in the modern cultivars. Molecular design breeding is currently one of the most available breeding methods. Screening-specific materials with some desired traits in a comparatively genetically diverse source of germplasm including old genotypes with high levels of erucic acid and seed glucosinolates and discovering the advantageous allelic variants by genetic analysis may advance molecular design breeding.</p><p>Genetic mapping of quantitative trait loci (QTL) in rapeseed is well established and has been employed to localize QTLs for quantitative traits, such as oil content,<sup><a reveal-id="DSU002C6" data-open="DSU002C6" class="link link-ref link-reveal xref-bibr">6</a></sup> glucosinolate content,<sup><a reveal-id="DSU002C7" data-open="DSU002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> fatty acid composition of the seed oil,<sup><a reveal-id="DSU002C8" data-open="DSU002C8" class="link link-ref link-reveal xref-bibr">8</a></sup> flowering time,<sup><a reveal-id="DSU002C9" data-open="DSU002C9" class="link link-ref link-reveal xref-bibr">9</a></sup> yield components,<sup><a reveal-id="DSU002C10  DSU002C13" data-open="DSU002C10  DSU002C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup> etc. In all these studies, although QTL mapping is very successful in detecting QTL, the genetic variation in the mapping population is restricted to only the two parents, and the markers for detected QTL are not necessarily transferable to other materials. Furthermore, the ability of fine mapping a QTL in these studies is limited by the frequency of polymorphic loci between the two parents and requires a population consisting of several thousands of individuals, making it laborious and time-consuming to perform. There is, so far, no report on map-based cloning of a causal gene in a QTL of rapeseed.</p><p>Association mapping, also called linkage disequilibrium (LD) mapping, which directly studies statistical associations between genetic markers and phenotypes in natural populations, is an alternative to QTL mapping.<sup><a reveal-id="DSU002C14" data-open="DSU002C14" class="link link-ref link-reveal xref-bibr">14</a></sup> An association mapping study utilizes the higher number of historical recombination events that have occurred throughout the entire evolutionary history of the mapping population, allowing ﬁne-scale QTL mapping.<sup><a reveal-id="DSU002C14" data-open="DSU002C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU002C15" data-open="DSU002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> With the rapid developments in genomics and dramatically decreasing cost of genotyping technology, association mapping has rapidly became a promising approach for the genetic dissection of complex traits. For a species with available genome sequences such as <em>Arabidopsis thaliana</em>, rice, maize, etc., genome-wide association studies (GWASs) have contributed to revealing rich genetic architectures of complex traits.<sup><a reveal-id="DSU002C16  DSU002C21" data-open="DSU002C16  DSU002C21" class="link link-ref link-reveal xref-bibr">16–21</a></sup></p><p>Since rapeseed contains two homologous but divergent subgenomes A and C, which were shaped by whole-genome triplication followed by extensive diploidization,<sup><a reveal-id="DSU002C22" data-open="DSU002C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSU002C23" data-open="DSU002C23" class="link link-ref link-reveal xref-bibr">23</a></sup> its genome structure is complex, which has hindered genomic research and high throughput discovery of high-quality molecular markers. The population structure, LD and association mapping in rapeseed were studied using amplified fragment length polymorphism and simple sequence repeat markers.<sup><a reveal-id="DSU002C24  DSU002C29" data-open="DSU002C24  DSU002C29" class="link link-ref link-reveal xref-bibr">24–29</a></sup> However, due to the limited number of DNA markers and low-efficient genotyping technologies, a small number of lines or markers were used in these studies. Recent advances in sequencing and computational technology have enabled the discovery and efficient assay of large numbers of single-nucleotide polymorphism (SNP) markers in rapeseed.<sup><a reveal-id="DSU002C30" data-open="DSU002C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSU002C31" data-open="DSU002C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Delourme <em>et al.</em><sup><a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> analysed the genetic diversity and LD of a rapeseed collection of 313 inbred lines from different geographical origins using &gt;4300 SNPs that were localized on an integrated map of rapeseed. The genomic research of <em>Brassica</em> genera has developed rapidly in recent years. The <em>Brassica</em> A genome sequence from <em>B. rapa</em> has been published,<sup><a reveal-id="DSU002C33" data-open="DSU002C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and <em>Brassica</em> C genome sequencing from <em>B. oleracea</em> has also been completed and will be published soon (<a class="link link-uri" href="http://www.ocri-genomics.org/bolbase/">http://www.ocri-genomics.org/bolbase/</a>). Owing to the complicated genome of <em>B. napus</em>, genome sequencing has not yet been completed. Based on a high-density (∼21 K) SNP linkage map of rapeseed,<sup><a reveal-id="DSU002C31" data-open="DSU002C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Harper <em>et al.</em><sup><a reveal-id="DSU002C34" data-open="DSU002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> refined the order and orientation of the A and C genome sequence scaffolds, constructed pseudomolecules representative of the 19 chromosomes of <em>B. napus</em> and successfully carried out associative transcriptomics of glucosinolate and erucic acid content in a population of 53 <em>B. napus</em> lines.</p><p>In this study, we genotyped a panel of 472 rapeseed accessions from all over the world using a 60K <em>Brassica</em> Infinium<sup>®</sup> SNP array recently developed by the international <em>Brassica</em> Illumina SNP consortium. By <em>in silico</em> mapping of the SNPs of the array to ‘pseudomolecules’ representative of the <em>B. napus</em> genome to obtain their hypothetical position, we performed a GWAS of seed weight and seed quality in <em>B. napus</em>. The SNPs significantly associated with traits were identified, and the candidate genes were confirmed or predicted.</p>                            <h2 scrollto-destination=6655761 id="6655761" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6655762 id="6655762" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>A set of 472 rapeseed inbred lines collected from the National Mid-term Gene Bank for Oil Crops of China were used for association analysis. According to the information from the gene bank and our own observations, the accessions were assigned to five different germplasm types, i.e. winter oilseed rape (OSR) (160), semi-winter OSR (200), spring OSR (110), spring fodder (1) and winter fodder (1). Based on their origins, 266 accessions originated from Asia, 128 from Western Europe, 20 from Oceania, 26 from North America and 32 from Eastern Europe (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). For SNP array quality control, 12 doubled haploid (DH) lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), a resynthesized <em>B. napus</em> SBn8, a cultivar Zhongshuang9 and their F<sub>1</sub> hybrid named F1ZSSB were employed.</p>                            <h3 scrollto-destination=6655764 id="6655764" class="section-title" data-legacy-id=s2b>2.2. Experiment design and traits measurement</h3>
<p>One representative plant of each accession of the association population was self-pollinated in the 2010/2011 winter–spring growing season, and the leaves of each plant were sampled to extract genomic DNA. The self-pollinated seeds of each accession were grown in the experimental farm at Wuhan (114.31°E, 30.52°N), China, in the 2011/2012 growing season. Each accession was grown in a plot with three rows and 10–12 plants in each row, with a distance of 0.2 m between plants within each row and 0.3 m between rows. Before flowering time, each plot was separated by nylon net with a 0.35-mm square hole to propagate seeds. Fatty acid analysis was performed by using gas liquid chromatography (GC) with a Model 6890N GC analyzer (Agilent Technologies, Inc., Wilmington, DE, USA), following the protocol as described.<sup><a reveal-id="DSU002C35" data-open="DSU002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The erucic acid content was expressed as its percentage of total fatty acids in mature seeds.</p><p>The 472 accessions of the association population were grown using a randomized complete block design with three replications in the experimental farm at Wuhan, and Nanchang (116.27°E, 28.37°N), China, in the 2012/2013 growing season. Each accession was grown in a plot with five rows with the same plant density abovementioned. The weight of randomly selected 1000 well-ﬁlled, open-pollinated seeds of each plot was measured to represent seed weight for one replicate of each accession. Seed oil content and total glucosinolate content were estimated by near-infrared reflectance spectroscopy (NIRS). Approximately 3 g seeds per accession were measured using a Foss NIRS Systems 5000 instrument on a reflectance scanning mode.</p><p>As the traits of the association panel were investigated in multi-environments with three replications, an R script (<a class="link link-uri" href="http://www.eXtension.org/pages/61006">www.eXtension.org/pages/61006</a>) based on a linear model described by Merk <em>et al.</em><sup><a reveal-id="DSU002C36" data-open="DSU002C36" class="link link-ref link-reveal xref-bibr">36</a></sup> was used to obtain the best linear unbiased prediction (BLUP) of each trait of each line. The resulting values were used as phenotypes for the association analysis.</p>                            <h3 scrollto-destination=6655768 id="6655768" class="section-title" data-legacy-id=s2c>2.3. SNP genotyping and filtering</h3>
<p>SNP genotyping was performed using the <em>Brassica</em> 60K Illumina<sup>®</sup> Infinium SNP array by Emei Tongde Co. (Beijing) according to the manufacture's protocol (<a class="link link-uri" href="http://www.illumina.com/technology/infinium_hd_assay.ilmn">http://www.illumina.com/technology/infinium_hd_assay.ilmn</a>). The SNP data were clustered and called automatically using the Illumina BeadStudio genotyping software. Those SNPs with AA or BB frequency equal to zero, call frequency &lt;0.8 or minor frequency &lt;0.05 were excluded. The remaining SNPs were scrutinized visually and those SNPs that did not show three clearly defined clusters representing the three possible genotypes (AA, AB and BB) were also excluded.</p>                            <h3 scrollto-destination=6655770 id="6655770" class="section-title" data-legacy-id=s2d>2.4. <em>In silico</em> mapping of SNPs</h3>
<p>The source sequences for designing SNP probes of the SNP array were used to perform a BLAST<sup><a reveal-id="DSU002C37" data-open="DSU002C37" class="link link-ref link-reveal xref-bibr">37</a></sup> search against version 4 of the ‘pseudomolecules’ representative of the genome of <em>B. napus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Only the top BLAST hits against the pseudomolecules were considered. BLAST matches to multiple loci, with the same top identity, were not considered to be mapped.</p>                            <h3 scrollto-destination=6655772 id="6655772" class="section-title" data-legacy-id=s2e>2.5. Genetic diversity, population structure and linkage disequilibrium analysis</h3>
<p>Polymorphism information content (PIC) of the SNPs were estimated using the PowerMarker version 3.51.<sup><a reveal-id="DSU002C38" data-open="DSU002C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The differences of PIC between linkage groups were assessed using one-way analysis of variance implemented in SPSS 9.0. All the SNPs were used to estimate the genetic relatedness between individuals by principal component analysis (PCA) using the GCTA tool.<sup><a reveal-id="DSU002C39" data-open="DSU002C39" class="link link-ref link-reveal xref-bibr">39</a></sup> A total of 3900 SNPs [minor allele frequency (MAF) ≥ 0.2] evenly distributed across the whole genome were selected to perform the following population structure and relative kinship analysis. The software package STRUCTURE v2.3.4<sup><a reveal-id="DSU002C40" data-open="DSU002C40" class="link link-ref link-reveal xref-bibr">40</a></sup> was used to infer population structure. Five independent runs were performed with a <em>K-</em>value (the putative number of genetic groups) varying from 1 to 10, with the length of burnin period and the number of MCMC (Markov Chain Monte Carlo) replications after burnin both to 100 000 under the ‘admixture model’. The most likely <em>k-</em>value was determined by the log probability of data [Ln<em>P</em>(D)] and an <em>ad hoc</em> statistic Δ<em>k</em> based on the rate of change of Ln<em>P</em>(D) between successive <em>k</em> as described by Evanno <em>et al.</em><sup><a reveal-id="DSU002C41" data-open="DSU002C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The cluster membership coefficient matrices of replicate runs from STRUCTURE were integrated to get a Q matrix by the CLUMPP software<sup><a reveal-id="DSU002C42" data-open="DSU002C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and graphically displayed using the DISTRUCT software package.<sup><a reveal-id="DSU002C43" data-open="DSU002C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Accessions with the probability of membership &gt;0.7 were assigned to corresponding clusters, and those &lt;0.7 were assigned to a mixed group. Nei's genetic distance<sup><a reveal-id="DSU002C44" data-open="DSU002C44" class="link link-ref link-reveal xref-bibr">44</a></sup> was estimated and used for constructing an unrooted neighbour-joining tree by the PowerMarker software, and the tree was visualized using FigTree (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>). The relative kinship matrix comparing all pairs of the 472 accessions was calculated using the software package SPAGeDi.<sup><a reveal-id="DSU002C45" data-open="DSU002C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Negative values between two individuals were set to 0, as described by Yu <em>et al.</em><sup><a reveal-id="DSU002C46" data-open="DSU002C46" class="link link-ref link-reveal xref-bibr">46</a></sup> LD was estimated as the squared allele frequency correlations (<em>r</em><sup>2</sup>) between all pairs of the SNPs, using the software TASSEL3.0.<sup><a reveal-id="DSU002C47" data-open="DSU002C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p>                            <h3 scrollto-destination=6655774 id="6655774" class="section-title" data-legacy-id=s2f>2.6 Genome-wide association analysis</h3>
<p>Trait–SNP association analysis was performed using six models to evaluate the effects of population structure (Q, PC) and kinship (K): (i) naive—without controlling for Q and K, (ii) Q model, controlling for Q, (iii) PCA model, controlling for PC, the top 10 principal components were used as fixed effects, (iv) K model, controlling for K, (v) Q + K model, controlling for both Q and K and (vi) P + K model, controlling for both PC and K. The naive, Q and PCA models were performed using a general linear model; the K, Q + K and P + K models were performed using a mixed linear model with optimum compression and population parameters previously determined (P3D) variance component estimation in TASSEL 3.0.<sup><a reveal-id="DSU002C46" data-open="DSU002C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSU002C48" data-open="DSU002C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The distribution of observed −log<sub>10</sub>(<em>p</em>) for each SNP from marker–trait associations was compared with the expected distribution in a quantile–quantile plot. Signiﬁcance of associations between SNPs and traits was based on threshold <em>p</em> &lt; 2 × 10<sup>−6</sup> (i.e. −log<sub>10</sub>(<em>p</em>) = 5.7), a stringent Bonferroni correction calculated by dividing 0.05 by the total number of SNPs in the analysis, 24 256. The signiﬁcant association of more than one SNP was regarded as a true association. Furthermore, false discovery rates (FDRs) were calculated as [(<em>m</em>× <em>P</em>)/<em>n</em>] × 100%,<sup><a reveal-id="DSU002C49" data-open="DSU002C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSU002C50" data-open="DSU002C50" class="link link-ref link-reveal xref-bibr">50</a></sup> where <em>m</em> is the total number of SNPs (i.e. 24 256), <em>P</em> is the <em>p</em>-value threshold for detecting significant association (i.e. 2 × 10<sup>−6</sup>) and <em>n</em> is the total number of signiﬁcant associations per trait.</p>                            <h2 scrollto-destination=6655776 id="6655776" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6655777 id="6655777" class="section-title" data-legacy-id=s3a>3.1. Phenotypic variations of measured quantitative traits</h3>
<p>Glucosinolate content, oil content and seed weight of the <em>B. napus</em> association panel comprising 472 accessions were measured for three replications in two locations. As the key genes responsible for the natural variation of the erucic acid content are known, this trait was investigated for only one replication in one location and used for evaluating the ability and accuracy of the GWAS of this panel. Extensive phenotypic variations were observed as shown in the descriptive statistics in Table <a reveal-id="DSU002TB1" data-open="DSU002TB1" class="link link-reveal link-table xref-fig">1</a>. The erucic acid content, which varied from 0 to 53.7 with an average of 22.7, had the maximum coefficient of variation of 73.4%, whereas oil content in Nanchan, which varied from 33.4 to 52.6 with an average of 43.6, had the lowest coefficient of variation of 5.4%. Seed weight and glucosinolate content were quite consistent across different geographic location replicates with the correlation coefficient of 0.876 and 0.914, respectively, and oil content has the correlation coefficient of ∼0.775 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="6655779" scrollto-destination="6655779"></a>
<div content-id="DSU002TB1" class="table-modal"><div class="table-wrap"><div id="DSU002TB1" class="table-wrap-title" data-id="DSU002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Phenotypic variations for seed weight and seed quality in this <em>B. napus</em> panel</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Traits </th><th>Min ± SD<sup>a</sup> </th><th>Max ± SD </th><th>Mean ± SD </th><th>CV (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Glucosinolate content (µmol/g)—Wuhan 2013 </td><td>28.5 ± 2.0 </td><td>142.8 ± 0.7 </td><td>91.5 ± 0.3 </td><td>29.4 </td></tr><tr><td>Glucosinolate content (µmol/g)—Nanchan 2013 </td><td>30.1 ± 0.9 </td><td>138.0 ± 3.6 </td><td>90.1 ± 0.9 </td><td>27.4 </td></tr><tr><td>Oil content (%)—Wuhan 2013 </td><td>34.2 ± 0.8 </td><td>51.4 ± 0.4 </td><td>42.6 ± 0.3 </td><td>5.9 </td></tr><tr><td>Oil content (%)—Nanchan 2013 </td><td>33.4 ± 1.9 </td><td>52.6 ± 1.5 </td><td>43.6 ± 0.7 </td><td>5.4 </td></tr><tr><td>Erucic acid content (%)—Wuhan 2012 </td><td>0 </td><td>53.7 </td><td>22.7 </td><td>73.4 </td></tr><tr><td>Seed weight (g)—Wuhan 2013 </td><td>2.3 ± 0.1 </td><td>5.9 ± 0.2 </td><td>3.5 ± 0.0 </td><td>14.2 </td></tr><tr><td>Seed weight(g)—Nanchan 2013 </td><td>2.3 ± 0.1 </td><td>5.2 ± 0.2 </td><td>3.6 ± 0.1 </td><td>13.8 </td></tr></tbody></table></div><div content-id="DSU002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Traits </th><th>Min ± SD<sup>a</sup> </th><th>Max ± SD </th><th>Mean ± SD </th><th>CV (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Glucosinolate content (µmol/g)—Wuhan 2013 </td><td>28.5 ± 2.0 </td><td>142.8 ± 0.7 </td><td>91.5 ± 0.3 </td><td>29.4 </td></tr><tr><td>Glucosinolate content (µmol/g)—Nanchan 2013 </td><td>30.1 ± 0.9 </td><td>138.0 ± 3.6 </td><td>90.1 ± 0.9 </td><td>27.4 </td></tr><tr><td>Oil content (%)—Wuhan 2013 </td><td>34.2 ± 0.8 </td><td>51.4 ± 0.4 </td><td>42.6 ± 0.3 </td><td>5.9 </td></tr><tr><td>Oil content (%)—Nanchan 2013 </td><td>33.4 ± 1.9 </td><td>52.6 ± 1.5 </td><td>43.6 ± 0.7 </td><td>5.4 </td></tr><tr><td>Erucic acid content (%)—Wuhan 2012 </td><td>0 </td><td>53.7 </td><td>22.7 </td><td>73.4 </td></tr><tr><td>Seed weight (g)—Wuhan 2013 </td><td>2.3 ± 0.1 </td><td>5.9 ± 0.2 </td><td>3.5 ± 0.0 </td><td>14.2 </td></tr><tr><td>Seed weight(g)—Nanchan 2013 </td><td>2.3 ± 0.1 </td><td>5.2 ± 0.2 </td><td>3.6 ± 0.1 </td><td>13.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>SD is an abbreviation of standard deviation, which was calculated based on the measured values of seeds from three replicated experimental blocks.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>CV is an abbreviation of coefficient of variation, which was estimated as the ratio of the standard deviation to the mean of all accessions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655779" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU002TB1" class="table-wrap-title" data-id="DSU002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Phenotypic variations for seed weight and seed quality in this <em>B. napus</em> panel</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Traits </th><th>Min ± SD<sup>a</sup> </th><th>Max ± SD </th><th>Mean ± SD </th><th>CV (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Glucosinolate content (µmol/g)—Wuhan 2013 </td><td>28.5 ± 2.0 </td><td>142.8 ± 0.7 </td><td>91.5 ± 0.3 </td><td>29.4 </td></tr><tr><td>Glucosinolate content (µmol/g)—Nanchan 2013 </td><td>30.1 ± 0.9 </td><td>138.0 ± 3.6 </td><td>90.1 ± 0.9 </td><td>27.4 </td></tr><tr><td>Oil content (%)—Wuhan 2013 </td><td>34.2 ± 0.8 </td><td>51.4 ± 0.4 </td><td>42.6 ± 0.3 </td><td>5.9 </td></tr><tr><td>Oil content (%)—Nanchan 2013 </td><td>33.4 ± 1.9 </td><td>52.6 ± 1.5 </td><td>43.6 ± 0.7 </td><td>5.4 </td></tr><tr><td>Erucic acid content (%)—Wuhan 2012 </td><td>0 </td><td>53.7 </td><td>22.7 </td><td>73.4 </td></tr><tr><td>Seed weight (g)—Wuhan 2013 </td><td>2.3 ± 0.1 </td><td>5.9 ± 0.2 </td><td>3.5 ± 0.0 </td><td>14.2 </td></tr><tr><td>Seed weight(g)—Nanchan 2013 </td><td>2.3 ± 0.1 </td><td>5.2 ± 0.2 </td><td>3.6 ± 0.1 </td><td>13.8 </td></tr></tbody></table></div><div content-id="DSU002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Traits </th><th>Min ± SD<sup>a</sup> </th><th>Max ± SD </th><th>Mean ± SD </th><th>CV (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Glucosinolate content (µmol/g)—Wuhan 2013 </td><td>28.5 ± 2.0 </td><td>142.8 ± 0.7 </td><td>91.5 ± 0.3 </td><td>29.4 </td></tr><tr><td>Glucosinolate content (µmol/g)—Nanchan 2013 </td><td>30.1 ± 0.9 </td><td>138.0 ± 3.6 </td><td>90.1 ± 0.9 </td><td>27.4 </td></tr><tr><td>Oil content (%)—Wuhan 2013 </td><td>34.2 ± 0.8 </td><td>51.4 ± 0.4 </td><td>42.6 ± 0.3 </td><td>5.9 </td></tr><tr><td>Oil content (%)—Nanchan 2013 </td><td>33.4 ± 1.9 </td><td>52.6 ± 1.5 </td><td>43.6 ± 0.7 </td><td>5.4 </td></tr><tr><td>Erucic acid content (%)—Wuhan 2012 </td><td>0 </td><td>53.7 </td><td>22.7 </td><td>73.4 </td></tr><tr><td>Seed weight (g)—Wuhan 2013 </td><td>2.3 ± 0.1 </td><td>5.9 ± 0.2 </td><td>3.5 ± 0.0 </td><td>14.2 </td></tr><tr><td>Seed weight(g)—Nanchan 2013 </td><td>2.3 ± 0.1 </td><td>5.2 ± 0.2 </td><td>3.6 ± 0.1 </td><td>13.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>SD is an abbreviation of standard deviation, which was calculated based on the measured values of seeds from three replicated experimental blocks.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>CV is an abbreviation of coefficient of variation, which was estimated as the ratio of the standard deviation to the mean of all accessions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655779" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6655780 id="6655780" class="section-title" data-legacy-id=s3b>3.2. SNP performance and quality</h3>
<p>Calling SNP genotype data using the BeadStudio genotyping software generally produced three clear clusters, i.e. the AA homozygote, BB homozygote and AB heterozygote. Of the 52 157 SNPs in the array, 10 389 which had zero call frequency of AA or BB were excluded. Then, with a cut-off of missing data &gt;0.2 and MAF &lt;0.05, 1866 and 1428 SNPs were filtered, respectively, reducing the number of SNPs to 38 474. Those that remained were scrutinized visually and 29 027 SNPs with three clearly defined clusters were selected. Theoretically, the DH lines are homozygous throughout their genome, while the 12 DH lines derived from geographically different inbred lines of <em>B. napus</em> showed a heterozygosity rate of 1.8∼2.8%, which might be caused by miss-calling or homologous sequences and should be excluded. As a result, 2186 SNPs with the heterozygous genotype in any DH lines were filtered and 26 841 SNPs were finally selected.</p><p>To further control SNP quality, a parent/F<sub>1</sub> triplet and four DNA duplicates were used to analyse the pedigree consistency and technical reproducibility, respectively. As a result, the allele calls were highly reproducible (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>) with no or negligible inconsistencies between technical replicates. Furthermore, the parent/F<sub>1</sub> triplet showed a pedigree inconsistency of &lt;1%, confirming the high quality of the genotype calling of the selected SNPs in the array.</p>                            <h3 scrollto-destination=6655783 id="6655783" class="section-title" data-legacy-id=s3c>3.3. SNP <em>in silico</em> mapping and diversity</h3>
<p>By BLAST analysis of the resource sequences of the SNPs in the array against ‘pseudomolecules’ representative of the genome of <em>B. napus</em>, 47 805 of 52 157 SNPs were <em>in silico</em> mapped (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Among the selected 26 841 SNPs with fine performance, 24 256 were mapped (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5 and Fig. S2</a>). Linkage group A2 had the least markers of 506 with a marker density of one per 52.9 kb, and C4 had the most markers of 2104 with a marker density of one per 25.0 kb. The SNP diversity in the whole collection was expressed by a PIC value, and the results were shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a> and summarized in Table <a reveal-id="DSU002TB2" data-open="DSU002TB2" class="link link-reveal link-table xref-fig">2</a>. Except for linkage groups C5, C7 and C8, 75% of SNPs in each linkage group had PIC values over 0.25. The levels of polymorphism of linkage groups in the A subgenome (with the exception of A2, A4 and A5) were generally higher than those in the C subgenome. The mean PIC values of A8 and A9 were highest, nearly 0.32, whereas that of C7 was lowest, only 0.254. </p>                        <a id="6655785" scrollto-destination="6655785"></a>
<div content-id="DSU002TB2" class="table-modal"><div class="table-wrap"><div id="DSU002TB2" class="table-wrap-title" data-id="DSU002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of the PIC values in different linkage groups of the <em>B. napus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkgae group </th><th rowspan="2">Number of SNPs </th><th colspan="8">PIC<sup>a</sup> value<hr /> </th></tr><tr><th>0.05–0.1 </th><th>0.1–0.15 </th><th>0.15–0.2 </th><th>0.2–0.25 </th><th>0.25–0.3 </th><th>0.3–0.35 </th><th>0.35–0.4 </th><th>Average PIC<sup>b</sup> </th></tr></thead><tbody><tr><td>A1 </td><td>1072 </td><td>9 (0.8%) </td><td>34 (3.2%) </td><td>83 (7.7%) </td><td>117 (10.9%) </td><td>102 (9.5%) </td><td>248 (23.1%) </td><td>479 (44.7%) </td><td>0.31 g </td></tr><tr><td>A2 </td><td>506 </td><td>2 (0.4%) </td><td>34 (6.7%) </td><td>25 (4.9%) </td><td>46 (9.1%) </td><td>115 (22.7%) </td><td>130 (25.7%) </td><td>154 (30.4%) </td><td>0.294 cd </td></tr><tr><td>A3 </td><td>1469 </td><td>6 (0.4%) </td><td>45 (3.1%) </td><td>93 (6.3%) </td><td>130 (8.8%) </td><td>210 (14.3%) </td><td>383 (26.1%) </td><td>602 (41%) </td><td>0.311 g </td></tr><tr><td>A4 </td><td>1035 </td><td>6 (0.6%) </td><td>61 (5.9%) </td><td>56 (5.4%) </td><td>131 (12.7%) </td><td>209 (20.2%) </td><td>272 (26.3%) </td><td>300 (29%) </td><td>0.293 bcd </td></tr><tr><td>A5 </td><td>1123 </td><td>1 (0.1%) </td><td>35 (3.1%) </td><td>69 (6.1%) </td><td>115 (10.2%) </td><td>179 (15.9%) </td><td>376 (33.5%) </td><td>348 (31%) </td><td>0.307 fg </td></tr><tr><td>A6 </td><td>1099 </td><td>3 (0.3%) </td><td>36 (3.3%) </td><td>58 (5.3%) </td><td>91 (8.3%) </td><td>168 (15.3%) </td><td>317 (28.8%) </td><td>426 (38.8%) </td><td>0.312 g </td></tr><tr><td>A7 </td><td>1427 </td><td>11 (0.8%) </td><td>50 (3.5%) </td><td>70 (4.9%) </td><td>126 (8.8%) </td><td>217 (15.2%) </td><td>382 (26.8%) </td><td>571 (40%) </td><td>0.31 g </td></tr><tr><td>A8 </td><td>691 </td><td>7 (1%) </td><td>35 (5.1%) </td><td>38 (5.5%) </td><td>40 (5.8%) </td><td>53 (7.7%) </td><td>143 (20.7%) </td><td>375 (54.3%) </td><td>0.319 h </td></tr><tr><td>A9 </td><td>1225 </td><td>9 (0.7%) </td><td>45 (3.7%) </td><td>39 (3.2%) </td><td>68 (5.6%) </td><td>190 (15.5%) </td><td>261 (21.3%) </td><td>613 (50%) </td><td>0.32 h </td></tr><tr><td>A10 </td><td>805 </td><td>8 (1%) </td><td>33 (4.1%) </td><td>38 (4.7%) </td><td>72 (8.9%) </td><td>104 (12.9%) </td><td>193 (24%) </td><td>357 (44.3%) </td><td>0.312 g </td></tr><tr><td>C1 </td><td>2012 </td><td>1 (0%) </td><td>73 (3.6%) </td><td>124 (6.2%) </td><td>238 (11.8%) </td><td>298 (14.8%) </td><td>1074 (53.4%) </td><td>204 (10.1%) </td><td>0.298 cde </td></tr><tr><td>C2 </td><td>1292 </td><td>2 (0.2%) </td><td>23 (1.8%) </td><td>54 (4.2%) </td><td>116 (9%) </td><td>397 (30.7%) </td><td>321 (24.8%) </td><td>379 (29.3%) </td><td>0.3 de </td></tr><tr><td>C3 </td><td>2201 </td><td>29 (1.3%) </td><td>77 (3.5%) </td><td>159 (7.2%) </td><td>307 (13.9%) </td><td>352 (16%) </td><td>516 (23.4%) </td><td>761 (34.6%) </td><td>0.298 cde </td></tr><tr><td>C4 </td><td>2104 </td><td>32 (1.5%) </td><td>94 (4.5%) </td><td>163 (7.7%) </td><td>181 (8.6%) </td><td>351 (16.7%) </td><td>560 (26.6%) </td><td>723 (34.4%) </td><td>0.301 ef </td></tr><tr><td>C5 </td><td>719 </td><td>2 (0.3%) </td><td>32 (4.5%) </td><td>59 (8.2%) </td><td>105 (14.6%) </td><td>110 (15.3%) </td><td>121 (16.8%) </td><td>290 (40.3%) </td><td>0.298 cde </td></tr><tr><td>C6 </td><td>1539 </td><td>28 (1.8%) </td><td>52 (3.4%) </td><td>101 (6.6%) </td><td>136 (8.8%) </td><td>403 (26.2%) </td><td>417 (27.1%) </td><td>402 (26.1%) </td><td>0.296 cde </td></tr><tr><td>C7 </td><td>2006 </td><td>44 (2.2%) </td><td>591 (29.5%) </td><td>52 (2.6%) </td><td>185 (9.2%) </td><td>156 (7.8%) </td><td>587 (29.3%) </td><td>391 (19.5%) </td><td>0.254 a </td></tr><tr><td>C8 </td><td>1096 </td><td>5 (0.5%) </td><td>23 (2.1%) </td><td>40 (3.6%) </td><td>362 (33%) </td><td>90 (8.2%) </td><td>229 (20.9%) </td><td>347 (31.7%) </td><td>0.292 bc </td></tr><tr><td>C9 </td><td>835 </td><td>14 (1.7%) </td><td>18 (2.2%) </td><td>78 (9.3%) </td><td>97 (11.6%) </td><td>227 (27.2%) </td><td>192 (23%) </td><td>209 (25%) </td><td>0.287 b </td></tr></tbody></table></div><div content-id="DSU002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkgae group </th><th rowspan="2">Number of SNPs </th><th colspan="8">PIC<sup>a</sup> value<hr /> </th></tr><tr><th>0.05–0.1 </th><th>0.1–0.15 </th><th>0.15–0.2 </th><th>0.2–0.25 </th><th>0.25–0.3 </th><th>0.3–0.35 </th><th>0.35–0.4 </th><th>Average PIC<sup>b</sup> </th></tr></thead><tbody><tr><td>A1 </td><td>1072 </td><td>9 (0.8%) </td><td>34 (3.2%) </td><td>83 (7.7%) </td><td>117 (10.9%) </td><td>102 (9.5%) </td><td>248 (23.1%) </td><td>479 (44.7%) </td><td>0.31 g </td></tr><tr><td>A2 </td><td>506 </td><td>2 (0.4%) </td><td>34 (6.7%) </td><td>25 (4.9%) </td><td>46 (9.1%) </td><td>115 (22.7%) </td><td>130 (25.7%) </td><td>154 (30.4%) </td><td>0.294 cd </td></tr><tr><td>A3 </td><td>1469 </td><td>6 (0.4%) </td><td>45 (3.1%) </td><td>93 (6.3%) </td><td>130 (8.8%) </td><td>210 (14.3%) </td><td>383 (26.1%) </td><td>602 (41%) </td><td>0.311 g </td></tr><tr><td>A4 </td><td>1035 </td><td>6 (0.6%) </td><td>61 (5.9%) </td><td>56 (5.4%) </td><td>131 (12.7%) </td><td>209 (20.2%) </td><td>272 (26.3%) </td><td>300 (29%) </td><td>0.293 bcd </td></tr><tr><td>A5 </td><td>1123 </td><td>1 (0.1%) </td><td>35 (3.1%) </td><td>69 (6.1%) </td><td>115 (10.2%) </td><td>179 (15.9%) </td><td>376 (33.5%) </td><td>348 (31%) </td><td>0.307 fg </td></tr><tr><td>A6 </td><td>1099 </td><td>3 (0.3%) </td><td>36 (3.3%) </td><td>58 (5.3%) </td><td>91 (8.3%) </td><td>168 (15.3%) </td><td>317 (28.8%) </td><td>426 (38.8%) </td><td>0.312 g </td></tr><tr><td>A7 </td><td>1427 </td><td>11 (0.8%) </td><td>50 (3.5%) </td><td>70 (4.9%) </td><td>126 (8.8%) </td><td>217 (15.2%) </td><td>382 (26.8%) </td><td>571 (40%) </td><td>0.31 g </td></tr><tr><td>A8 </td><td>691 </td><td>7 (1%) </td><td>35 (5.1%) </td><td>38 (5.5%) </td><td>40 (5.8%) </td><td>53 (7.7%) </td><td>143 (20.7%) </td><td>375 (54.3%) </td><td>0.319 h </td></tr><tr><td>A9 </td><td>1225 </td><td>9 (0.7%) </td><td>45 (3.7%) </td><td>39 (3.2%) </td><td>68 (5.6%) </td><td>190 (15.5%) </td><td>261 (21.3%) </td><td>613 (50%) </td><td>0.32 h </td></tr><tr><td>A10 </td><td>805 </td><td>8 (1%) </td><td>33 (4.1%) </td><td>38 (4.7%) </td><td>72 (8.9%) </td><td>104 (12.9%) </td><td>193 (24%) </td><td>357 (44.3%) </td><td>0.312 g </td></tr><tr><td>C1 </td><td>2012 </td><td>1 (0%) </td><td>73 (3.6%) </td><td>124 (6.2%) </td><td>238 (11.8%) </td><td>298 (14.8%) </td><td>1074 (53.4%) </td><td>204 (10.1%) </td><td>0.298 cde </td></tr><tr><td>C2 </td><td>1292 </td><td>2 (0.2%) </td><td>23 (1.8%) </td><td>54 (4.2%) </td><td>116 (9%) </td><td>397 (30.7%) </td><td>321 (24.8%) </td><td>379 (29.3%) </td><td>0.3 de </td></tr><tr><td>C3 </td><td>2201 </td><td>29 (1.3%) </td><td>77 (3.5%) </td><td>159 (7.2%) </td><td>307 (13.9%) </td><td>352 (16%) </td><td>516 (23.4%) </td><td>761 (34.6%) </td><td>0.298 cde </td></tr><tr><td>C4 </td><td>2104 </td><td>32 (1.5%) </td><td>94 (4.5%) </td><td>163 (7.7%) </td><td>181 (8.6%) </td><td>351 (16.7%) </td><td>560 (26.6%) </td><td>723 (34.4%) </td><td>0.301 ef </td></tr><tr><td>C5 </td><td>719 </td><td>2 (0.3%) </td><td>32 (4.5%) </td><td>59 (8.2%) </td><td>105 (14.6%) </td><td>110 (15.3%) </td><td>121 (16.8%) </td><td>290 (40.3%) </td><td>0.298 cde </td></tr><tr><td>C6 </td><td>1539 </td><td>28 (1.8%) </td><td>52 (3.4%) </td><td>101 (6.6%) </td><td>136 (8.8%) </td><td>403 (26.2%) </td><td>417 (27.1%) </td><td>402 (26.1%) </td><td>0.296 cde </td></tr><tr><td>C7 </td><td>2006 </td><td>44 (2.2%) </td><td>591 (29.5%) </td><td>52 (2.6%) </td><td>185 (9.2%) </td><td>156 (7.8%) </td><td>587 (29.3%) </td><td>391 (19.5%) </td><td>0.254 a </td></tr><tr><td>C8 </td><td>1096 </td><td>5 (0.5%) </td><td>23 (2.1%) </td><td>40 (3.6%) </td><td>362 (33%) </td><td>90 (8.2%) </td><td>229 (20.9%) </td><td>347 (31.7%) </td><td>0.292 bc </td></tr><tr><td>C9 </td><td>835 </td><td>14 (1.7%) </td><td>18 (2.2%) </td><td>78 (9.3%) </td><td>97 (11.6%) </td><td>227 (27.2%) </td><td>192 (23%) </td><td>209 (25%) </td><td>0.287 b </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p><sup>a</sup>PIC is an abbreviation of polymorphism information content.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>b</sup>Values followed by different letters in this column are significantly different at the level of 0.05.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655785" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU002TB2" class="table-wrap-title" data-id="DSU002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of the PIC values in different linkage groups of the <em>B. napus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkgae group </th><th rowspan="2">Number of SNPs </th><th colspan="8">PIC<sup>a</sup> value<hr /> </th></tr><tr><th>0.05–0.1 </th><th>0.1–0.15 </th><th>0.15–0.2 </th><th>0.2–0.25 </th><th>0.25–0.3 </th><th>0.3–0.35 </th><th>0.35–0.4 </th><th>Average PIC<sup>b</sup> </th></tr></thead><tbody><tr><td>A1 </td><td>1072 </td><td>9 (0.8%) </td><td>34 (3.2%) </td><td>83 (7.7%) </td><td>117 (10.9%) </td><td>102 (9.5%) </td><td>248 (23.1%) </td><td>479 (44.7%) </td><td>0.31 g </td></tr><tr><td>A2 </td><td>506 </td><td>2 (0.4%) </td><td>34 (6.7%) </td><td>25 (4.9%) </td><td>46 (9.1%) </td><td>115 (22.7%) </td><td>130 (25.7%) </td><td>154 (30.4%) </td><td>0.294 cd </td></tr><tr><td>A3 </td><td>1469 </td><td>6 (0.4%) </td><td>45 (3.1%) </td><td>93 (6.3%) </td><td>130 (8.8%) </td><td>210 (14.3%) </td><td>383 (26.1%) </td><td>602 (41%) </td><td>0.311 g </td></tr><tr><td>A4 </td><td>1035 </td><td>6 (0.6%) </td><td>61 (5.9%) </td><td>56 (5.4%) </td><td>131 (12.7%) </td><td>209 (20.2%) </td><td>272 (26.3%) </td><td>300 (29%) </td><td>0.293 bcd </td></tr><tr><td>A5 </td><td>1123 </td><td>1 (0.1%) </td><td>35 (3.1%) </td><td>69 (6.1%) </td><td>115 (10.2%) </td><td>179 (15.9%) </td><td>376 (33.5%) </td><td>348 (31%) </td><td>0.307 fg </td></tr><tr><td>A6 </td><td>1099 </td><td>3 (0.3%) </td><td>36 (3.3%) </td><td>58 (5.3%) </td><td>91 (8.3%) </td><td>168 (15.3%) </td><td>317 (28.8%) </td><td>426 (38.8%) </td><td>0.312 g </td></tr><tr><td>A7 </td><td>1427 </td><td>11 (0.8%) </td><td>50 (3.5%) </td><td>70 (4.9%) </td><td>126 (8.8%) </td><td>217 (15.2%) </td><td>382 (26.8%) </td><td>571 (40%) </td><td>0.31 g </td></tr><tr><td>A8 </td><td>691 </td><td>7 (1%) </td><td>35 (5.1%) </td><td>38 (5.5%) </td><td>40 (5.8%) </td><td>53 (7.7%) </td><td>143 (20.7%) </td><td>375 (54.3%) </td><td>0.319 h </td></tr><tr><td>A9 </td><td>1225 </td><td>9 (0.7%) </td><td>45 (3.7%) </td><td>39 (3.2%) </td><td>68 (5.6%) </td><td>190 (15.5%) </td><td>261 (21.3%) </td><td>613 (50%) </td><td>0.32 h </td></tr><tr><td>A10 </td><td>805 </td><td>8 (1%) </td><td>33 (4.1%) </td><td>38 (4.7%) </td><td>72 (8.9%) </td><td>104 (12.9%) </td><td>193 (24%) </td><td>357 (44.3%) </td><td>0.312 g </td></tr><tr><td>C1 </td><td>2012 </td><td>1 (0%) </td><td>73 (3.6%) </td><td>124 (6.2%) </td><td>238 (11.8%) </td><td>298 (14.8%) </td><td>1074 (53.4%) </td><td>204 (10.1%) </td><td>0.298 cde </td></tr><tr><td>C2 </td><td>1292 </td><td>2 (0.2%) </td><td>23 (1.8%) </td><td>54 (4.2%) </td><td>116 (9%) </td><td>397 (30.7%) </td><td>321 (24.8%) </td><td>379 (29.3%) </td><td>0.3 de </td></tr><tr><td>C3 </td><td>2201 </td><td>29 (1.3%) </td><td>77 (3.5%) </td><td>159 (7.2%) </td><td>307 (13.9%) </td><td>352 (16%) </td><td>516 (23.4%) </td><td>761 (34.6%) </td><td>0.298 cde </td></tr><tr><td>C4 </td><td>2104 </td><td>32 (1.5%) </td><td>94 (4.5%) </td><td>163 (7.7%) </td><td>181 (8.6%) </td><td>351 (16.7%) </td><td>560 (26.6%) </td><td>723 (34.4%) </td><td>0.301 ef </td></tr><tr><td>C5 </td><td>719 </td><td>2 (0.3%) </td><td>32 (4.5%) </td><td>59 (8.2%) </td><td>105 (14.6%) </td><td>110 (15.3%) </td><td>121 (16.8%) </td><td>290 (40.3%) </td><td>0.298 cde </td></tr><tr><td>C6 </td><td>1539 </td><td>28 (1.8%) </td><td>52 (3.4%) </td><td>101 (6.6%) </td><td>136 (8.8%) </td><td>403 (26.2%) </td><td>417 (27.1%) </td><td>402 (26.1%) </td><td>0.296 cde </td></tr><tr><td>C7 </td><td>2006 </td><td>44 (2.2%) </td><td>591 (29.5%) </td><td>52 (2.6%) </td><td>185 (9.2%) </td><td>156 (7.8%) </td><td>587 (29.3%) </td><td>391 (19.5%) </td><td>0.254 a </td></tr><tr><td>C8 </td><td>1096 </td><td>5 (0.5%) </td><td>23 (2.1%) </td><td>40 (3.6%) </td><td>362 (33%) </td><td>90 (8.2%) </td><td>229 (20.9%) </td><td>347 (31.7%) </td><td>0.292 bc </td></tr><tr><td>C9 </td><td>835 </td><td>14 (1.7%) </td><td>18 (2.2%) </td><td>78 (9.3%) </td><td>97 (11.6%) </td><td>227 (27.2%) </td><td>192 (23%) </td><td>209 (25%) </td><td>0.287 b </td></tr></tbody></table></div><div content-id="DSU002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkgae group </th><th rowspan="2">Number of SNPs </th><th colspan="8">PIC<sup>a</sup> value<hr /> </th></tr><tr><th>0.05–0.1 </th><th>0.1–0.15 </th><th>0.15–0.2 </th><th>0.2–0.25 </th><th>0.25–0.3 </th><th>0.3–0.35 </th><th>0.35–0.4 </th><th>Average PIC<sup>b</sup> </th></tr></thead><tbody><tr><td>A1 </td><td>1072 </td><td>9 (0.8%) </td><td>34 (3.2%) </td><td>83 (7.7%) </td><td>117 (10.9%) </td><td>102 (9.5%) </td><td>248 (23.1%) </td><td>479 (44.7%) </td><td>0.31 g </td></tr><tr><td>A2 </td><td>506 </td><td>2 (0.4%) </td><td>34 (6.7%) </td><td>25 (4.9%) </td><td>46 (9.1%) </td><td>115 (22.7%) </td><td>130 (25.7%) </td><td>154 (30.4%) </td><td>0.294 cd </td></tr><tr><td>A3 </td><td>1469 </td><td>6 (0.4%) </td><td>45 (3.1%) </td><td>93 (6.3%) </td><td>130 (8.8%) </td><td>210 (14.3%) </td><td>383 (26.1%) </td><td>602 (41%) </td><td>0.311 g </td></tr><tr><td>A4 </td><td>1035 </td><td>6 (0.6%) </td><td>61 (5.9%) </td><td>56 (5.4%) </td><td>131 (12.7%) </td><td>209 (20.2%) </td><td>272 (26.3%) </td><td>300 (29%) </td><td>0.293 bcd </td></tr><tr><td>A5 </td><td>1123 </td><td>1 (0.1%) </td><td>35 (3.1%) </td><td>69 (6.1%) </td><td>115 (10.2%) </td><td>179 (15.9%) </td><td>376 (33.5%) </td><td>348 (31%) </td><td>0.307 fg </td></tr><tr><td>A6 </td><td>1099 </td><td>3 (0.3%) </td><td>36 (3.3%) </td><td>58 (5.3%) </td><td>91 (8.3%) </td><td>168 (15.3%) </td><td>317 (28.8%) </td><td>426 (38.8%) </td><td>0.312 g </td></tr><tr><td>A7 </td><td>1427 </td><td>11 (0.8%) </td><td>50 (3.5%) </td><td>70 (4.9%) </td><td>126 (8.8%) </td><td>217 (15.2%) </td><td>382 (26.8%) </td><td>571 (40%) </td><td>0.31 g </td></tr><tr><td>A8 </td><td>691 </td><td>7 (1%) </td><td>35 (5.1%) </td><td>38 (5.5%) </td><td>40 (5.8%) </td><td>53 (7.7%) </td><td>143 (20.7%) </td><td>375 (54.3%) </td><td>0.319 h </td></tr><tr><td>A9 </td><td>1225 </td><td>9 (0.7%) </td><td>45 (3.7%) </td><td>39 (3.2%) </td><td>68 (5.6%) </td><td>190 (15.5%) </td><td>261 (21.3%) </td><td>613 (50%) </td><td>0.32 h </td></tr><tr><td>A10 </td><td>805 </td><td>8 (1%) </td><td>33 (4.1%) </td><td>38 (4.7%) </td><td>72 (8.9%) </td><td>104 (12.9%) </td><td>193 (24%) </td><td>357 (44.3%) </td><td>0.312 g </td></tr><tr><td>C1 </td><td>2012 </td><td>1 (0%) </td><td>73 (3.6%) </td><td>124 (6.2%) </td><td>238 (11.8%) </td><td>298 (14.8%) </td><td>1074 (53.4%) </td><td>204 (10.1%) </td><td>0.298 cde </td></tr><tr><td>C2 </td><td>1292 </td><td>2 (0.2%) </td><td>23 (1.8%) </td><td>54 (4.2%) </td><td>116 (9%) </td><td>397 (30.7%) </td><td>321 (24.8%) </td><td>379 (29.3%) </td><td>0.3 de </td></tr><tr><td>C3 </td><td>2201 </td><td>29 (1.3%) </td><td>77 (3.5%) </td><td>159 (7.2%) </td><td>307 (13.9%) </td><td>352 (16%) </td><td>516 (23.4%) </td><td>761 (34.6%) </td><td>0.298 cde </td></tr><tr><td>C4 </td><td>2104 </td><td>32 (1.5%) </td><td>94 (4.5%) </td><td>163 (7.7%) </td><td>181 (8.6%) </td><td>351 (16.7%) </td><td>560 (26.6%) </td><td>723 (34.4%) </td><td>0.301 ef </td></tr><tr><td>C5 </td><td>719 </td><td>2 (0.3%) </td><td>32 (4.5%) </td><td>59 (8.2%) </td><td>105 (14.6%) </td><td>110 (15.3%) </td><td>121 (16.8%) </td><td>290 (40.3%) </td><td>0.298 cde </td></tr><tr><td>C6 </td><td>1539 </td><td>28 (1.8%) </td><td>52 (3.4%) </td><td>101 (6.6%) </td><td>136 (8.8%) </td><td>403 (26.2%) </td><td>417 (27.1%) </td><td>402 (26.1%) </td><td>0.296 cde </td></tr><tr><td>C7 </td><td>2006 </td><td>44 (2.2%) </td><td>591 (29.5%) </td><td>52 (2.6%) </td><td>185 (9.2%) </td><td>156 (7.8%) </td><td>587 (29.3%) </td><td>391 (19.5%) </td><td>0.254 a </td></tr><tr><td>C8 </td><td>1096 </td><td>5 (0.5%) </td><td>23 (2.1%) </td><td>40 (3.6%) </td><td>362 (33%) </td><td>90 (8.2%) </td><td>229 (20.9%) </td><td>347 (31.7%) </td><td>0.292 bc </td></tr><tr><td>C9 </td><td>835 </td><td>14 (1.7%) </td><td>18 (2.2%) </td><td>78 (9.3%) </td><td>97 (11.6%) </td><td>227 (27.2%) </td><td>192 (23%) </td><td>209 (25%) </td><td>0.287 b </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p><sup>a</sup>PIC is an abbreviation of polymorphism information content.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>b</sup>Values followed by different letters in this column are significantly different at the level of 0.05.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655785" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6655786 id="6655786" class="section-title" data-legacy-id=s3d>3.4. Population structure, relative kinship and linkage disequilibrium</h3>
<p>The population structure of the association panel was calculated using 3900 SNPs by STRUCTURE, and clustering inference performed with possible clusters (<em>k</em>) from 1 to 10 showed that the most signiﬁcant change of likelihood occurred when <em>K</em> increased from 3 to 4, and the highest Δ<em>k</em> value was observed at <em>k</em> = 3 (Fig. <a reveal-id="DSU002F1" data-open="DSU002F1" class="link link-reveal link-table xref-fig">1</a>). Both parameters suggested that the 472 genotypes could be assigned into three groups. Using a probability of membership threshold of 70%, 164, 29 and 17 lines were assigned into the three groups, respectively. The remaining 262 lines were classiﬁed into a mixed group (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Most of the lines in Group 1 were from Asia and belong to semi-winter OSR (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). European lines were the main component of the Groups 2 and 3, in which the larger part were winter OSR and spring OSR, respectively. The NJ phylogenetic tree based on Nei's genetic distances displayed three clear clades (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), corresponding to the three groups estimated by STRUCTURE. The lines belong to mixed groups distributed across the whole tree. The PCA based on the 24 256 genome-wide SNPs showed that the ﬁrst two principal components explained 11.7 and 5.8% of the genetic variance, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The three major groups identified using STRUCTURE (i.e. except for the mixed group), winter OSR, spring OSR and semi-winter OSR were assigned to three major clusters. The analysis of relative kinship showed that the average relative kinship between any two lines was 0.0856. A total of 51.4% of kinship coefﬁcients between lines were equal to 0, and 32.6% kinship coefﬁcients ranged from 0 to 0.2 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). This pattern of genetic relatedness revealed that most lines have no or weak kinship in this OSR panel. To assess the extent of LD, we calculated the mean pairwise <em>r</em><sup>2</sup> for all SNPs as 0.0176. </p>                        <a id="6655788" scrollto-destination="6655788"></a>
<div id="DSU002F1" data-id="DSU002F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00201.png?Expires=2147483647&Signature=ikNMireHZhAsbhSuaeOOhgjZG0wjSTAWDAYVoMp7Zb23ygal0mL~nvNtA2K6S6OLokp9yL-XTIfYuF0fprHfdOZQy9bDQ~KGJb2e2QcrNjiNL6N8BbRsifig3dvXV9yF~Z20QlKH59S9LOaBn-UaIx5gjOWTQryMdsaYg2pIjzaQA9KmLjGSrIncP~kbDrtvN6h98ZqRWPjApUZffFdTyOi0tA2dJwQwitO8EQ-ImfvE1b8Vqf1bWf3-~ugWVe-Mjj7jLQwhmelrUBPQ-TaYFWHWl67AKurKNy1fBS6n6t5vzOlx0Bn5z4rQAcwkfiCoBuHrntndDn3wOjguhiMm8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the population structure of 472 rapeseed accessions by STRUCTURE. (a) Estimated LnP(D) of possible clusters (k) from 1 to 10; (b) Δk based on the rate of change of LnP(D) between successive k; (c) population structure based on k = 3. Each individual is represented by a vertical bar, partitioned into coloured segments with the length of each segment representing the proportion of the individual's genome. A given group is represented: Red, Group 1; Green, Group 2; Purple, Group 3." path-from-xml="dsu00201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655788/dsu00201.tiff" path-from-xml="dsu00201.tiff" target="_blank">View large</a><a section="6655788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00201.png?Expires=2147483647&Signature=2TFpB6gN2VHc4ydemUOh64KS7~C5hENoEKLajP9qSg-EOb~o19jCsy4UbDjFejapq9OQ-zTG~1hnC3yIH06KzNtO6Be6vNFnSwoXw6o7UtDVE7hit5T~Uid4bLGwWl8c9TFc8MLJc6Z-SCv0XBBh5TKQli1txjfe-hapmwq7NEikLWtbpHdIUrj-KDLkJCNFcEnvk4nQvqXY8SG0IkAx735gBmm~QM61VJyb~FIpS9Fl4b5fLb88rA~D2NWRYLhlIVKks7q2Strbl6Qj3RxcO8KJ9KnP1TFqcAOre1q-w6vy4~pumEAgUnrFXrv5i7nxdnhvSSLOFCL7S~AtGugQmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655788&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the population structure of 472 rapeseed accessions by STRUCTURE. (a) Estimated Ln<em>P</em>(D) of possible clusters (<em>k</em>) from 1 to 10; (b) Δ<em>k</em> based on the rate of change of Ln<em>P</em>(D) between successive <em>k</em>; (c) population structure based on <em>k</em> = 3. Each individual is represented by a vertical bar, partitioned into coloured segments with the length of each segment representing the proportion of the individual's genome. A given group is represented: Red, Group 1; Green, Group 2; Purple, Group 3.</p></div></div><div content-id="DSU002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00201.png?Expires=2147483647&Signature=ikNMireHZhAsbhSuaeOOhgjZG0wjSTAWDAYVoMp7Zb23ygal0mL~nvNtA2K6S6OLokp9yL-XTIfYuF0fprHfdOZQy9bDQ~KGJb2e2QcrNjiNL6N8BbRsifig3dvXV9yF~Z20QlKH59S9LOaBn-UaIx5gjOWTQryMdsaYg2pIjzaQA9KmLjGSrIncP~kbDrtvN6h98ZqRWPjApUZffFdTyOi0tA2dJwQwitO8EQ-ImfvE1b8Vqf1bWf3-~ugWVe-Mjj7jLQwhmelrUBPQ-TaYFWHWl67AKurKNy1fBS6n6t5vzOlx0Bn5z4rQAcwkfiCoBuHrntndDn3wOjguhiMm8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the population structure of 472 rapeseed accessions by STRUCTURE. (a) Estimated LnP(D) of possible clusters (k) from 1 to 10; (b) Δk based on the rate of change of LnP(D) between successive k; (c) population structure based on k = 3. Each individual is represented by a vertical bar, partitioned into coloured segments with the length of each segment representing the proportion of the individual's genome. A given group is represented: Red, Group 1; Green, Group 2; Purple, Group 3." path-from-xml="dsu00201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655788/dsu00201.tiff" path-from-xml="dsu00201.tiff" target="_blank">View large</a><a section="6655788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00201.png?Expires=2147483647&Signature=2TFpB6gN2VHc4ydemUOh64KS7~C5hENoEKLajP9qSg-EOb~o19jCsy4UbDjFejapq9OQ-zTG~1hnC3yIH06KzNtO6Be6vNFnSwoXw6o7UtDVE7hit5T~Uid4bLGwWl8c9TFc8MLJc6Z-SCv0XBBh5TKQli1txjfe-hapmwq7NEikLWtbpHdIUrj-KDLkJCNFcEnvk4nQvqXY8SG0IkAx735gBmm~QM61VJyb~FIpS9Fl4b5fLb88rA~D2NWRYLhlIVKks7q2Strbl6Qj3RxcO8KJ9KnP1TFqcAOre1q-w6vy4~pumEAgUnrFXrv5i7nxdnhvSSLOFCL7S~AtGugQmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655788&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the population structure of 472 rapeseed accessions by STRUCTURE. (a) Estimated Ln<em>P</em>(D) of possible clusters (<em>k</em>) from 1 to 10; (b) Δ<em>k</em> based on the rate of change of Ln<em>P</em>(D) between successive <em>k</em>; (c) population structure based on <em>k</em> = 3. Each individual is represented by a vertical bar, partitioned into coloured segments with the length of each segment representing the proportion of the individual's genome. A given group is represented: Red, Group 1; Green, Group 2; Purple, Group 3.</p></div></div>                            <h3 scrollto-destination=6655789 id="6655789" class="section-title" data-legacy-id=s3e>3.5. Association mapping</h3>
<p>We assessed the utility of six statistical models for controlling type I and II errors in this <em>B. napus</em> panel. As can be seen in the QQ plots (Fig. <a reveal-id="DSU002F2" data-open="DSU002F2" class="link link-reveal link-table xref-fig">2</a>), the distribution of observed −log<sub>10</sub>(<em>p</em>) values from the naive model departed quite far from the expected distribution leading to a high level of false-positive signals. For all four traits, any model controlling population structure or relative kinship performed signiﬁcantly better than the naive model. The models controlling relative kinship, i.e. K, Q + K and P + K, had similar effects in reducing the false positives, while P + K showed slightly greater effect than K and Q + K models. For erucic acid content and glucosinolate content, the PCA, K, Q + K and P + K models were better than the Q model. All of the models except the naive model performed well for oil content and seed weight, and no significant difference was observed among these five models for the seed weight. There were no significant associations between SNPs and oil content in the Q, P + K models based on threshold −log<sub>10</sub>(<em>p</em>) = 5.7, which might indicate false negatives in this analysis. </p>                        <a id="6655791" scrollto-destination="6655791"></a>
<div id="DSU002F2" data-id="DSU002F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00202.png?Expires=2147483647&Signature=wgFXQgyEktTC-WBq2Qgvj5YTuPehM0U1dbWG24rtkK7G1~aS5BiLkSNU86NG6I5c~wCQUSTjZRDfRVSrr8UWkeMtteN6Z8MabYK1sRSryHG5agiPIsKUkhQo3x6oscvhoSQB-CNhZKZ7uYCpSraoR7mqWUiFdQ5hKaOrsXaYmp3hJGhC2hEG03uO~-UQe2RIrBYmCfJS3KeRpVPSQ6mryj3w0U6rZJa9XPWZBH7fyNSh5X3UUn0155WvoJ-MnO5xZWHwI0VyyRgl56uWImtG0NpDkue6XIaysFPlPLzgP5QrU6yuQ8GEoVB~cPKb318NsZWfbwr4aJUGbk0yeLmB8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantile–quantile plots of estimated −log10(p) from association analysis using six methods for four traits: (a) Erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. The black line is the expected line under the null distribution, and the deviations from expectation indicate that the statistical analysis may cause spurious associations. The horizontal dashed green lines indicates genome-wide significance threshold −log10(p) = 5.7." path-from-xml="dsu00202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655791/dsu00202.tiff" path-from-xml="dsu00202.tiff" target="_blank">View large</a><a section="6655791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00202.png?Expires=2147483647&Signature=Rsr3eNHexf8cLcEmdL3kNnE4gvAFbIJjHC9t1uijO7X4qRHz2mxJquhjPiJeFjkF4DJ2ZNM28TOmFclTsJUI~5JAN6gH3szSqXJNWMmRRckwLLEetkHPCRTAIZfPiXlZPsEabFHxTwRMZzvN5SEz69xvH~WVF6ly-J1bSdFK4C49RYyj2lH0C-OdURQ2rgAkml9pGF9IseRrKZ1cZTNtMGBRpLPjg2xYQTesQkzD9XVq09IIxDgaO~BsLKLlJhy~GWAwZaPBRA1x4w6d2mWiuPOLZG2Qdqhp~ogIiA4aHI4CROQGIL3eNdWoMaWndUsSiZqjsHDjKw0meWBxfPEloA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655791&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantile–quantile plots of estimated −log<sub>10</sub>(<em>p</em>) from association analysis using six methods for four traits: (a) Erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. The black line is the expected line under the null distribution, and the deviations from expectation indicate that the statistical analysis may cause spurious associations. The horizontal dashed green lines indicates genome-wide significance threshold −log<sub>10</sub>(<em>p</em>) = 5.7.</p></div></div><div content-id="DSU002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00202.png?Expires=2147483647&Signature=wgFXQgyEktTC-WBq2Qgvj5YTuPehM0U1dbWG24rtkK7G1~aS5BiLkSNU86NG6I5c~wCQUSTjZRDfRVSrr8UWkeMtteN6Z8MabYK1sRSryHG5agiPIsKUkhQo3x6oscvhoSQB-CNhZKZ7uYCpSraoR7mqWUiFdQ5hKaOrsXaYmp3hJGhC2hEG03uO~-UQe2RIrBYmCfJS3KeRpVPSQ6mryj3w0U6rZJa9XPWZBH7fyNSh5X3UUn0155WvoJ-MnO5xZWHwI0VyyRgl56uWImtG0NpDkue6XIaysFPlPLzgP5QrU6yuQ8GEoVB~cPKb318NsZWfbwr4aJUGbk0yeLmB8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantile–quantile plots of estimated −log10(p) from association analysis using six methods for four traits: (a) Erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. The black line is the expected line under the null distribution, and the deviations from expectation indicate that the statistical analysis may cause spurious associations. The horizontal dashed green lines indicates genome-wide significance threshold −log10(p) = 5.7." path-from-xml="dsu00202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655791/dsu00202.tiff" path-from-xml="dsu00202.tiff" target="_blank">View large</a><a section="6655791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00202.png?Expires=2147483647&Signature=Rsr3eNHexf8cLcEmdL3kNnE4gvAFbIJjHC9t1uijO7X4qRHz2mxJquhjPiJeFjkF4DJ2ZNM28TOmFclTsJUI~5JAN6gH3szSqXJNWMmRRckwLLEetkHPCRTAIZfPiXlZPsEabFHxTwRMZzvN5SEz69xvH~WVF6ly-J1bSdFK4C49RYyj2lH0C-OdURQ2rgAkml9pGF9IseRrKZ1cZTNtMGBRpLPjg2xYQTesQkzD9XVq09IIxDgaO~BsLKLlJhy~GWAwZaPBRA1x4w6d2mWiuPOLZG2Qdqhp~ogIiA4aHI4CROQGIL3eNdWoMaWndUsSiZqjsHDjKw0meWBxfPEloA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655791&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantile–quantile plots of estimated −log<sub>10</sub>(<em>p</em>) from association analysis using six methods for four traits: (a) Erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. The black line is the expected line under the null distribution, and the deviations from expectation indicate that the statistical analysis may cause spurious associations. The horizontal dashed green lines indicates genome-wide significance threshold −log<sub>10</sub>(<em>p</em>) = 5.7.</p></div></div><p>According to the Q−Q plots of the six models, we used PCA and Q + K models to identify association signals. First, in order to evaluate the power of association analysis in this panel, the erucic acid content, for which genetic control was well known, was first used to perform association analysis. Two significant regions located at 9.5 and 63.7 Mb of A8 and C3, respectively, in the ‘pseudomolecules’ of <em>B. napus</em> were detected by both models (Fig. <a reveal-id="DSU002F3" data-open="DSU002F3" class="link link-reveal link-table xref-fig">3</a>a, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>a and Table <a reveal-id="DSU002TB3" data-open="DSU002TB3" class="link link-reveal link-table xref-fig">3</a>). The FDRs for associations detected in Q + K and PCA models were 0.046 and 0.312%, respectively. The two GWAS peaks, at Bn-A08-p12599446 and Bn-scaff_15794_3-p29807, contributed to 12.2 and 7.9% of phenotypic variance, respectively, based on <em>R</em><sup>2</sup> values. These two peak SNPs were found to be 233 and 128 kb away from the key genes <em>BnaA.FAE1</em> and <em>BnaC.FAE1</em>, respectively, indicating that our association genetics approach was successful. </p>                        <a id="6655793" scrollto-destination="6655793"></a>
<div content-id="DSU002TB3" class="table-modal"><div class="table-wrap"><div id="DSU002TB3" class="table-wrap-title" data-id="DSU002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Genome-wide significant association signals of seed quality and seed weight</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>SNP </th><th>Linkage group </th><th>Position in ‘pseudomolecules’ (kb) </th><th>Major allele </th><th>Minor allele </th><th>Minor allele frequency </th><th>−log<sub>10</sub> (<em>p</em>)<sup>a</sup> </th><th>Contribution<sup>a</sup> (%) </th><th>Candidate gene </th><th>Position of candidate genes in ‘pseudomolecules' (kb) </th></tr></thead><tbody><tr><td rowspan="2">Erucic acid content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>12.0 </td><td>12.27 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td>Bn-scaff_15794_3-p29807 </td><td>C3 </td><td>63 672 </td><td>A </td><td>C </td><td>0.446 </td><td>8.0 </td><td>7.9 </td><td><em>BnaC.FAE1</em> </td><td>C1: 63 800 </td></tr><tr><td rowspan="5">Glucosinolate content </td><td>Bn-A01-p9004629 </td><td>A9 </td><td>3210 </td><td>T </td><td>C </td><td>0.314 </td><td>23.6 </td><td>25.17 </td><td><em>BnaA.HAG1c</em> </td><td>A9: 3426 </td></tr><tr><td>Bn-scaff_26086_1-p11779 </td><td>C2 </td><td>50 098 </td><td>G </td><td>A </td><td>0.292 </td><td>12.7 </td><td>13.19 </td><td><em>BnaC.HAG1a</em> </td><td>C2: 49 620 </td></tr><tr><td>Bn-scaff_16534_1-p2070156<sup>b</sup> </td><td>C4 </td><td>6327 </td><td>T </td><td>C </td><td>0.084 </td><td>10.1 </td><td>5 </td><td> </td><td> </td></tr><tr><td>Bn-scaff_15705_1-p1628802 </td><td>C7 </td><td>39 871 </td><td>G </td><td>A </td><td>0.284 </td><td>6.2 </td><td>6.06 </td><td><em>BnaC.HAG1b</em> </td><td>C7: 40 676 </td></tr><tr><td>Bn-scaff_17526_1-p1140588 </td><td>C9 </td><td>2823 </td><td>C </td><td>T </td><td>0.335 </td><td>10.5 </td><td>12.26 </td><td><em>BnaC.HAG1c</em> </td><td>C9: 4725 </td></tr><tr><td>Oil content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>6.1 </td><td>6.22 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td rowspan="2">Seed weight </td><td>Bn-A10-p12639538<sup>b</sup> </td><td>A7 </td><td>2674 </td><td>T </td><td>G </td><td>0.265 </td><td>6.2 </td><td>4.9 </td><td> </td><td> </td></tr><tr><td>Bn-A09-p30654305 </td><td>A9 </td><td>34 653 </td><td>A </td><td>C </td><td>0.275 </td><td>11.4 </td><td>13.87 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSU002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>SNP </th><th>Linkage group </th><th>Position in ‘pseudomolecules’ (kb) </th><th>Major allele </th><th>Minor allele </th><th>Minor allele frequency </th><th>−log<sub>10</sub> (<em>p</em>)<sup>a</sup> </th><th>Contribution<sup>a</sup> (%) </th><th>Candidate gene </th><th>Position of candidate genes in ‘pseudomolecules' (kb) </th></tr></thead><tbody><tr><td rowspan="2">Erucic acid content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>12.0 </td><td>12.27 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td>Bn-scaff_15794_3-p29807 </td><td>C3 </td><td>63 672 </td><td>A </td><td>C </td><td>0.446 </td><td>8.0 </td><td>7.9 </td><td><em>BnaC.FAE1</em> </td><td>C1: 63 800 </td></tr><tr><td rowspan="5">Glucosinolate content </td><td>Bn-A01-p9004629 </td><td>A9 </td><td>3210 </td><td>T </td><td>C </td><td>0.314 </td><td>23.6 </td><td>25.17 </td><td><em>BnaA.HAG1c</em> </td><td>A9: 3426 </td></tr><tr><td>Bn-scaff_26086_1-p11779 </td><td>C2 </td><td>50 098 </td><td>G </td><td>A </td><td>0.292 </td><td>12.7 </td><td>13.19 </td><td><em>BnaC.HAG1a</em> </td><td>C2: 49 620 </td></tr><tr><td>Bn-scaff_16534_1-p2070156<sup>b</sup> </td><td>C4 </td><td>6327 </td><td>T </td><td>C </td><td>0.084 </td><td>10.1 </td><td>5 </td><td> </td><td> </td></tr><tr><td>Bn-scaff_15705_1-p1628802 </td><td>C7 </td><td>39 871 </td><td>G </td><td>A </td><td>0.284 </td><td>6.2 </td><td>6.06 </td><td><em>BnaC.HAG1b</em> </td><td>C7: 40 676 </td></tr><tr><td>Bn-scaff_17526_1-p1140588 </td><td>C9 </td><td>2823 </td><td>C </td><td>T </td><td>0.335 </td><td>10.5 </td><td>12.26 </td><td><em>BnaC.HAG1c</em> </td><td>C9: 4725 </td></tr><tr><td>Oil content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>6.1 </td><td>6.22 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td rowspan="2">Seed weight </td><td>Bn-A10-p12639538<sup>b</sup> </td><td>A7 </td><td>2674 </td><td>T </td><td>G </td><td>0.265 </td><td>6.2 </td><td>4.9 </td><td> </td><td> </td></tr><tr><td>Bn-A09-p30654305 </td><td>A9 </td><td>34 653 </td><td>A </td><td>C </td><td>0.275 </td><td>11.4 </td><td>13.87 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn5"><div class="fn"><p><sup>a</sup>Only the values estimated by the Q + K model were shown, if the SNPs were detected by both Q + K and PCA models.</p></div></div><div content-id="tbnfn6"><div class="fn"><p><sup>b</sup>Association signals identified only by the PCA model.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655793" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU002TB3" class="table-wrap-title" data-id="DSU002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Genome-wide significant association signals of seed quality and seed weight</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait </th><th>SNP </th><th>Linkage group </th><th>Position in ‘pseudomolecules’ (kb) </th><th>Major allele </th><th>Minor allele </th><th>Minor allele frequency </th><th>−log<sub>10</sub> (<em>p</em>)<sup>a</sup> </th><th>Contribution<sup>a</sup> (%) </th><th>Candidate gene </th><th>Position of candidate genes in ‘pseudomolecules' (kb) </th></tr></thead><tbody><tr><td rowspan="2">Erucic acid content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>12.0 </td><td>12.27 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td>Bn-scaff_15794_3-p29807 </td><td>C3 </td><td>63 672 </td><td>A </td><td>C </td><td>0.446 </td><td>8.0 </td><td>7.9 </td><td><em>BnaC.FAE1</em> </td><td>C1: 63 800 </td></tr><tr><td rowspan="5">Glucosinolate content </td><td>Bn-A01-p9004629 </td><td>A9 </td><td>3210 </td><td>T </td><td>C </td><td>0.314 </td><td>23.6 </td><td>25.17 </td><td><em>BnaA.HAG1c</em> </td><td>A9: 3426 </td></tr><tr><td>Bn-scaff_26086_1-p11779 </td><td>C2 </td><td>50 098 </td><td>G </td><td>A </td><td>0.292 </td><td>12.7 </td><td>13.19 </td><td><em>BnaC.HAG1a</em> </td><td>C2: 49 620 </td></tr><tr><td>Bn-scaff_16534_1-p2070156<sup>b</sup> </td><td>C4 </td><td>6327 </td><td>T </td><td>C </td><td>0.084 </td><td>10.1 </td><td>5 </td><td> </td><td> </td></tr><tr><td>Bn-scaff_15705_1-p1628802 </td><td>C7 </td><td>39 871 </td><td>G </td><td>A </td><td>0.284 </td><td>6.2 </td><td>6.06 </td><td><em>BnaC.HAG1b</em> </td><td>C7: 40 676 </td></tr><tr><td>Bn-scaff_17526_1-p1140588 </td><td>C9 </td><td>2823 </td><td>C </td><td>T </td><td>0.335 </td><td>10.5 </td><td>12.26 </td><td><em>BnaC.HAG1c</em> </td><td>C9: 4725 </td></tr><tr><td>Oil content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>6.1 </td><td>6.22 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td rowspan="2">Seed weight </td><td>Bn-A10-p12639538<sup>b</sup> </td><td>A7 </td><td>2674 </td><td>T </td><td>G </td><td>0.265 </td><td>6.2 </td><td>4.9 </td><td> </td><td> </td></tr><tr><td>Bn-A09-p30654305 </td><td>A9 </td><td>34 653 </td><td>A </td><td>C </td><td>0.275 </td><td>11.4 </td><td>13.87 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSU002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait </th><th>SNP </th><th>Linkage group </th><th>Position in ‘pseudomolecules’ (kb) </th><th>Major allele </th><th>Minor allele </th><th>Minor allele frequency </th><th>−log<sub>10</sub> (<em>p</em>)<sup>a</sup> </th><th>Contribution<sup>a</sup> (%) </th><th>Candidate gene </th><th>Position of candidate genes in ‘pseudomolecules' (kb) </th></tr></thead><tbody><tr><td rowspan="2">Erucic acid content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>12.0 </td><td>12.27 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td>Bn-scaff_15794_3-p29807 </td><td>C3 </td><td>63 672 </td><td>A </td><td>C </td><td>0.446 </td><td>8.0 </td><td>7.9 </td><td><em>BnaC.FAE1</em> </td><td>C1: 63 800 </td></tr><tr><td rowspan="5">Glucosinolate content </td><td>Bn-A01-p9004629 </td><td>A9 </td><td>3210 </td><td>T </td><td>C </td><td>0.314 </td><td>23.6 </td><td>25.17 </td><td><em>BnaA.HAG1c</em> </td><td>A9: 3426 </td></tr><tr><td>Bn-scaff_26086_1-p11779 </td><td>C2 </td><td>50 098 </td><td>G </td><td>A </td><td>0.292 </td><td>12.7 </td><td>13.19 </td><td><em>BnaC.HAG1a</em> </td><td>C2: 49 620 </td></tr><tr><td>Bn-scaff_16534_1-p2070156<sup>b</sup> </td><td>C4 </td><td>6327 </td><td>T </td><td>C </td><td>0.084 </td><td>10.1 </td><td>5 </td><td> </td><td> </td></tr><tr><td>Bn-scaff_15705_1-p1628802 </td><td>C7 </td><td>39 871 </td><td>G </td><td>A </td><td>0.284 </td><td>6.2 </td><td>6.06 </td><td><em>BnaC.HAG1b</em> </td><td>C7: 40 676 </td></tr><tr><td>Bn-scaff_17526_1-p1140588 </td><td>C9 </td><td>2823 </td><td>C </td><td>T </td><td>0.335 </td><td>10.5 </td><td>12.26 </td><td><em>BnaC.HAG1c</em> </td><td>C9: 4725 </td></tr><tr><td>Oil content </td><td>Bn-A08-p12599446 </td><td>A8 </td><td>9514 </td><td>T </td><td>C </td><td>0.432 </td><td>6.1 </td><td>6.22 </td><td><em>BnaA.FAE1</em> </td><td>A8: 9281 </td></tr><tr><td rowspan="2">Seed weight </td><td>Bn-A10-p12639538<sup>b</sup> </td><td>A7 </td><td>2674 </td><td>T </td><td>G </td><td>0.265 </td><td>6.2 </td><td>4.9 </td><td> </td><td> </td></tr><tr><td>Bn-A09-p30654305 </td><td>A9 </td><td>34 653 </td><td>A </td><td>C </td><td>0.275 </td><td>11.4 </td><td>13.87 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn5"><div class="fn"><p><sup>a</sup>Only the values estimated by the Q + K model were shown, if the SNPs were detected by both Q + K and PCA models.</p></div></div><div content-id="tbnfn6"><div class="fn"><p><sup>b</sup>Association signals identified only by the PCA model.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6655793" target="_blank">View Large</a></div></div>                        <a id="6655794" scrollto-destination="6655794"></a>
<div id="DSU002F3" data-id="DSU002F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00203.png?Expires=2147483647&Signature=yaA3KzcFDIJncgYM9~evqZ3As~cMKOTIHMPiL31chBCD48KRC~YyakztCU4ZSV1NlDKm7xyztf3~X8-bxxCabUc6Zuzl8orlT62RT0ZaS~moqrqDgXC63bFG33hhX~hrRfTzG0Pwapueu9KvU-hxrbHA4Dt3boaEfVNA9f2J7AoHsBCKuF-XWPnZcsviAST5dKoFwrKC47GKLeaiHH1P9mN-vXBEd0PC2puFaEatNcAenxoWGBu~ZDZq9F7Wfj430ZDOCu1ZSviDQNdvU07tIJfSNId7PLRUMNqJXDQv5QEde0IdQvq8oh9wUq3PUhi36uosxiWniomahifbKinOow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Manhattan plots of association analysis using the Q + K model for four traits: (a) erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. Each dot represents a SNP. The horizontal dashed green line represents the Bonferroni-corrected significance threshold −log10(p) = 5.7." path-from-xml="dsu00203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655794/dsu00203.tiff" path-from-xml="dsu00203.tiff" target="_blank">View large</a><a section="6655794" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00203.png?Expires=2147483647&Signature=KYfZ4lhOJ3OYY02z9n2P4vlNmUrLntHCwckVmq6dHFk63tUuL6Y4mzSwk3istHCvc9BwwTdN0IR-rc-TdH0ocYTOee21cOabcAYZBoGGptWiFOnNKzQD~QVgfKQlElZAgR80oU4qboe9Ex6th7BgU4DyiJh7qCerHasOH6MvC0Lm1IUAfyOJzRZejt~9e2FfBsQs2PB8GOAHAl1tjmP0KO-gpxKvRYQiAbiccPvR6xibaBunv8mC9oo7eVtF4BWhMidN82gsCpTlonZIR-w8CutmxbLJRTFGWXZO8nLAh-jqFQ6-R1s1Rch-Gt9OCSsmV6H~u1FuI~G6wekvRohXJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655794&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Manhattan plots of association analysis using the Q + K model for four traits: (a) erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. Each dot represents a SNP. The horizontal dashed green line represents the Bonferroni-corrected significance threshold −log<sub>10</sub>(<em>p</em>) = 5.7.</p></div></div><div content-id="DSU002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/m_dsu00203.png?Expires=2147483647&Signature=yaA3KzcFDIJncgYM9~evqZ3As~cMKOTIHMPiL31chBCD48KRC~YyakztCU4ZSV1NlDKm7xyztf3~X8-bxxCabUc6Zuzl8orlT62RT0ZaS~moqrqDgXC63bFG33hhX~hrRfTzG0Pwapueu9KvU-hxrbHA4Dt3boaEfVNA9f2J7AoHsBCKuF-XWPnZcsviAST5dKoFwrKC47GKLeaiHH1P9mN-vXBEd0PC2puFaEatNcAenxoWGBu~ZDZq9F7Wfj430ZDOCu1ZSviDQNdvU07tIJfSNId7PLRUMNqJXDQv5QEde0IdQvq8oh9wUq3PUhi36uosxiWniomahifbKinOow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Manhattan plots of association analysis using the Q + K model for four traits: (a) erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. Each dot represents a SNP. The horizontal dashed green line represents the Bonferroni-corrected significance threshold −log10(p) = 5.7." path-from-xml="dsu00203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6655794/dsu00203.tiff" path-from-xml="dsu00203.tiff" target="_blank">View large</a><a section="6655794" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu00203.png?Expires=2147483647&Signature=KYfZ4lhOJ3OYY02z9n2P4vlNmUrLntHCwckVmq6dHFk63tUuL6Y4mzSwk3istHCvc9BwwTdN0IR-rc-TdH0ocYTOee21cOabcAYZBoGGptWiFOnNKzQD~QVgfKQlElZAgR80oU4qboe9Ex6th7BgU4DyiJh7qCerHasOH6MvC0Lm1IUAfyOJzRZejt~9e2FfBsQs2PB8GOAHAl1tjmP0KO-gpxKvRYQiAbiccPvR6xibaBunv8mC9oo7eVtF4BWhMidN82gsCpTlonZIR-w8CutmxbLJRTFGWXZO8nLAh-jqFQ6-R1s1Rch-Gt9OCSsmV6H~u1FuI~G6wekvRohXJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6655794&ar=445136&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Manhattan plots of association analysis using the Q + K model for four traits: (a) erucic acid content; (b) glucosinolate content; (c) oil content and (d) seed weight. Each dot represents a SNP. The horizontal dashed green line represents the Bonferroni-corrected significance threshold −log<sub>10</sub>(<em>p</em>) = 5.7.</p></div></div><p>Four and five significant associations for glucosinolate content were detected by Q + K and PCA models, respectively, with the FDR of 0.083 and 0.106%. Both models detected four common regions at 3.2, 50.0, 39.9 and 2.8 Mb of A9, C2, C7 and C9, respectively, in the ‘pseudomolecules’ of <em>B. napus</em> (Fig. <a reveal-id="DSU002F3" data-open="DSU002F3" class="link link-reveal link-table xref-fig">3</a>b, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>b and Table <a reveal-id="DSU002TB3" data-open="DSU002TB3" class="link link-reveal link-table xref-fig">3</a>). The cumulative phenotypic variance explained by all signiﬁcant SNPs was 56.7%. As the deletions of the <em>A. thaliana</em> orthologous gene <em>HAG1</em> (At5g61420) in C2 and A9 were regarded to lead to low glucosinolate content,<sup><a reveal-id="DSU002C34" data-open="DSU002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> we searched the <em>HAG1</em> paralogues in the ‘pseudomolecules’ of <em>B. napus</em> and found four paralogues at 3.4 Mb of A9, 49.6 Mb of C2, 40.7 Mb of C7 and 4.7 Mb of C9, which were very similar to the four GWAS peaks. The candidate gene <em>BnaC.HAG1c</em> was 1.9 Mb away from the peak SNP Bn-scaff_17526_1-p1140588, which might be due to the low density of SNPs around the candidate gene. In addition, the PCA model also detected a novel locus at 6.33 Mb of C4, which contributed 5.0% of the phenotypic variation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>b and Table <a reveal-id="DSU002TB3" data-open="DSU002TB3" class="link link-reveal link-table xref-fig">3</a>). For oil content, only one significantly associated region in A8 was detected by Q + K and PCA models with FDRs of 2.5 and 0.714%, respectively (Fig. <a reveal-id="DSU002F3" data-open="DSU002F3" class="link link-reveal link-table xref-fig">3</a>c, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>c and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsu002/-/DC1" target="_blank">Table S3</a>). The GWAS peaks explained 6.1% of the total phenotypic variance. The GWAS peak for oil content was similar to that for erucic acid. For seed weight, Q + K and PCA models detected one association in A9 and A7 with the FDR of 1.667 and 0.384%, respectively. The GWAS peak detected by Q + K, i.e. Bn-A09-p30654305, at 34.65 Mb of A9, explained 11.4% of the phenotypic variance (Fig. <a reveal-id="DSU002F3" data-open="DSU002F3" class="link link-reveal link-table xref-fig">3</a>d, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>d and Table <a reveal-id="DSU002TB3" data-open="DSU002TB3" class="link link-reveal link-table xref-fig">3</a>). The SNP Bn-A09-p30654305 was also detected by the PCA model as a single associated SNP in A9. In addition, using the PCA model, we also detected an associated SNP Bn-A10-p12639538 at 2.67 Mb of A7 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>d and Table <a reveal-id="DSU002TB3" data-open="DSU002TB3" class="link link-reveal link-table xref-fig">3</a>), which explained 4.9% of the total seed weight variation.</p>                            <h2 scrollto-destination=6655796 id="6655796" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>High-quality molecular markers and reliable genotype data are the precondition of genetic diversity analysis and association mapping. The amphidiploid nature of the <em>B. napus</em> genome might make automated SNP detection challenging and confounding, with homologue, paralogue and also homoeologue variation.<sup><a reveal-id="DSU002C30" data-open="DSU002C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSU002C51" data-open="DSU002C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Trick <em>et al.</em><sup><a reveal-id="DSU002C30" data-open="DSU002C30" class="link link-ref link-reveal xref-bibr">30</a></sup> termed three types of SNPs, i.e. inter-homoeologue polymorphisms, hemi-SNPs and simple SNPs in <em>B. napus</em>. The inter-homoeologue polymorphisms do not represent allelic variation <em>per se</em>, and as hemi-SNPs are the allelic polymorphisms in the homoeologous sequences, the short flanking sequences of the hemi-SNPs could not be specifically mapped in the ‘pseudomolecules’ representative of the genome of <em>B. napus</em>. By scanning the rapeseed association panel using the 60K SNP array, the clustering results showed that ∼10 000 SNPs had zero call frequency of AA or BB, which could, in part, be caused by inter-homoeologue polymorphisms. Meanwhile, ∼10 000 SNPs showed more than three clusters, which are indicative of two loci in a duplicated sequence as suggested by Ganal <em>et al.</em>,<sup><a reveal-id="DSU002C52" data-open="DSU002C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and are possibly hemi-SNPs. By strictly filtering, the genotypes of the finally selected SNPs of all DH lines were completely homozygous, and the DNA pedigree consistency and technical reproducibility were nearly perfect, suggesting the high quality of these selected SNPs.</p><p>On the whole, over 76% of the SNPs have the PIC of 0.25–0.5, which was regarded as the PIC range of medium polymorphism of the DNA marker.<sup><a reveal-id="DSU002C53" data-open="DSU002C53" class="link link-ref link-reveal xref-bibr">53</a></sup> On average, the polymorphism level was slightly higher for the A than for the C linkage groups in our population, contrary to the results of Delourme <em>et al.</em><sup><a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In our panel, nearly half of the genotypes were Asiatic, while those of Delourme <em>et al.</em> were European.<sup><a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Asiatic cultivars were selected for improved adaptation to the local environment by introgression of <em>B. rapa</em> into <em>B. napus</em> germplasm,<sup><a reveal-id="DSU002C54" data-open="DSU002C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSU002C55" data-open="DSU002C55" class="link link-ref link-reveal xref-bibr">55</a></sup> increasing the genetic diversity of the A genome. Our ﬁnding suggests that genetic diversity of <em>B. napus</em> germplasm should be broadened by introgression of <em>B. oleracea</em>, as described by previous studies.<sup><a reveal-id="DSU002C24" data-open="DSU002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU002C56" data-open="DSU002C56" class="link link-ref link-reveal xref-bibr">56</a></sup></p><p>Understanding the population structure of the association mapping population contributes to reducing both Type I and II errors between molecular markers and traits of interest.<sup><a reveal-id="DSU002C46" data-open="DSU002C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSU002C57" data-open="DSU002C57" class="link link-ref link-reveal xref-bibr">57</a></sup> In our study, the association population was classified into three groups, semi-winter, winter and spring OSR were the main components of these three groups. Distinct clustering of winter and spring types has been reported by previous studies.<sup><a reveal-id="DSU002C24" data-open="DSU002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSU002C58" data-open="DSU002C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSU002C59" data-open="DSU002C59" class="link link-ref link-reveal xref-bibr">59</a></sup> In our study, although Groups 2 and 3 mainly distinguished winter and spring types, respectively, some winter and spring types were found in the mixed group, located at an intermediate position between Groups 2 and 3. Delourme <em>et al.</em><sup><a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> also observed that some spring OSR lines did not show distinct clustering. These intermediate lines might be derived from hybrids of the lines from different gene pools.<sup><a reveal-id="DSU002C60" data-open="DSU002C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSU002C61" data-open="DSU002C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The kinship analysis revealed that most lines in the panel have no or weak kinship, together with significant differences in phenotype performance, indicating that this panel is suitable for association analysis. The mean pairwise <em>r</em><sup>2</sup> values is close to previous estimates of 0.0117<sup><a reveal-id="DSU002C26" data-open="DSU002C26" class="link link-ref link-reveal xref-bibr">26</a></sup> or 0.0247<sup><a reveal-id="DSU002C34" data-open="DSU002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and lower than 0.037 estimated for the population used by Delourme <em>et al.</em>,<sup><a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> confirming the low overall level of LD in <em>B. napus</em>. At the genome level, mean LD decay was estimated to be 0.5–1.2 cM in rapeseed germplasm.<sup><a reveal-id="DSU002C24" data-open="DSU002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU002C26" data-open="DSU002C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSU002C32" data-open="DSU002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Given the length of the genetic map of 2500 cM, &gt;2100–5000 evenly spaced markers would be necessary to perform GWASs in rapeseed. Therefore, by selecting in excess of 20 000 SNPs, our study should have more than sufficient markers to perform a good association analysis.</p><p>In the previous association analysis study in rapeseed, generally only population structure estimated by STRUCTURE<sup><a reveal-id="DSU002C40" data-open="DSU002C40" class="link link-ref link-reveal xref-bibr">40</a></sup> was considered.<sup><a reveal-id="DSU002C27  DSU002C29" data-open="DSU002C27  DSU002C29" class="link link-ref link-reveal xref-bibr">27–29</a>,<a reveal-id="DSU002C62" data-open="DSU002C62" class="link link-ref link-reveal xref-bibr">62</a></sup> However, spurious associations cannot be controlled completely by population structure since the Q matrix only gives a rough dissection of population differentiation. Furthermore, the programme STRUCTURE needs intensive computational cost on large data sets. The current PCA software such as EIGENSTRAT<sup><a reveal-id="DSU002C63" data-open="DSU002C63" class="link link-ref link-reveal xref-bibr">63</a></sup> and GCTA<sup><a reveal-id="DSU002C39" data-open="DSU002C39" class="link link-ref link-reveal xref-bibr">39</a></sup> could analyse tens of thousands of samples with millions of SNPs with high performance and infer continuous axes of genetic variation; therefore, the PCA based on genome-wide SNPs was broadly used to detect and correct for population stratiﬁcation in GWAS;<sup><a reveal-id="DSU002C18" data-open="DSU002C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSU002C19" data-open="DSU002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> however, a few residual false-positive associations still existed in the PCA model.<sup><a reveal-id="DSU002C64" data-open="DSU002C64" class="link link-ref link-reveal xref-bibr">64</a></sup> Yu <em>et al.</em><sup><a reveal-id="DSU002C46" data-open="DSU002C46" class="link link-ref link-reveal xref-bibr">46</a></sup> suggested that integration of the pairwise kinship into a mixed model to correct for relatedness could reduce spurious association, which was subsequently supported by the study in maize,<sup><a reveal-id="DSU002C48" data-open="DSU002C48" class="link link-ref link-reveal xref-bibr">48</a></sup> sorghum<sup><a reveal-id="DSU002C65" data-open="DSU002C65" class="link link-ref link-reveal xref-bibr">65</a></sup> and barley.<sup><a reveal-id="DSU002C66" data-open="DSU002C66" class="link link-ref link-reveal xref-bibr">66</a></sup> In our study, six models, i.e. naive, Q, PCA, K, Q + K and P + K models, were first compared for controlling the false positives in <em>B. napus</em> association mapping. The models considering population structure or relatedness, especially PCA, K, Q + K and P + K models, could effectively eliminate the excess of low <em>p</em>-values for all traits; however, the P + K model also likely eliminated true positives, which is a common problem seen in other systems as well.<sup><a reveal-id="DSU002C16" data-open="DSU002C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU002C67" data-open="DSU002C67" class="link link-ref link-reveal xref-bibr">67</a></sup> Therefore, in order to reduce the false positives and false negatives, both Q + K and PCA models were considered to identify association signals.</p><p>The position of the DNA markers relative to the genome sequence or linkage map is also preferable for successful association mapping. In this study, the SNPs of the array were <em>in silico</em> mapped in <em>B. napus</em> ‘pseudomolecules’ to establish their hypothetical order. The positions of the significant association signals detected in this study are consistent with previous studies. It has been shown that two homoeologues, namely <em>BnaA.FAE1</em> and <em>BnaC.FAE1</em> on linkage groups A8 and C3<em>,</em> respectively, are responsible for controlling erucic acid content.<sup><a reveal-id="DSU002C68" data-open="DSU002C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DSU002C69" data-open="DSU002C69" class="link link-ref link-reveal xref-bibr">69</a></sup> In the present study, the peaks of the two association signals for erucic acid content (represented by the markers Bn-A08-p12599446 and Bn-scaff_15794_3-p29807, which have the highest significance of association on A8 and C3, respectively) were within 233 kb of these two genes, indicating that our association approach was successful.</p><p>For glucosinolate content, we identified four association signals on A9, C2, C7 and C9, which were detected independently in different studies.<sup><a reveal-id="DSU002C7" data-open="DSU002C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU002C70" data-open="DSU002C70" class="link link-ref link-reveal xref-bibr">70</a>,<a reveal-id="DSU002C71" data-open="DSU002C71" class="link link-ref link-reveal xref-bibr">71</a></sup> Howell <em>et al.</em><sup><a reveal-id="DSU002C70" data-open="DSU002C70" class="link link-ref link-reveal xref-bibr">70</a></sup> regarded that the QTLs on A9, C2 and C9 were homoeologous loci, whereas the underlying control genes were not uncovered until Harper <em>et al.</em><sup><a reveal-id="DSU002C34" data-open="DSU002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> identified genomic deletions that underlie the QTLs on A9 and C2. In the low-glucosinolate rapeseed accessions, the deleted segments contained orthologs of the transcription factor <em>HAG1</em> (At5g61420),<sup><a reveal-id="DSU002C34" data-open="DSU002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> which controls aliphatic glucosinolate biosynthesis in <em>A. thaliana</em>.<sup><a reveal-id="DSU002C72" data-open="DSU002C72" class="link link-ref link-reveal xref-bibr">72</a></sup> In <em>Brassica juncea</em>, silencing of the orthologs of <em>HAG1</em> was reported to cause low glucosinolate content.<sup><a reveal-id="DSU002C73" data-open="DSU002C73" class="link link-ref link-reveal xref-bibr">73</a></sup> Interestingly, in the present study, four peak SNPs associated with glucosinolate content were identified to be close to the paralogues of <em>HAG1</em>. Further studies are necessary to reveal the DNA variation of the other two paralogues of <em>HAG1</em> on C7 and C9.</p><p>Control of seed oil content is complex and previous studies have detected at least 10 QTLs, which were environmentally sensitive and with minor effects.<sup><a reveal-id="DSU002C74  DSU002C76" data-open="DSU002C74  DSU002C76" class="link link-ref link-reveal xref-bibr">74–76</a></sup> These studies commonly regarded that two QTLs on A8 and C3 overlapped with the QTLs for erucic acid content. Since the <em>BnaA.FAE1</em> and <em>BnaC.FAE1</em> were cloned and confirmed as the key factors controlling the synthesis of erucic acid in rapeseed,<sup><a reveal-id="DSU002C68" data-open="DSU002C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DSU002C69" data-open="DSU002C69" class="link link-ref link-reveal xref-bibr">69</a></sup> it was regarded that the pleiotropy of <em>FAE1</em> was responsible for the decrease in seed oil content along with the reduction of seed erucic acid content in the modern cultivars. Our study showed that the peak SNP on A8 was only 233 kb away from the <em>BnaA.FAE1</em>, indicating the role of fine mapping of GWAS.</p><p>Seed weight of rapeseed is also a complex trait, which has a relatively high heritability and may primarily be controlled by genes with additive effects.<sup><a reveal-id="DSU002C10" data-open="DSU002C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU002C11" data-open="DSU002C11" class="link link-ref link-reveal xref-bibr">11</a></sup> So far, &gt;80 QTLs across the 19 linkage groups have been identiﬁed and individual QTL contributed to 2.0–28.2% of the phenotypic variation.<sup><a reveal-id="DSU002C10  DSU002C12" data-open="DSU002C10  DSU002C12" class="link link-ref link-reveal xref-bibr">10–12</a>,<a reveal-id="DSU002C77  DSU002C80" data-open="DSU002C77  DSU002C80" class="link link-ref link-reveal xref-bibr">77–80</a></sup> The seed weight QTLs on A7 were repeatedly detected in many studies with diverse genetic materials,<sup><a reveal-id="DSU002C10  DSU002C12" data-open="DSU002C10  DSU002C12" class="link link-ref link-reveal xref-bibr">10–12</a>,<a reveal-id="DSU002C78" data-open="DSU002C78" class="link link-ref link-reveal xref-bibr">78</a>,<a reveal-id="DSU002C80" data-open="DSU002C80" class="link link-ref link-reveal xref-bibr">80</a></sup> and the QTL on A9 was identified as a major QTL with the most contribution of 28.2% for the total seed weight variation.<sup><a reveal-id="DSU002C12" data-open="DSU002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Interestingly, our association mapping also detected a major QTL on A9 and a minor QTL on A7, indicating our association panel might be suitable for dissecting complex traits in <em>B. napus</em>. Since associated markers and causal polymorphisms underlying traits of interest are in a narrow interval equivalent to the distance of LD decay,<sup><a reveal-id="DSU002C57" data-open="DSU002C57" class="link link-ref link-reveal xref-bibr">57</a></sup> the genes flanking the peak SNPs and falling within the window of LD decay were always used to predict the candidate genes.<sup><a reveal-id="DSU002C81" data-open="DSU002C81" class="link link-ref link-reveal xref-bibr">81</a></sup> As the average LD across our panel is very low (mean pairwise <em>r</em><sup>2</sup> = 0.0176), it is reasonable to suppose that the detected loci in our study will be located in close proximity to the candidate genes controlling seed weight, and that these tightly associated SNPs would be of significant benefit in a molecular design breeding approach to improve seed weight.</p>                            <h2 scrollto-destination=6655805 id="6655805" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6655807 id="6655807" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Chinese National Basic Research and Development Program (2011CB109302), the National Natural Science Foundation of China (31301360), Strategic Japanese-Chinese Cooperative Program on ‘Climate Change’ (2012DFG90290), National Science and Technology Pillar Program during the Twelfth Five-year Plan Period (2011BAD35B09) and Chengguang Program for Young Scientists of Wuhan Municipal Government (2013070104010031).</p>                            <h2 scrollto-destination=6655809 id="6655809" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rana</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">van den Boogaart</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">O'Neill</div> <div class="given-names">C.M.</div></div>, et al. </div><div class="article-title">Conservation of the microstructure of genome segments in <em>Brassica napus</em> and its diploid relatives</div>, <div class="source">Plant J.</div> , <div class="year">2004</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">725</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20the%20microstructure%20of%20genome%20segments%20in%20Brassica%20napus%20and%20its%20diploid%20relatives&amp;author=D.%20Rana&amp;author=T.%20van%20den%20Boogaart&amp;author=C.M.%20O'Neill&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=40&amp;pages=725-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.40.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.40.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.40.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15546355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagaharu</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Genome-analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>, <div class="source">Japan J. Bot.</div> , <div class="year">1935</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="DSU002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gomez-Campo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Prakash</div> <div class="given-names">S.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Gomez-Campo</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Origin and domestication</div>, <div class="source">Biology of Brassica coenospecies</div> , <div class="year">1999</div><div class="publisher-loc">Amsterdam</div><div class="publisher-name">Elsevier</div>(pg. <div class="fpage">33</div>-<div class="lpage">58</div>)</div></div></div></div><div content-id="DSU002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prakash</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.-M.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">S.R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Janick</div> <div class="given-names">J.</div></div>. </div><div class="article-title">History, evolution, and domestication of <em>Brassica</em> crops</div>, <div class="source">Plant Breeding Reviews</div> , <div class="year">2011</div>, vol. <div class="volume">vol. 35</div> <div class="publisher-loc">New Jersey</div><div class="publisher-name">John Wiley &amp; Sons, Inc.</div>(pg. <div class="fpage">19</div>-<div class="lpage">84</div>)</div></div></div></div><div content-id="DSU002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toxopeus</div> <div class="given-names">H.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Van Marrewijk</div> <div class="given-names">N.P.A.</div></div>,  <div class="name"><div class="surname">Toxopeus</div> <div class="given-names">H.</div></div>. </div>, <div class="source">The domestication of <em>Brassica</em> crops</div> , <div class="year">1979</div><div class="publisher-loc">Wageningen</div><div class="publisher-name">Proc Eucarpia Conference on the breeding of Cruciferous crops</div>(pg. <div class="fpage">47</div>-<div class="lpage">56</div>)</div></div></div></div><div content-id="DSU002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Conditional QTL mapping of oil content in rapeseed with respect to protein content and traits related to plant development and grain yield</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">33</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conditional%20QTL%20mapping%20of%20oil%20content%20in%20rapeseed%20with%20respect%20to%20protein%20content%20and%20traits%20related%20to%20plant%20development%20and%20grain%20yield&amp;author=J.%20Zhao&amp;author=H.%20Becker&amp;author=D.%20Zhang&amp;author=Y.%20Zhang&amp;author=W.%20Ecke&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=33-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0267-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0267-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0267-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16614833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uzunova</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weissleder</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Röbbelen</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Mapping the genome of rapeseed (<em>Brassica napus</em> L.). I. Construction of an RFLP linkage map and localization of QTLs for seed glucosinolate content</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">194</div>-<div class="lpage">204</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20the%20genome%20of%20rapeseed%20(Brassica%20napus%20L.).%20I.%20Construction%20of%20an%20RFLP%20linkage%20map%20and%20localization%20of%20QTLs%20for%20seed%20glucosinolate%20content&amp;author=M.%20Uzunova&amp;author=W.%20Ecke&amp;author=K.%20Weissleder&amp;author=G.%20R%C3%B6bbelen&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=90&amp;pages=194-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24173891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dimov</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Möllers</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Mapping QTL controlling fatty acid composition in a doubled haploid rapeseed population segregating for oil content</div>, <div class="source">Mol. Breed</div> , <div class="year">2008</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">115</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20QTL%20controlling%20fatty%20acid%20composition%20in%20a%20doubled%20haploid%20rapeseed%20population%20segregating%20for%20oil%20content&amp;author=J.%20Zhao&amp;author=Z.%20Dimov&amp;author=H.%20Becker&amp;author=W.%20Ecke&amp;author=C.%20M%C3%B6llers&amp;publication_year=2008&amp;journal=Mol.%20Breed&amp;volume=21&amp;pages=115-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-007-9113-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-007-9113-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-007-9113-y"> </span></div> </div></div></div></div></div><div content-id="DSU002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Flowering time quantitative trait loci analysis of oilseed <em>Brassica</em> in multiple environments and genomewide alignment with Arabidopsis</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">2433</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flowering%20time%20quantitative%20trait%20loci%20analysis%20of%20oilseed%20Brassica%20in%20multiple%20environments%20and%20genomewide%20alignment%20with%20Arabidopsis&amp;author=Y.%20Long&amp;author=J.%20Shi&amp;author=D.%20Qiu&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=2433-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18073439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Unraveling the complex trait of crop yield with quantitative trait loci mapping in <em>Brassica napus</em></div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">851</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unraveling%20the%20complex%20trait%20of%20crop%20yield%20with%20quantitative%20trait%20loci%20mapping%20in%20Brassica%20napus&amp;author=J.%20Shi&amp;author=R.%20Li&amp;author=D.%20Qiu&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=851-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.101642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.101642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.101642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19414564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Radoev</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Genetic analysis of heterosis for yield and yield components in rapeseed (<em>Brassica napus</em> L.) by quantitative trait locus mapping</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">1547</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20heterosis%20for%20yield%20and%20yield%20components%20in%20rapeseed%20(Brassica%20napus%20L.)%20by%20quantitative%20trait%20locus%20mapping&amp;author=M.%20Radoev&amp;author=H.%20Becker&amp;author=W.%20Ecke&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=1547-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.089680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.089680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.089680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18562665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Shu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Identification of a major QTL for silique length and seed weight in oilseed rape (<em>Brassica napus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">285</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20major%20QTL%20for%20silique%20length%20and%20seed%20weight%20in%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=P.%20Yang&amp;author=C.%20Shu&amp;author=L.%20Chen&amp;author=J.%20Xu&amp;author=J.%20Wu&amp;author=K.%20Liu&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=285-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1833-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1833-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1833-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22406980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Identification and mapping of a major dominant quantitative trait locus controlling seeds per silique as a single Mendelian factor in <em>Brassica napus</em> L.</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">695</div>-<div class="lpage">705</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20mapping%20of%20a%20major%20dominant%20quantitative%20trait%20locus%20controlling%20seeds%20per%20silique%20as%20a%20single%20Mendelian%20factor%20in%20Brassica%20napus%20L.&amp;author=L.%20Zhang&amp;author=S.%20Li&amp;author=L.%20Chen&amp;author=G.%20Yang&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=695-705" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1861-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1861-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1861-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22487878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nordborg</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Next-generation genetics in plants</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">720</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20genetics%20in%20plants&amp;author=M.%20Nordborg&amp;author=D.%20Weigel&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=720-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07629" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07629" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07629"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19079047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nordborg</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tavare</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Linkage disequilibrium: what history has to tell us</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">83</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%3A%20what%20history%20has%20to%20tell%20us&amp;author=M.%20Nordborg&amp;author=S.%20Tavare&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=83-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02557-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02557-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02557-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11818140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tung</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Eizenga</div> <div class="given-names">G.C.</div></div>, et al. </div><div class="article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em></div>, <div class="source">Nat. Commun.</div> , <div class="year">2011</div>, vol. <div class="volume">2</div> pg. <div class="fpage">467</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20reveals%20a%20rich%20genetic%20architecture%20of%20complex%20traits%20in%20Oryza%20sativa&amp;author=K.%20Zhao&amp;author=C.W.%20Tung&amp;author=G.C.%20Eizenga&amp;publication_year=2011&amp;journal=Nat.%20Commun.&amp;volume=2&amp;pages=467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms1467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms1467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms1467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21915109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.J.</div></div>, et al. </div><div class="article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">159</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20leaf%20architecture%20in%20the%20maize%20nested%20association%20mapping%20population&amp;author=F.%20Tian&amp;author=P.J.%20Bradbury&amp;author=P.J.%20Brown&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=159-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">961</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">32</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20flowering%20time%20and%20grain%20yield%20traits%20in%20a%20worldwide%20collection%20of%20rice%20germplasm&amp;author=X.%20Huang&amp;author=Y.%20Zhao&amp;author=X.%20Wei&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=32-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.1018"> </span></div> </div></div></div></div></div><div content-id="DSU002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genome-wide association study dissects the genetic architecture of oil biosynthesis in maize kernels</div>, <div class="source">Nat. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">43</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20dissects%20the%20genetic%20architecture%20of%20oil%20biosynthesis%20in%20maize%20kernels&amp;author=H.%20Li&amp;author=Z.%20Peng&amp;author=X.%20Yang&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=43-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23242369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Atwell</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Vilhjalmsson</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">465</div> (pg. <div class="fpage">627</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20107%20phenotypes%20in%20Arabidopsis%20thaliana%20inbred%20lines&amp;author=S.%20Atwell&amp;author=Y.S.%20Huang&amp;author=B.J.%20Vilhjalmsson&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=627-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20336072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mandakova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Chromosomal phylogeny and karyotype evolution in x=7 crucifer species (Brassicaceae)</div>, <div class="source">Plant Cell</div> , <div class="year">2008</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2559</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20phylogeny%20and%20karyotype%20evolution%20in%20x%3D7%20crucifer%20species%20(Brassicaceae)&amp;author=T.%20Mandakova&amp;author=M.A.%20Lysak&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=2559-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.062166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.062166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.062166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18836039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkin</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Gulden</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Segmental structure of the <em>Brassica napus</em> genome based on comparative analysis with <em>Arabidopsis thaliana</em></div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">765</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20structure%20of%20the%20Brassica%20napus%20genome%20based%20on%20comparative%20analysis%20with%20Arabidopsis%20thaliana&amp;author=I.%20Parkin&amp;author=S.%20Gulden&amp;author=A.%20Sharpe&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=765-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16020789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bus</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Korber</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Stich</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Patterns of molecular variation in a species-wide germplasm set of <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">1413</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20molecular%20variation%20in%20a%20species-wide%20germplasm%20set%20of%20Brassica%20napus&amp;author=A.%20Bus&amp;author=N.%20Korber&amp;author=R.J.%20Snowdon&amp;author=B.%20Stich&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=1413-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1676-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1676-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1676-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21847624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Clemens</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Honsdorf</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.C.</div></div>. </div><div class="article-title">Extent and structure of linkage disequilibrium in canola quality winter rapeseed (<em>Brassica napus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">921</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extent%20and%20structure%20of%20linkage%20disequilibrium%20in%20canola%20quality%20winter%20rapeseed%20(Brassica%20napus%20L.)&amp;author=W.%20Ecke&amp;author=R.%20Clemens&amp;author=N.%20Honsdorf&amp;author=H.C.%20Becker&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=921-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1221-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1221-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1221-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19953219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Genetic structure and linkage disequilibrium pattern of a rapeseed (<em>Brassica napus</em> L.) association mapping panel revealed by microsatellites</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">437</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20linkage%20disequilibrium%20pattern%20of%20a%20rapeseed%20(Brassica%20napus%20L.)%20association%20mapping%20panel%20revealed%20by%20microsatellites&amp;author=Y.%20Xiao&amp;author=D.%20Cai&amp;author=W.%20Yang&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1843-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1843-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1843-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22437490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zou</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Association mapping of seed oil content in <em>Brassica napus</em> and comparison with quantitative trait loci identified from linkage mapping</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">908</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20seed%20oil%20content%20in%20Brassica%20napus%20and%20comparison%20with%20quantitative%20trait%20loci%20identified%20from%20linkage%20mapping&amp;author=J.%20Zou&amp;author=C.%20Jiang&amp;author=Z.%20Cao&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=908-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Honsdorf</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Association mapping for phenological, morphological, and quality traits in canola quality winter rapeseed (<em>Brassica napus</em> L.)</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">899</div>-<div class="lpage">907</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20for%20phenological%2C%20morphological%2C%20and%20quality%20traits%20in%20canola%20quality%20winter%20rapeseed%20(Brassica%20napus%20L.)&amp;author=N.%20Honsdorf&amp;author=H.C.%20Becker&amp;author=W.%20Ecke&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=899-907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Wittkop</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Rezaidad</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Regional association analysis delineates a sequenced chromosome region influencing antinutritive seed meal compounds in oilseed rape</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">917</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regional%20association%20analysis%20delineates%20a%20sequenced%20chromosome%20region%20influencing%20antinutritive%20seed%20meal%20compounds%20in%20oilseed%20rape&amp;author=R.J.%20Snowdon&amp;author=B.%20Wittkop&amp;author=A.%20Rezaidad&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=917-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Single nucleotide polymorphism (SNP) discovery in the polyploid <em>Brassica napus</em> using Solexa transcriptome sequencing</div>, <div class="source">Plant Biotechnol. J</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">334</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20(SNP)%20discovery%20in%20the%20polyploid%20Brassica%20napus%20using%20Solexa%20transcriptome%20sequencing&amp;author=M.%20Trick&amp;author=Y.%20Long&amp;author=J.%20Meng&amp;author=I.%20Bancroft&amp;publication_year=2009&amp;journal=Plant%20Biotechnol.%20J&amp;volume=7&amp;pages=334-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2009.7.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2009.7.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2009.7.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19207216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bancroft</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Dissecting the genome of the polyploid crop oilseed rape by transcriptome sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">762</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20the%20genome%20of%20the%20polyploid%20crop%20oilseed%20rape%20by%20transcriptome%20sequencing&amp;author=I.%20Bancroft&amp;author=C.%20Morgan&amp;author=F.%20Fraser&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=762-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21804563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delourme</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Falentin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fomeju</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">High-density SNP-based genetic map development and linkage disequilibrium assessment in <em>Brassica napus</em> L.</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">120</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20SNP-based%20genetic%20map%20development%20and%20linkage%20disequilibrium%20assessment%20in%20Brassica%20napus%20L.&amp;author=R.%20Delourme&amp;author=C.%20Falentin&amp;author=B.%20Fomeju&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23432809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">1035</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harper</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Associative transcriptomics of traits in the polyploid crop species <em>Brassica napus</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">798</div>-<div class="lpage">802</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associative%20transcriptomics%20of%20traits%20in%20the%20polyploid%20crop%20species%20Brassica%20napus&amp;author=A.L.%20Harper&amp;author=M.%20Trick&amp;author=J.%20Higgins&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=798-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22820317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thies</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Schnelle und einfache Analysen der Fettsäurezusammensetzung in einzelnen Rapskotyledonen I. Gaschromatographische und papierchromatographische Methoden</div>, <div class="source">Z Pflanzenzücht</div> , <div class="year">1971</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">181</div>-<div class="lpage">202</div>)</div></div></div></div><div content-id="DSU002C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Merk</div> <div class="given-names">H.L.</div></div>,  <div class="name"><div class="surname">Yarnes</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Trait diversity and potential for selection indices based on variation among regionally adapted processing tomato germplasm</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">427</div>-<div class="lpage">37</div>)</div></div></div></div><div content-id="DSU002C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Muse</div> <div class="given-names">S.</div></div>. </div><div class="article-title">PowerMarker: an integrated analysis environment for genetic marker analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2128</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PowerMarker%3A%20an%20integrated%20analysis%20environment%20for%20genetic%20marker%20analysis&amp;author=K.%20Liu&amp;author=S.%20Muse&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2128-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15705655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Goddard</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Visscher</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">GCTA: a tool for genome-wide complex trait analysis</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">76</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GCTA%3A%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis&amp;author=J.%20Yang&amp;author=S.H.%20Lee&amp;author=M.E.%20Goddard&amp;author=P.M.%20Visscher&amp;publication_year=2011&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=88&amp;pages=76-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajhg.2010.11.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2010.11.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajhg.2010.11.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21167468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">945</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Evanno</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Regnaut</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Goudet</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Detecting the number of clusters of individuals using the software structure: a simulation study</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2611</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20structure%3A%20a%20simulation%20study&amp;author=G.%20Evanno&amp;author=S.%20Regnaut&amp;author=J.%20Goudet&amp;publication_year=2005&amp;journal=Mol.%20Ecol.&amp;volume=14&amp;pages=2611-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2005.14.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2005.14.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2005.14.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jakobsson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rosenberg</div> <div class="given-names">N.A.</div></div>. </div><div class="article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1801</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUMPP%3A%20a%20cluster%20matching%20and%20permutation%20program%20for%20dealing%20with%20label%20switching%20and%20multimodality%20in%20analysis%20of%20population%20structure&amp;author=M.%20Jakobsson&amp;author=N.A.%20Rosenberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1801-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosenberg</div> <div class="given-names">N.</div></div>. </div><div class="article-title">DISTRUCT: a program for the graphical display of population structure</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2004</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">137</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DISTRUCT%3A%20a%20program%20for%20the%20graphical%20display%20of%20population%20structure&amp;author=N.%20Rosenberg&amp;publication_year=2004&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=4&amp;pages=137-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2003.00566.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2003.00566.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2003.00566.x"> </span></div> </div></div></div></div></div><div content-id="DSU002C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Takezaki</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Estimation of genetic distances and phylogenetic trees from DNA analysis</div>, <div class="year">1983</div><div class="conf-name">Proceedings of the Fifth World Congress on Genetics Applied to Livestock Production</div><div class="publisher-loc">Guelph, Ontario, Canada</div><div class="publisher-name">University of Guelph</div>(pg. <div class="fpage">405</div>-<div class="lpage">12</div>)</div></div></div></div><div content-id="DSU002C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardy</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Vekemans</div> <div class="given-names">X.</div></div>. </div><div class="article-title">SPAGeDi: a versatile computer program to analyse spatial genetic structure at the individual or population levels</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">618</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPAGeDi%3A%20a%20versatile%20computer%20program%20to%20analyse%20spatial%20genetic%20structure%20at%20the%20individual%20or%20population%20levels&amp;author=O.%20Hardy&amp;author=X.%20Vekemans&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=2&amp;pages=618-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00305.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00305.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00305.x"> </span></div> </div></div></div></div></div><div content-id="DSU002C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Briggs</div> <div class="given-names">W.H.</div></div>, et al. </div><div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">203</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kroon</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2633</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.E.%20Kroon&amp;author=T.M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genetic analysis and characterization of a new maize association mapping panel for quantitative trait loci dissection</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">417</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20and%20characterization%20of%20a%20new%20maize%20association%20mapping%20panel%20for%20quantitative%20trait%20loci%20dissection&amp;author=X.%20Yang&amp;author=J.%20Yan&amp;author=T.%20Shah&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=417-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1320-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1320-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1320-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20349034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. R. Stat. Soc. B</div> , <div class="year">1995</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DSU002C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olsen</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Hayes</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Kent</div> <div class="given-names">M.P.</div></div>, et al. </div><div class="article-title">Genome-wide association mapping in Norwegian Red cattle identifies quantitative trait loci for fertility and milk production on BTA12</div>, <div class="source">Anim. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">466</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20in%20Norwegian%20Red%20cattle%20identifies%20quantitative%20trait%20loci%20for%20fertility%20and%20milk%20production%20on%20BTA12&amp;author=H.G.%20Olsen&amp;author=B.J.%20Hayes&amp;author=M.P.%20Kent&amp;publication_year=2011&amp;journal=Anim.%20Genet.&amp;volume=42&amp;pages=466-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2011.42.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2011.42.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2011.42.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21906098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Francki</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Forster</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Identification, characterization and interpretation of single-nucleotide sequence variation in allopolyploid crop species</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">125</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%2C%20characterization%20and%20interpretation%20of%20single-nucleotide%20sequence%20variation%20in%20allopolyploid%20crop%20species&amp;author=S.%20Kaur&amp;author=M.G.%20Francki&amp;author=J.W.%20Forster&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=125-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.10.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.10.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.10.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21831136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ganal</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Polley</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A large maize (<em>Zea mays</em> L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome</div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e28334</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large%20maize%20(Zea%20mays%20L.)%20SNP%20genotyping%20array%3A%20development%20and%20germplasm%20genotyping%2C%20and%20genetic%20mapping%20to%20compare%20with%20the%20B73%20reference%20genome&amp;author=M.W.%20Ganal&amp;author=G.%20Durstewitz&amp;author=A.%20Polley&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e28334" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0028334" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0028334" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0028334"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wlllte</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Skolinck</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">314</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=R.L.%20Wlllte&amp;author=M.%20Skolinck&amp;publication_year=1980&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=32&amp;pages=314-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6247908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qian</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Introgression of genomic components from Chinese <em>Brassica rapa</em> contributes to widening the genetic diversity in rapeseed (<em>B. napus</em> L.), with emphasis on the evolution of Chinese rapeseed</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">49</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introgression%20of%20genomic%20components%20from%20Chinese%20Brassica%20rapa%20contributes%20to%20widening%20the%20genetic%20diversity%20in%20rapeseed%20(B.%20napus%20L.)%2C%20with%20emphasis%20on%20the%20evolution%20of%20Chinese%20rapeseed&amp;author=W.%20Qian&amp;author=J.%20Meng&amp;author=M.%20Li&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=49-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0269-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0269-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0269-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16604336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiga</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Rape breeding by interspecific crossing between <em>Brassica napus</em> and <em>Brassica campestris</em> in Japan</div>, <div class="source">Japan Agric. Res. Quart.</div> , <div class="year">1970</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">5</div>-<div class="lpage">10</div>)</div></div></div></div><div content-id="DSU002C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rahman</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">R.-C.</div></div>,  <div class="name"><div class="surname">Kebede</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Thiagarajah</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Exploitation of the late flowering species <em>Brassica oleracea</em> L. for the improvement of earliness in <em>B. napus</em> L.: an untraditional approach</div>, <div class="source">Euphytica</div> , <div class="year">2011</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">365</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploitation%20of%20the%20late%20flowering%20species%20Brassica%20oleracea%20L.%20for%20the%20improvement%20of%20earliness%20in%20B.%20napus%20L.%3A%20an%20untraditional%20approach&amp;author=M.H.%20Rahman&amp;author=R.%20Bennett&amp;author=R.-C.%20Yang&amp;author=B.%20Kebede&amp;author=M.%20Thiagarajah&amp;publication_year=2011&amp;journal=Euphytica&amp;volume=177&amp;pages=365-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0253-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0253-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0253-5"> </span></div> </div></div></div></div></div><div content-id="DSU002C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flint-Garcia</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Thornsberry</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div>IV</div>. </div><div class="article-title">Structure of linkage disequilibrium in plants</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">357</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20linkage%20disequilibrium%20in%20plants&amp;author=S.A.%20Flint-Garcia&amp;author=J.M.%20Thornsberry&amp;author=E.S.%20Buckler&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=54&amp;pages=357-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.54.031902.134907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.54.031902.134907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.54.031902.134907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14502995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diers</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Genetic diversity of oilseed <em>Brassica napus</em> germplasm based on restriction fragment length polymorphisms</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">662</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20oilseed%20Brassica%20napus%20germplasm%20based%20on%20restriction%20fragment%20length%20polymorphisms&amp;author=B.%20Diers&amp;author=T.%20Osborn&amp;publication_year=1994&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=88&amp;pages=662-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01253968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01253968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01253968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24186160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Seyis</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Badani</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Analysis of genetic diversity in the <em>Brassica napus</em> L. Gene pool using SSR markers</div>, <div class="source">Genet. Res. Crop Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">793</div>-<div class="lpage">802</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20genetic%20diversity%20in%20the%20Brassica%20napus%20L.%20Gene%20pool%20using%20SSR%20markers&amp;author=M.%20Hasan&amp;author=F.%20Seyis&amp;author=A.%20Badani&amp;publication_year=2006&amp;journal=Genet.%20Res.%20Crop%20Evol.&amp;volume=53&amp;pages=793-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-004-5541-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-004-5541-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-004-5541-2"> </span></div> </div></div></div></div></div><div content-id="DSU002C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qian</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zou</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Intersubgenomic heterosis in seed yield potential observed in a new type of <em>Brassica napus</em> introgressed with partial <em>Brassica rapa</em> genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1187</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intersubgenomic%20heterosis%20in%20seed%20yield%20potential%20observed%20in%20a%20new%20type%20of%20Brassica%20napus%20introgressed%20with%20partial%20Brassica%20rapa%20genome&amp;author=W.%20Qian&amp;author=X.%20Chen&amp;author=D.%20Fu&amp;author=J.%20Zou&amp;author=J.%20Meng&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1187-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1932-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1932-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1932-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hammerli</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Reusch</div> <div class="given-names">T.B.</div></div>. </div><div class="article-title">Genetic neighbourhood of clone structures in eelgrass meadows quantified by spatial autocorrelation of microsatellite markers</div>, <div class="source">Heredity (Edinb)</div> , <div class="year">2003</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">448</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20neighbourhood%20of%20clone%20structures%20in%20eelgrass%20meadows%20quantified%20by%20spatial%20autocorrelation%20of%20microsatellite%20markers&amp;author=A.%20Hammerli&amp;author=T.B.%20Reusch&amp;publication_year=2003&amp;journal=Heredity%20(Edinb)&amp;volume=91&amp;pages=448-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14576737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Friedt</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Pons-Kuhnemann</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Freitag</div> <div class="given-names">N.M.</div></div>,  <div class="name"><div class="surname">Link</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Association of gene-linked SSR markers to seed glucosinolate content in oilseed rape (<em>Brassica napus</em> ssp. <em>napus</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">1035</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20gene-linked%20SSR%20markers%20to%20seed%20glucosinolate%20content%20in%20oilseed%20rape%20(Brassica%20napus%20ssp.%20napus)&amp;author=M.%20Hasan&amp;author=W.%20Friedt&amp;author=J.%20Pons-Kuhnemann&amp;author=N.M.%20Freitag&amp;author=K.%20Link&amp;author=R.J.%20Snowdon&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=1035-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0733-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0733-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0733-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18322671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Patterson</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Plenge</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Weinblatt</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Shadick</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Reich</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Principal components analysis corrects for stratification in genome-wide association studies</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">904</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&amp;author=A.L.%20Price&amp;author=N.J.%20Patterson&amp;author=R.M.%20Plenge&amp;author=M.E.%20Weinblatt&amp;author=N.A.%20Shadick&amp;author=D.%20Reich&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=904-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16862161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Characterization of a global germplasm collection and its potential utilization for analysis of complex quantitative traits in maize</div>, <div class="source">Mol. Breed.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">511</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20global%20germplasm%20collection%20and%20its%20potential%20utilization%20for%20analysis%20of%20complex%20quantitative%20traits%20in%20maize&amp;author=X.%20Yang&amp;author=S.%20Gao&amp;author=S.%20Xu&amp;publication_year=2011&amp;journal=Mol.%20Breed.&amp;volume=28&amp;pages=511-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9500-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9500-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9500-7"> </span></div> </div></div></div></div></div><div content-id="DSU002C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Upadhyaya</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.-H.</div></div>,  <div class="name"><div class="surname">Gowda</div> <div class="given-names">C.L.L.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Association mapping of maturity and plant height using SNP markers with the sorghum mini core collection</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">2003</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20maturity%20and%20plant%20height%20using%20SNP%20markers%20with%20the%20sorghum%20mini%20core%20collection&amp;author=H.%20Upadhyaya&amp;author=Y.-H.%20Wang&amp;author=C.L.L.%20Gowda&amp;author=S.%20Sharma&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=2003-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2113-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2113-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2113-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23649651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pasam</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Malosetti</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide association studies for agronomical traits in a world wide spring barley collection</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">16</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20for%20agronomical%20traits%20in%20a%20world%20wide%20spring%20barley%20collection&amp;author=R.K.%20Pasam&amp;author=R.%20Sharma&amp;author=M.%20Malosetti&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22284310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Aranzana</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">An Arabidopsis example of association mapping in structured samples</div>, <div class="source">PLoS Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e4</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20Arabidopsis%20example%20of%20association%20mapping%20in%20structured%20samples&amp;author=K.%20Zhao&amp;author=M.J.%20Aranzana&amp;author=S.%20Kim&amp;publication_year=2007&amp;journal=PLoS%20Genet.&amp;volume=3&amp;pages=e4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17238287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">A functional genomics resource for <em>Brassica napus</em>: development of an EMS mutagenized population and discovery of FAE1 point mutations by TILLING</div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">751</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20functional%20genomics%20resource%20for%20Brassica%20napus%3A%20development%20of%20an%20EMS%20mutagenized%20population%20and%20discovery%20of%20FAE1%20point%20mutations%20by%20TILLING&amp;author=N.%20Wang&amp;author=Y.%20Wang&amp;author=F.%20Tian&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=180&amp;pages=751-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.180.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.180.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.180.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18811617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Zero erucic acid trait of rapeseed (<em>Brassica napus</em> L.) results from a deletion of four base pairs in the fatty acid elongase 1 gene</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">491</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zero%20erucic%20acid%20trait%20of%20rapeseed%20(Brassica%20napus%20L.)%20results%20from%20a%20deletion%20of%20four%20base%20pairs%20in%20the%20fatty%20acid%20elongase%201%20gene&amp;author=G.%20Wu&amp;author=Y.%20Wu&amp;author=L.%20Xiao&amp;author=X.%20Li&amp;author=C.%20Lu&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=491-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0685-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0685-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0685-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18075728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Howell</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Homoeologous loci control the accumulation of seed glucosinolates in oilseed rape (<em>Brassica napus</em>)</div>, <div class="source">Genome</div> , <div class="year">2003</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">454</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homoeologous%20loci%20control%20the%20accumulation%20of%20seed%20glucosinolates%20in%20oilseed%20rape%20(Brassica%20napus)&amp;author=P.%20Howell&amp;author=A.%20Sharpe&amp;author=D.%20Lydiate&amp;publication_year=2003&amp;journal=Genome&amp;volume=46&amp;pages=454-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12834062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Detection of loci controlling seed glucosinolate content and their association with <em>Sclerotinia</em> resistance in <em>Brassica napus</em></div>, <div class="source">Plant Breed.</div> , <div class="year">2003</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">19</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20loci%20controlling%20seed%20glucosinolate%20content%20and%20their%20association%20with%20Sclerotinia%20resistance%20in%20Brassica%20napus&amp;author=J.%20Zhao&amp;author=J.%20Meng&amp;publication_year=2003&amp;journal=Plant%20Breed.&amp;volume=122&amp;pages=19-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.2003.00784.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.2003.00784.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.2003.00784.x"> </span></div> </div></div></div></div></div><div content-id="DSU002C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirai</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Sugiyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">6478</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omics-based%20identification%20of%20Arabidopsis%20Myb%20transcription%20factors%20regulating%20aliphatic%20glucosinolate%20biosynthesis&amp;author=M.Y.%20Hirai&amp;author=K.%20Sugiyama&amp;author=Y.%20Sawada&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=6478-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611629104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611629104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611629104"> </span></div> </div></div></div></div></div><div content-id="DSU002C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Augustine</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Mukhopadhyay</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bisht</div> <div class="given-names">N. C.</div></div>. </div><div class="article-title">Targeted silencing of BjMYB28 transcription factor gene directs development of low glucosinolate lines in oilseed <em>Brassica juncea</em></div>, <div class="source">Plant Biotechnol. J</div> , <div class="year">2013</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">855</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20silencing%20of%20BjMYB28%20transcription%20factor%20gene%20directs%20development%20of%20low%20glucosinolate%20lines%20in%20oilseed%20Brassica%20juncea&amp;author=R.%20Augustine&amp;author=A.%20Mukhopadhyay&amp;author=N.%20C.%20Bisht&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J&amp;volume=11&amp;pages=855-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2013.11.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2013.11.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2013.11.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23721233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burns</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Barnes</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Bowman</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Clarke</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Werner</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Kearsey</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">QTL analysis of an intervarietal set of substitution lines in <em>Brassica napus</em>: (i) seed oil content and fatty acid composition</div>, <div class="source">Heredity (Edinb)</div> , <div class="year">2003</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">39</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20an%20intervarietal%20set%20of%20substitution%20lines%20in%20Brassica%20napus%3A%20(i)%20seed%20oil%20content%20and%20fatty%20acid%20composition&amp;author=M.J.%20Burns&amp;author=S.R.%20Barnes&amp;author=J.G.%20Bowman&amp;author=M.H.%20Clarke&amp;author=C.P.%20Werner&amp;author=M.J.%20Kearsey&amp;publication_year=2003&amp;journal=Heredity%20(Edinb)&amp;volume=90&amp;pages=39-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6800176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6800176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12522424" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Uzunova</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Weißleder</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Mapping the genome of rapeseed (<em>Brassica napus</em> L.). II. Localization of genes controlling erucic acid synthesis and seed oil content</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">972</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20the%20genome%20of%20rapeseed%20(Brassica%20napus%20L.).%20II.%20Localization%20of%20genes%20controlling%20erucic%20acid%20synthesis%20and%20seed%20oil%20content&amp;author=W.%20Ecke&amp;author=M.%20Uzunova&amp;author=K.%20Wei%C3%9Fleder&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=972-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Morgan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A comparative linkage map of oilseed rape and its use for QTL analysis of seed oil and erucic acid content</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">67</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20linkage%20map%20of%20oilseed%20rape%20and%20its%20use%20for%20QTL%20analysis%20of%20seed%20oil%20and%20erucic%20acid%20content&amp;author=D.%20Qiu&amp;author=C.%20Morgan&amp;author=J.%20Shi&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=67-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0411-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0411-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0411-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Basunanda</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Radoev</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Friedt</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Comparative mapping of quantitative trait loci involved in heterosis for seedling and yield traits in oilseed rape (<em>Brassica napus</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">271</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20of%20quantitative%20trait%20loci%20involved%20in%20heterosis%20for%20seedling%20and%20yield%20traits%20in%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=P.%20Basunanda&amp;author=M.%20Radoev&amp;author=W.%20Ecke&amp;author=W.%20Friedt&amp;author=H.%20Becker&amp;author=R.%20Snowdon&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1133-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1133-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1133-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19707740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Qin</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Mapping of quantitative trait loci and development of allele-specific markers for seed weight in <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">1289</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20quantitative%20trait%20loci%20and%20development%20of%20allele-specific%20markers%20for%20seed%20weight%20in%20Brassica%20napus&amp;author=C.%20Fan&amp;author=G.%20Cai&amp;author=J.%20Qin&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=1289-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1388-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1388-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1388-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20574694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quijada</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Udall</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (<em>Brassica napus</em> L.): 1. Identification of genomic regions from winter germplasm</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">549</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20analysis%20of%20seed%20yield%20and%20other%20complex%20traits%20in%20hybrid%20spring%20rapeseed%20(Brassica%20napus%20L.)%3A%201.%20Identification%20of%20genomic%20regions%20from%20winter%20germplasm&amp;author=P.%20Quijada&amp;author=J.%20Udall&amp;author=B.%20Lambert&amp;author=T.%20Osborn&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=549-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0323-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0323-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0323-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16767447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">Genetic and correlation analysis of silique-traits in <em>Brassica napus</em> L. by quantitative trait locus mapping</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">21</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20correlation%20analysis%20of%20silique-traits%20in%20Brassica%20napus%20L.%20by%20quantitative%20trait%20locus%20mapping&amp;author=L.%20Zhang&amp;author=G.%20Yang&amp;author=P.%20Liu&amp;author=D.%20Hong&amp;author=S.%20Li&amp;author=Q.%20He&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=21-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1419-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1419-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1419-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20686746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU002C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU002C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hao</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Genome-wide association study identifies candidate genes that affect plant height in Chinese elite maize (<em>Zea mays</em> L.) inbred lines</div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e29229</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20identifies%20candidate%20genes%20that%20affect%20plant%20height%20in%20Chinese%20elite%20maize%20(Zea%20mays%20L.)%20inbred%20lines&amp;author=J.%20Weng&amp;author=C.%20Xie&amp;author=Z.%20Hao&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e29229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22216221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=445136&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables6.xls?Expires=2147483647&amp;Signature=ufMBywU4lRu1p6wzt7BhcBJFys7WjMdrDzBWdQBelHalKEnUEah4JDopgRMZiut3io-aThof9h8oujadRF8diY4upV0NlY41MjpcDA-LRNP-4tzsNhMCCctrNYl5q1w7wbNdUmYTCgoygKfEaBfarIw8Q3io0Yvn4IE4OdEsANhrcBfyiNZw55lztQCpyN9FNuv~bLc~dOmoEOvkrn9-xf9ZSIkg3KGs-GAyYylnxpHK6RMcedrftoUKY98zVvfRqBfMrP627hm-UDElUA46qfn8lwzts31g0VSonSpTmOzFDUtH2gag7Z4jduKCCozt-rogR2KnOVSz0u0fwNKVsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables5.xls?Expires=2147483647&amp;Signature=WrO3leZHj4a8Mn5OIykdWlxJktIzcKjfKRWN8B9cQ0N9s5PXvfzVkbMVstotja-AwRinMIGv9dF-YxqVfvloRGA4YDeA2Q2eRisn6zPRvY6h8Cbnw-Igv~N0MEU-KcdNb3~W-4Zx-1xCrqP0Y1re~FgCQUSRds6B3FNUSG36VDhS5BAXCRKcZsZ7Pk0S5MMLF-O7wpRSIU~N9jhcYenwFhBBrcxlv9QH17s~iNY4pgpsY7lg8ibFC4aJQFd0z7ULxcp7czt1ngDbkqgLvxpWTZbob0O3bnATHzN5BL5yofdtTt0BypYIXBr5qhjgMbgkolGyYiqpGUnoAr5407b5-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables4.xls?Expires=2147483647&amp;Signature=c2DtcZJFtwOF7zFtjgRujn1mM4so-KzPoE87LCRptDu4KjYvD1IXiBe9jHSv7IkJiGc9RTtvw6Bn0w-0-Sa6k97e38PnKt~IhdKJD0HXGZ5und6kv3P9t3YK4RFSEcOrvJbHBXz--qZKHM1TyjfI7ZJ03oGlT-ILSrq972iNhWYOPS2uZcJEUWAd2HXnHOEodJ~EBDJQV~QSbuM0ftFOCUBW2mA1-JJq~M1mrtdVjMDg4qWO9UmY20cJGywGFZJMv~1obiylCWkhQhkwYUwvACAoeQNgb3pqBJrnw-YUSIDdjyNaZoqGm-aUPYe~vQvqcZnTfP5yY7TqxmMrPNuDaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables3.xlsx?Expires=2147483647&amp;Signature=IaV7PvxT6if~S1vS-VB8-NJVuBC38sTlzVuO0Re9zHAKO-GH~IDHL09hOmbhH7sAesWYVM0O6a5DTOiH7RwYqOvSVlwQZEWgo3VxN~BuXpWqSPbGKYfet7pSxpcTBs-jh7BpIuhBADP-ZhzfMFVdxBc~VosfcMzBqWsEIBCUoqTOoLPz0L3T2NfMdgNWljuWSoBCN6UtH7QxlGrRus4O4ItqJCPziYdatKYtmachHan0ltUO9P4YGeNy~hGWcQOd0vt~MHle-jgzDwQVqRVMTrkM55KIeH01OHDa7p14bKxQ~2oR9LbLzF-eNpKgWCnJ6DtJai2ahlqnQgeni36qRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables2.xls?Expires=2147483647&amp;Signature=3eJMHwCa5DS3PBVz97MMNoqjdHxdgcRQl2538lKzg2g-tw7Ydikdl8R1Dn1eGx1hBaMMgPKYbhXYdaBCYPirW3tnBpVPsIh0fhnwySZ2--owNOLoZuNjrr~NxllAt8wRPK14Rg03XQ5K1wEpg9zAUQoBOHejQYwQmkQvLB1sO9V9nQS-PDzbzY5U9dSrGLjHW8GvBhgDRuMKsP7jkKMqlzRP49cPz4TJLnEX-zjFQjv0HUs4QrMnsuzgTsOrab3cNtLbOFxJL4e3qEuZnOzgYMNPKVpkSsy9u5YZjhQjzSoNqB1hyMbKAsbDhT~ADMXFfMeM~AgnoTWpBGMrMjyInQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_tables1.xls?Expires=2147483647&amp;Signature=WK9akc6tfLVP9XfNASKW5SSYGRoryzc05dLs7sRUt~PaAh~IFekozyK81hNJot~KPWQZf0garr7CNWstYJa~WZth0aSoTsy7M5W0USrYxiD4QDvNPFX5DlecccOZlVwfTPqoAcO2CqXMhZWYcdyQVxfcyzc0ICOgliE0dooMaGRciZUe~zRhMr69SO2Hnum9gq6U6QKy6jUUKtInKT3EbRpfikltEZVPqMsjB~hyijYUDQksAhl1aMO1oXCj228lYnszMaIHgCIjsv5weulFOMMLHb1tIdeGjgb8377D2MAny7i7UsV3JMEKHLtZxu~~zjkLeGf8ZL7D9qxN7VAnYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu002/2/dsu002-dsu002supp_data.doc?Expires=2147483647&amp;Signature=IiHuAmIhqdhUbjW01YwIgiYEqSpyNjhs65K7-11V1dT624lvgeqEHoHxx2-WtbaGYRWQxoNcrVnHIXl8byY-l4BUHjIBuk2Liadv~P8jKWV8Jfrp2iUe21JSKBVqo5fSkKwG4Pn5Q35UQxEwVvmMHSlMzZ9SGi5a1tJkLmZKiuEzZCJqeg2HxkKO7mdkYqda~2lyPprQywi8Iq0qaRNcBsVx59sjnQwIgvDHHWWTJAkJFrCbStDuX69Hw6DTWAhnbjh0QPP4P26p8NyvyjI-4o10VOToDDG4Enmg3FkXnbz0U3XfjKQj6-tz5zbeE0sVfgdudP0zJie5WE--tzes5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,066</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">76</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="445136"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f355%2f445136">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (76)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F355%2F445136" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/445136'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Association Study Dissects the Genetic Architecture of Seed Weight and Seed Quality in Rapeseed (Brassica napus L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f355%2f445136">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

