

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/5/467/2236168","@type":"ScholarlyArticle","name":"An automated system for evaluation of the potential functionome: MAPLE version 2.1.0","datePublished":"2016-07-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2016-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw030","keywords":["metagenome","MAPLE","metabolic pathway","global ocean sampling (GOS)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/5/467/2236168","author":[{"name":"Takami, Hideto","affiliation":"Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan","@type":"Person"},{"name":"Taniguchi, Takeaki","affiliation":"Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo 152-8550, Japan","@type":"Person"},{"name":"Arai, Wataru","affiliation":"Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan","@type":"Person"},{"name":"Takemoto, Kazuhiro","affiliation":"Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan","@type":"Person"},{"name":"Moriya, Yuki","affiliation":"Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan","@type":"Person"},{"name":"Goto, Susumu","affiliation":"Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan","@type":"Person"}],"description":"Abstract.  Metabolic and physiological potential evaluator (MAPLE) is an automatic system that can perform a series of steps used in the evaluation of potential","pageStart":"467","pageEnd":"475","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An automated system for evaluation of the potential functionome: MAPLE version 2.1.0","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="automated system for evaluation of the potential functionome: MAPLE version 2.1.0" />
    <meta property="og:description" content="Abstract.  Metabolic and physiological potential evaluator (MAPLE) is an automatic system that can perform a series of steps used in the evaluation of potential" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/5/467/2236168" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takami, Hideto" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan" /><meta name="citation_author" content="Taniguchi, Takeaki" /><meta name="citation_author_institution" content="Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Arai, Wataru" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan" /><meta name="citation_author" content="Takemoto, Kazuhiro" /><meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan" /><meta name="citation_author" content="Moriya, Yuki" /><meta name="citation_author_institution" content="Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan" /><meta name="citation_author" content="Goto, Susumu" /><meta name="citation_author_institution" content="Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan" /><meta name="citation_title" content="An automated system for evaluation of the potential functionome: MAPLE version 2.1.0" /><meta name="citation_firstpage" content="467" /><meta name="citation_lastpage" content="475" /><meta name="citation_doi" content="10.1093/dnares/dsw030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2016/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes,; citation_author=Kanehisa  M.; citation_author=Goto  S.;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes; citation_author=Overbeek  R.; citation_author=Begley  T.; citation_author=Butler  R.M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=5691-702; " /><meta name="citation_reference" content="citation_title=KAAS: an automatic genome annotation and pathway reconstruction server; citation_author=Moriya  Y.; citation_author=Itoh  M.; citation_author=Okuda  S.; citation_author=Yoshizawa  A.; citation_author=Kanehisa  M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W182-5; " /><meta name="citation_reference" content="citation_title=The metagenomics RAST server–a public resource for the automatic phylogenetic and functional analysis of metagenomes; citation_author=Meyer  F.; citation_author=Paarmann  D.; citation_author=D’Souza  M.;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=386" /><meta name="citation_reference" content="citation_title=Integrative analysis of environmental sequences using MEGAN4; citation_author=Huson  D.H.; citation_author=Mitra  S.; citation_author=Ruscheweyh  H.J.; citation_author=Weber  N.; citation_author=Schuster  S.C.;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=1552-60; " /><meta name="citation_reference" content="citation_title=Bioinformatic approaches for functional annotation and pathway inference in metagenomic data; citation_author=Filippo  C.D.; citation_author=Ramazzotti  M.; citation_author=Fontana  R.; citation_author=Cavalieri  D.;  citation_journal_title=Brief Bioinform;  citation_year=2012;  citation_volume=13;  citation_pages=696-710; " /><meta name="citation_reference" content="citation_title=KEGG for linking genomes to life and environment; citation_author=Kanehisa  M.; citation_author=Araki  M.; citation_author=Goto  S.;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=D480-4; " /><meta name="citation_reference" content="citation_title=Evaluation method for the potential functionome harbored in the genome and metagenome; citation_author=Takami  H.; citation_author=Taniguchi  T.; citation_author=Moriya  Y.; citation_author=Kuwahara  T.; citation_author=Kanehisa  M.; citation_author=Goto  S.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=699" /><meta name="citation_reference" content="citation_author=Takami  H.; citation_author=K.  Nelson; citation_publisher=Springer-Verlag, Berlin, Heidelberg, Germany;  citation_title=Encyclopedia of Metagenomics;  citation_year=2014;  citation_pages=525-39; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; citation_author=Venter  J.C; citation_author=Remington  K.; citation_author=Heidelberg  J.F.;  citation_journal_title=Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=Assessing diversity and biogeography of aerobic anoxygenic phototrophic bacteria in surface waters of the Atlantic and Pacific Oceans using the Global Ocean Sampling expedition metagenomes; citation_author=Yutin  N.; citation_author=Suzuki  M.T.; citation_author=Teeling  H.;  citation_journal_title=Environ. Microbiol;  citation_year=2007;  citation_volume=9;  citation_pages=1464-75; " /><meta name="citation_reference" content="citation_title=Effect of counterion behavior on the frictional-compressive properties of chondroitin sulfate solutions; citation_author=Baeurle  S.A.; citation_author=Kiselev  M.G.; citation_author=Makorava  E.S.; citation_author=Nogovitsin  E.A.;  citation_journal_title=Polymer;  citation_year=2009;  citation_volume=50;  citation_pages=1805-13; " /><meta name="citation_reference" content="citation_title=Subsistence strategies in traditional societies distinguish gut microbiomes; citation_author=Obregon-Tito  A.J.; citation_author=Tito  R.Y.; citation_author=Metcalf  J.;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=6505" /><meta name="citation_reference" content="citation_title=Genome sequence of the soil bacterium Janthinobacterium sp. KBS0711; citation_author=Shoemaker  W.R.; citation_author=Muscarella  M.E.; citation_author=Lennon  J.T.;  citation_journal_title=Genome Announc;  citation_year=2015;  citation_volume=3;  citation_pages=e00689-15; " /><meta name="citation_reference" content="citation_title=Functional classification of uncultured “Candidatus Caldiarchaeum subterraneum” using the MAPLE system; citation_author=Takami  H.; citation_author=Arai  W.; citation_author=Takemoto  K.; citation_author=Uchiyama  I.; citation_author=Taniguchi  T.;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0132994" /><meta name="citation_reference" content="citation_title=Metagenomics: DNA sequencing of environmental samples; citation_author=Tringe  S.G.; citation_author=Rubin  E.M.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2005;  citation_volume=6;  citation_pages=805-14; " /><meta name="citation_reference" content="citation_title=Short pyrosequencing reads suffice for accurate microbial community analysis; citation_author=Liu  Z.; citation_author=Lozupone  C.; citation_author=Hamady  M.; citation_author=Bushman  F.D.; citation_author=Knight  R.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=e120" /><meta name="citation_reference" content="citation_title=Accurate and universal delineation of prokaryotic species; citation_author=Mende  D.R.; citation_author=Sunagawa  S.; citation_author=Aeller  G.; citation_author=Bork  P.;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=881-4; " /><meta name="citation_reference" content="citation_title=A human gut microbial gene catalogue established by metagenomic sequencing; citation_author=Qin  J.; citation_author=Li  R.; citation_author=Raes  J.;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=59-65; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="description" content="Abstract.  Metabolic and physiological potential evaluator (MAPLE) is an automatic system that can perform a series of steps used in the evaluation of potential" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/5/467/2236168" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/5/467/2236168" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/5/467/2236168");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw030';
    </script>
<title>
	automated system for evaluation of the potential functionome: MAPLE version 2.1.0 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2236168" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYBGPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthbWksIEhpZGV0byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pY3JvYmlhbCBHZW5vbWUgUmVzZWFyY2ggR3JvdXAsIFlva29oYW1hIEluc3RpdHV0ZSwgSmFwYW4gQWdlbmN5IGZvciBNYXJpbmUtRWFydGggU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoSkFNU1RFQyksIFlva29oYW1hLCBLYW5hZ2F3YSAyMzYtMDAwMSBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmlndWNoaSwgVGFrZWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgTWVndXJvLWt1LCBUb2t5byAxNTItODU1MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmFpLCBXYXRhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWNyb2JpYWwgR2Vub21lIFJlc2VhcmNoIEdyb3VwLCBZb2tvaGFtYSBJbnN0aXR1dGUsIEphcGFuIEFnZW5jeSBmb3IgTWFyaW5lLUVhcnRoIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKEpBTVNURUMpLCBZb2tvaGFtYSwgS2FuYWdhd2EgMjM2LTAwMDEgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWtlbW90bywgS2F6dWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UgYW5kIEJpb2luZm9ybWF0aWNzLCBLeXVzaHUgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIElpenVrYSwgRnVrdW9rYSA4MjAtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpeWEsIFl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBDZW50ZXIsIEluc3RpdHV0ZSBmb3IgQ2hlbWljYWwgUmVzZWFyY2gsIEt5b3RvIFVuaXZlcnNpdHksIFVqaSwgS3lvdG8gNjExLTAwMTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR290bywgU3VzdW11IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgQ2VudGVyLCBJbnN0aXR1dGUgZm9yIENoZW1pY2FsIFJlc2VhcmNoLCBLeW90byBVbml2ZXJzaXR5LCBVamksIEt5b3RvIDYxMS0wMDExLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW4gYXV0b21hdGVkIHN5c3RlbSBmb3IgZXZhbHVhdGlvbiBvZiB0aGUgcG90ZW50aWFsIGZ1bmN0aW9ub21lOiBNQVBMRSB2ZXJzaW9uIDIuMS4wIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtFR0c6IEt5b3RvIEVuY3ljbG9wZWRpYSBvZiBHZW5lcyBhbmQgR2Vub21lcyw7IGNpdGF0aW9uX2F1dGhvcj1LYW5laGlzYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Hb3RvICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN1YnN5c3RlbXMgYXBwcm9hY2ggdG8gZ2Vub21lIGFubm90YXRpb24gYW5kIGl0cyB1c2UgaW4gdGhlIHByb2plY3QgdG8gYW5ub3RhdGUgMTAwMCBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9T3ZlcmJlZWsgIFIuOyBjaXRhdGlvbl9hdXRob3I9QmVnbGV5ICBULjsgY2l0YXRpb25fYXV0aG9yPUJ1dGxlciAgUi5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz01NjkxLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0FBUzogYW4gYXV0b21hdGljIGdlbm9tZSBhbm5vdGF0aW9uIGFuZCBwYXRod2F5IHJlY29uc3RydWN0aW9uIHNlcnZlcjsgY2l0YXRpb25fYXV0aG9yPU1vcml5YSAgWS47IGNpdGF0aW9uX2F1dGhvcj1JdG9oICBNLjsgY2l0YXRpb25fYXV0aG9yPU9rdWRhICBTLjsgY2l0YXRpb25fYXV0aG9yPVlvc2hpemF3YSAgQS47IGNpdGF0aW9uX2F1dGhvcj1LYW5laGlzYSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzE4Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWV0YWdlbm9taWNzIFJBU1Qgc2VydmVy4oCTYSBwdWJsaWMgcmVzb3VyY2UgZm9yIHRoZSBhdXRvbWF0aWMgcGh5bG9nZW5ldGljIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIG1ldGFnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9TWV5ZXIgIEYuOyBjaXRhdGlvbl9hdXRob3I9UGFhcm1hbm4gIEQuOyBjaXRhdGlvbl9hdXRob3I9ROKAmVNvdXphICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSBhbmFseXNpcyBvZiBlbnZpcm9ubWVudGFsIHNlcXVlbmNlcyB1c2luZyBNRUdBTjQ7IGNpdGF0aW9uX2F1dGhvcj1IdXNvbiAgRC5ILjsgY2l0YXRpb25fYXV0aG9yPU1pdHJhICBTLjsgY2l0YXRpb25fYXV0aG9yPVJ1c2NoZXdleWggIEguSi47IGNpdGF0aW9uX2F1dGhvcj1XZWJlciAgTi47IGNpdGF0aW9uX2F1dGhvcj1TY2h1c3RlciAgUy5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE1NTItNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWMgYXBwcm9hY2hlcyBmb3IgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIGFuZCBwYXRod2F5IGluZmVyZW5jZSBpbiBtZXRhZ2Vub21pYyBkYXRhOyBjaXRhdGlvbl9hdXRob3I9RmlsaXBwbyAgQy5ELjsgY2l0YXRpb25fYXV0aG9yPVJhbWF6em90dGkgIE0uOyBjaXRhdGlvbl9hdXRob3I9Rm9udGFuYSAgUi47IGNpdGF0aW9uX2F1dGhvcj1DYXZhbGllcmkgIEQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZiBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02OTYtNzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHIGZvciBsaW5raW5nIGdlbm9tZXMgdG8gbGlmZSBhbmQgZW52aXJvbm1lbnQ7IGNpdGF0aW9uX2F1dGhvcj1LYW5laGlzYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1BcmFraSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Hb3RvICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENDgwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gbWV0aG9kIGZvciB0aGUgcG90ZW50aWFsIGZ1bmN0aW9ub21lIGhhcmJvcmVkIGluIHRoZSBnZW5vbWUgYW5kIG1ldGFnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1UYWthbWkgIEguOyBjaXRhdGlvbl9hdXRob3I9VGFuaWd1Y2hpICBULjsgY2l0YXRpb25fYXV0aG9yPU1vcml5YSAgWS47IGNpdGF0aW9uX2F1dGhvcj1LdXdhaGFyYSAgVC47IGNpdGF0aW9uX2F1dGhvcj1LYW5laGlzYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Hb3RvICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9Njk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPVRha2FtaSAgSC47IGNpdGF0aW9uX2F1dGhvcj1LLiAgTmVsc29uOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW4sIEhlaWRlbGJlcmcsIEdlcm1hbnk7ICBjaXRhdGlvbl90aXRsZT1FbmN5Y2xvcGVkaWEgb2YgTWV0YWdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fcGFnZXM9NTI1LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIGdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmcgb2YgdGhlIFNhcmdhc3NvIFNlYTsgY2l0YXRpb25fYXV0aG9yPVZlbnRlciAgSi5DOyBjaXRhdGlvbl9hdXRob3I9UmVtaW5ndG9uICBLLjsgY2l0YXRpb25fYXV0aG9yPUhlaWRlbGJlcmcgIEouRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA0OyAgY2l0YXRpb25fcGFnZXM9NjYtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyBkaXZlcnNpdHkgYW5kIGJpb2dlb2dyYXBoeSBvZiBhZXJvYmljIGFub3h5Z2VuaWMgcGhvdG90cm9waGljIGJhY3RlcmlhIGluIHN1cmZhY2Ugd2F0ZXJzIG9mIHRoZSBBdGxhbnRpYyBhbmQgUGFjaWZpYyBPY2VhbnMgdXNpbmcgdGhlIEdsb2JhbCBPY2VhbiBTYW1wbGluZyBleHBlZGl0aW9uIG1ldGFnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9WXV0aW4gIE4uOyBjaXRhdGlvbl9hdXRob3I9U3V6dWtpICBNLlQuOyBjaXRhdGlvbl9hdXRob3I9VGVlbGluZyAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTQ2NC03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIGNvdW50ZXJpb24gYmVoYXZpb3Igb24gdGhlIGZyaWN0aW9uYWwtY29tcHJlc3NpdmUgcHJvcGVydGllcyBvZiBjaG9uZHJvaXRpbiBzdWxmYXRlIHNvbHV0aW9uczsgY2l0YXRpb25fYXV0aG9yPUJhZXVybGUgIFMuQS47IGNpdGF0aW9uX2F1dGhvcj1LaXNlbGV2ICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9TWFrb3JhdmEgIEUuUy47IGNpdGF0aW9uX2F1dGhvcj1Ob2dvdml0c2luICBFLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qb2x5bWVyOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MTgwNS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vic2lzdGVuY2Ugc3RyYXRlZ2llcyBpbiB0cmFkaXRpb25hbCBzb2NpZXRpZXMgZGlzdGluZ3Vpc2ggZ3V0IG1pY3JvYmlvbWVzOyBjaXRhdGlvbl9hdXRob3I9T2JyZWdvbi1UaXRvICBBLkouOyBjaXRhdGlvbl9hdXRob3I9VGl0byAgUi5ZLjsgY2l0YXRpb25fYXV0aG9yPU1ldGNhbGYgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjUwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgc29pbCBiYWN0ZXJpdW0gSmFudGhpbm9iYWN0ZXJpdW0gc3AuIEtCUzA3MTE7IGNpdGF0aW9uX2F1dGhvcj1TaG9lbWFrZXIgIFcuUi47IGNpdGF0aW9uX2F1dGhvcj1NdXNjYXJlbGxhICBNLkUuOyBjaXRhdGlvbl9hdXRob3I9TGVubm9uICBKLlQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQW5ub3VuYzsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTAwNjg5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGNsYXNzaWZpY2F0aW9uIG9mIHVuY3VsdHVyZWQg4oCcQ2FuZGlkYXR1cyBDYWxkaWFyY2hhZXVtIHN1YnRlcnJhbmV1beKAnSB1c2luZyB0aGUgTUFQTEUgc3lzdGVtOyBjaXRhdGlvbl9hdXRob3I9VGFrYW1pICBILjsgY2l0YXRpb25fYXV0aG9yPUFyYWkgIFcuOyBjaXRhdGlvbl9hdXRob3I9VGFrZW1vdG8gIEsuOyBjaXRhdGlvbl9hdXRob3I9VWNoaXlhbWEgIEkuOyBjaXRhdGlvbl9hdXRob3I9VGFuaWd1Y2hpICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDEzMjk5NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljczogRE5BIHNlcXVlbmNpbmcgb2YgZW52aXJvbm1lbnRhbCBzYW1wbGVzOyBjaXRhdGlvbl9hdXRob3I9VHJpbmdlICBTLkcuOyBjaXRhdGlvbl9hdXRob3I9UnViaW4gIEUuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODA1LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9ydCBweXJvc2VxdWVuY2luZyByZWFkcyBzdWZmaWNlIGZvciBhY2N1cmF0ZSBtaWNyb2JpYWwgY29tbXVuaXR5IGFuYWx5c2lzOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBaLjsgY2l0YXRpb25fYXV0aG9yPUxvenVwb25lICBDLjsgY2l0YXRpb25fYXV0aG9yPUhhbWFkeSAgTS47IGNpdGF0aW9uX2F1dGhvcj1CdXNobWFuICBGLkQuOyBjaXRhdGlvbl9hdXRob3I9S25pZ2h0ICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1lMTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgYW5kIHVuaXZlcnNhbCBkZWxpbmVhdGlvbiBvZiBwcm9rYXJ5b3RpYyBzcGVjaWVzOyBjaXRhdGlvbl9hdXRob3I9TWVuZGUgIEQuUi47IGNpdGF0aW9uX2F1dGhvcj1TdW5hZ2F3YSAgUy47IGNpdGF0aW9uX2F1dGhvcj1BZWxsZXIgIEcuOyBjaXRhdGlvbl9hdXRob3I9Qm9yayAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTg4MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGh1bWFuIGd1dCBtaWNyb2JpYWwgZ2VuZSBjYXRhbG9ndWUgZXN0YWJsaXNoZWQgYnkgbWV0YWdlbm9taWMgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPVFpbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1MaSAgUi47IGNpdGF0aW9uX2F1dGhvcj1SYWVzICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBNZXRhYm9saWMgYW5kIHBoeXNpb2xvZ2ljYWwgcG90ZW50aWFsIGV2YWx1YXRvciAoTUFQTEUpIGlzIGFuIGF1dG9tYXRpYyBzeXN0ZW0gdGhhdCBjYW4gcGVyZm9ybSBhIHNlcmllcyBvZiBzdGVwcyB1c2VkIGluIHRoZSBldmFsdWF0aW9uIG9mIHBvdGVudGlhbCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjIzNjE2OCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGR0kIflPKqD2CH1/0fDRGwxagHGkQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f467%2f2236168" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f467%2f2236168" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=79980" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&amp;Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/5">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200537">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200537">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200541">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200548">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200548">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200570">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200570">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200603">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200603">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200607">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200607">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200615">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/5/451/2236165">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/5/477/2236181">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An automated system for evaluation of the potential functionome: MAPLE version 2.1.0
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideto Takami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw030-cor1" data-open="dsw030-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel: +81-45-778-5459. Fax: +81-45-778-5588. Email: <a href="mailto:takamih@jamstec.go.jp" target="_blank">takamih@jamstec.go.jp</a> (H.T.); Tel: +81-774-38-3271. Fax: +81-774-38-3269. Email: <a href="mailto:goto@kuicr.kyoto-u.ac.jp" target="_blank">goto@kuicr.kyoto-u.ac.jp</a> (S.G.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideto+Takami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takami H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideto Takami</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeaki Taniguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo 152-8550, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeaki+Taniguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Taniguchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Taniguchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeaki Taniguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wataru Arai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Microbial Genome Research Group, Yokohama Institute, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001 Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wataru+Arai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arai W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arai W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wataru Arai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhiro Takemoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div>Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhiro+Takemoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takemoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takemoto K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuhiro Takemoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Moriya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div>Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Moriya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moriya Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moriya Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Moriya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susumu Goto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div>Bioinformatics Center, Institute for Chemical Research, Kyoto University, Uji, Kyoto 611-0011, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw030-cor1" data-open="dsw030-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel: +81-45-778-5459. Fax: +81-45-778-5588. Email: <a href="mailto:takamih@jamstec.go.jp" target="_blank">takamih@jamstec.go.jp</a> (H.T.); Tel: +81-774-38-3271. Fax: +81-774-38-3269. Email: <a href="mailto:goto@kuicr.kyoto-u.ac.jp" target="_blank">goto@kuicr.kyoto-u.ac.jp</a> (S.G.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Susumu+Goto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Goto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Goto S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Susumu Goto</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw030-FM1"><div class="fn"><div class="footnote-content"><sup>†</sup>Present address: Database Center for Life Science, Research Organization of Information and Systems, Kashiwa, Chiba 277-0871, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw030-FM2"><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 467–475, <a href='https://doi.org/10.1093/dnares/dsw030'>https://doi.org/10.1093/dnares/dsw030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                            
                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideto Takami, Takeaki Taniguchi, Wataru Arai, Kazuhiro Takemoto, Yuki Moriya, Susumu Goto;  An automated system for evaluation of the potential functionome: MAPLE version 2.1.0, <em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 467–475, <a href='https://doi.org/10.1093/dnares/dsw030'>https://doi.org/10.1093/dnares/dsw030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236168?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20automated%20system%20for%20evaluation%20of%20the%20potential%20functionome%3A%20MAPLE%20version%202.1.0&amp;publicationDate=2016-07-03&amp;volumeNum=23&amp;issueNum=5&amp;author=Takami%2C%20Hideto%3B%20Taniguchi%2C%20Takeaki&amp;startPage=467&amp;endPage=475&amp;contentId=10.1093%2Fdnares%2Fdsw030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f467%2f2236168" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200537 id="87200537" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Metabolic and physiological potential evaluator (MAPLE) is an automatic system that can perform a series of steps used in the evaluation of potential comprehensive functions (functionome) harboured in the genome and metagenome. MAPLE first assigns KEGG Orthology (KO) to the query gene, maps the KO-assigned genes to the Kyoto Encyclopedia of Genes and Genomes (KEGG) functional modules, and then calculates the module completion ratio (MCR) of each functional module to characterize the potential functionome in the user’s own genomic and metagenomic data. In this study, we added two more useful functions to calculate module abundance and <em>Q</em>-value, which indicate the functional abundance and statistical significance of the MCR results, respectively, to the new version of MAPLE for more detailed comparative genomic and metagenomic analyses. Consequently, MAPLE version 2.1.0 reported significant differences in the potential functionome, functional abundance, and diversity of contributors to each function among four metagenomic datasets generated by the global ocean sampling expedition, one of the most popular environmental samples to use with this system. MAPLE version 2.1.0 is now available through the web interface (<a class="link link-uri" href="http://www.genome.jp/tools/maple/" target="_blank">http://www.genome.jp/tools/maple/</a>) 17 June 2016, date last accessed.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenome">metagenome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MAPLE">MAPLE</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;metabolic pathway&quot;">metabolic pathway</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;global ocean sampling (GOS)&quot;">global ocean sampling (GOS)</a></div>                            <h2 scrollto-destination=87200541 id="87200541" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>A primary objective of genomic and metagenomic analyses is to deduce potential comprehensive functions (the functionome) harbored by an individual organism or an entire community in diverse environments. However, evaluation of the potential functionome still remains difficult in comparison with the functional annotation of individual genes or proteins, principally because no standard methodology to extract functional category information, such as metabolism, energy generation, and membrane transport systems, has yet been established. In recent years, detailed and comprehensive functional categories present in the Kyoto Encyclopedia of Genes and Genomes (KEGG)<a reveal-id="dsw030-B1" data-open="dsw030-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and SEED<a reveal-id="dsw030-B2" data-open="dsw030-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> databases have been used to identify functional features in comparative genomics and metagenomics represented by the KEGG Automatic Annotation Server (KAAS),<a reveal-id="dsw030-B3" data-open="dsw030-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Metagenomics Rapid Annotation using Subsystem Technology,<a reveal-id="dsw030-B4" data-open="dsw030-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and Metagenome Analyzer<a reveal-id="dsw030-B5" data-open="dsw030-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> systems. Each system employs a similarity-based method for functional annotations but utilizes different databases for protein sequences, default threshold values, and ortholog identification (ID) numbers for mapping annotated sequences to functional categories depending on the desired outputs, specifically pathways in KEGG or subsystems in SEED. However, these functional categories still remain too broad to distinguish metabolic or physiological features. Thus, the evaluation of existing tools and development of new tools are required to characterize potential physiological and metabolic pathways in actual ecosystems.<a reveal-id="dsw030-B6" data-open="dsw030-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p><p>To resolve this problem, we previously developed a new method to evaluate the potential functionome based on calculating the completion ratio of four types of KEGG modules, i.e., pathways, molecular complexes, functional sets, and signatures,<a reveal-id="dsw030-B7" data-open="dsw030-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> represented as the percentage of a module component filled with the input KEGG Orthology (KO)-assigned genes by KAAS. In conjunction with the development of this method, a stand-alone calculation system of the module completion ratio (MCR) has been developed.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, it is difficult for general scientists who are not familiar with bioinformatic tools to handle the massive sequence data required for evaluation of potential functionomes in their own data. In addition, it is not easy to visualize the mapping patterns of KO-assigned query genes to the KEGG modules in detail. Thus, in December of 2013, we launched a prototype system, the metabolic and physiological potential evaluator (MAPLE), to automate our newly developed method.<a reveal-id="dsw030-B9" data-open="dsw030-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> MCR is an easy-to-understand measure to evaluate the functional potential, and generally, there is a correlation between the completeness of a KEGG module and the likelihood that an organism can perform the physiological function corresponding to the module; however, its interpretation may become difficult when the KOs used for a module are shared with other independent modules, (e.g. the modules such as the reductive TCA cycle and TCA cycle, glycolysis and gluconeogenesis), and the MCR does not necessarily reflect the working probability of each functional module. Moreover, when the module is complete, its abundance becomes a more important measure for comparing the functional robustness among different genomes or metagenomes from diverse environments. Thus, we aimed to improve the MAPLE system to increase the usefulness of our evaluation method and to facilitate the interpretation of the results of functional analyses using this system.</p><p>In this study, we present MAPLE version 2.1.0, an improvement of the original MAPLE system that permits the estimation of functional abundance and indicates the working probability of the MCR results by statistical testing. We also highlight the significant differences in abundance and biodiversity of the physiological functions among four metagenomic datasets generated by the global ocean sampling (GOS) expedition with this system.</p>                            <h2 scrollto-destination=87200548 id="87200548" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200549 id="87200549" class="section-title" >2.1. Overview</h3>
<p>MAPLE is an automatic system for mapping genes in an individual genome and metagenome to the functional modules defined by KEGG and for calculating the MCR and module abundance. Overview of MAPLE system is described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary text and Figure S1</a></span>.</p>                            <h3 scrollto-destination=87200552 id="87200552" class="section-title" >2.2. Module completion ratio</h3>
<p>The completion ratio of all KEGG functional modules in each sample was calculated based on a Boolean algebra-like equation as previously described.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The calculation program was slightly modified according to the changes in the Boolean algebra-like equation defined for some modules by the KEGG.</p>                            <h3 scrollto-destination=87200557 id="87200557" class="section-title" >2.3. Calculation of module abundance</h3>
<p>The total number of sequence reads assigned to each KO constructing a module was divided by the average length of each KO group for normalization of KO abundance. This normalized KO abundance is described at the lower right corner of each KO box when the mapping pattern of the query genes to the module is displayed. The module abundance is calculated based on the normalized KO abundance. For example, the module M00529, defined as a reaction of denitrification, is composed of four reaction steps (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). In each K number set, vertically connected K numbers indicate a complex whereas horizontally located K numbers indicate alternatives.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw030-B9" data-open="dsw030-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Because the enzyme responsible for the first reaction (nitrate reductase) is composed of four (left side) or two KO complexes (right side), the abundance of the first reaction step becomes 0 unless all KOs vertically connected are filled with the KO assigned genes. When all vertically connected KOs are filled, the minimal value of KO abundance in the vertically connected boxes becomes the abundance at the first step. Thus, the abundance of the first step in module M00529 is 63. As the second step, when two horizontally located KOs are filled with the KO assigned genes, the abundance at the second step becomes 1,129, which is sum of both KOs. The abundance at the third step becomes 56 in a similar manner as the first step, and that of the last step is 46 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Because the module abundance becomes the minimal value among all steps, the abundance of module M00529 is calculated to be 46. The method for comparison of the module abundances among different metagenomic samples is described in Supplementary text.</p>                            <h3 scrollto-destination=87200559 id="87200559" class="section-title" >2.4. Analysis of the <em>Q</em>-value for determining the significance of module completeness</h3>
<p>Generally, it is expected that the MCR is linked to the likelihood that the organisms perform the physiological function corresponding to the module. However, when the KOs used for a module are shared with the other modules, the MCR does not necessarily reflect the working probability of each functional module (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3-F</a></span>). Thus, the relationships among module completion of the targeted module, module completion of other modules to which the same KOs are assigned, and the contribution of specific KOs of each module to module completion should be considered when a module is not complete.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Namely, even if the same MCR was observed in different modules, the working probability of the physiological function is not necessarily equal among these MCRs. To avoid these problems, we proposed the use of the <em>Q</em>-value for determining the significance of module completion (see Supplementary text). This measure, which represents the probability that a reaction module is identified by chance, is calculated based on the statistics of sequence similarity scores (e.g. <em>E</em>-values) using the concept of multiple testing corrections, according to the definition of the KEGG reaction module (i.e. Boolean algebra-like equation).</p>                            <h3 scrollto-destination=87200563 id="87200563" class="section-title" >2.5. Characterization of the microbial community structure based on ribosomal proteins</h3>
<p>Ribosomal proteins are well conserved among all organisms and possess sequences specific to each individual organism; therefore, ribosomal proteins can be used for identification of organisms. Accordingly, we examined how accurately KAAS, which is used in the MAPLE system, can annotate the ribosomal proteins taxonomically. All prokaryotic ribosomal proteins available from the NCBI database (189,020 proteins) were subjected to annotation using MAPLE, and the results were then compared with the original taxonomic annotation. Consequently, since there was no significant difference between the original annotation at the phylum level and those by KAAS, despite the fact that the MAPLE system uses the KEGG database, which contains only completed genome data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), we concluded that MAPLE could be effectively applied for identification of organisms by construction of a metagenome based on ribosomal proteins. For application of MAPLE to taxonomic analysis, we calculated the ratio of bacteria and archaea in the metagenome based on the mapping pattern to the module M90000 for prokaryotic ribosomes and the taxonomic annotation of each ribosomal protein. As mentioned above, because the archaeal ribosome, which contains 58 ribosomal proteins, has six more proteins than the bacterial ribosome, we normalized the total number of archaeal ribosomal proteins to the number of bacterial ribosomal proteins by multiplying by 52/58. We summed up the number of bacterial ribosomes and normalized archaeal ribosomes and then used this sum as a denominator to calculate the ratio of archaea and bacteria. We also calculate the ratio of each taxonomic level defined by KEGG, such as phylum and class, in the metagenome using the same method.</p>                            <h3 scrollto-destination=87200566 id="87200566" class="section-title" >2.6. Metagenomic analysis of environmental samples using the MAPLE system</h3>
<p>We downloaded the metagenomic sequences generated by GOS expedition<a reveal-id="dsw030-B10" data-open="dsw030-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw030-B11" data-open="dsw030-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and the environmental data for each sampling site from the <em>i</em>Microbe database (<a class="link link-uri" href="http://data.imicrobe.us/project/view/26" target="_blank">http://data.imicrobe.us/project/view/26</a> 17 June 2016, date last accessed). Among 83 samples, we selected four metagenomic samples including over 1 million protein-coding genes, two from Sargasso Sea (GS000c and GS000d) and two from near the Galápagos Islands (GS030 and GS031). Detail information about these sites is provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Approximately 1.21 million to 1.43 million amino acid sequences in the multi-FASTA format were submitted to MAPLE through the web interface (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1-1</a></span>).</p>                            <h2 scrollto-destination=87200570 id="87200570" class="section-title" data-legacy-id=SEC3>3. Results</h2>
                            <h3 scrollto-destination=87200572 id="87200572" class="section-title" >3.1. Comparison of complete modules</h3>
<p>KEGG modules (587 modules as of December 2014) are modular functional units based on the KEGG pathways and are categorized into pathway modules, structural complexes, functional sets, and genetic signatures. We applied metagenomic sequence data from four GOS sites to the MAPLE system based on the KEGG modules and summarized all MAPLE results (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Using the MAPLE results, we first investigated the ratio of complete modules to all KEGG modules to characterize the differences in potential functionomes among the four GOS sites. According to individual taxonomic rank (ITR), the highest ratio of the complete module was found in the GS000c site (33.9%), whereas the lowest ratio (26.8%) was found in the GS030 site (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A–2 and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). ITR is a second taxonomic subcategory, containing mostly phylum-level information, such as that for <em>Firmicutes</em>, <em>Actinobacteria</em>, and <em>Alphaproteobacteria</em>, as well as class and order information, as defined in the KEGG Organisms database. These values for ITR were 5.3% lower than those for the whole community (WC) containing various taxonomic ranks appearing in GS031 and 9.7% lower than those for the WC in GS000c. Namely, some of incomplete modules by ITR were completed by WC, which is composed of a combination of various taxonomic ranks. We defined the module that was completed only by the WC as the OWC module (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). A similar trend was observed for the other two sites (GS000d and GS031), and there were approximate differences of 5–7% between the ratios of complete modules by ITR and WC. The GS000c site tended to have a high completion module rate for pathway modules, structural complexes, and functional sets, but not for signature modules, with a particularly remarkable completion rate in the functional set module, which contained functions such as aminoacyl-tRNA biosynthesis, nucleotide sugar biosynthesis, and various two-component regulatory systems (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A–2). </p>                        <a id="87200576" scrollto-destination="87200576"></a>
<div id="dsw030-F1" data-id="dsw030-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f1p.png?Expires=2147483647&Signature=KQj9C84TSqST1LiVbC1i87S9fv-KpsXONScHxEOUXAueVJVqppzpEQN23ob9kiaIKnrC3b98Z7WGlLS3XfO2sYacUX2c84Qamjh3KN7HBmVxtQUebg6th~3xBT4gfwY0R2J0QXrTo7ypXOWlc5ojTveI86YwNGR--sr4XmqyknSGHg2iiglZfIs4lqxhnhJJnyGrCQnBkGPRDHdwDYWcdOPUprsJRQvmTqv6ZE2wwVshN4HY42gVWd2U7ic9z1SNRL19euBQlI7FMVfCgts1l61RKdx4U~w8kpYE1rNUnTq9lMcH0nq6RzVrLD41Ka2L~KyiEP33kemOBZDlabDDXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the complete module patterns among the four GOS sites. A: Classification of complete modules based on the module type defined by the KEGG database. 1: Ratio of each module type in the KEGG module. 2: Comparison of the ratio of complete modules in each type of KEGG module among the four GOS sites. B: Classification of complete modules by module class based on the MCR patterns of 1,488 prokaryotes. 1: Ratio of each module class in the KEGG module. 2: Comparison of ratios of complete modules in each module class among the four GOS sites. ITR: individual taxonomic rank. WC: whole community, OWC: only whole community." path-from-xml="dsw030f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200576/dsw030f1p.tiff" path-from-xml="dsw030f1p.tiff" target="_blank">View large</a><a section="87200576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f1p.png?Expires=2147483647&Signature=tbFxF53dK7R4JCZKJkzP7JyTflUKSsQtWGDJFk1Z9gGobxjL6laH~9EWA8xXf-XjJ-WfyIUF2s7OSplOeBpVnaljXzjrrbOIV-o8LO~eNJGNtfhTaqpZXHTCwq5PNnPIHoOZFhT0NTW4SpdBdzmgL1mWz9lmOvQS6wN1jB8VUlrOOUy5vKHAt7DtRtffTrtuJp4ib4lInOqDf7QqT0oEynBdsA6JNvfaOntVUcAxefi0UsInFSITffYv4SMMoLdt8cwUu1bBYHNriaUulC0Jm11cJdWbE9sFn7veLDDaJQOAi6~3edQXIi7C~~i4ttVIz1xCiUZ4P8ypR8v95Qs0eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200576&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the complete module patterns among the four GOS sites. <strong>A</strong>: Classification of complete modules based on the module type defined by the KEGG database. 1: Ratio of each module type in the KEGG module. 2: Comparison of the ratio of complete modules in each type of KEGG module among the four GOS sites. <strong>B</strong>: Classification of complete modules by module class based on the MCR patterns of 1,488 prokaryotes. 1: Ratio of each module class in the KEGG module. 2: Comparison of ratios of complete modules in each module class among the four GOS sites. ITR: individual taxonomic rank. WC: whole community, OWC: only whole community.</p></div></div><div content-id="dsw030-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f1p.png?Expires=2147483647&Signature=KQj9C84TSqST1LiVbC1i87S9fv-KpsXONScHxEOUXAueVJVqppzpEQN23ob9kiaIKnrC3b98Z7WGlLS3XfO2sYacUX2c84Qamjh3KN7HBmVxtQUebg6th~3xBT4gfwY0R2J0QXrTo7ypXOWlc5ojTveI86YwNGR--sr4XmqyknSGHg2iiglZfIs4lqxhnhJJnyGrCQnBkGPRDHdwDYWcdOPUprsJRQvmTqv6ZE2wwVshN4HY42gVWd2U7ic9z1SNRL19euBQlI7FMVfCgts1l61RKdx4U~w8kpYE1rNUnTq9lMcH0nq6RzVrLD41Ka2L~KyiEP33kemOBZDlabDDXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the complete module patterns among the four GOS sites. A: Classification of complete modules based on the module type defined by the KEGG database. 1: Ratio of each module type in the KEGG module. 2: Comparison of the ratio of complete modules in each type of KEGG module among the four GOS sites. B: Classification of complete modules by module class based on the MCR patterns of 1,488 prokaryotes. 1: Ratio of each module class in the KEGG module. 2: Comparison of ratios of complete modules in each module class among the four GOS sites. ITR: individual taxonomic rank. WC: whole community, OWC: only whole community." path-from-xml="dsw030f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200576/dsw030f1p.tiff" path-from-xml="dsw030f1p.tiff" target="_blank">View large</a><a section="87200576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f1p.png?Expires=2147483647&Signature=tbFxF53dK7R4JCZKJkzP7JyTflUKSsQtWGDJFk1Z9gGobxjL6laH~9EWA8xXf-XjJ-WfyIUF2s7OSplOeBpVnaljXzjrrbOIV-o8LO~eNJGNtfhTaqpZXHTCwq5PNnPIHoOZFhT0NTW4SpdBdzmgL1mWz9lmOvQS6wN1jB8VUlrOOUy5vKHAt7DtRtffTrtuJp4ib4lInOqDf7QqT0oEynBdsA6JNvfaOntVUcAxefi0UsInFSITffYv4SMMoLdt8cwUu1bBYHNriaUulC0Jm11cJdWbE9sFn7veLDDaJQOAi6~3edQXIi7C~~i4ttVIz1xCiUZ4P8ypR8v95Qs0eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200576&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the complete module patterns among the four GOS sites. <strong>A</strong>: Classification of complete modules based on the module type defined by the KEGG database. 1: Ratio of each module type in the KEGG module. 2: Comparison of the ratio of complete modules in each type of KEGG module among the four GOS sites. <strong>B</strong>: Classification of complete modules by module class based on the MCR patterns of 1,488 prokaryotes. 1: Ratio of each module class in the KEGG module. 2: Comparison of ratios of complete modules in each module class among the four GOS sites. ITR: individual taxonomic rank. WC: whole community, OWC: only whole community.</p></div></div><p>The KEGG modules primarily fall into four classes (A, B, C, and D) based on distribution patterns of MCRs in all species registered in the KEGG Organisms database.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Of the 587 KEGG functional modules, only 31 (5%) of modules in prokaryotic species were grouped into class A defined as ‘universal’. All modules grouped into class A were completed by ITR except for module M00526 (lysine biosynthesis, DAP dehydrogenase pathway) but M00526 was not completed even by WC for all GOS sites (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>B–2). The ratio of complete modules by ITR in class B defined as ‘restricted’ was &lt;20%, except for GS000c, and even those by WC did not exceed 25% for sites GS000d, GS030, and GS031, although the GS000c site had a ratio of more than 30%. Modules over 63% in class C defined as ‘diversified’ were completed by the ITR in GS000c, GS000d, and GS031, and the ratio of complete modules by the WC ranged from 71% to 76% for all GOS sites except GS030. As an overall trend, there were no significant differences in the ratios of complete modules in four categorized classes based on the distribution patterns of MCRs in all classes from A to D (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>B–2).</p><p>Modules completed by &lt;10% of the prokaryotic or eukaryotic species were defined as ‘rare’ in the modules grouped into classes B and C; 269 modules (pathway: 78, structural complex: 110, functional set: 66, and signature: 15) were identified as rare modules in the prokaryotic species (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C–1). Generally, few rare modules were completed by the ITR, and the ratio of complete modules was &lt;20% for all GOS sites. However, some incomplete rare modules became complete by the WC, and the highest ratio (∼36%) of complete modules was observed in the pathway modules for all GOS sites (<a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>C–2). Class D, which accounted for 26% (149 modules) of all modules, comprised nonprokaryotic modules that were not completed by any single prokaryotic species. Unlike the case of all pathway modules shown in <a reveal-id="dsw030-F1" data-open="dsw030-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A–2, the GS000d and GS031 sites tended to have higher completion rations for rare pathway modules than the GS000c site.</p><p>In the natural environment, the microbial community is mainly composed of various uncultivable microbes, including those that are phylogenetically distant from the registered microbes in the KEGG Organisms database. Thus, the genes derived even from the same microbial genome in the environmental sample are not necessarily assigned to those from the microbes within the same taxonomic rank. This is one of major explanations for the differences in the ratios of complete modules between the WC and ITR, which were observed mainly for the pathway module. Indeed, almost all modules that were completed only by the WC were categorized into class D, with rare modules belonging to classes B and C. Two class D modules (M00013: malonate semialdehyde pathway and M00077: chondroichin sulphate degradation) and two other rare modules (M00358: coenzyme M biosynthesis and M00088: ketone body biosynthesis) were common modules completed only by the WC for all four GOS sites. Module M00013 was completed by organisms within several eukaryotic taxonomic ranks; however, M00077 was completed by microbes within several types of bacterial taxonomic ranks, such as <em>Planctomyces</em>, <em>Firmicutes</em>, and <em>Bacteroidetes</em>, with the exception of GS0030, despite the fact that M00077 was a nonprokaryotic module. Chondroichin sulphate, a sulphated glycosaminoglycan composed of a chain of alternating sugars, is an important structural component of cartilage.<a reveal-id="dsw030-B12" data-open="dsw030-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Thus, several bacterial species may cooperatively degrade chondroichin sulphate derived from the cartilage of marine creatures or some unknown single microbes that are phylogenetically distant from known prokaryotic species.</p>                            <h3 scrollto-destination=87200584 id="87200584" class="section-title" >3.2. Biodiversity and abundance of the complete modules</h3>
<p>MAPLE highlights the differences in the potential functions of organisms and environmental samples if the MCRs of various modules differ. However, in the first version of MAPLE, we could not clarify the differences in the functional potential of modules commonly completed in all samples. Accordingly, we improved the MAPLE system in order to determine the diversity of the ITRs that completed the module and the module abundance for every ITR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). We then characterized the common complete modules among the four GOS sites based on these two measures. Out of 444 KEGG modules (excluding the nonprokaryotic modules), 155 modules were completed by the WC, whereas 136 modules were completed by ITRs. Among 155 complete modules (88 pathway, 55 complex, and 12 functional set modules), there were 42 pathway modules and 24 structural complex modules showing more than 2-fold differences in module abundance ratios among the four GOS sites; these data were normalized according to the average length of each KO and number of ribosomal proteins identified at each GOS site (see Materials and methods; <a reveal-id="dsw030-F2" data-open="dsw030-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). For example, module M00572, responsible for pimeloyl-ACP biosynthesis related to biotin metabolism, showed a difference of more than 10-fold in the module abundance ratio although the module was completed by only one ITR, <em>Gammaproteobacteria-</em>others, i.e. <em>Gammaproteobacteria</em> excluding the order <em>Enterobacteriales</em> (<em>γ</em>-others) for the four GOS sites. This module, which included six reaction steps, contained two module-specific KOs, K02169 and K02170, and there were substantial differences in the abundances of K02170 between GS031 and the other three sites. Indeed, most of the sequences assigned to K02170 were derived from <em>γ</em>-others, whereas some sequences from <em>Deltaproteobacteria</em>, having MCRs of 83.3%, also contributed to the increase in the total module abundance. To compare the abundance of this module by <em>γ</em>-others among GS000c, GS000d, GS030, and GS031 sites, the module abundance was normalized by the number of ribosomal proteins appearing in each metagenome (see Supplementary Text). The normalized abundances of each site were 0.009 (GS000c), 0.004 (GS000d), 0.003 (GS030), and 0.025 (GS031), respectively, with a ratio of 3:1.3:1:8.3. In contrast, for the structural complex module responsible for the fructose transport system (M00218), there was a 12-fold greater difference in the module abundance ratio between Sargasso Sea sites (GS000c and GS000d) and the other two sites near the Galápagos Islands (GS030 and GS031), despite the fact that the module was completed by only one ITR of <em>Alphaproteobacteria</em> for all sites (<a reveal-id="dsw030-F2" data-open="dsw030-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The population of <em>Alphaproteobacteria</em> for the GS000c site was 36%, whereas that of GS000d was almost the same as those from the Galápagos Islands (57–59%). Thus, the abundance of this module did not necessarily depend on the total bacterial population within <em>Alphaproteobacteria</em>, and only limited species within this class presumably contribute to the abundance of this function. In contrast to module M00218, the two Sargasso Sea sites exhibited abundance ratios more than 10-fold higher than those of the other two sites near the Galápagos Islands in module M00222 (phosphate transport system), which was categorized into the universal module (class A). Although most major ITRs contributing to the abundance of module M00222 were from <em>Alphaproteobacteria</em>, this module was also completed by two other common ITRs, <em>γ</em>-others, and <em>Deltaproteobacteria</em> for all sites. However, the diversity of ITRs contributing to the module abundance for the Sargasso Sea sites was greater than that of the sites near the Galápagos Islands, and other ITRs, such as <em>Thaumarchaeota</em>, <em>Firmicutes-Clostridia</em> (<em>Firmicutes-C</em>), <em>Chloroflexi</em>, and <em>Cyanobacteria</em>, also contributed to the total module abundance. Besides these modules, four other modules classified in functional categories, such as pyrimidine metabolism (M00051 and M00053), photosynthesis (M00597), and DNA polymerase (M00264), showed 10-fold greater differences in module abundance ratios among the four GOS sites. </p>                        <a id="87200588" scrollto-destination="87200588"></a>
<div id="dsw030-F2" data-id="dsw030-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f2p.png?Expires=2147483647&Signature=JA9fjblHIHBG5w1EeK8-YubEevfYqmB0qUg-MNOHgWgt6J1TtibT2KhUmqidWxFJOCzTpIErZ~GONLcMWTdDWmZOOPK16e7aKKIFxAyX~MNEca3cC-HPqMGFEoyvtj2zJPJk305MzQB8QqNey19e2YPDeJqZzvUINx4XSswUBBi8HrsQQmqH1Cj~i5z1YiNTxoXQZweYYBOGB3jlWfV08DI8uUX2Ku7iEJ8vDW1N1-F~M~lXBMecHxws6dv-kImXConXXK0XowbtxAsmoKotTkvELvkeFLvrNZgFpTDV3OBYUzB9jU7QathE4ssUd9I0DQ-o9nTcRQMe5SDbV8RVeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of biodiversity contributing to the complete module and relative module abundance in the pathway and structural complex modules among the four GOS sites. The upper histogram shows MCR patterns of the KEGG module. The bars outlined by dashed lines indicate the MCRs of the WC. Uppercase characters A to D under the upper histogram show the module class used in Fig. 1, and ‘R’ indicates rare modules. The middle histogram shows the number of ITRs that completed the functional module. The lower histogram shows the relative module abundance ratio (MAR). The MAR was calculated by dividing abundance of each module by the minimum abundance among the four GOS sites. The modules showing 2-fold greater differences in the MAR among the four GOS sites are represented." path-from-xml="dsw030f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200588/dsw030f2p.tiff" path-from-xml="dsw030f2p.tiff" target="_blank">View large</a><a section="87200588" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f2p.png?Expires=2147483647&Signature=tBWr23MSzibBx-h5g4633uz6ojRmZ-lEhivzYBeh4wgbqGGNxWJ8Xym87OHbe-5u1UzFeKUPZUAhWHxfVDSCyJux-FjBzC0bccMjdiH0vJnujqanrFvgbAvKoWc2koRpkNJhJf0EV2VuCS4VwgFDLhHbP6LNm74i~4AiZew6Wt~-ShoQXhrPuRto37vKcII01J7jCUvjlERB9nXnmub8r-u6rK8dC38KGXmsuxSRnbn90NjTPQalEN8ogz3LVulV~Q~yy71pxzYPKUb0Y6gaMOIdxR8hgFq2kydZ8kapgsy6iKpg1yxyM4uWXAc1qyJghJd9MxtLG404yuJmSfRScg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200588&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of biodiversity contributing to the complete module and relative module abundance in the pathway and structural complex modules among the four GOS sites. The upper histogram shows MCR patterns of the KEGG module. The bars outlined by dashed lines indicate the MCRs of the WC. Uppercase characters A to D under the upper histogram show the module class used in Fig. 1, and ‘R’ indicates rare modules. The middle histogram shows the number of ITRs that completed the functional module. The lower histogram shows the relative module abundance ratio (MAR). The MAR was calculated by dividing abundance of each module by the minimum abundance among the four GOS sites. The modules showing 2-fold greater differences in the MAR among the four GOS sites are represented.</p></div></div><div content-id="dsw030-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f2p.png?Expires=2147483647&Signature=JA9fjblHIHBG5w1EeK8-YubEevfYqmB0qUg-MNOHgWgt6J1TtibT2KhUmqidWxFJOCzTpIErZ~GONLcMWTdDWmZOOPK16e7aKKIFxAyX~MNEca3cC-HPqMGFEoyvtj2zJPJk305MzQB8QqNey19e2YPDeJqZzvUINx4XSswUBBi8HrsQQmqH1Cj~i5z1YiNTxoXQZweYYBOGB3jlWfV08DI8uUX2Ku7iEJ8vDW1N1-F~M~lXBMecHxws6dv-kImXConXXK0XowbtxAsmoKotTkvELvkeFLvrNZgFpTDV3OBYUzB9jU7QathE4ssUd9I0DQ-o9nTcRQMe5SDbV8RVeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of biodiversity contributing to the complete module and relative module abundance in the pathway and structural complex modules among the four GOS sites. The upper histogram shows MCR patterns of the KEGG module. The bars outlined by dashed lines indicate the MCRs of the WC. Uppercase characters A to D under the upper histogram show the module class used in Fig. 1, and ‘R’ indicates rare modules. The middle histogram shows the number of ITRs that completed the functional module. The lower histogram shows the relative module abundance ratio (MAR). The MAR was calculated by dividing abundance of each module by the minimum abundance among the four GOS sites. The modules showing 2-fold greater differences in the MAR among the four GOS sites are represented." path-from-xml="dsw030f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200588/dsw030f2p.tiff" path-from-xml="dsw030f2p.tiff" target="_blank">View large</a><a section="87200588" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f2p.png?Expires=2147483647&Signature=tBWr23MSzibBx-h5g4633uz6ojRmZ-lEhivzYBeh4wgbqGGNxWJ8Xym87OHbe-5u1UzFeKUPZUAhWHxfVDSCyJux-FjBzC0bccMjdiH0vJnujqanrFvgbAvKoWc2koRpkNJhJf0EV2VuCS4VwgFDLhHbP6LNm74i~4AiZew6Wt~-ShoQXhrPuRto37vKcII01J7jCUvjlERB9nXnmub8r-u6rK8dC38KGXmsuxSRnbn90NjTPQalEN8ogz3LVulV~Q~yy71pxzYPKUb0Y6gaMOIdxR8hgFq2kydZ8kapgsy6iKpg1yxyM4uWXAc1qyJghJd9MxtLG404yuJmSfRScg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200588&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of biodiversity contributing to the complete module and relative module abundance in the pathway and structural complex modules among the four GOS sites. The upper histogram shows MCR patterns of the KEGG module. The bars outlined by dashed lines indicate the MCRs of the WC. Uppercase characters A to D under the upper histogram show the module class used in Fig. 1, and ‘R’ indicates rare modules. The middle histogram shows the number of ITRs that completed the functional module. The lower histogram shows the relative module abundance ratio (MAR). The MAR was calculated by dividing abundance of each module by the minimum abundance among the four GOS sites. The modules showing 2-fold greater differences in the MAR among the four GOS sites are represented.</p></div></div><p>M00018 (threonine biosynthesis), which was categorized into the universal module (class A), was completed by various ITRs; however, there were no substantial differences in module abundance ratios among the four GOS sites. As shown in <a reveal-id="dsw030-F3" data-open="dsw030-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, the population patterns of ITRs contributing to the module abundance were similar for the four GOS sites, with the exception of GS000c. <em>Alphaproteobacteria</em> accounted for 80% or more of the module abundance in GS000d, GS030, and GS031, and after adding the second and third major ITRs, i.e. <em>Gammaproteobacteria</em> and <em>Bacteroidetes</em>, more than 98% of the modules were accounted for. With respect to GS000c, the ratio of <em>Alphaproteobacteria</em> was low (63%), whereas that of <em>Gammaproteobacteria</em> for the second major ITR was more than two times higher than that of the other three sites. The remaining 15% was nearly evenly occupied by various ITRs, such as <em>Bacteroidetes</em>, <em>Firmicutes-C</em>, <em>Deltaproteobacteria</em>, and <em>Thaumarchaeota</em>. Modules M00338 and M00082, which were categorized as restricted (class B) and diversified (class C) modules, respectively, did not show significant differences in module abundance ratios, except for GS000d, which had a lower module abundance than the other sites (<a reveal-id="dsw030-F2" data-open="dsw030-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Similar to the case of module M00018, the patterns of a variety of ITRs, including those that completed the module M00082 (fatty acid biosynthesis-initiation), and the module abundance ratio of each ITR were similar for GS030 and GS031 but obviously different for GS000c and GS000d, despite the fact that <em>Alphaproteobacteria</em> was the major ITR contributing to module abundance for both sites (<a reveal-id="dsw030-F3" data-open="dsw030-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In contrast, with respect to M00338 (cysteine biosynthesis), various ITRs and the module abundance ratios differed completely from each other. However, <em>Bacteroidetes</em>, one of the major contributors to the module abundance, was shared among the four GOS sites. For example, this module was completed by eight types of ITRs in GS000c but only by two types of ITRs in GS031, and two eukaryotic ITRs (<em>Vertebrate</em> and <em>Amoebozoa</em>) accounted for 14% of the total module abundance (28% in total) in GS030. In GS000c, two ITRs (<em>γ</em>-others and <em>Alphaproteobacteria</em>) that never appeared at the other three sites accounted for 38% and 11% of the total module abundance, respectively. </p>                        <a id="87200593" scrollto-destination="87200593"></a>
<div id="dsw030-F3" data-id="dsw030-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f3p.png?Expires=2147483647&Signature=IC~NWWL-feOcIyZIrS9n~0VoI3lzErUeT2AfciIgqL9HAxp5-H91ruBzAuw6sfMUpJJivNAJEPKReRgFvQhfvf9GXmBPAynSF~3WMiAEfKNngE9JJ0HYcsk98-cLqQVaDYgcENoj749DbXIzGzPqAM7-uBs6pjLCfNy4KUmW5Q1LI3ni-OSrq6jes0p00Iys7nPEnxinFVsbmRNWpFmSelAYBCMhEPlmJBrIwBYvduCTov7c9RitECjRNVWNjsZqaLIvm2IkbxDA4Pxsk6d7QO9FwbYnp717yA7jsHSYPQhtaznOns3rjBY2rOXxjfQyFdwZuBXAFQi7CjqdzgylnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of abundance ratios of the modules completed by ITRs. Three typical pathway modules categorized into classes A (universal), B (restricted), and C (diversified). There were no substantial differences in relative module abundances among the four GOS sites selected for comparison of the contributions of ITR patterns to the module abundance." path-from-xml="dsw030f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200593/dsw030f3p.tiff" path-from-xml="dsw030f3p.tiff" target="_blank">View large</a><a section="87200593" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f3p.png?Expires=2147483647&Signature=ZtJK1FiSiiEd04x7XML0LWNWURQZz1eVTF97OpRrNJfO-sH7fANWTUlgP1kNsaixWDzFZVJeJsaON~pBhYbu6~b7zGbRqT9Jye8qSV1nMlq7AZyCaLHBehNCjJPn25MOGy7nHi3a3HpxD0YRC1Eq0~TL-ChJpI5xahspjlD8bQcWtrbt85wGjXL8gh4D59PylUF~p54AVzOgKIBwSuGmj7S6LnZUY~C1Rjnk2-W6uxRVOOK-bZhkyCGc2WFVBTTw07zX4p3ZI2F2XH1WpLeDJzoK-0KR2JoPh-qNhA3RtXbNTvqhLLe0Bjhuwkn2mHkbJUevJipptO09JLhv4217sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200593&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of abundance ratios of the modules completed by ITRs. Three typical pathway modules categorized into classes A (universal), B (restricted), and C (diversified). There were no substantial differences in relative module abundances among the four GOS sites selected for comparison of the contributions of ITR patterns to the module abundance.</p></div></div><div content-id="dsw030-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f3p.png?Expires=2147483647&Signature=IC~NWWL-feOcIyZIrS9n~0VoI3lzErUeT2AfciIgqL9HAxp5-H91ruBzAuw6sfMUpJJivNAJEPKReRgFvQhfvf9GXmBPAynSF~3WMiAEfKNngE9JJ0HYcsk98-cLqQVaDYgcENoj749DbXIzGzPqAM7-uBs6pjLCfNy4KUmW5Q1LI3ni-OSrq6jes0p00Iys7nPEnxinFVsbmRNWpFmSelAYBCMhEPlmJBrIwBYvduCTov7c9RitECjRNVWNjsZqaLIvm2IkbxDA4Pxsk6d7QO9FwbYnp717yA7jsHSYPQhtaznOns3rjBY2rOXxjfQyFdwZuBXAFQi7CjqdzgylnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of abundance ratios of the modules completed by ITRs. Three typical pathway modules categorized into classes A (universal), B (restricted), and C (diversified). There were no substantial differences in relative module abundances among the four GOS sites selected for comparison of the contributions of ITR patterns to the module abundance." path-from-xml="dsw030f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200593/dsw030f3p.tiff" path-from-xml="dsw030f3p.tiff" target="_blank">View large</a><a section="87200593" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f3p.png?Expires=2147483647&Signature=ZtJK1FiSiiEd04x7XML0LWNWURQZz1eVTF97OpRrNJfO-sH7fANWTUlgP1kNsaixWDzFZVJeJsaON~pBhYbu6~b7zGbRqT9Jye8qSV1nMlq7AZyCaLHBehNCjJPn25MOGy7nHi3a3HpxD0YRC1Eq0~TL-ChJpI5xahspjlD8bQcWtrbt85wGjXL8gh4D59PylUF~p54AVzOgKIBwSuGmj7S6LnZUY~C1Rjnk2-W6uxRVOOK-bZhkyCGc2WFVBTTw07zX4p3ZI2F2XH1WpLeDJzoK-0KR2JoPh-qNhA3RtXbNTvqhLLe0Bjhuwkn2mHkbJUevJipptO09JLhv4217sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200593&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of abundance ratios of the modules completed by ITRs. Three typical pathway modules categorized into classes A (universal), B (restricted), and C (diversified). There were no substantial differences in relative module abundances among the four GOS sites selected for comparison of the contributions of ITR patterns to the module abundance.</p></div></div>                            <h3 scrollto-destination=87200594 id="87200594" class="section-title" >3.3. Evaluation of the MCR according to the <em>Q</em>-value</h3>
<p>As mentioned in the section Analysis of the <em>Q</em>-value for determining the significance of module completeness, the <em>Q</em>-value is useful because the MCR has limitations. We here investigate the relationship between MCR and <em>Q</em>-value in each module for the four GOS sites, in order to show the advantage of <em>Q</em>-value.</p><p>We found higher MCRs were generally associated with lower <em>Q</em>-values, and the <em>Q</em>-value was almost 0 when the MCR was 100%. However, the <em>Q</em>-value is not necessarily negatively correlated with the MCR. For example, we considered the module M00532 (photorespiration). This module was not completed by the WC for the GS000d, GS030, and GS031 sites; however, the MCR was higher than 80% (<a reveal-id="dsw030-F4" data-open="dsw030-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Notably, the <em>Q</em>-value of this module was high for each site (0.875 for GS000d and 0.75 for both GS030 and GS031; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In this module, which comprised 10 reaction steps, the KOs assigned to seven steps (steps 2–5 and 8–10) were module-specific, sites GS030 and GS031 lacked K14272 in step 5, and GS000d lacked K03781 and K14272 in step 4. Moreover, module M00373 (ethylmalonyl pathway related to glyoxylate and dicarboxylate metabolism) was not completed by the WC at sites GS000c, GS000d, and GS030, although the MCR was 92.9%. Half of the 14 reaction steps included in this module were module-specific, and these three sites lacked only one KO, K14451, which was assigned to the last module-specific step in this module. According to the definition, the <em>Q</em>-value emphasizes the completion of module-specific KOs for evaluating the working probability; thus, even though a high MCR value is shown, the <em>Q</em>-value is high when the module-specific KO is missing. </p>                        <a id="87200597" scrollto-destination="87200597"></a>
<div id="dsw030-F4" data-id="dsw030-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f4p.png?Expires=2147483647&Signature=ar8RUKu6BrDbpnMRmapPwcJZEppcLAwNbkDjIDaksG5IAmih~PcOtTsLexI5NXf7QXPoViwcgCpMI4CrpC02K~Q-8XxQ5kKVB3f~UBR~Ti-Ow9O3vqwKD6MEblK4SEGAKfUga-3yZOsLsi5VXm2Rw3bkWPh7gFzStG8ba2RwOm0nGvSciIOQbA7YdnEbn2cEaip0MfiSCBgpgFAho-WVCl5jhiwTVbTRVgxNycczTJpL08BRtl6fSsu7ylBg3s3ShVRcNhD2W-sYOxHtTNKjq8EF7meGXxw7Sntf2Kb27Y9qe6lQ63X6wkNg2UeQIEqU8C1Z3sCshCLu1Onisajp~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of MCR and Q-value patterns of pathway modules by ITRs and WCs among the four GOS sites. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031. 1: ITR, 2: WC. Triangle: photorespiration (M00532), diamond: ethylmalonyl pathway (M00373), square: C5-isoprenoid biosynthesis (M00095). Number of modules with the Q-value &lt;0.5 in ITR and WC is shown in each sample. Among those with the Q-value &lt;0.5, the number of modules with that of nearly zero is shown in the parentheses. The number of modules with the Q-value &lt;0.5 in WC is somewhat larger than that in ITR in all GOS sites." path-from-xml="dsw030f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200597/dsw030f4p.tiff" path-from-xml="dsw030f4p.tiff" target="_blank">View large</a><a section="87200597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f4p.png?Expires=2147483647&Signature=Ib2KXStW-nA5GALVd0DlD2kDKGkQdpRNczHX2Owo2~wv5ky~FxDIFOlojemh9Too8qKucBVvfdT7ZUS7oIhIkFF0vugrbyx3abiMlXHFJF5ypfmolCGho4dmonwI5ujH-KR6hbqVlNeFO9JD36xoG7yM9Ih2tuuDSpgMDiEW0TD9bo~bnv3qcbiLM2w-r3-wfMiaGNsKuVvh70SkNIhtNm3tSotCrhMu3F53c3nKGYbRO6pze5DUTsM4xZS9QOtXB~2w-4KFWxokmS3nVqQW4VRbW3fRbhw2w~WF~9FHtOtWWu8fFQRNDsB9EjLaWY-pcuRliFVZCEDJd7sq-uoHjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200597&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of MCR and <em>Q</em>-value patterns of pathway modules by ITRs and WCs among the four GOS sites. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031. 1: ITR, 2: WC. Triangle: photorespiration (M00532), diamond: ethylmalonyl pathway (M00373), square: C<sub>5</sub>-isoprenoid biosynthesis (M00095). Number of modules with the <em>Q</em>-value &lt;0.5 in ITR and WC is shown in each sample. Among those with the <em>Q</em>-value &lt;0.5, the number of modules with that of nearly zero is shown in the parentheses. The number of modules with the <em>Q</em>-value &lt;0.5 in WC is somewhat larger than that in ITR in all GOS sites.</p></div></div><div content-id="dsw030-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f4p.png?Expires=2147483647&Signature=ar8RUKu6BrDbpnMRmapPwcJZEppcLAwNbkDjIDaksG5IAmih~PcOtTsLexI5NXf7QXPoViwcgCpMI4CrpC02K~Q-8XxQ5kKVB3f~UBR~Ti-Ow9O3vqwKD6MEblK4SEGAKfUga-3yZOsLsi5VXm2Rw3bkWPh7gFzStG8ba2RwOm0nGvSciIOQbA7YdnEbn2cEaip0MfiSCBgpgFAho-WVCl5jhiwTVbTRVgxNycczTJpL08BRtl6fSsu7ylBg3s3ShVRcNhD2W-sYOxHtTNKjq8EF7meGXxw7Sntf2Kb27Y9qe6lQ63X6wkNg2UeQIEqU8C1Z3sCshCLu1Onisajp~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of MCR and Q-value patterns of pathway modules by ITRs and WCs among the four GOS sites. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031. 1: ITR, 2: WC. Triangle: photorespiration (M00532), diamond: ethylmalonyl pathway (M00373), square: C5-isoprenoid biosynthesis (M00095). Number of modules with the Q-value &lt;0.5 in ITR and WC is shown in each sample. Among those with the Q-value &lt;0.5, the number of modules with that of nearly zero is shown in the parentheses. The number of modules with the Q-value &lt;0.5 in WC is somewhat larger than that in ITR in all GOS sites." path-from-xml="dsw030f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200597/dsw030f4p.tiff" path-from-xml="dsw030f4p.tiff" target="_blank">View large</a><a section="87200597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f4p.png?Expires=2147483647&Signature=Ib2KXStW-nA5GALVd0DlD2kDKGkQdpRNczHX2Owo2~wv5ky~FxDIFOlojemh9Too8qKucBVvfdT7ZUS7oIhIkFF0vugrbyx3abiMlXHFJF5ypfmolCGho4dmonwI5ujH-KR6hbqVlNeFO9JD36xoG7yM9Ih2tuuDSpgMDiEW0TD9bo~bnv3qcbiLM2w-r3-wfMiaGNsKuVvh70SkNIhtNm3tSotCrhMu3F53c3nKGYbRO6pze5DUTsM4xZS9QOtXB~2w-4KFWxokmS3nVqQW4VRbW3fRbhw2w~WF~9FHtOtWWu8fFQRNDsB9EjLaWY-pcuRliFVZCEDJd7sq-uoHjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200597&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of MCR and <em>Q</em>-value patterns of pathway modules by ITRs and WCs among the four GOS sites. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031. 1: ITR, 2: WC. Triangle: photorespiration (M00532), diamond: ethylmalonyl pathway (M00373), square: C<sub>5</sub>-isoprenoid biosynthesis (M00095). Number of modules with the <em>Q</em>-value &lt;0.5 in ITR and WC is shown in each sample. Among those with the <em>Q</em>-value &lt;0.5, the number of modules with that of nearly zero is shown in the parentheses. The number of modules with the <em>Q</em>-value &lt;0.5 in WC is somewhat larger than that in ITR in all GOS sites.</p></div></div><p>Although module M00095 (C<sub>5</sub> isoprenoid biosynthesis in the mevalonate pathway) was not completed in the WC, the MCR was high (85.7%) for the four GOS sites, similar to those of the modules M00532; however, the <em>Q</em>-value of this module was low (0.25). There are four module-specific reaction steps (steps 3–6) in the module comprising seven reaction steps, and the KO assigned to step 5 was missing for all GOS sites, similar to the results observed for the module M00532. However, there was a notable difference in the module structure of the missing reaction step; that is, reaction step 5 consisted of alternative genes (KOs), i.e. isozymes. According to the definition, the <em>Q</em>-value becomes lower when more isozymes are available; thus, it is low in such a case, even if module-specific KOs are missing.</p>                            <h3 scrollto-destination=87200599 id="87200599" class="section-title" >3.4. Microbial community structure based on ribosomal proteins</h3>
<p>To compare the microbial community structure among the four GOS sites, we analysed the taxonomic composition at the phylum level based on the ribosomal proteins comprising prokaryotic and eukaryotic ribosomes (see Materials and methods). As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, because filters with a pore size of 0.1–0.22 µm were used to recover the cells from seawater collected at the four GOS sites in the previous studies,<a reveal-id="dsw030-B10" data-open="dsw030-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw030-B11" data-open="dsw030-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> after prefiltration with filters having a pore size of 0.8 µm, most of the recovered cells were expected to be prokaryotic. As expected, more than 99% of the microbial community was composed of prokaryotes for all GOS sites, particularly the GS030 site; bacteria accounted for 99.9%, whereas archaea accounted for ∼2–6% at the other three sites (<a reveal-id="dsw030-F5" data-open="dsw030-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Although the ratio of archaea at site GS000c (5.9%) was higher than that at other sites (GS000d: 2.3%, GS030: 0.07%, and GS031: 3.5%), the archaeal population in the microbial community was generally very low. </p>                        <a id="87200601" scrollto-destination="87200601"></a>
<div id="dsw030-F5" data-id="dsw030-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f5p.png?Expires=2147483647&Signature=vuMQTLjGMSkQ518yYQTx-Hxp8XxBVUVPz7sTV8nHwAbdnGMYUBAQ8kzbTmSmD0Ayz6B56baEEmDSdhTASTNbTu0IPtBEv0zuSmPqYyRPENVmT6lSMVEZnQTvoLbNmOK0L8qUhmwqocnqXGVzLbGXEINShWZm7dwBXymwyxirmp6ZTPBTh3F2LxOCXfyC4eYQDke3471jkQQhJLroKgSQWL~E7E~ZBGzBIY04q03kmbYQOldYhE2GmdWKmq8FeoIojW1cDyRjZZr4Dy~2DuOfYsDmqA4JWEompvMmiONK4xgsGnjd1~QxObN8ez7RTm702lKBf~4ZKOzsA6N5MH94KA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microbial community structure for the four GOS sites based on ribosomal proteins. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031." path-from-xml="dsw030f5p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200601/dsw030f5p.tiff" path-from-xml="dsw030f5p.tiff" target="_blank">View large</a><a section="87200601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f5p.png?Expires=2147483647&Signature=sxr8BntsTP20lmG7xNB7CXj-Ut9iQvc8DnkOEIw0RVvJoSGNpakulrQom4J25V061ZySGhGOJfQgnSZazc5UJVAFAy25QBuc4xlXpYycZklLBMWf9sFotqWU0hfm8~-7E~7UoYwzD3NGAkWRGZh5Bi3jiyPDsyMWsEAF4uSt~7Py-fL5X1jKrffYKSkKM-70uwkh1eQeiKZ9niJwwP3tX5nJjrO~xWOaYglz5iVD5BF77fc8uycoo8gww0ILRZ9OJkuspkEbMOwfyZxF8Y3f-Qvl~dU3nRp2ZwjTWSveSCuDgThI7Jn1w4DGZzaUCG5GlQ6Zb4GCb-Z7WxnpjaW~fQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200601&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f5p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microbial community structure for the four GOS sites based on ribosomal proteins. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031.</p></div></div><div content-id="dsw030-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/m_dsw030f5p.png?Expires=2147483647&Signature=vuMQTLjGMSkQ518yYQTx-Hxp8XxBVUVPz7sTV8nHwAbdnGMYUBAQ8kzbTmSmD0Ayz6B56baEEmDSdhTASTNbTu0IPtBEv0zuSmPqYyRPENVmT6lSMVEZnQTvoLbNmOK0L8qUhmwqocnqXGVzLbGXEINShWZm7dwBXymwyxirmp6ZTPBTh3F2LxOCXfyC4eYQDke3471jkQQhJLroKgSQWL~E7E~ZBGzBIY04q03kmbYQOldYhE2GmdWKmq8FeoIojW1cDyRjZZr4Dy~2DuOfYsDmqA4JWEompvMmiONK4xgsGnjd1~QxObN8ez7RTm702lKBf~4ZKOzsA6N5MH94KA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microbial community structure for the four GOS sites based on ribosomal proteins. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031." path-from-xml="dsw030f5p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200601/dsw030f5p.tiff" path-from-xml="dsw030f5p.tiff" target="_blank">View large</a><a section="87200601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030f5p.png?Expires=2147483647&Signature=sxr8BntsTP20lmG7xNB7CXj-Ut9iQvc8DnkOEIw0RVvJoSGNpakulrQom4J25V061ZySGhGOJfQgnSZazc5UJVAFAy25QBuc4xlXpYycZklLBMWf9sFotqWU0hfm8~-7E~7UoYwzD3NGAkWRGZh5Bi3jiyPDsyMWsEAF4uSt~7Py-fL5X1jKrffYKSkKM-70uwkh1eQeiKZ9niJwwP3tX5nJjrO~xWOaYglz5iVD5BF77fc8uycoo8gww0ILRZ9OJkuspkEbMOwfyZxF8Y3f-Qvl~dU3nRp2ZwjTWSveSCuDgThI7Jn1w4DGZzaUCG5GlQ6Zb4GCb-Z7WxnpjaW~fQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200601&ar=2236168&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw030f5p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microbial community structure for the four GOS sites based on ribosomal proteins. (A) GS000c, (B) GS000d, (C) GS030, and (D) GS031.</p></div></div><p>The major bacterial taxon at all GOS sites was <em>Alphaproteobacteria</em>; the proportion of <em>Alphaproteobacteria</em> within the WC was ∼60%, except for at site GS000c, where the proportion was only 36%. Conversely, site GS000c exhibited the highest proportion of <em>Gammaproteobacteria</em> (26%), which was the second most common taxon for all sites. Another major difference between sites GS000c and GS000d was the proportions of <em>Firmicuctes</em> and <em>Bacteroidetes</em>; for both organisms, the proportion tended to be higher at site GS000c than at site GS000d. In contrast, the population of <em>Cyanobacteria</em> was four times higher in GS000d (8%) than in GS000c, despite the similarities in environmental and sampling conditions of these two sites.<a reveal-id="dsw030-B10" data-open="dsw030-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> With respect to the other two sites near the Galápagos Islands, there were no substantial differences in bacterial proportions, with the exception that the proportion of <em>Actinobacteria</em> (6%) was three times higher at site GS031 than at site GS0030, whereas almost no archaea were detected at site GS030.</p>                            <h2 scrollto-destination=87200603 id="87200603" class="section-title" data-legacy-id=SEC4>4. Discussion</h2>
<p>The original MAPLE system has been used for metagenomic analysis of the human gut microbiome<a reveal-id="dsw030-B13" data-open="dsw030-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and for comparative functional analysis of individual microbes, even within candidate phyla.<a reveal-id="dsw030-B14" data-open="dsw030-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw030-B15" data-open="dsw030-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> There is generally a correlation between the completeness of a KEGG module and the likelihood that an organism or microbial community performs the physiological function. However, the MCR does not necessarily reflect the working probability of each functional module because the KOs used for each module are shared between other independent modules (e.g. between TCA cycle [M00009] and the reductive TCA cycle [M00173], and between glycolysis [M00001] and gluconeogenesis [M00003]). Thus, we introduced the <em>Q</em>-value parameter to the MAPLE system to statistically re-evaluate the working probability based on the MCR, and several modules, which were deduced to have a higher working probability, were suggested based on the <em>Q</em>-values of the incomplete modules in this study. The <em>Q</em>-value is also helpful for evaluating the working probability of the functional module in the individual genome, and modules with lower <em>Q</em>-values, such as glycolysis (MCR: 90%, <em>Q</em>-value: 0.281) and gluconeogenesis (MCR: 85.7%, <em>Q</em>-value: 0.375), could be completed by manual curation in the genome of <em>Ca</em>. C. subterraneum.<a reveal-id="dsw030-B15" data-open="dsw030-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Thus, the <em>Q</em>-value is thought to be a good suggestive index to determine the working probability when the functional modules are not completed in the metagenome or in individual genomes.</p><p>In addition to the <em>Q</em>-value, we also introduced the concepts of biodiversity and the abundance of complete modules for more detailed analyses of the complete modules because we could not distinguish the properties of complete modules shared between the different environmental samples using the original MAPLE system. To demonstrate the usefulness of the new MAPLE system, we used four metagenomic sequence datasets generated by the GOS expedition. When we compared the abundance of each functional module among the four GOS sites, we used the module abundance normalized by the sum of ribosomal proteins constructing the prokaryotic ribosome module (M90000). Generally, most environmental metagenomic analyses target prokaryotes, and very few eukaryotic sequences (&lt;1%) were detected in the metagenomic sample in this study. In such cases, it is easy to normalize the module abundance according to the total number of prokaryotic ribosomal proteins; however, when eukaryotic sequences are frequently detected, eukaryotic ribosomal proteins also should be considered for normalization of the module abundance. The eukaryotic ribosome is composed of 79 ribosomal proteins, including two accessory proteins; importantly, 33 KOs corresponding to the ribosomal proteins are shared between archaea and eukaryotes, similar to the case of bacteria and archaea. Thus, when the number of eukaryotic ribosomal proteins is calculated, we have to consider this point.</p><p>In many metagenomic analyses, 16S rRNA gene sequences obtained by polymerase chain reaction (PCR) amplification are used to compare microbial community structures among different environments.<a reveal-id="dsw030-B16" data-open="dsw030-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> In recent studies, this PCR-based amplicon approach has targeted the V4 region because different regions of the 16S rRNA gene have been shown to yield varying degrees of accuracy in taxonomic assignments.<a reveal-id="dsw030-B17" data-open="dsw030-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> However, prokaryotic species exhibit variations in the copy number of the 16S rRNA gene, and it is impossible to determine the copy numbers of individual uncultivable, unknown microbes present in actual microbial communities. Thus, because taxonomic compositions based on amplicon sequences are strongly influenced by copy numbers in addition to basic PCR bias, this approach is not useful for the analysis of microbial community structure. On the other hand, a new method based on universal single-copy genes, which provide prokaryotic species boundaries at higher resolution than 16S rRNA gene, has been used to estimate relative abundances of known and currently unknown microbial community members with metagenomic data at species-level resolution.<a reveal-id="dsw030-B18" data-open="dsw030-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> However, community structure analysis at such high resolution is not necessarily required in metagenomic analyses of natural environments, unlike that of the human gut microbiome, because many community members have not yet been cultivated or identified at the species level. Thus, community structure analyses at the phylum or class level based on ribosomal proteins using the new version (version 2.1.0) of the MAPLE system are thought to be feasible for analysis of metagenomes from the natural environments. Although the sequence lengths generated by Illumina HiSeq and MiSeq sequencers (at most 400–500 bp as an assembled contig of the paired-end sequences) are shorter than the Sanger sequences used in this study, we have already confirmed that there are no discernible differences in KO ID assignment and mapping ratios of KO-assigned coding sequences for the KEGG modules in the metagenomic sequences of the human gut microbiome when comparing Sanger and Illumina reads.<a reveal-id="dsw030-B8" data-open="dsw030-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw030-B19" data-open="dsw030-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Finally, user’s guide of MAPLE is described in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h2 scrollto-destination=87200607 id="87200607" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank H. Uehara of SGI Japan Ltd. for assistance with optimization of the computer environment. We also thank Dr H. Huang of Chuo University for technical assistance.</p>                            <h3 scrollto-destination=87200609 id="87200609" class="section-title" >Conflict of interest</h3>
<p>None declared. </p>                            <h3 scrollto-destination=87200611 id="87200611" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=87200613 id="87200613" class="section-title" >Funding</h3>
<p>This work was supported in part by grants from the collaborative research program of the Institute for Chemical Research, Kyoto University to H.T. and S.G. (grants #2013-23 and #2014-24). This work was also supported in part by KAKENHI to H.T. and K.T. (No. 26550053) and by a grant from the Cross-ministerial Strategic Innovation Promotion Program to H.T. and W.A.</p>                            <h2 scrollto-destination=87200615 id="87200615" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw030-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Goto</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2000</div>, 
<div class="article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">27</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes%2C&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Overbeek</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Begley</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Butler</div>
               <div class="given-names">R.M.</div>
            </div>
          </div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">5691</div>–<div class="lpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20subsystems%20approach%20to%20genome%20annotation%20and%20its%20use%20in%20the%20project%20to%20annotate%201000%20genomes&amp;author=R.%20Overbeek&amp;author=T.%20Begley&amp;author=R.M.%20Butler&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=5691-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16214803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Moriya</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Itoh</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Okuda</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Yoshizawa</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2007</div>, 
<div class="article-title">KAAS: an automatic genome annotation and pathway reconstruction server</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W182</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KAAS%3A%20an%20automatic%20genome%20annotation%20and%20pathway%20reconstruction%20server&amp;author=Y.%20Moriya&amp;author=M.%20Itoh&amp;author=S.%20Okuda&amp;author=A.%20Yoshizawa&amp;author=M.%20Kanehisa&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Meyer</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Paarmann</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">D’Souza</div>
               <div class="given-names">M.</div>
            </div>
          </div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">The metagenomics RAST server–a public resource for the automatic phylogenetic and functional analysis of metagenomes</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">386</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20metagenomics%20RAST%20server%E2%80%93a%20public%20resource%20for%20the%20automatic%20phylogenetic%20and%20functional%20analysis%20of%20metagenomes&amp;author=F.%20Meyer&amp;author=D.%20Paarmann&amp;author=M.%20D%E2%80%99Souza&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-386" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-386" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-386"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18803844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Huson</div>
               <div class="given-names">D.H.</div>
            </div>
             <div class="name">
              <div class="surname">Mitra</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Ruscheweyh</div>
               <div class="given-names">H.J.</div>
            </div>
             <div class="name">
              <div class="surname">Weber</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Schuster</div>
               <div class="given-names">S.C.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">Integrative analysis of environmental sequences using MEGAN4</div>. <div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">1552</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20analysis%20of%20environmental%20sequences%20using%20MEGAN4&amp;author=D.H.%20Huson&amp;author=S.%20Mitra&amp;author=H.J.%20Ruscheweyh&amp;author=N.%20Weber&amp;author=S.C.%20Schuster&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=1552-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.120618.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.120618.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.120618.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21690186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Filippo</div>
               <div class="given-names">C.D.</div>
            </div>
             <div class="name">
              <div class="surname">Ramazzotti</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Fontana</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Cavalieri</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Bioinformatic approaches for functional annotation and pathway inference in metagenomic data</div>. <div class="source">Brief Bioinform</div> ., <div class="volume">13</div>, <div class="fpage">696</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20approaches%20for%20functional%20annotation%20and%20pathway%20inference%20in%20metagenomic%20data&amp;author=C.D.%20Filippo&amp;author=M.%20Ramazzotti&amp;author=R.%20Fontana&amp;author=D.%20Cavalieri&amp;publication_year=2012&amp;journal=Brief%20Bioinform&amp;volume=13&amp;pages=696-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23175748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Araki</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Goto</div>
               <div class="given-names">S.</div>
            </div>
          </div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">KEGG for linking genomes to life and environment</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">D480</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%20for%20linking%20genomes%20to%20life%20and%20environment&amp;author=M.%20Kanehisa&amp;author=M.%20Araki&amp;author=S.%20Goto&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D480-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm882" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm882" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm882"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18077471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Takami</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Taniguchi</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Moriya</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kuwahara</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Kanehisa</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Goto</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Evaluation method for the potential functionome harbored in the genome and metagenome</div>. <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">699</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20method%20for%20the%20potential%20functionome%20harbored%20in%20the%20genome%20and%20metagenome&amp;author=H.%20Takami&amp;author=T.%20Taniguchi&amp;author=Y.%20Moriya&amp;author=T.%20Kuwahara&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=699" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23234305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Takami</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2014</div>, New method for comparative functional genomics and metagenomics using KEGG modules. In: <div class="person-group"><div class="name"><div class="surname">K.</div> <div class="given-names">Nelson</div></div></div> ed., <div class="source">Encyclopedia of Metagenomics</div> . 
<div class="publisher-name">Springer-Verlag</div>, 
<div class="publisher-loc">Berlin, Heidelberg, Germany</div>, pp. <div class="fpage">525</div>–<div class="lpage">39</div>.</div></div></div></div><div content-id="dsw030-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Venter</div>
               <div class="given-names">J.C</div>
            </div>
             <div class="name">
              <div class="surname">Remington</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Heidelberg</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>, et al.  . <div class="year">2004</div>, 
<div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>. <div class="source">Science</div> , <div class="volume">304</div>, <div class="fpage">66</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.C%20Venter&amp;author=K.%20Remington&amp;author=J.F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yutin</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Suzuki</div>
               <div class="given-names">M.T.</div>
            </div>
             <div class="name">
              <div class="surname">Teeling</div>
               <div class="given-names">H.</div>
            </div>
          </div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Assessing diversity and biogeography of aerobic anoxygenic phototrophic bacteria in surface waters of the Atlantic and Pacific Oceans using the Global Ocean Sampling expedition metagenomes</div>. <div class="source">Environ. Microbiol</div> ., <div class="volume">9</div>, <div class="fpage">1464</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20diversity%20and%20biogeography%20of%20aerobic%20anoxygenic%20phototrophic%20bacteria%20in%20surface%20waters%20of%20the%20Atlantic%20and%20Pacific%20Oceans%20using%20the%20Global%20Ocean%20Sampling%20expedition%20metagenomes&amp;author=N.%20Yutin&amp;author=M.T.%20Suzuki&amp;author=H.%20Teeling&amp;publication_year=2007&amp;journal=Environ.%20Microbiol&amp;volume=9&amp;pages=1464-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2007.9.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2007.9.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2007.9.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17504484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Baeurle</div>
               <div class="given-names">S.A.</div>
            </div>
             <div class="name">
              <div class="surname">Kiselev</div>
               <div class="given-names">M.G.</div>
            </div>
             <div class="name">
              <div class="surname">Makorava</div>
               <div class="given-names">E.S.</div>
            </div>
             <div class="name">
              <div class="surname">Nogovitsin</div>
               <div class="given-names">E.A.</div>
            </div>
          </div>
          <div class="year">2009</div>, 
<div class="article-title">Effect of counterion behavior on the frictional-compressive properties of chondroitin sulfate solutions</div>. <div class="source">Polymer</div> , <div class="volume">50</div>, <div class="fpage">1805</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20counterion%20behavior%20on%20the%20frictional-compressive%20properties%20of%20chondroitin%20sulfate%20solutions&amp;author=S.A.%20Baeurle&amp;author=M.G.%20Kiselev&amp;author=E.S.%20Makorava&amp;author=E.A.%20Nogovitsin&amp;publication_year=2009&amp;journal=Polymer&amp;volume=50&amp;pages=1805-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.polymer.2009.01.066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.polymer.2009.01.066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.polymer.2009.01.066"> </span></div> </div></div></div></div></div><div content-id="dsw030-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Obregon-Tito</div>
               <div class="given-names">A.J.</div>
            </div>
             <div class="name">
              <div class="surname">Tito</div>
               <div class="given-names">R.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Metcalf</div>
               <div class="given-names">J.</div>
            </div>
          </div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Subsistence strategies in traditional societies distinguish gut microbiomes</div>. <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">6505</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Subsistence%20strategies%20in%20traditional%20societies%20distinguish%20gut%20microbiomes&amp;author=A.J.%20Obregon-Tito&amp;author=R.Y.%20Tito&amp;author=J.%20Metcalf&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=6505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms7505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms7505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms7505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25807110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shoemaker</div>
               <div class="given-names">W.R.</div>
            </div>
             <div class="name">
              <div class="surname">Muscarella</div>
               <div class="given-names">M.E.</div>
            </div>
             <div class="name">
              <div class="surname">Lennon</div>
               <div class="given-names">J.T.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Genome sequence of the soil bacterium <em>Janthinobacterium</em> sp. KBS0711</div>. <div class="source">Genome Announc</div> ., <div class="volume">3</div>, <div class="fpage">e00689</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20soil%20bacterium%20Janthinobacterium%20sp.%20KBS0711&amp;author=W.R.%20Shoemaker&amp;author=M.E.%20Muscarella&amp;author=J.T.%20Lennon&amp;publication_year=2015&amp;journal=Genome%20Announc&amp;volume=3&amp;pages=e00689-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/genomeA.00689-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.00689-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FgenomeA.00689-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26089434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Takami</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Arai</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Takemoto</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Uchiyama</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Taniguchi</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">Functional classification of uncultured “<em>Candidatus</em> Caldiarchaeum subterraneum” using the MAPLE system</div>. <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0132994</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20classification%20of%20uncultured%20%E2%80%9CCandidatus%20Caldiarchaeum%20subterraneum%E2%80%9D%20using%20the%20MAPLE%20system&amp;author=H.%20Takami&amp;author=W.%20Arai&amp;author=K.%20Takemoto&amp;author=I.%20Uchiyama&amp;author=T.%20Taniguchi&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0132994" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0132994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0132994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0132994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26196861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tringe</div>
               <div class="given-names">S.G.</div>
            </div>
             <div class="name">
              <div class="surname">Rubin</div>
               <div class="given-names">E.M.</div>
            </div>
          </div>
          <div class="year">2005</div>, 
<div class="article-title">Metagenomics: DNA sequencing of environmental samples</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">6</div>, <div class="fpage">805</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20DNA%20sequencing%20of%20environmental%20samples&amp;author=S.G.%20Tringe&amp;author=E.M.%20Rubin&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet&amp;volume=6&amp;pages=805-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16304596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Lozupone</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Hamady</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Bushman</div>
               <div class="given-names">F.D.</div>
            </div>
             <div class="name">
              <div class="surname">Knight</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2007</div>, 
<div class="article-title">Short pyrosequencing reads suffice for accurate microbial community analysis</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">e120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20pyrosequencing%20reads%20suffice%20for%20accurate%20microbial%20community%20analysis&amp;author=Z.%20Liu&amp;author=C.%20Lozupone&amp;author=M.%20Hamady&amp;author=F.D.%20Bushman&amp;author=R.%20Knight&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=e120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17881377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mende</div>
               <div class="given-names">D.R.</div>
            </div>
             <div class="name">
              <div class="surname">Sunagawa</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Aeller</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Bork</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">Accurate and universal delineation of prokaryotic species</div>. <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">881</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20and%20universal%20delineation%20of%20prokaryotic%20species&amp;author=D.R.%20Mende&amp;author=S.%20Sunagawa&amp;author=G.%20Aeller&amp;author=P.%20Bork&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=881-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2575"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23892899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw030-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw030-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Qin</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Raes</div>
               <div class="given-names">J.</div>
            </div>
          </div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">A human gut microbial gene catalogue established by metagenomic sequencing</div>. <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">59</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;author=J.%20Qin&amp;author=R.%20Li&amp;author=J.%20Raes&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=59-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2236168&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw030/1/dsw030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=CUxDxq5mmK6QO8dvPCzRgfxblrnJVuigxw8BfB4MejY5LB2j05kXs2TO0x~dMt0Npl3EEMGtOQ73iZCz8MGFzOFqa9TnySZNwFkh0kTgJ4rBh~oTWaYgZbaN8WAPKDq8NHnQQU9fZocq2GYWXplIx2cEa2rLNfZjwuToe4KlLQwxk9aal-m~cHd6TCTrx8agyuSOVT1wsWzGm4wPwgw5OlyWFO-m-Pryg2N~KdumlXHAmJO-Px3awZDlbT3xFTvZdwnngtAqfkT4QW1u9LYUr~DV894tx1z9qVuIJZ9KLcv~lEEJ2r0HKBzg5N3CQOHEVW33XWEAbKT55LKtq5wh-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">659</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2236168"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000386964800005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f5%2f467%2f2236168">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000386964800005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F5%2F467%2F2236168" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2236168'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="automated system for evaluation of the potential functionome: MAPLE version 2.1.0" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f467%2f2236168">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

