

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/511/5047107","@type":"ScholarlyArticle","name":"Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in  Brassica rapa  L.","datePublished":"2018-06-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy021","keywords":["DNA methylation","histone H3 lysine 9 di-methylation","small RNA","epigenetic","Brassica"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/511/5047107","author":[{"name":"Takahashi, Satoshi","affiliation":"RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Osabe, Kenji","affiliation":"Plant Epigenetics Unit, Okinawa Institute of Science and Technology Graduate University, Onna-son, Okinawa, Japan","@type":"Person"},{"name":"Fukushima, Naoki","affiliation":"Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan","@type":"Person"},{"name":"Takuno, Shohei","affiliation":"Department of Evolutionary Studies of Biosystems, SOKENDAI (The Graduate University for Advanced Studies), Hayama, Kanagawa, Japan","@type":"Person"},{"name":"Miyaji, Naomi","affiliation":"Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan","@type":"Person"},{"name":"Shimizu, Motoki","affiliation":"Iwate Biotechnology Research Center, Narita Kitakami, Iwate, Japan","@type":"Person"},{"name":"Takasaki-Yasuda, Takeshi","affiliation":"Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Dennis, Elizabeth S","affiliation":"CSIRO Agriculture and Food, Canberra, ACT, Australia  University of Technology, Sydney, Broadway, NSW, Australia","@type":"Person"},{"name":"Seki, Motoaki","affiliation":"RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan  Core Research for Evolutional Science and Technology, Japan Science and Technology, Kawaguchi, Saitama, Japan  RIKEN Cluster for Pioneering Research, Wako, Saitama, Japan","@type":"Person"},{"name":"Fujimoto, Ryo","affiliation":"Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan","@type":"Person","sameAs":"http://orcid.org/0000-0003-4690-1739"}],"description":"Abstract.  Epigenetic gene regulation is crucial to plant life and can involve dynamic interactions between various histone modifications, DNA methylation, and ","pageStart":"511","pageEnd":"520","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in  Brassica rapa  L.","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in  Brassica rapa  L." />
    <meta property="og:description" content="Abstract.  Epigenetic gene regulation is crucial to plant life and can involve dynamic interactions between various histone modifications, DNA methylation, and " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/511/5047107" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-06-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takahashi, Satoshi" /><meta name="citation_author_institution" content="RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Osabe, Kenji" /><meta name="citation_author_institution" content="Plant Epigenetics Unit, Okinawa Institute of Science and Technology Graduate University, Onna-son, Okinawa, Japan" /><meta name="citation_author" content="Fukushima, Naoki" /><meta name="citation_author_institution" content="Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan" /><meta name="citation_author" content="Takuno, Shohei" /><meta name="citation_author_institution" content="Department of Evolutionary Studies of Biosystems, SOKENDAI (The Graduate University for Advanced Studies), Hayama, Kanagawa, Japan" /><meta name="citation_author" content="Miyaji, Naomi" /><meta name="citation_author_institution" content="Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan" /><meta name="citation_author" content="Shimizu, Motoki" /><meta name="citation_author_institution" content="Iwate Biotechnology Research Center, Narita Kitakami, Iwate, Japan" /><meta name="citation_author" content="Takasaki-Yasuda, Takeshi" /><meta name="citation_author_institution" content="Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_author" content="Dennis, Elizabeth S" /><meta name="citation_author_institution" content="CSIRO Agriculture and Food, Canberra, ACT, Australia" /><meta name="citation_author_institution" content="University of Technology, Sydney, Broadway, NSW, Australia" /><meta name="citation_author" content="Seki, Motoaki" /><meta name="citation_author_institution" content="RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan" /><meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology, Japan Science and Technology, Kawaguchi, Saitama, Japan" /><meta name="citation_author_institution" content="RIKEN Cluster for Pioneering Research, Wako, Saitama, Japan" /><meta name="citation_author" content="Fujimoto, Ryo" /><meta name="citation_author_institution" content="Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan" /><meta name="citation_title" content="Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in Brassica rapa L." /><meta name="citation_firstpage" content="511" /><meta name="citation_lastpage" content="520" /><meta name="citation_doi" content="10.1093/dnares/dsy021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Molecular mechanisms of epigenetic variation in plants;  citation_journal_title=Int. J. Mol. Sci;  citation_year=2012;  citation_volume=13;  citation_pages=9900-22; " /><meta name="citation_reference" content="citation_title=Plant transgenerational epigenetics;  citation_journal_title=Annu. Rev. Genet;  citation_year=2016;  citation_volume=50;  citation_pages=467-91; " /><meta name="citation_reference" content="citation_title=Chromosomal histone modification patterns-from conservation to diversity;  citation_journal_title=Trends Plant Sci;  citation_year=2006;  citation_volume=11;  citation_pages=199-208; " /><meta name="citation_reference" content="citation_title=Tug of war: adding and removing histone lysine methylation in Arabidopsis;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2016;  citation_volume=34;  citation_pages=41-53; " /><meta name="citation_reference" content="citation_title=Genome-wide association of histone H3 lysine nine methylation with CHG DNA methylation in Arabidopsis thaliana;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e3156" /><meta name="citation_reference" content="citation_title=DNA methylation pathways and their crosstalk with histone methylation;  citation_journal_title=Nat. Rev. Mol. Cell Biol;  citation_year=2015;  citation_volume=16;  citation_pages=519-32; " /><meta name="citation_reference" content="citation_title=Spontaneous epigenetic variation in the Arabidopsis thaliana methylome;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=480;  citation_pages=245-9; " /><meta name="citation_reference" content="citation_title=Transgenerational epigenetic instability is a source of novel methylation variants;  citation_journal_title=Science;  citation_year=2011;  citation_volume=334;  citation_pages=369-73; " /><meta name="citation_reference" content="citation_publisher=NOVA Publishers, U.S.A, New York, Hauppauge;  citation_title=DNA Methylation: Principles, Mechanisms and Challenges;  citation_year=2012;  citation_pages=35-66; " /><meta name="citation_reference" content="citation_title=The Arabidopsis nucleosome remodeler DDM1 allows DNA methyltransferases to access H1-containing heterochromatin;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=153;  citation_pages=193-205; " /><meta name="citation_reference" content="citation_title=Non-CG methylation patterns shape the epigenetic landscape in Arabidopsis;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2014;  citation_volume=21;  citation_pages=64-72; " /><meta name="citation_reference" content="citation_title=RNA-directed DNA methylation: an epigenetic pathway of increasing complexity;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=394-408; " /><meta name="citation_reference" content="citation_title=Principles and challenges of genome-wide DNA methylation analysis;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=191-203; " /><meta name="citation_reference" content="citation_title=Patterns of population epigenomic diversity;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=193-8; " /><meta name="citation_reference" content="citation_title=Epigenomic diversity in a global collection of Arabidopsis thaliana accessions;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=166;  citation_pages=492-505; " /><meta name="citation_reference" content="citation_title=Widespread natural variation of DNA methylation within angiosperms;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=194" /><meta name="citation_reference" content="citation_title=Methylome evolution in plants;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=264" /><meta name="citation_reference" content="citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis;  citation_journal_title=Cell;  citation_year=2008;  citation_volume=133;  citation_pages=523-36; " /><meta name="citation_reference" content="citation_title=Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=452;  citation_pages=215-9; " /><meta name="citation_reference" content="citation_title=Self-incompatibility;  citation_journal_title=Adv. Bot. Res;  citation_year=2007;  citation_volume=45;  citation_pages=139-54; " /><meta name="citation_reference" content="citation_title=Molecular and cellular characteristics of hybrid vigour in a commercial hybrid of Chinese cabbage;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=45" /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=Subgenome parallel selection is associated with morphotype diversification and convergent crop domestication in Brassica rapa and Brassica oleracea;  citation_journal_title=Nat. Genet;  citation_year=2016;  citation_volume=48;  citation_pages=1218-24; " /><meta name="citation_reference" content="citation_title=Biased gene fractionation and dominant gene expression among the subgenomes of Brassica rapa;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e36442" /><meta name="citation_reference" content="citation_title=Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy;  citation_journal_title=Genetics;  citation_year=2012;  citation_volume=190;  citation_pages=1563-74; " /><meta name="citation_reference" content="citation_title=Genetic distance of inbred lines of Chinese cabbage and its relationship to heterosis;  citation_journal_title=Plant Gene;  citation_year=2016;  citation_volume=5;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1571-2; " /><meta name="citation_reference" content="citation_title=Body-methylated genes in Arabidopsis thaliana are functionally important and evolve slowly;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=219-27; " /><meta name="citation_reference" content="citation_title=Development of primer sets that can verify the enrichment of histone modifications, and their application to examining vernalization-mediated chromatin changes in Brassica rapa L;  citation_journal_title=Genes Genet. Syst;  citation_year=2016;  citation_volume=91;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Identification of DNA methylated regions by using methylated DNA immunoprecipitation sequencing in Brassica rapa;  citation_journal_title=Crop Pasture Sci;  citation_year=2018;  citation_volume=69;  citation_pages=107-20; " /><meta name="citation_reference" content="citation_title=Identification of candidate genes for fusarium yellows resistance in Chinese cabbage by differential expression analysis;  citation_journal_title=Plant Mol. Biol;  citation_year=2014;  citation_volume=85;  citation_pages=247-57; " /><meta name="citation_reference" content="citation_title=A comparison of transcriptome and epigenetic status between closely related species in the genus Arabidopsis;  citation_journal_title=Gene;  citation_year=2012;  citation_volume=506;  citation_pages=301-9; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA methylation profiling by modified reduced representation bisulfite sequencing in Brassica rapa suggests that epigenetic modifications play a key role in polyploid genome evolution;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=836" /><meta name="citation_reference" content="citation_title=Hypomethylation and transcriptional reactivation of retrotransposon-like sequences in ddm1 transgenic plants of Brassica rapa;  citation_journal_title=Plant Mol. Biol;  citation_year=2008;  citation_volume=66;  citation_pages=463-73; " /><meta name="citation_reference" content="citation_title=Analysis of target sequences of DDM1s in Brassica rapa by MSAP;  citation_journal_title=Plant Cell Rep;  citation_year=2011;  citation_volume=30;  citation_pages=81-8; " /><meta name="citation_reference" content="citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in Arabidopsis;  citation_journal_title=Cell;  citation_year=2006;  citation_volume=126;  citation_pages=1189-201; " /><meta name="citation_reference" content="citation_title=Single-base resolution maps of cultivated and wild rice methylomes and regulatory roles of DNA methylation in plant gene expression;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=300" /><meta name="citation_reference" content="citation_title=Whole-genome DNA methylation patterns and complex associations with gene structure and expression during flower development in Arabidopsis;  citation_journal_title=Plant J;  citation_year=2015;  citation_volume=81;  citation_pages=268-81; " /><meta name="citation_reference" content="citation_title=DNA methylation of the first exon is tightly linked to transcriptional silencing;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e14524" /><meta name="citation_reference" content="citation_title=Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1663-74; " /><meta name="citation_reference" content="citation_title=Evolution of DNA methylation patterns in the Brassicaceae is driven by differences in genome organization;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004785" /><meta name="citation_reference" content="citation_title=Comparative sequence analysis for Brassica oleracea with similar sequences in B. rapa and Arabidopsis thaliana;  citation_journal_title=Plant Cell Rep;  citation_year=2009;  citation_volume=28;  citation_pages=649-61; " /><meta name="citation_reference" content="citation_title=Gene body methylation is conserved between plant orthologs and is of evolutionary consequence;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2013;  citation_volume=110;  citation_pages=1797-802; " /><meta name="citation_reference" content="citation_title=Evolutionary patterns of genic DNA methylation vary across land plants;  citation_journal_title=Nat. Plants;  citation_year=2016;  citation_volume=2;  citation_pages=15222" /><meta name="citation_reference" content="citation_title=On the origin and evolutionary consequences of gene body DNA methylation;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2016;  citation_volume=113;  citation_pages=9111-6; " /><meta name="citation_reference" content="citation_title=Locus-specific control of DNA methylation by the Arabidopsis SUVH5 histone methyltransferase;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1166-76; " /><meta name="citation_reference" content="citation_title=Control of CpNpG DNA methylation by the KRYPTONITE histone H3 methyltransferase;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=416;  citation_pages=556-60; " /><meta name="citation_reference" content="citation_title=Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=4903-17; " /><meta name="citation_reference" content="citation_title=Characterization of DNA methyltransferase genes in Brassica rapa;  citation_journal_title=Genes Genet. Syst;  citation_year=2006;  citation_volume=81;  citation_pages=235-42; " /><meta name="citation_reference" content="citation_title=Diversification and evolution of the SDG gene family in Brassica rapa after the whole genome triplication;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=16851" /><meta name="citation_reference" content="citation_title=Epigenetic chromatin modifications in Brassica genomes;  citation_journal_title=Genome;  citation_year=2010;  citation_volume=53;  citation_pages=203-10; " /><meta name="citation_reference" content="citation_title=Histone H3 methylation patterns in Brassica nigra, Brassica juncea, and Brassica carinata species;  citation_journal_title=Genome;  citation_year=2012;  citation_volume=55;  citation_pages=68-74; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/511/26076923/dsy021.pdf" /><meta name="description" content="Abstract.  Epigenetic gene regulation is crucial to plant life and can involve dynamic interactions between various histone modifications, DNA methylation, and " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/511/5047107" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/511/5047107" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/511/5047107");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy021';
    </script>
<title>
	Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in Brassica rapa L. | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5047107" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeqDATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWhhc2hpLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gQ2VudGVyIGZvciBTdXN0YWluYWJsZSBSZXNvdXJjZSBTY2llbmNlLCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT3NhYmUsIEtlbmppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgRXBpZ2VuZXRpY3MgVW5pdCwgT2tpbmF3YSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBHcmFkdWF0ZSBVbml2ZXJzaXR5LCBPbm5hLXNvbiwgT2tpbmF3YSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWt1c2hpbWEsIE5hb2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBLb2JlIFVuaXZlcnNpdHksIFJva2tvZGFpLCBOYWRhLWt1LCBLb2JlLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRha3VubywgU2hvaGVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFdm9sdXRpb25hcnkgU3R1ZGllcyBvZiBCaW9zeXN0ZW1zLCBTT0tFTkRBSSAoVGhlIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMpLCBIYXlhbWEsIEthbmFnYXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1peWFqaSwgTmFvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIEtvYmUgVW5pdmVyc2l0eSwgUm9ra29kYWksIE5hZGEta3UsIEtvYmUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dSwgTW90b2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSXdhdGUgQmlvdGVjaG5vbG9neSBSZXNlYXJjaCBDZW50ZXIsIE5hcml0YSBLaXRha2FtaSwgSXdhdGUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYXNha2ktWWFzdWRhLCBUYWtlc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBLb2JlIFVuaXZlcnNpdHksIFJva2tvZGFpLCBOYWRhLWt1LCBLb2JlLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgWXV0YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIEthc2hpd2EsIENoaWJhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRlbm5pcywgRWxpemFiZXRoIFMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDU0lSTyBBZ3JpY3VsdHVyZSBhbmQgRm9vZCwgQ2FuYmVycmEsIEFDVCwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5LCBTeWRuZXksIEJyb2Fkd2F5LCBOU1csIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNla2ksIE1vdG9ha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBDZW50ZXIgZm9yIFN1c3RhaW5hYmxlIFJlc291cmNlIFNjaWVuY2UsIFlva29oYW1hLCBLYW5hZ2F3YSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb3JlIFJlc2VhcmNoIGZvciBFdm9sdXRpb25hbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBKYXBhbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBLYXdhZ3VjaGksIFNhaXRhbWEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gQ2x1c3RlciBmb3IgUGlvbmVlcmluZyBSZXNlYXJjaCwgV2FrbywgU2FpdGFtYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppbW90bywgUnlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBLb2JlIFVuaXZlcnNpdHksIFJva2tvZGFpLCBOYWRhLWt1LCBLb2JlLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lLXdpZGUgY2hhcmFjdGVyaXphdGlvbiBvZiBETkEgbWV0aHlsYXRpb24sIHNtYWxsIFJOQSBleHByZXNzaW9uLCBhbmQgaGlzdG9uZSBIMyBseXNpbmUgbmluZSBkaS1tZXRoeWxhdGlvbiBpbiBCcmFzc2ljYSByYXBhIEwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1MjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtZWNoYW5pc21zIG9mIGVwaWdlbmV0aWMgdmFyaWF0aW9uIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBNb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTk5MDAtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHRyYW5zZ2VuZXJhdGlvbmFsIGVwaWdlbmV0aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NDY3LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBoaXN0b25lIG1vZGlmaWNhdGlvbiBwYXR0ZXJucy1mcm9tIGNvbnNlcnZhdGlvbiB0byBkaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xOTktMjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UdWcgb2Ygd2FyOiBhZGRpbmcgYW5kIHJlbW92aW5nIGhpc3RvbmUgbHlzaW5lIG1ldGh5bGF0aW9uIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIG9mIGhpc3RvbmUgSDMgbHlzaW5lIG5pbmUgbWV0aHlsYXRpb24gd2l0aCBDSEcgRE5BIG1ldGh5bGF0aW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTMxNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gcGF0aHdheXMgYW5kIHRoZWlyIGNyb3NzdGFsayB3aXRoIGhpc3RvbmUgbWV0aHlsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBNb2wuIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTUxOS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BvbnRhbmVvdXMgZXBpZ2VuZXRpYyB2YXJpYXRpb24gaW4gdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIG1ldGh5bG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ4MDsgIGNpdGF0aW9uX3BhZ2VzPTI0NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2dlbmVyYXRpb25hbCBlcGlnZW5ldGljIGluc3RhYmlsaXR5IGlzIGEgc291cmNlIG9mIG5vdmVsIG1ldGh5bGF0aW9uIHZhcmlhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTMzNDsgIGNpdGF0aW9uX3BhZ2VzPTM2OS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPU5PVkEgUHVibGlzaGVycywgVS5TLkEsIE5ldyBZb3JrLCBIYXVwcGF1Z2U7ICBjaXRhdGlvbl90aXRsZT1ETkEgTWV0aHlsYXRpb246IFByaW5jaXBsZXMsIE1lY2hhbmlzbXMgYW5kIENoYWxsZW5nZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl9wYWdlcz0zNS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFyYWJpZG9wc2lzIG51Y2xlb3NvbWUgcmVtb2RlbGVyIERETTEgYWxsb3dzIEROQSBtZXRoeWx0cmFuc2ZlcmFzZXMgdG8gYWNjZXNzIEgxLWNvbnRhaW5pbmcgaGV0ZXJvY2hyb21hdGluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTE5My0yMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbi1DRyBtZXRoeWxhdGlvbiBwYXR0ZXJucyBzaGFwZSB0aGUgZXBpZ2VuZXRpYyBsYW5kc2NhcGUgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02NC03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLWRpcmVjdGVkIEROQSBtZXRoeWxhdGlvbjogYW4gZXBpZ2VuZXRpYyBwYXRod2F5IG9mIGluY3JlYXNpbmcgY29tcGxleGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9Mzk0LTQwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbmNpcGxlcyBhbmQgY2hhbGxlbmdlcyBvZiBnZW5vbWUtd2lkZSBETkEgbWV0aHlsYXRpb24gYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE5MS0yMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIHBvcHVsYXRpb24gZXBpZ2Vub21pYyBkaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTU7ICBjaXRhdGlvbl9wYWdlcz0xOTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2Vub21pYyBkaXZlcnNpdHkgaW4gYSBnbG9iYWwgY29sbGVjdGlvbiBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBhY2Nlc3Npb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2NjsgIGNpdGF0aW9uX3BhZ2VzPTQ5Mi01MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVzcHJlYWQgbmF0dXJhbCB2YXJpYXRpb24gb2YgRE5BIG1ldGh5bGF0aW9uIHdpdGhpbiBhbmdpb3NwZXJtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xOTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRoeWxvbWUgZXZvbHV0aW9uIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yNjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgaW50ZWdyYXRlZCBzaW5nbGUtYmFzZSByZXNvbHV0aW9uIG1hcHMgb2YgdGhlIGVwaWdlbm9tZSBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz01MjMtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNob3RndW4gYmlzdWxwaGl0ZSBzZXF1ZW5jaW5nIG9mIHRoZSBBcmFiaWRvcHNpcyBnZW5vbWUgcmV2ZWFscyBETkEgbWV0aHlsYXRpb24gcGF0dGVybmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTIxNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxmLWluY29tcGF0aWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBCb3QuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTEzOS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuZCBjZWxsdWxhciBjaGFyYWN0ZXJpc3RpY3Mgb2YgaHlicmlkIHZpZ291ciBpbiBhIGNvbW1lcmNpYWwgaHlicmlkIG9mIENoaW5lc2UgY2FiYmFnZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIG1lc29wb2x5cGxvaWQgY3JvcCBzcGVjaWVzIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1Ymdlbm9tZSBwYXJhbGxlbCBzZWxlY3Rpb24gaXMgYXNzb2NpYXRlZCB3aXRoIG1vcnBob3R5cGUgZGl2ZXJzaWZpY2F0aW9uIGFuZCBjb252ZXJnZW50IGNyb3AgZG9tZXN0aWNhdGlvbiBpbiBCcmFzc2ljYSByYXBhIGFuZCBCcmFzc2ljYSBvbGVyYWNlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEyMTgtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpYXNlZCBnZW5lIGZyYWN0aW9uYXRpb24gYW5kIGRvbWluYW50IGdlbmUgZXhwcmVzc2lvbiBhbW9uZyB0aGUgc3ViZ2Vub21lcyBvZiBCcmFzc2ljYSByYXBhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTM2NDQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJlZCBwYXR0ZXJucyBvZiBmcmFjdGlvbmF0aW9uIGFuZCBleG9uIGRlbGV0aW9ucyBpbiBCcmFzc2ljYSByYXBhIHN1cHBvcnQgYSB0d28tc3RlcCBtb2RlbCBvZiBwYWxlb2hleGFwbG9pZHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjMtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGlzdGFuY2Ugb2YgaW5icmVkIGxpbmVzIG9mIENoaW5lc2UgY2FiYmFnZSBhbmQgaXRzIHJlbGF0aW9uc2hpcCB0byBoZXRlcm9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlzbWFyazogYSBmbGV4aWJsZSBhbGlnbmVyIGFuZCBtZXRoeWxhdGlvbiBjYWxsZXIgZm9yIEJpc3VsZml0ZS1TZXEgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE1NzEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qm9keS1tZXRoeWxhdGVkIGdlbmVzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGFyZSBmdW5jdGlvbmFsbHkgaW1wb3J0YW50IGFuZCBldm9sdmUgc2xvd2x5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yMTktMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHByaW1lciBzZXRzIHRoYXQgY2FuIHZlcmlmeSB0aGUgZW5yaWNobWVudCBvZiBoaXN0b25lIG1vZGlmaWNhdGlvbnMsIGFuZCB0aGVpciBhcHBsaWNhdGlvbiB0byBleGFtaW5pbmcgdmVybmFsaXphdGlvbi1tZWRpYXRlZCBjaHJvbWF0aW4gY2hhbmdlcyBpbiBCcmFzc2ljYSByYXBhIEw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIEdlbmV0LiBTeXN0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgRE5BIG1ldGh5bGF0ZWQgcmVnaW9ucyBieSB1c2luZyBtZXRoeWxhdGVkIEROQSBpbW11bm9wcmVjaXBpdGF0aW9uIHNlcXVlbmNpbmcgaW4gQnJhc3NpY2EgcmFwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBQYXN0dXJlIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxODsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY2FuZGlkYXRlIGdlbmVzIGZvciBmdXNhcml1bSB5ZWxsb3dzIHJlc2lzdGFuY2UgaW4gQ2hpbmVzZSBjYWJiYWdlIGJ5IGRpZmZlcmVudGlhbCBleHByZXNzaW9uIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNDctNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGFyaXNvbiBvZiB0cmFuc2NyaXB0b21lIGFuZCBlcGlnZW5ldGljIHN0YXR1cyBiZXR3ZWVuIGNsb3NlbHkgcmVsYXRlZCBzcGVjaWVzIGluIHRoZSBnZW51cyBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01MDY7ICBjaXRhdGlvbl9wYWdlcz0zMDEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgRE5BIG1ldGh5bGF0aW9uIHByb2ZpbGluZyBieSBtb2RpZmllZCByZWR1Y2VkIHJlcHJlc2VudGF0aW9uIGJpc3VsZml0ZSBzZXF1ZW5jaW5nIGluIEJyYXNzaWNhIHJhcGEgc3VnZ2VzdHMgdGhhdCBlcGlnZW5ldGljIG1vZGlmaWNhdGlvbnMgcGxheSBhIGtleSByb2xlIGluIHBvbHlwbG9pZCBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04MzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IeXBvbWV0aHlsYXRpb24gYW5kIHRyYW5zY3JpcHRpb25hbCByZWFjdGl2YXRpb24gb2YgcmV0cm90cmFuc3Bvc29uLWxpa2Ugc2VxdWVuY2VzIGluIGRkbTEgdHJhbnNnZW5pYyBwbGFudHMgb2YgQnJhc3NpY2EgcmFwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9NDYzLTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0YXJnZXQgc2VxdWVuY2VzIG9mIERETTFzIGluIEJyYXNzaWNhIHJhcGEgYnkgTVNBUDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz04MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBoaWdoLXJlc29sdXRpb24gbWFwcGluZyBhbmQgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiBETkEgbWV0aHlsYXRpb24gaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MTE4OS0yMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1iYXNlIHJlc29sdXRpb24gbWFwcyBvZiBjdWx0aXZhdGVkIGFuZCB3aWxkIHJpY2UgbWV0aHlsb21lcyBhbmQgcmVndWxhdG9yeSByb2xlcyBvZiBETkEgbWV0aHlsYXRpb24gaW4gcGxhbnQgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgRE5BIG1ldGh5bGF0aW9uIHBhdHRlcm5zIGFuZCBjb21wbGV4IGFzc29jaWF0aW9ucyB3aXRoIGdlbmUgc3RydWN0dXJlIGFuZCBleHByZXNzaW9uIGR1cmluZyBmbG93ZXIgZGV2ZWxvcG1lbnQgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0yNjgtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBvZiB0aGUgZmlyc3QgZXhvbiBpcyB0aWdodGx5IGxpbmtlZCB0byB0cmFuc2NyaXB0aW9uYWwgc2lsZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE0NTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2Vub21lLXdpZGUgaW5oZXJpdGFuY2Ugb2YgY3l0b3NpbmUgbWV0aHlsYXRpb24gdmFyaWFudHMgaW4gYSByZWNvbWJpbmFudCBpbmJyZWQgcG9wdWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTE2NjMtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBETkEgbWV0aHlsYXRpb24gcGF0dGVybnMgaW4gdGhlIEJyYXNzaWNhY2VhZSBpcyBkcml2ZW4gYnkgZGlmZmVyZW5jZXMgaW4gZ2Vub21lIG9yZ2FuaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA0Nzg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgc2VxdWVuY2UgYW5hbHlzaXMgZm9yIEJyYXNzaWNhIG9sZXJhY2VhIHdpdGggc2ltaWxhciBzZXF1ZW5jZXMgaW4gQi4gcmFwYSBhbmQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NjQ5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGJvZHkgbWV0aHlsYXRpb24gaXMgY29uc2VydmVkIGJldHdlZW4gcGxhbnQgb3J0aG9sb2dzIGFuZCBpcyBvZiBldm9sdXRpb25hcnkgY29uc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xNzk3LTgwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHBhdHRlcm5zIG9mIGdlbmljIEROQSBtZXRoeWxhdGlvbiB2YXJ5IGFjcm9zcyBsYW5kIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE1MjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIG9yaWdpbiBhbmQgZXZvbHV0aW9uYXJ5IGNvbnNlcXVlbmNlcyBvZiBnZW5lIGJvZHkgRE5BIG1ldGh5bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9OTExMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb2N1cy1zcGVjaWZpYyBjb250cm9sIG9mIEROQSBtZXRoeWxhdGlvbiBieSB0aGUgQXJhYmlkb3BzaXMgU1VWSDUgaGlzdG9uZSBtZXRoeWx0cmFuc2ZlcmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTExNjYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2wgb2YgQ3BOcEcgRE5BIG1ldGh5bGF0aW9uIGJ5IHRoZSBLUllQVE9OSVRFIGhpc3RvbmUgSDMgbWV0aHlsdHJhbnNmZXJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTY7ICBjaXRhdGlvbl9wYWdlcz01NTYtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY2Vzc2libGUgRE5BIGFuZCByZWxhdGl2ZSBkZXBsZXRpb24gb2YgSDNLOW1lMiBhdCBtYWl6ZSBsb2NpIHVuZGVyZ29pbmcgUk5BLWRpcmVjdGVkIEROQSBtZXRoeWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQ5MDMtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgRE5BIG1ldGh5bHRyYW5zZmVyYXNlIGdlbmVzIGluIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIEdlbmV0LiBTeXN0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9MjM1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpZmljYXRpb24gYW5kIGV2b2x1dGlvbiBvZiB0aGUgU0RHIGdlbmUgZmFtaWx5IGluIEJyYXNzaWNhIHJhcGEgYWZ0ZXIgdGhlIHdob2xlIGdlbm9tZSB0cmlwbGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNjg1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgY2hyb21hdGluIG1vZGlmaWNhdGlvbnMgaW4gQnJhc3NpY2EgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MjAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b25lIEgzIG1ldGh5bGF0aW9uIHBhdHRlcm5zIGluIEJyYXNzaWNhIG5pZ3JhLCBCcmFzc2ljYSBqdW5jZWEsIGFuZCBCcmFzc2ljYSBjYXJpbmF0YSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz02OC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzUvNTExLzI2MDc2OTIzL2RzeTAyMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRXBpZ2VuZXRpYyBnZW5lIHJlZ3VsYXRpb24gaXMgY3J1Y2lhbCB0byBwbGFudCBsaWZlIGFuZCBjYW4gaW52b2x2ZSBkeW5hbWljIGludGVyYWN0aW9ucyBiZXR3ZWVuIHZhcmlvdXMgaGlzdG9uZSBtb2RpZmljYXRpb25zLCBETkEgbWV0aHlsYXRpb24sIGFuZCAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwNDcxMDcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkCbKOf/+f7o3HmFn1vWCQf329HGI=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f511%2f5047107" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f511%2f5047107" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122670911">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670911">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670913">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670913">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670920">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670920">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670937">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670937">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670964">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670964">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670981">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670981">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670987">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670987">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/499/5045548">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/5/521/5056130">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in <em>Brassica rapa</em> L.
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoshi Takahashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Osabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Plant Epigenetics Unit, Okinawa Institute of Science and Technology Graduate University, Onna-son, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Osabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Osabe K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Osabe K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenji Osabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Fukushima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Fukushima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukushima N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukushima N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naoki Fukushima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shohei Takuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolutionary Studies of Biosystems, SOKENDAI (The Graduate University for Advanced Studies), Hayama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shohei+Takuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takuno S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takuno S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shohei Takuno</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naomi Miyaji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naomi+Miyaji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyaji N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyaji N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naomi Miyaji</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoki Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Iwate Biotechnology Research Center, Narita Kitakami, Iwate, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoki+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Motoki Shimizu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Takasaki-Yasuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Takasaki-Yasuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takasaki-Yasuda T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takasaki-Yasuda T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takeshi Takasaki-Yasuda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elizabeth S Dennis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CSIRO Agriculture and Food, Canberra, ACT, Australia</div><div class="aff">University of Technology, Sydney, Broadway, NSW, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elizabeth+S+Dennis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dennis E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dennis E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Elizabeth S Dennis</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoaki Seki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">RIKEN Center for Sustainable Resource Science, Yokohama, Kanagawa, Japan</div><div class="aff">Core Research for Evolutional Science and Technology, Japan Science and Technology, Kawaguchi, Saitama, Japan</div><div class="aff">RIKEN Cluster for Pioneering Research, Wako, Saitama, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoaki+Seki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seki M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seki M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Motoaki Seki</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryo Fujimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Agricultural Science, Kobe University, Rokkodai, Nada-ku, Kobe, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy021-cor1" data-open="dsy021-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 78 803 5827. Fax. +81 78 803 5827. Email: <a href="mailto:leo@people.kobe-u.ac.jp" target="_blank">leo@people.kobe-u.ac.jp</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-4690-1739">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0003-4690-1739
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryo+Fujimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujimoto R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujimoto R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ryo Fujimoto</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy021-FM1"><div class="fn"><div class="footnote-content">Satoshi Takahashi, Kenji Osabe and Naoki Fukushima authors equally contributed to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 511–520, <a href='https://doi.org/10.1093/dnares/dsy021'>https://doi.org/10.1093/dnares/dsy021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 June 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 March 2018</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 May 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5047107" href="/dnaresearch/article-pdf/25/5/511/26076923/dsy021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Satoshi Takahashi, Kenji Osabe, Naoki Fukushima, Shohei Takuno, Naomi Miyaji, Motoki Shimizu, Takeshi Takasaki-Yasuda, Yutaka Suzuki, Elizabeth S Dennis, Motoaki Seki, Ryo Fujimoto;  Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in <em>Brassica rapa</em> L., <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 511–520, <a href='https://doi.org/10.1093/dnares/dsy021'>https://doi.org/10.1093/dnares/dsy021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5047107?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20characterization%20of%20DNA%20methylation%2C%20small%20RNA%20expression%2C%20and%20histone%20H3%20lysine%20nine%20di-methylation%20in%20%26lt%3Bem%26gt%3BBrassica%20rapa%26lt%3B%2Fem%26gt%3B%20L.&amp;publicationDate=2018-06-29&amp;volumeNum=25&amp;issueNum=5&amp;author=Takahashi%2C%20Satoshi%3B%20Osabe%2C%20Kenji&amp;startPage=511&amp;endPage=520&amp;contentId=10.1093%2Fdnares%2Fdsy021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f511%2f5047107" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122670911 id="122670911" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Epigenetic gene regulation is crucial to plant life and can involve dynamic interactions between various histone modifications, DNA methylation, and small RNAs. Detailed analysis of epigenome information is anticipated to reveal how the DNA sequence of the genome is translated into the plant’s phenotype. The aim of this study was to map the DNA methylation state at the whole genome level and to clarify the relationship between DNA methylation and transcription, small RNA expression, and histone H3 lysine 9 di-methylation (H3K9me2) in <em>Brassica rapa</em>. We performed whole genome bisulfite sequencing, small RNA sequencing, and chromatin immunoprecipitation sequencing using H3K9me2 antibody in a Chinese cabbage inbred line, RJKB-T24, and examined the impact of epigenetic states on transcription. Cytosine methylation in DNA was analysed in different sequence contexts (CG, CHG, and CHH) (where H could be A, C, or T) and position (promoter, exon, intron, terminator, interspersed repeat regions), and the H3K9me2 and 24 nucleotide small interfering RNAs (24 nt-siRNA) were overlaid onto the <em>B. rapa</em> reference genome. The epigenome was compared with that of <em>Arabidopsis thaliana</em> and the relationship between the position of DNA methylation and gene expression, and the involvement of 24 nt siRNAs and H3K9me2 are discussed.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;histone H3 lysine 9 di-methylation&quot;">histone H3 lysine 9 di-methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;small RNA&quot;">small RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetic">epigenetic</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Brassica">Brassica</a></div>                            <h2 scrollto-destination=122670913 id="122670913" class="section-title" >1. Introduction</h2>
<p>Genetic variation can cause phenotypic variation in organisms; due to the changes in the structure of an encoded protein or gene expression level by mutations of the DNA sequence. In addition to genetic variation, gene expression can be controlled by changes in the structure of chromatin without changing the DNA sequence, and this phenomenon is termed ‘epigenetic’ control. DNA methylation and histone modification are well-known epigenetic modifications.<a reveal-id="dsy021-B1" data-open="dsy021-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy021-B2" data-open="dsy021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a></p><p>Nucleosomes are formed by a histone octamer containing two of the core histones H2A, H2B, H3, and H4, and 147 bp of DNA is wrapped around this core. The N-terminal regions of histone proteins are subject to various chemical modifications, such as methylation or acetylation, and these histone modifications are associated with gene transcription. Different histone marks can be controlled by different histone methyltransferase and can lead to different effects on gene regulation.<a reveal-id="dsy021-B3" data-open="dsy021-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy021-B4" data-open="dsy021-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In <em>Arabidopsis thaliana</em>, histone H3 lysine 9 di-methylation (H3K9me2) is associated with heterochromatic regions,<a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and histone lysine methyltransferases, KRYPTONITE (KYP)/SU(VAR)3-9 HOMOLOG 4 (SUVH4), SUVH5, and SUVH6, catalyze addition of H3K9me2.<a reveal-id="dsy021-B6" data-open="dsy021-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p><p>In plants, cytosines in all contexts, CG, CHG, and CHH (H is A, T, or C), can be methylated. Spontaneous epimutation is defined as heritable stochastic changes in the methylation states at CG, CHG, and CHH sites, and the rate of epimutation is overwhelmingly higher than the rate of genetic mutations in <em>A. thaliana.</em><a reveal-id="dsy021-B7" data-open="dsy021-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy021-B8" data-open="dsy021-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Sometimes, epimutation becomes the driving force of phenotypic variation.<a reveal-id="dsy021-B1" data-open="dsy021-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy021-B2" data-open="dsy021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> DNA methylation has an important role in the regulation of gene expression, silencing of repeat sequences and transposons, and genome imprinting.<a reveal-id="dsy021-B2" data-open="dsy021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy021-B9" data-open="dsy021-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> DNA methylation in CG contexts is largely maintained by METHYLTRANSFERASE I (MET1), and those in CHG contexts are largely maintained by CHROMOMETHYLASE 3 (CMT3)-associated with H3K9me2.<a reveal-id="dsy021-B2" data-open="dsy021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy021-B6" data-open="dsy021-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy021-B9" data-open="dsy021-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> CHH site methylation is maintained by CMT2 or DOMAINS REARRANGED METHYLTRANSFERASE 2 (DRM2).<a reveal-id="dsy021-B10" data-open="dsy021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy021-B11" data-open="dsy021-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The <em>de novo</em> methylation in all contexts is catalyzed by DRM2 and is triggered by 24 nucleotide small-interfering RNAs (24 nt-siRNAs) produced by the RNA interference (RNAi) pathway, termed RNA-directed DNA methylation (RdDM). Two plant-specific RNA polymerases, Polymerase IV (Pol IV) and Pol V, RNA-DEPENDENT RNA POLYMERASE 2 (RDR2), DICER-LIKE 3 (DCL3), and ARGONAUTE 4 (AGO4) proteins function in this RNAi pathway.<a reveal-id="dsy021-B2" data-open="dsy021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy021-B12" data-open="dsy021-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>Genome-wide profiles of epigenetic information (the epigenome) are available in plants using technologies such as tiling arrays or high-throughput sequencing. High-resolution maps of epigenetic features have been obtained from whole genome bisulfite sequencing (WGBS) (bisulfite converted DNA is directly sequenced) or a combination of chromatin immunoprecipitation (ChIP) technology and high-throughput sequencing (ChIP-seq).<a reveal-id="dsy021-B4" data-open="dsy021-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy021-B13" data-open="dsy021-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Using these technologies, variations of DNA methylation states between accessions in a specific species or between species have been shown at the whole genome level.<a reveal-id="dsy021-B14 dsy021-B15 dsy021-B16 dsy021-B17" data-open="dsy021-B14 dsy021-B15 dsy021-B16 dsy021-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> In <em>A. thaliana</em>, DNA methylation is enriched in heterochromatic regions of genomes, which consist of repetitive sequences and transposable elements (TEs).<a reveal-id="dsy021-B18" data-open="dsy021-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy021-B19" data-open="dsy021-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> DNA methylation is also observed in the euchromatic regions, and genes having DNA methylation at CG sites in transcribed regions, termed gene-body methylation (gbM), showed high expression level.<a reveal-id="dsy021-B18" data-open="dsy021-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy021-B19" data-open="dsy021-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p><em>Brassica rapa</em> L. comprises commercially important vegetable crops consumed worldwide including Chinese cabbage (var. <em>pekinensis</em>), pak choi (var. <em>chinensis</em>), and komatsuna (var. <em>perviridis</em>), root vegetables including turnip (var. <em>rapa</em>), and oilseed (var. <em>oleifera</em>). Most cultivars of <em>B. rapa</em> are self-incompatible, preventing self-fertilization, and are F<sub>1</sub> hybrid cultivars, which have increased yields relative to their parents (heterosis/hybrid vigour).<a reveal-id="dsy021-B20" data-open="dsy021-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsy021-B21" data-open="dsy021-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Chinese cabbage forms a head with large pale-green-coloured leaves and wide white midribs and is an important vegetable in Asia. The draft genome sequence of Chinese cabbage (Chiifu-401-42) has been released and 199 accessions in <em>B. rapa</em> representing various morphotypes have been re-sequenced.<a reveal-id="dsy021-B22" data-open="dsy021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy021-B23" data-open="dsy021-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The <em>B. rapa</em> genome has undergone a whole-genome triplication (WGT), which results in multiple copies of paralogous genes. Three subgenomes, the least fractioned (LF) subgenome and two more fractionated subgenomes (MF1 and MF2), were found in <em>B. rapa</em> genome, and biased gene fractionation and dominant gene expression pattern among the subgenomes were shown.<a reveal-id="dsy021-B22" data-open="dsy021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy021-B24" data-open="dsy021-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsy021-B25" data-open="dsy021-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p><p>The aim of this study was to identify the DNA methylation states at the whole genome levels and to clarify the relationship between DNA methylation and transcription, small RNA expression, and H3K9me2 modification in <em>B. rapa</em>. <em>B. rapa</em> is related to the model plant <em>A. thaliana</em>, which enables us to compare the conservation and diversification of epigenetic states between orthologous genes of these species. In addition, <em>B. rapa</em> has more repetitive sequences than <em>A. thaliana</em> and has experienced WGT after speciation, which allows us to examine the effect of differences in genome composition on epigenetic states. Furthermore, the knowledge of the DNA methylation state, small RNA expression, and H3K9me2 placement at the whole genome level will be useful for examining natural variation, change of epigenetic states by abiotic or biotic stress, and understanding epigenetic states related to agronomically important traits using segregation of loci.</p>                            <h2 scrollto-destination=122670920 id="122670920" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122670921 id="122670921" class="section-title" >2.1. Plant materials and growth conditions</h3>
<p>A Chinese cabbage inbred line developed in a previous study, RJKB-T24, was used as plant material.<a reveal-id="dsy021-B26" data-open="dsy021-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Plants were grown on agar-solidified Murashige and Skoog (MS) plates with 1.0% (w/v) sucrose under long day condition (16 h light) at 21°C. For WGBS, small RNA sequencing (small RNA-seq), and ChIP analyses, first and second leaves were harvested from plants at 14 days after sowing.</p>                            <h3 scrollto-destination=122670923 id="122670923" class="section-title" >2.2. Whole genome bisulfite sequencing (WGBS)</h3>
<p>Genomic DNAs from 14 days first and second leaves were isolated using DNeasy Plant Mini Kit (Qiagen). Genomic DNAs were fragmented to 100–300 bp length by sonication. DNA-end repair, 3′-dA overhang, ligation of methylated sequence adaptors, and bisulfite treatment by ZYMO EZ DNA Methylation-Gold kit (ZYMO RESEARCH) were performed for constructing bisulfite sequence library. The library was sequenced using Illumina Hiseq2000 (100 bp paired end).</p><p>The reads of WGBS were mapped to the <em>B. rapa</em> reference genome v.1.5 (<a class="link link-uri" href="http://brassicadb.org/brad/" target="_blank">http://brassicadb.org/brad/</a>) or to RJKB-T24 re-sequenced genome, whose SNPs were replaced in the reference genome, using Bowtie2 version 2.2.3 and Bismark v0.14.3.<a reveal-id="dsy021-B27" data-open="dsy021-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> In order to estimate methylation levels of CG, CHG, and CHH contexts, the numbers of methylated and unmethylated reads were extracted on each cytosine position using bismark_methylation_extractor script with the paired-end parameter. The methylation level at each cytosine position was calculated by dividing the number of methylated cytosines (mC) reads by the total reads.</p>                            <h3 scrollto-destination=122670926 id="122670926" class="section-title" >2.3. Classification of gene-body methylation (gbM)</h3>
<p>We classified cytosine sites into methylated and unmethylated cytosines by employing a binomial test, following Lister <em>et al.</em><a reveal-id="dsy021-B18" data-open="dsy021-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> We estimated the sequence context-specific error rates of bisulfite conversion (0.0173 for CG, 0.0178 for CHG, and 0.0215 for CHH sites) from the mapping result of the unmethylated phage genome, and used these values for the binomial test. We applied the test only to the cytosine site with &gt;2 WGBS coverage, and used a significant cut-off of two-tailed <em>P</em>-value &lt; 0.01 to detect mC.</p><div class="block-child-p">We quantified the level of DNA methylation for each protein-coding region that was defined as the region from start to stop codons, which includes both exons and introns. We screened genes for which ≥60% of all cytosine residues and ≥20 cytosines in the CG context have &gt;2 aligned WGBS reads, and calculated CG, CHG, and CHH methylation levels for these genes. We further classified genes into body- and under-methylated genes, following Takuno and Gaut<a reveal-id="dsy021-B28" data-open="dsy021-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> with a slight modification. We calculated the number of cytosine residues with &gt;2 coverage at CG sites (<em>n</em><sub>CG</sub>) and the number of mC residues at CG sites (<em>m</em><sub>CG</sub>) for each gene. We calculated <em>P</em>-values (<em>P</em><sub>CG</sub>) for the departure of CG methylation levels from the average CG methylation level across all genes (<em>p</em><sub>CG</sub>): <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>P</mi><mrow><mtext>CG</mtext></mrow></msub><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><msub><mi>m</mi><mrow><mtext>CG</mtext></mrow></msub></mrow><mrow><msub><mi>n</mi><mrow><mtext>CG</mtext></mrow></msub></mrow></munderover><mrow><mrow><mo stretchy="true">(</mo><mrow><msubsup><mrow></mrow><mi>i</mi><mrow><msub><mi>n</mi><mrow><mtext>CG</mtext></mrow></msub></mrow></msubsup></mrow><mo stretchy="true">)</mo></mrow></mrow></mstyle><msubsup><mi>p</mi><mrow><mtext>CG</mtext></mrow><mi>i</mi></msubsup><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mrow><mtext>CG</mtext></mrow></msub></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><msub><mi>n</mi><mrow><mtext>CG</mtext></mrow></msub><mo>−</mo><mi>i</mi></mrow></msup></mrow></math></div></div></div></div><p>The smaller <em>P</em><sub>CG</sub> is, the higher CG methylation level is. We defined body- and under-methylated genes as genes with <em>P</em><sub>CG</sub> &lt; 0.05 and <em>P</em><sub>CG</sub> &gt; 0.95, respectively. Using the same rationale, we calculated <em>P</em><sub>CHG</sub> and <em>P</em><sub>CHH</sub> for CHG and CHH methylation, respectively.</p>                            <h3 scrollto-destination=122670930 id="122670930" class="section-title" >2.4. Small RNA sequencing (small RNA-seq)</h3>
<p>Total RNA was isolated from 14 days first and second leaves of <em>B. rapa</em> using TRIzol Reagent (Thermo Fisher) with DNase I treatments. Small RNAs of 17–30 nucleotide (nt) were purified. The sequence library was prepared using Oligonucleotide sequences for TruSeq<sup>TM</sup> Small RNA Sample Prep kits (Illumina), and small RNA-seq was performed using Illumina Hiseq2000 (36 bp single end).</p><p>The reads of small RNA-seq were purged from low quality reads or adapter sequences using cutadapt version 1.7.1 and Trim Galore! version 0.3.7. Then the reads were mapped to the <em>B. rapa</em> reference genome v.1.5 using Bowtie2 version 2.2.3. To examine the size distribution of mapped small RNAs, we classified the alignment reads by length. The 24 nt aligned reads were used to identify 24 nt-siRNA clusters by clustering adjacent reads using bedtools 2.15.0 with the ‘merge –d 200 (reads within 200 nt from each other were merged in one cluster)’ parameter. Clusters that contain at least five reads were picked up.</p>                            <h3 scrollto-destination=122670933 id="122670933" class="section-title" >2.5. Chromatin immunoprecipitation sequencing (ChIP-seq)</h3>
<p>ChIP experiments were performed as described by Kawanabe <em>et al.</em><a reveal-id="dsy021-B29" data-open="dsy021-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> One gram of 14 days first and second leaves of <em>B. rapa</em> was used for ChIP analysis and anti-H3K9me2 (ACTIVE MOTIF, 39753) antibody was used. Before the ChIP-seq, we validated the enrichment of purified immunoprecipitated DNAs by qPCR using the positive and negative control primer sets of H3K9me2 previously developed.<a reveal-id="dsy021-B29" data-open="dsy021-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Purified immunoprecipitated DNAs and input DNA were sequenced by Hiseq2000 (36 bp single end). The reads of ChIP-seq were purged from low quality reads or adapter sequences using cutadapt version 1.7.1 and Trim Galore! version 0.3.7. Then the reads were mapped to the <em>B. rapa</em> reference genome v.1.5 using Bowtie2 version 2.2.3. We performed peak calling on alignment results using model-based analysis for ChIP-seq (MACS) 2 2.1.0 and identified the regions having H3K9me2 as peaks. Primer sequences used in this study are shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h3 scrollto-destination=122670935 id="122670935" class="section-title" >2.6. Bisulfite sequencing</h3>
<p>Genomic DNAs from 14 days first and second leaves was isolated using DNeasy Plant Mini Kit (Qiagen). A total of 500 ng DNA was fragmented by sonication and fragments were approximately 300–800 bp in length. MethylCode Bisulfite Conversion Kit (Thermo Fisher) was used for chemical bisulfite reaction and PCR was performed using bisulfite treated DNAs as templates. PCR conditions were 95 °C for 2 min followed by 40 cycles of 95 °C for 30 s, 55 °C for 30 s, and 72 °C for 30 s. Amplified PCR fragments were gel purified using GENECLEAN III Kit (MP Biomedicals) and cloned into pGEM-T Easy vector (Promega). Ten independent clones were sequenced. Primers used for bisulfite sequencing are listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h2 scrollto-destination=122670937 id="122670937" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=122670938 id="122670938" class="section-title" >3.1. Whole genome DNA methylation states of RJKB-T24</h3>
<p>We performed WGBS using 14-day old first and second leaves of the Chinese cabbage (<em>B. rapa</em>) inbred line, RJKB-T24. Approximately 192 M clear reads were obtained and 70 M reads were uniquely mapped on the <em>B. rapa</em> reference genome (from chromosome 1 to 10) using Bismark, about 27 times the depth of the sequenced genome. Approximately 86% of cytosine sites in the regions from A01 to A10 were covered (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Most of the methylated CG sites were heavily methylated, while methylated CHG and CHH sites had low methylation levels (<a reveal-id="dsy021-F1" data-open="dsy021-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). A total of 70% of methyl-CG sites were fully methylated (<a reveal-id="dsy021-F1" data-open="dsy021-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>), which is higher than that in <em>A. thaliana</em> (20–30%).<a reveal-id="dsy021-B18" data-open="dsy021-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The average of methylation levels of the whole genome (total) for CG, CHG, and CHH sites were 36.5%, 13.4%, and 5.3%, respectively, and interspersed repeat regions (IRRs) were heavily methylated (CG, 73.7%; CHG, 33.8%; CHH, 13.0%) (<a reveal-id="dsy021-F1" data-open="dsy021-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). These methylation levels were slightly lower but similar to those seen using the reference genome with scaffold sequences or RJKB-T24 re-sequenced genome as a template of mapped reads (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), indicating that sequence polymorphisms between RJKB-T24 and the reference line did not greatly affect the quantification of the DNA methylation levels. We validated the WGBS results in the five selected regions by bisulfite sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). </p>                        <a id="122670940" scrollto-destination="122670940"></a>
<div id="dsy021-F1" data-id="dsy021-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f1.png?Expires=2147483647&Signature=lFuE1nrUpqck3oWvHIt5NGodLRsruhTTllogskjLMYAYzpG~Oqov0XE7e84mmMr5b-WdG-7voouosQF3JKvKQd3RMS6GCXTe7zh5QAggns1v3TSA-vPFUi-pwzaU2Xb1tukAbK9EC5uHngv2e5Hs4Jb8pGRBwSfE3SU~yXaDn~BCdtAuX4TMa2xhbH~NmsCDcNFJaVYkjsEauiSKw42n2iIur3v8Gqoc1PB~wTPO62cmY5OwmKaPSTAXeCdWPsRPTItbBRx9JaGEReP--KYySCDvYHBmWsbiwhuahofPtmwtvV3EiIHqgN9I0Q6fPCi3TSCLFtVftetR2x3HQLG2fA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation level detected by WGBS. (A) Distribution of the percentage of methylation site at each context. (B) Average of DNA methylation levels in the 2-kb upstream (2k-up), exon, intron, and 2-kb downstream (2k-down) regions, and interspersed repeats regions (IRRs). ‘Total’ represents the average of DNA methylation levels in all regions of the genome." path-from-xml="dsy021f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670940/dsy021f1.tif" path-from-xml="dsy021f1.tif" target="_blank">View large</a><a section="122670940" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f1.png?Expires=2147483647&Signature=A9XjuOiQyqVIOk4Om77fywBVXMWTWfvYppK-EyxZkEGNBCP~haP5YL3c8l6sHhpny7nGWcvYA~wIjhQOrZw9HDUVYOYrBe9-c5KT7Jggxx7sR-zSm2xdl3O3fNoO-JXHyvoLiate8T-X4ZbdIR4ntfq4YDvmOew27obknATHo9l~KDBXDx9s~n~vhPeYx9Pe8r2G2M~LVqFtqObrLiQI9Fh-NFtvLsf~lMoB5kOy5g70K3NmMh0krKuUxq0168NykZssJRec3TenED8kazJ1bXj68U3VrWFlJm0zKTuCnekmnl4tWGk1cADsPwtV5XDQwhWOLuBNyAq6kXbgZ9YiTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670940&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation level detected by WGBS. (A) Distribution of the percentage of methylation site at each context. (B) Average of DNA methylation levels in the 2-kb upstream (2k-up), exon, intron, and 2-kb downstream (2k-down) regions, and interspersed repeats regions (IRRs). ‘Total’ represents the average of DNA methylation levels in all regions of the genome.</p></div></div><div content-id="dsy021-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f1.png?Expires=2147483647&Signature=lFuE1nrUpqck3oWvHIt5NGodLRsruhTTllogskjLMYAYzpG~Oqov0XE7e84mmMr5b-WdG-7voouosQF3JKvKQd3RMS6GCXTe7zh5QAggns1v3TSA-vPFUi-pwzaU2Xb1tukAbK9EC5uHngv2e5Hs4Jb8pGRBwSfE3SU~yXaDn~BCdtAuX4TMa2xhbH~NmsCDcNFJaVYkjsEauiSKw42n2iIur3v8Gqoc1PB~wTPO62cmY5OwmKaPSTAXeCdWPsRPTItbBRx9JaGEReP--KYySCDvYHBmWsbiwhuahofPtmwtvV3EiIHqgN9I0Q6fPCi3TSCLFtVftetR2x3HQLG2fA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation level detected by WGBS. (A) Distribution of the percentage of methylation site at each context. (B) Average of DNA methylation levels in the 2-kb upstream (2k-up), exon, intron, and 2-kb downstream (2k-down) regions, and interspersed repeats regions (IRRs). ‘Total’ represents the average of DNA methylation levels in all regions of the genome." path-from-xml="dsy021f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670940/dsy021f1.tif" path-from-xml="dsy021f1.tif" target="_blank">View large</a><a section="122670940" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f1.png?Expires=2147483647&Signature=A9XjuOiQyqVIOk4Om77fywBVXMWTWfvYppK-EyxZkEGNBCP~haP5YL3c8l6sHhpny7nGWcvYA~wIjhQOrZw9HDUVYOYrBe9-c5KT7Jggxx7sR-zSm2xdl3O3fNoO-JXHyvoLiate8T-X4ZbdIR4ntfq4YDvmOew27obknATHo9l~KDBXDx9s~n~vhPeYx9Pe8r2G2M~LVqFtqObrLiQI9Fh-NFtvLsf~lMoB5kOy5g70K3NmMh0krKuUxq0168NykZssJRec3TenED8kazJ1bXj68U3VrWFlJm0zKTuCnekmnl4tWGk1cADsPwtV5XDQwhWOLuBNyAq6kXbgZ9YiTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670940&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation level detected by WGBS. (A) Distribution of the percentage of methylation site at each context. (B) Average of DNA methylation levels in the 2-kb upstream (2k-up), exon, intron, and 2-kb downstream (2k-down) regions, and interspersed repeats regions (IRRs). ‘Total’ represents the average of DNA methylation levels in all regions of the genome.</p></div></div><p>Previously, we examined DNA methylation states by methylated DNA immunoprecipitation sequencing (MeDIP-seq) using the same plant material and tissues but independently harvested.<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> We compared the results from the two methods, WGBS and MeDIP-seq, using genes having DNA methylation in the genic regions. More genes were detected by WGBS than by MeDIP-seq, and about 90% of genes having MeDIP-peaks in the 2-kb upstream, exon, intron, and 2-kb downstream regions overlapped with genes having DNA methylation detected by WGBS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The correlation coefficient between the methylation levels of all cytosine sites detected by WGBS and methylation levels detected by MeDIP-seq [log 10 score of reads per kilobase of exon per million mapped reads (RPKM), RPKM &gt; 0.01] in genic regions (200-bp upstream/exon/intron/200 bp-downstream regions) was high (<em>r </em>=<em> </em>0.76, <em>P </em>&lt;<em> </em>10e-6), indicating that WGBS data is consistent with the MeDIP-seq data. These results indicate that MeDIP-seq detected highly methylated regions, while WGBS can examine all DNA methylation contexts including the regions detected by MeDIP-seq analysis.</p>                            <h3 scrollto-destination=122670942 id="122670942" class="section-title" >3.2. Genes having DNA methylation and their expression levels</h3>
<p>We examined the impact of DNA methylation on transcription by comparing the DNA methylation data to previous RNA sequencing (RNA-seq) data with two replicates generated from samples using the same tissue and stage, but independently harvested.<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy021-B31" data-open="dsy021-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The correlation coefficient between the CG, CHG, and CHH methylation levels at specific genic regions (2-kb, 500-bp, and 200-bp upstream and downstream regions, first exon, exon, first intron, and intron) and the expression levels (log 2 score of FPKM, FPKM &gt; 0.01) was examined. There was a negative correlation between methylation levels and expression levels, except for CG methylation in exon regions, CHG methylation in the 2-kb upstream region, and CHH methylation in the 500-bp and 2-kb upstream regions (<a reveal-id="dsy021-T1" data-open="dsy021-T1" class="link link-reveal link-table xref-fig">Table 1</a>). CG and CHG methylation levels in the 200-bp upstream and 500-bp downstream regions and CG, CHG, and CHH methylation levels in the 200-bp downstream regions correlated with lower expression levels. CG, CHG, and CHH methylation levels in the first exon and first intron also showed negative correlation with expression levels. In summary, there were negative correlations between CHG and CHH methylation levels of the exon and intron regions and expression levels, whereas no negative correlation was found between the expression level and CG methylation levels of the exon and intron region (<a reveal-id="dsy021-T1" data-open="dsy021-T1" class="link link-reveal link-table xref-fig">Table 1</a>). <div content-id="dsy021-T1" class="table-modal"><div class="table-wrap"><div id="dsy021-T1" data-id="dsy021-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Correlation coefficients between DNA methylation levels and expression levels</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>  </th><th colspan="2">CG<hr /> </th><th colspan="2">CHG<hr /> </th><th colspan="2">CHH<hr /> </th></tr><tr><th>  </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th></tr></thead><tbody><tr><td>2-kb upstream </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.002 </td><td>NS </td><td>0.016 </td><td>&lt;0.05 </td></tr><tr><td>500-bp upstream </td><td>−0.081 </td><td>&lt;1.0e-5 </td><td>−0.043 </td><td>&lt;1.0e-5 </td><td>−0.009 </td><td>NS </td></tr><tr><td>200-bp upstream </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.087 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>First exon </td><td>−0.123 </td><td>&lt;1.0e-5 </td><td>−0.108 </td><td>&lt;1.0e-5 </td><td>−0.078 </td><td>&lt;1.0e-5 </td></tr><tr><td>Exon </td><td>−0.012 </td><td>NS </td><td>−0.137 </td><td>&lt;1.0e-5 </td><td>−0.129 </td><td>&lt;1.0e-5 </td></tr><tr><td>First intron </td><td>−0.091 </td><td>&lt;1.0e-5 </td><td>−0.109 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>Intron </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.133 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td></tr><tr><td>200-bp downstream </td><td>−0.079 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>500-bp downstream </td><td>−0.100 </td><td>&lt;1.0e-5 </td><td>−0.095 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>2-kb downstream </td><td>−0.058 </td><td>&lt;1.0e-5 </td><td>−0.049 </td><td>&lt;1.0e-5 </td><td>−0.034 </td><td>&lt;1.0e-5 </td></tr></tbody></table></div><div content-id="dsy021-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>  </th><th colspan="2">CG<hr /> </th><th colspan="2">CHG<hr /> </th><th colspan="2">CHH<hr /> </th></tr><tr><th>  </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th></tr></thead><tbody><tr><td>2-kb upstream </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.002 </td><td>NS </td><td>0.016 </td><td>&lt;0.05 </td></tr><tr><td>500-bp upstream </td><td>−0.081 </td><td>&lt;1.0e-5 </td><td>−0.043 </td><td>&lt;1.0e-5 </td><td>−0.009 </td><td>NS </td></tr><tr><td>200-bp upstream </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.087 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>First exon </td><td>−0.123 </td><td>&lt;1.0e-5 </td><td>−0.108 </td><td>&lt;1.0e-5 </td><td>−0.078 </td><td>&lt;1.0e-5 </td></tr><tr><td>Exon </td><td>−0.012 </td><td>NS </td><td>−0.137 </td><td>&lt;1.0e-5 </td><td>−0.129 </td><td>&lt;1.0e-5 </td></tr><tr><td>First intron </td><td>−0.091 </td><td>&lt;1.0e-5 </td><td>−0.109 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>Intron </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.133 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td></tr><tr><td>200-bp downstream </td><td>−0.079 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>500-bp downstream </td><td>−0.100 </td><td>&lt;1.0e-5 </td><td>−0.095 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>2-kb downstream </td><td>−0.058 </td><td>&lt;1.0e-5 </td><td>−0.049 </td><td>&lt;1.0e-5 </td><td>−0.034 </td><td>&lt;1.0e-5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>r</em>, correlation coefficient.</p></div></div><div content-id="tblfn2"><div class="fn"><p>Expression levels is log 2 score of FPKM (FPKM &gt; 0.01).</p></div></div><div content-id="tblfn3"><div class="fn"><p>NS, not significantly.</p></div></div><div content-id="tblfn4"><div class="fn"><p><em>P</em>-values are calculated by permutation test.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy021-T1" data-id="dsy021-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Correlation coefficients between DNA methylation levels and expression levels</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>  </th><th colspan="2">CG<hr /> </th><th colspan="2">CHG<hr /> </th><th colspan="2">CHH<hr /> </th></tr><tr><th>  </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th></tr></thead><tbody><tr><td>2-kb upstream </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.002 </td><td>NS </td><td>0.016 </td><td>&lt;0.05 </td></tr><tr><td>500-bp upstream </td><td>−0.081 </td><td>&lt;1.0e-5 </td><td>−0.043 </td><td>&lt;1.0e-5 </td><td>−0.009 </td><td>NS </td></tr><tr><td>200-bp upstream </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.087 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>First exon </td><td>−0.123 </td><td>&lt;1.0e-5 </td><td>−0.108 </td><td>&lt;1.0e-5 </td><td>−0.078 </td><td>&lt;1.0e-5 </td></tr><tr><td>Exon </td><td>−0.012 </td><td>NS </td><td>−0.137 </td><td>&lt;1.0e-5 </td><td>−0.129 </td><td>&lt;1.0e-5 </td></tr><tr><td>First intron </td><td>−0.091 </td><td>&lt;1.0e-5 </td><td>−0.109 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>Intron </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.133 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td></tr><tr><td>200-bp downstream </td><td>−0.079 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>500-bp downstream </td><td>−0.100 </td><td>&lt;1.0e-5 </td><td>−0.095 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>2-kb downstream </td><td>−0.058 </td><td>&lt;1.0e-5 </td><td>−0.049 </td><td>&lt;1.0e-5 </td><td>−0.034 </td><td>&lt;1.0e-5 </td></tr></tbody></table></div><div content-id="dsy021-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>  </th><th colspan="2">CG<hr /> </th><th colspan="2">CHG<hr /> </th><th colspan="2">CHH<hr /> </th></tr><tr><th>  </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th><th><em>r</em> </th><th><em>P</em>-values </th></tr></thead><tbody><tr><td>2-kb upstream </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.002 </td><td>NS </td><td>0.016 </td><td>&lt;0.05 </td></tr><tr><td>500-bp upstream </td><td>−0.081 </td><td>&lt;1.0e-5 </td><td>−0.043 </td><td>&lt;1.0e-5 </td><td>−0.009 </td><td>NS </td></tr><tr><td>200-bp upstream </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.087 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>First exon </td><td>−0.123 </td><td>&lt;1.0e-5 </td><td>−0.108 </td><td>&lt;1.0e-5 </td><td>−0.078 </td><td>&lt;1.0e-5 </td></tr><tr><td>Exon </td><td>−0.012 </td><td>NS </td><td>−0.137 </td><td>&lt;1.0e-5 </td><td>−0.129 </td><td>&lt;1.0e-5 </td></tr><tr><td>First intron </td><td>−0.091 </td><td>&lt;1.0e-5 </td><td>−0.109 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>Intron </td><td>−0.035 </td><td>&lt;1.0e-5 </td><td>−0.133 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td></tr><tr><td>200-bp downstream </td><td>−0.079 </td><td>&lt;1.0e-5 </td><td>−0.103 </td><td>&lt;1.0e-5 </td><td>−0.081 </td><td>&lt;1.0e-5 </td></tr><tr><td>500-bp downstream </td><td>−0.100 </td><td>&lt;1.0e-5 </td><td>−0.095 </td><td>&lt;1.0e-5 </td><td>−0.057 </td><td>&lt;1.0e-5 </td></tr><tr><td>2-kb downstream </td><td>−0.058 </td><td>&lt;1.0e-5 </td><td>−0.049 </td><td>&lt;1.0e-5 </td><td>−0.034 </td><td>&lt;1.0e-5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>r</em>, correlation coefficient.</p></div></div><div content-id="tblfn2"><div class="fn"><p>Expression levels is log 2 score of FPKM (FPKM &gt; 0.01).</p></div></div><div content-id="tblfn3"><div class="fn"><p>NS, not significantly.</p></div></div><div content-id="tblfn4"><div class="fn"><p><em>P</em>-values are calculated by permutation test.</p></div></div></div></div></p>                            <h3 scrollto-destination=122670944 id="122670944" class="section-title" >3.3. Comparison of DNA methylation states of <em>B. rapa</em> genes with orthologous genes of <em>A. thaliana</em></h3>
<p>We first identified gbM genes (methylated only in the CG context), under-methylated genes (under-methylated at all three types of sites), and genes having methylation at all three types of sites, in <em>B. rapa</em> by a probabilistic approach (see Section 2.3).<a reveal-id="dsy021-B28" data-open="dsy021-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> A total of 3,521 of 29,612 genes (11.9%) were classified as gbM genes. The proportion of gbM genes in <em>B. rapa</em> is lower than that in <em>A. thaliana</em> (23.3%; using the same calculation methods on the data from Lister <em>et al.</em><a reveal-id="dsy021-B18" data-open="dsy021-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a>) (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). A total of 3,141 of 29,612 genes (10.6%) had methylation both in the CG, CHG, and CHH contexts, which was higher than that in <em>A. thaliana</em> (3.6%), while 14,583 of 29,612 genes (49.2%) were categorized into under-methylated genes in <em>B. rapa</em>, similar to that in <em>A. thaliana</em> (44.1%) (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p><p>Next, we compared the DNA methylation states between the orthologous genes of <em>B. rapa</em> and <em>A. thaliana</em>. Because <em>B. rapa</em> experienced a WGT event,<a reveal-id="dsy021-B22" data-open="dsy021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy021-B24" data-open="dsy021-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsy021-B25" data-open="dsy021-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> there are from one to three <em>A. thaliana</em> orthologous genes in <em>B. rapa</em>. The previous study provided the list of orthologous genes based on the similarity of protein sequences and syntenic information.<a reveal-id="dsy021-B24" data-open="dsy021-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The orthologous genes were found in <em>A. thaliana</em> genes for 3,448 of 3,521 gbM genes in <em>B. rapa</em>, 2,976 of which were one-to-one orthologous genes between the two species (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>). A total of 1,904 of these 2,976 genes (64.0%) had gbM in <em>A. thaliana.</em> A total of 14,147 of 14,583 under-methylated genes in <em>B. rapa</em> had the orthologous genes in <em>A. thaliana</em>, and this was reduced to 10,333 one-to-one orthologous genes. Only 1,842 of these 10,333 genes (17.8%) had gbM in the <em>A. thaliana</em> genome (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>). These results suggest gbM tended to be conserved between orthologous genes in <em>B. rapa</em> and <em>A. thaliana</em>. <div content-id="dsy021-T2" class="table-modal"><div class="table-wrap"><div id="dsy021-T2" data-id="dsy021-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparison of gbM states between orthologous genes in <em>B. rapa</em> and <em>A. thaliana</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>  </th><th colspan="3"><em>B. rapa</em><hr /> </th><th colspan="3"><em>A. thaliana</em><hr /> </th></tr><tr><th>  </th><th>Number </th><th>Orthologous genes </th><th>Percentage (%) </th><th>Orthlogous genes<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>gbM genes </th><th>Percentage (%) </th></tr></thead><tbody><tr><td>gbM genes </td><td>3,521 </td><td>3,448 </td><td>97.9 </td><td>2,976 </td><td>1,904 </td><td>64.0 </td></tr><tr><td>Under-methylated genes </td><td>14,583 </td><td>14,147 </td><td>97.0 </td><td>10,333 </td><td>1,842 </td><td>17.8 </td></tr><tr><td>Heavily methylated genes </td><td>911 </td><td>424 </td><td>46.5 </td><td>395 </td><td>91 </td><td>23.0 </td></tr></tbody></table></div><div content-id="dsy021-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>  </th><th colspan="3"><em>B. rapa</em><hr /> </th><th colspan="3"><em>A. thaliana</em><hr /> </th></tr><tr><th>  </th><th>Number </th><th>Orthologous genes </th><th>Percentage (%) </th><th>Orthlogous genes<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>gbM genes </th><th>Percentage (%) </th></tr></thead><tbody><tr><td>gbM genes </td><td>3,521 </td><td>3,448 </td><td>97.9 </td><td>2,976 </td><td>1,904 </td><td>64.0 </td></tr><tr><td>Under-methylated genes </td><td>14,583 </td><td>14,147 </td><td>97.0 </td><td>10,333 </td><td>1,842 </td><td>17.8 </td></tr><tr><td>Heavily methylated genes </td><td>911 </td><td>424 </td><td>46.5 </td><td>395 </td><td>91 </td><td>23.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>gbM, gene-body methylation.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>a</sup><em>B. rapa</em> genes having sequence homology to <em>A. thaliana</em> genes with removal where more than two <em>B. rapa</em> genes in <em>A. thaliana</em>.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy021-T2" data-id="dsy021-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Comparison of gbM states between orthologous genes in <em>B. rapa</em> and <em>A. thaliana</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>  </th><th colspan="3"><em>B. rapa</em><hr /> </th><th colspan="3"><em>A. thaliana</em><hr /> </th></tr><tr><th>  </th><th>Number </th><th>Orthologous genes </th><th>Percentage (%) </th><th>Orthlogous genes<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>gbM genes </th><th>Percentage (%) </th></tr></thead><tbody><tr><td>gbM genes </td><td>3,521 </td><td>3,448 </td><td>97.9 </td><td>2,976 </td><td>1,904 </td><td>64.0 </td></tr><tr><td>Under-methylated genes </td><td>14,583 </td><td>14,147 </td><td>97.0 </td><td>10,333 </td><td>1,842 </td><td>17.8 </td></tr><tr><td>Heavily methylated genes </td><td>911 </td><td>424 </td><td>46.5 </td><td>395 </td><td>91 </td><td>23.0 </td></tr></tbody></table></div><div content-id="dsy021-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>  </th><th colspan="3"><em>B. rapa</em><hr /> </th><th colspan="3"><em>A. thaliana</em><hr /> </th></tr><tr><th>  </th><th>Number </th><th>Orthologous genes </th><th>Percentage (%) </th><th>Orthlogous genes<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>gbM genes </th><th>Percentage (%) </th></tr></thead><tbody><tr><td>gbM genes </td><td>3,521 </td><td>3,448 </td><td>97.9 </td><td>2,976 </td><td>1,904 </td><td>64.0 </td></tr><tr><td>Under-methylated genes </td><td>14,583 </td><td>14,147 </td><td>97.0 </td><td>10,333 </td><td>1,842 </td><td>17.8 </td></tr><tr><td>Heavily methylated genes </td><td>911 </td><td>424 </td><td>46.5 </td><td>395 </td><td>91 </td><td>23.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>gbM, gene-body methylation.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>a</sup><em>B. rapa</em> genes having sequence homology to <em>A. thaliana</em> genes with removal where more than two <em>B. rapa</em> genes in <em>A. thaliana</em>.</p></div></div></div></div></p><p>We further compared the level of gbM (level of CG methylation) between orthologous genes in <em>B. rapa</em> and <em>A. thaliana</em>. Again, we found significant correlations between them in all three subgenome domains of <em>B. rapa</em>, LF, MF1, and MF2 (<em>r </em>=<em> </em>0.433 for LF; <em>r </em>=<em> </em>0.393 for MF1; <em>r </em>=<em> </em>0.402 for MF2; <em>P </em>&lt;<em> </em>10<sup>−5</sup> by a permutation test in all three cases; <a reveal-id="dsy021-F2" data-open="dsy021-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).<a reveal-id="dsy021-B24" data-open="dsy021-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> We also compared the levels of gbM between paralogous genes in <em>B. rapa</em> that were derived in WGT event, and found significant correlations between them (<em>r </em>=<em> </em>0.467 for LF vs. MF1; <em>r </em>=<em> </em>0.494 for LF vs. MF2; <em>r </em>=<em> </em>0.442 for MF1 vs. MF2; <em>P </em>&lt;<em> </em>10<sup>−5</sup> by a permutation test in all three cases; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), indicating that gbM was conserved between paralogous genes in <em>B. rapa</em>. </p>                        <a id="122670948" scrollto-destination="122670948"></a>
<div id="dsy021-F2" data-id="dsy021-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f2.png?Expires=2147483647&Signature=1FTh61CK-rN1k9MNvIDqbEKJ1PBzz7MLQgjQpUhUbY0Z7SX-BZrSUUZWYvwfodrnJVzXTcGbnIDSn9JD8a5BDtqPABMBQUYo8xPoaTEMffV8fwdXTJmwdVHLcjxY04xVmykkSkKiXlif-DZ0zVMsnYTzvWkcoCh3NjssrK5CHjGjlh2nFl7JlHtkX3lNgEThVUF7QcRyrN0Fnj3byu6uZQ6W~ecZ73Z0B7MSq~4LcdLGznQmPOhXM6DQkGoxbf8zwJlwfYm7wjtu3ekbHFnibOf9mj5nVGMFPebM~3nGQtzfOEuWsKCbiDf232paCJ57QWtgIPPdoqGpnDF2Xoh5Hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The scatter plots for the level of CG methylation between A. thaliana and B. rapa orthologous genes in the three subgenomes. Correlation of CG methylation levels between A. thaliana and B. rapa orthologous genes in LF, MF1, and MF2 were 0.433 (n = 8, 040; P-value &lt; 10−5), 0.393 (n = 5, 223; P-value &lt; 10−5), and 0.402 (n = 4, 580; P-value &lt; 10−5), respectively. P-values were calculated by a permutation test." path-from-xml="dsy021f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670948/dsy021f2.tif" path-from-xml="dsy021f2.tif" target="_blank">View large</a><a section="122670948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f2.png?Expires=2147483647&Signature=eebWRMYtSSUI0UhCNS3ttcwYNRSoalcBW3onwBhjMXt72p1DVbRZ6CBa7d~Yo6G0~W-dRtioPKCaHdnKurvV10xF73bRmJG~ggRUtS7xFfPRw64yWjgB1o~2mbHKjSqUQiWOnESNddMWZwpnyWs8Zsejg72vWSGExKkC0fl6fh3eYQ0zD0RDFN9LILsbR8iZqAGVXXukG0Ho0hDTmS7DlxmUoafvGW4n9Rd3HIU~Gb9VJeBp3Iv0fwL1wl0JA8U~0yYwv0FonxKhSPUnZiZ-AQXR2tfhgqAIzEj54fH0tKe~WkaT5nk0no662vOS2CxjpFXIM1DMhbmhqorhLceyGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670948&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The scatter plots for the level of CG methylation between <em>A. thaliana</em> and <em>B. rapa</em> orthologous genes in the three subgenomes. Correlation of CG methylation levels between <em>A. thaliana</em> and <em>B. rapa</em> orthologous genes in LF, MF1, and MF2 were 0.433 (<em>n</em> = 8, 040; <em>P-</em>value &lt; 10<sup>−5</sup>), 0.393 (<em>n</em> = 5, 223; <em>P-</em>value &lt; 10<sup>−5</sup>), and 0.402 (<em>n</em> = 4, 580; <em>P-</em>value &lt; 10<sup>−5</sup>), respectively. <em>P</em>-values were calculated by a permutation test.</p></div></div><div content-id="dsy021-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f2.png?Expires=2147483647&Signature=1FTh61CK-rN1k9MNvIDqbEKJ1PBzz7MLQgjQpUhUbY0Z7SX-BZrSUUZWYvwfodrnJVzXTcGbnIDSn9JD8a5BDtqPABMBQUYo8xPoaTEMffV8fwdXTJmwdVHLcjxY04xVmykkSkKiXlif-DZ0zVMsnYTzvWkcoCh3NjssrK5CHjGjlh2nFl7JlHtkX3lNgEThVUF7QcRyrN0Fnj3byu6uZQ6W~ecZ73Z0B7MSq~4LcdLGznQmPOhXM6DQkGoxbf8zwJlwfYm7wjtu3ekbHFnibOf9mj5nVGMFPebM~3nGQtzfOEuWsKCbiDf232paCJ57QWtgIPPdoqGpnDF2Xoh5Hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The scatter plots for the level of CG methylation between A. thaliana and B. rapa orthologous genes in the three subgenomes. Correlation of CG methylation levels between A. thaliana and B. rapa orthologous genes in LF, MF1, and MF2 were 0.433 (n = 8, 040; P-value &lt; 10−5), 0.393 (n = 5, 223; P-value &lt; 10−5), and 0.402 (n = 4, 580; P-value &lt; 10−5), respectively. P-values were calculated by a permutation test." path-from-xml="dsy021f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670948/dsy021f2.tif" path-from-xml="dsy021f2.tif" target="_blank">View large</a><a section="122670948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f2.png?Expires=2147483647&Signature=eebWRMYtSSUI0UhCNS3ttcwYNRSoalcBW3onwBhjMXt72p1DVbRZ6CBa7d~Yo6G0~W-dRtioPKCaHdnKurvV10xF73bRmJG~ggRUtS7xFfPRw64yWjgB1o~2mbHKjSqUQiWOnESNddMWZwpnyWs8Zsejg72vWSGExKkC0fl6fh3eYQ0zD0RDFN9LILsbR8iZqAGVXXukG0Ho0hDTmS7DlxmUoafvGW4n9Rd3HIU~Gb9VJeBp3Iv0fwL1wl0JA8U~0yYwv0FonxKhSPUnZiZ-AQXR2tfhgqAIzEj54fH0tKe~WkaT5nk0no662vOS2CxjpFXIM1DMhbmhqorhLceyGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670948&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The scatter plots for the level of CG methylation between <em>A. thaliana</em> and <em>B. rapa</em> orthologous genes in the three subgenomes. Correlation of CG methylation levels between <em>A. thaliana</em> and <em>B. rapa</em> orthologous genes in LF, MF1, and MF2 were 0.433 (<em>n</em> = 8, 040; <em>P-</em>value &lt; 10<sup>−5</sup>), 0.393 (<em>n</em> = 5, 223; <em>P-</em>value &lt; 10<sup>−5</sup>), and 0.402 (<em>n</em> = 4, 580; <em>P-</em>value &lt; 10<sup>−5</sup>), respectively. <em>P</em>-values were calculated by a permutation test.</p></div></div>                            <h3 scrollto-destination=122670949 id="122670949" class="section-title" >3.4. Small RNA sequencing</h3>
<p>24 nt siRNAs are associated with targeting DNA methylation to complementary sequences by RdDM.<a reveal-id="dsy021-B12" data-open="dsy021-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> To examine the relationship between the distribution of 24 nt siRNAs and DNA methylation, we performed small RNA-seq. A total of 47.4 M clean reads were obtained, and 6.6 M reads (14.0%) were uniquely mapped on the <em>B. rapa</em> genome. A total of 3.7 of 6.6 M reads (55.3%) were mapped on the IRRs (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The majority of small RNAs were 24 nt in length. The second and third most abundant small RNAs were 23 nt and 21 nt in length, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). We identified 65,142 clusters of 24 nt siRNAs in total, and 11,354, 1,678, 2,057, and 8,795 genes had more than one 24 nt-siRNA cluster in the 2-kb upstream, exon, intron, and 2-kb downstream regions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p><p>We compared 24 nt-siRNA level (RPKM) and DNA methylation level of each sliding window of per 100 kb, and 24 nt-siRNA levels were most associated with CHH methylation levels (<a reveal-id="dsy021-F3" data-open="dsy021-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). We compared the regions having 24 nt-siRNA clusters and regions having DNA methylation detected by WGBS. About 70–80% of genes having 24 nt-siRNA clusters in the 2-kb upstream, exon, intron, and 2-kb downstream regions overlapped with genes having DNA methylation detected by WGBS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). About 48.0%, 39.0%, and 60.0% of total 24 nt-siRNA clusters overlapped with CG, CHG, and CHH methylation, respectively, and 45.2%, 35.9%, and 60.5% of 24 nt-siRNA clusters in IRRs overlapped with CG, CHG, and CHH methylation, respectively (<a reveal-id="dsy021-F3" data-open="dsy021-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). This indicates that regions having 24 nt-siRNA clusters tended to have CHH methylation rather than CG and CHG methylation. The average methylation levels for CG, CHG, and CHH sites in the regions overlapping 24 nt-siRNA clusters were 87.7%, 51.4%, and 21.6%, respectively (<a reveal-id="dsy021-F4" data-open="dsy021-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). DNA methylation levels overlapping the 24 nt-siRNA clusters were high regardless of where the IRRs were located within the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1 and S7</a></span>). </p>                        <a id="122670952" scrollto-destination="122670952"></a>
<div id="dsy021-F3" data-id="dsy021-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f3.png?Expires=2147483647&Signature=fUK1H0ZILR7Kt984kL2JmSyclJa~4Z4SpNxYMtcm0RgO5dobnTy~i~DY0-cu321d6JVKHEwNtg7wrccUjZmKgz2g1M-tliEvwg4Ah3Ko-9mSBhIb3t8yN66JeCTntqFYG~aAH0oHAq4frgUy8THS1lc0zKMmyxxw1kYXDCmEBdB1-BG-rMYTt0s77UkeBapZTfwTANM~ejpAD9sCNze3oknBk-2SHJ8uPMBTIvW3z99FdI4apOiOnT2iz1ZerUevmI-VRBI8AmjbZN0I8TrKAvaSA7od-uvq2FjZwFYxl5PEOkw4KD5TWjNFpboFhTNgHKnXHMoSYJ0ps0-DeApYWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The comparison between 24 nt-siRNA or H3K9me2 level and DNA methylation level. (A) Comparison of the 24 nt-siRNA or H3K9me2 level of the RPKM and DNA methylation level in each sliding window per 100 kb. (B) Percentage of genes both having 24 nt-siRNAs clusters and CG, CHG, and CHH methylation or both H3K9me2 peaks and CG, CHG, and CHH methylation. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively, with 24 nt-siRNA clusters or H3K9me2 peaks." path-from-xml="dsy021f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670952/dsy021f3.tif" path-from-xml="dsy021f3.tif" target="_blank">View large</a><a section="122670952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f3.png?Expires=2147483647&Signature=jBxBVXMInw0X8ENO0tX3O2aItbxY-gEGBoFXAGk9vy~yVzW9rCuLBbJbgVm-lQ1WwNxfw4JB8cX9BpJW4DQXJkVI7gWrWbL5WNVUBRxxaqBxHGg4EubcwOZfKwOHdRCH0EEJxqgPKb~fq2JlytDWXKusu6XP5H~t8QxGvB7wHn2IPg9f9PbbcZxIGmmPZqKfECpw7vE79rtJNVBX3X8JgTBgyKRvsOjuk5Bha~ZG63WmRaG~ixGxOvTL6CZ0xfyPA7g4jLHawhrgneaGuYpsR5LLFcXEjkTpgwoOseJR~A41wMOgMRoEZZEyLAiXPSXIGP8RZuZ4JPU5Op~s37Yn0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670952&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The comparison between 24 nt-siRNA or H3K9me2 level and DNA methylation level. (A) Comparison of the 24 nt-siRNA or H3K9me2 level of the RPKM and DNA methylation level in each sliding window per 100 kb. (B) Percentage of genes both having 24 nt-siRNAs clusters and CG, CHG, and CHH methylation or both H3K9me2 peaks and CG, CHG, and CHH methylation. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively, with 24 nt-siRNA clusters or H3K9me2 peaks.</p></div></div><div content-id="dsy021-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f3.png?Expires=2147483647&Signature=fUK1H0ZILR7Kt984kL2JmSyclJa~4Z4SpNxYMtcm0RgO5dobnTy~i~DY0-cu321d6JVKHEwNtg7wrccUjZmKgz2g1M-tliEvwg4Ah3Ko-9mSBhIb3t8yN66JeCTntqFYG~aAH0oHAq4frgUy8THS1lc0zKMmyxxw1kYXDCmEBdB1-BG-rMYTt0s77UkeBapZTfwTANM~ejpAD9sCNze3oknBk-2SHJ8uPMBTIvW3z99FdI4apOiOnT2iz1ZerUevmI-VRBI8AmjbZN0I8TrKAvaSA7od-uvq2FjZwFYxl5PEOkw4KD5TWjNFpboFhTNgHKnXHMoSYJ0ps0-DeApYWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The comparison between 24 nt-siRNA or H3K9me2 level and DNA methylation level. (A) Comparison of the 24 nt-siRNA or H3K9me2 level of the RPKM and DNA methylation level in each sliding window per 100 kb. (B) Percentage of genes both having 24 nt-siRNAs clusters and CG, CHG, and CHH methylation or both H3K9me2 peaks and CG, CHG, and CHH methylation. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively, with 24 nt-siRNA clusters or H3K9me2 peaks." path-from-xml="dsy021f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670952/dsy021f3.tif" path-from-xml="dsy021f3.tif" target="_blank">View large</a><a section="122670952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f3.png?Expires=2147483647&Signature=jBxBVXMInw0X8ENO0tX3O2aItbxY-gEGBoFXAGk9vy~yVzW9rCuLBbJbgVm-lQ1WwNxfw4JB8cX9BpJW4DQXJkVI7gWrWbL5WNVUBRxxaqBxHGg4EubcwOZfKwOHdRCH0EEJxqgPKb~fq2JlytDWXKusu6XP5H~t8QxGvB7wHn2IPg9f9PbbcZxIGmmPZqKfECpw7vE79rtJNVBX3X8JgTBgyKRvsOjuk5Bha~ZG63WmRaG~ixGxOvTL6CZ0xfyPA7g4jLHawhrgneaGuYpsR5LLFcXEjkTpgwoOseJR~A41wMOgMRoEZZEyLAiXPSXIGP8RZuZ4JPU5Op~s37Yn0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670952&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The comparison between 24 nt-siRNA or H3K9me2 level and DNA methylation level. (A) Comparison of the 24 nt-siRNA or H3K9me2 level of the RPKM and DNA methylation level in each sliding window per 100 kb. (B) Percentage of genes both having 24 nt-siRNAs clusters and CG, CHG, and CHH methylation or both H3K9me2 peaks and CG, CHG, and CHH methylation. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively, with 24 nt-siRNA clusters or H3K9me2 peaks.</p></div></div>                        <a id="122670953" scrollto-destination="122670953"></a>
<div id="dsy021-F4" data-id="dsy021-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f4.png?Expires=2147483647&Signature=iNSEEexyT~aFFx4WzqyGR08DiptjRbOP~PN~EkSXLhSkn09J1BI2KUoOWHkN8aGFctjdBXRkFLrWyehOdv7VGukFoR8ykrI732G~s8HmPyw3HvL5xjjOCGnzADSk6RI0R~onDSA0MjRaE0TAv4zcSeUDfoLOYkrKSWL7B6nHmWnaACu5zDHoXjTivEsATvArGX8Tt2eMlNo204czTwdltNj12GWMEWGhffufToGn2Xs7I4vt~BZiqzILkmwykGH3jl3FzBDx5ppEbMw8YiyVftBitHKwYNFHezgEz8j8sk6kgqOSBZQvfbgyy7bexfKov~7LDV3W7Ka~h06YD3t~QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average of DNA methylation levels in the regions covering 24 nt-siRNA clusters or H3K9me2 peaks. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively." path-from-xml="dsy021f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670953/dsy021f4.tif" path-from-xml="dsy021f4.tif" target="_blank">View large</a><a section="122670953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f4.png?Expires=2147483647&Signature=h5rPbdP5usCRGXLTf0S~skJDnCD-xEyPyBOdyBn-Fpec0F9rHdZwE33fqHYh6X2aZsx80l-hb9k1fDFquQUqlNmRlGC8pCYbMTCDSm8by62JDcV4B4XaejMhtG2jEQ96Fh0NXFSmPOXbe0CC4wbLpiGJogJm2jKsnPITyAfvygEb20ywxny2v5DKD5VGAuuXnpzf-gDJ~nDLvAbgEfideT9ctogH3mP-NJBZelbUzp8qSbxR1hU38A4MtchvhKWtvupfvnVzO3~OzTlCHjjahp~kH3mrrSIISl3Mh3jUvUah-K3NDNdoGN~OWKItjelkJ6ESnqz9~6ltrfijuQybJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670953&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average of DNA methylation levels in the regions covering 24 nt-siRNA clusters or H3K9me2 peaks. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively.</p></div></div><div content-id="dsy021-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f4.png?Expires=2147483647&Signature=iNSEEexyT~aFFx4WzqyGR08DiptjRbOP~PN~EkSXLhSkn09J1BI2KUoOWHkN8aGFctjdBXRkFLrWyehOdv7VGukFoR8ykrI732G~s8HmPyw3HvL5xjjOCGnzADSk6RI0R~onDSA0MjRaE0TAv4zcSeUDfoLOYkrKSWL7B6nHmWnaACu5zDHoXjTivEsATvArGX8Tt2eMlNo204czTwdltNj12GWMEWGhffufToGn2Xs7I4vt~BZiqzILkmwykGH3jl3FzBDx5ppEbMw8YiyVftBitHKwYNFHezgEz8j8sk6kgqOSBZQvfbgyy7bexfKov~7LDV3W7Ka~h06YD3t~QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average of DNA methylation levels in the regions covering 24 nt-siRNA clusters or H3K9me2 peaks. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively." path-from-xml="dsy021f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670953/dsy021f4.tif" path-from-xml="dsy021f4.tif" target="_blank">View large</a><a section="122670953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f4.png?Expires=2147483647&Signature=h5rPbdP5usCRGXLTf0S~skJDnCD-xEyPyBOdyBn-Fpec0F9rHdZwE33fqHYh6X2aZsx80l-hb9k1fDFquQUqlNmRlGC8pCYbMTCDSm8by62JDcV4B4XaejMhtG2jEQ96Fh0NXFSmPOXbe0CC4wbLpiGJogJm2jKsnPITyAfvygEb20ywxny2v5DKD5VGAuuXnpzf-gDJ~nDLvAbgEfideT9ctogH3mP-NJBZelbUzp8qSbxR1hU38A4MtchvhKWtvupfvnVzO3~OzTlCHjjahp~kH3mrrSIISl3Mh3jUvUah-K3NDNdoGN~OWKItjelkJ6ESnqz9~6ltrfijuQybJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670953&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average of DNA methylation levels in the regions covering 24 nt-siRNA clusters or H3K9me2 peaks. ‘All’ and ‘IRRs’ indicate the average of DNA methylation levels in all regions and interspersed repeats regions, respectively.</p></div></div><p>We examined the relationship between the presence of 24 nt-siRNA clusters and expression levels (log 2 score of FPKM, FPKM &gt; 0.01). Genes having 24 nt-siRNAs in exon regions tended to show a wide range of expression (<a reveal-id="dsy021-F5" data-open="dsy021-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Average expression level of genes having 24 nt-siRNAs in exons was higher than that of total genes, and it was also significantly higher than that of genes not having 24 nt-siRNAs in an exon (permutation test, <em>P </em>&lt;<em> </em>0.001) (<a reveal-id="dsy021-F5" data-open="dsy021-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Of genes having 24 nt-siRNAs in exons, genes that were more lowly expressed tended to have CG, CHG, and CHH methylation, while more highly expressed genes tended to be under-methylated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). </p>                        <a id="122670955" scrollto-destination="122670955"></a>
<div id="dsy021-F5" data-id="dsy021-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f5.png?Expires=2147483647&Signature=PZaZFNWLfvuQHwreAbdpZzM9XnnOjnZLXR3fDRyuxLVaAZoUbQQMaY1tXJM8s1ObBGUMTG2zDUq-KbO170YMPaFte7tdL0Lb7gn3M0ntwyGuXfmITjHIcoHsOcslrLc2rzeUxmfvIE-CXY4VbmLd8lCujYbK~i8-uCxwI05J3LZfbvGv700v~IDiTkOxzL7Z1dh5KwiNLJxGGyLZFMl8ThZBhyi80GQnwWZaAKZC2rabmUOByUOO7CuMrPBOmNRfXTyiVnghIPmrs8a2s5ByX0C8j3AvKVt4chpB2KsNXPsE4jFqbPPvcOClzcGk-JOXJW2GGKTP8v52ll-EvWhgGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Box plots of the expression levels of log 2 score of FPKM in genes having 24 nt-siRNA clusters or H3K9me2 peaks in the 2-kb upstream regions Up (2k), exon, intron, or 2-kb downstream regions Down (2k) of RJKB-T24. Total indicates the log 2 score of FPKM in all genes (FPKM &lt; 0.01)." path-from-xml="dsy021f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670955/dsy021f5.tif" path-from-xml="dsy021f5.tif" target="_blank">View large</a><a section="122670955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f5.png?Expires=2147483647&Signature=Ysc32qjMsukrN2MLtckvkDoMLSbe4rie6Iuv0dAor-aXYDZYTDNjXFahu9~9XoIBAGu4PrbcY-wTDdkOBFek~LqYqfDq0RaePHDzMEnLJO89c20ajw5uHISPnSRTYNInxDryxpxuBil3A6aYHdahlX7EQd~~2pZzZ-R81pPkbPsDx8RMv4ZIIYOJOFneWLu99YS5Z7epmgDdmFWOBplo1ydOV~hXSSz0g3kqHtbMqWxPMHMpF03WbwbEWGPWKAk4XVTXm5ziOFfcXnpvfwgEpmheB9aRUkF7IUTrjfsEQBNuvnNqachTqFS1Aeap6CwfNb7skBSxcQBhRbidS9lABw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670955&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Box plots of the expression levels of log 2 score of FPKM in genes having 24 nt-siRNA clusters or H3K9me2 peaks in the 2-kb upstream regions Up (2k), exon, intron, or 2-kb downstream regions Down (2k) of RJKB-T24. Total indicates the log 2 score of FPKM in all genes (FPKM &lt; 0.01).</p></div></div><div content-id="dsy021-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/m_dsy021f5.png?Expires=2147483647&Signature=PZaZFNWLfvuQHwreAbdpZzM9XnnOjnZLXR3fDRyuxLVaAZoUbQQMaY1tXJM8s1ObBGUMTG2zDUq-KbO170YMPaFte7tdL0Lb7gn3M0ntwyGuXfmITjHIcoHsOcslrLc2rzeUxmfvIE-CXY4VbmLd8lCujYbK~i8-uCxwI05J3LZfbvGv700v~IDiTkOxzL7Z1dh5KwiNLJxGGyLZFMl8ThZBhyi80GQnwWZaAKZC2rabmUOByUOO7CuMrPBOmNRfXTyiVnghIPmrs8a2s5ByX0C8j3AvKVt4chpB2KsNXPsE4jFqbPPvcOClzcGk-JOXJW2GGKTP8v52ll-EvWhgGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Box plots of the expression levels of log 2 score of FPKM in genes having 24 nt-siRNA clusters or H3K9me2 peaks in the 2-kb upstream regions Up (2k), exon, intron, or 2-kb downstream regions Down (2k) of RJKB-T24. Total indicates the log 2 score of FPKM in all genes (FPKM &lt; 0.01)." path-from-xml="dsy021f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670955/dsy021f5.tif" path-from-xml="dsy021f5.tif" target="_blank">View large</a><a section="122670955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021f5.png?Expires=2147483647&Signature=Ysc32qjMsukrN2MLtckvkDoMLSbe4rie6Iuv0dAor-aXYDZYTDNjXFahu9~9XoIBAGu4PrbcY-wTDdkOBFek~LqYqfDq0RaePHDzMEnLJO89c20ajw5uHISPnSRTYNInxDryxpxuBil3A6aYHdahlX7EQd~~2pZzZ-R81pPkbPsDx8RMv4ZIIYOJOFneWLu99YS5Z7epmgDdmFWOBplo1ydOV~hXSSz0g3kqHtbMqWxPMHMpF03WbwbEWGPWKAk4XVTXm5ziOFfcXnpvfwgEpmheB9aRUkF7IUTrjfsEQBNuvnNqachTqFS1Aeap6CwfNb7skBSxcQBhRbidS9lABw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670955&ar=5047107&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy021f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Box plots of the expression levels of log 2 score of FPKM in genes having 24 nt-siRNA clusters or H3K9me2 peaks in the 2-kb upstream regions Up (2k), exon, intron, or 2-kb downstream regions Down (2k) of RJKB-T24. Total indicates the log 2 score of FPKM in all genes (FPKM &lt; 0.01).</p></div></div>                            <h3 scrollto-destination=122670956 id="122670956" class="section-title" >3.5. Chromatin immunoprecipitation sequencing using H3K9me2 antibody</h3>
<p>To examine the relationship between DNA methylation and H3K9me2 in the <em>B. rapa</em> genome, we performed ChIP-seq analysis using an antibody against H3K9me2. A total of 37.8 M and 40.0 M clean reads were obtained from input-DNA-seq and ChIP-seq, respectively. A total of 18.2 M (input-DNA-seq, 48.2%) and 11.8 M reads (ChIP-seq, 29.5%) were uniquely mapped on the <em>B. rapa</em> genome, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). In total, 8.4 M ChIP-seq reads were mapped on the genic regions including exon, intron, and 2-kb upstream and downstream regions. The proportion of mapped reads of ChIP-seq in each of the four regions was similar to those of Input-DNA-seq (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). In the ChIP-seq data, 5.6 M (47.4%) reads mapped on the IRRs was higher than the percentage of mapped reads on the IRRs using input-DNA-seq data (39.1%) (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), but lower than using the previous MeDIP-seq data (63.9%; 36 bp single end).<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> A total of 9,920 H3K9me2 peaks were identified in RJKB-T24. A total of 1,293, 1,654, 449, and 895 genes had more than one H3K9me2 peak within 2-kb upstream, exon, intron, and 2-kb downstream regions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). A total of 7,268 of 9,920 H3K9me2 peaks (73.3%) were found in the IRRs (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>).</p><p>We examined the relationship between having H3K9me2 marks and expression levels (log 2 score of FPKM, FPKM &gt; 0.01). The average expression level in genes having H3K9me2 peaks in all four regions, especially in the exon and intron regions, was lower than that of total genes, and the average expression level of genes having H3K9me2 in the exon and intron regions was significantly lower than genes that do not have H3K9me2 peaks (permutation test, <em>P </em>&lt;<em> </em>0.001) (<a reveal-id="dsy021-F5" data-open="dsy021-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>).</p><p>We compared H3K9me2 levels (RPKM) and DNA methylation level or 24 nt-siRNA levels (RPKM) of each sliding window of per 100 kb. H3K9me2 levels were associated with DNA methylation levels, but there was no difference among the three contexts. In contrast, the association between H3K9me2 levels and 24 nt-siRNA levels was low (<a reveal-id="dsy021-F3" data-open="dsy021-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). In genes having H3K9me2 peaks, about 50% of them overlapped with genes having DNA methylation detected by WGBS in each of the 2-kb upstream, intron, and 2-kb downstream regions, while about 20% of genes overlapped with genes having DNA methylation in exon regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). The regions having H3K9me2 had CHH methylation, especially in IRRs, rather than CG or CHG methylation (<a reveal-id="dsy021-F3" data-open="dsy021-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The average of methylation levels for CG, CHG, and CHH sites in the regions overlapping H3K9me2 peaks were 53.6%, 24.2%, and 8.6%, respectively (<a reveal-id="dsy021-F4" data-open="dsy021-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), indicating increased DNA methylation levels in the region covering the H3K9me2 peaks. In the regions overlapping H3K9me2 peaks, DNA methylation levels were higher in IRRs (CG; 77.5%, CHG; 31.2%, CHH; 10.4%) in comparison to all regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). However, 6.2% of H3K9me2 peaks overlapped with 24 nt-siRNA clusters, and 4.3% of H3K9me2 peaks overlapped with 24 nt-siRNA clusters in IRRs, indicating that majority of H3K9me2 accumulation was not associated with 24 nt-siRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>).</p>                            <h3 scrollto-destination=122670960 id="122670960" class="section-title" >3.6. Genetic and DNA methylation states variation between <em>B. rapa</em> and <em>A. thaliana</em></h3>
<p>We compared DNA methylation states at specific <em>B. rapa</em> genes against the orthologous genes of <em>A. thaliana</em>. A total of 1,514 and 1,472 <em>B. rapa</em> genes of MeDIP-peaks had CG, CHG, and CHH methylation (heavily methylated genes) in the exon and intron regions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>), and 911 genes were in common between methylated exons and introns and distributed evenly over all chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). About 90% of these 911 heavily methylated genes had IRRs in exon and/or intron, and 99% of these IRRs were methylated (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>), indicating that heavy methylation in exon and intron regions of these 911 genes is due to the presence of IRRs targeted for DNA methylation.</p><p>A total of 424 of the 911 genes from <em>B. rapa</em> had putative orthologous genes in <em>A. thaliana</em> including paralogous genes. This was reduced to 395 genes in <em>A. thaliana</em> following the removal of genes where two or three <em>B. rapa</em> genes showed sequence similarity to one <em>A. thaliana</em> gene (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Of the 395 genes, 217 (54.9%) were categorized into under-methylated genes and 91 (23.0%) into gbM genes (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The frequency of under-methylated and gbM genes was similar to that of the total genes (under-methylated genes, 64.7%; gbM genes, 17.7%), suggesting that heavily methylated states were not conserved between orthologous genes between <em>B. rapa</em> and <em>A. thaliana</em>.</p><p>Of the 424 genes in <em>B. rapa</em>, 234 genes (55.2%) showed no expression, categorized into Group-0 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). In <em>A. thaliana</em>, we have divided the level of gene expression into six categories using microarray data; the first and sixth groups had the lowest and highest levels of expression, respectively.<a reveal-id="dsy021-B32" data-open="dsy021-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Of 395 <em>A. thaliana</em> genes with sequence homology to heavily methylated <em>B. rapa</em>, 325 genes were on the microarray, and the proportion classified from first to sixth groups were 11% (first, lowest), 11% (second), 14% (third), 16% (fourth), 23% (fifth), and 25% (sixth, highest), respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>), indicating that there was no tendency of transcription suppression. These results indicate that <em>A. thaliana</em> genes, which showed sequence similarity to heavily methylated genes in <em>B. rapa</em>, had neither heavy DNA methylation nor low expression levels.</p>                            <h2 scrollto-destination=122670964 id="122670964" class="section-title" >4. Discussion</h2>
<p>We examined the DNA methylation states and distribution of 24 nt-siRNAs and H3K9me2 of <em>B. rapa</em> at the whole genome level. DNA methylation and H3K9me2 are classified as repressive epigenetic marks and 24 nt-siRNAs are involved in RdDM. We showed how these three factors interact with each other and affect gene expression levels in <em>B. rapa</em>.</p>                            <h3 scrollto-destination=122670966 id="122670966" class="section-title" >4.1. DNA methylation states at the whole genome levels in <em>B. rapa</em></h3>
<p>We performed WGBS using an inbred line of Chinese cabbage, RJKB-T24, and these data were consistent with our previous MeDIP-seq data.<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The WGBS allowed us to assess different DNA methylation sequence contexts and was more sensitive. The average methylation levels for CG, CHG, and CHH sites in chromosome A01–A10 were 36.5%, 13.4%, and 5.3%, respectively, in RJKB-T24. Previously, WGBS was performed in <em>B. rapa</em> and the average methylation levels for CG, CHG, and CHH sites were 52.4%, 31.8%, and 8.3%, respectively,<a reveal-id="dsy021-B33" data-open="dsy021-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and 37.2%, 17.3%, and 4.4%, respectively.<a reveal-id="dsy021-B16" data-open="dsy021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Our results were similar to the latter report, though DNA methylation levels varied between lines or tissues. Most CG sites were highly methylated unlike in <em>A. thaliana</em>, but similar to previous reports of <em>B. rapa.</em><a reveal-id="dsy021-B16" data-open="dsy021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy021-B33" data-open="dsy021-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> High levels of DNA methylation in the IRRs including TEs or repetitive sequences were detected, which is a common observation not only in <em>B. rapa</em> but also in other plant species.<a reveal-id="dsy021-B17" data-open="dsy021-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy021-B33 dsy021-B34 dsy021-B35" data-open="dsy021-B33 dsy021-B34 dsy021-B35" class="link link-ref link-reveal xref-bibr"><sup>33–35</sup></a></p>                            <h3 scrollto-destination=122670968 id="122670968" class="section-title" >4.2. Heavy methylation in genic regions results in silencing of transcription</h3>
<p>We examined how DNA methylation affects gene expression levels using WGBS and RNA-seq data of identical tissues and stages. Promoter methylated genes tended to have lower expression levels in <em>A. thaliana.</em><a reveal-id="dsy021-B36" data-open="dsy021-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Promoter methylation also repressed gene expression in rice, and gene repression by DNA methylation in the transcriptional termination regions was stronger than the effect of DNA methylation in the promoter region.<a reveal-id="dsy021-B37" data-open="dsy021-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In our study, DNA methylation in the upstream and downstream regions of genes was negatively associated with expression levels, especially DNA methylation in the 200-bp upstream and downstream regions.</p><p>There is a strong negative relationship between DNA methylation at all sequence contexts in the first exon/intron and expression in <em>A. thaliana.</em><a reveal-id="dsy021-B38" data-open="dsy021-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> DNA methylation in the region of the first exon is much more tightly linked to transcriptional silencing than is methylation in the upstream promoter region in humans.<a reveal-id="dsy021-B39" data-open="dsy021-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Our results also showed methylation at CG, CHG, or CHH in the first exon or first intron associated with gene silencing. In soybean, there is positive correlation between CG gbM and expression levels, while CHG or CHH methylation in gene bodies was negatively associated with expression levels.<a reveal-id="dsy021-B40" data-open="dsy021-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> In our study, CHG and CHH methylation in exon or intron regions results in lower expression levels, while there was no negative association between CG methylation in exons (except for in the first exon) and expression levels. This indicates that CHG and CHH methylation in exon or intron regions are associated with gene silencing. In contrast, the frequency of moderately expressed genes (Group-4) having only CG methylation or gbM was higher than in the total genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>), indicating that genes having gbM showed higher expression levels, which is consistent with gbM genes in other plant species.<a reveal-id="dsy021-B17" data-open="dsy021-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>                            <h3 scrollto-destination=122670971 id="122670971" class="section-title" >4.3. Conservation and diversification of DNA methylation states of genic regions between <em>B. rapa</em> and <em>A. thaliana</em></h3>
<p>We categorized genes into three groups by DNA methylation types, gbM, heavily methylated, and under-methylated genes (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Of 911 heavily methylated genes, 424 (46.5%) had sequence homology to genes in <em>A. thaliana</em>, while more than 97% of gbM genes and under-methylated genes had sequence homology to genes in <em>A. thaliana</em> (<a reveal-id="dsy021-T2" data-open="dsy021-T2" class="link link-reveal link-table xref-fig">Table 2</a>), indicating that heavily methylated genes tend to lose orthologous genes in <em>B. rapa</em>. <em>A. thaliana</em> genes, which showed sequence similarity to heavily methylated genes in <em>B. rapa</em>, did not tend to have DNA methylation. In <em>B. rapa</em>, most heavily methylated genes contained IRRs in the exon/intron regions that were heavily methylated. These results suggest heavy methylation in genes is not conserved between species and heavy methylation occurs after speciation and is caused by transposon insertion that can drive variation of DNA methylation between species.<a reveal-id="dsy021-B41" data-open="dsy021-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsy021-B42" data-open="dsy021-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p><p>Conservation of gbM in orthologous genes between species, especially in closely related species, has been found.<a reveal-id="dsy021-B16" data-open="dsy021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy021-B41" data-open="dsy021-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsy021-B43" data-open="dsy021-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsy021-B44" data-open="dsy021-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> In our study, there was a significant correlation in gbM between orthologous genes in <em>B. rapa</em> and <em>A. thaliana</em>. Significant correlation in gbM between paralogous genes was also found in <em>B. rapa</em>. These results suggest that gbM is conserved not only after speciation but also after WGT. Although gbM is conserved between <em>A. thaliana</em> and <em>B. rapa</em>, about 14% of genes had gbM in <em>A. thaliana</em>, while only 0.5% of genes had gbM in <em>B. rapa.</em><a reveal-id="dsy021-B16" data-open="dsy021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> In this study, about 23% of genes had gbM in <em>A. thaliana</em>, and 12% of genes had gbM in <em>B. rapa.</em> Although the percentages are different because of the different analysis methods, the lower frequency of gbM genes in <em>B. rapa</em> than in <em>A. thaliana</em> is consistent. In <em>Eutrema salsugineum</em> and <em>Conringia planisiliqua</em>, loss of CMT3 function resulted in loss of gbM,<a reveal-id="dsy021-B16" data-open="dsy021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy021-B45" data-open="dsy021-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> suggesting that the lower numbers of gbM genes in <em>B. rapa</em> than in <em>A. thaliana</em> may be difference of CMT3 function.</p>                            <h3 scrollto-destination=122670974 id="122670974" class="section-title" >4.4. siRNAs are associated with DNA methylation</h3>
<p>24 nt-siRNAs guide methylation of cytosine in all classes of sequence contexts, termed RdDM, and the most abundant targets of RdDM are repetitive sequences or transposons, especially smaller and younger ones in euchromatic regions in <em>A. thaliana.</em><a reveal-id="dsy021-B10" data-open="dsy021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy021-B12" data-open="dsy021-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> We identified 65,142 clusters of 24 nt-siRNAs using uniquely mapped reads and 56% of the clusters were detected in IRRs. This frequency of the 24 nt-siRNA clusters in IRRs is lower than that of H3K9me2 peaks (73.3%) and MeDIP-peaks (86.5%) in IRRs.<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The 24 nt-siRNA levels or the 24 nt-siRNA clusters were more associated with CHH methylation than CG and CHG methylation in <em>B. rapa</em>, suggesting that this CHH methylation was via RdDM. Surprisingly, the average of methylation levels for CG, CHG, and CHH sites in the regions overlapping 24 nt-siRNA clusters was quite high even in the non-IRRs, indicating that 24 nt-siRNA clusters are strongly associated with DNA methylation.</p><p>We examined whether 24 nt-siRNA clusters nearby or within genes affect their expression. Genes having 24 nt-siRNA clusters in the 2-kb upstream or 2-kb downstream did not affect gene expression levels. In contrast, genes having 24 nt-siRNA clusters in exons showed a broad range of expression levels and genes having 24 nt-siRNA clusters in the exon regions tended to show higher expression levels. There is a tendency for negative correlation between the DNA methylation and expression level in genes having 24 nt-siRNA clusters in the exon regions, however, 24 nt-siRNA clusters in exons were not associated with DNA methylation in some genes, especially genes showing higher expression.</p>                            <h3 scrollto-destination=122670977 id="122670977" class="section-title" >4.5. Moderate association between H3K9me2 and DNA methylation</h3>
<p>The level of H3K9me2 associated with DNA methylation levels and methylation levels at CG, CHG, and CHH sites in the H3K9me2 regions was higher than the genome average levels, similar to the results in <em>A. thaliana.</em><a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The greatest fraction of CHG methylation overlapped with H3K9me2 positive regions in <em>A. thaliana</em>,<a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> while there was no tendency for H3K9me2 to be associated with CHG methylation in <em>B. rapa.</em> A greater fraction of CHH methylation overlapped with H3K9me2 in the IRRs. In <em>A. thaliana</em>, self-reinforcing mechanisms between CMT2/3 and KYP (SUVH4)/SUVH5/SUVH6 have been proposed, and CMT2 and CMT3 add CHH and CHG methylation, respectively.<a reveal-id="dsy021-B11" data-open="dsy021-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy021-B46" data-open="dsy021-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsy021-B47" data-open="dsy021-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The genome sequences of most euchromatic regions were determined in <em>B. rapa</em>, which is approximately half of the total genome size, but the sequences of the heterochromatic regions including centromere or pericentromere covering TEs and repetitive sequences were not determined.<a reveal-id="dsy021-B22" data-open="dsy021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> This may result in the weak correlation in <em>B. rapa</em> because association between H3K9me2 and CHG methylation levels are mostly high in TEs.<a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In maize there is no orthologous gene of AtCMT2, and there is no strong correlation between CHG methylation and H3K9me2.<a reveal-id="dsy021-B48" data-open="dsy021-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Although the putative orthologous genes of <em>AtCMT2</em>, <em>AtCMT3</em>, <em>AtKYP/AtSUVH4</em>, <em>AtSUVH5</em>, <em>AtSUVH6</em> were confirmed in the <em>B. rapa</em> genome,<a reveal-id="dsy021-B22" data-open="dsy021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy021-B49" data-open="dsy021-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsy021-B50" data-open="dsy021-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> the basis of no strong association between CHG and H3K9me2 is that the interaction between cytosine methyltransferase and H3K9 methyltransferase in <em>B. rapa</em> may be different from that in <em>A. thaliana</em>, which may explain the smaller number of gbM genes in <em>B. rapa</em>.</p><p>In <em>A. thaliana</em>, H3K9me2 methylation is associated with gene silencing, especially in pericentromeric regions.<a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Although the number of genes having H3K9me2 peaks in exon and intron regions was small, the average expression levels in genes having H3K9me2 in the exon and intron regions were lower than the total genes. However, the proportion of non-expressed genes having H3K9me2 was lower than genes having CG, CHG, and CHH methylation or MeDIP-peaks,<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> suggesting that in <em>B. rapa</em> some genes that were expressed at a lower level but not completely silenced, had H3K9me2.</p><p>In <em>A. thaliana</em>, TEs located in pericentromeric regions are associated more with high levels of H3K9me2 than in euchromatic arms. In this study, reads mapped to IRRs from ChIP-seq using H3K9me2 antibody (47%) was higher than that of Input-DNA-seq (39%), but this percentage is lower than that of MeDIP-seq (64%),<a reveal-id="dsy021-B30" data-open="dsy021-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> suggesting that H3K9me2 shows moderate association with IRRs in <em>B. rapa</em>. In <em>A. thaliana</em>, siRNA clusters were associated with H3K9me2 in pericentromeric regions, but generally not in euchromatic arms.<a reveal-id="dsy021-B5" data-open="dsy021-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In maize, H3K9me2 levels at RdDM loci were similar to genic loci and 2-fold lower than non-RdDM loci.<a reveal-id="dsy021-B48" data-open="dsy021-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> In <em>B. rapa</em>, an association between H3K9me2 and 24 nt-siRNA clusters was not detected. We could not examine the relationship between H3K9me2 and 24 nt-siRNA clusters or IRRs in pericentromere or centromere regions due to the limitation of available reference genome sequence information. At least in the euchromatic regions of <em>B. rapa</em>, H3K9me2 is moderately associated with IRRs, but the 24 nt-siRNA clusters did not associate with H3K9me2. However, immunostaining using H3K9me2 antibody in interphase nuclei revealed H3K9me2 was present in heterochromatin of chromocenters in <em>B. rapa</em>, similar to the results in <em>A. thaliana</em>,<a reveal-id="dsy021-B51" data-open="dsy021-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsy021-B52" data-open="dsy021-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> suggesting that H3K9me2 might be associated with IRRs or 24 nt-siRNA clusters in pericentromere or centromere regions in <em>B. rapa</em>.</p>                            <h2 scrollto-destination=122670981 id="122670981" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Etsuko Itabashi, Kiyomi Imamura, Makiko Tosaka, Taiji Kikuchi, and Terumi Horiuchi for their technical assistance. This work was supported by MEXT KAKENHI (No. 221S0002), by an Open Partnership Joint Projects of JSPS Bilateral Joint Research Projects (14544567), and by Grant-in-Aid for Young Scientists (B) (2478002), Scientific Research on Innovative Areas (24113509), and Grant-in-Aid Young Scientists (A) (15H05614) (JSPS) to R. Fujimoto, and grants from RIKEN and Japan Science and Technology Agency (JST) [Core Research for Evolutionary Science and Technology (JPMJCR13B4) (CREST)] to M.S.</p>                            <h3 scrollto-destination=122670983 id="122670983" class="section-title" >Accession numbers</h3>
<p>All sequence data have been deposited with DDBJ under DRA003120, DRA003121, and DRA003122.</p>                            <h3 scrollto-destination=122670985 id="122670985" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=122670987 id="122670987" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy021-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Osabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kawanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Molecular mechanisms of epigenetic variation in plants</div>, <div class="source">Int. J. Mol. Sci</div> ., <div class="volume">13</div>, <div class="fpage">9900</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20epigenetic%20variation%20in%20plants&amp;author=R.%20Fujimoto&amp;author=T.%20Sasaki&amp;author=R.%20Ishikawa&amp;author=K.%20Osabe&amp;author=T.%20Kawanabe&amp;author=E.%20S.%20Dennis&amp;publication_year=2012&amp;journal=Int.%20J.%20Mol.%20Sci&amp;volume=13&amp;pages=9900-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms13089900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms13089900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms13089900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22949838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quadrana</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Colot</div> <div class="given-names">V.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Plant transgenerational epigenetics</div>, <div class="source">Annu. Rev. Genet</div> ., <div class="volume">50</div>, <div class="fpage">467</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transgenerational%20epigenetics&amp;author=L.%20Quadrana&amp;author=V.%20Colot&amp;publication_year=2016&amp;journal=Annu.%20Rev.%20Genet&amp;volume=50&amp;pages=467-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-120215-035254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-120215-035254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-120215-035254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27732791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuchs</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Demidov</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Houben</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Schubert</div> <div class="given-names">I.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Chromosomal histone modification patterns-from conservation to diversity</div>, <div class="source">Trends Plant Sci</div> ., <div class="volume">11</div>, <div class="fpage">199</div>–<div class="lpage">208</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20histone%20modification%20patterns-from%20conservation%20to%20diversity&amp;author=J.%20Fuchs&amp;author=D.%20Demidov&amp;author=A.%20Houben&amp;author=I.%20Schubert&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci&amp;volume=11&amp;pages=199-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.02.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.02.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.02.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16546438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">U. S.</div></div>, <div class="name string-name"><div class="surname">Wagner</div> <div class="given-names">D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Tug of war: adding and removing histone lysine methylation in <em>Arabidopsis</em></div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">34</div>, <div class="fpage">41</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tug%20of%20war%3A%20adding%20and%20removing%20histone%20lysine%20methylation%20in%20Arabidopsis&amp;author=J.%20Xiao&amp;author=U.%20S.%20Lee&amp;author=D.%20Wagner&amp;publication_year=2016&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=34&amp;pages=41-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2016.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2016.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2016.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27614255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bernatavichute</div> <div class="given-names">Y. V.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pellegrini</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Genome-wide association of histone H3 lysine nine methylation with CHG DNA methylation in <em>Arabidopsis thaliana</em></div>, <div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e3156</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20of%20histone%20H3%20lysine%20nine%20methylation%20with%20CHG%20DNA%20methylation%20in%20Arabidopsis%20thaliana&amp;author=Y.%20V.%20Bernatavichute&amp;author=X.%20Zhang&amp;author=S.%20Cokus&amp;author=M.%20Pellegrini&amp;author=S.%20E.%20Jacobsen&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e3156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18776934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">L. M.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div>, <div class="name string-name"><div class="surname">Patel</div> <div class="given-names">D. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">DNA methylation pathways and their crosstalk with histone methylation</div>, <div class="source">Nat. Rev. Mol. Cell Biol</div> ., <div class="volume">16</div>, <div class="fpage">519</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20pathways%20and%20their%20crosstalk%20with%20histone%20methylation&amp;author=J.%20Du&amp;author=L.%20M.%20Johnson&amp;author=S.%20E.%20Jacobsen&amp;author=D.%20J.%20Patel&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;volume=16&amp;pages=519-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm4043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm4043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm4043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26296162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Becker</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hagmann</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Müller</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Spontaneous epigenetic variation in the <em>Arabidopsis thaliana</em> methylome</div>, <div class="source">Nature</div> , <div class="volume">480</div>, <div class="fpage">245</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spontaneous%20epigenetic%20variation%20in%20the%20Arabidopsis%20thaliana%20methylome&amp;author=C.%20Becker&amp;author=J.%20Hagmann&amp;author=J.%20M%C3%BCller&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=245-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22057020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Lewsey</div> <div class="given-names">M. G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Transgenerational epigenetic instability is a source of novel methylation variants</div>, <div class="source">Science</div> , <div class="volume">334</div>, <div class="fpage">369</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transgenerational%20epigenetic%20instability%20is%20a%20source%20of%20novel%20methylation%20variants&amp;author=R.%20J.%20Schmitz&amp;author=M.%20D.%20Schultz&amp;author=M.%20G.%20Lewsey&amp;publication_year=2011&amp;journal=Science&amp;volume=334&amp;pages=369-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1212959" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1212959" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1212959"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Osabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div></div> <div class="year">2012</div>, The role of DNA methylation in plants. In: <div class="person-group"><div class="name string-name"><div class="surname">Tatarinova</div> <div class="given-names">T.V.</div></div>, <div class="name string-name"><div class="surname">Sablok</div> <div class="given-names">G.</div></div></div> (eds) <div class="source">DNA Methylation: Principles, Mechanisms and Challenges</div> , 
<div class="publisher-name">NOVA Publishers</div>: 
<div class="publisher-loc">U.S.A, New York, Hauppauge</div>, pp. <div class="fpage">35</div>–<div class="lpage">66</div>.</div></div></div></div><div content-id="dsy021-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zemach</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">M. Y.</div></div>, <div class="name string-name"><div class="surname">Hsieh</div> <div class="given-names">P. H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The <em>Arabidopsis</em> nucleosome remodeler DDM1 allows DNA methyltransferases to access H1-containing heterochromatin</div>, <div class="source">Cell</div> , <div class="volume">153</div>, <div class="fpage">193</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20nucleosome%20remodeler%20DDM1%20allows%20DNA%20methyltransferases%20to%20access%20H1-containing%20heterochromatin&amp;author=A.%20Zemach&amp;author=M.%20Y.%20Kim&amp;author=P.%20H.%20Hsieh&amp;publication_year=2013&amp;journal=Cell&amp;volume=153&amp;pages=193-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.02.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.02.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.02.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23540698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stroud</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Do</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Non-CG methylation patterns shape the epigenetic landscape in <em>Arabidopsis</em></div>, <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">21</div>, <div class="fpage">64</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-CG%20methylation%20patterns%20shape%20the%20epigenetic%20landscape%20in%20Arabidopsis&amp;author=H.%20Stroud&amp;author=T.%20Do&amp;author=J.%20Du&amp;publication_year=2014&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=21&amp;pages=64-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2735" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2735" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2735"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24336224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matzke</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Mosher</div> <div class="given-names">R. A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">RNA-directed DNA methylation: an epigenetic pathway of increasing complexity</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">394</div>–<div class="lpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-directed%20DNA%20methylation%3A%20an%20epigenetic%20pathway%20of%20increasing%20complexity&amp;author=M.%20A.%20Matzke&amp;author=R.%20A.%20Mosher&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=394-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24805120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laird</div> <div class="given-names">P. W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Principles and challenges of genome-wide DNA methylation analysis</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">191</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principles%20and%20challenges%20of%20genome-wide%20DNA%20methylation%20analysis&amp;author=P.%20W.%20Laird&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=191-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2732" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20125086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Urich</div> <div class="given-names">M. A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Patterns of population epigenomic diversity</div>, <div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">193</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20population%20epigenomic%20diversity&amp;author=R.%20J.%20Schmitz&amp;author=M.%20D.%20Schultz&amp;author=M.%20A.%20Urich&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=193-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23467092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawakatsu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Jupe</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Epigenomic diversity in a global collection of <em>Arabidopsis thaliana</em> accessions</div>, <div class="source">Cell</div> , <div class="volume">166</div>, <div class="fpage">492</div>–<div class="lpage">505</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenomic%20diversity%20in%20a%20global%20collection%20of%20Arabidopsis%20thaliana%20accessions&amp;author=T.%20Kawakatsu&amp;author=S.%20C.%20Huang&amp;author=F.%20Jupe&amp;publication_year=2016&amp;journal=Cell&amp;volume=166&amp;pages=492-505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.06.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.06.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.06.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27419873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Niederhuth</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Bewick</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Ji</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Widespread natural variation of DNA methylation within angiosperms</div>, <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">194</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20natural%20variation%20of%20DNA%20methylation%20within%20angiosperms&amp;author=C.%20E.%20Niederhuth&amp;author=A.%20J.%20Bewick&amp;author=L.%20Ji&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-1059-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-1059-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-1059-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27671052" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vidalis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Živković</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wardenaar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Roquis</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tellier</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Johannes</div> <div class="given-names">F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Methylome evolution in plants</div>, <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methylome%20evolution%20in%20plants&amp;author=A.%20Vidalis&amp;author=D.%20%C5%BDivkovi%C4%87&amp;author=R.%20Wardenaar&amp;author=D.%20Roquis&amp;author=A.%20Tellier&amp;author=F.%20Johannes&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-1127-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-1127-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-1127-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27998290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lister</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">O'Malley</div> <div class="given-names">R. C.</div></div>, <div class="name string-name"><div class="surname">Tonti-Filippini</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Highly integrated single-base resolution maps of the epigenome in <em>Arabidopsis</em></div>, <div class="source">Cell</div> , <div class="volume">133</div>, <div class="fpage">523</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20integrated%20single-base%20resolution%20maps%20of%20the%20epigenome%20in%20Arabidopsis&amp;author=R.%20Lister&amp;author=R.%20C.%20O'Malley&amp;author=J.%20Tonti-Filippini&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=523-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.03.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.03.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.03.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18423832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Shotgun bisulphite sequencing of the <em>Arabidopsis</em> genome reveals DNA methylation patterning</div>, <div class="source">Nature</div> , <div class="volume">452</div>, <div class="fpage">215</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shotgun%20bisulphite%20sequencing%20of%20the%20Arabidopsis%20genome%20reveals%20DNA%20methylation%20patterning&amp;author=S.%20J.%20Cokus&amp;author=S.%20Feng&amp;author=X.%20Zhang&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=215-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Self-incompatibility</div>, <div class="source">Adv. Bot. Res</div> ., <div class="volume">45</div>, <div class="fpage">139</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-incompatibility&amp;author=R.%20Fujimoto&amp;author=T.%20Nishio&amp;publication_year=2007&amp;journal=Adv.%20Bot.%20Res&amp;volume=45&amp;pages=139-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0065-2296(07)45005-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0065-2296(07)45005-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0065-2296(07)45005-4"> </span></div> </div></div></div></div></div><div content-id="dsy021-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saeki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kawanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ying</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Molecular and cellular characteristics of hybrid vigour in a commercial hybrid of Chinese cabbage</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20cellular%20characteristics%20of%20hybrid%20vigour%20in%20a%20commercial%20hybrid%20of%20Chinese%20cabbage&amp;author=N.%20Saeki&amp;author=T.%20Kawanabe&amp;author=H.%20Ying&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-016-0734-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0734-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-016-0734-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26882898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em></div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">1035</div>–<div class="lpage">9</div>. <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Subgenome parallel selection is associated with morphotype diversification and convergent crop domestication in <em>Brassica rapa</em> and <em>Brassica oleracea</em></div>, <div class="source">Nat. Genet</div> , <div class="volume">48</div>, <div class="fpage">1218</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Subgenome%20parallel%20selection%20is%20associated%20with%20morphotype%20diversification%20and%20convergent%20crop%20domestication%20in%20Brassica%20rapa%20and%20Brassica%20oleracea&amp;author=F.%20Cheng&amp;author=R.%20Sun&amp;author=X.%20Hou&amp;publication_year=2016&amp;journal=Nat.%20Genet&amp;volume=48&amp;pages=1218-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3634" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3634" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3634"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27526322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Biased gene fractionation and dominant gene expression among the subgenomes of <em>Brassica rapa</em></div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e36442</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20gene%20fractionation%20and%20dominant%20gene%20expression%20among%20the%20subgenomes%20of%20Brassica%20rapa&amp;author=F.%20Cheng&amp;author=J.%20Wu&amp;author=L.%20Fang&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e36442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0036442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0036442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0036442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22567157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Woodhouse</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Altered patterns of fractionation and exon deletions in <em>Brassica rapa</em> support a two-step model of paleohexaploidy</div>, <div class="source">Genetics</div> , <div class="volume">190</div>, <div class="fpage">1563</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20patterns%20of%20fractionation%20and%20exon%20deletions%20in%20Brassica%20rapa%20support%20a%20two-step%20model%20of%20paleohexaploidy&amp;author=H.%20Tang&amp;author=M.%20R.%20Woodhouse&amp;author=F.%20Cheng&amp;publication_year=2012&amp;journal=Genetics&amp;volume=190&amp;pages=1563-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.111.137349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.111.137349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.111.137349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22308264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kawanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genetic distance of inbred lines of Chinese cabbage and its relationship to heterosis</div>, <div class="source">Plant Gene</div> , <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20distance%20of%20inbred%20lines%20of%20Chinese%20cabbage%20and%20its%20relationship%20to%20heterosis&amp;author=K.%20Kawamura&amp;author=T.%20Kawanabe&amp;author=M.%20Shimizu&amp;publication_year=2016&amp;journal=Plant%20Gene&amp;volume=5&amp;pages=1-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plgene.2015.10.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plgene.2015.10.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plgene.2015.10.003"> </span></div> </div></div></div></div></div><div content-id="dsy021-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krueger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Andrews</div> <div class="given-names">S. R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">1571</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bismark%3A%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications&amp;author=F.%20Krueger&amp;author=S.%20R.%20Andrews&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1571-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21493656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takuno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Body-methylated genes in <em>Arabidopsis thaliana</em> are functionally important and evolve slowly</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">219</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Body-methylated%20genes%20in%20Arabidopsis%20thaliana%20are%20functionally%20important%20and%20evolve%20slowly&amp;author=S.%20Takuno&amp;author=B.%20S.%20Gaut&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=219-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21813466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Osabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Itabashi</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Okazaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Development of primer sets that can verify the enrichment of histone modifications, and their application to examining vernalization-mediated chromatin changes in <em>Brassica rapa</em> L</div>, <div class="source">Genes Genet. Syst</div> ., <div class="volume">91</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20primer%20sets%20that%20can%20verify%20the%20enrichment%20of%20histone%20modifications%2C%20and%20their%20application%20to%20examining%20vernalization-mediated%20chromatin%20changes%20in%20Brassica%20rapa%20L&amp;author=T.%20Kawanabe&amp;author=K.%20Osabe&amp;author=E.%20Itabashi&amp;author=K.%20Okazaki&amp;author=E.%20S.%20Dennis&amp;author=R.%20Fujimoto&amp;publication_year=2016&amp;journal=Genes%20Genet.%20Syst&amp;volume=91&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.15-00058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.15-00058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.15-00058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27074983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fukushima</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Osabe</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2018</div>, 
<div class="article-title">Identification of DNA methylated regions by using methylated DNA immunoprecipitation sequencing in <em>Brassica rapa</em></div>, <div class="source">Crop Pasture Sci</div> ., <div class="volume">69</div>, <div class="fpage">107</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20DNA%20methylated%20regions%20by%20using%20methylated%20DNA%20immunoprecipitation%20sequencing%20in%20Brassica%20rapa&amp;author=S.%20Takahashi&amp;author=N.%20Fukushima&amp;author=K.%20Osabe&amp;publication_year=2018&amp;journal=Crop%20Pasture%20Sci&amp;volume=69&amp;pages=107-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/CP17394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FCP17394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FCP17394"> </span></div> </div></div></div></div></div><div content-id="dsy021-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ying</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Identification of candidate genes for fusarium yellows resistance in Chinese cabbage by differential expression analysis</div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">85</div>, <div class="fpage">247</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20candidate%20genes%20for%20fusarium%20yellows%20resistance%20in%20Chinese%20cabbage%20by%20differential%20expression%20analysis&amp;author=M.%20Shimizu&amp;author=R.%20Fujimoto&amp;author=H.%20Ying&amp;publication_year=2014&amp;journal=Plant%20Mol.%20Biol&amp;volume=85&amp;pages=247-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-014-0182-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-014-0182-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-014-0182-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24668026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A comparison of transcriptome and epigenetic status between closely related species in the genus <em>Arabidopsis</em></div>, <div class="source">Gene</div> , <div class="volume">506</div>, <div class="fpage">301</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20transcriptome%20and%20epigenetic%20status%20between%20closely%20related%20species%20in%20the%20genus%20Arabidopsis&amp;author=T.%20Kawanabe&amp;author=R.%20Fujimoto&amp;author=T.%20Sasaki&amp;author=J.%20M.%20Taylor&amp;author=E.%20S.%20Dennis&amp;publication_year=2012&amp;journal=Gene&amp;volume=506&amp;pages=301-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.07.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.07.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.07.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22796129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ge</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">King</div> <div class="given-names">G. J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">K.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide DNA methylation profiling by modified reduced representation bisulfite sequencing in <em>Brassica rapa</em> suggests that epigenetic modifications play a key role in polyploid genome evolution</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">836</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20DNA%20methylation%20profiling%20by%20modified%20reduced%20representation%20bisulfite%20sequencing%20in%20Brassica%20rapa%20suggests%20that%20epigenetic%20modifications%20play%20a%20key%20role%20in%20polyploid%20genome%20evolution&amp;author=X.%20Chen&amp;author=X.%20Ge&amp;author=J.%20Wang&amp;author=C.%20Tan&amp;author=G.%20J.%20King&amp;author=K.%20Liu&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=836" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26500672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Hypomethylation and transcriptional reactivation of retrotransposon-like sequences in <em>ddm1</em> transgenic plants of <em>Brassica rapa</em></div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">66</div>, <div class="fpage">463</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypomethylation%20and%20transcriptional%20reactivation%20of%20retrotransposon-like%20sequences%20in%20ddm1%20transgenic%20plants%20of%20Brassica%20rapa&amp;author=R.%20Fujimoto&amp;author=T.%20Sasaki&amp;author=H.%20Inoue&amp;author=T.%20Nishio&amp;publication_year=2008&amp;journal=Plant%20Mol.%20Biol&amp;volume=66&amp;pages=463-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-007-9285-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-007-9285-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-007-9285-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18236011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kishitani</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Analysis of target sequences of DDM1s in <em>Brassica rapa</em> by MSAP</div>, <div class="source">Plant Cell Rep</div> ., <div class="volume">30</div>, <div class="fpage">81</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20target%20sequences%20of%20DDM1s%20in%20Brassica%20rapa%20by%20MSAP&amp;author=T.%20Sasaki&amp;author=R.%20Fujimoto&amp;author=S.%20Kishitani&amp;author=T.%20Nishio&amp;publication_year=2011&amp;journal=Plant%20Cell%20Rep&amp;volume=30&amp;pages=81-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-010-0946-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-010-0946-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-010-0946-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21072521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yazaki</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sundaresan</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Genome-wide high-resolution mapping and functional analysis of DNA methylation in <em>Arabidopsis</em></div>, <div class="source">Cell</div> , <div class="volume">126</div>, <div class="fpage">1189</div>–<div class="lpage">201</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20high-resolution%20mapping%20and%20functional%20analysis%20of%20DNA%20methylation%20in%20Arabidopsis&amp;author=X.%20Zhang&amp;author=J.%20Yazaki&amp;author=A.%20Sundaresan&amp;publication_year=2006&amp;journal=Cell&amp;volume=126&amp;pages=1189-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16949657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Single-base resolution maps of cultivated and wild rice methylomes and regulatory roles of DNA methylation in plant gene expression</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-base%20resolution%20maps%20of%20cultivated%20and%20wild%20rice%20methylomes%20and%20regulatory%20roles%20of%20DNA%20methylation%20in%20plant%20gene%20expression&amp;author=X.%20Li&amp;author=J.%20Zhu&amp;author=F.%20Hu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">You</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Whole-genome DNA methylation patterns and complex associations with gene structure and expression during flower development in Arabidopsis</div>, <div class="source">Plant J</div> ., <div class="volume">81</div>, <div class="fpage">268</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20DNA%20methylation%20patterns%20and%20complex%20associations%20with%20gene%20structure%20and%20expression%20during%20flower%20development%20in%20Arabidopsis&amp;author=H.%20Yang&amp;author=F.%20Chang&amp;author=C.%20You&amp;publication_year=2015&amp;journal=Plant%20J&amp;volume=81&amp;pages=268-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2015.81.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2015.81.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2015.81.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25404462" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brenet</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Moh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Funk</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">DNA methylation of the first exon is tightly linked to transcriptional silencing</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e14524</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20of%20the%20first%20exon%20is%20tightly%20linked%20to%20transcriptional%20silencing&amp;author=F.%20Brenet&amp;author=M.%20Moh&amp;author=P.%20Funk&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e14524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0014524" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0014524" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0014524"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21267076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Valdés-López</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">1663</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenome-wide%20inheritance%20of%20cytosine%20methylation%20variants%20in%20a%20recombinant%20inbred%20population&amp;author=R.%20J.%20Schmitz&amp;author=Y.%20He&amp;author=O.%20Vald%C3%A9s-L%C3%B3pez&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1663-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.152538.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.152538.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.152538.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23739894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seymour</div> <div class="given-names">D. K.</div></div>, <div class="name string-name"><div class="surname">Koenig</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Hagmann</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Becker</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Evolution of DNA methylation patterns in the Brassicaceae is driven by differences in genome organization</div>, <div class="source">PLoS Genet</div> ., <div class="volume">10</div>, <div class="fpage">e1004785</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20DNA%20methylation%20patterns%20in%20the%20Brassicaceae%20is%20driven%20by%20differences%20in%20genome%20organization&amp;author=D.%20K.%20Seymour&amp;author=D.%20Koenig&amp;author=J.%20Hagmann&amp;author=C.%20Becker&amp;author=D.%20Weigel&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004785" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25393550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Quiros</div> <div class="given-names">C.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Comparative sequence analysis for <em>Brassica oleracea</em> with similar sequences in <em>B. rapa</em> and <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell Rep</div> ., <div class="volume">28</div>, <div class="fpage">649</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20sequence%20analysis%20for%20Brassica%20oleracea%20with%20similar%20sequences%20in%20B.%20rapa%20and%20Arabidopsis%20thaliana&amp;author=D.%20Qiu&amp;author=M.%20Gao&amp;author=G.%20Li&amp;author=C.%20Quiros&amp;publication_year=2009&amp;journal=Plant%20Cell%20Rep&amp;volume=28&amp;pages=649-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-008-0661-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-008-0661-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-008-0661-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19112567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takuno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Gene body methylation is conserved between plant orthologs and is of evolutionary consequence</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">110</div>, <div class="fpage">1797</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20body%20methylation%20is%20conserved%20between%20plant%20orthologs%20and%20is%20of%20evolutionary%20consequence&amp;author=S.%20Takuno&amp;author=B.%20S.%20Gaut&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=110&amp;pages=1797-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1215380110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1215380110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1215380110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23319627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takuno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ran</div> <div class="given-names">J. H.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Evolutionary patterns of genic DNA methylation vary across land plants</div>, <div class="source">Nat. Plants</div> , <div class="volume">2</div>, <div class="fpage">15222</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20patterns%20of%20genic%20DNA%20methylation%20vary%20across%20land%20plants&amp;author=S.%20Takuno&amp;author=J.%20H.%20Ran&amp;author=B.%20S.%20Gaut&amp;publication_year=2016&amp;journal=Nat.%20Plants&amp;volume=2&amp;pages=15222" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2015.222" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2015.222" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2015.222"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27249194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bewick</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Ji</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Niederhuth</div> <div class="given-names">C. E.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">On the origin and evolutionary consequences of gene body DNA methylation</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">113</div>, <div class="fpage">9111</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20origin%20and%20evolutionary%20consequences%20of%20gene%20body%20DNA%20methylation&amp;author=A.%20J.%20Bewick&amp;author=L.%20Ji&amp;author=C.%20E.%20Niederhuth&amp;publication_year=2016&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=113&amp;pages=9111-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1604666113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1604666113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1604666113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27457936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ebbs</div> <div class="given-names">M. L.</div></div>, <div class="name string-name"><div class="surname">Bender</div> <div class="given-names">J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Locus-specific control of DNA methylation by the <em>Arabidopsis</em> SUVH5 histone methyltransferase</div>, <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">1166</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locus-specific%20control%20of%20DNA%20methylation%20by%20the%20Arabidopsis%20SUVH5%20histone%20methyltransferase&amp;author=M.%20L.%20Ebbs&amp;author=J.%20Bender&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1166-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.041400" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.041400" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.041400"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16582009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Lindroth</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Control of CpNpG DNA methylation by the KRYPTONITE histone H3 methyltransferase</div>, <div class="source">Nature</div> , <div class="volume">416</div>, <div class="fpage">556</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20CpNpG%20DNA%20methylation%20by%20the%20KRYPTONITE%20histone%20H3%20methyltransferase&amp;author=J.%20P.%20Jackson&amp;author=A.%20M.%20Lindroth&amp;author=X.%20Cao&amp;author=S.%20E.%20Jacobsen&amp;publication_year=2002&amp;journal=Nature&amp;volume=416&amp;pages=556-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature731" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature731" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature731"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11898023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gent</div> <div class="given-names">J. I.</div></div>, <div class="name string-name"><div class="surname">Madzima</div> <div class="given-names">T. F.</div></div>, <div class="name string-name"><div class="surname">Bader</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation</div>, <div class="source">Plant Cell</div> ., <div class="volume">26</div>, <div class="fpage">4903</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accessible%20DNA%20and%20relative%20depletion%20of%20H3K9me2%20at%20maize%20loci%20undergoing%20RNA-directed%20DNA%20methylation&amp;author=J.%20I.%20Gent&amp;author=T.%20F.%20Madzima&amp;author=R.%20Bader&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=4903-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.130427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.130427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.130427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25465407" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujimoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Characterization of DNA methyltransferase genes in <em>Brassica rapa</em></div>, <div class="source">Genes Genet. Syst</div> ., <div class="volume">81</div>, <div class="fpage">235</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20DNA%20methyltransferase%20genes%20in%20Brassica%20rapa&amp;author=R.%20Fujimoto&amp;author=T.%20Sasaki&amp;author=T.%20Nishio&amp;publication_year=2006&amp;journal=Genes%20Genet.%20Syst&amp;volume=81&amp;pages=235-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.81.235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.81.235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.81.235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17038795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Diversification and evolution of the <em>SDG</em> gene family in <em>Brassica rapa</em> after the whole genome triplication</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">16851</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20and%20evolution%20of%20the%20SDG%20gene%20family%20in%20Brassica%20rapa%20after%20the%20whole%20genome%20triplication&amp;author=H.%20Dong&amp;author=D.%20Liu&amp;author=T.%20Han&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=16851" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep16851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep16851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep16851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26596461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Braszewska-Zalewska</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bernas</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Maluszynska</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Epigenetic chromatin modifications in <em>Brassica</em> genomes</div>, <div class="source">Genome</div> , <div class="volume">53</div>, <div class="fpage">203</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20chromatin%20modifications%20in%20Brassica%20genomes&amp;author=A.%20Braszewska-Zalewska&amp;author=T.%20Bernas&amp;author=J.%20Maluszynska&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=203-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20237597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy021-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy021-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Braszewska-Zalewska</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dziurlikowska</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Maluszynska</div> <div class="given-names">J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Histone H3 methylation patterns in <em>Brassica nigra</em>, <em>Brassica juncea</em>, and <em>Brassica carinata</em> species</div>, <div class="source">Genome</div> , <div class="volume">55</div>, <div class="fpage">68</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Histone%20H3%20methylation%20patterns%20in%20Brassica%20nigra%2C%20Brassica%20juncea%2C%20and%20Brassica%20carinata%20species&amp;author=A.%20Braszewska-Zalewska&amp;author=A.%20Dziurlikowska&amp;author=J.%20Maluszynska&amp;publication_year=2012&amp;journal=Genome&amp;volume=55&amp;pages=68-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22195975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743808" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5047107&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_figures.pptx?Expires=2147483647&amp;Signature=GUDg5IcJyHEfjjSnrWe~icQw64IO-KXTKo37SQwnvNqbN79BYISn5eCx52ikmN2b-jjeZu3MoYC-9qW0DeX2cRMPBzWitTXTwNHyC1UZ5V77SHCaN8ArEvgoM9MlWkjbqMmtjgXqHSBcbEjTGo3Y6u8635jX4QFLvKoT-JLh0g3CbEyFwJHh21BC8nqllfvKn~hQvQV2~QmI7bRGPQrMo358DUKe5oNlif6kjzjGIG1zfRp72107Tf5WqvWPHKl0xApSHw069f1T~~nIHtI810d8nvjR36VOSvGHWuuQzUfrR-zcJBfZVF7bqXc8WSbGwX7jDkOD1-cEeMpPkZuZDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures</div></a> - pptx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy021/1/dsy021_supplementary_tables.xls?Expires=2147483647&amp;Signature=fxo9o-499LExP4PY2ei4a~0O0H4pxXMhY9fZ5s~RvH~Z5JonPJOvUjlaxOefszB5HmCkfo1OJ4awcle4cYyKbAU0L9ubGez4sOCTgtUVz-t9MpuFvZSZPfNQimOmAnQvvxg836bRahHpy0yM1cnBZwyYz-DoZyYfSFXvA6Ap8tUC-PjBTEPsEPVGg8zCcx4os41dtlvCzSTrKzUr89enEyaNCSctEiXCp~RQYuYwogdA4OEzsmWqOXNAREIWsxkDWQZago6gRpY6XIakNiZYg38sFX5MvwWV34IyQFgf595xZgR39twJmN0xgmLbB5~jBLcUXU-pp7KzetZ37NC7zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Tables</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">959</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5047107"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f511%2f5047107">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F511%2F5047107" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5047107'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide characterization of DNA methylation, small RNA expression, and histone H3 lysine nine di-methylation in Brassica rapa L." />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f511%2f5047107">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

