

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/2/197/334072","@type":"ScholarlyArticle","name":"Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [ Setaria italica  (L.)]","datePublished":"2013-02-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst002","keywords":["foxtail millet (<italic>Setaria italica</italic> L.)","microsatellite","comparative mapping","physical mapping","transferability"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/2/197/334072","author":[{"name":"Pandey, Garima","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Misra, Gopal","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Kumari, Kajal","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Gupta, Sarika","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Parida, Swarup Kumar","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Chattopadhyay, Debasis","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Prasad, Manoj","affiliation":"National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"}],"description":"Abstract.  The availability of well-validated informative co-dominant microsatellite markers and saturated genetic linkage map has been limited in foxtail mille","pageStart":"197","pageEnd":"207","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [ Setaria italica  (L.)]","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [ Setaria italica  (L.)]" />
    <meta property="og:description" content="Abstract.  The availability of well-validated informative co-dominant microsatellite markers and saturated genetic linkage map has been limited in foxtail mille" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/2/197/334072" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-02-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Pandey, Garima" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Misra, Gopal" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Kumari, Kajal" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Gupta, Sarika" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Parida, Swarup Kumar" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Chattopadhyay, Debasis" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Prasad, Manoj" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_title" content="Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [Setaria italica (L.)]" /><meta name="citation_firstpage" content="197" /><meta name="citation_lastpage" content="207" /><meta name="citation_doi" content="10.1093/dnares/dst002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2013/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome relationships: the grass model in current research; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=637-46; " /><meta name="citation_reference" content="citation_title=Foxtail millet: a sequence-driven grass model system; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=137-41; " /><meta name="citation_reference" content="citation_title=Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses; citation_text=. (September 18, 2012); Crit. Rev. Biotechnol.;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Setaria viridis and Setaria italica, model genetic systems for Panicoid grasses; J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=3031-37; " /><meta name="citation_reference" content="citation_title=Reference genome sequence of the model plant Setaria; Nature Biotech.;  citation_year=2012;  citation_volume=30;  citation_pages=555-61; " /><meta name="citation_reference" content="citation_title=Genome sequence of foxtail millet (Setaria italica) provides insights into grass evolution and biofuel potential; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=549-54; " /><meta name="citation_reference" content="citation_title=The foxtail millet genome and its relationship amongst the grasses; citation_text=W328 (abstract); citation_publisher=San Diego, CA;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Detection of sequences with Z-DNA forming potential in higher plants; Biochem. Biophys. Res. Commun.;  citation_year=1983;  citation_volume=116;  citation_pages=113-20; " /><meta name="citation_reference" content="citation_title=Simple sequences are ubiquitous repetitive components of eukaryotic genomes; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=4127-38; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in plants: features and applications; Trends Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Unigene derived microsatellite markers for the cereal genomes; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=808-17; " /><meta name="citation_reference" content="citation_title=Informative genomic microsatellite markers for efficient genotyping applications in sugarcane; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=327-38; " /><meta name="citation_reference" content="citation_title=Microsatellites in Brassica unigenes: relative abundance, marker design and use in comparative physical mapping and genome analysis; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Development of simple sequence repeats (SSR) markers in Setaria italica (Poaceae) and cross-amplification in related species; Int. J. Mol. Sci.;  citation_year=2011;  citation_volume=12;  citation_pages=7835-45; " /><meta name="citation_reference" content="citation_title=Sequence based novel genomic microsatellite markers for robust genotyping purposes in foxtail millet [Setaria italica (L.) P. Beauv]; Plant Cell Rep.;  citation_year=2012;  citation_volume=31;  citation_pages=323-37; " /><meta name="citation_reference" content="citation_title=Development of EST-SSR in foxtail millet (Setaria italica); J. Genet. Res. Crop Evol.;  citation_year=2007;  citation_volume=54;  citation_pages=233-6; " /><meta name="citation_reference" content="citation_title=Highly variable SSR markers suitable for rice genotyping using agarose gels; Mol. Breed.;  citation_year=2010;  citation_volume=25;  citation_pages=359-64; " /><meta name="citation_reference" content="citation_title=Development and application of a whole-genome simple sequence repeat panel for high-throughput genotyping in soybean; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=107-15; " /><meta name="citation_reference" content="citation_title=Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in Brachypodium; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21298" /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Nouvelle recherches sur la distribution florale; Bull Soc. Vaud. Sci. Nat.;  citation_year=1908;  citation_volume=44;  citation_pages=223-70; " /><meta name="citation_reference" content="Rohlf F.J. 1997 Setauket, NY NTSYS-pc numerical taxonomy and multivariance analysis system Version 2.02e, Exeter Software " /><meta name="citation_reference" content="citation_title=Microsatellites markers: an overview of the recent progress in Plants; Euphytica;  citation_year=2011;  citation_volume=177;  citation_pages=309-34; " /><meta name="citation_reference" content="citation_title=PCR-amplified microsatellites as markers in plant genetics; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=175-82; " /><meta name="citation_reference" content="citation_title=Mining microsatellites in eukaryotic genomes; Trends Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=490-8; " /><meta name="citation_reference" content="citation_title=Characterization of microsatellites in the tribe bambuseae; Gene Conserv.;  citation_year=2011;  citation_volume=10;  citation_pages=51-64; " /><meta name="citation_reference" content="citation_title=A genome-wide survey of switchgrass genome structure and organization; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e33892" /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, lengthvariation, transposon associations and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-52; " /><meta name="citation_reference" content="citation_title=Assesment of genetic relationship of foxtail millet with its wild ancestor and close relatives by ISSR markers; J. Int. Agri.;  citation_year=2012;  citation_volume=11;  citation_pages=556-66; " /><meta name="citation_reference" content="citation_title=Genetic structure of landraces in foxtail millet (Setaria italica (L.) P. Beauv.) revealed with transposon display and interpretation to crop evolution of foxtail millet; Genome;  citation_year=2011;  citation_volume=54;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Development and characterization of simple sequence repeat (SSR) markers for perennial ryegrass (Lolium perenne. L.); Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=405-15; " /><meta name="citation_reference" content="citation_title=Transferability of rice SSR markers to bamboo; Euphytica;  citation_year=2010;  citation_volume=175;  citation_pages=23-33; " /><meta name="citation_reference" content="citation_title=Novel set of groundnut SSRs for genetic diversity and inter-specific transferability; Int. J. Integr. Biol.;  citation_year=2009;  citation_volume=7;  citation_pages=100-6; " /><meta name="citation_reference" content="citation_title=Comparative genetic maps of foxtail millet (Setaria italica) and rice (Oryza sativa); Theor. Appl. Genet.;  citation_year=1998;  citation_volume=96;  citation_pages=63-8; " /><meta name="citation_reference" content="citation_title=Updating the crop circle; Curr. Opin. Plant Biol.;  citation_year=2005;  citation_volume=8;  citation_pages=155-62; " /><meta name="citation_reference" content="citation_title=Comparative analyses reveal high levels of conserved colinearity between the finger millet and rice genomes; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=489-99; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/2/197/955939/dst002.pdf" /><meta name="description" content="Abstract.  The availability of well-validated informative co-dominant microsatellite markers and saturated genetic linkage map has been limited in foxtail mille" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/2/197/334072" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/2/197/334072" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/2/197/334072");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst002';
    </script>
<title>
	Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [Setaria italica (L.)] | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./334072" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BepTPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYW5kZXksIEdhcmltYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWlzcmEsIEdvcGFsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdW1hcmksIEthamFsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdXB0YSwgU2FyaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQYXJpZGEsIFN3YXJ1cCBLdW1hciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hhdHRvcGFkaHlheSwgRGViYXNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUHJhc2FkLCBNYW5vaiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtV2lkZSBEZXZlbG9wbWVudCBhbmQgVXNlIG9mIE1pY3Jvc2F0ZWxsaXRlIE1hcmtlcnMgZm9yIExhcmdlLVNjYWxlIEdlbm90eXBpbmcgQXBwbGljYXRpb25zIGluIEZveHRhaWwgTWlsbGV0IFtTZXRhcmlhIGl0YWxpY2EgKEwuKV0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxOTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHJlbGF0aW9uc2hpcHM6IHRoZSBncmFzcyBtb2RlbCBpbiBjdXJyZW50IHJlc2VhcmNoOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3h0YWlsIG1pbGxldDogYSBzZXF1ZW5jZS1kcml2ZW4gZ3Jhc3MgbW9kZWwgc3lzdGVtOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZveHRhaWwgbWlsbGV0OiBhIG1vZGVsIGNyb3AgZm9yIGdlbmV0aWMgYW5kIGdlbm9taWMgc3R1ZGllcyBpbiBiaW9lbmVyZ3kgZ3Jhc3NlczsgY2l0YXRpb25fdGV4dD0uIChTZXB0ZW1iZXIgMTgsIDIwMTIpOyBDcml0LiBSZXYuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXRhcmlhIHZpcmlkaXMgYW5kIFNldGFyaWEgaXRhbGljYSwgbW9kZWwgZ2VuZXRpYyBzeXN0ZW1zIGZvciBQYW5pY29pZCBncmFzc2VzOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0zMDMxLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWZlcmVuY2UgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBtb2RlbCBwbGFudCBTZXRhcmlhOyBOYXR1cmUgQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBmb3h0YWlsIG1pbGxldCAoU2V0YXJpYSBpdGFsaWNhKSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGdyYXNzIGV2b2x1dGlvbiBhbmQgYmlvZnVlbCBwb3RlbnRpYWw7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NDktNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmb3h0YWlsIG1pbGxldCBnZW5vbWUgYW5kIGl0cyByZWxhdGlvbnNoaXAgYW1vbmdzdCB0aGUgZ3Jhc3NlczsgY2l0YXRpb25fdGV4dD1XMzI4IChhYnN0cmFjdCk7IGNpdGF0aW9uX3B1Ymxpc2hlcj1TYW4gRGllZ28sIENBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2Ygc2VxdWVuY2VzIHdpdGggWi1ETkEgZm9ybWluZyBwb3RlbnRpYWwgaW4gaGlnaGVyIHBsYW50czsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlcyBhcmUgdWJpcXVpdG91cyByZXBldGl0aXZlIGNvbXBvbmVudHMgb2YgZXVrYXJ5b3RpYyBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MTI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHBsYW50czogZmVhdHVyZXMgYW5kIGFwcGxpY2F0aW9uczsgVHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaWdlbmUgZGVyaXZlZCBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZvciB0aGUgY2VyZWFsIGdlbm9tZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9ODA4LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZvcm1hdGl2ZSBnZW5vbWljIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIGVmZmljaWVudCBnZW5vdHlwaW5nIGFwcGxpY2F0aW9ucyBpbiBzdWdhcmNhbmU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MzI3LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgaW4gQnJhc3NpY2EgdW5pZ2VuZXM6IHJlbGF0aXZlIGFidW5kYW5jZSwgbWFya2VyIGRlc2lnbiBhbmQgdXNlIGluIGNvbXBhcmF0aXZlIHBoeXNpY2FsIG1hcHBpbmcgYW5kIGdlbm9tZSBhbmFseXNpczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9NTUtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXRzIChTU1IpIG1hcmtlcnMgaW4gU2V0YXJpYSBpdGFsaWNhIChQb2FjZWFlKSBhbmQgY3Jvc3MtYW1wbGlmaWNhdGlvbiBpbiByZWxhdGVkIHNwZWNpZXM7IEludC4gSi4gTW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NzgzNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgYmFzZWQgbm92ZWwgZ2Vub21pYyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZvciByb2J1c3QgZ2Vub3R5cGluZyBwdXJwb3NlcyBpbiBmb3h0YWlsIG1pbGxldCBbU2V0YXJpYSBpdGFsaWNhIChMLikgUC4gQmVhdXZdOyBQbGFudCBDZWxsIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjMtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIEVTVC1TU1IgaW4gZm94dGFpbCBtaWxsZXQgKFNldGFyaWEgaXRhbGljYSk7IEouIEdlbmV0LiBSZXMuIENyb3AgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IHZhcmlhYmxlIFNTUiBtYXJrZXJzIHN1aXRhYmxlIGZvciByaWNlIGdlbm90eXBpbmcgdXNpbmcgYWdhcm9zZSBnZWxzOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM1OS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGFwcGxpY2F0aW9uIG9mIGEgd2hvbGUtZ2Vub21lIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgcGFuZWwgZm9yIGhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIGluIHNveWJlYW47IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTA3LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkaXN0cmlidXRpb24gYW5kIG9yZ2FuaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcGxhbnRzOiBhbiBpbnNpZ2h0IGludG8gbWFya2VyIGRldmVsb3BtZW50IGluIEJyYWNoeXBvZGl1bTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMTI5OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3V2ZWxsZSByZWNoZXJjaGVzIHN1ciBsYSBkaXN0cmlidXRpb24gZmxvcmFsZTsgQnVsbCBTb2MuIFZhdWQuIFNjaS4gTmF0LjsgIGNpdGF0aW9uX3llYXI9MTkwODsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTIyMy03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUm9obGYgRi5KLiAxOTk3IFNldGF1a2V0LCBOWSBOVFNZUy1wYyBudW1lcmljYWwgdGF4b25vbXkgYW5kIG11bHRpdmFyaWFuY2UgYW5hbHlzaXMgc3lzdGVtIFZlcnNpb24gMi4wMmUsIEV4ZXRlciBTb2Z0d2FyZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZXMgbWFya2VyczogYW4gb3ZlcnZpZXcgb2YgdGhlIHJlY2VudCBwcm9ncmVzcyBpbiBQbGFudHM7IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzc7ICBjaXRhdGlvbl9wYWdlcz0zMDktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBDUi1hbXBsaWZpZWQgbWljcm9zYXRlbGxpdGVzIGFzIG1hcmtlcnMgaW4gcGxhbnQgZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmluZyBtaWNyb3NhdGVsbGl0ZXMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBUcmVuZHMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00OTAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gdGhlIHRyaWJlIGJhbWJ1c2VhZTsgR2VuZSBDb25zZXJ2LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTUxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIHN1cnZleSBvZiBzd2l0Y2hncmFzcyBnZW5vbWUgc3RydWN0dXJlIGFuZCBvcmdhbml6YXRpb247IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzM4OTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuZCBleHBlcmltZW50YWwgYW5hbHlzaXMgb2YgbWljcm9zYXRlbGxpdGVzIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik6IGZyZXF1ZW5jeSwgbGVuZ3RodmFyaWF0aW9uLCB0cmFuc3Bvc29uIGFzc29jaWF0aW9ucyBhbmQgZ2VuZXRpYyBtYXJrZXIgcG90ZW50aWFsOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE0NDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2VzbWVudCBvZiBnZW5ldGljIHJlbGF0aW9uc2hpcCBvZiBmb3h0YWlsIG1pbGxldCB3aXRoIGl0cyB3aWxkIGFuY2VzdG9yIGFuZCBjbG9zZSByZWxhdGl2ZXMgYnkgSVNTUiBtYXJrZXJzOyBKLiBJbnQuIEFncmkuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTU2LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHN0cnVjdHVyZSBvZiBsYW5kcmFjZXMgaW4gZm94dGFpbCBtaWxsZXQgKFNldGFyaWEgaXRhbGljYSAoTC4pIFAuIEJlYXV2LikgcmV2ZWFsZWQgd2l0aCB0cmFuc3Bvc29uIGRpc3BsYXkgYW5kIGludGVycHJldGF0aW9uIHRvIGNyb3AgZXZvbHV0aW9uIG9mIGZveHRhaWwgbWlsbGV0OyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKFNTUikgbWFya2VycyBmb3IgcGVyZW5uaWFsIHJ5ZWdyYXNzIChMb2xpdW0gcGVyZW5uZS4gTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTQwNS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXJhYmlsaXR5IG9mIHJpY2UgU1NSIG1hcmtlcnMgdG8gYmFtYm9vOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MjMtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIHNldCBvZiBncm91bmRudXQgU1NScyBmb3IgZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIGludGVyLXNwZWNpZmljIHRyYW5zZmVyYWJpbGl0eTsgSW50LiBKLiBJbnRlZ3IuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2VuZXRpYyBtYXBzIG9mIGZveHRhaWwgbWlsbGV0IChTZXRhcmlhIGl0YWxpY2EpIGFuZCByaWNlIChPcnl6YSBzYXRpdmEpOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBkYXRpbmcgdGhlIGNyb3AgY2lyY2xlOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNlcyByZXZlYWwgaGlnaCBsZXZlbHMgb2YgY29uc2VydmVkIGNvbGluZWFyaXR5IGJldHdlZW4gdGhlIGZpbmdlciBtaWxsZXQgYW5kIHJpY2UgZ2Vub21lczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz00ODktOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC8yLzE5Ny85NTU5MzkvZHN0MDAyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgYXZhaWxhYmlsaXR5IG9mIHdlbGwtdmFsaWRhdGVkIGluZm9ybWF0aXZlIGNvLWRvbWluYW50IG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgYW5kIHNhdHVyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwIGhhcyBiZWVuIGxpbWl0ZWQgaW4gZm94dGFpbCBtaWxsZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzM0MDcyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZIY1CmnEjkbDgBYaQpHj8CjLJtzj" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f197%2f334072" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f197%2f334072" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12698" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&amp;Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/2">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4631774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631774">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631776">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631776">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631781">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631781">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631793">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631793">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631819">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631819">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631821">Conflict of Interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631823">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631823">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631825">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631825">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631827">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631827">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4631829">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4631829">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/2/185/333880">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [<em>Setaria italica</em> (L.)]
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Garima Pandey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Garima+Pandey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pandey G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pandey G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Garima Pandey</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gopal Misra</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gopal+Misra">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Misra G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Misra G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gopal Misra</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kajal Kumari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kajal+Kumari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumari K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumari K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kajal Kumari</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sarika Gupta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sarika+Gupta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gupta S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gupta S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sarika Gupta</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup Kumar Parida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Swarup+Kumar+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parida S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Swarup Kumar Parida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Debasis Chattopadhyay</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Debasis+Chattopadhyay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chattopadhyay D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chattopadhyay D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Debasis Chattopadhyay</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manoj Prasad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +0091 11-2673-5160. Fax. +0091 11-2674-1658. Email: <a href="mailto:manoj_prasad@nipgr.ac.in" target="_blank">manoj_prasad@nipgr.ac.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manoj+Prasad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Prasad M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Prasad M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Manoj Prasad</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Authors contributed equally.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 197–207, <a href='https://doi.org/10.1093/dnares/dst002'>https://doi.org/10.1093/dnares/dst002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 February 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 November 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 January 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="334072" href="/dnaresearch/article-pdf/20/2/197/955939/dst002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Garima Pandey, Gopal Misra, Kajal Kumari, Sarika Gupta, Swarup Kumar Parida, Debasis Chattopadhyay, Manoj Prasad;  Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [<em>Setaria italica</em> (L.)], <em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 197–207, <a href='https://doi.org/10.1093/dnares/dst002'>https://doi.org/10.1093/dnares/dst002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/334072?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/334072?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Development%20and%20Use%20of%20Microsatellite%20Markers%20for%20Large-Scale%20Genotyping%20Applications%20in%20Foxtail%20Millet%20%5B%26lt%3Bem%26gt%3BSetaria%20italica%26lt%3B%2Fem%26gt%3B%20(L.)%5D&amp;publicationDate=2013-02-03&amp;volumeNum=20&amp;issueNum=2&amp;author=Pandey%2C%20Garima%3B%20Misra%2C%20Gopal&amp;startPage=197&amp;endPage=207&amp;contentId=10.1093%2Fdnares%2Fdst002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f197%2f334072" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4631774 id="4631774" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The availability of well-validated informative co-dominant microsatellite markers and saturated genetic linkage map has been limited in foxtail millet (<em>Setaria italica</em> L.). In view of this, we conducted a genome-wide analysis and identified 28 342 microsatellite repeat-motifs spanning 405.3 Mb of foxtail millet genome. The trinucleotide repeats (∼48%) was prevalent when compared with dinucleotide repeats (∼46%). Of the 28 342 microsatellites, 21 294 (∼75%) primer pairs were successfully designed, and a total of 15 573 markers were physically mapped on 9 chromosomes of foxtail millet. About 159 markers were validated successfully in 8 accessions of <em>Setaria</em> sp. with ∼67% polymorphic potential. The high percentage (89.3%) of cross-genera transferability across millet and non-millet species with higher transferability percentage in bioenergy grasses (∼79%, Switchgrass and ∼93%, Pearl millet) signifies their importance in studying the bioenergy grasses. <em>In silico</em> comparative mapping of 15 573 foxtail millet microsatellite markers against the mapping data of sorghum (16.9%), maize (14.5%) and rice (6.4%) indicated syntenic relationships among the chromosomes of foxtail millet and target species. The results, thus, demonstrate the immense applicability of developed microsatellite markers in germplasm characterization, phylogenetics, construction of genetic linkage map for gene/quantitative trait loci discovery, comparative mapping in foxtail millet, including other millets and bioenergy grass species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;foxtail millet (Setaria italica L.)&quot;">foxtail millet (<em>Setaria italica</em> L.)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microsatellite">microsatellite</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative mapping&quot;">comparative mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;physical mapping&quot;">physical mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transferability">transferability</a></div>                            <h2 scrollto-destination=4631776 id="4631776" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Foxtail millet [(<em>Setaria italica</em> (L.) P. Beauv.; 2<em>n</em> = 18)], a member of the Poaceae family, is the second most-widely planted species of millets, next to pearl millet (<em>Pennisetum glaucum</em>) (FAOSTAT 2005; <a class="link link-uri" href="http://faostat.fao/org/">http://faostat.fao/org/</a>). It is mainly cultivated in semi-arid regions of India, China and other parts of Asia, North Africa and America. It is a potential model system for biofuel crops due to its small genome size (∼515 Mb), less repetitive DNA and self-pollinated nature<sup><a reveal-id="DST002C1  DST002C3" data-open="DST002C1  DST002C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> and the most important are its close phylogenetic relationships with several biofuel crops such as switchgrass (<em>Panicum virgatum</em>), napier grass (<em>Pennisetum purpureum</em>) and pearl millet (<em>P. glaucum</em>). Furthermore, foxtail millet is an excellent genetic system to study evolution and physiology of C4 photosynthesis and abiotic stress tolerance mechanism particularly for salinity and dehydration stresses.<sup><a reveal-id="DST002C2  DST002C4" data-open="DST002C2  DST002C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Recently, the Joint Genome Institute of the Department of Energy, USA and Beijing Genomics Institute, China have sequenced the genome of two foxtail millet accessions.<sup><a reveal-id="DST002C5" data-open="DST002C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST002C6" data-open="DST002C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The sequence-based comparative genome mapping has revealed a high degree of syntenic relationship of foxtail millet with rice and sorghum, in spite of their divergence more than 50 million years ago.<sup><a reveal-id="DST002C7" data-open="DST002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> With the availability of genome-wide sequence resources in foxtail millet, it is now required to develop and validate useful sequence-based genetic markers and genes in a larger genotype sets and utilize them in genetic improvement of target millet and its related crop species.</p><p>Among the available different genetic marker system, the microsatellite markers are considered important in various genetic studies because of their high reproducibility, co-dominant inheritance, multi-allelic variation and abundance in the genome.<sup><a reveal-id="DST002C8" data-open="DST002C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST002C9" data-open="DST002C9" class="link link-ref link-reveal xref-bibr">9</a></sup> These desirable characteristics of microsatellite markers had made them advantageous in various applications of structural, functional and comparative genomics, including variety identification, studying genetic diversity and phylogenetic relationships, construction of high-density genome maps, mapping of useful genes, comparative genome mapping and marker-assisted selection.<sup><a reveal-id="DST002C10  DST002C15" data-open="DST002C10  DST002C15" class="link link-ref link-reveal xref-bibr">10–15</a></sup> Till now, a total of 218 genic and genomic microsatellite markers have been developed from Expressed Sequence Tag sequences (26)<sup><a reveal-id="DST002C16" data-open="DST002C16" class="link link-ref link-reveal xref-bibr">16</a></sup>, random amplified polymorphic DNA (RAPD)-enriched genomic sequences from GenBank (45)<sup><a reveal-id="DST002C14" data-open="DST002C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and microsatellite-enriched genomic libraries (147)<sup><a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> in foxtail millet. The availability of limited EST sequence resources and involvement of huge cost, time and labor in mining microsatellite markers from enriched genomic libraries have impeded the development of large-scale microsatellite markers in foxtail millet. However, foxtail millet with large genome size would require a large number of microsatellite markers for construction of high-density genetic linkage map and identification and mapping of genes/quantitative trait loci (QTLs) for useful agronomic traits.</p><p>A large number of microsatellite markers have been developed by utilizing the pseudomolecule genomic sequences from the completely sequenced plant genomes, including rice (IRGSP 2005, <a class="link link-uri" href="http://rgp.dna.affrc.go.jp/IRGSP/">http://rgp.dna.affrc.go.jp/IRGSP/</a>), <em>Populus</em> (genome.jgi.doe.gov/poplar/), sorghum (genome.jgi-psf.org/Sorbi1/Sorbi1.home.htm), maize (mips.helmholtz-muenchen.de/plant/maize/) and <em>Arabidopsis thaliana</em> (<em>Arabidopsis</em> Genome Initiative 2001, <a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>), and these markers were further utilized in genome mapping and genotyping applications in target crop species. A similar approach has been implemented in rice, sorghum, <em>Brachypodium</em>, <em>Brassica</em> and <em>Arabidopsis</em> for the development of thousands of microsatellite markers from whole genome sequences.<sup><a reveal-id="DST002C12" data-open="DST002C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST002C13" data-open="DST002C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST002C17  DST002C19" data-open="DST002C17  DST002C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> Unfortunately, no such marker resource is currently available for recently sequenced foxtail millet genome. With the availability of foxtail millet whole genome sequence in public domain, it is now possible to mine and develop large-scale <em>in silico</em> microsatellite markers that could be utilized for various applications of structural and comparative genomics in foxtail millet. The development and large-scale validation of such genomic microsatellite markers in a genome-wide scale could be useful for not only genetic improvement of foxtail millet, but also for other under-utilized/orphan crop species for which very little or no genomic information is available.</p><p>Considering the utility of whole genome sequence-based microsatellite markers, we made an attempt to develop and validate microsatellite markers from foxtail millet genome sequences and determine their genomic distribution on foxtail millet genome. Furthermore, a selected set of validated and physically mapped markers were utilized in evaluating polymorphic and diversity potential in foxtail millet accessions, studying cross-transferability across millet and non-millet species and <em>in silico</em> comparative genome mapping among foxtail millet and three members of grass family.</p>                            <h2 scrollto-destination=4631781 id="4631781" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4631782 id="4631782" class="section-title" data-legacy-id=s2a>2.1. Identification of microsatellites and primer design</h3>
<p>The whole genome sequences of <em>S. italica</em> were retrieved from Phytozome (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>) and searched for microsatellites using the default parameter of MIcroSAtellite (MISA) identification tool (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa">http://pgrc.ipk-gatersleben.de/misa</a>). The search criteria were: six repeat units for dinucleotide repeats (DNRs), five repeat units for trinucleotide repeats (TNRs), tetranucleotide repeats (TeNRs), pentanucleotide repeats (PNRs) and hexanucleotide repeats (HNRs). The forward and reverse primers flanking the identified microsatellite repeat motifs were designed in batches using the default parameters of integrated Perl 5 interface module of MISA-Primer3 software (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/primer3.html">http://pgrc.ipk-gatersleben.de/misa/primer3.html</a>).</p>                            <h3 scrollto-destination=4631784 id="4631784" class="section-title" data-legacy-id=s2b>2.2. Physical mapping of foxtail millet genomic microsatellite markers</h3>
<p>For determining the specific physical location (bp) of designed microsatellite markers on the nine foxtail millet chromosomes, the flanking genomic sequences of identified microsatellite repeat motifs were BLAST searched against the whole genome sequence of foxtail millet at Phytozome (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>). Optimized BLASTN search parameters with E-value = 0 and low-complexity filter options were used for this analysis. The microsatellite markers were plotted separately for each of the nine foxtail millet chromosomes according to their ascending order of physical position (bp), from the short arm telomere to the long arm telomere and finally visualized in MapChart software.<sup><a reveal-id="DST002C20" data-open="DST002C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p>                            <h3 scrollto-destination=4631786 id="4631786" class="section-title" data-legacy-id=s2c>2.3. Validation of genomic microsatellite markers</h3>
<p>For validation of selected set of physically mapped microsatellite markers distributed over nine chromosomes of foxtail millet, the genomic DNA was isolated from the leaves of eight foxtail millet accessions (Table <a reveal-id="DST002TB1" data-open="DST002TB1" class="link link-reveal link-table xref-fig">1</a>) using CTAB method. The isolated genomic DNA was resolved on 0.8% agarose gel (Cambrex, USA) in Tris-borate EDTA (TBE) buffer (pH 8.0), quantified using different concentration of λ DNA standard and analysed in the GelDoc-It imaging system (UVP, Cambridge, UK). The polymerase chain reaction (PCR) was carried out in MyCycler thermal controller (Bio-Rad) in a 25-µl total volume containing one unit of Taq DNA polymerase (Bangalore Genei), 50 ng of genomic DNA, 10 µmol/l of each primer, 0.5 mmol/l of each dNTPs and 2.5 µl of 10× PCR buffer (Bangalore Genei). DNA amplification was done following the PCR program; a preliminary denaturation of 3 min at 94°C, followed by 40 cycles of 1 min at 94°C, 1.5 min at 48–55°C and 2 min at 72°C and a final extension for 10 min at 72°C. The amplicons were resolved on 2% agarose gel (Cambrex, USA) in TBE buffer (pH 8.0), and analysed in the GelDoc-It imaging system. Standard size marker of 100 bp (NEB) was used to determine the fragment sizes for each markers. One of the markers showing amplification in all the accessions used in the study was cloned and sequenced. For cloning, the amplicons amplified by microsatellite marker were eluted using <em>AccuPrep</em> DNA Gel Purification Kit (Bioneer, Korea) and cloned into pGEM-T Easy vector (Promega, USA) following the manufacturer's instructions. The plasmids were transformed into competent cells of <em>Escherichia coli</em> DH5α and grown on LB media supplemented with ampicillin (100 μg/ml), X-Gal (20 μg/ml) and IPTG (0.1 M). The transformants were selected based on blue–white screening, and recombinant plasmids were isolated using <em>AccuPrep</em> Plasmid MiniPrep DNA Extraction Kit following the manufacturer's protocol. The plasmids were sequenced in automated sequencer (3730xI DNA Analyzer, Applied Biosystem) using M13 forward and reverse primers. Multiple sequence alignment was performed with the obtained sequences along with reference <em>S. italica</em> sequence using ClustalW2 program (<a class="link link-uri" href="http://www.ebi.ac.uk/Tools/clustalw2/index.html">http://www.ebi.ac.uk/Tools/clustalw2/index.html</a>). </p>                        <a id="4631788" scrollto-destination="4631788"></a>
<div content-id="DST002TB1" class="table-modal"><div class="table-wrap"><div id="DST002TB1" class="table-wrap-title" data-id="DST002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Details of plant materials used in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Common name </th><th>Genus/species </th><th>Accessions </th></tr></thead><tbody><tr><td>1. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Prasad </td></tr><tr><td>2. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Lepakshi </td></tr><tr><td>3. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC403579 </td></tr><tr><td>4. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC480117 </td></tr><tr><td>5. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>GS2038 </td></tr><tr><td>6. </td><td>Green millet </td><td><em>S. italica</em> subsp.viridis </td><td>EC539251 </td></tr><tr><td>7. </td><td>African bristlegrass </td><td><em>S. sphacelata</em> </td><td>EC539291 </td></tr><tr><td>8. </td><td>Bristly foxtail </td><td><em>S. verticillata</em> </td><td>EC539293 </td></tr><tr><td>9. </td><td>Barnyard millet </td><td><em>Echinochloa frumentacea</em> </td><td>CO-LV2 </td></tr><tr><td>10. </td><td>Finger millet </td><td><em>Eleusine coracana</em> </td><td>CO-RA114 </td></tr><tr><td>11. </td><td>Kodo millet </td><td><em>Paspalum scrobiculatum</em> </td><td>C0-3 </td></tr><tr><td>12. </td><td>Little millet </td><td><em>Panicum sumatrense</em> </td><td>C0-4 </td></tr><tr><td>13. </td><td>Pearl millet </td><td><em>Pennisetum glaucum</em> </td><td>CO-CU 9 </td></tr><tr><td>14. </td><td>Proso millet </td><td><em>Panicum miliaceum</em> </td><td>CO-5 </td></tr><tr><td>15. </td><td>Switchgrass </td><td><em>P. virgatum</em> </td><td>PI421521 </td></tr><tr><td>16. </td><td>Guinea grass </td><td><em>Panicum maximum</em> </td><td>SPM92 </td></tr><tr><td>17. </td><td>Sorghum </td><td><em>Sorghum bicolor</em> </td><td>CO30 </td></tr><tr><td>18. </td><td>Wheat </td><td><em>Triticum aestivum</em> </td><td>PH132 </td></tr><tr><td>19. </td><td>Rice </td><td><em>Oryza sativa</em> </td><td>cv. Pusa Basmati </td></tr><tr><td>20. </td><td>Maize </td><td><em>Zea mays</em> </td><td>B73 </td></tr></tbody></table></div><div content-id="DST002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Common name </th><th>Genus/species </th><th>Accessions </th></tr></thead><tbody><tr><td>1. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Prasad </td></tr><tr><td>2. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Lepakshi </td></tr><tr><td>3. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC403579 </td></tr><tr><td>4. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC480117 </td></tr><tr><td>5. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>GS2038 </td></tr><tr><td>6. </td><td>Green millet </td><td><em>S. italica</em> subsp.viridis </td><td>EC539251 </td></tr><tr><td>7. </td><td>African bristlegrass </td><td><em>S. sphacelata</em> </td><td>EC539291 </td></tr><tr><td>8. </td><td>Bristly foxtail </td><td><em>S. verticillata</em> </td><td>EC539293 </td></tr><tr><td>9. </td><td>Barnyard millet </td><td><em>Echinochloa frumentacea</em> </td><td>CO-LV2 </td></tr><tr><td>10. </td><td>Finger millet </td><td><em>Eleusine coracana</em> </td><td>CO-RA114 </td></tr><tr><td>11. </td><td>Kodo millet </td><td><em>Paspalum scrobiculatum</em> </td><td>C0-3 </td></tr><tr><td>12. </td><td>Little millet </td><td><em>Panicum sumatrense</em> </td><td>C0-4 </td></tr><tr><td>13. </td><td>Pearl millet </td><td><em>Pennisetum glaucum</em> </td><td>CO-CU 9 </td></tr><tr><td>14. </td><td>Proso millet </td><td><em>Panicum miliaceum</em> </td><td>CO-5 </td></tr><tr><td>15. </td><td>Switchgrass </td><td><em>P. virgatum</em> </td><td>PI421521 </td></tr><tr><td>16. </td><td>Guinea grass </td><td><em>Panicum maximum</em> </td><td>SPM92 </td></tr><tr><td>17. </td><td>Sorghum </td><td><em>Sorghum bicolor</em> </td><td>CO30 </td></tr><tr><td>18. </td><td>Wheat </td><td><em>Triticum aestivum</em> </td><td>PH132 </td></tr><tr><td>19. </td><td>Rice </td><td><em>Oryza sativa</em> </td><td>cv. Pusa Basmati </td></tr><tr><td>20. </td><td>Maize </td><td><em>Zea mays</em> </td><td>B73 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631788" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST002TB1" class="table-wrap-title" data-id="DST002TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Details of plant materials used in the present study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Common name </th><th>Genus/species </th><th>Accessions </th></tr></thead><tbody><tr><td>1. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Prasad </td></tr><tr><td>2. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Lepakshi </td></tr><tr><td>3. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC403579 </td></tr><tr><td>4. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC480117 </td></tr><tr><td>5. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>GS2038 </td></tr><tr><td>6. </td><td>Green millet </td><td><em>S. italica</em> subsp.viridis </td><td>EC539251 </td></tr><tr><td>7. </td><td>African bristlegrass </td><td><em>S. sphacelata</em> </td><td>EC539291 </td></tr><tr><td>8. </td><td>Bristly foxtail </td><td><em>S. verticillata</em> </td><td>EC539293 </td></tr><tr><td>9. </td><td>Barnyard millet </td><td><em>Echinochloa frumentacea</em> </td><td>CO-LV2 </td></tr><tr><td>10. </td><td>Finger millet </td><td><em>Eleusine coracana</em> </td><td>CO-RA114 </td></tr><tr><td>11. </td><td>Kodo millet </td><td><em>Paspalum scrobiculatum</em> </td><td>C0-3 </td></tr><tr><td>12. </td><td>Little millet </td><td><em>Panicum sumatrense</em> </td><td>C0-4 </td></tr><tr><td>13. </td><td>Pearl millet </td><td><em>Pennisetum glaucum</em> </td><td>CO-CU 9 </td></tr><tr><td>14. </td><td>Proso millet </td><td><em>Panicum miliaceum</em> </td><td>CO-5 </td></tr><tr><td>15. </td><td>Switchgrass </td><td><em>P. virgatum</em> </td><td>PI421521 </td></tr><tr><td>16. </td><td>Guinea grass </td><td><em>Panicum maximum</em> </td><td>SPM92 </td></tr><tr><td>17. </td><td>Sorghum </td><td><em>Sorghum bicolor</em> </td><td>CO30 </td></tr><tr><td>18. </td><td>Wheat </td><td><em>Triticum aestivum</em> </td><td>PH132 </td></tr><tr><td>19. </td><td>Rice </td><td><em>Oryza sativa</em> </td><td>cv. Pusa Basmati </td></tr><tr><td>20. </td><td>Maize </td><td><em>Zea mays</em> </td><td>B73 </td></tr></tbody></table></div><div content-id="DST002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Common name </th><th>Genus/species </th><th>Accessions </th></tr></thead><tbody><tr><td>1. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Prasad </td></tr><tr><td>2. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>cv. Lepakshi </td></tr><tr><td>3. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC403579 </td></tr><tr><td>4. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>IC480117 </td></tr><tr><td>5. </td><td>Foxtail millet </td><td><em>S. italica</em> </td><td>GS2038 </td></tr><tr><td>6. </td><td>Green millet </td><td><em>S. italica</em> subsp.viridis </td><td>EC539251 </td></tr><tr><td>7. </td><td>African bristlegrass </td><td><em>S. sphacelata</em> </td><td>EC539291 </td></tr><tr><td>8. </td><td>Bristly foxtail </td><td><em>S. verticillata</em> </td><td>EC539293 </td></tr><tr><td>9. </td><td>Barnyard millet </td><td><em>Echinochloa frumentacea</em> </td><td>CO-LV2 </td></tr><tr><td>10. </td><td>Finger millet </td><td><em>Eleusine coracana</em> </td><td>CO-RA114 </td></tr><tr><td>11. </td><td>Kodo millet </td><td><em>Paspalum scrobiculatum</em> </td><td>C0-3 </td></tr><tr><td>12. </td><td>Little millet </td><td><em>Panicum sumatrense</em> </td><td>C0-4 </td></tr><tr><td>13. </td><td>Pearl millet </td><td><em>Pennisetum glaucum</em> </td><td>CO-CU 9 </td></tr><tr><td>14. </td><td>Proso millet </td><td><em>Panicum miliaceum</em> </td><td>CO-5 </td></tr><tr><td>15. </td><td>Switchgrass </td><td><em>P. virgatum</em> </td><td>PI421521 </td></tr><tr><td>16. </td><td>Guinea grass </td><td><em>Panicum maximum</em> </td><td>SPM92 </td></tr><tr><td>17. </td><td>Sorghum </td><td><em>Sorghum bicolor</em> </td><td>CO30 </td></tr><tr><td>18. </td><td>Wheat </td><td><em>Triticum aestivum</em> </td><td>PH132 </td></tr><tr><td>19. </td><td>Rice </td><td><em>Oryza sativa</em> </td><td>cv. Pusa Basmati </td></tr><tr><td>20. </td><td>Maize </td><td><em>Zea mays</em> </td><td>B73 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631788" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4631789 id="4631789" class="section-title" data-legacy-id=s2d>2.4. Assessment of genetic diversity pattern and phylogenetic relationships</h3>
<p>The microsatellite alleles amplified in millets and non-millets were scored in a binary format to infer the molecular diversity and genetic relationship among them. The genetic similarity coefficient was calculated according to Jaccard's coefficient<sup><a reveal-id="DST002C21" data-open="DST002C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using the NTSYS-pc software package (version 2.02e, Exeter Software, Setauket, NY, USA).<sup><a reveal-id="DST002C22" data-open="DST002C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The pairwise similarity matrix of the Jaccard coefficient was used to construct a phylogenetic tree using the neighbor-joining module of NTSYS-pc software.</p>                            <h3 scrollto-destination=4631791 id="4631791" class="section-title" data-legacy-id=s2e>2.5. Comparative genome mapping between foxtail and other grass species</h3>
<p>The flanking sequences of microsatellite marker loci that were physically mapped on the nine chromosomes of foxtail millet were BLAST searched against genome sequences of sorghum, maize and rice (<a class="link link-uri" href="http://gramene.org/">http://gramene.org/</a>; <a class="link link-uri" href="http://www.phytozome.net">www.phytozome.net</a>) to derive marker-based syntenic relationships among the chromosomes of foxtail millet and three other grass species. A cutoff bit score of 54.7 and E-value of &lt;1e−05 were considered significant for BLAST analysis. The marker-based syntenic relationships among foxtail millet, rice, sorghum and maize were finally visualized with visualization blocks using Circos 0.55 (<a class="link link-uri" href="http://circos.ca">http://circos.ca</a>).</p>                            <h2 scrollto-destination=4631793 id="4631793" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4631794 id="4631794" class="section-title" data-legacy-id=s3a>3.1. Frequency, distribution and characterization of microsatellites in foxtail millet genome</h3>
<p>The 405.73 Mb available genome sequences of foxtail millet were searched for microsatellites, and a total of 28 342 microsatellites comprising different kinds of desirable repeat-motifs (from DNRs to HNRs) were identified with average frequency of about 69 microsatellites per megabase of genomic sequences. The frequency and distribution of different microsatellite repeat-motif types in the genomic sequences of foxtail millet are shown in Fig. <a reveal-id="DST002F1" data-open="DST002F1" class="link link-reveal link-table xref-fig">1</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The frequency of microsatellites (69/Mb) estimated in the foxtail millet genomic sequences at genome-wide level is comparable to that documented in <em>Brachypodium</em> (78.5/Mb) genome, but lower than that reported earlier in other related monocot plant genomes such as rice (189.4/Mb) and sorghum (99.8/Mb) and dicot <em>Arabidopsis</em> (127.5/Mb).<sup><a reveal-id="DST002C19" data-open="DST002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> These differences could be due to the variation in search criteria, size of the database and bioinformatics software tools used in different studies for identification of microsatellites.<sup><a reveal-id="DST002C10" data-open="DST002C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST002C11" data-open="DST002C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST002C23" data-open="DST002C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                        <a id="4631796" scrollto-destination="4631796"></a>
<div id="DST002F1" data-id="DST002F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00201.jpeg?Expires=2147483647&Signature=Mea2QJxMyi5YzpxggZ3ExqPH9fxltwhed-G7QuCDrH57J~CXt9upoQkawQbKZDxFDUQdpQiAouCbuzl42hv7m7FT7lyMPsDKzh22GttoZ5qCgqSUTZ2L0Ey3qp3cheq2qZYcNkArX5PTvx4ocOhQ3NG-bTcWaO3s0Yz8Sb1OUOMINMOKJHAOFIOp1yn2FJX8eQMGoOJGIuNMzh3H4sam58Jt7NyWavEGjTQ2lxVM0T0M0SEzqZFr-CFrPOYavt70RNqoRUtbJ~~JSdknnlh~0cZhkSnGgiP6XNXAex49bxSgnsL~ZQe1lL3DMOj846E8aT5cIt1i0V2t6oeY5CIo4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of simple sequence repeats from foxtail millet genome. (A) Relative frequency, proportion (%) and number of selected microsatellite repeat-motif types. (B) Different classes of selected microsatellite repeats." path-from-xml="dst00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631796/dst00201.jpeg" path-from-xml="dst00201.jpeg" target="_blank">View large</a><a section="4631796" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00201.jpeg?Expires=2147483647&Signature=yL1Ml9QkZjkKZ-JWsdtt2Algq3JBGGWqCdx9Gz307i6SEeEbuXMBDF9v6Hlz3x17rQvmCRtgftCkr~b~5xIa~uzTH0yqyPCLgJckhreWqWA5LG3QdkbTmaWtRv0lAeCWSXGjYPrkLQbqL~F-zueSQNxU~4kOJEMdqpOh6NRxx9BLnSUEajF6bB2xdvAjbt1qR6NRF-5MyLL8CHim3WAOGfUWxBD-DEGnPGlfl9zK8YsceDjt7NH9vhHVE2Y4DDLsKeTk6CNjK8lVlHDi~bOrRZxB19ReFlS19zAOTBfGQMSImbdEWcvjM9ZRnEtT64wKY6MrKjGNoOpyZ0TVcynuCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631796&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of simple sequence repeats from foxtail millet genome. (A) Relative frequency, proportion (%) and number of selected microsatellite repeat-motif types. (B) Different classes of selected microsatellite repeats.</p></div></div><div content-id="DST002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00201.jpeg?Expires=2147483647&Signature=Mea2QJxMyi5YzpxggZ3ExqPH9fxltwhed-G7QuCDrH57J~CXt9upoQkawQbKZDxFDUQdpQiAouCbuzl42hv7m7FT7lyMPsDKzh22GttoZ5qCgqSUTZ2L0Ey3qp3cheq2qZYcNkArX5PTvx4ocOhQ3NG-bTcWaO3s0Yz8Sb1OUOMINMOKJHAOFIOp1yn2FJX8eQMGoOJGIuNMzh3H4sam58Jt7NyWavEGjTQ2lxVM0T0M0SEzqZFr-CFrPOYavt70RNqoRUtbJ~~JSdknnlh~0cZhkSnGgiP6XNXAex49bxSgnsL~ZQe1lL3DMOj846E8aT5cIt1i0V2t6oeY5CIo4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of simple sequence repeats from foxtail millet genome. (A) Relative frequency, proportion (%) and number of selected microsatellite repeat-motif types. (B) Different classes of selected microsatellite repeats." path-from-xml="dst00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631796/dst00201.jpeg" path-from-xml="dst00201.jpeg" target="_blank">View large</a><a section="4631796" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00201.jpeg?Expires=2147483647&Signature=yL1Ml9QkZjkKZ-JWsdtt2Algq3JBGGWqCdx9Gz307i6SEeEbuXMBDF9v6Hlz3x17rQvmCRtgftCkr~b~5xIa~uzTH0yqyPCLgJckhreWqWA5LG3QdkbTmaWtRv0lAeCWSXGjYPrkLQbqL~F-zueSQNxU~4kOJEMdqpOh6NRxx9BLnSUEajF6bB2xdvAjbt1qR6NRF-5MyLL8CHim3WAOGfUWxBD-DEGnPGlfl9zK8YsceDjt7NH9vhHVE2Y4DDLsKeTk6CNjK8lVlHDi~bOrRZxB19ReFlS19zAOTBfGQMSImbdEWcvjM9ZRnEtT64wKY6MrKjGNoOpyZ0TVcynuCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631796&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of simple sequence repeats from foxtail millet genome. (A) Relative frequency, proportion (%) and number of selected microsatellite repeat-motif types. (B) Different classes of selected microsatellite repeats.</p></div></div><p>Among the 28 342 microsatellites obtained, the TNRs were the most abundant (13 521) with a proportion of 47.7%, followed by DNRs (13072, 46.1%), TeNRs (1354, 4.8%), PNRs (236, 0.8%) and HNRs (159, 0.56%) (Fig. <a reveal-id="DST002F1" data-open="DST002F1" class="link link-reveal link-table xref-fig">1</a>A). The distribution of different microsatellite motifs is not uniform, particularly as in case of TNRs and DNRs. Among DNRs, AG/CT motifs (21.4%) were more frequent, followed by AT/AT (11.9%), AC/GT (9.8%) and GC/CG (2.9%) motifs. Among the TNRs, CCG/CGG (18.3%) motifs were most abundant followed by AGG/CCT (8.1%) and AGC/GCT (5.1%), whereas ACT/AGT (0.7%) motifs were less abundant (Fig. <a reveal-id="DST002F1" data-open="DST002F1" class="link link-reveal link-table xref-fig">1</a>B). Based on length of the repeat motifs, a total of 7387 (26%) microsatellites were classified as long and hypervariable class I (≥20 bp) types and remaining 20 995 (74%) microsatellites as variable class II (12–19 bp) types. Interestingly, as in case of class I microsatellites, the proportion of DNRs (47%) was higher when compared with TNRs (30%) (Fig. <a reveal-id="DST002F2" data-open="DST002F2" class="link link-reveal link-table xref-fig">2</a>). In contrast, in the class II microsatellite types, the proportion of TNRs (54%) was more than that of DNRs (46%). This converse distribution pattern of microsatellite repeat motifs in the foxtail millet genome overall resulted in slightly higher frequency of TNRs when compared with DNRs in this study. Generally, DNRs are reported to be abundant in sequenced plant genomes such as rice, sorghum and <em>Arabidopsis</em>.<sup><a reveal-id="DST002C19" data-open="DST002C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DST002C24" data-open="DST002C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST002C25" data-open="DST002C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Conversely, in our study, we observed an abundance of TNRs (47.7%) in foxtail millet genome that agreed well with the earlier reports in <em>Brachypodium</em>,<sup><a reveal-id="DST002C19" data-open="DST002C19" class="link link-ref link-reveal xref-bibr">19</a></sup> bamboo<sup><a reveal-id="DST002C26" data-open="DST002C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and switchgrass.<sup><a reveal-id="DST002C27" data-open="DST002C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The earlier reports of higher frequency of TNRs in the transcribed genic sequence components of the plant genomes suggested that the genomic sequences used in this study for mining microsatellites possibly representing more of expressed sequence components of the foxtail millet genome.<sup><a reveal-id="DST002C10" data-open="DST002C10" class="link link-ref link-reveal xref-bibr">10</a>–<a reveal-id="DST002C13" data-open="DST002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Furthermore, the prevalence of AG/CT (21.4%) dinucleotide and CCG/CGG (18.3%), AGG/CCT (8.1%) and AGC/GCT (5.1%) trinucleotide repeat motifs in the foxtail millet genome is comparable to earlier similar genome-wide microsatellite identification studies in monocot and dicot plant species such as <em>Brachypodium</em>, rice, sorghum and bamboo.<sup><a reveal-id="DST002C19" data-open="DST002C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DST002C26" data-open="DST002C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                        <a id="4631798" scrollto-destination="4631798"></a>
<div id="DST002F2" data-id="DST002F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00202.jpeg?Expires=2147483647&Signature=Z4toEdbQSORcOxkdQ3n0GSzp8dXclhZDAIl3jXFCiQmEy6d1jwptXJDmryfMIsxOB8hbHE95fyLfE0sr1wsNnXHpzg-8j~ThETMBavINbL32svpHf5OiFb1Ldr7MyMk3zJQA4br7qAcqaonDw9VGUYvudW1fh6sgCLVGBWWk792IVmtzCM2LfMa3aCxwHvUCz1wEIKM8FsixUN5HAojD~4frN4PfFLVvAybBF0-bUpuM-NGBGV1MJ~U-D5IJ5VSyFH0ECHuBjRthn0wW1CkoGEwDfiEBupuL9xP58ajqa1LJUeRO6cZ4gvkiX9K3zWn2Rf8zPsFelpl-ewWyhDKHfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency and relative distribution of long and hypervariable class I and variable class II microsatellite repeats in the foxtail millet genome." path-from-xml="dst00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631798/dst00202.jpeg" path-from-xml="dst00202.jpeg" target="_blank">View large</a><a section="4631798" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00202.jpeg?Expires=2147483647&Signature=RZw-nvbAZz6fsa3sofc28ULFpxOH6XzTdCLFplRkkMHPdbt48wVoIZF6MFvCc~EVmte5rOvm94eUdoTrn1iFVvfOJmd6D5mDsXay2bbyOaC6naiw8wCw46Kag~RB5N7AszT6iXpqMUS0A87e8cuDpYulpmam4MqihZHVAwUP-KYHr3fbX308Rjnrel61PV5R~8nD2XvAUstvOQmW-8qK4ZRT6UaPDZvqPxfVTOES44pOPyeNQ4rTciAAL-vo6R7a1UDJif2QOdbz-IqYdKei3OgM-BRYHE3nU~6c3b3pR~vc88qMsYWwpFv2AXvyRmkSiqwpsKvrnMjPo3Y5u0extQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631798&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency and relative distribution of long and hypervariable class I and variable class II microsatellite repeats in the foxtail millet genome.</p></div></div><div content-id="DST002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00202.jpeg?Expires=2147483647&Signature=Z4toEdbQSORcOxkdQ3n0GSzp8dXclhZDAIl3jXFCiQmEy6d1jwptXJDmryfMIsxOB8hbHE95fyLfE0sr1wsNnXHpzg-8j~ThETMBavINbL32svpHf5OiFb1Ldr7MyMk3zJQA4br7qAcqaonDw9VGUYvudW1fh6sgCLVGBWWk792IVmtzCM2LfMa3aCxwHvUCz1wEIKM8FsixUN5HAojD~4frN4PfFLVvAybBF0-bUpuM-NGBGV1MJ~U-D5IJ5VSyFH0ECHuBjRthn0wW1CkoGEwDfiEBupuL9xP58ajqa1LJUeRO6cZ4gvkiX9K3zWn2Rf8zPsFelpl-ewWyhDKHfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency and relative distribution of long and hypervariable class I and variable class II microsatellite repeats in the foxtail millet genome." path-from-xml="dst00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631798/dst00202.jpeg" path-from-xml="dst00202.jpeg" target="_blank">View large</a><a section="4631798" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00202.jpeg?Expires=2147483647&Signature=RZw-nvbAZz6fsa3sofc28ULFpxOH6XzTdCLFplRkkMHPdbt48wVoIZF6MFvCc~EVmte5rOvm94eUdoTrn1iFVvfOJmd6D5mDsXay2bbyOaC6naiw8wCw46Kag~RB5N7AszT6iXpqMUS0A87e8cuDpYulpmam4MqihZHVAwUP-KYHr3fbX308Rjnrel61PV5R~8nD2XvAUstvOQmW-8qK4ZRT6UaPDZvqPxfVTOES44pOPyeNQ4rTciAAL-vo6R7a1UDJif2QOdbz-IqYdKei3OgM-BRYHE3nU~6c3b3pR~vc88qMsYWwpFv2AXvyRmkSiqwpsKvrnMjPo3Y5u0extQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631798&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency and relative distribution of long and hypervariable class I and variable class II microsatellite repeats in the foxtail millet genome.</p></div></div>                            <h3 scrollto-destination=4631799 id="4631799" class="section-title" data-legacy-id=s3b>3.2. Development of microsatellite markers and their physical mapping in foxtail millet genome</h3>
<p>From 28 342 identified microsatellites, forward and reverse primer pairs could be designed from either side of the flanking genomic sequences of 21 294 microsatellites with successful primer designing potential of 75.1%. The primers could not be designed for remaining 7048 (24.9%) microsatellites due to constraint of obtaining enough flanking sequences from either side of identified microsatellites. Similar observations have also been observed in other genome-wide microsatellite mining studies in monocot and dicot crop plants.<sup><a reveal-id="DST002C12" data-open="DST002C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST002C13" data-open="DST002C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST002C19" data-open="DST002C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>The determination of genomic distribution of 21 294 microsatellite markers on the foxtail millet genome revealed physical localization of 15 573 markers on the 9 chromosomes of foxtail millet with average marker density of 42 markers/Mb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). All the physically mapped 15 573 genomic microsatellite markers were reported in publicly available NCBI Probe Database with Accession numbers from PUID16589829 to PUID 16605401. The average marker density was maximum 46.2/Mb in chromosome 9, followed by 42.8/Mb in chromosome 5 and 41.6/Mb in chromosome 1 and minimum 30/Mb in chromosome 8. The comprehensive analysis of chromosome-wise distribution and frequency of these physically mapped microsatellite markers showed higher frequency of physically mapped markers on chromosome 9 (2719 markers, 17.5%) and minimum in chromosome 8 (1222, 7.8%) with average physical gap size of ∼24 kb between corresponding mapped markers (Table <a reveal-id="DST002TB2" data-open="DST002TB2" class="link link-reveal link-table xref-fig">2</a>). These observations suggested that the high-density microsatellite marker-based physical map constructed in this study for foxtail millet genome could be useful for rapid selection of genome-wide microsatellite markers well distributed over nine chromosomes for various large-scale genotyping applications, including mapping of genes and genomes and comparative genome mapping involving foxtail millet and other related crop plants. </p>                        <a id="4631802" scrollto-destination="4631802"></a>
<div content-id="DST002TB2" class="table-modal"><div class="table-wrap"><div id="DST002TB2" class="table-wrap-title" data-id="DST002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of chromosomal distribution and average density of microsatellite markers mapped on the nine chromosomes of foxtail millet</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>1751 (11.3) </td><td>41.59 </td></tr><tr><td>Chr. 2 </td><td>1929 (12.4) </td><td>39.21 </td></tr><tr><td>Chr. 3 </td><td>1969 (12.6) </td><td>38.91 </td></tr><tr><td>Chr. 4 </td><td>1352 (8.8) </td><td>33.55 </td></tr><tr><td>Chr. 5 </td><td>2019 (12.9) </td><td>42.78 </td></tr><tr><td>Chr. 6 </td><td>1235 (7.9) </td><td>34.31 </td></tr><tr><td>Chr. 7 </td><td>1377 (8.8) </td><td>38.36 </td></tr><tr><td>Chr. 8 </td><td>1222 (7.8) </td><td>30.02 </td></tr><tr><td>Chr. 9 </td><td>2719 (17.5) </td><td>46.16 </td></tr></tbody></table></div><div content-id="DST002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>1751 (11.3) </td><td>41.59 </td></tr><tr><td>Chr. 2 </td><td>1929 (12.4) </td><td>39.21 </td></tr><tr><td>Chr. 3 </td><td>1969 (12.6) </td><td>38.91 </td></tr><tr><td>Chr. 4 </td><td>1352 (8.8) </td><td>33.55 </td></tr><tr><td>Chr. 5 </td><td>2019 (12.9) </td><td>42.78 </td></tr><tr><td>Chr. 6 </td><td>1235 (7.9) </td><td>34.31 </td></tr><tr><td>Chr. 7 </td><td>1377 (8.8) </td><td>38.36 </td></tr><tr><td>Chr. 8 </td><td>1222 (7.8) </td><td>30.02 </td></tr><tr><td>Chr. 9 </td><td>2719 (17.5) </td><td>46.16 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631802" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST002TB2" class="table-wrap-title" data-id="DST002TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of chromosomal distribution and average density of microsatellite markers mapped on the nine chromosomes of foxtail millet</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>1751 (11.3) </td><td>41.59 </td></tr><tr><td>Chr. 2 </td><td>1929 (12.4) </td><td>39.21 </td></tr><tr><td>Chr. 3 </td><td>1969 (12.6) </td><td>38.91 </td></tr><tr><td>Chr. 4 </td><td>1352 (8.8) </td><td>33.55 </td></tr><tr><td>Chr. 5 </td><td>2019 (12.9) </td><td>42.78 </td></tr><tr><td>Chr. 6 </td><td>1235 (7.9) </td><td>34.31 </td></tr><tr><td>Chr. 7 </td><td>1377 (8.8) </td><td>38.36 </td></tr><tr><td>Chr. 8 </td><td>1222 (7.8) </td><td>30.02 </td></tr><tr><td>Chr. 9 </td><td>2719 (17.5) </td><td>46.16 </td></tr></tbody></table></div><div content-id="DST002TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Markers mapped (%) </th><th>Density (markers/Mb) </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>1751 (11.3) </td><td>41.59 </td></tr><tr><td>Chr. 2 </td><td>1929 (12.4) </td><td>39.21 </td></tr><tr><td>Chr. 3 </td><td>1969 (12.6) </td><td>38.91 </td></tr><tr><td>Chr. 4 </td><td>1352 (8.8) </td><td>33.55 </td></tr><tr><td>Chr. 5 </td><td>2019 (12.9) </td><td>42.78 </td></tr><tr><td>Chr. 6 </td><td>1235 (7.9) </td><td>34.31 </td></tr><tr><td>Chr. 7 </td><td>1377 (8.8) </td><td>38.36 </td></tr><tr><td>Chr. 8 </td><td>1222 (7.8) </td><td>30.02 </td></tr><tr><td>Chr. 9 </td><td>2719 (17.5) </td><td>46.16 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631802" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4631803 id="4631803" class="section-title" data-legacy-id=s3c>3.3. Amplification and polymorphic potential of microsatellite markers</h3>
<p>A selected set of 159 genomic microsatellite markers (148 from class I and 11 from class II), distributed over 9 chromosomes of foxtail millet, gave clear, successful and reproducible amplification with expected product size (bp) in <em>S. italica</em> cv. Prasad. It indicated about 100% amplification potential of designed genomic microsatellite markers, and, thus, the developed 15 573 microsatellite markers for foxtail millet genome are a useful resource for foxtail millet genomics and molecular breeding. One hundred fifty-two (95.6%) of the 159 microsatellite markers amplified unique single allele, whereas 7 markers amplified more than 1 allele/multiple alleles. Thus, a total of 167 alleles were amplified by 159 genomic microsatellite markers in <em>S. italica</em> cv. Prasad (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Furthermore, 132 out of 159 microsatellite markers produced fragments of desired sizes, whereas the amplification products for 27 markers differed from the expected sizes. This inconsistency may be either due to the use of different accessions of foxtail millet in this study and sequenced genome or due to sequencing error.</p><p>To evaluate polymorphism and molecular diversity potential of developed microsatellite markers, the validated and physically mapped 159 microsatellite markers were amplified in a set of 7 accessions of <em>Setaria</em>, including 4 cultivated and 3 wild species. One hundred seven (67.2%) of the 159 genomic microsatellite markers showed polymorphism in 8 cultivated and wild accessions of foxtail millet. A total of 342 alleles ranging from 1 to 6 alleles were amplified by 159 microsatellite markers in 8 foxtail accessions with an average 2.15 alleles per marker locus (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>; Fig. <a reveal-id="DST002F3" data-open="DST002F3" class="link link-reveal link-table xref-fig">3</a>). The polymorphic potentials of these 159 microsatellite markers were further evaluated in the genomic DNA of 2 parental foxtail millet accessions, Prasad and Lepakshi of an F<sub>2</sub> mapping population (cv. Prasad x cv. Lepakshi). Thirty (∼19%) of the 159 markers showed polymorphism between parental accessions of F<sub>2</sub> mapping population. It includes 20 (21.5%), 8 (14.8%) and 2 (16.6%) DNR, TNR and TeNR motifs containing markers, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Remarkably, all the 30 markers that showed parental polymorphism belonged to class I microsatellite types. This clearly suggests the added advantage of long hypervariable class I microsatellites over variable class II microsatellites for revealing higher degree of polymorphism in the foxtail millet accessions and, thus, could be utilized for large-scale genotyping applications. Such observations have also been reported in other genome-wide studies in plant species using a larger set of class I and class II microsatellite markers.<sup><a reveal-id="DST002C13" data-open="DST002C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST002C28" data-open="DST002C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The polymorphic potential (67.2%) of genomic microsatellite markers estimated among foxtail millet cultivated and wild accessions was higher than that reported using the microsatellite markers derived from foxtail millet genomic sequences.<sup><a reveal-id="DST002C14" data-open="DST002C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The higher polymorphic potential of genomic microsatellite markers is expected because of their derivation from non-coding sequence components of the foxtail genome.<sup><a reveal-id="DST002C10" data-open="DST002C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST002C12" data-open="DST002C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Besides, the validated physically mapped markers could enable to discriminate all the eight cultivated and wild foxtail millet accessions from each other with a level diversity from 32 to 59% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The distant clustering patterns of <em>S. italica</em> subsp. viridis and other related <em>Setaria</em> wild species such as <em>Setaria sphacelata</em> and <em>Setaria verticillata</em> from the cultivated <em>Setaria</em> species were expected. The clustering patterns derived among cultivated and wild <em>Setaria</em> species corresponded well with their phylogenetic relationships and taxonomic classifications. Moreover, this accords well with the earlier studies conducted using ISSR and transposon-based markers.<sup><a reveal-id="DST002C29" data-open="DST002C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST002C30" data-open="DST002C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The microsatellite markers, thus, showed differentiation between cultivated and wild <em>Setaria</em> species that could be useful for introgression breeding for transferring genes/alleles of agronomic importance like stress tolerance from the wild species to the cultivated genetic backgrounds of foxtail millet. Therefore, the genomic microsatellite markers with high amplification and polymorphic potential distributed over nine chromosomes of foxtail millet genome designed in the study could be useful for many large-scale genotyping applications in foxtail millet. </p>                        <a id="4631806" scrollto-destination="4631806"></a>
<div id="DST002F3" data-id="DST002F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00203.jpeg?Expires=2147483647&Signature=Oa1x6OajzRy7tMnB7wx9F~YuwYWMh8~Zt4~sSdzGF5kMD6VU4jMx2dkK5i0uMigs9n76mBE0zVoiNyfQdDdG4WrZJrGSENJOYTVnDem1sXjASAsiVeo~lNYQENEcEyo6~I8h2cyKZWe8g6YH1lhqrWMicZdq~JjATxdw9vfrwF~EGqloErJuw4rr~ZNfBeFdSmYXABPys1cVRlGgo03j2zjgCDtdNj9pDECSpI4~tG2mI1OIiBTWag0Ofllz~hLZy94AN9rth3nS25Y6U~Kk2WpSXof7eK8ycnAzXGDm1fOTr-CqVF1gxnVmHD7fKIjMw9CLO67zVMS4EQQ6vgj7kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gel showing amplification profiles of one microsatellite marker SiGMS 3261 and its fragment length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard." path-from-xml="dst00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631806/dst00203.jpeg" path-from-xml="dst00203.jpeg" target="_blank">View large</a><a section="4631806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00203.jpeg?Expires=2147483647&Signature=G8zGDX3ZMRshCz8-JRZgoKUDwOXxGjStwAhMYbLixeiOHoDtyKq8kMEE9UdLUgrP-sT5wgo9GfsgeLX34Qq5NPE5V9nfPFiXO~vO9usaeK~J1Xnc2qrfSqkeL33mCMfbvwtS5L1eutAW3Z4soPNXpRSEVsdEuamm~ShWxMQHIkHsvO8qNcrYucIAtIIYw911RKFxRDyMSeCIoYupX0nmEpdeKnskakeKqk-A3dQmreZc31WxwtTtzxZq11JSIdQdJv-f9~NsikzDERZ2ICHhW-14W5Ouv3SMZKCa7kYpgeYN70POGJ~31rTEwm-3Sqrhj~nSJrTmJYmCqR70lrJnGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631806&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gel showing amplification profiles of one microsatellite marker SiGMS 3261 and its fragment length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard.</p></div></div><div content-id="DST002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00203.jpeg?Expires=2147483647&Signature=Oa1x6OajzRy7tMnB7wx9F~YuwYWMh8~Zt4~sSdzGF5kMD6VU4jMx2dkK5i0uMigs9n76mBE0zVoiNyfQdDdG4WrZJrGSENJOYTVnDem1sXjASAsiVeo~lNYQENEcEyo6~I8h2cyKZWe8g6YH1lhqrWMicZdq~JjATxdw9vfrwF~EGqloErJuw4rr~ZNfBeFdSmYXABPys1cVRlGgo03j2zjgCDtdNj9pDECSpI4~tG2mI1OIiBTWag0Ofllz~hLZy94AN9rth3nS25Y6U~Kk2WpSXof7eK8ycnAzXGDm1fOTr-CqVF1gxnVmHD7fKIjMw9CLO67zVMS4EQQ6vgj7kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gel showing amplification profiles of one microsatellite marker SiGMS 3261 and its fragment length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard." path-from-xml="dst00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631806/dst00203.jpeg" path-from-xml="dst00203.jpeg" target="_blank">View large</a><a section="4631806" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00203.jpeg?Expires=2147483647&Signature=G8zGDX3ZMRshCz8-JRZgoKUDwOXxGjStwAhMYbLixeiOHoDtyKq8kMEE9UdLUgrP-sT5wgo9GfsgeLX34Qq5NPE5V9nfPFiXO~vO9usaeK~J1Xnc2qrfSqkeL33mCMfbvwtS5L1eutAW3Z4soPNXpRSEVsdEuamm~ShWxMQHIkHsvO8qNcrYucIAtIIYw911RKFxRDyMSeCIoYupX0nmEpdeKnskakeKqk-A3dQmreZc31WxwtTtzxZq11JSIdQdJv-f9~NsikzDERZ2ICHhW-14W5Ouv3SMZKCa7kYpgeYN70POGJ~31rTEwm-3Sqrhj~nSJrTmJYmCqR70lrJnGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631806&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gel showing amplification profiles of one microsatellite marker SiGMS 3261 and its fragment length polymorphism among foxtail millet and related species. The amplicons are resolved in 2% agarose gel along with 100 bp DNA size standard.</p></div></div>                            <h3 scrollto-destination=4631807 id="4631807" class="section-title" data-legacy-id=s3d>3.4. Cross-genera transferability of microsatellite markers</h3>
<p>Of the 159 genomic microsatellite markers validated, 58 markers were randomly selected to assess their utility for cross-genera transferability among millet and non-millet species. Of the 58 microsatellite markers assayed, 53 (91.3%) gave consistent amplification in Barnyard millet, 55 (94.8%) in finger millet, 51 (87.9%) in kodo millet, 55 (94.9%) in little millet, 54 (93.1%) in pearl millet, 51 (87.9%) in proso millet, 46 (79.3%) in switchgrass, 57 (98.2%) in guinea grass, 52 (89.6%) in sorghum, 42 (71.2%) in wheat, 51 (87.9%) in rice and 53 (91.4%) in maize with an average cross-genera transferability potential of ∼89% (Fig. <a reveal-id="DST002F3" data-open="DST002F3" class="link link-reveal link-table xref-fig">3</a>). All the 58 microsatellite markers have ability to distinguish the investigated 13 millet and non-millet species into 2 distinct groups (Fig. <a reveal-id="DST002F4" data-open="DST002F4" class="link link-reveal link-table xref-fig">4</a>). The obtained high average cross-genera transferability potential (∼89%) of genomic microsatellite markers is consistent with our previous study, where also a higher level of transferability (73%) of foxtail millet genomic microsatellite markers across various millet and non-millet species was observed.<sup><a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Apart from this, our results are in close agreement to several previous studies reporting high cross-species transferability of genomic microsatellite markers from <em>Lolium</em> to <em>Festuca</em> complex (∼71%),<sup><a reveal-id="DST002C31" data-open="DST002C31" class="link link-ref link-reveal xref-bibr">31</a></sup> rice to bamboo (∼68.3%)<sup><a reveal-id="DST002C32" data-open="DST002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and groundnut to other legumes (∼71%).<sup><a reveal-id="DST002C33" data-open="DST002C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The high degree of cross-transferability of genomic microsatellite markers across related foxtail millet species could be due to more representation of microsatellite markers that were derived from the conserved expressed component of the foxtail millet genome sequences and, thus, resulting in higher transferability of microsatellite markers like genic microsatellite markers across closely related and even distantly related genera.<sup><a reveal-id="DST002C10" data-open="DST002C10" class="link link-ref link-reveal xref-bibr">10</a>–<a reveal-id="DST002C12" data-open="DST002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The sequencing of PCR products of microsatellite markers amplifying multiple alleles in millet and non-millet species revealed the presence of expected microsatellite repeat motifs in foxtail millet as predicted <em>in silico</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). The fragment length polymorphism among the accessions explained the variation in the length of the microsatellite repeats and the presence of point mutations, including insertion/deletions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Therefore, based on amplicon sequencing, the microsatellite markers showing cross-transferability across millet and non-millet species were valid and, thus, have utility in comparative genome mapping studies in both millets and non-millets. Besides, the markers showing high-cross transferability across millet and non-millet species can be utilized for genetic enhancement of orphan crops using the marker-related genetic information of foxtail millet genome. </p>                        <a id="4631809" scrollto-destination="4631809"></a>
<div id="DST002F4" data-id="DST002F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00204.jpeg?Expires=2147483647&Signature=kMfih-dzwEm7oTJDm1X313q9P4PlxNp0gUEpoFBPAsO~W-doBmKadtWSa46HzY02x-C1KTy~5cfEQ4ACZybD~~0NEFI9HshFOlueY5BQY8poV~vsjMlnFiqkeVs0bBLEfZQBKOp01RcJ0mv0qEMJLWx9k3ZlV~f~yozuB1NBYwi~h99GKlZ3mKItlt5gAO9e3Pd9XW7avU5O1XaCJ3TgEO2bc3~2wCvjYjYkomJzvqGcFXZmHtAl0Vz3gs1317oWIhRKv9q1K01smZIh3cKn-BMOF-a3B48Crmy4i6SFtSkLvanGbBQc80AOF35DIcotzcNSGwuF871HxqppFvOUGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic relationships among 9 millet and 4 non-millet grass species based on 58 foxtail millet microsatellite markers using neighbor-joining clustering. Nine millet species, including foxtail millet, were clearly differentiated from the four non-millet grass species, and expected genetic relationships among species under study were also evident." path-from-xml="dst00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631809/dst00204.jpeg" path-from-xml="dst00204.jpeg" target="_blank">View large</a><a section="4631809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00204.jpeg?Expires=2147483647&Signature=QOXGLWrtrNr9rJx7KaRfBv~WGxTJ34NJ~7WFq4CSNj2bQiemgcm7lkqb7u8zjg1TavbWIe4e~W0at7aO3q~7LStsDM~1zsZhA52kKvD7~5WJq7XdcAmjqHSBbPIqPiX8W4IVPYz~rRrGjAN0DeJQ8tWPTj-ySSXay4DKND80aweGHtLvyuCKTk8C6zvLjQWeUQpiAyy9XlgNBRpifgC1rmt6On7y1t5~nQUcJBRirKh5NfeZLAfJLol4UMDC0aD2UFqZX1qBljCwxDQgIBXOcGvZdPtytSFFMa2KSggDYdMjmYDlU84bn-5mwBM0uqA9mWNvNtfbQwsMa3nfSC3aCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631809&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic relationships among 9 millet and 4 non-millet grass species based on 58 foxtail millet microsatellite markers using neighbor-joining clustering. Nine millet species, including foxtail millet, were clearly differentiated from the four non-millet grass species, and expected genetic relationships among species under study were also evident.</p></div></div><div content-id="DST002F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00204.jpeg?Expires=2147483647&Signature=kMfih-dzwEm7oTJDm1X313q9P4PlxNp0gUEpoFBPAsO~W-doBmKadtWSa46HzY02x-C1KTy~5cfEQ4ACZybD~~0NEFI9HshFOlueY5BQY8poV~vsjMlnFiqkeVs0bBLEfZQBKOp01RcJ0mv0qEMJLWx9k3ZlV~f~yozuB1NBYwi~h99GKlZ3mKItlt5gAO9e3Pd9XW7avU5O1XaCJ3TgEO2bc3~2wCvjYjYkomJzvqGcFXZmHtAl0Vz3gs1317oWIhRKv9q1K01smZIh3cKn-BMOF-a3B48Crmy4i6SFtSkLvanGbBQc80AOF35DIcotzcNSGwuF871HxqppFvOUGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic relationships among 9 millet and 4 non-millet grass species based on 58 foxtail millet microsatellite markers using neighbor-joining clustering. Nine millet species, including foxtail millet, were clearly differentiated from the four non-millet grass species, and expected genetic relationships among species under study were also evident." path-from-xml="dst00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631809/dst00204.jpeg" path-from-xml="dst00204.jpeg" target="_blank">View large</a><a section="4631809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00204.jpeg?Expires=2147483647&Signature=QOXGLWrtrNr9rJx7KaRfBv~WGxTJ34NJ~7WFq4CSNj2bQiemgcm7lkqb7u8zjg1TavbWIe4e~W0at7aO3q~7LStsDM~1zsZhA52kKvD7~5WJq7XdcAmjqHSBbPIqPiX8W4IVPYz~rRrGjAN0DeJQ8tWPTj-ySSXay4DKND80aweGHtLvyuCKTk8C6zvLjQWeUQpiAyy9XlgNBRpifgC1rmt6On7y1t5~nQUcJBRirKh5NfeZLAfJLol4UMDC0aD2UFqZX1qBljCwxDQgIBXOcGvZdPtytSFFMa2KSggDYdMjmYDlU84bn-5mwBM0uqA9mWNvNtfbQwsMa3nfSC3aCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631809&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic relationships among 9 millet and 4 non-millet grass species based on 58 foxtail millet microsatellite markers using neighbor-joining clustering. Nine millet species, including foxtail millet, were clearly differentiated from the four non-millet grass species, and expected genetic relationships among species under study were also evident.</p></div></div>                            <h3 scrollto-destination=4631810 id="4631810" class="section-title" data-legacy-id=s3e>3.5. <em>In silico</em> comparative genome mapping between foxtail millet and other grass species</h3>
<p>For comparative mapping, the physically mapped 15 573 microsatellite markers on the 9 chromosomes of foxtail millet were compared with their physical location on the chromosomes of other related grass genomes namely sorghum, maize and rice (Table <a reveal-id="DST002TB3" data-open="DST002TB3" class="link link-reveal link-table xref-fig">3</a>; Fig. <a reveal-id="DST002F5" data-open="DST002F5" class="link link-reveal link-table xref-fig">5</a>A–C; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S4–S6</a>). <em>In silico</em> comparative genome mapping revealed maximum proportion of sequence-based orthology and, thus, syntenic relationship of microsatellite markers distributed over 9 foxtail millet chromosomes with 10 chromosomes of sorghum (16.9%, 2631) and maize (14.5%, 2252) and 12 rice (6.4%, 1005) chromosomes. Many of the physically mapped microsatellite markers of foxtail millet also showed syntenic relationships with more than one chromosome of sorghum, maize and rice, reflecting the occurrence of segmental duplication events between the chromosomal stretches of respective genomes. This is expected considering the presence of higher number (10–12) of chromosomes in rice, maize and sorghum when compared with 9 chromosomes of foxtail millet. A number of foxtail millet microsatellite markers showed significant matches with different chromosomes of sorghum, maize and rice. </p>                        <a id="4631812" scrollto-destination="4631812"></a>
<div content-id="DST002TB3" class="table-modal"><div class="table-wrap"><div id="DST002TB3" class="table-wrap-title" data-id="DST002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>A summary of microsatellite marker-based comparative mapping showing maximum syntenic relationships of foxtail millet chromosomes with sorghum, maize and rice chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>Chr. 4 (309, 92.5%) </td><td>Chr. 5 (184, 62.5%) and Chr. 4 (85, 28.9%) </td><td>Chr. 2 (131, 86.1%) </td></tr><tr><td>Chr. 2 </td><td>Chr. 2 (271, 86.9%) </td><td>Chr. 7(155, 54.1%) and Chr. 2 (90, 31.4%) </td><td>Chr. 9 (45, 46.8%) and Chr. 7 (31, 32.3%) </td></tr><tr><td>Chr. 3 </td><td>Chr. 9 (157, 49.8%) and Chr. 8 (85, 26.9%) </td><td>Chr. 6 (83, 33.1%), Chr. 8 (42, 16.7%) and Chr. 10 (28, 11.1%) </td><td>Chr. 5 (55, 50%) and Chr. 12 (37, 33.6%) </td></tr><tr><td>Chr. 4 </td><td>Chr. 10 (169, 84.07%) </td><td>Chr. 9 (75, 42.1%), Chr. 6 (51, 28.6%) and Chr. 5 (28, 15.7%) </td><td>Chr. 6 (37, 69.8%) </td></tr><tr><td>Chr. 5 </td><td>Chr. 3 (332, 88.06%) </td><td>Chr. 3 (206, 60.2%) and Chr. 8 (93, 27.2%) </td><td>Chr. 1 (163, 89.5%) </td></tr><tr><td>Chr. 6 </td><td>Chr. 7 (140, 76.5%) </td><td>Chr. 1 (56, 34.3%) and Chr. 4 (51, 31.3%) </td><td>Chr. 8 (35, 67.3%) </td></tr><tr><td>Chr. 7 </td><td>Chr. 6 (185, 71.7%) </td><td>Chr. 2 (99, 43.8%) and Chr. 10 (74, 32.7%) </td><td>Chr. 4 (68, 64.1%) </td></tr><tr><td>Chr. 8 </td><td>Chr. 5 (36, 48.6%) </td><td>Chr. 4 (50, 48.1%) </td><td>Chr. 1 (39, 79.6%) and Chr. 11 (29, 59.2%) </td></tr><tr><td>Chr. 9 </td><td>Chr. 1 (495, 86.9%) </td><td>Chr. 1 (262, 64.2%), Chr. 9 (92, 22.5%) and Chr. 5 (64, 15.6%) </td><td>Chr. 3 (151, 73.7%), Chr. 1 (41, 20%) and Chr. 10 (31, 15.1%) </td></tr></tbody></table></div><div content-id="DST002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>Chr. 4 (309, 92.5%) </td><td>Chr. 5 (184, 62.5%) and Chr. 4 (85, 28.9%) </td><td>Chr. 2 (131, 86.1%) </td></tr><tr><td>Chr. 2 </td><td>Chr. 2 (271, 86.9%) </td><td>Chr. 7(155, 54.1%) and Chr. 2 (90, 31.4%) </td><td>Chr. 9 (45, 46.8%) and Chr. 7 (31, 32.3%) </td></tr><tr><td>Chr. 3 </td><td>Chr. 9 (157, 49.8%) and Chr. 8 (85, 26.9%) </td><td>Chr. 6 (83, 33.1%), Chr. 8 (42, 16.7%) and Chr. 10 (28, 11.1%) </td><td>Chr. 5 (55, 50%) and Chr. 12 (37, 33.6%) </td></tr><tr><td>Chr. 4 </td><td>Chr. 10 (169, 84.07%) </td><td>Chr. 9 (75, 42.1%), Chr. 6 (51, 28.6%) and Chr. 5 (28, 15.7%) </td><td>Chr. 6 (37, 69.8%) </td></tr><tr><td>Chr. 5 </td><td>Chr. 3 (332, 88.06%) </td><td>Chr. 3 (206, 60.2%) and Chr. 8 (93, 27.2%) </td><td>Chr. 1 (163, 89.5%) </td></tr><tr><td>Chr. 6 </td><td>Chr. 7 (140, 76.5%) </td><td>Chr. 1 (56, 34.3%) and Chr. 4 (51, 31.3%) </td><td>Chr. 8 (35, 67.3%) </td></tr><tr><td>Chr. 7 </td><td>Chr. 6 (185, 71.7%) </td><td>Chr. 2 (99, 43.8%) and Chr. 10 (74, 32.7%) </td><td>Chr. 4 (68, 64.1%) </td></tr><tr><td>Chr. 8 </td><td>Chr. 5 (36, 48.6%) </td><td>Chr. 4 (50, 48.1%) </td><td>Chr. 1 (39, 79.6%) and Chr. 11 (29, 59.2%) </td></tr><tr><td>Chr. 9 </td><td>Chr. 1 (495, 86.9%) </td><td>Chr. 1 (262, 64.2%), Chr. 9 (92, 22.5%) and Chr. 5 (64, 15.6%) </td><td>Chr. 3 (151, 73.7%), Chr. 1 (41, 20%) and Chr. 10 (31, 15.1%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631812" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST002TB3" class="table-wrap-title" data-id="DST002TB3"><div class="label -label">Table 3.</div><div class="caption"><p>A summary of microsatellite marker-based comparative mapping showing maximum syntenic relationships of foxtail millet chromosomes with sorghum, maize and rice chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>Chr. 4 (309, 92.5%) </td><td>Chr. 5 (184, 62.5%) and Chr. 4 (85, 28.9%) </td><td>Chr. 2 (131, 86.1%) </td></tr><tr><td>Chr. 2 </td><td>Chr. 2 (271, 86.9%) </td><td>Chr. 7(155, 54.1%) and Chr. 2 (90, 31.4%) </td><td>Chr. 9 (45, 46.8%) and Chr. 7 (31, 32.3%) </td></tr><tr><td>Chr. 3 </td><td>Chr. 9 (157, 49.8%) and Chr. 8 (85, 26.9%) </td><td>Chr. 6 (83, 33.1%), Chr. 8 (42, 16.7%) and Chr. 10 (28, 11.1%) </td><td>Chr. 5 (55, 50%) and Chr. 12 (37, 33.6%) </td></tr><tr><td>Chr. 4 </td><td>Chr. 10 (169, 84.07%) </td><td>Chr. 9 (75, 42.1%), Chr. 6 (51, 28.6%) and Chr. 5 (28, 15.7%) </td><td>Chr. 6 (37, 69.8%) </td></tr><tr><td>Chr. 5 </td><td>Chr. 3 (332, 88.06%) </td><td>Chr. 3 (206, 60.2%) and Chr. 8 (93, 27.2%) </td><td>Chr. 1 (163, 89.5%) </td></tr><tr><td>Chr. 6 </td><td>Chr. 7 (140, 76.5%) </td><td>Chr. 1 (56, 34.3%) and Chr. 4 (51, 31.3%) </td><td>Chr. 8 (35, 67.3%) </td></tr><tr><td>Chr. 7 </td><td>Chr. 6 (185, 71.7%) </td><td>Chr. 2 (99, 43.8%) and Chr. 10 (74, 32.7%) </td><td>Chr. 4 (68, 64.1%) </td></tr><tr><td>Chr. 8 </td><td>Chr. 5 (36, 48.6%) </td><td>Chr. 4 (50, 48.1%) </td><td>Chr. 1 (39, 79.6%) and Chr. 11 (29, 59.2%) </td></tr><tr><td>Chr. 9 </td><td>Chr. 1 (495, 86.9%) </td><td>Chr. 1 (262, 64.2%), Chr. 9 (92, 22.5%) and Chr. 5 (64, 15.6%) </td><td>Chr. 3 (151, 73.7%), Chr. 1 (41, 20%) and Chr. 10 (31, 15.1%) </td></tr></tbody></table></div><div content-id="DST002TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Foxtail millet chromosomes </th><th>Sorghum chromosomes </th><th>Maize chromosomes </th><th>Rice chromosomes </th></tr></thead><tbody><tr><td>Chr. 1 </td><td>Chr. 4 (309, 92.5%) </td><td>Chr. 5 (184, 62.5%) and Chr. 4 (85, 28.9%) </td><td>Chr. 2 (131, 86.1%) </td></tr><tr><td>Chr. 2 </td><td>Chr. 2 (271, 86.9%) </td><td>Chr. 7(155, 54.1%) and Chr. 2 (90, 31.4%) </td><td>Chr. 9 (45, 46.8%) and Chr. 7 (31, 32.3%) </td></tr><tr><td>Chr. 3 </td><td>Chr. 9 (157, 49.8%) and Chr. 8 (85, 26.9%) </td><td>Chr. 6 (83, 33.1%), Chr. 8 (42, 16.7%) and Chr. 10 (28, 11.1%) </td><td>Chr. 5 (55, 50%) and Chr. 12 (37, 33.6%) </td></tr><tr><td>Chr. 4 </td><td>Chr. 10 (169, 84.07%) </td><td>Chr. 9 (75, 42.1%), Chr. 6 (51, 28.6%) and Chr. 5 (28, 15.7%) </td><td>Chr. 6 (37, 69.8%) </td></tr><tr><td>Chr. 5 </td><td>Chr. 3 (332, 88.06%) </td><td>Chr. 3 (206, 60.2%) and Chr. 8 (93, 27.2%) </td><td>Chr. 1 (163, 89.5%) </td></tr><tr><td>Chr. 6 </td><td>Chr. 7 (140, 76.5%) </td><td>Chr. 1 (56, 34.3%) and Chr. 4 (51, 31.3%) </td><td>Chr. 8 (35, 67.3%) </td></tr><tr><td>Chr. 7 </td><td>Chr. 6 (185, 71.7%) </td><td>Chr. 2 (99, 43.8%) and Chr. 10 (74, 32.7%) </td><td>Chr. 4 (68, 64.1%) </td></tr><tr><td>Chr. 8 </td><td>Chr. 5 (36, 48.6%) </td><td>Chr. 4 (50, 48.1%) </td><td>Chr. 1 (39, 79.6%) and Chr. 11 (29, 59.2%) </td></tr><tr><td>Chr. 9 </td><td>Chr. 1 (495, 86.9%) </td><td>Chr. 1 (262, 64.2%), Chr. 9 (92, 22.5%) and Chr. 5 (64, 15.6%) </td><td>Chr. 3 (151, 73.7%), Chr. 1 (41, 20%) and Chr. 10 (31, 15.1%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4631812" target="_blank">View Large</a></div></div>                        <a id="4631813" scrollto-destination="4631813"></a>
<div id="DST002F5" data-id="DST002F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00205.jpeg?Expires=2147483647&Signature=C6Iaow~OiDYZmv7HNVq4PV3K2Gf6ahqbkfnMjWO9owRQ4SOvI7Jd3~wOuWT6F-OXq908QE9DdpHGmqcuIkqt5VUoNXElWpp8d8HX9Zm83Wu7VGEk1Pkp7kFPWamSp6wECq0r95A3zesF3tbRYxZedA0hjY9Rfh0CussmehkUUir7PJVS87rNARqUaG10RX9Wud0Ur4jnVGs5QAdv6ARNQLPNK1~ZprWUIpl~jdteu3C~c~FM4XXuzP9q7NpF3VwnQL9vTNqkkB-BcKUTaMaQd2ssOj8Tbhn1agVfT187v~6I21ZK6vqRAf~D25E9jJMOen56el3xtuiSEHbHisYNSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome relationships of foxtail millet with other grass species. Syntenic relationship of foxtail millet genome with (A) sorghum, (B) maize and (C) rice chromosomes using 15 573 physically mapped foxtail millet microsatellite markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on microsatellite markers were apparent." path-from-xml="dst00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631813/dst00205.jpeg" path-from-xml="dst00205.jpeg" target="_blank">View large</a><a section="4631813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00205.jpeg?Expires=2147483647&Signature=teI1-I2b3r1WmqZck2dBoxlsOlG1XXwuowWkgMo2k2-e~HS7o1Qv~fAPKFMOzmAeuNOWXV6rUKuyoMcFeiEI~Gs5mQuhR19wjG5WtQMbWsDLi07s74hz7VwNmGLGdQuTwzDMNLvtXrbC0yrMs1hLSWvjtMBF75Xd~KCMv4YmXEasSG1sxeDOghJHp~FdYnZvZyuKwQaCVk-Jn1ZC9wDjYNaYTpAbBMOqwOTvfQL8CBkjG~0pMSTURPzZVwvGUJb3otPUjGx0kEvcJJZjQWTXCiNZlqKuOkr2xMSl6nGM7h4lnOcUFDxK3f0knDXSC7Vv9rjmt~T-5ZuSVespy3yDjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631813&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome relationships of foxtail millet with other grass species. Syntenic relationship of foxtail millet genome with (A) sorghum, (B) maize and (C) rice chromosomes using 15 573 physically mapped foxtail millet microsatellite markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on microsatellite markers were apparent.</p></div></div><div content-id="DST002F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/m_dst00205.jpeg?Expires=2147483647&Signature=C6Iaow~OiDYZmv7HNVq4PV3K2Gf6ahqbkfnMjWO9owRQ4SOvI7Jd3~wOuWT6F-OXq908QE9DdpHGmqcuIkqt5VUoNXElWpp8d8HX9Zm83Wu7VGEk1Pkp7kFPWamSp6wECq0r95A3zesF3tbRYxZedA0hjY9Rfh0CussmehkUUir7PJVS87rNARqUaG10RX9Wud0Ur4jnVGs5QAdv6ARNQLPNK1~ZprWUIpl~jdteu3C~c~FM4XXuzP9q7NpF3VwnQL9vTNqkkB-BcKUTaMaQd2ssOj8Tbhn1agVfT187v~6I21ZK6vqRAf~D25E9jJMOen56el3xtuiSEHbHisYNSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome relationships of foxtail millet with other grass species. Syntenic relationship of foxtail millet genome with (A) sorghum, (B) maize and (C) rice chromosomes using 15 573 physically mapped foxtail millet microsatellite markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on microsatellite markers were apparent." path-from-xml="dst00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4631813/dst00205.jpeg" path-from-xml="dst00205.jpeg" target="_blank">View large</a><a section="4631813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst00205.jpeg?Expires=2147483647&Signature=teI1-I2b3r1WmqZck2dBoxlsOlG1XXwuowWkgMo2k2-e~HS7o1Qv~fAPKFMOzmAeuNOWXV6rUKuyoMcFeiEI~Gs5mQuhR19wjG5WtQMbWsDLi07s74hz7VwNmGLGdQuTwzDMNLvtXrbC0yrMs1hLSWvjtMBF75Xd~KCMv4YmXEasSG1sxeDOghJHp~FdYnZvZyuKwQaCVk-Jn1ZC9wDjYNaYTpAbBMOqwOTvfQL8CBkjG~0pMSTURPzZVwvGUJb3otPUjGx0kEvcJJZjQWTXCiNZlqKuOkr2xMSl6nGM7h4lnOcUFDxK3f0knDXSC7Vv9rjmt~T-5ZuSVespy3yDjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4631813&ar=334072&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome relationships of foxtail millet with other grass species. Syntenic relationship of foxtail millet genome with (A) sorghum, (B) maize and (C) rice chromosomes using 15 573 physically mapped foxtail millet microsatellite markers. Maximum syntenic relationships of foxtail millet chromosomes with sorghum chromosomes based on microsatellite markers were apparent.</p></div></div><p>Foxtail millet–sorghum synteny: between foxtail millet and sorghum genomes, 2631 microsatellite marker loci distributed over 9 chromosomes of foxtail millet showed significant matches with 2635 genomic regions of 10 sorghum chromosomes (Table <a reveal-id="DST002TB3" data-open="DST002TB3" class="link link-reveal link-table xref-fig">3</a>; Fig. <a reveal-id="DST002F5" data-open="DST002F5" class="link link-reveal link-table xref-fig">5</a>A). All the nine foxtail millet chromosomes showed considerable and higher average frequency (16.9%) of microsatellite marker-based syntenic relationship with specific sorghum chromosomes. The physically mapped microsatellite markers on the foxtail millet chromosome 1 (309) showed maximum synteny (92.5%) with sorghum chromosome 4, followed between foxtail millet chromosome 5 and sorghum chromosome 3 (88.1%), between foxtail millet chromosome 2 and sorghum chromosome 2 (86.9%), between foxtail millet chromosome 9 and sorghum chromosome 1 (86.9%) and minimum between foxtail millet chromosome 8 and sorghum chromosome 5 (48.6%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).</p><p>Foxtail millet–maize synteny: the comparative mapping between foxtail millet and maize genomes revealed syntenic relationship of 2252 microsatellite marker loci distributed over 9 chromosomes of foxtail millet with 2337 genomic regions on 10 chromosomes of maize (Table <a reveal-id="DST002TB3" data-open="DST002TB3" class="link link-reveal link-table xref-fig">3</a>; Fig. <a reveal-id="DST002F5" data-open="DST002F5" class="link link-reveal link-table xref-fig">5</a>B). On an average, 14.5% syntenic relationship of microsatellite marker loci between foxtail millet and maize chromosomes was observed. The syntenic relationships of microsatellite marker loci were maximum between foxtail millet chromosome 9 with maize chromosome 1 (64.2%), followed between foxtail millet chromosome 1 and maize chromosome 5 (62.5%), between foxtail millet chromosome 5 and maize chromosome 3 (60.2%) and minimum between foxtail millet chromosome 3 and maize chromosome 6 (33.1%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p><p>Foxtail millet–rice synteny: the microsatellite markers physically mapped on the foxtail millet chromosomes showed least synteny with 12 rice chromosomes with average frequency of 6.4% that is much lower than that with the sorghum and maize chromosomes (Table <a reveal-id="DST002TB3" data-open="DST002TB3" class="link link-reveal link-table xref-fig">3</a>; Fig. <a reveal-id="DST002F5" data-open="DST002F5" class="link link-reveal link-table xref-fig">5</a>C). Only 1005 foxtail microsatellite marker loci showed significant matches with 1130 genomic regions spanning over 12 chromosomes of rice genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Maximum synteny of microsatellite marker loci be-tween foxtail millet chromosome 5 and rice chromosome 1 (89.5%) and minimum between foxtail millet chromosome 2 and rice chromosome 9 (46.8%) was observed.</p><p>Previous reports showed the mapping of similar microsatellite markers either genetically or physically on orthologous or syntenic chromosomes of different related plant genomes.<sup><a reveal-id="DST002C1" data-open="DST002C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST002C6" data-open="DST002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST002C34  DST002C36" data-open="DST002C34  DST002C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup> Comparative microsatellite marker-based genome mapping revealed higher degree of synteny between foxtail and sorghum genome, followed by maize and rice that is expected due to their taxonomic relationship in which foxtail millet, sorghum and maize belong to the same subfamily Panicoideae, whereas rice belongs to Ehrhartoideae. This result also indicates declination of synteny with increasing phylogenetic distance among plant species. The syntenic relationships inferred among the chromosomes of foxtail millet and three cereal species (sorghum, maize and rice) in the present study agreed well with the earlier study of Devos<em>.</em><sup><a reveal-id="DST002C35" data-open="DST002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> However, using the genome-wide microsatellite markers, the present study generated few previously unobserved interesting information like syntenic relationships between foxtail chromosome 9 and rice chromosome 3 (73.7%) and foxtail millet chromosome 8 with rice chromosome 1 (79.5%) that was not reported earlier.<sup><a reveal-id="DST002C35" data-open="DST002C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>Our results are in accordance with our earlier study, where we reported ∼18% sequence homology of microsatellite markers in the foxtail millet with sorghum, ∼16% with maize and ∼5% with rice.<sup><a reveal-id="DST002C15" data-open="DST002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In the synteny analysis, the observed high level of similarity provides an idea on genome conservation among grasses. Chromosomal rearrangements such as duplication, inversion and translocation were also evidenced as many of foxtail millet marker loci that were mapped to more than one chromosome of analysed species. Although our results do not provide a whole genome view due to the limitations of regions studied and biasness of markers used, our comparative studies have been insightful in understanding the evolutionary process among grasses involving the foxtail millet genome. Therefore, further investigation is required to evaluate the detailed level of collinearity between sorghum, maize and rice with foxtail millet genomes. The microsatellite marker-based comparative genome mapping between foxtail millet and other diploid grass species such as sorghum, maize and rice would be useful in map-based isolation of genes of agronomic importance from foxtail millet using the marker-based genotyping information of other related small and diploid grass members.</p>                            <h2 scrollto-destination=4631819 id="4631819" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>In the present study, we conducted a genome-wide analysis and identified 28 342 microsatellite motifs in the foxtail millet. A total of 21 294 primer pairs were designed, of which 15 573 microsatellites were physically mapped on 9 chromosomes of foxtail millet genome, and 159 markers were validated successfully in foxtail millet accessions. The microsatellite markers developed here showed their utility in high amplification and polymorphic potential, cross-genera transferability and <em>in silico</em> comparative physical mapping. These genomic microsatellite markers reported here would be of enormous and immense use for various large-scale genotyping applications, including germplasm characterization, cultivar identification, construction of high-density microsatellite marker-based physical linkage map for gene/QTL discovery and comparative genome mapping involving foxtail millet and other millets, cereals and bioenergy grass species. More importantly, the constructed high-density physical map in this study could act as reference for future studies of researchers for rapid selection of microsatellite markers either across genome-wide/chromosomal level or at targeted chromosomal location. Thus, it would expedite the construction of high-density genetic linkage map and fine mapping of genes/QTLs for important agronomic traits in foxtail millet.</p>                            <h2 scrollto-destination=4631821 id="4631821" class="section-title" data-legacy-id=s5>Conflict of Interest statement</h2>
<p>None declared.</p>                            <h2 scrollto-destination=4631823 id="4631823" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4631825 id="4631825" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was financially supported by the core grant of National Institute of Plant Genome Research (NIPGR), New Delhi, India.</p>                            <h2 scrollto-destination=4631827 id="4631827" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Grateful thanks are due to the Director, National Institute of Plant Genome Research (NIPGR), New Delhi, India for providing facilities. Ms Garima Pandey acknowledges the award of Junior Research Fellowship from University Grants Commission, New Delhi. We are also thankful to the National Bureau of Plant Genetic Resources, New Delhi/Hyderabad/Akola, India and Tamil Nadu Agricultural University, India for providing the seed material. We thank Mr Mehanathan Muthamilarasan, NIPGR, New Delhi for critically reading the manuscript. Computational resources under BTISNET program (DISC facility) are gratefully acknowledged.</p>                            <h2 scrollto-destination=4631829 id="4631829" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Genome relationships: the grass model in current research</div>, <div class="source">Plant Cell</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">637</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20relationships%3A%20the%20grass%20model%20in%20current%20research&amp;author=K.M.%20Devos&amp;author=M.D.%20Gale&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=637-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.5.637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.5.637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.5.637"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10810140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doust</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Foxtail millet: a sequence-driven grass model system</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">137</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxtail%20millet%3A%20a%20sequence-driven%20grass%20model%20system&amp;author=A.N.%20Doust&amp;author=E.A.%20Kellogg&amp;author=K.M.%20Devos&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=137-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.129627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.129627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.129627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lata</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Foxtail millet: a model crop for genetic and genomic studies in bioenergy grasses</div>, <div class="source">Crit. Rev. Biotechnol.</div> , <div class="year">2012</div> <div class="comment">. (September 18, 2012)</div><div class="pub-id-doi"><a href="https://doi.org/10.3109/07388551.2012.716809" class="link link-pub-id-doi" target="_blank">https://doi.org/10.3109/07388551.2012.716809</a></div></div></div></div></div><div content-id="DST002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brutnell</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title"><em>Setaria viridis</em> and <em>Setaria italica</em>, model genetic systems for Panicoid grasses</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">3031</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Setaria%20viridis%20and%20Setaria%20italica%2C%20model%20genetic%20systems%20for%20Panicoid%20grasses&amp;author=P.%20Li&amp;author=T.P.%20Brutnell&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=3031-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21459768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.S.H.</div></div>,  <div class="name"><div class="surname">Percifield</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Reference genome sequence of the model plant <em>Setaria</em></div>, <div class="source">Nature Biotech.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">555</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reference%20genome%20sequence%20of%20the%20model%20plant%20Setaria&amp;author=J.L.%20Bennetzen&amp;author=J.S.H.%20Wang&amp;author=R.%20Percifield&amp;publication_year=2012&amp;journal=Nature%20Biotech.&amp;volume=30&amp;pages=555-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2196"> </span></div> </div></div></div></div></div><div content-id="DST002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Quan</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Genome sequence of foxtail millet (<em>Setaria italica</em>) provides insights into grass evolution and biofuel potential</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">549</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;author=G.%20Zhang&amp;author=X.%20Liu&amp;author=Z.%20Quan&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=549-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22580950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitros</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rokhsar</div> <div class="given-names">D.S.</div></div>. </div><div class="article-title">The foxtail millet genome and its relationship amongst the grasses</div>, <div class="year">2010</div><div class="conf-name">Plant and Animal Genomes XVIII Conference</div><div class="conf-date">January 9–13</div><div class="publisher-loc">San Diego, CA</div> <div class="comment">W328 (abstract)</div></div></div></div></div><div content-id="DST002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delseny</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Laroche</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Penon</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Detection of sequences with Z-DNA forming potential in higher plants</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">1983</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">113</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20sequences%20with%20Z-DNA%20forming%20potential%20in%20higher%20plants&amp;author=M.%20Delseny&amp;author=M.%20Laroche&amp;author=P.%20Penon&amp;publication_year=1983&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=116&amp;pages=113-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(83)90388-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(83)90388-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(83)90388-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6639651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Renz</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Simple sequences are ubiquitous repetitive components of eukaryotic genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1984</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">4127</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequences%20are%20ubiquitous%20repetitive%20components%20of%20eukaryotic%20genomes&amp;author=D.%20Tautz&amp;author=M.%20Renz&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=4127-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.10.4127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.10.4127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.10.4127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6328411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graver</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graver&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotechnol.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15629858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Rajkumar</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Dalal</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Unigene derived microsatellite markers for the cereal genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">808</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unigene%20derived%20microsatellite%20markers%20for%20the%20cereal%20genomes&amp;author=S.K.%20Parida&amp;author=K.A.%20Rajkumar&amp;author=V.%20Dalal&amp;author=N.K.%20Singh&amp;author=T.%20Mohapatra&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=808-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0182-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0182-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0182-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16429310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Kalia</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Sunita</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Informative genomic microsatellite markers for efficient genotyping applications in sugarcane</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">327</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informative%20genomic%20microsatellite%20markers%20for%20efficient%20genotyping%20applications%20in%20sugarcane&amp;author=S.K.%20Parida&amp;author=S.K.%20Kalia&amp;author=K.%20Sunita&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=327-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0902-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0902-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0902-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18946655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Yadav</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Microsatellites in <em>Brassica</em> unigenes: relative abundance, marker design and use in comparative physical mapping and genome analysis</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">55</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20Brassica%20unigenes%3A%20relative%20abundance%2C%20marker%20design%20and%20use%20in%20comparative%20physical%20mapping%20and%20genome%20analysis&amp;author=S.K.%20Parida&amp;author=D.K.%20Yadav&amp;author=T.%20Mohapatra&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20130749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">H.S.</div></div>,  <div class="name"><div class="surname">Chiang</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Kuoh</div> <div class="given-names">C.S.</div></div>. </div><div class="article-title">Development of simple sequence repeats (SSR) markers in <em>Setaria italica</em> (Poaceae) and cross-amplification in related species</div>, <div class="source">Int. J. Mol. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">7835</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20simple%20sequence%20repeats%20(SSR)%20markers%20in%20Setaria%20italica%20(Poaceae)%20and%20cross-amplification%20in%20related%20species&amp;author=H.S.%20Lin&amp;author=C.Y.%20Chiang&amp;author=S.B.%20Chang&amp;author=C.S.%20Kuoh&amp;publication_year=2011&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;volume=12&amp;pages=7835-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms12117835" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms12117835" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms12117835"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sahu</div> <div class="given-names">P.P.</div></div>,  <div class="name"><div class="surname">Vidapu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Sequence based novel genomic microsatellite markers for robust genotyping purposes in foxtail millet [<em>Setaria italica</em> (L.) P. Beauv]</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2012</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">323</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20based%20novel%20genomic%20microsatellite%20markers%20for%20robust%20genotyping%20purposes%20in%20foxtail%20millet%20%5BSetaria%20italica%20(L.)%20P.%20Beauv%5D&amp;author=S.%20Gupta&amp;author=K.%20Kumari&amp;author=P.P.%20Sahu&amp;author=S.%20Vidapu&amp;author=M.%20Prasad&amp;publication_year=2012&amp;journal=Plant%20Cell%20Rep.&amp;volume=31&amp;pages=323-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-011-1168-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-011-1168-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-011-1168-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21993813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jia</div> <div class="given-names">X.P.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">G.Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">T.Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Development of EST-SSR in foxtail millet (<em>Setaria italica</em>)</div>, <div class="source">J. Genet. Res. Crop Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">233</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20EST-SSR%20in%20foxtail%20millet%20(Setaria%20italica)&amp;author=X.P.%20Jia&amp;author=Y.S.%20Shi&amp;author=Y.C.%20Song&amp;author=G.Y.%20Wang&amp;author=T.Y.%20Wang&amp;author=Y.%20Li&amp;publication_year=2007&amp;journal=J.%20Genet.%20Res.%20Crop%20Evol.&amp;volume=54&amp;pages=233-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-006-9139-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-006-9139-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-006-9139-8"> </span></div> </div></div></div></div></div><div content-id="DST002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Deshmukh</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Gaikwad</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Highly variable SSR markers suitable for rice genotyping using agarose gels</div>, <div class="source">Mol. Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">359</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20variable%20SSR%20markers%20suitable%20for%20rice%20genotyping%20using%20agarose%20gels&amp;author=H.%20Singh&amp;author=R.K.%20Deshmukh&amp;author=A.%20Singh&amp;author=A.K.%20Singh&amp;author=K.%20Gaikwad&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=25&amp;pages=359-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9328-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9328-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9328-1"> </span></div> </div></div></div></div></div><div content-id="DST002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sayama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">T.-Y.</div></div>,  <div class="name"><div class="surname">Komatsu</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Development and application of a whole-genome simple sequence repeat panel for high-throughput genotyping in soybean</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">107</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20application%20of%20a%20whole-genome%20simple%20sequence%20repeat%20panel%20for%20high-throughput%20genotyping%20in%20soybean&amp;author=T.%20Sayama&amp;author=T.-Y.%20Hwang&amp;author=K.%20Komatsu&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=107-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21454301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sonah</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Deshmukh</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genome-wide distribution and organization of microsatellites in plants: an insight into marker development in <em>Brachypodium</em></div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e21298</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0021298ma1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0021298ma1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20distribution%20and%20organization%20of%20microsatellites%20in%20plants%3A%20an%20insight%20into%20marker%20development%20in%20Brachypodium&amp;author=H.%20Sonah&amp;author=R.K.%20Deshmukh&amp;author=A.%20Sharma&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e21298" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21713003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaccard</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Nouvelle recherches sur la distribution florale</div>, <div class="source">Bull Soc. Vaud. Sci. Nat.</div> , <div class="year">1908</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">223</div>-<div class="lpage">70</div>)</div></div></div></div><div content-id="DST002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rohlf</div> <div class="given-names">F.J.</div></div>. </div>, <div class="year">1997</div><div class="publisher-loc">Setauket, NY</div> <div class="comment">NTSYS-pc numerical taxonomy and multivariance analysis system Version 2.02e, Exeter Software</div></div></div></div></div><div content-id="DST002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kalia</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Rai</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Kalia</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dhawan</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Microsatellites markers: an overview of the recent progress in Plants</div>, <div class="source">Euphytica</div> , <div class="year">2011</div>, vol. <div class="volume">177</div> (pg. <div class="fpage">309</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20markers%3A%20an%20overview%20of%20the%20recent%20progress%20in%20Plants&amp;author=R.K.%20Kalia&amp;author=M.K.%20Rai&amp;author=S.%20Kalia&amp;author=R.%20Singh&amp;author=A.K.%20Dhawan&amp;publication_year=2011&amp;journal=Euphytica&amp;volume=177&amp;pages=309-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0286-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0286-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0286-9"> </span></div> </div></div></div></div></div><div content-id="DST002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Olivieri</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">PCR-amplified microsatellites as markers in plant genetics</div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">175</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR-amplified%20microsatellites%20as%20markers%20in%20plant%20genetics&amp;author=M.%20Morgante&amp;author=A.M.%20Olivieri&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=175-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.1993.tb00020.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.1993.tb00020.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.1993.tb00020.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8401603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Grover</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kahl</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Mining microsatellites in eukaryotic genomes</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">490</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20microsatellites%20in%20eukaryotic%20genomes&amp;author=P.C.%20Sharma&amp;author=A.%20Grover&amp;author=G.%20Kahl&amp;publication_year=2007&amp;journal=Trends%20Biotechnol.&amp;volume=25&amp;pages=490-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2007.07.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2007.07.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2007.07.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17945369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yasodha</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Characterization of microsatellites in the tribe bambuseae</div>, <div class="source">Gene Conserv.</div> , <div class="year">2011</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">51</div>-<div class="lpage">64</div>)</div></div></div></div><div content-id="DST002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A genome-wide survey of switchgrass genome structure and organization</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e33892</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0033892" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0033892</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20survey%20of%20switchgrass%20genome%20structure%20and%20organization&amp;author=M.K.%20Sharma&amp;author=R.%20Sharma&amp;author=P.%20Cao&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e33892" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0033892" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0033892" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0033892"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22511929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Declerk</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashover</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, lengthvariation, transposon associations and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20lengthvariation%2C%20transposon%20associations%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20Declerk&amp;author=A.%20Lukashover&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hui</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yong-fang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hai-quan</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xian-min</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Assesment of genetic relationship of foxtail millet with its wild ancestor and close relatives by ISSR markers</div>, <div class="source">J. Int. Agri.</div> , <div class="year">2012</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">556</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assesment%20of%20genetic%20relationship%20of%20foxtail%20millet%20with%20its%20wild%20ancestor%20and%20close%20relatives%20by%20ISSR%20markers&amp;author=L.%20Wei&amp;author=Z.%20Hui&amp;author=W.%20Yong-fang&amp;author=L.%20Hai-quan&amp;author=D.%20Xian-min&amp;publication_year=2012&amp;journal=J.%20Int.%20Agri.&amp;volume=11&amp;pages=556-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S2095-3119(12)60042-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS2095-3119(12)60042-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS2095-3119(12)60042-2"> </span></div> </div></div></div></div></div><div content-id="DST002C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirano</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Naito</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fukunaga</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">K.N.</div></div>,  <div class="name"><div class="surname">Ohsawa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kawase</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic structure of landraces in foxtail millet (Setaria italica (L.) P. Beauv.) revealed with transposon display and interpretation to crop evolution of foxtail millet</div>, <div class="source">Genome</div> , <div class="year">2011</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20of%20landraces%20in%20foxtail%20millet%20(Setaria%20italica%20(L.)%20P.%20Beauv.)%20revealed%20with%20transposon%20display%20and%20interpretation%20to%20crop%20evolution%20of%20foxtail%20millet&amp;author=R.%20Hirano&amp;author=K.%20Naito&amp;author=K.%20Fukunaga&amp;author=K.N.%20Watanabe&amp;author=R.%20Ohsawa&amp;author=M.%20Kawase&amp;publication_year=2011&amp;journal=Genome&amp;volume=54&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Dupal</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Kollier</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Drayton</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Forster</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Development and characterization of simple sequence repeat (SSR) markers for perennial ryegrass (<em>Lolium perenne</em>. L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">405</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20simple%20sequence%20repeat%20(SSR)%20markers%20for%20perennial%20ryegrass%20(Lolium%20perenne.%20L.)&amp;author=E.S.%20Jones&amp;author=M.P.%20Dupal&amp;author=R.%20Kollier&amp;author=M.C.%20Drayton&amp;author=J.W.%20Forster&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=405-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051661"> </span></div> </div></div></div></div></div><div content-id="DST002C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.Y.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.T.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Ku</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">Transferability of rice SSR markers to bamboo</div>, <div class="source">Euphytica</div> , <div class="year">2010</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">23</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%20of%20rice%20SSR%20markers%20to%20bamboo&amp;author=S.Y.%20Chen&amp;author=Y.T.%20Lin&amp;author=C.W.%20Lin&amp;author=W.Y.%20Chen&amp;author=C.H.%20Yang&amp;author=H.M.%20Ku&amp;publication_year=2010&amp;journal=Euphytica&amp;volume=175&amp;pages=23-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-010-0159-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-010-0159-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-010-0159-2"> </span></div> </div></div></div></div></div><div content-id="DST002C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gautami</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ravi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Narasu</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Novel set of groundnut SSRs for genetic diversity and inter-specific transferability</div>, <div class="source">Int. J. Integr. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">100</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DST002C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.M.</div></div>,  <div class="name"><div class="surname">Beales</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Comparative genetic maps of foxtail millet (<em>Setaria italica</em>) and rice (<em>Oryza sativa</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">63</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20maps%20of%20foxtail%20millet%20(Setaria%20italica)%20and%20rice%20(Oryza%20sativa)&amp;author=K.M.%20Devos&amp;author=Z.M.%20Wang&amp;author=J.%20Beales&amp;author=T.%20Sasaki&amp;author=M.D.%20Gale&amp;publication_year=1998&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=96&amp;pages=63-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050709"> </span></div> </div></div></div></div></div><div content-id="DST002C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Updating the crop circle</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">155</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Updating%20the%20crop%20circle&amp;author=K.M.%20Devos&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=8&amp;pages=155-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2005.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2005.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2005.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15752995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST002C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST002C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Srinivasachary</div> <div class="given-names">D.M.M.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Comparative analyses reveal high levels of conserved colinearity between the finger millet and rice genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">489</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analyses%20reveal%20high%20levels%20of%20conserved%20colinearity%20between%20the%20finger%20millet%20and%20rice%20genomes&amp;author=D.M.M.%20Srinivasachary&amp;author=M.D.%20Gale&amp;author=K.M.%20Devos&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=489-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0582-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0582-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0582-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17619853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=334072&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp_Table2.pdf?Expires=2147483647&amp;Signature=lqg2wzvSszHhSoc5dSMJ4Zrq5qsYz~goKFRsigGYxNdm~buvIrbLBH1mC-2CkVKWYsxAGPUKLl-DaBcSqXFNWG3YnhEgLkDlHSdL18zPCMUT7iifi0hg-yTZDTEQqGMcJNYiucezhp5Rs3XZVVU622RrwEcQ3pInieiJenNCm68HEMPHzNI4crvrfFidn9q7swAh9vo~AtibA1nigFLEFiGDVGlY4F3jmwDldmhs2cVYnKtoSlvpI60YbSCodjZKyco~QPWlCDPZLWjyhk3KfnBC71stF1dFM3JqSueVhWYZMKwUL-nkxSDBp~hDFmSl-eV~kXzzsaRdydbwLbQsEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp_Table1.doc?Expires=2147483647&amp;Signature=Pf1DTRiCOj2HkpLZIldMpqJQ3KULl0o19DB~JRwZTGqoA3eOkawbU14Wd4TpvHoViZOBcOXj3E~aRIdD6Pmbab0xbCvCDFA2-MS2zEb9VOrDQabYgy-dC0QGFLX-k0ZJY2wmbSwho689nhA1JyeilTkxcxOV87rKafaV9pbv1UqnqWzOCXeLdwEiMAk-eJFBxzSjtJBBQvMj8w56aWQOqI4OXSUCHaJY9vo~db~HPiAyJQSOCFJUGwEINJ7Z-yTdItUrt3bvfvxPz3zH6FvJhF5AEok2kNDev2b6g-aPYax2RJvvSmmB8hFC2Ty86yTrkSYJxl061RbkQ3IA6Qsxbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp_FigureS3.jpg?Expires=2147483647&amp;Signature=aXX95Vh~e4TKXHnjOhZNNvjPO5AO4ePIDBEOQ9hFzMMWX3Y348PbYALk5k0oQdGIDRL~Fdoiar8Mdp~WQzrkjoWuea5UNiL5Rng-BgMiHhTPCW8hW4VdrCGRYcIQvJS~rnVe2MKK91GmzH-2v53n7EJ2LW0wOUCEm7SKXSEgy6AcWbm1NHcWgVhYGdaNH~zJQrTJpbCpIqcX5L-B6ZO33owj4MRKA0kb79d7AHIgPLUy8t51k04kowslAzXd2H73nOFeNJxwQhYF0ZIZIWjAfTpZzwCeUbR1oTeeV6WeDSf4mFa8gdSbJsmQ5MgqBLFybwxFS31837r1QMZ9bTZp-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp_FigureS2.jpg?Expires=2147483647&amp;Signature=uOD245UYg1zbMGTppNXouemKKplf7K5kxyiBHwzYOHbg1b0nb8A-WU-rvo4ElhdNmaS6IhURhaNGLVHHDurSWSqh9J5q~FZvNDFdNJCcRwbB4-KhLZ~YYLZYhVq3J5EqNoqhHEfwKPEF7O3Im0dYrTpQEfMumb2OZtmuWAJ6Djou3kIMhrL0G0y4C8CdUjNfruJgcIZ8Vtk5hVw2dlh1ir1fjV44R-5s4Ku-SJyghrYVstnn5XKkWJ-LamHGxGKEOxrm1P9bZT-G3FsruuQsbeWQeVRfNtt3-8H25hLiIYQwc7X1WU21cW36ZGSiBLMgPKhZp5bh-bfDkTeRlVqk2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - jpg file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp_FigureS1.png?Expires=2147483647&amp;Signature=Sus2HZDQq4UE9V7HlFqZruCh-IgBBYwosO4s3xzqak7qtmjV6Zi43J4B5z-3j7dplkZhBZNiJ-SPVW0kiFK9Dm4ZVdqgNFs-IPxfqtMyU8Wo0w5R7yQU6fqsjb~7B4np1DRU7A1il98mR2uGsHAEnr2LcoLq9O7WOamVTs21CC0JYuKwFoYPyrdgGeNoY8Hep~0lp05hfRiGgSGcKagGIuuk0MfGPJwY5ndOfR4HzPNX171J6byTfkDl8avsKz~e2jT5In0ljazavnVr3xr2QdEMXTDpUhzDixVor5xjjTOdutQzuTP1TfSVZ3vA0RO9nP9ljGXF6LrJhIpEtcUY~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst002/2/dst002-dst002supp.doc?Expires=2147483647&amp;Signature=nBtNAF9GjUPf0F3ejtrDIKCq0oFzqMIKFVG7xQ2gA7QtaM5AkCv4tXUHZFFPLA1-CsW4gyqQH86Iavbq178tqB5Xpao9mrYH7r2~zyYREbEHhNJsF7jS8XWH7tTZckUoYeFf3YoPU8oG81wJnV~UB7iiKlzBOKLITfjr7gyttdAyy2wykKhU1LSQQM4UQroweDRWglZw7oE4rl2M20AtKN4X7i9HPxS-XC-ciYlHurUlIeuLtzLeUpa7KdMIOn0EOUMFIHhn33NJvyabe6b2zM1Bm~Pe7JcfoJcAGciUOYhTNxao3f1kZrES5Tq9L3UcpROrMAwalbDvozXo8EuTYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">549</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">62</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="334072"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f2%2f197%2f334072">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000318584600008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (62)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F2%2F197%2F334072" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/334072'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Development and Use of Microsatellite Markers for Large-Scale Genotyping Applications in Foxtail Millet [Setaria italica (L.)]" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f197%2f334072">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

