

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/4/347/451164","@type":"ScholarlyArticle","name":"Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA","datePublished":"2012-06-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2012-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss017","keywords":["picosecond-resolved FRET","HLA-DQB1 typing","insulin-dependent diabetes mellitus","time-correlated single-photon counting"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/4/347/451164","author":[{"name":"Nardo, Luca","affiliation":"1  Department of Science and High Tech ,  C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria ,  Via Valleggio 11, Como 22100 ,  Italy    2  C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria ,  Via Valleggio 11, Como 22100 ,  Italy","@type":"Person"},{"name":"Tosi, Giovanna","affiliation":"3  Department of Surgical and Morphologic Sciences ,  University of Insubria ,  Viale Borri 57, Varese 21100 ,  Italy","@type":"Person"},{"name":"Bondani, Maria","affiliation":"4  Institute for Photonics and Nanotechnology ,  C.N.R. (Consiglio Nazionale delle Ricerche) ,  Via Valleggio 11, Como 22100 ,  Italy","@type":"Person"},{"name":"Accolla, Roberto S.","affiliation":"3  Department of Surgical and Morphologic Sciences ,  University of Insubria ,  Viale Borri 57, Varese 21100 ,  Italy","@type":"Person"},{"name":"Andreoni, Alessandra","affiliation":"1  Department of Science and High Tech ,  C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria ,  Via Valleggio 11, Como 22100 ,  Italy    2  C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria ,  Via Valleggio 11, Como 22100 ,  Italy","@type":"Person"}],"description":"Abstract.  This work concerns the identification of the alleles of the polymorphic DQB1 gene of the human leukocyte antigen system, conferring susceptibility to","pageStart":"347","pageEnd":"355","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA" />
    <meta property="og:description" content="Abstract.  This work concerns the identification of the alleles of the polymorphic DQB1 gene of the human leukocyte antigen system, conferring susceptibility to" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/4/347/451164" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-06-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nardo, Luca" /><meta name="citation_author_institution" content="Department of Science and High Tech, C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria, Via Valleggio 11, Como 22100, Italy" /><meta name="citation_author_institution" content="C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria, Via Valleggio 11, Como 22100, Italy" /><meta name="citation_author" content="Tosi, Giovanna" /><meta name="citation_author_institution" content="Department of Surgical and Morphologic Sciences, University of Insubria, Viale Borri 57, Varese 21100, Italy" /><meta name="citation_author" content="Bondani, Maria" /><meta name="citation_author_institution" content="Institute for Photonics and Nanotechnology, C.N.R. (Consiglio Nazionale delle Ricerche), Via Valleggio 11, Como 22100, Italy" /><meta name="citation_author" content="Accolla, Roberto S." /><meta name="citation_author_institution" content="Department of Surgical and Morphologic Sciences, University of Insubria, Viale Borri 57, Varese 21100, Italy" /><meta name="citation_author" content="Andreoni, Alessandra" /><meta name="citation_author_institution" content="Department of Science and High Tech, C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria, Via Valleggio 11, Como 22100, Italy" /><meta name="citation_author_institution" content="C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria, Via Valleggio 11, Como 22100, Italy" /><meta name="citation_title" content="Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA" /><meta name="citation_firstpage" content="347" /><meta name="citation_lastpage" content="355" /><meta name="citation_doi" content="10.1093/dnares/dss017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2012/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=HLA and type I diabetes; Immunol. Today;  citation_year=1990;  citation_volume=11;  citation_pages=314-5; " /><meta name="citation_reference" content="citation_title=Assessment of the DQB1-DQA1 complete genotype allows best prediction for IDDM; Diabetes Care;  citation_year=1994;  citation_volume=17;  citation_pages=1045-9; " /><meta name="citation_reference" content="citation_title=Human leukocyte antigen class II susceptibility conferring alleles among non-insulin dependent diabetes mellitus patients; J. Coll. Phys. Surg. Pak.;  citation_year=2011;  citation_volume=21;  citation_pages=26-9; " /><meta name="citation_reference" content="citation_title=Sensitive mutation detection in heterogeneous cancer specimens by massively parallel picoliter reactor sequencing; Nat. Med.;  citation_year=2006;  citation_volume=12;  citation_pages=852-5; " /><meta name="citation_reference" content="citation_title=HLA-DQB gene contributes to susceptibility and resistance to insulin dependent diabetes mellitus; Nature;  citation_year=1987;  citation_volume=329;  citation_pages=599-604; " /><meta name="citation_reference" content="citation_title=HLA-DQB1 typing of north-east Italian IDDM patients using amplified DNA, oligonucleotide probes, and a rapid DNA-enzyme immunoassay (DEIA); Mol. Immunol.;  citation_year=1993;  citation_volume=30;  citation_pages=69-76; " /><meta name="citation_reference" content="citation_title=The complex interplay of the DQB1 and DQB2 loci in the generation of the susceptibility and protective phenotype for insulin-dependent diabetes mellitus; Mol. Immunol.;  citation_year=1994;  citation_volume=31;  citation_pages=429-37; " /><meta name="citation_reference" content="citation_title=Multicolor molecular beacons for allele discrimination; Nat. Biotechnol.;  citation_year=1998;  citation_volume=16;  citation_pages=49-53; " /><meta name="citation_reference" content="citation_title=SNP detection using peptide nucleic acid probes and conjugated polymers: applications in neurodegenerative disease identification; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=34-9; " /><meta name="citation_reference" content="citation_title=Improved single nucleotide polymorphism detection using conjugated polymer/surfactant system and peptide nucleic acid; Biosens Bioelectron;  citation_year=2008;  citation_volume=23;  citation_pages=1466-72; " /><meta name="citation_reference" content="citation_title=Time-resolved FRET method for typing polymorphic alleles of the human leukocyte antigen system by using a single DNA probe; Photochem. Photobiol. Sci.;  citation_year=2009;  citation_volume=8;  citation_pages=1202-6; " /><meta name="citation_reference" content="citation_title=Feasibility of single nucleotide polymorphism genotyping with a single-probe by time-resolved Förster resonance energy transfer; Mol. Cell. Probes;  citation_year=2009;  citation_volume=23;  citation_pages=119-21; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/4/347/1232928/dss017.pdf" /><meta name="description" content="Abstract.  This work concerns the identification of the alleles of the polymorphic DQB1 gene of the human leukocyte antigen system, conferring susceptibility to" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/4/347/451164" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/4/347/451164" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/4/347/451164");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss017';
    </script>
<title>
	Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./451164" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb0tPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYXJkbywgTHVjYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgU2NpZW5jZSBhbmQgSGlnaCBUZWNoLCBDLk4uSS5TLk0uIChDb25zb3J6aW8gTmF6aW9uYWxlIEludGVydW5pdmVyc2l0YXJpbyBwZXIgbGUgU2NpZW56ZSBGaXNpY2hlIGRlbGxhIE1hdGVyaWEpLCBVbml2ZXJzaXR5IG9mIEluc3VicmlhLCBWaWEgVmFsbGVnZ2lvIDExLCBDb21vIDIyMTAwLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkMuTi5JLlMuTS4gKENvbnNvcnppbyBOYXppb25hbGUgSW50ZXJ1bml2ZXJzaXRhcmlvIHBlciBsZSBTY2llbnplIEZpc2ljaGUgZGVsbGEgTWF0ZXJpYSksIFVuaXZlcnNpdHkgb2YgSW5zdWJyaWEsIFZpYSBWYWxsZWdnaW8gMTEsIENvbW8gMjIxMDAsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9zaSwgR2lvdmFubmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFN1cmdpY2FsIGFuZCBNb3JwaG9sb2dpYyBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBJbnN1YnJpYSwgVmlhbGUgQm9ycmkgNTcsIFZhcmVzZSAyMTEwMCwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb25kYW5pLCBNYXJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBmb3IgUGhvdG9uaWNzIGFuZCBOYW5vdGVjaG5vbG9neSwgQy5OLlIuIChDb25zaWdsaW8gTmF6aW9uYWxlIGRlbGxlIFJpY2VyY2hlKSwgVmlhIFZhbGxlZ2dpbyAxMSwgQ29tbyAyMjEwMCwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBY2NvbGxhLCBSb2JlcnRvIFMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBTdXJnaWNhbCBhbmQgTW9ycGhvbG9naWMgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgSW5zdWJyaWEsIFZpYWxlIEJvcnJpIDU3LCBWYXJlc2UgMjExMDAsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQW5kcmVvbmksIEFsZXNzYW5kcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFNjaWVuY2UgYW5kIEhpZ2ggVGVjaCwgQy5OLkkuUy5NLiAoQ29uc29yemlvIE5hemlvbmFsZSBJbnRlcnVuaXZlcnNpdGFyaW8gcGVyIGxlIFNjaWVuemUgRmlzaWNoZSBkZWxsYSBNYXRlcmlhKSwgVW5pdmVyc2l0eSBvZiBJbnN1YnJpYSwgVmlhIFZhbGxlZ2dpbyAxMSwgQ29tbyAyMjEwMCwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDLk4uSS5TLk0uIChDb25zb3J6aW8gTmF6aW9uYWxlIEludGVydW5pdmVyc2l0YXJpbyBwZXIgbGUgU2NpZW56ZSBGaXNpY2hlIGRlbGxhIE1hdGVyaWEpLCBVbml2ZXJzaXR5IG9mIEluc3VicmlhLCBWaWEgVmFsbGVnZ2lvIDExLCBDb21vIDIyMTAwLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVHlwaW5nIG9mIGEgUG9seW1vcnBoaWMgSHVtYW4gR2VuZSBDb25mZXJyaW5nIFN1c2NlcHRpYmlsaXR5IHRvIEluc3VsaW4tRGVwZW5kZW50IERpYWJldGVzIE1lbGxpdHVzIGJ5IFBpY29zZWNvbmQtUmVzb2x2ZWQgRlJFVCBvbiBOb24tUHVyaWZpZWQvTm9uLUFtcGxpZmllZCBHZW5vbWljIEROQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITEEgYW5kIHR5cGUgSSBkaWFiZXRlczsgSW1tdW5vbC4gVG9kYXk7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zMTQtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiB0aGUgRFFCMS1EUUExIGNvbXBsZXRlIGdlbm90eXBlIGFsbG93cyBiZXN0IHByZWRpY3Rpb24gZm9yIElERE07IERpYWJldGVzIENhcmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMDQ1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIGxldWtvY3l0ZSBhbnRpZ2VuIGNsYXNzIElJIHN1c2NlcHRpYmlsaXR5IGNvbmZlcnJpbmcgYWxsZWxlcyBhbW9uZyBub24taW5zdWxpbiBkZXBlbmRlbnQgZGlhYmV0ZXMgbWVsbGl0dXMgcGF0aWVudHM7IEouIENvbGwuIFBoeXMuIFN1cmcuIFBhay47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZW5zaXRpdmUgbXV0YXRpb24gZGV0ZWN0aW9uIGluIGhldGVyb2dlbmVvdXMgY2FuY2VyIHNwZWNpbWVucyBieSBtYXNzaXZlbHkgcGFyYWxsZWwgcGljb2xpdGVyIHJlYWN0b3Igc2VxdWVuY2luZzsgTmF0LiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODUyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhMQS1EUUIgZ2VuZSBjb250cmlidXRlcyB0byBzdXNjZXB0aWJpbGl0eSBhbmQgcmVzaXN0YW5jZSB0byBpbnN1bGluIGRlcGVuZGVudCBkaWFiZXRlcyBtZWxsaXR1czsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTU5OS02MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhMQS1EUUIxIHR5cGluZyBvZiBub3J0aC1lYXN0IEl0YWxpYW4gSURETSBwYXRpZW50cyB1c2luZyBhbXBsaWZpZWQgRE5BLCBvbGlnb251Y2xlb3RpZGUgcHJvYmVzLCBhbmQgYSByYXBpZCBETkEtZW56eW1lIGltbXVub2Fzc2F5IChERUlBKTsgTW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTY5LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxleCBpbnRlcnBsYXkgb2YgdGhlIERRQjEgYW5kIERRQjIgbG9jaSBpbiB0aGUgZ2VuZXJhdGlvbiBvZiB0aGUgc3VzY2VwdGliaWxpdHkgYW5kIHByb3RlY3RpdmUgcGhlbm90eXBlIGZvciBpbnN1bGluLWRlcGVuZGVudCBkaWFiZXRlcyBtZWxsaXR1czsgTW9sLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTQyOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGljb2xvciBtb2xlY3VsYXIgYmVhY29ucyBmb3IgYWxsZWxlIGRpc2NyaW1pbmF0aW9uOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NDktNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkZXRlY3Rpb24gdXNpbmcgcGVwdGlkZSBudWNsZWljIGFjaWQgcHJvYmVzIGFuZCBjb25qdWdhdGVkIHBvbHltZXJzOiBhcHBsaWNhdGlvbnMgaW4gbmV1cm9kZWdlbmVyYXRpdmUgZGlzZWFzZSBpZGVudGlmaWNhdGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0zNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGV0ZWN0aW9uIHVzaW5nIGNvbmp1Z2F0ZWQgcG9seW1lci9zdXJmYWN0YW50IHN5c3RlbSBhbmQgcGVwdGlkZSBudWNsZWljIGFjaWQ7IEJpb3NlbnMgQmlvZWxlY3Ryb247ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNDY2LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW1lLXJlc29sdmVkIEZSRVQgbWV0aG9kIGZvciB0eXBpbmcgcG9seW1vcnBoaWMgYWxsZWxlcyBvZiB0aGUgaHVtYW4gbGV1a29jeXRlIGFudGlnZW4gc3lzdGVtIGJ5IHVzaW5nIGEgc2luZ2xlIEROQSBwcm9iZTsgUGhvdG9jaGVtLiBQaG90b2Jpb2wuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEyMDItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmVhc2liaWxpdHkgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGdlbm90eXBpbmcgd2l0aCBhIHNpbmdsZS1wcm9iZSBieSB0aW1lLXJlc29sdmVkIEbDtnJzdGVyIHJlc29uYW5jZSBlbmVyZ3kgdHJhbnNmZXI7IE1vbC4gQ2VsbC4gUHJvYmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTE5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvNC8zNDcvMTIzMjkyOC9kc3MwMTcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoaXMgd29yayBjb25jZXJucyB0aGUgaWRlbnRpZmljYXRpb24gb2YgdGhlIGFsbGVsZXMgb2YgdGhlIHBvbHltb3JwaGljIERRQjEgZ2VuZSBvZiB0aGUgaHVtYW4gbGV1a29jeXRlIGFudGlnZW4gc3lzdGVtLCBjb25mZXJyaW5nIHN1c2NlcHRpYmlsaXR5IHRvIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NTExNjQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkkHF9uy58YPOq/Y0vzLcuEaKPTo0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f347%2f451164" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f347%2f451164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17634" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&amp;Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/4">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6771629">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771629">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771631">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771631">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771638">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771656">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771656">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771667">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771667">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771674">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771674">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771676">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771676">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6771678">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6771678">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/4/335/451025">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luca Nardo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Science and High Tech</div>, <div class="insititution">C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria</div>, <div class="addr-line">Via Valleggio 11, Como 22100</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria</div>, <div class="addr-line">Via Valleggio 11, Como 22100</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Luca+Nardo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nardo L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nardo L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Luca Nardo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Giovanna Tosi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Surgical and Morphologic Sciences</div>, <div class="insititution">University of Insubria</div>, <div class="addr-line">Viale Borri 57, Varese 21100</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Giovanna+Tosi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tosi G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tosi G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Giovanna Tosi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Bondani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Institute for Photonics and Nanotechnology</div>, <div class="insititution">C.N.R. (Consiglio Nazionale delle Ricerche)</div>, <div class="addr-line">Via Valleggio 11, Como 22100</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Bondani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bondani M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bondani M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maria Bondani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roberto S. Accolla</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Surgical and Morphologic Sciences</div>, <div class="insititution">University of Insubria</div>, <div class="addr-line">Viale Borri 57, Varese 21100</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roberto+S.+Accolla">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Accolla R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Accolla R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Roberto S. Accolla</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alessandra Andreoni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Science and High Tech</div>, <div class="insititution">C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria</div>, <div class="addr-line">Via Valleggio 11, Como 22100</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">C.N.I.S.M. (Consorzio Nazionale Interuniversitario per le Scienze Fisiche della Materia), University of Insubria</div>, <div class="addr-line">Via Valleggio 11, Como 22100</div>, <div class="country">Italy</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +39-(0)-31-2386210. Fax. +39-(0)-31-2386119. Email: <a href="mailto:alessandra.andreoni@uninsubria.it" target="_blank">alessandra.andreoni@uninsubria.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alessandra+Andreoni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Andreoni A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Andreoni A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alessandra Andreoni</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 347–355, <a href='https://doi.org/10.1093/dnares/dss017'>https://doi.org/10.1093/dnares/dss017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 June 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 December 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 May 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="451164" href="/dnaresearch/article-pdf/19/4/347/1232928/dss017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Luca Nardo, Giovanna Tosi, Maria Bondani, Roberto S. Accolla, Alessandra Andreoni;  Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA, <em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 347–355, <a href='https://doi.org/10.1093/dnares/dss017'>https://doi.org/10.1093/dnares/dss017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/451164?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/451164?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Typing%20of%20a%20Polymorphic%20Human%20Gene%20Conferring%20Susceptibility%20to%20Insulin-Dependent%20Diabetes%20Mellitus%20by%20Picosecond-Resolved%20FRET%20on%20Non-Purified%2FNon-Amplified%20Genomic%20DNA&amp;publicationDate=2012-06-28&amp;volumeNum=19&amp;issueNum=4&amp;author=Nardo%2C%20Luca%3B%20Tosi%2C%20Giovanna&amp;startPage=347&amp;endPage=355&amp;contentId=10.1093%2Fdnares%2Fdss017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f347%2f451164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6771629 id="6771629" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>This work concerns the identification of the alleles of the polymorphic DQB1 gene of the human leukocyte antigen system, conferring susceptibility to the development of insulin-dependent diabetes mellitus (IDDM) in non-PCR amplified DNA samples and, more importantly, in crude cell extracts. Our method is based on the time-resolved analysis of a Förster energy-transfer mechanism that occurs in a dual-labelled fluorescent probe specific for the IDDM-associated DQB1-0201 allele. Such an oligonucleotide probe is labelled, at the two ends, by a pair of chromophores that operate as donor and acceptor in a Förster resonant energy transfer. The donor fluorescence is quenched with an efficiency that is strongly dependent on the donor-to-acceptor distance, hence on the configuration of the probe after hybridization with the various DQB1 alleles. By time-correlated single-photon counting, performed with an excitation/detection system endowed with 30-ps resolution, we measure the time-resolved fluorescence decay of the donor and discriminate, by means of the decay–time value, the DNA bearing the ‘susceptible’ allele from the DNAs bearing any other sequence in the same region of the DQB1 gene. We could also distinguish the presence of the DQB1-0201 allele in a homozygous versus a heterozygous condition.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;picosecond-resolved FRET&quot;">picosecond-resolved FRET</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;HLA-DQB1 typing&quot;">HLA-DQB1 typing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;insulin-dependent diabetes mellitus&quot;">insulin-dependent diabetes mellitus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;time-correlated single-photon counting&quot;">time-correlated single-photon counting</a></div>                            <h2 scrollto-destination=6771631 id="6771631" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The human leukocyte antigen (HLA) system encodes highly polymorphic molecules which are fundamental for the triggering of the immune response. Moreover, it is known that the expression of certain HLA molecules is associated with the susceptibility to many autoimmune diseases, including insulin-dependent diabetes mellitus (IDDM).<sup><a reveal-id="DSS017C1" data-open="DSS017C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS017C2" data-open="DSS017C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Distinguishing the subjects with susceptibility genes, before the onset of the disease, calls for the development of rapid high-throughput HLA typing technologies. On the one hand, such an ambitious aim seems to contrast with the increasing number of susceptibility conferring alleles that are being discovered in patients affected by specific pathologies.<sup><a reveal-id="DSS017C3" data-open="DSS017C3" class="link link-ref link-reveal xref-bibr">3</a></sup> On the other hand, it is encouraging that a massively parallel sequencing technology has recently allowed mutation detection in picolitres of cancer specimens.<sup><a reveal-id="DSS017C4" data-open="DSS017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> For large-scale HLA analysis, we decided to set a method allowing the typing of non-PCR amplified genomic DNA, purified from leukocytes, and non-purified DNA present in cell lysates.</p><p>We adopted a DNA probe, which is depicted in Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>, matching the sequence of one of the most common alleles conferring susceptibility to IDDM in the Caucasian population and, by making a novel use of Förster resonant energy transfer (FRET), we found a method for detecting sequence variations of even a single nucleotide in DNA template used as target (see Fig. <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>). Markers of susceptibility to IDDM have been recognized in the HLA-II region and, in particular, in HLA-II DQA1 and DQB1 loci, coding for the α and β chains of DQ heterodimer, respectively.<sup><a reveal-id="DSS017C5" data-open="DSS017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Extensive molecular studies of HLA-DQ genotype in patients with IDDM led to correlate the susceptibility with the polymorphism of a single amino acid in both DQα and DQβ chains.<sup><a reveal-id="DSS017C6" data-open="DSS017C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS017C7" data-open="DSS017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In particular, the combined presence of an arginine residue in position 52 of the DQα chain and of a neutral residue (serine, alanine or valine) in position 57 of the DQβ chain is strongly associated with diabetes onset. On the contrary, the presence of aspartic acid at position 57 of DQβ confers resistance to IDDM.<sup><a reveal-id="DSS017C2" data-open="DSS017C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p>                        <a id="6771635" scrollto-destination="6771635"></a>
<div id="DSS017F1" data-id="DSS017F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01701.jpeg?Expires=2147483647&Signature=i8~IaxZeDZPqFrf57VLx86BWBJi1UtiwpeqVJn9ouP5C-NMM6gO9N8yl4~ODLxAIPMgAip~ZGeRjuTceKuLFTaOlmF6d-Wnoh2Z4vQ3BPLlitygvAVwVRtzve1sFuHORxMCVEWAze8g2VwbkPMjVlCujhvWjAJeYHneRboPmKHhg3UZ4qPF7~2JwxikK8KWnjTQvKDwYcrO6hbwRJzcqnDNcQ11ZlJGB6u2ts98IA8K4a1qjDDnlxXgAJcGPJt4FrlVKo71TmFsu5z-mxwG~v9fxWyvcN8aJaHqevEDiGDWICKtOGzWHQVF-k9aXDlNOVNHmSrGOo8b0rRbZoE2Hhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FRET probe matching the DQB1-0201 sequence between codons 51 and 58 (22 bases) and carrying the TAMRA donor and the BHQ2 quencher, whose structures are shown in the bottom." path-from-xml="dss01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771635/dss01701.jpeg" path-from-xml="dss01701.jpeg" target="_blank">View large</a><a section="6771635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01701.jpeg?Expires=2147483647&Signature=pc2zouaHwSz9vkXw4J2xsJR3mrDZ~A0fXaLA9hLNP31KDrZug~bO4uWqUzXHskmnCUEcikgq-7aE0XfTQsXAjjDCknRKiegBpzlGcMF6k3t5OP7rK0fd~hADGKrpdRovQ3GMWuP5nXIQnnfIHKutSxSkYXFLq7ko3ePu2c-zWROxtsnJmr~i9da1~VVQFtWd6-FdzJxdy8GHvPnJOEzWY78uSWa-vyiGDeDsAkkR7ilN0YHX38mSV4Qp7dbG4J7KYw8XLCOEQl2TN0KezkozISQzA1y8RPnhYS7zaG6TkHCyMDleJPmVn2nzOLZgFgKcuR4fkIHbEDxfGOIcRMgx3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771635&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FRET probe matching the DQB1-0201 sequence between codons 51 and 58 (22 bases) and carrying the TAMRA donor and the BHQ2 quencher, whose structures are shown in the bottom.</p></div></div><div content-id="DSS017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01701.jpeg?Expires=2147483647&Signature=i8~IaxZeDZPqFrf57VLx86BWBJi1UtiwpeqVJn9ouP5C-NMM6gO9N8yl4~ODLxAIPMgAip~ZGeRjuTceKuLFTaOlmF6d-Wnoh2Z4vQ3BPLlitygvAVwVRtzve1sFuHORxMCVEWAze8g2VwbkPMjVlCujhvWjAJeYHneRboPmKHhg3UZ4qPF7~2JwxikK8KWnjTQvKDwYcrO6hbwRJzcqnDNcQ11ZlJGB6u2ts98IA8K4a1qjDDnlxXgAJcGPJt4FrlVKo71TmFsu5z-mxwG~v9fxWyvcN8aJaHqevEDiGDWICKtOGzWHQVF-k9aXDlNOVNHmSrGOo8b0rRbZoE2Hhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FRET probe matching the DQB1-0201 sequence between codons 51 and 58 (22 bases) and carrying the TAMRA donor and the BHQ2 quencher, whose structures are shown in the bottom." path-from-xml="dss01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771635/dss01701.jpeg" path-from-xml="dss01701.jpeg" target="_blank">View large</a><a section="6771635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01701.jpeg?Expires=2147483647&Signature=pc2zouaHwSz9vkXw4J2xsJR3mrDZ~A0fXaLA9hLNP31KDrZug~bO4uWqUzXHskmnCUEcikgq-7aE0XfTQsXAjjDCknRKiegBpzlGcMF6k3t5OP7rK0fd~hADGKrpdRovQ3GMWuP5nXIQnnfIHKutSxSkYXFLq7ko3ePu2c-zWROxtsnJmr~i9da1~VVQFtWd6-FdzJxdy8GHvPnJOEzWY78uSWa-vyiGDeDsAkkR7ilN0YHX38mSV4Qp7dbG4J7KYw8XLCOEQl2TN0KezkozISQzA1y8RPnhYS7zaG6TkHCyMDleJPmVn2nzOLZgFgKcuR4fkIHbEDxfGOIcRMgx3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771635&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FRET probe matching the DQB1-0201 sequence between codons 51 and 58 (22 bases) and carrying the TAMRA donor and the BHQ2 quencher, whose structures are shown in the bottom.</p></div></div>                        <a id="6771636" scrollto-destination="6771636"></a>
<div id="DSS017F2" data-id="DSS017F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01702.jpeg?Expires=2147483647&Signature=3lvvMXe84t7p4uV-GoaI~ZmsDL4OroaiE3f43~0MStwe3GKkEAf-xny0EsaJiwEbxk5nWb8Vy28P-r6ZBdPQa-l5pY-tQuE6mP2FBrDPv3ZwZAEi6gzC2O91NGarg5U20~1K1sUtaMCq5w8o9O9K61UUiA07CdgMFhC0eLtz3Zh~MAtO~incHtEdr-fADtHqhkuOO2DgZtEkfc8KP6ITj~D~eOSdZ2TU4AJ5G7QmE4ab1TlnncU3q3R0hkg24s2oijziVDW4li3TKOK5r9dYMFg8CerM~HpmnAIqpobWtuQDzWHumn9~RRsBFMLZ2MiFgf0p~HNd6bO3M7LURUY0Zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequences (antisense) of the region comprising codons 51–60 of the eight relevant DQB1 allelic variants (listed on the left). Nucleotide changes with respect to DQB1-0201 are marked. The regions shaded in grey are those not covered by the oligonucleotide used as a probe (see Fig. 1)." path-from-xml="dss01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771636/dss01702.jpeg" path-from-xml="dss01702.jpeg" target="_blank">View large</a><a section="6771636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01702.jpeg?Expires=2147483647&Signature=KgCRgt~wSXdMz7N8iLwqXosQ53oSKjuncPF~XpawrlUeJNJgugZESHBnKUzlN6taNx5B00oqANf-jyHY1JrupejesjT8kPmOAZ4h8ezuSqIZ~Sje0bv6p5VlSHrKC2ItGIMCQ-kPsvJF1KrJtDFnhSGqLoDjJenMFgCWt35Fl4Q229li~yydPKg~Y2EUzA28Dzdp6XCfWjRN3AKsgS-KGulaEsN4PHrz2KYnfc00Vp4KKsOZlq57ehVkqUpxGaRI4L5wOcB9gx984CZSQvIonEqtKo~MuvGq4wgJkaQPFsLD9eKP6E~st0qHUjQQuZr0mMGCQU0srr7Bjj-nJV~mdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771636&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequences (antisense) of the region comprising codons 51–60 of the eight relevant DQB1 allelic variants (listed on the left). Nucleotide changes with respect to DQB1-0201 are marked. The regions shaded in grey are those not covered by the oligonucleotide used as a probe (see Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>).</p></div></div><div content-id="DSS017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01702.jpeg?Expires=2147483647&Signature=3lvvMXe84t7p4uV-GoaI~ZmsDL4OroaiE3f43~0MStwe3GKkEAf-xny0EsaJiwEbxk5nWb8Vy28P-r6ZBdPQa-l5pY-tQuE6mP2FBrDPv3ZwZAEi6gzC2O91NGarg5U20~1K1sUtaMCq5w8o9O9K61UUiA07CdgMFhC0eLtz3Zh~MAtO~incHtEdr-fADtHqhkuOO2DgZtEkfc8KP6ITj~D~eOSdZ2TU4AJ5G7QmE4ab1TlnncU3q3R0hkg24s2oijziVDW4li3TKOK5r9dYMFg8CerM~HpmnAIqpobWtuQDzWHumn9~RRsBFMLZ2MiFgf0p~HNd6bO3M7LURUY0Zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequences (antisense) of the region comprising codons 51–60 of the eight relevant DQB1 allelic variants (listed on the left). Nucleotide changes with respect to DQB1-0201 are marked. The regions shaded in grey are those not covered by the oligonucleotide used as a probe (see Fig. 1)." path-from-xml="dss01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771636/dss01702.jpeg" path-from-xml="dss01702.jpeg" target="_blank">View large</a><a section="6771636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01702.jpeg?Expires=2147483647&Signature=KgCRgt~wSXdMz7N8iLwqXosQ53oSKjuncPF~XpawrlUeJNJgugZESHBnKUzlN6taNx5B00oqANf-jyHY1JrupejesjT8kPmOAZ4h8ezuSqIZ~Sje0bv6p5VlSHrKC2ItGIMCQ-kPsvJF1KrJtDFnhSGqLoDjJenMFgCWt35Fl4Q229li~yydPKg~Y2EUzA28Dzdp6XCfWjRN3AKsgS-KGulaEsN4PHrz2KYnfc00Vp4KKsOZlq57ehVkqUpxGaRI4L5wOcB9gx984CZSQvIonEqtKo~MuvGq4wgJkaQPFsLD9eKP6E~st0qHUjQQuZr0mMGCQU0srr7Bjj-nJV~mdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771636&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequences (antisense) of the region comprising codons 51–60 of the eight relevant DQB1 allelic variants (listed on the left). Nucleotide changes with respect to DQB1-0201 are marked. The regions shaded in grey are those not covered by the oligonucleotide used as a probe (see Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>).</p></div></div><p>In the present work we focussed on the typing of the DQB1 locus by hybridizing target DNAs to a single probe corresponding to DQB1-0201 ‘susceptible’ allele, which was dual labelled by a fluorescent reporter in 5′ and a non-fluorescing quencher in 3′. In such a probe, the fluorescence of the donor tetramethylrhodamine (TAMRA) is quenched via FRET by the black-hole quencher (BHQ2) depending on the conformational differences among the hybrids due to the probe/target mismatches. It is worth noting that fluorescence quenching by FRET is basic to many genotyping methods that include a step in which allele-specific hybridization products must be recognized. These methods rely on the fact that FRET removes the excitation energy deposited on the donor (D) towards the acceptor (A) with an efficiency that depends on the D-to-A distance. As a result, when D and A are in close proximity, the D fluorescence is weaker than that emitted when they are at greater distances from each other. Multiple allele-specific probes, each labelled with a different D–A pair, can be used for querying allelic sequences.<sup><a reveal-id="DSS017C8" data-open="DSS017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The sequence of each probe is complementary to a specific polymorphic sequence and probes are mixed all at once with the PCR-amplified template. The highly stringent hybridization conditions allow annealing only of the perfectly matching probe, which results in a greater D-to-A distance, thus in an enhanced fluorescence emission of D. Conversely, as the non-matching probes preserve the native coiled conformation, their fluorescence is quenched. The genotype of the template DNA is inferred by assessing which D is the fluorescing one. We exploit the fact that FRET can produce quantitatively different fluorescence responses depending on the probe–target hybrid conformation, thus making it feasible SNP genotyping by means of a single probe. Pioneering attempts in this direction have been made by Gaylord <em>et al</em>.<sup><a reveal-id="DSS017C9" data-open="DSS017C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and, later, by Al Attar <em>et al</em>.<sup><a reveal-id="DSS017C10" data-open="DSS017C10" class="link link-ref link-reveal xref-bibr">10</a></sup>. The latter group used a cationic polymer conjugated to the targets as D and monitored the fluorescence intensity of A attached to a single peptide nucleic acid probe. As the neutral peptide backbone of the probe did not bind to D, the acceptor A received the excitation energy only upon hybridization of the probe to the targets. By applying this scheme to typing of ABL polymorphism in the BCR-ABL oncogene, Al Attar <em>et al</em>. have determined significantly different values of the FRET efficiency between D and A when the probe was hybridized to each of the possible target sequences (i.e. to each of the possible mutated alleles). Thus, they demonstrated the feasibility of ABL typing by using a single probe. It is easy to realize that distinguishing among the hybrids that our probe forms with the different allelic sequences by absolute measurements of D-fluorescence intensity is a hopeless approach to the development of a typing method suitable for non-purified/non-amplified genomic DNA. Indeed, a high-throughput screening method based on FRET should work with cellular lysates, where the DNA concentration is variable from sample to sample, and utilize detectors whose response is stable against environmental stray light conditions. In our approach we quantify the FRET efficiencies of each probe/template hybrid by analysing the D fluorescence temporal decay on a very fast time scale. We measure the TAMRA-fluorescence decay time (<em>τ</em><sub>D</sub>) upon picosecond laser excitation, at a wavelength that is only absorbed by this chromophore, and detect the fluorescence by a time-correlated single-photon counting (TCSPC) apparatus, which is endowed with a temporal resolution of &lt;30 ps. Though the acquisition rate is less than one photon per excitation pulse, which ensures the applicability of our technique to the detection of even few probe–target duplexes, good determinations of the <em>τ</em><sub>D</sub> values are done in few minutes due to the megahertz repetition rate of the excitation laser. The slight differences in the distance of the BHQ2 acceptor from the TAMRA donor and the consequent differences in FRET efficiencies produce distinct <em>τ</em><sub>D</sub> values for the probe hybridized to each of the allelic sequences.</p>                            <h2 scrollto-destination=6771638 id="6771638" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6771639 id="6771639" class="section-title" data-legacy-id=s2a>2.1. Hybridization of oligonucleotides</h3>
<p>The D–A pair in the single-stranded dual-labelled oligonucleotide probe in Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>, D-5′(ACGCTGCTGGGGCTGCCTGCCG)3′-A, was the TAMRA analogue 5′-Dimethoxytrityloxy-5-[<em>N</em>-((tetramethylrhodaminyl)-aminohexyl)-3-acrylimido]-2′-deoxyUridine-3′-[(2-cyanoethyl)-(<em>N</em>,<em>N</em>-diisopropyl)]-phosphoramidite (TAMRA) and the black hole quencher 4′-(4-Nitro-phenyldiazo)-2′-methoxy-5′-methoxy-azobenzene-4″-(<em>N</em>-ethyl)-<em>N</em>-ethyl-2-cyanoethyl-(<em>N</em>,<em>N</em>-diisopropyl)-phosphoramidite (BHQ2). The dual-labelled probe, purified by high performance liquid chromatography, was purchased from Eurofins MWG Operon (Ebersberg, Germany). The other oligonucleotides (see Fig. <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>) were purchased from TIB Molbiol (Genova, Italy). Twenty nanomoles of each lyophilized samples were resuspended with Tris–HCl/EDTA buffer (pH 7.6; 10 mM ionic strength). The probe and the target oligonucleotides were diluted to the same final concentration (500 nM) and mixed in equal volumes. The mixtures were gradually heated at a rate of +3°C/min to 98°C and kept at this temperature for 10 min to allow for the denaturation of secondary structures of the single strands and to destroy the dimers that could be present. Each probe–target solution was then cooled down, at a rate of −1°C/min, to a temperature that was lower by 1°C than the melting temperature of the duplex: 73°C for the DQB1-0201 oligonucleotide, 67°C for 0302, 64°C for 0301, 61°C for 05031 and 0402, 58°C for 0501, 0502 and 0602. The samples were kept at these plateau temperatures for 20 min, in order to maximize annealing. Finally, the hybridized solutions were slowly brought to room temperature (−2°C/min) to be analysed with the TCSPC apparatus. The whole procedure of temperature-controlled annealing was carried out in the thermostat of a ThermoQuest-Finnigan gas chromatograph (San Jose, CA, USA).</p>                            <h3 scrollto-destination=6771641 id="6771641" class="section-title" data-legacy-id=s2b>2.2. Hybridization of genomic DNA</h3>
<p>Genomic DNA was extracted by standard procedures from five HLA homozygous lymphoblastoid cell lines, namely PITOUT(DQB1-0201), HOM-2 (DQB1-0501), JVM (DQB1-0301), SAVC (DQB1-0302), and OLL (DQB1-0402)<sup><a reveal-id="DSS017C6" data-open="DSS017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and from the heterozygous cell line RAJI (DQB1 0201-0501; G. Tosi, personal communication). The quality of DNA was verified by running the samples on agarose gel. Although DNA were purified about 20 yrs ago and stored at 4°C, they were not grossly degraded (data not shown). The DNA concentrations were in the range of 100–600 μg/ml, depending on the cell line. The probe was added at ≈1 pM final concentration. The procedure of temperature-controlled annealing performed after DNA denaturation at 98°C was similar to that adopted for oligonucleotides, except for the fact that the samples were put together in the thermostat and exposed to a sequence of five 20-min periods, in which the plateau temperatures were set, in order, at the values of 73, 67, 64, 61, and 58°C. These multiple exposures to decreasing temperature values should allow optimal annealing of the probe to the DQB1 antisense alleles in the following order: 0201 first, then 0302, 0301, 0402, and 0501 at last. In the same way we treated 50:50 mixtures of SAVC DNA (0302) with either PITOUT (0201) or OLL (0402) at equal concentrations (100 μg/ml) and added, as above, with the probe.</p><p>Cell lysates were prepared from PITOUT and RAJI cells. Briefly, 10<sup>6</sup> cells were resuspended with 125 μl of solution A (10 mM Tris–HCl pH 8.3, 100 mM KCl) plus 110 μl of solution B (10 mM TRIS–HCl pH 8.3, 1% Tween 20, 1% Nonidet P40) added with 15 μl Proteinase K (10 mg/ml). The cellular suspensions were incubated at 56°C for 1 h and at 95°C for 15 min. An aliquot of the cell lysate was incubated with RNAse A (1 mg/ml) for 2 min at room temperature and then at 95°C for 10 min. Both untreated and RNAse-treated cell lysates were added with equal volumes of probe (1 pM final concentration). Denaturation and annealing were performed as described for purified DNA samples.</p>                            <h3 scrollto-destination=6771644 id="6771644" class="section-title" data-legacy-id=s2c>2.3. Measurements of D-fluorescence decay time</h3>
<p>All TCSPC measurements were performed at room temperature. In the case of oligonucleotides the fluorescence was detected at 90° with respect to laser excitation and the sample was contained in 1 × 1 cm quartz cuvette with four polished windows. Owing to the smallness of the available sample volumes, in the cases of both DNAs and lysates we adopted an epi-fluorescence 20× microscope geometry, in which ∼50 μl of the samples contained in 200 μl microwells were excited and the fluorescence collected along the same direction. The D-fluorescence was separated from the reflected/back-scattered excitation by a dichroic mirror at 45° (DMLP567, Thorlabs GmbH, Dachau, Germany). The excitation/detection apparatus, extensively described in a previous work of ours,<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> is entirely made up of commercial components. Its main features are excitation pulses at 532 nm (6.4 ps duration at 113 MHz repetition rate), typically attenuated by a factor of &gt;10<sup>3</sup> to achieve single-photon regime in the fluorescence detection; single-photon avalanche diode with built-in active quenching circuitry and thermoelectric cooling as the detector; fluorescence decay patterns recorded with 2.44 ps/channel resolution; full-width at a half-maximum duration of the detected excitation pulse of &lt;30 ps; time-to-amplitude conversion operated in reversed start–stop modality so that each detected photon produced a valid start signal; fluorescence detected &gt;550 nm; each decay pattern acquired up to a fixed number of counts at the peak, namely 65535; each acquisition time recorded; each measurement repeated three times at least. The zero of the time scale was set at the peak channel for all decay patterns.</p><p>In addition to the measurements on the hybrids, we also measured the fluorescence decay of a solution of the pure single-stranded probe at the same concentration as in the hybridized samples. Preliminarily to data analysis, the raw decay patterns acquired for both this solution and those containing the hybridized oligonucleotides were normalized to the acquisition time and to the absorbance values at 556 nm (TAMRA peak). The experimental decay data were treated as described below.</p>                            <h4 scrollto-destination=6771647 id="6771647" class="section-title" data-legacy-id=s2c1>2.3.1. Data analysis for <em>τ</em><sub>D</sub> determination: oligonucleotides</h4>
<p>All decay patterns exhibit fast initial transients of negligible amplitudes, thus showing that dual-labelled fluorescent probes left free in solution are virtually absent in our oligonucleotide samples. Such a fast transient represents the majority of the fluorescence decay of the solution of the pure single-stranded probe, in which it is summed to the slow decay component emitted by free TAMRA. Thus, after normalizing to acquisition time and TAMRA absorbance, we subtract this slow component (with the amplitude and lifetime detected for the pure single-stranded probe, see below) from all decays and fit the difference starting from 400 to 800 ps after the peak. For all hybridized oligonucleotide samples we find very good fittings by using single exponentials whose time constants we interpret as the <em>τ</em><sub>D</sub> values (see column (a) in Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>). As an example we show the case of the oligonucleotide mimicking the DQB1-0201 in Fig. <a reveal-id="DSS017F3" data-open="DSS017F3" class="link link-reveal link-table xref-fig">3</a>. </p>                        <a id="6771649" scrollto-destination="6771649"></a>
<div content-id="DSS017TB1" class="table-modal"><div class="table-wrap"><div id="DSS017TB1" class="table-wrap-title" data-id="DSS017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Values of the TAMRA-donor fluorescence decay time</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>DQB1-allelic variant </th><th>(a) Oligonucleotides, <em>τ</em><sub>D</sub> ± SD (ps) </th><th>(b) DNAs, <em>τ</em><sub>D</sub> ± SD (ps) </th></tr></thead><tbody><tr><td>0201 </td><td>2725 ± 3 </td><td>2723 ± 3 </td></tr><tr><td>0501 </td><td>2514 ± 3 </td><td>2512 ± 5 </td></tr><tr><td>0502 </td><td>2404 ± 11 </td><td>NA </td></tr><tr><td>05031 </td><td>2427 ± 2 </td><td>NA </td></tr><tr><td>0602 </td><td>2430 ± 9 </td><td>NA </td></tr><tr><td>0301 </td><td>2481 ± 3 </td><td>2479 ± 3 </td></tr><tr><td>0302 </td><td>2599 ± 3 </td><td>2603 ± 5 </td></tr><tr><td>0402 </td><td>2462 ± 3 </td><td>2466 ± 5 </td></tr></tbody></table></div><div content-id="DSS017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>DQB1-allelic variant </th><th>(a) Oligonucleotides, <em>τ</em><sub>D</sub> ± SD (ps) </th><th>(b) DNAs, <em>τ</em><sub>D</sub> ± SD (ps) </th></tr></thead><tbody><tr><td>0201 </td><td>2725 ± 3 </td><td>2723 ± 3 </td></tr><tr><td>0501 </td><td>2514 ± 3 </td><td>2512 ± 5 </td></tr><tr><td>0502 </td><td>2404 ± 11 </td><td>NA </td></tr><tr><td>05031 </td><td>2427 ± 2 </td><td>NA </td></tr><tr><td>0602 </td><td>2430 ± 9 </td><td>NA </td></tr><tr><td>0301 </td><td>2481 ± 3 </td><td>2479 ± 3 </td></tr><tr><td>0302 </td><td>2599 ± 3 </td><td>2603 ± 5 </td></tr><tr><td>0402 </td><td>2462 ± 3 </td><td>2466 ± 5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>NA, sample not available. <em>τ</em><sub>D</sub> ± standard deviation (three independent measurements at least on each sample), for the 22-base long dual-labelled DQB1-0201 probe (see Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>) bound to (a) synthetic oligonucleotides corresponding to the region between codons 51 and 58 of the listed allelic variants (see Fig. <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>) and (b) purified DNAs extracted from HLA-homozygous lymphoblastoid cell lines.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6771649" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS017TB1" class="table-wrap-title" data-id="DSS017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Values of the TAMRA-donor fluorescence decay time</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>DQB1-allelic variant </th><th>(a) Oligonucleotides, <em>τ</em><sub>D</sub> ± SD (ps) </th><th>(b) DNAs, <em>τ</em><sub>D</sub> ± SD (ps) </th></tr></thead><tbody><tr><td>0201 </td><td>2725 ± 3 </td><td>2723 ± 3 </td></tr><tr><td>0501 </td><td>2514 ± 3 </td><td>2512 ± 5 </td></tr><tr><td>0502 </td><td>2404 ± 11 </td><td>NA </td></tr><tr><td>05031 </td><td>2427 ± 2 </td><td>NA </td></tr><tr><td>0602 </td><td>2430 ± 9 </td><td>NA </td></tr><tr><td>0301 </td><td>2481 ± 3 </td><td>2479 ± 3 </td></tr><tr><td>0302 </td><td>2599 ± 3 </td><td>2603 ± 5 </td></tr><tr><td>0402 </td><td>2462 ± 3 </td><td>2466 ± 5 </td></tr></tbody></table></div><div content-id="DSS017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>DQB1-allelic variant </th><th>(a) Oligonucleotides, <em>τ</em><sub>D</sub> ± SD (ps) </th><th>(b) DNAs, <em>τ</em><sub>D</sub> ± SD (ps) </th></tr></thead><tbody><tr><td>0201 </td><td>2725 ± 3 </td><td>2723 ± 3 </td></tr><tr><td>0501 </td><td>2514 ± 3 </td><td>2512 ± 5 </td></tr><tr><td>0502 </td><td>2404 ± 11 </td><td>NA </td></tr><tr><td>05031 </td><td>2427 ± 2 </td><td>NA </td></tr><tr><td>0602 </td><td>2430 ± 9 </td><td>NA </td></tr><tr><td>0301 </td><td>2481 ± 3 </td><td>2479 ± 3 </td></tr><tr><td>0302 </td><td>2599 ± 3 </td><td>2603 ± 5 </td></tr><tr><td>0402 </td><td>2462 ± 3 </td><td>2466 ± 5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>NA, sample not available. <em>τ</em><sub>D</sub> ± standard deviation (three independent measurements at least on each sample), for the 22-base long dual-labelled DQB1-0201 probe (see Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>) bound to (a) synthetic oligonucleotides corresponding to the region between codons 51 and 58 of the listed allelic variants (see Fig. <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>) and (b) purified DNAs extracted from HLA-homozygous lymphoblastoid cell lines.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6771649" target="_blank">View Large</a></div></div>                        <a id="6771650" scrollto-destination="6771650"></a>
<div id="DSS017F3" data-id="DSS017F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01703.jpeg?Expires=2147483647&Signature=alvEx1aY7V5AkCzEGQ2uohutnzTi5ywjchcob5BKJ3zRo1XWeDbVyQe0wfSmvoMQJ-L~jNHc5bIj8ekhPWA03uTVLJszF5~E6ZkNfwJL2SLU6-4Qt2Std3XnhAlh16AJS~NbBh0nIYFS2blXRB~Tuspj7b2J9McrXxXDTmgMJ9biz1ibhvXZ1qXSJIq2PLGaSB~tHSbdEpnR238PpT0vMiVDHWMFL2KdiipFi0xiYUnUzgygh7m6y6HkKmPGDHxvQlJyTbc9JAQY5jV8J3zMVVOwtrXuZk9o~ogoNHZYbnna5KkYPcWXdxIOEKqwzO81C5VYaKzasihuQaLu2kVvpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fluorescence decay of the TAMRA donor of the probe bound to the oligonucleotide mimicking the sequence of the DQB1-0201 allele. Grey dots represent the experimental data after subtraction of the slow decay of TAMRA molecules left free in the solution and black line as a fitting curve with best-fitting parameters τD = 2722.61 ps and 652.3 counts as the background. Open dots: residuals." path-from-xml="dss01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771650/dss01703.jpeg" path-from-xml="dss01703.jpeg" target="_blank">View large</a><a section="6771650" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01703.jpeg?Expires=2147483647&Signature=lB5uOU~SrvQFVS1ZwvntQT2S-d9wVeTNJRobANBZzIDzBwtovkdqQLpLf38ZS1Szjwciew-dSrH24t4zU9KfjkWwfnUH1gN3pM1u5seQEj8fWIfN5z2AsgCoMlwZWmwGL5sMkIfQcOOrFrD6ZHjxe2W8ufsUKoV9bSItwXAwSXuUkZrz-X7t~CuPVwKnJXP6Ee0wUQJzCSsG1iy-cronOX1c~6~AbAFrFyRHyacZZkZo53ePGiybeRF2JWn0hY04WnXbpdx3vhvVAfSBpOyqnRzlNZbULBmBWg2oSQpaiOfddRfiDiqXNlobJ9SShVz6kD0yu7OLSVEuaNN8sN2kzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771650&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fluorescence decay of the TAMRA donor of the probe bound to the oligonucleotide mimicking the sequence of the DQB1-0201 allele. Grey dots represent the experimental data after subtraction of the slow decay of TAMRA molecules left free in the solution and black line as a fitting curve with best-fitting parameters <em>τ</em><sub>D</sub> = 2722.61 ps and 652.3 counts as the background. Open dots: residuals.</p></div></div><div content-id="DSS017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01703.jpeg?Expires=2147483647&Signature=alvEx1aY7V5AkCzEGQ2uohutnzTi5ywjchcob5BKJ3zRo1XWeDbVyQe0wfSmvoMQJ-L~jNHc5bIj8ekhPWA03uTVLJszF5~E6ZkNfwJL2SLU6-4Qt2Std3XnhAlh16AJS~NbBh0nIYFS2blXRB~Tuspj7b2J9McrXxXDTmgMJ9biz1ibhvXZ1qXSJIq2PLGaSB~tHSbdEpnR238PpT0vMiVDHWMFL2KdiipFi0xiYUnUzgygh7m6y6HkKmPGDHxvQlJyTbc9JAQY5jV8J3zMVVOwtrXuZk9o~ogoNHZYbnna5KkYPcWXdxIOEKqwzO81C5VYaKzasihuQaLu2kVvpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fluorescence decay of the TAMRA donor of the probe bound to the oligonucleotide mimicking the sequence of the DQB1-0201 allele. Grey dots represent the experimental data after subtraction of the slow decay of TAMRA molecules left free in the solution and black line as a fitting curve with best-fitting parameters τD = 2722.61 ps and 652.3 counts as the background. Open dots: residuals." path-from-xml="dss01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771650/dss01703.jpeg" path-from-xml="dss01703.jpeg" target="_blank">View large</a><a section="6771650" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01703.jpeg?Expires=2147483647&Signature=lB5uOU~SrvQFVS1ZwvntQT2S-d9wVeTNJRobANBZzIDzBwtovkdqQLpLf38ZS1Szjwciew-dSrH24t4zU9KfjkWwfnUH1gN3pM1u5seQEj8fWIfN5z2AsgCoMlwZWmwGL5sMkIfQcOOrFrD6ZHjxe2W8ufsUKoV9bSItwXAwSXuUkZrz-X7t~CuPVwKnJXP6Ee0wUQJzCSsG1iy-cronOX1c~6~AbAFrFyRHyacZZkZo53ePGiybeRF2JWn0hY04WnXbpdx3vhvVAfSBpOyqnRzlNZbULBmBWg2oSQpaiOfddRfiDiqXNlobJ9SShVz6kD0yu7OLSVEuaNN8sN2kzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771650&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fluorescence decay of the TAMRA donor of the probe bound to the oligonucleotide mimicking the sequence of the DQB1-0201 allele. Grey dots represent the experimental data after subtraction of the slow decay of TAMRA molecules left free in the solution and black line as a fitting curve with best-fitting parameters <em>τ</em><sub>D</sub> = 2722.61 ps and 652.3 counts as the background. Open dots: residuals.</p></div></div>                            <h4 scrollto-destination=6771651 id="6771651" class="section-title" data-legacy-id=s2c2>2.3.2. Data analysis for τ<sub>D</sub> determination: DNAs</h4>
<p>We apply a modified version of the procedure described in a previous work of ours.<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In the case of extracted DNAs, as in all measured decay patterns, preliminarily normalized to acquisition time and TAMRA absorbance, we recognize a slow decay component typical of not-quenched TAMRA, either free in solution or bound to hybridized probes lacking the BHQ2 acceptor. We subtract this component from the rough experimental decays of all the hybridized DNA samples with the initial amplitude and lifetime that we detected in the decay pattern measured on the solution of pure probe. The resulting patterns are normalized to their integrals. In the case of cell lysates we detect a further slow-decay component (∼5 ns) that is typical of flavine mononucleotide, which is the main endogenous fluorofore absorbing light at 532 nm. We subtract this component from the experimental decays measured for the lysates before proceeding as above. In both cases, the resulting patterns are normalized to their integrals.</p><div class="block-child-p">Let us call the resulting normalized decay patterns <em>F<sub>x</sub></em><sub>,i</sub>, where ‘<em>x</em>’ denotes the genotype and ‘<em>i</em>’ stands for either extracted-DNA samples (<em>i</em> = EX) or lysate (<em>i</em> = LY) samples. The subtraction of not-quenched TAMRA contribution is performed also on the probe decay pattern, which is then normalized to its integral to obtain <em>F</em><sub>PROBE</sub>. We then observe that <em>F</em><sub>PROBE</sub> is well fitted by a three-exponential decays <a id="DSS017M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS017M1"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss017eq1.gif?Expires=2147483647&Signature=B96dTxTabyAPqvfvFZNcKbNxmM1wgGjWJ6iUrBLbEyOfvfD7UArqcSjYoHrwLAlX1pRe4820Y00G4b6-1afXRiS-DL-~xClwH-PBTRygbhVU1~d0RH9f8qVBiHFUtGZ5WFGRSR~rzG9t5snwh~o6cBxp3srXECx3AB1nL64EB1ZFB2PEGuOe7uUbWjupP4-UiqjQQALXDXAP2PivUMnz1fRmRIBpGU8MKSyEBKNOyzCWQ7U1DMRL7vWx4S4WiPuHaCtLm7s2RE7hRMgSq-DIlWGjmB-Scolk7Lwl6hGUE-0gGRzxVhGnpFVCDiB6hkTaeI5wUNckll5mqhMG70TQaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss017eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss017eq1.gif?Expires=2147483647&Signature=czbDHOBHbfQU~0XA2x9svU2I8e9BMHxlegshMOQoseveB0ClH3teQzYXPUsIR~-i2~5lXBBkhrsXbrKuQ8vq4X8ImYQAH-g5y~JDGG7vP2a4cFAfftSmR8Ch9R-YdBtsRNa8kscKClQ5uZplKG7bFuXgBefgUZacbL9vNerkvxoI5Ku3I44d3iQZNuWu4VFsOQk9hAfRvLKxSI6rhPmRNA5t2Tda13IrQocVxKoOqZiV6nTVWDy3czbRFeZogM2d~NFWEqUJqL9reNstEr9abL4WCoDNg7EaGw2wMKrCPa5BsgB3sgyZ6RvPHaYfE~GoQb~9hqORTInXZj0Pb07f4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss017eq1.gif" /></a></div></div><div class="label">(1)</div></div></div><div class="block-child-p">The best procedure we found to get rid of the <em>F</em><sub>PROBE</sub> contributions in the decays <em>F<sub>x</sub></em><sub>,<em>i</em></sub> is as follows: we carry out multi-exponential fits of the latter decays from which we only extract the values <em>A</em><sub>1<em>x</em>,<em>i</em></sub>, <em>A</em><sub>2<em>x</em>,<em>i</em></sub>, and <em>A</em><sub>3<em>x</em>,<em>i</em></sub> of the three fastest components and use them to perform the following weighted subtractions <a id="DSS017M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS017M2"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss017eq2.gif?Expires=2147483647&Signature=qQ6ENKVcpV1VOdNjtpmGuGWKFkvt7Sb6LtHms-UP1bG8kvO2ygV7Be~BIBC4x6mGcR0pFeXkh84~akc~Wwk5-2B2qt5MaslNad0nZMi5njSj8LWLPAQyJ7aoIHvegfbD7iljE~WtFCm5Ojzj45HLhM7CQUjFk4mjKI~qiET6IZn1F2MBe7WWqEg3VrRrHLv801dG524Beo~8Bk1X6ZsDAVqvjqWYrC0THOdWNKCgWFID9WGpMtGAVmembZIEzj92hzFA1m9aZbJpjfAMX4-C1itjwBIh-nitOf~9caM5m7X~dzyN0w6QOwcb59VDXGy1BYO-BqnCTHS2MOq6nUSaEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss017eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss017eq2.gif?Expires=2147483647&Signature=08mOwFuXUbLSf0ct5QXAhyU-9CN2lXTET5um-ropdb9HL2qAQqYd1IVU~i-2nEraiFBtAsI3NaPS77n3-eurvUqo~Tz3Q5vxSL4S3Q5dWrfKnYXlr2ipW7edW2kTsdJP8aMwSzizWktXG75qywZ-6RrRJI9QdhU~LxFvQQwLUNWPgLKTXAaOwjE6vlOFWJymSVVKHXe816I931JxlaRq5I0hUa0kIkFYgFBvGdNC5lZWVWCr0dGVGOzI2f7bDH2hUUg3tsG7npHzOBzqHiDZhyXRCBAxskKTr~csBl6NghKDyTqIVjr7320yBzAeyE7X5ddOQajIyAPfWWUmonmuaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss017eq2.gif" /></a></div></div><div class="label">(2)</div></div> that yield the final curves <em>D<sub>x</sub></em><sub>,</sub><em><sub>i</sub></em> Curves <em>D<sub>x</sub></em><sub>,</sub><em><sub>i</sub></em> should only contain the fluorescence arising from the desired double-stranded fraction of each sample with decay time <em>τ</em><sub>D</sub> and, possibly, from the probe molecules partially annealed to other regions of the genomic DNA target. Actually, all curves <em>D<sub>x</sub></em><sub>,</sub><em><sub>i</sub></em> had dynamic ranges covering more than one decade. In the case of homozygous genotypes the <em>D<sub>x</sub></em><sub>,</sub><em><sub>i</sub></em> curves exhibited bi-exponential decays starting from ∼500 ps after the peak. Best fits of the <em>D<sub>x</sub></em><sub>,</sub><em><sub>i</sub></em> curves provide the values of the corresponding decay times <em>τ</em>′ and <em>τ</em><sub>D</sub>. The whole procedure is illustrated in Fig. <a reveal-id="DSS017F4" data-open="DSS017F4" class="link link-reveal link-table xref-fig">4</a> for the DNA extracted from the PITOUT cell line, homozygous for the DQB1-0201 allelic variant. </div>                        <a id="6771655" scrollto-destination="6771655"></a>
<div id="DSS017F4" data-id="DSS017F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01704.jpeg?Expires=2147483647&Signature=jSS8rlI39TsSzCt08tJnEiEZJjoxU1qelcQT69F05nCmS~jR8KEdwnD1UUiSgP7rxdNxNQsaj-bDpAUbsGb89yuy0OOtBHt9igUmdqMs2wtgQ~U3mZv~4l5IMdSc2oBSGThg~uEfetEAPcz26OnnAikSasyRjeDpHYrLd0BXODeTcoYYd1gxK1GXDC3aWSxVt9u1GunW2qd3yQC3NboXoEIN0q0GuphleGByqp-LNvAL-46GMIEDGfJ5Fi0zTaj9MZILpW1rjlY09CKjYMdtiymufj6LP6KGFmeOFxQbjKMEMKtp1R7yBNUoy1aZQCtYH19MTJH1TTjFv~s2SCXqYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) Raw experimental decay patterns of the samples containing (black dots) the probe hybridized to genomic DNA extracted from the PITOUT cell line, homozygous for the DQB1-0201 allelic variant; (open dots) the dual-labelled probe only. (b) Integral-normalized fluorescence decays F201 (black dots) and FPROBE (open dots) as obtained from the decays in (a) upon subtraction of the slow decay component of not-quenched TAMRA. Black line represents the fitting curve YPROBE in Equation (1). (c) Decay D201 (grey dots) as obtained by applying Equation (2) to the data in (b). Black line represents the bi-exponential fit with best-fitting parameters τ’ = 709.92 ps and τD = 2724.43 ps. Open dots: residuals. The arbitrary units in (b and c) are identical to allow the comparison, in amplitude, of D201 to F201 (see Section 4)." path-from-xml="dss01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771655/dss01704.jpeg" path-from-xml="dss01704.jpeg" target="_blank">View large</a><a section="6771655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01704.jpeg?Expires=2147483647&Signature=cfckmoiw6~dbPxFs6VRq8AJxBVdeNGGJsO0AzyFplRtMF~Jt9DZNsxkPyma0JAvASEpmJ0n3OkOzO0qK~LILp49rfE558PdZvvnhdTEXlqprPT298avjEupO08Yn01TF3jzHGhuOQpR6~4tTa0bFRsNgIdAzRf6uNN4Sd~~zdeeF-pCVyu6Yctk2ykGWgsZFo~ho2iSLHxsF6ZzqWEWxGbqPw4UvdswOTP-HPhiF-lTy~QLZAduiANxPv35t6xNahnBQHVeYQPv-mhlyWezywVXlrroUZmACsJNKR2j3-00dR9Z4Hd4d02SSnmZNWapbkYTWfynrKF5E9FzuDBRN2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771655&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) Raw experimental decay patterns of the samples containing (black dots) the probe hybridized to genomic DNA extracted from the PITOUT cell line, homozygous for the DQB1-0201 allelic variant; (open dots) the dual-labelled probe only. (b) Integral-normalized fluorescence decays <em>F</em><sub>201</sub> (black dots) and <em>F</em><sub>PROBE</sub> (open dots) as obtained from the decays in (a) upon subtraction of the slow decay component of not-quenched TAMRA. Black line represents the fitting curve <em>Y</em><sub>PROBE</sub> in Equation (1). (c) Decay <em>D</em><sub>201</sub> (grey dots) as obtained by applying Equation (2) to the data in (b). Black line represents the bi-exponential fit with best-fitting parameters <em>τ</em>’ = 709.92 ps and <em>τ</em><sub>D</sub> = 2724.43 ps. Open dots: residuals. The arbitrary units in (b and c) are identical to allow the comparison, in amplitude, of <em>D</em><sub>201</sub> to <em>F</em><sub>201</sub> (see Section 4).</p></div></div><div content-id="DSS017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/m_dss01704.jpeg?Expires=2147483647&Signature=jSS8rlI39TsSzCt08tJnEiEZJjoxU1qelcQT69F05nCmS~jR8KEdwnD1UUiSgP7rxdNxNQsaj-bDpAUbsGb89yuy0OOtBHt9igUmdqMs2wtgQ~U3mZv~4l5IMdSc2oBSGThg~uEfetEAPcz26OnnAikSasyRjeDpHYrLd0BXODeTcoYYd1gxK1GXDC3aWSxVt9u1GunW2qd3yQC3NboXoEIN0q0GuphleGByqp-LNvAL-46GMIEDGfJ5Fi0zTaj9MZILpW1rjlY09CKjYMdtiymufj6LP6KGFmeOFxQbjKMEMKtp1R7yBNUoy1aZQCtYH19MTJH1TTjFv~s2SCXqYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) Raw experimental decay patterns of the samples containing (black dots) the probe hybridized to genomic DNA extracted from the PITOUT cell line, homozygous for the DQB1-0201 allelic variant; (open dots) the dual-labelled probe only. (b) Integral-normalized fluorescence decays F201 (black dots) and FPROBE (open dots) as obtained from the decays in (a) upon subtraction of the slow decay component of not-quenched TAMRA. Black line represents the fitting curve YPROBE in Equation (1). (c) Decay D201 (grey dots) as obtained by applying Equation (2) to the data in (b). Black line represents the bi-exponential fit with best-fitting parameters τ’ = 709.92 ps and τD = 2724.43 ps. Open dots: residuals. The arbitrary units in (b and c) are identical to allow the comparison, in amplitude, of D201 to F201 (see Section 4)." path-from-xml="dss01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6771655/dss01704.jpeg" path-from-xml="dss01704.jpeg" target="_blank">View large</a><a section="6771655" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss017/2/dss01704.jpeg?Expires=2147483647&Signature=cfckmoiw6~dbPxFs6VRq8AJxBVdeNGGJsO0AzyFplRtMF~Jt9DZNsxkPyma0JAvASEpmJ0n3OkOzO0qK~LILp49rfE558PdZvvnhdTEXlqprPT298avjEupO08Yn01TF3jzHGhuOQpR6~4tTa0bFRsNgIdAzRf6uNN4Sd~~zdeeF-pCVyu6Yctk2ykGWgsZFo~ho2iSLHxsF6ZzqWEWxGbqPw4UvdswOTP-HPhiF-lTy~QLZAduiANxPv35t6xNahnBQHVeYQPv-mhlyWezywVXlrroUZmACsJNKR2j3-00dR9Z4Hd4d02SSnmZNWapbkYTWfynrKF5E9FzuDBRN2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6771655&ar=451164&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) Raw experimental decay patterns of the samples containing (black dots) the probe hybridized to genomic DNA extracted from the PITOUT cell line, homozygous for the DQB1-0201 allelic variant; (open dots) the dual-labelled probe only. (b) Integral-normalized fluorescence decays <em>F</em><sub>201</sub> (black dots) and <em>F</em><sub>PROBE</sub> (open dots) as obtained from the decays in (a) upon subtraction of the slow decay component of not-quenched TAMRA. Black line represents the fitting curve <em>Y</em><sub>PROBE</sub> in Equation (1). (c) Decay <em>D</em><sub>201</sub> (grey dots) as obtained by applying Equation (2) to the data in (b). Black line represents the bi-exponential fit with best-fitting parameters <em>τ</em>’ = 709.92 ps and <em>τ</em><sub>D</sub> = 2724.43 ps. Open dots: residuals. The arbitrary units in (b and c) are identical to allow the comparison, in amplitude, of <em>D</em><sub>201</sub> to <em>F</em><sub>201</sub> (see Section 4).</p></div></div>                            <h2 scrollto-destination=6771656 id="6771656" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6771657 id="6771657" class="section-title" data-legacy-id=s3a>3.1. Identification of the hybrids formed by allele-specific HLA-II DQB1 oligonucleotides and the 0201 oligoprobe</h3>
<p>In order to establish our FRET-based HLA-II DQB1 typing method we first set the system by using the panel of synthetic oligonucleotides listed in Fig. <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>. These oligonucleotides are complementary to the coding sequences of the eight relevant DQB1 alleles and encompass codons of 51–60.<sup><a reveal-id="DSS017C6" data-open="DSS017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Nucleotide variations with respect to the 0201 allele, shown on the top, are underlined. Each oligonucleotide was hybridized with the probe corresponding to the 0201 variant having the GCC codon for alanine in position 57. The oligoprobe and the structures of the TAMRA donor and BHQ2 acceptor molecules are shown in Fig. <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a>. In general, to increase the specificity of a probe for its target DNA, it is desirable to design the oligonucleotide with a great rather than a small number of nucleotides. However, this is not true for probes dual labelled at the 5′- and 3′-positions by a D and an A interacting by FRET, in that the D-to-A distance cannot remarkably overcome the Förster radius. We designed a probe of 22 nucleotides covering from codon 51 up to the first base of codon 58 as we preliminarily demonstrated that probe/target mismatches next to the probe ends, where D or A are covalently bound, cause the greatest changes in the FRET efficiency.<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS017C12" data-open="DSS017C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In the present case, mismatches at the polymorphic codon 57, which occur in the cases of all variants except 0201 and 0302, are expected to prevent pairing of the last three–four bases on the side of the BHQ2. Such an ‘unzipping’ of the duplex structure should let A approach D and cause a sizeable decrease in the donor-fluorescence decay time (<em>τ</em><sub>D</sub>). It is worth noting that the eight DQB1 alleles are all identical from codon 58 to codon 60, thus further extending the probe would prevent unzipping.</p><p>As mentioned in Section 2.3.1 the decay patterns recorded in the case of oligonucleotides exhibit fast initial transients of negligible amplitudes, thus showing that dual-labelled fluorescent probes left free in solution are virtually absent in our oligonucleotide samples and testifying that the affinity of our new 22-nucleotide probe for the DQB1 template is notably enhanced with respect to that of the 18-nucleotide probe, with which we detected sizeable amounts of non-hybridized probe molecules.<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The <em>τ</em><sub>D</sub> values that we determined for the DQB1-0201 probe binding to each of the eight synthetic oligonucleotides are listed in column (a) of Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>. For all hybrids, except those that the probe forms with the variant sequences 05031 and 0602, we find significantly different <em>τ</em><sub>D</sub> values. Most importantly, the greatest value corresponds to the perfect matching of the probe with the complementary 0201 oligonucleotide. The next highest <em>τ</em><sub>D</sub> value is exhibited by the probe binding the 0302 sequence, which is the only one displaying a base composition identical to that of the 0201 allele at codon 57. The manner in which the other values of <em>τ</em><sub>D</sub> scale agrees with what we previously observed by using a probe containing 18 instead of 22 nucleotides.<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Moreover, the differences among the data in column (a) confirm that the type of mismatch remarkably influences the decay time. In fact, the 0501 and 0602 alleles, having only one nucleotide variation in the same position of codon 57 (CAA and CTA, respectively) display significantly different <em>τ</em><sub>D</sub> values upon hybridization to the 0201 probe.</p>                            <h3 scrollto-destination=6771660 id="6771660" class="section-title" data-legacy-id=s3b>3.2. HLA-II DQB1 typing of purified, non-amplified genomic DNA</h3>
<p>In order to validate our FRET-based HLA-II DQB1 typing approach set with synthetic oligonucleotides, we tested the system with non-PCR-amplified genomic DNAs extracted from five HLA-homozygous lymphoblastoid cell lines with different DQB1 specificities.<sup><a reveal-id="DSS017C6" data-open="DSS017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Upon the modifications of the setup as to the excitation/collection optics described in Sub-section 2.3, the corresponding <em>τ</em><sub>D</sub> values were measured (see data listed in column (b) of Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>). These values are surprisingly equal, within the experimental errors, to those obtained with the oligonucleotides corresponding to the same DQB1 alleles, which are listed on the same lines in column (a), in spite of the fact that these measurements were performed on entire genomic DNA.</p><p>To assess the possibility of recognizing the presence of the 0201 variant also in the case of a heterozygous genomic DNA, we measured the fluorescence decays for the two 50:50 mixtures of 0201 with 0302 and of 0302 with 0402. Such samples were chosen in that the <em>τ</em><sub>D</sub> values obtained from the fittings of <em>D</em><sub>201,EX</sub> and <em>D</em><sub>302,EX</sub> are the most similar (see column (b) in Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>). A predominant decay component with time constant equal to the <em>τ</em><sub>D</sub> of <em>D</em><sub>201,EX</sub> was found in the fitting of curve <em>D</em><sub>201-302,EX</sub> (former sample) only. In the fitting of <em>D</em><sub>302-402,EX</sub> the predominant component had time constant equal to the <em>τ</em><sub>D</sub> of <em>D</em><sub>302,EX</sub>.</p><p>We conclude that the probe–template annealing is correctly attained in both homozygous and heterozygous situations.</p>                            <h3 scrollto-destination=6771664 id="6771664" class="section-title" data-legacy-id=s3c>3.3. Identification of DQB1-0201 allele in cell lysates</h3>
<p>In order to verify whether our typing method is sensitive enough to detect the DQB1-0201 allele directly in cell lysates without prior DNA purification, we performed measurements on both untreated and RNAse-treated cell lysates obtained from PITOUT (homozygous DQB1-0201) and RAJI (heterozygous DQB1 0201-0501) cell lines. DQB1-0201 probe was designed on the basis of the sequence of the ‘plus’ strand of DNA to avoid annealing with RNA. However, to completely exclude any possible spurious and non-specific annealing/interference derived from RNA in a ‘dirty’ system such as that of the cell extracts, we treated the cell lysates with RNAse. For both cell lines, the fluorescence responses were not improved in RNAse-treated as compared with untreated lysates (data not shown), suggesting that RNAse treatment is not a prerequisite step. Moreover, the fitting of <em>D</em><sub>201,LY</sub> returned a value <em>τ</em><sub>D</sub> = 2724 ± 4 ps, which coincides with that returned by the fitting of <em>D</em><sub>201,EX</sub> in the first line of Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>, column (b) within the experimental errors. We add that the <em>τ</em><sub>D</sub> component is so much more relevant in <em>D</em><sub>201,LY</sub> than in <em>D</em><sub>201,EX</sub> as to make the <em>τ</em>′ component negligible in amplitude.</p><p>In the case of the RJ 2.2.5 lysate we also measured the decay for its extracted DNA and obtained a <em>D</em><sub>201-501,EX</sub> curve in which the predominant component had time constant equal to the <em>τ</em><sub>D</sub> of <em>D</em><sub>201,EX</sub>. This was also the major component in <em>D</em><sub>201-501,LY</sub>.</p>                            <h2 scrollto-destination=6771667 id="6771667" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study we successfully developed a new method to identify the DQB1-0201 variant, an allele that is considered a genetic marker of susceptibility to IDDM. Indeed, this allele is significantly over-represented in patients with IDDM. The method is based on the measurements of fluorescence decay times performed with single photon sensitivity and picosecond resolution. Non-amplified genomic DNAs, purified from lymphoblastoid cell lines and bearing different sequences in the polymorphic region of the HLA-DQB1 gene, were hybridized with a single DNA probe corresponding to the DQB1-0201 allele. The same probe was also hybridized to non-purified DNA contained in crude cell extracts. The probe was dual-labelled with fluorescence donor-and-acceptor chromophores at its opposite ends. The chromophores undergo FRET, whose efficiency is characterized by measuring the fluorescence decay time <em>τ</em><sub>D</sub> of the donor. As the FRET efficiency, and hence the <em>τ</em><sub>D</sub> value, is differently modified by the number of probe-to-target mismatches and by their positions, the hybrids formed by the probe with each allelic variant are identified by distinct values of <em>τ</em><sub>D</sub>. It is worth noting that the picosecond-laser excitation wavelength we adopted, 532 nm, is very close to the absorption peak of the TAMRA donor, which falls at 556 nm, being not even absorbed by the non-fluorescent BHQ2 acceptor.</p><p>We achieved our main goal by successive steps. We first showed that the hybrids formed by allele-specific HLA-II DQB1 oligonucleotides and the 0201 oligoprobe can be identified through the different <em>τ</em><sub>D</sub> values. Though in a previous work of ours we obtained such a result by using an 18-nucleotides probe,<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reproducing it with our 22-nucleotides probe, which we deemed ideal for typing of genomic DNA templates, was not obvious in that this probe covered codon 51, which is identical in all allelic variants (see Figs <a reveal-id="DSS017F1" data-open="DSS017F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSS017F2" data-open="DSS017F2" class="link link-reveal link-table xref-fig">2</a>). So much so that, as shown in Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>, for the mismatching oligonucleotides mimicking the sequences of the DQB1 allelic variants other than 0201, we measure <em>τ</em><sub>D</sub> values that are both greater than those for the 18-nucleotide probe (see, in particular, the cases of alleles 0501, 0502, 05031, and 0602, which produced very fast decays when hybridized to the 18-nucleotide probe) and much more similar to that measured for the perfectly matching target oligonucleotide.<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> For these target sequences we had hypothesized an unzipping involving the first five nucleotides, due to the double mismatch of the second and fifth nucleotides. However, the addition of the three nucleotides corresponding to codon 51 confers a greatly enhanced selectivity to the 22-nucleotide probe, as reported in Section 3.1, and does not prevent discrimination of the DQB1-0201 genotype. Beside the above considerations, the scaling of the values in column (a) of Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a> agrees with the explanations that we presented in the quoted work of the dependence of the <em>τ</em><sub>D</sub> changes on both the number and the position of the base mismatches. Moreover, the 22-nucleotide probe preserves the <em>τ</em><sub>D</sub> sensitivity to the variation of a single nucleotide, as shown by the results reported in Section 3.1 for the 0501 and 0602 alleles.</p><p>On the one hand, the <em>τ</em><sub>D</sub> measurements on entire genomic DNA represent a very strong test of the probe selectivity. On the other hand, as they lead for each allelic species to the same <em>τ</em><sub>D</sub> values that are found for the corresponding oligonucleotide, these <em>τ</em><sub>D</sub> measurements constitute the core result of our work. The coincidence between the values in the two columns in Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a> and, in the case of DQB1-0201, with the <em>τ</em><sub>D</sub> value obtained with cell lysate, demonstrates that the conformation of the probe–template duplex is identical in the three instances. This is not surprising once the probe–template annealing is correctly attained, which actually is the main assumption underlying the rationale of the method. In fact, on the local scale of 22 base pairs, the tertiary structure of genomic DNA is not relevant, as the persistence length of a DNA B-structured double-helix is estimated to be ∼50 base pairs. In our opinion the really astonishing feature is the notable template specificity of the probe that leads to the observation of a detectable <em>τ</em><sub>D</sub> component in the decays obtained with genomic DNA (even when non-purified), which prevails in all cases. We can only attribute this result to the goodness of both the design of the probe and the scaling of temperatures during annealing. Moreover, designing the DQB1-0201 probe on the basis of the sequence of the ‘plus’ strand of DNA prevents annealing with RNA, thus making RNAse treatment of the cell extracts not necessary.</p><p>We think that a very important role is played in this work by the strategies we devised to determine the values of <em>τ</em><sub>D</sub>. First they are suitable for taking into account that not all the TAMRA molecules that contribute to the detected fluorescence-decay patterns decay with time-constant <em>τ</em><sub>D</sub>. There can be dual-labelled fluorescent probes free in solution, thus keeping their natural conformation as single-stranded probe and contributing decay-components shorter than <em>τ</em><sub>D</sub>. There might also be probes lacking the BHQ2 acceptor as well as TAMRA molecules not bound to the probe: both of them decay slower than any TAMRA quenched via FRET by the presence of a more-or-less distant BHQ2. Moreover, in the case of lysates, flavine mononucleotide fluorescence adds to the TAMRA decay. The overall effect is that the experimental decay patterns deviate from single exponentials. Finally, the binding of the probe to the specific target sequences is contrasted by partial annealing to other genomic DNA regions other than the allelic-variant under examination. However, a neutral application to all experimental decay patterns of the strategy described in Section 2 and already used in a previous work of ours,<sup><a reveal-id="DSS017C11" data-open="DSS017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> brought to decays (<em>D</em><sub>x</sub>) of remarkable amplitudes that could be fitted by the sum of two exponentials in the case of homozygous genotypes. The faster one has a decay time τ′ = 709 ± 67 ps, which is attributable to the partially annealed probes, while the slower one decays with the <em>τ</em><sub>D</sub> values reported in Table <a reveal-id="DSS017TB1" data-open="DSS017TB1" class="link link-reveal link-table xref-fig">1</a>. For heterozygous genotypes, beside <em>τ</em>′, we recognized the decay of the allele best matching the probe as the predominant one. The further decay component was too low in amplitude to allow identification of the second allele, which might be defined by using a second allele-specific probe. Nonetheless, we attained our primary goal, that is, the identification of the DQB1-0201 allele in a heterozygous genotype even in the presence of the allele with the most similar <em>τ</em><sub>D</sub> (i.e. DQB1-0302). The result obtained for the allele mixture of DQB1-0302 with DQB1-0402 showed that the method is free from the risk of detecting false positives (i.e. erroneously inferring the presence of the 0201 allele) even when the 0302 allele is present. Finally, it is worth noting that, when DQB1-0201 was present, we could distinguish between homozygous and heterozygous genotypes.</p><p>The fact that superimposable results were obtained with crude cell extracts without the need of DNA purification, makes our HLA typing method greatly advantageous with respect to other current techniques. As to the performance in a clinical setting, we think it is worth noting that, once the cell lysates are prepared, our technique only requires common buffers and negligible amounts of dual-labelled probes, with no need of enzymatic reactions or temperature-controlled incubation steps. Firstly, this makes the analysis easier and reduces the risks of sample contamination/degradation. Moreover, by virtue of the reduced amount of consumables needed, our method combines the timelines of the most state-of-art genotyping techniques based on PCR amplification with superior cost-effectiveness, which is a desirable feature in view of an application to population screening. Finally, the HLA typing method reported here is relatively simple; the instrumentation that is needed is not particularly expensive and is prone to be adapted for routine use in a diagnostic laboratory.</p><p>The single-photon sensitivity, which is intrinsic to all TCSPC techniques, also deserves some comment, in that it can bring the low detection limit of genomic DNA down to unprecedented values. In fact, as specified in Section 2.2, to keep the detected-photon rate within the single-photon regime, we attenuated the excitation laser by a factor of &gt;10<sup>3</sup> and obtained good fluorescence decay patterns from DNA samples of few hundredths picomolar concentration. However, there is no reason why, by using the full power of the laser, we could not lower the DNA concentrations by a factor of one thousand and obtain the same results without increasing the data-acquisition times. We think that, such a performance would be extremely desirable in a clinical setting for applications in which the sensitivity issue is definitely a matter of concern. Discriminating among the different alleles by the <em>τ</em><sub>D</sub> values, as shown in this paper in the case of a highly polymorphic gene, may allow the use of this method not only for routine typing of HLA, but also for the identification of specific ‘pathologic’ sequences, for example mutations of genes causing inherited genetic diseases or mutated oncogenes, in order to identify the specific mutation in tumour tissues and, most importantly, the extent of mutated genes in pre-neoplastic lesions.</p>                            <h2 scrollto-destination=6771674 id="6771674" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the following grants to R.S.A.: Fondazione Cariplo 2008-2230 ‘Cellular and molecular basis of human retroviral-dependent pathology’; A.I.R.C IG 8862 ‘New strategies of tumor vaccination and immunotherapy based on optimized triggering of anti-tumor CD4+ T cells'; MIUR-PRIN project 2008-WXF7KK ‘New strategies of immunointervention against tumors’. A.A. and G.T. also acknowledge the support by University of Insubria through projects ‘FAR 2009’ and ‘FAR 2010’.</p>                            <h2 scrollto-destination=6771676 id="6771676" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank A. Maspero for making the Finnigan instrument available and N. Camera for his technical assistance. We also acknowledge the funding of the equipment maintenance that is granted by Centro Servizi e Grandi Attrezzature per lo Studio e la Caratterizzazione della Materia of the University of Insubria. L.N. position is funded by the project POR FSE Regione Lombardia, Ob. 2 Asse IV 2007-2013, through an FP7 cooperation programme. This paper is dedicated to the Mother of M.B., who recently lost her struggle against diabetes.</p>                            <h2 scrollto-destination=6771678 id="6771678" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nepom</div> <div class="given-names">G.T.</div></div>. </div><div class="article-title">HLA and type I diabetes</div>, <div class="source">Immunol. Today</div> , <div class="year">1990</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">314</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0167-5699(10)80003-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0167-5699(10)80003-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HLA%20and%20type%20I%20diabetes&amp;author=G.T.%20Nepom&amp;publication_year=1990&amp;journal=Immunol.%20Today&amp;volume=11&amp;pages=314-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-5699(10)80003-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-5699(10)80003-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-5699(10)80003-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2206277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tosi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Facchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pinelli</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Accolla</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Assessment of the DQB1-DQA1 complete genotype allows best prediction for IDDM</div>, <div class="source">Diabetes Care</div> , <div class="year">1994</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1045</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2337/diacare.17.9.1045" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2337/diacare.17.9.1045</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20the%20DQB1-DQA1%20complete%20genotype%20allows%20best%20prediction%20for%20IDDM&amp;author=G.%20Tosi&amp;author=A.%20Facchin&amp;author=L.%20Pinelli&amp;author=R.S.%20Accolla&amp;publication_year=1994&amp;journal=Diabetes%20Care&amp;volume=17&amp;pages=1045-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2337/diacare.17.9.1045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2337%2Fdiacare.17.9.1045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2337%2Fdiacare.17.9.1045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7988304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tipu</div> <div class="given-names">H.N.</div></div>,  <div class="name"><div class="surname">Ahmed</div> <div class="given-names">T.A.</div></div>,  <div class="name"><div class="surname">Bashir</div> <div class="given-names">M.M.</div></div>. </div><div class="article-title">Human leukocyte antigen class II susceptibility conferring alleles among non-insulin dependent diabetes mellitus patients</div>, <div class="source">J. Coll. Phys. Surg. Pak.</div> , <div class="year">2011</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">26</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="DSS017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Nickerson</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Simons</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">Sensitive mutation detection in heterogeneous cancer specimens by massively parallel picoliter reactor sequencing</div>, <div class="source">Nat. Med.</div> , <div class="year">2006</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">852</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nm1437" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nm1437</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sensitive%20mutation%20detection%20in%20heterogeneous%20cancer%20specimens%20by%20massively%20parallel%20picoliter%20reactor%20sequencing&amp;author=R.K.%20Thomas&amp;author=E.%20Nickerson&amp;author=J.F.%20Simons&amp;publication_year=2006&amp;journal=Nat.%20Med.&amp;volume=12&amp;pages=852-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm1437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm1437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm1437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16799556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Todd</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Bell</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">McDevitt</div> <div class="given-names">H.O.</div></div>. </div><div class="article-title">HLA-DQB gene contributes to susceptibility and resistance to insulin dependent diabetes mellitus</div>, <div class="source">Nature</div> , <div class="year">1987</div>, vol. <div class="volume">329</div> (pg. <div class="fpage">599</div>-<div class="lpage">604</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/329599a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/329599a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HLA-DQB%20gene%20contributes%20to%20susceptibility%20and%20resistance%20to%20insulin%20dependent%20diabetes%20mellitus&amp;author=J.A.%20Todd&amp;author=J.I.%20Bell&amp;author=H.O.%20McDevitt&amp;publication_year=1987&amp;journal=Nature&amp;volume=329&amp;pages=599-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/329599a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F329599a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F329599a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3309680" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tosi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Mantero</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Magalini</div> <div class="given-names">A.R.</div></div>, et al. </div><div class="article-title">HLA-DQB1 typing of north-east Italian IDDM patients using amplified DNA, oligonucleotide probes, and a rapid DNA-enzyme immunoassay (DEIA)</div>, <div class="source">Mol. Immunol.</div> , <div class="year">1993</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">69</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0161-5890(93)90427-D" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0161-5890(93)90427-D</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HLA-DQB1%20typing%20of%20north-east%20Italian%20IDDM%20patients%20using%20amplified%20DNA%2C%20oligonucleotide%20probes%2C%20and%20a%20rapid%20DNA-enzyme%20immunoassay%20(DEIA)&amp;author=G.%20Tosi&amp;author=G.%20Mantero&amp;author=A.R.%20Magalini&amp;publication_year=1993&amp;journal=Mol.%20Immunol.&amp;volume=30&amp;pages=69-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0161-5890(93)90427-D" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0161-5890(93)90427-D" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0161-5890(93)90427-D"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8417376" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tosi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Brunelli</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mantero</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">The complex interplay of the DQB1 and DQB2 loci in the generation of the susceptibility and protective phenotype for insulin-dependent diabetes mellitus</div>, <div class="source">Mol. Immunol.</div> , <div class="year">1994</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">429</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0161-5890(94)90062-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0161-5890(94)90062-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20interplay%20of%20the%20DQB1%20and%20DQB2%20loci%20in%20the%20generation%20of%20the%20susceptibility%20and%20protective%20phenotype%20for%20insulin-dependent%20diabetes%20mellitus&amp;author=G.%20Tosi&amp;author=S.%20Brunelli&amp;author=G.%20Mantero&amp;publication_year=1994&amp;journal=Mol.%20Immunol.&amp;volume=31&amp;pages=429-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0161-5890(94)90062-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0161-5890(94)90062-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0161-5890(94)90062-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8183282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tyagi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bratu</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Russell Kramer</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Multicolor molecular beacons for allele discrimination</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">1998</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">49</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt0198-49" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt0198-49</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multicolor%20molecular%20beacons%20for%20allele%20discrimination&amp;author=S.%20Tyagi&amp;author=D.P.%20Bratu&amp;author=F.%20Russell%20Kramer&amp;publication_year=1998&amp;journal=Nat.%20Biotechnol.&amp;volume=16&amp;pages=49-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt0198-49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt0198-49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt0198-49"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9447593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gaylord</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Massie</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Feinstein</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Bazan</div> <div class="given-names">G.C.</div></div>. </div><div class="article-title">SNP detection using peptide nucleic acid probes and conjugated polymers: applications in neurodegenerative disease identification</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">34</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0407578101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0407578101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20detection%20using%20peptide%20nucleic%20acid%20probes%20and%20conjugated%20polymers%3A%20applications%20in%20neurodegenerative%20disease%20identification&amp;author=B.S.%20Gaylord&amp;author=M.R.%20Massie&amp;author=S.C.%20Feinstein&amp;author=G.C.%20Bazan&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=34-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407578101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407578101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407578101"> </span></div> </div></div></div></div></div><div content-id="DSS017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Al Attar</div> <div class="given-names">H.A.</div></div>,  <div class="name"><div class="surname">Norden</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">O'Brien</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Monkman</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Improved single nucleotide polymorphism detection using conjugated polymer/surfactant system and peptide nucleic acid</div>, <div class="source">Biosens Bioelectron</div> , <div class="year">2008</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1466</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bios.2008.01.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bios.2008.01.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20single%20nucleotide%20polymorphism%20detection%20using%20conjugated%20polymer%2Fsurfactant%20system%20and%20peptide%20nucleic%20acid&amp;author=H.A.%20Al%20Attar&amp;author=J.%20Norden&amp;author=S.%20O'Brien&amp;author=A.P.%20Monkman&amp;publication_year=2008&amp;journal=Biosens%20Bioelectron&amp;volume=23&amp;pages=1466-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bios.2008.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bios.2008.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bios.2008.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18296040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andreoni</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bondani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nardo</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Time-resolved FRET method for typing polymorphic alleles of the human leukocyte antigen system by using a single DNA probe</div>, <div class="source">Photochem. Photobiol. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1202</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1039/b906043j" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1039/b906043j</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Time-resolved%20FRET%20method%20for%20typing%20polymorphic%20alleles%20of%20the%20human%20leukocyte%20antigen%20system%20by%20using%20a%20single%20DNA%20probe&amp;author=A.%20Andreoni&amp;author=M.%20Bondani&amp;author=L.%20Nardo&amp;publication_year=2009&amp;journal=Photochem.%20Photobiol.%20Sci.&amp;volume=8&amp;pages=1202-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/b906043j" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fb906043j" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fb906043j"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19639124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andreoni</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bondani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nardo</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Feasibility of single nucleotide polymorphism genotyping with a single-probe by time-resolved Förster resonance energy transfer</div>, <div class="source">Mol. Cell. Probes</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">119</div>-<div class="lpage">21</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.mcp.2008.12.008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.mcp.2008.12.008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Feasibility%20of%20single%20nucleotide%20polymorphism%20genotyping%20with%20a%20single-probe%20by%20time-resolved%20F%C3%B6rster%20resonance%20energy%20transfer&amp;author=A.%20Andreoni&amp;author=M.%20Bondani&amp;author=L.%20Nardo&amp;publication_year=2009&amp;journal=Mol.%20Cell.%20Probes&amp;volume=23&amp;pages=119-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mcp.2008.12.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mcp.2008.12.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mcp.2008.12.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19162175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=451164&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">83</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="451164"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f4%2f347%2f451164">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000308385400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F4%2F347%2F451164" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/451164'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Typing of a Polymorphic Human Gene Conferring Susceptibility to Insulin-Dependent Diabetes Mellitus by Picosecond-Resolved FRET on Non-Purified/Non-Amplified Genomic DNA" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f347%2f451164">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

