

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/6/237/350369","@type":"ScholarlyArticle","name":" A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in  E. coli ","datePublished":"2008-01-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm024","keywords":["biofilm","deletion mutant","<italic>E. coli</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/6/237/350369","author":[{"name":"Niba, Emma Tabe Eko","affiliation":"1  Graduate School of Science and Technology  ,  Kobe University  ,  Kobe, Hyogo 657-8501  ,  Japan","@type":"Person"},{"name":"Naka, Yoshiaki","affiliation":"1  Graduate School of Science and Technology  ,  Kobe University  ,  Kobe, Hyogo 657-8501  ,  Japan","@type":"Person"},{"name":"Nagase, Megumi","affiliation":"2  Faculty of Science, Department of Biology  ,  Kobe University  ,  1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501  ,  Japan","@type":"Person"},{"name":"Mori, Hirotada","affiliation":"3  Graduate School of Biological Sciences  ,  Nara Institute of Science and Technology  ,  8916-5 Takayama, Ikoma, Nara 630-0101  ,  Japan","@type":"Person"},{"name":"Kitakawa, Madoka","affiliation":"2  Faculty of Science, Department of Biology  ,  Kobe University  ,  1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501  ,  Japan","@type":"Person"}],"description":"Abstract.   Biofilm forming cells are distinctive from the well-investigated planktonic cells and exhibit a different type of gene expression. Several new Esche","pageStart":"237","pageEnd":"246","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/6/1/m_cover.gif?Expires=2147483647&Signature=xrXZb4IVgc6UYk71I2zNykdY6Hcu-0o1Rf4ZKOrE~uXoiG-mdTKPhqNdFmH4ehQ4MPZTammdyjXsQdbho5ga8D2dYThH31rrf-Dw8J~5baCDD40eMtwiKxfnRT5KmXh6eOvxPG2d7hiyAa-0fCQwEFgjHJmIrL7D5vi8GlbWCry8lK-KZdvXib567QEU1maDt4QxvUhkZaw1thCqeTyQskUYKnuKs7h1O3pZmmsqJ9w9uXpuPJ8erv6egAUV0yVmUjLtruvHDbGbsk4ZO66ovuaAC1YvJAr1owWUkixxXjbOtI3XbVkos8neDYqwS40rtKzsb-gOqgRAnMI-u8A-hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in  E. coli ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/6/1/m_cover.gif?Expires=2147483647&Signature=xrXZb4IVgc6UYk71I2zNykdY6Hcu-0o1Rf4ZKOrE~uXoiG-mdTKPhqNdFmH4ehQ4MPZTammdyjXsQdbho5ga8D2dYThH31rrf-Dw8J~5baCDD40eMtwiKxfnRT5KmXh6eOvxPG2d7hiyAa-0fCQwEFgjHJmIrL7D5vi8GlbWCry8lK-KZdvXib567QEU1maDt4QxvUhkZaw1thCqeTyQskUYKnuKs7h1O3pZmmsqJ9w9uXpuPJ8erv6egAUV0yVmUjLtruvHDbGbsk4ZO66ovuaAC1YvJAr1owWUkixxXjbOtI3XbVkos8neDYqwS40rtKzsb-gOqgRAnMI-u8A-hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in  E. coli " />
    <meta property="og:description" content="Abstract.   Biofilm forming cells are distinctive from the well-investigated planktonic cells and exhibit a different type of gene expression. Several new Esche" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/6/237/350369" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-01-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Niba, Emma Tabe Eko" /><meta name="citation_author_institution" content="Graduate School of Science and Technology , Kobe University , Kobe, Hyogo 657-8501 , Japan" /><meta name="citation_author" content="Naka, Yoshiaki" /><meta name="citation_author_institution" content="Graduate School of Science and Technology , Kobe University , Kobe, Hyogo 657-8501 , Japan" /><meta name="citation_author" content="Nagase, Megumi" /><meta name="citation_author_institution" content="Faculty of Science, Department of Biology , Kobe University , 1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501 , Japan" /><meta name="citation_author" content="Mori, Hirotada" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences , Nara Institute of Science and Technology , 8916-5 Takayama, Ikoma, Nara 630-0101 , Japan" /><meta name="citation_author" content="Kitakawa, Madoka" /><meta name="citation_author_institution" content="Faculty of Science, Department of Biology , Kobe University , 1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501 , Japan" /><meta name="citation_title" content=" A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in E. coli" /><meta name="citation_firstpage" content="237" /><meta name="citation_lastpage" content="246" /><meta name="citation_doi" content="10.1093/dnares/dsm024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Microbial biofilms; Annu. Rev. Microbiol.;  citation_year=1995;  citation_volume=49;  citation_pages=711-745; " /><meta name="citation_reference" content="citation_title=Bacterial resistance to antibiotics: the role of biofilms; Prog. Drug. Res.;  citation_year=1991;  citation_volume=37;  citation_pages=91-105; " /><meta name="citation_reference" content="citation_title=A complex transcription network controls the early stages of biofilm development by Escherichia coli; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=3731-3739; " /><meta name="citation_reference" content="citation_title=Global gene expression in Escherichia coli biofilms; Mol. Microbiol.;  citation_year=2003;  citation_volume=48;  citation_pages=253-267; " /><meta name="citation_reference" content="citation_title=Global impact of mature biofilm lifestyle on Escherichia coli K-12 gene expression; Mol. Microbiol.;  citation_year=2004;  citation_volume=51;  citation_pages=659-674; " /><meta name="citation_reference" content="citation_title=Gene expression in Escherichia coli biofilms; Appl. Microbiol. Biotechnol.;  citation_year=2004;  citation_volume=64;  citation_pages=515-524; " /><meta name="citation_reference" content="citation_title=Transcription in Escherichia coli PHL628 biofilms; FEMS Microbiol. Lett.;  citation_year=2007;  citation_volume=268;  citation_pages=237-243; " /><meta name="citation_reference" content="citation_title=Temporal gene-expression in Escherichia coli K-12 biofilms; Environ. Microbiol.;  citation_year=2007;  citation_volume=9;  citation_pages=332-346; " /><meta name="citation_reference" content="citation_title=Lessons from DNA microarray analysis: the gene expression profile of biofilms; Curr. Opin. Microbiol.;  citation_year=2005;  citation_volume=8;  citation_pages=222-227; " /><meta name="citation_reference" content="citation_title=Developmental pathway for biofilm formation in curli-producing Escherichia coli strains: role of flagella, curli and colanic acid; Environ. Microbiol.;  citation_year=2000;  citation_volume=2;  citation_pages=450-464; " /><meta name="citation_reference" content="citation_title=Development and maturation of Escherichia coli K-12 biofilms; Mol. Microbiol.;  citation_year=2003;  citation_volume=48;  citation_pages=933-946; " /><meta name="citation_reference" content="citation_title=Natural conjugative plasmids induce bacterial biofilm development; Nature;  citation_year=2001;  citation_volume=412;  citation_pages=442-445; " /><meta name="citation_reference" content="citation_title=Autoinducer 2 controls biofilm formation in Escherichia coli through a novel motility quorum-sensing regulator (MqsR, B3022); J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=305-316; " /><meta name="citation_reference" content="citation_title=Hha, YbaJ, and OmpA regulate Escherichia coli K12 biofilm formation and conjugation plasmids abolish motility; Biotechnol. Bioeng.;  citation_year=2006;  citation_volume=93;  citation_pages=188-200; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection; Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2;  citation_pages=E1-E11; " /><meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=6640-6645; " /><meta name="citation_reference" content="citation_title=Complete set of ORF clones of Escherichia coli ASKA library. A complete set of E. coli K-12 ORF archive: unique resources for biological research; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=291-299; " /><meta name="citation_reference" content="citation_title=Genetic approaches to study of biofilms; Methods Enzymol.;  citation_year=1999;  citation_volume=310;  citation_pages=91-109; " /><meta name="citation_reference" content="citation_title=Migration of bacteria in semisolid agar; Proc. Natl. Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=6973-6977; " /><meta name="citation_reference" content="citation_title=Antigen 43 and type 1 fimbriae determine colony morphology of Escherichia coli K-12; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=1089-1095; " /><meta name="citation_reference" content="citation_title=Identification of genes involved in biofilm formation, In; citation_publisher=Bioline, Cardiff; Biofilm Community Interactions: Chance or Necessity;  citation_year=2001;  citation_pages=119-129; " /><meta name="citation_reference" content="citation_title=Characterization of six lipoproteins in the sigma E regulon; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=4552-4561; " /><meta name="citation_reference" content="citation_title=Systematic characterization of E. coli genes/ORFs affecting biofilm formation; FEMS Microbiol. Lett.;  citation_year=2003;  citation_volume=225;  citation_pages=107-114; " /><meta name="citation_reference" content="citation_title=Role of bacterial cell surface structures in Escherichia coli biofilm formation; Res. Microbiol.;  citation_year=2005;  citation_volume=156;  citation_pages=626-633; " /><meta name="citation_reference" content="citation_title=Coupling of flagellar gene expression to flagellar Assembly in Salmonella enterica Serovar Typhimurium and Escherichia coli; Microbiol. Mol. Biol. Rev.;  citation_year=2000;  citation_volume=64;  citation_pages=694-708; " /><meta name="citation_reference" content="citation_title=Molecular characterization of the Salmonella typhimurium flhB operon and its protein products; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=7630-7637; " /><meta name="citation_reference" content="citation_title=FliL is a membrane-associated component of the flagellar basal body of Salmonella; Microbiology;  citation_year=1999;  citation_volume=145;  citation_pages=1769-1775; " /><meta name="citation_reference" content="citation_title=Substrate complexes and domain organization of the Salmonella flagellar export chaperones FlgN and FliT; Mol. Microbiol.;  citation_year=2001;  citation_volume=39;  citation_pages=781-791; " /><meta name="citation_reference" content="citation_title=Salmonella typhimurium flhE , a conserved flagellar regulon gene required for swarming; Microbiology;  citation_year=2007;  citation_volume=153;  citation_issue=Pt 2;  citation_pages=541-547; " /><meta name="citation_reference" content="citation_title=Mutational analysis of the disulfide catalysts DsbA and DsbB; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=1504-1510; " /><meta name="citation_reference" content="citation_title=Mutants in disulfide bond formation that disrupt flagellar assembly in Escherichia coli; Proc. Natl. Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=1043-1047; " /><meta name="citation_reference" content="citation_title=Lesions in two Escherichia coli type 1 pilus genes alter pilus number and length without affecting receptor binding; J. Bacteriol.;  citation_year=1992;  citation_volume=174;  citation_pages=5923-5935; " /><meta name="citation_reference" content="citation_title=Interaction of FimB and FimE with the fim switch that controls the phase variation of type 1 fimbriae in Escherichia coli K-12; Mol. Microbiol.;  citation_year=1996;  citation_volume=21;  citation_pages=725-738; " /><meta name="citation_reference" content="citation_title=The Product of the fimI gene is necessary for Escherichia coli type 1 pilus biosynthesis; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=5007-5011; " /><meta name="citation_reference" content="citation_title=Curli biogenesis and function; Annu. Rev. Microbiol.;  citation_year=2006;  citation_volume=60;  citation_pages=131-147; " /><meta name="citation_reference" content="citation_title=Metabolic block at early stages of the glycolytic pathway activates the Rcs phosphorelay system via increased synthesis of dTDP-glucose in Escherichia coli; Mol. Microbiol.;  citation_year=2004;  citation_volume=51;  citation_pages=1117-1128; " /><meta name="citation_reference" content="citation_title=Molecular basis for structural diversity in the core regions of the lipopolysaccharides of Escherichia coli and Salmonella enterica; Mol. Microbiol.;  citation_year=1998;  citation_volume=30;  citation_pages=221-232; " /><meta name="citation_reference" content="citation_title=Genetics of lipopolysaccharide biosynthesis in enteric bacteria; Microbiol. Rev.;  citation_year=1993;  citation_volume=57;  citation_pages=655-682; " /><meta name="citation_reference" content="citation_title=The transcriptional antiterminator RfaH represses biofilm formation in Escherichia coli; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=1316-1331; " /><meta name="citation_reference" content="citation_title=The surA periplasmic PPIase lacking its paruvilin domains function in vivo and has chaperone activities; EMBO J.;  citation_year=2001;  citation_volume=20;  citation_pages=285-294; " /><meta name="citation_reference" content="citation_title=SurA, a periplasmic protein with peptidyl-prolyl isomerase activity, participates in the assembly of outer membrane porins; Genes Dev.;  citation_year=1996;  citation_volume=10;  citation_pages=3170-3182; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/6/237/7525612/dsm024.pdf" /><meta name="description" content="Abstract.   Biofilm forming cells are distinctive from the well-investigated planktonic cells and exhibit a different type of gene expression. Several new Esche" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/6/237/350369" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/6/237/350369" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/6/237/350369");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm024';
    </script>
<title>
	 A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in E. coli | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./350369" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcRbPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWJhLCBFbW1hIFRhYmUgRWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgLCBLb2JlIFVuaXZlcnNpdHkgLCBLb2JlLCBIeW9nbyA2NTctODUwMSAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYSwgWW9zaGlha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAsIEtvYmUgVW5pdmVyc2l0eSAsIEtvYmUsIEh5b2dvIDY1Ny04NTAxICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWdhc2UsIE1lZ3VtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgU2NpZW5jZSwgRGVwYXJ0bWVudCBvZiBCaW9sb2d5ICwgS29iZSBVbml2ZXJzaXR5ICwgMS0xIFJva2tvZGFpLCBOYWRhLWt1LCBLb2JlLCBIeW9nbyA2NTctODUwMSAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaSwgSGlyb3RhZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcyAsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgLCA4OTE2LTUgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXRha2F3YSwgTWFkb2thIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBTY2llbmNlLCBEZXBhcnRtZW50IG9mIEJpb2xvZ3kgLCBLb2JlIFVuaXZlcnNpdHkgLCAxLTEgUm9ra29kYWksIE5hZGEta3UsIEtvYmUsIEh5b2dvIDY1Ny04NTAxICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IiBBIEdlbm9tZS13aWRlIEFwcHJvYWNoIHRvIElkZW50aWZ5IHRoZSBHZW5lcyBJbnZvbHZlZCBpbiBCaW9maWxtIEZvcm1hdGlvbiBpbiBFLiBjb2xpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc20wMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNy8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBiaW9maWxtczsgQW5udS4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9NzExLTc0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIHJlc2lzdGFuY2UgdG8gYW50aWJpb3RpY3M6IHRoZSByb2xlIG9mIGJpb2ZpbG1zOyBQcm9nLiBEcnVnLiBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9OTEtMTA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBsZXggdHJhbnNjcmlwdGlvbiBuZXR3b3JrIGNvbnRyb2xzIHRoZSBlYXJseSBzdGFnZXMgb2YgYmlvZmlsbSBkZXZlbG9wbWVudCBieSBFc2NoZXJpY2hpYSBjb2xpOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTM3MzEtMzczOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGdlbmUgZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIGJpb2ZpbG1zOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0yNTMtMjY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgaW1wYWN0IG9mIG1hdHVyZSBiaW9maWxtIGxpZmVzdHlsZSBvbiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgZ2VuZSBleHByZXNzaW9uOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz02NTktNjc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saSBiaW9maWxtczsgQXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTUxNS01MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gaW4gRXNjaGVyaWNoaWEgY29saSBQSEw2MjggYmlvZmlsbXM7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz0yMzctMjQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UZW1wb3JhbCBnZW5lLWV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saSBLLTEyIGJpb2ZpbG1zOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMzItMzQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZXNzb25zIGZyb20gRE5BIG1pY3JvYXJyYXkgYW5hbHlzaXM6IHRoZSBnZW5lIGV4cHJlc3Npb24gcHJvZmlsZSBvZiBiaW9maWxtczsgQ3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjIyLTIyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnRhbCBwYXRod2F5IGZvciBiaW9maWxtIGZvcm1hdGlvbiBpbiBjdXJsaS1wcm9kdWNpbmcgRXNjaGVyaWNoaWEgY29saSBzdHJhaW5zOiByb2xlIG9mIGZsYWdlbGxhLCBjdXJsaSBhbmQgY29sYW5pYyBhY2lkOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00NTAtNDY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgbWF0dXJhdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgYmlvZmlsbXM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTkzMy05NDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgY29uanVnYXRpdmUgcGxhc21pZHMgaW5kdWNlIGJhY3RlcmlhbCBiaW9maWxtIGRldmVsb3BtZW50OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDEyOyAgY2l0YXRpb25fcGFnZXM9NDQyLTQ0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b2luZHVjZXIgMiBjb250cm9scyBiaW9maWxtIGZvcm1hdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIHRocm91Z2ggYSBub3ZlbCBtb3RpbGl0eSBxdW9ydW0tc2Vuc2luZyByZWd1bGF0b3IgKE1xc1IsIEIzMDIyKTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODg7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMzE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaGEsIFliYUosIGFuZCBPbXBBIHJlZ3VsYXRlIEVzY2hlcmljaGlhIGNvbGkgSzEyIGJpb2ZpbG0gZm9ybWF0aW9uIGFuZCBjb25qdWdhdGlvbiBwbGFzbWlkcyBhYm9saXNoIG1vdGlsaXR5OyBCaW90ZWNobm9sLiBCaW9lbmcuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MTg4LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgSy0xMiBpbi1mcmFtZSwgc2luZ2xlLWdlbmUga25vY2tvdXQgbXV0YW50czogdGhlIEtlaW8gY29sbGVjdGlvbjsgTW9sLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9RTEtRTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmUtc3RlcCBpbmFjdGl2YXRpb24gb2YgY2hyb21vc29tYWwgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saSBLLTEyIHVzaW5nIFBDUiBwcm9kdWN0czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz02NjQwLTY2NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHNldCBvZiBPUkYgY2xvbmVzIG9mIEVzY2hlcmljaGlhIGNvbGkgQVNLQSBsaWJyYXJ5LiBBIGNvbXBsZXRlIHNldCBvZiBFLiBjb2xpIEstMTIgT1JGIGFyY2hpdmU6IHVuaXF1ZSByZXNvdXJjZXMgZm9yIGJpb2xvZ2ljYWwgcmVzZWFyY2g7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjkxLTI5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhcHByb2FjaGVzIHRvIHN0dWR5IG9mIGJpb2ZpbG1zOyBNZXRob2RzIEVuenltb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTMxMDsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWlncmF0aW9uIG9mIGJhY3RlcmlhIGluIHNlbWlzb2xpZCBhZ2FyOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTY5NzMtNjk3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWdlbiA0MyBhbmQgdHlwZSAxIGZpbWJyaWFlIGRldGVybWluZSBjb2xvbnkgbW9ycGhvbG9neSBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9MTA4OS0xMDk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBnZW5lcyBpbnZvbHZlZCBpbiBiaW9maWxtIGZvcm1hdGlvbiwgSW47IGNpdGF0aW9uX3B1Ymxpc2hlcj1CaW9saW5lLCBDYXJkaWZmOyBCaW9maWxtIENvbW11bml0eSBJbnRlcmFjdGlvbnM6IENoYW5jZSBvciBOZWNlc3NpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl9wYWdlcz0xMTktMTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHNpeCBsaXBvcHJvdGVpbnMgaW4gdGhlIHNpZ21hIEUgcmVndWxvbjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz00NTUyLTQ1NjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgY2hhcmFjdGVyaXphdGlvbiBvZiBFLiBjb2xpIGdlbmVzL09SRnMgYWZmZWN0aW5nIGJpb2ZpbG0gZm9ybWF0aW9uOyBGRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjI1OyAgY2l0YXRpb25fcGFnZXM9MTA3LTExNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBiYWN0ZXJpYWwgY2VsbCBzdXJmYWNlIHN0cnVjdHVyZXMgaW4gRXNjaGVyaWNoaWEgY29saSBiaW9maWxtIGZvcm1hdGlvbjsgUmVzLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTYyNi02MzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdXBsaW5nIG9mIGZsYWdlbGxhciBnZW5lIGV4cHJlc3Npb24gdG8gZmxhZ2VsbGFyIEFzc2VtYmx5IGluIFNhbG1vbmVsbGEgZW50ZXJpY2EgU2Vyb3ZhciBUeXBoaW11cml1bSBhbmQgRXNjaGVyaWNoaWEgY29saTsgTWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz02OTQtNzA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgU2FsbW9uZWxsYSB0eXBoaW11cml1bSBmbGhCIG9wZXJvbiBhbmQgaXRzIHByb3RlaW4gcHJvZHVjdHM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9NzYzMC03NjM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbGlMIGlzIGEgbWVtYnJhbmUtYXNzb2NpYXRlZCBjb21wb25lbnQgb2YgdGhlIGZsYWdlbGxhciBiYXNhbCBib2R5IG9mIFNhbG1vbmVsbGE7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDU7ICBjaXRhdGlvbl9wYWdlcz0xNzY5LTE3NzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1YnN0cmF0ZSBjb21wbGV4ZXMgYW5kIGRvbWFpbiBvcmdhbml6YXRpb24gb2YgdGhlIFNhbG1vbmVsbGEgZmxhZ2VsbGFyIGV4cG9ydCBjaGFwZXJvbmVzIEZsZ04gYW5kIEZsaVQ7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTc4MS03OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbG1vbmVsbGEgdHlwaGltdXJpdW0gZmxoRSAsIGEgY29uc2VydmVkIGZsYWdlbGxhciByZWd1bG9uIGdlbmUgcmVxdWlyZWQgZm9yIHN3YXJtaW5nOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25faXNzdWU9UHQgMjsgIGNpdGF0aW9uX3BhZ2VzPTU0MS01NDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uYWwgYW5hbHlzaXMgb2YgdGhlIGRpc3VsZmlkZSBjYXRhbHlzdHMgRHNiQSBhbmQgRHNiQjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz0xNTA0LTE1MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGFudHMgaW4gZGlzdWxmaWRlIGJvbmQgZm9ybWF0aW9uIHRoYXQgZGlzcnVwdCBmbGFnZWxsYXIgYXNzZW1ibHkgaW4gRXNjaGVyaWNoaWEgY29saTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0xMDQzLTEwNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlc2lvbnMgaW4gdHdvIEVzY2hlcmljaGlhIGNvbGkgdHlwZSAxIHBpbHVzIGdlbmVzIGFsdGVyIHBpbHVzIG51bWJlciBhbmQgbGVuZ3RoIHdpdGhvdXQgYWZmZWN0aW5nIHJlY2VwdG9yIGJpbmRpbmc7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTc0OyAgY2l0YXRpb25fcGFnZXM9NTkyMy01OTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGlvbiBvZiBGaW1CIGFuZCBGaW1FIHdpdGggdGhlIGZpbSBzd2l0Y2ggdGhhdCBjb250cm9scyB0aGUgcGhhc2UgdmFyaWF0aW9uIG9mIHR5cGUgMSBmaW1icmlhZSBpbiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTcyNS03Mzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQcm9kdWN0IG9mIHRoZSBmaW1JIGdlbmUgaXMgbmVjZXNzYXJ5IGZvciBFc2NoZXJpY2hpYSBjb2xpIHR5cGUgMSBwaWx1cyBiaW9zeW50aGVzaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTg1OyAgY2l0YXRpb25fcGFnZXM9NTAwNy01MDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXJsaSBiaW9nZW5lc2lzIGFuZCBmdW5jdGlvbjsgQW5udS4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MTMxLTE0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWJvbGljIGJsb2NrIGF0IGVhcmx5IHN0YWdlcyBvZiB0aGUgZ2x5Y29seXRpYyBwYXRod2F5IGFjdGl2YXRlcyB0aGUgUmNzIHBob3NwaG9yZWxheSBzeXN0ZW0gdmlhIGluY3JlYXNlZCBzeW50aGVzaXMgb2YgZFREUC1nbHVjb3NlIGluIEVzY2hlcmljaGlhIGNvbGk7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTExMTctMTEyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGJhc2lzIGZvciBzdHJ1Y3R1cmFsIGRpdmVyc2l0eSBpbiB0aGUgY29yZSByZWdpb25zIG9mIHRoZSBsaXBvcG9seXNhY2NoYXJpZGVzIG9mIEVzY2hlcmljaGlhIGNvbGkgYW5kIFNhbG1vbmVsbGEgZW50ZXJpY2E7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIyMS0yMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWNzIG9mIGxpcG9wb2x5c2FjY2hhcmlkZSBiaW9zeW50aGVzaXMgaW4gZW50ZXJpYyBiYWN0ZXJpYTsgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9NjU1LTY4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBhbnRpdGVybWluYXRvciBSZmFIIHJlcHJlc3NlcyBiaW9maWxtIGZvcm1hdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTEzMTYtMTMzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN1ckEgcGVyaXBsYXNtaWMgUFBJYXNlIGxhY2tpbmcgaXRzIHBhcnV2aWxpbiBkb21haW5zIGZ1bmN0aW9uIGluIHZpdm8gYW5kIGhhcyBjaGFwZXJvbmUgYWN0aXZpdGllczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4NS0yOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1ckEsIGEgcGVyaXBsYXNtaWMgcHJvdGVpbiB3aXRoIHBlcHRpZHlsLXByb2x5bCBpc29tZXJhc2UgYWN0aXZpdHksIHBhcnRpY2lwYXRlcyBpbiB0aGUgYXNzZW1ibHkgb2Ygb3V0ZXIgbWVtYnJhbmUgcG9yaW5zOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzE3MC0zMTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvNi8yMzcvNzUyNTYxMi9kc20wMjQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBCaW9maWxtIGZvcm1pbmcgY2VsbHMgYXJlIGRpc3RpbmN0aXZlIGZyb20gdGhlIHdlbGwtaW52ZXN0aWdhdGVkIHBsYW5rdG9uaWMgY2VsbHMgYW5kIGV4aGliaXQgYSBkaWZmZXJlbnQgdHlwZSBvZiBnZW5lIGV4cHJlc3Npb24uIFNldmVyYWwgbmV3IEVzY2hlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTAzNjkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkEzN/eJRmwddj5nls/Jj/skzDq4A=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f6%2f237%2f350369" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f6%2f237%2f350369" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13551" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/6/1/m_cover.gif?Expires=2147483647&amp;Signature=xrXZb4IVgc6UYk71I2zNykdY6Hcu-0o1Rf4ZKOrE~uXoiG-mdTKPhqNdFmH4ehQ4MPZTammdyjXsQdbho5ga8D2dYThH31rrf-Dw8J~5baCDD40eMtwiKxfnRT5KmXh6eOvxPG2d7hiyAa-0fCQwEFgjHJmIrL7D5vi8GlbWCry8lK-KZdvXib567QEU1maDt4QxvUhkZaw1thCqeTyQskUYKnuKs7h1O3pZmmsqJ9w9uXpuPJ8erv6egAUV0yVmUjLtruvHDbGbsk4ZO66ovuaAC1YvJAr1owWUkixxXjbOtI3XbVkos8neDYqwS40rtKzsb-gOqgRAnMI-u8A-hQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/6">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="42022104">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022104">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022106">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022106">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022112">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022112">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022140">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022140">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022206">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022206">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022209">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022209">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="42022211">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#42022211">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/6/235/351236">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/6/247/350732">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in <em>E. coli</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Emma Tabe Eko Niba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Science and Technology</div> , <div class="insititution">Kobe University</div> , <div class="addr-line">Kobe, Hyogo 657-8501</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Emma+Tabe+Eko+Niba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Niba E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Niba E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Emma Tabe Eko Niba</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiaki Naka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Science and Technology</div> , <div class="insititution">Kobe University</div> , <div class="addr-line">Kobe, Hyogo 657-8501</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiaki+Naka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Naka Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Naka Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiaki Naka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Megumi Nagase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Science, Department of Biology</div> , <div class="insititution">Kobe University</div> , <div class="addr-line">1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Megumi+Nagase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagase M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagase M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Megumi Nagase</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirotada Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Biological Sciences</div> , <div class="insititution">Nara Institute of Science and Technology</div> , <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirotada+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirotada Mori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Madoka Kitakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Science, Department of Biology</div> , <div class="insititution">Kobe University</div> , <div class="addr-line">1-1 Rokkodai, Nada-ku, Kobe, Hyogo 657-8501</div> , <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div> To whom correspondence should be addressed. Tel. +81 78-803-5724 . Fax. <div class="fax">+81 78-803-5783</div> . E-mail: <a href="mailto:madoka@kobe-u.ac.jp" target="_blank">madoka@kobe-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Madoka+Kitakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kitakawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kitakawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Madoka Kitakawa</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 6, 1 January 2007, Pages 237–246, <a href='https://doi.org/10.1093/dnares/dsm024'>https://doi.org/10.1093/dnares/dsm024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 January 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 October 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 November 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="350369" href="/dnaresearch/article-pdf/14/6/237/7525612/dsm024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Emma Tabe Eko Niba, Yoshiaki Naka, Megumi Nagase, Hirotada Mori, Madoka Kitakawa;   A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in <em>E. coli</em>, <em>DNA Research</em>, Volume 14, Issue 6, 1 January 2007, Pages 237–246, <a href='https://doi.org/10.1093/dnares/dsm024'>https://doi.org/10.1093/dnares/dsm024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/350369?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/350369?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20A%20Genome-wide%20Approach%20to%20Identify%20the%20Genes%20Involved%20in%20Biofilm%20Formation%20in%20%26lt%3Bem%26gt%3BE.%20coli%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2008-01-07&amp;volumeNum=14&amp;issueNum=6&amp;author=Niba%2C%20Emma%20Tabe%20Eko%3B%20Naka%2C%20Yoshiaki&amp;startPage=237&amp;endPage=246&amp;contentId=10.1093%2Fdnares%2Fdsm024&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f6%2f237%2f350369" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=42022104 id="42022104" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> Biofilm forming cells are distinctive from the well-investigated planktonic cells and exhibit a different type of gene expression. Several new <em>Escherichia coli</em> genes related to biofilm formation have recently been identified through genomic approaches such as DNA microarray analysis. However, many others involved in this process might have escaped detection due to poor expression, regulatory mechanism, or genetic backgrounds. Here, we screened a collection of single-gene deletion mutants of <em>E. coli</em> named ‘Keio collection’ to identify genes required for biofilm formation. Of the 3985 mutants of non-essential genes in the collection thus examined, 110 showed a reduction in biofilm formation nine of which have not been well characterized yet. Systematic and quantitative analysis revealed the involvement of genes of various functions and reinforced the importance in biofilm formation of the genes for cell surface structures and cell membrane. Characterization of the nine mutants of function-unknown genes indicated that some of them, such as <em>yfgA</em> that genetically interacts with a periplasmic chaperone gene <em>surA</em> together with <em>yciB</em> and <em>yciM</em> , might be required for the integrity of outer membrane. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="biofilm">biofilm</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;deletion mutant&quot;">deletion mutant</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;E. coli&quot;"><em>E. coli</em></a></div>                            <h2 scrollto-destination=42022106 id="42022106" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p> Bacteria have evolved elaborate mechanisms for adhering to and colonizing solid surfaces, thereby establishing microbial communities known as biofilms. <sup><a reveal-id="DSM024C1" data-open="DSM024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> These represent a distinct lifestyle for bacteria that provides protection from deleterious conditions, thereby raising various problems to our life such as causing persistent and chronic human infections <sup><a reveal-id="DSM024C2" data-open="DSM024C2" class="link link-ref link-reveal xref-bibr">2</a></sup> or contamination of food products. </p><p> The transition from a planktonic to a sedentary biofilm mode of life requires the coordinated regulation of genes involved in the development of biofilms, which is an interesting theme to investigate the intricate network of signal transduction for gene expression in bacterial cells. <sup><a reveal-id="DSM024C3" data-open="DSM024C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The latter lifestyle would require the expression of genes that have not been investigated in studies with planktonic cells. Recent analyses of biofilms using DNA microarray <sup><a reveal-id="DSM024C4  DSM024C6" data-open="DSM024C4  DSM024C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> revealed that hundreds of genes including many of uncharacterized are differentially expressed in biofilms, which would provide insights into the genetic basis for biofilm formation. However, the agreement for differential gene expression is limited among these studies, probably reflecting differences in experimental conditions as well as the nature of biofilm itself. <sup><a reveal-id="DSM024C7" data-open="DSM024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The environment within the biofilm is heterogeneous and biofilm formation is a dynamic process. <sup><a reveal-id="DSM024C8" data-open="DSM024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Furthermore, the different expression of some genes may be due to differences in growth of planktonic cells used as control. <sup><a reveal-id="DSM024C9" data-open="DSM024C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Genetic analysis revealed that surface structures such as flagella and specific outer-membrane adhesins, Type 1, and curli fimbriae of <em>Escherichia coli</em> are important for biofilm formation, though they are not indispensable. <sup><a reveal-id="DSM024C10" data-open="DSM024C10" class="link link-ref link-reveal xref-bibr">10</a> , <a reveal-id="DSM024C11" data-open="DSM024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The extent of biofilm formation and the effect of mutation seem to be variable depending on the strains used. <sup><a reveal-id="DSM024C12  DSM024C14" data-open="DSM024C12  DSM024C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup></p><p> Keeping the above problems in mind, we started identifying genes that cause upon disruption a reduced biofilmformation in the same genetic background. In this study, we used a collection of single-gene deletion mutants of all non-essential genes of <em>E. coli</em> called ‘Keio collection’ <sup><a reveal-id="DSM024C15" data-open="DSM024C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and performed a quantitative analysis of their biofilm formation. </p>                            <h2 scrollto-destination=42022112 id="42022112" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=42022115 id="42022115" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and plasmid</h3>
<p><em>Escherichia coli</em> K-12 strain BW25113 and isogenic deletion mutants of the Keio collection <sup><a reveal-id="DSM024C15" data-open="DSM024C15" class="link link-ref link-reveal xref-bibr">15</a></sup> were used. KR0401 is a derivative of BW25113 spontaneously obtained (see text). Kanamycin sensitive derivatives of deletion mutants were constructed using plasmid pCP20 as described. <sup><a reveal-id="DSM024C16" data-open="DSM024C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Plasmids p <em>yfgA</em> is one clone of ASKA library, a set of plasmid clones containing all predicted ORFs of <em>E. coli</em> . <sup><a reveal-id="DSM024C17" data-open="DSM024C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=42022119 id="42022119" class="section-title" data-legacy-id=s2b>2.2. Assay for biofilm formation</h3>
<div class="block-child-p"> The procedure to characterize the biofilm-forming capacity of bacteria described by O'Toole et al. <sup><a reveal-id="DSM024C18" data-open="DSM024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> was generally followed. A 3 µL of cells from overnight culture was inoculated in 100 µL of Luria–Bertani (LB) medium and biofilm was allowed to form in 96-well polystyrene microtiter plates (Bio Medical Equipment, Japan) at 25°C for 24 h. Growth of cell was measured by reading the absorbance (OD <sub>650</sub> ) of each well using a plate reader (Molecular Device, USA). Medium was discarded and individual wells were stained with 0.1% crystal violet (CV). Subsequently, the amount of cells attached was estimated by measuring the absorbance (OD <sub>650</sub> ) of CV dissolved in 0.5% SDS by the plate reader. Then, the value of biofilm was normalized according to the amount of cells. This value (CV/growth) was termed ‘relative biofilm’, and for each strain, it was indicated as the ratio of its relative biofilm to that of wild type (KR0401). <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/dsm02403.gif?Expires=2147483647&Signature=SE3a2scIVTnLjlf7-4qR9YmVL-DNtihIcqaJK7rdUiF7zx0rokfBHBUTikAoPTTybugSDOVavHCL11jzqYtBHXVO-7Ra3MSO4ath0sODK3Io4i4MFlALKPTiMXH6yuXbquMeHz8it1KmWUVFK4oczG1EHE96vdPWGlx1ipBLMM3P0iqKxaEcDyR5HTMwtVIZJQB88dgf00RARL78uJ7lzAlbjytR4eQBWID6YxozbFoCj6byhtvVaWCWWH29gigalFz~fwMjRyh05tyBgC2aA7vLsDKFG2gb0ZrXUKkTHlZwUsEXv~q98ZciW-P~q5z~w9w2Nd6i-J2DNlk9hm6KAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm02403.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/m_dsm02403.gif?Expires=2147483647&Signature=C0wLPz9CzRvMPPaDAqIVt3jFZYpBixYtvG5l6P72HCifwPEjvLKNYXTcchTd6LxpPmZkicywZhyicv-~y6Qn6Um5Bo7JeiafHn~R~zcZkl5YbPPziDPS8tncaQqvM4QYBbJYEMPGY-AImFt32FcKH4H3RlHT-SckVlA4UYhA7Ww09NoaLU0Frie2Tb10OyerRp7HUAdg2AkSq~X1Ka0IIHmMAAUcjCqux8N2OCekxoQLQjW12RCuG~eoZV0PN8qjkrmshmQEG6ZEUDnIFdHzezcZjwQCemO86v0P41UczPuOmEVghb0yBo36Uw-RfYJQ7BUaorzw1G-q8~qoSzTGSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm02403.gif" /></a></div></div></div></div>                            <h3 scrollto-destination=42022126 id="42022126" class="section-title" data-legacy-id=s2c>2.3. Phenotype assays</h3>
<p> Motility was observed essentially as described by Wolfe and Berg. <sup><a reveal-id="DSM024C19" data-open="DSM024C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Three microlitres of overnight cultures were spotted on semi-solid agar plates (1% Tryptone, 0.5% NaCl, and 0.3% Difco agar) and incubated at 30°C for 6 h. The diameter of swarming colony was measured and clones that showed more than 50% of the wild-type control as well as those that were less than 50% but apparently motile were scored as positive and intermediate phenotype, respectively. Mannose-binding Type 1 fimbriae production was examined by the ability of cells to agglutinate budding yeast cells. <sup><a reveal-id="DSM024C20" data-open="DSM024C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Equal volumes of bacterial and yeast cultures were mixed in a titer plate and observed for agglutination either with the naked eyes or under the microscope. Clones that showed no visible aggregation were evaluated as Type 1 fimbriae deficient. Curli fimbriae production of colonies was judged on CFA plates containing 0.1 mg/mL of Congo Red dye. <sup><a reveal-id="DSM024C21" data-open="DSM024C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Colonies were observed for uptake of the red colour after 3 days incubation at 25°C. Clones that were stained as much as wild-type control cells were scored as curli positive and those that remained uncoloured as deletion mutants of <em>csgA</em> were considered negative. Cells that showed in between colour level were evaluated as intermediate. </p>                            <h3 scrollto-destination=42022131 id="42022131" class="section-title" data-legacy-id=s2d>2.4. Preparation and analysis of OMPs</h3>
<p> Strains were grown in 10 mL LB medium containing 30 µg/mL kanamycin to an OD <sub>600</sub> of 0.8–1.0 and the OD <sub>600</sub> of each culture recorded at the time of harvest. OMPs were prepared following the procedure described by Onufryk et al. <sup><a reveal-id="DSM024C22" data-open="DSM024C22" class="link link-ref link-reveal xref-bibr">22</a></sup> . Finally, the sample was suspended in 40 µL SDS buffer and a portion corresponding to 2.0 OD <sub>600</sub> of cells was analyzed by loading on a 10% polyacrylamide-SDS gel. </p>                            <h3 scrollto-destination=42022135 id="42022135" class="section-title" data-legacy-id=s2e>2.5. Synthetic phenotype analysis of double deletion mutants</h3>
<p> To examine the effect of introducing a second deletion mutation, the kanamycin resistance gene was first eliminated from deletion mutants through FLP recombination mediated by pCP20. <sup><a reveal-id="DSM024C16" data-open="DSM024C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The resultant Km <sup>S</sup> deletion mutants were used as recipients and another deletion mutation harbouring the antibiotic resistant cassette was introduced by P1 transduction. Appearance of Km <sup>R</sup> transductants was monitored after overnight incubation at 37°C. </p>                            <h2 scrollto-destination=42022140 id="42022140" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=42022143 id="42022143" class="section-title" data-legacy-id=s3a>3.1. Screening of genes involved in biofilm formation</h3>
<p> To investigate the effect of deletion of individual genes on biofilm formation, mutants were grown in LB medium and the amount of cells attached to 96-well polystyrene microtiter plates was measured. The results obtained revealed that the growth of deletion mutants was quite variable (Fig.  <a reveal-id="DSM024F1" data-open="DSM024F1" class="link link-reveal link-table xref-fig">1</a> ), which might affect the amount of attached cells. For this reason, we normalized the value of biofilm formation against the cell growth and termed it as ‘relative biofilm’ as described in experimental procedures. Throughout these analyses, we used strain KR0401, a derivative of BW25113 (parental strain of Keio collection), as a control and the relative biofilm formed by each mutant was expressed as a ratio to that of KR0401 to minimize the experimental fluctuation. KR0401 that was obtained spontaneously from BW25113 showed a stable and higher level of biofilm formation compared with BW25113 itself (data not shown). Although it is unclear why the biofilm formation of BW25113 varies from one batch to another, this feature might explain the difference in the biofilm formation observed between two deletion strains of the same gene within the Keio collection. </p>                        <a id="42022148" scrollto-destination="42022148"></a>
<div id="DSM024F1" data-id="DSM024F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/m_dsm02401.jpeg?Expires=2147483647&Signature=NGwVtgm4j~9y0vM-UZelJ4Vi28Yk8wQSG9LAuBWbiquG0P30gWZK99ZwmfteSRDoZ8lc12w709kD-mVxfdD6qdZm85KqBwnESM2a1ewYGkuii1lRfkZKMmhPuQwpLdZ3HX94f1dKSXDNTKB1w69R72ZvwiHpysOSYAv2ZEc8RunLPU5X-rSeG1X6GvP1xbOWSlzy7H9iPHvPBekooeXB7yttoyDX~N~N66hCQctd~JEgK3IgGhQ66sbqts-bfG41PMWHexaDwmkb2LUSnDdXNyUpgoKnsWrLQanPyp2NROTWSmyGi3kvLsISTAqTxszdhW1HndUb9URgGXdXrMqxQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Correlation between the growth and the amount of attached cells of deletion mutants of the Keio collection. Gray squares represent the selected deletion clones listed in Table  1 shown with the total mutants examined (black squares), parental strain BW25113 (white circle), and KR0401 used as a control (white triangle). The amounts of attached cells (cv) relative to that of the control are plotted against growth. " path-from-xml="dsm02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42022148/dsm02401.jpeg" path-from-xml="dsm02401.jpeg" target="_blank">View large</a><a section="42022148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/dsm02401.jpeg?Expires=2147483647&Signature=oohNhuj0NEL1CAQbrXYMMjR~VapGygPK0G0HJ3VL5JHlRVSl0ElDCqc69J5N8rXIVg5ekmJp986D0uCD8B1-W2h0sw1GtkpZ-XF7nM23A1Anl0H9aK5lzfCYFTPvwlUI62oT7slRwiQmkl9iwTY12eiEVCKPIjQFv8UcpPfY9wfKmUFlbWmbSGcvzsA9RMRrLxqTZU6HZtqOMMIySxFI90YNxkrhFPoSWQsJEzCpb3INYU2NN6y64OUW6PloSdWAvlgtCdsG8jgixRZu7oauYlQbWrYNhpQPg-15TX0p9J4EIZ31JSqMjH1i0UmstLPE0zwnaMlRAQ-dupTa2TEOVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42022148&ar=350369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Correlation between the growth and the amount of attached cells of deletion mutants of the Keio collection. Gray squares represent the selected deletion clones listed in Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> shown with the total mutants examined (black squares), parental strain BW25113 (white circle), and KR0401 used as a control (white triangle). The amounts of attached cells (cv) relative to that of the control are plotted against growth. </p></div></div><div content-id="DSM024F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/m_dsm02401.jpeg?Expires=2147483647&Signature=NGwVtgm4j~9y0vM-UZelJ4Vi28Yk8wQSG9LAuBWbiquG0P30gWZK99ZwmfteSRDoZ8lc12w709kD-mVxfdD6qdZm85KqBwnESM2a1ewYGkuii1lRfkZKMmhPuQwpLdZ3HX94f1dKSXDNTKB1w69R72ZvwiHpysOSYAv2ZEc8RunLPU5X-rSeG1X6GvP1xbOWSlzy7H9iPHvPBekooeXB7yttoyDX~N~N66hCQctd~JEgK3IgGhQ66sbqts-bfG41PMWHexaDwmkb2LUSnDdXNyUpgoKnsWrLQanPyp2NROTWSmyGi3kvLsISTAqTxszdhW1HndUb9URgGXdXrMqxQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Correlation between the growth and the amount of attached cells of deletion mutants of the Keio collection. Gray squares represent the selected deletion clones listed in Table  1 shown with the total mutants examined (black squares), parental strain BW25113 (white circle), and KR0401 used as a control (white triangle). The amounts of attached cells (cv) relative to that of the control are plotted against growth. " path-from-xml="dsm02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42022148/dsm02401.jpeg" path-from-xml="dsm02401.jpeg" target="_blank">View large</a><a section="42022148" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/dsm02401.jpeg?Expires=2147483647&Signature=oohNhuj0NEL1CAQbrXYMMjR~VapGygPK0G0HJ3VL5JHlRVSl0ElDCqc69J5N8rXIVg5ekmJp986D0uCD8B1-W2h0sw1GtkpZ-XF7nM23A1Anl0H9aK5lzfCYFTPvwlUI62oT7slRwiQmkl9iwTY12eiEVCKPIjQFv8UcpPfY9wfKmUFlbWmbSGcvzsA9RMRrLxqTZU6HZtqOMMIySxFI90YNxkrhFPoSWQsJEzCpb3INYU2NN6y64OUW6PloSdWAvlgtCdsG8jgixRZu7oauYlQbWrYNhpQPg-15TX0p9J4EIZ31JSqMjH1i0UmstLPE0zwnaMlRAQ-dupTa2TEOVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42022148&ar=350369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Correlation between the growth and the amount of attached cells of deletion mutants of the Keio collection. Gray squares represent the selected deletion clones listed in Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> shown with the total mutants examined (black squares), parental strain BW25113 (white circle), and KR0401 used as a control (white triangle). The amounts of attached cells (cv) relative to that of the control are plotted against growth. </p></div></div><p> We analyzed the biofilm formation of each mutant at least twice and selected ∼160 genes that showed, on average, less than 36% biofilm formation of KR0401 as candidates of biofilm-related gene. However, the Keio collection contains two independent sets of deletion mutants and the results for some genes were inconsistent between them. Therefore, we also selected those clones if one of the two showed less than 30% of the biofilm formation of KR0401, which is why the values of some selected clones in Fig.  <a reveal-id="DSM024F1" data-open="DSM024F1" class="link link-reveal link-table xref-fig">1</a> are rather high. Next, we tried to confirm that the deletion mutations of selected genes are indeed responsible for the reduced biofilm formation by transforming with the plasmid clone of ASKA library <sup><a reveal-id="DSM024C17" data-open="DSM024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> containing each corresponding wild-type gene. However, a considerable number of deletion mutants were only partially complemented by them or not at all (data not shown), which is probably because the overproduction or uncoordinated expression from the plasmid clone is also deleterious to biofilm formation. <sup><a reveal-id="DSM024C23" data-open="DSM024C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Therefore, we transferred the deletion mutation of selected genes into KR0401 strain by P1 transduction, selecting for Kanamycin resistance to examine the relation between biofilm phenotype and the deleted gene. </p><p> Biofilm formation of four independent transductants was analyzed for each mutant and the average and standard deviation of their relative biofilm formation was calculated. Transductants of some deletion mutants did not show apparent reduction in biofilm formation. Some others grew very poorly such that the reduced biofilm formation they exhibited was considered to be the result of bad growth. The genes of these deletion mutants were therefore excluded from the list of biofilm-related genes. In a few cases, one of the two deletion mutants of the Keio collection was found to be incorrect by PCR analysis of their chromosome, which probably occurred due to the contamination of other mutants during the transfer of clones, and only the data for correct clones were incorporated in Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> . </p>                        <a id="42022156" scrollto-destination="42022156"></a>
<div content-id="DSM024TB1" class="table-modal"><div class="table-wrap"><div id="DSM024TB1" class="table-wrap-title" data-id="DSM024TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p><em>Escherichia coli</em> genes of which mutation cause defective biofilm formation <sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align="bottom"><tr><th rowspan="2">Function </th><th rowspan="2">Gene </th><th colspan="2"> Biofilm <sup>b</sup> (%)  </th><th colspan="2"> Phenotype <sup>c</sup> </th><th colspan="2">COG </th><th>Description </th></tr><tr><th>av </th><th>SD </th><th>motility </th><th>Type 1 </th><th>curli </th><th> </th><th> </th></tr></thead><tbody><tr><td>Motility </td><td><em>flgA</em> </td><td>16.1 </td><td>7.6 </td><td>− </td><td>+ </td><td>+ </td><td>N, O </td><td>Assembly protein for flagellar basal-body periplasmic P ring </td></tr><tr><td> </td><td><em>flgB</em> </td><td>17.5 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgC</em> </td><td>16.9 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgD</em> </td><td>15.4 </td><td>10.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook assembly protein </td></tr><tr><td> </td><td><em>flgE</em> </td><td>14.0 </td><td>6.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook protein </td></tr><tr><td> </td><td><em>flgF</em> </td><td>12.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgG</em> </td><td>20.6 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-distal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgH</em> </td><td>16.8 </td><td>5.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar protein of basal-body outer-membrane L ring </td></tr><tr><td> </td><td><em>flgI</em> </td><td>20.2 </td><td>11.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Predicted flagellar basal body protein </td></tr><tr><td> </td><td><em>flgJ</em> </td><td>22.5 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, M, O, U </td><td>Muramidase </td></tr><tr><td> </td><td><em>flgK</em> </td><td>16.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein 1 </td></tr><tr><td> </td><td><em>flgL</em> </td><td>10.2 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein </td></tr><tr><td> </td><td><em>flgN</em> </td><td>47.1 </td><td>16.3 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Export chaperone for FlgK and FlgL </td></tr><tr><td> </td><td><em>flhA</em> </td><td>22.1 </td><td>5.6 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhB</em> </td><td>16.4 </td><td>5.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhC</em> </td><td>17.7 </td><td>3.5 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional regulator with FlhD </td></tr><tr><td> </td><td><em>flhD</em> </td><td>28.7 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional dual regulator with FlhC </td></tr><tr><td> </td><td><em>flhE</em> </td><td>56.1 </td><td>27.7 </td><td>v </td><td>+ </td><td>+ </td><td> </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliA</em> </td><td>18.9 </td><td>6.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>RNA polymerase, sigma 28 (sigma F) factor </td></tr><tr><td> </td><td><em>fliC</em> </td><td>18.1 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament structural protein (flagellin) </td></tr><tr><td> </td><td><em>fliD</em> </td><td>13.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament capping protein </td></tr><tr><td> </td><td><em> fliE <sup>a</sup></em> </td><td>13.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body component </td></tr><tr><td> </td><td><em>fliF</em> </td><td>19.1 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body MS-ring and collar protein </td></tr><tr><td> </td><td><em>fliG</em> </td><td>14.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliH</em> </td><td>13.6 </td><td>8.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliI</em> </td><td>18.7 </td><td>3.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellum-specific ATP synthase </td></tr><tr><td> </td><td><em>fliJ</em> </td><td>10.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliK</em> </td><td>19.1 </td><td>4.1 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-length control protein </td></tr><tr><td> </td><td><em>fliL</em> </td><td>46.6 </td><td>4.4 </td><td>+ </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliM</em> </td><td>21.9 </td><td>9.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliN</em> </td><td>18.7 </td><td>4.9 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliO</em> </td><td>9.5 </td><td>3.8 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliP</em> </td><td>18.1 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliQ</em> </td><td>17.0 </td><td>3.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliR</em> </td><td>15.3 </td><td>5.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar export pore protein </td></tr><tr><td> </td><td><em>fliS</em> </td><td>26.8 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein potentiates polymerization </td></tr><tr><td> </td><td><em>fliT</em> </td><td>59.0 </td><td>4.9 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted chaperone </td></tr><tr><td> </td><td><em>motA</em> </td><td>52.5 </td><td>14.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Proton conductor component of flagella motor </td></tr><tr><td> </td><td><em>motB</em> </td><td>15.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Protein that enables flagellar motor rotation </td></tr><tr><td>Type 1 </td><td><em>fimA</em> </td><td>3.9 </td><td>3.9 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Major Type 1 subunit fimbrin (pilin) </td></tr><tr><td> </td><td><em>fimB</em> </td><td>1.2 </td><td>1.4 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Tyrosine recombinase/inversion of on/off regulator of fimA </td></tr><tr><td> </td><td><em>fimC</em> </td><td>1.0 </td><td>1.3 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Chaperone, periplasmic </td></tr><tr><td> </td><td><em>fimD</em> </td><td>1.4 </td><td>1.9 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Outer membrane usher protein, Type 1 fimbrial synthesis </td></tr><tr><td> </td><td><em>fimF</em> </td><td>1.2 </td><td>1.8 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimG</em> </td><td>17.5 </td><td>15.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimH</em> </td><td>1.1 </td><td>1.5 </td><td>+ </td><td>− </td><td>+ </td><td> </td><td>minor component of Type 1 fimbriae </td></tr><tr><td>Curli </td><td><em>csgA</em> </td><td>32.9 </td><td>4.8 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Cryptic curlin major subunit </td></tr><tr><td> </td><td><em>csgB</em> </td><td>33.1 </td><td>3.3 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Curlin nucleator protein, minor subunit in curli complex </td></tr><tr><td> </td><td><em>csgD</em> </td><td>52.9 </td><td>1.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>DNA-binding transcriptional activator in two-component regulatory system </td></tr><tr><td> </td><td><em>csgE</em> </td><td>43.8 </td><td>7.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgF</em> </td><td>45.2 </td><td>8.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgG</em> </td><td>42.9 </td><td>2.5 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Outer membrane lipoprotein </td></tr><tr><td>LPS </td><td><em>lpcA</em> </td><td>9.6 </td><td>1.0 </td><td>− </td><td>+ </td><td>+ </td><td>G </td><td><span class="small-caps">d</span> -sedoheptulose 7-phosphate isomerase  </td></tr><tr><td> </td><td><em>gmhB</em> </td><td>24.7 </td><td>4.1 </td><td>± </td><td>+ </td><td>+ </td><td>E, G, M </td><td><span class="small-caps">d</span> , <span class="small-caps">d</span> -heptose 1,7-bisphosphate phosphatase  </td></tr><tr><td> </td><td><em>rfaD</em> </td><td>12.5 </td><td>2.0 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td> ADP- <span class="small-caps">l</span> -glycero- <span class="small-caps">d</span> -mannoheptose-6-epimerase, NAD(P)-binding  </td></tr><tr><td> </td><td><em>rfaE</em> </td><td>11.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>M </td><td>Fused heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase </td></tr><tr><td> </td><td><em>rfaF</em> </td><td>13.2 </td><td>2.2 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>ADP-heptose:LPS heptosyltransferase II </td></tr><tr><td> </td><td><em>rfaG</em> </td><td>19.1 </td><td>9.9 </td><td>+ </td><td>+ </td><td>± </td><td>M </td><td>glucosyltransferase I </td></tr><tr><td> </td><td><em>rfaH</em> </td><td>22.7 </td><td>13.7 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional antiterminator </td></tr><tr><td> </td><td><em>rfaP</em> </td><td>12.3 </td><td>6.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Kinase that phosphorylates core heptose of lipopolysaccharide </td></tr><tr><td>Other </td><td><em>btuB</em> </td><td>72.5 </td><td>27.2 </td><td>+ </td><td>v </td><td>v </td><td>H </td><td>Vitamin B12/cobalamin outer membrane transporter </td></tr><tr><td> </td><td><em>cheZ</em> </td><td>65.4 </td><td>15.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, T </td><td>Chemotaxis regulator </td></tr><tr><td> </td><td><em>crp</em> </td><td>8.9 </td><td>4.5 </td><td>− </td><td>+ </td><td>± </td><td>T </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>crr</em> </td><td>72.5 </td><td>10.6 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>Glucose-specific enzyme IIA component of PTS </td></tr><tr><td> </td><td><em>cyaA</em> </td><td>4.5 </td><td>4.3 </td><td>− </td><td>+ </td><td>− </td><td>F </td><td>Adenylate cyclase </td></tr><tr><td> </td><td><em>degP</em> </td><td>38.4 </td><td>7.4 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>Serine endoprotease (protease Do), membrane-associated </td></tr><tr><td> </td><td><em>dgkA</em> </td><td>76.7 </td><td>6.3 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Diacylglycerol kinase </td></tr><tr><td> </td><td><em>dnaK</em> </td><td>46.2 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>O </td><td>Chaperone Hsp70, co-chaperone with DnaJ </td></tr><tr><td> </td><td><em>dsbA</em> </td><td>8.5 </td><td>3.3 </td><td>+ </td><td>+ </td><td>+ </td><td>C,O </td><td>Periplasmic protein disulfide isomerase I </td></tr><tr><td> </td><td><em>dsbB</em> </td><td>43.6 </td><td>11.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Oxidoreductase that catalyzes reoxidation of DsbA protein disulfide isomerase I </td></tr><tr><td> </td><td><em>fruR</em> </td><td>52.3 </td><td>3.0 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>galU</em> </td><td>24.3 </td><td>11.7 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucose-1-phosphate uridylyltransferase </td></tr><tr><td> </td><td><em>gcvA</em> </td><td>74.6 </td><td>16.9 </td><td>+ </td><td>+ </td><td>± </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>greA</em> </td><td>49.8 </td><td>3.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Transcription elongation factor </td></tr><tr><td> </td><td><em>hfq</em> </td><td>43.6 </td><td>4.1 </td><td>+ </td><td>+ </td><td>± </td><td>R </td><td>HF-I, host factor for RNA phage Q beta replication </td></tr><tr><td> </td><td><em>hscB</em> </td><td>68.0 </td><td>14.6 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>DnaJ-like molecular chaperone specific for IscU </td></tr><tr><td> </td><td><em>hsrA/yieO</em> </td><td>70.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>E, G, P, R </td><td>Predicted multidrug or homocysteine efflux system </td></tr><tr><td> </td><td><em>ihfB</em> </td><td>2.2 </td><td>2.0 </td><td>+ </td><td>− </td><td>− </td><td>L </td><td>Integration host factor (IHF), DNA-binding protein, beta subunit </td></tr><tr><td> </td><td><em>lon</em> </td><td>37.3 </td><td>4.3 </td><td>+ </td><td>+ </td><td>− </td><td>O </td><td>DNA-binding ATP-dependent protease La </td></tr><tr><td> </td><td><em>mdoH</em> </td><td>60.1 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucan biosynthesis: glycosyl transferase </td></tr><tr><td> </td><td><em>mlrA</em> </td><td>61.5 </td><td>4.9 </td><td>+ </td><td>+ </td><td>− </td><td>K </td><td>DNA-binding transcriptional regulator </td></tr><tr><td> </td><td><em>mltE</em> </td><td>72.0 </td><td>16.6 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>Lytic murein endotransglycosylase E </td></tr><tr><td> </td><td><em>mog</em> </td><td>68.3 </td><td>12.6 </td><td>± </td><td>+ </td><td>+ </td><td>H </td><td>Predicted molybdochelatase </td></tr><tr><td> </td><td><em>nagA</em> </td><td>64.1 </td><td>5.1 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td><em>N</em> -acetylglucosamine-6-phosphate deacetylase  </td></tr><tr><td> </td><td><em>yjhA/ nanC</em> </td><td>43.5 </td><td>3.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td><em>N</em> -acetylnuraminic acid outer membrane channel protein  </td></tr><tr><td> </td><td><em>nifU</em> </td><td>68.6 </td><td>4.1 </td><td>± </td><td>+ </td><td>± </td><td>C </td><td>Scaffold protein </td></tr><tr><td> </td><td><em>nlpD</em> </td><td>60.0 </td><td>4.2 </td><td>+ </td><td>+ </td><td>− </td><td>M </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>nlpI</em> </td><td>56.8 </td><td>4.6 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ompR</em> </td><td>47.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>K, T </td><td>DNA-binding response regulator in two-component regulatory system with EnvZ </td></tr><tr><td> </td><td><em>pgi</em> </td><td>59.2 </td><td>4.0 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td>Glucosephosphate isomerase </td></tr><tr><td> </td><td><em>proQ</em> </td><td>66.6 </td><td>16.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted structural transport element </td></tr><tr><td> </td><td><em>ptsI</em> </td><td>63.4 </td><td>9.3 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>PEP-protein phosphotransferase of PTS system (enzyme I) </td></tr><tr><td> </td><td><em>rcsC</em> </td><td>72.9 </td><td>3.9 </td><td>+ </td><td>+ </td><td>+ </td><td>T </td><td>Hybrid sensory kinase in two-component regulatory system with RcsB and YojN </td></tr><tr><td> </td><td><em>rpmE</em> </td><td>59.8 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>J </td><td>50S ribosomal subunit protein L31 </td></tr><tr><td> </td><td><em>rpoS</em> </td><td>58.6 </td><td>8.6 </td><td>+ </td><td>v </td><td>− </td><td>K </td><td>RNA polymerase, sigma S (sigma 38) factor </td></tr><tr><td> </td><td><em>sdhC</em> </td><td>73.3 </td><td>5.6 </td><td>+ </td><td>+ </td><td>+ </td><td>C </td><td>Succinate dehydrogenase, membrane subunit, binds cytochrome b556 </td></tr><tr><td> </td><td><em>surA</em> </td><td>3.3 </td><td>2.9 </td><td>+ </td><td>− </td><td>+ </td><td>O </td><td> peptidyl-prolyl <em>cis</em> – <em>trans</em> isomerase (PPIase)  </td></tr><tr><td> </td><td><em>tolA</em> </td><td>58.6 </td><td>4.1 </td><td>v </td><td>+ </td><td>+ </td><td>M </td><td>Membrane anchored protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>tolB</em> </td><td>45.6 </td><td>4.8 </td><td>+ </td><td>+ </td><td>± </td><td>N, U </td><td>Periplasmic protein </td></tr><tr><td> </td><td><em>tolR</em> </td><td>45.9 </td><td>2.3 </td><td>± </td><td>+ </td><td>± </td><td>N, U </td><td>Membrane spanning protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>yfgL</em> </td><td>37.5 </td><td>10.5 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Protein assembly complex, lipoprotein component </td></tr><tr><td>Uncharacterized </td><td><em>ycfM</em> </td><td>50.4 </td><td>7.3 </td><td>± </td><td>+ </td><td>+ </td><td>N, R </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>yciB/ ispZ</em> </td><td>48.0 </td><td>12.2 </td><td>± </td><td>+ </td><td>+ </td><td>D </td><td>Predicted inner membrane protein </td></tr><tr><td> </td><td><em>yciM</em> </td><td>39.9 </td><td>14.0 </td><td>+ </td><td>+ </td><td>± </td><td>G </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ydaM</em> </td><td>58.2 </td><td>4.3 </td><td>± </td><td>+ </td><td>± </td><td>T </td><td>Predicted diguanylate cyclase, GGDEF domain signalling protein </td></tr><tr><td> </td><td><em>ydeT/(fimD)</em> </td><td>60.5 </td><td>18.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted protein </td></tr><tr><td> </td><td><em>yfgA</em> </td><td>63.3 </td><td>15.1 </td><td>− </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yhcB</em> </td><td>68.3 </td><td>19.4 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yicO</em> </td><td>75.7 </td><td>39.0 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Predicted xanthine/uracil permase </td></tr><tr><td> </td><td><em>ynjC</em> </td><td>41.0 </td><td>7.1 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Fused transporter subunits/membrane component of ABC superfamily </td></tr></tbody></table></div><div content-id="DSM024TB1" class="table-modal"><table><thead name="thead" align="bottom"><tr><th rowspan="2">Function </th><th rowspan="2">Gene </th><th colspan="2"> Biofilm <sup>b</sup> (%)  </th><th colspan="2"> Phenotype <sup>c</sup> </th><th colspan="2">COG </th><th>Description </th></tr><tr><th>av </th><th>SD </th><th>motility </th><th>Type 1 </th><th>curli </th><th> </th><th> </th></tr></thead><tbody><tr><td>Motility </td><td><em>flgA</em> </td><td>16.1 </td><td>7.6 </td><td>− </td><td>+ </td><td>+ </td><td>N, O </td><td>Assembly protein for flagellar basal-body periplasmic P ring </td></tr><tr><td> </td><td><em>flgB</em> </td><td>17.5 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgC</em> </td><td>16.9 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgD</em> </td><td>15.4 </td><td>10.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook assembly protein </td></tr><tr><td> </td><td><em>flgE</em> </td><td>14.0 </td><td>6.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook protein </td></tr><tr><td> </td><td><em>flgF</em> </td><td>12.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgG</em> </td><td>20.6 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-distal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgH</em> </td><td>16.8 </td><td>5.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar protein of basal-body outer-membrane L ring </td></tr><tr><td> </td><td><em>flgI</em> </td><td>20.2 </td><td>11.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Predicted flagellar basal body protein </td></tr><tr><td> </td><td><em>flgJ</em> </td><td>22.5 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, M, O, U </td><td>Muramidase </td></tr><tr><td> </td><td><em>flgK</em> </td><td>16.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein 1 </td></tr><tr><td> </td><td><em>flgL</em> </td><td>10.2 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein </td></tr><tr><td> </td><td><em>flgN</em> </td><td>47.1 </td><td>16.3 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Export chaperone for FlgK and FlgL </td></tr><tr><td> </td><td><em>flhA</em> </td><td>22.1 </td><td>5.6 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhB</em> </td><td>16.4 </td><td>5.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhC</em> </td><td>17.7 </td><td>3.5 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional regulator with FlhD </td></tr><tr><td> </td><td><em>flhD</em> </td><td>28.7 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional dual regulator with FlhC </td></tr><tr><td> </td><td><em>flhE</em> </td><td>56.1 </td><td>27.7 </td><td>v </td><td>+ </td><td>+ </td><td> </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliA</em> </td><td>18.9 </td><td>6.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>RNA polymerase, sigma 28 (sigma F) factor </td></tr><tr><td> </td><td><em>fliC</em> </td><td>18.1 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament structural protein (flagellin) </td></tr><tr><td> </td><td><em>fliD</em> </td><td>13.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament capping protein </td></tr><tr><td> </td><td><em> fliE <sup>a</sup></em> </td><td>13.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body component </td></tr><tr><td> </td><td><em>fliF</em> </td><td>19.1 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body MS-ring and collar protein </td></tr><tr><td> </td><td><em>fliG</em> </td><td>14.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliH</em> </td><td>13.6 </td><td>8.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliI</em> </td><td>18.7 </td><td>3.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellum-specific ATP synthase </td></tr><tr><td> </td><td><em>fliJ</em> </td><td>10.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliK</em> </td><td>19.1 </td><td>4.1 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-length control protein </td></tr><tr><td> </td><td><em>fliL</em> </td><td>46.6 </td><td>4.4 </td><td>+ </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliM</em> </td><td>21.9 </td><td>9.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliN</em> </td><td>18.7 </td><td>4.9 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliO</em> </td><td>9.5 </td><td>3.8 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliP</em> </td><td>18.1 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliQ</em> </td><td>17.0 </td><td>3.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliR</em> </td><td>15.3 </td><td>5.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar export pore protein </td></tr><tr><td> </td><td><em>fliS</em> </td><td>26.8 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein potentiates polymerization </td></tr><tr><td> </td><td><em>fliT</em> </td><td>59.0 </td><td>4.9 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted chaperone </td></tr><tr><td> </td><td><em>motA</em> </td><td>52.5 </td><td>14.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Proton conductor component of flagella motor </td></tr><tr><td> </td><td><em>motB</em> </td><td>15.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Protein that enables flagellar motor rotation </td></tr><tr><td>Type 1 </td><td><em>fimA</em> </td><td>3.9 </td><td>3.9 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Major Type 1 subunit fimbrin (pilin) </td></tr><tr><td> </td><td><em>fimB</em> </td><td>1.2 </td><td>1.4 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Tyrosine recombinase/inversion of on/off regulator of fimA </td></tr><tr><td> </td><td><em>fimC</em> </td><td>1.0 </td><td>1.3 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Chaperone, periplasmic </td></tr><tr><td> </td><td><em>fimD</em> </td><td>1.4 </td><td>1.9 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Outer membrane usher protein, Type 1 fimbrial synthesis </td></tr><tr><td> </td><td><em>fimF</em> </td><td>1.2 </td><td>1.8 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimG</em> </td><td>17.5 </td><td>15.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimH</em> </td><td>1.1 </td><td>1.5 </td><td>+ </td><td>− </td><td>+ </td><td> </td><td>minor component of Type 1 fimbriae </td></tr><tr><td>Curli </td><td><em>csgA</em> </td><td>32.9 </td><td>4.8 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Cryptic curlin major subunit </td></tr><tr><td> </td><td><em>csgB</em> </td><td>33.1 </td><td>3.3 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Curlin nucleator protein, minor subunit in curli complex </td></tr><tr><td> </td><td><em>csgD</em> </td><td>52.9 </td><td>1.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>DNA-binding transcriptional activator in two-component regulatory system </td></tr><tr><td> </td><td><em>csgE</em> </td><td>43.8 </td><td>7.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgF</em> </td><td>45.2 </td><td>8.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgG</em> </td><td>42.9 </td><td>2.5 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Outer membrane lipoprotein </td></tr><tr><td>LPS </td><td><em>lpcA</em> </td><td>9.6 </td><td>1.0 </td><td>− </td><td>+ </td><td>+ </td><td>G </td><td><span class="small-caps">d</span> -sedoheptulose 7-phosphate isomerase  </td></tr><tr><td> </td><td><em>gmhB</em> </td><td>24.7 </td><td>4.1 </td><td>± </td><td>+ </td><td>+ </td><td>E, G, M </td><td><span class="small-caps">d</span> , <span class="small-caps">d</span> -heptose 1,7-bisphosphate phosphatase  </td></tr><tr><td> </td><td><em>rfaD</em> </td><td>12.5 </td><td>2.0 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td> ADP- <span class="small-caps">l</span> -glycero- <span class="small-caps">d</span> -mannoheptose-6-epimerase, NAD(P)-binding  </td></tr><tr><td> </td><td><em>rfaE</em> </td><td>11.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>M </td><td>Fused heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase </td></tr><tr><td> </td><td><em>rfaF</em> </td><td>13.2 </td><td>2.2 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>ADP-heptose:LPS heptosyltransferase II </td></tr><tr><td> </td><td><em>rfaG</em> </td><td>19.1 </td><td>9.9 </td><td>+ </td><td>+ </td><td>± </td><td>M </td><td>glucosyltransferase I </td></tr><tr><td> </td><td><em>rfaH</em> </td><td>22.7 </td><td>13.7 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional antiterminator </td></tr><tr><td> </td><td><em>rfaP</em> </td><td>12.3 </td><td>6.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Kinase that phosphorylates core heptose of lipopolysaccharide </td></tr><tr><td>Other </td><td><em>btuB</em> </td><td>72.5 </td><td>27.2 </td><td>+ </td><td>v </td><td>v </td><td>H </td><td>Vitamin B12/cobalamin outer membrane transporter </td></tr><tr><td> </td><td><em>cheZ</em> </td><td>65.4 </td><td>15.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, T </td><td>Chemotaxis regulator </td></tr><tr><td> </td><td><em>crp</em> </td><td>8.9 </td><td>4.5 </td><td>− </td><td>+ </td><td>± </td><td>T </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>crr</em> </td><td>72.5 </td><td>10.6 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>Glucose-specific enzyme IIA component of PTS </td></tr><tr><td> </td><td><em>cyaA</em> </td><td>4.5 </td><td>4.3 </td><td>− </td><td>+ </td><td>− </td><td>F </td><td>Adenylate cyclase </td></tr><tr><td> </td><td><em>degP</em> </td><td>38.4 </td><td>7.4 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>Serine endoprotease (protease Do), membrane-associated </td></tr><tr><td> </td><td><em>dgkA</em> </td><td>76.7 </td><td>6.3 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Diacylglycerol kinase </td></tr><tr><td> </td><td><em>dnaK</em> </td><td>46.2 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>O </td><td>Chaperone Hsp70, co-chaperone with DnaJ </td></tr><tr><td> </td><td><em>dsbA</em> </td><td>8.5 </td><td>3.3 </td><td>+ </td><td>+ </td><td>+ </td><td>C,O </td><td>Periplasmic protein disulfide isomerase I </td></tr><tr><td> </td><td><em>dsbB</em> </td><td>43.6 </td><td>11.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Oxidoreductase that catalyzes reoxidation of DsbA protein disulfide isomerase I </td></tr><tr><td> </td><td><em>fruR</em> </td><td>52.3 </td><td>3.0 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>galU</em> </td><td>24.3 </td><td>11.7 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucose-1-phosphate uridylyltransferase </td></tr><tr><td> </td><td><em>gcvA</em> </td><td>74.6 </td><td>16.9 </td><td>+ </td><td>+ </td><td>± </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>greA</em> </td><td>49.8 </td><td>3.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Transcription elongation factor </td></tr><tr><td> </td><td><em>hfq</em> </td><td>43.6 </td><td>4.1 </td><td>+ </td><td>+ </td><td>± </td><td>R </td><td>HF-I, host factor for RNA phage Q beta replication </td></tr><tr><td> </td><td><em>hscB</em> </td><td>68.0 </td><td>14.6 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>DnaJ-like molecular chaperone specific for IscU </td></tr><tr><td> </td><td><em>hsrA/yieO</em> </td><td>70.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>E, G, P, R </td><td>Predicted multidrug or homocysteine efflux system </td></tr><tr><td> </td><td><em>ihfB</em> </td><td>2.2 </td><td>2.0 </td><td>+ </td><td>− </td><td>− </td><td>L </td><td>Integration host factor (IHF), DNA-binding protein, beta subunit </td></tr><tr><td> </td><td><em>lon</em> </td><td>37.3 </td><td>4.3 </td><td>+ </td><td>+ </td><td>− </td><td>O </td><td>DNA-binding ATP-dependent protease La </td></tr><tr><td> </td><td><em>mdoH</em> </td><td>60.1 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucan biosynthesis: glycosyl transferase </td></tr><tr><td> </td><td><em>mlrA</em> </td><td>61.5 </td><td>4.9 </td><td>+ </td><td>+ </td><td>− </td><td>K </td><td>DNA-binding transcriptional regulator </td></tr><tr><td> </td><td><em>mltE</em> </td><td>72.0 </td><td>16.6 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>Lytic murein endotransglycosylase E </td></tr><tr><td> </td><td><em>mog</em> </td><td>68.3 </td><td>12.6 </td><td>± </td><td>+ </td><td>+ </td><td>H </td><td>Predicted molybdochelatase </td></tr><tr><td> </td><td><em>nagA</em> </td><td>64.1 </td><td>5.1 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td><em>N</em> -acetylglucosamine-6-phosphate deacetylase  </td></tr><tr><td> </td><td><em>yjhA/ nanC</em> </td><td>43.5 </td><td>3.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td><em>N</em> -acetylnuraminic acid outer membrane channel protein  </td></tr><tr><td> </td><td><em>nifU</em> </td><td>68.6 </td><td>4.1 </td><td>± </td><td>+ </td><td>± </td><td>C </td><td>Scaffold protein </td></tr><tr><td> </td><td><em>nlpD</em> </td><td>60.0 </td><td>4.2 </td><td>+ </td><td>+ </td><td>− </td><td>M </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>nlpI</em> </td><td>56.8 </td><td>4.6 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ompR</em> </td><td>47.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>K, T </td><td>DNA-binding response regulator in two-component regulatory system with EnvZ </td></tr><tr><td> </td><td><em>pgi</em> </td><td>59.2 </td><td>4.0 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td>Glucosephosphate isomerase </td></tr><tr><td> </td><td><em>proQ</em> </td><td>66.6 </td><td>16.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted structural transport element </td></tr><tr><td> </td><td><em>ptsI</em> </td><td>63.4 </td><td>9.3 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>PEP-protein phosphotransferase of PTS system (enzyme I) </td></tr><tr><td> </td><td><em>rcsC</em> </td><td>72.9 </td><td>3.9 </td><td>+ </td><td>+ </td><td>+ </td><td>T </td><td>Hybrid sensory kinase in two-component regulatory system with RcsB and YojN </td></tr><tr><td> </td><td><em>rpmE</em> </td><td>59.8 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>J </td><td>50S ribosomal subunit protein L31 </td></tr><tr><td> </td><td><em>rpoS</em> </td><td>58.6 </td><td>8.6 </td><td>+ </td><td>v </td><td>− </td><td>K </td><td>RNA polymerase, sigma S (sigma 38) factor </td></tr><tr><td> </td><td><em>sdhC</em> </td><td>73.3 </td><td>5.6 </td><td>+ </td><td>+ </td><td>+ </td><td>C </td><td>Succinate dehydrogenase, membrane subunit, binds cytochrome b556 </td></tr><tr><td> </td><td><em>surA</em> </td><td>3.3 </td><td>2.9 </td><td>+ </td><td>− </td><td>+ </td><td>O </td><td> peptidyl-prolyl <em>cis</em> – <em>trans</em> isomerase (PPIase)  </td></tr><tr><td> </td><td><em>tolA</em> </td><td>58.6 </td><td>4.1 </td><td>v </td><td>+ </td><td>+ </td><td>M </td><td>Membrane anchored protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>tolB</em> </td><td>45.6 </td><td>4.8 </td><td>+ </td><td>+ </td><td>± </td><td>N, U </td><td>Periplasmic protein </td></tr><tr><td> </td><td><em>tolR</em> </td><td>45.9 </td><td>2.3 </td><td>± </td><td>+ </td><td>± </td><td>N, U </td><td>Membrane spanning protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>yfgL</em> </td><td>37.5 </td><td>10.5 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Protein assembly complex, lipoprotein component </td></tr><tr><td>Uncharacterized </td><td><em>ycfM</em> </td><td>50.4 </td><td>7.3 </td><td>± </td><td>+ </td><td>+ </td><td>N, R </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>yciB/ ispZ</em> </td><td>48.0 </td><td>12.2 </td><td>± </td><td>+ </td><td>+ </td><td>D </td><td>Predicted inner membrane protein </td></tr><tr><td> </td><td><em>yciM</em> </td><td>39.9 </td><td>14.0 </td><td>+ </td><td>+ </td><td>± </td><td>G </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ydaM</em> </td><td>58.2 </td><td>4.3 </td><td>± </td><td>+ </td><td>± </td><td>T </td><td>Predicted diguanylate cyclase, GGDEF domain signalling protein </td></tr><tr><td> </td><td><em>ydeT/(fimD)</em> </td><td>60.5 </td><td>18.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted protein </td></tr><tr><td> </td><td><em>yfgA</em> </td><td>63.3 </td><td>15.1 </td><td>− </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yhcB</em> </td><td>68.3 </td><td>19.4 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yicO</em> </td><td>75.7 </td><td>39.0 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Predicted xanthine/uracil permase </td></tr><tr><td> </td><td><em>ynjC</em> </td><td>41.0 </td><td>7.1 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Fused transporter subunits/membrane component of ABC superfamily </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup> Genes are classified according to their known function. Name, clusters of orthologous group (COG), and description of genes are adapted from GenoBase ( <a class="link link-uri" href="http://ecoli.naist.jp">http://ecoli.naist.jp</a> ). An alternative gene name is given in addition to the systematic name when available. </p></div></div><div content-id=""><div class="fn"><p><sup>b</sup> Average (av) and standard deviation (SD) of relative biofilm formation were calculated and normalized to the values of KR0401 with more than four transductants for each deletion mutation. </p></div></div><div content-id=""><div class="fn"><p><sup>c</sup> Phenotype of motility, Type 1, and curli fimbriae were examined as described in experimental procedures. +, −, ±, and v indicate normal, defective, intermediate, and variable among transductants, respectively. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42022156" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM024TB1" class="table-wrap-title" data-id="DSM024TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p><em>Escherichia coli</em> genes of which mutation cause defective biofilm formation <sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align="bottom"><tr><th rowspan="2">Function </th><th rowspan="2">Gene </th><th colspan="2"> Biofilm <sup>b</sup> (%)  </th><th colspan="2"> Phenotype <sup>c</sup> </th><th colspan="2">COG </th><th>Description </th></tr><tr><th>av </th><th>SD </th><th>motility </th><th>Type 1 </th><th>curli </th><th> </th><th> </th></tr></thead><tbody><tr><td>Motility </td><td><em>flgA</em> </td><td>16.1 </td><td>7.6 </td><td>− </td><td>+ </td><td>+ </td><td>N, O </td><td>Assembly protein for flagellar basal-body periplasmic P ring </td></tr><tr><td> </td><td><em>flgB</em> </td><td>17.5 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgC</em> </td><td>16.9 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgD</em> </td><td>15.4 </td><td>10.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook assembly protein </td></tr><tr><td> </td><td><em>flgE</em> </td><td>14.0 </td><td>6.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook protein </td></tr><tr><td> </td><td><em>flgF</em> </td><td>12.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgG</em> </td><td>20.6 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-distal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgH</em> </td><td>16.8 </td><td>5.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar protein of basal-body outer-membrane L ring </td></tr><tr><td> </td><td><em>flgI</em> </td><td>20.2 </td><td>11.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Predicted flagellar basal body protein </td></tr><tr><td> </td><td><em>flgJ</em> </td><td>22.5 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, M, O, U </td><td>Muramidase </td></tr><tr><td> </td><td><em>flgK</em> </td><td>16.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein 1 </td></tr><tr><td> </td><td><em>flgL</em> </td><td>10.2 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein </td></tr><tr><td> </td><td><em>flgN</em> </td><td>47.1 </td><td>16.3 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Export chaperone for FlgK and FlgL </td></tr><tr><td> </td><td><em>flhA</em> </td><td>22.1 </td><td>5.6 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhB</em> </td><td>16.4 </td><td>5.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhC</em> </td><td>17.7 </td><td>3.5 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional regulator with FlhD </td></tr><tr><td> </td><td><em>flhD</em> </td><td>28.7 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional dual regulator with FlhC </td></tr><tr><td> </td><td><em>flhE</em> </td><td>56.1 </td><td>27.7 </td><td>v </td><td>+ </td><td>+ </td><td> </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliA</em> </td><td>18.9 </td><td>6.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>RNA polymerase, sigma 28 (sigma F) factor </td></tr><tr><td> </td><td><em>fliC</em> </td><td>18.1 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament structural protein (flagellin) </td></tr><tr><td> </td><td><em>fliD</em> </td><td>13.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament capping protein </td></tr><tr><td> </td><td><em> fliE <sup>a</sup></em> </td><td>13.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body component </td></tr><tr><td> </td><td><em>fliF</em> </td><td>19.1 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body MS-ring and collar protein </td></tr><tr><td> </td><td><em>fliG</em> </td><td>14.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliH</em> </td><td>13.6 </td><td>8.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliI</em> </td><td>18.7 </td><td>3.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellum-specific ATP synthase </td></tr><tr><td> </td><td><em>fliJ</em> </td><td>10.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliK</em> </td><td>19.1 </td><td>4.1 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-length control protein </td></tr><tr><td> </td><td><em>fliL</em> </td><td>46.6 </td><td>4.4 </td><td>+ </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliM</em> </td><td>21.9 </td><td>9.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliN</em> </td><td>18.7 </td><td>4.9 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliO</em> </td><td>9.5 </td><td>3.8 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliP</em> </td><td>18.1 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliQ</em> </td><td>17.0 </td><td>3.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliR</em> </td><td>15.3 </td><td>5.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar export pore protein </td></tr><tr><td> </td><td><em>fliS</em> </td><td>26.8 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein potentiates polymerization </td></tr><tr><td> </td><td><em>fliT</em> </td><td>59.0 </td><td>4.9 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted chaperone </td></tr><tr><td> </td><td><em>motA</em> </td><td>52.5 </td><td>14.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Proton conductor component of flagella motor </td></tr><tr><td> </td><td><em>motB</em> </td><td>15.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Protein that enables flagellar motor rotation </td></tr><tr><td>Type 1 </td><td><em>fimA</em> </td><td>3.9 </td><td>3.9 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Major Type 1 subunit fimbrin (pilin) </td></tr><tr><td> </td><td><em>fimB</em> </td><td>1.2 </td><td>1.4 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Tyrosine recombinase/inversion of on/off regulator of fimA </td></tr><tr><td> </td><td><em>fimC</em> </td><td>1.0 </td><td>1.3 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Chaperone, periplasmic </td></tr><tr><td> </td><td><em>fimD</em> </td><td>1.4 </td><td>1.9 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Outer membrane usher protein, Type 1 fimbrial synthesis </td></tr><tr><td> </td><td><em>fimF</em> </td><td>1.2 </td><td>1.8 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimG</em> </td><td>17.5 </td><td>15.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimH</em> </td><td>1.1 </td><td>1.5 </td><td>+ </td><td>− </td><td>+ </td><td> </td><td>minor component of Type 1 fimbriae </td></tr><tr><td>Curli </td><td><em>csgA</em> </td><td>32.9 </td><td>4.8 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Cryptic curlin major subunit </td></tr><tr><td> </td><td><em>csgB</em> </td><td>33.1 </td><td>3.3 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Curlin nucleator protein, minor subunit in curli complex </td></tr><tr><td> </td><td><em>csgD</em> </td><td>52.9 </td><td>1.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>DNA-binding transcriptional activator in two-component regulatory system </td></tr><tr><td> </td><td><em>csgE</em> </td><td>43.8 </td><td>7.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgF</em> </td><td>45.2 </td><td>8.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgG</em> </td><td>42.9 </td><td>2.5 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Outer membrane lipoprotein </td></tr><tr><td>LPS </td><td><em>lpcA</em> </td><td>9.6 </td><td>1.0 </td><td>− </td><td>+ </td><td>+ </td><td>G </td><td><span class="small-caps">d</span> -sedoheptulose 7-phosphate isomerase  </td></tr><tr><td> </td><td><em>gmhB</em> </td><td>24.7 </td><td>4.1 </td><td>± </td><td>+ </td><td>+ </td><td>E, G, M </td><td><span class="small-caps">d</span> , <span class="small-caps">d</span> -heptose 1,7-bisphosphate phosphatase  </td></tr><tr><td> </td><td><em>rfaD</em> </td><td>12.5 </td><td>2.0 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td> ADP- <span class="small-caps">l</span> -glycero- <span class="small-caps">d</span> -mannoheptose-6-epimerase, NAD(P)-binding  </td></tr><tr><td> </td><td><em>rfaE</em> </td><td>11.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>M </td><td>Fused heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase </td></tr><tr><td> </td><td><em>rfaF</em> </td><td>13.2 </td><td>2.2 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>ADP-heptose:LPS heptosyltransferase II </td></tr><tr><td> </td><td><em>rfaG</em> </td><td>19.1 </td><td>9.9 </td><td>+ </td><td>+ </td><td>± </td><td>M </td><td>glucosyltransferase I </td></tr><tr><td> </td><td><em>rfaH</em> </td><td>22.7 </td><td>13.7 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional antiterminator </td></tr><tr><td> </td><td><em>rfaP</em> </td><td>12.3 </td><td>6.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Kinase that phosphorylates core heptose of lipopolysaccharide </td></tr><tr><td>Other </td><td><em>btuB</em> </td><td>72.5 </td><td>27.2 </td><td>+ </td><td>v </td><td>v </td><td>H </td><td>Vitamin B12/cobalamin outer membrane transporter </td></tr><tr><td> </td><td><em>cheZ</em> </td><td>65.4 </td><td>15.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, T </td><td>Chemotaxis regulator </td></tr><tr><td> </td><td><em>crp</em> </td><td>8.9 </td><td>4.5 </td><td>− </td><td>+ </td><td>± </td><td>T </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>crr</em> </td><td>72.5 </td><td>10.6 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>Glucose-specific enzyme IIA component of PTS </td></tr><tr><td> </td><td><em>cyaA</em> </td><td>4.5 </td><td>4.3 </td><td>− </td><td>+ </td><td>− </td><td>F </td><td>Adenylate cyclase </td></tr><tr><td> </td><td><em>degP</em> </td><td>38.4 </td><td>7.4 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>Serine endoprotease (protease Do), membrane-associated </td></tr><tr><td> </td><td><em>dgkA</em> </td><td>76.7 </td><td>6.3 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Diacylglycerol kinase </td></tr><tr><td> </td><td><em>dnaK</em> </td><td>46.2 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>O </td><td>Chaperone Hsp70, co-chaperone with DnaJ </td></tr><tr><td> </td><td><em>dsbA</em> </td><td>8.5 </td><td>3.3 </td><td>+ </td><td>+ </td><td>+ </td><td>C,O </td><td>Periplasmic protein disulfide isomerase I </td></tr><tr><td> </td><td><em>dsbB</em> </td><td>43.6 </td><td>11.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Oxidoreductase that catalyzes reoxidation of DsbA protein disulfide isomerase I </td></tr><tr><td> </td><td><em>fruR</em> </td><td>52.3 </td><td>3.0 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>galU</em> </td><td>24.3 </td><td>11.7 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucose-1-phosphate uridylyltransferase </td></tr><tr><td> </td><td><em>gcvA</em> </td><td>74.6 </td><td>16.9 </td><td>+ </td><td>+ </td><td>± </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>greA</em> </td><td>49.8 </td><td>3.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Transcription elongation factor </td></tr><tr><td> </td><td><em>hfq</em> </td><td>43.6 </td><td>4.1 </td><td>+ </td><td>+ </td><td>± </td><td>R </td><td>HF-I, host factor for RNA phage Q beta replication </td></tr><tr><td> </td><td><em>hscB</em> </td><td>68.0 </td><td>14.6 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>DnaJ-like molecular chaperone specific for IscU </td></tr><tr><td> </td><td><em>hsrA/yieO</em> </td><td>70.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>E, G, P, R </td><td>Predicted multidrug or homocysteine efflux system </td></tr><tr><td> </td><td><em>ihfB</em> </td><td>2.2 </td><td>2.0 </td><td>+ </td><td>− </td><td>− </td><td>L </td><td>Integration host factor (IHF), DNA-binding protein, beta subunit </td></tr><tr><td> </td><td><em>lon</em> </td><td>37.3 </td><td>4.3 </td><td>+ </td><td>+ </td><td>− </td><td>O </td><td>DNA-binding ATP-dependent protease La </td></tr><tr><td> </td><td><em>mdoH</em> </td><td>60.1 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucan biosynthesis: glycosyl transferase </td></tr><tr><td> </td><td><em>mlrA</em> </td><td>61.5 </td><td>4.9 </td><td>+ </td><td>+ </td><td>− </td><td>K </td><td>DNA-binding transcriptional regulator </td></tr><tr><td> </td><td><em>mltE</em> </td><td>72.0 </td><td>16.6 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>Lytic murein endotransglycosylase E </td></tr><tr><td> </td><td><em>mog</em> </td><td>68.3 </td><td>12.6 </td><td>± </td><td>+ </td><td>+ </td><td>H </td><td>Predicted molybdochelatase </td></tr><tr><td> </td><td><em>nagA</em> </td><td>64.1 </td><td>5.1 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td><em>N</em> -acetylglucosamine-6-phosphate deacetylase  </td></tr><tr><td> </td><td><em>yjhA/ nanC</em> </td><td>43.5 </td><td>3.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td><em>N</em> -acetylnuraminic acid outer membrane channel protein  </td></tr><tr><td> </td><td><em>nifU</em> </td><td>68.6 </td><td>4.1 </td><td>± </td><td>+ </td><td>± </td><td>C </td><td>Scaffold protein </td></tr><tr><td> </td><td><em>nlpD</em> </td><td>60.0 </td><td>4.2 </td><td>+ </td><td>+ </td><td>− </td><td>M </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>nlpI</em> </td><td>56.8 </td><td>4.6 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ompR</em> </td><td>47.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>K, T </td><td>DNA-binding response regulator in two-component regulatory system with EnvZ </td></tr><tr><td> </td><td><em>pgi</em> </td><td>59.2 </td><td>4.0 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td>Glucosephosphate isomerase </td></tr><tr><td> </td><td><em>proQ</em> </td><td>66.6 </td><td>16.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted structural transport element </td></tr><tr><td> </td><td><em>ptsI</em> </td><td>63.4 </td><td>9.3 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>PEP-protein phosphotransferase of PTS system (enzyme I) </td></tr><tr><td> </td><td><em>rcsC</em> </td><td>72.9 </td><td>3.9 </td><td>+ </td><td>+ </td><td>+ </td><td>T </td><td>Hybrid sensory kinase in two-component regulatory system with RcsB and YojN </td></tr><tr><td> </td><td><em>rpmE</em> </td><td>59.8 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>J </td><td>50S ribosomal subunit protein L31 </td></tr><tr><td> </td><td><em>rpoS</em> </td><td>58.6 </td><td>8.6 </td><td>+ </td><td>v </td><td>− </td><td>K </td><td>RNA polymerase, sigma S (sigma 38) factor </td></tr><tr><td> </td><td><em>sdhC</em> </td><td>73.3 </td><td>5.6 </td><td>+ </td><td>+ </td><td>+ </td><td>C </td><td>Succinate dehydrogenase, membrane subunit, binds cytochrome b556 </td></tr><tr><td> </td><td><em>surA</em> </td><td>3.3 </td><td>2.9 </td><td>+ </td><td>− </td><td>+ </td><td>O </td><td> peptidyl-prolyl <em>cis</em> – <em>trans</em> isomerase (PPIase)  </td></tr><tr><td> </td><td><em>tolA</em> </td><td>58.6 </td><td>4.1 </td><td>v </td><td>+ </td><td>+ </td><td>M </td><td>Membrane anchored protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>tolB</em> </td><td>45.6 </td><td>4.8 </td><td>+ </td><td>+ </td><td>± </td><td>N, U </td><td>Periplasmic protein </td></tr><tr><td> </td><td><em>tolR</em> </td><td>45.9 </td><td>2.3 </td><td>± </td><td>+ </td><td>± </td><td>N, U </td><td>Membrane spanning protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>yfgL</em> </td><td>37.5 </td><td>10.5 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Protein assembly complex, lipoprotein component </td></tr><tr><td>Uncharacterized </td><td><em>ycfM</em> </td><td>50.4 </td><td>7.3 </td><td>± </td><td>+ </td><td>+ </td><td>N, R </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>yciB/ ispZ</em> </td><td>48.0 </td><td>12.2 </td><td>± </td><td>+ </td><td>+ </td><td>D </td><td>Predicted inner membrane protein </td></tr><tr><td> </td><td><em>yciM</em> </td><td>39.9 </td><td>14.0 </td><td>+ </td><td>+ </td><td>± </td><td>G </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ydaM</em> </td><td>58.2 </td><td>4.3 </td><td>± </td><td>+ </td><td>± </td><td>T </td><td>Predicted diguanylate cyclase, GGDEF domain signalling protein </td></tr><tr><td> </td><td><em>ydeT/(fimD)</em> </td><td>60.5 </td><td>18.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted protein </td></tr><tr><td> </td><td><em>yfgA</em> </td><td>63.3 </td><td>15.1 </td><td>− </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yhcB</em> </td><td>68.3 </td><td>19.4 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yicO</em> </td><td>75.7 </td><td>39.0 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Predicted xanthine/uracil permase </td></tr><tr><td> </td><td><em>ynjC</em> </td><td>41.0 </td><td>7.1 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Fused transporter subunits/membrane component of ABC superfamily </td></tr></tbody></table></div><div content-id="DSM024TB1" class="table-modal"><table><thead name="thead" align="bottom"><tr><th rowspan="2">Function </th><th rowspan="2">Gene </th><th colspan="2"> Biofilm <sup>b</sup> (%)  </th><th colspan="2"> Phenotype <sup>c</sup> </th><th colspan="2">COG </th><th>Description </th></tr><tr><th>av </th><th>SD </th><th>motility </th><th>Type 1 </th><th>curli </th><th> </th><th> </th></tr></thead><tbody><tr><td>Motility </td><td><em>flgA</em> </td><td>16.1 </td><td>7.6 </td><td>− </td><td>+ </td><td>+ </td><td>N, O </td><td>Assembly protein for flagellar basal-body periplasmic P ring </td></tr><tr><td> </td><td><em>flgB</em> </td><td>17.5 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgC</em> </td><td>16.9 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgD</em> </td><td>15.4 </td><td>10.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook assembly protein </td></tr><tr><td> </td><td><em>flgE</em> </td><td>14.0 </td><td>6.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook protein </td></tr><tr><td> </td><td><em>flgF</em> </td><td>12.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-proximal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgG</em> </td><td>20.6 </td><td>16.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar component of cell-distal portion of basal-body rod </td></tr><tr><td> </td><td><em>flgH</em> </td><td>16.8 </td><td>5.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar protein of basal-body outer-membrane L ring </td></tr><tr><td> </td><td><em>flgI</em> </td><td>20.2 </td><td>11.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Predicted flagellar basal body protein </td></tr><tr><td> </td><td><em>flgJ</em> </td><td>22.5 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, M, O, U </td><td>Muramidase </td></tr><tr><td> </td><td><em>flgK</em> </td><td>16.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein 1 </td></tr><tr><td> </td><td><em>flgL</em> </td><td>10.2 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-filament junction protein </td></tr><tr><td> </td><td><em>flgN</em> </td><td>47.1 </td><td>16.3 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Export chaperone for FlgK and FlgL </td></tr><tr><td> </td><td><em>flhA</em> </td><td>22.1 </td><td>5.6 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhB</em> </td><td>16.4 </td><td>5.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted flagellar export pore protein </td></tr><tr><td> </td><td><em>flhC</em> </td><td>17.7 </td><td>3.5 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional regulator with FlhD </td></tr><tr><td> </td><td><em>flhD</em> </td><td>28.7 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>DNA-binding transcriptional dual regulator with FlhC </td></tr><tr><td> </td><td><em>flhE</em> </td><td>56.1 </td><td>27.7 </td><td>v </td><td>+ </td><td>+ </td><td> </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliA</em> </td><td>18.9 </td><td>6.9 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>RNA polymerase, sigma 28 (sigma F) factor </td></tr><tr><td> </td><td><em>fliC</em> </td><td>18.1 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament structural protein (flagellin) </td></tr><tr><td> </td><td><em>fliD</em> </td><td>13.4 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar filament capping protein </td></tr><tr><td> </td><td><em> fliE <sup>a</sup></em> </td><td>13.8 </td><td>5.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body component </td></tr><tr><td> </td><td><em>fliF</em> </td><td>19.1 </td><td>8.1 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar basal-body MS-ring and collar protein </td></tr><tr><td> </td><td><em>fliG</em> </td><td>14.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliH</em> </td><td>13.6 </td><td>8.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliI</em> </td><td>18.7 </td><td>3.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellum-specific ATP synthase </td></tr><tr><td> </td><td><em>fliJ</em> </td><td>10.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein </td></tr><tr><td> </td><td><em>fliK</em> </td><td>19.1 </td><td>4.1 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar hook-length control protein </td></tr><tr><td> </td><td><em>fliL</em> </td><td>46.6 </td><td>4.4 </td><td>+ </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliM</em> </td><td>21.9 </td><td>9.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliN</em> </td><td>18.7 </td><td>4.9 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar motor switching and energizing component </td></tr><tr><td> </td><td><em>fliO</em> </td><td>9.5 </td><td>3.8 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliP</em> </td><td>18.1 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliQ</em> </td><td>17.0 </td><td>3.7 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar biosynthesis protein </td></tr><tr><td> </td><td><em>fliR</em> </td><td>15.3 </td><td>5.2 </td><td>− </td><td>+ </td><td>+ </td><td>N, U </td><td>Flagellar export pore protein </td></tr><tr><td> </td><td><em>fliS</em> </td><td>26.8 </td><td>6.0 </td><td>− </td><td>+ </td><td>+ </td><td>N, O, U </td><td>Flagellar protein potentiates polymerization </td></tr><tr><td> </td><td><em>fliT</em> </td><td>59.0 </td><td>4.9 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted chaperone </td></tr><tr><td> </td><td><em>motA</em> </td><td>52.5 </td><td>14.5 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Proton conductor component of flagella motor </td></tr><tr><td> </td><td><em>motB</em> </td><td>15.9 </td><td>4.4 </td><td>− </td><td>+ </td><td>+ </td><td>N </td><td>Protein that enables flagellar motor rotation </td></tr><tr><td>Type 1 </td><td><em>fimA</em> </td><td>3.9 </td><td>3.9 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Major Type 1 subunit fimbrin (pilin) </td></tr><tr><td> </td><td><em>fimB</em> </td><td>1.2 </td><td>1.4 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Tyrosine recombinase/inversion of on/off regulator of fimA </td></tr><tr><td> </td><td><em>fimC</em> </td><td>1.0 </td><td>1.3 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Chaperone, periplasmic </td></tr><tr><td> </td><td><em>fimD</em> </td><td>1.4 </td><td>1.9 </td><td>+ </td><td>− </td><td>+ </td><td>L </td><td>Outer membrane usher protein, Type 1 fimbrial synthesis </td></tr><tr><td> </td><td><em>fimF</em> </td><td>1.2 </td><td>1.8 </td><td>+ </td><td>− </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimG</em> </td><td>17.5 </td><td>15.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Minor component of Type 1 fimbriae </td></tr><tr><td> </td><td><em>fimH</em> </td><td>1.1 </td><td>1.5 </td><td>+ </td><td>− </td><td>+ </td><td> </td><td>minor component of Type 1 fimbriae </td></tr><tr><td>Curli </td><td><em>csgA</em> </td><td>32.9 </td><td>4.8 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Cryptic curlin major subunit </td></tr><tr><td> </td><td><em>csgB</em> </td><td>33.1 </td><td>3.3 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Curlin nucleator protein, minor subunit in curli complex </td></tr><tr><td> </td><td><em>csgD</em> </td><td>52.9 </td><td>1.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>DNA-binding transcriptional activator in two-component regulatory system </td></tr><tr><td> </td><td><em>csgE</em> </td><td>43.8 </td><td>7.6 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgF</em> </td><td>45.2 </td><td>8.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted transport protein </td></tr><tr><td> </td><td><em>csgG</em> </td><td>42.9 </td><td>2.5 </td><td>+ </td><td>+ </td><td>− </td><td> </td><td>Outer membrane lipoprotein </td></tr><tr><td>LPS </td><td><em>lpcA</em> </td><td>9.6 </td><td>1.0 </td><td>− </td><td>+ </td><td>+ </td><td>G </td><td><span class="small-caps">d</span> -sedoheptulose 7-phosphate isomerase  </td></tr><tr><td> </td><td><em>gmhB</em> </td><td>24.7 </td><td>4.1 </td><td>± </td><td>+ </td><td>+ </td><td>E, G, M </td><td><span class="small-caps">d</span> , <span class="small-caps">d</span> -heptose 1,7-bisphosphate phosphatase  </td></tr><tr><td> </td><td><em>rfaD</em> </td><td>12.5 </td><td>2.0 </td><td>− </td><td>+ </td><td>+ </td><td> </td><td> ADP- <span class="small-caps">l</span> -glycero- <span class="small-caps">d</span> -mannoheptose-6-epimerase, NAD(P)-binding  </td></tr><tr><td> </td><td><em>rfaE</em> </td><td>11.4 </td><td>2.3 </td><td>− </td><td>+ </td><td>+ </td><td>M </td><td>Fused heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase </td></tr><tr><td> </td><td><em>rfaF</em> </td><td>13.2 </td><td>2.2 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>ADP-heptose:LPS heptosyltransferase II </td></tr><tr><td> </td><td><em>rfaG</em> </td><td>19.1 </td><td>9.9 </td><td>+ </td><td>+ </td><td>± </td><td>M </td><td>glucosyltransferase I </td></tr><tr><td> </td><td><em>rfaH</em> </td><td>22.7 </td><td>13.7 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional antiterminator </td></tr><tr><td> </td><td><em>rfaP</em> </td><td>12.3 </td><td>6.4 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Kinase that phosphorylates core heptose of lipopolysaccharide </td></tr><tr><td>Other </td><td><em>btuB</em> </td><td>72.5 </td><td>27.2 </td><td>+ </td><td>v </td><td>v </td><td>H </td><td>Vitamin B12/cobalamin outer membrane transporter </td></tr><tr><td> </td><td><em>cheZ</em> </td><td>65.4 </td><td>15.4 </td><td>− </td><td>+ </td><td>+ </td><td>N, T </td><td>Chemotaxis regulator </td></tr><tr><td> </td><td><em>crp</em> </td><td>8.9 </td><td>4.5 </td><td>− </td><td>+ </td><td>± </td><td>T </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>crr</em> </td><td>72.5 </td><td>10.6 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>Glucose-specific enzyme IIA component of PTS </td></tr><tr><td> </td><td><em>cyaA</em> </td><td>4.5 </td><td>4.3 </td><td>− </td><td>+ </td><td>− </td><td>F </td><td>Adenylate cyclase </td></tr><tr><td> </td><td><em>degP</em> </td><td>38.4 </td><td>7.4 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>Serine endoprotease (protease Do), membrane-associated </td></tr><tr><td> </td><td><em>dgkA</em> </td><td>76.7 </td><td>6.3 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Diacylglycerol kinase </td></tr><tr><td> </td><td><em>dnaK</em> </td><td>46.2 </td><td>5.8 </td><td>− </td><td>+ </td><td>+ </td><td>O </td><td>Chaperone Hsp70, co-chaperone with DnaJ </td></tr><tr><td> </td><td><em>dsbA</em> </td><td>8.5 </td><td>3.3 </td><td>+ </td><td>+ </td><td>+ </td><td>C,O </td><td>Periplasmic protein disulfide isomerase I </td></tr><tr><td> </td><td><em>dsbB</em> </td><td>43.6 </td><td>11.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Oxidoreductase that catalyzes reoxidation of DsbA protein disulfide isomerase I </td></tr><tr><td> </td><td><em>fruR</em> </td><td>52.3 </td><td>3.0 </td><td>+ </td><td>+ </td><td>+ </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>galU</em> </td><td>24.3 </td><td>11.7 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucose-1-phosphate uridylyltransferase </td></tr><tr><td> </td><td><em>gcvA</em> </td><td>74.6 </td><td>16.9 </td><td>+ </td><td>+ </td><td>± </td><td>K </td><td>DNA-binding transcriptional dual regulator </td></tr><tr><td> </td><td><em>greA</em> </td><td>49.8 </td><td>3.5 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Transcription elongation factor </td></tr><tr><td> </td><td><em>hfq</em> </td><td>43.6 </td><td>4.1 </td><td>+ </td><td>+ </td><td>± </td><td>R </td><td>HF-I, host factor for RNA phage Q beta replication </td></tr><tr><td> </td><td><em>hscB</em> </td><td>68.0 </td><td>14.6 </td><td>+ </td><td>+ </td><td>+ </td><td>O </td><td>DnaJ-like molecular chaperone specific for IscU </td></tr><tr><td> </td><td><em>hsrA/yieO</em> </td><td>70.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>E, G, P, R </td><td>Predicted multidrug or homocysteine efflux system </td></tr><tr><td> </td><td><em>ihfB</em> </td><td>2.2 </td><td>2.0 </td><td>+ </td><td>− </td><td>− </td><td>L </td><td>Integration host factor (IHF), DNA-binding protein, beta subunit </td></tr><tr><td> </td><td><em>lon</em> </td><td>37.3 </td><td>4.3 </td><td>+ </td><td>+ </td><td>− </td><td>O </td><td>DNA-binding ATP-dependent protease La </td></tr><tr><td> </td><td><em>mdoH</em> </td><td>60.1 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>M </td><td>Glucan biosynthesis: glycosyl transferase </td></tr><tr><td> </td><td><em>mlrA</em> </td><td>61.5 </td><td>4.9 </td><td>+ </td><td>+ </td><td>− </td><td>K </td><td>DNA-binding transcriptional regulator </td></tr><tr><td> </td><td><em>mltE</em> </td><td>72.0 </td><td>16.6 </td><td>± </td><td>+ </td><td>+ </td><td>M </td><td>Lytic murein endotransglycosylase E </td></tr><tr><td> </td><td><em>mog</em> </td><td>68.3 </td><td>12.6 </td><td>± </td><td>+ </td><td>+ </td><td>H </td><td>Predicted molybdochelatase </td></tr><tr><td> </td><td><em>nagA</em> </td><td>64.1 </td><td>5.1 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td><em>N</em> -acetylglucosamine-6-phosphate deacetylase  </td></tr><tr><td> </td><td><em>yjhA/ nanC</em> </td><td>43.5 </td><td>3.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td><em>N</em> -acetylnuraminic acid outer membrane channel protein  </td></tr><tr><td> </td><td><em>nifU</em> </td><td>68.6 </td><td>4.1 </td><td>± </td><td>+ </td><td>± </td><td>C </td><td>Scaffold protein </td></tr><tr><td> </td><td><em>nlpD</em> </td><td>60.0 </td><td>4.2 </td><td>+ </td><td>+ </td><td>− </td><td>M </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>nlpI</em> </td><td>56.8 </td><td>4.6 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ompR</em> </td><td>47.8 </td><td>3.7 </td><td>+ </td><td>+ </td><td>− </td><td>K, T </td><td>DNA-binding response regulator in two-component regulatory system with EnvZ </td></tr><tr><td> </td><td><em>pgi</em> </td><td>59.2 </td><td>4.0 </td><td>± </td><td>+ </td><td>+ </td><td>G </td><td>Glucosephosphate isomerase </td></tr><tr><td> </td><td><em>proQ</em> </td><td>66.6 </td><td>16.8 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Predicted structural transport element </td></tr><tr><td> </td><td><em>ptsI</em> </td><td>63.4 </td><td>9.3 </td><td>+ </td><td>+ </td><td>+ </td><td>G </td><td>PEP-protein phosphotransferase of PTS system (enzyme I) </td></tr><tr><td> </td><td><em>rcsC</em> </td><td>72.9 </td><td>3.9 </td><td>+ </td><td>+ </td><td>+ </td><td>T </td><td>Hybrid sensory kinase in two-component regulatory system with RcsB and YojN </td></tr><tr><td> </td><td><em>rpmE</em> </td><td>59.8 </td><td>10.4 </td><td>+ </td><td>+ </td><td>+ </td><td>J </td><td>50S ribosomal subunit protein L31 </td></tr><tr><td> </td><td><em>rpoS</em> </td><td>58.6 </td><td>8.6 </td><td>+ </td><td>v </td><td>− </td><td>K </td><td>RNA polymerase, sigma S (sigma 38) factor </td></tr><tr><td> </td><td><em>sdhC</em> </td><td>73.3 </td><td>5.6 </td><td>+ </td><td>+ </td><td>+ </td><td>C </td><td>Succinate dehydrogenase, membrane subunit, binds cytochrome b556 </td></tr><tr><td> </td><td><em>surA</em> </td><td>3.3 </td><td>2.9 </td><td>+ </td><td>− </td><td>+ </td><td>O </td><td> peptidyl-prolyl <em>cis</em> – <em>trans</em> isomerase (PPIase)  </td></tr><tr><td> </td><td><em>tolA</em> </td><td>58.6 </td><td>4.1 </td><td>v </td><td>+ </td><td>+ </td><td>M </td><td>Membrane anchored protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>tolB</em> </td><td>45.6 </td><td>4.8 </td><td>+ </td><td>+ </td><td>± </td><td>N, U </td><td>Periplasmic protein </td></tr><tr><td> </td><td><em>tolR</em> </td><td>45.9 </td><td>2.3 </td><td>± </td><td>+ </td><td>± </td><td>N, U </td><td>Membrane spanning protein in TolA-TolQ-TolR complex </td></tr><tr><td> </td><td><em>yfgL</em> </td><td>37.5 </td><td>10.5 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Protein assembly complex, lipoprotein component </td></tr><tr><td>Uncharacterized </td><td><em>ycfM</em> </td><td>50.4 </td><td>7.3 </td><td>± </td><td>+ </td><td>+ </td><td>N, R </td><td>Predicted outer membrane lipoprotein </td></tr><tr><td> </td><td><em>yciB/ ispZ</em> </td><td>48.0 </td><td>12.2 </td><td>± </td><td>+ </td><td>+ </td><td>D </td><td>Predicted inner membrane protein </td></tr><tr><td> </td><td><em>yciM</em> </td><td>39.9 </td><td>14.0 </td><td>+ </td><td>+ </td><td>± </td><td>G </td><td>Conserved protein </td></tr><tr><td> </td><td><em>ydaM</em> </td><td>58.2 </td><td>4.3 </td><td>± </td><td>+ </td><td>± </td><td>T </td><td>Predicted diguanylate cyclase, GGDEF domain signalling protein </td></tr><tr><td> </td><td><em>ydeT/(fimD)</em> </td><td>60.5 </td><td>18.1 </td><td>+ </td><td>+ </td><td>+ </td><td>N, U </td><td>Predicted protein </td></tr><tr><td> </td><td><em>yfgA</em> </td><td>63.3 </td><td>15.1 </td><td>− </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yhcB</em> </td><td>68.3 </td><td>19.4 </td><td>+ </td><td>+ </td><td>+ </td><td>S </td><td>Conserved protein </td></tr><tr><td> </td><td><em>yicO</em> </td><td>75.7 </td><td>39.0 </td><td>+ </td><td>+ </td><td>+ </td><td>R </td><td>Predicted xanthine/uracil permase </td></tr><tr><td> </td><td><em>ynjC</em> </td><td>41.0 </td><td>7.1 </td><td>+ </td><td>+ </td><td>+ </td><td> </td><td>Fused transporter subunits/membrane component of ABC superfamily </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup> Genes are classified according to their known function. Name, clusters of orthologous group (COG), and description of genes are adapted from GenoBase ( <a class="link link-uri" href="http://ecoli.naist.jp">http://ecoli.naist.jp</a> ). An alternative gene name is given in addition to the systematic name when available. </p></div></div><div content-id=""><div class="fn"><p><sup>b</sup> Average (av) and standard deviation (SD) of relative biofilm formation were calculated and normalized to the values of KR0401 with more than four transductants for each deletion mutation. </p></div></div><div content-id=""><div class="fn"><p><sup>c</sup> Phenotype of motility, Type 1, and curli fimbriae were examined as described in experimental procedures. +, −, ±, and v indicate normal, defective, intermediate, and variable among transductants, respectively. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42022156" target="_blank">View Large</a></div></div><p> Several deletion mutations were non-transferable by P1 phage, because strains harbouring them were refractory to P1 propagation. Therefore, we initially introduced the corresponding archive clone (ASKA library) <sup><a reveal-id="DSM024C17" data-open="DSM024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> by transformation and then propagated P1 from the resulting transformants. These genes were <em>lpcA, rfaD</em> , and <em>rfaE</em> , which are involved in LPS biosynthesis. The P1 lysates thus obtained were subsequently transduced into KR0401 and the biofilm forming ability examined. However, in the case of <em>rfaC</em> , <em>tolQ</em> , and <em>yiiS</em> gene mutants, transformants of the corresponding archive clones still could not propagate P1 and thus we failed to confirm the effect of the deletion mutations in biofilm formation. Therefore, these genes were not included in the final list of biofilm-related genes. </p><p> As a consequence, 110 genes were identified to be associated with biofilm formation upon disruption, although there were variations in the degree of reduction observed in each gene deletion mutant (Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> ). They are classified according to their functions and their possible roles in biofilm formation are discussed in the following sections. </p>                            <h3 scrollto-destination=42022161 id="42022161" class="section-title" data-legacy-id=s3b>3.2. Motility and fimbriae genes</h3>
<p> Various investigations reported so far showed that bacterial cell surface structures are important for cell adhesion during the development of biofilm. In <em>E. coli</em> K12 strains, flagella, Type 1, and curli fimbriae are the main structures implicated in biofilm formation. <sup><a reveal-id="DSM024C24" data-open="DSM024C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Therefore, we subsequently examined deletion mutants of these three surface appendages as listed in Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> . </p><p> As expected, most mutants of flagella biosynthesis and motility genes including <em>cyaA</em> and <em>crp</em> that are required for the expression of flagella genes <sup><a reveal-id="DSM024C25" data-open="DSM024C25" class="link link-ref link-reveal xref-bibr">25</a></sup> showed severe biofilm defects and loss of motility, whereas <em>flhE</em> , <em>fliL</em> , and <em>fliT</em> mutants retained the motility as reported previously <sup><a reveal-id="DSM024C26  DSM024C28" data-open="DSM024C26  DSM024C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup> and showed a lesser extent of reduction in biofilm formation. This was also the case with an <em>flgN</em> mutant. Proteins encoded by <em>flgN</em> and <em>fliT</em> act as chaperones in flagella assembly but they are not structural proteins. <sup><a reveal-id="DSM024C28" data-open="DSM024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The functions of <em>flhE</em> and <em>fliL</em> are still not clear, although they reside in the flagella operon. <sup><a reveal-id="DSM024C26" data-open="DSM024C26" class="link link-ref link-reveal xref-bibr">26</a> , <a reveal-id="DSM024C27" data-open="DSM024C27" class="link link-ref link-reveal xref-bibr">27</a> , <a reveal-id="DSM024C29" data-open="DSM024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> These results indicate that the biofilm formation can reflect even a subtle functional or structural difference in flagella. This possibility was also indicated in the results for mutants of disulfide interchange proteins. A severe biofilm defect (8.5% of wild type) was observed with a <em>dsbA</em> mutant, whereas <em>dsbB</em> disruption caused only a mild reduction (50%). DsbA protein catalyzes the disulfide bond formation, whereas the role of DsbB is to supply the material and its requirement can be suppressed in a medium supplied with cysteine. <sup><a reveal-id="DSM024C30" data-open="DSM024C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Dsb proteins are implicated in flagella assembly and mutants are non-motile in the absence of cystine. <sup><a reveal-id="DSM024C31" data-open="DSM024C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Flagella are probably important for the initial cell-to-surface contact and the spread of bacteria along the surface. <sup><a reveal-id="DSM024C10" data-open="DSM024C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Previously we reported that the overproduction of several genes for flagella biosynthesis also showed reduced biofilm formation. <sup><a reveal-id="DSM024C23" data-open="DSM024C23" class="link link-ref link-reveal xref-bibr">23</a></sup> This reduced biofilm phenotype might have resulted from the uncoordinated gene expression that has led to the deficiency of flagella and reduced motility. </p><p> Apart from flagella-related genes, mutants of <em>lpcA, rfaD</em> , and <em>rfaE</em> that are required for LPS biosynthesis were found to be non-motile and highly impaired in biofilm formation. On the other hand, <em>cheZ</em> , <em>dnaK</em> , and <em>yfgA</em> mutants are non-motile and yet showed only moderate biofilm phenotypes. These observations indicate that the structure of flagella and membrane that supports the assembly of flagella might be more important than the motility itself for biofilm formation. </p><p> The <em>fim</em> gene cluster encodes proteins involved in the biosynthesis of Type 1 fimbriae. Mutants of all <em>fim</em> genes except <em>fimE</em> , <em>fimG</em> , and <em>fimI</em> were negative in agglutination assay and exhibited severe defects in biofilm formation. The <em>fimG</em> mutants showed less severe biofilm compared with the other <em>fim</em> genes, which might reflect the function of the <em>fimG</em> product that may act as an inhibitor of pilus polymerization. <sup><a reveal-id="DSM024C32" data-open="DSM024C32" class="link link-ref link-reveal xref-bibr">32</a></sup> FimE negatively regulates the fimbriae synthesis <sup><a reveal-id="DSM024C33" data-open="DSM024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and the function of FimI is not clear. <sup><a reveal-id="DSM024C34" data-open="DSM024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Mutants of these genes showed increased and normal biofilm formation, respectively (data not shown). Apart from <em>fim</em> genes, <em>ihfB</em> and <em>surA</em> mutants were defective in agglutination assay and exhibited severe biofilm-defective phenotype. Type 1 fimbriae-deficient strains showed most severe biofilm reduction among mutants investigated, which may reflect their importance in the irreversible attachment of cells to the surface. </p><p> Six genes in two operons <em>csgBA</em> and <em>csgDEFG</em> function in the curli formation in <em>E. coli</em> (reviewed by Barnhart and Chapman <sup><a reveal-id="DSM024C35" data-open="DSM024C35" class="link link-ref link-reveal xref-bibr">35</a></sup> ). Deletion mutants of all these genes showed reduction in biofilm formation. The effect of loss of CsgA and CsgB, a structural subunit and a nucleator protein, respectively, was greater than those of the accessory proteins CsgD, CsgE, CsgF, and CsgG that are required for curli assembly. In addition to curli genes, <em>cya, ihfB, lon, mlrA, nlpD, ompR, rpoS</em> , and <em>yieO</em> genes showed impaired curli production upon disruption. Curli synthesis is under the complex regulatory network, which includes <em>ihf</em> , <em>mlrA</em> , <em>ompR</em> , <em>rpoS</em> genes, and the Rcs system. The Rcs system negatively regulates curli synthesis and is activated by mutations affecting the cell envelope (reviewed by Barnhart and Chapman <sup><a reveal-id="DSM024C35" data-open="DSM024C35" class="link link-ref link-reveal xref-bibr">35</a></sup> ) and metabolic stress via the alteration of membrane-related oligosaccharides. <sup><a reveal-id="DSM024C36" data-open="DSM024C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The effect on biofilm formation of the deletion mutation in these genes was moderate except for <em>cyaA</em> and <em>ihfB</em> genes, mutants of which are also defective in motility and Type 1 fimbriae, respectively, as shown above. Curli-deficient mutants exhibited moderate biofilm reduction compared with the former two surface structures, although curli were reported to be important for the initial adhesion and subsequent biofilm development. <sup><a reveal-id="DSM024C10" data-open="DSM024C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We observed that curli mutants initially adhered to the surface to a certain extent but probably detached after a while (data not shown), which suggests that curli contribute more to the maturation of the biofilm formed rather than the initial cell attachment. </p><p>The results mentioned above not only confirmed the importance of these cell surface structures but also indicated that the function of a specific deletion was well reflected in the biofilm formation. On the other hand, the existence of many more mutants that seem to be intact in these surface structures but exhibit reduced biofilm phenotype showed that indeed various genes are required for the proper development of biofilm.</p>                            <h3 scrollto-destination=42022175 id="42022175" class="section-title" data-legacy-id=s3c>3.3. Lipopolysaccharide genes</h3>
<p> Genome-wide analysis of deletion mutants revealed that eight genes involved in lipopolysaccharide (LPS) synthesis exhibited a significant degree of reduction in biofilm formation when disrupted. All of them except <em>rfaH</em> encode enzymes that catalyze the synthesis of <span class="small-caps">l</span> -glycero- <span class="small-caps">d</span> -manno-heptose and inner core assembly of the LPS indicating that the heptose region of the core oligosaccharide is important not only for the outer membrane stability <sup><a reveal-id="DSM024C37" data-open="DSM024C37" class="link link-ref link-reveal xref-bibr">37</a></sup> but also for the adhesion of cell whereas the outer core and O-antigen are not critical for biofilm formation. Their defective biofilm formation seems not simply due to the lack of the major surface structures mentioned above, at least in the case of <em>rfaF</em> , <em>rfaG</em> , <em>rfaH</em> , and <em>rfaP</em> mutants, because they were normal in motility and agglutination assay. LPS seems to be important for the initial attachment to the surface, since mutants of LPS synthesis genes showed biofilm reduction similar to flagella-defective mutants. Some of them also exhibited motility-defective phenotype as described earlier. The gene <em>rfaH</em> encodes a transcriptional antiterminator required for the expression of the <em>rfa</em> operon <sup><a reveal-id="DSM024C38" data-open="DSM024C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and, in contrast to our result, its inactivation was reported to increase the initial adhesion and biofilm formation. <sup><a reveal-id="DSM024C39" data-open="DSM024C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The reason for this discrepancy is not clear but it might be due to the difference in culture conditions. </p>                            <h3 scrollto-destination=42022179 id="42022179" class="section-title" data-legacy-id=s3d>3.4. Others</h3>
<p> Apart from LPS, many genes related to cell membrane were identified to exhibit mild reduction in biofilm formation upon disruption. However, our analysis could not identify the genes for colanic acid synthesis. This exopolysaccharide is not required for the initial attachment but important for the biofilm development into a complex three-dimensional structure (reviewed by Van Houdt and Michiels <sup><a reveal-id="DSM024C24" data-open="DSM024C24" class="link link-ref link-reveal xref-bibr">24</a></sup> ). Therefore, the possible reason for this failure is that our procedure was not suitable for the analysis of the late stage of biofilm formation or that the parental strain of the Keio collection does not develop such a structure as reported for some laboratory <em>E. coli</em> strains. </p><p>In addition, genes of more variable functions, including nine genes of unknown function, were identified in our analysis. Although the precise mechanism how the deletion of these genes leads to defective biofilm formation remains to be clarified, it is conceivable that the perturbation of cellular activity such as metabolism and energy production causes some deficiency or stress in the cell membrane, thereby affecting biofilm formation.</p>                            <h3 scrollto-destination=42022186 id="42022186" class="section-title" data-legacy-id=s3e>3.5. Characterization of function-unknown genes</h3>
<p> Some of the function-unknown genes identified are predicted to encode membrane proteins based on their primary structure and/or the phenotypes of their mutants (Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> ). Moreover, it is well expected that membrane proteins, in particular OMPs, contribute to the developmental processes of biofilm formation. Therefore, we examined the profiles of major OMPs isolated from mutants of function-unknown genes and <em>surA</em> . SurA possesses both periplasmic chaperone and peptidyl prolyl isomerase (PPIase) activities and facilitates OMP biogenesis. <sup><a reveal-id="DSM024C40" data-open="DSM024C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Its mutant shows a profile of significantly reduced OMPs <sup><a reveal-id="DSM024C41" data-open="DSM024C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and highly impaired biofilm formation (Table  <a reveal-id="DSM024TB1" data-open="DSM024TB1" class="link link-reveal link-table xref-fig">1</a> ). The amounts of OmpA, OmpC, OmpF, and LamB were found to be reduced in <em>yciB</em> , <em>yciM</em> , and <em>yfgA</em> mutants, although the effect was not so drastic as that observed with the <em>surA</em> mutant (Fig.  <a reveal-id="DSM024F2" data-open="DSM024F2" class="link link-reveal link-table xref-fig">2</a> ). This result indicates that these genes might be required for membrane integrity. However, our analysis did not identify mutants of major OMP genes: the reason could be that deficiency of a single OMP gene might not cause an observable reduction in biofilm formation. </p>                        <a id="42022191" scrollto-destination="42022191"></a>
<div id="DSM024F2" data-id="DSM024F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/m_dsm02402.jpeg?Expires=2147483647&Signature=bzNRdCglxqfmuY4BUDtlQopijumfdAI98qLASJOmkeicr8d3fhP~KKnVTrZmfT3fdbakWPj1vzmRTfVYCNVopydDuFUlXy~zB6sJwAYlKQ8diUJ9Z-Ld1Rbn9Vs5cdJ8KxApo0z4hUkWuBcJ3D4wa2i21Fn60TxkzjW0GvQEiRdOiMQ6sNlQ8bZYnVTHUGdkFtQjm3TQAWFlHHYHnIg8-If5ctfqOOHWLOiBqd8I3-PWwuxFVIyZREtAEECK-c7eLDa2Ee6FaMCJrwiJzJyMUzUmuP~RqocB-EyrM3VirHRwsJQkb-2lFtGJeu5mRyzjXx-o1k6LykM~ponn2Pgh1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The OMP profiles of deletion mutants of uncharacterized genes. Outer membrane fractions prepared from the equivalent amount of cells of KR0401 (lane 1) and its derivative strains harboring Δ yfgA ::Km, Δ yciM ::Km, Δ yciB ::Km, and Δ surA ::Km mutations (lanes 2–5, respectively) were analyzed on a 10 % SDS-polyacrylamide gel and stained with Coomassie Brilliant Blue. " path-from-xml="dsm02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42022191/dsm02402.jpeg" path-from-xml="dsm02402.jpeg" target="_blank">View large</a><a section="42022191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/dsm02402.jpeg?Expires=2147483647&Signature=Rh3HaaCKK33g8B9C~X6G83VQ9lEmaiHrFjvwg-AQrHypAH6m2uJDPVaUA-5aAA~LASKbkhvr1Q9xp8~gwkKElj0art6ESfbOg0l4FtUhcfP~EGAywgtIh9A2oF7b-w1ht5aqz5N0zPVAGrzrM~k86VVPXSjYTwYZYu6gjOZK28aBUyq9S75Zg3dKLxFAuxd~99bawk0jHy-9EwpoB1Ad1FizUFLT0fvsTs7x-OgV-mM6mP71YtMMYOtO2nMMALesB2Zeb2hvvNICgdSt7zCkbewI2yQlMwZb3D9IteGzBXkUOvcy7QnKCVv1Sd2XseTk5b8J4Wn0Fro4T8R81YVaqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42022191&ar=350369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The OMP profiles of deletion mutants of uncharacterized genes. Outer membrane fractions prepared from the equivalent amount of cells of KR0401 (lane 1) and its derivative strains harboring Δ <em>yfgA</em> ::Km, Δ <em>yciM</em> ::Km, Δ <em>yciB</em> ::Km, and Δ <em>surA</em> ::Km mutations (lanes 2–5, respectively) were analyzed on a 10 % SDS-polyacrylamide gel and stained with Coomassie Brilliant Blue. </p></div></div><div content-id="DSM024F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/m_dsm02402.jpeg?Expires=2147483647&Signature=bzNRdCglxqfmuY4BUDtlQopijumfdAI98qLASJOmkeicr8d3fhP~KKnVTrZmfT3fdbakWPj1vzmRTfVYCNVopydDuFUlXy~zB6sJwAYlKQ8diUJ9Z-Ld1Rbn9Vs5cdJ8KxApo0z4hUkWuBcJ3D4wa2i21Fn60TxkzjW0GvQEiRdOiMQ6sNlQ8bZYnVTHUGdkFtQjm3TQAWFlHHYHnIg8-If5ctfqOOHWLOiBqd8I3-PWwuxFVIyZREtAEECK-c7eLDa2Ee6FaMCJrwiJzJyMUzUmuP~RqocB-EyrM3VirHRwsJQkb-2lFtGJeu5mRyzjXx-o1k6LykM~ponn2Pgh1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The OMP profiles of deletion mutants of uncharacterized genes. Outer membrane fractions prepared from the equivalent amount of cells of KR0401 (lane 1) and its derivative strains harboring Δ yfgA ::Km, Δ yciM ::Km, Δ yciB ::Km, and Δ surA ::Km mutations (lanes 2–5, respectively) were analyzed on a 10 % SDS-polyacrylamide gel and stained with Coomassie Brilliant Blue. " path-from-xml="dsm02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/42022191/dsm02402.jpeg" path-from-xml="dsm02402.jpeg" target="_blank">View large</a><a section="42022191" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/6/10.1093_dnares_dsm024/3/dsm02402.jpeg?Expires=2147483647&Signature=Rh3HaaCKK33g8B9C~X6G83VQ9lEmaiHrFjvwg-AQrHypAH6m2uJDPVaUA-5aAA~LASKbkhvr1Q9xp8~gwkKElj0art6ESfbOg0l4FtUhcfP~EGAywgtIh9A2oF7b-w1ht5aqz5N0zPVAGrzrM~k86VVPXSjYTwYZYu6gjOZK28aBUyq9S75Zg3dKLxFAuxd~99bawk0jHy-9EwpoB1Ad1FizUFLT0fvsTs7x-OgV-mM6mP71YtMMYOtO2nMMALesB2Zeb2hvvNICgdSt7zCkbewI2yQlMwZb3D9IteGzBXkUOvcy7QnKCVv1Sd2XseTk5b8J4Wn0Fro4T8R81YVaqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=42022191&ar=350369&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The OMP profiles of deletion mutants of uncharacterized genes. Outer membrane fractions prepared from the equivalent amount of cells of KR0401 (lane 1) and its derivative strains harboring Δ <em>yfgA</em> ::Km, Δ <em>yciM</em> ::Km, Δ <em>yciB</em> ::Km, and Δ <em>surA</em> ::Km mutations (lanes 2–5, respectively) were analyzed on a 10 % SDS-polyacrylamide gel and stained with Coomassie Brilliant Blue. </p></div></div><p> The genetic interaction of function-unknown genes with the <em>surA</em> gene was then investigated by analyzing the synthetic phenotype of double mutations as described in the experimental procedures. By observing the phenotype of the double mutants thus constructed, Δ <em>yfgA</em> was found to exhibit a synthetic lethal phenotype with Δ <em>surA</em> . Similarly, Δ <em>yciB</em> , Δ <em>yciM</em> , and Δ <em>yhcB</em> were synthetically lethal with Δ <em>yfgA</em> and the phenotype was rescued by introducing a plasmid carrying the wild-type allele of <em>yfgA</em> (Table  <a reveal-id="DSM024TB2" data-open="DSM024TB2" class="link link-reveal link-table xref-fig">2</a> ). Mutants of <em>yciM</em> and <em>yfgA</em> were also more sensitive to SDS and Novobiocin, an amphipathic antibiotic, compared with the wild-type strain (data not shown). </p>                        <a id="42022197" scrollto-destination="42022197"></a>
<div content-id="DSM024TB2" class="table-modal"><div class="table-wrap"><div id="DSM024TB2" class="table-wrap-title" data-id="DSM024TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Synthetic phenotypes displayed by function-unknown genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">P1 donors </th><th colspan="3">Recipients </th></tr><tr><th> Δ <em>surA</em> </th><th> Δ <em>yfgA</em> </th><th> Δ <em>yfgA(pyfgA)</em> </th></tr></thead><tbody><tr><td> Δ <em>yciB::Km</em> </td><td>+ </td><td>vs/ − </td><td>+ </td></tr><tr><td> Δ <em>yciM::Km</em> </td><td>vs </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>yfgA::Km</em> </td><td>− </td><td>NT </td><td>NT </td></tr><tr><td> Δ <em>yhcB::Km</em> </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>surA::Km</em> </td><td>NT </td><td>− </td><td>+ </td></tr></tbody></table></div><div content-id="DSM024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">P1 donors </th><th colspan="3">Recipients </th></tr><tr><th> Δ <em>surA</em> </th><th> Δ <em>yfgA</em> </th><th> Δ <em>yfgA(pyfgA)</em> </th></tr></thead><tbody><tr><td> Δ <em>yciB::Km</em> </td><td>+ </td><td>vs/ − </td><td>+ </td></tr><tr><td> Δ <em>yciM::Km</em> </td><td>vs </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>yfgA::Km</em> </td><td>− </td><td>NT </td><td>NT </td></tr><tr><td> Δ <em>yhcB::Km</em> </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>surA::Km</em> </td><td>NT </td><td>− </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>+ indicates much the same phenotype as that of the corresponding single mutant, whereas – indicates that no transductant appeared, and vs very small colonies, respectively. NT, not tested.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42022197" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM024TB2" class="table-wrap-title" data-id="DSM024TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Synthetic phenotypes displayed by function-unknown genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">P1 donors </th><th colspan="3">Recipients </th></tr><tr><th> Δ <em>surA</em> </th><th> Δ <em>yfgA</em> </th><th> Δ <em>yfgA(pyfgA)</em> </th></tr></thead><tbody><tr><td> Δ <em>yciB::Km</em> </td><td>+ </td><td>vs/ − </td><td>+ </td></tr><tr><td> Δ <em>yciM::Km</em> </td><td>vs </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>yfgA::Km</em> </td><td>− </td><td>NT </td><td>NT </td></tr><tr><td> Δ <em>yhcB::Km</em> </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>surA::Km</em> </td><td>NT </td><td>− </td><td>+ </td></tr></tbody></table></div><div content-id="DSM024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">P1 donors </th><th colspan="3">Recipients </th></tr><tr><th> Δ <em>surA</em> </th><th> Δ <em>yfgA</em> </th><th> Δ <em>yfgA(pyfgA)</em> </th></tr></thead><tbody><tr><td> Δ <em>yciB::Km</em> </td><td>+ </td><td>vs/ − </td><td>+ </td></tr><tr><td> Δ <em>yciM::Km</em> </td><td>vs </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>yfgA::Km</em> </td><td>− </td><td>NT </td><td>NT </td></tr><tr><td> Δ <em>yhcB::Km</em> </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td> Δ <em>surA::Km</em> </td><td>NT </td><td>− </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>+ indicates much the same phenotype as that of the corresponding single mutant, whereas – indicates that no transductant appeared, and vs very small colonies, respectively. NT, not tested.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/42022197" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=42022200 id="42022200" class="section-title" data-legacy-id=s3f>3.6. Concluding remarks</h3>
<p>Our genome-wide analysis demonstrated that variable genes are indeed required for biofilm formation and that the cell surface structures and envelope are important factors. It could be that cells are required to re-organize their membrane structures during the process of developing biofilms and it is in such a process that the hitherto uncharacterized genes described above might be involved.</p><p> The analysis described here by using deletion mutants could directly identify genes involved in biofilm formation. However, certain genes not identified in this analysis might probably be required under different conditions and/or in different genetic backgrounds. Also, we found genes that increased biofilm formation upon disruption (Fig.  <a reveal-id="DSM024F1" data-open="DSM024F1" class="link link-reveal link-table xref-fig">1</a> ). They include various genes such as those for signal transduction, transcription, carbohydrate metabolism as well as those of unknown function. Further analysis of these genes would provide clues with respect to how bacteria change their life-style. </p>                            <h2 scrollto-destination=42022206 id="42022206" class="section-title" data-legacy-id=s4>Funding</h2>
<p>E. T. E. Niba was supported by a foreign student fellowship from the Japanese Ministry of Education, Culture, Sports, Science and Technology.</p>                            <h2 scrollto-destination=42022209 id="42022209" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr Tomoya Baba of Research Organization of Information and Systems for providing us with the Keio collection and ASKA clones. We thank Dr Katsumi Isono of Kazusa DNA Research Institute for his critical reading of the manuscript and discussion, and Dr Elizabeth Tenorio for her valuable suggestions.</p>                            <h2 scrollto-destination=42022211 id="42022211" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM024C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Costerton</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Lewandowski</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Caldwell</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Korber</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Lappin-Scott</div> <div class="given-names">H. M.</div></div>. </div><div class="article-title">Microbial biofilms</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">1995</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">711</div>-<div class="lpage">745</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20biofilms&amp;author=J.%20W.%20Costerton&amp;author=Z.%20Lewandowski&amp;author=D.%20E.%20Caldwell&amp;author=D.%20R.%20Korber&amp;author=H.%20M.%20Lappin-Scott&amp;publication_year=1995&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=49&amp;pages=711-745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.mi.49.100195.003431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.mi.49.100195.003431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.mi.49.100195.003431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8561477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoyle</div> <div class="given-names">B. D.</div></div>,  <div class="name"><div class="surname">Costerton</div> <div class="given-names">W. J.</div></div>. </div><div class="article-title">Bacterial resistance to antibiotics: the role of biofilms</div>, <div class="source">Prog. Drug. Res.</div> , <div class="year">1991</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">91</div>-<div class="lpage">105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20resistance%20to%20antibiotics%3A%20the%20role%20of%20biofilms&amp;author=B.%20D.%20Hoyle&amp;author=W.%20J.%20Costerton&amp;publication_year=1991&amp;journal=Prog.%20Drug.%20Res.&amp;volume=37&amp;pages=91-105" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1763187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prüß</div> <div class="given-names">B. M.</div></div>,  <div class="name"><div class="surname">Besemann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Denton</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">A. J.</div></div>. </div><div class="article-title"> A complex transcription network controls the early stages of biofilm development by <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">3731</div>-<div class="lpage">3739</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complex%20transcription%20network%20controls%20the%20early%20stages%20of%20biofilm%20development%20by%20Escherichia%20coli&amp;author=B.%20M.%20Pr%C3%BC%C3%9F&amp;author=C.%20Besemann&amp;author=A.%20Denton&amp;author=A.%20J.%20Wolfe&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=3731-3739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01780-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01780-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01780-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16707665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schembri</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kjærgaard</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Klemm</div> <div class="given-names">P.</div></div>. </div><div class="article-title"> Global gene expression in <em>Escherichia coli</em> biofilms </div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">253</div>-<div class="lpage">267</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20gene%20expression%20in%20Escherichia%20coli%20biofilms&amp;author=M.%20A.%20Schembri&amp;author=K.%20Kj%C3%A6rgaard&amp;author=P.%20Klemm&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=48&amp;pages=253-267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03432.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03432.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03432.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12657059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beloin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Valle</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Latour-Lambert</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Faure</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kzreminski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Balestrino</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title"> Global impact of mature biofilm lifestyle on <em>Escherichia coli</em> K-12 gene expression </div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">659</div>-<div class="lpage">674</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20impact%20of%20mature%20biofilm%20lifestyle%20on%20Escherichia%20coli%20K-12%20gene%20expression&amp;author=C.%20Beloin&amp;author=J.%20Valle&amp;author=P.%20Latour-Lambert&amp;author=P.%20Faure&amp;author=M.%20Kzreminski&amp;author=D.%20Balestrino&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=51&amp;pages=659-674" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03865.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03865.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03865.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14731270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ren</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bedzyk</div> <div class="given-names">L. A.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Ye</div> <div class="given-names">R. W.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">T. K.</div></div>. </div><div class="article-title"> Gene expression in <em>Escherichia coli</em> biofilms </div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2004</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">515</div>-<div class="lpage">524</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20in%20Escherichia%20coli%20biofilms&amp;author=D.%20Ren&amp;author=L.%20A.%20Bedzyk&amp;author=S.%20M.%20Thomas&amp;author=R.%20W.%20Ye&amp;author=T.%20K.%20Wood&amp;publication_year=2004&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=64&amp;pages=515-524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-003-1517-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-003-1517-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-003-1517-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14727089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Junker</div> <div class="given-names">L. M.</div></div>,  <div class="name"><div class="surname">Toba</div> <div class="given-names">F. A.</div></div>,  <div class="name"><div class="surname">Hay</div> <div class="given-names">A. G.</div></div>. </div><div class="article-title"> Transcription in <em>Escherichia coli</em> PHL628 biofilms </div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">237</div>-<div class="lpage">243</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20in%20Escherichia%20coli%20PHL628%20biofilms&amp;author=L.%20M.%20Junker&amp;author=F.%20A.%20Toba&amp;author=A.%20G.%20Hay&amp;publication_year=2007&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=268&amp;pages=237-243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2007.268.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2007.268.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2007.268.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17227468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Domka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bansal</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">T. K.</div></div>. </div><div class="article-title"> Temporal gene-expression in <em>Escherichia coli</em> K-12 biofilms </div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">332</div>-<div class="lpage">346</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%20gene-expression%20in%20Escherichia%20coli%20K-12%20biofilms&amp;author=J.%20Domka&amp;author=J.%20Lee&amp;author=T.%20Bansal&amp;author=T.%20K.%20Wood&amp;publication_year=2007&amp;journal=Environ.%20Microbiol.&amp;volume=9&amp;pages=332-346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2007.9.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2007.9.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2007.9.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17222132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lazazzera</div> <div class="given-names">B. A.</div></div>. </div><div class="article-title">Lessons from DNA microarray analysis: the gene expression profile of biofilms</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">222</div>-<div class="lpage">227</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lessons%20from%20DNA%20microarray%20analysis%3A%20the%20gene%20expression%20profile%20of%20biofilms&amp;author=B.%20A.%20Lazazzera&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=8&amp;pages=222-227" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2005.02.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2005.02.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2005.02.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15802256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prigent-Combaret</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Prensier</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Le Thi</div> <div class="given-names">T. T.</div></div>,  <div class="name"><div class="surname">Vidal</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Lejeune</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Dorel</div> <div class="given-names">C.</div></div>. </div><div class="article-title"> Developmental pathway for biofilm formation in curli-producing <em>Escherichia coli</em> strains: role of flagella, curli and colanic acid </div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">450</div>-<div class="lpage">464</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20pathway%20for%20biofilm%20formation%20in%20curli-producing%20Escherichia%20coli%20strains%3A%20role%20of%20flagella%2C%20curli%20and%20colanic%20acid&amp;author=C.%20Prigent-Combaret&amp;author=G.%20Prensier&amp;author=T.%20T.%20Le%20Thi&amp;author=O.%20Vidal&amp;author=P.%20Lejeune&amp;author=C.%20Dorel&amp;publication_year=2000&amp;journal=Environ.%20Microbiol.&amp;volume=2&amp;pages=450-464" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1462-2920.2000.00128.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1462-2920.2000.00128.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1462-2920.2000.00128.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11234933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reisner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Haagensen</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Schembri</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Zechner</div> <div class="given-names">E. L.</div></div>,  <div class="name"><div class="surname">Molin</div> <div class="given-names">S.</div></div>. </div><div class="article-title"> Development and maturation of <em>Escherichia coli</em> K-12 biofilms </div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">933</div>-<div class="lpage">946</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20maturation%20of%20Escherichia%20coli%20K-12%20biofilms&amp;author=A.%20Reisner&amp;author=J.%20A.%20Haagensen&amp;author=M.%20A.%20Schembri&amp;author=E.%20L.%20Zechner&amp;author=S.%20Molin&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=48&amp;pages=933-946" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03490.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03490.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03490.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12753187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghingo</div> <div class="given-names">J-M.</div></div>. </div><div class="article-title">Natural conjugative plasmids induce bacterial biofilm development</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">412</div> (pg. <div class="fpage">442</div>-<div class="lpage">445</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20conjugative%20plasmids%20induce%20bacterial%20biofilm%20development&amp;author=J-M.%20Ghingo&amp;publication_year=2001&amp;journal=Nature&amp;volume=412&amp;pages=442-445" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35086581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35086581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35086581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11473319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrios</div> <div class="given-names">A. F. G.</div></div>,  <div class="name"><div class="surname">Zuo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bentley</div> <div class="given-names">W. E.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">T. K.</div></div>. </div><div class="article-title"> Autoinducer 2 controls biofilm formation in <em>Escherichia coli</em> through a novel motility quorum-sensing regulator (MqsR, B3022) </div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">305</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autoinducer%202%20controls%20biofilm%20formation%20in%20Escherichia%20coli%20through%20a%20novel%20motility%20quorum-sensing%20regulator%20(MqsR%2C%20B3022)&amp;author=A.%20F.%20G.%20Barrios&amp;author=R.%20Zuo&amp;author=Y.%20Hashimoto&amp;author=L.%20Yang&amp;author=W.%20E.%20Bentley&amp;author=T.%20K.%20Wood&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=305-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.1.305-316.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.1.305-316.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.1.305-316.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16352847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrios</div> <div class="given-names">A. F. G.</div></div>,  <div class="name"><div class="surname">Zuo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">T. K.</div></div>. </div><div class="article-title"> Hha, YbaJ, and OmpA regulate <em>Escherichia coli</em> K12 biofilm formation and conjugation plasmids abolish motility </div>, <div class="source">Biotechnol. Bioeng.</div> , <div class="year">2006</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">188</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hha%2C%20YbaJ%2C%20and%20OmpA%20regulate%20Escherichia%20coli%20K12%20biofilm%20formation%20and%20conjugation%20plasmids%20abolish%20motility&amp;author=A.%20F.%20G.%20Barrios&amp;author=R.%20Zuo&amp;author=D.%20Ren&amp;author=T.%20K.%20Wood&amp;publication_year=2006&amp;journal=Biotechnol.%20Bioeng.&amp;volume=93&amp;pages=188-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16317765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Takai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Okumura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Baba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Datsenko</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title"> Construction of <em>Escherichia coli</em> K-12 in-frame, single-gene knockout mutants: the Keio collection </div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">E1</div>-<div class="lpage">E11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20Escherichia%20coli%20K-12%20in-frame%2C%20single-gene%20knockout%20mutants%3A%20the%20Keio%20collection&amp;author=T.%20Baba&amp;author=T.%20Ara&amp;author=M.%20Hasegawa&amp;author=Y.%20Takai&amp;author=Y.%20Okumura&amp;author=M.%20Baba&amp;author=K.%20Datsenko&amp;publication_year=2006&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=2&amp;pages=E1-E11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100050"> </span></div> </div></div></div></div></div><div content-id="DSM024C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Datsenko</div> <div class="given-names">K. A.</div></div>,  <div class="name"><div class="surname">Wanner</div> <div class="given-names">B. L.</div></div>. </div><div class="article-title"> One-step inactivation of chromosomal genes in <em>Escherichia coli</em> K-12 using PCR products </div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">6640</div>-<div class="lpage">6645</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products&amp;author=K.%20A.%20Datsenko&amp;author=B.%20L.%20Wanner&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=6640-6645" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.120163297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.120163297"> </span></div> </div></div></div></div></div><div content-id="DSM024C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kitagawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Arifuzzaman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ioka-Nakamichi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Inamoto</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Toyonaga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>. </div><div class="article-title"> Complete set of ORF clones of <em>Escherichia coli</em> ASKA library. A complete set of <em>E. coli</em> K-12 ORF archive: unique resources for biological research </div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">291</div>-<div class="lpage">299</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20set%20of%20ORF%20clones%20of%20Escherichia%20coli%20ASKA%20library.%20A%20complete%20set%20of%20E.%20coli%20K-12%20ORF%20archive%3A%20unique%20resources%20for%20biological%20research&amp;author=M.%20Kitagawa&amp;author=T.%20Ara&amp;author=M.%20Arifuzzaman&amp;author=T.%20Ioka-Nakamichi&amp;author=E.%20Inamoto&amp;author=H.%20Toyonaga&amp;author=H.%20Mori&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=291-299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">O'Toole</div> <div class="given-names">G. A.</div></div>,  <div class="name"><div class="surname">Pratt</div> <div class="given-names">L. A.</div></div>,  <div class="name"><div class="surname">Watnick</div> <div class="given-names">P. I.</div></div>,  <div class="name"><div class="surname">Newman</div> <div class="given-names">D. K.</div></div>,  <div class="name"><div class="surname">Weaver</div> <div class="given-names">V. B.</div></div>,  <div class="name"><div class="surname">Kolter</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genetic approaches to study of biofilms</div>, <div class="source">Methods Enzymol.</div> , <div class="year">1999</div>, vol. <div class="volume">310</div> (pg. <div class="fpage">91</div>-<div class="lpage">109</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20approaches%20to%20study%20of%20biofilms&amp;author=G.%20A.%20O'Toole&amp;author=L.%20A.%20Pratt&amp;author=P.%20I.%20Watnick&amp;author=D.%20K.%20Newman&amp;author=V.%20B.%20Weaver&amp;author=R.%20Kolter&amp;publication_year=1999&amp;journal=Methods%20Enzymol.&amp;volume=310&amp;pages=91-109" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Berg</div> <div class="given-names">H. C.</div></div>. </div><div class="article-title">Migration of bacteria in semisolid agar</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1989</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">6973</div>-<div class="lpage">6977</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Migration%20of%20bacteria%20in%20semisolid%20agar&amp;author=A.%20J.%20Wolfe&amp;author=H.%20C.%20Berg&amp;publication_year=1989&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=6973-6977" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.18.6973" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.18.6973" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.18.6973"> </span></div> </div></div></div></div></div><div content-id="DSM024C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Schembri</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Klemm</div> <div class="given-names">P.</div></div>. </div><div class="article-title"> Antigen 43 and type 1 fimbriae determine colony morphology of <em>Escherichia coli</em> K-12 </div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">1089</div>-<div class="lpage">1095</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antigen%2043%20and%20type%201%20fimbriae%20determine%20colony%20morphology%20of%20Escherichia%20coli%20K-12&amp;author=H.%20Hasman&amp;author=M.%20A.%20Schembri&amp;author=P.%20Klemm&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=1089-1095" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.4.1089-1095.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.4.1089-1095.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.4.1089-1095.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10648536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moore</div> <div class="given-names">G. F.</div></div>,  <div class="name"><div class="surname">Hamiltion</div> <div class="given-names">M. S.</div></div>,  <div class="name"><div class="surname">Holah</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Woordward</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Humpherey</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">Lappin-Scott</div> <div class="given-names">H. M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Gilbert</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Identification of genes involved in biofilm formation, In</div>, <div class="source">Biofilm Community Interactions: Chance or Necessity</div> , <div class="year">2001</div><div class="publisher-loc">Cardiff</div><div class="publisher-name">Bioline</div>(pg. <div class="fpage">119</div>-<div class="lpage">129</div>)</div></div></div></div><div content-id="DSM024C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Onufryk</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">F. C.</div></div>. </div><div class="article-title">Characterization of six lipoproteins in the sigma E regulon</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">4552</div>-<div class="lpage">4561</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20six%20lipoproteins%20in%20the%20sigma%20E%20regulon&amp;author=C.%20Onufryk&amp;author=M.%20L.%20Crouch&amp;author=F.%20C.%20Fang&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=4552-4561" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.13.4552-4561.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.13.4552-4561.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.13.4552-4561.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15968066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tenorio</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kitakawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Baba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Isono</div> <div class="given-names">K.</div></div>. </div><div class="article-title"> Systematic characterization of <em>E. coli</em> genes/ORFs affecting biofilm formation </div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2003</div>, vol. <div class="volume">225</div> (pg. <div class="fpage">107</div>-<div class="lpage">114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20characterization%20of%20E.%20coli%20genes%2FORFs%20affecting%20biofilm%20formation&amp;author=E.%20Tenorio&amp;author=T.%20Saeki&amp;author=K.%20Fujita&amp;author=M.%20Kitakawa&amp;author=T.%20Baba&amp;author=H.%20Mori&amp;author=K.%20Isono&amp;publication_year=2003&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=225&amp;pages=107-114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1097(03)00507-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1097(03)00507-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1097(03)00507-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12900028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Houdt</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Michiels</div> <div class="given-names">C. W.</div></div>. </div><div class="article-title"> Role of bacterial cell surface structures in <em>Escherichia coli</em> biofilm formation </div>, <div class="source">Res. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">626</div>-<div class="lpage">633</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20bacterial%20cell%20surface%20structures%20in%20Escherichia%20coli%20biofilm%20formation&amp;author=R.%20Van%20Houdt&amp;author=C.%20W.%20Michiels&amp;publication_year=2005&amp;journal=Res.%20Microbiol.&amp;volume=156&amp;pages=626-633" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2005.02.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2005.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2005.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15950122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chilcott</div> <div class="given-names">G. S.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">K. T.</div></div>. </div><div class="article-title"> Coupling of flagellar gene expression to flagellar Assembly in <em>Salmonella enterica</em> Serovar Typhimurium and <em>Escherichia coli</em></div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">694</div>-<div class="lpage">708</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coupling%20of%20flagellar%20gene%20expression%20to%20flagellar%20Assembly%20in%20Salmonella%20enterica%20Serovar%20Typhimurium%20and%20Escherichia%20coli&amp;author=G.%20S.%20Chilcott&amp;author=K.%20T.%20Hughes&amp;publication_year=2000&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=64&amp;pages=694-708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.64.4.694-708.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.64.4.694-708.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.64.4.694-708.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11104815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minamino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Iino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kutsukake</div> <div class="given-names">K.</div></div>. </div><div class="article-title"> Molecular characterization of the <em>Salmonella typhimurium flhB</em> operon and its protein products </div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">7630</div>-<div class="lpage">7637</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20the%20Salmonella%20typhimurium%20flhB%20operon%20and%20its%20protein%20products&amp;author=T.%20Minamino&amp;author=T.%20Iino&amp;author=K.%20Kutsukake&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=7630-7637" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.24.7630-7637.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.24.7630-7637.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.24.7630-7637.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8002587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schoenhals</div> <div class="given-names">G. J.</div></div>,  <div class="name"><div class="surname">Macnab</div> <div class="given-names">R. M.</div></div>. </div><div class="article-title"> FliL is a membrane-associated component of the flagellar basal body of <em>Salmonella</em></div>, <div class="source">Microbiology</div> , <div class="year">1999</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">1769</div>-<div class="lpage">1775</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FliL%20is%20a%20membrane-associated%20component%20of%20the%20flagellar%20basal%20body%20of%20Salmonella&amp;author=G.%20J.%20Schoenhals&amp;author=R.%20M.%20Macnab&amp;publication_year=1999&amp;journal=Microbiology&amp;volume=145&amp;pages=1769-1775" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/13500872-145-7-1769" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F13500872-145-7-1769" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F13500872-145-7-1769"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10439416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennett</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Substrate complexes and domain organization of the Salmonella flagellar export chaperones FlgN and FliT</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">781</div>-<div class="lpage">791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate%20complexes%20and%20domain%20organization%20of%20the%20Salmonella%20flagellar%20export%20chaperones%20FlgN%20and%20FliT&amp;author=J.%20C.%20Bennett&amp;author=J.%20Thomas&amp;author=G.%20M.%20Fraser&amp;author=C.%20Hughes&amp;publication_year=2001&amp;journal=Mol.%20Microbiol.&amp;volume=39&amp;pages=781-791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02268.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02268.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02268.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11169117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stafford</div> <div class="given-names">G. P.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">C.</div></div>. </div><div class="article-title"><em>Salmonella typhimurium flhE</em> , a conserved flagellar regulon gene required for swarming </div>, <div class="source">Microbiology</div> , <div class="year">2007</div>, vol. <div class="volume">153</div> <div class="issue">Pt 2</div>(pg. <div class="fpage">541</div>-<div class="lpage">547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salmonella%20typhimurium%20flhE%20%2C%20a%20conserved%20flagellar%20regulon%20gene%20required%20for%20swarming&amp;author=G.%20P.%20Stafford&amp;author=C.%20Hughes&amp;publication_year=2007&amp;journal=Microbiology&amp;volume=153&amp;pages=541-547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2006/002576-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2006%2F002576-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2006%2F002576-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17259626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bardwell</div> <div class="given-names">J. C.</div></div>. </div><div class="article-title">Mutational analysis of the disulfide catalysts DsbA and DsbB</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">1504</div>-<div class="lpage">1510</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20analysis%20of%20the%20disulfide%20catalysts%20DsbA%20and%20DsbB&amp;author=J.%20Tan&amp;author=Y.%20Lu&amp;author=J.%20C.%20Bardwell&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=1504-1510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.4.1504-1510.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.4.1504-1510.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.4.1504-1510.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15687215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dailey</div> <div class="given-names">F. E</div></div>,  <div class="name"><div class="surname">Berg</div> <div class="given-names">H. C.</div></div>. </div><div class="article-title"> Mutants in disulfide bond formation that disrupt flagellar assembly in <em>Escherichia coli</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">1043</div>-<div class="lpage">1047</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutants%20in%20disulfide%20bond%20formation%20that%20disrupt%20flagellar%20assembly%20in%20Escherichia%20coli&amp;author=F.%20E%20Dailey&amp;author=H.%20C.%20Berg&amp;publication_year=1993&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=1043-1047" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.3.1043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.3.1043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.3.1043"> </span></div> </div></div></div></div></div><div content-id="DSM024C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Russell</div> <div class="given-names">P. W.</div></div>,  <div class="name"><div class="surname">Orndorf</div> <div class="given-names">P. E.</div></div>. </div><div class="article-title"> Lesions in two <em>Escherichia coli</em> type 1 pilus genes alter pilus number and length without affecting receptor binding </div>, <div class="source">J. Bacteriol.</div> , <div class="year">1992</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">5923</div>-<div class="lpage">5935</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lesions%20in%20two%20Escherichia%20coli%20type%201%20pilus%20genes%20alter%20pilus%20number%20and%20length%20without%20affecting%20receptor%20binding&amp;author=P.%20W.%20Russell&amp;author=P.%20E.%20Orndorf&amp;publication_year=1992&amp;journal=J.%20Bacteriol.&amp;volume=174&amp;pages=5923-5935" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1355769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galley</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Leathart</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Blomfield</div> <div class="given-names">I. C.</div></div>. </div><div class="article-title"> Interaction of FimB and FimE with the fim switch that controls the phase variation of type 1 fimbriae in <em>Escherichia coli</em> K-12 </div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1996</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">725</div>-<div class="lpage">738</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20of%20FimB%20and%20FimE%20with%20the%20fim%20switch%20that%20controls%20the%20phase%20variation%20of%20type%201%20fimbriae%20in%20Escherichia%20coli%20K-12&amp;author=D.%20L.%20Galley&amp;author=J.%20Leathart&amp;author=I.%20C.%20Blomfield&amp;publication_year=1996&amp;journal=Mol.%20Microbiol.&amp;volume=21&amp;pages=725-738" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.311388.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1996.311388.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1996.311388.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8878036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Valenski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Spears</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Horton</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Orndorff</div> <div class="given-names">P. E.</div></div>. </div><div class="article-title"> The Product of the <em>fimI</em> gene is necessary for <em>Escherichia coli</em> type 1 pilus biosynthesis </div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">5007</div>-<div class="lpage">5011</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Product%20of%20the%20fimI%20gene%20is%20necessary%20for%20Escherichia%20coli%20type%201%20pilus%20biosynthesis&amp;author=M.%20Valenski&amp;author=S.%20L.%20Harris&amp;author=P.%20A.%20Spears&amp;author=J.%20R.%20Horton&amp;author=P.%20E.%20Orndorff&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=5007-5011" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.16.5007-5011.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.16.5007-5011.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.16.5007-5011.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12897022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barnhart</div> <div class="given-names">M. M.</div></div>,  <div class="name"><div class="surname">Chapman</div> <div class="given-names">M. R.</div></div>. </div><div class="article-title">Curli biogenesis and function</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">131</div>-<div class="lpage">147</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Curli%20biogenesis%20and%20function&amp;author=M.%20M.%20Barnhart&amp;author=M.%20R.%20Chapman&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=60&amp;pages=131-147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.60.080805.142106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.60.080805.142106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.60.080805.142106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16704339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">El-Kazzaz</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Morita</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tagami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Aiba</div> <div class="given-names">H.</div></div>. </div><div class="article-title"> Metabolic block at early stages of the glycolytic pathway activates the Rcs phosphorelay system via increased synthesis of dTDP-glucose in <em>Escherichia coli</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">1117</div>-<div class="lpage">1128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20block%20at%20early%20stages%20of%20the%20glycolytic%20pathway%20activates%20the%20Rcs%20phosphorelay%20system%20via%20increased%20synthesis%20of%20dTDP-glucose%20in%20Escherichia%20coli&amp;author=W.%20El-Kazzaz&amp;author=T.%20Morita&amp;author=H.%20Tagami&amp;author=T.%20Inada&amp;author=H.%20Aiba&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=51&amp;pages=1117-1128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03888.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03888.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03888.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14763984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heinrichs</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Yethon</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Whitfield</div> <div class="given-names">C.</div></div>. </div><div class="article-title"> Molecular basis for structural diversity in the core regions of the lipopolysaccharides of <em>Escherichia coli</em> and <em>Salmonella enterica</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">221</div>-<div class="lpage">232</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20for%20structural%20diversity%20in%20the%20core%20regions%20of%20the%20lipopolysaccharides%20of%20Escherichia%20coli%20and%20Salmonella%20enterica&amp;author=D.%20E.%20Heinrichs&amp;author=J.%20A.%20Yethon&amp;author=C.%20Whitfield&amp;publication_year=1998&amp;journal=Mol.%20Microbiol.&amp;volume=30&amp;pages=221-232" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.01063.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1998.01063.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1998.01063.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9791168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnaitman</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Klena</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">Genetics of lipopolysaccharide biosynthesis in enteric bacteria</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1993</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">655</div>-<div class="lpage">682</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20lipopolysaccharide%20biosynthesis%20in%20enteric%20bacteria&amp;author=C.%20A.%20Schnaitman&amp;author=J.%20D.%20Klena&amp;publication_year=1993&amp;journal=Microbiol.%20Rev.&amp;volume=57&amp;pages=655-682" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7504166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beloin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Michaelis</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lindner</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Landini</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hacker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ghigo</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Dobrindt</div> <div class="given-names">U.</div></div>. </div><div class="article-title"> The transcriptional antiterminator RfaH represses biofilm formation in <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">1316</div>-<div class="lpage">1331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20antiterminator%20RfaH%20represses%20biofilm%20formation%20in%20Escherichia%20coli&amp;author=C.%20Beloin&amp;author=K.%20Michaelis&amp;author=K.%20Lindner&amp;author=P.%20Landini&amp;author=J.%20Hacker&amp;author=J.%20M.%20Ghigo&amp;author=U.%20Dobrindt&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=1316-1331" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.4.1316-1331.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.4.1316-1331.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.4.1316-1331.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16452414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Behrens</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Maier</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">de Cook</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Schmid</div> <div class="given-names">F. X.</div></div>,  <div class="name"><div class="surname">Gross</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title"> The surA periplasmic PPIase lacking its paruvilin domains function <em>in vivo</em> and has chaperone activities </div>, <div class="source">EMBO J.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">285</div>-<div class="lpage">294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20surA%20periplasmic%20PPIase%20lacking%20its%20paruvilin%20domains%20function%20in%20vivo%20and%20has%20chaperone%20activities&amp;author=S.%20Behrens&amp;author=R.%20Maier&amp;author=H.%20de%20Cook&amp;author=F.%20X.%20Schmid&amp;author=C.%20A.%20Gross&amp;publication_year=2001&amp;journal=EMBO%20J.&amp;volume=20&amp;pages=285-294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/20.1.285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F20.1.285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F20.1.285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11226178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM024C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM024C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouviere</div> <div class="given-names">P. E.</div></div>,  <div class="name"><div class="surname">Gross</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">SurA, a periplasmic protein with peptidyl-prolyl isomerase activity, participates in the assembly of outer membrane porins</div>, <div class="source">Genes Dev.</div> , <div class="year">1996</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">3170</div>-<div class="lpage">3182</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SurA%2C%20a%20periplasmic%20protein%20with%20peptidyl-prolyl%20isomerase%20activity%2C%20participates%20in%20the%20assembly%20of%20outer%20membrane%20porins&amp;author=P.%20E.%20Rouviere&amp;author=C.%20A.%20Gross&amp;publication_year=1996&amp;journal=Genes%20Dev.&amp;volume=10&amp;pages=3170-3182" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.10.24.3170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.10.24.3170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.10.24.3170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8985185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=350369&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">549</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000254115800002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">54</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="350369"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000254115800002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f6%2f237%2f350369">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000254115800002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (54)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F6%2F237%2F350369" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/350369'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" A Genome-wide Approach to Identify the Genes Involved in Biofilm Formation in E. coli" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f6%2f237%2f350369">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

