

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/451/2462493","@type":"ScholarlyArticle","name":"Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils","datePublished":"2015-10-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv026","keywords":["microbial ecology","functional diversity","sequence capture","454 pyrosequencing","comparative metagenomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/451/2462493","author":[{"name":"Manoharan, Lokeshwaran","affiliation":"1  Department of Biology ,  Lund University ,  Lund 223 62 ,  Sweden","@type":"Person"},{"name":"Kushwaha, Sandeep K.","affiliation":"1  Department of Biology ,  Lund University ,  Lund 223 62 ,  Sweden    2  Bioinformatics Infrastructure for Life Sciences (BILS) ,  Lund University ,  Lund ,  Sweden    3  PlantLink ,  Swedish University of Agriculture Sciences ,  Alnarp ,  Sweden","@type":"Person"},{"name":"Hedlund, Katarina","affiliation":"1  Department of Biology ,  Lund University ,  Lund 223 62 ,  Sweden","@type":"Person"},{"name":"Ahrén, Dag","affiliation":"1  Department of Biology ,  Lund University ,  Lund 223 62 ,  Sweden    2  Bioinformatics Infrastructure for Life Sciences (BILS) ,  Lund University ,  Lund ,  Sweden","@type":"Person"}],"description":"Abstract.  Microbial enzyme diversity is a key to understand many ecosystem processes. Whole metagenome sequencing (WMG) obtains information on functional genes","pageStart":"451","pageEnd":"460","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils" />
    <meta property="og:description" content="Abstract.  Microbial enzyme diversity is a key to understand many ecosystem processes. Whole metagenome sequencing (WMG) obtains information on functional genes" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/451/2462493" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-10-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Manoharan, Lokeshwaran" /><meta name="citation_author_institution" content="Department of Biology, Lund University, Lund 223 62, Sweden" /><meta name="citation_author" content="Kushwaha, Sandeep K." /><meta name="citation_author_institution" content="Department of Biology, Lund University, Lund 223 62, Sweden" /><meta name="citation_author_institution" content="Bioinformatics Infrastructure for Life Sciences (BILS), Lund University, Lund, Sweden" /><meta name="citation_author_institution" content="PlantLink, Swedish University of Agriculture Sciences, Alnarp, Sweden" /><meta name="citation_author" content="Hedlund, Katarina" /><meta name="citation_author_institution" content="Department of Biology, Lund University, Lund 223 62, Sweden" /><meta name="citation_author" content="Ahrén, Dag" /><meta name="citation_author_institution" content="Department of Biology, Lund University, Lund 223 62, Sweden" /><meta name="citation_author_institution" content="Bioinformatics Infrastructure for Life Sciences (BILS), Lund University, Lund, Sweden" /><meta name="citation_title" content="Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils" /><meta name="citation_firstpage" content="451" /><meta name="citation_lastpage" content="460" /><meta name="citation_doi" content="10.1093/dnares/dsv026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The taxonomic and functional diversity of microbes at a temperate coastal site: a ‘multi-omic’ study of seasonal and diel temporal variation;  citation_journal_title=PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e15545" /><meta name="citation_reference" content="citation_title=Vive la difference: plant functional diversity matters to ecosystem processes;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2001;  citation_volume=16;  citation_pages=646-55; " /><meta name="citation_reference" content="citation_title=Phylogenetic constraints on ecosystem functioning;  citation_journal_title=Nat. Commun.;  citation_year=2012;  citation_volume=3;  citation_pages=1117" /><meta name="citation_reference" content="citation_title=Biodiversity and ecosystem function in soil;  citation_journal_title=Funct. Ecol.;  citation_year=2005;  citation_volume=19;  citation_pages=369-77; " /><meta name="citation_reference" content="citation_title=Linking species richness, biodiversity and ecosystem function in soil systems;  citation_journal_title=Pedobiologia;  citation_year=2005;  citation_volume=49;  citation_pages=479-97; " /><meta name="citation_reference" content="citation_title=Soil enzymes in a changing environment: current knowledge and future directions;  citation_journal_title=Soil Biol. Biochem.;  citation_year=2013;  citation_volume=58;  citation_pages=216-34; " /><meta name="citation_reference" content="citation_title=Microbial metagenomics: beyond the genome;  citation_journal_title=Annu. Rev. Mar. Sci.;  citation_year=2011;  citation_volume=3;  citation_pages=347-71; " /><meta name="citation_reference" content="citation_title=A bioinformatician's guide to metagenomics;  citation_journal_title=Microbiol. Mol. Biol. Rev.;  citation_year=2008;  citation_volume=72;  citation_pages=557-78; " /><meta name="citation_reference" content="citation_title=A primer on metagenomics;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000667" /><meta name="citation_reference" content="citation_title=Molecular detection and diversity of xylanase genes in alpine tundra soil;  citation_journal_title=Appl. Microbiol. Biot.;  citation_year=2010;  citation_volume=87;  citation_pages=1383-93; " /><meta name="citation_reference" content="citation_title=The ammonia monooxygenase structural gene amoA as a functional marker: molecular fine-scale analysis of natural ammonia-oxidizing populations;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=1997;  citation_volume=63;  citation_pages=4704-12; " /><meta name="citation_reference" content="citation_title=Development and application of functional gene arrays for microbial community analysis;  citation_journal_title=T. Nonferr. Metal. Soc.;  citation_year=2008;  citation_volume=18;  citation_pages=1319-27; " /><meta name="citation_reference" content="citation_title=Analysis of microbial communities by functional gene arrays; citation_publisher=Springer, The Netherlands;  citation_title=Geomicrobiology: molecular and environmental perspective;  citation_year=2010;  citation_pages=109-26; " /><meta name="citation_reference" content="citation_title=Targeted metagenomics: a high-resolution metagenomics approach for specific gene clusters in complex microbial communities;  citation_journal_title=Environ. Microbiol.;  citation_year=2012;  citation_volume=14;  citation_pages=13-22; " /><meta name="citation_reference" content="citation_title=Direct selection of human genomic loci by microarray hybridization;  citation_journal_title=Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=903-5; " /><meta name="citation_reference" content="citation_title=Targeted investigation of the Neandertal genome by array-based sequence capture;  citation_journal_title=Science;  citation_year=2010;  citation_volume=328;  citation_pages=723-5; " /><meta name="citation_reference" content="citation_title=Targeted capture and massively parallel sequencing of 12 human exomes;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=461;  citation_pages=272-6; " /><meta name="citation_reference" content="citation_title=Targeted capture and next-generation sequencing identifies C9orf75, encoding taperin, as the mutated gene in nonsyndromic deafness DFNB79;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2010;  citation_volume=86;  citation_pages=378-88; " /><meta name="citation_reference" content="citation_title=A high frequency of activating extracellular domain ERBB2 (HER2) mutation in micropapillary urothelial carcinoma;  citation_journal_title=Clin. Cancer Res.;  citation_year=2014;  citation_volume=20;  citation_pages=68-75; " /><meta name="citation_reference" content="citation_title=Targeted next generation sequencing of clinically significant gene mutations and translocations in leukemia;  citation_journal_title=Modern Pathol.;  citation_year=2012;  citation_volume=25;  citation_pages=795-804; " /><meta name="citation_reference" content="citation_title=Transcriptome-based exon capture enables highly cost-effective comparative genomic data collection at moderate evolutionary scales;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=403" /><meta name="citation_reference" content="citation_title=Enrichment of genomic DNA for polymorphism detection in a non-model highly polyploid crop plant;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=657-67; " /><meta name="citation_reference" content="citation_title=Exome-wide DNA capture and next generation sequencing in domestic and wild species;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=347" /><meta name="citation_reference" content="citation_title=Gene capture coupled to high-throughput sequencing as a strategy for targeted metagenome exploration;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=185-96; " /><meta name="citation_reference" content="citation_title=The Carbohydrate-Active EnZymes database (CAZy): an expert resource for Glycogenomics;  citation_journal_title=Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D233-8; " /><meta name="citation_reference" content="citation_title=FOLy: an integrated database for the classification and functional annotation of fungal oxidoreductases potentially involved in the degradation of lignin and related aromatic compounds;  citation_journal_title=Fungal Genet. Biol.;  citation_year=2008;  citation_volume=45;  citation_pages=638-45; " /><meta name="citation_reference" content="citation_title=MEROPS: the database of proteolytic enzymes, their substrates and inhibitors;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D343-50; " /><meta name="citation_reference" content="citation_title=MetCap: a bioinformatics probe design pipeline for large-scale targeted metagenomics;  citation_journal_title=BMC Bioinformatics;  citation_year=2015;  citation_volume=16;  citation_pages=65" /><meta name="citation_reference" content="citation_title=Intensive agriculture reduces soil biodiversity across Europe;  citation_journal_title=Global Chang. Biol;  citation_year=2014;  citation_volume=21;  citation_pages=973-85; " /><meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP and related tools;  citation_journal_title=Nat. Protoc.;  citation_year=2007;  citation_volume=2;  citation_pages=953-71; " /><meta name="citation_reference" content="citation_title=Transitions in bacterial communities along the 2000km salinity gradient of the Baltic Sea;  citation_journal_title=Isme J.;  citation_year=2011;  citation_volume=5;  citation_pages=1571-9; " /><meta name="citation_reference" content="citation_title=The metagenomics RAST server - a public resource for the automatic phylogenetic and functional analysis of metagenomes;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=386" /><meta name="citation_reference" content="citation_title=Systematic artifacts in metagenomes from complex microbial communities;  citation_journal_title=Isme J.;  citation_year=2009;  citation_volume=3;  citation_pages=1314-7; " /><meta name="citation_reference" content="citation_title=The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5691-702; " /><meta name="citation_reference" content="citation_title=CAZymes Analysis Toolkit (CAT): Web service for searching and analyzing carbohydrate-active enzymes in a newly sequenced organism using CAZy database;  citation_journal_title=Glycobiology;  citation_year=2010;  citation_volume=20;  citation_pages=1574-84; " /><meta name="citation_reference" content="citation_title=QIIME allows analysis of high-throughput community sequencing data;  citation_journal_title=Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=335-6; " /><meta name="citation_reference" content="citation_title=An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea;  citation_journal_title=Isme J.;  citation_year=2012;  citation_volume=6;  citation_pages=610-8; " /><meta name="citation_reference" content="  Braithwaite D., Keegan K.  2014 , matR: Metagenomics Analysis Tools for R , http://CRAN.R-project.org/package=matR(1 August 2014, date last accessed). " /><meta name="citation_reference" content="citation_title=STAMP: statistical analysis of taxonomic and functional profiles;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3123-4; " /><meta name="citation_reference" content="citation_title=Seeing the forest for the genes: using metagenomics to infer the aggregated traits of microbial communities;  citation_journal_title=Front. Microbiol.;  citation_year=2014;  citation_volume=5;  citation_pages=614" /><meta name="citation_reference" content="citation_title=Systematic exploration of error sources in pyrosequencing flowgram data;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=i304-9; " /><meta name="citation_reference" content="citation_title=Synergistic use of plant-prokaryote comparative genomics for functional annotations;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_issue=Suppl 1;  citation_pages=S2" /><meta name="citation_reference" content="citation_title=Functional metagenomics to mine the human gut microbiome for dietary fiber catabolic enzymes;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1605-12; " /><meta name="citation_reference" content="citation_title=PCR primers to study the diversity of expressed fungal genes encoding lignocellulolytic enzymes in soils using high-throughput sequencing;  citation_journal_title=PloS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e116264" /><meta name="citation_reference" content="citation_title=Structural and evolutionary aspects of two families of non-catalytic domains present in starch and glycogen binding proteins from microbes, plants and animals;  citation_journal_title=Enzyme Microb. Tech.;  citation_year=2011;  citation_volume=49;  citation_pages=429-40; " /><meta name="citation_reference" content="citation_title=Identification and characterization of a novel trehalose synthase gene derived from saline-alkali soil metagenomes;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e77437" /><meta name="citation_reference" content="citation_title=Three genomes from the phylum Acidobacteria provide insight into the lifestyles of these microorganisms in soils;  citation_journal_title=Appl. Environ. Microb.;  citation_year=2009;  citation_volume=75;  citation_pages=2046-56; " /><meta name="citation_reference" content="citation_title=Acidobacteria form a coherent but highly diverse group within the bacterial domain: evidence from environmental genomics;  citation_journal_title=Mol. Microbiol.;  citation_year=2003;  citation_volume=50;  citation_pages=563-75; " /><meta name="citation_reference" content="citation_title=Insights into the phylogeny and coding potential of microbial dark matter;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=499;  citation_pages=431-7; " /><meta name="citation_reference" content="citation_title=A phylogeny-driven genomic encyclopaedia of Bacteria and Archaea;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=462;  citation_pages=1056-60; " /><meta name="citation_reference" content="citation_title=The functional potential of high Arctic permafrost revealed by metagenomic sequencing, qPCR and microarray analyses;  citation_journal_title=Isme J.;  citation_year=2010;  citation_volume=4;  citation_pages=1206-14; " /><meta name="citation_reference" content="citation_title=Strategies for enhancing the effectiveness of metagenomic-based enzyme discovery in lignocellulolytic microbial communities;  citation_journal_title=Bioenerg. Res.;  citation_year=2010;  citation_volume=3;  citation_pages=146-58; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis of the bioremediation of diesel-contaminated Canadian high arctic soils;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e30058" /><meta name="citation_reference" content="citation_title=Biases in the experimental annotations of protein function and their effect on our understanding of protein function space;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003063" /><meta name="citation_reference" content="citation_title=Tackling soil diversity with the assembly of large, complex metagenomes (vol 111, pg 4904, 2014);  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=6115" /><meta name="citation_reference" content="citation_title=GeoChip: a comprehensive microarray for investigating biogeochemical, ecological and environmental processes;  citation_journal_title=Isme J.;  citation_year=2007;  citation_volume=1;  citation_pages=67-77; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/451/17463278/dsv026.pdf" /><meta name="description" content="Abstract.  Microbial enzyme diversity is a key to understand many ecosystem processes. Whole metagenome sequencing (WMG) obtains information on functional genes" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/451/2462493" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/451/2462493" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/451/2462493");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv026';
    </script>
<title>
	Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462493" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaODATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFub2hhcmFuLCBMb2tlc2h3YXJhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgTHVuZCBVbml2ZXJzaXR5LCBMdW5kIDIyMyA2MiwgU3dlZGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VzaHdhaGEsIFNhbmRlZXAgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIEx1bmQgVW5pdmVyc2l0eSwgTHVuZCAyMjMgNjIsIFN3ZWRlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIEluZnJhc3RydWN0dXJlIGZvciBMaWZlIFNjaWVuY2VzIChCSUxTKSwgTHVuZCBVbml2ZXJzaXR5LCBMdW5kLCBTd2VkZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudExpbmssIFN3ZWRpc2ggVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSBTY2llbmNlcywgQWxuYXJwLCBTd2VkZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIZWRsdW5kLCBLYXRhcmluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSwgTHVuZCBVbml2ZXJzaXR5LCBMdW5kIDIyMyA2MiwgU3dlZGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWhyw6luLCBEYWciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIEx1bmQgVW5pdmVyc2l0eSwgTHVuZCAyMjMgNjIsIFN3ZWRlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIEluZnJhc3RydWN0dXJlIGZvciBMaWZlIFNjaWVuY2VzIChCSUxTKSwgTHVuZCBVbml2ZXJzaXR5LCBMdW5kLCBTd2VkZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNhcHR1cmVkIG1ldGFnZW5vbWljczogbGFyZ2Utc2NhbGUgdGFyZ2V0aW5nIG9mIGdlbmVzIGJhc2VkIG9uIOKAmHNlcXVlbmNlIGNhcHR1cmXigJkgcmV2ZWFscyBmdW5jdGlvbmFsIGRpdmVyc2l0eSBpbiBzb2lscyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdGF4b25vbWljIGFuZCBmdW5jdGlvbmFsIGRpdmVyc2l0eSBvZiBtaWNyb2JlcyBhdCBhIHRlbXBlcmF0ZSBjb2FzdGFsIHNpdGU6IGEg4oCYbXVsdGktb21pY+KAmSBzdHVkeSBvZiBzZWFzb25hbCBhbmQgZGllbCB0ZW1wb3JhbCB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTU1NDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXZlIGxhIGRpZmZlcmVuY2U6IHBsYW50IGZ1bmN0aW9uYWwgZGl2ZXJzaXR5IG1hdHRlcnMgdG8gZWNvc3lzdGVtIHByb2Nlc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEVjb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjQ2LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgY29uc3RyYWludHMgb24gZWNvc3lzdGVtIGZ1bmN0aW9uaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTExMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9kaXZlcnNpdHkgYW5kIGVjb3N5c3RlbSBmdW5jdGlvbiBpbiBzb2lsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5jdC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zNjktNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmtpbmcgc3BlY2llcyByaWNobmVzcywgYmlvZGl2ZXJzaXR5IGFuZCBlY29zeXN0ZW0gZnVuY3Rpb24gaW4gc29pbCBzeXN0ZW1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QZWRvYmlvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz00NzktOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvaWwgZW56eW1lcyBpbiBhIGNoYW5naW5nIGVudmlyb25tZW50OiBjdXJyZW50IGtub3dsZWRnZSBhbmQgZnV0dXJlIGRpcmVjdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNvaWwgQmlvbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0yMTYtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBtZXRhZ2Vub21pY3M6IGJleW9uZCB0aGUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIE1hci4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGJpb2luZm9ybWF0aWNpYW4ncyBndWlkZSB0byBtZXRhZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9NTU3LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByaW1lciBvbiBtZXRhZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA2NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZGV0ZWN0aW9uIGFuZCBkaXZlcnNpdHkgb2YgeHlsYW5hc2UgZ2VuZXMgaW4gYWxwaW5lIHR1bmRyYSBzb2lsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3QuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9MTM4My05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFtbW9uaWEgbW9ub294eWdlbmFzZSBzdHJ1Y3R1cmFsIGdlbmUgYW1vQSBhcyBhIGZ1bmN0aW9uYWwgbWFya2VyOiBtb2xlY3VsYXIgZmluZS1zY2FsZSBhbmFseXNpcyBvZiBuYXR1cmFsIGFtbW9uaWEtb3hpZGl6aW5nIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9NDcwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGFwcGxpY2F0aW9uIG9mIGZ1bmN0aW9uYWwgZ2VuZSBhcnJheXMgZm9yIG1pY3JvYmlhbCBjb21tdW5pdHkgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVQuIE5vbmZlcnIuIE1ldGFsLiBTb2MuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTMxOS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgbWljcm9iaWFsIGNvbW11bml0aWVzIGJ5IGZ1bmN0aW9uYWwgZ2VuZSBhcnJheXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgVGhlIE5ldGhlcmxhbmRzOyAgY2l0YXRpb25fdGl0bGU9R2VvbWljcm9iaW9sb2d5OiBtb2xlY3VsYXIgYW5kIGVudmlyb25tZW50YWwgcGVyc3BlY3RpdmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl9wYWdlcz0xMDktMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIG1ldGFnZW5vbWljczogYSBoaWdoLXJlc29sdXRpb24gbWV0YWdlbm9taWNzIGFwcHJvYWNoIGZvciBzcGVjaWZpYyBnZW5lIGNsdXN0ZXJzIGluIGNvbXBsZXggbWljcm9iaWFsIGNvbW11bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpcmVjdCBzZWxlY3Rpb24gb2YgaHVtYW4gZ2Vub21pYyBsb2NpIGJ5IG1pY3JvYXJyYXkgaHlicmlkaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz05MDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgaW52ZXN0aWdhdGlvbiBvZiB0aGUgTmVhbmRlcnRhbCBnZW5vbWUgYnkgYXJyYXktYmFzZWQgc2VxdWVuY2UgY2FwdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjg7ICBjaXRhdGlvbl9wYWdlcz03MjMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgY2FwdHVyZSBhbmQgbWFzc2l2ZWx5IHBhcmFsbGVsIHNlcXVlbmNpbmcgb2YgMTIgaHVtYW4gZXhvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYxOyAgY2l0YXRpb25fcGFnZXM9MjcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIGNhcHR1cmUgYW5kIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGlkZW50aWZpZXMgQzlvcmY3NSwgZW5jb2RpbmcgdGFwZXJpbiwgYXMgdGhlIG11dGF0ZWQgZ2VuZSBpbiBub25zeW5kcm9taWMgZGVhZm5lc3MgREZOQjc5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz0zNzgtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaCBmcmVxdWVuY3kgb2YgYWN0aXZhdGluZyBleHRyYWNlbGx1bGFyIGRvbWFpbiBFUkJCMiAoSEVSMikgbXV0YXRpb24gaW4gbWljcm9wYXBpbGxhcnkgdXJvdGhlbGlhbCBjYXJjaW5vbWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNsaW4uIENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjgtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIG5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIG9mIGNsaW5pY2FsbHkgc2lnbmlmaWNhbnQgZ2VuZSBtdXRhdGlvbnMgYW5kIHRyYW5zbG9jYXRpb25zIGluIGxldWtlbWlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2Rlcm4gUGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTc5NS04MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUtYmFzZWQgZXhvbiBjYXB0dXJlIGVuYWJsZXMgaGlnaGx5IGNvc3QtZWZmZWN0aXZlIGNvbXBhcmF0aXZlIGdlbm9taWMgZGF0YSBjb2xsZWN0aW9uIGF0IG1vZGVyYXRlIGV2b2x1dGlvbmFyeSBzY2FsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVucmljaG1lbnQgb2YgZ2Vub21pYyBETkEgZm9yIHBvbHltb3JwaGlzbSBkZXRlY3Rpb24gaW4gYSBub24tbW9kZWwgaGlnaGx5IHBvbHlwbG9pZCBjcm9wIHBsYW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTY1Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhvbWUtd2lkZSBETkEgY2FwdHVyZSBhbmQgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgaW4gZG9tZXN0aWMgYW5kIHdpbGQgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBjYXB0dXJlIGNvdXBsZWQgdG8gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgYXMgYSBzdHJhdGVneSBmb3IgdGFyZ2V0ZWQgbWV0YWdlbm9tZSBleHBsb3JhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xODUtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDYXJib2h5ZHJhdGUtQWN0aXZlIEVuWnltZXMgZGF0YWJhc2UgKENBWnkpOiBhbiBleHBlcnQgcmVzb3VyY2UgZm9yIEdseWNvZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQyMzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rk9MeTogYW4gaW50ZWdyYXRlZCBkYXRhYmFzZSBmb3IgdGhlIGNsYXNzaWZpY2F0aW9uIGFuZCBmdW5jdGlvbmFsIGFubm90YXRpb24gb2YgZnVuZ2FsIG94aWRvcmVkdWN0YXNlcyBwb3RlbnRpYWxseSBpbnZvbHZlZCBpbiB0aGUgZGVncmFkYXRpb24gb2YgbGlnbmluIGFuZCByZWxhdGVkIGFyb21hdGljIGNvbXBvdW5kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTYzOC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVST1BTOiB0aGUgZGF0YWJhc2Ugb2YgcHJvdGVvbHl0aWMgZW56eW1lcywgdGhlaXIgc3Vic3RyYXRlcyBhbmQgaW5oaWJpdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDM0My01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0Q2FwOiBhIGJpb2luZm9ybWF0aWNzIHByb2JlIGRlc2lnbiBwaXBlbGluZSBmb3IgbGFyZ2Utc2NhbGUgdGFyZ2V0ZWQgbWV0YWdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz02NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVuc2l2ZSBhZ3JpY3VsdHVyZSByZWR1Y2VzIHNvaWwgYmlvZGl2ZXJzaXR5IGFjcm9zcyBFdXJvcGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdsb2JhbCBDaGFuZy4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTk3My04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgcHJvdGVpbnMgaW4gdGhlIGNlbGwgdXNpbmcgVGFyZ2V0UCwgU2lnbmFsUCBhbmQgcmVsYXRlZCB0b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz05NTMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zaXRpb25zIGluIGJhY3RlcmlhbCBjb21tdW5pdGllcyBhbG9uZyB0aGUgMjAwMGttIHNhbGluaXR5IGdyYWRpZW50IG9mIHRoZSBCYWx0aWMgU2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Jc21lIEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtZXRhZ2Vub21pY3MgUkFTVCBzZXJ2ZXIgLSBhIHB1YmxpYyByZXNvdXJjZSBmb3IgdGhlIGF1dG9tYXRpYyBwaHlsb2dlbmV0aWMgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgbWV0YWdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mzg2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBhcnRpZmFjdHMgaW4gbWV0YWdlbm9tZXMgZnJvbSBjb21wbGV4IG1pY3JvYmlhbCBjb21tdW5pdGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SXNtZSBKLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTMxNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3Vic3lzdGVtcyBhcHByb2FjaCB0byBnZW5vbWUgYW5ub3RhdGlvbiBhbmQgaXRzIHVzZSBpbiB0aGUgcHJvamVjdCB0byBhbm5vdGF0ZSAxMDAwIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTU2OTEtNzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DQVp5bWVzIEFuYWx5c2lzIFRvb2xraXQgKENBVCk6IFdlYiBzZXJ2aWNlIGZvciBzZWFyY2hpbmcgYW5kIGFuYWx5emluZyBjYXJib2h5ZHJhdGUtYWN0aXZlIGVuenltZXMgaW4gYSBuZXdseSBzZXF1ZW5jZWQgb3JnYW5pc20gdXNpbmcgQ0FaeSBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2x5Y29iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTU3NC04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UUlJTUUgYWxsb3dzIGFuYWx5c2lzIG9mIGhpZ2gtdGhyb3VnaHB1dCBjb21tdW5pdHkgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTMzNS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbXByb3ZlZCBHcmVlbmdlbmVzIHRheG9ub215IHdpdGggZXhwbGljaXQgcmFua3MgZm9yIGVjb2xvZ2ljYWwgYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNlcyBvZiBiYWN0ZXJpYSBhbmQgYXJjaGFlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SXNtZSBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjEwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgQnJhaXRod2FpdGUgRC4sIEtlZWdhbiBLLiAgMjAxNCAsIG1hdFI6IE1ldGFnZW5vbWljcyBBbmFseXNpcyBUb29scyBmb3IgUiAsIGh0dHA6Ly9DUkFOLlItcHJvamVjdC5vcmcvcGFja2FnZT1tYXRSKDEgQXVndXN0IDIwMTQsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNUQU1QOiBzdGF0aXN0aWNhbCBhbmFseXNpcyBvZiB0YXhvbm9taWMgYW5kIGZ1bmN0aW9uYWwgcHJvZmlsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzEyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWVpbmcgdGhlIGZvcmVzdCBmb3IgdGhlIGdlbmVzOiB1c2luZyBtZXRhZ2Vub21pY3MgdG8gaW5mZXIgdGhlIGFnZ3JlZ2F0ZWQgdHJhaXRzIG9mIG1pY3JvYmlhbCBjb21tdW5pdGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTYxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgZXhwbG9yYXRpb24gb2YgZXJyb3Igc291cmNlcyBpbiBweXJvc2VxdWVuY2luZyBmbG93Z3JhbSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPWkzMDQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3luZXJnaXN0aWMgdXNlIG9mIHBsYW50LXByb2thcnlvdGUgY29tcGFyYXRpdmUgZ2Vub21pY3MgZm9yIGZ1bmN0aW9uYWwgYW5ub3RhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsIDE7ICBjaXRhdGlvbl9wYWdlcz1TMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgbWV0YWdlbm9taWNzIHRvIG1pbmUgdGhlIGh1bWFuIGd1dCBtaWNyb2Jpb21lIGZvciBkaWV0YXJ5IGZpYmVyIGNhdGFib2xpYyBlbnp5bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE2MDUtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBDUiBwcmltZXJzIHRvIHN0dWR5IHRoZSBkaXZlcnNpdHkgb2YgZXhwcmVzc2VkIGZ1bmdhbCBnZW5lcyBlbmNvZGluZyBsaWdub2NlbGx1bG9seXRpYyBlbnp5bWVzIGluIHNvaWxzIHVzaW5nIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTExNjI2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgYW5kIGV2b2x1dGlvbmFyeSBhc3BlY3RzIG9mIHR3byBmYW1pbGllcyBvZiBub24tY2F0YWx5dGljIGRvbWFpbnMgcHJlc2VudCBpbiBzdGFyY2ggYW5kIGdseWNvZ2VuIGJpbmRpbmcgcHJvdGVpbnMgZnJvbSBtaWNyb2JlcywgcGxhbnRzIGFuZCBhbmltYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fbnp5bWUgTWljcm9iLiBUZWNoLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTQyOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBub3ZlbCB0cmVoYWxvc2Ugc3ludGhhc2UgZ2VuZSBkZXJpdmVkIGZyb20gc2FsaW5lLWFsa2FsaSBzb2lsIG1ldGFnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTc3NDM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyZWUgZ2Vub21lcyBmcm9tIHRoZSBwaHlsdW0gQWNpZG9iYWN0ZXJpYSBwcm92aWRlIGluc2lnaHQgaW50byB0aGUgbGlmZXN0eWxlcyBvZiB0aGVzZSBtaWNyb29yZ2FuaXNtcyBpbiBzb2lsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTIwNDYtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjaWRvYmFjdGVyaWEgZm9ybSBhIGNvaGVyZW50IGJ1dCBoaWdobHkgZGl2ZXJzZSBncm91cCB3aXRoaW4gdGhlIGJhY3RlcmlhbCBkb21haW46IGV2aWRlbmNlIGZyb20gZW52aXJvbm1lbnRhbCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NTYzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBpbnRvIHRoZSBwaHlsb2dlbnkgYW5kIGNvZGluZyBwb3RlbnRpYWwgb2YgbWljcm9iaWFsIGRhcmsgbWF0dGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk5OyAgY2l0YXRpb25fcGFnZXM9NDMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGh5bG9nZW55LWRyaXZlbiBnZW5vbWljIGVuY3ljbG9wYWVkaWEgb2YgQmFjdGVyaWEgYW5kIEFyY2hhZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NjI7ICBjaXRhdGlvbl9wYWdlcz0xMDU2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZnVuY3Rpb25hbCBwb3RlbnRpYWwgb2YgaGlnaCBBcmN0aWMgcGVybWFmcm9zdCByZXZlYWxlZCBieSBtZXRhZ2Vub21pYyBzZXF1ZW5jaW5nLCBxUENSIGFuZCBtaWNyb2FycmF5IGFuYWx5c2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Jc21lIEouOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjA2LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhdGVnaWVzIGZvciBlbmhhbmNpbmcgdGhlIGVmZmVjdGl2ZW5lc3Mgb2YgbWV0YWdlbm9taWMtYmFzZWQgZW56eW1lIGRpc2NvdmVyeSBpbiBsaWdub2NlbGx1bG9seXRpYyBtaWNyb2JpYWwgY29tbXVuaXRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2VuZXJnLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xNDYtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljIGFuYWx5c2lzIG9mIHRoZSBiaW9yZW1lZGlhdGlvbiBvZiBkaWVzZWwtY29udGFtaW5hdGVkIENhbmFkaWFuIGhpZ2ggYXJjdGljIHNvaWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMwMDU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlhc2VzIGluIHRoZSBleHBlcmltZW50YWwgYW5ub3RhdGlvbnMgb2YgcHJvdGVpbiBmdW5jdGlvbiBhbmQgdGhlaXIgZWZmZWN0IG9uIG91ciB1bmRlcnN0YW5kaW5nIG9mIHByb3RlaW4gZnVuY3Rpb24gc3BhY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDMwNjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYWNrbGluZyBzb2lsIGRpdmVyc2l0eSB3aXRoIHRoZSBhc3NlbWJseSBvZiBsYXJnZSwgY29tcGxleCBtZXRhZ2Vub21lcyAodm9sIDExMSwgcGcgNDkwNCwgMjAxNCk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9NjExNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlb0NoaXA6IGEgY29tcHJlaGVuc2l2ZSBtaWNyb2FycmF5IGZvciBpbnZlc3RpZ2F0aW5nIGJpb2dlb2NoZW1pY2FsLCBlY29sb2dpY2FsIGFuZCBlbnZpcm9ubWVudGFsIHByb2Nlc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SXNtZSBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi82LzQ1MS8xNzQ2MzI3OC9kc3YwMjYucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE1pY3JvYmlhbCBlbnp5bWUgZGl2ZXJzaXR5IGlzIGEga2V5IHRvIHVuZGVyc3RhbmQgbWFueSBlY29zeXN0ZW0gcHJvY2Vzc2VzLiBXaG9sZSBtZXRhZ2Vub21lIHNlcXVlbmNpbmcgKFdNRykgb2J0YWlucyBpbmZvcm1hdGlvbiBvbiBmdW5jdGlvbmFsIGdlbmVzIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDYyNDkzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZAUPZQq5aGhU4gBsHaoDOcpYK7JZ" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f451%2f2462493" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f451%2f2462493" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200430">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200430">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200432">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200432">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200436">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200436">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200454">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200454">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200477">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200477">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200486">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200486">Conflict of interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200488">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200490">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200490">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200492">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/439/2462479">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/461/2462501">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lokeshwaran Manoharan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund 223 62</div>, <div class="country">Sweden</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +46 735 99 61 24. E-mail: <a href="mailto:lokeshwaran.manoharan@biol.lu.se" target="_blank">lokeshwaran.manoharan@biol.lu.se</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lokeshwaran+Manoharan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Manoharan L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Manoharan L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lokeshwaran Manoharan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sandeep K. Kushwaha</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund 223 62</div>, <div class="country">Sweden</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Bioinformatics Infrastructure for Life Sciences (BILS)</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund</div>, <div class="country">Sweden</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">PlantLink</div>, <div class="insititution">Swedish University of Agriculture Sciences</div>, <div class="addr-line">Alnarp</div>, <div class="country">Sweden</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sandeep+K.+Kushwaha">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kushwaha S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kushwaha S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sandeep K. Kushwaha</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katarina Hedlund</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund 223 62</div>, <div class="country">Sweden</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katarina+Hedlund">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hedlund K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hedlund K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katarina Hedlund</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dag Ahr&#233;n</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund 223 62</div>, <div class="country">Sweden</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Bioinformatics Infrastructure for Life Sciences (BILS)</div>, <div class="insititution">Lund University</div>, <div class="addr-line">Lund</div>, <div class="country">Sweden</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dag+Ahr%c3%a9n">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ahr&#233;n D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ahr&#233;n D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dag Ahrén</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 451–460, <a href='https://doi.org/10.1093/dnares/dsv026'>https://doi.org/10.1093/dnares/dsv026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 September 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462493" href="/dnaresearch/article-pdf/22/6/451/17463278/dsv026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Lokeshwaran Manoharan, Sandeep K. Kushwaha, Katarina Hedlund, Dag Ahrén;  Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 451–460, <a href='https://doi.org/10.1093/dnares/dsv026'>https://doi.org/10.1093/dnares/dsv026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462493?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Captured%20metagenomics%3A%20large-scale%20targeting%20of%20genes%20based%20on%20%E2%80%98sequence%20capture%E2%80%99%20reveals%20functional%20diversity%20in%20soils&amp;publicationDate=2015-10-21&amp;volumeNum=22&amp;issueNum=6&amp;author=Manoharan%2C%20Lokeshwaran%3B%20Kushwaha%2C%20Sandeep%20K.&amp;startPage=451&amp;endPage=460&amp;contentId=10.1093%2Fdnares%2Fdsv026&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f451%2f2462493" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200430 id="87200430" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Microbial enzyme diversity is a key to understand many ecosystem processes. Whole metagenome sequencing (WMG) obtains information on functional genes, but it is costly and inefficient due to large amount of sequencing that is required. In this study, we have applied a captured metagenomics technique for functional genes in soil microorganisms, as an alternative to WMG. Large-scale targeting of functional genes, coding for enzymes related to organic matter degradation, was applied to two agricultural soil communities through captured metagenomics. Captured metagenomics uses custom-designed, hybridization-based oligonucleotide probes that enrich functional genes of interest in metagenomic libraries where only probe-bound DNA fragments are sequenced. The captured metagenomes were highly enriched with targeted genes while maintaining their target diversity and their taxonomic distribution correlated well with the traditional ribosomal sequencing. The captured metagenomes were highly enriched with genes related to organic matter degradation; at least five times more than similar, publicly available soil WMG projects. This target enrichment technique also preserves the functional representation of the soils, thereby facilitating comparative metagenomics projects. Here, we present the first study that applies the captured metagenomics approach in large scale, and this novel method allows deep investigations of central ecosystem processes by studying functional gene abundances.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial ecology&quot;">microbial ecology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional diversity&quot;">functional diversity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence capture&quot;">sequence capture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;454 pyrosequencing&quot;">454 pyrosequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative metagenomics&quot;">comparative metagenomics</a></div>                            <h2 scrollto-destination=87200432 id="87200432" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The knowledge on the link between functional diversity and species richness is one of the key areas that is lacking for better understanding of ecosystem functioning in soils,<sup><a reveal-id="DSV026C1  DSV026C3" data-open="DSV026C1  DSV026C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Ecosystems such as soil have vast diverse microorganisms with several thousand complex functional capabilities.<sup><a reveal-id="DSV026C4" data-open="DSV026C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV026C5" data-open="DSV026C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Novel technological advancements are thought to be a way towards the understanding of microbial functional diversity in biogeochemical processes more clearly.<sup><a reveal-id="DSV026C6" data-open="DSV026C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Several molecular methods like PCR, RFLP, microarrays and sequencing have been utilized in the field of ecology, and in recent years high-throughput sequencing has proven to be efficient in characterizing the diversity of microorganisms in ecological systems.<sup><a reveal-id="DSV026C7" data-open="DSV026C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Whole metagenomic sequencing is currently limited by its coverage as the interesting regions can form a very low proportion of the whole nucleic acid amounts and will thus not be obtained in the large data set.<sup><a reveal-id="DSV026C8" data-open="DSV026C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV026C9" data-open="DSV026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Amplicon sequencing through PCR primers is an alternative way to specifically obtain the less abundant genomic regions of interest, which becomes harder when there are several thousand functional targets.<sup><a reveal-id="DSV026C10" data-open="DSV026C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV026C11" data-open="DSV026C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Large-scale functional gene microarrays such as the Geochip<sup><a reveal-id="DSV026C12" data-open="DSV026C12" class="link link-ref link-reveal xref-bibr">12</a></sup> has been chosen as an alternative method by targeting several specific genes involved in ecosystem functioning. This method was useful but also had problems related to specificity and sensitivity of the fragments of DNA binding to the probes which are difficult to control.<sup><a reveal-id="DSV026C13" data-open="DSV026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Although targeted approach could be very informative, it is important to note that these are confined only to known and well-annotated sequence information available in public databases.<sup><a reveal-id="DSV026C14" data-open="DSV026C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>We suggest using a technique, called ‘sequence capture’, to sequence a large number of genes that could be used for analysing the functions of environmental communities at a significantly higher resolution than what has been possible with other approaches. It involves a selection of specific genomic loci through adhesion to probes and sequencing only the DNA fragments that are bound to the probes.<sup><a reveal-id="DSV026C15" data-open="DSV026C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The main advantage of the method is its ability to efficiently enrich for genes that are in very low abundance in the gene pool.<sup><a reveal-id="DSV026C16" data-open="DSV026C16" class="link link-ref link-reveal xref-bibr">16</a></sup> It has been predominantly used in the field of medicine to study disease-related point mutations in the human genome.<sup><a reveal-id="DSV026C17  DSV026C20" data-open="DSV026C17  DSV026C20" class="link link-ref link-reveal xref-bibr">17–20</a></sup> Sequence capture has also been implemented for use in several non-model organisms such as chipmunks (Tamilas),<sup><a reveal-id="DSV026C21" data-open="DSV026C21" class="link link-ref link-reveal xref-bibr">21</a></sup> sugarcane (Saccharum)<sup><a reveal-id="DSV026C22" data-open="DSV026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and bison (Bos)<sup><a reveal-id="DSV026C23" data-open="DSV026C23" class="link link-ref link-reveal xref-bibr">23</a></sup> targeting several thousand genomic regions through their closely related species. Sequence capture was also used in a small-scale targeting of two enzymatic regions in a freshwater metagenome from Lake Pavin in France.<sup><a reveal-id="DSV026C24" data-open="DSV026C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>In this study, we aim to demonstrate the application of sequence capture on a large scale for the first time on microbial communities of agricultural soils. The focus was to identify and enhance the capture of functional genes coding for carbohydrate-active enzymes and secretory proteases that are related to organic matter degradation were identified from the public databases were targeted in this method<sup><a reveal-id="DSV026C25  DSV026C27" data-open="DSV026C25  DSV026C27" class="link link-ref link-reveal xref-bibr">25–27</a></sup> and subsequently targeted using sequence capture. The probe design was customized to target functional enzymes of soil microorganisms in natural soil communities using MetCap<sup><a reveal-id="DSV026C28" data-open="DSV026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and allows for efficient targeting hundreds of thousands of genes. In short, MetCap is a web-based probe-designing pipeline that takes user's sequences of interest and design probes for targeting those sequences optimized for sequence capture. The soil metagenomic DNA was targeted at four different probe hybridization stringencies and sequenced with high-throughput sequencing. Here we determine that: (i) the custom-designed sequence capture is efficient in enriching for functional targets in the metagenomic data sets and the effect of hybridization stringencies was investigated, (ii) the correlation of taxonomic distribution of the targeted metagenomes to their respective traditional ribosomal 16S rDNA libraries is very high and (iii) a comparative metagenomic analyses of the eight captured metagenomes with 22 publicly available soil whole metagenomes from the MG-RAST metagenome database illustrated the efficient enrichment using sequence capture compared with standard shotgun whole metagenome sequencing (WMG).</p>                            <h2 scrollto-destination=87200436 id="87200436" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200437 id="87200437" class="section-title" data-legacy-id=s2a>2.1. Soil samples and DNA extraction</h3>
<p>Two soils from Bjornstorp, located in the southern region (Scania) of Sweden that is part of a land-use management study,<sup><a reveal-id="DSV026C29" data-open="DSV026C29" class="link link-ref link-reveal xref-bibr">29</a></sup> were sampled to test the proposed sequence capture method. The soil samples were from agricultural fields, one from a winter wheat field and the other from grassland nearby the wheat field (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). At each field, several sub-samples were collected from different spots (0–15 cm depth) and mixed together. The soils were transported in cold boxes and sieved (2.5 mm) and then stored in a −20°C freezer. Then refrigerated at 4°C before proceeding to DNA extraction. DNA from both samples was extracted using Nucleospin soil DNA isolation kit (Macherey &amp; Nagel, Duren, Germany). The extractions were carried out according to the manufacturer with 0.5 g of soil as the starting material. The extracted DNA was tested for quality (A260/280) and concentration using NanoDrop 2000 spectrophotometer (NanoDrop Technologies, Wilmington, NC, USA). Multiple extractions were carried out for each sample, and the extraction with highest yield and best quality was chosen for amplicon and sequence capture (SeqCap) library preparation steps.</p>                            <h3 scrollto-destination=87200439 id="87200439" class="section-title" data-legacy-id=s2b>2.2. SeqCap EZ probe generation</h3>
<p>Enzymes from Carbohydrate-Active Enzyme database (CAZy)<sup><a reveal-id="DSV026C25" data-open="DSV026C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and proteases from the MEROPS database<sup><a reveal-id="DSV026C27" data-open="DSV026C27" class="link link-ref link-reveal xref-bibr">27</a></sup> having a predicted secretion signal based on signalP v4<sup><a reveal-id="DSV026C30" data-open="DSV026C30" class="link link-ref link-reveal xref-bibr">30</a></sup> were chosen. In total, 306,525 sequences were selected for probe generation of which 260,731 were from CAZy and 45,794 were from MEROPS. The nucleotide coding sequences of these genes were used to design probes for sequence capture, and a local sequence database was created with these sequences (subsequently called target database, TDB). The probes were generated based on these sequences using the MetCap pipeline, where the sequences were clustered with 90% sequence similarity and on an average three probes were generated from each cluster and more details of these probes were described in Kushwaha et al.<sup><a reveal-id="DSV026C28" data-open="DSV026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In total, 406,277 probes were generated with 351,482 from CAZy and 54,795 from MEROPS. They were generated with melting temperature (50°C) and probe length (50mer) that are suitable to use with protocol based on NimbleGen SeqCap EZ (Roche NimbleGen, Inc.).</p>                            <h3 scrollto-destination=87200441 id="87200441" class="section-title" data-legacy-id=s2c>2.3. SeqCap design</h3>
<p>The design of the SeqCap protocol has been modified and tested for the hybridization stringency on target sequence binding specificity, to account for the combination of variability in metagenomic samples and the number of target regions. To test for this stringency, ‘hybridization time’ has been chosen as a factor, since it is effective, and it is also least dependent on other parameters of the protocol making it easy to handle according to the recommendations of Roche NimbleGen, Inc. The metagenomic SeqCap setup was designed to test both soil samples at four different hybridization times. Along with the recommended 72 h (Roche NimbleGen, Inc.) hybridization time for probes and DNA fragments at 47°C, three (24, 16 and 8 h) less-stringent hybridization times have been tested, where 72 h was expected to be most stringent and 8 h to be the least stringent in binding specificity against the target genes.</p>                            <h3 scrollto-destination=87200443 id="87200443" class="section-title" data-legacy-id=s2d>2.4. SeqCap library preparation</h3>
<p>Four libraries were constructed from each soil DNA sample (4 hybridization times × 2 soil samples), following manufacturer's instructions from GS FLX rapid library preparation method (Roche). A total of 500 ng of each DNA sample was fragmented using a Nebulizer (Roche) along with nebulization buffers at 2.1 bar pressure to get the average fragment length in the range of 700 bp. This length is the recommended fragment length for the combination of sequence captures using SeqCap EZ probes coupled with 454 GS FLX sequencing (NimbleGen, Roche). The cleaned fragments were then subjected to the end repair treatment (Roche) followed by 454-adapter ligation. To make multiplexing possible in both cases of sequence capture and sequencing, each library was prepared with a specific MID (Multiplexing Identifier) on its 454 adapters (Lib-L). Each of the eight libraries contained a unique MID, and the qualities of the libraries were checked with High Sensitivity DNA chip (Bioanalyzer, Agilent) at the end of the library preparation step. Before subjecting the fragments to hybridization with probes, a ligation-mediated (LM) PCR (pre-capture) with the 454 adapters as primers was done with FastStart High Fidelity PCR System (Roche Applied Science) with only 12 cycles and purified with QIAquick PCR purification kit (Qiagen, Limburg, The Netherlands), so as to increase the amount of DNA fragments for hybridization step that passed adapter ligation step.</p>                            <h3 scrollto-destination=87200445 id="87200445" class="section-title" data-legacy-id=s2e>2.5. Sequence capture</h3>
<p>The actual sequence capture process has been carried out with the capture protocol (Roche NimbleGen, Inc.) with adjustments in hybridization time as described in the design. The following steps are according to the standard SeqCap EZ protocol (Roche NimbleGen, Inc.). The hybridization step was multiplexed and carried out for each hybridization time as the libraries contain MIDs. For each hybridization time, 1 µg (500 ng from each soil DNA) of amplified libraries together with hybridization-enhancing oligos was dried in a DNA vacuum concentrator. The corresponding DNA libraries were dried, so that there were four dried samples representing each hybridization time. Hybridization buffers were added to the libraries and heated at 95°C for 10 min to denature the DNA fragments. Then the probes (6.5 µl/reaction) were added to the denatured libraries immediately and incubated at 47°C in a thermal cycler with a heated cover at 57°C for respective hybridization times. After the hybridization, the libraries were washed with buffers specified in the SeqCap EZ protocol (Roche NimbleGen, Inc.) along with streptavidin dynabeads at 47°C and magnetic device to retain just the hybridized fragments with probes and remove unbound fragments. These captured DNA fragments bound to probes that were attached to the dynabeads were used as template in a LM-PCR (post-capture) reaction by which the captured DNA fragments were amplified and also separated from the beads/probes. These samples were checked again with a Bioanalyzer to ensure that the DNA fragments were captured and to check for primer dimers. The captured DNA fragments were purified two times using AmpPure Bead (Beckman Coulter Inc., Brea, USA) purification method for the removal of any primer dimers from the post-capture LM PCR that could hinder in the sequencing step. The quantities of double-stranded DNA in these four captured DNA libraries were measured using Quant-it Pico Green kit (Invitrogen, Carlsbad, USA).</p>                            <h3 scrollto-destination=87200447 id="87200447" class="section-title" data-legacy-id=s2f>2.6. 16s rDNA amplicon library preparation</h3>
<p>As part of the sample analysis, bacterial species composition was studied through amplicon sequencing of ribosomal DNA from these soil samples. Amplicon libraries were prepared by running a PCR with fusion primers optimized for 454 sequencing. Fusion primers for bacterial 16S rDNA V3-V4 region were designed using the forward primer B341F [‘CCTACGGGNGGCWGCAG’] and 454 (Lib-A) adapter-A and the reverse primer B805R [‘GACTACHVGGGTATCTAATCC’] preceded by 454 (Lib-A) adapter-B with a MID.<sup><a reveal-id="DSV026C31" data-open="DSV026C31" class="link link-ref link-reveal xref-bibr">31</a></sup> All amplicon libraries were prepared using reagents based on Phire Hot Start DNA Polymerase (Thermo Fisher Scientific Inc., Waltham, MA, USA). The pre-PCR mix was prepared in the same proportion for each sample totaling to 25 µl (5 µl-5× Buffer; 0.5 µl-10 mM dNTPs; 1 µl-10 µM forward primer; 2 µl-5 µM reverse primer; 0.5 µl-DNA polymerase; 0.5 µl-bovine serum albumin; 2.5 µl-Template DNA; 13 µl-MilliQ water). The PCR conditions for 16S rDNA amplification were as follows: initial denaturation step at 98°C for 30 s; 27 cycles of denaturation at 98°C for 5 s, annealing at 56°C for 5 s, extension at 72°C for 10 s; final extension at 72°C for 60 s. Three 25 µl PCR reactions were run separately for each sample and pooled together. These PCR amplicons were purified with QIAquick PCR purification kit (Qiagen), and the quantity was measured using Quant-it Pico Green kit (Invitrogen).</p>                            <h3 scrollto-destination=87200449 id="87200449" class="section-title" data-legacy-id=s2g>2.7. 454 sequencing</h3>
<p>The SeqCap and amplicon libraries were sequenced separately with Lib-L and Lib-A chemistry, respectively (Roche 454, Shirley, NY, USA). The multiplexing (MID) option used in both SeqCap and amplicon libraries facilitate sequencing them as respective pools to yield more sequencing depth and then to separate the libraries computationally. For SeqCap pool, equal amounts of captured DNA from all four different hybridizations were pooled together. The pooled captured libraries were sequenced in the sequencing facility at Lund University on a whole plate of GS FLX Titanium series with Lib-L chemistry in two regions. For amplicon libraries, both 16S rDNA amplicon libraries were sequenced as part of an amplicon pool made from equal amounts of 24 other amplicon libraries. This amplicon pool was sequenced in a 1/4 plate using a GS FLX Titanium series with Lib-A chemistry also sequenced at the same facility. The sequence data related to the captured metagenomes can be found on the MG-RAST server with their specific Metagenome IDs (WS-72: 4527652.3, WS-24: 4529373.3, WS-16: 4529786.3, WS-8: 4528934.3, GL-72: 4527653.3, GL-24: 4529374.3, GL-16: 4529787.3 and GL-8: 4528937.3). The 16S rDNA amplicon sequencing data can be found in the study accession number PRJEB9530 at EMBL (WS-16S: ERS743453 and GL-16S: ERS743454).</p>                            <h3 scrollto-destination=87200451 id="87200451" class="section-title" data-legacy-id=s2h>2.8. Data analysis</h3>
<p>The sequencing output from both regions in the captured libraries was separated based on their MID tags into their respective eight different data sets. The sequence reads from each of these captured data sets were processed through MG-RAST,<sup><a reveal-id="DSV026C32" data-open="DSV026C32" class="link link-ref link-reveal xref-bibr">32</a></sup> an online tool that is mainly used for functional annotations of metagenomic sequences. The default parameters for quality filtering in MG-RAST were used to remove reads with bad quality and to remove artificial sequence duplicates.<sup><a reveal-id="DSV026C33" data-open="DSV026C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The filtered sequence reads were used in further analysis. The optimal hybridization time with respect to this particular set of probe design method was deduced based on the enrichment efficiency (fraction of on-targets) for each hybridization time. On-targets were identified through sequence similarity (blastx, <em>E</em>-value of 10<sup>−5</sup>) with the target database (TDB) consisting of carbohydrate-active enzymes and secreted proteases.<sup><a reveal-id="DSV026C25  DSV026C27" data-open="DSV026C25  DSV026C27" class="link link-ref link-reveal xref-bibr">25–27</a></sup></p><p>The functional assignment of sequences from captured metagenomes was also carried out in MG-RAST based on sequence similarity matches with different databases along with SEED<sup><a reveal-id="DSV026C34" data-open="DSV026C34" class="link link-ref link-reveal xref-bibr">34</a></sup> database where reads were annotated with different subsystems (<em>E</em>-value of 10<sup>−5</sup>). The CAZy domains were predicted from the captured metagenomes using CAZy Analysis Toolkit (CAT) with <em>E</em>-value cut-off of 10<sup>−5</sup>.<sup><a reveal-id="DSV026C35" data-open="DSV026C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The taxonomic analysis of captured metagenomes has been analysed mainly based on counts for each taxa obtained from MG-RAST.<sup><a reveal-id="DSV026C32" data-open="DSV026C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The amplicon metagenomes were processed with QIIME<sup><a reveal-id="DSV026C36" data-open="DSV026C36" class="link link-ref link-reveal xref-bibr">36</a></sup> in combination with Greengenes<sup><a reveal-id="DSV026C37" data-open="DSV026C37" class="link link-ref link-reveal xref-bibr">37</a></sup> for 16S rDNA amplicons as a resource for taxonomy assignments. The taxonomic distributions from the amplicon metagenomes were compared with that of the captured metagenomes. A comparative metagenomic analyses between the eight captured metagenomes along with 22 publicly available whole metagenomes (WMG) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>) also from soil sequenced with 454 were achieved through the R package ‘matR’ in MG-RAST.<sup><a reveal-id="DSV026C38" data-open="DSV026C38" class="link link-ref link-reveal xref-bibr">38</a></sup> These metagenomes were also normalized with log transformation and centred for each sample using ‘matR’. STAMP<sup><a reveal-id="DSV026C39" data-open="DSV026C39" class="link link-ref link-reveal xref-bibr">39</a></sup> was used for the visualization and the statistics of the comparative metagenomic data sets with Bonferroni correction for <em>P</em>-values. The CAZy domains were predicted for two of the intensively sequenced, publicly available whole metagenomes and compared with the captured metagenomes from our study.</p>                            <h2 scrollto-destination=87200454 id="87200454" class="section-title" data-legacy-id=s3>3. Results</h2>
<p>In total, 914,996 sequence reads were obtained from the SeqCap pool. The number of sequences with good quality (after QC and dereplication) for all eight samples was uniform and ranged between 89,281 and 129,198 per sample (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The targeted metagenomes that were sequenced with 454 pyrosequencing generated &gt;850,000 reads with an average read length of 348 after quality control (QC). The fraction of reads removed through QC was consistent (∼7%) among all eight captured metagenomes. In total from the amplicon pool, 4,990 sequence reads were obtained from 16S rDNA amplicons after QC from both the wheat soil (WS) and grassland soil (GL), respectively.</p>                            <h3 scrollto-destination=87200456 id="87200456" class="section-title" data-legacy-id=s3a>3.1. Capture efficiency</h3>
<p>The efficiency of different hybridization times to enrich for the sequences of interest was measured as the proportion of reads with significant similarity to proteins sequences in the TDB (Fig. <a reveal-id="DSV026F1" data-open="DSV026F1" class="link link-reveal link-table xref-fig">1</a>A). The fraction of reads that matched (blastx) to TDB varied from 27 ± 1% for the shortest hybridization time (8 h) to 40 ± 2% for the longest hybridization time (72 h). The total number of reads that belonged to the carbohydrate subsystem from SEED<sup><a reveal-id="DSV026C34" data-open="DSV026C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and the fraction of reads that have a predicted CAZy domain from CAT<sup><a reveal-id="DSV026C35" data-open="DSV026C35" class="link link-ref link-reveal xref-bibr">35</a></sup> also increased with increase in hybridization times for both samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The capture efficiencies of the different hybridization times obtained through three independent methods (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) showed that 72 h hybridization time was most efficient in enriching targets from metagenomes. </p>                        <a id="87200458" scrollto-destination="87200458"></a>
<div id="DSV026F1" data-id="DSV026F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02601.png?Expires=2147483647&Signature=e5OYjt3oTGXhWG2CeCRweNrrWD1qU7veLPLKMn2s11ibyM5AjUNnbakt4CLxiG4eKz5uEVBTTCsWnXa9AVOJF-wPu5Y3IGHg3x1ODUoaM0R7NUAy4YOBEF9op84YKvUmF1kIeRhPGnyDJCHUHemvMt5cPv6m6QCshRFs90AnNlZ~~mcbmA5onbJwaJz2rBUb7jUCQcEZiCtZPxC7tcopym0lFS-KnEfMviegetwDaLdFgO7SXcN1dJ5ujBxlbJq4XIe2rC5d1IqizQ-2huiBceIF0nbG~hME9NR7wLCkT60mPlfCm2KkNGJ5hhMROkEP~f6CfTxtcfs9R3j4aCxTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The capture efficiency in relation to different hybridization times from the wheat (WS) and grassland (GL) soils calculated as fraction (%) of total filtered reads that had a significant sequence match (blastx) to the protein sequences from TDB. (B) Rarefaction curves of unique enzyme IDs (UniProt) from random sampling of filtered sequence reads from the captured metagenomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200458/dsv02601.tiff" path-from-xml="dsv02601.tiff" target="_blank">View large</a><a section="87200458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02601.png?Expires=2147483647&Signature=kip15Roi1nRvblu7mFLcPdEpD~ExOB~OjKF5n~Z8GBK8OpIT3vHo7aqdWH5IQDOisimUyKSKnBJh5cG1PJKqu6V969Sc-uW7jhsrgy8zsIDVvSAIe3oZP3c3pa52H6x~Zh7MzhglIVJaMnaSE-0yxKU8-S1GQM7lG2NCBfZf3nT8Vuy1oflC4bX74BhiS3x1A5~AO17NFdP9NPSQjGqqAQEatg717qisCtHJF2Qxik7LBzhMAqv-NXU8LuClX70qkgkdlM7snQxGOmHTgJU7MLvhnl-rQ9r2yDcZFE2Tt0UEbIC7JP43oNPEoS-NnLTp29eGAwfAu2hRdQ0B3SQutA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200458&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The capture efficiency in relation to different hybridization times from the wheat (WS) and grassland (GL) soils calculated as fraction (%) of total filtered reads that had a significant sequence match (blastx) to the protein sequences from TDB. (B) Rarefaction curves of unique enzyme IDs (UniProt) from random sampling of filtered sequence reads from the captured metagenomes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02601.png?Expires=2147483647&Signature=e5OYjt3oTGXhWG2CeCRweNrrWD1qU7veLPLKMn2s11ibyM5AjUNnbakt4CLxiG4eKz5uEVBTTCsWnXa9AVOJF-wPu5Y3IGHg3x1ODUoaM0R7NUAy4YOBEF9op84YKvUmF1kIeRhPGnyDJCHUHemvMt5cPv6m6QCshRFs90AnNlZ~~mcbmA5onbJwaJz2rBUb7jUCQcEZiCtZPxC7tcopym0lFS-KnEfMviegetwDaLdFgO7SXcN1dJ5ujBxlbJq4XIe2rC5d1IqizQ-2huiBceIF0nbG~hME9NR7wLCkT60mPlfCm2KkNGJ5hhMROkEP~f6CfTxtcfs9R3j4aCxTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The capture efficiency in relation to different hybridization times from the wheat (WS) and grassland (GL) soils calculated as fraction (%) of total filtered reads that had a significant sequence match (blastx) to the protein sequences from TDB. (B) Rarefaction curves of unique enzyme IDs (UniProt) from random sampling of filtered sequence reads from the captured metagenomes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02601.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200458/dsv02601.tiff" path-from-xml="dsv02601.tiff" target="_blank">View large</a><a section="87200458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02601.png?Expires=2147483647&Signature=kip15Roi1nRvblu7mFLcPdEpD~ExOB~OjKF5n~Z8GBK8OpIT3vHo7aqdWH5IQDOisimUyKSKnBJh5cG1PJKqu6V969Sc-uW7jhsrgy8zsIDVvSAIe3oZP3c3pa52H6x~Zh7MzhglIVJaMnaSE-0yxKU8-S1GQM7lG2NCBfZf3nT8Vuy1oflC4bX74BhiS3x1A5~AO17NFdP9NPSQjGqqAQEatg717qisCtHJF2Qxik7LBzhMAqv-NXU8LuClX70qkgkdlM7snQxGOmHTgJU7MLvhnl-rQ9r2yDcZFE2Tt0UEbIC7JP43oNPEoS-NnLTp29eGAwfAu2hRdQ0B3SQutA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200458&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02601.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The capture efficiency in relation to different hybridization times from the wheat (WS) and grassland (GL) soils calculated as fraction (%) of total filtered reads that had a significant sequence match (blastx) to the protein sequences from TDB. (B) Rarefaction curves of unique enzyme IDs (UniProt) from random sampling of filtered sequence reads from the captured metagenomes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The abundance of each enzyme (Uniprot enzyme ID) based on the number of reads that matched the enzymes in TDB sequences through blastx was obtained for each metagenome. The rarefaction curve (Fig. <a reveal-id="DSV026F1" data-open="DSV026F1" class="link link-reveal link-table xref-fig">1</a>B) for unique enzymes in each metagenome also depicts the variation between hybridization times for both soil samples. The metagenomes in terms of enzyme abundance counts were significantly explained by hybridization time when they were homogenized for soil types (ANOSIM: <em>P</em> = 0.04). The 72 h hybridization time had more unique enzymes and more on-target matches than the other times in any given amount of random sampling of sequences.</p>                            <h3 scrollto-destination=87200460 id="87200460" class="section-title" data-legacy-id=s3b>3.2. Taxonomic distribution of the metagenomes</h3>
<p>The captured metagenomes were dominated by sequences of the bacterial phyla Actinobacteria and Proteobacteria in both soil samples. Comparative taxonomic analysis of metagenomes was based on the relative abundances of different genera (both bacteria and fungi). The taxonomic groups of the captured metagenomes from the same soil type were significantly similar (PERMANOVA: <em>P</em> &lt; 0.001, <em>R</em><sup>2</sup> = 0.74; ANOSIM: <em>P</em> = 0.03), while the hybridization time did not have any effect on their taxonomic distribution (PERMANOVA: <em>P</em> = 0.45; ANOSIM: <em>P</em> = 0.22) (Fig. <a reveal-id="DSV026F2" data-open="DSV026F2" class="link link-reveal link-table xref-fig">2</a>A). The high-stringency (72 h) metagenomes were used for the taxonomic comparison to the taxonomic distribution obtained from the 16S rDNA amplicon sequencing of respective soil samples (WS-16 and GL-16). The relative abundances of bacterial taxa in the captured metagenomes (72 h) from MG-RAST were well correlated with their respective 16S rDNA amplicons (Fig. <a reveal-id="DSV026F2" data-open="DSV026F2" class="link link-reveal link-table xref-fig">2</a>B). The taxonomic representations of the captured metagenomes were similar to that of the representations obtained from the ribosomal amplicon metagenomes. The taxonomy distributions between amplicon libraries and the captured metagenomes were significantly correlated at different levels of taxa for both soil types (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). </p>                        <a id="87200462" scrollto-destination="87200462"></a>
<div id="DSV026F2" data-id="DSV026F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02602.png?Expires=2147483647&Signature=lpgVYtMdKLWu9KUBt5aHcf6qTxtC8VyLiwZOK9nP2VKDZ2P01xN-mLqp5yx~Z9HtwDmAe5FUhfpL2OL8J98ukUPbqnGGWM7fYWMvOI~NLteRuY63g9L2K7t3pzUYtBE2tD-YLuzRKNymr5vGuceA4PVYQJzboQV0qrq6SZysUlhTdTT2A8UF9PQbReysK-Ct2QffgCth2rj0qfATZQNoyefDHSOA93xQ8Jq4aIYBD-BU8ch2za4t5Dg~xgxZ3eg-OyslY~dCWsTtLyNaWplaX2BKCGgUs5H8s3HYHQ~sOIl2FYZjDwfkzR-zOGYg9CcS-TLb1yX6aKZKOs3e24U0pQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) In an NMDS plot based on Bray–Curtis distances, the taxonomic distributions from the eight captured metagenomes show that the metagenomes from the same soil wheat (WS) and grassland (GL) cluster together (NMDS: stress = 0.04; PERMANOVA: R2 = 0.74, P &lt; 0.001). (B) The relative abundances of different taxonomic groups (phyla) from the captured metagenomes (WS-72 and GL-72) and 16S rDNA amplicon metagenomes (WS-16S and GL-16S). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200462/dsv02602.tiff" path-from-xml="dsv02602.tiff" target="_blank">View large</a><a section="87200462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02602.png?Expires=2147483647&Signature=EFoKw-rtnH0~oNJ8uAogR7O9N4zJ3AAGTdU8-uL6N8JfUUE6iBWgH2jIIdTIr9qId7fL86FLpQedAlt2u4LzIMCAKj6PzWIu95~YqUCf7aR8GxciS0haJkyIZXUGLcRi~L0kG81CKGhyjCJ~8SuGah1WmWQcQqCz7ulkcN9AVtHfAR-TK1K2WAT4Qzk3D~oWnBHrjHIx27FtTQIiQH-uKZrM6tmkbQXKFlZT9CxG-lEqD4S1Z7Q9fu16RYRD2ygO3b8T8CXvvTn6Nbblnmkk97DmdFZkYxWFJ3qc4FtLiQrp1E0vePbiQ~qziBwRgHknIpHjZ0VVwVFKIvb3qrCU-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200462&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) In an NMDS plot based on Bray–Curtis distances, the taxonomic distributions from the eight captured metagenomes show that the metagenomes from the same soil wheat (WS) and grassland (GL) cluster together (NMDS: stress = 0.04; PERMANOVA: <em>R</em><sup>2</sup> = 0.74, <em>P</em> &lt; 0.001). (B) The relative abundances of different taxonomic groups (phyla) from the captured metagenomes (WS-72 and GL-72) and 16S rDNA amplicon metagenomes (WS-16S and GL-16S). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02602.png?Expires=2147483647&Signature=lpgVYtMdKLWu9KUBt5aHcf6qTxtC8VyLiwZOK9nP2VKDZ2P01xN-mLqp5yx~Z9HtwDmAe5FUhfpL2OL8J98ukUPbqnGGWM7fYWMvOI~NLteRuY63g9L2K7t3pzUYtBE2tD-YLuzRKNymr5vGuceA4PVYQJzboQV0qrq6SZysUlhTdTT2A8UF9PQbReysK-Ct2QffgCth2rj0qfATZQNoyefDHSOA93xQ8Jq4aIYBD-BU8ch2za4t5Dg~xgxZ3eg-OyslY~dCWsTtLyNaWplaX2BKCGgUs5H8s3HYHQ~sOIl2FYZjDwfkzR-zOGYg9CcS-TLb1yX6aKZKOs3e24U0pQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) In an NMDS plot based on Bray–Curtis distances, the taxonomic distributions from the eight captured metagenomes show that the metagenomes from the same soil wheat (WS) and grassland (GL) cluster together (NMDS: stress = 0.04; PERMANOVA: R2 = 0.74, P &lt; 0.001). (B) The relative abundances of different taxonomic groups (phyla) from the captured metagenomes (WS-72 and GL-72) and 16S rDNA amplicon metagenomes (WS-16S and GL-16S). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02602.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200462/dsv02602.tiff" path-from-xml="dsv02602.tiff" target="_blank">View large</a><a section="87200462" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02602.png?Expires=2147483647&Signature=EFoKw-rtnH0~oNJ8uAogR7O9N4zJ3AAGTdU8-uL6N8JfUUE6iBWgH2jIIdTIr9qId7fL86FLpQedAlt2u4LzIMCAKj6PzWIu95~YqUCf7aR8GxciS0haJkyIZXUGLcRi~L0kG81CKGhyjCJ~8SuGah1WmWQcQqCz7ulkcN9AVtHfAR-TK1K2WAT4Qzk3D~oWnBHrjHIx27FtTQIiQH-uKZrM6tmkbQXKFlZT9CxG-lEqD4S1Z7Q9fu16RYRD2ygO3b8T8CXvvTn6Nbblnmkk97DmdFZkYxWFJ3qc4FtLiQrp1E0vePbiQ~qziBwRgHknIpHjZ0VVwVFKIvb3qrCU-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200462&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02602.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) In an NMDS plot based on Bray–Curtis distances, the taxonomic distributions from the eight captured metagenomes show that the metagenomes from the same soil wheat (WS) and grassland (GL) cluster together (NMDS: stress = 0.04; PERMANOVA: <em>R</em><sup>2</sup> = 0.74, <em>P</em> &lt; 0.001). (B) The relative abundances of different taxonomic groups (phyla) from the captured metagenomes (WS-72 and GL-72) and 16S rDNA amplicon metagenomes (WS-16S and GL-16S). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200463 id="87200463" class="section-title" data-legacy-id=s3c>3.3. Functional distribution of the metagenomes</h3>
<p>The functional annotations based on the different databases available in MG-RAST showed that the fraction of reads annotated with functional proteins increased with longer hybridization time for both soil samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). However, the fraction of reads that had neither protein similarity nor functional annotation was constant in all captured metagenomes (12.5 ± 1%). The subgroup of reads with KEGG orthology annotations (KO) (identity ≥60% and length ≥15 aa) showed that &gt;85% of these proteins were involved in metabolism (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Also, &gt;70% of those metabolic reads were particularly involved in carbohydrate metabolism in both wheat and grassland soil with high-stringent hybridization time (WS-72: 18,628 reads; GL-72: 15,900 reads) (Fig. <a reveal-id="DSV026F3" data-open="DSV026F3" class="link link-reveal link-table xref-fig">3</a>A) and are part of &gt;200 different KO functional groups. This is followed by synthesis of secondary metabolites, amino acid metabolism and glycan biosynthesis and metabolism. These four metabolic categories constitute &gt;90% of all the metabolic annotations. </p>                        <a id="87200465" scrollto-destination="87200465"></a>
<div id="DSV026F3" data-id="DSV026F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02603.png?Expires=2147483647&Signature=hGf5o6SkjCg~KsU6AxbFF~-iGa~uFBMWQmKE3SNahwiK79r2p-F3I98aI7Rf52~moQ0DAlv03Za7PpUdH7wwPxa2vMGPjwvxgc6xRhZXCQ4sKK02VCallIkAZuzRjysG-DC5CsF7FuYIPv7dOjJBZM1Y3QK2fakd~Re4b82hJCTCV0f94315ZbpC-p18RTYQqA8UtY-PeuaVniWSO4sO~5Q-dUFgUktodgSxxJjc7FhRs7CmO8WceG8osHfcQGIuIHEHGdz4Lh8jEt3fh93dswXL-ekBS5rlZ1P8QM087fsuSjTWbda7ghciwpNg~WFyQdlmu0bTAw4SBLX6kkaOww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The distribution of different metabolic functions (%) obtained and based on KEGG orthology (KO) annotations. (B) The number of different CAZy enzyme families captured under the different enzyme classes based on three different annotation sources (CAT, MG-RAST and blastx against TDB) from each or both the captured metagenomes WS-72 and GL-72. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200465/dsv02603.tiff" path-from-xml="dsv02603.tiff" target="_blank">View large</a><a section="87200465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02603.png?Expires=2147483647&Signature=zLn5Boqyu-vf~7M9aOQTa2kkEgGvmAchs7-yHZlVWaP~p57JOi0bFfjd8IYZ3-gTd4Lv~UDXgOlN8ogHTe005bLsOBIh7ksxW3~6OASqeVN6XWg81670TETuSD8ADBC9FUnYE5axbi0YjZ1rnYVEAVuGC9EYpe5jTjWZ8DoswlbMfPG71ktQkR3ThIiTahNKe5jWf-Snj9ivfNxoyOS45NSp2TrxKBd8uuc-xsY-pyMs-HAZ4mfBY8aKDGOinNrKPDXQgJx1zc-NsZp5zCO7SHs0XHLcohlZILeQOlDAQqGUopIHXj3dYpUhPZ1n8UglUrnz2CbI4onlJGQgUqYcHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200465&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The distribution of different metabolic functions (%) obtained and based on KEGG orthology (KO) annotations. (B) The number of different CAZy enzyme families captured under the different enzyme classes based on three different annotation sources (CAT, MG-RAST and blastx against TDB) from each or both the captured metagenomes WS-72 and GL-72. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02603.png?Expires=2147483647&Signature=hGf5o6SkjCg~KsU6AxbFF~-iGa~uFBMWQmKE3SNahwiK79r2p-F3I98aI7Rf52~moQ0DAlv03Za7PpUdH7wwPxa2vMGPjwvxgc6xRhZXCQ4sKK02VCallIkAZuzRjysG-DC5CsF7FuYIPv7dOjJBZM1Y3QK2fakd~Re4b82hJCTCV0f94315ZbpC-p18RTYQqA8UtY-PeuaVniWSO4sO~5Q-dUFgUktodgSxxJjc7FhRs7CmO8WceG8osHfcQGIuIHEHGdz4Lh8jEt3fh93dswXL-ekBS5rlZ1P8QM087fsuSjTWbda7ghciwpNg~WFyQdlmu0bTAw4SBLX6kkaOww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The distribution of different metabolic functions (%) obtained and based on KEGG orthology (KO) annotations. (B) The number of different CAZy enzyme families captured under the different enzyme classes based on three different annotation sources (CAT, MG-RAST and blastx against TDB) from each or both the captured metagenomes WS-72 and GL-72. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02603.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200465/dsv02603.tiff" path-from-xml="dsv02603.tiff" target="_blank">View large</a><a section="87200465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02603.png?Expires=2147483647&Signature=zLn5Boqyu-vf~7M9aOQTa2kkEgGvmAchs7-yHZlVWaP~p57JOi0bFfjd8IYZ3-gTd4Lv~UDXgOlN8ogHTe005bLsOBIh7ksxW3~6OASqeVN6XWg81670TETuSD8ADBC9FUnYE5axbi0YjZ1rnYVEAVuGC9EYpe5jTjWZ8DoswlbMfPG71ktQkR3ThIiTahNKe5jWf-Snj9ivfNxoyOS45NSp2TrxKBd8uuc-xsY-pyMs-HAZ4mfBY8aKDGOinNrKPDXQgJx1zc-NsZp5zCO7SHs0XHLcohlZILeQOlDAQqGUopIHXj3dYpUhPZ1n8UglUrnz2CbI4onlJGQgUqYcHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200465&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02603.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The distribution of different metabolic functions (%) obtained and based on KEGG orthology (KO) annotations. (B) The number of different CAZy enzyme families captured under the different enzyme classes based on three different annotation sources (CAT, MG-RAST and blastx against TDB) from each or both the captured metagenomes WS-72 and GL-72. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Among the different annotation sources in the MG-RAST server, the SEED-based subsystem had the most abundant annotations with 943 different subsystems (Level 3) annotated at least two times among the eight captured metagenomes. Comparing the abundances of broader subsystems (Level 1), the carbohydrates subsystem was the most abundant (WS-72: 35% and GL-72: 36%) followed by the clustering-based subsystem (functionally coupled genes) (WS-72: 33% and GL-72: 35%). The three most abundant finer (Level 3) subsystems that were found in both wheat (WS-72) and grassland soil (GL-72) were trehalose biosynthesis, glycogen metabolism and glycogen metabolism cluster which form ∼50% of the entire subsystems. The blastx matches obtained against the TDB sequences showed that ∼75% of the matches were annotated to CAZy enzymes and the rest were proteases for both soils (WS-72, GL-72). Among the different enzyme families, the most abundant bacterial family was the glycosyl hydrolase 13 (GH13) in the CAZy database (WS-72: 41%, GL-72: 45%). This was followed by metalloproteases (WS-72: 13%, GL-72: 11%) and serine proteases (WS-72: 13%, GL-72: 10%) in both soils.</p>                            <h3 scrollto-destination=87200467 id="87200467" class="section-title" data-legacy-id=s3d>3.4. CAZy analysis</h3>
<p>Out of 331 different CAZy enzyme families, 243 were found in both soil samples (WS-72 and GL-72) at least through one of the three methods of annotation (CAT, MG-RAST and blastx) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The number of different enzyme families captured in each CAZy enzyme class varied between the soil samples and the enzyme class (Fig. <a reveal-id="DSV026F3" data-open="DSV026F3" class="link link-reveal link-table xref-fig">3</a>B). Larger number of enzyme families from glycosyl transferase (GT) class was not found in both metagenomes (WS-72 and GL-72) compared with other enzyme classes. Glycosyl hydrolases were the most abundant enzyme class from all three methods in both metagenomes (WS-72 and GL-72). The most abundant glycosyl hydrolase enzyme family GH13 was predicted along with carbon binding motif CBM48 from the domain predictor (CAT). The fractions of reads with GH13 domain along with CBM48 (WS-72: 39%, GL-72: 46%) were the most abundant next to GH13 domains (WS-72: 17%, GL-72: 17%) that were predicted independently.</p>                            <h3 scrollto-destination=87200469 id="87200469" class="section-title" data-legacy-id=s3e>3.5. Comparative metagenomics</h3>
<p>As all the captured metagenomes in this study represented two soils, the metagenomes from the different hybridization times were treated as replicates and the different enzyme IDs among the portion of on-targets (removing the off-targets) that matched (blastx) to TDB was very similar within each soil type (Fig. <a reveal-id="DSV026F4" data-open="DSV026F4" class="link link-reveal link-table xref-fig">4</a>A). Similarly, the abundances of different SEED-based subsystems (Level 3) from MG-RAST after normalization were analysed for the eight captured metagenomes (Fig. <a reveal-id="DSV026F4" data-open="DSV026F4" class="link link-reveal link-table xref-fig">4</a>B). The subsystem abundances in these eight captured metagenomes were significantly affected by both soil type (PERMANOVA: <em>P</em> &lt; 0.001) and hybridization times (PERMANOVA: <em>P</em> &lt; 0.01). Enzymes such as amino-peptidisases, GH10 and GH36 for example were highly enriched in grassland (GL) while enzymes related to peptidoglycan biosynthesis and GH46 were highly enriched in the wheat field (WS) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>). </p>                        <a id="87200471" scrollto-destination="87200471"></a>
<div id="DSV026F4" data-id="DSV026F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02604.png?Expires=2147483647&Signature=Ti0~kp7GpKTSaYtc9LQv7kjbv2YTrI7wXUEB7akvKua3zKZscO1CSwWuKAUY7v~y~CCxD38LQASULwnWTzsRMefhxBTApItvzSHS5wdpSg6540gTxlJ5pZqGcNkeLbMlydpWZTrFl6ncrMCBQgcjPazXu-RaWTuXMnLdEwgwpTw5wcCOQsgjaWSb~~yZeUKg3TEyp3SfQ3ymn9eMJqA7raowjYGj6Z-~~iaeY-LQWS3-cenceLIeHNegTsrMRaXofts6AVj5J47~bIBD~4I1OfpySfKYxHT4BhxY5fQOOfz0zz4FpTZbpgukgYkp422gNYqBrzGpf0~~LP4zmjPOoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Rarefaction of unique enzyme IDs (UniProt) obtained from random sampling (sequences) of on-targets from the blastx matches against TDB shows that the enzyme diversity of samples converges based on the soil type. (B) A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes showing that the metagenomes from wheat (WS) and grassland (GL) cluster separately (PERMANOVA: P &lt; 0.001). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02604.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200471/dsv02604.tiff" path-from-xml="dsv02604.tiff" target="_blank">View large</a><a section="87200471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02604.png?Expires=2147483647&Signature=pY1tRXJ1GtAny2ofVJFE6Lv4T6bbYEsIf64nSxmjb2YuaPiqMQun8RRC0AyS2suFluoSWRcGjFzKMmc6cV9mUhIGDhvd05u0Ojxc2F6X-vHQdHfa0kUuJT~0AlbEbSthIt~ljRKjEfW906fNRQyxw5z~MLckCg5~mSkywcpLR7bds2TN4M3lOJ~JR2qdsYv7BH-jSZB1QUcLf-Li1WxMm2~-bdTnizUBe00rg1Gz5NwwrSGm3yU6ZpqcrwGiCQGJC6RwzbEQnZXD9hcb~8rGqxTL~SYr-q15xS63jJ7ny2BdSlKIOWUD-P7~ROFx2yzwHF~l357eqzFSQZ9sAcguyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200471&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02604.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Rarefaction of unique enzyme IDs (UniProt) obtained from random sampling (sequences) of on-targets from the blastx matches against TDB shows that the enzyme diversity of samples converges based on the soil type. (B) A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes showing that the metagenomes from wheat (WS) and grassland (GL) cluster separately (PERMANOVA: <em>P</em> &lt; 0.001). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02604.png?Expires=2147483647&Signature=Ti0~kp7GpKTSaYtc9LQv7kjbv2YTrI7wXUEB7akvKua3zKZscO1CSwWuKAUY7v~y~CCxD38LQASULwnWTzsRMefhxBTApItvzSHS5wdpSg6540gTxlJ5pZqGcNkeLbMlydpWZTrFl6ncrMCBQgcjPazXu-RaWTuXMnLdEwgwpTw5wcCOQsgjaWSb~~yZeUKg3TEyp3SfQ3ymn9eMJqA7raowjYGj6Z-~~iaeY-LQWS3-cenceLIeHNegTsrMRaXofts6AVj5J47~bIBD~4I1OfpySfKYxHT4BhxY5fQOOfz0zz4FpTZbpgukgYkp422gNYqBrzGpf0~~LP4zmjPOoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Rarefaction of unique enzyme IDs (UniProt) obtained from random sampling (sequences) of on-targets from the blastx matches against TDB shows that the enzyme diversity of samples converges based on the soil type. (B) A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes showing that the metagenomes from wheat (WS) and grassland (GL) cluster separately (PERMANOVA: P &lt; 0.001). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02604.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200471/dsv02604.tiff" path-from-xml="dsv02604.tiff" target="_blank">View large</a><a section="87200471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02604.png?Expires=2147483647&Signature=pY1tRXJ1GtAny2ofVJFE6Lv4T6bbYEsIf64nSxmjb2YuaPiqMQun8RRC0AyS2suFluoSWRcGjFzKMmc6cV9mUhIGDhvd05u0Ojxc2F6X-vHQdHfa0kUuJT~0AlbEbSthIt~ljRKjEfW906fNRQyxw5z~MLckCg5~mSkywcpLR7bds2TN4M3lOJ~JR2qdsYv7BH-jSZB1QUcLf-Li1WxMm2~-bdTnizUBe00rg1Gz5NwwrSGm3yU6ZpqcrwGiCQGJC6RwzbEQnZXD9hcb~8rGqxTL~SYr-q15xS63jJ7ny2BdSlKIOWUD-P7~ROFx2yzwHF~l357eqzFSQZ9sAcguyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200471&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02604.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Rarefaction of unique enzyme IDs (UniProt) obtained from random sampling (sequences) of on-targets from the blastx matches against TDB shows that the enzyme diversity of samples converges based on the soil type. (B) A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes showing that the metagenomes from wheat (WS) and grassland (GL) cluster separately (PERMANOVA: <em>P</em> &lt; 0.001). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To elucidate the ability of captured metagenomics, a number of whole metagenomic data sets from soil (WMG) that were deeply sequenced by pyrosequencing and were publicly available in MG-RAST were chosen for comparison. Some of these public metagenomes were sequenced to a depth &gt;10 times that of the captured metagenomes in our study. Two of WMG have been analysed for CAT, since it only allows 50,000 sequences to be analysed per run. The fraction of reads that contained CAZy domains in the captured metagenomes was more than five times higher than the possible untargeted metagenomes despite the coverage of sequencing (Table <a reveal-id="DSV026TB1" data-open="DSV026TB1" class="link link-reveal link-table xref-fig">1</a>). The most abundant enzyme family GT2 in the two public metagenomes formed only ∼0.8 and 0.2% of the total reads, whereas in our targeted approach the two most common enzyme families were GH13 together with CBM48 and represented 12% of the WS-72 and 13% of the GL-72 metagenomes. </p>                        <a id="87200473" scrollto-destination="87200473"></a>
<div content-id="DSV026TB1" class="table-modal"><div class="table-wrap"><div id="DSV026TB1" class="table-wrap-title" data-id="DSV026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The fraction of reads predicted with CAZy domains from the captured metagenomes is at least five times more than intensively sequenced public WMG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Metagenomes </th><th>Filtered reads </th><th>CAZy domains (reads) </th><th>%CAZy </th><th>Expected CAZy domains (per million reads) </th></tr></thead><tbody><tr><td>WS-72 </td><td>105,190 </td><td>30,999 </td><td>29.50 </td><td>294,695 </td></tr><tr><td>GL-72 </td><td>89,281 </td><td>24,637 </td><td>27.60 </td><td>275,949 </td></tr><tr><td>Public-A </td><td>937,368 </td><td>52,817 </td><td>5.60 </td><td>56,346 </td></tr><tr><td>Public-2M </td><td>354,345 </td><td>4,692 </td><td>1.30 </td><td>13,241 </td></tr></tbody></table></div><div content-id="DSV026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Metagenomes </th><th>Filtered reads </th><th>CAZy domains (reads) </th><th>%CAZy </th><th>Expected CAZy domains (per million reads) </th></tr></thead><tbody><tr><td>WS-72 </td><td>105,190 </td><td>30,999 </td><td>29.50 </td><td>294,695 </td></tr><tr><td>GL-72 </td><td>89,281 </td><td>24,637 </td><td>27.60 </td><td>275,949 </td></tr><tr><td>Public-A </td><td>937,368 </td><td>52,817 </td><td>5.60 </td><td>56,346 </td></tr><tr><td>Public-2M </td><td>354,345 </td><td>4,692 </td><td>1.30 </td><td>13,241 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>CAZy, Carbohydrate-Active Enzymes; WMG, whole metagenome sequencing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200473" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV026TB1" class="table-wrap-title" data-id="DSV026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The fraction of reads predicted with CAZy domains from the captured metagenomes is at least five times more than intensively sequenced public WMG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Metagenomes </th><th>Filtered reads </th><th>CAZy domains (reads) </th><th>%CAZy </th><th>Expected CAZy domains (per million reads) </th></tr></thead><tbody><tr><td>WS-72 </td><td>105,190 </td><td>30,999 </td><td>29.50 </td><td>294,695 </td></tr><tr><td>GL-72 </td><td>89,281 </td><td>24,637 </td><td>27.60 </td><td>275,949 </td></tr><tr><td>Public-A </td><td>937,368 </td><td>52,817 </td><td>5.60 </td><td>56,346 </td></tr><tr><td>Public-2M </td><td>354,345 </td><td>4,692 </td><td>1.30 </td><td>13,241 </td></tr></tbody></table></div><div content-id="DSV026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Metagenomes </th><th>Filtered reads </th><th>CAZy domains (reads) </th><th>%CAZy </th><th>Expected CAZy domains (per million reads) </th></tr></thead><tbody><tr><td>WS-72 </td><td>105,190 </td><td>30,999 </td><td>29.50 </td><td>294,695 </td></tr><tr><td>GL-72 </td><td>89,281 </td><td>24,637 </td><td>27.60 </td><td>275,949 </td></tr><tr><td>Public-A </td><td>937,368 </td><td>52,817 </td><td>5.60 </td><td>56,346 </td></tr><tr><td>Public-2M </td><td>354,345 </td><td>4,692 </td><td>1.30 </td><td>13,241 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>CAZy, Carbohydrate-Active Enzymes; WMG, whole metagenome sequencing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200473" target="_blank">View Large</a></div></div><p>In total, 22 publicly available whole metagenomes were used for comparison towards the two captured metagenomes (WS-72 and GL-72). After normalization of these 24 metagenomes, there were 1,124 subsystems (Level 3) whose abundances were compared between the metagenomes. Among these subsystems, there were 390 subsystems that had a significant difference in the relative frequencies (<em>P</em> &lt; 0.05, <em>q</em> &lt; 0.05) between the whole metagenomes and captured metagenomes (Fig. <a reveal-id="DSV026F5" data-open="DSV026F5" class="link link-reveal link-table xref-fig">5</a>) (PERMANOVA: <em>P</em> &lt; 0.001, <em>R</em><sup>2</sup> = 0.59). The most common subsystems that were significantly enriched in the captured metagenomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) belonged to: carbohydrates (43), clustering-based subsystem (45) and protein metabolism (18). The captured metagenomes are highly efficient in enriching for sequences related to carbohydrate subsystem as expected through design compared with the whole metagenomes (Fig. <a reveal-id="DSV026F6" data-open="DSV026F6" class="link link-reveal link-table xref-fig">6</a>). </p>                        <a id="87200475" scrollto-destination="87200475"></a>
<div id="DSV026F5" data-id="DSV026F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02605.png?Expires=2147483647&Signature=P9Ow7qv~RCqXVkad49NN2ZuPR7M46hTxm~MtzUZ-LxbPtjM0heHdkhJ9V~uMUIfFMUgK9b8dxw59AbX4A-Q9e83iyWdRgDKgFvWGG04gMh4oNDVdzyvNeHXkJR3Fj0bZO5dSmdlEaTjUTz8aXZfczuIUXXBdNfDp4fJBlECwn2Ur9SE2HUPWCnAku-NGCuM8SIwHCOs8j1EJpjJvHx7tUnOs7fsJiqomuUZ1PaOihh9BSCQm7tBVCeIUtWSX4qe7d-w0m4tvZoNlxVzEK2cQjVRpDDBrrnRvCZkDOXekPJAP9zL1cpn1vV-my8x~gLeD-AMKGL3tsnml1ygFDY74Wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes (SeqCap) and whole metagenomes from the MG-RAST database (WMG) showing that the captured metagenomes are clustered completely different from the whole metagenomes (P &lt; 0.001). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200475/dsv02605.tiff" path-from-xml="dsv02605.tiff" target="_blank">View large</a><a section="87200475" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02605.png?Expires=2147483647&Signature=tIyuGgayO0HqWIZnNBRxhfkIPDPoS2kedTR5gNlwEqXBtwyAsO~C6~E15WT-SNntBQGPL9Z9ZG8rgYu3~815hY79XgMjs~SrPoAG3crd3W1JXs~Q-z8D8wgUO7vJGR-na2tFSARXSbXM~nyotMrLz~G2fWB5BbcuqA6aQOzXT4LYY4SK~y2VDz1jmJ5vd1UlHEX67wBHe~V6e-uOqe9qKi6KWBjENx9DzIFMJ9wiq~RcxvMUg9x733LkiSz3kn-NAgMBCQQMsaFVK2J-MopmR-txIAzg7Hp-rRI0Rk2JPy-JB2wgWMSe6BLr7mGNfS-sx6qvoVqfrWBKz~Zp9PlKjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200475&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes (SeqCap) and whole metagenomes from the MG-RAST database (WMG) showing that the captured metagenomes are clustered completely different from the whole metagenomes (<em>P</em> &lt; 0.001). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02605.png?Expires=2147483647&Signature=P9Ow7qv~RCqXVkad49NN2ZuPR7M46hTxm~MtzUZ-LxbPtjM0heHdkhJ9V~uMUIfFMUgK9b8dxw59AbX4A-Q9e83iyWdRgDKgFvWGG04gMh4oNDVdzyvNeHXkJR3Fj0bZO5dSmdlEaTjUTz8aXZfczuIUXXBdNfDp4fJBlECwn2Ur9SE2HUPWCnAku-NGCuM8SIwHCOs8j1EJpjJvHx7tUnOs7fsJiqomuUZ1PaOihh9BSCQm7tBVCeIUtWSX4qe7d-w0m4tvZoNlxVzEK2cQjVRpDDBrrnRvCZkDOXekPJAP9zL1cpn1vV-my8x~gLeD-AMKGL3tsnml1ygFDY74Wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes (SeqCap) and whole metagenomes from the MG-RAST database (WMG) showing that the captured metagenomes are clustered completely different from the whole metagenomes (P &lt; 0.001). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02605.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200475/dsv02605.tiff" path-from-xml="dsv02605.tiff" target="_blank">View large</a><a section="87200475" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02605.png?Expires=2147483647&Signature=tIyuGgayO0HqWIZnNBRxhfkIPDPoS2kedTR5gNlwEqXBtwyAsO~C6~E15WT-SNntBQGPL9Z9ZG8rgYu3~815hY79XgMjs~SrPoAG3crd3W1JXs~Q-z8D8wgUO7vJGR-na2tFSARXSbXM~nyotMrLz~G2fWB5BbcuqA6aQOzXT4LYY4SK~y2VDz1jmJ5vd1UlHEX67wBHe~V6e-uOqe9qKi6KWBjENx9DzIFMJ9wiq~RcxvMUg9x733LkiSz3kn-NAgMBCQQMsaFVK2J-MopmR-txIAzg7Hp-rRI0Rk2JPy-JB2wgWMSe6BLr7mGNfS-sx6qvoVqfrWBKz~Zp9PlKjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200475&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02605.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A PCA based on normalized subsystem (Level 3) abundances between the captured metagenomes (SeqCap) and whole metagenomes from the MG-RAST database (WMG) showing that the captured metagenomes are clustered completely different from the whole metagenomes (<em>P</em> &lt; 0.001). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="87200476" scrollto-destination="87200476"></a>
<div id="DSV026F6" data-id="DSV026F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02606.png?Expires=2147483647&Signature=G5RBmE6pj4been3Q~FFQw1o0-tvy4-Eq8vttwf8bCnxX6dRPPOAZNvYzWBgig~e3g2VC8MtiStD7t71fTultKcQCxgkwfdiGDxGwYpatN5BNZQDjjwtqe63JRKqvdiVkyfGr05yOZgo3lHmEFCLp468p0ga-klxRLkBt33c3JLBmFJiYVvbYgLyn-~KmiEzCnN6G13ZvY3Al1laP2Jw2glP9K5PsXmMzacyM0vMFDFDQnt2BTyJac7PlmIDkqyycX4v90LvA2yjIGfjjYmpWeikteKE6Jb43a-PUeepnG7XD7QJA6Af3erx7L70YbLpGp2YMwmD840W~9P9w0iYmBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The mean proportions of the significantly different subsystems (Level 1) in the captured metagenomes (SeqCap) from the 22 whole metagenomes from the MG-RAST database (WMG) with their respective 95% CI and corrected P-values. (B) The proportion of carbohydrate subsystem in the captured metagenomes (SeqCap) compared with 22 whole metagenomes (WMG). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02606.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200476/dsv02606.tiff" path-from-xml="dsv02606.tiff" target="_blank">View large</a><a section="87200476" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02606.png?Expires=2147483647&Signature=W7HVjME6lyQFizBkVaoVvhFmWoz6xggw824BJ1A~yBfBBul354eCJ7lwBLOaohM7iFwO94zK~gBDjFyeX0RbGaZSt5Q19S1d1wPGbc3mTaBO36Ze44-o4nQvBM-WCe8GhfHNR6irMPwtpEb0uiUP8-fkApIDhIUnG8AUdTodkHeJx5UpO794RhA3kB-g7c5YZ~wvwvgueeJwlEum5tH5bHnb3uN8r0uGbkeNcBiVW7fFtF3FRT~B~E6HEV-y~fV6H8Rj5e4bnM00U3oDoH1RAxlrFBJ-4Lf1AkAuZu58AMCeU1JgHb9XMpQwKfr8xiEUei2bA7R88sRfXbN1QDuCOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200476&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02606.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The mean proportions of the significantly different subsystems (Level 1) in the captured metagenomes (SeqCap) from the 22 whole metagenomes from the MG-RAST database (WMG) with their respective 95% CI and corrected <em>P</em>-values. (B) The proportion of carbohydrate subsystem in the captured metagenomes (SeqCap) compared with 22 whole metagenomes (WMG). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV026F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/m_dsv02606.png?Expires=2147483647&Signature=G5RBmE6pj4been3Q~FFQw1o0-tvy4-Eq8vttwf8bCnxX6dRPPOAZNvYzWBgig~e3g2VC8MtiStD7t71fTultKcQCxgkwfdiGDxGwYpatN5BNZQDjjwtqe63JRKqvdiVkyfGr05yOZgo3lHmEFCLp468p0ga-klxRLkBt33c3JLBmFJiYVvbYgLyn-~KmiEzCnN6G13ZvY3Al1laP2Jw2glP9K5PsXmMzacyM0vMFDFDQnt2BTyJac7PlmIDkqyycX4v90LvA2yjIGfjjYmpWeikteKE6Jb43a-PUeepnG7XD7QJA6Af3erx7L70YbLpGp2YMwmD840W~9P9w0iYmBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The mean proportions of the significantly different subsystems (Level 1) in the captured metagenomes (SeqCap) from the 22 whole metagenomes from the MG-RAST database (WMG) with their respective 95% CI and corrected P-values. (B) The proportion of carbohydrate subsystem in the captured metagenomes (SeqCap) compared with 22 whole metagenomes (WMG). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02606.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200476/dsv02606.tiff" path-from-xml="dsv02606.tiff" target="_blank">View large</a><a section="87200476" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv02606.png?Expires=2147483647&Signature=W7HVjME6lyQFizBkVaoVvhFmWoz6xggw824BJ1A~yBfBBul354eCJ7lwBLOaohM7iFwO94zK~gBDjFyeX0RbGaZSt5Q19S1d1wPGbc3mTaBO36Ze44-o4nQvBM-WCe8GhfHNR6irMPwtpEb0uiUP8-fkApIDhIUnG8AUdTodkHeJx5UpO794RhA3kB-g7c5YZ~wvwvgueeJwlEum5tH5bHnb3uN8r0uGbkeNcBiVW7fFtF3FRT~B~E6HEV-y~fV6H8Rj5e4bnM00U3oDoH1RAxlrFBJ-4Lf1AkAuZu58AMCeU1JgHb9XMpQwKfr8xiEUei2bA7R88sRfXbN1QDuCOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200476&ar=2462493&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02606.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The mean proportions of the significantly different subsystems (Level 1) in the captured metagenomes (SeqCap) from the 22 whole metagenomes from the MG-RAST database (WMG) with their respective 95% CI and corrected <em>P</em>-values. (B) The proportion of carbohydrate subsystem in the captured metagenomes (SeqCap) compared with 22 whole metagenomes (WMG). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87200477 id="87200477" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>This study has for the first time applied a targeted metagenomic approach to study genes of more than hundreds of different enzyme families in environmental samples. The approach was successfully demonstrated for soil from two different land-use types, grass and wheat, but is applicable to any metagenome. The soil functional diversity can be efficiently investigated by using the sequence capture technique that has been developed with an online web tool for probe design<sup><a reveal-id="DSV026C28" data-open="DSV026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> for genes encoding enzymes regulating specific functional mechanisms in ecosystems. This method circumvents the inefficiency due to coverage or high labour demand for gene targeting, of the techniques that are commonly applied for the study of community functions.<sup><a reveal-id="DSV026C8" data-open="DSV026C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV026C9" data-open="DSV026C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSV026C13" data-open="DSV026C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV026C40" data-open="DSV026C40" class="link link-ref link-reveal xref-bibr">40</a></sup> We have applied this technique successfully for two different soil metagenomes to target enzymes involved in organic matter degradation.</p><p>The abundances of these target enzymes in each captured metagenome were significantly affected by the hybridization time. However, the diversity of these targeted enzymes was consistent within each soil regardless of their hybridization time. This shows that stringent hybridization, in terms of hybridization time, increases the enrichment of target genes while maintaining the diversity. It could also be inferred that this technique is reproducible as there is an increase in target enrichment while maintaining diversity at higher hybridization stringencies for both metagenomes. Lower hybridization stringencies also increase the fraction of proteins without any annotations in the public databases in the metagenome. This is mainly due to the random hybridizations of DNA fragments to probes (i.e. increased number of off-targets), which would be less likely at higher stringencies. Since this technique involves sequencing of the hybridized fragments, it provides the advantage of separating out the randomly bound fragments from the ones that contain functional annotation compared with microarray-based techniques.<sup><a reveal-id="DSV026C13" data-open="DSV026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> It is also important to note that these fragments were sequenced with 454 for longer sequence reads. As 454 frequently does not sequence the entire DNA fragment, adjacent non-coding region instead of the region bound to the probe might have been sequenced, which could also hinder their functional annotations. In addition, 454 also has higher per-base sequencing errors like homopolymer errors<sup><a reveal-id="DSV026C41" data-open="DSV026C41" class="link link-ref link-reveal xref-bibr">41</a></sup> that could cause problems while matching sequence reads for homology in databases. These issues could be solved with other less error-prone sequencing technologies such as Illumina.</p><p>Most of the metabolic gene predictions in the captured metagenomic datasets for both soils were annotated to carbohydrate metabolism. Other major metabolic components (e.g. amino acid metabolism) of the captured metagenomes are also related to organic matter degradation. This clearly illustrates the efficiency of using our customized probes to enrich for regions of interest in a complex metagenome and is a highly suitable approach to increase the understanding of a particular environmental process. Similarly in the subsystem analysis, the most abundant subsystem (Level 1) was carbohydrates, also showing the efficiency of this capture technique. The second most abundant subsystem, clustering-based subsystems, are the genetic regions that are collocated to functional genes in the genomes of different taxa, but their functions are not well known.<sup><a reveal-id="DSV026C42" data-open="DSV026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> We suggest that these genes are abundant, because they are sequenced along with the targeted genetic DNA fragments. The functional subsystems distribution between the captured metagenomes was mainly influenced by their soil type rather than their hybridization stringency, although both properties significantly affected their distribution. This shows that this targeted metagenomics approach does not bias the functional representation of the soils even at very high stringency. The results clearly show the efficiency of applying this method to provide insights into important ecological questions, such as understanding key processes in complex environments.</p><p>The number of different CAZy enzyme families found from these metagenomes also shows the ability of this target enrichment strategy to obtain reliable data for &gt;200 enzyme families at different taxonomic levels. It is to be noted that an efficient enrichment of proteases was also obtained along with CAZy enzymes, although there are common enzymes between the two databases CAZy and MEROPS. Due to our approach, the numbers of CAZy enzyme families found in our metagenomes are much higher to what has been reported in earlier findings.<sup><a reveal-id="DSV026C43" data-open="DSV026C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSV026C44" data-open="DSV026C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Despite this, some enzyme families from certain enzyme class like GT were not acquired completely. This could be due to the inefficiency of probes targeting of these families or the very low abundance of these enzyme classes in these soils. Also, these enzyme classes could co-occur with other enzyme families and the annotations like blast would predict one of those with higher scores. For example, the glycosyl hydrolase 13, one of the major CAZy enzyme families, frequently co-occur with CBM48<sup><a reveal-id="DSV026C45" data-open="DSV026C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and is the most abundant enzyme family in captured metagenomes. GH13 enzyme catalysis is known to be an important step in trehalose biosynthesis,<sup><a reveal-id="DSV026C46" data-open="DSV026C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and this biosynthesis is also the most abundant subsystem in the captured metagenomes.</p><p>The taxonomic analysis of the captured metagenomes from MG-RAST showed that the target enrichment strategy applied here does not appear to bias the targeting towards any particular taxonomic group. The taxonomic distribution between all captured metagenomes was significantly explained by their soil type rather than the hybridization times. It is evident that even at the high stringency, captured metagenomes have a similar taxonomic distribution as the other metagenomes from the same soil. Hence, the developed sequence capture method does not have any taxonomic bias. This was further supported by the significant correlations between captured metagenomes (WS-72 and GL-72) and the amplicon metagenomes based on the relative abundances of taxa at different levels. The changes in certain taxa between captured metagenomes and amplicon sequencing could be explained through the availability of functional information of particular taxa. The phylum acidobacteria, for example, is represented highly in 16S rDNA amplicon data but not in case of the captured metagenomes. This can be explained partially that it is well known for its abundance in soil communities but has not been extensively studied for their functions.<sup><a reveal-id="DSV026C47" data-open="DSV026C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSV026C48" data-open="DSV026C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The fraction of genes coding for enzymes from Acidobacteria used for our probe were measured to be only 0.6% in TDB, also only 0.9% among the other bacterial enzymes. This limitation is not due to the capture technique, but rather due to biases in the public databases. Several ongoing projects are aiming at increasing the resolution of these under-studied organisms through genome sequencing,<sup><a reveal-id="DSV026C49" data-open="DSV026C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSV026C50" data-open="DSV026C50" class="link link-ref link-reveal xref-bibr">50</a></sup> which will be helpful in understanding the ecosystems better in the future.</p><p>The comparison of CAZy domains from captured metagenomes to publicly available whole metagenomes showed that enrichment through probes was more efficient than intensive sequencing to identify lowly abundant functional genes. Even the most abundant CAZy domains were &lt;1% in WMG, showing the inability of whole metagenomic approach to obtain these important genes.<sup><a reveal-id="DSV026C51" data-open="DSV026C51" class="link link-ref link-reveal xref-bibr">51</a></sup> However, our targeted approach efficiently enriched for genes that were targeted, at considerably lower sequencing depths. The abundance of enzymes related to carbon cycling from captured metagenomes was much higher than the whole metagenomes (WMG) despite that the WMG was very deeply sequenced. These 22 WMG were obtained from different soils at different conditions, including metagenomes from places like rain forest in Puerto Rico, permafrost and high arctic soils which are known to have communities with higher organic matter degradation capabilities.<sup><a reveal-id="DSV026C51  DSV026C53" data-open="DSV026C51  DSV026C53" class="link link-ref link-reveal xref-bibr">51–53</a></sup></p><p>The differences between two soil communities (WS and GL) were deducible with the availability of abundance and richness of the enzymes in captured metagenomes. It was clear that enzyme families such as GH10 coding for xylanases<sup><a reveal-id="DSV026C10" data-open="DSV026C10" class="link link-ref link-reveal xref-bibr">10</a></sup> were abundant in grassland as expected since it is related to degradation of plant cell-wall material. Similarly amino peptidases are also found abundant in grassland, as the soil does not receive any free nitrogen as in the case of wheat soil through fertilization. This also shows the ability of captured metagenomics to explain soil functionality. Although, it is important to note that the probes used in this method depend on known functions/enzymes in databases and the information related to unknown functions/enzymes in communities may only be obtained if they were genetically similar. WMG could still be a way of obtaining such information in these particular cases but not to forget that annotating them could still be a subject of biases.<sup><a reveal-id="DSV026C54" data-open="DSV026C54" class="link link-ref link-reveal xref-bibr">54</a></sup></p><p>This study has for the first time implemented the sequence capture technique to study the functional diversity of enzymes degrading organic matter in natural soil communities. This approach has the ability to solve the coverage issues with the WMG to get enough amounts of representative sequences.<sup><a reveal-id="DSV026C55" data-open="DSV026C55" class="link link-ref link-reveal xref-bibr">55</a></sup> We also argue that it is superior to the large-scale microarrays<sup><a reveal-id="DSV026C56" data-open="DSV026C56" class="link link-ref link-reveal xref-bibr">56</a></sup> since it is able to detect unspecific binding of DNA fragments to the probes and facilitates the discovery of novel genes. It has also been clearly shown, based on multiple hybridization stringencies, that this technique is highly stable and reproducible. As this method represents the taxonomic diversity very well, it could be used to understand the relation between the taxonomic and functional diversity present in the environmental communities.<sup><a reveal-id="DSV026C1" data-open="DSV026C1" class="link link-ref link-reveal xref-bibr">1</a></sup> As mentioned in Fierer et al.,<sup><a reveal-id="DSV026C40" data-open="DSV026C40" class="link link-ref link-reveal xref-bibr">40</a></sup> this approach would be an excellent tool for measuring ‘community aggregated traits’ (CAT) and hence proved an integrated understanding of the functional capabilities of complex microbial ecosystems. The relative functional gene abundances from different samples, as traits irrespective of its taxonomic origin, are a better way to determine the functional capabilities of the community. Overall, this method demonstrates its ability to improve our understanding of those community functions in different ecological processes of interest and is applicable even to the most complex metagenomes.</p>                            <h2 scrollto-destination=87200486 id="87200486" class="section-title" data-legacy-id=s5>Conflict of interest statement</h2>
<p>None declared.</p>                            <h2 scrollto-destination=87200488 id="87200488" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by The Swedish Research Council for Environment, Agricultural Sciences and Spatial Planning (215-2010-1679). Funding to pay the Open Access publication charges for this article was provided by Lund University library and The Swedish Research Council for Environment, Agricultural Sciences and Spatial Planning (215-2010-1679).</p>                            <h2 scrollto-destination=87200490 id="87200490" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Helene Bracht Jørgensen (Lund University, Sweden) for helping us with the sampling of soils and Dimitrios Floudas (Lund University, Sweden) for the valuable discussions. We also thank Xavier Miro (Roche) for his suggestions and help with the sequence capture experimental procedures.</p>                            <h2 scrollto-destination=87200492 id="87200492" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV026C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gilbert</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Field</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Swift</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">The taxonomic and functional diversity of microbes at a temperate coastal site: a ‘multi-omic’ study of seasonal and diel temporal variation</div>, <div class="source">PLoS ONE</div> , <div class="volume">5</div>, <div class="fpage">e15545</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20taxonomic%20and%20functional%20diversity%20of%20microbes%20at%20a%20temperate%20coastal%20site%3A%20a%20%E2%80%98multi-omic%E2%80%99%20study%20of%20seasonal%20and%20diel%20temporal%20variation&amp;author=J.A.%20Gilbert&amp;author=D.%20Field&amp;author=P.%20Swift&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e15545" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0015545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0015545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0015545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21124740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Diaz</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Cabido</div> <div class="given-names">M.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Vive la difference: plant functional diversity matters to ecosystem processes</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="volume">16</div>, <div class="fpage">646</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vive%20la%20difference%3A%20plant%20functional%20diversity%20matters%20to%20ecosystem%20processes&amp;author=S.%20Diaz&amp;author=M.%20Cabido&amp;publication_year=2001&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=16&amp;pages=646-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(01)02283-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(01)02283-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(01)02283-2"> </span></div> </div></div></div></div></div><div content-id="DSV026C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gravel</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Bell</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Barbera</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Combe</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pommier</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mouquet</div> <div class="given-names">N.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Phylogenetic constraints on ecosystem functioning</div>, <div class="source">Nat. Commun.</div> , <div class="volume">3</div>, <div class="fpage">1117</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20constraints%20on%20ecosystem%20functioning&amp;author=D.%20Gravel&amp;author=T.%20Bell&amp;author=C.%20Barbera&amp;author=M.%20Combe&amp;author=T.%20Pommier&amp;author=N.%20Mouquet&amp;publication_year=2012&amp;journal=Nat.%20Commun.&amp;volume=3&amp;pages=1117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fitter</div>
               <div class="given-names">A.H.</div>
            </div>, <div class="name string-name"><div class="surname">Gilligan</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Hollingworth</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Biodiversity and ecosystem function in soil</div>, <div class="source">Funct. Ecol.</div> , <div class="volume">19</div>, <div class="fpage">369</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biodiversity%20and%20ecosystem%20function%20in%20soil&amp;author=A.H.%20Fitter&amp;author=C.A.%20Gilligan&amp;author=K.%20Hollingworth&amp;publication_year=2005&amp;journal=Funct.%20Ecol.&amp;volume=19&amp;pages=369-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fec.2005.19.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffec.2005.19.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffec.2005.19.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSV026C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coleman</div>
               <div class="given-names">D.C.</div>
            </div>, <div class="name string-name"><div class="surname">Whitman</div> <div class="given-names">W.B.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Linking species richness, biodiversity and ecosystem function in soil systems</div>, <div class="source">Pedobiologia</div> , <div class="volume">49</div>, <div class="fpage">479</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linking%20species%20richness%2C%20biodiversity%20and%20ecosystem%20function%20in%20soil%20systems&amp;author=D.C.%20Coleman&amp;author=W.B.%20Whitman&amp;publication_year=2005&amp;journal=Pedobiologia&amp;volume=49&amp;pages=479-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pedobi.2005.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pedobi.2005.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pedobi.2005.05.006"> </span></div> </div></div></div></div></div><div content-id="DSV026C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burns</div>
               <div class="given-names">R.G.</div>
            </div>, <div class="name string-name"><div class="surname">DeForest</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Marxsen</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Soil enzymes in a changing environment: current knowledge and future directions</div>, <div class="source">Soil Biol. Biochem.</div> , <div class="volume">58</div>, <div class="fpage">216</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soil%20enzymes%20in%20a%20changing%20environment%3A%20current%20knowledge%20and%20future%20directions&amp;author=R.G.%20Burns&amp;author=J.L.%20DeForest&amp;author=J.%20Marxsen&amp;publication_year=2013&amp;journal=Soil%20Biol.%20Biochem.&amp;volume=58&amp;pages=216-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.soilbio.2012.11.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.soilbio.2012.11.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.soilbio.2012.11.009"> </span></div> </div></div></div></div></div><div content-id="DSV026C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gilbert</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Dupont</div> <div class="given-names">C.L.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Microbial metagenomics: beyond the genome</div>, <div class="source">Annu. Rev. Mar. Sci.</div> , <div class="volume">3</div>, <div class="fpage">347</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20metagenomics%3A%20beyond%20the%20genome&amp;author=J.A.%20Gilbert&amp;author=C.L.%20Dupont&amp;publication_year=2011&amp;journal=Annu.%20Rev.%20Mar.%20Sci.&amp;volume=3&amp;pages=347-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-marine-120709-142811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-marine-120709-142811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-marine-120709-142811"> </span></div> </div></div></div></div></div><div content-id="DSV026C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kunin</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Copeland</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lapidus</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mavromatis</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">A bioinformatician's guide to metagenomics</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="volume">72</div>, <div class="fpage">557</div>–<div class="lpage">78</div>, <div class="comment">Table of Contents</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20bioinformatician's%20guide%20to%20metagenomics&amp;author=V.%20Kunin&amp;author=A.%20Copeland&amp;author=A.%20Lapidus&amp;author=K.%20Mavromatis&amp;author=P.%20Hugenholtz&amp;publication_year=2008&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=72&amp;pages=557-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00009-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00009-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00009-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19052320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wooley</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Friedberg</div> <div class="given-names">I.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">A primer on metagenomics</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">6</div>, <div class="fpage">e1000667</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20primer%20on%20metagenomics&amp;author=J.C.%20Wooley&amp;author=A.%20Godzik&amp;author=I.%20Friedberg&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=6&amp;pages=e1000667" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000667" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000667" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000667"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20195499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">G.Z.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.R.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">P.L.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Molecular detection and diversity of xylanase genes in alpine tundra soil</div>, <div class="source">Appl. Microbiol. Biot.</div> , <div class="volume">87</div>, <div class="fpage">1383</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20detection%20and%20diversity%20of%20xylanase%20genes%20in%20alpine%20tundra%20soil&amp;author=G.Z.%20Wang&amp;author=Y.R.%20Wang&amp;author=P.L.%20Yang&amp;publication_year=2010&amp;journal=Appl.%20Microbiol.%20Biot.&amp;volume=87&amp;pages=1383-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-010-2564-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-010-2564-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-010-2564-9"> </span></div> </div></div></div></div></div><div content-id="DSV026C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rotthauwe</div>
               <div class="given-names">J.H.</div>
            </div>, <div class="name string-name"><div class="surname">Witzel</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Liesack</div> <div class="given-names">W.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">The ammonia monooxygenase structural gene amoA as a functional marker: molecular fine-scale analysis of natural ammonia-oxidizing populations</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">63</div>, <div class="fpage">4704</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ammonia%20monooxygenase%20structural%20gene%20amoA%20as%20a%20functional%20marker%3A%20molecular%20fine-scale%20analysis%20of%20natural%20ammonia-oxidizing%20populations&amp;author=J.H.%20Rotthauwe&amp;author=K.P.%20Witzel&amp;author=W.%20Liesack&amp;publication_year=1997&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=63&amp;pages=4704-12" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9406389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">He</div>
               <div class="given-names">Z.L.</div>
            </div>, <div class="name string-name"><div class="surname">Van Nostrand</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">L.Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.Z.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Development and application of functional gene arrays for microbial community analysis</div>, <div class="source">T. Nonferr. Metal. Soc.</div> , <div class="volume">18</div>, <div class="fpage">1319</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20application%20of%20functional%20gene%20arrays%20for%20microbial%20community%20analysis&amp;author=Z.L.%20He&amp;author=J.D.%20Van%20Nostrand&amp;author=L.Y.%20Wu&amp;author=J.Z.%20Zhou&amp;publication_year=2008&amp;journal=T.%20Nonferr.%20Metal.%20Soc.&amp;volume=18&amp;pages=1319-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1003-6326(09)60004-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1003-6326(09)60004-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1003-6326(09)60004-2"> </span></div> </div></div></div></div></div><div content-id="DSV026C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barton</div>
               <div class="given-names">L.L.</div>
            </div>, <div class="name string-name"><div class="surname">Mandl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Loy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nostrand</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Analysis of microbial communities by functional gene arrays</div>. In: <div class="source">Geomicrobiology: molecular and environmental perspective</div> . <div class="publisher-name">Springer</div>: <div class="publisher-loc">The Netherlands</div>, pp.<div class="fpage">109</div>–<div class="lpage">26</div>.</div></div></div></div><div content-id="DSV026C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Suenaga</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2012</div>, <div class="article-title">Targeted metagenomics: a high-resolution metagenomics approach for specific gene clusters in complex microbial communities</div>, <div class="source">Environ. Microbiol.</div> , <div class="volume">14</div>, <div class="fpage">13</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20metagenomics%3A%20a%20high-resolution%20metagenomics%20approach%20for%20specific%20gene%20clusters%20in%20complex%20microbial%20communities&amp;author=H.%20Suenaga&amp;publication_year=2012&amp;journal=Environ.%20Microbiol.&amp;volume=14&amp;pages=13-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2011.02438.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2011.02438.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2011.02438.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21366818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Albert</div>
               <div class="given-names">T.J.</div>
            </div>, <div class="name string-name"><div class="surname">Molla</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Muzny</div> <div class="given-names">D.M.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Direct selection of human genomic loci by microarray hybridization</div>, <div class="source">Nat. Methods</div> , <div class="volume">4</div>, <div class="fpage">903</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization&amp;author=T.J.%20Albert&amp;author=M.N.%20Molla&amp;author=D.M.%20Muzny&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burbano</div>
               <div class="given-names">H.A.</div>
            </div>, <div class="name string-name"><div class="surname">Hodges</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">R.E.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Targeted investigation of the Neandertal genome by array-based sequence capture</div>, <div class="source">Science</div> , <div class="volume">328</div>, <div class="fpage">723</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20investigation%20of%20the%20Neandertal%20genome%20by%20array-based%20sequence%20capture&amp;author=H.A.%20Burbano&amp;author=E.%20Hodges&amp;author=R.E.%20Green&amp;publication_year=2010&amp;journal=Science&amp;volume=328&amp;pages=723-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1188046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1188046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1188046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20448179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ng</div>
               <div class="given-names">S.B.</div>
            </div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">E.H.</div></div>, <div class="name string-name"><div class="surname">Robertson</div> <div class="given-names">P.D.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Targeted capture and massively parallel sequencing of 12 human exomes</div>, <div class="source">Nature</div> , <div class="volume">461</div>, <div class="fpage">272</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20capture%20and%20massively%20parallel%20sequencing%20of%2012%20human%20exomes&amp;author=S.B.%20Ng&amp;author=E.H.%20Turner&amp;author=P.D.%20Robertson&amp;publication_year=2009&amp;journal=Nature&amp;volume=461&amp;pages=272-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19684571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rehman</div>
               <div class="given-names">A.U.</div>
            </div>, <div class="name string-name"><div class="surname">Morell</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Belyantseva</div> <div class="given-names">I.A.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Targeted capture and next-generation sequencing identifies C9orf75, encoding taperin, as the mutated gene in nonsyndromic deafness DFNB79</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">86</div>, <div class="fpage">378</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20capture%20and%20next-generation%20sequencing%20identifies%20C9orf75%2C%20encoding%20taperin%2C%20as%20the%20mutated%20gene%20in%20nonsyndromic%20deafness%20DFNB79&amp;author=A.U.%20Rehman&amp;author=R.J.%20Morell&amp;author=I.A.%20Belyantseva&amp;publication_year=2010&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=86&amp;pages=378-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajhg.2010.01.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2010.01.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajhg.2010.01.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20170899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ross</div>
               <div class="given-names">J.S.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Gay</div> <div class="given-names">L.M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">A high frequency of activating extracellular domain ERBB2 (HER2) mutation in micropapillary urothelial carcinoma</div>, <div class="source">Clin. Cancer Res.</div> , <div class="volume">20</div>, <div class="fpage">68</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20frequency%20of%20activating%20extracellular%20domain%20ERBB2%20(HER2)%20mutation%20in%20micropapillary%20urothelial%20carcinoma&amp;author=J.S.%20Ross&amp;author=K.%20Wang&amp;author=L.M.%20Gay&amp;publication_year=2014&amp;journal=Clin.%20Cancer%20Res.&amp;volume=20&amp;pages=68-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-13-1992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-13-1992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-13-1992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24192927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duncavage</div>
               <div class="given-names">E.J.</div>
            </div>, <div class="name string-name"><div class="surname">Abel</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Szankasi</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Pfeifer</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Targeted next generation sequencing of clinically significant gene mutations and translocations in leukemia</div>, <div class="source">Modern Pathol.</div> , <div class="volume">25</div>, <div class="fpage">795</div>–<div class="lpage">804</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20next%20generation%20sequencing%20of%20clinically%20significant%20gene%20mutations%20and%20translocations%20in%20leukemia&amp;author=E.J.%20Duncavage&amp;author=H.J.%20Abel&amp;author=P.%20Szankasi&amp;author=T.W.%20Kelley&amp;author=J.D.%20Pfeifer&amp;publication_year=2012&amp;journal=Modern%20Pathol.&amp;volume=25&amp;pages=795-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/modpathol.2012.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmodpathol.2012.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmodpathol.2012.29"> </span></div> </div></div></div></div></div><div content-id="DSV026C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bi</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Vanderpool</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Singhal</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Linderoth</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Moritz</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Good</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Transcriptome-based exon capture enables highly cost-effective comparative genomic data collection at moderate evolutionary scales</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome-based%20exon%20capture%20enables%20highly%20cost-effective%20comparative%20genomic%20data%20collection%20at%20moderate%20evolutionary%20scales&amp;author=K.%20Bi&amp;author=D.%20Vanderpool&amp;author=S.%20Singhal&amp;author=T.%20Linderoth&amp;author=C.%20Moritz&amp;author=J.M.%20Good&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22900609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bundock</div>
               <div class="given-names">P.C.</div>
            </div>, <div class="name string-name"><div class="surname">Casu</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Henry</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Enrichment of genomic DNA for polymorphism detection in a non-model highly polyploid crop plant</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">10</div>, <div class="fpage">657</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20genomic%20DNA%20for%20polymorphism%20detection%20in%20a%20non-model%20highly%20polyploid%20crop%20plant&amp;author=P.C.%20Bundock&amp;author=R.E.%20Casu&amp;author=R.J.%20Henry&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=657-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00707.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00707.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00707.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22624722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cosart</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Beja-Pereira</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ng</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Luikart</div> <div class="given-names">G.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Exome-wide DNA capture and next generation sequencing in domestic and wild species</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">347</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exome-wide%20DNA%20capture%20and%20next%20generation%20sequencing%20in%20domestic%20and%20wild%20species&amp;author=T.%20Cosart&amp;author=A.%20Beja-Pereira&amp;author=S.%20Chen&amp;author=S.B.%20Ng&amp;author=J.%20Shendure&amp;author=G.%20Luikart&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=347" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-347" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-347" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-347"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21729323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Denonfoux</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Parisot</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Dugat-Bony</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Gene capture coupled to high-throughput sequencing as a strategy for targeted metagenome exploration</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">185</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20capture%20coupled%20to%20high-throughput%20sequencing%20as%20a%20strategy%20for%20targeted%20metagenome%20exploration&amp;author=J.%20Denonfoux&amp;author=N.%20Parisot&amp;author=E.%20Dugat-Bony&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=185-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23364577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cantarel</div>
               <div class="given-names">B.L.</div>
            </div>, <div class="name string-name"><div class="surname">Coutinho</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Rancurel</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bernard</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">The Carbohydrate-Active EnZymes database (CAZy): an expert resource for Glycogenomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">37</div>, <div class="fpage">D233</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Carbohydrate-Active%20EnZymes%20database%20(CAZy)%3A%20an%20expert%20resource%20for%20Glycogenomics&amp;author=B.L.%20Cantarel&amp;author=P.M.%20Coutinho&amp;author=C.%20Rancurel&amp;author=T.%20Bernard&amp;author=V.%20Lombard&amp;author=B.%20Henrissat&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D233-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18838391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levasseur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Piumi</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Coutinho</div> <div class="given-names">P.M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">FOLy: an integrated database for the classification and functional annotation of fungal oxidoreductases potentially involved in the degradation of lignin and related aromatic compounds</div>, <div class="source">Fungal Genet. Biol.</div> , <div class="volume">45</div>, <div class="fpage">638</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FOLy%3A%20an%20integrated%20database%20for%20the%20classification%20and%20functional%20annotation%20of%20fungal%20oxidoreductases%20potentially%20involved%20in%20the%20degradation%20of%20lignin%20and%20related%20aromatic%20compounds&amp;author=A.%20Levasseur&amp;author=F.%20Piumi&amp;author=P.M.%20Coutinho&amp;publication_year=2008&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=45&amp;pages=638-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2008.01.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2008.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2008.01.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18308593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rawlings</div>
               <div class="given-names">N.D.</div>
            </div>, <div class="name string-name"><div class="surname">Barrett</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">MEROPS: the database of proteolytic enzymes, their substrates and inhibitors</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D343</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEROPS%3A%20the%20database%20of%20proteolytic%20enzymes%2C%20their%20substrates%20and%20inhibitors&amp;author=N.D.%20Rawlings&amp;author=A.J.%20Barrett&amp;author=A.%20Bateman&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D343-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kushwaha</div>
               <div class="given-names">S.K.</div>
            </div>, <div class="name string-name"><div class="surname">Manoharan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Meerupati</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hedlund</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ahrén</div> <div class="given-names">D.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">MetCap: a bioinformatics probe design pipeline for large-scale targeted metagenomics</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">16</div>, <div class="fpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetCap%3A%20a%20bioinformatics%20probe%20design%20pipeline%20for%20large-scale%20targeted%20metagenomics&amp;author=S.K.%20Kushwaha&amp;author=L.%20Manoharan&amp;author=T.%20Meerupati&amp;author=K.%20Hedlund&amp;author=D.%20Ahr%C3%A9n&amp;publication_year=2015&amp;journal=BMC%20Bioinformatics&amp;volume=16&amp;pages=65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12859-015-0501-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-015-0501-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12859-015-0501-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25880302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsiafouli</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Thebault</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Sgardelis</div> <div class="given-names">S.P.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Intensive agriculture reduces soil biodiversity across Europe</div>, <div class="source">Global Chang. Biol</div> ., <div class="volume">21</div>, <div class="fpage">973</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intensive%20agriculture%20reduces%20soil%20biodiversity%20across%20Europe&amp;author=M.A.%20Tsiafouli&amp;author=E.%20Thebault&amp;author=S.P.%20Sgardelis&amp;publication_year=2014&amp;journal=Global%20Chang.%20Biol&amp;volume=21&amp;pages=973-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/gcb.12752" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgcb.12752" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgcb.12752"> </span></div> </div></div></div></div></div><div content-id="DSV026C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Emanuelsson</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Locating proteins in the cell using TargetP, SignalP and related tools</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">2</div>, <div class="fpage">953</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20proteins%20in%20the%20cell%20using%20TargetP%2C%20SignalP%20and%20related%20tools&amp;author=O.%20Emanuelsson&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2007&amp;journal=Nat.%20Protoc.&amp;volume=2&amp;pages=953-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2007.131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2007.131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Herlemann</div>
               <div class="given-names">D.P.R.</div>
            </div>, <div class="name string-name"><div class="surname">Labrenz</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jurgens</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bertilsson</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Waniek</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Andersson</div> <div class="given-names">A.F.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Transitions in bacterial communities along the 2000km salinity gradient of the Baltic Sea</div>, <div class="source">Isme J.</div> , <div class="volume">5</div>, <div class="fpage">1571</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transitions%20in%20bacterial%20communities%20along%20the%202000km%20salinity%20gradient%20of%20the%20Baltic%20Sea&amp;author=D.P.R.%20Herlemann&amp;author=M.%20Labrenz&amp;author=K.%20Jurgens&amp;author=S.%20Bertilsson&amp;author=J.J.%20Waniek&amp;author=A.F.%20Andersson&amp;publication_year=2011&amp;journal=Isme%20J.&amp;volume=5&amp;pages=1571-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2011.41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2011.41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21472016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Meyer</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Paarmann</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">D'Souza</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">The metagenomics RAST server - a public resource for the automatic phylogenetic and functional analysis of metagenomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">386</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20metagenomics%20RAST%20server%20-%20a%20public%20resource%20for%20the%20automatic%20phylogenetic%20and%20functional%20analysis%20of%20metagenomes&amp;author=F.%20Meyer&amp;author=D.%20Paarmann&amp;author=M.%20D'Souza&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-386" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-386" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-386"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18803844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gomez-Alvarez</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Teal</div> <div class="given-names">T.K.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">T.M.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Systematic artifacts in metagenomes from complex microbial communities</div>, <div class="source">Isme J.</div> , <div class="volume">3</div>, <div class="fpage">1314</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20artifacts%20in%20metagenomes%20from%20complex%20microbial%20communities&amp;author=V.%20Gomez-Alvarez&amp;author=T.K.%20Teal&amp;author=T.M.%20Schmidt&amp;publication_year=2009&amp;journal=Isme%20J.&amp;volume=3&amp;pages=1314-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19587772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Overbeek</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Begley</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">R.M.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">5691</div>–<div class="lpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20subsystems%20approach%20to%20genome%20annotation%20and%20its%20use%20in%20the%20project%20to%20annotate%201000%20genomes&amp;author=R.%20Overbeek&amp;author=T.%20Begley&amp;author=R.M.%20Butler&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5691-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16214803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Park</div>
               <div class="given-names">B.H.</div>
            </div>, <div class="name string-name"><div class="surname">Karpinets</div> <div class="given-names">T.V.</div></div>, <div class="name string-name"><div class="surname">Syed</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Leuze</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Uberbacher</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">CAZymes Analysis Toolkit (CAT): Web service for searching and analyzing carbohydrate-active enzymes in a newly sequenced organism using CAZy database</div>, <div class="source">Glycobiology</div> , <div class="volume">20</div>, <div class="fpage">1574</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAZymes%20Analysis%20Toolkit%20(CAT)%3A%20Web%20service%20for%20searching%20and%20analyzing%20carbohydrate-active%20enzymes%20in%20a%20newly%20sequenced%20organism%20using%20CAZy%20database&amp;author=B.H.%20Park&amp;author=T.V.%20Karpinets&amp;author=M.H.%20Syed&amp;author=M.R.%20Leuze&amp;author=E.C.%20Uberbacher&amp;publication_year=2010&amp;journal=Glycobiology&amp;volume=20&amp;pages=1574-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/glycob/cwq106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fglycob%2Fcwq106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fglycob%2Fcwq106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20696711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Caporaso</div>
               <div class="given-names">J.G.</div>
            </div>, <div class="name string-name"><div class="surname">Kuczynski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Stombaugh</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">QIIME allows analysis of high-throughput community sequencing data</div>, <div class="source">Nat. Methods</div> , <div class="volume">7</div>, <div class="fpage">335</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QIIME%20allows%20analysis%20of%20high-throughput%20community%20sequencing%20data&amp;author=J.G.%20Caporaso&amp;author=J.%20Kuczynski&amp;author=J.%20Stombaugh&amp;publication_year=2010&amp;journal=Nat.%20Methods&amp;volume=7&amp;pages=335-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.f.303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.f.303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.f.303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20383131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McDonald</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Price</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Goodrich</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea</div>, <div class="source">Isme J.</div> , <div class="volume">6</div>, <div class="fpage">610</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20Greengenes%20taxonomy%20with%20explicit%20ranks%20for%20ecological%20and%20evolutionary%20analyses%20of%20bacteria%20and%20archaea&amp;author=D.%20McDonald&amp;author=M.N.%20Price&amp;author=J.%20Goodrich&amp;publication_year=2012&amp;journal=Isme%20J.&amp;volume=6&amp;pages=610-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2011.139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2011.139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22134646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Braithwaite</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Keegan</div> <div class="given-names">K.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">matR: Metagenomics Analysis Tools for R</div>, <div class="comment"><a class="link link-uri" href="http://CRAN.R-project.org/package=matR">http://CRAN.R-project.org/package=matR</a> (1 August 2014, date last accessed)</div>.</div></div></div></div><div content-id="DSV026C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parks</div>
               <div class="given-names">D.H.</div>
            </div>, <div class="name string-name"><div class="surname">Tyson</div> <div class="given-names">G.W.</div></div>, <div class="name string-name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Beiko</div> <div class="given-names">R.G.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">STAMP: statistical analysis of taxonomic and functional profiles</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3123</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAMP%3A%20statistical%20analysis%20of%20taxonomic%20and%20functional%20profiles&amp;author=D.H.%20Parks&amp;author=G.W.%20Tyson&amp;author=P.%20Hugenholtz&amp;author=R.G.%20Beiko&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3123-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu494" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu494" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu494"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25061070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fierer</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Barberan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Laughlin</div> <div class="given-names">D.C.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Seeing the forest for the genes: using metagenomics to infer the aggregated traits of microbial communities</div>, <div class="source">Front. Microbiol.</div> , <div class="volume">5</div>, <div class="fpage">614</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Seeing%20the%20forest%20for%20the%20genes%3A%20using%20metagenomics%20to%20infer%20the%20aggregated%20traits%20of%20microbial%20communities&amp;author=N.%20Fierer&amp;author=A.%20Barberan&amp;author=D.C.%20Laughlin&amp;publication_year=2014&amp;journal=Front.%20Microbiol.&amp;volume=5&amp;pages=614" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2014.00614" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2014.00614" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2014.00614"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25429288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Balzer</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Malde</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jonassen</div> <div class="given-names">I.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Systematic exploration of error sources in pyrosequencing flowgram data</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">i304</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20exploration%20of%20error%20sources%20in%20pyrosequencing%20flowgram%20data&amp;author=S.%20Balzer&amp;author=K.%20Malde&amp;author=I.%20Jonassen&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=i304-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gerdes</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">El Yacoubi</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bailly</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Synergistic use of plant-prokaryote comparative genomics for functional annotations</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div> (<div class="issue">Suppl 1</div>), <div class="fpage">S2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synergistic%20use%20of%20plant-prokaryote%20comparative%20genomics%20for%20functional%20annotations&amp;author=S.%20Gerdes&amp;author=B.%20El%20Yacoubi&amp;author=M.%20Bailly&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=S2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-S1-S2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-S1-S2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-S1-S2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21810204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tasse</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Bercovici</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pizzut-Serin</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Functional metagenomics to mine the human gut microbiome for dietary fiber catabolic enzymes</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">1605</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20metagenomics%20to%20mine%20the%20human%20gut%20microbiome%20for%20dietary%20fiber%20catabolic%20enzymes&amp;author=L.%20Tasse&amp;author=J.%20Bercovici&amp;author=S.%20Pizzut-Serin&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1605-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.108332.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.108332.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.108332.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20841432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Barbi</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Bragalini</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Vallon</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">PCR primers to study the diversity of expressed fungal genes encoding lignocellulolytic enzymes in soils using high-throughput sequencing</div>, <div class="source">PloS ONE</div> , <div class="volume">9</div>, <div class="fpage">e116264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR%20primers%20to%20study%20the%20diversity%20of%20expressed%20fungal%20genes%20encoding%20lignocellulolytic%20enzymes%20in%20soils%20using%20high-throughput%20sequencing&amp;author=F.%20Barbi&amp;author=C.%20Bragalini&amp;author=L.%20Vallon&amp;publication_year=2014&amp;journal=PloS%20ONE&amp;volume=9&amp;pages=e116264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0116264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0116264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0116264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25545363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Janecek</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Svensson</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">MacGregor</div> <div class="given-names">E.A.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Structural and evolutionary aspects of two families of non-catalytic domains present in starch and glycogen binding proteins from microbes, plants and animals</div>, <div class="source">Enzyme Microb. Tech.</div> , <div class="volume">49</div>, <div class="fpage">429</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20evolutionary%20aspects%20of%20two%20families%20of%20non-catalytic%20domains%20present%20in%20starch%20and%20glycogen%20binding%20proteins%20from%20microbes%2C%20plants%20and%20animals&amp;author=S.%20Janecek&amp;author=B.%20Svensson&amp;author=E.A.%20MacGregor&amp;publication_year=2011&amp;journal=Enzyme%20Microb.%20Tech.&amp;volume=49&amp;pages=429-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.enzmictec.2011.07.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.enzmictec.2011.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.enzmictec.2011.07.002"> </span></div> </div></div></div></div></div><div content-id="DSV026C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Identification and characterization of a novel trehalose synthase gene derived from saline-alkali soil metagenomes</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e77437</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20a%20novel%20trehalose%20synthase%20gene%20derived%20from%20saline-alkali%20soil%20metagenomes&amp;author=L.%20Jiang&amp;author=M.%20Lin&amp;author=Y.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e77437" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0077437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0077437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0077437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24146994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ward</div>
               <div class="given-names">N.L.</div>
            </div>, <div class="name string-name"><div class="surname">Challacombe</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Janssen</div> <div class="given-names">P.H.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Three genomes from the phylum Acidobacteria provide insight into the lifestyles of these microorganisms in soils</div>, <div class="source">Appl. Environ. Microb.</div> , <div class="volume">75</div>, <div class="fpage">2046</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20genomes%20from%20the%20phylum%20Acidobacteria%20provide%20insight%20into%20the%20lifestyles%20of%20these%20microorganisms%20in%20soils&amp;author=N.L.%20Ward&amp;author=J.F.%20Challacombe&amp;author=P.H.%20Janssen&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microb.&amp;volume=75&amp;pages=2046-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02294-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02294-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02294-08"> </span></div> </div></div></div></div></div><div content-id="DSV026C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quaiser</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Ochsenreiter</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lanz</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Acidobacteria form a coherent but highly diverse group within the bacterial domain: evidence from environmental genomics</div>, <div class="source">Mol. Microbiol.</div> , <div class="volume">50</div>, <div class="fpage">563</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acidobacteria%20form%20a%20coherent%20but%20highly%20diverse%20group%20within%20the%20bacterial%20domain%3A%20evidence%20from%20environmental%20genomics&amp;author=A.%20Quaiser&amp;author=T.%20Ochsenreiter&amp;author=C.%20Lanz&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=50&amp;pages=563-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03707.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03707.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03707.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14617179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rinke</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Schwientek</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sczyrba</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Insights into the phylogeny and coding potential of microbial dark matter</div>, <div class="source">Nature</div> , <div class="volume">499</div>, <div class="fpage">431</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20phylogeny%20and%20coding%20potential%20of%20microbial%20dark%20matter&amp;author=C.%20Rinke&amp;author=P.%20Schwientek&amp;author=A.%20Sczyrba&amp;publication_year=2013&amp;journal=Nature&amp;volume=499&amp;pages=431-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23851394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">D.Y.</div>
            </div>, <div class="name string-name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mavromatis</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">A phylogeny-driven genomic encyclopaedia of Bacteria and Archaea</div>, <div class="source">Nature</div> , <div class="volume">462</div>, <div class="fpage">1056</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogeny-driven%20genomic%20encyclopaedia%20of%20Bacteria%20and%20Archaea&amp;author=D.Y.%20Wu&amp;author=P.%20Hugenholtz&amp;author=K.%20Mavromatis&amp;publication_year=2009&amp;journal=Nature&amp;volume=462&amp;pages=1056-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08656" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08656" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08656"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20033048" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yergeau</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Hogues</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Whyte</div> <div class="given-names">L.G.</div></div>, <div class="name string-name"><div class="surname">Greer</div> <div class="given-names">C.W.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">The functional potential of high Arctic permafrost revealed by metagenomic sequencing, qPCR and microarray analyses</div>, <div class="source">Isme J.</div> , <div class="volume">4</div>, <div class="fpage">1206</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20potential%20of%20high%20Arctic%20permafrost%20revealed%20by%20metagenomic%20sequencing%2C%20qPCR%20and%20microarray%20analyses&amp;author=E.%20Yergeau&amp;author=H.%20Hogues&amp;author=L.G.%20Whyte&amp;author=C.W.%20Greer&amp;publication_year=2010&amp;journal=Isme%20J.&amp;volume=4&amp;pages=1206-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2010.41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2010.41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2010.41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20393573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">DeAngelis</div>
               <div class="given-names">K.M.</div>
            </div>, <div class="name string-name"><div class="surname">Gladden</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Allgaier</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Strategies for enhancing the effectiveness of metagenomic-based enzyme discovery in lignocellulolytic microbial communities</div>, <div class="source">Bioenerg. Res.</div> , <div class="volume">3</div>, <div class="fpage">146</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strategies%20for%20enhancing%20the%20effectiveness%20of%20metagenomic-based%20enzyme%20discovery%20in%20lignocellulolytic%20microbial%20communities&amp;author=K.M.%20DeAngelis&amp;author=J.M.%20Gladden&amp;author=M.%20Allgaier&amp;publication_year=2010&amp;journal=Bioenerg.%20Res.&amp;volume=3&amp;pages=146-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12155-010-9089-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12155-010-9089-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12155-010-9089-z"> </span></div> </div></div></div></div></div><div content-id="DSV026C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yergeau</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Sanschagrin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Beaumier</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Greer</div> <div class="given-names">C.W.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Metagenomic analysis of the bioremediation of diesel-contaminated Canadian high arctic soils</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e30058</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20of%20the%20bioremediation%20of%20diesel-contaminated%20Canadian%20high%20arctic%20soils&amp;author=E.%20Yergeau&amp;author=S.%20Sanschagrin&amp;author=D.%20Beaumier&amp;author=C.W.%20Greer&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e30058" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schnoes</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Ream</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Thorman</div> <div class="given-names">A.W.</div></div>, <div class="name string-name"><div class="surname">Babbitt</div> <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Friedberg</div> <div class="given-names">I.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Biases in the experimental annotations of protein function and their effect on our understanding of protein function space</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">9</div>, <div class="fpage">e1003063</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biases%20in%20the%20experimental%20annotations%20of%20protein%20function%20and%20their%20effect%20on%20our%20understanding%20of%20protein%20function%20space&amp;author=A.M.%20Schnoes&amp;author=D.C.%20Ream&amp;author=A.W.%20Thorman&amp;author=P.C.%20Babbitt&amp;author=I.%20Friedberg&amp;publication_year=2013&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=9&amp;pages=e1003063" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23737737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV026C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Howe</div>
               <div class="given-names">A.C.</div>
            </div>, <div class="name string-name"><div class="surname">Jansson</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Malfatti</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Tringe</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Tiedje</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">C.T.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Tackling soil diversity with the assembly of large, complex metagenomes (vol 111, pg 4904, 2014)</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">6115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tackling%20soil%20diversity%20with%20the%20assembly%20of%20large%2C%20complex%20metagenomes%20(vol%20111%2C%20pg%204904%2C%202014)&amp;author=A.C.%20Howe&amp;author=J.K.%20Jansson&amp;author=S.A.%20Malfatti&amp;author=S.G.%20Tringe&amp;author=J.M.%20Tiedje&amp;author=C.T.%20Brown&amp;publication_year=2014&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=6115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1402564111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1402564111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1402564111"> </span></div> </div></div></div></div></div><div content-id="DSV026C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV026C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">He</div>
               <div class="given-names">Z.L.</div>
            </div>, <div class="name string-name"><div class="surname">Gentry</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Schadt</div> <div class="given-names">C.W.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">GeoChip: a comprehensive microarray for investigating biogeochemical, ecological and environmental processes</div>, <div class="source">Isme J.</div> , <div class="volume">1</div>, <div class="fpage">67</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeoChip%3A%20a%20comprehensive%20microarray%20for%20investigating%20biogeochemical%2C%20ecological%20and%20environmental%20processes&amp;author=Z.L.%20He&amp;author=T.J.%20Gentry&amp;author=C.W.%20Schadt&amp;publication_year=2007&amp;journal=Isme%20J.&amp;volume=1&amp;pages=67-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2007.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2007.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2007.2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18043615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462493&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv026/1/dsv026_Supplementary_Data.zip?Expires=2147483647&amp;Signature=Pvuq~TrgEq5yF2n3pjsbFzJo99WT0M0biPxy52MuupPYCFeXqG7BT7M8omOoCBrMOj1l8s0jvdjW-BlSShwrnLtYVsCknEDAHHZkchdyytoUmAqS~jfIhos9Tdwx5abyGBZ0Ieywlx54qI-vaALvhrNK3e8Pgl-87hhfKr68GppNtf193FUlNjjKxek3Oeyq28hsj8XwIjw83gAK4KjlwzKWu99-yCEiVf4yLWkT3m2Us4PX4AvPCWi-dHUAvvXyDmNA3gyo8YbG9~tGvunGE5TKO8~4w8aoxbxTzlKlzUKSfL1LOous~yxR7uyqC18gNHtbh7XthcMMZrBp6hxh9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">633</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv026"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462493"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f451%2f2462493">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F451%2F2462493" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462493'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Captured metagenomics: large-scale targeting of genes based on ‘sequence capture’ reveals functional diversity in soils" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv026");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f451%2f2462493">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

