

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/535/5055375","@type":"ScholarlyArticle","name":"Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome","datePublished":"2018-07-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy023","keywords":["target enrichment","sequence capture","SNPs","Handroanthus"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/535/5055375","author":[{"name":"Silva-Junior, Orzenil Bonfim","affiliation":"EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil  Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil","@type":"Person","sameAs":"http://orcid.org/0000-0002-2104-2010"},{"name":"Grattapaglia, Dario","affiliation":"EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil  Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil","@type":"Person"},{"name":"Novaes, Evandro","affiliation":"Departamento de Biologia, Universidade Federal de Lavras, Lavras, MG, Brazil","@type":"Person"},{"name":"Collevatti, Rosane G","affiliation":"Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas, Universidade Federal de Goiás, Goiânia, GO, Brazil","@type":"Person","sameAs":"http://orcid.org/0000-0002-3733-7059"}],"description":"Abstract.  Targeted sequence capture coupled to high-throughput sequencing has become a powerful method for the study of genome-wide sequence variation. Followi","pageStart":"535","pageEnd":"545","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome" />
    <meta property="og:description" content="Abstract.  Targeted sequence capture coupled to high-throughput sequencing has become a powerful method for the study of genome-wide sequence variation. Followi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/535/5055375" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-07-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Silva-Junior, Orzenil Bonfim" /><meta name="citation_author_institution" content="EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil" /><meta name="citation_author_institution" content="Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil" /><meta name="citation_author" content="Grattapaglia, Dario" /><meta name="citation_author_institution" content="EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil" /><meta name="citation_author_institution" content="Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil" /><meta name="citation_author" content="Novaes, Evandro" /><meta name="citation_author_institution" content="Departamento de Biologia, Universidade Federal de Lavras, Lavras, MG, Brazil" /><meta name="citation_author" content="Collevatti, Rosane G" /><meta name="citation_author_institution" content="Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas, Universidade Federal de Goiás, Goiânia, GO, Brazil" /><meta name="citation_title" content="Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome" /><meta name="citation_firstpage" content="535" /><meta name="citation_lastpage" content="545" /><meta name="citation_doi" content="10.1093/dnares/dsy023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Adaptation genomics: the next generation;  citation_journal_title=Trends Ecol. Evol;  citation_year=2010;  citation_volume=25;  citation_pages=705-12; " /><meta name="citation_reference" content="citation_title=Population genetic inference from genomic sequence variation;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=291-300; " /><meta name="citation_reference" content="citation_title=SNPs in ecology, evolution and conservation;  citation_journal_title=Trends Ecol. Evol;  citation_year=2004;  citation_volume=19;  citation_pages=208-16; " /><meta name="citation_reference" content="citation_title=Advances and limits of using population genetics to understand local adaptation;  citation_journal_title=Trends Ecol. Evol;  citation_year=2014;  citation_volume=29;  citation_pages=673-80; " /><meta name="citation_reference" content="citation_title=A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=409;  citation_pages=928-33; " /><meta name="citation_reference" content="citation_title=The pattern of polymorphism in Arabidopsis thaliana;  citation_journal_title=PLoS Biol;  citation_year=2005;  citation_volume=3;  citation_pages=e196-1299; " /><meta name="citation_reference" content="citation_title=Development of genome-wide DNA polymorphism database for map-based cloning of rice genes;  citation_journal_title=Plant Physiol;  citation_year=2004;  citation_volume=135;  citation_pages=1198-205; " /><meta name="citation_reference" content="citation_title=Crop genomics: advances and applications;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=13;  citation_pages=85-96; " /><meta name="citation_reference" content="citation_title=More genomic resources for less-studied crops;  citation_journal_title=Trends Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=452-60; " /><meta name="citation_reference" content="citation_title=Forest tree genomics: growing resources and applications;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=111-22; " /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=Target-enrichment strategies for next-generation sequencing;  citation_journal_title=Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=111-8; " /><meta name="citation_reference" content="citation_title=Sequence capture by hybridization to explore modern and ancient genomic diversity in model and nonmodel organisms;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=4504-18; " /><meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing;  citation_journal_title=Nat. Biotechnol;  citation_year=2009;  citation_volume=27;  citation_pages=182-9; " /><meta name="citation_reference" content="citation_title=Detection of inherited mutations for breast and ovarian cancer using genomic capture and massively parallel sequencing;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2010;  citation_volume=107;  citation_pages=12629-33; " /><meta name="citation_reference" content="citation_title=Repeat subtraction-mediated sequence capture from a complex genome;  citation_journal_title=Plant J;  citation_year=2010;  citation_volume=62;  citation_pages=898-909; " /><meta name="citation_reference" content="citation_title=Development of genetic markers in eucalyptus species by target enrichment and exome sequencing;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0116528" /><meta name="citation_reference" content="citation_title=Targeted enrichment of the black cottonwood (Populus trichocarpa) gene space using sequence capture;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=703" /><meta name="citation_reference" content="citation_title=Whole-exome targeted sequencing of the uncharacterized pine genome;  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=75;  citation_pages=146-56; " /><meta name="citation_reference" content="citation_title=Targeted capture in evolutionary and ecological genomics;  citation_journal_title=Mol. Ecol;  citation_year=2016;  citation_volume=25;  citation_pages=185-202; " /><meta name="citation_reference" content="citation_title=Using targeted enrichment of nuclear genes to increase phylogenetic resolution in the Neotropical rain forest genus Inga (Leguminosae: Mimosoideae);  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=710" /><meta name="citation_reference" content="citation_title=Patterns of species distributions in the dry seasonal forests of South America;  citation_journal_title=Ann. Missouri Bot. Gard;  citation_year=1993;  citation_volume=80;  citation_pages=902-27; " /><meta name="citation_reference" content="citation_title=A coupled phylogeographical and species distribution modelling approach recovers the demographical history of a Neotropical seasonally dry forest tree species;  citation_journal_title=Mol. Ecol;  citation_year=2012;  citation_volume=21;  citation_pages=5845-63; " /><meta name="citation_reference" content="citation_title=Inferring phylogeny despite incomplete lineage sorting;  citation_journal_title=Syst. Biol;  citation_year=2006;  citation_volume=55;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=Evaluating ipe (Tabebuia, Bignoniaceae) logging in Amazonia: sustainable management or catalyst for forest degradation?;  citation_journal_title=Biol. Conserv;  citation_year=2008;  citation_volume=141;  citation_pages=2071-85; " /><meta name="citation_reference" content="citation_title=Genomics and the challenging translation into conservation practice;  citation_journal_title=Trends Ecol. Evol;  citation_year=2015;  citation_volume=30;  citation_pages=78-87; " /><meta name="citation_reference" content="citation_title=Genome assembly of the Pink Ipê (Handroanthus impetiginosus, Bignoniaceae), a highly-valued ecologically keystone Neotropical timber forest tree and, a natural product producer;  citation_journal_title=Gigascience;  citation_year=2018;  citation_volume=7;  citation_pages=16" /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=De novo assembly and genotyping of variants using colored de Bruijn graphs;  citation_journal_title=Nat. Genet;  citation_year=2012;  citation_volume=44;  citation_pages=226-32; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=High-resolution linkage map and chromosome-scale genome assembly for Cassava (Manihot esculenta Crantz) from 10 populations; citation_author=ICGMC;  citation_journal_title=G3 (Bethesda);  citation_year=2015;  citation_volume=5;  citation_pages=133" /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; " /><meta name="citation_reference" content="citation_title=RIG: recalibration and Interrelation of Genomic Sequence Data with the GATK;  citation_journal_title=G3 (Bethesda);  citation_year=2015;  citation_volume=5;  citation_pages=655-65; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff;  citation_journal_title=Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets;  citation_journal_title=Gigascience;  citation_year=2015;  citation_volume=4;  citation_pages=16" /><meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination;  citation_journal_title=Theor. Popul. Biol;  citation_year=1975;  citation_volume=7;  citation_pages=256-76; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GbS) approach for high diversity species;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Improving fruit and wine: what does genomics have to offer?;  citation_journal_title=Trends Genet;  citation_year=2013;  citation_volume=29;  citation_pages=190-6; " /><meta name="citation_reference" content="citation_title=Breaking RAD: an evaluation of the utility of restriction site‐associated DNA sequencing for genome scans of adaptation;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2016;  citation_volume=17;  citation_pages=142-52; " /><meta name="citation_reference" content="citation_title=The effect of RAD allele dropout on the estimation of genetic variation within and between populations;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=3165-78; " /><meta name="citation_reference" content="citation_title=RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=3179-90; " /><meta name="citation_reference" content="citation_title=Sequence capture versus restriction site associated DNA sequencing for shallow systematics;  citation_journal_title=Syst. Biol;  citation_year=2016;  citation_volume=65;  citation_pages=910-24; " /><meta name="citation_reference" content="citation_title=RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2016;  citation_volume=16;  citation_pages=1264-78; " /><meta name="citation_reference" content="citation_title=An evaluation of transcriptome-based exon capture for frog phylogenomics across multiple scales of divergence (Class: Amphibia, Order: Anura);  citation_journal_title=Mol. Ecol. Resour;  citation_year=2016;  citation_volume=16;  citation_pages=1069-83; " /><meta name="citation_reference" content="citation_title=Sequencing depth and coverage: key considerations in genomic analyses;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=121-32; " /><meta name="citation_reference" content="citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=936-9; " /><meta name="citation_reference" content="citation_title=Simultaneous alignment of short reads against multiple genomes;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R98" /><meta name="citation_reference" content="citation_title=SplitMEM: a graphical algorithm for pan-genome analysis with suffix skips;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3476-83; " /><meta name="citation_reference" content="citation_title=Short read alignment with populations of genomes;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=i361-70; " /><meta name="citation_reference" content="citation_title=A representation of a compressed de Bruijn graph for pan-genome analysis that enables search;  citation_journal_title=Algorithms Mol. Biol;  citation_year=2016;  citation_volume=11;  citation_pages=20" /><meta name="citation_reference" content="citation_title=Systematic comparison of variant calling pipelines using gold standard personal exome variants;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=5;  citation_pages=17875" /><meta name="citation_reference" content="citation_title=The next generation of target capture technologies - large DNA fragment enrichment and sequencing determines regional genomic variation of high complexity;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=486" /><meta name="citation_reference" content="citation_title=A robust protocol to increase NimbleGen SeqCap EZ multiplexing capacity to 96 samples;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0123872" /><meta name="citation_reference" content="citation_title=Targeted capture sequencing in whitebark pine reveals range-wide demographic and adaptive patterns despite challenges of a large, repetitive genome;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7;  citation_pages=484" /><meta name="citation_reference" content="citation_title=Genomic distribution and estimation of nucleotide diversity in natural populations: perspectives from the collared flycatcher (Ficedula albicollis) genome;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2017;  citation_volume=17;  citation_pages=586-97; " /><meta name="citation_reference" content="citation_title=Natural selection and recombination rate variation shape nucleotide polymorphism across the genomes of three related populus species;  citation_journal_title=Genetics;  citation_year=2016;  citation_volume=202;  citation_pages=1185-200; " /><meta name="citation_reference" content="citation_title=Coalescent simulation and paleodistribution modeling for Tabebuia rosealba do not support South American dry forest refugia hypothesis;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0159314" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/535/26076910/dsy023.pdf" /><meta name="description" content="Abstract.  Targeted sequence capture coupled to high-throughput sequencing has become a powerful method for the study of genome-wide sequence variation. Followi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/535/5055375" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/535/5055375" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/535/5055375");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy023';
    </script>
<title>
	Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5055375" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZeGATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2lsdmEtSnVuaW9yLCBPcnplbmlsIEJvbmZpbSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVNQlJBUEEgUmVjdXJzb3MgR2Vuw6l0aWNvcyBlIEJpb3RlY25vbG9naWEsIEVQcUIsIEJyYXPDrWxpYSwgREYsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlByb2dyYW1hIGRlIENpw6puY2lhcyBHZW7DtG1pY2FzIGUgQmlvdGVjbm9sb2dpYSwgVW5pdmVyc2lkYWRlIENhdMOzbGljYSBkZSBCcmFzw61saWEsIFNHQU4gOTE2IE1vZHVsbyBCLCBCcmFzaWxpYSwgREYsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdyYXR0YXBhZ2xpYSwgRGFyaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFTUJSQVBBIFJlY3Vyc29zIEdlbsOpdGljb3MgZSBCaW90ZWNub2xvZ2lhLCBFUHFCLCBCcmFzw61saWEsIERGLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQcm9ncmFtYSBkZSBDacOqbmNpYXMgR2Vuw7RtaWNhcyBlIEJpb3RlY25vbG9naWEsIFVuaXZlcnNpZGFkZSBDYXTDs2xpY2EgZGUgQnJhc8OtbGlhLCBTR0FOIDkxNiBNb2R1bG8gQiwgQnJhc2lsaWEsIERGLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb3ZhZXMsIEV2YW5kcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRhbWVudG8gZGUgQmlvbG9naWEsIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRlIExhdnJhcywgTGF2cmFzLCBNRywgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ29sbGV2YXR0aSwgUm9zYW5lIEciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0w7NyaW8gZGUgR2Vuw6l0aWNhICZhbXA7IEJpb2RpdmVyc2lkYWRlLCBJbnN0aXR1dG8gZGUgQ2nDqm5jaWFzIEJpb2zDs2dpY2FzLCBVbml2ZXJzaWRhZGUgRmVkZXJhbCBkZSBHb2nDoXMsIEdvacOibmlhLCBHTywgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXNpZ24gYW5kIGV2YWx1YXRpb24gb2YgYSBzZXF1ZW5jZSBjYXB0dXJlIHN5c3RlbSBmb3IgZ2Vub21lLXdpZGUgU05QIGdlbm90eXBpbmcgaW4gaGlnaGx5IGhldGVyb3p5Z291cyBwbGFudCBnZW5vbWVzOiBhIGNhc2Ugc3R1ZHkgd2l0aCBhIGtleXN0b25lIE5lb3Ryb3BpY2FsIGhhcmR3b29kIHRyZWUgZ2Vub21lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1NDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0YXRpb24gZ2Vub21pY3M6IHRoZSBuZXh0IGdlbmVyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBFY29sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NzA1LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGdlbmV0aWMgaW5mZXJlbmNlIGZyb20gZ2Vub21pYyBzZXF1ZW5jZSB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yOTEtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlBzIGluIGVjb2xvZ3ksIGV2b2x1dGlvbiBhbmQgY29uc2VydmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgRWNvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTIwOC0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgYW5kIGxpbWl0cyBvZiB1c2luZyBwb3B1bGF0aW9uIGdlbmV0aWNzIHRvIHVuZGVyc3RhbmQgbG9jYWwgYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEVjb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWFwIG9mIGh1bWFuIGdlbm9tZSBzZXF1ZW5jZSB2YXJpYXRpb24gY29udGFpbmluZyAxLjQyIG1pbGxpb24gc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTkyOC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBhdHRlcm4gb2YgcG9seW1vcnBoaXNtIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUxOTYtMTI5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgZ2Vub21lLXdpZGUgRE5BIHBvbHltb3JwaGlzbSBkYXRhYmFzZSBmb3IgbWFwLWJhc2VkIGNsb25pbmcgb2YgcmljZSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzU7ICBjaXRhdGlvbl9wYWdlcz0xMTk4LTIwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3JvcCBnZW5vbWljczogYWR2YW5jZXMgYW5kIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9ODUtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vcmUgZ2Vub21pYyByZXNvdXJjZXMgZm9yIGxlc3Mtc3R1ZGllZCBjcm9wczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz00NTItNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvcmVzdCB0cmVlIGdlbm9taWNzOiBncm93aW5nIHJlc291cmNlcyBhbmQgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGdlbmV0aWMgbWFya2VyIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDk5LTUxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0LWVucmljaG1lbnQgc3RyYXRlZ2llcyBmb3IgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTExLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGNhcHR1cmUgYnkgaHlicmlkaXphdGlvbiB0byBleHBsb3JlIG1vZGVybiBhbmQgYW5jaWVudCBnZW5vbWljIGRpdmVyc2l0eSBpbiBtb2RlbCBhbmQgbm9ubW9kZWwgb3JnYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTQ1MDQtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHV0aW9uIGh5YnJpZCBzZWxlY3Rpb24gd2l0aCB1bHRyYS1sb25nIG9saWdvbnVjbGVvdGlkZXMgZm9yIG1hc3NpdmVseSBwYXJhbGxlbCB0YXJnZXRlZCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xODItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIGluaGVyaXRlZCBtdXRhdGlvbnMgZm9yIGJyZWFzdCBhbmQgb3ZhcmlhbiBjYW5jZXIgdXNpbmcgZ2Vub21pYyBjYXB0dXJlIGFuZCBtYXNzaXZlbHkgcGFyYWxsZWwgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLlMuQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xMjYyOS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwZWF0IHN1YnRyYWN0aW9uLW1lZGlhdGVkIHNlcXVlbmNlIGNhcHR1cmUgZnJvbSBhIGNvbXBsZXggZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9ODk4LTkwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgZ2VuZXRpYyBtYXJrZXJzIGluIGV1Y2FseXB0dXMgc3BlY2llcyBieSB0YXJnZXQgZW5yaWNobWVudCBhbmQgZXhvbWUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDExNjUyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIGVucmljaG1lbnQgb2YgdGhlIGJsYWNrIGNvdHRvbndvb2QgKFBvcHVsdXMgdHJpY2hvY2FycGEpIGdlbmUgc3BhY2UgdXNpbmcgc2VxdWVuY2UgY2FwdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NzAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZXhvbWUgdGFyZ2V0ZWQgc2VxdWVuY2luZyBvZiB0aGUgdW5jaGFyYWN0ZXJpemVkIHBpbmUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9MTQ2LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBjYXB0dXJlIGluIGV2b2x1dGlvbmFyeSBhbmQgZWNvbG9naWNhbCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTg1LTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgdGFyZ2V0ZWQgZW5yaWNobWVudCBvZiBudWNsZWFyIGdlbmVzIHRvIGluY3JlYXNlIHBoeWxvZ2VuZXRpYyByZXNvbHV0aW9uIGluIHRoZSBOZW90cm9waWNhbCByYWluIGZvcmVzdCBnZW51cyBJbmdhIChMZWd1bWlub3NhZTogTWltb3NvaWRlYWUpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBvZiBzcGVjaWVzIGRpc3RyaWJ1dGlvbnMgaW4gdGhlIGRyeSBzZWFzb25hbCBmb3Jlc3RzIG9mIFNvdXRoIEFtZXJpY2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gTWlzc291cmkgQm90LiBHYXJkOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9OTAyLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvdXBsZWQgcGh5bG9nZW9ncmFwaGljYWwgYW5kIHNwZWNpZXMgZGlzdHJpYnV0aW9uIG1vZGVsbGluZyBhcHByb2FjaCByZWNvdmVycyB0aGUgZGVtb2dyYXBoaWNhbCBoaXN0b3J5IG9mIGEgTmVvdHJvcGljYWwgc2Vhc29uYWxseSBkcnkgZm9yZXN0IHRyZWUgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTg0NS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJyaW5nIHBoeWxvZ2VueSBkZXNwaXRlIGluY29tcGxldGUgbGluZWFnZSBzb3J0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpbmcgaXBlIChUYWJlYnVpYSwgQmlnbm9uaWFjZWFlKSBsb2dnaW5nIGluIEFtYXpvbmlhOiBzdXN0YWluYWJsZSBtYW5hZ2VtZW50IG9yIGNhdGFseXN0IGZvciBmb3Jlc3QgZGVncmFkYXRpb24/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBDb25zZXJ2OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzEtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIGFuZCB0aGUgY2hhbGxlbmdpbmcgdHJhbnNsYXRpb24gaW50byBjb25zZXJ2YXRpb24gcHJhY3RpY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBFY29sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NzgtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBhc3NlbWJseSBvZiB0aGUgUGluayBJcMOqIChIYW5kcm9hbnRodXMgaW1wZXRpZ2lub3N1cywgQmlnbm9uaWFjZWFlKSwgYSBoaWdobHktdmFsdWVkIGVjb2xvZ2ljYWxseSBrZXlzdG9uZSBOZW90cm9waWNhbCB0aW1iZXIgZm9yZXN0IHRyZWUgYW5kLCBhIG5hdHVyYWwgcHJvZHVjdCBwcm9kdWNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2lnYXNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTg7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmcmFtZXdvcmsgZm9yIHZhcmlhdGlvbiBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz00OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBhbmQgZ2Vub3R5cGluZyBvZiB2YXJpYW50cyB1c2luZyBjb2xvcmVkIGRlIEJydWlqbiBncmFwaHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0yMjYtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gbGlua2FnZSBtYXAgYW5kIGNocm9tb3NvbWUtc2NhbGUgZ2Vub21lIGFzc2VtYmx5IGZvciBDYXNzYXZhIChNYW5paG90IGVzY3VsZW50YSBDcmFudHopIGZyb20gMTAgcG9wdWxhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1JQ0dNQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgKEJldGhlc2RhKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkVEVG9vbHM6IGEgZmxleGlibGUgc3VpdGUgb2YgdXRpbGl0aWVzIGZvciBjb21wYXJpbmcgZ2Vub21pYyBmZWF0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz04NDEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UklHOiByZWNhbGlicmF0aW9uIGFuZCBJbnRlcnJlbGF0aW9uIG9mIEdlbm9taWMgU2VxdWVuY2UgRGF0YSB3aXRoIHRoZSBHQVRLOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02NTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvZ3JhbSBmb3IgYW5ub3RhdGluZyBhbmQgcHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zLCBTbnBFZmY7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZseTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODAtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlY29uZC1nZW5lcmF0aW9uIFBMSU5LOiByaXNpbmcgdG8gdGhlIGNoYWxsZW5nZSBvZiBsYXJnZXIgYW5kIHJpY2hlciBkYXRhc2V0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2lnYXNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIG51bWJlciBvZiBzZWdyZWdhdGluZyBzaXRlcyBpbiBnZW5ldGljYWwgbW9kZWxzIHdpdGhvdXQgcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIFBvcHVsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNTYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9idXN0LCBzaW1wbGUgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIChHYlMpIGFwcHJvYWNoIGZvciBoaWdoIGRpdmVyc2l0eSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE5Mzc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92aW5nIGZydWl0IGFuZCB3aW5lOiB3aGF0IGRvZXMgZ2Vub21pY3MgaGF2ZSB0byBvZmZlcj87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTE5MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmVha2luZyBSQUQ6IGFuIGV2YWx1YXRpb24gb2YgdGhlIHV0aWxpdHkgb2YgcmVzdHJpY3Rpb24gc2l0ZeKAkGFzc29jaWF0ZWQgRE5BIHNlcXVlbmNpbmcgZm9yIGdlbm9tZSBzY2FucyBvZiBhZGFwdGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE0Mi01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiBSQUQgYWxsZWxlIGRyb3BvdXQgb24gdGhlIGVzdGltYXRpb24gb2YgZ2VuZXRpYyB2YXJpYXRpb24gd2l0aGluIGFuZCBiZXR3ZWVuIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zMTY1LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SQURzZXEgdW5kZXJlc3RpbWF0ZXMgZGl2ZXJzaXR5IGFuZCBpbnRyb2R1Y2VzIGdlbmVhbG9naWNhbCBiaWFzZXMgZHVlIHRvIG5vbnJhbmRvbSBoYXBsb3R5cGUgc2FtcGxpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTMxNzktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGNhcHR1cmUgdmVyc3VzIHJlc3RyaWN0aW9uIHNpdGUgYXNzb2NpYXRlZCBETkEgc2VxdWVuY2luZyBmb3Igc2hhbGxvdyBzeXN0ZW1hdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTkxMC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFEY2FwOiBzZXF1ZW5jZSBjYXB0dXJlIG9mIGR1YWwtZGlnZXN0IFJBRHNlcSBsaWJyYXJpZXMgd2l0aCBpZGVudGlmaWFibGUgZHVwbGljYXRlcyBhbmQgcmVkdWNlZCBtaXNzaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTI2NC03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZhbHVhdGlvbiBvZiB0cmFuc2NyaXB0b21lLWJhc2VkIGV4b24gY2FwdHVyZSBmb3IgZnJvZyBwaHlsb2dlbm9taWNzIGFjcm9zcyBtdWx0aXBsZSBzY2FsZXMgb2YgZGl2ZXJnZW5jZSAoQ2xhc3M6IEFtcGhpYmlhLCBPcmRlcjogQW51cmEpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEwNjktODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgZGVwdGggYW5kIGNvdmVyYWdlOiBrZXkgY29uc2lkZXJhdGlvbnMgaW4gZ2Vub21pYyBhbmFseXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFtcHk6IGEgc3RhdGlzdGljYWwgYWxnb3JpdGhtIGZvciBzZW5zaXRpdmUgYW5kIGZhc3QgbWFwcGluZyBvZiBJbGx1bWluYSBzZXF1ZW5jZSByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTkzNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW11bHRhbmVvdXMgYWxpZ25tZW50IG9mIHNob3J0IHJlYWRzIGFnYWluc3QgbXVsdGlwbGUgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SOTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpdE1FTTogYSBncmFwaGljYWwgYWxnb3JpdGhtIGZvciBwYW4tZ2Vub21lIGFuYWx5c2lzIHdpdGggc3VmZml4IHNraXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTM0NzYtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggcG9wdWxhdGlvbnMgb2YgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz1pMzYxLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJlcHJlc2VudGF0aW9uIG9mIGEgY29tcHJlc3NlZCBkZSBCcnVpam4gZ3JhcGggZm9yIHBhbi1nZW5vbWUgYW5hbHlzaXMgdGhhdCBlbmFibGVzIHNlYXJjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxnb3JpdGhtcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgY29tcGFyaXNvbiBvZiB2YXJpYW50IGNhbGxpbmcgcGlwZWxpbmVzIHVzaW5nIGdvbGQgc3RhbmRhcmQgcGVyc29uYWwgZXhvbWUgdmFyaWFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNzg3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZXh0IGdlbmVyYXRpb24gb2YgdGFyZ2V0IGNhcHR1cmUgdGVjaG5vbG9naWVzIC0gbGFyZ2UgRE5BIGZyYWdtZW50IGVucmljaG1lbnQgYW5kIHNlcXVlbmNpbmcgZGV0ZXJtaW5lcyByZWdpb25hbCBnZW5vbWljIHZhcmlhdGlvbiBvZiBoaWdoIGNvbXBsZXhpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQ4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9idXN0IHByb3RvY29sIHRvIGluY3JlYXNlIE5pbWJsZUdlbiBTZXFDYXAgRVogbXVsdGlwbGV4aW5nIGNhcGFjaXR5IHRvIDk2IHNhbXBsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMjM4NzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBjYXB0dXJlIHNlcXVlbmNpbmcgaW4gd2hpdGViYXJrIHBpbmUgcmV2ZWFscyByYW5nZS13aWRlIGRlbW9ncmFwaGljIGFuZCBhZGFwdGl2ZSBwYXR0ZXJucyBkZXNwaXRlIGNoYWxsZW5nZXMgb2YgYSBsYXJnZSwgcmVwZXRpdGl2ZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQ4NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZGlzdHJpYnV0aW9uIGFuZCBlc3RpbWF0aW9uIG9mIG51Y2xlb3RpZGUgZGl2ZXJzaXR5IGluIG5hdHVyYWwgcG9wdWxhdGlvbnM6IHBlcnNwZWN0aXZlcyBmcm9tIHRoZSBjb2xsYXJlZCBmbHljYXRjaGVyIChGaWNlZHVsYSBhbGJpY29sbGlzKSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NTg2LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIHNlbGVjdGlvbiBhbmQgcmVjb21iaW5hdGlvbiByYXRlIHZhcmlhdGlvbiBzaGFwZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBhY3Jvc3MgdGhlIGdlbm9tZXMgb2YgdGhyZWUgcmVsYXRlZCBwb3B1bHVzIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIwMjsgIGNpdGF0aW9uX3BhZ2VzPTExODUtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2FsZXNjZW50IHNpbXVsYXRpb24gYW5kIHBhbGVvZGlzdHJpYnV0aW9uIG1vZGVsaW5nIGZvciBUYWJlYnVpYSByb3NlYWxiYSBkbyBub3Qgc3VwcG9ydCBTb3V0aCBBbWVyaWNhbiBkcnkgZm9yZXN0IHJlZnVnaWEgaHlwb3RoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE1OTMxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS81LzUzNS8yNjA3NjkxMC9kc3kwMjMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRhcmdldGVkIHNlcXVlbmNlIGNhcHR1cmUgY291cGxlZCB0byBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBoYXMgYmVjb21lIGEgcG93ZXJmdWwgbWV0aG9kIGZvciB0aGUgc3R1ZHkgb2YgZ2Vub21lLXdpZGUgc2VxdWVuY2UgdmFyaWF0aW9uLiBGb2xsb3dpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01MDU1Mzc1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZOoDRpT0Rqk2Xq/nJ1MvntkNNrKj" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f535%2f5055375" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f535%2f5055375" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122670738">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670738">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670740">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670740">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670745">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670745">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670760">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670760">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670778">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670778">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670800">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670800">5. Data and scripts availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670802">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670802">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122670808">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122670808">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/521/5056130">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/5/547/5059530">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Orzenil Bonfim Silva-Junior</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil</div><div class="aff">Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy023-cor1" data-open="dsy023-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +55 61 3448 4792. Fax. +55 61 3340 3624. Email: <a href="mailto:orzenil.silva@embrapa.br" target="_blank">orzenil.silva@embrapa.br</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-2104-2010">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0002-2104-2010
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Orzenil+Bonfim+Silva-Junior">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Silva-Junior O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Silva-Junior O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Orzenil Bonfim Silva-Junior</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dario Grattapaglia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">EMBRAPA Recursos Genéticos e Biotecnologia, EPqB, Brasília, DF, Brazil</div><div class="aff">Programa de Ciências Genômicas e Biotecnologia, Universidade Católica de Brasília, SGAN 916 Modulo B, Brasilia, DF, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dario+Grattapaglia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Grattapaglia D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Grattapaglia D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dario Grattapaglia</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Evandro Novaes</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Biologia, Universidade Federal de Lavras, Lavras, MG, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Evandro+Novaes">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Novaes E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Novaes E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Evandro Novaes</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rosane G Collevatti</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas, Universidade Federal de Goiás, Goiânia, GO, Brazil</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-3733-7059">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0002-3733-7059
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rosane+G+Collevatti">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Collevatti R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Collevatti R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rosane G Collevatti</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 535–545, <a href='https://doi.org/10.1093/dnares/dsy023'>https://doi.org/10.1093/dnares/dsy023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 July 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 December 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 June 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5055375" href="/dnaresearch/article-pdf/25/5/535/26076910/dsy023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Orzenil Bonfim Silva-Junior, Dario Grattapaglia, Evandro Novaes, Rosane G Collevatti;  Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome, <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 535–545, <a href='https://doi.org/10.1093/dnares/dsy023'>https://doi.org/10.1093/dnares/dsy023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5055375?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Design%20and%20evaluation%20of%20a%20sequence%20capture%20system%20for%20genome-wide%20SNP%20genotyping%20in%20highly%20heterozygous%20plant%20genomes%3A%20a%20case%20study%20with%20a%20keystone%20Neotropical%20hardwood%20tree%20genome&amp;publicationDate=2018-07-17&amp;volumeNum=25&amp;issueNum=5&amp;author=Silva-Junior%2C%20Orzenil%20Bonfim%3B%20Grattapaglia%2C%20Dario&amp;startPage=535&amp;endPage=545&amp;contentId=10.1093%2Fdnares%2Fdsy023&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f535%2f5055375" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122670738 id="122670738" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Targeted sequence capture coupled to high-throughput sequencing has become a powerful method for the study of genome-wide sequence variation. Following our recent development of a genome assembly for the Pink Ipê tree (<em>Handroanthus impetiginosus</em>), a widely distributed Neotropical timber species, we now report the development of a set of 24,751 capture probes for single-nucleotide polymorphisms (SNPs) characterization and genotyping across 18,216 distinct loci, sampling more than 10 Mbp of the species genome. This system identifies nearly 200,000 SNPs located inside or in close proximity to almost 14,000 annotated protein-coding genes, generating quality genotypic data in populations spanning wide geographic distances across the species native range. To provide recommendations for future developments of similar systems for highly heterozygous plant genomes we investigated issues such as probe design, sequencing coverage and bioinformatics, including the evaluation of the capture efficiency and a reassessment of the technical reproducibility of the assay for SNPs recall and genotyping precision. Our results highlight the value of a detailed probe screening on a preliminary genome assembly to produce reliable data for downstream genetic studies. This work should inspire and assist the development of similar genomic resources for other orphan crops and forest trees with highly heterozygous genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;target enrichment&quot;">target enrichment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence capture&quot;">sequence capture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNPs">SNPs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Handroanthus">Handroanthus</a></div>                            <h2 scrollto-destination=122670740 id="122670740" class="section-title" >1. Introduction</h2>
<p>The remarkable advances in whole-genome sequencing (WGS) and genome-wide genotyping technologies have provided unprecedented opportunities to gain detailed insights into the overall patterns of genetic variation underpinning the basis of adaptive evolution.<a reveal-id="dsy023-B1 dsy023-B2 dsy023-B3 dsy023-B4" data-open="dsy023-B1 dsy023-B2 dsy023-B3 dsy023-B4" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Such advances took place first in species for which reference genome sequences and high-throughput single-nucleotide polymorphism (SNP) genotyping were readily available, including humans<a reveal-id="dsy023-B5" data-open="dsy023-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and model plant species.<a reveal-id="dsy023-B6" data-open="dsy023-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy023-B7" data-open="dsy023-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> However, little advance has taken place for the vast majority of plant species besides the mainstream crops,<a reveal-id="dsy023-B8" data-open="dsy023-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> some minor ones<a reveal-id="dsy023-B9" data-open="dsy023-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and the main plantation forest trees.<a reveal-id="dsy023-B10" data-open="dsy023-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>The dropping costs and increasing data yields of next-generation sequencing (NGS) technologies have fostered the development of several sequencing-based genotyping methods that allow for simultaneous discovery and genotyping of very large numbers of markers.<a reveal-id="dsy023-B11" data-open="dsy023-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Among these methods, targeted enrichment or sequence capture<a reveal-id="dsy023-B12" data-open="dsy023-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> has been increasingly used in a large array of organisms.<a reveal-id="dsy023-B13" data-open="dsy023-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Early reports published in humans<a reveal-id="dsy023-B14" data-open="dsy023-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy023-B15" data-open="dsy023-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and in plants such as maize,<a reveal-id="dsy023-B16" data-open="dsy023-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> eucalypts,<a reveal-id="dsy023-B17" data-open="dsy023-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> poplars<a reveal-id="dsy023-B18" data-open="dsy023-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and pines<a reveal-id="dsy023-B19" data-open="dsy023-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> showed the potential of the capture systems to assay sequence variants across the genomes. Sequence capture can significantly reduce costs in comparison to WGS because only specific loci of interest are sequenced at high coverage, increasing confidence of true variant detection. The practical advantages and increasing accessibility of target sequence capture methods have been reviewed in the context of evolutionary and ecological genomics, predicting a rapid expansion of this approach.<a reveal-id="dsy023-B20" data-open="dsy023-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> For example, targeted enrichment was recently used to increase phylogenetic resolution within the Neotropical tree genus Inga (Leguminosae),<a reveal-id="dsy023-B21" data-open="dsy023-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> generating alignments of over 0.3 Mbp of coding sequences revealing nearly 5,000 informative sites. Although providing large volumes of phylogenetically informative sequence data, analyses indicated that to further enhance understanding of the evolution of Inga, for instance by increasing ability to discriminate among more parameter-rich models and more recent divergence histories, a much larger set of sites, covering several thousand loci, would be needed.</p><p><em>Handroanthus impetiginosus</em> (Mart. ex DC.) Mattos (syn. <em>Tabebuia impetiginosa</em>, Bignoniaceae) commonly known as Pink Ipê or Lapacho tree, is a Neotropical hardwood widely distributed in seasonally dry tropical forests (SDTFs) of South America and Mesoamerica.<a reveal-id="dsy023-B22" data-open="dsy023-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy023-B23" data-open="dsy023-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Our previous studies with <em>H. impetiginosus</em> populations in Central Brazil have revealed incomplete lineage sorting and a highly structured genetic diversity,<a reveal-id="dsy023-B23" data-open="dsy023-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> representing an excellent system to better understand evolutionary processes that result in correlation between ecological and genetic differentiation in tropical biomes. We are now interested in disentangle the uncertainty in relating phylogenetic inference of ancestral geography and genealogical histories of individual loci in <em>H. impetiginosus</em>. Attempts to reconstruct species phylogeny using gene sequences has shown to be affected by the number of loci used to estimate the phylogeny and the number of individuals sampled per species.<a reveal-id="dsy023-B24" data-open="dsy023-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The availability of a high-throughput genotyping system at large numbers of loci in a reasonable number of individuals can provide, despite the presence of retention and sorting of ancestral polymorphism, clues on the occurrence of past contacts that may have facilitated evolutionary divergence of modern population sub-groups. From a practical standpoint, an improved knowledge of the evolutionary history and population genomics of this highly exploited species may also aid the precise identification of geographical origin of timber trade in forensic applications to combat the significant illegal trading pressure<a reveal-id="dsy023-B25" data-open="dsy023-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and help define management units for conservation planning.<a reveal-id="dsy023-B26" data-open="dsy023-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p><p>We have recently generated a preliminary genome assembly and annotation of the nuclear genome sequence of a single individual of <em>H. impetiginosus</em>.<a reveal-id="dsy023-B27" data-open="dsy023-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Using these resources as reference, in this work we designed and carried out an evaluation of a set of 30,795 sequence capture probes for the identification and genotyping of nearly 200,000 SNPs located inside <em>circa</em> of 14,000 annotated protein-coding genes or in close proximity to them to support multiple sequence-based genetic analyses in the species.</p>                            <h2 scrollto-destination=122670745 id="122670745" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122670746 id="122670746" class="section-title" >2.1. Biological material and sequencing</h3>
<p>Total genomic DNA samples from six unrelated trees of <em>H. impetiginosus</em> were extracted using the Qiagen DNeasy Plant Min kit (Qiagen, DK) and pooled in equimolar amounts to be whole-genome sequenced. These six trees used in the pooled WGS were part of the 24 trees later used for the sequence capture system. They were sampled at the rate of one per each population group in an attempt to sample a representative sequence diversity of the species for designing probes. One barcoded DNA library was built on the pooled sample and paired-end sequencing of this library (2 × 150 nt) was performed in half lane of an Illumina HiSeq 2500 instrument (Illumina, CA, USA) at the High-Throughput Sequencing and Genotyping Center of the University of Illinois Urbana-Champaign, USA. DNA samples of 24 <em>H. impetiginosus</em> individual haplotypes sampled from six subpopulations spanning a wide geographic range of the species (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were used for the development and evaluation of the sequence capture system.</p>                            <h3 scrollto-destination=122670748 id="122670748" class="section-title" >2.2. Sequence-capture probe design</h3>
<p>Two different sets of probe sequences were designed. The first one targeted transcript sequences of protein-coding genes predicted from the genome assembly of <em>H. impetiginosus</em> UFG-1.<a reveal-id="dsy023-B27" data-open="dsy023-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> These probes were designed by RAPiD Genomics LLC, Gainesville, FL, USA, from a total of 17,424 transcript sequences, which were tiled with 26,526 120-mer DNA sequence probes. The second set, designed by our group, sampled SNPs identified in pooled low coverage WGS data. We used nearly 20 Gbp of data (70,049,993 pairs of reads) for the SNP calling process. From the empirically determined haploid genome size of 557 Mbp/1C for <em>H. impetiginosus</em>,<a reveal-id="dsy023-B27" data-open="dsy023-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> this read set corresponded to nearly 3× of sequencing coverage in the pool of 12 chromosomes. Based on the 23,508 variants derived from the GATK<a reveal-id="dsy023-B28" data-open="dsy023-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and the 9,079 from the Cortex<a reveal-id="dsy023-B29" data-open="dsy023-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> analysis, the genome assembly of the species was used as a template for the design of 120-mer probes surrounding the SNP vicinity. A total of 4,269 120-mer DNA probe sequences were extracted from the template genome for subsequent use. The 4,269 WGS derived probes together with the 26,526 probes from transcript data, composed the capture system with a total of 30,795 probes. Additional details on the design of these probes are described in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=122670750 id="122670750" class="section-title" >2.3. Sequence-capture data generation</h3>
<p>Rapid Genomics LLC (Gainesville, FL, USA) Capture-Seq method provided target DNA enrichment across this custom probe set on the 24 individuals using an 8-plex capture pool.<a reveal-id="dsy023-B19" data-open="dsy023-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The products of multiplexed reactions were sequenced in a single lane of an Illumina HiSeq2000 instrument, 1 × 101 bp.</p>                            <h3 scrollto-destination=122670752 id="122670752" class="section-title" >2.4. Analyses of sequence capture data and definition of target loci</h3>
<p>Sequencing reads were trimmed for adapters and aligned to a genome sequence assembly of <em>H. impetiginosus</em><a reveal-id="dsy023-B27" data-open="dsy023-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with BWA<a reveal-id="dsy023-B30" data-open="dsy023-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> version 0.5.9 using BWA-backtrack algorithm, default settings. These steps were performed in a grid computing cluster software system using a pipeline available from the International Cassava Genetic Map Consortium.<a reveal-id="dsy023-B31" data-open="dsy023-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Preliminary analysis of variants was performed by joint-sample SNV calling and genotyping of the 24 individuals using GATK-GenotypeGVCFs method on gVCF files produced by HaplotypeCaller on each sample (options: <em>-mbq 10 -mmq 10 -hets 0.01 -ERC GVCF -ploidy 2</em>). For the resulting VCF file, the raw SNPs were analyzed. Coordinates of the 30,795 sequence probes on the genome were reputable initially as ‘on-target’ regions, while all the complementary intervals on the genome assembly were considered ‘off-target’. Following the preliminary analysis of on-target and off-target variants, sequence coordinates for the probes passing preliminary quality control were adjusted to include 200 bp in each direction. All book-ended (‘touching’) entries were then merged into a single interval using, respectively, <em>slopBed</em> and <em>mergeBed</em> utilities from the BedTools v2.25 package.<a reveal-id="dsy023-B32" data-open="dsy023-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> A set of non-overlapping intervals thereafter called <em>loci</em> was defined. This criterion was set to reduce the negative impact of unintended sequence capture while allowing for accurate variant detection in genomic regions contiguous to the region of interest thus extending the utility of the targeted sequencing to novel genomic sequences that would be more divergent from the probe sequence. It also allowed the identification of faulty or improper probe sequences in our initial design. Based on this criterion, the coordinates of the 30,795 probe sequences have determined 23,232 distinct target loci across the genome assembly. These encompass 19,962 loci determined by the coordinates of probe sequences from transcripts and 3,270 loci for the low-coverage WGS. For the assessment of the capture efficiency, only the defined loci were considered as callable regions for on-target SNP discovery and further genotyping, while calls outside these regions were discarded. Sequencing coverage and capture efficiency for loci determined by probe coordinates from transcripts and low-coverage WGS were computed per sample. Coverage was quoted as the ratio of the aligned read depth, which denotes the number of quality reads after alignment to the genome assembly, to the number of sequenced loci in the corresponding sample. Capture efficiency was estimated as the ratio of the number of loci for which at least one quality SNP was detected to the total number of loci. Finally, following GATK best practices and other recommendations to improve variant calling in NGS data,<a reveal-id="dsy023-B33" data-open="dsy023-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> we devised a procedure to establish useful quality dataset for further refinement of the SNP calls in the defined target loci as described in the <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=122670754 id="122670754" class="section-title" >2.5. Performance of the initially designed probe sequences for target enrichment and capture</h3>
<p>Bioinformatics filtering steps were developed to obtain quality SNP data across the defined 23,232 loci. The successive application of these filters led us to define call sets that encompassed a collection of variant calls across loci and probe sequences with increasing stringency, which were named STANDARD, GQ20+VQSR and GQ20+VQSR+MM80 (see Results for definitions). To detect improper designed or faulty probes for reliable SNP detection, capture efficiencies across these call sets were inspected by counting the number of probe sequences across target loci for which at least one quality SNP was detected. A success rate for probes was obtained by dividing the number of successful probes by their totals in the initial design, i.e. 30,795, whether the source of design were transcripts (26,526 probes) or low-coverage WGS (4,269 probes). Conversely, a failure rate was defined as (1 − success rate). The Success/Failure rate was also stratified according the probe location in the genome assembly for predicted gene model features or intergenic region.</p>                            <h3 scrollto-destination=122670756 id="122670756" class="section-title" >2.6. Reproducibility of the capture assay</h3>
<p>Reproducibility was assessed in terms of SNP recall rate and precision between technical replicates conducted in the same laboratory. Reproducibility was assessed for a partial set of 14,135 sequence capture probes selected out of the 30,795 probes (23,232 loci). The coordinates of these 14,135 probe sequences fell within 11,026 loci on the genome assembly. Nearly 50% (5,978 out of 11,026) of these loci were randomly selected in the well-curated GQ20+VQSR set of probes. It included 6,843 probe sequences from transcripts and 1,313 probes from low-coverage WGS. The remaining 5,048 loci (3,023 probe sequences from transcripts and 2,956 from low-coverage WGS) were taken randomly from the remaining set of loci. For an adequate reproducibility assay, the same level of DNA pooling for the capture assay, i.e. 8 samples per pool, and sequencing effort, i.e. 24 barcoded samples in one lane of a HiSeq2000 instrument, were kept between replicates. The sequence data in the first assay (called HIMP-1 thereafter) and the second partial replicate (called HIMP-2 thereafter) were processed using the analytical protocol developed in this study, including variant recalibration using a database of likelihoods for markers in the HIMP-1 study. VCF files for the two assays were combined and evaluated as described in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>.</p>                            <h3 scrollto-destination=122670758 id="122670758" class="section-title" >2.7. Genomic variant characterization</h3>
<p>We combined the sequence data in HIMP-1 and HIMP-2 capture assays and used our pipeline to compile a catalog of SNPs across the genome assembly. SnpEff<a reveal-id="dsy023-B34" data-open="dsy023-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> version 4.3t was used to annotate variants in the genome assembly based on their targeted regions and predicted coding effects. As a first glimpse of the ability of this sequence capture system to carry out population genomics studies of <em>H. impetiginosus</em>, we used the SNPs data to look at the site frequency spectrum (SFS) and some population genetics features. To account for the fact that the sequence capture system provides data in ‘blocks’, with SNPs likely in complete linkage disequilibrium (LD), we used Plink v1.90b2h<a reveal-id="dsy023-B35" data-open="dsy023-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> to greedily prune SNPs. A window size of variant count was used such that no SNP pairs remained with <em>r</em><sup>2</sup> greater than a threshold for tight LD (–indep-pairwise 50 5 0.5), and only included SNPs for which missing data was ≤20% (–geno 0.2). We used the data of the LD-pruned SNPs to obtain estimates of population heterozygosity, the ratio of transitions and transversion (Ts/Tv) and the number of SNP with heterozygous or non-reference homozygous genotypes (het/non-ref-hom).</p>                            <h2 scrollto-destination=122670760 id="122670760" class="section-title" >3. Results</h2>
<p>To facilitate following the sequence of methods and results obtained along the development and evaluation of the sequence capture system, a comprehensive flowchart is presented (<a reveal-id="dsy023-F1" data-open="dsy023-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The overall process involved five steps, starting with the sequence probe design, all the way to the definition of a set of loci (non-overlapping intervals), recognition of improper or faulty probes, and genome annotation of the sampled SNPs. Each step is detailed in its components such that one may follow the numbers of input sequences, probes or SNPs, and the resulting output numbers achieved. </p>                        <a id="122670762" scrollto-destination="122670762"></a>
<div id="dsy023-F1" data-id="dsy023-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/m_dsy023f1.png?Expires=2147483647&Signature=lvOkD4W5Dm8cN78~UhDXzINGaH21SRfjTdvJg-qfc96Ri4IuGMTz02dt~jRNVK3C-1fS4GKSKZ9faZHTiqKO2pxyr7kuGknQOT3Pe3XNPHCHTqN076fmKylw5lDyHfr9xlGOEpcOJ7GSVnqaLGVBwwE3k~a1L~klTbXmONfVdj1a5I~kJKtcmm-SbD9luRvmhHBbENug~kvQQwcIAKQwEsNWHC4hiIqjrR0dxDerMRJH~Z9UtyTOVPLduroEFgcJV5q-izcn-UKEPtryxaG~fpHMGER8l2swJ-EXSnDBdpQEoxsBeaYM70RXVmhCG6rX-P-gh37YImzS6J8oxhon1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the sequence of steps and corresponding input and output results in terms of sequence data, probes and SNPs obtained along the development and evaluation of the sequence capture system for Handroanthus impetiginosus." path-from-xml="dsy023f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670762/dsy023f1.tif" path-from-xml="dsy023f1.tif" target="_blank">View large</a><a section="122670762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023f1.png?Expires=2147483647&Signature=pkpjUnEGCblhpgjaBDE-jhHbucrCtYrrNOPXORUZBz5Q9LSjWHjcpwe0v3bOIyhFemAOf-iAJSxKo5LgNBjpRqVE3xM21pT~2qluhLEn6RYFlXLJQFI27zEGr-vfgiIEygiStmwVNk1mLFJnnvUqElsIFawj8a4dyJqaNcWOv9hRzSqpQnt3Uc1-1dcx6jUHoNPLM2kK-IpEhvur30cvh73lFXWprRSqOV9A480ZqxRZ-vmz~orVxB77xL2ZPqTk2NwO32~xzHH1Ckk65jwoqsiAaG8MDt5GbjwRjstcr2um5slKDOkyl8qKbP~HipeJI6a1DMneJL-SVH02OOA65w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670762&ar=5055375&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy023f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the sequence of steps and corresponding input and output results in terms of sequence data, probes and SNPs obtained along the development and evaluation of the sequence capture system for <em>Handroanthus impetiginosus</em>.</p></div></div><div content-id="dsy023-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/m_dsy023f1.png?Expires=2147483647&Signature=lvOkD4W5Dm8cN78~UhDXzINGaH21SRfjTdvJg-qfc96Ri4IuGMTz02dt~jRNVK3C-1fS4GKSKZ9faZHTiqKO2pxyr7kuGknQOT3Pe3XNPHCHTqN076fmKylw5lDyHfr9xlGOEpcOJ7GSVnqaLGVBwwE3k~a1L~klTbXmONfVdj1a5I~kJKtcmm-SbD9luRvmhHBbENug~kvQQwcIAKQwEsNWHC4hiIqjrR0dxDerMRJH~Z9UtyTOVPLduroEFgcJV5q-izcn-UKEPtryxaG~fpHMGER8l2swJ-EXSnDBdpQEoxsBeaYM70RXVmhCG6rX-P-gh37YImzS6J8oxhon1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the sequence of steps and corresponding input and output results in terms of sequence data, probes and SNPs obtained along the development and evaluation of the sequence capture system for Handroanthus impetiginosus." path-from-xml="dsy023f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670762/dsy023f1.tif" path-from-xml="dsy023f1.tif" target="_blank">View large</a><a section="122670762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023f1.png?Expires=2147483647&Signature=pkpjUnEGCblhpgjaBDE-jhHbucrCtYrrNOPXORUZBz5Q9LSjWHjcpwe0v3bOIyhFemAOf-iAJSxKo5LgNBjpRqVE3xM21pT~2qluhLEn6RYFlXLJQFI27zEGr-vfgiIEygiStmwVNk1mLFJnnvUqElsIFawj8a4dyJqaNcWOv9hRzSqpQnt3Uc1-1dcx6jUHoNPLM2kK-IpEhvur30cvh73lFXWprRSqOV9A480ZqxRZ-vmz~orVxB77xL2ZPqTk2NwO32~xzHH1Ckk65jwoqsiAaG8MDt5GbjwRjstcr2um5slKDOkyl8qKbP~HipeJI6a1DMneJL-SVH02OOA65w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670762&ar=5055375&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy023f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the sequence of steps and corresponding input and output results in terms of sequence data, probes and SNPs obtained along the development and evaluation of the sequence capture system for <em>Handroanthus impetiginosus</em>.</p></div></div>                            <h3 scrollto-destination=122670763 id="122670763" class="section-title" >3.1. Sequence capture data</h3>
<p>A total of 156,896, 801 single-end reads were generated from sequencing of DNA fragments captured by the 30,795 probes in the sample of 24 individuals of <em>H. impetiginosus</em> (mean: 6,537,367; sd: 2,865,856) (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Alignment of the reads to the genome assembly was determined for each sample. We observed a varying fraction of unmapped reads which ranged from 10% to nearly 40% for some individuals. For mapped reads, mappability within targeted regions was further evaluated using HCMappingQualityFilter in GATK HaplotypeCaller (-mmq 10, default value). Overall good read mappability was achieved, with only 3–4% of the reads being discarded prior to variant calling.</p>                            <h3 scrollto-destination=122670765 id="122670765" class="section-title" >3.2. Preliminary analyses of SNPs and definition of target loci</h3>
<p>A total of 251,169 SNPs were located in the genomic intervals of the 30,795 probe coordinates (‘on-target variants’). We identified a set of targeted regions that consistently produced data in the sampled individuals. For three samples the sequencing coverage was &lt;40× (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). With the exclusion of these samples, a total of 26,451 probe sequences successfully captured on-target SNPs. In our evaluation of off-target variants (see <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>), we found that additional 134,126 quality SNPs were spread across the genome assembly in regions up to 25 kbp from the closest targeted location (<a reveal-id="dsy023-F2" data-open="dsy023-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). These distances were unevenly distributed and most of them (75% of the total off-target SNPs) were only 200 bp upstream or downstream to the targeted location on the genome assembly. Sequence coordinates for these 26,451 probes passing the preliminary analyses were adjusted then to include 200 bp in each direction (see Materials and Methods). A set of 19,627 capturable <em>loci</em> was thus defined (average length: 607 bp; median length: 520 bp) and none of them had matches to annotated coordinates to repeats in the genome assembly of <em>H. impetiginosus</em>. Consequently, 4,344 probe sequences (∼14% of 30,795) were considered improper in the design of the capture system. </p>                        <a id="122670767" scrollto-destination="122670767"></a>
<div id="dsy023-F2" data-id="dsy023-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/m_dsy023f2.png?Expires=2147483647&Signature=dkozdBG7Q43uFZkq12ewIOXx0dqIsej3mkeBXbMTnjsOne-4-r5D4hjkoLr3AiqbHRPpGBTiZm4C4yTRmVmMrzH1XxbrxMNFj-mASGCmK1o3XFuMnRWuG5FCYkGSWRKtY3c9XuPA24P49JB2e59HDuRb9uf-Z0LKbTqXtkljQE3xl24ad9iLeG1sNcFiOGinhRt4p1dZJ1d2~bxLGBjT2qHzN7z~tFqU7akWQgCDvStwEw1~Az0TXScyeoKWua66pfsauuolzh-unDY2hFQ7y4csLfUNU~qhPm9vubnz4jU9Ez3y2msSOLtnikeiEfe~NE0cAH5fZzdPIalSaa1mFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of off-target SNPs distances to the closest probe sequence coordinate. (A) The narrow spectrum of distances indicates that most of the identified SNPs are located up to 200 bp upstream or downstream to a targeted location in the genome. (B) The wide spectrum of distances shows that off-target variants were found spread across the genome in regions up to 25 kbp from the closest targeted probe location." path-from-xml="dsy023f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670767/dsy023f2.tif" path-from-xml="dsy023f2.tif" target="_blank">View large</a><a section="122670767" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023f2.png?Expires=2147483647&Signature=3BN2vnVSn2ZpzHess6oHyZ~Dk0hoC7bKoE8-d-61VQFTtNJ6kJ4KEuPIeOw-n0cQh6k4P4xRmGVjXuOd7q5zlcmylDKB2KY63cJLpXPxTGIHajflElYLBR4qyWMtnZEb-Osqk3nEVzZiqI3PQ7Pa1BJoEFuv9vHeyOELdhaMbTZXpuD~wGYaeBfFL1lYCnS-qS9HcLG758KBFg1Uk3SfFikRpwkh2VtVgRHYy09GoWsqp~EF93alPxJNr2-rQLwfgmRu51oVThka8ujUi26HlALSSjyLLIzvrg4btQk4uVxzbcL0zHlBI7kmjPSB9B-WYMN23sUosXJfcafnM7HGgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670767&ar=5055375&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy023f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of off-target SNPs distances to the closest probe sequence coordinate. (A) The narrow spectrum of distances indicates that most of the identified SNPs are located up to 200 bp upstream or downstream to a targeted location in the genome. (B) The wide spectrum of distances shows that off-target variants were found spread across the genome in regions up to 25 kbp from the closest targeted probe location.</p></div></div><div content-id="dsy023-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/m_dsy023f2.png?Expires=2147483647&Signature=dkozdBG7Q43uFZkq12ewIOXx0dqIsej3mkeBXbMTnjsOne-4-r5D4hjkoLr3AiqbHRPpGBTiZm4C4yTRmVmMrzH1XxbrxMNFj-mASGCmK1o3XFuMnRWuG5FCYkGSWRKtY3c9XuPA24P49JB2e59HDuRb9uf-Z0LKbTqXtkljQE3xl24ad9iLeG1sNcFiOGinhRt4p1dZJ1d2~bxLGBjT2qHzN7z~tFqU7akWQgCDvStwEw1~Az0TXScyeoKWua66pfsauuolzh-unDY2hFQ7y4csLfUNU~qhPm9vubnz4jU9Ez3y2msSOLtnikeiEfe~NE0cAH5fZzdPIalSaa1mFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of off-target SNPs distances to the closest probe sequence coordinate. (A) The narrow spectrum of distances indicates that most of the identified SNPs are located up to 200 bp upstream or downstream to a targeted location in the genome. (B) The wide spectrum of distances shows that off-target variants were found spread across the genome in regions up to 25 kbp from the closest targeted probe location." path-from-xml="dsy023f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122670767/dsy023f2.tif" path-from-xml="dsy023f2.tif" target="_blank">View large</a><a section="122670767" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023f2.png?Expires=2147483647&Signature=3BN2vnVSn2ZpzHess6oHyZ~Dk0hoC7bKoE8-d-61VQFTtNJ6kJ4KEuPIeOw-n0cQh6k4P4xRmGVjXuOd7q5zlcmylDKB2KY63cJLpXPxTGIHajflElYLBR4qyWMtnZEb-Osqk3nEVzZiqI3PQ7Pa1BJoEFuv9vHeyOELdhaMbTZXpuD~wGYaeBfFL1lYCnS-qS9HcLG758KBFg1Uk3SfFikRpwkh2VtVgRHYy09GoWsqp~EF93alPxJNr2-rQLwfgmRu51oVThka8ujUi26HlALSSjyLLIzvrg4btQk4uVxzbcL0zHlBI7kmjPSB9B-WYMN23sUosXJfcafnM7HGgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122670767&ar=5055375&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy023f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of off-target SNPs distances to the closest probe sequence coordinate. (A) The narrow spectrum of distances indicates that most of the identified SNPs are located up to 200 bp upstream or downstream to a targeted location in the genome. (B) The wide spectrum of distances shows that off-target variants were found spread across the genome in regions up to 25 kbp from the closest targeted probe location.</p></div></div>                            <h3 scrollto-destination=122670768 id="122670768" class="section-title" >3.3. Improvement procedure for SNP calling and genotyping refinement</h3>
<p>We applied a bioinformatics protocol for SNP discovery and progressive filtering of low-confidence variants across the 26,451 120-mer sequences and the 19,627 targeted loci defined across this probe set. Overall, the numbers of aligned reads were 63,868,629 (∼41% of the total reads) and 107,275,224 (∼60% of the total reads) across probe and loci coordinates, respectively. The average sequencing coverage and standard deviation for the samples across the loci was 136x ± 75x (9x − 304x). Sequencing coverage and capture efficiency for loci determined by probe coordinates from transcripts and low-coverage WGS were estimated per sample (<a reveal-id="dsy023-T1" data-open="dsy023-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Considering the samples with capture efficiency greater than 60%, the coverage was equal or greater than 70x. This coverage is enough to produce accurate joint genotype calls for common variants across the samples (minimum allele frequency, MAF &gt; 5%) using per-sample read depth as low as 3 (70x0.05) at the variant site. The call set across these target loci resulted in 688,754 SNPs. It was named ‘STANDARD’ and its main features are shown in <a reveal-id="dsy023-T2" data-open="dsy023-T2" class="link link-reveal link-table xref-fig">Table 2</a>. <div content-id="dsy023-T1" class="table-modal"><div class="table-wrap"><div id="dsy023-T1" data-id="dsy023-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of per sample coverage and capture efficiency over the 23,232 target loci in the genome assembly of <em>H. impetiginosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th><strong>Coverage</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>Number of sequenced loci out of the 23,232</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>Efficiency over 19,962 loci determined by probes from transcripts</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(a)</strong></sup> </th><th><strong>Efficiency over 3,270 loci determined by probes from low-coverage WGS</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(b)</strong></sup> </th><th><strong>Efficiency over all 23,232 loci</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(c)</strong></sup> </th></tr></thead><tbody><tr><td>POS-15-1 </td><td>304 </td><td>19,627 </td><td>72% </td><td>63% </td><td>71% </td></tr><tr><td>POT-05-1 </td><td>265 </td><td>18,951 </td><td>70% </td><td>66% </td><td>69% </td></tr><tr><td>CAR-05-1 </td><td>210 </td><td>19,077 </td><td>70% </td><td>67% </td><td>69% </td></tr><tr><td>POS-10-1 </td><td>211 </td><td>18,671 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>SEC-23-1 </td><td>217 </td><td>18,944 </td><td>70% </td><td>63% </td><td>69% </td></tr><tr><td>SEC-08-1 </td><td>245 </td><td>18,784 </td><td>69% </td><td>65% </td><td>68% </td></tr><tr><td>SEC-05-1 </td><td>146 </td><td>18,524 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>CAR-02-1 </td><td>100 </td><td>18,047 </td><td>66% </td><td>63% </td><td>66% </td></tr><tr><td>SUM-13-1 </td><td>132 </td><td>17,909 </td><td>68% </td><td>47% </td><td>65% </td></tr><tr><td>SUM-12-1 </td><td>114 </td><td>17,797 </td><td>68% </td><td>46% </td><td>65% </td></tr><tr><td>MOC-08-1 </td><td>112 </td><td>17,877 </td><td>70% </td><td>37% </td><td>65% </td></tr><tr><td>MOC-11-1 </td><td>136 </td><td>17,788 </td><td>69% </td><td>44% </td><td>65% </td></tr><tr><td>MOC-02-1 </td><td>148 </td><td>17,765 </td><td>68% </td><td>41% </td><td>64% </td></tr><tr><td>MOC-09-1 </td><td>129 </td><td>17,547 </td><td>68% </td><td>39% </td><td>64% </td></tr><tr><td>MOC-12-1 </td><td>133 </td><td>17,625 </td><td>68% </td><td>42% </td><td>64% </td></tr><tr><td>MOC-13-1 </td><td>125 </td><td>17,705 </td><td>68% </td><td>43% </td><td>64% </td></tr><tr><td>MOC-07-1 </td><td>118 </td><td>17,524 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>MOC-03-1 </td><td>112 </td><td>17,436 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>SUM-10-1 </td><td>72 </td><td>17,443 </td><td>66% </td><td>48% </td><td>63% </td></tr><tr><td>MOC-06-1 </td><td>94 </td><td>17,226 </td><td>66% </td><td>42% </td><td>63% </td></tr><tr><td>MOC-10-1 </td><td>70 </td><td>16,986 </td><td>65% </td><td>43% </td><td>62% </td></tr><tr><td>MOC-05-1 </td><td>25 </td><td>16,652 </td><td>62% </td><td>45% </td><td>60% </td></tr><tr><td>SEC-09-1 </td><td>9 </td><td>16,229 </td><td>58% </td><td>65% </td><td>59% </td></tr><tr><td>MOC-04-1 </td><td>28 </td><td>15,942 </td><td>61% </td><td>44% </td><td>58% </td></tr></tbody></table></div><div content-id="dsy023-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th><strong>Coverage</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>Number of sequenced loci out of the 23,232</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>Efficiency over 19,962 loci determined by probes from transcripts</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(a)</strong></sup> </th><th><strong>Efficiency over 3,270 loci determined by probes from low-coverage WGS</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(b)</strong></sup> </th><th><strong>Efficiency over all 23,232 loci</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(c)</strong></sup> </th></tr></thead><tbody><tr><td>POS-15-1 </td><td>304 </td><td>19,627 </td><td>72% </td><td>63% </td><td>71% </td></tr><tr><td>POT-05-1 </td><td>265 </td><td>18,951 </td><td>70% </td><td>66% </td><td>69% </td></tr><tr><td>CAR-05-1 </td><td>210 </td><td>19,077 </td><td>70% </td><td>67% </td><td>69% </td></tr><tr><td>POS-10-1 </td><td>211 </td><td>18,671 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>SEC-23-1 </td><td>217 </td><td>18,944 </td><td>70% </td><td>63% </td><td>69% </td></tr><tr><td>SEC-08-1 </td><td>245 </td><td>18,784 </td><td>69% </td><td>65% </td><td>68% </td></tr><tr><td>SEC-05-1 </td><td>146 </td><td>18,524 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>CAR-02-1 </td><td>100 </td><td>18,047 </td><td>66% </td><td>63% </td><td>66% </td></tr><tr><td>SUM-13-1 </td><td>132 </td><td>17,909 </td><td>68% </td><td>47% </td><td>65% </td></tr><tr><td>SUM-12-1 </td><td>114 </td><td>17,797 </td><td>68% </td><td>46% </td><td>65% </td></tr><tr><td>MOC-08-1 </td><td>112 </td><td>17,877 </td><td>70% </td><td>37% </td><td>65% </td></tr><tr><td>MOC-11-1 </td><td>136 </td><td>17,788 </td><td>69% </td><td>44% </td><td>65% </td></tr><tr><td>MOC-02-1 </td><td>148 </td><td>17,765 </td><td>68% </td><td>41% </td><td>64% </td></tr><tr><td>MOC-09-1 </td><td>129 </td><td>17,547 </td><td>68% </td><td>39% </td><td>64% </td></tr><tr><td>MOC-12-1 </td><td>133 </td><td>17,625 </td><td>68% </td><td>42% </td><td>64% </td></tr><tr><td>MOC-13-1 </td><td>125 </td><td>17,705 </td><td>68% </td><td>43% </td><td>64% </td></tr><tr><td>MOC-07-1 </td><td>118 </td><td>17,524 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>MOC-03-1 </td><td>112 </td><td>17,436 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>SUM-10-1 </td><td>72 </td><td>17,443 </td><td>66% </td><td>48% </td><td>63% </td></tr><tr><td>MOC-06-1 </td><td>94 </td><td>17,226 </td><td>66% </td><td>42% </td><td>63% </td></tr><tr><td>MOC-10-1 </td><td>70 </td><td>16,986 </td><td>65% </td><td>43% </td><td>62% </td></tr><tr><td>MOC-05-1 </td><td>25 </td><td>16,652 </td><td>62% </td><td>45% </td><td>60% </td></tr><tr><td>SEC-09-1 </td><td>9 </td><td>16,229 </td><td>58% </td><td>65% </td><td>59% </td></tr><tr><td>MOC-04-1 </td><td>28 </td><td>15,942 </td><td>61% </td><td>44% </td><td>58% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>Coverage is the ratio of the aligned read depth, which denotes the number of quality reads after alignment to the genome assembly, to the number of sequenced loci in the corresponding sample.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>Denotes the number of target loci in the initial design of the capture system for which at least one quality read was detected after alignment to the genome assembly.</p></div></div><div content-id="tblfn3"><div class="fn"><div class="label -label">c</div><p>Capture efficiency was computed at sample level by the ratio between the number of loci for which at least one quality SNP was detected using the improvement procedure for SNP calling and genotyping refinement and the total number of loci determined by probes in the corresponding design: 19,962 loci from transcripts of protein-coding genes (c(a)); 3,270 loci from low-coverage WGS (c(b)); total of 23,232 loci (c(c)).</p></div></div></div></div></div><div class="table-wrap"><div id="dsy023-T1" data-id="dsy023-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of per sample coverage and capture efficiency over the 23,232 target loci in the genome assembly of <em>H. impetiginosus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th><strong>Coverage</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>Number of sequenced loci out of the 23,232</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>Efficiency over 19,962 loci determined by probes from transcripts</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(a)</strong></sup> </th><th><strong>Efficiency over 3,270 loci determined by probes from low-coverage WGS</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(b)</strong></sup> </th><th><strong>Efficiency over all 23,232 loci</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(c)</strong></sup> </th></tr></thead><tbody><tr><td>POS-15-1 </td><td>304 </td><td>19,627 </td><td>72% </td><td>63% </td><td>71% </td></tr><tr><td>POT-05-1 </td><td>265 </td><td>18,951 </td><td>70% </td><td>66% </td><td>69% </td></tr><tr><td>CAR-05-1 </td><td>210 </td><td>19,077 </td><td>70% </td><td>67% </td><td>69% </td></tr><tr><td>POS-10-1 </td><td>211 </td><td>18,671 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>SEC-23-1 </td><td>217 </td><td>18,944 </td><td>70% </td><td>63% </td><td>69% </td></tr><tr><td>SEC-08-1 </td><td>245 </td><td>18,784 </td><td>69% </td><td>65% </td><td>68% </td></tr><tr><td>SEC-05-1 </td><td>146 </td><td>18,524 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>CAR-02-1 </td><td>100 </td><td>18,047 </td><td>66% </td><td>63% </td><td>66% </td></tr><tr><td>SUM-13-1 </td><td>132 </td><td>17,909 </td><td>68% </td><td>47% </td><td>65% </td></tr><tr><td>SUM-12-1 </td><td>114 </td><td>17,797 </td><td>68% </td><td>46% </td><td>65% </td></tr><tr><td>MOC-08-1 </td><td>112 </td><td>17,877 </td><td>70% </td><td>37% </td><td>65% </td></tr><tr><td>MOC-11-1 </td><td>136 </td><td>17,788 </td><td>69% </td><td>44% </td><td>65% </td></tr><tr><td>MOC-02-1 </td><td>148 </td><td>17,765 </td><td>68% </td><td>41% </td><td>64% </td></tr><tr><td>MOC-09-1 </td><td>129 </td><td>17,547 </td><td>68% </td><td>39% </td><td>64% </td></tr><tr><td>MOC-12-1 </td><td>133 </td><td>17,625 </td><td>68% </td><td>42% </td><td>64% </td></tr><tr><td>MOC-13-1 </td><td>125 </td><td>17,705 </td><td>68% </td><td>43% </td><td>64% </td></tr><tr><td>MOC-07-1 </td><td>118 </td><td>17,524 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>MOC-03-1 </td><td>112 </td><td>17,436 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>SUM-10-1 </td><td>72 </td><td>17,443 </td><td>66% </td><td>48% </td><td>63% </td></tr><tr><td>MOC-06-1 </td><td>94 </td><td>17,226 </td><td>66% </td><td>42% </td><td>63% </td></tr><tr><td>MOC-10-1 </td><td>70 </td><td>16,986 </td><td>65% </td><td>43% </td><td>62% </td></tr><tr><td>MOC-05-1 </td><td>25 </td><td>16,652 </td><td>62% </td><td>45% </td><td>60% </td></tr><tr><td>SEC-09-1 </td><td>9 </td><td>16,229 </td><td>58% </td><td>65% </td><td>59% </td></tr><tr><td>MOC-04-1 </td><td>28 </td><td>15,942 </td><td>61% </td><td>44% </td><td>58% </td></tr></tbody></table></div><div content-id="dsy023-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th><strong>Coverage</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>Number of sequenced loci out of the 23,232</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>Efficiency over 19,962 loci determined by probes from transcripts</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(a)</strong></sup> </th><th><strong>Efficiency over 3,270 loci determined by probes from low-coverage WGS</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(b)</strong></sup> </th><th><strong>Efficiency over all 23,232 loci</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><sup><strong>(c)</strong></sup> </th></tr></thead><tbody><tr><td>POS-15-1 </td><td>304 </td><td>19,627 </td><td>72% </td><td>63% </td><td>71% </td></tr><tr><td>POT-05-1 </td><td>265 </td><td>18,951 </td><td>70% </td><td>66% </td><td>69% </td></tr><tr><td>CAR-05-1 </td><td>210 </td><td>19,077 </td><td>70% </td><td>67% </td><td>69% </td></tr><tr><td>POS-10-1 </td><td>211 </td><td>18,671 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>SEC-23-1 </td><td>217 </td><td>18,944 </td><td>70% </td><td>63% </td><td>69% </td></tr><tr><td>SEC-08-1 </td><td>245 </td><td>18,784 </td><td>69% </td><td>65% </td><td>68% </td></tr><tr><td>SEC-05-1 </td><td>146 </td><td>18,524 </td><td>68% </td><td>63% </td><td>68% </td></tr><tr><td>CAR-02-1 </td><td>100 </td><td>18,047 </td><td>66% </td><td>63% </td><td>66% </td></tr><tr><td>SUM-13-1 </td><td>132 </td><td>17,909 </td><td>68% </td><td>47% </td><td>65% </td></tr><tr><td>SUM-12-1 </td><td>114 </td><td>17,797 </td><td>68% </td><td>46% </td><td>65% </td></tr><tr><td>MOC-08-1 </td><td>112 </td><td>17,877 </td><td>70% </td><td>37% </td><td>65% </td></tr><tr><td>MOC-11-1 </td><td>136 </td><td>17,788 </td><td>69% </td><td>44% </td><td>65% </td></tr><tr><td>MOC-02-1 </td><td>148 </td><td>17,765 </td><td>68% </td><td>41% </td><td>64% </td></tr><tr><td>MOC-09-1 </td><td>129 </td><td>17,547 </td><td>68% </td><td>39% </td><td>64% </td></tr><tr><td>MOC-12-1 </td><td>133 </td><td>17,625 </td><td>68% </td><td>42% </td><td>64% </td></tr><tr><td>MOC-13-1 </td><td>125 </td><td>17,705 </td><td>68% </td><td>43% </td><td>64% </td></tr><tr><td>MOC-07-1 </td><td>118 </td><td>17,524 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>MOC-03-1 </td><td>112 </td><td>17,436 </td><td>67% </td><td>41% </td><td>63% </td></tr><tr><td>SUM-10-1 </td><td>72 </td><td>17,443 </td><td>66% </td><td>48% </td><td>63% </td></tr><tr><td>MOC-06-1 </td><td>94 </td><td>17,226 </td><td>66% </td><td>42% </td><td>63% </td></tr><tr><td>MOC-10-1 </td><td>70 </td><td>16,986 </td><td>65% </td><td>43% </td><td>62% </td></tr><tr><td>MOC-05-1 </td><td>25 </td><td>16,652 </td><td>62% </td><td>45% </td><td>60% </td></tr><tr><td>SEC-09-1 </td><td>9 </td><td>16,229 </td><td>58% </td><td>65% </td><td>59% </td></tr><tr><td>MOC-04-1 </td><td>28 </td><td>15,942 </td><td>61% </td><td>44% </td><td>58% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>Coverage is the ratio of the aligned read depth, which denotes the number of quality reads after alignment to the genome assembly, to the number of sequenced loci in the corresponding sample.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>Denotes the number of target loci in the initial design of the capture system for which at least one quality read was detected after alignment to the genome assembly.</p></div></div><div content-id="tblfn3"><div class="fn"><div class="label -label">c</div><p>Capture efficiency was computed at sample level by the ratio between the number of loci for which at least one quality SNP was detected using the improvement procedure for SNP calling and genotyping refinement and the total number of loci determined by probes in the corresponding design: 19,962 loci from transcripts of protein-coding genes (c(a)); 3,270 loci from low-coverage WGS (c(b)); total of 23,232 loci (c(c)).</p></div></div></div></div><div content-id="dsy023-T2" class="table-modal"><div class="table-wrap"><div id="dsy023-T2" data-id="dsy023-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of the numbers of loci, size in base pairs (bp), numbers of probes, SNPs, gene models and intergenic regions retained following each filtering step in the variant analysis using the GATK framework</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SNP call set </th><th rowspan="2">Number of loci </th><th colspan="3">Target loci<hr /> </th><th rowspan="2">Number of probes </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of gene models </th></tr><tr><th>Total size (bp) </th><th>Mean (bp) </th><th>Median (bp) </th></tr></thead><tbody><tr><td>STANDARD </td><td>19,627 </td><td>11,913,787 </td><td>607 </td><td>520 </td><td>26,451 </td><td>688,754 </td><td>11,024 </td></tr><tr><td>GQ20+VQSR </td><td>18,216 </td><td>10,983,469 </td><td>603 </td><td>520 </td><td>24,771 </td><td>352,879 </td><td>10,400 </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>11,748 </td><td>7,392,636 </td><td>630 </td><td>520 </td><td>16,901 </td><td>83,476 </td><td>7,991 </td></tr></tbody></table></div><div content-id="dsy023-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SNP call set </th><th rowspan="2">Number of loci </th><th colspan="3">Target loci<hr /> </th><th rowspan="2">Number of probes </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of gene models </th></tr><tr><th>Total size (bp) </th><th>Mean (bp) </th><th>Median (bp) </th></tr></thead><tbody><tr><td>STANDARD </td><td>19,627 </td><td>11,913,787 </td><td>607 </td><td>520 </td><td>26,451 </td><td>688,754 </td><td>11,024 </td></tr><tr><td>GQ20+VQSR </td><td>18,216 </td><td>10,983,469 </td><td>603 </td><td>520 </td><td>24,771 </td><td>352,879 </td><td>10,400 </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>11,748 </td><td>7,392,636 </td><td>630 </td><td>520 </td><td>16,901 </td><td>83,476 </td><td>7,991 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>See Results for the definitions of the SNP call sets.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy023-T2" data-id="dsy023-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of the numbers of loci, size in base pairs (bp), numbers of probes, SNPs, gene models and intergenic regions retained following each filtering step in the variant analysis using the GATK framework</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SNP call set </th><th rowspan="2">Number of loci </th><th colspan="3">Target loci<hr /> </th><th rowspan="2">Number of probes </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of gene models </th></tr><tr><th>Total size (bp) </th><th>Mean (bp) </th><th>Median (bp) </th></tr></thead><tbody><tr><td>STANDARD </td><td>19,627 </td><td>11,913,787 </td><td>607 </td><td>520 </td><td>26,451 </td><td>688,754 </td><td>11,024 </td></tr><tr><td>GQ20+VQSR </td><td>18,216 </td><td>10,983,469 </td><td>603 </td><td>520 </td><td>24,771 </td><td>352,879 </td><td>10,400 </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>11,748 </td><td>7,392,636 </td><td>630 </td><td>520 </td><td>16,901 </td><td>83,476 </td><td>7,991 </td></tr></tbody></table></div><div content-id="dsy023-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SNP call set </th><th rowspan="2">Number of loci </th><th colspan="3">Target loci<hr /> </th><th rowspan="2">Number of probes </th><th rowspan="2">Number of SNPs </th><th rowspan="2">Number of gene models </th></tr><tr><th>Total size (bp) </th><th>Mean (bp) </th><th>Median (bp) </th></tr></thead><tbody><tr><td>STANDARD </td><td>19,627 </td><td>11,913,787 </td><td>607 </td><td>520 </td><td>26,451 </td><td>688,754 </td><td>11,024 </td></tr><tr><td>GQ20+VQSR </td><td>18,216 </td><td>10,983,469 </td><td>603 </td><td>520 </td><td>24,771 </td><td>352,879 </td><td>10,400 </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>11,748 </td><td>7,392,636 </td><td>630 </td><td>520 </td><td>16,901 </td><td>83,476 </td><td>7,991 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>See Results for the definitions of the SNP call sets.</p></div></div></div></div></p><p>Additional filtering based on SNP quality score recalibration using a custom database of likelihoods and genotyping refinement steps (see <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>) resulted in a set of 352,879 SNPs (51.2% of the variants in the STANDARD call set). Coordinates of these SNPs were within 18,216 targeted loci (≈11 Mbp), encompassing the locations of 24,771 probe sequences (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). This call set was named ‘GQ20+VQSR’ (<a reveal-id="dsy023-T2" data-open="dsy023-T2" class="link link-reveal link-table xref-fig">Table 2</a>). With this improvement procedure, an additional set of 1,680 probe sequences were deemed faulty for SNP detection. Together with the preliminary analysis, this additional step to detect faulty probes resulted in the exclusion of 6,024 (4,344 + 1,680) probes. It represents ∼20% of the probes in the initial design. Based on our assessment of genetic diversity from the sequence data, using average θ (the population mutation rate) of 0.009 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), and sample size (<em>N</em> = 21 diploid individuals), we found that the number of quality SNPs closely matches the expectation of ∼367,000 polymorphic sites using the Watterson estimator<a reveal-id="dsy023-B36" data-open="dsy023-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (=18,216 × 520 × 0,009 × 4.3), where 4.3 is the partial sum of the harmonic series with (2*<em>N</em> – 1) terms. Finally, we applied a threshold of less than 20% of missing data for any site on the former ‘GQ20+VQSR’ call set. Considering the 21 individuals, the number of retained polymorphic SNPs dropped from 352,879 to 83,476, within 11,748 loci (∼65% of 18,216). This call set resulting from all three filters applied was named ‘GQ20+VQSR+MM80’ (<a reveal-id="dsy023-T2" data-open="dsy023-T2" class="link link-reveal link-table xref-fig">Table 2</a>).</p>                            <h3 scrollto-destination=122670771 id="122670771" class="section-title" >3.4. Efficiency of the capture system</h3>
<p>To assess the efficiency of the system to detect genomic variants, we considered the initial number of 30,795 probe sequences and 23,232 distinct target loci. The number of loci targeted by probes in which we successful detected quality SNPs ranged from 11,748 (50.6% of 23,232) to 18,216 (78.4% of 23,232) across quality detected variants, whether we accounted for polymorphism (MAF &gt; 5%) and low-missing data at the genotype level (&lt;20%) or only polymorphism requirement, respectively. <a reveal-id="dsy023-T3" data-open="dsy023-T3" class="link link-reveal link-table xref-fig">Table 3</a> summarizes the capture efficiencies for probes across the progressively filtered call sets whether the source of design were transcripts or low-coverage WGS and considering the location of the probes in the genome assembly. <div content-id="dsy023-T3" class="table-modal"><div class="table-wrap"><div id="dsy023-T3" data-id="dsy023-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Performance of the initially designed probe sequences for target enrichment and capture</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="6">Genic region<hr /> </th><th colspan="2" rowspan="2">Intergenic region<hr /> </th><th colspan="2" rowspan="2">Total<hr /> </th></tr><tr><th>Source </th><th>Filter </th><th colspan="2">Exon<hr /> </th><th colspan="2">Exon + flanking<hr /> </th><th colspan="2">Intron<hr /> </th></tr><tr><th> </th><th> </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th></tr></thead><tbody><tr><td rowspan="3">Probes from transcripts </td><td>STANDARD </td><td>3,062 (88%) </td><td>405 (12%) </td><td>16,208 (90%) </td><td>1,870 (10%) </td><td>1,163 (83%) </td><td>230 (17%) </td><td>2,070 (58%) </td><td>1,518 (42%) </td><td>22,503 (85%) </td><td>4,023 (15%) </td></tr><tr><td>GQ20+VQSR </td><td>2,872 (83%) </td><td>595 (17%) </td><td>15,197 (84%) </td><td>2,881 (16%) </td><td>1,051 (75%) </td><td>342 (25%) </td><td>1,821 (51%) </td><td>1,767 (49%) </td><td>20,941 (79%) </td><td>5,585 (21%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>2,447 (71%) </td><td>1,020 (29%) </td><td>11,529 (64%) </td><td>6,549 (36%) </td><td>511 (37%) </td><td>882 (63%) </td><td>1,101 (31%) </td><td>2,487 (69%) </td><td>15,588 (59%) </td><td>10,938 (41%) </td></tr><tr><td rowspan="3">Probes from low-coverage WGS </td><td>STANDARD </td><td>102 (90%) </td><td>11 (10%) </td><td>1,813 (94%) </td><td>106 (6%) </td><td>214 (94%) </td><td>14 (6%) </td><td>1,819 (91%) </td><td>190 (9%) </td><td>3,948 (92%) </td><td>321 (8%) </td></tr><tr><td>GQ20+VQSR </td><td>102 (90%) </td><td>11 (10%) </td><td>1,799 (94%) </td><td>120 (6%) </td><td>209 (92%) </td><td>19 (8%) </td><td>1,700 (85%) </td><td>309 (15%) </td><td>3,810 (89%) </td><td>459 (11%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>69 (61%) </td><td>44 (39%) </td><td>966 (50%) </td><td>953 (50%) </td><td>71 (31%) </td><td>157 (69%) </td><td>207 (10%) </td><td>1,802 (90%) </td><td>1,313 (31%) </td><td>2,956 (69%) </td></tr></tbody></table></div><div content-id="dsy023-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="6">Genic region<hr /> </th><th colspan="2" rowspan="2">Intergenic region<hr /> </th><th colspan="2" rowspan="2">Total<hr /> </th></tr><tr><th>Source </th><th>Filter </th><th colspan="2">Exon<hr /> </th><th colspan="2">Exon + flanking<hr /> </th><th colspan="2">Intron<hr /> </th></tr><tr><th> </th><th> </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th></tr></thead><tbody><tr><td rowspan="3">Probes from transcripts </td><td>STANDARD </td><td>3,062 (88%) </td><td>405 (12%) </td><td>16,208 (90%) </td><td>1,870 (10%) </td><td>1,163 (83%) </td><td>230 (17%) </td><td>2,070 (58%) </td><td>1,518 (42%) </td><td>22,503 (85%) </td><td>4,023 (15%) </td></tr><tr><td>GQ20+VQSR </td><td>2,872 (83%) </td><td>595 (17%) </td><td>15,197 (84%) </td><td>2,881 (16%) </td><td>1,051 (75%) </td><td>342 (25%) </td><td>1,821 (51%) </td><td>1,767 (49%) </td><td>20,941 (79%) </td><td>5,585 (21%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>2,447 (71%) </td><td>1,020 (29%) </td><td>11,529 (64%) </td><td>6,549 (36%) </td><td>511 (37%) </td><td>882 (63%) </td><td>1,101 (31%) </td><td>2,487 (69%) </td><td>15,588 (59%) </td><td>10,938 (41%) </td></tr><tr><td rowspan="3">Probes from low-coverage WGS </td><td>STANDARD </td><td>102 (90%) </td><td>11 (10%) </td><td>1,813 (94%) </td><td>106 (6%) </td><td>214 (94%) </td><td>14 (6%) </td><td>1,819 (91%) </td><td>190 (9%) </td><td>3,948 (92%) </td><td>321 (8%) </td></tr><tr><td>GQ20+VQSR </td><td>102 (90%) </td><td>11 (10%) </td><td>1,799 (94%) </td><td>120 (6%) </td><td>209 (92%) </td><td>19 (8%) </td><td>1,700 (85%) </td><td>309 (15%) </td><td>3,810 (89%) </td><td>459 (11%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>69 (61%) </td><td>44 (39%) </td><td>966 (50%) </td><td>953 (50%) </td><td>71 (31%) </td><td>157 (69%) </td><td>207 (10%) </td><td>1,802 (90%) </td><td>1,313 (31%) </td><td>2,956 (69%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>Capture efficiencies across filtered call sets (see Results for the definitions of the filter criteria) were inspected by counting the number of loci and probe sequences for which at least one quality SNP was detected. A success rate was obtained by dividing the number of successful probes by their totals in the initial design, i.e. 30,795 probe sequences, whether the source of design were mRNA (26,526 probes) or low-coverage WGS (4,269 probes). Conversely, a failure rate was defined as (1 − success rate). The Success/Failure rate was also stratified according the probe location in the genome assembly for predicted gene model features or intergenic region.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy023-T3" data-id="dsy023-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Performance of the initially designed probe sequences for target enrichment and capture</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="6">Genic region<hr /> </th><th colspan="2" rowspan="2">Intergenic region<hr /> </th><th colspan="2" rowspan="2">Total<hr /> </th></tr><tr><th>Source </th><th>Filter </th><th colspan="2">Exon<hr /> </th><th colspan="2">Exon + flanking<hr /> </th><th colspan="2">Intron<hr /> </th></tr><tr><th> </th><th> </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th></tr></thead><tbody><tr><td rowspan="3">Probes from transcripts </td><td>STANDARD </td><td>3,062 (88%) </td><td>405 (12%) </td><td>16,208 (90%) </td><td>1,870 (10%) </td><td>1,163 (83%) </td><td>230 (17%) </td><td>2,070 (58%) </td><td>1,518 (42%) </td><td>22,503 (85%) </td><td>4,023 (15%) </td></tr><tr><td>GQ20+VQSR </td><td>2,872 (83%) </td><td>595 (17%) </td><td>15,197 (84%) </td><td>2,881 (16%) </td><td>1,051 (75%) </td><td>342 (25%) </td><td>1,821 (51%) </td><td>1,767 (49%) </td><td>20,941 (79%) </td><td>5,585 (21%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>2,447 (71%) </td><td>1,020 (29%) </td><td>11,529 (64%) </td><td>6,549 (36%) </td><td>511 (37%) </td><td>882 (63%) </td><td>1,101 (31%) </td><td>2,487 (69%) </td><td>15,588 (59%) </td><td>10,938 (41%) </td></tr><tr><td rowspan="3">Probes from low-coverage WGS </td><td>STANDARD </td><td>102 (90%) </td><td>11 (10%) </td><td>1,813 (94%) </td><td>106 (6%) </td><td>214 (94%) </td><td>14 (6%) </td><td>1,819 (91%) </td><td>190 (9%) </td><td>3,948 (92%) </td><td>321 (8%) </td></tr><tr><td>GQ20+VQSR </td><td>102 (90%) </td><td>11 (10%) </td><td>1,799 (94%) </td><td>120 (6%) </td><td>209 (92%) </td><td>19 (8%) </td><td>1,700 (85%) </td><td>309 (15%) </td><td>3,810 (89%) </td><td>459 (11%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>69 (61%) </td><td>44 (39%) </td><td>966 (50%) </td><td>953 (50%) </td><td>71 (31%) </td><td>157 (69%) </td><td>207 (10%) </td><td>1,802 (90%) </td><td>1,313 (31%) </td><td>2,956 (69%) </td></tr></tbody></table></div><div content-id="dsy023-T3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th colspan="6">Genic region<hr /> </th><th colspan="2" rowspan="2">Intergenic region<hr /> </th><th colspan="2" rowspan="2">Total<hr /> </th></tr><tr><th>Source </th><th>Filter </th><th colspan="2">Exon<hr /> </th><th colspan="2">Exon + flanking<hr /> </th><th colspan="2">Intron<hr /> </th></tr><tr><th> </th><th> </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th><th>#of successes </th><th>#of failures </th></tr></thead><tbody><tr><td rowspan="3">Probes from transcripts </td><td>STANDARD </td><td>3,062 (88%) </td><td>405 (12%) </td><td>16,208 (90%) </td><td>1,870 (10%) </td><td>1,163 (83%) </td><td>230 (17%) </td><td>2,070 (58%) </td><td>1,518 (42%) </td><td>22,503 (85%) </td><td>4,023 (15%) </td></tr><tr><td>GQ20+VQSR </td><td>2,872 (83%) </td><td>595 (17%) </td><td>15,197 (84%) </td><td>2,881 (16%) </td><td>1,051 (75%) </td><td>342 (25%) </td><td>1,821 (51%) </td><td>1,767 (49%) </td><td>20,941 (79%) </td><td>5,585 (21%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>2,447 (71%) </td><td>1,020 (29%) </td><td>11,529 (64%) </td><td>6,549 (36%) </td><td>511 (37%) </td><td>882 (63%) </td><td>1,101 (31%) </td><td>2,487 (69%) </td><td>15,588 (59%) </td><td>10,938 (41%) </td></tr><tr><td rowspan="3">Probes from low-coverage WGS </td><td>STANDARD </td><td>102 (90%) </td><td>11 (10%) </td><td>1,813 (94%) </td><td>106 (6%) </td><td>214 (94%) </td><td>14 (6%) </td><td>1,819 (91%) </td><td>190 (9%) </td><td>3,948 (92%) </td><td>321 (8%) </td></tr><tr><td>GQ20+VQSR </td><td>102 (90%) </td><td>11 (10%) </td><td>1,799 (94%) </td><td>120 (6%) </td><td>209 (92%) </td><td>19 (8%) </td><td>1,700 (85%) </td><td>309 (15%) </td><td>3,810 (89%) </td><td>459 (11%) </td></tr><tr><td>GQ20+VQSR+MM80 </td><td>69 (61%) </td><td>44 (39%) </td><td>966 (50%) </td><td>953 (50%) </td><td>71 (31%) </td><td>157 (69%) </td><td>207 (10%) </td><td>1,802 (90%) </td><td>1,313 (31%) </td><td>2,956 (69%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>Capture efficiencies across filtered call sets (see Results for the definitions of the filter criteria) were inspected by counting the number of loci and probe sequences for which at least one quality SNP was detected. A success rate was obtained by dividing the number of successful probes by their totals in the initial design, i.e. 30,795 probe sequences, whether the source of design were mRNA (26,526 probes) or low-coverage WGS (4,269 probes). Conversely, a failure rate was defined as (1 − success rate). The Success/Failure rate was also stratified according the probe location in the genome assembly for predicted gene model features or intergenic region.</p></div></div></div></div></p>                            <h3 scrollto-destination=122670773 id="122670773" class="section-title" >3.5. Technical reproducibility and the performance of the sequence capture system for genotyping</h3>
<p>Sequence data generated in the partially replicated assay, named HIMP-2, produced 105,665,340 reads in total. Considering the sequencing intervals defined by the 14,135 probe sequences and 11,026 loci in this second assay (see Materials and Methods), 52,717,622 reads (∼50% of the total reads) and 75,394,548 reads (∼71% of the total reads) were aligned across on-target regions, respectively. After SNP detection using the improvement procedure for variant analysis, the VCF files contained records of 129,423 SNPs in HIMP-1 and a total of 175,361 in HIMP-2. The overall sensitivity—the fraction of ‘true called SNPs’ to ‘true SNPs’—was 61% while the precision—the fraction of ‘true called SNPs’ to ‘called SNPs’—was 83%. False positive (FP) rate, calculated as the FP/(FP + TN) ratio (TN is the true negative rate), was 8% while the overall accuracy was 81%. Contingency table values were summarized (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The true positive (TP) set of SNPs consisted of 93,441 variants with high confidence to the genotype calls between replicates (see <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S4</a></span>).</p><p>For the 14,135 probe sequences and 11,026 loci (47.5% of the 23,232 total targeted loci) evaluated across replicates, the overall performance of the capture system was 84% (8,315 out of 9,866) for the transcript-based probes and only 12% (524 out of 4,269) for the WGS-designed probes. Taking into account only the well-curated set of probes (GQ20+VQSR), the efficiency between replicates for transcript-based probes was 6,651 out of 6,843 (97%) and 430 out of 1,313 (33%) for probes from low-coverage WGS. Applying these results to the complete set of 24,751 probe sequences in the GQ20+VQSR set, we infer that <em>circa of</em> 21,570 (20,941 × 97% + 3,810 × 33%) probes constitute a highly efficient capture-based genotyping system for the species. Regarding the SNP call and genotyping confidence and considering the overall accuracy we estimated that the GQ20+VQSR probe set can sample nearly 204,000 true SNPs [93,441 × (23,232/11,026) × (21,570/24,751) × (1 + (1 − 81%))] across 18,216 loci (11.1 Mbp) in the genome assembly. Sample-to-sample variation in terms of read counts indicates that this outcome is influenced by the quality of the input DNA and sources of technical variation not evaluated here.</p>                            <h3 scrollto-destination=122670776 id="122670776" class="section-title" >3.6. Genomic variant annotation and functional effect prediction</h3>
<p>For the 18,216 genomic regions considered reliable analysis targets of this capture system, we produced an annotation of the genetic variants to improve the characterization of the recently reported genome assembly of <em>H. impetiginosus</em> and allow for further genetic research and candidate genes studies in this species. Sequence data in HIMP-1 and HIMP-2 capture assays were combined and processed in our pipeline (see Data and scripts availability) to compile a list of high quality SNP sites across the genome assembly. Given the high sequencing coverage acquired following the combined data sets of the two replicates, the number of quality SNPs found was 210,006, which agrees with our previous expectation of 204,000 SNPs based on the assessment of the sequence capture efficiency. The average sample call rate was 92% (82–99%) and the average SNP call rate was also 92% (62–100%). The 210,006 high-quality candidate SNPs were found in 14,499 genomic regions of 3,973 scaffolds in the assembly. These scaffolds encompass ∼290 Mbp of sequence in the 557 Mbp/1C genome of the species. SnpEff annotated 389,568 effects with the largest numbers of effects related to SNPs located in downstream or upstream gene regions (45.80%), exons (32.04%), introns and untranslated gene regions (16.06%) and intergenic regions far from genes (6.09%).</p>                            <h2 scrollto-destination=122670778 id="122670778" class="section-title" >4. Discussion</h2>
<p>We have described the development and evaluation of an initial collection of 30,795 probe sequences for SNP discovery and genotyping in the highly heterozygous plant genome <em>Handroanthus impetiginosus</em>. These probes were used as input to a commercially available system for custom target DNA enrichment followed by Illumina sequencing in a sample of 24 trees from six population groups spanning a wide geographic range of the species. With the aid of a genome assembly of a single tree, we performed extensive evaluation of the capture system to provide recommendations for future developments of similar plant genomes. It resulted in a final set of 24,751 capture probes covering 18,216 distinct loci, sampling more than 10 Mbp of the species genome. Variant analysis and SNP discovery allowed characterization of 210,006 SNPs and genotyping with overall level of missing data low for each site (&lt;20%) across samples. We investigated issues such as probe design, sequencing coverage and bioinformatics protocols including the evaluation of the capture efficiency and a reassessment of the technical reproducibility of the assay for SNPs recall and genotyping precision.</p><p>The methods described here for target enrichment share some common aspects to other genome complexity reduction approaches such as genotype by sequencing (GBS) and RAD sequencing and its variations as they also are intended to provide large numbers of SNPs at a lower cost per sample.<a reveal-id="dsy023-B37" data-open="dsy023-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsy023-B38" data-open="dsy023-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Several additional challenges are faced, however, by these alternative genome complexity reduction methods for direct SNP genotyping. The low sequencing coverage associated to the variable efficiency of DNA digestion results in inconsistent sampling of loci, seriously affecting reproducibility across experiments that tend to exacerbate in heterozygous genomes,<a reveal-id="dsy023-B39" data-open="dsy023-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> ultimately biasing diversity measures.<a reveal-id="dsy023-B40 dsy023-B41 dsy023-B42" data-open="dsy023-B40 dsy023-B41 dsy023-B42" class="link link-ref link-reveal xref-bibr"><sup>40–42</sup></a> The final number of robust SNPs assayed in large samples is typically only a small fraction of the initial set, defeating the alleged cost advantage.</p><p>A direct comparison of our sequence capture system with alternative genome complexity reduction methods for direct SNP genotyping is evidently beyond the scope of our research. Studies approaching this issue, however, pointed out that sequence capture consistently provided more reliable and portable data across experiments than RAD for shallow systematics, such as within species.<a reveal-id="dsy023-B43" data-open="dsy023-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Such observations have, indeed, lead to the recent development of combined protocols that ultimately benefit from the lower cost of RAD and the robustness of targeted capture.<a reveal-id="dsy023-B44" data-open="dsy023-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p><p>It should be said, however, that laboratory methods for capture systems involve decisions which also have the potential to significantly impact the capture efficiency outcome.<a reveal-id="dsy023-B45" data-open="dsy023-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Finally, while whole-genome re-sequencing analysis has also been trending for natural population studies, it is still largely cost-prohibitive for the vast majority of species that are investigated on very low budgets even those with relatively small genomes. Because we were aware of all these issues regarding the challenges of reduced complexity-based methods, we devised several strategies in our analytical pipeline to assess the quality of the sequence data generated in the assay, including a partial replication of the experiment.</p>                            <h3 scrollto-destination=122670783 id="122670783" class="section-title" >4.1 Quality of sequence data and challenges in bioinformatics</h3>
<p>The alignment of the sequencing reads obtained for each sample to the genome assembly is an important challenge. Accurate aligner algorithms such as BWA implement mismatch penalty scheme in which sequencing error rate is less than 10% (∼6%, for BWA). We observed a varying fraction of unmapped reads which ranged from 10% to nearly 40% for some individuals. These results indicated that genetic variants that distinguish the individuals’ genomes from the reference sequence may have not only caused reads to be misaligned but also led to a variable proportion of unmapped reads, resulting in a biased accounting of sequencing coverage across the genetically diverse populations of <em>H. impetiginosus</em>. As already noted in the literature, the inability to map reads is most likely due to structural rearrangements or insertions in the query genomes, or deletions in the reference.<a reveal-id="dsy023-B46" data-open="dsy023-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> For mapped reads, however, mappability within targeted regions was further evaluated using HCMappingQualityFilter in GATK HaplotypeCaller (-mmq 10, default value). This filter is applied as a prelude to variant calling to ensure that only reads that are likely to be informative will be used as evidence of possible variation. Overall good read mappability was achieved, with only 3–4% of the reads being discarded prior to variant calling.</p><p>Available algorithms for read mapping of divergent reads to a single reference, say up to 10–15%, have been developed such as Stampy.<a reveal-id="dsy023-B47" data-open="dsy023-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Recently developed bioinformatics tools aim to integrate multiple genomes of the same or closely related species into a single space-efficient representation of the graph structure and align read data to this complex structure.<a reveal-id="dsy023-B48 dsy023-B49 dsy023-B50 dsy023-B51" data-open="dsy023-B48 dsy023-B49 dsy023-B50 dsy023-B51" class="link link-ref link-reveal xref-bibr"><sup>48–51</sup></a> Unfortunately, these tools are not fully integrated in confident tools for SNP calling and genotyping such as the GATK-HaplotypeCaller framework which performs better with aligner tool less tolerable to divergent reads (up to 3–6%) such as BWA.<a reveal-id="dsy023-B52" data-open="dsy023-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a></p><p>Sequencing coverage contributes significantly to the total physical coverage of the targeted region and to an accuracy of sequence characterization of that region. At the single sample level, usually 150–200x of read depth is necessary to obtain over than 99.9% total coverage and accuracy across targets.<a reveal-id="dsy023-B53" data-open="dsy023-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> However, a depth of coverage as low as 1–2x has been found sufficient for accurate allele calls<a reveal-id="dsy023-B46" data-open="dsy023-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> to find common SNPs in a multi-sample sequencing design for a single species or closely related species. In a diploid organism, beginning with a 2x depth of coverage and dividing by the expected allele frequency to be observed (say 5%), the minimum depth necessary would be 40x. In one of our experiments, three of the 24 samples produced sequencing coverage of less than 40x possibly due to poor DNA quality or issues during library preparation.</p><p>We identified a set of targeted regions containing SNPs that consistently produced data in the sampled individuals with a capture efficiency of 60% or superior at appropriate coverage (&gt;70x). However, we also recognized in our pipeline that some unintended regions with high homology to the target sequence had been captured. In fact, the number of quality calls in the non-targeted regions was ∼35% of the total calls (134,126 out of 385,295) as assessed in the preliminary analysis of variation. This indicates that the protocol used for target enrichment and sequence capture can generate a considerable number of reads that maps outside the targeted regions, which limits the degree of sample multiplexing per NGS unit run and also represents a challenge to the read alignment process.<a reveal-id="dsy023-B53" data-open="dsy023-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Nevertheless, while off-target read mapping does represent a challenge to SNP callers it can also be viewed as an opportunity to extend genotyping beyond the designed probe sequences.</p><p>These off-target variants can be due to many different factors depending on the capture platform, such as the quantity of input DNA, the degree of the target DNA entanglement with other strands during isolation for library preparation, and the ability of the probes to capture DNA segments that are much larger than their desired length based on the probe sequence. These unintended segments may contain repetitive sequences that can be present in many copies resulting in self-priming events during any subsequent amplification step. We speculate that the fraction of calls in off-target regions in our study was most likely caused by the relatively simplified capture and multiplexing system used by the custom multiplexed Capture-Seq method used for DNA enrichment and indexing sequencing libraries.<a reveal-id="dsy023-B19" data-open="dsy023-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Designed to be a fast, low-cost and high throughput method, it does not include fragment size selection and the use of blocking oligos to prevent the enrichment of non-specific targets in the custom enrichment, which are steps commonly used in extended but significantly more expensive protocols. Noteworthy, our first capture assay resulted in 40% of total reads mapping on-target related to the probe sequence coordinates. A partially replicated assay resulted in 50% of total reads mapping on-target. However, with an appropriate definition of target loci, which include 200 bp in each direction from the probe sequence coordinates, the amount of on-target read relative to the total generated reads can reach 60–70%. Given the low cost protocol and a customized probe set for a significantly more heterozygous genome, it is satisfactorily competitive with more refined commercial capture probe assays in humans which report average of on-target reads within 68.4% ± 7.8% (<span class="small-caps">s</span>.<span class="small-caps">d</span>.) such as the NimbleGen SeqCap EZ (Roche, Basel, Switzerland).<a reveal-id="dsy023-B54" data-open="dsy023-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a></p>                            <h3 scrollto-destination=122670789 id="122670789" class="section-title" >4.2. The challenge of designing a benchmark strategy to assesses the performance outcome of the custom probe set</h3>
<p>We observed that the bioinformatics analysis of the sequence data is another important challenge. Caution is needed to adequately remove probe sequences improperly designed or unreliable for SNP detection. Typically, the design of overlapping sequences tiled across the entire transcript sequences was much satisfactory. Sequences spanning across exon boundaries cannot work well, but they will have neighboring probes which will still give quality data. However, any short exons (below the probe sequence length) may not be recoverable unless they can be ‘padded’ with intronic sequence. The computational prediction of gene models from a preliminary genome assembly provides the adequate resource to target these regions. Furthermore, because the experimental conditions of the genotyping assay cannot be directly predicted, we recommend an evaluation across replicates to assess the overall accuracy in the performance outcome of the custom probe set. In our work, with a careful bioinformatics protocol for variant analysis we could recognize 4,344 probes that were deemed faulty or improper. Additionally, we inferred from a partial replication that another 4,881 probes contributed negatively to the measures of performance of a binary classification test of success/failure to detect the true genotype of samples.</p><p>In terms of the number of distinct sampled loci, both methods used for probe design led to confident SNP calls in regions flanking the exons in transcripts indicating that the whole transcript can be targeted. Intergenic regions will produce reliable detection of true variants, but can introduce penalties in terms of missing data when the sampled individuals are very diverse. Limited read alignment to a single reference genome due to high rates and/or different patterns of sequence polymorphism between individuals can lead to biased accounting of sequencing coverage around the intergenic, UTRs and introns regions, which can result in inflation of missing data at shared sites when genotyping. Noteworthy, we observe that target capture systems can benefit of alternative approaches for probe capture design that directly use transcript sequences, instead of limiting the design to longer exons as is often the choice.</p><p>Based on these assessments, we estimated that the efficiency of our custom capture system to reveal high-confidence SNP calls for reliable genotyping in populations of <em>H. impetiginosus</em> from which the samples were drawn is 70% (21,570 successful capture probe sequences out of a total of 30, 795 designed probes). To get higher efficiencies, an interested user can simply use the recommended subset of probes in the VQSR+GQ20 set (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>). In this case, under the expectation of the performance evaluation, the percentage of probe regions that may be produce aligned read depth greater than 40x will be ∼87.1% (21,570 probe regions in 24,751). This is within the same order of magnitude, for instance, of the performance of a sequence capture system targeting the gene space of another highly heterozygous forest trees. In <em>Populus trichocarpa</em> and <em>Pinus taeda</em>, 86.8% and 79.1% of the probe regions were on average sequenced at a depth ≥10x.<a reveal-id="dsy023-B18" data-open="dsy023-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy023-B19" data-open="dsy023-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p>                            <h3 scrollto-destination=122670793 id="122670793" class="section-title" >4.3. Missing data at SNPs detected in the capture system</h3>
<p>A common issue in studies attempting to use capture probes on divergent genomes is the amount of missing data at any shared site in a diverse sample of individuals. In plants, for instance, a probe set developed based on transcript sequences from loblolly pine (<em>Pinus taeda</em>) was used for DNA enrichment across targeted sequences in 48 individuals of whitebark pine (<em>Pinus albicaulis</em>), resulting in the identification of 528,873 SNPs in 7,849 transcripts.<a reveal-id="dsy023-B55" data-open="dsy023-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> In that study, data on genotypes at SNPs was found useful only for 12,390 segregating sites (2% out of 528,873 identified SNPs) in 4,452 transcripts (56.7% out of 7,849). In our study, missing data for any site was concentrated in particular SNPs and populations, suggesting a significant underlying population structure likely leading to an important impact on the sharing of SNP sites. In fact, the SNP data clearly recovered patterns of structure in the populations (see details in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_figure_s1.doc?Expires=2147483647&amp;Signature=a0ulO-6~sy0TzuDbimPrlijLIVfaZhA7isPmaCcqGR3hLauLgD-A8wtIYwID-VRvpBANIHaKFdZiF8drZdKOXCKLTFAJUToGBAswMRDNCZlbgLwKeDZGMFeaWjyWDXHD7QFcb8IY2SDjvL~GpCsu15xsALVfvEDRDKglwfZ9Ox0DF~K~lIMc-XA3dA7VzgQMSG1-GjuLCpjcA9WKmZh1LTapS2wT6h~leDc-WpJBPcRAtGDGmizUr9nXRksyEpxj3WplY3-WebxszJt8hsbQfyz9dGX6-x1tRXbmYjw~seBBKvi9xVHTIDbFLWt-u~PB3pX9YOEwChgG5kISXQP~Bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1B and C</a></span>). Besides structure in the data, we could also observe that the patterns of polymorphisms in the sampled individuals differ from a single reference genome at different rates across intergenic, transcribed or protein-coding regions. It is an expected outcome since population-level whole-genome resequencing has revealed that diversity levels in large outbred population—assessed by different measures of nucleotide diversity—within stretches of DNA transcribed into a mRNA molecule is reduced in comparison to intergenic DNA. A recent study in outbred bird species,<a reveal-id="dsy023-B56" data-open="dsy023-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> for instance, reported that diversity is related to selective constraint such that in comparison with intergenic DNA, diversity at fourfold degenerate sites was reduced to 85%, 3′ UTRs to 82%, 5′ UTRs to 70% and non-degenerate sites to 12%. Other studies including outbred tree species such as poplars<a reveal-id="dsy023-B57" data-open="dsy023-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> have shown that, among various genomic contexts, the levels of nucleotide diversity were highest at intergenic sites, followed by fourfold synonymous sites, 3′ UTRs, 5′ UTRs, and introns and were the lowest at 0-fold degenerate sites.</p>                            <h3 scrollto-destination=122670795 id="122670795" class="section-title" >4.4. Requirements and recommendations</h3>
<p>Given the per-sample sequencing coverage (i.e. the ratio between the number of quality reads after alignment to a genome and the number of loci) greater than 40x, capture efficiencies &gt;60% are expected and genotype call rates at SNPs may be ≥92%. Our results suggest that the successful use of this sequence capture system would encompass the generation of &gt;3.9 million of 100 bp single-end reads per-sample across the 18,216 targeted loci defined by location of the recommended 24,771 probe sequences in the call set ‘GQ20+VQSR’. This lower-bound value was adjusted considering the observed ratio between on-target and off-target read alignments (35%, in our data). Adjustment has also been performed due to the limited read alignment to the reference assembly (60–90%, in our data), because of structural rearrangements or insertions in the query genome, or deletions in the reference assembly due to divergent evolutionary history of the population to which the sample belongs. The bioinformatics protocol for data analysis is standard for analysis of variation using Illumina’s NGS technology and reference-based methods of SNP identification and involves the steps summarized in <a reveal-id="dsy023-F1" data-open="dsy023-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> and detailed in the <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>. It includes the target loci definition, the creation of a database of variant resources and the likelihoods that these variants exist and the recalibration for all evaluated variants.</p>                            <h3 scrollto-destination=122670797 id="122670797" class="section-title" >4.5. Prospects of the sequence capture system for population genetic studies</h3>
<p>We used the genotypic data on the cataloged 210,006 SNP from the two genotyping assays using our custom probe set to look at some population features as a preamble to population genomics studies in <em>H. impetiginosus.</em> From a purely observational standpoint, due to the small number of sampled individuals, the shape of the SFS distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_figure_s1.doc?Expires=2147483647&amp;Signature=a0ulO-6~sy0TzuDbimPrlijLIVfaZhA7isPmaCcqGR3hLauLgD-A8wtIYwID-VRvpBANIHaKFdZiF8drZdKOXCKLTFAJUToGBAswMRDNCZlbgLwKeDZGMFeaWjyWDXHD7QFcb8IY2SDjvL~GpCsu15xsALVfvEDRDKglwfZ9Ox0DF~K~lIMc-XA3dA7VzgQMSG1-GjuLCpjcA9WKmZh1LTapS2wT6h~leDc-WpJBPcRAtGDGmizUr9nXRksyEpxj3WplY3-WebxszJt8hsbQfyz9dGX6-x1tRXbmYjw~seBBKvi9xVHTIDbFLWt-u~PB3pX9YOEwChgG5kISXQP~Bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>) may suggest a pattern consistent with incomplete lineage sorting and/or mutation bias among the sampled sub-groups. These preliminary results are consistent with previous phylogenetic analyses showing evidences of incomplete lineage sorting in <em>Handroanthus</em> species across populations in Central Brazil but without recovering the shared ancestral haplotypes between them.<a reveal-id="dsy023-B23" data-open="dsy023-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsy023-B58" data-open="dsy023-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> If there is an underlying structure in the data (see <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_figure_s1.doc?Expires=2147483647&amp;Signature=a0ulO-6~sy0TzuDbimPrlijLIVfaZhA7isPmaCcqGR3hLauLgD-A8wtIYwID-VRvpBANIHaKFdZiF8drZdKOXCKLTFAJUToGBAswMRDNCZlbgLwKeDZGMFeaWjyWDXHD7QFcb8IY2SDjvL~GpCsu15xsALVfvEDRDKglwfZ9Ox0DF~K~lIMc-XA3dA7VzgQMSG1-GjuLCpjcA9WKmZh1LTapS2wT6h~leDc-WpJBPcRAtGDGmizUr9nXRksyEpxj3WplY3-WebxszJt8hsbQfyz9dGX6-x1tRXbmYjw~seBBKvi9xVHTIDbFLWt-u~PB3pX9YOEwChgG5kISXQP~Bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1B and C</a></span>) and if genes did not coalesce at the same time that the sub-groups delimitation occurred in a recent past, differences in the allele frequency distribution can be observed on the entire SFS in comparison with the expected distribution under mutation-drift equilibrium. Thus, the small number of generations that have passed until sub-group delimitation may have contributed in maintaining larger than expected SNPs at intermediate to high frequency. Based on these observations, we expect that this capture system along with available tools for phase and haplotype identification should provide the necessary framework to provide more definitive evidences in support of these findings.</p><p>In conclusion, we have reported a targeted SNP resource for <em>Handroanthus impetiginosus</em>, a highly valued tree, with a notable ecological keystone status in SDTFs of South America and Mesoamerica. The successful use of this 24,751 capture probes system requires the generation of ∼4 million of 100 bp single-end reads per-sample to provide robust genotyping across 18,216 distinct loci sampling more than 10 Mbp of the species genome. This system identifies nearly 200,000 SNPs located inside or in close proximity to almost 14,000 annotated protein-coding genes, generating quality genotypic data in populations spanning wide geographic distances across the species native range. Prospects of population genetic studies indicate that this platform can be used to estimate population genetics parameters and carry on investigations of the interplay of ecology and evolution at the genome-wide scale. We emphasize though that the preliminary population-level analyses presented here with a limited sample of individuals was only meant to provide a first exploratory assessment on the evolution of genome sequences in <em>H. impetiginosus</em>, and by no means we imply that these analyses are representative but rather observational. To that end, we are currently using this capture system to characterize polymorphisms in a follow-up study with a larger set of individuals sampled in 13 demographic groups across the Brazilian territory. Going beyond the significance of these results for the species, this study paves the way for the development of similar genomic resources for other Neotropical forest trees of equivalent or higher relevance. This in turn will open exceptional prospects to advance the understanding of the evolutionary history, species distribution and population demography of the still largely neglected forest trees of the mega diverse tropical biomes.</p>                            <h2 scrollto-destination=122670800 id="122670800" class="section-title" >5. Data and scripts availability</h2>
<p>Sequences and corresponding protein-coding gene annotations for the genome assembly of <em>H. impetiginosus</em> have been deposited into GenBank, accession <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/NKXS00000000.1" target="_blank">NKXS00000000.1</a>. Raw sequencing reads produced in the Capture-Seq experiments HIMP-1 and HIMP-2 are available into SRA under accession numbers <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR6369264-SRR6369287" target="_blank">SRR6369264-SRR6369287</a> and <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR6369569-SRR6369592" target="_blank">SRR6369569-SRR6369592</a>, respectively. BioSample descriptions and all available data are accessible from the BioProject PRJNA324125. Scripts used to process the sequence data and to generate the SNP calls and genotypes across the validated loci in the species genome are available from <a class="link link-uri" href="https://github.com/biozzyn/handroanthus-variant-analysis" target="_blank">https://github.com/biozzyn/handroanthus-variant-analysis</a>.</p>                            <h2 scrollto-destination=122670802 id="122670802" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by competitive grants from CNPq to R.G.C. (project no. 471366/2007-2 and Rede Cerrado CNPq/PPBio project no. 457406/2012-7), to E.N. (CNPq Proc. 476709/2012-1) and to D.G. (PRONEX FAP-DF Project Grant “NEXTREE” 193.000.570/2009). R.G.C. and D.G. have been supported by productivity grants from CNPq, which we gratefully acknowledge. O.B.S. has been supported by an EMBRAPA doctoral fellowship.</p>                            <h3 scrollto-destination=122670804 id="122670804" class="section-title" >Accession numbers</h3>
<p>GenBank <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/NKXS00000000.1" target="_blank">NKXS00000000.1</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRA SRR6369264-SRR6369287" target="_blank">SRA SRR6369264-SRR6369287</a>, SRA <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR6369569-SRR6369592" target="_blank">SRR6369569-SRR6369592</a></p>                            <h3 scrollto-destination=122670806 id="122670806" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=122670808 id="122670808" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy023-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stapley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Reger</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Feulner</div> <div class="given-names">P. G. D.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Adaptation genomics: the next generation</div>, <div class="source">Trends Ecol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">705</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20genomics%3A%20the%20next%20generation&amp;author=J.%20Stapley&amp;author=J.%20Reger&amp;author=P.%20G.%20D.%20Feulner&amp;publication_year=2010&amp;journal=Trends%20Ecol.%20Evol&amp;volume=25&amp;pages=705-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2010.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2010.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2010.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20952088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pool</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Hellmann</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Population genetic inference from genomic sequence variation</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">291</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genetic%20inference%20from%20genomic%20sequence%20variation&amp;author=J.%20E.%20Pool&amp;author=I.%20Hellmann&amp;author=J.%20D.%20Jensen&amp;author=R.%20Nielsen&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=291-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.079509.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.079509.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.079509.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20067940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morin</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Luikart</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wayne</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Grp</div> <div class="given-names">S. W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">SNPs in ecology, evolution and conservation</div>, <div class="source">Trends Ecol. Evol</div> ., <div class="volume">19</div>, <div class="fpage">208</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNPs%20in%20ecology%2C%20evolution%20and%20conservation&amp;author=P.%20A.%20Morin&amp;author=G.%20Luikart&amp;author=R.%20K.%20Wayne&amp;author=S.%20W.%20Grp&amp;publication_year=2004&amp;journal=Trends%20Ecol.%20Evol&amp;volume=19&amp;pages=208-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2004.01.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2004.01.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2004.01.009"> </span></div> </div></div></div></div></div><div content-id="dsy023-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tiffin</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ross-Ibarra</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Advances and limits of using population genetics to understand local adaptation</div>, <div class="source">Trends Ecol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">673</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20and%20limits%20of%20using%20population%20genetics%20to%20understand%20local%20adaptation&amp;author=P.%20Tiffin&amp;author=J.%20Ross-Ibarra&amp;publication_year=2014&amp;journal=Trends%20Ecol.%20Evol&amp;volume=29&amp;pages=673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2014.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2014.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2014.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25454508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sachidanandam</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">S. C.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms</div>, <div class="source">Nature</div> , <div class="volume">409</div>, <div class="fpage">928</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map%20of%20human%20genome%20sequence%20variation%20containing%201.42%20million%20single%20nucleotide%20polymorphisms&amp;author=R.%20Sachidanandam&amp;author=D.%20Weissman&amp;author=S.%20C.%20Schmidt&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=928-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nordborg</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">T. T.</div></div>, <div class="name string-name"><div class="surname">Ishino</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">The pattern of polymorphism in Arabidopsis thaliana</div>, <div class="source">PLoS Biol</div> ., <div class="volume">3</div>, <div class="fpage">e196</div>–<div class="lpage">1299</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pattern%20of%20polymorphism%20in%20Arabidopsis%20thaliana&amp;author=M.%20Nordborg&amp;author=T.%20T.%20Hu&amp;author=Y.%20Ishino&amp;publication_year=2005&amp;journal=PLoS%20Biol&amp;volume=3&amp;pages=e196-1299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15907155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">Y. J.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">J. P.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Development of genome-wide DNA polymorphism database for map-based cloning of rice genes</div>, <div class="source">Plant Physiol</div> ., <div class="volume">135</div>, <div class="fpage">1198</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genome-wide%20DNA%20polymorphism%20database%20for%20map-based%20cloning%20of%20rice%20genes&amp;author=Y.%20J.%20Shen&amp;author=H.%20Jiang&amp;author=J.%20P.%20Jin&amp;publication_year=2004&amp;journal=Plant%20Physiol&amp;volume=135&amp;pages=1198-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.038463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.038463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.038463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15266053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morrell</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Buckler</div> <div class="given-names">E. S.</div></div>, <div class="name string-name"><div class="surname">Ross-Ibarra</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Crop genomics: advances and applications</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">13</div>, <div class="fpage">85</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20genomics%3A%20advances%20and%20applications&amp;author=P.%20L.%20Morrell&amp;author=E.%20S.%20Buckler&amp;author=J.%20Ross-Ibarra&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=13&amp;pages=85-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22207165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Glaszmann</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Leung</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ribaut</div> <div class="given-names">J. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">More genomic resources for less-studied crops</div>, <div class="source">Trends Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">452</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=More%20genomic%20resources%20for%20less-studied%20crops&amp;author=R.%20K.%20Varshney&amp;author=J.%20C.%20Glaszmann&amp;author=H.%20Leung&amp;author=J.%20M.%20Ribaut&amp;publication_year=2010&amp;journal=Trends%20Biotechnol&amp;volume=28&amp;pages=452-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2010.06.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2010.06.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2010.06.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20692061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Neale</div> <div class="given-names">D. B.</div></div>, <div class="name string-name"><div class="surname">Kremer</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Forest tree genomics: growing resources and applications</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">111</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forest%20tree%20genomics%3A%20growing%20resources%20and%20applications&amp;author=D.%20B.%20Neale&amp;author=A.%20Kremer&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=111-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2931" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2931"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21245829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davey</div> <div class="given-names">J. W.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Boone</div> <div class="given-names">J. Q.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M. L.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">499</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.%20W.%20Davey&amp;author=P.%20A.%20Hohenlohe&amp;author=P.%20D.%20Etter&amp;author=J.%20Q.%20Boone&amp;author=J.%20M.%20Catchen&amp;author=M.%20L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mamanova</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Coffey</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Scott</div> <div class="given-names">C. E.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Target-enrichment strategies for next-generation sequencing</div>, <div class="source">Nat. Methods</div> ., <div class="volume">7</div>, <div class="fpage">111</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target-enrichment%20strategies%20for%20next-generation%20sequencing&amp;author=L.%20Mamanova&amp;author=A.%20J.%20Coffey&amp;author=C.%20E.%20Scott&amp;publication_year=2010&amp;journal=Nat.%20Methods&amp;volume=7&amp;pages=111-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20111037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gasc</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Peyretaillade</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Peyret</div> <div class="given-names">P.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Sequence capture by hybridization to explore modern and ancient genomic diversity in model and nonmodel organisms</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">4504</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20capture%20by%20hybridization%20to%20explore%20modern%20and%20ancient%20genomic%20diversity%20in%20model%20and%20nonmodel%20organisms&amp;author=C.%20Gasc&amp;author=E.%20Peyretaillade&amp;author=P.%20Peyret&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=4504-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27105841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gnirke</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Melnikov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Maguire</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">27</div>, <div class="fpage">182</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;author=A.%20Gnirke&amp;author=A.%20Melnikov&amp;author=J.%20Maguire&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol&amp;volume=27&amp;pages=182-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walsh</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Casadei</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Detection of inherited mutations for breast and ovarian cancer using genomic capture and massively parallel sequencing</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">107</div>, <div class="fpage">12629</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20inherited%20mutations%20for%20breast%20and%20ovarian%20cancer%20using%20genomic%20capture%20and%20massively%20parallel%20sequencing&amp;author=T.%20Walsh&amp;author=M.%20K.%20Lee&amp;author=S.%20Casadei&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=107&amp;pages=12629-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1007983107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1007983107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1007983107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20616022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Springer</div> <div class="given-names">N. M.</div></div>, <div class="name string-name"><div class="surname">Gerhardt</div> <div class="given-names">D. J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Repeat subtraction-mediated sequence capture from a complex genome</div>, <div class="source">Plant J</div> ., <div class="volume">62</div>, <div class="fpage">898</div>–<div class="lpage">909</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeat%20subtraction-mediated%20sequence%20capture%20from%20a%20complex%20genome&amp;author=Y.%20Fu&amp;author=N.%20M.%20Springer&amp;author=D.%20J.%20Gerhardt&amp;publication_year=2010&amp;journal=Plant%20J&amp;volume=62&amp;pages=898-909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.62.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.62.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.62.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20230488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dasgupta</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Dharanishanthi</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Agarwal</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Krutovsky</div> <div class="given-names">K. V.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Development of genetic markers in eucalyptus species by target enrichment and exome sequencing</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0116528</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genetic%20markers%20in%20eucalyptus%20species%20by%20target%20enrichment%20and%20exome%20sequencing&amp;author=M.%20G.%20Dasgupta&amp;author=V.%20Dharanishanthi&amp;author=I.%20Agarwal&amp;author=K.%20V.%20Krutovsky&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0116528" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0116528" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0116528" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0116528"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25602379" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Holliday</div> <div class="given-names">J. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Targeted enrichment of the black cottonwood (<em>Populus trichocarpa</em>) gene space using sequence capture</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20enrichment%20of%20the%20black%20cottonwood%20(Populus%20trichocarpa)%20gene%20space%20using%20sequence%20capture&amp;author=L.%20Zhou&amp;author=J.%20A.%20Holliday&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-703" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-703" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-703"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23241106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Neves</div> <div class="given-names">L. G.</div></div>, <div class="name string-name"><div class="surname">Davis</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Barbazuk</div> <div class="given-names">W. B.</div></div>, <div class="name string-name"><div class="surname">Kirst</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Whole-exome targeted sequencing of the uncharacterized pine genome</div>, <div class="source">Plant J</div> ., <div class="volume">75</div>, <div class="fpage">146</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-exome%20targeted%20sequencing%20of%20the%20uncharacterized%20pine%20genome&amp;author=L.%20G.%20Neves&amp;author=J.%20M.%20Davis&amp;author=W.%20B.%20Barbazuk&amp;author=M.%20Kirst&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=75&amp;pages=146-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.12193" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12193" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.12193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23551702" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Good</div> <div class="given-names">J. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Targeted capture in evolutionary and ecological genomics</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">25</div>, <div class="fpage">185</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20capture%20in%20evolutionary%20and%20ecological%20genomics&amp;author=M.%20R.%20Jones&amp;author=J.%20M.%20Good&amp;publication_year=2016&amp;journal=Mol.%20Ecol&amp;volume=25&amp;pages=185-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.13304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.13304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.13304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26137993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nicholls</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pennington</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Koenen</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Using targeted enrichment of nuclear genes to increase phylogenetic resolution in the Neotropical rain forest genus Inga (Leguminosae: Mimosoideae)</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20targeted%20enrichment%20of%20nuclear%20genes%20to%20increase%20phylogenetic%20resolution%20in%20the%20Neotropical%20rain%20forest%20genus%20Inga%20(Leguminosae%3A%20Mimosoideae)&amp;author=J.%20Nicholls&amp;author=R.%20Pennington&amp;author=E.%20Koenen&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00710" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00710"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26442024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Prado</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Gibbs</div> <div class="given-names">P. E.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Patterns of species distributions in the dry seasonal forests of South America</div>, <div class="source">Ann. Missouri Bot. Gard</div> ., <div class="volume">80</div>, <div class="fpage">902</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20species%20distributions%20in%20the%20dry%20seasonal%20forests%20of%20South%20America&amp;author=D.%20E.%20Prado&amp;author=P.%20E.%20Gibbs&amp;publication_year=1993&amp;journal=Ann.%20Missouri%20Bot.%20Gard&amp;volume=80&amp;pages=902-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2399937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2399937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2399937"> </span></div> </div></div></div></div></div><div content-id="dsy023-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Collevatti</div> <div class="given-names">R. G.</div></div>, <div class="name string-name"><div class="surname">Terribile</div> <div class="given-names">L. C.</div></div>, <div class="name string-name"><div class="surname">Lima-Ribeiro</div> <div class="given-names">M. S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A coupled phylogeographical and species distribution modelling approach recovers the demographical history of a Neotropical seasonally dry forest tree species</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">21</div>, <div class="fpage">5845</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20coupled%20phylogeographical%20and%20species%20distribution%20modelling%20approach%20recovers%20the%20demographical%20history%20of%20a%20Neotropical%20seasonally%20dry%20forest%20tree%20species&amp;author=R.%20G.%20Collevatti&amp;author=L.%20C.%20Terribile&amp;author=M.%20S.%20Lima-Ribeiro&amp;publication_year=2012&amp;journal=Mol.%20Ecol&amp;volume=21&amp;pages=5845-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23094833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maddison</div> <div class="given-names">W. P.</div></div>, <div class="name string-name"><div class="surname">Knowles</div> <div class="given-names">L. L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Inferring phylogeny despite incomplete lineage sorting</div>, <div class="source">Syst. Biol</div> ., <div class="volume">55</div>, <div class="fpage">21</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20phylogeny%20despite%20incomplete%20lineage%20sorting&amp;author=W.%20P.%20Maddison&amp;author=L.%20L.%20Knowles&amp;publication_year=2006&amp;journal=Syst.%20Biol&amp;volume=55&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150500354928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150500354928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150500354928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16507521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schulze</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Grogan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Uhl</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lentini</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Vidal</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Evaluating ipe (Tabebuia, Bignoniaceae) logging in Amazonia: sustainable management or catalyst for forest degradation?</div> <div class="source">Biol. Conserv</div> ., <div class="volume">141</div>, <div class="fpage">2071</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluating%20ipe%20(Tabebuia%2C%20Bignoniaceae)%20logging%20in%20Amazonia%3A%20sustainable%20management%20or%20catalyst%20for%20forest%20degradation%3F&amp;author=M.%20Schulze&amp;author=J.%20Grogan&amp;author=C.%20Uhl&amp;author=M.%20Lentini&amp;author=E.%20Vidal&amp;publication_year=2008&amp;journal=Biol.%20Conserv&amp;volume=141&amp;pages=2071-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biocon.2008.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biocon.2008.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biocon.2008.06.003"> </span></div> </div></div></div></div></div><div content-id="dsy023-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shafer</div> <div class="given-names">A. B. A.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">J. B. W.</div></div>, <div class="name string-name"><div class="surname">Alves</div> <div class="given-names">P. C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genomics and the challenging translation into conservation practice</div>, <div class="source">Trends Ecol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">78</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20and%20the%20challenging%20translation%20into%20conservation%20practice&amp;author=A.%20B.%20A.%20Shafer&amp;author=J.%20B.%20W.%20Wolf&amp;author=P.%20C.%20Alves&amp;publication_year=2015&amp;journal=Trends%20Ecol.%20Evol&amp;volume=30&amp;pages=78-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2014.11.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2014.11.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2014.11.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25534246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Silva-Junior</div> <div class="given-names">O. B.</div></div>, <div class="name string-name"><div class="surname">Grattapaglia</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Novaes</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Collevatti</div> <div class="given-names">R. G.</div></div></div> <div class="year">2018</div>, 
<div class="article-title">Genome assembly of the Pink Ipê (<em>Handroanthus impetiginosus</em>, Bignoniaceae), a highly-valued ecologically keystone Neotropical timber forest tree and, a natural product producer</div>. <div class="source">Gigascience</div> , <div class="volume">7</div>, <div class="fpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20assembly%20of%20the%20Pink%20Ip%C3%AA%20(Handroanthus%20impetiginosus%2C%20Bignoniaceae)%2C%20a%20highly-valued%20ecologically%20keystone%20Neotropical%20timber%20forest%20tree%20and%2C%20a%20natural%20product%20producer&amp;author=O.%20B.%20Silva-Junior&amp;author=D.%20Grattapaglia&amp;author=E.%20Novaes&amp;author=R.%20G.%20Collevatti&amp;publication_year=2018&amp;journal=Gigascience&amp;volume=7&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gigascience/gix125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgigascience%2Fgix125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgigascience%2Fgix125"> </span></div> </div></div></div></div></div><div content-id="dsy023-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DePristo</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.%20A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iqbal</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Caccamo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Flicek</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">McVean</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">De novo assembly and genotyping of variants using colored de Bruijn graphs</div>, <div class="source">Nat. Genet</div> ., <div class="volume">44</div>, <div class="fpage">226</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20genotyping%20of%20variants%20using%20colored%20de%20Bruijn%20graphs&amp;author=Z.%20Iqbal&amp;author=M.%20Caccamo&amp;author=I.%20Turner&amp;author=P.%20Flicek&amp;author=G.%20McVean&amp;publication_year=2012&amp;journal=Nat.%20Genet&amp;volume=44&amp;pages=226-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.1028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.1028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.1028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22231483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="collab">ICGMC</div> <div class="year">2015</div>, 
<div class="article-title">High-resolution linkage map and chromosome-scale genome assembly for Cassava (<em>Manihot esculenta</em> Crantz) from 10 populations</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">5</div>, <div class="fpage">133</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.015008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.015008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.015008"> </span></div> </div></div></div></div></div><div content-id="dsy023-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McCormick</div> <div class="given-names">R. F.</div></div>, <div class="name string-name"><div class="surname">Truong</div> <div class="given-names">S. K.</div></div>, <div class="name string-name"><div class="surname">Mullet</div> <div class="given-names">J. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">RIG: recalibration and Interrelation of Genomic Sequence Data with the GATK</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">5</div>, <div class="fpage">655</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RIG%3A%20recalibration%20and%20Interrelation%20of%20Genomic%20Sequence%20Data%20with%20the%20GATK&amp;author=R.%20F.%20McCormick&amp;author=S.%20K.%20Truong&amp;author=J.%20E.%20Mullet&amp;publication_year=2015&amp;journal=G3%20(Bethesda)&amp;volume=5&amp;pages=655-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.115.017012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.115.017012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.115.017012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25681258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cingolani</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Platts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L. L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff</div>, <div class="source">Fly</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20L.%20Wang&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chang</div> <div class="given-names">C. C.</div></div>, <div class="name string-name"><div class="surname">Chow</div> <div class="given-names">C. C.</div></div>, <div class="name string-name"><div class="surname">Tellier</div> <div class="given-names">L. C. A. M.</div></div>, <div class="name string-name"><div class="surname">Vattikuti</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Purcell</div> <div class="given-names">S. M.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</div>, <div class="source">Gigascience</div> , <div class="volume">4</div>, <div class="fpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;author=C.%20C.%20Chang&amp;author=C.%20C.%20Chow&amp;author=L.%20C.%20A.%20M.%20Tellier&amp;author=S.%20Vattikuti&amp;author=S.%20M.%20Purcell&amp;author=J.%20J.%20Lee&amp;publication_year=2015&amp;journal=Gigascience&amp;volume=4&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13742-015-0047-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13742-015-0047-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13742-015-0047-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25859341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watterson</div> <div class="given-names">G. A.</div></div></div> <div class="year">1975</div>, 
<div class="article-title">On the number of segregating sites in genetical models without recombination</div>, <div class="source">Theor. Popul. Biol</div> ., <div class="volume">7</div>, <div class="fpage">256</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetical%20models%20without%20recombination&amp;author=G.%20A.%20Watterson&amp;publication_year=1975&amp;journal=Theor.%20Popul.%20Biol&amp;volume=7&amp;pages=256-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0040-5809(75)90020-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0040-5809(75)90020-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1145509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baird</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Atwood</div> <div class="given-names">T. S.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elshire</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A robust, simple genotyping-by-sequencing (GbS) approach for high diversity species</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GbS)%20approach%20for%20high%20diversity%20species&amp;author=R.%20J.%20Elshire&amp;author=J.%20C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Myles</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Improving fruit and wine: what does genomics have to offer?</div> <div class="source">Trends Genet</div> ., <div class="volume">29</div>, <div class="fpage">190</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20fruit%20and%20wine%3A%20what%20does%20genomics%20have%20to%20offer%3F&amp;author=S.%20Myles&amp;publication_year=2013&amp;journal=Trends%20Genet&amp;volume=29&amp;pages=190-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2013.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2013.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2013.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23428114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowry David</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hoban</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kelley Joanna</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Breaking RAD: an evaluation of the utility of restriction site‐associated DNA sequencing for genome scans of adaptation</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">17</div>, <div class="fpage">142</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Breaking%20RAD%3A%20an%20evaluation%20of%20the%20utility%20of%20restriction%20site%E2%80%90associated%20DNA%20sequencing%20for%20genome%20scans%20of%20adaptation&amp;author=B.%20Lowry%20David&amp;author=S.%20Hoban&amp;author=L.%20Kelley%20Joanna&amp;publication_year=2016&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=17&amp;pages=142-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1755-0998.12635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1755-0998.12635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1755-0998.12635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27860289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cezard</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The effect of RAD allele dropout on the estimation of genetic variation within and between populations</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">3165</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20RAD%20allele%20dropout%20on%20the%20estimation%20of%20genetic%20variation%20within%20and%20between%20populations&amp;author=M.%20Gautier&amp;author=K.%20Gharbi&amp;author=T.%20Cezard&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=3165-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23110526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arnold</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Corbett-Detig</div> <div class="given-names">R. B.</div></div>, <div class="name string-name"><div class="surname">Hartl</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bomblies</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">3179</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RADseq%20underestimates%20diversity%20and%20introduces%20genealogical%20biases%20due%20to%20nonrandom%20haplotype%20sampling&amp;author=B.%20Arnold&amp;author=R.%20B.%20Corbett-Detig&amp;author=D.%20Hartl&amp;author=K.%20Bomblies&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=3179-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12276" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12276" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12276"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23551379" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harvey</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">B. T.</div></div>, <div class="name string-name"><div class="surname">Glenn</div> <div class="given-names">T. C.</div></div>, <div class="name string-name"><div class="surname">Faircloth</div> <div class="given-names">B. C.</div></div>, <div class="name string-name"><div class="surname">Brumfield</div> <div class="given-names">R. T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Sequence capture versus restriction site associated DNA sequencing for shallow systematics</div>, <div class="source">Syst. Biol</div> ., <div class="volume">65</div>, <div class="fpage">910</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20capture%20versus%20restriction%20site%20associated%20DNA%20sequencing%20for%20shallow%20systematics&amp;author=M.%20G.%20Harvey&amp;author=B.%20T.%20Smith&amp;author=T.%20C.%20Glenn&amp;author=B.%20C.%20Faircloth&amp;author=R.%20T.%20Brumfield&amp;publication_year=2016&amp;journal=Syst.%20Biol&amp;volume=65&amp;pages=910-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syw036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyw036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsyw036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27288477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoffberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Kieran</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J. M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">16</div>, <div class="fpage">1264</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RADcap%3A%20sequence%20capture%20of%20dual-digest%20RADseq%20libraries%20with%20identifiable%20duplicates%20and%20reduced%20missing%20data&amp;author=S.%20L.%20Hoffberg&amp;author=T.%20J.%20Kieran&amp;author=J.%20M.%20Catchen&amp;publication_year=2016&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=16&amp;pages=1264-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1755-0998.12566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1755-0998.12566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1755-0998.12566"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27416967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Portik</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Bi</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">An evaluation of transcriptome-based exon capture for frog phylogenomics across multiple scales of divergence (Class: Amphibia, Order: Anura)</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">16</div>, <div class="fpage">1069</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20transcriptome-based%20exon%20capture%20for%20frog%20phylogenomics%20across%20multiple%20scales%20of%20divergence%20(Class%3A%20Amphibia%2C%20Order%3A%20Anura)&amp;author=D.%20Portik&amp;author=L.%20Smith&amp;author=K.%20Bi&amp;publication_year=2016&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=16&amp;pages=1069-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1755-0998.12541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1755-0998.12541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1755-0998.12541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27241806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sims</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sudbery</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ilott</div> <div class="given-names">N. E.</div></div>, <div class="name string-name"><div class="surname">Heger</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C. P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Sequencing depth and coverage: key considerations in genomic analyses</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">121</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20depth%20and%20coverage%3A%20key%20considerations%20in%20genomic%20analyses&amp;author=D.%20Sims&amp;author=I.%20Sudbery&amp;author=N.%20E.%20Ilott&amp;author=A.%20Heger&amp;author=C.%20P.%20Ponting&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=121-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24434847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lunter</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Goodson</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads</div>, <div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">936</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stampy%3A%20a%20statistical%20algorithm%20for%20sensitive%20and%20fast%20mapping%20of%20Illumina%20sequence%20reads&amp;author=G.%20Lunter&amp;author=M.%20Goodson&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=936-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.111120.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.111120.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.111120.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20980556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hagmann</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Simultaneous alignment of short reads against multiple genomes</div>, <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simultaneous%20alignment%20of%20short%20reads%20against%20multiple%20genomes&amp;author=K.%20Schneeberger&amp;author=J.%20Hagmann&amp;author=S.%20Ossowski&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-9-r98" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-9-r98" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-9-r98"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19761611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marcus</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schatz</div> <div class="given-names">M. C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">SplitMEM: a graphical algorithm for pan-genome analysis with suffix skips</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3476</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SplitMEM%3A%20a%20graphical%20algorithm%20for%20pan-genome%20analysis%20with%20suffix%20skips&amp;author=S.%20Marcus&amp;author=H.%20Lee&amp;author=M.%20C.%20Schatz&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3476-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25398610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Popic</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Batzoglou</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Short read alignment with populations of genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">i361</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20read%20alignment%20with%20populations%20of%20genomes&amp;author=L.%20Huang&amp;author=V.%20Popic&amp;author=S.%20Batzoglou&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=i361-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23813006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ohlebusch</div> <div class="given-names">E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A representation of a compressed de Bruijn graph for pan-genome analysis that enables search</div>, <div class="source">Algorithms Mol. Biol</div> ., <div class="volume">11</div>, <div class="fpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20representation%20of%20a%20compressed%20de%20Bruijn%20graph%20for%20pan-genome%20analysis%20that%20enables%20search&amp;author=T.%20Beller&amp;author=E.%20Ohlebusch&amp;publication_year=2016&amp;journal=Algorithms%20Mol.%20Biol&amp;volume=11&amp;pages=20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13015-016-0083-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13015-016-0083-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13015-016-0083-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27437028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hwang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Marcotte</div> <div class="given-names">E. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Systematic comparison of variant calling pipelines using gold standard personal exome variants</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">17875</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20comparison%20of%20variant%20calling%20pipelines%20using%20gold%20standard%20personal%20exome%20variants&amp;author=S.%20Hwang&amp;author=E.%20Kim&amp;author=I.%20Lee&amp;author=E.%20M.%20Marcotte&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=17875" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep17875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep17875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep17875"> </span></div> </div></div></div></div></div><div content-id="dsy023-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dapprich</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ferriola</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mackiewicz</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The next generation of target capture technologies - large DNA fragment enrichment and sequencing determines regional genomic variation of high complexity</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">486</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20next%20generation%20of%20target%20capture%20technologies%20-%20large%20DNA%20fragment%20enrichment%20and%20sequencing%20determines%20regional%20genomic%20variation%20of%20high%20complexity&amp;author=J.%20Dapprich&amp;author=D.%20Ferriola&amp;author=K.%20Mackiewicz&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=486" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2836-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2836-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2836-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27393338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van der Werf</div> <div class="given-names">I. M.</div></div>, <div class="name string-name"><div class="surname">Kooy</div> <div class="given-names">R. F.</div></div>, <div class="name string-name"><div class="surname">Vandeweyer</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A robust protocol to increase NimbleGen SeqCap EZ multiplexing capacity to 96 samples</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0123872</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%20protocol%20to%20increase%20NimbleGen%20SeqCap%20EZ%20multiplexing%20capacity%20to%2096%20samples&amp;author=I.%20M.%20van%20der%20Werf&amp;author=R.%20F.%20Kooy&amp;author=G.%20Vandeweyer&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0123872" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0123872" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0123872" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0123872"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25875648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Syring</div> <div class="given-names">J. V.</div></div>, <div class="name string-name"><div class="surname">Tennessen</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Jennings</div> <div class="given-names">T. N.</div></div>, <div class="name string-name"><div class="surname">Wegrzyn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Scelfo-Dalbey</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Cronn</div> <div class="given-names">R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Targeted capture sequencing in whitebark pine reveals range-wide demographic and adaptive patterns despite challenges of a large, repetitive genome</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">484</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20capture%20sequencing%20in%20whitebark%20pine%20reveals%20range-wide%20demographic%20and%20adaptive%20patterns%20despite%20challenges%20of%20a%20large%2C%20repetitive%20genome&amp;author=J.%20V.%20Syring&amp;author=J.%20A.%20Tennessen&amp;author=T.%20N.%20Jennings&amp;author=J.%20Wegrzyn&amp;author=C.%20Scelfo-Dalbey&amp;author=R.%20Cronn&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=7&amp;pages=484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2016.00484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2016.00484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2016.00484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27148310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dutoit</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Burri</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Nater</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mugal</div> <div class="given-names">C. F.</div></div>, <div class="name string-name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Genomic distribution and estimation of nucleotide diversity in natural populations: perspectives from the collared flycatcher (<em>Ficedula albicollis</em>) genome</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">17</div>, <div class="fpage">586</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20distribution%20and%20estimation%20of%20nucleotide%20diversity%20in%20natural%20populations%3A%20perspectives%20from%20the%20collared%20flycatcher%20(Ficedula%20albicollis)%20genome&amp;author=L.%20Dutoit&amp;author=R.%20Burri&amp;author=A.%20Nater&amp;author=C.%20F.%20Mugal&amp;author=H.%20Ellegren&amp;publication_year=2017&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=17&amp;pages=586-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2017.17.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2017.17.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2017.17.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27717155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Street</div> <div class="given-names">N. R.</div></div>, <div class="name string-name"><div class="surname">Scofield</div> <div class="given-names">D. G.</div></div>, <div class="name string-name"><div class="surname">Ingvarsson</div> <div class="given-names">P. K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Natural selection and recombination rate variation shape nucleotide polymorphism across the genomes of three related populus species</div>, <div class="source">Genetics</div> , <div class="volume">202</div>, <div class="fpage">1185</div>–<div class="lpage">200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20selection%20and%20recombination%20rate%20variation%20shape%20nucleotide%20polymorphism%20across%20the%20genomes%20of%20three%20related%20populus%20species&amp;author=J.%20Wang&amp;author=N.%20R.%20Street&amp;author=D.%20G.%20Scofield&amp;author=P.%20K.%20Ingvarsson&amp;publication_year=2016&amp;journal=Genetics&amp;volume=202&amp;pages=1185-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.115.183152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.115.183152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.115.183152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26721855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy023-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy023-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Melo</div> <div class="given-names">W. A.</div></div>, <div class="name string-name"><div class="surname">Lima-Ribeiro</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Terribile</div> <div class="given-names">L. C.</div></div>, <div class="name string-name"><div class="surname">Collevatti</div> <div class="given-names">R. G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Coalescent simulation and paleodistribution modeling for <em>Tabebuia rosealba</em> do not support South American dry forest refugia hypothesis</div>, <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0159314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coalescent%20simulation%20and%20paleodistribution%20modeling%20for%20Tabebuia%20rosealba%20do%20not%20support%20South%20American%20dry%20forest%20refugia%20hypothesis&amp;author=W.%20A.%20de%20Melo&amp;author=M.%20S.%20Lima-Ribeiro&amp;author=L.%20C.%20Terribile&amp;author=R.%20G.%20Collevatti&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0159314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0159314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0159314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0159314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27458982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743790" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhiro Sato</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhiro+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato K&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Kazuhiro Sato</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5055375&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_figure_s1.doc?Expires=2147483647&amp;Signature=a0ulO-6~sy0TzuDbimPrlijLIVfaZhA7isPmaCcqGR3hLauLgD-A8wtIYwID-VRvpBANIHaKFdZiF8drZdKOXCKLTFAJUToGBAswMRDNCZlbgLwKeDZGMFeaWjyWDXHD7QFcb8IY2SDjvL~GpCsu15xsALVfvEDRDKglwfZ9Ox0DF~K~lIMc-XA3dA7VzgQMSG1-GjuLCpjcA9WKmZh1LTapS2wT6h~leDc-WpJBPcRAtGDGmizUr9nXRksyEpxj3WplY3-WebxszJt8hsbQfyz9dGX6-x1tRXbmYjw~seBBKvi9xVHTIDbFLWt-u~PB3pX9YOEwChgG5kISXQP~Bg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s1.doc?Expires=2147483647&amp;Signature=HPsIJZqiHj6k57s6wvaBywnwovVR63EgJ-8hBX6ak~Spmv~8MnRcBmpvtPXh4bAdCFC7LoCVD6r9B6PnWsG41hOMxREit3BKXzoAHT4NAAOcPb54qejBtsOpIaUhXkjZblWZhoVOi9mGjUBTi3jPZipA1599Rpg5GK4dxScKaXEDyntjjB6gSfOnKI6EF3O~p5Jh9d7QxAZzFf75gJ6XLVfBIBheWZXR~6Nd6z4fmHH6YHcV47srCEXpYqXdRRv0RpAOAqw2Q4BQil2gcD58hl4e3cffN9W-Tue-bj8IWeTwSJO~IEKeEloMLldnK-XZpMhyCUfVXXyYgvFIsiPK3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S1</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy023/1/dsy023_supplementary_file_s2.xls?Expires=2147483647&amp;Signature=OpatlEJ3mEKNJ~44vJD1k98yoOZSmtGltwkOn-0Zf0ZEAitbhywqoBiEFGqUZNLbqLnghQVxp0DLvkT67mY2iIiGp3Z2H0h-~kIfaAoepw4-fAzHsaX56qZiVir3QIES1P2eO~zXswxN4wZmAiHniMwwyPda9Llz47Y~N~jFqVOWGCddBZXA7w~QZoQTUx3kt--XJE-JsBMk4DWIKlTl6kClmDf3AN3WzkzVzNElWwYNAxlUk5F2X2-f6MvVxJJhMn61vj8O3wbrMHR12VdDRZ59JazrdKZyujXIzppX2FZTtgL4~uwDWTd4FPRyfLz7hwI0yDX0BRifKM~KHE4keQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary File S2</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">662</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5055375"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f535%2f5055375">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F535%2F5055375" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5055375'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Design and evaluation of a sequence capture system for genome-wide SNP genotyping in highly heterozygous plant genomes: a case study with a keystone Neotropical hardwood tree genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f535%2f5055375">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

