

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/5/491/2754541","@type":"ScholarlyArticle","name":"Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform","datePublished":"2014-06-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2014-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu015","keywords":["DNA fingerprinting","high-throughput sequencing","molecular marker","retrotransposon","sweet potato"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/5/491/2754541","author":[{"name":"Monden, Yuki","affiliation":"1  Graduate School of Environmental and Life Science ,  Okayama University ,  1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530 ,  Japan","@type":"Person"},{"name":"Yamamoto, Ayaka","affiliation":"2  Faculty of Agriculture ,  Okayama University ,  1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530 ,  Japan","@type":"Person"},{"name":"Shindo, Akiko","affiliation":"1  Graduate School of Environmental and Life Science ,  Okayama University ,  1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530 ,  Japan","@type":"Person"},{"name":"Tahara, Makoto","affiliation":"1  Graduate School of Environmental and Life Science ,  Okayama University ,  1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530 ,  Japan","@type":"Person"}],"description":"In many crop species, DNA fingerprinting is required for the precise identification of cultivars to protect the rights of breeders. Many families of retrotransp","pageStart":"491","pageEnd":"498","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform" />
    <meta property="og:description" content="In many crop species, DNA fingerprinting is required for the precise identification of cultivars to protect the rights of breeders. Many families of retrotransp" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/5/491/2754541" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-06-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Monden, Yuki" /><meta name="citation_author_institution" content="Graduate School of Environmental and Life Science, Okayama University, 1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530, Japan" /><meta name="citation_author" content="Yamamoto, Ayaka" /><meta name="citation_author_institution" content="Faculty of Agriculture, Okayama University, 1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530, Japan" /><meta name="citation_author" content="Shindo, Akiko" /><meta name="citation_author_institution" content="Graduate School of Environmental and Life Science, Okayama University, 1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530, Japan" /><meta name="citation_author" content="Tahara, Makoto" /><meta name="citation_author_institution" content="Graduate School of Environmental and Life Science, Okayama University, 1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530, Japan" /><meta name="citation_title" content="Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform" /><meta name="citation_firstpage" content="491" /><meta name="citation_lastpage" content="498" /><meta name="citation_doi" content="10.1093/dnares/dsu015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2014/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Autopolyploidy versus allopolyploidy and low-density randomly amplified polymorphic DNA linkage maps of sweetpotato;  citation_journal_title=J. Am. Soc. Hort. Sci.;  citation_year=1997;  citation_volume=122;  citation_pages=822-8; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of sweetpotato (Ipomoea batatas (L.) Lam.) based on AFLP markers;  citation_journal_title=Mol. Breeding;  citation_year=2003;  citation_volume=11;  citation_pages=169-85; " /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci for dry-matter, starch, and Î²-carotene content in sweetpotato;  citation_journal_title=Mol. Breeding;  citation_year=2010;  citation_volume=28;  citation_pages=201-16; " /><meta name="citation_reference" content="citation_title=Establishment of molecular linkage maps using SRAP markers in sweet potato;  citation_journal_title=Acta Agronom. Sin.;  citation_year=2010;  citation_volume=36;  citation_pages=1286-95; " /><meta name="citation_reference" content="citation_title=A genetic linkage map based on AFLP and SSR markers and mapping of QTL for dry-matter content in sweetpotato;  citation_journal_title=Mol. Breeding;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Plant retrotransposons;  citation_journal_title=Annu. Rev. Genet.;  citation_year=1999;  citation_volume=33;  citation_pages=479-532; " /><meta name="citation_reference" content="citation_title=Plant transposable elements: where genetics meets genomics;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=329-41; " /><meta name="citation_reference" content="citation_title=Dynamic interactions between transposable elements and their hosts;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=615-27; " /><meta name="citation_reference" content="citation_title=Isolation of an active element from a high-copy-number family of retrotransposons in the sweetpotato genome;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2004;  citation_volume=272;  citation_pages=116-27; " /><meta name="citation_reference" content="citation_title=A LINE-type retrotransposon active in meristem stem cells causes heritable transpositions in the sweetpotato genome;  citation_journal_title=Plant Mol. Biol.;  citation_year=2006;  citation_volume=61;  citation_pages=79-94; " /><meta name="citation_reference" content="citation_title=Genetic distribution of Bare-1-like retrotransposable elements in the barley genome revealed by sequence-specific amplification polymorphisms (S-SAP);  citation_journal_title=Mol. Gen. Genet.;  citation_year=1997;  citation_volume=253;  citation_pages=687-94; " /><meta name="citation_reference" content="citation_title=Applications of retrotransposons as genetic tools in plant biology;  citation_journal_title=Trends Plant Sci.;  citation_year=2001;  citation_volume=6;  citation_pages=127-34; " /><meta name="citation_reference" content="citation_title=Ty1-copia retrotransposon-based SSAP marker development in cashew (Anacardium occidentale L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1195-202; " /><meta name="citation_reference" content="citation_title=The use of retrotransposon-based molecular markers to analyze genetic diversity;  citation_journal_title=Ratar. Povrt.;  citation_year=2011;  citation_volume=48;  citation_pages=261-74; " /><meta name="citation_reference" content="citation_title=Analysis of plant diversity with retrotransposon-based molecular markers;  citation_journal_title=Heredity;  citation_year=2011;  citation_volume=106;  citation_pages=520-30; " /><meta name="citation_reference" content="citation_title=Advances in plant gene-targeted and functional markers: a review;  citation_journal_title=Plant Methods;  citation_year=2013;  citation_volume=9;  citation_pages=6" /><meta name="citation_reference" content="citation_title=Performance comparison of benchtop high-throughput sequencing platforms;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=434-9; " /><meta name="citation_reference" content="citation_title=Reply to updating benchtop sequencing performance comparison;  citation_journal_title=Nat. Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=296" /><meta name="citation_reference" content="citation_title=Mobile element biology: new possibilities with high-throughput sequencing;  citation_journal_title=Trends Genet.;  citation_year=2013;  citation_volume=29;  citation_pages=280-9; " /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=BLATâThe BLAST-like alignment tool;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1979;  citation_volume=76;  citation_pages=5269-73; " /><meta name="citation_reference" content="citation_title=FreeTreeâFreeware program for construction of phylogenetic trees on the basis of distance data and bootstrap/jackknife analysis of the tree robustness. Application in the RAPD analysis of genus;  citation_journal_title=Folia Biol.;  citation_year=1999;  citation_volume=45;  citation_pages=97-9; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=High potential of a transposon mPing as a marker system in japonica times japonica cross in rice;  citation_journal_title=DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=131-40; " /><meta name="citation_reference" content="citation_title=SCAR markers: a potential tool for authentication of herbal drugs;  citation_journal_title=Fitoterapia;  citation_year=2010;  citation_volume=81;  citation_pages=969-76; " /><meta name="citation_reference" content="citation_title=A new SCAR marker potentially useful to distinguish Italian cattle breeds;  citation_journal_title=Food Chem.;  citation_year=2012;  citation_volume=130;  citation_pages=172-6; " /><meta name="citation_reference" content="citation_title=Digital gene expression analysis based on integrated de novo transcriptome assembly of sweet potato [Ipomoea batatas (L.) Lam];  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e36234" /><meta name="citation_reference" content="citation_title=A sweetpotato gene index established by de novo assembly of pyrosequencing and Sanger sequences and mining for gene-based microsatellite markers;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=604" /><meta name="citation_reference" content="citation_title=De novo assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of cSSR markers in sweet potato (Ipomoea batatas);  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=726" /><meta name="citation_reference" content="citation_title=Linkage and association mapping of Arabidopsis thaliana flowering time in nature;  citation_journal_title=PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000940" /><meta name="citation_reference" content="citation_title=Genome-wide association studies coming of age in rice;  citation_journal_title=Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=926-7; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces;  citation_journal_title=Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection;  citation_journal_title=Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1053-9; " /><meta name="citation_reference" content="citation_title=Association mapping of local climate-sensitive quantitative trait loci in Arabidopsis thaliana;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=21199-204; " /><meta name="citation_reference" content="citation_title=Genotype and SNP calling from next-generation sequencing data;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=443-51; " /><meta name="citation_reference" content="citation_title=Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=267" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/5/491/17462860/dsu015.pdf" /><meta name="description" content="In many crop species, DNA fingerprinting is required for the precise identification of cultivars to protect the rights of breeders. Many families of retrotransp" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/5/491/2754541" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/5/491/2754541" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/5/491/2754541");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu015';
    </script>
<title>
	Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754541" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd5bPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb25kZW4sIFl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgRW52aXJvbm1lbnRhbCBhbmQgTGlmZSBTY2llbmNlLCBPa2F5YW1hIFVuaXZlcnNpdHksIDEtMS0xIFRzdXNoaW1hLU5ha2EsIEtpdGEta3UsIE9rYXlhbWEgNzAwLTg1MzAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYW1vdG8sIEF5YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgT2theWFtYSBVbml2ZXJzaXR5LCAxLTEtMSBUc3VzaGltYS1OYWthLCBLaXRhLWt1LCBPa2F5YW1hIDcwMC04NTMwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaW5kbywgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgRW52aXJvbm1lbnRhbCBhbmQgTGlmZSBTY2llbmNlLCBPa2F5YW1hIFVuaXZlcnNpdHksIDEtMS0xIFRzdXNoaW1hLU5ha2EsIEtpdGEta3UsIE9rYXlhbWEgNzAwLTg1MzAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFoYXJhLCBNYWtvdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgRW52aXJvbm1lbnRhbCBhbmQgTGlmZSBTY2llbmNlLCBPa2F5YW1hIFVuaXZlcnNpdHksIDEtMS0xIFRzdXNoaW1hLU5ha2EsIEtpdGEta3UsIE9rYXlhbWEgNzAwLTg1MzAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFZmZpY2llbnQgRE5BIEZpbmdlcnByaW50aW5nIEJhc2VkIG9uIHRoZSBUYXJnZXRlZCBTZXF1ZW5jaW5nIG9mIEFjdGl2ZSBSZXRyb3RyYW5zcG9zb24gSW5zZXJ0aW9uIFNpdGVzIFVzaW5nIGEgQmVuY2gtVG9wIEhpZ2gtVGhyb3VnaHB1dCBTZXF1ZW5jaW5nIFBsYXRmb3JtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0OTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9wb2x5cGxvaWR5IHZlcnN1cyBhbGxvcG9seXBsb2lkeSBhbmQgbG93LWRlbnNpdHkgcmFuZG9tbHkgYW1wbGlmaWVkIHBvbHltb3JwaGljIEROQSBsaW5rYWdlIG1hcHMgb2Ygc3dlZXRwb3RhdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFtLiBTb2MuIEhvcnQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9ODIyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBzd2VldHBvdGF0byAoSXBvbW9lYSBiYXRhdGFzIChMLikgTGFtLikgYmFzZWQgb24gQUZMUCBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTY5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBmb3IgZHJ5LW1hdHRlciwgc3RhcmNoLCBhbmQgzrItY2Fyb3RlbmUgY29udGVudCBpbiBzd2VldHBvdGF0bzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTIwMS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0YWJsaXNobWVudCBvZiBtb2xlY3VsYXIgbGlua2FnZSBtYXBzIHVzaW5nIFNSQVAgbWFya2VycyBpbiBzd2VldCBwb3RhdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEgQWdyb25vbS4gU2luLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTEyODYtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBiYXNlZCBvbiBBRkxQIGFuZCBTU1IgbWFya2VycyBhbmQgbWFwcGluZyBvZiBRVEwgZm9yIGRyeS1tYXR0ZXIgY29udGVudCBpbiBzd2VldHBvdGF0bzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgcmV0cm90cmFuc3Bvc29uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz00NzktNTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCB0cmFuc3Bvc2FibGUgZWxlbWVudHM6IHdoZXJlIGdlbmV0aWNzIG1lZXRzIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zMjktNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgaW50ZXJhY3Rpb25zIGJldHdlZW4gdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGFuZCB0aGVpciBob3N0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYxNS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIGFuIGFjdGl2ZSBlbGVtZW50IGZyb20gYSBoaWdoLWNvcHktbnVtYmVyIGZhbWlseSBvZiByZXRyb3RyYW5zcG9zb25zIGluIHRoZSBzd2VldHBvdGF0byBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTExNi0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBMSU5FLXR5cGUgcmV0cm90cmFuc3Bvc29uIGFjdGl2ZSBpbiBtZXJpc3RlbSBzdGVtIGNlbGxzIGNhdXNlcyBoZXJpdGFibGUgdHJhbnNwb3NpdGlvbnMgaW4gdGhlIHN3ZWV0cG90YXRvIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTc5LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpc3RyaWJ1dGlvbiBvZiBCYXJlLTEtbGlrZSByZXRyb3RyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiB0aGUgYmFybGV5IGdlbm9tZSByZXZlYWxlZCBieSBzZXF1ZW5jZS1zcGVjaWZpYyBhbXBsaWZpY2F0aW9uIHBvbHltb3JwaGlzbXMgKFMtU0FQKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTM7ICBjaXRhdGlvbl9wYWdlcz02ODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9ucyBvZiByZXRyb3RyYW5zcG9zb25zIGFzIGdlbmV0aWMgdG9vbHMgaW4gcGxhbnQgYmlvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEyNy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHkxLWNvcGlhIHJldHJvdHJhbnNwb3Nvbi1iYXNlZCBTU0FQIG1hcmtlciBkZXZlbG9wbWVudCBpbiBjYXNoZXcgKEFuYWNhcmRpdW0gb2NjaWRlbnRhbGUgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTExOTUtMjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdXNlIG9mIHJldHJvdHJhbnNwb3Nvbi1iYXNlZCBtb2xlY3VsYXIgbWFya2VycyB0byBhbmFseXplIGdlbmV0aWMgZGl2ZXJzaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SYXRhci4gUG92cnQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MjYxLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBwbGFudCBkaXZlcnNpdHkgd2l0aCByZXRyb3RyYW5zcG9zb24tYmFzZWQgbW9sZWN1bGFyIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTUyMC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gcGxhbnQgZ2VuZS10YXJnZXRlZCBhbmQgZnVuY3Rpb25hbCBtYXJrZXJzOiBhIHJldmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmZvcm1hbmNlIGNvbXBhcmlzb24gb2YgYmVuY2h0b3AgaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcGxhdGZvcm1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NDM0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGx5IHRvIHVwZGF0aW5nIGJlbmNodG9wIHNlcXVlbmNpbmcgcGVyZm9ybWFuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI5NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vYmlsZSBlbGVtZW50IGJpb2xvZ3k6IG5ldyBwb3NzaWJpbGl0aWVzIHdpdGggaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yODAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBjb250cm9sIGFuZCBwcmVwcm9jZXNzaW5nIG9mIG1ldGFnZW5vbWljIGRhdGFzZXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTg2My00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFU4oCUVGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjU2LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGFsIFcgYW5kIENsdXN0YWwgWCB2ZXJzaW9uIDIuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yOTQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hdGhlbWF0aWNhbCBtb2RlbCBmb3Igc3R1ZHlpbmcgZ2VuZXRpYyB2YXJpYXRpb24gaW4gdGVybXMgb2YgcmVzdHJpY3Rpb24gZW5kb251Y2xlYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz01MjY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GcmVlVHJlZeKAlEZyZWV3YXJlIHByb2dyYW0gZm9yIGNvbnN0cnVjdGlvbiBvZiBwaHlsb2dlbmV0aWMgdHJlZXMgb24gdGhlIGJhc2lzIG9mIGRpc3RhbmNlIGRhdGEgYW5kIGJvb3RzdHJhcC9qYWNra25pZmUgYW5hbHlzaXMgb2YgdGhlIHRyZWUgcm9idXN0bmVzcy4gQXBwbGljYXRpb24gaW4gdGhlIFJBUEQgYW5hbHlzaXMgb2YgZ2VudXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvbGlhIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9OTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIHBvdGVudGlhbCBvZiBhIHRyYW5zcG9zb24gbVBpbmcgYXMgYSBtYXJrZXIgc3lzdGVtIGluIGphcG9uaWNhIHRpbWVzIGphcG9uaWNhIGNyb3NzIGluIHJpY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTMxLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TQ0FSIG1hcmtlcnM6IGEgcG90ZW50aWFsIHRvb2wgZm9yIGF1dGhlbnRpY2F0aW9uIG9mIGhlcmJhbCBkcnVnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rml0b3RlcmFwaWE7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz05NjktNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IFNDQVIgbWFya2VyIHBvdGVudGlhbGx5IHVzZWZ1bCB0byBkaXN0aW5ndWlzaCBJdGFsaWFuIGNhdHRsZSBicmVlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvb2QgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9MTcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZ2l0YWwgZ2VuZSBleHByZXNzaW9uIGFuYWx5c2lzIGJhc2VkIG9uIGludGVncmF0ZWQgZGUgbm92byB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IG9mIHN3ZWV0IHBvdGF0byBbSXBvbW9lYSBiYXRhdGFzIChMLikgTGFtXTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzNjIzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3dlZXRwb3RhdG8gZ2VuZSBpbmRleCBlc3RhYmxpc2hlZCBieSBkZSBub3ZvIGFzc2VtYmx5IG9mIHB5cm9zZXF1ZW5jaW5nIGFuZCBTYW5nZXIgc2VxdWVuY2VzIGFuZCBtaW5pbmcgZm9yIGdlbmUtYmFzZWQgbWljcm9zYXRlbGxpdGUgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiByb290IHRyYW5zY3JpcHRvbWUgdXNpbmcgSWxsdW1pbmEgcGFpcmVkLWVuZCBzZXF1ZW5jaW5nIGFuZCBkZXZlbG9wbWVudCBvZiBjU1NSIG1hcmtlcnMgaW4gc3dlZXQgcG90YXRvIChJcG9tb2VhIGJhdGF0YXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz03MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIGFuZCBhc3NvY2lhdGlvbiBtYXBwaW5nIG9mIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGZsb3dlcmluZyB0aW1lIGluIG5hdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwOTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBjb21pbmcgb2YgYWdlIGluIHJpY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTI2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXMgb2YgMTQgYWdyb25vbWljIHRyYWl0cyBpbiByaWNlIGxhbmRyYWNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz05NjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZXF1ZW5jaW5nIG9mIDMxIHdpbGQgYW5kIGN1bHRpdmF0ZWQgc295YmVhbiBnZW5vbWVzIGlkZW50aWZpZXMgcGF0dGVybnMgb2YgZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIHNlbGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDUzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG1hcHBpbmcgb2YgbG9jYWwgY2xpbWF0ZS1zZW5zaXRpdmUgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTIxMTk5LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub3R5cGUgYW5kIFNOUCBjYWxsaW5nIGZyb20gbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZSBkZWZpbml0aW9uIG9mIHRoZSBwZWRpZ3JlZSBoYXBsb3R5cGVzIG9mIGNsb3NlbHkgcmVsYXRlZCByaWNlIGN1bHRpdmFycyBieSBtZWFucyBvZiBnZW5vbWUtd2lkZSBkaXNjb3Zlcnkgb2Ygc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzUvNDkxLzE3NDYyODYwL2RzdTAxNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkluIG1hbnkgY3JvcCBzcGVjaWVzLCBETkEgZmluZ2VycHJpbnRpbmcgaXMgcmVxdWlyZWQgZm9yIHRoZSBwcmVjaXNlIGlkZW50aWZpY2F0aW9uIG9mIGN1bHRpdmFycyB0byBwcm90ZWN0IHRoZSByaWdodHMgb2YgYnJlZWRlcnMuIE1hbnkgZmFtaWxpZXMgb2YgcmV0cm90cmFuc3AiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3NTQ1NDEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkWQkVraLdaCUqkynJUrg17DkTC2k=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f491%2f2754541" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f491%2f2754541" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15969" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&amp;Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/5">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197013">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197013">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197018">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197018">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197036">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197036">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197058">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197058">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197063">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197065">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197067">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197067">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/5/481/2754538">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/5/499/2754542">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Monden</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Environmental and Life Science</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Monden">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Monden Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Monden Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Monden</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayaka Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayaka+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ayaka Yamamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Shindo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Environmental and Life Science</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Shindo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shindo A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shindo A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akiko Shindo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Tahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Environmental and Life Science</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">1-1-1 Tsushima-Naka, Kita-ku, Okayama 700-8530</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 86-251-8312. Fax. +81 86-251-8388. E-mail: <a href="mailto:tahara@cc.okayama-u.ac.jp" target="_blank">tahara@cc.okayama-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Tahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tahara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tahara M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Makoto Tahara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="an1"><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 491â498, <a href='https://doi.org/10.1093/dnares/dsu015'>https://doi.org/10.1093/dnares/dsu015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 June 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 November 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 April 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754541" href="/dnaresearch/article-pdf/21/5/491/17462860/dsu015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuki Monden, Ayaka Yamamoto, Akiko Shindo, Makoto Tahara;  Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform, <em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 491â498, <a href='https://doi.org/10.1093/dnares/dsu015'>https://doi.org/10.1093/dnares/dsu015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754541?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Efficient%20DNA%20Fingerprinting%20Based%20on%20the%20Targeted%20Sequencing%20of%20Active%20Retrotransposon%20Insertion%20Sites%20Using%20a%20Bench-Top%20High-Throughput%20Sequencing%20Platform&amp;publicationDate=2014-06-16&amp;volumeNum=21&amp;issueNum=5&amp;author=Monden%2C%20Yuki%3B%20Yamamoto%2C%20Ayaka&amp;startPage=491&amp;endPage=498&amp;contentId=10.1093%2Fdnares%2Fdsu015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f491%2f2754541" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>In many crop species, DNA fingerprinting is required for the precise identification of cultivars to protect the rights of breeders. Many families of retrotransposons have multiple copies throughout the eukaryotic genome and their integrated copies are inherited genetically. Thus, their insertion polymorphisms among cultivars are useful for DNA fingerprinting. In this study, we conducted a DNA fingerprinting based on the insertion polymorphisms of active retrotransposon families (<em>Rtsp-1</em> and <em>LIb</em>) in sweet potato. Using 38 cultivars, we identified 2,024 insertion sites in the two families with an Illumina MiSeq sequencing platform. Of these insertion sites, 91.4% appeared to be polymorphic among the cultivars and 376 cultivar-specific insertion sites were identified, which were converted directly into cultivar-specific sequence-characterized amplified region (SCAR) markers. A phylogenetic tree was constructed using these insertion sites, which corresponded well with known pedigree information, thereby indicating their suitability for genetic diversity studies. Thus, the genome-wide comparative analysis of active retrotransposon insertion sites using the bench-top MiSeq sequencing platform is highly effective for DNA fingerprinting without any requirement for whole genome sequence information. This approach may facilitate the development of practical polymerase chain reaction-based cultivar diagnostic system and could also be applied to the determination of genetic relationships.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA fingerprinting&quot;">DNA fingerprinting</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-throughput sequencing&quot;">high-throughput sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular marker&quot;">molecular marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="retrotransposon">retrotransposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sweet potato&quot;">sweet potato</a></div>                            <h2 scrollto-destination=87197013 id="87197013" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Globally, the sweet potato [<em>Ipomoea batatas</em> (L.) Lam] is an important food crop with an annual production of 104 million tons (Food and Agriculture Organization of the United Nations; <a class="link link-uri" href="http://faostat3.fao.org/">http://faostat3.fao.org/</a>, 20 May 2014, date last accessed). This crop is known to be a valuable food source that contains various nutrients, including a high starch content, complex carbohydrates, dietary fibre, vitamins and anthocyanins. In addition to its high nutrient content, this crop is resilient, easy to propagate and grows relatively well in infertile and nitrogen-poor soils. There are over 8,000 varieties, including wild accessions, landraces and breeding lines registered with the International Potato Center (<a class="link link-uri" href="http://cipotato.org/sweetpotato/">http://cipotato.org/sweetpotato/</a>, 20 May 2014, date last accessed). Thus, the sweet potato is grown widely and is used for various purposes throughout the world. However, the sweet potato is a hexaploid (2<em>n</em> = 6<em>x</em> = 90) outbreeding species with a high degree of heterozygosity, and genome sequence information is not available for this species, which has hindered genetic analyses, the development of molecular markers and DNA fingerprinting.<sup><a reveal-id="DSU015C1" data-open="DSU015C1" class="link link-ref link-reveal xref-bibr">1</a>â<a reveal-id="DSU015C5" data-open="DSU015C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>Retrotransposons are major components of eukaryotic genomes and they are present in high copy numbers in most of the plants.<sup><a reveal-id="DSU015C6" data-open="DSU015C6" class="link link-ref link-reveal xref-bibr">6</a>â<a reveal-id="DSU015C8" data-open="DSU015C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Retrotransposons are divided into two major groups: a group with a long terminal repeat (LTR) sequence at both ends and another group without LTRs (non-LTR). The non-LTR retrotransposons are further divided into long interspersed nuclear elements (LINEs) and short interspersed nuclear elements (SINEs).<sup><a reveal-id="DSU015C6" data-open="DSU015C6" class="link link-ref link-reveal xref-bibr">6</a>â<a reveal-id="DSU015C8" data-open="DSU015C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In our previous study, we identified two active retrotransposon families in sweet potato, i.e. an LTR-type <em>Rtsp-1</em> and a LINE-type <em>LIb</em>.<sup><a reveal-id="DSU015C9" data-open="DSU015C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU015C10" data-open="DSU015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> These two families exhibited a high degree of insertion polymorphism and newly inserted copies were inherited steadily throughout clonal propagation.<sup><a reveal-id="DSU015C10" data-open="DSU015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The retrotransposon insertion polymorphisms found in different cultivars can be used for DNA fingerprinting.<sup><a reveal-id="DSU015C11" data-open="DSU015C11" class="link link-ref link-reveal xref-bibr">11</a>â<a reveal-id="DSU015C16" data-open="DSU015C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>Next-generation sequencing technology has developed rapidly in the past few years and it has been utilized in various fields of genetics, thereby accelerating genomic and genetic research dramatically. More recently, bench-top high-throughput sequencing instruments such as the Roche/454 GS Junior (Roche, Basel, Switzerland), Illumina MiSeq and Ion Torrent PGM (Life Technologies) have been released, which are capable of sequencing several megabase to gigabase pairs in a single lane-sequencing run with enhanced speed and cost performance.<sup><a reveal-id="DSU015C17" data-open="DSU015C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU015C18" data-open="DSU015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In particular, the MiSeq system has the highest throughput per run and the lowest error rate,<sup><a reveal-id="DSU015C17" data-open="DSU015C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the latest version enables to generate up to 15 Gb per run with 2 Ã 300 bp reads using new reagent kits (<a class="link link-uri" href="http://www.illumina.com/systems/miseq.ilmn">http://www.illumina.com/systems/miseq.ilmn</a>, 20 May 2014, date last accessed). These bench-top systems produce smaller sequence datasets than traditional high-throughput sequencing instruments (Roche/454 GS FLX+ and Illumina HiSeq 2000), but their utilization is increasingly widespread because of their greater time and cost performance during sequencing analysis, although with modest performance.</p><p>In the present study, we conducted DNA fingerprinting based on the insertion polymorphisms of the active retrotransposon families (<em>Rtsp-1</em> and <em>LIb</em>) in sweet potato. Using a bench-top sequencing platform, we successfully identified 2,024 insertion sites in 38 cultivars based on a single lane-sequencing run. It should be noted that the insertion sites of the active retrotransposon families were highly polymorphic among those cultivars. Using the cultivar-specific insertion sites, we successfully developed sequence-characterized amplified region (SCAR) markers, which may facilitate the precise identification of cultivars. Moreover, the genotyping data related to the insertion sites can be used to determine the genetic relationships among cultivars. The sweet potato is a non-model crop species and its whole genome sequencing data are not available, but our method enables efficient genotyping based on a number of polymorphic retrotransposon insertion sites.</p>                            <h2 scrollto-destination=87197018 id="87197018" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87197019 id="87197019" class="section-title" data-legacy-id=s2a>2.1. DNA samples</h3>
<p>DNA samples from the 38 sweet potato cultivars listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> were provided by the National Agriculture and Food Research Organization, Kyusyu Okinawa Agricultural Research Center. Young leaves were collected from field grown plants and genomic DNA was extracted using a DNeasy Plant Mini Kit (Qiagen, Inc., Germany).</p>                            <h3 scrollto-destination=87197021 id="87197021" class="section-title" data-legacy-id=s2b>2.2. MiSeq library construction</h3>
<div class="block-child-p">Genomic DNA (5 Î¼g) was separated into fragments of â¼6 kb by g-TUBE (Covaris) centrifugation and purified using a QIAquick PCR Purification Kit (Qiagen). The purified products were treated with DNA polymerase I and T4 DNA polymerase to convert the heterogeneous ends via their physical fragmentation into blunt ends. The blunt ends were adenylated at the 3â² end using a Klenow fragment that lacked a 5â² to 3â² exonuclease activity. A forked adapter with a T overhang at the 5â² double-stranded end was ligated using T4 DNA ligase. The retrotransposon insertion sites were specifically amplified using nested polymerase chain reaction (PCR), where the first PCR was performed with retrotransposon-specific (<em>Rtsp-1</em>_ppt or <em>LIb</em>_L_D4) and adapter-specific (AP2) primer combinations using the ligated products as the template, and the second PCR was performed with primers for the 3â² end sequence of <em>Rtsp-1</em> or <em>LIb</em> [adjacent to the 3â² end polyadenosine sequence (poly(A)] and an adapter sequence (AP3) using the initial PCR products as the template. The PCR comprises an initial denaturation at 94Â°C for 2 min, which was followed by 30 cycles at 94Â°C for 30 s, 80Â°C for 30 s, 58Â°C for 30 s and 72Â°C for 1 min, with a final extension at 72Â°C for 15 min. The PCR products were selected by size (300â500 bp) using gel electrophoresis and purified with a QIAquick Gel Extraction Kit (Qiagen). The MiSeq sequencing adapters were ligated to the size-selected products from each cultivar using a TruSeq DNA-Sampling Preparation Kit (Illumina) to obtain cultivar-specific combinations of the dual index sequences in the adapters (Fig.Â <a reveal-id="DSU015F1" data-open="DSU015F1" class="link link-reveal link-table xref-fig">1</a>). Equal amounts of the ligated products from the 38 cultivars were pooled before MiSeq sequencing, and a paired-end sequence of 150 bp was determined at each end (Fig.Â <a reveal-id="DSU015F1" data-open="DSU015F1" class="link link-reveal link-table xref-fig">1</a>). MiSeq sequencing was conducted using the MS-102-1001 MiSeq Reagen Kit (300 cycles) (Illumina). In sequencing run, we added a PhiX Control v3 (Illumina) at 50% of our library, because Illumina recommends spike-in a PhiX control DNA for low diversity library where a significant number of the reads have the same sequence. It is known that a high-concentration spike-in (40% or higher) of PhiX improves matrix generation and phasing/prephasing calculations and helps balance the overall lack of sequence diversity. Our library was considered to have low diversity nucleotide composition because of the <em>Rtsp-1</em> or <em>LIb</em> sequences, or an adaptor AP3 sequence at the beginning of one read of a sequence pair. This should be combined with a reduction of the amount of library used. The sequences of the adapters and primers are reported in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. The MiSeq reads of this study have been submitted to DDBJ (<a class="link link-uri" href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</a>, 20 May 2014, date last accessed) under accession no. DRA001340. <div data-id="DSU015F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01501.jpeg?Expires=2147483647&amp;Signature=KVlWOTl~H0fTpeDbA41QVgLvlR7otH9JB6XIzvJHJ8Pe6GfhWIAoVPCPDPimyiDbBLbHjvg0~lk8INhxWDhLTtl9d9WY4gsDtPldl3mP~sDAbNGYsmNAkQkQkh7U6An4BzYIf92OmGLhKD6CGLjCTzSlM4Cqx2vlZpDiIZEd3y4JjsTOiLSXACGjmrk8yFkdqKulu5GFkIO1gg~YK2yQj0-CXf92x5w9HkCqd9fQ5LSPx0fLTttEI5R2~XLPe-qOVU2Y1y7s98XFbOVSLTXfiq2NmY~lkjEqOQzjCKX0-YMCdwcTU~KSycSaETnPzcfNZAzL~R0IV4Mx76ewIu3JNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of the MiSeq sequencing library in this study. The 150-bp paired reads (black arrow) represent the junction sequence of the retrotransposon (Rtsp-1 or LIb) (red box) and the insertion site sequence from the AP3 adapter (purple box). The TruSeq adapters used in this study were indexed with short 6â8 bp sequences to facilitate multiplexing in a single sequencing run. The P5 and P7 sequences (blue box) allowed the final products to hybridize to the Illumina flow cell. The Rd1 and Rd2 sequences (green box) served as sequencing primers." path-from-xml="dsu01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197022/dsu01501.jpeg" path-from-xml="dsu01501.jpeg" target="_blank">View large</a><a section="87197022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01501.jpeg?Expires=2147483647&amp;Signature=nCm9LLl4cXiKjSIN-mfht~VsN96uUk77D0KBPErOHd8uJSMvBOqMiRwXEWXzxNtb7KN7hCP903drvSnqwu~zHnK-U1sJIZTKZJQS1fuXByXuc780AyZk~qLIvcHtrRE85qGrYiw626rXzW8tp1pDut3GTv7b8NrT2D--~WSPM0GT8WdOvWjLfRlPgTnr4~Nj-FUtGLw6VNMcbtt4h7HTcZEFxDCS~p1AHg2bv-gdlgNHvKi2BSR5oUBOG~EAmiPF3dvqFdnMNeQzQLRLaV4VlcjQcpZea7P2mbrAfvxmFU-eohfDgdDiFn52keLUfESbydrFH47pgD71e-PZcITfVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197022&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme of the MiSeq sequencing library in this study. The 150-bp paired reads (black arrow) represent the junction sequence of the retrotransposon (<em>Rtsp</em>-1 or <em>LIb</em>) (red box) and the insertion site sequence from the AP3 adapter (purple box). The TruSeq adapters used in this study were indexed with short 6â8 bp sequences to facilitate multiplexing in a single sequencing run. The P5 and P7 sequences (blue box) allowed the final products to hybridize to the Illumina flow cell. The Rd1 and Rd2 sequences (green box) served as sequencing primers.</p></div></div><div content-id="DSU015F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01501.jpeg?Expires=2147483647&amp;Signature=KVlWOTl~H0fTpeDbA41QVgLvlR7otH9JB6XIzvJHJ8Pe6GfhWIAoVPCPDPimyiDbBLbHjvg0~lk8INhxWDhLTtl9d9WY4gsDtPldl3mP~sDAbNGYsmNAkQkQkh7U6An4BzYIf92OmGLhKD6CGLjCTzSlM4Cqx2vlZpDiIZEd3y4JjsTOiLSXACGjmrk8yFkdqKulu5GFkIO1gg~YK2yQj0-CXf92x5w9HkCqd9fQ5LSPx0fLTttEI5R2~XLPe-qOVU2Y1y7s98XFbOVSLTXfiq2NmY~lkjEqOQzjCKX0-YMCdwcTU~KSycSaETnPzcfNZAzL~R0IV4Mx76ewIu3JNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of the MiSeq sequencing library in this study. The 150-bp paired reads (black arrow) represent the junction sequence of the retrotransposon (Rtsp-1 or LIb) (red box) and the insertion site sequence from the AP3 adapter (purple box). The TruSeq adapters used in this study were indexed with short 6â8 bp sequences to facilitate multiplexing in a single sequencing run. The P5 and P7 sequences (blue box) allowed the final products to hybridize to the Illumina flow cell. The Rd1 and Rd2 sequences (green box) served as sequencing primers." path-from-xml="dsu01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197022/dsu01501.jpeg" path-from-xml="dsu01501.jpeg" target="_blank">View large</a><a section="87197022" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01501.jpeg?Expires=2147483647&amp;Signature=nCm9LLl4cXiKjSIN-mfht~VsN96uUk77D0KBPErOHd8uJSMvBOqMiRwXEWXzxNtb7KN7hCP903drvSnqwu~zHnK-U1sJIZTKZJQS1fuXByXuc780AyZk~qLIvcHtrRE85qGrYiw626rXzW8tp1pDut3GTv7b8NrT2D--~WSPM0GT8WdOvWjLfRlPgTnr4~Nj-FUtGLw6VNMcbtt4h7HTcZEFxDCS~p1AHg2bv-gdlgNHvKi2BSR5oUBOG~EAmiPF3dvqFdnMNeQzQLRLaV4VlcjQcpZea7P2mbrAfvxmFU-eohfDgdDiFn52keLUfESbydrFH47pgD71e-PZcITfVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197022&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scheme of the MiSeq sequencing library in this study. The 150-bp paired reads (black arrow) represent the junction sequence of the retrotransposon (<em>Rtsp</em>-1 or <em>LIb</em>) (red box) and the insertion site sequence from the AP3 adapter (purple box). The TruSeq adapters used in this study were indexed with short 6â8 bp sequences to facilitate multiplexing in a single sequencing run. The P5 and P7 sequences (blue box) allowed the final products to hybridize to the Illumina flow cell. The Rd1 and Rd2 sequences (green box) served as sequencing primers.</p></div></div></div>                            <h3 scrollto-destination=87197023 id="87197023" class="section-title" data-legacy-id=s2c>2.3. Data analysis</h3>
<p>The MiSeq output was divided into each cultivar based on the dual index TruSeq sequence. The sequencing data were handled using Maser, the pipeline execution system of the Cell Innovation Program at the National Institute of Genetics (<a class="link link-uri" href="http://cell-innovation.nig.ac.jp/index_en.html">http://cell-innovation.nig.ac.jp/index_en.html</a>, 20 May 2014, date last accessed). When one read of a sequence pair was filtered for invalid <em>Rtsp-1</em> or <em>LIb</em> primer sequences, the entire pair was discarded. The LINE element of <em>LIb</em> has a poly(A) tail with variable lengths at the integration site.<sup><a reveal-id="DSU015C10" data-open="DSU015C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU015C19" data-open="DSU015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Those poly(A) sequences were removed from the <em>LIb</em> reads using the PRINSEQ tool<sup><a reveal-id="DSU015C20" data-open="DSU015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> before subsequent data processing. After trimming the <em>Rtsp-1</em> or <em>LIb</em> sequences from the reads, they were trimmed further to 50 bp from the retrotransposon junction and filtered based on the quality value (QV) for all base calls â¥20. Next, the outliers were filtered by trimming them to a specific length that included most of the sequences (&gt;99 or &gt;98%), whereas reads shorter than this specific length were filtered out. Identical sequences observed in â¥10 reads were treated as a single sequence using the FASTA format with read count information, whereas those with &lt;10 reads were discarded. Using the sequences that passed through these processes, we determined <em>Rtsp-1</em> and <em>LIb</em> insertion sites according to the following steps: (i) we conducted all-to-all comparison of these sequences to reveal their similarities, which were used to build clusters that represent an individual insertion site. It started with the self-alignment of the produced sequences via BLAT analysis<sup><a reveal-id="DSU015C21" data-open="DSU015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using the following parameter settings: -tileSize = 8, -minMatch = 1, -minScore = 10, -repMatch = â1 and -oneOff = 2. BLAT is one of the local pairwise alignment programmes.<sup><a reveal-id="DSU015C21" data-open="DSU015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (ii) We clustered these sequences into groups based on their sequence similarity with the results of pairwise alignments. In each cluster, sequences were collected in FASTA format. (iii) We performed a multiple sequence alignment using these FASTA files for each cluster to reveal the sequence similarity with ClustalW program,<sup><a reveal-id="DSU015C22" data-open="DSU015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> which produced an alignment file per cluster. Then, we determined the representative sequence in each cluster based on the number of sequence. The sequence with the highest number was extracted as the representative sequence in each cluster. Our results showed that the sequences to be aligned in each cluster were very similar to each other (identity of &gt;90%). These processes generated the cluster of sequence or non-clustered sequence with â¥10 reads as an individual insertion site that a copy of <em>Rtsp-1</em> or <em>LIb</em> was inserted in at least one or more cultivars. However, assigning the insertion sites to cultivars would probably have caused some errors, because sequence errors in the index sequence may have allocated the reads to an incorrect cultivar while sequences with small read numbers may have been incorrectly included in large clusters by the BLAT analysis. These erroneous assignments would have resulted in a very small number of reads. Thus, we set a critical value for determining the presence of <em>Rtsp-1</em> and <em>LIb</em> insertions, i.e. if the reads for a cultivar at a specific insertion site comprise &lt;0.01% of the entire reads for that cultivar, we assumed that the retrotransposon was absent from that site. Each cultivar was genotyped by combining the presence (1) or absence (0) of information at each site for all the insertion sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p>                            <h3 scrollto-destination=87197027 id="87197027" class="section-title" data-legacy-id=s2d>2.4. Development of SCAR markers</h3>
<p>We designed the PCR primers to amplify the junction of the retrotransposon and its insertion site. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> showed all genotyping information (scores 0 and 1) in the cultivars, the representative sequence in each cluster and non-cluster sequence. We focused on the putative cultivar-specific insertion site that was present in one cultivar (scoring; 1) and absent from others (scoring; 0). The primer sequence of SCAR marker was designed based on the representative sequence for the cluster of these cultivar-specific insertion site (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The PCR was performed with <em>Rtsp-1-</em>specific primer (<em>Rtsp-1</em>_ppt) and insertion site-specific (RP_*) primer combinations using the genomic DNA as the template (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The PCR comprises an initial denaturation at 95Â°C for 2 min, which was followed by 30 cycles at 95Â°C for 30 s, 58Â°C for 30 s and 72Â°C for 30 s, with a final extension at 72Â°C for 15 min. The PCR products were visualized using 1.5% agarose gel electrophoresis.</p>                            <h3 scrollto-destination=87197031 id="87197031" class="section-title" data-legacy-id=s2e>2.5. Phylogenetic analysis</h3>
<p>A distance matrix was calculated from the presence/absence matrices using Nei's and Li's coefficient index<sup><a reveal-id="DSU015C23" data-open="DSU015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> with the FreeTree program (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).<sup><a reveal-id="DSU015C24" data-open="DSU015C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The phylogenetic reconstruction was based on the neighbour-joining method. Support for the internal branches in the phylogeny was assessed using 1,000 bootstrap replicates. A dendrogram was constructed with MEGA5.<sup><a reveal-id="DSU015C25" data-open="DSU015C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>                            <h2 scrollto-destination=87197036 id="87197036" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197038 id="87197038" class="section-title" data-legacy-id=s3a>3.1. MiSeq sequencing of the <em>Rtsp-1 and LIb</em> insertion sites</h3>
<p>We constructed a MiSeq sequencing library to screen the junction sequences of the <em>Rtsp-1</em> and <em>LIb</em> insertion sites in 38 sweet potato cultivars (Fig.Â <a reveal-id="DSU015F1" data-open="DSU015F1" class="link link-reveal link-table xref-fig">1</a>). Each cultivar sample was labelled using a TruSeq DNA-Sampling Preparation Kit with dual indexing (Fig.Â <a reveal-id="DSU015F1" data-open="DSU015F1" class="link link-reveal link-table xref-fig">1</a>). The <em>Rtsp-1</em> and <em>LIb</em> libraries were mixed at a ratio of 4 : 1 to prepare a single sequencing sample, because the copy number of <em>Rtsp-1</em> was estimated to be approximately four times higher than that of <em>LIb</em>.<sup><a reveal-id="DSU015C9" data-open="DSU015C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU015C10" data-open="DSU015C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>A total of 4,401,639 read pairs were obtained from a single lane-sequencing run using the MiSeq platform, i.e. 3,567,587 and 834,052 read pairs represented <em>Rtsp-1</em> and <em>LIb</em> insertion sites, respectively (TableÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a>). The results indicated that the read number ratio in each library (<em>Rtsp-1</em> reads = 81.1% and <em>LIb</em> reads = 18.9%) was close to the expected value (<em>Rtsp-1</em> reads = 80% and <em>LIb</em> reads = 20%). Of these read pairs, 86.8% for <em>Rtsp-1</em> and 89.8% for <em>LIb</em> had valid retrotransposon-specific sequences (TableÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a>), which indicated that nested PCR could specifically amplify DNA fragments from the junction sequences of these elements. The read numbers for each sample are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. <div content-id="DSU015TB1" class="table-modal"><div class="table-wrap"><div id="DSU015TB1" data-id="DSU015TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Summary of the MiSeq read data processing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>RetrotransposonÂ </th><th>AnalysisÂ </th><th>No. of read pairsÂ </th><th>No. of collapsed reads (â¥10)Â </th><th>Ratio (%)Â </th><th>No. of clustersÂ </th></tr></thead><tbody><tr><td rowspan="6"><em>Rtsp-1</em>Â </td><td>Raw dataÂ </td><td>3,567,587Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>3,095,771Â </td><td>16,428Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>3,095,771Â </td><td>8,038Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>2,292,344Â </td><td>5,245Â </td><td>64.3Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;99%)Â </td><td>2,273,707Â </td><td>5,218Â </td><td>63.7Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>1,497Â </td></tr><tr><td rowspan="7"><em>LIb</em>Â </td><td>Raw dataÂ </td><td>834,052Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>743,945Â </td><td>3,608Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Poly(A) tail trimming (A â¥ 3)Â </td><td>743,935Â </td><td>3,571Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>743,935Â </td><td>3,154Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>434,502Â </td><td>1,619Â </td><td>52.1Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;98%)Â </td><td>427,120Â </td><td>1,603Â </td><td>51.2Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>527Â </td></tr></tbody></table></div><div content-id="DSU015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>RetrotransposonÂ </th><th>AnalysisÂ </th><th>No. of read pairsÂ </th><th>No. of collapsed reads (â¥10)Â </th><th>Ratio (%)Â </th><th>No. of clustersÂ </th></tr></thead><tbody><tr><td rowspan="6"><em>Rtsp-1</em>Â </td><td>Raw dataÂ </td><td>3,567,587Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>3,095,771Â </td><td>16,428Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>3,095,771Â </td><td>8,038Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>2,292,344Â </td><td>5,245Â </td><td>64.3Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;99%)Â </td><td>2,273,707Â </td><td>5,218Â </td><td>63.7Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>1,497Â </td></tr><tr><td rowspan="7"><em>LIb</em>Â </td><td>Raw dataÂ </td><td>834,052Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>743,945Â </td><td>3,608Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Poly(A) tail trimming (A â¥ 3)Â </td><td>743,935Â </td><td>3,571Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>743,935Â </td><td>3,154Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>434,502Â </td><td>1,619Â </td><td>52.1Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;98%)Â </td><td>427,120Â </td><td>1,603Â </td><td>51.2Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>527Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The clusters include groups of two or more sequences and non-clustered single sequences for the insertion sites of <em>Rtsp-1</em>and <em>LIb</em>, where the minimum number of reads for an identical sequence was 10.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU015TB1" data-id="DSU015TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Summary of the MiSeq read data processing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>RetrotransposonÂ </th><th>AnalysisÂ </th><th>No. of read pairsÂ </th><th>No. of collapsed reads (â¥10)Â </th><th>Ratio (%)Â </th><th>No. of clustersÂ </th></tr></thead><tbody><tr><td rowspan="6"><em>Rtsp-1</em>Â </td><td>Raw dataÂ </td><td>3,567,587Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>3,095,771Â </td><td>16,428Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>3,095,771Â </td><td>8,038Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>2,292,344Â </td><td>5,245Â </td><td>64.3Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;99%)Â </td><td>2,273,707Â </td><td>5,218Â </td><td>63.7Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>1,497Â </td></tr><tr><td rowspan="7"><em>LIb</em>Â </td><td>Raw dataÂ </td><td>834,052Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>743,945Â </td><td>3,608Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Poly(A) tail trimming (A â¥ 3)Â </td><td>743,935Â </td><td>3,571Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>743,935Â </td><td>3,154Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>434,502Â </td><td>1,619Â </td><td>52.1Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;98%)Â </td><td>427,120Â </td><td>1,603Â </td><td>51.2Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>527Â </td></tr></tbody></table></div><div content-id="DSU015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>RetrotransposonÂ </th><th>AnalysisÂ </th><th>No. of read pairsÂ </th><th>No. of collapsed reads (â¥10)Â </th><th>Ratio (%)Â </th><th>No. of clustersÂ </th></tr></thead><tbody><tr><td rowspan="6"><em>Rtsp-1</em>Â </td><td>Raw dataÂ </td><td>3,567,587Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>3,095,771Â </td><td>16,428Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>3,095,771Â </td><td>8,038Â </td><td>86.8Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>2,292,344Â </td><td>5,245Â </td><td>64.3Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;99%)Â </td><td>2,273,707Â </td><td>5,218Â </td><td>63.7Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>1,497Â </td></tr><tr><td rowspan="7"><em>LIb</em>Â </td><td>Raw dataÂ </td><td>834,052Â </td><td>Â </td><td>100Â </td><td>Â </td></tr><tr><td>Validation of retrotransposon sequenceÂ </td><td>743,945Â </td><td>3,608Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Poly(A) tail trimming (A â¥ 3)Â </td><td>743,935Â </td><td>3,571Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>Trimming to 50 bpÂ </td><td>743,935Â </td><td>3,154Â </td><td>89.2Â </td><td>Â </td></tr><tr><td>QV filtering (â¥20)Â </td><td>434,502Â </td><td>1,619Â </td><td>52.1Â </td><td>Â </td></tr><tr><td>Outlier filtering (&gt;98%)Â </td><td>427,120Â </td><td>1,603Â </td><td>51.2Â </td><td>Â </td></tr><tr><td>Clustering with BLAT<sup>a</sup>Â </td><td>Â </td><td>Â </td><td>Â </td><td>527Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>The clusters include groups of two or more sequences and non-clustered single sequences for the insertion sites of <em>Rtsp-1</em>and <em>LIb</em>, where the minimum number of reads for an identical sequence was 10.</p></div></div></div></div></p>                            <h3 scrollto-destination=87197044 id="87197044" class="section-title" data-legacy-id=s3b>3.2. Identification of the <em>Rtsp-1 and LIb</em> insertion sites in 38 cultivars</h3>
<div class="block-child-p">To identify the <em>Rtsp-1</em> and <em>LIb</em> insertion sites, we selected one end of a paired read that contained the junction sequence of the <em>Rtsp-1</em> and <em>LIb</em> ends and their insertion sites. A sequence of 150 bp in the read was filtered based on the valid retrotransposon sequences and QV scores with base calls â¥20 in the insertion site sequence of 50 bp (see Materials and Methods) (TableÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a>). After these filtering processes, 63.7 and 51.2% of the reads remained for <em>Rtsp-1</em> and <em>LIb</em>, respectively (TableÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a>). These reads were used in the clustering analysis to obtain sequence information for each insertion site (TableÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a>). The clustering process was conducted using BLAT,<sup><a reveal-id="DSU015C21" data-open="DSU015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> where the parameter settings were -tileSize = 8, -minMatch = 1, -minScore = 10, -repMatch = â1 and -oneOff = 2, which generated 1,497 and 527 clusters or non-clustered single sequences for the insertion sites of <em>Rtsp-</em>1 and <em>LIb</em>, respectively, where the minimum number of reads required for an identical sequence was 10 (TablesÂ <a reveal-id="DSU015TB1" data-open="DSU015TB1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSU015TB2" data-open="DSU015TB2" class="link link-reveal link-table xref-fig">2</a>). We detected <em>Rtsp-1</em> and <em>LIb</em> in each insertion site of the 38 cultivars (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). In the 38 cultivars, the median number of <em>Rtsp-1</em> insertion sites per cultivar was 257.5 (range = 216â347), whereas that for <em>LIb</em> was 92.5 (range = 67â147) (Fig.Â <a reveal-id="DSU015F2" data-open="DSU015F2" class="link link-reveal link-table xref-fig">2</a>). Also, the average number of <em>Rtsp-1</em> insertion sites among 38 cultivars was 259, whereas that of <em>LIb</em> was 102.6, respectively. Of the 1,497 <em>Rtsp-1</em> insertion sites, 88.6% were polymorphic among cultivars and 19.4% were identified as cultivar-specific sites, which were present in only one cultivar and absent from others (TableÂ <a reveal-id="DSU015TB2" data-open="DSU015TB2" class="link link-reveal link-table xref-fig">2</a>). For <em>LIb</em>, the same figures were 99.2 and 16.1%, respectively (TableÂ <a reveal-id="DSU015TB2" data-open="DSU015TB2" class="link link-reveal link-table xref-fig">2</a>). Thus, the insertion sites of these two active retrotransposon families were found to be highly polymorphic among sweet potato cultivars. <div content-id="DSU015TB2" class="table-modal"><div class="table-wrap"><div id="DSU015TB2" data-id="DSU015TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>Summary of the insertion sites identified</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>All insertion sitesÂ </th><th>No. of polymorphic sitesÂ </th><th>Polymorphic ratio (%)Â </th><th>No. of cultivar-specific sitesÂ </th><th>Cultivar-specific ratio (%)Â </th></tr></thead><tbody><tr><td><em>Rtsp-1</em>Â </td><td>1,497Â </td><td>1,327Â </td><td>88.6Â </td><td>291Â </td><td>19.4Â </td></tr><tr><td><em>LIb</em>Â </td><td>527Â </td><td>523Â </td><td>99.2Â </td><td>85Â </td><td>16.1Â </td></tr><tr><td>TotalÂ </td><td>2,024Â </td><td>1,850Â </td><td>91.4Â </td><td>376Â </td><td>18.6Â </td></tr></tbody></table></div><div content-id="DSU015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>All insertion sitesÂ </th><th>No. of polymorphic sitesÂ </th><th>Polymorphic ratio (%)Â </th><th>No. of cultivar-specific sitesÂ </th><th>Cultivar-specific ratio (%)Â </th></tr></thead><tbody><tr><td><em>Rtsp-1</em>Â </td><td>1,497Â </td><td>1,327Â </td><td>88.6Â </td><td>291Â </td><td>19.4Â </td></tr><tr><td><em>LIb</em>Â </td><td>527Â </td><td>523Â </td><td>99.2Â </td><td>85Â </td><td>16.1Â </td></tr><tr><td>TotalÂ </td><td>2,024Â </td><td>1,850Â </td><td>91.4Â </td><td>376Â </td><td>18.6Â </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU015TB2" data-id="DSU015TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>Summary of the insertion sites identified</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Â </th><th>All insertion sitesÂ </th><th>No. of polymorphic sitesÂ </th><th>Polymorphic ratio (%)Â </th><th>No. of cultivar-specific sitesÂ </th><th>Cultivar-specific ratio (%)Â </th></tr></thead><tbody><tr><td><em>Rtsp-1</em>Â </td><td>1,497Â </td><td>1,327Â </td><td>88.6Â </td><td>291Â </td><td>19.4Â </td></tr><tr><td><em>LIb</em>Â </td><td>527Â </td><td>523Â </td><td>99.2Â </td><td>85Â </td><td>16.1Â </td></tr><tr><td>TotalÂ </td><td>2,024Â </td><td>1,850Â </td><td>91.4Â </td><td>376Â </td><td>18.6Â </td></tr></tbody></table></div><div content-id="DSU015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Â </th><th>All insertion sitesÂ </th><th>No. of polymorphic sitesÂ </th><th>Polymorphic ratio (%)Â </th><th>No. of cultivar-specific sitesÂ </th><th>Cultivar-specific ratio (%)Â </th></tr></thead><tbody><tr><td><em>Rtsp-1</em>Â </td><td>1,497Â </td><td>1,327Â </td><td>88.6Â </td><td>291Â </td><td>19.4Â </td></tr><tr><td><em>LIb</em>Â </td><td>527Â </td><td>523Â </td><td>99.2Â </td><td>85Â </td><td>16.1Â </td></tr><tr><td>TotalÂ </td><td>2,024Â </td><td>1,850Â </td><td>91.4Â </td><td>376Â </td><td>18.6Â </td></tr></tbody></table></div></div><div data-id="DSU015F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01502.jpeg?Expires=2147483647&amp;Signature=RvZyPV-fCwFc~asaM5V3AuTpFFGhmby6OEkcMqymCZXa9OoUyIo7G5TX6HFkle7O~HrjOa6Fr8EggEoP9LZrDei~lPoow~BNJP2CODQZD-SNI45tLWqYjm8WPjrV9sVyrPt62RgeLwqZgTcGcYYLG5J0Cbi2kZsIc5TPda4xj8XM1Mhfn3keergn3fF4dpel8DTvnVAqOvyWTDVmh-tjiY1U1BRMK-W-he9b5PF1TzPAMLYlQnWayY~B6ckMJyS1-iZyWe~Zc6p6VbZa5hDCxMoD2EV703vltKvGyUkzLqMK8UvEwKJbqlhdVR64vUskT75hRDIbp2oTcPB7WLyY7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of Rtsp_1 and LIb insertion sites identified in this study. The box plots show the distributions of these sites in the 38 cultivars. The central rectangle spans the first quartile to the third quartile. The line inside the rectangle shows the median number (257.5 and 92.5 for Rtsp-1 and LIb, respectively), and the whiskers above and below the box show the maximum and minimum number, respectively." path-from-xml="dsu01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197046/dsu01502.jpeg" path-from-xml="dsu01502.jpeg" target="_blank">View large</a><a section="87197046" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01502.jpeg?Expires=2147483647&amp;Signature=thf3SMkZf0dpDfH02GiTc95GzeMY03Yjl29uIg9r1x0PNOLNeXKZKmhOsFQPuVulaSDQtwrh7xDyETcKfbXYYBeGbb2PvhhPUSnXkYocyQUGIKrXAU9F7ESgOnFmsIDIHRq0~SzERYaxIGOOLnh1BIFuHdxta7v1uejhGu1jTjOlAYJpgBZsDgtA0D7kEhLg1akIPaAgs6shHsmYwoEmaZ7wCbPkivCeafrbJAbMckpxU8d9yYqjJj-3etplLeSnqgiHR3A5FPxKWxQH8pz-tADHazjEea~Z3lS2x5zKTOXxTybCopAsXZiUr-ZlqWwJm1TZabm7odzGOOCLZOioUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197046&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of <em>Rtsp_1</em> and <em>LIb</em> insertion sites identified in this study. The box plots show the distributions of these sites in the 38 cultivars. The central rectangle spans the first quartile to the third quartile. The line inside the rectangle shows the median number (257.5 and 92.5 for <em>Rtsp-1</em> and <em>LIb</em>, respectively), and the whiskers above and below the box show the maximum and minimum number, respectively.</p></div></div><div content-id="DSU015F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01502.jpeg?Expires=2147483647&amp;Signature=RvZyPV-fCwFc~asaM5V3AuTpFFGhmby6OEkcMqymCZXa9OoUyIo7G5TX6HFkle7O~HrjOa6Fr8EggEoP9LZrDei~lPoow~BNJP2CODQZD-SNI45tLWqYjm8WPjrV9sVyrPt62RgeLwqZgTcGcYYLG5J0Cbi2kZsIc5TPda4xj8XM1Mhfn3keergn3fF4dpel8DTvnVAqOvyWTDVmh-tjiY1U1BRMK-W-he9b5PF1TzPAMLYlQnWayY~B6ckMJyS1-iZyWe~Zc6p6VbZa5hDCxMoD2EV703vltKvGyUkzLqMK8UvEwKJbqlhdVR64vUskT75hRDIbp2oTcPB7WLyY7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of Rtsp_1 and LIb insertion sites identified in this study. The box plots show the distributions of these sites in the 38 cultivars. The central rectangle spans the first quartile to the third quartile. The line inside the rectangle shows the median number (257.5 and 92.5 for Rtsp-1 and LIb, respectively), and the whiskers above and below the box show the maximum and minimum number, respectively." path-from-xml="dsu01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197046/dsu01502.jpeg" path-from-xml="dsu01502.jpeg" target="_blank">View large</a><a section="87197046" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01502.jpeg?Expires=2147483647&amp;Signature=thf3SMkZf0dpDfH02GiTc95GzeMY03Yjl29uIg9r1x0PNOLNeXKZKmhOsFQPuVulaSDQtwrh7xDyETcKfbXYYBeGbb2PvhhPUSnXkYocyQUGIKrXAU9F7ESgOnFmsIDIHRq0~SzERYaxIGOOLnh1BIFuHdxta7v1uejhGu1jTjOlAYJpgBZsDgtA0D7kEhLg1akIPaAgs6shHsmYwoEmaZ7wCbPkivCeafrbJAbMckpxU8d9yYqjJj-3etplLeSnqgiHR3A5FPxKWxQH8pz-tADHazjEea~Z3lS2x5zKTOXxTybCopAsXZiUr-ZlqWwJm1TZabm7odzGOOCLZOioUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197046&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of <em>Rtsp_1</em> and <em>LIb</em> insertion sites identified in this study. The box plots show the distributions of these sites in the 38 cultivars. The central rectangle spans the first quartile to the third quartile. The line inside the rectangle shows the median number (257.5 and 92.5 for <em>Rtsp-1</em> and <em>LIb</em>, respectively), and the whiskers above and below the box show the maximum and minimum number, respectively.</p></div></div></div>                            <h3 scrollto-destination=87197049 id="87197049" class="section-title" data-legacy-id=s3c>3.3. Development of SCAR markers for cultivar identification</h3>
<div class="block-child-p">We designed PCR primers using the <em>Rtsp-1</em> insertion sequences of five cultivar-specific sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). We conducted PCR amplification with the 38 sweet potato cultivars using these cultivar-specific insertion primer (RP_*) and <em>Rtsp-1</em>-specific primer (<em>Rtsp-1</em>_ppt), which produced single, distinct and clear bands (Fig.Â <a reveal-id="DSU015F3" data-open="DSU015F3" class="link link-reveal link-table xref-fig">3</a>). The SCAR markers developed in this study can be treated as dominant markers and used for the simple and rapid screening of cultivars. <div data-id="DSU015F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01503.jpeg?Expires=2147483647&amp;Signature=diWAm1vbJyDTUz0CCtn5PLdhRn2~IEnCoskI8krYHJnP7GlKPI~4i8jG8msPxzsz8-eDgumUK3YIwKylfk6Xf-AxkYM6HxrbVCSf0L0cfAJYSUtVQ7GZiQKP4MK3~eCWP9JJ6gxRRa-yM4ZwDxpCe0jsTYDdwUWZ7xt6-36~z4T-xwt7BBINrhDN1J0sLmf8IDB47MBkAAJyaB06CrJwlMzLXiNIsnxYuKc-OK3O6bHdAPap1iYIf6RHHkZ3ZbmZYC28QrHCbFtf9yNOgpRD1CjXJwnL26lg4iFE3ZCOJL9Mb3IR0Nxw4veb9gwmXl6fvic~V2S6iZyIyk9A683WaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel image showing the SCAR markers developed in this study. PCR amplification was conducted using the cultivar-specific insertion primer (RP_*) and the Rtsp-1-specific primer (Rtsp-1_ppt). Descriptions of these primers are provided in Supplementary Data. The arrows indicate the strong and reproducible bands that were specific for each cultivar. Lanes 1â38 correspond to the cultivars listed in Supplementary Data." path-from-xml="dsu01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197052/dsu01503.jpeg" path-from-xml="dsu01503.jpeg" target="_blank">View large</a><a section="87197052" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01503.jpeg?Expires=2147483647&amp;Signature=GF4ZPun4yAlfYu8EMcq3vWkcsbd0pNCA5qcE1Js5HMd3bunaqxlZF2-OA9gPucpGgCkzo8GmJ6xBK55goBaUiwero5646vNsmP6GNigx2JmgzTLpHtp~EW-iIk8HYcAFRiE2TT8F80eViNB3E1yji3yVnYKUgvX0jZSYkzHafLeMqXFTeY3kYsV3pgR9ViymuMMzOcycJ5sNf4lNJ8GNyn2pEmt7~96g3jGmWP7HPyaYVzi-kjukUVo8VHprVFDuP8iYHaOn2UXZbWg6hrakgDDZBHJ90-4sPCrcNpcu5ljBFYS1eC-P5O2dha2F5XYqik4X4WHhgytlG3yFIg~cCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197052&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel image showing the SCAR markers developed in this study. PCR amplification was conducted using the cultivar-specific insertion primer (RP_*) and the <em>Rtsp-1</em>-specific primer (<em>Rtsp-1_</em>ppt). Descriptions of these primers are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. The arrows indicate the strong and reproducible bands that were specific for each cultivar. Lanes 1â38 correspond to the cultivars listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div><div content-id="DSU015F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01503.jpeg?Expires=2147483647&amp;Signature=diWAm1vbJyDTUz0CCtn5PLdhRn2~IEnCoskI8krYHJnP7GlKPI~4i8jG8msPxzsz8-eDgumUK3YIwKylfk6Xf-AxkYM6HxrbVCSf0L0cfAJYSUtVQ7GZiQKP4MK3~eCWP9JJ6gxRRa-yM4ZwDxpCe0jsTYDdwUWZ7xt6-36~z4T-xwt7BBINrhDN1J0sLmf8IDB47MBkAAJyaB06CrJwlMzLXiNIsnxYuKc-OK3O6bHdAPap1iYIf6RHHkZ3ZbmZYC28QrHCbFtf9yNOgpRD1CjXJwnL26lg4iFE3ZCOJL9Mb3IR0Nxw4veb9gwmXl6fvic~V2S6iZyIyk9A683WaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Agarose gel image showing the SCAR markers developed in this study. PCR amplification was conducted using the cultivar-specific insertion primer (RP_*) and the Rtsp-1-specific primer (Rtsp-1_ppt). Descriptions of these primers are provided in Supplementary Data. The arrows indicate the strong and reproducible bands that were specific for each cultivar. Lanes 1â38 correspond to the cultivars listed in Supplementary Data." path-from-xml="dsu01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197052/dsu01503.jpeg" path-from-xml="dsu01503.jpeg" target="_blank">View large</a><a section="87197052" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01503.jpeg?Expires=2147483647&amp;Signature=GF4ZPun4yAlfYu8EMcq3vWkcsbd0pNCA5qcE1Js5HMd3bunaqxlZF2-OA9gPucpGgCkzo8GmJ6xBK55goBaUiwero5646vNsmP6GNigx2JmgzTLpHtp~EW-iIk8HYcAFRiE2TT8F80eViNB3E1yji3yVnYKUgvX0jZSYkzHafLeMqXFTeY3kYsV3pgR9ViymuMMzOcycJ5sNf4lNJ8GNyn2pEmt7~96g3jGmWP7HPyaYVzi-kjukUVo8VHprVFDuP8iYHaOn2UXZbWg6hrakgDDZBHJ90-4sPCrcNpcu5ljBFYS1eC-P5O2dha2F5XYqik4X4WHhgytlG3yFIg~cCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197052&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Agarose gel image showing the SCAR markers developed in this study. PCR amplification was conducted using the cultivar-specific insertion primer (RP_*) and the <em>Rtsp-1</em>-specific primer (<em>Rtsp-1_</em>ppt). Descriptions of these primers are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. The arrows indicate the strong and reproducible bands that were specific for each cultivar. Lanes 1â38 correspond to the cultivars listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div>                            <h3 scrollto-destination=87197054 id="87197054" class="section-title" data-legacy-id=s3d>3.4. Genetic relationships among sweet potato cultivars</h3>
<div class="block-child-p">To investigate the genetic relationships among the 38 sweet potato cultivars, we conducted a phylogenetic analysis based on the <em>Rtsp-1</em> and <em>LIb</em> insertion sites. Basically, sweet potato is cultivated by vegetative propagation. However, Japanese sweet potato cultivars had been developed with crossing the limited number of cultivars, which indicated most cultivars used in this study were genetically related. The genetic distance scores were calculated based on Nei's and Li's coefficient index (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The phylogenetic trees were constructed using the neighbour-joining method (Fig.Â <a reveal-id="DSU015F4" data-open="DSU015F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The genetic relationships among cultivars based on the <em>Rtsp-1</em> and <em>LIb</em> insertion polymorphisms were consistent well each other (Fig.Â <a reveal-id="DSU015F4" data-open="DSU015F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Pedigree information was known for some cultivars (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), which was also consistent with the genetic relationships in the phylogenetic trees. In addition, the parent and offspring cultivars shown in the pedigree information had lower genetic distance (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), which indicated they are genetically closely-related. This suggests that obtaining genotyping information based on the active retrotransposon insertion sites may be useful for determining the genetic relationships among sweet potato cultivars. <div data-id="DSU015F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01504.jpeg?Expires=2147483647&amp;Signature=nej6L~24melib-H~cBfW4Zpa7pXtzmt6kamPc8VHnov7jvlHBIVubXWV2UeKGFhoRYDMder-F8HDR-Gh8nGDqIWQP94N9-jUkwVUE~whoGPFHIgjNV0XNU3n2m4W2~5TnGCDyLhCZAEYemm7F3YT7JEmwR62v1PDeVEYmUtRAV85bwo6PXMpRO0oQJW0Nwa15In6eUUT439yzCLesazo0kzHOPpMHIimFoeS-pgeiz8lk3~6pZ-xfBX32hK8GHJEuQHZIw7oXMPUta0sCpBRl~g2bG-9aIMMim5TltZperK7tBbdq0nrmKNBQA-oITuCpIW7GZVIaHRhlW-cNqx7Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis based on the Rtsp-1 insertion polymorphisms of the cultivars. The phylogenetic tree was constructed using the neighbour-joining method. The bootstrap values are shown. +The genetic relationships based on the Rtsp-1 and LIb (Supplementary Data) insertion polymorphisms were well consistent. *The genetic relationships among cultivars agreed well with the pedigree information (Supplementary Data)." path-from-xml="dsu01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197055/dsu01504.jpeg" path-from-xml="dsu01504.jpeg" target="_blank">View large</a><a section="87197055" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01504.jpeg?Expires=2147483647&amp;Signature=2MTDqb4UrLaHbQikCMqIwpvdb9Dd7s3aWzzymMsDWeuUOKaZ21tO9hV2L1d6ydRYqSt5KEJrd61pz2QbC~fQEH~OAbCI4ZFVTxtKXqFTxKviiE~U7JvilIXsV9BdXbW8A~JLyMkfBQuTJQscNjXi5cB0cDQ7lOxyJzWaZ7XsoavAZgidfzgr1zaQhD3o5CTduISHgHCDiLE4A5LKNX3mMBnzpJ5gm9rOwSN4OgHXknFcDX3Nujs5DTCs-xTsn1GNwoFjOXdmErqkUJchOO-UWgtVYMt8xhpK28a7KAe5ZtmiydlSWFQ3PRWj49CiCbdU0SuVaGV0w~ng0-WVcXAk~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197055&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis based on the <em>Rtsp-1</em> insertion polymorphisms of the cultivars. The phylogenetic tree was constructed using the neighbour-joining method. The bootstrap values are shown. <sup>+</sup>The genetic relationships based on the <em>Rtsp-1</em> and <em>LIb</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>) insertion polymorphisms were well consistent. *The genetic relationships among cultivars agreed well with the pedigree information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p></div></div><div content-id="DSU015F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/m_dsu01504.jpeg?Expires=2147483647&amp;Signature=nej6L~24melib-H~cBfW4Zpa7pXtzmt6kamPc8VHnov7jvlHBIVubXWV2UeKGFhoRYDMder-F8HDR-Gh8nGDqIWQP94N9-jUkwVUE~whoGPFHIgjNV0XNU3n2m4W2~5TnGCDyLhCZAEYemm7F3YT7JEmwR62v1PDeVEYmUtRAV85bwo6PXMpRO0oQJW0Nwa15In6eUUT439yzCLesazo0kzHOPpMHIimFoeS-pgeiz8lk3~6pZ-xfBX32hK8GHJEuQHZIw7oXMPUta0sCpBRl~g2bG-9aIMMim5TltZperK7tBbdq0nrmKNBQA-oITuCpIW7GZVIaHRhlW-cNqx7Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis based on the Rtsp-1 insertion polymorphisms of the cultivars. The phylogenetic tree was constructed using the neighbour-joining method. The bootstrap values are shown. +The genetic relationships based on the Rtsp-1 and LIb (Supplementary Data) insertion polymorphisms were well consistent. *The genetic relationships among cultivars agreed well with the pedigree information (Supplementary Data)." path-from-xml="dsu01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197055/dsu01504.jpeg" path-from-xml="dsu01504.jpeg" target="_blank">View large</a><a section="87197055" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu01504.jpeg?Expires=2147483647&amp;Signature=2MTDqb4UrLaHbQikCMqIwpvdb9Dd7s3aWzzymMsDWeuUOKaZ21tO9hV2L1d6ydRYqSt5KEJrd61pz2QbC~fQEH~OAbCI4ZFVTxtKXqFTxKviiE~U7JvilIXsV9BdXbW8A~JLyMkfBQuTJQscNjXi5cB0cDQ7lOxyJzWaZ7XsoavAZgidfzgr1zaQhD3o5CTduISHgHCDiLE4A5LKNX3mMBnzpJ5gm9rOwSN4OgHXknFcDX3Nujs5DTCs-xTsn1GNwoFjOXdmErqkUJchOO-UWgtVYMt8xhpK28a7KAe5ZtmiydlSWFQ3PRWj49CiCbdU0SuVaGV0w~ng0-WVcXAk~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197055&amp;ar=2754541&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis based on the <em>Rtsp-1</em> insertion polymorphisms of the cultivars. The phylogenetic tree was constructed using the neighbour-joining method. The bootstrap values are shown. <sup>+</sup>The genetic relationships based on the <em>Rtsp-1</em> and <em>LIb</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>) insertion polymorphisms were well consistent. *The genetic relationships among cultivars agreed well with the pedigree information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p></div></div></div>                            <h2 scrollto-destination=87197058 id="87197058" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, we conducted DNA fingerprinting based on the insertion polymorphisms of the active retrotransposon families (<em>Rtsp-1</em> and <em>LIb</em>) in the sweet potato. The application of a MiSeq sequencing platform, which is a bench-top high-throughput sequencer, allowed &gt;1.3 Gb of high-quality sequencing data to be produced at a low cost in a short time. We successfully identified 2,024 insertion sites for two types of retrotransposon families and obtained 76,912 (2,024 insertion sites Ã 38 cultivars) genotyping data points in a single sequencing run. We developed several SCAR markers based on the cultivar-specific insertion sites (Fig.Â <a reveal-id="DSU015F3" data-open="DSU015F3" class="link link-reveal link-table xref-fig">3</a>). This type of marker has several advantages compared with simple sequence repeat (SSR) marker or amplified fragment length polymorphism (AFLP) marker, because they can be used to produce specific and easily recognizable bands by agarose gel electrophoresis (Fig.Â <a reveal-id="DSU015F3" data-open="DSU015F3" class="link link-reveal link-table xref-fig">3</a>).<sup><a reveal-id="DSU015C26" data-open="DSU015C26" class="link link-ref link-reveal xref-bibr">26</a>â<a reveal-id="DSU015C28" data-open="DSU015C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Thus, the SCAR markers developed in this study could be used for the rapid and precise screening of sweet potato cultivars in breeding programmes. Furthermore, the genetic relationships among sweet potato cultivars can be determined based on the genotyping data, which were demonstrated by the phylogenetic analysis (Fig.Â <a reveal-id="DSU015F4" data-open="DSU015F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Our methods should accelerate sweet potato genotyping and they provide a practical method for cultivar identification.</p><p>The sweet potato is a non-model crop species and its whole genome sequence information is not available. It is a hexaploid plant (2<em>n</em> = 6<em>x</em> = 90) with an estimated genome size of 2,200â3,000 Mb.<sup><a reveal-id="DSU015C29" data-open="DSU015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Recently, comprehensive transcriptome sequencing of the sweet potato was reported in three studies.<sup><a reveal-id="DSU015C29" data-open="DSU015C29" class="link link-ref link-reveal xref-bibr">29</a>â<a reveal-id="DSU015C31" data-open="DSU015C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Thus, genetic analyses of the sweet potato based on molecular markers have been delayed compared with other plant species. In general, genotyping with a high-throughput sequencing system, such as the Illumina HiSeq 2000, has been conducted based on single nucleotide polymorphism (SNP) loci in model species.<sup><a reveal-id="DSU015C32" data-open="DSU015C32" class="link link-ref link-reveal xref-bibr">32</a>â<a reveal-id="DSU015C38" data-open="DSU015C38" class="link link-ref link-reveal xref-bibr">38</a></sup> If a reference genome sequence is available, whole genome resequencing or exome resequencing may be effective for genome-wide SNP and/or indel calling using multiple samples after mapping the reads to the reference genome. In non-model crop species such as the sweet potato, however, these resequencing methods cannot be applied because of the lack of reference sequences. Thus, targeting the sequences of highly polymorphic sites in cultivars may be an efficient alternative method for genotyping these species. Moreover, high-sensitivity targeted sequencing is far less expensive and requires less sequencing data per sample than whole genome resequencing.<sup><a reveal-id="DSU015C14" data-open="DSU015C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU015C16" data-open="DSU015C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU015C19" data-open="DSU015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Our results demonstrated that the active retrotransposon insertion sites were highly polymorphic in cultivated sweet potato varieties, and the targeted sequencing of these sites could be achieved using a MiSeq bench-top system. Thus, the low cost and time-saving DNA fingerprinting methods developed in this study could facilitate the precise identification of cultivars, as well as being used for assessments of genetic diversity and conducting linkage analyses.</p>                            <h2 scrollto-destination=87197063 id="87197063" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by a Research and Development Project for Application in Promoting the New Policy of Agriculture Forestry and Fisheries grant from the Ministry of Agriculture, Forestry and Fisheries of Japan and by the Programme to Disseminate Tenure Tracking System, from the Ministry of Education, Culture, Sports, Science and Technology (MEXT), Japan (to Y.M.).</p>                            <h2 scrollto-destination=87197065 id="87197065" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Drs Yoshihiro Okada and Yasuhiro Takahata at the National Agriculture and Food Research Organization, Kyusyu Okinawa Agricultural Research Center, for providing plant materials and their experimental support. We also thank Drs Nobuyuki Fujii and Kazuho Ikeo at the National Institute of Genetics for the development of data analysis pipelines.</p>                            <h2 scrollto-destination=87197067 id="87197067" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ukoskit</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">Autopolyploidy versus allopolyploidy and low-density randomly amplified polymorphic DNA linkage maps of sweetpotato</div>
          <div class="source">J. Am. Soc. Hort. Sci.</div>Â 
          <div class="year">1997</div>
          <div class="volume">122</div>
          <div class="fpage">822</div>
          <div class="lpage">8</div>
        </div></div></div></div><div content-id="DSU015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kriegner</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Burg</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mwanga</div>
               <div class="given-names">R.O.</div>
            </div>
          </div>
          <div class="article-title">A genetic linkage map of sweetpotato (<em>Ipomoea batatas</em> (L.) Lam.) based on AFLP markers</div>
          <div class="source">Mol. Breeding</div>Â 
          <div class="year">2003</div>
          <div class="volume">11</div>
          <div class="fpage">169</div>
          <div class="lpage">85</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20sweetpotato%20(Ipomoea%20batatas%20(L.)%20Lam.)%20based%20on%20AFLP%20markers&amp;author=A.%20Kriegner&amp;author=K.%20Burg&amp;author=R.O.%20Mwanga&amp;publication_year=2003&amp;journal=Mol.%20Breeding&amp;volume=11&amp;pages=169-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022870917230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022870917230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022870917230"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cervantes-Flores</div>
               <div class="given-names">J.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sosinski</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pecota</div>
               <div class="given-names">K.V.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Identification of quantitative trait loci for dry-matter, starch, and Î²-carotene content in sweetpotato</div>
          <div class="source">Mol. Breeding</div>Â 
          <div class="year">2010</div>
          <div class="volume">28</div>
          <div class="fpage">201</div>
          <div class="lpage">16</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20for%20dry-matter%2C%20starch%2C%20and%20%CE%B2-carotene%20content%20in%20sweetpotato&amp;author=J.C.%20Cervantes-Flores&amp;author=B.%20Sosinski&amp;author=K.V.%20Pecota&amp;publication_year=2010&amp;journal=Mol.%20Breeding&amp;volume=28&amp;pages=201-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-010-9474-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-010-9474-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-010-9474-5"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">A.X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Q.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">Q.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">L.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhai</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">S.Z.</div>
            </div>
          </div>
          <div class="article-title">Establishment of molecular linkage maps using SRAP markers in sweet potato</div>
          <div class="source">Acta Agronom. Sin.</div>Â 
          <div class="year">2010</div>
          <div class="volume">36</div>
          <div class="fpage">1286</div>
          <div class="lpage">95</div>
        </div></div></div></div><div content-id="DSU015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jie</div>
               <div class="given-names">Q.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">A genetic linkage map based on AFLP and SSR markers and mapping of QTL for dry-matter content in sweetpotato</div>
          <div class="source">Mol. Breeding</div>Â 
          <div class="year">2013</div>
          <div class="comment">
            <a class="link link-uri" href="http://dx.doi.org/10.1007/s11032-013-9908-y" target="_blank">doi:10.1007/s11032-013-9908-y</a>
          </div>
        </div></div></div></div><div content-id="DSU015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bennetzen</div>
               <div class="given-names">J.L.</div>
            </div>
          </div>
          <div class="article-title">Plant retrotransposons</div>
          <div class="source">Annu. Rev. Genet.</div>Â 
          <div class="year">1999</div>
          <div class="volume">33</div>
          <div class="fpage">479</div>
          <div class="lpage">532</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20retrotransposons&amp;author=A.%20Kumar&amp;author=J.L.%20Bennetzen&amp;publication_year=1999&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=33&amp;pages=479-532" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.33.1.479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.33.1.479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.33.1.479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10690416" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Feschotte</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wessler</div>
               <div class="given-names">S.R.</div>
            </div>
          </div>
          <div class="article-title">Plant transposable elements: where genetics meets genomics</div>
          <div class="source">Nat. Rev. Genet.</div>Â 
          <div class="year">2002</div>
          <div class="volume">3</div>
          <div class="fpage">329</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transposable%20elements%3A%20where%20genetics%20meets%20genomics&amp;author=C.%20Feschotte&amp;author=N.%20Jiang&amp;author=S.R.%20Wessler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=329-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988759" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levin</div>
               <div class="given-names">H.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Moran</div>
               <div class="given-names">J.V.</div>
            </div>
          </div>
          <div class="article-title">Dynamic interactions between transposable elements and their hosts</div>
          <div class="source">Nat. Rev. Genet.</div>Â 
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">615</div>
          <div class="lpage">27</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20interactions%20between%20transposable%20elements%20and%20their%20hosts&amp;author=H.L.%20Levin&amp;author=J.V.%20Moran&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=615-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21850042" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tahara</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Aoki</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Suzuka</div>
               <div class="given-names">S.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Isolation of an active element from a high-copy-number family of retrotransposons in the sweetpotato genome</div>
          <div class="source">Mol. Genet. Genomics</div>Â 
          <div class="year">2004</div>
          <div class="volume">272</div>
          <div class="fpage">116</div>
          <div class="lpage">27</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20an%20active%20element%20from%20a%20high-copy-number%20family%20of%20retrotransposons%20in%20the%20sweetpotato%20genome&amp;author=M.%20Tahara&amp;author=T.%20Aoki&amp;author=S.%20Suzuka&amp;publication_year=2004&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=272&amp;pages=116-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-004-1044-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-004-1044-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-004-1044-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15480792" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamashita</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tahara</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">A LINE-type retrotransposon active in meristem stem cells causes heritable transpositions in the sweetpotato genome</div>
          <div class="source">Plant Mol. Biol.</div>Â 
          <div class="year">2006</div>
          <div class="volume">61</div>
          <div class="fpage">79</div>
          <div class="lpage">94</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20LINE-type%20retrotransposon%20active%20in%20meristem%20stem%20cells%20causes%20heritable%20transpositions%20in%20the%20sweetpotato%20genome&amp;author=H.%20Yamashita&amp;author=M.%20Tahara&amp;publication_year=2006&amp;journal=Plant%20Mol.%20Biol.&amp;volume=61&amp;pages=79-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-005-6002-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-005-6002-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-005-6002-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16786293" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Waugh</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">McLean</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Flavell</div>
               <div class="given-names">A.J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Genetic distribution of Bare-1-like retrotransposable elements in the barley genome revealed by sequence-specific amplification polymorphisms (S-SAP)</div>
          <div class="source">Mol. Gen. Genet.</div>Â 
          <div class="year">1997</div>
          <div class="volume">253</div>
          <div class="fpage">687</div>
          <div class="lpage">94</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20distribution%20of%20Bare-1-like%20retrotransposable%20elements%20in%20the%20barley%20genome%20revealed%20by%20sequence-specific%20amplification%20polymorphisms%20(S-SAP)&amp;author=R.%20Waugh&amp;author=K.%20McLean&amp;author=A.J.%20Flavell&amp;publication_year=1997&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=253&amp;pages=687-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380050372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380050372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9079879" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hirochika</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="article-title">Applications of retrotransposons as genetic tools in plant biology</div>
          <div class="source">Trends Plant Sci.</div>Â 
          <div class="year">2001</div>
          <div class="volume">6</div>
          <div class="fpage">127</div>
          <div class="lpage">34</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20retrotransposons%20as%20genetic%20tools%20in%20plant%20biology&amp;author=A.%20Kumar&amp;author=H.%20Hirochika&amp;publication_year=2001&amp;journal=Trends%20Plant%20Sci.&amp;volume=6&amp;pages=127-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(00)01860-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(00)01860-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(00)01860-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11239612" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Syed</div>
               <div class="given-names">N.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sureshsundar</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wilkinson</div>
               <div class="given-names">M.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bhau</div>
               <div class="given-names">B.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cavalcanti</div>
               <div class="given-names">J.J.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Flavell</div>
               <div class="given-names">A.J.</div>
            </div>
          </div>
          <div class="article-title">Ty1-<em>copia</em> retrotransposon-based SSAP marker development in cashew (<em>Anacardium occidentale</em> L.)</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2005</div>
          <div class="volume">110</div>
          <div class="fpage">1195</div>
          <div class="lpage">202</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ty1-copia%20retrotransposon-based%20SSAP%20marker%20development%20in%20cashew%20(Anacardium%20occidentale%20L.)&amp;author=N.H.%20Syed&amp;author=S.%20Sureshsundar&amp;author=M.J.%20Wilkinson&amp;author=B.S.%20Bhau&amp;author=J.J.V.%20Cavalcanti&amp;author=A.J.%20Flavell&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1195-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1948-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1948-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1948-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15761718" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kalendar</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">The use of retrotransposon-based molecular markers to analyze genetic diversity</div>
          <div class="source">Ratar. Povrt.</div>Â 
          <div class="year">2011</div>
          <div class="volume">48</div>
          <div class="fpage">261</div>
          <div class="lpage">74</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20retrotransposon-based%20molecular%20markers%20to%20analyze%20genetic%20diversity&amp;author=R.%20Kalendar&amp;publication_year=2011&amp;journal=Ratar.%20Povrt.&amp;volume=48&amp;pages=261-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5937/ratpov1102261K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5937%2Fratpov1102261K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5937%2Fratpov1102261K"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kalendar</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Flavell</div>
               <div class="given-names">A.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ellis</div>
               <div class="given-names">T.H.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sjakste</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Moisy</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schulman</div>
               <div class="given-names">A.H.</div>
            </div>
          </div>
          <div class="article-title">Analysis of plant diversity with retrotransposon-based molecular markers</div>
          <div class="source">Heredity</div>Â 
          <div class="year">2011</div>
          <div class="volume">106</div>
          <div class="fpage">520</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20plant%20diversity%20with%20retrotransposon-based%20molecular%20markers&amp;author=R.%20Kalendar&amp;author=A.J.%20Flavell&amp;author=T.H.N.%20Ellis&amp;author=T.%20Sjakste&amp;author=C.%20Moisy&amp;author=A.H.%20Schulman&amp;publication_year=2011&amp;journal=Heredity&amp;volume=106&amp;pages=520-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.93" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.93"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20683483" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Poczai</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Varga</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laos</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Advances in plant gene-targeted and functional markers: a review</div>
          <div class="source">Plant Methods</div>Â 
          <div class="year">2013</div>
          <div class="volume">9</div>
          <div class="fpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20gene-targeted%20and%20functional%20markers%3A%20a%20review&amp;author=P.%20Poczai&amp;author=I.%20Varga&amp;author=M.%20Laos&amp;publication_year=2013&amp;journal=Plant%20Methods&amp;volume=9&amp;pages=6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1746-4811-9-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1746-4811-9-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1746-4811-9-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23406322" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Loman</div>
               <div class="given-names">N.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Misra</div>
               <div class="given-names">R.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dallman</div>
               <div class="given-names">T.J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Performance comparison of benchtop high-throughput sequencing platforms</div>
          <div class="source">Nat. Biotechnol.</div>Â 
          <div class="year">2012</div>
          <div class="volume">30</div>
          <div class="fpage">434</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20comparison%20of%20benchtop%20high-throughput%20sequencing%20platforms&amp;author=N.J.%20Loman&amp;author=R.V.%20Misra&amp;author=T.J.%20Dallman&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=434-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22522955" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pallen</div>
               <div class="given-names">M.J.</div>
            </div>
          </div>
          <div class="article-title">Reply to updating benchtop sequencing performance comparison</div>
          <div class="source">Nat. Biotechnol.</div>Â 
          <div class="year">2013</div>
          <div class="volume">31</div>
          <div class="fpage">296</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reply%20to%20updating%20benchtop%20sequencing%20performance%20comparison&amp;author=M.J.%20Pallen&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.&amp;volume=31&amp;pages=296" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563422" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xing</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Witherspoon</div>
               <div class="given-names">D.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jorde</div>
               <div class="given-names">L.B.</div>
            </div>
          </div>
          <div class="article-title">Mobile element biology: new possibilities with high-throughput sequencing</div>
          <div class="source">Trends Genet.</div>Â 
          <div class="year">2013</div>
          <div class="volume">29</div>
          <div class="fpage">280</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20element%20biology%3A%20new%20possibilities%20with%20high-throughput%20sequencing&amp;author=J.%20Xing&amp;author=D.J.%20Witherspoon&amp;author=L.B.%20Jorde&amp;publication_year=2013&amp;journal=Trends%20Genet.&amp;volume=29&amp;pages=280-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23312846" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmieder</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Edwards</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Quality control and preprocessing of metagenomic datasets</div>
          <div class="source">Bioinformatics</div>Â 
          <div class="year">2011</div>
          <div class="volume">27</div>
          <div class="fpage">863</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kent</div>
               <div class="given-names">W.J.</div>
            </div>
          </div>
          <div class="article-title">BLATâThe BLAST-like alignment tool</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2002</div>
          <div class="volume">12</div>
          <div class="fpage">656</div>
          <div class="lpage">64</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94The%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larkin</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Blackshields</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">N.P.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Clustal W and Clustal X version 2.0</div>
          <div class="source">Bioinformatics</div>Â 
          <div class="year">2007</div>
          <div class="volume">23</div>
          <div class="fpage">2947</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.H.</div>
            </div>
          </div>
          <div class="article-title">Mathematical model for studying genetic variation in terms of restriction endonucleases</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">1979</div>
          <div class="volume">76</div>
          <div class="fpage">5269</div>
          <div class="lpage">73</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mathematical%20model%20for%20studying%20genetic%20variation%20in%20terms%20of%20restriction%20endonucleases&amp;author=M.%20Nei&amp;author=W.H.%20Li&amp;publication_year=1979&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=76&amp;pages=5269-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.76.10.5269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.76.10.5269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.76.10.5269"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pavlicek</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hrda</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Flegr</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">FreeTreeâFreeware program for construction of phylogenetic trees on the basis of distance data and bootstrap/jackknife analysis of the tree robustness. Application in the RAPD analysis of genus</div>
          <div class="source">Folia Biol.</div>Â 
          <div class="year">1999</div>
          <div class="volume">45</div>
          <div class="fpage">97</div>
          <div class="lpage">9</div>
        </div></div></div></div><div content-id="DSU015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stecher</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>
          <div class="source">Mol. Biol. Evol.</div>Â 
          <div class="year">2011</div>
          <div class="volume">28</div>
          <div class="fpage">2731</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Monden</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Naito</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Okumoto</div>
               <div class="given-names">Y.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">High potential of a transposon <em>mPing</em> as a marker system in <em>japonica</em> times <em>japonica</em> cross in rice</div>
          <div class="source">DNA Res.</div>Â 
          <div class="year">2009</div>
          <div class="volume">16</div>
          <div class="fpage">131</div>
          <div class="lpage">40</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20potential%20of%20a%20transposon%20mPing%20as%20a%20marker%20system%20in%20japonica%20times%20japonica%20cross%20in%20rice&amp;author=Y.%20Monden&amp;author=K.%20Naito&amp;author=Y.%20Okumoto&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=131-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19270311" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kiran</div>
               <div class="given-names">U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Khan</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mirza</div>
               <div class="given-names">K.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ram</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Abdin</div>
               <div class="given-names">M.Z.</div>
            </div>
          </div>
          <div class="article-title">SCAR markers: a potential tool for authentication of herbal drugs</div>
          <div class="source">Fitoterapia</div>Â 
          <div class="year">2010</div>
          <div class="volume">81</div>
          <div class="fpage">969</div>
          <div class="lpage">76</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SCAR%20markers%3A%20a%20potential%20tool%20for%20authentication%20of%20herbal%20drugs&amp;author=U.%20Kiran&amp;author=S.%20Khan&amp;author=K.J.%20Mirza&amp;author=M.%20Ram&amp;author=M.Z.%20Abdin&amp;publication_year=2010&amp;journal=Fitoterapia&amp;volume=81&amp;pages=969-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fitote.2010.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fitote.2010.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fitote.2010.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20723584" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cirillo</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gaudio</div>
               <div class="given-names">S.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bernardo</div>
               <div class="given-names">G.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Galano</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Galderisi</div>
               <div class="given-names">U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cipollaro</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">A new SCAR marker potentially useful to distinguish Italian cattle breeds</div>
          <div class="source">Food Chem.</div>Â 
          <div class="year">2012</div>
          <div class="volume">130</div>
          <div class="fpage">172</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20SCAR%20marker%20potentially%20useful%20to%20distinguish%20Italian%20cattle%20breeds&amp;author=A.%20Cirillo&amp;author=S.D.%20Gaudio&amp;author=G.B.%20Bernardo&amp;author=G.%20Galano&amp;author=U.%20Galderisi&amp;author=M.%20Cipollaro&amp;publication_year=2012&amp;journal=Food%20Chem.&amp;volume=130&amp;pages=172-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodchem.2011.06.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodchem.2011.06.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodchem.2011.06.043"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tao</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">H.Y.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Digital gene expression analysis based on integrated de novo transcriptome assembly of sweet potato [<em>Ipomoea batatas</em> (L.) Lam]</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e36234</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20gene%20expression%20analysis%20based%20on%20integrated%20de%20novo%20transcriptome%20assembly%20of%20sweet%20potato%20%5BIpomoea%20batatas%20(L.)%20Lam%5D&amp;author=X.%20Tao&amp;author=Y.H.%20Gu&amp;author=H.Y.%20Wang&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e36234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0036234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0036234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0036234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22558397" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schafleitner</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tincopa</div>
               <div class="given-names">L.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Palomino</div>
               <div class="given-names">O.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">A sweetpotato gene index established by de novo assembly of pyrosequencing and Sanger sequences and mining for gene-based microsatellite markers</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2010</div>
          <div class="volume">11</div>
          <div class="fpage">604</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sweetpotato%20gene%20index%20established%20by%20de%20novo%20assembly%20of%20pyrosequencing%20and%20Sanger%20sequences%20and%20mining%20for%20gene-based%20microsatellite%20markers&amp;author=R.%20Schafleitner&amp;author=L.R.%20Tincopa&amp;author=O.%20Palomino&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-604"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20977749" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">Z.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fang</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">
            <em>De novo</em> assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of cSSR markers in sweet potato (<em>Ipomoea batatas</em>)</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2010</div>
          <div class="volume">11</div>
          <div class="fpage">726</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20root%20transcriptome%20using%20Illumina%20paired-end%20sequencing%20and%20development%20of%20cSSR%20markers%20in%20sweet%20potato%20(Ipomoea%20batatas)&amp;author=Z.%20Wang&amp;author=B.%20Fang&amp;author=J.%20Chen&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=726" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-726" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-726" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-726"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21182800" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brachi</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Faure</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Horton</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Linkage and association mapping of <em>Arabidopsis thaliana</em> flowering time in nature</div>
          <div class="source">PLoS Genet.</div>Â 
          <div class="year">2010</div>
          <div class="volume">6</div>
          <div class="fpage">e1000940</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20and%20association%20mapping%20of%20Arabidopsis%20thaliana%20flowering%20time%20in%20nature&amp;author=B.%20Brachi&amp;author=N.%20Faure&amp;author=M.%20Horton&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1000940" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000940" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000940" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000940"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20463887" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clark</div>
               <div class="given-names">R.M.</div>
            </div>
          </div>
          <div class="article-title">Genome-wide association studies coming of age in rice</div>
          <div class="source">Nat. Genet.</div>Â 
          <div class="year">2010</div>
          <div class="volume">42</div>
          <div class="fpage">926</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20coming%20of%20age%20in%20rice&amp;author=R.M.%20Clark&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=926-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1110-926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1110-926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1110-926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20980987" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wei</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sang</div>
               <div class="given-names">T.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>
          <div class="source">Nat. Genet.</div>Â 
          <div class="year">2010</div>
          <div class="volume">42</div>
          <div class="fpage">961</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lam</div>
               <div class="given-names">H.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">X.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection</div>
          <div class="source">Nat. Genet.</div>Â 
          <div class="year">2010</div>
          <div class="volume">42</div>
          <div class="fpage">1053</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%20of%2031%20wild%20and%20cultivated%20soybean%20genomes%20identifies%20patterns%20of%20genetic%20diversity%20and%20selection&amp;author=H.M.%20Lam&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1053-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076406" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bergelson</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nordborg</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Borevitz</div>
               <div class="given-names">J.O.</div>
            </div>
          </div>
          <div class="article-title">Association mapping of local climate-sensitive quantitative trait loci in <em>Arabidopsis thaliana</em></div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2010</div>
          <div class="volume">107</div>
          <div class="fpage">21199</div>
          <div class="lpage">204</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20local%20climate-sensitive%20quantitative%20trait%20loci%20in%20Arabidopsis%20thaliana&amp;author=Y.%20Li&amp;author=Y.%20Huang&amp;author=J.%20Bergelson&amp;author=M.%20Nordborg&amp;author=J.O.%20Borevitz&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=21199-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1007431107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1007431107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1007431107"> </span></div>Â </div></div></div></div></div><div content-id="DSU015C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Paul</div>
               <div class="given-names">J.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Albrechtsen</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Song</div>
               <div class="given-names">Y.S.</div>
            </div>
          </div>
          <div class="article-title">Genotype and SNP calling from next-generation sequencing data</div>
          <div class="source">Nat. Rev. Genet.</div>Â 
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">443</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%20and%20SNP%20calling%20from%20next-generation%20sequencing%20data&amp;author=R.%20Nielsen&amp;author=J.S.%20Paul&amp;author=A.%20Albrechtsen&amp;author=Y.S.%20Song&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=443-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21587300" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU015C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU015C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamoto</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nagasaki</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yonemaru</div>
               <div class="given-names">J.I.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2010</div>
          <div class="volume">11</div>
          <div class="fpage">267</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20definition%20of%20the%20pedigree%20haplotypes%20of%20closely%20related%20rice%20cultivars%20by%20means%20of%20genome-wide%20discovery%20of%20single-nucleotide%20polymorphisms&amp;author=T.%20Yamamoto&amp;author=H.%20Nagasaki&amp;author=J.I.%20Yonemaru&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20423466" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754541&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu015/2/dsu015_Supplementary_Data.zip?Expires=2147483647&amp;Signature=f7yLjLJIxbi1wz3SshlY2pGt~YK-yYgjxJ~MSauNVK8ZW4AMnJyU0yt0mi3IjJQkf5nwlwDEXNYlArCUPI4GOB1fOimD7p3LUcgWXY0f70PK3syQzt1MpbVAEclvxwSzaNrSwsY2VsVUzvDLSK-yVFCxKi0wAASy0wossgwJbghwixJHAQ4pbfp1t~V9G-2-yYnO3brs230uRFfbYGxMsO8h8zFGLrS~pQHoyM8xUHvZQGXjKUZCfb26s7jeIUVXyWyme-387~fCL6wQIEXCaX1xh3FMI4YQzwiftVTPdJ4ApoVBXK3MobSqQyVR9fqlzR1l5PTTtFMPSP6kkmVqGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">391</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754541"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f5%2f491%2f2754541">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344244900004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F5%2F491%2F2754541" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754541'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Efficient DNA Fingerprinting Based on the Targeted Sequencing of Active Retrotransposon Insertion Sites Using a Bench-Top High-Throughput Sequencing Platform" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f491%2f2754541">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

