

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/203/1745366","@type":"ScholarlyArticle","name":"Dissecting the stochastic transcription initiation process in live  Escherichia coli ","datePublished":"2016-03-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw009","keywords":["free RNA polymerase","<italic>in vivo</italic> transcription dynamics","rate-limiting steps","reversible closed complex formation","repressor binding dynamics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/203/1745366","author":[{"name":"Lloyd-Price, Jason","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Startceva, Sofia","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Kandavalli, Vinodh","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Chandraseelan, Jerome G.","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Goncalves, Nadia","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Oliveira, Samuel M. D.","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Häkkinen, Antti","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"},{"name":"Ribeiro, Andre S.","affiliation":"Laboratory of Biosystem Dynamics, Department of Signal Processing ,  Tampere University of Technology ,  PO Box 553, Office TC336 ,  33101 Tampere ,  Finland","@type":"Person"}],"description":"Abstract.  We investigate the hypothesis that, in Escherichia coli, while the concentration of RNA polymerases differs in different growth conditions, the fract","pageStart":"203","pageEnd":"214","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Dissecting the stochastic transcription initiation process in live  Escherichia coli ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Dissecting the stochastic transcription initiation process in live  Escherichia coli " />
    <meta property="og:description" content="Abstract.  We investigate the hypothesis that, in Escherichia coli, while the concentration of RNA polymerases differs in different growth conditions, the fract" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/203/1745366" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-03-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Lloyd-Price, Jason" /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Startceva, Sofia" /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Kandavalli, Vinodh" /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Chandraseelan, Jerome G." /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Goncalves, Nadia" /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Oliveira, Samuel M. D." /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Häkkinen, Antti" /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_author" content="Ribeiro, Andre S." /><meta name="citation_author_institution" content="Laboratory of Biosystem Dynamics, Department of Signal Processing, Tampere University of Technology, PO Box 553, Office TC336, 33101 Tampere, Finland" /><meta name="citation_title" content="Dissecting the stochastic transcription initiation process in live Escherichia coli" /><meta name="citation_firstpage" content="203" /><meta name="citation_lastpage" content="214" /><meta name="citation_doi" content="10.1093/dnares/dsw009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=RNA dynamics in live Escherichia coli cells;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=11310-5; " /><meta name="citation_reference" content="citation_title=Real-time kinetics of gene activity in individual bacteria;  citation_journal_title=Cell;  citation_year=2005;  citation_volume=123;  citation_pages=1025-36; " /><meta name="citation_reference" content="citation_title=Probing gene expression in live cells, one protein molecule at a time;  citation_journal_title=Science;  citation_year=2006;  citation_volume=311;  citation_pages=1600-3; " /><meta name="citation_reference" content="citation_title=In vivo kinetics of transcription initiation of the lar promoter in Escherichia coli. Evidence for a sequential mechanism with two rate-limiting steps;  citation_journal_title=BMC Syst. Biol.;  citation_year=2011;  citation_volume=5;  citation_pages=149" /><meta name="citation_reference" content="citation_title=Dynamics of transcription driven by the tetA promoter, one event at a time, in live Escherichia coli cells;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=8472-83; " /><meta name="citation_reference" content="citation_title=Single mRNA Molecules Demonstrate Probabilistic Movement in Living Mammalian Cells;  citation_journal_title=Curr. Biol.;  citation_year=2003;  citation_volume=13;  citation_pages=161-7; " /><meta name="citation_reference" content="citation_title=Stochastic mRNA synthesis in mammalian cells;  citation_journal_title=PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=1707-19; " /><meta name="citation_reference" content="citation_title=Stochasticity in gene expression: from theories to phenotypes;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=451-64; " /><meta name="citation_reference" content="citation_title=Stochastic kinetic analysis of developmental pathway bifurcation in phage λ-infected Escherichia coli cells;  citation_journal_title=Genetics;  citation_year=1998;  citation_volume=149;  citation_pages=1633-48; " /><meta name="citation_reference" content="citation_title=Stochastic gene expression in a single cell;  citation_journal_title=Science;  citation_year=2002;  citation_volume=297;  citation_pages=1183-6; " /><meta name="citation_reference" content="citation_title=Noise in gene expression: origins, consequences, and control;  citation_journal_title=Science;  citation_year=2005;  citation_volume=309;  citation_pages=2010-3; " /><meta name="citation_reference" content="citation_title=Regulation of noise in the expression of a single gene;  citation_journal_title=Nat. Genet.;  citation_year=2002;  citation_volume=31;  citation_pages=69-73; " /><meta name="citation_reference" content="citation_title=It's a noisy business! Genetic regulation at the nanomolar scale;  citation_journal_title=Trends Genet.;  citation_year=1999;  citation_volume=15;  citation_pages=65-9; " /><meta name="citation_reference" content="citation_title=An excitable gene regulatory circuit induces transient cellular differentiation;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=440;  citation_pages=545-50; " /><meta name="citation_reference" content="citation_title=Stochastic protein expression in individual cells at the single molecule level;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=440;  citation_pages=358-62; " /><meta name="citation_reference" content="citation_title=The generation of promoter-mediated transcriptional noise in bacteria;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000109" /><meta name="citation_reference" content="citation_title=General properties of transcriptional time series in Escherichia coli;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=554-60; " /><meta name="citation_reference" content="citation_title=Kinetic models of gene expression including non-coding RNAs;  citation_journal_title=Phys. Rep.;  citation_year=2011;  citation_volume=500;  citation_pages=1-42; " /><meta name="citation_reference" content="citation_title=Mechanism of transcriptional bursting in bacteria;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=158;  citation_pages=314-26; " /><meta name="citation_reference" content="citation_title=Single-molecule dynamics of transcription of the lar promoter;  citation_journal_title=Phys. Biol.;  citation_year=2012;  citation_volume=9; " /><meta name="citation_reference" content="citation_title=Regulation of mean and noise of the in vivo kinetics of transcription under the control of the lac/ara-1 promoter;  citation_journal_title=FEBS Lett.;  citation_year=2012;  citation_volume=586;  citation_pages=3870-5; " /><meta name="citation_reference" content="citation_title=Rate-limiting steps in RNA chain initiation;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1980;  citation_volume=77;  citation_pages=5634-8; " /><meta name="citation_reference" content="citation_title=Mechanism and control of transcription initiation in prokaryotes;  citation_journal_title=Annu. Rev. Biochem.;  citation_year=1985;  citation_volume=54;  citation_pages=171-204; " /><meta name="citation_reference" content="citation_title=A new experimental approach for studying the association between RNA polymeras and the tet promoter of pBR322;  citation_journal_title=Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=1697-706; " /><meta name="citation_reference" content="citation_title=Characterizing rate limiting steps in transcription from RNA production times in live cells;  citation_journal_title=Bioinformatics;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Mechanism of transcription initiation at an activator-dependent promoter defined by single-molecule observation;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=148;  citation_pages=679-89; " /><meta name="citation_reference" content="citation_title=Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=533-8; " /><meta name="citation_reference" content="citation_title=Effect of promoter architecture on the cell-to-cell variability in gene expression;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2011;  citation_volume=7;  citation_pages=e1001100" /><meta name="citation_reference" content="citation_title=Dissecting the functional program of Escherichia coli promoters: the combined mode of action of Lac repressor and AraC activator;  citation_journal_title=Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=3873-81; " /><meta name="citation_reference" content="citation_title=Operator sequence alters gene expression independently of transcription factor occupancy in bacteria;  citation_journal_title=Cell Rep.;  citation_year=2012;  citation_volume=2;  citation_pages=150-61; " /><meta name="citation_reference" content="citation_title=Increased RNA polymerase availability directs resources towards growth at the expense of maintenance;  citation_journal_title=EMBO J.;  citation_year=2009;  citation_volume=28;  citation_pages=2209-19; " /><meta name="citation_reference" content="citation_title=Modulation of Chemical Composition and Other Parameters of the Cell by Growth Rate; citation_publisher=ASM Press, Washington, DC;  citation_title=Escherichia Coli and Salmonella;  citation_year=1996;  citation_pages=1553-69; " /><meta name="citation_reference" content="citation_title=Spatial distribution and diffusive motion of RNA polymerase in live Escherichia coli.;  citation_journal_title=J. Bacteriol.;  citation_year=2011;  citation_volume=193;  citation_pages=5138-46; " /><meta name="citation_reference" content="citation_title=Bacterial nucleoid-associated proteins, nucleoid structure and gene expression;  citation_journal_title=Nat. Rev. Microbiol.;  citation_year=2010;  citation_volume=8;  citation_pages=185-95; " /><meta name="citation_reference" content="citation_title=Activities of constitutive promoters in Escherichia coli.;  citation_journal_title=J. Mol. Biol.;  citation_year=1999;  citation_volume=292;  citation_pages=19-37; " /><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2-ΔΔCt Method;  citation_journal_title=Methods;  citation_year=2001;  citation_volume=25;  citation_pages=402-8; " /><meta name="citation_reference" content="citation_title=Robustness of the division symmetry in Escherichia coli and functional consequences of symmetry breaking;  citation_journal_title=Phys. Biol.;  citation_year=2014;  citation_volume=11;  citation_pages=066005" /><meta name="citation_reference" content="citation_title=Cell segmentation by multi-resolution analysis and maximum likelihood estimation (MAMLE);  citation_journal_title=BMC Bioinformatics;  citation_year=2013;  citation_volume=14;  citation_pages=S8" /><meta name="citation_reference" content="citation_title=CellAging: A tool to study segregation and partitioning in division in cell lineages of Escherichia coli;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=1708-9; " /><meta name="citation_reference" content="citation_title=Estimation of fluorescence-tagged RNA numbers from spot intensities;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1146-53; " /><meta name="citation_reference" content="citation_title=Estimation of GFP-tagged RNA numbers from temporal fluorescence intensity data;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=69-75; " /><meta name="citation_reference" content="citation_title=Kinetics of the cellular intake of a gene expression inducer at high concentrations;  citation_journal_title=Mol. Biosyst.;  citation_year=2015;  citation_volume=11;  citation_pages=2579-87; " /><meta name="citation_reference" content="citation_title=A thermodynamic analysis of the sequence-specific binding of RNA by bacteriophage MS2 coat protein;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=9244-9; " /><meta name="citation_reference" content="citation_title=Physical Nature of Bacterial Cytoplasm;  citation_journal_title=Phys. Rev. Lett.;  citation_year=2006;  citation_volume=96;  citation_pages=98102" /><meta name="citation_reference" content="citation_title=Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=9697-702; " /><meta name="citation_reference" content="citation_title=Mechanism of bacterial transcription initiation: RNA polymerase - promoter binding, isomerization to initiation-competent open complexes, and initiation of RNA synthesis;  citation_journal_title=J. Mol. Biol.;  citation_year=2011;  citation_volume=412;  citation_pages=754-71; " /><meta name="citation_reference" content="citation_title=RNA polymerase-promoter interactions: The comings and goings of RNA polymerase;  citation_journal_title=J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=3019-25; " /><meta name="citation_reference" content="citation_title=The selectivity of transcription;  citation_journal_title=Annu. Rev. Biochem.;  citation_year=1974;  citation_volume=43;  citation_pages=721-75; " /><meta name="citation_reference" content="citation_title=Monitoring abortive initiation;  citation_journal_title=Methods;  citation_year=2009;  citation_volume=47;  citation_pages=25-36; " /><meta name="citation_reference" content="citation_title=Escherichia coli promoter sequences predict in vitro RNA polymerase selectivity;  citation_journal_title=Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=789-800; " /><meta name="citation_reference" content="citation_title=Single-molecule analysis of RNA polymerase transcription;  citation_journal_title=Annu. Rev. Biophys. Biomol. Struct.;  citation_year=2006;  citation_volume=35;  citation_pages=343-60; " /><meta name="citation_reference" content="citation_title=Single-molecule studies of transcription: From one RNA polymerase at a time to the gene expression profile of a cell;  citation_journal_title=J. Mol. Biol.;  citation_year=2011;  citation_volume=412;  citation_pages=814-31; " /><meta name="citation_reference" content="citation_title=Pausing by bacterial RNA polymerase is mediated by mechanistically distinct classes of signals;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=7090-5; " /><meta name="citation_reference" content="citation_title=Effects of transcriptional pausing on gene expression dynamics;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000704" /><meta name="citation_reference" content="citation_title=Isolation and characterization of σ70-retaining transcription elongation complexes from Escherichia coli;  citation_journal_title=Cell;  citation_year=2001;  citation_volume=106;  citation_pages=443-51; " /><meta name="citation_reference" content="citation_title=Insights into transcriptional regulation and sigma competition from an equilibrium model of RNA polymerase binding to DNA;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=5332-7; " /><meta name="citation_reference" content="citation_title=Competition among seven Escherichia coli σ subunits: relative binding affinities to the core RNA polymerase;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=3497-503; " /><meta name="citation_reference" content="citation_title=Bayes Factors;  citation_journal_title=J. Am. Stat. Assoc.;  citation_year=1995;  citation_volume=90;  citation_pages=773-95; " /><meta name="citation_reference" content="citation_title=Promoter clearance and escape in prokaryotes;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2002;  citation_volume=15,77;  citation_pages=191-207; " /><meta name="citation_reference" content="citation_publisher=Duxbury Press, Pacific Grove, CA;  citation_title=The Delta Method. Statistical Inference;  citation_year=2001;  citation_pages=240-5; " /><meta name="citation_reference" content="citation_title=A weighted total least-squares algorithm for fitting a straight line;  citation_journal_title=Meas. Sci. Technol.;  citation_year=2008;  citation_volume=19;  citation_pages=79801" /><meta name="citation_reference" content="citation_title=Growth-rate-dependent partitioning of RNA polymerases in bacteria;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=20245-50; " /><meta name="citation_reference" content="citation_title=Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=1203-10; " /><meta name="citation_reference" content="citation_title=A fast, robust and tunable synthetic gene oscillator;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=516-9; " /><meta name="citation_reference" content="citation_title=Dynamics of transcription of closely spaced promoters in Escherichia coli, one event at a time;  citation_journal_title=J. Theor. Biol.;  citation_year=2012;  citation_volume=301;  citation_pages=83-94; " /><meta name="citation_reference" content="citation_title=In vivo single-molecule kinetics of activation and subsequent activity of the arabinose promoter;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=6544-52; " /><meta name="citation_reference" content="citation_title=Effects of Mg2+ on in vivo transcriptional dynamics of the lar promoter;  citation_journal_title=BioSystems;  citation_year=2012;  citation_volume=107;  citation_pages=129-34; " /><meta name="citation_reference" content="citation_title=Effects of molecular memory and bursting on fluctuations in gene expression;  citation_journal_title=Science;  citation_year=2008;  citation_volume=319;  citation_pages=339-43; " /><meta name="citation_reference" content="citation_publisher=Duxbury Press, Pacific Grove, CA, USA;  citation_title=Simultaneous Estimation and Confidence Bands. Statistical Inference;  citation_year=2001;  citation_pages=559-63; " /><meta name="citation_reference" content="citation_title=Estimating the dimension of a model;  citation_journal_title=Ann. Stat.;  citation_year=1978;  citation_volume=6;  citation_pages=461-4; " /><meta name="citation_reference" content="citation_title=Mechanism of CRP-cAMP activation of lac operon transcription initiation activation of the P1 promoter;  citation_journal_title=J. Mol. Biol.;  citation_year=1984;  citation_volume=180;  citation_pages=881-909; " /><meta name="citation_reference" content="citation_title=Kinetics of open complex formation between Escherichia coli RNA polymerase and the lac UV5 promoter. Evidence for a sequential mechanism involving three steps;  citation_journal_title=Biochemistry;  citation_year=1985;  citation_volume=24;  citation_pages=2712-23; " /><meta name="citation_reference" content="citation_title=Stochastic models of transcription: From single molecules to single cells;  citation_journal_title=Methods;  citation_year=2013;  citation_volume=62;  citation_pages=13-25; " /><meta name="citation_reference" content="citation_title=Transcription stochasticity of complex gene regulation models;  citation_journal_title=Biophys. J.;  citation_year=2012;  citation_volume=103;  citation_pages=1152-61; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/203/17463027/dsw009.pdf" /><meta name="description" content="Abstract.  We investigate the hypothesis that, in Escherichia coli, while the concentration of RNA polymerases differs in different growth conditions, the fract" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/203/1745366" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/203/1745366" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/203/1745366");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw009';
    </script>
<title>
	Dissecting the stochastic transcription initiation process in live Escherichia coli | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745366" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeenATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGxveWQtUHJpY2UsIEphc29uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBCaW9zeXN0ZW0gRHluYW1pY3MsIERlcGFydG1lbnQgb2YgU2lnbmFsIFByb2Nlc3NpbmcsIFRhbXBlcmUgVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5LCBQTyBCb3ggNTUzLCBPZmZpY2UgVEMzMzYsIDMzMTAxIFRhbXBlcmUsIEZpbmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdGFydGNldmEsIFNvZmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBCaW9zeXN0ZW0gRHluYW1pY3MsIERlcGFydG1lbnQgb2YgU2lnbmFsIFByb2Nlc3NpbmcsIFRhbXBlcmUgVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5LCBQTyBCb3ggNTUzLCBPZmZpY2UgVEMzMzYsIDMzMTAxIFRhbXBlcmUsIEZpbmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5kYXZhbGxpLCBWaW5vZGgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEJpb3N5c3RlbSBEeW5hbWljcywgRGVwYXJ0bWVudCBvZiBTaWduYWwgUHJvY2Vzc2luZywgVGFtcGVyZSBVbml2ZXJzaXR5IG9mIFRlY2hub2xvZ3ksIFBPIEJveCA1NTMsIE9mZmljZSBUQzMzNiwgMzMxMDEgVGFtcGVyZSwgRmlubGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoYW5kcmFzZWVsYW4sIEplcm9tZSBHLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvc3lzdGVtIER5bmFtaWNzLCBEZXBhcnRtZW50IG9mIFNpZ25hbCBQcm9jZXNzaW5nLCBUYW1wZXJlIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgUE8gQm94IDU1MywgT2ZmaWNlIFRDMzM2LCAzMzEwMSBUYW1wZXJlLCBGaW5sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR29uY2FsdmVzLCBOYWRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvc3lzdGVtIER5bmFtaWNzLCBEZXBhcnRtZW50IG9mIFNpZ25hbCBQcm9jZXNzaW5nLCBUYW1wZXJlIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgUE8gQm94IDU1MywgT2ZmaWNlIFRDMzM2LCAzMzEwMSBUYW1wZXJlLCBGaW5sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2xpdmVpcmEsIFNhbXVlbCBNLiBELiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvc3lzdGVtIER5bmFtaWNzLCBEZXBhcnRtZW50IG9mIFNpZ25hbCBQcm9jZXNzaW5nLCBUYW1wZXJlIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgUE8gQm94IDU1MywgT2ZmaWNlIFRDMzM2LCAzMzEwMSBUYW1wZXJlLCBGaW5sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSMOka2tpbmVuLCBBbnR0aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvc3lzdGVtIER5bmFtaWNzLCBEZXBhcnRtZW50IG9mIFNpZ25hbCBQcm9jZXNzaW5nLCBUYW1wZXJlIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgUE8gQm94IDU1MywgT2ZmaWNlIFRDMzM2LCAzMzEwMSBUYW1wZXJlLCBGaW5sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmliZWlybywgQW5kcmUgUy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEJpb3N5c3RlbSBEeW5hbWljcywgRGVwYXJ0bWVudCBvZiBTaWduYWwgUHJvY2Vzc2luZywgVGFtcGVyZSBVbml2ZXJzaXR5IG9mIFRlY2hub2xvZ3ksIFBPIEJveCA1NTMsIE9mZmljZSBUQzMzNiwgMzMxMDEgVGFtcGVyZSwgRmlubGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGlzc2VjdGluZyB0aGUgc3RvY2hhc3RpYyB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb24gcHJvY2VzcyBpbiBsaXZlIEVzY2hlcmljaGlhIGNvbGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIGR5bmFtaWNzIGluIGxpdmUgRXNjaGVyaWNoaWEgY29saSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMTMxMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWFsLXRpbWUga2luZXRpY3Mgb2YgZ2VuZSBhY3Rpdml0eSBpbiBpbmRpdmlkdWFsIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTEwMjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JpbmcgZ2VuZSBleHByZXNzaW9uIGluIGxpdmUgY2VsbHMsIG9uZSBwcm90ZWluIG1vbGVjdWxlIGF0IGEgdGltZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTE7ICBjaXRhdGlvbl9wYWdlcz0xNjAwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHZpdm8ga2luZXRpY3Mgb2YgdHJhbnNjcmlwdGlvbiBpbml0aWF0aW9uIG9mIHRoZSBsYXIgcHJvbW90ZXIgaW4gRXNjaGVyaWNoaWEgY29saS4gRXZpZGVuY2UgZm9yIGEgc2VxdWVudGlhbCBtZWNoYW5pc20gd2l0aCB0d28gcmF0ZS1saW1pdGluZyBzdGVwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljcyBvZiB0cmFuc2NyaXB0aW9uIGRyaXZlbiBieSB0aGUgdGV0QSBwcm9tb3Rlciwgb25lIGV2ZW50IGF0IGEgdGltZSwgaW4gbGl2ZSBFc2NoZXJpY2hpYSBjb2xpIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz04NDcyLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbVJOQSBNb2xlY3VsZXMgRGVtb25zdHJhdGUgUHJvYmFiaWxpc3RpYyBNb3ZlbWVudCBpbiBMaXZpbmcgTWFtbWFsaWFuIENlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdG9jaGFzdGljIG1STkEgc3ludGhlc2lzIGluIG1hbW1hbGlhbiBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTcwNy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RvY2hhc3RpY2l0eSBpbiBnZW5lIGV4cHJlc3Npb246IGZyb20gdGhlb3JpZXMgdG8gcGhlbm90eXBlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDUxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdG9jaGFzdGljIGtpbmV0aWMgYW5hbHlzaXMgb2YgZGV2ZWxvcG1lbnRhbCBwYXRod2F5IGJpZnVyY2F0aW9uIGluIHBoYWdlIM67LWluZmVjdGVkIEVzY2hlcmljaGlhIGNvbGkgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzMtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0b2NoYXN0aWMgZ2VuZSBleHByZXNzaW9uIGluIGEgc2luZ2xlIGNlbGw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk3OyAgY2l0YXRpb25fcGFnZXM9MTE4My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob2lzZSBpbiBnZW5lIGV4cHJlc3Npb246IG9yaWdpbnMsIGNvbnNlcXVlbmNlcywgYW5kIGNvbnRyb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9MjAxMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIG5vaXNlIGluIHRoZSBleHByZXNzaW9uIG9mIGEgc2luZ2xlIGdlbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NjktNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUl0J3MgYSBub2lzeSBidXNpbmVzcyEgR2VuZXRpYyByZWd1bGF0aW9uIGF0IHRoZSBuYW5vbW9sYXIgc2NhbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz02NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleGNpdGFibGUgZ2VuZSByZWd1bGF0b3J5IGNpcmN1aXQgaW5kdWNlcyB0cmFuc2llbnQgY2VsbHVsYXIgZGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQwOyAgY2l0YXRpb25fcGFnZXM9NTQ1LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdG9jaGFzdGljIHByb3RlaW4gZXhwcmVzc2lvbiBpbiBpbmRpdmlkdWFsIGNlbGxzIGF0IHRoZSBzaW5nbGUgbW9sZWN1bGUgbGV2ZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDA7ICBjaXRhdGlvbl9wYWdlcz0zNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5lcmF0aW9uIG9mIHByb21vdGVyLW1lZGlhdGVkIHRyYW5zY3JpcHRpb25hbCBub2lzZSBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDEwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYWwgcHJvcGVydGllcyBvZiB0cmFuc2NyaXB0aW9uYWwgdGltZSBzZXJpZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz01NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtpbmV0aWMgbW9kZWxzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbmNsdWRpbmcgbm9uLWNvZGluZyBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTUwMDsgIGNpdGF0aW9uX3BhZ2VzPTEtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiB0cmFuc2NyaXB0aW9uYWwgYnVyc3RpbmcgaW4gYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU4OyAgY2l0YXRpb25fcGFnZXM9MzE0LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtbW9sZWN1bGUgZHluYW1pY3Mgb2YgdHJhbnNjcmlwdGlvbiBvZiB0aGUgbGFyIHByb21vdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIG1lYW4gYW5kIG5vaXNlIG9mIHRoZSBpbiB2aXZvIGtpbmV0aWNzIG9mIHRyYW5zY3JpcHRpb24gdW5kZXIgdGhlIGNvbnRyb2wgb2YgdGhlIGxhYy9hcmEtMSBwcm9tb3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01ODY7ICBjaXRhdGlvbl9wYWdlcz0zODcwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhdGUtbGltaXRpbmcgc3RlcHMgaW4gUk5BIGNoYWluIGluaXRpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz01NjM0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBhbmQgY29udHJvbCBvZiB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb24gaW4gcHJva2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzEtMjA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBleHBlcmltZW50YWwgYXBwcm9hY2ggZm9yIHN0dWR5aW5nIHRoZSBhc3NvY2lhdGlvbiBiZXR3ZWVuIFJOQSBwb2x5bWVyYXMgYW5kIHRoZSB0ZXQgcHJvbW90ZXIgb2YgcEJSMzIyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNjk3LTcwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXppbmcgcmF0ZSBsaW1pdGluZyBzdGVwcyBpbiB0cmFuc2NyaXB0aW9uIGZyb20gUk5BIHByb2R1Y3Rpb24gdGltZXMgaW4gbGl2ZSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb24gYXQgYW4gYWN0aXZhdG9yLWRlcGVuZGVudCBwcm9tb3RlciBkZWZpbmVkIGJ5IHNpbmdsZS1tb2xlY3VsZSBvYnNlcnZhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz02NzktODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aWZ5aW5nIEUuIGNvbGkgcHJvdGVvbWUgYW5kIHRyYW5zY3JpcHRvbWUgd2l0aCBzaW5nbGUtbW9sZWN1bGUgc2Vuc2l0aXZpdHkgaW4gc2luZ2xlIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTUzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3Qgb2YgcHJvbW90ZXIgYXJjaGl0ZWN0dXJlIG9uIHRoZSBjZWxsLXRvLWNlbGwgdmFyaWFiaWxpdHkgaW4gZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMTAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGluZyB0aGUgZnVuY3Rpb25hbCBwcm9ncmFtIG9mIEVzY2hlcmljaGlhIGNvbGkgcHJvbW90ZXJzOiB0aGUgY29tYmluZWQgbW9kZSBvZiBhY3Rpb24gb2YgTGFjIHJlcHJlc3NvciBhbmQgQXJhQyBhY3RpdmF0b3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTM4NzMtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wZXJhdG9yIHNlcXVlbmNlIGFsdGVycyBnZW5lIGV4cHJlc3Npb24gaW5kZXBlbmRlbnRseSBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvciBvY2N1cGFuY3kgaW4gYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTUwLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNyZWFzZWQgUk5BIHBvbHltZXJhc2UgYXZhaWxhYmlsaXR5IGRpcmVjdHMgcmVzb3VyY2VzIHRvd2FyZHMgZ3Jvd3RoIGF0IHRoZSBleHBlbnNlIG9mIG1haW50ZW5hbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjIwOS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kdWxhdGlvbiBvZiBDaGVtaWNhbCBDb21wb3NpdGlvbiBhbmQgT3RoZXIgUGFyYW1ldGVycyBvZiB0aGUgQ2VsbCBieSBHcm93dGggUmF0ZTsgY2l0YXRpb25fcHVibGlzaGVyPUFTTSBQcmVzcywgV2FzaGluZ3RvbiwgREM7ICBjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBDb2xpIGFuZCBTYWxtb25lbGxhOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fcGFnZXM9MTU1My02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BhdGlhbCBkaXN0cmlidXRpb24gYW5kIGRpZmZ1c2l2ZSBtb3Rpb24gb2YgUk5BIHBvbHltZXJhc2UgaW4gbGl2ZSBFc2NoZXJpY2hpYSBjb2xpLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz01MTM4LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJpYWwgbnVjbGVvaWQtYXNzb2NpYXRlZCBwcm90ZWlucywgbnVjbGVvaWQgc3RydWN0dXJlIGFuZCBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xODUtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjdGl2aXRpZXMgb2YgY29uc3RpdHV0aXZlIHByb21vdGVycyBpbiBFc2NoZXJpY2hpYSBjb2xpLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTI7ICBjaXRhdGlvbl9wYWdlcz0xOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgcmVsYXRpdmUgZ2VuZSBleHByZXNzaW9uIGRhdGEgdXNpbmcgcmVhbC10aW1lIHF1YW50aXRhdGl2ZSBQQ1IgYW5kIHRoZSAyLc6UzpRDdCBNZXRob2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00MDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9idXN0bmVzcyBvZiB0aGUgZGl2aXNpb24gc3ltbWV0cnkgaW4gRXNjaGVyaWNoaWEgY29saSBhbmQgZnVuY3Rpb25hbCBjb25zZXF1ZW5jZXMgb2Ygc3ltbWV0cnkgYnJlYWtpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXMuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MDY2MDA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbCBzZWdtZW50YXRpb24gYnkgbXVsdGktcmVzb2x1dGlvbiBhbmFseXNpcyBhbmQgbWF4aW11bSBsaWtlbGlob29kIGVzdGltYXRpb24gKE1BTUxFKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsQWdpbmc6IEEgdG9vbCB0byBzdHVkeSBzZWdyZWdhdGlvbiBhbmQgcGFydGl0aW9uaW5nIGluIGRpdmlzaW9uIGluIGNlbGwgbGluZWFnZXMgb2YgRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xNzA4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpb24gb2YgZmx1b3Jlc2NlbmNlLXRhZ2dlZCBSTkEgbnVtYmVycyBmcm9tIHNwb3QgaW50ZW5zaXRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTE0Ni01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBHRlAtdGFnZ2VkIFJOQSBudW1iZXJzIGZyb20gdGVtcG9yYWwgZmx1b3Jlc2NlbmNlIGludGVuc2l0eSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTY5LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LaW5ldGljcyBvZiB0aGUgY2VsbHVsYXIgaW50YWtlIG9mIGEgZ2VuZSBleHByZXNzaW9uIGluZHVjZXIgYXQgaGlnaCBjb25jZW50cmF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9zeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI1NzktODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdGhlcm1vZHluYW1pYyBhbmFseXNpcyBvZiB0aGUgc2VxdWVuY2Utc3BlY2lmaWMgYmluZGluZyBvZiBSTkEgYnkgYmFjdGVyaW9waGFnZSBNUzIgY29hdCBwcm90ZWluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9OTI0NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaWNhbCBOYXR1cmUgb2YgQmFjdGVyaWFsIEN5dG9wbGFzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5cy4gUmV2LiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTk4MTAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGFuYWx5c2lzIG9mIG1STkEgZGVjYXkgYW5kIGFidW5kYW5jZSBpbiBFc2NoZXJpY2hpYSBjb2xpIGF0IHNpbmdsZS1nZW5lIHJlc29sdXRpb24gdXNpbmcgdHdvLWNvbG9yIGZsdW9yZXNjZW50IEROQSBtaWNyb2FycmF5czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTk2OTctNzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gb2YgYmFjdGVyaWFsIHRyYW5zY3JpcHRpb24gaW5pdGlhdGlvbjogUk5BIHBvbHltZXJhc2UgLSBwcm9tb3RlciBiaW5kaW5nLCBpc29tZXJpemF0aW9uIHRvIGluaXRpYXRpb24tY29tcGV0ZW50IG9wZW4gY29tcGxleGVzLCBhbmQgaW5pdGlhdGlvbiBvZiBSTkEgc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQxMjsgIGNpdGF0aW9uX3BhZ2VzPTc1NC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIHBvbHltZXJhc2UtcHJvbW90ZXIgaW50ZXJhY3Rpb25zOiBUaGUgY29taW5ncyBhbmQgZ29pbmdzIG9mIFJOQSBwb2x5bWVyYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTMwMTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZWxlY3Rpdml0eSBvZiB0cmFuc2NyaXB0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NzIxLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb25pdG9yaW5nIGFib3J0aXZlIGluaXRpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0yNS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjaGVyaWNoaWEgY29saSBwcm9tb3RlciBzZXF1ZW5jZXMgcHJlZGljdCBpbiB2aXRybyBSTkEgcG9seW1lcmFzZSBzZWxlY3Rpdml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9Nzg5LTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLW1vbGVjdWxlIGFuYWx5c2lzIG9mIFJOQSBwb2x5bWVyYXNlIHRyYW5zY3JpcHRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gQmlvcGh5cy4gQmlvbW9sLiBTdHJ1Y3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MzQzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtbW9sZWN1bGUgc3R1ZGllcyBvZiB0cmFuc2NyaXB0aW9uOiBGcm9tIG9uZSBSTkEgcG9seW1lcmFzZSBhdCBhIHRpbWUgdG8gdGhlIGdlbmUgZXhwcmVzc2lvbiBwcm9maWxlIG9mIGEgY2VsbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MTI7ICBjaXRhdGlvbl9wYWdlcz04MTQtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdXNpbmcgYnkgYmFjdGVyaWFsIFJOQSBwb2x5bWVyYXNlIGlzIG1lZGlhdGVkIGJ5IG1lY2hhbmlzdGljYWxseSBkaXN0aW5jdCBjbGFzc2VzIG9mIHNpZ25hbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz03MDkwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgdHJhbnNjcmlwdGlvbmFsIHBhdXNpbmcgb24gZ2VuZSBleHByZXNzaW9uIGR5bmFtaWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNzA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIM+DNzAtcmV0YWluaW5nIHRyYW5zY3JpcHRpb24gZWxvbmdhdGlvbiBjb21wbGV4ZXMgZnJvbSBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbiBhbmQgc2lnbWEgY29tcGV0aXRpb24gZnJvbSBhbiBlcXVpbGlicml1bSBtb2RlbCBvZiBSTkEgcG9seW1lcmFzZSBiaW5kaW5nIHRvIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz01MzMyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBldGl0aW9uIGFtb25nIHNldmVuIEVzY2hlcmljaGlhIGNvbGkgz4Mgc3VidW5pdHM6IHJlbGF0aXZlIGJpbmRpbmcgYWZmaW5pdGllcyB0byB0aGUgY29yZSBSTkEgcG9seW1lcmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzQ5Ny01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzIEZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFtLiBTdGF0LiBBc3NvYy47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz03NzMtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb21vdGVyIGNsZWFyYW5jZSBhbmQgZXNjYXBlIGluIHByb2thcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE1LDc3OyAgY2l0YXRpb25fcGFnZXM9MTkxLTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUR1eGJ1cnkgUHJlc3MsIFBhY2lmaWMgR3JvdmUsIENBOyAgY2l0YXRpb25fdGl0bGU9VGhlIERlbHRhIE1ldGhvZC4gU3RhdGlzdGljYWwgSW5mZXJlbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fcGFnZXM9MjQwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd2VpZ2h0ZWQgdG90YWwgbGVhc3Qtc3F1YXJlcyBhbGdvcml0aG0gZm9yIGZpdHRpbmcgYSBzdHJhaWdodCBsaW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZWFzLiBTY2kuIFRlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9Nzk4MDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Hcm93dGgtcmF0ZS1kZXBlbmRlbnQgcGFydGl0aW9uaW5nIG9mIFJOQSBwb2x5bWVyYXNlcyBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0yMDI0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kZXBlbmRlbnQgYW5kIHRpZ2h0IHJlZ3VsYXRpb24gb2YgdHJhbnNjcmlwdGlvbmFsIHVuaXRzIGluIEVzY2hlcmljaGlhIGNvbGkgdmlhIHRoZSBMYWNSL08sIHRoZSBUZXRSL08gYW5kIEFyYUMvSTEtSTIgcmVndWxhdG9yeSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTIwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmYXN0LCByb2J1c3QgYW5kIHR1bmFibGUgc3ludGhldGljIGdlbmUgb3NjaWxsYXRvcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1NjsgIGNpdGF0aW9uX3BhZ2VzPTUxNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljcyBvZiB0cmFuc2NyaXB0aW9uIG9mIGNsb3NlbHkgc3BhY2VkIHByb21vdGVycyBpbiBFc2NoZXJpY2hpYSBjb2xpLCBvbmUgZXZlbnQgYXQgYSB0aW1lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUaGVvci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzAxOyAgY2l0YXRpb25fcGFnZXM9ODMtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHZpdm8gc2luZ2xlLW1vbGVjdWxlIGtpbmV0aWNzIG9mIGFjdGl2YXRpb24gYW5kIHN1YnNlcXVlbnQgYWN0aXZpdHkgb2YgdGhlIGFyYWJpbm9zZSBwcm9tb3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NjU0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBNZzIrIG9uIGluIHZpdm8gdHJhbnNjcmlwdGlvbmFsIGR5bmFtaWNzIG9mIHRoZSBsYXIgcHJvbW90ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb1N5c3RlbXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RzIG9mIG1vbGVjdWxhciBtZW1vcnkgYW5kIGJ1cnN0aW5nIG9uIGZsdWN0dWF0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzE5OyAgY2l0YXRpb25fcGFnZXM9MzM5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RHV4YnVyeSBQcmVzcywgUGFjaWZpYyBHcm92ZSwgQ0EsIFVTQTsgIGNpdGF0aW9uX3RpdGxlPVNpbXVsdGFuZW91cyBFc3RpbWF0aW9uIGFuZCBDb25maWRlbmNlIEJhbmRzLiBTdGF0aXN0aWNhbCBJbmZlcmVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl9wYWdlcz01NTktNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgdGhlIGRpbWVuc2lvbiBvZiBhIG1vZGVsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFN0YXQuOyAgY2l0YXRpb25feWVhcj0xOTc4OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00NjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtIG9mIENSUC1jQU1QIGFjdGl2YXRpb24gb2YgbGFjIG9wZXJvbiB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb24gYWN0aXZhdGlvbiBvZiB0aGUgUDEgcHJvbW90ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9ODgxLTkwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2luZXRpY3Mgb2Ygb3BlbiBjb21wbGV4IGZvcm1hdGlvbiBiZXR3ZWVuIEVzY2hlcmljaGlhIGNvbGkgUk5BIHBvbHltZXJhc2UgYW5kIHRoZSBsYWMgVVY1IHByb21vdGVyLiBFdmlkZW5jZSBmb3IgYSBzZXF1ZW50aWFsIG1lY2hhbmlzbSBpbnZvbHZpbmcgdGhyZWUgc3RlcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI3MTItMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0b2NoYXN0aWMgbW9kZWxzIG9mIHRyYW5zY3JpcHRpb246IEZyb20gc2luZ2xlIG1vbGVjdWxlcyB0byBzaW5nbGUgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xMy0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBzdG9jaGFzdGljaXR5IG9mIGNvbXBsZXggZ2VuZSByZWd1bGF0aW9uIG1vZGVsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvcGh5cy4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTE1Mi02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzMvMjAzLzE3NDYzMDI3L2RzdzAwOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgaW52ZXN0aWdhdGUgdGhlIGh5cG90aGVzaXMgdGhhdCwgaW4gRXNjaGVyaWNoaWEgY29saSwgd2hpbGUgdGhlIGNvbmNlbnRyYXRpb24gb2YgUk5BIHBvbHltZXJhc2VzIGRpZmZlcnMgaW4gZGlmZmVyZW50IGdyb3d0aCBjb25kaXRpb25zLCB0aGUgZnJhY3QiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE3NDUzNjYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk1jYP0OlURMuM3O1MF97iQhnFzqo=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f203%2f1745366" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f203%2f1745366" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198393">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198393">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198395">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198395">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198403">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198403">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198446">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198446">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198486">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198486">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198492">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198494">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198494">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198496">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198496">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/193/1745343">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/215/1745460">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Dissecting the stochastic transcription initiation process in live <em>Escherichia coli</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jason Lloyd-Price</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jason+Lloyd-Price">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lloyd-Price J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lloyd-Price J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jason Lloyd-Price</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sofia Startceva</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sofia+Startceva">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Startceva S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Startceva S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sofia Startceva</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vinodh Kandavalli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vinodh+Kandavalli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kandavalli V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kandavalli V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vinodh Kandavalli</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jerome G. Chandraseelan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jerome+G.+Chandraseelan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chandraseelan J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chandraseelan J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jerome G. Chandraseelan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nadia Goncalves</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nadia+Goncalves">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Goncalves N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Goncalves N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nadia Goncalves</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Samuel M. D. Oliveira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Samuel+M.+D.+Oliveira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oliveira S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oliveira S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Samuel M. D. Oliveira</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antti H&#228;kkinen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antti+H%c3%a4kkinen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=H&#228;kkinen A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;H&#228;kkinen A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antti Häkkinen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andre S. Ribeiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Laboratory of Biosystem Dynamics, Department of Signal Processing</div>, <div class="insititution">Tampere University of Technology</div>, <div class="addr-line">PO Box 553, Office TC336</div>, <div class="addr-line">33101 Tampere</div>, <div class="country">Finland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +358 408490736. Fax. +358 331154989. Email: <a href="mailto:andre.ribeiro@tut.fi" target="_blank">andre.ribeiro@tut.fi</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andre+S.+Ribeiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ribeiro A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ribeiro A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andre S. Ribeiro</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 203–214, <a href='https://doi.org/10.1093/dnares/dsw009'>https://doi.org/10.1093/dnares/dsw009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 March 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">01 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745366" href="/dnaresearch/article-pdf/23/3/203/17463027/dsw009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jason Lloyd-Price, Sofia Startceva, Vinodh Kandavalli, Jerome G. Chandraseelan, Nadia Goncalves, Samuel M. D. Oliveira, Antti Häkkinen, Andre S. Ribeiro;  Dissecting the stochastic transcription initiation process in live <em>Escherichia coli</em>, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 203–214, <a href='https://doi.org/10.1093/dnares/dsw009'>https://doi.org/10.1093/dnares/dsw009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745366?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Dissecting%20the%20stochastic%20transcription%20initiation%20process%20in%20live%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2016-03-28&amp;volumeNum=23&amp;issueNum=3&amp;author=Lloyd-Price%2C%20Jason%3B%20Startceva%2C%20Sofia&amp;startPage=203&amp;endPage=214&amp;contentId=10.1093%2Fdnares%2Fdsw009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f203%2f1745366" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198393 id="87198393" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We investigate the hypothesis that, in <em>Escherichia coli</em>, while the concentration of RNA polymerases differs in different growth conditions, the fraction of RNA polymerases free for transcription remains approximately constant within a certain range of these conditions. After establishing this, we apply a standard model-fitting procedure to fully characterize the <em>in vivo</em> kinetics of the rate-limiting steps in transcription initiation of the P<em><sub>lac/ara-1</sub></em> promoter from distributions of intervals between transcription events in cells with different RNA polymerase concentrations. We find that, under full induction, the closed complex lasts ∼788 s while subsequent steps last ∼193 s, on average. We then establish that the closed complex formation usually occurs multiple times prior to each successful initiation event. Furthermore, the promoter intermittently switches to an inactive state that, on average, lasts ∼87 s. This is shown to arise from the intermittent repression of the promoter by LacI. The methods employed here should be of use to resolve the rate-limiting steps governing the <em>in vivo</em> dynamics of initiation of prokaryotic promoters, similar to established steady-state assays to resolve the <em>in vitro</em> dynamics.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;free RNA polymerase&quot;">free RNA polymerase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;in vivo transcription dynamics&quot;"><em>in vivo</em> transcription dynamics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rate-limiting steps&quot;">rate-limiting steps</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reversible closed complex formation&quot;">reversible closed complex formation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;repressor binding dynamics&quot;">repressor binding dynamics</a></div>                            <h2 scrollto-destination=87198395 id="87198395" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Gene expression has been intensively studied with the relatively new tools provided by fluorescent proteins and microscopy techniques with single-molecule resolution, in both prokaryotic<sup><a reveal-id="DSW009C1  DSW009C5" data-open="DSW009C1  DSW009C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> and eukaryotic<sup><a reveal-id="DSW009C6" data-open="DSW009C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSW009C7" data-open="DSW009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> systems. These studies have established that this process cannot be fully characterized by the mean protein production rate,<sup><a reveal-id="DSW009C8  DSW009C12" data-open="DSW009C8  DSW009C12" class="link link-ref link-reveal xref-bibr">8–12</a></sup> since cells exhibit fluctuations (i.e. noise) over time and diversity in numbers across populations,<sup><a reveal-id="DSW009C13" data-open="DSW009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> which, among other things, generates phenotypic diversity.<sup><a reveal-id="DSW009C8" data-open="DSW009C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The noise has generally been investigated through indirect means, such as by observing the diversity in RNA and protein numbers in cell populations.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C3" data-open="DSW009C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW009C10" data-open="DSW009C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW009C11" data-open="DSW009C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSW009C14" data-open="DSW009C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Other, more direct means consist of observing the distribution of intervals between RNA productions<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and between protein bursts in individual cells.<sup><a reveal-id="DSW009C3" data-open="DSW009C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW009C15" data-open="DSW009C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>From these observations, a wide range of gene expression behaviours have been reported and, therefore, significantly different probabilistic models of transcription have been proposed.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C16  DSW009C18" data-open="DSW009C16  DSW009C18" class="link link-ref link-reveal xref-bibr">16–18</a></sup> In general, higher-than-Poissonian variability in RNA numbers has been explained by models in which the promoter intermittently switched into an inactive state, resulting in bursty RNA production dynamics.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C16" data-open="DSW009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Meanwhile, lower-than-Poissonian variability appears to be more consistent with models assuming multiple rate-limiting steps.<sup><a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C16" data-open="DSW009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW009C20" data-open="DSW009C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSW009C21" data-open="DSW009C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>There is direct experimental evidence for the existence of both mechanisms. Recently, Chong et al.<sup><a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a></sup> showed that bursts of RNA production can emerge due to positive supercoiling build-up on a DNA segment, which eventually stops transcription initiation for a short period until the release of the supercoiling by gyrase. On the other hand, the existence of rate-limiting steps was established by studies using steady-state assays.<sup><a reveal-id="DSW009C22  DSW009C24" data-open="DSW009C22  DSW009C24" class="link link-ref link-reveal xref-bibr">22–24</a></sup> Also, more recently, by fitting a monotone piecewise-constant function to the fluorescence signal from MS2-GFP tagged RNAs in individual cells, it was shown that <em>in vivo</em> RNA production can be a sub-Poissonian process.<sup><a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C20" data-open="DSW009C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSW009C21" data-open="DSW009C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Recent studies have considered the possibility that both mechanisms can be present in a single promoter.<sup><a reveal-id="DSW009C16" data-open="DSW009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In ref. <a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a>, a model including both mechanisms was proposed, and statistical methods were developed to select the relevant components and estimate the kinetics of the intermediate steps in initiation based on empirical data. However, this method cannot distinguish the order of the steps which occur after the start of transcription initiation, nor can it determine their reversibility, which recent evidence suggests may play a significant role in the dynamics of RNA production.<sup><a reveal-id="DSW009C26" data-open="DSW009C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>A complete model for transcription in prokaryotes must account, apart from the genome-wide variability in noise levels,<sup><a reveal-id="DSW009C17" data-open="DSW009C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSW009C27" data-open="DSW009C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSW009C28" data-open="DSW009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> for the well-established genome-wide variability in mean transcription rate<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C3" data-open="DSW009C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW009C8" data-open="DSW009C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and in fold change (ratio of production rate between zero and full induction)<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> in response to induction found, e.g. in <em>Escherichia coli</em> promoters. For example, <em>in vitro</em> measurements on fully induced variants of the <em>lar</em> promoter showed that the mean interval between transcription events of these variants differs by hundreds of seconds.<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Promoters also differ widely in range of induction, even when differing only by a couple of nucleotides.<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSW009C30" data-open="DSW009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> For example, while P<em><sub>larS17</sub></em> has an induction range of 500 fold, P<em><sub>larconS17</sub></em> has an induction range of 4.5-fold, even though it only differs by 3 point mutations.<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> This wide behavioural diversity is likely made possible by the sequence dependence of each step in transcription initiation.<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>Thus far, the strategies used <em>in vitro</em> to characterize the kinetics of the steps involved in transcription initiation<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW009C26" data-open="DSW009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> have not been applied <em>in vivo</em> since they rely on measuring transcription for different RNA polymerase (RNAp) concentrations. Such a change in cells is expected to have a multitude of unforeseen effects<sup><a reveal-id="DSW009C31" data-open="DSW009C31" class="link link-ref link-reveal xref-bibr">31</a></sup> (in addition to the side effects of the means used to alter RNAp concentrations), which hampers the assessment of its consequences to the duration of the closed complex formation of a specific promoter. However, it is reasonable to hypothesize that, for certain small ranges of RNAp concentrations, these side effects will be negligible and thus, in such ranges, the inverse of the rate of transcription will be linear with respect to the inverse of the free RNAp concentration.</p><p>Importantly, in <em>E. coli</em>, RNAp concentrations have been shown to vary widely with differing growth conditions.<sup><a reveal-id="DSW009C32" data-open="DSW009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> As such, here we make use of different media richness to achieve different RNAp concentrations and test whether within this range of conditions, the RNA production rate changes hyperbolically with the RNAp concentrations (i.e. if the inverse of this rate changes linearly with the inverse of the RNAp concentration). Having established this relationship, we make use of it to study the <em>in vivo</em> kinetics of transcription initiation of P<em><sub>lac/ara-1</sub></em>. In particular, we perform measurements of the time intervals between RNA productions at the single molecule level in different intracellular RNAp and inducer concentration conditions, which we use to derive a more detailed model of transcription initiation of P<em><sub>lac/ara-1</sub></em>. For this, we first extrapolate the mean interval between production events to the limit of infinite RNAp concentration, so as to estimate the <em>in vivo</em> durations of the open and closed complex formations of this promoter. Next, we examine the significance of an intermittent inactive promoter state, and the role of LacI in the emergence of this state. Finally, for the first time <em>in vivo</em>, we determine the reversibility of the closed complex formation.</p>                            <h2 scrollto-destination=87198403 id="87198403" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87198404 id="87198404" class="section-title" data-legacy-id=s2a>2.1. Cells and plasmids</h3>
<p>For single-cell RNAp fluorescence measurements, we used <em>E. coli</em> W3110 and RL1314,<sup><a reveal-id="DSW009C33" data-open="DSW009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> generously provided by Robert Landick, University of Wisconsin-Madison. For single-cell transcription interval measurements, we used <em>E. coli</em> DH5α-PRO (generously provided by Ido Golding, Baylor College of Medicine, Houston). The strain information is: deoR, endA1, gyrA96, hsdR17(rK- mK+), recA1, relA1, supE44, thi-1, Δ(lacZYA-argF)U169, Φ80δlacZΔM15, F-, λ-, PN25/tetR, PlacIq/lacI and SpR. This strain contains two constructs: a high-copy reporter plasmid vector PROTET-K133 (carrying MS2d-GFP under the control of P<em><sub>LtetO-1</sub></em>) and a single-copy plasmid vector pIG-BAC carrying the target transcript (mRFP1 followed by 96 MS2-binding sites) under the control of P<em><sub>lac/ara-1</sub></em>.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This promoter is located approximately 2 and 9 kb from the origin of replication (Ori2) and the plasmid size is 11.5 kb.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This system has been used to measure the distribution of time intervals between RNA production events due to its ability to detect individual target RNA molecules consisting of numerous MS2 coat protein binding sites, which are rapidly bound by fluorescently tagged MS2 coat proteins. These can be seen as they are produced under a fluorescence microscope as fluorescent foci.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C20" data-open="DSW009C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSW009C21" data-open="DSW009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Finally, we used the plasmid pAB332 carrying <em>hupA-mCherry</em> to visualize nucleoids (generously provided by Nancy Kleckner, Harvard University, Cambridge, MA, USA). For our measurements, we inserted this plasmid into DH5α-PRO cells so as to detect nucleoids in individual cells during the live cell microscopy sessions. HupA is a major nucleoid associated protein (NAP) that participates in its structural organization.<sup><a reveal-id="DSW009C34" data-open="DSW009C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=87198406 id="87198406" class="section-title" data-legacy-id=s2b>2.2. Chemicals</h3>
<p>The components of Lysogeny Broth (LB) were purchased from LabM (UK), and antibiotics from Sigma-Aldrich (USA). For RT-PCR, cells were ﬁxed with RNAprotect bacteria reagent (Qiagen, USA). Tris and EDTA for lysis buffer were purchased from Sigma-Aldrich and lysozyme from Fermentas (USA). The total RNA extraction was done with RNeasy RNA puriﬁcation kit (Qiagen). DNase I, RNasefree for RNA puriﬁcation, was purchased from Promega (USA). iScript Reverse Transcription Supermix for cDNA synthesis and iQ SYBR Green supermix for RT-PCR were purchased from Biorad (USA). Agarose, isopropyl β-<span class="small-caps">d</span>-1-thiogalactopyranoside (IPTG), arabinose, and anhydrotetracycline (aTc) are from Sigma-Aldrich.</p>                            <h3 scrollto-destination=87198408 id="87198408" class="section-title" data-legacy-id=s2c>2.3. Growth media</h3>
<p>To achieve different RNAp concentrations in cells, we altered their growth conditions as in.<sup><a reveal-id="DSW009C35" data-open="DSW009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> For this, we used modified LB media which differed in the concentrations of some of their components. The media used are denoted as <em>m</em>×, where the composition per 100 ml are: <em>m</em> grams of tryptone, <em>m</em>/2 gram of yeast extract and 1 g of NaCl (pH = 7.0). For example, 0.25× media has 0.25 g of tryptone and 0.125 g of yeast extract per 100 ml.</p>                            <h3 scrollto-destination=87198410 id="87198410" class="section-title" data-legacy-id=s2d>2.4. Relative RNAp quantification</h3>
<p>We measured relative RNAp concentrations in cells using four different methods. First, relative RNAp concentrations in the strains W3110 and DH5α-PRO were measured from the relative <em>rpoC</em> transcript levels obtained using RT-PCR. Cells containing the target plasmid with P<em><sub>lac/ara-1</sub></em>-mRFP1-96BS and the reporter plasmids were grown overnight in respective media. Cells were diluted into fresh media to an OD<sub>600</sub> of 0.05. After 110 min, cells were re-diluted to an OD<sub>600</sub> of 0.05 into respective media containing IPTG (1 mM) and arabinose (1%). After 70 min, RNA protect reagent was added to fix the cells, followed by enzymatic lysis with Tris–EDTA lysozyme buffer (pH 8.3). RNA was isolated from cells using RNeasy mini-kit (Qiagen). One microgram of RNA was used as the starting material. The RNA samples were treated with DNase free of RNase to remove residual DNA. Next, RNA was reverse transcribed into cDNA using iSCRIPT reverse transcription super mix (Biorad). RT-PCR was performed using Power SYBR-green master mix (Life Technologies) with primers for the amplification of the target gene at a concentration of 200 nM. Reactions were carried out in triplicate with 500 nM per primer with a total reaction volume 20 µl. The following primers were used for quantification: RpoC-F: CGTCAGATGCTGCGTAAAGC, RpoC-R: GCGATCTTGACGCGAGAGTA, mRFP1-F: TACGACGCCGAGGTCAAG, mRFP1-R: TTGTGGGAGGTGATGTCCA. Estimated relative RNAp concentrations <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></math></span> in each condition <em>m</em>, and their standard uncertainties <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msub><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo></math>⁠</span>, were calculated according to the <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><mrow xmlns=""><msubsup><mrow><mi>C</mi></mrow><mi>T</mi><mrow><mi mathvariant="normal">′</mi></mrow></msubsup></mrow></math></span> method.<sup><a reveal-id="DSW009C36" data-open="DSW009C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>Second, <em>E. coli</em> RL1314 cells with fluorescently tagged β′ subunits were grown overnight in respective media. A pre-culture was prepared by diluting cells to an OD<sub>600</sub> of 0.1 with fresh specific medium, and grown to an OD<sub>600</sub> of 0.5 at 37°C at 250 rpm. Cells were pelleted by centrifugation and re-suspended in saline. Fluorescence from the cell population was measured using a fluorescent plate-reader (Thermo Scientiﬁc Fluoroskan Ascent Microplate Fluorometer).</p><p>Third, relative RNAp concentrations were also estimated based on the growth rates of DH5α-PRO cells in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>. First, we fit a power law function to the ‘RNA polymerase molecules per cell’ row of Table <a reveal-id="DSW009TB3" data-open="DSW009TB3" class="link link-reveal link-table xref-fig">3</a> from ref. <a reveal-id="DSW009C32" data-open="DSW009C32" class="link link-ref link-reveal xref-bibr">32</a>, which we found to be <em>R</em>=10<sup>6</sup><em>µ</em><sup>−1.426</sup>, where <em>µ</em> is the cell doubling time. Relative RNAp concentrations were then estimated from the measured cell doubling times.</p><p>Lastly, we measured the relative RNAp concentrations in RL1314 cells under the microscope using fluorescently tagged RpoC (described in the next section).</p>                            <h3 scrollto-destination=87198415 id="87198415" class="section-title" data-legacy-id=s2e>2.5. Microscopy</h3>
<p>DH5α-PRO cells containing the target and the reporter plasmids were grown as described previously. Briefly, cells were grown overnight in respective media, diluted into fresh media to an OD<sub>600</sub> of 0.1, and allowed to grow to an OD<sub>600</sub> of ∼0.3. For the reporter plasmid induction, aTc (100 ng/ml) was added 1 h before the start of the measurements. For the target plasmid, arabinose (1%) was added at the same time as aTc (following the protocol in ref. <a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>), and IPTG (1 mM) was added 10 min before the start of the measurements. Cells were pelleted and resuspended to fresh medium. A few microliters of cells were placed between a coverslip and an agarose gel pad (2%), which contains the respective inducers, in a thermal imaging chamber (FCS2, Bioptechs), heated to 37°C. The cells were visualized using a Nikon Eclipse (Ti-E, Nikon, Japan) inverted microscope with a C2+ confocal laser-scanning system using a 100× Apo TIRF objective. Images were acquired using the Nikon Nis-Elements software. GFP ﬂuorescence was measured using a 488 nm argon ion laser (Melles-Griot) and 514/30 nm emission ﬁlter. Phase-contrast images were acquired with the external phase contrast system and a Nikon DS-Fi2 camera. Fluorescence images were acquired every 1 min for a total duration of 2 h. Phase-contrast images were acquired simultaneously every 5 min during the measurements.</p><p>We tested for phototoxicity due to the fluorescence and the phase-contrast imaging in these measurements. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Results</a></span> suggest that there is no significant phototoxicity. Additionally, we verified that the relative RNAp concentrations under the microscope are similar to those measured in the previous section by repeating the above procedure with RL1314 cells and imaging RpoC::GFP fluorescence, 1 h after being placed in the thermal imaging chamber (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). The relative RNAp concentration was estimated from the mean fluorescence concentrations of cells growing in each media.</p>                            <h3 scrollto-destination=87198418 id="87198418" class="section-title" data-legacy-id=s2f>2.6. Image analysis</h3>
<p>Cells were detected from the phase contrast images as described in ref. <a reveal-id="DSW009C37" data-open="DSW009C37" class="link link-ref link-reveal xref-bibr">37</a>. First, the images were temporally aligned using cross-correlation. Next, an automatic segmentation of the cells was performed by MAMLE,<sup><a reveal-id="DSW009C38" data-open="DSW009C38" class="link link-ref link-reveal xref-bibr">38</a></sup> which was checked and corrected manually. Next, cell lineages were constructed by CellAging.<sup><a reveal-id="DSW009C39" data-open="DSW009C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Alignment of the phase-contrast images with the confocal images was done by manually selecting 5–7 landmarks in both images, and using thin-plate spline interpolation for the registration transform. Fluorescent spots and their intensities were detected from the confocal images using the Gaussian surface-fitting algorithm from.<sup><a reveal-id="DSW009C40" data-open="DSW009C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>Jumps were detected in each cell's spot intensity timeseries using a least-deviation jump-detection method.<sup><a reveal-id="DSW009C41" data-open="DSW009C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Given the level of noise in the timeseries, jump sizes, i.e. the intensity of ‘one RNA’, were selected by manual inspection of the timeseries of total foreground spot intensities within cells of a given timeseries, and cross-referencing these values with the observed numbers of spots in the cells. After performing the jump detection process making use of the complete timeseries, jumps occurring within 5 min of the beginning or end of a cell's lifetime were disregarded due to our observation that the jump detection method tends to produce spurious jumps in these regions due to insufficient data. The remaining jumps were interpreted as RNA production times, from which intervals between transcription events were calculated. Finally, censored intervals were calculated as the time from the last RNA production in a cell until the last time at which a jump could have been observed (i.e. until 5 min prior to cell division or the end of the timeseries). This removes the possibility of false positives while not affecting the distribution of intervals.</p><p>This method, when first proposed, made two assumptions on the fluorescence of MS2-GFP tagged RNAs (named ‘spots’). Importantly, both assumptions were recently shown to be valid.<sup><a reveal-id="DSW009C42" data-open="DSW009C42" class="link link-ref link-reveal xref-bibr">42</a></sup> First, an individual spot is bound sufficiently rapidly by MS2-GFPs such that its fluorescence intensity, when first detected, is already within the range of fluorescence of fully formed MS2-GFP-RNA spots (when taking one image per minute). In other words, the spot intensity of a newly transcribed RNA jumps from 0 to ‘full’ in &lt;1 min, rather than slowly ramping up. Namely, since the transcription elongation rate of mRNA in <em>E. coli</em> is ∼50 nt/s<sup><a reveal-id="DSW009C32" data-open="DSW009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and the target gene is ∼3,200 bp long,<sup><a reveal-id="DSW009C1" data-open="DSW009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> the time to elongate the MS2-binding site region of the target RNA is ∼60 s. Provided that MS2-GFP binding to its RNA-binding sites is fast, there will therefore be a maximum of one timepoint at which the fully transcribed target RNA may have reduced fluorescence. Since MS2-GFP is produced in excess in the cell and its binding affinity is strong (dissociation constant of ∼0.04 nM<sup><a reveal-id="DSW009C43" data-open="DSW009C43" class="link link-ref link-reveal xref-bibr">43</a></sup>), most binding sites will be saturated very shortly after being produced. In agreement with ref. <a reveal-id="DSW009C42" data-open="DSW009C42" class="link link-ref link-reveal xref-bibr">42</a>, no gradual increase in spot fluorescence was observed around the time of the first appearance of a spot.</p><p>Second, once formed, MS2-GFP-RNA spots, as well as their fluorescence, are resistant to degradation for the duration of our measurements (2 h). This was shown by measurements of the dissociation rate of MS2 coat proteins from their RNA binding sites (on the order of several hours<sup><a reveal-id="DSW009C43" data-open="DSW009C43" class="link link-ref link-reveal xref-bibr">43</a></sup>), and by measurements of the lifetimes of the fluorescence of MS2-GFP tagged RNAs kept under observation for more than 2 h.<sup><a reveal-id="DSW009C1" data-open="DSW009C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C42" data-open="DSW009C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSW009C44" data-open="DSW009C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Relevantly, no detectable decrease in fluorescence was observed during this time.<sup><a reveal-id="DSW009C42" data-open="DSW009C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=87198423 id="87198423" class="section-title" data-legacy-id=s2g>2.7. Model of transcription initiation</h3>
<p>We first consider a model that allows for RNA production dynamics to range from sub-Poissonian to super-Poissonian, given the results from genome-wide studies of the variability in RNA numbers<sup><a reveal-id="DSW009C27" data-open="DSW009C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSW009C45" data-open="DSW009C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and from studies of the transcription dynamics of individual genes.<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C17" data-open="DSW009C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSW009C20" data-open="DSW009C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The features of the model that allow it to reproduce these numbers are based on processes known to occur during transcription initiation in <em>E. coli</em> (e.g. the open complex formation<sup><a reveal-id="DSW009C16" data-open="DSW009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW009C23" data-open="DSW009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and an ON/OFF mechanism<sup><a reveal-id="DSW009C16" data-open="DSW009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a></sup>). Then, based on our novel empirical data and methodology, we aim to obtain the most parsimonious version of the model that fits the data for a given promoter. We expect this procedure to be applicable to any promoter, and to result in slightly different models due to their differing dynamics and regulatory mechanisms.</p><div class="block-child-p">The full model of transcription initiation considered here consists of the following set of reactions: <a id="DSW009M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW009M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">R</mtext><mo xmlns="">+</mo><mrow xmlns=""><msub><mtext>P</mtext><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mrow xmlns=""><munderover><mrow><mrow><mrow><mover><mo stretchy="false">⟵</mo><mo stretchy="false">⟶</mo></mover></mrow></mrow></mrow><mrow><mrow><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow></mrow></munderover></mrow><mtext xmlns="">R</mtext><mrow xmlns=""><msub><mtext>P</mtext><mrow><mi mathvariant="normal">c</mi></mrow></msub></mrow><mrow xmlns=""><mover><mo stretchy="false">⟶</mo><mrow><mrow><msub><mi>k</mi><mrow><mn>2</mn></mrow></msub></mrow></mrow></mover></mrow><mtext xmlns="">R</mtext><mrow xmlns=""><msub><mtext>P</mtext><mrow><mi mathvariant="normal">o</mi></mrow></msub></mrow><mrow xmlns=""><mover><mo stretchy="false">⟶</mo><mrow><mrow><msub><mi>k</mi><mrow><mn>3</mn></mrow></msub></mrow></mrow></mover></mrow><mtext xmlns="">R</mtext><mo xmlns="">+</mo><mrow xmlns=""><msub><mtext>P</mtext><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">+</mo><mtext xmlns="">RNA</mtext></math></div></div><div class="label">(1)</div></div><a id="DSW009M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW009M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mtext>P</mtext><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mrow xmlns=""><munderover><mrow><mrow><mrow><mover><mo stretchy="false">⟵</mo><mo stretchy="false">⟶</mo></mover></mrow></mrow></mrow><mrow><mrow><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></mrow></munderover></mrow><mrow xmlns=""><msub><mtext>P</mtext><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math></div></div><div class="label">(2)</div></div></div><p>Reaction (1) represents the multi-step process of transcription initiation of an active promoter in prokaryotes.<sup><a reveal-id="DSW009C23" data-open="DSW009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSW009C24" data-open="DSW009C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSW009C46" data-open="DSW009C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSW009C47" data-open="DSW009C47" class="link link-ref link-reveal xref-bibr">47</a></sup> It begins with the formation of the closed complex (RP<sub>c</sub>), i.e. the binding of the RNA polymerase (R) to a free promoter (P<sub>ON</sub>). Once at the start site, the polymerase must open the DNA double helix, a process that includes several long-lived intermediate states,<sup><a reveal-id="DSW009C23" data-open="DSW009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSW009C26" data-open="DSW009C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSW009C46" data-open="DSW009C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSW009C48" data-open="DSW009C48" class="link link-ref link-reveal xref-bibr">48</a></sup> resulting in the open complex (RP<sub>o</sub>). Finally, the polymerase begins RNA elongation, though before clearing the promoter, it may engage in abortive RNA synthesis in which short RNA transcripts (&lt;10 nt) are produced.<sup><a reveal-id="DSW009C47" data-open="DSW009C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSW009C49" data-open="DSW009C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The reactions in (1) should not be interpreted as elementary transitions. Rather, they represent the effective rates of the rate-limiting steps in the process, thus defining the promoter strength, and have been shown to be sequence-dependent.<sup><a reveal-id="DSW009C50" data-open="DSW009C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p><p>Specifically, <em>k</em><sub>1</sub> represents the rate at which polymerases find and bind to the promoter region, which is the overall result of the promoter search process which includes non-specific binding of the polymerases to the DNA, followed by a 1D diffusive search,<sup><a reveal-id="DSW009C51" data-open="DSW009C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSW009C52" data-open="DSW009C52" class="link link-ref link-reveal xref-bibr">52</a></sup> collectively referred to here as the closed complex formation. Subsequently, several rapid, possibly reversible isomerization reactions occur until the polymerase melts the DNA and forms the transcription ‘bubble’.<sup><a reveal-id="DSW009C51" data-open="DSW009C51" class="link link-ref link-reveal xref-bibr">51</a></sup> In Reaction (1), the RP<sub>c</sub> state represents all substates until the first irreversible reaction in this chain. Consequently, <em>k</em><sub>2</sub> and <em>k</em><sub>−1</sub> should be interpreted as the product of the rates of the elementary reactions which exit from this group of substates, and the steady-state probability of being in the appropriate substates for these reactions to occur.</p><p>Similarly, the RP<sub>o</sub> state may represent numerous substates between the first state after which the complex is committed to initiation, and successful initiation. However, after this point, we cannot distinguish the reversibility of any of the following steps, since the time-interval distribution of a sequence of elementary reversible reactions of arbitrary rates is observationally equivalent to a sequence of irreversible reactions.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The remaining steps (here, only <em>k</em><sub>3</sub>) therefore represent the rates of the slowest of these irreversible reactions. Such steps may include additional isomerization reactions, abortive RNA synthesis and promoter escape and clearance.<sup><a reveal-id="DSW009C35" data-open="DSW009C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>Reaction (2) represents the promoter intermittently transitioning to a transcriptionally inactive state (<em>P</em><sub>OFF</sub>). Experimentally verified mechanisms by which this can occur are the binding and unbinding of repressors and activators,<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> the accumulation of positive supercoiling in the DNA.<sup><a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Additional mechanisms have also been hypothesized, such as transcriptional pausing<sup><a reveal-id="DSW009C53" data-open="DSW009C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSW009C54" data-open="DSW009C54" class="link link-ref link-reveal xref-bibr">54</a></sup> and others.<sup><a reveal-id="DSW009C55" data-open="DSW009C55" class="link link-ref link-reveal xref-bibr">55</a></sup></p><div class="block-child-p">For a given concentration of R, the interval distribution between transcription events described by Reactions (1) and (2) (i.e. the first-passage time distribution to reach the final state, starting in the <em>P</em><sub>ON</sub> state) is observationally equivalent to the interval distribution described by a model of the form: <a id="DSW009M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW009M3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>S</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow><mrow xmlns=""><munderover><mrow><mrow><mrow><mover><mo stretchy="false">⟵</mo><mo stretchy="false">⟶</mo></mover></mrow></mrow></mrow><mrow><mrow><msub><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></mrow><mrow><mrow><msub><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow></mrow></munderover></mrow><mrow xmlns=""><msub><mi>S</mi><mrow><mn>0</mn></mrow></msub></mrow><mrow xmlns=""><mover><mo stretchy="false">⟶</mo><mrow><mrow><msub><mi>λ</mi><mn>1</mn></msub></mrow></mrow></mover></mrow><mrow xmlns=""><msub><mi>S</mi><mn>1</mn></msub></mrow><mrow xmlns=""><mover><mo stretchy="false">⟶</mo><mrow><mrow><msub><mi>λ</mi><mn>2</mn></msub></mrow></mrow></mover></mrow><mrow xmlns=""><msub><mi>S</mi><mn>2</mn></msub></mrow><mrow xmlns=""><mover><mo stretchy="false">⟶</mo><mrow><mrow><msub><mi>λ</mi><mn>3</mn></msub></mrow></mrow></mover></mrow><mrow xmlns=""><msub><mi>S</mi><mn>0</mn></msub></mrow><mo xmlns="">+</mo><mtext xmlns="">RNA,</mtext></math></div></div><div class="label">(3)</div></div> where the system starts in state <em>S</em><sub>0</sub>. The relationship between the parameters of these two models is described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Note that the states <em>S<sub>i</sub></em> do not correspond to the promoter states in Reactions (1) and (2). For details on how to derive and evaluate the distribution function for this model, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> and.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup></div><p>It is noted that this model assumes that only one copy of the promoter is present in each cell at any given time. In the experiments performed here, in all conditions tested, the bacteria divided sufficiently slowly such that they spent most lifetime with only one chromosome. Specifically, cells spent no more than 11.4 ± 1.0% of their lifetime with two copies of the target promoter (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>).</p><p>Finally, it is noted that the present model does not consider the influence of σ factors' numbers on the dynamics of transcription initiation, focussing instead solely on the concentration of RNA polymerases (in particular, on the concentration of holoenzymes containing a σ<sup>70</sup>, i.e. Eσ<sup>70</sup>, since our promoter of interest can only be transcribed by Eσ<sup>70</sup>). This is based on the fact that, in all conditions tested, most RNA polymerases are occupied by σ factors.<sup><a reveal-id="DSW009C56" data-open="DSW009C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSW009C57" data-open="DSW009C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Further, this occupation is made largely by σ<sup>70</sup> since, first, when altering media richness, only σ<sup>32</sup>'s concentration is significantly altered<sup><a reveal-id="DSW009C56" data-open="DSW009C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and, second, the binding affinity of σ<sup>70</sup> to E is much higher than that of any other σ factor (e.g. it is approximately 9 times higher than that of σ<sup>32</sup>).<sup><a reveal-id="DSW009C57" data-open="DSW009C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p>                            <h3 scrollto-destination=87198433 id="87198433" class="section-title" data-legacy-id=s2h>2.8. Parameter estimation</h3>
<p>Parameter estimates in Tables <a reveal-id="DSW009TB1  DSW009TB3" data-open="DSW009TB1  DSW009TB3" class="link link-reveal link-table xref-fig">1–3</a> were obtained by a maximum likelihood fit using the samples of the distribution of time intervals between production events obtained above (the intervals and censored intervals), as in.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The complete model-fitting procedure is detailed in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>. The uncertainty of the fit of the model parameters was estimated using the negative of the Hessian of the log-likelihood surface, evaluated at the maximum likelihood estimate. </p>                        <a id="87198435" scrollto-destination="87198435"></a>
<div content-id="DSW009TB1" class="table-modal"><div class="table-wrap"><div id="DSW009TB1" class="table-wrap-title" data-id="DSW009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the measured distributions of intervals between transcription events from <em>lac/ara-1</em> promoters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Condition </th><th>Number of cells </th><th>Number of intervals </th><th>Number of censored intervals </th><th>Inferred interval mean and uncertainty (s) </th><th>Inferred CV<sup>2</sup> </th></tr></thead><tbody><tr><td>0.25× </td><td>196 </td><td>371 </td><td>323 </td><td>1,899 ± 105 </td><td>1.08 </td></tr><tr><td>0.5× </td><td>302 </td><td>1,027 </td><td>605 </td><td>1,553 ± 50 </td><td>1.06 </td></tr><tr><td>0.75× </td><td>146 </td><td>620 </td><td>345 </td><td>1,205 ± 51 </td><td>1.09 </td></tr><tr><td>1× </td><td>206 </td><td>1,202 </td><td>573 </td><td>1,005 ± 112 </td><td>1.21 </td></tr></tbody></table></div><div content-id="DSW009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Condition </th><th>Number of cells </th><th>Number of intervals </th><th>Number of censored intervals </th><th>Inferred interval mean and uncertainty (s) </th><th>Inferred CV<sup>2</sup> </th></tr></thead><tbody><tr><td>0.25× </td><td>196 </td><td>371 </td><td>323 </td><td>1,899 ± 105 </td><td>1.08 </td></tr><tr><td>0.5× </td><td>302 </td><td>1,027 </td><td>605 </td><td>1,553 ± 50 </td><td>1.06 </td></tr><tr><td>0.75× </td><td>146 </td><td>620 </td><td>345 </td><td>1,205 ± 51 </td><td>1.09 </td></tr><tr><td>1× </td><td>206 </td><td>1,202 </td><td>573 </td><td>1,005 ± 112 </td><td>1.21 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Shown are the condition, the number of cells (which is the cell count at the start of the measurements), the numbers of whole and censored intervals extracted, and finally the inferred mean (and its standard uncertainty) and CV<sup>2</sup> of the interval distribution.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198435" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW009TB1" class="table-wrap-title" data-id="DSW009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the measured distributions of intervals between transcription events from <em>lac/ara-1</em> promoters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Condition </th><th>Number of cells </th><th>Number of intervals </th><th>Number of censored intervals </th><th>Inferred interval mean and uncertainty (s) </th><th>Inferred CV<sup>2</sup> </th></tr></thead><tbody><tr><td>0.25× </td><td>196 </td><td>371 </td><td>323 </td><td>1,899 ± 105 </td><td>1.08 </td></tr><tr><td>0.5× </td><td>302 </td><td>1,027 </td><td>605 </td><td>1,553 ± 50 </td><td>1.06 </td></tr><tr><td>0.75× </td><td>146 </td><td>620 </td><td>345 </td><td>1,205 ± 51 </td><td>1.09 </td></tr><tr><td>1× </td><td>206 </td><td>1,202 </td><td>573 </td><td>1,005 ± 112 </td><td>1.21 </td></tr></tbody></table></div><div content-id="DSW009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Condition </th><th>Number of cells </th><th>Number of intervals </th><th>Number of censored intervals </th><th>Inferred interval mean and uncertainty (s) </th><th>Inferred CV<sup>2</sup> </th></tr></thead><tbody><tr><td>0.25× </td><td>196 </td><td>371 </td><td>323 </td><td>1,899 ± 105 </td><td>1.08 </td></tr><tr><td>0.5× </td><td>302 </td><td>1,027 </td><td>605 </td><td>1,553 ± 50 </td><td>1.06 </td></tr><tr><td>0.75× </td><td>146 </td><td>620 </td><td>345 </td><td>1,205 ± 51 </td><td>1.09 </td></tr><tr><td>1× </td><td>206 </td><td>1,202 </td><td>573 </td><td>1,005 ± 112 </td><td>1.21 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Shown are the condition, the number of cells (which is the cell count at the start of the measurements), the numbers of whole and censored intervals extracted, and finally the inferred mean (and its standard uncertainty) and CV<sup>2</sup> of the interval distribution.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198435" target="_blank">View Large</a></div></div>                        <a id="87198436" scrollto-destination="87198436"></a>
<div content-id="DSW009TB2" class="table-modal"><div class="table-wrap"><div id="DSW009TB2" class="table-wrap-title" data-id="DSW009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Fit parameters of the transcription initiation model in Reactions (1) and (2), and the models derived by applying the listed simplifying assumptions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Limiting mechanisms </th><th>Simplifications </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (Rs) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>3</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th><th>ΔBIC<sub>C</sub> </th></tr></thead><tbody><tr><td>Full model </td><td> </td><td>87 </td><td>Fast </td><td>8,313 </td><td>Fast </td><td>2,247 </td><td>Fast </td><td>177 </td><td>Fast </td><td>14.8 </td><td>15.7 </td></tr><tr><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math></span> </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>7,446 </td><td> </td><td>192 </td><td>Fast </td><td>8.1 </td><td>8.5 </td></tr><tr><td>I, <em>k</em><sub>3</sub> = ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>6,469 </td><td> </td><td>192 </td><td> </td><td>0.0 </td><td>0.0 </td></tr><tr><td>II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td>90 </td><td>Fast </td><td>0.10 </td><td>Fast </td><td> </td><td> </td><td>7 </td><td>7 </td><td>18.3 </td><td>18.8 </td></tr><tr><td>II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>86 </td><td>Fast </td><td>0.09 </td><td>Fast </td><td> </td><td> </td><td>10 </td><td> </td><td>10.7 </td><td>10.7 </td></tr><tr><td>No ON/OFF </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>Fast </td><td>0.49 </td><td>Fast </td><td>326 </td><td>Fast </td><td>188.1 </td><td>188.1 </td></tr><tr><td>No ON/OFF, I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td>Fast </td><td>180.1 </td><td>179.6 </td></tr><tr><td>No ON/OFF, I, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td> </td><td>172.0 </td><td>171.1 </td></tr><tr><td>No ON/OFF, II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td>Fast </td><td>201.0 </td><td>200.6 </td></tr><tr><td>No ON/OFF, II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td> </td><td>192.9 </td><td>192.0 </td></tr></tbody></table></div><div content-id="DSW009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Limiting mechanisms </th><th>Simplifications </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (Rs) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>3</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th><th>ΔBIC<sub>C</sub> </th></tr></thead><tbody><tr><td>Full model </td><td> </td><td>87 </td><td>Fast </td><td>8,313 </td><td>Fast </td><td>2,247 </td><td>Fast </td><td>177 </td><td>Fast </td><td>14.8 </td><td>15.7 </td></tr><tr><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math></span> </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>7,446 </td><td> </td><td>192 </td><td>Fast </td><td>8.1 </td><td>8.5 </td></tr><tr><td>I, <em>k</em><sub>3</sub> = ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>6,469 </td><td> </td><td>192 </td><td> </td><td>0.0 </td><td>0.0 </td></tr><tr><td>II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td>90 </td><td>Fast </td><td>0.10 </td><td>Fast </td><td> </td><td> </td><td>7 </td><td>7 </td><td>18.3 </td><td>18.8 </td></tr><tr><td>II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>86 </td><td>Fast </td><td>0.09 </td><td>Fast </td><td> </td><td> </td><td>10 </td><td> </td><td>10.7 </td><td>10.7 </td></tr><tr><td>No ON/OFF </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>Fast </td><td>0.49 </td><td>Fast </td><td>326 </td><td>Fast </td><td>188.1 </td><td>188.1 </td></tr><tr><td>No ON/OFF, I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td>Fast </td><td>180.1 </td><td>179.6 </td></tr><tr><td>No ON/OFF, I, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td> </td><td>172.0 </td><td>171.1 </td></tr><tr><td>No ON/OFF, II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td>Fast </td><td>201.0 </td><td>200.6 </td></tr><tr><td>No ON/OFF, II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td> </td><td>192.9 </td><td>192.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Parameters denoted ‘fast’ are too fast to present on the timescale of seconds. When competing fast reactions occur, relevant ratios are given. ΔBIC values are given as the difference of the model's BIC from the BIC of the best-fitting model (the one with ΔBIC = 0). Models with lower ΔBIC are favoured over models with higher ΔBIC.<sup><a reveal-id="DSW009C58" data-open="DSW009C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Censored intervals were included in ΔBIC<sub>C</sub>, but not in ΔBIC. The best-fitting model is shaded. Rates (and ratios) involving <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> are given relative to the intracellular RNAp concentration in the 1× media.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>a</sup><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">=</mo><mn xmlns="">0.11</mn></math>⁠</span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198436" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW009TB2" class="table-wrap-title" data-id="DSW009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Fit parameters of the transcription initiation model in Reactions (1) and (2), and the models derived by applying the listed simplifying assumptions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Limiting mechanisms </th><th>Simplifications </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (Rs) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>3</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th><th>ΔBIC<sub>C</sub> </th></tr></thead><tbody><tr><td>Full model </td><td> </td><td>87 </td><td>Fast </td><td>8,313 </td><td>Fast </td><td>2,247 </td><td>Fast </td><td>177 </td><td>Fast </td><td>14.8 </td><td>15.7 </td></tr><tr><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math></span> </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>7,446 </td><td> </td><td>192 </td><td>Fast </td><td>8.1 </td><td>8.5 </td></tr><tr><td>I, <em>k</em><sub>3</sub> = ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>6,469 </td><td> </td><td>192 </td><td> </td><td>0.0 </td><td>0.0 </td></tr><tr><td>II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td>90 </td><td>Fast </td><td>0.10 </td><td>Fast </td><td> </td><td> </td><td>7 </td><td>7 </td><td>18.3 </td><td>18.8 </td></tr><tr><td>II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>86 </td><td>Fast </td><td>0.09 </td><td>Fast </td><td> </td><td> </td><td>10 </td><td> </td><td>10.7 </td><td>10.7 </td></tr><tr><td>No ON/OFF </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>Fast </td><td>0.49 </td><td>Fast </td><td>326 </td><td>Fast </td><td>188.1 </td><td>188.1 </td></tr><tr><td>No ON/OFF, I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td>Fast </td><td>180.1 </td><td>179.6 </td></tr><tr><td>No ON/OFF, I, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td> </td><td>172.0 </td><td>171.1 </td></tr><tr><td>No ON/OFF, II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td>Fast </td><td>201.0 </td><td>200.6 </td></tr><tr><td>No ON/OFF, II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td> </td><td>192.9 </td><td>192.0 </td></tr></tbody></table></div><div content-id="DSW009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Limiting mechanisms </th><th>Simplifications </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (Rs) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (R<sup>−1</sup>) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>3</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th><th>ΔBIC<sub>C</sub> </th></tr></thead><tbody><tr><td>Full model </td><td> </td><td>87 </td><td>Fast </td><td>8,313 </td><td>Fast </td><td>2,247 </td><td>Fast </td><td>177 </td><td>Fast </td><td>14.8 </td><td>15.7 </td></tr><tr><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math></span> </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>7,446 </td><td> </td><td>192 </td><td>Fast </td><td>8.1 </td><td>8.5 </td></tr><tr><td>I, <em>k</em><sub>3</sub> = ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>87 </td><td>Fast<sup>a</sup> </td><td> </td><td> </td><td>6,469 </td><td> </td><td>192 </td><td> </td><td>0.0 </td><td>0.0 </td></tr><tr><td>II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td>90 </td><td>Fast </td><td>0.10 </td><td>Fast </td><td> </td><td> </td><td>7 </td><td>7 </td><td>18.3 </td><td>18.8 </td></tr><tr><td>II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td>86 </td><td>Fast </td><td>0.09 </td><td>Fast </td><td> </td><td> </td><td>10 </td><td> </td><td>10.7 </td><td>10.7 </td></tr><tr><td>No ON/OFF </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>Fast </td><td>0.49 </td><td>Fast </td><td>326 </td><td>Fast </td><td>188.1 </td><td>188.1 </td></tr><tr><td>No ON/OFF, I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td>Fast </td><td>180.1 </td><td>179.6 </td></tr><tr><td>No ON/OFF, I, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td> </td><td>0.50 </td><td> </td><td>328 </td><td> </td><td>172.0 </td><td>171.1 </td></tr><tr><td>No ON/OFF, II </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td>Fast </td><td>201.0 </td><td>200.6 </td></tr><tr><td>No ON/OFF, II, <em>k</em><sub>3</sub>= ∞ </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><mo xmlns="">≫</mo><mrow xmlns=""><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow></math>⁠</span>, <em>k</em><sub>3</sub>= ∞ </td><td> </td><td> </td><td> </td><td>910 </td><td> </td><td> </td><td>Fast </td><td> </td><td>192.9 </td><td>192.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Parameters denoted ‘fast’ are too fast to present on the timescale of seconds. When competing fast reactions occur, relevant ratios are given. ΔBIC values are given as the difference of the model's BIC from the BIC of the best-fitting model (the one with ΔBIC = 0). Models with lower ΔBIC are favoured over models with higher ΔBIC.<sup><a reveal-id="DSW009C58" data-open="DSW009C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Censored intervals were included in ΔBIC<sub>C</sub>, but not in ΔBIC. The best-fitting model is shaded. Rates (and ratios) involving <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> are given relative to the intracellular RNAp concentration in the 1× media.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>a</sup><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mn>1</mn></msub></mrow><mrow xmlns=""><msub><mi>k</mi><mn>2</mn></msub></mrow><msubsup xmlns=""><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">=</mo><mn xmlns="">0.11</mn></math>⁠</span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198436" target="_blank">View Large</a></div></div>                        <a id="87198437" scrollto-destination="87198437"></a>
<div content-id="DSW009TB3" class="table-modal"><div class="table-wrap"><div id="DSW009TB3" class="table-wrap-title" data-id="DSW009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Fit parameters of the transcription initiation model in Reaction (3) to the measured intervals in the 1× media with and without induction by IPTG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Joint parameters </th><th>Condition </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th></tr></thead><tbody><tr><td rowspan="2">Independent </td><td>IPTG+ </td><td>110 </td><td>Fast </td><td>0.11 </td><td>Fast </td><td>5 </td><td rowspan="2">14.3 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td><td>Fast </td><td>Fast </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">4,444 </td><td>Fast </td><td>11.50 </td><td>Fast </td><td>964 </td><td rowspan="2">120.3 </td></tr><tr><td>IPTG− </td><td>Fast </td><td>∞ </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>7 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">152.9 </td></tr><tr><td>IPTG− </td><td>320 </td><td>1.86 </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">326 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">145.7 </td></tr><tr><td>IPTG− </td><td>1.94 </td><td>Fast </td><td>2,918 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>106 </td><td>Fast </td><td>0.11 </td><td rowspan="2">Fast </td><td rowspan="2">Fast </td><td rowspan="2">0.0 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td></tr></tbody></table></div><div content-id="DSW009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Joint parameters </th><th>Condition </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th></tr></thead><tbody><tr><td rowspan="2">Independent </td><td>IPTG+ </td><td>110 </td><td>Fast </td><td>0.11 </td><td>Fast </td><td>5 </td><td rowspan="2">14.3 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td><td>Fast </td><td>Fast </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">4,444 </td><td>Fast </td><td>11.50 </td><td>Fast </td><td>964 </td><td rowspan="2">120.3 </td></tr><tr><td>IPTG− </td><td>Fast </td><td>∞ </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>7 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">152.9 </td></tr><tr><td>IPTG− </td><td>320 </td><td>1.86 </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">326 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">145.7 </td></tr><tr><td>IPTG− </td><td>1.94 </td><td>Fast </td><td>2,918 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>106 </td><td>Fast </td><td>0.11 </td><td rowspan="2">Fast </td><td rowspan="2">Fast </td><td rowspan="2">0.0 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The relationship between these parameters and the parameters in Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a> are discussed in the Materials and methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>. Five models are considered, differing in which parameters are assumed to be the same between the two induction conditions. Parameters denoted ‘fast’ are too fast to present on the timescale of seconds. As <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> were found to be fast in all models, the <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> ratio is also shown. ΔBIC values are given as the difference of the model's BIC from the BIC of the best-fitting model (the one with ΔBIC = 0). Models with lower ΔBIC are favoured over models with higher ΔBIC.<sup><a reveal-id="DSW009C58" data-open="DSW009C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198437" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW009TB3" class="table-wrap-title" data-id="DSW009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Fit parameters of the transcription initiation model in Reaction (3) to the measured intervals in the 1× media with and without induction by IPTG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Joint parameters </th><th>Condition </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th></tr></thead><tbody><tr><td rowspan="2">Independent </td><td>IPTG+ </td><td>110 </td><td>Fast </td><td>0.11 </td><td>Fast </td><td>5 </td><td rowspan="2">14.3 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td><td>Fast </td><td>Fast </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">4,444 </td><td>Fast </td><td>11.50 </td><td>Fast </td><td>964 </td><td rowspan="2">120.3 </td></tr><tr><td>IPTG− </td><td>Fast </td><td>∞ </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>7 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">152.9 </td></tr><tr><td>IPTG− </td><td>320 </td><td>1.86 </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">326 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">145.7 </td></tr><tr><td>IPTG− </td><td>1.94 </td><td>Fast </td><td>2,918 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>106 </td><td>Fast </td><td>0.11 </td><td rowspan="2">Fast </td><td rowspan="2">Fast </td><td rowspan="2">0.0 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td></tr></tbody></table></div><div content-id="DSW009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Joint parameters </th><th>Condition </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (s) </th><th>ΔBIC </th></tr></thead><tbody><tr><td rowspan="2">Independent </td><td>IPTG+ </td><td>110 </td><td>Fast </td><td>0.11 </td><td>Fast </td><td>5 </td><td rowspan="2">14.3 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td><td>Fast </td><td>Fast </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">4,444 </td><td>Fast </td><td>11.50 </td><td>Fast </td><td>964 </td><td rowspan="2">120.3 </td></tr><tr><td>IPTG− </td><td>Fast </td><td>∞ </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>7 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">152.9 </td></tr><tr><td>IPTG− </td><td>320 </td><td>1.86 </td><td>Fast </td><td>2,919 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td rowspan="2">326 </td><td rowspan="2">Fast </td><td>∞ </td><td>Fast </td><td>964 </td><td rowspan="2">145.7 </td></tr><tr><td>IPTG− </td><td>1.94 </td><td>Fast </td><td>2,918 </td></tr><tr><td rowspan="2"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> </td><td>IPTG+ </td><td>106 </td><td>Fast </td><td>0.11 </td><td rowspan="2">Fast </td><td rowspan="2">Fast </td><td rowspan="2">0.0 </td></tr><tr><td>IPTG− </td><td>48 </td><td>Fast </td><td>0.01 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The relationship between these parameters and the parameters in Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a> are discussed in the Materials and methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>. Five models are considered, differing in which parameters are assumed to be the same between the two induction conditions. Parameters denoted ‘fast’ are too fast to present on the timescale of seconds. As <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> were found to be fast in all models, the <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mn>1</mn></msub></mrow><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> ratio is also shown. ΔBIC values are given as the difference of the model's BIC from the BIC of the best-fitting model (the one with ΔBIC = 0). Models with lower ΔBIC are favoured over models with higher ΔBIC.<sup><a reveal-id="DSW009C58" data-open="DSW009C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198437" target="_blank">View Large</a></div></div><div class="block-child-p">The mean of the time interval distribution between transcription initiation events, <em>I</em>(<em>R</em>), predicted by Reactions (1) and (2) is, for a given RNAp concentration <em>R</em>: <a id="DSW009M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSW009M4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">I</mi><mo stretchy="false" xmlns="">(</mo><mi xmlns="">R</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mo stretchy="false">(</mo><mrow><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow></msub></mrow><mo stretchy="false">)</mo><mo stretchy="false">(</mo><mrow><msub><mi>k</mi><mrow><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo>+</mo><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow><mo stretchy="false">)</mo></mrow><mrow><mi>R</mi><mrow><msub><mi>k</mi><mn>1</mn></msub></mrow><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow><mrow><msub><mi>k</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow></msub></mrow></mrow></mfrac></mrow><mo>+</mo><mstyle><mrow><mfrac><mn>1</mn><mrow><mrow><msub><mi>k</mi><mn>2</mn></msub></mrow></mrow></mfrac></mrow><mo>+</mo><mstyle><mrow><mfrac><mn>1</mn><mrow><mrow><msub><mi>k</mi><mn>3</mn></msub></mrow></mrow></mfrac></mrow><mo>=</mo><mrow><msub><mi>τ</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow></msub></mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></mstyle></mstyle></mstyle></math></div></div><div class="label">(4)</div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow></msub></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">R</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mrow xmlns=""><msup><mi>R</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math></span> is the mean time taken by the initial binding of RNAp for a given RNAp concentration, and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> is the mean time taken by the steps occurring after the polymerase has committed to transcription until the clearance of the promoter region (due to the initiation of elongation). As such, we expect the majority of the duration of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> to consist of the open complex formation as defined in.<sup><a reveal-id="DSW009C46" data-open="DSW009C46" class="link link-ref link-reveal xref-bibr">46</a></sup> The remaining of its duration we attribute to failures in promoter escape.<sup><a reveal-id="DSW009C59" data-open="DSW009C59" class="link link-ref link-reveal xref-bibr">59</a></sup></div><p>Estimates of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, denoted <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, were obtained from the best-fit parameters of the most parsimonious model, as given in Table <a reveal-id="DSW009TB3" data-open="DSW009TB3" class="link link-reveal link-table xref-fig">3</a>. The standard uncertainties of the estimators <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, denoted <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msubsup><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow><mo stretchy="false" xmlns="">)</mo></math>⁠</span>, were obtained using the Delta Method<sup><a reveal-id="DSW009C60" data-open="DSW009C60" class="link link-ref link-reveal xref-bibr">60</a></sup> from the uncertainties of the model parameters.</p><p>Finally, mean durations of intervals between transcription events for each media condition <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></math>⁠</span>, were estimated by fitting the model in Reaction (3) to the data from only that condition, and taking the mean of the distribution. This procedure was followed to include the censored intervals in the estimate of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></math></span> to avoid underestimating the mean interval duration due to the limited observation times. The standard uncertainty <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mrow><msub><mrow><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></mrow><mo stretchy="false" xmlns="">)</mo></math></span> was estimated using the Delta Method.<sup><a reveal-id="DSW009C60" data-open="DSW009C60" class="link link-ref link-reveal xref-bibr">60</a></sup></p>                            <h3 scrollto-destination=87198441 id="87198441" class="section-title" data-legacy-id=s2i>2.9. Validation of the <em>τ</em>-plot slope</h3>
<p>We verified the slope of the <em>τ</em>-plot in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a> using the RT-PCR measurements from Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a>. These measurements are both linear with respect to <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, but differ by an unknown scaling factor. We denote the estimated production rate as measured by RT-PCR in media condition m as <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>S</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></math>⁠</span>, with standard uncertainty <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mrow><msub><mrow><mrow><mover><mi>S</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></mrow><mo stretchy="false" xmlns="">)</mo></math>⁠</span>. We found this scaling factor by fitting the parameter <em>c</em> in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow><mo xmlns="">=</mo><mi xmlns="">c</mi><msubsup xmlns=""><mrow><mover><mi>S</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> by weighted total least squares<sup><a reveal-id="DSW009C61" data-open="DSW009C61" class="link link-ref link-reveal xref-bibr">61</a></sup> (WTLS), with the measurements weighted by the inverse of their uncertainty (i.e. <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>σ</mi><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msubsup><mrow><mover><mi>S</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow><mo stretchy="false" xmlns="">)</mo></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>σ</mi><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mrow><msub><mrow><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></mrow><mo stretchy="false" xmlns="">)</mo></math>⁠</span>). This method was chosen since it accounts for the uncertainty in both of the measurements. It results in the estimate <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mover><mi>c</mi><mo stretchy="false">ˆ</mo></mover></mrow></math>⁠</span>. The dashed line in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a> was obtained by fitting the scaled points <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mover><mi>c</mi><mo stretchy="false">ˆ</mo></mover></mrow><msubsup xmlns=""><mrow><mover><mi>S</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> against <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mi mathvariant="normal">m</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> by WTLS. The uncertainty shown includes both the uncertainty in the WTLS fit of this line, as well as the uncertainty in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mover><mi>c</mi><mo stretchy="false">ˆ</mo></mover></mrow></math>⁠</span>.</p>                            <h3 scrollto-destination=87198443 id="87198443" class="section-title" data-legacy-id=s2j>2.10. Method to infer the duration of the closed complex of a promoter</h3>
<p>The method to infer the kinetics of transcription initiation <em>in vivo</em> is illustrated in Fig. <a reveal-id="DSW009F1" data-open="DSW009F1" class="link link-reveal link-table xref-fig">1</a>. First, conditions are selected that differ widely in free intracellular RNAp concentrations (step A in Fig. <a reveal-id="DSW009F1" data-open="DSW009F1" class="link link-reveal link-table xref-fig">1</a>). Next, an <em>in vivo</em> single-molecule detection technique is used to sample the time interval distribution between consecutive transcription events in individual cells in each of the conditions (step C in Fig. <a reveal-id="DSW009F1" data-open="DSW009F1" class="link link-reveal link-table xref-fig">1</a>). To obtain these intervals, here we used the MS2d-GFP single RNA detection system<sup><a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a></sup> (step B in Fig. <a reveal-id="DSW009F1" data-open="DSW009F1" class="link link-reveal link-table xref-fig">1</a>). Then, we fit a general model of transcription initiation to the empirical data (see above), which includes both the multi-step nature of transcription initiation as well as the possibility of an intermittently inactive promoter state<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> (Reactions (1) and (2)). From this fit, we obtain an estimate of the <em>in vivo</em> mean duration of the open complex formation by extrapolating the duration of intervals between transcription events to infinite RNAp concentrations, similar to the <em>in vitro</em> extrapolation presented in ref. <a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a> (step D in Fig. <a reveal-id="DSW009F1" data-open="DSW009F1" class="link link-reveal link-table xref-fig">1</a>). The model fit will also assess the importance of an intermittent inactive promoter state and the reversibility and kinetics of the closed complex formation. </p>                        <a id="87198445" scrollto-destination="87198445"></a>
<div id="DSW009F1" data-id="DSW009F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00901.png?Expires=2147483647&Signature=Trgkv-zlcmQSE7E9PxwTx2qCt7JIBz2npZJo9xM~ttxd0SWHVIjuFlzNGW6WqTXZ-H3Kao9n2xV1lYmE5uRGhvM-CAkAPjjKqt~HGc0Cqs69rFjVuP~XJtnWLoSHvbWVKEpwHU1sZCAaDkuIcXhg7nr7nonM6Rfd7l~lbP3MBfTBA33iaWZtSh9~hF1BXQvZEe-XMiBmu3PCt~xdFhIr2~ON3d5bKDtfpwXlK1J3k0nulN75oMh9mFO2z~I~2VQafxrAdiKXGCi~NdqVX1JULYZXyvSZHkCbsOoPWnXP8irM~rCYwj8-B4wOou2W9YA-e4sarwZvtpqXLb5WzSyxPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the in vivo measurement of the initiation kinetics, using simulated data. (A) First, several conditions are selected, labelled I–IV, differing in intracellular RNAp concentration, R. (B) Next, we obtain timeseries of fluorescence and phase contrast (for cell segmentation purposes) images of cells expressing MS2d-GFP and target RNA under the control of the promoter of interest in each condition, from which time intervals between individual transcription events are determined. This is done by jump detection in the total RNA spot intensity of each cell (lower-left in B), from which the interval distribution is obtained (lower-right in B). (C) Mean interval durations are then estimated from these interval distributions for each condition. (D) Finally, the mean interval durations and measurements of R are combined into a τ-plot,22 from which estimates of the mean times taken by the closed complex and open complex formation are obtained for each condition. Arrows depict the flow of information in the measurement procedure." path-from-xml="dsw00901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198445/dsw00901.tiff" path-from-xml="dsw00901.tiff" target="_blank">View large</a><a section="87198445" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00901.png?Expires=2147483647&Signature=qIHY4ZProaVx9zml94VwWxT17xKedJ3S-czapr6BOyL5WZ5-1wraJAzkWWY6fxYlUqsIoPgv2OCJyTCPiHGnosenP5YTEU~6EMo4s8RDkOlCjlb4Tqk1D3hr114k7-wugpIFCbWI~nisEGee~QkyaGcVrTWU8oa88iN0raNN8B5ec3JZegiqvs9jKj80sbozWob7tc-eUrCaiTsreMBi8YmLLwhOb3wq4Q~N7oO-t6BLPEtxTGd8E75Jbq~RiRK66Zh2--j1NpAgNo2pH3yFnuCc6JxAP8l--reioKF9FKqIOKxdnLySeeCIuQ9Vuaav~kSHx2eUDiLPvdhneFkDsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198445&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the <em>in vivo</em> measurement of the initiation kinetics, using simulated data. (A) First, several conditions are selected, labelled I–IV, differing in intracellular RNAp concentration, <em>R</em>. (B) Next, we obtain timeseries of fluorescence and phase contrast (for cell segmentation purposes) images of cells expressing MS2d-GFP and target RNA under the control of the promoter of interest in each condition, from which time intervals between individual transcription events are determined. This is done by jump detection in the total RNA spot intensity of each cell (lower-left in B), from which the interval distribution is obtained (lower-right in B). (C) Mean interval durations are then estimated from these interval distributions for each condition. (D) Finally, the mean interval durations and measurements of <em>R</em> are combined into a <em>τ</em>-plot,<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a></sup> from which estimates of the mean times taken by the closed complex and open complex formation are obtained for each condition. Arrows depict the flow of information in the measurement procedure.</p></div></div><div content-id="DSW009F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00901.png?Expires=2147483647&Signature=Trgkv-zlcmQSE7E9PxwTx2qCt7JIBz2npZJo9xM~ttxd0SWHVIjuFlzNGW6WqTXZ-H3Kao9n2xV1lYmE5uRGhvM-CAkAPjjKqt~HGc0Cqs69rFjVuP~XJtnWLoSHvbWVKEpwHU1sZCAaDkuIcXhg7nr7nonM6Rfd7l~lbP3MBfTBA33iaWZtSh9~hF1BXQvZEe-XMiBmu3PCt~xdFhIr2~ON3d5bKDtfpwXlK1J3k0nulN75oMh9mFO2z~I~2VQafxrAdiKXGCi~NdqVX1JULYZXyvSZHkCbsOoPWnXP8irM~rCYwj8-B4wOou2W9YA-e4sarwZvtpqXLb5WzSyxPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the in vivo measurement of the initiation kinetics, using simulated data. (A) First, several conditions are selected, labelled I–IV, differing in intracellular RNAp concentration, R. (B) Next, we obtain timeseries of fluorescence and phase contrast (for cell segmentation purposes) images of cells expressing MS2d-GFP and target RNA under the control of the promoter of interest in each condition, from which time intervals between individual transcription events are determined. This is done by jump detection in the total RNA spot intensity of each cell (lower-left in B), from which the interval distribution is obtained (lower-right in B). (C) Mean interval durations are then estimated from these interval distributions for each condition. (D) Finally, the mean interval durations and measurements of R are combined into a τ-plot,22 from which estimates of the mean times taken by the closed complex and open complex formation are obtained for each condition. Arrows depict the flow of information in the measurement procedure." path-from-xml="dsw00901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198445/dsw00901.tiff" path-from-xml="dsw00901.tiff" target="_blank">View large</a><a section="87198445" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00901.png?Expires=2147483647&Signature=qIHY4ZProaVx9zml94VwWxT17xKedJ3S-czapr6BOyL5WZ5-1wraJAzkWWY6fxYlUqsIoPgv2OCJyTCPiHGnosenP5YTEU~6EMo4s8RDkOlCjlb4Tqk1D3hr114k7-wugpIFCbWI~nisEGee~QkyaGcVrTWU8oa88iN0raNN8B5ec3JZegiqvs9jKj80sbozWob7tc-eUrCaiTsreMBi8YmLLwhOb3wq4Q~N7oO-t6BLPEtxTGd8E75Jbq~RiRK66Zh2--j1NpAgNo2pH3yFnuCc6JxAP8l--reioKF9FKqIOKxdnLySeeCIuQ9Vuaav~kSHx2eUDiLPvdhneFkDsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198445&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the <em>in vivo</em> measurement of the initiation kinetics, using simulated data. (A) First, several conditions are selected, labelled I–IV, differing in intracellular RNAp concentration, <em>R</em>. (B) Next, we obtain timeseries of fluorescence and phase contrast (for cell segmentation purposes) images of cells expressing MS2d-GFP and target RNA under the control of the promoter of interest in each condition, from which time intervals between individual transcription events are determined. This is done by jump detection in the total RNA spot intensity of each cell (lower-left in B), from which the interval distribution is obtained (lower-right in B). (C) Mean interval durations are then estimated from these interval distributions for each condition. (D) Finally, the mean interval durations and measurements of <em>R</em> are combined into a <em>τ</em>-plot,<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a></sup> from which estimates of the mean times taken by the closed complex and open complex formation are obtained for each condition. Arrows depict the flow of information in the measurement procedure.</p></div></div>                            <h2 scrollto-destination=87198446 id="87198446" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87198447 id="87198447" class="section-title" data-legacy-id=s3a>3.1. Changing free RNA polymerase concentrations</h3>
<p>We first verified that it is possible to change intracellular RNAp concentration by a wide range by changing the growth conditions of the cells.<sup><a reveal-id="DSW009C32" data-open="DSW009C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSW009C35" data-open="DSW009C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSW009C62" data-open="DSW009C62" class="link link-ref link-reveal xref-bibr">62</a></sup> As such, we grew cells in four media (described in the Materials and methods), labelled 1×, 0.75×, 0.5×, and 0.25×, which solely differ in richness of two components (tryptone and yeast extract). We then measured the relative RNAp concentrations in cells grown in these four media using RT-PCR of the <em>rpoC</em> gene, i.e. the gene coding for the β′ subunit, which is the limiting factor in the assembly of the RNAp holoenzyme.<sup><a reveal-id="DSW009C48" data-open="DSW009C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSW009C57" data-open="DSW009C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSW009C62" data-open="DSW009C62" class="link link-ref link-reveal xref-bibr">62</a></sup> Results in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> (dark grey bars) show that, in the range tested, the RNAp concentration in the cells increases significantly with increasing media richness. </p>                        <a id="87198449" scrollto-destination="87198449"></a>
<div id="DSW009F2" data-id="DSW009F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00902.png?Expires=2147483647&Signature=bA4elKytkxcZqPGZjk7-YIyrDUgtYmfNtilgcOY1kwkqLlL7MxpkvyaZVyrYe9QJihPiiz2BvJl1ZysAv3wOVfA-4r8km7KnUhfBtRGSmefWxduEKtuEIYfUNWPa-ccEjNYU1JdJxk-~NkdAJNMw5YbZ8DyzkHGHUO1k7O2YpgfiwR7YAABmdb1zyPydOYYmU9aGXBRGsg4DBYzI0ZX11VbVzuuJ9cWAIA~gqOHvEDuKAD2gUVmPPuj1nlydYbbfg4VKjuommQftU8KrgFxOkuBGX8tEuBAZ3j2fpATlDyUW3YPs8ZIQLiKoiuYAxllKeExFJvK-Q8tM9zvfyGKl8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Measurements of the relative intracellular RNAp concentrations (Rˆm) for cells growing in the four different media. Bars show the standard uncertainties (σ(Rˆm)) of the measurements. Data is from two replicates with 3 technical replicates each (DH5α-PRO, RT-PCR, and W3110, RT-PCR), and three replicates with three technical replicates each (RL1314, RpoC::GFP). All data are presented relative to the RNAp concentration at 1×. The media used are denoted as m×, where the composition per 100 ml is: m grams of tryptone, m/2 grams of yeast extract and 1 g of NaCl (pH = 7.0).For example, 0.25× media has 0.25 g of tryptone and 0.125 g of yeast extract per 100 ml." path-from-xml="dsw00902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198449/dsw00902.tiff" path-from-xml="dsw00902.tiff" target="_blank">View large</a><a section="87198449" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00902.png?Expires=2147483647&Signature=g7hVsy3v~lz6XpvAvYJiwDF84muFBourHuAA9EK3luLoggxTMY~bDfVxWdCeKBg90w-JNtOjwMHKuNoXz9GlJ83jiWToCDro0xUWC2FSPO6RT2ANsN0MYEHeFttXYIX6-5Yg9GPL5kn0Hf2ClsdqdlVWqNzuYSj8du8mnaJHEIbpEDNMODNFMC6GX8ES-h~IGks74zbydKLc4QEggh58bg0qLl21T6YsR4u0mSVrXOSNkevCvWKdDG-Xv8GX0CAEuq8cal48~CVzHj2hcp9-m4ihIYwStXSxlBqgp68zIGFQZOYdzWvQXvr-byM3KojbLOJt7NE2SFc7WBvjcqQQ3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198449&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Measurements of the relative intracellular RNAp concentrations <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mrow><msub><mrow><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mi>m</mi></msub></mrow></mrow><mo stretchy="false" xmlns="">)</mo></math></span> for cells growing in the four different media. Bars show the standard uncertainties <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mi>σ</mi><mo stretchy="false">(</mo><mrow><mrow><msub><mrow><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></mrow><mo stretchy="false">)</mo></mrow><mo stretchy="false" xmlns="">)</mo></math></span> of the measurements. Data is from two replicates with 3 technical replicates each (DH5α-PRO, RT-PCR, and W3110, RT-PCR), and three replicates with three technical replicates each (RL1314, RpoC::GFP). All data are presented relative to the RNAp concentration at 1×. The media used are denoted as <em>m</em>×, where the composition per 100 ml is: <em>m</em> grams of tryptone, <em>m</em>/2 grams of yeast extract and 1 g of NaCl (pH = 7.0).For example, 0.25× media has 0.25 g of tryptone and 0.125 g of yeast extract per 100 ml.</p></div></div><div content-id="DSW009F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00902.png?Expires=2147483647&Signature=bA4elKytkxcZqPGZjk7-YIyrDUgtYmfNtilgcOY1kwkqLlL7MxpkvyaZVyrYe9QJihPiiz2BvJl1ZysAv3wOVfA-4r8km7KnUhfBtRGSmefWxduEKtuEIYfUNWPa-ccEjNYU1JdJxk-~NkdAJNMw5YbZ8DyzkHGHUO1k7O2YpgfiwR7YAABmdb1zyPydOYYmU9aGXBRGsg4DBYzI0ZX11VbVzuuJ9cWAIA~gqOHvEDuKAD2gUVmPPuj1nlydYbbfg4VKjuommQftU8KrgFxOkuBGX8tEuBAZ3j2fpATlDyUW3YPs8ZIQLiKoiuYAxllKeExFJvK-Q8tM9zvfyGKl8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Measurements of the relative intracellular RNAp concentrations (Rˆm) for cells growing in the four different media. Bars show the standard uncertainties (σ(Rˆm)) of the measurements. Data is from two replicates with 3 technical replicates each (DH5α-PRO, RT-PCR, and W3110, RT-PCR), and three replicates with three technical replicates each (RL1314, RpoC::GFP). All data are presented relative to the RNAp concentration at 1×. The media used are denoted as m×, where the composition per 100 ml is: m grams of tryptone, m/2 grams of yeast extract and 1 g of NaCl (pH = 7.0).For example, 0.25× media has 0.25 g of tryptone and 0.125 g of yeast extract per 100 ml." path-from-xml="dsw00902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198449/dsw00902.tiff" path-from-xml="dsw00902.tiff" target="_blank">View large</a><a section="87198449" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00902.png?Expires=2147483647&Signature=g7hVsy3v~lz6XpvAvYJiwDF84muFBourHuAA9EK3luLoggxTMY~bDfVxWdCeKBg90w-JNtOjwMHKuNoXz9GlJ83jiWToCDro0xUWC2FSPO6RT2ANsN0MYEHeFttXYIX6-5Yg9GPL5kn0Hf2ClsdqdlVWqNzuYSj8du8mnaJHEIbpEDNMODNFMC6GX8ES-h~IGks74zbydKLc4QEggh58bg0qLl21T6YsR4u0mSVrXOSNkevCvWKdDG-Xv8GX0CAEuq8cal48~CVzHj2hcp9-m4ihIYwStXSxlBqgp68zIGFQZOYdzWvQXvr-byM3KojbLOJt7NE2SFc7WBvjcqQQ3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198449&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Measurements of the relative intracellular RNAp concentrations <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mrow><msub><mrow><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mi>m</mi></msub></mrow></mrow><mo stretchy="false" xmlns="">)</mo></math></span> for cells growing in the four different media. Bars show the standard uncertainties <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mi>σ</mi><mo stretchy="false">(</mo><mrow><mrow><msub><mrow><mrow><mover><mi>R</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow></mrow><mo stretchy="false">)</mo></mrow><mo stretchy="false" xmlns="">)</mo></math></span> of the measurements. Data is from two replicates with 3 technical replicates each (DH5α-PRO, RT-PCR, and W3110, RT-PCR), and three replicates with three technical replicates each (RL1314, RpoC::GFP). All data are presented relative to the RNAp concentration at 1×. The media used are denoted as <em>m</em>×, where the composition per 100 ml is: <em>m</em> grams of tryptone, <em>m</em>/2 grams of yeast extract and 1 g of NaCl (pH = 7.0).For example, 0.25× media has 0.25 g of tryptone and 0.125 g of yeast extract per 100 ml.</p></div></div><p>To validate this result, we measured the relative RNAp concentrations by plate reader in cells expressing fluorescently tagged RpoC in the strain RL1314 (derived from W3110),<sup><a reveal-id="DSW009C33" data-open="DSW009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> in the same four media. In addition, we also measured the levels of the <em>rpoC</em> transcripts in the strain W3110 by RT-PCR in the 0.5× and 1× conditions. Results (Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a>) show that the relative changes in the protein and mRNA levels of <em>rpoC</em> match the measurements by RT-PCR of the <em>rpoC</em> gene in DH5α-PRO.</p><p>Note that, even though the experimental procedures and strains differ, our measurements are in agreement with the relative changes in RNAp concentrations reported in ref. <a reveal-id="DSW009C32" data-open="DSW009C32" class="link link-ref link-reveal xref-bibr">32</a>, for the difference in growth rates observed here between the 0.25× and 1× conditions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), which we estimate to be ∼0.48 (Materials and methods). In this regard, given that the same result applies to (at least) three different strains, we expect it to be significantly strain-independent.</p><p>Finally, to verify that the relative RNAp concentrations measured in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> are maintained under the microscope, we measured the relative RNAp concentration in the RL1314 cells expressing fluorescently tagged RpoC under the microscope between the two extreme conditions (0.25× and 1×), after 1 h in the thermal imaging chamber (Materials and methods). The relative RNAp concentration between the conditions was measured to be 0.367 ± 0.012, which is consistent with the measurements in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a>. Lastly, from these images, we did not observe significant cell-to-cell variability in the RNAp concentrations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), indicating that the mean concentrations reported in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> are representative of the populations.</p><p>These measurements show that the relative RNAp concentration changes widely between the selected growth conditions. However, the variable affecting transcription kinetics is the relative free RNAp concentration. As such, we must verify whether the relative total RNAp concentration can be used as a proxy for the relative free RNAp concentrations. If this holds true and there are no other factors affecting the production rate of the promoter of interest in these conditions, then the RNA production rate should be hyperbolic with respect to the RNAp concentration. That is, the reciprocal of the RNA production rate from this promoter should be linear when plotted against the reciprocal of the measured relative RNAp concentrations, and one should obtain a line on a Lineweaver–Burk plot.</p><p>There are several reasons why this plot may not be linear. If, for example, the ratio of free RNAp to total RNAp is not constant in this range of growth conditions, with a higher fraction of free RNAp in the poorer growth conditions due to increased ppGpp,<sup><a reveal-id="DSW009C31" data-open="DSW009C31" class="link link-ref link-reveal xref-bibr">31</a></sup> then we expect a curve with positive curvature on this plot. Meanwhile, a negative curvature would be obtained if the promoter of interest could be induced by increased cAMP in the poorer growth conditions, or if the cells spent, on average, a significantly increased amount of time with multiple copies of the plasmid in the richer growth conditions, among other possibilities. In these cases, to dissect the transcription initiation kinetics of such promoters, another method of modifying the free RNAp concentration will be required.</p><p>Given the above, we interpret a straight line on the Lineweaver–Burk plot as evidence that, for the conditions tested, (i) the relative free RNAp concentrations can be assessed from the total RNAp concentrations, and (ii) no factors other than the changes in the free RNAp concentration affect the target promoter.</p><p>Here, we tested this by measuring the RNA production rate from P<em><sub>lac/ara-1</sub></em> in <em>E. coli</em> DH5α-PRO by RT-PCR in the same four media conditions as in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a>. We selected this promoter, since its dynamics has been extensively characterized<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C21" data-open="DSW009C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSW009C63" data-open="DSW009C63" class="link link-ref link-reveal xref-bibr">63</a>–<a reveal-id="DSW009C67" data-open="DSW009C67" class="link link-ref link-reveal xref-bibr">67</a></sup> and because it has the same logical structure as the <em>lac</em> promoter, with an activator and a repressor.<sup><a reveal-id="DSW009C63" data-open="DSW009C63" class="link link-ref link-reveal xref-bibr">63</a></sup> The resulting Lineweaver–Burk plot is shown in Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a> where a linear relationship is clearly observed between these points (black points). To determine whether the small deviations from linearity are statistically significant, we performed a likelihood ratio test between a linear fit by WTLS<sup><a reveal-id="DSW009C61" data-open="DSW009C61" class="link link-ref link-reveal xref-bibr">61</a></sup> (shown as a line in Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a>), and fits with higher order polynomials (also by WTLS by minimizing <em>χ</em><sup>2</sup> as in<sup><a reveal-id="DSW009C61" data-open="DSW009C61" class="link link-ref link-reveal xref-bibr">61</a></sup>). No test rejected the linear model (all <em>P</em> &gt; 0.25). As noted earlier, this relationship is only expected to occur in a limited range of growth conditions. To illustrate this, we repeated the same measurements in 1.5× media (grey point in Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a>). The result shows that this hyperbolic relationship is lost in very rich media (including this point causes the likelihood ratio test to reject the linear model, <em>P</em> = 0.0014). We conclude that, for the growth conditions in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a>, the relative free RNAp concentrations are well-approximated by the total RNAp concentrations, and there are no significant other factors affecting the initiation dynamics of P<em><sub>lac/ara-1</sub></em>. </p>                        <a id="87198457" scrollto-destination="87198457"></a>
<div id="DSW009F3" data-id="DSW009F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00903.png?Expires=2147483647&Signature=h~rZScSiYiRkgK~IjTfwFE1gsR23s8by9dIxwXv6GTznmfBowSo87n3VVNEkUlF0xT5-7sYqNXzlNqgIDIZ9yOUpywenshSw5rWnE~Eb1Wvawr-wyBK4B06wYqzmdEVEzVGVKB5lIPqebBFrE1hP7eeQjDTCClY1yRgPdcjayAlqGk75uLUgnzMcDdutUTc~eZYrg1Rip2t8DDbLZhFaaZvZijqO1JJxQSO6~pCELFb5Mt9SzEgUtTfAH2bfjkoEUiyxKP1gGiCP~z4xLf-hCiRKQ~ZH6PYk8L7GJ4V620fxrgsPl-gBkm0dYcd4osIZjdBEDtcxxrkJGzpkP5xAaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lineweaver–Burk plot of the inverse of the production rate of mRFP1 from the Plac/ara-1 promoter against the inverse of the total RNAp concentrations for the same growth conditions as in Fig. 2 (black points), and for 1.50× media (grey point). Standard uncertainties are shown for both quantities (horizontal and vertical error bars). Relative production rates were measured by RT-PCR with two biological replicates with three technical replicates each." path-from-xml="dsw00903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198457/dsw00903.tiff" path-from-xml="dsw00903.tiff" target="_blank">View large</a><a section="87198457" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00903.png?Expires=2147483647&Signature=x92kVIcU3d9twAYZwNYX2Aa8udlfaTHjVdX0c6~q92BjxeShIyrwSdfLROouOokVqQoP14xq9g6qAgtt940OGnn~GtENHJY0Ra2-uBr7p~G2ZaYRJv~caTMSnsYQoy0ldGrwPbN27ouOh4puc9vxwjKdiM-Ok7EY0Xp2AcRjG814yvI9ag3O9HxtpyZLtGiCLEUrhtIZTDh13xQWPSssndLfvAtH856jn7cl~F404mtNddAo-UqpK~zQGZjXE92tML~JMAT7aMuNQ1mhOPmOpS8X97xHy5EAHALwTE8HXPmqsxLfR9CT~LFcPG~qiDJQEUv~rm3wG5LdsDXeC03q~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198457&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lineweaver–Burk plot of the inverse of the production rate of mRFP1 from the P<em><sub>lac/ara-1</sub></em> promoter against the inverse of the total RNAp concentrations for the same growth conditions as in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> (black points), and for 1.50× media (grey point). Standard uncertainties are shown for both quantities (horizontal and vertical error bars). Relative production rates were measured by RT-PCR with two biological replicates with three technical replicates each.</p></div></div><div content-id="DSW009F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00903.png?Expires=2147483647&Signature=h~rZScSiYiRkgK~IjTfwFE1gsR23s8by9dIxwXv6GTznmfBowSo87n3VVNEkUlF0xT5-7sYqNXzlNqgIDIZ9yOUpywenshSw5rWnE~Eb1Wvawr-wyBK4B06wYqzmdEVEzVGVKB5lIPqebBFrE1hP7eeQjDTCClY1yRgPdcjayAlqGk75uLUgnzMcDdutUTc~eZYrg1Rip2t8DDbLZhFaaZvZijqO1JJxQSO6~pCELFb5Mt9SzEgUtTfAH2bfjkoEUiyxKP1gGiCP~z4xLf-hCiRKQ~ZH6PYk8L7GJ4V620fxrgsPl-gBkm0dYcd4osIZjdBEDtcxxrkJGzpkP5xAaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lineweaver–Burk plot of the inverse of the production rate of mRFP1 from the Plac/ara-1 promoter against the inverse of the total RNAp concentrations for the same growth conditions as in Fig. 2 (black points), and for 1.50× media (grey point). Standard uncertainties are shown for both quantities (horizontal and vertical error bars). Relative production rates were measured by RT-PCR with two biological replicates with three technical replicates each." path-from-xml="dsw00903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198457/dsw00903.tiff" path-from-xml="dsw00903.tiff" target="_blank">View large</a><a section="87198457" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00903.png?Expires=2147483647&Signature=x92kVIcU3d9twAYZwNYX2Aa8udlfaTHjVdX0c6~q92BjxeShIyrwSdfLROouOokVqQoP14xq9g6qAgtt940OGnn~GtENHJY0Ra2-uBr7p~G2ZaYRJv~caTMSnsYQoy0ldGrwPbN27ouOh4puc9vxwjKdiM-Ok7EY0Xp2AcRjG814yvI9ag3O9HxtpyZLtGiCLEUrhtIZTDh13xQWPSssndLfvAtH856jn7cl~F404mtNddAo-UqpK~zQGZjXE92tML~JMAT7aMuNQ1mhOPmOpS8X97xHy5EAHALwTE8HXPmqsxLfR9CT~LFcPG~qiDJQEUv~rm3wG5LdsDXeC03q~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198457&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lineweaver–Burk plot of the inverse of the production rate of mRFP1 from the P<em><sub>lac/ara-1</sub></em> promoter against the inverse of the total RNAp concentrations for the same growth conditions as in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> (black points), and for 1.50× media (grey point). Standard uncertainties are shown for both quantities (horizontal and vertical error bars). Relative production rates were measured by RT-PCR with two biological replicates with three technical replicates each.</p></div></div>                            <h3 scrollto-destination=87198458 id="87198458" class="section-title" data-legacy-id=s3b>3.2. Interval distributions between consecutive RNA productions</h3>
<p>Given this, it is possible to apply a standard model-fitting procedure to fully characterize the <em>in vivo</em> kinetics of the rate-limiting steps in transcription initiation of the P<em><sub>lac/ara-1</sub></em> promoter from distributions of intervals between transcription events in cells with different RNA polymerase concentrations.</p><p>We measured the distribution of time intervals between transcription events (hereafter referred to as ‘intervals’) for P<em><sub>lac/ara-1</sub></em> in each cell growth condition using the MS2d-GFP single-RNA detection system,<sup><a reveal-id="DSW009C1" data-open="DSW009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> with a least-deviation jump-detection procedure<sup><a reveal-id="DSW009C41" data-open="DSW009C41" class="link link-ref link-reveal xref-bibr">41</a></sup> (Materials and methods). This measurement results in samples from the interval distribution as well as ‘censored’ intervals, i.e. intervals for which we only observe the beginning due to cell division or the end of the time series. Both censored and uncensored intervals were accounted for in all parameter estimates to avoid biasing the estimates. For example, note that taking the mean of the uncensored intervals alone would underestimate the mean of the true interval distribution since long unobservable intervals would be absent from the estimate. Including the censored intervals balances this by considering long intervals that are at least as long as the censored interval length.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>From these distributions, we estimated the true mean and the squared coefficient of variation (CV<sup>2</sup>, defined as the variance over the squared mean) of the interval distributions (Materials and methods). We chose CV<sup>2</sup> for quantifying the noise in the interval distribution since, to a good approximation, this quantity reflects the level of noise in the protein levels regardless of the actual shape of the transcription interval distribution.<sup><a reveal-id="DSW009C68" data-open="DSW009C68" class="link link-ref link-reveal xref-bibr">68</a></sup> Further, this variable equals 1 for the interval distribution of a Poisson process (i.e. an exponential distribution), regardless of the mean rate. These results, along with the amount of empirical data used, are shown in Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>.</p><p>From Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>, the mean interval decreases significantly with increasing media richness, as expected from the increased RNAp concentrations. Meanwhile, the CV<sup>2</sup> does not exhibit the same dependence on the media richness, and remains slightly &gt;1 in all conditions tested.</p>                            <h3 scrollto-destination=87198463 id="87198463" class="section-title" data-legacy-id=s3c>3.3. Decomposition of the <em>in vivo</em> kinetics</h3>
<p>From the data in Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>, we next recreate the Lineweaver–Burk plot in Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a> (white circles in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a>), using the mean interval durations between RNA productions, as this quantity is an absolute measure of the inverse rate of RNA production (this plot is called a <em>τ</em>-plot). </p>                        <a id="87198465" scrollto-destination="87198465"></a>
<div id="DSW009F4" data-id="DSW009F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00904.png?Expires=2147483647&Signature=h-NI9CV7stW0GkHhkankrxQw7Qjt0b-P0ISBoUsh0578uTsMSQ2R4sAJYiVs7TkEdp~BjZ8HihyuiolKBaDNeeGHn00x-Gt~g2ypRuCdUNxT5k2EP6KN5eg3KVWgB1vAqQ4NOCMjGpSvvEEWXuVDz5GRClwl2zRVpt4KttofhfmY9iKQ41hq7ydd1FJoOzuoXD3v1AB2w9Mh0MofPGNxTwpjgTQiVj9ql7gS4Xah36nX6h1phUXXF641zD6aPcdtWYbmsqgv3FHtxJm3d~Nc0I~S6YmUJmT0l9IJ3iDlm8LqadeoyFilRvd8oS2qV~c8ICGWaPtxrRHwPA-PgRXzRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="τ-plot for Plac/ara-1, showing the mean interval between transcription events in individual cells for each media condition (white circles), with their standard uncertainties (vertical error bars) and the standard uncertainties of the relative RNAp concentrations (horizontal error bars). Also shown is the best-fit line (solid line), as determined by the intercept and slope obtained from the best-fitting model (Table 3), with one standard uncertainty estimated by Scheffé's method69 combined with the Delta Method60 (grey area). In addition, the figure shows the data from Fig. 3 (triangles), and the best-fitting line (dashed line, see Materials and methods) with one standard uncertainty estimated by Scheffé's method69 (dotted black curves)." path-from-xml="dsw00904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198465/dsw00904.tiff" path-from-xml="dsw00904.tiff" target="_blank">View large</a><a section="87198465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00904.png?Expires=2147483647&Signature=SczvcVRhL4i-q6blVavxwgSQ-XYKqz6FadUazSU-HV18mtISZca4wEsYt0rD3mQfTAC-MSEvnDmmzvSfbtmMKQEt1oBLKOOcQL3u9rRvaplbiBZUdLPVZSNI6cGtx12Xerv8NXxzm1VD01kpqnhjbzbOkVb3wCiiLlW~oS0o2ubAS4oiVjnt8-hq987uC6OXjyHOP1BvVlxTJicDxFDdlk4zT9TsHbkqWuFO3BCUEhfPtEBvGf8Mj3ASjSo0o2UUPGlugYMZeIs21g57qgzKie~75ViuGi2rEFUqE3OzQdYrJTo1WgiQ3IyCOm9puRFfGJ9NU87V6j1vdLAMjP00cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198465&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>τ</em>-plot for P<em><sub>lac/ara-1</sub></em>, showing the mean interval between transcription events in individual cells for each media condition (white circles), with their standard uncertainties (vertical error bars) and the standard uncertainties of the relative RNAp concentrations (horizontal error bars). Also shown is the best-fit line (solid line), as determined by the intercept and slope obtained from the best-fitting model (Table <a reveal-id="DSW009TB3" data-open="DSW009TB3" class="link link-reveal link-table xref-fig">3</a>), with one standard uncertainty estimated by Scheffé's method<sup><a reveal-id="DSW009C69" data-open="DSW009C69" class="link link-ref link-reveal xref-bibr">69</a></sup> combined with the Delta Method<sup><a reveal-id="DSW009C60" data-open="DSW009C60" class="link link-ref link-reveal xref-bibr">60</a></sup> (grey area). In addition, the figure shows the data from Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a> (triangles), and the best-fitting line (dashed line, see Materials and methods) with one standard uncertainty estimated by Scheffé's method<sup><a reveal-id="DSW009C69" data-open="DSW009C69" class="link link-ref link-reveal xref-bibr">69</a></sup> (dotted black curves).</p></div></div><div content-id="DSW009F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00904.png?Expires=2147483647&Signature=h-NI9CV7stW0GkHhkankrxQw7Qjt0b-P0ISBoUsh0578uTsMSQ2R4sAJYiVs7TkEdp~BjZ8HihyuiolKBaDNeeGHn00x-Gt~g2ypRuCdUNxT5k2EP6KN5eg3KVWgB1vAqQ4NOCMjGpSvvEEWXuVDz5GRClwl2zRVpt4KttofhfmY9iKQ41hq7ydd1FJoOzuoXD3v1AB2w9Mh0MofPGNxTwpjgTQiVj9ql7gS4Xah36nX6h1phUXXF641zD6aPcdtWYbmsqgv3FHtxJm3d~Nc0I~S6YmUJmT0l9IJ3iDlm8LqadeoyFilRvd8oS2qV~c8ICGWaPtxrRHwPA-PgRXzRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="τ-plot for Plac/ara-1, showing the mean interval between transcription events in individual cells for each media condition (white circles), with their standard uncertainties (vertical error bars) and the standard uncertainties of the relative RNAp concentrations (horizontal error bars). Also shown is the best-fit line (solid line), as determined by the intercept and slope obtained from the best-fitting model (Table 3), with one standard uncertainty estimated by Scheffé's method69 combined with the Delta Method60 (grey area). In addition, the figure shows the data from Fig. 3 (triangles), and the best-fitting line (dashed line, see Materials and methods) with one standard uncertainty estimated by Scheffé's method69 (dotted black curves)." path-from-xml="dsw00904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198465/dsw00904.tiff" path-from-xml="dsw00904.tiff" target="_blank">View large</a><a section="87198465" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00904.png?Expires=2147483647&Signature=SczvcVRhL4i-q6blVavxwgSQ-XYKqz6FadUazSU-HV18mtISZca4wEsYt0rD3mQfTAC-MSEvnDmmzvSfbtmMKQEt1oBLKOOcQL3u9rRvaplbiBZUdLPVZSNI6cGtx12Xerv8NXxzm1VD01kpqnhjbzbOkVb3wCiiLlW~oS0o2ubAS4oiVjnt8-hq987uC6OXjyHOP1BvVlxTJicDxFDdlk4zT9TsHbkqWuFO3BCUEhfPtEBvGf8Mj3ASjSo0o2UUPGlugYMZeIs21g57qgzKie~75ViuGi2rEFUqE3OzQdYrJTo1WgiQ3IyCOm9puRFfGJ9NU87V6j1vdLAMjP00cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198465&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>τ</em>-plot for P<em><sub>lac/ara-1</sub></em>, showing the mean interval between transcription events in individual cells for each media condition (white circles), with their standard uncertainties (vertical error bars) and the standard uncertainties of the relative RNAp concentrations (horizontal error bars). Also shown is the best-fit line (solid line), as determined by the intercept and slope obtained from the best-fitting model (Table <a reveal-id="DSW009TB3" data-open="DSW009TB3" class="link link-reveal link-table xref-fig">3</a>), with one standard uncertainty estimated by Scheffé's method<sup><a reveal-id="DSW009C69" data-open="DSW009C69" class="link link-ref link-reveal xref-bibr">69</a></sup> combined with the Delta Method<sup><a reveal-id="DSW009C60" data-open="DSW009C60" class="link link-ref link-reveal xref-bibr">60</a></sup> (grey area). In addition, the figure shows the data from Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a> (triangles), and the best-fitting line (dashed line, see Materials and methods) with one standard uncertainty estimated by Scheffé's method<sup><a reveal-id="DSW009C69" data-open="DSW009C69" class="link link-ref link-reveal xref-bibr">69</a></sup> (dotted black curves).</p></div></div><p>Previously, using <em>in vitro</em> techniques, it has only been possible to extract from a <em>τ</em>-plot the mean duration of the open complex formation (the <em>y</em>-intercept of the plot, here denoted <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math>⁠</span>), because the plot is based on the steady-state assay which only measures the mean rate of abortive transcription initiations. However, the distributions of time intervals between RNA productions contain information about the stochasticity of the process (i.e. the variability between intervals). As such, it is possible to extract a more complete model of the process of transcription. Namely, aside from the open complex formation, as mentioned in Materials and methods, it is possible to extract information on the closed complex and on an intermittent state prior to the closed complex formation.</p><p>In particular, we consider the detailed model of transcription initiation presented in Materials and methods (Reactions (1) and (2)), along with simplified models that can be considered if certain steps of the more detailed model do not influence the distribution of intervals. This model assumes that only one copy of the promoter is present in each cell at any given time, since in all conditions, the bacteria divided slowly, which suggests that they spent most lifetime with only one chromosome. We then consider three simplified models. First, if the time spent in the OFF state is very small, or if the system switches between OFF and ON very rapidly when compared with the forward reaction, then Reaction (2) will not affect the RNA production dynamics. A sufficient condition for both of these situations is that <em>k</em><sub>ON</sub> &gt;&gt; <em>k</em><sub>1</sub>. The other two simplifications are two limits of the closed complex formation, first considered in<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a></sup>: (i) <em>k</em><sub>−1</sub> &gt;&gt; <em>k</em><sub>2</sub>, i.e. it is reversible (Limiting Mechanism I), and (ii) <em>k</em><sub>2</sub> &gt;&gt; <em>k</em><sub>−1</sub>, i.e. irreversible (Limiting Mechanism II). Limiting Mechanism I was found to be more likely in several <em>in vitro</em> measurements of various promoters.<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW009C23" data-open="DSW009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSW009C26" data-open="DSW009C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>While all three simplifications are consistent with a line on a <em>τ</em>-plot, they produce significantly different distributions of intervals between RNA production events. For example, a significant ON/OFF mechanism will result in a more noisy distribution (a higher CV<sup>2</sup>).<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Similarly, Limiting Mechanism I effectively eliminates one limiting step, which also results in higher noise when compared with Limiting Mechanism II (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p><p>We fit the full and simplified models of transcription initiation to the observed dynamics of P<em><sub>lac/ara-1</sub></em> from all media conditions (Materials and methods). We used the Bayesian Information Criterion<sup><a reveal-id="DSW009C70" data-open="DSW009C70" class="link link-ref link-reveal xref-bibr">70</a></sup> (BIC) to compare the fits. The BIC is a model selection criterion which balances goodness-of-fit with the number of parameters to determine which model is most likely the ‘truth’. The difference between BIC values (ΔBIC) can be interpreted as evidence <em>against</em> the model with <em>higher</em> BIC, with a ΔBIC &gt; 5 being interpreted as strong evidence.<sup><a reveal-id="DSW009C58" data-open="DSW009C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Results are shown in Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a>. Since, for several of the models, the optimal fit was for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mn>3</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo xmlns="">=</mo><mn xmlns="">0</mn></math>⁠</span>, we also considered models that do not include another rate-limiting step after the open complex formation.</p><p>From Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a>, the initiation kinetics of P<em><sub>lac/ara-1</sub></em> is best-fit by Limiting Mechanism I (i.e. a reversible closed complex), with very high certainty (ΔBIC of all other models &gt;8). We also find evidence for a significant ON/OFF mechanism. Though the time spent in each OFF state is short (∼87 s), it will turn OFF, on average, ∼9.1 times before committing to transcription in the 1× case (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>). This results in an interval distribution which is only slightly more noisy than what would be expected if the production process were Poissonian (i.e. a CV<sup>2</sup> of the interval distribution of 1; see the CV<sup>2</sup> values in Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>). Interestingly, this implies that the noise in transcription of this promoter is representative of the behaviour of the majority of promoters in <em>E. coli</em>.<sup><a reveal-id="DSW009C27" data-open="DSW009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Finally, the steps after the commitment to transcription are fast, indicating that abortive initiation events do not play a significant role in the dynamics of RNA production by P<em><sub>lac/ara-1</sub></em>. This model is depicted graphically in Fig. <a reveal-id="DSW009F5" data-open="DSW009F5" class="link link-reveal link-table xref-fig">5</a>. </p>                        <a id="87198471" scrollto-destination="87198471"></a>
<div id="DSW009F5" data-id="DSW009F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00905.png?Expires=2147483647&Signature=gC5kj8mP5jSKmUfMs6SF6pFNMfLeWkwL9GVn2fSzHWENYbaM6YSOWw-6e3QqJDOrWU3qlf8rIJizd6pIlbA~WStFrxSV2VwFC0GxlEkEcFCWT6Rh5ZlNGzhrZ-w7s4nwcZ9L6rnhooU-q9BYrSXZ3DbZGaBzCDU2z1VLHjGyNpratBacnjUV2XxT47e18eGF3iSM7qnqS04svMJlMy6RhPYGdAXPMyvxVD18H3fLBwfHYjqjZem7Pf7M6TngiCHR7~uTJcgWPiq6TuIiTjJ4MNqStGZ3POdz6HjJ8~zXM5LMvUGBmOlNIDWGoARAgadE3tpovkCo1lhoNvSmFpWFfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Best fitting model of transcription initiation (with ON/OFF mechanism and reversible close complex formation). The model parameters are specified in black and estimated durations of the transcription initiation steps for 1× LB media are shown in grey." path-from-xml="dsw00905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198471/dsw00905.tiff" path-from-xml="dsw00905.tiff" target="_blank">View large</a><a section="87198471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00905.png?Expires=2147483647&Signature=0sQVS1psoJoIWw1OjTZpZa8Cu7XcxyCyU6sBzqCU9fHYB9TAjOd0~oUCJ8fMXb2RmeRjx7I2pVQsoufDnt5FqqrYI4LJHVvNTRE5ESqO0FoSJr63sJeocCOydgB~nJLpBkgdSVd3CNH6AHuBZW3nxjti9Ai2EgMyDdVKc~4AwTqDrKuhaE~aoE3vdo~zOCx7iLSJrg30tKEDS-9Noypd1XLoDg344xM6gLBzQRgeUIAMTYxRcHAsZUctTcAuUM7hMAEefZbFfe3vZzVlrtAvsdrjVrt5QpUhIl4GdtisVM0BCvuxQS024FuEtBSmANzNZAdfLY2hRs54uPwtUAjRkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198471&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Best fitting model of transcription initiation (with ON/OFF mechanism and reversible close complex formation). The model parameters are specified in black and estimated durations of the transcription initiation steps for 1× LB media are shown in grey.</p></div></div><div content-id="DSW009F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/m_dsw00905.png?Expires=2147483647&Signature=gC5kj8mP5jSKmUfMs6SF6pFNMfLeWkwL9GVn2fSzHWENYbaM6YSOWw-6e3QqJDOrWU3qlf8rIJizd6pIlbA~WStFrxSV2VwFC0GxlEkEcFCWT6Rh5ZlNGzhrZ-w7s4nwcZ9L6rnhooU-q9BYrSXZ3DbZGaBzCDU2z1VLHjGyNpratBacnjUV2XxT47e18eGF3iSM7qnqS04svMJlMy6RhPYGdAXPMyvxVD18H3fLBwfHYjqjZem7Pf7M6TngiCHR7~uTJcgWPiq6TuIiTjJ4MNqStGZ3POdz6HjJ8~zXM5LMvUGBmOlNIDWGoARAgadE3tpovkCo1lhoNvSmFpWFfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Best fitting model of transcription initiation (with ON/OFF mechanism and reversible close complex formation). The model parameters are specified in black and estimated durations of the transcription initiation steps for 1× LB media are shown in grey." path-from-xml="dsw00905.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198471/dsw00905.tiff" path-from-xml="dsw00905.tiff" target="_blank">View large</a><a section="87198471" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw00905.png?Expires=2147483647&Signature=0sQVS1psoJoIWw1OjTZpZa8Cu7XcxyCyU6sBzqCU9fHYB9TAjOd0~oUCJ8fMXb2RmeRjx7I2pVQsoufDnt5FqqrYI4LJHVvNTRE5ESqO0FoSJr63sJeocCOydgB~nJLpBkgdSVd3CNH6AHuBZW3nxjti9Ai2EgMyDdVKc~4AwTqDrKuhaE~aoE3vdo~zOCx7iLSJrg30tKEDS-9Noypd1XLoDg344xM6gLBzQRgeUIAMTYxRcHAsZUctTcAuUM7hMAEefZbFfe3vZzVlrtAvsdrjVrt5QpUhIl4GdtisVM0BCvuxQS024FuEtBSmANzNZAdfLY2hRs54uPwtUAjRkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198471&ar=1745366&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00905.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Best fitting model of transcription initiation (with ON/OFF mechanism and reversible close complex formation). The model parameters are specified in black and estimated durations of the transcription initiation steps for 1× LB media are shown in grey.</p></div></div><p>In addition, from Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a>, we find that <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> is 193 ± 49 s. Meanwhile, the slope of the line on the <em>τ</em>-plot, here denoted <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, is 788 ± 59 <em>R</em>·s (<em>R</em> is the polymerase concentration such that <em>R</em> = 1 is the polymerase concentration in 1× media). The line given by these values is shown in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a> (solid line). As a side note, the uncertainties of these estimates exaggerate the uncertainty of the inference, since the estimates are highly correlated (correlation coefficient of −0.6). This correlation is responsible for the hyperbolic shape of the confidence bounds (grey region in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>We verified the slope of the solid line in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a> using the RT-PCR measurements presented in Fig. <a reveal-id="DSW009F3" data-open="DSW009F3" class="link link-reveal link-table xref-fig">3</a>, scaled to match the timescale of the intervals (Materials and methods). The resulting line is shown in Fig. <a reveal-id="DSW009F4" data-open="DSW009F4" class="link link-reveal link-table xref-fig">4</a> (dashed line), and is in good agreement with both the line given by our estimates of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> (solid line), and the inferred interval means (white circles).</p><p>Lastly, we note that the BIC depends on the number of samples used to calculate the likelihood. Thus, BIC values calculated assuming that each censored interval is ‘one sample’ will over-penalize models with more parameters, while removing them will under-penalize them. Both sets of ΔBIC values are presented in Table <a reveal-id="DSW009TB2" data-open="DSW009TB2" class="link link-reveal link-table xref-fig">2</a> and, in our case, both result in the same conclusion, and thus the distinction does not affect the results for P<em><sub>lac/ara-1</sub></em>. If, for another promoter, this turns out to be the case, additional measurements will be required to distinguish between the models.</p><p>Our results are in agreement with previous measurements of the kinetics of this and similar promoters. For example, a previous study reported that, under full induction in LB media (1× media here), P<em><sub>lac/ara-1</sub></em> expresses ∼4 RNA/h<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> (i.e. 1 RNA every ∼900 s), while we inferred the time between transcription events to be ∼980 s. Using the steady-state assay, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> was measured to be ∼330 s for P<em><sub>lac</sub></em><sup><a reveal-id="DSW009C71" data-open="DSW009C71" class="link link-ref link-reveal xref-bibr">71</a></sup> (with or without CRP-cAMP), while we obtained ∼193 s.</p>                            <h3 scrollto-destination=87198476 id="87198476" class="section-title" data-legacy-id=s3d>3.4. Determining the source of the intermittent inactive state for P<em><sub>lac/ara-1</sub></em></h3>
<p>We identified the presence of an ON/OFF mechanism in the dynamics of P<em><sub>lac/ara-1</sub></em>. It is worth noting that this ON/OFF phenomenon differs from the one reported in refs <a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a> and <a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a> since, first, we only observe OFF periods on the order of ∼87 s, while in ref. <a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a> the OFF periods reported for P<em><sub>lac/ara-1</sub></em>were on the order of 37 min. In addition, both here and in ref. <a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>, the promoter of interest is integrated in a single-copy plasmid, and thus the OFF periods cannot be explained by the buildup of positive supercoiling, since the plasmid is not topologically constrained.<sup><a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a></sup> We therefore hypothesized that the OFF periods observed here more likely result from the intermittent formation of a DNA loop, due to the transient binding of LacI, which exists in high concentration in DH5α-PRO (∼3,000 copies vs. ∼20 in wild type<sup><a reveal-id="DSW009C63" data-open="DSW009C63" class="link link-ref link-reveal xref-bibr">63</a></sup>).</p><p>If LacI is responsible for the ON/OFF behaviour, then reducing the concentration of IPTG should affect the ON/OFF dynamics, and not change the dynamics following the closed complex formation.<sup><a reveal-id="DSW009C29" data-open="DSW009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> To test this prediction, and demonstrate the utility of the model-fitting approach, besides considering the interval measurements in 1× in Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>, we also measured the interval distribution of P<em><sub>lac/ara-1</sub></em> using MS2d-GFP in the 1× media without induction by IPTG. From 130 cells, we extracted 57 intervals and 117 censored intervals between transcription events. From these, we inferred a mean interval of 3,374 ± 462 s, and a CV<sup>2</sup> of 1.03. This mean is significantly greater than the mean measured in the fully induced condition (1,005 ± 112 s), consistent with the much stronger repression of the promoter by LacI in this condition.</p><p>Given the wide difference in dynamics of RNA production between the induced and non-induced cases, we used the model fitting procedure to determine which steps are significantly affected by LacI. For this, we performed independent fits of a reduced model of initiation to the induced and the non-induced conditions. This model is observationally equivalent to the full model of initiation (Reactions (1) and (2)) for a single value of R, and is presented in Reaction (3). This reduced model is necessary since we do not have measurements of the uninduced case at multiple values of <em>R</em> with which to fit all parameters of the full model. The reduced model's parameters are denoted by <em>λ<sub>x</sub></em>, which are related to, but are not equal to the values of <em>k<sub>x</sub></em>. Their relationship is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. The fitting results are shown in Table <a reveal-id="DSW009TB3" data-open="DSW009TB3" class="link link-reveal link-table xref-fig">3</a> (labelled ‘Independent’). We also considered joint models where parameters were fixed between conditions, and used the BIC to select the most likely model.</p><p>The first three models with joint parameters test for whether or not the parameters controlling the ON/OFF mechanism change with induction strength. Consistent with this hypothesis, the models with joint <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> are strongly rejected (ΔBIC much higher than that of the Independent model). Surprisingly, the model with only joint <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> was also rejected, implying that the mean OFF times might also vary with induction strength. Additional studies are needed to elucidate why such OFF times depend on the induction strength.</p><p>Having established that <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> differ between conditions, we next assessed whether only these parameters differ. For that, we fixed <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, and verified that this model is the most parsimonious model (ΔBIC relative to the Independent model of −14.3). We conclude that only <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">OFF</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> differ between conditions, confirming the prediction that LacI is responsible for the ON/OFF mechanism affecting the RNA production dynamics.</p><p>Finally, other models were considered, e.g. the hypothesis that <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>1</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mn>2</mn><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, and/or <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>λ</mi><mrow><mrow><mi mathvariant="normal">ON</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> do not differ between conditions. These models were also strongly rejected in favour of the parsimonious model, and are not shown for brevity.</p>                            <h3 scrollto-destination=87198483 id="87198483" class="section-title" data-legacy-id=s3e>3.5. Precision of the estimates</h3>
<p>We define the precision of the estimates of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>τ</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>k</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span> as the ratio between the timescale of the intervals (i.e. the mean interval in the condition with greatest <em>R</em>) and the standard uncertainties of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>, respectively. Specifically, the precision of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span>'s estimate is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>P</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow><mo xmlns="">=</mo><mrow xmlns=""><mrow><mrow><msub><mrow><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mn>1</mn></msub></mrow></mrow><mrow><mo>/</mo></mrow><mrow><mi>σ</mi><mo stretchy="false">(</mo><mrow><mrow><msub><mrow><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></math>⁠</span>, and the precision of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math></span>'s estimate is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow></msub></mrow><mo xmlns="">=</mo><mrow xmlns=""><mrow><mrow><msub><mrow><mrow><mover><mi>I</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mn>1</mn></msub></mrow></mrow><mrow><mo>/</mo></mrow><mrow><mi>σ</mi><mo stretchy="false">(</mo><mrow><msubsup><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math>⁠</span>. Given this, here, with the volume of data in Table <a reveal-id="DSW009TB1" data-open="DSW009TB1" class="link link-reveal link-table xref-fig">1</a>, we achieved <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>P</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow><mo xmlns="">=</mo><mtext xmlns="">2</mtext><mn xmlns="">0.7</mn></math></span> and <em>P</em><sub>CC</sub> = 17.0, corresponding to errors of ∼5 and ∼6%, respectively.</p><p>In addition, we found that this precision is highly dependent on the dynamic range of RNAp concentrations. For example, for a small dynamic range of 1.5 (our measurements in Fig. <a reveal-id="DSW009F2" data-open="DSW009F2" class="link link-reveal link-table xref-fig">2</a> have a range of ∼2.4), the precisions <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>P</mi><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math></span> (in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover><mi>τ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mover><mrow><mi mathvariant="normal">CC</mi></mrow><mo accent="false">¯</mo></mover></mrow></msub></mrow></math>⁠</span>) and <em>P</em><sub>CC</sub>(in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mover><mi>k</mi><mo stretchy="false">ˆ</mo></mover></mrow><mrow><mrow><mi mathvariant="normal">CC</mi></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></math>⁠</span>) would have been reduced to ∼11.2 and ∼6.7, respectively. Losses in precision due to reduced dynamic ranges can, however, to some extent, be offset by collecting more samples for the interval distributions (see estimation of precision in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>).</p>                            <h2 scrollto-destination=87198486 id="87198486" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We established that, in <em>E. coli</em>, the concentration of free RNA polymerases differs significantly within a certain range of growth conditions, and that the inverse of the target RNA production rate under the control of P<em><sub>lac/ara-1</sub></em> varies linearly with the inverse of the free RNAp concentration (which are the conditions imposed in the <em>in vitro</em> measurements the open complex formation by steady state assays<sup><a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW009C24" data-open="DSW009C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSW009C72" data-open="DSW009C72" class="link link-ref link-reveal xref-bibr">72</a></sup>). Thus, we were able to apply a standard model-fitting procedure to fully characterize the <em>in vivo</em> kinetics of the rate-limiting steps in transcription initiation of the P<em><sub>lac/ara-1</sub></em> promoter from distributions of intervals between transcription events in cells with different RNA polymerase concentrations. This revealed that this promoter has two rate-limiting steps: a reversible closed complex formation and a significant open complex formation. Further, it also intermittently switches to a short-lived inactive state. Based on the inferred timescale of this inactive state, we predicted that this state is the result of the intermittent binding of the repressor LacI, which we verified by measuring the interval distribution when the promoter is not induced by IPTG. We believe that the complexity of this process is the reason why it has not been reported before. Namely, previous studies only considered either multiple rate-limiting steps,<sup><a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C5" data-open="DSW009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW009C22" data-open="DSW009C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSW009C23" data-open="DSW009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSW009C66" data-open="DSW009C66" class="link link-ref link-reveal xref-bibr">66</a></sup> or an ON/OFF process,<sup><a reveal-id="DSW009C2" data-open="DSW009C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW009C17" data-open="DSW009C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSW009C19" data-open="DSW009C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSW009C73" data-open="DSW009C73" class="link link-ref link-reveal xref-bibr">73</a>,<a reveal-id="DSW009C74" data-open="DSW009C74" class="link link-ref link-reveal xref-bibr">74</a></sup> while this promoter exhibits both.</p><p>We note that, provided that the promoter has a reversible closed complex formation, the model fitting procedure proposed here allows the duration and order of two steps following the closed complex to be obtained (specifically, the ratio between <em>k</em><sub>2</sub> and <em>k</em><sub>3</sub> can be determined from how the CV<sup>2</sup> of the interval distribution changes with R; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Here, this additional step was not found. However, we expect that, for other promoters, or in different conditions (e.g. low temperatures<sup><a reveal-id="DSW009C72" data-open="DSW009C72" class="link link-ref link-reveal xref-bibr">72</a></sup>), this step may be significant. Meanwhile, if Limiting Mechanism II is found to be the best-fitting model, the order of the last two steps will remain ambiguous due to the lack of reversibility.</p><p>Finally, it is worth noting that in previous works, we have not found evidence for an ON/OFF mechanism for P<em><sub>lac/ara-1</sub></em>, due to the low levels of noise detected in the time intervals between transcription events.<sup><a reveal-id="DSW009C4" data-open="DSW009C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW009C21" data-open="DSW009C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSW009C66" data-open="DSW009C66" class="link link-ref link-reveal xref-bibr">66</a></sup> This can be explained by, first, we did not consider censored intervals, which contribute significantly to the increase of the tail of the distribution of intervals.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Second, the OFF period is quite short, and thus its detection requires a large volume of data and a sensitive inference methodology.<sup><a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Our results show that, by solving these two issues (by applying the methods in refs <a reveal-id="DSW009C41" data-open="DSW009C41" class="link link-ref link-reveal xref-bibr">41</a> and <a reveal-id="DSW009C25" data-open="DSW009C25" class="link link-ref link-reveal xref-bibr">25</a>), our methodology can identify and characterize many relevant steps in transcription initiation, including those with lesser influence.</p><p>In the future, it would be of interest to extend the model to consider what occurs when more than one copy of a promoter is present in the cell. We expect that variations in the promoter copy numbers would, in that case, explain some of the variance of the data, instead of this variance being solely determined by the ON/OFF mechanism and the sequential steps.</p><p>We expect the methodology employed here to be applicable to promoters, native or synthetic, whose changes in the inverse of the transcription rate are linear with the inverse of the free RNAp concentrations. Also, it should be applicable to promoters evolved to interact with multiple transcription factors (TF), provided their fast binding and unbinding (compared with competing events), as they could be accounted for by tuning the rate constants of some of the reactions of the model. Further, multiple slow TFs, including activators, can be accounted for by adding appropriate TF-bound states, with differing production rates, in a similar manner to the ON/OFF model. As such, the methodology should be applicable at a genome wide scale. It should also be applicable to eukaryotes, provided suitable means to alter polymerase concentrations. Lastly, it should be useful in detecting differences in transcription initiation kinetics of a promoter subject to different intra- or extra-cellular conditions.</p>                            <h2 scrollto-destination=87198492 id="87198492" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Academy of Finland (257603 to A.S.R.); Centre of International Mobility (13.1.2014/TM-14-91361/CIMO to A.S.R.); Jenny and Antti Wihuri Foundation (to A.H.); and the Tampere University of Technology President's Graduate Programme (to J.L.-P. and S.S.). Funding to pay the Open Access publication charges for this article was provided by Academy of Finland (257603 to A.S.R.).</p>                            <h2 scrollto-destination=87198494 id="87198494" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Axel Oikari, Abhishekh Gupta, and Antti Martikainen for valuable advice.</p>                            <h2 scrollto-destination=87198496 id="87198496" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Golding</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">RNA dynamics in live <em>Escherichia coli</em> cells</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">101</div>, <div class="fpage">11310</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20dynamics%20in%20live%20Escherichia%20coli%20cells&amp;author=I.%20Golding&amp;author=E.C.%20Cox&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=11310-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404443101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404443101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404443101"> </span></div> </div></div></div></div></div><div content-id="DSW009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Golding</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Paulsson</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zawilski</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Real-time kinetics of gene activity in individual bacteria</div>, <div class="source">Cell</div> , <div class="volume">123</div>, <div class="fpage">1025</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Real-time%20kinetics%20of%20gene%20activity%20in%20individual%20bacteria&amp;author=I.%20Golding&amp;author=J.%20Paulsson&amp;author=S.M.%20Zawilski&amp;author=E.C.%20Cox&amp;publication_year=2005&amp;journal=Cell&amp;volume=123&amp;pages=1025-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.09.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.09.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.09.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16360033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ren</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lao</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">X.S.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Probing gene expression in live cells, one protein molecule at a time</div>, <div class="source">Science</div> , <div class="volume">311</div>, <div class="fpage">1600</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probing%20gene%20expression%20in%20live%20cells%2C%20one%20protein%20molecule%20at%20a%20time&amp;author=J.%20Yu&amp;author=J.%20Xiao&amp;author=X.%20Ren&amp;author=K.%20Lao&amp;author=X.S.%20Xie&amp;publication_year=2006&amp;journal=Science&amp;volume=311&amp;pages=1600-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1119623" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1119623" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1119623"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16543458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kandhavelu</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Mannerström</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title"><em>In vivo</em> kinetics of transcription initiation of the <em>lar</em> promoter in <em>Escherichia coli</em>. Evidence for a sequential mechanism with two rate-limiting steps</div>, <div class="source">BMC Syst. Biol.</div> , <div class="volume">5</div>, <div class="fpage">149</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20kinetics%20of%20transcription%20initiation%20of%20the%20lar%20promoter%20in%20Escherichia%20coli.%20Evidence%20for%20a%20sequential%20mechanism%20with%20two%20rate-limiting%20steps&amp;author=M.%20Kandhavelu&amp;author=H.%20Mannerstr%C3%B6m&amp;author=A.%20Gupta&amp;publication_year=2011&amp;journal=BMC%20Syst.%20Biol.&amp;volume=5&amp;pages=149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1752-0509-5-149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1752-0509-5-149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1752-0509-5-149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21943372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Muthukrishnan</div>
               <div class="given-names">A.-B.</div>
            </div>, <div class="name string-name"><div class="surname">Kandhavelu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lloyd-Price</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Dynamics of transcription driven by the <em>tetA</em> promoter, one event at a time, in live <em>Escherichia coli</em> cells</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">8472</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20of%20transcription%20driven%20by%20the%20tetA%20promoter%2C%20one%20event%20at%20a%20time%2C%20in%20live%20Escherichia%20coli%20cells&amp;author=A.-B.%20Muthukrishnan&amp;author=M.%20Kandhavelu&amp;author=J.%20Lloyd-Price&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=8472-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks583" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks583"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22730294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fusco</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Accornero</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Lavoie</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Single mRNA Molecules Demonstrate Probabilistic Movement in Living Mammalian Cells</div>, <div class="source">Curr. Biol.</div> , <div class="volume">13</div>, <div class="fpage">161</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20mRNA%20Molecules%20Demonstrate%20Probabilistic%20Movement%20in%20Living%20Mammalian%20Cells&amp;author=D.%20Fusco&amp;author=N.%20Accornero&amp;author=B.%20Lavoie&amp;publication_year=2003&amp;journal=Curr.%20Biol.&amp;volume=13&amp;pages=161-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)01436-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)01436-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)01436-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12546792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Raj</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Peskin</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Tranchina</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Vargas</div> <div class="given-names">D.Y.</div></div>, <div class="name string-name"><div class="surname">Tyagi</div> <div class="given-names">S.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Stochastic mRNA synthesis in mammalian cells</div>, <div class="source">PLoS Biol.</div> , <div class="volume">4</div>, <div class="fpage">1707</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20mRNA%20synthesis%20in%20mammalian%20cells&amp;author=A.%20Raj&amp;author=C.S.%20Peskin&amp;author=D.%20Tranchina&amp;author=D.Y.%20Vargas&amp;author=S.%20Tyagi&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=1707-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040309"> </span></div> </div></div></div></div></div><div content-id="DSW009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kaern</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Elston</div> <div class="given-names">T.C.</div></div>, <div class="name string-name"><div class="surname">Blake</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Collins</div> <div class="given-names">J.J.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Stochasticity in gene expression: from theories to phenotypes</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">6</div>, <div class="fpage">451</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochasticity%20in%20gene%20expression%3A%20from%20theories%20to%20phenotypes&amp;author=M.%20Kaern&amp;author=T.C.%20Elston&amp;author=W.J.%20Blake&amp;author=J.J.%20Collins&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=451-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15883588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arkin</div>
               <div class="given-names">A.P.</div>
            </div>, <div class="name string-name"><div class="surname">Ross</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">McAdams</div> <div class="given-names">H.H.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Stochastic kinetic analysis of developmental pathway bifurcation in phage λ-infected <em>Escherichia coli</em> cells</div>, <div class="source">Genetics</div> , <div class="volume">149</div>, <div class="fpage">1633</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20kinetic%20analysis%20of%20developmental%20pathway%20bifurcation%20in%20phage%20%CE%BB-infected%20Escherichia%20coli%20cells&amp;author=A.P.%20Arkin&amp;author=J.%20Ross&amp;author=H.H.%20McAdams&amp;publication_year=1998&amp;journal=Genetics&amp;volume=149&amp;pages=1633-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9691025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Elowitz</div>
               <div class="given-names">M.B.</div>
            </div>, <div class="name string-name"><div class="surname">Levine</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Siggia</div> <div class="given-names">E.D.</div></div>, <div class="name string-name"><div class="surname">Swain</div> <div class="given-names">P.S.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Stochastic gene expression in a single cell</div>, <div class="source">Science</div> , <div class="volume">297</div>, <div class="fpage">1183</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20gene%20expression%20in%20a%20single%20cell&amp;author=M.B.%20Elowitz&amp;author=A.J.%20Levine&amp;author=E.D.%20Siggia&amp;author=P.S.%20Swain&amp;publication_year=2002&amp;journal=Science&amp;volume=297&amp;pages=1183-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1070919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1070919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1070919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12183631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Raser</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">O'Shea</div> <div class="given-names">E.K.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Noise in gene expression: origins, consequences, and control</div>, <div class="source">Science</div> , <div class="volume">309</div>, <div class="fpage">2010</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noise%20in%20gene%20expression%3A%20origins%2C%20consequences%2C%20and%20control&amp;author=J.M.%20Raser&amp;author=E.K.%20O'Shea&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=2010-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1105891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1105891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1105891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16179466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ozbudak</div>
               <div class="given-names">E.M.</div>
            </div>, <div class="name string-name"><div class="surname">Thattai</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kurtser</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Grossman</div> <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">van Oudenaarden</div> <div class="given-names">A.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Regulation of noise in the expression of a single gene</div>, <div class="source">Nat. Genet.</div> , <div class="volume">31</div>, <div class="fpage">69</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20noise%20in%20the%20expression%20of%20a%20single%20gene&amp;author=E.M.%20Ozbudak&amp;author=M.%20Thattai&amp;author=I.%20Kurtser&amp;author=A.D.%20Grossman&amp;author=A.%20van%20Oudenaarden&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=31&amp;pages=69-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng869" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng869"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11967532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McAdams</div>
               <div class="given-names">H.H.</div>
            </div>, <div class="name string-name"><div class="surname">Arkin</div> <div class="given-names">A.P.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">It's a noisy business! Genetic regulation at the nanomolar scale</div>, <div class="source">Trends Genet.</div> , <div class="volume">15</div>, <div class="fpage">65</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=It's%20a%20noisy%20business!%20Genetic%20regulation%20at%20the%20nanomolar%20scale&amp;author=H.H.%20McAdams&amp;author=A.P.%20Arkin&amp;publication_year=1999&amp;journal=Trends%20Genet.&amp;volume=15&amp;pages=65-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(98)01659-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(98)01659-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(98)01659-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10098409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Süel</div>
               <div class="given-names">G.M.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia-Ojalvo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liberman</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Elowitz</div> <div class="given-names">M.B.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">An excitable gene regulatory circuit induces transient cellular differentiation</div>, <div class="source">Nature</div> , <div class="volume">440</div>, <div class="fpage">545</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20excitable%20gene%20regulatory%20circuit%20induces%20transient%20cellular%20differentiation&amp;author=G.M.%20S%C3%BCel&amp;author=J.%20Garcia-Ojalvo&amp;author=L.M.%20Liberman&amp;author=M.B.%20Elowitz&amp;publication_year=2006&amp;journal=Nature&amp;volume=440&amp;pages=545-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04588" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04588" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04588"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16554821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cai</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Friedman</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">X.S.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Stochastic protein expression in individual cells at the single molecule level</div>, <div class="source">Nature</div> , <div class="volume">440</div>, <div class="fpage">358</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20protein%20expression%20in%20individual%20cells%20at%20the%20single%20molecule%20level&amp;author=L.%20Cai&amp;author=N.%20Friedman&amp;author=X.S.%20Xie&amp;publication_year=2006&amp;journal=Nature&amp;volume=440&amp;pages=358-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04599"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16541077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mitarai</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Dodd</div> <div class="given-names">I.B.</div></div>, <div class="name string-name"><div class="surname">Crooks</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Sneppen</div> <div class="given-names">K.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">The generation of promoter-mediated transcriptional noise in bacteria</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">4</div>, <div class="fpage">e1000109</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20generation%20of%20promoter-mediated%20transcriptional%20noise%20in%20bacteria&amp;author=N.%20Mitarai&amp;author=I.B.%20Dodd&amp;author=M.T.%20Crooks&amp;author=K.%20Sneppen&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18617999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">So</div>
               <div class="given-names">L.-H.</div>
            </div>, <div class="name string-name"><div class="surname">Ghosh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zong</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sepúlveda</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Segev</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Golding</div> <div class="given-names">I.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">General properties of transcriptional time series in <em>Escherichia coli</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">554</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20properties%20of%20transcriptional%20time%20series%20in%20Escherichia%20coli&amp;author=L.-H.%20So&amp;author=A.%20Ghosh&amp;author=C.%20Zong&amp;author=L.A.%20Sep%C3%BAlveda&amp;author=R.%20Segev&amp;author=I.%20Golding&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=554-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21532574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhdanov</div>
               <div class="given-names">V.P.</div>
            </div>
          </div>
          <div class="year">2011</div>, <div class="article-title">Kinetic models of gene expression including non-coding RNAs</div>, <div class="source">Phys. Rep.</div> , <div class="volume">500</div>, <div class="fpage">1</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kinetic%20models%20of%20gene%20expression%20including%20non-coding%20RNAs&amp;author=V.P.%20Zhdanov&amp;publication_year=2011&amp;journal=Phys.%20Rep.&amp;volume=500&amp;pages=1-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.physrep.2010.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.physrep.2010.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.physrep.2010.12.002"> </span></div> </div></div></div></div></div><div content-id="DSW009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chong</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ge</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">X.S.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Mechanism of transcriptional bursting in bacteria</div>, <div class="source">Cell</div> , <div class="volume">158</div>, <div class="fpage">314</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20transcriptional%20bursting%20in%20bacteria&amp;author=S.%20Chong&amp;author=C.%20Chen&amp;author=H.%20Ge&amp;author=X.S.%20Xie&amp;publication_year=2014&amp;journal=Cell&amp;volume=158&amp;pages=314-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.05.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.05.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.05.038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25036631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kandhavelu</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Häkkinen</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Single-molecule dynamics of transcription of the <em>lar</em> promoter</div>, <div class="source">Phys. Biol.</div> , <div class="volume">9</div>, <div class="comment">026004</div>.</div></div></div></div><div content-id="DSW009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kandhavelu</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Lloyd-Price</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Muthukrishnan</div> <div class="given-names">A.-B.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Regulation of mean and noise of the <em>in vivo</em> kinetics of transcription under the control of the <em>lac/ara</em>-1 promoter</div>, <div class="source">FEBS Lett.</div> , <div class="volume">586</div>, <div class="fpage">3870</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20mean%20and%20noise%20of%20the%20in%20vivo%20kinetics%20of%20transcription%20under%20the%20control%20of%20the%20lac%2Fara-1%20promoter&amp;author=M.%20Kandhavelu&amp;author=J.%20Lloyd-Price&amp;author=A.%20Gupta&amp;author=A.-B.%20Muthukrishnan&amp;author=O.%20Yli-Harja&amp;author=A.S.%20Ribeiro&amp;publication_year=2012&amp;journal=FEBS%20Lett.&amp;volume=586&amp;pages=3870-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2012.09.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2012.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2012.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23017207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McClure</div>
               <div class="given-names">W.R.</div>
            </div>
          </div>
          <div class="year">1980</div>, <div class="article-title">Rate-limiting steps in RNA chain initiation</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">77</div>, <div class="fpage">5634</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rate-limiting%20steps%20in%20RNA%20chain%20initiation&amp;author=W.R.%20McClure&amp;publication_year=1980&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=77&amp;pages=5634-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.77.10.5634" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.77.10.5634" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.77.10.5634"> </span></div> </div></div></div></div></div><div content-id="DSW009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McClure</div>
               <div class="given-names">W.R.</div>
            </div>
          </div>
          <div class="year">1985</div>, <div class="article-title">Mechanism and control of transcription initiation in prokaryotes</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="volume">54</div>, <div class="fpage">171</div>–<div class="lpage">204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20and%20control%20of%20transcription%20initiation%20in%20prokaryotes&amp;author=W.R.%20McClure&amp;publication_year=1985&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=54&amp;pages=171-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.54.070185.001131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.54.070185.001131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.54.070185.001131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3896120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bertrand-Burggraf</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Lefèvre</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Daune</div> <div class="given-names">M.</div></div></div>
          <div class="year">1984</div>, <div class="article-title">A new experimental approach for studying the association between RNA polymeras and the <em>tet</em> promoter of pBR322</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">12</div>, <div class="fpage">1697</div>–<div class="lpage">706</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20experimental%20approach%20for%20studying%20the%20association%20between%20RNA%20polymeras%20and%20the%20tet%20promoter%20of%20pBR322&amp;author=E.%20Bertrand-Burggraf&amp;author=J.F.%20Lef%C3%A8vre&amp;author=M.%20Daune&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=1697-706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.3.1697" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.3.1697" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.3.1697"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6366739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Häkkinen</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2016</div>, <div class="article-title">Characterizing rate limiting steps in transcription from RNA production times in live cells</div>, <div class="source">Bioinformatics</div> , <div class="comment"><a class="link link-uri" href="http://bioinformatics.oxfordjournals.org/content/early/2016/01/28/bioinformatics.btv744.abstract">http://bioinformatics.oxfordjournals.org/content/early/2016/01/28/bioinformatics.btv744.abstract</a></div>.</div></div></div></div><div content-id="DSW009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Friedman</div>
               <div class="given-names">L.J.</div>
            </div>, <div class="name string-name"><div class="surname">Gelles</div> <div class="given-names">J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Mechanism of transcription initiation at an activator-dependent promoter defined by single-molecule observation</div>, <div class="source">Cell</div> , <div class="volume">148</div>, <div class="fpage">679</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20transcription%20initiation%20at%20an%20activator-dependent%20promoter%20defined%20by%20single-molecule%20observation&amp;author=L.J.%20Friedman&amp;author=J.%20Gelles&amp;publication_year=2012&amp;journal=Cell&amp;volume=148&amp;pages=679-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.01.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.01.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.01.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22341441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Taniguchi</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.-W.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Quantifying <em>E. coli</em> proteome and transcriptome with single-molecule sensitivity in single cells</div>, <div class="source">Science</div> , <div class="volume">329</div>, <div class="fpage">533</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20E.%20coli%20proteome%20and%20transcriptome%20with%20single-molecule%20sensitivity%20in%20single%20cells&amp;author=Y.%20Taniguchi&amp;author=P.J.%20Choi&amp;author=G.-W.%20Li&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=533-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1188308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1188308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1188308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20671182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sanchez</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">H.G.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kondev</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Effect of promoter architecture on the cell-to-cell variability in gene expression</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">7</div>, <div class="fpage">e1001100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20promoter%20architecture%20on%20the%20cell-to-cell%20variability%20in%20gene%20expression&amp;author=A.%20Sanchez&amp;author=H.G.%20Garcia&amp;author=D.%20Jones&amp;author=R.%20Phillips&amp;author=J.%20Kondev&amp;publication_year=2011&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=7&amp;pages=e1001100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1001100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1001100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1001100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21390269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lutz</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Lozinski</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ellinger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bujard</div> <div class="given-names">H.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Dissecting the functional program of <em>Escherichia coli</em> promoters: the combined mode of action of Lac repressor and AraC activator</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">29</div>, <div class="fpage">3873</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20the%20functional%20program%20of%20Escherichia%20coli%20promoters%3A%20the%20combined%20mode%20of%20action%20of%20Lac%20repressor%20and%20AraC%20activator&amp;author=R.%20Lutz&amp;author=T.%20Lozinski&amp;author=T.%20Ellinger&amp;author=H.%20Bujard&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=3873-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.18.3873" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.18.3873" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.18.3873"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11557820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Garcia</div>
               <div class="given-names">H.G.</div>
            </div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Boedicker</div> <div class="given-names">J.Q.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Operator sequence alters gene expression independently of transcription factor occupancy in bacteria</div>, <div class="source">Cell Rep.</div> , <div class="volume">2</div>, <div class="fpage">150</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Operator%20sequence%20alters%20gene%20expression%20independently%20of%20transcription%20factor%20occupancy%20in%20bacteria&amp;author=H.G.%20Garcia&amp;author=A.%20Sanchez&amp;author=J.Q.%20Boedicker&amp;publication_year=2012&amp;journal=Cell%20Rep.&amp;volume=2&amp;pages=150-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2012.06.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2012.06.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2012.06.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22840405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gummesson</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Magnusson</div> <div class="given-names">L.U.</div></div>, <div class="name string-name"><div class="surname">Lovmar</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Increased RNA polymerase availability directs resources towards growth at the expense of maintenance</div>, <div class="source">EMBO J.</div> , <div class="volume">28</div>, <div class="fpage">2209</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increased%20RNA%20polymerase%20availability%20directs%20resources%20towards%20growth%20at%20the%20expense%20of%20maintenance&amp;author=B.%20Gummesson&amp;author=L.U.%20Magnusson&amp;author=M.%20Lovmar&amp;publication_year=2009&amp;journal=EMBO%20J.&amp;volume=28&amp;pages=2209-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/emboj.2009.181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Femboj.2009.181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Femboj.2009.181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19574956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bremer</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">P.P.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Modulation of Chemical Composition and Other Parameters of the Cell by Growth Rate</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Neidhardt</div> <div class="given-names">F.C.</div></div></div>, (ed.), <div class="source">Escherichia Coli and Salmonella</div> , <div class="edition">2nd ed</div>. <div class="publisher-name">ASM Press</div>, <div class="publisher-loc">Washington, DC</div>, pp. <div class="fpage">1553</div>–<div class="lpage">69</div>.</div></div></div></div><div content-id="DSW009C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bratton</div>
               <div class="given-names">B.P.</div>
            </div>, <div class="name string-name"><div class="surname">Mooney</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">J.C.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Spatial distribution and diffusive motion of RNA polymerase in live <em>Escherichia coli</em>.</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">193</div>, <div class="fpage">5138</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatial%20distribution%20and%20diffusive%20motion%20of%20RNA%20polymerase%20in%20live%20Escherichia%20coli.&amp;author=B.P.%20Bratton&amp;author=R.A.%20Mooney&amp;author=J.C.%20Weisshaar&amp;publication_year=2011&amp;journal=J.%20Bacteriol.&amp;volume=193&amp;pages=5138-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00198-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00198-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00198-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21784927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dillon</div>
               <div class="given-names">S.C.</div>
            </div>, <div class="name string-name"><div class="surname">Dorman</div> <div class="given-names">C.J.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="volume">8</div>, <div class="fpage">185</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20nucleoid-associated%20proteins%2C%20nucleoid%20structure%20and%20gene%20expression&amp;author=S.C.%20Dillon&amp;author=C.J.%20Dorman&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=8&amp;pages=185-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liang</div>
               <div class="given-names">S.-T.</div>
            </div>, <div class="name string-name"><div class="surname">Bipatnath</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.-C.</div></div>et al.  </div>. <div class="year">1999</div>, <div class="article-title">Activities of constitutive promoters in <em>Escherichia coli</em>.</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">292</div>, <div class="fpage">19</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activities%20of%20constitutive%20promoters%20in%20Escherichia%20coli.&amp;author=S.-T.%20Liang&amp;author=M.%20Bipatnath&amp;author=Y.-C.%20Xu&amp;publication_year=1999&amp;journal=J.%20Mol.%20Biol.&amp;volume=292&amp;pages=19-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1999.3056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1999.3056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1999.3056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10493854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Livak</div>
               <div class="given-names">K.J.</div>
            </div>, <div class="name string-name"><div class="surname">Schmittgen</div> <div class="given-names">T.D.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2<sup>-ΔΔCt</sup> Method</div>, <div class="source">Methods</div> , <div class="volume">25</div>, <div class="fpage">402</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202-%CE%94%CE%94Ct%20Method&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen&amp;publication_year=2001&amp;journal=Methods&amp;volume=25&amp;pages=402-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmeth.2001.1262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmeth.2001.1262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gupta</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Lloyd-Price</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Oliveira</div> <div class="given-names">S.M.D.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Muthukrishnan</div> <div class="given-names">A.-B.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Robustness of the division symmetry in <em>Escherichia coli</em> and functional consequences of symmetry breaking</div>, <div class="source">Phys. Biol.</div> , <div class="volume">11</div>, <div class="fpage">066005</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robustness%20of%20the%20division%20symmetry%20in%20Escherichia%20coli%20and%20functional%20consequences%20of%20symmetry%20breaking&amp;author=A.%20Gupta&amp;author=J.%20Lloyd-Price&amp;author=S.M.D.%20Oliveira&amp;author=O.%20Yli-Harja&amp;author=A.-B.%20Muthukrishnan&amp;author=A.S.%20Ribeiro&amp;publication_year=2014&amp;journal=Phys.%20Biol.&amp;volume=11&amp;pages=066005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1088/1478-3975/11/6/066005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1088%2F1478-3975%2F11%2F6%2F066005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1088%2F1478-3975%2F11%2F6%2F066005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25382420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chowdhury</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Kandhavelu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Cell segmentation by multi-resolution analysis and maximum likelihood estimation (MAMLE)</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">S8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20segmentation%20by%20multi-resolution%20analysis%20and%20maximum%20likelihood%20estimation%20(MAMLE)&amp;author=S.%20Chowdhury&amp;author=M.%20Kandhavelu&amp;author=O.%20Yli-Harja&amp;author=A.S.%20Ribeiro&amp;publication_year=2013&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=S8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-14-196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-14-196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24267594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Häkkinen</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Muthukrishnan</div> <div class="given-names">A.-B.</div></div>, <div class="name string-name"><div class="surname">Mora</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fonseca</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">CellAging: A tool to study segregation and partitioning in division in cell lineages of <em>Escherichia coli</em></div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">1708</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CellAging%3A%20A%20tool%20to%20study%20segregation%20and%20partitioning%20in%20division%20in%20cell%20lineages%20of%20Escherichia%20coli&amp;author=A.%20H%C3%A4kkinen&amp;author=A.-B.%20Muthukrishnan&amp;author=A.%20Mora&amp;author=J.M.%20Fonseca&amp;author=A.S.%20Ribeiro&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=1708-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt194"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23613488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Häkkinen</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Kandhavelu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Garasto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Estimation of fluorescence-tagged RNA numbers from spot intensities</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1146</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20fluorescence-tagged%20RNA%20numbers%20from%20spot%20intensities&amp;author=A.%20H%C3%A4kkinen&amp;author=M.%20Kandhavelu&amp;author=S.%20Garasto&amp;author=A.S.%20Ribeiro&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1146-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt766"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24389660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Häkkinen</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Estimation of GFP-tagged RNA numbers from temporal fluorescence intensity data</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">69</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20GFP-tagged%20RNA%20numbers%20from%20temporal%20fluorescence%20intensity%20data&amp;author=A.%20H%C3%A4kkinen&amp;author=A.S.%20Ribeiro&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=69-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25189780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tran</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Oliveira</div> <div class="given-names">S.M.D.</div></div>, <div class="name string-name"><div class="surname">Goncalves</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Kinetics of the cellular intake of a gene expression inducer at high concentrations</div>, <div class="source">Mol. Biosyst.</div> , <div class="volume">11</div>, <div class="fpage">2579</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kinetics%20of%20the%20cellular%20intake%20of%20a%20gene%20expression%20inducer%20at%20high%20concentrations&amp;author=H.%20Tran&amp;author=S.M.D.%20Oliveira&amp;author=N.%20Goncalves&amp;author=A.S.%20Ribeiro&amp;publication_year=2015&amp;journal=Mol.%20Biosyst.&amp;volume=11&amp;pages=2579-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/C5MB00244C" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2FC5MB00244C" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2FC5MB00244C"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26223179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Johansson</div>
               <div class="given-names">H.E.</div>
            </div>, <div class="name string-name"><div class="surname">Dertinger</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">LeCuyer</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Behlen</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Greef</div> <div class="given-names">C.H.</div></div>, <div class="name string-name"><div class="surname">Uhlenbeck</div> <div class="given-names">O.C.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">A thermodynamic analysis of the sequence-specific binding of RNA by bacteriophage MS2 coat protein</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">95</div>, <div class="fpage">9244</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20thermodynamic%20analysis%20of%20the%20sequence-specific%20binding%20of%20RNA%20by%20bacteriophage%20MS2%20coat%20protein&amp;author=H.E.%20Johansson&amp;author=D.%20Dertinger&amp;author=K.A.%20LeCuyer&amp;author=L.S.%20Behlen&amp;author=C.H.%20Greef&amp;author=O.C.%20Uhlenbeck&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=9244-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9244"> </span></div> </div></div></div></div></div><div content-id="DSW009C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Golding</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Physical Nature of Bacterial Cytoplasm</div>, <div class="source">Phys. Rev. Lett.</div> , <div class="volume">96</div>, <div class="fpage">98102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20Nature%20of%20Bacterial%20Cytoplasm&amp;author=I.%20Golding&amp;author=E.C.%20Cox&amp;publication_year=2006&amp;journal=Phys.%20Rev.%20Lett.&amp;volume=96&amp;pages=98102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1103/PhysRevLett.96.098102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1103%2FPhysRevLett.96.098102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1103%2FPhysRevLett.96.098102"> </span></div> </div></div></div></div></div><div content-id="DSW009C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bernstein</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Khodursky</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Pei-Hsun</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lin-Chao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cohen</div> <div class="given-names">S.N.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Global analysis of mRNA decay and abundance in <em>Escherichia coli</em> at single-gene resolution using two-color fluorescent DNA microarrays</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">99</div>, <div class="fpage">9697</div>–<div class="lpage">702</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20mRNA%20decay%20and%20abundance%20in%20Escherichia%20coli%20at%20single-gene%20resolution%20using%20two-color%20fluorescent%20DNA%20microarrays&amp;author=J.A.%20Bernstein&amp;author=A.B.%20Khodursky&amp;author=L.%20Pei-Hsun&amp;author=S.%20Lin-Chao&amp;author=S.N.%20Cohen&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=9697-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.112318199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.112318199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.112318199"> </span></div> </div></div></div></div></div><div content-id="DSW009C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saecker</div>
               <div class="given-names">R.M.</div>
            </div>, <div class="name string-name"><div class="surname">Record</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Dehaseth</div> <div class="given-names">P.L.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Mechanism of bacterial transcription initiation: RNA polymerase - promoter binding, isomerization to initiation-competent open complexes, and initiation of RNA synthesis</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">412</div>, <div class="fpage">754</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20bacterial%20transcription%20initiation%3A%20RNA%20polymerase%20-%20promoter%20binding%2C%20isomerization%20to%20initiation-competent%20open%20complexes%2C%20and%20initiation%20of%20RNA%20synthesis&amp;author=R.M.%20Saecker&amp;author=M.T.%20Record&amp;author=P.L.%20Dehaseth&amp;publication_year=2011&amp;journal=J.%20Mol.%20Biol.&amp;volume=412&amp;pages=754-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2011.01.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2011.01.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2011.01.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21371479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">DeHaseth</div>
               <div class="given-names">P.L.</div>
            </div>, <div class="name string-name"><div class="surname">Zupancic</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Record</div> <div class="given-names">M.T.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">RNA polymerase-promoter interactions: The comings and goings of RNA polymerase</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">180</div>, <div class="fpage">3019</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20polymerase-promoter%20interactions%3A%20The%20comings%20and%20goings%20of%20RNA%20polymerase&amp;author=P.L.%20DeHaseth&amp;author=M.L.%20Zupancic&amp;author=M.T.%20Record&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=3019-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9620948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chamberlin</div>
               <div class="given-names">M.J.</div>
            </div>
          </div>
          <div class="year">1974</div>, <div class="article-title">The selectivity of transcription</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="volume">43</div>, <div class="fpage">721</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selectivity%20of%20transcription&amp;author=M.J.%20Chamberlin&amp;publication_year=1974&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=43&amp;pages=721-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.43.070174.003445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.43.070174.003445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.43.070174.003445"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4605337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hsu</div>
               <div class="given-names">L.M.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">Monitoring abortive initiation</div>, <div class="source">Methods</div> , <div class="volume">47</div>, <div class="fpage">25</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monitoring%20abortive%20initiation&amp;author=L.M.%20Hsu&amp;publication_year=2009&amp;journal=Methods&amp;volume=47&amp;pages=25-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2008.10.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2008.10.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2008.10.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mulligan</div>
               <div class="given-names">M.E.</div>
            </div>, <div class="name string-name"><div class="surname">Hawley</div> <div class="given-names">D.K.</div></div>, <div class="name string-name"><div class="surname">Entriken</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">McClure</div> <div class="given-names">W.R.</div></div></div>
          <div class="year">1984</div>, <div class="article-title"><em>Escherichia coli</em> promoter sequences predict in vitro RNA polymerase selectivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">12</div>, <div class="fpage">789</div>–<div class="lpage">800</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20promoter%20sequences%20predict%20in%20vitro%20RNA%20polymerase%20selectivity&amp;author=M.E.%20Mulligan&amp;author=D.K.%20Hawley&amp;author=R.%20Entriken&amp;author=W.R.%20McClure&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=789-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.1Part2.789" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.1Part2.789" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.1Part2.789"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6364042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bai</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Santangelo</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.D.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Single-molecule analysis of RNA polymerase transcription</div>, <div class="source">Annu. Rev. Biophys. Biomol. Struct.</div> , <div class="volume">35</div>, <div class="fpage">343</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-molecule%20analysis%20of%20RNA%20polymerase%20transcription&amp;author=L.%20Bai&amp;author=T.J.%20Santangelo&amp;author=M.D.%20Wang&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Biophys.%20Biomol.%20Struct.&amp;volume=35&amp;pages=343-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biophys.35.010406.150153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biophys.35.010406.150153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biophys.35.010406.150153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16689640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Greene</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Single-molecule studies of transcription: From one RNA polymerase at a time to the gene expression profile of a cell</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">412</div>, <div class="fpage">814</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-molecule%20studies%20of%20transcription%3A%20From%20one%20RNA%20polymerase%20at%20a%20time%20to%20the%20gene%20expression%20profile%20of%20a%20cell&amp;author=F.%20Wang&amp;author=E.C.%20Greene&amp;publication_year=2011&amp;journal=J.%20Mol.%20Biol.&amp;volume=412&amp;pages=814-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2011.01.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2011.01.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2011.01.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21255583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Artsimovitch</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Landick</div> <div class="given-names">R.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Pausing by bacterial RNA polymerase is mediated by mechanistically distinct classes of signals</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">97</div>, <div class="fpage">7090</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pausing%20by%20bacterial%20RNA%20polymerase%20is%20mediated%20by%20mechanistically%20distinct%20classes%20of%20signals&amp;author=I.%20Artsimovitch&amp;author=R.%20Landick&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=7090-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.13.7090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.13.7090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.13.7090"> </span></div> </div></div></div></div></div><div content-id="DSW009C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rajala</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Häkkinen</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Healy</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Effects of transcriptional pausing on gene expression dynamics</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="volume">6</div>, <div class="fpage">e1000704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20transcriptional%20pausing%20on%20gene%20expression%20dynamics&amp;author=T.%20Rajala&amp;author=A.%20H%C3%A4kkinen&amp;author=S.%20Healy&amp;author=O.%20Yli-Harja&amp;author=A.S.%20Ribeiro&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=6&amp;pages=e1000704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20300642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bar-Nahum</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Nudler</div> <div class="given-names">E.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Isolation and characterization of σ<sup>70</sup>-retaining transcription elongation complexes from <em>Escherichia coli</em></div>, <div class="source">Cell</div> , <div class="volume">106</div>, <div class="fpage">443</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20%CF%8370-retaining%20transcription%20elongation%20complexes%20from%20Escherichia%20coli&amp;author=G.%20Bar-Nahum&amp;author=E.%20Nudler&amp;publication_year=2001&amp;journal=Cell&amp;volume=106&amp;pages=443-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(01)00461-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(01)00461-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(01)00461-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11525730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grigorova</div>
               <div class="given-names">I.L.</div>
            </div>, <div class="name string-name"><div class="surname">Phleger</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Mutalik</div> <div class="given-names">V.K.</div></div>, <div class="name string-name"><div class="surname">Gross</div> <div class="given-names">C.a.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Insights into transcriptional regulation and sigma competition from an equilibrium model of RNA polymerase binding to DNA</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">5332</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20transcriptional%20regulation%20and%20sigma%20competition%20from%20an%20equilibrium%20model%20of%20RNA%20polymerase%20binding%20to%20DNA&amp;author=I.L.%20Grigorova&amp;author=N.J.%20Phleger&amp;author=V.K.%20Mutalik&amp;author=C.a.%20Gross&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=5332-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0600828103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0600828103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0600828103"> </span></div> </div></div></div></div></div><div content-id="DSW009C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maeda</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ishihama</div> <div class="given-names">A.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Competition among seven <em>Escherichia coli</em> σ subunits: relative binding affinities to the core RNA polymerase</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">28</div>, <div class="fpage">3497</div>–<div class="lpage">503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Competition%20among%20seven%20Escherichia%20coli%20%CF%83%20subunits%3A%20relative%20binding%20affinities%20to%20the%20core%20RNA%20polymerase&amp;author=H.%20Maeda&amp;author=N.%20Fujita&amp;author=A.%20Ishihama&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=3497-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.18.3497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.18.3497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.18.3497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10982868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kass</div>
               <div class="given-names">R.E.</div>
            </div>, <div class="name string-name"><div class="surname">Raftery</div> <div class="given-names">A.E.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Bayes Factors</div>, <div class="source">J. Am. Stat. Assoc.</div> , <div class="volume">90</div>, <div class="fpage">773</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayes%20Factors&amp;author=R.E.%20Kass&amp;author=A.E.%20Raftery&amp;publication_year=1995&amp;journal=J.%20Am.%20Stat.%20Assoc.&amp;volume=90&amp;pages=773-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01621459.1995.10476572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01621459.1995.10476572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01621459.1995.10476572"> </span></div> </div></div></div></div></div><div content-id="DSW009C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hsu</div>
               <div class="given-names">L.M.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">Promoter clearance and escape in prokaryotes</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">15,77</div>, <div class="fpage">191</div>–<div class="lpage">207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20clearance%20and%20escape%20in%20prokaryotes&amp;author=L.M.%20Hsu&amp;publication_year=2002&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=15,77&amp;pages=191-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-4781(02)00452-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-4781(02)00452-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-4781(02)00452-9"> </span></div> </div></div></div></div></div><div content-id="DSW009C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Casella</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Berger</div> <div class="given-names">R.L.</div></div></div>
          <div class="year">2001</div>, <div class="source">The Delta Method. Statistical Inference</div> , <div class="edition">2nd ed</div>. <div class="publisher-name">Duxbury Press</div>, <div class="publisher-loc">Pacific Grove, CA</div>, pp. <div class="fpage">240</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="DSW009C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krystek</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Anton</div> <div class="given-names">M.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">A weighted total least-squares algorithm for fitting a straight line</div>, <div class="source">Meas. Sci. Technol.</div> , <div class="volume">19</div>, <div class="fpage">79801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20weighted%20total%20least-squares%20algorithm%20for%20fitting%20a%20straight%20line&amp;author=M.%20Krystek&amp;author=M.%20Anton&amp;publication_year=2008&amp;journal=Meas.%20Sci.%20Technol.&amp;volume=19&amp;pages=79801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1088/0957-0233/19/7/079801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1088%2F0957-0233%2F19%2F7%2F079801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1088%2F0957-0233%2F19%2F7%2F079801"> </span></div> </div></div></div></div></div><div content-id="DSW009C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Klumpp</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hwa</div> <div class="given-names">T.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Growth-rate-dependent partitioning of RNA polymerases in bacteria</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">105</div>, <div class="fpage">20245</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Growth-rate-dependent%20partitioning%20of%20RNA%20polymerases%20in%20bacteria&amp;author=S.%20Klumpp&amp;author=T.%20Hwa&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=20245-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0804953105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0804953105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0804953105"> </span></div> </div></div></div></div></div><div content-id="DSW009C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lutz</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Bujard</div> <div class="given-names">H.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Independent and tight regulation of transcriptional units in <em>Escherichia coli</em> via the LacR/O, the TetR/O and AraC/I<sub>1</sub>-I<sub>2</sub> regulatory elements</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">1203</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Independent%20and%20tight%20regulation%20of%20transcriptional%20units%20in%20Escherichia%20coli%20via%20the%20LacR%2FO%2C%20the%20TetR%2FO%20and%20AraC%2FI1-I2%20regulatory%20elements&amp;author=R.%20Lutz&amp;author=H.%20Bujard&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=1203-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.6.1203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.6.1203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.6.1203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9092630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stricker</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Cookson</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Mather</div> <div class="given-names">W.H.</div></div>, <div class="name string-name"><div class="surname">Tsimring</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Hasty</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">A fast, robust and tunable synthetic gene oscillator</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">516</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20robust%20and%20tunable%20synthetic%20gene%20oscillator&amp;author=J.%20Stricker&amp;author=S.%20Cookson&amp;author=M.R.%20Bennett&amp;author=W.H.%20Mather&amp;author=L.S.%20Tsimring&amp;author=J.%20Hasty&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=516-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18971928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martins</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Mäkelä</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Häkkinen</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Dynamics of transcription of closely spaced promoters in <em>Escherichia coli</em>, one event at a time</div>, <div class="source">J. Theor. Biol.</div> , <div class="volume">301</div>, <div class="fpage">83</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamics%20of%20transcription%20of%20closely%20spaced%20promoters%20in%20Escherichia%20coli%2C%20one%20event%20at%20a%20time&amp;author=L.%20Martins&amp;author=J.%20M%C3%A4kel%C3%A4&amp;author=A.%20H%C3%A4kkinen&amp;publication_year=2012&amp;journal=J.%20Theor.%20Biol.&amp;volume=301&amp;pages=83-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2012.02.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2012.02.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2012.02.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22370562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mäkelä</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Kandhavelu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Oliveira</div> <div class="given-names">S.M.D.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title"><em>In vivo</em> single-molecule kinetics of activation and subsequent activity of the arabinose promoter</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">6544</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20single-molecule%20kinetics%20of%20activation%20and%20subsequent%20activity%20of%20the%20arabinose%20promoter&amp;author=J.%20M%C3%A4kel%C3%A4&amp;author=M.%20Kandhavelu&amp;author=S.M.D.%20Oliveira&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=6544-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kandhavelu</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Lihavainen</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Muthukrishnan</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Yli-Harja</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Ribeiro</div> <div class="given-names">A.S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Effects of Mg<sup>2+</sup> on <em>in vivo</em> transcriptional dynamics of the <em>lar</em> promoter</div>, <div class="source">BioSystems</div> , <div class="volume">107</div>, <div class="fpage">129</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20Mg2%2B%20on%20in%20vivo%20transcriptional%20dynamics%20of%20the%20lar%20promoter&amp;author=M.%20Kandhavelu&amp;author=E.%20Lihavainen&amp;author=A.B.%20Muthukrishnan&amp;author=O.%20Yli-Harja&amp;author=A.S.%20Ribeiro&amp;publication_year=2012&amp;journal=BioSystems&amp;volume=107&amp;pages=129-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biosystems.2011.11.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biosystems.2011.11.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biosystems.2011.11.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22101250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pedraza</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Paulsson</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Effects of molecular memory and bursting on fluctuations in gene expression</div>, <div class="source">Science</div> , <div class="volume">319</div>, <div class="fpage">339</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20molecular%20memory%20and%20bursting%20on%20fluctuations%20in%20gene%20expression&amp;author=J.M.%20Pedraza&amp;author=J.%20Paulsson&amp;publication_year=2008&amp;journal=Science&amp;volume=319&amp;pages=339-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1144331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1144331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1144331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Casella</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Berger</div> <div class="given-names">R.L.</div></div></div>
          <div class="year">2001</div>, <div class="source">Simultaneous Estimation and Confidence Bands. Statistical Inference</div> , <div class="edition">2nd ed</div>. <div class="publisher-name">Duxbury Press</div>, <div class="publisher-loc">Pacific Grove, CA, USA</div>, pp. <div class="fpage">559</div>–<div class="lpage">63</div>.</div></div></div></div><div content-id="DSW009C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schwarz</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1978</div>, <div class="article-title">Estimating the dimension of a model</div>, <div class="source">Ann. Stat.</div> , <div class="volume">6</div>, <div class="fpage">461</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20dimension%20of%20a%20model&amp;author=G.%20Schwarz&amp;publication_year=1978&amp;journal=Ann.%20Stat.&amp;volume=6&amp;pages=461-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/aos/1176344136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faos%2F1176344136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1214%2Faos%2F1176344136"> </span></div> </div></div></div></div></div><div content-id="DSW009C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Malan</div>
               <div class="given-names">T.P.</div>
            </div>, <div class="name string-name"><div class="surname">Kolb</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">McClure</div> <div class="given-names">W.R.</div></div></div>
          <div class="year">1984</div>, <div class="article-title">Mechanism of CRP-cAMP activation of <em>lac</em> operon transcription initiation activation of the <em>P1</em> promoter</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">180</div>, <div class="fpage">881</div>–<div class="lpage">909</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20CRP-cAMP%20activation%20of%20lac%20operon%20transcription%20initiation%20activation%20of%20the%20P1%20promoter&amp;author=T.P.%20Malan&amp;author=A.%20Kolb&amp;author=H.%20Buc&amp;author=W.R.%20McClure&amp;publication_year=1984&amp;journal=J.%20Mol.%20Biol.&amp;volume=180&amp;pages=881-909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(84)90262-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(84)90262-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(84)90262-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6098691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Buc</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">McClure</div> <div class="given-names">W.R.</div></div></div>
          <div class="year">1985</div>, <div class="article-title">Kinetics of open complex formation between <em>Escherichia coli</em> RNA polymerase and the <em>lac</em> UV5 promoter. Evidence for a sequential mechanism involving three steps</div>, <div class="source">Biochemistry</div> , <div class="volume">24</div>, <div class="fpage">2712</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kinetics%20of%20open%20complex%20formation%20between%20Escherichia%20coli%20RNA%20polymerase%20and%20the%20lac%20UV5%20promoter.%20Evidence%20for%20a%20sequential%20mechanism%20involving%20three%20steps&amp;author=H.%20Buc&amp;author=W.R.%20McClure&amp;publication_year=1985&amp;journal=Biochemistry&amp;volume=24&amp;pages=2712-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00332a018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00332a018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00332a018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3896304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sanchez</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Choubey</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kondev</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Stochastic models of transcription: From single molecules to single cells</div>, <div class="source">Methods</div> , <div class="volume">62</div>, <div class="fpage">13</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stochastic%20models%20of%20transcription%3A%20From%20single%20molecules%20to%20single%20cells&amp;author=A.%20Sanchez&amp;author=S.%20Choubey&amp;author=J.%20Kondev&amp;publication_year=2013&amp;journal=Methods&amp;volume=62&amp;pages=13-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2013.03.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2013.03.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2013.03.026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23557991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW009C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW009C74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schwabe</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Rybakova</div> <div class="given-names">K.N.</div></div>, <div class="name string-name"><div class="surname">Bruggeman</div> <div class="given-names">F.J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Transcription stochasticity of complex gene regulation models</div>, <div class="source">Biophys. J.</div> , <div class="volume">103</div>, <div class="fpage">1152</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20stochasticity%20of%20complex%20gene%20regulation%20models&amp;author=A.%20Schwabe&amp;author=K.N.%20Rybakova&amp;author=F.J.%20Bruggeman&amp;publication_year=2012&amp;journal=Biophys.%20J.&amp;volume=103&amp;pages=1152-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bpj.2012.07.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bpj.2012.07.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bpj.2012.07.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22995487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745366&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw009/1/dsw009_Supplementary_Data.zip?Expires=2147483647&amp;Signature=aZFby0S82tn0dTUlef5SFU6NIGkB1CseGPRBYeOWJjLwLDP70zqwDa3ADGcs5USZmRCVoMup02IzxNEOaGnAYoGBYqVGEF-lqI7n1e-7ppchGhTg-22zfEsDIP6vx1QCpyPoZk3m0LRwYThhlSqR3DlUGI-4XxuKULmly9VnsqE865jYQhR-5Q7yXw4AymsA7lUmZf~FFPT1p9Q1SOhGwzw-FjTpCz0xVs2hluINe6I6sSyD3sZcW~Ra0HN1vfNFS2pM1AsJ4nCjeVdbUcmw-yi-6VC7E8bB1dreeh-wsNKS~BEydS3hMsTq90V5vQqUi2HF8lPiU26mZLa9wcdQOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">401</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745366"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f203%2f1745366">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F203%2F1745366" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745366'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Dissecting the stochastic transcription initiation process in live Escherichia coli" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f203%2f1745366">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

