

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/5/367/348265","@type":"ScholarlyArticle","name":"High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers","datePublished":"2015-09-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv019","keywords":["sex determination","RAD-seq","genome assembly","<italic>Actinidia chinensis</italic>","<italic>Actinidia rufa</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/5/367/348265","author":[{"name":"Zhang, Qiong","affiliation":"1  Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden ,  Chinese Academy of Sciences ,  Wuhan 430074 ,  China","@type":"Person"},{"name":"Liu, Chunyan","affiliation":"1  Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden ,  Chinese Academy of Sciences ,  Wuhan 430074 ,  China    2  University of Chinese Academy of Sciences ,  Beijing 100039 ,  China","@type":"Person"},{"name":"Liu, Yifei","affiliation":"3  South China Botanical Garden, Chinese Academy of Sciences ,  Guangzhou 510650 ,  China","@type":"Person"},{"name":"VanBuren, Robert","affiliation":"4  Donald Danforth Plant Science Center ,  St Louis, MO ,  USA","@type":"Person"},{"name":"Yao, Xiaohong","affiliation":"1  Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden ,  Chinese Academy of Sciences ,  Wuhan 430074 ,  China","@type":"Person"},{"name":"Zhong, Caihong","affiliation":"1  Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden ,  Chinese Academy of Sciences ,  Wuhan 430074 ,  China","@type":"Person"},{"name":"Huang, Hongwen","affiliation":"1  Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden ,  Chinese Academy of Sciences ,  Wuhan 430074 ,  China","@type":"Person"}],"description":"Abstract.  Kiwifruit (Actinidia chinensis Planchon) is an important specialty fruit crop that suffers from narrow genetic diversity stemming from recent global ","pageStart":"367","pageEnd":"375","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers" />
    <meta property="og:description" content="Abstract.  Kiwifruit (Actinidia chinensis Planchon) is an important specialty fruit crop that suffers from narrow genetic diversity stemming from recent global " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/5/367/348265" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-09-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhang, Qiong" /><meta name="citation_author_institution" content="Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China" /><meta name="citation_author" content="Liu, Chunyan" /><meta name="citation_author_institution" content="Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing 100039, China" /><meta name="citation_author" content="Liu, Yifei" /><meta name="citation_author_institution" content="South China Botanical Garden, Chinese Academy of Sciences, Guangzhou 510650, China" /><meta name="citation_author" content="VanBuren, Robert" /><meta name="citation_author_institution" content="Donald Danforth Plant Science Center, St Louis, MO, USA" /><meta name="citation_author" content="Yao, Xiaohong" /><meta name="citation_author_institution" content="Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China" /><meta name="citation_author" content="Zhong, Caihong" /><meta name="citation_author_institution" content="Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China" /><meta name="citation_author" content="Huang, Hongwen" /><meta name="citation_author_institution" content="Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan 430074, China" /><meta name="citation_title" content="High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers" /><meta name="citation_firstpage" content="367" /><meta name="citation_lastpage" content="375" /><meta name="citation_doi" content="10.1093/dnares/dsv019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2015/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Science Press, Beijing;  citation_title=Actinidiaceae/Flora of China;  citation_year=2007;  citation_volume=vol. 12;  citation_pages=334-60; " /><meta name="citation_reference" content="citation_title=‘Jinyan’, an interspecific hybrid kiwifruit with Brilliant Yellow Flesh and Good Storage Quality;  citation_journal_title=HortScience;  citation_year=2012;  citation_volume=47;  citation_pages=1187-90; " /><meta name="citation_reference" content="citation_title=The evolution of sex chromosomes;  citation_journal_title=Science;  citation_year=1991;  citation_volume=251;  citation_pages=1030-3; " /><meta name="citation_reference" content="citation_title=Sex determination in the monoecious species cucumber is confined to specific floral whorls;  citation_journal_title=Plant Cell;  citation_year=2001;  citation_volume=13;  citation_pages=481-93; " /><meta name="citation_reference" content="citation_title=A primitive Y chromosome in papaya marks incipient sex chromosome evolution;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=427;  citation_pages=348-52; " /><meta name="citation_reference" content="citation_title=Identification of sex-linked SNP markers using RAD sequencing suggests ZW/ZZ sex determination in Pistacia vera L.;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=98" /><meta name="citation_reference" content="citation_title=Genetic mapping of sex determination in a wild strawberry, Fragaria virginiana, reveals earliest form of sex chromosome;  citation_journal_title=Heredity;  citation_year=2008;  citation_volume=101;  citation_pages=507-17; " /><meta name="citation_reference" content="citation_title=Independent origin of sex chromosomes in two species of the genus Silene;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=1129-33; " /><meta name="citation_reference" content="citation_publisher=Science Press, Beijing;  citation_title=The Genus of Actinidia: A Word Monograph;  citation_year=2002;  citation_pages=34-6; " /><meta name="citation_reference" content="citation_title=Flower and fruit characters in a kiwifruit hermaphrodite;  citation_journal_title=Euphytica;  citation_year=1998;  citation_volume=101;  citation_pages=69-72; " /><meta name="citation_reference" content="citation_title=Genetic resources of kiwifruit: domestication and breeding;  citation_journal_title=Hortic. Rev.;  citation_year=2007;  citation_volume=33;  citation_pages=1-121; " /><meta name="citation_reference" content="citation_title=A kiwifruit (Actinidia spp.) linkage map based on microsatellites and integrated with AFLP markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=30-6; " /><meta name="citation_reference" content="citation_title=A gene-rich linkage map in the dioecious species Actinidia chinensis (kiwifruit) reveals putative X/Y sex-determining chromosomes;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=102" /><meta name="citation_reference" content="citation_title=Development of sex-linked PCR markers for gender identification in Actinidia;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1998;  citation_volume=97;  citation_pages=439-45; " /><meta name="citation_reference" content="citation_title=Draft genome of the kiwifruit Actinidia chinensis;  citation_journal_title=Nat. Commun.;  citation_year=2013;  citation_volume=4; " /><meta name="citation_reference" content="citation_title=Mapping with RAD (restriction-site associated DNA) markers to rapidly identify QTL for stem rust resistance in Lolium perenne;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1467-80; " /><meta name="citation_reference" content="citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19315" /><meta name="citation_reference" content="citation_title=High-density detection of restriction-site-associated DNA markers for rapid mapping of mutated loci in Neurospora;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=1163-71; " /><meta name="citation_reference" content="citation_title=An integrated genetic map based on four mapping populations and quantitative trait loci associated with economically important traits in watermelon (Citrullus lanatus);  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Construction and application for QTL analysis of a restriction site associated DNA (RAD) linkage map in barley;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis;  citation_journal_title=DNA Res.;  citation_year=2015;  citation_volume=22;  citation_pages=161-70; " /><meta name="citation_reference" content="citation_title=Isolation of plant DNA from fresh tissue;  citation_journal_title=Focus;  citation_year=1990;  citation_volume=12;  citation_pages=13-5; " /><meta name="citation_reference" content="citation_title=Optimization of linkage mapping strategy and construction of a high-density American lotus linkage map;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=372" /><meta name="citation_reference" content="citation_title=Resolving postglacial phylogeography using high-throughput sequencing;  citation_journal_title=Proc. Natl. Acad. Sci.;  citation_year=2010;  citation_volume=107;  citation_pages=16196-200; " /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=SNP detection for massively parallel whole-genome resequencing;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1124-32; " /><meta name="citation_reference" content="citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers;  citation_journal_title=Genetics;  citation_year=1994;  citation_volume=137;  citation_pages=1121-37; " /><meta name="citation_reference" content="citation_publisher=Kyazma BV, Wageningen, Netherlands;  citation_title=JoinMap 4. Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Control of plant germline proliferation by SCFFBL17 degradation of cell cycle inhibitors;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=455;  citation_pages=1134-7; " /><meta name="citation_reference" content="citation_title=A hybrid BAC physical map of potato: a framework for sequencing a heterozygous genome;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=594" /><meta name="citation_reference" content="citation_title=Large-scale recombination rate patterns are conserved among human populations;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1547-52; " /><meta name="citation_reference" content="citation_title=Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps;  citation_journal_title=Genome Biol. Evol.;  citation_year=2014;  citation_volume=6;  citation_pages=3295-313; " /><meta name="citation_reference" content="citation_title=Integration of the genetic map and genome assembly of fugu facilitates insights into distinct features of genome evolution in teleosts and mammals;  citation_journal_title=Genome Biol. Evol.;  citation_year=2011;  citation_volume=3;  citation_pages=424-42; " /><meta name="citation_reference" content="citation_title=Comparative studies of de novo assembly tools for next-generation sequencing technologies;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2031-7; " /><meta name="citation_reference" content="citation_title=Integration of physical and genetic maps in apple confirms whole-genome and segmental duplications in the apple genome;  citation_journal_title=J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=5117-30; " /><meta name="citation_reference" content="citation_title=A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e47864" /><meta name="citation_reference" content="citation_title=Dioecy and its correlates in the flowering plants;  citation_journal_title=Am. J. Bot.;  citation_year=1995;  citation_volume=82;  citation_pages=596-606; " /><meta name="citation_reference" content="citation_title=Sex chromosomes in land plants;  citation_journal_title=Ann. Rev. Plant Biol.;  citation_year=2011;  citation_volume=62;  citation_pages=485-514; " /><meta name="citation_reference" content="citation_title=Sex determination in Actinidia. 1. Sex-linked markers and progeny sex ratio in diploid A. chinensis;  citation_journal_title=Sex. Plant Reprod.;  citation_year=1997;  citation_volume=10;  citation_pages=149-54; " /><meta name="citation_reference" content="citation_title=Plant sex chromosomes: molecular structure and function;  citation_journal_title=Cytogenet Genome Res.;  citation_year=2007;  citation_volume=120;  citation_pages=255-64; " /><meta name="citation_reference" content="citation_title=Sequencing papaya X and Yh chromosomes reveals molecular basis of incipient sex chromosome evolution;  citation_journal_title=Proc. Natl. Acad. Sci.;  citation_year=2012;  citation_volume=109;  citation_pages=13710-15; " /><meta name="citation_reference" content="citation_title=Genome structure and emerging evidence of an incipient sex chromosome in Populus;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=422-30; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/5/367/998037/dsv019.pdf" /><meta name="description" content="Abstract.  Kiwifruit (Actinidia chinensis Planchon) is an important specialty fruit crop that suffers from narrow genetic diversity stemming from recent global " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/5/367/348265" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/5/367/348265" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/5/367/348265");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv019';
    </script>
<title>
	High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348265" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdhpPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgUWlvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBQbGFudCBHZXJtcGxhc20gRW5oYW5jZW1lbnQgYW5kIFNwZWNpYWx0eSBBZ3JpY3VsdHVyZSwgV3VoYW4gQm90YW5pY2FsIEdhcmRlbiwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBXdWhhbiA0MzAwNzQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBDaHVueWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgUGxhbnQgR2VybXBsYXNtIEVuaGFuY2VtZW50IGFuZCBTcGVjaWFsdHkgQWdyaWN1bHR1cmUsIFd1aGFuIEJvdGFuaWNhbCBHYXJkZW4sIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgV3VoYW4gNDMwMDc0LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDAzOSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIFlpZmVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU291dGggQ2hpbmEgQm90YW5pY2FsIEdhcmRlbiwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBHdWFuZ3pob3UgNTEwNjUwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhbkJ1cmVuLCBSb2JlcnQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEb25hbGQgRGFuZm9ydGggUGxhbnQgU2NpZW5jZSBDZW50ZXIsIFN0IExvdWlzLCBNTywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFvLCBYaWFvaG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIFBsYW50IEdlcm1wbGFzbSBFbmhhbmNlbWVudCBhbmQgU3BlY2lhbHR5IEFncmljdWx0dXJlLCBXdWhhbiBCb3RhbmljYWwgR2FyZGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFd1aGFuIDQzMDA3NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaG9uZywgQ2FpaG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIFBsYW50IEdlcm1wbGFzbSBFbmhhbmNlbWVudCBhbmQgU3BlY2lhbHR5IEFncmljdWx0dXJlLCBXdWhhbiBCb3RhbmljYWwgR2FyZGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFd1aGFuIDQzMDA3NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgSG9uZ3dlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIFBsYW50IEdlcm1wbGFzbSBFbmhhbmNlbWVudCBhbmQgU3BlY2lhbHR5IEFncmljdWx0dXJlLCBXdWhhbiBCb3RhbmljYWwgR2FyZGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFd1aGFuIDQzMDA3NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtZGVuc2l0eSBpbnRlcnNwZWNpZmljIGdlbmV0aWMgbWFwcyBvZiBraXdpZnJ1aXQgYW5kIHRoZSBpZGVudGlmaWNhdGlvbiBvZiBzZXgtc3BlY2lmaWMgbWFya2VycyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2NpZW5jZSBQcmVzcywgQmVpamluZzsgIGNpdGF0aW9uX3RpdGxlPUFjdGluaWRpYWNlYWUvRmxvcmEgb2YgQ2hpbmE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiAxMjsgIGNpdGF0aW9uX3BhZ2VzPTMzNC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU94oCYSmlueWFu4oCZLCBhbiBpbnRlcnNwZWNpZmljIGh5YnJpZCBraXdpZnJ1aXQgd2l0aCBCcmlsbGlhbnQgWWVsbG93IEZsZXNoIGFuZCBHb29kIFN0b3JhZ2UgUXVhbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SG9ydFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xMTg3LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIHNleCBjaHJvbW9zb21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTE7ICBjaXRhdGlvbl9wYWdlcz0xMDMwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNleCBkZXRlcm1pbmF0aW9uIGluIHRoZSBtb25vZWNpb3VzIHNwZWNpZXMgY3VjdW1iZXIgaXMgY29uZmluZWQgdG8gc3BlY2lmaWMgZmxvcmFsIHdob3JsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ4MS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmltaXRpdmUgWSBjaHJvbW9zb21lIGluIHBhcGF5YSBtYXJrcyBpbmNpcGllbnQgc2V4IGNocm9tb3NvbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDI3OyAgY2l0YXRpb25fcGFnZXM9MzQ4LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzZXgtbGlua2VkIFNOUCBtYXJrZXJzIHVzaW5nIFJBRCBzZXF1ZW5jaW5nIHN1Z2dlc3RzIFpXL1paIHNleCBkZXRlcm1pbmF0aW9uIGluIFBpc3RhY2lhIHZlcmEgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYXBwaW5nIG9mIHNleCBkZXRlcm1pbmF0aW9uIGluIGEgd2lsZCBzdHJhd2JlcnJ5LCBGcmFnYXJpYSB2aXJnaW5pYW5hLCByZXZlYWxzIGVhcmxpZXN0IGZvcm0gb2Ygc2V4IGNocm9tb3NvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTUwNy0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kZXBlbmRlbnQgb3JpZ2luIG9mIHNleCBjaHJvbW9zb21lcyBpbiB0d28gc3BlY2llcyBvZiB0aGUgZ2VudXMgU2lsZW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzk7ICBjaXRhdGlvbl9wYWdlcz0xMTI5LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U2NpZW5jZSBQcmVzcywgQmVpamluZzsgIGNpdGF0aW9uX3RpdGxlPVRoZSBHZW51cyBvZiBBY3RpbmlkaWE6IEEgV29yZCBNb25vZ3JhcGg7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl9wYWdlcz0zNC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG93ZXIgYW5kIGZydWl0IGNoYXJhY3RlcnMgaW4gYSBraXdpZnJ1aXQgaGVybWFwaHJvZGl0ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTY5LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHJlc291cmNlcyBvZiBraXdpZnJ1aXQ6IGRvbWVzdGljYXRpb24gYW5kIGJyZWVkaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ib3J0aWMuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xLTEyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBraXdpZnJ1aXQgKEFjdGluaWRpYSBzcHAuKSBsaW5rYWdlIG1hcCBiYXNlZCBvbiBtaWNyb3NhdGVsbGl0ZXMgYW5kIGludGVncmF0ZWQgd2l0aCBBRkxQIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lLXJpY2ggbGlua2FnZSBtYXAgaW4gdGhlIGRpb2VjaW91cyBzcGVjaWVzIEFjdGluaWRpYSBjaGluZW5zaXMgKGtpd2lmcnVpdCkgcmV2ZWFscyBwdXRhdGl2ZSBYL1kgc2V4LWRldGVybWluaW5nIGNocm9tb3NvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBzZXgtbGlua2VkIFBDUiBtYXJrZXJzIGZvciBnZW5kZXIgaWRlbnRpZmljYXRpb24gaW4gQWN0aW5pZGlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NDM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgb2YgdGhlIGtpd2lmcnVpdCBBY3RpbmlkaWEgY2hpbmVuc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyB3aXRoIFJBRCAocmVzdHJpY3Rpb24tc2l0ZSBhc3NvY2lhdGVkIEROQSkgbWFya2VycyB0byByYXBpZGx5IGlkZW50aWZ5IFFUTCBmb3Igc3RlbSBydXN0IHJlc2lzdGFuY2UgaW4gTG9saXVtIHBlcmVubmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9MTQ2Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlua2FnZSBtYXBwaW5nIGFuZCBjb21wYXJhdGl2ZSBnZW5vbWljcyB1c2luZyBuZXh0LWdlbmVyYXRpb24gUkFEIHNlcXVlbmNpbmcgb2YgYSBub24tbW9kZWwgb3JnYW5pc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTkzMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWRlbnNpdHkgZGV0ZWN0aW9uIG9mIHJlc3RyaWN0aW9uLXNpdGUtYXNzb2NpYXRlZCBETkEgbWFya2VycyBmb3IgcmFwaWQgbWFwcGluZyBvZiBtdXRhdGVkIGxvY2kgaW4gTmV1cm9zcG9yYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9MTE2My03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBnZW5ldGljIG1hcCBiYXNlZCBvbiBmb3VyIG1hcHBpbmcgcG9wdWxhdGlvbnMgYW5kIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGFzc29jaWF0ZWQgd2l0aCBlY29ub21pY2FsbHkgaW1wb3J0YW50IHRyYWl0cyBpbiB3YXRlcm1lbG9uIChDaXRydWxsdXMgbGFuYXR1cyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBhcHBsaWNhdGlvbiBmb3IgUVRMIGFuYWx5c2lzIG9mIGEgcmVzdHJpY3Rpb24gc2l0ZSBhc3NvY2lhdGVkIEROQSAoUkFEKSBsaW5rYWdlIG1hcCBpbiBiYXJsZXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBTTlAgaWRlbnRpZmljYXRpb24gZm9yIHRoZSBjb25zdHJ1Y3Rpb24gb2YgYSBoaWdoLXJlc29sdXRpb24gZ2VuZXRpYyBtYXAgb2YgSmFwYW5lc2UgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpOiBhcHBsaWNhdGlvbnMgdG8gUVRMIG1hcHBpbmcgb2YgVmlicmlvIGFuZ3VpbGxhcnVtIGRpc2Vhc2UgcmVzaXN0YW5jZSBhbmQgY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiBwbGFudCBETkEgZnJvbSBmcmVzaCB0aXNzdWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvY3VzOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemF0aW9uIG9mIGxpbmthZ2UgbWFwcGluZyBzdHJhdGVneSBhbmQgY29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IEFtZXJpY2FuIGxvdHVzIGxpbmthZ2UgbWFwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zNzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNvbHZpbmcgcG9zdGdsYWNpYWwgcGh5bG9nZW9ncmFwaHkgdXNpbmcgaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MTYxOTYtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQMjogYW4gaW1wcm92ZWQgdWx0cmFmYXN0IHRvb2wgZm9yIHNob3J0IHJlYWQgYWxpZ25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE5NjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRldGVjdGlvbiBmb3IgbWFzc2l2ZWx5IHBhcmFsbGVsIHdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTEyNC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcHMgb2YgRXVjYWx5cHR1cyBncmFuZGlzIGFuZCBFdWNhbHlwdHVzIHVyb3BoeWxsYSB1c2luZyBhIHBzZXVkby10ZXN0Y3Jvc3M6IG1hcHBpbmcgc3RyYXRlZ3kgYW5kIFJBUEQgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9MTEyMS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUt5YXptYSBCViwgV2FnZW5pbmdlbiwgTmV0aGVybGFuZHM7ICBjaXRhdGlvbl90aXRsZT1Kb2luTWFwIDQuIFNvZnR3YXJlIGZvciB0aGUgQ2FsY3VsYXRpb24gb2YgR2VuZXRpYyBMaW5rYWdlIE1hcHMgaW4gRXhwZXJpbWVudGFsIFBvcHVsYXRpb25zOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIHBsYW50IGdlcm1saW5lIHByb2xpZmVyYXRpb24gYnkgU0NGRkJMMTcgZGVncmFkYXRpb24gb2YgY2VsbCBjeWNsZSBpbmhpYml0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU1OyAgY2l0YXRpb25fcGFnZXM9MTEzNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGh5YnJpZCBCQUMgcGh5c2ljYWwgbWFwIG9mIHBvdGF0bzogYSBmcmFtZXdvcmsgZm9yIHNlcXVlbmNpbmcgYSBoZXRlcm96eWdvdXMgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01OTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSByZWNvbWJpbmF0aW9uIHJhdGUgcGF0dGVybnMgYXJlIGNvbnNlcnZlZCBhbW9uZyBodW1hbiBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNTQ3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgb3JpZ2lucyBhbmQgZHluYW1pY3Mgb2Ygb2N0b3Bsb2lkIHN0cmF3YmVycnkgc3ViZ2Vub21lcyByZXZlYWxlZCBieSBkZW5zZSB0YXJnZXRlZCBjYXB0dXJlIGxpbmthZ2UgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMjk1LTMxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpb24gb2YgdGhlIGdlbmV0aWMgbWFwIGFuZCBnZW5vbWUgYXNzZW1ibHkgb2YgZnVndSBmYWNpbGl0YXRlcyBpbnNpZ2h0cyBpbnRvIGRpc3RpbmN0IGZlYXR1cmVzIG9mIGdlbm9tZSBldm9sdXRpb24gaW4gdGVsZW9zdHMgYW5kIG1hbW1hbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NDI0LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBzdHVkaWVzIG9mIGRlIG5vdm8gYXNzZW1ibHkgdG9vbHMgZm9yIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMDMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW9uIG9mIHBoeXNpY2FsIGFuZCBnZW5ldGljIG1hcHMgaW4gYXBwbGUgY29uZmlybXMgd2hvbGUtZ2Vub21lIGFuZCBzZWdtZW50YWwgZHVwbGljYXRpb25zIGluIHRoZSBhcHBsZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTUxMTctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXVsdGktcG9wdWxhdGlvbiBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgcmV2ZWFscyBpbmNvbnNpc3RlbnQgbWFya2VyIG9yZGVyIGFtb25nIG1hcHMgbGlrZWx5IGF0dHJpYnV0ZWQgdG8gc3RydWN0dXJhbCB2YXJpYXRpb25zIGluIHRoZSBhcHBsZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDc4NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaW9lY3kgYW5kIGl0cyBjb3JyZWxhdGVzIGluIHRoZSBmbG93ZXJpbmcgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTU5Ni02MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNleCBjaHJvbW9zb21lcyBpbiBsYW5kIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NDg1LTUxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V4IGRldGVybWluYXRpb24gaW4gQWN0aW5pZGlhLiAxLiBTZXgtbGlua2VkIG1hcmtlcnMgYW5kIHByb2dlbnkgc2V4IHJhdGlvIGluIGRpcGxvaWQgQS4gY2hpbmVuc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TZXguIFBsYW50IFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNDktNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHNleCBjaHJvbW9zb21lczogbW9sZWN1bGFyIHN0cnVjdHVyZSBhbmQgZnVuY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldCBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0yNTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgcGFwYXlhIFggYW5kIFloIGNocm9tb3NvbWVzIHJldmVhbHMgbW9sZWN1bGFyIGJhc2lzIG9mIGluY2lwaWVudCBzZXggY2hyb21vc29tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTM3MTAtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzdHJ1Y3R1cmUgYW5kIGVtZXJnaW5nIGV2aWRlbmNlIG9mIGFuIGluY2lwaWVudCBzZXggY2hyb21vc29tZSBpbiBQb3B1bHVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzUvMzY3Lzk5ODAzNy9kc3YwMTkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEtpd2lmcnVpdCAoQWN0aW5pZGlhIGNoaW5lbnNpcyBQbGFuY2hvbikgaXMgYW4gaW1wb3J0YW50IHNwZWNpYWx0eSBmcnVpdCBjcm9wIHRoYXQgc3VmZmVycyBmcm9tIG5hcnJvdyBnZW5ldGljIGRpdmVyc2l0eSBzdGVtbWluZyBmcm9tIHJlY2VudCBnbG9iYWwgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDgyNjUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk2tsEBBjZsFphFjyTjgAV4ChBBko=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f367%2f348265" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f367%2f348265" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&amp;Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/5">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4942849">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942849">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942851">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942851">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942855">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942855">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942869">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942869">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942885">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942885">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942892">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942892">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942894">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942894">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4942896">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4942896">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/5/357/348184">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/5/377/348317">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiong Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430074</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qiong+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qiong Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chunyan Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430074</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 100039</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chunyan+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chunyan Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yifei Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">South China Botanical Garden, Chinese Academy of Sciences</div>, <div class="addr-line">Guangzhou 510650</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yifei+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yifei Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert VanBuren</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Donald Danforth Plant Science Center</div>, <div class="addr-line">St Louis, MO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+VanBuren">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=VanBuren R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;VanBuren R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert VanBuren</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaohong Yao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430074</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaohong+Yao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yao X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yao X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaohong Yao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Caihong Zhong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430074</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Caihong+Zhong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhong C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhong C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Caihong Zhong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongwen Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Wuhan 430074</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-87510232. Fax. +86 27-87510331. E-mail: <a href="mailto:huanghw@scbg.ac.cn" target="_blank">huanghw@scbg.ac.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongwen+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hongwen Huang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="TN1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 367–375, <a href='https://doi.org/10.1093/dnares/dsv019'>https://doi.org/10.1093/dnares/dsv019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 September 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 April 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 August 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348265" href="/dnaresearch/article-pdf/22/5/367/998037/dsv019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Qiong Zhang, Chunyan Liu, Yifei Liu, Robert VanBuren, Xiaohong Yao, Caihong Zhong, Hongwen Huang;  High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers, <em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 367–375, <a href='https://doi.org/10.1093/dnares/dsv019'>https://doi.org/10.1093/dnares/dsv019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348265?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348265?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-density%20interspecific%20genetic%20maps%20of%20kiwifruit%20and%20the%20identification%20of%20sex-specific%20markers&amp;publicationDate=2015-09-14&amp;volumeNum=22&amp;issueNum=5&amp;author=Zhang%2C%20Qiong%3B%20Liu%2C%20Chunyan&amp;startPage=367&amp;endPage=375&amp;contentId=10.1093%2Fdnares%2Fdsv019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f367%2f348265" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4942849 id="4942849" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Kiwifruit (<em>Actinidia chinensis</em> Planchon) is an important specialty fruit crop that suffers from narrow genetic diversity stemming from recent global commercialization and limited cultivar improvement. Here, we present high-density RAD-seq-based genetic maps using an interspecific F<sub>1</sub> cross between <em>Actinidia rufa</em> ‘MT570001’ and <em>A. chinensis</em> ‘Guihai No4’. The <em>A. rufa</em> (maternal) map consists of 2,426 single-nucleotide polymorphism (SNP) markers with a total length of 2,651 cM in 29 linkage groups (LGs) corresponding to the 29 chromosomes. The <em>A. chinensis</em> (paternal) map consists of 4,214 SNP markers over 3,142 cM in 29 LGs. Using these maps, we were able to anchor an additional 440 scaffolds from the kiwifruit draft genome assembly. Kiwifruit is functionally dioecious, which presents unique challenges for breeding and production. Three sex-specific simple sequence repeats (SSR) markers can be used to accurately sex type male and female kiwifruit in breeding programmes. The sex-determination region (SDR) in kiwifruit was narrowed to a 1-Mb subtelomeric region on chromosome 25. Localizing the SDR will expedite the discovery of genes controlling carpel abortion in males and pollen sterility in females.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sex determination&quot;">sex determination</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RAD-seq">RAD-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome assembly&quot;">genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Actinidia chinensis&quot;"><em>Actinidia chinensis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Actinidia rufa&quot;"><em>Actinidia rufa</em></a></div>                            <h2 scrollto-destination=4942851 id="4942851" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Kiwifruit (<em>Actinidia chinensis</em> Planchon) is one of the most recently domesticated specialty fruit crops and is currently grown commercially worldwide. The genus <em>Actinidia</em> (2<em>n</em> = 2<em>x</em> = 58 chromosomes) comprises 54 species and 75 taxa in total, and most of these species occur naturally in China.<sup><a reveal-id="DSV019C1" data-open="DSV019C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Most commercial cultivars were developed from a narrow pool of <em>A. chinensis</em> germplasm, and the current lack of genetic diversity makes kiwifruit vulnerable to new diseases and hinders cultivar improvement. The underexplored wild <em>Actinidia</em> germplasm has a wide range of desirable fruit characteristics and high potential for developing new kiwifruit cultivars. Interspecific hybridization is a proved approach to combine desirable traits from different species and surmount obstacles of paternal selection of dioecious plants, for example, ‘Jinyan’, a new cultivar bred by interspecific hybridization between <em>Actinidia eriantha</em> and <em>A. chinensis</em>, has excellent storage quality which is attributed to <em>A. eriantha</em>.<sup><a reveal-id="DSV019C2" data-open="DSV019C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>The vast majority of flowering plants, including most crop plants, are hermaphroditic, but a small portion (6%) are dioecious with separate male and female plants. Dioecy likely evolves from hermaphrodite or monoecious ancestors by two mutations with the first causing male sterility and the second reducing female fertility.<sup><a reveal-id="DSV019C3" data-open="DSV019C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Dioecy can be controlled by individual gene (e.g. cucumis),<sup><a reveal-id="DSV019C4" data-open="DSV019C4" class="link link-ref link-reveal xref-bibr">4</a></sup> or fully developed sex-specific regions or sex chromosomes as observed in papaya,<sup><a reveal-id="DSV019C5" data-open="DSV019C5" class="link link-ref link-reveal xref-bibr">5</a></sup> pistachio,<sup><a reveal-id="DSV019C6" data-open="DSV019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> strawberry,<sup><a reveal-id="DSV019C7" data-open="DSV019C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and the model plant Silene.<sup><a reveal-id="DSV019C8" data-open="DSV019C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Kiwifruit is a functionally dioecious plant. Females bear flowers that are hermaphroditic in appearance but produce empty pollen, and male flowers have a rudimentary carpel that aborts before style elongation. Because it is dioecious, kiwifruit has an inevitable disadvantage in breeding; paternal parents are selected with unknown fruit quality since male plants cannot bear fruit.<sup><a reveal-id="DSV019C9" data-open="DSV019C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In a typical cross population, male plants represent 50% of the progeny that are a waste of breeding land and resources.<sup><a reveal-id="DSV019C10" data-open="DSV019C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV019C11" data-open="DSV019C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In interspecific breeding programmes for dioecious plants, high-density genetic linkage maps of both maternal and paternal species are one of the most important tools for mapping important agricultural traits and marker assistant selection. However, few interspecific kiwifruit genetic maps have been generated, except a map constructed using microsatellite and amplified fragments length polymorphism (AFLP) markers from a cross between <em>A. chinensis</em> and <em>Actinidia callosa.</em><sup><a reveal-id="DSV019C12" data-open="DSV019C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The sex-determination region (SDR) was mapped to a linkage group in <em>A. callosa</em>, but the relatively low marker density limited the practical use of marker-assisted selection or map-based gene cloning in the interspecific kiwifruit breeding programmes. Subsequently, more saturated genetic maps of <em>A. chinensis</em> were constructed with 644 simple sequence repeats (SSR) markers, and two sex-linked sequence characterized amplified region (SCAR) markers (SmX and SmY) were mapped to a shared linkage group.<sup><a reveal-id="DSV019C13" data-open="DSV019C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV019C14" data-open="DSV019C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The SDR in kiwifruit was mapped to the subtelomere of LG17 using sex-specific SCAR markers,<sup><a reveal-id="DSV019C13" data-open="DSV019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> which corresponded to 25 chromosome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). However, scoring the sex-specific markers SmX and SmY in a population of <em>Actinidia rufa</em> × <em>A. chinensis</em> revealed that they were not robust and amplified poorly across species which limited their utility for breeding. Sex-linked markers can reduce the time, labour, and costs associated with breeding programmes, and facilitate dissecting the sex-determination system.<sup><a reveal-id="DSV019C6" data-open="DSV019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Recently, a high-density genetic map based on single-nucleotide polymorphism (SNP) markers was constructed between <em>A. chinensis</em> ‘Hongyang-MS-01’ (male) and <em>A. chinensis</em> × <em>A. eriantha</em> cv. ‘Jiangshanjiao’ (female) to order scaffolds in the kiwifruit draft genome assembly.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> However, ∼25% of the scaffolds are currently unanchored to the chromosome scale assembly.</p><p>Traditionally genotyping methods were expensive and labour intensive; recent advances in next-generation sequencing technologies have provided new opportunities for detecting a large number of DNA markers rapidly. Restriction-associated DNA (RAD) sequencing can produce dominant markers within the restriction sites and co-dominant markers adjacent to the restriction sites.<sup><a reveal-id="DSV019C16" data-open="DSV019C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Detection of DNA polymorphisms using next-generation RAD sequencing (RAD-seq) is efficient and requires no prior genome sequence knowledge for the species under investigation. Linkage maps for insects,<sup><a reveal-id="DSV019C17" data-open="DSV019C17" class="link link-ref link-reveal xref-bibr">17</a></sup> fungi,<sup><a reveal-id="DSV019C18" data-open="DSV019C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and plants<sup><a reveal-id="DSV019C19" data-open="DSV019C19" class="link link-ref link-reveal xref-bibr">19</a></sup> have been constructed using RAD-seq, with broad applications in most model and non-model organisms.<sup><a reveal-id="DSV019C20" data-open="DSV019C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSV019C21" data-open="DSV019C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Recently, sex-linked SNP markers in pistachio were identified through RAD-seq in an F<sub>1</sub> segregating population, which was beneficial to cost-effective marker-assisted selection in breeding programmes.<sup><a reveal-id="DSV019C6" data-open="DSV019C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Here, we present high-density interspecific kiwifruit genetic maps based on SNP markers using RAD-seq. The high-density genetic maps are beneficial for kiwifruit breeding programmes and improving the kiwifruit draft genome assembly. The three sex-related markers developed in the SDR can accurately distinguish male and female plants, which can be utilized in kiwifruit breeding and commercial production for marker-assisted selection for sex.</p>                            <h2 scrollto-destination=4942855 id="4942855" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4942856 id="4942856" class="section-title" data-legacy-id=s2a>2.1. Plant material and DNA isolation</h3>
<p>An F<sub>1</sub> mapping population was generated by crossing <em>A. rufa</em> ‘MT570001’ and <em>A. chinensis</em> ‘Guihai No4’, and 174 F<sub>1</sub> individuals consisting of 87 male progeny and 87 female progeny were selected for genotyping and mapping. Young leaf tissue of the parents and F<sub>1</sub> individuals was harvested for genomic DNA extraction using the modified cetyltrimethylammonium bromide method.<sup><a reveal-id="DSV019C22" data-open="DSV019C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=4942858 id="4942858" class="section-title" data-legacy-id=s2b>2.2. RAD library preparation and sequencing</h3>
<p>A reduced representation restriction-associated DNA (RAD) sequencing method was used for library construction following the protocol outlined in Zhang et al.<sup><a reveal-id="DSV019C23" data-open="DSV019C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In brief, genomic DNA (1 µg) from each sample was digested for 15 min at 37°C in a 50 ml reaction with 20 units (U) of <em>Eco</em>RI (NEB, USA), which recognized a six-nucleotide sequence (5′G/AATTC3′). A modified Illumina P1 adapter containing specific nucleotide barcodes 4–8 bp long was using for sample tracking. After adapter ligation, individual libraries were pooled to equimolar concentrations with the exception of the parents which had 3× representation. DNA fragments were randomly sheared (Bioruptor Branson sonicator 450) to an average size of 500 bp, and sheared DNA fragments were run out on a 1% agarose gel (Sigma). DNA fragments in the range of 300–500 bp were isolated using the MinElute Gel Extraction Kit (Qiagen). The dsDNA ends end repaired, a 3′-adenine overhangs were added, and a modified Illumina P2 adapter was ligated. Finally, DNA products were PCR-amplified using Phusion Master Mix (NEB, USA), with 4 µM modified Solexa amplification primer mix (Illumina, USA) for 18 cycles. DNA library was sequenced by using the Illumina Hiseq2000 instrumentation.</p>                            <h3 scrollto-destination=4942863 id="4942863" class="section-title" data-legacy-id=s2c>2.3. SNP discovery and genotyping</h3>
<p>Raw sequence reads were segregated by barcodes assigned to individuals and low-quality reads, and those that lacked a correct barcode were removed.<sup><a reveal-id="DSV019C24" data-open="DSV019C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The reads were first assigned to each individual by the unambiguous barcodes, and the reads without the unique barcodes were discarded. Reads were quality-filtered by discarding adapter pollutions and the reads containing &gt;50% low-quality bases (quality value of ≤5) were removing. The quality-filtered RAD reads of each individual were mapped onto the kiwifruit repeat masked genome (Kiwifruit_pseudomolecule.masked.fa)<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with the alignment software SOAP2.<sup><a reveal-id="DSV019C25" data-open="DSV019C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Based on the alignment result, the RAD-based SNP calling was done by SOAPsnp.<sup><a reveal-id="DSV019C26" data-open="DSV019C26" class="link link-ref link-reveal xref-bibr">26</a></sup> To minimize errors in calling markers, the consensus alignments with &lt;5 or &gt;100 sequences were discarded. Polymorphic markers between maternal and paternal parents were identified by pairwise grouping with a copy number of ≤1.5. The ratio of marker segregation was calculated by the <em>χ</em><sup>2</sup> test and markers showing significantly distorted segregation (<em>P</em>-value &lt;0.01) were excluded from the map construction.</p>                            <h3 scrollto-destination=4942865 id="4942865" class="section-title" data-legacy-id=s2d>2.4. Construction of linkage maps</h3>
<p>For linkage analysis, the double pseudo-test cross strategy was applied.<sup><a reveal-id="DSV019C27" data-open="DSV019C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The flower sex phenotype was mapped as a dominant marker ‘Sex-f’. The RAD-seq markers with &gt;20% missing data were removed, then the ‘Sex-f’ marker and the remaining high-quality RAD-seq markers were used to construct a genetic map using the JoinMap 4.0 software with cross pollinated population type codes.<sup><a reveal-id="DSV019C28" data-open="DSV019C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The female and male linkage maps were created by maternal and paternal population nodes, respectively. Markers were assigned to linkage groups with an independence log-of-odds (LOD) score of 8.0. The genetic distances between loci and the recombination rate were calculated with Kosambi's function and the regression mapping algorithm with a recombination frequency threshold of 0.5. Markers that were physically within 100 kb of each other in the two parental maps were used to assess marker co-linearity between the maps. The markers between two parental genetic maps were linked using custom perl scripts, and Circos was used to visualize the collinearity of two parental maps.</p>                            <h3 scrollto-destination=4942867 id="4942867" class="section-title" data-legacy-id=s2e>2.5. Sex-related markers screening and verification</h3>
<p>The assembled kiwifruit genome sequence and gene annotation were downloaded from the GenBank nucleotide core database.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The MIcroSAtellite identification tool (MISA) program was used to identify microsatellite markers in the kiwifruit scaffolds, and dimers to hexamers of at least 22 bp in length were selected; primers were designed based on flank sequence using Primer 3 program. All 40 SSR markers in genic regions were screened in 10 male and 10 female plants from the mapping population, then sex-specific markers were validated by screening against the 174 F1 individuals and further tested in another population and 6 <em>A. chinensis</em> (‘Hongyang’, ‘Cuiyu’, ‘Guihai’, ‘Huaguang No2’, ‘QS-14’, and ‘QS-5’) and 3 <em>A. rufa</em> (‘Shanli 3’, ‘Shanli 4’, and ‘Shanli 5’) germplasm accessions. The PCR products were separated on 6% denaturing polyacrylamide gel. Candidate sex-determination genes were annotated using BLAST with a minimum <em>e</em>-value of 1<em>e</em>−04.</p>                            <h2 scrollto-destination=4942869 id="4942869" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4942870 id="4942870" class="section-title" data-legacy-id=s3a>3.1. RAD-seq and SNP discovering</h3>
<p>A total of 176 RAD-seq libraries of two parents (<em>A. rufa</em> ‘MT570001’ and <em>A. chinensis</em> ‘Guihai No4’) and their 174 F<sub>1</sub> offspring were constructed and sequenced using the Illumina HigSeq 2000 platform. After data trimming and filtering, 2,658 million clean reads remained, representing ∼231 Gb of sequences. The RAD-seq reads were deposited at NCBI with accession number PRJNA290688 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/bioproject/290688">http://www.ncbi.nlm.nih.gov/bioproject/290688</a>). RAD reads were sorted by individual based on the barcode information. The parental libraries had a combined 118 million filtered reads, and the 174 F<sub>1</sub> offspring have a total of 2,540 million filtered reads with an average of 14.6 million reads per progeny. Among sequencing reads, 53.81% reads were mapped to the reference genome, and unique reads accounted for 92.94% of mapping reads. Most of the unmapped reads likely corresponded to repeat regions which were masked in the reference genome to prevent erroneous marker calls. The average sequencing depths of sequenced loci per parent and progeny were 30.35 and 20.25 fold, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). RAD tags from each individual were clustered and called for SNP markers after stringent selection following the protocol described in the Materials and Methods section. In total, 6,783 high-confidence SNPs across the 174 offspring were generated, including 2,479 heterozygous SNPs in <em>A. rufa</em> ‘MT570001’ and 4,304 heterozygous SNPs in <em>A. chinensis</em> ‘Guihai No4’.</p>                            <h3 scrollto-destination=4942872 id="4942872" class="section-title" data-legacy-id=s3b>3.2. Construction of the high-density genetic maps</h3>
<p>We constructed two high-density genetic maps from an interspecific cross of <em>A. rufa</em> ‘MT570001’ <em>×</em><em>A. chinensis</em> ‘Guihai No4’ using JoinMap 4.0, with an independence LOD score of 8.0. The <em>A. rufa</em> ‘MT570001’ (maternal) map consists of 2,426 SNP markers in 29 linkage groups (LGs) corresponding to the 29 haploid chromosomes. The marker names and positions for all loci in the 29 LGs of genetic map are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. The total length of the <em>A. rufa</em> map is 2651.3 cM with an average distance of 1.09 cM between adjacent markers, and the genetic length for each LG ranged from 67.3 to 122.5 cM (Fig. <a reveal-id="DSV019F1" data-open="DSV019F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSV019TB1" data-open="DSV019TB1" class="link link-reveal link-table xref-fig">1</a>). The <em>A. chinensis</em> ‘Guihai No4’ (paternal) map consists of 4,214 SNP markers in 29 LGs. The locus name and genetic map position for all SNP loci in the 29 LGs of genetic map are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. The total length of the <em>A. chinensis</em> map is 3,142.4 cM with an average distance of 0.75 cM between adjacent markers, with linkage groups ranging from 72.4 to 151.7 cM in size (Fig. <a reveal-id="DSV019F2" data-open="DSV019F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSV019TB1" data-open="DSV019TB1" class="link link-reveal link-table xref-fig">1</a>). The ‘Sex-f’ marker was mapped to the telomere of LG25 in <em>A. chinensis</em> map. It is notable that a few markers mapped on linkage groups were not located on the corresponding chromosomes compared with the kiwifruit draft genome.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p>                        <a id="4942874" scrollto-destination="4942874"></a>
<div content-id="DSV019TB1" class="table-modal"><div class="table-wrap"><div id="DSV019TB1" class="table-wrap-title" data-id="DSV019TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of mapping result in <em>A. rufa</em> and <em>A. chinensis</em> genetic maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4"><em>A. rufa</em><hr /> </th><th rowspan="2">Linkage group </th><th colspan="4"><em>A. chinensis</em><hr /> </th></tr><tr><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th></tr></thead><tbody><tr><td>LG1 </td><td>97 </td><td>108.7 </td><td>1.12 </td><td>26 </td><td>LG1 </td><td>148 </td><td>108.7 </td><td>0.73 </td><td>31 </td></tr><tr><td>LG2 </td><td>71 </td><td>77.9 </td><td>1.1 </td><td>1 </td><td>LG2 </td><td>96 </td><td>105.6 </td><td>1.1 </td><td>12 </td></tr><tr><td>LG3 </td><td>134 </td><td>132 </td><td>0.99 </td><td>40 </td><td>LG3 </td><td>270 </td><td>147.6 </td><td>0.55 </td><td>59 </td></tr><tr><td>LG4 </td><td>55 </td><td>67.3 </td><td>1.22 </td><td>1 </td><td>LG4 </td><td>111 </td><td>72.4 </td><td>0.65 </td><td>8 </td></tr><tr><td>LG5 </td><td>74 </td><td>83.2 </td><td>1.12 </td><td>26 </td><td>LG5 </td><td>145 </td><td>88.9 </td><td>0.61 </td><td>21 </td></tr><tr><td>LG6 </td><td>48 </td><td>87.3 </td><td>1.82 </td><td>11 </td><td>LG6 </td><td>153 </td><td>129.4 </td><td>0.85 </td><td>31 </td></tr><tr><td>LG7 </td><td>89 </td><td>90.7 </td><td>1.02 </td><td>11 </td><td>LG7 </td><td>146 </td><td>113.3 </td><td>0.78 </td><td>24 </td></tr><tr><td>LG8 </td><td>77 </td><td>119.2 </td><td>1.55 </td><td>5 </td><td>LG8 </td><td>187 </td><td>136.1 </td><td>0.73 </td><td>35 </td></tr><tr><td>LG9 </td><td>91 </td><td>76.5 </td><td>0.84 </td><td>38 </td><td>LG9 </td><td>146 </td><td>103.6 </td><td>0.71 </td><td>80 </td></tr><tr><td>LG10 </td><td>104 </td><td>86.5 </td><td>0.83 </td><td>84 </td><td>LG10 </td><td>168 </td><td>104.5 </td><td>0.62 </td><td>116 </td></tr><tr><td>LG11 </td><td>53 </td><td>75.4 </td><td>1.42 </td><td>3 </td><td>LG11 </td><td>112 </td><td>95.7 </td><td>0.85 </td><td>19 </td></tr><tr><td>LG12 </td><td>88 </td><td>88.2 </td><td>1 </td><td>26 </td><td>LG12 </td><td>114 </td><td>101.4 </td><td>0.89 </td><td>51 </td></tr><tr><td>LG13 </td><td>86 </td><td>115 </td><td>1.34 </td><td>16 </td><td>LG13 </td><td>158 </td><td>136.4 </td><td>0.86 </td><td>11 </td></tr><tr><td>LG14 </td><td>86 </td><td>103.5 </td><td>1.2 </td><td>56 </td><td>LG14 </td><td>165 </td><td>108.8 </td><td>0.66 </td><td>111 </td></tr><tr><td>LG15 </td><td>73 </td><td>103 </td><td>1.41 </td><td>0 </td><td>LG15 </td><td>169 </td><td>106.5 </td><td>0.63 </td><td>4 </td></tr><tr><td>LG16 </td><td>86 </td><td>73.6 </td><td>0.86 </td><td>34 </td><td>LG16 </td><td>135 </td><td>82.1 </td><td>0.61 </td><td>51 </td></tr><tr><td>LG17 </td><td>59 </td><td>78.7 </td><td>1.33 </td><td>14 </td><td>LG17 </td><td>89 </td><td>101.3 </td><td>1.14 </td><td>33 </td></tr><tr><td>LG18 </td><td>97 </td><td>87.8 </td><td>0.9 </td><td>23 </td><td>LG18 </td><td>82 </td><td>132.9 </td><td>1.62 </td><td>42 </td></tr><tr><td>LG19 </td><td>85 </td><td>105.5 </td><td>1.24 </td><td>21 </td><td>LG19 </td><td>118 </td><td>112.5 </td><td>0.95 </td><td>24 </td></tr><tr><td>LG20 </td><td>108 </td><td>94.7 </td><td>0.88 </td><td>51 </td><td>LG20 </td><td>174 </td><td>112.5 </td><td>0.65 </td><td>78 </td></tr><tr><td>LG21 </td><td>69 </td><td>83.5 </td><td>1.21 </td><td>17 </td><td>LG21 </td><td>116 </td><td>110.1 </td><td>0.95 </td><td>14 </td></tr><tr><td>LG22 </td><td>79 </td><td>74.6 </td><td>0.94 </td><td>39 </td><td>LG22 </td><td>176 </td><td>83.4 </td><td>0.47 </td><td>119 </td></tr><tr><td>LG23 </td><td>104 </td><td>122.5 </td><td>1.18 </td><td>0 </td><td>LG23 </td><td>230 </td><td>151.7 </td><td>0.66 </td><td>8 </td></tr><tr><td>LG24 </td><td>62 </td><td>109.4 </td><td>1.76 </td><td>4 </td><td>LG24 </td><td>136 </td><td>120.4 </td><td>0.89 </td><td>8 </td></tr><tr><td>LG25 </td><td>129 </td><td>89.6 </td><td>0.69 </td><td>58 </td><td>LG25 </td><td>154 </td><td>97.8 </td><td>0.63 </td><td>44 </td></tr><tr><td>LG26 </td><td>89 </td><td>83.7 </td><td>0.94 </td><td>20 </td><td>LG26 </td><td>176 </td><td>99 </td><td>0.56 </td><td>32 </td></tr><tr><td>LG27 </td><td>71 </td><td>74.8 </td><td>1.05 </td><td>27 </td><td>LG27 </td><td>104 </td><td>78.9 </td><td>0.76 </td><td>45 </td></tr><tr><td>LG28 </td><td>80 </td><td>83.3 </td><td>1.04 </td><td>18 </td><td>LG28 </td><td>107 </td><td>92.2 </td><td>0.86 </td><td>17 </td></tr><tr><td>LG29 </td><td>82 </td><td>75.3 </td><td>0.92 </td><td>36 </td><td>LG29 </td><td>129 </td><td>108.7 </td><td>0.84 </td><td>59 </td></tr><tr><td>Total </td><td>2,426 </td><td>2651.3 </td><td>1.09 </td><td>706 </td><td>Total </td><td>4,214 </td><td>3142.4 </td><td>0.75 </td><td>1,187 </td></tr></tbody></table></div><div content-id="DSV019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4"><em>A. rufa</em><hr /> </th><th rowspan="2">Linkage group </th><th colspan="4"><em>A. chinensis</em><hr /> </th></tr><tr><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th></tr></thead><tbody><tr><td>LG1 </td><td>97 </td><td>108.7 </td><td>1.12 </td><td>26 </td><td>LG1 </td><td>148 </td><td>108.7 </td><td>0.73 </td><td>31 </td></tr><tr><td>LG2 </td><td>71 </td><td>77.9 </td><td>1.1 </td><td>1 </td><td>LG2 </td><td>96 </td><td>105.6 </td><td>1.1 </td><td>12 </td></tr><tr><td>LG3 </td><td>134 </td><td>132 </td><td>0.99 </td><td>40 </td><td>LG3 </td><td>270 </td><td>147.6 </td><td>0.55 </td><td>59 </td></tr><tr><td>LG4 </td><td>55 </td><td>67.3 </td><td>1.22 </td><td>1 </td><td>LG4 </td><td>111 </td><td>72.4 </td><td>0.65 </td><td>8 </td></tr><tr><td>LG5 </td><td>74 </td><td>83.2 </td><td>1.12 </td><td>26 </td><td>LG5 </td><td>145 </td><td>88.9 </td><td>0.61 </td><td>21 </td></tr><tr><td>LG6 </td><td>48 </td><td>87.3 </td><td>1.82 </td><td>11 </td><td>LG6 </td><td>153 </td><td>129.4 </td><td>0.85 </td><td>31 </td></tr><tr><td>LG7 </td><td>89 </td><td>90.7 </td><td>1.02 </td><td>11 </td><td>LG7 </td><td>146 </td><td>113.3 </td><td>0.78 </td><td>24 </td></tr><tr><td>LG8 </td><td>77 </td><td>119.2 </td><td>1.55 </td><td>5 </td><td>LG8 </td><td>187 </td><td>136.1 </td><td>0.73 </td><td>35 </td></tr><tr><td>LG9 </td><td>91 </td><td>76.5 </td><td>0.84 </td><td>38 </td><td>LG9 </td><td>146 </td><td>103.6 </td><td>0.71 </td><td>80 </td></tr><tr><td>LG10 </td><td>104 </td><td>86.5 </td><td>0.83 </td><td>84 </td><td>LG10 </td><td>168 </td><td>104.5 </td><td>0.62 </td><td>116 </td></tr><tr><td>LG11 </td><td>53 </td><td>75.4 </td><td>1.42 </td><td>3 </td><td>LG11 </td><td>112 </td><td>95.7 </td><td>0.85 </td><td>19 </td></tr><tr><td>LG12 </td><td>88 </td><td>88.2 </td><td>1 </td><td>26 </td><td>LG12 </td><td>114 </td><td>101.4 </td><td>0.89 </td><td>51 </td></tr><tr><td>LG13 </td><td>86 </td><td>115 </td><td>1.34 </td><td>16 </td><td>LG13 </td><td>158 </td><td>136.4 </td><td>0.86 </td><td>11 </td></tr><tr><td>LG14 </td><td>86 </td><td>103.5 </td><td>1.2 </td><td>56 </td><td>LG14 </td><td>165 </td><td>108.8 </td><td>0.66 </td><td>111 </td></tr><tr><td>LG15 </td><td>73 </td><td>103 </td><td>1.41 </td><td>0 </td><td>LG15 </td><td>169 </td><td>106.5 </td><td>0.63 </td><td>4 </td></tr><tr><td>LG16 </td><td>86 </td><td>73.6 </td><td>0.86 </td><td>34 </td><td>LG16 </td><td>135 </td><td>82.1 </td><td>0.61 </td><td>51 </td></tr><tr><td>LG17 </td><td>59 </td><td>78.7 </td><td>1.33 </td><td>14 </td><td>LG17 </td><td>89 </td><td>101.3 </td><td>1.14 </td><td>33 </td></tr><tr><td>LG18 </td><td>97 </td><td>87.8 </td><td>0.9 </td><td>23 </td><td>LG18 </td><td>82 </td><td>132.9 </td><td>1.62 </td><td>42 </td></tr><tr><td>LG19 </td><td>85 </td><td>105.5 </td><td>1.24 </td><td>21 </td><td>LG19 </td><td>118 </td><td>112.5 </td><td>0.95 </td><td>24 </td></tr><tr><td>LG20 </td><td>108 </td><td>94.7 </td><td>0.88 </td><td>51 </td><td>LG20 </td><td>174 </td><td>112.5 </td><td>0.65 </td><td>78 </td></tr><tr><td>LG21 </td><td>69 </td><td>83.5 </td><td>1.21 </td><td>17 </td><td>LG21 </td><td>116 </td><td>110.1 </td><td>0.95 </td><td>14 </td></tr><tr><td>LG22 </td><td>79 </td><td>74.6 </td><td>0.94 </td><td>39 </td><td>LG22 </td><td>176 </td><td>83.4 </td><td>0.47 </td><td>119 </td></tr><tr><td>LG23 </td><td>104 </td><td>122.5 </td><td>1.18 </td><td>0 </td><td>LG23 </td><td>230 </td><td>151.7 </td><td>0.66 </td><td>8 </td></tr><tr><td>LG24 </td><td>62 </td><td>109.4 </td><td>1.76 </td><td>4 </td><td>LG24 </td><td>136 </td><td>120.4 </td><td>0.89 </td><td>8 </td></tr><tr><td>LG25 </td><td>129 </td><td>89.6 </td><td>0.69 </td><td>58 </td><td>LG25 </td><td>154 </td><td>97.8 </td><td>0.63 </td><td>44 </td></tr><tr><td>LG26 </td><td>89 </td><td>83.7 </td><td>0.94 </td><td>20 </td><td>LG26 </td><td>176 </td><td>99 </td><td>0.56 </td><td>32 </td></tr><tr><td>LG27 </td><td>71 </td><td>74.8 </td><td>1.05 </td><td>27 </td><td>LG27 </td><td>104 </td><td>78.9 </td><td>0.76 </td><td>45 </td></tr><tr><td>LG28 </td><td>80 </td><td>83.3 </td><td>1.04 </td><td>18 </td><td>LG28 </td><td>107 </td><td>92.2 </td><td>0.86 </td><td>17 </td></tr><tr><td>LG29 </td><td>82 </td><td>75.3 </td><td>0.92 </td><td>36 </td><td>LG29 </td><td>129 </td><td>108.7 </td><td>0.84 </td><td>59 </td></tr><tr><td>Total </td><td>2,426 </td><td>2651.3 </td><td>1.09 </td><td>706 </td><td>Total </td><td>4,214 </td><td>3142.4 </td><td>0.75 </td><td>1,187 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942874" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV019TB1" class="table-wrap-title" data-id="DSV019TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of mapping result in <em>A. rufa</em> and <em>A. chinensis</em> genetic maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4"><em>A. rufa</em><hr /> </th><th rowspan="2">Linkage group </th><th colspan="4"><em>A. chinensis</em><hr /> </th></tr><tr><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th></tr></thead><tbody><tr><td>LG1 </td><td>97 </td><td>108.7 </td><td>1.12 </td><td>26 </td><td>LG1 </td><td>148 </td><td>108.7 </td><td>0.73 </td><td>31 </td></tr><tr><td>LG2 </td><td>71 </td><td>77.9 </td><td>1.1 </td><td>1 </td><td>LG2 </td><td>96 </td><td>105.6 </td><td>1.1 </td><td>12 </td></tr><tr><td>LG3 </td><td>134 </td><td>132 </td><td>0.99 </td><td>40 </td><td>LG3 </td><td>270 </td><td>147.6 </td><td>0.55 </td><td>59 </td></tr><tr><td>LG4 </td><td>55 </td><td>67.3 </td><td>1.22 </td><td>1 </td><td>LG4 </td><td>111 </td><td>72.4 </td><td>0.65 </td><td>8 </td></tr><tr><td>LG5 </td><td>74 </td><td>83.2 </td><td>1.12 </td><td>26 </td><td>LG5 </td><td>145 </td><td>88.9 </td><td>0.61 </td><td>21 </td></tr><tr><td>LG6 </td><td>48 </td><td>87.3 </td><td>1.82 </td><td>11 </td><td>LG6 </td><td>153 </td><td>129.4 </td><td>0.85 </td><td>31 </td></tr><tr><td>LG7 </td><td>89 </td><td>90.7 </td><td>1.02 </td><td>11 </td><td>LG7 </td><td>146 </td><td>113.3 </td><td>0.78 </td><td>24 </td></tr><tr><td>LG8 </td><td>77 </td><td>119.2 </td><td>1.55 </td><td>5 </td><td>LG8 </td><td>187 </td><td>136.1 </td><td>0.73 </td><td>35 </td></tr><tr><td>LG9 </td><td>91 </td><td>76.5 </td><td>0.84 </td><td>38 </td><td>LG9 </td><td>146 </td><td>103.6 </td><td>0.71 </td><td>80 </td></tr><tr><td>LG10 </td><td>104 </td><td>86.5 </td><td>0.83 </td><td>84 </td><td>LG10 </td><td>168 </td><td>104.5 </td><td>0.62 </td><td>116 </td></tr><tr><td>LG11 </td><td>53 </td><td>75.4 </td><td>1.42 </td><td>3 </td><td>LG11 </td><td>112 </td><td>95.7 </td><td>0.85 </td><td>19 </td></tr><tr><td>LG12 </td><td>88 </td><td>88.2 </td><td>1 </td><td>26 </td><td>LG12 </td><td>114 </td><td>101.4 </td><td>0.89 </td><td>51 </td></tr><tr><td>LG13 </td><td>86 </td><td>115 </td><td>1.34 </td><td>16 </td><td>LG13 </td><td>158 </td><td>136.4 </td><td>0.86 </td><td>11 </td></tr><tr><td>LG14 </td><td>86 </td><td>103.5 </td><td>1.2 </td><td>56 </td><td>LG14 </td><td>165 </td><td>108.8 </td><td>0.66 </td><td>111 </td></tr><tr><td>LG15 </td><td>73 </td><td>103 </td><td>1.41 </td><td>0 </td><td>LG15 </td><td>169 </td><td>106.5 </td><td>0.63 </td><td>4 </td></tr><tr><td>LG16 </td><td>86 </td><td>73.6 </td><td>0.86 </td><td>34 </td><td>LG16 </td><td>135 </td><td>82.1 </td><td>0.61 </td><td>51 </td></tr><tr><td>LG17 </td><td>59 </td><td>78.7 </td><td>1.33 </td><td>14 </td><td>LG17 </td><td>89 </td><td>101.3 </td><td>1.14 </td><td>33 </td></tr><tr><td>LG18 </td><td>97 </td><td>87.8 </td><td>0.9 </td><td>23 </td><td>LG18 </td><td>82 </td><td>132.9 </td><td>1.62 </td><td>42 </td></tr><tr><td>LG19 </td><td>85 </td><td>105.5 </td><td>1.24 </td><td>21 </td><td>LG19 </td><td>118 </td><td>112.5 </td><td>0.95 </td><td>24 </td></tr><tr><td>LG20 </td><td>108 </td><td>94.7 </td><td>0.88 </td><td>51 </td><td>LG20 </td><td>174 </td><td>112.5 </td><td>0.65 </td><td>78 </td></tr><tr><td>LG21 </td><td>69 </td><td>83.5 </td><td>1.21 </td><td>17 </td><td>LG21 </td><td>116 </td><td>110.1 </td><td>0.95 </td><td>14 </td></tr><tr><td>LG22 </td><td>79 </td><td>74.6 </td><td>0.94 </td><td>39 </td><td>LG22 </td><td>176 </td><td>83.4 </td><td>0.47 </td><td>119 </td></tr><tr><td>LG23 </td><td>104 </td><td>122.5 </td><td>1.18 </td><td>0 </td><td>LG23 </td><td>230 </td><td>151.7 </td><td>0.66 </td><td>8 </td></tr><tr><td>LG24 </td><td>62 </td><td>109.4 </td><td>1.76 </td><td>4 </td><td>LG24 </td><td>136 </td><td>120.4 </td><td>0.89 </td><td>8 </td></tr><tr><td>LG25 </td><td>129 </td><td>89.6 </td><td>0.69 </td><td>58 </td><td>LG25 </td><td>154 </td><td>97.8 </td><td>0.63 </td><td>44 </td></tr><tr><td>LG26 </td><td>89 </td><td>83.7 </td><td>0.94 </td><td>20 </td><td>LG26 </td><td>176 </td><td>99 </td><td>0.56 </td><td>32 </td></tr><tr><td>LG27 </td><td>71 </td><td>74.8 </td><td>1.05 </td><td>27 </td><td>LG27 </td><td>104 </td><td>78.9 </td><td>0.76 </td><td>45 </td></tr><tr><td>LG28 </td><td>80 </td><td>83.3 </td><td>1.04 </td><td>18 </td><td>LG28 </td><td>107 </td><td>92.2 </td><td>0.86 </td><td>17 </td></tr><tr><td>LG29 </td><td>82 </td><td>75.3 </td><td>0.92 </td><td>36 </td><td>LG29 </td><td>129 </td><td>108.7 </td><td>0.84 </td><td>59 </td></tr><tr><td>Total </td><td>2,426 </td><td>2651.3 </td><td>1.09 </td><td>706 </td><td>Total </td><td>4,214 </td><td>3142.4 </td><td>0.75 </td><td>1,187 </td></tr></tbody></table></div><div content-id="DSV019TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4"><em>A. rufa</em><hr /> </th><th rowspan="2">Linkage group </th><th colspan="4"><em>A. chinensis</em><hr /> </th></tr><tr><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th><th>Marker no. </th><th>Size (cM) </th><th>Average distance (cM) </th><th>Marker on unanchored scaffolds </th></tr></thead><tbody><tr><td>LG1 </td><td>97 </td><td>108.7 </td><td>1.12 </td><td>26 </td><td>LG1 </td><td>148 </td><td>108.7 </td><td>0.73 </td><td>31 </td></tr><tr><td>LG2 </td><td>71 </td><td>77.9 </td><td>1.1 </td><td>1 </td><td>LG2 </td><td>96 </td><td>105.6 </td><td>1.1 </td><td>12 </td></tr><tr><td>LG3 </td><td>134 </td><td>132 </td><td>0.99 </td><td>40 </td><td>LG3 </td><td>270 </td><td>147.6 </td><td>0.55 </td><td>59 </td></tr><tr><td>LG4 </td><td>55 </td><td>67.3 </td><td>1.22 </td><td>1 </td><td>LG4 </td><td>111 </td><td>72.4 </td><td>0.65 </td><td>8 </td></tr><tr><td>LG5 </td><td>74 </td><td>83.2 </td><td>1.12 </td><td>26 </td><td>LG5 </td><td>145 </td><td>88.9 </td><td>0.61 </td><td>21 </td></tr><tr><td>LG6 </td><td>48 </td><td>87.3 </td><td>1.82 </td><td>11 </td><td>LG6 </td><td>153 </td><td>129.4 </td><td>0.85 </td><td>31 </td></tr><tr><td>LG7 </td><td>89 </td><td>90.7 </td><td>1.02 </td><td>11 </td><td>LG7 </td><td>146 </td><td>113.3 </td><td>0.78 </td><td>24 </td></tr><tr><td>LG8 </td><td>77 </td><td>119.2 </td><td>1.55 </td><td>5 </td><td>LG8 </td><td>187 </td><td>136.1 </td><td>0.73 </td><td>35 </td></tr><tr><td>LG9 </td><td>91 </td><td>76.5 </td><td>0.84 </td><td>38 </td><td>LG9 </td><td>146 </td><td>103.6 </td><td>0.71 </td><td>80 </td></tr><tr><td>LG10 </td><td>104 </td><td>86.5 </td><td>0.83 </td><td>84 </td><td>LG10 </td><td>168 </td><td>104.5 </td><td>0.62 </td><td>116 </td></tr><tr><td>LG11 </td><td>53 </td><td>75.4 </td><td>1.42 </td><td>3 </td><td>LG11 </td><td>112 </td><td>95.7 </td><td>0.85 </td><td>19 </td></tr><tr><td>LG12 </td><td>88 </td><td>88.2 </td><td>1 </td><td>26 </td><td>LG12 </td><td>114 </td><td>101.4 </td><td>0.89 </td><td>51 </td></tr><tr><td>LG13 </td><td>86 </td><td>115 </td><td>1.34 </td><td>16 </td><td>LG13 </td><td>158 </td><td>136.4 </td><td>0.86 </td><td>11 </td></tr><tr><td>LG14 </td><td>86 </td><td>103.5 </td><td>1.2 </td><td>56 </td><td>LG14 </td><td>165 </td><td>108.8 </td><td>0.66 </td><td>111 </td></tr><tr><td>LG15 </td><td>73 </td><td>103 </td><td>1.41 </td><td>0 </td><td>LG15 </td><td>169 </td><td>106.5 </td><td>0.63 </td><td>4 </td></tr><tr><td>LG16 </td><td>86 </td><td>73.6 </td><td>0.86 </td><td>34 </td><td>LG16 </td><td>135 </td><td>82.1 </td><td>0.61 </td><td>51 </td></tr><tr><td>LG17 </td><td>59 </td><td>78.7 </td><td>1.33 </td><td>14 </td><td>LG17 </td><td>89 </td><td>101.3 </td><td>1.14 </td><td>33 </td></tr><tr><td>LG18 </td><td>97 </td><td>87.8 </td><td>0.9 </td><td>23 </td><td>LG18 </td><td>82 </td><td>132.9 </td><td>1.62 </td><td>42 </td></tr><tr><td>LG19 </td><td>85 </td><td>105.5 </td><td>1.24 </td><td>21 </td><td>LG19 </td><td>118 </td><td>112.5 </td><td>0.95 </td><td>24 </td></tr><tr><td>LG20 </td><td>108 </td><td>94.7 </td><td>0.88 </td><td>51 </td><td>LG20 </td><td>174 </td><td>112.5 </td><td>0.65 </td><td>78 </td></tr><tr><td>LG21 </td><td>69 </td><td>83.5 </td><td>1.21 </td><td>17 </td><td>LG21 </td><td>116 </td><td>110.1 </td><td>0.95 </td><td>14 </td></tr><tr><td>LG22 </td><td>79 </td><td>74.6 </td><td>0.94 </td><td>39 </td><td>LG22 </td><td>176 </td><td>83.4 </td><td>0.47 </td><td>119 </td></tr><tr><td>LG23 </td><td>104 </td><td>122.5 </td><td>1.18 </td><td>0 </td><td>LG23 </td><td>230 </td><td>151.7 </td><td>0.66 </td><td>8 </td></tr><tr><td>LG24 </td><td>62 </td><td>109.4 </td><td>1.76 </td><td>4 </td><td>LG24 </td><td>136 </td><td>120.4 </td><td>0.89 </td><td>8 </td></tr><tr><td>LG25 </td><td>129 </td><td>89.6 </td><td>0.69 </td><td>58 </td><td>LG25 </td><td>154 </td><td>97.8 </td><td>0.63 </td><td>44 </td></tr><tr><td>LG26 </td><td>89 </td><td>83.7 </td><td>0.94 </td><td>20 </td><td>LG26 </td><td>176 </td><td>99 </td><td>0.56 </td><td>32 </td></tr><tr><td>LG27 </td><td>71 </td><td>74.8 </td><td>1.05 </td><td>27 </td><td>LG27 </td><td>104 </td><td>78.9 </td><td>0.76 </td><td>45 </td></tr><tr><td>LG28 </td><td>80 </td><td>83.3 </td><td>1.04 </td><td>18 </td><td>LG28 </td><td>107 </td><td>92.2 </td><td>0.86 </td><td>17 </td></tr><tr><td>LG29 </td><td>82 </td><td>75.3 </td><td>0.92 </td><td>36 </td><td>LG29 </td><td>129 </td><td>108.7 </td><td>0.84 </td><td>59 </td></tr><tr><td>Total </td><td>2,426 </td><td>2651.3 </td><td>1.09 </td><td>706 </td><td>Total </td><td>4,214 </td><td>3142.4 </td><td>0.75 </td><td>1,187 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942874" target="_blank">View Large</a></div></div>                        <a id="4942875" scrollto-destination="4942875"></a>
<div id="DSV019F1" data-id="DSV019F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01901.png?Expires=2147483647&Signature=K94y2zPIB5E-RCrFTeI4Z6QHUizb6vg2XjOs5QRvsfJQSH7B5eqZs3cbghIl-5sx~-NHx3l0NCD29~SPpMklax3YLvNCPzzYN20LX0oFYMGiz~DQ~VYX3DYI12XqOvMjvyoNZTgZ1iqBlGJ8jSbjWQ1p3rpyIMupC4TQ~6OaCIQiu52vJFW2k17hEh0gJlxjGoEv-r-DCPOVT8SYwGAeTCFJVZxMbJQ7P2ecgTanAHPz5PTeOAmxIxxvAFNqpHxIF8hIiUa30LuRbSuV~4xJPErxH3xEvdfBUvaiNLQDwxIliwOhUlFFlv2Nh1xyAsm1rYL2Y7xMHReSLS8SWEzdDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The linkage groups of A. rufa. Genetic distance was shown by the vertical scale line with centimorgans (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942875/dsv01901.tiff" path-from-xml="dsv01901.tiff" target="_blank">View large</a><a section="4942875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01901.png?Expires=2147483647&Signature=vtwiVyxfvffBLD5IVQs3EIxd0NwkISjK3qxERXqz0FZzcZGCSuFuF6LsUrFsgfsICvaxmm6B7m4riX1ih8YOxrtNNzYHXwWgE8tfZq1zPlRBBms5LRrdnNcQKiNh~5pfbf2psthMwpNjRzvNQB4D6n1x2PPGBjmsQBnT3mXQddCiTt2W1oMcdIv9aWv5INjP9cfDzcu46~fK6tc5qPF2l5p9pJV1ETj1McSvpYk0~v9KRmHeNX~1KKWufSchB7tSiRfjTL1jO89UXo148SteeYauaU0cTqCx8uocmj6Ud5Dh7a-YT~pvVX7nyCVdj4mekcnq1iOeOw7Pr6YYmv7jyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942875&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The linkage groups of <em>A. rufa.</em> Genetic distance was shown by the vertical scale line with centimorgans (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV019F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01901.png?Expires=2147483647&Signature=K94y2zPIB5E-RCrFTeI4Z6QHUizb6vg2XjOs5QRvsfJQSH7B5eqZs3cbghIl-5sx~-NHx3l0NCD29~SPpMklax3YLvNCPzzYN20LX0oFYMGiz~DQ~VYX3DYI12XqOvMjvyoNZTgZ1iqBlGJ8jSbjWQ1p3rpyIMupC4TQ~6OaCIQiu52vJFW2k17hEh0gJlxjGoEv-r-DCPOVT8SYwGAeTCFJVZxMbJQ7P2ecgTanAHPz5PTeOAmxIxxvAFNqpHxIF8hIiUa30LuRbSuV~4xJPErxH3xEvdfBUvaiNLQDwxIliwOhUlFFlv2Nh1xyAsm1rYL2Y7xMHReSLS8SWEzdDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The linkage groups of A. rufa. Genetic distance was shown by the vertical scale line with centimorgans (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942875/dsv01901.tiff" path-from-xml="dsv01901.tiff" target="_blank">View large</a><a section="4942875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01901.png?Expires=2147483647&Signature=vtwiVyxfvffBLD5IVQs3EIxd0NwkISjK3qxERXqz0FZzcZGCSuFuF6LsUrFsgfsICvaxmm6B7m4riX1ih8YOxrtNNzYHXwWgE8tfZq1zPlRBBms5LRrdnNcQKiNh~5pfbf2psthMwpNjRzvNQB4D6n1x2PPGBjmsQBnT3mXQddCiTt2W1oMcdIv9aWv5INjP9cfDzcu46~fK6tc5qPF2l5p9pJV1ETj1McSvpYk0~v9KRmHeNX~1KKWufSchB7tSiRfjTL1jO89UXo148SteeYauaU0cTqCx8uocmj6Ud5Dh7a-YT~pvVX7nyCVdj4mekcnq1iOeOw7Pr6YYmv7jyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942875&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The linkage groups of <em>A. rufa.</em> Genetic distance was shown by the vertical scale line with centimorgans (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="4942876" scrollto-destination="4942876"></a>
<div id="DSV019F2" data-id="DSV019F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01902.png?Expires=2147483647&Signature=fqOxYrCPjRkNyul0cBvYk~3orcbI6PfXNIhnfifJVRu1I1133-Rt318oyQh3-BIZ8joHcVqUsx6UK5y4XezduykiUmpEkh7qWpm7VG1y2i3ZxyEfVB7mSQvSv5tFvs7bYQ82XvWFto2SCFW1dyUR~IclYVZQtRdn2RbuC4fffk1oro-2UpxM-WvXjPjBETm~4x37Anb9IRYOacIEacHsER7e7t8HZilnmoh43e-q4wN4Iy50wHn1eepm41IyNV1L6taJsMkpzw8dawfhdK8qPKLO3~D-d6uwdJ2Wr9qQJAK7NunPJvP0j2sa6PYz84XyfmJtanUnAgbFihMf5zxHJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The linkage groups of A. chinensis. Genetic distance was shown by the vertical scale line with centimorgan (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942876/dsv01902.tiff" path-from-xml="dsv01902.tiff" target="_blank">View large</a><a section="4942876" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01902.png?Expires=2147483647&Signature=P4RRQOS0GlQf0UlTEeNRiRK5fCKwCOtrpqPyVCCKiQyYAWH-osZqwPRLhzVqd1A4Q0p9g~4yOK2TWLOfRK4jOq5NGnFipTbJnd3-1uJtlWoaUJjsjuF1~7lQefA5r9qm-xLu4xvQFF0O6~~sEwP4JRXbLNTvla~8nOskhlk11~mS~TxthdVK9hpXmtK~E~fOySErWcM7omWzOiohm58G7nr~oPNGRQ8xHQ3opWjaF5xZcGwvkv2iUca9ZLJfSe61azeIIfVh6gxqJC8thalchHcYOqy3u67WlyRPaGElwXaZciBUFScHwOdQbK9vvoI6pQtJ9dtYzNz8daAMUVrL3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942876&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The linkage groups of <em>A. chinensis</em>. Genetic distance was shown by the vertical scale line with centimorgan (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV019F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01902.png?Expires=2147483647&Signature=fqOxYrCPjRkNyul0cBvYk~3orcbI6PfXNIhnfifJVRu1I1133-Rt318oyQh3-BIZ8joHcVqUsx6UK5y4XezduykiUmpEkh7qWpm7VG1y2i3ZxyEfVB7mSQvSv5tFvs7bYQ82XvWFto2SCFW1dyUR~IclYVZQtRdn2RbuC4fffk1oro-2UpxM-WvXjPjBETm~4x37Anb9IRYOacIEacHsER7e7t8HZilnmoh43e-q4wN4Iy50wHn1eepm41IyNV1L6taJsMkpzw8dawfhdK8qPKLO3~D-d6uwdJ2Wr9qQJAK7NunPJvP0j2sa6PYz84XyfmJtanUnAgbFihMf5zxHJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The linkage groups of A. chinensis. Genetic distance was shown by the vertical scale line with centimorgan (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942876/dsv01902.tiff" path-from-xml="dsv01902.tiff" target="_blank">View large</a><a section="4942876" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01902.png?Expires=2147483647&Signature=P4RRQOS0GlQf0UlTEeNRiRK5fCKwCOtrpqPyVCCKiQyYAWH-osZqwPRLhzVqd1A4Q0p9g~4yOK2TWLOfRK4jOq5NGnFipTbJnd3-1uJtlWoaUJjsjuF1~7lQefA5r9qm-xLu4xvQFF0O6~~sEwP4JRXbLNTvla~8nOskhlk11~mS~TxthdVK9hpXmtK~E~fOySErWcM7omWzOiohm58G7nr~oPNGRQ8xHQ3opWjaF5xZcGwvkv2iUca9ZLJfSe61azeIIfVh6gxqJC8thalchHcYOqy3u67WlyRPaGElwXaZciBUFScHwOdQbK9vvoI6pQtJ9dtYzNz8daAMUVrL3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942876&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The linkage groups of <em>A. chinensis</em>. Genetic distance was shown by the vertical scale line with centimorgan (cM). The 29 linkage groups are displayed by the vertical bars with lines in each linkage group indicating a marker position. Black lines represented markers from assembled chromosomes sequences, and red lines represented markers from unassembled scaffold sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The high quality and density of the genetic maps allowed us to anchor and orient a number of scaffolds, which were previously unanchored in the kiwifruit draft genome. Among unmapped scaffolds, 228 scaffolds were anchored by 694 markers from <em>A. rufa</em> map, and 373 scaffolds were anchored by 1,187 markers from <em>A. chinensis</em> map. In total, 440 newly anchored genome scaffolds with 101 Mb were covered by the markers of the two genetic maps. The scaffold information of markers allowed us to assess the collinearity and accuracy of the genetic maps as well as integration of the two maps. The markers used to access collinearity were based on physical distances, including those mapped on conflicting chromosomes. The kiwifruit genetic maps are largely collinear, with all 29 LGs having shared scaffolds or genome sequences between the two maps (Figs <a reveal-id="DSV019F3" data-open="DSV019F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DSV019F4" data-open="DSV019F4" class="link link-reveal link-table xref-fig">4</a> and Table <a reveal-id="DSV019TB2" data-open="DSV019TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="4942878" scrollto-destination="4942878"></a>
<div content-id="DSV019TB2" class="table-modal"><div class="table-wrap"><div id="DSV019TB2" class="table-wrap-title" data-id="DSV019TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Integration of two parental genetic maps using anchored markers on scaffolds or genome sequences</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>A. rufa</em> </th><th rowspan="2">No. of anchored markers </th><th><em>A. chinensis</em> </th></tr><tr><th>Linkage group </th><th>Linkage group </th></tr></thead><tbody><tr><td>1 </td><td>126 </td><td>1 </td></tr><tr><td>2 </td><td>54 </td><td>2 </td></tr><tr><td>3 </td><td>304 </td><td>3 </td></tr><tr><td>4 </td><td>102 </td><td>4 </td></tr><tr><td>5 </td><td>93 </td><td>5 </td></tr><tr><td>6 </td><td>65 </td><td>6 </td></tr><tr><td>7 </td><td>133 </td><td>7 </td></tr><tr><td>8 </td><td>127 </td><td>8 </td></tr><tr><td>9 </td><td>160 </td><td>9 </td></tr><tr><td>10 </td><td>209 </td><td>10 </td></tr><tr><td>11 </td><td>76 </td><td>11 </td></tr><tr><td>12 </td><td>110 </td><td>12 </td></tr><tr><td>13 </td><td>128 </td><td>13 </td></tr><tr><td>14 </td><td>205 </td><td>14 </td></tr><tr><td>15 </td><td>144 </td><td>15 </td></tr><tr><td>16 </td><td>147 </td><td>16 </td></tr><tr><td>17 </td><td>62 </td><td>17 </td></tr><tr><td>18 </td><td>90 </td><td>18 </td></tr><tr><td>19 </td><td>92 </td><td>19 </td></tr><tr><td>20 </td><td>282 </td><td>20 </td></tr><tr><td>21 </td><td>74 </td><td>21 </td></tr><tr><td>22 </td><td>125 </td><td>22 </td></tr><tr><td>23 </td><td>214 </td><td>23 </td></tr><tr><td>24 </td><td>87 </td><td>24 </td></tr><tr><td>25 </td><td>210 </td><td>25 </td></tr><tr><td>26 </td><td>202 </td><td>26 </td></tr><tr><td>27 </td><td>104 </td><td>27 </td></tr><tr><td>28 </td><td>71 </td><td>28 </td></tr><tr><td>29 </td><td>90 </td><td>29 </td></tr></tbody></table></div><div content-id="DSV019TB2" class="table-modal"><table><thead name="thead" align=""><tr><th><em>A. rufa</em> </th><th rowspan="2">No. of anchored markers </th><th><em>A. chinensis</em> </th></tr><tr><th>Linkage group </th><th>Linkage group </th></tr></thead><tbody><tr><td>1 </td><td>126 </td><td>1 </td></tr><tr><td>2 </td><td>54 </td><td>2 </td></tr><tr><td>3 </td><td>304 </td><td>3 </td></tr><tr><td>4 </td><td>102 </td><td>4 </td></tr><tr><td>5 </td><td>93 </td><td>5 </td></tr><tr><td>6 </td><td>65 </td><td>6 </td></tr><tr><td>7 </td><td>133 </td><td>7 </td></tr><tr><td>8 </td><td>127 </td><td>8 </td></tr><tr><td>9 </td><td>160 </td><td>9 </td></tr><tr><td>10 </td><td>209 </td><td>10 </td></tr><tr><td>11 </td><td>76 </td><td>11 </td></tr><tr><td>12 </td><td>110 </td><td>12 </td></tr><tr><td>13 </td><td>128 </td><td>13 </td></tr><tr><td>14 </td><td>205 </td><td>14 </td></tr><tr><td>15 </td><td>144 </td><td>15 </td></tr><tr><td>16 </td><td>147 </td><td>16 </td></tr><tr><td>17 </td><td>62 </td><td>17 </td></tr><tr><td>18 </td><td>90 </td><td>18 </td></tr><tr><td>19 </td><td>92 </td><td>19 </td></tr><tr><td>20 </td><td>282 </td><td>20 </td></tr><tr><td>21 </td><td>74 </td><td>21 </td></tr><tr><td>22 </td><td>125 </td><td>22 </td></tr><tr><td>23 </td><td>214 </td><td>23 </td></tr><tr><td>24 </td><td>87 </td><td>24 </td></tr><tr><td>25 </td><td>210 </td><td>25 </td></tr><tr><td>26 </td><td>202 </td><td>26 </td></tr><tr><td>27 </td><td>104 </td><td>27 </td></tr><tr><td>28 </td><td>71 </td><td>28 </td></tr><tr><td>29 </td><td>90 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942878" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV019TB2" class="table-wrap-title" data-id="DSV019TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Integration of two parental genetic maps using anchored markers on scaffolds or genome sequences</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>A. rufa</em> </th><th rowspan="2">No. of anchored markers </th><th><em>A. chinensis</em> </th></tr><tr><th>Linkage group </th><th>Linkage group </th></tr></thead><tbody><tr><td>1 </td><td>126 </td><td>1 </td></tr><tr><td>2 </td><td>54 </td><td>2 </td></tr><tr><td>3 </td><td>304 </td><td>3 </td></tr><tr><td>4 </td><td>102 </td><td>4 </td></tr><tr><td>5 </td><td>93 </td><td>5 </td></tr><tr><td>6 </td><td>65 </td><td>6 </td></tr><tr><td>7 </td><td>133 </td><td>7 </td></tr><tr><td>8 </td><td>127 </td><td>8 </td></tr><tr><td>9 </td><td>160 </td><td>9 </td></tr><tr><td>10 </td><td>209 </td><td>10 </td></tr><tr><td>11 </td><td>76 </td><td>11 </td></tr><tr><td>12 </td><td>110 </td><td>12 </td></tr><tr><td>13 </td><td>128 </td><td>13 </td></tr><tr><td>14 </td><td>205 </td><td>14 </td></tr><tr><td>15 </td><td>144 </td><td>15 </td></tr><tr><td>16 </td><td>147 </td><td>16 </td></tr><tr><td>17 </td><td>62 </td><td>17 </td></tr><tr><td>18 </td><td>90 </td><td>18 </td></tr><tr><td>19 </td><td>92 </td><td>19 </td></tr><tr><td>20 </td><td>282 </td><td>20 </td></tr><tr><td>21 </td><td>74 </td><td>21 </td></tr><tr><td>22 </td><td>125 </td><td>22 </td></tr><tr><td>23 </td><td>214 </td><td>23 </td></tr><tr><td>24 </td><td>87 </td><td>24 </td></tr><tr><td>25 </td><td>210 </td><td>25 </td></tr><tr><td>26 </td><td>202 </td><td>26 </td></tr><tr><td>27 </td><td>104 </td><td>27 </td></tr><tr><td>28 </td><td>71 </td><td>28 </td></tr><tr><td>29 </td><td>90 </td><td>29 </td></tr></tbody></table></div><div content-id="DSV019TB2" class="table-modal"><table><thead name="thead" align=""><tr><th><em>A. rufa</em> </th><th rowspan="2">No. of anchored markers </th><th><em>A. chinensis</em> </th></tr><tr><th>Linkage group </th><th>Linkage group </th></tr></thead><tbody><tr><td>1 </td><td>126 </td><td>1 </td></tr><tr><td>2 </td><td>54 </td><td>2 </td></tr><tr><td>3 </td><td>304 </td><td>3 </td></tr><tr><td>4 </td><td>102 </td><td>4 </td></tr><tr><td>5 </td><td>93 </td><td>5 </td></tr><tr><td>6 </td><td>65 </td><td>6 </td></tr><tr><td>7 </td><td>133 </td><td>7 </td></tr><tr><td>8 </td><td>127 </td><td>8 </td></tr><tr><td>9 </td><td>160 </td><td>9 </td></tr><tr><td>10 </td><td>209 </td><td>10 </td></tr><tr><td>11 </td><td>76 </td><td>11 </td></tr><tr><td>12 </td><td>110 </td><td>12 </td></tr><tr><td>13 </td><td>128 </td><td>13 </td></tr><tr><td>14 </td><td>205 </td><td>14 </td></tr><tr><td>15 </td><td>144 </td><td>15 </td></tr><tr><td>16 </td><td>147 </td><td>16 </td></tr><tr><td>17 </td><td>62 </td><td>17 </td></tr><tr><td>18 </td><td>90 </td><td>18 </td></tr><tr><td>19 </td><td>92 </td><td>19 </td></tr><tr><td>20 </td><td>282 </td><td>20 </td></tr><tr><td>21 </td><td>74 </td><td>21 </td></tr><tr><td>22 </td><td>125 </td><td>22 </td></tr><tr><td>23 </td><td>214 </td><td>23 </td></tr><tr><td>24 </td><td>87 </td><td>24 </td></tr><tr><td>25 </td><td>210 </td><td>25 </td></tr><tr><td>26 </td><td>202 </td><td>26 </td></tr><tr><td>27 </td><td>104 </td><td>27 </td></tr><tr><td>28 </td><td>71 </td><td>28 </td></tr><tr><td>29 </td><td>90 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942878" target="_blank">View Large</a></div></div>                        <a id="4942879" scrollto-destination="4942879"></a>
<div id="DSV019F3" data-id="DSV019F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01903.png?Expires=2147483647&Signature=K-2GF44fT-T8u2ZVlByaj3j~GNGgpKmhHadNI6U6k5Hefn0K9LPsn6YO77U76cKT~qX8XpJk~nz3ir5oiSPU~GH1jJhqJbu6a63EtxY8A-wvCtr0HbNwhJ5v4LaSL7A7t9Alh1RcK3rQ5S7PWHrfZn3IzahBmUhWPwWjPuuBYcnRHMRYjSxCcM1suBjEt9VzVSzNzUg1ywHUlXjkzRGg0gVMbP80Kzq5kQCfDFvC2qkSxTBStrUzWbHsf0UY32v1Nvv5PpllGNPViUSdNeyzvTusiEJd1585zzMX0IqWYP3YGvTXRDtj7C~AN5p3qkAizNVS~c75K71IFvMZQyB93g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity between the A. rufa and A. chinensis genetic maps. Linkage groups for the A. rufa genetic map are colourized, and LGs for the A. chinensis are in grey. The two parental SNP markers that aligned to the genome scaffold within 100 kb were linked by coloured lines, with each colour corresponding to an A. rufa LG. ‘C’ and ‘R’ after each linkage group meant A. chinensis and A. rufa, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942879/dsv01903.tiff" path-from-xml="dsv01903.tiff" target="_blank">View large</a><a section="4942879" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01903.png?Expires=2147483647&Signature=S0zWW7nolfXtI7wIDdZLyEV1wiSO~DkzN-A9yKMriNSu7QkALc3Hcwq9qNGCkDcRaRx3F-gWY-nf4qxMzta40olyao5wqjCnt5RWi~J-XofHKW9bI232Fz7vbOuLN~3p~ROscWofBreBicGWLqEh4OlrmjFR2vLkB0Caqa0Y1TRCLwso9oorB2q4vz-iSRaOW2OF4kwsjB-FTUC2YlGxpHOm8iJwr0Xt0GhhTj6yWKEeJu6DvpNFCs56zkys-peFOdcPW0o2PUMTI6mv5BTK4b-~tnFcLe1-c7j8bv~T4ATt~rAcqiTa9ySyr1YHwxTC0-1m2WngE7OfqhTAa~ssYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942879&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity between the <em>A. rufa</em> and <em>A. chinensis</em> genetic maps. Linkage groups for the <em>A. rufa</em> genetic map are colourized, and LGs for the <em>A. chinensis</em> are in grey. The two parental SNP markers that aligned to the genome scaffold within 100 kb were linked by coloured lines, with each colour corresponding to an <em>A. rufa</em> LG. ‘C’ and ‘R’ after each linkage group meant <em>A. chinensis</em> and <em>A. rufa</em>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV019F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01903.png?Expires=2147483647&Signature=K-2GF44fT-T8u2ZVlByaj3j~GNGgpKmhHadNI6U6k5Hefn0K9LPsn6YO77U76cKT~qX8XpJk~nz3ir5oiSPU~GH1jJhqJbu6a63EtxY8A-wvCtr0HbNwhJ5v4LaSL7A7t9Alh1RcK3rQ5S7PWHrfZn3IzahBmUhWPwWjPuuBYcnRHMRYjSxCcM1suBjEt9VzVSzNzUg1ywHUlXjkzRGg0gVMbP80Kzq5kQCfDFvC2qkSxTBStrUzWbHsf0UY32v1Nvv5PpllGNPViUSdNeyzvTusiEJd1585zzMX0IqWYP3YGvTXRDtj7C~AN5p3qkAizNVS~c75K71IFvMZQyB93g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity between the A. rufa and A. chinensis genetic maps. Linkage groups for the A. rufa genetic map are colourized, and LGs for the A. chinensis are in grey. The two parental SNP markers that aligned to the genome scaffold within 100 kb were linked by coloured lines, with each colour corresponding to an A. rufa LG. ‘C’ and ‘R’ after each linkage group meant A. chinensis and A. rufa, respectively. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942879/dsv01903.tiff" path-from-xml="dsv01903.tiff" target="_blank">View large</a><a section="4942879" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01903.png?Expires=2147483647&Signature=S0zWW7nolfXtI7wIDdZLyEV1wiSO~DkzN-A9yKMriNSu7QkALc3Hcwq9qNGCkDcRaRx3F-gWY-nf4qxMzta40olyao5wqjCnt5RWi~J-XofHKW9bI232Fz7vbOuLN~3p~ROscWofBreBicGWLqEh4OlrmjFR2vLkB0Caqa0Y1TRCLwso9oorB2q4vz-iSRaOW2OF4kwsjB-FTUC2YlGxpHOm8iJwr0Xt0GhhTj6yWKEeJu6DvpNFCs56zkys-peFOdcPW0o2PUMTI6mv5BTK4b-~tnFcLe1-c7j8bv~T4ATt~rAcqiTa9ySyr1YHwxTC0-1m2WngE7OfqhTAa~ssYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942879&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity between the <em>A. rufa</em> and <em>A. chinensis</em> genetic maps. Linkage groups for the <em>A. rufa</em> genetic map are colourized, and LGs for the <em>A. chinensis</em> are in grey. The two parental SNP markers that aligned to the genome scaffold within 100 kb were linked by coloured lines, with each colour corresponding to an <em>A. rufa</em> LG. ‘C’ and ‘R’ after each linkage group meant <em>A. chinensis</em> and <em>A. rufa</em>, respectively. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4942880 id="4942880" class="section-title" data-legacy-id=s3c>3.3. Sex-specific marker identification and localization of the SDR</h3>
<p>The subtelomeric region of chromosome 25 (0–5 Mb) was confirmed to be the SDR based on the physical position of markers close to ‘Sex-f’, consistent with the previous findings of Fraser et al. A total of 1,095 SSR markers in position 0–5 Mb were identified by MISA following the criteria outlined in the Materials and methods section. The SSR markers were thinned to eight per Mb and 40 genic SSR markers were screened. All 40 SSR markers in the 0–5 Mb region were screened in 10 male and 10 female plants from the mapping population, of which 12 SSR markers were polymorphic in 20 lines tested, and only three SSR markers presented distinct bands for sex identification (Table <a reveal-id="DSV019TB3" data-open="DSV019TB3" class="link link-reveal link-table xref-fig">3</a>). The three sex-specific markers were further validated by screening against the 174 F1 individuals, and the results showed that they could clearly distinguish male and female individuals, respectively. A001 amplifies a 202-bp fragment in female individuals with no product in male individuals, and A002 and A003 produce diverse bands between the male and female plants. Furthermore, the three sex-specific SSR markers were validated in another set of individuals crossed by <em>A. rufa</em> and <em>A. chinensis</em>, and also in 6 <em>A. chinensis</em> (‘Hongyang’, ‘Cuiyu’, ‘Guihai’, ‘Huaguang No2’, ‘QS-14’, and ‘QS-5’) and 3 <em>A. rufa</em> (‘Shanli 3’, ‘Shanli 4’, and ‘Shanli 5’) germplasm accessions. All three markers can be used to accurately sex type in another interspecific cross. The A003 marker could distinguish all <em>A. chinensis</em> and <em>A. rufa</em> samples, and A001 marker could identify <em>A. rufa</em> plants, and combining A001 and A002 could also be used to sex type all <em>A. chinensis</em> and <em>A. rufa</em> samples. The positions of three SSR markers were located in 1–2 Mb on chromosome 25, which narrowed down the SDR to this 1 Mb region. The genotypes of three SSR markers were added to the database for grouping and mapping to the linkage group. All three SSR markers were closely adjacent to sex-determination locus, and the minimum genetic distance between ‘sex-f’ and SSR marker is 0.9 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="4942882" scrollto-destination="4942882"></a>
<div content-id="DSV019TB3" class="table-modal"><div class="table-wrap"><div id="DSV019TB3" class="table-wrap-title" data-id="DSV019TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Sex-related markers information</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker </th><th rowspan="2">Primer-F </th><th rowspan="2">Primer-R </th><th colspan="2">Size of products (bp)<hr /> </th><th rowspan="2">Gene ID </th><th rowspan="2">Position on chromosome 25 </th><th rowspan="2">Gene function </th></tr><tr><th>Female </th><th>Male </th></tr></thead><tbody><tr><td>A001 </td><td>TCAATGCATTTAGACA
TTCCTTTGTCCA </td><td>TGGGTAAACATAA
CCACATGCCAAC </td><td>202 </td><td>0 </td><td>Achn098711 </td><td>1,541,845–1,542,047 </td><td>RNA-binding protein, putative; contains IPR001876 (Zinc finger, RanBP2-type), IPR012677 (nucleotide-binding, alpha-beta plait) </td></tr><tr><td>A002 </td><td>TACTGACGGTCACT
CCCTAATCCC </td><td>CATGGATGGAACTG
GTGGAGGAAG </td><td>219 </td><td>230 </td><td>Achn098721 </td><td>1,612,522–1,612,771 </td><td>Oxidoreductase, 2OG-Fe(II) oxygenase family protein; contains IPR027450 (alpha-ketoglutarate-dependent dioxygenase AlkB-like) </td></tr><tr><td>A003 </td><td>GCAAGCGGGGGTAAA
TTTGTACAG </td><td>GGATAGGAGGAGC
TTTACGGACCT </td><td>304 </td><td>287 </td><td>Achn342741 </td><td>1,899,177–1,899,503 </td><td>Ribosome-recycling factor; contains IPR002661 (ribosome-recycling factor), IPR023584 (ribosome-recycling factor domain), IPR024946 (arginine repressor C-terminal-like domain) </td></tr></tbody></table></div><div content-id="DSV019TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker </th><th rowspan="2">Primer-F </th><th rowspan="2">Primer-R </th><th colspan="2">Size of products (bp)<hr /> </th><th rowspan="2">Gene ID </th><th rowspan="2">Position on chromosome 25 </th><th rowspan="2">Gene function </th></tr><tr><th>Female </th><th>Male </th></tr></thead><tbody><tr><td>A001 </td><td>TCAATGCATTTAGACA
TTCCTTTGTCCA </td><td>TGGGTAAACATAA
CCACATGCCAAC </td><td>202 </td><td>0 </td><td>Achn098711 </td><td>1,541,845–1,542,047 </td><td>RNA-binding protein, putative; contains IPR001876 (Zinc finger, RanBP2-type), IPR012677 (nucleotide-binding, alpha-beta plait) </td></tr><tr><td>A002 </td><td>TACTGACGGTCACT
CCCTAATCCC </td><td>CATGGATGGAACTG
GTGGAGGAAG </td><td>219 </td><td>230 </td><td>Achn098721 </td><td>1,612,522–1,612,771 </td><td>Oxidoreductase, 2OG-Fe(II) oxygenase family protein; contains IPR027450 (alpha-ketoglutarate-dependent dioxygenase AlkB-like) </td></tr><tr><td>A003 </td><td>GCAAGCGGGGGTAAA
TTTGTACAG </td><td>GGATAGGAGGAGC
TTTACGGACCT </td><td>304 </td><td>287 </td><td>Achn342741 </td><td>1,899,177–1,899,503 </td><td>Ribosome-recycling factor; contains IPR002661 (ribosome-recycling factor), IPR023584 (ribosome-recycling factor domain), IPR024946 (arginine repressor C-terminal-like domain) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942882" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV019TB3" class="table-wrap-title" data-id="DSV019TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Sex-related markers information</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Marker </th><th rowspan="2">Primer-F </th><th rowspan="2">Primer-R </th><th colspan="2">Size of products (bp)<hr /> </th><th rowspan="2">Gene ID </th><th rowspan="2">Position on chromosome 25 </th><th rowspan="2">Gene function </th></tr><tr><th>Female </th><th>Male </th></tr></thead><tbody><tr><td>A001 </td><td>TCAATGCATTTAGACA
TTCCTTTGTCCA </td><td>TGGGTAAACATAA
CCACATGCCAAC </td><td>202 </td><td>0 </td><td>Achn098711 </td><td>1,541,845–1,542,047 </td><td>RNA-binding protein, putative; contains IPR001876 (Zinc finger, RanBP2-type), IPR012677 (nucleotide-binding, alpha-beta plait) </td></tr><tr><td>A002 </td><td>TACTGACGGTCACT
CCCTAATCCC </td><td>CATGGATGGAACTG
GTGGAGGAAG </td><td>219 </td><td>230 </td><td>Achn098721 </td><td>1,612,522–1,612,771 </td><td>Oxidoreductase, 2OG-Fe(II) oxygenase family protein; contains IPR027450 (alpha-ketoglutarate-dependent dioxygenase AlkB-like) </td></tr><tr><td>A003 </td><td>GCAAGCGGGGGTAAA
TTTGTACAG </td><td>GGATAGGAGGAGC
TTTACGGACCT </td><td>304 </td><td>287 </td><td>Achn342741 </td><td>1,899,177–1,899,503 </td><td>Ribosome-recycling factor; contains IPR002661 (ribosome-recycling factor), IPR023584 (ribosome-recycling factor domain), IPR024946 (arginine repressor C-terminal-like domain) </td></tr></tbody></table></div><div content-id="DSV019TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Marker </th><th rowspan="2">Primer-F </th><th rowspan="2">Primer-R </th><th colspan="2">Size of products (bp)<hr /> </th><th rowspan="2">Gene ID </th><th rowspan="2">Position on chromosome 25 </th><th rowspan="2">Gene function </th></tr><tr><th>Female </th><th>Male </th></tr></thead><tbody><tr><td>A001 </td><td>TCAATGCATTTAGACA
TTCCTTTGTCCA </td><td>TGGGTAAACATAA
CCACATGCCAAC </td><td>202 </td><td>0 </td><td>Achn098711 </td><td>1,541,845–1,542,047 </td><td>RNA-binding protein, putative; contains IPR001876 (Zinc finger, RanBP2-type), IPR012677 (nucleotide-binding, alpha-beta plait) </td></tr><tr><td>A002 </td><td>TACTGACGGTCACT
CCCTAATCCC </td><td>CATGGATGGAACTG
GTGGAGGAAG </td><td>219 </td><td>230 </td><td>Achn098721 </td><td>1,612,522–1,612,771 </td><td>Oxidoreductase, 2OG-Fe(II) oxygenase family protein; contains IPR027450 (alpha-ketoglutarate-dependent dioxygenase AlkB-like) </td></tr><tr><td>A003 </td><td>GCAAGCGGGGGTAAA
TTTGTACAG </td><td>GGATAGGAGGAGC
TTTACGGACCT </td><td>304 </td><td>287 </td><td>Achn342741 </td><td>1,899,177–1,899,503 </td><td>Ribosome-recycling factor; contains IPR002661 (ribosome-recycling factor), IPR023584 (ribosome-recycling factor domain), IPR024946 (arginine repressor C-terminal-like domain) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4942882" target="_blank">View Large</a></div></div>                        <a id="4942883" scrollto-destination="4942883"></a>
<div id="DSV019F4" data-id="DSV019F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01904.png?Expires=2147483647&Signature=v741v7oPk4~07iyzwEJdVf-FLIGSwdeyzkPgXhjbPLbYNuE0Vs~gseulbJtsSWvLL639RAsoEo3AmWED47nxvThqMcXkJnFPTxhwZE56Hw56UEYr02bfAJOb-aQ8VIo3ypvnRKjMxJFcM5lHOo5hLImdahhmAbldsL76UDGfWCfWq5vt96P4OIA3aT5R41juQPMz7lNt6FoT2mj-QybMITEyuWjMqeDHEXMkQaniIlXm24L8Wb~1e5-x8kTvxgc6FX~BqLdI8j1u4uPCrXCI1jcDrF71leCoEbfNR~JZaBGWWW6W-2uxM-hzoPmXON6NwRDs99HTDWS21t0o30AeJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Markers were mapped on linkage groups that differed with corresponding assembled chromosomes. The chromosomes displayed by blue vertical bars are named beginning with ‘Chr’. Actinidia rufa linkage groups displaced by red vertical bars are named with ‘Ar_LG’, and A. chinensis linkage groups displaced by red vertical bars are named with ‘Ac_LG’. Green lines show links between linkage groups and corresponding chromosomes based on the marker position. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942883/dsv01904.tiff" path-from-xml="dsv01904.tiff" target="_blank">View large</a><a section="4942883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01904.png?Expires=2147483647&Signature=ZPsd3Nz~vDqJtTbqz5kIfZAn9EGyQbL1BWZyE2wiunSZ3f4Up9B-ViJ5J0qkN6sFYOHL-c8KoJ20vO3oe850s31D4GMSDL7aViiuf5DARsbfIOgICB9E4Wj0~mkFFXwLikFYLrLqD2YbG8BzBaNn-atHDyQwrsclKJqxwd2IIG5sqN9YCXZWve7Cscx8beuOdRyxpGiDN4PlKYq4Rx6MserzU4US1JExgdXrOV8IT-U9UeiKkUzTwglsvUHi-6jNM9YyTjdyNCB-K1Fu6Q8YfBWgtnrLs8x-vQhhAMQyNh~RD4qkC77hIkswyXbQhYefU7YJRuso64SakmZBqnLphw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942883&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Markers were mapped on linkage groups that differed with corresponding assembled chromosomes. The chromosomes displayed by blue vertical bars are named beginning with ‘Chr’. <em>Actinidia rufa</em> linkage groups displaced by red vertical bars are named with ‘Ar_LG’, and <em>A. chinensis</em> linkage groups displaced by red vertical bars are named with ‘Ac_LG’. Green lines show links between linkage groups and corresponding chromosomes based on the marker position. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV019F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/m_dsv01904.png?Expires=2147483647&Signature=v741v7oPk4~07iyzwEJdVf-FLIGSwdeyzkPgXhjbPLbYNuE0Vs~gseulbJtsSWvLL639RAsoEo3AmWED47nxvThqMcXkJnFPTxhwZE56Hw56UEYr02bfAJOb-aQ8VIo3ypvnRKjMxJFcM5lHOo5hLImdahhmAbldsL76UDGfWCfWq5vt96P4OIA3aT5R41juQPMz7lNt6FoT2mj-QybMITEyuWjMqeDHEXMkQaniIlXm24L8Wb~1e5-x8kTvxgc6FX~BqLdI8j1u4uPCrXCI1jcDrF71leCoEbfNR~JZaBGWWW6W-2uxM-hzoPmXON6NwRDs99HTDWS21t0o30AeJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Markers were mapped on linkage groups that differed with corresponding assembled chromosomes. The chromosomes displayed by blue vertical bars are named beginning with ‘Chr’. Actinidia rufa linkage groups displaced by red vertical bars are named with ‘Ar_LG’, and A. chinensis linkage groups displaced by red vertical bars are named with ‘Ac_LG’. Green lines show links between linkage groups and corresponding chromosomes based on the marker position. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv01904.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4942883/dsv01904.tiff" path-from-xml="dsv01904.tiff" target="_blank">View large</a><a section="4942883" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv01904.png?Expires=2147483647&Signature=ZPsd3Nz~vDqJtTbqz5kIfZAn9EGyQbL1BWZyE2wiunSZ3f4Up9B-ViJ5J0qkN6sFYOHL-c8KoJ20vO3oe850s31D4GMSDL7aViiuf5DARsbfIOgICB9E4Wj0~mkFFXwLikFYLrLqD2YbG8BzBaNn-atHDyQwrsclKJqxwd2IIG5sqN9YCXZWve7Cscx8beuOdRyxpGiDN4PlKYq4Rx6MserzU4US1JExgdXrOV8IT-U9UeiKkUzTwglsvUHi-6jNM9YyTjdyNCB-K1Fu6Q8YfBWgtnrLs8x-vQhhAMQyNh~RD4qkC77hIkswyXbQhYefU7YJRuso64SakmZBqnLphw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4942883&ar=348265&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01904.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Markers were mapped on linkage groups that differed with corresponding assembled chromosomes. The chromosomes displayed by blue vertical bars are named beginning with ‘Chr’. <em>Actinidia rufa</em> linkage groups displaced by red vertical bars are named with ‘Ar_LG’, and <em>A. chinensis</em> linkage groups displaced by red vertical bars are named with ‘Ac_LG’. Green lines show links between linkage groups and corresponding chromosomes based on the marker position. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The SDR region is subtelomeric and has a much lower gene density than the genome-wide average with mere 150 genes across the 5-Mb region. Despite the low gene density, the SDR region is enriched for genes involved in reproduction and it contains several candidate sex-determination genes. Among these are Achn342721, a homologue to <em>FBL17</em> in Arabidopsis with an <em>e</em>-value of 8<em>e</em>−04, which is essential for male fertility and germ line proliferation<sup><a reveal-id="DSV019C29" data-open="DSV019C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and Achn226501, a homologue of Caleosin, which is involved in lipid storage in maturing seeds with an <em>e</em>-value of 1<em>e</em>−79.</p>                            <h2 scrollto-destination=4942885 id="4942885" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Although a number of AFLP markers were developed to create linkage map for anchoring genome scaffolds,<sup><a reveal-id="DSV019C30" data-open="DSV019C30" class="link link-ref link-reveal xref-bibr">30</a></sup> screening traditional non-informative DNA markers, such as AFLP, RAPD, and ISSR, is a labour intensive, expensive, and low-throughput process that is unsuitable for aiding in the genome assembly. Fortunately, the advent of RAD-seq has overcome these problems, providing a cost-effective, high-throughput, automated protocol for identifying SNP markers. Using this method, a genetic map of kiwifruit was constructed to aid in the kiwifruit genome assembly using an F<sub>1</sub> population of <em>A. chinensis</em>.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Except for <em>A. chinensis</em>, few genetic and genomic resources are available for other <em>Actinidia</em> species which are especially problematic, given the limited diversity of useful kiwifruit germplam. <em>Actinidia rufa</em> native to China, Korea, and Japan is characterized by disease resistance, high yield, excellent storage, and other numerous good fruit traits. The high-density genetic map of <em>A. rufa</em> presented here is a useful prerequisite for quantitative trait locus mapping and marker-assisted breeding to facilitate identification of new desirable traits.</p><p>The <em>A. chinensis</em> genetic map had a higher marker density than the <em>A. rufa</em> map, with 4,215 markers in 29 linkage groups, spanning 3,142.4 cM. In comparison, the <em>A. rufa</em> map contained 2,426 markers with 29 linkage groups spanning over 2,651.3 cM. The higher marker density suggests that <em>A. chinensis</em> may have higher within genome heterozygosity than <em>A. rufa</em>. The lower nucleotide diversity in <em>A. rufa</em> is likely because it naturally occurs in a narrow geographic area (Taiwan in China, Korea, and Japan), which probably has an independent evolutionary and geographic origin. Moreover, <em>A. rufa</em> may have more sequence divergence that would result in fewer shared restriction sites and thus less mapable markers obtained. The accuracy of genetic maps can be estimated by comparing the collinearity between the two maps. Comparison of marker positions between the two maps revealed a high degree of collinearity; almost all 29 linkage groups are collinear on corresponding chromosomes between both maps, which positively supported the reliability of marker order in genetic maps.</p><p>Owing to large population size and high sequencing depth, abundant SNP markers were developed in a high-density genetic map, which was greatly beneficial to the kiwifruit genome assembly. Population size is a strong confounding factor of LD measurement, smaller populations have less variation, and lower recombination rates could be detected.<sup><a reveal-id="DSV019C31" data-open="DSV019C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We constructed a high-density genetic map using an interspecific population with 174 individuals, 1.6 times more than those (108 individuals) employed for map construction in previous research.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The significant genetic differences between parents and large size of the population facilitated to identify more SNPs. Moreover, the average sequencing depth of sequenced loci per progeny in these maps is 20.25 fold, almost two times more than that (10.73 fold) in Huang's study. In the previous map, SNP calling and genotyping were mainly depended on the parental data due to lack of a kiwifruit reference genome, which limited the number of useful markers. The previous mean marker density is 1.28 cM per marker, and 3,379 markers were aligned to the assembled scaffolds.<sup><a reveal-id="DSV019C15" data-open="DSV019C15" class="link link-ref link-reveal xref-bibr">15</a></sup> By comparison, we generated 2,426 and 4,214 SNP markers in the maternal and paternal maps with average marker distance 1.09 and 0.75 cM, respectively. The higher marker density allowed some previously unmapped scaffolds to be anchored. Roughly 440 unmapped scaffolds from the kiwifruit reference genome totalling 101 Mb were anchored to chromosomes; 228 and 373 unmapped scaffolds were anchored on <em>A. rufa</em> and <em>A. chinensis</em> map, respectively. The new anchored scaffolds considerably improved the kiwifruit genome assembly.</p><p>It is notable that a few markers mapped on linkage groups were differed with the corresponding assembled chromosomes. Multiple-clustered markers on chromosome 6 and chromosome 5 were re-mapped on linkage groups (chromosomes) 19 and 21 on both parental maps, which reconfirmed the high reliability of the genetic maps. It is possible that entire scaffolds were misplaced, which is common in draft genome assemblies.<sup><a reveal-id="DSV019C32" data-open="DSV019C32" class="link link-ref link-reveal xref-bibr">32</a></sup> We assumed that scaffolds were assembled correctly, then interchromosomal rearrangement appeared in both <em>A. chinensis</em> (paternal) and <em>A. rufa</em> (maternal)<em>.</em> Thus, it is contradictory that interchromosomal rearrangement occurred in interspecific <em>A. rufa</em> and <em>A. chinensis</em>, rather than the intraspecific <em>A. chinensis</em> cv. ‘Guihai No4’ and <em>A. chinensis</em> cv. ‘Hongyang’ genome. Much higher synteny with interspecific plants than intraspecific plants suggested that it was likely assembly errors rather than true genomic rearrangements.<sup><a reveal-id="DSV019C33" data-open="DSV019C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Moreover, if genome interchromosomal rearrangement was so rampant, we would expect to detect at least one between <em>A. rufa</em> and <em>A. chinensis.</em> Genome assemblies were imperfect and genomic rearrangements seem to be more common than they really are due to assembly errors.<sup><a reveal-id="DSV019C34" data-open="DSV019C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Upon integrating physical and genetic maps of the apple, homoeologous chromosome pairs and multiple locus markers on the same chromosome were detected.<sup><a reveal-id="DSV019C35" data-open="DSV019C35" class="link link-ref link-reveal xref-bibr">35</a></sup> These genome-wide and segmental duplications in the genome were likely to cause assembly errors. Besides assembly errors, some inconsistencies among marker positions were attributed either to structural variations within the genome or among mapping populations, or genotyping technical errors.<sup><a reveal-id="DSV019C36" data-open="DSV019C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The kiwifruit genome could be considerably improved by means of correcting misplaced scaffolds and increasing new anchored scaffolds.</p><p>Roughly 6% of flowering plants are dioecious with separate male and female plants, and dioecy has evolved independently numerous times.<sup><a reveal-id="DSV019C37" data-open="DSV019C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Dioecy can be controlled by simple mechanisms such as a single gene with two independent mutations or more complex mechanisms such as sex chromosomes. Two sex-determinant genes were mapped on a common linkage group in wild strawberry, and recombination between them resulted in male, female, hermaphrodite, and neuter progeny.<sup><a reveal-id="DSV019C38" data-open="DSV019C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Kiwifruit has an SDR that behaves like an XY chromosome pair with presumably two linked genes controlling sex.<sup><a reveal-id="DSV019C14" data-open="DSV019C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV019C39" data-open="DSV019C39" class="link link-ref link-reveal xref-bibr">39</a></sup> One gene likely controls carpel abortion which happens near inception in male flowers and the second gene controls pollen sterility as female flowers produce empty pollen grains.<sup><a reveal-id="DSV019C40" data-open="DSV019C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The sex-related marker ‘Sex-f’ was mapped on the telomere of LG25 in male map, while failed to map on female map. The similar results appeared on kiwifruit maps using AFLP markers in <em>A.</em> c<em>hinensis</em><em>×</em><em>A. callosa</em> and using SSR markers in <em>A. chinensis</em> × <em>A.</em> c<em>hinensis</em> due to the gap in female map or the low recombination of sexual chromosomes.<sup><a reveal-id="DSV019C12" data-open="DSV019C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV019C13" data-open="DSV019C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We constructed the higher density female genetic map with the average marker distance of 0.69 cM in LG25; therefore, it is possible that recombination suppression led to no markers mapping on the female sex chromosome.</p><p>We produced three additional SSR markers in the SDR that can be used to accurately sex type male and female plants, which has both commercial and breeding applications. The SCAR markers, SmX and SmY, were available to identify sex type in <em>A. chinensis</em>, but had limited use in sex typing interspecific seedlings. The SCAR markers relied on more specific sequences, which usually cause the failure of interspecific amplification, whereas SSR markers in the genic region were more stable than SCAR marker because of gene conservation. Three sex-related SSR markers were selected on genes in 0–5 Mb on chromosome 25; their predicted function was RNA-binding protein, oxidoreductase, and Ribosome-recycling factor, respectively (Table <a reveal-id="DSV019TB3" data-open="DSV019TB3" class="link link-reveal link-table xref-fig">3</a>). Because of its subtelomeric position, the SDR is remarkably gene poor and contains 150 genes over the ∼5 Mb region. SDR regions in plants tend to be pericentric as is the case of papaya<sup><a reveal-id="DSV019C41" data-open="DSV019C41" class="link link-ref link-reveal xref-bibr">41</a></sup> or telomeric as seen in poplar,<sup><a reveal-id="DSV019C42" data-open="DSV019C42" class="link link-ref link-reveal xref-bibr">42</a></sup> as these regions typically have lower rates of recombination which allows the fixation of sex-determination loci. The SDR in kiwifruit has several genes, which may be involved in sex determination including a homologue of <em>FBL17</em> in Arabidopsis, which is essential for male fertility and germ line proliferation.<sup><a reveal-id="DSV019C29" data-open="DSV019C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Mapping the SDR region will help expedite the discovery of the sex-determination genes in kiwifruit, which will allow the development of true breeding hermaphrodite varieties.</p>                            <h2 scrollto-destination=4942892 id="4942892" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4942894 id="4942894" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This project was supported by the National Natural Science Foundation of China (grants no. 31301749) and Wuhan Applied Basic Research Project (2015020101010075). Funding to pay the Open Access publication charges for this article was provided by the National Natural Science Foundation of China (Grants No. 31301749).</p>                            <h2 scrollto-destination=4942896 id="4942896" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV019C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Soejarto</div>  <div class="given-names">D.</div></div></div> <div class="year">2007</div>, <div class="source">Actinidiaceae/Flora of China</div> , <div class="volume">vol. 12</div>. <div class="publisher-name">Science Press</div>: <div class="publisher-loc">Beijing</div>; <div class="comment">Missouri Botanical Gardens: St. Louis, Missouri</div>, p. <div class="fpage">334</div>–<div class="lpage">60</div>.</div></div></div></div><div content-id="DSV019C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhong</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jiang</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">H.</div></div></div> <div class="year">2012</div>, <div class="article-title">‘Jinyan’, an interspecific hybrid kiwifruit with Brilliant Yellow Flesh and Good Storage Quality</div>, <div class="source">HortScience</div> , <div class="volume">47</div>, <div class="fpage">1187</div>–<div class="lpage">90</div>.</div></div></div></div><div content-id="DSV019C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Charlesworth</div>  <div class="given-names">B.</div></div></div> <div class="year">1991</div>, <div class="article-title">The evolution of sex chromosomes</div>, <div class="source">Science</div> , <div class="volume">251</div>, <div class="fpage">1030</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20sex%20chromosomes&amp;author=B.%20Charlesworth&amp;publication_year=1991&amp;journal=Science&amp;volume=251&amp;pages=1030-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1998119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1998119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1998119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1998119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kater</div>  <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Franken</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Carney</div>  <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Colombo</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Angenent</div>  <div class="given-names">G.C.</div></div></div> <div class="year">2001</div>, <div class="article-title">Sex determination in the monoecious species cucumber is confined to specific floral whorls</div>, <div class="source">Plant Cell</div> , <div class="volume">13</div>, <div class="fpage">481</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex%20determination%20in%20the%20monoecious%20species%20cucumber%20is%20confined%20to%20specific%20floral%20whorls&amp;author=M.M.%20Kater&amp;author=J.%20Franken&amp;author=K.J.%20Carney&amp;author=L.%20Colombo&amp;author=G.C.%20Angenent&amp;publication_year=2001&amp;journal=Plant%20Cell&amp;volume=13&amp;pages=481-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.13.3.481" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.13.3.481" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.13.3.481"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11251091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Moore</div>  <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Ma</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2004</div>, <div class="article-title">A primitive Y chromosome in papaya marks incipient sex chromosome evolution</div>, <div class="source">Nature</div> , <div class="volume">427</div>, <div class="fpage">348</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20primitive%20Y%20chromosome%20in%20papaya%20marks%20incipient%20sex%20chromosome%20evolution&amp;author=Z.%20Liu&amp;author=P.H.%20Moore&amp;author=H.%20Ma&amp;publication_year=2004&amp;journal=Nature&amp;volume=427&amp;pages=348-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14737167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kafkas</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Khodaeiaminjan</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Güney</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kafkas</div>  <div class="given-names">E.</div></div></div> <div class="year">2015</div>, <div class="article-title">Identification of sex-linked SNP markers using RAD sequencing suggests ZW/ZZ sex determination in <em>Pistacia vera</em> L.</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20sex-linked%20SNP%20markers%20using%20RAD%20sequencing%20suggests%20ZW%2FZZ%20sex%20determination%20in%20Pistacia%20vera%20L.&amp;author=S.%20Kafkas&amp;author=M.%20Khodaeiaminjan&amp;author=M.%20G%C3%BCney&amp;author=E.%20Kafkas&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1326-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1326-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1326-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25765114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Spigler</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lewers</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Main</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ashman</div>  <div class="given-names">T.</div></div></div> <div class="year">2008</div>, <div class="article-title">Genetic mapping of sex determination in a wild strawberry, <em>Fragaria virginiana</em>, reveals earliest form of sex chromosome</div>, <div class="source">Heredity</div> , <div class="volume">101</div>, <div class="fpage">507</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20sex%20determination%20in%20a%20wild%20strawberry%2C%20Fragaria%20virginiana%2C%20reveals%20earliest%20form%20of%20sex%20chromosome&amp;author=R.%20Spigler&amp;author=K.%20Lewers&amp;author=D.%20Main&amp;author=T.%20Ashman&amp;publication_year=2008&amp;journal=Heredity&amp;volume=101&amp;pages=507-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2008.100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2008.100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2008.100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18797475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mrackova</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nicolas</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hobza</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Independent origin of sex chromosomes in two species of the genus Silene</div>, <div class="source">Genetics</div> , <div class="volume">179</div>, <div class="fpage">1129</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Independent%20origin%20of%20sex%20chromosomes%20in%20two%20species%20of%20the%20genus%20Silene&amp;author=M.%20Mrackova&amp;author=M.%20Nicolas&amp;author=R.%20Hobza&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=1129-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.085670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.085670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.085670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18558658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">H.</div></div></div> <div class="year">2002</div>, <div class="source">The Genus of Actinidia: A Word Monograph</div> . <div class="publisher-loc">Beijing</div>: <div class="publisher-name">Science Press</div>, p. <div class="fpage">34</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSV019C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McNeilage</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Steinhagen</div>  <div class="given-names">S.</div></div></div> <div class="year">1998</div>, <div class="article-title">Flower and fruit characters in a kiwifruit hermaphrodite</div>, <div class="source">Euphytica</div> , <div class="volume">101</div>, <div class="fpage">69</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flower%20and%20fruit%20characters%20in%20a%20kiwifruit%20hermaphrodite&amp;author=M.%20McNeilage&amp;author=S.%20Steinhagen&amp;publication_year=1998&amp;journal=Euphytica&amp;volume=101&amp;pages=69-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1018398924819" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1018398924819" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1018398924819"> </span></div> </div></div></div></div></div><div content-id="DSV019C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferguson</div>  <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">H.</div></div></div> <div class="year">2007</div>, <div class="article-title">Genetic resources of kiwifruit: domestication and breeding</div>, <div class="source">Hortic. Rev.</div> , <div class="volume">33</div>, <div class="fpage">1</div>–<div class="lpage">121</div>.</div></div></div></div><div content-id="DSV019C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Testolin</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Lain</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Messina</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Vecchione</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cipriani</div>  <div class="given-names">G.</div></div></div> <div class="year">2001</div>, <div class="article-title">A kiwifruit (<em>Actinidia</em> spp.) linkage map based on microsatellites and integrated with AFLP markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">103</div>, <div class="fpage">30</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20kiwifruit%20(Actinidia%20spp.)%20linkage%20map%20based%20on%20microsatellites%20and%20integrated%20with%20AFLP%20markers&amp;author=R.%20Testolin&amp;author=W.%20Huang&amp;author=O.%20Lain&amp;author=R.%20Messina&amp;author=A.%20Vecchione&amp;author=G.%20Cipriani&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=30-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-001-0555-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-001-0555-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-001-0555-z"> </span></div> </div></div></div></div></div><div content-id="DSV019C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fraser</div>  <div class="given-names">L.G.</div></div>, <div class="name string-name"><div class="surname">Tsang</div>  <div class="given-names">G.K.</div></div>, <div class="name string-name"><div class="surname">Datson</div>  <div class="given-names">P.M.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">A gene-rich linkage map in the dioecious species <em>Actinidia chinensis</em> (kiwifruit) reveals putative X/Y sex-determining chromosomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">10</div>, <div class="fpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene-rich%20linkage%20map%20in%20the%20dioecious%20species%20Actinidia%20chinensis%20(kiwifruit)%20reveals%20putative%20X%2FY%20sex-determining%20chromosomes&amp;author=L.G.%20Fraser&amp;author=G.K.%20Tsang&amp;author=P.M.%20Datson&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19284545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gill</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Harvey</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gardner</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Fraser</div>  <div class="given-names">L.</div></div></div> <div class="year">1998</div>, <div class="article-title">Development of sex-linked PCR markers for gender identification in Actinidia</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">97</div>, <div class="fpage">439</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20sex-linked%20PCR%20markers%20for%20gender%20identification%20in%20Actinidia&amp;author=G.%20Gill&amp;author=C.%20Harvey&amp;author=R.%20Gardner&amp;author=L.%20Fraser&amp;publication_year=1998&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=97&amp;pages=439-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050914"> </span></div> </div></div></div></div></div><div content-id="DSV019C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ding</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Deng</div>  <div class="given-names">D.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Draft genome of the kiwifruit <em>Actinidia chinensis</em></div>, <div class="source">Nat. Commun.</div> , <div class="volume">4</div>.</div></div></div></div><div content-id="DSV019C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pfender</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Saha</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Johnson</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Slabaugh</div>  <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <div class="article-title">Mapping with RAD (restriction-site associated DNA) markers to rapidly identify QTL for stem rust resistance in <em>Lolium perenne</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">122</div>, <div class="fpage">1467</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20with%20RAD%20(restriction-site%20associated%20DNA)%20markers%20to%20rapidly%20identify%20QTL%20for%20stem%20rust%20resistance%20in%20Lolium%20perenne&amp;author=W.%20Pfender&amp;author=M.%20Saha&amp;author=E.%20Johnson&amp;author=M.%20Slabaugh&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1467-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1546-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1546-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1546-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21344184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baxter</div>  <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Davey</div>  <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Johnston</div>  <div class="given-names">J.S.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20mapping%20and%20comparative%20genomics%20using%20next-generation%20RAD%20sequencing%20of%20a%20non-model%20organism&amp;author=S.W.%20Baxter&amp;author=J.W.%20Davey&amp;author=J.S.%20Johnston&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e19315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21541297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lewis</div>  <div class="given-names">Z.A.</div></div>, <div class="name string-name"><div class="surname">Shiver</div>  <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Stiffler</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Miller</div>  <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Johnson</div>  <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Selker</div>  <div class="given-names">E.U.</div></div></div> <div class="year">2007</div>, <div class="article-title">High-density detection of restriction-site-associated DNA markers for rapid mapping of mutated loci in Neurospora</div>, <div class="source">Genetics</div> , <div class="volume">177</div>, <div class="fpage">1163</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20detection%20of%20restriction-site-associated%20DNA%20markers%20for%20rapid%20mapping%20of%20mutated%20loci%20in%20Neurospora&amp;author=Z.A.%20Lewis&amp;author=A.L.%20Shiver&amp;author=N.%20Stiffler&amp;author=M.R.%20Miller&amp;author=E.A.%20Johnson&amp;author=E.U.%20Selker&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=1163-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.078147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.078147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.078147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ren</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">McGregor</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">Y.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">An integrated genetic map based on four mapping populations and quantitative trait loci associated with economically important traits in watermelon (<em>Citrullus lanatus</em>)</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20map%20based%20on%20four%20mapping%20populations%20and%20quantitative%20trait%20loci%20associated%20with%20economically%20important%20traits%20in%20watermelon%20(Citrullus%20lanatus)&amp;author=Y.%20Ren&amp;author=C.%20McGregor&amp;author=Y.%20Zhang&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24443961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chutimanitsakun</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nipper</div>  <div class="given-names">R.W.</div></div>, <div class="name string-name"><div class="surname">Cuesta-Marcos</div>  <div class="given-names">A.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Construction and application for QTL analysis of a restriction site associated DNA (RAD) linkage map in barley</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20application%20for%20QTL%20analysis%20of%20a%20restriction%20site%20associated%20DNA%20(RAD)%20linkage%20map%20in%20barley&amp;author=Y.%20Chutimanitsakun&amp;author=R.W.%20Nipper&amp;author=A.%20Cuesta-Marcos&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21205322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shao</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Niu</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Rastas</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of <em>Vibrio anguillarum</em> disease resistance and comparative genomic analysis</div>, <div class="source">DNA Res.</div> , <div class="volume">22</div>, <div class="fpage">161</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20identification%20for%20the%20construction%20of%20a%20high-resolution%20genetic%20map%20of%20Japanese%20flounder%20(Paralichthys%20olivaceus)%3A%20applications%20to%20QTL%20mapping%20of%20Vibrio%20anguillarum%20disease%20resistance%20and%20comparative%20genomic%20analysis&amp;author=C.%20Shao&amp;author=Y.%20Niu&amp;author=P.%20Rastas&amp;publication_year=2015&amp;journal=DNA%20Res.&amp;volume=22&amp;pages=161-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25762582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doyle</div>  <div class="given-names">J.J.</div></div></div> <div class="year">1990</div>, <div class="article-title">Isolation of plant DNA from fresh tissue</div>, <div class="source">Focus</div> , <div class="volume">12</div>, <div class="fpage">13</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="DSV019C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">VanBuren</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Optimization of linkage mapping strategy and construction of a high-density American lotus linkage map</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">372</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20linkage%20mapping%20strategy%20and%20construction%20of%20a%20high-density%20American%20lotus%20linkage%20map&amp;author=Q.%20Zhang&amp;author=L.%20Li&amp;author=R.%20VanBuren&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=372" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24885335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Emerson</div>  <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Merz</div>  <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Catchen</div>  <div class="given-names">J.M.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Resolving postglacial phylogeography using high-throughput sequencing</div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="volume">107</div>, <div class="fpage">16196</div>–<div class="lpage">200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolving%20postglacial%20phylogeography%20using%20high-throughput%20sequencing&amp;author=K.J.%20Emerson&amp;author=C.R.%20Merz&amp;author=J.M.%20Catchen&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=107&amp;pages=16196-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1006538107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1006538107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1006538107"> </span></div> </div></div></div></div></div><div content-id="DSV019C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">Y.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1966</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fang</div>  <div class="given-names">X.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">SNP detection for massively parallel whole-genome resequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">1124</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20detection%20for%20massively%20parallel%20whole-genome%20resequencing&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=X.%20Fang&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1124-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.088013.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.088013.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.088013.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grattapaglia</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sederoff</div>  <div class="given-names">R.</div></div></div> <div class="year">1994</div>, <div class="article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers</div>, <div class="source">Genetics</div> , <div class="volume">137</div>, <div class="fpage">1121</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20of%20Eucalyptus%20grandis%20and%20Eucalyptus%20urophylla%20using%20a%20pseudo-testcross%3A%20mapping%20strategy%20and%20RAPD%20markers&amp;author=D.%20Grattapaglia&amp;author=R.%20Sederoff&amp;publication_year=1994&amp;journal=Genetics&amp;volume=137&amp;pages=1121-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7982566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ooijen</div>  <div class="given-names">J.</div></div></div> <div class="year">2006</div>, <div class="source">JoinMap 4. Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> . <div class="publisher-name">Kyazma BV</div>: <div class="publisher-loc">Wageningen, Netherlands</div>.</div></div></div></div><div content-id="DSV019C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div>  <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Oh</div>  <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Brownfield</div>  <div class="given-names">L.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Control of plant germline proliferation by SCFFBL17 degradation of cell cycle inhibitors</div>, <div class="source">Nature</div> , <div class="volume">455</div>, <div class="fpage">1134</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20plant%20germline%20proliferation%20by%20SCFFBL17%20degradation%20of%20cell%20cycle%20inhibitors&amp;author=H.J.%20Kim&amp;author=S.A.%20Oh&amp;author=L.%20Brownfield&amp;publication_year=2008&amp;journal=Nature&amp;volume=455&amp;pages=1134-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07289"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Boer</div>  <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Borm</div>  <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Jesse</div>  <div class="given-names">T.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">A hybrid BAC physical map of potato: a framework for sequencing a heterozygous genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">594</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hybrid%20BAC%20physical%20map%20of%20potato%3A%20a%20framework%20for%20sequencing%20a%20heterozygous%20genome&amp;author=J.M.%20de%20Boer&amp;author=T.J.%20Borm&amp;author=T.%20Jesse&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=594" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22142254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Serre</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Nadon</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hudson</div>  <div class="given-names">T.J.</div></div></div> <div class="year">2005</div>, <div class="article-title">Large-scale recombination rate patterns are conserved among human populations</div>, <div class="source">Genome Res.</div> , <div class="volume">15</div>, <div class="fpage">1547</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20recombination%20rate%20patterns%20are%20conserved%20among%20human%20populations&amp;author=D.%20Serre&amp;author=R.%20Nadon&amp;author=T.J.%20Hudson&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1547-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4211905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4211905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4211905"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16251464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tennessen</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Govindarajulu</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ashman</div>  <div class="given-names">T.-L.</div></div>, <div class="name string-name"><div class="surname">Liston</div>  <div class="given-names">A.</div></div></div> <div class="year">2014</div>, <div class="article-title">Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">6</div>, <div class="fpage">3295</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origins%20and%20dynamics%20of%20octoploid%20strawberry%20subgenomes%20revealed%20by%20dense%20targeted%20capture%20linkage%20maps&amp;author=J.A.%20Tennessen&amp;author=R.%20Govindarajulu&amp;author=T.-L.%20Ashman&amp;author=A.%20Liston&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol.&amp;volume=6&amp;pages=3295-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25477420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kai</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kikuchi</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tohari</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Integration of the genetic map and genome assembly of fugu facilitates insights into distinct features of genome evolution in teleosts and mammals</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">3</div>, <div class="fpage">424</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20the%20genetic%20map%20and%20genome%20assembly%20of%20fugu%20facilitates%20insights%20into%20distinct%20features%20of%20genome%20evolution%20in%20teleosts%20and%20mammals&amp;author=W.%20Kai&amp;author=K.%20Kikuchi&amp;author=S.%20Tohari&amp;publication_year=2011&amp;journal=Genome%20Biol.%20Evol.&amp;volume=3&amp;pages=424-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evr041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevr041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21551351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shen</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Papasian</div>  <div class="given-names">C.J.</div></div></div> <div class="year">2011</div>, <div class="article-title">Comparative studies of de novo assembly tools for next-generation sequencing technologies</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2031</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20studies%20of%20de%20novo%20assembly%20tools%20for%20next-generation%20sequencing%20technologies&amp;author=Y.%20Lin&amp;author=J.%20Li&amp;author=H.%20Shen&amp;author=L.%20Zhang&amp;author=C.J.%20Papasian&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2031-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zheng</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Vimolmangkang</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Khan</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Beever</div>  <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Korban</div>  <div class="given-names">S.S.</div></div></div> <div class="year">2011</div>, <div class="article-title">Integration of physical and genetic maps in apple confirms whole-genome and segmental duplications in the apple genome</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">62</div>, <div class="fpage">5117</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20physical%20and%20genetic%20maps%20in%20apple%20confirms%20whole-genome%20and%20segmental%20duplications%20in%20the%20apple%20genome&amp;author=Y.%20Han&amp;author=D.%20Zheng&amp;author=S.%20Vimolmangkang&amp;author=M.A.%20Khan&amp;author=J.E.%20Beever&amp;author=S.S.%20Korban&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=5117-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khan</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Han</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhao</div>  <div class="given-names">Y.F.</div></div>, <div class="name string-name"><div class="surname">Troggio</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Korban</div>  <div class="given-names">S.S.</div></div></div> <div class="year">2012</div>, <div class="article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e47864</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20multi-population%20consensus%20genetic%20map%20reveals%20inconsistent%20marker%20order%20among%20maps%20likely%20attributed%20to%20structural%20variations%20in%20the%20apple%20genome&amp;author=M.A.%20Khan&amp;author=Y.%20Han&amp;author=Y.F.%20Zhao&amp;author=M.%20Troggio&amp;author=S.S.%20Korban&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e47864" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0047864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0047864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0047864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23144832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV019C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Renner</div>  <div class="given-names">S.S.</div></div>, <div class="name string-name"><div class="surname">Ricklefs</div>  <div class="given-names">R.E.</div></div></div> <div class="year">1995</div>, <div class="article-title">Dioecy and its correlates in the flowering plants</div>, <div class="source">Am. J. Bot.</div> , <div class="volume">82</div>, <div class="fpage">596</div>–<div class="lpage">606</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dioecy%20and%20its%20correlates%20in%20the%20flowering%20plants&amp;author=S.S.%20Renner&amp;author=R.E.%20Ricklefs&amp;publication_year=1995&amp;journal=Am.%20J.%20Bot.&amp;volume=82&amp;pages=596-606" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2445418" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2445418" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2445418"> </span></div> </div></div></div></div></div><div content-id="DSV019C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ming</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bendahmane</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Renner</div>  <div class="given-names">S.S.</div></div></div> <div class="year">2011</div>, <div class="article-title">Sex chromosomes in land plants</div>, <div class="source">Ann. Rev. Plant Biol.</div> , <div class="volume">62</div>, <div class="fpage">485</div>–<div class="lpage">514</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex%20chromosomes%20in%20land%20plants&amp;author=R.%20Ming&amp;author=A.%20Bendahmane&amp;author=S.S.%20Renner&amp;publication_year=2011&amp;journal=Ann.%20Rev.%20Plant%20Biol.&amp;volume=62&amp;pages=485-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-042110-103914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-042110-103914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-042110-103914"> </span></div> </div></div></div></div></div><div content-id="DSV019C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harvey</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gill</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Fraser</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">McNeilage</div>  <div class="given-names">M.</div></div></div> <div class="year">1997</div>, <div class="article-title">Sex determination in Actinidia. 1. Sex-linked markers and progeny sex ratio in diploid <em>A. chinensis</em></div>, <div class="source">Sex. Plant Reprod.</div> , <div class="volume">10</div>, <div class="fpage">149</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex%20determination%20in%20Actinidia.%201.%20Sex-linked%20markers%20and%20progeny%20sex%20ratio%20in%20diploid%20A.%20chinensis&amp;author=C.%20Harvey&amp;author=G.%20Gill&amp;author=L.%20Fraser&amp;author=M.%20McNeilage&amp;publication_year=1997&amp;journal=Sex.%20Plant%20Reprod.&amp;volume=10&amp;pages=149-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004970050082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004970050082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004970050082"> </span></div> </div></div></div></div></div><div content-id="DSV019C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jamilena</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mariotti</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Manzano</div>  <div class="given-names">S.</div></div></div> <div class="year">2007</div>, <div class="article-title">Plant sex chromosomes: molecular structure and function</div>, <div class="source">Cytogenet Genome Res.</div> , <div class="volume">120</div>, <div class="fpage">255</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20sex%20chromosomes%3A%20molecular%20structure%20and%20function&amp;author=M.%20Jamilena&amp;author=B.%20Mariotti&amp;author=S.%20Manzano&amp;publication_year=2007&amp;journal=Cytogenet%20Genome%20Res.&amp;volume=120&amp;pages=255-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000121075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000121075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000121075"> </span></div> </div></div></div></div></div><div content-id="DSV019C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Na</div>  <div class="given-names">J.-K.</div></div>, <div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Q.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Sequencing papaya X and Yh chromosomes reveals molecular basis of incipient sex chromosome evolution</div>, <div class="source">Proc. Natl. Acad. Sci.</div> , <div class="volume">109</div>, <div class="fpage">13710</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20papaya%20X%20and%20Yh%20chromosomes%20reveals%20molecular%20basis%20of%20incipient%20sex%20chromosome%20evolution&amp;author=J.%20Wang&amp;author=J.-K.%20Na&amp;author=Q.%20Yu&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=109&amp;pages=13710-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1207833109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1207833109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1207833109"> </span></div> </div></div></div></div></div><div content-id="DSV019C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV019C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yin</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">DiFazio</div>  <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Gunter</div>  <div class="given-names">L.E.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Genome structure and emerging evidence of an incipient sex chromosome in Populus</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">422</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20and%20emerging%20evidence%20of%20an%20incipient%20sex%20chromosome%20in%20Populus&amp;author=T.%20Yin&amp;author=S.P.%20DiFazio&amp;author=L.E.%20Gunter&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=422-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7076308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7076308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7076308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18256239" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348265&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table6.xlsx?Expires=2147483647&amp;Signature=nlMwmmpKdiR-dBNp0TAcwzGTa6kRxoQbKVBGvI5oJwLPKC0yMnpNWbnVUqth16WU9Xuo0kv2G03zmcHbBXTowfaPfv42NTdtWp3dbEldr5brANoFgyOxioXMNdq~q~70qd4ujDDHfplxhaw7seZ3XqGDLmNxhJ7VCgMyLpz-DX~T8mlIPv7blaNYv0goW54TQl6eOSewyZaomhEGvz20UOtm4jUXe5DIUCSLayy7qD5FQ7hDoArLTYqSjrez~UkE2nkc3yYbsnWO9N2LEHm9t3R8BVfS2ImHKSUsmtGnNepc95UbN-IzRkjnCcokH91YBWULIQpzqKYeuVAlnYOWog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table5.xlsx?Expires=2147483647&amp;Signature=4FcDdnnJmLb9MV4qC~1ueSh5q-MrMcDLMiwvvh2xfSf30Y7aKCdGCmz4nhAbiuM3qlOvtNpB8Zu6AwrIN8aDCqvem-T--QrZFEdftC3WFcvKZAwAczic1o4g4qeFn~eqGCWpGpOD6ckgUJHx6dlxEGyHsXgikf5dfUZY8vyRE7s0snZAQqWSCA3jz-i3sg0N3YZEn86zYr~aVP9eZpejn0J-Jp8b1JWq8cHhwWIH-72CTxoGGLfNu51yjuvoUKXEkjIhac~jOJa1c4csDX1pcMQh79~RKiCLtd77aghIhh2I1qT7WQAAEKZ7tMLMb8WRNSAzENPCznCF9oc6ZJgVjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table4.xls?Expires=2147483647&amp;Signature=WlwqNB2ppwxgWtPT~zwiL-Ddo0eylvxlohCqwZOCS7c5KCz-tm3p58BPzuWTaDagcOTj~t6fd2ocijZLRroQKCM0DvdXs0LWvBdJ-bcgzVS6sjR~B-58jZUx89lLLxrlty57LjP2CR8CubBkfdUPKJlSeivi8wXUz8a8xEXpb~RgbrG2whMpBTiZQkGOOzxQO3gJzTa7ZI-oFD25D7~vRpNqtTZ6GtRT198SkjSRCdhiUVmgjHjertuUauLnWpCjAantO33hqMYu-G-wRVIWwyi-gJSxEWLKlcHrTWkWb3kwiwS8vbo1njLjz9k9ZAc8m4LGLqbzFLZ4md2CeD40kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table3.xls?Expires=2147483647&amp;Signature=pEuMCW7rG0g7uoxTBFFzBgUal4kYNgMy~UJ89YUndMfBjY0u3tJO8U62KK2L3T5RmD9BoN9ntvJFPpqM7A4pTlu6uqRw~qX9OP6DpaJt5ZRyLG5CxbWVd8Gsp9YALl4SqjWUVThZfTdSoIlTy8wyaVeco6vtfLZhygPV2XNsOIEnVRFEhvzaeYDvW3wWhQBVkcXJvHdte1zWDscb26EzzhZ6Fuwk4lJxWh-qHHmDZCc8gZ~kDpNcr8ccBCuTCrKelsi8mxis6rxnyupbPut7sCx8qGdHfy4Vsr~TbbzMvXJecvB3Y8saNfOZSmXWr7WlVdKZZ9OJx6FPeWl-ERSjDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table2.xlsx?Expires=2147483647&amp;Signature=gjNdcbn5CEAfQe5pFI5eyL1zjFGi-PXdphXfIOUtnYYkVcjG81ALZom21IGDwjBJ0FOSv5lXAexJsC0M9cl5L-v7rgl~pXJjDZtUpGVU~9-~-br6TXXv5dWh0cSNQFaCcQ-mb0FaD5StLRbywKG5sM76JEJiwUsdTgnTo79~AFcKXN-JfSrrqZo0ilAfW9dakno9sTzjxh88RX0rQ2X97CgJ~XU41nGNRPfTvI59~cqaYvZSvLnZGKchLOUn7Gz7MxaZNGqTyj3ycXdwF-l48KQgG~bgtzEBKZHuzRQugKgsEzWse~gyTbwJngKsGLa2EQXJGYXYHFBxCWoE0hExxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_table1.xlsx?Expires=2147483647&amp;Signature=hRd~KgAxqUzI2ekFMJlnCaCY7W1a51xUZrOx0sCEMcSL~~ATHKaFlME94npKaCLE6wpwT1cP9R~j0Hac773DShuHEwLYbndeMOkejuIuKZjIgcywQgFvmy1WT0Wt1JpV5-dErV3UAOYrkhwe7f7yHlqt~wxz4QbVE4XxBToFZNbFxmQZEqiiTyvOD73EG5zhFHDgMFGo0Ga9VKfVWRDUvCKyPvV6-Z4cEENwwhOmzq8ikXj3ZoEw5LbMqd8gFqb1J7RWQMrtmDjI~l7sJZjsVd1-04YpcjmSn0uoAAHMsmA7yJzlAtyyj1rMs~VMgZDmdIzedE-81iyclgg55xk4Kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv019/2/dsv019-dsv019supp_fig1.pdf?Expires=2147483647&amp;Signature=h6ausbIeRmCcITl1PLjBEtH~z89CNgpXi0lOxTufWJPPZJT8fYnK20lFsHIa1F6PC0BYWLhyb68s51g1kx3UGTczWoWmArcCVFXHkceW~ikOE4v0aA8vzQX5ShBUXDOwZwH7wfJdegnh84ALAP9S6qtnNi~9rMQvJ0oc7qVi8uh8qRDpU5HkYZxaaFKCgzMYO3NqVQvnLRBtzXJIlczgzbPTqZFDwp6sn2LuYmBg8Yih5-LFl-gTr6FnHUQxVWyWn7-BVzdp9QM85aHxR6C89ngD1zFK3q-woflqcx4-QWxLYvPD8jij0dm3Jk6Jz-QHahgCaj3ArH8QF2Hul4rDkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">850</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">23</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348265"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f5%2f367%2f348265">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362820100006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (23)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F5%2F367%2F348265" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348265'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-density interspecific genetic maps of kiwifruit and the identification of sex-specific markers" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f367%2f348265">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

