

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/1/21/2462418","@type":"ScholarlyArticle","name":"Improvement of barley genome annotations by deciphering the Haruna Nijo genome","datePublished":"2015-11-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2016-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv033","keywords":["genome sequencing","full-length cDNA","<italic>Hordeum</italic>","RNA-Seq"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/1/21/2462418","author":[{"name":"Sato, Kazuhiro","affiliation":"1  Institute of Plant Science and Resources ,  Okayama University ,  Kurashiki 710-0046 ,  Japan","@type":"Person"},{"name":"Tanaka, Tsuyoshi","affiliation":"2  National Institute of Agrobiological Sciences ,  Tsukuba 305-8602 ,  Japan","@type":"Person"},{"name":"Shigenobu, Shuji","affiliation":"3  National Institute for Basic Biology ,  Okazaki 444-8585 ,  Japan","@type":"Person"},{"name":"Motoi, Yuka","affiliation":"1  Institute of Plant Science and Resources ,  Okayama University ,  Kurashiki 710-0046 ,  Japan","@type":"Person"},{"name":"Wu, Jianzhong","affiliation":"2  National Institute of Agrobiological Sciences ,  Tsukuba 305-8602 ,  Japan","@type":"Person"},{"name":"Itoh, Takeshi","affiliation":"2  National Institute of Agrobiological Sciences ,  Tsukuba 305-8602 ,  Japan","@type":"Person"}],"description":"Abstract.  Full-length (FL) cDNA sequences provide the most reliable evidence for the presence of genes in genomes. In this report, detailed gene structures of ","pageStart":"21","pageEnd":"28","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Improvement of barley genome annotations by deciphering the Haruna Nijo genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Improvement of barley genome annotations by deciphering the Haruna Nijo genome" />
    <meta property="og:description" content="Abstract.  Full-length (FL) cDNA sequences provide the most reliable evidence for the presence of genes in genomes. In this report, detailed gene structures of " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/1/21/2462418" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-11-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Kazuhiro" /><meta name="citation_author_institution" content="Institute of Plant Science and Resources, Okayama University, Kurashiki 710-0046, Japan" /><meta name="citation_author" content="Tanaka, Tsuyoshi" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba 305-8602, Japan" /><meta name="citation_author" content="Shigenobu, Shuji" /><meta name="citation_author_institution" content="National Institute for Basic Biology, Okazaki 444-8585, Japan" /><meta name="citation_author" content="Motoi, Yuka" /><meta name="citation_author_institution" content="Institute of Plant Science and Resources, Okayama University, Kurashiki 710-0046, Japan" /><meta name="citation_author" content="Wu, Jianzhong" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba 305-8602, Japan" /><meta name="citation_author" content="Itoh, Takeshi" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, Tsukuba 305-8602, Japan" /><meta name="citation_title" content="Improvement of barley genome annotations by deciphering the Haruna Nijo genome" /><meta name="citation_firstpage" content="21" /><meta name="citation_lastpage" content="28" /><meta name="citation_doi" content="10.1093/dnares/dsv033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2016/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evolution of the grain dispersal system in barley;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=162;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_publisher=Springer, Heidelberg;  citation_title=Biotechnological approaches to barley improvement, biotechnology in agriculture and forestry;  citation_year=2014;  citation_pages=21-36; " /><meta name="citation_reference" content="citation_title=A physical, genetic and functional sequence assembly of the barley genome; citation_author=The International Barley Genome Sequencing Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=711-6; " /><meta name="citation_reference" content="citation_title=A sequence-ready physical map of barley anchored genetically by two million SNPs;  citation_journal_title=Plant Physiol.;  citation_year=2014;  citation_volume=164;  citation_pages=412-23; " /><meta name="citation_reference" content="citation_title=Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ);  citation_journal_title=Plant J.;  citation_year=2013;  citation_volume=76;  citation_pages=718-27; " /><meta name="citation_reference" content="citation_title=The draft genome of Tibetan hulless barley reveals adaptive patterns to the high stressful Tibetan Plateau;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=1095-100; " /><meta name="citation_reference" content="citation_title=Worldwide distribution of β-amylase thermostability in barley;  citation_journal_title=Euhytica;  citation_year=2001;  citation_volume=121;  citation_pages=225-8; " /><meta name="citation_reference" content="citation_title=Discovery of novel Bmy1 alleles increasing β-amylase activity in Chinese landraces and Tibetan wild barley for improvement of malting quality via MAS;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e72875" /><meta name="citation_reference" content="citation_title=Mapping and validation of chromosome regions associated with high malt extract in barley (Hordeum vulgare L.);  citation_journal_title=Aust. J. Agr. Res.;  citation_year=2003;  citation_volume=54;  citation_pages=1223-40; " /><meta name="citation_reference" content="citation_title=A high-density transcript linkage map of barley derived from a single population;  citation_journal_title=Heredity (Edinb);  citation_year=2009;  citation_volume=103;  citation_pages=110-7; " /><meta name="citation_reference" content="citation_title=Development and implementation of high-throughput SNP genotyping in barley;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=582" /><meta name="citation_reference" content="citation_title=Construction and characterization of a bacterial artificial chromosome (BAC) library for Japanese malting barley ‘Haruna Nijo;  citation_journal_title=Breed. Sci.;  citation_year=2007;  citation_volume=57;  citation_pages=29-38; " /><meta name="citation_reference" content="citation_title=Development of 5006 full-length CDNAs in barley: a tool for accessing cereal genomics resources;  citation_journal_title=DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=81-9; " /><meta name="citation_reference" content="citation_title=Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries;  citation_journal_title=Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=20-8; " /><meta name="citation_reference" content="citation_title=Development of genetic map and alignment of recombinant chromo-some substitution lines from a cross of EST donors by high accuracy SNP typing in barley;  citation_journal_title=Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=728-37; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=CENSOR-a program for identification and elimination of repetitive elements from DNA sequences;  citation_journal_title=Comput. Chem.;  citation_year=1996;  citation_volume=20;  citation_pages=119-21; " /><meta name="citation_reference" content="citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome; citation_author=The International Wheat Genome Sequencing Consortium;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345; " /><meta name="citation_reference" content="citation_title=Draft genome of the wheat A-genome progenitor Triticum urartu;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=Aegilops tauschii draft genome sequence reveals a gene repertoire for wheat adaptation;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=91-5; " /><meta name="citation_reference" content="citation_title=bex-db:Bioinformatics workbench for comprehensive analysis of barley-expressed genes;  citation_journal_title=Breeding Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=430-4; " /><meta name="citation_reference" content="citation_title=EST_GENOME: a program to align spliced DNA sequences to unspliced genomic DNA;  citation_journal_title=Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=477-8; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina Sequence Data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=ToHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotech.;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=AgBase: a functional genomics resource for agriculture;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=229" /><meta name="citation_reference" content="citation_title=The SILVA ribosomal RNA gene database project: improved data processing and web-based tools;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D590-6; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-65; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=Highly diversified molecular evolution of downstream transcription start sites in Oryza sativa and Arabidopsis thaliana;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=1316-24; " /><meta name="citation_reference" content="citation_title=Cleistogamous flowering in barley arises from the suppression of microRNA-guided HvAP2 mRNA cleavage;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=490-5; " /><meta name="citation_reference" content="citation_title=Ribosomal RNA genes on homoeologous chromosomes of groups 5 and 6 in hexaploid wheat;  citation_journal_title=Heredity;  citation_year=1976;  citation_volume=37;  citation_pages=377-85; " /><meta name="citation_reference" content="citation_title=Next-generation survey sequencing and the molecular organization of wheat chromosome 6B;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=103-14; " /><meta name="citation_reference" content="citation_title=Characterization of internal structure of the nucleolar organizing region in rice (Oryza sativa L.);  citation_journal_title=Cytonegetic Genome Res.;  citation_year=2008;  citation_volume=121;  citation_pages=282-5; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=The Generic genome browser: a building block for a model organism system database;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1599-610; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/1/21/17462847/dsv033.pdf" /><meta name="description" content="Abstract.  Full-length (FL) cDNA sequences provide the most reliable evidence for the presence of genes in genomes. In this report, detailed gene structures of " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/1/21/2462418" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/1/21/2462418" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/1/21/2462418");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv033';
    </script>
<title>
	Improvement of barley genome annotations by deciphering the Haruna Nijo genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462418" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd9cPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBLYXp1aGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBQbGFudCBTY2llbmNlIGFuZCBSZXNvdXJjZXMsIE9rYXlhbWEgVW5pdmVyc2l0eSwgS3VyYXNoaWtpIDcxMC0wMDQ2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmFrYSwgVHN1eW9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIFRzdWt1YmEgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpZ2Vub2J1LCBTaHVqaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgQmFzaWMgQmlvbG9neSwgT2themFraSA0NDQtODU4NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3RvaSwgWXVrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBQbGFudCBTY2llbmNlIGFuZCBSZXNvdXJjZXMsIE9rYXlhbWEgVW5pdmVyc2l0eSwgS3VyYXNoaWtpIDcxMC0wMDQ2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild1LCBKaWFuemhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIFRzdWt1YmEgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgVGFrZXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgVHN1a3ViYSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkltcHJvdmVtZW50IG9mIGJhcmxleSBnZW5vbWUgYW5ub3RhdGlvbnMgYnkgZGVjaXBoZXJpbmcgdGhlIEhhcnVuYSBOaWpvIGdlbm9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdjAzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHRoZSBncmFpbiBkaXNwZXJzYWwgc3lzdGVtIGluIGJhcmxleTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjI7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIEhlaWRlbGJlcmc7ICBjaXRhdGlvbl90aXRsZT1CaW90ZWNobm9sb2dpY2FsIGFwcHJvYWNoZXMgdG8gYmFybGV5IGltcHJvdmVtZW50LCBiaW90ZWNobm9sb2d5IGluIGFncmljdWx0dXJlIGFuZCBmb3Jlc3RyeTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeXNpY2FsLCBnZW5ldGljIGFuZCBmdW5jdGlvbmFsIHNlcXVlbmNlIGFzc2VtYmx5IG9mIHRoZSBiYXJsZXkgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9VGhlIEludGVybmF0aW9uYWwgQmFybGV5IEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTE7ICBjaXRhdGlvbl9wYWdlcz03MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZXF1ZW5jZS1yZWFkeSBwaHlzaWNhbCBtYXAgb2YgYmFybGV5IGFuY2hvcmVkIGdlbmV0aWNhbGx5IGJ5IHR3byBtaWxsaW9uIFNOUHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTQxMi0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5jaG9yaW5nIGFuZCBvcmRlcmluZyBOR1MgY29udGlnIGFzc2VtYmxpZXMgYnkgcG9wdWxhdGlvbiBzZXF1ZW5jaW5nIChQT1BTRVEpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTcxOC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBvZiBUaWJldGFuIGh1bGxlc3MgYmFybGV5IHJldmVhbHMgYWRhcHRpdmUgcGF0dGVybnMgdG8gdGhlIGhpZ2ggc3RyZXNzZnVsIFRpYmV0YW4gUGxhdGVhdTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xMDk1LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ybGR3aWRlIGRpc3RyaWJ1dGlvbiBvZiDOsi1hbXlsYXNlIHRoZXJtb3N0YWJpbGl0eSBpbiBiYXJsZXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1aHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2Ygbm92ZWwgQm15MSBhbGxlbGVzIGluY3JlYXNpbmcgzrItYW15bGFzZSBhY3Rpdml0eSBpbiBDaGluZXNlIGxhbmRyYWNlcyBhbmQgVGliZXRhbiB3aWxkIGJhcmxleSBmb3IgaW1wcm92ZW1lbnQgb2YgbWFsdGluZyBxdWFsaXR5IHZpYSBNQVM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzI4NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGFuZCB2YWxpZGF0aW9uIG9mIGNocm9tb3NvbWUgcmVnaW9ucyBhc3NvY2lhdGVkIHdpdGggaGlnaCBtYWx0IGV4dHJhY3QgaW4gYmFybGV5IChIb3JkZXVtIHZ1bGdhcmUgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BdXN0LiBKLiBBZ3IuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjIzLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSB0cmFuc2NyaXB0IGxpbmthZ2UgbWFwIG9mIGJhcmxleSBkZXJpdmVkIGZyb20gYSBzaW5nbGUgcG9wdWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHkgKEVkaW5iKTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xMTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGltcGxlbWVudGF0aW9uIG9mIGhpZ2gtdGhyb3VnaHB1dCBTTlAgZ2Vub3R5cGluZyBpbiBiYXJsZXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhIGJhY3RlcmlhbCBhcnRpZmljaWFsIGNocm9tb3NvbWUgKEJBQykgbGlicmFyeSBmb3IgSmFwYW5lc2UgbWFsdGluZyBiYXJsZXkg4oCYSGFydW5hIE5pam87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIDUwMDYgZnVsbC1sZW5ndGggQ0ROQXMgaW4gYmFybGV5OiBhIHRvb2wgZm9yIGFjY2Vzc2luZyBjZXJlYWwgZ2Vub21pY3MgcmVzb3VyY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgc2VxdWVuY2UgYW5hbHlzaXMgb2YgMjQsNzgzIGJhcmxleSBmdWxsLWxlbmd0aCBjRE5BcyBkZXJpdmVkIGZyb20gMTIgY2xvbmUgbGlicmFyaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0yMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBnZW5ldGljIG1hcCBhbmQgYWxpZ25tZW50IG9mIHJlY29tYmluYW50IGNocm9tby1zb21lIHN1YnN0aXR1dGlvbiBsaW5lcyBmcm9tIGEgY3Jvc3Mgb2YgRVNUIGRvbm9ycyBieSBoaWdoIGFjY3VyYWN5IFNOUCB0eXBpbmcgaW4gYmFybGV5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NzI4LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRU5TT1ItYSBwcm9ncmFtIGZvciBpZGVudGlmaWNhdGlvbiBhbmQgZWxpbWluYXRpb24gb2YgcmVwZXRpdGl2ZSBlbGVtZW50cyBmcm9tIEROQSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXB1dC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tZS1iYXNlZCBkcmFmdCBzZXF1ZW5jZSBvZiB0aGUgaGV4YXBsb2lkIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSkgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9VGhlIEludGVybmF0aW9uYWwgV2hlYXQgR2Vub21lIFNlcXVlbmNpbmcgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBvZiB0aGUgd2hlYXQgQS1nZW5vbWUgcHJvZ2VuaXRvciBUcml0aWN1bSB1cmFydHU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz04Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWVnaWxvcHMgdGF1c2NoaWkgZHJhZnQgZ2Vub21lIHNlcXVlbmNlIHJldmVhbHMgYSBnZW5lIHJlcGVydG9pcmUgZm9yIHdoZWF0IGFkYXB0YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz05MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1iZXgtZGI6QmlvaW5mb3JtYXRpY3Mgd29ya2JlbmNoIGZvciBjb21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIGJhcmxleS1leHByZXNzZWQgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyZWVkaW5nIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz00MzAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNUX0dFTk9NRTogYSBwcm9ncmFtIHRvIGFsaWduIHNwbGljZWQgRE5BIHNlcXVlbmNlcyB0byB1bnNwbGljZWQgZ2Vub21pYyBETkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ3Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmltbW9tYXRpYzogYSBmbGV4aWJsZSB0cmltbWVyIGZvciBJbGx1bWluYSBTZXF1ZW5jZSBEYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIxMTQtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvSGF0MjogYWNjdXJhdGUgYWxpZ25tZW50IG9mIHRyYW5zY3JpcHRvbWVzIGluIHRoZSBwcmVzZW5jZSBvZiBpbnNlcnRpb25zLCBkZWxldGlvbnMgYW5kIGdlbmUgZnVzaW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdCBhc3NlbWJseSBhbmQgcXVhbnRpZmljYXRpb24gYnkgUk5BLVNlcSByZXZlYWxzIHVuYW5ub3RhdGVkIHRyYW5zY3JpcHRzIGFuZCBpc29mb3JtIHN3aXRjaGluZyBkdXJpbmcgY2VsbCBkaWZmZXJlbnRpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01MTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm9TY2FuIDU6IGdlbm9tZS1zY2FsZSBwcm90ZWluIGZ1bmN0aW9uIGNsYXNzaWZpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEyMzYtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFnQmFzZTogYSBmdW5jdGlvbmFsIGdlbm9taWNzIHJlc291cmNlIGZvciBhZ3JpY3VsdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU0lMVkEgcmlib3NvbWFsIFJOQSBnZW5lIGRhdGFiYXNlIHByb2plY3Q6IGltcHJvdmVkIGRhdGEgcHJvY2Vzc2luZyBhbmQgd2ViLWJhc2VkIHRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENTkwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HTUFQOiBhIGdlbm9taWMgbWFwcGluZyBhbmQgYWxpZ25tZW50IHByb2dyYW0gZm9yIG1STkEgYW5kIEVTVCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTg1OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IGRpdmVyc2lmaWVkIG1vbGVjdWxhciBldm9sdXRpb24gb2YgZG93bnN0cmVhbSB0cmFuc2NyaXB0aW9uIHN0YXJ0IHNpdGVzIGluIE9yeXphIHNhdGl2YSBhbmQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTEzMTYtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsZWlzdG9nYW1vdXMgZmxvd2VyaW5nIGluIGJhcmxleSBhcmlzZXMgZnJvbSB0aGUgc3VwcHJlc3Npb24gb2YgbWljcm9STkEtZ3VpZGVkIEh2QVAyIG1STkEgY2xlYXZhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NDkwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpYm9zb21hbCBSTkEgZ2VuZXMgb24gaG9tb2VvbG9nb3VzIGNocm9tb3NvbWVzIG9mIGdyb3VwcyA1IGFuZCA2IGluIGhleGFwbG9pZCB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTE5NzY7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0zNzctODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQtZ2VuZXJhdGlvbiBzdXJ2ZXkgc2VxdWVuY2luZyBhbmQgdGhlIG1vbGVjdWxhciBvcmdhbml6YXRpb24gb2Ygd2hlYXQgY2hyb21vc29tZSA2QjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgaW50ZXJuYWwgc3RydWN0dXJlIG9mIHRoZSBudWNsZW9sYXIgb3JnYW5pemluZyByZWdpb24gaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b25lZ2V0aWMgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9MjgyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYXAtYmFzZWQgc2VxdWVuY2Ugb2YgdGhlIHJpY2UgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBSaWNlIEdlbm9tZSBTZXF1ZW5jaW5nIFByb2plY3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz03OTMtODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2VuZXJpYyBnZW5vbWUgYnJvd3NlcjogYSBidWlsZGluZyBibG9jayBmb3IgYSBtb2RlbCBvcmdhbmlzbSBzeXN0ZW0gZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTU5OS02MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy8xLzIxLzE3NDYyODQ3L2RzdjAzMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRnVsbC1sZW5ndGggKEZMKSBjRE5BIHNlcXVlbmNlcyBwcm92aWRlIHRoZSBtb3N0IHJlbGlhYmxlIGV2aWRlbmNlIGZvciB0aGUgcHJlc2VuY2Ugb2YgZ2VuZXMgaW4gZ2Vub21lcy4gSW4gdGhpcyByZXBvcnQsIGRldGFpbGVkIGdlbmUgc3RydWN0dXJlcyBvZiAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI0NjI0MTgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkmJM+nPjW9snLu06wOjs6tCiMDgs=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f21%2f2462418" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f21%2f2462418" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83835" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&amp;Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/1">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196927">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196927">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196929">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196929">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196935">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196935">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196968">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196968">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196998">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196998">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197001">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197001">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197003">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197003">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197005">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197005">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/1/11/2462413">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/1/29/2462430">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Improvement of barley genome annotations by deciphering the Haruna Nijo genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhiro Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Plant Science and Resources</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">Kurashiki 710-0046</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>Shared first authors.</p>
                </div>
                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhiro+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuhiro Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyoshi Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">Tsukuba 305-8602</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>Shared first authors.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyoshi+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tsuyoshi Tanaka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuji Shigenobu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Institute for Basic Biology</div>, <div class="addr-line">Okazaki 444-8585</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuji+Shigenobu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shigenobu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shigenobu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shuji Shigenobu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuka Motoi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Plant Science and Resources</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">Kurashiki 710-0046</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuka+Motoi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Motoi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Motoi Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuka Motoi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianzhong Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">Tsukuba 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianzhong+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jianzhong Wu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">Tsukuba 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeshi Itoh</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Shared first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 21–28, <a href='https://doi.org/10.1093/dnares/dsv033'>https://doi.org/10.1093/dnares/dsv033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 November 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 August 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462418" href="/dnaresearch/article-pdf/23/1/21/17462847/dsv033.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kazuhiro Sato, Tsuyoshi Tanaka, Shuji Shigenobu, Yuka Motoi, Jianzhong Wu, Takeshi Itoh;  Improvement of barley genome annotations by deciphering the Haruna Nijo genome, <em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 21–28, <a href='https://doi.org/10.1093/dnares/dsv033'>https://doi.org/10.1093/dnares/dsv033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462418?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Improvement%20of%20barley%20genome%20annotations%20by%20deciphering%20the%20Haruna%20Nijo%20genome&amp;publicationDate=2015-11-29&amp;volumeNum=23&amp;issueNum=1&amp;author=Sato%2C%20Kazuhiro%3B%20Tanaka%2C%20Tsuyoshi&amp;startPage=21&amp;endPage=28&amp;contentId=10.1093%2Fdnares%2Fdsv033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f21%2f2462418" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87196927 id="87196927" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Full-length (FL) cDNA sequences provide the most reliable evidence for the presence of genes in genomes. In this report, detailed gene structures of barley, whole genome shotgun (WGS) and additional transcript data of the cultivar Haruna Nijo were quality controlled and compared with the published Morex genome information. Haruna Nijo scaffolds have longer total sequence length with much higher N50 and fewer sequences than those in Morex WGS contigs. The longer Haruna Nijo scaffolds provided efficient FLcDNA mapping, resulting in high coverage and detection of the transcription start sites. In combination with FLcDNAs and RNA-Seq data from four different tissue samples of Haruna Nijo, we identified 51,249 gene models on 30,606 loci. Overall sequence similarity between Haruna Nijo and Morex genome was 95.99%, while that of exon regions was higher (99.71%). These sequence and annotation data of Haruna Nijo are combined with Morex genome data and released from a genome browser. The genome sequence of Haruna Nijo may provide detailed gene structures in addition to the current Morex barley genome information.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Hordeum"><em>Hordeum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-Seq">RNA-Seq</a></div>                            <h2 scrollto-destination=87196929 id="87196929" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Barley (<em>Hordeum vulgare</em> L.) is used for many purposes including human food, malting and animal feed. Coupled with its wide adaptability in environments ranging from the highlands of Africa at the equator to the Arctic Regions in Scandinavia, barley is the fourth most important cereal crop (<a class="link link-uri" href="http://faostat.fao.org">http://faostat.fao.org</a>, 7 November 2015, date accessed). In addition, barley is an ancient crop that was domesticated ca. 10,000 yrs ago in the Fertile Crescent.<sup><a reveal-id="DSV033C1" data-open="DSV033C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Decades of collection and curation of barley germplasm have resulted in substantial germplasm collections including: IPK (Germany), USDA-ARS (USA) and Okayama University (Japan).<sup><a reveal-id="DSV033C2" data-open="DSV033C2" class="link link-ref link-reveal xref-bibr">2</a></sup> To fully exploit these <em>in situ</em> and <em>ex situ</em> collections for barley breeding and gene discovery, barley genome sequences of multiple genotypes are needed.</p><p>To generate the first genome sequence of barley, the International Barley Sequencing Consortium (IBSC) conducted BAC fingerprinting, BAC end sequencing, whole genome shotgun (WGS) sequencing, RNA-Seq analysis and genetic mapping and integrated these data to develop a gene-based genome sequence of the North American six-row spring malting barley cultivar Morex.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Morex traces to barley germplasm of Manchurian origin with six-row inflorescence, which is not common for malting barleys other than the USA. Currently, Morex is the reference genome for barley genetics studies, and additional efforts are in progress to improve the Morex genome sequence.<sup><a reveal-id="DSV033C4" data-open="DSV033C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV033C5" data-open="DSV033C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Three other barley genomes including Bowman, Barke and a Tibetan hulless genotype have been WGS sequenced<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV033C6" data-open="DSV033C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and provide additional sequence information for comparison to Morex.</p><p>Haruna Nijo is a Japanese malting barley cultivar exhibiting excellent malting quality for brewing beer. Historically, Japanese malting barleys have been developed by crossing European malting barley cultivars and Japanese landraces and selecting for malting quality and adaptation to Japanese environments. Haruna Nijo was also developed by this approach in 1979 and has been extensively used as a foundation genotype of current Japanese breeding. Haruna Nijo has a high thermostability of β-amylase,<sup><a reveal-id="DSV033C7" data-open="DSV033C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and it also shows different Bmy1 allele for β-amylase activity from Morex.<sup><a reveal-id="DSV033C8" data-open="DSV033C8" class="link link-ref link-reveal xref-bibr">8</a></sup> It is also shown that Haruna Nijo has a high malt extract (related to beer production) by QTL analysis.<sup><a reveal-id="DSV033C9" data-open="DSV033C9" class="link link-ref link-reveal xref-bibr">9</a></sup> To enhance the utility of Haruna Nijo for breeding, a suite of genomics resources were developed including: an EST collection, transcript map construction,<sup><a reveal-id="DSV033C10" data-open="DSV033C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV033C11" data-open="DSV033C11" class="link link-ref link-reveal xref-bibr">11</a></sup> a BAC library,<sup><a reveal-id="DSV033C12" data-open="DSV033C12" class="link link-ref link-reveal xref-bibr">12</a></sup> 454-based WGS sequencing<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and full-length (FL) cDNA sequencing and analysis<sup><a reveal-id="DSV033C13" data-open="DSV033C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV033C14" data-open="DSV033C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (available at <a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/bexdb/">http://barleyflc.dna.affrc.go.jp/bexdb/</a>, 7 November 2015, date accessed, with functions of expression profiling and genome browsing).</p><p>In the case of barley genome resources, the Morex reference genome and the Haruna Nijo FLcDNA sequences provide unique resources in different genotypes. Thus, to enhance the utility of both resources they need to be compared and integrated. Sequence comparisons of the two different malting barley haplotypes will be much more useful than single haplotype information, as they provide the basis of structural and functional allelic diversity. Previous comparisons of WGS data and FLcDNA sequences from Haruna Nijo and the genome sequence from Morex demonstrated a large amount of sequence polymorphisms between the two genotypes.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> For example, the genetic distance (dissimilarity) between Morex and Haruna Nijo by 1,536 SNPs of Illumina GoldenGate Assay is 0.50, which is larger than the difference (0.48) between Morex and Akashinriki (Japanese food barley).<sup><a reveal-id="DSV033C15" data-open="DSV033C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Thus, the Haruna Nijo FLcDNA and WGS sequences may provide useful gene annotations for the reference genotype Morex and a resource for future breeding and gene discovery activities.</p><p>Our overall goal is to provide genome information of Haruna Nijo to annotate gene structures on the genome sequence of Morex. The four specific objectives of this study were to (i) generate high-quality genomic sequence of Haruna Nijo, (ii) map the precise position of genes identified in the Haruna Nijo genome, (iii) compare gene information between Haruna Nijo and Morex, and (iv) develop gene models based on the Haruna Nijo sequence data.</p>                            <h2 scrollto-destination=87196935 id="87196935" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196936 id="87196936" class="section-title" data-legacy-id=s2a>2.1. Plant materials and nucleic acid isolation</h3>
<p>For DNA and RNA isolation from seedling tissues, seeds of Haruna Nijo were germinated on moist filter paper in Petri dishes at 20°C in the dark. For DNA isolation, shoots of ca. 5 cm were harvested and high-quality genomic DNA was isolated by DNeasy Plant Mini Kit (QIAGEN K.K., Japan). For RNA isolation, seedling shoot and seedling root tissues were sampled from plants exhibiting 5 cm shoots. For the RNA samples from immature spike and immature seed samples, barley seeds were planted in pot with soil mixture (N: 120 mg l<sup>−1</sup>; P: 100 mg l<sup>−1</sup>; K: 160 mg l<sup>−1</sup>, pH 5.5) and grown in the greenhouse at 20/15°C day/night temperature under natural light conditions. Immature spike samples were harvested from the leaf sheath 5 days before heading. Immature seeds were sampled 35 days after flowering (soft dough stage). Total RNA from immature seeds was isolated by TRIzol<sup>®</sup> Reagent (Life Technologies, Japan) following the Plant RNA Isolation protocol. All RNA samples were purified by RNeasy Plant Mini Kit (QIAGEN K.K., Japan), and DNA was removed by RNase-Free DNase Set (QIAGEN K.K., Japan).</p>                            <h3 scrollto-destination=87196938 id="87196938" class="section-title" data-legacy-id=s2b>2.2. Library development</h3>
                            <h4 scrollto-destination=87196939 id="87196939" class="section-title" data-legacy-id=s2b1>2.2.1. 454 long paired-end library</h4>
<p>The high-quality genomic DNA was fragmented in 8 k and 20 k by the Hydroshear (Digilab Inc., Holliston, MA, USA). The library was developed by a library preparation kit (Roche diagnostics, Japan) for long paired-end libraries (8 k and 20 k). Circularization was performed according to the Roche Paired End Library Protocol, using Roche circularization adapters (Paired End Library Preparation Method Manual 20 kb and 8 kb Span; Roche Diagnostics, October 2009, Steps 3.1–3.7.3). Subsequently, the circularized fragments were fragmented again by nebulization to develop a sequencing library.</p>                            <h4 scrollto-destination=87196941 id="87196941" class="section-title" data-legacy-id=s2b2>2.2.2. Illumina HiSeq sequencing library</h4>
<p>DNA (2 µg) was fragmented by nebulization. Libraries were prepared according to the manufacturer's instruction ‘Preparing Samples for Paired-End Sequencing, Part # 1005063 Rev. A June 2008’ of TruSeq DNA Sample Prep Kit (Illumina Japan). DNA fragments were size selected (500 bp) following the Low-Throughput Protocol of TruSeq DNA Sample Prep Kit (Illumina Japan). The quality of the library (fragment length distribution) was checked by Agilent Bioanalyzer High Sensitivity DNA Assay (Agilent Technologies, Japan) and KAPA Library Quantification Kit (KK4835, Kapa Biosystems, MA, USA).</p>                            <h4 scrollto-destination=87196943 id="87196943" class="section-title" data-legacy-id=s2b3>2.2.3. MiSeq RNA sequencing library</h4>
<p>Libraries for RNA-Seq analysis were developed from each RNA sample using TruSeq RNA Sample Prep Kit V2 (Illumina Japan). The protocol of TruSeq RNA Sample Preparation V2 Guide (Illumina Japan) was used with modification of fragment isolation from agarose gel electrophoresis and elution in EB (elution buffer) following the method in the TruSeq Sample Preparation Guide (Illumina Japan). The library was quantified with Agilent High Sensitivity DNA Kit (Agilent Technologies Japan) and Qubit 2.0 Fluorometer (Life Technologies Japan).</p>                            <h3 scrollto-destination=87196945 id="87196945" class="section-title" data-legacy-id=s2c>2.3. Sequencing</h3>
                            <h4 scrollto-destination=87196946 id="87196946" class="section-title" data-legacy-id=s2c1>2.3.1. 454 FLX Titanium platform</h4>
<p>Each library was emulsion PCR amplified. The PCR-amplified fragments on beads were washed, and the bead number was counted using a Coulter Counter Z1 single threshold instrument (Beckman Coulter Japan). The appropriate number of beads was applied on a pico titre plate according to the manufacturer's protocol. The FLX Titanium platform was used for sequencing (average read length 500 bp). The pyrosequencing reaction data were base-called to generate sff format files using the software installed on the analysis server of the 454 sequencer (Roche diagnostics, Japan).</p>                            <h4 scrollto-destination=87196948 id="87196948" class="section-title" data-legacy-id=s2c2>2.3.2. Illumina HiSeq platform</h4>
<p>The shotgun library was sequenced on an Illumina HiSeq 2000 to produce 2 × 101 paired-end reads. Raw data processing, base calling and quality control were performed with manufacture's standard pipeline. The quality of the output sequences was inspected using the FastQC program (<a class="link link-uri" href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc/">http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc/</a>, 7 November 2015, date accessed).</p>                            <h4 scrollto-destination=87196950 id="87196950" class="section-title" data-legacy-id=s2c3>2.3.3. Illumina MiSeq RNA sequencing</h4>
<p>The RNA-Seq library was sequenced with MiSeq Reagent Kit V3 (2 × 300 bp cycles) on MiSeq NGS system according to the MiSeq System User Guide (Illumina Japan) and fastq files were generated from both ends of the fragments.</p>                            <h3 scrollto-destination=87196952 id="87196952" class="section-title" data-legacy-id=s2d>2.4. Assembly, sequence trimming and repeat masking</h3>
                            <h4 scrollto-destination=87196953 id="87196953" class="section-title" data-legacy-id=s2d1>2.4.1. WGS assembly</h4>
<p>Illimina PE reads, 454 single-end reads and 454 long paired-end reads were hybrid assembled by the <em>de novo</em> assemble algorithm of CLC Assembly Cell ver. 3.2.2 installed on a linux server with a main memory of 256 Gb.</p>                            <h4 scrollto-destination=87196955 id="87196955" class="section-title" data-legacy-id=s2d2>2.4.2. Sequence trimming and repeat masking</h4>
<p>Human, fungi (33 species) and microbial (2,777 species) nucleotide sequence data were obtained from NCBI. Each assembled contig sequence was queried on sequences of these species, and the contigs showing &gt;80% identity and &gt;50% of cumulative coverage by blastn<sup><a reveal-id="DSV033C16" data-open="DSV033C16" class="link link-ref link-reveal xref-bibr">16</a></sup> were assumed as alien sequences against the barley genome. For repeat masking, fasta files of Triticeae repeat sequence database (TREP Release 10: <em>n</em> = 1,717) were downloaded from GrainGenes (<a class="link link-uri" href="http://wheat.pw.usda.gov/ITMI/Repeats/">http://wheat.pw.usda.gov/ITMI/Repeats/</a>, 7 November 2015, date accessed). In addition, new repetitive elements were generated from the Haruna Nijo genome assembly by RepeatModeler (<a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler.html">http://www.repeatmasker.org/RepeatModeler.html</a>, 7 November 2015, date accessed). These two libraries were used for repeat masking the Haruna Nijo assembly by censor<sup><a reveal-id="DSV033C17" data-open="DSV033C17" class="link link-ref link-reveal xref-bibr">17</a></sup> with -mode norm mode. For the analysis of repeat distribution, only results of repeat masking by the TREP library were used. For comparison, genomes of <em>Triticum aestivum</em>,<sup><a reveal-id="DSV033C18" data-open="DSV033C18" class="link link-ref link-reveal xref-bibr">18</a></sup><em>Triticum urartu</em><sup><a reveal-id="DSV033C19" data-open="DSV033C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and <em>Aegilops tauschii</em><sup><a reveal-id="DSV033C20" data-open="DSV033C20" class="link link-ref link-reveal xref-bibr">20</a></sup> were repeat masked in a similar manner as the Haruna Nijo genome.</p>                            <h3 scrollto-destination=87196957 id="87196957" class="section-title" data-legacy-id=s2e>2.5. FLcDNA/RNA-Seq mapping</h3>
                            <h4 scrollto-destination=87196958 id="87196958" class="section-title" data-legacy-id=s2e1>2.5.1. FLcDNA mapping</h4>
<p>Of the two independent projects,<sup><a reveal-id="DSV033C13" data-open="DSV033C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV033C14" data-open="DSV033C14" class="link link-ref link-reveal xref-bibr">14</a></sup> 5,006 FLcDNAs were retrieved from NCBI<sup><a reveal-id="DSV033C13" data-open="DSV033C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and 23,614 FLcDNAs<sup><a reveal-id="DSV033C14" data-open="DSV033C14" class="link link-ref link-reveal xref-bibr">14</a></sup> were downloaded from the bex-db (<a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/bexdb/">http://barleyflc.dna.affrc.go.jp/bexdb/</a>).<sup><a reveal-id="DSV033C21" data-open="DSV033C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Morex genome assembly (contigs) was retrieved from the PGSB PlantsDB database (<a class="link link-uri" href="http://pgsb.helmholtz-muenchen.de/plant/plantsdb.jsp">http://pgsb.helmholtz-muenchen.de/plant/plantsdb.jsp</a>, 7 November 2015, date accessed).<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> FLcDNAs were mapped both on the Haruna Nijo scaffolds and on the Morex contigs using blast+ with the parameters ‘-task blastn -evalue 10e<sup>−10</sup> -lcase_masking’, and with est2genome<sup><a reveal-id="DSV033C22" data-open="DSV033C22" class="link link-ref link-reveal xref-bibr">22</a></sup> in the EMBOSS package with the parameters ‘-align –mode both -gappenalty 8 -mismatch 6 -minscore 10’. FLcDNAs mapped to scaffolds with &gt;95% identity and &gt;50% coverage were accepted. Mapped regions with the highest identity were adopted for each FLcDNA. Overlapped FLcDNA sequences were clustered onto a locus on Haruna Nijo genome.</p>                            <h4 scrollto-destination=87196960 id="87196960" class="section-title" data-legacy-id=s2e2>2.5.2. RNA-Seq analysis</h4>
<p>After trimming of low-quality nucleotide and adapter sequences by Trimmomatic,<sup><a reveal-id="DSV033C23" data-open="DSV033C23" class="link link-ref link-reveal xref-bibr">23</a></sup> reads derived from rRNA sequences were discarded by Bowtie2.<sup><a reveal-id="DSV033C24" data-open="DSV033C24" class="link link-ref link-reveal xref-bibr">24</a></sup> A set of programs, Bowtie2,<sup><a reveal-id="DSV033C24" data-open="DSV033C24" class="link link-ref link-reveal xref-bibr">24</a></sup> TopHat2<sup><a reveal-id="DSV033C25" data-open="DSV033C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and Cufflinks,<sup><a reveal-id="DSV033C26" data-open="DSV033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> were used to map reads on the Haruna Nijo scaffolds. Gene structures derived from FLcDNA mapping were utilized to develop consensus gene models of the four library reads. To predict ORFs on the gene structures, blastx searches were conducted to RefSeq and UniProtKB data sets. From the best hit of blastx results, the most reliable ORFs were determined with at least 70 amino acids in length. To assign gene function, InterProScan 5<sup><a reveal-id="DSV033C27" data-open="DSV033C27" class="link link-ref link-reveal xref-bibr">27</a></sup> was conducted on the predicted ORFs. Based on the InterPro domain, GO terms were assigned to each ORF, and GOslim<sup><a reveal-id="DSV033C28" data-open="DSV033C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (<a class="link link-uri" href="http://agbase.msstate.edu/cgi-bin/tools/goslimviewer_select.pl">http://agbase.msstate.edu/cgi-bin/tools/goslimviewer_select.pl</a>, 7 November 2015, date accessed) was conducted.</p>                            <h4 scrollto-destination=87196962 id="87196962" class="section-title" data-legacy-id=s2e3>2.5.3. RNA gene prediction</h4>
<p>From the silva database<sup><a reveal-id="DSV033C29" data-open="DSV033C29" class="link link-ref link-reveal xref-bibr">29</a></sup> (<a class="link link-uri" href="http://www.arb-silva.de/">http://www.arb-silva.de/</a>, 7 November 2015, date accessed), 17 barley rRNA sequences were downloaded. They were mapped on the Haruna Nijo scaffolds using blast+ with the parameters ‘-task blastn -evalue 0.01 -lcase_masking -num_descriptions 100’ and blast hits with &gt;98% identity and &gt;50% coverage were assumed as rRNAs. tRNA genes were predicted using the tRNAscan-SE ver. 1.3.1 program.<sup><a reveal-id="DSV033C30" data-open="DSV033C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Any tRNAs that were annotated as ‘possible pseudogenes’ were not counted.</p>                            <h4 scrollto-destination=87196964 id="87196964" class="section-title" data-legacy-id=s2e4>2.5.4. Genome sequence comparison between Haruna Nijo and Morex</h4>
<p>The Haruna Nijo scaffolds were mapped on the Morex contigs using megablast with the parameters ‘-evalue 10e<sup>−10</sup> -num_descriptions 5’. Since several query sequences were hit on the same target sequence, the best hit regions were identified. If more than one Morex contig was positioned on a Haruna Nijo scaffold, these contigs were aligned on the scaffold allowing 10% overlap of the contig sequences.</p>                            <h4 scrollto-destination=87196966 id="87196966" class="section-title" data-legacy-id=s2e5>2.5.5. Morex gene mapping</h4>
<p>High confidence (HC) and low confidence (LC) genes of Morex were downloaded from the PGSB barley genome database and mapped to the Haruna Nijo scaffolds by GMAP<sup><a reveal-id="DSV033C31" data-open="DSV033C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with &gt;95% identity and &gt;90% coverage.</p>                            <h2 scrollto-destination=87196968 id="87196968" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87196969 id="87196969" class="section-title" data-legacy-id=s3a>3.1. Assembly of the Haruna Nijo genome sequence</h3>
<p>We analysed different platforms of WGS sequences of Haruna Nijo by hybrid assembly. A total of 305 Gbp of sequence data were generated by Illumina HiSeq paired-end reads and 454 Titanium long paired-end reads (8 K and 20 K) (Table <a reveal-id="DSV033TB1" data-open="DSV033TB1" class="link link-reveal link-table xref-fig">1</a>). The hybrid assembly of Illumina and 454 reads, which also include the published 454 Titanium single-end reads (25Gbp),<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> generated a total of 2,055,601,874 bp in 1,753,384 scaffolds (contigs and gaps). After sequence trimming, the Haruna Nijo genome size was 2,005,970,762 bp in 1,712,236 scaffolds (Table <a reveal-id="DSV033TB2" data-open="DSV033TB2" class="link link-reveal link-table xref-fig">2</a>). The N50 was ∼3.5 kb, which is 2.5 times larger than the published Morex WGS contigs.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The average sequence length in the Haruna Nijo scaffolds was longer than the Morex WGS contigs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Compared with Morex WGS contigs, which were assembled with only Illumina paired-end reads, the longer Haruna Nijo WGS assembly is likely due to assembling the Illumina paired-end reads and 454 longer single-end reads. </p>                        <a id="87196971" scrollto-destination="87196971"></a>
<div content-id="DSV033TB1" class="table-modal"><div class="table-wrap"><div id="DSV033TB1" class="table-wrap-title" data-id="DSV033TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sources of WGS sequencing data for Haruna Nijo</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Read </th><th>No. of reads </th><th>No. of bases (bp) </th></tr></thead><tbody><tr><td>Illumina </td><td>Paired end (PE) </td><td>3,005,632,276 </td><td>298,945,920,500 </td></tr><tr><td>454 </td><td>PE 20 kb </td><td>14,242,510 </td><td>4,569,072,624 </td></tr><tr><td>454 </td><td>PE 8 kb </td><td>2,955,788 </td><td>1,016,746,348 </td></tr><tr><td>454 </td><td>Single end 500 bp<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </td><td>70,936,197 </td><td>25,410,875,001 </td></tr></tbody></table></div><div content-id="DSV033TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Read </th><th>No. of reads </th><th>No. of bases (bp) </th></tr></thead><tbody><tr><td>Illumina </td><td>Paired end (PE) </td><td>3,005,632,276 </td><td>298,945,920,500 </td></tr><tr><td>454 </td><td>PE 20 kb </td><td>14,242,510 </td><td>4,569,072,624 </td></tr><tr><td>454 </td><td>PE 8 kb </td><td>2,955,788 </td><td>1,016,746,348 </td></tr><tr><td>454 </td><td>Single end 500 bp<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </td><td>70,936,197 </td><td>25,410,875,001 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196971" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV033TB1" class="table-wrap-title" data-id="DSV033TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sources of WGS sequencing data for Haruna Nijo</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Read </th><th>No. of reads </th><th>No. of bases (bp) </th></tr></thead><tbody><tr><td>Illumina </td><td>Paired end (PE) </td><td>3,005,632,276 </td><td>298,945,920,500 </td></tr><tr><td>454 </td><td>PE 20 kb </td><td>14,242,510 </td><td>4,569,072,624 </td></tr><tr><td>454 </td><td>PE 8 kb </td><td>2,955,788 </td><td>1,016,746,348 </td></tr><tr><td>454 </td><td>Single end 500 bp<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </td><td>70,936,197 </td><td>25,410,875,001 </td></tr></tbody></table></div><div content-id="DSV033TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Read </th><th>No. of reads </th><th>No. of bases (bp) </th></tr></thead><tbody><tr><td>Illumina </td><td>Paired end (PE) </td><td>3,005,632,276 </td><td>298,945,920,500 </td></tr><tr><td>454 </td><td>PE 20 kb </td><td>14,242,510 </td><td>4,569,072,624 </td></tr><tr><td>454 </td><td>PE 8 kb </td><td>2,955,788 </td><td>1,016,746,348 </td></tr><tr><td>454 </td><td>Single end 500 bp<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </td><td>70,936,197 </td><td>25,410,875,001 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196971" target="_blank">View Large</a></div></div>                        <a id="87196972" scrollto-destination="87196972"></a>
<div content-id="DSV033TB2" class="table-modal"><div class="table-wrap"><div id="DSV033TB2" class="table-wrap-title" data-id="DSV033TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Results of whole-genome assembly of Haruna Nijo and Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Data </th><th>Haruna Nijo </th><th>Morex </th></tr></thead><tbody><tr><td>Total length (bp) </td><td>2,005,970,672 </td><td>1,868,648,155 </td></tr><tr><td>N50 bp </td><td>3,539 </td><td>1,425 </td></tr><tr><td>No. of contigs/scaffolds </td><td>1,712,236 </td><td>2,670,738 </td></tr><tr><td>No. of ‘N’ </td><td>225,899,677 </td><td>50,995,654 </td></tr><tr><td>No. of non-‘N’ </td><td>1,780,070,995 </td><td>1,817,652,501 </td></tr><tr><td>GC (%) </td><td>45.0 </td><td>44.4 </td></tr></tbody></table></div><div content-id="DSV033TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Data </th><th>Haruna Nijo </th><th>Morex </th></tr></thead><tbody><tr><td>Total length (bp) </td><td>2,005,970,672 </td><td>1,868,648,155 </td></tr><tr><td>N50 bp </td><td>3,539 </td><td>1,425 </td></tr><tr><td>No. of contigs/scaffolds </td><td>1,712,236 </td><td>2,670,738 </td></tr><tr><td>No. of ‘N’ </td><td>225,899,677 </td><td>50,995,654 </td></tr><tr><td>No. of non-‘N’ </td><td>1,780,070,995 </td><td>1,817,652,501 </td></tr><tr><td>GC (%) </td><td>45.0 </td><td>44.4 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196972" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV033TB2" class="table-wrap-title" data-id="DSV033TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Results of whole-genome assembly of Haruna Nijo and Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Data </th><th>Haruna Nijo </th><th>Morex </th></tr></thead><tbody><tr><td>Total length (bp) </td><td>2,005,970,672 </td><td>1,868,648,155 </td></tr><tr><td>N50 bp </td><td>3,539 </td><td>1,425 </td></tr><tr><td>No. of contigs/scaffolds </td><td>1,712,236 </td><td>2,670,738 </td></tr><tr><td>No. of ‘N’ </td><td>225,899,677 </td><td>50,995,654 </td></tr><tr><td>No. of non-‘N’ </td><td>1,780,070,995 </td><td>1,817,652,501 </td></tr><tr><td>GC (%) </td><td>45.0 </td><td>44.4 </td></tr></tbody></table></div><div content-id="DSV033TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Data </th><th>Haruna Nijo </th><th>Morex </th></tr></thead><tbody><tr><td>Total length (bp) </td><td>2,005,970,672 </td><td>1,868,648,155 </td></tr><tr><td>N50 bp </td><td>3,539 </td><td>1,425 </td></tr><tr><td>No. of contigs/scaffolds </td><td>1,712,236 </td><td>2,670,738 </td></tr><tr><td>No. of ‘N’ </td><td>225,899,677 </td><td>50,995,654 </td></tr><tr><td>No. of non-‘N’ </td><td>1,780,070,995 </td><td>1,817,652,501 </td></tr><tr><td>GC (%) </td><td>45.0 </td><td>44.4 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196972" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87196973 id="87196973" class="section-title" data-legacy-id=s3b>3.2. Repeat content of the Haruna Nijo genome</h3>
<p>Previous studies have shown that the barley genome contains ∼80% repeated elements.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In the published WGS assemblies,<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> the relative frequency was reduced to ∼60%, due to the degradation of redundant repeated sequences. On the assembly of Haruna Nijo WGS data, repeat analysis by the Triticeae repeat library (TREP) detected 55.9% as repeated sequences. After deleting unknown genomic regions (‘N’ sites), 60.8% of the assembled sequences were classified as repeated sequences. The composition and distribution of repeat sequence categories were similar between Haruna Nijo and Morex (Fig. <a reveal-id="DSV033F1" data-open="DSV033F1" class="link link-reveal link-table xref-fig">1</a>). Noteworthy, <em>Gypsy</em>, <em>Copia</em> retrotransposons and the <em>CACTA</em> DNA transposon showed higher frequencies than those normally observed in other Triticeae species. When we compared repeat categories from Haruna Nijo, <em>T. aestivum</em> (common wheat) cv. Chinese Spring, <em>T. urartu</em> and <em>A. tauschii</em>, the frequency of <em>Copia</em> in Haruna Nijo was higher than those in the wheat genomes, while Haruna Nijo contained more <em>CACTA</em> and less <em>Gypsy</em> than those in wheat genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). </p>                        <a id="87196975" scrollto-destination="87196975"></a>
<div id="DSV033F1" data-id="DSV033F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03301.png?Expires=2147483647&Signature=NsOsjzDD0jTux5XpRkbmn1dThCG9d9IZnIT0npsjHtAwb6VrqmYI4ETwY33ZeFzd6OYJxqOV6-xA7FX-775JWntU07UJy5d9Gm0FOZX0kTG~25IjGpq4xKgwpH8vSAFXUpWSl5M~MYyD8YJUJiJq8WHQOe-e8JPzM525XE-n245swAfSJuPOJCx-6i408UeUEtGoRWDa9VWPTU~fjdiWnY6QyAQ9uXx8LmOgmMfJqlDHxB6NOcaEEF-gHmfpvlf3BiHYdXXOAs7B57cBhxF1Pv5Z9s-fuTyNWBo9T9ONAqW5l9v6gyafY4DcT-Faxmn--a9CVSzJCGhw3rCfBqYvqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of TREP repeat categories on the Haruna Nijo and Morex barley genome assemblies. Frequency of each repeat content was calculated by the result of repeat masking using the TREP library. Total bases of each repeat region on the genomes were divided by those of all repeat regions." path-from-xml="dsv03301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196975/dsv03301.tiff" path-from-xml="dsv03301.tiff" target="_blank">View large</a><a section="87196975" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03301.png?Expires=2147483647&Signature=Wr8n3Sd-reTe~ziF98MjuJif4CPovjqYkeHSQ4C0xc0jvgnE-CPjnHW4pRspXPFHhWiIZgBm7wh2u4yJVRCuVNLCFdydFtrY6nBrTfw9QNvCjrgGd1U2GrJAfLj0IH0NifLlq9PfzkwC0i8~nqv6GdMHfXl1-K1deMce6KcMu9ZvNhRhggWVcaqppWS5oHK-90bNhqBpfky-aFfL~UqurmWSYcgvGWj~HGCX2doDhpJmCrps6YzAeoVPX32JTACO3p1T6J0YZ4M~4jpdKEdPMnSjA775ZGoM-7EhqYA4IydMt8CPMoWvnA3zsI~cDu9GGx6UpmYtDI0QM7vsjPA-Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196975&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of TREP repeat categories on the Haruna Nijo and Morex barley genome assemblies. Frequency of each repeat content was calculated by the result of repeat masking using the TREP library. Total bases of each repeat region on the genomes were divided by those of all repeat regions.</p></div></div><div content-id="DSV033F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03301.png?Expires=2147483647&Signature=NsOsjzDD0jTux5XpRkbmn1dThCG9d9IZnIT0npsjHtAwb6VrqmYI4ETwY33ZeFzd6OYJxqOV6-xA7FX-775JWntU07UJy5d9Gm0FOZX0kTG~25IjGpq4xKgwpH8vSAFXUpWSl5M~MYyD8YJUJiJq8WHQOe-e8JPzM525XE-n245swAfSJuPOJCx-6i408UeUEtGoRWDa9VWPTU~fjdiWnY6QyAQ9uXx8LmOgmMfJqlDHxB6NOcaEEF-gHmfpvlf3BiHYdXXOAs7B57cBhxF1Pv5Z9s-fuTyNWBo9T9ONAqW5l9v6gyafY4DcT-Faxmn--a9CVSzJCGhw3rCfBqYvqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of TREP repeat categories on the Haruna Nijo and Morex barley genome assemblies. Frequency of each repeat content was calculated by the result of repeat masking using the TREP library. Total bases of each repeat region on the genomes were divided by those of all repeat regions." path-from-xml="dsv03301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196975/dsv03301.tiff" path-from-xml="dsv03301.tiff" target="_blank">View large</a><a section="87196975" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03301.png?Expires=2147483647&Signature=Wr8n3Sd-reTe~ziF98MjuJif4CPovjqYkeHSQ4C0xc0jvgnE-CPjnHW4pRspXPFHhWiIZgBm7wh2u4yJVRCuVNLCFdydFtrY6nBrTfw9QNvCjrgGd1U2GrJAfLj0IH0NifLlq9PfzkwC0i8~nqv6GdMHfXl1-K1deMce6KcMu9ZvNhRhggWVcaqppWS5oHK-90bNhqBpfky-aFfL~UqurmWSYcgvGWj~HGCX2doDhpJmCrps6YzAeoVPX32JTACO3p1T6J0YZ4M~4jpdKEdPMnSjA775ZGoM-7EhqYA4IydMt8CPMoWvnA3zsI~cDu9GGx6UpmYtDI0QM7vsjPA-Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196975&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of TREP repeat categories on the Haruna Nijo and Morex barley genome assemblies. Frequency of each repeat content was calculated by the result of repeat masking using the TREP library. Total bases of each repeat region on the genomes were divided by those of all repeat regions.</p></div></div><p>To identify unique repeat sequences in barley, <em>de novo</em> repeat sequences in the Haruna Nijo scaffolds were detected by RepeatModeler, which identified a total of 979 consensus sequences. The <em>de novo</em> repeat library and the TREP library were used for repeat masking. As a result, 74.8% of the assembled sequences were masked (‘N’s were not masked and thus deleted from the calculation). The repeat masking by the <em>de novo</em> library identified 68.1% repeat sequences, which were much more than the masked repeats (60.8%) detected by the TREP library.</p>                            <h3 scrollto-destination=87196977 id="87196977" class="section-title" data-legacy-id=s3c>3.3. FLcDNA mapping</h3>
<p>FLcDNA mapping on genome sequence provides reliable exon vs. intron structures on gene models. A non-redundant set of 28,620 FLcDNA sequences<sup><a reveal-id="DSV033C13" data-open="DSV033C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV033C14" data-open="DSV033C14" class="link link-ref link-reveal xref-bibr">14</a></sup> was mapped to the Haruna Nijo scaffolds. Blastn mapping revealed that 27,784 FLcDNAs (97%) showed sequence similarities with the Haruna Nijo scaffolds. According to the results of the FLcDNA mapping to scaffolds, we analysed mapping coverage of 5′-end of each FLcDNA on Haruna Nijo scaffolds. The ratio of completely mapped FLcDNAs, which were mapped from the first nucleotide sequence, was &lt;10%. This shows that most of the FLcDNAs do not have the complete sequences on the 5′-end side exon. To analyse additional mapped FLcDNAs, we relaxed the mapping position to the first 10 nucleotide sequences. Then, the ratio of mapped FLcDNA was raised to 80% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). GC contents and CG-skew were analysed on the FLcDNAs having mapped 5′-ends within 10 bp. The typical high GC contents and CG-skewed peaks at the transcription start sites were identified as previously observed in rice and Arabidopsis<sup><a reveal-id="DSV033C32" data-open="DSV033C32" class="link link-ref link-reveal xref-bibr">32</a></sup> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p><p>The longer Haruna Nijo scaffolds may contribute to the efficiency in FLcDNA mapping. The number of mapped FLcDNAs under the threshold of 50% mapping coverage was 26,240 in the Haruna Nijo scaffolds and 24,261 in the Morex contigs (Table <a reveal-id="DSV033TB3" data-open="DSV033TB3" class="link link-reveal link-table xref-fig">3</a>). These numbers included FLcDNAs that mapped to either the Haruna Nijo scaffolds (1,985) or the Morex contigs (379). However, if the threshold of mapping coverage was raised to 95%, the number of mapped FLcDNAs decreased to 14,044 on the Morex contigs, while 19,485 were still mapped on the Haruna Nijo scaffolds. </p>                        <a id="87196980" scrollto-destination="87196980"></a>
<div content-id="DSV033TB3" class="table-modal"><div class="table-wrap"><div id="DSV033TB3" class="table-wrap-title" data-id="DSV033TB3"><div class="label -label">Table 3.</div><div class="caption"><p>FLcDNA mapping on genome assemblies</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Coverage (%) </th><th colspan="2">No. of hit FLcDNAs (ratio/total)<hr /> </th></tr><tr><th>Haruna Nijo </th><th>Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </th></tr></thead><tbody><tr><td>50 </td><td>26,241 (0.917) </td><td>24,621 (0.860) </td></tr><tr><td>70 </td><td>22,835 (0.798) </td><td>19,926 (0.696) </td></tr><tr><td>80 </td><td>21,429 (0.749) </td><td>17,706 (0.619) </td></tr><tr><td>90 </td><td>20,153 (0.704) </td><td>15,361 (0.537) </td></tr><tr><td>95 </td><td>19,485 (0.681) </td><td>14,044 (0.491) </td></tr></tbody></table></div><div content-id="DSV033TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Coverage (%) </th><th colspan="2">No. of hit FLcDNAs (ratio/total)<hr /> </th></tr><tr><th>Haruna Nijo </th><th>Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </th></tr></thead><tbody><tr><td>50 </td><td>26,241 (0.917) </td><td>24,621 (0.860) </td></tr><tr><td>70 </td><td>22,835 (0.798) </td><td>19,926 (0.696) </td></tr><tr><td>80 </td><td>21,429 (0.749) </td><td>17,706 (0.619) </td></tr><tr><td>90 </td><td>20,153 (0.704) </td><td>15,361 (0.537) </td></tr><tr><td>95 </td><td>19,485 (0.681) </td><td>14,044 (0.491) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196980" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV033TB3" class="table-wrap-title" data-id="DSV033TB3"><div class="label -label">Table 3.</div><div class="caption"><p>FLcDNA mapping on genome assemblies</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Coverage (%) </th><th colspan="2">No. of hit FLcDNAs (ratio/total)<hr /> </th></tr><tr><th>Haruna Nijo </th><th>Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </th></tr></thead><tbody><tr><td>50 </td><td>26,241 (0.917) </td><td>24,621 (0.860) </td></tr><tr><td>70 </td><td>22,835 (0.798) </td><td>19,926 (0.696) </td></tr><tr><td>80 </td><td>21,429 (0.749) </td><td>17,706 (0.619) </td></tr><tr><td>90 </td><td>20,153 (0.704) </td><td>15,361 (0.537) </td></tr><tr><td>95 </td><td>19,485 (0.681) </td><td>14,044 (0.491) </td></tr></tbody></table></div><div content-id="DSV033TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Coverage (%) </th><th colspan="2">No. of hit FLcDNAs (ratio/total)<hr /> </th></tr><tr><th>Haruna Nijo </th><th>Morex<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> </th></tr></thead><tbody><tr><td>50 </td><td>26,241 (0.917) </td><td>24,621 (0.860) </td></tr><tr><td>70 </td><td>22,835 (0.798) </td><td>19,926 (0.696) </td></tr><tr><td>80 </td><td>21,429 (0.749) </td><td>17,706 (0.619) </td></tr><tr><td>90 </td><td>20,153 (0.704) </td><td>15,361 (0.537) </td></tr><tr><td>95 </td><td>19,485 (0.681) </td><td>14,044 (0.491) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196980" target="_blank">View Large</a></div></div><p>We also analysed the coverage of FLcDNA mapped reads on the Haruna Nijo scaffolds and the Morex contigs by comparing the mapped exon numbers of FLcDNAs between Haruna Nijo and Morex. Of the 24,255 FLcDNA gene models, 19,207 (79.2%) were mapped with the same number of exons both on the Haruna Nijo scaffolds and on the Morex contigs. Of these, 3,066 gene models had more exons in Haruna Nijo than that in Morex, while 1,982 gene models had more exons in Morex than Haruna Nijo. The start position analysis also showed that the Haruna Nijo scaffolds (6,842) had more mapped 5′-end sequences than the Morex contigs (1,721).</p><p>These results indicated that the available genome sequences of Morex and Haruna Nijo are different, and there may be more cases where FLcDNAs mapped with higher coverage on Haruna Nijo genomes. For example, the entire FLcDNA (AK371953) for the <em>cleistogamy1</em> (<em>cly1</em> for closed flowering) gene,<sup><a reveal-id="DSV033C33" data-open="DSV033C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which was mapped on a Haruna Nijo scaffold, while the 5′ region was not mapped on a Morex contig (Fig. <a reveal-id="DSV033F2" data-open="DSV033F2" class="link link-reveal link-table xref-fig">2</a>). Moreover, flanking regions for the FLcDNA (3.6 Kb for upstream and 2.9 Kb for downstream regions) were present on the Haruna Nijo scaffold to provide the opportunity to analyse the <em>cis</em>-element(s) of the gene. </p>                        <a id="87196983" scrollto-destination="87196983"></a>
<div id="DSV033F2" data-id="DSV033F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03302.png?Expires=2147483647&Signature=KAJamzVptHH~61-PfD-1SZhIsPtvOG~Vy9f-Bg2OgMLGCDhZEm8PNZ9mroPqdL7psCzVMYQgPVK12hDmZED69AZnyVrE6gFhDt6cfGi56xVG4BH2Qx6kVwRIOCOIAA5DDD6lYKb2AJOFIIiXHJ7C7vSHrWU9lbrhHdeYhIVF~Z8bIkvZSav2dtD9CONi1ssi3F~AgLj1hPH-zqeq289O6EGsELxuojs6TJcK2plNuCyB2KogQnVAPk3K7CpnvWrXWz18MjlidiZmkml9tDVMku5mtHh3xOIvT9RXBLoGYDy-jhn9tjL-LXtiuHoy6W4~7KwsB1GDTNX6t~faUxxZ7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene structure of closed flowering locus cly1 (AK371953). In the Haruna Nijo genome, the complete gene structures of cly1 including upstream and downstream regions were identified. In the Morex genome,3 the first exon and upstream regions were truncated because of insufficient length of the contig." path-from-xml="dsv03302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196983/dsv03302.tiff" path-from-xml="dsv03302.tiff" target="_blank">View large</a><a section="87196983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03302.png?Expires=2147483647&Signature=2cjrzqZOk3KauHC6TaFPjnVbFCSpBSdkLCgDH1gS9IW60f8Bh~ELHq4ZKizQJMWpSKflmz~VQg7scUK6qccskgFfXhidkdTAXjD7nYeUSxbXGsdP4qk7laATYfAeDNm0ZoQuSPxOszeDp3MuoYaSR8qPmIajKiDFqHO0RihbhHIp72OHAcLxyeJlzh6p-e6ircWnI8~W8A7egFHwh6EDejeLSrr~T5JOfPxLJjuHwOar7danU37EhVDumvtag~eHfqqm1H87WzbGdPZg~pN8FfGBT4uvhDu1bh7zR43rYzups6ND0CXjpOg6iueUZlVI0IMmp50B0tZewWS0FCrj7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196983&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene structure of closed flowering locus <em>cly1</em> (AK371953). In the Haruna Nijo genome, the complete gene structures of <em>cly1</em> including upstream and downstream regions were identified. In the Morex genome,<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> the first exon and upstream regions were truncated because of insufficient length of the contig.</p></div></div><div content-id="DSV033F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03302.png?Expires=2147483647&Signature=KAJamzVptHH~61-PfD-1SZhIsPtvOG~Vy9f-Bg2OgMLGCDhZEm8PNZ9mroPqdL7psCzVMYQgPVK12hDmZED69AZnyVrE6gFhDt6cfGi56xVG4BH2Qx6kVwRIOCOIAA5DDD6lYKb2AJOFIIiXHJ7C7vSHrWU9lbrhHdeYhIVF~Z8bIkvZSav2dtD9CONi1ssi3F~AgLj1hPH-zqeq289O6EGsELxuojs6TJcK2plNuCyB2KogQnVAPk3K7CpnvWrXWz18MjlidiZmkml9tDVMku5mtHh3xOIvT9RXBLoGYDy-jhn9tjL-LXtiuHoy6W4~7KwsB1GDTNX6t~faUxxZ7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene structure of closed flowering locus cly1 (AK371953). In the Haruna Nijo genome, the complete gene structures of cly1 including upstream and downstream regions were identified. In the Morex genome,3 the first exon and upstream regions were truncated because of insufficient length of the contig." path-from-xml="dsv03302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196983/dsv03302.tiff" path-from-xml="dsv03302.tiff" target="_blank">View large</a><a section="87196983" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03302.png?Expires=2147483647&Signature=2cjrzqZOk3KauHC6TaFPjnVbFCSpBSdkLCgDH1gS9IW60f8Bh~ELHq4ZKizQJMWpSKflmz~VQg7scUK6qccskgFfXhidkdTAXjD7nYeUSxbXGsdP4qk7laATYfAeDNm0ZoQuSPxOszeDp3MuoYaSR8qPmIajKiDFqHO0RihbhHIp72OHAcLxyeJlzh6p-e6ircWnI8~W8A7egFHwh6EDejeLSrr~T5JOfPxLJjuHwOar7danU37EhVDumvtag~eHfqqm1H87WzbGdPZg~pN8FfGBT4uvhDu1bh7zR43rYzups6ND0CXjpOg6iueUZlVI0IMmp50B0tZewWS0FCrj7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196983&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene structure of closed flowering locus <em>cly1</em> (AK371953). In the Haruna Nijo genome, the complete gene structures of <em>cly1</em> including upstream and downstream regions were identified. In the Morex genome,<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> the first exon and upstream regions were truncated because of insufficient length of the contig.</p></div></div>                            <h3 scrollto-destination=87196984 id="87196984" class="section-title" data-legacy-id=s3d>3.4. Gene annotation</h3>
<p>The IBSC presented 26,159 HC (high confidence) and 53,220 LC (low confidence) gene models by using RNA-Seq-based transcript and FLcDNAs.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> This number of genes was larger than that of the Haruna Nijo FLcDNAs. Actually, of the 24,243 HC Morex gene models (in GFF file), only 7,645 had corresponding Haruna Nijo FLcDNAs, indicating that the FLcDNAs sequence data were not deep enough to construct a comprehensive set of barley gene models. To obtain more comprehensive transcript sequences of Haruna Nijo, RNA-Seq data from four libraries of seedling shoot, seedling root, immature spike and immature seed were collected (Table <a reveal-id="DSV033TB4" data-open="DSV033TB4" class="link link-reveal link-table xref-fig">4</a>). As reference gene structures, 26,240 gene models on 18,226 loci determined by FLcDNA mapping with 50% coverage (Table <a reveal-id="DSV033TB3" data-open="DSV033TB3" class="link link-reveal link-table xref-fig">3</a>) were used for mapping RNA-Seq reads. Sequences of each RNA-Seq library mapped 73.2–81.0% to each other, indicating that the concordant pair alignment rates were not much different among the libraries (Table <a reveal-id="DSV033TB4" data-open="DSV033TB4" class="link link-reveal link-table xref-fig">4</a>). Sequences from all four libraries were combined by cuffmerge<sup><a reveal-id="DSV033C26" data-open="DSV033C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and 30,606 loci (51,249 gene models) were estimated. Of these, 12,390 loci and 5,635 alternative variants were added on the Haruna Nijo gene models. </p>                        <a id="87196986" scrollto-destination="87196986"></a>
<div content-id="DSV033TB4" class="table-modal"><div class="table-wrap"><div id="DSV033TB4" class="table-wrap-title" data-id="DSV033TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Sequencing and mapping results of Haruna Nijo RNA-Seq data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>No. of read pairs </th><th>Aligned pairs </th><th>Concordant pair alignment rate (%) </th></tr></thead><tbody><tr><td>Leaf </td><td>4,977,702 </td><td>3,738,352 </td><td>75.0 </td></tr><tr><td>Root </td><td>3,981,366 </td><td>2,921,944 </td><td>73.2 </td></tr><tr><td>Seed </td><td>7,024,371 </td><td>5,691,954 </td><td>81.0 </td></tr><tr><td>Spike </td><td>4,292,946 </td><td>3,399,764 </td><td>79.1 </td></tr></tbody></table></div><div content-id="DSV033TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>No. of read pairs </th><th>Aligned pairs </th><th>Concordant pair alignment rate (%) </th></tr></thead><tbody><tr><td>Leaf </td><td>4,977,702 </td><td>3,738,352 </td><td>75.0 </td></tr><tr><td>Root </td><td>3,981,366 </td><td>2,921,944 </td><td>73.2 </td></tr><tr><td>Seed </td><td>7,024,371 </td><td>5,691,954 </td><td>81.0 </td></tr><tr><td>Spike </td><td>4,292,946 </td><td>3,399,764 </td><td>79.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196986" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV033TB4" class="table-wrap-title" data-id="DSV033TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Sequencing and mapping results of Haruna Nijo RNA-Seq data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>No. of read pairs </th><th>Aligned pairs </th><th>Concordant pair alignment rate (%) </th></tr></thead><tbody><tr><td>Leaf </td><td>4,977,702 </td><td>3,738,352 </td><td>75.0 </td></tr><tr><td>Root </td><td>3,981,366 </td><td>2,921,944 </td><td>73.2 </td></tr><tr><td>Seed </td><td>7,024,371 </td><td>5,691,954 </td><td>81.0 </td></tr><tr><td>Spike </td><td>4,292,946 </td><td>3,399,764 </td><td>79.1 </td></tr></tbody></table></div><div content-id="DSV033TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>No. of read pairs </th><th>Aligned pairs </th><th>Concordant pair alignment rate (%) </th></tr></thead><tbody><tr><td>Leaf </td><td>4,977,702 </td><td>3,738,352 </td><td>75.0 </td></tr><tr><td>Root </td><td>3,981,366 </td><td>2,921,944 </td><td>73.2 </td></tr><tr><td>Seed </td><td>7,024,371 </td><td>5,691,954 </td><td>81.0 </td></tr><tr><td>Spike </td><td>4,292,946 </td><td>3,399,764 </td><td>79.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196986" target="_blank">View Large</a></div></div><p>We compared the Haruna Nijo gene models with the Morex HC and LC gene models.<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> We mapped 78.0% of Morex HC gene models and 72.2% of Morex LC gene models on Haruna Nijo gene models with 95% coverage and 90% identity. Of these, 1,446 Morex gene models were mapped on multiple Haruna Nijo gene models. Clustering analysis showed that 21,381 Haruna Nijo gene models overlapped with Morex gene models on the Haruna Nijo scaffolds, while 9,225 gene models did not overlap each other.</p><p>Blastx search on RefSeq/UniProt showed that 48,619 gene models of Haruna Nijo were similar to known protein sequences. In addition, 48,105 predicted ORFs on 28,422 gene models of 24,467 scaffolds were &gt;70 aa in length. Of these, 44,133 ORFs had Met at the start positions. Noteworthy, 23,882 ORFs derived from 25,897 mapped FLcDNA contained Met at the start position.</p><p>InterProScan analysis showed that 26,956 ORFs on 19,438 loci contained 67,293 InterPro domains. While 2,760 InterPro domains were in the ‘REPEAT’ category, which may not have biological function, other 44,769 domains were ‘DOMAIN’ category (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Moreover, GOslim analysis revealed that the gene content of Haruna Nijo (barley) and Nipponbare (rice) was quite similar (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p><p>We also predicted rRNA genes on Haruna Nijo scaffolds. Based on the silva database, nine scaffolds were identified as rRNA regions. In the case of rice, rDNA genes compose nucleolar-organizing region (NOR) in which rDNA are aligned in tandem. In wheat, the chromosome-based WGS analysis showed that the chromosome 6B contains 60% of the total rRNA gene (5,500 genes).<sup><a reveal-id="DSV033C34" data-open="DSV033C34" class="link link-ref link-reveal xref-bibr">34</a></sup> However, these regions could not be assembled by NGS data<sup><a reveal-id="DSV033C35" data-open="DSV033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> due to the degeneration in the process of assembly. This was also true for the Haruna Nijo scaffolds which showed only nine scaffolds carrying rRNA sequences. The difficulty of assembling NOR was also reported in rice,<sup><a reveal-id="DSV033C36" data-open="DSV033C36" class="link link-ref link-reveal xref-bibr">36</a></sup> but the NOR assembly in WGS might be more difficult in general.</p><p>tRNAs were also annotated by tRNAscan-SE. The spectrum of tRNA distribution across the genome was uneven but similar to the rice genome<sup><a reveal-id="DSV033C37" data-open="DSV033C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). It is reported that tRNA<sup>Lys</sup> sequences are extraordinarily abundant in wheat chromosome 6B due to the abundance of the retrotransposon <em>Gypsy</em>.<sup><a reveal-id="DSV033C35" data-open="DSV033C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The Haruna Nijo scaffolds also have higher rate of <em>Gypsy</em>; however, the number of tRNAs was smaller than that in wheat.</p>                            <h3 scrollto-destination=87196992 id="87196992" class="section-title" data-legacy-id=s3e>3.5. WGS assembly comparison between Haruna Nijo and Morex</h3>
<p>There are several published<sup><a reveal-id="DSV033C3" data-open="DSV033C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and ongoing efforts of sequencing BACs derived from Morex. BAC-based assemblies may provide longer sequences than WGS assemblies to estimate gene models on the genome. The final goal for developing Haruna Nijo gene models is to map them on the BAC-based Morex genome sequences. To understand the quality of current Haruna Nijo genome resources, sequences of the Haruna Nijo scaffolds and the Morex contigs were aligned by megablast. An average sequence identity between Haruna Nijo and Morex was estimated as 95.99% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). We found that mapped regions of query were different when query and database sequences were exchanged between Haruna Nijo and Morex. This was caused by the difference of sequence length in two assemblies. In this analysis, we used the best hit of megablast for the calculation, and the other hits with lower coverage on Haruna Nijo scaffolds were discarded. Therefore, the total length of aligned regions was decreased. Genome comparison of Haruna Nijo and Morex identified the occasions that the genomic regions of Haruna Nijo and Morex do not match. An example of mismatch between two assemblies on chromosome 5H was that a region of the Haruna Nijo gene model of TCONS_00033948 or TCONS_00033949 (Fig. <a reveal-id="DSV033F3" data-open="DSV033F3" class="link link-reveal link-table xref-fig">3</a>A), which is supported by a FLcDNA, AK370496 that encodes bZIP transcription factor family protein on a contig_55003 (scaffold), was divided into two loci of MLOC_14578.1 on morex_contig_156845 (Fig. <a reveal-id="DSV033F3" data-open="DSV033F3" class="link link-reveal link-table xref-fig">3</a>B) and MLOC_75543 on morex_contig_68185 (Fig. <a reveal-id="DSV033F3" data-open="DSV033F3" class="link link-reveal link-table xref-fig">3</a>C). Thus, we may find a better gene model if we compare multiple gene models derived from different assemblies. </p>                        <a id="87196994" scrollto-destination="87196994"></a>
<div id="DSV033F3" data-id="DSV033F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03303.png?Expires=2147483647&Signature=GuzSng6Vr9wz0g7UiTWQeifv547PsFuoImb9VeHrFrEmC7~arjb4Ruq0Fqw33N8Ontt61yg-U9YiAdH-1VSspnkKGl~qinxdzcaiRRm6amjtZOofQhli8o1I2EEJC3knEsu2RvXU3mH7m0plYb2i2Q9ejmeWWZ88Dy8Q8QIbhH3dOFUvF6wHYkQaLR8icyO1A1DolWKURjjHkVeh4HIEcpWPOVqfiupoxsWE69XoltOotZjbPPEhZFc3dPwBJUllW7LXvMHIRsW2BZS4j-4e1t3ONGdm~HEk9WXfpIMoFGTRrx5tZp7zX1~tcveph8o-XXw5-clN2QGPVjeeM~~OLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example screenshots of (A) Haruna Nijo contig_55003, (B) Morex contig_156845 and (C) Morex contig_68185 under the platform of Gbrowse. Separated genomic regions of Morex which correspond to the single FLcDNA region in Haruna Nijo. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196994/dsv03303.tiff" path-from-xml="dsv03303.tiff" target="_blank">View large</a><a section="87196994" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03303.png?Expires=2147483647&Signature=B473w9nYQFBsD60pJHHRHgMnkx1YotL4MSU97ouKEIHf9U7~Sz6bphmF1CPAncrPS~~JJk1n3uzNTZeQe5VAgj7Ml4moLruQYU94MUAqACoDHojU0iJ5cSBSSAiXqeFLtgmuYrPGbWJDnaaJbqK12nUIJwXBbqRdN-gtAjbTh5aLxPOkgcQ9Tj8~dOmyPRVWnC8dmqj8QbyP6Z~FSxNfB61MqOlv4eR-iQi2TqRITGRSP1M0WF-~VjeRfmCNWznyvuNcP2-rSqVj8tq88Es3-OLfLUFsFw7bbjgihhK3G~zpIWtlCIXjobFRoTSy-arbyQi6BCSsnygMsg5dI8uqmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196994&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example screenshots of (A) Haruna Nijo contig_55003, (B) Morex contig_156845 and (C) Morex contig_68185 under the platform of Gbrowse. Separated genomic regions of Morex which correspond to the single FLcDNA region in Haruna Nijo. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV033F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03303.png?Expires=2147483647&Signature=GuzSng6Vr9wz0g7UiTWQeifv547PsFuoImb9VeHrFrEmC7~arjb4Ruq0Fqw33N8Ontt61yg-U9YiAdH-1VSspnkKGl~qinxdzcaiRRm6amjtZOofQhli8o1I2EEJC3knEsu2RvXU3mH7m0plYb2i2Q9ejmeWWZ88Dy8Q8QIbhH3dOFUvF6wHYkQaLR8icyO1A1DolWKURjjHkVeh4HIEcpWPOVqfiupoxsWE69XoltOotZjbPPEhZFc3dPwBJUllW7LXvMHIRsW2BZS4j-4e1t3ONGdm~HEk9WXfpIMoFGTRrx5tZp7zX1~tcveph8o-XXw5-clN2QGPVjeeM~~OLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example screenshots of (A) Haruna Nijo contig_55003, (B) Morex contig_156845 and (C) Morex contig_68185 under the platform of Gbrowse. Separated genomic regions of Morex which correspond to the single FLcDNA region in Haruna Nijo. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196994/dsv03303.tiff" path-from-xml="dsv03303.tiff" target="_blank">View large</a><a section="87196994" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03303.png?Expires=2147483647&Signature=B473w9nYQFBsD60pJHHRHgMnkx1YotL4MSU97ouKEIHf9U7~Sz6bphmF1CPAncrPS~~JJk1n3uzNTZeQe5VAgj7Ml4moLruQYU94MUAqACoDHojU0iJ5cSBSSAiXqeFLtgmuYrPGbWJDnaaJbqK12nUIJwXBbqRdN-gtAjbTh5aLxPOkgcQ9Tj8~dOmyPRVWnC8dmqj8QbyP6Z~FSxNfB61MqOlv4eR-iQi2TqRITGRSP1M0WF-~VjeRfmCNWznyvuNcP2-rSqVj8tq88Es3-OLfLUFsFw7bbjgihhK3G~zpIWtlCIXjobFRoTSy-arbyQi6BCSsnygMsg5dI8uqmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196994&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example screenshots of (A) Haruna Nijo contig_55003, (B) Morex contig_156845 and (C) Morex contig_68185 under the platform of Gbrowse. Separated genomic regions of Morex which correspond to the single FLcDNA region in Haruna Nijo. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>We also analysed conserved genic regions between Haruna Nijo and Morex by using mapped regions of FLcDNAs. The difference between Haruna Nijo scaffolds and FLcDNA sequences was 0.05% (1 bp of 2000bp in exon), which is still five times higher than the value of 0.01% in rice genome.<sup><a reveal-id="DSV033C37" data-open="DSV033C37" class="link link-ref link-reveal xref-bibr">37</a></sup> However, the sequence difference of exon regions between Haruna Nijo and Morex (identified by Haruna Nijo FLcDNAs) was 0.29%, indicating that the sequence quality of scaffolds in Haruna Nijo was high compared with the haplotype difference between Haruna Nijo and Morex.</p>                            <h3 scrollto-destination=87196996 id="87196996" class="section-title" data-legacy-id=s3f>3.6. Conclusion</h3>
<p>Based on this study, there are several advantages of using the Haruna Nijo sequence resources. The Haruna Nijo scaffolds provide a high-quality genome sequence for the barley research community. In particular, the FLcDNA sequences, which are available only for Haruna Nijo, provide a key resource. In addition, the gene structures of Haruna Nijo provide <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary data</a></span> sets to the Morex genome sequence. Finally, the haplotype of Haruna Nijo is popular among the malting barley cultivars in the world and can be used as an alternative source of alleles for important traits in barley.</p>                            <h2 scrollto-destination=87196998 id="87196998" class="section-title" data-legacy-id=s4>4. Availability</h2>
<p>The short genomic reads used in the study are deposited at DDBJ-SRA under accession ID PRJDB4103. The scaffolds for the <em>Hordeum vulgare</em> cv. Haruna Nijo are available at <a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/bexdb/pages/harunanijo_index.jsp">http://barleyflc.dna.affrc.go.jp/bexdb/pages/harunanijo_index.jsp</a> for download. Genome browsing of the Haruna Nijo gene models are available on GBrowse [(<a class="link link-uri" href="http://gmod.org/wiki/GBrowse">http://gmod.org/wiki/GBrowse</a>)<sup><a reveal-id="DSV033C38" data-open="DSV033C38" class="link link-ref link-reveal xref-bibr">38</a></sup> at <a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/gb2/gbrowse/HarunaNijo_genome/.21">http://barleyflc.dna.affrc.go.jp/gb2/gbrowse/HarunaNijo_genome/.21</a>]. All scaffolds were concatenated by 100 ‘Ns’ from the longer scaffolds. Haruna Nijo scaffolds, FLcDNAs, RNA-Seq genes, repeat information, Morex contigs and Morex HC/LC genes are displayed (Fig. <a reveal-id="DSV033F3" data-open="DSV033F3" class="link link-reveal link-table xref-fig">3</a>). From the Haruna Nijo genome browser, FLcDNA annotation data and Morex genome browser of bex-db could be accessed (Fig. <a reveal-id="DSV033F4" data-open="DSV033F4" class="link link-reveal link-table xref-fig">4</a>). Moreover, users can access the Haruna Nijo genome from FLcDNA information of bex-db. </p>                        <a id="87197000" scrollto-destination="87197000"></a>
<div id="DSV033F4" data-id="DSV033F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03304.png?Expires=2147483647&Signature=MUltpAgpM5bDjIyhB9vM9u~8alDAZe5GIIiWpxgZy5ygXJ7nDt3VvrKeoTDd1KyxdL6URcmPJFtWOYnU6eGM5edcWk8kdY6bvRqLfrDx1h3Tex7MQ7g5Jo1WvWoPtg8EsRkl4zR6YJMQE9mr6F3kZyiergGeVRtp2Y1Qerd3pQEKvOs6vjHJZzSdV0HuS7HEM30TquUHBcKqszhKPt3ZuDBeggw1PtK99n~z5wPCcWXetZGVszD-5PDR3KpVCOy91fcAE90kFLJ9Y62c~poGudnTv3YaxntIA-4zwwpDqkxFrqLfZOuYCrUBqXQbXU4KoU6zxhn301y~oMt3vji3QQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example screenshots of bex-db (http://barleyflc.dna.affrc.go.jp/bexdb/) for a FLcDNA (AK370496). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197000/dsv03304.tiff" path-from-xml="dsv03304.tiff" target="_blank">View large</a><a section="87197000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03304.png?Expires=2147483647&Signature=0Ro254NT2jDI~3Jec4yR~q8VFFSBjPLSu-KWqOz2tBcQYbYqDufkFYD4BrIFl0Mvprx5nw-2gfVd-KZ1WYu7YCZc-pv9PMynwjKNqwUuUpFKiVDnNHZ-pKcFIaNhPEZGVEOnbhX2OqqGIXEFHA7UuqbeFIXdyvAGBu45M6G7DT2IWkFQXUxZzZN2iIi6EmlbyGo6BUQwIZknG3WrmLLD2PuObsfcaC2VgKFUbLba8bFPdAs35k1tn-dlQt~-HsWWphIreVI7oJItAHYBYMrNNfXVEeynAEc1Pxk-yFoush58TzLQc1pNvtBzAiyyGMrVvIKuwU-9rFnI7zw5mCbiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197000&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example screenshots of bex-db (<a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/bexdb/">http://barleyflc.dna.affrc.go.jp/bexdb/</a>) for a FLcDNA (AK370496). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV033F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/m_dsv03304.png?Expires=2147483647&Signature=MUltpAgpM5bDjIyhB9vM9u~8alDAZe5GIIiWpxgZy5ygXJ7nDt3VvrKeoTDd1KyxdL6URcmPJFtWOYnU6eGM5edcWk8kdY6bvRqLfrDx1h3Tex7MQ7g5Jo1WvWoPtg8EsRkl4zR6YJMQE9mr6F3kZyiergGeVRtp2Y1Qerd3pQEKvOs6vjHJZzSdV0HuS7HEM30TquUHBcKqszhKPt3ZuDBeggw1PtK99n~z5wPCcWXetZGVszD-5PDR3KpVCOy91fcAE90kFLJ9Y62c~poGudnTv3YaxntIA-4zwwpDqkxFrqLfZOuYCrUBqXQbXU4KoU6zxhn301y~oMt3vji3QQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example screenshots of bex-db (http://barleyflc.dna.affrc.go.jp/bexdb/) for a FLcDNA (AK370496). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197000/dsv03304.tiff" path-from-xml="dsv03304.tiff" target="_blank">View large</a><a section="87197000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv03304.png?Expires=2147483647&Signature=0Ro254NT2jDI~3Jec4yR~q8VFFSBjPLSu-KWqOz2tBcQYbYqDufkFYD4BrIFl0Mvprx5nw-2gfVd-KZ1WYu7YCZc-pv9PMynwjKNqwUuUpFKiVDnNHZ-pKcFIaNhPEZGVEOnbhX2OqqGIXEFHA7UuqbeFIXdyvAGBu45M6G7DT2IWkFQXUxZzZN2iIi6EmlbyGo6BUQwIZknG3WrmLLD2PuObsfcaC2VgKFUbLba8bFPdAs35k1tn-dlQt~-HsWWphIreVI7oJItAHYBYMrNNfXVEeynAEc1Pxk-yFoush58TzLQc1pNvtBzAiyyGMrVvIKuwU-9rFnI7zw5mCbiKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197000&ar=2462418&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example screenshots of bex-db (<a class="link link-uri" href="http://barleyflc.dna.affrc.go.jp/bexdb/">http://barleyflc.dna.affrc.go.jp/bexdb/</a>) for a FLcDNA (AK370496). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87197001 id="87197001" class="section-title" data-legacy-id=s6>Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by Okayama University to Kazuhiro Sato, corresponding author.</p>                            <h2 scrollto-destination=87197003 id="87197003" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to express our thanks to Prof. Gary Muehlbauer, University of Minnesota for his critical reading and linguistic editing of the manuscript. Barley seeds were provided through the National Bioresource Project of Barley, MEXT of Japan. Analysis of Illumina HiSeq 2000 sequence reads was performed by the Japan Advanced Plant Science Research Network (<a class="link link-uri" href="http://www.psr-net.riken.jp/index.html">http://www.psr-net.riken.jp/index.html</a>). This work was partly supported by the scientific technique research promotion program for agriculture, forestry fisheries and food industry to K.S.</p>                            <h2 scrollto-destination=87197005 id="87197005" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV033C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pourkheirandish</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Hensel</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kilian</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Evolution of the grain dispersal system in barley</div>, <div class="source">Cell</div> , <div class="volume">162</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20grain%20dispersal%20system%20in%20barley&amp;author=M.%20Pourkheirandish&amp;author=G.%20Hensel&amp;author=B.%20Kilian&amp;publication_year=2015&amp;journal=Cell&amp;volume=162&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.07.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.07.002"> </span></div> </div></div></div></div></div><div content-id="DSV033C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Flavell</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Börner</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Valkoun</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Kumlehn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">N.</div></div></div> (eds), <div class="source">Biotechnological approaches to barley improvement, biotechnology in agriculture and forestry</div> . <div class="publisher-name">Springer</div>: <div class="publisher-loc">Heidelberg</div>, pp.<div class="fpage">21</div>–<div class="lpage">36</div>.</div></div></div></div><div content-id="DSV033C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="collab">The International Barley Genome Sequencing Consortium</div>. <div class="year">2012</div>, <div class="article-title">A physical, genetic and functional sequence assembly of the barley genome</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">711</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ariyadasa</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Mascher</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nussbaumer</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">A sequence-ready physical map of barley anchored genetically by two million SNPs</div>, <div class="source">Plant Physiol.</div> , <div class="volume">164</div>, <div class="fpage">412</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sequence-ready%20physical%20map%20of%20barley%20anchored%20genetically%20by%20two%20million%20SNPs&amp;author=R.%20Ariyadasa&amp;author=M.%20Mascher&amp;author=T.%20Nussbaumer&amp;publication_year=2014&amp;journal=Plant%20Physiol.&amp;volume=164&amp;pages=412-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.228213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.228213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.228213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24243933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mascher</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Muehlbauer</div> <div class="given-names">G.J.</div></div>, <div class="name string-name"><div class="surname">Rokhsar</div> <div class="given-names">D.S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ)</div>, <div class="source">Plant J.</div> , <div class="volume">76</div>, <div class="fpage">718</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anchoring%20and%20ordering%20NGS%20contig%20assemblies%20by%20population%20sequencing%20(POPSEQ)&amp;author=M.%20Mascher&amp;author=G.J.%20Muehlbauer&amp;author=D.S.%20Rokhsar&amp;publication_year=2013&amp;journal=Plant%20J.&amp;volume=76&amp;pages=718-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.76.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.76.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.76.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23998490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zeng</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">The draft genome of Tibetan hulless barley reveals adaptive patterns to the high stressful Tibetan Plateau</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">1095</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Tibetan%20hulless%20barley%20reveals%20adaptive%20patterns%20to%20the%20high%20stressful%20Tibetan%20Plateau&amp;author=X.%20Zeng&amp;author=H.%20Long&amp;author=Z.%20Wang&amp;publication_year=2015&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=1095-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1423628112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1423628112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1423628112"> </span></div> </div></div></div></div></div><div content-id="DSV033C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.S.</div></div>, <div class="name string-name"><div class="surname">Ito</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Worldwide distribution of β-amylase thermostability in barley</div>, <div class="source">Euhytica</div> , <div class="volume">121</div>, <div class="fpage">225</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worldwide%20distribution%20of%20%CE%B2-amylase%20thermostability%20in%20barley&amp;author=T.%20Kaneko&amp;author=W.S.%20Zhang&amp;author=K.%20Ito&amp;publication_year=2001&amp;journal=Euhytica&amp;volume=121&amp;pages=225-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1012095811104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1012095811104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1012095811104"> </span></div> </div></div></div></div></div><div content-id="DSV033C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gong</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Westcott</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Discovery of novel Bmy1 alleles increasing β-amylase activity in Chinese landraces and Tibetan wild barley for improvement of malting quality via MAS</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e72875</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20novel%20Bmy1%20alleles%20increasing%20%CE%B2-amylase%20activity%20in%20Chinese%20landraces%20and%20Tibetan%20wild%20barley%20for%20improvement%20of%20malting%20quality%20via%20MAS&amp;author=X.%20Gong&amp;author=S.%20Westcott&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e72875" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0072875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0072875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0072875"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24019884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Collins</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Panozzo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Logue</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Mapping and validation of chromosome regions associated with high malt extract in barley (<em>Hordeum vulgare</em> L.)</div>, <div class="source">Aust. J. Agr. Res.</div> , <div class="volume">54</div>, <div class="fpage">1223</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20validation%20of%20chromosome%20regions%20associated%20with%20high%20malt%20extract%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=H.%20Collins&amp;author=J.%20Panozzo&amp;author=S.%20Logue&amp;publication_year=2003&amp;journal=Aust.%20J.%20Agr.%20Res.&amp;volume=54&amp;pages=1223-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/AR02201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FAR02201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FAR02201"> </span></div> </div></div></div></div></div><div content-id="DSV033C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Nankaku</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Takeda</div> <div class="given-names">K.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">A high-density transcript linkage map of barley derived from a single population</div>, <div class="source">Heredity (Edinb)</div> , <div class="volume">103</div>, <div class="fpage">110</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20transcript%20linkage%20map%20of%20barley%20derived%20from%20a%20single%20population&amp;author=K.%20Sato&amp;author=N.%20Nankaku&amp;author=K.%20Takeda&amp;publication_year=2009&amp;journal=Heredity%20(Edinb)&amp;volume=103&amp;pages=110-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2009.57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2009.57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2009.57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19455180" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Close</div>
               <div class="given-names">T.J.</div>
            </div>, <div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>, <div class="name string-name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Development and implementation of high-throughput SNP genotyping in barley</div>, <div class="source">BMC Genomics</div> , <div class="volume">10</div>, <div class="fpage">582</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20high-throughput%20SNP%20genotyping%20in%20barley&amp;author=T.J.%20Close&amp;author=P.R.%20Bhat&amp;author=S.%20Lonardi&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saisho</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Myoraku</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Takeda</div> <div class="given-names">K.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Construction and characterization of a bacterial artificial chromosome (BAC) library for Japanese malting barley ‘Haruna Nijo</div>’, <div class="source">Breed. Sci.</div> , <div class="volume">57</div>, <div class="fpage">29</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20a%20bacterial%20artificial%20chromosome%20(BAC)%20library%20for%20Japanese%20malting%20barley%20%E2%80%98Haruna%20Nijo&amp;author=D.%20Saisho&amp;author=E.%20Myoraku&amp;author=S.%20Kawasaki&amp;author=K.%20Sato&amp;author=K.%20Takeda&amp;publication_year=2007&amp;journal=Breed.%20Sci.&amp;volume=57&amp;pages=29-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.57.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.57.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.57.29"> </span></div> </div></div></div></div></div><div content-id="DSV033C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Shin</div> <div class="given-names">I.T.</div></div>, <div class="name string-name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Development of 5006 full-length CDNAs in barley: a tool for accessing cereal genomics resources</div>, <div class="source">DNA Res.</div> , <div class="volume">16</div>, <div class="fpage">81</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%205006%20full-length%20CDNAs%20in%20barley%3A%20a%20tool%20for%20accessing%20cereal%20genomics%20resources&amp;author=K.%20Sato&amp;author=I.T.%20Shin&amp;author=M.%20Seki&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=81-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19150987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Matsumoto</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries</div>, <div class="source">Plant Physiol.</div> , <div class="volume">156</div>, <div class="fpage">20</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20sequence%20analysis%20of%2024%2C783%20barley%20full-length%20cDNAs%20derived%20from%2012%20clone%20libraries&amp;author=T.%20Matsumoto&amp;author=T.%20Tanaka&amp;author=H.%20Sakai&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=20-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.171579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.171579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.171579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21415278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Development of genetic map and alignment of recombinant chromo-some substitution lines from a cross of EST donors by high accuracy SNP typing in barley</div>, <div class="source">Plant Cell Physiol.</div> , <div class="volume">52</div>, <div class="fpage">728</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genetic%20map%20and%20alignment%20of%20recombinant%20chromo-some%20substitution%20lines%20from%20a%20cross%20of%20EST%20donors%20by%20high%20accuracy%20SNP%20typing%20in%20barley&amp;author=K.%20Sato&amp;author=T.J.%20Close&amp;author=P.%20Bhat&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=728-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21427110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jurka</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Dagman</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pelton</div> <div class="given-names">P.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">CENSOR-a program for identification and elimination of repetitive elements from DNA sequences</div>, <div class="source">Comput. Chem.</div> , <div class="volume">20</div>, <div class="fpage">119</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CENSOR-a%20program%20for%20identification%20and%20elimination%20of%20repetitive%20elements%20from%20DNA%20sequences&amp;author=J.%20Jurka&amp;author=P.%20Klonowski&amp;author=V.%20Dagman&amp;author=P.%20Pelton&amp;publication_year=1996&amp;journal=Comput.%20Chem.&amp;volume=20&amp;pages=119-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0097-8485(96)80013-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(96)80013-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0097-8485(96)80013-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8867843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="collab">The International Wheat Genome Sequencing Consortium</div>. <div class="year">2014</div>, <div class="article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome</div>, <div class="source">Science</div> , <div class="volume">345</div>, <div class="comment">1251788</div>.</div></div></div></div><div content-id="DSV033C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ling</div>
               <div class="given-names">H.Q.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Draft genome of the wheat A-genome progenitor <em>Triticum urartu</em></div>, <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">87</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20wheat%20A-genome%20progenitor%20Triticum%20urartu&amp;author=H.Q.%20Ling&amp;author=S.%20Zhao&amp;author=D.%20Liu&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jia</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title"><em>Aegilops tauschii</em> draft genome sequence reveals a gene repertoire for wheat adaptation</div>, <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">91</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aegilops%20tauschii%20draft%20genome%20sequence%20reveals%20a%20gene%20repertoire%20for%20wheat%20adaptation&amp;author=J.%20Jia&amp;author=S.%20Zhao&amp;author=X.%20Kong&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=91-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">bex-db:Bioinformatics workbench for comprehensive analysis of barley-expressed genes</div>, <div class="source">Breeding Sci.</div> , <div class="volume">63</div>, <div class="fpage">430</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=bex-db%3ABioinformatics%20workbench%20for%20comprehensive%20analysis%20of%20barley-expressed%20genes&amp;author=T.%20Tanaka&amp;author=H.%20Sakai&amp;author=N.%20Fujii&amp;publication_year=2013&amp;journal=Breeding%20Sci.&amp;volume=63&amp;pages=430-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.430"> </span></div> </div></div></div></div></div><div content-id="DSV033C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mott</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">1997</div>, <div class="article-title">EST_GENOME: a program to align spliced DNA sequences to unspliced genomic DNA</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="volume">13</div>, <div class="fpage">477</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST_GENOME%3A%20a%20program%20to%20align%20spliced%20DNA%20sequences%20to%20unspliced%20genomic%20DNA&amp;author=R.%20Mott&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=477-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9283765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bolger</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Trimmomatic: a flexible trimmer for Illumina Sequence Data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20Sequence%20Data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">ToHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>, <div class="source">Genome Biol.</div> , <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ToHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotech.</div> , <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=B.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotech.&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div> </div></div></div></div></div><div content-id="DSV033C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jones</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H.Y.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">InterProScan 5: genome-scale protein function classification</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1236</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McCarthy</div>
               <div class="given-names">F.M.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Magee</div> <div class="given-names">G.B.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">AgBase: a functional genomics resource for agriculture</div>, <div class="source">BMC Genomics</div> , <div class="volume">7</div>, <div class="fpage">229</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AgBase%3A%20a%20functional%20genomics%20resource%20for%20agriculture&amp;author=F.M.%20McCarthy&amp;author=N.%20Wang&amp;author=G.B.%20Magee&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16961921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quast</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Pruesse</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Yilmaz</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The SILVA ribosomal RNA gene database project: improved data processing and web-based tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">D590</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SILVA%20ribosomal%20RNA%20gene%20database%20project%3A%20improved%20data%20processing%20and%20web-based%20tools&amp;author=C.%20Quast&amp;author=E.%20Pruesse&amp;author=P.%20Yilmaz&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D590-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">T.D.</div>
            </div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Koyanagi</div> <div class="given-names">K.O.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Highly diversified molecular evolution of downstream transcription start sites in Oryza sativa and Arabidopsis thaliana</div>, <div class="source">Plant Physiol.</div> , <div class="volume">149</div>, <div class="fpage">1316</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20diversified%20molecular%20evolution%20of%20downstream%20transcription%20start%20sites%20in%20Oryza%20sativa%20and%20Arabidopsis%20thaliana&amp;author=T.%20Tanaka&amp;author=K.O.%20Koyanagi&amp;author=T.%20Itoh&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=1316-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.131656" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.131656" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.131656"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19118127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nair</div>
               <div class="given-names">S.K.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Turuspekov</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Cleistogamous flowering in barley arises from the suppression of microRNA-guided <em>HvAP2</em> mRNA cleavage</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">490</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cleistogamous%20flowering%20in%20barley%20arises%20from%20the%20suppression%20of%20microRNA-guided%20HvAP2%20mRNA%20cleavage&amp;author=S.K.%20Nair&amp;author=N.%20Wang&amp;author=Y.%20Turuspekov&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=490-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909097107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909097107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909097107"> </span></div> </div></div></div></div></div><div content-id="DSV033C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Flavell</div>
               <div class="given-names">R.B.</div>
            </div>, <div class="name string-name"><div class="surname">O'Dell</div> <div class="given-names">M.</div></div></div>
          <div class="year">1976</div>, <div class="article-title">Ribosomal RNA genes on homoeologous chromosomes of groups 5 and 6 in hexaploid wheat</div>, <div class="source">Heredity</div> , <div class="volume">37</div>, <div class="fpage">377</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosomal%20RNA%20genes%20on%20homoeologous%20chromosomes%20of%20groups%205%20and%206%20in%20hexaploid%20wheat&amp;author=R.B.%20Flavell&amp;author=M.%20O'Dell&amp;publication_year=1976&amp;journal=Heredity&amp;volume=37&amp;pages=377-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.1976.102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.1976.102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.1976.102"> </span></div> </div></div></div></div></div><div content-id="DSV033C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Joshi</div> <div class="given-names">G.P.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Next-generation survey sequencing and the molecular organization of wheat chromosome 6B</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">103</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20survey%20sequencing%20and%20the%20molecular%20organization%20of%20wheat%20chromosome%206B&amp;author=T.%20Tanaka&amp;author=F.%20Kobayashi&amp;author=G.P.%20Joshi&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=103-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mizuno</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Characterization of internal structure of the nucleolar organizing region in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Cytonegetic Genome Res.</div> , <div class="volume">121</div>, <div class="fpage">282</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20internal%20structure%20of%20the%20nucleolar%20organizing%20region%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=H.%20Mizuno&amp;author=T.%20Sasaki&amp;author=T.%20Matsumoto&amp;publication_year=2008&amp;journal=Cytonegetic%20Genome%20Res.&amp;volume=121&amp;pages=282-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000138898" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000138898" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000138898"> </span></div> </div></div></div></div></div><div content-id="DSV033C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="collab">International Rice Genome Sequencing Project</div>. <div class="year">2005</div>, <div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="volume">436</div>, <div class="fpage">793</div>–<div class="lpage">800</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV033C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV033C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stein</div>
               <div class="given-names">L.D.</div>
            </div>, <div class="name string-name"><div class="surname">Mungall</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">S.Q.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">The Generic genome browser: a building block for a model organism system database</div>, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">1599</div>–<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Generic%20genome%20browser%3A%20a%20building%20block%20for%20a%20model%20organism%20system%20database&amp;author=L.D.%20Stein&amp;author=C.%20Mungall&amp;author=S.Q.%20Shu&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1599-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.403602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.403602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.403602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462418&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv033/1/dsv033_Supplementary_Data.zip?Expires=2147483647&amp;Signature=St5Sja~ihdPXn-tclE-csLJmK-AnUVoaFc5Wd2lyh~AC8RBKAqbvEEIRqi~P7oKJCUfMxR9m3OJZhtXLngnm7S5U7Uj7paq6qfbpEUzriGclAFE8q4mjS~QasMd-zZPXQ36gFqFGaXM1DCkfeNHdVfiPkb7pzm0x-L3ri~n6c~HlasIVqUdXafztmCfnBo2JR6vrKUL4TzT37Cq00OTcmirQmaS1O85CkupJf7VWt4kNBl6n51JaMcktQXy~9GXUZ26nB5F~vvqnkur51Ej5iJWhXmlq-n~~WAbSWCu4M6026x4MBEnO7TjJ3vH4pQh1mUrW2PIevDxLPC0TK5Kk9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,799</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv033"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462418"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f1%2f21%2f2462418">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000370973100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F1%2F21%2F2462418" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462418'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Improvement of barley genome annotations by deciphering the Haruna Nijo genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv033");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f21%2f2462418">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

