

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/2/115/401903","@type":"ScholarlyArticle","name":"Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem","datePublished":"2013-10-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2014-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst044","keywords":["<italic>Ovis aries</italic>","microbiome","16S subunit"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/2/115/401903","author":[{"name":"Ellison, Melinda J.","affiliation":"1  Department of Animal Science ,  University of Wyoming ,  Laramie, WY ,  USA","@type":"Person"},{"name":"Conant, Gavin C.","affiliation":"2  Division of Animal Sciences ,  University of Missouri ,  Columbia, MO ,  USA    3  Informatics Institute ,  University of Missouri ,  Columbia, MO ,  USA","@type":"Person"},{"name":"Cockrum, Rebecca R.","affiliation":"4  Department of Animal Science ,  Colorado State University ,  Fort Collins, CO ,  USA","@type":"Person"},{"name":"Austin, Kathy J.","affiliation":"1  Department of Animal Science ,  University of Wyoming ,  Laramie, WY ,  USA","@type":"Person"},{"name":"Truong, Huan","affiliation":"3  Informatics Institute ,  University of Missouri ,  Columbia, MO ,  USA","@type":"Person"},{"name":"Becchi, Michela","affiliation":"3  Informatics Institute ,  University of Missouri ,  Columbia, MO ,  USA    5  Department of Electrical and Computer Engineering ,  University of Missouri ,  Columbia, MO ,  USA","@type":"Person"},{"name":"Lamberson, William R.","affiliation":"2  Division of Animal Sciences ,  University of Missouri ,  Columbia, MO ,  USA","@type":"Person"},{"name":"Cammack, Kristi M.","affiliation":"1  Department of Animal Science ,  University of Wyoming ,  Laramie, WY ,  USA","@type":"Person"}],"description":"Abstract.  We surveyed the ruminal metagenomes of 16 sheep under two different diets using Illumina pair-end DNA sequencing of raw microbial DNA extracted from ","pageStart":"115","pageEnd":"125","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem" />
    <meta property="og:description" content="Abstract.  We surveyed the ruminal metagenomes of 16 sheep under two different diets using Illumina pair-end DNA sequencing of raw microbial DNA extracted from " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/2/115/401903" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-10-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ellison, Melinda J." /><meta name="citation_author_institution" content="Department of Animal Science, University of Wyoming, Laramie, WY, USA" /><meta name="citation_author" content="Conant, Gavin C." /><meta name="citation_author_institution" content="Division of Animal Sciences, University of Missouri, Columbia, MO, USA" /><meta name="citation_author_institution" content="Informatics Institute, University of Missouri, Columbia, MO, USA" /><meta name="citation_author" content="Cockrum, Rebecca R." /><meta name="citation_author_institution" content="Department of Animal Science, Colorado State University, Fort Collins, CO, USA" /><meta name="citation_author" content="Austin, Kathy J." /><meta name="citation_author_institution" content="Department of Animal Science, University of Wyoming, Laramie, WY, USA" /><meta name="citation_author" content="Truong, Huan" /><meta name="citation_author_institution" content="Informatics Institute, University of Missouri, Columbia, MO, USA" /><meta name="citation_author" content="Becchi, Michela" /><meta name="citation_author_institution" content="Informatics Institute, University of Missouri, Columbia, MO, USA" /><meta name="citation_author_institution" content="Department of Electrical and Computer Engineering, University of Missouri, Columbia, MO, USA" /><meta name="citation_author" content="Lamberson, William R." /><meta name="citation_author_institution" content="Division of Animal Sciences, University of Missouri, Columbia, MO, USA" /><meta name="citation_author" content="Cammack, Kristi M." /><meta name="citation_author_institution" content="Department of Animal Science, University of Wyoming, Laramie, WY, USA" /><meta name="citation_title" content="Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem" /><meta name="citation_firstpage" content="115" /><meta name="citation_lastpage" content="125" /><meta name="citation_doi" content="10.1093/dnares/dst044" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2014/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Metagenomics: DNA sequencing of environmental samples; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=805-14; " /><meta name="citation_reference" content="citation_title=Mutualistic fermentative digestion in the gastrointestinal tract: diversity and evolution; Integr. Comput. Biol.;  citation_year=2002;  citation_volume=42;  citation_pages=319-26; " /><meta name="citation_reference" content="citation_title=Worlds within worlds: evolution of the vertebrate gut microbiota; Nat. Rev. Microbiol.;  citation_year=2008;  citation_volume=6;  citation_pages=776-88; " /><meta name="citation_reference" content="citation_title=Evolution of mammals and their gut microbes; Science;  citation_year=2008;  citation_volume=320;  citation_pages=1647-51; " /><meta name="citation_reference" content="citation_title=Structure, function and diversity of the healthy human microbiome; citation_author=Human Microbiome Project Consortium; Nature;  citation_year=2012;  citation_volume=486;  citation_pages=207-14; " /><meta name="citation_reference" content="citation_title=Obesity alters gut microbial ecology; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=11070-5; " /><meta name="citation_reference" content="citation_title=An obesity-associated gut microbiome with increased capacity for energy harvest; Nature;  citation_year=2006;  citation_volume=444;  citation_pages=1027-31; " /><meta name="citation_reference" content="citation_title=Gene-centric metagenomics of the fiber-adherent bovine rumen microbiome reveals forage specific glycoside hydrolases; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=1948-53; " /><meta name="citation_reference" content="citation_title=Status of the phylogenetic diversity census of ruminal microbiomes; FEMS Microbiol. Ecol.;  citation_year=2011;  citation_volume=76;  citation_pages=49-63; " /><meta name="citation_reference" content="citation_title=Unsuspected diversity among marine aerobic anoxygenic phototrophs; Nature;  citation_year=2002;  citation_volume=415;  citation_pages=630-3; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of rumen bacteria by comparative sequence analysis of cloned 16S rRNA genes; Anaerobe;  citation_year=1998;  citation_volume=4;  citation_pages=153-63; " /><meta name="citation_reference" content="citation_title=Metagenomic discovery of biomass-degrading genes and genomes from cow rumen; Science;  citation_year=2011;  citation_volume=331;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=16S rDNA library-based analysis of ruminal bacterial diversity; Antonie Van Leeuwenhoek;  citation_year=2004;  citation_volume=86;  citation_pages=263-81; " /><meta name="citation_reference" content="citation_title=Linkage of microbial ecology to phenotype: correlation of rumen microbial ecology to cattle's feed efficiency; FEMS Microbiol. Lett.;  citation_year=2008;  citation_volume=288;  citation_pages=85-91; " /><meta name="citation_reference" content="citation_title=Assessment of the microbial ecology of ruminal methanogens in cattle with different feed efficiencies; Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=6524-33; " /><meta name="citation_reference" content="citation_title=Composition, spatial distribution, and diversity of the bacterial communities in the rumen of cows fed different forages; FEMS Microbiol. Ecol.;  citation_year=2010;  citation_volume=74;  citation_pages=612-22; " /><meta name="citation_reference" content="citation_title=Rumen bacterial diversity dynamics associated with changing from bermudagrass hay to grazed winter wheat diets; Microb. Ecol.;  citation_year=2010;  citation_volume=59;  citation_pages=511-22; " /><meta name="citation_reference" content="citation_title=Estimates of genetic parameters for feed intake, feeding behavior, and daily gain in composite ram lambs; J. Anim. Sci.;  citation_year=2005;  citation_volume=83;  citation_pages=777-85; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=186-94; " /><meta name="citation_reference" content="citation_title=EMIRGE: reconstruction of full-length ribosomal genes from microbial community short read sequencing data; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R44" /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=The Ribosomal Database Project: improved alignments and new tools for rRNA analysis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D141-5; " /><meta name="citation_reference" content="citation_title=GenBank; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D36-42; " /><meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins; J. Mol. Biol.;  citation_year=1970;  citation_volume=48;  citation_pages=443-53; " /><meta name="citation_reference" content="citation_title=A distributed CPU-GPU framework for pairwise alignments on large-scale sequence datasets;  citation_year=2013;  citation_pages=329-38; " /><meta name="citation_reference" content="citation_title=Altered patterns of gene duplication and differential gene gain and loss in fungal pathogens; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=147" /><meta name="citation_reference" content=" citation_edition=3rd edition; citation_publisher=W. H. Freeman and Company, New York; Biometry;  citation_year=1995; " /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information: update; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D35-40; " /><meta name="citation_reference" content="citation_title=Species abundance distributions: moving beyond single prediction theories to integration within an ecological framework; Ecol. Lett.;  citation_year=2007;  citation_volume=10;  citation_pages=995-1015; " /><meta name="citation_reference" content="citation_title=Links between the species abundance distribution and the shape of the corresponding rank abundance curve; Ecol. Indicators;  citation_year=2012;  citation_volume=14;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, New York; Numerical Recipes in C;  citation_year=1992; " /><meta name="citation_reference" content="citation_title=Functional partitioning of yeast co-expression networks after genome duplication; PLoS. Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e109" /><meta name="citation_reference" content="citation_title=Changes in Atmospheric Constituents and in Radiative Forcing; citation_publisher=Cambridge University Press, Cambridge, UK; Climate Change 2007: The Physical Science Basis. Contribution of Working Group I to the Fourth Assessment Report of the Intergovernmental Panel on Climate Change;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=The role of pH in regulating ruminal methane and ammonia production; J. Anim. Sci.;  citation_year=1998;  citation_volume=76;  citation_pages=2190-6; " /><meta name="citation_reference" content="citation_title=Species abundance distributions and richness estimations in fungal metagenomics—lessons learned from community ecology; Mol. Ecol.;  citation_year=2011;  citation_volume=20;  citation_pages=275-85; " /><meta name="citation_reference" content="citation_title=Genetic diversity and diet specificity of ruminal Prevotella revealed by 16S rRNA gene-based analysis; FEMS Microbiol. Lett.;  citation_year=2010;  citation_volume=305;  citation_pages=49-57; " /><meta name="citation_reference" content="citation_title=Dominance of Prevotella and low abundance of classical ruminal bacterial species in the bovine rumen revealed by relative quantification real-time PCR; Appl. Microbiol. Biotechnol.;  citation_year=2007;  citation_volume=75;  citation_pages=165-74; " /><meta name="citation_reference" content="citation_title=Consequences of dominance: a review of evenness effects on local and regional ecosystem processes; Ecology;  citation_year=2008;  citation_volume=89;  citation_pages=1510-20; " /><meta name="citation_reference" content="citation_title=A consistent terminology for quantifying species diversity? Yes, it does exist; Oecologia;  citation_year=2010;  citation_volume=164;  citation_pages=853-60; " /><meta name="citation_reference" content="citation_title=Prokaryotic diversity–magnitude, dynamics, and controlling factors; Science;  citation_year=2002;  citation_volume=296;  citation_pages=1064-6; " /><meta name="citation_reference" content="citation_title=Phenotypic diversity and ecosystem functioning in changing environments: a theoretical framework; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11376-81; " /><meta name="citation_reference" content="citation_title=How important are species richness, species evenness and interspeciﬁc differences to productivity? A mathematical model; Oikos;  citation_year=2000;  citation_volume=88;  citation_pages=57-66; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/2/115/1124352/dst044.pdf" /><meta name="description" content="Abstract.  We surveyed the ruminal metagenomes of 16 sheep under two different diets using Illumina pair-end DNA sequencing of raw microbial DNA extracted from " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/2/115/401903" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/2/115/401903" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/2/115/401903");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst044';
    </script>
<title>
	Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./401903" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be1iPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFbGxpc29uLCBNZWxpbmRhIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBXeW9taW5nLCBMYXJhbWllLCBXWSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ29uYW50LCBHYXZpbiBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEFuaW1hbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBNaXNzb3VyaSwgQ29sdW1iaWEsIE1PLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbmZvcm1hdGljcyBJbnN0aXR1dGUsIFVuaXZlcnNpdHkgb2YgTWlzc291cmksIENvbHVtYmlhLCBNTywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ29ja3J1bSwgUmViZWNjYSBSLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIFNjaWVuY2UsIENvbG9yYWRvIFN0YXRlIFVuaXZlcnNpdHksIEZvcnQgQ29sbGlucywgQ08sIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkF1c3RpbiwgS2F0aHkgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBTY2llbmNlLCBVbml2ZXJzaXR5IG9mIFd5b21pbmcsIExhcmFtaWUsIFdZLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUcnVvbmcsIEh1YW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbmZvcm1hdGljcyBJbnN0aXR1dGUsIFVuaXZlcnNpdHkgb2YgTWlzc291cmksIENvbHVtYmlhLCBNTywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmVjY2hpLCBNaWNoZWxhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5mb3JtYXRpY3MgSW5zdGl0dXRlLCBVbml2ZXJzaXR5IG9mIE1pc3NvdXJpLCBDb2x1bWJpYSwgTU8sIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRWxlY3RyaWNhbCBhbmQgQ29tcHV0ZXIgRW5naW5lZXJpbmcsIFVuaXZlcnNpdHkgb2YgTWlzc291cmksIENvbHVtYmlhLCBNTywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGFtYmVyc29uLCBXaWxsaWFtIFIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgQW5pbWFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE1pc3NvdXJpLCBDb2x1bWJpYSwgTU8sIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhbW1hY2ssIEtyaXN0aSBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgV3lvbWluZywgTGFyYW1pZSwgV1ksIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGlldCBBbHRlcnMgQm90aCB0aGUgU3RydWN0dXJlIGFuZCBUYXhvbm9teSBvZiB0aGUgT3ZpbmUgR3V0IE1pY3JvYmlhbCBFY29zeXN0ZW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDA0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWNzOiBETkEgc2VxdWVuY2luZyBvZiBlbnZpcm9ubWVudGFsIHNhbXBsZXM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTgwNS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0dWFsaXN0aWMgZmVybWVudGF0aXZlIGRpZ2VzdGlvbiBpbiB0aGUgZ2FzdHJvaW50ZXN0aW5hbCB0cmFjdDogZGl2ZXJzaXR5IGFuZCBldm9sdXRpb247IEludGVnci4gQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTMxOS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ybGRzIHdpdGhpbiB3b3JsZHM6IGV2b2x1dGlvbiBvZiB0aGUgdmVydGVicmF0ZSBndXQgbWljcm9iaW90YTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTc3Ni04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIG1hbW1hbHMgYW5kIHRoZWlyIGd1dCBtaWNyb2JlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjA7ICBjaXRhdGlvbl9wYWdlcz0xNjQ3LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUsIGZ1bmN0aW9uIGFuZCBkaXZlcnNpdHkgb2YgdGhlIGhlYWx0aHkgaHVtYW4gbWljcm9iaW9tZTsgY2l0YXRpb25fYXV0aG9yPUh1bWFuIE1pY3JvYmlvbWUgUHJvamVjdCBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg2OyAgY2l0YXRpb25fcGFnZXM9MjA3LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PYmVzaXR5IGFsdGVycyBndXQgbWljcm9iaWFsIGVjb2xvZ3k7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTExMDcwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIG9iZXNpdHktYXNzb2NpYXRlZCBndXQgbWljcm9iaW9tZSB3aXRoIGluY3JlYXNlZCBjYXBhY2l0eSBmb3IgZW5lcmd5IGhhcnZlc3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMDI3LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lLWNlbnRyaWMgbWV0YWdlbm9taWNzIG9mIHRoZSBmaWJlci1hZGhlcmVudCBib3ZpbmUgcnVtZW4gbWljcm9iaW9tZSByZXZlYWxzIGZvcmFnZSBzcGVjaWZpYyBnbHljb3NpZGUgaHlkcm9sYXNlczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTk0OC01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdHVzIG9mIHRoZSBwaHlsb2dlbmV0aWMgZGl2ZXJzaXR5IGNlbnN1cyBvZiBydW1pbmFsIG1pY3JvYmlvbWVzOyBGRU1TIE1pY3JvYmlvbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz00OS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5zdXNwZWN0ZWQgZGl2ZXJzaXR5IGFtb25nIG1hcmluZSBhZXJvYmljIGFub3h5Z2VuaWMgcGhvdG90cm9waHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTU7ICBjaXRhdGlvbl9wYWdlcz02MzAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW52aXJvbm1lbnRhbCBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jaW5nIG9mIHRoZSBTYXJnYXNzbyBTZWE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA0OyAgY2l0YXRpb25fcGFnZXM9NjYtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBydW1lbiBiYWN0ZXJpYSBieSBjb21wYXJhdGl2ZSBzZXF1ZW5jZSBhbmFseXNpcyBvZiBjbG9uZWQgMTZTIHJSTkEgZ2VuZXM7IEFuYWVyb2JlOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTMtNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljIGRpc2NvdmVyeSBvZiBiaW9tYXNzLWRlZ3JhZGluZyBnZW5lcyBhbmQgZ2Vub21lcyBmcm9tIGNvdyBydW1lbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzE7ICBjaXRhdGlvbl9wYWdlcz00NjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9MTZTIHJETkEgbGlicmFyeS1iYXNlZCBhbmFseXNpcyBvZiBydW1pbmFsIGJhY3RlcmlhbCBkaXZlcnNpdHk7IEFudG9uaWUgVmFuIExlZXV3ZW5ob2VrOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9MjYzLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIG9mIG1pY3JvYmlhbCBlY29sb2d5IHRvIHBoZW5vdHlwZTogY29ycmVsYXRpb24gb2YgcnVtZW4gbWljcm9iaWFsIGVjb2xvZ3kgdG8gY2F0dGxlJ3MgZmVlZCBlZmZpY2llbmN5OyBGRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Mjg4OyAgY2l0YXRpb25fcGFnZXM9ODUtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgdGhlIG1pY3JvYmlhbCBlY29sb2d5IG9mIHJ1bWluYWwgbWV0aGFub2dlbnMgaW4gY2F0dGxlIHdpdGggZGlmZmVyZW50IGZlZWQgZWZmaWNpZW5jaWVzOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9NjUyNC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb24sIHNwYXRpYWwgZGlzdHJpYnV0aW9uLCBhbmQgZGl2ZXJzaXR5IG9mIHRoZSBiYWN0ZXJpYWwgY29tbXVuaXRpZXMgaW4gdGhlIHJ1bWVuIG9mIGNvd3MgZmVkIGRpZmZlcmVudCBmb3JhZ2VzOyBGRU1TIE1pY3JvYmlvbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz02MTItMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJ1bWVuIGJhY3RlcmlhbCBkaXZlcnNpdHkgZHluYW1pY3MgYXNzb2NpYXRlZCB3aXRoIGNoYW5naW5nIGZyb20gYmVybXVkYWdyYXNzIGhheSB0byBncmF6ZWQgd2ludGVyIHdoZWF0IGRpZXRzOyBNaWNyb2IuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NTExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0ZXMgb2YgZ2VuZXRpYyBwYXJhbWV0ZXJzIGZvciBmZWVkIGludGFrZSwgZmVlZGluZyBiZWhhdmlvciwgYW5kIGRhaWx5IGdhaW4gaW4gY29tcG9zaXRlIHJhbSBsYW1iczsgSi4gQW5pbS4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT04MzsgIGNpdGF0aW9uX3BhZ2VzPTc3Ny04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzZS1jYWxsaW5nIG9mIGF1dG9tYXRlZCBzZXF1ZW5jZXIgdHJhY2VzIHVzaW5nIHBocmVkLiBJSS4gRXJyb3IgcHJvYmFiaWxpdGllczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE4Ni05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RU1JUkdFOiByZWNvbnN0cnVjdGlvbiBvZiBmdWxsLWxlbmd0aCByaWJvc29tYWwgZ2VuZXMgZnJvbSBtaWNyb2JpYWwgY29tbXVuaXR5IHNob3J0IHJlYWQgc2VxdWVuY2luZyBkYXRhOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1SNDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYWZhc3QgYW5kIG1lbW9yeS1lZmZpY2llbnQgYWxpZ25tZW50IG9mIHNob3J0IEROQSBzZXF1ZW5jZXMgdG8gdGhlIGh1bWFuIGdlbm9tZTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJpYm9zb21hbCBEYXRhYmFzZSBQcm9qZWN0OiBpbXByb3ZlZCBhbGlnbm1lbnRzIGFuZCBuZXcgdG9vbHMgZm9yIHJSTkEgYW5hbHlzaXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQxNDEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuQmFuazsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDM2LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmVyYWwgbWV0aG9kIGFwcGxpY2FibGUgdG8gdGhlIHNlYXJjaCBmb3Igc2ltaWxhcml0aWVzIGluIHRoZSBhbWlubyBhY2lkIHNlcXVlbmNlIG9mIHR3byBwcm90ZWluczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk3MDsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkaXN0cmlidXRlZCBDUFUtR1BVIGZyYW1ld29yayBmb3IgcGFpcndpc2UgYWxpZ25tZW50cyBvbiBsYXJnZS1zY2FsZSBzZXF1ZW5jZSBkYXRhc2V0czsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3BhZ2VzPTMyOS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJlZCBwYXR0ZXJucyBvZiBnZW5lIGR1cGxpY2F0aW9uIGFuZCBkaWZmZXJlbnRpYWwgZ2VuZSBnYWluIGFuZCBsb3NzIGluIGZ1bmdhbCBwYXRob2dlbnM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249M3JkIGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1XLiBILiBGcmVlbWFuIGFuZCBDb21wYW55LCBOZXcgWW9yazsgQmlvbWV0cnk7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGFiYXNlIHJlc291cmNlcyBvZiB0aGUgTmF0aW9uYWwgQ2VudGVyIGZvciBCaW90ZWNobm9sb2d5IEluZm9ybWF0aW9uOiB1cGRhdGU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQzNS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2llcyBhYnVuZGFuY2UgZGlzdHJpYnV0aW9uczogbW92aW5nIGJleW9uZCBzaW5nbGUgcHJlZGljdGlvbiB0aGVvcmllcyB0byBpbnRlZ3JhdGlvbiB3aXRoaW4gYW4gZWNvbG9naWNhbCBmcmFtZXdvcms7IEVjb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9OTk1LTEwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmtzIGJldHdlZW4gdGhlIHNwZWNpZXMgYWJ1bmRhbmNlIGRpc3RyaWJ1dGlvbiBhbmQgdGhlIHNoYXBlIG9mIHRoZSBjb3JyZXNwb25kaW5nIHJhbmsgYWJ1bmRhbmNlIGN1cnZlOyBFY29sLiBJbmRpY2F0b3JzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyBOdW1lcmljYWwgUmVjaXBlcyBpbiBDOyAgY2l0YXRpb25feWVhcj0xOTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIHBhcnRpdGlvbmluZyBvZiB5ZWFzdCBjby1leHByZXNzaW9uIG5ldHdvcmtzIGFmdGVyIGdlbm9tZSBkdXBsaWNhdGlvbjsgUExvUy4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFuZ2VzIGluIEF0bW9zcGhlcmljIENvbnN0aXR1ZW50cyBhbmQgaW4gUmFkaWF0aXZlIEZvcmNpbmc7IGNpdGF0aW9uX3B1Ymxpc2hlcj1DYW1icmlkZ2UgVW5pdmVyc2l0eSBQcmVzcywgQ2FtYnJpZGdlLCBVSzsgQ2xpbWF0ZSBDaGFuZ2UgMjAwNzogVGhlIFBoeXNpY2FsIFNjaWVuY2UgQmFzaXMuIENvbnRyaWJ1dGlvbiBvZiBXb3JraW5nIEdyb3VwIEkgdG8gdGhlIEZvdXJ0aCBBc3Nlc3NtZW50IFJlcG9ydCBvZiB0aGUgSW50ZXJnb3Zlcm5tZW50YWwgUGFuZWwgb24gQ2xpbWF0ZSBDaGFuZ2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIHBIIGluIHJlZ3VsYXRpbmcgcnVtaW5hbCBtZXRoYW5lIGFuZCBhbW1vbmlhIHByb2R1Y3Rpb247IEouIEFuaW0uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NzY7ICBjaXRhdGlvbl9wYWdlcz0yMTkwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZXMgYWJ1bmRhbmNlIGRpc3RyaWJ1dGlvbnMgYW5kIHJpY2huZXNzIGVzdGltYXRpb25zIGluIGZ1bmdhbCBtZXRhZ2Vub21pY3PigJRsZXNzb25zIGxlYXJuZWQgZnJvbSBjb21tdW5pdHkgZWNvbG9neTsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI3NS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgYW5kIGRpZXQgc3BlY2lmaWNpdHkgb2YgcnVtaW5hbCBQcmV2b3RlbGxhIHJldmVhbGVkIGJ5IDE2UyByUk5BIGdlbmUtYmFzZWQgYW5hbHlzaXM7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDU7ICBjaXRhdGlvbl9wYWdlcz00OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9taW5hbmNlIG9mIFByZXZvdGVsbGEgYW5kIGxvdyBhYnVuZGFuY2Ugb2YgY2xhc3NpY2FsIHJ1bWluYWwgYmFjdGVyaWFsIHNwZWNpZXMgaW4gdGhlIGJvdmluZSBydW1lbiByZXZlYWxlZCBieSByZWxhdGl2ZSBxdWFudGlmaWNhdGlvbiByZWFsLXRpbWUgUENSOyBBcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9MTY1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXF1ZW5jZXMgb2YgZG9taW5hbmNlOiBhIHJldmlldyBvZiBldmVubmVzcyBlZmZlY3RzIG9uIGxvY2FsIGFuZCByZWdpb25hbCBlY29zeXN0ZW0gcHJvY2Vzc2VzOyBFY29sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MTUxMC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zaXN0ZW50IHRlcm1pbm9sb2d5IGZvciBxdWFudGlmeWluZyBzcGVjaWVzIGRpdmVyc2l0eT8gWWVzLCBpdCBkb2VzIGV4aXN0OyBPZWNvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTY0OyAgY2l0YXRpb25fcGFnZXM9ODUzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9rYXJ5b3RpYyBkaXZlcnNpdHnigJNtYWduaXR1ZGUsIGR5bmFtaWNzLCBhbmQgY29udHJvbGxpbmcgZmFjdG9yczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz0xMDY0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZW5vdHlwaWMgZGl2ZXJzaXR5IGFuZCBlY29zeXN0ZW0gZnVuY3Rpb25pbmcgaW4gY2hhbmdpbmcgZW52aXJvbm1lbnRzOiBhIHRoZW9yZXRpY2FsIGZyYW1ld29yazsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xMTM3Ni04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IGltcG9ydGFudCBhcmUgc3BlY2llcyByaWNobmVzcywgc3BlY2llcyBldmVubmVzcyBhbmQgaW50ZXJzcGVjae+sgWMgZGlmZmVyZW5jZXMgdG8gcHJvZHVjdGl2aXR5PyBBIG1hdGhlbWF0aWNhbCBtb2RlbDsgT2lrb3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz01Ny02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzIvMTE1LzExMjQzNTIvZHN0MDQ0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBzdXJ2ZXllZCB0aGUgcnVtaW5hbCBtZXRhZ2Vub21lcyBvZiAxNiBzaGVlcCB1bmRlciB0d28gZGlmZmVyZW50IGRpZXRzIHVzaW5nIElsbHVtaW5hIHBhaXItZW5kIEROQSBzZXF1ZW5jaW5nIG9mIHJhdyBtaWNyb2JpYWwgRE5BIGV4dHJhY3RlZCBmcm9tICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDAxOTAzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKu2y2OrC+7GZLG4tvKHfzUfH0mt" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f115%2f401903" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f115%2f401903" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15712" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&amp;Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/2">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5934366">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934366">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934369">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934369">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934375">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934375">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934400">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934400">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934409">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934409">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934421">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934421">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934423">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934423">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934425">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934425">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5934427">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5934427">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/2/103/401582">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/2/127/402275">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Melinda J. Ellison</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Science</div>, <div class="insititution">University of Wyoming</div>, <div class="addr-line">Laramie, WY</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Melinda+J.+Ellison">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ellison M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ellison M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Melinda J. Ellison</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gavin C. Conant</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Animal Sciences</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Informatics Institute</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. E-mail: <a href="mailto:conantg@missouri.edu" target="_blank">conantg@missouri.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gavin+C.+Conant">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Conant G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Conant G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gavin C. Conant</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rebecca R. Cockrum</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Animal Science</div>, <div class="insititution">Colorado State University</div>, <div class="addr-line">Fort Collins, CO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rebecca+R.+Cockrum">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cockrum R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cockrum R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rebecca R. Cockrum</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kathy J. Austin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Science</div>, <div class="insititution">University of Wyoming</div>, <div class="addr-line">Laramie, WY</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kathy+J.+Austin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Austin K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Austin K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kathy J. Austin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huan Truong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Informatics Institute</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huan+Truong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Truong H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Truong H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huan Truong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michela Becchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Informatics Institute</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Electrical and Computer Engineering</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michela+Becchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Becchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Becchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michela Becchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">William R. Lamberson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Animal Sciences</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=William+R.+Lamberson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lamberson W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lamberson W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">William R. Lamberson</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kristi M. Cammack</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Science</div>, <div class="insititution">University of Wyoming</div>, <div class="addr-line">Laramie, WY</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kristi+M.+Cammack">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cammack K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cammack K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kristi M. Cammack</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 115–125, <a href='https://doi.org/10.1093/dnares/dst044'>https://doi.org/10.1093/dnares/dst044</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 April 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="401903" href="/dnaresearch/article-pdf/21/2/115/1124352/dst044.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Melinda J. Ellison, Gavin C. Conant, Rebecca R. Cockrum, Kathy J. Austin, Huan Truong, Michela Becchi, William R. Lamberson, Kristi M. Cammack;  Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem, <em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 115–125, <a href='https://doi.org/10.1093/dnares/dst044'>https://doi.org/10.1093/dnares/dst044</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/401903?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/401903?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Diet%20Alters%20Both%20the%20Structure%20and%20Taxonomy%20of%20the%20Ovine%20Gut%20Microbial%20Ecosystem&amp;publicationDate=2013-10-29&amp;volumeNum=21&amp;issueNum=2&amp;author=Ellison%2C%20Melinda%20J.%3B%20Conant%2C%20Gavin%20C.&amp;startPage=115&amp;endPage=125&amp;contentId=10.1093%2Fdnares%2Fdst044&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f115%2f401903" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5934366 id="5934366" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We surveyed the ruminal metagenomes of 16 sheep under two different diets using Illumina pair-end DNA sequencing of raw microbial DNA extracted from rumen samples. The resulting sequence data were bioinformatically mapped to known prokaryotic 16S rDNA sequences to identify the taxa present in the samples and then analysed for the presence of potentially new taxa. Strikingly, the majority of the microbial individuals found did not map to known taxa from 16S sequence databases. We used a novel statistical modelling approach to compare the taxonomic distributions between animals fed a forage-based diet and those fed concentrated grains. With this model, we found significant differences between the two groups both in the dominant taxa present in the rumen and in the overall shape of the taxa abundance curves. In general, forage-fed animals have a more diverse microbial ecosystem, whereas the concentrate-fed animals have ruminal systems more heavily dominated by a few taxa. As expected, organisms from methanogenic groups are more prevalent in forage-fed animals. Finally, all of these differences appear to be grounded in an underlying common input of new microbial individuals into the rumen environment, with common organisms from one feed group being present in the other, but at much lower abundance.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Ovis aries&quot;"><em>Ovis aries</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="microbiome">microbiome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;16S subunit&quot;">16S subunit</a></div>                            <h2 scrollto-destination=5934369 id="5934369" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microbial symbionts of mammals are ubiquitous, taxonomically diverse and highly abundant.<sup><a reveal-id="DST044C1" data-open="DST044C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Moreover, the word symbiont is used advisedly: among their many roles, gut microbes are critical in extracting nutrition for their hosts from the varied mammalian diets,<sup><a reveal-id="DST044C2" data-open="DST044C2" class="link link-ref link-reveal xref-bibr">2</a></sup> with both diet and host phylogeny being necessary predictors for understanding gut microbe diversity.<sup><a reveal-id="DST044C3" data-open="DST044C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST044C4" data-open="DST044C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The complement of these organisms varies by individual within a species,<sup><a reveal-id="DST044C5" data-open="DST044C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and this variation can alter host phenotypes,<sup><a reveal-id="DST044C6" data-open="DST044C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST044C7" data-open="DST044C7" class="link link-ref link-reveal xref-bibr">7</a></sup> a fact that makes understanding the diversity and function of these organisms of more than ecological interest.</p><p>Although other techniques are becoming available,<sup><a reveal-id="DST044C8" data-open="DST044C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the majority of current metagenomic studies have employed the sequence of the 16S subunit of the prokaryotic ribosome for taxa identification.<sup><a reveal-id="DST044C9" data-open="DST044C9" class="link link-ref link-reveal xref-bibr">9</a></sup> This gene is appealing as it should be universal and permits the use of generic PCR primers that allow amplification from very diverse taxa in a single thermocycler reaction.<sup><a reveal-id="DST044C1" data-open="DST044C1" class="link link-ref link-reveal xref-bibr">1</a></sup> As such, sequencing of 16S genes avoids the very serious biases inherent in any approach to microbial diversity that requires culturing.<sup><a reveal-id="DST044C1" data-open="DST044C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST044C9  DST044C12" data-open="DST044C9  DST044C12" class="link link-ref link-reveal xref-bibr">9–12</a></sup> More recently, it has become possible to shotgun sequence raw metagenomic samples at high depth,<sup><a reveal-id="DST044C8" data-open="DST044C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST044C13" data-open="DST044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> presumably avoiding the potential for PCR-based artefacts that can occur when directly amplifying the 16S gene<sup><a reveal-id="DST044C14" data-open="DST044C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and allowing researchers to more fully explore the genic diversity of this ecosystem.</p><p>Such ecosystem probing may be especially rewarding when studying ruminants, because they are particularly dependent on their gut microbial symbionts. The reason for this dependence is that the cellulose and other plant materials that form the basis of their diets cannot be degraded by enzymes encoded in their own genomes.<sup><a reveal-id="DST044C2" data-open="DST044C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Instead, many different microbial taxa<sup><a reveal-id="DST044C9" data-open="DST044C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST044C15" data-open="DST044C15" class="link link-ref link-reveal xref-bibr">15</a></sup> are responsible for producing a variety of enzymes that break down these plant cell components.<sup><a reveal-id="DST044C8" data-open="DST044C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST044C13" data-open="DST044C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Thus, in addition to the health-related concerns seen in microbiome studies in humans,<sup><a reveal-id="DST044C5" data-open="DST044C5" class="link link-ref link-reveal xref-bibr">5</a></sup> understanding the microbiome of domestic animals has ecological and economic relevance.</p><p>The complement of microbes in the rumen can alter several host phenotypes: both the overall microbe composition and the distribution of methanogenic microbes differ between cattle with high efficiency of converting ingested food into biomass and those with lesser efficiency.<sup><a reveal-id="DST044C15" data-open="DST044C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST044C16" data-open="DST044C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The precise nature of the animal's diet also directly influences the gut microbiota. In cattle, there are clear differences in the relative abundances of different microbial taxa (hereafter microbial distributions), depending on the type of grass consumed.<sup><a reveal-id="DST044C17" data-open="DST044C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST044C18" data-open="DST044C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Here, we sought to better understand how diet alters rumen microbial diversity, using a shotgun sequencing approach that allowed deep sampling of microbial diversity across multiple individuals. Our goal was to understand the structural differences between two ecosystems, each defined by the host diet.</p>                            <h2 scrollto-destination=5934375 id="5934375" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=5934376 id="5934376" class="section-title" data-legacy-id=s2a>2.1. Animal trial and DNA sample collection</h3>
<p>Growing wethers (<em>n</em> = 77; initial body weight = 51.3 ± 1.2 kg) of Rambouillet, Hampshire, and Suffolk breed types were randomly allocated by body weight to receive either a concentrate- (<em>CONCEN</em>: 50% corn, 31% wheat middlings, yielding a measured dietary intake of 91.6% dry matter including 12.1% crude protein, 17.6% neutral detergent fibre, and a mean energy of 2.98 Mcal/kg; <em>n</em> = 39 animals) or forage-based (<em>FORG</em>: 67.7% alfalfa, 27.5% wheat middlings, yielding a measured dietary intake of 92.3% dry matter including 16.2% crude protein, 36.3% neutral detergent fibre, and a mean energy of 2.31 Mcal/kg; <em>n</em> = 38 animals) pelleted diet. Lambs were acclimated to diets using a 20% increase in the proportion of new-to-old feed every 4–5 days until the diet consisted of 100% new pelleted diet <em>ad libitum</em>. To give the clearest sense of the microbial diversity across these two diets, individuals were selected for metagenomic sequencing based on their rate of body weight gain relative to feed intake. To do so, individual feed intake was measured using the GrowSafe System for a 49-day trial period. Two-day average initial and final body weights were obtained to calculate daily gain. We used residual feed intake (RFI) in order to select 16 animals for metagenomic sequencing. Thus, RFI was calculated as the deviation of true feed intake from expected feed intake. Expected feed intake was determined by regressing daily gain and metabolic midweight on actual feed intake.<sup><a reveal-id="DST044C19" data-open="DST044C19" class="link link-ref link-reveal xref-bibr">19</a></sup> RFI calculations were used to rank wether efficiency. Rumen fluid samples were collected at the end of the feeding trial and frozen at −80°C. DNA was then extracted from the fluid of the 10% most (<em>n</em> = 4, low RFI) and the 10% least (<em>n</em> = 4, high RFI) efficient wethers from each diet (eight animals per diet, <em>n</em> = 16).</p>                            <h3 scrollto-destination=5934378 id="5934378" class="section-title" data-legacy-id=s2b>2.2. DNA extraction and library preparation</h3>
<p>Sterilized zirconia (0.3 g of 0.1 mm) and silicon (0.1 g of 0.5 mm) beads and 1 ml of lysis buffer were added to thawed rumen fluid samples, and tubes were homogenized using a Mini-Beadbeater-8 at maximum speed for 3 min, incubated at 70°C for 15 min with gentle mixing every 5 min, and centrifuged at 4°C for 5 min. Supernatant was transferred into new 2-ml flat cap tubes and fresh lysis buffer was added to the pelleted beads. The homogenization, incubation, and centrifugation were repeated, and the supernatants were pooled. Precipitation of nucleic acids, removal of RNA and proteins, and purification were completed using the protocol of the QIAamp DNA Stool Mini Kit (Qiagen, Santa Clarita, CA, USA). Genomic libraries from these 16 samples were constructed following the manufacturer's recommended protocol with reagents supplied in Illumina's DNA sample preparation kit. Briefly, genomic DNA was sheared using standard Diagenode BioRuptor methods to generate fragment sizes of ∼300 bp. The resulting 3′ and 5′ overhangs were removed by an end-repair reaction that uses a 3′- to 5′-exonuclease activity and polymerase activity to blunt the fragment ends. A single adenosine nucleotide was added to the 3′ ends of the blunt fragment followed by the ligation of Illumina adapters. The resulting adapter-ligated fragments were size selected on an agarose gel. Fragments of ∼420 bp were excised from the gel and recovered from the gel slice by elution and ethanol precipitation as described by the Illumina protocol. Each purified library was quantified with a Qubit assay and library fragment size confirmed by the Agilent BioAnalyzer High Sensitivity DNA assay.</p>                            <h3 scrollto-destination=5934380 id="5934380" class="section-title" data-legacy-id=s2c>2.3. Metagenomic sequencing, quality filtering, and identification of novel 16S genes</h3>
<p>Libraries were diluted and sequenced according to Illumina's standard sequencing protocol on a HiSeq 2000. The 16 libraries were multiplexed four libraries per lane, resulting in 100 bp, paired-end sequences. The mean insert size across the 16 samples was 309 bp, corresponding to an unsequenced insert between reads of ∼109 bp. Raw sequence reads are available from NCBI's short read archive (Project SRP028527).</p><p>Paired-end reads were quality filtered by truncating each read after the first run of three bases, with a phred quality score of &lt;15.<sup><a reveal-id="DST044C20" data-open="DST044C20" class="link link-ref link-reveal xref-bibr">20</a></sup> From the filtered reads, any read pair where one or both reads were &lt;85 bases long or had an average quality score of &lt;25 was omitted. The resulting reads represent 96 gigabases of sequence.</p><p>We then used the software package EMIRGE<sup><a reveal-id="DST044C21" data-open="DST044C21" class="link link-ref link-reveal xref-bibr">21</a></sup> to identify potentially unknown 16S rDNA sequences in these data. EMIRGE uses a reference 16S database (see below) and the Bowtie alignment tool<sup><a reveal-id="DST044C22" data-open="DST044C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to identify sequence reads that are potentially derived from 16S rDNA genes. It then iteratively constructs a set of new consensus 16S sequences found in the metagenomic sample, but not in the reference database.</p>                            <h3 scrollto-destination=5934384 id="5934384" class="section-title" data-legacy-id=s2d>2.4. Classification of 16S rDNA-derived reads</h3>
<p>To identify reads derived from 16S rDNA genes, we compared the filtered reads to two distinct reference databases of 16S rDNA genes. The first database (16S_Ref) was constructed by combining the Ribosomal Database collection of sequences<sup><a reveal-id="DST044C23" data-open="DST044C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and the set of 16S rDNA genes from the sequenced prokaryotic genomes at NCBI GenBank.<sup><a reveal-id="DST044C24" data-open="DST044C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Identical sequences were purged from the database, as were sequences of &lt;1450 bases long and those with undetermined nucleotides (e.g. ‘N's), resulting in a final database of 27 290 sequences. The second database (16S_Merge) comprised the union of 16S_Ref and the novel 16S sequences identified above with EMIRGE. We then used Bowtie<sup><a reveal-id="DST044C22" data-open="DST044C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to align reads from our 16 animals to these two databases. For both the forward and reverse reads, we required at least 97% sequence identity between the read and the database sequences. We retained both the best hit for each read and a second list of all database sequences, where both members of a read pair aligned with a ≥97% sequence identity. This second list was retained in order to perform the sequence clustering and operational taxonomic unit (OTU) identification described below. In Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>, we list the number of identified bacterial individuals in each sample that met these criteria. </p>                        <a id="5934386" scrollto-destination="5934386"></a>
<div content-id="DST044TB1" class="table-modal"><div class="table-wrap"><div id="DST044TB1" class="table-wrap-title" data-id="DST044TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Mapping Illumina reads to 16S rDNA databases and OTU identification</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Diet </th><th>Million paired reads<sup>a</sup> </th><th>Individual 16S genes<sup>b</sup> </th><th>% of reads from 16S<sup>c</sup> </th><th><em>P</em><sup>d</sup> </th><th>Total OTUs<sup>e</sup> </th></tr></thead><tbody><tr><td>1003 </td><td rowspan="9"><em>FORG</em> </td><td>16.8 </td><td>860/2718 </td><td>0.016 </td><td rowspan="8"> </td><td>109/419 </td></tr><tr><td>1009 </td><td>35.9 </td><td>2548/8935 </td><td>0.025 </td><td>161/489 </td></tr><tr><td>1127 </td><td>41.1 </td><td>1744/5229 </td><td>0.013 </td><td>137/467 </td></tr><tr><td>1208 </td><td>44.8 </td><td>2731/11 431 </td><td>0.026 </td><td>140/539 </td></tr><tr><td>1248 </td><td>22.7 </td><td>2078/5232 </td><td>0.023 </td><td>127/470 </td></tr><tr><td>1366 </td><td>18.1 </td><td>1615/3264 </td><td>0.018 </td><td>119/440 </td></tr><tr><td>1397 </td><td>32.2 </td><td>2184/5327 </td><td>0.017 </td><td>137/491 </td></tr><tr><td>7505 </td><td>47.2 </td><td>3049/7571 </td><td>0.016 </td><td>177/510 </td></tr><tr><td>Total </td><td>258.9 </td><td>16 809/49 707 </td><td>0.019 </td><td>&lt;10<sup>−10</sup> </td><td>280/801 </td></tr><tr><td>1026 </td><td rowspan="9"><em>CONCEN</em> </td><td>29.8 </td><td>6174/22 787 </td><td>0.076 </td><td rowspan="9"> </td><td>108/225 </td></tr><tr><td>1101 </td><td>54.9 </td><td>6401/13 579 </td><td>0.024 </td><td>142/297 </td></tr><tr><td>1111 </td><td>26.7 </td><td>2904/18 633 </td><td>0.070 </td><td>137/289 </td></tr><tr><td>1220 </td><td>7.8 </td><td>929/3758 </td><td>0.048 </td><td>75/172 </td></tr><tr><td>1239 </td><td>42.2 </td><td>5296/19 310 </td><td>0.046 </td><td>138/276 </td></tr><tr><td>1348 </td><td>13.6 </td><td>1825/5055 </td><td>0.037 </td><td>102/222 </td></tr><tr><td>1396 </td><td>18.3 </td><td>1996/8497 </td><td>0.046 </td><td>124/289 </td></tr><tr><td>7429 </td><td>30.2 </td><td>3745/12 735 </td><td>0.042 </td><td>135/345 </td></tr><tr><td>Total </td><td>223.6 </td><td>29 270/104 354 </td><td>0.047 </td><td>250/574 </td></tr><tr><td>Grand total </td><td> </td><td>482.5 </td><td>46 079/154 061 </td><td>0.032 </td><td> </td><td>349/992 </td></tr></tbody></table></div><div content-id="DST044TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Diet </th><th>Million paired reads<sup>a</sup> </th><th>Individual 16S genes<sup>b</sup> </th><th>% of reads from 16S<sup>c</sup> </th><th><em>P</em><sup>d</sup> </th><th>Total OTUs<sup>e</sup> </th></tr></thead><tbody><tr><td>1003 </td><td rowspan="9"><em>FORG</em> </td><td>16.8 </td><td>860/2718 </td><td>0.016 </td><td rowspan="8"> </td><td>109/419 </td></tr><tr><td>1009 </td><td>35.9 </td><td>2548/8935 </td><td>0.025 </td><td>161/489 </td></tr><tr><td>1127 </td><td>41.1 </td><td>1744/5229 </td><td>0.013 </td><td>137/467 </td></tr><tr><td>1208 </td><td>44.8 </td><td>2731/11 431 </td><td>0.026 </td><td>140/539 </td></tr><tr><td>1248 </td><td>22.7 </td><td>2078/5232 </td><td>0.023 </td><td>127/470 </td></tr><tr><td>1366 </td><td>18.1 </td><td>1615/3264 </td><td>0.018 </td><td>119/440 </td></tr><tr><td>1397 </td><td>32.2 </td><td>2184/5327 </td><td>0.017 </td><td>137/491 </td></tr><tr><td>7505 </td><td>47.2 </td><td>3049/7571 </td><td>0.016 </td><td>177/510 </td></tr><tr><td>Total </td><td>258.9 </td><td>16 809/49 707 </td><td>0.019 </td><td>&lt;10<sup>−10</sup> </td><td>280/801 </td></tr><tr><td>1026 </td><td rowspan="9"><em>CONCEN</em> </td><td>29.8 </td><td>6174/22 787 </td><td>0.076 </td><td rowspan="9"> </td><td>108/225 </td></tr><tr><td>1101 </td><td>54.9 </td><td>6401/13 579 </td><td>0.024 </td><td>142/297 </td></tr><tr><td>1111 </td><td>26.7 </td><td>2904/18 633 </td><td>0.070 </td><td>137/289 </td></tr><tr><td>1220 </td><td>7.8 </td><td>929/3758 </td><td>0.048 </td><td>75/172 </td></tr><tr><td>1239 </td><td>42.2 </td><td>5296/19 310 </td><td>0.046 </td><td>138/276 </td></tr><tr><td>1348 </td><td>13.6 </td><td>1825/5055 </td><td>0.037 </td><td>102/222 </td></tr><tr><td>1396 </td><td>18.3 </td><td>1996/8497 </td><td>0.046 </td><td>124/289 </td></tr><tr><td>7429 </td><td>30.2 </td><td>3745/12 735 </td><td>0.042 </td><td>135/345 </td></tr><tr><td>Total </td><td>223.6 </td><td>29 270/104 354 </td><td>0.047 </td><td>250/574 </td></tr><tr><td>Grand total </td><td> </td><td>482.5 </td><td>46 079/154 061 </td><td>0.032 </td><td> </td><td>349/992 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Total number of paired reads (over 1 million) analysed after quality filtering.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of paired reads that both mapped onto at least one 16S gene in the database with a &gt;97% identity. A/B # of reads mapped onto 16S_Ref /16S_Merge (Methods).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>% of reads identifiable as 16S genes when both database and EMIRGE sequences are considered.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup><em>P</em>-value for the hypothesis test that the proportion of mappable 16S reads was the same for the forage and concentrate diets (for both databases 16S_Ref and 16S_Merge; see Results).</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Number of distinct OTUs observed for each sample: A/B: # of OTUs when considering 16S_Ref versus when considering 16S_Merge (Methods).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5934386" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST044TB1" class="table-wrap-title" data-id="DST044TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Mapping Illumina reads to 16S rDNA databases and OTU identification</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Diet </th><th>Million paired reads<sup>a</sup> </th><th>Individual 16S genes<sup>b</sup> </th><th>% of reads from 16S<sup>c</sup> </th><th><em>P</em><sup>d</sup> </th><th>Total OTUs<sup>e</sup> </th></tr></thead><tbody><tr><td>1003 </td><td rowspan="9"><em>FORG</em> </td><td>16.8 </td><td>860/2718 </td><td>0.016 </td><td rowspan="8"> </td><td>109/419 </td></tr><tr><td>1009 </td><td>35.9 </td><td>2548/8935 </td><td>0.025 </td><td>161/489 </td></tr><tr><td>1127 </td><td>41.1 </td><td>1744/5229 </td><td>0.013 </td><td>137/467 </td></tr><tr><td>1208 </td><td>44.8 </td><td>2731/11 431 </td><td>0.026 </td><td>140/539 </td></tr><tr><td>1248 </td><td>22.7 </td><td>2078/5232 </td><td>0.023 </td><td>127/470 </td></tr><tr><td>1366 </td><td>18.1 </td><td>1615/3264 </td><td>0.018 </td><td>119/440 </td></tr><tr><td>1397 </td><td>32.2 </td><td>2184/5327 </td><td>0.017 </td><td>137/491 </td></tr><tr><td>7505 </td><td>47.2 </td><td>3049/7571 </td><td>0.016 </td><td>177/510 </td></tr><tr><td>Total </td><td>258.9 </td><td>16 809/49 707 </td><td>0.019 </td><td>&lt;10<sup>−10</sup> </td><td>280/801 </td></tr><tr><td>1026 </td><td rowspan="9"><em>CONCEN</em> </td><td>29.8 </td><td>6174/22 787 </td><td>0.076 </td><td rowspan="9"> </td><td>108/225 </td></tr><tr><td>1101 </td><td>54.9 </td><td>6401/13 579 </td><td>0.024 </td><td>142/297 </td></tr><tr><td>1111 </td><td>26.7 </td><td>2904/18 633 </td><td>0.070 </td><td>137/289 </td></tr><tr><td>1220 </td><td>7.8 </td><td>929/3758 </td><td>0.048 </td><td>75/172 </td></tr><tr><td>1239 </td><td>42.2 </td><td>5296/19 310 </td><td>0.046 </td><td>138/276 </td></tr><tr><td>1348 </td><td>13.6 </td><td>1825/5055 </td><td>0.037 </td><td>102/222 </td></tr><tr><td>1396 </td><td>18.3 </td><td>1996/8497 </td><td>0.046 </td><td>124/289 </td></tr><tr><td>7429 </td><td>30.2 </td><td>3745/12 735 </td><td>0.042 </td><td>135/345 </td></tr><tr><td>Total </td><td>223.6 </td><td>29 270/104 354 </td><td>0.047 </td><td>250/574 </td></tr><tr><td>Grand total </td><td> </td><td>482.5 </td><td>46 079/154 061 </td><td>0.032 </td><td> </td><td>349/992 </td></tr></tbody></table></div><div content-id="DST044TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Diet </th><th>Million paired reads<sup>a</sup> </th><th>Individual 16S genes<sup>b</sup> </th><th>% of reads from 16S<sup>c</sup> </th><th><em>P</em><sup>d</sup> </th><th>Total OTUs<sup>e</sup> </th></tr></thead><tbody><tr><td>1003 </td><td rowspan="9"><em>FORG</em> </td><td>16.8 </td><td>860/2718 </td><td>0.016 </td><td rowspan="8"> </td><td>109/419 </td></tr><tr><td>1009 </td><td>35.9 </td><td>2548/8935 </td><td>0.025 </td><td>161/489 </td></tr><tr><td>1127 </td><td>41.1 </td><td>1744/5229 </td><td>0.013 </td><td>137/467 </td></tr><tr><td>1208 </td><td>44.8 </td><td>2731/11 431 </td><td>0.026 </td><td>140/539 </td></tr><tr><td>1248 </td><td>22.7 </td><td>2078/5232 </td><td>0.023 </td><td>127/470 </td></tr><tr><td>1366 </td><td>18.1 </td><td>1615/3264 </td><td>0.018 </td><td>119/440 </td></tr><tr><td>1397 </td><td>32.2 </td><td>2184/5327 </td><td>0.017 </td><td>137/491 </td></tr><tr><td>7505 </td><td>47.2 </td><td>3049/7571 </td><td>0.016 </td><td>177/510 </td></tr><tr><td>Total </td><td>258.9 </td><td>16 809/49 707 </td><td>0.019 </td><td>&lt;10<sup>−10</sup> </td><td>280/801 </td></tr><tr><td>1026 </td><td rowspan="9"><em>CONCEN</em> </td><td>29.8 </td><td>6174/22 787 </td><td>0.076 </td><td rowspan="9"> </td><td>108/225 </td></tr><tr><td>1101 </td><td>54.9 </td><td>6401/13 579 </td><td>0.024 </td><td>142/297 </td></tr><tr><td>1111 </td><td>26.7 </td><td>2904/18 633 </td><td>0.070 </td><td>137/289 </td></tr><tr><td>1220 </td><td>7.8 </td><td>929/3758 </td><td>0.048 </td><td>75/172 </td></tr><tr><td>1239 </td><td>42.2 </td><td>5296/19 310 </td><td>0.046 </td><td>138/276 </td></tr><tr><td>1348 </td><td>13.6 </td><td>1825/5055 </td><td>0.037 </td><td>102/222 </td></tr><tr><td>1396 </td><td>18.3 </td><td>1996/8497 </td><td>0.046 </td><td>124/289 </td></tr><tr><td>7429 </td><td>30.2 </td><td>3745/12 735 </td><td>0.042 </td><td>135/345 </td></tr><tr><td>Total </td><td>223.6 </td><td>29 270/104 354 </td><td>0.047 </td><td>250/574 </td></tr><tr><td>Grand total </td><td> </td><td>482.5 </td><td>46 079/154 061 </td><td>0.032 </td><td> </td><td>349/992 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Total number of paired reads (over 1 million) analysed after quality filtering.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of paired reads that both mapped onto at least one 16S gene in the database with a &gt;97% identity. A/B # of reads mapped onto 16S_Ref /16S_Merge (Methods).</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>% of reads identifiable as 16S genes when both database and EMIRGE sequences are considered.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup><em>P</em>-value for the hypothesis test that the proportion of mappable 16S reads was the same for the forage and concentrate diets (for both databases 16S_Ref and 16S_Merge; see Results).</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Number of distinct OTUs observed for each sample: A/B: # of OTUs when considering 16S_Ref versus when considering 16S_Merge (Methods).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5934386" target="_blank">View Large</a></div></div><p>There were 8472 and 9188 gene sequences from 16S_Ref and 16S_Merge found to match our reads, respectively. In each case, we performed single-linkage clustering using custom software. To do so, we first computed all possible pairwise global alignments between the genes using our new GPU-based global pairwise alignment package.<sup><a reveal-id="DST044C25" data-open="DST044C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DST044C26" data-open="DST044C26" class="link link-ref link-reveal xref-bibr">26</a></sup> We next created a graph where each node was a 16S rDNA gene. We defined edges between pairs of genes if their pairwise global sequence identity was ≥97%.<sup><a reveal-id="DST044C27" data-open="DST044C27" class="link link-ref link-reveal xref-bibr">27</a></sup> We then defined the OTUs to be the connected components in this graph.</p><p>Using in-house perl scripts, we mapped these OTUs back onto the reads, using each read pair's top hit to assign that pair to an OTU. We identified 349 OTUs using 16S_Ref and 992 OTUs with 16S_Merge (Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>). To test whether the percentage of reads mapped onto the rDNA database was the same for the two feed groups, we fit the number of reads mapped over the total number of reads to a binomial distribution, first requiring that proportion of reads mapped (<em>p</em>) be the same for both groups, then allowing <em>p</em> to differ between diets. Twice the difference in ln-likelihood for these models was compared with a <em>χ</em><sup>2</sup> distribution with one degree of freedom (e.g. a likelihood ratio test).<sup><a reveal-id="DST044C28" data-open="DST044C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=5934389 id="5934389" class="section-title" data-legacy-id=s2e>2.5. Phylum-level analysis</h3>
<p>Using the taxonomic names from the 16S_Ref database, we analysed the phyla-level distribution of our OTUs, mapping each prokaryotic taxa or genus name to the NCBI taxonomy database<sup><a reveal-id="DST044C29" data-open="DST044C29" class="link link-ref link-reveal xref-bibr">29</a></sup> to retrieve the corresponding phylum.</p>                            <h3 scrollto-destination=5934391 id="5934391" class="section-title" data-legacy-id=s2f>2.6. Statistical comparison of metagenome populations between individuals differing in diet</h3>
<div class="block-child-p">The metagenomic sequence data collected here are unusual in that similar environments have been sampled multiple times (e.g. sheep fed the same diet). We therefore require computational and statistical approaches able to statistically assess if the two diets induce a difference in microbe distribution. To detect any significant differences in microbial taxa (OTU) abundance between the animals with different feeds, we developed a partial statistical model, implemented in custom c++ programs. The input data for this model are the raw counts of OTU observations from each animal (Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>). However, because different numbers of total microbial individuals were sequenced for each animal host, it is not appropriate to directly compare these counts. Instead, the model is based on the underlying assumption that the relative abundances of the different OTUs in the rumen follow a multinomial distribution. In other words, OTUs <em>i</em> = 1…<em>n</em> each have a relative frequency <em>p<sub>i</sub></em> in the environment such that: <a id="DST044M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST044M1"><div class="tex-math display-math">$$\sum\limits_{i = 1}^n {\,p_{\rm i} = 1} $$</div></div><div class="label">(1)</div></div></div><div class="block-child-p">These <em>p<sub>i</sub>s</em> then give the probability that a single microbial individual drawn from that animal would come from OTU <em>i</em>. The probability of the observed bacterial OTU counts from an animal <em>j</em> (<em>D<sub>j</sub></em>) is then given by: <a id="DST044M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST044M2"><div class="tex-math display-math">$$P(D_j ) = \left( {\displaystyle{{n\hbox{!}} \over {\sum\limits_{i = 1}^n {x_i \hbox{!}} }}} \right) \cdot \prod\limits_{i = 1}^n {\,p_i^{x_i } } $$</div></div><div class="label">(2)</div></div></div><div class="block-child-p">where the <em>x<sub>i</sub>s</em> give the number of individuals observed from OTU <em>i</em>. The obvious difficulty with this model is that it has <em>n</em>− 1 unknown parameters (the <em>p<sub>i</sub>s</em>). With a sample of only 16 individuals, estimating so many unknowns is infeasible. Instead, we assumed that the rank-ordered values of the <em>p<sub>i</sub>s</em> followed one of two discrete probability distributions: a discrete power-law or a geometric distribution (for discussion of this assumption, see McGill <em>et al</em>.<sup><a reveal-id="DST044C30" data-open="DST044C30" class="link link-ref link-reveal xref-bibr">30</a></sup>, and Izsák and Pavoine<sup><a reveal-id="DST044C31" data-open="DST044C31" class="link link-ref link-reveal xref-bibr">31</a></sup>). Thus, we took the total number of microbial individuals from each OTU across all animals and sorted this sum across all OTUs. We then defined <em>p</em><sub>1</sub> as the proportion of all microbial individuals that belonged to the most abundant OTU, <em>p</em><sub>2</sub> as the proportion belonging to the next most abundant and so forth. In this framework, the two probability distributions define the relationships between <em>p</em><sub>1</sub>, <em>p</em><sub>2</sub>, <em>…</em>, <em>p<sub>n</sub></em>. Specifically, for the power-law distribution, the value of <em>p<sub>i</sub></em> for the <em>i</em>th most abundant OTU (across all animals) is given by: <a id="DST044M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST044M3"><div class="tex-math display-math">$$p_i = \displaystyle{{i^{ - a} } \over {\sum\limits_{\,j = 1}^\infty {\,j^{ - a} } }}$$</div></div><div class="label">(3)</div></div></div><div class="block-child-p">where <em>a</em> is a parameter estimated from data (see below). Similarly, under the geometric distribution, the <em>p<sub>i</sub></em> for the <em>i</em>th most abundant OTU is: <a id="DST044M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DST044M4"><div class="tex-math display-math">$$p_i = \pi \cdot (1 - \pi )^{i - 1} $$</div></div><div class="label">(4)</div></div></div><p>Where <em>π</em> is a parameter to be estimated. Thus, in both cases, we have reduced the problem from estimating <em>n</em> − 1 parameters to estimating one parameter. To do so, we fit the observed OTU counts to these models by maximum-likelihood. The likelihood of an entire sample of animals <em>L</em> is then given by the product of the <em>D<sub>j</sub>s</em> from (2). We estimate <em>a</em> or <em>π</em> using numerical optimization to find the value that maximizes <em>L.</em><sup><a reveal-id="DST044C32" data-open="DST044C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Now that the data have been placed into a modelling framework, we can use the models to ask if different samples follow different multinomial distributions. To test for differences between the samples due to diet, we adopted a partitioning and randomization approach. First, we divided the OTU distributions into the two dietary groups: <em>FORG</em> and <em>CONCEN</em> described above. We then individually calculated ln(<em>L</em><sub>F</sub>) for <em>FORG</em> and ln(<em>L</em><sub>C</sub>) for <em>CONCEN</em> and computed <em>D</em><em>=</em> [ln(<em>L</em><sub>F</sub>) + ln(<em>L</em><sub>C</sub>)]<em>–</em>ln(<em>L</em>). Note that <em>FORG</em> and <em>CONCEN</em> differ from the full dataset potentially in both the rankings of the 349 or 992 OTUs and the value of <em>a</em> or <em>π</em>. Thus, <em>D</em> is a measure of how much samples <em>FORG</em> and <em>CONCEN</em> differ. To assess if the observed difference would be expected by chance, we randomly repartitioned the full dataset <em>A</em> into samples of the same size as <em>CONCEN</em> and <em>FORG</em> 1000 times. For each such randomization, we calculated the value of <em>D</em><sub>rand</sub>. If <em>D</em> for the real dataset is exceeded by not more than 5% of the values of <em>D</em><sub>rand</sub>, we can statistically conclude that there is sufficient evidence to reject the null hypothesis of the same species distribution in <em>CONCEN</em> and <em>FORG</em>.</p>                            <h3 scrollto-destination=5934398 id="5934398" class="section-title" data-legacy-id=s2g>2.7. Identifying OTU-level differences between feeds</h3>
<p>The above approach only indicates whether or not the two feed groups are statistically distinguishable. It cannot describe the particular OTUs that drive this difference. In order to do so, we slightly modified our model to consist of three distinct multinomial distributions of the form of (1): <em>M</em><sub>S</sub>, <em>M</em><sub>F</sub>, and <em>M</em><sub>C</sub>. Each distribution has its own value of <em>a</em> or <em>π.</em> Among the <em>n</em> OTUs, each can either be assigned to the shared distribution (<em>M</em><sub>S</sub>) or to the distinct distributions (<em>M</em><sub>F</sub> and <em>M</em><sub>C</sub>): this assignment is coded as a binary vector <span class="inline-formula no-formula-id">|$\overline S $|</span> of length <em>n</em>. The likelihood of a sample is then the product of the likelihood of the shared OTUs (<em>s<sub>i</sub></em> = 0) under <em>M</em><sub>S</sub> and the distinct OTUs (<em>s<sub>i</sub></em> = 1) under either <em>M</em><sub>F</sub> or <em>M</em><sub>C</sub>, depending on the feed treatment for that sample. There are 2<em><sup>n</sup></em> possible values of <span class="inline-formula no-formula-id">|$\overline S $|⁠</span>, and we used our previously described simulated annealing software to search for the combination of the entries of<span class="inline-formula no-formula-id">|$\overline S $|</span> and the values of the three <em>a</em>’s or <em>π</em>'s that give the maximum likelihood of observing the data collected.<sup><a reveal-id="DST044C33" data-open="DST044C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We also compared the proportion of individuals who were members of the Methanobacteria group between the two feeds using the same binomial model used to test the read-mapping proportion.</p>                            <h2 scrollto-destination=5934400 id="5934400" class="section-title" data-legacy-id=s3>3. Results</h2>
<p>Using Illumina sequencing, we obtained &gt;480 million paired-end reads from the rumen metagenomes of 16 sheep. We used two strategies for analysing the microbial taxonomic diversity present in these animals. First, by mapping the reads to known 16S rDNA genes (16S_Ref, Methods), we identified 349 known prokaryotic OTUs present in at least one of our 16 animals (Methods; Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>). Secondly, by using the EMIRGE package,<sup><a reveal-id="DST044C21" data-open="DST044C21" class="link link-ref link-reveal xref-bibr">21</a></sup> we assembled probabilistic consensus sequences for new 16S rDNA genes (16S_Merge), resulting in between a 2- and 4-fold increase in the number of reads identified as coming from 16S rDNA genes and roughly a 3-fold increase in the number of OTUs seen (Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>In keeping with EMIRGE's described function of identifying new 16S rDNA sequences, &lt;2% of the OTUs derived from EMIRGE 16S rDNA assemblies also included sequences from the existing database, strongly suggesting the presence of many unknown taxa in these samples.</p><p>When considering gross, phylum-level differences between the animals in known taxa (16S_Ref), there is a clear distinction between the two feed conditions (Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>A). Interestingly, the proportion of Illumina reads mapped onto 16S_Merge was roughly 2-fold higher among the concentrate-fed animals (1 in 2100 versus 1 in 5200), a significant difference (<em>P</em> &lt; 10<sup>−10</sup>, likelihood ratio test, Methods). This bias is not attributable to an overall lower efficiency in obtaining DNA from these animals, as the raw number of reads obtained for each group is comparable (Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="5934404" scrollto-destination="5934404"></a>
<div id="DST044F1" data-id="DST044F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/m_dst04401.png?Expires=2147483647&Signature=x8HM3ahnzBrk0MFt-df3h1Z2aLi9I6w7dpKBgaRjQTApUJDpx6og1n~fZjlmKQKQ8JBa-9Garp3k1Zs0QoJSLy~b4DDnYIiu8DxuH8WKUvyNgEG50b-rk3VxoQpof2TFCCG2Ypt-zrbZJ7ptlnRut~c2QAOttiooOJiBf2vRXZ6JoJBMUeXPvxOibm0YijrJ5OwLAi5IKytlnSQbDPPreRIRPjgajmFDWWU09uizCnh463eeKvJ8jrcGW0V0IzwBv67uKQclUZWjE7G7FB~BBYufXMSp3kOvPKG1aHSN0aE44D~pmw4G6gR~P7ATTNJ9lyeyMdOzQpLIE4v32Xe8xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microbial diversity in forage- and concentrate-fed animals. (A) Phylum-level breakdown of the microbial diversity, showing the top seven detected phyla for genes drawn from the 16S_Ref database (Methods). While there is considerable variation among individuals, there are clear differences between the two diets. Because all archaeans seen were from the Class Methanobacteria, this name is indicated. (B) Models of the species abundance curves for the forage diet (FORG), including all OTUs (e.g. 16S_Merge; Methods). On the x-axis is the rank abundance of each OTU: the most abundant OTU is rank 1 and so forth. On the y-axis is the proportion of the total sample for that individual that rank makes up. We fit two statistical distributions to these data: a discrete power-law (purple) and a geometric (green; Methods). For this diet, the geometric distribution provides a better fit (ln-likelihood of −278 825 versus −284 582 for the power-law distribution). (C) As in (B), but for the concentrate-fed animals (CONCEN). Here, the power-law distribution is a better statistical fit (ln-likelihood of −416 927 and −380 366 for the geometric and power-law distributions, respectively)." path-from-xml="dst04401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5934404/dst04401.tiff" path-from-xml="dst04401.tiff" target="_blank">View large</a><a section="5934404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/dst04401.png?Expires=2147483647&Signature=OWMi-HDpKzMoPYD-SzdOnc9bus7~dEihSwonboQ1pAicFBiVjKek1FfEdHkEWCGwE6hhlI51bpem460-Ip~Pi6EV2~S5ismr5tf8Z7cMd~If1VmSe4yDowj3dbsfORAHN7yDKpelP~E~HiVAuXzuHdgfkgIukv4bG-nWsj3EnBvDOAolpsEI483EPox2gSfaC95xK43RSeoViq~tqpgBIyCA9aLDSFNbN13~E2icSAdeOu~6aV-XIgCef1ap~mGUaHineccBTfL-C-MsJlM6VRcpjXyydLmK33SWgGNB1ybc4P5MgyQavdeaiH4R3RgG0vyGI8fwRnrxSwn5u85cMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5934404&ar=401903&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microbial diversity in forage- and concentrate-fed animals. (A) Phylum-level breakdown of the microbial diversity, showing the top seven detected phyla for genes drawn from the 16S_Ref database (Methods). While there is considerable variation among individuals, there are clear differences between the two diets. Because all archaeans seen were from the Class Methanobacteria, this name is indicated. (B) Models of the species abundance curves for the forage diet (<em>FORG</em>), including all OTUs (e.g. 16S_Merge; Methods). On the <em>x</em>-axis is the rank abundance of each OTU: the most abundant OTU is rank 1 and so forth. On the <em>y</em>-axis is the proportion of the total sample for that individual that rank makes up. We fit two statistical distributions to these data: a discrete power-law (purple) and a geometric (green; Methods). For this diet, the geometric distribution provides a better fit (ln-likelihood of −278 825 versus −284 582 for the power-law distribution). (C) As in (B), but for the concentrate-fed animals (<em>CONCEN</em>). Here, the power-law distribution is a better statistical fit (ln-likelihood of −416 927 and −380 366 for the geometric and power-law distributions, respectively).</p></div></div><div content-id="DST044F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/m_dst04401.png?Expires=2147483647&Signature=x8HM3ahnzBrk0MFt-df3h1Z2aLi9I6w7dpKBgaRjQTApUJDpx6og1n~fZjlmKQKQ8JBa-9Garp3k1Zs0QoJSLy~b4DDnYIiu8DxuH8WKUvyNgEG50b-rk3VxoQpof2TFCCG2Ypt-zrbZJ7ptlnRut~c2QAOttiooOJiBf2vRXZ6JoJBMUeXPvxOibm0YijrJ5OwLAi5IKytlnSQbDPPreRIRPjgajmFDWWU09uizCnh463eeKvJ8jrcGW0V0IzwBv67uKQclUZWjE7G7FB~BBYufXMSp3kOvPKG1aHSN0aE44D~pmw4G6gR~P7ATTNJ9lyeyMdOzQpLIE4v32Xe8xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Microbial diversity in forage- and concentrate-fed animals. (A) Phylum-level breakdown of the microbial diversity, showing the top seven detected phyla for genes drawn from the 16S_Ref database (Methods). While there is considerable variation among individuals, there are clear differences between the two diets. Because all archaeans seen were from the Class Methanobacteria, this name is indicated. (B) Models of the species abundance curves for the forage diet (FORG), including all OTUs (e.g. 16S_Merge; Methods). On the x-axis is the rank abundance of each OTU: the most abundant OTU is rank 1 and so forth. On the y-axis is the proportion of the total sample for that individual that rank makes up. We fit two statistical distributions to these data: a discrete power-law (purple) and a geometric (green; Methods). For this diet, the geometric distribution provides a better fit (ln-likelihood of −278 825 versus −284 582 for the power-law distribution). (C) As in (B), but for the concentrate-fed animals (CONCEN). Here, the power-law distribution is a better statistical fit (ln-likelihood of −416 927 and −380 366 for the geometric and power-law distributions, respectively)." path-from-xml="dst04401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5934404/dst04401.tiff" path-from-xml="dst04401.tiff" target="_blank">View large</a><a section="5934404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/dst04401.png?Expires=2147483647&Signature=OWMi-HDpKzMoPYD-SzdOnc9bus7~dEihSwonboQ1pAicFBiVjKek1FfEdHkEWCGwE6hhlI51bpem460-Ip~Pi6EV2~S5ismr5tf8Z7cMd~If1VmSe4yDowj3dbsfORAHN7yDKpelP~E~HiVAuXzuHdgfkgIukv4bG-nWsj3EnBvDOAolpsEI483EPox2gSfaC95xK43RSeoViq~tqpgBIyCA9aLDSFNbN13~E2icSAdeOu~6aV-XIgCef1ap~mGUaHineccBTfL-C-MsJlM6VRcpjXyydLmK33SWgGNB1ybc4P5MgyQavdeaiH4R3RgG0vyGI8fwRnrxSwn5u85cMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5934404&ar=401903&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Microbial diversity in forage- and concentrate-fed animals. (A) Phylum-level breakdown of the microbial diversity, showing the top seven detected phyla for genes drawn from the 16S_Ref database (Methods). While there is considerable variation among individuals, there are clear differences between the two diets. Because all archaeans seen were from the Class Methanobacteria, this name is indicated. (B) Models of the species abundance curves for the forage diet (<em>FORG</em>), including all OTUs (e.g. 16S_Merge; Methods). On the <em>x</em>-axis is the rank abundance of each OTU: the most abundant OTU is rank 1 and so forth. On the <em>y</em>-axis is the proportion of the total sample for that individual that rank makes up. We fit two statistical distributions to these data: a discrete power-law (purple) and a geometric (green; Methods). For this diet, the geometric distribution provides a better fit (ln-likelihood of −278 825 versus −284 582 for the power-law distribution). (C) As in (B), but for the concentrate-fed animals (<em>CONCEN</em>). Here, the power-law distribution is a better statistical fit (ln-likelihood of −416 927 and −380 366 for the geometric and power-law distributions, respectively).</p></div></div><p>Methane production is a topic of considerable current interest,<sup><a reveal-id="DST044C34" data-open="DST044C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and consequently, we sought to assess if the abundance of methanogenic microbes differed between the two diet groups. As can be seen from Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>A, there is considerable variation in the proportion of archaeans among the samples. All of these individuals were derived from one class among the Euryarchaeota, namely the Methanobacteria: they are indicated in pink in Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>A. Nevertheless, on average, there are significantly more such microbes in animals administered a forage diet (<em>P</em> &lt; 10<sup>−10</sup>; likelihood ratio test), a fact potentially related to the lowering of rumen pH under concentrate-type diets.<sup><a reveal-id="DST044C35" data-open="DST044C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>To explore these differences in a rigorous statistical manner, we examined the relative abundance differences between samples (Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B and C). To assess whether there were systematic differences in the OTU abundances depending on feed source, we fit maximum-likelihood models of species abundances to our 16 samples under both an assumed power-law and geometric distribution of rank abundances (Methods). We first asked if the animals fed concentrate diets showed differing OTU distributions than did those fed forage diets. For both the power-law and geometric models, there was a significant improvement in fit by allowing the two feed groups to have their own multinomial distributions (<em>P</em> &lt; 0.001 using either 16S_Ref or 16S_Merge).</p><p>This observed improvement in fit could result from a range of circumstances, from a large difference in abundance for a few OTUs to nearly non-overlapping OTU for the two treatments. Therefore, to understand the source of these differences, we applied a partitioning model that broke the OTUs down into two groups, one for which abundance was similar in both treatments and one for which each treatment had an independent abundance rank for that OTU (Methods). This approach is most appropriate when the OTUs analysed can be mapped to known taxa, and so we applied it to the OTUs found with 16S_Ref. We sought the maximum-likelihood arrangement of OTUs into these two groups. The two treatments are generally different in their most abundant OTUs (Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>C and D: c.f. to A), with a group of more rarely observed OTUs with similar (low) abundances between the two treatments (Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>B). </p>                        <a id="5934408" scrollto-destination="5934408"></a>
<div id="DST044F2" data-id="DST044F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/m_dst04402.png?Expires=2147483647&Signature=Fa0PWKg1iyMh8K8TOHraxGHbOGq78QQgueDLQdG3-VDgzRmpdE5CpSuDv5lHeyeWY8XRYbgghhkfGqq~URroTdVNf4Bq5MAiPZQ2W~-RiubQSa34Abmo0ZuQZeCRwjqd5DBpZaTsd~ZYwfpU6NENH53bs~ctQ-kCe6wSjmDo9ZGRzE3u4MoclpYJ4CAyf4-3Opd2UmJv4EceQ2Kno-7K7OeeQbriPnoxNIFNbhcDuw~LENY51BEK28zepjYgtkBHAsoE9X9fsUPZkpahFlEm925XtddnzTTuJKDYhVjRAzvCq9oRcKocWijpLfzhvrl2lG9XJn4i7zlYhGbQHcq27A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distinct sets of high-abundance taxa between forage- and concentrate-fed animals are overlaid on a common core of rare organisms. For each panel, the x-axis gives the rank of each OTU (according the scheme for that panel), whereas the y-axis is the frequency of that OTU in a particular animal. Unlike Fig. 1B, here only genes matching to 16S_Ref are included (Methods). (A) The OTU distribution seen when all animals' OTU frequencies are plotted against the average OTU abundance across all 16 animals. The predicted abundance curves from our power-law and geometric distributions provide a visually very poor fit to the data, with obvious differences in abundance between the two feed groups (red, forage and blue, concentrate). (B–D) A machine-learning approach was applied to partition the set of 349 OTUs into either a ‘shared’ group common to both feeds or a feed-specific group (Methods). Generally speaking, this approach placed the abundant taxa into feed-specific groups (C: forage-feed animals, FORG; D: concentrate-fed animals, CONCEN), while there was a set of low-abundance microbes that did not appear to differ between the feeds. Thus, in C and D, the OTUs are individually ranked for the forage- and concentrate-fed animals, while in B, both groups share a common ranking. Note that, unlike Fig. 1, this partitioning approach yields curves that visually match the geometric distribution well. Representative taxa names are given above abundant organisms for reference." path-from-xml="dst04402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5934408/dst04402.tiff" path-from-xml="dst04402.tiff" target="_blank">View large</a><a section="5934408" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/dst04402.png?Expires=2147483647&Signature=UG74XjhUfVGcMyKdg-bdlDyyaGaLezHU2FI3kwILqo3Lj-ikeaFW2Xs3MdhMaZFMLE9xMXDuZw5u1NdqFPWFRCHQwPcmUZQgV8n1u81VWh0Sx08pu3GDCCJREMmWJavSR~W6Avm--hf8cYxLpOG~CaYTOXg5RcoIiAV16Srk8fQESTJQrlu5VpMfhHFZoWvpaSs2gLnoNEubYAN~K~BfDSO7-~ZoNmvW1rlZlRDnhBjcJ~xkiwTdaG4ppGpy96nOpmuNGB64ISapeROUfV4G9q~i4NcZVVC81K9lpo~DySJsmjBOz0Q3Qv-AyJMhpOEtV1xM9vUNmrp4Gf7Cmnx1JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5934408&ar=401903&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distinct sets of high-abundance taxa between forage- and concentrate-fed animals are overlaid on a common core of rare organisms. For each panel, the <em>x</em>-axis gives the rank of each OTU (according the scheme for that panel), whereas the <em>y</em>-axis is the frequency of that OTU in a particular animal. Unlike Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B, here only genes matching to 16S_Ref are included (Methods). (A) The OTU distribution seen when all animals' OTU frequencies are plotted against the average OTU abundance across all 16 animals. The predicted abundance curves from our power-law and geometric distributions provide a visually very poor fit to the data, with obvious differences in abundance between the two feed groups (red, forage and blue, concentrate). (B–D) A machine-learning approach was applied to partition the set of 349 OTUs into either a ‘shared’ group common to both feeds or a feed-specific group (Methods). Generally speaking, this approach placed the abundant taxa into feed-specific groups (C: forage-feed animals, <em>FORG</em>; D: concentrate-fed animals, <em>CONCEN</em>), while there was a set of low-abundance microbes that did not appear to differ between the feeds. Thus, in C and D, the OTUs are individually ranked for the forage- and concentrate-fed animals, while in B, both groups share a common ranking. Note that, unlike Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>, this partitioning approach yields curves that visually match the geometric distribution well. Representative taxa names are given above abundant organisms for reference.</p></div></div><div content-id="DST044F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/m_dst04402.png?Expires=2147483647&Signature=Fa0PWKg1iyMh8K8TOHraxGHbOGq78QQgueDLQdG3-VDgzRmpdE5CpSuDv5lHeyeWY8XRYbgghhkfGqq~URroTdVNf4Bq5MAiPZQ2W~-RiubQSa34Abmo0ZuQZeCRwjqd5DBpZaTsd~ZYwfpU6NENH53bs~ctQ-kCe6wSjmDo9ZGRzE3u4MoclpYJ4CAyf4-3Opd2UmJv4EceQ2Kno-7K7OeeQbriPnoxNIFNbhcDuw~LENY51BEK28zepjYgtkBHAsoE9X9fsUPZkpahFlEm925XtddnzTTuJKDYhVjRAzvCq9oRcKocWijpLfzhvrl2lG9XJn4i7zlYhGbQHcq27A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distinct sets of high-abundance taxa between forage- and concentrate-fed animals are overlaid on a common core of rare organisms. For each panel, the x-axis gives the rank of each OTU (according the scheme for that panel), whereas the y-axis is the frequency of that OTU in a particular animal. Unlike Fig. 1B, here only genes matching to 16S_Ref are included (Methods). (A) The OTU distribution seen when all animals' OTU frequencies are plotted against the average OTU abundance across all 16 animals. The predicted abundance curves from our power-law and geometric distributions provide a visually very poor fit to the data, with obvious differences in abundance between the two feed groups (red, forage and blue, concentrate). (B–D) A machine-learning approach was applied to partition the set of 349 OTUs into either a ‘shared’ group common to both feeds or a feed-specific group (Methods). Generally speaking, this approach placed the abundant taxa into feed-specific groups (C: forage-feed animals, FORG; D: concentrate-fed animals, CONCEN), while there was a set of low-abundance microbes that did not appear to differ between the feeds. Thus, in C and D, the OTUs are individually ranked for the forage- and concentrate-fed animals, while in B, both groups share a common ranking. Note that, unlike Fig. 1, this partitioning approach yields curves that visually match the geometric distribution well. Representative taxa names are given above abundant organisms for reference." path-from-xml="dst04402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5934408/dst04402.tiff" path-from-xml="dst04402.tiff" target="_blank">View large</a><a section="5934408" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/dst04402.png?Expires=2147483647&Signature=UG74XjhUfVGcMyKdg-bdlDyyaGaLezHU2FI3kwILqo3Lj-ikeaFW2Xs3MdhMaZFMLE9xMXDuZw5u1NdqFPWFRCHQwPcmUZQgV8n1u81VWh0Sx08pu3GDCCJREMmWJavSR~W6Avm--hf8cYxLpOG~CaYTOXg5RcoIiAV16Srk8fQESTJQrlu5VpMfhHFZoWvpaSs2gLnoNEubYAN~K~BfDSO7-~ZoNmvW1rlZlRDnhBjcJ~xkiwTdaG4ppGpy96nOpmuNGB64ISapeROUfV4G9q~i4NcZVVC81K9lpo~DySJsmjBOz0Q3Qv-AyJMhpOEtV1xM9vUNmrp4Gf7Cmnx1JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5934408&ar=401903&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distinct sets of high-abundance taxa between forage- and concentrate-fed animals are overlaid on a common core of rare organisms. For each panel, the <em>x</em>-axis gives the rank of each OTU (according the scheme for that panel), whereas the <em>y</em>-axis is the frequency of that OTU in a particular animal. Unlike Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B, here only genes matching to 16S_Ref are included (Methods). (A) The OTU distribution seen when all animals' OTU frequencies are plotted against the average OTU abundance across all 16 animals. The predicted abundance curves from our power-law and geometric distributions provide a visually very poor fit to the data, with obvious differences in abundance between the two feed groups (red, forage and blue, concentrate). (B–D) A machine-learning approach was applied to partition the set of 349 OTUs into either a ‘shared’ group common to both feeds or a feed-specific group (Methods). Generally speaking, this approach placed the abundant taxa into feed-specific groups (C: forage-feed animals, <em>FORG</em>; D: concentrate-fed animals, <em>CONCEN</em>), while there was a set of low-abundance microbes that did not appear to differ between the feeds. Thus, in C and D, the OTUs are individually ranked for the forage- and concentrate-fed animals, while in B, both groups share a common ranking. Note that, unlike Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>, this partitioning approach yields curves that visually match the geometric distribution well. Representative taxa names are given above abundant organisms for reference.</p></div></div>                            <h2 scrollto-destination=5934409 id="5934409" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5934410 id="5934410" class="section-title" data-legacy-id=s4a>4.1. High diversity in sheep ruminal metagenomes, with strong distinctions due to diet</h3>
<p>We highlight two key findings from our analyses of rumen metagenomic DNA from sheep. First, there is evidence for a large number of currently unclassified microbes in this environment. EMIRGE predicted a number of new 16S rDNA sequences that do not cluster with existing sequences in the 16S database, and these sequences represent the majority of the 16S rDNA reads identified. Secondly, there are large differences in microbial distributions between the two diets examined, regardless of the 16S database used (16S_Merge, Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B and C and 16S_Ref, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=5934412 id="5934412" class="section-title" data-legacy-id=s4b>4.2. Comparing microbial diversity across individual animals</h3>
<p>Many discussions of the rumen microbial community quantify the complexity of the microbial community in terms of the number of species or OTUs.<sup><a reveal-id="DST044C9" data-open="DST044C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST044C14" data-open="DST044C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST044C17" data-open="DST044C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST044C18" data-open="DST044C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Here, we have chosen not to use that metric for several reasons. First, and trivially, the highly skewed distributions of the form of Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a> suggest that while there may be a large number of lowly abundant taxa, it seems unreasonable to believe that the major differences between animals or diets result from these rare individuals. Secondly, most communities are described by two inter-related parameters, the richness (related to the number of taxa present) and the evenness (describing those taxa's relative abundance). Species abundance curves link these two concepts with a probability distribution, allowing fair comparisons between samples.<sup><a reveal-id="DST044C36" data-open="DST044C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Finally, we believe that the methods used to define OTUs in metagenomic contexts are unstable relative to sample size. We, like other researchers, have defined OTUs based on a 97% or greater sequence identity in the 16S rDNA gene. While this approach is sensible, it rests on an implicit network clustering approach whereby sequences are first linked by sequence identity, followed by a clustering step that defines connected components in a graph and hence OTUs (see Methods). However, adding sequences increases the chance of a new sequence bridging two previously separate OTUs. Thus, we expect that larger samples, while increasing the OTU count with new taxa, will also tend to compress that count through OTU merging. This effect is unlikely to have serious consequences in most cases, but it does mean that the OTU counts for different studies should not be directly compared. Our results are also unusual in that, because of our Illumina-based approach, we clustered not the sequence data but rather the ∼9000 database sequences that those reads matched to (Methods). As a result, our OTU estimates should not be compared with PCR-based analyses.<sup><a reveal-id="DST044C17" data-open="DST044C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST044C18" data-open="DST044C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=5934414 id="5934414" class="section-title" data-legacy-id=s4c>4.3. Caveats</h3>
<p>Our Illumina sequencing-based approach has different biases than do culture or PCR-based methods. Our read-mapping strategy precludes the identification of taxa with 16S genes &lt;97% identical to known samples. This limitation is likely the reason that, although we had similar numbers of sequence reads for the two diets, the number of identified 16S genes was lower in the forage-fed group (Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>). Likewise, because we did not sequence entire 16S genes, it is possible that certain OTUs might contain individuals who, while having 97% identity in some regions of the gene, are more dissimilar in other regions. Fortunately, this bias is constant across our samples. Another issue with all 16S-based approaches is that 16S copy number is taken as a proxy for microbe abundance, even though 16S copy number is not constant across genomes. Again, this effect should not bias our analyses, because it influences them all equally. Finally, the EMRIGE approach, while powerful, has a few shortcomings. First, the sequences inferred do not necessarily represent particular microbes from the sample, but are rather consensus inferences. It is therefore potentially dangerous to try to place them in a phylogenetic context. Moreover, the EMIRGE pipeline requires known 16S rDNA sequences as input: there still may be highly diverged 16S rDNA that have been missed.</p><p>Our results differ in detail somewhat from a previous analysis of the bacterial composition of both forage and concentrate-fed sheep that focused on the genus <em>Prevotella.</em><sup><a reveal-id="DST044C37" data-open="DST044C37" class="link link-ref link-reveal xref-bibr">37</a></sup> These authors found a higher percentage of <em>Prevotella</em> individuals in concentrate-fed animals, in contrast to our results finding that <em>Prevotella</em> was the dominant genus in forage-, but not concentrate-, fed animals (Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>). Given the very different methods employed, it is difficult to know what to make of this difference. While the majority of the <em>Prevotella</em> found in an earlier bovine survey were from taxa not in 16S_Ref,<sup><a reveal-id="DST044C38" data-open="DST044C38" class="link link-ref link-reveal xref-bibr">38</a></sup> they are unlikely to represent the most common OTUs here, since none of the five most abundant 16S rDNA sequences produced by EMIRGE had <em>Prevotella</em> as the strongest BLAST hit (data not shown). We note, however, that the general conclusion in both cases was that there was a greater diversity in the forage group.<sup><a reveal-id="DST044C37" data-open="DST044C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=5934417 id="5934417" class="section-title" data-legacy-id=s4d>4.4. Diet-based differences in highly abundant microbes derived from a common core of taxa</h3>
<p>As an alternative to the OTU counting approach mentioned above, we have described microbial diversity in terms of simple mathematical models (Figs <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>). One apparent trend is the presence of a universal rare ‘core’ of organisms present in both groups (Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>B). It is possible that this core is the result insufficient statistical power in our model. However, inspection of Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a> shows some taxa with clear separation between the feeds (e.g. <em>Prevotella ruminicola</em> and <em>Dialister succinatiphilus</em> in Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>A) and others with overlapping distributions (e.g. <em>Selenomonas bovis</em> in Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>B). Instead, we suggest that another possibility is that a relatively large number of new microbial individuals enter the rumen, a suggestion supported by the observation that there are almost no OTUs of high abundance in one animal that are not at least found occasionally in all the other animals. Indeed, in only two microbial groups (<em>Parascardovia denticolens</em> and <em>Allisonella histaminiformans</em>) were 100 or more microbial individuals present in one feed group, with no individuals being present in the other. Thus, under this common inputs hypothesis, the observed differences are not a result of differences in microbes entering the system, but rather in the niches available to them when they arrive.</p><p>In support of this idea of reasonably high microbe turnover is the fact that the two diets differ not only in the OTUs present, but also in the nature of the taxa abundance curves. When the diets are treated separately (Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>) and all 16S rDNA sequences are used, the microbial ecosystem induced by the forage diet is clearly more diverse than that induced by the concentrate diet (a ‘flatter’ power-law curve in Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B for the forage diet versus Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>C and the concentrate-fed animals). This result may appear to contradict the data of Fig. <a reveal-id="DST044F2" data-open="DST044F2" class="link link-reveal link-table xref-fig">2</a>C and D, where the forage diet has a rumen community that is dominated by a single OTU (<em>P. ruminicola</em>). However, we believe that this apparent discrepancy results from the fact that the reference database used in that figure (e.g. 16S_Ref) more poorly represents the highest abundance taxa from the forage environment than from the concentrate-induced one. Thus, the slope seen in Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>B implies that the forage diet has a greater diversity of rare OTUs relative to Fig. <a reveal-id="DST044F1" data-open="DST044F1" class="link link-reveal link-table xref-fig">1</a>C. This fact can be observed in Table <a reveal-id="DST044TB1" data-open="DST044TB1" class="link link-reveal link-table xref-fig">1</a>, where the ‘long-tailed’ distribution of abundances means that there are more total OTUs observed among the forage-fed animals, despite these animals having many fewer total individuals.</p><p>The ecological literature on species richness (the number of OTUs present in our case) and species evenness (whether the numbers of individuals of those species are present in relatively equal numbers) is considerable.<sup><a reveal-id="DST044C39  DST044C41" data-open="DST044C39  DST044C41" class="link link-ref link-reveal xref-bibr">39–41</a></sup> However, the exact role of species evenness, in particular, is complex and incompletely understood.<sup><a reveal-id="DST044C39" data-open="DST044C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Under some conditions, such as a constant environment, dominance by a few taxa may increase productivity.<sup><a reveal-id="DST044C42" data-open="DST044C42" class="link link-ref link-reveal xref-bibr">42</a></sup> However, if the environment is more complex (e.g. certain local regions are more suitable to different taxa, or the environment changes in time), greater evenness of taxa abundance (less dominance) will improve productivity.<sup><a reveal-id="DST044C42" data-open="DST044C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DST044C43" data-open="DST044C43" class="link link-ref link-reveal xref-bibr">43</a></sup> One can make a plausible argument that the variety and complexity of the nutrients in a forage diet are greater, yielding greater evenness in the OTU abundances. On the other hand, the rumen is a system that has adapted over a long period for forage-like diets, and the differences seen might also be due to this fact. It would be most helpful to develop theories and tests able to distinguish between these two hypotheses.</p>                            <h2 scrollto-destination=5934421 id="5934421" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5934423 id="5934423" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the USDA National Research Initiative (NRI) (grant 2011-68006-30185).</p>                            <h2 scrollto-destination=5934425 id="5934425" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank R. Schnabel for insights into analysing high-throughput sequence data, D. Li and K. Sajjapongse for assistance with GPU analyses of 16S rDNA genes, and M. Kerley and J. Taylor for helpful discussions.</p>                            <h2 scrollto-destination=5934427 id="5934427" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST044C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tringe</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">E.M.</div></div>. </div><div class="article-title">Metagenomics: DNA sequencing of environmental samples</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">805</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20DNA%20sequencing%20of%20environmental%20samples&amp;author=S.G.%20Tringe&amp;author=E.M.%20Rubin&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=805-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16304596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackie</div> <div class="given-names">R.I.</div></div>. </div><div class="article-title">Mutualistic fermentative digestion in the gastrointestinal tract: diversity and evolution</div>, <div class="source">Integr. Comput. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">319</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutualistic%20fermentative%20digestion%20in%20the%20gastrointestinal%20tract%3A%20diversity%20and%20evolution&amp;author=R.I.%20Mackie&amp;publication_year=2002&amp;journal=Integr.%20Comput.%20Biol.&amp;volume=42&amp;pages=319-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/icb/42.2.319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ficb%2F42.2.319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ficb%2F42.2.319"> </span></div> </div></div></div></div></div><div content-id="DST044C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Lozupone</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Worlds within worlds: evolution of the vertebrate gut microbiota</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">776</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worlds%20within%20worlds%3A%20evolution%20of%20the%20vertebrate%20gut%20microbiota&amp;author=R.E.%20Ley&amp;author=C.A.%20Lozupone&amp;author=M.%20Hamady&amp;author=R.%20Knight&amp;author=J.I.%20Gordon&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=6&amp;pages=776-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18794915" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lozupone</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Evolution of mammals and their gut microbes</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">320</div> (pg. <div class="fpage">1647</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20mammals%20and%20their%20gut%20microbes&amp;author=R.E.%20Ley&amp;author=M.%20Hamady&amp;author=C.%20Lozupone&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1647-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1155725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1155725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1155725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18497261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="collab">Human Microbiome Project Consortium</div><div class="article-title">Structure, function and diversity of the healthy human microbiome</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">486</div> (pg. <div class="fpage">207</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Backhed</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Lozupone</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Obesity alters gut microbial ecology</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">11070</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Obesity%20alters%20gut%20microbial%20ecology&amp;author=R.E.%20Ley&amp;author=F.%20Backhed&amp;author=P.%20Turnbaugh&amp;author=C.A.%20Lozupone&amp;author=R.D.%20Knight&amp;author=J.I.%20Gordon&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=11070-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0504978102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0504978102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0504978102"> </span></div> </div></div></div></div></div><div content-id="DST044C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Ley</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Mahowald</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Magrini</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Mardis</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">An obesity-associated gut microbiome with increased capacity for energy harvest</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">444</div> (pg. <div class="fpage">1027</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20obesity-associated%20gut%20microbiome%20with%20increased%20capacity%20for%20energy%20harvest&amp;author=P.J.%20Turnbaugh&amp;author=R.E.%20Ley&amp;author=M.A.%20Mahowald&amp;author=V.%20Magrini&amp;author=E.R.%20Mardis&amp;author=J.I.%20Gordon&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=1027-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brulc</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Antonopoulos</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">M.E.</div></div>, et al. </div><div class="article-title">Gene-centric metagenomics of the fiber-adherent bovine rumen microbiome reveals forage specific glycoside hydrolases</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1948</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene-centric%20metagenomics%20of%20the%20fiber-adherent%20bovine%20rumen%20microbiome%20reveals%20forage%20specific%20glycoside%20hydrolases&amp;author=J.M.%20Brulc&amp;author=D.A.%20Antonopoulos&amp;author=M.E.%20Miller&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=1948-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0806191105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0806191105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0806191105"> </span></div> </div></div></div></div></div><div content-id="DST044C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Morrison</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Status of the phylogenetic diversity census of ruminal microbiomes</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2011</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">49</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Status%20of%20the%20phylogenetic%20diversity%20census%20of%20ruminal%20microbiomes&amp;author=M.%20Kim&amp;author=M.%20Morrison&amp;author=Z.%20Yu&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=76&amp;pages=49-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2011.76.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2011.76.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffem.2011.76.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21223325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beja</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">Unsuspected diversity among marine aerobic anoxygenic phototrophs</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">415</div> (pg. <div class="fpage">630</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unsuspected%20diversity%20among%20marine%20aerobic%20anoxygenic%20phototrophs&amp;author=O.%20Beja&amp;author=M.T.%20Suzuki&amp;author=J.F.%20Heidelberg&amp;publication_year=2002&amp;journal=Nature&amp;volume=415&amp;pages=630-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/415630a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F415630a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F415630a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11832943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Remington</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">66</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.C.%20Venter&amp;author=K.%20Remington&amp;author=J.F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whitford</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Forster</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Beard</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Gong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Teather</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Phylogenetic analysis of rumen bacteria by comparative sequence analysis of cloned 16S rRNA genes</div>, <div class="source">Anaerobe</div> , <div class="year">1998</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">153</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20rumen%20bacteria%20by%20comparative%20sequence%20analysis%20of%20cloned%2016S%20rRNA%20genes&amp;author=M.F.%20Whitford&amp;author=R.J.%20Forster&amp;author=C.E.%20Beard&amp;author=J.%20Gong&amp;author=R.M.%20Teather&amp;publication_year=1998&amp;journal=Anaerobe&amp;volume=4&amp;pages=153-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/anae.1998.0155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fanae.1998.0155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fanae.1998.0155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16887636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hess</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sczyrba</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Egan</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Metagenomic discovery of biomass-degrading genes and genomes from cow rumen</div>, <div class="source">Science</div> , <div class="year">2011</div>, vol. <div class="volume">331</div> (pg. <div class="fpage">463</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20discovery%20of%20biomass-degrading%20genes%20and%20genomes%20from%20cow%20rumen&amp;author=M.%20Hess&amp;author=A.%20Sczyrba&amp;author=R.%20Egan&amp;publication_year=2011&amp;journal=Science&amp;volume=331&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1200387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1200387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1200387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21273488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwards</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">McEwan</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Travis</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">John Wallace</div> <div class="given-names">R.</div></div>. </div><div class="article-title">16S rDNA library-based analysis of ruminal bacterial diversity</div>, <div class="source">Antonie Van Leeuwenhoek</div> , <div class="year">2004</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">263</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=16S%20rDNA%20library-based%20analysis%20of%20ruminal%20bacterial%20diversity&amp;author=J.E.%20Edwards&amp;author=N.R.%20McEwan&amp;author=A.J.%20Travis&amp;author=R.%20John%20Wallace&amp;publication_year=2004&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=86&amp;pages=263-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:ANTO.0000047942.69033.24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:ANTO.0000047942.69033.24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:ANTO.0000047942.69033.24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15539930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guan</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Nkrumah</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Basarab</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">S.S.</div></div>. </div><div class="article-title">Linkage of microbial ecology to phenotype: correlation of rumen microbial ecology to cattle's feed efficiency</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2008</div>, vol. <div class="volume">288</div> (pg. <div class="fpage">85</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20of%20microbial%20ecology%20to%20phenotype%3A%20correlation%20of%20rumen%20microbial%20ecology%20to%20cattle's%20feed%20efficiency&amp;author=L.L.%20Guan&amp;author=J.D.%20Nkrumah&amp;author=J.A.%20Basarab&amp;author=S.S.%20Moore&amp;publication_year=2008&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=288&amp;pages=85-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2008.288.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2008.288.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2008.288.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18785930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hernandez-Sanabria</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Guan</div> <div class="given-names">L.L.</div></div>. </div><div class="article-title">Assessment of the microbial ecology of ruminal methanogens in cattle with different feed efficiencies</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">6524</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20the%20microbial%20ecology%20of%20ruminal%20methanogens%20in%20cattle%20with%20different%20feed%20efficiencies&amp;author=M.%20Zhou&amp;author=E.%20Hernandez-Sanabria&amp;author=L.L.%20Guan&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=6524-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02815-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02815-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02815-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19717632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Teather</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Forster</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Composition, spatial distribution, and diversity of the bacterial communities in the rumen of cows fed different forages</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">612</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composition%2C%20spatial%20distribution%2C%20and%20diversity%20of%20the%20bacterial%20communities%20in%20the%20rumen%20of%20cows%20fed%20different%20forages&amp;author=Y.%20Kong&amp;author=R.%20Teather&amp;author=R.%20Forster&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=74&amp;pages=612-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6941.2010.00977.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6941.2010.00977.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6941.2010.00977.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pitta</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Pinchak</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Dowd</div> <div class="given-names">S.E.</div></div>, et al. </div><div class="article-title">Rumen bacterial diversity dynamics associated with changing from bermudagrass hay to grazed winter wheat diets</div>, <div class="source">Microb. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">511</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rumen%20bacterial%20diversity%20dynamics%20associated%20with%20changing%20from%20bermudagrass%20hay%20to%20grazed%20winter%20wheat%20diets&amp;author=D.W.%20Pitta&amp;author=E.%20Pinchak&amp;author=S.E.%20Dowd&amp;publication_year=2010&amp;journal=Microb.%20Ecol.&amp;volume=59&amp;pages=511-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00248-009-9609-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00248-009-9609-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00248-009-9609-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20037795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cammack</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Leymaster</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Jenkins</div> <div class="given-names">T.G.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">M.K.</div></div>. </div><div class="article-title">Estimates of genetic parameters for feed intake, feeding behavior, and daily gain in composite ram lambs</div>, <div class="source">J. Anim. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">777</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimates%20of%20genetic%20parameters%20for%20feed%20intake%2C%20feeding%20behavior%2C%20and%20daily%20gain%20in%20composite%20ram%20lambs&amp;author=K.M.%20Cammack&amp;author=K.A.%20Leymaster&amp;author=T.G.%20Jenkins&amp;author=M.K.%20Nielsen&amp;publication_year=2005&amp;journal=J.%20Anim.%20Sci.&amp;volume=83&amp;pages=777-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/2005.834777x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2F2005.834777x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2F2005.834777x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15753331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">186</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities&amp;author=B.%20Ewing&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=186-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Baker</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Singer</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Banfield</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">EMIRGE: reconstruction of full-length ribosomal genes from microbial community short read sequencing data</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R44</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMIRGE%3A%20reconstruction%20of%20full-length%20ribosomal%20genes%20from%20microbial%20community%20short%20read%20sequencing%20data&amp;author=C.S.%20Miller&amp;author=B.J.%20Baker&amp;author=B.C.%20Thomas&amp;author=S.W.%20Singer&amp;author=J.F.%20Banfield&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-5-r44" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-5-r44" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-5-r44"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21595876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R25</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Cardenas</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D141</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;author=J.R.%20Cole&amp;author=Q.%20Wang&amp;author=E.%20Cardenas&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D141-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Cavanaugh</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">GenBank</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D36</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBank&amp;author=D.A.%20Benson&amp;author=M.%20Cavanaugh&amp;author=K.%20Clark&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D36-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Needleman</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Wunsch</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1970</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">443</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;author=S.B.%20Needleman&amp;author=C.D.%20Wunsch&amp;publication_year=1970&amp;journal=J.%20Mol.%20Biol.&amp;volume=48&amp;pages=443-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(70)90057-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(70)90057-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(70)90057-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5420325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sajjapongse</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Truong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Conant</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Becchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A distributed CPU-GPU framework for pairwise alignments on large-scale sequence datasets</div>, <div class="year">2013</div><div class="conf-name">The 24th IEEE International Conference on Application-specific Systems, Architectures and Processors (ASAP13)</div><div class="conf-loc">George Washington University, VA</div>(pg. <div class="fpage">329</div>-<div class="lpage">38</div>)</div></div></div></div><div content-id="DST044C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Powell</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Carbone</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Dean</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Altered patterns of gene duplication and differential gene gain and loss in fungal pathogens</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">147</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20patterns%20of%20gene%20duplication%20and%20differential%20gene%20gain%20and%20loss%20in%20fungal%20pathogens&amp;author=A.J.%20Powell&amp;author=G.C.%20Conant&amp;author=D.E.%20Brown&amp;author=I.%20Carbone&amp;author=R.A.%20Dean&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=147" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18373860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sokal</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Rohlf</div> <div class="given-names">F.J.</div></div>. </div>, <div class="source">Biometry</div> , <div class="year">1995</div><div class="edition">3rd edition</div><div class="publisher-loc">New York</div><div class="publisher-name">W. H. Freeman and Company</div></div></div></div></div><div content-id="DST044C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Database resources of the National Center for Biotechnology Information: update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D35</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information%3A%20update&amp;author=D.L.%20Wheeler&amp;author=D.M.%20Church&amp;author=R.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D35-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McGill</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Etienne</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Species abundance distributions: moving beyond single prediction theories to integration within an ecological framework</div>, <div class="source">Ecol. Lett.</div> , <div class="year">2007</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">995</div>-<div class="lpage">1015</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20abundance%20distributions%3A%20moving%20beyond%20single%20prediction%20theories%20to%20integration%20within%20an%20ecological%20framework&amp;author=B.J.%20McGill&amp;author=R.S.%20Etienne&amp;author=J.S.%20Gray&amp;publication_year=2007&amp;journal=Ecol.%20Lett.&amp;volume=10&amp;pages=995-1015" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ele.2007.10.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fele.2007.10.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fele.2007.10.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17845298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Izsák</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pavoine</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Links between the species abundance distribution and the shape of the corresponding rank abundance curve</div>, <div class="source">Ecol. Indicators</div> , <div class="year">2012</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Links%20between%20the%20species%20abundance%20distribution%20and%20the%20shape%20of%20the%20corresponding%20rank%20abundance%20curve&amp;author=J.%20Izs%C3%A1k&amp;author=S.%20Pavoine&amp;publication_year=2012&amp;journal=Ecol.%20Indicators&amp;volume=14&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ecolind.2011.06.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ecolind.2011.06.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ecolind.2011.06.030"> </span></div> </div></div></div></div></div><div content-id="DST044C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Press</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Teukolsky</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Vetterling</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Flannery</div> <div class="given-names">B.P.</div></div>. </div>, <div class="source">Numerical Recipes in C</div> , <div class="year">1992</div><div class="publisher-loc">New York</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DST044C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>. </div><div class="article-title">Functional partitioning of yeast co-expression networks after genome duplication</div>, <div class="source">PLoS. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e109</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20partitioning%20of%20yeast%20co-expression%20networks%20after%20genome%20duplication&amp;author=G.C.%20Conant&amp;author=K.H.%20Wolfe&amp;publication_year=2006&amp;journal=PLoS.%20Biol.&amp;volume=4&amp;pages=e109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16555924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Forster</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ramaswamy</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Artaxo</div> <div class="given-names">P.</div></div>, et al.  , et al. </div><div class="person-group"><div class="name"><div class="surname">Solomon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Qin</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Manning</div> <div class="given-names">M</div></div>. </div><div class="article-title">Changes in Atmospheric Constituents and in Radiative Forcing</div>, <div class="source">Climate Change 2007: The Physical Science Basis. Contribution of Working Group I to the Fourth Assessment Report of the Intergovernmental Panel on Climate Change</div> , <div class="year">2007</div><div class="publisher-loc">Cambridge, UK</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DST044C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lana</div> <div class="given-names">R.R.P.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Van Amburgh</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">The role of pH in regulating ruminal methane and ammonia production</div>, <div class="source">J. Anim. Sci.</div> , <div class="year">1998</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">2190</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20pH%20in%20regulating%20ruminal%20methane%20and%20ammonia%20production&amp;author=R.R.P.%20Lana&amp;author=J.B.%20Russell&amp;author=M.E.%20Van%20Amburgh&amp;publication_year=1998&amp;journal=J.%20Anim.%20Sci.&amp;volume=76&amp;pages=2190-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/1998.7682190x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2F1998.7682190x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2F1998.7682190x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9734871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Unterseher</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jumpponen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Opik</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Species abundance distributions and richness estimations in fungal metagenomics—lessons learned from community ecology</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">275</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20abundance%20distributions%20and%20richness%20estimations%20in%20fungal%20metagenomics%E2%80%94lessons%20learned%20from%20community%20ecology&amp;author=M.%20Unterseher&amp;author=A.%20Jumpponen&amp;author=M.%20Opik&amp;publication_year=2011&amp;journal=Mol.%20Ecol.&amp;volume=20&amp;pages=275-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.20.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.20.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.20.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21155911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bekele</div> <div class="given-names">A.Z.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Genetic diversity and diet specificity of ruminal <em>Prevotella</em> revealed by 16S rRNA gene-based analysis</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2010</div>, vol. <div class="volume">305</div> (pg. <div class="fpage">49</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20diet%20specificity%20of%20ruminal%20Prevotella%20revealed%20by%2016S%20rRNA%20gene-based%20analysis&amp;author=A.Z.%20Bekele&amp;author=S.%20Koike&amp;author=Y.%20Kobayashi&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=305&amp;pages=49-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2010.305.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2010.305.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2010.305.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20158525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stevenson</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Weimer</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">Dominance of <em>Prevotella</em> and low abundance of classical ruminal bacterial species in the bovine rumen revealed by relative quantification real-time PCR</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">165</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dominance%20of%20Prevotella%20and%20low%20abundance%20of%20classical%20ruminal%20bacterial%20species%20in%20the%20bovine%20rumen%20revealed%20by%20relative%20quantification%20real-time%20PCR&amp;author=D.M.%20Stevenson&amp;author=P.J.%20Weimer&amp;publication_year=2007&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=75&amp;pages=165-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-006-0802-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-006-0802-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-006-0802-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17235560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hillebrand</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Cadotte</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">Consequences of dominance: a review of evenness effects on local and regional ecosystem processes</div>, <div class="source">Ecology</div> , <div class="year">2008</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1510</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consequences%20of%20dominance%3A%20a%20review%20of%20evenness%20effects%20on%20local%20and%20regional%20ecosystem%20processes&amp;author=H.%20Hillebrand&amp;author=D.M.%20Bennett&amp;author=M.W.%20Cadotte&amp;publication_year=2008&amp;journal=Ecology&amp;volume=89&amp;pages=1510-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/07-1053.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F07-1053.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F07-1053.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18589516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuomisto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A consistent terminology for quantifying species diversity? Yes, it does exist</div>, <div class="source">Oecologia</div> , <div class="year">2010</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">853</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consistent%20terminology%20for%20quantifying%20species%20diversity%3F%20Yes%2C%20it%20does%20exist&amp;author=H.%20Tuomisto&amp;publication_year=2010&amp;journal=Oecologia&amp;volume=164&amp;pages=853-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00442-010-1812-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00442-010-1812-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00442-010-1812-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20978798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Torsvik</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Ovreas</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Thingstad</div> <div class="given-names">T.F.</div></div>. </div><div class="article-title">Prokaryotic diversity–magnitude, dynamics, and controlling factors</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">1064</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotic%20diversity%E2%80%93magnitude%2C%20dynamics%2C%20and%20controlling%20factors&amp;author=V.%20Torsvik&amp;author=L.%20Ovreas&amp;author=T.F.%20Thingstad&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=1064-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1071698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1071698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1071698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12004116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST044C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Norberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Swaney</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Dushoff</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Casagrandi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Levin</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Phenotypic diversity and ecosystem functioning in changing environments: a theoretical framework</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">11376</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20diversity%20and%20ecosystem%20functioning%20in%20changing%20environments%3A%20a%20theoretical%20framework&amp;author=J.%20Norberg&amp;author=D.P.%20Swaney&amp;author=J.%20Dushoff&amp;author=J.%20Lin&amp;author=R.%20Casagrandi&amp;author=S.A.%20Levin&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11376-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.171315998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.171315998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.171315998"> </span></div> </div></div></div></div></div><div content-id="DST044C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST044C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nijs</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Roy</div> <div class="given-names">J.</div></div>. </div><div class="article-title">How important are species richness, species evenness and interspeciﬁc differences to productivity? A mathematical model</div>, <div class="source">Oikos</div> , <div class="year">2000</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">57</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20important%20are%20species%20richness%2C%20species%20evenness%20and%20interspeci%EF%AC%81c%20differences%20to%20productivity%3F%20A%20mathematical%20model&amp;author=I.%20Nijs&amp;author=J.%20Roy&amp;publication_year=2000&amp;journal=Oikos&amp;volume=88&amp;pages=57-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1034/j.1600-0706.2000.880107.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1034%2Fj.1600-0706.2000.880107.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1034%2Fj.1600-0706.2000.880107.x"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=401903&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst044/2/dst044-dst044supp.pdf?Expires=2147483647&amp;Signature=p04f7BHB-irfKw1pOJsq~lJHV8mbpp16KFEHq4NmQmYNhOyLFiTb1~hXXGSN58RmzsUjNM67ye-~VSl597oQLnbHv0J6mxSbwnl2A6-0o0yFrWYqjAO79H1BxWBQqZIQYFHySdKSm2EeJjJsvu7StQdKAZuCeYhlskGF1zTi6P~EqtKhEZBgG5ngz9NeDXX54DS2yLPXJm7dRpkFLrJLMr3lxibge4jVoyPl2VlmOb4NB5ENs3G0eQCm60rRz1NVb3IvVn-jSn1VFYXhh9XbPlqSsOWD55Um5s6jZR3xZaRAbu3mRL7WZ4w9x2U21KFc-KYxZzu0oRGaiydWuQs8sw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">298</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst044"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="401903"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f2%2f115%2f401903">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897500002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F2%2F115%2F401903" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/401903'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Diet Alters Both the Structure and Taxonomy of the Ovine Gut Microbial Ecosystem" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst044");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f115%2f401903">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

