

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/81/2656160","@type":"ScholarlyArticle","name":"Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP","datePublished":"2016-12-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw050","keywords":["target enrichment","sequence reconstruction","heterozygous variants","web tool analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/81/2656160","author":[{"name":"Ruggieri, Valentino","affiliation":"1 Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy   2 Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain","@type":"Person"},{"name":"Anzar, Irantzu","affiliation":"2 Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain","@type":"Person"},{"name":"Paytuvi, Andreu","affiliation":"2 Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain","@type":"Person"},{"name":"Calafiore, Roberta","affiliation":"1 Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy","@type":"Person"},{"name":"Cigliano, Riccardo Aiese","affiliation":"2 Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain","@type":"Person"},{"name":"Sanseverino, Walter","affiliation":"2 Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain","@type":"Person"},{"name":"Barone, Amalia","affiliation":"1 Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy","@type":"Person"}],"description":"Abstract.  The recent development of Sequence Capture methodology represents a powerful strategy for enhancing data generation to assess genetic variation of ta","pageStart":"81","pageEnd":"91","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP" />
    <meta property="og:description" content="Abstract.  The recent development of Sequence Capture methodology represents a powerful strategy for enhancing data generation to assess genetic variation of ta" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/81/2656160" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-12-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ruggieri, Valentino" /><meta name="citation_author_institution" content="Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy" /><meta name="citation_author_institution" content="Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain" /><meta name="citation_author" content="Anzar, Irantzu" /><meta name="citation_author_institution" content="Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain" /><meta name="citation_author" content="Paytuvi, Andreu" /><meta name="citation_author_institution" content="Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain" /><meta name="citation_author" content="Calafiore, Roberta" /><meta name="citation_author_institution" content="Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy" /><meta name="citation_author" content="Cigliano, Riccardo Aiese" /><meta name="citation_author_institution" content="Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain" /><meta name="citation_author" content="Sanseverino, Walter" /><meta name="citation_author_institution" content="Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain" /><meta name="citation_author" content="Barone, Amalia" /><meta name="citation_author_institution" content="Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy" /><meta name="citation_title" content="Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP" /><meta name="citation_firstpage" content="81" /><meta name="citation_lastpage" content="91" /><meta name="citation_doi" content="10.1093/dnares/dsw050" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Springer, Netherlands, Dordrecht;  citation_title=Cereal Genomics II;  citation_year=2013;  citation_pages=57-76; " /><meta name="citation_reference" content="citation_title=Next generation sequencing technologies for next generation plant breeding,;  citation_journal_title=Front. Plant. Sci;  citation_year=2014;  citation_volume=5;  citation_pages=367" /><meta name="citation_reference" content="citation_title=Genotyping a large collection of pepper (Capsicum spp.) with SSR loci brings new evidence for the wild origin of cultivated C. annuum and the structuring of genetic diversity by human selection of cultivar types,;  citation_journal_title=Genet. Resour. Crop Evol;  citation_year=2013;  citation_volume=60;  citation_pages=2375-90; " /><meta name="citation_reference" content="citation_title=A clarified position for Solanum lycopersicum var. cerasiforme in the evolutionary history of tomatoes (solanaceae);  citation_journal_title=BMC Plant Biol;  citation_year=2008;  citation_volume=8;  citation_pages=130" /><meta name="citation_reference" content="citation_title=Genomic variation in tomato, from wild ancestors to contemporary breeding accessions,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=257" /><meta name="citation_reference" content="citation_title=Development of genomics-based genotyping platforms and their applications in rice breeding,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2013;  citation_volume=16;  citation_pages=247-54; " /><meta name="citation_reference" content="citation_title=Population structure and genetic diversity characterization of a sunflower association mapping population using SSR and SNP markers,;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato,;  citation_journal_title=DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=593-603; " /><meta name="citation_reference" content="citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e40563" /><meta name="citation_reference" content="citation_title=Whole genome resequencing in tomato reveals variation associated with introgression and breeding events,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Patchwork sequencing of tomato San Marzano and Vesuviano varieties highlights genome-wide variations,;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=138" /><meta name="citation_reference" content="citation_title=Performance comparison of whole-genome sequencing platforms,;  citation_journal_title=Nat. Biotech;  citation_year=2012;  citation_volume=30;  citation_pages=78-82; " /><meta name="citation_reference" content="citation_title=Reference-guided assembly of four diverse Arabidopsis thaliana genomes,;  citation_journal_title=Proc. Natl. Acad. Sci. U S A;  citation_year=2011;  citation_volume=108;  citation_pages=10249-54; " /><meta name="citation_reference" content="citation_title=Direct selection of human genomic loci by microarray hybridization,;  citation_journal_title=Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=903-5; " /><meta name="citation_reference" content="citation_title=Microarray-based genomic selection for high-throughput resequencing,;  citation_journal_title=Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=907-9; " /><meta name="citation_reference" content="citation_title=Enrichment of super-sized resequencing targets from the human genome,;  citation_journal_title=Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=891-2; " /><meta name="citation_reference" content="citation_title=Phylogenetic properties of 50 nuclear loci in Medicago (Leguminosae) generated using multiplexed sequence capture and next-generation sequencing,;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e109704" /><meta name="citation_reference" content="citation_title=The promise of whole-exome sequencing in medical genetics,;  citation_journal_title=J. Hum. Genet;  citation_year=2014;  citation_volume=59;  citation_pages=5-15; " /><meta name="citation_reference" content="citation_title=Targeted sequencing reveals large-scale sequence polymorphism in maize candidate genes for biomass production and composition,;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0132120" /><meta name="citation_reference" content="citation_title=Multilocus sex determination revealed in two populations of gynodioecious wild strawberry, Fragaria vesca subsp. bracteata;  citation_journal_title=G3 (Bethesda);  citation_year=2015;  citation_volume=5;  citation_pages=2759-73; " /><meta name="citation_reference" content="citation_title=Genomic DNA enrichment using sequence capture microarrays: a novel approach to discover sequence nucleotide polymorphisms (SNP) in Brassica napus L,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e81992" /><meta name="citation_reference" content="citation_title=Capturing sequence variation among flowering-time regulatory gene homologs in the allopolyploid crop species Brassica napus,;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=404" /><meta name="citation_reference" content="citation_title=Targeted enrichment of the black cottonwood (Populus trichocarpa) gene space using sequence capture,;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e62355" /><meta name="citation_reference" content="citation_title=Using genic sequence capture in combination with a syntenic pseudo genome to map a deletion mutant in a wheat species,;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=80;  citation_pages=895-904; " /><meta name="citation_reference" content="citation_title=Targeted re-sequencing of the allohexaploid wheat exome,;  citation_journal_title=Plant Biotechnol. J;  citation_year=2012;  citation_volume=10;  citation_pages=733-42; " /><meta name="citation_reference" content="citation_title=Large-scale SNP discovery through RNA sequencing and SNP genotyping by targeted enrichment sequencing in cassava (Manihot esculenta Crantz),;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e116028" /><meta name="citation_reference" content="citation_title=Transposon Insertions, Structural Variations, and SNPs Contribute to the Evolution of the Melon Genome,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2760-74; " /><meta name="citation_reference" content="citation_title=Detecting false-positive signals in exome sequencing,;  citation_journal_title=Hum. Mutat;  citation_year=2012;  citation_volume=33;  citation_pages=609-13; " /><meta name="citation_reference" content="citation_title=Exploring a tomato landraces collection for fruit-related traits by the aid of a high-throughput genomic platform,;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0137139" /><meta name="citation_reference" content="citation_title=Integrated bioinformatics to decipher the ascorbic acid metabolic network in tomato,;  citation_journal_title=Plant. Mol. Biol;  citation_year=2016;  citation_volume=91;  citation_issue=4–5;  citation_pages=397-412; " /><meta name="citation_reference" content="citation_title=An association mapping approach to identify favourable alleles for tomato fruit quality breeding,;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=337" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=LUMPY: a probabilistic framework for structural variant discovery,;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=R84" /><meta name="citation_reference" content="citation_title=Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift,;  citation_journal_title=Front. Genet;  citation_year=2012;  citation_volume=3;  citation_pages=35" /><meta name="citation_reference" content="citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2987-93; " /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features,;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data,;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Fluorescence in situ hybridization and optical mapping to correct scaffold arrangement in the tomato genome,;  citation_journal_title=G3 (Bethesda);  citation_year=2014;  citation_volume=4;  citation_pages=1395-1405; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples,;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-35; " /><meta name="citation_reference" content="citation_title=Genomic analyses provide insights into the history of tomato breeding,;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=1220-6; " /><meta name="citation_reference" content="citation_title=The variant call format and VCFtools,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; " /><meta name="citation_reference" content="citation_title=Comparison of commercially available target enrichment methods for next-generation sequencing,;  citation_journal_title=J. Biomol. Tech;  citation_year=2013;  citation_volume=24;  citation_pages=73-86; " /><meta name="citation_reference" content="citation_title=Sequencing depth and coverage: key considerations in genomic analyses,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=121-32; " /><meta name="citation_reference" content="citation_title=Accurate whole human genome sequencing using reversible terminator chemistry,;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=53-9; " /><meta name="citation_reference" content="citation_title=Problems and solutions for estimating indel rates and length distributions,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2009;  citation_volume=26;  citation_pages=473-80; " /><meta name="citation_reference" content="citation_title=Probabilistic whole-genome alignments reveal high indel rates in the human and mouse genomes,;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=i289-96; " /><meta name="citation_reference" content="citation_title=Direct comparisons of Illumina vs. Roche 454 sequencing technologies on the same microbial community DNA sample,;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30087" /><meta name="citation_reference" content="citation_title=A rare SNP identified a TCP transcription factor essential for tendril development in cucumber,;  citation_journal_title=Mol. Plant;  citation_year=2015;  citation_volume=8;  citation_pages=1795-808; " /><meta name="citation_reference" content="citation_title=Targeted exon sequencing fails to identify rare coding variants with large effect in rheumatoid arthritis,;  citation_journal_title=Arthritis Res. Ther;  citation_year=2014;  citation_volume=16;  citation_pages=447" /><meta name="citation_reference" content="citation_title=Allelic genome structural variations in maize detected by array comparative genome hybridization;  citation_journal_title=Theor. Appl. Genet;  citation_year=2010;  citation_volume=120;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content,;  citation_journal_title=PLoS Genet;  citation_year=2009;  citation_volume=5;  citation_pages=e1000734" /><meta name="citation_reference" content="citation_title=Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor,;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1689-99; " /><meta name="citation_reference" content="citation_title=A large genome center's improvements to the Illumina sequencing system,;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=1005-10; " /><meta name="citation_reference" content="citation_title=Heads or tails: a simple reliability check for multiple sequence alignments,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1380-3; " /><meta name="citation_reference" content="citation_title=Galaxy: a web-based genome analysis tool for experimentalists,;  citation_journal_title=Curr. Protoc. Mol. Biol;  citation_year=2010; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/81/10731846/dsw050.pdf" /><meta name="description" content="Abstract.  The recent development of Sequence Capture methodology represents a powerful strategy for enhancing data generation to assess genetic variation of ta" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/81/2656160" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/81/2656160" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/1/81/2656160");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw050';
    </script>
<title>
	Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2656160" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbCHATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUnVnZ2llcmksIFZhbGVudGlubyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE5hcGxlcyBGZWRlcmljbyBJSSwgVmlhIFVuaXZlcnNpdMOgIDEwMCwgODAwNTUgUG9ydGljaSAoTkEpLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNlcXVlbnRpYSBCaW90ZWNoIFNMLCBDYWxsZSBDb21wdGUgZOKAmVVyZ2VsbCwgMjQwLCAwODAzNSBCYXJjZWxvbmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQW56YXIsIElyYW50enUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTZXF1ZW50aWEgQmlvdGVjaCBTTCwgQ2FsbGUgQ29tcHRlIGTigJlVcmdlbGwsIDI0MCwgMDgwMzUgQmFyY2Vsb25hLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBheXR1dmksIEFuZHJldSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNlcXVlbnRpYSBCaW90ZWNoIFNMLCBDYWxsZSBDb21wdGUgZOKAmVVyZ2VsbCwgMjQwLCAwODAzNSBCYXJjZWxvbmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FsYWZpb3JlLCBSb2JlcnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTmFwbGVzIEZlZGVyaWNvIElJLCBWaWEgVW5pdmVyc2l0w6AgMTAwLCA4MDA1NSBQb3J0aWNpIChOQSksIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2lnbGlhbm8sIFJpY2NhcmRvIEFpZXNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2VxdWVudGlhIEJpb3RlY2ggU0wsIENhbGxlIENvbXB0ZSBk4oCZVXJnZWxsLCAyNDAsIDA4MDM1IEJhcmNlbG9uYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYW5zZXZlcmlubywgV2FsdGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2VxdWVudGlhIEJpb3RlY2ggU0wsIENhbGxlIENvbXB0ZSBk4oCZVXJnZWxsLCAyNDAsIDA4MDM1IEJhcmNlbG9uYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYXJvbmUsIEFtYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE5hcGxlcyBGZWRlcmljbyBJSSwgVmlhIFVuaXZlcnNpdMOgIDEwMCwgODAwNTUgUG9ydGljaSAoTkEpLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRXhwbG9pdGluZyB0aGUgZ3JlYXQgcG90ZW50aWFsIG9mIFNlcXVlbmNlIENhcHR1cmUgZGF0YSBieSBhIG5ldyB0b29sLCBTVVBFUi1DQVAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgTmV0aGVybGFuZHMsIERvcmRyZWNodDsgIGNpdGF0aW9uX3RpdGxlPUNlcmVhbCBHZW5vbWljcyBJSTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3BhZ2VzPTU3LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2dpZXMgZm9yIG5leHQgZ2VuZXJhdGlvbiBwbGFudCBicmVlZGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaW5nIGEgbGFyZ2UgY29sbGVjdGlvbiBvZiBwZXBwZXIgKENhcHNpY3VtIHNwcC4pIHdpdGggU1NSIGxvY2kgYnJpbmdzIG5ldyBldmlkZW5jZSBmb3IgdGhlIHdpbGQgb3JpZ2luIG9mIGN1bHRpdmF0ZWQgQy4gYW5udXVtIGFuZCB0aGUgc3RydWN0dXJpbmcgb2YgZ2VuZXRpYyBkaXZlcnNpdHkgYnkgaHVtYW4gc2VsZWN0aW9uIG9mIGN1bHRpdmFyIHR5cGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlc291ci4gQ3JvcCBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MjM3NS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjbGFyaWZpZWQgcG9zaXRpb24gZm9yIFNvbGFudW0gbHljb3BlcnNpY3VtIHZhci4gY2VyYXNpZm9ybWUgaW4gdGhlIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIHRvbWF0b2VzIChzb2xhbmFjZWFlKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgdmFyaWF0aW9uIGluIHRvbWF0bywgZnJvbSB3aWxkIGFuY2VzdG9ycyB0byBjb250ZW1wb3JhcnkgYnJlZWRpbmcgYWNjZXNzaW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGdlbm9taWNzLWJhc2VkIGdlbm90eXBpbmcgcGxhdGZvcm1zIGFuZCB0aGVpciBhcHBsaWNhdGlvbnMgaW4gcmljZSBicmVlZGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yNDctNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIGFuZCBnZW5ldGljIGRpdmVyc2l0eSBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgc3VuZmxvd2VyIGFzc29jaWF0aW9uIG1hcHBpbmcgcG9wdWxhdGlvbiB1c2luZyBTU1IgYW5kIFNOUCBtYXJrZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIHVzaW5nIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIGRldmVsb3BlZCBieSByZS1zZXF1ZW5jaW5nIG9mIHRoZSBnZW5vbWVzIG9mIGN1bHRpdmF0ZWQgdG9tYXRvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU5My02MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgbGFyZ2UgU05QIGdlbm90eXBpbmcgYXJyYXkgYW5kIGdlbmVyYXRpb24gb2YgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwcyBpbiB0b21hdG8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQwNTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIHJlc2VxdWVuY2luZyBpbiB0b21hdG8gcmV2ZWFscyB2YXJpYXRpb24gYXNzb2NpYXRlZCB3aXRoIGludHJvZ3Jlc3Npb24gYW5kIGJyZWVkaW5nIGV2ZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdGNod29yayBzZXF1ZW5jaW5nIG9mIHRvbWF0byBTYW4gTWFyemFubyBhbmQgVmVzdXZpYW5vIHZhcmlldGllcyBoaWdobGlnaHRzIGdlbm9tZS13aWRlIHZhcmlhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJmb3JtYW5jZSBjb21wYXJpc29uIG9mIHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nIHBsYXRmb3Jtcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTc4LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWZlcmVuY2UtZ3VpZGVkIGFzc2VtYmx5IG9mIGZvdXIgZGl2ZXJzZSBBcmFiaWRvcHNpcyB0aGFsaWFuYSBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVIFMgQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMDI0OS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IHNlbGVjdGlvbiBvZiBodW1hbiBnZW5vbWljIGxvY2kgYnkgbWljcm9hcnJheSBoeWJyaWRpemF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz05MDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9hcnJheS1iYXNlZCBnZW5vbWljIHNlbGVjdGlvbiBmb3IgaGlnaC10aHJvdWdocHV0IHJlc2VxdWVuY2luZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9OTA3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVucmljaG1lbnQgb2Ygc3VwZXItc2l6ZWQgcmVzZXF1ZW5jaW5nIHRhcmdldHMgZnJvbSB0aGUgaHVtYW4gZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz04OTEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHByb3BlcnRpZXMgb2YgNTAgbnVjbGVhciBsb2NpIGluIE1lZGljYWdvIChMZWd1bWlub3NhZSkgZ2VuZXJhdGVkIHVzaW5nIG11bHRpcGxleGVkIHNlcXVlbmNlIGNhcHR1cmUgYW5kIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDk3MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcHJvbWlzZSBvZiB3aG9sZS1leG9tZSBzZXF1ZW5jaW5nIGluIG1lZGljYWwgZ2VuZXRpY3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIdW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgc2VxdWVuY2luZyByZXZlYWxzIGxhcmdlLXNjYWxlIHNlcXVlbmNlIHBvbHltb3JwaGlzbSBpbiBtYWl6ZSBjYW5kaWRhdGUgZ2VuZXMgZm9yIGJpb21hc3MgcHJvZHVjdGlvbiBhbmQgY29tcG9zaXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTMyMTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlsb2N1cyBzZXggZGV0ZXJtaW5hdGlvbiByZXZlYWxlZCBpbiB0d28gcG9wdWxhdGlvbnMgb2YgZ3lub2Rpb2VjaW91cyB3aWxkIHN0cmF3YmVycnksIEZyYWdhcmlhIHZlc2NhIHN1YnNwLiBicmFjdGVhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTI3NTktNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgRE5BIGVucmljaG1lbnQgdXNpbmcgc2VxdWVuY2UgY2FwdHVyZSBtaWNyb2FycmF5czogYSBub3ZlbCBhcHByb2FjaCB0byBkaXNjb3ZlciBzZXF1ZW5jZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgKFNOUCkgaW4gQnJhc3NpY2EgbmFwdXMgTCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lODE5OTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXB0dXJpbmcgc2VxdWVuY2UgdmFyaWF0aW9uIGFtb25nIGZsb3dlcmluZy10aW1lIHJlZ3VsYXRvcnkgZ2VuZSBob21vbG9ncyBpbiB0aGUgYWxsb3BvbHlwbG9pZCBjcm9wIHNwZWNpZXMgQnJhc3NpY2EgbmFwdXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBlbnJpY2htZW50IG9mIHRoZSBibGFjayBjb3R0b253b29kIChQb3B1bHVzIHRyaWNob2NhcnBhKSBnZW5lIHNwYWNlIHVzaW5nIHNlcXVlbmNlIGNhcHR1cmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIG1ldGhvZCBmb3IgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgYXV0b3RldHJhcGxvaWQgcG90YXRvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MjM1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIGdlbmljIHNlcXVlbmNlIGNhcHR1cmUgaW4gY29tYmluYXRpb24gd2l0aCBhIHN5bnRlbmljIHBzZXVkbyBnZW5vbWUgdG8gbWFwIGEgZGVsZXRpb24gbXV0YW50IGluIGEgd2hlYXQgc3BlY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz04OTUtOTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCByZS1zZXF1ZW5jaW5nIG9mIHRoZSBhbGxvaGV4YXBsb2lkIHdoZWF0IGV4b21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvdGVjaG5vbC4gSjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTczMy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgU05QIGRpc2NvdmVyeSB0aHJvdWdoIFJOQSBzZXF1ZW5jaW5nIGFuZCBTTlAgZ2Vub3R5cGluZyBieSB0YXJnZXRlZCBlbnJpY2htZW50IHNlcXVlbmNpbmcgaW4gY2Fzc2F2YSAoTWFuaWhvdCBlc2N1bGVudGEgQ3JhbnR6KSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTE2MDI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NvbiBJbnNlcnRpb25zLCBTdHJ1Y3R1cmFsIFZhcmlhdGlvbnMsIGFuZCBTTlBzIENvbnRyaWJ1dGUgdG8gdGhlIEV2b2x1dGlvbiBvZiB0aGUgTWVsb24gR2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9Mjc2MC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIGZhbHNlLXBvc2l0aXZlIHNpZ25hbHMgaW4gZXhvbWUgc2VxdWVuY2luZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTXV0YXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz02MDktMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBhIHRvbWF0byBsYW5kcmFjZXMgY29sbGVjdGlvbiBmb3IgZnJ1aXQtcmVsYXRlZCB0cmFpdHMgYnkgdGhlIGFpZCBvZiBhIGhpZ2gtdGhyb3VnaHB1dCBnZW5vbWljIHBsYXRmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDEzNzEzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0ZWQgYmlvaW5mb3JtYXRpY3MgdG8gZGVjaXBoZXIgdGhlIGFzY29yYmljIGFjaWQgbWV0YWJvbGljIG5ldHdvcmsgaW4gdG9tYXRvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX2lzc3VlPTTigJM1OyAgY2l0YXRpb25fcGFnZXM9Mzk3LTQxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXNzb2NpYXRpb24gbWFwcGluZyBhcHByb2FjaCB0byBpZGVudGlmeSBmYXZvdXJhYmxlIGFsbGVsZXMgZm9yIHRvbWF0byBmcnVpdCBxdWFsaXR5IGJyZWVkaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTZXF1ZW5jZSBBbGlnbm1lbnQvTWFwIGZvcm1hdCBhbmQgU0FNdG9vbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TFVNUFk6IGEgcHJvYmFiaWxpc3RpYyBmcmFtZXdvcmsgZm9yIHN0cnVjdHVyYWwgdmFyaWFudCBkaXNjb3ZlcnksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPVI4NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGFzIGEgTW9kZWwgZm9yIEdlbm90b3hpYyBDaGVtaWNhbCBNdXRhdGlvbmFsIFN0dWRpZXMgd2l0aCBhIE5ldyBQcm9ncmFtLCBTbnBTaWZ0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3RhdGlzdGljYWwgZnJhbWV3b3JrIGZvciBTTlAgY2FsbGluZywgbXV0YXRpb24gZGlzY292ZXJ5LCBhc3NvY2lhdGlvbiBtYXBwaW5nIGFuZCBwb3B1bGF0aW9uIGdlbmV0aWNhbCBwYXJhbWV0ZXIgZXN0aW1hdGlvbiBmcm9tIHNlcXVlbmNpbmcgZGF0YSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9Mjk4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkVEVG9vbHM6IGEgZmxleGlibGUgc3VpdGUgb2YgdXRpbGl0aWVzIGZvciBjb21wYXJpbmcgZ2Vub21pYyBmZWF0dXJlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmltbW9tYXRpYzogYSBmbGV4aWJsZSB0cmltbWVyIGZvciBJbGx1bWluYSBzZXF1ZW5jZSBkYXRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMTE0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIGFuZCBvcHRpY2FsIG1hcHBpbmcgdG8gY29ycmVjdCBzY2FmZm9sZCBhcnJhbmdlbWVudCBpbiB0aGUgdG9tYXRvIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEzOTUtMTQwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VEFTU0VMOiBzb2Z0d2FyZSBmb3IgYXNzb2NpYXRpb24gbWFwcGluZyBvZiBjb21wbGV4IHRyYWl0cyBpbiBkaXZlcnNlIHNhbXBsZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MzMtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgYW5hbHlzZXMgcHJvdmlkZSBpbnNpZ2h0cyBpbnRvIHRoZSBoaXN0b3J5IG9mIHRvbWF0byBicmVlZGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0xMjIwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB2YXJpYW50IGNhbGwgZm9ybWF0IGFuZCBWQ0Z0b29scyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE1Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGNvbW1lcmNpYWxseSBhdmFpbGFibGUgdGFyZ2V0IGVucmljaG1lbnQgbWV0aG9kcyBmb3IgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9tb2wuIFRlY2g7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz03My04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBkZXB0aCBhbmQgY292ZXJhZ2U6IGtleSBjb25zaWRlcmF0aW9ucyBpbiBnZW5vbWljIGFuYWx5c2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSB3aG9sZSBodW1hbiBnZW5vbWUgc2VxdWVuY2luZyB1c2luZyByZXZlcnNpYmxlIHRlcm1pbmF0b3IgY2hlbWlzdHJ5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1NjsgIGNpdGF0aW9uX3BhZ2VzPTUzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JsZW1zIGFuZCBzb2x1dGlvbnMgZm9yIGVzdGltYXRpbmcgaW5kZWwgcmF0ZXMgYW5kIGxlbmd0aCBkaXN0cmlidXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NDczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9iYWJpbGlzdGljIHdob2xlLWdlbm9tZSBhbGlnbm1lbnRzIHJldmVhbCBoaWdoIGluZGVsIHJhdGVzIGluIHRoZSBodW1hbiBhbmQgbW91c2UgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9aTI4OS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IGNvbXBhcmlzb25zIG9mIElsbHVtaW5hIHZzLiBSb2NoZSA0NTQgc2VxdWVuY2luZyB0ZWNobm9sb2dpZXMgb24gdGhlIHNhbWUgbWljcm9iaWFsIGNvbW11bml0eSBETkEgc2FtcGxlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMDA4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmFyZSBTTlAgaWRlbnRpZmllZCBhIFRDUCB0cmFuc2NyaXB0aW9uIGZhY3RvciBlc3NlbnRpYWwgZm9yIHRlbmRyaWwgZGV2ZWxvcG1lbnQgaW4gY3VjdW1iZXIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xNzk1LTgwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgZXhvbiBzZXF1ZW5jaW5nIGZhaWxzIHRvIGlkZW50aWZ5IHJhcmUgY29kaW5nIHZhcmlhbnRzIHdpdGggbGFyZ2UgZWZmZWN0IGluIHJoZXVtYXRvaWQgYXJ0aHJpdGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXJ0aHJpdGlzIFJlcy4gVGhlcjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTQ0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbGVsaWMgZ2Vub21lIHN0cnVjdHVyYWwgdmFyaWF0aW9ucyBpbiBtYWl6ZSBkZXRlY3RlZCBieSBhcnJheSBjb21wYXJhdGl2ZSBnZW5vbWUgaHlicmlkaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFpemUgaW5icmVkcyBleGhpYml0IGhpZ2ggbGV2ZWxzIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbiAoQ05WKSBhbmQgcHJlc2VuY2UvYWJzZW5jZSB2YXJpYXRpb24gKFBBVikgaW4gZ2Vub21lIGNvbnRlbnQsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDczNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcnZhc2l2ZSBnZW5lIGNvbnRlbnQgdmFyaWF0aW9uIGFuZCBjb3B5IG51bWJlciB2YXJpYXRpb24gaW4gbWFpemUgYW5kIGl0cyB1bmRvbWVzdGljYXRlZCBwcm9nZW5pdG9yLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE2ODktOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbGFyZ2UgZ2Vub21lIGNlbnRlcidzIGltcHJvdmVtZW50cyB0byB0aGUgSWxsdW1pbmEgc2VxdWVuY2luZyBzeXN0ZW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYWRzIG9yIHRhaWxzOiBhIHNpbXBsZSByZWxpYWJpbGl0eSBjaGVjayBmb3IgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FsYXh5OiBhIHdlYi1iYXNlZCBnZW5vbWUgYW5hbHlzaXMgdG9vbCBmb3IgZXhwZXJpbWVudGFsaXN0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFByb3RvYy4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMS84MS8xMDczMTg0Ni9kc3cwNTAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSByZWNlbnQgZGV2ZWxvcG1lbnQgb2YgU2VxdWVuY2UgQ2FwdHVyZSBtZXRob2RvbG9neSByZXByZXNlbnRzIGEgcG93ZXJmdWwgc3RyYXRlZ3kgZm9yIGVuaGFuY2luZyBkYXRhIGdlbmVyYXRpb24gdG8gYXNzZXNzIGdlbmV0aWMgdmFyaWF0aW9uIG9mIHRhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNjU2MTYwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZEiQ2qOwNe5B5dRMVA80TruuI7dX" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f81%2f2656160" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f81%2f2656160" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=96271" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&amp;Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61463565">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463565">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463567">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463567">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463572">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463593">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463593">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463618">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463618">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61463626">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61463626">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/71/2726856">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/93/2738680">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Valentino Ruggieri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy</div><div class="aff"><div class="label -label">2</div>Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Valentino+Ruggieri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ruggieri V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ruggieri V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Valentino Ruggieri</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Irantzu Anzar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Irantzu+Anzar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Anzar I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Anzar I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Irantzu Anzar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andreu Paytuvi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andreu+Paytuvi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paytuvi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paytuvi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andreu Paytuvi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roberta Calafiore</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roberta+Calafiore">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Calafiore R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Calafiore R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Roberta Calafiore</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Riccardo Aiese Cigliano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw050-cor1" data-open="dsw050-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 0034 930107368. Email: <a href="mailto:raiesecigliano@sequentiabiotech.com" target="_blank">raiesecigliano@sequentiabiotech.com</a> (R.A.C.); <a href="mailto:wsanseverino@sequentiabiotech.com" target="_blank">wsanseverino@sequentiabiotech.com</a> (W.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Riccardo+Aiese+Cigliano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cigliano R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cigliano R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Riccardo Aiese Cigliano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Walter Sanseverino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Sequentia Biotech SL, Calle Compte d’Urgell, 240, 08035 Barcelona, Spain</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw050-cor1" data-open="dsw050-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 0034 930107368. Email: <a href="mailto:raiesecigliano@sequentiabiotech.com" target="_blank">raiesecigliano@sequentiabiotech.com</a> (R.A.C.); <a href="mailto:wsanseverino@sequentiabiotech.com" target="_blank">wsanseverino@sequentiabiotech.com</a> (W.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Walter+Sanseverino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sanseverino W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sanseverino W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Walter Sanseverino</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Amalia Barone</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Agricultural Sciences, University of Naples Federico II, Via Università 100, 80055 Portici (NA), Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Amalia+Barone">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Barone A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Barone A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Amalia Barone</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw050-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Mikita Suyama</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 81–91, <a href='https://doi.org/10.1093/dnares/dsw050'>https://doi.org/10.1093/dnares/dsw050</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 May 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 October 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2656160" href="/dnaresearch/article-pdf/24/1/81/10731846/dsw050.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Valentino Ruggieri, Irantzu Anzar, Andreu Paytuvi, Roberta Calafiore, Riccardo Aiese Cigliano, Walter Sanseverino, Amalia Barone;  Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 81–91, <a href='https://doi.org/10.1093/dnares/dsw050'>https://doi.org/10.1093/dnares/dsw050</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2656160?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Exploiting%20the%20great%20potential%20of%20Sequence%20Capture%20data%20by%20a%20new%20tool%2C%20SUPER-CAP&amp;publicationDate=2016-12-08&amp;volumeNum=24&amp;issueNum=1&amp;author=Ruggieri%2C%20Valentino%3B%20Anzar%2C%20Irantzu&amp;startPage=81&amp;endPage=91&amp;contentId=10.1093%2Fdnares%2Fdsw050&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f81%2f2656160" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61463565 id="61463565" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The recent development of Sequence Capture methodology represents a powerful strategy for enhancing data generation to assess genetic variation of targeted genomic regions. Here, we present SUPER-CAP, a bioinformatics web tool aimed at handling Sequence Capture data, fine calculating the allele frequency of variations and building genotype-specific sequence of captured genes. The dataset used to develop this in silico strategy consists of 378 loci and related regulative regions in a collection of 44 tomato landraces. About 14,000 high-quality variants were identified. The high depth (&gt;40×) of coverage and adopting the correct filtering criteria allowed identification of about 4,000 rare variants and 10 genes with a different copy number variation. We also show that the tool is capable to reconstruct genotype-specific sequences for each genotype by using the detected variants. This allows evaluating the combined effect of multiple variants in the same protein. The architecture and functionality of SUPER-CAP makes the software appropriate for a broad set of analyses including SNP discovery and mining. Its functionality, together with the capability to process large data sets and efficient detection of sequence variation, makes SUPER-CAP a valuable bioinformatics tool for genomics and breeding purposes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;target enrichment&quot;">target enrichment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence reconstruction&quot;">sequence reconstruction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;heterozygous variants&quot;">heterozygous variants</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;web tool analysis&quot;">web tool analysis</a></div>                            <h2 scrollto-destination=61463567 id="61463567" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Identifying sequence polymorphisms responsible for phenotypic variation and understanding the genetic basis of complex traits have been two major challenges of plant molecular genetics since the development of the first molecular markers up to the impressive high-throughput genomics technologies available today.<a reveal-id="dsw050-B1" data-open="dsw050-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw050-B2" data-open="dsw050-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The process of genotype-phenotype association is one of the central goals in the path towards plant improvement, and this requires the use of all the genetic and genomic information available for a given individual and/or population. The association of the genomic variation with traits of interest requires the reliable detection and the systematic investigation of the entire spectrum of DNA variability, including single nucleotide polymorphisms (SNPs), insertions/deletions (INDELs) as well as copy number variations (CNVs) and presence/absence of variations (PAV). Substantial progress towards this goal was made in the last few years. Large germoplasm collections have been characterized using SSR,<a reveal-id="dsw050-B3" data-open="dsw050-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw050-B4" data-open="dsw050-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> SNP markers and SNP arrays.<a reveal-id="dsw050-B5 dsw050-B6 dsw050-B7 dsw050-B8 dsw050-B9" data-open="dsw050-B5 dsw050-B6 dsw050-B7 dsw050-B8 dsw050-B9" class="link link-ref link-reveal xref-bibr"><sup>5–9</sup></a> However, most of these approaches had known disadvantages that limited the power of the detection, in particular because the SNPs used in these studies were selected from a limited number of divergent sources and were commonly chosen to exceed a minimum frequency of the rare allele. Whole genome sequencing has also been used to explore individual variation at the genomic level in plants<a reveal-id="dsw050-B10 dsw050-B11 dsw050-B12 dsw050-B13" data-open="dsw050-B10 dsw050-B11 dsw050-B12 dsw050-B13" class="link link-ref link-reveal xref-bibr"><sup>10–13</sup></a> but, due to its high cost and complexity in data analysis, it is not expected to be widely applied for investigation of variants underlying specific traits of interest at a sufficient coverage. Recently, the development of microarray based or liquid-based genomic selection methods, commonly referred to as ‘Sequence Capture’, provided an affordable way to produce high-quality variants, thus solving most of the ascertained bias reported.<a reveal-id="dsw050-B14 dsw050-B15 dsw050-B16" data-open="dsw050-B14 dsw050-B15 dsw050-B16" class="link link-ref link-reveal xref-bibr"><sup>14–16</sup></a> Combining the recently developed targeted sequence enrichment with Next-Generation Sequencing (NGS) technologies, Sequence Capture methodology represents a powerful strategy for enhancing data generation to assess genetic variation of target regions, and it is likely to replace PCR as the main target enrichment method in both plants and animals.<a reveal-id="dsw050-B17" data-open="dsw050-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In addition, setting the experiments to obtain a sufficient depth of coverage would support the detection of rare functional variants and allow the discovery of complete haplotypes of genes, as well as CNV and PAV. With the ever-decreasing costs of sequencing and the advances in Sequence Capture technologies, these approaches are nowadays largely applied in different fields. Thanks to its high reliability and moderate cost per experiment, Sequence Capture is becoming an affordable diagnostic tool for medical and personalized medicine purposes.<a reveal-id="dsw050-B18" data-open="dsw050-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Despite its large employment in the medical field, the use of Sequence Capture in plant science is still emerging. Sequence Capture experiments were carried out for few species, including maize,<a reveal-id="dsw050-B19" data-open="dsw050-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> strawberry,<a reveal-id="dsw050-B20" data-open="dsw050-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> rapeseed canola,<a reveal-id="dsw050-B21" data-open="dsw050-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw050-B22" data-open="dsw050-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> black cottonwood,<a reveal-id="dsw050-B23" data-open="dsw050-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> potato,<a reveal-id="dsw050-B24" data-open="dsw050-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> wheat,<a reveal-id="dsw050-B25" data-open="dsw050-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw050-B26" data-open="dsw050-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> medicago<a reveal-id="dsw050-B17" data-open="dsw050-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and cassava.<a reveal-id="dsw050-B27" data-open="dsw050-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p><p>The work herein proposed, which investigates the sequence variation of a group of 378 genes and the related regulative regions in a collection of 44 landraces, represents the first study of Sequence Capture in tomato species. Although software tools are available for variant calling and variant mining, their application is not straightforward, requiring users to install various packages and to convert data into different formats. This lack of easily accessible software pushed us to propose a web-based tool, named SUPER-CAP (<a class="link link-uri" href="http://supercap.sequentiabiotech.com/" target="_blank">http://supercap.sequentiabiotech.com/</a>), to boost quick, proficient and affordable analysis of sequence capture data. This tool, benefitting from SUPER-W pipeline<a reveal-id="dsw050-B28" data-open="dsw050-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and combining different software and customizable procedures, assists the user in sequence capture experiments from the identification of single-nucleotide variants (SNVs) and small insertions and deletions (INDELs) to the reconstruction of genotype-specific (hereafter referred to as ‘private’) sequences/gene features for each target region of each sample.</p><p>In addition, in this study, an explorative investigation on different aspects affecting sequence capture data was also carried out. For instance, since one of the major challenges in the enrichment/capture technologies are to avoid spurious variants calling for heterozygous (He) loci, this work evaluated how the depth of the reads and the procedure set-ups impact the variant calling of He variants. In fact, being the tomato a highly homozygous (Ho) species, it represents a good model for testing He variant calling into the gene set considered. In addition, spurious callings are frequently referred to multi-copy gene families (duplicated region/high homologous regions).<a reveal-id="dsw050-B29" data-open="dsw050-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The gene set considered in this study represents a large range of variability in terms of gene families. This allowed for suitable assessment and estimation of the effective presence of He variants among the multi-copy genes.</p><p>As a final point, the variant set was also used to investigate: (i) the level and the type of sequence polymorphism in captured genes across the 44 tomato samples, (ii) the pattern of variants distribution among the lines, with respect to the identification of rare variants, (iii) the number of genes CNV, PAV and (iv) the functional annotation of the variants in order to prioritize the study of highly relevant variants. The work herein proposed represents a framework for easy Sequence Capture-related experiments that will promote similar studies, as well as in differing species. </p>                            <h2 scrollto-destination=61463572 id="61463572" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=61463573 id="61463573" class="section-title" >2.1. Plant material and selection of target genes</h3>
<p>A panel of 44 genetically diverse <em>Solanum</em><em>lycopersicum</em> genotypes was selected from a wide collection of tomato landraces<a reveal-id="dsw050-B30" data-open="dsw050-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> available at the University of Naples, department of Agricultural Sciences. Plants were grown in a greenhouse under controlled conditions at the aforementioned Department; detailed information, including source and geographic origin for each genotype, is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p><p>Genomic DNA from the 44 genotypes was isolated from young tomato leaves using the DNeasy Plant Mini Kit (Qiagen, Valencia, CA, USA). DNA quantity and purity/degradation was firstly checked on 1% agarose gel. In order to fit the standard parameters for Sequence Capture analyses, DNA concentration and quality were determined by Nanodrop spectrophotometer (Thermo Fisher Scientific, Waltham, MA, USA) and Qubit fluorometer (Life Technologies, Darmstadt, Germany) according to the manufacturer’s requirements.</p><p>An inventory of 378 candidate genes representing possible targets for antioxidant metabolism in tomatoes was compiled from published literature, previous research<a reveal-id="dsw050-B31" data-open="dsw050-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw050-B32" data-open="dsw050-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and exploration of metabolic pathway databases (LycoCyc at <a class="link link-uri" href="http://solgenomics.net/" target="_blank">http://solgenomics.net/</a>and KEGG at <a class="link link-uri" href="http://www.genome.jp/kegg/" target="_blank">http://www.genome.jp/kegg/</a>). Genes were also selected to represent different gene family sizes. In total 235 out of the 378 genes completely represent 25 gene families. In particular, 171 genes belong to eight large gene families (we declared a big family if composed by more than 10 copies in the genome), 40 belong to seven medium gene families (between nine and four gene copies), and 24 belong to 10 small gene families (two or three copies). Six out of the 378 genes represent single copy genes. Moreover, in order to take into account variations in the regulatory region of each selected gene, a 3 Kbp promoter region was also included in the analysis. For genes with an intergenic distance shorter than 3 Kbp, only the related shorter portion of the promoter was considered. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> shows detailed information of the genomic regions considered.</p>                            <h3 scrollto-destination=61463577 id="61463577" class="section-title" >2.2. Sequence capture design and sequencing</h3>
<p>Probe design and gene enrichment were performed following the protocol provided with the solution-based Roche NimbleGen SeqCap EZ Library (Roche-NimbleGen, Madison, WI, USA). <em>Loci</em> coordinates of the genic and promotor regions were identified and submitted to Roche Diagnostics for probe design using NimbleDesign and SignalMap software (<a class="link link-uri" href="http://www.nimblegen.com/products/software/index.html" target="_blank">http://www.nimblegen.com/products/software/index.html</a>). This probe set contains probes with up to 20 close matches in the genome as determined by the Sequence Search and Alignment by Hashing Algorithm (<a class="link link-uri" href="http://www.sanger.ac.uk/resources/software/ssaha/" target="_blank">http://www.sanger.ac.uk/resources/software/ssaha/</a>). Acco rding to NimbleGen specification, we considered a probe to match the genome if the variation ratio was &lt;0.05. Following the SeqCapEZ protocol, 44 paired-end libraries were prepared using the Illumina Kapa Library Prep Kit and were multiplexed and sequenced with HiSeq 1,500 (read size:100 bp) according to Illumina specifications.</p>                            <h3 scrollto-destination=61463579 id="61463579" class="section-title" >2.3. SUPER-CAP usage and procedures</h3>
<div class="block-child-p">SUPER-CAP is a bioinformatics tool specifically designed for accurate mapping, variant calling and reconstruction of private sequences using the variations detected from Sequence Capture data. SUPER-CAP includes an updated version of SUPER-W (release 4).<a reveal-id="dsw050-B28" data-open="dsw050-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> SUPER-CAP is a user-friendly web tool which only needs two input files to work: the captured region file in BED format and the filtered NGS reads in FASTQ format. The main steps and procedure underlying the SUPER-CAP tool, including the calling variants, the variants filtering and the targeted sequence reconstruction are presented in <a reveal-id="dsw050-F1" data-open="dsw050-F1" class="link link-reveal link-table xref-fig">Figure 1</a>. <div data-id="dsw050-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f1.png?Expires=2147483647&Signature=y30It9Y6gr0LmhG8GMAm6aWP69lDgXznCw9KWcgeqsl91NzKs2hS9BsY7Z6g6cYEHlA3U9LN6SDHCkBaOJKOkGqtNPPRqooJ0FDlb2lKJHwgNJHl94W2bSOpVO3DmK4aa-mL01YthEE-GRH76y8RSX~VaejZmTlEHT~0dHBsDc5CHsWGF7kYShMzXJg6r7yeM4uP-uyM1hXqQmx9a-dBa0aw1OEHHzPtMsv2X8cXuFYPU3lFmuRNMh7ZRSWibyTKSLQDngb7szFA1SysJKCoQjM8Knck7NJ7ZXtOPuCnaLuXNAIzjsctcViguXqRp~nVQhLoGw6LlLzf6Kk7G9UaIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The SUPER-CAP tool. The main steps and procedures underlying the tool are graphically reported in the workflow." path-from-xml="dsw050f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463580/dsw050f1.tif" path-from-xml="dsw050f1.tif" target="_blank">View large</a><a section="61463580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f1.png?Expires=2147483647&Signature=n2DWDcGMCPCQl7e9VTFkqzVauJMJ3Tvu5ny2hhazJZtwgGbiFNCSKceqDJ77~5bElkh0UIkmLX3-3DZZx-JNj1RNJuzW3NvS3JUwIfJKiptA~fx8LtzEolRXjo5NEDPdKlDg-KjUlp~gFGVXrOyTLoFioZh74mMiLsCiHqVbLt6EtjyTUCBhssphXgZne5gZDXUV6YMyB7ufV7~2dkpGwAk8i4SvN53UaLLwSu4NofB7ZohJRQg2RyE-gKcZULL2DU6~2LGTrGg2y19Eu2SdfmtVoFkMNbIjHvBKq4OUKNKqyrv6YpfWc28awpKjFYpcC~0Vl7kRwSIWO7eWlHJHAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463580&ar=2656160&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw050f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The SUPER-CAP tool. The main steps and procedures underlying the tool are graphically reported in the workflow.</p></div></div><div content-id="dsw050-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f1.png?Expires=2147483647&Signature=y30It9Y6gr0LmhG8GMAm6aWP69lDgXznCw9KWcgeqsl91NzKs2hS9BsY7Z6g6cYEHlA3U9LN6SDHCkBaOJKOkGqtNPPRqooJ0FDlb2lKJHwgNJHl94W2bSOpVO3DmK4aa-mL01YthEE-GRH76y8RSX~VaejZmTlEHT~0dHBsDc5CHsWGF7kYShMzXJg6r7yeM4uP-uyM1hXqQmx9a-dBa0aw1OEHHzPtMsv2X8cXuFYPU3lFmuRNMh7ZRSWibyTKSLQDngb7szFA1SysJKCoQjM8Knck7NJ7ZXtOPuCnaLuXNAIzjsctcViguXqRp~nVQhLoGw6LlLzf6Kk7G9UaIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The SUPER-CAP tool. The main steps and procedures underlying the tool are graphically reported in the workflow." path-from-xml="dsw050f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463580/dsw050f1.tif" path-from-xml="dsw050f1.tif" target="_blank">View large</a><a section="61463580" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f1.png?Expires=2147483647&Signature=n2DWDcGMCPCQl7e9VTFkqzVauJMJ3Tvu5ny2hhazJZtwgGbiFNCSKceqDJ77~5bElkh0UIkmLX3-3DZZx-JNj1RNJuzW3NvS3JUwIfJKiptA~fx8LtzEolRXjo5NEDPdKlDg-KjUlp~gFGVXrOyTLoFioZh74mMiLsCiHqVbLt6EtjyTUCBhssphXgZne5gZDXUV6YMyB7ufV7~2dkpGwAk8i4SvN53UaLLwSu4NofB7ZohJRQg2RyE-gKcZULL2DU6~2LGTrGg2y19Eu2SdfmtVoFkMNbIjHvBKq4OUKNKqyrv6YpfWc28awpKjFYpcC~0Vl7kRwSIWO7eWlHJHAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61463580&ar=2656160&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw050f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The SUPER-CAP tool. The main steps and procedures underlying the tool are graphically reported in the workflow.</p></div></div></div><p>SUPER-W has been modified to specifically handle sequence capture data. In this new version, SUPER-W uses as input the BED file of capture probe design (it can also handle a whole exome) and the filtered NGS reads. The first step is to map all the samples against a reference genome (specified by the user) with BWA (version 0.7.5; options used: mem:BWA-MEM algorithm).<a reveal-id="dsw050-B33" data-open="dsw050-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The mapped files are processed for PCR duplicates (Picard, MarkDuplicates tool, version 1.118), filtered for quality (minimum quality required: 30), sorted and indexed.<a reveal-id="dsw050-B34" data-open="dsw050-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The resulting BAM file is then used for the variant calling step. Small variations (SNPs and short deletion and insertion polymorphisms), large variations (deletions, inversions and duplications), or both, can be set for detection. The small variations are called with SAMtools<a reveal-id="dsw050-B34" data-open="dsw050-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> through an accurate and sensitive double calling step, while the structural variants are detected using LUMPY (version 0.0.11; options used: -mv 4 –tt 0 –pe –sr).<a reveal-id="dsw050-B35" data-open="dsw050-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p>Once the variant calling step has been successfully completed, statistics on sequence capture experiment are calculated. The statistics report can be considered as a checkpoint, allowing the user to check the IN/OFF target read count, which highlights the specificity or sensitivity of the sequence capture experiment.</p><p>Raw genomic variants annotated by SUPER-W are then classified and filtered out. As default, only variants with a coverage &gt;6× and a Phred quality &gt;30 were used for subsequent analysis. Furthermore, variants overlapping each other are removed from downstream analysis as they cause a low confidence region. Moreover, a special option has been released to allow calling variants according to customized allele frequency (AF). As default value, variants with an AF between 0 and 0.2 are considered Ho for the reference allele, variants with a AF between 0.4 and 0.6 are considered He while variants with an AF between 0.8 and 1 are considered Ho for the alternative allele. Variants with an AF ranging out of boundaries set (i.e. between 0.2 and 0.4 and between 0.6 and 0.8) are called with an alert comment in the output VCF file. Coverage and quality filters as well as the AF parameters can be manually modified by the user. Finally, the filtered variants were classified according to type (SNP or INDEL). For SNP variants the breakdown into transitions and transversions was also determined. To improve the variant features, SnpSift and SnpEff<a reveal-id="dsw050-B36" data-open="dsw050-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> were implemented in the workflow, which allows imputing the gene region in which the variants fall (promoter, intron, exon, Untranslated regions (UTRs)) and the putative impact of the variants on protein functionality.</p><p>The high-quality variants, the reference genome and the BED file of the captured regions were used to perform a targeted sequence reconstruction. Using bcftools<a reveal-id="dsw050-B37" data-open="dsw050-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and BEDtools<a reveal-id="dsw050-B38" data-open="dsw050-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> utilities, a new consensus sequence was created by applying the filtered variants to the reference sequence. He SNPs were introduced using IUPAC ambiguity codes and He INDELs were discarded. Through a liftover process, which recalculates the gene coordinates after the insertion or deletion of INDELs, a FASTA file with reconstructed sequences was created. Variants reconstructed were also displayed in the SUPER-CAP’s browser results page, allowing the user to explore the type and the number of variants easily.</p>                            <h3 scrollto-destination=61463585 id="61463585" class="section-title" >2.4. Web tool development and design</h3>
<p>With the aim of supplying SUPER-CAP with a graphical user interface, we developed a NodeJS-based web tool (version 4.4.1. Available at <a class="link link-uri" href="https://nodejs.org/" target="_blank">https://nodejs.org/</a>), available at ‘<a class="link link-uri" href="http://supercap.sequentiabiotech.com/" target="_blank">http://supercap.sequentiabiotech.com/</a>’. Using this tool, the user can upload the two required files (the BED file of the target regions and the FASTQ files of the sequenced reads) and choose the analysis parameters (i.e. the quality thresholds for the filtering procedure and the AF range for the He /Ho assignation). An ID is given after completion of the procedure, which is required for accessing the results page. Both the reconstructed sequences in FASTA format and the whole variants in VCF format can be downloaded from the ‘Download section’. In addition, the visualization of the reconstructed sequence is also shown in an interactive graphical interface in the results page. Here the user can explore, for each gene in each genotype, the type and the number of variants that have been incorporated and the effect they produced on the reference protein. General mapping statistics, including IN/OFF targets, as well as the sensitivity of the experiment, are displayed in the same page.</p>                            <h3 scrollto-destination=61463587 id="61463587" class="section-title" >2.5. Use of SUPER-CAP on tomato experimental data</h3>
<p>The reads obtained by sequencing the tomato DNA samples were quality checked using FastQC (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Then, reads were filtered for quality and trimmed with the Trimmomatic tool (version 0.30; options used: –windowsSize 3 –requiredQuality 22 –MINLEN 30).<a reveal-id="dsw050-B39" data-open="dsw050-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> By using the filtered reads, the following procedures were undertaken: reads falling in the designed regions with a PHRED quality &gt;30 and a minimum length of 30 bp were mapped onto the tomato genome SL2.50.<a reveal-id="dsw050-B40" data-open="dsw050-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Then, PCR duplicates and multiple mapped reads were removed by filtering on a meaningful MAPQ score of 30. Once the variants were called, in order to reduce the number of false positives calls, an additional filter was applied and only variants with coverage higher than 6× and a PHRED quality &gt;30 were selected. The total numbers of He and Ho variants, as well as their ratio, were calculated with default parameters (0.4–0.6 as optimal AF range for the He calling). A Neighbour-joining dendrogram was built in TASSEL v.5<a reveal-id="dsw050-B41" data-open="dsw050-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> by using the detected filtered variants, and the related tree was graphically refined using the Fig-tree software (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a>). SNPs and INDELs were also structurally and functionally annotated with SnpEff v4.2<a reveal-id="dsw050-B36" data-open="dsw050-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> using the tomato iTAG2.40 annotation.</p>                            <h3 scrollto-destination=61463589 id="61463589" class="section-title" >2.6. Validation of variants</h3>
<p>To test the reliability of the adopted procedures, a validation of the variants detected was achieved by exploring publicly available data for tomato. In particular, data from a previous genotyping experiment which detected variants for 7,720 SNPs through the SolCAP array,<a reveal-id="dsw050-B30" data-open="dsw050-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw050-B32" data-open="dsw050-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> as well as data concerning the resequencing of 360 tomato genomes,<a reveal-id="dsw050-B42" data-open="dsw050-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> were considered. Of the 44 genotypes used in this study, 37 were common to those used in the SolCAP experiment and 9 in the 360 genome project. The validation was performed by comparing the number and the concordance of shared variations among the common genotypes. Only Ho variations were used for the validation. Intersection of the sets was carried out by intersectBed tool<a reveal-id="dsw050-B38" data-open="dsw050-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and the comparison analysis by using VCFtools.<a reveal-id="dsw050-B43" data-open="dsw050-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p>                            <h3 scrollto-destination=61463591 id="61463591" class="section-title" >2.7. CNV and PAV identification</h3>
<p>The coverage and total number of reads were evaluated for each sample. Alignments were visualized using the IGV browser version 2.3.3 (<a class="link link-uri" href="http://www.broadinstitute.org/igv/" target="_blank">http://www.broadinstitute.org/igv/</a>). CNV was computed following a modified procedure of Schiessl <em>et al.</em> (2014)<a reveal-id="dsw050-B22" data-open="dsw050-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using a normalized read coverage for each captured region. CNV in a given target region was assumed if the ratio of normalized coverage (genotype)/normalized coverage (all genotypes) was &lt;0.5 or &gt;1.5, respectively. PAV was assumed if the ratio was &lt;0.05.</p>                            <h2 scrollto-destination=61463593 id="61463593" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=61463594 id="61463594" class="section-title" >3.1. Sequence capture experiment</h3>
<p>We designed a custom capture probe experiment by using a customizable and cost-effective approach based on Roche-NimbleGen SeqCap EZ technology. This framework helped to set up an <em>in silico</em> approach (SUPER-CAP) with the aim of facilitating further sequence capture studies.</p><p>This study investigated the sequence variation of a group of 378 genes and related regulative regions. The total size of the sequenced region was 2,338,578 bp, of which 1,353,817 bp corresponded to genic regions and 984,761 bp to their related promoter regions. Capturing probes were developed by Roche NimbleGen to target the specific regions. The final design was non-redundant and covered about 92.6% of the targeted regions. The 7.4% of non-covered regions represents those that did not match the minimum parameters for the capture and in the majority of the cases (&gt;95%) these were promoter regions. It has already been reported that the cover design slightly varies depending on different factors, including the commercial technology adopted.<a reveal-id="dsw050-B44" data-open="dsw050-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p><p>After the enrichment of the targeted regions, each capture library was sequenced in one Illumina HiSeq lane. The number of raw reads generated <em>per</em> sample varied between 1,103,352 and 2,683,868 (<a reveal-id="dsw050-T1" data-open="dsw050-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Sequenced reads were analyzed in order to remove low-quality regions. After quality trimming, between 71 and 93% of the reads were retained and then used as input for SUPER-CAP. Data were then statistically evaluated for: (i) alignment rates, the number of mapped reads on the total reads, (ii) specificity, the number of reads that map to the targeted sequence and (iii) sensitivity, the percentage of targeted bases covered by sequence reads. The reads from samples 21A, 1A and 64A showed the highest alignment rates (&gt;98%). In contrast, the lowest alignment rate was observed in genotypes 85A, 79A and 38A (&lt;78%). The alignment success was however independent of the total number of reads. Specificity of the capture was imputed considering the number of reads in the target interval on the number of mapped reads. It showed an average value of 74.80%. The minimum value 67.2% was detected for the genotype 14A and the maximum (77.02%) for the genotype 43A. In addition, considering an extension of flanking regions of 200 bp, the specificity was found to vary only slightly, showing an average increase of &lt;1%. The range, considering flanking regions, was indeed from 67.75 to 77.59%. <div content-id="dsw050-T1" class="table-modal"><div class="table-wrap"><div id="dsw050-T1" data-id="dsw050-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Alignment statistics for the 44 individual captures (Sample ID)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample ID </th><th>Raw reads </th><th>Mapped reads </th><th>Mapped reads Q &gt; 30 </th><th>Mapped reads Q &gt; 30 WD </th><th>Alignment rate (%) </th><th>On target reads (%) </th><th>On target reads + 200 bp (%) </th><th>Mean depth of coverage </th></tr></thead><tbody><tr><td>1A </td><td char=".">1,344,456 </td><td char=".">1,327,567 </td><td char=".">1,269,157 </td><td char=".">1,251,088 </td><td char=".">98.74 </td><td char=".">75.45 </td><td char=".">76.06 </td><td char=".">43.2 </td></tr><tr><td>3A </td><td char=".">1,314,574 </td><td char=".">1,293,498 </td><td char=".">1,234,679 </td><td char=".">1,213,125 </td><td char=".">98.4 </td><td char=".">75.07 </td><td char=".">75.65 </td><td char=".">42.42 </td></tr><tr><td>5A </td><td char=".">1,413,706 </td><td char=".">1,391,398 </td><td char=".">1,331,163 </td><td char=".">1,310,092 </td><td char=".">98.42 </td><td char=".">75.94 </td><td char=".">76.55 </td><td char=".">45.97 </td></tr><tr><td>8A </td><td char=".">1,376,510 </td><td char=".">1,342,280 </td><td char=".">1,279,940 </td><td char=".">1,262,285 </td><td char=".">97.51 </td><td char=".">74.93 </td><td char=".">75.51 </td><td char=".">43.94 </td></tr><tr><td>14A </td><td char=".">1,580,074 </td><td char=".">1,231,066 </td><td char=".">1,142,665 </td><td char=".">1,128,063 </td><td char=".">77.91 </td><td char=".">67.22 </td><td char=".">67.75 </td><td char=".">39.67 </td></tr><tr><td>15A </td><td char=".">1,686,492 </td><td char=".">1,340,263 </td><td char=".">1,274,719 </td><td char=".">1,262,193 </td><td char=".">79.47 </td><td char=".">75.66 </td><td char=".">76.27 </td><td char=".">44.92 </td></tr><tr><td>20A </td><td char=".">1,480,340 </td><td char=".">1,162,656 </td><td char=".">1,101,676 </td><td char=".">1,056,514 </td><td char=".">78.54 </td><td char=".">71.73 </td><td char=".">72.28 </td><td char=".">40.27 </td></tr><tr><td>21A </td><td char=".">1,304,914 </td><td char=".">1,289,290 </td><td char=".">1,228,450 </td><td char=".">1,211,447 </td><td char=".">98.8 </td><td char=".">74.62 </td><td char=".">75.21 </td><td char=".">41.77 </td></tr><tr><td>26A </td><td char=".">1,103,352 </td><td char=".">871,462 </td><td char=".">830,910 </td><td char=".">821,995 </td><td char=".">78.98 </td><td char=".">75.72 </td><td char=".">76.35 </td><td char=".">30.03 </td></tr><tr><td>27A </td><td char=".">1,469,212 </td><td char=".">1,436,034 </td><td char=".">1,365,188 </td><td char=".">1,344,283 </td><td char=".">97.74 </td><td char=".">74.73 </td><td char=".">75.3 </td><td char=".">47.02 </td></tr><tr><td>28A </td><td char=".">1,354,892 </td><td char=".">1,332,311 </td><td char=".">1,264,586 </td><td char=".">1,190,895 </td><td char=".">98.33 </td><td char=".">70.39 </td><td char=".">70.87 </td><td char=".">40.76 </td></tr><tr><td>30A </td><td char=".">1,397,308 </td><td char=".">1,379,189 </td><td char=".">1,319,320 </td><td char=".">1,300,299 </td><td char=".">98.7 </td><td char=".">75.76 </td><td char=".">76.43 </td><td char=".">45.43 </td></tr><tr><td>32A </td><td char=".">1,893,224 </td><td char=".">1,480,474 </td><td char=".">1,410,253 </td><td char=".">1,394,575 </td><td char=".">78.2 </td><td char=".">74.55 </td><td char=".">75.18 </td><td char=".">48.69 </td></tr><tr><td>34A </td><td char=".">1,891,936 </td><td char=".">1,478,312 </td><td char=".">1,411,043 </td><td char=".">1,396,990 </td><td char=".">78.14 </td><td char=".">75.18 </td><td char=".">75.84 </td><td char=".">48.54 </td></tr><tr><td>35A </td><td char=".">1,788,530 </td><td char=".">1,398,636 </td><td char=".">1,333,126 </td><td char=".">1,316,721 </td><td char=".">78.2 </td><td char=".">74.51 </td><td char=".">75.16 </td><td char=".">46.02 </td></tr><tr><td>38A </td><td char=".">1,886,966 </td><td char=".">1,468,035 </td><td char=".">1,399,866 </td><td char=".">1,386,113 </td><td char=".">77.8 </td><td char=".">74.57 </td><td char=".">75.21 </td><td char=".">48.23 </td></tr><tr><td>40A </td><td char=".">2,651,822 </td><td char=".">2,575,073 </td><td char=".">2,463,126 </td><td char=".">2,432,466 </td><td char=".">97.11 </td><td char=".">76.67 </td><td char=".">77.25 </td><td char=".">88.92 </td></tr><tr><td>41A </td><td char=".">2,683,868 </td><td char=".">2,594,090 </td><td char=".">2,482,444 </td><td char=".">2,451,378 </td><td char=".">96.65 </td><td char=".">76.75 </td><td char=".">77.31 </td><td char=".">90.36 </td></tr><tr><td>42A </td><td char=".">1,450,568 </td><td char=".">1,424,424 </td><td char=".">1,348,207 </td><td char=".">1,329,068 </td><td char=".">98.2 </td><td char=".">74.28 </td><td char=".">74.84 </td><td char=".">46.26 </td></tr><tr><td>43A </td><td char=".">2,082,552 </td><td char=".">2,006,847 </td><td char=".">1,921,991 </td><td char=".">1,899,495 </td><td char=".">96.36 </td><td char=".">77.02 </td><td char=".">77.59 </td><td char=".">70.57 </td></tr><tr><td>45A </td><td char=".">2,409,462 </td><td char=".">2,316,282 </td><td char=".">2,221,361 </td><td char=".">2,193,201 </td><td char=".">96.13 </td><td char=".">76.59 </td><td char=".">77.18 </td><td char=".">80.52 </td></tr><tr><td>57A </td><td char=".">1,297,772 </td><td char=".">1,275,552 </td><td char=".">1,216,245 </td><td char=".">1,198,785 </td><td char=".">98.29 </td><td char=".">75 </td><td char=".">75.56 </td><td char=".">42.68 </td></tr><tr><td>64A </td><td char=".">1,446,368 </td><td char=".">1,429,441 </td><td char=".">1,366,044 </td><td char=".">1,346,218 </td><td char=".">98.83 </td><td char=".">75.98 </td><td char=".">76.56 </td><td char=".">47.44 </td></tr><tr><td>66A </td><td char=".">2,267,108 </td><td char=".">2,176,890 </td><td char=".">2,081,441 </td><td char=".">2,057,807 </td><td char=".">96.02 </td><td char=".">76.59 </td><td char=".">77.17 </td><td char=".">74.86 </td></tr><tr><td>70A </td><td char=".">1,372,572 </td><td char=".">1,297,126 </td><td char=".">1,223,744 </td><td char=".">1,206,911 </td><td char=".">94.5 </td><td char=".">72.75 </td><td char=".">73.28 </td><td char=".">41.49 </td></tr><tr><td>75A </td><td char=".">1,799,384 </td><td char=".">1,396,835 </td><td char=".">1,323,958 </td><td char=".">1,312,523 </td><td char=".">77.63 </td><td char=".">73.81 </td><td char=".">74.38 </td><td char=".">45.59 </td></tr><tr><td>78A </td><td char=".">2,060,104 </td><td char=".">1,992,077 </td><td char=".">1,903,906 </td><td char=".">1,876,376 </td><td char=".">96.7 </td><td char=".">76.81 </td><td char=".">77.4 </td><td char=".">72.14 </td></tr><tr><td>79A </td><td char=".">1,666,724 </td><td char=".">1,295,896 </td><td char=".">1,235,321 </td><td char=".">1,224,643 </td><td char=".">77.75 </td><td char=".">75.02 </td><td char=".">75.63 </td><td char=".">44.71 </td></tr><tr><td>85A </td><td char=".">1,879,204 </td><td char=".">1,439,306 </td><td char=".">1,364,666 </td><td char=".">1,344,083 </td><td char=".">76.59 </td><td char=".">73.83 </td><td char=".">74.42 </td><td char=".">47.2 </td></tr><tr><td>87A </td><td char=".">2,342,600 </td><td char=".">2,247,877 </td><td char=".">2,150,105 </td><td char=".">2,124,074 </td><td char=".">95.96 </td><td char=".">76.24 </td><td char=".">76.83 </td><td char=".">77.91 </td></tr><tr><td>92A </td><td char=".">2,247,742 </td><td char=".">2,165,543 </td><td char=".">2,069,466 </td><td char=".">2,043,464 </td><td char=".">96.34 </td><td char=".">75.25 </td><td char=".">75.84 </td><td char=".">73.65 </td></tr><tr><td>93A </td><td char=".">1,877,134 </td><td char=".">1,465,142 </td><td char=".">1,394,154 </td><td char=".">1,379,154 </td><td char=".">78.05 </td><td char=".">74.36 </td><td char=".">74.95 </td><td char=".">48.29 </td></tr><tr><td>94A </td><td char=".">2,105,320 </td><td char=".">2,019,814 </td><td char=".">1,927,953 </td><td char=".">1,904,794 </td><td char=".">95.94 </td><td char=".">75.59 </td><td char=".">76.25 </td><td char=".">68.51 </td></tr><tr><td>97A </td><td char=".">2,489,394 </td><td char=".">2,402,528 </td><td char=".">2,299,398 </td><td char=".">2,270,879 </td><td char=".">96.51 </td><td char=".">76.11 </td><td char=".">76.72 </td><td char=".">82.59 </td></tr><tr><td>99A </td><td char=".">2,479,650 </td><td char=".">2,380,804 </td><td char=".">2,280,336 </td><td char=".">2,251,710 </td><td char=".">96.01 </td><td char=".">76.49 </td><td char=".">77.11 </td><td char=".">81.97 </td></tr><tr><td>102A </td><td char=".">1,760,260 </td><td char=".">1,374,039 </td><td char=".">1,305,068 </td><td char=".">1,293,203 </td><td char=".">78.06 </td><td char=".">73.31 </td><td char=".">73.91 </td><td char=".">44.39 </td></tr><tr><td>103A </td><td char=".">1,969,444 </td><td char=".">1,886,176 </td><td char=".">1,780,373 </td><td char=".">1,747,969 </td><td char=".">95.77 </td><td char=".">71.5 </td><td char=".">72 </td><td char=".">61.9 </td></tr><tr><td>105A </td><td char=".">1,942,196 </td><td char=".">1,516,300 </td><td char=".">1,450,772 </td><td char=".">1,438,620 </td><td char=".">78.07 </td><td char=".">76.3 </td><td char=".">76.9 </td><td char=".">51.32 </td></tr><tr><td>109A </td><td char=".">1,835,360 </td><td char=".">1,428,227 </td><td char=".">1,356,340 </td><td char=".">1,343,895 </td><td char=".">77.82 </td><td char=".">74.8 </td><td char=".">75.39 </td><td char=".">49.28 </td></tr><tr><td>111A </td><td char=".">2,541,684 </td><td char=".">2,006,126 </td><td char=".">1,913,540 </td><td char=".">1,893,325 </td><td char=".">78.93 </td><td char=".">76.22 </td><td char=".">76.82 </td><td char=".">68.21 </td></tr><tr><td>115A </td><td char=".">1,547,164 </td><td char=".">1,229,818 </td><td char=".">1,166,492 </td><td char=".">1,099,375 </td><td char=".">79.49 </td><td char=".">70.75 </td><td char=".">71.3 </td><td char=".">41.5 </td></tr><tr><td>117A </td><td char=".">1,682,246 </td><td char=".">1,333,089 </td><td char=".">1,273,573 </td><td char=".">1,262,226 </td><td char=".">79.24 </td><td char=".">76.42 </td><td char=".">77.03 </td><td char=".">44.83 </td></tr><tr><td>118A </td><td char=".">1,665,304 </td><td char=".">1,320,669 </td><td char=".">1,260,016 </td><td char=".">1,248,760 </td><td char=".">79.3 </td><td char=".">75.09 </td><td char=".">75.72 </td><td char=".">43.63 </td></tr><tr><td>120A </td><td char=".">1,651,964 </td><td char=".">1,308,823 </td><td char=".">1,249,979 </td><td char=".">1,237,454 </td><td char=".">79.23 </td><td char=".">76.03 </td><td char=".">76.64 </td><td char=".">43.62 </td></tr><tr><td><strong>Mean</strong> </td><td char="."><strong>1,806,624</strong> </td><td char="."><strong>1,602,892</strong> </td><td char="."><strong>1,527,880</strong> </td><td char="."><strong>1,505,875</strong> </td><td char="."><strong>88.72</strong> </td><td char="."><strong>74.80</strong> </td><td char="."><strong>75.40</strong> </td><td char="."><strong>53.89</strong> </td></tr></tbody></table></div><div content-id="dsw050-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample ID </th><th>Raw reads </th><th>Mapped reads </th><th>Mapped reads Q &gt; 30 </th><th>Mapped reads Q &gt; 30 WD </th><th>Alignment rate (%) </th><th>On target reads (%) </th><th>On target reads + 200 bp (%) </th><th>Mean depth of coverage </th></tr></thead><tbody><tr><td>1A </td><td char=".">1,344,456 </td><td char=".">1,327,567 </td><td char=".">1,269,157 </td><td char=".">1,251,088 </td><td char=".">98.74 </td><td char=".">75.45 </td><td char=".">76.06 </td><td char=".">43.2 </td></tr><tr><td>3A </td><td char=".">1,314,574 </td><td char=".">1,293,498 </td><td char=".">1,234,679 </td><td char=".">1,213,125 </td><td char=".">98.4 </td><td char=".">75.07 </td><td char=".">75.65 </td><td char=".">42.42 </td></tr><tr><td>5A </td><td char=".">1,413,706 </td><td char=".">1,391,398 </td><td char=".">1,331,163 </td><td char=".">1,310,092 </td><td char=".">98.42 </td><td char=".">75.94 </td><td char=".">76.55 </td><td char=".">45.97 </td></tr><tr><td>8A </td><td char=".">1,376,510 </td><td char=".">1,342,280 </td><td char=".">1,279,940 </td><td char=".">1,262,285 </td><td char=".">97.51 </td><td char=".">74.93 </td><td char=".">75.51 </td><td char=".">43.94 </td></tr><tr><td>14A </td><td char=".">1,580,074 </td><td char=".">1,231,066 </td><td char=".">1,142,665 </td><td char=".">1,128,063 </td><td char=".">77.91 </td><td char=".">67.22 </td><td char=".">67.75 </td><td char=".">39.67 </td></tr><tr><td>15A </td><td char=".">1,686,492 </td><td char=".">1,340,263 </td><td char=".">1,274,719 </td><td char=".">1,262,193 </td><td char=".">79.47 </td><td char=".">75.66 </td><td char=".">76.27 </td><td char=".">44.92 </td></tr><tr><td>20A </td><td char=".">1,480,340 </td><td char=".">1,162,656 </td><td char=".">1,101,676 </td><td char=".">1,056,514 </td><td char=".">78.54 </td><td char=".">71.73 </td><td char=".">72.28 </td><td char=".">40.27 </td></tr><tr><td>21A </td><td char=".">1,304,914 </td><td char=".">1,289,290 </td><td char=".">1,228,450 </td><td char=".">1,211,447 </td><td char=".">98.8 </td><td char=".">74.62 </td><td char=".">75.21 </td><td char=".">41.77 </td></tr><tr><td>26A </td><td char=".">1,103,352 </td><td char=".">871,462 </td><td char=".">830,910 </td><td char=".">821,995 </td><td char=".">78.98 </td><td char=".">75.72 </td><td char=".">76.35 </td><td char=".">30.03 </td></tr><tr><td>27A </td><td char=".">1,469,212 </td><td char=".">1,436,034 </td><td char=".">1,365,188 </td><td char=".">1,344,283 </td><td char=".">97.74 </td><td char=".">74.73 </td><td char=".">75.3 </td><td char=".">47.02 </td></tr><tr><td>28A </td><td char=".">1,354,892 </td><td char=".">1,332,311 </td><td char=".">1,264,586 </td><td char=".">1,190,895 </td><td char=".">98.33 </td><td char=".">70.39 </td><td char=".">70.87 </td><td char=".">40.76 </td></tr><tr><td>30A </td><td char=".">1,397,308 </td><td char=".">1,379,189 </td><td char=".">1,319,320 </td><td char=".">1,300,299 </td><td char=".">98.7 </td><td char=".">75.76 </td><td char=".">76.43 </td><td char=".">45.43 </td></tr><tr><td>32A </td><td char=".">1,893,224 </td><td char=".">1,480,474 </td><td char=".">1,410,253 </td><td char=".">1,394,575 </td><td char=".">78.2 </td><td char=".">74.55 </td><td char=".">75.18 </td><td char=".">48.69 </td></tr><tr><td>34A </td><td char=".">1,891,936 </td><td char=".">1,478,312 </td><td char=".">1,411,043 </td><td char=".">1,396,990 </td><td char=".">78.14 </td><td char=".">75.18 </td><td char=".">75.84 </td><td char=".">48.54 </td></tr><tr><td>35A </td><td char=".">1,788,530 </td><td char=".">1,398,636 </td><td char=".">1,333,126 </td><td char=".">1,316,721 </td><td char=".">78.2 </td><td char=".">74.51 </td><td char=".">75.16 </td><td char=".">46.02 </td></tr><tr><td>38A </td><td char=".">1,886,966 </td><td char=".">1,468,035 </td><td char=".">1,399,866 </td><td char=".">1,386,113 </td><td char=".">77.8 </td><td char=".">74.57 </td><td char=".">75.21 </td><td char=".">48.23 </td></tr><tr><td>40A </td><td char=".">2,651,822 </td><td char=".">2,575,073 </td><td char=".">2,463,126 </td><td char=".">2,432,466 </td><td char=".">97.11 </td><td char=".">76.67 </td><td char=".">77.25 </td><td char=".">88.92 </td></tr><tr><td>41A </td><td char=".">2,683,868 </td><td char=".">2,594,090 </td><td char=".">2,482,444 </td><td char=".">2,451,378 </td><td char=".">96.65 </td><td char=".">76.75 </td><td char=".">77.31 </td><td char=".">90.36 </td></tr><tr><td>42A </td><td char=".">1,450,568 </td><td char=".">1,424,424 </td><td char=".">1,348,207 </td><td char=".">1,329,068 </td><td char=".">98.2 </td><td char=".">74.28 </td><td char=".">74.84 </td><td char=".">46.26 </td></tr><tr><td>43A </td><td char=".">2,082,552 </td><td char=".">2,006,847 </td><td char=".">1,921,991 </td><td char=".">1,899,495 </td><td char=".">96.36 </td><td char=".">77.02 </td><td char=".">77.59 </td><td char=".">70.57 </td></tr><tr><td>45A </td><td char=".">2,409,462 </td><td char=".">2,316,282 </td><td char=".">2,221,361 </td><td char=".">2,193,201 </td><td char=".">96.13 </td><td char=".">76.59 </td><td char=".">77.18 </td><td char=".">80.52 </td></tr><tr><td>57A </td><td char=".">1,297,772 </td><td char=".">1,275,552 </td><td char=".">1,216,245 </td><td char=".">1,198,785 </td><td char=".">98.29 </td><td char=".">75 </td><td char=".">75.56 </td><td char=".">42.68 </td></tr><tr><td>64A </td><td char=".">1,446,368 </td><td char=".">1,429,441 </td><td char=".">1,366,044 </td><td char=".">1,346,218 </td><td char=".">98.83 </td><td char=".">75.98 </td><td char=".">76.56 </td><td char=".">47.44 </td></tr><tr><td>66A </td><td char=".">2,267,108 </td><td char=".">2,176,890 </td><td char=".">2,081,441 </td><td char=".">2,057,807 </td><td char=".">96.02 </td><td char=".">76.59 </td><td char=".">77.17 </td><td char=".">74.86 </td></tr><tr><td>70A </td><td char=".">1,372,572 </td><td char=".">1,297,126 </td><td char=".">1,223,744 </td><td char=".">1,206,911 </td><td char=".">94.5 </td><td char=".">72.75 </td><td char=".">73.28 </td><td char=".">41.49 </td></tr><tr><td>75A </td><td char=".">1,799,384 </td><td char=".">1,396,835 </td><td char=".">1,323,958 </td><td char=".">1,312,523 </td><td char=".">77.63 </td><td char=".">73.81 </td><td char=".">74.38 </td><td char=".">45.59 </td></tr><tr><td>78A </td><td char=".">2,060,104 </td><td char=".">1,992,077 </td><td char=".">1,903,906 </td><td char=".">1,876,376 </td><td char=".">96.7 </td><td char=".">76.81 </td><td char=".">77.4 </td><td char=".">72.14 </td></tr><tr><td>79A </td><td char=".">1,666,724 </td><td char=".">1,295,896 </td><td char=".">1,235,321 </td><td char=".">1,224,643 </td><td char=".">77.75 </td><td char=".">75.02 </td><td char=".">75.63 </td><td char=".">44.71 </td></tr><tr><td>85A </td><td char=".">1,879,204 </td><td char=".">1,439,306 </td><td char=".">1,364,666 </td><td char=".">1,344,083 </td><td char=".">76.59 </td><td char=".">73.83 </td><td char=".">74.42 </td><td char=".">47.2 </td></tr><tr><td>87A </td><td char=".">2,342,600 </td><td char=".">2,247,877 </td><td char=".">2,150,105 </td><td char=".">2,124,074 </td><td char=".">95.96 </td><td char=".">76.24 </td><td char=".">76.83 </td><td char=".">77.91 </td></tr><tr><td>92A </td><td char=".">2,247,742 </td><td char=".">2,165,543 </td><td char=".">2,069,466 </td><td char=".">2,043,464 </td><td char=".">96.34 </td><td char=".">75.25 </td><td char=".">75.84 </td><td char=".">73.65 </td></tr><tr><td>93A </td><td char=".">1,877,134 </td><td char=".">1,465,142 </td><td char=".">1,394,154 </td><td char=".">1,379,154 </td><td char=".">78.05 </td><td char=".">74.36 </td><td char=".">74.95 </td><td char=".">48.29 </td></tr><tr><td>94A </td><td char=".">2,105,320 </td><td char=".">2,019,814 </td><td char=".">1,927,953 </td><td char=".">1,904,794 </td><td char=".">95.94 </td><td char=".">75.59 </td><td char=".">76.25 </td><td char=".">68.51 </td></tr><tr><td>97A </td><td char=".">2,489,394 </td><td char=".">2,402,528 </td><td char=".">2,299,398 </td><td char=".">2,270,879 </td><td char=".">96.51 </td><td char=".">76.11 </td><td char=".">76.72 </td><td char=".">82.59 </td></tr><tr><td>99A </td><td char=".">2,479,650 </td><td char=".">2,380,804 </td><td char=".">2,280,336 </td><td char=".">2,251,710 </td><td char=".">96.01 </td><td char=".">76.49 </td><td char=".">77.11 </td><td char=".">81.97 </td></tr><tr><td>102A </td><td char=".">1,760,260 </td><td char=".">1,374,039 </td><td char=".">1,305,068 </td><td char=".">1,293,203 </td><td char=".">78.06 </td><td char=".">73.31 </td><td char=".">73.91 </td><td char=".">44.39 </td></tr><tr><td>103A </td><td char=".">1,969,444 </td><td char=".">1,886,176 </td><td char=".">1,780,373 </td><td char=".">1,747,969 </td><td char=".">95.77 </td><td char=".">71.5 </td><td char=".">72 </td><td char=".">61.9 </td></tr><tr><td>105A </td><td char=".">1,942,196 </td><td char=".">1,516,300 </td><td char=".">1,450,772 </td><td char=".">1,438,620 </td><td char=".">78.07 </td><td char=".">76.3 </td><td char=".">76.9 </td><td char=".">51.32 </td></tr><tr><td>109A </td><td char=".">1,835,360 </td><td char=".">1,428,227 </td><td char=".">1,356,340 </td><td char=".">1,343,895 </td><td char=".">77.82 </td><td char=".">74.8 </td><td char=".">75.39 </td><td char=".">49.28 </td></tr><tr><td>111A </td><td char=".">2,541,684 </td><td char=".">2,006,126 </td><td char=".">1,913,540 </td><td char=".">1,893,325 </td><td char=".">78.93 </td><td char=".">76.22 </td><td char=".">76.82 </td><td char=".">68.21 </td></tr><tr><td>115A </td><td char=".">1,547,164 </td><td char=".">1,229,818 </td><td char=".">1,166,492 </td><td char=".">1,099,375 </td><td char=".">79.49 </td><td char=".">70.75 </td><td char=".">71.3 </td><td char=".">41.5 </td></tr><tr><td>117A </td><td char=".">1,682,246 </td><td char=".">1,333,089 </td><td char=".">1,273,573 </td><td char=".">1,262,226 </td><td char=".">79.24 </td><td char=".">76.42 </td><td char=".">77.03 </td><td char=".">44.83 </td></tr><tr><td>118A </td><td char=".">1,665,304 </td><td char=".">1,320,669 </td><td char=".">1,260,016 </td><td char=".">1,248,760 </td><td char=".">79.3 </td><td char=".">75.09 </td><td char=".">75.72 </td><td char=".">43.63 </td></tr><tr><td>120A </td><td char=".">1,651,964 </td><td char=".">1,308,823 </td><td char=".">1,249,979 </td><td char=".">1,237,454 </td><td char=".">79.23 </td><td char=".">76.03 </td><td char=".">76.64 </td><td char=".">43.62 </td></tr><tr><td><strong>Mean</strong> </td><td char="."><strong>1,806,624</strong> </td><td char="."><strong>1,602,892</strong> </td><td char="."><strong>1,527,880</strong> </td><td char="."><strong>1,505,875</strong> </td><td char="."><strong>88.72</strong> </td><td char="."><strong>74.80</strong> </td><td char="."><strong>75.40</strong> </td><td char="."><strong>53.89</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The number of raw reads (Raw reads), mapped reads (Mapped reads), mapped reads with quality Q &gt; 30 (Mapped reads Q&gt;30) and without duplications (Mapped reads Q &gt; 30 WD) were reported. For each sample, the alignment rate (Alignment rate), the specificity (On target reads), the specificity including the flanking regions of 200 bp (On target reads + 200 bp) and the average depth of coverage (Mean depth of coverage ) are also reported.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw050-T1" data-id="dsw050-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Alignment statistics for the 44 individual captures (Sample ID)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample ID </th><th>Raw reads </th><th>Mapped reads </th><th>Mapped reads Q &gt; 30 </th><th>Mapped reads Q &gt; 30 WD </th><th>Alignment rate (%) </th><th>On target reads (%) </th><th>On target reads + 200 bp (%) </th><th>Mean depth of coverage </th></tr></thead><tbody><tr><td>1A </td><td char=".">1,344,456 </td><td char=".">1,327,567 </td><td char=".">1,269,157 </td><td char=".">1,251,088 </td><td char=".">98.74 </td><td char=".">75.45 </td><td char=".">76.06 </td><td char=".">43.2 </td></tr><tr><td>3A </td><td char=".">1,314,574 </td><td char=".">1,293,498 </td><td char=".">1,234,679 </td><td char=".">1,213,125 </td><td char=".">98.4 </td><td char=".">75.07 </td><td char=".">75.65 </td><td char=".">42.42 </td></tr><tr><td>5A </td><td char=".">1,413,706 </td><td char=".">1,391,398 </td><td char=".">1,331,163 </td><td char=".">1,310,092 </td><td char=".">98.42 </td><td char=".">75.94 </td><td char=".">76.55 </td><td char=".">45.97 </td></tr><tr><td>8A </td><td char=".">1,376,510 </td><td char=".">1,342,280 </td><td char=".">1,279,940 </td><td char=".">1,262,285 </td><td char=".">97.51 </td><td char=".">74.93 </td><td char=".">75.51 </td><td char=".">43.94 </td></tr><tr><td>14A </td><td char=".">1,580,074 </td><td char=".">1,231,066 </td><td char=".">1,142,665 </td><td char=".">1,128,063 </td><td char=".">77.91 </td><td char=".">67.22 </td><td char=".">67.75 </td><td char=".">39.67 </td></tr><tr><td>15A </td><td char=".">1,686,492 </td><td char=".">1,340,263 </td><td char=".">1,274,719 </td><td char=".">1,262,193 </td><td char=".">79.47 </td><td char=".">75.66 </td><td char=".">76.27 </td><td char=".">44.92 </td></tr><tr><td>20A </td><td char=".">1,480,340 </td><td char=".">1,162,656 </td><td char=".">1,101,676 </td><td char=".">1,056,514 </td><td char=".">78.54 </td><td char=".">71.73 </td><td char=".">72.28 </td><td char=".">40.27 </td></tr><tr><td>21A </td><td char=".">1,304,914 </td><td char=".">1,289,290 </td><td char=".">1,228,450 </td><td char=".">1,211,447 </td><td char=".">98.8 </td><td char=".">74.62 </td><td char=".">75.21 </td><td char=".">41.77 </td></tr><tr><td>26A </td><td char=".">1,103,352 </td><td char=".">871,462 </td><td char=".">830,910 </td><td char=".">821,995 </td><td char=".">78.98 </td><td char=".">75.72 </td><td char=".">76.35 </td><td char=".">30.03 </td></tr><tr><td>27A </td><td char=".">1,469,212 </td><td char=".">1,436,034 </td><td char=".">1,365,188 </td><td char=".">1,344,283 </td><td char=".">97.74 </td><td char=".">74.73 </td><td char=".">75.3 </td><td char=".">47.02 </td></tr><tr><td>28A </td><td char=".">1,354,892 </td><td char=".">1,332,311 </td><td char=".">1,264,586 </td><td char=".">1,190,895 </td><td char=".">98.33 </td><td char=".">70.39 </td><td char=".">70.87 </td><td char=".">40.76 </td></tr><tr><td>30A </td><td char=".">1,397,308 </td><td char=".">1,379,189 </td><td char=".">1,319,320 </td><td char=".">1,300,299 </td><td char=".">98.7 </td><td char=".">75.76 </td><td char=".">76.43 </td><td char=".">45.43 </td></tr><tr><td>32A </td><td char=".">1,893,224 </td><td char=".">1,480,474 </td><td char=".">1,410,253 </td><td char=".">1,394,575 </td><td char=".">78.2 </td><td char=".">74.55 </td><td char=".">75.18 </td><td char=".">48.69 </td></tr><tr><td>34A </td><td char=".">1,891,936 </td><td char=".">1,478,312 </td><td char=".">1,411,043 </td><td char=".">1,396,990 </td><td char=".">78.14 </td><td char=".">75.18 </td><td char=".">75.84 </td><td char=".">48.54 </td></tr><tr><td>35A </td><td char=".">1,788,530 </td><td char=".">1,398,636 </td><td char=".">1,333,126 </td><td char=".">1,316,721 </td><td char=".">78.2 </td><td char=".">74.51 </td><td char=".">75.16 </td><td char=".">46.02 </td></tr><tr><td>38A </td><td char=".">1,886,966 </td><td char=".">1,468,035 </td><td char=".">1,399,866 </td><td char=".">1,386,113 </td><td char=".">77.8 </td><td char=".">74.57 </td><td char=".">75.21 </td><td char=".">48.23 </td></tr><tr><td>40A </td><td char=".">2,651,822 </td><td char=".">2,575,073 </td><td char=".">2,463,126 </td><td char=".">2,432,466 </td><td char=".">97.11 </td><td char=".">76.67 </td><td char=".">77.25 </td><td char=".">88.92 </td></tr><tr><td>41A </td><td char=".">2,683,868 </td><td char=".">2,594,090 </td><td char=".">2,482,444 </td><td char=".">2,451,378 </td><td char=".">96.65 </td><td char=".">76.75 </td><td char=".">77.31 </td><td char=".">90.36 </td></tr><tr><td>42A </td><td char=".">1,450,568 </td><td char=".">1,424,424 </td><td char=".">1,348,207 </td><td char=".">1,329,068 </td><td char=".">98.2 </td><td char=".">74.28 </td><td char=".">74.84 </td><td char=".">46.26 </td></tr><tr><td>43A </td><td char=".">2,082,552 </td><td char=".">2,006,847 </td><td char=".">1,921,991 </td><td char=".">1,899,495 </td><td char=".">96.36 </td><td char=".">77.02 </td><td char=".">77.59 </td><td char=".">70.57 </td></tr><tr><td>45A </td><td char=".">2,409,462 </td><td char=".">2,316,282 </td><td char=".">2,221,361 </td><td char=".">2,193,201 </td><td char=".">96.13 </td><td char=".">76.59 </td><td char=".">77.18 </td><td char=".">80.52 </td></tr><tr><td>57A </td><td char=".">1,297,772 </td><td char=".">1,275,552 </td><td char=".">1,216,245 </td><td char=".">1,198,785 </td><td char=".">98.29 </td><td char=".">75 </td><td char=".">75.56 </td><td char=".">42.68 </td></tr><tr><td>64A </td><td char=".">1,446,368 </td><td char=".">1,429,441 </td><td char=".">1,366,044 </td><td char=".">1,346,218 </td><td char=".">98.83 </td><td char=".">75.98 </td><td char=".">76.56 </td><td char=".">47.44 </td></tr><tr><td>66A </td><td char=".">2,267,108 </td><td char=".">2,176,890 </td><td char=".">2,081,441 </td><td char=".">2,057,807 </td><td char=".">96.02 </td><td char=".">76.59 </td><td char=".">77.17 </td><td char=".">74.86 </td></tr><tr><td>70A </td><td char=".">1,372,572 </td><td char=".">1,297,126 </td><td char=".">1,223,744 </td><td char=".">1,206,911 </td><td char=".">94.5 </td><td char=".">72.75 </td><td char=".">73.28 </td><td char=".">41.49 </td></tr><tr><td>75A </td><td char=".">1,799,384 </td><td char=".">1,396,835 </td><td char=".">1,323,958 </td><td char=".">1,312,523 </td><td char=".">77.63 </td><td char=".">73.81 </td><td char=".">74.38 </td><td char=".">45.59 </td></tr><tr><td>78A </td><td char=".">2,060,104 </td><td char=".">1,992,077 </td><td char=".">1,903,906 </td><td char=".">1,876,376 </td><td char=".">96.7 </td><td char=".">76.81 </td><td char=".">77.4 </td><td char=".">72.14 </td></tr><tr><td>79A </td><td char=".">1,666,724 </td><td char=".">1,295,896 </td><td char=".">1,235,321 </td><td char=".">1,224,643 </td><td char=".">77.75 </td><td char=".">75.02 </td><td char=".">75.63 </td><td char=".">44.71 </td></tr><tr><td>85A </td><td char=".">1,879,204 </td><td char=".">1,439,306 </td><td char=".">1,364,666 </td><td char=".">1,344,083 </td><td char=".">76.59 </td><td char=".">73.83 </td><td char=".">74.42 </td><td char=".">47.2 </td></tr><tr><td>87A </td><td char=".">2,342,600 </td><td char=".">2,247,877 </td><td char=".">2,150,105 </td><td char=".">2,124,074 </td><td char=".">95.96 </td><td char=".">76.24 </td><td char=".">76.83 </td><td char=".">77.91 </td></tr><tr><td>92A </td><td char=".">2,247,742 </td><td char=".">2,165,543 </td><td char=".">2,069,466 </td><td char=".">2,043,464 </td><td char=".">96.34 </td><td char=".">75.25 </td><td char=".">75.84 </td><td char=".">73.65 </td></tr><tr><td>93A </td><td char=".">1,877,134 </td><td char=".">1,465,142 </td><td char=".">1,394,154 </td><td char=".">1,379,154 </td><td char=".">78.05 </td><td char=".">74.36 </td><td char=".">74.95 </td><td char=".">48.29 </td></tr><tr><td>94A </td><td char=".">2,105,320 </td><td char=".">2,019,814 </td><td char=".">1,927,953 </td><td char=".">1,904,794 </td><td char=".">95.94 </td><td char=".">75.59 </td><td char=".">76.25 </td><td char=".">68.51 </td></tr><tr><td>97A </td><td char=".">2,489,394 </td><td char=".">2,402,528 </td><td char=".">2,299,398 </td><td char=".">2,270,879 </td><td char=".">96.51 </td><td char=".">76.11 </td><td char=".">76.72 </td><td char=".">82.59 </td></tr><tr><td>99A </td><td char=".">2,479,650 </td><td char=".">2,380,804 </td><td char=".">2,280,336 </td><td char=".">2,251,710 </td><td char=".">96.01 </td><td char=".">76.49 </td><td char=".">77.11 </td><td char=".">81.97 </td></tr><tr><td>102A </td><td char=".">1,760,260 </td><td char=".">1,374,039 </td><td char=".">1,305,068 </td><td char=".">1,293,203 </td><td char=".">78.06 </td><td char=".">73.31 </td><td char=".">73.91 </td><td char=".">44.39 </td></tr><tr><td>103A </td><td char=".">1,969,444 </td><td char=".">1,886,176 </td><td char=".">1,780,373 </td><td char=".">1,747,969 </td><td char=".">95.77 </td><td char=".">71.5 </td><td char=".">72 </td><td char=".">61.9 </td></tr><tr><td>105A </td><td char=".">1,942,196 </td><td char=".">1,516,300 </td><td char=".">1,450,772 </td><td char=".">1,438,620 </td><td char=".">78.07 </td><td char=".">76.3 </td><td char=".">76.9 </td><td char=".">51.32 </td></tr><tr><td>109A </td><td char=".">1,835,360 </td><td char=".">1,428,227 </td><td char=".">1,356,340 </td><td char=".">1,343,895 </td><td char=".">77.82 </td><td char=".">74.8 </td><td char=".">75.39 </td><td char=".">49.28 </td></tr><tr><td>111A </td><td char=".">2,541,684 </td><td char=".">2,006,126 </td><td char=".">1,913,540 </td><td char=".">1,893,325 </td><td char=".">78.93 </td><td char=".">76.22 </td><td char=".">76.82 </td><td char=".">68.21 </td></tr><tr><td>115A </td><td char=".">1,547,164 </td><td char=".">1,229,818 </td><td char=".">1,166,492 </td><td char=".">1,099,375 </td><td char=".">79.49 </td><td char=".">70.75 </td><td char=".">71.3 </td><td char=".">41.5 </td></tr><tr><td>117A </td><td char=".">1,682,246 </td><td char=".">1,333,089 </td><td char=".">1,273,573 </td><td char=".">1,262,226 </td><td char=".">79.24 </td><td char=".">76.42 </td><td char=".">77.03 </td><td char=".">44.83 </td></tr><tr><td>118A </td><td char=".">1,665,304 </td><td char=".">1,320,669 </td><td char=".">1,260,016 </td><td char=".">1,248,760 </td><td char=".">79.3 </td><td char=".">75.09 </td><td char=".">75.72 </td><td char=".">43.63 </td></tr><tr><td>120A </td><td char=".">1,651,964 </td><td char=".">1,308,823 </td><td char=".">1,249,979 </td><td char=".">1,237,454 </td><td char=".">79.23 </td><td char=".">76.03 </td><td char=".">76.64 </td><td char=".">43.62 </td></tr><tr><td><strong>Mean</strong> </td><td char="."><strong>1,806,624</strong> </td><td char="."><strong>1,602,892</strong> </td><td char="."><strong>1,527,880</strong> </td><td char="."><strong>1,505,875</strong> </td><td char="."><strong>88.72</strong> </td><td char="."><strong>74.80</strong> </td><td char="."><strong>75.40</strong> </td><td char="."><strong>53.89</strong> </td></tr></tbody></table></div><div content-id="dsw050-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample ID </th><th>Raw reads </th><th>Mapped reads </th><th>Mapped reads Q &gt; 30 </th><th>Mapped reads Q &gt; 30 WD </th><th>Alignment rate (%) </th><th>On target reads (%) </th><th>On target reads + 200 bp (%) </th><th>Mean depth of coverage </th></tr></thead><tbody><tr><td>1A </td><td char=".">1,344,456 </td><td char=".">1,327,567 </td><td char=".">1,269,157 </td><td char=".">1,251,088 </td><td char=".">98.74 </td><td char=".">75.45 </td><td char=".">76.06 </td><td char=".">43.2 </td></tr><tr><td>3A </td><td char=".">1,314,574 </td><td char=".">1,293,498 </td><td char=".">1,234,679 </td><td char=".">1,213,125 </td><td char=".">98.4 </td><td char=".">75.07 </td><td char=".">75.65 </td><td char=".">42.42 </td></tr><tr><td>5A </td><td char=".">1,413,706 </td><td char=".">1,391,398 </td><td char=".">1,331,163 </td><td char=".">1,310,092 </td><td char=".">98.42 </td><td char=".">75.94 </td><td char=".">76.55 </td><td char=".">45.97 </td></tr><tr><td>8A </td><td char=".">1,376,510 </td><td char=".">1,342,280 </td><td char=".">1,279,940 </td><td char=".">1,262,285 </td><td char=".">97.51 </td><td char=".">74.93 </td><td char=".">75.51 </td><td char=".">43.94 </td></tr><tr><td>14A </td><td char=".">1,580,074 </td><td char=".">1,231,066 </td><td char=".">1,142,665 </td><td char=".">1,128,063 </td><td char=".">77.91 </td><td char=".">67.22 </td><td char=".">67.75 </td><td char=".">39.67 </td></tr><tr><td>15A </td><td char=".">1,686,492 </td><td char=".">1,340,263 </td><td char=".">1,274,719 </td><td char=".">1,262,193 </td><td char=".">79.47 </td><td char=".">75.66 </td><td char=".">76.27 </td><td char=".">44.92 </td></tr><tr><td>20A </td><td char=".">1,480,340 </td><td char=".">1,162,656 </td><td char=".">1,101,676 </td><td char=".">1,056,514 </td><td char=".">78.54 </td><td char=".">71.73 </td><td char=".">72.28 </td><td char=".">40.27 </td></tr><tr><td>21A </td><td char=".">1,304,914 </td><td char=".">1,289,290 </td><td char=".">1,228,450 </td><td char=".">1,211,447 </td><td char=".">98.8 </td><td char=".">74.62 </td><td char=".">75.21 </td><td char=".">41.77 </td></tr><tr><td>26A </td><td char=".">1,103,352 </td><td char=".">871,462 </td><td char=".">830,910 </td><td char=".">821,995 </td><td char=".">78.98 </td><td char=".">75.72 </td><td char=".">76.35 </td><td char=".">30.03 </td></tr><tr><td>27A </td><td char=".">1,469,212 </td><td char=".">1,436,034 </td><td char=".">1,365,188 </td><td char=".">1,344,283 </td><td char=".">97.74 </td><td char=".">74.73 </td><td char=".">75.3 </td><td char=".">47.02 </td></tr><tr><td>28A </td><td char=".">1,354,892 </td><td char=".">1,332,311 </td><td char=".">1,264,586 </td><td char=".">1,190,895 </td><td char=".">98.33 </td><td char=".">70.39 </td><td char=".">70.87 </td><td char=".">40.76 </td></tr><tr><td>30A </td><td char=".">1,397,308 </td><td char=".">1,379,189 </td><td char=".">1,319,320 </td><td char=".">1,300,299 </td><td char=".">98.7 </td><td char=".">75.76 </td><td char=".">76.43 </td><td char=".">45.43 </td></tr><tr><td>32A </td><td char=".">1,893,224 </td><td char=".">1,480,474 </td><td char=".">1,410,253 </td><td char=".">1,394,575 </td><td char=".">78.2 </td><td char=".">74.55 </td><td char=".">75.18 </td><td char=".">48.69 </td></tr><tr><td>34A </td><td char=".">1,891,936 </td><td char=".">1,478,312 </td><td char=".">1,411,043 </td><td char=".">1,396,990 </td><td char=".">78.14 </td><td char=".">75.18 </td><td char=".">75.84 </td><td char=".">48.54 </td></tr><tr><td>35A </td><td char=".">1,788,530 </td><td char=".">1,398,636 </td><td char=".">1,333,126 </td><td char=".">1,316,721 </td><td char=".">78.2 </td><td char=".">74.51 </td><td char=".">75.16 </td><td char=".">46.02 </td></tr><tr><td>38A </td><td char=".">1,886,966 </td><td char=".">1,468,035 </td><td char=".">1,399,866 </td><td char=".">1,386,113 </td><td char=".">77.8 </td><td char=".">74.57 </td><td char=".">75.21 </td><td char=".">48.23 </td></tr><tr><td>40A </td><td char=".">2,651,822 </td><td char=".">2,575,073 </td><td char=".">2,463,126 </td><td char=".">2,432,466 </td><td char=".">97.11 </td><td char=".">76.67 </td><td char=".">77.25 </td><td char=".">88.92 </td></tr><tr><td>41A </td><td char=".">2,683,868 </td><td char=".">2,594,090 </td><td char=".">2,482,444 </td><td char=".">2,451,378 </td><td char=".">96.65 </td><td char=".">76.75 </td><td char=".">77.31 </td><td char=".">90.36 </td></tr><tr><td>42A </td><td char=".">1,450,568 </td><td char=".">1,424,424 </td><td char=".">1,348,207 </td><td char=".">1,329,068 </td><td char=".">98.2 </td><td char=".">74.28 </td><td char=".">74.84 </td><td char=".">46.26 </td></tr><tr><td>43A </td><td char=".">2,082,552 </td><td char=".">2,006,847 </td><td char=".">1,921,991 </td><td char=".">1,899,495 </td><td char=".">96.36 </td><td char=".">77.02 </td><td char=".">77.59 </td><td char=".">70.57 </td></tr><tr><td>45A </td><td char=".">2,409,462 </td><td char=".">2,316,282 </td><td char=".">2,221,361 </td><td char=".">2,193,201 </td><td char=".">96.13 </td><td char=".">76.59 </td><td char=".">77.18 </td><td char=".">80.52 </td></tr><tr><td>57A </td><td char=".">1,297,772 </td><td char=".">1,275,552 </td><td char=".">1,216,245 </td><td char=".">1,198,785 </td><td char=".">98.29 </td><td char=".">75 </td><td char=".">75.56 </td><td char=".">42.68 </td></tr><tr><td>64A </td><td char=".">1,446,368 </td><td char=".">1,429,441 </td><td char=".">1,366,044 </td><td char=".">1,346,218 </td><td char=".">98.83 </td><td char=".">75.98 </td><td char=".">76.56 </td><td char=".">47.44 </td></tr><tr><td>66A </td><td char=".">2,267,108 </td><td char=".">2,176,890 </td><td char=".">2,081,441 </td><td char=".">2,057,807 </td><td char=".">96.02 </td><td char=".">76.59 </td><td char=".">77.17 </td><td char=".">74.86 </td></tr><tr><td>70A </td><td char=".">1,372,572 </td><td char=".">1,297,126 </td><td char=".">1,223,744 </td><td char=".">1,206,911 </td><td char=".">94.5 </td><td char=".">72.75 </td><td char=".">73.28 </td><td char=".">41.49 </td></tr><tr><td>75A </td><td char=".">1,799,384 </td><td char=".">1,396,835 </td><td char=".">1,323,958 </td><td char=".">1,312,523 </td><td char=".">77.63 </td><td char=".">73.81 </td><td char=".">74.38 </td><td char=".">45.59 </td></tr><tr><td>78A </td><td char=".">2,060,104 </td><td char=".">1,992,077 </td><td char=".">1,903,906 </td><td char=".">1,876,376 </td><td char=".">96.7 </td><td char=".">76.81 </td><td char=".">77.4 </td><td char=".">72.14 </td></tr><tr><td>79A </td><td char=".">1,666,724 </td><td char=".">1,295,896 </td><td char=".">1,235,321 </td><td char=".">1,224,643 </td><td char=".">77.75 </td><td char=".">75.02 </td><td char=".">75.63 </td><td char=".">44.71 </td></tr><tr><td>85A </td><td char=".">1,879,204 </td><td char=".">1,439,306 </td><td char=".">1,364,666 </td><td char=".">1,344,083 </td><td char=".">76.59 </td><td char=".">73.83 </td><td char=".">74.42 </td><td char=".">47.2 </td></tr><tr><td>87A </td><td char=".">2,342,600 </td><td char=".">2,247,877 </td><td char=".">2,150,105 </td><td char=".">2,124,074 </td><td char=".">95.96 </td><td char=".">76.24 </td><td char=".">76.83 </td><td char=".">77.91 </td></tr><tr><td>92A </td><td char=".">2,247,742 </td><td char=".">2,165,543 </td><td char=".">2,069,466 </td><td char=".">2,043,464 </td><td char=".">96.34 </td><td char=".">75.25 </td><td char=".">75.84 </td><td char=".">73.65 </td></tr><tr><td>93A </td><td char=".">1,877,134 </td><td char=".">1,465,142 </td><td char=".">1,394,154 </td><td char=".">1,379,154 </td><td char=".">78.05 </td><td char=".">74.36 </td><td char=".">74.95 </td><td char=".">48.29 </td></tr><tr><td>94A </td><td char=".">2,105,320 </td><td char=".">2,019,814 </td><td char=".">1,927,953 </td><td char=".">1,904,794 </td><td char=".">95.94 </td><td char=".">75.59 </td><td char=".">76.25 </td><td char=".">68.51 </td></tr><tr><td>97A </td><td char=".">2,489,394 </td><td char=".">2,402,528 </td><td char=".">2,299,398 </td><td char=".">2,270,879 </td><td char=".">96.51 </td><td char=".">76.11 </td><td char=".">76.72 </td><td char=".">82.59 </td></tr><tr><td>99A </td><td char=".">2,479,650 </td><td char=".">2,380,804 </td><td char=".">2,280,336 </td><td char=".">2,251,710 </td><td char=".">96.01 </td><td char=".">76.49 </td><td char=".">77.11 </td><td char=".">81.97 </td></tr><tr><td>102A </td><td char=".">1,760,260 </td><td char=".">1,374,039 </td><td char=".">1,305,068 </td><td char=".">1,293,203 </td><td char=".">78.06 </td><td char=".">73.31 </td><td char=".">73.91 </td><td char=".">44.39 </td></tr><tr><td>103A </td><td char=".">1,969,444 </td><td char=".">1,886,176 </td><td char=".">1,780,373 </td><td char=".">1,747,969 </td><td char=".">95.77 </td><td char=".">71.5 </td><td char=".">72 </td><td char=".">61.9 </td></tr><tr><td>105A </td><td char=".">1,942,196 </td><td char=".">1,516,300 </td><td char=".">1,450,772 </td><td char=".">1,438,620 </td><td char=".">78.07 </td><td char=".">76.3 </td><td char=".">76.9 </td><td char=".">51.32 </td></tr><tr><td>109A </td><td char=".">1,835,360 </td><td char=".">1,428,227 </td><td char=".">1,356,340 </td><td char=".">1,343,895 </td><td char=".">77.82 </td><td char=".">74.8 </td><td char=".">75.39 </td><td char=".">49.28 </td></tr><tr><td>111A </td><td char=".">2,541,684 </td><td char=".">2,006,126 </td><td char=".">1,913,540 </td><td char=".">1,893,325 </td><td char=".">78.93 </td><td char=".">76.22 </td><td char=".">76.82 </td><td char=".">68.21 </td></tr><tr><td>115A </td><td char=".">1,547,164 </td><td char=".">1,229,818 </td><td char=".">1,166,492 </td><td char=".">1,099,375 </td><td char=".">79.49 </td><td char=".">70.75 </td><td char=".">71.3 </td><td char=".">41.5 </td></tr><tr><td>117A </td><td char=".">1,682,246 </td><td char=".">1,333,089 </td><td char=".">1,273,573 </td><td char=".">1,262,226 </td><td char=".">79.24 </td><td char=".">76.42 </td><td char=".">77.03 </td><td char=".">44.83 </td></tr><tr><td>118A </td><td char=".">1,665,304 </td><td char=".">1,320,669 </td><td char=".">1,260,016 </td><td char=".">1,248,760 </td><td char=".">79.3 </td><td char=".">75.09 </td><td char=".">75.72 </td><td char=".">43.63 </td></tr><tr><td>120A </td><td char=".">1,651,964 </td><td char=".">1,308,823 </td><td char=".">1,249,979 </td><td char=".">1,237,454 </td><td char=".">79.23 </td><td char=".">76.03 </td><td char=".">76.64 </td><td char=".">43.62 </td></tr><tr><td><strong>Mean</strong> </td><td char="."><strong>1,806,624</strong> </td><td char="."><strong>1,602,892</strong> </td><td char="."><strong>1,527,880</strong> </td><td char="."><strong>1,505,875</strong> </td><td char="."><strong>88.72</strong> </td><td char="."><strong>74.80</strong> </td><td char="."><strong>75.40</strong> </td><td char="."><strong>53.89</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The number of raw reads (Raw reads), mapped reads (Mapped reads), mapped reads with quality Q &gt; 30 (Mapped reads Q&gt;30) and without duplications (Mapped reads Q &gt; 30 WD) were reported. For each sample, the alignment rate (Alignment rate), the specificity (On target reads), the specificity including the flanking regions of 200 bp (On target reads + 200 bp) and the average depth of coverage (Mean depth of coverage ) are also reported.</p></div></div></div></div></p><p>Even though investigators are seeking experimental designs that generate robust scientific findings at the lowest sequencing cost, an adequate depth of coverage is very important in order to reduce the variant-error rate and the assembly gaps, and to obtain the correct call of variants, in particular for the He ones.<a reveal-id="dsw050-B45" data-open="dsw050-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> It has indeed been demonstrated that for short Illumina reads a coverage comprised between 30× and 55× appears necessary to correctly identify SNVs and small INDELs with a proper degree of reliability.<a reveal-id="dsw050-B45" data-open="dsw050-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsw050-B46" data-open="dsw050-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a></p><p>In our experiment, the normalized mean coverage of the total targeted regions showed an average value of 53.89×, allowing very accurate detection of variants. Analysing the sensitivity of the experiment (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), which shows how well the targeted region is covered for reads at depths from 1× to the max depth (&gt;150×), we observed that at a minimum coverage of 1× about 95% of the designed target regions resulted completely covered. Between 91 and 94% of the target was covered when a minimum coverage of ten reads was applied. At 20×, 76–93% of the target regions was still covered.</p><p>Although Sequence Capture experiments represent a promising technology, only few studies have used Sequence Capture in plants (&lt;20 in PubMed in April 2016). Probably the lacking of straightforward procedures for data analysis makes this technology still emerging. For this reason, SUPER-CAP, representing a user-friendly tool, could boost the use of this technology also for plant organisms. Moreover, in this study, the high rates reached for the capture parameters, in particular for the specificity (&gt;74%), highlighted the high efficiency reached by combing an accurate probe design with a high performant aligner included in the pipeline (BWA), for which high confidence of sequence alignments has been already reported.<a reveal-id="dsw050-B19" data-open="dsw050-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> This makes the performances obtained in this study in terms of alignment rate and sensitivity in line with performances reported for other organisms, for example humans.<a reveal-id="dsw050-B44" data-open="dsw050-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p>                            <h3 scrollto-destination=61463601 id="61463601" class="section-title" >3.2. Variant calling and reconstruction of private sequences</h3>
<div class="block-child-p">SUPER-CAP allowed to detect 25,654 unfiltered variants in the targeted regions considered. Then the filtering procedure allowed to reduce the number of false positives/low-quality variants. Only variants exhibiting coverage higher than 6× and PHRED quality higher than 30 were selected. Even if a different threshold could be applied to the filtering procedure, we observed that in our experiment a depth coverage of six represented the minimum coverage to maximize the PHRED quality (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Subsequent criteria were adopted to appropriately detect the number of He variants. A preliminary survey was carried out in order to estimate the number of He variants in accordance with the range of the AF (AF of mapped reads for each locus) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span><strong>).</strong> As expected, an inverse correlation between the range of AF and number of He variants was observed. This indicated that the set-up of the AF-range deeply affects the estimation of the He variants in the calling procedure. In order to make this sensitive phase of the variant calling easily customizable, we allow the SUPER-CAP user to set their own AF range. In this study, we set an AF range of 0.4-0.6 for the He variants, in line with a previous study in tomato.<a reveal-id="dsw050-B10" data-open="dsw050-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> At this threshold, 86,120 variants in 14,116 sites were scored. Only 5% (4,970) of the total variants in 2,543 sites resulted to be He. Looking at the distribution of the He variants, we observed that they were not uniformly distributed across the chromosomes and genotypes. Chromosome 11 harboured the highest number of He variants per Mbp of captured region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) while chromosomes 10 and 0 harboured the lowest number. Considering the 44 samples analysed, the heterozygosity ranged from 1.1 to 47% (<a reveal-id="dsw050-F2" data-open="dsw050-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), with a minimum value for sample 99A and a maximum value for sample 85A. Sample 85A showed an unexpected high percentage of residual heterozygosity, suggesting a possible sampling error of the plant material for this genotype (which could be mixed with other genotypes or to a residual segregation during the germoplasm conservation). For this reason, the sample was excluded from subsequent analyses. <div data-id="dsw050-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f2.png?Expires=2147483647&amp;Signature=NNIBghKrQXzI5A6QRDjEEzE6i2aaGDoNkZCASi84rkR9N-5YX-fRTgLiEErOTS58fpet4ohYO11QtR3ZDYxryzexQrcMfS-aFzcLgfVSFPJtvrwlAzau7fzStbAXQVKepH-jMUHaII--LgCLQIv~wj9qwe3L6VPYe5Hq10kIQqYWik6at1TVIRqkLQt9-5bKmEUB6UDbdckM7~TPWsgzvibNOvRDqvEXB3SONz8HAXM9RlvWIWtSoCFtBV5UHtZwLLLukDc2mGzWsDKP8VhC2HHGTS1fzwMJtV72Q8~6DHbDOyReXh6Ay6Lt~O8IyZI7-81dvR4vQ6U-EfhlZiVA6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of variants detected for each genotype according to the type of variants. Proportion of Ho and He SNP and INDEL are reported as well as the number of private variants. A dendrogram built by using the complete set of variants shows the relationship among the genotypes." path-from-xml="dsw050f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463602/dsw050f2.tif" path-from-xml="dsw050f2.tif" target="_blank">View large</a><a section="61463602" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f2.png?Expires=2147483647&amp;Signature=oNWvXWilvovHb32W0ceFM-~7IjFZJo64KPFAbEU7r6GbEiILnqp8z6Frb9YiQHwJBdaBE3IOJFKhzWQPDyBq8p4s8V4uh7dQ3W~9qay6uGhrU-MgcoJrB6Cx6ERzyLnt~GOjH5imtZS74XOuRPS0oe4YJwJ56PNkVYukpZpQATn~1S210FnWWHOZlTGYo8OxT-fJ-BAOAEWsX9WBjyY7WfjbfXa~KoCuMtonfQQPZOwZABYuTPUAtjMi~VhiARS-AF0UCfjZcgXd0DVYOJPAn9D7rHnFfMIFYkOOgLX8Tmyksd~HNKj78uYpCascw27wBMRvMM-56fMp~uvLZGFuhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463602&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of variants detected for each genotype according to the type of variants. Proportion of Ho and He SNP and INDEL are reported as well as the number of private variants. A dendrogram built by using the complete set of variants shows the relationship among the genotypes.</p></div></div><div content-id="dsw050-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f2.png?Expires=2147483647&amp;Signature=NNIBghKrQXzI5A6QRDjEEzE6i2aaGDoNkZCASi84rkR9N-5YX-fRTgLiEErOTS58fpet4ohYO11QtR3ZDYxryzexQrcMfS-aFzcLgfVSFPJtvrwlAzau7fzStbAXQVKepH-jMUHaII--LgCLQIv~wj9qwe3L6VPYe5Hq10kIQqYWik6at1TVIRqkLQt9-5bKmEUB6UDbdckM7~TPWsgzvibNOvRDqvEXB3SONz8HAXM9RlvWIWtSoCFtBV5UHtZwLLLukDc2mGzWsDKP8VhC2HHGTS1fzwMJtV72Q8~6DHbDOyReXh6Ay6Lt~O8IyZI7-81dvR4vQ6U-EfhlZiVA6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of variants detected for each genotype according to the type of variants. Proportion of Ho and He SNP and INDEL are reported as well as the number of private variants. A dendrogram built by using the complete set of variants shows the relationship among the genotypes." path-from-xml="dsw050f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463602/dsw050f2.tif" path-from-xml="dsw050f2.tif" target="_blank">View large</a><a section="61463602" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f2.png?Expires=2147483647&amp;Signature=oNWvXWilvovHb32W0ceFM-~7IjFZJo64KPFAbEU7r6GbEiILnqp8z6Frb9YiQHwJBdaBE3IOJFKhzWQPDyBq8p4s8V4uh7dQ3W~9qay6uGhrU-MgcoJrB6Cx6ERzyLnt~GOjH5imtZS74XOuRPS0oe4YJwJ56PNkVYukpZpQATn~1S210FnWWHOZlTGYo8OxT-fJ-BAOAEWsX9WBjyY7WfjbfXa~KoCuMtonfQQPZOwZABYuTPUAtjMi~VhiARS-AF0UCfjZcgXd0DVYOJPAn9D7rHnFfMIFYkOOgLX8Tmyksd~HNKj78uYpCascw27wBMRvMM-56fMp~uvLZGFuhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463602&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of variants detected for each genotype according to the type of variants. Proportion of Ho and He SNP and INDEL are reported as well as the number of private variants. A dendrogram built by using the complete set of variants shows the relationship among the genotypes.</p></div></div></div><p><a reveal-id="dsw050-F2" data-open="dsw050-F2" class="link link-reveal link-table xref-fig">Figure 2</a> shows the number of variants detected for each sample, with respect to the proportion of SNPs and INDELs at both Ho and He level. Although INDELs are present at lower rates than SNPs, small INDELs represent functionally important types of genomic variation.<a reveal-id="dsw050-B47" data-open="dsw050-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsw050-B48" data-open="dsw050-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> It is reported that, compared with other sequencing technologies, Illumina does not appear to have high error rates in homopolymer regions.<a reveal-id="dsw050-B49" data-open="dsw050-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> However, in this study the INDELs surveyed were obtained after discarding positions overlapping with homopolymers &gt;7 bp, both in the tomato reference and in resequencing sequences. This allowed to avoid possible errors due to technical bias and enabled to collect a final high-quality set of variants.</p><p>As a whole, the sample with the highest number of variants was 42A, which harbours &gt;4,000 SNPs and about 1,000 INDELs. The sample with the lowest number of variants was 102A which exhibits 1,123 variants between SNPs and INDELs. On average, 1,357 SNPs <em>per</em> sample were scored, of which 6.8% turned out to be He. The average of INDELs <em>per</em> sample turned out to be about 600, and only 3% were He.</p><p>The high depth of coverage reached in this experiment supported the detection of rare variants. It has been suggested that rare or low-frequency variants, which are not fully captured using other conventional genotyping technology, could largely contribute to explain the effects of some traits<a reveal-id="dsw050-B50" data-open="dsw050-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> or to characterize their genetic architecture.<a reveal-id="dsw050-B51" data-open="dsw050-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> In this study 5,441 Ho and 442 He genotype-specific variants were identified. Among these, 1,965 Ho and 216 He variants resulted still private variants when compared with external public repository for SNPs in tomato (360 tomato genomes<a reveal-id="dsw050-B42" data-open="dsw050-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>). This could be of interest when designing specific SNP markers or for studying the potential functionality of specific haplotypes in the landraces collection used in this survey. However, the rare variants identified alone do not seem to explain the phylogenetic differences observed (<a reveal-id="dsw050-F2" data-open="dsw050-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The genetic divergence was associated with the high number of private alleles only in a few samples (i.e. 103A and 97A).</p><p>The possibility of reconstructing private sequences from high-throughput resequencing data is still a challenge today.<a reveal-id="dsw050-B28" data-open="dsw050-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> For this reason, the last step in the SUPER-CAP workflow allows to specifically insert the detected variants into the sequence of each sample. This step, which integrates the single information of the variants in a genotype-specific gene-based view, enables to obtain a necessary starting point for further functional validation analyses. In this study, a total of 16,254 regions (378*43), each including the gene and the related promoter region, were reconstructed. In order to easily exploit the combined effect of multiple variants in the same protein, reconstruction of the CDS was also performed. Future works, benefitting from this effort, will evaluate the combined effect of the variants for each gene in order to find relevant association with antioxidant content in tomato.</p>                            <h3 scrollto-destination=61463607 id="61463607" class="section-title" >3.3. Validation of variants</h3>
<p>In order to validate the variants detected, we performed a survey by exploring the available resources already present for tomato. About 8,600 variant sites detected in the present study overlapped with those reported in the genome resequencing data of 360 tomatoes.<a reveal-id="dsw050-B42" data-open="dsw050-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Most of the non-overlapping variants were excluded since they classified as rare variants in the experiment, being present only in specific samples. Moreover, by using nine common accessions, an analysis on the concordance of the variant calling was performed. Each of the nine samples shared a different number of variants, ranging from 191 to 985. Taking into account the Ho alternative variants only, an average of 95% concordance was found. Another validation was carried out by comparing SNPs detected on a tomato population genotyped using a SolCAP arrays,<a reveal-id="dsw050-B30" data-open="dsw050-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw050-B32" data-open="dsw050-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> which included 7,720 SNPs. Only 170 of the 7,720 SNPs fall in the targeted region considered in the present study, since the targeted regions represent ∼1% (378/34,725) of the tomato genes. Of these 170 SNPs, a subset of 97 belongs to common accessions and was used for validation. On average, a concordance &gt;95% was observed. In most cases the discrepancies were genotyped as He either on the SolCAP array (2.5%) or in the Sequence capture (0.6%). When excluding the He cases, the similarity increased to 98%.</p>                            <h3 scrollto-destination=61463609 id="61463609" class="section-title" >3.4. CNV and PAV</h3>
<p>CNV and PAV have been recently reported as sources of important phenotypic variation in plants.<a reveal-id="dsw050-B52 dsw050-B53 dsw050-B54" data-open="dsw050-B52 dsw050-B53 dsw050-B54" class="link link-ref link-reveal xref-bibr"><sup>52–54</sup></a> The adequate depth coverage reached in this study allowed to hypothesize on the possible structural variations occurring among the samples considered. To do this, CNV and PAV analyses were performed to detect additional or deleted homologous loci of the 378 genes among the 43 samples, by analysing the normalized depth coverage of each sample. The comparison of read depths along the loci revealed at least 10 loci with a significant variation in depth in at least one line. In particular, nine genes (four Pectinesterases, two Phenylalanine ammonia-lyase, one Reductase, one Inositol-3-phosphate synthase, one Caffeoyl-CoA O-methyltransferase) showed a CNV, and one gene (Dehydroascorbate reductase) showed a PAV (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Three genes showed a putative reduction of the CNV (ratio &lt;0.5) while six showed an increase (ratio &gt;1.5). The very low number of reads captured for two samples (103A and 15A) for the gene Solyc09g056180 was associated with a PAV and we assumed that this gene was deleted in the two samples. In addition, an experimental validation by genomic PCR corroborated this result, highlighting that no amplification of the Solyc09g056180 was present for these two genotypes as showed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S5</a></span>. The analysis of the coverage revealed a uniform high level of coverage across all the samples for seven genes (Solyc00g027770, Solyc00g282510, Solyc00g030510, Solyc02g075620, Solyc03g042560, Solyc03g036470, Solyc03g07 1860). Indeed, these seven genes showed levels of coverage ranging between 104.58× and 192.48×, representing an increase of two to four fold compared with the average coverage encountered among all the genes (53.89×). This could be due to the lack of regions in the current release of tomato. A high number of reads collapsing on specific regions could be caused by unassembled/misassembled regions in the genome.<a reveal-id="dsw050-B29" data-open="dsw050-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> To validate this unexpected high coverage, we evaluated the depth of coverage of these genes in two further independent resequencing experiments (Heinz and Moneymaker) by using publicly available data (variation data from SGN databases). We observed that significant high coverage was obtained for three genes in at least one of the two resequencing experiments and one gene in both the resequencing experiments (data not shown). This corroborates that the reference genome could carry misleading sequences. An additional indication of this hypothesis, as also discussed below, come from the high ratio of He observed for these genes, showing that similar but not identical reads were mapped on the reference regions.</p>                            <h3 scrollto-destination=61463611 id="61463611" class="section-title" >3.5. Heterozygous variants into gene families</h3>
<p>He variant assignments represent one crucial difficulty in sequence experiments.<a reveal-id="dsw050-B10" data-open="dsw050-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw050-B29" data-open="dsw050-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw050-B55" data-open="dsw050-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Since one of the causes of spurious He call was hypothesized to derive from duplicated genes, a specific investigation was carried out, aided by the high quality of variants detected by SUPER-CAP. Previous studies reported that false-positive signals could arise in regions with low complexity,<a reveal-id="dsw050-B56" data-open="dsw050-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> or result from misalignment of multiple copies of genes, paralogues, or pseudogenes<a reveal-id="dsw050-B57" data-open="dsw050-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> but, to our knowledge, no study has analysed the behaviour of the He variants in the gene family context.</p><div class="block-child-p">In our collection, 235 out 378 genes completely represent 25 gene families. In particular, 171 genes belong to eight large gene families, 40 belong to seven medium gene families, 24 belong to 10 small gene families and six are single copy genes. In order to estimate the number of He variants in each group we evaluated (<a reveal-id="dsw050-F3" data-open="dsw050-F3" class="link link-reveal link-table xref-fig">Fig. 3</a><strong>)</strong> the average He density (number of He variants for Kbp) for each gene family. On average, a lower number of He variants was identified in the single genes compared with the gene family groups. Significant differences were observed at <em>t</em>-test between single genes and Large, Medium and Small gene families (<em>P</em> = 0.007) but not among gene families themselves (<em>P</em> &gt; 0.05). Although these results evidence that gene families, despite the number of copies, are more prone to produce He variants compared with single genes, the high standard deviation detected showed that differences seem to be associated with specific cases in each gene family and not due to a general behaviour. It appears evident that in each gene family only specific genes showed a very high level of heterozygosity in almost all the genotypes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In particular it was evident that Solyc00g282510, Solyc03g042560, Solyc00g027770 and Solyc01g091060 (belonging to large gene families) and Solyc12g098090 (belonging to a medium gene family) showed a skewed rate of He variants. Some possible explanations of this result were hypothesized. In particular, exploring if neighbouring regions of these genes showed a similar rate of heterozygosity (by using information from the 360 genome project), we evidenced that a similar high rate of He variants was found only for Solyc03g042560 in a region spanning ∼300 Kbp. <div data-id="dsw050-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f3.png?Expires=2147483647&amp;Signature=u1K3DmEpBEKKoUHssecw5dhw1t2gz4KdgsprYaQvpYm8oeNI8NyOrYVUJ0mUXsLj0PBbx-HBxZZPjnR8X16ul2~f4cFVa-91ikE58Khnx2TuOzg5Dmhs0VIONWG71nj4uZbiCn8RyBibp1O-L-WoUywF5NCI41FJWZGAUVfuZGscZR9VKcrG4rASUFMhMVLJGIRV04KH7NtZKV0Y6DyUwxMeA5gWO-NST6~XcrrVj06Re1Mh7mBqQysCoF1EM5xWzA2OiaZSa4WltF4eZlAmfZbJVAK8dKyuznVCYBWVyCvICIS871zNmwGOWlGzB2UkeBG9wdzZV57jpL1LBPAcSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average He variants density (He density) for each gene family (reported as EC number) in each category (LARGE, MEDIUM, SMALL, SINGLE). He density is expressed as the average number of He variants for 10 Kbp. Bar errors represent SD of the mean." path-from-xml="dsw050f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463613/dsw050f3.tif" path-from-xml="dsw050f3.tif" target="_blank">View large</a><a section="61463613" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f3.png?Expires=2147483647&amp;Signature=GIIFWa9F1bwjePGBMHMJWICs3JMYPvV0S0cKvVbbnWxUW0QJTHpFOeC77824gcVonEICiOb9xtKziSBNAJzwlEtx3LsiiOvt~tikRCvWKeDEqUGJmXsY5rT-fa6sJ6PFONPMnBPW3CfyoeTO0tsHg5hXt3kgodqDyrHv3uJ4jxL3RoVAsZ6a7CJUaH0jobX2yjLQ7aFoZTv~eMAsoqSM4gKYIa213Mo9NpXY86pC7tBGssQjjH5EvYgTleZf9upQ5BSPCuKz0PhbTUSlZjCdwpavoVGlCMRGTFm0CedDf3VRVDAsakOU9qN-Rec-9npx6e11~g0R4v41BhZtnjmA-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463613&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average He variants density (He density) for each gene family (reported as EC number) in each category (LARGE, MEDIUM, SMALL, SINGLE). He density is expressed as the average number of He variants for 10 Kbp. Bar errors represent SD of the mean.</p></div></div><div content-id="dsw050-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f3.png?Expires=2147483647&amp;Signature=u1K3DmEpBEKKoUHssecw5dhw1t2gz4KdgsprYaQvpYm8oeNI8NyOrYVUJ0mUXsLj0PBbx-HBxZZPjnR8X16ul2~f4cFVa-91ikE58Khnx2TuOzg5Dmhs0VIONWG71nj4uZbiCn8RyBibp1O-L-WoUywF5NCI41FJWZGAUVfuZGscZR9VKcrG4rASUFMhMVLJGIRV04KH7NtZKV0Y6DyUwxMeA5gWO-NST6~XcrrVj06Re1Mh7mBqQysCoF1EM5xWzA2OiaZSa4WltF4eZlAmfZbJVAK8dKyuznVCYBWVyCvICIS871zNmwGOWlGzB2UkeBG9wdzZV57jpL1LBPAcSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average He variants density (He density) for each gene family (reported as EC number) in each category (LARGE, MEDIUM, SMALL, SINGLE). He density is expressed as the average number of He variants for 10 Kbp. Bar errors represent SD of the mean." path-from-xml="dsw050f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463613/dsw050f3.tif" path-from-xml="dsw050f3.tif" target="_blank">View large</a><a section="61463613" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f3.png?Expires=2147483647&amp;Signature=GIIFWa9F1bwjePGBMHMJWICs3JMYPvV0S0cKvVbbnWxUW0QJTHpFOeC77824gcVonEICiOb9xtKziSBNAJzwlEtx3LsiiOvt~tikRCvWKeDEqUGJmXsY5rT-fa6sJ6PFONPMnBPW3CfyoeTO0tsHg5hXt3kgodqDyrHv3uJ4jxL3RoVAsZ6a7CJUaH0jobX2yjLQ7aFoZTv~eMAsoqSM4gKYIa213Mo9NpXY86pC7tBGssQjjH5EvYgTleZf9upQ5BSPCuKz0PhbTUSlZjCdwpavoVGlCMRGTFm0CedDf3VRVDAsakOU9qN-Rec-9npx6e11~g0R4v41BhZtnjmA-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463613&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average He variants density (He density) for each gene family (reported as EC number) in each category (LARGE, MEDIUM, SMALL, SINGLE). He density is expressed as the average number of He variants for 10 Kbp. Bar errors represent SD of the mean.</p></div></div></div><p>Another source of false positive He variants might be due to duplicated regions found in the samples that are not found in the reference genome. In such cases, the variants will be due to reads from the two copies (or more) that are piled in the only copy found in the reference genome. Such biases could occur because of the limited number of genotypes on which the original reference sequence was based, or sequencing and alignment errors (Lander <em>et al</em>., 2001). Fortunately, these cases can be highlighted looking at excesses of depth of coverage. Three out of the five genes (Solyc00g282510, Solyc00g027770, Solyc01g091060) exhibiting a high He density showed a high depth of coverage in parallel (as also evidenced in the previous paragraph), generally doubled or tripled in comparison to the average value of all the samples. This proves that the tomato reference genome might carry misleading sequences for those regions and that the He calls observed are likely to be false-positives.</p>                            <h3 scrollto-destination=61463615 id="61463615" class="section-title" >3.6. Variants classification and annotation</h3>
<div class="block-child-p">As an additional step, the functional annotation of the variants has also been investigated by using SnpEff and SNPsift programs. The SNPs discovered were classified according to their type (transition vs transversion and insertions vs deletion) and the genomic region they were found in. A significant number of the SNPs discovered (6,098, 44.2%) were located in the genic region and, among these, about 21% were found within exons, 73% within introns, 1.73% within the 5′-UTR and 3.68% within the 3′-UTR (<a reveal-id="dsw050-F4" data-open="dsw050-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The remaining 7,674 (55.8%) were found to be within the promoter region. Considering the number of variants per Kbp, eight variants per Kbp were found in the promoter region on average, while only 2.2 variants per Kbp were found in the exonic regions. Introns and UTRs showed more than double the number of variants per Kbp compared with exonic regions. Genic and intergenic variants were classified as transition/transversion and insertion/deletion typologies. Transitions represent approximately 64.54% of all post-filtered SNPs (11,495), with 44.2% (5,160) of them being located in genic regions. Deletions in average were more represented than insertions, particularly in promoter and intron regions. Then, the 6,098 variants of the genic regions were annotated according to their putative effect on the protein by using SnpEff v4 (<a reveal-id="dsw050-T2" data-open="dsw050-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The goal of annotating variants is to provide a prediction of which ones are functionally relevant. Our detected variants were assigned to a diverse range of functional classes, with the majority (78%) classified as ‘modifier’, therefore without predictable effects being located in the UTR regions or in the introns. These may have little or no effect on the phenotype. Among the remaining variants, about 12% showed a ‘low effect’, since they were variants in coding regions that did not change the amino acid sequence (synonymous variant). About 9% showed a ‘moderate’ effect. These variants are predominantly non-synonymous amino acid changes (missense variant) and in few cases in-frame deletion/insertion. They are the most likely candidates for causal mutations, since they could alter the structure and function of relevant proteins. Last, about 1% of the variants were predicted to have a ‘high effect’. In particular, 14 variants were predicted to cause a frameshift: two of them a splice site acceptor modification, and nine and three a start or stop codon gain or loss, respectively. On average, we observed that the percentage of INDELs with ‘high effect’ was higher than for SNPs, since an INDEL may rapidly cause a frameshift in the sequence. The list of the variant with a moderate or a high effect with respect to the gene they affected is provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. <div data-id="dsw050-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f4.png?Expires=2147483647&amp;Signature=cmBBYeiiitILWnulopSnR7CgzZngpAzLRGUY1R7FfC7EYkH8b-LCL56OzajShZGj5Dior0Xw4dWrh5FyQB28rq7XHiK1w725hB1ut3K4fSrgySz1lTjrhD0oRoTuvQGME8KAgZxE0zlKseeIpfNBxtZ8EpQHLQBTNR1tQLJpP16RS7yPT8ox996vNmQodZ6Naoq4BBhlII3OJ8yZbzcfj6MxihdBHpG-nMtceRiDQ4VTecfJd-ngO09crE4fBpaEHkJ~8T94mbcr6Kt-AsmENhkvjQkzxQPrC0bgxFYBIfUgiQ0~YsnB7NNu7D4VW51KCBwWyIoI9idTNgTtqRWq8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and classification of the discovered variants according to type (transition vs transversion and insertions vs deletion) and according to the genetic feature they were found in (promoter, exon, intron, UTRs). " path-from-xml="dsw050f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463616/dsw050f4.tif" path-from-xml="dsw050f4.tif" target="_blank">View large</a><a section="61463616" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f4.png?Expires=2147483647&amp;Signature=fflFCikVQ3BzBD-mPyeNydGYw~eZfZcMQBqPko2ghLQTC2IlvS8d~9iHckmvJsCabfieh6bBxS7H0DO~dxbGA6HRERdQZJi0yqkuYfx6xEjbWC2Es58iOsdcUqsm-g9XoWUVRERgUAOsAz1S52uQifsbG2llReixGMpcjXRxJdetVgBQ1ByrdqvtlW2HpgGgMdM9EA~TJP8QWflpMLRDnPUJXuJBc6njJGIM0LqL2NelxyY~C5fuGq-vWOjrwg~~Q1QvG8X7j3tNfX~QEVWbmmW9bViZ29HF21qPvAKBR9uwPvDsj2cJzPbzI54Su~cKu1H24HDZTAPgT6dfmaetTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463616&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and classification of the discovered variants according to type (transition vs transversion and insertions vs deletion) and according to the genetic feature they were found in (promoter, exon, intron, UTRs). </p></div></div><div content-id="dsw050-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/m_dsw050f4.png?Expires=2147483647&amp;Signature=cmBBYeiiitILWnulopSnR7CgzZngpAzLRGUY1R7FfC7EYkH8b-LCL56OzajShZGj5Dior0Xw4dWrh5FyQB28rq7XHiK1w725hB1ut3K4fSrgySz1lTjrhD0oRoTuvQGME8KAgZxE0zlKseeIpfNBxtZ8EpQHLQBTNR1tQLJpP16RS7yPT8ox996vNmQodZ6Naoq4BBhlII3OJ8yZbzcfj6MxihdBHpG-nMtceRiDQ4VTecfJd-ngO09crE4fBpaEHkJ~8T94mbcr6Kt-AsmENhkvjQkzxQPrC0bgxFYBIfUgiQ0~YsnB7NNu7D4VW51KCBwWyIoI9idTNgTtqRWq8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and classification of the discovered variants according to type (transition vs transversion and insertions vs deletion) and according to the genetic feature they were found in (promoter, exon, intron, UTRs). " path-from-xml="dsw050f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61463616/dsw050f4.tif" path-from-xml="dsw050f4.tif" target="_blank">View large</a><a section="61463616" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050f4.png?Expires=2147483647&amp;Signature=fflFCikVQ3BzBD-mPyeNydGYw~eZfZcMQBqPko2ghLQTC2IlvS8d~9iHckmvJsCabfieh6bBxS7H0DO~dxbGA6HRERdQZJi0yqkuYfx6xEjbWC2Es58iOsdcUqsm-g9XoWUVRERgUAOsAz1S52uQifsbG2llReixGMpcjXRxJdetVgBQ1ByrdqvtlW2HpgGgMdM9EA~TJP8QWflpMLRDnPUJXuJBc6njJGIM0LqL2NelxyY~C5fuGq-vWOjrwg~~Q1QvG8X7j3tNfX~QEVWbmmW9bViZ29HF21qPvAKBR9uwPvDsj2cJzPbzI54Su~cKu1H24HDZTAPgT6dfmaetTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61463616&amp;ar=2656160&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw050f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and classification of the discovered variants according to type (transition vs transversion and insertions vs deletion) and according to the genetic feature they were found in (promoter, exon, intron, UTRs). </p></div></div><div content-id="dsw050-T2" class="table-modal"><div class="table-wrap"><div id="dsw050-T2" data-id="dsw050-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of 6,098 genic variants <em>per</em> type of predicted effect (PREDICTED EFFECT) on the relative protein as predicted by SNPeff.</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PREDICTED EFFECT </th><th>VARIANTS No </th></tr></thead><tbody><tr><td><strong>Modifier</strong> </td><td><strong>4,765</strong> </td></tr><tr><td>3′ UTR variant </td><td>256 </td></tr><tr><td>5′ UTR variant </td><td>155 </td></tr><tr><td>Intron variant </td><td>4,354 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Low</strong> </td><td><strong>729</strong> </td></tr><tr><td>5′ UTR premature start codon gain variant </td><td>13 </td></tr><tr><td>Splice region variant &amp; intron variant </td><td>115 </td></tr><tr><td>Synonymous_variant </td><td>601 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Moderate</strong> </td><td><strong>576</strong> </td></tr><tr><td>Missense variant </td><td>559 </td></tr><tr><td>Inframe deletion </td><td>7 </td></tr><tr><td>Inframe insertion </td><td>10 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>High</strong> </td><td><strong>28</strong> </td></tr><tr><td>Frameshift_variant </td><td>14 </td></tr><tr><td>Splice acceptor variant </td><td>2 </td></tr><tr><td>Stop_gained variant </td><td>9 </td></tr><tr><td>Stop_lost variant </td><td>3 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Total</strong> </td><td><strong>6,098</strong> </td></tr></tbody></table></div><div content-id="dsw050-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>PREDICTED EFFECT </th><th>VARIANTS No </th></tr></thead><tbody><tr><td><strong>Modifier</strong> </td><td><strong>4,765</strong> </td></tr><tr><td>3′ UTR variant </td><td>256 </td></tr><tr><td>5′ UTR variant </td><td>155 </td></tr><tr><td>Intron variant </td><td>4,354 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Low</strong> </td><td><strong>729</strong> </td></tr><tr><td>5′ UTR premature start codon gain variant </td><td>13 </td></tr><tr><td>Splice region variant &amp; intron variant </td><td>115 </td></tr><tr><td>Synonymous_variant </td><td>601 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Moderate</strong> </td><td><strong>576</strong> </td></tr><tr><td>Missense variant </td><td>559 </td></tr><tr><td>Inframe deletion </td><td>7 </td></tr><tr><td>Inframe insertion </td><td>10 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>High</strong> </td><td><strong>28</strong> </td></tr><tr><td>Frameshift_variant </td><td>14 </td></tr><tr><td>Splice acceptor variant </td><td>2 </td></tr><tr><td>Stop_gained variant </td><td>9 </td></tr><tr><td>Stop_lost variant </td><td>3 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Total</strong> </td><td><strong>6,098</strong> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw050-T2" data-id="dsw050-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Distribution of 6,098 genic variants <em>per</em> type of predicted effect (PREDICTED EFFECT) on the relative protein as predicted by SNPeff.</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PREDICTED EFFECT </th><th>VARIANTS No </th></tr></thead><tbody><tr><td><strong>Modifier</strong> </td><td><strong>4,765</strong> </td></tr><tr><td>3′ UTR variant </td><td>256 </td></tr><tr><td>5′ UTR variant </td><td>155 </td></tr><tr><td>Intron variant </td><td>4,354 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Low</strong> </td><td><strong>729</strong> </td></tr><tr><td>5′ UTR premature start codon gain variant </td><td>13 </td></tr><tr><td>Splice region variant &amp; intron variant </td><td>115 </td></tr><tr><td>Synonymous_variant </td><td>601 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Moderate</strong> </td><td><strong>576</strong> </td></tr><tr><td>Missense variant </td><td>559 </td></tr><tr><td>Inframe deletion </td><td>7 </td></tr><tr><td>Inframe insertion </td><td>10 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>High</strong> </td><td><strong>28</strong> </td></tr><tr><td>Frameshift_variant </td><td>14 </td></tr><tr><td>Splice acceptor variant </td><td>2 </td></tr><tr><td>Stop_gained variant </td><td>9 </td></tr><tr><td>Stop_lost variant </td><td>3 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Total</strong> </td><td><strong>6,098</strong> </td></tr></tbody></table></div><div content-id="dsw050-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>PREDICTED EFFECT </th><th>VARIANTS No </th></tr></thead><tbody><tr><td><strong>Modifier</strong> </td><td><strong>4,765</strong> </td></tr><tr><td>3′ UTR variant </td><td>256 </td></tr><tr><td>5′ UTR variant </td><td>155 </td></tr><tr><td>Intron variant </td><td>4,354 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Low</strong> </td><td><strong>729</strong> </td></tr><tr><td>5′ UTR premature start codon gain variant </td><td>13 </td></tr><tr><td>Splice region variant &amp; intron variant </td><td>115 </td></tr><tr><td>Synonymous_variant </td><td>601 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Moderate</strong> </td><td><strong>576</strong> </td></tr><tr><td>Missense variant </td><td>559 </td></tr><tr><td>Inframe deletion </td><td>7 </td></tr><tr><td>Inframe insertion </td><td>10 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>High</strong> </td><td><strong>28</strong> </td></tr><tr><td>Frameshift_variant </td><td>14 </td></tr><tr><td>Splice acceptor variant </td><td>2 </td></tr><tr><td>Stop_gained variant </td><td>9 </td></tr><tr><td>Stop_lost variant </td><td>3 </td></tr><tr><td colspan="2"><hr /> </td></tr><tr><td><strong>Total</strong> </td><td><strong>6,098</strong> </td></tr></tbody></table></div></div></div><p>Thus, a larger effort would be warranted to study potential links between the identified variants and trait variation in tomato, and to determine how they affect the regulation of biological pathways and processes. The variants selected or prioritized in this way would be highly preferred marker sets to be subjected to association studies using suitably larger populations, for which panels with considerable genotype and phenotype information has already been collected.<a reveal-id="dsw050-B30" data-open="dsw050-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw050-B32" data-open="dsw050-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p>                            <h2 scrollto-destination=61463618 id="61463618" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors wish to thank the Genomix4Life S.r.l (<a class="link link-uri" href="http://www.genomix4life.com" target="_blank">http://www.genomix4life.com</a>) for the genotyping analyses performed with ILLUMINA Infinium Technology.</p>                            <h3 scrollto-destination=61463620 id="61463620" class="section-title" >Conflict of interest</h3>
<p>None declared. </p>                            <h3 scrollto-destination=61463622 id="61463622" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=61463624 id="61463624" class="section-title" >Funding</h3>
<p>This work was supported by the Italian Ministry of University and Research (MIUR) (grant GenoPOMpro PON02_00395_3082360). The funders had no role in the study design, data collection or analysis, the decision to publish, or in the preparation of the article. </p>                            <h2 scrollto-destination=61463626 id="61463626" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw050-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">P. K.</div></div></div> <div class="year">2013</div>, Sequence based DNA markers and genotyping for cereal genomics and breeding, In: <div class="person-group"><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">K.P.</div></div><div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">K.R.</div></div></div> (eds), <div class="source">Cereal Genomics II</div> , 
<div class="publisher-loc">Netherlands, Dordrecht</div>: 
<div class="publisher-name">Springer</div>, pp. <div class="fpage">57</div>–<div class="lpage">76</div>.</div></div></div></div><div content-id="dsw050-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ray</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Satya</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Next generation sequencing technologies for next generation plant breeding,</div> <div class="source">Front. Plant. Sci</div> ., <div class="volume">5</div>, <div class="fpage">367</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20technologies%20for%20next%20generation%20plant%20breeding%2C&amp;author=S.%20Ray&amp;author=P.%20Satya&amp;publication_year=2014&amp;journal=Front.%20Plant.%20Sci&amp;volume=5&amp;pages=367" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25126091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nicolaï</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Cantet</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Lefebvre</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Sage-Palloix</div> <div class="given-names">A.-M.</div></div><div class="name string-name"><div class="surname">Palloix</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genotyping a large collection of pepper (Capsicum spp.) with SSR loci brings new evidence for the wild origin of cultivated C. annuum and the structuring of genetic diversity by human selection of cultivar types,</div> <div class="source">Genet. Resour. Crop Evol</div> ., <div class="volume">60</div>, <div class="fpage">2375</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping%20a%20large%20collection%20of%20pepper%20(Capsicum%20spp.)%20with%20SSR%20loci%20brings%20new%20evidence%20for%20the%20wild%20origin%20of%20cultivated%20C.%20annuum%20and%20the%20structuring%20of%20genetic%20diversity%20by%20human%20selection%20of%20cultivar%20types%2C&amp;author=M.%20Nicola%C3%AF&amp;author=M.%20Cantet&amp;author=V.%20Lefebvre&amp;author=A.-M.%20Sage-Palloix&amp;author=A.%20Palloix&amp;publication_year=2013&amp;journal=Genet.%20Resour.%20Crop%20Evol&amp;volume=60&amp;pages=2375-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-013-0006-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-013-0006-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-013-0006-0"> </span></div> </div></div></div></div></div><div content-id="dsw050-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ranc</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Munos</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Santoni</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Causse</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A clarified position for Solanum lycopersicum var. cerasiforme in the evolutionary history of tomatoes (solanaceae)</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">8</div>, <div class="fpage">130</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20clarified%20position%20for%20Solanum%20lycopersicum%20var.%20cerasiforme%20in%20the%20evolutionary%20history%20of%20tomatoes%20(solanaceae)&amp;author=N.%20Ranc&amp;author=S.%20Munos&amp;author=S.%20Santoni&amp;author=M.%20Causse&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol&amp;volume=8&amp;pages=130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19099601" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanca</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Montero-Pau</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Sauvage</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Genomic variation in tomato, from wild ancestors to contemporary breeding accessions,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">257</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20variation%20in%20tomato%2C%20from%20wild%20ancestors%20to%20contemporary%20breeding%20accessions%2C&amp;author=J.%20Blanca&amp;author=J.%20Montero-Pau&amp;author=C.%20Sauvage&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=257" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1444-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1444-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1444-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25880392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">He</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X. W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Development of genomics-based genotyping platforms and their applications in rice breeding,</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">247</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genomics-based%20genotyping%20platforms%20and%20their%20applications%20in%20rice%20breeding%2C&amp;author=H.%20Chen&amp;author=H.%20He&amp;author=F.%20Zhou&amp;author=H.%20Yu&amp;author=X.%20W.%20Deng&amp;publication_year=2013&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=16&amp;pages=247-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2013.04.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2013.04.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2013.04.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23706659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Filippi</div> <div class="given-names">C. V.</div></div><div class="name string-name"><div class="surname">Aguirre</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Rivas</div> <div class="given-names">J. G.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Population structure and genetic diversity characterization of a sunflower association mapping population using SSR and SNP markers,</div> <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20genetic%20diversity%20characterization%20of%20a%20sunflower%20association%20mapping%20population%20using%20SSR%20and%20SNP%20markers%2C&amp;author=C.%20V.%20Filippi&amp;author=N.%20Aguirre&amp;author=J.%20G.%20Rivas&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0360-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0360-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0360-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25592487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Matsunaga</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato,</div> <div class="source">DNA Res</div> ., <div class="volume">20</div>, <div class="fpage">593</div>–<div class="lpage">603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20using%20single%20nucleotide%20polymorphism%20markers%20developed%20by%20re-sequencing%20of%20the%20genomes%20of%20cultivated%20tomato%2C&amp;author=K.%20Shirasawa&amp;author=H.%20Fukuoka&amp;author=H.%20Matsunaga&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=593-603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23903436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sim</div> <div class="given-names">S. C.</div></div><div class="name string-name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e40563</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20large%20SNP%20genotyping%20array%20and%20generation%20of%20high-density%20genetic%20maps%20in%20tomato%2C&amp;author=S.%20C.%20Sim&amp;author=G.%20Durstewitz&amp;author=J.%20Plieske&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e40563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22802968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Causse</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Desplat</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Pascual</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Whole genome resequencing in tomato reveals variation associated with introgression and breeding events,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20resequencing%20in%20tomato%20reveals%20variation%20associated%20with%20introgression%20and%20breeding%20events%2C&amp;author=M.%20Causse&amp;author=N.%20Desplat&amp;author=L.%20Pascual&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-791" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23323973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ercolano</div> <div class="given-names">M. R.</div></div><div class="name string-name"><div class="surname">Sacco</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Ferriello</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Patchwork sequencing of tomato San Marzano and Vesuviano varieties highlights genome-wide variations,</div> <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">138</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patchwork%20sequencing%20of%20tomato%20San%20Marzano%20and%20Vesuviano%20varieties%20highlights%20genome-wide%20variations%2C&amp;author=M.%20R.%20Ercolano&amp;author=A.%20Sacco&amp;author=F.%20Ferriello&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-138" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-138" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-138"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24548308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lam</div> <div class="given-names">H. Y. K.</div></div><div class="name string-name"><div class="surname">Clark</div> <div class="given-names">M. J.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Performance comparison of whole-genome sequencing platforms,</div> <div class="source">Nat. Biotech</div> ., <div class="volume">30</div>, <div class="fpage">78</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20comparison%20of%20whole-genome%20sequencing%20platforms%2C&amp;author=H.%20Y.%20K.%20Lam&amp;author=M.%20J.%20Clark&amp;author=R.%20Chen&amp;publication_year=2012&amp;journal=Nat.%20Biotech&amp;volume=30&amp;pages=78-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2065"> </span></div> </div></div></div></div></div><div content-id="dsw050-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ott</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Reference-guided assembly of four diverse Arabidopsis thaliana genomes,</div> <div class="source">Proc. Natl. Acad. Sci. U S A</div> , <div class="volume">108</div>, <div class="fpage">10249</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reference-guided%20assembly%20of%20four%20diverse%20Arabidopsis%20thaliana%20genomes%2C&amp;author=K.%20Schneeberger&amp;author=S.%20Ossowski&amp;author=F.%20Ott&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;volume=108&amp;pages=10249-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1107739108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1107739108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1107739108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21646520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">T. J.</div></div><div class="name string-name"><div class="surname">Molla</div> <div class="given-names">M. N.</div></div><div class="name string-name"><div class="surname">Muzny</div> <div class="given-names">D. M.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Direct selection of human genomic loci by microarray hybridization,</div> <div class="source">Nat. Methods</div> , <div class="volume">4</div>, <div class="fpage">903</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization%2C&amp;author=T.%20J.%20Albert&amp;author=M.%20N.%20Molla&amp;author=D.%20M.%20Muzny&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okou</div> <div class="given-names">D. T.</div></div><div class="name string-name"><div class="surname">Steinberg</div> <div class="given-names">K. M.</div></div><div class="name string-name"><div class="surname">Middle</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Cutler</div> <div class="given-names">D. J.</div></div><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">T. J.</div></div><div class="name string-name"><div class="surname">Zwick</div> <div class="given-names">M. E.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Microarray-based genomic selection for high-throughput resequencing,</div> <div class="source">Nat. Methods</div> , <div class="volume">4</div>, <div class="fpage">907</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray-based%20genomic%20selection%20for%20high-throughput%20resequencing%2C&amp;author=D.%20T.%20Okou&amp;author=K.%20M.%20Steinberg&amp;author=C.%20Middle&amp;author=D.%20J.%20Cutler&amp;author=T.%20J.%20Albert&amp;author=M.%20E.%20Zwick&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=907-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Olson</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Enrichment of super-sized resequencing targets from the human genome,</div> <div class="source">Nat. Methods</div> , <div class="volume">4</div>, <div class="fpage">891</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20super-sized%20resequencing%20targets%20from%20the%20human%20genome%2C&amp;author=M.%20Olson&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=891-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1107-891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1107-891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1107-891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17971778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Sousa</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Bertrand</div> <div class="given-names">Y. J.</div></div><div class="name string-name"><div class="surname">Nylinder</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Oxelman</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Eriksson</div> <div class="given-names">J. S.</div></div><div class="name string-name"><div class="surname">Pfeil</div> <div class="given-names">B. E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Phylogenetic properties of 50 nuclear loci in Medicago (Leguminosae) generated using multiplexed sequence capture and next-generation sequencing,</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e109704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20properties%20of%2050%20nuclear%20loci%20in%20Medicago%20(Leguminosae)%20generated%20using%20multiplexed%20sequence%20capture%20and%20next-generation%20sequencing%2C&amp;author=F.%20de%20Sousa&amp;author=Y.%20J.%20Bertrand&amp;author=S.%20Nylinder&amp;author=B.%20Oxelman&amp;author=J.%20S.%20Eriksson&amp;author=B.%20E.%20Pfeil&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e109704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0109704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0109704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0109704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25329401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rabbani</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Tekin</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Mahdieh</div> <div class="given-names">N.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The promise of whole-exome sequencing in medical genetics,</div> <div class="source">J. Hum. Genet</div> ., <div class="volume">59</div>, <div class="fpage">5</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20promise%20of%20whole-exome%20sequencing%20in%20medical%20genetics%2C&amp;author=B.%20Rabbani&amp;author=M.%20Tekin&amp;author=N.%20Mahdieh&amp;publication_year=2014&amp;journal=J.%20Hum.%20Genet&amp;volume=59&amp;pages=5-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/jhg.2013.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fjhg.2013.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fjhg.2013.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24196381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Muraya</div> <div class="given-names">M. M.</div></div><div class="name string-name"><div class="surname">Schmutzer</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Ulpinnis</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">U.</div></div><div class="name string-name"><div class="surname">Altmann</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Targeted sequencing reveals large-scale sequence polymorphism in maize candidate genes for biomass production and composition,</div> <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0132120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20sequencing%20reveals%20large-scale%20sequence%20polymorphism%20in%20maize%20candidate%20genes%20for%20biomass%20production%20and%20composition%2C&amp;author=M.%20M.%20Muraya&amp;author=T.%20Schmutzer&amp;author=C.%20Ulpinnis&amp;author=U.%20Scholz&amp;author=T.%20Altmann&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0132120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0132120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0132120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0132120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26151830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashman</div> <div class="given-names">T. L.</div></div><div class="name string-name"><div class="surname">Tennessen</div> <div class="given-names">J. A.</div></div><div class="name string-name"><div class="surname">Dalton</div> <div class="given-names">R. M.</div></div><div class="name string-name"><div class="surname">Govindarajulu</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Koski</div> <div class="given-names">M. H.</div></div><div class="name string-name"><div class="surname">Liston</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Multilocus sex determination revealed in two populations of gynodioecious wild strawberry, Fragaria vesca subsp. bracteata</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">5</div>, <div class="fpage">2759</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multilocus%20sex%20determination%20revealed%20in%20two%20populations%20of%20gynodioecious%20wild%20strawberry%2C%20Fragaria%20vesca%20subsp.%20bracteata&amp;author=T.%20L.%20Ashman&amp;author=J.%20A.%20Tennessen&amp;author=R.%20M.%20Dalton&amp;author=R.%20Govindarajulu&amp;author=M.%20H.%20Koski&amp;author=A.%20Liston&amp;publication_year=2015&amp;journal=G3%20(Bethesda)&amp;volume=5&amp;pages=2759-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.115.023358" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.115.023358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.115.023358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26483011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clarke</div> <div class="given-names">W. E.</div></div><div class="name string-name"><div class="surname">Parkin</div> <div class="given-names">I. A.</div></div><div class="name string-name"><div class="surname">Gajardo</div> <div class="given-names">H. A.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Genomic DNA enrichment using sequence capture microarrays: a novel approach to discover sequence nucleotide polymorphisms (SNP) in Brassica napus L,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e81992</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20DNA%20enrichment%20using%20sequence%20capture%20microarrays%3A%20a%20novel%20approach%20to%20discover%20sequence%20nucleotide%20polymorphisms%20(SNP)%20in%20Brassica%20napus%20L%2C&amp;author=W.%20E.%20Clarke&amp;author=I.%20A.%20Parkin&amp;author=H.%20A.%20Gajardo&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e81992" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0081992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0081992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0081992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24312619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schiessl</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Samans</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Huttel</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Reinhard</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Snowdon</div> <div class="given-names">R. J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Capturing sequence variation among flowering-time regulatory gene homologs in the allopolyploid crop species Brassica napus,</div> <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20sequence%20variation%20among%20flowering-time%20regulatory%20gene%20homologs%20in%20the%20allopolyploid%20crop%20species%20Brassica%20napus%2C&amp;author=S.%20Schiessl&amp;author=B.%20Samans&amp;author=B.%20Huttel&amp;author=R.%20Reinhard&amp;author=R.%20J.%20Snowdon&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25202314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Holliday</div> <div class="given-names">J. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Targeted enrichment of the black cottonwood (Populus trichocarpa) gene space using sequence capture,</div> <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20enrichment%20of%20the%20black%20cottonwood%20(Populus%20trichocarpa)%20gene%20space%20using%20sequence%20capture%2C&amp;author=L.%20Zhou&amp;author=J.%20A.%20Holliday&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uitdewilligen</div> <div class="given-names">J. G.</div></div><div class="name string-name"><div class="surname">Wolters</div> <div class="given-names">A. M.</div></div><div class="name string-name"><div class="surname">D’Hoop</div> <div class="given-names">B. B.</div></div><div class="name string-name"><div class="surname">Borm</div> <div class="given-names">T. J.</div></div><div class="name string-name"><div class="surname">Visser</div> <div class="given-names">R. G.</div></div><div class="name string-name"><div class="surname">van Eck</div> <div class="given-names">H. J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e62355</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20next-generation%20sequencing%20method%20for%20genotyping-by-sequencing%20of%20highly%20heterozygous%20autotetraploid%20potato%2C&amp;author=J.%20G.%20Uitdewilligen&amp;author=A.%20M.%20Wolters&amp;author=B.%20B.%20D%E2%80%99Hoop&amp;author=T.%20J.%20Borm&amp;author=R.%20G.%20Visser&amp;author=H.%20J.%20van%20Eck&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e62355" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0062355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0062355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0062355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23667470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gardiner</div> <div class="given-names">L. J.</div></div><div class="name string-name"><div class="surname">Gawronski</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Olohan</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Schnurbusch</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Using genic sequence capture in combination with a syntenic pseudo genome to map a deletion mutant in a wheat species,</div> <div class="source">Plant J</div> ., <div class="volume">80</div>, <div class="fpage">895</div>–<div class="lpage">904</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20genic%20sequence%20capture%20in%20combination%20with%20a%20syntenic%20pseudo%20genome%20to%20map%20a%20deletion%20mutant%20in%20a%20wheat%20species%2C&amp;author=L.%20J.%20Gardiner&amp;author=P.%20Gawronski&amp;author=L.%20Olohan&amp;author=T.%20Schnurbusch&amp;author=N.%20Hall&amp;author=A.%20Hall&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=80&amp;pages=895-904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.80.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.80.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.80.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25205592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winfield</div> <div class="given-names">M. O.</div></div><div class="name string-name"><div class="surname">Wilkinson</div> <div class="given-names">P. A.</div></div><div class="name string-name"><div class="surname">Allen</div> <div class="given-names">A. M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Targeted re-sequencing of the allohexaploid wheat exome,</div> <div class="source">Plant Biotechnol. J</div> ., <div class="volume">10</div>, <div class="fpage">733</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20re-sequencing%20of%20the%20allohexaploid%20wheat%20exome%2C&amp;author=M.%20O.%20Winfield&amp;author=P.%20A.%20Wilkinson&amp;author=A.%20M.%20Allen&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J&amp;volume=10&amp;pages=733-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00713.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00713.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00713.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pootakham</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Shearman</div> <div class="given-names">J. R.</div></div><div class="name string-name"><div class="surname">Ruang-Areerate</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Large-scale SNP discovery through RNA sequencing and SNP genotyping by targeted enrichment sequencing in cassava (Manihot esculenta Crantz),</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e116028</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20SNP%20discovery%20through%20RNA%20sequencing%20and%20SNP%20genotyping%20by%20targeted%20enrichment%20sequencing%20in%20cassava%20(Manihot%20esculenta%20Crantz)%2C&amp;author=W.%20Pootakham&amp;author=J.%20R.%20Shearman&amp;author=P.%20Ruang-Areerate&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e116028" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0116028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0116028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0116028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25551642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sanseverino</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Henaff</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Vives</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Transposon Insertions, Structural Variations, and SNPs Contribute to the Evolution of the Melon Genome,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2760</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposon%20Insertions%2C%20Structural%20Variations%2C%20and%20SNPs%20Contribute%20to%20the%20Evolution%20of%20the%20Melon%20Genome%2C&amp;author=W.%20Sanseverino&amp;author=E.%20Henaff&amp;author=C.%20Vives&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2760-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msv152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsv152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26174143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuentes Fajardo</div> <div class="given-names">K. V.</div></div><div class="name string-name"><div class="surname">Adams</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Program</div> <div class="given-names">N. C. S.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Detecting false-positive signals in exome sequencing,</div> <div class="source">Hum. Mutat</div> ., <div class="volume">33</div>, <div class="fpage">609</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20false-positive%20signals%20in%20exome%20sequencing%2C&amp;author=K.%20V.%20Fuentes%20Fajardo&amp;author=D.%20Adams&amp;author=N.%20C.%20S.%20Program&amp;publication_year=2012&amp;journal=Hum.%20Mutat&amp;volume=33&amp;pages=609-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.22033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.22033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.22033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22294350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sacco</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Ruggieri</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Parisi</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Exploring a tomato landraces collection for fruit-related traits by the aid of a high-throughput genomic platform,</div> <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0137139</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20a%20tomato%20landraces%20collection%20for%20fruit-related%20traits%20by%20the%20aid%20of%20a%20high-throughput%20genomic%20platform%2C&amp;author=A.%20Sacco&amp;author=V.%20Ruggieri&amp;author=M.%20Parisi&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0137139" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0137139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0137139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0137139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26393929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ruggieri</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Bostan</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Barone</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Frusciante</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Chiusano</div> <div class="given-names">M. L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Integrated bioinformatics to decipher the ascorbic acid metabolic network in tomato,</div> <div class="source">Plant. Mol. Biol</div> ., <div class="volume">91</div> (<div class="issue">4–5</div>), <div class="fpage">397</div>–<div class="lpage">412</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20bioinformatics%20to%20decipher%20the%20ascorbic%20acid%20metabolic%20network%20in%20tomato%2C&amp;author=V.%20Ruggieri&amp;author=H.%20Bostan&amp;author=A.%20Barone&amp;author=L.%20Frusciante&amp;author=M.%20L.%20Chiusano&amp;publication_year=2016&amp;journal=Plant.%20Mol.%20Biol&amp;volume=91&amp;pages=397-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-016-0469-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-016-0469-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-016-0469-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27007138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ruggieri</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Francese</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Sacco</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">An association mapping approach to identify favourable alleles for tomato fruit quality breeding,</div> <div class="source">BMC Plant Biol</div> , <div class="volume">14</div>, <div class="fpage">337</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20association%20mapping%20approach%20to%20identify%20favourable%20alleles%20for%20tomato%20fruit%20quality%20breeding%2C&amp;author=V.%20Ruggieri&amp;author=G.%20Francese&amp;author=A.%20Sacco&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0337-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0337-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0337-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25465385" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform%2C&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">The Sequence Alignment/Map format and SAMtools,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools%2C&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Layer</div> <div class="given-names">R. M.</div></div><div class="name string-name"><div class="surname">Chiang</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A. R.</div></div><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I. M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">LUMPY: a probabilistic framework for structural variant discovery,</div> <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">R84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LUMPY%3A%20a%20probabilistic%20framework%20for%20structural%20variant%20discovery%2C&amp;author=R.%20M.%20Layer&amp;author=C.%20Chiang&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-6-r84" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-6-r84" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-6-r84"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24970577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cingolani</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Patel</div> <div class="given-names">V. M.</div></div><div class="name string-name"><div class="surname">Coon</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift,</div> <div class="source">Front. Genet</div> ., <div class="volume">3</div>, <div class="fpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20Drosophila%20melanogaster%20as%20a%20Model%20for%20Genotoxic%20Chemical%20Mutational%20Studies%20with%20a%20New%20Program%2C%20SnpSift%2C&amp;author=P.%20Cingolani&amp;author=V.%20M.%20Patel&amp;author=M.%20Coon&amp;publication_year=2012&amp;journal=Front.%20Genet&amp;volume=3&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2012.00035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2012.00035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2012.00035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22435069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2987</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20framework%20for%20SNP%20calling%2C%20mutation%20discovery%2C%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data%2C&amp;author=H.%20Li&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2987-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A. R.</div></div><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features,</div> <div class="source">Bioinformatics</div> , <div class="volume">26</div>.</div></div></div></div><div content-id="dsw050-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div><div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data,</div> <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data%2C&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shearer</div> <div class="given-names">L. A.</div></div><div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">L. K.</div></div><div class="name string-name"><div class="surname">de Jong</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Fluorescence in situ hybridization and optical mapping to correct scaffold arrangement in the tomato genome,</div> <div class="source">G3 (Bethesda)</div> , <div class="volume">4</div>, <div class="fpage">1395</div>–<div class="lpage">1405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescence%20in%20situ%20hybridization%20and%20optical%20mapping%20to%20correct%20scaffold%20arrangement%20in%20the%20tomato%20genome%2C&amp;author=L.%20A.%20Shearer&amp;author=L.%20K.%20Anderson&amp;author=H.%20de%20Jong&amp;publication_year=2014&amp;journal=G3%20(Bethesda)&amp;volume=4&amp;pages=1395-1405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.011197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.011197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.011197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24879607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bradbury</div> <div class="given-names">P. J.</div></div><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Kroon</div> <div class="given-names">D. E.</div></div><div class="name string-name"><div class="surname">Casstevens</div> <div class="given-names">T. M.</div></div><div class="name string-name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Buckler</div> <div class="given-names">E. S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples,</div> <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2633</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples%2C&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.%20S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Genomic analyses provide insights into the history of tomato breeding,</div> <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">1220</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analyses%20provide%20insights%20into%20the%20history%20of%20tomato%20breeding%2C&amp;author=T.%20Lin&amp;author=G.%20Zhu&amp;author=J.%20Zhang&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=1220-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25305757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Danecek</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The variant call format and VCFtools,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools%2C&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bodi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Perera</div> <div class="given-names">A. G.</div></div><div class="name string-name"><div class="surname">Adams</div> <div class="given-names">P. S.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Comparison of commercially available target enrichment methods for next-generation sequencing,</div> <div class="source">J. Biomol. Tech</div> ., <div class="volume">24</div>, <div class="fpage">73</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20commercially%20available%20target%20enrichment%20methods%20for%20next-generation%20sequencing%2C&amp;author=K.%20Bodi&amp;author=A.%20G.%20Perera&amp;author=P.%20S.%20Adams&amp;publication_year=2013&amp;journal=J.%20Biomol.%20Tech&amp;volume=24&amp;pages=73-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23814499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sims</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Sudbery</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Ilott</div> <div class="given-names">N. E.</div></div><div class="name string-name"><div class="surname">Heger</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C. P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Sequencing depth and coverage: key considerations in genomic analyses,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">121</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20depth%20and%20coverage%3A%20key%20considerations%20in%20genomic%20analyses%2C&amp;author=D.%20Sims&amp;author=I.%20Sudbery&amp;author=N.%20E.%20Ilott&amp;author=A.%20Heger&amp;author=C.%20P.%20Ponting&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=121-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24434847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bentley</div> <div class="given-names">D. R.</div></div><div class="name string-name"><div class="surname">Balasubramanian</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Swerdlow</div> <div class="given-names">H. P.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Accurate whole human genome sequencing using reversible terminator chemistry,</div> <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry%2C&amp;author=D.%20R.%20Bentley&amp;author=S.%20Balasubramanian&amp;author=H.%20P.%20Swerdlow&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=53-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18987734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cartwright</div> <div class="given-names">R. A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Problems and solutions for estimating indel rates and length distributions,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">26</div>, <div class="fpage">473</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Problems%20and%20solutions%20for%20estimating%20indel%20rates%20and%20length%20distributions%2C&amp;author=R.%20A.%20Cartwright&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol&amp;volume=26&amp;pages=473-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19042944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lunter</div> <div class="given-names">G.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Probabilistic whole-genome alignments reveal high indel rates in the human and mouse genomes,</div> <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">i289</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probabilistic%20whole-genome%20alignments%20reveal%20high%20indel%20rates%20in%20the%20human%20and%20mouse%20genomes%2C&amp;author=G.%20Lunter&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=i289-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Tsementzi</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Kyrpides</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Read</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Konstantinidis</div> <div class="given-names">K. T.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Direct comparisons of Illumina vs. Roche 454 sequencing technologies on the same microbial community DNA sample,</div> <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e30087</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20comparisons%20of%20Illumina%20vs.%20Roche%20454%20sequencing%20technologies%20on%20the%20same%20microbial%20community%20DNA%20sample%2C&amp;author=C.%20Luo&amp;author=D.%20Tsementzi&amp;author=N.%20Kyrpides&amp;author=T.%20Read&amp;author=K.%20T.%20Konstantinidis&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30087" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22347999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">A rare SNP identified a TCP transcription factor essential for tendril development in cucumber,</div> <div class="source">Mol. Plant</div> , <div class="volume">8</div>, <div class="fpage">1795</div>–<div class="lpage">808</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rare%20SNP%20identified%20a%20TCP%20transcription%20factor%20essential%20for%20tendril%20development%20in%20cucumber%2C&amp;author=S.%20Wang&amp;author=X.%20Yang&amp;author=M.%20Xu&amp;publication_year=2015&amp;journal=Mol.%20Plant&amp;volume=8&amp;pages=1795-808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2015.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2015.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2015.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26597500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bang</div> <div class="given-names">S. Y.</div></div><div class="name string-name"><div class="surname">Na</div> <div class="given-names">Y. J.</div></div><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Targeted exon sequencing fails to identify rare coding variants with large effect in rheumatoid arthritis,</div> <div class="source">Arthritis Res. Ther</div> ., <div class="volume">16</div>, <div class="fpage">447</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20exon%20sequencing%20fails%20to%20identify%20rare%20coding%20variants%20with%20large%20effect%20in%20rheumatoid%20arthritis%2C&amp;author=S.%20Y.%20Bang&amp;author=Y.%20J.%20Na&amp;author=K.%20Kim&amp;publication_year=2014&amp;journal=Arthritis%20Res.%20Ther&amp;volume=16&amp;pages=447" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13075-014-0447-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13075-014-0447-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13075-014-0447-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25267259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Belo</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Beatty</div> <div class="given-names">M. K.</div></div><div class="name string-name"><div class="surname">Hondred</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Fengler</div> <div class="given-names">K. A.</div></div><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Allelic genome structural variations in maize detected by array comparative genome hybridization</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">120</div>, <div class="fpage">355</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20genome%20structural%20variations%20in%20maize%20detected%20by%20array%20comparative%20genome%20hybridization&amp;author=A.%20Belo&amp;author=M.%20K.%20Beatty&amp;author=D.%20Hondred&amp;author=K.%20A.%20Fengler&amp;author=B.%20Li&amp;author=A.%20Rafalski&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet&amp;volume=120&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1128-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1128-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1128-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19756477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Springer</div> <div class="given-names">N. M.</div></div><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content,</div> <div class="source">PLoS Genet</div> , <div class="volume">5</div>, <div class="fpage">e1000734</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maize%20inbreds%20exhibit%20high%20levels%20of%20copy%20number%20variation%20(CNV)%20and%20presence%2Fabsence%20variation%20(PAV)%20in%20genome%20content%2C&amp;author=N.%20M.%20Springer&amp;author=K.%20Ying&amp;author=Y.%20Fu&amp;publication_year=2009&amp;journal=PLoS%20Genet&amp;volume=5&amp;pages=e1000734" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19956538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Swanson-Wagner</div> <div class="given-names">R. A.</div></div><div class="name string-name"><div class="surname">Eichten</div> <div class="given-names">S. R.</div></div><div class="name string-name"><div class="surname">Kumari</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Pervasive gene content variation and copy number variation in maize and its undomesticated progenitor,</div> <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1689</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pervasive%20gene%20content%20variation%20and%20copy%20number%20variation%20in%20maize%20and%20its%20undomesticated%20progenitor%2C&amp;author=R.%20A.%20Swanson-Wagner&amp;author=S.%20R.%20Eichten&amp;author=S.%20Kumari&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1689-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.109165.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.109165.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.109165.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21036921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quail</div> <div class="given-names">M. A.</div></div><div class="name string-name"><div class="surname">Kozarewa</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">A large genome center's improvements to the Illumina sequencing system,</div> <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">1005</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large%20genome%20center's%20improvements%20to%20the%20Illumina%20sequencing%20system%2C&amp;author=M.%20A.%20Quail&amp;author=I.%20Kozarewa&amp;author=F.%20Smith&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=1005-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1270" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1270" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19034268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Landan</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Graur</div> <div class="given-names">D.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Heads or tails: a simple reliability check for multiple sequence alignments,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1380</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heads%20or%20tails%3A%20a%20simple%20reliability%20check%20for%20multiple%20sequence%20alignments%2C&amp;author=G.%20Landan&amp;author=D.%20Graur&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1380-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm060" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm060" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm060"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17387100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw050-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw050-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blankenberg</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Von Kuster</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Coraor</div> <div class="given-names">N.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Galaxy: a web-based genome analysis tool for experimentalists,</div> <div class="source">Curr. Protoc. Mol. Biol</div> ., Chapter 19, Unit 19.10.11-21.</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2656160&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw050/1/dsw050_Supp.zip?Expires=2147483647&amp;Signature=lTKHwTzpefrkcPd6Pqu23S2D37ZuptdkETSg8WPGNXqCOlonlfiryvN9TGc8JI1Jwt3NbYaYqZ3MR8i1oVNINcoZOn2P~R~BcAxuEzmSfriBU9pmQf1DZYUu96ZCnsxpe2aINGBxTc5Um27ffvYF757cIooVHvA9QeUjDv0B9hY--3qv9gviBaaSFIzuCMjQMUFtXxyfY6~xzEG742zWvReY3NcW5Jh~dyvOQLhX1iseof0URqvnV7B1G0orbChiQ~SfuVC8IX3dNb6URoOgCsdVpE~yRt6hTbBt7ApNhLHbfY4I8hSp2p5yPS7401LinQYfZGKbjiM8SRHgQpK~3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">560</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw050"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2656160"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f81%2f2656160">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000397051500008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F81%2F2656160" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2656160'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Exploiting the great potential of Sequence Capture data by a new tool, SUPER-CAP" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw050");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f81%2f2656160">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

