

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/5/471/359636","@type":"ScholarlyArticle","name":"Identification and Evolution of the Silkworm  Helitrons  and their Contribution to Transcripts","datePublished":"2013-06-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2013-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst024","keywords":["<italic>Helitron</italic>","silkworm","gene fragment acquisition","evolution","transcript"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/5/471/359636","author":[{"name":"Han, Min-Jin","affiliation":"1  State Key Laboratory of Silkworm Genome Biology ,  The Key Sericultural Laboratory of Agricultural Ministry, Southwest University ,  Chongqing  400715 ,  China","@type":"Person"},{"name":"Shen, Yi-Hong","affiliation":"1  State Key Laboratory of Silkworm Genome Biology ,  The Key Sericultural Laboratory of Agricultural Ministry, Southwest University ,  Chongqing  400715 ,  China","@type":"Person"},{"name":"Xu, Meng-Shu","affiliation":"1  State Key Laboratory of Silkworm Genome Biology ,  The Key Sericultural Laboratory of Agricultural Ministry, Southwest University ,  Chongqing  400715 ,  China","@type":"Person"},{"name":"Liang, Hong-Yu","affiliation":"1  State Key Laboratory of Silkworm Genome Biology ,  The Key Sericultural Laboratory of Agricultural Ministry, Southwest University ,  Chongqing  400715 ,  China","@type":"Person"},{"name":"Zhang, Hua-Hao","affiliation":"2  School of Life Sciences ,  Chongqing University ,  Chongqing  400044 ,  China","@type":"Person"},{"name":"Zhang, Ze","affiliation":"2  School of Life Sciences ,  Chongqing University ,  Chongqing  400044 ,  China","@type":"Person"}],"description":"Abstract.  In this study, we developed a structure-based approach to identify Helitrons in four lepidopterans and systematically analysed Helitrons in the silkw","pageStart":"471","pageEnd":"484","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification and Evolution of the Silkworm  Helitrons  and their Contribution to Transcripts","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification and Evolution of the Silkworm  Helitrons  and their Contribution to Transcripts" />
    <meta property="og:description" content="Abstract.  In this study, we developed a structure-based approach to identify Helitrons in four lepidopterans and systematically analysed Helitrons in the silkw" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/5/471/359636" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-06-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Han, Min-Jin" /><meta name="citation_author_institution" content="State Key Laboratory of Silkworm Genome Biology, The Key Sericultural Laboratory of Agricultural Ministry, Southwest University, Chongqing400715, China" /><meta name="citation_author" content="Shen, Yi-Hong" /><meta name="citation_author_institution" content="State Key Laboratory of Silkworm Genome Biology, The Key Sericultural Laboratory of Agricultural Ministry, Southwest University, Chongqing400715, China" /><meta name="citation_author" content="Xu, Meng-Shu" /><meta name="citation_author_institution" content="State Key Laboratory of Silkworm Genome Biology, The Key Sericultural Laboratory of Agricultural Ministry, Southwest University, Chongqing400715, China" /><meta name="citation_author" content="Liang, Hong-Yu" /><meta name="citation_author_institution" content="State Key Laboratory of Silkworm Genome Biology, The Key Sericultural Laboratory of Agricultural Ministry, Southwest University, Chongqing400715, China" /><meta name="citation_author" content="Zhang, Hua-Hao" /><meta name="citation_author_institution" content="School of Life Sciences, Chongqing University, Chongqing400044, China" /><meta name="citation_author" content="Zhang, Ze" /><meta name="citation_author_institution" content="School of Life Sciences, Chongqing University, Chongqing400044, China" /><meta name="citation_title" content="Identification and Evolution of the Silkworm Helitrons and their Contribution to Transcripts" /><meta name="citation_firstpage" content="471" /><meta name="citation_lastpage" content="484" /><meta name="citation_doi" content="10.1093/dnares/dst024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2013/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Rolling-circle transposons in eukaryotes; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8714-19; " /><meta name="citation_reference" content="citation_title=Differential roles of the transposon termini in IS91 transposition; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=1922-26; " /><meta name="citation_reference" content="citation_title=Helitron-N1_XT, a family of non-autonomous Helitron from frog; Repbase Reports;  citation_year=2006;  citation_volume=6;  citation_pages=494-6; " /><meta name="citation_reference" content="citation_title=Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization; Science;  citation_year=2007;  citation_volume=317;  citation_pages=86-94; " /><meta name="citation_reference" content="citation_title=Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzae; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1105-15; " /><meta name="citation_reference" content="citation_title=Massive amplification of rolling-circle transposons in the lineage of the bat Myotis lucifugus; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=1895-900; " /><meta name="citation_reference" content="citation_title=The polychromatic Helitron landscape of the maize genome; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=19916-21; " /><meta name="citation_reference" content="citation_title=Distribution, diversity, evolution, and survival of Helitrons in the maize genome; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=19922-7; " /><meta name="citation_reference" content="citation_title=Helitrons in fruit flies; Repbase Reports;  citation_year=2007;  citation_volume=7;  citation_pages=127-32; " /><meta name="citation_reference" content="citation_title=Transposable element distribution, abundance and role in genome size variation in the genus Oryza; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=152" /><meta name="citation_reference" content="citation_title=Structure-based discovery and description of plant and animal Helitrons; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=12832-7; " /><meta name="citation_reference" content="citation_title=Gene capture by Helitron transposons reshuffles the transcriptome of maize; Genetics;  citation_year=2012;  citation_volume=190;  citation_pages=965-75; " /><meta name="citation_reference" content="citation_title=Gene duplication and exon shuffling by Helitron-like transposons generate intraspecies diversity in maize; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=997-1002; " /><meta name="citation_reference" content="citation_title=A draft sequence for the genome of the domesticated silkworm (Bombyx mori); Science;  citation_year=2004;  citation_volume=306;  citation_pages=1937-40; " /><meta name="citation_reference" content="citation_title=The monarch butterfly genome yields insights into long-distance migration; Cell;  citation_year=2011;  citation_volume=147;  citation_pages=1171-85; " /><meta name="citation_reference" content="citation_title=Butterfly genome reveals promiscuous exchange of mimicry adaptations among species; Nature;  citation_year=2012;  citation_volume=487;  citation_pages=94-8; " /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, New York; Evolution of the insects;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Genome-wide screening and characterization of transposable elements and their distribution analysis in the silkworm, Bombyx mori; Insect. Biochem. Mol. Biol.;  citation_year=2008;  citation_volume=38;  citation_pages=1046-57; " /><meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-1; " /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=DAMBE: software package for data analysis in molecular biology and evolution; J. Hered.;  citation_year=2001;  citation_volume=92;  citation_pages=371-3; " /><meta name="citation_reference" content="citation_title=UNAFold: software for nucleic acid folding and hybridization; Methods Mol. Biol.;  citation_year=2008;  citation_volume=453;  citation_pages=3-31; " /><meta name="citation_reference" content="citation_title=ISfinder: the reference centre for bacterial insertion sequences; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D32-6; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements; , Cytogenet. Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="citation_title=Genome fingerprinting of the silkworm, Bombyx mori, using random arbitrary primers; , Electrophoresis;  citation_year=1995;  citation_volume=16;  citation_pages=1633-8; " /><meta name="citation_reference" content="citation_title=A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide-sequences; J. Mol. Evol.;  citation_year=1980;  citation_volume=16;  citation_pages=111-20; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland, Massachusetts; Molecular evolution;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Complete resequencing of 40 genomes reveals domestication events and genes in silkworm (Bombyx); Science;  citation_year=2009;  citation_volume=326;  citation_pages=433-6; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis; Evol. Bioinform. Online;  citation_year=2005;  citation_volume=1;  citation_pages=47-50; " /><meta name="citation_reference" content="citation_title=SilkDB v2.0: a platform for silkworm (Bombyx mori) genome biology; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=453-6; " /><meta name="citation_reference" content="citation_title=Burst expansion, distribution and diversification of MITEs in the silkworm genome; BMC genomics;  citation_year=2010;  citation_volume=11;  citation_pages=520" /><meta name="citation_reference" content="citation_title=Pervasive horizontal transfer of rolling-circle transposons among animals; Genome Biol. Evol.;  citation_year=2010;  citation_volume=2;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Mobilizing the genome of Lepidoptera through novel sequence gains and end creation by non-autonomous Lep1 Helitrons; DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=11-21; " /><meta name="citation_reference" content="citation_title=Contrasting evolutionary patterns and target specificities among three Tourist-like MITE families in the maize genome; Plant Mol. Biol.;  citation_year=2009;  citation_volume=71;  citation_pages=99-114; " /><meta name="citation_reference" content="citation_title=Population growth makes waves in the distribution of pairwise genetic differences; Mol. Biol. Evol.;  citation_year=1992;  citation_volume=9;  citation_pages=552-69; " /><meta name="citation_reference" content="citation_title=A comparative computational analysis of nonautonomous Helitron elements between maize and rice; BMC genomics;  citation_year=2008;  citation_volume=9;  citation_pages=467" /><meta name="citation_reference" content="citation_title=A Helitron-Like Transposon Superfamily from Lepidoptera Disrupts (GAAA)(n) microsatellites and is responsible for flanking sequence similarity within a microsatellite family; J. Mol. Evol.;  citation_year=2010;  citation_volume=70;  citation_pages=275-88; " /><meta name="citation_reference" content="Smit A.F.A. 2006 Repeat Masker edition, Release 20061006 , Repbase Update " /><meta name="citation_reference" content="citation_title=Population and evolutionary dynamics of Helitron transposable elements in Arabidopsis thaliana; Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=2515-24; " /><meta name="citation_reference" content="citation_title=Tuned for transposition: molecular determinants underlying the hyperactivity of a Stowaway MITE; Science;  citation_year=2009;  citation_volume=325;  citation_pages=1391-4; " /><meta name="citation_reference" content="citation_title=An active DNA transposon family in rice; Nature;  citation_year=2003;  citation_volume=421;  citation_pages=163-7; " /><meta name="citation_reference" content="citation_title=Mobilization of a transposon in the rice genome; Nature;  citation_year=2003;  citation_volume=421;  citation_pages=170-2; " /><meta name="citation_reference" content="citation_title=Dramatic amplification of a rice transposable element during recent domestication; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=17620-5; " /><meta name="citation_reference" content="citation_title=Phylogenetic aspects on the origin of Japanese race of the silkworm, Bombyx mori L; J. Sericol. Sci. Jpn;  citation_year=1968;  citation_volume=37;  citation_pages=83-7; " /><meta name="citation_reference" content="citation_title=Phylogeny and evolutionary history of the silkworm; Sci. China Life Sci.;  citation_year=2012;  citation_volume=55;  citation_pages=483-96; " /><meta name="citation_reference" content="citation_title=A cornucopia of Helitrons shapes the maize genome; Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=19747-8; " /><meta name="citation_reference" content="citation_title=Treasures in the attic: rolling circle transposons discovered in eukaryotic genomes; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8923-4; " /><meta name="citation_reference" content="citation_title=Transposable elements, gene creation and genome rearrangement in flowering plants; Curr. Opin. Genet. Dev.;  citation_year=2005;  citation_volume=15;  citation_pages=621-7; " /><meta name="citation_reference" content="citation_title=Origins, genetic organization and transcription of a family of non-autonomous Helitron elements in maize; Plant J.;  citation_year=2005;  citation_volume=43;  citation_pages=799-810; " /><meta name="citation_reference" content="citation_title=Helitrons: Enigmatic abductors and mobilizers of host genome sequences; Plant Sci.;  citation_year=2009;  citation_volume=176;  citation_pages=181-6; " /><meta name="citation_reference" content="citation_title=Structural characterization of Helitrons and their stepwise capturing of gene fragments in the maize genome; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=609" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/5/471/1029513/dst024.pdf" /><meta name="description" content="Abstract.  In this study, we developed a structure-based approach to identify Helitrons in four lepidopterans and systematically analysed Helitrons in the silkw" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/5/471/359636" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/5/471/359636" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/5/471/359636");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst024';
    </script>
<title>
	Identification and Evolution of the Silkworm Helitrons and their Contribution to Transcripts | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359636" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be1sPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW4sIE1pbi1KaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBTaWxrd29ybSBHZW5vbWUgQmlvbG9neSwgVGhlIEtleSBTZXJpY3VsdHVyYWwgTGFib3JhdG9yeSBvZiBBZ3JpY3VsdHVyYWwgTWluaXN0cnksIFNvdXRod2VzdCBVbml2ZXJzaXR5LCBDaG9uZ3Fpbmc0MDA3MTUsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hlbiwgWWktSG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFNpbGt3b3JtIEdlbm9tZSBCaW9sb2d5LCBUaGUgS2V5IFNlcmljdWx0dXJhbCBMYWJvcmF0b3J5IG9mIEFncmljdWx0dXJhbCBNaW5pc3RyeSwgU291dGh3ZXN0IFVuaXZlcnNpdHksIENob25ncWluZzQwMDcxNSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJYdSwgTWVuZy1TaHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBTaWxrd29ybSBHZW5vbWUgQmlvbG9neSwgVGhlIEtleSBTZXJpY3VsdHVyYWwgTGFib3JhdG9yeSBvZiBBZ3JpY3VsdHVyYWwgTWluaXN0cnksIFNvdXRod2VzdCBVbml2ZXJzaXR5LCBDaG9uZ3Fpbmc0MDA3MTUsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlhbmcsIEhvbmctWXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBTaWxrd29ybSBHZW5vbWUgQmlvbG9neSwgVGhlIEtleSBTZXJpY3VsdHVyYWwgTGFib3JhdG9yeSBvZiBBZ3JpY3VsdHVyYWwgTWluaXN0cnksIFNvdXRod2VzdCBVbml2ZXJzaXR5LCBDaG9uZ3Fpbmc0MDA3MTUsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIEh1YS1IYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgQ2hvbmdxaW5nIFVuaXZlcnNpdHksIENob25ncWluZzQwMDA0NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgWmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgQ2hvbmdxaW5nIFVuaXZlcnNpdHksIENob25ncWluZzQwMDA0NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IklkZW50aWZpY2F0aW9uIGFuZCBFdm9sdXRpb24gb2YgdGhlIFNpbGt3b3JtIEhlbGl0cm9ucyBhbmQgdGhlaXIgQ29udHJpYnV0aW9uIHRvIFRyYW5zY3JpcHRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0ODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGxpbmctY2lyY2xlIHRyYW5zcG9zb25zIGluIGV1a2FyeW90ZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9ODcxNC0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIHJvbGVzIG9mIHRoZSB0cmFuc3Bvc29uIHRlcm1pbmkgaW4gSVM5MSB0cmFuc3Bvc2l0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTE5MjItMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlbGl0cm9uLU4xX1hULCBhIGZhbWlseSBvZiBub24tYXV0b25vbW91cyBIZWxpdHJvbiBmcm9tIGZyb2c7IFJlcGJhc2UgUmVwb3J0czsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDk0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlYSBhbmVtb25lIGdlbm9tZSByZXZlYWxzIGFuY2VzdHJhbCBldW1ldGF6b2FuIGdlbmUgcmVwZXJ0b2lyZSBhbmQgZ2Vub21pYyBvcmdhbml6YXRpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE3OyAgY2l0YXRpb25fcGFnZXM9ODYtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgb2YgQXNwZXJnaWxsdXMgbmlkdWxhbnMgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIHdpdGggQS4gZnVtaWdhdHVzIGFuZCBBLiBvcnl6YWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlIGFtcGxpZmljYXRpb24gb2Ygcm9sbGluZy1jaXJjbGUgdHJhbnNwb3NvbnMgaW4gdGhlIGxpbmVhZ2Ugb2YgdGhlIGJhdCBNeW90aXMgbHVjaWZ1Z3VzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTE4OTUtOTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcG9seWNocm9tYXRpYyBIZWxpdHJvbiBsYW5kc2NhcGUgb2YgdGhlIG1haXplIGdlbm9tZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xOTkxNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdHJpYnV0aW9uLCBkaXZlcnNpdHksIGV2b2x1dGlvbiwgYW5kIHN1cnZpdmFsIG9mIEhlbGl0cm9ucyBpbiB0aGUgbWFpemUgZ2Vub21lOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE5OTIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlbGl0cm9ucyBpbiBmcnVpdCBmbGllczsgUmVwYmFzZSBSZXBvcnRzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50IGRpc3RyaWJ1dGlvbiwgYWJ1bmRhbmNlIGFuZCByb2xlIGluIGdlbm9tZSBzaXplIHZhcmlhdGlvbiBpbiB0aGUgZ2VudXMgT3J5emE7IEJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLWJhc2VkIGRpc2NvdmVyeSBhbmQgZGVzY3JpcHRpb24gb2YgcGxhbnQgYW5kIGFuaW1hbCBIZWxpdHJvbnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTI4MzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBjYXB0dXJlIGJ5IEhlbGl0cm9uIHRyYW5zcG9zb25zIHJlc2h1ZmZsZXMgdGhlIHRyYW5zY3JpcHRvbWUgb2YgbWFpemU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTk2NS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkdXBsaWNhdGlvbiBhbmQgZXhvbiBzaHVmZmxpbmcgYnkgSGVsaXRyb24tbGlrZSB0cmFuc3Bvc29ucyBnZW5lcmF0ZSBpbnRyYXNwZWNpZXMgZGl2ZXJzaXR5IGluIG1haXplOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTk5Ny0xMDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRyYWZ0IHNlcXVlbmNlIGZvciB0aGUgZ2Vub21lIG9mIHRoZSBkb21lc3RpY2F0ZWQgc2lsa3dvcm0gKEJvbWJ5eCBtb3JpKTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDY7ICBjaXRhdGlvbl9wYWdlcz0xOTM3LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9uYXJjaCBidXR0ZXJmbHkgZ2Vub21lIHlpZWxkcyBpbnNpZ2h0cyBpbnRvIGxvbmctZGlzdGFuY2UgbWlncmF0aW9uOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTExNzEtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1dHRlcmZseSBnZW5vbWUgcmV2ZWFscyBwcm9taXNjdW91cyBleGNoYW5nZSBvZiBtaW1pY3J5IGFkYXB0YXRpb25zIGFtb25nIHNwZWNpZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODc7ICBjaXRhdGlvbl9wYWdlcz05NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyBFdm9sdXRpb24gb2YgdGhlIGluc2VjdHM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHNjcmVlbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHMgYW5kIHRoZWlyIGRpc3RyaWJ1dGlvbiBhbmFseXNpcyBpbiB0aGUgc2lsa3dvcm0sIEJvbWJ5eCBtb3JpOyBJbnNlY3QuIEJpb2NoZW0uIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMDQ2LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWFyY2ggYW5kIGNsdXN0ZXJpbmcgb3JkZXJzIG9mIG1hZ25pdHVkZSBmYXN0ZXIgdGhhbiBCTEFTVDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0yNDYwLTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1VU0NMRTogbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHdpdGggaGlnaCBhY2N1cmFjeSBhbmQgaGlnaCB0aHJvdWdocHV0OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xNzkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURBTUJFOiBzb2Z0d2FyZSBwYWNrYWdlIGZvciBkYXRhIGFuYWx5c2lzIGluIG1vbGVjdWxhciBiaW9sb2d5IGFuZCBldm9sdXRpb247IEouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTM3MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VTkFGb2xkOiBzb2Z0d2FyZSBmb3IgbnVjbGVpYyBhY2lkIGZvbGRpbmcgYW5kIGh5YnJpZGl6YXRpb247IE1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTM7ICBjaXRhdGlvbl9wYWdlcz0zLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JU2ZpbmRlcjogdGhlIHJlZmVyZW5jZSBjZW50cmUgZm9yIGJhY3RlcmlhbCBpbnNlcnRpb24gc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1EMzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYmFzZSBVcGRhdGUsIGEgZGF0YWJhc2Ugb2YgZXVrYXJ5b3RpYyByZXBldGl0aXZlIGVsZW1lbnRzOyAsIEN5dG9nZW5ldC4gR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9NDYyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBmaW5nZXJwcmludGluZyBvZiB0aGUgc2lsa3dvcm0sIEJvbWJ5eCBtb3JpLCB1c2luZyByYW5kb20gYXJiaXRyYXJ5IHByaW1lcnM7ICwgRWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTYzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtZXRob2QgZm9yIGVzdGltYXRpbmcgZXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGJhc2Ugc3Vic3RpdHV0aW9ucyB0aHJvdWdoIGNvbXBhcmF0aXZlIHN0dWRpZXMgb2YgbnVjbGVvdGlkZS1zZXF1ZW5jZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZXMsIFN1bmRlcmxhbmQsIE1hc3NhY2h1c2V0dHM7IE1vbGVjdWxhciBldm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHJlc2VxdWVuY2luZyBvZiA0MCBnZW5vbWVzIHJldmVhbHMgZG9tZXN0aWNhdGlvbiBldmVudHMgYW5kIGdlbmVzIGluIHNpbGt3b3JtIChCb21ieXgpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNjsgIGNpdGF0aW9uX3BhZ2VzPTQzMy02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyAoTUVHQSkgc29mdHdhcmUgdmVyc2lvbiA0LjA7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFybGVxdWluICh2ZXJzaW9uIDMuMCk6IGFuIGludGVncmF0ZWQgc29mdHdhcmUgcGFja2FnZSBmb3IgcG9wdWxhdGlvbiBnZW5ldGljcyBkYXRhIGFuYWx5c2lzOyBFdm9sLiBCaW9pbmZvcm0uIE9ubGluZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NDctNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbGtEQiB2Mi4wOiBhIHBsYXRmb3JtIGZvciBzaWxrd29ybSAoQm9tYnl4IG1vcmkpIGdlbm9tZSBiaW9sb2d5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz00NTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnVyc3QgZXhwYW5zaW9uLCBkaXN0cmlidXRpb24gYW5kIGRpdmVyc2lmaWNhdGlvbiBvZiBNSVRFcyBpbiB0aGUgc2lsa3dvcm0gZ2Vub21lOyBCTUMgZ2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01MjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJ2YXNpdmUgaG9yaXpvbnRhbCB0cmFuc2ZlciBvZiByb2xsaW5nLWNpcmNsZSB0cmFuc3Bvc29ucyBhbW9uZyBhbmltYWxzOyBHZW5vbWUgQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9iaWxpemluZyB0aGUgZ2Vub21lIG9mIExlcGlkb3B0ZXJhIHRocm91Z2ggbm92ZWwgc2VxdWVuY2UgZ2FpbnMgYW5kIGVuZCBjcmVhdGlvbiBieSBub24tYXV0b25vbW91cyBMZXAxIEhlbGl0cm9uczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJhc3RpbmcgZXZvbHV0aW9uYXJ5IHBhdHRlcm5zIGFuZCB0YXJnZXQgc3BlY2lmaWNpdGllcyBhbW9uZyB0aHJlZSBUb3VyaXN0LWxpa2UgTUlURSBmYW1pbGllcyBpbiB0aGUgbWFpemUgZ2Vub21lOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9OTktMTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGdyb3d0aCBtYWtlcyB3YXZlcyBpbiB0aGUgZGlzdHJpYnV0aW9uIG9mIHBhaXJ3aXNlIGdlbmV0aWMgZGlmZmVyZW5jZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU1Mi02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJhdGl2ZSBjb21wdXRhdGlvbmFsIGFuYWx5c2lzIG9mIG5vbmF1dG9ub21vdXMgSGVsaXRyb24gZWxlbWVudHMgYmV0d2VlbiBtYWl6ZSBhbmQgcmljZTsgQk1DIGdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEhlbGl0cm9uLUxpa2UgVHJhbnNwb3NvbiBTdXBlcmZhbWlseSBmcm9tIExlcGlkb3B0ZXJhIERpc3J1cHRzIChHQUFBKShuKSBtaWNyb3NhdGVsbGl0ZXMgYW5kIGlzIHJlc3BvbnNpYmxlIGZvciBmbGFua2luZyBzZXF1ZW5jZSBzaW1pbGFyaXR5IHdpdGhpbiBhIG1pY3Jvc2F0ZWxsaXRlIGZhbWlseTsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTI3NS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iU21pdCBBLkYuQS4gMjAwNiBSZXBlYXQgTWFza2VyIGVkaXRpb24sIFJlbGVhc2UgMjAwNjEwMDYgLCBSZXBiYXNlIFVwZGF0ZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIGFuZCBldm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgSGVsaXRyb24gdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBNb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yNTE1LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UdW5lZCBmb3IgdHJhbnNwb3NpdGlvbjogbW9sZWN1bGFyIGRldGVybWluYW50cyB1bmRlcmx5aW5nIHRoZSBoeXBlcmFjdGl2aXR5IG9mIGEgU3Rvd2F3YXkgTUlURTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjU7ICBjaXRhdGlvbl9wYWdlcz0xMzkxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFjdGl2ZSBETkEgdHJhbnNwb3NvbiBmYW1pbHkgaW4gcmljZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyMTsgIGNpdGF0aW9uX3BhZ2VzPTE2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGl6YXRpb24gb2YgYSB0cmFuc3Bvc29uIGluIHRoZSByaWNlIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyMTsgIGNpdGF0aW9uX3BhZ2VzPTE3MC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFtYXRpYyBhbXBsaWZpY2F0aW9uIG9mIGEgcmljZSB0cmFuc3Bvc2FibGUgZWxlbWVudCBkdXJpbmcgcmVjZW50IGRvbWVzdGljYXRpb247IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTc2MjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFzcGVjdHMgb24gdGhlIG9yaWdpbiBvZiBKYXBhbmVzZSByYWNlIG9mIHRoZSBzaWxrd29ybSwgQm9tYnl4IG1vcmkgTDsgSi4gU2VyaWNvbC4gU2NpLiBKcG47ICBjaXRhdGlvbl95ZWFyPTE5Njg7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz04My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbnkgYW5kIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIHRoZSBzaWxrd29ybTsgU2NpLiBDaGluYSBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz00ODMtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29ybnVjb3BpYSBvZiBIZWxpdHJvbnMgc2hhcGVzIHRoZSBtYWl6ZSBnZW5vbWU7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTE5NzQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyZWFzdXJlcyBpbiB0aGUgYXR0aWM6IHJvbGxpbmcgY2lyY2xlIHRyYW5zcG9zb25zIGRpc2NvdmVyZWQgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9ODkyMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc3Bvc2FibGUgZWxlbWVudHMsIGdlbmUgY3JlYXRpb24gYW5kIGdlbm9tZSByZWFycmFuZ2VtZW50IGluIGZsb3dlcmluZyBwbGFudHM7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NjIxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbnMsIGdlbmV0aWMgb3JnYW5pemF0aW9uIGFuZCB0cmFuc2NyaXB0aW9uIG9mIGEgZmFtaWx5IG9mIG5vbi1hdXRvbm9tb3VzIEhlbGl0cm9uIGVsZW1lbnRzIGluIG1haXplOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTc5OS04MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlbGl0cm9uczogRW5pZ21hdGljIGFiZHVjdG9ycyBhbmQgbW9iaWxpemVycyBvZiBob3N0IGdlbm9tZSBzZXF1ZW5jZXM7IFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgY2hhcmFjdGVyaXphdGlvbiBvZiBIZWxpdHJvbnMgYW5kIHRoZWlyIHN0ZXB3aXNlIGNhcHR1cmluZyBvZiBnZW5lIGZyYWdtZW50cyBpbiB0aGUgbWFpemUgZ2Vub21lOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02MDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvNS80NzEvMTAyOTUxMy9kc3QwMjQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEluIHRoaXMgc3R1ZHksIHdlIGRldmVsb3BlZCBhIHN0cnVjdHVyZS1iYXNlZCBhcHByb2FjaCB0byBpZGVudGlmeSBIZWxpdHJvbnMgaW4gZm91ciBsZXBpZG9wdGVyYW5zIGFuZCBzeXN0ZW1hdGljYWxseSBhbmFseXNlZCBIZWxpdHJvbnMgaW4gdGhlIHNpbGt3IiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTk2MzYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk0xV7H/K/TAKAFyt6C8mT4UFFutw=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f471%2f359636" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f471%2f359636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13963" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&amp;Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/5">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5185488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185488">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185490">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185490">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185496">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185496">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185514">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185514">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185543">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185543">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185565">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185565">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185567">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185567">Authors' contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185569">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185569">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185571">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185571">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185573">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185573">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5185575">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5185575">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/5/461/359543">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/5/485/359815">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification and Evolution of the Silkworm <em>Helitrons</em> and their Contribution to Transcripts
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Min-Jin Han</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory of Silkworm Genome Biology</div>, <div class="insititution">The Key Sericultural Laboratory of Agricultural Ministry, Southwest University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400715</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Min-Jin+Han">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Han M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Han M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Min-Jin Han</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yi-Hong Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory of Silkworm Genome Biology</div>, <div class="insititution">The Key Sericultural Laboratory of Agricultural Ministry, Southwest University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400715</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yi-Hong+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shen Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yi-Hong Shen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meng-Shu Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory of Silkworm Genome Biology</div>, <div class="insititution">The Key Sericultural Laboratory of Agricultural Ministry, Southwest University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400715</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meng-Shu+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meng-Shu Xu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hong-Yu Liang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory of Silkworm Genome Biology</div>, <div class="insititution">The Key Sericultural Laboratory of Agricultural Ministry, Southwest University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400715</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hong-Yu+Liang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hong-Yu Liang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hua-Hao Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">Chongqing University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400044</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hua-Hao+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hua-Hao Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ze Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">Chongqing University</div>, <div class="addr-line">Chongqing</div><div class="addr-line">400044</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86-23-65122685. E-mail: <a href="mailto:zezhang@cqu.edu.cn" target="_blank">zezhang@cqu.edu.cn</a>, <a href="mailto:ze_zhang@126.com" target="_blank">ze_zhang@126.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ze+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ze Zhang</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 471–484, <a href='https://doi.org/10.1093/dnares/dst024'>https://doi.org/10.1093/dnares/dst024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 June 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 December 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 May 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359636" href="/dnaresearch/article-pdf/20/5/471/1029513/dst024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Min-Jin Han, Yi-Hong Shen, Meng-Shu Xu, Hong-Yu Liang, Hua-Hao Zhang, Ze Zhang;  Identification and Evolution of the Silkworm <em>Helitrons</em> and their Contribution to Transcripts, <em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 471–484, <a href='https://doi.org/10.1093/dnares/dst024'>https://doi.org/10.1093/dnares/dst024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359636?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359636?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20and%20Evolution%20of%20the%20Silkworm%20%26lt%3Bem%26gt%3BHelitrons%26lt%3B%2Fem%26gt%3B%20and%20their%20Contribution%20to%20Transcripts&amp;publicationDate=2013-06-14&amp;volumeNum=20&amp;issueNum=5&amp;author=Han%2C%20Min-Jin%3B%20Shen%2C%20Yi-Hong&amp;startPage=471&amp;endPage=484&amp;contentId=10.1093%2Fdnares%2Fdst024&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f471%2f359636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5185488 id="5185488" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, we developed a structure-based approach to identify <em>Helitrons</em> in four lepidopterans and systematically analysed <em>Helitrons</em> in the silkworm genome. We found that the content of <em>Helitrons</em> varied greatly among genomes. The silkworm genome harboured 67 555 <em>Helitron</em>-related sequences that could be classified into 21 families and accounted for ∼4.23% of the genome. Thirteen of the families were new. Three families were putatively autonomous and included the replication initiator motif and helicase domain. The silkworm <em>Helitrons</em> were widely and randomly distributed in the genome. Most <em>Helitron</em> families radiated within the past 2 million years and experienced a single burst of expansion. These <em>Helitron</em> families captured 3724 gene fragments and contributed to at least 1.4% of the silkworm full-length cDNAs, suggesting important roles of <em>Helitrons</em> in the evolution of the silkworm genes. In addition, we found that some new <em>Helitrons</em> were generated by combinations of other <em>Helitrons</em>. Overall, the results presented in this study provided insights into the generation and evolution of <em>Helitron</em> transposons and their contribution to transcripts.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Helitron"><em>Helitron</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="silkworm">silkworm</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene fragment acquisition&quot;">gene fragment acquisition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcript">transcript</a></div>                            <h2 scrollto-destination=5185490 id="5185490" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Helitrons</em>, originally discovered in the genomes of the model organisms <em>Arabidopsis</em>, rice, and <em>Caenorhabditis elegans</em>, are classified as DNA transposable elements (TEs).<sup><a reveal-id="DST024C1" data-open="DST024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, their sequence structures and mechanisms of transposition are completely different from those of other DNA transposons. They are thought to transpose via a rolling circle mechanism, because some intact <em>Helitrons</em> encode proteins that include the replication initiator (Rep) motif and helicase domain. These two genetic elements are necessary for bacterial IS91 (insertion sequence 91) transposition through a rolling circle mechanism.<sup><a reveal-id="DST024C2" data-open="DST024C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Although <em>Helitrons</em> do not have terminal inverted repeats or target site duplications, they preferentially insert into the dinucleotide AT and are characterized by a TC dinucleotide at the 5′-end, a CTRR motif at the 3′-end, and often a palindromic sequence near the 3′-terminus.</p><p>In recent years, <em>Helitrons</em> have been identified in almost all eukaryotic genomes. They constitute 0–5% of total genomic DNA in some model organisms. For example, they comprise &gt;2% of the genome in <em>C. elegans</em>,<sup><a reveal-id="DST024C1" data-open="DST024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> &gt;0.5% in frog,<sup><a reveal-id="DST024C3" data-open="DST024C3" class="link link-ref link-reveal xref-bibr">3</a></sup> ∼3% in <em>Nematostella vectensis</em>,<sup><a reveal-id="DST024C4" data-open="DST024C4" class="link link-ref link-reveal xref-bibr">4</a></sup> &lt;0.1% in <em>Aspergillus nidulans</em>,<sup><a reveal-id="DST024C5" data-open="DST024C5" class="link link-ref link-reveal xref-bibr">5</a></sup> ∼3% in bat,<sup><a reveal-id="DST024C6" data-open="DST024C6" class="link link-ref link-reveal xref-bibr">6</a></sup> ∼2% in maize,<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and 1–5% in fruit flies.<sup><a reveal-id="DST024C9" data-open="DST024C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Furthermore, <em>Helitron</em> content is often highly variable among closely related species. For instance, they occupy 1–5% of genomic DNA in different fruit flies and 0.03–2.09% in different rice species.<sup><a reveal-id="DST024C9  DST024C11" data-open="DST024C9  DST024C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup></p><p><em>Helitrons</em> vary greatly in sequence length, even within the same <em>Helitron</em> family, in part because different gene fragments are captured by these elements. More than half of the <em>Helitrons</em> in the maize B73 genome contained gene fragments whose lengths ranged from tens of base pairs (bp) to ten or more kilobase pairs (kbp).<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Furthermore, genes captured by <em>Helitrons</em> reshuffled the transcriptome of maize.<sup><a reveal-id="DST024C12" data-open="DST024C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Hence, <em>Helitrons</em>, creating the diversity of coding regions, can lead to the evolution of new functional genes.<sup><a reveal-id="DST024C12" data-open="DST024C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST024C13" data-open="DST024C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Although an increasing number of <em>Helitrons</em> are being identified in eukaryotic genomes, little is known about <em>Helitrons</em> in Lepidoptera. Recently, the genome sequences of three lepidopterans, <em>Heliconius melpomene</em>, <em>Danaus plexippus</em> (both Nymphalidae), and <em>Manduca sexta</em> (Sphingidae) were released, in addition to the previously available silkworm (<em>Bombyx mori</em>; Bombycidae) genome.<sup><a reveal-id="DST024C14  DST024C16" data-open="DST024C14  DST024C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> Taken together, they provide an excellent resource for investigating <em>Helitrons</em> in Lepidoptera. The silkworm and <em>M. sexta</em> are moths, while <em>H. melpomene</em> and <em>D. plexippus</em> belong to butterfly. They diverged ∼100 million years ago (mya).<sup><a reveal-id="DST024C15  DST024C17" data-open="DST024C15  DST024C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup></p><p>The silkworm is a model insect for Lepidoptera and has important economic value for its silk and as a bioreactor. Approximately 40% of its genome consists of known TEs, with <em>Helitrons</em> comprising only 0.1%.<sup><a reveal-id="DST024C18" data-open="DST024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In this study, we developed a structure-based approach to rescan the new silkworm genome assembly to identify <em>Helitrons</em>. We found that the silkworm genome harbours 21 <em>Helitron</em> families that occupy ∼4.23% of the genomic DNA. Thirteen of these families are new and three are putative autonomous elements. Estimates of insertion date and diversity for each <em>Helitron</em> family suggested that most <em>Helitron</em> families experienced a single rapid expansion within the past 2 million years (my). Strikingly, these <em>Helitron</em> families captured 3724 fragments from 268 genes and contributed to at least 1.4% of silkworm full-length cDNAs. A comparative analysis of <em>Helitrons</em> within Lepidoptera was also performed.</p>                            <h2 scrollto-destination=5185496 id="5185496" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=5185497 id="5185497" class="section-title" data-legacy-id=s2a>2.1. Identification and characterization of <em>Helitrons</em></h3>
<p>Genome sequences were downloaded for the following Lepidopterans as indicated: silkworm new assembly from SilkDB (<a class="link link-uri" href="http://silkworm.swu.edu.cn/silkdb">http://silkworm.swu.edu.cn/silkdb</a>); <em>H. melpomene</em> from the <em>Heliconius</em> genome project (<a class="link link-uri" href="http://butterflygenome.org/">http://butterflygenome.org/</a>); <em>D. plexippus</em> V3 from MonarchBase (<a class="link link-uri" href="http://monarchbase.umassmed.edu/home.html">http://monarchbase.umassmed.edu/home.html</a>), and <em>M. sexta</em> from NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/nuccore/AIXA00000000">http://www.ncbi.nlm.nih.gov/nuccore/AIXA00000000</a>).</p><p>To identify <em>Helitrons</em>, a series of Perl scripts were written to search for <em>Helitron</em> sequence characteristics, similar to ‘HelSearch’.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Briefly, the method included four steps (Fig. <a reveal-id="DST024F1" data-open="DST024F1" class="link link-reveal link-table xref-fig">1</a>): (I) a Perl script found <em>Helitron</em> end structures, includes hairpins, loops, and CTRRT motifs; (II) another Perl script scanned upstream from <em>Helitron</em> end structure; (III) all sequences were clustered using Usearch;<sup><a reveal-id="DST024C19" data-open="DST024C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (IV) the <em>Helitron</em> boundaries were sought. In the step IV, we extended the sequences of each cluster in both directions using a Perl script and aligned them using MUSCLE,<sup><a reveal-id="DST024C20" data-open="DST024C20" class="link link-ref link-reveal xref-bibr">20</a></sup> then the <em>Helitron</em> boundaries were manually defined. Finally, we modified Yang and Bennetzen's method of classification.<sup><a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Sequences with identities &gt;80% in the 30 bp of both their 5′- and 3′-ends were classified as members of the same family. Full-length sequences with identity &gt;80% were classified in the same subfamily. Our programme and readme file are available upon request. </p>                        <a id="5185500" scrollto-destination="5185500"></a>
<div id="DST024F1" data-id="DST024F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02401.jpeg?Expires=2147483647&Signature=g5aTUbdZ2teDoUDgfJ4oX5YwFmh43w~C1AIolTKkSi5mLi5uGBGUFhakadd5-UFLLFG1rzIiujquge9y0i3GP0SGk~2b5BmrWGTtrlhjUux5RizFkZbd27QFs19pgYe7GZHyoPwP8REaR47eGXH5l2S~reeoc~lG42IOYJ1u-mk3FPI0wcoNVBUDnA0TCokJq39Orw2PikQLHm1VTbZZRpZOMPVYTBWndZImzxBV~bdCNm8P91BLOPGKxHTlZU1AZkxXvrrrO352aMPTVFcziafUmeVVpOlE92nF3EYaFapm-Ocd17xCyuOi-dOwSbv220GyFiJBXFubPDysjoVzOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Helitrons in the silkworm. (A) Helitron structure. (B) Pipeline for Helitron identification." path-from-xml="dst02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185500/dst02401.jpeg" path-from-xml="dst02401.jpeg" target="_blank">View large</a><a section="5185500" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02401.jpeg?Expires=2147483647&Signature=nqnTjgvfsjO3onebaB8Q~mLCRgssMvlKiCOl2IQg~GC8X0m4ChLhlhVYeIDSSzhd7fb9WtpDhBOUBs~YAXKgdqetcE784FDZ~J~HYCfPwbnepiUjV62O0l7ms2rw9Gso1PEpC9WYzsE4f5JgAdnWPYQx~vDCuIxCzUuAkNFSsFjJsxOrzQLh0OsN4jgsGcCFaNjxte-z-jcRbCRCJwnlrlzrpiP8ppoPAh3BhmIeeNhzRPGYW~q-Cz4OUf4V3xgZqaJMOAulpeDoxZsIszzb~0vge3H6EySvFCgTHlnNADhxRIN7mrGyL95FvJtL435-BzPCh56f1tp68hFy1SBJIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185500&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Helitrons</em> in the silkworm. (A) <em>Helitron</em> structure. (B) Pipeline for <em>Helitron</em> identification.</p></div></div><div content-id="DST024F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02401.jpeg?Expires=2147483647&Signature=g5aTUbdZ2teDoUDgfJ4oX5YwFmh43w~C1AIolTKkSi5mLi5uGBGUFhakadd5-UFLLFG1rzIiujquge9y0i3GP0SGk~2b5BmrWGTtrlhjUux5RizFkZbd27QFs19pgYe7GZHyoPwP8REaR47eGXH5l2S~reeoc~lG42IOYJ1u-mk3FPI0wcoNVBUDnA0TCokJq39Orw2PikQLHm1VTbZZRpZOMPVYTBWndZImzxBV~bdCNm8P91BLOPGKxHTlZU1AZkxXvrrrO352aMPTVFcziafUmeVVpOlE92nF3EYaFapm-Ocd17xCyuOi-dOwSbv220GyFiJBXFubPDysjoVzOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Helitrons in the silkworm. (A) Helitron structure. (B) Pipeline for Helitron identification." path-from-xml="dst02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185500/dst02401.jpeg" path-from-xml="dst02401.jpeg" target="_blank">View large</a><a section="5185500" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02401.jpeg?Expires=2147483647&Signature=nqnTjgvfsjO3onebaB8Q~mLCRgssMvlKiCOl2IQg~GC8X0m4ChLhlhVYeIDSSzhd7fb9WtpDhBOUBs~YAXKgdqetcE784FDZ~J~HYCfPwbnepiUjV62O0l7ms2rw9Gso1PEpC9WYzsE4f5JgAdnWPYQx~vDCuIxCzUuAkNFSsFjJsxOrzQLh0OsN4jgsGcCFaNjxte-z-jcRbCRCJwnlrlzrpiP8ppoPAh3BhmIeeNhzRPGYW~q-Cz4OUf4V3xgZqaJMOAulpeDoxZsIszzb~0vge3H6EySvFCgTHlnNADhxRIN7mrGyL95FvJtL435-BzPCh56f1tp68hFy1SBJIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185500&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Helitrons</em> in the silkworm. (A) <em>Helitron</em> structure. (B) Pipeline for <em>Helitron</em> identification.</p></div></div><p>To estimate copy numbers, we generated a consensus sequence for each <em>Helitron</em> family using DAMBE.<sup><a reveal-id="DST024C21" data-open="DST024C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We used these consensus sequences as queries for BLASTN searches of the corresponding genome database. In this step, a <em>Helitron</em> family was defined by <em>E</em> &lt; <em>e</em><sup>–6</sup>, pair-wise length &gt;80 bp, and a minimum nucleotide identity rate of &gt;80%. Sequences with a TC dinucleotide at the 5′-end and a CTRR motif at the 3′-end were defined as intact <em>Helitrons</em>.</p><p>Copy numbers of relatively long <em>Helitron</em> sequences (&lt;15 kb) were estimated as follows: (i) when each end almost perfectly matched the ends of a <em>Helitron</em> family's consensus sequence (identity &gt;80%, pair-wise length &gt;80 bp for each end), and when the sequence had a TC dinucleotide and a CTRR motif at the 5′- and 3′-ends, respectively, it was defined as a <em>Helitron</em> copy (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A–C</a>). (ii) When each end almost perfectly matched (identity &gt;80%, pair-wise length &gt;80 bp) non-terminal regions of a <em>Helitron</em> family's consensus sequence and the sequence had no or one <em>Helitron</em> terminal sequence (either a 5′ TC or a 3′ CTRR), it was treated as two or more <em>Helitron</em> fragments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1D–F</a>).</p><p>To better understand the composition and structure of <em>Helitron</em> sequences, the AT content of each <em>Helitron</em> family consensus sequence was estimated using BioEdit (<a class="link link-uri" href="http://www.mbio.ncsu.edu/bioedit/bioedit.html">http://www.mbio.ncsu.edu/bioedit/bioedit.html</a>). UNAFOLD (<a class="link link-uri" href="http://mfold.rna.albany.edu/">http://mfold.rna.albany.edu/</a>) was used to predict the Gibbs free energy (–dG) of each <em>Helitron</em>.<sup><a reveal-id="DST024C22" data-open="DST024C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Finally, all <em>Helitron</em> families were screened against the ISfinder (<a class="link link-uri" href="http://www-is.biotoul.fr/">http://www-is.biotoul.fr/</a>),<sup><a reveal-id="DST024C23" data-open="DST024C23" class="link link-ref link-reveal xref-bibr">23</a></sup> RepBase (v. 17.08),<sup><a reveal-id="DST024C24" data-open="DST024C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and NCBI non-redundant (nr) databases to identify known families. Putative autonomous <em>Helitrons</em> were identified by using known autonomous <em>Helitrons</em> downloaded from RepBase as queries and performing TBLASTN searches against all the silkworm <em>Helitron</em> databases.</p>                            <h3 scrollto-destination=5185504 id="5185504" class="section-title" data-legacy-id=s2b>2.2. PCR validation of predicted <em>Helitrons</em></h3>
<p>Fifteen accessions (02–320, DaZao, Ri9, 872, Ou18, Yi16, YinDuSanMian, WuLin1Hao, BH863, YingWenXing, LuoSa, RiXian2Hao, ALiKeSi, SanMianBai, and Zhong4010) representing the four main geographic strains of silkworm (Chinese, Japanese, European, and tropical) were used for insertion validation. DNA was extracted from individual pupae and moths using a standard phenol–chloroform protocol.<sup><a reveal-id="DST024C25" data-open="DST024C25" class="link link-ref link-reveal xref-bibr">25</a></sup> A <em>Helitron</em> (BmHel-8) was randomly selected for insertion validation, and primers (BmHel-8-S: 5′-ATTGTCAGTGGTATCGTTGCTCC-3′, BmHel-8-A: 5′-TAAGGGAATACAATAGAGCCGTG-3′) were designed based on the flanking insertion sites.</p>                            <h3 scrollto-destination=5185506 id="5185506" class="section-title" data-legacy-id=s2c>2.3. Estimates of insertion time and expansion events</h3>
<p>To estimate <em>Helitron</em> age, all full-length sequences of each <em>Helitron</em> family were aligned using MUSCLE,<sup><a reveal-id="DST024C20" data-open="DST024C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and the amount of nucleotide substitution (<em>k</em>) between each <em>Helitron</em> and the family consensus sequence was calculated using Kimura 2-parameter (K2P) distance.<sup><a reveal-id="DST024C26" data-open="DST024C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The age of each <em>Helitron</em> was estimated using the formula <em>T</em> = <em>k</em>/2<em>r</em>, where <em>r</em> = 1.56 × 10<sup>−8</sup>, the neutral rate of substitutions per year in fruit fly, which has previously been used in silkworms.<sup><a reveal-id="DST024C27" data-open="DST024C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST024C28" data-open="DST024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Neighbor-joining trees (pair-wise deletion of gaps and K2P substitution model) for <em>Helitron</em> families were reconstructed using MEGA4.<sup><a reveal-id="DST024C29" data-open="DST024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Within each <em>Helitron</em> family, the frequency distribution of the number of pair-wise differences between sequences was calculated with Arlequin v. 3.11.<sup><a reveal-id="DST024C30" data-open="DST024C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h3 scrollto-destination=5185508 id="5185508" class="section-title" data-legacy-id=s2d>2.4. Distribution of <em>Helitrons</em> on chromosomes</h3>
<p>All identified members of each <em>Helitron</em> family were mapped onto chromosomes using SilkMap (<a class="link link-uri" href="http://silkworm.swu.edu.cn/silksoft/silkmap.html">http://silkworm.swu.edu.cn/silksoft/silkmap.html</a>), and the copy number of each family on each chromosome was counted. We divided each chromosome into 100 uniformly-sized segments and estimated each <em>Helitron</em> family's distribution in these fragments using a Perl script. The positions of predicted genes in scaffolds and the lengths of scaffolds were downloaded from SilkDB (<a class="link link-uri" href="http://silkworm.swu.edu.cn/silkdb">http://silkworm.swu.edu.cn/silkdb</a>);<sup><a reveal-id="DST024C31" data-open="DST024C31" class="link link-ref link-reveal xref-bibr">31</a></sup> a Perl script was used to identify the genes near to or containing <em>Helitrons</em>. To determine whether <em>Helitron</em> insertions into genes were due to chance, a computer simulation was performed.<sup><a reveal-id="DST024C32" data-open="DST024C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=5185510 id="5185510" class="section-title" data-legacy-id=s2e>2.5. Gene fragment acquisition and contribution to transcripts</h3>
<p>Gene fragments captured by <em>Helitrons</em> were identified by using all identified <em>Helitrons</em> in a BLASTX search against the NCBI nr protein database (as of 22 January 2013). Captured gene fragments were identified if a homologue had a maximum expected value of <em>e</em><sup>–10</sup> in the silkworm, or of <em>e</em><sup>–5</sup> in a species other than silkworm. TE-related proteins were discarded.</p><p>To evaluate the contributions of silkworm <em>Helitrons</em> to transcripts, we used all intact <em>Helitron</em> sequences as queries in BLASTN analyses against the expressed sequence tag (EST) database of the silkworm (<a class="link link-uri" href="http://silkworm.swu.edu.cn/silkdb">http://silkworm.swu.edu.cn/silkdb</a>). A match was defined when the fragments had at least 99% identity and <em>E</em> &lt; <em>e</em><sup>−10</sup>. All matching ESTs were classified as either parental transcripts (with a similarity between the EST and the parental genes of the captured fragment greater than that between the EST and the corresponding <em>Helitron</em>) or <em>Helitron</em> transcripts.</p><p>To estimate whether the silkworm <em>Helitrons</em> contributed to 5′ untranslated regions (UTRs), coding regions, or 3′-UTRs of the silkworm full-length cDNAs, we downloaded the silkworm full-length cDNA database from SilkBase (<a class="link link-uri" href="http://silkbase.ab.a.u-tokyo.ac.jp/cgi-bin/index.cgi">http://silkbase.ab.a.u-tokyo.ac.jp/cgi-bin/index.cgi</a>). A Perl script was written to split each full-length cDNA into 5′-UTR, coding region, and 3′-UTR. Then, we performed BLASTN analyses against these three datasets with cut-off values of at least 99% identity and <em>E</em> &lt; <em>e</em><sup>−10</sup>. All matching transcripts were classified as either (i) parental transcripts (with the similarity between the transcript and parental gene greater than that between the transcript and corresponding <em>Helitron</em>); (ii) transcripts of <em>Helitron</em> transposase; or (iii) chimerical transcripts composed of other genes and <em>Helitron</em> fragments if a full-length <em>Helitron</em> copy matches a cDNA sequence and there is an overlapping region (matched region) between the position of the full-length cDNA sequence and the corresponding full-length <em>Helitron</em> copy in the silkworm genome.</p>                            <h2 scrollto-destination=5185514 id="5185514" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5185515 id="5185515" class="section-title" data-legacy-id=s3a>3.1. Identification, classification, and characterization of <em>Helitrons</em></h3>
<p>We used a structure-based approach to search for <em>Helitrons</em> in the silkworm genome (Fig. <a reveal-id="DST024F1" data-open="DST024F1" class="link link-reveal link-table xref-fig">1</a>). First, we searched the silkworm genome for sequences with a characteristic <em>Helitron</em> end structure (Fig. <a reveal-id="DST024F1" data-open="DST024F1" class="link link-reveal link-table xref-fig">1</a>A) and found 106 766 candidate sequences. We extended the sequence of each candidate upstream, clustered all candidates, and generated a consensus sequence for each cluster. In total, we obtained 1805 consensus sequences. Each consensus sequence was used as a query in BLASTN search against BmTEdb (<a class="link link-uri" href="http://202.202.1.217/BmTEdb/">http://202.202.1.217/BmTEdb/</a>), and sequences that hit to other known non-<em>Helitron</em> TEs were discarded. The remaining 854 consensus sequences were used in a BLASTN search against the silkworm genome. We extracted no more than 20 most-similar sequences for each consensus sequence, aligned them, and manually defined the <em>Helitron</em> element boundaries (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Finally, sequences were classified by similarity into families and subfamilies. This pipeline (Fig. <a reveal-id="DST024F1" data-open="DST024F1" class="link link-reveal link-table xref-fig">1</a>B) identified 21 <em>Helitron</em> families (Table <a reveal-id="DST024TB1" data-open="DST024TB1" class="link link-reveal link-table xref-fig">1</a>) that were designated BmHel-1 through BmHel-21. </p>                        <a id="5185517" scrollto-destination="5185517"></a>
<div content-id="DST024TB1" class="table-modal"><div class="table-wrap"><div id="DST024TB1" class="table-wrap-title" data-id="DST024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary information for the <em>Helitron</em> families in four lepidopteran species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Family </th><th>Subfamily </th><th>Length (bp) </th><th>Copies </th><th>AT (%) </th><th>−dG </th><th>Annotation<sup>ref</sup> </th></tr></thead><tbody><tr><td rowspan="22"><em>B. mori</em> </td><td>BmHel-1 </td><td> </td><td>198–781 </td><td>514 </td><td>56.6 </td><td>33.8 </td><td>Bm_283<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-2 </td><td> </td><td>822–9452 </td><td>79 </td><td>63.3 </td><td>141.7 </td><td>bm_691<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-3 </td><td>BmHel-3a </td><td>196–8700 </td><td>1386 </td><td>63.9 </td><td>30.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td> </td><td>BmHel-3b </td><td>206–10479 </td><td>1995 </td><td>69.4 </td><td>144.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-4 </td><td> </td><td>290–9930 </td><td>661 </td><td>64.0 </td><td>69.0 </td><td>Helianu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-5 </td><td> </td><td>3121–6696 </td><td>80 </td><td>68.7 </td><td>535.3 </td><td>Heliminu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-6 </td><td> </td><td>594–738 </td><td>25 </td><td>69.5 </td><td>82.8 </td><td>Novel </td></tr><tr><td>BmHel-7 </td><td> </td><td>126–10099 </td><td>6007 </td><td>61.9 </td><td>35.3 </td><td>Novel </td></tr><tr><td>BmHel-8 </td><td> </td><td>105–10644 </td><td>14656 </td><td>71.9 </td><td>55.4 </td><td>Novel </td></tr><tr><td>BmHel-9 </td><td> </td><td>608–1256 </td><td>11 </td><td>68.9 </td><td>52.6 </td><td>Novel </td></tr><tr><td>BmHel-10 </td><td> </td><td>455–935 </td><td>21 </td><td>63.6 </td><td>72.0 </td><td>Novel </td></tr><tr><td>BmHel-11 </td><td> </td><td>96–9788 </td><td>12206 </td><td>65.4 </td><td>21.9 </td><td>Novel </td></tr><tr><td>BmHel-12 </td><td> </td><td>152–10623 </td><td>3138 </td><td>59.3 </td><td>62.6 </td><td>Novel </td></tr><tr><td>BmHel-13 </td><td> </td><td>285–8964 </td><td>3428 </td><td>66.9 </td><td>43.3 </td><td>Novel </td></tr><tr><td>BmHel-14 </td><td> </td><td>258–2723 </td><td>3431 </td><td>69.9 </td><td>69.7 </td><td>Novel </td></tr><tr><td>BmHel-15 </td><td> </td><td>142–10468 </td><td>8297 </td><td>67.1 </td><td>71.2 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>BmHel-16 </td><td> </td><td>136–10228 </td><td>6537 </td><td>65.9 </td><td>25.9 </td><td>Novel </td></tr><tr><td>BmHel-17 </td><td> </td><td>288–683 </td><td>645 </td><td>66.9 </td><td>57.5 </td><td>Novel </td></tr><tr><td>BmHel-18 </td><td> </td><td>113–9945 </td><td>3768 </td><td>64.7 </td><td>26.8 </td><td>Novel </td></tr><tr><td>BmHel-19 </td><td> </td><td>300–2462 </td><td>149 </td><td>58.0 </td><td>138.4 </td><td>Novel </td></tr><tr><td>BmHel-20 </td><td> </td><td>296–2355 </td><td>144 </td><td>65.9 </td><td>43.8 </td><td>HeligloriaAi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-21 </td><td> </td><td>149–6073 </td><td>377 </td><td>67.9 </td><td>82.1 </td><td>HeligloriaAii<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td rowspan="10"><em>H. mel</em> </td><td>HmHel-1 </td><td> </td><td>134–10671 </td><td>6148 </td><td>70.1 </td><td>122.6 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>HmHel-2 </td><td> </td><td>120–10381 </td><td>6827 </td><td>66.3 </td><td>103.2 </td><td><em>Helitron</em>-5<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-3 </td><td> </td><td>127–11371 </td><td>3940 </td><td>66.9 </td><td>52.2 </td><td><em>Helitron</em>-4,7<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-4 </td><td> </td><td>134–10068 </td><td>5665 </td><td>59.5 </td><td>44.1 </td><td><em>Helitron</em>-15<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-5 </td><td> </td><td>268–7417 </td><td>4398 </td><td>64.8 </td><td>191.9 </td><td><em>Helitron</em>-6,11<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-6 </td><td> </td><td>134–15055 </td><td>15359 </td><td>47.5 </td><td>72.8 </td><td>Novel </td></tr><tr><td>HmHel-7 </td><td> </td><td>276–7883 </td><td>7103 </td><td>60.5 </td><td>563.1 </td><td><em>Helitron</em>-13<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-8 </td><td> </td><td>289–373 </td><td>74 </td><td>64.5 </td><td>53.5 </td><td><em>Helitron</em>-16<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-9 </td><td> </td><td>217–1404 </td><td>621 </td><td>67.8 </td><td>37.4 </td><td><em>Helitron</em>-1<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-10 </td><td> </td><td>192–5100 </td><td>888 </td><td>70.7 </td><td>103.5 </td><td><em>Helitron</em>-9<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td><em>D. ple</em> </td><td>DpHel-1 </td><td> </td><td>111–4232 </td><td>1010 </td><td>66.6 </td><td>82.0 </td><td>Novel </td></tr><tr><td rowspan="8"><em>M. ext</em> </td><td>MsHel-1 </td><td> </td><td>156–9446 </td><td>6386 </td><td>58.5 </td><td>133.5 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>MsHel-2 </td><td> </td><td>134–6559 </td><td>2975 </td><td>57.6 </td><td>53.3 </td><td>Novel </td></tr><tr><td>MsHel-3 </td><td>MsHel-3a </td><td>134–7735 </td><td>3270 </td><td>66.1 </td><td>49.5 </td><td>Novel </td></tr><tr><td> </td><td>MsHel-3b </td><td>119–9845 </td><td>5632 </td><td>60.4 </td><td>33.0. </td><td>Novel </td></tr><tr><td>MsHel-4 </td><td> </td><td>120–10147 </td><td>697 </td><td>60.3 </td><td>91.1 </td><td>Novel </td></tr><tr><td>MsHel-5 </td><td> </td><td>120–3351 </td><td>1957 </td><td>62.1 </td><td>49.2 </td><td>Novel </td></tr><tr><td>MsHel-6 </td><td> </td><td>122–9236 </td><td>2574 </td><td>65.5 </td><td>31.4 </td><td>Novel </td></tr><tr><td>MsHel-7 </td><td> </td><td>107–3867 </td><td>2691 </td><td>62.3 </td><td>119.2 </td><td>Novel </td></tr></tbody></table></div><div content-id="DST024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Family </th><th>Subfamily </th><th>Length (bp) </th><th>Copies </th><th>AT (%) </th><th>−dG </th><th>Annotation<sup>ref</sup> </th></tr></thead><tbody><tr><td rowspan="22"><em>B. mori</em> </td><td>BmHel-1 </td><td> </td><td>198–781 </td><td>514 </td><td>56.6 </td><td>33.8 </td><td>Bm_283<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-2 </td><td> </td><td>822–9452 </td><td>79 </td><td>63.3 </td><td>141.7 </td><td>bm_691<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-3 </td><td>BmHel-3a </td><td>196–8700 </td><td>1386 </td><td>63.9 </td><td>30.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td> </td><td>BmHel-3b </td><td>206–10479 </td><td>1995 </td><td>69.4 </td><td>144.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-4 </td><td> </td><td>290–9930 </td><td>661 </td><td>64.0 </td><td>69.0 </td><td>Helianu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-5 </td><td> </td><td>3121–6696 </td><td>80 </td><td>68.7 </td><td>535.3 </td><td>Heliminu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-6 </td><td> </td><td>594–738 </td><td>25 </td><td>69.5 </td><td>82.8 </td><td>Novel </td></tr><tr><td>BmHel-7 </td><td> </td><td>126–10099 </td><td>6007 </td><td>61.9 </td><td>35.3 </td><td>Novel </td></tr><tr><td>BmHel-8 </td><td> </td><td>105–10644 </td><td>14656 </td><td>71.9 </td><td>55.4 </td><td>Novel </td></tr><tr><td>BmHel-9 </td><td> </td><td>608–1256 </td><td>11 </td><td>68.9 </td><td>52.6 </td><td>Novel </td></tr><tr><td>BmHel-10 </td><td> </td><td>455–935 </td><td>21 </td><td>63.6 </td><td>72.0 </td><td>Novel </td></tr><tr><td>BmHel-11 </td><td> </td><td>96–9788 </td><td>12206 </td><td>65.4 </td><td>21.9 </td><td>Novel </td></tr><tr><td>BmHel-12 </td><td> </td><td>152–10623 </td><td>3138 </td><td>59.3 </td><td>62.6 </td><td>Novel </td></tr><tr><td>BmHel-13 </td><td> </td><td>285–8964 </td><td>3428 </td><td>66.9 </td><td>43.3 </td><td>Novel </td></tr><tr><td>BmHel-14 </td><td> </td><td>258–2723 </td><td>3431 </td><td>69.9 </td><td>69.7 </td><td>Novel </td></tr><tr><td>BmHel-15 </td><td> </td><td>142–10468 </td><td>8297 </td><td>67.1 </td><td>71.2 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>BmHel-16 </td><td> </td><td>136–10228 </td><td>6537 </td><td>65.9 </td><td>25.9 </td><td>Novel </td></tr><tr><td>BmHel-17 </td><td> </td><td>288–683 </td><td>645 </td><td>66.9 </td><td>57.5 </td><td>Novel </td></tr><tr><td>BmHel-18 </td><td> </td><td>113–9945 </td><td>3768 </td><td>64.7 </td><td>26.8 </td><td>Novel </td></tr><tr><td>BmHel-19 </td><td> </td><td>300–2462 </td><td>149 </td><td>58.0 </td><td>138.4 </td><td>Novel </td></tr><tr><td>BmHel-20 </td><td> </td><td>296–2355 </td><td>144 </td><td>65.9 </td><td>43.8 </td><td>HeligloriaAi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-21 </td><td> </td><td>149–6073 </td><td>377 </td><td>67.9 </td><td>82.1 </td><td>HeligloriaAii<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td rowspan="10"><em>H. mel</em> </td><td>HmHel-1 </td><td> </td><td>134–10671 </td><td>6148 </td><td>70.1 </td><td>122.6 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>HmHel-2 </td><td> </td><td>120–10381 </td><td>6827 </td><td>66.3 </td><td>103.2 </td><td><em>Helitron</em>-5<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-3 </td><td> </td><td>127–11371 </td><td>3940 </td><td>66.9 </td><td>52.2 </td><td><em>Helitron</em>-4,7<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-4 </td><td> </td><td>134–10068 </td><td>5665 </td><td>59.5 </td><td>44.1 </td><td><em>Helitron</em>-15<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-5 </td><td> </td><td>268–7417 </td><td>4398 </td><td>64.8 </td><td>191.9 </td><td><em>Helitron</em>-6,11<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-6 </td><td> </td><td>134–15055 </td><td>15359 </td><td>47.5 </td><td>72.8 </td><td>Novel </td></tr><tr><td>HmHel-7 </td><td> </td><td>276–7883 </td><td>7103 </td><td>60.5 </td><td>563.1 </td><td><em>Helitron</em>-13<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-8 </td><td> </td><td>289–373 </td><td>74 </td><td>64.5 </td><td>53.5 </td><td><em>Helitron</em>-16<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-9 </td><td> </td><td>217–1404 </td><td>621 </td><td>67.8 </td><td>37.4 </td><td><em>Helitron</em>-1<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-10 </td><td> </td><td>192–5100 </td><td>888 </td><td>70.7 </td><td>103.5 </td><td><em>Helitron</em>-9<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td><em>D. ple</em> </td><td>DpHel-1 </td><td> </td><td>111–4232 </td><td>1010 </td><td>66.6 </td><td>82.0 </td><td>Novel </td></tr><tr><td rowspan="8"><em>M. ext</em> </td><td>MsHel-1 </td><td> </td><td>156–9446 </td><td>6386 </td><td>58.5 </td><td>133.5 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>MsHel-2 </td><td> </td><td>134–6559 </td><td>2975 </td><td>57.6 </td><td>53.3 </td><td>Novel </td></tr><tr><td>MsHel-3 </td><td>MsHel-3a </td><td>134–7735 </td><td>3270 </td><td>66.1 </td><td>49.5 </td><td>Novel </td></tr><tr><td> </td><td>MsHel-3b </td><td>119–9845 </td><td>5632 </td><td>60.4 </td><td>33.0. </td><td>Novel </td></tr><tr><td>MsHel-4 </td><td> </td><td>120–10147 </td><td>697 </td><td>60.3 </td><td>91.1 </td><td>Novel </td></tr><tr><td>MsHel-5 </td><td> </td><td>120–3351 </td><td>1957 </td><td>62.1 </td><td>49.2 </td><td>Novel </td></tr><tr><td>MsHel-6 </td><td> </td><td>122–9236 </td><td>2574 </td><td>65.5 </td><td>31.4 </td><td>Novel </td></tr><tr><td>MsHel-7 </td><td> </td><td>107–3867 </td><td>2691 </td><td>62.3 </td><td>119.2 </td><td>Novel </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>–dG, average Gibbs energy (kcal/mol) for each <em>Helitron</em> family consensus sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5185517" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST024TB1" class="table-wrap-title" data-id="DST024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary information for the <em>Helitron</em> families in four lepidopteran species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Family </th><th>Subfamily </th><th>Length (bp) </th><th>Copies </th><th>AT (%) </th><th>−dG </th><th>Annotation<sup>ref</sup> </th></tr></thead><tbody><tr><td rowspan="22"><em>B. mori</em> </td><td>BmHel-1 </td><td> </td><td>198–781 </td><td>514 </td><td>56.6 </td><td>33.8 </td><td>Bm_283<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-2 </td><td> </td><td>822–9452 </td><td>79 </td><td>63.3 </td><td>141.7 </td><td>bm_691<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-3 </td><td>BmHel-3a </td><td>196–8700 </td><td>1386 </td><td>63.9 </td><td>30.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td> </td><td>BmHel-3b </td><td>206–10479 </td><td>1995 </td><td>69.4 </td><td>144.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-4 </td><td> </td><td>290–9930 </td><td>661 </td><td>64.0 </td><td>69.0 </td><td>Helianu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-5 </td><td> </td><td>3121–6696 </td><td>80 </td><td>68.7 </td><td>535.3 </td><td>Heliminu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-6 </td><td> </td><td>594–738 </td><td>25 </td><td>69.5 </td><td>82.8 </td><td>Novel </td></tr><tr><td>BmHel-7 </td><td> </td><td>126–10099 </td><td>6007 </td><td>61.9 </td><td>35.3 </td><td>Novel </td></tr><tr><td>BmHel-8 </td><td> </td><td>105–10644 </td><td>14656 </td><td>71.9 </td><td>55.4 </td><td>Novel </td></tr><tr><td>BmHel-9 </td><td> </td><td>608–1256 </td><td>11 </td><td>68.9 </td><td>52.6 </td><td>Novel </td></tr><tr><td>BmHel-10 </td><td> </td><td>455–935 </td><td>21 </td><td>63.6 </td><td>72.0 </td><td>Novel </td></tr><tr><td>BmHel-11 </td><td> </td><td>96–9788 </td><td>12206 </td><td>65.4 </td><td>21.9 </td><td>Novel </td></tr><tr><td>BmHel-12 </td><td> </td><td>152–10623 </td><td>3138 </td><td>59.3 </td><td>62.6 </td><td>Novel </td></tr><tr><td>BmHel-13 </td><td> </td><td>285–8964 </td><td>3428 </td><td>66.9 </td><td>43.3 </td><td>Novel </td></tr><tr><td>BmHel-14 </td><td> </td><td>258–2723 </td><td>3431 </td><td>69.9 </td><td>69.7 </td><td>Novel </td></tr><tr><td>BmHel-15 </td><td> </td><td>142–10468 </td><td>8297 </td><td>67.1 </td><td>71.2 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>BmHel-16 </td><td> </td><td>136–10228 </td><td>6537 </td><td>65.9 </td><td>25.9 </td><td>Novel </td></tr><tr><td>BmHel-17 </td><td> </td><td>288–683 </td><td>645 </td><td>66.9 </td><td>57.5 </td><td>Novel </td></tr><tr><td>BmHel-18 </td><td> </td><td>113–9945 </td><td>3768 </td><td>64.7 </td><td>26.8 </td><td>Novel </td></tr><tr><td>BmHel-19 </td><td> </td><td>300–2462 </td><td>149 </td><td>58.0 </td><td>138.4 </td><td>Novel </td></tr><tr><td>BmHel-20 </td><td> </td><td>296–2355 </td><td>144 </td><td>65.9 </td><td>43.8 </td><td>HeligloriaAi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-21 </td><td> </td><td>149–6073 </td><td>377 </td><td>67.9 </td><td>82.1 </td><td>HeligloriaAii<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td rowspan="10"><em>H. mel</em> </td><td>HmHel-1 </td><td> </td><td>134–10671 </td><td>6148 </td><td>70.1 </td><td>122.6 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>HmHel-2 </td><td> </td><td>120–10381 </td><td>6827 </td><td>66.3 </td><td>103.2 </td><td><em>Helitron</em>-5<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-3 </td><td> </td><td>127–11371 </td><td>3940 </td><td>66.9 </td><td>52.2 </td><td><em>Helitron</em>-4,7<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-4 </td><td> </td><td>134–10068 </td><td>5665 </td><td>59.5 </td><td>44.1 </td><td><em>Helitron</em>-15<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-5 </td><td> </td><td>268–7417 </td><td>4398 </td><td>64.8 </td><td>191.9 </td><td><em>Helitron</em>-6,11<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-6 </td><td> </td><td>134–15055 </td><td>15359 </td><td>47.5 </td><td>72.8 </td><td>Novel </td></tr><tr><td>HmHel-7 </td><td> </td><td>276–7883 </td><td>7103 </td><td>60.5 </td><td>563.1 </td><td><em>Helitron</em>-13<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-8 </td><td> </td><td>289–373 </td><td>74 </td><td>64.5 </td><td>53.5 </td><td><em>Helitron</em>-16<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-9 </td><td> </td><td>217–1404 </td><td>621 </td><td>67.8 </td><td>37.4 </td><td><em>Helitron</em>-1<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-10 </td><td> </td><td>192–5100 </td><td>888 </td><td>70.7 </td><td>103.5 </td><td><em>Helitron</em>-9<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td><em>D. ple</em> </td><td>DpHel-1 </td><td> </td><td>111–4232 </td><td>1010 </td><td>66.6 </td><td>82.0 </td><td>Novel </td></tr><tr><td rowspan="8"><em>M. ext</em> </td><td>MsHel-1 </td><td> </td><td>156–9446 </td><td>6386 </td><td>58.5 </td><td>133.5 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>MsHel-2 </td><td> </td><td>134–6559 </td><td>2975 </td><td>57.6 </td><td>53.3 </td><td>Novel </td></tr><tr><td>MsHel-3 </td><td>MsHel-3a </td><td>134–7735 </td><td>3270 </td><td>66.1 </td><td>49.5 </td><td>Novel </td></tr><tr><td> </td><td>MsHel-3b </td><td>119–9845 </td><td>5632 </td><td>60.4 </td><td>33.0. </td><td>Novel </td></tr><tr><td>MsHel-4 </td><td> </td><td>120–10147 </td><td>697 </td><td>60.3 </td><td>91.1 </td><td>Novel </td></tr><tr><td>MsHel-5 </td><td> </td><td>120–3351 </td><td>1957 </td><td>62.1 </td><td>49.2 </td><td>Novel </td></tr><tr><td>MsHel-6 </td><td> </td><td>122–9236 </td><td>2574 </td><td>65.5 </td><td>31.4 </td><td>Novel </td></tr><tr><td>MsHel-7 </td><td> </td><td>107–3867 </td><td>2691 </td><td>62.3 </td><td>119.2 </td><td>Novel </td></tr></tbody></table></div><div content-id="DST024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Family </th><th>Subfamily </th><th>Length (bp) </th><th>Copies </th><th>AT (%) </th><th>−dG </th><th>Annotation<sup>ref</sup> </th></tr></thead><tbody><tr><td rowspan="22"><em>B. mori</em> </td><td>BmHel-1 </td><td> </td><td>198–781 </td><td>514 </td><td>56.6 </td><td>33.8 </td><td>Bm_283<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-2 </td><td> </td><td>822–9452 </td><td>79 </td><td>63.3 </td><td>141.7 </td><td>bm_691<sup>BmTEdb</sup> </td></tr><tr><td>BmHel-3 </td><td>BmHel-3a </td><td>196–8700 </td><td>1386 </td><td>63.9 </td><td>30.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td> </td><td>BmHel-3b </td><td>206–10479 </td><td>1995 </td><td>69.4 </td><td>144.5 </td><td>Helisimi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-4 </td><td> </td><td>290–9930 </td><td>661 </td><td>64.0 </td><td>69.0 </td><td>Helianu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-5 </td><td> </td><td>3121–6696 </td><td>80 </td><td>68.7 </td><td>535.3 </td><td>Heliminu<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-6 </td><td> </td><td>594–738 </td><td>25 </td><td>69.5 </td><td>82.8 </td><td>Novel </td></tr><tr><td>BmHel-7 </td><td> </td><td>126–10099 </td><td>6007 </td><td>61.9 </td><td>35.3 </td><td>Novel </td></tr><tr><td>BmHel-8 </td><td> </td><td>105–10644 </td><td>14656 </td><td>71.9 </td><td>55.4 </td><td>Novel </td></tr><tr><td>BmHel-9 </td><td> </td><td>608–1256 </td><td>11 </td><td>68.9 </td><td>52.6 </td><td>Novel </td></tr><tr><td>BmHel-10 </td><td> </td><td>455–935 </td><td>21 </td><td>63.6 </td><td>72.0 </td><td>Novel </td></tr><tr><td>BmHel-11 </td><td> </td><td>96–9788 </td><td>12206 </td><td>65.4 </td><td>21.9 </td><td>Novel </td></tr><tr><td>BmHel-12 </td><td> </td><td>152–10623 </td><td>3138 </td><td>59.3 </td><td>62.6 </td><td>Novel </td></tr><tr><td>BmHel-13 </td><td> </td><td>285–8964 </td><td>3428 </td><td>66.9 </td><td>43.3 </td><td>Novel </td></tr><tr><td>BmHel-14 </td><td> </td><td>258–2723 </td><td>3431 </td><td>69.9 </td><td>69.7 </td><td>Novel </td></tr><tr><td>BmHel-15 </td><td> </td><td>142–10468 </td><td>8297 </td><td>67.1 </td><td>71.2 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>BmHel-16 </td><td> </td><td>136–10228 </td><td>6537 </td><td>65.9 </td><td>25.9 </td><td>Novel </td></tr><tr><td>BmHel-17 </td><td> </td><td>288–683 </td><td>645 </td><td>66.9 </td><td>57.5 </td><td>Novel </td></tr><tr><td>BmHel-18 </td><td> </td><td>113–9945 </td><td>3768 </td><td>64.7 </td><td>26.8 </td><td>Novel </td></tr><tr><td>BmHel-19 </td><td> </td><td>300–2462 </td><td>149 </td><td>58.0 </td><td>138.4 </td><td>Novel </td></tr><tr><td>BmHel-20 </td><td> </td><td>296–2355 </td><td>144 </td><td>65.9 </td><td>43.8 </td><td>HeligloriaAi<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td>BmHel-21 </td><td> </td><td>149–6073 </td><td>377 </td><td>67.9 </td><td>82.1 </td><td>HeligloriaAii<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> </td></tr><tr><td rowspan="10"><em>H. mel</em> </td><td>HmHel-1 </td><td> </td><td>134–10671 </td><td>6148 </td><td>70.1 </td><td>122.6 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>HmHel-2 </td><td> </td><td>120–10381 </td><td>6827 </td><td>66.3 </td><td>103.2 </td><td><em>Helitron</em>-5<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-3 </td><td> </td><td>127–11371 </td><td>3940 </td><td>66.9 </td><td>52.2 </td><td><em>Helitron</em>-4,7<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-4 </td><td> </td><td>134–10068 </td><td>5665 </td><td>59.5 </td><td>44.1 </td><td><em>Helitron</em>-15<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-5 </td><td> </td><td>268–7417 </td><td>4398 </td><td>64.8 </td><td>191.9 </td><td><em>Helitron</em>-6,11<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-6 </td><td> </td><td>134–15055 </td><td>15359 </td><td>47.5 </td><td>72.8 </td><td>Novel </td></tr><tr><td>HmHel-7 </td><td> </td><td>276–7883 </td><td>7103 </td><td>60.5 </td><td>563.1 </td><td><em>Helitron</em>-13<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-8 </td><td> </td><td>289–373 </td><td>74 </td><td>64.5 </td><td>53.5 </td><td><em>Helitron</em>-16<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-9 </td><td> </td><td>217–1404 </td><td>621 </td><td>67.8 </td><td>37.4 </td><td><em>Helitron</em>-1<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td>HmHel-10 </td><td> </td><td>192–5100 </td><td>888 </td><td>70.7 </td><td>103.5 </td><td><em>Helitron</em>-9<sup><a reveal-id="DST024C17" data-open="DST024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> </td></tr><tr><td><em>D. ple</em> </td><td>DpHel-1 </td><td> </td><td>111–4232 </td><td>1010 </td><td>66.6 </td><td>82.0 </td><td>Novel </td></tr><tr><td rowspan="8"><em>M. ext</em> </td><td>MsHel-1 </td><td> </td><td>156–9446 </td><td>6386 </td><td>58.5 </td><td>133.5 </td><td>Lep1<sup><a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> </td></tr><tr><td>MsHel-2 </td><td> </td><td>134–6559 </td><td>2975 </td><td>57.6 </td><td>53.3 </td><td>Novel </td></tr><tr><td>MsHel-3 </td><td>MsHel-3a </td><td>134–7735 </td><td>3270 </td><td>66.1 </td><td>49.5 </td><td>Novel </td></tr><tr><td> </td><td>MsHel-3b </td><td>119–9845 </td><td>5632 </td><td>60.4 </td><td>33.0. </td><td>Novel </td></tr><tr><td>MsHel-4 </td><td> </td><td>120–10147 </td><td>697 </td><td>60.3 </td><td>91.1 </td><td>Novel </td></tr><tr><td>MsHel-5 </td><td> </td><td>120–3351 </td><td>1957 </td><td>62.1 </td><td>49.2 </td><td>Novel </td></tr><tr><td>MsHel-6 </td><td> </td><td>122–9236 </td><td>2574 </td><td>65.5 </td><td>31.4 </td><td>Novel </td></tr><tr><td>MsHel-7 </td><td> </td><td>107–3867 </td><td>2691 </td><td>62.3 </td><td>119.2 </td><td>Novel </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>–dG, average Gibbs energy (kcal/mol) for each <em>Helitron</em> family consensus sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5185517" target="_blank">View Large</a></div></div><p>The silkworm <em>Helitron</em> families were annotated based on homology. Using a consensus sequence for each <em>Helitron</em> family as queries, we searched the BmTEdb, ISfinder, RepBase, and NCBI nr databases and found that eight families (BmHel-1, 2, 3 4, 5, 15, 20, and 21) had been previously identified. The other 13 families had no matches to any known <em>Helitron</em> (Table <a reveal-id="DST024TB1" data-open="DST024TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>To estimate the abundance of these 21 families, we searched the silkworm genome. We identified 67 555 <em>Helitrons</em> in total, which constitute about 19.7 Mb (∼4.23%) of the silkworm genome. The insertion sites of these <em>Helitrons</em> into accession numbers (NCBI) are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Similar to previous reports on <em>Helitrons</em>,<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the silkworm <em>Helitron</em> size varied greatly both among and within a family; sizes ranged from 96 to 10 644 bp (Table <a reveal-id="DST024TB1" data-open="DST024TB1" class="link link-reveal link-table xref-fig">1</a>). There were 202 very long (from 6000–10 644 bp) <em>Helitron</em> copies. The internal sequences of these <em>Helitrons</em> had at most 50% identity, but their ends (100 bp) had at least 80% identity. This pattern could be caused by different DNA sequences being captured either by the <em>Helitrons</em> or by the insertion of other TEs into the <em>Helitrons</em>. We identified 19 580 intact <em>Helitrons</em>, with a TC dinucleotide at the 5′-end and a CTRR motif at the 3′-end; they made up 10.7 Mb (∼2.30%) of the silkworm genome. Of these, 15 272 (∼78%) had at least 80% identity and 8615 (∼44%) had at least 90% identity.</p><p>The <em>Helitron</em> families were AT rich, with AT contents ranging from 56.61% to 71.9%. The average AT content of the silkworm genome is ∼62%. Four <em>Helitron</em> families (BmHel-1, 7, 12, and 19) had AT contents that did not exceed the genome average (Table <a reveal-id="DST024TB1" data-open="DST024TB1" class="link link-reveal link-table xref-fig">1</a>). Almost all of the silkworm <em>Helitron</em> families had high predicted –dG values, indicating that most silkworm <em>Helitrons</em> can form stable secondary structures.</p><p>Putative autonomous silkworm <em>Helitrons</em> were founded based on homology. An autonomous <em>Helitron</em> should encode a Rep/helicase protein, because both the Rep motif and DNA helicase domain are necessary for transposition. All 141 known autonomous <em>Helitrons</em> were downloaded from RepBase and screened against all intact silkworm <em>Helitrons</em>. Three silkworm <em>Helitron</em> families (BmHel-2, 3, and 5) were putatively autonomous (Fig. <a reveal-id="DST024F2" data-open="DST024F2" class="link link-reveal link-table xref-fig">2</a>). These candidate autonomous elements encoded complete open reading frames (ORFs), in addition to a Rep motif and helicase domain. For example, the SilkDB accession numbers of transposase for BmHel-2, BmHel-3, and BmHel-5 were BGIBMGA003354-TA, BGIBMGA012372-TA, and BGIBMGA008616-TA, respectively. Furthermore, two of the three putative autonomous families had EST evidence; BGIBMGA003354-TA matched the EST BY927485 (identity, 0.98; length, 760 bp) and BGIBMGA012372-TA matched ESTs BB983132 (identity, 0.94; length, 681 bp), BY932007 (identity, 0.99; length, 702 bp), CK528421 (identity, 0.95; length, 632 bp), and BY916909 (identity, 0.96; length, 657 bp). Thus, we concluded that these elements could be active in the silkworm genome. </p>                        <a id="5185522" scrollto-destination="5185522"></a>
<div id="DST024F2" data-id="DST024F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02402.jpeg?Expires=2147483647&Signature=zsspd~SG04Wnd2TdmIa~ESujFDehPo7aYAZbrYJF1vNk9sELO3tsWKrp2xPQtuTANCwXNiN8vAaraEwzoq54fFNzbUFiXBY~aHhPATfoV3W6CugQ5rxpyCWCbYvI7KdgqolbuQwWzAOgxGSm9Lp3G3KDeug5S-DhhYf9XvCi7R7Fj~VdBxXLiR1osCsTmUjyRD3UunJ0DH~4yftnVILwY-vpluUKTqs3GBMtGMVrtbJzZMcaxF94vQdxx-KBh-jLd5CWWW0GZTkvYjo1BQJN-ScFC~53D9ydJ9RCO3SmQsN2PBv1g0CzzQHzWy7zAP7E~xuapFim5ftBmZFoKVpsNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three predicted putative autonomous elements in the silkworm based on protein domain. (A) A schematic representation of the putative autonomous Helitrons; Rep, rolling circle replication initiator motif; Helicase, region similar to SF1 superfamily of DNA helicases. (B) Alignment of REP motifs between silkworm and 12 other species. (C) Alignment of eight conserved motifs of the SF1 superfamily of DNA helicases." path-from-xml="dst02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185522/dst02402.jpeg" path-from-xml="dst02402.jpeg" target="_blank">View large</a><a section="5185522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02402.jpeg?Expires=2147483647&Signature=JYA5pKZc0l~tcYcOdtEjR-cdoKrt6X1hvCEqHbDVlR1doXDUpHAjFDSUPq1IluQfD-wXJCP-1yvtB870YjSorxZviw3aAsI6WEIsH4pJ2Mt3cHB-nP~YUFgviOZrNmalnII6V8FCgjK0-KiSHd65F-I~g2ty-vGQJ-v9qs18ABDhBa8oKtMeASHiz56e0YFV~vstooUPjZUxANeXhJlhSuL-PosE34fQcNk1p1ah1EyPap~ux3fVZCQZEF9WOHes~X1xFU5-ejlyJSST0DlCQZpSKpJ6f-P78xhB3sACNl0Gi4bDlZ3hPpwcj7ZTUA6dcKTmr-2DJALRlPu0JAKfAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185522&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three predicted putative autonomous elements in the silkworm based on protein domain. (A) A schematic representation of the putative autonomous <em>Helitrons</em>; Rep, rolling circle replication initiator motif; Helicase, region similar to SF1 superfamily of DNA helicases. (B) Alignment of REP motifs between silkworm and 12 other species. (C) Alignment of eight conserved motifs of the SF1 superfamily of DNA helicases.</p></div></div><div content-id="DST024F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02402.jpeg?Expires=2147483647&Signature=zsspd~SG04Wnd2TdmIa~ESujFDehPo7aYAZbrYJF1vNk9sELO3tsWKrp2xPQtuTANCwXNiN8vAaraEwzoq54fFNzbUFiXBY~aHhPATfoV3W6CugQ5rxpyCWCbYvI7KdgqolbuQwWzAOgxGSm9Lp3G3KDeug5S-DhhYf9XvCi7R7Fj~VdBxXLiR1osCsTmUjyRD3UunJ0DH~4yftnVILwY-vpluUKTqs3GBMtGMVrtbJzZMcaxF94vQdxx-KBh-jLd5CWWW0GZTkvYjo1BQJN-ScFC~53D9ydJ9RCO3SmQsN2PBv1g0CzzQHzWy7zAP7E~xuapFim5ftBmZFoKVpsNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three predicted putative autonomous elements in the silkworm based on protein domain. (A) A schematic representation of the putative autonomous Helitrons; Rep, rolling circle replication initiator motif; Helicase, region similar to SF1 superfamily of DNA helicases. (B) Alignment of REP motifs between silkworm and 12 other species. (C) Alignment of eight conserved motifs of the SF1 superfamily of DNA helicases." path-from-xml="dst02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185522/dst02402.jpeg" path-from-xml="dst02402.jpeg" target="_blank">View large</a><a section="5185522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02402.jpeg?Expires=2147483647&Signature=JYA5pKZc0l~tcYcOdtEjR-cdoKrt6X1hvCEqHbDVlR1doXDUpHAjFDSUPq1IluQfD-wXJCP-1yvtB870YjSorxZviw3aAsI6WEIsH4pJ2Mt3cHB-nP~YUFgviOZrNmalnII6V8FCgjK0-KiSHd65F-I~g2ty-vGQJ-v9qs18ABDhBa8oKtMeASHiz56e0YFV~vstooUPjZUxANeXhJlhSuL-PosE34fQcNk1p1ah1EyPap~ux3fVZCQZEF9WOHes~X1xFU5-ejlyJSST0DlCQZpSKpJ6f-P78xhB3sACNl0Gi4bDlZ3hPpwcj7ZTUA6dcKTmr-2DJALRlPu0JAKfAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185522&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three predicted putative autonomous elements in the silkworm based on protein domain. (A) A schematic representation of the putative autonomous <em>Helitrons</em>; Rep, rolling circle replication initiator motif; Helicase, region similar to SF1 superfamily of DNA helicases. (B) Alignment of REP motifs between silkworm and 12 other species. (C) Alignment of eight conserved motifs of the SF1 superfamily of DNA helicases.</p></div></div>                            <h3 scrollto-destination=5185523 id="5185523" class="section-title" data-legacy-id=s3b>3.2. Validation of predicted <em>Helitrons</em></h3>
<p>A BmHel-8 insert site was selected for PCR verification in 15 silkworm accessions representing four main geographic strains. The results indicated that BmHel-8 was present in most of the strains, but absent in YinDuSanMian, YingWenXing, LuoSa, and RiXian2Hao (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). This polymorphism indicated that the <em>Helitron</em> was not fixed in the silkworm genome and verified the efficacy of our approach.</p>                            <h3 scrollto-destination=5185525 id="5185525" class="section-title" data-legacy-id=s3c>3.3. <em>Helitron</em> abundance in other lepidopteran genomes</h3>
<p>To investigate whether <em>Helitrons</em> were pervasive in lepidopteran, three other recently released lepidopteran genomes were searched for <em>Helitrons. Helitron</em> abundance varied greatly among these genomes (Table <a reveal-id="DST024TB1" data-open="DST024TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DST024F3" data-open="DST024F3" class="link link-reveal link-table xref-fig">3</a>). For instance, <em>H. melpomene</em> had 10 <em>Helitron</em> families that comprised ∼6.62% (17.1/260 Mb) of the genome. <em>Manduca sexta</em> harboured seven <em>Helitron</em> families that made up ∼1.86% (7.23/388 Mb) of the genome. However, <em>D. plexippus</em> had only one <em>Helitron</em> family that represented only ∼0.20% (0.48/237 Mb) of the genome. The locations of each <em>Helitron</em> in these three genomes are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> (<em>H. melpomene</em>), <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> (<em>D. plexippus</em>), and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> (<em>M. sexta</em>). </p>                        <a id="5185527" scrollto-destination="5185527"></a>
<div id="DST024F3" data-id="DST024F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02403.jpeg?Expires=2147483647&Signature=Dgqvj5TYybLezCHcqM0dE7Q6ocFwlspSXpvqfkDn2PGaQAICfbN5YdxXLhC7ahHz0ncg5zrqdDrnS~T~Bi8df7D944ZyQMAEKVGqnJXAi3AQ8CB8R5zCzXVJN6ZTZmCRhLRVIBRiRVgLuFBmmA92CQ0REBWbkICBdhdxmhE2sQkltLLNTlYHzFhHFB79Xz9k0QSTj4sqpZYim7cao4YVpNwkGud-xD0rks~qaeKHERKzgU~bTAxrBl7K3~tujGAqcyROCnaZYU8QQrnK6yzTqEtg3Ml5HtW41jMgYxbdnmGl6Fc~ecg5BLBx7HA~P94e0ANT5W0gviiJR0OKw4I6Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Helitron abundances in different organisms. Underlined Helitron contents were identified in this study, while others came from previous studies.1,3–11" path-from-xml="dst02403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185527/dst02403.jpeg" path-from-xml="dst02403.jpeg" target="_blank">View large</a><a section="5185527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02403.jpeg?Expires=2147483647&Signature=CICGcIZoc3fJ8WkFIeWLY~h~P0mjyxdwMOsGJ4D4p6-W6Rtidf4f~xwFQyqDpjKS~-YaGFf0VpDTmy282aKQKo~hDEH6NPIbOtueN22wKA-iN2Lv5laRFjKNc~mfD2EQU78lD3N~ifbRyIubSfgs5yrsfY6827LcT9WU3Yy5cRUlO-bG4mcfPREVCtSFg87Yjh47YlJRB5BbbAkM8xmGlrr3NpexOafigtdmpMZTJMfrC2N7Q879C6sxRv5T8xhowb~8XO-gWEvlPrPJTUBnRPaBTtxlUFCAp5N22JmyYURheXvCRVHU9Y9k9eiZuUN7xz-jeg3AEb3rgj-RK1aR3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185527&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Helitron</em> abundances in different organisms. Underlined <em>Helitron</em> contents were identified in this study, while others came from previous studies.<sup><a reveal-id="DST024C1" data-open="DST024C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST024C3  DST024C11" data-open="DST024C3  DST024C11" class="link link-ref link-reveal xref-bibr">3–11</a></sup></p></div></div><div content-id="DST024F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02403.jpeg?Expires=2147483647&Signature=Dgqvj5TYybLezCHcqM0dE7Q6ocFwlspSXpvqfkDn2PGaQAICfbN5YdxXLhC7ahHz0ncg5zrqdDrnS~T~Bi8df7D944ZyQMAEKVGqnJXAi3AQ8CB8R5zCzXVJN6ZTZmCRhLRVIBRiRVgLuFBmmA92CQ0REBWbkICBdhdxmhE2sQkltLLNTlYHzFhHFB79Xz9k0QSTj4sqpZYim7cao4YVpNwkGud-xD0rks~qaeKHERKzgU~bTAxrBl7K3~tujGAqcyROCnaZYU8QQrnK6yzTqEtg3Ml5HtW41jMgYxbdnmGl6Fc~ecg5BLBx7HA~P94e0ANT5W0gviiJR0OKw4I6Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Helitron abundances in different organisms. Underlined Helitron contents were identified in this study, while others came from previous studies.1,3–11" path-from-xml="dst02403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185527/dst02403.jpeg" path-from-xml="dst02403.jpeg" target="_blank">View large</a><a section="5185527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02403.jpeg?Expires=2147483647&Signature=CICGcIZoc3fJ8WkFIeWLY~h~P0mjyxdwMOsGJ4D4p6-W6Rtidf4f~xwFQyqDpjKS~-YaGFf0VpDTmy282aKQKo~hDEH6NPIbOtueN22wKA-iN2Lv5laRFjKNc~mfD2EQU78lD3N~ifbRyIubSfgs5yrsfY6827LcT9WU3Yy5cRUlO-bG4mcfPREVCtSFg87Yjh47YlJRB5BbbAkM8xmGlrr3NpexOafigtdmpMZTJMfrC2N7Q879C6sxRv5T8xhowb~8XO-gWEvlPrPJTUBnRPaBTtxlUFCAp5N22JmyYURheXvCRVHU9Y9k9eiZuUN7xz-jeg3AEb3rgj-RK1aR3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185527&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Helitron</em> abundances in different organisms. Underlined <em>Helitron</em> contents were identified in this study, while others came from previous studies.<sup><a reveal-id="DST024C1" data-open="DST024C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST024C3  DST024C11" data-open="DST024C3  DST024C11" class="link link-ref link-reveal xref-bibr">3–11</a></sup></p></div></div>                            <h3 scrollto-destination=5185528 id="5185528" class="section-title" data-legacy-id=s3d>3.4. Distributions of <em>Helitrons</em> on chromosomes</h3>
<p>The silkworm <em>Helitrons</em> were distributed on all 28 silkworm chromosomes and were uniformly distributed among chromosomes (<em>P</em>&gt; 0.05; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). We also found that most silkworm <em>Helitrons</em> had no distinct insertion bias within chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). When we examined whether the <em>Helitrons</em> preferentially inserted into or near genes, we discovered that their frequencies within introns and &gt;1 kb from genes were significantly higher than expected (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>), suggesting the silkworm <em>Helitrons</em> preferential insertion into these regions.</p>                            <h3 scrollto-destination=5185530 id="5185530" class="section-title" data-legacy-id=s3e>3.5. Insertion times and expansion patterns</h3>
<p>We estimated the age of each intact <em>Helitron</em> by first estimating <em>k</em> between each intact <em>Helitron</em> and its family consensus sequence based on K2P distances.<sup><a reveal-id="DST024C28" data-open="DST024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The range was 0–0.69, but 15 443 copies (∼79% of the 19 580 copies) had <em>k</em>≤ 0.06. Insertion dates based on these <em>k-</em>values ranged from 0 to &gt;10 mya (Fig. <a reveal-id="DST024F4" data-open="DST024F4" class="link link-reveal link-table xref-fig">4</a>A), but most expansion events appeared to have happened within the most recent 2 my (corresponding to <em>k</em> = ∼0.06). </p>                        <a id="5185532" scrollto-destination="5185532"></a>
<div id="DST024F4" data-id="DST024F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02404.jpeg?Expires=2147483647&Signature=F5rewMGIP7UMLwfgVCPLObm4KSHeuS0g9jtJnmQGNpeSAlpI9jk11eVd9ao2ZjucWt9BX2Cx8jd04OWUNjzQtliquEdLZK17-IubiQaDbTCYcB669m33t4LnV-NSns2HH8B3NPFKQeBbGBM1vYuQze0No1wtvMbFR20PEXRpcrdPM9mZYIUtjhpy8trdAdGXknBJH~~1MiZjlENoRImt6vXwQnib2WSlEbYEF6l46AeljWg60-kQxJHmhTPKAVfe8nAHZZGtby7ba8w816XbhIABOSF4EnEnbVk2WbNrdT7VgUbARz~2dY~hylMU57kvWSQwpei8md5coRyqkxBysA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary history of Helitrons in the silkworm. (A) Helitron amplification dates. (B) Distributions of pair-wise nucleotide diversity among full-length elements of BmHel-3a (with a bimodal distribution, suggesting more than one round of amplification) and BmHel-18 (with a unimodal distribution, suggesting one amplification burst). (C) Phylogenetic trees of BmHel-3a (bimodal pair-wise nucleotide diversity and more than one clade) and BmHel-18 (unimodal pair-wise nucleotide diversity and a single clade)." path-from-xml="dst02404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185532/dst02404.jpeg" path-from-xml="dst02404.jpeg" target="_blank">View large</a><a section="5185532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02404.jpeg?Expires=2147483647&Signature=ty0eVcAgO7IegWCCGr3Qd8VQXtHTEVd7rv70x479Ewuq8ipjvTAaiVz3SWMTfr-usgMOcBjObZjHzXspdQ4WCn~LzW73wy3W2lLZyiv4Qd42Q4CRxcYR09iLL2pFQ6WSSBgvJMC9s9g7csUZjqb-cg2j-7mDnxH2GQMkCFPo7iocBjCiuhYzpZ1xd92agwbBvZYpFJst0jL3Rwa2~KgtxnwshLyVU6yOpQnWklV6w1LGUXV-D4aiVVuMYq2zKrEJcySQOyQ0KnYtHU4LUfI3IN9y01-mx5shTJltvIxqI9~uspw5eAFWyASiAmfzPhNNVEGm59Gy9TZrCeqTO2JJyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185532&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary history of <em>Helitrons</em> in the silkworm. (A) <em>Helitron</em> amplification dates. (B) Distributions of pair-wise nucleotide diversity among full-length elements of BmHel-3a (with a bimodal distribution, suggesting more than one round of amplification) and BmHel-18 (with a unimodal distribution, suggesting one amplification burst). (C) Phylogenetic trees of BmHel-3a (bimodal pair-wise nucleotide diversity and more than one clade) and BmHel-18 (unimodal pair-wise nucleotide diversity and a single clade).</p></div></div><div content-id="DST024F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02404.jpeg?Expires=2147483647&Signature=F5rewMGIP7UMLwfgVCPLObm4KSHeuS0g9jtJnmQGNpeSAlpI9jk11eVd9ao2ZjucWt9BX2Cx8jd04OWUNjzQtliquEdLZK17-IubiQaDbTCYcB669m33t4LnV-NSns2HH8B3NPFKQeBbGBM1vYuQze0No1wtvMbFR20PEXRpcrdPM9mZYIUtjhpy8trdAdGXknBJH~~1MiZjlENoRImt6vXwQnib2WSlEbYEF6l46AeljWg60-kQxJHmhTPKAVfe8nAHZZGtby7ba8w816XbhIABOSF4EnEnbVk2WbNrdT7VgUbARz~2dY~hylMU57kvWSQwpei8md5coRyqkxBysA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary history of Helitrons in the silkworm. (A) Helitron amplification dates. (B) Distributions of pair-wise nucleotide diversity among full-length elements of BmHel-3a (with a bimodal distribution, suggesting more than one round of amplification) and BmHel-18 (with a unimodal distribution, suggesting one amplification burst). (C) Phylogenetic trees of BmHel-3a (bimodal pair-wise nucleotide diversity and more than one clade) and BmHel-18 (unimodal pair-wise nucleotide diversity and a single clade)." path-from-xml="dst02404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185532/dst02404.jpeg" path-from-xml="dst02404.jpeg" target="_blank">View large</a><a section="5185532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02404.jpeg?Expires=2147483647&Signature=ty0eVcAgO7IegWCCGr3Qd8VQXtHTEVd7rv70x479Ewuq8ipjvTAaiVz3SWMTfr-usgMOcBjObZjHzXspdQ4WCn~LzW73wy3W2lLZyiv4Qd42Q4CRxcYR09iLL2pFQ6WSSBgvJMC9s9g7csUZjqb-cg2j-7mDnxH2GQMkCFPo7iocBjCiuhYzpZ1xd92agwbBvZYpFJst0jL3Rwa2~KgtxnwshLyVU6yOpQnWklV6w1LGUXV-D4aiVVuMYq2zKrEJcySQOyQ0KnYtHU4LUfI3IN9y01-mx5shTJltvIxqI9~uspw5eAFWyASiAmfzPhNNVEGm59Gy9TZrCeqTO2JJyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185532&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary history of <em>Helitrons</em> in the silkworm. (A) <em>Helitron</em> amplification dates. (B) Distributions of pair-wise nucleotide diversity among full-length elements of BmHel-3a (with a bimodal distribution, suggesting more than one round of amplification) and BmHel-18 (with a unimodal distribution, suggesting one amplification burst). (C) Phylogenetic trees of BmHel-3a (bimodal pair-wise nucleotide diversity and more than one clade) and BmHel-18 (unimodal pair-wise nucleotide diversity and a single clade).</p></div></div><p>To investigate the history of <em>Helitron</em> expansion in silkworms, pair-wise nucleotide diversities of intact <em>Helitrons</em> were calculated and histograms were drawn for each <em>Helitron</em> family. Most histograms were wave-like (Fig. <a reveal-id="DST024F4" data-open="DST024F4" class="link link-reveal link-table xref-fig">4</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). These histograms indicated that each family may have experienced a rapid population expansion (burst) during its evolutionary history.<sup><a reveal-id="DST024C35" data-open="DST024C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DST024C36" data-open="DST024C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Thirteen families (BmHel-4, 7, 8, 10, 11, 12, 13, 14, 15, 16, 17, 20, and 21) of silkworm <em>Helitrons</em> had unimodal distributions, two (BmHel-2 and 3) had bimodal distributions, and the other six families displayed multimodal distributions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>), indicating that these <em>Helitron</em> families had experienced one, two, or multiple expansions, respectively.</p><p>To further investigate the histories of these <em>Helitron</em> families, phylogenetic trees were reconstructed (Fig. <a reveal-id="DST024F4" data-open="DST024F4" class="link link-reveal link-table xref-fig">4</a>C and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). Families with unimodal histograms formed star-shaped clades, indicating a rapid amplification from a single master element. Those with bi- or multimodal distributions had more than one clade, providing evidence for amplification bursts at different times. Most silkworm <em>Helitron</em> families experienced a single evolutionary radiation.</p>                            <h3 scrollto-destination=5185535 id="5185535" class="section-title" data-legacy-id=s3f>3.6. Gene fragment acquisition and contribution to transcripts</h3>
<p>To estimate the numbers of gene fragments captured by silkworm <em>Helitrons</em>, we performed a BLASTX search against the NCBI nr protein database. More than 18% (3546/19 580) of the intact elements captured one or more gene fragments (Fig. <a reveal-id="DST024F5" data-open="DST024F5" class="link link-reveal link-table xref-fig">5</a>A). The number of captured gene fragments ranged from one to six. Most intact <em>Helitrons</em> (∼96%) captured no more than one gene fragment. A total of 3724 gene fragments from 268 genes were captured (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Examples of genes captured by <em>Helitrons</em> are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>. </p>                        <a id="5185537" scrollto-destination="5185537"></a>
<div id="DST024F5" data-id="DST024F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02405.jpeg?Expires=2147483647&Signature=3StC2U~tXHn1EfulDipsISbCTFba7BTMI3E-zv8hqrShrL0lErO4cufsd9NoLSZa4nU1GohHLmm2n2PAXGdWWeqyce7TfWoOSpTmTn9vwtbKWjlmHquSEnswOMW308kbBgXqyy4OWevIehkhbEbrFl9gF3K910eWBevNulja0vwZ6IjWz8c4MxP5dkKN~JQaQzgE6GhELp5p~e955KdEa9PqaqgGVj0y9L-pJ3LLxXPbjUn9UwEvIBJ3X0JkhV4ksyktJL4eA0wrcVwRtzZTX0xKC7RIISKjrtqAKfNBIKNFLfBSu-0JkVRi339QRN2~m8FEdVFKeVbW47UjCpGIUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Silkworm Helitrons within genes. (A) Distribution of the number of gene fragments captured by silkworm Helitrons. (B) Silkworm Helitrons contributed to coding regions and to 5′- and 3′-UTRs of full-length cDNA." path-from-xml="dst02405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185537/dst02405.jpeg" path-from-xml="dst02405.jpeg" target="_blank">View large</a><a section="5185537" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02405.jpeg?Expires=2147483647&Signature=RZM78nErVwVWh7u1le9FdDyzMoFsOnWUr6fEuFaL48U8eWNarshTM8I67WIQsLiruGG64QgbyjWa1MzW3vzAKbzZXZ0dy5NmmDdpWO79A6jvYR3umqBpxUpw4DNzf3xAXhJ0RiCM6jCdOrzmErQiSdxtQ97Pg-4CMX7yNUk0W5MfbyL8iWClv8Nq~7dhYHvkDgb4J06DTqN7krNyWbmXZYSK6P~ZcV9EktbyqoGcJzoeSOzAGrKWrVLXA9I9AXBy40VrHt6xR2LrjShHP6dVwLU2m3vM9F9icH6m2r4QNqvUejKBUO9~990HDJCMlsw-P~FFhDz3NTIWI4v5AgOIsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185537&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Silkworm <em>Helitrons</em> within genes. (A) Distribution of the number of gene fragments captured by silkworm <em>Helitrons</em>. (B) Silkworm <em>Helitrons</em> contributed to coding regions and to 5′- and 3′-UTRs of full-length cDNA.</p></div></div><div content-id="DST024F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02405.jpeg?Expires=2147483647&Signature=3StC2U~tXHn1EfulDipsISbCTFba7BTMI3E-zv8hqrShrL0lErO4cufsd9NoLSZa4nU1GohHLmm2n2PAXGdWWeqyce7TfWoOSpTmTn9vwtbKWjlmHquSEnswOMW308kbBgXqyy4OWevIehkhbEbrFl9gF3K910eWBevNulja0vwZ6IjWz8c4MxP5dkKN~JQaQzgE6GhELp5p~e955KdEa9PqaqgGVj0y9L-pJ3LLxXPbjUn9UwEvIBJ3X0JkhV4ksyktJL4eA0wrcVwRtzZTX0xKC7RIISKjrtqAKfNBIKNFLfBSu-0JkVRi339QRN2~m8FEdVFKeVbW47UjCpGIUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Silkworm Helitrons within genes. (A) Distribution of the number of gene fragments captured by silkworm Helitrons. (B) Silkworm Helitrons contributed to coding regions and to 5′- and 3′-UTRs of full-length cDNA." path-from-xml="dst02405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185537/dst02405.jpeg" path-from-xml="dst02405.jpeg" target="_blank">View large</a><a section="5185537" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02405.jpeg?Expires=2147483647&Signature=RZM78nErVwVWh7u1le9FdDyzMoFsOnWUr6fEuFaL48U8eWNarshTM8I67WIQsLiruGG64QgbyjWa1MzW3vzAKbzZXZ0dy5NmmDdpWO79A6jvYR3umqBpxUpw4DNzf3xAXhJ0RiCM6jCdOrzmErQiSdxtQ97Pg-4CMX7yNUk0W5MfbyL8iWClv8Nq~7dhYHvkDgb4J06DTqN7krNyWbmXZYSK6P~ZcV9EktbyqoGcJzoeSOzAGrKWrVLXA9I9AXBy40VrHt6xR2LrjShHP6dVwLU2m3vM9F9icH6m2r4QNqvUejKBUO9~990HDJCMlsw-P~FFhDz3NTIWI4v5AgOIsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185537&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Silkworm <em>Helitrons</em> within genes. (A) Distribution of the number of gene fragments captured by silkworm <em>Helitrons</em>. (B) Silkworm <em>Helitrons</em> contributed to coding regions and to 5′- and 3′-UTRs of full-length cDNA.</p></div></div><p>To investigate whether these <em>Helitrons</em> had transcriptional activity, we performed a BLASTN search against the silkworm EST database and discarded parental transcripts. We found that 1317 (∼6.7%) intact <em>Helitrons</em> had transcriptional activity, contributing to 1210 ESTs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Among these ESTs, five matched the transposases of BmHel-2 (BY927485) and BmHel-3a (BB983132, BY932007, CK528421, and BY916909), while 1205 matched 1317 intact <em>Helitrons</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). We could not distinguish between <em>Helitron</em> transcripts and transcripts composed of <em>Helitron</em> fragments plus other genes (the 1205 ESTs), because most ESTs were too short.</p><p>To estimate whether the <em>Helitrons</em> contributed to 5′-UTRs, ORFs, and 3′-UTRs of full-length silkworm cDNAs, we performed BLASTN searches and discarded parental and <em>Helitron</em> transposase transcripts. The intact <em>Helitrons</em> contributed to the 5′­-UTRs of 35 full-length cDNAs, to the ORFs of 44, and to the 3′-UTRs of 44 (Fig. <a reveal-id="DST024F5" data-open="DST024F5" class="link link-reveal link-table xref-fig">5</a>B; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7–9</a>). These donated fragments contributed to 123 full-length cDNAs, which represented ∼1.4% (123 of 8,654) of the silkworm full-length cDNAs. Examples are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>.</p>                            <h3 scrollto-destination=5185540 id="5185540" class="section-title" data-legacy-id=s3g>3.7. New <em>Helitron</em> creation through combinations of different <em>Helitrons</em></h3>
<p>By clustering of all silkworm <em>Helitron</em> family consensus sequences with an all-versus-all BLAST search, we found that some distinct <em>Helitrons</em> had merged to form new <em>Helitrons</em>. Three examples are shown in Fig. <a reveal-id="DST024F6" data-open="DST024F6" class="link link-reveal link-table xref-fig">6</a>: BmHel-6 was formed from forward-oriented segments of BmHel-7 and BmHel-8; BmHel-9 comprised forward BmHel-7 and reverse BmHel-8 segments; and BmHel-10 united forward segments of BmHel-7 and dBmHel-11. </p>                        <a id="5185542" scrollto-destination="5185542"></a>
<div id="DST024F6" data-id="DST024F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02406.jpeg?Expires=2147483647&Signature=cSpH7FH8rUqos9VsBhczQvnZm8350XmZhy9V0dsgUjLQ5vbGSYEsyn2Jh-mLC~~wgD1NI1YqtTArCDmgKpsB-dY1eRtMzWQGakxaPjcLg51QTeMcktwU096aJfSzJH~kP4VhCB8QAAlUkJcpHRWQp3HLqfCUFR9hmIh~Dy~SeX73Hh~FUuEWHXZRA0kveAyExGBVei9kxDL3je6ETuPcG2Tbj4MHl2IKdhxiAQuF0w9rIvJlCqh1S3PGcXEJWok6pTqB1uSSXHuoOl5pcf2EpHsV~Qeq-WqXz9Y7SpUWlBDWUHK8L-xzmufXdGGJ2BdNTUM~Sx50YRj6XZBVE1TpdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Possible mechanism of new Helitron generation through Helitron sequence acquisition and new end creation." path-from-xml="dst02406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185542/dst02406.jpeg" path-from-xml="dst02406.jpeg" target="_blank">View large</a><a section="5185542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02406.jpeg?Expires=2147483647&Signature=27rygBDlKlkwXyn4o251XmPYRmCU4op~bMUagd34vngAIWRdMIJajt4Q6rgNPa9tlEve6Ut6L16NkZ24-7-aiUPVVAs3YkXu~fS9LYCaS2UWCyVrPEiU45mfaVKADTymYeh2~EppErSpy6KQ7zwaSFlPAYToIED2btOJYb0IYb9YMnPRpkY6weIDowCswqEhOvzsQEOe-P1-6kidSyIgeKyYoNV1ju39KsrlvWzBTV1yuYjl~xRRv6SXKdVH9u~94YFypJ3UHDcPTT0Xco2ZKOUYvV~iixB6G1lPK~PcbWYs1zBR0CIugC5~Bx4Lq44GmXJrIpt8fgZidkMmJ93dvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185542&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Possible mechanism of new <em>Helitron</em> generation through <em>Helitron</em> sequence acquisition and new end creation.</p></div></div><div content-id="DST024F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/m_dst02406.jpeg?Expires=2147483647&Signature=cSpH7FH8rUqos9VsBhczQvnZm8350XmZhy9V0dsgUjLQ5vbGSYEsyn2Jh-mLC~~wgD1NI1YqtTArCDmgKpsB-dY1eRtMzWQGakxaPjcLg51QTeMcktwU096aJfSzJH~kP4VhCB8QAAlUkJcpHRWQp3HLqfCUFR9hmIh~Dy~SeX73Hh~FUuEWHXZRA0kveAyExGBVei9kxDL3je6ETuPcG2Tbj4MHl2IKdhxiAQuF0w9rIvJlCqh1S3PGcXEJWok6pTqB1uSSXHuoOl5pcf2EpHsV~Qeq-WqXz9Y7SpUWlBDWUHK8L-xzmufXdGGJ2BdNTUM~Sx50YRj6XZBVE1TpdA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Possible mechanism of new Helitron generation through Helitron sequence acquisition and new end creation." path-from-xml="dst02406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5185542/dst02406.jpeg" path-from-xml="dst02406.jpeg" target="_blank">View large</a><a section="5185542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst02406.jpeg?Expires=2147483647&Signature=27rygBDlKlkwXyn4o251XmPYRmCU4op~bMUagd34vngAIWRdMIJajt4Q6rgNPa9tlEve6Ut6L16NkZ24-7-aiUPVVAs3YkXu~fS9LYCaS2UWCyVrPEiU45mfaVKADTymYeh2~EppErSpy6KQ7zwaSFlPAYToIED2btOJYb0IYb9YMnPRpkY6weIDowCswqEhOvzsQEOe-P1-6kidSyIgeKyYoNV1ju39KsrlvWzBTV1yuYjl~xRRv6SXKdVH9u~94YFypJ3UHDcPTT0Xco2ZKOUYvV~iixB6G1lPK~PcbWYs1zBR0CIugC5~Bx4Lq44GmXJrIpt8fgZidkMmJ93dvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5185542&ar=359636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Possible mechanism of new <em>Helitron</em> generation through <em>Helitron</em> sequence acquisition and new end creation.</p></div></div>                            <h2 scrollto-destination=5185543 id="5185543" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5185544 id="5185544" class="section-title" data-legacy-id=s4a>4.1. Identification and characterization of silkworm <em>Helitrons</em></h3>
<p>TEs in higher eukaryotic genomes are identified in two main ways: homology-based and structure-based methods. Homology-based methods are biased toward detecting previously identified families; its major limitation is that it cannot detect TEs that are very distinct in sequence from known TEs. In contrast, structure-based method use prior knowledge about the common structural features and can effectively identify unique TEs. However, a precondition for this method is that TEs have conserved sequence structures. Homology- and structure-based methods have been developed to identify <em>Helitrons</em>. The homology-based <em>Helitron</em>Finder has been used to identify <em>Helitrons</em> in the maize genome,<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> but its applicability to other organisms is limited by the fact that <em>Helitrons</em> vary greatly among organisms. HelSearch is a structure-based programme.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>We developed a new structure-based pipeline to identify <em>Helitrons</em> in the silkworm genome. This method was fast and effective. We used Usearch to cluster sequences with <em>Helitron</em> end structures rather than BLASTALL, which greatly sped the clustering but also generated a consensus sequence for each cluster. In addition, we discarded false-positive matches to non-<em>Helitron</em> TEs when all candidate <em>Helitron</em> sequences were used to BLAST against the known silkworm TEs. A total of 951 such consensus sequences were discarded. Finally, the computational requirements of this method were very low; an ordinary PC machine could complete all of the work. Our pipeline was structure-based like HelSearch, so both methods should have similar efficacy.</p><p>Although the silkworm genome is rich in various types of TEs,<sup><a reveal-id="DST024C18" data-open="DST024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> few <em>Helitrons</em> have been identified. We performed a genome-wide structure-based scan for <em>Helitrons</em> and identified 21 <em>Helitron</em> families with a total of 67 555 copies. These <em>Helitron</em> families comprised ∼4.23% of the silkworm genome, a proportion higher than in other organisms. For instance, <em>Helitrons</em> occupied &gt;2% of the genome sequence in <em>C. elegans</em>, &gt;0.5% in frog, ∼3% in the <em>N. vectensis</em>, &lt;0.1% in <em>A. nidulans</em>, ∼1.3% in <em>Arabidopsis thaliana</em>, ∼1.29% in <em>Medicago truncatula</em>, ∼3% in bat, and ∼2% in maize.<sup><a reveal-id="DST024C1" data-open="DST024C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST024C3  DST024C8" data-open="DST024C3  DST024C8" class="link link-ref link-reveal xref-bibr">3–8</a>,<a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Furthermore, our estimated proportion was much higher than the value of ∼0.1% previously reported for silkworm.<sup><a reveal-id="DST024C18" data-open="DST024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This discrepancy may be because the previous study used an homology-based search. <em>Helitrons</em> vary greatly among organisms,<sup><a reveal-id="DST024C9" data-open="DST024C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST024C37" data-open="DST024C37" class="link link-ref link-reveal xref-bibr">37</a></sup> so an homology-based search could greatly underestimate <em>Helitron</em> content.</p><p>We found that 13 of 21 <em>Helitron</em> families were new, while eight had been previously published in the BmTEdb or RepBase database or in papers.<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST024C34" data-open="DST024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> One silkworm <em>Helitron</em>-like family (Bm_1607) published in the BmTEdb was not identified in this study, because it does not have typical <em>Helitron</em> characteristics, such the 5′-TC dinucleotide and 3′-CTRR motif. These results, together with PCR verification, indicated that our approach was reliable and efficient at identifying <em>Helitrons</em>, but it could not identify structurally atypical <em>Helitron</em> families.</p><p>We found that the silkworm genome had 19 580 intact <em>Helitrons</em>, many more than the 1930 intact <em>Helitrons</em> in maize, the 281 in <em>Arabidopsis</em> and <em>C. elegans</em>, the 230 in <em>Medicago</em>, the 651 in rice, and the 608 in sorghum.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Their abundance and high sequence identities (&gt;90% in 8615 of 19 580 sequences, or ∼44%) in the silkworm genome implied a recent amplification history. Two <em>Helitron</em> families (BmHel-2 and -3) exhibited features of putative autonomous families. BmHel-5 (BmHel1p) was previously identified as putatively autonomous,<sup><a reveal-id="DST024C38" data-open="DST024C38" class="link link-ref link-reveal xref-bibr">38</a></sup> suggesting that autonomous <em>Helitrons</em> may exist in the silkworm. However, BmHel-2 and -5 had very small copy numbers, just 79 and 80 copies, respectively. In contrast, some silkworm <em>Helitron</em> families with many copies did not have features of putative autonomous elements. A possible reason will be discussed below.</p>                            <h3 scrollto-destination=5185550 id="5185550" class="section-title" data-legacy-id=s4b>4.2. <em>Helitron</em> abundances in other lepidopterans</h3>
<p>We analysed <em>Helitron</em> abundances in three recently released lepidopteran genomes using our approach. <em>Helitron</em> abundance varied strikingly among lepidopterans. For instance, there were 10 <em>Helitron</em> families that constituted ∼6.62% of the genome in <em>H. melpomene</em>. This proportion was much larger than the previously reported 5.37%,<sup><a reveal-id="DST024C16" data-open="DST024C16" class="link link-ref link-reveal xref-bibr">16</a></sup> because we identified an additional big <em>Helitron</em> family (HmHel-6) that comprised ∼1.17% of the genome. However, we did not find the low-copy-number families of <em>Helitron</em>-like-2, 8, and 10, because they lacked the 5′-TC dinucleotide and 3′-CTRR motif. Unexpectedly, only one family, representing ∼0.20% of the <em>D. plexippus</em> genome, was identified. Thus, different lepidopteran genomes contained very different numbers of <em>Helitrons</em>, consistent with previous reports that <em>Helitron</em> content was highly variable even among closely-related species. For instance, <em>Helitrons</em> make up 1–5% of genomic DNA in different fruit fly species,<sup><a reveal-id="DST024C9" data-open="DST024C9" class="link link-ref link-reveal xref-bibr">9</a></sup> 0–3% in mammals.<sup><a reveal-id="DST024C6" data-open="DST024C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST024C39" data-open="DST024C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>Different <em>Helitron</em> abundances could be caused by three factors: (i) different rates of <em>Helitron</em> expansion or deletion in different lineages; (ii) vertical transfer with frequent diversification and extinction; and (iii) horizontal acquisition of <em>Helitrons</em>. For example, there were 21 <em>Helitron</em> families in the silkworm genome, but only one in the <em>D. plexippus</em> genome. These results could be caused by horizontal transfer. A previous study reported that five <em>Helitron</em> families (BmHel-3a, 4, 5, 20, and 21) experienced horizontal transfer in the silkworm.<sup><a reveal-id="DST024C33" data-open="DST024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Furthermore, we found that an intact homologue of BmHel-16 was also present in the <em>Cotesia sesamiae</em> Kitale bracovirus genome. A phylogenetic analysis indicated that the silkworm BmHel-16 was more closely related to the <em>C. sesamiae</em> bracovirus copy than that to sequences from other Lepidoptera (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>), suggesting horizontal transfer. However, BmHel-16 and the <em>C. sesamiae</em> bracovirus sequence had only 79% identity, implying that the horizontal transfer happened long ago.</p>                            <h3 scrollto-destination=5185553 id="5185553" class="section-title" data-legacy-id=s4c>4.3. Distribution of <em>Helitrons</em> on chromosomes</h3>
<p>Previous studies indicated that <em>Helitrons</em> preferentially insert into gene-poor regions.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> For instance, <em>Helitrons</em> in <em>Arabidopsis</em> were rich in pericentromeric regions. Similarly, <em>Helitrons</em> in the <em>C. elegans</em> genome were most abundant in the terminal regions of each chromosome, which are often in the heterochromatin state. However, the silkworm <em>Helitrons</em> were randomly distributed on chromosomes. If <em>Helitrons</em> generally insert into heterochromatin regions, their random distribution in the silkworm genome is expected, because silkworm chromosomes are holocentromeres.</p><p>We also found that the numbers of <em>Helitrons</em> that inserted into introns and &gt;1 kb away from genes were higher than expected. The reasons for this observation are not clear. A previous study proved that genes captured by <em>Helitrons</em> reshuffled the transcriptome of maize.<sup><a reveal-id="DST024C12" data-open="DST024C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Thus, preferential accumulation in intron regions could drive gene evolution through gene capture and exonization of <em>Helitrons</em>.</p>                            <h3 scrollto-destination=5185556 id="5185556" class="section-title" data-legacy-id=s4d>4.4 Massive expansions and diversity patterns</h3>
<p>Our results indicated that major expansion events of silkworm <em>Helitrons</em> occurred in the past 2 my (Fig. <a reveal-id="DST024F4" data-open="DST024F4" class="link link-reveal link-table xref-fig">4</a>A). Similarly, about 87% of <em>BaShos</em> insertions occurred within the most recent 5 my in <em>A. thaliana</em> and ∼71% of Hel1-105 elements and 69% of Hel-106 elements inserted within the past 1 my in maize.<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST024C40" data-open="DST024C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>To investigate history of the silkworm <em>Helitron</em> amplifications, we estimated pair-wise nucleotide diversity and phylogenetic trees of the full-length <em>Helitrons</em> (Fig. <a reveal-id="DST024F4" data-open="DST024F4" class="link link-reveal link-table xref-fig">4</a>B and C). Most <em>Helitron</em> families experienced single expansions. However, the mechanism was not clear. As discussed above, we did not find putative autonomous <em>Helitron</em> copies in some high-copy-number families (e.g. BmHel-8, 11, and 15). In contrast, some low-copy-number families (BmHel-2 and 5) appeared to be putative autonomous elements. These results were reminiscent of miniature inverted-repeat TEs (MITEs) that were highly transposable because of transposases encoded by distantly related and self-restrained autonomous elements in rice,<sup><a reveal-id="DST024C41" data-open="DST024C41" class="link link-ref link-reveal xref-bibr">41</a></sup> a mechanism known as cross-mobilization. Thus, some non-autonomous <em>Helitrons</em> might move using transposases encoded by autonomous <em>Helitrons</em>. Whether this is true in silkworm <em>Helitrons</em> remains to be investigated.</p><p>Why some silkworm <em>Helitrons</em> experienced bursts of expansion is not clear. Most <em>Helitrons</em> probably remain inactive for most of their evolutionary histories, and they may be suddenly activated by ‘genome shock’, as observed in rice MITEs. For instance, <em>mPing</em> is known to be activated by irradiation, cell culture, and recent domestication.<sup><a reveal-id="DST024C42  DST024C44" data-open="DST024C42  DST024C44" class="link link-ref link-reveal xref-bibr">42–44</a></sup> The silkworm was domesticated from wild Chinese silkworms about 5000 years ago.<sup><a reveal-id="DST024C45" data-open="DST024C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DST024C46" data-open="DST024C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Whether some silkworm <em>Helitrons</em> were activated by domestication is an interesting question.</p>                            <h3 scrollto-destination=5185560 id="5185560" class="section-title" data-legacy-id=s4e>4.5. Gene fragments acquisition and contribution to transcripts</h3>
<p><em>Helitrons</em> vary greatly in sequence length, even within a family. One explanation is that these elements capture different gene fragments.<sup><a reveal-id="DST024C7" data-open="DST024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST024C12" data-open="DST024C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST024C47" data-open="DST024C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Although some molecular mechanisms for gene capture have been proposed,<sup><a reveal-id="DST024C48  DST024C51" data-open="DST024C48  DST024C51" class="link link-ref link-reveal xref-bibr">48–51</a></sup> clear experimental evidence for a particular mechanism is lacking. In this study, we found that 3546 intact <em>Helitrons</em> (&gt;18% of all intact <em>Helitrons</em>) had captured one or more gene fragments, for a total of 3724 captured fragments. The average number of captured gene fragments per intact <em>Helitron</em> was 1.08, similar to the value (1.81) for maize <em>Helitrons</em>.<sup><a reveal-id="DST024C8" data-open="DST024C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Furthermore, we found that ∼6.7% of intact silkworm <em>Helitrons</em> (1317 of 19 580) had EST evidence. Based on a homology search against silkworm full-length cDNAs, we found that these intact <em>Helitrons</em> contributed to about 123 full-length cDNAs (∼1.4% of the published total) by donating one or more exons. A recent study suggested that ∼9% of maize <em>Helitrons</em> had EST evidence and could generate abundant transcripts through alternative splicing.<sup><a reveal-id="DST024C12" data-open="DST024C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Thus, <em>Helitrons</em> may play important roles in the evolution of silkworm transcripts.</p>                            <h3 scrollto-destination=5185563 id="5185563" class="section-title" data-legacy-id=s4f>4.6. Generation of new <em>Helitrons</em></h3>
<p>Previous studies indicated that <em>Helitrons</em> could acquire new sequences by recognizing either a new 3′ termination site or a new 5′ start site.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST024C52" data-open="DST024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> A hairpin was proposed to serve as a stop signal during <em>Helitron</em> transposition. When this hairpin is destroyed by unknown mechanism, a new hairpin-like sequence could be acquired, perhaps from nearby <em>Helitrons</em>, to generate chimeric elements.<sup><a reveal-id="DST024C11" data-open="DST024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Interestingly, we found that two <em>Helitrons</em> could combine to produce a new <em>Helitron</em> (Fig. <a reveal-id="DST024F6" data-open="DST024F6" class="link link-reveal link-table xref-fig">6</a>). Similarly, in maize, <em>Helitron</em>_mc2 was composed of ZmHelA5 and <em>Helitron</em>_mc.<sup><a reveal-id="DST024C52" data-open="DST024C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Thus, new <em>Helitrons</em> can be generated in different ways, making them the most diverse class of transposons.</p>                            <h2 scrollto-destination=5185565 id="5185565" class="section-title" data-legacy-id=s5>5. Conclusions</h2>
<p>In present study, we developed a structure-based approach to identify <em>Helitrons</em> in a genome and analysed their presence in four Lepidoptera species. <em>Helitron</em> abundance and the number of families varied greatly among these insect genomes. One plausible explanation is that horizontal transfer caused these differences. A systematic analysis of silkworm <em>Helitrons</em> revealed that they accounted for ∼4.23% of the genome, much more than the previously reported ∼0.1%.<sup><a reveal-id="DST024C18" data-open="DST024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> A total of 21 <em>Helitron</em> families were identified in the silkworm, and 13 were new families. Most <em>Helitron</em> families expanded within the past 2 my in a single radiation. Furthermore, we found that <em>Helitrons</em> contributed to at least 1.4% of silkworm full-length cDNAs, indicating their important roles in the evolution of the silkworm genes. In addition, existent <em>Helitrons</em> could generate new families by combining. Our results provided insights into the generation and evolution of <em>Helitron</em> transposons as well as their contribution to transcripts.</p>                            <h2 scrollto-destination=5185567 id="5185567" class="section-title" data-legacy-id=s6>Authors' contribution</h2>
<p>Z.Z. and M.J.H. designed the study. Y.H.S., M.S.X., H.Y.L., H.H.Z., and M.J.H. analysed the data. Z.Z. provided the platform for analysis. Z.Z. and M.J.H. drafted and revised the manuscript. All authors read and approved the final manuscript.</p>                            <h2 scrollto-destination=5185569 id="5185569" class="section-title" data-legacy-id=s7>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5185571 id="5185571" class="section-title" data-legacy-id=s8>Funding</h2>
<p>This work was supported by the Hi-Tech Research and Development (863) Program of China (2013AA102507), a grant from Natural Science Foundation Project of CQ CSTC (cstc2012jjB80007), and the Doctorial Innovation Fund of Southwest University (kb2010016).</p>                            <h2 scrollto-destination=5185573 id="5185573" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to two anonymous reviewers for their valuable comments and critiques, which helped us to greatly improve the paper. We thank Dr Fang-Yin Dai for help in collecting domesticated silkworm samples and the members of Z.Z.'s laboratory group for their help with data analysis and helpful discussions.</p>                            <h2 scrollto-destination=5185575 id="5185575" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST024C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Rolling-circle transposons in eukaryotes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8714</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rolling-circle%20transposons%20in%20eukaryotes&amp;author=V.V.%20Kapitonov&amp;author=J.%20Jurka&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8714-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.151269298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.151269298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.151269298"> </span></div> </div></div></div></div></div><div content-id="DST024C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mendiola</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Bernales</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">de la Cruz</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Differential roles of the transposon termini in IS91 transposition</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1922</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20roles%20of%20the%20transposon%20termini%20in%20IS91%20transposition&amp;author=M.V.%20Mendiola&amp;author=I.%20Bernales&amp;author=F.%20de%20la%20Cruz&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=1922-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.5.1922" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.5.1922" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.5.1922"> </span></div> </div></div></div></div></div><div content-id="DST024C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>. </div><div class="article-title"><em>Helitron-N1_XT, a family of non-autonomous</em> Helitron <em>from frog</em></div>, <div class="source">Repbase Reports</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">494</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DST024C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Putnam</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Srivastava</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hellsten</div> <div class="given-names">U.</div></div>, et al. </div><div class="article-title">Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">317</div> (pg. <div class="fpage">86</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sea%20anemone%20genome%20reveals%20ancestral%20eumetazoan%20gene%20repertoire%20and%20genomic%20organization&amp;author=N.H.%20Putnam&amp;author=M.%20Srivastava&amp;author=U.%20Hellsten&amp;publication_year=2007&amp;journal=Science&amp;volume=317&amp;pages=86-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1139158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1139158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1139158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17615350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galagan</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Calvo</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Cuomo</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Sequencing of <em>Aspergillus nidulans</em> and comparative analysis with <em>A. fumigatus</em> and <em>A. oryzae</em></div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">1105</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20Aspergillus%20nidulans%20and%20comparative%20analysis%20with%20A.%20fumigatus%20and%20A.%20oryzae&amp;author=J.E.%20Galagan&amp;author=S.E.%20Calvo&amp;author=C.%20Cuomo&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1105-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16372000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pritham</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Massive amplification of rolling-circle transposons in the lineage of the bat <em>Myotis lucifugus</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">1895</div>-<div class="lpage">900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20amplification%20of%20rolling-circle%20transposons%20in%20the%20lineage%20of%20the%20bat%20Myotis%20lucifugus&amp;author=E.J.%20Pritham&amp;author=C.%20Feschotte&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=1895-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0609601104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0609601104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0609601104"> </span></div> </div></div></div></div></div><div content-id="DST024C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fefelova</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Caronna</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Dooner</div> <div class="given-names">H.K.</div></div>. </div><div class="article-title">The polychromatic <em>Helitron</em> landscape of the maize genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">19916</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20polychromatic%20Helitron%20landscape%20of%20the%20maize%20genome&amp;author=C.%20Du&amp;author=N.%20Fefelova&amp;author=J.%20Caronna&amp;author=L.%20He&amp;author=H.K.%20Dooner&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=19916-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0904742106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0904742106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0904742106"> </span></div> </div></div></div></div></div><div content-id="DST024C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Distributio<em>n, diversity, evolution, and survival of</em> Helitrons <em>in the maize genome</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">19922</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20diversity%2C%20evolution%2C%20and%20survival%20of%20Helitrons%20in%20the%20maize%20genome&amp;author=L.%20Yang&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=19922-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0908008106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0908008106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0908008106"> </span></div> </div></div></div></div></div><div content-id="DST024C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title"><em>Helitrons</em> in fruit flies</div>, <div class="source">Repbase Reports</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">127</div>-<div class="lpage">32</div>)</div></div></div></div><div content-id="DST024C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sebastian</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Talag</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Transposable <em>element distribution, abundance and role in genome size variation in the genus Oryza</em></div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> pg. <div class="fpage">152</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20element%20distribution%2C%20abundance%20and%20role%20in%20genome%20size%20variation%20in%20the%20genus%20Oryza&amp;author=A.%20Zuccolo&amp;author=A.%20Sebastian&amp;author=J.%20Talag&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17727727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Structure-based discovery and description of plant and animal <em>Helitrons</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">12832</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure-based%20discovery%20and%20description%20of%20plant%20and%20animal%20Helitrons&amp;author=L.%20Yang&amp;author=J.L.%20Bennetzen&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=12832-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905563106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905563106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905563106"> </span></div> </div></div></div></div></div><div content-id="DST024C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barbaglia</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Klusman</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shaw</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Hannah</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Lal</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">Gene capture by <em>Helitron</em> transposons reshuffles the transcriptome of maize</div>, <div class="source">Genetics</div> , <div class="year">2012</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">965</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20capture%20by%20Helitron%20transposons%20reshuffles%20the%20transcriptome%20of%20maize&amp;author=A.M.%20Barbaglia&amp;author=K.M.%20Klusman&amp;author=J.%20Higgins&amp;author=J.R.%20Shaw&amp;author=L.C.%20Hannah&amp;author=S.K.%20Lal&amp;publication_year=2012&amp;journal=Genetics&amp;volume=190&amp;pages=965-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.111.136176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.111.136176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.111.136176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pea</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fengler</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zuccolo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene duplication and exon shuffling by <em>Helitron</em>-like transposons generate intraspecies diversity in maize</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">997</div>-<div class="lpage">1002</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20exon%20shuffling%20by%20Helitron-like%20transposons%20generate%20intraspecies%20diversity%20in%20maize&amp;author=M.%20Morgante&amp;author=S.%20Brunner&amp;author=G.%20Pea&amp;author=K.%20Fengler&amp;author=A.%20Zuccolo&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=997-1002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A draft sequence for the genome of the domesticated silkworm (<em>Bombyx mori</em>)</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">1937</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20for%20the%20genome%20of%20the%20domesticated%20silkworm%20(Bombyx%20mori)&amp;author=Q.%20Xia&amp;author=Z.%20Zhou&amp;author=C.%20Lu&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=1937-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1102210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1102210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1102210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15591204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Merlin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Reppert</div> <div class="given-names">S.M.</div></div>. </div><div class="article-title">The monarch butterfly genome yields insights into long-distance migration</div>, <div class="source">Cell</div> , <div class="year">2011</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">1171</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20monarch%20butterfly%20genome%20yields%20insights%20into%20long-distance%20migration&amp;author=S.%20Zhan&amp;author=C.%20Merlin&amp;author=J.L.%20Boore&amp;author=S.M.%20Reppert&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=1171-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.09.052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.09.052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.09.052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22118469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Consortium</div> <div class="given-names">T.H.G.</div></div>. </div><div class="article-title">Butterfly genome reveals promiscuous exchange of mimicry adaptations among species</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">487</div> (pg. <div class="fpage">94</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Butterfly%20genome%20reveals%20promiscuous%20exchange%20of%20mimicry%20adaptations%20among%20species&amp;author=T.H.G.%20Consortium&amp;publication_year=2012&amp;journal=Nature&amp;volume=487&amp;pages=94-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grimaldi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Engel</div> <div class="given-names">M.S.</div></div>. </div>, <div class="source">Evolution of the insects</div> , <div class="year">2005</div><div class="publisher-loc">New York</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DST024C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osanai-Futahashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suetsugu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujiwara</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genome-wide screening and characterization of transposable elements and their distribution analysis in the silkworm, <em>Bombyx mori</em></div>, <div class="source">Insect. Biochem. Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">1046</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20screening%20and%20characterization%20of%20transposable%20elements%20and%20their%20distribution%20analysis%20in%20the%20silkworm%2C%20Bombyx%20mori&amp;author=M.%20Osanai-Futahashi&amp;author=Y.%20Suetsugu&amp;author=K.%20Mita&amp;author=H.%20Fujiwara&amp;publication_year=2008&amp;journal=Insect.%20Biochem.%20Mol.%20Biol.&amp;volume=38&amp;pages=1046-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ibmb.2008.05.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ibmb.2008.05.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ibmb.2008.05.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19280695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2460</div>-<div class="lpage">1</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=R.C.%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">1792</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">DAMBE: software package for data analysis in molecular biology and evolution</div>, <div class="source">J. Hered.</div> , <div class="year">2001</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">371</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DAMBE%3A%20software%20package%20for%20data%20analysis%20in%20molecular%20biology%20and%20evolution&amp;author=X.%20Xia&amp;author=Z.%20Xie&amp;publication_year=2001&amp;journal=J.%20Hered.&amp;volume=92&amp;pages=371-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/92.4.371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F92.4.371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F92.4.371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11535656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Markham</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Zuker</div> <div class="given-names">M.</div></div>. </div><div class="article-title">UNAFold: software for nucleic acid folding and hybridization</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">453</div> (pg. <div class="fpage">3</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UNAFold%3A%20software%20for%20nucleic%20acid%20folding%20and%20hybridization&amp;author=N.R.%20Markham&amp;author=M.%20Zuker&amp;publication_year=2008&amp;journal=Methods%20Mol.%20Biol.&amp;volume=453&amp;pages=3-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18712296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siguier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Perochon</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lestrade</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Mahillon</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ISfinder: the reference centre for bacterial insertion sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D32</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISfinder%3A%20the%20reference%20centre%20for%20bacterial%20insertion%20sequences&amp;author=P.%20Siguier&amp;author=J.%20Perochon&amp;author=L.%20Lestrade&amp;author=J.%20Mahillon&amp;author=M.%20Chandler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D32-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Repbase Update, a database of eukaryotic repetitive elements</div>, <div class="source">, <em>Cytogenet. Genome Res</em>.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">462</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=%2C%20Cytogenet.%20Genome%20Res.&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagaraja</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Nagaraju</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genome fingerprinting of the silkworm, <em>Bombyx mori</em>, using random arbitrary primers</div>, <div class="source">, <em>Electrophoresis</em></div> , <div class="year">1995</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1633</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20fingerprinting%20of%20the%20silkworm%2C%20Bombyx%20mori%2C%20using%20random%20arbitrary%20primers&amp;author=G.M.%20Nagaraja&amp;author=J.%20Nagaraju&amp;publication_year=1995&amp;journal=%2C%20Electrophoresis&amp;volume=16&amp;pages=1633-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8582347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide-sequences</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1980</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">111</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20estimating%20evolutionary%20rates%20of%20base%20substitutions%20through%20comparative%20studies%20of%20nucleotide-sequences&amp;author=M.%20Kimura&amp;publication_year=1980&amp;journal=J.%20Mol.%20Evol.&amp;volume=16&amp;pages=111-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01731581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01731581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01731581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div>. </div>, <div class="source">Molecular evolution</div> , <div class="year">1997</div><div class="publisher-loc">Sunderland, Massachusetts</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="DST024C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Complete resequencing of 40 genomes reveals domestication events and genes in silkworm (Bombyx)</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">433</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20resequencing%20of%2040%20genomes%20reveals%20domestication%20events%20and%20genes%20in%20silkworm%20(Bombyx)&amp;author=Q.%20Xia&amp;author=Y.%20Guo&amp;author=Z.%20Zhang&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=433-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1176620" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1176620" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1176620"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19713493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Excoffier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Laval</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Schneider</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis</div>, <div class="source">Evol. Bioinform. Online</div> , <div class="year">2005</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">47</div>-<div class="lpage">50</div>)</div></div></div></div><div content-id="DST024C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">SilkDB v2.0: a platform for silkworm (<em>Bombyx mori</em>) genome biology</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">453</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SilkDB%20v2.0%3A%20a%20platform%20for%20silkworm%20(Bombyx%20mori)%20genome%20biology&amp;author=J.%20Duan&amp;author=R.%20Li&amp;author=D.%20Cheng&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=453-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp801"> </span></div> </div></div></div></div></div><div content-id="DST024C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.Y.</div></div>,  <div class="name"><div class="surname">Xiang</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Burst expansion, distribution and diversification of MITEs in the silkworm genome</div>, <div class="source">BMC genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">520</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Burst%20expansion%2C%20distribution%20and%20diversification%20of%20MITEs%20in%20the%20silkworm%20genome&amp;author=M.J.%20Han&amp;author=Y.H.%20Shen&amp;author=Y.H.%20Gao&amp;author=L.Y.%20Chen&amp;author=Z.H.%20Xiang&amp;author=Z.%20Zhang&amp;publication_year=2010&amp;journal=BMC%20genomics&amp;volume=11&amp;pages=520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20875122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schaack</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pritham</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">Pervasive horizontal transfer of rolling-circle transposons among animals</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pervasive%20horizontal%20transfer%20of%20rolling-circle%20transposons%20among%20animals&amp;author=J.%20Thomas&amp;author=S.%20Schaack&amp;author=E.J.%20Pritham&amp;publication_year=2010&amp;journal=Genome%20Biol.%20Evol.&amp;volume=2&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evq050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevq050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevq050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20693155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coates</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Hellmich</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Grant</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Abel</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Mobilizing the genome of Lepidoptera through novel sequence gains and end creation by non-autonomous Lep1 <em>Helitrons</em></div>, <div class="source">DNA Res.</div> , <div class="year">2012</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">11</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobilizing%20the%20genome%20of%20Lepidoptera%20through%20novel%20sequence%20gains%20and%20end%20creation%20by%20non-autonomous%20Lep1%20Helitrons&amp;author=B.S.%20Coates&amp;author=R.L.%20Hellmich&amp;author=D.M.%20Grant&amp;author=C.A.%20Abel&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=11-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerjal</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Joets</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Alix</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Grandbastien</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Tenaillon</div> <div class="given-names">M.I.</div></div>. </div><div class="article-title">Contrasting evolutionary patterns and target specificities among three Tourist-like MITE families in the maize genome</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">99</div>-<div class="lpage">114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contrasting%20evolutionary%20patterns%20and%20target%20specificities%20among%20three%20Tourist-like%20MITE%20families%20in%20the%20maize%20genome&amp;author=T.%20Zerjal&amp;author=J.%20Joets&amp;author=K.%20Alix&amp;author=M.A.%20Grandbastien&amp;author=M.I.%20Tenaillon&amp;publication_year=2009&amp;journal=Plant%20Mol.%20Biol.&amp;volume=71&amp;pages=99-114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-009-9511-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-009-9511-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-009-9511-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19533380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogers</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Harpending</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Population growth makes waves in the distribution of pairwise genetic differences</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1992</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">552</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20growth%20makes%20waves%20in%20the%20distribution%20of%20pairwise%20genetic%20differences&amp;author=A.R.%20Rogers&amp;author=H.%20Harpending&amp;publication_year=1992&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=9&amp;pages=552-69" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1316531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sweredoski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">DeRose-Wilson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>. </div><div class="article-title">A comparative computational analysis of nonautonomous <em>Helitron</em> elements between maize and rice</div>, <div class="source">BMC genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">467</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20computational%20analysis%20of%20nonautonomous%20Helitron%20elements%20between%20maize%20and%20rice&amp;author=M.%20Sweredoski&amp;author=L.%20DeRose-Wilson&amp;author=B.S.%20Gaut&amp;publication_year=2008&amp;journal=BMC%20genomics&amp;volume=9&amp;pages=467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18842139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coates</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Sumerford</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Hellmich</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Lewis</div> <div class="given-names">L.C.</div></div>. </div><div class="article-title">A <em>Helitron</em>-Like Transposon Superfamily from Lepidoptera Disrupts (GAAA)(n) microsatellites and is responsible for flanking sequence similarity within a microsatellite family</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">275</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Helitron-Like%20Transposon%20Superfamily%20from%20Lepidoptera%20Disrupts%20(GAAA)(n)%20microsatellites%20and%20is%20responsible%20for%20flanking%20sequence%20similarity%20within%20a%20microsatellite%20family&amp;author=B.S.%20Coates&amp;author=D.V.%20Sumerford&amp;author=R.L.%20Hellmich&amp;author=L.C.%20Lewis&amp;publication_year=2010&amp;journal=J.%20Mol.%20Evol.&amp;volume=70&amp;pages=275-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-010-9330-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-010-9330-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-010-9330-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20217059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smit</div> <div class="given-names">A.F.A.</div></div>. </div>, <div class="year">2006</div> <div class="comment">Repeat Masker edition, <em>Release 20061006</em>, Repbase Update</div></div></div></div></div><div content-id="DST024C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hollister</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>. </div><div class="article-title">Population and evolutionary dynamics of <em>Helitron</em> transposable elements in Arabidopsis thaliana</div>, <div class="source">Mol. Biol. Evol</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2515</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20and%20evolutionary%20dynamics%20of%20Helitron%20transposable%20elements%20in%20Arabidopsis%20thaliana&amp;author=J.D.%20Hollister&amp;author=B.S.%20Gaut&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=2515-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17890239" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nagel</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hancock</div> <div class="given-names">C.N.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Tuned for transposition: molecular determinants underlying the hyperactivity of a Stowaway MITE</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">325</div> (pg. <div class="fpage">1391</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tuned%20for%20transposition%3A%20molecular%20determinants%20underlying%20the%20hyperactivity%20of%20a%20Stowaway%20MITE&amp;author=G.%20Yang&amp;author=D.H.%20Nagel&amp;author=C.%20Feschotte&amp;author=C.N.%20Hancock&amp;author=S.R.%20Wessler&amp;publication_year=2009&amp;journal=Science&amp;volume=325&amp;pages=1391-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1175688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1175688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1175688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19745152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">An active DNA transposon family in rice</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">421</div> (pg. <div class="fpage">163</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20active%20DNA%20transposon%20family%20in%20rice&amp;author=N.%20Jiang&amp;author=Z.%20Bao&amp;author=X.%20Zhang&amp;publication_year=2003&amp;journal=Nature&amp;volume=421&amp;pages=163-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakazaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okumoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horibata</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Mobilization of a transposon in the rice genome</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">421</div> (pg. <div class="fpage">170</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobilization%20of%20a%20transposon%20in%20the%20rice%20genome&amp;author=T.%20Nakazaki&amp;author=Y.%20Okumoto&amp;author=A.%20Horibata&amp;publication_year=2003&amp;journal=Nature&amp;volume=421&amp;pages=170-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naito</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">Dramatic amplification of a rice transposable element during recent domestication</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">17620</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dramatic%20amplification%20of%20a%20rice%20transposable%20element%20during%20recent%20domestication&amp;author=K.%20Naito&amp;author=E.%20Cho&amp;author=G.J.%20Yang&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=17620-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0605421103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605421103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0605421103"> </span></div> </div></div></div></div></div><div content-id="DST024C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshitake</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Phylogenetic aspects on the origin of Japanese race of the silkworm, <em>Bombyx mori</em> L</div>, <div class="source">J. Sericol. Sci. Jpn</div> , <div class="year">1968</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">83</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DST024C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Banno</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xiang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Phylogeny and evolutionary history of the silkworm</div>, <div class="source">Sci. China Life Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">483</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20evolutionary%20history%20of%20the%20silkworm&amp;author=W.%20Sun&amp;author=H.%20Yu&amp;author=Y.%20Shen&amp;author=Y.%20Banno&amp;author=Z.%20Xiang&amp;author=Z.%20Zhang&amp;publication_year=2012&amp;journal=Sci.%20China%20Life%20Sci.&amp;volume=55&amp;pages=483-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11427-012-4334-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11427-012-4334-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11427-012-4334-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22744178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pritham</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">A cornucopia of <em>Helitrons</em> shapes the maize genome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">19747</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20cornucopia%20of%20Helitrons%20shapes%20the%20maize%20genome&amp;author=C.%20Feschotte&amp;author=E.J.%20Pritham&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=19747-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0910273106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0910273106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0910273106"> </span></div> </div></div></div></div></div><div content-id="DST024C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Treasures in the attic: rolling circle transposons discovered in eukaryotic genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8923</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Treasures%20in%20the%20attic%3A%20rolling%20circle%20transposons%20discovered%20in%20eukaryotic%20genomes&amp;author=C.%20Feschotte&amp;author=S.R.%20Wessler&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8923-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.171326198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.171326198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.171326198"> </span></div> </div></div></div></div></div><div content-id="DST024C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Transposable elements, gene creation and genome rearrangement in flowering plants</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">621</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%2C%20gene%20creation%20and%20genome%20rearrangement%20in%20flowering%20plants&amp;author=J.L.%20Bennetzen&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=15&amp;pages=621-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2005.09.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2005.09.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2005.09.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16219458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pea</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Origins, genetic organization and transcription of a family of non-autonomous Helitron elements in maize</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">799</div>-<div class="lpage">810</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%2C%20genetic%20organization%20and%20transcription%20of%20a%20family%20of%20non-autonomous%20Helitron%20elements%20in%20maize&amp;author=S.%20Brunner&amp;author=G.%20Pea&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=43&amp;pages=799-810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.43.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.43.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.43.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16146520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST024C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lal</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oetjens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hannah</div> <div class="given-names">L.C.</div></div>. </div><div class="article-title"><em>Helitrons</em>: Enigmatic abductors and mobilizers of host genome sequences</div>, <div class="source">Plant Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">181</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Helitrons%3A%20Enigmatic%20abductors%20and%20mobilizers%20of%20host%20genome%20sequences&amp;author=S.%20Lal&amp;author=M.%20Oetjens&amp;author=L.C.%20Hannah&amp;publication_year=2009&amp;journal=Plant%20Sci.&amp;volume=176&amp;pages=181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2008.11.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2008.11.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2008.11.004"> </span></div> </div></div></div></div></div><div content-id="DST024C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST024C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Structural characterization of <em>Helitrons</em> and their stepwise capturing of gene fragments in the maize genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">609</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20characterization%20of%20Helitrons%20and%20their%20stepwise%20capturing%20of%20gene%20fragments%20in%20the%20maize%20genome&amp;author=Y.%20Dong&amp;author=X.%20Lu&amp;author=W.%20Song&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=609" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-609" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-609" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-609"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22177531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Takashi Ito</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359636&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst024-dst024supp_table3-9.pdf?Expires=2147483647&amp;Signature=nX1CUK2WF6VGNLpKtxfp5jOoccGiofQl8juX~wrDMjrfmmYzkXKrwK46oGvi5WHlN~TOZq0nInZPI~0tvXT-0rtrGllPTIppsI5WEtR3gNjYngfP1rbsjFSuA3T~LkSpVbX3KhuN0bZ4ca8iYFjwWol2CC~viK4fN5X-6N0ArlS7TerZImTb65aXF6NifSGkygJrOZsB5iMPxjQG40cf~zPLgpusf4YMdlPfB34xkSOzlO6Tmr2BH~cZGhV~B9t-7BjBz-AUO6euO4~1uinqQJw92BHsAwhEJGsI71O6xeIiwLReSRv5C-hfFEph0OhW2rh5t3tszmUtJE6A8c070w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst024-dst024supp_table2.pdf?Expires=2147483647&amp;Signature=zbDMemhxi3jkyDG9olc5yDyH3NsayjCoEKgTimdxYL0zuOKvXn5VGwGtaiiFI4HOme-ZXseIG-Tg6qsq1nDjIdF4zIwJS~vQjI5FuAkPVAPUUuu34194jx23PtZU7hqwT0O7HxDa9nA-oPtPVNrGbsElEyXchQHqoEJXupUaZt5k-1u9IymWCVT7HwLAeNhJnbblkjO7vnubidhaf4Hw-g2RPpyCgU9mEwozuAr7wvR3CTG~CZ2Bz-spovG1GEC7PpKaEkZ7TKOVQqHYYWSEpxbqRf0OkJ4CaU1NTtyzv6Vj0fAQsh1KVSSufKh~LWY3WAspHmWhUmHsJek0OtPFDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst024-dst024supp_table1.pdf?Expires=2147483647&amp;Signature=R5EdwcRUv-Rxv5~8rnZPjAIeBj8bINXmadnLusVEpv32COLESAsbd9WLosOVIbLz5~no~VFHLVQwhpnc-3J~BRk38GvE0zNNSF6NUql~d~Ft1DUSoGLh7X1d4~XqTInODeUQRDobrLYoqiUk~ZnSPLMKl6p5OvArOCaicPNTtA6ilgxzj~IjmbCFUvw-nnlPXNfQmm75qsP6skZn-DKy2O-Dwlip9AGHAInPGB1mIcth6Xu02KH9QnAUhZhXv528FLyWRSGACizMTARq~512gA1dF9cvbpZvZoF8~M0Z0XazqQeL6EmqjDdtIeBye2F44sZ0Weir3RMXMSP6cL0yUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst024/2/dst024-dst024supp_figs1-11.pdf?Expires=2147483647&amp;Signature=WbN-zxj0lg2FWRmiRTAutpz4GS3rIXBDvhVMEhJ5tzFK3YbZajSIaqf8XXWP6poUPZQpTbi-2wPDYB-~q4~NOiKnu93zcPQM8nVCBSqZ7y6e-XaBINaTZAxxP6vxlA7wf8FV7e8lZ1htHZ95gGy89BssUOQ2Ij8hrbYa3Q2x6r8Wkaj~e8rAG3CHoUeR2PQerTiB6oj94IjhLlLi3N5ND309HZSqNyq~ADJSItx8IjvzF2i8w-dKwix~FXBLGpJQEwQerwlnjtZDQ4Xg8ZAhRhH99xfN~8IorSqCBKKkV5HqnbgmzGsbB2GpknnyKRFyu8Rh5SPHE7V-~7cACIepig__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">253</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">17</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359636"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f5%2f471%2f359636">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330167500005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (17)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F5%2F471%2F359636" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359636'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification and Evolution of the Silkworm Helitrons and their Contribution to Transcripts" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f471%2f359636">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

