

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2273,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/549/3865558","@type":"ScholarlyArticle","name":"Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs","datePublished":"2017-06-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx024","keywords":["meiotic recombination","recombination hotspot","genotyping by sequencing","DNA motifs","tomato"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/549/3865558","author":[{"name":"de Haas, Lars S","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"},{"name":"Koopmans, Roy","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"},{"name":"Lelivelt, Cilia L C","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"},{"name":"Ursem, Remco","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"},{"name":"Dirks, Rob","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"},{"name":"Velikkakam James, Geo","affiliation":"1 Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands","@type":"Person"}],"description":"Abstract.  Traditional plant breeding relies on meiotic recombination for mixing of parental alleles to create novel allele combinations. Detailed analysis of r","pageStart":"549","pageEnd":"558","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs" />
    <meta property="og:description" content="Abstract.  Traditional plant breeding relies on meiotic recombination for mixing of parental alleles to create novel allele combinations. Detailed analysis of r" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/549/3865558" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-06-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="de Haas, Lars S" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_author" content="Koopmans, Roy" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_author" content="Lelivelt, Cilia L C" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_author" content="Ursem, Remco" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_author" content="Dirks, Rob" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_author" content="Velikkakam James, Geo" /><meta name="citation_author_institution" content="Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands" /><meta name="citation_title" content="Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs" /><meta name="citation_firstpage" content="549" /><meta name="citation_lastpage" content="558" /><meta name="citation_doi" content="10.1093/dnares/dsx024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The molecular biology of meiosis in plants,;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2015;  citation_volume=66;  citation_pages=297-327; " /><meta name="citation_reference" content="citation_title=Intraspecific variation of recombination rate in maize,;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R103" /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics,;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=Variation in crossover rates across a 3-Mb contig of bread wheat (Triticum aestivum) reveals the presence of a meiotic recombination hotspot,;  citation_journal_title=Chromosoma;  citation_year=2011;  citation_volume=120;  citation_pages=185-98; " /><meta name="citation_reference" content="citation_title=The genomic landscape of meiotic crossovers and gene conversions in Arabidopsis thaliana,;  citation_journal_title=eLife;  citation_year=2013;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=The recombination landscape in Arabidopsis thaliana F2 populations,;  citation_journal_title=Heredity;  citation_year=2012;  citation_volume=108;  citation_pages=447-55; " /><meta name="citation_reference" content="citation_title=Fine-scale population recombination rates, hotspots, and correlates of recombination in the Medicago truncatula genome,;  citation_journal_title=Genome Biol. Evol;  citation_year=2012;  citation_volume=4;  citation_pages=726-37; " /><meta name="citation_reference" content="citation_title=Arabidopsis meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters,;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=1327-36; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of mono-, di-and trimethylation of histone H3 lysine 4 in Arabidopsis thaliana,;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R62" /><meta name="citation_reference" content="citation_title=DNA crossover motifs associated with epigenetic modifications delineate open chromatin regions in Arabidopsis,;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=2427-36; " /><meta name="citation_reference" content="citation_title=Meiotic recombination and genome evolution in plants,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2016;  citation_volume=30;  citation_pages=82-7; " /><meta name="citation_reference" content="citation_title=Great majority of recombination events in Arabidopsis are gene conversion events,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2012;  citation_volume=109;  citation_pages=20992-7; " /><meta name="citation_reference" content="citation_title=Detection of genomic variations and DNA polymorphisms and impact on analysis of meiotic recombination and genetic mapping,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2014;  citation_volume=111;  citation_pages=10007-12; " /><meta name="citation_reference" content="citation_title=Identifying the loci responsible for natural variation in fruit size and shape in tomato,;  citation_journal_title=Theor. Appl. Genet;  citation_year=1999;  citation_volume=99;  citation_pages=978-87; " /><meta name="citation_reference" content="citation_title=RFLP mapping of QTLs conferring cold tolerance during seed germination in an interspecific cross of tomato;  citation_journal_title=Mol. Breed;  citation_year=1998;  citation_volume=4;  citation_pages=519-29; " /><meta name="citation_reference" content="citation_title=Mapping of QTLs for lycopene and other fruit traits in a Lycopersicon esculentum × L. pimpinellifolium cross and comparison of QTLs across tomato species;  citation_journal_title=Mol. Breed;  citation_year=1999;  citation_volume=5;  citation_pages=283-99; " /><meta name="citation_reference" content="citation_title=A new genetic linkage map of tomato based on a Solanum lycopersicum × S. pimpinellifolium RIL population displaying locations of candidate pathogen response genes;  citation_journal_title=Genome Natl. Res. Counc. Can. Génome Cons. Natl. Rech. Can;  citation_year=2009;  citation_volume=52;  citation_pages=935-56; " /><meta name="citation_reference" content="citation_title=Mapping in the era of sequencing: high density genotyping and its application for mapping TYLCV resistance in Solanum pimpinellifolium,;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1152" /><meta name="citation_reference" content="citation_title=Recombination around the Tm2a and Mi resistance genes in different crosses of Lycopersicon peruvianum,;  citation_journal_title=Theor. Appl. Genet;  citation_year=1996;  citation_volume=92;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Genome mapping and molecular breeding of tomato,;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2007;  citation_volume=2007; " /><meta name="citation_reference" content="citation_title=Tomato chromosome 6: effect of alien chromosomal segments on recombinant frequencies,;  citation_journal_title=Genome;  citation_year=1996;  citation_volume=39;  citation_pages=485-91; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Structural differences in chromosomes distinguish species in the tomato clade,;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2010;  citation_volume=129;  citation_pages=24-34; " /><meta name="citation_reference" content="citation_title=Exploring genetic variation in the tomato (Solanum section Lycopersicon) clade by whole-genome sequencing,; citation_author=The 100 Tomato Genome Sequencing Consortium;  citation_journal_title=Plant J;  citation_year=2014;  citation_volume=80;  citation_pages=136-48; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data,;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism genotyping using Kompetitive Allele Specific PCR (KASP): overview of the technology and its application in crop improvement,;  citation_journal_title=Mol. Breed;  citation_year=2014;  citation_volume=33;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes,;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=MEME Suite: tools for motif discovery and searching,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=W202-8; " /><meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers,;  citation_journal_title=Proc. Int. Conf. Intell. Syst. Mol. Biol;  citation_year=1994;  citation_volume=2;  citation_pages=28-36; " /><meta name="citation_reference" content="citation_title=Motif Enrichment Analysis: a unified framework and an evaluation on ChIP data,;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=165" /><meta name="citation_reference" content="citation_title=Euchromatin and pericentromeric heterochromatin: comparative composition in the tomato genome;  citation_journal_title=Genetics;  citation_year=2006;  citation_volume=172;  citation_pages=2529-40; " /><meta name="citation_reference" content="citation_title=Large-scale gene function analysis with the PANTHER classification system,;  citation_journal_title=Nat. Protoc;  citation_year=2013;  citation_volume=8;  citation_pages=1551-66; " /><meta name="citation_reference" content="citation_title=Defining the full tomato NB-LRR resistance gene repertoire using genomic and cDNA RenSeq,;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=120" /><meta name="citation_reference" content="citation_title=A general modeling framework for genome ancestral origins in multiparental populations,;  citation_journal_title=Genetics;  citation_year=2014;  citation_volume=198;  citation_pages=87-101; " /><meta name="citation_reference" content="citation_title=Fluorescence in situ hybridization and optical mapping to correct scaffold arrangement in the tomato genome,;  citation_journal_title=G3 GenesGenomesGenetics;  citation_year=2014;  citation_volume=4;  citation_pages=1395-405; " /><meta name="citation_reference" content="citation_title=Involvement of a citrus meiotic recombination TTC-repeat motif in the formation of gross deletions generated by ionizing radiation and MULE activation,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=69" /><meta name="citation_reference" content="citation_title=Poly-dA:dT tracts form an in vivo nucleosomal turnstile,;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e110479" /><meta name="citation_reference" content="citation_title=User guide for mapping-by-sequencing in Arabidopsis,;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R61" /><meta name="citation_reference" content="citation_title=Dissecting meiotic recombination based on tetrad analysis by single-microspore sequencing in maize,;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=6648" /><meta name="citation_reference" content="citation_title=Fine-scale variation in meiotic recombination in Mimulus inferred from population shotgun sequencing,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2013;  citation_volume=110;  citation_pages=19478-82; " /><meta name="citation_reference" content="citation_title=Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants,;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=206;  citation_pages=1491-502; " /><meta name="citation_reference" content="citation_title=Recombination rate heterogeneity within Arabidopsis disease resistance genes,;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1006179" /><meta name="citation_reference" content="citation_title=Initiation of meiotic recombination: how and where? Conservation and specificities among eukaryotes,;  citation_journal_title=Annu. Rev. Genet;  citation_year=2013;  citation_volume=47;  citation_pages=563-99; " /><meta name="citation_reference" content="citation_title=Poly(dA:dT) Tracts: major determinants of nucleosome organization,;  citation_journal_title=Curr. Opin. Struct. Biol;  citation_year=2009;  citation_volume=19;  citation_pages=65-71; " /><meta name="citation_reference" content="citation_title=A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation,;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=144;  citation_pages=719-31; " /><meta name="citation_reference" content="citation_title=Unstable tandem repeats in promoters confer transcriptional evolvability,;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=1213-6; " /><meta name="citation_reference" content="citation_title=Variant histone H2A.Z is globally localized to the promoters of inactive yeast genes and regulates nucleosome positioning;  citation_journal_title=PLoS Biol;  citation_year=2005;  citation_volume=3; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/549/22295658/dsx024.pdf" /><meta name="description" content="Abstract.  Traditional plant breeding relies on meiotic recombination for mixing of parental alleles to create novel allele combinations. Detailed analysis of r" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/549/3865558" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/549/3865558" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/549/3865558");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx024';
    </script>
<title>
	Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3865558" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BctxPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJkZSBIYWFzLCBMYXJzIFMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSaWprIFp3YWFuIEJyZWVkaW5nIEIuVi4sIDQ3OTMgUlMgRmlqbmFhcnQsIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktvb3BtYW5zLCBSb3kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSaWprIFp3YWFuIEJyZWVkaW5nIEIuVi4sIDQ3OTMgUlMgRmlqbmFhcnQsIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlbGl2ZWx0LCBDaWxpYSBMIEMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSaWprIFp3YWFuIEJyZWVkaW5nIEIuVi4sIDQ3OTMgUlMgRmlqbmFhcnQsIFRoZSBOZXRoZXJsYW5kcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVyc2VtLCBSZW1jbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJpamsgWndhYW4gQnJlZWRpbmcgQi5WLiwgNDc5MyBSUyBGaWpuYWFydCwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGlya3MsIFJvYiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJpamsgWndhYW4gQnJlZWRpbmcgQi5WLiwgNDc5MyBSUyBGaWpuYWFydCwgVGhlIE5ldGhlcmxhbmRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmVsaWtrYWthbSBKYW1lcywgR2VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmlqayBad2FhbiBCcmVlZGluZyBCLlYuLCA0NzkzIFJTIEZpam5hYXJ0LCBUaGUgTmV0aGVybGFuZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikxvdy1jb3ZlcmFnZSByZXNlcXVlbmNpbmcgZGV0ZWN0cyBtZWlvdGljIHJlY29tYmluYXRpb24gcGF0dGVybiBhbmQgZmVhdHVyZXMgaW4gdG9tYXRvIFJJTHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjU1OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vbGVjdWxhciBiaW9sb2d5IG9mIG1laW9zaXMgaW4gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9Mjk3LTMyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cmFzcGVjaWZpYyB2YXJpYXRpb24gb2YgcmVjb21iaW5hdGlvbiByYXRlIGluIG1haXplLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMTAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEI3MyBtYWl6ZSBnZW5vbWU6IGNvbXBsZXhpdHksIGRpdmVyc2l0eSwgYW5kIGR5bmFtaWNzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMTEyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiBjcm9zc292ZXIgcmF0ZXMgYWNyb3NzIGEgMy1NYiBjb250aWcgb2YgYnJlYWQgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtKSByZXZlYWxzIHRoZSBwcmVzZW5jZSBvZiBhIG1laW90aWMgcmVjb21iaW5hdGlvbiBob3RzcG90LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0xODUtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWljIGxhbmRzY2FwZSBvZiBtZWlvdGljIGNyb3Nzb3ZlcnMgYW5kIGdlbmUgY29udmVyc2lvbnMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1lTGlmZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVjb21iaW5hdGlvbiBsYW5kc2NhcGUgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgRjIgcG9wdWxhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz00NDctNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUtc2NhbGUgcG9wdWxhdGlvbiByZWNvbWJpbmF0aW9uIHJhdGVzLCBob3RzcG90cywgYW5kIGNvcnJlbGF0ZXMgb2YgcmVjb21iaW5hdGlvbiBpbiB0aGUgTWVkaWNhZ28gdHJ1bmNhdHVsYSBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NzI2LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBtZWlvdGljIGNyb3Nzb3ZlciBob3Qgc3BvdHMgb3ZlcmxhcCB3aXRoIEgyQS5aIG51Y2xlb3NvbWVzIGF0IGdlbmUgcHJvbW90ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjctMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIG1vbm8tLCBkaS1hbmQgdHJpbWV0aHlsYXRpb24gb2YgaGlzdG9uZSBIMyBseXNpbmUgNCBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGNyb3Nzb3ZlciBtb3RpZnMgYXNzb2NpYXRlZCB3aXRoIGVwaWdlbmV0aWMgbW9kaWZpY2F0aW9ucyBkZWxpbmVhdGUgb3BlbiBjaHJvbWF0aW4gcmVnaW9ucyBpbiBBcmFiaWRvcHNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yNDI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWlvdGljIHJlY29tYmluYXRpb24gYW5kIGdlbm9tZSBldm9sdXRpb24gaW4gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTgyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyZWF0IG1ham9yaXR5IG9mIHJlY29tYmluYXRpb24gZXZlbnRzIGluIEFyYWJpZG9wc2lzIGFyZSBnZW5lIGNvbnZlcnNpb24gZXZlbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTIwOTkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBvZiBnZW5vbWljIHZhcmlhdGlvbnMgYW5kIEROQSBwb2x5bW9ycGhpc21zIGFuZCBpbXBhY3Qgb24gYW5hbHlzaXMgb2YgbWVpb3RpYyByZWNvbWJpbmF0aW9uIGFuZCBnZW5ldGljIG1hcHBpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9MTAwMDctMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZ5aW5nIHRoZSBsb2NpIHJlc3BvbnNpYmxlIGZvciBuYXR1cmFsIHZhcmlhdGlvbiBpbiBmcnVpdCBzaXplIGFuZCBzaGFwZSBpbiB0b21hdG8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz05NzgtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJGTFAgbWFwcGluZyBvZiBRVExzIGNvbmZlcnJpbmcgY29sZCB0b2xlcmFuY2UgZHVyaW5nIHNlZWQgZ2VybWluYXRpb24gaW4gYW4gaW50ZXJzcGVjaWZpYyBjcm9zcyBvZiB0b21hdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTUxOS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBvZiBRVExzIGZvciBseWNvcGVuZSBhbmQgb3RoZXIgZnJ1aXQgdHJhaXRzIGluIGEgTHljb3BlcnNpY29uIGVzY3VsZW50dW0gw5cgTC4gcGltcGluZWxsaWZvbGl1bSBjcm9zcyBhbmQgY29tcGFyaXNvbiBvZiBRVExzIGFjcm9zcyB0b21hdG8gc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjgzLTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHRvbWF0byBiYXNlZCBvbiBhIFNvbGFudW0gbHljb3BlcnNpY3VtIMOXIFMuIHBpbXBpbmVsbGlmb2xpdW0gUklMIHBvcHVsYXRpb24gZGlzcGxheWluZyBsb2NhdGlvbnMgb2YgY2FuZGlkYXRlIHBhdGhvZ2VuIHJlc3BvbnNlIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgTmF0bC4gUmVzLiBDb3VuYy4gQ2FuLiBHw6lub21lIENvbnMuIE5hdGwuIFJlY2guIENhbjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTkzNS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBpbiB0aGUgZXJhIG9mIHNlcXVlbmNpbmc6IGhpZ2ggZGVuc2l0eSBnZW5vdHlwaW5nIGFuZCBpdHMgYXBwbGljYXRpb24gZm9yIG1hcHBpbmcgVFlMQ1YgcmVzaXN0YW5jZSBpbiBTb2xhbnVtIHBpbXBpbmVsbGlmb2xpdW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb21iaW5hdGlvbiBhcm91bmQgdGhlIFRtMmEgYW5kIE1pIHJlc2lzdGFuY2UgZ2VuZXMgaW4gZGlmZmVyZW50IGNyb3NzZXMgb2YgTHljb3BlcnNpY29uIHBlcnV2aWFudW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0xMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG1hcHBpbmcgYW5kIG1vbGVjdWxhciBicmVlZGluZyBvZiB0b21hdG8sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBsYW50IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvbWF0byBjaHJvbW9zb21lIDY6IGVmZmVjdCBvZiBhbGllbiBjaHJvbW9zb21hbCBzZWdtZW50cyBvbiByZWNvbWJpbmFudCBmcmVxdWVuY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTQ4NS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRvbWF0byBnZW5vbWUgc2VxdWVuY2UgcHJvdmlkZXMgaW5zaWdodHMgaW50byBmbGVzaHkgZnJ1aXQgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9NjM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGRpZmZlcmVuY2VzIGluIGNocm9tb3NvbWVzIGRpc3Rpbmd1aXNoIHNwZWNpZXMgaW4gdGhlIHRvbWF0byBjbGFkZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldC4gR2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0yNC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9yaW5nIGdlbmV0aWMgdmFyaWF0aW9uIGluIHRoZSB0b21hdG8gKFNvbGFudW0gc2VjdGlvbiBMeWNvcGVyc2ljb24pIGNsYWRlIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nLDsgY2l0YXRpb25fYXV0aG9yPVRoZSAxMDAgVG9tYXRvIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xMzYtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93c+KAk1doZWVsZXIgdHJhbnNmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZyYW1ld29yayBmb3IgdmFyaWF0aW9uIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz00OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGdlbm90eXBpbmcgdXNpbmcgS29tcGV0aXRpdmUgQWxsZWxlIFNwZWNpZmljIFBDUiAoS0FTUCk6IG92ZXJ2aWV3IG9mIHRoZSB0ZWNobm9sb2d5IGFuZCBpdHMgYXBwbGljYXRpb24gaW4gY3JvcCBpbXByb3ZlbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZXJzYXRpbGUgYW5kIG9wZW4gc29mdHdhcmUgZm9yIGNvbXBhcmluZyBsYXJnZSBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FTUUgU3VpdGU6IHRvb2xzIGZvciBtb3RpZiBkaXNjb3ZlcnkgYW5kIHNlYXJjaGluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9VzIwMi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaXR0aW5nIGEgbWl4dHVyZSBtb2RlbCBieSBleHBlY3RhdGlvbiBtYXhpbWl6YXRpb24gdG8gZGlzY292ZXIgbW90aWZzIGluIGJpb3BvbHltZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gSW50LiBDb25mLiBJbnRlbGwuIFN5c3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjgtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vdGlmIEVucmljaG1lbnQgQW5hbHlzaXM6IGEgdW5pZmllZCBmcmFtZXdvcmsgYW5kIGFuIGV2YWx1YXRpb24gb24gQ2hJUCBkYXRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXVjaHJvbWF0aW4gYW5kIHBlcmljZW50cm9tZXJpYyBoZXRlcm9jaHJvbWF0aW46IGNvbXBhcmF0aXZlIGNvbXBvc2l0aW9uIGluIHRoZSB0b21hdG8gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzI7ICBjaXRhdGlvbl9wYWdlcz0yNTI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBnZW5lIGZ1bmN0aW9uIGFuYWx5c2lzIHdpdGggdGhlIFBBTlRIRVIgY2xhc3NpZmljYXRpb24gc3lzdGVtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE1NTEtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmluaW5nIHRoZSBmdWxsIHRvbWF0byBOQi1MUlIgcmVzaXN0YW5jZSBnZW5lIHJlcGVydG9pcmUgdXNpbmcgZ2Vub21pYyBhbmQgY0ROQSBSZW5TZXEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtb2RlbGluZyBmcmFtZXdvcmsgZm9yIGdlbm9tZSBhbmNlc3RyYWwgb3JpZ2lucyBpbiBtdWx0aXBhcmVudGFsIHBvcHVsYXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTk4OyAgY2l0YXRpb25fcGFnZXM9ODctMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIGFuZCBvcHRpY2FsIG1hcHBpbmcgdG8gY29ycmVjdCBzY2FmZm9sZCBhcnJhbmdlbWVudCBpbiB0aGUgdG9tYXRvIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIEdlbmVzR2Vub21lc0dlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMzk1LTQwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52b2x2ZW1lbnQgb2YgYSBjaXRydXMgbWVpb3RpYyByZWNvbWJpbmF0aW9uIFRUQy1yZXBlYXQgbW90aWYgaW4gdGhlIGZvcm1hdGlvbiBvZiBncm9zcyBkZWxldGlvbnMgZ2VuZXJhdGVkIGJ5IGlvbml6aW5nIHJhZGlhdGlvbiBhbmQgTVVMRSBhY3RpdmF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5LWRBOmRUIHRyYWN0cyBmb3JtIGFuIGluIHZpdm8gbnVjbGVvc29tYWwgdHVybnN0aWxlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTA0NzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2VyIGd1aWRlIGZvciBtYXBwaW5nLWJ5LXNlcXVlbmNpbmcgaW4gQXJhYmlkb3BzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPVI2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3RpbmcgbWVpb3RpYyByZWNvbWJpbmF0aW9uIGJhc2VkIG9uIHRldHJhZCBhbmFseXNpcyBieSBzaW5nbGUtbWljcm9zcG9yZSBzZXF1ZW5jaW5nIGluIG1haXplLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTY2NDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lLXNjYWxlIHZhcmlhdGlvbiBpbiBtZWlvdGljIHJlY29tYmluYXRpb24gaW4gTWltdWx1cyBpbmZlcnJlZCBmcm9tIHBvcHVsYXRpb24gc2hvdGd1biBzZXF1ZW5jaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTE5NDc4LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlbHkgZGlzdHJpYnV0ZWQgaG90IGFuZCBjb2xkIHNwb3RzIGluIG1laW90aWMgcmVjb21iaW5hdGlvbiBhcyBzaG93biBieSB0aGUgc2VxdWVuY2luZyBvZiByaWNlIEYyIHBsYW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjA2OyAgY2l0YXRpb25fcGFnZXM9MTQ5MS01MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29tYmluYXRpb24gcmF0ZSBoZXRlcm9nZW5laXR5IHdpdGhpbiBBcmFiaWRvcHNpcyBkaXNlYXNlIHJlc2lzdGFuY2UgZ2VuZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDYxNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWF0aW9uIG9mIG1laW90aWMgcmVjb21iaW5hdGlvbjogaG93IGFuZCB3aGVyZT8gQ29uc2VydmF0aW9uIGFuZCBzcGVjaWZpY2l0aWVzIGFtb25nIGV1a2FyeW90ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NTYzLTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5KGRBOmRUKSBUcmFjdHM6IG1ham9yIGRldGVybWluYW50cyBvZiBudWNsZW9zb21lIG9yZ2FuaXphdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFN0cnVjdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTY1LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZXJhcmNoaWNhbCBjb21iaW5hdGlvbiBvZiBmYWN0b3JzIHNoYXBlcyB0aGUgZ2Vub21lLXdpZGUgdG9wb2dyYXBoeSBvZiB5ZWFzdCBtZWlvdGljIHJlY29tYmluYXRpb24gaW5pdGlhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTQ0OyAgY2l0YXRpb25fcGFnZXM9NzE5LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnN0YWJsZSB0YW5kZW0gcmVwZWF0cyBpbiBwcm9tb3RlcnMgY29uZmVyIHRyYW5zY3JpcHRpb25hbCBldm9sdmFiaWxpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTEyMTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWFudCBoaXN0b25lIEgyQS5aIGlzIGdsb2JhbGx5IGxvY2FsaXplZCB0byB0aGUgcHJvbW90ZXJzIG9mIGluYWN0aXZlIHllYXN0IGdlbmVzIGFuZCByZWd1bGF0ZXMgbnVjbGVvc29tZSBwb3NpdGlvbmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC82LzU0OS8yMjI5NTY1OC9kc3gwMjQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRyYWRpdGlvbmFsIHBsYW50IGJyZWVkaW5nIHJlbGllcyBvbiBtZWlvdGljIHJlY29tYmluYXRpb24gZm9yIG1peGluZyBvZiBwYXJlbnRhbCBhbGxlbGVzIHRvIGNyZWF0ZSBub3ZlbCBhbGxlbGUgY29tYmluYXRpb25zLiBEZXRhaWxlZCBhbmFseXNpcyBvZiByIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODY1NTU4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZH90cJobXecIrkhEzK80wxfvuPtt" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f549%2f3865558" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f549%2f3865558" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104085989">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104085989">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104085991">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104085991">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104085996">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104085996">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086022">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086022">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086045">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086045">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086052">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086052">Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086054">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086054">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086062">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086062">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/559/3868561">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lars S de Haas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lars+S+de+Haas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=de Haas L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;de Haas L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lars S de Haas</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roy Koopmans</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roy+Koopmans">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koopmans R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koopmans R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Roy Koopmans</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cilia L C Lelivelt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Cilia+L+C+Lelivelt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lelivelt C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lelivelt C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Cilia L C Lelivelt</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Remco Ursem</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Remco+Ursem">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ursem R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ursem R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Remco Ursem</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rob Dirks</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rob+Dirks">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dirks R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dirks R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rob Dirks</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Geo Velikkakam James</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Rijk Zwaan Breeding B.V., 4793 RS Fijnaart, The Netherlands</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx024-cor1" data-open="dsx024-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Email: <a href="mailto:g.james@rijkzwaan.nl" target="_blank">g.james@rijkzwaan.nl</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Geo+Velikkakam+James">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Velikkakam James G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Velikkakam James G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Geo Velikkakam James</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx024-FM2"><div class="fn"><div class="footnote-content">Present address: Department of Plant Production, Faculty of Bioscience Engineering, Ghent University, Coupure links 653, 9000 Gent, Belgium</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx024-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 549–558, <a href='https://doi.org/10.1093/dnares/dsx024'>https://doi.org/10.1093/dnares/dsx024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 June 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 May 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3865558" href="/dnaresearch/article-pdf/24/6/549/22295658/dsx024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Lars S de Haas, Roy Koopmans, Cilia L C Lelivelt, Remco Ursem, Rob Dirks, Geo Velikkakam James;  Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 549–558, <a href='https://doi.org/10.1093/dnares/dsx024'>https://doi.org/10.1093/dnares/dsx024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3865558?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Low-coverage%20resequencing%20detects%20meiotic%20recombination%20pattern%20and%20features%20in%20tomato%20RILs&amp;publicationDate=2017-06-09&amp;volumeNum=24&amp;issueNum=6&amp;author=de%20Haas%2C%20Lars%20S%3B%20Koopmans%2C%20Roy&amp;startPage=549&amp;endPage=558&amp;contentId=10.1093%2Fdnares%2Fdsx024&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f549%2f3865558" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104085989 id="104085989" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Traditional plant breeding relies on meiotic recombination for mixing of parental alleles to create novel allele combinations. Detailed analysis of recombination patterns in model organisms shows that recombination is tightly regulated within the genome, but frequencies vary extensively along chromosomes. Despite being a model organism for fruit developmental studies, high-resolution recombination patterns are lacking in tomato. In this study, we developed a novel methodology to use low-coverage resequencing to identify genome-wide recombination patterns and applied this methodology on 60 tomato Recombinant Inbred Lines (RILs). Our methodology identifies polymorphic markers from the low-coverage resequencing population data and utilizes the same data to locate the recombination breakpoints in individuals by using a variable sliding window. We identified 1,445 recombination sites comprising 112 recombination prone regions enriched for AT-rich DNA motifs. Furthermore, the recombination prone regions in tomato preferably occurred in gene promoters over intergenic regions, an observation consistent with <em>Arabidopsis thaliana</em>, <em>Zea mays</em> and <em>Mimulus guttatus</em>. Overall, our cost effective method and findings enhance the understanding of meiotic recombination in tomato and suggest evolutionarily conserved recombination associated genomic features.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;meiotic recombination&quot;">meiotic recombination</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;recombination hotspot&quot;">recombination hotspot</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genotyping by sequencing&quot;">genotyping by sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA motifs&quot;">DNA motifs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="tomato">tomato</a></div>                            <h2 scrollto-destination=104085991 id="104085991" class="section-title" >1. Introduction</h2>
<p>Meiotic recombination plays a vital role in sexual reproduction by mixing parental alleles to create novel allele combinations. In general, both genetic mapping and traditional breeding depends upon the distribution of recombination along chromosomes. In plants, the meiotic recombination rate is tightly regulated within the genome, but varies between species.<a reveal-id="dsx024-B1" data-open="dsx024-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Furthermore, the distribution of recombination events is not random along the chromosome. Regions that are preferred and suppressed for recombination are known as recombination hotspots and coldspots, respectively. Various studies have shown that recombination prone regions vary in size and magnitude between species (<em>Zea mays</em>,<a reveal-id="dsx024-B2" data-open="dsx024-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx024-B3" data-open="dsx024-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><em>Triticum aestivum</em>,<a reveal-id="dsx024-B4" data-open="dsx024-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>Arabidopsis thaliana</em><a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B6" data-open="dsx024-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and <em>Medicago trancatula</em><a reveal-id="dsx024-B7" data-open="dsx024-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>). Extended knowledge about genome-wide recombination patterns in a species would not only improve the understanding of recombination preferences but also accelerate fine mapping of genetic traits and breeding design of the crop. Moreover, high-resolution recombination patterns provide the opportunity to study recombination associated genomic features, thereby the possibility to modulate recombination to accelerate breeding.</p><p>Advancement in sequencing technology has not only enabled rapid generation of draft genome assemblies for individual species but also identification of genome-wide ultra-dense markers in a population. Hence, whole genome resequencing has also enabled high resolution recombination pattern identification. Recent studies have produced base pair resolution recombination patterns in <em>Arabidopsis thaliana</em> and shown that the distribution of recombination is influenced by chromatin structure, histone variants and DNA methylation, among other factors.<a reveal-id="dsx024-B8" data-open="dsx024-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx024-B9" data-open="dsx024-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Further analysis using next generation resequencing data revealed poly-A, CCN-repeat and CTT-repeat motifs associated with recombination prone regions in <em>Arabidopsis thaliana.</em><a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B8" data-open="dsx024-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx024-B10" data-open="dsx024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The first motif was associated with nucleosome depleted regions, whereas CCN- and CTT-repeat motifs were enriched within genes.<a reveal-id="dsx024-B10" data-open="dsx024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx024-B11" data-open="dsx024-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> However, lack of stringent resequencing data analysis led to false positive markers that inflate recombination rate.<a reveal-id="dsx024-B12" data-open="dsx024-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Therefore, it is of the utmost importance to select an accurate marker-set while applying resequencing data for recombination identification.<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B13" data-open="dsx024-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> This caveat is even bigger in crops with a large genome, or the ones possessing a high repeat content. Besides, an incomplete assembled reference genome representing only one haploid genome may hamper proper evaluation of recombination rates. Therefore, in crop plants, it is crucial to have a cost effective methodology that can identify genome-wide recombination patterns using low-coverage resequencing data. This is desired to reduce sequencing efforts to a minimum and at the same time maintain a low false-positive identification rate. In this study, we describe a novel methodology using meticulous filtering steps to conservatively select polymorphic markers from low-coverage resequencing population data to identify recombination sites. We applied our method to identify genome-wide recombination pattern on resequencing data from 60 tomato recombinant inbred lines (RILs) from an interspecific cross between <em>Solanum lycopersicum</em> cv. Moneymaker and <em>Solanum pimpinellifolium</em> CGN 15528.</p><p>In crop plants, tomato is a model species for fruit developmental studies and interspecific crosses are primarily used for genetic mapping. <em>S. pimpinellifolium</em>, a wild but closely related species of <em>S. lycopersicum</em>, is a source for introgression breeding including various disease resistance, abiotic stress tolerance, and fruit quality improvements.<a reveal-id="dsx024-B14 dsx024-B15 dsx024-B16 dsx024-B17 dsx024-B18" data-open="dsx024-B14 dsx024-B15 dsx024-B16 dsx024-B17 dsx024-B18" class="link link-ref link-reveal xref-bibr"><sup>14–18</sup></a> However, these interspecific crosses usually cause recombination suppression within the introgression region and require extensive additional breeding efforts for later improvements. For example, the tomato chromosomes 6 and 9 are known to contain introgression fragments for nematode and TMV resistance and conceive low or no recombination.<a reveal-id="dsx024-B19 dsx024-B20 dsx024-B21" data-open="dsx024-B19 dsx024-B20 dsx024-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a> Despite the utility of local recombination pattern for an informed breeding design, there is no availability of genome-wide high-resolution interspecific recombination pattern in tomato. This is mainly due to the complexity and lack of methodology to apply resequencing to a relatively large genome for genome-wide recombination pattern identification. Like the majority of crop species, the tomato genome sequence is complex and incomplete. Current tomato reference genome has assembled 760 mega base pair (Mbp) from an estimated genome size of 900 Mbp.<sup>22</sup> This hinders the use of a resequencing approach for genome-wide recombination studies. Moreover, structural variations are reported between <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em>, which impede the analysis of interspecific genomes even further.<a reveal-id="dsx024-B23" data-open="dsx024-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>In this study, we applied our novel methodology to identify a reliable set of polymorphic markers from the resequencing data from an interspecific RIL population. By using allele segregation in the population and resequencing data from one of the parents, we inferred the genotype of the second parent and estimated haplotypes for 60 RILs to identify recombination sites. We identified high-resolution genome-wide recombination patterns and identified several DNA motifs enriched at recombination prone regions, including two AT-rich motifs, which have not been reported previously in relation with recombination in plants. With this study, we have identified high-resolution recombination prone regions in tomato and genomic features associated with it.</p>                            <h2 scrollto-destination=104085996 id="104085996" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104085997 id="104085997" class="section-title" >2.1. Data used and variant calling</h3>
<p>We used low-coverage (∼6.3×) Illumina HiSeq 2000 resequencing data of 60 RILs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s1_dsx024.doc?Expires=2147483647&Signature=P6ILI5PE4Vehg1ttVp9BD~M2S31yKps~0mSyoBG3QJN1FGSKvV-uFv8sDVGK0POISlCRitxvvwXUa0Q5DagjII2ae6bwjBjHyvSJDDHx-~nBkORIstE6e4KkzOUxi7HB-5BdUYMZgp5LbZBhcMtBkFmTSn0UyCIMeAe0ydF53QZAm~EK0MZv6F0aQOY8ciYaImHpqIKWsCMBemZWspMZ70J5MB9K7YE9wJVd97OIsSK-CB~ezIXONqvITNWHQ8T3RYFf40aWmhIlXKAzCZ0kV2yDYOwGkxdtjVT~Ni1b8m30ZQJ-MyyxWhXLG-7aJPXeNACht52cnBRS1Se1-5aOgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). This F6 population was generated by an interspecific cross between <em>S. lycopersicum</em> (cv. Moneymaker) and <em>S. pimpinellifolium</em> (CGN 15528, also referred to as CGN 14498).<sup>24</sup> Resequencing data from one of the parents, <em>S. lycopersicum</em>, was publicly available with an average genome coverage of ∼38×, whereas the sequencing data for <em>S. pimpinellifolium</em> was unavailable. Both RILs and <em>S. lycopersicum</em> were part of the 150 tomato Genome Resequencing Project (<a class="link link-uri" href="http://www.tomatogenome.net/" target="_blank">http://www.tomatogenome.net/</a>).<a reveal-id="dsx024-B24" data-open="dsx024-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> We aligned the short reads from 60 RILs and <em>S. lycopersicum</em> separately to <em>S. lycopersicum</em> cv. Heinz 1706 reference genome (version Sl2.40, <a class="link link-uri" href="http://www.solgenomics.net" target="_blank">http://www.solgenomics.net</a>) using a modified version of BWA (based on version 0.5.9.) default settings.<a reveal-id="dsx024-B25" data-open="dsx024-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> For each sample read groups were corrected and duplicate reads were removed using PICARD tools (version 1.107, <a class="link link-uri" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>) with default settings. We improved the alignment by local realignment using GATK-lite (version 2.39) and these improved aligned resequencing data were subjected to variant calling by the UnifiedGenotyper of GATK-lite default settings.<a reveal-id="dsx024-B26" data-open="dsx024-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=104085999 id="104085999" class="section-title" >2.2. Population based marker-set identification</h3>
<p>After variant calling, we strictly filtered single nucleotide variants (SNPs). To consider a genomic position a genuine marker, in contrast to sequencing and read alignment artefacts, characteristics of a SNP were needed to meet six criteria. There parameters were chosen based on the expected characteristics of the used RIL population.</p><p>By making sure that the same marker is reported in multiple plants, we made use of the recurrence of SNP information and took advantage of population’s properties. Although it was expected to have approximately 30 homozygous alternative allele calls on a position, we incorporated a substantial margin to account for missing calls due to the low coverage data used and variation in the uniformity of the expected segregation pattern, considering the limited size of the population. For this study, at least 10 out of 60 RILs were required to report a homozygous alternative call on a single position.</p><p>Since the population is expected to be mostly homozygous, heterozygous SNPs have a higher chance of being either sequencing or read alignment artefacts. At the same time, it is harder to call heterozygous genotypes than it is to call homozygous genotypes in low coverage sequencing data. Therefore, we also required that the ratio of homozygous calls to the total number of calls had to be above 0.8. This way we avoided the selection of positions that were susceptible to show heterozygous genotypes throughout the majority of the population, which might for example have been caused by misaligned reads or paralogous sequences in the genome.</p><p>Meanwhile, the ratio of alternative calls to the total number of homozygous can assure the selection of markers that are evenly segregating through the population. This ratio is expected to be equal between alternative and reference calls. However, this ratio is heavily influenced by the number of plants having, or missing, a genotype call on the respective position, which is expected to vary due to sequencing coverage. So, considering the limited population size and the amount of missing genotype calls, the ratio of alternative homozygous calls and total number of homozygous calls was set to be between 0.25 and 0.75. This was especially important in this particular study, because we were not only constructing a marker-set but also inferring the genotypes of the second parent. By confirming that both reference and alternative allele were segregating within the population, we confirmed that the parents have opposite genotypes on this position.</p><p>To correct for individual sequencing artefacts, only biallelic SNPs with a coverage of four or more for at least one of the alleles were considered. In total we yielded 4,463,846 markers after filtering and these markers were used for genotyping. A correlation between the abundance of available markers and the average coverage of a RIL was expected. The number of available markers and coverage per plant were visualized to estimate the strength of the expected correlation.</p>                            <h3 scrollto-destination=104086005 id="104086005" class="section-title" >2.3. Inferring markers of second parent</h3>
<p>Since resequencing data for <em>S. pimpinellifolium</em> was not available, we inferred the <em>S. pimpinellifolium</em> genotypes by assigning non-<em>S. lycopersicum</em> allele to previously identified population based markers. Markers with reference base calls reported in the <em>S. lycopersicum</em> accession were considered to have alternative base in <em>S. pimpinellifolium</em>. Out of 4,463,846 markers, 75,526 were inferred to be alternative base in <em>S. lycopersicum</em> and consequently reference base in <em>S. pimpinellifolium</em>. <em>S. lycopersicum</em> was expected to have less alternative base calls, because it is more closely related to the used reference genome as opposed to <em>S. pimpinellifolium</em><em>.</em><a reveal-id="dsx024-B24" data-open="dsx024-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Using 130 Kompetitive Allele Specific PCR (KASP) assays,<a reveal-id="dsx024-B27" data-open="dsx024-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> we validated the inferred genotype of <em>S. pimpinellifolium</em> and estimated its accuracy.</p>                            <h3 scrollto-destination=104086007 id="104086007" class="section-title" >2.4. Determining haplotype blocks and estimating recombination landscape</h3>
<p>A variable sliding window approach was used to impute the haplotype blocks of RILs. Imputation of haplotypes enabled us to resolve conflicting genotypes as a result of erroneous parental markers, sequencing errors and read alignment artefacts. The used variable window was adjusted in size to the local density of available markers, with at least nine markers over a genomic distance of at least 10 kbp. Scoring markers in a variable window allowed the collective use of markers, correcting faulty individual markers and genotypes.</p><p>The use of a variable window also contributed to cope with missing markers due to low-coverage resequencing, without compromising the yielded resolution. The score of a window was calculated by adding up all genotypes (+1 for <em>S. lycopersicum</em>, 0 for heterozygous, −1 for <em>S. pimpinellifolium</em>) and dividing it by the number of markers in the window. When 50% or more of the markers in a window were heterozygous, the window was given an intermediate score of 0 (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s1_dsx024.pdf?Expires=2147483647&Signature=03UKJGngcFcIKvKIoyy8CHRSKa3hhxDdZd7E-kzNgMVzD2hIzN3z61Hqp2OxfUbfw61J~sItUwVewHSkRq4okPVxFQ0YLn63DRIwhKQnfotk2XCU2RDo0iUsG6BF8XLEHljO3JcVOsx6OQBAzkXJkefzRHaSyyWVd5wXDDlVOHEpmHD7OUluhm2AjM-oWke7rtYMxjgygMZ0keJEh~q1UFm34qBzBRvYsOZGE6Ne9EVuyPlsIKuzb7tCMjSP2hNmRa~kxkD2MohDZ1oeEgeaGJ7YYdxSSJAqEh172AqNXNAUEwfNl5yYaWmv7Z-HGLpSma-qUMGdZsp1m1ho8RtGAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>: second panel).</p><p>Each time the score of a forthcoming window (w) passed one of the thresholds <em>w</em> &gt; 0.25, −0.25 &lt; = <em>w</em> &lt; = 0.25 or <em>w</em> &lt; −0.25 in respect to the previous window, the markers of both windows were merged and subjected to a second analysis. This was done to avoid false positive recombination events due to fluctuation in scores, caused by sequencing and read alignment artefacts. Again a variable window approach was used, but over a greater distance of at least 250kbp with 50 markers both upwards and downwards of the presumed recombination region. This confirmed whether variation in scores was persistent, thus must have been caused by a recombination event. Finally, the inner unambiguous markers with contrasting parental genotypes were selected as borders of the recombination event.</p><p>When two or more haplotype changes were detected within 250kbp, this was considered to be most likely the result of fluctuation in scores caused by artefacts. In such cases, all involved events were merged and processed as a single event, which may or may not have resulted in the detection of a genuine recombination event. Our tool reports all the recombination sites per individual as well as the summarised recombination pattern per chromosome (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s1_dsx024.pdf?Expires=2147483647&Signature=03UKJGngcFcIKvKIoyy8CHRSKa3hhxDdZd7E-kzNgMVzD2hIzN3z61Hqp2OxfUbfw61J~sItUwVewHSkRq4okPVxFQ0YLn63DRIwhKQnfotk2XCU2RDo0iUsG6BF8XLEHljO3JcVOsx6OQBAzkXJkefzRHaSyyWVd5wXDDlVOHEpmHD7OUluhm2AjM-oWke7rtYMxjgygMZ0keJEh~q1UFm34qBzBRvYsOZGE6Ne9EVuyPlsIKuzb7tCMjSP2hNmRa~kxkD2MohDZ1oeEgeaGJ7YYdxSSJAqEh172AqNXNAUEwfNl5yYaWmv7Z-HGLpSma-qUMGdZsp1m1ho8RtGAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>, first panel). This intermediate score and recombination landscape were visualised to ease manual inspection and optimization of the parameters described in “Population based marker-set identification” (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s1_dsx024.pdf?Expires=2147483647&Signature=03UKJGngcFcIKvKIoyy8CHRSKa3hhxDdZd7E-kzNgMVzD2hIzN3z61Hqp2OxfUbfw61J~sItUwVewHSkRq4okPVxFQ0YLn63DRIwhKQnfotk2XCU2RDo0iUsG6BF8XLEHljO3JcVOsx6OQBAzkXJkefzRHaSyyWVd5wXDDlVOHEpmHD7OUluhm2AjM-oWke7rtYMxjgygMZ0keJEh~q1UFm34qBzBRvYsOZGE6Ne9EVuyPlsIKuzb7tCMjSP2hNmRa~kxkD2MohDZ1oeEgeaGJ7YYdxSSJAqEh172AqNXNAUEwfNl5yYaWmv7Z-HGLpSma-qUMGdZsp1m1ho8RtGAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). In the population of 60 RILs, we calculated the average number of recombination per chromosome by considering all marker positions within the chromosome conceived with at least one recombination. The background recombination rate per chromosome was calculated by adding average recombination rate with one standard deviation. Regions having a recombination rate above the background rate were considered as recombination prone regions. The longest region of each chromosome without any recombination over all 60 RILs was considered recombination deprived region.</p>                            <h3 scrollto-destination=104086012 id="104086012" class="section-title" >2.5. False-positive recombination events caused by the used reference genome assembly</h3>
<p>Two versions (Sl2.40 and Sl2.50) of the <em>S. lycopersicum</em> cv. Heinz 1706 reference genome assembly were aligned to detect large inversions and relocations between the two reference genomes using MUMmer3’s NUCmer.<a reveal-id="dsx024-B28" data-open="dsx024-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> For this, a minimum cluster size of 1kbp and all anchor matches regardless of their uniqueness were used. Large scale differences in alignment between these two reference assemblies indicated inconsistencies between assemblies. These differences were used to eliminate recombination prone regions overlapping with these regions to prevent false positives caused by the used assembly (version Sl2.40).</p>                            <h3 scrollto-destination=104086014 id="104086014" class="section-title" >2.6. Confirmation of inferred parental genotype and RIL haplotypes</h3>
<p>In order to confirm both the inferred <em>S. pimpinellifolium</em> genotype and individual RIL haplotypes, we ordered the seeds of 60 RILs together with their corresponding parental lines from the 150 Tomato Project. The seeds were sown on wet tissue paper at 25 °C with a continuous exposure to light for 12 days. Due to poor seed germination, we proceeded with 55 RIL plants. Whole seedlings were harvested and outsourced to LGC Teddington, London, for genomic DNA extraction and KASP genotyping in duplo. Genotyping was done using LGC’s KASP Master Mix, containing two universal cassettes (FAM and HEX), ROX<sup>tm</sup> passive reference dye, Taq polymerase, free nucleotides and MgCl<sub>2</sub> in an optimized buffer solution. One hundred and fifty SNPs were selected for KASP assay-design of which 90 SNPs around recombination prone regions, 20 random, 24 in removed regions and 16 to confirm suspected false positives were indeed caused by sequencing and read alignment artefacts (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s2_dsx024.xlsx?Expires=2147483647&Signature=2mZbRb-Y1HdGLq-9wLpBWvCYXvE71MjfSmbHcsSpxy~CwEWtX6gSNEoPq7ljg79cbaxtOBTcwiEBaAVOGRdTcsPd5I-VW-4uQQtHDMdy-VynIyOnvPtxbvZKy4wvUuzVfNsdtruaiW6Jy6V~8IpnSuZ2Y2a3y35CBlaBhabNugHltbeWIXnitwnCsG8JdoUjXpN0cBwcUvzzt6Y4LfLxrIdilzm6ucfEb3~iBWKk-ypzFhlgYF9DuEsGsGcT2nDT7OvWcFuHzumxtMC0Ve9B80VqhmUXifopu1DS9aiIYUW8WO3dRPZxQ-xIitX1p0I9FWpYPs9uw96ygg6mebGiww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Selected SNPs were subjected to LGC’s Primer Picker software for an <em>in silico</em> examination of possible drop-outs prior to shipping.</p><p>We used an ad-hoc script to compare LGC’s KASP genotypes with our predicted haplotype blocks for an exact match. Whenever one or both of the two KASP replicate assays had failed, or replicate assays did not match, the concerned assay was discarded in order to eliminate errors introduced by the used method of confirmation. This allowed us to validate polymorphic marker identification, as well as inferred genotypes of <em>S. pimpinellifolium</em> and our approach to estimate haplotype blocks for RILs.</p>                            <h3 scrollto-destination=104086017 id="104086017" class="section-title" >2.7. Discovery of DNA motifs enriched with recombination prone regions</h3>
<p>Identified recombination prone regions were assessed for over-representation of sequence motifs. Associated sequences were selected by our pipeline and subjected to <em>de novo</em> motif discovery by the MEME Suite.<a reveal-id="dsx024-B29" data-open="dsx024-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> To include sequence motifs regulating recombination upstream and downstream of the processed recombination prone regions, sets with various sized flanking regions (0, 1, 2.5 and 5kbp) were created and subjected to motif discovery by MEME.<a reveal-id="dsx024-B30" data-open="dsx024-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The resulting collections of sequences had total sizes of 278kbp, 503kbp, 839kbp and 1,400kbp, respectively.</p><p>MEME was used for <em>de novo</em> 8–15bp DNA sequence motif discovery at above-mentioned collection of sequences using a Zero or One Occurrence Per Sequence model. The maximum allowed dataset size was increased to enable MEME to process the data. A control set of 1,000 random sequence collections was constructed, each containing 112 random <em>S. lycopersicum</em> cv. Heinz 1706 (reference version 2.40) sequences of lengths equal to the identified recombination prone regions together with their respective flanking regions. <em>De novo</em> discovered sequence motifs were analysed using default settings of AME<a reveal-id="dsx024-B31" data-open="dsx024-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and the control set, to determine whether these motifs were over-enriched in recombination prone region compared with the random background.</p>                            <h3 scrollto-destination=104086020 id="104086020" class="section-title" >2.8. Identification of genomic features associated with recombination prone regions</h3>
<p>We annotated each base pair of recombination prone regions to identify genomic features, including 5′-UTR, gene start, gene, 3′-UTR, gene end and intergenic region using the ITAG2.3 annotation (<a class="link link-uri" href="http://www.solgenomics.net" target="_blank">http://www.solgenomics.net</a>). We defined promoter and gene start features as 3kbp reverse and 200bp forward from the transcription start site, respectively. Similarly, gene end was defined as 200bp before transcription end site. To assess whether the distance of recombination events to genes is significantly lower compared with randomly sampled positions, the genomic distance between all 112 regions and their nearest gene was mapped using the ITAG version 2.3 annotation. Two sets of 1000 random genomic regions were created with and without pericentric heterochromatin. These pericentric heterochromatins were discarded in one of the two sets to exclude influence due to their low gene density.<a reveal-id="dsx024-B32" data-open="dsx024-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Differences were visualized, showing the average of all randomly sampled positions in 1kbp windows, their corresponding standard deviation and observed distance to genes. We listed the annotated genes within the recombination prone regions for gene ontology (GO) enrichment analysis using the PANTHER classification system (<a class="link link-uri" href="http://www.geneontology.org/" target="_blank">http://www.geneontology.org/ (12 October 2016, date last accessed</a>), which is up to date with GO annotations.<a reveal-id="dsx024-B33" data-open="dsx024-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> We also compared genes overlapping with recombination prone regions to nucleotide-binding site leucine-rich repeat (NB-LRR) resistance genes in tomato.<a reveal-id="dsx024-B34" data-open="dsx024-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h2 scrollto-destination=104086022 id="104086022" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086023 id="104086023" class="section-title" >3.1. Identification of population based marker-set from low-coverage resequencing data</h3>
<p>Genotyping by resequencing allows identification of recombination break points at a high-resolution, but non-allelic structural variations between parental lines with a high homology in sequence can lead to false-positive markers, especially when using short-read resequencing data.<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B13" data-open="dsx024-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Due to independent assortment of these loci, genotypes of these false positive markers will be random. Additionally, genotyping bias will increase if one of the parental lines is more closely related to the used reference genome. Together, these biases will produce false-positive markers and result in exaggerated recombination rates. Thus, it is crucial to filter out false-positive markers when estimating the recombination landscape, particularly while using low-coverage short-read resequencing data. We illustrated a novel methodology to conservatively select markers in the context to identify recombination sites by implementing fundamental population genetics principals with modifications to incorporate regions with low sequencing coverage.</p><p>Short reads from individual RILs were aligned to tomato reference genome to identify SNPs using GATK-lite pipeline. To increase the specificity of SNP identification, we applied three rounds of filters, addressing the independent allele segregation within population, fixation or near fixation of alleles and sufficient sequencing depth at SNP position in individuals as well as within the population (see section Materials and methods). As shown in <a reveal-id="dsx024-F1" data-open="dsx024-F1" class="link link-reveal link-table xref-fig">Fig. 1A–C</a>, the characteristics of most markers corresponded to what was expected from a RIL population. The majority of markers were predominantly homozygous calls, with relatively few heterozygous calls (<a reveal-id="dsx024-F1" data-open="dsx024-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The ratio between homozygous calls and the combined number of both homozygous and heterozygous genotypes was near to one, confirming the homozygous nature of a RIL population (<a reveal-id="dsx024-F1" data-open="dsx024-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). We examined the segregation of reference alleles and alternative alleles and identified that the majority of markers were located around a ratio of 0.5 as expected from a biparental population (<a reveal-id="dsx024-F1" data-open="dsx024-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Confirmation of these characteristics justifies the reasoning behind our filtering strategy. In total 4,463,846 out of 7,979,788 unique genomic positions with reported SNPs passed all filtering steps. Of these positions, on average 2,548,113 markers were available per individual in the low-coverage sequencing data, resulting in over 150 million available markers across the population. Despite stringent filtering, marker distribution was well spread across the genome allowing us to identify recombination patterns with high-resolution (<a reveal-id="dsx024-F1" data-open="dsx024-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>). The lowest and the highest number of markers for a single chromosome of an individual were 31,707 and 563,533, respectively. The average genome coverage per individual played a crucial role in the number of available markers and showed a Pearson correlation coefficient of 0.92 (<a reveal-id="dsx024-F2" data-open="dsx024-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Using the selected marker positions from the population, we used available parental genotype for haplotype phasing and inferred the missing parental genotype. </p>                        <a id="104086026" scrollto-destination="104086026"></a>
<div id="dsx024-F1" data-id="dsx024-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f1.png?Expires=2147483647&Signature=dUZcKmgD~lVh8rIbLIjbdYNzddti5Kl4EqEov2rLNCJCYUJ9KUlbpbS-KNSmIvwJ9BqnpXvsJsL5Tzf0GaCGaPJ3UeRTxq8eaft~lSgjN3OQBRg-cb8sBNSIkdRcjCTru5bgkWqJvORAXOkIB~XFMm9qLSnX-BwQ7lnVZDO-FRoJy0GJ9U66Wz9LEN0KoFzO3r54ITjUhZT6Qd9IrBAs3f7zvojYdsAKWZ9a3JU06smsQ0yJf8sw6oxJBPfp40zhEmPfoZbe8EzqXQKcSK0zbN76MsHFgz22KduLetzKQjpBTSGIf27Mv5TlzuKpqdXlYG6EG1gNKmsI~Kv3~2JtAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Two-dimensional binning plot showing the number of homozygous and heterozygous calls per locus in the population. (B) Histogram showing the ratio between homozygous and the total number of calls. (C) Histogram showing the ratio between alternative and total number of homozygous calls. Cut offs are shown in dotted lines. (D) Genome-wide distribution of markers in 100kbp bins (y-axis). Chromosomes are visualized in blue lines, pericentric heterochromatin are visualized in thick blue lines. (See online article for colour version of this figure). " path-from-xml="dsx024f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086026/dsx024f1.tif" path-from-xml="dsx024f1.tif" target="_blank">View large</a><a section="104086026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f1.png?Expires=2147483647&Signature=t6qRSH89WAp3Vjph3-aqznP6kJKTRX9fBaQhCEtvJWxqZrrGMmbo9c8gc61~1wLTpb1fNEj~Gjx~2mXI-OaUOjyyjJWDw2e4EDLQaDXsTARdZv2oqarfiqGUo2o2OORIvZmRbfRlH2EC4EznJV9xC0DhSl8mvLml7kBtAHJwZsD~xqiLvr4NbwxKIWREUXCtzEE8rAtcMVVPrOUQBmz~DoI8x-ymXZb-qluPu0BKH6bNuZJ2oJab2ABn-HDNTHqSgqvbcOTGFLpMgJebKbL45D61ZrgkBdOgHcEz6E27tMvPUJyOsj8l0kspO8E5vPK-7sXWY4zRusKjWBVQ48sq-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086026&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Two-dimensional binning plot showing the number of homozygous and heterozygous calls per locus in the population. (B) Histogram showing the ratio between homozygous and the total number of calls. (C) Histogram showing the ratio between alternative and total number of homozygous calls. Cut offs are shown in dotted lines. (D) Genome-wide distribution of markers in 100kbp bins (<em>y</em>-axis). Chromosomes are visualized in blue lines, pericentric heterochromatin are visualized in thick blue lines. (See online article for colour version of this figure). </p></div></div><div content-id="dsx024-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f1.png?Expires=2147483647&Signature=dUZcKmgD~lVh8rIbLIjbdYNzddti5Kl4EqEov2rLNCJCYUJ9KUlbpbS-KNSmIvwJ9BqnpXvsJsL5Tzf0GaCGaPJ3UeRTxq8eaft~lSgjN3OQBRg-cb8sBNSIkdRcjCTru5bgkWqJvORAXOkIB~XFMm9qLSnX-BwQ7lnVZDO-FRoJy0GJ9U66Wz9LEN0KoFzO3r54ITjUhZT6Qd9IrBAs3f7zvojYdsAKWZ9a3JU06smsQ0yJf8sw6oxJBPfp40zhEmPfoZbe8EzqXQKcSK0zbN76MsHFgz22KduLetzKQjpBTSGIf27Mv5TlzuKpqdXlYG6EG1gNKmsI~Kv3~2JtAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Two-dimensional binning plot showing the number of homozygous and heterozygous calls per locus in the population. (B) Histogram showing the ratio between homozygous and the total number of calls. (C) Histogram showing the ratio between alternative and total number of homozygous calls. Cut offs are shown in dotted lines. (D) Genome-wide distribution of markers in 100kbp bins (y-axis). Chromosomes are visualized in blue lines, pericentric heterochromatin are visualized in thick blue lines. (See online article for colour version of this figure). " path-from-xml="dsx024f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086026/dsx024f1.tif" path-from-xml="dsx024f1.tif" target="_blank">View large</a><a section="104086026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f1.png?Expires=2147483647&Signature=t6qRSH89WAp3Vjph3-aqznP6kJKTRX9fBaQhCEtvJWxqZrrGMmbo9c8gc61~1wLTpb1fNEj~Gjx~2mXI-OaUOjyyjJWDw2e4EDLQaDXsTARdZv2oqarfiqGUo2o2OORIvZmRbfRlH2EC4EznJV9xC0DhSl8mvLml7kBtAHJwZsD~xqiLvr4NbwxKIWREUXCtzEE8rAtcMVVPrOUQBmz~DoI8x-ymXZb-qluPu0BKH6bNuZJ2oJab2ABn-HDNTHqSgqvbcOTGFLpMgJebKbL45D61ZrgkBdOgHcEz6E27tMvPUJyOsj8l0kspO8E5vPK-7sXWY4zRusKjWBVQ48sq-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086026&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Two-dimensional binning plot showing the number of homozygous and heterozygous calls per locus in the population. (B) Histogram showing the ratio between homozygous and the total number of calls. (C) Histogram showing the ratio between alternative and total number of homozygous calls. Cut offs are shown in dotted lines. (D) Genome-wide distribution of markers in 100kbp bins (<em>y</em>-axis). Chromosomes are visualized in blue lines, pericentric heterochromatin are visualized in thick blue lines. (See online article for colour version of this figure). </p></div></div>                        <a id="104086027" scrollto-destination="104086027"></a>
<div id="dsx024-F2" data-id="dsx024-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f2.png?Expires=2147483647&Signature=U7XhFfqxYSMpGKrOfohwXAb8Udd1GIVxl9~XSrEDsS2nrsUMC~S0stP6tTqqUoVBmNUC5ACif2cI4FYpwj6T4vfqJM-OH7ZJRGFPAllyy35ivsujc6QadPsEX95-BpAbx4XAODk20yvKWPdks3GlnGW2p7~TaJOl-TXhzQPN3qGXxH3J3TylyIL1dDTLmcTtzD9Ix7N2F98SXkmkCZXxjkkb~z7kBjoECpMAgU97P4TLazpBZ2xd~2tOZdIK95UsrNVI3JPAXtTcuAZG5lbeCTPCgt7XsaGkQrb4vVxeknHgKrYOU-dCffZksldw-CjWaZR77lFoSgi7KweSxyr7Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between abundance of available markers (x-axis) and average genome coverage of the each RIL (y-axis)." path-from-xml="dsx024f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086027/dsx024f2.tif" path-from-xml="dsx024f2.tif" target="_blank">View large</a><a section="104086027" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f2.png?Expires=2147483647&Signature=svD86g~14GBdqVvpCO0Lh~P9UO1RZylyeZ~jPpLOJk9gvLzyaVbUhUeOy-AKVolpCXG1Miko1JXjgumEzqy6Sm2ped17ymvm2o7y0ay3S-2CP8XlaGRutOQnUyrsbDVDy0tQjO5~v1nZymWlLKyRswIL84y8fEHDoj4OQ3KI9Tz5YbC7B6KHb-9Dxn5ImNaN6-ozHnSxWnSTqMy0PhKjXQe11MDp0DpCxb2BSRuFnbdT2B8U50IQYIIVCDCfZVUXxqgz8-d8XKhAYwmng3cL8ERCP2BltwKaJhcyTAp7U73qd0F~2lZBLsfeS3Jujab0qKmC6X2zEeCdotTGpaDIzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086027&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between abundance of available markers (<em>x</em>-axis) and average genome coverage of the each RIL (<em>y</em>-axis).</p></div></div><div content-id="dsx024-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f2.png?Expires=2147483647&Signature=U7XhFfqxYSMpGKrOfohwXAb8Udd1GIVxl9~XSrEDsS2nrsUMC~S0stP6tTqqUoVBmNUC5ACif2cI4FYpwj6T4vfqJM-OH7ZJRGFPAllyy35ivsujc6QadPsEX95-BpAbx4XAODk20yvKWPdks3GlnGW2p7~TaJOl-TXhzQPN3qGXxH3J3TylyIL1dDTLmcTtzD9Ix7N2F98SXkmkCZXxjkkb~z7kBjoECpMAgU97P4TLazpBZ2xd~2tOZdIK95UsrNVI3JPAXtTcuAZG5lbeCTPCgt7XsaGkQrb4vVxeknHgKrYOU-dCffZksldw-CjWaZR77lFoSgi7KweSxyr7Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between abundance of available markers (x-axis) and average genome coverage of the each RIL (y-axis)." path-from-xml="dsx024f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086027/dsx024f2.tif" path-from-xml="dsx024f2.tif" target="_blank">View large</a><a section="104086027" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f2.png?Expires=2147483647&Signature=svD86g~14GBdqVvpCO0Lh~P9UO1RZylyeZ~jPpLOJk9gvLzyaVbUhUeOy-AKVolpCXG1Miko1JXjgumEzqy6Sm2ped17ymvm2o7y0ay3S-2CP8XlaGRutOQnUyrsbDVDy0tQjO5~v1nZymWlLKyRswIL84y8fEHDoj4OQ3KI9Tz5YbC7B6KHb-9Dxn5ImNaN6-ozHnSxWnSTqMy0PhKjXQe11MDp0DpCxb2BSRuFnbdT2B8U50IQYIIVCDCfZVUXxqgz8-d8XKhAYwmng3cL8ERCP2BltwKaJhcyTAp7U73qd0F~2lZBLsfeS3Jujab0qKmC6X2zEeCdotTGpaDIzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086027&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between abundance of available markers (<em>x</em>-axis) and average genome coverage of the each RIL (<em>y</em>-axis).</p></div></div>                            <h3 scrollto-destination=104086028 id="104086028" class="section-title" >3.2. Estimated recombination rates and identified recombination prone regions</h3>
<p>Using a variable sliding window approach to identify genotype breaks, we identified 1,445 recombination events in 60 RIL tomato plants with 123 recombination prone regions (see section Materials and Methods, <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s3_dsx024.xlsx?Expires=2147483647&amp;Signature=o09hobCAWSwVE4~r9WtG~EFFZ0ymwu51OmhcOeQQe6XIZNa13B6DEo6lZEpil8ggYdI3NVwQxMEmaEaN-Y9zW~VllcQYpIA6U2-V5HU~X772EtoGFCOhjsk6cNZdHnGiGb4d5vQp-XNHulZ~8cv4UTFObTrkCUFCdaN2lPm0Z44UrAFlPqQO6qIdZVxLmhcl91bQiof1VBwRgdFQy01XXQs8lpYH3d8uiRCq4dGnZSqanmzp3o-XEduD3XI4oox0sxbGRp7SxC9ifHqE9zBT-aAhy0~deuEMmELLe-rpauCvrW~sHehSDMoqx5eCAfESFgJhxsTAQQi4KVOmegkXzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s4_dsx024.doc?Expires=2147483647&amp;Signature=LNiBpm-eBw9G2oz-ktBffk9-wXbxO69fYj7s-3awk9plBmzMTYP4ByPIgrX0v~X5LZxSMRhjUSLMPZ50EPe91n1z2SosyIcP0a4UTgCMNrMzrG0tnyqWyq2f2j13lgHRG3jOJWpBdXZeZxf7GDnIsr608CWxFePQjEKpegD9pO2ztBtMwOPgRsu2eWoOuYz1ZP7Lrv0qQ58rmNIz5TolXnRhK5KbNOV22E10OWJ8zXbObHy9zMMtHYL0VuYkvraUk2H1k90bVFdL4SK9OgeBtFdWoVjJlpF6SOEN4FSzOmLRuss7ieQ0EllM7~Q9XR3AmTEiIV4Dd8KkzpT-7vkLMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S4</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s2_dsx024.pdf?Expires=2147483647&amp;Signature=shpFJXwWd8hhmpcJKNC0RhVyqxo63U9QmpJUA1gYFTbU-eq7RHBw5XzNrt18jCcL-tQkTxH1ZIyMp536brkWyYRrBMX7MzDKdiJkRyoleKfRstUaHrO3XBstWAWVz6U-51r-yzHLEKy0~aP-dTm8m3rVvWvE~r7Hns4yNTLlb0dwU-qbPuZmkVfA6Qdv1cYSN9aijoFxqxAYeW-C2fxrJ9nSHPze5ZUGAjavKvuxutHL22KBdh~HGQLijtSII5tT5Q3-WlZkllRbaeW7FK~VWUC0t20iQmnUSYFm5SN16cQvgzk2GGAmT7dgahFKqsGqk4P-5Hi3cn38LvNgsX2fWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S2</a></span>). We identified recombination events with a median track length of 2,134bp that is comparable to <em>Arabidopsis thaliana</em><em>.</em><a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> On an average, we identified 24 recombination events per RIL, which is in agreement with the expected number of observable recombinations.<a reveal-id="dsx024-B35" data-open="dsx024-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p>During genotyping, three genomic regions were shown to be polymorphic for individual markers, but haplotypes of these genomic regions could not clearly be assigned to one of the parental genomes. As shown in previous studies, abnormalities in reference genome could lead to such patterns.<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B13" data-open="dsx024-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Therefore, we removed recombination prone regions overlapping with these regions from further analysis. As a result, 0.32% of the markers were removed from further analysis. We further extended the above observation as large structural variation due to conflict in reference genome assembly, which can introduce false recombination events and consequently false recombination prone regions.<a reveal-id="dsx024-B23" data-open="dsx024-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsx024-B36" data-open="dsx024-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> We aligned the two most recent <em>S. lycopersicum</em> cv. Heinz 1706 reference genomes (Sl2.40 and Sl2.50) and identified large inversions and rearrangements on chromosome 1, 3 and 12 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s3_dsx024.xlsx?Expires=2147483647&Signature=o09hobCAWSwVE4~r9WtG~EFFZ0ymwu51OmhcOeQQe6XIZNa13B6DEo6lZEpil8ggYdI3NVwQxMEmaEaN-Y9zW~VllcQYpIA6U2-V5HU~X772EtoGFCOhjsk6cNZdHnGiGb4d5vQp-XNHulZ~8cv4UTFObTrkCUFCdaN2lPm0Z44UrAFlPqQO6qIdZVxLmhcl91bQiof1VBwRgdFQy01XXQs8lpYH3d8uiRCq4dGnZSqanmzp3o-XEduD3XI4oox0sxbGRp7SxC9ifHqE9zBT-aAhy0~deuEMmELLe-rpauCvrW~sHehSDMoqx5eCAfESFgJhxsTAQQi4KVOmegkXzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Structural variation was also seen in other chromosomes, but these cases did not interfere with recombination events. In total, 11 recombination prone regions (two from chromosome 1 and all from chromosomes 3 and 12) were attributed to structural variation in the used reference genome. This reduced set of 112 recombination prone regions with a median of 1,204bp, making up a total size of 278,937bp was used for identification of genomic features. Because of differences in number of available markers, due to variance in sequence coverage, recombination site resolution varied between individuals. However, by utilizing the shared regions in the individually observed recombination events to define recombination prone regions allowed us to improve the resolution of these regions. These high-resolution recombination intervals in an interspecific tomato genome, allowed us to expand the knowledge of recombination patterns in tomato at a sequence level.</p><p>The low number of recombinations per chromosome hampered fine resolution identification of recombination deprived regions. However, regions on all chromosomes without any recombination event at all were excessively large, ranging from 17.8 to 60.0Mbp (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s5_dsx024.doc?Expires=2147483647&Signature=cULqQ-MZwN0dvjpplBEzFv3sWmhIGlwQRj~ZEIFcbCT2p6bltMbPlRTv4mGTE2ZG3LQoQKOn~c-OcY9Ob7uto2uPyI4F3d0KYbfQseEqFpJLwS0yw9lt-iCXMF2IqVLFHF7JF1ozTRW~KPAToIdA~EnR3UbMZW~0t43dJEw1d28f3ciRwf89dHdmJPrUfot~NoeNuPKuK8V4tYLsElBxVIdw3n1yK0ylYa1SN~E7Ig34yOvNc1NwiuS~-DN4kjrszfj5D~V2MyF0fQ6d0qmh1Aqe~7lgVCEZKL9gkXHgGRAwd9IcmG803TfSw3OCParEt05kxpxRzipqZ5bpIA-zjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Recombination prone and deprived regions were clearly overlapping with euchromatin and pericentric heterochromatin regions of the chromosome, respectively (<a reveal-id="dsx024-F3" data-open="dsx024-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>).<sup>22,<a reveal-id="dsx024-B32" data-open="dsx024-B32" class="link link-ref link-reveal xref-bibr">32</a></sup> This consistent pattern has been observed in other plant species indicating an evolutionarily conserved preference mechanism either due to local genomics features or open chromatin structure.<a reveal-id="dsx024-B1" data-open="dsx024-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> We used recombination prone regions to perform <em>de novo</em> DNA motif discovery, together with other genomic features identification. </p>                        <a id="104086032" scrollto-destination="104086032"></a>
<div id="dsx024-F3" data-id="dsx024-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f3.png?Expires=2147483647&Signature=f9iApp6FI9te0CjfSWuf8~WJyoHRs3HehuOQJbkXrNlb~1OOZSSClNGn8-puSB4Rj5kyNOSQ6HZmnBNtL0WCGSzDy2am1dYuXapiBEB8fFyUPfOZFIOZLskWN9HkUojMvik3Yyt7LR89M1ZPrGZoqiwAJQWAyRctQyzR-F9sa5nOM4fqCI2IHA~nIuVsmh1KymAzxG~cInC70cZzJGHRCbyMWnPLy760xif5TczW1OqG72YW7jEbvvhU7bmFLfyWu~be7sOTSj-6b6phczjbkBDS08nTD9P14Y5C3EAis~Di25yvl-0ZXj8h~Tw~eiySCbEnwU9UKsqsGq-Qx6R0Qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary plot of recombination prone and deprived regions along chromosomes 1 and 2 and 4–11. Pericentric heterochromatins are shown in thick blue lines.22 Per chromosome, the biggest contiguous region completely deprived of recombination events are coloured blue, the remaining is coloured in yellow. Regions prone to recombination are indicated by orange triangles. (See online article for colour version of this figure)." path-from-xml="dsx024f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086032/dsx024f3.tif" path-from-xml="dsx024f3.tif" target="_blank">View large</a><a section="104086032" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f3.png?Expires=2147483647&Signature=uArYqUxXcWM2IC7pKOFl8mWjTOMRpgbACYFnMnsP8bBxJyEgUxuLF-~ys9fYm5O4HPnlJqHXcOCm-oxH97xkN9ruVAQFzYvZCFyz1WXhMk4zjZXrrSGgtL2Sec~XB-LStZQbKdVfxWq0NkP4lo-KuiF7NbpKIF2EcT6qD1RRVnvBObPRz4Ht0poK69isrdnsS02pGCKd4fZ78i5WqaSkhSSm6iV5mRSpZEzLmwnM40SF-rpSXO-wXapjAS9~dDTeDflYG1xMiYnte1ohBfcxjxVwVCQcrtrUtafMtDnWzIm-UlKtB4NTL2Iv6ILWnjkdJCv0lvGzUBcs8ygwEnxTNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086032&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary plot of recombination prone and deprived regions along chromosomes 1 and 2 and 4–11. Pericentric heterochromatins are shown in thick blue lines.<a reveal-id="dsx024-B22" data-open="dsx024-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Per chromosome, the biggest contiguous region completely deprived of recombination events are coloured blue, the remaining is coloured in yellow. Regions prone to recombination are indicated by orange triangles. (See online article for colour version of this figure).</p></div></div><div content-id="dsx024-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f3.png?Expires=2147483647&Signature=f9iApp6FI9te0CjfSWuf8~WJyoHRs3HehuOQJbkXrNlb~1OOZSSClNGn8-puSB4Rj5kyNOSQ6HZmnBNtL0WCGSzDy2am1dYuXapiBEB8fFyUPfOZFIOZLskWN9HkUojMvik3Yyt7LR89M1ZPrGZoqiwAJQWAyRctQyzR-F9sa5nOM4fqCI2IHA~nIuVsmh1KymAzxG~cInC70cZzJGHRCbyMWnPLy760xif5TczW1OqG72YW7jEbvvhU7bmFLfyWu~be7sOTSj-6b6phczjbkBDS08nTD9P14Y5C3EAis~Di25yvl-0ZXj8h~Tw~eiySCbEnwU9UKsqsGq-Qx6R0Qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary plot of recombination prone and deprived regions along chromosomes 1 and 2 and 4–11. Pericentric heterochromatins are shown in thick blue lines.22 Per chromosome, the biggest contiguous region completely deprived of recombination events are coloured blue, the remaining is coloured in yellow. Regions prone to recombination are indicated by orange triangles. (See online article for colour version of this figure)." path-from-xml="dsx024f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086032/dsx024f3.tif" path-from-xml="dsx024f3.tif" target="_blank">View large</a><a section="104086032" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f3.png?Expires=2147483647&Signature=uArYqUxXcWM2IC7pKOFl8mWjTOMRpgbACYFnMnsP8bBxJyEgUxuLF-~ys9fYm5O4HPnlJqHXcOCm-oxH97xkN9ruVAQFzYvZCFyz1WXhMk4zjZXrrSGgtL2Sec~XB-LStZQbKdVfxWq0NkP4lo-KuiF7NbpKIF2EcT6qD1RRVnvBObPRz4Ht0poK69isrdnsS02pGCKd4fZ78i5WqaSkhSSm6iV5mRSpZEzLmwnM40SF-rpSXO-wXapjAS9~dDTeDflYG1xMiYnte1ohBfcxjxVwVCQcrtrUtafMtDnWzIm-UlKtB4NTL2Iv6ILWnjkdJCv0lvGzUBcs8ygwEnxTNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086032&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary plot of recombination prone and deprived regions along chromosomes 1 and 2 and 4–11. Pericentric heterochromatins are shown in thick blue lines.<a reveal-id="dsx024-B22" data-open="dsx024-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Per chromosome, the biggest contiguous region completely deprived of recombination events are coloured blue, the remaining is coloured in yellow. Regions prone to recombination are indicated by orange triangles. (See online article for colour version of this figure).</p></div></div>                            <h3 scrollto-destination=104086033 id="104086033" class="section-title" >3.3. Validated inferred parental genotype and RIL haplotype blocks</h3>
<p>Validation of our genotyping approach for both parental lines and RILs was done using KASP assays. Five plants were excluded because either seed germination or DNA isolation was unsuccessful. An additional four plants were discarded after genotyping because of high levels of heterozygosity. Two plants showing random genotypes were also removed, as these plants were noticeably different from the corresponding resequenced RILs and were most likely the result of a mix up of supplied seeds, either before or during genotyping. During data analysis, three out of 133 successful KASP assays were discarded. Two of them failed for more than half of the population and one was non-polymorphic. All the remaining assays were polymorphic, confirming the identification of polymorphic markers from resequencing data.</p><p>After above-mentioned filtering steps, we successfully genotyped 49 RILs using 130 KASP assays. RIL haplotypes estimated with low-coverage resequencing data were validated by KASP assays with an accuracy of 96.8%. This accuracy increased to 98.0% when only homozygous markers were considered. These high accuracies confirmed our methodology for both marker selection and haplotype block estimation using low-coverage resequencing data. Markers identified by resequencing of <em>S. lycopersicum</em> and inferred markers of <em>S. pimpinellifolium</em> matched with 99.2% of the assays, proving high accuracy of our methodology to infer markers for one of the parental lines.</p>                            <h3 scrollto-destination=104086036 id="104086036" class="section-title" >3.4. AT-rich DNA motifs are enriched at recombination prone regions</h3>
<p>Previous reports in <em>Arabidopsis thaliana</em> and <em>Citrus clementina</em> have identified DNA motifs associated with recombination sites.<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B8" data-open="dsx024-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx024-B37" data-open="dsx024-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In tomato, since the genome-wide high-resolution recombination pattern was unavailable, it was not possible to analyse recombination associated DNA sequence motifs so far. We carried out a <em>de novo</em> DNA motif discovery in recombination prone regions together with 0, 1, 2.5 and 5kbp flanking regions to discover sequence motifs in tomato. Several motifs were discovered out of which six motifs were found to be over-enriched compared to equivalent random sequences. These motifs were consistently enriched in recombination prone regions with 0–5kbp flanking regions (see section Materials and methods; <a reveal-id="dsx024-F4" data-open="dsx024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). </p>                        <a id="104086038" scrollto-destination="104086038"></a>
<div id="dsx024-F4" data-id="dsx024-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f4.png?Expires=2147483647&Signature=q0FOxK9snHbFFEFth~Q4p5~dGJ8C7wb~G96om6ZYkeC2k06tVyk7U1nT6J1fwpagHu5GXwHVvpJ3FNxw4HuL9q~otamPU1wL~07TpDrhTyJLvHZjvSU-Dat7pd19LED3i8q2c1fQ4LRPa9Etcj4P65YdGRQH8imfmEHx0JmyYWImazpqpr3j2NiGUhsoh0c8rn8Z75oV9ucUbdKtJxHPKnDBzmlSvP1ANF3B95VxrL0Y5CEeeGDye7Csf7kvYSKmrFqYdLBQJoyC~pCazWW1EdRCAV1WCaxBL8vLRA5snO0rFCzJopgdwgLB7xdL85AOJyXhnA5~53~sZwa89DhpzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence motifs enriched at recombination prone regions, including 1000bp flanking regions. (See online article for colour version of this figure)." path-from-xml="dsx024f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086038/dsx024f4.tif" path-from-xml="dsx024f4.tif" target="_blank">View large</a><a section="104086038" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f4.png?Expires=2147483647&Signature=TwzauYZBgn05AFY5diYt47kkVy5lyaImCes1Lhfx0d1WECmlutJeMIxNiMTD-eDsOeVlZ8Z2Sqfxfy4qTbt3H2q0y6n8YSnXyphrO~jnVKM72jruNEsxiqOnujrumnnutpYL4BztBv-j0VQoYeZMhCeuuOQVJBQHjfkwC--QACgulwiESdNZk1zgMoVfsgezXW-pE~zWJuhhh3elNVBS2bnBr07VqZUTAR54HzktVIfkzQf9Nr8eOLIp9CVXhRE3uDfJICd~3X3Uuu-RjPqQxeo~ITgzbQEyvj1uZ0kvCfFj-dbr1PggR1ijMTmhPKOSINna9rZ7DyuS~r0RzNLb2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086038&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence motifs enriched at recombination prone regions, including 1000bp flanking regions. (See online article for colour version of this figure).</p></div></div><div content-id="dsx024-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f4.png?Expires=2147483647&Signature=q0FOxK9snHbFFEFth~Q4p5~dGJ8C7wb~G96om6ZYkeC2k06tVyk7U1nT6J1fwpagHu5GXwHVvpJ3FNxw4HuL9q~otamPU1wL~07TpDrhTyJLvHZjvSU-Dat7pd19LED3i8q2c1fQ4LRPa9Etcj4P65YdGRQH8imfmEHx0JmyYWImazpqpr3j2NiGUhsoh0c8rn8Z75oV9ucUbdKtJxHPKnDBzmlSvP1ANF3B95VxrL0Y5CEeeGDye7Csf7kvYSKmrFqYdLBQJoyC~pCazWW1EdRCAV1WCaxBL8vLRA5snO0rFCzJopgdwgLB7xdL85AOJyXhnA5~53~sZwa89DhpzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA sequence motifs enriched at recombination prone regions, including 1000bp flanking regions. (See online article for colour version of this figure)." path-from-xml="dsx024f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086038/dsx024f4.tif" path-from-xml="dsx024f4.tif" target="_blank">View large</a><a section="104086038" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f4.png?Expires=2147483647&Signature=TwzauYZBgn05AFY5diYt47kkVy5lyaImCes1Lhfx0d1WECmlutJeMIxNiMTD-eDsOeVlZ8Z2Sqfxfy4qTbt3H2q0y6n8YSnXyphrO~jnVKM72jruNEsxiqOnujrumnnutpYL4BztBv-j0VQoYeZMhCeuuOQVJBQHjfkwC--QACgulwiESdNZk1zgMoVfsgezXW-pE~zWJuhhh3elNVBS2bnBr07VqZUTAR54HzktVIfkzQf9Nr8eOLIp9CVXhRE3uDfJICd~3X3Uuu-RjPqQxeo~ITgzbQEyvj1uZ0kvCfFj-dbr1PggR1ijMTmhPKOSINna9rZ7DyuS~r0RzNLb2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086038&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA sequence motifs enriched at recombination prone regions, including 1000bp flanking regions. (See online article for colour version of this figure).</p></div></div><p>Within enriched motifs, homopolymer poly-A and poly-T occurred in all recombination prone regions. Additionally, we identified a poly-T/poly-A motif in all recombination prone regions with the highest adjusted <em>P</em>-value compared to the whole tomato genome as background (<a reveal-id="dsx024-F4" data-open="dsx024-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). This motif is similar to motifs reported in earlier studies<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B8" data-open="dsx024-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and associated with nucleosome boundaries.<a reveal-id="dsx024-B38" data-open="dsx024-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Besides the poly-T/poly-A motif, a 15bp TATA-repeat was discovered. Both the TATA-repeat and poly-T/poly-A motif were reported for the first time and further studies are required to establish the functional correlation. In contrast, the CCN-repeat motif previously identified in <em>Arabidopsis</em> was not identified in our study. Meanwhile, the CTT-repeat motif identified in <em>Arabidopsis</em> was detected, but was not enriched.</p>                            <h3 scrollto-destination=104086040 id="104086040" class="section-title" >3.5. Recombination prone regions in tomato occurred at gene and promoter regions</h3>
<p>We further analysed other genomic features associated with recombination prone regions. All recombination prone regions were observed within euchromatin regions. We examined the position of recombination prone regions in the genome using tomato gene annotation. Recombination prone regions showed a tendency to be at or near to genes. This is consistent to previous observations in <em>Arabidopsis</em><em>.</em><a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> We have found that gene body was significantly over-represented in recombination prone regions (<em>P</em>-value 0.01 [permutation test]). Notably, features such as 3′-UTR and promoter showed significant enrichment in recombination prone regions (<em>P</em>-value 1.403 × 10<sup>−6</sup> and 0.00043, respectively [permutation test]). Forty-eight recombination prone regions overlapped with at least one annotated gene. We repeated the analysis by excluding pericentric heterochromatin from the expected probability analysis. This stringent analysis also showed consistent enrichment (<a reveal-id="dsx024-F5" data-open="dsx024-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). However, for genes overlapping with recombination prone regions, no enrichment was found for either any gene ontology category or NB-LRR resistance genes. </p>                        <a id="104086042" scrollto-destination="104086042"></a>
<div id="dsx024-F5" data-id="dsx024-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f5.png?Expires=2147483647&Signature=I30QpL3C5I8GTqVxGz6AllLHhov0I4wQ5BoYsRIgsNQf0R00AqfiWBFDcPU86jVt0dYyr82O0VWpWxdSzQFkzZtCN62xDkZe94EpR1Zv-9HKYvd~uvUC9rRW-9jqU7-9tnsXUSlWbU-jUoAP0UH926XP-zGWrmE1CYwU5zmknSN4JN7eLGNof6glvevvnxjwjBfV6ScZZlerC-FIWymDuHNXjd4qxIM~covLrWa9UDl1PWUiJltu4xprSDUFeqh0jTO5wknTV0Wtr8oAnH0eZN50GrqPwedjTt8-pZLfbN0NviN-fhMdM1X9cUT73vtdcMBfoC9bpL6vIwkmTTvvaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positions of recombination prone regions based on tomato genome annotation. Standard error is represented by grey error bars. (See online article for colour version of this figure)." path-from-xml="dsx024f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086042/dsx024f5.tif" path-from-xml="dsx024f5.tif" target="_blank">View large</a><a section="104086042" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f5.png?Expires=2147483647&Signature=2XIGtlQ175Nq-lEYOoSim-cYr~IIuusdv58yC~-wtpeVlh2V~80mx~wN-N8I~pm-dYYYnOuuGMvGUzkvvgtiwFUMtwNjYBMOHCwI6TyJx-4jq12qiKhfpcy3i7wyMMJ~reoxip9j6bG9YXG2cM9WFsXyqmPEL8woyC~zUoyeV41OKPy6mi2Bpk5iUYe13wnRyDlyJVVP8I00bvKK-fWSwiV~zkhWcTj9kTcKSNDwEBcq-EISo9uOM2ZFVmcNC8x96p90nuANj~V8Pw40W2TXcJrxahRWUZlwhbXMGTXMXPjMJYcRuU~J5dzS0R4W-yskWA0qy5~DcwSJaJF5VBgtfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086042&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positions of recombination prone regions based on tomato genome annotation. Standard error is represented by grey error bars. (See online article for colour version of this figure).</p></div></div><div content-id="dsx024-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f5.png?Expires=2147483647&Signature=I30QpL3C5I8GTqVxGz6AllLHhov0I4wQ5BoYsRIgsNQf0R00AqfiWBFDcPU86jVt0dYyr82O0VWpWxdSzQFkzZtCN62xDkZe94EpR1Zv-9HKYvd~uvUC9rRW-9jqU7-9tnsXUSlWbU-jUoAP0UH926XP-zGWrmE1CYwU5zmknSN4JN7eLGNof6glvevvnxjwjBfV6ScZZlerC-FIWymDuHNXjd4qxIM~covLrWa9UDl1PWUiJltu4xprSDUFeqh0jTO5wknTV0Wtr8oAnH0eZN50GrqPwedjTt8-pZLfbN0NviN-fhMdM1X9cUT73vtdcMBfoC9bpL6vIwkmTTvvaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positions of recombination prone regions based on tomato genome annotation. Standard error is represented by grey error bars. (See online article for colour version of this figure)." path-from-xml="dsx024f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086042/dsx024f5.tif" path-from-xml="dsx024f5.tif" target="_blank">View large</a><a section="104086042" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f5.png?Expires=2147483647&Signature=2XIGtlQ175Nq-lEYOoSim-cYr~IIuusdv58yC~-wtpeVlh2V~80mx~wN-N8I~pm-dYYYnOuuGMvGUzkvvgtiwFUMtwNjYBMOHCwI6TyJx-4jq12qiKhfpcy3i7wyMMJ~reoxip9j6bG9YXG2cM9WFsXyqmPEL8woyC~zUoyeV41OKPy6mi2Bpk5iUYe13wnRyDlyJVVP8I00bvKK-fWSwiV~zkhWcTj9kTcKSNDwEBcq-EISo9uOM2ZFVmcNC8x96p90nuANj~V8Pw40W2TXcJrxahRWUZlwhbXMGTXMXPjMJYcRuU~J5dzS0R4W-yskWA0qy5~DcwSJaJF5VBgtfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086042&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positions of recombination prone regions based on tomato genome annotation. Standard error is represented by grey error bars. (See online article for colour version of this figure).</p></div></div><p>We visualized the genomic distance from recombination prone regions to the nearest gene and observed that the distance to the nearest gene was lower when starting from a recombination prone region, than when starting from a randomly sampled region (<a reveal-id="dsx024-F6" data-open="dsx024-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). We observed that a remarkable 74% of recombination prone regions were at gene or promoter regions, compared with the random average expected rate of 32%. Furthermore, this pattern was persistent despite removing pericentric heterochromatin, known for low gene density, from the analysis. Similar observations were made in <em>Arabidopsis thaliana</em> where recombination targets AT-rich promoter sites.<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Although tomato has lower gene density compared with <em>Arabidopsis thaliana</em>, strikingly, it has over-enrichment of recombination prone regions at gene body. This might be due to longer tomato genes and the genomic composition of the tomato genome. These observations indicate that gene or promoter associated genomic features, such as DNA motifs and chromatin stage, may influence recombination sites. </p>                        <a id="104086044" scrollto-destination="104086044"></a>
<div id="dsx024-F6" data-id="dsx024-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f6.png?Expires=2147483647&Signature=ySbI0MDTuZKOmAQHfStTFeYTL8iXINv~cwHbT3eIBmOPpghHGD9UNVllkHbEpZuJJgbrSi7Ygoxp1JHhK73N2NzUDn2LzjBlVKJx1~2HhPMAxpZibU7sc-oz5AlF77xcgvpSi7X9LmSnLlsvK6VFHpYI6hllfnmVpS~s10TQ6Rb5YW7wX6Sx6~H0S-kHJDLJ6EjwDAK9soDlQ7RIok~J0ci6h2~LKxSaTO-ZRc6~7~~9bytd1SmuuCRpLeRJvmC5~kq6UftnSO6BlVaU2LvNSiNgc3rqotDNaBySan6cS0F4mXMBPt6bUv9x4A-yv4-qGgLhxGybjKMTqrZ6iMCN5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic distance from recombination prone regions to the nearest annotated gene in 500bp bins. Black line shows the observed distance from recombination prone region to the nearest annotated gene, whereas the expected background mean and standard deviation are shown in blue and orange. Magnified version of first 5kbp is shown for visual easiness without using bins. (See online article for colour version of this figure). " path-from-xml="dsx024f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086044/dsx024f6.tif" path-from-xml="dsx024f6.tif" target="_blank">View large</a><a section="104086044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f6.png?Expires=2147483647&Signature=dgrnxlcm0refs2kST-6sBhxNurF7F9ZHbr~00cr2dBSWT~EBZ5NWp7w8TR3PUOGD4HUd35h~XtrsJBdi7IjhcDIvPwwAeXSVRJnQ4V-Q0mbdUmH3VGlhye5D~Rr5A-VFv1Go8IIHE4qJmx9cMVwb7nFF3mjxgwMdJBMTF~DEkjhC8qndaSpPwDrY2xoW9VHUbvxsQzzCVmwQBbmOKoLM6WWeY9Xl29R02hWR1c~S1EiEVTB6pfuHlUIbwc67Ff67svEyuMeqoJUbZQd0OO4xKJZ369PT2tRObj2KY1FuNIUnRb~nteAFSs~SidCkdFo85CxhiJs4ruzeJaCs1FgsrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086044&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic distance from recombination prone regions to the nearest annotated gene in 500bp bins. Black line shows the observed distance from recombination prone region to the nearest annotated gene, whereas the expected background mean and standard deviation are shown in blue and orange. Magnified version of first 5kbp is shown for visual easiness without using bins. (See online article for colour version of this figure). </p></div></div><div content-id="dsx024-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/m_dsx024f6.png?Expires=2147483647&Signature=ySbI0MDTuZKOmAQHfStTFeYTL8iXINv~cwHbT3eIBmOPpghHGD9UNVllkHbEpZuJJgbrSi7Ygoxp1JHhK73N2NzUDn2LzjBlVKJx1~2HhPMAxpZibU7sc-oz5AlF77xcgvpSi7X9LmSnLlsvK6VFHpYI6hllfnmVpS~s10TQ6Rb5YW7wX6Sx6~H0S-kHJDLJ6EjwDAK9soDlQ7RIok~J0ci6h2~LKxSaTO-ZRc6~7~~9bytd1SmuuCRpLeRJvmC5~kq6UftnSO6BlVaU2LvNSiNgc3rqotDNaBySan6cS0F4mXMBPt6bUv9x4A-yv4-qGgLhxGybjKMTqrZ6iMCN5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic distance from recombination prone regions to the nearest annotated gene in 500bp bins. Black line shows the observed distance from recombination prone region to the nearest annotated gene, whereas the expected background mean and standard deviation are shown in blue and orange. Magnified version of first 5kbp is shown for visual easiness without using bins. (See online article for colour version of this figure). " path-from-xml="dsx024f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086044/dsx024f6.tif" path-from-xml="dsx024f6.tif" target="_blank">View large</a><a section="104086044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/dsx024f6.png?Expires=2147483647&Signature=dgrnxlcm0refs2kST-6sBhxNurF7F9ZHbr~00cr2dBSWT~EBZ5NWp7w8TR3PUOGD4HUd35h~XtrsJBdi7IjhcDIvPwwAeXSVRJnQ4V-Q0mbdUmH3VGlhye5D~Rr5A-VFv1Go8IIHE4qJmx9cMVwb7nFF3mjxgwMdJBMTF~DEkjhC8qndaSpPwDrY2xoW9VHUbvxsQzzCVmwQBbmOKoLM6WWeY9Xl29R02hWR1c~S1EiEVTB6pfuHlUIbwc67Ff67svEyuMeqoJUbZQd0OO4xKJZ369PT2tRObj2KY1FuNIUnRb~nteAFSs~SidCkdFo85CxhiJs4ruzeJaCs1FgsrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086044&ar=3865558&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx024f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic distance from recombination prone regions to the nearest annotated gene in 500bp bins. Black line shows the observed distance from recombination prone region to the nearest annotated gene, whereas the expected background mean and standard deviation are shown in blue and orange. Magnified version of first 5kbp is shown for visual easiness without using bins. (See online article for colour version of this figure). </p></div></div>                            <h2 scrollto-destination=104086045 id="104086045" class="section-title" >4. Discussion</h2>
<p>Marker identification and selection are crucial when markers are identified using low-coverage resequencing data. Therefore, we developed a population-based methodology to compile marker sets, using various criteria to ensure selected markers were genuine markers rather than sequencing or reference genome artefacts. The methodology is based on only selecting genomic positions that have multiple unequivocal called SNPs that match with the genetic characteristics of those expected from a RIL population. By selecting these positions, the sequencing and read alignment artefacts introduced with low-coverage sequencing were reduced to less than one per cent. In order to achieve this, markers for a RIL population were filtered to contain high percentages of homozygous calls and an equal ratio between reference and alternative calls. Different parameters will be desired when using other population types and sizes. Also, the quality of the used sequencing technique and the average depth strongly influence the desired parameters. We observed that in tomato the added value of increased genome coverage is depreciating above a coverage of 10, similar to <em>Arabidopsis thaliana</em> resequencing recommendations.<a reveal-id="dsx024-B39" data-open="dsx024-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Therefore, we suggest a minimum coverage of 10 for projects with similar experimental setup and aim.</p><p>We confirmed the efficiency of our approach to estimate haplotypes by KASP validation, and identified a specificity of 97.96% for homozygous haplotypes. The same KASP validation was used to show the efficiency and specificity of our inferred <em>S. pimpinellifolium</em> genotypes and validated 99.63% of these inferred genotypes. Only one assay was found to be monomorphic between parental lines. However, within the RIL population, this locus had 18 reference calls, 24 alternative calls and 1 heterozygous call, pointing to the likelihood that this error was introduced by the used KASP assay.</p><p>Despite conservative selection of markers, we were able to use 4,463,846 markers for genotyping. This in combination with the variable sliding window approach we developed, allowed us to identify recombination patterns in tomato at an unprecedented resolution, with a median size of 2,134bp. By accumulating the identified recombination events, we managed to increase the resolution of recombination prone regions to a median size of 1,204bp. The achieved resolution is the highest so far in tomato, improving the accuracy of subsequent identification of associated genomic features. Although the false-negative rate of our variable sliding window approach is unknown, we deliberately selected parameters and elected specificity over sensitivity in order to accurately predict genomic features associated with recombination prone regions. Genomes with high repeat content and incomplete reference genome would benefit the most from our marker selection and methodology to estimate haplotype blocks.</p><p>We have identified 112 recombination prone regions from 60 interspecific tomato RILs in which 74% of recombination prone regions overlapped with gene or promoter regions. This strong preference is consistent with previous reports from <em>Arabidopsis thaliana</em>,<a reveal-id="dsx024-B5" data-open="dsx024-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx024-B10" data-open="dsx024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>Zea mays</em><a reveal-id="dsx024-B40" data-open="dsx024-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and <em>Mimulus guttatus</em><em>.</em><a reveal-id="dsx024-B41" data-open="dsx024-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Contradictory to <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em>, where high recombination rate in resistance genes was reported, our recombination prone regions showed no enrichment for these type of genes.<a reveal-id="dsx024-B42" data-open="dsx024-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsx024-B43" data-open="dsx024-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> This might be due to heterogeneity in resistance genes or diversity in used parental lines. In all chromosomes, we identified recombination deprived regions to be centric to centromere. This might be due to the extended pericentric heterochromatin in tomato.<a reveal-id="dsx024-B32" data-open="dsx024-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p><p>Previous studies, particularly those in <em>Arabidopsis thaliana</em>, have found poly-A and CTT-repeat motifs to be enriched in hotspot sequences. Recombination prone regions described in this study were found to be enriched with AT-rich motifs. The presence of poly-A motif in all recombination prone regions was prominent in our analysis, whereas CTT-repeats were not enriched. However, lately it has been shown in <em>Arabidopsis</em> that CCN repeats were enriched at crossover regions though they were preferentially associated with genes but not with promoters.<a reveal-id="dsx024-B10" data-open="dsx024-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> These observations indicate that although similarities in DNA motif and genomic features are found across plant species, there are differences too. Differences between eukaryotic species in mechanistic performances also are likely to occur, when looking at preferred recombination sites in human, mice, plants and yeast.<a reveal-id="dsx024-B1" data-open="dsx024-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx024-B44" data-open="dsx024-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p><p>Besides previously reported poly-A motif, we identified a novel TATA-repeat and poly-T/poly-A motif. According to the literature, these AT-rich motifs are associated with strong nucleosome free regions, of which poly-T/A results in the most robust nucleosome-free region.<a reveal-id="dsx024-B38" data-open="dsx024-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Furthermore, the flanking DNA on both sides of the motif is considered to be nucleosome depleted as well, and this effect would remain significant over ∼100–150bp.<a reveal-id="dsx024-B45" data-open="dsx024-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> In yeast, nucleosome depletion is in fact especially pronounced around AT-rich repeats and nucleosomes containing the histone variant H2A.Z tend to border tandem repeats.<a reveal-id="dsx024-B46" data-open="dsx024-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx024-B47" data-open="dsx024-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Additionally, H2A.Z is associated with transcriptional regulation, anti-silencing, silencing and genome stability suggesting that H2A.Z and recombination rates could have several other biological relevancies.<a reveal-id="dsx024-B48" data-open="dsx024-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Hence the presence of poly-T to poly-A tract, among the other enriched motifs in recombination-related sequences in tomato, infers the possibility of nucleosome depletion at recombination prone region. These consistent and emergent observations about the association between recombination prone region, gene promoter region, nucleosome depletion and AT-rich DNA motifs indicate an evolutionary conserved mechanism within eukaryotes and demands further causality studies. Better understanding of the effect of these genetic and genomics features on recombination will provide tools to modulate recombination for the advancement of plant breeding.</p>                            <h2 scrollto-destination=104086052 id="104086052" class="backsection-title" >Data availability</h2>
<p>This tool is available for academic research. Contact corresponding author to request a copy. Nucleotide sequences are deposited to the European Nucleotide Archive by the Plant Research International. The accession number for this study is PRJEB6659. The filtered marker-set can be downloaded from <a class="link link-uri" href="http://ftp://ftp.solgenomics.net/manuscripts/Geo_2017/Supplementary_table_S2_Marker-set.txt" target="_blank">ftp://ftp.solgenomics.net/manuscripts/Geo_2017/Supplementary_table_S2_Marker-set.txt</a></p>                            <h2 scrollto-destination=104086054 id="104086054" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to all the members of 150 tomato genome project consortium and in particularly to Dr. Richard Finkers for sharing the RIL seeds. Also, we would like to thank Ms. Eva Paulina Verhofstad and Dr. Shushimita Shushimita for their detailed comments in the final stage of our manuscript. Finally, we would like to thank Martijn van Elk for his assistance during read alignment and variant calling.</p>                            <h3 scrollto-destination=104086056 id="104086056" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=104086058 id="104086058" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s1_dsx024.doc?Expires=2147483647&Signature=P6ILI5PE4Vehg1ttVp9BD~M2S31yKps~0mSyoBG3QJN1FGSKvV-uFv8sDVGK0POISlCRitxvvwXUa0Q5DagjII2ae6bwjBjHyvSJDDHx-~nBkORIstE6e4KkzOUxi7HB-5BdUYMZgp5LbZBhcMtBkFmTSn0UyCIMeAe0ydF53QZAm~EK0MZv6F0aQOY8ciYaImHpqIKWsCMBemZWspMZ70J5MB9K7YE9wJVd97OIsSK-CB~ezIXONqvITNWHQ8T3RYFf40aWmhIlXKAzCZ0kV2yDYOwGkxdtjVT~Ni1b8m30ZQJ-MyyxWhXLG-7aJPXeNACht52cnBRS1Se1-5aOgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">DNARES online</a>.</p>                            <h3 scrollto-destination=104086060 id="104086060" class="section-title" >Funding</h3>
<p>No external funding.</p>                            <h2 scrollto-destination=104086062 id="104086062" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx024-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mézard</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jenczewski</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Macaisne</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Grelon</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The molecular biology of meiosis in plants,</div> <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">66</div>, <div class="fpage">297</div>–<div class="lpage">327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20biology%20of%20meiosis%20in%20plants%2C&amp;author=R.%20Mercier&amp;author=C.%20M%C3%A9zard&amp;author=E.%20Jenczewski&amp;author=N.%20Macaisne&amp;author=M.%20Grelon&amp;publication_year=2015&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=66&amp;pages=297-327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050213-035923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050213-035923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050213-035923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25494464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bauer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Falque</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Walter</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Intraspecific variation of recombination rate in maize,</div> <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intraspecific%20variation%20of%20recombination%20rate%20in%20maize%2C&amp;author=E.%20Bauer&amp;author=M.%20Falque&amp;author=H.%20Walter&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-9-r103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-9-r103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-9-r103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24050704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The B73 maize genome: complexity, diversity, and dynamics,</div> <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1112</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics%2C&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saintenac</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Faure</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Remay</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Variation in crossover rates across a 3-Mb contig of bread wheat (Triticum aestivum) reveals the presence of a meiotic recombination hotspot,</div> <div class="source">Chromosoma</div> , <div class="volume">120</div>, <div class="fpage">185</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20crossover%20rates%20across%20a%203-Mb%20contig%20of%20bread%20wheat%20(Triticum%20aestivum)%20reveals%20the%20presence%20of%20a%20meiotic%20recombination%20hotspot%2C&amp;author=C.%20Saintenac&amp;author=S.%20Faure&amp;author=A.%20Remay&amp;publication_year=2011&amp;journal=Chromosoma&amp;volume=120&amp;pages=185-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00412-010-0302-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-010-0302-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00412-010-0302-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21161258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wijnker</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">James</div> <div class="given-names">G.V.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The genomic landscape of meiotic crossovers and gene conversions in Arabidopsis thaliana,</div> <div class="source">eLife</div> , <div class="volume">2</div>, e01426.</div></div></div></div><div content-id="dsx024-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salomé</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Bomblies</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fitz</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The recombination landscape in Arabidopsis thaliana F2 populations,</div> <div class="source">Heredity</div> , <div class="volume">108</div>, <div class="fpage">447</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20recombination%20landscape%20in%20Arabidopsis%20thaliana%20F2%20populations%2C&amp;author=P.A.%20Salom%C3%A9&amp;author=K.%20Bomblies&amp;author=J.%20Fitz&amp;publication_year=2012&amp;journal=Heredity&amp;volume=108&amp;pages=447-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2011.95" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2011.95" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2011.95"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22072068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paape</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Branca</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Briskine</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Tiffin</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fine-scale population recombination rates, hotspots, and correlates of recombination in the Medicago truncatula genome,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">726</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20population%20recombination%20rates%2C%20hotspots%2C%20and%20correlates%20of%20recombination%20in%20the%20Medicago%20truncatula%20genome%2C&amp;author=T.%20Paape&amp;author=P.%20Zhou&amp;author=A.%20Branca&amp;author=R.%20Briskine&amp;author=N.%20Young&amp;author=P.%20Tiffin&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol&amp;volume=4&amp;pages=726-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22554552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">K.A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Arabidopsis meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters,</div> <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">1327</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20meiotic%20crossover%20hot%20spots%20overlap%20with%20H2A.Z%20nucleosomes%20at%20gene%20promoters%2C&amp;author=K.%20Choi&amp;author=X.%20Zhao&amp;author=K.A.%20Kelly&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=1327-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2766"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24056716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Bernatavichute</div> <div class="given-names">Y.V.</div></div>, <div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pellegrini</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide analysis of mono-, di-and trimethylation of histone H3 lysine 4 in <em>Arabidopsis thaliana</em>,</div> <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20mono-%2C%20di-and%20trimethylation%20of%20histone%20H3%20lysine%204%20in%20Arabidopsis%20thaliana%2C&amp;author=X.%20Zhang&amp;author=Y.V.%20Bernatavichute&amp;author=S.%20Cokus&amp;author=M.%20Pellegrini&amp;author=S.E.%20Jacobsen&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-6-r62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-6-r62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-6-r62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19508735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shilo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Melamed-Bessudo</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Dorone</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Barkai</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Levy</div> <div class="given-names">A.A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">DNA crossover motifs associated with epigenetic modifications delineate open chromatin regions in Arabidopsis,</div> <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">2427</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20crossover%20motifs%20associated%20with%20epigenetic%20modifications%20delineate%20open%20chromatin%20regions%20in%20Arabidopsis%2C&amp;author=S.%20Shilo&amp;author=C.%20Melamed-Bessudo&amp;author=Y.%20Dorone&amp;author=N.%20Barkai&amp;author=A.A.%20Levy&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=2427-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.15.00391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.15.00391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.15.00391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26381163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Melamed-Bessudo</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Shilo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Levy</div> <div class="given-names">A.A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Meiotic recombination and genome evolution in plants,</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">30</div>, <div class="fpage">82</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiotic%20recombination%20and%20genome%20evolution%20in%20plants%2C&amp;author=C.%20Melamed-Bessudo&amp;author=S.%20Shilo&amp;author=A.A.%20Levy&amp;publication_year=2016&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=30&amp;pages=82-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2016.02.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2016.02.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2016.02.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26939088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Great majority of recombination events in Arabidopsis are gene conversion events,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">109</div>, <div class="fpage">20992</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Great%20majority%20of%20recombination%20events%20in%20Arabidopsis%20are%20gene%20conversion%20events%2C&amp;author=S.%20Yang&amp;author=Y.%20Yuan&amp;author=L.%20Wang&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=109&amp;pages=20992-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1211827110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1211827110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1211827110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23213238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qi</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Copenhaver</div> <div class="given-names">G.P.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Detection of genomic variations and DNA polymorphisms and impact on analysis of meiotic recombination and genetic mapping,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">111</div>, <div class="fpage">10007</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20genomic%20variations%20and%20DNA%20polymorphisms%20and%20impact%20on%20analysis%20of%20meiotic%20recombination%20and%20genetic%20mapping%2C&amp;author=J.%20Qi&amp;author=Y.%20Chen&amp;author=G.P.%20Copenhaver&amp;author=H.%20Ma&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=111&amp;pages=10007-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1321897111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1321897111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1321897111"> </span></div> </div></div></div></div></div><div content-id="dsx024-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grandillo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ku</div> <div class="given-names">H.M.</div></div>, <div class="name string-name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Identifying the loci responsible for natural variation in fruit size and shape in tomato,</div> <div class="source">Theor. Appl. Genet</div> ., <div class="volume">99</div>, <div class="fpage">978</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20the%20loci%20responsible%20for%20natural%20variation%20in%20fruit%20size%20and%20shape%20in%20tomato%2C&amp;author=S.%20Grandillo&amp;author=H.M.%20Ku&amp;author=S.D.%20Tanksley&amp;publication_year=1999&amp;journal=Theor.%20Appl.%20Genet&amp;volume=99&amp;pages=978-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051405"> </span></div> </div></div></div></div></div><div content-id="dsx024-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">F.Q.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">G.Y.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">RFLP mapping of QTLs conferring cold tolerance during seed germination in an interspecific cross of tomato</div>, <div class="source">Mol. Breed</div> ., <div class="volume">4</div>, <div class="fpage">519</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP%20mapping%20of%20QTLs%20conferring%20cold%20tolerance%20during%20seed%20germination%20in%20an%20interspecific%20cross%20of%20tomato&amp;author=M.R.%20Foolad&amp;author=F.Q.%20Chen&amp;author=G.Y.%20Lin&amp;publication_year=1998&amp;journal=Mol.%20Breed&amp;volume=4&amp;pages=519-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1009648112491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1009648112491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1009648112491"> </span></div> </div></div></div></div></div><div content-id="dsx024-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">F.Q.</div></div>, <div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Hyman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Clair</div> <div class="given-names">D.A.S.</div></div>, <div class="name string-name"><div class="surname">Beelaman</div> <div class="given-names">R.B.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Mapping of QTLs for lycopene and other fruit traits in a Lycopersicon esculentum × L. pimpinellifolium cross and comparison of QTLs across tomato species</div>, <div class="source">Mol. Breed</div> ., <div class="volume">5</div>, <div class="fpage">283</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20QTLs%20for%20lycopene%20and%20other%20fruit%20traits%20in%20a%20Lycopersicon%20esculentum%20%C3%97%20L.%20pimpinellifolium%20cross%20and%20comparison%20of%20QTLs%20across%20tomato%20species&amp;author=F.Q.%20Chen&amp;author=M.R.%20Foolad&amp;author=J.%20Hyman&amp;author=D.A.S.%20Clair&amp;author=R.B.%20Beelaman&amp;publication_year=1999&amp;journal=Mol.%20Breed&amp;volume=5&amp;pages=283-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1009656910457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1009656910457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1009656910457"> </span></div> </div></div></div></div></div><div content-id="dsx024-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashrafi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kinkade</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">A new genetic linkage map of tomato based on a Solanum lycopersicum × S. pimpinellifolium RIL population displaying locations of candidate pathogen response genes</div>, <div class="source">Genome Natl. Res. Counc. Can. Génome Cons. Natl. Rech. Can</div> ., <div class="volume">52</div>, <div class="fpage">935</div>–<div class="lpage">56</div>.</div></div></div></div><div content-id="dsx024-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Víquez-Zamora</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Caro</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Finkers</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Mapping in the era of sequencing: high density genotyping and its application for mapping TYLCV resistance in Solanum pimpinellifolium,</div> <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1152</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20in%20the%20era%20of%20sequencing%3A%20high%20density%20genotyping%20and%20its%20application%20for%20mapping%20TYLCV%20resistance%20in%20Solanum%20pimpinellifolium%2C&amp;author=M.%20V%C3%ADquez-Zamora&amp;author=M.%20Caro&amp;author=R.%20Finkers&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-1152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-1152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25526885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ganal</div> <div class="given-names">M.W.</div></div>, <div class="name string-name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Recombination around the Tm2a and Mi resistance genes in different crosses of <em>Lycopersicon peruvianum</em>,</div> <div class="source">Theor. Appl. Genet</div> ., <div class="volume">92</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20around%20the%20Tm2a%20and%20Mi%20resistance%20genes%20in%20different%20crosses%20of%20Lycopersicon%20peruvianum%2C&amp;author=M.W.%20Ganal&amp;author=S.D.%20Tanksley&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet&amp;volume=92&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24166123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Genome mapping and molecular breeding of tomato,</div> <div class="source">Int. J. Plant Genomics</div> , <div class="volume">2007</div>.</div></div></div></div><div content-id="dsx024-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liharska</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wordragen</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kammen</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zabel</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Koornneef</div> <div class="given-names">M.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Tomato chromosome 6: effect of alien chromosomal segments on recombinant frequencies,</div> <div class="source">Genome</div> , <div class="volume">39</div>, <div class="fpage">485</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20chromosome%206%3A%20effect%20of%20alien%20chromosomal%20segments%20on%20recombinant%20frequencies%2C&amp;author=T.%20Liharska&amp;author=M.%20Wordragen&amp;author=A.%20Kammen&amp;author=P.%20Zabel&amp;author=M.%20Koornneef&amp;publication_year=1996&amp;journal=Genome&amp;volume=39&amp;pages=485-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g96-062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg96-062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg96-062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">635</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tomato%20genome%20sequence%20provides%20insights%20into%20fleshy%20fruit%20evolution&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;author=H.%20Hirakawa&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=635-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">L.K.</div></div>, <div class="name string-name"><div class="surname">Covey</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Larsen</div> <div class="given-names">L.R.</div></div>, <div class="name string-name"><div class="surname">Bedinger</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Stack</div> <div class="given-names">S.M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Structural differences in chromosomes distinguish species in the tomato clade,</div> <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">129</div>, <div class="fpage">24</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20differences%20in%20chromosomes%20distinguish%20species%20in%20the%20tomato%20clade%2C&amp;author=L.K.%20Anderson&amp;author=P.A.%20Covey&amp;author=L.R.%20Larsen&amp;author=P.%20Bedinger&amp;author=S.M.%20Stack&amp;publication_year=2010&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=129&amp;pages=24-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000313850" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000313850" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000313850"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20551606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="collab">The 100 Tomato Genome Sequencing Consortium</div>, <div class="person-group"><div class="name string-name"><div class="surname">Aflitos</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schijlen</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Exploring genetic variation in the tomato (<em>Solanum</em> section <em>Lycopersicon</em>) clade by whole-genome sequencing,</div> <div class="source">Plant J</div> ., <div class="volume">80</div>, <div class="fpage">136</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20genetic%20variation%20in%20the%20tomato%20(Solanum%20section%20Lycopersicon)%20clade%20by%20whole-genome%20sequencing%2C&amp;author=S.%20Aflitos&amp;author=E.%20Schijlen&amp;publication_year=2014&amp;journal=Plant%20J&amp;volume=80&amp;pages=136-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.80.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.80.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.80.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25039268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows–Wheeler transform,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform%2C&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DePristo</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data,</div> <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data%2C&amp;author=M.A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Semagn</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Babu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hearne</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Single nucleotide polymorphism genotyping using Kompetitive Allele Specific PCR (KASP): overview of the technology and its application in crop improvement,</div> <div class="source">Mol. Breed</div> ., <div class="volume">33</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20genotyping%20using%20Kompetitive%20Allele%20Specific%20PCR%20(KASP)%3A%20overview%20of%20the%20technology%20and%20its%20application%20in%20crop%20improvement%2C&amp;author=K.%20Semagn&amp;author=R.%20Babu&amp;author=S.%20Hearne&amp;author=M.%20Olsen&amp;publication_year=2014&amp;journal=Mol.%20Breed&amp;volume=33&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-013-9917-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-013-9917-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-013-9917-x"> </span></div> </div></div></div></div></div><div content-id="dsx024-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Versatile and open software for comparing large genomes,</div> <div class="source">Genome Biol</div> ., <div class="volume">5</div>, <div class="fpage">R12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes%2C&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Boden</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Buske</div> <div class="given-names">F.A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">MEME Suite: tools for motif discovery and searching,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">W202</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEME%20Suite%3A%20tools%20for%20motif%20discovery%20and%20searching%2C&amp;author=T.L.%20Bailey&amp;author=M.%20Boden&amp;author=F.A.%20Buske&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=W202-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp335" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp335" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp335"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19458158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Elkan</div> <div class="given-names">C.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers,</div> <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol</div> ., <div class="volume">2</div>, <div class="fpage">28</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fitting%20a%20mixture%20model%20by%20expectation%20maximization%20to%20discover%20motifs%20in%20biopolymers%2C&amp;author=T.L.%20Bailey&amp;author=C.%20Elkan&amp;publication_year=1994&amp;journal=Proc.%20Int.%20Conf.%20Intell.%20Syst.%20Mol.%20Biol&amp;volume=2&amp;pages=28-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McLeay</div> <div class="given-names">R.C.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Motif Enrichment Analysis: a unified framework and an evaluation on ChIP data,</div> <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Motif%20Enrichment%20Analysis%3A%20a%20unified%20framework%20and%20an%20evaluation%20on%20ChIP%20data%2C&amp;author=R.C.%20McLeay&amp;author=T.L.%20Bailey&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20356413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Mueller</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Giovannoni</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Euchromatin and pericentromeric heterochromatin: comparative composition in the tomato genome</div>, <div class="source">Genetics</div> , <div class="volume">172</div>, <div class="fpage">2529</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Euchromatin%20and%20pericentromeric%20heterochromatin%3A%20comparative%20composition%20in%20the%20tomato%20genome&amp;author=Y.%20Wang&amp;author=X.%20Tang&amp;author=Z.%20Cheng&amp;author=L.%20Mueller&amp;author=J.%20Giovannoni&amp;author=S.D.%20Tanksley&amp;publication_year=2006&amp;journal=Genetics&amp;volume=172&amp;pages=2529-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.055772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.055772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.055772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16489216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Muruganujan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Casagrande</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">P.D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Large-scale gene function analysis with the PANTHER classification system,</div> <div class="source">Nat. Protoc</div> ., <div class="volume">8</div>, <div class="fpage">1551</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20gene%20function%20analysis%20with%20the%20PANTHER%20classification%20system%2C&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=J.T.%20Casagrande&amp;author=P.D.%20Thomas&amp;publication_year=2013&amp;journal=Nat.%20Protoc&amp;volume=8&amp;pages=1551-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2013.092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2013.092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2013.092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23868073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andolfo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Jupe</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Witek</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Etherington</div> <div class="given-names">G.J.</div></div>, <div class="name string-name"><div class="surname">Ercolano</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J.D.G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Defining the full tomato NB-LRR resistance gene repertoire using genomic and cDNA RenSeq,</div> <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="fpage">120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20full%20tomato%20NB-LRR%20resistance%20gene%20repertoire%20using%20genomic%20and%20cDNA%20RenSeq%2C&amp;author=G.%20Andolfo&amp;author=F.%20Jupe&amp;author=K.%20Witek&amp;author=G.J.%20Etherington&amp;author=M.R.%20Ercolano&amp;author=J.D.G.%20Jones&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-14-120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-14-120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24885638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">P Boer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">van Eeuwijk</div> <div class="given-names">F.A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A general modeling framework for genome ancestral origins in multiparental populations,</div> <div class="source">Genetics</div> , <div class="volume">198</div>, <div class="fpage">87</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20modeling%20framework%20for%20genome%20ancestral%20origins%20in%20multiparental%20populations%2C&amp;author=C.%20Zheng&amp;author=M.%20P%20Boer&amp;author=F.A.%20van%20Eeuwijk&amp;publication_year=2014&amp;journal=Genetics&amp;volume=198&amp;pages=87-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.114.163006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.114.163006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.114.163006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25236451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shearer</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">L.K.</div></div>, <div class="name string-name"><div class="surname">de Jong</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Fluorescence in situ hybridization and optical mapping to correct scaffold arrangement in the tomato genome,</div> <div class="source">G3 GenesGenomesGenetics</div> , <div class="volume">4</div>, <div class="fpage">1395</div>–<div class="lpage">405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescence%20in%20situ%20hybridization%20and%20optical%20mapping%20to%20correct%20scaffold%20arrangement%20in%20the%20tomato%20genome%2C&amp;author=L.A.%20Shearer&amp;author=L.K.%20Anderson&amp;author=H.%20de%20Jong&amp;publication_year=2014&amp;journal=G3%20GenesGenomesGenetics&amp;volume=4&amp;pages=1395-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.011197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.011197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.011197"> </span></div> </div></div></div></div></div><div content-id="dsx024-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ibañez</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Carbonell</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Involvement of a citrus meiotic recombination TTC-repeat motif in the formation of gross deletions generated by ionizing radiation and MULE activation,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20a%20citrus%20meiotic%20recombination%20TTC-repeat%20motif%20in%20the%20formation%20of%20gross%20deletions%20generated%20by%20ionizing%20radiation%20and%20MULE%20activation%2C&amp;author=J.%20Terol&amp;author=V.%20Iba%C3%B1ez&amp;author=J.%20Carbonell&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1280-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1280-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1280-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25758634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Boer</div> <div class="given-names">C.G.</div></div>, <div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">T.R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Poly-dA:dT tracts form an in vivo nucleosomal turnstile,</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e110479</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poly-dA%3AdT%20tracts%20form%20an%20in%20vivo%20nucleosomal%20turnstile%2C&amp;author=C.G.%20de%20Boer&amp;author=T.R.%20Hughes&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e110479" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0110479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0110479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0110479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25353956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">James</div> <div class="given-names">G.V.</div></div>, <div class="name string-name"><div class="surname">Patel</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Nordström</div> <div class="given-names">K.J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">User guide for mapping-by-sequencing in Arabidopsis,</div> <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=User%20guide%20for%20mapping-by-sequencing%20in%20Arabidopsis%2C&amp;author=G.V.%20James&amp;author=V.%20Patel&amp;author=K.J.%20Nordstr%C3%B6m&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23773572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Dissecting meiotic recombination based on tetrad analysis by single-microspore sequencing in maize,</div> <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">6648</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissecting%20meiotic%20recombination%20based%20on%20tetrad%20analysis%20by%20single-microspore%20sequencing%20in%20maize%2C&amp;author=X.%20Li&amp;author=L.%20Li&amp;author=J.%20Yan&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=6648" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms7648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms7648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms7648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25800954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hellsten</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Fine-scale variation in meiotic recombination in Mimulus inferred from population shotgun sequencing,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">110</div>, <div class="fpage">19478</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20variation%20in%20meiotic%20recombination%20in%20Mimulus%20inferred%20from%20population%20shotgun%20sequencing%2C&amp;author=U.%20Hellsten&amp;author=K.M.%20Wright&amp;author=J.%20Jenkins&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=110&amp;pages=19478-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1319032110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1319032110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1319032110"> </span></div> </div></div></div></div></div><div content-id="dsx024-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Si</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Widely distributed hot and cold spots in meiotic recombination as shown by the sequencing of rice F2 plants,</div> <div class="source">New Phytol</div> ., <div class="volume">206</div>, <div class="fpage">1491</div>–<div class="lpage">502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widely%20distributed%20hot%20and%20cold%20spots%20in%20meiotic%20recombination%20as%20shown%20by%20the%20sequencing%20of%20rice%20F2%20plants%2C&amp;author=W.%20Si&amp;author=Y.%20Yuan&amp;author=J.%20Huang&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=206&amp;pages=1491-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25664766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Reinhard</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Serra</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Recombination rate heterogeneity within Arabidopsis disease resistance genes,</div> <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1006179</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20rate%20heterogeneity%20within%20Arabidopsis%20disease%20resistance%20genes%2C&amp;author=K.%20Choi&amp;author=C.%20Reinhard&amp;author=H.%20Serra&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1006179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1006179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1006179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27415776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Massy</div> <div class="given-names">B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Initiation of meiotic recombination: how and where? Conservation and specificities among eukaryotes,</div> <div class="source">Annu. Rev. Genet</div> ., <div class="volume">47</div>, <div class="fpage">563</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initiation%20of%20meiotic%20recombination%3A%20how%20and%20where%3F%20Conservation%20and%20specificities%20among%20eukaryotes%2C&amp;author=B.%20de%20Massy&amp;publication_year=2013&amp;journal=Annu.%20Rev.%20Genet&amp;volume=47&amp;pages=563-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110711-155423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110711-155423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110711-155423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24050176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Segal</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Widom</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Poly(dA:dT) Tracts: major determinants of nucleosome organization,</div> <div class="source">Curr. Opin. Struct. Biol</div> ., <div class="volume">19</div>, <div class="fpage">65</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poly(dA%3AdT)%20Tracts%3A%20major%20determinants%20of%20nucleosome%20organization%2C&amp;author=E.%20Segal&amp;author=J.%20Widom&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Struct.%20Biol&amp;volume=19&amp;pages=65-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2009.01.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2009.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2009.01.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19208466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kniewel</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation,</div> <div class="source">Cell</div> , <div class="volume">144</div>, <div class="fpage">719</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hierarchical%20combination%20of%20factors%20shapes%20the%20genome-wide%20topography%20of%20yeast%20meiotic%20recombination%20initiation%2C&amp;author=J.%20Pan&amp;author=M.%20Sasaki&amp;author=R.%20Kniewel&amp;publication_year=2011&amp;journal=Cell&amp;volume=144&amp;pages=719-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21376234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vinces</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Legendre</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Caldara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hagihara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Verstrepen</div> <div class="given-names">K.J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Unstable tandem repeats in promoters confer transcriptional evolvability,</div> <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">1213</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unstable%20tandem%20repeats%20in%20promoters%20confer%20transcriptional%20evolvability%2C&amp;author=M.D.%20Vinces&amp;author=M.%20Legendre&amp;author=M.%20Caldara&amp;author=M.%20Hagihara&amp;author=K.J.%20Verstrepen&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=1213-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19478187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx024-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx024-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guillemette</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bataille</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Gévry</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Variant histone H2A.Z is globally localized to the promoters of inactive yeast genes and regulates nucleosome positioning</div>, <div class="source">PLoS Biol</div> ., <div class="volume">3</div>.</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3865558&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s1_dsx024.doc?Expires=2147483647&amp;Signature=P6ILI5PE4Vehg1ttVp9BD~M2S31yKps~0mSyoBG3QJN1FGSKvV-uFv8sDVGK0POISlCRitxvvwXUa0Q5DagjII2ae6bwjBjHyvSJDDHx-~nBkORIstE6e4KkzOUxi7HB-5BdUYMZgp5LbZBhcMtBkFmTSn0UyCIMeAe0ydF53QZAm~EK0MZv6F0aQOY8ciYaImHpqIKWsCMBemZWspMZ70J5MB9K7YE9wJVd97OIsSK-CB~ezIXONqvITNWHQ8T3RYFf40aWmhIlXKAzCZ0kV2yDYOwGkxdtjVT~Ni1b8m30ZQJ-MyyxWhXLG-7aJPXeNACht52cnBRS1Se1-5aOgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s2_dsx024.xlsx?Expires=2147483647&amp;Signature=2mZbRb-Y1HdGLq-9wLpBWvCYXvE71MjfSmbHcsSpxy~CwEWtX6gSNEoPq7ljg79cbaxtOBTcwiEBaAVOGRdTcsPd5I-VW-4uQQtHDMdy-VynIyOnvPtxbvZKy4wvUuzVfNsdtruaiW6Jy6V~8IpnSuZ2Y2a3y35CBlaBhabNugHltbeWIXnitwnCsG8JdoUjXpN0cBwcUvzzt6Y4LfLxrIdilzm6ucfEb3~iBWKk-ypzFhlgYF9DuEsGsGcT2nDT7OvWcFuHzumxtMC0Ve9B80VqhmUXifopu1DS9aiIYUW8WO3dRPZxQ-xIitX1p0I9FWpYPs9uw96ygg6mebGiww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s3_dsx024.xlsx?Expires=2147483647&amp;Signature=o09hobCAWSwVE4~r9WtG~EFFZ0ymwu51OmhcOeQQe6XIZNa13B6DEo6lZEpil8ggYdI3NVwQxMEmaEaN-Y9zW~VllcQYpIA6U2-V5HU~X772EtoGFCOhjsk6cNZdHnGiGb4d5vQp-XNHulZ~8cv4UTFObTrkCUFCdaN2lPm0Z44UrAFlPqQO6qIdZVxLmhcl91bQiof1VBwRgdFQy01XXQs8lpYH3d8uiRCq4dGnZSqanmzp3o-XEduD3XI4oox0sxbGRp7SxC9ifHqE9zBT-aAhy0~deuEMmELLe-rpauCvrW~sHehSDMoqx5eCAfESFgJhxsTAQQi4KVOmegkXzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s4_dsx024.doc?Expires=2147483647&amp;Signature=LNiBpm-eBw9G2oz-ktBffk9-wXbxO69fYj7s-3awk9plBmzMTYP4ByPIgrX0v~X5LZxSMRhjUSLMPZ50EPe91n1z2SosyIcP0a4UTgCMNrMzrG0tnyqWyq2f2j13lgHRG3jOJWpBdXZeZxf7GDnIsr608CWxFePQjEKpegD9pO2ztBtMwOPgRsu2eWoOuYz1ZP7Lrv0qQ58rmNIz5TolXnRhK5KbNOV22E10OWJ8zXbObHy9zMMtHYL0VuYkvraUk2H1k90bVFdL4SK9OgeBtFdWoVjJlpF6SOEN4FSzOmLRuss7ieQ0EllM7~Q9XR3AmTEiIV4Dd8KkzpT-7vkLMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_table_s5_dsx024.doc?Expires=2147483647&amp;Signature=cULqQ-MZwN0dvjpplBEzFv3sWmhIGlwQRj~ZEIFcbCT2p6bltMbPlRTv4mGTE2ZG3LQoQKOn~c-OcY9Ob7uto2uPyI4F3d0KYbfQseEqFpJLwS0yw9lt-iCXMF2IqVLFHF7JF1ozTRW~KPAToIdA~EnR3UbMZW~0t43dJEw1d28f3ciRwf89dHdmJPrUfot~NoeNuPKuK8V4tYLsElBxVIdw3n1yK0ylYa1SN~E7Ig34yOvNc1NwiuS~-DN4kjrszfj5D~V2MyF0fQ6d0qmh1Aqe~7lgVCEZKL9gkXHgGRAwd9IcmG803TfSw3OCParEt05kxpxRzipqZ5bpIA-zjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s1_dsx024.pdf?Expires=2147483647&amp;Signature=03UKJGngcFcIKvKIoyy8CHRSKa3hhxDdZd7E-kzNgMVzD2hIzN3z61Hqp2OxfUbfw61J~sItUwVewHSkRq4okPVxFQ0YLn63DRIwhKQnfotk2XCU2RDo0iUsG6BF8XLEHljO3JcVOsx6OQBAzkXJkefzRHaSyyWVd5wXDDlVOHEpmHD7OUluhm2AjM-oWke7rtYMxjgygMZ0keJEh~q1UFm34qBzBRvYsOZGE6Ne9EVuyPlsIKuzb7tCMjSP2hNmRa~kxkD2MohDZ1oeEgeaGJ7YYdxSSJAqEh172AqNXNAUEwfNl5yYaWmv7Z-HGLpSma-qUMGdZsp1m1ho8RtGAQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s2_dsx024.pdf?Expires=2147483647&amp;Signature=shpFJXwWd8hhmpcJKNC0RhVyqxo63U9QmpJUA1gYFTbU-eq7RHBw5XzNrt18jCcL-tQkTxH1ZIyMp536brkWyYRrBMX7MzDKdiJkRyoleKfRstUaHrO3XBstWAWVz6U-51r-yzHLEKy0~aP-dTm8m3rVvWvE~r7Hns4yNTLlb0dwU-qbPuZmkVfA6Qdv1cYSN9aijoFxqxAYeW-C2fxrJ9nSHPze5ZUGAjavKvuxutHL22KBdh~HGQLijtSII5tT5Q3-WlZkllRbaeW7FK~VWUC0t20iQmnUSYFm5SN16cQvgzk2GGAmT7dgahFKqsGqk4P-5Hi3cn38LvNgsX2fWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx024/1/supplementary_figure_s3_dsx024.pdf?Expires=2147483647&amp;Signature=ISXmWS2oa8YSkf3LLXHysFF6LbJr8kj-UskCn9~sVNjJZ4LgtE0QNyZo6N9Wo9jASL8kZzNhQou1V852JeHTM5ImCpKwERemJDj-xk-YaLWV1ArWXsaQmyKMEPk0ANS7tjifTBwmykvI1ilNwJSOhASAvVuaVmxKuxPmgU3Tp0JaCm65c-K0PCX5MF~FdLOeO-4Wo1ocJiFuRjK63csLS3hGhPbHluM8~TW~vJOVRMs3BAPTSuBQHZchHuJIptMVHh0RGItYfiH0XQXxfke-YRFUes6~K9QnQVxlTGYQeKITc5LMmyjdQTtv1bD4VZK6ILMvWw6Q00I0fhvFwbtHTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,055</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3865558"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f549%2f3865558">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F549%2F3865558" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3865558'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Low-coverage resequencing detects meiotic recombination pattern and features in tomato RILs" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f549%2f3865558">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

