

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/5/523/3857773","@type":"ScholarlyArticle","name":"Genome-wide profiling of  Sus scrofa  circular RNAs across nine organs and three developmental stages","datePublished":"2017-05-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2017-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx022","keywords":["circRNAs","pig","profiling","organs","skeletal muscle"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/5/523/3857773","author":[{"name":"Liang, Guoming","affiliation":"1 State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China   2 Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China   3 Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China","@type":"Person"},{"name":"Yang, Yalan","affiliation":"1 State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China   2 Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China   3 Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China","@type":"Person"},{"name":"Niu, Guanglin","affiliation":"1 State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China","@type":"Person"},{"name":"Tang, Zhonglin","affiliation":"1 State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China   2 Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China   3 Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China","@type":"Person"},{"name":"Li, Kui","affiliation":"1 State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China   2 Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China","@type":"Person"}],"description":"Abstract.  The spatio-temporal expression patterns of Circular RNA (circRNA) across organs and developmental stages are critical for its function and evolution ","pageStart":"523","pageEnd":"535","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide profiling of  Sus scrofa  circular RNAs across nine organs and three developmental stages","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide profiling of  Sus scrofa  circular RNAs across nine organs and three developmental stages" />
    <meta property="og:description" content="Abstract.  The spatio-temporal expression patterns of Circular RNA (circRNA) across organs and developmental stages are critical for its function and evolution " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/5/523/3857773" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-05-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Liang, Guoming" /><meta name="citation_author_institution" content="State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author_institution" content="Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author" content="Yang, Yalan" /><meta name="citation_author_institution" content="State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author_institution" content="Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author" content="Niu, Guanglin" /><meta name="citation_author_institution" content="State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author" content="Tang, Zhonglin" /><meta name="citation_author_institution" content="State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author_institution" content="Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_author" content="Li, Kui" /><meta name="citation_author_institution" content="State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China" /><meta name="citation_title" content="Genome-wide profiling of Sus scrofa circular RNAs across nine organs and three developmental stages" /><meta name="citation_firstpage" content="523" /><meta name="citation_lastpage" content="535" /><meta name="citation_doi" content="10.1093/dnares/dsx022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2017/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Electron microscopic evidence for the circular form of RNA in the cytoplasm of eukaryotic cells,;  citation_journal_title=Nature;  citation_year=1979;  citation_volume=280;  citation_pages=339-40; " /><meta name="citation_reference" content="citation_title=Circular RNAs are a large class of animal RNAs with regulatory potency,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=333-8; " /><meta name="citation_reference" content="citation_title=Complementary sequence-mediated exon circularization,;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=159;  citation_pages=134-47; " /><meta name="citation_reference" content="citation_title=Expanded identification and characterization of mammalian circular RNAs,;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=014-0409; " /><meta name="citation_reference" content="citation_title=Single-cell RNA-seq transcriptome analysis of linear and circular RNAs in mouse preimplantation embryos,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=148" /><meta name="citation_reference" content="  Ye C.-Y., Chen L., Liu C., Zhu Q.-H., Fan L.  2015 , Widespread noncoding circular RNAs in plants, New Phytol. , 208, 88–95. " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Drosophila circular RNAs reveals their structural and sequence properties and age-dependent neural accumulation,;  citation_journal_title=Cell Rep;  citation_year=2014;  citation_volume=9;  citation_pages=15" /><meta name="citation_reference" content="citation_title=Spatio-temporal regulation of circular RNA expression during porcine embryonic brain development,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=015-0801; " /><meta name="citation_reference" content="citation_title=Statistically based splicing detection reveals neural enrichment and tissue-specific induction of circular RNA during human fetal development,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=015-0690; " /><meta name="citation_reference" content="citation_title=Natural RNA circles function as efficient microRNA sponges,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=384-8; " /><meta name="citation_reference" content="citation_title=Exon-intron circular RNAs regulate transcription in the nucleus,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2015;  citation_volume=22;  citation_pages=256-64; " /><meta name="citation_reference" content="citation_title=A molecular timescale for vertebrate evolution,;  citation_journal_title=Nature;  citation_year=1998;  citation_volume=392;  citation_pages=917-20; " /><meta name="citation_reference" content="citation_title=Worldwide phylogeography of wild boar reveals multiple centers of pig domestication,;  citation_journal_title=Science;  citation_year=2005;  citation_volume=307;  citation_pages=1618-21; " /><meta name="citation_reference" content="citation_title=Analyses of pig genomes provide insight into porcine demography and evolution,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=393-8; " /><meta name="citation_reference" content="citation_title=, Pigs in sequence space: a 0.66 X coverage pig genome survey based on shotgun sequencing,;  citation_journal_title=BMC Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Comparative gene expression signature of pig, human and mouse induced pluripotent stem cell lines reveals insight into pig pluripotency gene networks,;  citation_journal_title=Stem Cell Rev. Rep;  citation_year=2014;  citation_volume=10;  citation_pages=162-76; " /><meta name="citation_reference" content="citation_title=Are animal models as good as we think?,;  citation_journal_title=Theriogenology;  citation_year=2008;  citation_volume=69;  citation_pages=2-9; " /><meta name="citation_reference" content="citation_title=Completion of the swine genome will simplify the production of swine as a large animal biomedical model,;  citation_journal_title=BMC Med Genomics;  citation_year=2012;  citation_volume=5;  citation_pages=55" /><meta name="citation_reference" content="citation_title=Advances in swine transcriptomics,;  citation_journal_title=Int. J. Biol. Sci;  citation_year=2007;  citation_volume=3;  citation_pages=132-52; " /><meta name="citation_reference" content="citation_title=Porcine transcriptome analysis based on 97 non-normalized cDNA libraries and assembly of 1,021,891 expressed sequence tags,;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=R45" /><meta name="citation_reference" content="citation_title=EST-based gene discovery in pig: virtual expression patterns and comparative mapping to human,;  citation_journal_title=Mamm. Genome;  citation_year=2003;  citation_volume=14;  citation_pages=565-79; " /><meta name="citation_reference" content="citation_title=Dynamic transcriptome profiles of skeletal muscle tissue across 11 developmental stages for both Tongcheng and Yorkshire pigs,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=377" /><meta name="citation_reference" content="citation_title=LongSAGE analysis of skeletal muscle at three prenatal stages in Tongcheng and Landrace pigs,;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=R115" /><meta name="citation_reference" content="citation_title=Comparison of skeletal muscle miRNA and mRNA profiles among three pig breeds,;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2015;  citation_volume=291;  citation_pages=, 559-15; " /><meta name="citation_reference" content="citation_title=Integrated analysis of miRNA and mRNA paired expression profiling of prenatal skeletal muscle development in three genotype pigs,;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=15544" /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2,;  citation_journal_title=Nat. Meth;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2987-93; " /><meta name="citation_reference" content="citation_title=circBase: a database for circular RNAs,;  citation_journal_title=RNA;  citation_year=2014;  citation_volume=20;  citation_pages=1666-70; " /><meta name="citation_reference" content="citation_title=miRBase: annotating high confidence microRNAs using deep sequencing data,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D68-73; " /><meta name="citation_reference" content="citation_title=MicroTar: predicting microRNA targets from RNA duplexes,;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7 Suppl 5,;  citation_pages=S20" /><meta name="citation_reference" content="citation_title=Human microRNA targets,;  citation_journal_title=PLoS Biol;  citation_year=2004;  citation_volume=2;  citation_pages=e363" /><meta name="citation_reference" content="citation_title=RNAhybrid: microRNA target prediction easy, fast and flexible,;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W451-4; " /><meta name="citation_reference" content="citation_title=Cell-type specific features of circular RNA expression;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003777" /><meta name="citation_reference" content="citation_title=Circular RNAs are abundant, conserved, and associated with ALU repeats,;  citation_journal_title=RNA;  citation_year=2013;  citation_volume=19;  citation_pages=426" /><meta name="citation_reference" content="citation_title=Circular RNAs in the mammalian brain are highly abundant, conserved, and dynamically expressed,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=58;  citation_pages=870-85; " /><meta name="citation_reference" content="citation_title=Tracing the expression of circular RNAs in human pre-implantation embryos,;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=circRNA biogenesis competes with pre-mRNA splicing,;  citation_journal_title=Mol. Cell;  citation_year=2014;  citation_volume=56;  citation_pages=55-66; " /><meta name="citation_reference" content="citation_title=Short intronic repeat sequences facilitate circular RNA production,;  citation_journal_title=Genes Dev;  citation_year=2014;  citation_volume=28;  citation_pages=2233-47; " /><meta name="citation_reference" content="citation_title=Exon circularization requires canonical splice signals;  citation_journal_title=Cell Rep;  citation_volume=10;  citation_pages=103-111; " /><meta name="citation_reference" content="citation_title=Classification of introns: U2-type or U12-type,;  citation_journal_title=Cell;  citation_volume=91;  citation_pages=875-9; " /><meta name="citation_reference" content="citation_title=Mechanism of Dis3l2 substrate recognition in the Lin28-let-7 pathway,;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=514;  citation_pages=252-6; " /><meta name="citation_reference" content="citation_title=The exoribonuclease Dis3L2 defines a novel eukaryotic RNA degradation pathway,;  citation_journal_title=EMBO J;  citation_year=2013;  citation_volume=32;  citation_pages=1842-54; " /><meta name="citation_reference" content="citation_title=The let-7g microRNA promotes follicular granulosa cell apoptosis by targeting transforming growth factor-β type 1 receptor,;  citation_journal_title=Mol Cell. Endocrinol;  citation_year=2015;  citation_volume=409;  citation_pages=103-12; " /><meta name="citation_reference" content="citation_title=Circular RNA profiling reveals an abundant circHIPK3 that regulates cell growth by sponging multiple miRNAs,;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7,;  citation_pages=11215" /><meta name="citation_reference" content="citation_title=Diversifying microRNA sequence and function,;  citation_journal_title=Nat. Rev. Mol. Cell Biol;  citation_year=2013;  citation_volume=14;  citation_pages=475-88; " /><meta name="citation_reference" content="citation_title=Biogenesis, identification, and function of exonic circular RNAs,;  citation_journal_title=Wiley Interdiscip. Rev. RNA;  citation_year=2015;  citation_volume=6;  citation_pages=563-79; " /><meta name="citation_reference" content="citation_title=A low prevalence of MYH7/MYBPC3 mutations among Familial Hypertrophic Cardiomyopathy patients in India,;  citation_journal_title=Mol. Cell. Biochem;  citation_year=2012;  citation_volume=360;  citation_pages=373-82; " /><meta name="citation_reference" content="citation_title=Coding sequence rare variants identified in MYBPC3, MYH6, TPM1, TNNC1, and TNNI3 from 312 patients with familial or idiopathic dilated cardiomyopathy,;  citation_journal_title=Circ. Cardiovasc. Genet;  citation_year=2010;  citation_volume=3;  citation_pages=155-61; " /><meta name="citation_reference" content="citation_title=Identification of tissue-specific microRNAs from mouse,;  citation_journal_title=Curr Biol;  citation_year=2002;  citation_volume=12;  citation_pages=735-9; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of tissue-specific microRNA and transcription factor regulatory networks in 12 tissues,;  citation_journal_title=Sci. Rep;  citation_year=2014;  citation_volume=4;  citation_pages=5150" /><meta name="citation_reference" content="citation_title=Proteomics. Tissue-based map of the human proteome,;  citation_journal_title=Science;  citation_year=2015;  citation_volume=347;  citation_pages=1260419" /><meta name="citation_reference" content="citation_title=Effects of divergent selection for 8-week body weight on postnatal enzyme activity pattern of 3 fiber types in fast muscles of male broilers (Gallus gallus domesticus),;  citation_journal_title=Poult. Sci;  citation_year=2010;  citation_volume=89;  citation_pages=2651-9; " /><meta name="citation_reference" content="citation_title=Aging differentially affects human skeletal muscle microRNA expression at rest and after an anabolic stimulus of resistance exercise and essential amino acids,;  citation_journal_title=Am. J. Physiol. –Endocrinol. Metab;  citation_year=2008;  citation_volume=295;  citation_pages=E1333-40; " /><meta name="citation_reference" content="citation_title=Role of microRNAs in skeletal muscle hypertrophy,;  citation_journal_title=Front. Physiol,;  citation_year=2014;  citation_volume=4;  citation_pages=408" /><meta name="citation_reference" content="citation_title=Relating biochemistry and function in the myosin superfamily,;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2004;  citation_volume=16;  citation_pages=61-7; " /><meta name="citation_reference" content="citation_title=Fibre type-specific expression patterns of myosin heavy chain genes in adult torafugu Takifugu rubripes muscles,;  citation_journal_title=J. Exp. Biol;  citation_year=2010;  citation_volume=213;  citation_pages=137-45; " /><meta name="citation_reference" content="citation_title=Effects of intergenic single nucleotide polymorphisms in the fast myosin heavy chain cluster on muscle fiber characteristics and meat quality in Berkshire pigs,;  citation_journal_title=Meat Sci;  citation_year=2015;  citation_volume=110;  citation_pages=224-9; " /><meta name="citation_reference" content="citation_title=Moderately increased maternal dietary energy intake delays foetal skeletal muscle differentiation and maturity in pigs;  citation_journal_title=Eur. J. Nutr;  citation_year=2015;  citation_volume=55;  citation_pages=1777-87; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis reveals genetic loci and candidate genes for meat quality traits in Chinese Laiwu pigs,;  citation_journal_title=Mamm. Genome;  citation_year=2015;  citation_volume=26;  citation_pages=181-90; " /><meta name="citation_reference" content="citation_title=CIRI: an efficient and unbiased algorithm for de novo circular RNA identification,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=4" /><meta name="citation_reference" content="citation_title=CircNet: a database of circular RNAs derived from transcriptome sequencing data,;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=44;  citation_pages=D209-15; " /><meta name="citation_reference" content="citation_title=Circular transcripts of the testis-determining gene &lt;em&gt;Sry&lt;/em&gt; in adult mouse testis,;  citation_journal_title=Cell;  citation_volume=73;  citation_pages=1019-30; " /><meta name="citation_reference" content="citation_title=Transcriptome-wide investigation of circular RNAs in rice,;  citation_journal_title=RNA;  citation_year=2015;  citation_volume=21;  citation_pages=2076-87; " /><meta name="citation_reference" content="citation_title=Detecting and characterizing circular RNAs,;  citation_journal_title=Nat. Biotech;  citation_year=2014;  citation_volume=32;  citation_pages=453-61; " /><meta name="citation_reference" content="citation_title=Mutations of the cardiac ryanodine receptor (RyR2) gene in familial polymorphic ventricular tachycardia,;  citation_journal_title=Circulation;  citation_year=2001;  citation_volume=103;  citation_pages=485-90; " /><meta name="citation_reference" content="citation_title=Mutations in the slow skeletal muscle fiber myosin heavy chain gene (&lt;em&gt;MYH7&lt;/em&gt;) cause laing early-onset distal myopathy (MPD1),;  citation_journal_title=Am. J. Hum. Genet;  citation_volume=75;  citation_pages=703-8; " /><meta name="citation_reference" content="citation_title=Cardiomyopathy: a systematic review of disease-causing mutations in myosin heavy chain 7 and their phenotypic manifestations,;  citation_journal_title=Cardiology;  citation_year=2010;  citation_volume=115;  citation_pages=49-60; " /><meta name="citation_reference" content="citation_title=Mechanisms regulating skeletal muscle growth and atrophy,;  citation_journal_title=FEBS J;  citation_year=2013;  citation_volume=280;  citation_pages=4294-314; " /><meta name="citation_reference" content="citation_title=Tight junctions and the regulation of gene expression,;  citation_journal_title=Semin. Cell Dev. Biol;  citation_year=2014;  citation_volume=36;  citation_pages=213-23; " /><meta name="citation_reference" content="citation_title=Role of tight junctions in cell proliferation and cancer,;  citation_journal_title=Prog. Histochem. Cytochem;  citation_year=2007;  citation_volume=42;  citation_pages=1-57; " /><meta name="citation_reference" content="citation_title=Circular RNAs are the predominant transcript isoform from hundreds of human genes in diverse cell types;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30733" /><meta name="citation_reference" content="citation_title=Circ2Traits: a comprehensive database for circular RNA potentially associated with disease and traits,;  citation_journal_title=Front. Genet;  citation_year=2013;  citation_volume=4;  citation_pages=283" /><meta name="citation_reference" content="citation_title=deepBase v2.0: identification, expression, evolution and function of small RNAs, LncRNAs and circular RNAs from deep-sequencing data,;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D196-202; " /><meta name="citation_reference" content="citation_title=CircInteractome: a web tool for exploring circular RNAs and their interacting proteins and microRNAs;  citation_journal_title=RNA Biol;  citation_year=2015;  citation_volume=13;  citation_pages=34-42; " /><meta name="citation_reference" content="citation_title=SomamiR 2.0: a database of cancer somatic mutations altering microRNA–ceRNA interactions,;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D1005-10; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/5/523/22893199/dsx022.pdf" /><meta name="description" content="Abstract.  The spatio-temporal expression patterns of Circular RNA (circRNA) across organs and developmental stages are critical for its function and evolution " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/5/523/3857773" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/5/523/3857773" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/5/523/3857773");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx022';
    </script>
<title>
	Genome-wide profiling of Sus scrofa circular RNAs across nine organs and three developmental stages | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3857773" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcywATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlhbmcsIEd1b21pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTnV0cml0aW9uLCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGlnIEdlbm9taWMgRGVzaWduIGFuZCBCcmVlZGluZywgQWdyaWN1bHR1cmFsIEdlbm9tZSBJbnN0aXR1dGUgYXQgU2hlbnpoZW4sIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuIDUxODEyNCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaGVuemhlbiBLZXkgTGFib3JhdG9yeSBvZiBQaGVub3R5cGUgQW5hbHlzaXMgYW5kIFV0aWxpemF0aW9uIG9mIEFncmljdWx0dXJhbCBHZW5vbWUsIEFncmljdWx0dXJhbCBHZW5vbWUgSW5zdGl0dXRlIGF0IFNoZW56aGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBTaGVuemhlbiA1MTgxMjQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgWWFsYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTnV0cml0aW9uLCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGlnIEdlbm9taWMgRGVzaWduIGFuZCBCcmVlZGluZywgQWdyaWN1bHR1cmFsIEdlbm9tZSBJbnN0aXR1dGUgYXQgU2hlbnpoZW4sIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuIDUxODEyNCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaGVuemhlbiBLZXkgTGFib3JhdG9yeSBvZiBQaGVub3R5cGUgQW5hbHlzaXMgYW5kIFV0aWxpemF0aW9uIG9mIEFncmljdWx0dXJhbCBHZW5vbWUsIEFncmljdWx0dXJhbCBHZW5vbWUgSW5zdGl0dXRlIGF0IFNoZW56aGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBTaGVuemhlbiA1MTgxMjQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTml1LCBHdWFuZ2xpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEFuaW1hbCBOdXRyaXRpb24sIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQmVpamluZyAxMDAxOTMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuZywgWmhvbmdsaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBBbmltYWwgTnV0cml0aW9uLCBJbnN0aXR1dGUgb2YgQW5pbWFsIFNjaWVuY2UsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGlnIEdlbm9taWMgRGVzaWduIGFuZCBCcmVlZGluZywgQWdyaWN1bHR1cmFsIEdlbm9tZSBJbnN0aXR1dGUgYXQgU2hlbnpoZW4sIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuIDUxODEyNCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaGVuemhlbiBLZXkgTGFib3JhdG9yeSBvZiBQaGVub3R5cGUgQW5hbHlzaXMgYW5kIFV0aWxpemF0aW9uIG9mIEFncmljdWx0dXJhbCBHZW5vbWUsIEFncmljdWx0dXJhbCBHZW5vbWUgSW5zdGl0dXRlIGF0IFNoZW56aGVuLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBTaGVuemhlbiA1MTgxMjQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIEt1aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEFuaW1hbCBOdXRyaXRpb24sIEluc3RpdHV0ZSBvZiBBbmltYWwgU2NpZW5jZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQmVpamluZyAxMDAxOTMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQaWcgR2Vub21pYyBEZXNpZ24gYW5kIEJyZWVkaW5nLCBBZ3JpY3VsdHVyYWwgR2Vub21lIEluc3RpdHV0ZSBhdCBTaGVuemhlbiwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgU2hlbnpoZW4gNTE4MTI0LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lLXdpZGUgcHJvZmlsaW5nIG9mIFN1cyBzY3JvZmEgY2lyY3VsYXIgUk5BcyBhY3Jvc3MgbmluZSBvcmdhbnMgYW5kIHRocmVlIGRldmVsb3BtZW50YWwgc3RhZ2VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1MzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsZWN0cm9uIG1pY3Jvc2NvcGljIGV2aWRlbmNlIGZvciB0aGUgY2lyY3VsYXIgZm9ybSBvZiBSTkEgaW4gdGhlIGN5dG9wbGFzbSBvZiBldWthcnlvdGljIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTc5OyAgY2l0YXRpb25fdm9sdW1lPTI4MDsgIGNpdGF0aW9uX3BhZ2VzPTMzOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BcyBhcmUgYSBsYXJnZSBjbGFzcyBvZiBhbmltYWwgUk5BcyB3aXRoIHJlZ3VsYXRvcnkgcG90ZW5jeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTU7ICBjaXRhdGlvbl9wYWdlcz0zMzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxlbWVudGFyeSBzZXF1ZW5jZS1tZWRpYXRlZCBleG9uIGNpcmN1bGFyaXphdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9MTM0LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBhbmRlZCBpZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBtYW1tYWxpYW4gY2lyY3VsYXIgUk5Bcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MDE0LTA0MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1jZWxsIFJOQS1zZXEgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiBsaW5lYXIgYW5kIGNpcmN1bGFyIFJOQXMgaW4gbW91c2UgcHJlaW1wbGFudGF0aW9uIGVtYnJ5b3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE0OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgWWUgQy4tWS4sIENoZW4gTC4sIExpdSBDLiwgWmh1IFEuLUguLCBGYW4gTC4gIDIwMTUgLCBXaWRlc3ByZWFkIG5vbmNvZGluZyBjaXJjdWxhciBSTkFzIGluIHBsYW50cywgTmV3IFBoeXRvbC4gLCAyMDgsIDg44oCTOTUuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIERyb3NvcGhpbGEgY2lyY3VsYXIgUk5BcyByZXZlYWxzIHRoZWlyIHN0cnVjdHVyYWwgYW5kIHNlcXVlbmNlIHByb3BlcnRpZXMgYW5kIGFnZS1kZXBlbmRlbnQgbmV1cmFsIGFjY3VtdWxhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwYXRpby10ZW1wb3JhbCByZWd1bGF0aW9uIG9mIGNpcmN1bGFyIFJOQSBleHByZXNzaW9uIGR1cmluZyBwb3JjaW5lIGVtYnJ5b25pYyBicmFpbiBkZXZlbG9wbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MDE1LTA4MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsbHkgYmFzZWQgc3BsaWNpbmcgZGV0ZWN0aW9uIHJldmVhbHMgbmV1cmFsIGVucmljaG1lbnQgYW5kIHRpc3N1ZS1zcGVjaWZpYyBpbmR1Y3Rpb24gb2YgY2lyY3VsYXIgUk5BIGR1cmluZyBodW1hbiBmZXRhbCBkZXZlbG9wbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MDE1LTA2OTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgUk5BIGNpcmNsZXMgZnVuY3Rpb24gYXMgZWZmaWNpZW50IG1pY3JvUk5BIHNwb25nZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk1OyAgY2l0YXRpb25fcGFnZXM9Mzg0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4b24taW50cm9uIGNpcmN1bGFyIFJOQXMgcmVndWxhdGUgdHJhbnNjcmlwdGlvbiBpbiB0aGUgbnVjbGV1cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9sZWN1bGFyIHRpbWVzY2FsZSBmb3IgdmVydGVicmF0ZSBldm9sdXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MzkyOyAgY2l0YXRpb25fcGFnZXM9OTE3LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Xb3JsZHdpZGUgcGh5bG9nZW9ncmFwaHkgb2Ygd2lsZCBib2FyIHJldmVhbHMgbXVsdGlwbGUgY2VudGVycyBvZiBwaWcgZG9tZXN0aWNhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA3OyAgY2l0YXRpb25fcGFnZXM9MTYxOC0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzZXMgb2YgcGlnIGdlbm9tZXMgcHJvdmlkZSBpbnNpZ2h0IGludG8gcG9yY2luZSBkZW1vZ3JhcGh5IGFuZCBldm9sdXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9MzkzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPSwgUGlncyBpbiBzZXF1ZW5jZSBzcGFjZTogYSAwLjY2IFggY292ZXJhZ2UgcGlnIGdlbm9tZSBzdXJ2ZXkgYmFzZWQgb24gc2hvdGd1biBzZXF1ZW5jaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2VuZSBleHByZXNzaW9uIHNpZ25hdHVyZSBvZiBwaWcsIGh1bWFuIGFuZCBtb3VzZSBpbmR1Y2VkIHBsdXJpcG90ZW50IHN0ZW0gY2VsbCBsaW5lcyByZXZlYWxzIGluc2lnaHQgaW50byBwaWcgcGx1cmlwb3RlbmN5IGdlbmUgbmV0d29ya3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TdGVtIENlbGwgUmV2LiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNjItNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyZSBhbmltYWwgbW9kZWxzIGFzIGdvb2QgYXMgd2UgdGhpbms/LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlcmlvZ2Vub2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz0yLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRpb24gb2YgdGhlIHN3aW5lIGdlbm9tZSB3aWxsIHNpbXBsaWZ5IHRoZSBwcm9kdWN0aW9uIG9mIHN3aW5lIGFzIGEgbGFyZ2UgYW5pbWFsIGJpb21lZGljYWwgbW9kZWwsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgTWVkIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIHN3aW5lIHRyYW5zY3JpcHRvbWljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gQmlvbC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzItNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcmNpbmUgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBiYXNlZCBvbiA5NyBub24tbm9ybWFsaXplZCBjRE5BIGxpYnJhcmllcyBhbmQgYXNzZW1ibHkgb2YgMSwwMjEsODkxIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVI0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVC1iYXNlZCBnZW5lIGRpc2NvdmVyeSBpbiBwaWc6IHZpcnR1YWwgZXhwcmVzc2lvbiBwYXR0ZXJucyBhbmQgY29tcGFyYXRpdmUgbWFwcGluZyB0byBodW1hbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1hbW0uIEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTU2NS03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyB0cmFuc2NyaXB0b21lIHByb2ZpbGVzIG9mIHNrZWxldGFsIG11c2NsZSB0aXNzdWUgYWNyb3NzIDExIGRldmVsb3BtZW50YWwgc3RhZ2VzIGZvciBib3RoIFRvbmdjaGVuZyBhbmQgWW9ya3NoaXJlIHBpZ3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zNzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nU0FHRSBhbmFseXNpcyBvZiBza2VsZXRhbCBtdXNjbGUgYXQgdGhyZWUgcHJlbmF0YWwgc3RhZ2VzIGluIFRvbmdjaGVuZyBhbmQgTGFuZHJhY2UgcGlncyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1SMTE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBza2VsZXRhbCBtdXNjbGUgbWlSTkEgYW5kIG1STkEgcHJvZmlsZXMgYW1vbmcgdGhyZWUgcGlnIGJyZWVkcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI5MTsgIGNpdGF0aW9uX3BhZ2VzPSwgNTU5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGVkIGFuYWx5c2lzIG9mIG1pUk5BIGFuZCBtUk5BIHBhaXJlZCBleHByZXNzaW9uIHByb2ZpbGluZyBvZiBwcmVuYXRhbCBza2VsZXRhbCBtdXNjbGUgZGV2ZWxvcG1lbnQgaW4gdGhyZWUgZ2Vub3R5cGUgcGlncyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTU0NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDIsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGg7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0YXRpc3RpY2FsIGZyYW1ld29yayBmb3IgU05QIGNhbGxpbmcsIG11dGF0aW9uIGRpc2NvdmVyeSwgYXNzb2NpYXRpb24gbWFwcGluZyBhbmQgcG9wdWxhdGlvbiBnZW5ldGljYWwgcGFyYW1ldGVyIGVzdGltYXRpb24gZnJvbSBzZXF1ZW5jaW5nIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWNpcmNCYXNlOiBhIGRhdGFiYXNlIGZvciBjaXJjdWxhciBSTkFzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTY2Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bWlSQmFzZTogYW5ub3RhdGluZyBoaWdoIGNvbmZpZGVuY2UgbWljcm9STkFzIHVzaW5nIGRlZXAgc2VxdWVuY2luZyBkYXRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1ENjgtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvVGFyOiBwcmVkaWN0aW5nIG1pY3JvUk5BIHRhcmdldHMgZnJvbSBSTkEgZHVwbGV4ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NyBTdXBwbCA1LDsgIGNpdGF0aW9uX3BhZ2VzPVMyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIG1pY3JvUk5BIHRhcmdldHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUzNjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkFoeWJyaWQ6IG1pY3JvUk5BIHRhcmdldCBwcmVkaWN0aW9uIGVhc3ksIGZhc3QgYW5kIGZsZXhpYmxlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XNDUxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGwtdHlwZSBzcGVjaWZpYyBmZWF0dXJlcyBvZiBjaXJjdWxhciBSTkEgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDM3NzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBSTkFzIGFyZSBhYnVuZGFudCwgY29uc2VydmVkLCBhbmQgYXNzb2NpYXRlZCB3aXRoIEFMVSByZXBlYXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NDI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BcyBpbiB0aGUgbWFtbWFsaWFuIGJyYWluIGFyZSBoaWdobHkgYWJ1bmRhbnQsIGNvbnNlcnZlZCwgYW5kIGR5bmFtaWNhbGx5IGV4cHJlc3NlZCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTg3MC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhY2luZyB0aGUgZXhwcmVzc2lvbiBvZiBjaXJjdWxhciBSTkFzIGluIGh1bWFuIHByZS1pbXBsYW50YXRpb24gZW1icnlvcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Y2lyY1JOQSBiaW9nZW5lc2lzIGNvbXBldGVzIHdpdGggcHJlLW1STkEgc3BsaWNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz01NS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvcnQgaW50cm9uaWMgcmVwZWF0IHNlcXVlbmNlcyBmYWNpbGl0YXRlIGNpcmN1bGFyIFJOQSBwcm9kdWN0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjIzMy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhvbiBjaXJjdWxhcml6YXRpb24gcmVxdWlyZXMgY2Fub25pY2FsIHNwbGljZSBzaWduYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFJlcDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsYXNzaWZpY2F0aW9uIG9mIGludHJvbnM6IFUyLXR5cGUgb3IgVTEyLXR5cGUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9ODc1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBEaXMzbDIgc3Vic3RyYXRlIHJlY29nbml0aW9uIGluIHRoZSBMaW4yOC1sZXQtNyBwYXRod2F5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUxNDsgIGNpdGF0aW9uX3BhZ2VzPTI1Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXhvcmlib251Y2xlYXNlIERpczNMMiBkZWZpbmVzIGEgbm92ZWwgZXVrYXJ5b3RpYyBSTkEgZGVncmFkYXRpb24gcGF0aHdheSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTE4NDItNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBsZXQtN2cgbWljcm9STkEgcHJvbW90ZXMgZm9sbGljdWxhciBncmFudWxvc2EgY2VsbCBhcG9wdG9zaXMgYnkgdGFyZ2V0aW5nIHRyYW5zZm9ybWluZyBncm93dGggZmFjdG9yLc6yIHR5cGUgMSByZWNlcHRvciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbCBDZWxsLiBFbmRvY3Jpbm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY3VsYXIgUk5BIHByb2ZpbGluZyByZXZlYWxzIGFuIGFidW5kYW50IGNpcmNISVBLMyB0aGF0IHJlZ3VsYXRlcyBjZWxsIGdyb3d0aCBieSBzcG9uZ2luZyBtdWx0aXBsZSBtaVJOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT03LDsgIGNpdGF0aW9uX3BhZ2VzPTExMjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaWZ5aW5nIG1pY3JvUk5BIHNlcXVlbmNlIGFuZCBmdW5jdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBNb2wuIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ3NS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvZ2VuZXNpcywgaWRlbnRpZmljYXRpb24sIGFuZCBmdW5jdGlvbiBvZiBleG9uaWMgY2lyY3VsYXIgUk5Bcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdpbGV5IEludGVyZGlzY2lwLiBSZXYuIFJOQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NTYzLTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxvdyBwcmV2YWxlbmNlIG9mIE1ZSDcvTVlCUEMzIG11dGF0aW9ucyBhbW9uZyBGYW1pbGlhbCBIeXBlcnRyb3BoaWMgQ2FyZGlvbXlvcGF0aHkgcGF0aWVudHMgaW4gSW5kaWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzYwOyAgY2l0YXRpb25fcGFnZXM9MzczLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2Rpbmcgc2VxdWVuY2UgcmFyZSB2YXJpYW50cyBpZGVudGlmaWVkIGluIE1ZQlBDMywgTVlINiwgVFBNMSwgVE5OQzEsIGFuZCBUTk5JMyBmcm9tIDMxMiBwYXRpZW50cyB3aXRoIGZhbWlsaWFsIG9yIGlkaW9wYXRoaWMgZGlsYXRlZCBjYXJkaW9teW9wYXRoeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNpcmMuIENhcmRpb3Zhc2MuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRpc3N1ZS1zcGVjaWZpYyBtaWNyb1JOQXMgZnJvbSBtb3VzZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTczNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzdXJ2ZXkgb2YgdGlzc3VlLXNwZWNpZmljIG1pY3JvUk5BIGFuZCB0cmFuc2NyaXB0aW9uIGZhY3RvciByZWd1bGF0b3J5IG5ldHdvcmtzIGluIDEyIHRpc3N1ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NTE1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21pY3MuIFRpc3N1ZS1iYXNlZCBtYXAgb2YgdGhlIGh1bWFuIHByb3Rlb21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDc7ICBjaXRhdGlvbl9wYWdlcz0xMjYwNDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBkaXZlcmdlbnQgc2VsZWN0aW9uIGZvciA4LXdlZWsgYm9keSB3ZWlnaHQgb24gcG9zdG5hdGFsIGVuenltZSBhY3Rpdml0eSBwYXR0ZXJuIG9mIDMgZmliZXIgdHlwZXMgaW4gZmFzdCBtdXNjbGVzIG9mIG1hbGUgYnJvaWxlcnMgKEdhbGx1cyBnYWxsdXMgZG9tZXN0aWN1cyksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qb3VsdC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MjY1MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ2luZyBkaWZmZXJlbnRpYWxseSBhZmZlY3RzIGh1bWFuIHNrZWxldGFsIG11c2NsZSBtaWNyb1JOQSBleHByZXNzaW9uIGF0IHJlc3QgYW5kIGFmdGVyIGFuIGFuYWJvbGljIHN0aW11bHVzIG9mIHJlc2lzdGFuY2UgZXhlcmNpc2UgYW5kIGVzc2VudGlhbCBhbWlubyBhY2lkcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBQaHlzaW9sLiDigJNFbmRvY3Jpbm9sLiBNZXRhYjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yOTU7ICBjaXRhdGlvbl9wYWdlcz1FMTMzMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBtaWNyb1JOQXMgaW4gc2tlbGV0YWwgbXVzY2xlIGh5cGVydHJvcGh5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBoeXNpb2wsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxhdGluZyBiaW9jaGVtaXN0cnkgYW5kIGZ1bmN0aW9uIGluIHRoZSBteW9zaW4gc3VwZXJmYW1pbHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBDZWxsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz02MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaWJyZSB0eXBlLXNwZWNpZmljIGV4cHJlc3Npb24gcGF0dGVybnMgb2YgbXlvc2luIGhlYXZ5IGNoYWluIGdlbmVzIGluIGFkdWx0IHRvcmFmdWd1IFRha2lmdWd1IHJ1YnJpcGVzIG11c2NsZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjEzOyAgY2l0YXRpb25fcGFnZXM9MTM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RzIG9mIGludGVyZ2VuaWMgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiB0aGUgZmFzdCBteW9zaW4gaGVhdnkgY2hhaW4gY2x1c3RlciBvbiBtdXNjbGUgZmliZXIgY2hhcmFjdGVyaXN0aWNzIGFuZCBtZWF0IHF1YWxpdHkgaW4gQmVya3NoaXJlIHBpZ3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZWF0IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0yMjQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZXJhdGVseSBpbmNyZWFzZWQgbWF0ZXJuYWwgZGlldGFyeSBlbmVyZ3kgaW50YWtlIGRlbGF5cyBmb2V0YWwgc2tlbGV0YWwgbXVzY2xlIGRpZmZlcmVudGlhdGlvbiBhbmQgbWF0dXJpdHkgaW4gcGlnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVyLiBKLiBOdXRyOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTc3Ny04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gYW5hbHlzaXMgcmV2ZWFscyBnZW5ldGljIGxvY2kgYW5kIGNhbmRpZGF0ZSBnZW5lcyBmb3IgbWVhdCBxdWFsaXR5IHRyYWl0cyBpbiBDaGluZXNlIExhaXd1IHBpZ3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NYW1tLiBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNJUkk6IGFuIGVmZmljaWVudCBhbmQgdW5iaWFzZWQgYWxnb3JpdGhtIGZvciBkZSBub3ZvIGNpcmN1bGFyIFJOQSBpZGVudGlmaWNhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNOZXQ6IGEgZGF0YWJhc2Ugb2YgY2lyY3VsYXIgUk5BcyBkZXJpdmVkIGZyb20gdHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPUQyMDktMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIHRyYW5zY3JpcHRzIG9mIHRoZSB0ZXN0aXMtZGV0ZXJtaW5pbmcgZ2VuZSAmbHQ7ZW0mZ3Q7U3J5Jmx0Oy9lbSZndDsgaW4gYWR1bHQgbW91c2UgdGVzdGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUtd2lkZSBpbnZlc3RpZ2F0aW9uIG9mIGNpcmN1bGFyIFJOQXMgaW4gcmljZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzYtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGluZyBhbmQgY2hhcmFjdGVyaXppbmcgY2lyY3VsYXIgUk5Bcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTQ1My02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25zIG9mIHRoZSBjYXJkaWFjIHJ5YW5vZGluZSByZWNlcHRvciAoUnlSMikgZ2VuZSBpbiBmYW1pbGlhbCBwb2x5bW9ycGhpYyB2ZW50cmljdWxhciB0YWNoeWNhcmRpYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNpcmN1bGF0aW9uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTQ4NS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25zIGluIHRoZSBzbG93IHNrZWxldGFsIG11c2NsZSBmaWJlciBteW9zaW4gaGVhdnkgY2hhaW4gZ2VuZSAoJmx0O2VtJmd0O01ZSDcmbHQ7L2VtJmd0OykgY2F1c2UgbGFpbmcgZWFybHktb25zZXQgZGlzdGFsIG15b3BhdGh5IChNUEQxKSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBIdW0uIEdlbmV0OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9NzAzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcmRpb215b3BhdGh5OiBhIHN5c3RlbWF0aWMgcmV2aWV3IG9mIGRpc2Vhc2UtY2F1c2luZyBtdXRhdGlvbnMgaW4gbXlvc2luIGhlYXZ5IGNoYWluIDcgYW5kIHRoZWlyIHBoZW5vdHlwaWMgbWFuaWZlc3RhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYXJkaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc21zIHJlZ3VsYXRpbmcgc2tlbGV0YWwgbXVzY2xlIGdyb3d0aCBhbmQgYXRyb3BoeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yODA7ICBjaXRhdGlvbl9wYWdlcz00Mjk0LTMxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGlnaHQganVuY3Rpb25zIGFuZCB0aGUgcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TZW1pbi4gQ2VsbCBEZXYuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0yMTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgdGlnaHQganVuY3Rpb25zIGluIGNlbGwgcHJvbGlmZXJhdGlvbiBhbmQgY2FuY2VyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvZy4gSGlzdG9jaGVtLiBDeXRvY2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmN1bGFyIFJOQXMgYXJlIHRoZSBwcmVkb21pbmFudCB0cmFuc2NyaXB0IGlzb2Zvcm0gZnJvbSBodW5kcmVkcyBvZiBodW1hbiBnZW5lcyBpbiBkaXZlcnNlIGNlbGwgdHlwZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzA3MzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjMlRyYWl0czogYSBjb21wcmVoZW5zaXZlIGRhdGFiYXNlIGZvciBjaXJjdWxhciBSTkEgcG90ZW50aWFsbHkgYXNzb2NpYXRlZCB3aXRoIGRpc2Vhc2UgYW5kIHRyYWl0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZGVlcEJhc2UgdjIuMDogaWRlbnRpZmljYXRpb24sIGV4cHJlc3Npb24sIGV2b2x1dGlvbiBhbmQgZnVuY3Rpb24gb2Ygc21hbGwgUk5BcywgTG5jUk5BcyBhbmQgY2lyY3VsYXIgUk5BcyBmcm9tIGRlZXAtc2VxdWVuY2luZyBkYXRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1EMTk2LTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY0ludGVyYWN0b21lOiBhIHdlYiB0b29sIGZvciBleHBsb3JpbmcgY2lyY3VsYXIgUk5BcyBhbmQgdGhlaXIgaW50ZXJhY3RpbmcgcHJvdGVpbnMgYW5kIG1pY3JvUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zNC00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29tYW1pUiAyLjA6IGEgZGF0YWJhc2Ugb2YgY2FuY2VyIHNvbWF0aWMgbXV0YXRpb25zIGFsdGVyaW5nIG1pY3JvUk5B4oCTY2VSTkEgaW50ZXJhY3Rpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1EMTAwNS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzUvNTIzLzIyODkzMTk5L2RzeDAyMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHNwYXRpby10ZW1wb3JhbCBleHByZXNzaW9uIHBhdHRlcm5zIG9mIENpcmN1bGFyIFJOQSAoY2lyY1JOQSkgYWNyb3NzIG9yZ2FucyBhbmQgZGV2ZWxvcG1lbnRhbCBzdGFnZXMgYXJlIGNyaXRpY2FsIGZvciBpdHMgZnVuY3Rpb24gYW5kIGV2b2x1dGlvbiAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM4NTc3NzMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkTVr5YgJdC/2p0C6N/xLwtoV+1vw=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f523%2f3857773" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f523%2f3857773" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=99990" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&amp;Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/5">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106357139">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357139">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357141">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357141">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357146">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357146">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357153">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357153">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357188">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357188">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357194">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357194">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357196">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357196">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357202">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357202">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/5/509/3859850">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/5/537/3861839">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide profiling of <em>Sus scrofa</em> circular RNAs across nine organs and three developmental stages
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guoming Liang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China</div><div class="aff"><div class="label -label">2</div>Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div><div class="aff"><div class="label -label">3</div>Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guoming+Liang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liang G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liang G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guoming Liang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yalan Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China</div><div class="aff"><div class="label -label">2</div>Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div><div class="aff"><div class="label -label">3</div>Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yalan+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yalan Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guanglin Niu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guanglin+Niu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Niu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Niu G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guanglin Niu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhonglin Tang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China</div><div class="aff"><div class="label -label">2</div>Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div><div class="aff"><div class="label -label">3</div>Shenzhen Key Laboratory of Phenotype Analysis and Utilization of Agricultural Genome, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx022-cor1" data-open="dsx022-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10 6281 8180. Fax. +86 10 6281 8180. Email: <a href="mailto:zhonglinqy_99@sina.com" target="_blank">zhonglinqy_99@sina.com</a> (Z.T.); <a href="mailto:likui@caas.cn" target="_blank">likui@caas.cn</a> (K.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhonglin+Tang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tang Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhonglin Tang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kui Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>State Key Laboratory of Animal Nutrition, Institute of Animal Science, Chinese Academy of Agricultural Sciences, Beijing 100193, China</div><div class="aff"><div class="label -label">2</div>Department of Pig Genomic Design and Breeding, Agricultural Genome Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518124, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx022-cor1" data-open="dsx022-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10 6281 8180. Fax. +86 10 6281 8180. Email: <a href="mailto:zhonglinqy_99@sina.com" target="_blank">zhonglinqy_99@sina.com</a> (Z.T.); <a href="mailto:likui@caas.cn" target="_blank">likui@caas.cn</a> (K.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kui+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kui Li</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx022-FM1"><div class="fn"><div class="label -label">*</div><div class="footnote-content">Edited by Dr. Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 523–535, <a href='https://doi.org/10.1093/dnares/dsx022'>https://doi.org/10.1093/dnares/dsx022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 May 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 October 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 May 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3857773" href="/dnaresearch/article-pdf/24/5/523/22893199/dsx022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Guoming Liang, Yalan Yang, Guanglin Niu, Zhonglin Tang, Kui Li;  Genome-wide profiling of <em>Sus scrofa</em> circular RNAs across nine organs and three developmental stages, <em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 523–535, <a href='https://doi.org/10.1093/dnares/dsx022'>https://doi.org/10.1093/dnares/dsx022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3857773?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20profiling%20of%20%26lt%3Bem%26gt%3BSus%20scrofa%26lt%3B%2Fem%26gt%3B%20circular%20RNAs%20across%20nine%20organs%20and%20three%20developmental%20stages&amp;publicationDate=2017-05-29&amp;volumeNum=24&amp;issueNum=5&amp;author=Liang%2C%20Guoming%3B%20Yang%2C%20Yalan&amp;startPage=523&amp;endPage=535&amp;contentId=10.1093%2Fdnares%2Fdsx022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f523%2f3857773" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106357139 id="106357139" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The spatio-temporal expression patterns of Circular RNA (circRNA) across organs and developmental stages are critical for its function and evolution analysis. However, they remain largely unclear in mammals. Here, we comprehensively analysed circRNAs in nine organs and three skeletal muscles of Guizhou miniature pig (<em>S. scrofa</em>), a widely used biomedical model animal. We identified 5,934 circRNAs and analysed their molecular properties, sequence conservation, spatio-temporal expression pattern, potential function, and interaction with miRNAs. <em>S. scrofa</em> circRNAs show modest sequence conservation with human and mouse circRNAs, are flanked by long introns, exhibit low abundance, and are expressed dynamically in a spatio-temporally specific manner. <em>S. scrofa</em> circRNAs show the greatest abundance and complexity in the testis. Notably, 31% of circRNAs harbour well-conserved canonical miRNA seed matches, suggesting that some circRNAs act as miRNAs sponges. We identified 149 circRNAs potentially associated with muscle growth and found that their host genes were significantly involved in muscle development, contraction, chromatin modification, cation homeostasis, and ATP hydrolysis-coupled proton transport; moreover, this set of genes was markedly enriched in genes involved in tight junctions and the calcium signalling pathway. Finally, we constructed the first public <em>S. scrofa</em> circRNA database, allowing researchers to query comprehensive annotation, expression, and regulatory networks of circRNAs.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="circRNAs">circRNAs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="pig">pig</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="profiling">profiling</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="organs">organs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;skeletal muscle&quot;">skeletal muscle</a></div>                            <h2 scrollto-destination=106357141 id="106357141" class="section-title" >1. Introduction</h2>
<p>Circular RNAs (circRNAs) were generally believed to be linear until their circular form was observed via electron microscopy in 1979.<a reveal-id="dsx022-B1" data-open="dsx022-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> CircRNA is formed by exon-scrambling, in which a splice donor is spliced to an upstream acceptor rather than a downstream acceptor during transcription. While exons are most common, some introns can also form circRNAs during transcription. Usually, circRNAs are expressed at a relatively low level. CircRNAs have been identified in large numbers by deep RNA sequencing and are believed to be widely expressed in eukaryotic cells.<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2–6</sup></a> Most circRNA is flanked by GT/AG splice sites reflecting back-splicing. CircRNA back-splicing locations are in nearly the same locations in orthologous genes and exons in animals, showing that circRNAs are conserved in animals<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and plants.<a reveal-id="dsx022-B6" data-open="dsx022-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Although many circRNAs have been discovered in mouse and human, little is known about circRNAs in other mammals.</p><p>CircRNA expression is spatio-temporally regulated in <em>Drosophila</em> brains,<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> pig brains,<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and mouse pre-implantation embryos.<a reveal-id="dsx022-B5" data-open="dsx022-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> CircRNAs also show tissue-specific expression in the heart and lung during human fetal development.<a reveal-id="dsx022-B9" data-open="dsx022-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> It is believed that circRNAs bind miRNAs and thus regulate gene expression at the post-transcriptional level. For example, <em>CDR1as</em> functions as an efficient miRNA sponge, binding miR-7 and allowing mRNAs to escape degradation following miRNA binding.<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx022-B10" data-open="dsx022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>Sry</em> can form a circRNA that is also proposed to act as a miRNA sponge.<a reveal-id="dsx022-B10" data-open="dsx022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In addition to regulating mRNA translocation by acting as a miRNA sponge, circRNA can also regulate transcription through specific RNA–RNA interactions. A recently described subclass of circRNAs, EIciRNAs, contains introns among exons after RNA circling. In human cells, some EIciRNAs associate with RNA polymerase II, thus interacting with U1 snRNP and enhancing parental gene expression in <em>cis.</em><a reveal-id="dsx022-B11" data-open="dsx022-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The functions of circRNAs remain largely unclear. Investigating the spatio-temporal distribution of circRNAs is critical to understanding their biological functions. However, little is known about the expression patterns of circRNAs across organs and developmental stages in mammals.</p><p>The domestic pig (<em>Sus scrofa</em>) has had a close and complex relationship with humans for at least 10,000 years.<a reveal-id="dsx022-B12" data-open="dsx022-B12" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> The pig is an important domesticated farm animal that provides protein for humans, as well as a non-rodent animal model used in biomedical research. In comparison with traditional rodent models, the pig is more similar to humans in body size, growth, development, immunity, physiology, and metabolism, as well as genomic sequence.<a reveal-id="dsx022-B15" data-open="dsx022-B15" class="link link-ref link-reveal xref-bibr"><sup>15–17</sup></a> Recent studies of the <em>S. scrofa</em> genome and transcriptome have increased the utility of this species as a biomedical model.<a reveal-id="dsx022-B14" data-open="dsx022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx022-B18" data-open="dsx022-B18" class="link link-ref link-reveal xref-bibr"><sup>18–21</sup></a> However, in comparison with mice and humans, knowledge regarding the pig transcriptome across organs and developmental stages is very limited.<a reveal-id="dsx022-B22" data-open="dsx022-B22" class="link link-ref link-reveal xref-bibr"><sup>22–25</sup></a> Thousands of circRNAs have been discovered in fetal pig brains;<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> however, the identities and expression profiles of <em>S. scrofa</em> circRNAs remain largely uncharacterized.</p><p>We performed genome-wide analysis of circRNA across nine organs (heart, liver, spleen, lung, kidney, ovarium, testis, muscle, and fat) and three skeletal muscles at multiple developmental stages (0, 30, and 240 days after birth) from Guizhou miniature pigs (a native Chinese pig in the Guizhou province), which are widely used as an animal model in biomedical research. We systematically identified <em>S. scrofa</em> circRNAs and analysed their molecular properties, sequence conservation, spatio-temporal expression patterns, potential functions, and binding sites for miRNAs. Finally, we constructed the first public <em>S. scrofa</em> circRNA database.</p>                            <h2 scrollto-destination=106357146 id="106357146" class="section-title" >2. Materials and methods</h2>
<p>Nine organs, including the heart, liver, spleen, lung, kidney, ovarium, testis, skeletal muscle, and fat, were collected from three male and female Guizhou miniature pigs at postnatal day 240. We also collected longissimus muscle samples from three Guizhou miniature pigs at postnatal days 0 and 30. Total RNA was isolated from each sample by TRIZOL reagent according to the manufacturer's protocols, after which total RNA was pooled by tissue type and age for RNA-seq analysis. Total RNA was reverse-transcribed according to the manufacturer’s instructions. Paired-end reads 100 bp in length were sequenced by an Illumina HiSeq 2500. RNA-seq data were deposited in the Gene Expression Omnibus (accession codes GSE73763).</p><p>Reads were first mapped to the pig genome by Bowtie2,<a reveal-id="dsx022-B26" data-open="dsx022-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> after which the mapped reads were filtered out by SAMtools.<a reveal-id="dsx022-B27" data-open="dsx022-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> For the unmapped reads, 20 mers were cut off from both ends of each paired-end read, after which the remaining 60 mers were aligned to the pig genome in reverse orientation by Bowtie2.<a reveal-id="dsx022-B26" data-open="dsx022-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> We used the find_circ package<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> to identify circRNA. The workflow is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>.</p><p>In this study, a circRNA was designated as conserved when it was produced from orthologous genes in different species. We extracted pig-to-human and pig-to-mouse orthologous gene tables from BioMart (<a class="link link-uri" href="http://www.biomart.org" target="_blank">www.biomart.org</a>). Human and mouse circRNAs were identified by a previous study.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx022-B28" data-open="dsx022-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p><p>CircRNA expression was quantified by calculating the number of junction spanning reads per million reads of each sample. Junction spanning reads that were less than 5 reads supported were excluded when determining circRNA expression levels. Differentially expressed circRNAs were identified by the chi-square test with <em>P</em> ≤ 0.05 and had fold-change ≥ 2 (or ≤ 0.5) between any two tissues samples.</p><p>Four-hundred and eleven mature pig miRNA sequences were obtained from miRBase.<a reveal-id="dsx022-B29" data-open="dsx022-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> We used microtar,<a reveal-id="dsx022-B30" data-open="dsx022-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Miranda,<a reveal-id="dsx022-B31" data-open="dsx022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and RNAhybrid<a reveal-id="dsx022-B32" data-open="dsx022-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> to identify circRNA binding sites for miRNAs. To decrease the proportion of false-positive results, intersections predicted by all three programs were treated as the final set of target miRNAs.</p><p>The circRNA database is composed of a web interface and a SQLite database engine, which is used to store and manage all data. The data processing programs are written in Python (version 3.4.4). The web interface is built by Django (version 1.8.13) a Python Web framework. Google<sup>©</sup> charts (<a class="link link-uri" href="https://developers.google.com/chart/" target="_blank">https://developers.google.com/chart/</a>) and D3.js (<a class="link link-uri" href="https://d3js.org/" target="_blank">https://d3js.org/</a>) were used for interface development. The circRNA database is coded as a Django project and is deployed by uWSGI (<a class="link link-uri" href="https://uwsgi-docs.readthedocs.io" target="_blank">https://uwsgi-docs.readthedocs.io</a>). Web services were built using nginx (version 2.10.0), an HTTP and reverse proxy server.</p>                            <h2 scrollto-destination=106357153 id="106357153" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=106357154 id="106357154" class="section-title" >3.1. Identification and characteristics of <em>S. scrofa</em> circRNAs</h3>
<p>To perform comprehensive profiling of circular RNA in pigs, we carried out total RNA sequencing in nine adult organs (including fat, heart, kidney, liver, lung, spleen, testis, ovarium, and skeletal muscle) and three skeletal muscles at 0, 30, and 240 days after birth. In this study, the identified circRNAs were flanked by GT/AG splice sites and had unambiguous break points. The genomic distance between two splice sites was less than 100 kb, and each circRNA was supported by at least 2 reads in each sample. We identified 5,934 unique circRNAs in all assessed biological tissues (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s1.xlsx?Expires=2147483647&Signature=RuKJNnQQTnfx52hL3dQZsj9p5tIxvtJhVKqT3zQ1qV8AWYxXrsq33S6~QUZahTN~s37ReV5PEOIGZeJq07LaQK~oqrGIsQ5szpq69rmfef2GaiKcoLPXhScg3e1YdYZj9lM7SVTrEN7VOEOJPAX4nWCEe1~Gqc3ClT0TmmW35gpCkGgHzeMeFNZtBayA5Y2NEFPmf81kEfpGHI6O~VJEk2WVy9w3xrbA6WOyts9b4xy4KrooQ1nAaReAuzIw1Ecwbu8KctRbLibVTcvF1aX2juuTGWfBL5l-7Lci3hD~mY5Ydq6eLi4QlYioUGhOjhY7gqZKHL3M~Bc-KjcIFHFmRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p><p>The analysis of chromosome distribution showed that circRNAs are widely and unevenly transcribed from <em>S. scrofa</em> chromosomes (SSCs). More circRNAs were generated from SSC1 and SSC6 in comparison with the other SSCs. Only three circRNAs were detected on SSC-Y (<a reveal-id="dsx022-F1" data-open="dsx022-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The number of circRNAs was not significantly related to chromosome length or the number of genes contained in each chromosome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Most circRNAs contain multiple exons, and some circRNAs also retain introns, as indicated in a previous study.<a reveal-id="dsx022-B33" data-open="dsx022-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> According to the pig genes and splice junction annotations in the Ensembl database (release 78, <a class="link link-uri" href="http://ftp://ftp.ensembl.org/pub/release-78/gtf/sus_scrofa" target="_blank">ftp://ftp.ensembl.org/pub/release-78/gtf/sus_scrofa</a>), we found that most circularizing events typically encompass less than 5 exons (average exons = 4.8). More than 90% of circRNAs are formed by multiple exons, while only 9.9% are formed by a single exon (<a reveal-id="dsx022-F1" data-open="dsx022-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). This finding is in agreement with observations in mice.<a reveal-id="dsx022-B5" data-open="dsx022-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p>                        <a id="106357157" scrollto-destination="106357157"></a>
<div id="dsx022-F1" data-id="dsx022-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f1.png?Expires=2147483647&Signature=4AfQPCY3IUXvJ4Om~Kw6EsQKq9zOmpRJ-XD51oh13qcZrb4x-KRc3BEB0WNc2Lvqi0GnmOLwVoYwju1z63xsE5s7bdNNHMEmPVGf7d-p1UW90Rccv46yPPH18QdVE~9EHf-XdiM6mbAgkdym8fugIcsOg~NY7e2nez5t0VSX1iYhZukFJfYiIvLjO22g4X1POl~vTM5RzsyE5MDvRW7Y-qbsaHS58NwCC4DdInZE7AuGqrdqDt71RSXK071ha2qmuJYaPvr-ziRFoy5JsvqK3V3cFnmSpP~gcLEPGy~EbEUSbjT9HAhgd~FzgC0-GdF8rCVZOUwnXegdHqNHfF5faQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification and feature of circRNAs in pigs. (A) Chromosome distribution of circRNAs, the curve line represents the density of circRNAs along chromosome. (B) Distribution of exon number for circRNAs and mRNA. (C) Genomic location of circRNAs. (D) GC contents of circRNAs comparing with mRNA, miRNA and lincRNA." path-from-xml="dsx022f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357157/dsx022f1.png" path-from-xml="dsx022f1.png" target="_blank">View large</a><a section="106357157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f1.png?Expires=2147483647&Signature=gH9joyYRw-VTyZ0~D2CYiEgEFU1bAr6IhQKnwWbO48Hah5jQ9pW4J-wN-eui1~Z7Z2NMaCpoQPG7nsbT72x3iP~pqJbyd1v4a~C26IqgcSSVKIS6E-ko-ho71wCX7HwdEoeLRzMbDWZnbOAh7sNGJ2~BjgyFWU3GB8cZvZeTNlBzPuEtidHkZ7fpv5oiElSnkEfPlkltHZ90JeoGvf0z80uimXDx4juae5cfhZhw29J2EKsu1vCO4V9MxxiJM88TByS~unLgv67H2yua-k7iw19WqNXFYb3CSR66u6fYKGF18rf6LbGprP7ujjj4LYNRFxxA~Z-zVMc009gZTwHU2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357157&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification and feature of circRNAs in pigs. <strong>(A)</strong> Chromosome distribution of circRNAs, the curve line represents the density of circRNAs along chromosome. <strong>(B)</strong> Distribution of exon number for circRNAs and mRNA. <strong>(C)</strong> Genomic location of circRNAs. <strong>(D)</strong> GC contents of circRNAs comparing with mRNA, miRNA and lincRNA.</p></div></div><div content-id="dsx022-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f1.png?Expires=2147483647&Signature=4AfQPCY3IUXvJ4Om~Kw6EsQKq9zOmpRJ-XD51oh13qcZrb4x-KRc3BEB0WNc2Lvqi0GnmOLwVoYwju1z63xsE5s7bdNNHMEmPVGf7d-p1UW90Rccv46yPPH18QdVE~9EHf-XdiM6mbAgkdym8fugIcsOg~NY7e2nez5t0VSX1iYhZukFJfYiIvLjO22g4X1POl~vTM5RzsyE5MDvRW7Y-qbsaHS58NwCC4DdInZE7AuGqrdqDt71RSXK071ha2qmuJYaPvr-ziRFoy5JsvqK3V3cFnmSpP~gcLEPGy~EbEUSbjT9HAhgd~FzgC0-GdF8rCVZOUwnXegdHqNHfF5faQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification and feature of circRNAs in pigs. (A) Chromosome distribution of circRNAs, the curve line represents the density of circRNAs along chromosome. (B) Distribution of exon number for circRNAs and mRNA. (C) Genomic location of circRNAs. (D) GC contents of circRNAs comparing with mRNA, miRNA and lincRNA." path-from-xml="dsx022f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357157/dsx022f1.png" path-from-xml="dsx022f1.png" target="_blank">View large</a><a section="106357157" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f1.png?Expires=2147483647&Signature=gH9joyYRw-VTyZ0~D2CYiEgEFU1bAr6IhQKnwWbO48Hah5jQ9pW4J-wN-eui1~Z7Z2NMaCpoQPG7nsbT72x3iP~pqJbyd1v4a~C26IqgcSSVKIS6E-ko-ho71wCX7HwdEoeLRzMbDWZnbOAh7sNGJ2~BjgyFWU3GB8cZvZeTNlBzPuEtidHkZ7fpv5oiElSnkEfPlkltHZ90JeoGvf0z80uimXDx4juae5cfhZhw29J2EKsu1vCO4V9MxxiJM88TByS~unLgv67H2yua-k7iw19WqNXFYb3CSR66u6fYKGF18rf6LbGprP7ujjj4LYNRFxxA~Z-zVMc009gZTwHU2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357157&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification and feature of circRNAs in pigs. <strong>(A)</strong> Chromosome distribution of circRNAs, the curve line represents the density of circRNAs along chromosome. <strong>(B)</strong> Distribution of exon number for circRNAs and mRNA. <strong>(C)</strong> Genomic location of circRNAs. <strong>(D)</strong> GC contents of circRNAs comparing with mRNA, miRNA and lincRNA.</p></div></div><p>CircRNAs are mostly comprised of coding sequence (CDS, 68.40%) and intergenic regions (21.93%), whereas a smaller fraction (10%) also contain 5′-untranslated regions (UTRs), 3′-UTRs, or both UTRs (<a reveal-id="dsx022-F1" data-open="dsx022-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>), in agreement with a previous report.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The distribution of circRNA exonic sequence lengths resembles that of human circRNA, with a median length of 509 nucleotides, whereas that of human circRNA is 547 nucleotides.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Most <em>S. scrofa</em> circRNAs are shorter than 2 kb (4,243/5,934 = 71.5%), similar to mouse circRNAs.<a reveal-id="dsx022-B5" data-open="dsx022-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The GC content of <em>S. scrofa</em> circRNA is similar to that of other small RNAs, with an average GC content of 42.67% (<a reveal-id="dsx022-F1" data-open="dsx022-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>).</p>                            <h3 scrollto-destination=106357159 id="106357159" class="section-title" >3.2. Sequence conservation among pig, human, and mouse circRNAs</h3>
<p>Several studies suggest that some circRNAs are evolutionarily conserved among humans and mice.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx022-B34" data-open="dsx022-B34" class="link link-ref link-reveal xref-bibr"><sup>34–36</sup></a> Indeed, 81 of 1,903 mouse circRNAs were mapped to human circRNAs in a report by Memczak.<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Using the liftOver tool, another study found that 28.5% of mouse circRNAs are conserved in humans, while 33.3% of these circRNAs have no human homolog.<a reveal-id="dsx022-B35" data-open="dsx022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> To investigate circRNA sequence conservation between pigs and other mammals, we compared pig circRNAs to human and mouse circRNAs with clear one-to-one orthologs (<a class="link link-uri" href="http://asia.ensembl.org/biomart/martview/" target="_blank">http://asia.ensembl.org/biomart/martview/</a>). We found that 20.20% of pig circRNAs have human orthologs, whereas 16.96% of pig circRNAs have mouse orthologs (<a reveal-id="dsx022-F2" data-open="dsx022-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). These observations are comparable with previous reports. </p>                        <a id="106357161" scrollto-destination="106357161"></a>
<div id="dsx022-F2" data-id="dsx022-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f2.png?Expires=2147483647&Signature=gcD3LBvd1mMYNH1HxolXBJlXsRgA2gMIrt6lod8qMMwPb2xD~MdsN3JSUU0teuJw4T96R-vHbXhiVpqVeWll1CasK3Dg2A5b6lWDOHPb0bfjKj6L0Sy5N2gLK1AczpTCzKhM2cJ~v-6PfJEF6Lpu~STCvLuzohN7Dz0tjTDiuBXIZNEYN6t~zXhTEI4OcglUJKgwhBo3K6IuMURpdHG4cu~LPSCWJJiPNyPSfTQb7DWUtUMpxCEhzeoTlSmMGjFoMWCWJtsrnkzb3QeOk7T75jmz3yHz01g0ZUeLmGlIsfPLkc5Y8rK7AbSm7EuYvJRZno5-hBGB17fxOuAtIVJFTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular characteristics of S. scrofa circRNA. (A) Sequence conservation of S. scrofa circRNA compared with human and mouse ones, respectively. The percentage means that if there is a circRNA in pig, there is also have an otology circRNA in human or mouse. (B) Sequences identity of flanking introns of a circRNA 5' end upstream and a 3' end downstream. (C) Distribution of flanking intron sequences lengths for circRNA. (D) Distribution of sequences identity of each intron pair that flanked circRNA." path-from-xml="dsx022f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357161/dsx022f2.png" path-from-xml="dsx022f2.png" target="_blank">View large</a><a section="106357161" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f2.png?Expires=2147483647&Signature=veUrTePcUYm14pKs2b4mHUPi8cxVp25-r24sJDEv8~FQBgZQyTPc9mg4UtTXK3xvLXU12Eh3GWehB7QJis-37vYV5e-MG-q3wOvEDtzhtpX241zI5AATPOQWU-RMGSN9fADHs54w3MLYPEkazy-DCEOarY~jEzxYo-7T-qm2drhRoMPqkKPqVzKIQZWJ7bRtry7yK5V0U99MLBUAqUD1vG8ukS-p2vES753YAOwrm2yaHyqCh796Jp1YsAqsHCgeh1VaZMXxDTDKzpGR99Er-RCFcB4VAu3MvbF4qvX~uIcVO0mlIrSzHNrU0fvG0i7WUl1PQ1tEYUkKthzOdXCmkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357161&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular characteristics of <em>S. scrofa</em> circRNA. <strong>(A)</strong> Sequence conservation of <em>S. scrofa</em> circRNA compared with human and mouse ones, respectively. The percentage means that if there is a circRNA in pig, there is also have an otology circRNA in human or mouse. <strong>(B)</strong> Sequences identity of flanking introns of a circRNA 5' end upstream and a 3' end downstream. <strong>(C)</strong> Distribution of flanking intron sequences lengths for circRNA. <strong>(D)</strong> Distribution of sequences identity of each intron pair that flanked circRNA.</p></div></div><div content-id="dsx022-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f2.png?Expires=2147483647&Signature=gcD3LBvd1mMYNH1HxolXBJlXsRgA2gMIrt6lod8qMMwPb2xD~MdsN3JSUU0teuJw4T96R-vHbXhiVpqVeWll1CasK3Dg2A5b6lWDOHPb0bfjKj6L0Sy5N2gLK1AczpTCzKhM2cJ~v-6PfJEF6Lpu~STCvLuzohN7Dz0tjTDiuBXIZNEYN6t~zXhTEI4OcglUJKgwhBo3K6IuMURpdHG4cu~LPSCWJJiPNyPSfTQb7DWUtUMpxCEhzeoTlSmMGjFoMWCWJtsrnkzb3QeOk7T75jmz3yHz01g0ZUeLmGlIsfPLkc5Y8rK7AbSm7EuYvJRZno5-hBGB17fxOuAtIVJFTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular characteristics of S. scrofa circRNA. (A) Sequence conservation of S. scrofa circRNA compared with human and mouse ones, respectively. The percentage means that if there is a circRNA in pig, there is also have an otology circRNA in human or mouse. (B) Sequences identity of flanking introns of a circRNA 5' end upstream and a 3' end downstream. (C) Distribution of flanking intron sequences lengths for circRNA. (D) Distribution of sequences identity of each intron pair that flanked circRNA." path-from-xml="dsx022f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357161/dsx022f2.png" path-from-xml="dsx022f2.png" target="_blank">View large</a><a section="106357161" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f2.png?Expires=2147483647&Signature=veUrTePcUYm14pKs2b4mHUPi8cxVp25-r24sJDEv8~FQBgZQyTPc9mg4UtTXK3xvLXU12Eh3GWehB7QJis-37vYV5e-MG-q3wOvEDtzhtpX241zI5AATPOQWU-RMGSN9fADHs54w3MLYPEkazy-DCEOarY~jEzxYo-7T-qm2drhRoMPqkKPqVzKIQZWJ7bRtry7yK5V0U99MLBUAqUD1vG8ukS-p2vES753YAOwrm2yaHyqCh796Jp1YsAqsHCgeh1VaZMXxDTDKzpGR99Er-RCFcB4VAu3MvbF4qvX~uIcVO0mlIrSzHNrU0fvG0i7WUl1PQ1tEYUkKthzOdXCmkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357161&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Molecular characteristics of <em>S. scrofa</em> circRNA. <strong>(A)</strong> Sequence conservation of <em>S. scrofa</em> circRNA compared with human and mouse ones, respectively. The percentage means that if there is a circRNA in pig, there is also have an otology circRNA in human or mouse. <strong>(B)</strong> Sequences identity of flanking introns of a circRNA 5' end upstream and a 3' end downstream. <strong>(C)</strong> Distribution of flanking intron sequences lengths for circRNA. <strong>(D)</strong> Distribution of sequences identity of each intron pair that flanked circRNA.</p></div></div><p>To assess whether pig and human circRNAs arise from orthologous exons, we used whole-genome alignments to identify regions of the human genome that correspond to pig circRNAs and quantified the degree to which pig circRNAs overlapped with these regions. We found that 29.4% of pig circRNAs have orthologous exons in humans. In mice, 28.6% of circRNAs overlap with human circRNAs.<a reveal-id="dsx022-B35" data-open="dsx022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Blast analysis suggested that 1,510 (1510/5934 = 25.45%) and 5,189 (5189/5934 = 87.44%) <em>S. scrofa</em> circRNAs have orthologs in mice and humans, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s2.xlsx?Expires=2147483647&Signature=NExjX7EeMoapM9m8lENrtnDrVHUTJIyqceT1rt2flYOkZbnHZW6wDQzPxGME8PyX~eK2eMAtNN9n5eT6Qf8XKWaMjLtrl2ZiKPozXUNX0IGczuTre03P9Pk4dG-Nt52nYQR5sKaAS9ovPFkSn6M~aE94xczX4~gM1qbehf4Tz9dIOT1j6UfWjgGIxOtUERY3u9Zep2zXi~JO6F-20jhVWvNuL8kpCuaBG3Y0dglPNYyN~8h2NjZc8T-lp9xFiZNMVbJLqw~GOwY6wAvVWi405rZwpt-6CRBm85eu4JOoMOF4KP5Z94dp7G1fC6f4tu0E0854k0pjfvPC0PauhDX4Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>). These findings indicate that pig, human, and mouse circRNAs are modestly conserved and often generated from orthologous genes. Moreover, our sequence conservation analysis suggests that circRNA might have conserved functions in pigs, mice, and humans.</p>                            <h3 scrollto-destination=106357163 id="106357163" class="section-title" >3.3. <em>S. scrofa</em> circRNAs are flanked by long introns</h3>
<p>The sequence conservation of circRNAs among species indicated that their mechanisms of biogenesis might also be conserved. Previous studies suggested that most circRNAs are derived from exons with long flanking introns, which usually contain reverse complementary matches (RCMs) that facilitate back-splicing of the enclosed circRNAs.<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx022-B37" data-open="dsx022-B37" class="link link-ref link-reveal xref-bibr"><sup>37–39</sup></a> To determine whether introns flank pig circRNAs, we analysed 5kb sequences length of both 5′ upstream and 3′ downstream sequences flanking pig circRNAs. We observed that the majority of circularized exons are flanked by upstream and downstream introns. We found that 60.4% of 5′ sequences upstream of circRNAs contain introns, whereas 62.4% of 3′ sequences downstream of circRNAs contain introns. Moreover, 29.1 and 28.0% of the assessed regions upstream and downstream of circRNAs are intergenic regions. Few sequences flanking circRNAs are CDS, 5′-UTRs, or 3′-UTRs (<a reveal-id="dsx022-F2" data-open="dsx022-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). We found that 2,641 circRNAs are flanked by intron sequences on the 5′ and 3′ ends. The median length of the flanking introns on both sides of the identified circRNAs is 5,040 bp (<a reveal-id="dsx022-F2" data-open="dsx022-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a><strong>)</strong>. Our analysis suggests that long flanking introns are an intrinsic determinant of RNA circularization in pigs. Mammalian circRNAs have been reported to have relatively long flanking introns.<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx022-B33" data-open="dsx022-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> We aligned the introns of each intron pair that flanked a circRNA using the Basic Local Alignment Search Tool (BLAST). We observed that 1,810 circRNAs share RCMs of at least 100 bp in pigs (<a reveal-id="dsx022-F2" data-open="dsx022-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>). The lengths of long introns are not randomly distributed among circularized exons in the genome.<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Short intronic repeat sequences and complementary sequences facilitate RNA circularization.<a reveal-id="dsx022-B3" data-open="dsx022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx022-B38" data-open="dsx022-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> In our study, few circRNAs flanking intronic repeat sequences were detected, but the RCMs detected in our experiments are longer than previously reported RCMs; these characteristics may be typical of <em>S. scrofa</em> circRNAs.</p>                            <h3 scrollto-destination=106357165 id="106357165" class="section-title" >3.4. circRNA isoforms and diversity of circularization pattern</h3>
<p>Most genes generated one or two circRNAs, but some genes yielded multiple distinct circularized products (<a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). We identified 4,928 circRNAs generated from 2,538 annotated host genes. Among the identified circRNAs, 85.7% are produced by ‘hot-spot’ host genes that generate more than two circRNAs. This finding has been reported in other studies of circRNAs.<a reveal-id="dsx022-B3" data-open="dsx022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx022-B5" data-open="dsx022-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> A striking example is the chromodomain helicase DNA binding protein 2 gene (<em>CHD2</em>), which is involved in epigenetic regulation of chromatin structure and may generate as many as 40 distinct circRNAs (at least two unique back-spliced reads). CircRNA isoforms provide insight into the mechanism of circRNA biogenesis and the manner in which circRNAs are regulated. It is notable that, because de novo circRNA detection was performed without prior knowledge of annotated exons, 1,006 of the identified circRNAs fall outside the genomic regions of annotated genes. </p>                        <a id="106357167" scrollto-destination="106357167"></a>
<div id="dsx022-F3" data-id="dsx022-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f3.png?Expires=2147483647&Signature=SGRviSLJF~uzPktiX2QR7LyFwchZUEkta0TsPt4puYcc25KhfNZGyo745EYRlRVLhTAdgJTTylJMgr17YDQ7gwTNjyaDqQBkIlDZ5ooGzsi0StVCrmnF514n9q58worLyD~qkNzmsPhLYt3r0S-slgpaNvPeA7nfcl4n1kIN89xKiol1Th0k4aXgx9~ievOT~JHfzMGAyWM4wDljOONubZwmRz59HdW0Y-BM4HQqj883GFpC1-Xe-KRiNAVVfVwjXNVY~huYBdyBN06f5DXB7hF5h4IPg6boYD3XWjBUgrK0~7Dszo6gUkT3RnKPKX59lfLq-7lDNBvLAw1mFrKxgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Generation model and examples of circRNAs isoform in pig. (A) Distribution of circular RNAs among genes; (B) The genome location of three deduced donors for one acceptor to generate circRNA in DIS3L2. (C) The genome location of two deduced acceptors for one donor to generate circRNA in RBM25. (D) Expression profile of three circRNA isoforms for DIS3L2 in nine tissues. (E) ssc-ciR-03122 is abundantly expressed in kidney. (F) Tissue and age-dependent differential expression for two circRNA isoforms in PTGES3." path-from-xml="dsx022f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357167/dsx022f3.png" path-from-xml="dsx022f3.png" target="_blank">View large</a><a section="106357167" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f3.png?Expires=2147483647&Signature=15XqNoHhPvBTmiXodRL6zJ8g6Q2MuL-2VbBPbQjfugTeYMzUKwY0YukhxSrkDJJthHBh5KEuyI2~wITNA9~49e7OsUX-SQGX3BhVGz10lzyzL9s2zB66Oyupm~8eYut--k77Pbr6VXK8fYGW~xwjlf7HOVU9gOYN2gqXpNZDBtDOJu8PXPzqQI1zgfiKuDhQV74WmEafcLi8OMEB6xtfWzpINFhV~V3iMWF-EpOBsoljEltDIrZjXy-0Ho6p1cvUjhUITbDNx4YxBQAnFgbUHWqN7xUGLPfyqbilqsdIwie7bJSlOkQycCbckkhsw8j5J3VZQk6xHPfcSi5a3hSjyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357167&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Generation model and examples of circRNAs isoform in pig. <strong>(A)</strong> Distribution of circular RNAs among genes; <strong>(B)</strong> The genome location of three deduced donors for one acceptor to generate circRNA in DIS3L2. <strong>(C)</strong> The genome location of two deduced acceptors for one donor to generate circRNA in RBM25. <strong>(D)</strong> Expression profile of three circRNA isoforms for DIS3L2 in nine tissues. <strong>(E)</strong> ssc-ciR-03122 is abundantly expressed in kidney. <strong>(F)</strong> Tissue and age-dependent differential expression for two circRNA isoforms in PTGES3.</p></div></div><div content-id="dsx022-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f3.png?Expires=2147483647&Signature=SGRviSLJF~uzPktiX2QR7LyFwchZUEkta0TsPt4puYcc25KhfNZGyo745EYRlRVLhTAdgJTTylJMgr17YDQ7gwTNjyaDqQBkIlDZ5ooGzsi0StVCrmnF514n9q58worLyD~qkNzmsPhLYt3r0S-slgpaNvPeA7nfcl4n1kIN89xKiol1Th0k4aXgx9~ievOT~JHfzMGAyWM4wDljOONubZwmRz59HdW0Y-BM4HQqj883GFpC1-Xe-KRiNAVVfVwjXNVY~huYBdyBN06f5DXB7hF5h4IPg6boYD3XWjBUgrK0~7Dszo6gUkT3RnKPKX59lfLq-7lDNBvLAw1mFrKxgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Generation model and examples of circRNAs isoform in pig. (A) Distribution of circular RNAs among genes; (B) The genome location of three deduced donors for one acceptor to generate circRNA in DIS3L2. (C) The genome location of two deduced acceptors for one donor to generate circRNA in RBM25. (D) Expression profile of three circRNA isoforms for DIS3L2 in nine tissues. (E) ssc-ciR-03122 is abundantly expressed in kidney. (F) Tissue and age-dependent differential expression for two circRNA isoforms in PTGES3." path-from-xml="dsx022f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357167/dsx022f3.png" path-from-xml="dsx022f3.png" target="_blank">View large</a><a section="106357167" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f3.png?Expires=2147483647&Signature=15XqNoHhPvBTmiXodRL6zJ8g6Q2MuL-2VbBPbQjfugTeYMzUKwY0YukhxSrkDJJthHBh5KEuyI2~wITNA9~49e7OsUX-SQGX3BhVGz10lzyzL9s2zB66Oyupm~8eYut--k77Pbr6VXK8fYGW~xwjlf7HOVU9gOYN2gqXpNZDBtDOJu8PXPzqQI1zgfiKuDhQV74WmEafcLi8OMEB6xtfWzpINFhV~V3iMWF-EpOBsoljEltDIrZjXy-0Ho6p1cvUjhUITbDNx4YxBQAnFgbUHWqN7xUGLPfyqbilqsdIwie7bJSlOkQycCbckkhsw8j5J3VZQk6xHPfcSi5a3hSjyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357167&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Generation model and examples of circRNAs isoform in pig. <strong>(A)</strong> Distribution of circular RNAs among genes; <strong>(B)</strong> The genome location of three deduced donors for one acceptor to generate circRNA in DIS3L2. <strong>(C)</strong> The genome location of two deduced acceptors for one donor to generate circRNA in RBM25. <strong>(D)</strong> Expression profile of three circRNA isoforms for DIS3L2 in nine tissues. <strong>(E)</strong> ssc-ciR-03122 is abundantly expressed in kidney. <strong>(F)</strong> Tissue and age-dependent differential expression for two circRNA isoforms in PTGES3.</p></div></div><p>There are structural complexities among well-expressed circRNAs. circRNAs produced from back-splicing have splicing signals at their junctions.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Introns spliced by the major spliceosome usually contain the GT dinucleotide at their 5′ end (the splice donor) and the AG dinucleotide at their 3′ end (the splice acceptor).<a reveal-id="dsx022-B40" data-open="dsx022-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Most circRNAs contain one or a few exons; however, many circRNA are supported by abundant back-spliced reads that traverse multiple exons. More than one GT donor site often exists for a given AG acceptor splicing site. We found that this situation is common when GT donors are more than one exon away from AG acceptor sites; such genes generate multiple circular RNAs. For example, <em>DIS3L2</em>, an important factor in mRNA degradation,<a reveal-id="dsx022-B41" data-open="dsx022-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsx022-B42" data-open="dsx022-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> regulates Let-7g expression and apoptosis by targeting <em>TGFBR1</em> and down-regulating the activity of <em>TGF-β</em> pathway.<a reveal-id="dsx022-B43" data-open="dsx022-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> In our study, the <em>DIS3L2</em> gene was found to yield three circRNA isoforms, ssc-ciR-05526, ssc-ciR-05527, and ssc-ciR-05528, of 349 bp, 523 bp, and 603 bp in length, respectively (<a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The circular fractions of ssc-ciR-05526, ssc-ciR-05527, and ssc-ciR-05528 (ratio of the circular isoform relative to all transcripts from the same locus) are 3.8, 23.2, and 17.3%, respectively. Many circRNAs have different acceptors, but share the same donor. For example, host gene <em>RBM25</em> produces two circRNA isoforms as shown in <a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>. This analysis suggests the existence of alternative and interleaved splicing events, where the same splice sites can participate in multiple forward and backward splicing reactions, either to adjacent exons or to distant skipped exons.</p><p>We further assessed the abundance of circRNA isoforms, revealing dissimilar expression profiles among different circRNAs originating from the same host gene. Expression profiles of this type have also been observed by other groups.<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsx022-B44" data-open="dsx022-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> For example, among the three circRNA isoforms of <em>DIS3L2</em>, ssc-ciR-05527 is highly expressed in the lung, testis, and skeletal muscle at D240, whereas ssc-ciR-05526 and ssc-ciR-05528 were only detected in the testis (<a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). ssc-ciR-03122, the predominant circRNA from the <em>RBM25</em> gene, is abundantly expressed in the kidney, while its other four circRNA isoforms are weakly expressed or not detected in most tissues (<a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3E</a>). Among circRNA isoforms generated from <em>PTGES3</em>, ssc-ciR-01789 is widely expressed in all tissues and abundantly expressed in the testis and skeletal muscle. Expression of ssc-ciR-01789 in skeletal muscle was age-dependent. However, ssc-ciR-01788 from <em>PTGES3</em> is weakly expressed only in the kidney and testis (<a reveal-id="dsx022-F3" data-open="dsx022-F3" class="link link-reveal link-table xref-fig">Fig. 3F</a>). Another study revealed that some host genes produce multiple circRNAs with dissimilar expression profiles.<a reveal-id="dsx022-B45" data-open="dsx022-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Collectively, these results suggest that back-splicing by alternative spliceosomes generates circRNA isoforms and contributes to the diversity and functional complexity of circRNAs.</p>                            <h3 scrollto-destination=106357170 id="106357170" class="section-title" >3.5. <em>S. scrofa</em> circRNAs potentially act as miRNA sponges</h3>
<p>A previous study indicated that circRNAs function as miRNA sponges to indirectly regulate mRNA expression.<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx022-B10" data-open="dsx022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> To assess whether pig circRNAs regulate gene transcription by binding to miRNAs, we predicted miRNA binding sites of circRNAs using computational methods. We found 4,436 (74.8%), 4,432 (74.7%), and 2,279 (38.4%) putative miRNA-binding sites on 5,394 circRNAs using the micrnada, RNAhybrid, and microtar programs, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s4.xlsx?Expires=2147483647&amp;Signature=XpnLrKi7gUj5uOWxAtULHIh8CwbTOle4W1hOfdtY8Y1kihSDva55S9GBIl-RQCyQ567Lk6~GONQ6fB4AGUz6zOrTU8jHk7rOdlovMuYh93S~NqFuyRCkBC8WZlp29q~DulI7wp5VZmHy0XUGj8ImU-pFUOMEZEes-90NfhTrXKFBKLMGV9IF8aqGeGxcy3eS~FdwH6O6yMXTEUZSsudnp20iYchhbUuWfEbxswIOA~PS69NYCfIB4aWm-wTfVvbvvR1NdbNuJNa2jOT8onouWJsuJy6of~eddrH0IHShsMJ8URak6PGQCgeuJLOGaeQGnh6iAcjdUWhDvUmwjEnTNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>, <a reveal-id="dsx022-F4" data-open="dsx022-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Moreover, we observed a significant positive relationship between the number of miRNA binding sites and circRNA length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&amp;Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). To reduce the number of predicted false-positive miRNA targets, only miRNAs considered as targets of circRNAs identified using all three programs were further assessed in our study. We identified 1,842 circRNAs with miRNA binding sites that were predicted by all three prediction methods (<a reveal-id="dsx022-F4" data-open="dsx022-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>); while most circRNAs were predicted to have no potential binding sites (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s4.xlsx?Expires=2147483647&amp;Signature=XpnLrKi7gUj5uOWxAtULHIh8CwbTOle4W1hOfdtY8Y1kihSDva55S9GBIl-RQCyQ567Lk6~GONQ6fB4AGUz6zOrTU8jHk7rOdlovMuYh93S~NqFuyRCkBC8WZlp29q~DulI7wp5VZmHy0XUGj8ImU-pFUOMEZEes-90NfhTrXKFBKLMGV9IF8aqGeGxcy3eS~FdwH6O6yMXTEUZSsudnp20iYchhbUuWfEbxswIOA~PS69NYCfIB4aWm-wTfVvbvvR1NdbNuJNa2jOT8onouWJsuJy6of~eddrH0IHShsMJ8URak6PGQCgeuJLOGaeQGnh6iAcjdUWhDvUmwjEnTNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The miRNA-binding sites were not equally distributed in the circRNAs, only 441 circRNAs out of 1842 (22.20%) had more than 20 potential biding sites (<a reveal-id="dsx022-F4" data-open="dsx022-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). </p>                        <a id="106357172" scrollto-destination="106357172"></a>
<div id="dsx022-F4" data-id="dsx022-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f4.png?Expires=2147483647&Signature=NF7e8NlDCM6wxF0MXmZIW-UvXOY0mT4~L3NksIo0milahB~N06x2eI7l2ghIlVUbQRqA0HaLyil2e2tvOc7tSNDXjWREkP~Qbcjo6mXiqAlILPyMacFn7xvotUpNB8-iyjUf8BZ4JamMhkUt56hxyhiU7cBU0k5ZiHT18j7FVvgRQ5ELdaMMTcH-TWKlW8TVz5m3BYgrMZDryBKmVq6U0~fPSphgOFQKEqE4Hc2vb1GWijTfeTkadUA67pd3vq8CWvYelowePzhy98X02B3kZRjIClFlp16u-htdWihpwhbovPIA9zGDH9GLmYN61IGImYBVB76Jwmk2OK1fstdj3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis for circRNAs with the properties of miRNA sponges. (A) Prediction results of miRNA binding sites for circRNA by mirnada, RNAhybrid, and microtar program, respectively. (B) Overlapping of miRNA binding sites for circRNAs among three methods. (C) Number of miRNA binding sites for circRNAs. (D) A view of interaction between miRNA and circRNAs." path-from-xml="dsx022f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357172/dsx022f4.png" path-from-xml="dsx022f4.png" target="_blank">View large</a><a section="106357172" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f4.png?Expires=2147483647&Signature=2LoJZ4SiyI8wvo1uMqOiCuJB3rax6YfSyfMPFLR1klFm6Yr8ZecchqnD09eTY2EoWP9WR3seNsEGgT1wKhLfP3A5LMrb5tsb4BCraR9r83ScDuFTI0UEb7wkNG82PO1ef-dmZgZsDqO5ghyOJksWoV9y0ricZS7Wt-NCzhczSHQwE86mGOIJmc5C7dO7rqocpR1-8rvQSv5C32XYNHzjSZpk4nKvcRezSCbz-ZoRP802Eu6RSynIGoHkXj8cHGGL45S7MUw5hsTY9LvRI6S2z9fAaGQGj0AeUi7vgFNIsuxkbFZ6k5u5cUPztppqoVv-~MP2BgjY8~ClAAgUp5EGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357172&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis for circRNAs with the properties of miRNA sponges. <strong>(A)</strong> Prediction results of miRNA binding sites for circRNA by mirnada, RNAhybrid, and microtar program, respectively. <strong>(B)</strong> Overlapping of miRNA binding sites for circRNAs among three methods. <strong>(C)</strong> Number of miRNA binding sites for circRNAs. <strong>(D)</strong> A view of interaction between miRNA and circRNAs.</p></div></div><div content-id="dsx022-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f4.png?Expires=2147483647&Signature=NF7e8NlDCM6wxF0MXmZIW-UvXOY0mT4~L3NksIo0milahB~N06x2eI7l2ghIlVUbQRqA0HaLyil2e2tvOc7tSNDXjWREkP~Qbcjo6mXiqAlILPyMacFn7xvotUpNB8-iyjUf8BZ4JamMhkUt56hxyhiU7cBU0k5ZiHT18j7FVvgRQ5ELdaMMTcH-TWKlW8TVz5m3BYgrMZDryBKmVq6U0~fPSphgOFQKEqE4Hc2vb1GWijTfeTkadUA67pd3vq8CWvYelowePzhy98X02B3kZRjIClFlp16u-htdWihpwhbovPIA9zGDH9GLmYN61IGImYBVB76Jwmk2OK1fstdj3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis for circRNAs with the properties of miRNA sponges. (A) Prediction results of miRNA binding sites for circRNA by mirnada, RNAhybrid, and microtar program, respectively. (B) Overlapping of miRNA binding sites for circRNAs among three methods. (C) Number of miRNA binding sites for circRNAs. (D) A view of interaction between miRNA and circRNAs." path-from-xml="dsx022f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357172/dsx022f4.png" path-from-xml="dsx022f4.png" target="_blank">View large</a><a section="106357172" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f4.png?Expires=2147483647&Signature=2LoJZ4SiyI8wvo1uMqOiCuJB3rax6YfSyfMPFLR1klFm6Yr8ZecchqnD09eTY2EoWP9WR3seNsEGgT1wKhLfP3A5LMrb5tsb4BCraR9r83ScDuFTI0UEb7wkNG82PO1ef-dmZgZsDqO5ghyOJksWoV9y0ricZS7Wt-NCzhczSHQwE86mGOIJmc5C7dO7rqocpR1-8rvQSv5C32XYNHzjSZpk4nKvcRezSCbz-ZoRP802Eu6RSynIGoHkXj8cHGGL45S7MUw5hsTY9LvRI6S2z9fAaGQGj0AeUi7vgFNIsuxkbFZ6k5u5cUPztppqoVv-~MP2BgjY8~ClAAgUp5EGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357172&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis for circRNAs with the properties of miRNA sponges. <strong>(A)</strong> Prediction results of miRNA binding sites for circRNA by mirnada, RNAhybrid, and microtar program, respectively. <strong>(B)</strong> Overlapping of miRNA binding sites for circRNAs among three methods. <strong>(C)</strong> Number of miRNA binding sites for circRNAs. <strong>(D)</strong> A view of interaction between miRNA and circRNAs.</p></div></div><p>Subsequently, we analysed interactions between differentially expressed circRNAs and predicted target miRNAs. The resulting circRNA–miRNA association network provided nodes and connections between circRNAs and their target miRNAs. According to our data, no miRNA was regulated by only one circRNA. For example, miRNA-378 is involved in a regulatory pathway for circRNA ciR-90. In total, 141 miRNAs and 81 circRNA host genes were involved in this network (<a reveal-id="dsx022-F4" data-open="dsx022-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>). These results suggest that circRNAs could regulate gene expression by functioning as miRNA sponges. We found that each circRNA can contain more than one miRNA binding site and can bind to more than one miRNA. Although these <em>in silico</em> results should be investigated <em>in vivo</em>, these results illuminate the manner in which circRNAs regulate mRNA translation.</p>                            <h3 scrollto-destination=106357174 id="106357174" class="section-title" >3.6. Dominant circRNAs are expressed in a tissue-specific manner and highly enriched in the testis</h3>
<p>To act as miRNA sponges or perform other non-catalytic cellular functions, circRNAs must be expressed at a consequential level within a cell or region of tissue.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Here, in order to gain insight into the circRNA landscape across different organs and infer the abundance of each circRNA, we multiplied the circular fraction of each circRNA by the density of RNA-seq reads arising from the cognate gene locus (measured in fragment per kilobase of transcript per million fragments sequenced, or FPKM) (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s5.xlsx?Expires=2147483647&amp;Signature=omFH6Q6tvOebkpN98LLXBBhN6fHk29sLo6acWgeHIktl-GkP9P9miJ~C2oSMKkz5sfOS0RXzJdp1uGbSccFun5GVppdL6Tmg2bD-4huracFaKaRgC2EXi~o8e7RAe9DVobVOstrORu~9DIO6JTzbnswQBvMYZ~T-ZFGuy8PArZaGEjSL3JUl3fi4cE1OQWas9X0w9ci5Ki4XGkero9WE-oyqBsawAucy-tneVyxAzbhVqiphbO5rCDrZyT9tZklIixp3isEvFBNJoSMmxwEVnoAv84l3nv4my8BAuu4XAo0WTKbgMJ4oYOt~J38avd70jX6IAIcNt63ORR6VdyyLLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Most circRNAs are expressed with relatively low abundance.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx022-B46" data-open="dsx022-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> In this study, the average abundance of each <em>S. scrofa</em> circRNA was 0.45 FPKM. When considering the 395 circRNAs with inferred FPKM ≥ 1.0, only 43 had FPKM ≥ 10.0, while 4 had FPKM ≥ 100. To test whether circRNAs are expressed in a tissue-specific manner in pigs, a clustered heatmap was generated for nine organs (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&amp;Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). These results suggest that pig circRNAs have highly tissue-specific expression patterns (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). The largest cluster of circRNAs was in the testis and appeared to be testis-specific (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Interestingly, circRNA ssc-ciR-03062, which is generated from <em>MYH7</em> and <em>MYH6</em>, which are associated with familial hypertrophic cardiomyopathy,<a reveal-id="dsx022-B47" data-open="dsx022-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsx022-B48" data-open="dsx022-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> was the most abundantly expressed circRNA in the heart and was highly expressed in skeletal muscle. </p>                        <a id="106357176" scrollto-destination="106357176"></a>
<div id="dsx022-F5" data-id="dsx022-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f5.png?Expires=2147483647&Signature=uviM9KphhOU52GqBBTUAz7Qg0J0qRyR9LHOrOb8INwDuXpSwyFbDmchhGdxjobn4RrjZ72B0O9zEhySe8ec~VXEopAqt1r75S1MqErtA4Rs2lToMhikV-VMt3AxW95z-bcJZp0ADKQTO0H58Ggv1ujxQQ1Fxnyll9orTvdE21YRitVdU54Rbk3MkG8KmxndW9h-wJhQewRTr66-XMrOdj0QyWfkEJqj~3w3j99L69zuDYaHSCVZjnoYrWIcyJY904nf4VsEu6SW-hG1TG4kQK4-xvdTrPNwg3UCuzHqYbackcrlE~SayouglHEeEKFVHOXoRXB~J8dSpZeEq2mHVOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The tissue-specific expression of S. scrofa circRNAs in pigs. (A) Hierarchical clustering of the circular fraction of 5,399 circRNAs expressed in nine tissues with ≥2 reads. (B) Numbers of circRNAs identified each tissues. (C) Number of S. scrofa circRNAs in tissues analysed. (D) Percentage of tissues specific, -enrichment, and -enhanced circRNAs in each tissues. (E) KEGG term enrichments of circRNAs reveal many genes sets that were specific in tissues." path-from-xml="dsx022f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357176/dsx022f5.png" path-from-xml="dsx022f5.png" target="_blank">View large</a><a section="106357176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f5.png?Expires=2147483647&Signature=jiXh-Ew6fdi9o~Y0EGqsqHD2DGDSIxtcMew7WorKyiUB-k8MG0gLj-lYVBYGjoCbfwW82JxtnOJAvVS04VI9YzVH2XaFPWOYb4m00QJ-gMul4BWcG2wTD1lbU~2KHZrwqFvGWZqIzeTCFlbyxBZJFdcicpqOamKrdwuUk0tm1pPYbNPoeJfwYdigJJ8Z1PaBNqpKPXE4-ScJm6lDi1xGL-acj527WZboLZr~M6Lq~668nhiFiE2Blsp4zp~xrkijWbbaWx7MKmPqDMkPjIJTcVrmsQxg3emLdqgXFWgXy0eoc--DrSvK6DrjXVoKDaTWRVOzFJFC9VXaAUAloJuVcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357176&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The tissue-specific expression of <em>S. scrofa</em> circRNAs in pigs. <strong>(A)</strong> Hierarchical clustering of the circular fraction of 5,399 circRNAs expressed in nine tissues with ≥2 reads. <strong>(B)</strong> Numbers of circRNAs identified each tissues. <strong>(C)</strong> Number of <em>S. scrofa</em> circRNAs in tissues analysed. <strong>(D)</strong> Percentage of tissues specific, -enrichment, and -enhanced circRNAs in each tissues. <strong>(E)</strong> KEGG term enrichments of circRNAs reveal many genes sets that were specific in tissues.</p></div></div><div content-id="dsx022-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f5.png?Expires=2147483647&Signature=uviM9KphhOU52GqBBTUAz7Qg0J0qRyR9LHOrOb8INwDuXpSwyFbDmchhGdxjobn4RrjZ72B0O9zEhySe8ec~VXEopAqt1r75S1MqErtA4Rs2lToMhikV-VMt3AxW95z-bcJZp0ADKQTO0H58Ggv1ujxQQ1Fxnyll9orTvdE21YRitVdU54Rbk3MkG8KmxndW9h-wJhQewRTr66-XMrOdj0QyWfkEJqj~3w3j99L69zuDYaHSCVZjnoYrWIcyJY904nf4VsEu6SW-hG1TG4kQK4-xvdTrPNwg3UCuzHqYbackcrlE~SayouglHEeEKFVHOXoRXB~J8dSpZeEq2mHVOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The tissue-specific expression of S. scrofa circRNAs in pigs. (A) Hierarchical clustering of the circular fraction of 5,399 circRNAs expressed in nine tissues with ≥2 reads. (B) Numbers of circRNAs identified each tissues. (C) Number of S. scrofa circRNAs in tissues analysed. (D) Percentage of tissues specific, -enrichment, and -enhanced circRNAs in each tissues. (E) KEGG term enrichments of circRNAs reveal many genes sets that were specific in tissues." path-from-xml="dsx022f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357176/dsx022f5.png" path-from-xml="dsx022f5.png" target="_blank">View large</a><a section="106357176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f5.png?Expires=2147483647&Signature=jiXh-Ew6fdi9o~Y0EGqsqHD2DGDSIxtcMew7WorKyiUB-k8MG0gLj-lYVBYGjoCbfwW82JxtnOJAvVS04VI9YzVH2XaFPWOYb4m00QJ-gMul4BWcG2wTD1lbU~2KHZrwqFvGWZqIzeTCFlbyxBZJFdcicpqOamKrdwuUk0tm1pPYbNPoeJfwYdigJJ8Z1PaBNqpKPXE4-ScJm6lDi1xGL-acj527WZboLZr~M6Lq~668nhiFiE2Blsp4zp~xrkijWbbaWx7MKmPqDMkPjIJTcVrmsQxg3emLdqgXFWgXy0eoc--DrSvK6DrjXVoKDaTWRVOzFJFC9VXaAUAloJuVcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357176&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The tissue-specific expression of <em>S. scrofa</em> circRNAs in pigs. <strong>(A)</strong> Hierarchical clustering of the circular fraction of 5,399 circRNAs expressed in nine tissues with ≥2 reads. <strong>(B)</strong> Numbers of circRNAs identified each tissues. <strong>(C)</strong> Number of <em>S. scrofa</em> circRNAs in tissues analysed. <strong>(D)</strong> Percentage of tissues specific, -enrichment, and -enhanced circRNAs in each tissues. <strong>(E)</strong> KEGG term enrichments of circRNAs reveal many genes sets that were specific in tissues.</p></div></div><p>We found that 339–3,169 circRNAs were detected across most tissues types, and the testis had more than nine times as many circRNAs as the spleen (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). Some circRNAs are ubiquitously expressed, but most are found in only a few tissue types (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). We found that only 37 circRNAs are expressed in all nine organs analysed in this study. This variation was not explained by differences in sequencing depth. It is notable that 53.86% of the identified circRNAs (3,196/5,934 = 53.86%) were detected in the testis; however, 70.86% (2,265/3,196 = 70.86%) of the identified circRNAs were only observed in one other tissue type (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). Tissue-specific circRNAs are a group of circRNAs whose function and expression are confined to one or several tissues.<a reveal-id="dsx022-B49" data-open="dsx022-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsx022-B50" data-open="dsx022-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> As shown in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s6.xlsx?Expires=2147483647&amp;Signature=H2BsLWuD656yVe0AZ~xmxQOw~k1i4esgKwcym4uYvpob2sEQ-tG~~C~i9ognTlMdkmwn2bxqHJ5UUU7cp-49Td~APnrDTOWbiz~Si1Fa4xUI4ABhv9qR-5JqAfs1D9-j8vtoDvC6uZnD3GaLj8~B23D3hjZErxUNmaX4O32dNIF6vkpuDsmdlrFrFiuf5BPI~wOg1T8m~pNhQoOmrvgFyhsGYXOBMMX-ywGBoEgT2YHNiHPrXrAcV3RLnsYDrZsSQ82r9itZsqpoUWLrONHTZq~4VfySnb5pKPdIwne6pmal8yAdWbDpZ6naSbdlc5osdGAPFRKwIf3DbUFpKbyD5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, the testis shows the largest number of tissue-specific circRNAs (1,155), followed by the heart (205 circRNAs), muscle (174 circRNAs), and fat (147 circRNAs). The lowest number of tissue-specific circRNAs was found in the liver (53 circRNAs) (<a reveal-id="dsx022-T1" data-open="dsx022-T1" class="link link-reveal link-table xref-fig">Table 1</a>). It is notable that we did not detect any spleen-specific circRNAs. We assessed enrichment of gene ontology (GO) and Kyoto Encyclopaedia of Genes and Genomes (KEGG) terms among the tissue-specific circRNAs. In the heart, GO terms related to muscle development, construction, cell differentiation, and actin filament-based processes are highly enriched (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s7.xls?Expires=2147483647&amp;Signature=vlzUsYNlvu4lHzOpk3A5e0oMk-VNLX~3BTHOWBrqtmrshUyZOUDNgWjOBWLrKxvqQzGYpEaZ~Li3VjvSa0Xo4SYtGRBtZvtpYzs~0dVxmW52akm6p4og~8sBJcUOhZLLIOtGvNJsaKRJyTVLc1-vEVl29r1n5SnX6twKmb66vUJENIcBboravALzRZOkngdq52N5-oQfuIOlbSA7a7CDryuvJBkveSjZeOKNJfH6qt4W9jCH40mFIhIwlJxC2-oD8Yqp1ZfIh6RgjAI1-wb26d~VmbuXhBMsALFZFhUqgzlxPJmAgeYBeKLcdBK-wG4FAJTVlYh3nHv7irZq2igvjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Pathway analysis indicates that the set of circRNAs specific to the heart is significantly enriched in genes related to hypertrophic cardiomyopathy (HCM), cardiac muscle contraction, dilated cardiomyopathy, and the <em>Wnt</em> signalling pathway (<a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). <div content-id="dsx022-T1" class="table-modal"><div class="table-wrap"><div id="dsx022-T1" data-id="dsx022-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Classification of circRNA expression in different tissues</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissues </th><th>Tissues-specific circRNAs </th><th>Tissues-enrichment circRNAs </th><th>Tissues-enhanced circRNAs </th></tr></thead><tbody><tr><td>Spleen </td><td>0 </td><td>49 </td><td>182 </td></tr><tr><td>Lung </td><td>130 </td><td>125 </td><td>568 </td></tr><tr><td>Kidney </td><td>99 </td><td>106 </td><td>433 </td></tr><tr><td>Liver </td><td>53 </td><td>53 </td><td>232 </td></tr><tr><td>Fat </td><td>147 </td><td>149 </td><td>371 </td></tr><tr><td>Testis </td><td>1,155 </td><td>1,192 </td><td>1,596 </td></tr><tr><td>Muscle </td><td>174 </td><td>186 </td><td>412 </td></tr><tr><td>Ovarium </td><td>94 </td><td>95 </td><td>555 </td></tr><tr><td>Heart </td><td>205 </td><td>223 </td><td>512 </td></tr></tbody></table></div><div content-id="dsx022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissues </th><th>Tissues-specific circRNAs </th><th>Tissues-enrichment circRNAs </th><th>Tissues-enhanced circRNAs </th></tr></thead><tbody><tr><td>Spleen </td><td>0 </td><td>49 </td><td>182 </td></tr><tr><td>Lung </td><td>130 </td><td>125 </td><td>568 </td></tr><tr><td>Kidney </td><td>99 </td><td>106 </td><td>433 </td></tr><tr><td>Liver </td><td>53 </td><td>53 </td><td>232 </td></tr><tr><td>Fat </td><td>147 </td><td>149 </td><td>371 </td></tr><tr><td>Testis </td><td>1,155 </td><td>1,192 </td><td>1,596 </td></tr><tr><td>Muscle </td><td>174 </td><td>186 </td><td>412 </td></tr><tr><td>Ovarium </td><td>94 </td><td>95 </td><td>555 </td></tr><tr><td>Heart </td><td>205 </td><td>223 </td><td>512 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx022-T1" data-id="dsx022-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Classification of circRNA expression in different tissues</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Tissues </th><th>Tissues-specific circRNAs </th><th>Tissues-enrichment circRNAs </th><th>Tissues-enhanced circRNAs </th></tr></thead><tbody><tr><td>Spleen </td><td>0 </td><td>49 </td><td>182 </td></tr><tr><td>Lung </td><td>130 </td><td>125 </td><td>568 </td></tr><tr><td>Kidney </td><td>99 </td><td>106 </td><td>433 </td></tr><tr><td>Liver </td><td>53 </td><td>53 </td><td>232 </td></tr><tr><td>Fat </td><td>147 </td><td>149 </td><td>371 </td></tr><tr><td>Testis </td><td>1,155 </td><td>1,192 </td><td>1,596 </td></tr><tr><td>Muscle </td><td>174 </td><td>186 </td><td>412 </td></tr><tr><td>Ovarium </td><td>94 </td><td>95 </td><td>555 </td></tr><tr><td>Heart </td><td>205 </td><td>223 </td><td>512 </td></tr></tbody></table></div><div content-id="dsx022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Tissues </th><th>Tissues-specific circRNAs </th><th>Tissues-enrichment circRNAs </th><th>Tissues-enhanced circRNAs </th></tr></thead><tbody><tr><td>Spleen </td><td>0 </td><td>49 </td><td>182 </td></tr><tr><td>Lung </td><td>130 </td><td>125 </td><td>568 </td></tr><tr><td>Kidney </td><td>99 </td><td>106 </td><td>433 </td></tr><tr><td>Liver </td><td>53 </td><td>53 </td><td>232 </td></tr><tr><td>Fat </td><td>147 </td><td>149 </td><td>371 </td></tr><tr><td>Testis </td><td>1,155 </td><td>1,192 </td><td>1,596 </td></tr><tr><td>Muscle </td><td>174 </td><td>186 </td><td>412 </td></tr><tr><td>Ovarium </td><td>94 </td><td>95 </td><td>555 </td></tr><tr><td>Heart </td><td>205 </td><td>223 </td><td>512 </td></tr></tbody></table></div></div></p><p>Furthermore, we also analysed tissue-enriched (expression level at least 5-fold higher in a particular tissue as compared to all other tissues<a reveal-id="dsx022-B51" data-open="dsx022-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a>) and tissue-enhanced (at least 5-fold higher expression level in a particular tissue as compared to the average level in all tissues<a reveal-id="dsx022-B51" data-open="dsx022-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a>) circRNA. We identified 2,167 tissue-enriched circRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s8.xlsx?Expires=2147483647&amp;Signature=l7nDPPsjIb9xwcQtWHLN0qJyUz-YAHy4ZnNqbogvm0vJPoPuvOjp4r~l9AoYRvENvErZp4J~5M63riz~ZeNeakMNxJAuVjhu545Sl8~Iw6bNjT2A5C3-jOBg62gq6WjP3yX769bP9-Zm8PHHITaz13qjJtOYk9c2JhORJtIBvI7S5M~XA~VkSbRr6hT8ctka4GY3sSFwXvo-gW52gyqEfg9xi6wVZ8qPUBWo4bWkig2FnfgEwYX-YG1C7afdsWXufYXjNIPbcRvIYKyo-IVCjhZwR87EDixCzhS2~IEWGZ-9ozX8iNjuhDM6GCMxEmgk3mWesFtS~tjw19SHwB3b4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>) and 3,301 tissue-enhanced circRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s9.xlsx?Expires=2147483647&amp;Signature=eXedVugIRXNe8f5ed6-0zO~NdjHACIbU5X0GLAtBMkhrpIMkU6W9vIJvPSKmqaC2MCbZqJRUKgGbBFGM99Yn4OKrzfBguVQ8Kv43kf0HlzkTkUpJeSZYizxmnZXpJxIEkzNSQUMAZ~j1E62BJyCifXQKcA4IFk99bDHzRiez6QLn-d9Ok3T-FIP0KZ3Ck2zRn7nJPgsHYDSTPSMbZj9NDyvPaAMp2~0mxPmF4IfdhwbXUGg6Gyh~vAaDflnwaTqYwkWckfZRCtFY0wnlZyP-n6pUV5gQ7ezJBPGaoFjG93M7b14R8BaXc-5yI17RP9rFG6ooPSIwN-SVr46Xb2b53A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>) across all nine assessed tissue types. The numbers of tissue-enriched and tissue-enhanced genes are highly variable among the analysed tissue types (<a reveal-id="dsx022-T1" data-open="dsx022-T1" class="link link-reveal link-table xref-fig">Table 1</a>, <a reveal-id="dsx022-F5" data-open="dsx022-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). The testis shows the largest number of tissue-enriched circRNAs (1,192), followed by the heart (223) and skeletal muscle (186). These results indicate that circRNAs are expressed with highly tissue-specific patterns in pigs. A previous study also reported that dominant circular RNAs were expressed in a tissue-specific manner during human fetal development.<a reveal-id="dsx022-B9" data-open="dsx022-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p>                            <h3 scrollto-destination=106357179 id="106357179" class="section-title" >3.7. Differentially expressed circRNAs in postnatal skeletal muscle</h3>
<p>Skeletal muscles allow movement of the body via energy expenditure. Postnatal growth of skeletal muscle is mainly realized through increases in the length and girth of muscle fibres, but not by increases in muscle fibre number.<a reveal-id="dsx022-B52" data-open="dsx022-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> To understand the regulation of circRNAs in skeletal muscle, we performed expression profiling of circRNAs in skeletal muscle at postnatal days 0, 30, and 240 (designated D0, D30, and D240, respectively). This profiling allowed evaluation of dynamic changes in circRNA expression from birth until adulthood and identification of circRNAs associated with muscle growth. We detected 1,489 circRNAs expressed with more than two reads in at least one sample (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s10.xlsx?Expires=2147483647&amp;Signature=zvYBJWusLGpT6cn4-fMMV-vQfBF6oxSuOqTcTU6xZAAWRCuSvcxBRxrWYXczToeByIqcCzqobq1dniwVF6Xh7KoyNXBEJ68XQhmFzsuIvVs-yq-RaLtFuBFNQ8jY1l6--9WlY3vO-E8ZM0cIbghceRcJfGLuBmj2H66Ot~HoviuaTveZppCPCnhy6dTWnuEgvriRcLJ4UV5s7dETBuwE5-OeZ43Etf5su12Wm338RuuznXBIpTECFMpmo0lWjKWCXfaLXXT6if9Tunlsg5ZITiwu0PyFHqqWk4-zh3j9PHRTz-dgA3-buqruxmPS4Pr~FiiLnCcI69o-Oshwc-9u0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Of these 1,489 skeletal muscle circRNAs, 597 circRNAs were detected at D0, whereas 670 were detected at D30 and 673 were detected at D240; 57.2–63.9% of skeletal muscle circRNAs were detected at only one stage (<a reveal-id="dsx022-F6" data-open="dsx022-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>). The identified skeletal muscle circRNAs were studied further to assess changes in their expression levels during skeletal muscle growth. To obtain an overview of spatial circRNA expression changes in developmental skeletal muscle, we clustered pig circRNA expression data from D0, D30, and D240, revealing differential circRNA expression between growth stages. Interestingly, cluster analysis showed that skeletal muscle at D30 had a circRNA expression profile more similar to that of skeletal muscle at D240 than to that of skeletal muscle at D0 (<a reveal-id="dsx022-F6" data-open="dsx022-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). </p>                        <a id="106357181" scrollto-destination="106357181"></a>
<div id="dsx022-F6" data-id="dsx022-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f6.png?Expires=2147483647&Signature=vmV8EQG7pugxz6WSfxQA54QU3zMFTAYOF2GLymGJGe6XuCbQtc~piniOUBJj~5wCIHhaTnG2d922r2VdbIfaLuhjsCOHVqNejAdigMMs3zNzSB1~Iq5NPL7Bpxt9en3ipiJxCpsrP8mpbzs7w1sF81qVTOnyROLVQIReFjy03H-wJsfY-InAl7U4~SWgDUKucH~slBrf5vyOVNHtQTVK0oBk~hQQYvkLAkMi35Cng0FVYQ2RSZsfzBvZD8a-OTtdiAwf~7TPWZ4ncfE16CKDmkytw1jfjRf6PzSez6-hAK30fvuaanUHvzAhLPMdK71DnBboLWRnOYVlhhDXbIWgSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Temporal expression of circRNAs in skeletal muscle. (A) Venn chart of circRNAs detected in skeletal muscle. (B) Heatmap of circRNA expression abundance at 0 day, 30 days, and 240 days. (C) Top circRNAs expressed in skeletal muscle. (D–E) Differentially expressed circRNA between different growth stages in postnatal skeletal muscle." path-from-xml="dsx022f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357181/dsx022f6.png" path-from-xml="dsx022f6.png" target="_blank">View large</a><a section="106357181" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f6.png?Expires=2147483647&Signature=ZgbtRjR8om7NpKotIaz5HvrVwq-5A9hj~pEKzwtYn5xR0MjI1LCECCsp6cvU3uIiPNKkareK1zZ~3gDMpvj05DqsmO~Heuk9Umaot4R~UF5JhG5viJWPN5ODlSk8QB8jdq4zNVMOnRoc~BpWRQD-r-yYeJC58UQEXrfqQc76V29V1alVrO~1b-K0dEvTqhMcwlknHA98y9o2NEedi6WgZxcUr2hYVMBPH9MpljRlvTlvhFpd~tQ8F3QdXIocIxdoZAm33CjiuLI6-fQh6SGW8LlMUO5mKWzySdFc57s2ZXuPh7zFZ5ZnnLbLw9PVlRrIoRqSeFu9-Gby241YfU1JSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357181&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Temporal expression of circRNAs in skeletal muscle. <strong>(A)</strong> Venn chart of circRNAs detected in skeletal muscle. <strong>(B)</strong> Heatmap of circRNA expression abundance at 0 day, 30 days, and 240 days. <strong>(C)</strong> Top circRNAs expressed in skeletal muscle. <strong>(D–E)</strong> Differentially expressed circRNA between different growth stages in postnatal skeletal muscle.</p></div></div><div content-id="dsx022-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f6.png?Expires=2147483647&Signature=vmV8EQG7pugxz6WSfxQA54QU3zMFTAYOF2GLymGJGe6XuCbQtc~piniOUBJj~5wCIHhaTnG2d922r2VdbIfaLuhjsCOHVqNejAdigMMs3zNzSB1~Iq5NPL7Bpxt9en3ipiJxCpsrP8mpbzs7w1sF81qVTOnyROLVQIReFjy03H-wJsfY-InAl7U4~SWgDUKucH~slBrf5vyOVNHtQTVK0oBk~hQQYvkLAkMi35Cng0FVYQ2RSZsfzBvZD8a-OTtdiAwf~7TPWZ4ncfE16CKDmkytw1jfjRf6PzSez6-hAK30fvuaanUHvzAhLPMdK71DnBboLWRnOYVlhhDXbIWgSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Temporal expression of circRNAs in skeletal muscle. (A) Venn chart of circRNAs detected in skeletal muscle. (B) Heatmap of circRNA expression abundance at 0 day, 30 days, and 240 days. (C) Top circRNAs expressed in skeletal muscle. (D–E) Differentially expressed circRNA between different growth stages in postnatal skeletal muscle." path-from-xml="dsx022f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357181/dsx022f6.png" path-from-xml="dsx022f6.png" target="_blank">View large</a><a section="106357181" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f6.png?Expires=2147483647&Signature=ZgbtRjR8om7NpKotIaz5HvrVwq-5A9hj~pEKzwtYn5xR0MjI1LCECCsp6cvU3uIiPNKkareK1zZ~3gDMpvj05DqsmO~Heuk9Umaot4R~UF5JhG5viJWPN5ODlSk8QB8jdq4zNVMOnRoc~BpWRQD-r-yYeJC58UQEXrfqQc76V29V1alVrO~1b-K0dEvTqhMcwlknHA98y9o2NEedi6WgZxcUr2hYVMBPH9MpljRlvTlvhFpd~tQ8F3QdXIocIxdoZAm33CjiuLI6-fQh6SGW8LlMUO5mKWzySdFc57s2ZXuPh7zFZ5ZnnLbLw9PVlRrIoRqSeFu9-Gby241YfU1JSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357181&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Temporal expression of circRNAs in skeletal muscle. <strong>(A)</strong> Venn chart of circRNAs detected in skeletal muscle. <strong>(B)</strong> Heatmap of circRNA expression abundance at 0 day, 30 days, and 240 days. <strong>(C)</strong> Top circRNAs expressed in skeletal muscle. <strong>(D–E)</strong> Differentially expressed circRNA between different growth stages in postnatal skeletal muscle.</p></div></div><p>Next, we focused on abundant circRNAs in skeletal muscle. Notably, we found that several of the most abundant circRNAs in skeletal muscle originate from protein coding genes with pivotal roles in skeletal muscle growth and hypertrophy (e.g. <em>MYH1</em>, <em>MYH2</em>, <em>MYH3</em>, <em>MYH6</em>, <em>MYH7</em>, <em>MYL12B</em>, <em>IGFBP5</em>) or non-coding RNAs with roles in slow-twitch fibre formation (miRNA-208b encoded by introns of myosin genes) (<a reveal-id="dsx022-F6" data-open="dsx022-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>).<a reveal-id="dsx022-B53" data-open="dsx022-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsx022-B54" data-open="dsx022-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><em>MYH1</em>, <em>MYH6</em>, and <em>MYH7</em> belong to the <em>MYH</em> myosin superfamily. Myosin, the primary component of thick filaments, is widely expressed in mammalian skeletal muscle and involved in muscle contraction, phagocytosis, cell motility, and vesicle transport.<a reveal-id="dsx022-B55" data-open="dsx022-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsx022-B56" data-open="dsx022-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Recent studies reported that myosins are associated with skeletal muscle development and influence the quality of pork.<a reveal-id="dsx022-B57" data-open="dsx022-B57" class="link link-ref link-reveal xref-bibr"><sup>57–59</sup></a> We analysed the expression patterns of all circRNAs detected in skeletal muscle, revealing 16 expression patterns and significant enrichment in cluster 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). GO analysis of circRNAs expressed in skeletal muscle did not reveal significant enrichment in GO terms.</p><p>To study functional transitions in skeletal muscle development, we assessed two types of differentially expressed circRNAs (DEC) between time points. We focused on circRNAs with fold-change &gt;2, <em>P</em>-value &lt; 0.05, and FDR &lt; 0.1. The expression levels of most circRNAs are not significantly altered during postnatal skeletal muscle growth. Type 1 DECs were defined as those that were differentially expressed between two closed time points. We detected 101 and 83 differently expressed circRNAs in the D0 versus D30 and D30 versus D240 analyses, respectively. Among these DECs, 75 circRNAs were up-regulated at D30 in comparison with D0, whereas 26 circRNAs were up-regulated at D0 in comparison with D30 (<span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s11.xls?Expires=2147483647&amp;Signature=OAtIvyfQ5pzC-2LNIqWvlfX~NfvsRDPFBtGT4R02M3YhivagWW-MvM6RjHrEc8~IrpZ27-mJMlVohTs0zJFZiv2Fh9o87f3g9GcGj-dklmv-dnnHuvrT7P5eIo95wPt-TAhiuYulbRsXzF9CnL1y69VRXadFfK6Eqb5mnepDALSRYAOW-yNv8OO~Sh2WOuKloTazChFZQbikCeD5mK~Q5g-pgetpHhV61vsF-y4IRex9jikojPsgKKwhitRbN51mH62zJB2Umc~lRvB4b7keHXcjwxSRFycsKCA-HltvNqCRHNaQfyQN5XtphFeVtDDuecr2oxajeSG5AWY8geXYrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>, <a reveal-id="dsx022-F6" data-open="dsx022-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). In the D30 versus D240 group, 58 circRNAs were up-regulated at D240 in comparison with D30, whereas 25 circRNAs were up-regulated at D30 in comparison with D240 (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s12.xls?Expires=2147483647&amp;Signature=bWXY-UYi16eNWh-QGUDO4AEQK4BN5VHIdMixVHpqy-ioVG5CjlBkWGLTB1o1yIm6hz9oOP~fgTyumZVsezOxNB7yPN7jNWCD-v-sWUnJRONuzikxMEM6i8FeZ6HS-28XTjHNpW1Anp8rR713HT~xrGgl4q3Tz2itMdfeJRtkTySGTqvY4C-H15cuMzhY1MPLkQzXv3BvfKN0fTlsNlRq5g4Ze2HwIosUDycmjk-yDd1WRW5poyfgEdHrt-5HTzK328wcsTVnVH5lLt5EJjsxE9Ro5LtBpTOHrEhAVw4dVWXF6qMzDXVHwLkT0HGVGIrzI-gamo6MpTaQSdccaJkwNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>, <a reveal-id="dsx022-F6" data-open="dsx022-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a>). Under the assumption that the functions of circRNAs are related to the functions of their host genes, we performed GO and KEGG pathway analysis to predict the functions of DECs during postnatal growth of skeletal muscle. The GO analysis indicated that the host genes of the D0 versus D30 DECs are significantly associated with muscle development (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&amp;Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>). However, for the D30 versus D240 DECs, host genes are associated with homeostatic processes such as ion homeostasis and cation homeostasis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&amp;Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 4B</a></span>). Pathway analysis indicated that the host genes of the D0 versus D30 DECs are involved in tight junction signalling (<a reveal-id="dsx022-T2" data-open="dsx022-T2" class="link link-reveal link-table xref-fig">Table 2</a>). In the D30 versus D240 group, circRNA host genes are significantly associated with the calcium signalling pathway and aldosterone-regulated sodium reabsorption signalling pathway. The overall induction of circRNAs during skeletal muscle development indicates that circularization is likely important for muscle function, because many circRNAs are up- or down-regulated at different time points of skeletal muscle growth. <div content-id="dsx022-T2" class="table-modal"><div class="table-wrap"><div id="dsx022-T2" data-id="dsx022-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Pathway analysis of parent genes of circRNAs</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th> Term </th><th><em>P</em> value </th><th>Benjamini <em>q</em> value </th></tr></thead><tbody><tr><td>D0 versus D30 </td><td>Tight junction </td><td>7.67E−05 </td><td>0.0029 </td></tr><tr><td>D30 versus D240 </td><td>Calcium signalling pathway </td><td>0.0365 </td><td>0.67 </td></tr><tr><td>D30 versus D240 </td><td>Aldosterone-regulated sodium reabsorption </td><td>0.0703 </td><td>0.66 </td></tr></tbody></table></div><div content-id="dsx022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th> Term </th><th><em>P</em> value </th><th>Benjamini <em>q</em> value </th></tr></thead><tbody><tr><td>D0 versus D30 </td><td>Tight junction </td><td>7.67E−05 </td><td>0.0029 </td></tr><tr><td>D30 versus D240 </td><td>Calcium signalling pathway </td><td>0.0365 </td><td>0.67 </td></tr><tr><td>D30 versus D240 </td><td>Aldosterone-regulated sodium reabsorption </td><td>0.0703 </td><td>0.66 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>Note</em>: Significantly over-represented KEGG (Kyoto Encyclopaedia of Genes and Genomes) pathways detected with DAVID for parent genes of circRNAs that are differently expressed with at least a twofold changes in group D0 versus D30 and D30 versus D240, respectively.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx022-T2" data-id="dsx022-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Pathway analysis of parent genes of circRNAs</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th> Term </th><th><em>P</em> value </th><th>Benjamini <em>q</em> value </th></tr></thead><tbody><tr><td>D0 versus D30 </td><td>Tight junction </td><td>7.67E−05 </td><td>0.0029 </td></tr><tr><td>D30 versus D240 </td><td>Calcium signalling pathway </td><td>0.0365 </td><td>0.67 </td></tr><tr><td>D30 versus D240 </td><td>Aldosterone-regulated sodium reabsorption </td><td>0.0703 </td><td>0.66 </td></tr></tbody></table></div><div content-id="dsx022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th> Term </th><th><em>P</em> value </th><th>Benjamini <em>q</em> value </th></tr></thead><tbody><tr><td>D0 versus D30 </td><td>Tight junction </td><td>7.67E−05 </td><td>0.0029 </td></tr><tr><td>D30 versus D240 </td><td>Calcium signalling pathway </td><td>0.0365 </td><td>0.67 </td></tr><tr><td>D30 versus D240 </td><td>Aldosterone-regulated sodium reabsorption </td><td>0.0703 </td><td>0.66 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>Note</em>: Significantly over-represented KEGG (Kyoto Encyclopaedia of Genes and Genomes) pathways detected with DAVID for parent genes of circRNAs that are differently expressed with at least a twofold changes in group D0 versus D30 and D30 versus D240, respectively.</p></div></div></div></div></p><p>Type 2 DECs were defined as circRNAs that were differentially expressed only in the D0 versus D30 or D30 versus D240 analyses, representing circRNAs that were up- or down-regulated only between two consecutive time points during muscle growth. We found 29 circRNAs that were differentially expressed in the D0 versus D30 comparison, but not in the D30 versus D240 comparison (<span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s13.xls?Expires=2147483647&amp;Signature=q3VaPUh-Xh6J6FM~8bkPosWae5-DMe5PHXabjRxOlCaURv~7Rz10CKep~s~8~Qijub-h~KmL~iH7PMMDYBHcRfD-NN4Eth4ntm6p6r337lOyhKUwtZqjirvexl3J8PrlAHONOUQoAwPlhiR6kG9yBy0TzvetSisgD4t8MTLngz0UW9wI~a2cOsAph0UVEJvJoYmorhD~D0wxL8OrPfG9vDRme1n-F1DTGkaQ85nOm-kGaN2Ul0Xtwm4sruTWWmJ9ouxs~YF8XE0JI~OGiNXSqh7nijO1q3Albe~2WHFM8Z-sl0lTpVXblLi0JXKUz9hF5t~nMken1urdeRlNl1Y9hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 13</a></span>). Notably, several circRNAs generated from <em>MYH1</em>, <em>MYH2</em>, <em>MYH6</em>, <em>MYH7</em>, and <em>TNNC2</em>, as well as miRNA-208b, were significantly differentially expressed only in the D0 versus D30 comparison. These host genes mainly encoded proteins associated with muscle development and fibre twitching. We found 24 circRNAs that were differentially expressed in the comparison of D30 and D240, but not in the comparison of D0 and D30 (<span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s14.xls?Expires=2147483647&amp;Signature=fQBIQ1vyvtBHgQRmvIxQf8KzXiotZ3eQHoWLKn8VOYblZUJ1J0KZ2vDc0C2878Y7gTyJ8klDlVMqpaxgiar~KGu2T1vdSsoH7i-sBw3BBI0iaHP33GkTwF6p~zcApcoprnXnZiBX5SMxmHGk6q9cGhHZ5uSlGUvNlEPERQM2iGy4i9ypwS8zLnIQLlMySFq2hpNTgOGTbWnDJAxXJR879twE6C4EyswLxlI38tobH-psI3Z9DZ4e4Bwse3EFthUgD8Nwan0HS0uRc2EVknzqKUyNcnQ0lLmrwVN-lxsc50IPjx-06bfPgJ3CIKFYmOwihY2LMidjWggHpa9AfwXagQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). Interestingly, these circRNAs were mainly generated from <em>NDFIP2</em>, <em>RBM6</em>, <em>RYR1</em>, <em>SEC24A</em>, <em>RYR1</em>, <em>RBM24</em>, <em>SAR1B</em>, <em>PLEKHF1</em>, <em>ATP1A2</em>, <em>ATP1A4</em>, <em>DCUN1D1</em>, <em>ATP1A2</em>, <em>PRSS22</em>, and <em>PHKB</em>, which encode proteins associated with glycosaminoglycan metabolism and calcium channels. These circRNAs could signify the onset/termination of growth and/or physiological processes at a particular developmental stage.</p>                            <h3 scrollto-destination=106357185 id="106357185" class="section-title" >3.8. <em>S. scrofa</em> circRNA database</h3>
<p>In order to facilitate dissemination of information regarding circRNA annotation, circRNA expression, potential circRNA functions, and circRNA–miRNA–gene interactions, we created a database of <em>S. scrofa</em> circRNAs that was designated pigcirNet (<a class="link link-uri" href="http://lnc.rnanet.org/circ" target="_blank">http://lnc.rnanet.org/circ</a>). The web interface of pigcirNet is summarized in <a reveal-id="dsx022-F7" data-open="dsx022-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>. PigcirNet shows circRNA location, circular nucleotide length, strand signal, gene Entrez ID, Ensembl id, gene symbol, classification, and expression level across all organs and stages selected in this study. Information regarding prediction of circRNA targeting by miRNAs and circRNA–miRNA interactions are also available in pigcirNet. In order to investigate potential functions of circRNAs, we retrieve GO information with host-related genes. We also search the PubMed database to retrieve reports related to queried circRNAs. A Google<sup>©</sup> chart is used to display a diagram of circRNA abundance, whereas sigma.js is used to show a circRNA, mRNA, and miRNA interaction network. circRNAs can be browsed by their chromosomal location, classification, or tissue-specific expression pattern. We also provide a search box, in which users can find related circRNAs by searching circRNA id, gene symbol, gene Entrez ID, or gene Ensembl ID. To the best of our knowledge, our database is the first public circRNA resource for a non-rodent mammal. </p>                        <a id="106357187" scrollto-destination="106357187"></a>
<div id="dsx022-F7" data-id="dsx022-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f7.png?Expires=2147483647&Signature=B7fbbTjWL-9mLB90SAFEXQ1Rx2s3sZFVdRd8ir20H3GnNncCOM-HS-MxAQSeJKEJwL7YyruXdBUhb4ZykiSUQkX5nbQllxbuk~IFx-BxDKOZMLr4kY~SZqWncWPAwHmeoMvb7xeH28Zjzdek3e7QMjcEY2OIxyZNtCRtwyg0-S7KYlrm2DZJNOAEZBbdx41b2O45~7RVNdvIAZNsXQ9Zmg5KyUI4VmvssLE1yfOP16PGu57-aZaVemS8rkNCR2HpiBYJLIyFL9V~KGQnO3DqIIT3A6Wim5ZUq2ZLIVScp9NIDDFIBusoUum46e4l5m5fdNQ~twJft9pyFQmdzr4eLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Framework of the database construction in pigcirNet. The graph illustrates how the network in pigcirNet was constructed." path-from-xml="dsx022f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357187/dsx022f7.png" path-from-xml="dsx022f7.png" target="_blank">View large</a><a section="106357187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f7.png?Expires=2147483647&Signature=s1OP~890WAcICkooF89BMxBtALBE42BNERmzSmaS5GqieiVcxvb2~NjLbiOut04ILl4tmMUp789vi26CnjUNCwWEOeDDCD8VBi1GzcNwcuNYYYlK891OuPxa5eQEOyHauGx7~QENYmLXz8EAbZfOpYW3EERW6bhQ4YYa-GYi9XsKSKXbEMIulJvY97aXtvXRRjHyFWEIok-kOn4oS2O8L2E2jdYx6SFP8tBqb7vsb6p-ue26Qb81lbqBt2KAaA18o60Fyq3rV4PXqj0Z~RTTA7KemFIvTRFJZKqRovz0bo6mmFVYG8CT35WXDe5DxG1FP5Q3we9foY~vxtLnnMqv-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357187&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Framework of the database construction in pigcirNet. The graph illustrates how the network in pigcirNet was constructed.</p></div></div><div content-id="dsx022-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/m_dsx022f7.png?Expires=2147483647&Signature=B7fbbTjWL-9mLB90SAFEXQ1Rx2s3sZFVdRd8ir20H3GnNncCOM-HS-MxAQSeJKEJwL7YyruXdBUhb4ZykiSUQkX5nbQllxbuk~IFx-BxDKOZMLr4kY~SZqWncWPAwHmeoMvb7xeH28Zjzdek3e7QMjcEY2OIxyZNtCRtwyg0-S7KYlrm2DZJNOAEZBbdx41b2O45~7RVNdvIAZNsXQ9Zmg5KyUI4VmvssLE1yfOP16PGu57-aZaVemS8rkNCR2HpiBYJLIyFL9V~KGQnO3DqIIT3A6Wim5ZUq2ZLIVScp9NIDDFIBusoUum46e4l5m5fdNQ~twJft9pyFQmdzr4eLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Framework of the database construction in pigcirNet. The graph illustrates how the network in pigcirNet was constructed." path-from-xml="dsx022f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357187/dsx022f7.png" path-from-xml="dsx022f7.png" target="_blank">View large</a><a section="106357187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022f7.png?Expires=2147483647&Signature=s1OP~890WAcICkooF89BMxBtALBE42BNERmzSmaS5GqieiVcxvb2~NjLbiOut04ILl4tmMUp789vi26CnjUNCwWEOeDDCD8VBi1GzcNwcuNYYYlK891OuPxa5eQEOyHauGx7~QENYmLXz8EAbZfOpYW3EERW6bhQ4YYa-GYi9XsKSKXbEMIulJvY97aXtvXRRjHyFWEIok-kOn4oS2O8L2E2jdYx6SFP8tBqb7vsb6p-ue26Qb81lbqBt2KAaA18o60Fyq3rV4PXqj0Z~RTTA7KemFIvTRFJZKqRovz0bo6mmFVYG8CT35WXDe5DxG1FP5Q3we9foY~vxtLnnMqv-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357187&ar=3857773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx022f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Framework of the database construction in pigcirNet. The graph illustrates how the network in pigcirNet was constructed.</p></div></div>                            <h2 scrollto-destination=106357188 id="106357188" class="section-title" >4. Discussion</h2>
<p>Genome-wide analyses of RNA-seq data have revealed that circRNAs are abundant in animal transcriptomes and identified thousands of circRNAs in humans,<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx022-B60" data-open="dsx022-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> mice,<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> nematodes,<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> and drosophila.<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx022-B37" data-open="dsx022-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Guo et al. detected 7,112 human circRNAs using 39 biological samples of whole-cell non-poly(A)-selected RNA-seq data from the ENCODE project.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Gao et al. applied the CIRI algorithm to identify nearly 98,526 circRNAs based on ENCODE RNA-seq data from 15 cell lines.<a reveal-id="dsx022-B60" data-open="dsx022-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Based on 464 RNA-seq samples across 26 human tissues and 104 disease conditions,<a reveal-id="dsx022-B61" data-open="dsx022-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Liu and colleagues identified 212,950 human circRNAs (53,687 novel circRNAs). These findings suggest that a large number of circRNAs remain unknown and indicate that additional circRNAs can be identified by improving RNA analysis methods and accumulating deeper sequencing data. Because circRNAs are expressed in a highly spatio-temporal specific manner, it is essential that studies of circRNAs in mammals assess various tissues, conditions, and developmental stages. The pig is an important farm animal that provides protein for humans and an important non-rodent animal model that is widely used in biomedical research. Morten et al., the first group to study <em>S. scrofa</em> circRNAs, identified 4,634 unique circRNAs from 2,195 host genes in five brain tissue types at six time-points during fetal porcine development, and the results suggested that some circRNAs were conserved between humans and mice.<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> To perform genome-wide identification of <em>S. scrofa</em> circRNAs and explore their spatio-temporal expression patterns, we carried out total RNA sequencing across nine organs (heart, liver, spleen, lung, kidney, ovarium, testis, skeletal muscle, and fat) and skeletal muscle at three developmental stages (0, 30, and 240 days after birth). We identified 5,934 <em>S. scrofa</em> circRNAs, which represent a significant addition to the growing catalogue of annotated mammalian circRNAs. The circRNAs CDR1as and <em>Sry</em> were well-known studied. However, we did not detect circRNAs formed by a CDR1as orthologue or <em>Sry</em> in our data. <em>Sry</em> is induced during embryonic development.<a reveal-id="dsx022-B62" data-open="dsx022-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Because circRNAs are expressed in a tissue- and cell-specific manner, <em>S. scrofa</em> circRNAs orthologous to CDR1as and <em>Sry</em> were probably not expressed in the tissues selected in our study. To validate this conjecture, we further analysed expression of CDR1 and <em>Sry</em>. As expected, we did not detect expression of CDR1 or <em>Sry</em> in any tissue, Neither CDR1 nor Sry were mentioned in Morten’s study of the pig brain,<a reveal-id="dsx022-B8" data-open="dsx022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> indicating that these circRNAs were not expressed in these samples. We analysed a limited set of tissues and developmental stages in this study. Thus, the true number of <em>S. scrofa</em> circRNAs is almost certainly much larger than that reported in this study. In comparison with humans, knowledge of <em>S. scrofa</em> circRNAs remains limited. Therefore, deeper RNA-sequencing to identify circRNAs across additional organs and developmental stages in pigs is essential.</p><p>Although the molecular functions of circRNAs are mostly unclear, some circRNAs affect gene expression by acting as microRNA sponges. The most emblematical examples of circRNAs that act as microRNA sponges are CDR1as/ciRS-7 and <em>Sry</em>, which function in human and mouse brain development.<a reveal-id="dsx022-B2" data-open="dsx022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx022-B10" data-open="dsx022-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Guo et al. predicted that circRNAs from the human C2H2 zinc finger gene family function as miRNA sponges.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Another study showed that <em>Drosophila</em> circular RNAs harbour &gt;1000 well-conserved canonical miRNA seed matches.<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> To assess whether <em>S. scrofa</em> circRNAs affect post-transcriptional gene regulation by binding to miRNAs, we predicted circRNA-originating targets by three computational methods. We found that 1,842 of 5,934 (31.04%) circRNAs had putative miRNA-binding sites. In a study of rice, 17.33% (235/1356) of circRNAs were found to have putative miRNA-binding sites.<a reveal-id="dsx022-B63" data-open="dsx022-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> The identified circRNAs had, on average, 19 miRNA-binding sites. There was a significant positive relationship between the number of miRNA-binding sites and circRNA sequence length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Based on our analysis of circRNAs, we constructed an interaction network of miRNAs and circular RNAs. Certainly, the potential genome-wide interplay between miRNAs and circular RNAs warrants further experimental and computational investigation in the future. Although the aforementioned studies indicated that some circRNAs function as miRNA sponges, 4,092 of 5,934 (68.96%) <em>S. scrofa</em> circRNAs identified in this study have no putative miRNA binding site. Several studies have suggested that most circRNAs do not act as miRNA sponges.<a reveal-id="dsx022-B4" data-open="dsx022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx022-B63" data-open="dsx022-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> These findings indicate that circRNAs have functions other than binding to miRNAs, including regulation of host gene transcription, protein binding, and translation.<a reveal-id="dsx022-B11" data-open="dsx022-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx022-B64" data-open="dsx022-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a></p><p>Spatio-temporal gene expression patterns reveal the functions of genes within specific tissues/cells at specific times during development. Several studies have revealed that circRNAs tend to show tissue-/stage-specific expression<a reveal-id="dsx022-B7" data-open="dsx022-B7" class="link link-ref link-reveal xref-bibr"><sup>7–9</sup></a> and enrichment in the brain.<a reveal-id="dsx022-B35" data-open="dsx022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> We also identified tissue-specific, tissue-enriched, and tissue-enhanced circRNAs, providing important information regarding their functions. Of the identified heart-specific circRNAs, for example, ssc-ciR-03925, ssc-ciR-03394, ssc-ciR-03072, ssc-ciR-05708, ssc-ciR-02839, ssc-ciR-02761, ssc-ciR-04971, ssc-ciR-04972, and ssc-ciR-03852 were specifically expressed in the heart. ssc-ciR-04971 is generated from host gene <em>RYR2</em>, mutations in which are associated with stress-induced polymorphic ventricular tachycardia and arrhythmogenic right ventricular dysplasia.<a reveal-id="dsx022-B65" data-open="dsx022-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Ssc-ciR-03072 is produced from host gene <em>MYH7</em>, changes in the relative abundance of which are correlated with the contractile velocity of cardiac muscle; mutations in <em>MYH7</em> are associated with familial hypertrophic cardiomyopathy, myosin storage myopathy, dilated cardiomyopathy, and Laing early onset distal myopathy.<a reveal-id="dsx022-B66" data-open="dsx022-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><a reveal-id="dsx022-B67" data-open="dsx022-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> These findings suggest that heart-specific circRNAs might be involved in heart development and cardiac disease.</p><p>Skeletal muscle mass increases during postnatal animal development via hypertrophy.<a reveal-id="dsx022-B68" data-open="dsx022-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> The role of circRNAs in skeletal muscle development is unknown. Therefore, we analysed dynamic changes in circRNA abundance during postnatal growth of skeletal muscle. We identified 149 circRNAs that were differentially expressed between the selected time points and thus might be associated with postnatal growth of skeletal muscle. No studies have assessed the role of circRNAs in myogenesis. The D0 versus D30 DECs and D30 versus D240 DECs differed significantly with respect to enriched GO terms and signalling pathways. The D0 vs. D30 DECs were markedly enriched in GO terms associated with muscle contraction, muscle organ development, and chromatin modification, while the D30 versus D240 DECs were enriched in GO terms associated with cellular cation homeostasis, ATP hydrolysis-coupled proton transport, energy-coupled proton transport against the electrochemical gradient, and response to hypoxia. KEGG analysis suggested that DECs were mainly enriched in the tight junction pathway for the D0 versus D30 group, while DECs in the D30 versus D240 group were significantly associated with calcium signalling and aldosterone-regulated sodium reabsorption signalling. Tight junction proteins participate in the regulation of cell proliferation, gene expression, and cell differentiation.<a reveal-id="dsx022-B69" data-open="dsx022-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a><sup>,</sup><a reveal-id="dsx022-B70" data-open="dsx022-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> A previous study showed that proliferation and fusion of satellite cells, leading to an increase in the number of myonuclei, may also regulate muscle growth during the early, but not late, stages of postnatal development.<a reveal-id="dsx022-B68" data-open="dsx022-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Our study indicated that circRNAs (such as ssc-ciR-02753, ssc-ciR-04353, ssc-ciR-04335, ssc-ciR-04349, ssc-ciR-04348, ssc-ciR-04359, ssc-ciR-03066, ssc-ciR-03069, and ssc-ciR-03065) may regulate muscle growth by affecting cell proliferation and fusion during early postnatal muscle development. Clustering analysis suggested that the circRNA expression profile of skeletal muscle at D30 was more similar to that of skeletal muscle at D240 than to that of skeletal muscle at D0. We speculate that circRNAs such as ssc-ciR-01595, ssc-ciR-01592, ssc-ciR-03395, ssc-ciR-02589, ssc-ciR-02611, and ssc-ciR-02610 contribute to muscle plasticity and contraction. Some circRNAs are evolutionarily conserved in terms of sequence and expression, suggesting that they possess similar cellular functions in diverse species.<a reveal-id="dsx022-B34" data-open="dsx022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsx022-B35" data-open="dsx022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx022-B71" data-open="dsx022-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Our genome-wide analysis of the spatio-temporal expression patterns of <em>S. scrofa</em> circRNAs provides a foundation for studies aimed at understanding the molecular functions of mammalian circRNAs.</p><p>Thousands of circRNAs have been discovered in plants, animals, and humans. Given the emerging understanding of the biological importance of circRNAs and research efforts to understand them, researchers have constructed several circRNA databases, including Circ2Traits,<a reveal-id="dsx022-B72" data-open="dsx022-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> CircNet,<a reveal-id="dsx022-B61" data-open="dsx022-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> deepBase v2.0,<a reveal-id="dsx022-B73" data-open="dsx022-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> CircInteractome,<a reveal-id="dsx022-B74" data-open="dsx022-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> and SomamiR2.0.<a reveal-id="dsx022-B75" data-open="dsx022-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> At present, these databases include human, mouse, and <em>C. elegans</em> circRNAs, but not <em>S. scrofa</em> circRNAs, because identification of circRNAs in pigs has lagged far behind identification of circRNAs in humans, mice, and <em>C. elegans</em>. Exploring the pig genome can illuminate facets of human culture, organic evolution, biomedical research, and animal breeding. Therefore, we constructed the database of <em>S. scrofa</em> circRNAs based on our data. To the best of our knowledge, this database is the first public resource containing information about the circRNAs of a non-rodent mammal.</p>                            <h2 scrollto-destination=106357194 id="106357194" class="section-title" >Availability</h2>
<p>RNA-seq data were deposited in the Gene Expression Omnibus with accession codes was GSE73763. The identified circRNA was stored in <a class="link link-uri" href="http://lnc.rnanet.org/circ" target="_blank">http://lnc.rnanet.org/circ</a>.</p>                            <h2 scrollto-destination=106357196 id="106357196" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by National Key Basic Research Program of China (2014CB138504, 2015CB943101), National Natural Science Foundation of China (31171192, 31330074), the Agricultural Science and Technology Innovation Program (ASTIP-IAS16). This work was also supported by Key Laboratory of Shenzhen (ZDSYS20141118170111640). </p>                            <h3 scrollto-destination=106357198 id="106357198" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h3 scrollto-destination=106357200 id="106357200" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=106357202 id="106357202" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx022-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hsu</div>
               <div class="given-names">M.-T.</div>
            </div>, <div class="name string-name"><div class="surname">Coca-Prados</div> <div class="given-names">M.</div></div></div>
          <div class="year">1979</div>, 
<div class="article-title">Electron microscopic evidence for the circular form of RNA in the cytoplasm of eukaryotic cells,</div><div class="source">Nature</div> , <div class="volume">280</div>, <div class="fpage">339</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electron%20microscopic%20evidence%20for%20the%20circular%20form%20of%20RNA%20in%20the%20cytoplasm%20of%20eukaryotic%20cells%2C&amp;author=M.-T.%20Hsu&amp;author=M.%20Coca-Prados&amp;publication_year=1979&amp;journal=Nature&amp;volume=280&amp;pages=339-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/280339a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F280339a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F280339a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/460409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Memczak</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Jens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Elefsinioti</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Circular RNAs are a large class of animal RNAs with regulatory potency,</div><div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">333</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20a%20large%20class%20of%20animal%20RNAs%20with%20regulatory%20potency%2C&amp;author=S.%20Memczak&amp;author=M.%20Jens&amp;author=A.%20Elefsinioti&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=333-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">X.-O.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.-B.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.-L.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Complementary sequence-mediated exon circularization,</div><div class="source">Cell</div> , <div class="volume">159</div>, <div class="fpage">134</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complementary%20sequence-mediated%20exon%20circularization%2C&amp;author=X.-O.%20Zhang&amp;author=H.-B.%20Wang&amp;author=Y.%20Zhang&amp;author=X.%20Lu&amp;author=L.-L.%20Chen&amp;author=L.%20Yang&amp;publication_year=2014&amp;journal=Cell&amp;volume=159&amp;pages=134-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25242744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">J. U.</div>
            </div>, <div class="name string-name"><div class="surname">Agarwal</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Expanded identification and characterization of mammalian circular RNAs,</div><div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">014</div>–<div class="lpage">0409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expanded%20identification%20and%20characterization%20of%20mammalian%20circular%20RNAs%2C&amp;author=J.%20U.%20Guo&amp;author=V.%20Agarwal&amp;author=H.%20Guo&amp;author=D.%20P.%20Bartel&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=014-0409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-1-r14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-1-r14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-1-r14"> </span></div> </div></div></div></div></div><div content-id="dsx022-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fan</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Single-cell RNA-seq transcriptome analysis of linear and circular RNAs in mouse preimplantation embryos,</div><div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">148</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20RNA-seq%20transcriptome%20analysis%20of%20linear%20and%20circular%20RNAs%20in%20mouse%20preimplantation%20embryos%2C&amp;author=X.%20Fan&amp;author=X.%20Zhang&amp;author=X.%20Wu&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0706-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0706-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0706-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26201400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ye</div>
               <div class="given-names">C.-Y.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Q.-H.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">L.</div></div></div>
          <div class="year">2015</div>, Widespread noncoding circular RNAs in plants, <em>New Phytol.</em>, 208, 88–95.</div></div></div></div><div content-id="dsx022-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Westholm</div>
               <div class="given-names">J.O.</div>
            </div>, <div class="name string-name"><div class="surname">Miura</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Olson</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of Drosophila circular RNAs reveals their structural and sequence properties and age-dependent neural accumulation,</div><div class="source">Cell Rep</div> ., <div class="volume">9</div>, <div class="fpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Drosophila%20circular%20RNAs%20reveals%20their%20structural%20and%20sequence%20properties%20and%20age-dependent%20neural%20accumulation%2C&amp;author=J.O.%20Westholm&amp;author=P.%20Miura&amp;author=S.%20Olson&amp;publication_year=2014&amp;journal=Cell%20Rep&amp;volume=9&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.10.062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.10.062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.10.062"> </span></div> </div></div></div></div></div><div content-id="dsx022-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Veno</div>
               <div class="given-names">M. T.</div>
            </div>, <div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">T. B.</div></div>, <div class="name string-name"><div class="surname">Veno</div> <div class="given-names">S. T.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Spatio-temporal regulation of circular RNA expression during porcine embryonic brain development,</div><div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">015</div>–<div class="lpage">0801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatio-temporal%20regulation%20of%20circular%20RNA%20expression%20during%20porcine%20embryonic%20brain%20development%2C&amp;author=M.%20T.%20Veno&amp;author=T.%20B.%20Hansen&amp;author=S.%20T.%20Veno&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=015-0801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0801-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0801-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0801-3"> </span></div> </div></div></div></div></div><div content-id="dsx022-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Szabo</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Morey</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Palpant</div> <div class="given-names">N. J.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Statistically based splicing detection reveals neural enrichment and tissue-specific induction of circular RNA during human fetal development,</div><div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">015</div>–<div class="lpage">0690</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistically%20based%20splicing%20detection%20reveals%20neural%20enrichment%20and%20tissue-specific%20induction%20of%20circular%20RNA%20during%20human%20fetal%20development%2C&amp;author=L.%20Szabo&amp;author=R.%20Morey&amp;author=N.%20J.%20Palpant&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=015-0690" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0566-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0566-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0566-0"> </span></div> </div></div></div></div></div><div content-id="dsx022-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hansen</div>
               <div class="given-names">T. B.</div>
            </div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">T. I.</div></div>, <div class="name string-name"><div class="surname">Clausen</div> <div class="given-names">B. H.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Natural RNA circles function as efficient microRNA sponges,</div><div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">384</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20RNA%20circles%20function%20as%20efficient%20microRNA%20sponges%2C&amp;author=T.%20B.%20Hansen&amp;author=T.%20I.%20Jensen&amp;author=B.%20H.%20Clausen&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=384-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23446346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Exon-intron circular RNAs regulate transcription in the nucleus,</div><div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">22</div>, <div class="fpage">256</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exon-intron%20circular%20RNAs%20regulate%20transcription%20in%20the%20nucleus%2C&amp;author=Z.%20Li&amp;author=C.%20Huang&amp;author=C.%20Bao&amp;publication_year=2015&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=22&amp;pages=256-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2959" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2959" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2959"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25664725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hedges</div> <div class="given-names">S. B.</div></div></div>
          <div class="year">1998</div>, 
<div class="article-title">A molecular timescale for vertebrate evolution,</div><div class="source">Nature</div> , <div class="volume">392</div>, <div class="fpage">917</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20timescale%20for%20vertebrate%20evolution%2C&amp;author=S.%20Kumar&amp;author=S.%20B.%20Hedges&amp;publication_year=1998&amp;journal=Nature&amp;volume=392&amp;pages=917-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/31927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F31927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F31927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larson</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Dobney</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Albarella</div> <div class="given-names">U.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Worldwide phylogeography of wild boar reveals multiple centers of pig domestication,</div><div class="source">Science</div> , <div class="volume">307</div>, <div class="fpage">1618</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worldwide%20phylogeography%20of%20wild%20boar%20reveals%20multiple%20centers%20of%20pig%20domestication%2C&amp;author=G.%20Larson&amp;author=K.%20Dobney&amp;author=U.%20Albarella&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1618-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1106927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1106927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1106927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15761152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Groenen</div>
               <div class="given-names">M. A.</div>
            </div>, <div class="name string-name"><div class="surname">Archibald</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Uenishi</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Analyses of pig genomes provide insight into porcine demography and evolution,</div><div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">393</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20pig%20genomes%20provide%20insight%20into%20porcine%20demography%20and%20evolution%2C&amp;author=M.%20A.%20Groenen&amp;author=A.%20L.%20Archibald&amp;author=H.%20Uenishi&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=393-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wernersson</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Schierup</div> <div class="given-names">M. H.</div></div>, <div class="name string-name"><div class="surname">Jørgensen</div> <div class="given-names">F. G.</div></div></div>, et al.  <div class="year">2005</div><div class="article-title">, Pigs in sequence space: a 0.66 X coverage pig genome survey based on shotgun sequencing,</div><div class="source">BMC Genomics</div> , <div class="volume">6</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%2C%20Pigs%20in%20sequence%20space%3A%20a%200.66%20X%20coverage%20pig%20genome%20survey%20based%20on%20shotgun%20sequencing%2C&amp;author=R.%20Wernersson&amp;author=M.%20H.%20Schierup&amp;author=F.%20G.%20J%C3%B8rgensen&amp;publication_year=2005&amp;journal=BMC%20Genomics&amp;volume=6&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-6-70" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-6-70" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-6-70"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15627404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.-Y.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Comparative gene expression signature of pig, human and mouse induced pluripotent stem cell lines reveals insight into pig pluripotency gene networks,</div><div class="source">Stem Cell Rev. Rep</div> ., <div class="volume">10</div>, <div class="fpage">162</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20gene%20expression%20signature%20of%20pig%2C%20human%20and%20mouse%20induced%20pluripotent%20stem%20cell%20lines%20reveals%20insight%20into%20pig%20pluripotency%20gene%20networks%2C&amp;author=Y.%20Liu&amp;author=Y.%20Ma&amp;author=J.-Y.%20Yang&amp;publication_year=2014&amp;journal=Stem%20Cell%20Rev.%20Rep&amp;volume=10&amp;pages=162-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12015-013-9485-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12015-013-9485-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12015-013-9485-9"> </span></div> </div></div></div></div></div><div content-id="dsx022-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wall</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Shani</div> <div class="given-names">M.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Are animal models as good as we think?,</div><div class="source">Theriogenology</div> , <div class="volume">69</div>, <div class="fpage">2</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Are%20animal%20models%20as%20good%20as%20we%20think%3F%2C&amp;author=R.%20Wall&amp;author=M.%20Shani&amp;publication_year=2008&amp;journal=Theriogenology&amp;volume=69&amp;pages=2-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.theriogenology.2007.09.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.theriogenology.2007.09.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.theriogenology.2007.09.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17988725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Walters</div>
               <div class="given-names">E. M.</div>
            </div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Whyte</div> <div class="given-names">J. J.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Completion of the swine genome will simplify the production of swine as a large animal biomedical model,</div><div class="source">BMC Med Genomics</div> , <div class="volume">5</div>, <div class="fpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Completion%20of%20the%20swine%20genome%20will%20simplify%20the%20production%20of%20swine%20as%20a%20large%20animal%20biomedical%20model%2C&amp;author=E.%20M.%20Walters&amp;author=E.%20Wolf&amp;author=J.%20J.%20Whyte&amp;publication_year=2012&amp;journal=BMC%20Med%20Genomics&amp;volume=5&amp;pages=55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1755-8794-5-55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1755-8794-5-55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1755-8794-5-55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuggle</div>
               <div class="given-names">C. K.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Couture</div> <div class="given-names">O.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Advances in swine transcriptomics,</div><div class="source">Int. J. Biol. Sci</div> ., <div class="volume">3</div>, <div class="fpage">132</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20swine%20transcriptomics%2C&amp;author=C.%20K.%20Tuggle&amp;author=Y.%20Wang&amp;author=O.%20Couture&amp;publication_year=2007&amp;journal=Int.%20J.%20Biol.%20Sci&amp;volume=3&amp;pages=132-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7150/ijbs.3.132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7150%2Fijbs.3.132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7150%2Fijbs.3.132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17384733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gorodkin</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Cirera</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hedegaard</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Porcine transcriptome analysis based on 97 non-normalized cDNA libraries and assembly of 1,021,891 expressed sequence tags,</div><div class="source">Genome Biol</div> ., <div class="volume">8</div>, <div class="fpage">R45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Porcine%20transcriptome%20analysis%20based%20on%2097%20non-normalized%20cDNA%20libraries%20and%20assembly%20of%201%2C021%2C891%20expressed%20sequence%20tags%2C&amp;author=J.%20Gorodkin&amp;author=S.%20Cirera&amp;author=J.%20Hedegaard&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=R45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-4-r45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-4-r45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-4-r45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17407547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuggle</div>
               <div class="given-names">C. K.</div>
            </div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Fitzsimmons</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2003</div>, 
<div class="article-title">EST-based gene discovery in pig: virtual expression patterns and comparative mapping to human,</div><div class="source">Mamm. Genome</div> , <div class="volume">14</div>, <div class="fpage">565</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-based%20gene%20discovery%20in%20pig%3A%20virtual%20expression%20patterns%20and%20comparative%20mapping%20to%20human%2C&amp;author=C.%20K.%20Tuggle&amp;author=J.%20A.%20Green&amp;author=C.%20Fitzsimmons&amp;publication_year=2003&amp;journal=Mamm.%20Genome&amp;volume=14&amp;pages=565-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-002-2263-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-002-2263-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-002-2263-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12925889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Dynamic transcriptome profiles of skeletal muscle tissue across 11 developmental stages for both Tongcheng and Yorkshire pigs,</div><div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">377</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20transcriptome%20profiles%20of%20skeletal%20muscle%20tissue%20across%2011%20developmental%20stages%20for%20both%20Tongcheng%20and%20Yorkshire%20pigs%2C&amp;author=Y.%20Zhao&amp;author=J.%20Li&amp;author=H.%20Liu&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=377" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1580-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1580-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1580-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25962502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">P.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">LongSAGE analysis of skeletal muscle at three prenatal stages in Tongcheng and Landrace pigs,</div><div class="source">Genome Biol</div> ., <div class="volume">8</div>, <div class="fpage">R115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LongSAGE%20analysis%20of%20skeletal%20muscle%20at%20three%20prenatal%20stages%20in%20Tongcheng%20and%20Landrace%20pigs%2C&amp;author=Z.%20Tang&amp;author=Y.%20Li&amp;author=P.%20Wan&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=R115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-6-r115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-6-r115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-6-r115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17573972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hou</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Comparison of skeletal muscle miRNA and mRNA profiles among three pig breeds,</div><div class="source">Mol. Genet. Genomics</div> , <div class="volume">291</div><div class="fpage">, 559</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20skeletal%20muscle%20miRNA%20and%20mRNA%20profiles%20among%20three%20pig%20breeds%2C&amp;author=X.%20Hou&amp;author=Y.%20Yang&amp;author=S.%20Zhu&amp;publication_year=2015&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=291&amp;pages=, 559-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-015-1126-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-015-1126-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-015-1126-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26458558" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">K.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Integrated analysis of miRNA and mRNA paired expression profiling of prenatal skeletal muscle development in three genotype pigs,</div><div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">15544</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20analysis%20of%20miRNA%20and%20mRNA%20paired%20expression%20profiling%20of%20prenatal%20skeletal%20muscle%20development%20in%20three%20genotype%20pigs%2C&amp;author=Z.%20Tang&amp;author=Y.%20Yang&amp;author=Z.%20Wang&amp;author=S.%20Zhao&amp;author=Y.%20Mu&amp;author=K.%20Li&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=15544" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep15544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep15544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep15544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26496978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2,</div><div class="source">Nat. Meth</div> ., <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202%2C&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Meth&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div> </div></div></div></div></div><div content-id="dsx022-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data,</div><div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2987</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20framework%20for%20SNP%20calling%2C%20mutation%20discovery%2C%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data%2C&amp;author=H.%20Li&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2987-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Glažar</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Papavasileiou</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rajewsky</div> <div class="given-names">N.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">circBase: a database for circular RNAs,</div><div class="source">RNA</div> , <div class="volume">20</div>, <div class="fpage">1666</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=circBase%3A%20a%20database%20for%20circular%20RNAs%2C&amp;author=P.%20Gla%C5%BEar&amp;author=P.%20Papavasileiou&amp;author=N.%20Rajewsky&amp;publication_year=2014&amp;journal=RNA&amp;volume=20&amp;pages=1666-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.043687.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.043687.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.043687.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25234927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kozomara</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">miRBase: annotating high confidence microRNAs using deep sequencing data,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D68</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=miRBase%3A%20annotating%20high%20confidence%20microRNAs%20using%20deep%20sequencing%20data%2C&amp;author=A.%20Kozomara&amp;author=S.%20Griffiths-Jones&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D68-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24275495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thadani</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Tammi</div> <div class="given-names">M. T.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">MicroTar: predicting microRNA targets from RNA duplexes,</div><div class="source">BMC Bioinformatics</div> , <div class="volume">7 Suppl 5,</div><div class="fpage">S20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroTar%3A%20predicting%20microRNA%20targets%20from%20RNA%20duplexes%2C&amp;author=R.%20Thadani&amp;author=M.%20T.%20Tammi&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7 Suppl 5,&amp;pages=S20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-S5-S20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-S5-S20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-S5-S20"> </span></div> </div></div></div></div></div><div content-id="dsx022-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">John</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Enright</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Aravin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tuschl</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sander</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Marks</div> <div class="given-names">D. S.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Human microRNA targets,</div><div class="source">PLoS Biol</div> ., <div class="volume">2</div>, <div class="fpage">e363</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20microRNA%20targets%2C&amp;author=B.%20John&amp;author=A.%20J.%20Enright&amp;author=A.%20Aravin&amp;author=T.%20Tuschl&amp;author=C.%20Sander&amp;author=D.%20S.%20Marks&amp;publication_year=2004&amp;journal=PLoS%20Biol&amp;volume=2&amp;pages=e363" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15502875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krüger</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Rehmsmeier</div> <div class="given-names">M.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">RNAhybrid: microRNA target prediction easy, fast and flexible,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W451</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAhybrid%3A%20microRNA%20target%20prediction%20easy%2C%20fast%20and%20flexible%2C&amp;author=J.%20Kr%C3%BCger&amp;author=M.%20Rehmsmeier&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W451-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Salzman</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">R. E.</div></div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">M. N.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Cell-type specific features of circular RNA expression</div>. <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003777</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell-type%20specific%20features%20of%20circular%20RNA%20expression&amp;author=J.%20Salzman&amp;author=R.%20E.%20Chen&amp;author=M.%20N.%20Olsen&amp;author=P.%20L.%20Wang&amp;author=P.%20O.%20Brown&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003777" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003777" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003777" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003777"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24039610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jeck</div>
               <div class="given-names">W. R.</div>
            </div>, <div class="name string-name"><div class="surname">Sorrentino</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Circular RNAs are abundant, conserved, and associated with ALU repeats,</div><div class="source">RNA</div> , <div class="volume">19</div>, <div class="fpage">426</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20abundant%2C%20conserved%2C%20and%20associated%20with%20ALU%20repeats%2C&amp;author=W.%20R.%20Jeck&amp;author=J.%20A.%20Sorrentino&amp;author=K.%20Wang&amp;publication_year=2013&amp;journal=RNA&amp;volume=19&amp;pages=426" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.035667.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.035667.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.035667.112"> </span></div> </div></div></div></div></div><div content-id="dsx022-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rybak-Wolf</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Stottmeister</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Glažar</div> <div class="given-names">P.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Circular RNAs in the mammalian brain are highly abundant, conserved, and dynamically expressed,</div><div class="source">Mol. Cell</div> , <div class="volume">58</div>, <div class="fpage">870</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20in%20the%20mammalian%20brain%20are%20highly%20abundant%2C%20conserved%2C%20and%20dynamically%20expressed%2C&amp;author=A.%20Rybak-Wolf&amp;author=C.%20Stottmeister&amp;author=P.%20Gla%C5%BEar&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=58&amp;pages=870-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.03.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.03.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.03.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25921068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">B.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Tracing the expression of circular RNAs in human pre-implantation embryos,</div><div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracing%20the%20expression%20of%20circular%20RNAs%20in%20human%20pre-implantation%20embryos%2C&amp;author=Y.%20Dang&amp;author=L.%20Yan&amp;author=B.%20Hu&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-0991-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-0991-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-0991-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26753840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ashwal-Fluss</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pamudurti</div></div>, <div class="name string-name"><div class="surname">Nagarjuna</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">circRNA biogenesis competes with pre-mRNA splicing,</div><div class="source">Mol. Cell</div> , <div class="volume">56</div>, <div class="fpage">55</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=circRNA%20biogenesis%20competes%20with%20pre-mRNA%20splicing%2C&amp;author=R.%20Ashwal-Fluss&amp;author=M.%20Meyer&amp;author=%20Pamudurti&amp;author=R.%20Nagarjuna&amp;publication_year=2014&amp;journal=Mol.%20Cell&amp;volume=56&amp;pages=55-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2014.08.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2014.08.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2014.08.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25242144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liang</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Wilusz</div> <div class="given-names">J. E.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Short intronic repeat sequences facilitate circular RNA production,</div><div class="source">Genes Dev</div> ., <div class="volume">28</div>, <div class="fpage">2233</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20intronic%20repeat%20sequences%20facilitate%20circular%20RNA%20production%2C&amp;author=D.%20Liang&amp;author=J.%20E.%20Wilusz&amp;publication_year=2014&amp;journal=Genes%20Dev&amp;volume=28&amp;pages=2233-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.251926.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.251926.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.251926.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25281217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Starke</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Jost</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Rossbach</div> <div class="given-names">O.</div></div></div>, et al.  <div class="article-title">Exon circularization requires canonical splice signals</div>. <div class="source">Cell Rep</div> ., <div class="volume">10</div>, <div class="fpage">103</div>–<div class="lpage">111</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2014.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2014.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2014.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25543144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P. A.</div>
            </div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C. B.</div></div></div>
          <div class="article-title">Classification of introns: U2-type or U12-type,</div>
          <div class="source">Cell</div> , <div class="volume">91</div>, <div class="fpage">875</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)80479-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)80479-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)80479-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9428511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Faehnle</div>
               <div class="given-names">C. R.</div>
            </div>, <div class="name string-name"><div class="surname">Walleshauser</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Joshua-Tor</div> <div class="given-names">L.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Mechanism of Dis3l2 substrate recognition in the Lin28-let-7 pathway,</div><div class="source">Nature</div> , <div class="volume">514</div>, <div class="fpage">252</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20Dis3l2%20substrate%20recognition%20in%20the%20Lin28-let-7%20pathway%2C&amp;author=C.%20R.%20Faehnle&amp;author=J.%20Walleshauser&amp;author=L.%20Joshua-Tor&amp;publication_year=2014&amp;journal=Nature&amp;volume=514&amp;pages=252-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25119025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Malecki</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Viegas</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Carneiro</div> <div class="given-names">T.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The exoribonuclease Dis3L2 defines a novel eukaryotic RNA degradation pathway,</div><div class="source">EMBO J</div> , <div class="volume">32</div>, <div class="fpage">1842</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20exoribonuclease%20Dis3L2%20defines%20a%20novel%20eukaryotic%20RNA%20degradation%20pathway%2C&amp;author=M.%20Malecki&amp;author=S.%20C.%20Viegas&amp;author=T.%20Carneiro&amp;publication_year=2013&amp;journal=EMBO%20J&amp;volume=32&amp;pages=1842-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/emboj.2013.63" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Femboj.2013.63" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Femboj.2013.63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23503588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Z.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">The let-7g microRNA promotes follicular granulosa cell apoptosis by targeting transforming growth factor-β type 1 receptor,</div><div class="source">Mol Cell. Endocrinol</div> ., <div class="volume">409</div>, <div class="fpage">103</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20let-7g%20microRNA%20promotes%20follicular%20granulosa%20cell%20apoptosis%20by%20targeting%20transforming%20growth%20factor-%CE%B2%20type%201%20receptor%2C&amp;author=J.%20Zhou&amp;author=J.%20Liu&amp;author=Z.%20Pan&amp;publication_year=2015&amp;journal=Mol%20Cell.%20Endocrinol&amp;volume=409&amp;pages=103-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mce.2015.03.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mce.2015.03.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mce.2015.03.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25817543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zheng</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">W.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Circular RNA profiling reveals an abundant circHIPK3 that regulates cell growth by sponging multiple miRNAs,</div><div class="source">Nat. Commun</div> ., <div class="volume">7,</div><div class="fpage">11215</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNA%20profiling%20reveals%20an%20abundant%20circHIPK3%20that%20regulates%20cell%20growth%20by%20sponging%20multiple%20miRNAs%2C&amp;author=Q.%20Zheng&amp;author=C.%20Bao&amp;author=W.%20Guo&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7,&amp;pages=11215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms11215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms11215"> </span></div> </div></div></div></div></div><div content-id="dsx022-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ameres</div>
               <div class="given-names">S. L.</div>
            </div>, <div class="name string-name"><div class="surname">Zamore</div> <div class="given-names">P. D.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Diversifying microRNA sequence and function,</div><div class="source">Nat. Rev. Mol. Cell Biol</div> ., <div class="volume">14</div>, <div class="fpage">475</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversifying%20microRNA%20sequence%20and%20function%2C&amp;author=S.%20L.%20Ameres&amp;author=P.%20D.%20Zamore&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;volume=14&amp;pages=475-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm3611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm3611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm3611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23800994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.-Y.</div></div>, <div class="name string-name"><div class="surname">Chuang</div> <div class="given-names">T.-J.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Biogenesis, identification, and function of exonic circular RNAs,</div><div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="volume">6</div>, <div class="fpage">563</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biogenesis%2C%20identification%2C%20and%20function%20of%20exonic%20circular%20RNAs%2C&amp;author=I.%20Chen&amp;author=C.-Y.%20Chen&amp;author=T.-J.%20Chuang&amp;publication_year=2015&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=6&amp;pages=563-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.2015.6.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.2015.6.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.2015.6.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26230526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bashyam</div>
               <div class="given-names">M. D.</div>
            </div>, <div class="name string-name"><div class="surname">Purushotham</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chaudhary</div> <div class="given-names">A. K.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">A low prevalence of MYH7/MYBPC3 mutations among Familial Hypertrophic Cardiomyopathy patients in India,</div><div class="source">Mol. Cell. Biochem</div> ., <div class="volume">360</div>, <div class="fpage">373</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20low%20prevalence%20of%20MYH7%2FMYBPC3%20mutations%20among%20Familial%20Hypertrophic%20Cardiomyopathy%20patients%20in%20India%2C&amp;author=M.%20D.%20Bashyam&amp;author=G.%20Purushotham&amp;author=A.%20K.%20Chaudhary&amp;publication_year=2012&amp;journal=Mol.%20Cell.%20Biochem&amp;volume=360&amp;pages=373-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11010-011-1077-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11010-011-1077-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11010-011-1077-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hershberger</div>
               <div class="given-names">R. E.</div>
            </div>, <div class="name string-name"><div class="surname">Norton</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Morales</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Siegfried</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Gonzalez-Quintana</div> <div class="given-names">J.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Coding sequence rare variants identified in MYBPC3, MYH6, TPM1, TNNC1, and TNNI3 from 312 patients with familial or idiopathic dilated cardiomyopathy,</div><div class="source">Circ. Cardiovasc. Genet</div> ., <div class="volume">3</div>, <div class="fpage">155</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding%20sequence%20rare%20variants%20identified%20in%20MYBPC3%2C%20MYH6%2C%20TPM1%2C%20TNNC1%2C%20and%20TNNI3%20from%20312%20patients%20with%20familial%20or%20idiopathic%20dilated%20cardiomyopathy%2C&amp;author=R.%20E.%20Hershberger&amp;author=N.%20Norton&amp;author=A.%20Morales&amp;author=D.%20Li&amp;author=J.%20D.%20Siegfried&amp;author=J.%20Gonzalez-Quintana&amp;publication_year=2010&amp;journal=Circ.%20Cardiovasc.%20Genet&amp;volume=3&amp;pages=155-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1161/CIRCGENETICS.109.912345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1161%2FCIRCGENETICS.109.912345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1161%2FCIRCGENETICS.109.912345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20215591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lagos-Quintana</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Rauhut</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yalcin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lendeckel</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Tuschl</div> <div class="given-names">T.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">Identification of tissue-specific microRNAs from mouse,</div><div class="source">Curr Biol</div> ., <div class="volume">12</div>, <div class="fpage">735</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20tissue-specific%20microRNAs%20from%20mouse%2C&amp;author=M.%20Lagos-Quintana&amp;author=R.%20Rauhut&amp;author=A.%20Yalcin&amp;author=J.%20Meyer&amp;author=W.%20Lendeckel&amp;author=T.%20Tuschl&amp;publication_year=2002&amp;journal=Curr%20Biol&amp;volume=12&amp;pages=735-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)00809-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)00809-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)00809-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12007417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Maki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">L.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Genome-wide survey of tissue-specific microRNA and transcription factor regulatory networks in 12 tissues,</div><div class="source">Sci. Rep</div> ., <div class="volume">4</div>, <div class="fpage">5150</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20tissue-specific%20microRNA%20and%20transcription%20factor%20regulatory%20networks%20in%2012%20tissues%2C&amp;author=Z.%20Guo&amp;author=M.%20Maki&amp;author=R.%20Ding&amp;author=Y.%20Yang&amp;author=B.%20Zhang&amp;author=L.%20Xiong&amp;publication_year=2014&amp;journal=Sci.%20Rep&amp;volume=4&amp;pages=5150" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep05150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep05150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep05150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24889152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Uhlen</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Fagerberg</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Hallstrom</div> <div class="given-names">B. M.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Proteomics. Tissue-based map of the human proteome,</div><div class="source">Science</div> , <div class="volume">347</div>, <div class="fpage">1260419</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomics.%20Tissue-based%20map%20of%20the%20human%20proteome%2C&amp;author=M.%20Uhlen&amp;author=L.%20Fagerberg&amp;author=B.%20M.%20Hallstrom&amp;publication_year=2015&amp;journal=Science&amp;volume=347&amp;pages=1260419" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1260419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1260419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1260419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25613900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dahmane Gošnak</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Eržen</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Holcman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Škorjanc</div> <div class="given-names">D.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Effects of divergent selection for 8-week body weight on postnatal enzyme activity pattern of 3 fiber types in fast muscles of male broilers (<em>Gallus gallus domesticus</em>),</div><div class="source">Poult. Sci</div> ., <div class="volume">89</div>, <div class="fpage">2651</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20divergent%20selection%20for%208-week%20body%20weight%20on%20postnatal%20enzyme%20activity%20pattern%20of%203%20fiber%20types%20in%20fast%20muscles%20of%20male%20broilers%20(Gallus%20gallus%20domesticus)%2C&amp;author=R.%20Dahmane%20Go%C5%A1nak&amp;author=I.%20Er%C5%BEen&amp;author=A.%20Holcman&amp;author=D.%20%C5%A0korjanc&amp;publication_year=2010&amp;journal=Poult.%20Sci&amp;volume=89&amp;pages=2651-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3382/ps.2010-00641" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3382%2Fps.2010-00641" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3382%2Fps.2010-00641"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Drummond</div>
               <div class="given-names">M. J.</div>
            </div>, <div class="name string-name"><div class="surname">McCarthy</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Fry</div> <div class="given-names">C. S.</div></div>, <div class="name string-name"><div class="surname">Esser</div> <div class="given-names">K. A.</div></div>, <div class="name string-name"><div class="surname">Rasmussen</div> <div class="given-names">B. B.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Aging differentially affects human skeletal muscle microRNA expression at rest and after an anabolic stimulus of resistance exercise and essential amino acids,</div><div class="source">Am. J. Physiol. –Endocrinol. Metab</div> ., <div class="volume">295</div>, <div class="fpage">E1333</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aging%20differentially%20affects%20human%20skeletal%20muscle%20microRNA%20expression%20at%20rest%20and%20after%20an%20anabolic%20stimulus%20of%20resistance%20exercise%20and%20essential%20amino%20acids%2C&amp;author=M.%20J.%20Drummond&amp;author=J.%20J.%20McCarthy&amp;author=C.%20S.%20Fry&amp;author=K.%20A.%20Esser&amp;author=B.%20B.%20Rasmussen&amp;publication_year=2008&amp;journal=Am.%20J.%20Physiol.%20%E2%80%93Endocrinol.%20Metab&amp;volume=295&amp;pages=E1333-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/ajpendo.90562.2008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fajpendo.90562.2008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fajpendo.90562.2008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18827171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hitachi</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Tsuchida</div> <div class="given-names">K.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Role of microRNAs in skeletal muscle hypertrophy,</div><div class="source">Front. Physiol,</div> , <div class="volume">4</div>, <div class="fpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20microRNAs%20in%20skeletal%20muscle%20hypertrophy%2C&amp;author=K.%20Hitachi&amp;author=K.%20Tsuchida&amp;publication_year=2014&amp;journal=Front.%20Physiol%2C&amp;volume=4&amp;pages=408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fphys.2013.00408" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffphys.2013.00408" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffphys.2013.00408"> </span></div> </div></div></div></div></div><div content-id="dsx022-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De La Cruz</div>
               <div class="given-names">E. M.</div>
            </div>, <div class="name string-name"><div class="surname">Ostap</div> <div class="given-names">E. M.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Relating biochemistry and function in the myosin superfamily,</div><div class="source">Curr. Opin. Cell Biol</div> ., <div class="volume">16</div>, <div class="fpage">61</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relating%20biochemistry%20and%20function%20in%20the%20myosin%20superfamily%2C&amp;author=E.%20M.%20De%20La%20Cruz&amp;author=E.%20M.%20Ostap&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=16&amp;pages=61-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2003.11.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2003.11.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2003.11.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15037306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akolkar</div>
               <div class="given-names">D. B.</div>
            </div>, <div class="name string-name"><div class="surname">Kinoshita</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yasmin</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Fibre type-specific expression patterns of myosin heavy chain genes in adult torafugu Takifugu rubripes muscles,</div><div class="source">J. Exp. Biol</div> ., <div class="volume">213</div>, <div class="fpage">137</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fibre%20type-specific%20expression%20patterns%20of%20myosin%20heavy%20chain%20genes%20in%20adult%20torafugu%20Takifugu%20rubripes%20muscles%2C&amp;author=D.%20B.%20Akolkar&amp;author=S.%20Kinoshita&amp;author=L.%20Yasmin&amp;publication_year=2010&amp;journal=J.%20Exp.%20Biol&amp;volume=213&amp;pages=137-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jeb.030759" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjeb.030759" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjeb.030759"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20008370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lim</div>
               <div class="given-names">K.-S.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.-H.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">E.-A.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.-M.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">K.-C.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Effects of intergenic single nucleotide polymorphisms in the fast myosin heavy chain cluster on muscle fiber characteristics and meat quality in Berkshire pigs,</div><div class="source">Meat Sci</div> ., <div class="volume">110</div>, <div class="fpage">224</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20intergenic%20single%20nucleotide%20polymorphisms%20in%20the%20fast%20myosin%20heavy%20chain%20cluster%20on%20muscle%20fiber%20characteristics%20and%20meat%20quality%20in%20Berkshire%20pigs%2C&amp;author=K.-S.%20Lim&amp;author=S.-H.%20Lee&amp;author=E.-A.%20Lee&amp;author=J.-M.%20Kim&amp;author=K.-C.%20Hong&amp;publication_year=2015&amp;journal=Meat%20Sci&amp;volume=110&amp;pages=224-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.meatsci.2015.07.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.meatsci.2015.07.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.meatsci.2015.07.025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26263039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zou</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">B.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Moderately increased maternal dietary energy intake delays foetal skeletal muscle differentiation and maturity in pigs</div>, <div class="source">Eur. J. Nutr</div> ., <div class="volume">55</div>, <div class="fpage">1777</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Moderately%20increased%20maternal%20dietary%20energy%20intake%20delays%20foetal%20skeletal%20muscle%20differentiation%20and%20maturity%20in%20pigs&amp;author=T.%20Zou&amp;author=D.%20He&amp;author=B.%20Yu&amp;publication_year=2015&amp;journal=Eur.%20J.%20Nutr&amp;volume=55&amp;pages=1777-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00394-015-0996-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00394-015-0996-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00394-015-0996-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26179476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xiong</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Genome-wide association analysis reveals genetic loci and candidate genes for meat quality traits in Chinese Laiwu pigs,</div><div class="source">Mamm. Genome</div> , <div class="volume">26</div>, <div class="fpage">181</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20reveals%20genetic%20loci%20and%20candidate%20genes%20for%20meat%20quality%20traits%20in%20Chinese%20Laiwu%20pigs%2C&amp;author=X.%20Xiong&amp;author=X.%20Liu&amp;author=L.%20Zhou&amp;publication_year=2015&amp;journal=Mamm.%20Genome&amp;volume=26&amp;pages=181-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-015-9558-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-015-9558-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-015-9558-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25678226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">F.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification,</div><div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CIRI%3A%20an%20efficient%20and%20unbiased%20algorithm%20for%20de%20novo%20circular%20RNA%20identification%2C&amp;author=Y.%20Gao&amp;author=J.%20Wang&amp;author=F.%20Zhao&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0571-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0571-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0571-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Y.-C.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.-R.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">C.-H.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">CircNet: a database of circular RNAs derived from transcriptome sequencing data,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D209</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CircNet%3A%20a%20database%20of%20circular%20RNAs%20derived%20from%20transcriptome%20sequencing%20data%2C&amp;author=Y.-C.%20Liu&amp;author=J.-R.%20Li&amp;author=C.-H.%20Sun&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D209-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv940" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv940" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv940"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26450965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Capel</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Swain</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nicolis</div> <div class="given-names">S.</div></div></div>, et al.  <div class="article-title">Circular transcripts of the testis-determining gene &lt;em&gt;Sry&lt;/em&gt; in adult mouse testis,</div><div class="source">Cell</div> , <div class="volume">73</div>, <div class="fpage">1019</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90279-Y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90279-Y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90279-Y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7684656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Transcriptome-wide investigation of circular RNAs in rice,</div><div class="source">RNA</div> , <div class="volume">21</div>, <div class="fpage">2076</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome-wide%20investigation%20of%20circular%20RNAs%20in%20rice%2C&amp;author=T.%20Lu&amp;author=L.%20Cui&amp;author=Y.%20Zhou&amp;publication_year=2015&amp;journal=RNA&amp;volume=21&amp;pages=2076-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.052282.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.052282.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.052282.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26464523" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jeck</div>
               <div class="given-names">W. R.</div>
            </div>, <div class="name string-name"><div class="surname">Sharpless</div> <div class="given-names">N. E.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Detecting and characterizing circular RNAs,</div><div class="source">Nat. Biotech</div> ., <div class="volume">32</div>, <div class="fpage">453</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20and%20characterizing%20circular%20RNAs%2C&amp;author=W.%20R.%20Jeck&amp;author=N.%20E.%20Sharpless&amp;publication_year=2014&amp;journal=Nat.%20Biotech&amp;volume=32&amp;pages=453-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2890"> </span></div> </div></div></div></div></div><div content-id="dsx022-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Laitinen</div>
               <div class="given-names">P. J.</div>
            </div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Piippo</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2001</div>, 
<div class="article-title">Mutations of the cardiac ryanodine receptor (RyR2) gene in familial polymorphic ventricular tachycardia,</div><div class="source">Circulation</div> , <div class="volume">103</div>, <div class="fpage">485</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20of%20the%20cardiac%20ryanodine%20receptor%20(RyR2)%20gene%20in%20familial%20polymorphic%20ventricular%20tachycardia%2C&amp;author=P.%20J.%20Laitinen&amp;author=K.%20M.%20Brown&amp;author=K.%20Piippo&amp;publication_year=2001&amp;journal=Circulation&amp;volume=103&amp;pages=485-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1161/01.CIR.103.4.485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1161%2F01.CIR.103.4.485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1161%2F01.CIR.103.4.485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11157710" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Meredith</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Herrmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Parry</div> <div class="given-names">C.</div></div></div>, et al.  <div class="article-title">Mutations in the slow skeletal muscle fiber myosin heavy chain gene (&lt;em&gt;MYH7&lt;/em&gt;) cause laing early-onset distal myopathy (MPD1),</div><div class="source">Am. J. Hum. Genet</div> ., <div class="volume">75</div>, <div class="fpage">703</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/424760" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F424760" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F424760"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15322983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Walsh</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Rutland</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Loughna</div> <div class="given-names">S.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Cardiomyopathy: a systematic review of disease-causing mutations in myosin heavy chain 7 and their phenotypic manifestations,</div><div class="source">Cardiology</div> , <div class="volume">115</div>, <div class="fpage">49</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cardiomyopathy%3A%20a%20systematic%20review%20of%20disease-causing%20mutations%20in%20myosin%20heavy%20chain%207%20and%20their%20phenotypic%20manifestations%2C&amp;author=R.%20Walsh&amp;author=C.%20Rutland&amp;author=R.%20Thomas&amp;author=S.%20Loughna&amp;publication_year=2010&amp;journal=Cardiology&amp;volume=115&amp;pages=49-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000252808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000252808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000252808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19864899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schiaffino</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Dyar</div> <div class="given-names">K. A.</div></div>, <div class="name string-name"><div class="surname">Ciciliot</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Blaauw</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Sandri</div> <div class="given-names">M.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Mechanisms regulating skeletal muscle growth and atrophy,</div><div class="source">FEBS J</div> ., <div class="volume">280</div>, <div class="fpage">4294</div>–<div class="lpage">314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20regulating%20skeletal%20muscle%20growth%20and%20atrophy%2C&amp;author=S.%20Schiaffino&amp;author=K.%20A.%20Dyar&amp;author=S.%20Ciciliot&amp;author=B.%20Blaauw&amp;author=M.%20Sandri&amp;publication_year=2013&amp;journal=FEBS%20J&amp;volume=280&amp;pages=4294-314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/febs.2013.280.issue-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffebs.2013.280.issue-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffebs.2013.280.issue-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23517348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">González-Mariscal</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Domínguez-Calderón</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Raya-Sandino</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ortega-Olvera</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Vargas-Sierra</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Martínez-Revollar</div> <div class="given-names">G.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Tight junctions and the regulation of gene expression,</div><div class="source">Semin. Cell Dev. Biol</div> ., <div class="volume">36</div>, <div class="fpage">213</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tight%20junctions%20and%20the%20regulation%20of%20gene%20expression%2C&amp;author=L.%20Gonz%C3%A1lez-Mariscal&amp;author=A.%20Dom%C3%ADnguez-Calder%C3%B3n&amp;author=A.%20Raya-Sandino&amp;author=J.%20M.%20Ortega-Olvera&amp;author=O.%20Vargas-Sierra&amp;author=G.%20Mart%C3%ADnez-Revollar&amp;publication_year=2014&amp;journal=Semin.%20Cell%20Dev.%20Biol&amp;volume=36&amp;pages=213-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.semcdb.2014.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcdb.2014.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.semcdb.2014.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25152334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">González-Mariscal</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Lechuga</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Garay</div> <div class="given-names">E.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Role of tight junctions in cell proliferation and cancer,</div><div class="source">Prog. Histochem. Cytochem</div> ., <div class="volume">42</div>, <div class="fpage">1</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20tight%20junctions%20in%20cell%20proliferation%20and%20cancer%2C&amp;author=L.%20Gonz%C3%A1lez-Mariscal&amp;author=S.%20Lechuga&amp;author=E.%20Garay&amp;publication_year=2007&amp;journal=Prog.%20Histochem.%20Cytochem&amp;volume=42&amp;pages=1-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.proghi.2007.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.proghi.2007.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.proghi.2007.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17502225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Salzman</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Gawad</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">P. L.</div></div>, <div class="name string-name"><div class="surname">Lacayo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Circular RNAs are the predominant transcript isoform from hundreds of human genes in diverse cell types</div>. <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e30733</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20RNAs%20are%20the%20predominant%20transcript%20isoform%20from%20hundreds%20of%20human%20genes%20in%20diverse%20cell%20types&amp;author=J.%20Salzman&amp;author=C.%20Gawad&amp;author=P.%20L.%20Wang&amp;author=N.%20Lacayo&amp;author=P.%20O.%20Brown&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30733" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030733" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030733" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030733"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ghosal</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Das</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sen</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Basak</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Chakrabarti</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Circ2Traits: a comprehensive database for circular RNA potentially associated with disease and traits,</div><div class="source">Front. Genet</div> ., <div class="volume">4</div>, <div class="fpage">283</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circ2Traits%3A%20a%20comprehensive%20database%20for%20circular%20RNA%20potentially%20associated%20with%20disease%20and%20traits%2C&amp;author=S.%20Ghosal&amp;author=S.%20Das&amp;author=R.%20Sen&amp;author=P.%20Basak&amp;author=J.%20Chakrabarti&amp;publication_year=2013&amp;journal=Front.%20Genet&amp;volume=4&amp;pages=283" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2013.00283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2013.00283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2013.00283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24339831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zheng</div>
               <div class="given-names">L.-L.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.-H.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">deepBase v2.0: identification, expression, evolution and function of small RNAs, LncRNAs and circular RNAs from deep-sequencing data,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D196</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=deepBase%20v2.0%3A%20identification%2C%20expression%2C%20evolution%20and%20function%20of%20small%20RNAs%2C%20LncRNAs%20and%20circular%20RNAs%20from%20deep-sequencing%20data%2C&amp;author=L.-L.%20Zheng&amp;author=J.-H.%20Li&amp;author=J.%20Wu&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D196-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26590255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx022-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dudekula</div>
               <div class="given-names">D. B.</div>
            </div>, <div class="name string-name"><div class="surname">Panda</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Grammatikakis</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">De</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Abdelmohsen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Gorospe</div> <div class="given-names">M.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">CircInteractome: a web tool for exploring circular RNAs and their interacting proteins and microRNAs</div>, <div class="source">RNA Biol</div> ., <div class="volume">13</div>, <div class="fpage">34</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CircInteractome%3A%20a%20web%20tool%20for%20exploring%20circular%20RNAs%20and%20their%20interacting%20proteins%20and%20microRNAs&amp;author=D.%20B.%20Dudekula&amp;author=A.%20C.%20Panda&amp;author=I.%20Grammatikakis&amp;author=S.%20De&amp;author=K.%20Abdelmohsen&amp;author=M.%20Gorospe&amp;publication_year=2015&amp;journal=RNA%20Biol&amp;volume=13&amp;pages=34-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15476286.2015.1128065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15476286.2015.1128065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15476286.2015.1128065"> </span></div> </div></div></div></div></div><div content-id="dsx022-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx022-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bhattacharya</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">SomamiR 2.0: a database of cancer somatic mutations altering microRNA–ceRNA interactions,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D1005</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SomamiR%202.0%3A%20a%20database%20of%20cancer%20somatic%20mutations%20altering%20microRNA%E2%80%93ceRNA%20interactions%2C&amp;author=A.%20Bhattacharya&amp;author=Y.%20Cui&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D1005-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26578591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3857773&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_figure.pptx?Expires=2147483647&amp;Signature=XFpS1V6dtdgSXItn2-4fmXxp2Y5PNodsMGhC8XXsAI5QZ-PXvU-bor061xLtYuLPJr7zFQzMHRBWywT9aR8WAtzWjKORl2-7rsJ2GNU52ytKWD4nmASca9S1zucOfvh3vWG5c9nnnQw5G4wMEfoviGUCaCl4WMClDzPza3inTq22kSHufedjwJ-t1IZbuobajWljDziJiOBLSyAI48xbTc6ddyiFd5uohBznC3ejOvqhpQfT2cyY8a2MGW2ERAQdYoRxWtSeuZLQvjvlDibCIpgzz3XK97m1IdK1k~CNtqJ~QJDTTLZhCSD67PnxnbHViATplB2nGMm-MmB25mzFNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pptx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s1.xlsx?Expires=2147483647&amp;Signature=RuKJNnQQTnfx52hL3dQZsj9p5tIxvtJhVKqT3zQ1qV8AWYxXrsq33S6~QUZahTN~s37ReV5PEOIGZeJq07LaQK~oqrGIsQ5szpq69rmfef2GaiKcoLPXhScg3e1YdYZj9lM7SVTrEN7VOEOJPAX4nWCEe1~Gqc3ClT0TmmW35gpCkGgHzeMeFNZtBayA5Y2NEFPmf81kEfpGHI6O~VJEk2WVy9w3xrbA6WOyts9b4xy4KrooQ1nAaReAuzIw1Ecwbu8KctRbLibVTcvF1aX2juuTGWfBL5l-7Lci3hD~mY5Ydq6eLi4QlYioUGhOjhY7gqZKHL3M~Bc-KjcIFHFmRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s2.xlsx?Expires=2147483647&amp;Signature=NExjX7EeMoapM9m8lENrtnDrVHUTJIyqceT1rt2flYOkZbnHZW6wDQzPxGME8PyX~eK2eMAtNN9n5eT6Qf8XKWaMjLtrl2ZiKPozXUNX0IGczuTre03P9Pk4dG-Nt52nYQR5sKaAS9ovPFkSn6M~aE94xczX4~gM1qbehf4Tz9dIOT1j6UfWjgGIxOtUERY3u9Zep2zXi~JO6F-20jhVWvNuL8kpCuaBG3Y0dglPNYyN~8h2NjZc8T-lp9xFiZNMVbJLqw~GOwY6wAvVWi405rZwpt-6CRBm85eu4JOoMOF4KP5Z94dp7G1fC6f4tu0E0854k0pjfvPC0PauhDX4Lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s3.xlsx?Expires=2147483647&amp;Signature=AdkRoI0PDxlLMpihy7UedEr3ONk8nPSFO~szt9SS3afsFPhe0uWARHbaVJ9eyNhOVhDJcappFOnhoNTbA2nbpajd~r99aDfyHSF6F6WJkaVjxFmVaUUtMFdwMaGX4ZtIdNDyRchm55T7pLbQHV8ETMlLWLUyBUnJv79iS0f-p6yuUWvzYhwFZS2kDssPLgyqc1-cKYO30Y5f7Yaig-6CPKnPCC9sTMZ5HHOegN74kmKQkSi6K56H2Z-C-OJCs0h~Z9BLNEm56dzZ9ctPnA9SHg-skku4vQPNoRp39cmRd3m6dI~zL5AnTHoFZtG4LW4F08C~HVhW5jFi64QqzajUvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s4.xlsx?Expires=2147483647&amp;Signature=XpnLrKi7gUj5uOWxAtULHIh8CwbTOle4W1hOfdtY8Y1kihSDva55S9GBIl-RQCyQ567Lk6~GONQ6fB4AGUz6zOrTU8jHk7rOdlovMuYh93S~NqFuyRCkBC8WZlp29q~DulI7wp5VZmHy0XUGj8ImU-pFUOMEZEes-90NfhTrXKFBKLMGV9IF8aqGeGxcy3eS~FdwH6O6yMXTEUZSsudnp20iYchhbUuWfEbxswIOA~PS69NYCfIB4aWm-wTfVvbvvR1NdbNuJNa2jOT8onouWJsuJy6of~eddrH0IHShsMJ8URak6PGQCgeuJLOGaeQGnh6iAcjdUWhDvUmwjEnTNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s5.xlsx?Expires=2147483647&amp;Signature=omFH6Q6tvOebkpN98LLXBBhN6fHk29sLo6acWgeHIktl-GkP9P9miJ~C2oSMKkz5sfOS0RXzJdp1uGbSccFun5GVppdL6Tmg2bD-4huracFaKaRgC2EXi~o8e7RAe9DVobVOstrORu~9DIO6JTzbnswQBvMYZ~T-ZFGuy8PArZaGEjSL3JUl3fi4cE1OQWas9X0w9ci5Ki4XGkero9WE-oyqBsawAucy-tneVyxAzbhVqiphbO5rCDrZyT9tZklIixp3isEvFBNJoSMmxwEVnoAv84l3nv4my8BAuu4XAo0WTKbgMJ4oYOt~J38avd70jX6IAIcNt63ORR6VdyyLLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s6.xlsx?Expires=2147483647&amp;Signature=H2BsLWuD656yVe0AZ~xmxQOw~k1i4esgKwcym4uYvpob2sEQ-tG~~C~i9ognTlMdkmwn2bxqHJ5UUU7cp-49Td~APnrDTOWbiz~Si1Fa4xUI4ABhv9qR-5JqAfs1D9-j8vtoDvC6uZnD3GaLj8~B23D3hjZErxUNmaX4O32dNIF6vkpuDsmdlrFrFiuf5BPI~wOg1T8m~pNhQoOmrvgFyhsGYXOBMMX-ywGBoEgT2YHNiHPrXrAcV3RLnsYDrZsSQ82r9itZsqpoUWLrONHTZq~4VfySnb5pKPdIwne6pmal8yAdWbDpZ6naSbdlc5osdGAPFRKwIf3DbUFpKbyD5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s7.xls?Expires=2147483647&amp;Signature=vlzUsYNlvu4lHzOpk3A5e0oMk-VNLX~3BTHOWBrqtmrshUyZOUDNgWjOBWLrKxvqQzGYpEaZ~Li3VjvSa0Xo4SYtGRBtZvtpYzs~0dVxmW52akm6p4og~8sBJcUOhZLLIOtGvNJsaKRJyTVLc1-vEVl29r1n5SnX6twKmb66vUJENIcBboravALzRZOkngdq52N5-oQfuIOlbSA7a7CDryuvJBkveSjZeOKNJfH6qt4W9jCH40mFIhIwlJxC2-oD8Yqp1ZfIh6RgjAI1-wb26d~VmbuXhBMsALFZFhUqgzlxPJmAgeYBeKLcdBK-wG4FAJTVlYh3nHv7irZq2igvjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s8.xlsx?Expires=2147483647&amp;Signature=l7nDPPsjIb9xwcQtWHLN0qJyUz-YAHy4ZnNqbogvm0vJPoPuvOjp4r~l9AoYRvENvErZp4J~5M63riz~ZeNeakMNxJAuVjhu545Sl8~Iw6bNjT2A5C3-jOBg62gq6WjP3yX769bP9-Zm8PHHITaz13qjJtOYk9c2JhORJtIBvI7S5M~XA~VkSbRr6hT8ctka4GY3sSFwXvo-gW52gyqEfg9xi6wVZ8qPUBWo4bWkig2FnfgEwYX-YG1C7afdsWXufYXjNIPbcRvIYKyo-IVCjhZwR87EDixCzhS2~IEWGZ-9ozX8iNjuhDM6GCMxEmgk3mWesFtS~tjw19SHwB3b4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s9.xlsx?Expires=2147483647&amp;Signature=eXedVugIRXNe8f5ed6-0zO~NdjHACIbU5X0GLAtBMkhrpIMkU6W9vIJvPSKmqaC2MCbZqJRUKgGbBFGM99Yn4OKrzfBguVQ8Kv43kf0HlzkTkUpJeSZYizxmnZXpJxIEkzNSQUMAZ~j1E62BJyCifXQKcA4IFk99bDHzRiez6QLn-d9Ok3T-FIP0KZ3Ck2zRn7nJPgsHYDSTPSMbZj9NDyvPaAMp2~0mxPmF4IfdhwbXUGg6Gyh~vAaDflnwaTqYwkWckfZRCtFY0wnlZyP-n6pUV5gQ7ezJBPGaoFjG93M7b14R8BaXc-5yI17RP9rFG6ooPSIwN-SVr46Xb2b53A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s10.xlsx?Expires=2147483647&amp;Signature=zvYBJWusLGpT6cn4-fMMV-vQfBF6oxSuOqTcTU6xZAAWRCuSvcxBRxrWYXczToeByIqcCzqobq1dniwVF6Xh7KoyNXBEJ68XQhmFzsuIvVs-yq-RaLtFuBFNQ8jY1l6--9WlY3vO-E8ZM0cIbghceRcJfGLuBmj2H66Ot~HoviuaTveZppCPCnhy6dTWnuEgvriRcLJ4UV5s7dETBuwE5-OeZ43Etf5su12Wm338RuuznXBIpTECFMpmo0lWjKWCXfaLXXT6if9Tunlsg5ZITiwu0PyFHqqWk4-zh3j9PHRTz-dgA3-buqruxmPS4Pr~FiiLnCcI69o-Oshwc-9u0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s11.xls?Expires=2147483647&amp;Signature=OAtIvyfQ5pzC-2LNIqWvlfX~NfvsRDPFBtGT4R02M3YhivagWW-MvM6RjHrEc8~IrpZ27-mJMlVohTs0zJFZiv2Fh9o87f3g9GcGj-dklmv-dnnHuvrT7P5eIo95wPt-TAhiuYulbRsXzF9CnL1y69VRXadFfK6Eqb5mnepDALSRYAOW-yNv8OO~Sh2WOuKloTazChFZQbikCeD5mK~Q5g-pgetpHhV61vsF-y4IRex9jikojPsgKKwhitRbN51mH62zJB2Umc~lRvB4b7keHXcjwxSRFycsKCA-HltvNqCRHNaQfyQN5XtphFeVtDDuecr2oxajeSG5AWY8geXYrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s12.xls?Expires=2147483647&amp;Signature=bWXY-UYi16eNWh-QGUDO4AEQK4BN5VHIdMixVHpqy-ioVG5CjlBkWGLTB1o1yIm6hz9oOP~fgTyumZVsezOxNB7yPN7jNWCD-v-sWUnJRONuzikxMEM6i8FeZ6HS-28XTjHNpW1Anp8rR713HT~xrGgl4q3Tz2itMdfeJRtkTySGTqvY4C-H15cuMzhY1MPLkQzXv3BvfKN0fTlsNlRq5g4Ze2HwIosUDycmjk-yDd1WRW5poyfgEdHrt-5HTzK328wcsTVnVH5lLt5EJjsxE9Ro5LtBpTOHrEhAVw4dVWXF6qMzDXVHwLkT0HGVGIrzI-gamo6MpTaQSdccaJkwNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s13.xls?Expires=2147483647&amp;Signature=q3VaPUh-Xh6J6FM~8bkPosWae5-DMe5PHXabjRxOlCaURv~7Rz10CKep~s~8~Qijub-h~KmL~iH7PMMDYBHcRfD-NN4Eth4ntm6p6r337lOyhKUwtZqjirvexl3J8PrlAHONOUQoAwPlhiR6kG9yBy0TzvetSisgD4t8MTLngz0UW9wI~a2cOsAph0UVEJvJoYmorhD~D0wxL8OrPfG9vDRme1n-F1DTGkaQ85nOm-kGaN2Ul0Xtwm4sruTWWmJ9ouxs~YF8XE0JI~OGiNXSqh7nijO1q3Albe~2WHFM8Z-sl0lTpVXblLi0JXKUz9hF5t~nMken1urdeRlNl1Y9hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx022/2/dsx022_suppl_table-s14.xls?Expires=2147483647&amp;Signature=fQBIQ1vyvtBHgQRmvIxQf8KzXiotZ3eQHoWLKn8VOYblZUJ1J0KZ2vDc0C2878Y7gTyJ8klDlVMqpaxgiar~KGu2T1vdSsoH7i-sBw3BBI0iaHP33GkTwF6p~zcApcoprnXnZiBX5SMxmHGk6q9cGhHZ5uSlGUvNlEPERQM2iGy4i9ypwS8zLnIQLlMySFq2hpNTgOGTbWnDJAxXJR879twE6C4EyswLxlI38tobH-psI3Z9DZ4e4Bwse3EFthUgD8Nwan0HS0uRc2EVknzqKUyNcnQ0lLmrwVN-lxsc50IPjx-06bfPgJ3CIKFYmOwihY2LMidjWggHpa9AfwXagQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,668</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3857773"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f5%2f523%2f3857773">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000413018400007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F5%2F523%2F3857773" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3857773'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide profiling of Sus scrofa circular RNAs across nine organs and three developmental stages" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f523%2f3857773">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

