

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/6/535/2647444","@type":"ScholarlyArticle","name":"Draft genome sequence of an inbred line of  Chenopodium quinoa , an allotetraploid crop with great environmental adaptability and outstanding nutritional properties","datePublished":"2016-07-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2016-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw037","keywords":["<italic>Chenopodium quinoa</italic>","draft genome","NGS","inbred accession"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/6/535/2647444","author":[{"name":"Yasui, Yasuo","affiliation":"1 Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"2 Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Oikawa, Tetsuo","affiliation":"3 Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan","@type":"Person"},{"name":"Toyoshima, Masami","affiliation":"3 Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan","@type":"Person"},{"name":"Matsuzaki, Chiaki","affiliation":"4 Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan","@type":"Person"},{"name":"Ueno, Mariko","affiliation":"1 Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan","@type":"Person"},{"name":"Mizuno, Nobuyuki","affiliation":"1 Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan","@type":"Person"},{"name":"Nagatoshi, Yukari","affiliation":"3 Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan","@type":"Person"},{"name":"Imamura, Tomohiro","affiliation":"4 Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan","@type":"Person"},{"name":"Miyago, Manami","affiliation":"5 Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan","@type":"Person"},{"name":"Tanaka, Kojiro","affiliation":"5 Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan","@type":"Person"},{"name":"Mise, Kazuyuki","affiliation":"1 Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan","@type":"Person"},{"name":"Tanaka, Tsutomu","affiliation":"5 Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan","@type":"Person"},{"name":"Mizukoshi, Hiroharu","affiliation":"5 Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan","@type":"Person"},{"name":"Mori, Masashi","affiliation":"4 Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan","@type":"Person"},{"name":"Fujita, Yasunari","affiliation":"3 Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan","@type":"Person"}],"description":"Chenopodium quinoa Willd. (quinoa) originated from the Andean region of South America, and is a pseudocereal crop of the Amaranthaceae family. Quinoa is emergin","pageStart":"535","pageEnd":"546","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Draft genome sequence of an inbred line of  Chenopodium quinoa , an allotetraploid crop with great environmental adaptability and outstanding nutritional properties","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Draft genome sequence of an inbred line of  Chenopodium quinoa , an allotetraploid crop with great environmental adaptability and outstanding nutritional properties" />
    <meta property="og:description" content="Chenopodium quinoa Willd. (quinoa) originated from the Andean region of South America, and is a pseudocereal crop of the Amaranthaceae family. Quinoa is emergin" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/6/535/2647444" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yasui, Yasuo" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Oikawa, Tetsuo" /><meta name="citation_author_institution" content="Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan" /><meta name="citation_author" content="Toyoshima, Masami" /><meta name="citation_author_institution" content="Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan" /><meta name="citation_author" content="Matsuzaki, Chiaki" /><meta name="citation_author_institution" content="Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan" /><meta name="citation_author" content="Ueno, Mariko" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Mizuno, Nobuyuki" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Nagatoshi, Yukari" /><meta name="citation_author_institution" content="Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan" /><meta name="citation_author" content="Imamura, Tomohiro" /><meta name="citation_author_institution" content="Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan" /><meta name="citation_author" content="Miyago, Manami" /><meta name="citation_author_institution" content="Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan" /><meta name="citation_author" content="Tanaka, Kojiro" /><meta name="citation_author_institution" content="Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan" /><meta name="citation_author" content="Mise, Kazuyuki" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Tanaka, Tsutomu" /><meta name="citation_author_institution" content="Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan" /><meta name="citation_author" content="Mizukoshi, Hiroharu" /><meta name="citation_author_institution" content="Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan" /><meta name="citation_author" content="Mori, Masashi" /><meta name="citation_author_institution" content="Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan" /><meta name="citation_author" content="Fujita, Yasunari" /><meta name="citation_author_institution" content="Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan" /><meta name="citation_title" content="Draft genome sequence of an inbred line of Chenopodium quinoa, an allotetraploid crop with great environmental adaptability and outstanding nutritional properties" /><meta name="citation_firstpage" content="535" /><meta name="citation_lastpage" content="546" /><meta name="citation_doi" content="10.1093/dnares/dsw037" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2016/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A genetic linkage map of quinoa (Chenopodium quinoa) based on AFLP, RAPD, and SSR markers;  citation_journal_title=Theor. Appl. Genet;  citation_year=2004;  citation_volume=109;  citation_pages=1188-95; " /><meta name="citation_reference" content="citation_title=Breeding quinoa (Chenopodium quinoa Willd.): potential and perspectives;  citation_journal_title=Mol. Breed;  citation_year=2014;  citation_volume=34;  citation_pages=13-30; " /><meta name="citation_reference" content="citation_title=The pattern of genetic diversity in the Andean grain crop quinoa (Chenopodium quinoa Willd.). I. Associations between characteristics;  citation_journal_title=Euphytica;  citation_year=1989;  citation_volume=41;  citation_pages=147-62; " /><meta name="citation_reference" content="citation_title=Morphological analysis of quinoa germplasm grown under lowland desert conditions;  citation_journal_title=J. Agron. Crop Sci;  citation_year=2011;  citation_volume=197;  citation_pages=124-34; " /><meta name="citation_reference" content="citation_title=Genotypic variation of gas exchange parameters and leaf stable carbon and nitrogen isotopes in ten quinoa cultivars grown under drought;  citation_journal_title=J. Agron. Crop Sci;  citation_year=2011;  citation_volume=197;  citation_pages=81-93; " /><meta name="citation_reference" content="citation_title=Re-introduction of quínoa into arid Chile: cultivation of two lowland races under extremely low irrigation;  citation_journal_title=J. Agron. Crop Sci;  citation_year=2009;  citation_volume=195;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Evapotranspiration analysis and irrigation requirements of quinoa (Chenopodium quinoa) in the Bolivian highlands;  citation_journal_title=Agric. Water Manage;  citation_year=2003;  citation_volume=60;  citation_pages=119-34; " /><meta name="citation_reference" content="citation_title=Responses of two main Andean crops, quinoa (Chenopodium quinoa Willd) and papa amarga (Solanum juzepczukii Buk.) to drought on the Bolivian Altiplano: significance of local adaptation;  citation_journal_title=Agric. Ecosyst. Environ;  citation_year=1998;  citation_volume=68;  citation_pages=99-108; " /><meta name="citation_reference" content="citation_title=Water relations and transpiration of quinoa (Chenopodium quinoa Willd.) under salinity and soil drying;  citation_journal_title=J. Agron. Crop Sci;  citation_year=2011;  citation_volume=197;  citation_pages=348-60; " /><meta name="citation_reference" content="citation_title=Quinoa: an alternative crop for saline soils;  citation_journal_title=J. Exp. Bot;  citation_year=2003;  citation_volume=54;  citation_pages=i25" /><meta name="citation_reference" content="citation_title=Ionic and osmotic relations in quinoa (Chenopodium quinoa Willd.) plants grown at various salinity levels;  citation_journal_title=J. Exp. Bot;  citation_year=2011;  citation_volume=62;  citation_pages=185-93; " /><meta name="citation_reference" content="citation_title=Plant responses of quinoa (Chenopodium quinoa Willd.) to frost at various phenological stages;  citation_journal_title=Eur. J. Agron;  citation_year=2005;  citation_volume=22;  citation_pages=131-9; " /><meta name="citation_reference" content="citation_title=Nutrition facts and functional potential of quinoa (Chenopodium quinoa willd.), an ancient Andean grain: a review;  citation_journal_title=J. Sci. Food Agric;  citation_year=2010;  citation_volume=90;  citation_pages=2541-7; " /><meta name="citation_reference" content="citation_title=Quinoa: nutritional, functional and antinutritional aspects;  citation_journal_title=Crit. Rev. Food Sci. Nutr;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=Assessment of the nutritional composition of quinoa (Chenopodium quinoa Willd.);  citation_journal_title=Food Chem;  citation_year=2016;  citation_volume=193;  citation_pages=47-54; " /><meta name="citation_reference" content="citation_title=Protein content and amino acids profile of pseudocereals;  citation_journal_title=Food Chem;  citation_year=2016;  citation_volume=193;  citation_pages=55-61; " /><meta name="citation_reference" content="citation_publisher=Rome;  citation_title=State of the art report on quinoa around the world in 2013. Food and Agriculture Organization of the United Nations (FAO) &amp; Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD);  citation_year=2015; " /><meta name="citation_reference" content="citation_title=Quinoa: an emerging “new” crop with potential for celss;  citation_journal_title=NASA Tech. Pap;  citation_year=1993;  citation_volume=3422;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Inc, New Jersey;  citation_title=Quinoa: improvement and sustainable production;  citation_year=2015;  citation_pages=1-18; " /><meta name="citation_reference" content="citation_title=Assessment of genetic diversity in the USDA and CIP-FAO international nursery collections of quinoa (Chenopodium quinoa Willd.) using microsatellite markers;  citation_journal_title=Plant Genet. Resour;  citation_year=2007;  citation_volume=5;  citation_pages=82-95; " /><meta name="citation_reference" content="citation_title=Nuclear genome size and chromosome analysis in Chenopodium quinoa and C. berlandieri subsp. nuttalliae;  citation_journal_title=Euphytica;  citation_year=2008;  citation_volume=164;  citation_pages=221-30; " /><meta name="citation_reference" content="citation_title=Cytogenetic analysis of quinoa chromosomes using nanoscale imaging and spectroscopy techniques;  citation_journal_title=Nanoscale Res Lett;  citation_year=2013;  citation_volume=8;  citation_pages=463" /><meta name="citation_reference" content="citation_title=Chenopodium polyploidy inferences from Salt Overly Sensitive 1 (SOS1) data;  citation_journal_title=Am. J. Bot;  citation_year=2015;  citation_volume=102;  citation_pages=533-43; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism identification, characterization, and linkage mapping in quinoa;  citation_journal_title=Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=114-25; " /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Quality of prokaryote genome assembly: indispensable issues of factors affecting prokaryote genome assembly quality;  citation_journal_title=Gene;  citation_year=2012;  citation_volume=505;  citation_pages=365-7; " /><meta name="citation_reference" content="citation_title=Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e47768" /><meta name="citation_reference" content="citation_title=The draft genome of Primula veris yields insights into the molecular basis of heterostyly;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=12" /><meta name="citation_reference" content="citation_title=Deletion of the C-terminal 33 amino acids of cucumber mosaic virus movement protein enables a chimeric brome mosaic virus to move from cell to cell;  citation_journal_title=J. Virol;  citation_year=1997;  citation_volume=71;  citation_pages=2270-6; " /><meta name="citation_reference" content="citation_publisher=IRL Press at Oxford University Press, Oxford;  citation_title=Molecular plant pathology: a practical approach;  citation_year=1992;  citation_volume=vol. I;  citation_pages=23-34; " /><meta name="citation_reference" content="citation_title=Bromovirus movement protein genes play a crucial role in host specificity;  citation_journal_title=J. Virol;  citation_year=1993;  citation_volume=67;  citation_pages=2815-23; " /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons Inc, New Jersey;  citation_title=Current Protocols in Molecular Biology;  citation_year=1995; " /><meta name="citation_reference" content="citation_title=Qualitative de novo analysis of full length cDNA and quantitative analysis of gene expression for common marmoset (Callithrix jacchus) transcriptomes using parallel long-read technology and short-read sequencing;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e100936" /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=EMBnet. J;  citation_year=2011;  citation_volume=17;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=1513-8; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21 Suppl 1;  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1086-92; " /><meta name="citation_reference" content="citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3150-2; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=BRAKER1: unsupervised RNA-seq-based genome annotation with GeneMark-ET and AUGUSTUS;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=767-9; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19 Suppl 2;  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=323" /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes;  citation_journal_title=BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=The Gypsy Database (GyDB) of mobile genetic elements: release 2.0;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D70-4; " /><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference;  citation_journal_title=Genome Inform;  citation_year=2009;  citation_volume=23;  citation_pages=205-11; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W609-12; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol. Biol. Evol;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences;  citation_journal_title=Comput. Appl. Biosci;  citation_year=1992;  citation_volume=8;  citation_pages=275-82; " /><meta name="citation_reference" content="citation_title=Reduced tonoplast fast-activating and slow-activating channel activity is essential for conferring salinity tolerance in a facultative halophyte, quinoa;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=162;  citation_pages=940-52; " /><meta name="citation_reference" content="citation_title=Sensitivity of two quinoa (Chenopodium quinoa Willd.) varieties to progressive drought stress;  citation_journal_title=J. Agron. Crop Sci;  citation_year=2014;  citation_volume=200;  citation_pages=12-23; " /><meta name="citation_reference" content="citation_title=Salt tolerance mechanisms in quinoa (Chenopodium quinoa Willd.);  citation_journal_title=Environ. Exp. Bot;  citation_year=2013;  citation_volume=92;  citation_pages=43-54; " /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Inc, New Jersey;  citation_title=Current Protocols in Microbiology;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=A virus made from parts of the genomes of brome mosaic and cowpea chlorotic mottle virus. J. Gen.;  citation_journal_title=Virol;  citation_year=1972;  citation_volume=14;  citation_pages=223-8; " /><meta name="citation_reference" content="citation_title=Molecular studies on bromovirus capsid protein. IV. Coat protein exchanges between brome mosaic and cowpea chlorotic mottle viruses exhibit neutral effects in heterologous hosts;  citation_journal_title=Virology;  citation_year=1997;  citation_volume=238;  citation_pages=452-9; " /><meta name="citation_reference" content="citation_title=Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=215-24; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Genome survey sequencing for the characterization of the genetic background of Rosa roxburghii Tratt and leaf ascorbate metabolism genes;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0147530" /><meta name="citation_reference" content="citation_title=The power of single molecule real-time sequencing technology in the de novo assembly of a eukaryotic genome;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=16780" /><meta name="citation_reference" content="citation_title=Improvement of barley genome annotations by deciphering the Haruna Nijo genome;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=21-8; " /><meta name="citation_reference" content="citation_title=Evolution of plant genome architecture;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=37" /><meta name="citation_reference" content="citation_title=Characterization of Salt Overly Sensitive 1 (SOS1) gene homoeologs in quinoa (Chenopodium quinoa Willd.);  citation_journal_title=Genome;  citation_year=2009;  citation_volume=52;  citation_pages=647-57; " /><meta name="citation_reference" content="citation_title=Analysis of the bread wheat genome using whole-genome shotgun sequencing;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=705-10; " /><meta name="citation_reference" content="Müller K. Borsch T.  2005 , Phylogenetics of Amaranthaceae based on mat K/ trn K sequence data: evidence from parsimony, likelihood, and Bayesian analyses. Ann. Missouri Bot. Gard. , 92 , 66 – 102. " /><meta name="citation_reference" content="citation_title=LHT1, a lysine- and histidine-specific amino acid transporter in arabidopsis;  citation_journal_title=Plant Physiol;  citation_year=1997;  citation_volume=115;  citation_pages=1127-34; " /><meta name="citation_reference" content="citation_title=Selective expression of a novel high-affinity transport system for acidic and neutral amino acids in the tapetum cells of Arabidopsis flowers;  citation_journal_title=Plant J;  citation_year=2004;  citation_volume=40;  citation_pages=60-74; " /><meta name="citation_reference" content="citation_title=Ferulate-5-hydroxylase from Arabidopsis thaliana defines a new family of cytochrome P450-dependent monooxygenases;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=1996;  citation_volume=93;  citation_pages=6869-74; " /><meta name="citation_reference" content="citation_title=Lineage-specific gene radiations underlie the evolution of novel betalain pigmentation in Caryophyllales;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=207;  citation_pages=1170-80; " /><meta name="citation_reference" content="citation_title=Quinoa – a model crop for understanding salt-tolerance mechanisms in halophytes;  citation_journal_title=Plant Biosyst;  citation_year=2016;  citation_volume=150;  citation_pages=357-71; " /><meta name="citation_reference" content="citation_title=Structure and function of abscisic acid receptors;  citation_journal_title=Trends Plant Sci;  citation_year=2013;  citation_volume=18;  citation_pages=259-66; " /><meta name="citation_reference" content="citation_title=A natural variant of a host RNA-dependent RNA polymerase is associated with increased susceptibility to viruses by Nicotiana benthamiana;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=2004;  citation_volume=101;  citation_pages=6297-302; " /><meta name="citation_reference" content="citation_title=The extremophile Nicotiana benthamiana has traded viral defence for early vigour;  citation_journal_title=Nat. Plants;  citation_year=2015;  citation_volume=1;  citation_pages=15165" /><meta name="citation_reference" content="citation_title=Evolutionary connection between the catalytic subunits of DNA-dependent RNA polymerases and eukaryotic RNA-dependent RNA polymerases and the origin of RNA polymerases;  citation_journal_title=BMC Struct. Biol;  citation_year=2003;  citation_volume=3;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Optical mapping in plant comparative genomics;  citation_journal_title=Gigascience;  citation_year=2015;  citation_volume=4;  citation_pages=3" /><meta name="citation_reference" content="Phytozome 11, https://phytozome.jgi.doe.gov/pz/portal.html " /><meta name="citation_reference" content="The Beta vulgaris Resource, http://bvseq.molgen.mpg.de/index.shtml. " /><meta name="citation_reference" content="Nicotiana benthamiana Genome &amp; Transcriptome, http://benthgenome.qut.edu.au. " /><meta name="citation_reference" content="EMBOSS Transeq , http://www.ebi.ac.uk/Tools/st/emboss_transeq/. " /><meta name="citation_reference" content="citation_title=CDD: NCBI's conserved domain database;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D222-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/6/535/8325689/dsw037.pdf" /><meta name="description" content="Chenopodium quinoa Willd. (quinoa) originated from the Andean region of South America, and is a pseudocereal crop of the Amaranthaceae family. Quinoa is emergin" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/6/535/2647444" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/6/535/2647444" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/6/535/2647444");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw037';
    </script>
<title>
	Draft genome sequence of an inbred line of Chenopodium quinoa, an allotetraploid crop with great environmental adaptability and outstanding nutritional properties | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2647444" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BY7PATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFzdWksIFlhc3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlLCBLeW90byBVbml2ZXJzaXR5LCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2lrYXdhLCBUZXRzdW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFJlc291cmNlcyBhbmQgUG9zdC1oYXJ2ZXN0IERpdmlzaW9uLCBKYXBhbiBJbnRlcm5hdGlvbmFsIFJlc2VhcmNoIENlbnRlciBmb3IgQWdyaWN1bHR1cmFsIFNjaWVuY2VzIChKSVJDQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04Njg2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRveW9zaGltYSwgTWFzYW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9naWNhbCBSZXNvdXJjZXMgYW5kIFBvc3QtaGFydmVzdCBEaXZpc2lvbiwgSmFwYW4gSW50ZXJuYXRpb25hbCBSZXNlYXJjaCBDZW50ZXIgZm9yIEFncmljdWx0dXJhbCBTY2llbmNlcyAoSklSQ0FTKSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODY4NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdXpha2ksIENoaWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUGxhbnQgR2VuZSBGdW5jdGlvbiwgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciBCaW9yZXNvdXJjZXMgYW5kIEJpb3RlY2hub2xvZ3ksIElzaGlrYXdhIFByZWZlY3R1cmFsIFVuaXZlcnNpdHksIE5vbm9pY2hpLCBJc2hpa2F3YSA5MjEtODgzNiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVZW5vLCBNYXJpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIEt5b3RvIFVuaXZlcnNpdHksIFNha3lvLWt1LCBLeW90byA2MDYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXp1bm8sIE5vYnV5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlLCBLeW90byBVbml2ZXJzaXR5LCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYXRvc2hpLCBZdWthcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFJlc291cmNlcyBhbmQgUG9zdC1oYXJ2ZXN0IERpdmlzaW9uLCBKYXBhbiBJbnRlcm5hdGlvbmFsIFJlc2VhcmNoIENlbnRlciBmb3IgQWdyaWN1bHR1cmFsIFNjaWVuY2VzIChKSVJDQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04Njg2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkltYW11cmEsIFRvbW9oaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQbGFudCBHZW5lIEZ1bmN0aW9uLCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIEJpb3Jlc291cmNlcyBhbmQgQmlvdGVjaG5vbG9neSwgSXNoaWthd2EgUHJlZmVjdHVyYWwgVW5pdmVyc2l0eSwgTm9ub2ljaGksIElzaGlrYXdhIDkyMS04ODM2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1peWFnbywgTWFuYW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGVjaG5vbG9neSBEZXZlbG9wbWVudCBHcm91cCwgQWN0cmVlIENvLiwgSGFrdXNhbiwgSXNoaWthd2EgOTI0LTAwNTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBLb2ppcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZWNobm9sb2d5IERldmVsb3BtZW50IEdyb3VwLCBBY3RyZWUgQ28uLCBIYWt1c2FuLCBJc2hpa2F3YSA5MjQtMDA1MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXNlLCBLYXp1eXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgS3lvdG8gVW5pdmVyc2l0eSwgU2FreW8ta3UsIEt5b3RvIDYwNi04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmFrYSwgVHN1dG9tdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRlY2hub2xvZ3kgRGV2ZWxvcG1lbnQgR3JvdXAsIEFjdHJlZSBDby4sIEhha3VzYW4sIElzaGlrYXdhIDkyNC0wMDUzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1penVrb3NoaSwgSGlyb2hhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZWNobm9sb2d5IERldmVsb3BtZW50IEdyb3VwLCBBY3RyZWUgQ28uLCBIYWt1c2FuLCBJc2hpa2F3YSA5MjQtMDA1MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpLCBNYXNhc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQbGFudCBHZW5lIEZ1bmN0aW9uLCBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIEJpb3Jlc291cmNlcyBhbmQgQmlvdGVjaG5vbG9neSwgSXNoaWthd2EgUHJlZmVjdHVyYWwgVW5pdmVyc2l0eSwgTm9ub2ljaGksIElzaGlrYXdhIDkyMS04ODM2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1aml0YSwgWWFzdW5hcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFJlc291cmNlcyBhbmQgUG9zdC1oYXJ2ZXN0IERpdmlzaW9uLCBKYXBhbiBJbnRlcm5hdGlvbmFsIFJlc2VhcmNoIENlbnRlciBmb3IgQWdyaWN1bHR1cmFsIFNjaWVuY2VzIChKSVJDQVMpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04Njg2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIGFuIGluYnJlZCBsaW5lIG9mIENoZW5vcG9kaXVtIHF1aW5vYSwgYW4gYWxsb3RldHJhcGxvaWQgY3JvcCB3aXRoIGdyZWF0IGVudmlyb25tZW50YWwgYWRhcHRhYmlsaXR5IGFuZCBvdXRzdGFuZGluZyBudXRyaXRpb25hbCBwcm9wZXJ0aWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1NDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSkgYmFzZWQgb24gQUZMUCwgUkFQRCwgYW5kIFNTUiBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTE4OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJlZWRpbmcgcXVpbm9hIChDaGVub3BvZGl1bSBxdWlub2EgV2lsbGQuKTogcG90ZW50aWFsIGFuZCBwZXJzcGVjdGl2ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBhdHRlcm4gb2YgZ2VuZXRpYyBkaXZlcnNpdHkgaW4gdGhlIEFuZGVhbiBncmFpbiBjcm9wIHF1aW5vYSAoQ2hlbm9wb2RpdW0gcXVpbm9hIFdpbGxkLikuIEkuIEFzc29jaWF0aW9ucyBiZXR3ZWVuIGNoYXJhY3RlcmlzdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTQ3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3JwaG9sb2dpY2FsIGFuYWx5c2lzIG9mIHF1aW5vYSBnZXJtcGxhc20gZ3Jvd24gdW5kZXIgbG93bGFuZCBkZXNlcnQgY29uZGl0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWdyb24uIENyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTEyNC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub3R5cGljIHZhcmlhdGlvbiBvZiBnYXMgZXhjaGFuZ2UgcGFyYW1ldGVycyBhbmQgbGVhZiBzdGFibGUgY2FyYm9uIGFuZCBuaXRyb2dlbiBpc290b3BlcyBpbiB0ZW4gcXVpbm9hIGN1bHRpdmFycyBncm93biB1bmRlciBkcm91Z2h0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBZ3Jvbi4gQ3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTk3OyAgY2l0YXRpb25fcGFnZXM9ODEtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlLWludHJvZHVjdGlvbiBvZiBxdcOtbm9hIGludG8gYXJpZCBDaGlsZTogY3VsdGl2YXRpb24gb2YgdHdvIGxvd2xhbmQgcmFjZXMgdW5kZXIgZXh0cmVtZWx5IGxvdyBpcnJpZ2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBZ3Jvbi4gQ3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk1OyAgY2l0YXRpb25fcGFnZXM9MS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhcG90cmFuc3BpcmF0aW9uIGFuYWx5c2lzIGFuZCBpcnJpZ2F0aW9uIHJlcXVpcmVtZW50cyBvZiBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSkgaW4gdGhlIEJvbGl2aWFuIGhpZ2hsYW5kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWdyaWMuIFdhdGVyIE1hbmFnZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTExOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzcG9uc2VzIG9mIHR3byBtYWluIEFuZGVhbiBjcm9wcywgcXVpbm9hIChDaGVub3BvZGl1bSBxdWlub2EgV2lsbGQpIGFuZCBwYXBhIGFtYXJnYSAoU29sYW51bSBqdXplcGN6dWtpaSBCdWsuKSB0byBkcm91Z2h0IG9uIHRoZSBCb2xpdmlhbiBBbHRpcGxhbm86IHNpZ25pZmljYW5jZSBvZiBsb2NhbCBhZGFwdGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZ3JpYy4gRWNvc3lzdC4gRW52aXJvbjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTk5LTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2F0ZXIgcmVsYXRpb25zIGFuZCB0cmFuc3BpcmF0aW9uIG9mIHF1aW5vYSAoQ2hlbm9wb2RpdW0gcXVpbm9hIFdpbGxkLikgdW5kZXIgc2FsaW5pdHkgYW5kIHNvaWwgZHJ5aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBZ3Jvbi4gQ3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTk3OyAgY2l0YXRpb25fcGFnZXM9MzQ4LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWlub2E6IGFuIGFsdGVybmF0aXZlIGNyb3AgZm9yIHNhbGluZSBzb2lsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz1pMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jb25pYyBhbmQgb3Ntb3RpYyByZWxhdGlvbnMgaW4gcXVpbm9hIChDaGVub3BvZGl1bSBxdWlub2EgV2lsbGQuKSBwbGFudHMgZ3Jvd24gYXQgdmFyaW91cyBzYWxpbml0eSBsZXZlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MTg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCByZXNwb25zZXMgb2YgcXVpbm9hIChDaGVub3BvZGl1bSBxdWlub2EgV2lsbGQuKSB0byBmcm9zdCBhdCB2YXJpb3VzIHBoZW5vbG9naWNhbCBzdGFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1ci4gSi4gQWdyb247ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnV0cml0aW9uIGZhY3RzIGFuZCBmdW5jdGlvbmFsIHBvdGVudGlhbCBvZiBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSB3aWxsZC4pLCBhbiBhbmNpZW50IEFuZGVhbiBncmFpbjogYSByZXZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFNjaS4gRm9vZCBBZ3JpYzsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTI1NDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVpbm9hOiBudXRyaXRpb25hbCwgZnVuY3Rpb25hbCBhbmQgYW50aW51dHJpdGlvbmFsIGFzcGVjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyaXQuIFJldi4gRm9vZCBTY2kuIE51dHI7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgdGhlIG51dHJpdGlvbmFsIGNvbXBvc2l0aW9uIG9mIHF1aW5vYSAoQ2hlbm9wb2RpdW0gcXVpbm9hIFdpbGxkLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvb2QgQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz00Ny01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBjb250ZW50IGFuZCBhbWlubyBhY2lkcyBwcm9maWxlIG9mIHBzZXVkb2NlcmVhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvb2QgQ2hlbTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTM7ICBjaXRhdGlvbl9wYWdlcz01NS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVJvbWU7ICBjaXRhdGlvbl90aXRsZT1TdGF0ZSBvZiB0aGUgYXJ0IHJlcG9ydCBvbiBxdWlub2EgYXJvdW5kIHRoZSB3b3JsZCBpbiAyMDEzLiBGb29kIGFuZCBBZ3JpY3VsdHVyZSBPcmdhbml6YXRpb24gb2YgdGhlIFVuaXRlZCBOYXRpb25zIChGQU8pICZhbXA7IENlbnRyZSBkZSBDb29ww6lyYXRpb24gSW50ZXJuYXRpb25hbGUgZW4gUmVjaGVyY2hlIEFncm9ub21pcXVlIHBvdXIgbGUgRMOpdmVsb3BwZW1lbnQgKENJUkFEKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVpbm9hOiBhbiBlbWVyZ2luZyDigJxuZXfigJ0gY3JvcCB3aXRoIHBvdGVudGlhbCBmb3IgY2Vsc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5BU0EgVGVjaC4gUGFwOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM0MjI7ICBjaXRhdGlvbl9wYWdlcz0xLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5ICZhbXA7IFNvbnMsIEluYywgTmV3IEplcnNleTsgIGNpdGF0aW9uX3RpdGxlPVF1aW5vYTogaW1wcm92ZW1lbnQgYW5kIHN1c3RhaW5hYmxlIHByb2R1Y3Rpb247ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl9wYWdlcz0xLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIGdlbmV0aWMgZGl2ZXJzaXR5IGluIHRoZSBVU0RBIGFuZCBDSVAtRkFPIGludGVybmF0aW9uYWwgbnVyc2VyeSBjb2xsZWN0aW9ucyBvZiBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSBXaWxsZC4pIHVzaW5nIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEdlbmV0LiBSZXNvdXI7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTgyLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZWFyIGdlbm9tZSBzaXplIGFuZCBjaHJvbW9zb21lIGFuYWx5c2lzIGluIENoZW5vcG9kaXVtIHF1aW5vYSBhbmQgQy4gYmVybGFuZGllcmkgc3Vic3AuIG51dHRhbGxpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9nZW5ldGljIGFuYWx5c2lzIG9mIHF1aW5vYSBjaHJvbW9zb21lcyB1c2luZyBuYW5vc2NhbGUgaW1hZ2luZyBhbmQgc3BlY3Ryb3Njb3B5IHRlY2huaXF1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hbm9zY2FsZSBSZXMgTGV0dDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hlbm9wb2RpdW0gcG9seXBsb2lkeSBpbmZlcmVuY2VzIGZyb20gU2FsdCBPdmVybHkgU2Vuc2l0aXZlIDEgKFNPUzEpIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9NTMzLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gaWRlbnRpZmljYXRpb24sIGNoYXJhY3Rlcml6YXRpb24sIGFuZCBsaW5rYWdlIG1hcHBpbmcgaW4gcXVpbm9hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTExNC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2J1c3QsIHNpbXBsZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgKEdCUykgYXBwcm9hY2ggZm9yIGhpZ2ggZGl2ZXJzaXR5IHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTkzNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFsaXR5IG9mIHByb2thcnlvdGUgZ2Vub21lIGFzc2VtYmx5OiBpbmRpc3BlbnNhYmxlIGlzc3VlcyBvZiBmYWN0b3JzIGFmZmVjdGluZyBwcm9rYXJ5b3RlIGdlbm9tZSBhc3NlbWJseSBxdWFsaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUwNTsgIGNpdGF0aW9uX3BhZ2VzPTM2NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5kIHRoZSBnYXA6IHVwZ3JhZGluZyBnZW5vbWVzIHdpdGggUGFjaWZpYyBCaW9zY2llbmNlcyBSUyBsb25nLXJlYWQgc2VxdWVuY2luZyB0ZWNobm9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ3NzY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBvZiBQcmltdWxhIHZlcmlzIHlpZWxkcyBpbnNpZ2h0cyBpbnRvIHRoZSBtb2xlY3VsYXIgYmFzaXMgb2YgaGV0ZXJvc3R5bHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWxldGlvbiBvZiB0aGUgQy10ZXJtaW5hbCAzMyBhbWlubyBhY2lkcyBvZiBjdWN1bWJlciBtb3NhaWMgdmlydXMgbW92ZW1lbnQgcHJvdGVpbiBlbmFibGVzIGEgY2hpbWVyaWMgYnJvbWUgbW9zYWljIHZpcnVzIHRvIG1vdmUgZnJvbSBjZWxsIHRvIGNlbGw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFZpcm9sOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9MjI3MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9SVJMIFByZXNzIGF0IE94Zm9yZCBVbml2ZXJzaXR5IFByZXNzLCBPeGZvcmQ7ICBjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgcGxhbnQgcGF0aG9sb2d5OiBhIHByYWN0aWNhbCBhcHByb2FjaDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT12b2wuIEk7ICBjaXRhdGlvbl9wYWdlcz0yMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJvbW92aXJ1cyBtb3ZlbWVudCBwcm90ZWluIGdlbmVzIHBsYXkgYSBjcnVjaWFsIHJvbGUgaW4gaG9zdCBzcGVjaWZpY2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVmlyb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0yODE1LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obiBXaWxleSAmYW1wOyBTb25zIEluYywgTmV3IEplcnNleTsgIGNpdGF0aW9uX3RpdGxlPUN1cnJlbnQgUHJvdG9jb2xzIGluIE1vbGVjdWxhciBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFsaXRhdGl2ZSBkZSBub3ZvIGFuYWx5c2lzIG9mIGZ1bGwgbGVuZ3RoIGNETkEgYW5kIHF1YW50aXRhdGl2ZSBhbmFseXNpcyBvZiBnZW5lIGV4cHJlc3Npb24gZm9yIGNvbW1vbiBtYXJtb3NldCAoQ2FsbGl0aHJpeCBqYWNjaHVzKSB0cmFuc2NyaXB0b21lcyB1c2luZyBwYXJhbGxlbCBsb25nLXJlYWQgdGVjaG5vbG9neSBhbmQgc2hvcnQtcmVhZCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDkzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyaW1tb21hdGljOiBhIGZsZXhpYmxlIHRyaW1tZXIgZm9yIElsbHVtaW5hIHNlcXVlbmNlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjExNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGlvbiBvZiB0aGUgb2N0b3Bsb2lkIHN0cmF3YmVycnkgZ2Vub21lIGJ5IGRlZXAgc2VxdWVuY2luZyBvZiB0aGUgZ2Vub21lcyBvZiBGcmFnYXJpYSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTY5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1dGFkYXB0IHJlbW92ZXMgYWRhcHRlciBzZXF1ZW5jZXMgZnJvbSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CbmV0LiBKOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1xdWFsaXR5IGRyYWZ0IGFzc2VtYmxpZXMgb2YgbWFtbWFsaWFuIGdlbm9tZXMgZnJvbSBtYXNzaXZlbHkgcGFyYWxsZWwgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTUxMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgcmVwZWF0IGZhbWlsaWVzIGluIGxhcmdlIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxIFN1cHBsIDE7ICBjaXRhdGlvbl9wYWdlcz1pMzUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlbHZldDogYWxnb3JpdGhtcyBmb3IgZGUgbm92byBzaG9ydCByZWFkIGFzc2VtYmx5IHVzaW5nIGRlIEJydWlqbiBncmFwaHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2FzZXM6IHJvYnVzdCBkZSBub3ZvIFJOQS1zZXEgYXNzZW1ibHkgYWNyb3NzIHRoZSBkeW5hbWljIHJhbmdlIG9mIGV4cHJlc3Npb24gbGV2ZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNELUhJVDogYWNjZWxlcmF0ZWQgZm9yIGNsdXN0ZXJpbmcgdGhlIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzE1MC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HTUFQOiBhIGdlbm9taWMgbWFwcGluZyBhbmQgYWxpZ25tZW50IHByb2dyYW0gZm9yIG1STkEgYW5kIEVTVCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTg1OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CUkFLRVIxOiB1bnN1cGVydmlzZWQgUk5BLXNlcS1iYXNlZCBnZW5vbWUgYW5ub3RhdGlvbiB3aXRoIEdlbmVNYXJrLUVUIGFuZCBBVUdVU1RVUzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz03NjctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIHdpdGggYSBoaWRkZW4gTWFya292IG1vZGVsIGFuZCBhIG5ldyBpbnRyb24gc3VibW9kZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5IFN1cHBsIDI7ICBjaXRhdGlvbl9wYWdlcz1paTIxNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UlNFTTogYWNjdXJhdGUgdHJhbnNjcmlwdCBxdWFudGlmaWNhdGlvbiBmcm9tIFJOQS1TZXEgZGF0YSB3aXRoIG9yIHdpdGhvdXQgYSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVcxMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDT0cgZGF0YWJhc2U6IGFuIHVwZGF0ZWQgdmVyc2lvbiBpbmNsdWRlcyBldWthcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEd5cHN5IERhdGFiYXNlIChHeURCKSBvZiBtb2JpbGUgZ2VuZXRpYyBlbGVtZW50czogcmVsZWFzZSAyLjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9RDcwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGdlbmVyYXRpb24gb2YgaG9tb2xvZ3kgc2VhcmNoIHRvb2xzIGJhc2VkIG9uIHByb2JhYmlsaXN0aWMgaW5mZXJlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgSW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQUwyTkFMOiByb2J1c3QgY29udmVyc2lvbiBvZiBwcm90ZWluIHNlcXVlbmNlIGFsaWdubWVudHMgaW50byB0aGUgY29ycmVzcG9uZGluZyBjb2RvbiBhbGlnbm1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPVc2MDktMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTUwgNDogcGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2x1c3RhbCBXIGFuZCBDbHVzdGFsIFggdmVyc2lvbiAyLjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9Mjk0Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJhcGlkIGdlbmVyYXRpb24gb2YgbXV0YXRpb24gZGF0YSBtYXRyaWNlcyBmcm9tIHByb3RlaW4gc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIEFwcGwuIEJpb3NjaTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWR1Y2VkIHRvbm9wbGFzdCBmYXN0LWFjdGl2YXRpbmcgYW5kIHNsb3ctYWN0aXZhdGluZyBjaGFubmVsIGFjdGl2aXR5IGlzIGVzc2VudGlhbCBmb3IgY29uZmVycmluZyBzYWxpbml0eSB0b2xlcmFuY2UgaW4gYSBmYWN1bHRhdGl2ZSBoYWxvcGh5dGUsIHF1aW5vYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjI7ICBjaXRhdGlvbl9wYWdlcz05NDAtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbnNpdGl2aXR5IG9mIHR3byBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSBXaWxsZC4pIHZhcmlldGllcyB0byBwcm9ncmVzc2l2ZSBkcm91Z2h0IHN0cmVzczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWdyb24uIENyb3AgU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwMDsgIGNpdGF0aW9uX3BhZ2VzPTEyLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWx0IHRvbGVyYW5jZSBtZWNoYW5pc21zIGluIHF1aW5vYSAoQ2hlbm9wb2RpdW0gcXVpbm9hIFdpbGxkLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9NDMtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5ICZhbXA7IFNvbnMsIEluYywgTmV3IEplcnNleTsgIGNpdGF0aW9uX3RpdGxlPUN1cnJlbnQgUHJvdG9jb2xzIGluIE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB2aXJ1cyBtYWRlIGZyb20gcGFydHMgb2YgdGhlIGdlbm9tZXMgb2YgYnJvbWUgbW9zYWljIGFuZCBjb3dwZWEgY2hsb3JvdGljIG1vdHRsZSB2aXJ1cy4gSi4gR2VuLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Vmlyb2w7ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yMjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHN0dWRpZXMgb24gYnJvbW92aXJ1cyBjYXBzaWQgcHJvdGVpbi4gSVYuIENvYXQgcHJvdGVpbiBleGNoYW5nZXMgYmV0d2VlbiBicm9tZSBtb3NhaWMgYW5kIGNvd3BlYSBjaGxvcm90aWMgbW90dGxlIHZpcnVzZXMgZXhoaWJpdCBuZXV0cmFsIGVmZmVjdHMgaW4gaGV0ZXJvbG9nb3VzIGhvc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1WaXJvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzg7ICBjaXRhdGlvbl9wYWdlcz00NTItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibHkgb2YgdGhlIGRyYWZ0IGdlbm9tZSBvZiBidWNrd2hlYXQgYW5kIGl0cyBhcHBsaWNhdGlvbnMgaW4gaWRlbnRpZnlpbmcgYWdyb25vbWljYWxseSB1c2VmdWwgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMTUtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZSBub3ZvIGFzc2VtYmx5IG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lcyBmcm9tIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTM4NC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHN1cnZleSBzZXF1ZW5jaW5nIGZvciB0aGUgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgZ2VuZXRpYyBiYWNrZ3JvdW5kIG9mIFJvc2Egcm94YnVyZ2hpaSBUcmF0dCBhbmQgbGVhZiBhc2NvcmJhdGUgbWV0YWJvbGlzbSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE0NzUzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwb3dlciBvZiBzaW5nbGUgbW9sZWN1bGUgcmVhbC10aW1lIHNlcXVlbmNpbmcgdGVjaG5vbG9neSBpbiB0aGUgZGUgbm92byBhc3NlbWJseSBvZiBhIGV1a2FyeW90aWMgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTY3ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlbWVudCBvZiBiYXJsZXkgZ2Vub21lIGFubm90YXRpb25zIGJ5IGRlY2lwaGVyaW5nIHRoZSBIYXJ1bmEgTmlqbyBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgcGxhbnQgZ2Vub21lIGFyY2hpdGVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgU2FsdCBPdmVybHkgU2Vuc2l0aXZlIDEgKFNPUzEpIGdlbmUgaG9tb2VvbG9ncyBpbiBxdWlub2EgKENoZW5vcG9kaXVtIHF1aW5vYSBXaWxsZC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz02NDctNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBicmVhZCB3aGVhdCBnZW5vbWUgdXNpbmcgd2hvbGUtZ2Vub21lIHNob3RndW4gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTcwNS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTcO8bGxlciBLLiBCb3JzY2ggVC4gIDIwMDUgLCBQaHlsb2dlbmV0aWNzIG9mIEFtYXJhbnRoYWNlYWUgYmFzZWQgb24gbWF0IEsvIHRybiBLIHNlcXVlbmNlIGRhdGE6IGV2aWRlbmNlIGZyb20gcGFyc2ltb255LCBsaWtlbGlob29kLCBhbmQgQmF5ZXNpYW4gYW5hbHlzZXMuIEFubi4gTWlzc291cmkgQm90LiBHYXJkLiAsIDkyICwgNjYg4oCTIDEwMi4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TEhUMSwgYSBseXNpbmUtIGFuZCBoaXN0aWRpbmUtc3BlY2lmaWMgYW1pbm8gYWNpZCB0cmFuc3BvcnRlciBpbiBhcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz0xMTI3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3RpdmUgZXhwcmVzc2lvbiBvZiBhIG5vdmVsIGhpZ2gtYWZmaW5pdHkgdHJhbnNwb3J0IHN5c3RlbSBmb3IgYWNpZGljIGFuZCBuZXV0cmFsIGFtaW5vIGFjaWRzIGluIHRoZSB0YXBldHVtIGNlbGxzIG9mIEFyYWJpZG9wc2lzIGZsb3dlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02MC03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmVydWxhdGUtNS1oeWRyb3h5bGFzZSBmcm9tIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGRlZmluZXMgYSBuZXcgZmFtaWx5IG9mIGN5dG9jaHJvbWUgUDQ1MC1kZXBlbmRlbnQgbW9ub294eWdlbmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9Njg2OS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGluZWFnZS1zcGVjaWZpYyBnZW5lIHJhZGlhdGlvbnMgdW5kZXJsaWUgdGhlIGV2b2x1dGlvbiBvZiBub3ZlbCBiZXRhbGFpbiBwaWdtZW50YXRpb24gaW4gQ2FyeW9waHlsbGFsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjA3OyAgY2l0YXRpb25fcGFnZXM9MTE3MC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVpbm9hIOKAkyBhIG1vZGVsIGNyb3AgZm9yIHVuZGVyc3RhbmRpbmcgc2FsdC10b2xlcmFuY2UgbWVjaGFuaXNtcyBpbiBoYWxvcGh5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW9zeXN0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuZCBmdW5jdGlvbiBvZiBhYnNjaXNpYyBhY2lkIHJlY2VwdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTI1OS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuYXR1cmFsIHZhcmlhbnQgb2YgYSBob3N0IFJOQS1kZXBlbmRlbnQgUk5BIHBvbHltZXJhc2UgaXMgYXNzb2NpYXRlZCB3aXRoIGluY3JlYXNlZCBzdXNjZXB0aWJpbGl0eSB0byB2aXJ1c2VzIGJ5IE5pY290aWFuYSBiZW50aGFtaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9NjI5Ny0zMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBleHRyZW1vcGhpbGUgTmljb3RpYW5hIGJlbnRoYW1pYW5hIGhhcyB0cmFkZWQgdmlyYWwgZGVmZW5jZSBmb3IgZWFybHkgdmlnb3VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTUxNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY29ubmVjdGlvbiBiZXR3ZWVuIHRoZSBjYXRhbHl0aWMgc3VidW5pdHMgb2YgRE5BLWRlcGVuZGVudCBSTkEgcG9seW1lcmFzZXMgYW5kIGV1a2FyeW90aWMgUk5BLWRlcGVuZGVudCBSTkEgcG9seW1lcmFzZXMgYW5kIHRoZSBvcmlnaW4gb2YgUk5BIHBvbHltZXJhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgU3RydWN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aWNhbCBtYXBwaW5nIGluIHBsYW50IGNvbXBhcmF0aXZlIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HaWdhc2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlBoeXRvem9tZSAxMSwgaHR0cHM6Ly9waHl0b3pvbWUuamdpLmRvZS5nb3YvcHovcG9ydGFsLmh0bWwgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVGhlIEJldGEgdnVsZ2FyaXMgUmVzb3VyY2UsIGh0dHA6Ly9idnNlcS5tb2xnZW4ubXBnLmRlL2luZGV4LnNodG1sLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOaWNvdGlhbmEgYmVudGhhbWlhbmEgR2Vub21lICZhbXA7IFRyYW5zY3JpcHRvbWUsIGh0dHA6Ly9iZW50aGdlbm9tZS5xdXQuZWR1LmF1LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJFTUJPU1MgVHJhbnNlcSAsIGh0dHA6Ly93d3cuZWJpLmFjLnVrL1Rvb2xzL3N0L2VtYm9zc190cmFuc2VxLy4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0REOiBOQ0JJJ3MgY29uc2VydmVkIGRvbWFpbiBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1EMjIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy82LzUzNS84MzI1Njg5L2RzdzAzNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkNoZW5vcG9kaXVtIHF1aW5vYSBXaWxsZC4gKHF1aW5vYSkgb3JpZ2luYXRlZCBmcm9tIHRoZSBBbmRlYW4gcmVnaW9uIG9mIFNvdXRoIEFtZXJpY2EsIGFuZCBpcyBhIHBzZXVkb2NlcmVhbCBjcm9wIG9mIHRoZSBBbWFyYW50aGFjZWFlIGZhbWlseS4gUXVpbm9hIGlzIGVtZXJnaW4iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI2NDc0NDQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkh+GmnRsxgk+7kx1rXzCSSsYVd/8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f535%2f2647444" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f535%2f2647444" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=89339" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&amp;Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/6">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="48005208">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005208">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005214">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005214">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005244">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005244">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005272">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005272">4. Conclusion and future perspectives</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005275">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005275">5. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005277">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005277">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48005285">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48005285">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/6/527/2647443">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/6/547/2647446">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Draft genome sequence of an inbred line of <em>Chenopodium quinoa</em>, an allotetraploid crop with great environmental adaptability and outstanding nutritional properties
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuo Yasui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw037-cor1" data-open="dsw037-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 75-753-6480. Email: <a href="mailto:yasyas@kais.kyoto-u.ac.jp" target="_blank">yasyas@kais.kyoto-u.ac.jp</a> (Y.Y.); Tel. +81 76-227-7527. Fax. +81 76-227-7557. Email: <a href="mailto:mori@ishikawa-pu.ac.jp" target="_blank">mori@ishikawa-pu.ac.jp</a> (M.M.); and Tel. +81 29-838-6642. Fax. +81 29-838-6643. Email: <a href="mailto:yasuf@affrc.go.jp" target="_blank">yasuf@affrc.go.jp</a> (Y.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuo+Yasui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yasui Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yasui Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasuo Yasui</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuo Oikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuo+Oikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oikawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oikawa T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tetsuo Oikawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masami Toyoshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masami+Toyoshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoshima M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoshima M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masami Toyoshima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiaki Matsuzaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiaki+Matsuzaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsuzaki C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsuzaki C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chiaki Matsuzaki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariko Ueno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariko+Ueno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ueno M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ueno M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mariko Ueno</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuyuki Mizuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuyuki+Mizuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizuno N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mizuno N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nobuyuki Mizuno</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yukari Nagatoshi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yukari+Nagatoshi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagatoshi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagatoshi Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yukari Nagatoshi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomohiro Imamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomohiro+Imamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Imamura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Imamura T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomohiro Imamura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manami Miyago</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manami+Miyago">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyago M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyago M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Manami Miyago</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kojiro Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kojiro+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kojiro Tanaka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuyuki Mise</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Agriculture, Kyoto University, Sakyo-ku, Kyoto 606-8502, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuyuki+Mise">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mise K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mise K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kazuyuki Mise</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsutomu Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsutomu+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsutomu Tanaka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroharu Mizukoshi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Technology Development Group, Actree Co., Hakusan, Ishikawa 924-0053, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroharu+Mizukoshi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizukoshi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mizukoshi H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiroharu Mizukoshi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masashi Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Laboratory of Plant Gene Function, Research Institute for Bioresources and Biotechnology, Ishikawa Prefectural University, Nonoichi, Ishikawa 921-8836, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw037-cor1" data-open="dsw037-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 75-753-6480. Email: <a href="mailto:yasyas@kais.kyoto-u.ac.jp" target="_blank">yasyas@kais.kyoto-u.ac.jp</a> (Y.Y.); Tel. +81 76-227-7527. Fax. +81 76-227-7557. Email: <a href="mailto:mori@ishikawa-pu.ac.jp" target="_blank">mori@ishikawa-pu.ac.jp</a> (M.M.); and Tel. +81 29-838-6642. Fax. +81 29-838-6643. Email: <a href="mailto:yasuf@affrc.go.jp" target="_blank">yasuf@affrc.go.jp</a> (Y.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masashi+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masashi Mori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasunari Fujita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Biological Resources and Post-harvest Division, Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Ibaraki 305-8686, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw037-cor1" data-open="dsw037-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 75-753-6480. Email: <a href="mailto:yasyas@kais.kyoto-u.ac.jp" target="_blank">yasyas@kais.kyoto-u.ac.jp</a> (Y.Y.); Tel. +81 76-227-7527. Fax. +81 76-227-7557. Email: <a href="mailto:mori@ishikawa-pu.ac.jp" target="_blank">mori@ishikawa-pu.ac.jp</a> (M.M.); and Tel. +81 29-838-6642. Fax. +81 29-838-6643. Email: <a href="mailto:yasuf@affrc.go.jp" target="_blank">yasuf@affrc.go.jp</a> (Y.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasunari+Fujita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujita Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujita Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasunari Fujita</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw037-FM2"><div class="fn"><div class="footnote-content"><sup>†</sup>Co-second authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw037-FM1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 535–546, <a href='https://doi.org/10.1093/dnares/dsw037'>https://doi.org/10.1093/dnares/dsw037</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2647444" href="/dnaresearch/article-pdf/23/6/535/8325689/dsw037.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yasuo Yasui, Hideki Hirakawa, Tetsuo Oikawa, Masami Toyoshima, Chiaki Matsuzaki, Mariko Ueno, Nobuyuki Mizuno, Yukari Nagatoshi, Tomohiro Imamura, Manami Miyago, Kojiro Tanaka, Kazuyuki Mise, Tsutomu Tanaka, Hiroharu Mizukoshi, Masashi Mori, Yasunari Fujita;  Draft genome sequence of an inbred line of <em>Chenopodium quinoa</em>, an allotetraploid crop with great environmental adaptability and outstanding nutritional properties, <em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 535–546, <a href='https://doi.org/10.1093/dnares/dsw037'>https://doi.org/10.1093/dnares/dsw037</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647444?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Draft%20genome%20sequence%20of%20an%20inbred%20line%20of%20%26lt%3Bem%26gt%3BChenopodium%20quinoa%26lt%3B%2Fem%26gt%3B%2C%20an%20allotetraploid%20crop%20with%20great%20environmental%20adaptability%20and%20outstanding%20nutritional%20properties&amp;publicationDate=2016-07-25&amp;volumeNum=23&amp;issueNum=6&amp;author=Yasui%2C%20Yasuo%3B%20Hirakawa%2C%20Hideki&amp;startPage=535&amp;endPage=546&amp;contentId=10.1093%2Fdnares%2Fdsw037&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f535%2f2647444" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p><em>Chenopodium quinoa</em> Willd. (quinoa) originated from the Andean region of South America, and is a pseudocereal crop of the Amaranthaceae family. Quinoa is emerging as an important crop with the potential to contribute to food security worldwide and is considered to be an optimal food source for astronauts, due to its outstanding nutritional profile and ability to tolerate stressful environments. Furthermore, plant pathologists use quinoa as a representative diagnostic host to identify virus species. However, molecular analysis of quinoa is limited by its genetic heterogeneity due to outcrossing and its genome complexity derived from allotetraploidy. To overcome these obstacles, we established the inbred and standard quinoa accession Kd that enables rigorous molecular analysis, and presented the draft genome sequence of Kd, using an optimized combination of high-throughput next generation sequencing on the Illumina Hiseq 2500 and PacBio RS II sequencers. The <em>de novo</em> genome assembly contained 25 k scaffolds consisting of 1 Gbp with N50 length of 86 kbp. Based on these data, we constructed the free-access Quinoa Genome DataBase (QGDB). Thus, these findings provide insights into the mechanisms underlying agronomically important traits of quinoa and the effect of allotetraploidy on genome evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Chenopodium quinoa&quot;"><em>Chenopodium quinoa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;draft genome&quot;">draft genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NGS">NGS</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;inbred accession&quot;">inbred accession</a></div>                            <h2 scrollto-destination=48005208 id="48005208" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Quinoa (<em>Chenopodium quinoa</em> Willd.) is an annual herbaceous plant that originated from the Andean region of South America, and is a pseudocereal crop of the Amaranthaceae family, which also includes sugar beet (<em>Beta vulgaris</em> L. ssp. <em>vulgaris</em>) and spinach (<em>Spinacia oleracea</em> L.).<a reveal-id="dsw037-B1" data-open="dsw037-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The major area of quinoa cultivation ranges from Columbia to central Chile,<a reveal-id="dsw037-B3" data-open="dsw037-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw037-B4" data-open="dsw037-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and includes altitudes from sea level up to 4,000 m above sea level<a reveal-id="dsw037-B5" data-open="dsw037-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and annual rainfalls of 80 mm to 2,000 mm.<a reveal-id="dsw037-B1" data-open="dsw037-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw037-B6" data-open="dsw037-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Quinoa is therefore well adapted to grow under adverse climatic and soil conditions<a reveal-id="dsw037-B7" data-open="dsw037-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and displays high tolerance to drought,<a reveal-id="dsw037-B8" data-open="dsw037-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw037-B9" data-open="dsw037-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> soil salinity,<a reveal-id="dsw037-B10" data-open="dsw037-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw037-B11" data-open="dsw037-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and frost.<a reveal-id="dsw037-B12" data-open="dsw037-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Furthermore, quinoa is an exceptional nutritional source of a wide spectrum of minerals (e.g. Ca, Fe, Mg, P, and Zn), vitamins (e.g. A, B1, B2, B9, C, and E), dietary fiber, linolenate, natural antioxidants (e.g. polyphenols), and high-quality protein, containing high levels of essential amino acids, particularly methionine and lysine.<a reveal-id="dsw037-B13 dsw037-B14 dsw037-B15 dsw037-B16" data-open="dsw037-B13 dsw037-B14 dsw037-B15 dsw037-B16" class="link link-ref link-reveal xref-bibr"><sup>13–16</sup></a> Being gluten-free, quinoa is suitable for consumption by individuals who are allergic or intolerant to wheat.<a reveal-id="dsw037-B14" data-open="dsw037-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Owing to the outstanding nutritional value of quinoa seeds and the great adaptability of quinoa plants to adverse environments, quinoa is considered by the Food and Agriculture Organization of the United Nations (FAO) to be an important crop with the potential to contribute to food security worldwide.<a reveal-id="dsw037-B17" data-open="dsw037-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Moreover, the National Aeronautics and Space Administration, USA (NASA) deems quinoa as an optimal food source for astronauts on long-term space missions in isolated conditions.<a reveal-id="dsw037-B18" data-open="dsw037-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p><p>Quinoa has been cultivated in the Andes for several thousand years.<a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Although quinoa cultivation was forbidden during the Spanish Conquest of South America in the sixteenth century, quinoa is cultivated in over 50 countries today.<a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw037-B19" data-open="dsw037-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Indeed, several thousand quinoa accessions are stored in germplasm banks.<a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Although quinoa is considered to be a predominantly autogamous (i.e. self-pollinated) species, multiple reports indicate that quinoa accessions are genetically heterogeneous due to outcrossing based on carrying two kinds of flowers on the same plant.<a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw037-B20" data-open="dsw037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Nevertheless, no inbred quinoa accessions have been reported to date, and this is problematic because molecular genetics and biology studies of quinoa rely on the development of an inbred quinoa line.</p><p>Quinoa is an allotetraploid species (2<em>n </em>= 4<em>x </em>= 36 with a genome size of 1,448 Mbp)<a reveal-id="dsw037-B21" data-open="dsw037-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw037-B22" data-open="dsw037-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> that consists of two distinct genomes, A and B.<a reveal-id="dsw037-B23" data-open="dsw037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Genetic mapping of quinoa has been conducted using amplified fragment length polymorphism (AFLP) markers,<a reveal-id="dsw037-B1" data-open="dsw037-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> simple sequence repeat (SSR) markers,<a reveal-id="dsw037-B1" data-open="dsw037-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and array-platform markers.<a reveal-id="dsw037-B24" data-open="dsw037-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> However, the most recent quinoa map contains just 511 single nucleotide polymorphism (SNP) sites and does not span the entire quinoa genome.<a reveal-id="dsw037-B24" data-open="dsw037-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> To obtain a sufficient number of molecular markers to cover the entire genome, the draft genome sequence must be established. This would serve as a reference to identify not only SNPs/SSRs, but also next-generation sequencing (NGS)-based markers, such as genotyping-by-sequencing (GBS) markers.<a reveal-id="dsw037-B25" data-open="dsw037-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p><p>Illumina and Pacific Biosciences (PacBio) have developed powerful NGS techniques to sequence the genomes of all living organisms. The Illumina HiSeq 2500 sequencer generates a high number of short reads (&lt;250 bp) with a high quality of base calls. In contrast, the PacBio RS II sequencer produces long reads (average read length, 7 kbp), though its throughput and quality of base calls are lower.<a reveal-id="dsw037-B26" data-open="dsw037-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Recently, long reads have been shown to fill gaps within and between scaffolds assembled by short reads.<a reveal-id="dsw037-B27" data-open="dsw037-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Prompted by the finding that different types of sequencers have been successfully combined for the <em>de novo</em> assembly of genome scaffolds of a heterozygous plant, <em>Primula veris</em>,<a reveal-id="dsw037-B28" data-open="dsw037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> we used two distinct types of sequencers to produce the draft genome sequence of the tetraploid species quinoa.</p><p>In this study, we established an inbred and standard quinoa accession, Kd, suitable for molecular analyses, and provided the draft genome sequence of the quinoa accession using the Illumina HiSeq 2500 and PacBio RS II sequencers. Based on these data, we constructed the free-access Quinoa Genome DataBase (QGDB; <a class="link link-uri" href="http://quinoa.kazusa.or.jp" target="_blank">http://quinoa.kazusa.or.jp</a>), which provides annotations of <em>in silico</em> predicted genes. Furthermore, we utilized comparative genomics and experimental approaches to identify genes in quinoa that are involved in abiotic and biotic stress responses.</p>                            <h2 scrollto-destination=48005214 id="48005214" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=48005215 id="48005215" class="section-title" >2.1. Plant materials and growth conditions</h3>
<p>Quinoa (<em>Chenopodium quinoa</em> Willd.) seeds had been propagated in a temperature-controlled plant growth room at the Laboratory of Plant Pathology, Graduate School of Agriculture, Kyoto University in the absence of the other quinoa accessions for over 20 years.<a reveal-id="dsw037-B29" data-open="dsw037-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Then at the Japan International Research Center for Agricultural Sciences (JIRCAS), to establish an inbred quinoa accession, quinoa seeds have been propagated from a single plant derived from the seeds propagated in Kyoto University. To prevent cross-pollination, all of the inflorescences of these plants grown in JIRCAS were covered with non-woven pollination bags (Rizo, Tsukuba, Japan). The quinoa seeds were sown in a peat moss mix (Jiffy Mix, Sakata Seeds, Yokohama, Japan) in a cell tray and were grown in a growth chamber at 27 °C with a short-day photoperiod (11 h light/13 h darkness). After 14 days, the seedlings were transferred to a standard potting mix (Tsuchitaro, Sumitomo Forestry, Tokyo, Japan) in 20-L plant pots and were grown under ambient light in a temperature-controlled phytotron in JIRCAS with a temperature of 25 ± 5 °C and relative humidity of 55 ± 25%. For NGS analyses, a single plant was selected, and the progeny seeds were harvested.</p>                            <h3 scrollto-destination=48005217 id="48005217" class="section-title" >2.2. Evaluation of salt-tolerance in quinoa and <em>Arabidopsis</em></h3>
<p>Quinoa seeds (Kd) were sown in the peat moss mix in a cell tray and were grown in a growth chamber at 27 °C with a short-day photoperiod (11 h light/13 h darkness). At 14 days after sowing, the seedlings were transferred to a standard potting mixture (Professional-baido, Daio Chemical, Tokyo, Japan) in 0.16-L plant pots and were further grown under the same conditions. <em>Arabidopsis thaliana</em> (CS60000) seeds were sown on MS agar plates and then were stored at 4 °C in the darkness for 3 days. Then, the seedlings were grown in a growth cabinet (Biotron; NK systems, Japan) at 22 °C with a long-day photoperiod (16 h light/8 h darkness). At 14 days after sowing, the seedlings were transferred to the standard potting mix in 0.16-L plant pots and were grown in a growth chamber at 27 °C with a short-day photoperiod (11 h light/13 h darkness). At 21 days after sowing, the quinoa and the <em>Arabidopsis</em> seedlings were treated with 0 mM or 300 mM NaCl. The survival rates were measured at 22 days after the salt treatments. To maintain the concentration (300 mM NaCl corresponds to 2.90 S/m) and the default volume of the salt solution, the electrical conductivities of the salt solutions were measured using an electrical conductivity meter (Laqua DS-71: HORIBA, Kyoto, Japan) every 2 days.</p>                            <h3 scrollto-destination=48005219 id="48005219" class="section-title" >2.3. Inoculation of quinoa plants</h3>
<p>For virus inoculation, quinoa plants were grown in potting mix (Tsuchitaro, Sumitomo Forestry, Tokyo, Japan) in a plant growth room at 25 °C with 16 h of illumination per day. Capped full-length RNA transcripts of two bromoviruses, <em>Brome mosaic virus</em> (BMV) and <em>Cowpea chlorotic mottle virus</em> (CCMV), were synthesized <em>in vitro</em> using T7 RNA polymerase (Takara Bio, Kusatsu, Japan), as described previously.<a reveal-id="dsw037-B30" data-open="dsw037-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw037-B31" data-open="dsw037-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> A mixture of transcripts of viral RNAs 1, 2, and 3 was inoculated mechanically with Carborundum onto the four youngest fully expanded leaves of 5-week-old quinoa plants. The inoculated plants were kept in the growth room and observed for symptom expression.</p>                            <h3 scrollto-destination=48005221 id="48005221" class="section-title" >2.4. Extraction and purification of nuclear DNA</h3>
<p>Approximately 80 g of quinoa leaves was harvested from 52-day-old single plants. Cell lysis and the isolation of nuclei were performed using CelLytic PN Isolation/Extraction Kit (Sigma-Aldrich) according to the manufacturer’s instructions with some modifications. Twenty grams of leaves were homogenized in liquid nitrogen, and then the tissue powder was suspended in 800 ml Nuclei Isolation Buffer (Sigma-Aldrich). Homogenization and suspension were repeated four times. The suspension was passed through a Filter Mesh 100 (Sigma-Aldrich), and the supernatant was centrifuged at 1,300 g for 10 min at 4 °C. The precipitate was resuspended in 15 ml of Wash Buffer (Sigma-Aldrich), followed by centrifugation at 2,200 g for 3 min at 4 °C. The wash step was repeated three times. The washed precipitate was resuspended in 10 ml of AP1 Buffer (Qiagen), 40 μl of RNase A solution (Qiagen) was added, and the solution was incubated at 65 °C for 18 min. After centrifugation at 21,500 g for 10 min at 4 °C, the supernatant was transferred to a 50-ml tube. Then, 3.3 ml of Buffer P3 (Qiagen) was added to the tube, which was centrifuged at 21,500 g for 10 min at 4 °C. An equal volume of 2-propanol was added to the supernatant. After gentle mixing, the supernatant was centrifuged at 17,200 <em>g</em> for 3 min at 4 °C. The precipitate was rinsed twice with 70% ethanol. Finally, the resulting dried, pure nuclear DNA was dissolved in 10 ml of Buffer AE (Qiagen).</p>                            <h3 scrollto-destination=48005223 id="48005223" class="section-title" >2.5. DNA sequencing</h3>
<p>Library construction and sequencing were performed at Takara Bio. For sequencing using the Illumina HiSeq platform, a paired-end (PE) library with insert sizes of 185 bp and two mate-pair (MP) libraries with expected insert sizes of 2,700–3,500 bp and of 9,000–11,000 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were constructed from nuclear DNA according to the manufacturer’s protocol (Illumina Inc., CA, USA). These libraries were sequenced using a HiSeq 2500 sequencer. For sequencing by PacBio RS II platform, libraries with expected insert sizes (25,300–28,700 bp) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were constructed according to the manufacture’s protocol (Pacific Biosciences of California, Inc., CA, USA). The libraries were sequenced on 40 single-molecule real-time (SMRT) cells of PacBio RS II.</p>                            <h3 scrollto-destination=48005225 id="48005225" class="section-title" >2.6. Amplification and sequencing of <em>RDR1</em> genomic fragments</h3>
<p>Genomic fragments of <em>CqRDR1A</em> and <em>CqRDR1B</em> from quinoa (Kd), and C<em>pRDR1</em> of <em>C. pallidicaule</em> were amplified using PrimeSTAR GXL DNA Polymerase (Takara Bio). For amplification of <em>CqRDR1B</em>, two overlapping genomic fragments, named the 5′-half and 3′-half, were separately amplified. The amplified genomic fragments were purified using the Wizard SV Gel and PCR Clean-Up System (Promega). Sequence data were obtained using the ABI PRISM 3130x1 Genetic Analyzer (Life Technologies). To verify the segregation pattern of <em>CqRDR1A</em> and <em>CqRDR1B</em> genes in the F<sub>1</sub> progeny of Kd, a co-dominant PCR marker, rdr1-MF2/MR2, was designed. Genomic DNA from 20 self-pollinated F<sub>1</sub> progeny was used as PCR templates, and PCR products were analysed on a 4% agarose gel stained with ethidium bromide. Primers used are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=48005227 id="48005227" class="section-title" >2.7. Extraction of total RNA and DNase I treatment</h3>
<p>For RNA Sequencing (RNA-Seq) analysis, total RNA was extracted from various quinoa organs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), using a phenol-SDS method,<a reveal-id="dsw037-B32" data-open="dsw037-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> with minor modifications. Quinoa organs were homogenized with liquid nitrogen, and the tissue powders were suspended in TE-saturated-phenol/RNA extraction buffer (0.2 M Tris, 0.2 M LiCl, 5 mM EDTA, and 1% SDS) solution (1/1; vol/vol), mixed well, and centrifuged at 21,500 <em>g</em> for 5 min at 4 °C. The upper aqueous layer was transferred to phenol-chloroform-isoamyl alcohol solution followed, vortexed, and then centrifuged at 21,500 <em>g</em> for 5 min at 4 °C. The total RNA in the upper aqueous phase was precipitated by adding 1/100 volume of acetic acid and an equal volume of 2-propanol. After incubation at −30 °C for 20 min, the suspension was centrifuged at 21,500 <em>g</em> for 15 min at 4 °C. The precipitate was dissolved in nuclease-free water and centrifuged to remove insoluble matter. A quarter volume of 10 M LiCl was added to the supernatant in tubes. The samples were vortexed and incubated at 4 °C for at least 1 h. After centrifuging at 21,500 <em>g</em> for 15 min at 4 °C, the precipitate was rinsed with 2 M LiCl solution and then dissolved in nuclease-free water. The total RNA solution was subjected to phenol-chloroform extraction followed by ethanol precipitation, and then the precipitates were rinsed with 70% ethanol. The dried total RNA was dissolved in nuclease-free water, and the quality was checked using a Bioanalyzer (model 2100; Agilent). Forty micrograms of total RNA was treated with RQ1 DNase I (Promega), according to the manufacturer’s instructions. After DNase I treatment, the total RNA was purified using an RNeasy Plant Mini Kit (Qiagen). First-strand cDNA was synthesized from 1.0 µg total RNA using the PrimeScript High Fidelity RT-PCR Kit (Takara Bio) with oligo-dT (20) primer, according to the manufacturer’s instructions.</p>                            <h3 scrollto-destination=48005229 id="48005229" class="section-title" >2.8. RNA sequencing</h3>
<p>Library preparation, sequencing, and assembly were performed at Eurofins Genomics (Ebersberg, Germany). The normalization library was prepared according to Shimizu et al.<a reveal-id="dsw037-B33" data-open="dsw037-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The normalized library was sequenced using an Illumina MiSeq with a 2× 150-bp read module. Low-quality reads and adapter sequences were trimmed using Trimmomatic v0.32.<a reveal-id="dsw037-B34" data-open="dsw037-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h3 scrollto-destination=48005231 id="48005231" class="section-title" >2.9. Estimation of genome size</h3>
<p>For genome size estimation, PE reads with a k-mer size of 17 were used, as reported previously.<a reveal-id="dsw037-B35" data-open="dsw037-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> The k-mer distribution was investigated using Jellyfish 2.1.3.<a reveal-id="dsw037-B36" data-open="dsw037-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The genome size and coverage (i.e. the number of base pairs sequenced as a multiple of the number of base pairs present in the genome) were estimated using the peak at 97 on the k-mer frequency distribution curve (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) as described previously.<a reveal-id="dsw037-B35" data-open="dsw037-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                            <h3 scrollto-destination=48005233 id="48005233" class="section-title" >2.10. Genome assembly</h3>
<p>The genome was assembled at Takara Bio. The adaptor sequences of Illumina reads were trimmed using cutadapt v1.2.1.<a reveal-id="dsw037-B37" data-open="dsw037-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The trimmed Illumina reads were assembled using ALLPATHS-LG v52488,<a reveal-id="dsw037-B38" data-open="dsw037-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> with the setting HAPLOIDIFY= TRUE. PacBio reads of longer than 1,000 bp were used for gap-closing and further scaffolding of the assemblies obtained from the Illumina reads by PBJelly2 v14.1.14 under default settings.<a reveal-id="dsw037-B27" data-open="dsw037-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Sequences homologous to bacterial, fungal, and human (hg19) genome sequences, vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/" target="_blank">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>), chloroplast (Accession number: NC_000932.1) and mitochondrial (Accession number: NC_001284.2) genome sequences from <em>A. thaliana</em>, and the PhiX sequence used for background controlling in Illumina sequencing by BLASTN<a reveal-id="dsw037-B39" data-open="dsw037-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> searches with an E-value cutoff of 1E-10 and length coverage of ≥10%, were excluded as probable contamination. Finally, scaffolds longer than 300 bp were selected and designated Cqu_r1.0. Repetitive sequences in Cqu_r1.0 were detected using RepeatScout 1.0.5.<a reveal-id="dsw037-B40" data-open="dsw037-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and RepeatMasker 4.0.3 (<a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>) as described previously.<a reveal-id="dsw037-B35" data-open="dsw037-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                            <h3 scrollto-destination=48005235 id="48005235" class="section-title" >2.11. Gene prediction and annotation</h3>
<p>To analyse the relationship between the transcriptome and the quinoa genome sequences, we mapped a <em>de novo</em> assembly of transcriptome sequences against Cqu_r1.0. <em>De novo</em> assembly was conducted using the software tools Velvet v1.2.10 and Oases v0.2.08.<a reveal-id="dsw037-B41" data-open="dsw037-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsw037-B42" data-open="dsw037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> A multi-kmer approach was applied. In this approach, separate kmers are first assembled (kmers 59, 69, 79, 89) and then the set of assemblies is merged into a single ‘merged’ assembly (kmer 29). The assembled transcripts were clustered based on sequence identity (≥99%) using the software CD-HIT-EST v4.6.<a reveal-id="dsw037-B43" data-open="dsw037-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> These transcripts were mapped against the genome sequence (Cqu_r1.0) using GMAP v2016-05-01 software<a reveal-id="dsw037-B44" data-open="dsw037-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> with the threshold option of ≥95% identity and ≥80% coverage.</p><p>RNA-Seq reads were mapped onto the draft genome sequence (Cqu_r1.0) with TopHat 2.0.12.<a reveal-id="dsw037-B45" data-open="dsw037-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The bam file obtained was used to generate the training set for the gene prediction of BRAKER1 pipeline.<a reveal-id="dsw037-B46" data-open="dsw037-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Using the training set, the genes were predicted by Augustus 3.0.3.<a reveal-id="dsw037-B47" data-open="dsw037-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The RNA-Seq reads were mapped onto the predicted genes, and splicing variants were excluded by RSEM 1.2.15.<a reveal-id="dsw037-B48" data-open="dsw037-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> The predicted genes in the quinoa genome together with those in <em>S. oleracea</em> (Spinach-1.0, 21,702 genes), <em>B. vulgaris</em> (RefBeet-1.1, 27,421 genes), <em>Amaranthus hypochondriacus</em> (AhG2s, 30,564 genes), and <em>A. thaliana</em> genomes (TAIR10, 35,386 genes) were clustered using the CD-HIT program<a reveal-id="dsw037-B43" data-open="dsw037-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> with the parameters c = 0.4 and aL = 0.4.</p><p>The predicted genes were subjected to similarity searches against the NCBI NR database (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz</a>) and amino acid sequences of <em>A. thaliana</em> from TAIR10 (<a class="link link-uri" href="https://www.arabidopsis.org" target="_blank">https://www.arabidopsis.org</a>) using BLASTX with an E-value cutoff of 1E-10. The top hit was used to assign the product name. BLAST searches against UniProt (TrEMBL + Swiss-Prot) with an <em>E</em>-value cut-off of 1E-20 were also carried out. A domain search against InterPro (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/" target="_blank">http://www.ebi.ac.uk/interpro/</a>) was conducted using InterProScan<a reveal-id="dsw037-B49" data-open="dsw037-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> with an <em>E</em>-value cutoff of 1.0. Finally, genes were classified based on the NCBI euKaryotic clusters of Orthologous Groups (KOG) database<a reveal-id="dsw037-B50" data-open="dsw037-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> by performing BLAST searches with an <em>E</em>-value cutoff of 1E-4. In addition, the genes were mapped onto the KEGG reference pathways by BLAST searches against the KEGG GENES database (<a class="link link-uri" href="http://www.genome.jp/kegg/genes.html" target="_blank">http://www.genome.jp/kegg/genes.html</a>) with an <em>E</em>-value cut-off of 1E-4, length of coverage of 25%, and identity of 50%.</p><p>Genes related to transposable elements (TEs) were inferred based on a BLAST search against the NCBI NR database and conserved domains were identified based on a search against InterPro and GyDB 2.0<a reveal-id="dsw037-B51" data-open="dsw037-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> using hmmsearch in HMMER 3.0<a reveal-id="dsw037-B52" data-open="dsw037-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> with an <em>E</em>-value cutoff of 1.0. Transfer RNA genes (tRNAs) were predicted using tRNAscan-SE v.1.23.<a reveal-id="dsw037-B53" data-open="dsw037-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Ribosomal RNA genes (rRNAs) were predicted in BLASTN searches with an <em>E</em>-value cutoff of 1E-10 using <em>A. thaliana</em> 5.8S and 25S rRNAs (Accession number: X52320.1) and 18S rRNA (Accession number: X16077.1) as queries.</p>                            <h3 scrollto-destination=48005240 id="48005240" class="section-title" >2.12. Estimation of the pair-wise nucleotide divergence at synonymous site (Ks) between putative homoeologous genes</h3>
<p>CD-HIT was used to identify putative homoeologous genes. Using the amino acid data set containing 62,512 sequences annotated by BLASTP searches against the NCBI NR database, gene clusters containing two genes (i.e. putative homoeologous genes) were surveyed by the CD-HIT program with the parameters c = 0.9 and aS = 0.5. Paired genes on the same scaffold were not used for further analyses, as they might be tandemly duplicated genes. Alignments of two protein sequences were conducted by BLASTP, and subsequent codon alignment was conducted by PAL2NAL.<a reveal-id="dsw037-B54" data-open="dsw037-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Finally, Ks values were estimated by PAML<a reveal-id="dsw037-B55" data-open="dsw037-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> and the frequency of Ks (0 &lt; Ks &lt; 0.5) was plotted.</p>                            <h3 scrollto-destination=48005242 id="48005242" class="section-title" >2.13. Phylogenetic analyses</h3>
<p>Alignments of sequences were carried out using CLUSTALW2.<a reveal-id="dsw037-B56" data-open="dsw037-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> The evolutionary history was inferred using the neighbor-joining method,<a reveal-id="dsw037-B57" data-open="dsw037-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and the evolutionary distances were computed using the JTT matrix-based method.<a reveal-id="dsw037-B58" data-open="dsw037-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p>                            <h2 scrollto-destination=48005244 id="48005244" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=48005245 id="48005245" class="section-title" >3.1. A standard quinoa accession Kd for molecular genetic analyses</h3>
                            <h4 scrollto-destination=48005246 id="48005246" class="section-title" data-legacy-id=SEC3.1.1>3.1.1. Kd is an appropriate quinoa accession for genome sequencing</h4>
<p>Although quinoa is essentially self-pollinating, this species does exhibit genetic heterogeneity due to outcrossing.<a reveal-id="dsw037-B2" data-open="dsw037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw037-B20" data-open="dsw037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Quinoa is a gynomonoecious species that has both hermaphrodite and female flowers on the same plant, so that quinoa plants can accept pollen from the other individuals. Heterozygous alleles at a microsatellite locus were observed in 32% of quinoa accessions.<a reveal-id="dsw037-B20" data-open="dsw037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Thus, reducing the genome complexity by repeated self-pollination is a crucial step for sequencing the genome of the allotetraploid species quinoa.</p><div class="block-child-p">A quinoa line has been propagated in an air-conditioned plant growth room without outcrossing for over 20 years.<a reveal-id="dsw037-B29" data-open="dsw037-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> We multiplied seeds derived from this line with special non-woven pollination bags, in which inflorescences had been enclosed during flowering stage to prevent cross-pollination, producing inbred accession Kyoto-d (Kd) seeds. The inbred accession Kd (<a reveal-id="dsw037-F1" data-open="dsw037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A–H) is more stable in terms of phenotypic uniformity, which has been evaluated based on the extent of variation in size, colour, and morphological characteristics of seeds, seedlings, and flowers in every generation obtained thus far, than any of the more than 150 accessions collected from research institutes (<a reveal-id="dsw037-F1" data-open="dsw037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A; more detailed data will be published elsewhere), indicating that Kd is a suitable standard accession for molecular genetics and biology analyses. Kd seeds are approximately 2 mm in diameter (<a reveal-id="dsw037-F1" data-open="dsw037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>D). Like other quinoa lines (for example, Bonales-Alatorre et al.<a reveal-id="dsw037-B59" data-open="dsw037-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a>), epidermal bladder cells (salt bladders) are present on the surface of leaves (<a reveal-id="dsw037-F1" data-open="dsw037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>E and F). Guard cells are also observed on both abaxial and adaxial surface of leaves (<a reveal-id="dsw037-F1" data-open="dsw037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>G and H). <div data-id="dsw037-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f1p.png?Expires=2147483647&Signature=XYTpHLBLHC2U9nHENUw05lNfxrIO1SWvmk509yWGwYjSZjfzwaLu9NjOIfdUmRQ6xVgbo2wPReVWVvJqXhtQOTIRovoT8BG59XrbUA9myohSrXt95nPFnPEN37tk52GdCnfBis3J6c84kV1Q4DwJr86uSLdp7tsnccK7EA1OXB-JO6ln57VssKO46csPA3ZUKDcIrM4TXWDlHSmD6mwJxpRNMs3TjpzNCaPmcGW6DeOcNw4RZ-OkYdDWvEogLlQDurFdmpUByiRgg3qJzTciiUlhg4YOv1ETMruvHoQf962t9JJgWQj2YxLqLALH53zodL~JjdOsl5TCprcVd0XjWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Morphological characteristics of quinoa (Kd) plants. (A) 14 day-old quinoa (Kd) seedlings grown in soil. Scale bar = 1 cm. (B) 32 day-old quinoa (Kd) plant grown in soil. Scale bar = 5 cm. (C) A main panicle. Scale bar = 5 cm. (D) Dried mature quinoa (Kd) seeds. Scale bar = 1 cm. (E) Epidermal bladder cells (salt bladders) on abaxial surface of a young quinoa leaf (the leaf blade length: 15 mm). Scale bar = 0.5 mm. (F) Epidermal bladder cells (salt bladders) on abaxial surface of a fully expanded quinoa leaf. Scale bar = 0.5 mm. (G) Epidermal bladder and guard cells on abaxial surface of a fully expanded quinoa leaf were observed using a colour laser three-dimensional profile microscope (Keyence, Osaka, Japan), which shows epidermal bladder and stomatal aperture with no pretreatments. White arrows indicate epidermal bladder cells. Scale bar = 50 µm. (H) Enlarged view of guard cells on abaxial surface of a fully expanded quinoa leaf was provided using the microscope as described in (G). Scale bar = 50 µm." path-from-xml="dsw037f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005248/dsw037f1p.tif" path-from-xml="dsw037f1p.tif" target="_blank">View large</a><a section="48005248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f1p.png?Expires=2147483647&Signature=UfL4SfpOXv6r8rYATwNUP6Z3tQSaePXgnDr~34vBcxCCfRKXn72K8uHAZcfChX9AcOi8pQxxaAIIlCG9jDWvu7g3PROFCA3RLph5Umbm1UUxe1vxoQUXypOeQ-ThiGIiHIAgSp6n4d1vPQ-Txs6gUiWnY6T3ZJ7DBsQEttV9wfqaSjzXZLOW~qoFUw-nIKXnz8UtwBFX72szdajJ2Voo73lfTuYgCG1lAFPpXeQV-UUgu~qaUNDhafqZROFXRcx3QZlyndgdYC8Eds9eHZkaELUXFAKJHatoJtVWQaf4qSdhzZTFT~zmg3ZGii5Yxx2H1I2Jrc1u2HkD3FLz-KL0iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005248&ar=2647444&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw037f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Morphological characteristics of quinoa (Kd) plants. (A) 14 day-old quinoa (Kd) seedlings grown in soil. Scale bar = 1 cm. (B) 32 day-old quinoa (Kd) plant grown in soil. Scale bar = 5 cm. (C) A main panicle. Scale bar = 5 cm. (D) Dried mature quinoa (Kd) seeds. Scale bar = 1 cm. (E) Epidermal bladder cells (salt bladders) on abaxial surface of a young quinoa leaf (the leaf blade length: 15 mm). Scale bar = 0.5 mm. (F) Epidermal bladder cells (salt bladders) on abaxial surface of a fully expanded quinoa leaf. Scale bar = 0.5 mm. (G) Epidermal bladder and guard cells on abaxial surface of a fully expanded quinoa leaf were observed using a colour laser three-dimensional profile microscope (Keyence, Osaka, Japan), which shows epidermal bladder and stomatal aperture with no pretreatments. White arrows indicate epidermal bladder cells. Scale bar = 50 µm. (H) Enlarged view of guard cells on abaxial surface of a fully expanded quinoa leaf was provided using the microscope as described in (G). Scale bar = 50 µm.</p></div></div><div content-id="dsw037-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f1p.png?Expires=2147483647&Signature=XYTpHLBLHC2U9nHENUw05lNfxrIO1SWvmk509yWGwYjSZjfzwaLu9NjOIfdUmRQ6xVgbo2wPReVWVvJqXhtQOTIRovoT8BG59XrbUA9myohSrXt95nPFnPEN37tk52GdCnfBis3J6c84kV1Q4DwJr86uSLdp7tsnccK7EA1OXB-JO6ln57VssKO46csPA3ZUKDcIrM4TXWDlHSmD6mwJxpRNMs3TjpzNCaPmcGW6DeOcNw4RZ-OkYdDWvEogLlQDurFdmpUByiRgg3qJzTciiUlhg4YOv1ETMruvHoQf962t9JJgWQj2YxLqLALH53zodL~JjdOsl5TCprcVd0XjWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Morphological characteristics of quinoa (Kd) plants. (A) 14 day-old quinoa (Kd) seedlings grown in soil. Scale bar = 1 cm. (B) 32 day-old quinoa (Kd) plant grown in soil. Scale bar = 5 cm. (C) A main panicle. Scale bar = 5 cm. (D) Dried mature quinoa (Kd) seeds. Scale bar = 1 cm. (E) Epidermal bladder cells (salt bladders) on abaxial surface of a young quinoa leaf (the leaf blade length: 15 mm). Scale bar = 0.5 mm. (F) Epidermal bladder cells (salt bladders) on abaxial surface of a fully expanded quinoa leaf. Scale bar = 0.5 mm. (G) Epidermal bladder and guard cells on abaxial surface of a fully expanded quinoa leaf were observed using a colour laser three-dimensional profile microscope (Keyence, Osaka, Japan), which shows epidermal bladder and stomatal aperture with no pretreatments. White arrows indicate epidermal bladder cells. Scale bar = 50 µm. (H) Enlarged view of guard cells on abaxial surface of a fully expanded quinoa leaf was provided using the microscope as described in (G). Scale bar = 50 µm." path-from-xml="dsw037f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005248/dsw037f1p.tif" path-from-xml="dsw037f1p.tif" target="_blank">View large</a><a section="48005248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f1p.png?Expires=2147483647&Signature=UfL4SfpOXv6r8rYATwNUP6Z3tQSaePXgnDr~34vBcxCCfRKXn72K8uHAZcfChX9AcOi8pQxxaAIIlCG9jDWvu7g3PROFCA3RLph5Umbm1UUxe1vxoQUXypOeQ-ThiGIiHIAgSp6n4d1vPQ-Txs6gUiWnY6T3ZJ7DBsQEttV9wfqaSjzXZLOW~qoFUw-nIKXnz8UtwBFX72szdajJ2Voo73lfTuYgCG1lAFPpXeQV-UUgu~qaUNDhafqZROFXRcx3QZlyndgdYC8Eds9eHZkaELUXFAKJHatoJtVWQaf4qSdhzZTFT~zmg3ZGii5Yxx2H1I2Jrc1u2HkD3FLz-KL0iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005248&ar=2647444&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw037f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Morphological characteristics of quinoa (Kd) plants. (A) 14 day-old quinoa (Kd) seedlings grown in soil. Scale bar = 1 cm. (B) 32 day-old quinoa (Kd) plant grown in soil. Scale bar = 5 cm. (C) A main panicle. Scale bar = 5 cm. (D) Dried mature quinoa (Kd) seeds. Scale bar = 1 cm. (E) Epidermal bladder cells (salt bladders) on abaxial surface of a young quinoa leaf (the leaf blade length: 15 mm). Scale bar = 0.5 mm. (F) Epidermal bladder cells (salt bladders) on abaxial surface of a fully expanded quinoa leaf. Scale bar = 0.5 mm. (G) Epidermal bladder and guard cells on abaxial surface of a fully expanded quinoa leaf were observed using a colour laser three-dimensional profile microscope (Keyence, Osaka, Japan), which shows epidermal bladder and stomatal aperture with no pretreatments. White arrows indicate epidermal bladder cells. Scale bar = 50 µm. (H) Enlarged view of guard cells on abaxial surface of a fully expanded quinoa leaf was provided using the microscope as described in (G). Scale bar = 50 µm.</p></div></div></div>                            <h4 scrollto-destination=48005249 id="48005249" class="section-title" >3.1.2. Kd responds well to abiotic and biotic stresses</h4>
<div class="block-child-p">Quinoa plants are able to grow under a wide spectrum of harsh environments<a reveal-id="dsw037-B60" data-open="dsw037-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and are facultative C3 halophytes.<a reveal-id="dsw037-B11" data-open="dsw037-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw037-B59" data-open="dsw037-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsw037-B61" data-open="dsw037-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Indeed, all of the Kd plants survived for at least 22 days after treatment with 300 mM NaCl, whereas no <em>A. thaliana</em> plants (CS60000, the inbred line used for whole-genome sequencing) did (<a reveal-id="dsw037-F2" data-open="dsw037-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A–D), indicating that Kd also displays enhanced tolerance to high salinity in comparison with the C3 glycophyte <em>A. thaliana</em>. These data are consistent with previous findings for quinoa plants subjected to salt tolerance tests,<a reveal-id="dsw037-B11" data-open="dsw037-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw037-B59" data-open="dsw037-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> suggesting that Kd is appropriate for molecular analyses to identify the mechanisms underlying enhanced tolerance of quinoa to abiotic stresses. <div data-id="dsw037-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f2p.png?Expires=2147483647&Signature=R1PEVWKRUa0K75AiqIN3ARk3ZXzljBqkePDgJfQTyz0dXi3Wg6LFLZlVjpzo4~niEPKna~1HSqu-peOUqmc8EYY0RNaeKhak-c-2VhF-zIhcy8KUKxuelNELTCB3NAmYygziRwm-hXhmkpxIZxi8wixd4K-R2hR2Zy6pqBonxNxFQUnmVBdCIVMc8O4tfK7STDP9IRhnMs2hwX-r0hwhHMBHJZcWnqrThlySvwyxwB9C~vQiWEFdklHrXR4vhRiACjsHbxEBu-8IH5l-PlS-9fPiQ5hyiBI6EbbVX9JxkAlenFbLBckUT6bSymxR~~YpjESyThLpSePJKhV1Xe7R2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quinoa (Kd) plants exhibited higher salt tolerance and characteristic symptoms of virus infections. (A–D) Comparative analysis of salinity tolerance in quinoa (Kd) versus Arabidopsis plants grown in soil. 3 week-old quinoa (A, C) and Arabidopsis (B, D) plants were treated with 0 mM NaCl (A, B) or 300 mM NaCl (C, D) for 3 weeks and photographed at 6 weeks after germination. Scale bar = 8 cm. (E) Chlorotic or necrotic local lesions induced on the inoculated quinoa leaves mechanically inoculated with Brome mosaic virus (BMV) or Cowpea chlorotic mottle virus (CCMV) are viewed at 7 days post-inoculation, respectively. Mock, mock inoculation. Scale bar = 2 cm. (F) Systemic or nonsystemic infection of quinoa plants inoculated with BMV or CCMV viewed at 21 days post-inoculation, respectively. Scale bar = 8 cm." path-from-xml="dsw037f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005250/dsw037f2p.tif" path-from-xml="dsw037f2p.tif" target="_blank">View large</a><a section="48005250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f2p.png?Expires=2147483647&Signature=3-w6NMlA826UBUrcPiP--g2YRj8DfH62axuRqkI39GoK6QXV9QLHQG4uVm6SxRZIRf2uj22tVu5ulUyPiCto0N73fpr-zBY~rlr6exsJ-ti6qb~kkdb8LYfefHOe-53Q5u69ALrVH5ROxb47uXXLbhrbRVKT26mePlhXhCwVhVV8ZNpfo4Izg8A1SPdAjoE8rvIgazE6yuGwabPE8TUXrnBAPXkP736bGsiAD2uhdQdDYEyBBbmqWLIFV1qOwwJ9C23kL45VlrytARvr59~gure88pMCw99dt0q~K5r6TcjBnDiWEQCDKNnniki5c0HUXIwgxzj6JmrxJPzk~EzR5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005250&ar=2647444&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw037f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quinoa (Kd) plants exhibited higher salt tolerance and characteristic symptoms of virus infections. (A–D) Comparative analysis of salinity tolerance in quinoa (Kd) versus <em>Arabidopsis</em> plants grown in soil. 3 week-old quinoa (A, C) and <em>Arabidopsis</em> (B, D) plants were treated with 0 mM NaCl (A, B) or 300 mM NaCl (C, D) for 3 weeks and photographed at 6 weeks after germination. Scale bar = 8 cm. (E) Chlorotic or necrotic local lesions induced on the inoculated quinoa leaves mechanically inoculated with <em>Brome mosaic virus</em> (BMV) or <em>Cowpea chlorotic mottle virus</em> (CCMV) are viewed at 7 days post-inoculation, respectively. Mock, mock inoculation. Scale bar = 2 cm. (F) Systemic or nonsystemic infection of quinoa plants inoculated with BMV or CCMV viewed at 21 days post-inoculation, respectively. Scale bar = 8 cm.</p></div></div><div content-id="dsw037-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f2p.png?Expires=2147483647&Signature=R1PEVWKRUa0K75AiqIN3ARk3ZXzljBqkePDgJfQTyz0dXi3Wg6LFLZlVjpzo4~niEPKna~1HSqu-peOUqmc8EYY0RNaeKhak-c-2VhF-zIhcy8KUKxuelNELTCB3NAmYygziRwm-hXhmkpxIZxi8wixd4K-R2hR2Zy6pqBonxNxFQUnmVBdCIVMc8O4tfK7STDP9IRhnMs2hwX-r0hwhHMBHJZcWnqrThlySvwyxwB9C~vQiWEFdklHrXR4vhRiACjsHbxEBu-8IH5l-PlS-9fPiQ5hyiBI6EbbVX9JxkAlenFbLBckUT6bSymxR~~YpjESyThLpSePJKhV1Xe7R2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quinoa (Kd) plants exhibited higher salt tolerance and characteristic symptoms of virus infections. (A–D) Comparative analysis of salinity tolerance in quinoa (Kd) versus Arabidopsis plants grown in soil. 3 week-old quinoa (A, C) and Arabidopsis (B, D) plants were treated with 0 mM NaCl (A, B) or 300 mM NaCl (C, D) for 3 weeks and photographed at 6 weeks after germination. Scale bar = 8 cm. (E) Chlorotic or necrotic local lesions induced on the inoculated quinoa leaves mechanically inoculated with Brome mosaic virus (BMV) or Cowpea chlorotic mottle virus (CCMV) are viewed at 7 days post-inoculation, respectively. Mock, mock inoculation. Scale bar = 2 cm. (F) Systemic or nonsystemic infection of quinoa plants inoculated with BMV or CCMV viewed at 21 days post-inoculation, respectively. Scale bar = 8 cm." path-from-xml="dsw037f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005250/dsw037f2p.tif" path-from-xml="dsw037f2p.tif" target="_blank">View large</a><a section="48005250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f2p.png?Expires=2147483647&Signature=3-w6NMlA826UBUrcPiP--g2YRj8DfH62axuRqkI39GoK6QXV9QLHQG4uVm6SxRZIRf2uj22tVu5ulUyPiCto0N73fpr-zBY~rlr6exsJ-ti6qb~kkdb8LYfefHOe-53Q5u69ALrVH5ROxb47uXXLbhrbRVKT26mePlhXhCwVhVV8ZNpfo4Izg8A1SPdAjoE8rvIgazE6yuGwabPE8TUXrnBAPXkP736bGsiAD2uhdQdDYEyBBbmqWLIFV1qOwwJ9C23kL45VlrytARvr59~gure88pMCw99dt0q~K5r6TcjBnDiWEQCDKNnniki5c0HUXIwgxzj6JmrxJPzk~EzR5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48005250&ar=2647444&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw037f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quinoa (Kd) plants exhibited higher salt tolerance and characteristic symptoms of virus infections. (A–D) Comparative analysis of salinity tolerance in quinoa (Kd) versus <em>Arabidopsis</em> plants grown in soil. 3 week-old quinoa (A, C) and <em>Arabidopsis</em> (B, D) plants were treated with 0 mM NaCl (A, B) or 300 mM NaCl (C, D) for 3 weeks and photographed at 6 weeks after germination. Scale bar = 8 cm. (E) Chlorotic or necrotic local lesions induced on the inoculated quinoa leaves mechanically inoculated with <em>Brome mosaic virus</em> (BMV) or <em>Cowpea chlorotic mottle virus</em> (CCMV) are viewed at 7 days post-inoculation, respectively. Mock, mock inoculation. Scale bar = 2 cm. (F) Systemic or nonsystemic infection of quinoa plants inoculated with BMV or CCMV viewed at 21 days post-inoculation, respectively. Scale bar = 8 cm.</p></div></div></div><p>Quinoa and the tobacco species <em>Nicotiana benthamiana</em> are widely used as diagnostic hosts to identify virus species.<a reveal-id="dsw037-B62" data-open="dsw037-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> One well-studied and monocot-adapted bromovirus, BMV, induced chlorotic local lesions that developed into large chlorotic blotches on the inoculated Kd leaves, and then continued to spread systemically, resulting in severe symptoms, including leaf distortions and dwarfing (<a reveal-id="dsw037-F2" data-open="dsw037-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>E and F). In contrast, a closely related and dicot-adapted bromovirus, CCMV, caused just small necrotic local lesions on the inoculated Kd leaves. Interestingly, CCMV was arrested within the lesions and did not spread systemically (<a reveal-id="dsw037-F2" data-open="dsw037-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>E and F). These findings are in accordance with previous reports in the other accessions.<a reveal-id="dsw037-B63" data-open="dsw037-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw037-B64" data-open="dsw037-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Thus, Kd responds to abiotic and biotic stresses in a manner that is representative of quinoa species, and is therefore suitable for molecular analyses of its great adaptability and susceptibility to a wide range of viruses.</p>                            <h3 scrollto-destination=48005252 id="48005252" class="section-title" >3.2. Genome size estimation</h3>
<p>In <em>de novo</em> genome assemblies, most scaffolds are in fragmented, and thus it is often difficult to determine the genome size accurately from the total assembly length. As the first step of our quinoa genome project, we estimated the genome size using the frequency distribution curve of 17-mer obtained from Illumina short reads, as successfully reported in previous studies.<a reveal-id="dsw037-B35" data-open="dsw037-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw037-B65" data-open="dsw037-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> The k-mer frequency distribution curve (k-mer =17) using paired-ends (PEs) with a 185-bp insert size is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>. The highest peak, at a multiplicity of 97, was expected to be a ‘homo-peak’, that contained 17-mers from a homozygous region of the genome.<a reveal-id="dsw037-B66" data-open="dsw037-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Based on the highest peak, the genome size of quinoa was estimated to be 1.5 Gbp, in good agreement with that of 1,448 Mbp from a cytometry analysis.<a reveal-id="dsw037-B21" data-open="dsw037-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> It is noteworthy that no peaks around half of the multiplicity of 97 were detected. A previous study indicated that a ‘hetero-peak’, i.e. a peak at half the multiplicity of the homo-peak, is detected in heterozygous species.<a reveal-id="dsw037-B66" data-open="dsw037-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> A simulation also indicated that the greater the degree of heterozygosity, the greater the height of the hetero-peak.<a reveal-id="dsw037-B66" data-open="dsw037-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> The finding that no distinct hetero-peaks were present in our k-mer analysis implies that the repeated self-pollination of Kd has resulted in a highly homogenized genetic background. The smaller peak at around the doubled multiplicity of the homo-peak (i.e. at around ∼200) is of interest, because it may contain k-mers related to allotetraploidy. However, the k-mer curve distribution for the ‘diploid’ genome is known to show a smaller peak at the doubled multiplicity of the homo-peak.<a reveal-id="dsw037-B67" data-open="dsw037-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Thus, the smaller peak at around ∼200 in our k-mer analysis may relate to genomic sequences caused by both allotetraploidy and other simple duplications.</p>                            <h3 scrollto-destination=48005254 id="48005254" class="section-title" >3.3. Genome assembly of quinoa</h3>
<p>We obtained a large amount of quinoa DNA sequence data using Illumina HiSeq 2500 and PacBio RS II platforms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In total, 290.8 Gbp and 45.8 Gbp of raw data were generated by the HiSeq and PacBio RS II platforms, respectively. The average size of raw reads from PacBio RS II was 10.1 kbp. These Illumina and PacBio sequence data corresponded to 196× and 31× coverage of the quinoa genome, respectively. First, <em>de novo</em> genome assembly was performed using the short reads generated from the HiSeq platform using ALLPATHS-LG. We obtained 110,092 contigs, with a total length of 830.0 Mbp and an N50 size of 14,505 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). These contigs were grouped into 36,423 scaffolds, with a total length of 946.6 Mbp and an N50 size of 53,276 bp. Then, using PacBio long reads and PBJelly2 software, gap-closing and further scaffolding of the assembly generated from HiSeq reads were performed. This strategy reduced the gap lengths in the scaffolds generated using short reads, improving the genome assembly.<a reveal-id="dsw037-B28" data-open="dsw037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> After processing with PBJelly2 software, we obtained 24,847 scaffolds, with a total length of 1.1 Gbp and an N50 length of 86,941 bp. The number of observed gaps (number of ‘N’ and ‘Others’) resulted in a 76% reduction of the number observed using only HiSeq data, and the N50 length was 1.6-fold greater than in the assembly before an application of PBJelly2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). After trimming of two scaffolds that exhibited signs of contamination (identified in a BLAST search), 24,845 scaffolds were designated as the draft genome sequence, Cqu_r1.0 (<a reveal-id="dsw037-T1" data-open="dsw037-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The scaffolds were named ‘Cqu_sc’ followed by a five-digit identifier and the sequence version (e.g. Cqu_sc00001.1). The total length of Cqu_r1.0 was 1.1 Gbp, and the N50 length was 86,941 bp. As with the results of the genome assembly of the heterozygous plant <em>P. veris</em>,<a reveal-id="dsw037-B28" data-open="dsw037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> the length of the assembly for the tetraploid species quinoa was improved by including PacBio long reads. However, the genome coverage rates of the assembly were low both in quinoa (73%) and <em>P. veris</em> (63%). Recently, near-complete genome assembly of a diploid species, <em>Vigna angularis</em> (azuki bean), was achieved using 27.6 Gbp of the PacBio long reads, which corresponds to 51× coverage of the genome size.<a reveal-id="dsw037-B68" data-open="dsw037-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> The authors constructed 2,529 scaffolds (N50 = 3.0 Mbp), covering 97.1% of the <em>V. angularis</em> genome. In our analysis of quinoa, we attained a genome coverage of 30×. Given the complexity of the quinoa genome resulting from ploidy and large genome size, a much greater coverage than 51× of the genome size is needed to use the method reported by Sakai et al.<a reveal-id="dsw037-B68" data-open="dsw037-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Attaining a greater genome coverage is our next objective for refining our quinoa assembly. <div content-id="dsw037-T1" class="table-modal"><div class="table-wrap"><div id="dsw037-T1" data-id="dsw037-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of the draft genome sequences (Cqu_r1.0)</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of sequences </td><td char=".">24,845 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td char=".">1,087,413,657 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td char=".">43,768 </td></tr><tr><td>Max length of sequences (bases) </td><td char=".">641,516 </td></tr><tr><td>Min length of sequences (bases) </td><td char=".">332 </td></tr><tr><td>N50 length (bases) </td><td char=".">86,941 </td></tr><tr><td>Number of undetermined bases </td><td char=".">28,385,628 </td></tr><tr><td>GC% (GC/ATGC) </td><td char=".">36.9 </td></tr></tbody></table></div><div content-id="dsw037-T1" class="table-modal"><table><tbody><tr><td>Number of sequences </td><td char=".">24,845 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td char=".">1,087,413,657 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td char=".">43,768 </td></tr><tr><td>Max length of sequences (bases) </td><td char=".">641,516 </td></tr><tr><td>Min length of sequences (bases) </td><td char=".">332 </td></tr><tr><td>N50 length (bases) </td><td char=".">86,941 </td></tr><tr><td>Number of undetermined bases </td><td char=".">28,385,628 </td></tr><tr><td>GC% (GC/ATGC) </td><td char=".">36.9 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw037-T1" data-id="dsw037-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Statistics of the draft genome sequences (Cqu_r1.0)</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of sequences </td><td char=".">24,845 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td char=".">1,087,413,657 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td char=".">43,768 </td></tr><tr><td>Max length of sequences (bases) </td><td char=".">641,516 </td></tr><tr><td>Min length of sequences (bases) </td><td char=".">332 </td></tr><tr><td>N50 length (bases) </td><td char=".">86,941 </td></tr><tr><td>Number of undetermined bases </td><td char=".">28,385,628 </td></tr><tr><td>GC% (GC/ATGC) </td><td char=".">36.9 </td></tr></tbody></table></div><div content-id="dsw037-T1" class="table-modal"><table><tbody><tr><td>Number of sequences </td><td char=".">24,845 </td></tr><tr><td>Cumulative length of sequences (bases) </td><td char=".">1,087,413,657 </td></tr><tr><td>Average length of sequences per contig (bases) </td><td char=".">43,768 </td></tr><tr><td>Max length of sequences (bases) </td><td char=".">641,516 </td></tr><tr><td>Min length of sequences (bases) </td><td char=".">332 </td></tr><tr><td>N50 length (bases) </td><td char=".">86,941 </td></tr><tr><td>Number of undetermined bases </td><td char=".">28,385,628 </td></tr><tr><td>GC% (GC/ATGC) </td><td char=".">36.9 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=48005256 id="48005256" class="section-title" >3.4. Gene prediction and annotation</h3>
<p>Transcriptome sequencing of the normalized cDNA library, which was derived from 12 different tissues of quinoa, produced 14.5 million reads, corresponding to 4.3 Gbp after quality trimming (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We obtained a <em>de novo</em> transcriptome assembly containing 87,877 transcripts (≥100 bp). Of these transcripts, we were able to successfully map 71,744 (81.6%) onto Cqu_r1.0, which is higher than the mapping efficiency reported for barley<a reveal-id="dsw037-B69" data-open="dsw037-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> (74.9%). Our genome assembly covers much of the transcriptome assembly, suggesting that it represents a suitable framework for gene prediction of <em>C. quinoa</em>. Using these RNA-Seq reads as the training set to predict genes for Augustus 3.0.3, 226,647 CDSs (Cqu_r1.0_cds) consisting of 190.5 Mbp, were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p><p>Genes related to transposable elements (TEs) were inferred according to BLAST searches against the NCBI NR database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The total length of known repeats was 132.7 Mbp (12.2% of Cqu_r1.0) and Class I LTR elements were frequently found (7.2% of Cqu_r1.0). In this analysis, we identified unique repeats that had not previously been sequenced, and these had a total length of 535.5 Mbp and accounted for 49.2% of Cqu_r1.0. Genes annotated as transposons were tagged ‘TE’ in the sequence name.</p><p>Based on BLAST searches against the NCBI NR database, the genes were divided into four classes. The first class contains genes that include both a start and stop codon; the second contains genes that include either a start codon or a stop codon, or that lack both a start and stop codon; the third contains genes that include a stop codon in the coding region; and the fourth contains short sequences (encoding &lt; 50 amino acids). The tags based on this classification system and the numbers of genes in each tag are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>. We found that 150,029 genes in the first and second classes were not related to TEs, and we successfully annotated 62,512 of these 150,029 genes by BLAST searches against the UniProtKB database. Thus, the number of annotated genes (62,512 genes) is roughly twice that predicted in most other plant diploid genomes.<a reveal-id="dsw037-B70" data-open="dsw037-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> This appears to be reasonable, since quinoa is a tetraploid species. However, given that a large number of genes (87,517 genes) were not annotated by the BLAST searches against the UniProtKB database, the number of protein-coding genes of quinoa predicted in this study might still be an underestimation. In addition to the protein-coding genes, we identified 2,592 genes for tRNAs, and the number of genes for each tRNA is summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. The draft genome sequence (Cqu_r1.0), predicted gene sequences, deduced amino acid sequences, annotations derived from BLAST searches against the NCBI NR and TAIR10 databases, and domains identified in the search against InterPro were included in the free-access Quinoa Genome DataBase (QGDB; <a class="link link-uri" href="http://quinoa.kazusa.or.jp" target="_blank">http://quinoa.kazusa.or.jp</a>). In addition, local BLAST searches and keyword searches for gene names and their annotations were also implemented in the QGDB.</p><p>An amino acid sequence similarity search was performed for the predicted proteins encoded by the subset of 62,512 genes using the CD-HIT program with the parameters c = 0.9 and aS = 0.5, and 46,695 of sequence homology clusters were identified. Of these, 9,768 contained two genes; i.e. 19,536 of 62,512 predicted genes were paired. Of the 9,768 pairs, 131 were located on the same scaffold, and might be tandemly duplicated genes. The average of Ks in the remaining 9,637 clusters was 0.08, and the distribution of Ks values peaked at ∼0.035 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Considering that the Ks values of these duplicated genes are similar to the previously estimated Ks average between two homoeologous <em>SOS1</em> genes (Ks = 0.077),<a reveal-id="dsw037-B71" data-open="dsw037-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> most of these duplicated genes in a single cluster can be considered as putative homoeologs derived from two genomes. A major challenge in the <em>de novo</em> assembly of polyploids is to differentiate the homoeologous genomes and to assign the genes/scaffolds to each subgenome. In hexaploid wheat, transcriptome assemblies in ancestral diploid species were used to classify the genes predicted from a <em>de novo</em> genome assembly into the A, B, and D genomes.<a reveal-id="dsw037-B72" data-open="dsw037-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> This was not possible in quinoa because the donor diploid species of the B-genome has yet to be identified,<a reveal-id="dsw037-B23" data-open="dsw037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and no genome or transcriptome data of the A-genome species were available. Nevertheless, the fact that we could identify several putative homeologous genes suggests that we were successful in capturing the two subgenomes at least to a certain extent. Genome or transcriptome sequencing of <em>C. standleyanum</em> or other A-genome species, such as <em>C</em>. <em>pallidicaule</em>, will facilitate efforts to assign the predicted genes/scaffolds to the A- or B-genome in quinoa.</p>                            <h3 scrollto-destination=48005261 id="48005261" class="section-title" >3.5. Comparative analysis of the quinoa and other <em>Amaranthaceae</em> species gene sequences</h3>
<p>The entire data set of 226,647 genes was mapped onto the KEGG metabolic pathway and classified into the categories under ‘1. Metabolism’ and the numbers of mapped genes in each pathway are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. Briefly, 1,703 genes from quinoa were mapped onto 133 of the 158 categories of metabolic pathway in the KEGG database, whereas 1,748 genes of <em>S. oleracea</em>, 1,753 genes of <em>B. vulgaris</em>, and 1,009 genes of <em>A. hypochondriacus</em> were mapped onto 133, 135, and 124 pathways, respectively. Twenty-six pathways were only associated with genes in the quinoa genome. However, genes with similarity to the quinoa genes on these 26 pathways were identified in <em>S. oleracea</em>, <em>B. vulgaris</em>, or <em>A. hypochondriacus</em> by subsequent BLASTP searches against the NR database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). These would be false positive by the mapping analyses for KEGG metabolic pathways under the parameters with relatively low thresholds (<em>E</em>-value cut-off of 1E-4, length of coverage of 25%, and identity of 50%).</p><div class="block-child-p">The entire data set of 226,647 genes was also annotated by conducting similarity searches using the CD-HIT program (-c:0.4, -aL:0.4; <a reveal-id="dsw037-F3" data-open="dsw037-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). First, the predicted genes in the quinoa genome were clustered together with 30,564 genes in <em>A. hypochondriacus</em>, 27,421 in <em>B. vulgaris</em>, 21,702 in <em>S. oleracea</em>, and 35,386 in <em>A. thaliana</em>. The 3,393 clusters were common among five species, and 9,534, 4,543, 6,114, 5,805, and 9,996 genes belonged to the common clusters in <em>C. quinoa</em>, <em>A. hypochondriacus</em>, <em>B. vulgaris</em>, <em>S. oleracea</em>, and <em>A. thaliana</em>, respectively. Quinoa has 565 clusters of genes in a section shared only with <em>A. thaliana</em>, which is much less than the 1,224 genes present in sections shared with <em>S. oleracea</em>, 1,498 with <em>B. vulgaris</em>, and 3,215 with <em>A. hypochondriacus</em> in Amaranthaceae (<a reveal-id="dsw037-F3" data-open="dsw037-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Similar results were obtained in the comparative analysis of the filtered dataset of 62,512 genes of quinoa annotated by performing BLASTP searches against the NR database in NCBI (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&amp;Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). In the Amaranthaceae family, the genera <em>Chenopodium</em> and <em>Spinacia</em> belong to Chenopoideae, the genus <em>Beta</em> to Betoideae, and the genus <em>Amaranthus</em> to Amaranthoideae.<a reveal-id="dsw037-B73" data-open="dsw037-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> The genes, which were selected from a common section (central part in <a reveal-id="dsw037-F3" data-open="dsw037-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) on condition of a single copy being present in <em>A. hypochondriacus</em>, <em>B. vulgaris</em>, <em>S. oleracea</em>, and <em>A. thaliana</em> and of two copies being present in quinoa, were concatenated. Multiple alignment was carried out for the concatenated genes by ClustalW2,<a reveal-id="dsw037-B56" data-open="dsw037-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> and the dendrogram was constructed by the neighbour-joining method (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&amp;Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Quinoa and <em>S. oleracea</em> were phylogenetically closely related as expected, confirming that genes obtained from the QGDB were appropriate derivatives of those in the three other Amaranthaceae species. These basically single copy genes might be useful in phylogenetic analyses of core eudicots, as previously proposed for orthologous gene sets in <em>P. veris</em>.<a reveal-id="dsw037-B28" data-open="dsw037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><div data-id="dsw037-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f3p.png?Expires=2147483647&amp;Signature=favQrLokQu25YphFQD4Tmz~a4Lu9X1eIOCD7G8njzTne-g2UJcrHwpEBLjgfbjfnR10VvEHJrBFCZ2wQUPBQah21kIuR4Zk6GN3pOfBw82440-Ysp6PUT232aViPnc3~Fjcuj2tBdh2eqWBHTC0zaIHOi2bzAe1SiFBwjYLO-qoFwVx~Xf7Yo2C16ta~BDA1C2KghAAWYjacOhYRKW3uYDCrAepOaDFUDYQfjYjeh~EZUhzcGDxytRn0aKDxxvAwArng4k-KJEWGAbYv5a9IEF~1eisEwEsNSY-P~Qo2FdtvT-X5IfdEo3a-MGsEoqdKYb3Ql27ZZ6P1V4MkABoSQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cluster analysis of the predicted gene sequences. Predicted genes in Chenopodeum quinoa, Amaranthus hypochondriacus, Beta vulgaris, Spinacia oleracea, and Arabidopsis thaliana were clustered into gene families. The number in each section represents the number of clusters, and the numbers in parentheses in the central section represent the numbers of genes included from each species. The number below the species shows the total number of genes used as input for the CD-HIT (-c: 0.4, -aL: 0.4)." path-from-xml="dsw037f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005263/dsw037f3p.tif" path-from-xml="dsw037f3p.tif" target="_blank">View large</a><a section="48005263" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f3p.png?Expires=2147483647&amp;Signature=P3GkILySEcEyA8u5YgkjW-s7KX8kINy0ps35mq7WzHh4A7Ok9eUj3PzFTq5ZKabfTbqx191HXB9ZbFAJG33x6mNrUAGEvVB1udiErUP1iJz-2NzzRP1uNI3n4GBBC~McnlxC6AKfUl10vpm4jIpu0vo54gubRzDkcBWRb5zkvbuA1a3SLp8UtMqJAXKEGYWNroJ7K3UkIqW7fpOHbG7MlGIWWcN8x3~3ErPGkAl~vb4DfFIUp-KJlBL6G0LkLu62ovZDILzqoiElxWjBP-T-oLDEcL8aWUHDZVKDGY59CJtU4h0DUeXTuu-u-x9bYJAlbz~JvkSgSFUOw1rEFPYdcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005263&amp;ar=2647444&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw037f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cluster analysis of the predicted gene sequences. Predicted genes in <em>Chenopodeum quinoa</em>, <em>Amaranthus hypochondriacus</em>, <em>Beta vulgaris</em>, <em>Spinacia oleracea</em>, and <em>Arabidopsis thaliana</em> were clustered into gene families. The number in each section represents the number of clusters, and the numbers in parentheses in the central section represent the numbers of genes included from each species. The number below the species shows the total number of genes used as input for the CD-HIT (-c: 0.4, -aL: 0.4).</p></div></div><div content-id="dsw037-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f3p.png?Expires=2147483647&amp;Signature=favQrLokQu25YphFQD4Tmz~a4Lu9X1eIOCD7G8njzTne-g2UJcrHwpEBLjgfbjfnR10VvEHJrBFCZ2wQUPBQah21kIuR4Zk6GN3pOfBw82440-Ysp6PUT232aViPnc3~Fjcuj2tBdh2eqWBHTC0zaIHOi2bzAe1SiFBwjYLO-qoFwVx~Xf7Yo2C16ta~BDA1C2KghAAWYjacOhYRKW3uYDCrAepOaDFUDYQfjYjeh~EZUhzcGDxytRn0aKDxxvAwArng4k-KJEWGAbYv5a9IEF~1eisEwEsNSY-P~Qo2FdtvT-X5IfdEo3a-MGsEoqdKYb3Ql27ZZ6P1V4MkABoSQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cluster analysis of the predicted gene sequences. Predicted genes in Chenopodeum quinoa, Amaranthus hypochondriacus, Beta vulgaris, Spinacia oleracea, and Arabidopsis thaliana were clustered into gene families. The number in each section represents the number of clusters, and the numbers in parentheses in the central section represent the numbers of genes included from each species. The number below the species shows the total number of genes used as input for the CD-HIT (-c: 0.4, -aL: 0.4)." path-from-xml="dsw037f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005263/dsw037f3p.tif" path-from-xml="dsw037f3p.tif" target="_blank">View large</a><a section="48005263" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f3p.png?Expires=2147483647&amp;Signature=P3GkILySEcEyA8u5YgkjW-s7KX8kINy0ps35mq7WzHh4A7Ok9eUj3PzFTq5ZKabfTbqx191HXB9ZbFAJG33x6mNrUAGEvVB1udiErUP1iJz-2NzzRP1uNI3n4GBBC~McnlxC6AKfUl10vpm4jIpu0vo54gubRzDkcBWRb5zkvbuA1a3SLp8UtMqJAXKEGYWNroJ7K3UkIqW7fpOHbG7MlGIWWcN8x3~3ErPGkAl~vb4DfFIUp-KJlBL6G0LkLu62ovZDILzqoiElxWjBP-T-oLDEcL8aWUHDZVKDGY59CJtU4h0DUeXTuu-u-x9bYJAlbz~JvkSgSFUOw1rEFPYdcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005263&amp;ar=2647444&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw037f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cluster analysis of the predicted gene sequences. Predicted genes in <em>Chenopodeum quinoa</em>, <em>Amaranthus hypochondriacus</em>, <em>Beta vulgaris</em>, <em>Spinacia oleracea</em>, and <em>Arabidopsis thaliana</em> were clustered into gene families. The number in each section represents the number of clusters, and the numbers in parentheses in the central section represent the numbers of genes included from each species. The number below the species shows the total number of genes used as input for the CD-HIT (-c: 0.4, -aL: 0.4).</p></div></div></div><p>We also detected several expanded genes in quinoa from the results of CD-HIT analysis. Ten <em>lysine histidine transporter 2</em> (<em>LHT2</em>) like genes, which are related to amino acid uptake, were detected in the quinoa genome,<a reveal-id="dsw037-B74" data-open="dsw037-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a><sup>,</sup><a reveal-id="dsw037-B75" data-open="dsw037-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> but 2, 3, and 1 genes were found in <em>S. oleracea</em>, <em>B. vulgaris</em>, and <em>A. hypochondriacus</em>, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Some cytochrome P450 genes were also expanded in the quinoa genome. Two of these were annotated as cytochrome P450 71A1 and 76AD1, which play a role in the polyphenol- and betalain-biosynthesis pathway, respectively.<a reveal-id="dsw037-B76" data-open="dsw037-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a><sup>,</sup><a reveal-id="dsw037-B77" data-open="dsw037-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> Expansion and subsequent functional differentiation of these genes might be related to quinoa having a high protein content and abundant secondary metabolites.</p>                            <h3 scrollto-destination=48005265 id="48005265" class="section-title" >3.6. Genome designed for adaptation to hostile environments</h3>
                            <h4 scrollto-destination=48005266 id="48005266" class="section-title" data-legacy-id=SEC3.6.1>3.6.1. ABA signalling genes in quinoa</h4>
<p>Quinoa can withstand a variety of abiotic stresses,<a reveal-id="dsw037-B61" data-open="dsw037-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a><sup>,</sup><a reveal-id="dsw037-B78" data-open="dsw037-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> including high salinity (<a reveal-id="dsw037-F2" data-open="dsw037-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A–D). Transporter genes implicated in salt tolerance in quinoa, such as <em>CqSOS1A</em> and <em>CqSOS1B</em>, which are homologs of <em>Arabidopsis SOS1</em> involved in plasma membrane-localized Na<sup>+</sup>/H<sup>+ </sup>transport, have been analysed so far.<a reveal-id="dsw037-B71" data-open="dsw037-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Meanwhile, the phytohormone abscisic acid (ABA) is known to regulate abiotic stress tolerance.<a reveal-id="dsw037-B79" data-open="dsw037-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> Phylogenetic analyses revealed that gene families involved in ABA signalling were substantially expanded in the quinoa genome compared with other Amaranthaceae plant species. For example, the PYR/PYL/RCAR gene family, which encodes a bona fide ABA receptor, plays key roles in ABA signalling. Neighbour-joining analyses of AtPYLs and PYLs from Amaranthaceae species revealed three phylogenetic groups (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). In each group, we identified subgroups consisting of genes of Amaranthaceae species, including three in group I, two in group II, and three in group III (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Six of these subgroups contained two phylogenetically similar genes from <em>C. quinoa</em> and one from each of the other Amaranthaceae species included in the analysis. These pairs of <em>C. quinoa</em> genes are putative homoeologs, thought to have arisen from allotetraploidization. The gene family encoding SnRK2 protein kinases includes core positive regulators of ABA signalling, and similar phylogenetic results were obtained: four of the six subgroups identified contained two genes from <em>C. quinoa</em> and one from each of the other Amaranthaceae species examined (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The Ks values between putative homoeologs found in <em>PYLs</em> and <em>SnRK2</em> ranged from 0 to 0.088 and from 0 to 0.076, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). These findings are roughly consistent with the distribution of Ks values estimated from putative homoeologous gene sets obtained by CD-HIT analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Thus, our sequence results provide an important basis for studies aimed at determining the roles and precise numbers of candidate genes and pathways that regulate tolerance to abiotic stress in allotetraploid quinoa.</p>                            <h4 scrollto-destination=48005268 id="48005268" class="section-title" data-legacy-id=SEC3.6.2>3.6.2. <em>RDR1</em> homoeologs in quinoa</h4>
<p>Quinoa and <em>N. benthamiana</em> have been widely used as diagnostic hosts to identify virus species based on their own characteristic symptoms.<a reveal-id="dsw037-B62" data-open="dsw037-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> However, <em>N. benthamiana</em> is usually systemically infected with viruses, whereas quinoa is either locally or systemically infected, as shown in <a reveal-id="dsw037-F2" data-open="dsw037-F2" class="link link-reveal link-table xref-fig">Figure 2</a>E and F. Although mutations in <em>Rdr1</em>, which is involved in antiviral defence, have provided intriguing clues as to the mechanism by which a wide range of viruses can amplify well in <em>N. benthamiana</em>,<a reveal-id="dsw037-B80" data-open="dsw037-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><a reveal-id="dsw037-B81" data-open="dsw037-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> it remains unknown why a broad range of viruses can infect quinoa. We therefore analysed the <em>RDR1</em> genes in quinoa.</p><div class="block-child-p">We identified two <em>RDR1</em> orthologs in quinoa derived from a predicted gene (Cqu_c21957.1_g001.1) based on QGDB and a truncated gene on a scaffold (Cqu_c12210.1), and named these <em>CqRDR1A</em> and <em>CqRDR1B</em>, respectively. RT-PCR and Sanger sequencing revealed that <em>CqRDR1A</em> carries a complete ORF encoding 1,122 amino acids, and that <em>CqRDR1B</em> harbours a 2-bp deletion in the protein-coding region that induces a nonsense mutation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&amp;Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>), leading to the absence of a large portion of the RNA-dependent RNA polymerase (RdRP) domain that includes the active site (DxDGD).<a reveal-id="dsw037-B82" data-open="dsw037-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> Along with the observation that the RdRP domains are well conserved in angiosperms (<a reveal-id="dsw037-F4" data-open="dsw037-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), these findings clearly indicate that RdRP function is impaired in <em>CqRDR1B</em>. <div data-id="dsw037-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f4p.png?Expires=2147483647&amp;Signature=OnjbJ6qK6-xcl6pINjN7aN3ykr6zB09~I5PLIwf33PU5aUbKtdtb8uLp~Et6UxB6RyfT1samPxySJ3F1xVMg33tFOizuVorK6oBLmD1bFGkndXI9JXf9zxPK~ziNokElEAglJ61oSHjYhxdhlvDFABS-qSwbeszIuSVRMj99ViGe1BE3voJEoF3YSWBpfVWE7W4qYcKkpyUrnAJBQUaAjPDD9~jeDZyRhHwahPrTVkrK1-5iRvq6-Iia5nBHeIrHb2gRDmOyPnHZuKK0Q-5yUn03n-5IQXJQMSzZkn6S0O1uXM6TKYBZMr-uT1C1Q1dmlXXjWb8VMjqoCQbSpij88Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RDR1 of Amaranthaceae and other plant species. Amino acid sequences of Amaranthus hypochondriacus, Arabidopsis thaliana, and Oryza sativa were obtained from Phytozome 11,84 sequences of Beta vulgaris and Spinacia oleracea were from the Beta vulgaris Resource,85 and sequences of Nicotiana benthamiana (AAS78669.1) and N. tabacum were from NCBI. The nucleotide sequence of N. benthamiana (WA) was from Nicotiana benthamiana Genome and Transcriptome86 and translated to the putative amino acid sequence by EMBOSS Transeq.87 (A) Unrooted neighbor-joining tree based on amino acid sequences. The bootstrap values (500 replicates) not less than 50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. The root was assumed as the midpoint of the tree. (B) Schematic view of RDR1. The RNA-dependent RNA polymerase (RdRP) domain88 is indicated by a rounded rectangle and the active site of RDR182 is indicated by an arrowhead. The scale bar corresponds to 1,200 amino acids." path-from-xml="dsw037f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005270/dsw037f4p.tif" path-from-xml="dsw037f4p.tif" target="_blank">View large</a><a section="48005270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f4p.png?Expires=2147483647&amp;Signature=skgOhUjCurzGF3zIuzot3q0dP-omU6ioikfpvOhjYwpovADfSem46WdDqMIL-TMpgh9U2vzDd9wKB4Nlmf6i~DITbK1RL2sfbSmLHFdaVF4p2ufzUXVoFdMgZ6DMpNLlL0dPJmNwmNBmNUAQr0IKl7teFy6ZeprVaX5WML8n1N0TfTrnqFYhN1DBD1SS1s1TK8-IoAheNevRkAjHtO4KL6jJ5eZtGbLgLs~Zscz9yCcHX7eSrg8cKZBW1-VpYIIQkaILtaLME5oZqYN66BLHvuzPXuD5QFJPY6AR9H26OGcnWsui8PaxkyAFseIFmHgxe3VE~fKEj54vagPEMNSUdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005270&amp;ar=2647444&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw037f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RDR1 of Amaranthaceae and other plant species. Amino acid sequences of <em>Amaranthus hypochondriacus</em>, <em>Arabidopsis thaliana</em>, and <em>Oryza sativa</em> were obtained from Phytozome 11,<a reveal-id="dsw037-B84" data-open="dsw037-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> sequences of <em>Beta vulgaris</em> and <em>Spinacia oleracea</em> were from the Beta vulgaris Resource,<a reveal-id="dsw037-B85" data-open="dsw037-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> and sequences of <em>Nicotiana benthamiana</em> (AAS78669.1) and <em>N. tabacum</em> were from NCBI. The nucleotide sequence of <em>N. benthamiana</em> (WA) was from Nicotiana benthamiana Genome and Transcriptome<a reveal-id="dsw037-B86" data-open="dsw037-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> and translated to the putative amino acid sequence by EMBOSS Transeq.<a reveal-id="dsw037-B87" data-open="dsw037-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> (A) Unrooted neighbor-joining tree based on amino acid sequences. The bootstrap values (500 replicates) not less than 50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. The root was assumed as the midpoint of the tree. (B) Schematic view of RDR1. The RNA-dependent RNA polymerase (RdRP) domain<a reveal-id="dsw037-B88" data-open="dsw037-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> is indicated by a rounded rectangle and the active site of RDR1<a reveal-id="dsw037-B82" data-open="dsw037-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> is indicated by an arrowhead. The scale bar corresponds to 1,200 amino acids.</p></div></div><div content-id="dsw037-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/m_dsw037f4p.png?Expires=2147483647&amp;Signature=OnjbJ6qK6-xcl6pINjN7aN3ykr6zB09~I5PLIwf33PU5aUbKtdtb8uLp~Et6UxB6RyfT1samPxySJ3F1xVMg33tFOizuVorK6oBLmD1bFGkndXI9JXf9zxPK~ziNokElEAglJ61oSHjYhxdhlvDFABS-qSwbeszIuSVRMj99ViGe1BE3voJEoF3YSWBpfVWE7W4qYcKkpyUrnAJBQUaAjPDD9~jeDZyRhHwahPrTVkrK1-5iRvq6-Iia5nBHeIrHb2gRDmOyPnHZuKK0Q-5yUn03n-5IQXJQMSzZkn6S0O1uXM6TKYBZMr-uT1C1Q1dmlXXjWb8VMjqoCQbSpij88Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RDR1 of Amaranthaceae and other plant species. Amino acid sequences of Amaranthus hypochondriacus, Arabidopsis thaliana, and Oryza sativa were obtained from Phytozome 11,84 sequences of Beta vulgaris and Spinacia oleracea were from the Beta vulgaris Resource,85 and sequences of Nicotiana benthamiana (AAS78669.1) and N. tabacum were from NCBI. The nucleotide sequence of N. benthamiana (WA) was from Nicotiana benthamiana Genome and Transcriptome86 and translated to the putative amino acid sequence by EMBOSS Transeq.87 (A) Unrooted neighbor-joining tree based on amino acid sequences. The bootstrap values (500 replicates) not less than 50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. The root was assumed as the midpoint of the tree. (B) Schematic view of RDR1. The RNA-dependent RNA polymerase (RdRP) domain88 is indicated by a rounded rectangle and the active site of RDR182 is indicated by an arrowhead. The scale bar corresponds to 1,200 amino acids." path-from-xml="dsw037f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48005270/dsw037f4p.tif" path-from-xml="dsw037f4p.tif" target="_blank">View large</a><a section="48005270" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037f4p.png?Expires=2147483647&amp;Signature=skgOhUjCurzGF3zIuzot3q0dP-omU6ioikfpvOhjYwpovADfSem46WdDqMIL-TMpgh9U2vzDd9wKB4Nlmf6i~DITbK1RL2sfbSmLHFdaVF4p2ufzUXVoFdMgZ6DMpNLlL0dPJmNwmNBmNUAQr0IKl7teFy6ZeprVaX5WML8n1N0TfTrnqFYhN1DBD1SS1s1TK8-IoAheNevRkAjHtO4KL6jJ5eZtGbLgLs~Zscz9yCcHX7eSrg8cKZBW1-VpYIIQkaILtaLME5oZqYN66BLHvuzPXuD5QFJPY6AR9H26OGcnWsui8PaxkyAFseIFmHgxe3VE~fKEj54vagPEMNSUdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48005270&amp;ar=2647444&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw037f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RDR1 of Amaranthaceae and other plant species. Amino acid sequences of <em>Amaranthus hypochondriacus</em>, <em>Arabidopsis thaliana</em>, and <em>Oryza sativa</em> were obtained from Phytozome 11,<a reveal-id="dsw037-B84" data-open="dsw037-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> sequences of <em>Beta vulgaris</em> and <em>Spinacia oleracea</em> were from the Beta vulgaris Resource,<a reveal-id="dsw037-B85" data-open="dsw037-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> and sequences of <em>Nicotiana benthamiana</em> (AAS78669.1) and <em>N. tabacum</em> were from NCBI. The nucleotide sequence of <em>N. benthamiana</em> (WA) was from Nicotiana benthamiana Genome and Transcriptome<a reveal-id="dsw037-B86" data-open="dsw037-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> and translated to the putative amino acid sequence by EMBOSS Transeq.<a reveal-id="dsw037-B87" data-open="dsw037-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> (A) Unrooted neighbor-joining tree based on amino acid sequences. The bootstrap values (500 replicates) not less than 50 are shown next to the branches. The scale bar corresponds to 0.05 substitutions per site. The root was assumed as the midpoint of the tree. (B) Schematic view of RDR1. The RNA-dependent RNA polymerase (RdRP) domain<a reveal-id="dsw037-B88" data-open="dsw037-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> is indicated by a rounded rectangle and the active site of RDR1<a reveal-id="dsw037-B82" data-open="dsw037-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> is indicated by an arrowhead. The scale bar corresponds to 1,200 amino acids.</p></div></div></div><p>Our phylogenetic analysis revealed that <em>CqRDR1</em> duplication occurred after the speciation between <em>Chenopodium</em> and <em>Spinacia</em> (<a reveal-id="dsw037-F4" data-open="dsw037-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Based on a co-dominant marker that distinguishes between two <em>CqRDR1</em> genes, we confirmed that all 20 tested Kd progeny retained both genes at different loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Comparison of <em>RDR1</em> sequences between quinoa and <em>C. pallidicaule</em> indicated that <em>CqRDR1A</em> was derived from the <em>Chenopodium</em> A-genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). In addition, no PCR products were amplified from four <em>C</em>. <em>pallidicaule</em> lines using a B-genome-specific primer pair (data not shown), suggesting that <em>CqRDR1A</em> and <em>CqRDR1B</em> are homoeologs. Although further analyses will be required to reveal the mechanism supporting the susceptibility to a wide variety of viruses, these findings provide insight into the mechanism underlying the response to virus infections in quinoa.</p>                            <h2 scrollto-destination=48005272 id="48005272" class="section-title" data-legacy-id=SEC4>4. Conclusion and future perspectives</h2>
<p>The year 2013 was declared the International Year of Quinoa by FAO to heighten public awareness of the nutritional benefits of this durable plant, as part of a sustainable food production effort aimed at food security and nutrition.<a reveal-id="dsw037-B17" data-open="dsw037-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> However, molecular analysis of quinoa is limited by its genetic heterogeneity and its genome complexity, due to allotetraploidy. To overcome the former limitation, we established the inbred accession Kd that possesses standard quinoa properties in response to abiotic and biotic stresses. This provides a useful experimental standard material for basic and applied studies of quinoa, which have recently focused on its exceptional nutritional value, tolerance to unfavourable environments, and susceptibility to a broad range of viruses.</p><p>The allotetraploid nature of quinoa makes it difficult to construct a genome assembly using high-throughput sequencing. To overcome this limitation, we employed a combination of Illumina HiSeq 2500 and PacBio RS II NGSs, which produce short (approximately 200 bp) and long (mean length of 10 kbp) reads, respectively. The long reads were used for gap-closing and for further scaffolding of the assembly obtained from the Illumina short reads to improve the genome assembly, and resulted in a 76% reduction in gaps and a 1.6-fold increase in the N50 length. In our trial usage of the database QGDB, ABA signalling genes and <em>RDR1</em> were detected, and gene expansion probably caused by tetraploidization was observed in all cases. It should be noted that we were also able to identify putative homoeologous genes in both the clustering and Ks distribution analyses. Thus, although our assembly does not cover the entire genome and still contains large gaps, these findings have clearly demonstrated that the genome database created from the assembly is a useful basis for identifying agronomically important genes. Nevertheless, the length of scaffolds of our assembly could be increased by applying new technologies, such as high-throughput optical mapping,<a reveal-id="dsw037-B83" data-open="dsw037-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> which provides long-range genomic information through constructing restriction maps. In addition, a high-density linkage map generated using NGS-based GBS markers with Kd-based recombinant inbred lines (RILs) will also be required to anchor the improved scaffolds to each chromosome to generate pseudomolecules of the 18 chromosomes in quinoa.</p>                            <h2 scrollto-destination=48005275 id="48005275" class="section-title" data-legacy-id=SEC5>5. Data availability</h2>
<p>The Illumina and PacBio reads used in this study are available from DDBJ/EMBL/Genbank under the accession numbers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1 and S5</a></span>. The scaffold sequences are available under the accession numbers BDCQ01000001-BDCQ01024845 (24,845 entries). The draft genome sequence Cqu_r1.0, CDS and protein sequences, and annotation file (gff file) are also available from the free-access Quinoa Genome DataBase (QGDB; <a class="link link-uri" href="http://quinoa.kazusa.or.jp" target="_blank">http://quinoa.kazusa.or.jp</a>). DNA sequences of <em>RDR1s</em> are deposited in the DDBJ/EMBL/Genbank database under the accession numbers LC146405, LC146407, and LC149497.</p>                            <h2 scrollto-destination=48005277 id="48005277" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank S. Tabata (Kazusa) and A. Nakagawa (Ishikawa Pref. Univ.) for valuable suggestions, K. Amano, E. Ohgawara and E. Kishi (JIRCAS) for their excellent technical support, M. Fujita (RIKEN) for helpful comments and skilful graphics assistance, and K. L. Farquharson for language-editing support of the manuscript. We are deeply grateful to I. Furusawa (prof. emeritus of Kyoto Univ.), T. Okuno (Ryukoku Univ.; prof. emeritus of Kyoto Univ.), H. Nagano (Kyoto Univ.), and the other past members at the Lab. of Plant Pathology, Kyoto Univ., involved in the quinoa growth for more than 20 years. We apologize to all authors whose work could not be cited here due to space constraints.</p>                            <h3 scrollto-destination=48005279 id="48005279" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=48005281 id="48005281" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=48005283 id="48005283" class="section-title" >Funding</h3>
<p>This research was supported by cooperative research funds from Actree Co. Our work at JIRCAS was partly supported by the Ministry of Agriculture, Forestry and Fisheries (MAFF) of Japan.</p>                            <h2 scrollto-destination=48005285 id="48005285" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw037-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maughan</div> <div class="given-names">P.J.</div></div><div class="name string-name"><div class="surname">Bonifacio</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Jellen</div> <div class="given-names">E.N.</div></div></div>, et al.  . <div class="year">2004</div>, 
<div class="article-title">A genetic linkage map of quinoa (<em>Chenopodium quinoa</em>) based on AFLP, RAPD, and SSR markers</div>. <div class="source">Theor. Appl. Genet</div> ., <div class="volume">109</div>, <div class="fpage">1188</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20quinoa%20(Chenopodium%20quinoa)%20based%20on%20AFLP%2C%20RAPD%2C%20and%20SSR%20markers&amp;author=P.J.%20Maughan&amp;author=A.%20Bonifacio&amp;author=E.N.%20Jellen&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet&amp;volume=109&amp;pages=1188-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1730-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1730-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1730-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15309300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zurita-Silva</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Fuentes</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Zamora</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.-E.</div></div><div class="name string-name"><div class="surname">Schwember</div> <div class="given-names">A.R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Breeding quinoa (<em>Chenopodium quinoa</em> Willd.): potential and perspectives</div>. <div class="source">Mol. Breed</div> ., <div class="volume">34</div>, <div class="fpage">13</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Breeding%20quinoa%20(Chenopodium%20quinoa%20Willd.)%3A%20potential%20and%20perspectives&amp;author=A.%20Zurita-Silva&amp;author=F.%20Fuentes&amp;author=P.%20Zamora&amp;author=S.-E.%20Jacobsen&amp;author=A.R.%20Schwember&amp;publication_year=2014&amp;journal=Mol.%20Breed&amp;volume=34&amp;pages=13-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-014-0023-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-014-0023-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-014-0023-5"> </span></div> </div></div></div></div></div><div content-id="dsw037-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Risi</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Galwey</div> <div class="given-names">N.W.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">The pattern of genetic diversity in the Andean grain crop quinoa (<em>Chenopodium quinoa</em> Willd.). I. Associations between characteristics</div>. <div class="source">Euphytica</div> , <div class="volume">41</div>, <div class="fpage">147</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pattern%20of%20genetic%20diversity%20in%20the%20Andean%20grain%20crop%20quinoa%20(Chenopodium%20quinoa%20Willd.).%20I.%20Associations%20between%20characteristics&amp;author=J.%20Risi&amp;author=N.W.%20Galwey&amp;publication_year=1989&amp;journal=Euphytica&amp;volume=41&amp;pages=147-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00022424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00022424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00022424"> </span></div> </div></div></div></div></div><div content-id="dsw037-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fuentes</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Bhargava</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Morphological analysis of quinoa germplasm grown under lowland desert conditions</div>. <div class="source">J. Agron. Crop Sci</div> ., <div class="volume">197</div>, <div class="fpage">124</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Morphological%20analysis%20of%20quinoa%20germplasm%20grown%20under%20lowland%20desert%20conditions&amp;author=F.%20Fuentes&amp;author=A.%20Bhargava&amp;publication_year=2011&amp;journal=J.%20Agron.%20Crop%20Sci&amp;volume=197&amp;pages=124-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jac.2011.197.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjac.2011.197.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjac.2011.197.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsw037-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">González</div> <div class="given-names">J.A.</div></div><div class="name string-name"><div class="surname">Bruno</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Valoy</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Prado</div> <div class="given-names">F.E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genotypic variation of gas exchange parameters and leaf stable carbon and nitrogen isotopes in ten quinoa cultivars grown under drought</div>. <div class="source">J. Agron. Crop Sci</div> ., <div class="volume">197</div>, <div class="fpage">81</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotypic%20variation%20of%20gas%20exchange%20parameters%20and%20leaf%20stable%20carbon%20and%20nitrogen%20isotopes%20in%20ten%20quinoa%20cultivars%20grown%20under%20drought&amp;author=J.A.%20Gonz%C3%A1lez&amp;author=M.%20Bruno&amp;author=M.%20Valoy&amp;author=F.E.%20Prado&amp;publication_year=2011&amp;journal=J.%20Agron.%20Crop%20Sci&amp;volume=197&amp;pages=81-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jac.2011.197.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjac.2011.197.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjac.2011.197.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsw037-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">E.A.</div></div><div class="name string-name"><div class="surname">Veas</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Jorquera</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">San Martín</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Jara</div> <div class="given-names">P.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Re-introduction of quínoa into arid Chile: cultivation of two lowland races under extremely low irrigation</div>. <div class="source">J. Agron. Crop Sci</div> ., <div class="volume">195</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-introduction%20of%20qu%C3%ADnoa%20into%20arid%20Chile%3A%20cultivation%20of%20two%20lowland%20races%20under%20extremely%20low%20irrigation&amp;author=E.A.%20Mart%C3%ADnez&amp;author=E.%20Veas&amp;author=C.%20Jorquera&amp;author=R.%20San%20Mart%C3%ADn&amp;author=P.%20Jara&amp;publication_year=2009&amp;journal=J.%20Agron.%20Crop%20Sci&amp;volume=195&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jac.2009.195.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjac.2009.195.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjac.2009.195.issue-1"> </span></div> </div></div></div></div></div><div content-id="dsw037-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Raes</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.-E.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Evapotranspiration analysis and irrigation requirements of quinoa (<em>Chenopodium quinoa</em>) in the Bolivian highlands</div>. <div class="source">Agric. Water Manage</div> ., <div class="volume">60</div>, <div class="fpage">119</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evapotranspiration%20analysis%20and%20irrigation%20requirements%20of%20quinoa%20(Chenopodium%20quinoa)%20in%20the%20Bolivian%20highlands&amp;author=M.%20Garcia&amp;author=D.%20Raes&amp;author=S.-E.%20Jacobsen&amp;publication_year=2003&amp;journal=Agric.%20Water%20Manage&amp;volume=60&amp;pages=119-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-3774(02)00162-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-3774(02)00162-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-3774(02)00162-2"> </span></div> </div></div></div></div></div><div content-id="dsw037-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vacher</div> <div class="given-names">J.J.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Responses of two main Andean crops, quinoa (<em>Chenopodium quinoa</em> Willd) and papa amarga (<em>Solanum juzepczukii</em> Buk.) to drought on the Bolivian Altiplano: significance of local adaptation</div>. <div class="source">Agric. Ecosyst. Environ</div> ., <div class="volume">68</div>, <div class="fpage">99</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Responses%20of%20two%20main%20Andean%20crops%2C%20quinoa%20(Chenopodium%20quinoa%20Willd)%20and%20papa%20amarga%20(Solanum%20juzepczukii%20Buk.)%20to%20drought%20on%20the%20Bolivian%20Altiplano%3A%20significance%20of%20local%20adaptation&amp;author=J.J.%20Vacher&amp;publication_year=1998&amp;journal=Agric.%20Ecosyst.%20Environ&amp;volume=68&amp;pages=99-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-8809(97)00140-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-8809(97)00140-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-8809(97)00140-0"> </span></div> </div></div></div></div></div><div content-id="dsw037-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Razzaghi</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Ahmadi</div> <div class="given-names">S.H.</div></div><div class="name string-name"><div class="surname">Adolf</div> <div class="given-names">V.I.</div></div><div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">C.R.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div><div class="name string-name"><div class="surname">Andersen</div> <div class="given-names">M.N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Water relations and transpiration of quinoa (<em>Chenopodium quinoa</em> Willd.) under salinity and soil drying</div>. <div class="source">J. Agron. Crop Sci</div> ., <div class="volume">197</div>, <div class="fpage">348</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Water%20relations%20and%20transpiration%20of%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20under%20salinity%20and%20soil%20drying&amp;author=F.%20Razzaghi&amp;author=S.H.%20Ahmadi&amp;author=V.I.%20Adolf&amp;author=C.R.%20Jensen&amp;author=S.E.%20Jacobsen&amp;author=M.N.%20Andersen&amp;publication_year=2011&amp;journal=J.%20Agron.%20Crop%20Sci&amp;volume=197&amp;pages=348-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jac.2011.197.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjac.2011.197.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjac.2011.197.issue-5"> </span></div> </div></div></div></div></div><div content-id="dsw037-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div><div class="name string-name"><div class="surname">Mujica</div> <div class="given-names">A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Quinoa: an alternative crop for saline soils</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">54</div>, <div class="fpage">i25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quinoa%3A%20an%20alternative%20crop%20for%20saline%20soils&amp;author=S.E.%20Jacobsen&amp;author=A.%20Mujica&amp;publication_year=2003&amp;journal=J.%20Exp.%20Bot&amp;volume=54&amp;pages=i25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erg041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferg041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferg041"> </span></div> </div></div></div></div></div><div content-id="dsw037-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hariadi</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Marandon</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div><div class="name string-name"><div class="surname">Shabala</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Ionic and osmotic relations in quinoa (<em>Chenopodium quinoa</em> Willd.) plants grown at various salinity levels</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">62</div>, <div class="fpage">185</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ionic%20and%20osmotic%20relations%20in%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20plants%20grown%20at%20various%20salinity%20levels&amp;author=Y.%20Hariadi&amp;author=K.%20Marandon&amp;author=Y.%20Tian&amp;author=S.E.%20Jacobsen&amp;author=S.%20Shabala&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot&amp;volume=62&amp;pages=185-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erq257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferq257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferq257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20732880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div><div class="name string-name"><div class="surname">Monteros</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Christiansen</div> <div class="given-names">J.L.</div></div><div class="name string-name"><div class="surname">Bravo</div> <div class="given-names">L.A.</div></div><div class="name string-name"><div class="surname">Corcuera</div> <div class="given-names">L.J.</div></div><div class="name string-name"><div class="surname">Mujica</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Plant responses of quinoa (<em>Chenopodium quinoa</em> Willd.) to frost at various phenological stages</div>. <div class="source">Eur. J. Agron</div> ., <div class="volume">22</div>, <div class="fpage">131</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20responses%20of%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20to%20frost%20at%20various%20phenological%20stages&amp;author=S.E.%20Jacobsen&amp;author=C.%20Monteros&amp;author=J.L.%20Christiansen&amp;author=L.A.%20Bravo&amp;author=L.J.%20Corcuera&amp;author=A.%20Mujica&amp;publication_year=2005&amp;journal=Eur.%20J.%20Agron&amp;volume=22&amp;pages=131-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.eja.2004.01.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.eja.2004.01.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.eja.2004.01.003"> </span></div> </div></div></div></div></div><div content-id="dsw037-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vega-Galvez</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Miranda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Vergara</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Uribe</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Puente</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Martinez</div> <div class="given-names">E.A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Nutrition facts and functional potential of quinoa (<em>Chenopodium quinoa</em> willd.), an ancient Andean grain: a review</div>. <div class="source">J. Sci. Food Agric</div> ., <div class="volume">90</div>, <div class="fpage">2541</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nutrition%20facts%20and%20functional%20potential%20of%20quinoa%20(Chenopodium%20quinoa%20willd.)%2C%20an%20ancient%20Andean%20grain%3A%20a%20review&amp;author=A.%20Vega-Galvez&amp;author=M.%20Miranda&amp;author=J.%20Vergara&amp;author=E.%20Uribe&amp;author=L.%20Puente&amp;author=E.A.%20Martinez&amp;publication_year=2010&amp;journal=J.%20Sci.%20Food%20Agric&amp;volume=90&amp;pages=2541-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jsfa.v90:15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjsfa.v90:15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjsfa.v90:15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20814881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maradini Filho</div> <div class="given-names">A.M.</div></div><div class="name string-name"><div class="surname">Pirozi</div> <div class="given-names">M.R.</div></div><div class="name string-name"><div class="surname">Da Silva Borges</div> <div class="given-names">J.T.</div></div><div class="name string-name"><div class="surname">Pinheiro Sant'Ana</div> <div class="given-names">H.M.</div></div><div class="name string-name"><div class="surname">Paes Chaves</div> <div class="given-names">J.B.</div></div><div class="name string-name"><div class="surname">Dos Reis Coimbra</div> <div class="given-names">J.S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Quinoa: nutritional, functional and antinutritional aspects</div>. <div class="source">Crit. Rev. Food Sci. Nutr</div> ., doi: 10.1080/10408398.2014.1001811.</div></div></div></div><div content-id="dsw037-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nowak</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Charrondiere</div> <div class="given-names">U.R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Assessment of the nutritional composition of quinoa (<em>Chenopodium quinoa</em> Willd.)</div>. <div class="source">Food Chem</div> ., <div class="volume">193</div>, <div class="fpage">47</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20the%20nutritional%20composition%20of%20quinoa%20(Chenopodium%20quinoa%20Willd.)&amp;author=V.%20Nowak&amp;author=J.%20Du&amp;author=U.R.%20Charrondiere&amp;publication_year=2016&amp;journal=Food%20Chem&amp;volume=193&amp;pages=47-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodchem.2015.02.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodchem.2015.02.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodchem.2015.02.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26433286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mota</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Santos</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Mauro</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Protein content and amino acids profile of pseudocereals</div>. <div class="source">Food Chem</div> ., <div class="volume">193</div>, <div class="fpage">55</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20content%20and%20amino%20acids%20profile%20of%20pseudocereals&amp;author=C.%20Mota&amp;author=M.%20Santos&amp;author=R.%20Mauro&amp;publication_year=2016&amp;journal=Food%20Chem&amp;volume=193&amp;pages=55-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodchem.2014.11.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodchem.2014.11.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodchem.2014.11.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26433287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bazile</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bertero</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Nieto</div> <div class="given-names">C.</div></div></div> <div class="year">2015</div>, <div class="source">State of the art report on quinoa around the world in 2013. Food and Agriculture Organization of the United Nations (FAO) &amp; Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD)</div> : 
<div class="publisher-loc">Rome</div>.</div></div></div></div><div content-id="dsw037-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schlick</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Bubenheim</div> <div class="given-names">D.L.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Quinoa: an emerging “new” crop with potential for celss</div>. <div class="source">NASA Tech. Pap</div> ., <div class="volume">3422</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw037-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">González</div> <div class="given-names">J.A.</div></div><div class="name string-name"><div class="surname">Eisa</div> <div class="given-names">S.S.S.</div></div><div class="name string-name"><div class="surname">Hussin</div> <div class="given-names">S.A.E.S.</div></div><div class="name string-name"><div class="surname">Prado</div> <div class="given-names">F.E.</div></div></div> <div class="year">2015</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">K.S.</div> <div class="given-names">Murphy</div></div><div class="name string-name"><div class="surname">J.</div> <div class="given-names">Matanguihan</div></div></div> (eds), <div class="source">Quinoa: improvement and sustainable production</div> . 
<div class="publisher-name">John Wiley &amp; Sons, Inc</div>., 
<div class="publisher-loc">New Jersey</div>, pp. <div class="fpage">1</div>–<div class="lpage">18</div>.</div></div></div></div><div content-id="dsw037-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Christensen</div> <div class="given-names">S.A.</div></div><div class="name string-name"><div class="surname">Pratt</div> <div class="given-names">D.B.</div></div><div class="name string-name"><div class="surname">Pratt</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Assessment of genetic diversity in the USDA and CIP-FAO international nursery collections of quinoa (<em>Chenopodium quinoa</em> Willd.) using microsatellite markers</div>. <div class="source">Plant Genet. Resour</div> ., <div class="volume">5</div>, <div class="fpage">82</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20genetic%20diversity%20in%20the%20USDA%20and%20CIP-FAO%20international%20nursery%20collections%20of%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20using%20microsatellite%20markers&amp;author=S.A.%20Christensen&amp;author=D.B.%20Pratt&amp;author=C.%20Pratt&amp;publication_year=2007&amp;journal=Plant%20Genet.%20Resour&amp;volume=5&amp;pages=82-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1479262107672293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1479262107672293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1479262107672293"> </span></div> </div></div></div></div></div><div content-id="dsw037-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palomino</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Hernández</div> <div class="given-names">L.T.</div></div><div class="name string-name"><div class="surname">de la Cruz Torres</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Nuclear genome size and chromosome analysis in <em>Chenopodium quinoa</em> and <em>C</em>. <em>berlandieri</em> subsp. <em>nuttalliae</em></div>. <div class="source">Euphytica</div> , <div class="volume">164</div>, <div class="fpage">221</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20genome%20size%20and%20chromosome%20analysis%20in%20Chenopodium%20quinoa%20and%20C.%20berlandieri%20subsp.%20nuttalliae&amp;author=G.%20Palomino&amp;author=L.T.%20Hern%C3%A1ndez&amp;author=E.%20de%20la%20Cruz%20Torres&amp;publication_year=2008&amp;journal=Euphytica&amp;volume=164&amp;pages=221-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-008-9711-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-008-9711-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-008-9711-8"> </span></div> </div></div></div></div></div><div content-id="dsw037-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yangquanwei</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Neethirajan</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Karunakaran</div> <div class="given-names">C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Cytogenetic analysis of quinoa chromosomes using nanoscale imaging and spectroscopy techniques</div>. <div class="source">Nanoscale Res Lett</div> , <div class="volume">8</div>, <div class="fpage">463</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetic%20analysis%20of%20quinoa%20chromosomes%20using%20nanoscale%20imaging%20and%20spectroscopy%20techniques&amp;author=Z.%20Yangquanwei&amp;author=S.%20Neethirajan&amp;author=C.%20Karunakaran&amp;publication_year=2013&amp;journal=Nanoscale%20Res%20Lett&amp;volume=8&amp;pages=463" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1556-276X-8-463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1556-276X-8-463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1556-276X-8-463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24191931" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walsh</div> <div class="given-names">B.M.</div></div><div class="name string-name"><div class="surname">Adhikary</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Maughan</div> <div class="given-names">P.J.</div></div><div class="name string-name"><div class="surname">Emshwiller</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Jellen</div> <div class="given-names">E.N.</div></div></div> <div class="year">2015</div>, 
<div class="article-title"><em>Chenopodium</em> polyploidy inferences from <em>Salt Overly Sensitive 1</em> (<em>SOS1</em>) data</div>. <div class="source">Am. J. Bot</div> ., <div class="volume">102</div>, <div class="fpage">533</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chenopodium%20polyploidy%20inferences%20from%20Salt%20Overly%20Sensitive%201%20(SOS1)%20data&amp;author=B.M.%20Walsh&amp;author=D.%20Adhikary&amp;author=P.J.%20Maughan&amp;author=E.%20Emshwiller&amp;author=E.N.%20Jellen&amp;publication_year=2015&amp;journal=Am.%20J.%20Bot&amp;volume=102&amp;pages=533-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1400344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1400344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1400344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25878087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maughan</div> <div class="given-names">P.J.</div></div><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">S.M.</div></div><div class="name string-name"><div class="surname">Rojas-Beltrán</div> <div class="given-names">J.A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Single nucleotide polymorphism identification, characterization, and linkage mapping in quinoa</div>. <div class="source">Plant Genome</div> , <div class="volume">5</div>, <div class="fpage">114</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20identification%2C%20characterization%2C%20and%20linkage%20mapping%20in%20quinoa&amp;author=P.J.%20Maughan&amp;author=S.M.%20Smith&amp;author=J.A.%20Rojas-Beltr%C3%A1n&amp;publication_year=2012&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=114-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2012.06.0011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2012.06.0011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2012.06.0011"> </span></div> </div></div></div></div></div><div content-id="dsw037-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elshire</div> <div class="given-names">R.J.</div></div><div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</div>. <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.J.%20Elshire&amp;author=J.C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carneiro</div> <div class="given-names">A.R.</div></div><div class="name string-name"><div class="surname">Ramos</div> <div class="given-names">R.T.</div></div><div class="name string-name"><div class="surname">Barbosa</div> <div class="given-names">H.P.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Quality of prokaryote genome assembly: indispensable issues of factors affecting prokaryote genome assembly quality</div>. <div class="source">Gene</div> , <div class="volume">505</div>, <div class="fpage">365</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20of%20prokaryote%20genome%20assembly%3A%20indispensable%20issues%20of%20factors%20affecting%20prokaryote%20genome%20assembly%20quality&amp;author=A.R.%20Carneiro&amp;author=R.T.%20Ramos&amp;author=H.P.%20Barbosa&amp;publication_year=2012&amp;journal=Gene&amp;volume=505&amp;pages=365-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.06.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.06.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.06.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22721771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">English</div> <div class="given-names">A.C.</div></div><div class="name string-name"><div class="surname">Richards</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology</div>. <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e47768</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mind%20the%20gap%3A%20upgrading%20genomes%20with%20Pacific%20Biosciences%20RS%20long-read%20sequencing%20technology&amp;author=A.C.%20English&amp;author=S.%20Richards&amp;author=Y.%20Han&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e47768" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0047768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0047768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0047768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23185243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nowak</div> <div class="given-names">M.D.</div></div><div class="name string-name"><div class="surname">Russo</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Schlapbach</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Huu</div> <div class="given-names">C.N.</div></div><div class="name string-name"><div class="surname">Lenhard</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Conti</div> <div class="given-names">E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The draft genome of <em>Primula veris</em> yields insights into the molecular basis of heterostyly</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Primula%20veris%20yields%20insights%20into%20the%20molecular%20basis%20of%20heterostyly&amp;author=M.D.%20Nowak&amp;author=G.%20Russo&amp;author=R.%20Schlapbach&amp;author=C.N.%20Huu&amp;author=M.%20Lenhard&amp;author=E.%20Conti&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0567-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0567-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0567-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25651398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagano</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Okuno</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Mise</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Furusawa</div> <div class="given-names">I.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Deletion of the C-terminal 33 amino acids of cucumber mosaic virus movement protein enables a chimeric brome mosaic virus to move from cell to cell</div>. <div class="source">J. Virol</div> ., <div class="volume">71</div>, <div class="fpage">2270</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion%20of%20the%20C-terminal%2033%20amino%20acids%20of%20cucumber%20mosaic%20virus%20movement%20protein%20enables%20a%20chimeric%20brome%20mosaic%20virus%20to%20move%20from%20cell%20to%20cell&amp;author=H.%20Nagano&amp;author=T.%20Okuno&amp;author=K.%20Mise&amp;author=I.%20Furusawa&amp;publication_year=1997&amp;journal=J.%20Virol&amp;volume=71&amp;pages=2270-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9032362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kroner</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Ahlquist</div> <div class="given-names">P.</div></div></div> <div class="year">1992</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">S.J.</div> <div class="given-names">Gurr</div></div><div class="name string-name"><div class="surname">M.J.</div> <div class="given-names">McPherson</div></div><div class="name string-name"><div class="surname">D.J.</div> <div class="given-names">Bowles</div></div></div> (eds), <div class="source">Molecular plant pathology: a practical approach</div> , <div class="volume">vol. I</div>. 
<div class="publisher-name">IRL Press at Oxford University Press</div>, 
<div class="publisher-loc">Oxford</div>, <div class="fpage">23</div>–<div class="lpage">34</div>.</div></div></div></div><div content-id="dsw037-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mise</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Allison</div> <div class="given-names">R.F.</div></div><div class="name string-name"><div class="surname">Janda</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ahlquist</div> <div class="given-names">P.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Bromovirus movement protein genes play a crucial role in host specificity</div>. <div class="source">J. Virol</div> ., <div class="volume">67</div>, <div class="fpage">2815</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bromovirus%20movement%20protein%20genes%20play%20a%20crucial%20role%20in%20host%20specificity&amp;author=K.%20Mise&amp;author=R.F.%20Allison&amp;author=M.%20Janda&amp;author=P.%20Ahlquist&amp;publication_year=1993&amp;journal=J.%20Virol&amp;volume=67&amp;pages=2815-23" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7682628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">K.J.</div></div><div class="name string-name"><div class="surname">Gilman</div> <div class="given-names">M.</div></div></div> <div class="year">1995</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">F.M.</div> <div class="given-names">Ausubel</div></div><div class="name string-name"><div class="surname">R.</div> <div class="given-names">Brent</div></div><div class="name string-name"><div class="surname">R.E.</div> <div class="given-names">Kingston</div></div></div>, et al.  . (eds), <div class="source">Current Protocols in Molecular Biology</div> . 
<div class="publisher-name">John Wiley &amp; Sons Inc</div>., 
<div class="publisher-loc">New Jersey</div>. Chapter 4, Section 4.3.</div></div></div></div><div content-id="dsw037-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Iwano</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Uno</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Qualitative <em>de novo</em> analysis of full length cDNA and quantitative analysis of gene expression for common marmoset (<em>Callithrix jacchus</em>) transcriptomes using parallel long-read technology and short-read sequencing</div>. <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e100936</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Qualitative%20de%20novo%20analysis%20of%20full%20length%20cDNA%20and%20quantitative%20analysis%20of%20gene%20expression%20for%20common%20marmoset%20(Callithrix%20jacchus)%20transcriptomes%20using%20parallel%20long-read%20technology%20and%20short-read%20sequencing&amp;author=M.%20Shimizu&amp;author=S.%20Iwano&amp;author=Y.%20Uno&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e100936" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0100936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0100936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0100936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24977701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A.M.</div></div><div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>. <div class="source">EMBnet. J</div> ., <div class="volume">17</div>, <div class="fpage">10</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet.%20J&amp;volume=17&amp;pages=10-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="dsw037-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gnerre</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Maccallum</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Przybylski</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data</div>. <div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">1513</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-quality%20draft%20assemblies%20of%20mammalian%20genomes%20from%20massively%20parallel%20sequence%20data&amp;author=S.%20Gnerre&amp;author=I.%20Maccallum&amp;author=D.%20Przybylski&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=1513-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1017351108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1017351108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1017351108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21187386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div><div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div><div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>. <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div><div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title"><em>De novo</em> identification of repeat families in large genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">21 Suppl 1</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21 Suppl 1&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>. <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">821</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div><div class="name string-name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div><div class="name string-name"><div class="surname">Vingron</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Oases: robust <em>de novo</em> RNA-seq assembly across the dynamic range of expression levels</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1086</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oases%3A%20robust%20de%20novo%20RNA-seq%20assembly%20across%20the%20dynamic%20range%20of%20expression%20levels&amp;author=M.H.%20Schulz&amp;author=D.R.%20Zerbino&amp;author=M.%20Vingron&amp;author=E.%20Birney&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1086-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22368243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Niu</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">CD-HIT: accelerated for clustering the next-generation sequencing data</div>. <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">3150</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CD-HIT%3A%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data&amp;author=L.%20Fu&amp;author=B.%20Niu&amp;author=Z.%20Zhu&amp;author=S.%20Wu&amp;author=W.%20Li&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3150-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div><div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoff</div> <div class="given-names">K.J.</div></div><div class="name string-name"><div class="surname">Lange</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">BRAKER1: unsupervised RNA-seq-based genome annotation with GeneMark-ET and AUGUSTUS</div>. <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">767</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BRAKER1%3A%20unsupervised%20RNA-seq-based%20genome%20annotation%20with%20GeneMark-ET%20and%20AUGUSTUS&amp;author=K.J.%20Hoff&amp;author=S.%20Lange&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=767-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26559507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>. <div class="source">Bioinformatics</div> , <div class="volume">19 Suppl 2</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19 Suppl 2&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Dewey</div> <div class="given-names">C.N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=C.N.%20Dewey&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div><div class="name string-name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div><div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">The COG database: an updated version includes eukaryotes</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Llorens</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Futami</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Covelli</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The Gypsy Database (GyDB) of mobile genetic elements: release 2.0</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D70</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gypsy%20Database%20(GyDB)%20of%20mobile%20genetic%20elements%3A%20release%202.0&amp;author=C.%20Llorens&amp;author=R.%20Futami&amp;author=L.%20Covelli&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D70-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21036865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">A new generation of homology search tools based on probabilistic inference</div>. <div class="source">Genome Inform</div> , <div class="volume">23</div>, <div class="fpage">205</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Inform&amp;volume=23&amp;pages=205-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suyama</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Torrents</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W609</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAL2NAL%3A%20robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W609-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div><div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Clustal W and Clustal X version 2.0</div>. <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">406</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">D.T.</div></div><div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">W.R.</div></div><div class="name string-name"><div class="surname">Thornton</div> <div class="given-names">J.M.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">The rapid generation of mutation data matrices from protein sequences</div>. <div class="source">Comput. Appl. Biosci</div> ., <div class="volume">8</div>, <div class="fpage">275</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=D.T.%20Jones&amp;author=W.R.%20Taylor&amp;author=J.M.%20Thornton&amp;publication_year=1992&amp;journal=Comput.%20Appl.%20Biosci&amp;volume=8&amp;pages=275-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1633570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonales-Alatorre</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Shabala</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.H.</div></div><div class="name string-name"><div class="surname">Pottosin</div> <div class="given-names">I.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Reduced tonoplast fast-activating and slow-activating channel activity is essential for conferring salinity tolerance in a facultative halophyte, quinoa</div>. <div class="source">Plant Physiol</div> ., <div class="volume">162</div>, <div class="fpage">940</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20tonoplast%20fast-activating%20and%20slow-activating%20channel%20activity%20is%20essential%20for%20conferring%20salinity%20tolerance%20in%20a%20facultative%20halophyte%2C%20quinoa&amp;author=E.%20Bonales-Alatorre&amp;author=S.%20Shabala&amp;author=Z.H.%20Chen&amp;author=I.%20Pottosin&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=162&amp;pages=940-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.216572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.216572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.216572"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Bendevis</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Shabala</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Sensitivity of two quinoa (<em>Chenopodium quinoa</em> Willd.) varieties to progressive drought stress</div>. <div class="source">J. Agron. Crop Sci</div> ., <div class="volume">200</div>, <div class="fpage">12</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sensitivity%20of%20two%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20varieties%20to%20progressive%20drought%20stress&amp;author=Y.%20Sun&amp;author=F.%20Liu&amp;author=M.%20Bendevis&amp;author=S.%20Shabala&amp;author=S.E.%20Jacobsen&amp;publication_year=2014&amp;journal=J.%20Agron.%20Crop%20Sci&amp;volume=200&amp;pages=12-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jac.2013.200.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjac.2013.200.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjac.2013.200.issue-1"> </span></div> </div></div></div></div></div><div content-id="dsw037-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Adolf</div> <div class="given-names">V.I.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.-E.</div></div><div class="name string-name"><div class="surname">Shabala</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Salt tolerance mechanisms in quinoa (<em>Chenopodium quinoa</em> Willd.)</div>. <div class="source">Environ. Exp. Bot</div> ., <div class="volume">92</div>, <div class="fpage">43</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salt%20tolerance%20mechanisms%20in%20quinoa%20(Chenopodium%20quinoa%20Willd.)&amp;author=V.I.%20Adolf&amp;author=S.-E.%20Jacobsen&amp;author=S.%20Shabala&amp;publication_year=2013&amp;journal=Environ.%20Exp.%20Bot&amp;volume=92&amp;pages=43-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.envexpbot.2012.07.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.envexpbot.2012.07.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.envexpbot.2012.07.004"> </span></div> </div></div></div></div></div><div content-id="dsw037-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hull</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">M.</div> <div class="given-names">Grigg</div></div><div class="name string-name"><div class="surname">A.</div> <div class="given-names">McBride</div></div><div class="name string-name"><div class="surname">J.M.</div> <div class="given-names">Quarles</div></div></div>, et al.  . (eds), <div class="source">Current Protocols in Microbiology</div> , 
<div class="publisher-name">John Wiley &amp; Sons, Inc</div>., 
<div class="publisher-loc">New Jersey</div>, Chapter 16, 16A.2.1–A2.4.</div></div></div></div><div content-id="dsw037-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bancroft</div> <div class="given-names">J.B.</div></div></div> <div class="year">1972</div>, 
<div class="article-title">A virus made from parts of the genomes of brome mosaic and cowpea chlorotic mottle virus. <em>J. Gen.</em></div> <div class="source">Virol</div> ., <div class="volume">14</div>, <div class="fpage">223</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="dsw037-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Osman</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Grantham</div> <div class="given-names">G.L.</div></div><div class="name string-name"><div class="surname">Rao</div> <div class="given-names">A.L.N.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Molecular studies on bromovirus capsid protein. IV. Coat protein exchanges between brome mosaic and cowpea chlorotic mottle viruses exhibit neutral effects in heterologous hosts</div>. <div class="source">Virology</div> , <div class="volume">238</div>, <div class="fpage">452</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20studies%20on%20bromovirus%20capsid%20protein.%20IV.%20Coat%20protein%20exchanges%20between%20brome%20mosaic%20and%20cowpea%20chlorotic%20mottle%20viruses%20exhibit%20neutral%20effects%20in%20heterologous%20hosts&amp;author=F.%20Osman&amp;author=G.L.%20Grantham&amp;author=A.L.N.%20Rao&amp;publication_year=1997&amp;journal=Virology&amp;volume=238&amp;pages=452-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/viro.1997.8849" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fviro.1997.8849" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fviro.1997.8849"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9400617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yasui</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Ueno</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Assembly of the draft genome of buckwheat and its applications in identifying agronomically useful genes</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">215</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20of%20the%20draft%20genome%20of%20buckwheat%20and%20its%20applications%20in%20identifying%20agronomically%20useful%20genes&amp;author=Y.%20Yasui&amp;author=H.%20Hirakawa&amp;author=M.%20Ueno&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=215-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27037832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">An</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genome survey sequencing for the characterization of the genetic background of <em>Rosa roxburghii</em> Tratt and leaf ascorbate metabolism genes</div>. <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0147530</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20survey%20sequencing%20for%20the%20characterization%20of%20the%20genetic%20background%20of%20Rosa%20roxburghii%20Tratt%20and%20leaf%20ascorbate%20metabolism%20genes&amp;author=M.%20Lu&amp;author=H.%20An&amp;author=L.%20Li&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0147530" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0147530" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0147530" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0147530"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26849133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Naito</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Ogiso-Tanaka</div> <div class="given-names">E.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">The power of single molecule real-time sequencing technology in the <em>de novo</em> assembly of a eukaryotic genome</div>. <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">16780</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20power%20of%20single%20molecule%20real-time%20sequencing%20technology%20in%20the%20de%20novo%20assembly%20of%20a%20eukaryotic%20genome&amp;author=H.%20Sakai&amp;author=K.%20Naito&amp;author=E.%20Ogiso-Tanaka&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=16780" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep16780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep16780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep16780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26616024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Shigenobu</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Motoi</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Improvement of barley genome annotations by deciphering the Haruna Nijo genome</div>. <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">21</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20barley%20genome%20annotations%20by%20deciphering%20the%20Haruna%20Nijo%20genome&amp;author=K.%20Sato&amp;author=T.%20Tanaka&amp;author=S.%20Shigenobu&amp;author=Y.%20Motoi&amp;author=J.%20Wu&amp;author=T.%20Itoh&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=21-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26622062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div><div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div><div class="name string-name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div><div class="name string-name"><div class="surname">Wing</div> <div class="given-names">R.A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Evolution of plant genome architecture</div>. <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20plant%20genome%20architecture&amp;author=J.F.%20Wendel&amp;author=S.A.%20Jackson&amp;author=B.C.%20Meyers&amp;author=R.A.%20Wing&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-0908-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-0908-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-0908-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26926526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maughan</div> <div class="given-names">P.J.</div></div><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">T.B.</div></div><div class="name string-name"><div class="surname">Coleman</div> <div class="given-names">C.E.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Characterization of <em>Salt Overly Sensitive 1</em> (<em>SOS1</em>) gene homoeologs in quinoa (<em>Chenopodium quinoa</em> Willd.)</div>. <div class="source">Genome</div> , <div class="volume">52</div>, <div class="fpage">647</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Salt%20Overly%20Sensitive%201%20(SOS1)%20gene%20homoeologs%20in%20quinoa%20(Chenopodium%20quinoa%20Willd.)&amp;author=P.J.%20Maughan&amp;author=T.B.%20Turner&amp;author=C.E.%20Coleman&amp;publication_year=2009&amp;journal=Genome&amp;volume=52&amp;pages=647-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19767895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brenchley</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Spannagl</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Pfeifer</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Analysis of the bread wheat genome using whole-genome shotgun sequencing</div>. <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">705</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20bread%20wheat%20genome%20using%20whole-genome%20shotgun%20sequencing&amp;author=R.%20Brenchley&amp;author=M.%20Spannagl&amp;author=M.%20Pfeifer&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=705-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Müller</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Borsch</div> <div class="given-names">T.</div></div></div> <div class="year">2005</div>, Phylogenetics of Amaranthaceae based on <em>mat</em>K/<em>trn</em>K sequence data: evidence from parsimony, likelihood, and Bayesian analyses. <em>Ann. Missouri Bot. Gard.</em>, <div class="volume">92</div>, <div class="fpage">66</div>–<div class="lpage">102</div>.</div></div></div></div><div content-id="dsw037-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Bush</div> <div class="given-names">D.R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">LHT1, a lysine- and histidine-specific amino acid transporter in arabidopsis</div>. <div class="source">Plant Physiol</div> ., <div class="volume">115</div>, <div class="fpage">1127</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LHT1%2C%20a%20lysine-%20and%20histidine-specific%20amino%20acid%20transporter%20in%20arabidopsis&amp;author=L.%20Chen&amp;author=D.R.%20Bush&amp;publication_year=1997&amp;journal=Plant%20Physiol&amp;volume=115&amp;pages=1127-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.115.3.1127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.115.3.1127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.115.3.1127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9390441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">Y.H.</div></div><div class="name string-name"><div class="surname">Tegeder</div> <div class="given-names">M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Selective expression of a novel high-affinity transport system for acidic and neutral amino acids in the tapetum cells of <em>Arabidopsis</em> flowers</div>. <div class="source">Plant J</div> ., <div class="volume">40</div>, <div class="fpage">60</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20expression%20of%20a%20novel%20high-affinity%20transport%20system%20for%20acidic%20and%20neutral%20amino%20acids%20in%20the%20tapetum%20cells%20of%20Arabidopsis%20flowers&amp;author=Y.H.%20Lee&amp;author=M.%20Tegeder&amp;publication_year=2004&amp;journal=Plant%20J&amp;volume=40&amp;pages=60-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.40.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.40.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.40.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15361141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Cusumano</div> <div class="given-names">J.C.</div></div><div class="name string-name"><div class="surname">Somerville</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Chapple</div> <div class="given-names">C.C.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Ferulate-5-hydroxylase from <em>Arabidopsis thaliana</em> defines a new family of cytochrome P450-dependent monooxygenases</div>. <div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">93</div>, <div class="fpage">6869</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ferulate-5-hydroxylase%20from%20Arabidopsis%20thaliana%20defines%20a%20new%20family%20of%20cytochrome%20P450-dependent%20monooxygenases&amp;author=K.%20Meyer&amp;author=J.C.%20Cusumano&amp;author=C.%20Somerville&amp;author=C.C.%20Chapple&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=93&amp;pages=6869-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.14.6869" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.14.6869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.14.6869"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8692910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brockington</div> <div class="given-names">S.F.</div></div><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Gandia-Herrero</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Lineage-specific gene radiations underlie the evolution of novel betalain pigmentation in Caryophyllales</div>. <div class="source">New Phytol</div> ., <div class="volume">207</div>, <div class="fpage">1170</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lineage-specific%20gene%20radiations%20underlie%20the%20evolution%20of%20novel%20betalain%20pigmentation%20in%20Caryophyllales&amp;author=S.F.%20Brockington&amp;author=Y.%20Yang&amp;author=F.%20Gandia-Herrero&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=207&amp;pages=1170-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25966996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ruiz</div> <div class="given-names">K.B.</div></div><div class="name string-name"><div class="surname">Biondi</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">E.A.</div></div><div class="name string-name"><div class="surname">Orsini</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Antognoni</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Quinoa – a model crop for understanding salt-tolerance mechanisms in halophytes</div>. <div class="source">Plant Biosyst</div> ., <div class="volume">150</div>, <div class="fpage">357</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quinoa%20%E2%80%93%20a%20model%20crop%20for%20understanding%20salt-tolerance%20mechanisms%20in%20halophytes&amp;author=K.B.%20Ruiz&amp;author=S.%20Biondi&amp;author=E.A.%20Mart%C3%ADnez&amp;author=F.%20Orsini&amp;author=F.%20Antognoni&amp;author=S.E.%20Jacobsen&amp;publication_year=2016&amp;journal=Plant%20Biosyst&amp;volume=150&amp;pages=357-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/11263504.2015.1027317" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F11263504.2015.1027317" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F11263504.2015.1027317"> </span></div> </div></div></div></div></div><div content-id="dsw037-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miyakawa</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Tanokura</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Structure and function of abscisic acid receptors</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">18</div>, <div class="fpage">259</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20function%20of%20abscisic%20acid%20receptors&amp;author=T.%20Miyakawa&amp;author=Y.%20Fujita&amp;author=K.%20Yamaguchi-Shinozaki&amp;author=M.%20Tanokura&amp;publication_year=2013&amp;journal=Trends%20Plant%20Sci&amp;volume=18&amp;pages=259-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2012.11.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2012.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2012.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23265948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">S.J.</div></div><div class="name string-name"><div class="surname">Carter</div> <div class="given-names">S.A.</div></div><div class="name string-name"><div class="surname">Cole</div> <div class="given-names">A.B.</div></div><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">N.H.</div></div><div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">R.S.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">A natural variant of a host RNA-dependent RNA polymerase is associated with increased susceptibility to viruses by <em>Nicotiana benthamiana</em></div>. <div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">101</div>, <div class="fpage">6297</div>–<div class="lpage">302</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20natural%20variant%20of%20a%20host%20RNA-dependent%20RNA%20polymerase%20is%20associated%20with%20increased%20susceptibility%20to%20viruses%20by%20Nicotiana%20benthamiana&amp;author=S.J.%20Yang&amp;author=S.A.%20Carter&amp;author=A.B.%20Cole&amp;author=N.H.%20Cheng&amp;author=R.S.%20Nelson&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=101&amp;pages=6297-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0304346101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0304346101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0304346101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15079073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bally</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Nakasugi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Jia</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">The extremophile <em>Nicotiana benthamiana</em> has traded viral defence for early vigour</div>. <div class="source">Nat. Plants</div> , <div class="volume">1</div>, <div class="fpage">15165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extremophile%20Nicotiana%20benthamiana%20has%20traded%20viral%20defence%20for%20early%20vigour&amp;author=J.%20Bally&amp;author=K.%20Nakasugi&amp;author=F.%20Jia&amp;publication_year=2015&amp;journal=Nat.%20Plants&amp;volume=1&amp;pages=15165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2015.165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2015.165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2015.165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27251536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iyer</div> <div class="given-names">L.M.</div></div><div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div><div class="name string-name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Evolutionary connection between the catalytic subunits of DNA-dependent RNA polymerases and eukaryotic RNA-dependent RNA polymerases and the origin of RNA polymerases</div>. <div class="source">BMC Struct. Biol</div> ., <div class="volume">3</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20connection%20between%20the%20catalytic%20subunits%20of%20DNA-dependent%20RNA%20polymerases%20and%20eukaryotic%20RNA-dependent%20RNA%20polymerases%20and%20the%20origin%20of%20RNA%20polymerases&amp;author=L.M.%20Iyer&amp;author=E.V.%20Koonin&amp;author=L.%20Aravind&amp;publication_year=2003&amp;journal=BMC%20Struct.%20Biol&amp;volume=3&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1472-6807-3-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1472-6807-3-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1472-6807-3-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12553882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Lyons</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Town</div> <div class="given-names">C.D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Optical mapping in plant comparative genomics</div>. <div class="source">Gigascience</div> , <div class="volume">4</div>, <div class="fpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optical%20mapping%20in%20plant%20comparative%20genomics&amp;author=H.%20Tang&amp;author=E.%20Lyons&amp;author=C.D.%20Town&amp;publication_year=2015&amp;journal=Gigascience&amp;volume=4&amp;pages=3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13742-015-0044-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13742-015-0044-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13742-015-0044-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25699175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw037-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation">Phytozome 11, <a class="link link-uri" href="https://phytozome.jgi.doe.gov/pz/portal.html" target="_blank">https://phytozome.jgi.doe.gov/pz/portal.html</a></div></div></div></div><div content-id="dsw037-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation">The Beta vulgaris Resource, <a class="link link-uri" href="http://bvseq.molgen.mpg.de/index.shtml" target="_blank">http://bvseq.molgen.mpg.de/index.shtml</a>.</div></div></div></div><div content-id="dsw037-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation">Nicotiana benthamiana Genome &amp; Transcriptome, <a class="link link-uri" href="http://benthgenome.qut.edu.au" target="_blank">http://benthgenome.qut.edu.au</a>.</div></div></div></div><div content-id="dsw037-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="collab">EMBOSS Transeq</div>, <a class="link link-uri" href="http://www.ebi.ac.uk/Tools/st/emboss_transeq/" target="_blank">http://www.ebi.ac.uk/Tools/st/emboss_transeq/</a>.</div></div></div></div><div content-id="dsw037-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw037-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marchler-Bauer</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Derbyshire</div> <div class="given-names">M.K.</div></div><div class="name string-name"><div class="surname">Gonzales</div> <div class="given-names">N.R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">CDD: NCBI's conserved domain database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D222</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20NCBI's%20conserved%20domain%20database&amp;author=A.%20Marchler-Bauer&amp;author=M.K.%20Derbyshire&amp;author=N.R.%20Gonzales&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D222-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25414356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2647444&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw037/2/dsw037_Supp.zip?Expires=2147483647&amp;Signature=SYQbmgwNJGTDLPts8a7npbVjFrRkOuazU77S5R3ZYXu9V1EV6vxrU-o~E9CJEIqJ01CVUePKV8KitLHJhIdHYXtFm~FYmvuyermxJ68C32bf3DscsPk3FJJzuxVsNwyPwjTEikHGzaGYrPHwcg8mN9LvTW-C4d2yuDlKYInEIM7ylf7ClqCYtFk2npFwmtAbZj3ML4bcCE4N1qqjaBjeGsTjmESWKEnhLoXOxYPa10GaIGZnuvWAN6ByIM7hOe6Kyd2Qz5SmmKLwymefsvSu~IzJjs4P8qfy-XV7R9-JFJ1X1cgv5Uz-wbLZv74tUDg3NYEnFwF26KRYcpdJLePGPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,165</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw037"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2647444"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f6%2f535%2f2647444">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000392723300004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F6%2F535%2F2647444" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2647444'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Draft genome sequence of an inbred line of Chenopodium quinoa, an allotetraploid crop with great environmental adaptability and outstanding nutritional properties" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw037");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f535%2f2647444">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

