

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/4/201/500751","@type":"ScholarlyArticle","name":"Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation","datePublished":"2011-05-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2011-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr008","keywords":["transplantation","MHC","human resequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/4/201/500751","author":[{"name":"Pröll, Johannes","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Danzer, Martin","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Stabentheiner, Stephanie","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Niklas, Norbert","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Hackl, Christa","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Hofer, Katja","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Atzmüller, Sabine","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"},{"name":"Hufnagl, Peter","affiliation":"2  Roche Diagnostics GmbH, Professional Diagnostics and Research ,  Engelhorngasse 3, Wien ,  Austria","@type":"Person"},{"name":"Gülly, Christian","affiliation":"3  Center for Medical Research ,  Medical University of Graz ,  Stiftingtalstraße 24, Graz ,  Austria","@type":"Person"},{"name":"Hauser, Hanns","affiliation":"4  First Department of Internal Medicine ,  Elisabethinen Hospital  Linz, Fadingerstrasse 1 ,  Linz, Austria","@type":"Person"},{"name":"Krieger, Otto","affiliation":"4  First Department of Internal Medicine ,  Elisabethinen Hospital  Linz, Fadingerstrasse 1 ,  Linz, Austria","@type":"Person"},{"name":"Gabriel, Christian","affiliation":"1  Red Cross Transfusion Service for Upper Austria ,  Krankenhausstraße 7, Linz ,  Austria","@type":"Person"}],"description":"Abstract.  How cells coordinate the immune system activities is important for potentially life-saving organ or stem cell transplantations. Polymorphic immunoreg","pageStart":"201","pageEnd":"210","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation" />
    <meta property="og:description" content="Abstract.  How cells coordinate the immune system activities is important for potentially life-saving organ or stem cell transplantations. Polymorphic immunoreg" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/4/201/500751" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-05-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Pröll, Johannes" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Danzer, Martin" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Stabentheiner, Stephanie" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Niklas, Norbert" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Hackl, Christa" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Hofer, Katja" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Atzmüller, Sabine" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_author" content="Hufnagl, Peter" /><meta name="citation_author_institution" content="Roche Diagnostics GmbH, Professional Diagnostics and Research, Engelhorngasse 3, Wien, Austria" /><meta name="citation_author" content="Gülly, Christian" /><meta name="citation_author_institution" content="Center for Medical Research, Medical University of Graz, Stiftingtalstraße 24, Graz, Austria" /><meta name="citation_author" content="Hauser, Hanns" /><meta name="citation_author_institution" content="First Department of Internal Medicine, Elisabethinen HospitalLinz, Fadingerstrasse 1, Linz, Austria" /><meta name="citation_author" content="Krieger, Otto" /><meta name="citation_author_institution" content="First Department of Internal Medicine, Elisabethinen HospitalLinz, Fadingerstrasse 1, Linz, Austria" /><meta name="citation_author" content="Gabriel, Christian" /><meta name="citation_author_institution" content="Red Cross Transfusion Service for Upper Austria, Krankenhausstraße 7, Linz, Austria" /><meta name="citation_title" content="Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation" /><meta name="citation_firstpage" content="201" /><meta name="citation_lastpage" content="210" /><meta name="citation_doi" content="10.1093/dnares/dsr008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2011/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Advances in the selection of HLA-compatible donors: refinements in HLA typing and matching over the first 20 years of the National Marrow Donor Program Registry; Biol. Blood Marrow Transplant.;  citation_year=2008;  citation_volume=14;  citation_pages=37-44; " /><meta name="citation_reference" content="citation_title=Complete sequence and gene map of a human major histocompatibility complex. The MHC sequencing consortium; Nature;  citation_year=1999;  citation_volume=401;  citation_pages=921-3; " /><meta name="citation_reference" content="citation_title=Complete MHC haplotype sequencing for common disease gene mapping; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1176-87; " /><meta name="citation_reference" content="citation_title=Variation analysis and gene annotation of eight MHC haplotypes: the MHC Haplotype Project; Immunogenetics;  citation_year=2008;  citation_volume=60;  citation_pages=1-18; " /><meta name="citation_reference" content="citation_title=A genome-wide search for human type 1 diabetes susceptibility genes; Nature;  citation_year=1994;  citation_volume=371;  citation_pages=130-6; " /><meta name="citation_reference" content="citation_title=Impact of MHC class I diversity on immune control of immunodeficiency virus replication; Nat. Rev. Immunol.;  citation_year=2008;  citation_volume=8;  citation_pages=619-30; " /><meta name="citation_reference" content="citation_title=High-resolution donor-recipient HLA matching contributes to the success of unrelated donor marrow transplantation; Blood;  citation_year=2007;  citation_volume=110;  citation_pages=4576-83; " /><meta name="citation_reference" content="citation_title=HLA matching in allogeneic stem cell transplantation; Curr. Opin. Hematol.;  citation_year=2004;  citation_volume=11;  citation_pages=386-91; " /><meta name="citation_reference" content="citation_title=MHC haplotype matching for unrelated hematopoietic cell transplantation; PLoS Med.;  citation_year=2007;  citation_volume=4;  citation_pages=e8" /><meta name="citation_reference" content="citation_title=The structure of haplotype blocks in the human genome; Science;  citation_year=2002;  citation_volume=296;  citation_pages=2225-9; " /><meta name="citation_reference" content="citation_title=Long-range multilocus haplotype phasing of the MHC; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=6964-9; " /><meta name="citation_reference" content="citation_title=High-resolution, high-throughput HLA genotyping by next-generation sequencing; Tissue Antigens;  citation_year=2009;  citation_volume=74;  citation_pages=393-403; " /><meta name="citation_reference" content="citation_title=Rapid high-throughput human leukocyte antigen typing by massively parallel pyrosequencing for high-resolution allele identification; Hum. Immunol.;  citation_year=2009;  citation_volume=70;  citation_pages=960-4; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing: the solution for high-resolution; Unambiguous HLA Typing, Hum. Immunol.;  citation_year=2010;  citation_volume=71;  citation_pages=1033-42; " /><meta name="citation_reference" content="citation_title=Mechanisms of multidrug resistance: the potential role of microtubule-stabilizing agents; Ann. Oncol.;  citation_year=2007;  citation_volume=18;  citation_issue=Suppl. 5;  citation_pages=v3-v8; " /><meta name="citation_reference" content="citation_title=Potent anti-inflammatory activity of novel microtubule-modulating brominated noscapine analogs; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e9165" /><meta name="citation_reference" content="citation_title=Graft-versus-host disease: regulation by microbe-associated molecules and innate immune receptors; Blood;  citation_year=2010;  citation_volume=115;  citation_pages=1865-72; " /><meta name="citation_reference" content="citation_title=Enteropathogenic Escherichia coli type III effectors EspG and EspG2 disrupt the microtubule network of intestinal epithelial cells; Infect. Immun.;  citation_year=2005;  citation_volume=73;  citation_pages=4385-90; " /><meta name="citation_reference" content="citation_title=Transposons in the MHC: the Yin and Yang of the vertebrate immune system; Heredity;  citation_year=2009;  citation_volume=103;  citation_pages=190-1; " /><meta name="citation_reference" content="citation_title=The synergistic effect of tautomycetin on cyclosporine a-mediated immunosuppression in a rodent islet allograft model; Mol. Med.;  citation_year=2010;  citation_volume=16;  citation_pages=298-306; " /><meta name="citation_reference" content="citation_title=GNL3L stabilizes the TRF1 complex and promotes mitotic transition; J. Cell Biol.;  citation_year=2009;  citation_volume=185;  citation_pages=827-39; " /><meta name="citation_reference" content="citation_title=Dissection and molecular analysis of alloreactive CD8+ T cell responses in allogeneic haematopoietic stem cell transplantation; Cancer Immunol. Immunother.;  citation_year=2008;  citation_volume=57;  citation_pages=849-57; " /><meta name="citation_reference" content="citation_title=Immunobiology of human NKG2D and its ligands; Curr. Top. Microbiol. Immunol.;  citation_year=2006;  citation_volume=298;  citation_pages=121-38; " /><meta name="citation_reference" content="citation_title=Potential role of NKG2D and its ligands in organ transplantation: new target for immunointervention; Am. J. Transplant.;  citation_year=2009;  citation_volume=9;  citation_pages=251-7; " /><meta name="citation_reference" content="citation_title=The diversity of ubiquitin recognition: hot spots and varied specificity; Mol. Cell;  citation_year=2010;  citation_volume=38;  citation_pages=627-35; " /><meta name="citation_reference" content="citation_title=Immunoproteasome beta subunit 10 is increased in chronic antibody-mediated rejection; Kidney Int.;  citation_year=2010;  citation_volume=77;  citation_pages=880-90; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/4/201/1348494/dsr008.pdf" /><meta name="description" content="Abstract.  How cells coordinate the immune system activities is important for potentially life-saving organ or stem cell transplantations. Polymorphic immunoreg" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/4/201/500751" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/4/201/500751" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/4/201/500751");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr008';
    </script>
<title>
	Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./500751" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BedKPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQcsO2bGwsIEpvaGFubmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVkIENyb3NzIFRyYW5zZnVzaW9uIFNlcnZpY2UgZm9yIFVwcGVyIEF1c3RyaWEsIEtyYW5rZW5oYXVzc3RyYcOfZSA3LCBMaW56LCBBdXN0cmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGFuemVyLCBNYXJ0aW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZWQgQ3Jvc3MgVHJhbnNmdXNpb24gU2VydmljZSBmb3IgVXBwZXIgQXVzdHJpYSwgS3JhbmtlbmhhdXNzdHJhw59lIDcsIExpbnosIEF1c3RyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdGFiZW50aGVpbmVyLCBTdGVwaGFuaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZWQgQ3Jvc3MgVHJhbnNmdXNpb24gU2VydmljZSBmb3IgVXBwZXIgQXVzdHJpYSwgS3JhbmtlbmhhdXNzdHJhw59lIDcsIExpbnosIEF1c3RyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWtsYXMsIE5vcmJlcnQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZWQgQ3Jvc3MgVHJhbnNmdXNpb24gU2VydmljZSBmb3IgVXBwZXIgQXVzdHJpYSwgS3JhbmtlbmhhdXNzdHJhw59lIDcsIExpbnosIEF1c3RyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYWNrbCwgQ2hyaXN0YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlZCBDcm9zcyBUcmFuc2Z1c2lvbiBTZXJ2aWNlIGZvciBVcHBlciBBdXN0cmlhLCBLcmFua2VuaGF1c3N0cmHDn2UgNywgTGlueiwgQXVzdHJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhvZmVyLCBLYXRqYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlZCBDcm9zcyBUcmFuc2Z1c2lvbiBTZXJ2aWNlIGZvciBVcHBlciBBdXN0cmlhLCBLcmFua2VuaGF1c3N0cmHDn2UgNywgTGlueiwgQXVzdHJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkF0em3DvGxsZXIsIFNhYmluZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlZCBDcm9zcyBUcmFuc2Z1c2lvbiBTZXJ2aWNlIGZvciBVcHBlciBBdXN0cmlhLCBLcmFua2VuaGF1c3N0cmHDn2UgNywgTGlueiwgQXVzdHJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1Zm5hZ2wsIFBldGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUm9jaGUgRGlhZ25vc3RpY3MgR21iSCwgUHJvZmVzc2lvbmFsIERpYWdub3N0aWNzIGFuZCBSZXNlYXJjaCwgRW5nZWxob3JuZ2Fzc2UgMywgV2llbiwgQXVzdHJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkfDvGxseSwgQ2hyaXN0aWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBNZWRpY2FsIFJlc2VhcmNoLCBNZWRpY2FsIFVuaXZlcnNpdHkgb2YgR3JheiwgU3RpZnRpbmd0YWxzdHJhw59lIDI0LCBHcmF6LCBBdXN0cmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF1c2VyLCBIYW5ucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZpcnN0IERlcGFydG1lbnQgb2YgSW50ZXJuYWwgTWVkaWNpbmUsIEVsaXNhYmV0aGluZW4gSG9zcGl0YWxMaW56LCBGYWRpbmdlcnN0cmFzc2UgMSwgTGlueiwgQXVzdHJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktyaWVnZXIsIE90dG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGaXJzdCBEZXBhcnRtZW50IG9mIEludGVybmFsIE1lZGljaW5lLCBFbGlzYWJldGhpbmVuIEhvc3BpdGFsTGlueiwgRmFkaW5nZXJzdHJhc3NlIDEsIExpbnosIEF1c3RyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYWJyaWVsLCBDaHJpc3RpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZWQgQ3Jvc3MgVHJhbnNmdXNpb24gU2VydmljZSBmb3IgVXBwZXIgQXVzdHJpYSwgS3JhbmtlbmhhdXNzdHJhw59lIDcsIExpbnosIEF1c3RyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNlcXVlbmNlIENhcHR1cmUgYW5kIE5leHQgR2VuZXJhdGlvbiBSZXNlcXVlbmNpbmcgb2YgdGhlIE1IQyBSZWdpb24gSGlnaGxpZ2h0cyBQb3RlbnRpYWwgVHJhbnNwbGFudGF0aW9uIERldGVybWluYW50cyBpbiBITEEgSWRlbnRpY2FsIEhhZW1hdG9wb2lldGljIFN0ZW0gQ2VsbCBUcmFuc3BsYW50YXRpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWR2YW5jZXMgaW4gdGhlIHNlbGVjdGlvbiBvZiBITEEtY29tcGF0aWJsZSBkb25vcnM6IHJlZmluZW1lbnRzIGluIEhMQSB0eXBpbmcgYW5kIG1hdGNoaW5nIG92ZXIgdGhlIGZpcnN0IDIwIHllYXJzIG9mIHRoZSBOYXRpb25hbCBNYXJyb3cgRG9ub3IgUHJvZ3JhbSBSZWdpc3RyeTsgQmlvbC4gQmxvb2QgTWFycm93IFRyYW5zcGxhbnQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIHNlcXVlbmNlIGFuZCBnZW5lIG1hcCBvZiBhIGh1bWFuIG1ham9yIGhpc3RvY29tcGF0aWJpbGl0eSBjb21wbGV4LiBUaGUgTUhDIHNlcXVlbmNpbmcgY29uc29ydGl1bTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTQwMTsgIGNpdGF0aW9uX3BhZ2VzPTkyMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBNSEMgaGFwbG90eXBlIHNlcXVlbmNpbmcgZm9yIGNvbW1vbiBkaXNlYXNlIGdlbmUgbWFwcGluZzsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTc2LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gYW5hbHlzaXMgYW5kIGdlbmUgYW5ub3RhdGlvbiBvZiBlaWdodCBNSEMgaGFwbG90eXBlczogdGhlIE1IQyBIYXBsb3R5cGUgUHJvamVjdDsgSW1tdW5vZ2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIHNlYXJjaCBmb3IgaHVtYW4gdHlwZSAxIGRpYWJldGVzIHN1c2NlcHRpYmlsaXR5IGdlbmVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MzcxOyAgY2l0YXRpb25fcGFnZXM9MTMwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGFjdCBvZiBNSEMgY2xhc3MgSSBkaXZlcnNpdHkgb24gaW1tdW5lIGNvbnRyb2wgb2YgaW1tdW5vZGVmaWNpZW5jeSB2aXJ1cyByZXBsaWNhdGlvbjsgTmF0LiBSZXYuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02MTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtcmVzb2x1dGlvbiBkb25vci1yZWNpcGllbnQgSExBIG1hdGNoaW5nIGNvbnRyaWJ1dGVzIHRvIHRoZSBzdWNjZXNzIG9mIHVucmVsYXRlZCBkb25vciBtYXJyb3cgdHJhbnNwbGFudGF0aW9uOyBCbG9vZDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz00NTc2LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITEEgbWF0Y2hpbmcgaW4gYWxsb2dlbmVpYyBzdGVtIGNlbGwgdHJhbnNwbGFudGF0aW9uOyBDdXJyLiBPcGluLiBIZW1hdG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUhDIGhhcGxvdHlwZSBtYXRjaGluZyBmb3IgdW5yZWxhdGVkIGhlbWF0b3BvaWV0aWMgY2VsbCB0cmFuc3BsYW50YXRpb247IFBMb1MgTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3RydWN0dXJlIG9mIGhhcGxvdHlwZSBibG9ja3MgaW4gdGhlIGh1bWFuIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz0yMjI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmctcmFuZ2UgbXVsdGlsb2N1cyBoYXBsb3R5cGUgcGhhc2luZyBvZiB0aGUgTUhDOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz02OTY0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtcmVzb2x1dGlvbiwgaGlnaC10aHJvdWdocHV0IEhMQSBnZW5vdHlwaW5nIGJ5IG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyBUaXNzdWUgQW50aWdlbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0zOTMtNDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBoaWdoLXRocm91Z2hwdXQgaHVtYW4gbGV1a29jeXRlIGFudGlnZW4gdHlwaW5nIGJ5IG1hc3NpdmVseSBwYXJhbGxlbCBweXJvc2VxdWVuY2luZyBmb3IgaGlnaC1yZXNvbHV0aW9uIGFsbGVsZSBpZGVudGlmaWNhdGlvbjsgSHVtLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTk2MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzogdGhlIHNvbHV0aW9uIGZvciBoaWdoLXJlc29sdXRpb247IFVuYW1iaWd1b3VzIEhMQSBUeXBpbmcsIEh1bS4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0xMDMzLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc21zIG9mIG11bHRpZHJ1ZyByZXNpc3RhbmNlOiB0aGUgcG90ZW50aWFsIHJvbGUgb2YgbWljcm90dWJ1bGUtc3RhYmlsaXppbmcgYWdlbnRzOyBBbm4uIE9uY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsLiA1OyAgY2l0YXRpb25fcGFnZXM9djMtdjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvdGVudCBhbnRpLWluZmxhbW1hdG9yeSBhY3Rpdml0eSBvZiBub3ZlbCBtaWNyb3R1YnVsZS1tb2R1bGF0aW5nIGJyb21pbmF0ZWQgbm9zY2FwaW5lIGFuYWxvZ3M7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lOTE2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYWZ0LXZlcnN1cy1ob3N0IGRpc2Vhc2U6IHJlZ3VsYXRpb24gYnkgbWljcm9iZS1hc3NvY2lhdGVkIG1vbGVjdWxlcyBhbmQgaW5uYXRlIGltbXVuZSByZWNlcHRvcnM7IEJsb29kOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTE4NjUtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudGVyb3BhdGhvZ2VuaWMgRXNjaGVyaWNoaWEgY29saSB0eXBlIElJSSBlZmZlY3RvcnMgRXNwRyBhbmQgRXNwRzIgZGlzcnVwdCB0aGUgbWljcm90dWJ1bGUgbmV0d29yayBvZiBpbnRlc3RpbmFsIGVwaXRoZWxpYWwgY2VsbHM7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9NDM4NS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NvbnMgaW4gdGhlIE1IQzogdGhlIFlpbiBhbmQgWWFuZyBvZiB0aGUgdmVydGVicmF0ZSBpbW11bmUgc3lzdGVtOyBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xOTAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN5bmVyZ2lzdGljIGVmZmVjdCBvZiB0YXV0b215Y2V0aW4gb24gY3ljbG9zcG9yaW5lIGEtbWVkaWF0ZWQgaW1tdW5vc3VwcHJlc3Npb24gaW4gYSByb2RlbnQgaXNsZXQgYWxsb2dyYWZ0IG1vZGVsOyBNb2wuIE1lZC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yOTgtMzA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HTkwzTCBzdGFiaWxpemVzIHRoZSBUUkYxIGNvbXBsZXggYW5kIHByb21vdGVzIG1pdG90aWMgdHJhbnNpdGlvbjsgSi4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz04MjctMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3NlY3Rpb24gYW5kIG1vbGVjdWxhciBhbmFseXNpcyBvZiBhbGxvcmVhY3RpdmUgQ0Q4KyBUIGNlbGwgcmVzcG9uc2VzIGluIGFsbG9nZW5laWMgaGFlbWF0b3BvaWV0aWMgc3RlbSBjZWxsIHRyYW5zcGxhbnRhdGlvbjsgQ2FuY2VyIEltbXVub2wuIEltbXVub3RoZXIuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9ODQ5LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbW11bm9iaW9sb2d5IG9mIGh1bWFuIE5LRzJEIGFuZCBpdHMgbGlnYW5kczsgQ3Vyci4gVG9wLiBNaWNyb2Jpb2wuIEltbXVub2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI5ODsgIGNpdGF0aW9uX3BhZ2VzPTEyMS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG90ZW50aWFsIHJvbGUgb2YgTktHMkQgYW5kIGl0cyBsaWdhbmRzIGluIG9yZ2FuIHRyYW5zcGxhbnRhdGlvbjogbmV3IHRhcmdldCBmb3IgaW1tdW5vaW50ZXJ2ZW50aW9uOyBBbS4gSi4gVHJhbnNwbGFudC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI1MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGl2ZXJzaXR5IG9mIHViaXF1aXRpbiByZWNvZ25pdGlvbjogaG90IHNwb3RzIGFuZCB2YXJpZWQgc3BlY2lmaWNpdHk7IE1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTYyNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5vcHJvdGVhc29tZSBiZXRhIHN1YnVuaXQgMTAgaXMgaW5jcmVhc2VkIGluIGNocm9uaWMgYW50aWJvZHktbWVkaWF0ZWQgcmVqZWN0aW9uOyBLaWRuZXkgSW50LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTg4MC05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzQvMjAxLzEzNDg0OTQvZHNyMDA4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBIb3cgY2VsbHMgY29vcmRpbmF0ZSB0aGUgaW1tdW5lIHN5c3RlbSBhY3Rpdml0aWVzIGlzIGltcG9ydGFudCBmb3IgcG90ZW50aWFsbHkgbGlmZS1zYXZpbmcgb3JnYW4gb3Igc3RlbSBjZWxsIHRyYW5zcGxhbnRhdGlvbnMuIFBvbHltb3JwaGljIGltbXVub3JlZyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTAwNzUxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJcfKepGHzEOCcilE8c9uIgz1Bt8" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f201%2f500751" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f201%2f500751" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19820" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/4/1/m_cover.gif?Expires=2147483647&amp;Signature=ZFRbowJBxwXOTzTsPz03DWYsnGghS-nU1nO0exRyQHGXUuLV6Xu2GicBw3LWSCNHEfcWFiT4HGUNeIPDVRBi214rI7PVArgVPJo1BMA1-5xIUwDwxPOBmvPgwLB~aeQyV1kDNVOVp8DwQr2zKRbxPCH2HpyvBmkPeuMYcTbfTZ6okGIWEBcV0ISBBLmKp85KT4awLO8JGP1n1fW8sIxseMx8rvNF3xoz-lMI~mG~A46w5Xa~4G55BcTuFbAMKXLQhHv5lw1qw4EKRN23cP-CfP9rWU8Rh-AcS5~j9V91FL7CnLL4awypeZ7cbGrkuFLvsQNtFcL8nMFLefCIW-Ppsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/4">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7719733">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719733">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719735">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719735">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719742">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719742">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719765">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719765">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719802">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719802">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719806">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719806">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7719808">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7719808">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/18/4/211/500841">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Johannes Pr&#246;ll</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 43-732-777000-426. Fax.: 43-732-777000-430. E-mail: <a href="mailto:johannes.proell@o.roteskreuz.at" target="_blank">johannes.proell@o.roteskreuz.at</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Johannes+Pr%c3%b6ll">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pr&#246;ll J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pr&#246;ll J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Johannes Pröll</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Danzer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Danzer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Danzer M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Danzer M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Martin Danzer</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephanie Stabentheiner</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephanie+Stabentheiner">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stabentheiner S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Stabentheiner S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Stephanie Stabentheiner</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Norbert Niklas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Norbert+Niklas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Niklas N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Niklas N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Norbert Niklas</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christa Hackl</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christa+Hackl">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hackl C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hackl C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Christa Hackl</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katja Hofer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katja+Hofer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hofer K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hofer K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Katja Hofer</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sabine Atzm&#252;ller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sabine+Atzm%c3%bcller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Atzm&#252;ller S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Atzm&#252;ller S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sabine Atzmüller</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peter Hufnagl</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Roche Diagnostics GmbH, Professional Diagnostics and Research</div>, <div class="addr-line">Engelhorngasse 3, Wien</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peter+Hufnagl">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hufnagl P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hufnagl P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Peter Hufnagl</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christian G&#252;lly</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Center for Medical Research</div>, <div class="insititution">Medical University of Graz</div>, <div class="addr-line">Stiftingtalstraße 24, Graz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christian+G%c3%bclly">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=G&#252;lly C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;G&#252;lly C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Christian Gülly</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hanns Hauser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">First Department of Internal Medicine</div>, <div class="insititution">Elisabethinen Hospital</div><div class="addr-line">Linz, Fadingerstrasse 1</div>, <div class="country">Linz, Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hanns+Hauser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hauser H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hauser H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hanns Hauser</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Otto Krieger</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">First Department of Internal Medicine</div>, <div class="insititution">Elisabethinen Hospital</div><div class="addr-line">Linz, Fadingerstrasse 1</div>, <div class="country">Linz, Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Otto+Krieger">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Krieger O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Krieger O&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Otto Krieger</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christian Gabriel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Red Cross Transfusion Service for Upper Austria</div>, <div class="addr-line">Krankenhausstraße 7, Linz</div>, <div class="country">Austria</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christian+Gabriel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gabriel C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gabriel C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Christian Gabriel</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 201–210, <a href='https://doi.org/10.1093/dnares/dsr008'>https://doi.org/10.1093/dnares/dsr008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 May 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 October 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 April 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="500751" href="/dnaresearch/article-pdf/18/4/201/1348494/dsr008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Johannes Pröll, Martin Danzer, Stephanie Stabentheiner, Norbert Niklas, Christa Hackl, Katja Hofer, Sabine Atzmüller, Peter Hufnagl, Christian Gülly, Hanns Hauser, Otto Krieger, Christian Gabriel;  Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation, <em>DNA Research</em>, Volume 18, Issue 4, 1 August 2011, Pages 201–210, <a href='https://doi.org/10.1093/dnares/dsr008'>https://doi.org/10.1093/dnares/dsr008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/500751?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/500751?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sequence%20Capture%20and%20Next%20Generation%20Resequencing%20of%20the%20MHC%20Region%20Highlights%20Potential%20Transplantation%20Determinants%20in%20HLA%20Identical%20Haematopoietic%20Stem%20Cell%20Transplantation&amp;publicationDate=2011-05-28&amp;volumeNum=18&amp;issueNum=4&amp;author=Pr%C3%B6ll%2C%20Johannes%3B%20Danzer%2C%20Martin&amp;startPage=201&amp;endPage=210&amp;contentId=10.1093%2Fdnares%2Fdsr008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f201%2f500751" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7719733 id="7719733" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>How cells coordinate the immune system activities is important for potentially life-saving organ or stem cell transplantations. Polymorphic immunoregulatory genes, many of them located in the human major histocompatibility complex, impact the process and assure the proper execution of tolerance-versus-activity mechanisms. In haematopoietic stem cell transplantation, on the basis of fully human leukocyte antigen (HLA)-matched donor–recipient pairs, adverse effects like graft versus leukaemia and graft versus host are observed and difficult to handle. So far, high-resolution HLA typing was performed with Sanger sequencing, but for methodological reasons information on additional immunocompetent major histocompatibility complex loci has not been revealed. Now, we have used microarray sequence capture and targeted enrichment combined with next generation pyrosequencing for 3.5 million base pair human major histocompatibility complex resequencing in a clinical transplant setting and describe 3025 variant single nucleotide polymorphisms, insertions and deletions among recipient and donor in a single sequencing experiment. Taken together, the presented data show that sequence capture and massively parallel pyrosequencing can be used as a new tool for risk assessment in the setting of allogeneic stem cell transplantation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="transplantation">transplantation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MHC">MHC</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;human resequencing&quot;">human resequencing</a></div>                            <h2 scrollto-destination=7719735 id="7719735" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Haematopoietic stem cell transplantation (HSCT) is a well-established treatment in various haematologic malignancies. Advances in clinical practice and improvements in donor–patient matching have produced significant better outcomes in related and unrelated allogeneic HSCT. Especially, refinements in high-resolution DNA-based tissue typing and knowledge which transplantation antigens are most significant when selecting donors for recipients helped to improve and prolong survival rates.<sup><a reveal-id="DSR008C1" data-open="DSR008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The classical transplantation antigens defined as human leukocyte antigens (HLA) are physically located within a ∼4 Mb highly polymorphic multigene region, known as the major histocompatibility complex (MHC) on chromosome 6.<sup><a reveal-id="DSR008C2  DSR008C4" data-open="DSR008C2  DSR008C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Many of the genes annotated inside the MHC have immune-related functions, for instance genes involved in HLA expression and transport or antigen degradation and presentation. Hence, MHC-located genes with distinct extensive polymorphisms play a fundamental role in transplantation medicine and moreover in autoimmune, inflammatory and infection diseases in humans.<sup><a reveal-id="DSR008C5" data-open="DSR008C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR008C6" data-open="DSR008C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>From the clinical perspective, exact HLA typing for class I and class II genes is a necessity for haematologic stem cell transplantation.<sup><a reveal-id="DSR008C7" data-open="DSR008C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR008C8" data-open="DSR008C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In addition to HLA, MHC-resident unidentified transplantation determinants recently were proposed to be responsible for graft-versus-host disease (GVHD).<sup><a reveal-id="DSR008C9" data-open="DSR008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Petersdorf <em>et al</em>. identified significantly increased risks of severe acute GVHD and lower incidence of disease recurrence in haplotype-mismatched HLA-matched unrelated HSCT. Extended haplotypes of unrelated donors and recipients were allusively defined by applying a mapping approach using the physical linkage of HLA-A, -B and -DRB1. Another methodological approach, measuring risks associated with donor–recipient MHC microsatellite marker matches or mismatches, showed increased or decreased risk for death and GVHD development.<sup><a reveal-id="DSR008C10" data-open="DSR008C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>In case of yet unknown transplantation determinants located inside the MHC region, new technologies are essential for providing direct insights. Current approaches to define MHC-resident disease association are generally based on analytical methods for indirect haplotype determination. Definition of haplotype blocks<sup><a reveal-id="DSR008C11" data-open="DSR008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with linked dedicated genetic variants is an effective tool, however phasing of long-distance segments is required and comes with several challenging issues and systemic limitations.<sup><a reveal-id="DSR008C11" data-open="DSR008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Due to the long genetic distances between MHC loci and the necessity for phasing of uneven distributed single nucleotide polymorphisms (SNPs), long-range haplotyping is hard to achieve.</p><p>Now, we hypothesize that applying targeted sequence capture to the human MHC, ideally combined and enhanced by HLA typing<sup><a reveal-id="DSR008C12  DSR008C14" data-open="DSR008C12  DSR008C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> in one single diagnostic sequencing experiment, provides additional information on transplant outcome.</p><p>Here we describe for the first time NimbleGen sequence capture on solid surface for targeted MHC resequencing by 454 sequencing in a clinical transplant setting. Moreover, to address the accuracy of the sequencing approach, we analysed approximately 1500 SNP markers mapping along the sequenced MHC region using genome-wide SNP arrays. For this study, we selected a patient who underwent unrelated HLA-matched allogeneic stem cell transplantation, the fully matched unrelated donor and also the parents of the patient. The patient developed acute GVHD after transplantation and died 1.5 months later.</p><p>Thus, we show that MHC resequencing, although applied to two HLA-identical genomes, reveals a large number of genetic differences and variants potentially linked to GVHD development. Various differences causing non-synonymous amino acid exchanges are discussed for their influence on GVHD development or drug efficacy and donor selection. In contrast to microarray-based SNP analysis or haplotype association studies, direct resequencing offers the near-complete information on genotypic differences for important transplantation outcome determinants at the single nucleotide resolution level.</p>                            <h2 scrollto-destination=7719742 id="7719742" class="section-title" data-legacy-id=s2>2. Material and methods</h2>
                            <h3 scrollto-destination=7719743 id="7719743" class="section-title" data-legacy-id=s2a>2.1 Clinical setting and experimental design</h3>
<p>We performed a comprehensive MHC resequencing approach and analysed a patient with acute myeloid leukaemia undergoing HLA-matched allergenic HSCT, the unmatched donor and the patient's parents. Initial donor selection was performed using molecular high-resolution HLA typing by Sanger sequencing. Written consent was obtained from all patients and donors as required by the institutional review board in accordance with the Declaration of Helsinki. The transplant was matched for 10 out of 10 alleles. The patient underwent a myeloablative busulfan/cyclophosphamide-based conditioning regime before transplantation. Short-course methotrexate combined with mycophenolate mofetil was used as a standard GVHD prophylaxis, due to intolerance to calcineurin inhibitors. On Day +21 after transplantation the patient developed acute GVHD Grade IV (skin IV, liver IV, gut II; Seattle criteria) and died 3 weeks later.</p><p>After microarray sequence capture and enrichment (SCE), MHC-targeted resequencing of 3.5 million base pairs for all four DNA samples was performed on a single Genome Sequencer instrument run. Additionally, microarray based genome-wide SNP analysis (Affymetrix Genome-wide Human SNP Nsp/Sty 6.0) was performed for donor and recipient for further MHC and genome-wide SNP analysis.</p>                            <h3 scrollto-destination=7719746 id="7719746" class="section-title" data-legacy-id=s2b>2.2 DNA samples</h3>
<p>Genomic DNA was isolated and purified from venous blood samples using the magnetic bead extracting MagNaPure Compact instrument (Roche Diagnostics, Mannheim, Germany) following the manufactu-rer's instructions. DNA concentration and quality were photometrically determined (BioPhotometer, Eppendorf, Hamburg, Germany).</p>                            <h3 scrollto-destination=7719748 id="7719748" class="section-title" data-legacy-id=s2c>2.3 High-resolution sequence-based HLA typing (SBT)</h3>
<p>The samples were typed initially for loci A, C, DQB1 and DRB1 using a commercial typing kit (S4 HLA-DRB1 and S3 HLA-A/C/DQB1; Protrans, Hockenheim, Germany) and HLA-B (AlleleSEQR HLA-B; Atria Genetics, South San Francisco, CA). PCR products were treated with ExoSAP-iT (Amersham Biosciences, Freiburg, Germany) to remove primers and dNTPs and purified by size exclusion chromatography with Sephadex<sup>®</sup> G-50 (Sigma-Aldrich, Steinheim, Germany). Direct cycle sequencing was performed in a 9800 Fast Thermal Cycler (Applied Biosystems, Foster City, CA) using the BigDyeTerminator™ v1.1 chemistry (Applied Biosystems) with the respective sequencing primers. The sequencing platform was an ABI Prism 3100 Genetic Analyzer (Applied Biosystems). The sequences representing frequent and well-known alleles were analysed by the Assign 3.5 software (Conexio Genomics) and presented in Table <a reveal-id="DSR008TB1" data-open="DSR008TB1" class="link link-reveal link-table xref-fig">1</a>. </p>                        <a id="7719750" scrollto-destination="7719750"></a>
<div content-id="DSR008TB1" class="table-modal"><div class="table-wrap"><div id="DSR008TB1" class="table-wrap-title" data-id="DSR008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>HLA typing results</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Parent 1 </td><td> </td><td> </td><td>Parent 2 </td><td> </td><td> </td></tr><tr><td>A* </td><td>29 </td><td>01 </td><td>A* </td><td>02 </td><td>32 </td></tr><tr><td>B* </td><td>44 </td><td>57 </td><td>B* </td><td>44 </td><td>14 </td></tr><tr><td>C* </td><td>06 </td><td>06 </td><td>C* </td><td>05 </td><td>08 </td></tr><tr><td>DRB1* </td><td>03 </td><td>07 </td><td>DRB1* </td><td>04 </td><td>07 </td></tr><tr><td>DQB1* </td><td>03 </td><td>03 </td><td>DQB1* </td><td>03 </td><td>02 </td></tr><tr><td>DPB1* </td><td>04 </td><td>04 </td><td>DPB1* </td><td>04 </td><td>04 </td></tr><tr><td colspan="3">Recipient </td><td colspan="3">Donor </td></tr><tr><td>A* </td><td>01:01 </td><td>02:01 </td><td>A* </td><td>01:01 </td><td>02:01 </td></tr><tr><td>B* </td><td>57:01 </td><td>44:02 </td><td>B* </td><td>57:01 </td><td>44:02 </td></tr><tr><td>C* </td><td>06:02 </td><td>05:01 </td><td>C* </td><td>06:02 </td><td>05:01 </td></tr><tr><td>DRB1* </td><td>07:01 </td><td>04:01 </td><td>DRB1* </td><td>07:01 </td><td>04:01 </td></tr><tr><td>DQB1* </td><td>03:03 </td><td>03:02 </td><td>DQB1* </td><td>03:03 </td><td>03:02 </td></tr><tr><td>DPB1* </td><td>04:01 </td><td>04:01 </td><td>DPB1* </td><td>04:01 </td><td>04:01 </td></tr></tbody></table></div><div content-id="DSR008TB1" class="table-modal"><table><tbody><tr><td>Parent 1 </td><td> </td><td> </td><td>Parent 2 </td><td> </td><td> </td></tr><tr><td>A* </td><td>29 </td><td>01 </td><td>A* </td><td>02 </td><td>32 </td></tr><tr><td>B* </td><td>44 </td><td>57 </td><td>B* </td><td>44 </td><td>14 </td></tr><tr><td>C* </td><td>06 </td><td>06 </td><td>C* </td><td>05 </td><td>08 </td></tr><tr><td>DRB1* </td><td>03 </td><td>07 </td><td>DRB1* </td><td>04 </td><td>07 </td></tr><tr><td>DQB1* </td><td>03 </td><td>03 </td><td>DQB1* </td><td>03 </td><td>02 </td></tr><tr><td>DPB1* </td><td>04 </td><td>04 </td><td>DPB1* </td><td>04 </td><td>04 </td></tr><tr><td colspan="3">Recipient </td><td colspan="3">Donor </td></tr><tr><td>A* </td><td>01:01 </td><td>02:01 </td><td>A* </td><td>01:01 </td><td>02:01 </td></tr><tr><td>B* </td><td>57:01 </td><td>44:02 </td><td>B* </td><td>57:01 </td><td>44:02 </td></tr><tr><td>C* </td><td>06:02 </td><td>05:01 </td><td>C* </td><td>06:02 </td><td>05:01 </td></tr><tr><td>DRB1* </td><td>07:01 </td><td>04:01 </td><td>DRB1* </td><td>07:01 </td><td>04:01 </td></tr><tr><td>DQB1* </td><td>03:03 </td><td>03:02 </td><td>DQB1* </td><td>03:03 </td><td>03:02 </td></tr><tr><td>DPB1* </td><td>04:01 </td><td>04:01 </td><td>DPB1* </td><td>04:01 </td><td>04:01 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*DNA based typing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719750" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR008TB1" class="table-wrap-title" data-id="DSR008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>HLA typing results</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Parent 1 </td><td> </td><td> </td><td>Parent 2 </td><td> </td><td> </td></tr><tr><td>A* </td><td>29 </td><td>01 </td><td>A* </td><td>02 </td><td>32 </td></tr><tr><td>B* </td><td>44 </td><td>57 </td><td>B* </td><td>44 </td><td>14 </td></tr><tr><td>C* </td><td>06 </td><td>06 </td><td>C* </td><td>05 </td><td>08 </td></tr><tr><td>DRB1* </td><td>03 </td><td>07 </td><td>DRB1* </td><td>04 </td><td>07 </td></tr><tr><td>DQB1* </td><td>03 </td><td>03 </td><td>DQB1* </td><td>03 </td><td>02 </td></tr><tr><td>DPB1* </td><td>04 </td><td>04 </td><td>DPB1* </td><td>04 </td><td>04 </td></tr><tr><td colspan="3">Recipient </td><td colspan="3">Donor </td></tr><tr><td>A* </td><td>01:01 </td><td>02:01 </td><td>A* </td><td>01:01 </td><td>02:01 </td></tr><tr><td>B* </td><td>57:01 </td><td>44:02 </td><td>B* </td><td>57:01 </td><td>44:02 </td></tr><tr><td>C* </td><td>06:02 </td><td>05:01 </td><td>C* </td><td>06:02 </td><td>05:01 </td></tr><tr><td>DRB1* </td><td>07:01 </td><td>04:01 </td><td>DRB1* </td><td>07:01 </td><td>04:01 </td></tr><tr><td>DQB1* </td><td>03:03 </td><td>03:02 </td><td>DQB1* </td><td>03:03 </td><td>03:02 </td></tr><tr><td>DPB1* </td><td>04:01 </td><td>04:01 </td><td>DPB1* </td><td>04:01 </td><td>04:01 </td></tr></tbody></table></div><div content-id="DSR008TB1" class="table-modal"><table><tbody><tr><td>Parent 1 </td><td> </td><td> </td><td>Parent 2 </td><td> </td><td> </td></tr><tr><td>A* </td><td>29 </td><td>01 </td><td>A* </td><td>02 </td><td>32 </td></tr><tr><td>B* </td><td>44 </td><td>57 </td><td>B* </td><td>44 </td><td>14 </td></tr><tr><td>C* </td><td>06 </td><td>06 </td><td>C* </td><td>05 </td><td>08 </td></tr><tr><td>DRB1* </td><td>03 </td><td>07 </td><td>DRB1* </td><td>04 </td><td>07 </td></tr><tr><td>DQB1* </td><td>03 </td><td>03 </td><td>DQB1* </td><td>03 </td><td>02 </td></tr><tr><td>DPB1* </td><td>04 </td><td>04 </td><td>DPB1* </td><td>04 </td><td>04 </td></tr><tr><td colspan="3">Recipient </td><td colspan="3">Donor </td></tr><tr><td>A* </td><td>01:01 </td><td>02:01 </td><td>A* </td><td>01:01 </td><td>02:01 </td></tr><tr><td>B* </td><td>57:01 </td><td>44:02 </td><td>B* </td><td>57:01 </td><td>44:02 </td></tr><tr><td>C* </td><td>06:02 </td><td>05:01 </td><td>C* </td><td>06:02 </td><td>05:01 </td></tr><tr><td>DRB1* </td><td>07:01 </td><td>04:01 </td><td>DRB1* </td><td>07:01 </td><td>04:01 </td></tr><tr><td>DQB1* </td><td>03:03 </td><td>03:02 </td><td>DQB1* </td><td>03:03 </td><td>03:02 </td></tr><tr><td>DPB1* </td><td>04:01 </td><td>04:01 </td><td>DPB1* </td><td>04:01 </td><td>04:01 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*DNA based typing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719750" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7719751 id="7719751" class="section-title" data-legacy-id=s2d>2.4 Affymetrix genome-wide SNP analysis</h3>
<p>For genome-wide SNP determination, an Affymetrix Genome-wide Human SNP Array (Nsp/Sty Assay Kit 5.0/6.0; carrying 909,622 SNP markers) processing with sample preparation, digestion, ligation, amplification, labelling, hybridization, staining and scanning were conducted according to the manufacturer's instructions. For donor and recipient, 250 ng of double-stranded genomic DNA starting material were digested with <em>Nsp I</em> and <em>Sty I</em> (New England BioLabs GmbH, Frankfurt, Germany). Hybridizations were conducted with one replicate of all times and treatments concurrently. Each array image was visually screened to count for general signal quality and then submitted to Genotyping Console 4.0 (Affymetrix). Genotype calling was determined by the Birdseed version 2.0 using 90 HapMap reference samples for cluster analysis.</p>                            <h3 scrollto-destination=7719753 id="7719753" class="section-title" data-legacy-id=s2e>2.5 Sequence capture and enrichment</h3>
<p>Design and production of a GS FLX Titanium optimized sequence capture array (385k) was done at NimbleGen (Roche NimbleGen, Madison, WI) targeting 5 Mb reference sequence (human hg18) spanning the MHC complex on chromosome 6 (NCBI: chr6 NGS primary_target_region 29,594,756-33,046,546). Targeted sequence capture, with pre-capture ligation-mediated PCR (LM-PCR), hybridization, washing, elution and quantitative PCR (qPCR) to assess capture success, was performed according to the manufacturer's instructions. According to ENSEMBL v49 release (HG 18 in UCSC nomenclature) reference genome, the final target bases covering the complete MHC class I, II and III regions were defined to be 3 451 791 bp; of those 2 922 962, target bases (84.7%) were covered by capture oligonucleotides as defined by NimbleGens default settings for probe selection. 528 829 bp (15.3%) of the initial target region were omitted due to reasons of specificity and uniqueness.</p>                            <h3 scrollto-destination=7719755 id="7719755" class="section-title" data-legacy-id=s2f>2.6 MHC genotyping by Roche/454 Genome Sequencer FLX sequencing</h3>
<p>Using the titanium-optimized sequence, capture protocol provides the captured DNA ready for use in 454 emPCR because required adaptor sequences are integrated during the LM-PCR step. The four captured DNA-sequencing libraries were prepared separately and each one loaded on one of the four-lane gasket PicoTiterPlate device (PTP; 70×75 mm; Roche/454), respectively. Each library was quantitated by Pico-Green<sup>®</sup> (Quant-iT™, Molecular Probes, Invitrogen), diluted to 1 × 10<sup>5</sup> molecules/µl and used for emPCR at a ratio of 0.1 copies of library fragments per DNA capture bead. After the DNA capture bead recovery, DNA capture bead enrichment, to separate DNA-carrying beads from non-DNA carrying beads, was performed. Finally, we loaded 490 000–690 000 then single-strand DNA carrying beads on the PTP. The sequencing itself was performed on a Genome Sequencer FLX system using titanium chemistry and standard Roche 454 protocols (Lib-L SV; Rev. Jan 2010). Sequence information is available on NCBI (release on 2011-11-24; Accession nb: SRP004546).</p>                            <h3 scrollto-destination=7719757 id="7719757" class="section-title" data-legacy-id=s2g>2.7 Data analysis</h3>
                            <h4 scrollto-destination=7719758 id="7719758" class="section-title" data-legacy-id=s2g1>2.7.1 Image processing to gene annotation</h4>
<p>Image processing and base calling was performed with GS FLX software (Roche/454 Life Sciences). For 100% identity contiguous sequence production, the GS Reference Mapper (Roche/454 Life Sciences) was used. The MHC Haplotype Project with the analysis of eight MHC haplotypes provided a comprehensive reference sequence database, which is consequently used for variation assessments in the present study<sup><a reveal-id="DSR008C4" data-open="DSR008C4" class="link link-ref link-reveal xref-bibr">4</a></sup>. Haplotype (A3-B7-DR15) of the cell line PGF was designated as the new MHC reference sequence, representing the reference sequence for the used capture oligonucleotide design (NCBI human HG 18). Genes were annotated according to the Vertebrate Genome Annotation database (<a class="link link-uri" href="http://www.VEGA.sanger.ac.uk">http://www.VEGA.sanger.ac.uk</a>) on the basis of the referenced PGF cell line.</p>                            <h4 scrollto-destination=7719760 id="7719760" class="section-title" data-legacy-id=s2g2>2.7.2 Gene annotation to variant analysis</h4>
<p>A series of Perl scripts was developed and applied to determine the type (substitute, insertion, deletion, synonymous or non-synonymous) and quantity (number of reads per variant) of variation between the four samples and the reference cell line and in between donor and recipient. A simple, but clear-cut ratio of the number of matched or mismatched variants in relation to the total number of sequenced base pairs defines the similarity of actual or potential donor and recipient MHC, reliably enabling fast qualification of possible donor-recipient matches with a single quality factor over and above the obligatory HLA match.</p>                            <h3 scrollto-destination=7719762 id="7719762" class="section-title" data-legacy-id=s2h>2.8 Variant analysis to graft-versus-host or disease susceptibility</h3>
<p>Additional series of Perl scripts allow to search and compare for known disease associations or other valued medical information. Parameters such as intergenic, intronic, coding/non-coding, synonymous/non-synonymous and coverage as well as prevalence are used to define a list of SNPs to start batch analysis in NCBI's OMIM or related databases. Once the sequence data were generated by the sequencing instrument, it took just 2 h for the data analysis programs to analyse the raw data and generate report variation and disease association information. A summary table with all prominent types of SNPs in resequenced regions with at least 20-fold coverage is presented in Table <a reveal-id="DSR008TB2" data-open="DSR008TB2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="7719764" scrollto-destination="7719764"></a>
<div content-id="DSR008TB2" class="table-modal"><div class="table-wrap"><div id="DSR008TB2" class="table-wrap-title" data-id="DSR008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>MHC resequencing: summary of insertions, deletions and mismatches for Parents 1 and 2, donor and recipient compared with the reference cell line PGF</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th> </th><th>Parent 1 </th><th>Parent 2 </th><th>Donor </th><th>Recipient </th><th>Sum </th><th>(%) </th></tr></thead><tbody><tr><td colspan="7">Exon </td></tr><tr><td> Insertion </td><td>18 </td><td>28 </td><td>28 </td><td>23 </td><td>97 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>14 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>548 </td><td>518 </td><td>459 </td><td>316 </td><td>1841 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>570 (10.2) </td><td>550 (13.3) </td><td>490 (10.2) </td><td>342 (7.3) </td><td>1952 </td><td>(100) </td></tr><tr><td colspan="7">Intron </td></tr><tr><td> Insertion </td><td>122 </td><td>95 </td><td>110 </td><td>107 </td><td>434 </td><td>(7.3) </td></tr><tr><td> Deletion </td><td>16 </td><td>12 </td><td>11 </td><td>14 </td><td>53 </td><td>(0.9) </td></tr><tr><td> Mismatch </td><td>1664 </td><td>1109 </td><td>1406 </td><td>1278 </td><td>5457 </td><td>(91.8) </td></tr><tr><td> Sum (%) </td><td>1802 (32.3) </td><td>1216 (29.4) </td><td>1527 (31.9) </td><td>1399 (30.1) </td><td>5944 </td><td>(100) </td></tr><tr><td colspan="7">Intergenic </td></tr><tr><td> Insertion </td><td>138 </td><td>134 </td><td>137 </td><td>152 </td><td>561 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>17 </td><td>14 </td><td>22 </td><td>26 </td><td>79 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>3050 </td><td>2225 </td><td>2605 </td><td>2740 </td><td>10 620 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>3205 (57.5) </td><td>2373 (57.3) </td><td>2764 (57.8) </td><td>2918 (62.6) </td><td>11 260 </td><td>(100) </td></tr><tr><td>Total </td><td>5577 </td><td>4139 </td><td>4781 </td><td>4659 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th> </th><th>Parent 1 </th><th>Parent 2 </th><th>Donor </th><th>Recipient </th><th>Sum </th><th>(%) </th></tr></thead><tbody><tr><td colspan="7">Exon </td></tr><tr><td> Insertion </td><td>18 </td><td>28 </td><td>28 </td><td>23 </td><td>97 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>14 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>548 </td><td>518 </td><td>459 </td><td>316 </td><td>1841 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>570 (10.2) </td><td>550 (13.3) </td><td>490 (10.2) </td><td>342 (7.3) </td><td>1952 </td><td>(100) </td></tr><tr><td colspan="7">Intron </td></tr><tr><td> Insertion </td><td>122 </td><td>95 </td><td>110 </td><td>107 </td><td>434 </td><td>(7.3) </td></tr><tr><td> Deletion </td><td>16 </td><td>12 </td><td>11 </td><td>14 </td><td>53 </td><td>(0.9) </td></tr><tr><td> Mismatch </td><td>1664 </td><td>1109 </td><td>1406 </td><td>1278 </td><td>5457 </td><td>(91.8) </td></tr><tr><td> Sum (%) </td><td>1802 (32.3) </td><td>1216 (29.4) </td><td>1527 (31.9) </td><td>1399 (30.1) </td><td>5944 </td><td>(100) </td></tr><tr><td colspan="7">Intergenic </td></tr><tr><td> Insertion </td><td>138 </td><td>134 </td><td>137 </td><td>152 </td><td>561 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>17 </td><td>14 </td><td>22 </td><td>26 </td><td>79 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>3050 </td><td>2225 </td><td>2605 </td><td>2740 </td><td>10 620 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>3205 (57.5) </td><td>2373 (57.3) </td><td>2764 (57.8) </td><td>2918 (62.6) </td><td>11 260 </td><td>(100) </td></tr><tr><td>Total </td><td>5577 </td><td>4139 </td><td>4781 </td><td>4659 </td><td> </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719764" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR008TB2" class="table-wrap-title" data-id="DSR008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>MHC resequencing: summary of insertions, deletions and mismatches for Parents 1 and 2, donor and recipient compared with the reference cell line PGF</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th> </th><th>Parent 1 </th><th>Parent 2 </th><th>Donor </th><th>Recipient </th><th>Sum </th><th>(%) </th></tr></thead><tbody><tr><td colspan="7">Exon </td></tr><tr><td> Insertion </td><td>18 </td><td>28 </td><td>28 </td><td>23 </td><td>97 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>14 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>548 </td><td>518 </td><td>459 </td><td>316 </td><td>1841 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>570 (10.2) </td><td>550 (13.3) </td><td>490 (10.2) </td><td>342 (7.3) </td><td>1952 </td><td>(100) </td></tr><tr><td colspan="7">Intron </td></tr><tr><td> Insertion </td><td>122 </td><td>95 </td><td>110 </td><td>107 </td><td>434 </td><td>(7.3) </td></tr><tr><td> Deletion </td><td>16 </td><td>12 </td><td>11 </td><td>14 </td><td>53 </td><td>(0.9) </td></tr><tr><td> Mismatch </td><td>1664 </td><td>1109 </td><td>1406 </td><td>1278 </td><td>5457 </td><td>(91.8) </td></tr><tr><td> Sum (%) </td><td>1802 (32.3) </td><td>1216 (29.4) </td><td>1527 (31.9) </td><td>1399 (30.1) </td><td>5944 </td><td>(100) </td></tr><tr><td colspan="7">Intergenic </td></tr><tr><td> Insertion </td><td>138 </td><td>134 </td><td>137 </td><td>152 </td><td>561 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>17 </td><td>14 </td><td>22 </td><td>26 </td><td>79 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>3050 </td><td>2225 </td><td>2605 </td><td>2740 </td><td>10 620 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>3205 (57.5) </td><td>2373 (57.3) </td><td>2764 (57.8) </td><td>2918 (62.6) </td><td>11 260 </td><td>(100) </td></tr><tr><td>Total </td><td>5577 </td><td>4139 </td><td>4781 </td><td>4659 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th> </th><th>Parent 1 </th><th>Parent 2 </th><th>Donor </th><th>Recipient </th><th>Sum </th><th>(%) </th></tr></thead><tbody><tr><td colspan="7">Exon </td></tr><tr><td> Insertion </td><td>18 </td><td>28 </td><td>28 </td><td>23 </td><td>97 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>4 </td><td>4 </td><td>3 </td><td>3 </td><td>14 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>548 </td><td>518 </td><td>459 </td><td>316 </td><td>1841 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>570 (10.2) </td><td>550 (13.3) </td><td>490 (10.2) </td><td>342 (7.3) </td><td>1952 </td><td>(100) </td></tr><tr><td colspan="7">Intron </td></tr><tr><td> Insertion </td><td>122 </td><td>95 </td><td>110 </td><td>107 </td><td>434 </td><td>(7.3) </td></tr><tr><td> Deletion </td><td>16 </td><td>12 </td><td>11 </td><td>14 </td><td>53 </td><td>(0.9) </td></tr><tr><td> Mismatch </td><td>1664 </td><td>1109 </td><td>1406 </td><td>1278 </td><td>5457 </td><td>(91.8) </td></tr><tr><td> Sum (%) </td><td>1802 (32.3) </td><td>1216 (29.4) </td><td>1527 (31.9) </td><td>1399 (30.1) </td><td>5944 </td><td>(100) </td></tr><tr><td colspan="7">Intergenic </td></tr><tr><td> Insertion </td><td>138 </td><td>134 </td><td>137 </td><td>152 </td><td>561 </td><td>(5.0) </td></tr><tr><td> Deletion </td><td>17 </td><td>14 </td><td>22 </td><td>26 </td><td>79 </td><td>(0.7) </td></tr><tr><td> Mismatch </td><td>3050 </td><td>2225 </td><td>2605 </td><td>2740 </td><td>10 620 </td><td>(94.3) </td></tr><tr><td> Sum (%) </td><td>3205 (57.5) </td><td>2373 (57.3) </td><td>2764 (57.8) </td><td>2918 (62.6) </td><td>11 260 </td><td>(100) </td></tr><tr><td>Total </td><td>5577 </td><td>4139 </td><td>4781 </td><td>4659 </td><td> </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719764" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=7719765 id="7719765" class="section-title" data-legacy-id=s3>3. Results and Discussion</h2>
                            <h3 scrollto-destination=7719766 id="7719766" class="section-title" data-legacy-id=s3a>3.1 Efficiency of the enrichment process</h3>
<p>When estimating the efficiency of the enrichment process, the four different probes (NSC0237, NSC0247, NSC0268 and NSC0272) included in the sequence capture assay were quantified by qPCR before and after the enrichment process revealing that ∼337-fold enrichment was achieved (ranging from 164- to 851-fold). This is clearly above the quality standard of 100-fold enrichment, which should be realized when using microarray-based sequence capture systems.</p>                            <h3 scrollto-destination=7719768 id="7719768" class="section-title" data-legacy-id=s3b>3.2 Specificity of the enrichment process</h3>
<p>According to PGF reference alignment observations, a total of 706 431 reads representing 78% of the 908 152 high-quality reads produced were located inside the specified MHC target region. 61 455 reads (8.7%) were rejected due to length (&lt;50 b), repetitive or chimeric characteristics.</p>                            <h3 scrollto-destination=7719770 id="7719770" class="section-title" data-legacy-id=s3c>3.3 454 pyrosequencing</h3>
<p>Libraries 1–4 (MHC Parent 1, MHC Parent 2, MHC donor, MHC recipient) showed a post-emPCR and recovery enrichment efficiency of 2.5, 4.5, 5.7 and 5.8%. As a consequence 490 450, 544 500, 682 950 and 690 900 DNA-capture beads per lane were loaded to the GS-PTP (PicoTiterPlate) regions 1–4. The number of individual read sequences which passed internal quality control filters, with a mean read length of 350 bp, were 223 005, 225 385, 226 501 and 234 017, respectively, for each region.</p>                            <h3 scrollto-destination=7719772 id="7719772" class="section-title" data-legacy-id=s3d>3.4 MHC-resequencing coverage</h3>
<p>In total, we were able to annotate 273 genes to our MHC sequence data. For samples 1–4 (Parent 1, Parent 2, donor and recipient) 165 956, 167 376, 178 588 and 194 511 reads were MHC matched (Table <a reveal-id="DSR008TB3" data-open="DSR008TB3" class="link link-reveal link-table xref-fig">3</a>). For contig analysis (parameters: 90% homology; three reads minimum for base pair position), a sum of 1246 (3 009 032 bp, 87% of target), 1287 (2 931 090 bp, 85% of target), 1073 (3 036 931 bp (88% of target) and 1129 (3 048 429 bp, 88% of target) contigs with on average 140, 131, 174 and 179 reads per contig were generated. In total, 4735 contigs with 12 025 482 bp and 706 431 reads were submitted for further analysis. On average, 13% of the targeted 3.5 Mb MHC were missed by this resequencing approach. For an annotated gene list of 273 genes (pseudogenes included) from the telomeric gene DAQB-12N14.5 (MHC-extended class I) to the centromeric gene BRD2 (MHC-class II) for which we have full sequence, the coverage is on average 17-, 18-, 19- and 20-folds for each position for Parent 1, Parent 2, donor and recipient, respectively (Table <a reveal-id="DSR008TB3" data-open="DSR008TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DSR008F1" data-open="DSR008F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="7719774" scrollto-destination="7719774"></a>
<div content-id="DSR008TB3" class="table-modal"><div class="table-wrap"><div id="DSR008TB3" class="table-wrap-title" data-id="DSR008TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of 454 MHC resequencing data after sequence capture microarray enrichment for parents, donor and recipient</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="7">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th colspan="2">454 genome sequencer 1× instrument run (titanium chemistry) </th><th>Contigs (<em>n</em>)<sup>a</sup> </th><th>Reads (<em>n</em>) passed filter (%) </th><th>Reads (<em>n</em>) p. Contig (<em>n</em>) </th><th>Coverage read length (bp) </th><th>MHC covered (%)<sup>b</sup> </th><th>MHC not covered (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Region 1 </td><td>Parent 1 </td><td>1246 </td><td>174 382 (66%) </td><td>140 </td><td>17× (342) </td><td>3 009 032 (87%) </td><td>438 685 (13%) </td></tr><tr><td>Region 2 </td><td>Parent 2 </td><td>1287 </td><td>175 749 (66%) </td><td>137 </td><td>18× (347) </td><td>2 931 090 (85%) </td><td>518 853 (15%) </td></tr><tr><td>Region 3 </td><td>Donor </td><td>1073 </td><td>186 819 (58%) </td><td>174 </td><td>19× (352) </td><td>3 036 931 (88%) </td><td>410 878 (12%) </td></tr><tr><td>Region 4 </td><td>Recipient </td><td>1129 </td><td>202 358 (62%) </td><td>179 </td><td>20× (340) </td><td>3 048 429 (88%) </td><td>400 206 (12%) </td></tr><tr><td> </td><td>Sum </td><td>4735 </td><td>739 311 </td><td> </td><td> </td><td>12 025 485 </td><td>1 768 (623) </td></tr><tr><td> </td><td>Mean </td><td>1184 </td><td>184 827 (63%) </td><td>157 </td><td>18.5× (345) </td><td>3 006 371 (87%) </td><td>442 156 (13%) </td></tr></tbody></table></div><div content-id="DSR008TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="7">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th colspan="2">454 genome sequencer 1× instrument run (titanium chemistry) </th><th>Contigs (<em>n</em>)<sup>a</sup> </th><th>Reads (<em>n</em>) passed filter (%) </th><th>Reads (<em>n</em>) p. Contig (<em>n</em>) </th><th>Coverage read length (bp) </th><th>MHC covered (%)<sup>b</sup> </th><th>MHC not covered (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Region 1 </td><td>Parent 1 </td><td>1246 </td><td>174 382 (66%) </td><td>140 </td><td>17× (342) </td><td>3 009 032 (87%) </td><td>438 685 (13%) </td></tr><tr><td>Region 2 </td><td>Parent 2 </td><td>1287 </td><td>175 749 (66%) </td><td>137 </td><td>18× (347) </td><td>2 931 090 (85%) </td><td>518 853 (15%) </td></tr><tr><td>Region 3 </td><td>Donor </td><td>1073 </td><td>186 819 (58%) </td><td>174 </td><td>19× (352) </td><td>3 036 931 (88%) </td><td>410 878 (12%) </td></tr><tr><td>Region 4 </td><td>Recipient </td><td>1129 </td><td>202 358 (62%) </td><td>179 </td><td>20× (340) </td><td>3 048 429 (88%) </td><td>400 206 (12%) </td></tr><tr><td> </td><td>Sum </td><td>4735 </td><td>739 311 </td><td> </td><td> </td><td>12 025 485 </td><td>1 768 (623) </td></tr><tr><td> </td><td>Mean </td><td>1184 </td><td>184 827 (63%) </td><td>157 </td><td>18.5× (345) </td><td>3 006 371 (87%) </td><td>442 156 (13%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Contigs match parameter: 90% homology, minimum three reads for a given position.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Per cent of complete target length 3 451 791 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719774" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR008TB3" class="table-wrap-title" data-id="DSR008TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of 454 MHC resequencing data after sequence capture microarray enrichment for parents, donor and recipient</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="7">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th colspan="2">454 genome sequencer 1× instrument run (titanium chemistry) </th><th>Contigs (<em>n</em>)<sup>a</sup> </th><th>Reads (<em>n</em>) passed filter (%) </th><th>Reads (<em>n</em>) p. Contig (<em>n</em>) </th><th>Coverage read length (bp) </th><th>MHC covered (%)<sup>b</sup> </th><th>MHC not covered (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Region 1 </td><td>Parent 1 </td><td>1246 </td><td>174 382 (66%) </td><td>140 </td><td>17× (342) </td><td>3 009 032 (87%) </td><td>438 685 (13%) </td></tr><tr><td>Region 2 </td><td>Parent 2 </td><td>1287 </td><td>175 749 (66%) </td><td>137 </td><td>18× (347) </td><td>2 931 090 (85%) </td><td>518 853 (15%) </td></tr><tr><td>Region 3 </td><td>Donor </td><td>1073 </td><td>186 819 (58%) </td><td>174 </td><td>19× (352) </td><td>3 036 931 (88%) </td><td>410 878 (12%) </td></tr><tr><td>Region 4 </td><td>Recipient </td><td>1129 </td><td>202 358 (62%) </td><td>179 </td><td>20× (340) </td><td>3 048 429 (88%) </td><td>400 206 (12%) </td></tr><tr><td> </td><td>Sum </td><td>4735 </td><td>739 311 </td><td> </td><td> </td><td>12 025 485 </td><td>1 768 (623) </td></tr><tr><td> </td><td>Mean </td><td>1184 </td><td>184 827 (63%) </td><td>157 </td><td>18.5× (345) </td><td>3 006 371 (87%) </td><td>442 156 (13%) </td></tr></tbody></table></div><div content-id="DSR008TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="7">3.5 Mb human MHC resequencing<hr /> </th></tr><tr><th colspan="2">454 genome sequencer 1× instrument run (titanium chemistry) </th><th>Contigs (<em>n</em>)<sup>a</sup> </th><th>Reads (<em>n</em>) passed filter (%) </th><th>Reads (<em>n</em>) p. Contig (<em>n</em>) </th><th>Coverage read length (bp) </th><th>MHC covered (%)<sup>b</sup> </th><th>MHC not covered (%)<sup>b</sup> </th></tr></thead><tbody><tr><td>Region 1 </td><td>Parent 1 </td><td>1246 </td><td>174 382 (66%) </td><td>140 </td><td>17× (342) </td><td>3 009 032 (87%) </td><td>438 685 (13%) </td></tr><tr><td>Region 2 </td><td>Parent 2 </td><td>1287 </td><td>175 749 (66%) </td><td>137 </td><td>18× (347) </td><td>2 931 090 (85%) </td><td>518 853 (15%) </td></tr><tr><td>Region 3 </td><td>Donor </td><td>1073 </td><td>186 819 (58%) </td><td>174 </td><td>19× (352) </td><td>3 036 931 (88%) </td><td>410 878 (12%) </td></tr><tr><td>Region 4 </td><td>Recipient </td><td>1129 </td><td>202 358 (62%) </td><td>179 </td><td>20× (340) </td><td>3 048 429 (88%) </td><td>400 206 (12%) </td></tr><tr><td> </td><td>Sum </td><td>4735 </td><td>739 311 </td><td> </td><td> </td><td>12 025 485 </td><td>1 768 (623) </td></tr><tr><td> </td><td>Mean </td><td>1184 </td><td>184 827 (63%) </td><td>157 </td><td>18.5× (345) </td><td>3 006 371 (87%) </td><td>442 156 (13%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Contigs match parameter: 90% homology, minimum three reads for a given position.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Per cent of complete target length 3 451 791 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719774" target="_blank">View Large</a></div></div>                        <a id="7719775" scrollto-destination="7719775"></a>
<div id="DSR008F1" data-id="DSR008F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00801.jpeg?Expires=2147483647&Signature=PrRsX1Lada2Xw0o1ravV9epjLNTHAMIvGBK00FeUBY6jevD7pSaKKC38qfLXvdoSGqLb37BZgqCJM8AbLKbVtxx2arwcwEhapsoAeHgFOz-~gaG6ziGXMEOorBapb3gqlwaggKN-tTLid3oCaDbvL2bdMin9yV05KY~K7BsUkfbd-GpBUsaxcZvK19MBOBOHdmWSub4qDtfsgmfIHCiMtDcuXjVbIEAiludHLyKdvR64ym94~iqBldM573OVgv6WOdn4NafZzoyX9RZXhl61I2Zdf1bzmKVT~zD1sDuNNFbB79UZaEPq3cxlrUMAzgx-tNn5HkA1ZssSkduKX6uGdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results for parents, donor and recipient compared with annotated gene loci. Graphs are representing at least 3-fold coverage (present = 1) or gaps (absent = 0) for the complete 3.5 Mb spanning MHC region of the four sequenced DNA samples (P1, parent 1), (P2, parent 2), (D, donor) and (R, recipient). Gaps present in P1 to R are a consequence of repeat rich regions and poor sequence capture or failed amplification. Graph for reference cell line PGF show known annotated gene loci in black (gene annotation = 1)." path-from-xml="dsr00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719775/dsr00801.jpeg" path-from-xml="dsr00801.jpeg" target="_blank">View large</a><a section="7719775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00801.jpeg?Expires=2147483647&Signature=dk7r4Sj4AqFGkNIQznOnAwg98mGqnVs9RbAp2zb8gEDf4h0ZJSaXiTc7nSJz~9Fe4yfM8nuSQDm2J83OqJmwGk1u8Ec2gsmT75UgK6Oc6WhXn6raRjKZpRGuLrWWBNKirjB1K4SqvPEByL6YiGVB4Td-XwbOHrTt88WmivzLCwE~iUn6i-ObG~oOV0kTQCrdz2hXz~LBOclLo2iB4tHMWNp6pHKBMaA0NFW1CGLFgl6gFJcHWAfyHnsUBIcCmVuim4HyP00PCFNEVzoD9qZCpqAqfB~daPCaeP~by-eQPKg69heA0Rho0YjbiVsyNf-gJNfKOeKzxnUCVOh0eFUqIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719775&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results for parents, donor and recipient compared with annotated gene loci. Graphs are representing at least 3-fold coverage (present = 1) or gaps (absent = 0) for the complete 3.5 Mb spanning MHC region of the four sequenced DNA samples (P1, parent 1), (P2, parent 2), (D, donor) and (R, recipient). Gaps present in P1 to R are a consequence of repeat rich regions and poor sequence capture or failed amplification. Graph for reference cell line PGF show known annotated gene loci in black (gene annotation = 1).</p></div></div><div content-id="DSR008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00801.jpeg?Expires=2147483647&Signature=PrRsX1Lada2Xw0o1ravV9epjLNTHAMIvGBK00FeUBY6jevD7pSaKKC38qfLXvdoSGqLb37BZgqCJM8AbLKbVtxx2arwcwEhapsoAeHgFOz-~gaG6ziGXMEOorBapb3gqlwaggKN-tTLid3oCaDbvL2bdMin9yV05KY~K7BsUkfbd-GpBUsaxcZvK19MBOBOHdmWSub4qDtfsgmfIHCiMtDcuXjVbIEAiludHLyKdvR64ym94~iqBldM573OVgv6WOdn4NafZzoyX9RZXhl61I2Zdf1bzmKVT~zD1sDuNNFbB79UZaEPq3cxlrUMAzgx-tNn5HkA1ZssSkduKX6uGdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results for parents, donor and recipient compared with annotated gene loci. Graphs are representing at least 3-fold coverage (present = 1) or gaps (absent = 0) for the complete 3.5 Mb spanning MHC region of the four sequenced DNA samples (P1, parent 1), (P2, parent 2), (D, donor) and (R, recipient). Gaps present in P1 to R are a consequence of repeat rich regions and poor sequence capture or failed amplification. Graph for reference cell line PGF show known annotated gene loci in black (gene annotation = 1)." path-from-xml="dsr00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719775/dsr00801.jpeg" path-from-xml="dsr00801.jpeg" target="_blank">View large</a><a section="7719775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00801.jpeg?Expires=2147483647&Signature=dk7r4Sj4AqFGkNIQznOnAwg98mGqnVs9RbAp2zb8gEDf4h0ZJSaXiTc7nSJz~9Fe4yfM8nuSQDm2J83OqJmwGk1u8Ec2gsmT75UgK6Oc6WhXn6raRjKZpRGuLrWWBNKirjB1K4SqvPEByL6YiGVB4Td-XwbOHrTt88WmivzLCwE~iUn6i-ObG~oOV0kTQCrdz2hXz~LBOclLo2iB4tHMWNp6pHKBMaA0NFW1CGLFgl6gFJcHWAfyHnsUBIcCmVuim4HyP00PCFNEVzoD9qZCpqAqfB~daPCaeP~by-eQPKg69heA0Rho0YjbiVsyNf-gJNfKOeKzxnUCVOh0eFUqIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719775&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results for parents, donor and recipient compared with annotated gene loci. Graphs are representing at least 3-fold coverage (present = 1) or gaps (absent = 0) for the complete 3.5 Mb spanning MHC region of the four sequenced DNA samples (P1, parent 1), (P2, parent 2), (D, donor) and (R, recipient). Gaps present in P1 to R are a consequence of repeat rich regions and poor sequence capture or failed amplification. Graph for reference cell line PGF show known annotated gene loci in black (gene annotation = 1).</p></div></div>                            <h3 scrollto-destination=7719776 id="7719776" class="section-title" data-legacy-id=s3e>3.5 MHC-resequencing variation analysis</h3>
<p>When comparing the sequence data of the four individual MHC regions to PGF, a significant number of known (dbSNP included) and new (non-dbSNP included) variant positions were detected. For parent 1 5 577, for parent 2 4139, for donor 4781 and for the bone marrow transplanted patient 4659, variant positions were detected. Among these 6% are insertions, 1% are deletions and 93% represent substitutions. An estimated 10% of all detected variants affiliate to exonic, 31% to intronic and 59% to intergenic regions. The complete numbers are shown in Table <a reveal-id="DSR008TB1" data-open="DSR008TB1" class="link link-reveal link-table xref-fig">1</a>.</p><p>Comparing the MHC sequences of donor and recipient with the reference PGF cell line, in sum 32 586 variant positions were detected with a 3-fold coverage. Analysing donor and recipient sequences directly to each other, with the restriction to 10- or 20-fold coverage these numbers drop to 15 699 and 4596, respectively. Taking the mean of 3-, 10- and 20-fold coverage numbers, these variant positions are equally distributed and located 59% to intergenic, 7% to exonic and 34% to intronic regions.</p><p>Now, in the case of comparing the MHC sequences of donor and recipient directly to each other, 3025 different positions were observed with coverage greater or equal to 20-fold. Additional 1517 positions are shared among donor and recipient, but differ with PGF sequences. Comparing Parent 1 to recipient, 3639 variant positions differ. Comparing parent 2 and recipient showed 5790 differences. Analogous, 1574 and 2276 positions shared differ in comparison to the referenced cell line. Figure <a reveal-id="DSR008F2" data-open="DSR008F2" class="link link-reveal link-table xref-fig">2</a> shows the graphical interpretation of similarities and dissimilarities among possible donor and recipient genotypes. Although, being far from perfect, donor versus recipient correlation compared with recipient versus parent correlation show the influence of the HLA match. Ultra-deep sequencing would be necessary to bring the data closer to the homogenous or heterogenous distribution values (50, 100%). </p>                        <a id="7719780" scrollto-destination="7719780"></a>
<div id="DSR008F2" data-id="DSR008F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00802.jpeg?Expires=2147483647&Signature=psrijgJwb-qXd7im7yvh4r4n9K6VrPcnOY29JJbjCziUxipeYjwP90eySq~ZGhTRrn5FYFyMDfqaqaFpqVOerRV-L8kcnjIhOgII-R-azuHTMCBtA2NAbL4gKt87f06Tl6T9CSsHiVGPlYWZtNHYUUEz3bfSB1uVn-L2NUPbR7lPRJJR35pZqlQfqvxuA8~g5cEAamT7RJ2T6Vg-kADzejZYdY-qmYlev~~7lbaHSwva0CDZmW57NQzhIF3~-C8uKnFUeDsnQy-hyPsSKGUw6qbxPeOmtfolqOigcqzxUAoqPOYQ-JbUJAVqFhoxI7IFXFyLYkvnR1~IBj2bg97vbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results. Variant correlation of parents, donor and recipient. (A–C) Correlating all detected variant positions in percentage of reads per position for two individuals visualize the degree of sequence match. A clear reduction of variant positions is desplayed in (C). (D) Cumulative coverage in percentage of target bases (blue p1, green p2, lila donor, red recipient)." path-from-xml="dsr00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719780/dsr00802.jpeg" path-from-xml="dsr00802.jpeg" target="_blank">View large</a><a section="7719780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00802.jpeg?Expires=2147483647&Signature=0CdRRWa1JAxvczpaM3~xuH3H~NlwHwyYaNYZki9sI9aWqp5FyEbI8yYvs4F920Qsj3YPoxvSu~EOiUs-cGrQFRWlMsN4SlZoJ3n03-wVDouQalTYlIeMvatnaF86EiWEqpt~A4ou1ZatOHprE3cVRwMNpm3jXklkZEkUKFp~FIB-4FxI0DpRWjq1sHtF-Ag56iVC-C~UJrVUSXx3INiWmq5YheQZINjP1d52xDPX2es5s-SFOZ0yOoA1itQnKzh-NYPu93i61p2NpbrJcU00Hx8-PniZizwVAYiFv8ey5e4oaPztWHkHUPzIDpH54v-cBFzqWGtwvRkfRzARI2l51w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719780&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results. Variant correlation of parents, donor and recipient. (A–C) Correlating all detected variant positions in percentage of reads per position for two individuals visualize the degree of sequence match. A clear reduction of variant positions is desplayed in (C). (D) Cumulative coverage in percentage of target bases (blue p1, green p2, lila donor, red recipient).</p></div></div><div content-id="DSR008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00802.jpeg?Expires=2147483647&Signature=psrijgJwb-qXd7im7yvh4r4n9K6VrPcnOY29JJbjCziUxipeYjwP90eySq~ZGhTRrn5FYFyMDfqaqaFpqVOerRV-L8kcnjIhOgII-R-azuHTMCBtA2NAbL4gKt87f06Tl6T9CSsHiVGPlYWZtNHYUUEz3bfSB1uVn-L2NUPbR7lPRJJR35pZqlQfqvxuA8~g5cEAamT7RJ2T6Vg-kADzejZYdY-qmYlev~~7lbaHSwva0CDZmW57NQzhIF3~-C8uKnFUeDsnQy-hyPsSKGUw6qbxPeOmtfolqOigcqzxUAoqPOYQ-JbUJAVqFhoxI7IFXFyLYkvnR1~IBj2bg97vbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results. Variant correlation of parents, donor and recipient. (A–C) Correlating all detected variant positions in percentage of reads per position for two individuals visualize the degree of sequence match. A clear reduction of variant positions is desplayed in (C). (D) Cumulative coverage in percentage of target bases (blue p1, green p2, lila donor, red recipient)." path-from-xml="dsr00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719780/dsr00802.jpeg" path-from-xml="dsr00802.jpeg" target="_blank">View large</a><a section="7719780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00802.jpeg?Expires=2147483647&Signature=0CdRRWa1JAxvczpaM3~xuH3H~NlwHwyYaNYZki9sI9aWqp5FyEbI8yYvs4F920Qsj3YPoxvSu~EOiUs-cGrQFRWlMsN4SlZoJ3n03-wVDouQalTYlIeMvatnaF86EiWEqpt~A4ou1ZatOHprE3cVRwMNpm3jXklkZEkUKFp~FIB-4FxI0DpRWjq1sHtF-Ag56iVC-C~UJrVUSXx3INiWmq5YheQZINjP1d52xDPX2es5s-SFOZ0yOoA1itQnKzh-NYPu93i61p2NpbrJcU00Hx8-PniZizwVAYiFv8ey5e4oaPztWHkHUPzIDpH54v-cBFzqWGtwvRkfRzARI2l51w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719780&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results. Variant correlation of parents, donor and recipient. (A–C) Correlating all detected variant positions in percentage of reads per position for two individuals visualize the degree of sequence match. A clear reduction of variant positions is desplayed in (C). (D) Cumulative coverage in percentage of target bases (blue p1, green p2, lila donor, red recipient).</p></div></div><p>Analysis of the variant list with equal or greater than 20× coverage in donor and recipient showed, that 1492 are located in intergenic regions, 1173 in intronic regions and 360 in exons. These 360 exon variants represent, with 6.6 variant positions per gene, a list of 59 genes (Fig. <a reveal-id="DSR008F3" data-open="DSR008F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="7719782" scrollto-destination="7719782"></a>
<div id="DSR008F3" data-id="DSR008F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00803.jpeg?Expires=2147483647&Signature=RnzrdDjoULPEtaFc4ykPLu8SuTX9aR2VvafznCExOnQESjuty9aMhxZA01~BvrUZ7-z1EWrU5DQj3G-YRfp0OH7XxOhv8tUqlt5EZNRxnG9ezpNGPr6qPPDYxnOPqRQHRqoIxsun3HyMH4XnzuddllJYkAOHKqlnWr8SMGrKvbFMFkmFrQUqo4Ibs1LnAHbD2o22LeZRmqAoe22pbnvc8OgDCYJ3KhILcri-Igm0GuVl~dE50OHfHizNSW6fEEE7dT8OCa-ITqJjeelSExdXn8YC5vmi-V2una3tNXwi28dEugSrv75Ip4txLOEFzv-ovjQ0kXn5j~7aJE2bqjEImA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results. Summary of variant positions between parents, donor and recipient." path-from-xml="dsr00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719782/dsr00803.jpeg" path-from-xml="dsr00803.jpeg" target="_blank">View large</a><a section="7719782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00803.jpeg?Expires=2147483647&Signature=HPOfL4k1TVs-gFaEpJJ4Es-CQ6HpFetQbHPrcQRjjL4TGagdX1ZQypscJe-Z8pGJHj1ktCrIbziEeWIHMunlozyskhoBYP8ADUOsGMGCr2YBantpY7E-A2848yWUlMo-OaFMFRDKSi9M~JDpCSakwTlLZu7PfYbXIEZK-tDvtCQl6uk1-FU5u~u7Eklw5dUa0teD77PbVykzA1ncgGdaAjhUvtvmHV6VcrFW9-dfgZlQ6OxkEO0A~eRNjyEK1A39Yw5Q9e1XSz2AWp60~a3ZFO75sTB2-61a3~kdVnlOU~MQiOeeQK05-27lAgvzFcJEsdJ8v8HJiAgyfIV~F~8AKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719782&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results. Summary of variant positions between parents, donor and recipient.</p></div></div><div content-id="DSR008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00803.jpeg?Expires=2147483647&Signature=RnzrdDjoULPEtaFc4ykPLu8SuTX9aR2VvafznCExOnQESjuty9aMhxZA01~BvrUZ7-z1EWrU5DQj3G-YRfp0OH7XxOhv8tUqlt5EZNRxnG9ezpNGPr6qPPDYxnOPqRQHRqoIxsun3HyMH4XnzuddllJYkAOHKqlnWr8SMGrKvbFMFkmFrQUqo4Ibs1LnAHbD2o22LeZRmqAoe22pbnvc8OgDCYJ3KhILcri-Igm0GuVl~dE50OHfHizNSW6fEEE7dT8OCa-ITqJjeelSExdXn8YC5vmi-V2una3tNXwi28dEugSrv75Ip4txLOEFzv-ovjQ0kXn5j~7aJE2bqjEImA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MHC resequencing results. Summary of variant positions between parents, donor and recipient." path-from-xml="dsr00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719782/dsr00803.jpeg" path-from-xml="dsr00803.jpeg" target="_blank">View large</a><a section="7719782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00803.jpeg?Expires=2147483647&Signature=HPOfL4k1TVs-gFaEpJJ4Es-CQ6HpFetQbHPrcQRjjL4TGagdX1ZQypscJe-Z8pGJHj1ktCrIbziEeWIHMunlozyskhoBYP8ADUOsGMGCr2YBantpY7E-A2848yWUlMo-OaFMFRDKSi9M~JDpCSakwTlLZu7PfYbXIEZK-tDvtCQl6uk1-FU5u~u7Eklw5dUa0teD77PbVykzA1ncgGdaAjhUvtvmHV6VcrFW9-dfgZlQ6OxkEO0A~eRNjyEK1A39Yw5Q9e1XSz2AWp60~a3ZFO75sTB2-61a3~kdVnlOU~MQiOeeQK05-27lAgvzFcJEsdJ8v8HJiAgyfIV~F~8AKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719782&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MHC resequencing results. Summary of variant positions between parents, donor and recipient.</p></div></div>                            <h3 scrollto-destination=7719783 id="7719783" class="section-title" data-legacy-id=s3f>3.6 Variation analysis SNP microarray</h3>
<p>SNP microarray variants (Affymetrix) with at least 10-fold coverage for recipient and donor with 1345 and 1382 SNPs show that ∼93.38% (<em>n</em>= 1,256) for recipient and 94.07% (<em>n</em>= 1,300) for donor are validated by GS-FLX data, or vice versa. SNPs with no coverage with GS-FLX data represent 1.3%, meaning, that only 1.3% fall in gap regions, a value acceptable to be low. Among those SNPs which differ with PGF, 46% are heterozygous and 54% are homozygous. Data for SNPs outside MHC comprise 885 576 specified SNPs; among these, 551 954 (62.33%) are concordant between donor and recipient. In contrast inside MHC, 1365 (86.6%) SNPs match.</p><p>Taken together, here we report the performance of newly developed methods for human MHC locus resequencing, namely targeted microarray-mediated SCE by NimbleGen and massively-parallel clonal pyrosequencing with 454's next generation sequencing technology. The genomes we partly resequenced represent a typical HSCT situation with 10 out of 10 HLA-loci matched. The procedures we used were tested with respect to efficiency and specificity of on-surface sequence capture, and the ability to discover genetic variation in coding and non-coding regions comparing the genotypes with single-base resolution for selected 3.5 Mb MHC sequence. In total, four MHC captures from four different samples were performed using DNA extracted from peripheral blood, prepared for 454 sequencing separately and subjected to one of the four sequencing regions on a single picotitre plate. The intention of the study was not to look at known or unknown GVHD-determining parameters, but to show whether available technologies and analytical software were powerful enough to start considering a study that addressed this problem comprehensively.</p>                            <h3 scrollto-destination=7719786 id="7719786" class="section-title" data-legacy-id=s3g>3.7 Enrichment, sequencing and recipient-donor pairing</h3>
<p>In total, a mean of 337-fold enrichment achieved, among these sequences 83% could be adjusted to the targeted MHC sequences. We qualified both values (enrichment and specificity) highly sufficient for sound comparison of sequence among samples. 454-pyrosequencing gave approximately und genau? 200 000 reads per sample. A mean of 87% of the MHC was resequenced with at least 3-fold coverage. The MHC-resequencing variation analysis we chose resulted in a simple but clear score for comparing genotype variants. When comparing two individuals on the genotype level, we, first, describe and second, simply count the numbers of single nucleotide variant positions, including insertions or deletions neglecting any known functional background and score a simple number to each possible transplant-pair donor versus recipient 3025, Parent 1 versus recipient 3696 and Parent 2 versus recipient 5790. Applying this type of analysis can reveal the best fit donor out from a series of equally well fully HLA-matched candidates. As already mentioned, we cannot yet discriminate between expected transplant outcome, but we can analyse each MHC with single nucleotide resolution what can help with dependence on known disease association or possible immunologic activity generating as well as drug-acceptance potential for each and any SNP to select the best-suited donor candidate.</p>                            <h3 scrollto-destination=7719788 id="7719788" class="section-title" data-legacy-id=s3h>3.8 Genes and SNPs</h3>
<p>On the basis of this individual MHC resequencing data and gene annotation, all database including genetic variant information for a total of 273 genes was gathered. To date numerous SNPs are described for these regions. Among these, all variants located in exons of the ∼45 MHC resident genes with known immunologic function, presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a> only, show that a huge amount of possible proteinous structural variation, which could be responsible for inflammation induction when encountered by immunologic competent cells not tolerant against them. In total, we have 30 protein-coding genes with 56 non-synonymous (44%) and 71 synonymous (56%) sequence variants in exonic regions, three RNA genes (<em>HCG17</em>, <em>HCG27</em> and <em>NCRNA00171</em>) and one transposable element (<em>TIGD1L</em>). Genes with non-synonymous amino acid exchanges are <em>TUBB</em>, <em>GNL1</em>, <em>PRR3</em>, <em>HCG17</em>, <em>MDC1</em>, <em>HCG27</em>, <em>UBD</em>, <em>PSMB9</em>, <em>HSPA1L</em>, <em>TAP2</em>, <em>PPP1R10</em>, <em>MSH5</em>, <em>CLIC</em>, <em>CFB</em>, <em>C6orf10</em>, <em>TRIM40</em>, <em>CSNK2B, C6orf136 and BAT3</em>. The list of protein-coding genes with synonymous amino acid exchanges, includes the genes <em>PPP1R11</em>, <em>TRIM10</em>, <em>MICB</em>, <em>C6orf205</em>, <em>TRIM26</em>, <em>TCF19</em>, <em>ZFP57</em>, <em>NEU1</em>, <em>KIAA1949</em>, <em>GABBR1</em>, <em>DDR1</em>, <em>C6orf26</em> and <em>AIF1</em> (Table <a reveal-id="DSR008TB4" data-open="DSR008TB4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="7719790" scrollto-destination="7719790"></a>
<div content-id="DSR008TB4" class="table-modal"><div class="table-wrap"><div id="DSR008TB4" class="table-wrap-title" data-id="DSR008TB4"><div class="label -label">Table 4.</div><div class="caption"><p>List of genes with variant positions<sup>a</sup> between donor and recipient</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Non-synonymous amino acid changes: <em>TUBB</em>, <em>GNL1</em>, <em>PRR3</em>, <em>HCG17</em>, <em>MDC1</em>, <em>HCG27</em>, <em>UBD</em>, <em>PSMB9</em>, <em>HSPA1L</em>, <em>TAP2</em>, <em>PPP1R10</em>, <em>MSH5</em>, <em>CLIC</em>, <em>CFB</em>, <em>C6orf10</em>, <em>TRIM40, CSNK2B</em>, <em>C6orf136 and BAT3</em> </td></tr><tr><td>Synonymous amino acid changes: <em>PPP1R11</em>, <em>TRIM10</em>, <em>MICB</em>, <em>C6orf205</em>, <em>TRIM26</em>, <em>TCF19</em>, <em>ZFP57</em>, <em>NEU1</em>, <em>KIAA1949</em>, <em>GABBR1</em>, <em>DDR1 C6orf26</em> and <em>AIF1</em> </td></tr></tbody></table></div><div content-id="DSR008TB4" class="table-modal"><table><tbody><tr><td>Non-synonymous amino acid changes: <em>TUBB</em>, <em>GNL1</em>, <em>PRR3</em>, <em>HCG17</em>, <em>MDC1</em>, <em>HCG27</em>, <em>UBD</em>, <em>PSMB9</em>, <em>HSPA1L</em>, <em>TAP2</em>, <em>PPP1R10</em>, <em>MSH5</em>, <em>CLIC</em>, <em>CFB</em>, <em>C6orf10</em>, <em>TRIM40, CSNK2B</em>, <em>C6orf136 and BAT3</em> </td></tr><tr><td>Synonymous amino acid changes: <em>PPP1R11</em>, <em>TRIM10</em>, <em>MICB</em>, <em>C6orf205</em>, <em>TRIM26</em>, <em>TCF19</em>, <em>ZFP57</em>, <em>NEU1</em>, <em>KIAA1949</em>, <em>GABBR1</em>, <em>DDR1 C6orf26</em> and <em>AIF1</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Variants located in exonic regions only.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719790" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR008TB4" class="table-wrap-title" data-id="DSR008TB4"><div class="label -label">Table 4.</div><div class="caption"><p>List of genes with variant positions<sup>a</sup> between donor and recipient</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Non-synonymous amino acid changes: <em>TUBB</em>, <em>GNL1</em>, <em>PRR3</em>, <em>HCG17</em>, <em>MDC1</em>, <em>HCG27</em>, <em>UBD</em>, <em>PSMB9</em>, <em>HSPA1L</em>, <em>TAP2</em>, <em>PPP1R10</em>, <em>MSH5</em>, <em>CLIC</em>, <em>CFB</em>, <em>C6orf10</em>, <em>TRIM40, CSNK2B</em>, <em>C6orf136 and BAT3</em> </td></tr><tr><td>Synonymous amino acid changes: <em>PPP1R11</em>, <em>TRIM10</em>, <em>MICB</em>, <em>C6orf205</em>, <em>TRIM26</em>, <em>TCF19</em>, <em>ZFP57</em>, <em>NEU1</em>, <em>KIAA1949</em>, <em>GABBR1</em>, <em>DDR1 C6orf26</em> and <em>AIF1</em> </td></tr></tbody></table></div><div content-id="DSR008TB4" class="table-modal"><table><tbody><tr><td>Non-synonymous amino acid changes: <em>TUBB</em>, <em>GNL1</em>, <em>PRR3</em>, <em>HCG17</em>, <em>MDC1</em>, <em>HCG27</em>, <em>UBD</em>, <em>PSMB9</em>, <em>HSPA1L</em>, <em>TAP2</em>, <em>PPP1R10</em>, <em>MSH5</em>, <em>CLIC</em>, <em>CFB</em>, <em>C6orf10</em>, <em>TRIM40, CSNK2B</em>, <em>C6orf136 and BAT3</em> </td></tr><tr><td>Synonymous amino acid changes: <em>PPP1R11</em>, <em>TRIM10</em>, <em>MICB</em>, <em>C6orf205</em>, <em>TRIM26</em>, <em>TCF19</em>, <em>ZFP57</em>, <em>NEU1</em>, <em>KIAA1949</em>, <em>GABBR1</em>, <em>DDR1 C6orf26</em> and <em>AIF1</em> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Variants located in exonic regions only.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7719790" target="_blank">View Large</a></div></div><p>The gene most affected by sequence variation is <em>Tubulin beta</em>. It is different in 54 positions in all four exonic regions including 5 insertions (GG, T, TAAGA, AA and TG) and 3 deletions (A, TG and T) with all variants &gt;22-fold coverage, among those positions different 33 distinct variants result in 21 synonymous amino acid changes. The data need further verification with additional bioinformatic tools with the purpose to discriminate among tubulin family members and true variants. But, tubulin beta functions in microtubule formation and is therefore important for many aspects of cell activity, including morphogenesis, intracellular vesicle/signal transport and cell cycle regulation. In breast cancer, tubulin-associated SNPs are known to be responsible for taxol/taxane resistance.<sup><a reveal-id="DSR008C15" data-open="DSR008C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Another aspect of tubulin-drug interaction is that beside taxane, noscapine, its analoga and colchicine have tubulin-binding properties and influence anti-inflammatory activity by inhibition of cytokine and chemokine release from macrophages. Consequently, if tubulin-variant dependent microtubule dynamics influence cytosolic signalling, thus dampening inflammatory responses the described individual differences might possibly help explain why GVHD signals originate and sprout.<sup><a reveal-id="DSR008C16" data-open="DSR008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In that context it is interesting to mention that GVHD often starts in skin, gastrointestinal tract or liver and that tubulin is involved in pathogenic <em>Escherichia coli</em> infections, which cause inflammation and sepsis, therefore it should be allowed to think, that tubulin variants can contribute to GVHD in one or the other way.<sup><a reveal-id="DSR008C17" data-open="DSR008C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR008C18" data-open="DSR008C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>TIGD1L (tigger transposable element derived 1-like) is a DNA transposon, or class 2 transposable element, has also 17 genomic variants between recipient and donor—meaning full or not? Generally spoken, the influence of this class of elements on mammalian genome evolution is very interesting and it would be helpful to better understand their influence on the immune system and why some species/individuals are more susceptible to invasion by parasites or principal stress situations than others.<sup><a reveal-id="DSR008C19" data-open="DSR008C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>The next gene with high amounts of SNP differences in our specific recipient and donor setting are the genes for the inhibitory units 11 and 10 of protein phosphatase 1 (<em>PPP1R11</em>; protein phosphatase 1, regulatory (inhibitor) subunit 11 with 9 synonymous exchanges; <em>PPP1R10</em> 1 non-synonymous and 1 synonymous amino acid exchange). Especially, in combination with CsA, most immunosuppressive drugs that support successful allograft survival act by inhibiting or depleting T lymphocytes by inhibiting protein phosphatases.<sup><a reveal-id="DSR008C20" data-open="DSR008C20" class="link link-ref link-reveal xref-bibr">20</a></sup></p><p>Another gene with two non-synonymous, five synonymous amino acid exchange differences between recipient and donor is the gene guanine nucleotide-binding protein-like 1 (<em>GNL1</em>). Recently, guanine nucleotide binding protein-like 3-like (<em>GNL3L</em>), another member of the GNL family was described to bind TRF1 (telomeric repeat-binding factor 1). In addition to GNL, TRF1 can be bound and modulated also by two nucleolar GTP-binding proteins, nucleostemin, which exhibit apparently opposite effects on the protein degradation of TRF1. In particular, in a recent paper from Zhu <em>et al</em>., GNL3L is able to stabilize TRF1 protein during mitosis and promote the metaphase-to-anaphase transition.<sup><a reveal-id="DSR008C21" data-open="DSR008C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Telomeric repeat binding factor 1 (TRF1) is a component of the multiprotein complex ‘shelterin’, which organizes the telomere into a high-order structure.</p><p><em>TRIM10, TRIM 26, TRIM 40</em> have 4, 2 and 1 SNPs different with 1 non-synonymous amino acid exchange in <em>TRIM 40</em>. A study using a cDNA expression screening procedure in a patient–donor model with full HLA match, the major target antigen of donor lymphocytes was identified to be TRIM22-442 C, a polymorphic allele of the tripartite motif family member TRIM22 (synonym: STAF50). The authors described that an arginine(R)-to-cysteine(C) exchange at position 442 generated an immunogenic T cell epitope equivalent to a minor histocompatibility antigen (mHag). Approximately 1.3% of Caucasians carry TRIM22-442 C in association with HLA-A*02:01.<sup><a reveal-id="DSR008C22" data-open="DSR008C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p><em>MICB</em>, with only four synonymous exchanges encodes a heavily glycosylated protein, which is a ligand for the NKG2D type II receptor. Binding of the ligand activates the cytolytic response of natural killer (NK) cells, CD8 alpha/beta T cells and gamma/delta T cells, which express the receptor. This protein is stress induced and is similar to MHC class I molecules. <em>MICB</em> seems to have no role in antigen presentation, but acts as a stress-induced self-antigen, a ligand for KLRK1/NKG2D receptor, that is recognized by gamma/delta T cells.<sup><a reveal-id="DSR008C23" data-open="DSR008C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSR008C24" data-open="DSR008C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>Ubiquitin-binding domains of the gene ubiquitin D (<em>UBD</em>) are modular elements that bind non-covalently to the protein modifier ubiquitin. The sequence context of UBDs and the conformational changes that follow their binding to ubiquitin also contribute to ubiquitin signalling and can therefore influence ubiquitins functions in HLA molecule expression and antigen presentation as well as dendritic cell activation.<sup><a reveal-id="DSR008C25" data-open="DSR008C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p><em>PSMB9</em> is different at three exonic positions. A recent publication with a rat cardiac allograft model has shown that immunoproteasome beta subunit 10 was found to be specifically increased in the graft and blood samples during chronic active antibody-mediated rejection. Using this model, they found that administration of the proteasome inhibitor, bortezomib, delayed acute rejection and attenuated the humoral response in both the acute phase and established state of this syndrome in a dose-dependent manner.<sup><a reveal-id="DSR008C26" data-open="DSR008C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=7719799 id="7719799" class="section-title" data-legacy-id=s3i>3.9 Genome-wide SNP analysis</h3>
<p>The analysis of the SNPs located inside MHC by next generation sequencing was complemented by whole genome SNP analysis. In total, we compared 885 576 SNPs equally well distributed throughout the genome observing that 62% (551 954) are shared between donor and recipient outside the MHC, whereas 86% (1365) are shared inside the MHC, reflecting the higher linkage equilibrium of MHC localized loci (Fig. <a reveal-id="DSR008F4" data-open="DSR008F4" class="link link-reveal link-table xref-fig">4</a>). Taken into account that great variance exists between persons as well as demographic groups, this analysis can for the time being only be an estimate as a consequence of missing data on an individuals’ variant level. Validation results from Affymetrix-SNP data within the MHC region show 93.38% concordance with the next generation sequence data. On the basis of more individuals' sequence availability for MHC, the data for SNP design will improve accordingly, but whether it will ever meet the power of direct resequencing is an unanswered question. </p>                        <a id="7719801" scrollto-destination="7719801"></a>
<div id="DSR008F4" data-id="DSR008F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00804.jpeg?Expires=2147483647&Signature=1UtsMkevJFe~mKfGtZbMhZeCEyoTBeMqLCctDnpnNmWg2m2rerQFz3mYxc8aB61lGMPrziiGAFgn5Mhg8ayEB4JCsfdLBnzqDLCNHJ1T~KwrZCpRj-DR2c5lHuD1nVFw-Wx0dbdMB1vMwFiSqjlQPWc91DfyBWIwn1llLCpf7SEnh2V-A5aWWnIMb50tWhxXQ~hlcLQMpHYfCRmXcZrlbB2JcjOAcaVpf9ELlcJN32sPXXv0EHzWmzyorkTk4YTCKo5-YpSvhDCv4eaygX6uDvC~laDIC1eVsEwwkxYjKOdwLHwDksnUrRd-rS5Uj3ayGfNu1yWxk3axsv7iAEptFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of SNP array results for donor and recipient inside and outside MHC. Analysis of Affymetrix microarray HGW SNP 6.0 data for SNP density and signal intensities. (A) The slope of the graph represents the cumulative number of variant SNPs (x = number of SNPs; y = number of variant SNPs). Inset MHC region (Chr.6: 29 000 000–34 999 000 only) and with unrelated not HLA matched control sample (inset blue). (B) Correlation of signal intensities for concordant and variant SNP's inside and outside the MHC region. Taken together, showing that HLA matching in contrast to an unrelated and HLA unmatched control sample significantly reduces the variant SNP density in MHC regions (R = 0.892 versus R = 0.704; z = 6.76) and not in non-MHC regions (R = 0.801 versus R = 0.740; z = −34.53)." path-from-xml="dsr00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719801/dsr00804.jpeg" path-from-xml="dsr00804.jpeg" target="_blank">View large</a><a section="7719801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00804.jpeg?Expires=2147483647&Signature=xFMoEL0si6xVFvyPzgruwhNUAW27aHJgDqOFxQTDa6DfIS~vA12wqKsk6vgclxgQ4h5c0xtqNdLLO5izfP98c3zsja89PpujIAxred5re34xN09BFRLqGBR5RtEnUbBl8F9OY3MktKrrUsYKpObIWhKcWbTMah1ABn20-6HYAdowc7bML8nUzx98YVdM4i9KE8MX4y-JORBSY8enC5pfsWeqLWTSU6ZkWtxa1RqRzdGbDZAYcC5BsLNT792XgXLQNdEl0Xiv2QkWIp11l2UV4f8bajnFkiQE03ljP1SOQcB87RnltjpBqjkgWPLYZt8nHTMZV7sJ2V0AsD-39wj9Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719801&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of SNP array results for donor and recipient inside and outside MHC. Analysis of Affymetrix microarray HGW SNP 6.0 data for SNP density and signal intensities. (A) The slope of the graph represents the cumulative number of variant SNPs (x = number of SNPs; y = number of variant SNPs). Inset MHC region (Chr.6: 29 000 000–34 999 000 only) and with unrelated not HLA matched control sample (inset blue). (B) Correlation of signal intensities for concordant and variant SNP's inside and outside the MHC region. Taken together, showing that HLA matching in contrast to an unrelated and HLA unmatched control sample significantly reduces the variant SNP density in MHC regions (R = 0.892 versus R = 0.704; z = 6.76) and not in non-MHC regions (R = 0.801 versus R = 0.740; z = −34.53).</p></div></div><div content-id="DSR008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/m_dsr00804.jpeg?Expires=2147483647&Signature=1UtsMkevJFe~mKfGtZbMhZeCEyoTBeMqLCctDnpnNmWg2m2rerQFz3mYxc8aB61lGMPrziiGAFgn5Mhg8ayEB4JCsfdLBnzqDLCNHJ1T~KwrZCpRj-DR2c5lHuD1nVFw-Wx0dbdMB1vMwFiSqjlQPWc91DfyBWIwn1llLCpf7SEnh2V-A5aWWnIMb50tWhxXQ~hlcLQMpHYfCRmXcZrlbB2JcjOAcaVpf9ELlcJN32sPXXv0EHzWmzyorkTk4YTCKo5-YpSvhDCv4eaygX6uDvC~laDIC1eVsEwwkxYjKOdwLHwDksnUrRd-rS5Uj3ayGfNu1yWxk3axsv7iAEptFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of SNP array results for donor and recipient inside and outside MHC. Analysis of Affymetrix microarray HGW SNP 6.0 data for SNP density and signal intensities. (A) The slope of the graph represents the cumulative number of variant SNPs (x = number of SNPs; y = number of variant SNPs). Inset MHC region (Chr.6: 29 000 000–34 999 000 only) and with unrelated not HLA matched control sample (inset blue). (B) Correlation of signal intensities for concordant and variant SNP's inside and outside the MHC region. Taken together, showing that HLA matching in contrast to an unrelated and HLA unmatched control sample significantly reduces the variant SNP density in MHC regions (R = 0.892 versus R = 0.704; z = 6.76) and not in non-MHC regions (R = 0.801 versus R = 0.740; z = −34.53)." path-from-xml="dsr00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7719801/dsr00804.jpeg" path-from-xml="dsr00804.jpeg" target="_blank">View large</a><a section="7719801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr00804.jpeg?Expires=2147483647&Signature=xFMoEL0si6xVFvyPzgruwhNUAW27aHJgDqOFxQTDa6DfIS~vA12wqKsk6vgclxgQ4h5c0xtqNdLLO5izfP98c3zsja89PpujIAxred5re34xN09BFRLqGBR5RtEnUbBl8F9OY3MktKrrUsYKpObIWhKcWbTMah1ABn20-6HYAdowc7bML8nUzx98YVdM4i9KE8MX4y-JORBSY8enC5pfsWeqLWTSU6ZkWtxa1RqRzdGbDZAYcC5BsLNT792XgXLQNdEl0Xiv2QkWIp11l2UV4f8bajnFkiQE03ljP1SOQcB87RnltjpBqjkgWPLYZt8nHTMZV7sJ2V0AsD-39wj9Dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7719801&ar=500751&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of SNP array results for donor and recipient inside and outside MHC. Analysis of Affymetrix microarray HGW SNP 6.0 data for SNP density and signal intensities. (A) The slope of the graph represents the cumulative number of variant SNPs (x = number of SNPs; y = number of variant SNPs). Inset MHC region (Chr.6: 29 000 000–34 999 000 only) and with unrelated not HLA matched control sample (inset blue). (B) Correlation of signal intensities for concordant and variant SNP's inside and outside the MHC region. Taken together, showing that HLA matching in contrast to an unrelated and HLA unmatched control sample significantly reduces the variant SNP density in MHC regions (R = 0.892 versus R = 0.704; z = 6.76) and not in non-MHC regions (R = 0.801 versus R = 0.740; z = −34.53).</p></div></div>                            <h2 scrollto-destination=7719802 id="7719802" class="section-title" data-legacy-id=s4>5. Conclusions</h2>
<p>Rapidly evolving next generation sequencing technologies have reduced the hands on time and the cost per base compared with Sanger sequencing and are beginning to spread into disease prog- and diagnostics. New methodological applications, for intensive characterization of donor and recipient, minor and major histocompatibility antigens or other biomarker candidates affecting transplant outcome can clearly help select the best-matched donor by extending the list of genes screened. The list of candidate genes affecting GVHD induction and maintenance is long and the mechanisms are likely to be diverse. The data presented here demonstrate the utility of targeted sequence capture and massive parallel sequencing in detailed characterization of the genetic diversity of the donor and the recipient MHC repertoire. Such a wealth of information derived from such studies will lead to extended personalized genomic knowledge on individual MHC structures, SNPs disease associations, drug efficacy information, minor histocompatibility antigen polymorphisms or T-cell epitope formation, all crucial for further improvement of donor selection and T cell transfer strategies. Ongoing developments in targeted sequence capture in solution and the scheduled 1 kb read length in 454's pyrosequencing technology together with supplemental analysis for haplotype structures will increase the impact of that type of analysis in the field of transplant medicine.</p><p><strong>Acknowledgements:</strong> The authors thank Henry A. Erlich for critically reading the manuscript and helpful discussion. For this publication J.P. and M.D. wrote the manuscript and performed research. S.S., K.H., C.H., S.A. and C.G. performed research, M.D., J.P., P.H. and C.G. designed research and critically reviewed the manuscript, N.N. designed and performed all bioinformatic and statistical analyses. H.H., O.K. and C.G. organised sample preparation and critically reviewed the manuscript.</p><p><strong>Supplementary data:</strong><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7719806 id="7719806" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This project is supported by the Austrian Research Promotion Agency (FFG grant: Nr. 815468 B1).</p>                            <h2 scrollto-destination=7719808 id="7719808" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spellman</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Setterholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maiers</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Advances in the selection of HLA-compatible donors: refinements in HLA typing and matching over the first 20 years of the National Marrow Donor Program Registry</div>, <div class="source">Biol. Blood Marrow Transplant.</div> , <div class="year">2008</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">37</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbmt.2008.05.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbmt.2008.05.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20the%20selection%20of%20HLA-compatible%20donors%3A%20refinements%20in%20HLA%20typing%20and%20matching%20over%20the%20first%2020%20years%20of%20the%20National%20Marrow%20Donor%20Program%20Registry&amp;author=S.%20Spellman&amp;author=M.%20Setterholm&amp;author=M.%20Maiers&amp;publication_year=2008&amp;journal=Biol.%20Blood%20Marrow%20Transplant.&amp;volume=14&amp;pages=37-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbmt.2008.05.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbmt.2008.05.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbmt.2008.05.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18721779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="article-title">Complete sequence and gene map of a human major histocompatibility complex. The MHC sequencing consortium</div>, <div class="source">Nature</div> , <div class="year">1999</div>, vol. <div class="volume">401</div> (pg. <div class="fpage">921</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/44853" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/44853</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/44853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F44853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F44853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10553908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stewart</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Horton</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Allcock</div> <div class="given-names">R. J.</div></div>, et al. </div><div class="article-title">Complete MHC haplotype sequencing for common disease gene mapping</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1176</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2188104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2188104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20MHC%20haplotype%20sequencing%20for%20common%20disease%20gene%20mapping&amp;author=C.%20A.%20Stewart&amp;author=R.%20Horton&amp;author=R.%20J.%20Allcock&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1176-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2188104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2188104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2188104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15140828" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horton</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Coggill</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Variation analysis and gene annotation of eight MHC haplotypes: the MHC Haplotype Project</div>, <div class="source">Immunogenetics</div> , <div class="year">2008</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1</div>-<div class="lpage">18</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00251-007-0262-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00251-007-0262-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20analysis%20and%20gene%20annotation%20of%20eight%20MHC%20haplotypes%3A%20the%20MHC%20Haplotype%20Project&amp;author=R.%20Horton&amp;author=R.%20Gibson&amp;author=P.%20Coggill&amp;publication_year=2008&amp;journal=Immunogenetics&amp;volume=60&amp;pages=1-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00251-007-0262-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00251-007-0262-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00251-007-0262-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18193213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davies</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">S. T.</div></div>, et al. </div><div class="article-title">A genome-wide search for human type 1 diabetes susceptibility genes</div>, <div class="source">Nature</div> , <div class="year">1994</div>, vol. <div class="volume">371</div> (pg. <div class="fpage">130</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/371130a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/371130a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20search%20for%20human%20type%201%20diabetes%20susceptibility%20genes&amp;author=J.%20L.%20Davies&amp;author=Y.%20Kawaguchi&amp;author=S.%20T.%20Bennett&amp;publication_year=1994&amp;journal=Nature&amp;volume=371&amp;pages=130-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/371130a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F371130a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F371130a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8072542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goulder</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Watkins</div> <div class="given-names">D. I.</div></div>. </div><div class="article-title">Impact of MHC class I diversity on immune control of immunodeficiency virus replication</div>, <div class="source">Nat. Rev. Immunol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">619</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nri2357" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nri2357</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20MHC%20class%20I%20diversity%20on%20immune%20control%20of%20immunodeficiency%20virus%20replication&amp;author=P.%20J.%20Goulder&amp;author=D.%20I.%20Watkins&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Immunol.&amp;volume=8&amp;pages=619-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nri2357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnri2357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnri2357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18617886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Haagenson</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">High-resolution donor-recipient HLA matching contributes to the success of unrelated donor marrow transplantation</div>, <div class="source">Blood</div> , <div class="year">2007</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">4576</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1182/blood-2007-06-097386" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1182/blood-2007-06-097386</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20donor-recipient%20HLA%20matching%20contributes%20to%20the%20success%20of%20unrelated%20donor%20marrow%20transplantation&amp;author=S.%20J.%20Lee&amp;author=J.%20Klein&amp;author=M.%20Haagenson&amp;publication_year=2007&amp;journal=Blood&amp;volume=110&amp;pages=4576-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood-2007-06-097386" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood-2007-06-097386" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood-2007-06-097386"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17785583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petersdorf</div> <div class="given-names">E. W.</div></div>. </div><div class="article-title">HLA matching in allogeneic stem cell transplantation</div>, <div class="source">Curr. Opin. Hematol.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">386</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1097/01.moh.0000143701.88042.d9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1097/01.moh.0000143701.88042.d9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HLA%20matching%20in%20allogeneic%20stem%20cell%20transplantation&amp;author=E.%20W.%20Petersdorf&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Hematol.&amp;volume=11&amp;pages=386-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/01.moh.0000143701.88042.d9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2F01.moh.0000143701.88042.d9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2F01.moh.0000143701.88042.d9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15548992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petersdorf</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Malkki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gooley</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">MHC haplotype matching for unrelated hematopoietic cell transplantation</div>, <div class="source">PLoS Med.</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e8</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pmed.0040008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pmed.0040008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MHC%20haplotype%20matching%20for%20unrelated%20hematopoietic%20cell%20transplantation&amp;author=E.%20W.%20Petersdorf&amp;author=M.%20Malkki&amp;author=T.%20A.%20Gooley&amp;author=P.%20J.%20Martin&amp;author=Z.%20Guo&amp;publication_year=2007&amp;journal=PLoS%20Med.&amp;volume=4&amp;pages=e8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pmed.0040008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pmed.0040008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pmed.0040008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17378697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gabriel</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Schaffner</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The structure of haplotype blocks in the human genome</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">2225</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1069424" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1069424</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&amp;author=S.%20B.%20Gabriel&amp;author=S.%20F.%20Schaffner&amp;author=H.%20Nguyen&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=2225-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1069424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1069424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1069424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12029063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Hood</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Malkki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Petersdorf</div> <div class="given-names">E. W.</div></div>. </div><div class="article-title">Long-range multilocus haplotype phasing of the MHC</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">6964</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0602286103" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0602286103</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-range%20multilocus%20haplotype%20phasing%20of%20the%20MHC&amp;author=Z.%20Guo&amp;author=L.%20Hood&amp;author=M.%20Malkki&amp;author=E.%20W.%20Petersdorf&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=6964-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0602286103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0602286103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0602286103"> </span></div> </div></div></div></div></div><div content-id="DSR008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bentley</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Higuchi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hoglund</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">High-resolution, high-throughput HLA genotyping by next-generation sequencing</div>, <div class="source">Tissue Antigens</div> , <div class="year">2009</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">393</div>-<div class="lpage">403</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1399-0039.2009.01345.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1399-0039.2009.01345.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%2C%20high-throughput%20HLA%20genotyping%20by%20next-generation%20sequencing&amp;author=G.%20Bentley&amp;author=R.%20Higuchi&amp;author=B.%20Hoglund&amp;publication_year=2009&amp;journal=Tissue%20Antigens&amp;volume=74&amp;pages=393-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tan.2009.74.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftan.2009.74.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftan.2009.74.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19845894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gabriel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Danzer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hackl</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Rapid high-throughput human leukocyte antigen typing by massively parallel pyrosequencing for high-resolution allele identification</div>, <div class="source">Hum. Immunol.</div> , <div class="year">2009</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">960</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.humimm.2009.08.009" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.humimm.2009.08.009</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20high-throughput%20human%20leukocyte%20antigen%20typing%20by%20massively%20parallel%20pyrosequencing%20for%20high-resolution%20allele%20identification&amp;author=C.%20Gabriel&amp;author=M.%20Danzer&amp;author=C.%20Hackl&amp;publication_year=2009&amp;journal=Hum.%20Immunol.&amp;volume=70&amp;pages=960-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.humimm.2009.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.humimm.2009.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.humimm.2009.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19706315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lind</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ferriola</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mackiewicz</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Next-generation sequencing: the solution for high-resolution</div>, <div class="source">Unambiguous HLA Typing, Hum. Immunol.</div> , <div class="year">2010</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">1033</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%3A%20the%20solution%20for%20high-resolution&amp;author=C.%20Lind&amp;author=D.%20Ferriola&amp;author=K.%20Mackiewicz&amp;publication_year=2010&amp;journal=Unambiguous%20HLA%20Typing%2C%20Hum.%20Immunol.&amp;volume=71&amp;pages=1033-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.humimm.2010.06.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.humimm.2010.06.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.humimm.2010.06.016"> </span></div> </div></div></div></div></div><div content-id="DSR008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fojo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Menefee</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Mechanisms of multidrug resistance: the potential role of microtubule-stabilizing agents</div>, <div class="source">Ann. Oncol.</div> , <div class="year">2007</div>, vol. <div class="volume">18</div> <div class="issue">Suppl. 5</div>(pg. <div class="fpage">v3</div>-<div class="lpage">v8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/annonc/mdm172" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/annonc/mdm172</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20multidrug%20resistance%3A%20the%20potential%20role%20of%20microtubule-stabilizing%20agents&amp;author=T.%20Fojo&amp;author=M.%20Menefee&amp;publication_year=2007&amp;journal=Ann.%20Oncol.&amp;volume=18&amp;pages=v3-v8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/annonc/mdm172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fannonc%2Fmdm172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fannonc%2Fmdm172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17656560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zughaier</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Karna</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Aneja</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Potent anti-inflammatory activity of novel microtubule-modulating brominated noscapine analogs</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e9165</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0009165" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0009165</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Potent%20anti-inflammatory%20activity%20of%20novel%20microtubule-modulating%20brominated%20noscapine%20analogs&amp;author=S.%20Zughaier&amp;author=P.%20Karna&amp;author=D.%20Stephens&amp;author=R.%20Aneja&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e9165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20161797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Penack</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Holler</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">van den Brink</div> <div class="given-names">M. R.</div></div>. </div><div class="article-title">Graft-versus-host disease: regulation by microbe-associated molecules and innate immune receptors</div>, <div class="source">Blood</div> , <div class="year">2010</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">1865</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1182/blood-2009-09-242784" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1182/blood-2009-09-242784</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Graft-versus-host%20disease%3A%20regulation%20by%20microbe-associated%20molecules%20and%20innate%20immune%20receptors&amp;author=O.%20Penack&amp;author=E.%20Holler&amp;author=M.%20R.%20van%20den%20Brink&amp;publication_year=2010&amp;journal=Blood&amp;volume=115&amp;pages=1865-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood-2009-09-242784" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood-2009-09-242784" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood-2009-09-242784"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20042727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shaw</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Smollett</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cleary</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Enteropathogenic Escherichia coli type III effectors EspG and EspG2 disrupt the microtubule network of intestinal epithelial cells</div>, <div class="source">Infect. Immun.</div> , <div class="year">2005</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">4385</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/IAI.73.7.4385-4390.2005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/IAI.73.7.4385-4390.2005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enteropathogenic%20Escherichia%20coli%20type%20III%20effectors%20EspG%20and%20EspG2%20disrupt%20the%20microtubule%20network%20of%20intestinal%20epithelial%20cells&amp;author=R.%20K.%20Shaw&amp;author=K.%20Smollett&amp;author=J.%20Cleary&amp;publication_year=2005&amp;journal=Infect.%20Immun.&amp;volume=73&amp;pages=4385-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.73.7.4385-4390.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.73.7.4385-4390.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.73.7.4385-4390.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15972534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Oosterhout</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Transposons in the MHC: the Yin and Yang of the vertebrate immune system</div>, <div class="source">Heredity</div> , <div class="year">2009</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">190</div>-<div class="lpage">1</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/hdy.2009.46" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/hdy.2009.46</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposons%20in%20the%20MHC%3A%20the%20Yin%20and%20Yang%20of%20the%20vertebrate%20immune%20system&amp;author=C.%20van%20Oosterhout&amp;publication_year=2009&amp;journal=Heredity&amp;volume=103&amp;pages=190-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2009.46" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2009.46" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2009.46"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19401713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wee</div> <div class="given-names">Y. M.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">M. Y.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">C. H.</div></div>, et al. </div><div class="article-title">The synergistic effect of tautomycetin on cyclosporine a-mediated immunosuppression in a rodent islet allograft model</div>, <div class="source">Mol. Med.</div> , <div class="year">2010</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">298</div>-<div class="lpage">306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20synergistic%20effect%20of%20tautomycetin%20on%20cyclosporine%20a-mediated%20immunosuppression%20in%20a%20rodent%20islet%20allograft%20model&amp;author=Y.%20M.%20Wee&amp;author=M.%20Y.%20Choi&amp;author=C.%20H.%20Kang&amp;publication_year=2010&amp;journal=Mol.%20Med.&amp;volume=16&amp;pages=298-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2119/molmed.2009.00099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2119%2Fmolmed.2009.00099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2119%2Fmolmed.2009.00099"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20440443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hsu</div> <div class="given-names">J. K.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Teishima</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">R. Y.</div></div>. </div><div class="article-title">GNL3L stabilizes the TRF1 complex and promotes mitotic transition</div>, <div class="source">J. Cell Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">827</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1083/jcb.200812121" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1083/jcb.200812121</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GNL3L%20stabilizes%20the%20TRF1%20complex%20and%20promotes%20mitotic%20transition&amp;author=Q.%20Zhu&amp;author=L.%20Meng&amp;author=J.%20K.%20Hsu&amp;author=T.%20Lin&amp;author=J.%20Teishima&amp;author=R.%20Y.%20Tsai&amp;publication_year=2009&amp;journal=J.%20Cell%20Biol.&amp;volume=185&amp;pages=827-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200812121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200812121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200812121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19487455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lennerz</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Lindemann</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Dissection and molecular analysis of alloreactive CD8+ T cell responses in allogeneic haematopoietic stem cell transplantation</div>, <div class="source">Cancer Immunol. Immunother.</div> , <div class="year">2008</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">849</div>-<div class="lpage">57</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00262-007-0421-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00262-007-0421-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20and%20molecular%20analysis%20of%20alloreactive%20CD8%2B%20T%20cell%20responses%20in%20allogeneic%20haematopoietic%20stem%20cell%20transplantation&amp;author=C.%20Wolfel&amp;author=V.%20Lennerz&amp;author=E.%20Lindemann&amp;publication_year=2008&amp;journal=Cancer%20Immunol.%20Immunother.&amp;volume=57&amp;pages=849-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00262-007-0421-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00262-007-0421-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00262-007-0421-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18004563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gonzalez</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Groh</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Spies</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Immunobiology of human NKG2D and its ligands</div>, <div class="source">Curr. Top. Microbiol. Immunol.</div> , <div class="year">2006</div>, vol. <div class="volume">298</div> (pg. <div class="fpage">121</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/3-540-27743-9_6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/3-540-27743-9_6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunobiology%20of%20human%20NKG2D%20and%20its%20ligands&amp;author=S.%20Gonzalez&amp;author=V.%20Groh&amp;author=T.%20Spies&amp;publication_year=2006&amp;journal=Curr.%20Top.%20Microbiol.%20Immunol.&amp;volume=298&amp;pages=121-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16329186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suarez-Alvarez</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lopez-Vazquez</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Baltar</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Ortega</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Lopez-Larrea</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Potential role of NKG2D and its ligands in organ transplantation: new target for immunointervention</div>, <div class="source">Am. J. Transplant.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">251</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1600-6143.2008.02526.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1600-6143.2008.02526.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Potential%20role%20of%20NKG2D%20and%20its%20ligands%20in%20organ%20transplantation%3A%20new%20target%20for%20immunointervention&amp;author=B.%20Suarez-Alvarez&amp;author=A.%20Lopez-Vazquez&amp;author=J.%20M.%20Baltar&amp;author=F.%20Ortega&amp;author=C.%20Lopez-Larrea&amp;publication_year=2009&amp;journal=Am.%20J.%20Transplant.&amp;volume=9&amp;pages=251-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ajt.2009.9.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fajt.2009.9.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fajt.2009.9.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19178412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winget</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Mayor</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The diversity of ubiquitin recognition: hot spots and varied specificity</div>, <div class="source">Mol. Cell</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">627</div>-<div class="lpage">35</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.molcel.2010.05.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.molcel.2010.05.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20diversity%20of%20ubiquitin%20recognition%3A%20hot%20spots%20and%20varied%20specificity&amp;author=J.%20M.%20Winget&amp;author=T.%20Mayor&amp;publication_year=2010&amp;journal=Mol.%20Cell&amp;volume=38&amp;pages=627-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2010.05.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2010.05.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2010.05.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20541996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR008C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashton-Chess</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mai</div> <div class="given-names">H. L.</div></div>,  <div class="name"><div class="surname">Jovanovic</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Immunoproteasome beta subunit 10 is increased in chronic antibody-mediated rejection</div>, <div class="source">Kidney Int.</div> , <div class="year">2010</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">880</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ki.2010.15" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ki.2010.15</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunoproteasome%20beta%20subunit%2010%20is%20increased%20in%20chronic%20antibody-mediated%20rejection&amp;author=J.%20Ashton-Chess&amp;author=H.%20L.%20Mai&amp;author=V.%20Jovanovic&amp;publication_year=2010&amp;journal=Kidney%20Int.&amp;volume=77&amp;pages=880-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ki.2010.15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fki.2010.15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fki.2010.15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20182416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=500751&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/4/10.1093/dnares/dsr008/2/dsr008-dsr008supp.xlsx?Expires=2147483647&amp;Signature=xxvRBtvojEj38P2CKJBrmUSPdtXcUVYrUyKp7Si1lEOAQ85u4oMpv9tCRU3nUB0H9A0ghxTiGDo-9Yrj5Vip4bxVck3I984W7UgcY-neh-SObmEeh2ZnqXw2OhMCP0Ax6XBGLc0T5gZqBIEL9Gg--dpYFf1wKTPZ-MWDZL~sbs-L--jA2-ZJcWosSj70W4c-Y2oW4~vaZkXaS~97t7HjdEfdjrLKlUIIefINUzm9RKeB0t-ShJIvABs31ah7QsTaLpTnHnpeAhf1eadxc0XeoDw~sqyF8cu0mBWsu5qnGrY2CVEHHtf6vbIDuCKScyJNWNQBtaqf4~URG-hH0k1Bzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">247</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="500751"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000294936100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f4%2f201%2f500751">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000294936100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F4%2F201%2F500751" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/500751'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sequence Capture and Next Generation Resequencing of the MHC Region Highlights Potential Transplantation Determinants in HLA Identical Haematopoietic Stem Cell Transplantation" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f4%2f201%2f500751">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

