

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/5/377/348317","@type":"ScholarlyArticle","name":"Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea","datePublished":"2015-09-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv020","keywords":["chickpea","flowering time","InDel","maturity time","QTL"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/5/377/348317","author":[{"name":"Das, Shouvik","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Upadhyaya, Hari D.","affiliation":"2  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Telangana ,  India","@type":"Person"},{"name":"Srivastava, Rishi","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Bajaj, Deepak","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Gowda, C.L.L.","affiliation":"2  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Telangana ,  India","@type":"Person"},{"name":"Sharma, Shivali","affiliation":"2  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Telangana ,  India","@type":"Person"},{"name":"Singh, Sube","affiliation":"2  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324, Telangana ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"},{"name":"Parida, Swarup K.","affiliation":"1  National Institute of Plant Genome Research (NIPGR) ,  Aruna Asaf Ali Marg, New Delhi 110067 ,  India","@type":"Person"}],"description":"Abstract.  We developed 21,499 genome-wide insertion–deletion (InDel) markers (2- to 54-bp in silico fragment length polymorphism) by comparing the genomic sequ","pageStart":"377","pageEnd":"386","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea" />
    <meta property="og:description" content="Abstract.  We developed 21,499 genome-wide insertion–deletion (InDel) markers (2- to 54-bp in silico fragment length polymorphism) by comparing the genomic sequ" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/5/377/348317" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-09-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Das, Shouvik" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Upadhyaya, Hari D." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India" /><meta name="citation_author" content="Srivastava, Rishi" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Bajaj, Deepak" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Gowda, C.L.L." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India" /><meta name="citation_author" content="Sharma, Shivali" /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India" /><meta name="citation_author" content="Singh, Sube" /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, Telangana, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_author" content="Parida, Swarup K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110067, India" /><meta name="citation_title" content="Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea" /><meta name="citation_firstpage" content="377" /><meta name="citation_lastpage" content="386" /><meta name="citation_doi" content="10.1093/dnares/dsv020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2015/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Towards marker-assisted selection in pulses: a review;  citation_journal_title=Plant Breed.;  citation_year=2011;  citation_volume=130;  citation_pages=297-313; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement;  citation_journal_title=Nat. Biotechnol.,;  citation_year=2013;  citation_volume=31;  citation_pages=240-6; " /><meta name="citation_reference" content="citation_title=Achievements and prospects of genomics-assisted breeding in three legume crops of the semi-arid tropics;  citation_journal_title=Biotechnol. Adv.;  citation_year=2013;  citation_volume=31;  citation_pages=1120-34; " /><meta name="citation_reference" content="citation_title=Genetic analysis of seed size, yield and days to flowering in a chickpea recombinant inbred line population derived from a kabuli × desi cross;  citation_journal_title=Ann. Appl. Biol.;  citation_year=2007;  citation_volume=151;  citation_pages=33-42; " /><meta name="citation_reference" content="citation_title=Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with Medicago truncatula genome;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=1415-41; " /><meta name="citation_reference" content="citation_title=Mapping of QTLs governing agronomic and field traits in chickpea;  citation_journal_title=J. Appl. Genet.;  citation_year=2011;  citation_volume=52;  citation_pages=9-21; " /><meta name="citation_reference" content="citation_title=Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (Cicer arietinum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1577-89; " /><meta name="citation_reference" content="citation_title=Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume crop of the semi-arid tropics of Asia and Africa;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=922-31; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (Cicer arietinum L.);  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=357-73; " /><meta name="citation_reference" content="citation_title=Functionally relevant microsatellite markers from chickpea transcription factor genes for efﬁcient genotyping applications and trait association mapping;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=355-74; " /><meta name="citation_reference" content="citation_title=Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea;  citation_journal_title=Front. Plant Sci.;  citation_year=2015;  citation_volume=6;  citation_pages=162" /><meta name="citation_reference" content="citation_title=An integrated genomic approach for rapid delineation of candidate genes regulating agro-morphological traits in chickpea;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=695-710; " /><meta name="citation_reference" content="citation_title=Genetic dissection of drought tolerance in chickpea (Cicer arietinum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2014;  citation_volume=127;  citation_pages=445-62; " /><meta name="citation_reference" content="citation_title=A draft genome sequence of the pulse crop chickpea (Cicer arietinum L.);  citation_journal_title=Plant J.;  citation_year=2013;  citation_volume=74;  citation_pages=715-29; " /><meta name="citation_reference" content="citation_title=In silico identification and experimental validation of insertion-deletion polymorphisms in tomato genome;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=429-38; " /><meta name="citation_reference" content="citation_title=Development of genome-wide insertion/deletion markers in rice based on graphic pipeline platform;  citation_journal_title=J. Integr. Plant. Biol.;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=Developing market class specific InDel markers from next generation sequence data in Phaseolus vulgaris L.;  citation_journal_title=Front. Plant Sci.;  citation_year=2014;  citation_volume=5;  citation_pages=185" /><meta name="citation_reference" content="citation_title=Repeat instability: mechanisms of dynamic mutations;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=729-42; " /><meta name="citation_reference" content="citation_title=Mechanism of a genetic glissando: structural biology of InDel mutations;  citation_journal_title=Trends Biochem. Sci.;  citation_year=2006;  citation_volume=31;  citation_pages=206-14; " /><meta name="citation_reference" content="citation_title=Genome-wide discovery of DNA polymorphisms in rice cultivars with contrasting drought and salinity stress response and their functional relevance;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2014;  citation_volume=12;  citation_pages=253-64; " /><meta name="citation_reference" content="citation_title=Development of insertion and deletion markers based on biparental resequencing for fine mapping seed weight in soybean;  citation_journal_title=Plant Genome;  citation_year=2014;  citation_volume=7;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Exploration of presence/absence variation and corresponding polymorphic markers in soybean genome;  citation_journal_title=J. Integr. Plant. Biol.;  citation_year=2014;  citation_volume=56;  citation_pages=1009-19; " /><meta name="citation_reference" content="citation_title=PAV markers in Sorghum bicolour: genome pattern, affected genes and pathways, and genetic linkage map construction;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2015;  citation_volume=128;  citation_pages=623-37; " /><meta name="citation_reference" content="citation_title=The chickpea genomic web resource: visualization and analysis of the desi-type Cicer arietinum nuclear genome for comparative exploration of legumes;  citation_journal_title=BMC Plant Biol.;  citation_year=2014;  citation_volume=14;  citation_pages=315" /><meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic marker analysis;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2128-9; " /><meta name="citation_reference" content="citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations;  citation_journal_title=Genomics;  citation_year=1987;  citation_volume=1;  citation_pages=174-81; " /><meta name="citation_reference" content="citation_title=A combinatorial approach of comprehensive QTL-based comparative genome mapping and transcript profiling identified a seed weight-regulating candidate gene in chickpea;  citation_journal_title=Sci. Rep.;  citation_year=2015;  citation_volume=5;  citation_pages=9264" /><meta name="citation_reference" content="Van Ooijen J.W.  2009 , MapQTL 6: Software for the mapping of quantitative trait loci in experimental populations of diploid species , B.V. Kyazma : Wageningen, Netherlands. " /><meta name="citation_reference" content="citation_title=Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation;  citation_journal_title=Funct. Plant. Biol.;  citation_year=2003;  citation_volume=30;  citation_pages=1081-7; " /><meta name="citation_reference" content="citation_title=Molecular insights into the evolution of crop plants;  citation_journal_title=Am. J. Bot.;  citation_year=2008;  citation_volume=95;  citation_pages=113-22; " /><meta name="citation_reference" content="citation_title=Genome-wide patterns of large-size presence/absence variants in sorghum;  citation_journal_title=J. Integr. Plant. Biol.;  citation_year=2014;  citation_volume=56;  citation_pages=24-37; " /><meta name="citation_reference" content="citation_title=Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=716-32; " /><meta name="citation_reference" content="citation_title=Comparative analysis of kabuli chickpea transcriptome with desi and wild chickpea provides a rich resource for development of functional markers;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e52443" /><meta name="citation_reference" content="citation_title=Molecular diversity and phylogeny in geographical collection of chickpea (Cicer sp.) accessions;  citation_journal_title=J. Genet.;  citation_year=2011;  citation_volume=90;  citation_pages=e94-100; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=708" /><meta name="citation_reference" content="citation_title=Mapping genes for double podding and other morphological traits in chickpea;  citation_journal_title=Euphytica;  citation_year=2002;  citation_volume=128;  citation_pages=285-92; " /><meta name="citation_reference" content="citation_title=Inheritance of time to flowering in chickpea in a short-season temperate environments;  citation_journal_title=J. Hered.;  citation_year=2006;  citation_volume=97;  citation_pages=55-61; " /><meta name="citation_reference" content="citation_title=Genetic analysis of agronomic traits in a wide cross of chickpea;  citation_journal_title=Field Crops Res.;  citation_year=2009;  citation_volume=111;  citation_pages=130-6; " /><meta name="citation_reference" content="citation_title=Mapping a major gene for growth habit and QTLs for Ascochyta blight resistance and flowering time in a population between chickpea and Cicer reticulatum;  citation_journal_title=Euphytica;  citation_year=2010;  citation_volume=173;  citation_pages=307-19; " /><meta name="citation_reference" content="citation_title=Molecular mapping and characterization of genes governing time to flowering, seed weight, and plant height in an intraspecific genetic linkage map of chickpea (Cicer arietinum);  citation_journal_title=Biochem. Genet.;  citation_year=2013;  citation_volume=51;  citation_pages=387-97; " /><meta name="citation_reference" content="citation_title=The Medicago FLOWERING LOCUS T homolog, MtFTa1, is a key regulator of flowering time;  citation_journal_title=Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=2207-24; " /><meta name="citation_reference" content="citation_title=The genetic basis of ﬂowering responses to seasonal cues;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=627-39; " /><meta name="citation_reference" content="citation_title=GmFT4, a homolog of FLOWERING LOCUS T, is positively regulated by E1 and functions as a flowering repressor in soybean;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e89030" /><meta name="citation_reference" content="citation_title=Genetic control of flowering time in legumes;  citation_journal_title=Front. Plant Sci.;  citation_year=2015;  citation_volume=6;  citation_pages=207" /><meta name="citation_reference" content="citation_title=Characterization of MADS-domain transcription factor complexes in Arabidopsis flower development;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=1560-5; " /><meta name="citation_reference" content="citation_title=The role of the DNA-binding One Zinc Finger (DOF) transcription factor family in plants;  citation_journal_title=Plant Sci.;  citation_year=2013;  citation_volume=209;  citation_pages=32-45; " /><meta name="citation_reference" content="citation_title=MADS reloaded: evolution of the AGAMOUS subfamily genes;  citation_journal_title=New Phytol.;  citation_year=2014;  citation_volume=201;  citation_pages=717-32; " /><meta name="citation_reference" content="citation_title=TCP three-way handshake: linking developmental processes with plant immunity;  citation_journal_title=Trends Plant Sci.;  citation_year=2015;  citation_volume=20;  citation_pages=238-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/5/377/998207/dsv020.pdf" /><meta name="description" content="Abstract.  We developed 21,499 genome-wide insertion–deletion (InDel) markers (2- to 54-bp in silico fragment length polymorphism) by comparing the genomic sequ" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/5/377/348317" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/5/377/348317" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/5/377/348317");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv020';
    </script>
<title>
	Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348317" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bdp8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEYXMsIFNob3V2aWsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVXBhZGh5YXlhLCBIYXJpIEQuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyMzI0LCBUZWxhbmdhbmEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3JpdmFzdGF2YSwgUmlzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFqYWosIERlZXBhayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb3dkYSwgQy5MLkwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyMzI0LCBUZWxhbmdhbmEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hhcm1hLCBTaGl2YWxpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyMzI0LCBUZWxhbmdhbmEsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2luZ2gsIFN1YmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSA1MDIzMjQsIFRlbGFuZ2FuYSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUeWFnaSwgQWtoaWxlc2ggSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaWRhLCBTd2FydXAgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtd2lkZSBpbnNlcnRpb27igJNkZWxldGlvbiAoSW5EZWwpIG1hcmtlciBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgZm9yIGdlbm9taWNzLWFzc2lzdGVkIGJyZWVkaW5nIGFwcGxpY2F0aW9ucyBpbiBjaGlja3BlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzg2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmRzIG1hcmtlci1hc3Npc3RlZCBzZWxlY3Rpb24gaW4gcHVsc2VzOiBhIHJldmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEzMDsgIGNpdGF0aW9uX3BhZ2VzPTI5Ny0zMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSBwcm92aWRlcyBhIHJlc291cmNlIGZvciB0cmFpdCBpbXByb3ZlbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLiw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yNDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNoaWV2ZW1lbnRzIGFuZCBwcm9zcGVjdHMgb2YgZ2Vub21pY3MtYXNzaXN0ZWQgYnJlZWRpbmcgaW4gdGhyZWUgbGVndW1lIGNyb3BzIG9mIHRoZSBzZW1pLWFyaWQgdHJvcGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvdGVjaG5vbC4gQWR2LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTExMjAtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgb2Ygc2VlZCBzaXplLCB5aWVsZCBhbmQgZGF5cyB0byBmbG93ZXJpbmcgaW4gYSBjaGlja3BlYSByZWNvbWJpbmFudCBpbmJyZWQgbGluZSBwb3B1bGF0aW9uIGRlcml2ZWQgZnJvbSBhIGthYnVsaSDDlyBkZXNpIGNyb3NzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEFwcGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGlvbiBvZiBub3ZlbCBTU1IgYW5kIGdlbmUtYmFzZWQgU05QIG1hcmtlciBsb2NpIGluIHRoZSBjaGlja3BlYSBnZW5ldGljIG1hcCBhbmQgZXN0YWJsaXNobWVudCBvZiBuZXcgYW5jaG9yIHBvaW50cyB3aXRoIE1lZGljYWdvIHRydW5jYXR1bGEgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTE0MTUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgb2YgUVRMcyBnb3Zlcm5pbmcgYWdyb25vbWljIGFuZCBmaWVsZCB0cmFpdHMgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCB1c2Ugb2YgZ2VuaWMgbW9sZWN1bGFyIG1hcmtlcnMgKEdNTXMpIGZvciBjb25zdHJ1Y3Rpb24gb2YgYSB0cmFuc2NyaXB0IG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0xNTc3LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIGluIGNoaWNrcGVhIChDaWNlciBhcmlldGludW0gTC4pLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgdGhlIHNlbWktYXJpZCB0cm9waWNzIG9mIEFzaWEgYW5kIEFmcmljYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTkyMi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgZm9yIGNvbnN0cnVjdGluZyBhIHNhdHVyYXRlZCBsaW5rYWdlIG1hcCBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWxseSByZWxldmFudCBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZyb20gY2hpY2twZWEgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZ2VuZXMgZm9yIGVm76yBY2llbnQgZ2Vub3R5cGluZyBhcHBsaWNhdGlvbnMgYW5kIHRyYWl0IGFzc29jaWF0aW9uIG1hcHBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MzU1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbXBsb3lpbmcgZ2Vub21lLXdpZGUgU05QIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyBzdHJhdGVneSB0byBleHRyYXBvbGF0ZSB0aGUgbmF0dXJhbCBhbGxlbGljIGRpdmVyc2l0eSBhbmQgZG9tZXN0aWNhdGlvbiBwYXR0ZXJucyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0ZWQgZ2Vub21pYyBhcHByb2FjaCBmb3IgcmFwaWQgZGVsaW5lYXRpb24gb2YgY2FuZGlkYXRlIGdlbmVzIHJlZ3VsYXRpbmcgYWdyby1tb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpc3NlY3Rpb24gb2YgZHJvdWdodCB0b2xlcmFuY2UgaW4gY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9NDQ1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcHVsc2UgY3JvcCBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz03MTUtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluIHNpbGljbyBpZGVudGlmaWNhdGlvbiBhbmQgZXhwZXJpbWVudGFsIHZhbGlkYXRpb24gb2YgaW5zZXJ0aW9uLWRlbGV0aW9uIHBvbHltb3JwaGlzbXMgaW4gdG9tYXRvIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGdlbm9tZS13aWRlIGluc2VydGlvbi9kZWxldGlvbiBtYXJrZXJzIGluIHJpY2UgYmFzZWQgb24gZ3JhcGhpYyBwaXBlbGluZSBwbGF0Zm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSW50ZWdyLiBQbGFudC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BpbmcgbWFya2V0IGNsYXNzIHNwZWNpZmljIEluRGVsIG1hcmtlcnMgZnJvbSBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2UgZGF0YSBpbiBQaGFzZW9sdXMgdnVsZ2FyaXMgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBlYXQgaW5zdGFiaWxpdHk6IG1lY2hhbmlzbXMgb2YgZHluYW1pYyBtdXRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTcyOS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtIG9mIGEgZ2VuZXRpYyBnbGlzc2FuZG86IHN0cnVjdHVyYWwgYmlvbG9neSBvZiBJbkRlbCBtdXRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBCaW9jaGVtLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjA2LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBkaXNjb3Zlcnkgb2YgRE5BIHBvbHltb3JwaGlzbXMgaW4gcmljZSBjdWx0aXZhcnMgd2l0aCBjb250cmFzdGluZyBkcm91Z2h0IGFuZCBzYWxpbml0eSBzdHJlc3MgcmVzcG9uc2UgYW5kIHRoZWlyIGZ1bmN0aW9uYWwgcmVsZXZhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgaW5zZXJ0aW9uIGFuZCBkZWxldGlvbiBtYXJrZXJzIGJhc2VkIG9uIGJpcGFyZW50YWwgcmVzZXF1ZW5jaW5nIGZvciBmaW5lIG1hcHBpbmcgc2VlZCB3ZWlnaHQgaW4gc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmF0aW9uIG9mIHByZXNlbmNlL2Fic2VuY2UgdmFyaWF0aW9uIGFuZCBjb3JyZXNwb25kaW5nIHBvbHltb3JwaGljIG1hcmtlcnMgaW4gc295YmVhbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEludGVnci4gUGxhbnQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9MTAwOS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFWIG1hcmtlcnMgaW4gU29yZ2h1bSBiaWNvbG91cjogZ2Vub21lIHBhdHRlcm4sIGFmZmVjdGVkIGdlbmVzIGFuZCBwYXRod2F5cywgYW5kIGdlbmV0aWMgbGlua2FnZSBtYXAgY29uc3RydWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTYyMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNoaWNrcGVhIGdlbm9taWMgd2ViIHJlc291cmNlOiB2aXN1YWxpemF0aW9uIGFuZCBhbmFseXNpcyBvZiB0aGUgZGVzaS10eXBlIENpY2VyIGFyaWV0aW51bSBudWNsZWFyIGdlbm9tZSBmb3IgY29tcGFyYXRpdmUgZXhwbG9yYXRpb24gb2YgbGVndW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG93ZXJNYXJrZXI6IGFuIGludGVncmF0ZWQgYW5hbHlzaXMgZW52aXJvbm1lbnQgZm9yIGdlbmV0aWMgbWFya2VyIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIxMjgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFQTUFLRVI6IGFuIGludGVyYWN0aXZlIGNvbXB1dGVyIHBhY2thZ2UgZm9yIGNvbnN0cnVjdGluZyBwcmltYXJ5IGdlbmV0aWMgbGlua2FnZSBtYXBzIG9mIGV4cGVyaW1lbnRhbCBhbmQgbmF0dXJhbCBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE3NC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21iaW5hdG9yaWFsIGFwcHJvYWNoIG9mIGNvbXByZWhlbnNpdmUgUVRMLWJhc2VkIGNvbXBhcmF0aXZlIGdlbm9tZSBtYXBwaW5nIGFuZCB0cmFuc2NyaXB0IHByb2ZpbGluZyBpZGVudGlmaWVkIGEgc2VlZCB3ZWlnaHQtcmVndWxhdGluZyBjYW5kaWRhdGUgZ2VuZSBpbiBjaGlja3BlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05MjY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVmFuIE9vaWplbiBKLlcuICAyMDA5ICwgTWFwUVRMIDY6IFNvZnR3YXJlIGZvciB0aGUgbWFwcGluZyBvZiBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbiBleHBlcmltZW50YWwgcG9wdWxhdGlvbnMgb2YgZGlwbG9pZCBzcGVjaWVzICwgQi5WLiBLeWF6bWEgOiBXYWdlbmluZ2VuLCBOZXRoZXJsYW5kcy4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGN1bHRpdmF0ZWQgY2hpY2twZWE6IGZvdXIgYm90dGxlbmVja3MgbGltaXQgZGl2ZXJzaXR5IGFuZCBjb25zdHJhaW4gYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuY3QuIFBsYW50LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwODEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGluc2lnaHRzIGludG8gdGhlIGV2b2x1dGlvbiBvZiBjcm9wIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xMTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHBhdHRlcm5zIG9mIGxhcmdlLXNpemUgcHJlc2VuY2UvYWJzZW5jZSB2YXJpYW50cyBpbiBzb3JnaHVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBJbnRlZ3IuIFBsYW50LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTI0LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBkZXZlbG9wbWVudCBvZiBjb3N0LWVmZmVjdGl2ZSBTTlAgbWFya2VyIGFzc2F5cyBmb3IgZGl2ZXJzaXR5IGFzc2Vzc21lbnQgYW5kIGdlbmV0aWMgbWFwcGluZyBpbiBjaGlja3BlYSBhbmQgY29tcGFyYXRpdmUgbWFwcGluZyBpbiBsZWd1bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTcxNi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2Yga2FidWxpIGNoaWNrcGVhIHRyYW5zY3JpcHRvbWUgd2l0aCBkZXNpIGFuZCB3aWxkIGNoaWNrcGVhIHByb3ZpZGVzIGEgcmljaCByZXNvdXJjZSBmb3IgZGV2ZWxvcG1lbnQgb2YgZnVuY3Rpb25hbCBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTUyNDQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGRpdmVyc2l0eSBhbmQgcGh5bG9nZW55IGluIGdlb2dyYXBoaWNhbCBjb2xsZWN0aW9uIG9mIGNoaWNrcGVhIChDaWNlciBzcC4pIGFjY2Vzc2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPWU5NC0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIFNOUCBpZGVudGlmaWNhdGlvbiBpbiBjaGlja3BlYSBmb3IgdXNlIGluIGRldmVsb3BtZW50IG9mIGEgaGlnaCBkZW5zaXR5IGdlbmV0aWMgbWFwIGFuZCBpbXByb3ZlbWVudCBvZiBjaGlja3BlYSByZWZlcmVuY2UgZ2Vub21lIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz03MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGdlbmVzIGZvciBkb3VibGUgcG9kZGluZyBhbmQgb3RoZXIgbW9ycGhvbG9naWNhbCB0cmFpdHMgaW4gY2hpY2twZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0yODUtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaGVyaXRhbmNlIG9mIHRpbWUgdG8gZmxvd2VyaW5nIGluIGNoaWNrcGVhIGluIGEgc2hvcnQtc2Vhc29uIHRlbXBlcmF0ZSBlbnZpcm9ubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEhlcmVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTU1LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzIG9mIGFncm9ub21pYyB0cmFpdHMgaW4gYSB3aWRlIGNyb3NzIG9mIGNoaWNrcGVhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GaWVsZCBDcm9wcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTEzMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGEgbWFqb3IgZ2VuZSBmb3IgZ3Jvd3RoIGhhYml0IGFuZCBRVExzIGZvciBBc2NvY2h5dGEgYmxpZ2h0IHJlc2lzdGFuY2UgYW5kIGZsb3dlcmluZyB0aW1lIGluIGEgcG9wdWxhdGlvbiBiZXR3ZWVuIGNoaWNrcGVhIGFuZCBDaWNlciByZXRpY3VsYXR1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3MzsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1hcHBpbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2VuZXMgZ292ZXJuaW5nIHRpbWUgdG8gZmxvd2VyaW5nLCBzZWVkIHdlaWdodCwgYW5kIHBsYW50IGhlaWdodCBpbiBhbiBpbnRyYXNwZWNpZmljIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgY2hpY2twZWEgKENpY2VyIGFyaWV0aW51bSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1lZGljYWdvIEZMT1dFUklORyBMT0NVUyBUIGhvbW9sb2csIE10RlRhMSwgaXMgYSBrZXkgcmVndWxhdG9yIG9mIGZsb3dlcmluZyB0aW1lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTY7ICBjaXRhdGlvbl9wYWdlcz0yMjA3LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2VuZXRpYyBiYXNpcyBvZiDvrIJvd2VyaW5nIHJlc3BvbnNlcyB0byBzZWFzb25hbCBjdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjI3LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbUZUNCwgYSBob21vbG9nIG9mIEZMT1dFUklORyBMT0NVUyBULCBpcyBwb3NpdGl2ZWx5IHJlZ3VsYXRlZCBieSBFMSBhbmQgZnVuY3Rpb25zIGFzIGEgZmxvd2VyaW5nIHJlcHJlc3NvciBpbiBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTg5MDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjb250cm9sIG9mIGZsb3dlcmluZyB0aW1lIGluIGxlZ3VtZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIE1BRFMtZG9tYWluIHRyYW5zY3JpcHRpb24gZmFjdG9yIGNvbXBsZXhlcyBpbiBBcmFiaWRvcHNpcyBmbG93ZXIgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE1NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgdGhlIEROQS1iaW5kaW5nIE9uZSBaaW5jIEZpbmdlciAoRE9GKSB0cmFuc2NyaXB0aW9uIGZhY3RvciBmYW1pbHkgaW4gcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOTsgIGNpdGF0aW9uX3BhZ2VzPTMyLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQURTIHJlbG9hZGVkOiBldm9sdXRpb24gb2YgdGhlIEFHQU1PVVMgc3ViZmFtaWx5IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE7ICBjaXRhdGlvbl9wYWdlcz03MTctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRDUCB0aHJlZS13YXkgaGFuZHNoYWtlOiBsaW5raW5nIGRldmVsb3BtZW50YWwgcHJvY2Vzc2VzIHdpdGggcGxhbnQgaW1tdW5pdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjM4LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvNS8zNzcvOTk4MjA3L2RzdjAyMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgZGV2ZWxvcGVkIDIxLDQ5OSBnZW5vbWUtd2lkZSBpbnNlcnRpb27igJNkZWxldGlvbiAoSW5EZWwpIG1hcmtlcnMgKDItIHRvIDU0LWJwIGluIHNpbGljbyBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtKSBieSBjb21wYXJpbmcgdGhlIGdlbm9taWMgc2VxdSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ4MzE3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZAZvVLMz7fjJPYR8j9FC6T1vA/gw" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f377%2f348317" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f377%2f348317" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13431" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/5/1/m_cover.gif?Expires=2147483647&amp;Signature=xvv8vRJ-FfvUppijKWF0xFZfVSvzyQTCdlqTLUsfzwFVzAehecEgQ2S6mZY0yRx2f-hRCKeQv8DfoQDM8lMd5vYQgCqoUwn0S9LY5-40K3B5K1Lv9gOINfWoDdErPxCIb-Rwedq9I5ll3EBuknadKshNKIPi7rQ2b1h4wXSSR4sJ5C8FqsRxpoxIt-pXh08eaFz67EfoBSo8Nb7zB052xiRBaK5jRn1tr2IE8No45sk0vZG1Vmc0up6YQdBT~1Gu1ajO7vC-hp2YN3zKqenmGW4xiJby2aNmgQ8F2iOXrFmGCyXrbWdD4ph-9qD0QR7JQZGnUwswoPbGDu~QVIvQ0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/5">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4944019">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944019">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944021">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944021">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944026">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944026">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944036">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944036">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944061">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944061">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944063">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944065">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944067">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944067">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/5/367/348265">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shouvik Das</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shouvik+Das">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Das S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Das S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shouvik Das</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hari D. Upadhyaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Telangana</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hari+D.+Upadhyaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Upadhyaya H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Upadhyaya H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hari D. Upadhyaya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rishi Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rishi+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srivastava R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rishi Srivastava</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepak Bajaj</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deepak+Bajaj">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bajaj D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bajaj D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Deepak Bajaj</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C.L.L. Gowda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Telangana</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=C.L.L.+Gowda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gowda C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gowda C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">C.L.L. Gowda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shivali Sharma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Telangana</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shivali+Sharma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sharma S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sharma S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shivali Sharma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sube Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324, Telangana</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sube+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Singh S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sube Singh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tyagi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Swarup K. Parida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Institute of Plant Genome Research (NIPGR)</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91-11-26735228; Fax. +91-11-26741658; E-mail: <a href="mailto:swarup@nipgr.ac.in" target="_blank">swarup@nipgr.ac.in</a>; <a href="mailto:swarupdbt@gmail.com" target="_blank">swarupdbt@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Swarup+K.+Parida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parida S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parida S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Swarup K. Parida</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="TN2"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 377–386, <a href='https://doi.org/10.1093/dnares/dsv020'>https://doi.org/10.1093/dnares/dsv020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 September 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 May 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 August 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348317" href="/dnaresearch/article-pdf/22/5/377/998207/dsv020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shouvik Das, Hari D. Upadhyaya, Rishi Srivastava, Deepak Bajaj, C.L.L. Gowda, Shivali Sharma, Sube Singh, Akhilesh K. Tyagi, Swarup K. Parida;  Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea, <em>DNA Research</em>, Volume 22, Issue 5, 1 October 2015, Pages 377–386, <a href='https://doi.org/10.1093/dnares/dsv020'>https://doi.org/10.1093/dnares/dsv020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348317?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348317?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20insertion%E2%80%93deletion%20(InDel)%20marker%20discovery%20and%20genotyping%20for%20genomics-assisted%20breeding%20applications%20in%20chickpea&amp;publicationDate=2015-09-17&amp;volumeNum=22&amp;issueNum=5&amp;author=Das%2C%20Shouvik%3B%20Upadhyaya%2C%20Hari%20D.&amp;startPage=377&amp;endPage=386&amp;contentId=10.1093%2Fdnares%2Fdsv020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f377%2f348317" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4944019 id="4944019" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We developed 21,499 genome-wide insertion–deletion (InDel) markers (2- to 54-bp <em>in silico</em> fragment length polymorphism) by comparing the genomic sequences of four (<em>desi</em>, <em>kabuli</em> and wild <em>C. reticulatum</em>) chickpea [<em>Cicer arietinum</em> (L.)] accessions. InDel markers showing 2- to 6-bp fragment length polymorphism among accessions were abundant (76.8%) in the chickpea genome. The physically mapped 7,643 and 13,856 markers on eight chromosomes and unanchored scaffolds, respectively, were structurally and functionally annotated. The 4,506 coding (23% large-effect frameshift mutations) and regulatory InDel markers were identified from 3,228 genes (representing 11.7% of total 27,571 <em>desi</em> genes), suggesting their functional relevance for trait association/genetic mapping. High amplification (97%) and intra-specific polymorphic (60–83%) potential and wider genetic diversity (15–89%) were detected by genome-wide 6,254 InDel markers among <em>desi</em>, <em>kabuli</em> and wild accessions using even a simpler cost-effective agarose gel-based assay. This signifies added advantages of this user-friendly genetic marker system for manifold large-scale genotyping applications in laboratories with limited infrastructure and resources. Utilizing 6,254 InDel markers-based high-density (inter-marker distance: 0.212 cM) inter-specific genetic linkage map (ICC 4958 × ICC 17160) of chickpea as a reference, three major genomic regions harboring six flowering and maturity time robust QTLs (16.4–27.5% phenotypic variation explained, 8.1–11.5 logarithm of odds) were identified. Integration of genetic and physical maps at these target QTL intervals mapped on three chromosomes delineated five InDel markers-containing candidate genes tightly linked to the QTLs governing flowering and maturity time in chickpea. Taken together, our study demonstrated the practical utility of developing and high-throughput genotyping of such beneficial InDel markers at a genome-wide scale to expedite genomics-assisted breeding applications in chickpea.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="chickpea">chickpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;flowering time&quot;">flowering time</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="InDel">InDel</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;maturity time&quot;">maturity time</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="QTL">QTL</a></div>                            <h2 scrollto-destination=4944021 id="4944021" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Chickpea [<em>Cicer arietinum</em> (L.)] is a self-pollinated annual diploid legume with a genome size of ∼740 Mb.<sup><a reveal-id="DSV020C1  DSV020C3" data-open="DSV020C1  DSV020C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Globally, chickpea is the third most important food legume complementing cereals and considered a vital human dietary source of protein abundant in essential amino acids.<sup><a reveal-id="DSV020C2" data-open="DSV020C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV020C3" data-open="DSV020C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In the present era of climatic variability, resource scarcity and fast-growing population, it is necessary to enhance productivity as well as sustainability of agriculture for circumventing the global food insecurity. The prime objective of chickpea genomics-assisted breeding was thus inclined towards developing high-yielding durable stress tolerant (climate resilient) chickpea cultivars to meet the dietary demand of increasing population for attaining food security. To achieve these goals, an effective molecular dissection of these complex quantitative stress tolerance and yield-contributing traits by identification (map-based cloning) of potential trait-associated genes/quantitative trait loci (QTLs) through genetic/association mapping and their deployment in marker-assisted genetic enhancement of chickpea is essential. Significant progress in this regard has been made primarily through development and high-throughput genotyping of numerous genome/gene-derived simple sequence repeat (SSR) and single nucleotide polymorphism (SNP) markers in diverse natural and mapping populations to expedite the process of genetic and association mapping in chickpea.<sup><a reveal-id="DSV020C4  DSV020C12" data-open="DSV020C4  DSV020C12" class="link link-ref link-reveal xref-bibr">4–12</a></sup> Consequently, hitherto very limited number of markers tightly linked to genes/QTLs governing traits of agronomic importance has been identified to be exploited for marker-assisted genetic improvement of chickpea.<sup><a reveal-id="DSV020C13" data-open="DSV020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Several studies have documented the narrow genetic base causing low intra-specific marker polymorphism as the major hindrance for chickpea genetic improvement.<sup><a reveal-id="DSV020C2" data-open="DSV020C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Henceforth, it is imperative to develop and use numerous genome-wide informative sequence-based genetic markers in generating high-density linkage maps and for high-resolution genetic and association mapping in order to delineate functionally relevant trait-regulatory genes/QTLs for genomics-assisted crop improvement of chickpea.</p><p>Insertion-deletions (InDels) are one of the major sources of structural variation found widely distributed across the genomes of diverse plant species, including <em>Arabidopsis</em>, rice, wheat and tomato.<sup><a reveal-id="DSV020C15" data-open="DSV020C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSV020C16" data-open="DSV020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> InDels generally arise due to certain cellular mechanisms, including movement of transposable elements, replication slippage and unequal crossing-over within the genome.<sup><a reveal-id="DSV020C17" data-open="DSV020C17" class="link link-ref link-reveal xref-bibr">17</a></sup> InDel is an important phenomenon, which can exert a deleterious as well as beneficial effect on particular loci within the genome.<sup><a reveal-id="DSV020C18" data-open="DSV020C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSV020C19" data-open="DSV020C19" class="link link-ref link-reveal xref-bibr">19</a></sup> These InDels are valuable complement to other sequence-based genetic markers like SSRs and SNPs and well recognized as an effective marker system for genetic analysis in crop plants primarily due to myriad desirable inherent genetic attributes such as multi-allelic and co-dominant inheritance and wide genomic distribution. Besides, the InDel markers are easily detectable at a genome-wide scale (gene level) <em>in silico</em> with low cost, labor and time via comparison of freely accessible genomic (transcriptomic) sequence resources of different available genotypes through computational genomics tools. In addition to the user-friendly mining approach, these markers can be preferably selected on the basis of their predicted fragment length variations and further validated/genotyped in germplasm lines and mapping populations using a simple cost-effective agarose gel-based assay. This makes InDels an ideal marker system of choice for diverse genomics-assisted breeding applications in crop plants.<sup><a reveal-id="DSV020C15  DSV020C17" data-open="DSV020C15  DSV020C17" class="link link-ref link-reveal xref-bibr">15–17</a>,<a reveal-id="DSV020C20  DSV020C23" data-open="DSV020C20  DSV020C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup></p><p>Much recently, the completion of genome sequencing of <em>desi</em> (cv. ICC 4958) and <em>kabuli</em> (cv. CDC Frontier) cultivars of chickpea has accelerated the process of genome resequencing of diverse <em>desi</em>, <em>kabuli</em> and wild accessions.<sup><a reveal-id="DSV020C2" data-open="DSV020C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> It would be thus interesting to develop informative InDel markers <em>in silico</em> at a genome-wide scale by comparing these publicly accessible genomic (gene) sequences of different chickpea accessions with sub-optimal use of resources. Furthermore, the large-scale validation and genotyping of these developed InDel markers in natural germplasm lines and mapping populations with higher marker genetic polymorphic potential, could be utilized for construction of high-resolution genetic linkage maps as well as molecular mapping and genetic association analysis in order to identify trait-regulating genes/QTLs for marker-assisted genetic enhancement of chickpea.</p><p>In light of the above, the present study has made an effort to develop large-scale genome-wide InDel markers and to determine their genomic constitution in the chickpea genome. The potential of these markers was assessed to detect intra-specific polymorphism and genetic diversity among <em>desi</em>, <em>kabuli</em> and wild accessions, and construct high-density inter-specific genetic linkage map of chickpea. As a proof of concept, this InDel markers-based high-resolution genetic linkage map was utilized as a reference to identify and map the major genomic regions harboring robust QTLs governing flowering and maturity time, with an aim to accelerate genomics-assisted breeding in chickpea.</p>                            <h2 scrollto-destination=4944026 id="4944026" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4944027 id="4944027" class="section-title" data-legacy-id=s2a>2.1. Discovery of genome-wide InDel markers in chickpea</h3>
<p>The high-quality FASTQ sequence reads generated from whole-genome resequencing of four chickpea accessions (<em>desi</em> cv. ICC 4958 and ICC 4951, <em>kabuli</em> cv. ICC 12968 and wild <em>Cicer reticulatum</em> cv. ICC 17160) were acquired.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV020C24" data-open="DSV020C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The mapping of sequence reads of three chickpea accessions onto the chromosome pseudomolecules of reference <em>desi</em> (ICC 4958) genome and detection of high-quality InDels among accessions were performed following Jain et al.<sup><a reveal-id="DSV020C20" data-open="DSV020C20" class="link link-ref link-reveal xref-bibr">20</a></sup> A highly stringent criteria with a minimum read depth of 10 and a minimum sequence variant frequency (polymorphism call rate) of &gt;85% as well as the presence of at least one InDel within a 50-bp window were considered relevant for screening and detection of high-quality (minimal false-positive) InDels among four chickpea accessions. About 200–300 bp high-quality genomic sequences of ICC 4958 flanking the either side of identified InDels were retrieved. The forward and reverse primers targeting these InDels-carrying sequences were designed using the generic primer interface of BatchPrimer3 (<a class="link link-uri" href="http://probes.pw.usda.gov/batchprimer3">http://probes.pw.usda.gov/batchprimer3</a>) to develop genome-wide InDel markers. The genomic distribution of InDel markers on chromosome pseudomolecules of <em>desi</em> chickpea genome was determined and visualized using Circos visualization tool.<sup><a reveal-id="DSV020C11" data-open="DSV020C11" class="link link-ref link-reveal xref-bibr">11</a></sup> To determine the precise distribution of InDel markers in the diverse intergenic as well as coding and non-coding sequence components of genes/genomes (chromosomes/pseudomolecules and unanchored scaffolds), the physical positions (bp) of InDel markers were integrated/correlated with the GFF file containing genome annotation of <em>desi</em> chickpea (CGAP v1.0<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup>) using the customized Perl scripts. The putative function of InDel markers-carrying genes was predicted according to <em>desi</em> genome annotation<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and PFAM database v27.0 (<a class="link link-uri" href="http://pfam.sanger.ac.uk">http://pfam.sanger.ac.uk</a>).</p>                            <h3 scrollto-destination=4944029 id="4944029" class="section-title" data-legacy-id=s2b>2.2. Experimental validation, polymorphism and genetic diversity potential of InDel markers</h3>
<p>To assess the amplification and polymorphic potential of developed InDel markers, the InDel markers showing ≥10 bp <em>in silico</em> fragment length polymorphism among accessions were PCR amplified using the genomic DNA of 24 chickpea accessions. This included four accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160) from which the InDel markers were originally identified, and 20 additional <em>desi</em> (9) and <em>kabuli</em> (11) accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The standard PCR constituents and touchdown thermal cycling profiling for PCR amplification and 2.5% agarose gel to resolve the amplified PCR products were utilized as per Kujur et al.<sup><a reveal-id="DSV020C10" data-open="DSV020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> For experimental validation of InDel markers revealing 2–9 bp <em>in silico</em> fragment length polymorphism among accessions, these markers were PCR amplified using the genomic DNA of 24 aforesaid chickpea accessions. The amplified PCR products were purified and sequenced in both forward and reverse directions twice on a capillary-based automated DNA sequencer (Applied Biosystems, ABI 3730xl DNA Analyzer, Vernon Hills, IL, USA) using the BigDye Terminator v3.1 sequencing kit. The high-quality consensus sequences obtained for each InDel marker were aligned and compared among accessions used. The genotyping data of validated InDel markers were utilized to estimate the average polymorphic alleles per marker, percent polymorphism and polymorphism information content (PIC) among chickpea accessions employing the PowerMarker v3.51.<sup><a reveal-id="DSV020C25" data-open="DSV020C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The genetic diversity and phylogenetic relationships among accessions were determined using the genotyping data of validated polymorphic genome/gene-derived InDel markers. The Nei and Li similarity coefficient-based neighbor joining (NJ) method (with 1,000 bootstrap replicates) of PowerMarker v3.51 was employed for clustering analysis and construction of unrooted phylogenetic tree among accessions.</p>                            <h3 scrollto-destination=4944031 id="4944031" class="section-title" data-legacy-id=s2c>2.3. Genetic linkage map construction</h3>
<p>The InDel markers showing polymorphism between ICC 4958 and ICC 17160 were PCR amplified and genotyped using 190 individuals and parental accessions derived from an F<sub>5</sub> inter-specific mapping population (ICC 4958 × ICC 17160), following the aforementioned methods of agarose gel- and PCR amplicons resequencing-based assays. The genotyping data were analyzed employing the <em>χ</em><sup>2</sup>-test (<em>P</em> &lt; 0.05) to assess their goodness-of-fit to the expected Mendelian 1:1 segregation ratio. The MAPMAKER/EXP 3.0<sup><a reveal-id="DSV020C26" data-open="DSV020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> was used to estimate linkage analysis among the InDel markers, basing upon which different linkage groups (LGs) were classified. For further validation of linkages among markers, the genotyping data of markers grouped by MAPMAKER were analyzed through JoinMap 4.1 (<a class="link link-uri" href="http://www.kyazma.nl/index.php/mc.JoinMap">http://www.kyazma.nl/index.php/mc.JoinMap</a>) at higher logarithm of odds (LOD) threshold (4.0–8.0) with Kosambi mapping function. The InDel markers were incorporated into defined LGs (designated LG1 to LG8) based on their centiMorgan (cM) genetic distances and corresponding marker physical positions (bp) on the chromosomes. An inter-specific genetic linkage map was finally constructed and visualized using MapChart v2.2.<sup><a reveal-id="DSV020C10" data-open="DSV020C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p>                            <h3 scrollto-destination=4944033 id="4944033" class="section-title" data-legacy-id=s2d>2.4. QTL mapping</h3>
<p>The 190 individuals along with parental accessions of an inter-specific mapping population (ICC 4958 × ICC 17160) were grown in the field for two consecutive years during crop-growing season at two diverse geographical locations of India. Additionally, greenhouse trial was performed to measure the flowering and maturity time response of mapping individuals under both long- and short-day conditions at 22 ± 2°C. The days to 50% flowering (DF) and maturity (DM) time of each mapping individual (10–12 representative plants from each individual) was measured by counting the number of days from sowing (first irrigation) to the stage when 50% of their plants have begun to flower and 90% of pods have matured and turned yellow, respectively. The homogeneity, frequency distribution, coefficient of variation (CV) and broad-sense heritability (H<sup>2</sup>) of DF and DM traits in mapping population were determined following Bajaj et al.<sup><a reveal-id="DSV020C27" data-open="DSV020C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>For QTL mapping, the genotyping data of InDel markers physically/genetically mapped on eight LGs/chromosomes and field phenotypic data (DF and DM) of 190 mapping individuals and parental accessions were correlated using composite interval mapping (CIM) function of MapQTL 6.<sup><a reveal-id="DSV020C28" data-open="DSV020C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The LOD threshold score of more than 4.0 at 1,000 permutations was considered significant (<em>P</em> &lt; 0.05) to identify and map the major QTLs on LGs governing DF and DM traits in chickpea. At significant LOD, the positional genetic effect and phenotypic variation explained (PVE) by QTLs were measured. QTL Network v2.0 was used to determine the additive effect of marker loci underlying the QTLs. At 1.5-LOD support intervals (95% bootstrap CI), the confidence intervals (CI) around each significant major QTL peak was estimated.</p>                            <h2 scrollto-destination=4944036 id="4944036" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4944037 id="4944037" class="section-title" data-legacy-id=s3a>3.1. Genomic constitution of InDel markers in chickpea genome</h3>
<p>The comparison of resequencing data of three chickpea accessions ICC 4951 (cultivated <em>desi</em>), ICC 12968 (cultivated <em>kabuli</em>) and ICC 17160 (wild <em>C. reticulatum</em>) with reference genome of <em>desi</em> accession (ICC 4958) developed a total of 21,499 InDel markers (Fig. <a reveal-id="DSV020F1" data-open="DSV020F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). These developed markers with diverse potential of detecting <em>in silico</em> InDel polymorphism in all possible-pair combination of four different chickpea accessions were physically mapped on the eight chromosomes and unanchored scaffolds of <em>desi</em> chickpea genome (Fig. <a reveal-id="DSV020F2" data-open="DSV020F2" class="link link-reveal link-table xref-fig">2</a>A–D). A maximum of 16 839 (78.3%) markers showing <em>in silico</em> InDel-based fragment length polymorphism between ICC 4958 and ICC 17160, followed by 4699 (21.8%) markers between ICC 4958 and ICC 12968 and 3252 (15.1%) markers between ICC 4958 and ICC 4951, and minimum of 1173 (5.4%) markers between ICC 4951 and ICC 17160 were identified (Fig. <a reveal-id="DSV020F2" data-open="DSV020F2" class="link link-reveal link-table xref-fig">2</a>A–C). Interestingly, markers designed from 694 (3.2%) InDels were found to be commonly polymorphic among all four chickpea accessions used. The observed InDel marker-based genetic polymorphism overall reflect the close phylogenetic relatedness of <em>desi</em> rather than <em>kabuli</em> with wild accessions and evolutionary divergence of cultivated accessions from a common wild progenitor <em>C. reticulatum</em> as well as existence of a domestication-led bottleneck in cultivated <em>desi</em> and <em>kabuli</em> chickpea.<sup><a reveal-id="DSV020C2" data-open="DSV020C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV020C29" data-open="DSV020C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSV020C30" data-open="DSV020C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Remarkably, <em>in silico</em> fragment length polymorphism detected by markers based on their size (bp) of InDels varied from 2 to 54 bp with an average of 6 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). More than 76.8% (16,510) InDel markers showed 2 to 6 bp <em>in silico</em> fragment length polymorphism, whereas remaining 23.2% (4,989) markers had 7 to 54 bp fragment length polymorphism (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>). This trend of proportionate distribution of two different kinds of fragment length polymorphisms (2 to 6 bp: 75.3 to 87.5% and 7 to 54 bp: 12.5 to 24.7%) showing InDel markers detected in each possible-pair combination of four chickpea accessions used, remained almost similar (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>). A relatively higher frequency of markers with smaller size InDels (2 to 6 bp) identified in chickpea could be due to the use of NGS-based short sequence reads (&lt;200 bp) and/or user-specific computational genomics tools/algorithms for detection of InDels among chickpea accessions at a genome-wide scale.<sup><a reveal-id="DSV020C33" data-open="DSV020C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p>                        <a id="4944039" scrollto-destination="4944039"></a>
<div id="DSV020F1" data-id="DSV020F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02001.png?Expires=2147483647&Signature=WLpcnNu7AyyxriQr9I98qnxjItaqIyDgm~jnZkUk2uMtqYmQNKELLyMqD2kmH7ZNrdlrK36JZ1SfytiwSmUqFURlC2BgxXzLGOEj1eoPaK2Em0JqqLwZQwH0I2qHF6qBoqs-~BYkbPRR4aCyVj1eDivI00LnuXr3yMTxtVNc2ADn1AvE5oceCytx6tBlEuN5CQ1qCOjFWB54SYV7besUj3QdPVvmo9e6P2t1NUr8uUpf4SNW4ZulicL6CRxJnaiRHkLIo~WuVRbc3FTp~HN7E-PFle3BhTa8CJtmxCQR6VHUdV0BE-zWSJNkQmqINjaUe4t9SMd9g7PUbl9VfsNGQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic illustrating the major steps followed for successful development, validation and genotyping of InDel markers derived from diverse intergenic and gene (coding and non-coding) sequence components of chickpea genome for their genomics-assisted breeding applications. The forward (F) and reverse (R) primers designed from the sequences flanking the target InDels were developed as InDel markers and amplified in a diverse set of desi, kabuli and wild natural chickpea accessions and individuals of a mapping population. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944039/dsv02001.tiff" path-from-xml="dsv02001.tiff" target="_blank">View large</a><a section="4944039" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02001.png?Expires=2147483647&Signature=2NYadMJbUCklogu2fKJOm9q9g8x03b36URrwizPEFt6PrHI2g4y0Zicvs7YxTLrGAAjnj3U3czjwVc4hnHdsGI3wcv3TwGCsGYhf0h7uSqjoeyEGXwU93oYWvHI6sbv8UhNnIYlCYySQ3N0J9FHp1lgOR62dg2~xPucJSm0UPfSxHxv2VCTr2ct55YAlm5-dTQ6wNWu1~T9~hd7Wvq0fAmWj0IjHPmv4Bo3mj8fkQdav-SeAEz~oIEh37gxujllXimjcJoSpsaqNjvI8MtQ~9sTAGR3gYKYNkepSxNabiof~6f94kfmgnFdwqbNyMcxv-yUPZwbnSN8C1wPFZsF8fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944039&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic illustrating the major steps followed for successful development, validation and genotyping of InDel markers derived from diverse intergenic and gene (coding and non-coding) sequence components of chickpea genome for their genomics-assisted breeding applications. The forward (F) and reverse (R) primers designed from the sequences flanking the target InDels were developed as InDel markers and amplified in a diverse set of <em>desi</em>, <em>kabuli</em> and wild natural chickpea accessions and individuals of a mapping population. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV020F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02001.png?Expires=2147483647&Signature=WLpcnNu7AyyxriQr9I98qnxjItaqIyDgm~jnZkUk2uMtqYmQNKELLyMqD2kmH7ZNrdlrK36JZ1SfytiwSmUqFURlC2BgxXzLGOEj1eoPaK2Em0JqqLwZQwH0I2qHF6qBoqs-~BYkbPRR4aCyVj1eDivI00LnuXr3yMTxtVNc2ADn1AvE5oceCytx6tBlEuN5CQ1qCOjFWB54SYV7besUj3QdPVvmo9e6P2t1NUr8uUpf4SNW4ZulicL6CRxJnaiRHkLIo~WuVRbc3FTp~HN7E-PFle3BhTa8CJtmxCQR6VHUdV0BE-zWSJNkQmqINjaUe4t9SMd9g7PUbl9VfsNGQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic illustrating the major steps followed for successful development, validation and genotyping of InDel markers derived from diverse intergenic and gene (coding and non-coding) sequence components of chickpea genome for their genomics-assisted breeding applications. The forward (F) and reverse (R) primers designed from the sequences flanking the target InDels were developed as InDel markers and amplified in a diverse set of desi, kabuli and wild natural chickpea accessions and individuals of a mapping population. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944039/dsv02001.tiff" path-from-xml="dsv02001.tiff" target="_blank">View large</a><a section="4944039" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02001.png?Expires=2147483647&Signature=2NYadMJbUCklogu2fKJOm9q9g8x03b36URrwizPEFt6PrHI2g4y0Zicvs7YxTLrGAAjnj3U3czjwVc4hnHdsGI3wcv3TwGCsGYhf0h7uSqjoeyEGXwU93oYWvHI6sbv8UhNnIYlCYySQ3N0J9FHp1lgOR62dg2~xPucJSm0UPfSxHxv2VCTr2ct55YAlm5-dTQ6wNWu1~T9~hd7Wvq0fAmWj0IjHPmv4Bo3mj8fkQdav-SeAEz~oIEh37gxujllXimjcJoSpsaqNjvI8MtQ~9sTAGR3gYKYNkepSxNabiof~6f94kfmgnFdwqbNyMcxv-yUPZwbnSN8C1wPFZsF8fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944039&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic illustrating the major steps followed for successful development, validation and genotyping of InDel markers derived from diverse intergenic and gene (coding and non-coding) sequence components of chickpea genome for their genomics-assisted breeding applications. The forward (F) and reverse (R) primers designed from the sequences flanking the target InDels were developed as InDel markers and amplified in a diverse set of <em>desi</em>, <em>kabuli</em> and wild natural chickpea accessions and individuals of a mapping population. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="4944040" scrollto-destination="4944040"></a>
<div id="DSV020F2" data-id="DSV020F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02002.png?Expires=2147483647&Signature=CQqIKC2GZoVGa656CwGnYb9hu3oIakXInyuZQChOFhkC4-NJDJ31uRVxoccvUwl-wF9brr5tLGVp~1BudkRe16txHR9O5rcWQ~ASa5jtw~4SHf2OJywj0H2eTco7pv64Slhov2DCuDKqLQwec5jD7dSsXtX2bLE44KScq996TMgPREuvBPBE6Ii4mJmKl5gtWS9yqBzbs7A8uGmXGna11t8K1j~E2G~MAJ8LSI3jo4KJCVoMaZKhTvVxOmJ6y0XoiD5hoAGaO3QiPgVtNzVRLRNBloPKhErPHk-v1AEFmUG4riOEoAHCHJ2m~NP5i9YorvhU5lZXzcP2N5VXMEoNMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic constitution and genome-wide distribution pattern of 21,499 InDel markers developed by comparing the genome sequences of four different desi (ICC 4958 and ICC 4951), kabuli (ICC 12968) and wild (ICC 17160) chickpea accessions. InDel markers identified from diverse possible-pair combination (indicated by individual circles) of four accessions that were physically mapped on eight chromosomes (A) and unanchored scaffolds (B) as well as sum of the whole chickpea genome (C) are depicted by Venn diagrams. (D) The frequency and relative distribution of 7,643 InDel markers physically mapped on eight chromosomes of desi chickpea genome are illustrated by a Circos circular ideogram. The outermost circles signify the different physical size (Mb) of eight chromosomes coded with multiple colors as per the pseudomolecule size documented in desi chickpea genome.14 (E) Relative frequency of 21,499 InDel markers identified from the intergenic and various coding and non-coding (introns and regulatory regions) sequence components of 3,228 genes annotated from desi genome. Parenthesis indicates the proportion (%) of InDel markers. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of protein-coding genes were defined according to the available gene annotation of desi chickpea genome.14 This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944040/dsv02002.tiff" path-from-xml="dsv02002.tiff" target="_blank">View large</a><a section="4944040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02002.png?Expires=2147483647&Signature=DF6WtbbSVS0RRuKOWAXTytefxjJAO63H3v4C4SZhRG9nma4XQoJhIapuygq7neB4hTHNbynjSOZulwQt2LattHH802bFx4tT23XoU7pQi8nKC3o5VTgZWcYFdSVddmbMzzt5EB5ligbi0EsSzZC9KymvjEx5FwBc79P7pTDO65VSuaarVa1JT2Pq-G5tthUYsIepr-sn0RbVMkAFBD5ps3gqOOrFGbq8VpgqvjPBt4ucqHbXoEyFwiP0GBnGU3qKmLDmiK96Lf1CNhOIjSP7gUe7ifKZzJoSpGZiB3EKpfC6~KqkvK2wAZ-yzAQgjdq0CR-yZSTxOSVDQCV7xEgLOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944040&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic constitution and genome-wide distribution pattern of 21,499 InDel markers developed by comparing the genome sequences of four different <em>desi</em> (ICC 4958 and ICC 4951), <em>kabuli</em> (ICC 12968) and wild (ICC 17160) chickpea accessions. InDel markers identified from diverse possible-pair combination (indicated by individual circles) of four accessions that were physically mapped on eight chromosomes (A) and unanchored scaffolds (B) as well as sum of the whole chickpea genome (C) are depicted by Venn diagrams. (D) The frequency and relative distribution of 7,643 InDel markers physically mapped on eight chromosomes of <em>desi</em> chickpea genome are illustrated by a Circos circular ideogram. The outermost circles signify the different physical size (Mb) of eight chromosomes coded with multiple colors as per the pseudomolecule size documented in <em>desi</em> chickpea genome.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (E) Relative frequency of 21,499 InDel markers identified from the intergenic and various coding and non-coding (introns and regulatory regions) sequence components of 3,228 genes annotated from <em>desi</em> genome. Parenthesis indicates the proportion (%) of InDel markers. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of protein-coding genes were defined according to the available gene annotation of <em>desi</em> chickpea genome.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV020F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02002.png?Expires=2147483647&Signature=CQqIKC2GZoVGa656CwGnYb9hu3oIakXInyuZQChOFhkC4-NJDJ31uRVxoccvUwl-wF9brr5tLGVp~1BudkRe16txHR9O5rcWQ~ASa5jtw~4SHf2OJywj0H2eTco7pv64Slhov2DCuDKqLQwec5jD7dSsXtX2bLE44KScq996TMgPREuvBPBE6Ii4mJmKl5gtWS9yqBzbs7A8uGmXGna11t8K1j~E2G~MAJ8LSI3jo4KJCVoMaZKhTvVxOmJ6y0XoiD5hoAGaO3QiPgVtNzVRLRNBloPKhErPHk-v1AEFmUG4riOEoAHCHJ2m~NP5i9YorvhU5lZXzcP2N5VXMEoNMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic constitution and genome-wide distribution pattern of 21,499 InDel markers developed by comparing the genome sequences of four different desi (ICC 4958 and ICC 4951), kabuli (ICC 12968) and wild (ICC 17160) chickpea accessions. InDel markers identified from diverse possible-pair combination (indicated by individual circles) of four accessions that were physically mapped on eight chromosomes (A) and unanchored scaffolds (B) as well as sum of the whole chickpea genome (C) are depicted by Venn diagrams. (D) The frequency and relative distribution of 7,643 InDel markers physically mapped on eight chromosomes of desi chickpea genome are illustrated by a Circos circular ideogram. The outermost circles signify the different physical size (Mb) of eight chromosomes coded with multiple colors as per the pseudomolecule size documented in desi chickpea genome.14 (E) Relative frequency of 21,499 InDel markers identified from the intergenic and various coding and non-coding (introns and regulatory regions) sequence components of 3,228 genes annotated from desi genome. Parenthesis indicates the proportion (%) of InDel markers. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of protein-coding genes were defined according to the available gene annotation of desi chickpea genome.14 This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944040/dsv02002.tiff" path-from-xml="dsv02002.tiff" target="_blank">View large</a><a section="4944040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02002.png?Expires=2147483647&Signature=DF6WtbbSVS0RRuKOWAXTytefxjJAO63H3v4C4SZhRG9nma4XQoJhIapuygq7neB4hTHNbynjSOZulwQt2LattHH802bFx4tT23XoU7pQi8nKC3o5VTgZWcYFdSVddmbMzzt5EB5ligbi0EsSzZC9KymvjEx5FwBc79P7pTDO65VSuaarVa1JT2Pq-G5tthUYsIepr-sn0RbVMkAFBD5ps3gqOOrFGbq8VpgqvjPBt4ucqHbXoEyFwiP0GBnGU3qKmLDmiK96Lf1CNhOIjSP7gUe7ifKZzJoSpGZiB3EKpfC6~KqkvK2wAZ-yzAQgjdq0CR-yZSTxOSVDQCV7xEgLOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944040&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic constitution and genome-wide distribution pattern of 21,499 InDel markers developed by comparing the genome sequences of four different <em>desi</em> (ICC 4958 and ICC 4951), <em>kabuli</em> (ICC 12968) and wild (ICC 17160) chickpea accessions. InDel markers identified from diverse possible-pair combination (indicated by individual circles) of four accessions that were physically mapped on eight chromosomes (A) and unanchored scaffolds (B) as well as sum of the whole chickpea genome (C) are depicted by Venn diagrams. (D) The frequency and relative distribution of 7,643 InDel markers physically mapped on eight chromosomes of <em>desi</em> chickpea genome are illustrated by a Circos circular ideogram. The outermost circles signify the different physical size (Mb) of eight chromosomes coded with multiple colors as per the pseudomolecule size documented in <em>desi</em> chickpea genome.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (E) Relative frequency of 21,499 InDel markers identified from the intergenic and various coding and non-coding (introns and regulatory regions) sequence components of 3,228 genes annotated from <em>desi</em> genome. Parenthesis indicates the proportion (%) of InDel markers. The CDS (coding sequences), URR (upstream regulatory region) and DRR (downstream regulatory region) of protein-coding genes were defined according to the available gene annotation of <em>desi</em> chickpea genome.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Of the designed 21,499 InDel markers, 7,643 and 13,856 markers were physically mapped on eight chromosomes (Fig. <a reveal-id="DSV020F2" data-open="DSV020F2" class="link link-reveal link-table xref-fig">2</a>D) and unanchored scaffolds of <em>desi</em> chickpea genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Highest and lowest number of InDel markers were mapped on chromosomes 3 (1,468 markers with a mean map density: 15.9 kb) and 7 (488 markers with a mean map density: 17.3 kb), respectively. Interestingly, almost a similar level of map density of InDel markers was observed in all eight chickpea chromosomes, while it ranged from 15.4 kb for chromosome 6 to 18.8 kb in chromosome 8 with an average of 16.3 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Notably, only 120 (23.1%) of 520 Mb draft scaffold sequence (covering ∼70% of 740 Mb estimated genome size of <em>desi</em> chickpea) has been anchored onto eight chromosome pseudomolecules of chickpea genome so far.<sup><a reveal-id="DSV020C14" data-open="DSV020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The probability of physical mapping of InDel markers derived from the unanchored scaffolds onto chromosomes can be enriched by a subsequent increase in anchoring proportion of these scaffolds across the eight chickpea chromosomes. The detailed structural annotation of 21,499 InDel markers exhibited the presence of 16,993 (79%) and 4,506 (21%) markers in the intergenic regions and 3,228 protein-coding genes, respectively (Fig. <a reveal-id="DSV020F2" data-open="DSV020F2" class="link link-reveal link-table xref-fig">2</a>E). Notably, we observed the presence of InDel markers in the 3,228 genes representing 11.7% of the total genes (27,571) annotated from <em>desi</em> chickpea genome. This is almost comparable with the proportionate occurrence (9–10%) of InDels-carrying genes documented in many crop plants, including sorghum and <em>Arabidopsis</em>.<sup><a reveal-id="DSV020C23" data-open="DSV020C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The gene-derived InDel markers included a maximum of 3,420 (75.9%) markers from the intronic sequence components of 2,238 genes, followed by 544 (12.1%) markers from the CDS of 487 genes, 290 (6.4%) markers from the downstream regulatory regions (DRRs) of 278 genes and minimum of 252 (5.6%) markers from the upstream regulatory regions (URRs) of 225 genes (Fig. <a reveal-id="DSV020F2" data-open="DSV020F2" class="link link-reveal link-table xref-fig">2</a>E). Among the coding sequence variants, 87% of InDel markers affected initiation and stop codons as well as frameshift mutations. Interestingly, in both the genes and intergenic regions of <em>desi</em> chickpea genome, we observed a significantly higher relative distribution of InDel markers revealing 2 to 6 bp (75.7 to 77.1%) than 7 to 54 bp (22.9 to 24.3%) <em>in silico</em> fragment length polymorphism among four chickpea accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>). The functional annotation of 4,506 InDel markers-containing 3,228 genes revealed their maximum correspondence with genes encoding growth, development and metabolism-related proteins (34%), followed by transcription factors (14%) and signal transduction (10%) and disease resistance-related proteins (6%), and minimum to transposons/retrotransposons (1%). In contrast to previous reports (60–70%) in many crop plants,<sup><a reveal-id="DSV020C23" data-open="DSV020C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSV020C31" data-open="DSV020C31" class="link link-ref link-reveal xref-bibr">31</a></sup> a lower abundance of transposons/retrotransposons-associated InDel markers was observed in chickpea genome.</p><p>Collectively, the structurally and functionally annotated InDel markers derived from the diverse sequence components of the genome/genes are specifically responsible for frameshift mutations due to disruption of ORFs (open-reading frames) in the coding regions. These probably have large impact on transcriptional gene regulation (expression) leading to alteration of gene functions in chickpea. While comparing the average frequency between coding-InDel polymorphism and non-synonymous SNP substitutions detected among four chickpea accessions, we observed a higher frequency of non-synonymous mutations (506 SNPs/Mb) than that of InDels (252 InDels/Mb) causing frameshift mutations in the coding regions of chickpea genes.</p>                            <h3 scrollto-destination=4944043 id="4944043" class="section-title" data-legacy-id=s3b>3.2. Amplification and polymorphic potential of InDel markers</h3>
<p>To determine the amplification and polymorphic potential of designed InDel markers, 6,580, including 3,042 and 3,538 InDel markers revealing 10–54 and 2–9 bp <em>in silico</em> fragment length polymorphism among four chickpea accessions, were selected for experimental validation using the agarose gel- and PCR amplicons resequencing-based assays, respectively. The overall criteria used to select InDel markers from the total 21,499 designed markers for our experimental validation study is depicted in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S2</a>. All the 6,580 selected markers were validated primarily by gel- and sequencing-based assays using the genomic DNA of four chickpea accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160), from which the InDel markers were originally detected. This included 694 markers showing <em>in silico</em> fragment length polymorphism commonly among all four chickpea accessions. Notably, 6,482 of 6,580 markers produced single reproducible PCR amplicons in 2.5% agarose gel with a mean amplification success rate of 98.5% (Fig. <a reveal-id="DSV020F3" data-open="DSV020F3" class="link link-reveal link-table xref-fig">3</a>A). Of these, 6,254 (96.5%) amplified markers showing <em>in silico</em> polymorphism at least between two combination of four chickpea accessions were validated experimentally using both agarose gel- and PCR amplicons resequencing-based assays (Fig. <a reveal-id="DSV020F3" data-open="DSV020F3" class="link link-reveal link-table xref-fig">3</a>A, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S3</a>). The validation and genotyping of InDel markers particularly by PCR amplicons resequencing-based assay confirmed the presence of expected InDels, which further corresponded well with their <em>in silico</em> fragment length polymorphism detected among four chickpea accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S3</a>). </p>                        <a id="4944045" scrollto-destination="4944045"></a>
<div id="DSV020F3" data-id="DSV020F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02003.png?Expires=2147483647&Signature=QZ1VVtGBZ-tG~zb3169K5PHqTEblLFCp9kqlEoLQ89hPOrCeQv9UJdOUOeqC37auJZKyjoQt9UHmsGZ3SVTvCgUXf7lJQEbaUUZmCtzJdHCUVpiSwzNHxhmoErahbIICMc3tp-3~J5IolKJIy1EwEe6~KCeSQ4ALxO1dO~BDHb1XIzdMLKTsWWxqrULYCjL~dDJCf95Q4yJGWETxHpsy7-shzf9YASIKfUplFRp5T4pJ51qx5Qx941ps-czc3sYpEU1WpCgV6M4EgA2o~SWDbv9FaYGFWsiAY9SZzoVJHnNKaaDC9kcUE~-Qz3EEgAMSqdaMvp4e3Azyc8yC4Yx2Hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Allelic polymorphism detected among 24 desi, kabuli and wild chickpea accessions along with four control accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160; from which sequences the InDel markers were originally identified) using a representative intergenic InDel marker (CID_C_1862732) in gel-based assay. A maximum number of three polymorphic alleles was amplified among accessions. The detailed information regarding accessions are provided in the Supplementary Table S1. (B) Allelic segregation pattern of a selected intronic InDel marker (CID_C_11555849) in a representative set of 22 individuals and two parental accessions of an inter-specific mapping population (ICC 4958 × ICC 17160) scanned using agarose gel-based assay. Asterisk indicates the heterozygous alleles amplified by this InDel marker in two selected mapping individuals. The amplified fragment sizes (bp) of the polymorphic alleles are indicated. M: 50 bp DNA ladder size standard. The identities of two InDel markers with their detailed information are mentioned in the Supplementary Table S3." path-from-xml="dsv02003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944045/dsv02003.tiff" path-from-xml="dsv02003.tiff" target="_blank">View large</a><a section="4944045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02003.png?Expires=2147483647&Signature=LfmcWw~OrTAqZMnV~AL~1OjeyR88hnturIPOLscMTCFSkqfBzJRSC-U0~H96MepEQB9efw1LeNbnzyXSxgS-jxZGI5Q7Rmf-85iTv79KBRfv0OE7x7cNQXMH~4QEuxdxHZLBuVPlOKxp5ZKqjFbx1SwHB~pj~hrwwbekl8Vkw3ycawIrYoLNMkcUHe4DakH2SugfmzBN8-UghDn~SahYaixYJLRbDCA8qH9geCYCaLAPjJvyC82vMyFA338xWu~uLMtI4onRk8zGpTycFfJnlP4AN7f1oPGCchfc4gu8LhDFfeXsIJOjDQ1L-XW7u-lMrZsu5aCvPbPD-WbtvbgGyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944045&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Allelic polymorphism detected among 24 <em>desi</em>, <em>kabuli</em> and wild chickpea accessions along with four control accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160; from which sequences the InDel markers were originally identified) using a representative intergenic InDel marker (CID_C_1862732) in gel-based assay. A maximum number of three polymorphic alleles was amplified among accessions. The detailed information regarding accessions are provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. (B) Allelic segregation pattern of a selected intronic InDel marker (CID_C_11555849) in a representative set of 22 individuals and two parental accessions of an inter-specific mapping population (ICC 4958 × ICC 17160) scanned using agarose gel-based assay. Asterisk indicates the heterozygous alleles amplified by this InDel marker in two selected mapping individuals. The amplified fragment sizes (bp) of the polymorphic alleles are indicated. M: 50 bp DNA ladder size standard. The identities of two InDel markers with their detailed information are mentioned in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>.</p></div></div><div content-id="DSV020F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02003.png?Expires=2147483647&Signature=QZ1VVtGBZ-tG~zb3169K5PHqTEblLFCp9kqlEoLQ89hPOrCeQv9UJdOUOeqC37auJZKyjoQt9UHmsGZ3SVTvCgUXf7lJQEbaUUZmCtzJdHCUVpiSwzNHxhmoErahbIICMc3tp-3~J5IolKJIy1EwEe6~KCeSQ4ALxO1dO~BDHb1XIzdMLKTsWWxqrULYCjL~dDJCf95Q4yJGWETxHpsy7-shzf9YASIKfUplFRp5T4pJ51qx5Qx941ps-czc3sYpEU1WpCgV6M4EgA2o~SWDbv9FaYGFWsiAY9SZzoVJHnNKaaDC9kcUE~-Qz3EEgAMSqdaMvp4e3Azyc8yC4Yx2Hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Allelic polymorphism detected among 24 desi, kabuli and wild chickpea accessions along with four control accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160; from which sequences the InDel markers were originally identified) using a representative intergenic InDel marker (CID_C_1862732) in gel-based assay. A maximum number of three polymorphic alleles was amplified among accessions. The detailed information regarding accessions are provided in the Supplementary Table S1. (B) Allelic segregation pattern of a selected intronic InDel marker (CID_C_11555849) in a representative set of 22 individuals and two parental accessions of an inter-specific mapping population (ICC 4958 × ICC 17160) scanned using agarose gel-based assay. Asterisk indicates the heterozygous alleles amplified by this InDel marker in two selected mapping individuals. The amplified fragment sizes (bp) of the polymorphic alleles are indicated. M: 50 bp DNA ladder size standard. The identities of two InDel markers with their detailed information are mentioned in the Supplementary Table S3." path-from-xml="dsv02003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944045/dsv02003.tiff" path-from-xml="dsv02003.tiff" target="_blank">View large</a><a section="4944045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02003.png?Expires=2147483647&Signature=LfmcWw~OrTAqZMnV~AL~1OjeyR88hnturIPOLscMTCFSkqfBzJRSC-U0~H96MepEQB9efw1LeNbnzyXSxgS-jxZGI5Q7Rmf-85iTv79KBRfv0OE7x7cNQXMH~4QEuxdxHZLBuVPlOKxp5ZKqjFbx1SwHB~pj~hrwwbekl8Vkw3ycawIrYoLNMkcUHe4DakH2SugfmzBN8-UghDn~SahYaixYJLRbDCA8qH9geCYCaLAPjJvyC82vMyFA338xWu~uLMtI4onRk8zGpTycFfJnlP4AN7f1oPGCchfc4gu8LhDFfeXsIJOjDQ1L-XW7u-lMrZsu5aCvPbPD-WbtvbgGyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944045&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Allelic polymorphism detected among 24 <em>desi</em>, <em>kabuli</em> and wild chickpea accessions along with four control accessions (ICC 4958, ICC 4951, ICC 12968 and ICC 17160; from which sequences the InDel markers were originally identified) using a representative intergenic InDel marker (CID_C_1862732) in gel-based assay. A maximum number of three polymorphic alleles was amplified among accessions. The detailed information regarding accessions are provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. (B) Allelic segregation pattern of a selected intronic InDel marker (CID_C_11555849) in a representative set of 22 individuals and two parental accessions of an inter-specific mapping population (ICC 4958 × ICC 17160) scanned using agarose gel-based assay. Asterisk indicates the heterozygous alleles amplified by this InDel marker in two selected mapping individuals. The amplified fragment sizes (bp) of the polymorphic alleles are indicated. M: 50 bp DNA ladder size standard. The identities of two InDel markers with their detailed information are mentioned in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>.</p></div></div><p>The 6,254 informative polymorphic InDel markers were further genotyped in a diverse set of 24 <em>desi, kabuli</em> and wild accessions to assess their potential for detecting polymorphism among these chickpea accessions (Fig. <a reveal-id="DSV020F3" data-open="DSV020F3" class="link link-reveal link-table xref-fig">3</a>A, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S3</a>). These markers overall produced 15,623 alleles (with an average PIC of 0.71) among accessions. The number of alleles detected by these markers varied from 1 to 3 with an average of 2.5 alleles per marker. Six thousand forty-eight (96.7%, mean PIC: 0.65) of 6,254 markers showed polymorphism between cultivated and wild chickpea accessions. Five thousand one hundred ninty-two (83%, mean PIC: 0.60) of 6,254 markers were found to be polymorphic between <em>desi</em> and <em>kabuli</em>. Notably, 4,315 (69%) markers revealed polymorphism among 11 <em>desi</em> accessions (1 to 3 alleles with mean PIC: 0.57), whereas 3,746 (59.9%) markers exhibited polymorphism among 12 <em>kabuli</em> accessions (1 to 2 alleles with mean PIC: 0.51).</p><p>The inter-specific (96.7%) and also intra-specific (60–83%) polymorphic potential detected by the InDel markers among <em>desi</em>, <em>kabuli</em> and wild chickpea accessions is much higher than that estimated especially with random genome-wide SSR and SNP markers (∼35%) as well as <em>in silico</em> polymorphic SSR and SNP markers (50–60%).<sup><a reveal-id="DSV020C5" data-open="DSV020C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV020C7" data-open="DSV020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV020C32" data-open="DSV020C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSV020C33" data-open="DSV020C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The informative InDel markers with high intra-specific polymorphic potential developed by us at a genome-wide scale could serve as a valuable resource for their immense use in genomics-assisted breeding applications of chickpea. The aforementioned marker validation outcomes specifically inferred that 3,042 validated InDel markers revealing 10–54 bp <em>in silico</em> fragment length polymorphism among <em>desi</em>, <em>kabuli</em> and wild accessions have efficiency to be resolved and genotyped even by a simpler cost-effective agarose gel-based assay using numerous chickpea accessions.</p>                            <h3 scrollto-destination=4944048 id="4944048" class="section-title" data-legacy-id=s3c>3.3. Genetic diversity potential of InDel markers</h3>
<p>The evaluation of genetic diversity level based on pair-wise similarity among 24 <em>desi</em>, <em>kabuli</em> and wild chickpea accessions using 6,254 genome/gene-derived polymorphic InDel markers revealed a wide range of distance coefficient from 0.15 to 0.89 with an average of 0.57. The genetic distance among 23 cultivated <em>desi</em> and <em>kabuli</em> accessions ranged from 0.20 to 0.81 with an average of 0.51. The level of genetic diversity (15 to 89%) measured among the accessions using the InDel markers is comparatively much higher than that estimated previously with the microsatellite and SNP markers.<sup><a reveal-id="DSV020C5" data-open="DSV020C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV020C7" data-open="DSV020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV020C10" data-open="DSV020C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV020C11" data-open="DSV020C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV020C34" data-open="DSV020C34" class="link link-ref link-reveal xref-bibr">34</a></sup> A higher efficiency of InDel markers in assaying genetic diversity implies their significance in establishing distinctness among cultivated (<em>desi</em> and <em>kabuli</em>) and wild accessions and thus could be deployed in genomics-assisted varietal improvement program in chickpea. The genetic relationship among 24 <em>desi</em>, <em>kabuli</em> and wild chickpea accessions was depicted by an unrooted phylogenetic tree (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S4</a>). The genome/gene-derived InDel markers clearly discriminated all these accessions from each other and clustered into two major <em>desi</em> and <em>kabuli</em> groups. One wild <em>C. reticulatum</em> accession ICC 17160 being divergent from cultivated accessions; however, showing close evolutionary relationships with <em>desi</em> chickpea was clustered within a <em>desi</em> group. This overall reflects the correspondence of clustering patterns observed among these accessions with the known cultivar-specific origination, pedigree relationships and parentage.<sup><a reveal-id="DSV020C5" data-open="DSV020C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV020C7" data-open="DSV020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV020C10" data-open="DSV020C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV020C11" data-open="DSV020C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV020C34" data-open="DSV020C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=4944050 id="4944050" class="section-title" data-legacy-id=s3d>3.4. Generation of a high-density InDel marker-based inter-specific chickpea genetic linkage map</h3>
<p>For constructing a high-resolution inter-specific genetic linkage map, 6,219 InDel markers revealing polymorphism between parental accession ICC 4958 and ICC 17160 were genotyped among 190 individuals of a F<sub>5</sub> mapping population (ICC 4958 × ICC 17160). The co-dominant inheritance pattern of the InDel markers was evident from their efficiency to discriminate homozygous mapping individuals of either of the parental accessions from heterozygous individuals (Fig. <a reveal-id="DSV020F3" data-open="DSV020F3" class="link link-reveal link-table xref-fig">3</a>B). This suggests the broader practical applicability of developed genome-wide InDel markers for multiple large-scale rapid genotyping applications, including their suitability in construction of high-density genetic linkage maps and molecular mapping of high-resolution QTLs. By performing linkage analysis using the 6,219 marker genotyping data, we mapped 6,197 InDel markers across eight LGs of an inter-specific chickpea genetic map (Table <a reveal-id="DSV020TB1" data-open="DSV020TB1" class="link link-reveal link-table xref-fig">1</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>A). Highest number of markers were mapped on CaLG04 (1,123 markers), followed by CaLG03 (1,011 markers) and lowest on CaLG07 (457 markers). The generated eight LGs-based genetic maps spanned a total map length of 1,311.9 cM, with a mean inter-marker distance of 0.212 cM (Table <a reveal-id="DSV020TB1" data-open="DSV020TB1" class="link link-reveal link-table xref-fig">1</a>). CaLG03 and CaLG07 had longest and shortest map length spanning 199.4 and 135.3 cM, respectively. Most saturated genetic map was constructed in case of CaLG04 (mean inter-marker distance: 0.162 cM), followed by CaLG06 (0.175 cM) and least saturated map was generated for CaLG01 (0.253 cM) (Table <a reveal-id="DSV020TB1" data-open="DSV020TB1" class="link link-reveal link-table xref-fig">1</a>). An InDel marker-based inter-specific genetic linkage map (ICC 4958 × ICC 17160) generated in our study had greater map density (inter-marker distance: 0.212 cM) and thus highly saturated when compared with that reported so far in multiple intra- and inter-specific mapping population-derived genetic maps (0.50–8.01 cM) of chickpea.<sup><a reveal-id="DSV020C5" data-open="DSV020C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV020C7" data-open="DSV020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV020C8" data-open="DSV020C8" class="link link-ref link-reveal xref-bibr">8</a>–<a reveal-id="DSV020C10" data-open="DSV020C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV020C12" data-open="DSV020C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV020C27" data-open="DSV020C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSV020C35" data-open="DSV020C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                        <a id="4944052" scrollto-destination="4944052"></a>
<div content-id="DSV020TB1" class="table-modal"><div class="table-wrap"><div id="DSV020TB1" class="table-wrap-title" data-id="DSV020TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of an inter-specific genetic linkage map constructed using a 190 <em>F</em><sub>5</sub> chickpea mapping population (ICC 4958 × ICC 17160)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage groups (LGs) </th><th>InDel markers mapped </th><th>Map length covered (cM) </th><th>Mean inter-marker distance (cM) </th></tr></thead><tbody><tr><td>CaLG01 </td><td>651 </td><td>164.8 </td><td>0.253 </td></tr><tr><td>CaLG02 </td><td>697 </td><td>171.9 </td><td>0.247 </td></tr><tr><td>CaLG03 </td><td>1,011 </td><td>199.4 </td><td>0.197 </td></tr><tr><td>CaLG04 </td><td>1,123 </td><td>181.9 </td><td>0.162 </td></tr><tr><td>CaLG05 </td><td>808 </td><td>167.1 </td><td>0.207 </td></tr><tr><td>CaLG06 </td><td>805 </td><td>140.9 </td><td>0.175 </td></tr><tr><td>CaLG07 </td><td>457 </td><td>135.3 </td><td>0.296 </td></tr><tr><td>CaLG08 </td><td>645 </td><td>150.6 </td><td>0.233 </td></tr><tr><td>Total </td><td>6,197 </td><td>1,311.9 </td><td>0.212 </td></tr></tbody></table></div><div content-id="DSV020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage groups (LGs) </th><th>InDel markers mapped </th><th>Map length covered (cM) </th><th>Mean inter-marker distance (cM) </th></tr></thead><tbody><tr><td>CaLG01 </td><td>651 </td><td>164.8 </td><td>0.253 </td></tr><tr><td>CaLG02 </td><td>697 </td><td>171.9 </td><td>0.247 </td></tr><tr><td>CaLG03 </td><td>1,011 </td><td>199.4 </td><td>0.197 </td></tr><tr><td>CaLG04 </td><td>1,123 </td><td>181.9 </td><td>0.162 </td></tr><tr><td>CaLG05 </td><td>808 </td><td>167.1 </td><td>0.207 </td></tr><tr><td>CaLG06 </td><td>805 </td><td>140.9 </td><td>0.175 </td></tr><tr><td>CaLG07 </td><td>457 </td><td>135.3 </td><td>0.296 </td></tr><tr><td>CaLG08 </td><td>645 </td><td>150.6 </td><td>0.233 </td></tr><tr><td>Total </td><td>6,197 </td><td>1,311.9 </td><td>0.212 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944052" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV020TB1" class="table-wrap-title" data-id="DSV020TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Characteristics of an inter-specific genetic linkage map constructed using a 190 <em>F</em><sub>5</sub> chickpea mapping population (ICC 4958 × ICC 17160)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage groups (LGs) </th><th>InDel markers mapped </th><th>Map length covered (cM) </th><th>Mean inter-marker distance (cM) </th></tr></thead><tbody><tr><td>CaLG01 </td><td>651 </td><td>164.8 </td><td>0.253 </td></tr><tr><td>CaLG02 </td><td>697 </td><td>171.9 </td><td>0.247 </td></tr><tr><td>CaLG03 </td><td>1,011 </td><td>199.4 </td><td>0.197 </td></tr><tr><td>CaLG04 </td><td>1,123 </td><td>181.9 </td><td>0.162 </td></tr><tr><td>CaLG05 </td><td>808 </td><td>167.1 </td><td>0.207 </td></tr><tr><td>CaLG06 </td><td>805 </td><td>140.9 </td><td>0.175 </td></tr><tr><td>CaLG07 </td><td>457 </td><td>135.3 </td><td>0.296 </td></tr><tr><td>CaLG08 </td><td>645 </td><td>150.6 </td><td>0.233 </td></tr><tr><td>Total </td><td>6,197 </td><td>1,311.9 </td><td>0.212 </td></tr></tbody></table></div><div content-id="DSV020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage groups (LGs) </th><th>InDel markers mapped </th><th>Map length covered (cM) </th><th>Mean inter-marker distance (cM) </th></tr></thead><tbody><tr><td>CaLG01 </td><td>651 </td><td>164.8 </td><td>0.253 </td></tr><tr><td>CaLG02 </td><td>697 </td><td>171.9 </td><td>0.247 </td></tr><tr><td>CaLG03 </td><td>1,011 </td><td>199.4 </td><td>0.197 </td></tr><tr><td>CaLG04 </td><td>1,123 </td><td>181.9 </td><td>0.162 </td></tr><tr><td>CaLG05 </td><td>808 </td><td>167.1 </td><td>0.207 </td></tr><tr><td>CaLG06 </td><td>805 </td><td>140.9 </td><td>0.175 </td></tr><tr><td>CaLG07 </td><td>457 </td><td>135.3 </td><td>0.296 </td></tr><tr><td>CaLG08 </td><td>645 </td><td>150.6 </td><td>0.233 </td></tr><tr><td>Total </td><td>6,197 </td><td>1,311.9 </td><td>0.212 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944052" target="_blank">View Large</a></div></div>                        <a id="4944053" scrollto-destination="4944053"></a>
<div id="DSV020F4" data-id="DSV020F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02004.png?Expires=2147483647&Signature=vDA0t0DgBAjzDh1j0mhJ3xNnRcwpUeflWfYzx7dtLcOXl7zcCICZjw5zctqnOecarYnc93fLOAx2NlYrUGq3uleLf8B6yaRs~dz26dlddRNZNv1LsfmXUFDXLjWoBB0T~QQV~LxjtbCNDvU6W3mTRPYZ-8kpqZviHttSi55e3fVTocycXYpyixWm~y9iZg5jGpNYEoxcs58DdPgtvj8FFvtjUx5oVPGXAWyivn5tAtJ~wyWio1RdCmQC6T6KMcEpg3FYGHYTDifepm1Z6ayCypokxaeBzv9WhoWLKsfc1cQRk8hkarYiEPOTZPwhbDjTvbxj2YvECM-jPoZ6uhxdZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) A high-resolution inter-specific genetic map (ICC 4958 × ICC 17160) constructed by integrating 6,197 InDel markers on eight LGs of chickpea, are depicted in a Circos circular ideogram. The outermost circles represent the different genetic map length (cM) (spanning 20 cM uniform genetic distance intervals between bins) of eight LGs coded with multiple colors. (B) The integration of genetic and physical maps delineated five InDel markers-containing candidate genes at three major genomic regions harboring six robust DF and DM QTLs mapped on three desi chromosomes 1, 2 and 3. The InDel markers-carrying genes showing strong linkage with DF and DM QTLs selected as potential candidate for flowering and maturity time regulation in chickpea are highlighted with red color. The genetic (cM)/physical (bp) distance and identities of the InDel markers mapped on the LGs/chromosomes are indicated on the right and left side of the LGs/chromosomes, respectively. The detail information regarding InDel markers and QTLs are mentioned in Supplementary Table S3 and Table 2. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944053/dsv02004.tiff" path-from-xml="dsv02004.tiff" target="_blank">View large</a><a section="4944053" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02004.png?Expires=2147483647&Signature=q-IXYIhROFM2gawZOqbJYmgg~pymSq9vSNFmu9UGGG4sm9tXxMT7ODjMCqh7PuIqU~~oga4NarU6zaFSMMSPJMwZR6T9vVpQ9YoIYiuHucV3HRQ6fLI-AIYm6iM6l9~QTLX4mzjvrL6aG0JTXbCKeoTVOqCHQ6FGoX6O0K9yTHsyvcKZcOhQ3Ppo~XkSl1ju3yO5PtwfdTKEAvLabuQWQV74F5wJ4n3ITq51xjaM92pU6IKHWlsr1tBWio0G8teUYKbsOo0VhRFdacW0DXW7VVf1WUNpb2nw-ofmkrzhmAue6W7z-Tt0z5ZYvkeCLTzUvbm9G8I2H6YCVOGmUfYsLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944053&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) A high-resolution inter-specific genetic map (ICC 4958 × ICC 17160) constructed by integrating 6,197 InDel markers on eight LGs of chickpea, are depicted in a Circos circular ideogram. The outermost circles represent the different genetic map length (cM) (spanning 20 cM uniform genetic distance intervals between bins) of eight LGs coded with multiple colors. (B) The integration of genetic and physical maps delineated five InDel markers-containing candidate genes at three major genomic regions harboring six robust DF and DM QTLs mapped on three <em>desi</em> chromosomes 1, 2 and 3. The InDel markers-carrying genes showing strong linkage with DF and DM QTLs selected as potential candidate for flowering and maturity time regulation in chickpea are highlighted with red color. The genetic (cM)/physical (bp) distance and identities of the InDel markers mapped on the LGs/chromosomes are indicated on the right and left side of the LGs/chromosomes, respectively. The detail information regarding InDel markers and QTLs are mentioned in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> and Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV020F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/m_dsv02004.png?Expires=2147483647&Signature=vDA0t0DgBAjzDh1j0mhJ3xNnRcwpUeflWfYzx7dtLcOXl7zcCICZjw5zctqnOecarYnc93fLOAx2NlYrUGq3uleLf8B6yaRs~dz26dlddRNZNv1LsfmXUFDXLjWoBB0T~QQV~LxjtbCNDvU6W3mTRPYZ-8kpqZviHttSi55e3fVTocycXYpyixWm~y9iZg5jGpNYEoxcs58DdPgtvj8FFvtjUx5oVPGXAWyivn5tAtJ~wyWio1RdCmQC6T6KMcEpg3FYGHYTDifepm1Z6ayCypokxaeBzv9WhoWLKsfc1cQRk8hkarYiEPOTZPwhbDjTvbxj2YvECM-jPoZ6uhxdZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) A high-resolution inter-specific genetic map (ICC 4958 × ICC 17160) constructed by integrating 6,197 InDel markers on eight LGs of chickpea, are depicted in a Circos circular ideogram. The outermost circles represent the different genetic map length (cM) (spanning 20 cM uniform genetic distance intervals between bins) of eight LGs coded with multiple colors. (B) The integration of genetic and physical maps delineated five InDel markers-containing candidate genes at three major genomic regions harboring six robust DF and DM QTLs mapped on three desi chromosomes 1, 2 and 3. The InDel markers-carrying genes showing strong linkage with DF and DM QTLs selected as potential candidate for flowering and maturity time regulation in chickpea are highlighted with red color. The genetic (cM)/physical (bp) distance and identities of the InDel markers mapped on the LGs/chromosomes are indicated on the right and left side of the LGs/chromosomes, respectively. The detail information regarding InDel markers and QTLs are mentioned in Supplementary Table S3 and Table 2. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944053/dsv02004.tiff" path-from-xml="dsv02004.tiff" target="_blank">View large</a><a section="4944053" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv02004.png?Expires=2147483647&Signature=q-IXYIhROFM2gawZOqbJYmgg~pymSq9vSNFmu9UGGG4sm9tXxMT7ODjMCqh7PuIqU~~oga4NarU6zaFSMMSPJMwZR6T9vVpQ9YoIYiuHucV3HRQ6fLI-AIYm6iM6l9~QTLX4mzjvrL6aG0JTXbCKeoTVOqCHQ6FGoX6O0K9yTHsyvcKZcOhQ3Ppo~XkSl1ju3yO5PtwfdTKEAvLabuQWQV74F5wJ4n3ITq51xjaM92pU6IKHWlsr1tBWio0G8teUYKbsOo0VhRFdacW0DXW7VVf1WUNpb2nw-ofmkrzhmAue6W7z-Tt0z5ZYvkeCLTzUvbm9G8I2H6YCVOGmUfYsLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944053&ar=348317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) A high-resolution inter-specific genetic map (ICC 4958 × ICC 17160) constructed by integrating 6,197 InDel markers on eight LGs of chickpea, are depicted in a Circos circular ideogram. The outermost circles represent the different genetic map length (cM) (spanning 20 cM uniform genetic distance intervals between bins) of eight LGs coded with multiple colors. (B) The integration of genetic and physical maps delineated five InDel markers-containing candidate genes at three major genomic regions harboring six robust DF and DM QTLs mapped on three <em>desi</em> chromosomes 1, 2 and 3. The InDel markers-carrying genes showing strong linkage with DF and DM QTLs selected as potential candidate for flowering and maturity time regulation in chickpea are highlighted with red color. The genetic (cM)/physical (bp) distance and identities of the InDel markers mapped on the LGs/chromosomes are indicated on the right and left side of the LGs/chromosomes, respectively. The detail information regarding InDel markers and QTLs are mentioned in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> and Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=4944054 id="4944054" class="section-title" data-legacy-id=s3e>3.5. Molecular mapping of QTLs governing flowering and maturity time in chickpea</h3>
<p>A significant phenotypic difference of DF (25.7–111.0 with 81% H<sup>2</sup>) and DM (86.4–151.3 with 85% H<sup>2</sup>) traits in 190 mapping individuals (ICC 4958 × ICC 17160) and two parental accessions across 2 years was observed based on ANOVA (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). ANOVA outcomes inferred existence of a highly significant difference (<em>P</em> &lt; 0.001) of DF and DM traits among the mapping individuals although significant environmental (years) effects observed on these two traits in both seasons/years. A bi-directional transgressive segregation, including normal frequency distribution of DF and DM traits in mapping individuals and parental accessions across 2 years, was obtained (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5</a>A and B). These results collectively indicate that the complex quantitative genetic inheritance pattern of DF and DM traits in our developed inter-specific mapping population (ICC 4958 × ICC 17160) are regulated by multiple genes and thus have significance to be utilized for QTL mapping.</p><p>The 2 years multi-location field phenotyping data of DF and DM traits and genotyping information of 6,197 InDel markers genetically mapped on eight LGs were integrated for molecular mapping of QTLs. This analysis identified three major genomic regions harboring three of each significant (LOD: 8.1–11.5) DF (<em>CaqDF1.1</em>, <em>CaqDF2.1</em> and <em>CaqDF3.1</em>) and DM (<em>CaqDM1.1</em>, <em>CaqDM2.1</em> and <em>CaqDM3.1</em>) QTLs that were mapped across three LGs (CaLG01, CaLG02 and CaLG03) (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). The individual major QTL explained 17.5–27.5% and 16.4–26.9% phenotypic variations (<em>R</em><sup>2</sup>) for DF and DM traits, respectively. The PVE measured for all three of each robust DF and DM QTLs in combination was 68 and 65%, respectively. The identified six DF and DM QTLs being validated and exhibited consistent PVE (&gt;10% <em>R</em><sup>2</sup>) at higher LOD (8.1–11.5) across two seasons (years) were considered as robust QTLs. Three major genomic regions (1.036, 1.555 and 1.292 cM marker genetic intervals on CaLG01, CaLG02 and CaLG03, respectively) underlying these six robust DF and DM QTLs spanned with 45 InDel markers, were mapped on three different genomic regions with similar marker intervals across three LGs (Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). The molecular mapping and clustering of multiple DF and DM QTLs especially on a single major genomic region of three LGs infers complex genetic inheritance pattern of these quantitative traits in chickpea. All these six DF and DM QTLs exhibited positive additive gene effects (3.9–6.4) for early flowering and maturity time with major allelic contributions from ICC 4958 (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>). This could be due to strong positive correlation (Pearson's correlation coefficient, <em>r</em>: 91%, <em>P</em> &lt; 0.0001) between DF and DM traits observed in an inter-specific mapping population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S5</a>C). The five InDel markers tightly linked to six major DF and DM robust QTLs identified by both single marker- and CIM-based QTL analysis (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>) could have potential to be deployed in marker-assisted genetic enhancement for developing early flowering and maturing chickpea cultivars. </p>                        <a id="4944057" scrollto-destination="4944057"></a>
<div content-id="DSV020TB2" class="table-modal"><div class="table-wrap"><div id="DSV020TB2" class="table-wrap-title" data-id="DSV020TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Molecular mapping of significant QTLs governing DF and DM traits on chickpea chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">QTLs </th><th rowspan="2">LGs/chromosomes </th><th rowspan="2">Marker intervals with genetic positions (cM) </th><th rowspan="2">QTL physical intervals (kb) </th><th rowspan="2">Markers tightly linked with QTLs </th><th colspan="3">2012<hr /> </th><th colspan="3">2013<hr /> </th></tr><tr><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th></tr></thead><tbody><tr><td><em>CaqDF1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.8 </td><td>21.8 </td><td>5.8 </td><td>10.7 </td><td>23.6 </td><td>6.1 </td></tr><tr><td><em>CaqDF2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.5 </td><td>17.5 </td><td>3.9 </td><td>9.7 </td><td>20.4 </td><td>4.9 </td></tr><tr><td><em>CaqDF3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>10.3 </td><td>24.7 </td><td>4.6 </td><td>11.5 </td><td>27.5 </td><td>5.6 </td></tr><tr><td><em>CaqDM1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.5 </td><td>20.5 </td><td>4.7 </td><td>10.1 </td><td>22.7 </td><td>6.4 </td></tr><tr><td><em>CaqDM2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.1 </td><td>16.4 </td><td>4.1 </td><td>9.4 </td><td>19.6 </td><td>4.5 </td></tr><tr><td><em>CaqDM3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>9.8 </td><td>23.5 </td><td>4.2 </td><td>11.2 </td><td>26.9 </td><td>5.1 </td></tr></tbody></table></div><div content-id="DSV020TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">QTLs </th><th rowspan="2">LGs/chromosomes </th><th rowspan="2">Marker intervals with genetic positions (cM) </th><th rowspan="2">QTL physical intervals (kb) </th><th rowspan="2">Markers tightly linked with QTLs </th><th colspan="3">2012<hr /> </th><th colspan="3">2013<hr /> </th></tr><tr><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th></tr></thead><tbody><tr><td><em>CaqDF1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.8 </td><td>21.8 </td><td>5.8 </td><td>10.7 </td><td>23.6 </td><td>6.1 </td></tr><tr><td><em>CaqDF2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.5 </td><td>17.5 </td><td>3.9 </td><td>9.7 </td><td>20.4 </td><td>4.9 </td></tr><tr><td><em>CaqDF3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>10.3 </td><td>24.7 </td><td>4.6 </td><td>11.5 </td><td>27.5 </td><td>5.6 </td></tr><tr><td><em>CaqDM1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.5 </td><td>20.5 </td><td>4.7 </td><td>10.1 </td><td>22.7 </td><td>6.4 </td></tr><tr><td><em>CaqDM2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.1 </td><td>16.4 </td><td>4.1 </td><td>9.4 </td><td>19.6 </td><td>4.5 </td></tr><tr><td><em>CaqDM3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>9.8 </td><td>23.5 </td><td>4.2 </td><td>11.2 </td><td>26.9 </td><td>5.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>CaqDF1.1</em> (<em>Cicer arietinum</em> QTL for days to 50% flowering time on chromosome 1 number 1) and <em>CaqDM1.1</em> (<em>Cicer arietinum</em> QTL for days to maturity time on chromosome 1 number 1). PVE: percentage of phenotypic variation explained by QTLs. A: Additive effect; positive additive effect infers alleles from ICC 4958 with early flowering and maturity trait values. Details regarding CID_C and CID_S markers are mentioned in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. TF: transcription factor.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944057" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV020TB2" class="table-wrap-title" data-id="DSV020TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Molecular mapping of significant QTLs governing DF and DM traits on chickpea chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">QTLs </th><th rowspan="2">LGs/chromosomes </th><th rowspan="2">Marker intervals with genetic positions (cM) </th><th rowspan="2">QTL physical intervals (kb) </th><th rowspan="2">Markers tightly linked with QTLs </th><th colspan="3">2012<hr /> </th><th colspan="3">2013<hr /> </th></tr><tr><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th></tr></thead><tbody><tr><td><em>CaqDF1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.8 </td><td>21.8 </td><td>5.8 </td><td>10.7 </td><td>23.6 </td><td>6.1 </td></tr><tr><td><em>CaqDF2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.5 </td><td>17.5 </td><td>3.9 </td><td>9.7 </td><td>20.4 </td><td>4.9 </td></tr><tr><td><em>CaqDF3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>10.3 </td><td>24.7 </td><td>4.6 </td><td>11.5 </td><td>27.5 </td><td>5.6 </td></tr><tr><td><em>CaqDM1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.5 </td><td>20.5 </td><td>4.7 </td><td>10.1 </td><td>22.7 </td><td>6.4 </td></tr><tr><td><em>CaqDM2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.1 </td><td>16.4 </td><td>4.1 </td><td>9.4 </td><td>19.6 </td><td>4.5 </td></tr><tr><td><em>CaqDM3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>9.8 </td><td>23.5 </td><td>4.2 </td><td>11.2 </td><td>26.9 </td><td>5.1 </td></tr></tbody></table></div><div content-id="DSV020TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">QTLs </th><th rowspan="2">LGs/chromosomes </th><th rowspan="2">Marker intervals with genetic positions (cM) </th><th rowspan="2">QTL physical intervals (kb) </th><th rowspan="2">Markers tightly linked with QTLs </th><th colspan="3">2012<hr /> </th><th colspan="3">2013<hr /> </th></tr><tr><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th><th>LOD </th><th>PVE (<em>R</em><sup>2</sup>%) </th><th>A </th></tr></thead><tbody><tr><td><em>CaqDF1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.8 </td><td>21.8 </td><td>5.8 </td><td>10.7 </td><td>23.6 </td><td>6.1 </td></tr><tr><td><em>CaqDF2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.5 </td><td>17.5 </td><td>3.9 </td><td>9.7 </td><td>20.4 </td><td>4.9 </td></tr><tr><td><em>CaqDF3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>10.3 </td><td>24.7 </td><td>4.6 </td><td>11.5 </td><td>27.5 </td><td>5.6 </td></tr><tr><td><em>CaqDM1.1</em> </td><td>CaLG(Chr01) </td><td>1.036 cM [CID_C_403402 (5.997) to CID_C_492721 (7.033)] </td><td>89.3 </td><td>CID_C_477164 (TCP TF) </td><td>9.5 </td><td>20.5 </td><td>4.7 </td><td>10.1 </td><td>22.7 </td><td>6.4 </td></tr><tr><td><em>CaqDM2.1</em> </td><td>CaLG(Chr02) </td><td>1.555 [CID_C_4546528 (46.061) to CID_C_4703718 (47.616)] </td><td>157.2 </td><td>CID_C_4642382 (MADS TF)<br />CID_S_5036 (DOF Zinc Finger TF) </td><td>8.1 </td><td>16.4 </td><td>4.1 </td><td>9.4 </td><td>19.6 </td><td>4.5 </td></tr><tr><td><em>CaqDM3.1</em> </td><td>CaLG(Chr03) </td><td>1.292 [CID_C_4424175 (38.314) to CID_C_4575860 (39.606)] </td><td>151.7 </td><td>CID_C_4431329 (Phytochrome F-box)<br />CID_S_93427 (AGAMOUS) </td><td>9.8 </td><td>23.5 </td><td>4.2 </td><td>11.2 </td><td>26.9 </td><td>5.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>CaqDF1.1</em> (<em>Cicer arietinum</em> QTL for days to 50% flowering time on chromosome 1 number 1) and <em>CaqDM1.1</em> (<em>Cicer arietinum</em> QTL for days to maturity time on chromosome 1 number 1). PVE: percentage of phenotypic variation explained by QTLs. A: Additive effect; positive additive effect infers alleles from ICC 4958 with early flowering and maturity trait values. Details regarding CID_C and CID_S markers are mentioned in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. TF: transcription factor.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944057" target="_blank">View Large</a></div></div><p>To ascertain the novelty of our identified QTLs, the major genomic regions harboring six robust DF and DM QTLs were compared with that documented in earlier QTL mapping studies utilizing diverse inter- and intra-specific chickpea mapping populations.<sup><a reveal-id="DSV020C4" data-open="DSV020C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV020C6" data-open="DSV020C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV020C13" data-open="DSV020C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSV020C36  DSV020C40" data-open="DSV020C36  DSV020C40" class="link link-ref link-reveal xref-bibr">36–40</a></sup> We could not find any correspondence of these identified DF and DM QTLs with known QTLs reported previously based on their congruent genetic or physical positions on chickpea LGs/chromosomes. This indicates that DF and DM trait-associated QTLs identified by us are novel and possibly exhibit population-specific genomic distribution.</p>                            <h3 scrollto-destination=4944059 id="4944059" class="section-title" data-legacy-id=s3f>3.6. Delineation of candidate gene-based InDel markers regulating flowering and maturity time in chickpea</h3>
<p>The integration of genetic linkage map information of InDel markers spanning six robust DF and DM QTLs with that of physical maps of <em>desi</em> chickpea genome demarcated these three major QTL intervals into 89.3 (chromosome 1)–157.2 (chromosome 2) kb physical genomic regions on chromosomes 1, 2 and 3 (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). The structural and functional annotation of these target genomic physical intervals underlying DF and DM QTLs on three <em>desi</em> chromosomes identified multiple candidate protein-coding <em>desi</em> chickpea genes<em>.</em> Interestingly, five InDel markers-containing genes among these, at three target major QTL intervals exhibited tight linkage with six DF and DM-regulating QTLs based on our high-resolution QTL mapping (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). One InDel marker (CID_C_477164) in the CDS of a <em>TCP</em> [Teosinte branched 1 (<em>tb1</em>) Cycloidea, Proliferating cell factor (<em>PCF</em>)] transcription factor (TF) gene localized at <em>CaqDF1.1</em> and <em>CaqDM1.1</em> QTL region was associated strongly (<em>R</em><sup>2</sup>: 25.8–28.6% at 9.8–10.7 LOD) with DF and DM traits (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). Another two InDel markers (CID_C_4642382 and CID_S_5036) in the intron and CDS of <em>MADS</em> [<em>MCM</em> (Minichromosome maintenance) <em>AGAMOUS DEFICIENS SRF</em> (human serum response factor)] and <em>DOF</em> (DNA-binding One zinc Finger)-TF genes, respectively, annotated at <em>CaqDF2.1</em> and <em>CaqDM2.1</em> QTL region revealed strong association (<em>R</em><sup>2</sup>: 17.5–20.4% at 8.5–9.7 LOD) with DF and DM traits. Additionally, two InDel markers (CID_C_4431329 and CID_S_93427) in the DRR and introns of Phytochrome F-box and <em>AGAMOUS</em> genes, respectively annotated at <em>CaqDF3.1</em> and <em>CaqDM3.1</em> QTL interval had strong association (<em>R</em><sup>2</sup>: 28.7–30.2% at 10.3–11.5 LOD) with DF and DM traits (Table <a reveal-id="DSV020TB2" data-open="DSV020TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSV020F4" data-open="DSV020F4" class="link link-reveal link-table xref-fig">4</a>B). These InDel markers-carrying genes are known to be involved in transcriptional regulation of diverse known flowering time pathways in crop plants, including legumes.<sup><a reveal-id="DSV020C41" data-open="DSV020C41" class="link link-ref link-reveal xref-bibr">41</a>–<a reveal-id="DSV020C44" data-open="DSV020C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The <em>MADS</em>, <em>TCP</em> and <em>DOF</em> transcription factors, and Phytochrome F-box and <em>AGAMOUS</em> genes have significant role in regulating the ABC model of flower development by defining the genetic mechanism underlying the floral organ identity and floral architecture as well as through controlling signal perception during floral transitions from vegetative to reproductive phase specifically in <em>Arabidopsis</em> and other crop plants.<sup><a reveal-id="DSV020C45" data-open="DSV020C45" class="link link-ref link-reveal xref-bibr">45</a>–<a reveal-id="DSV020C48" data-open="DSV020C48" class="link link-ref link-reveal xref-bibr">48</a></sup> In this context, the InDel markers-associated QTLs and five potential candidate genes regulating flowering and maturity time delineated in the present study, thus have functional relevance to understand the genetic basis of complex flowering and maturity time traits in chickpea.</p>                            <h2 scrollto-destination=4944061 id="4944061" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a></p>                            <h2 scrollto-destination=4944063 id="4944063" class="section-title" data-legacy-id=s5>Funding</h2>
<p>The authors gratefully acknowledge the financial support for this study provided by a research grant (102/IFD/SAN/2161/2013-14) from the Department of Biotechnology (DBT), Government of India. Funding to pay the Open Access publication charges for this article was provided by the National Institute of Plant Genome Research (NIPGR).</p>                            <h2 scrollto-destination=4944065 id="4944065" class="backacknowledgements-title" >Acknowledgements</h2>
<p>S.D. acknowledges the DBT for Junior Research Fellowship award. The authors thank the Editor and reviewer for critically evaluating the manuscript and providing constructive comments.</p>                            <h2 scrollto-destination=4944067 id="4944067" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV020C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Choudhary</div>  <div class="given-names">A.K.</div></div>, <div class="name string-name"><div class="surname">Solanki</div>  <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Pratap</div>  <div class="given-names">A.</div></div></div> <div class="year">2011</div>, <div class="article-title">Towards marker-assisted selection in pulses: a review</div>, <div class="source">Plant Breed.</div> , <div class="volume">130</div>, <div class="fpage">297</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20marker-assisted%20selection%20in%20pulses%3A%20a%20review&amp;author=J.%20Kumar&amp;author=A.K.%20Choudhary&amp;author=R.K.%20Solanki&amp;author=A.%20Pratap&amp;publication_year=2011&amp;journal=Plant%20Breed.&amp;volume=130&amp;pages=297-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2011.130.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2011.130.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2011.130.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSV020C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div>  <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Song</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Saxena</div>  <div class="given-names">R.K.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement</div>, <div class="source">Nat. Biotechnol.,</div>  <div class="volume">31</div>, <div class="fpage">240</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.K.%20Varshney&amp;author=C.%20Song&amp;author=R.K.%20Saxena&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.%2C&amp;volume=31&amp;pages=240-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div> </div></div></div></div></div><div content-id="DSV020C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div>  <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Mohan</div>  <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Gaur</div>  <div class="given-names">P.M.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Achievements and prospects of genomics-assisted breeding in three legume crops of the semi-arid tropics</div>, <div class="source">Biotechnol. Adv.</div> , <div class="volume">31</div>, <div class="fpage">1120</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Achievements%20and%20prospects%20of%20genomics-assisted%20breeding%20in%20three%20legume%20crops%20of%20the%20semi-arid%20tropics&amp;author=R.K.%20Varshney&amp;author=S.M.%20Mohan&amp;author=P.M.%20Gaur&amp;publication_year=2013&amp;journal=Biotechnol.%20Adv.&amp;volume=31&amp;pages=1120-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2013.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2013.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2013.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23313999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cobos</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Rubio</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fernández-Romero</div>  <div class="given-names">M.D.</div></div> et al.  </div>. <div class="year">2007</div>, <div class="article-title">Genetic analysis of seed size, yield and days to flowering in a chickpea recombinant inbred line population derived from a <em>kabuli</em> × <em>desi</em> cross</div>, <div class="source">Ann. Appl. Biol.</div> , <div class="volume">151</div>, <div class="fpage">33</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20seed%20size%2C%20yield%20and%20days%20to%20flowering%20in%20a%20chickpea%20recombinant%20inbred%20line%20population%20derived%20from%20a%20kabuli%20%C3%97%20desi%20cross&amp;author=M.J.%20Cobos&amp;author=J.%20Rubio&amp;author=M.D.%20Fern%C3%A1ndez-Romero&amp;publication_year=2007&amp;journal=Ann.%20Appl.%20Biol.&amp;volume=151&amp;pages=33-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/aab.2007.151.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Faab.2007.151.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Faab.2007.151.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSV020C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nayak</div>  <div class="given-names">S.N.</div></div>, <div class="name string-name"><div class="surname">Zhu</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Varghese</div>  <div class="given-names">N.</div></div> et al.  </div>. <div class="year">2010</div>, <div class="article-title">Integration of novel SSR and gene-based SNP marker loci in the chickpea genetic map and establishment of new anchor points with <em>Medicago truncatula</em> genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">120</div>, <div class="fpage">1415</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20novel%20SSR%20and%20gene-based%20SNP%20marker%20loci%20in%20the%20chickpea%20genetic%20map%20and%20establishment%20of%20new%20anchor%20points%20with%20Medicago%20truncatula%20genome&amp;author=S.N.%20Nayak&amp;author=H.%20Zhu&amp;author=N.%20Varghese&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=1415-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1265-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1265-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1265-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gowda</div>  <div class="given-names">S.J.M.</div></div>, <div class="name string-name"><div class="surname">Radhika</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mhase</div>  <div class="given-names">L.B.</div></div>, <div class="name string-name"><div class="surname">Jamadagni</div>  <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">Gupta</div>  <div class="given-names">V.S.</div></div>, <div class="name string-name"><div class="surname">Kadro</div>  <div class="given-names">N.Y.</div></div></div> <div class="year">2011</div>, <div class="article-title">Mapping of QTLs governing agronomic and field traits in chickpea</div>, <div class="source">J. Appl. Genet.</div> , <div class="volume">52</div>, <div class="fpage">9</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20QTLs%20governing%20agronomic%20and%20field%20traits%20in%20chickpea&amp;author=S.J.M.%20Gowda&amp;author=P.%20Radhika&amp;author=L.B.%20Mhase&amp;author=B.M.%20Jamadagni&amp;author=V.S.%20Gupta&amp;author=N.Y.%20Kadro&amp;publication_year=2011&amp;journal=J.%20Appl.%20Genet.&amp;volume=52&amp;pages=9-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13353-010-0016-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13353-010-0016-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13353-010-0016-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21181334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gujaria</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dauthal</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Development and use of genic molecular markers (GMMs) for construction of a transcript map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">122</div>, <div class="fpage">1577</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20use%20of%20genic%20molecular%20markers%20(GMMs)%20for%20construction%20of%20a%20transcript%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=N.%20Gujaria&amp;author=A.%20Kumar&amp;author=P.%20Dauthal&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1577-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1556-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1556-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1556-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21384113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hiremath</div>  <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Farmer</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cannon</div>  <div class="given-names">S.B.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Large-scale transcriptome analysis in chickpea (<em>Cicer arietinum</em> L.), an orphan legume crop of the semi-arid tropics of Asia and Africa</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">9</div>, <div class="fpage">922</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptome%20analysis%20in%20chickpea%20(Cicer%20arietinum%20L.)%2C%20an%20orphan%20legume%20crop%20of%20the%20semi-arid%20tropics%20of%20Asia%20and%20Africa&amp;author=P.J.%20Hiremath&amp;author=A.%20Farmer&amp;author=S.B.%20Cannon&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=922-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2011.00625.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2011.00625.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2011.00625.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21615673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gaur</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Aza</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jeena</div>  <div class="given-names">G.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">High-throughput SNP discovery and genotyping for constructing a saturated linkage map of chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">357</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20genotyping%20for%20constructing%20a%20saturated%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.%20Gaur&amp;author=S.%20Aza&amp;author=G.%20Jeena&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=357-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kujur</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bajaj</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Saxena</div>  <div class="given-names">M.S.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Functionally relevant microsatellite markers from chickpea transcription factor genes for efﬁcient genotyping applications and trait association mapping</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">355</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functionally%20relevant%20microsatellite%20markers%20from%20chickpea%20transcription%20factor%20genes%20for%20ef%EF%AC%81cient%20genotyping%20applications%20and%20trait%20association%20mapping&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=M.S.%20Saxena&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=355-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23633531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kujur</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bajaj</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Upadhyaya</div>  <div class="given-names">H.D.</div></div> et al.  </div>. <div class="year">2015</div>, <div class="article-title">Employing genome-wide SNP discovery and genotyping strategy to extrapolate the natural allelic diversity and domestication patterns in chickpea</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">6</div>, <div class="fpage">162</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Employing%20genome-wide%20SNP%20discovery%20and%20genotyping%20strategy%20to%20extrapolate%20the%20natural%20allelic%20diversity%20and%20domestication%20patterns%20in%20chickpea&amp;author=A.%20Kujur&amp;author=D.%20Bajaj&amp;author=H.D.%20Upadhyaya&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci.&amp;volume=6&amp;pages=162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saxena</div>  <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Bajaj</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Das</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">An integrated genomic approach for rapid delineation of candidate genes regulating agro-morphological traits in chickpea</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">695</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genomic%20approach%20for%20rapid%20delineation%20of%20candidate%20genes%20regulating%20agro-morphological%20traits%20in%20chickpea&amp;author=M.S.%20Saxena&amp;author=D.%20Bajaj&amp;author=S.%20Das&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=695-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25335477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div>  <div class="given-names">R.K.</div></div>, <div class="name string-name"><div class="surname">Thudi</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nayak</div>  <div class="given-names">S.N.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genetic dissection of drought tolerance in chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">127</div>, <div class="fpage">445</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20dissection%20of%20drought%20tolerance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=R.K.%20Varshney&amp;author=M.%20Thudi&amp;author=S.N.%20Nayak&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=127&amp;pages=445-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2230-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2230-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2230-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24326458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jain</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Misra</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Patel</div>  <div class="given-names">R.K.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">A draft genome sequence of the pulse crop chickpea (<em>Cicer arietinum</em> L.)</div>, <div class="source">Plant J.</div> , <div class="volume">74</div>, <div class="fpage">715</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20genome%20sequence%20of%20the%20pulse%20crop%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=M.%20Jain&amp;author=G.%20Misra&amp;author=R.K.%20Patel&amp;publication_year=2013&amp;journal=Plant%20J.&amp;volume=74&amp;pages=715-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.74.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.74.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.74.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23489434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shen</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yang</div>  <div class="given-names">W.</div></div></div> <div class="year">2014</div>, <div class="article-title"><em>In silico</em> identification and experimental validation of insertion-deletion polymorphisms in tomato genome</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">429</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20identification%20and%20experimental%20validation%20of%20insertion-deletion%20polymorphisms%20in%20tomato%20genome&amp;author=J.%20Yang&amp;author=Y.%20Wang&amp;author=H.%20Shen&amp;author=W.%20Yang&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=429-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24618211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Cui</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">R.</div></div> et al.  </div>. <div class="year">2015</div>, <div class="article-title">Development of genome-wide insertion/deletion markers in rice based on graphic pipeline platform</div>, <div class="source">J. Integr. Plant. Biol.</div> , <div class="pub-id-doi"><a href="https://doi.org/10.1111/jipb.12354" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/jipb.12354</a></div>.</div></div></div></div><div content-id="DSV020C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moghaddam</div>  <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Song</div>  <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Mamidi</div>  <div class="given-names">S.</div></div></div> <div class="year">2014</div>, <div class="article-title">Developing market class specific InDel markers from next generation sequence data in <em>Phaseolus vulgaris</em> L.</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">5</div>, <div class="fpage">185</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developing%20market%20class%20specific%20InDel%20markers%20from%20next%20generation%20sequence%20data%20in%20Phaseolus%20vulgaris%20L.&amp;author=M.S.%20Moghaddam&amp;author=Q.%20Song&amp;author=S.%20Mamidi&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci.&amp;volume=5&amp;pages=185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24860578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pearson</div>  <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Edamura</div>  <div class="given-names">K.N.</div></div>, <div class="name string-name"><div class="surname">Cleary</div>  <div class="given-names">J.D.</div></div></div> <div class="year">2005</div>, <div class="article-title">Repeat instability: mechanisms of dynamic mutations</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">6</div>, <div class="fpage">729</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeat%20instability%3A%20mechanisms%20of%20dynamic%20mutations&amp;author=C.E.%20Pearson&amp;author=K.N.%20Edamura&amp;author=J.D.%20Cleary&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=729-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16205713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia-Diaz</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kunkel</div>  <div class="given-names">T.A.</div></div></div> <div class="year">2006</div>, <div class="article-title">Mechanism of a genetic glissando: structural biology of InDel mutations</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="volume">31</div>, <div class="fpage">206</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20a%20genetic%20glissando%3A%20structural%20biology%20of%20InDel%20mutations&amp;author=M.%20Garcia-Diaz&amp;author=T.A.%20Kunkel&amp;publication_year=2006&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=31&amp;pages=206-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibs.2006.02.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibs.2006.02.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibs.2006.02.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16545956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jain</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Moharana</div>  <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Shankar</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kumari</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Garg</div>  <div class="given-names">R.</div></div></div> <div class="year">2014</div>, <div class="article-title">Genome-wide discovery of DNA polymorphisms in rice cultivars with contrasting drought and salinity stress response and their functional relevance</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">12</div>, <div class="fpage">253</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20discovery%20of%20DNA%20polymorphisms%20in%20rice%20cultivars%20with%20contrasting%20drought%20and%20salinity%20stress%20response%20and%20their%20functional%20relevance&amp;author=M.%20Jain&amp;author=K.C.%20Moharana&amp;author=R.%20Shankar&amp;author=R.%20Kumari&amp;author=R.%20Garg&amp;publication_year=2014&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=12&amp;pages=253-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24460890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div>  <div class="given-names">Y.H.</div></div>, <div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Reif</div>  <div class="given-names">J.C.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Development of insertion and deletion markers based on biparental resequencing for fine mapping seed weight in soybean</div>, <div class="source">Plant Genome</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20insertion%20and%20deletion%20markers%20based%20on%20biparental%20resequencing%20for%20fine%20mapping%20seed%20weight%20in%20soybean&amp;author=Y.H.%20Li&amp;author=B.%20Liu&amp;author=J.C.%20Reif&amp;publication_year=2014&amp;journal=Plant%20Genome&amp;volume=7&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2014.04.0014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2014.04.0014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2014.04.0014"> </span></div> </div></div></div></div></div><div content-id="DSV020C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lu</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Exploration of presence/absence variation and corresponding polymorphic markers in soybean genome</div>, <div class="source">J. Integr. Plant. Biol.</div> , <div class="volume">56</div>, <div class="fpage">1009</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploration%20of%20presence%2Fabsence%20variation%20and%20corresponding%20polymorphic%20markers%20in%20soybean%20genome&amp;author=Y.%20Wang&amp;author=J.%20Lu&amp;author=S.%20Chen&amp;publication_year=2014&amp;journal=J.%20Integr.%20Plant.%20Biol.&amp;volume=56&amp;pages=1009-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.v56.10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.v56.10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.v56.10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24751174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shen</div>  <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Mocoeur</div>  <div class="given-names">A.R.J.</div></div>, <div class="name string-name"><div class="surname">Xia</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Jing</div>  <div class="given-names">H.C.</div></div></div> <div class="year">2015</div>, <div class="article-title">PAV markers in <em>Sorghum bicolour</em>: genome pattern, affected genes and pathways, and genetic linkage map construction</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">128</div>, <div class="fpage">623</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAV%20markers%20in%20Sorghum%20bicolour%3A%20genome%20pattern%2C%20affected%20genes%20and%20pathways%2C%20and%20genetic%20linkage%20map%20construction&amp;author=X.%20Shen&amp;author=Z.%20Liu&amp;author=A.R.J.%20Mocoeur&amp;author=Y.%20Xia&amp;author=H.C.%20Jing&amp;publication_year=2015&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=128&amp;pages=623-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-015-2458-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-015-2458-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-015-2458-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25634103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Misra</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Priya</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bandhiwal</div>  <div class="given-names">N.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">The chickpea genomic web resource: visualization and analysis of the <em>desi</em>-type <em>Cicer arietinum</em> nuclear genome for comparative exploration of legumes</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">14</div>, <div class="fpage">315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chickpea%20genomic%20web%20resource%3A%20visualization%20and%20analysis%20of%20the%20desi-type%20Cicer%20arietinum%20nuclear%20genome%20for%20comparative%20exploration%20of%20legumes&amp;author=G.%20Misra&amp;author=P.%20Priya&amp;author=N.%20Bandhiwal&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol.&amp;volume=14&amp;pages=315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-014-0315-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-014-0315-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-014-0315-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25518738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Muse</div>  <div class="given-names">S.V.</div></div></div> <div class="year">2005</div>, <div class="article-title">PowerMarker: an integrated analysis environment for genetic marker analysis</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">2128</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PowerMarker%3A%20an%20integrated%20analysis%20environment%20for%20genetic%20marker%20analysis&amp;author=K.%20Liu&amp;author=S.V.%20Muse&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2128-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15705655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lander</div>  <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Green</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Abrahamson</div>  <div class="given-names">J.</div></div> et al.  </div>. <div class="year">1987</div>, <div class="article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations</div>, <div class="source">Genomics</div> , <div class="volume">1</div>, <div class="fpage">174</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAPMAKER%3A%20an%20interactive%20computer%20package%20for%20constructing%20primary%20genetic%20linkage%20maps%20of%20experimental%20and%20natural%20populations&amp;author=E.S.%20Lander&amp;author=P.%20Green&amp;author=J.%20Abrahamson&amp;publication_year=1987&amp;journal=Genomics&amp;volume=1&amp;pages=174-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(87)90010-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(87)90010-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(87)90010-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3692487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bajaj</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Upadhyaya</div>  <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Khan</div>  <div class="given-names">Y.</div></div> et al.  </div>. <div class="year">2015</div>, <div class="article-title">A combinatorial approach of comprehensive QTL-based comparative genome mapping and transcript profiling identified a seed weight-regulating candidate gene in chickpea</div>, <div class="source">Sci. Rep.</div> , <div class="volume">5</div>, <div class="fpage">9264</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20combinatorial%20approach%20of%20comprehensive%20QTL-based%20comparative%20genome%20mapping%20and%20transcript%20profiling%20identified%20a%20seed%20weight-regulating%20candidate%20gene%20in%20chickpea&amp;author=D.%20Bajaj&amp;author=H.D.%20Upadhyaya&amp;author=Y.%20Khan&amp;publication_year=2015&amp;journal=Sci.%20Rep.&amp;volume=5&amp;pages=9264" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep09264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep09264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep09264"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25786576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ooijen</div>  <div class="given-names">J.W.</div></div></div> <div class="year">2009</div>, <div class="article-title">MapQTL 6: Software for the mapping of quantitative trait loci in experimental populations of diploid species</div>, <div class="publisher-name">B.V. Kyazma</div>: <div class="publisher-loc">Wageningen, Netherlands</div>.</div></div></div></div><div content-id="DSV020C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abbo</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Berger</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Turner</div>  <div class="given-names">N.C.</div></div></div> <div class="year">2003</div>, <div class="article-title">Evolution of cultivated chickpea: four bottlenecks limit diversity and constrain adaptation</div>, <div class="source">Funct. Plant. Biol.</div> , <div class="volume">30</div>, <div class="fpage">1081</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20cultivated%20chickpea%3A%20four%20bottlenecks%20limit%20diversity%20and%20constrain%20adaptation&amp;author=S.%20Abbo&amp;author=J.%20Berger&amp;author=N.C.%20Turner&amp;publication_year=2003&amp;journal=Funct.%20Plant.%20Biol.&amp;volume=30&amp;pages=1081-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/FP03084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FFP03084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FFP03084"> </span></div> </div></div></div></div></div><div content-id="DSV020C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burger</div>  <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Champan</div>  <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Burke</div>  <div class="given-names">J.M.</div></div></div> <div class="year">2008</div>, <div class="article-title">Molecular insights into the evolution of crop plants</div>, <div class="source">Am. J. Bot.</div> , <div class="volume">95</div>, <div class="fpage">113</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20insights%20into%20the%20evolution%20of%20crop%20plants&amp;author=J.C.%20Burger&amp;author=M.A.%20Champan&amp;author=J.M.%20Burke&amp;publication_year=2008&amp;journal=Am.%20J.%20Bot.&amp;volume=95&amp;pages=113-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.95.2.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.95.2.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.95.2.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21632337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div>  <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Luo</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">Z.Q.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide patterns of large-size presence/absence variants in sorghum</div>, <div class="source">J. Integr. Plant. Biol.</div> , <div class="volume">56</div>, <div class="fpage">24</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20patterns%20of%20large-size%20presence%2Fabsence%20variants%20in%20sorghum&amp;author=L.M.%20Zhang&amp;author=H.%20Luo&amp;author=Z.Q.%20Liu&amp;publication_year=2014&amp;journal=J.%20Integr.%20Plant.%20Biol.&amp;volume=56&amp;pages=24-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.v56.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.v56.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.v56.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24428208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hiremath</div>  <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Penmetsa</div>  <div class="given-names">R.V.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">10</div>, <div class="fpage">716</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20development%20of%20cost-effective%20SNP%20marker%20assays%20for%20diversity%20assessment%20and%20genetic%20mapping%20in%20chickpea%20and%20comparative%20mapping%20in%20legumes&amp;author=P.J.%20Hiremath&amp;author=A.%20Kumar&amp;author=R.V.%20Penmetsa&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=716-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00710.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00710.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00710.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Agarwal</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Jhanwar</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Priya</div>  <div class="given-names">P.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Comparative analysis of <em>kabuli</em> chickpea transcriptome with <em>desi</em> and wild chickpea provides a rich resource for development of functional markers</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e52443</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20kabuli%20chickpea%20transcriptome%20with%20desi%20and%20wild%20chickpea%20provides%20a%20rich%20resource%20for%20development%20of%20functional%20markers&amp;author=G.%20Agarwal&amp;author=S.%20Jhanwar&amp;author=P.%20Priya&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e52443" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0052443" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0052443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bharadwaj</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Srivastava</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chauhan</div>  <div class="given-names">S.K.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">Molecular diversity and phylogeny in geographical collection of chickpea (<em>Cicer</em> sp.) accessions</div>, <div class="source">J. Genet.</div> , <div class="volume">90</div>, <div class="fpage">e94</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20diversity%20and%20phylogeny%20in%20geographical%20collection%20of%20chickpea%20(Cicer%20sp.)%20accessions&amp;author=C.%20Bharadwaj&amp;author=R.%20Srivastava&amp;author=S.K.%20Chauhan&amp;publication_year=2011&amp;journal=J.%20Genet.&amp;volume=90&amp;pages=e94-100" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22232200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deokar</div>  <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Ramsay</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Sharpe</div>  <div class="given-names">A.G.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide SNP identification in chickpea for use in development of a high density genetic map and improvement of chickpea reference genome assembly</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">708</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20identification%20in%20chickpea%20for%20use%20in%20development%20of%20a%20high%20density%20genetic%20map%20and%20improvement%20of%20chickpea%20reference%20genome%20assembly&amp;author=A.A.%20Deokar&amp;author=L.%20Ramsay&amp;author=A.G.%20Sharpe&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25150411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cho</div>  <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shultz</div>  <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Anupama</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tefera</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Muehlbauer</div>  <div class="given-names">F.J.</div></div></div> <div class="year">2002</div>, <div class="article-title">Mapping genes for double podding and other morphological traits in chickpea</div>, <div class="source">Euphytica</div> , <div class="volume">128</div>, <div class="fpage">285</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20genes%20for%20double%20podding%20and%20other%20morphological%20traits%20in%20chickpea&amp;author=S.H.%20Cho&amp;author=J.%20Kumar&amp;author=J.L.%20Shultz&amp;author=K.%20Anupama&amp;author=F.%20Tefera&amp;author=F.J.%20Muehlbauer&amp;publication_year=2002&amp;journal=Euphytica&amp;volume=128&amp;pages=285-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1020872009306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1020872009306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1020872009306"> </span></div> </div></div></div></div></div><div content-id="DSV020C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anbessa</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Warkentin</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Vandenberg</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ball</div>  <div class="given-names">R.</div></div></div> <div class="year">2006</div>, <div class="article-title">Inheritance of time to flowering in chickpea in a short-season temperate environments</div>, <div class="source">J. Hered.</div> , <div class="volume">97</div>, <div class="fpage">55</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20time%20to%20flowering%20in%20chickpea%20in%20a%20short-season%20temperate%20environments&amp;author=Y.%20Anbessa&amp;author=T.%20Warkentin&amp;author=A.%20Vandenberg&amp;author=R.%20Ball&amp;publication_year=2006&amp;journal=J.%20Hered.&amp;volume=97&amp;pages=55-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esj009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesj009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesj009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16394253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cobos</div>  <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Winter</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kharrat</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2009</div>, <div class="article-title">Genetic analysis of agronomic traits in a wide cross of chickpea</div>, <div class="source">Field Crops Res.</div> , <div class="volume">111</div>, <div class="fpage">130</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20agronomic%20traits%20in%20a%20wide%20cross%20of%20chickpea&amp;author=M.J.%20Cobos&amp;author=P.%20Winter&amp;author=M.%20Kharrat&amp;publication_year=2009&amp;journal=Field%20Crops%20Res.&amp;volume=111&amp;pages=130-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fcr.2008.11.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fcr.2008.11.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fcr.2008.11.006"> </span></div> </div></div></div></div></div><div content-id="DSV020C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aryamanesh</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Nelson</div>  <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Yan</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Clarke</div>  <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Siddique</div>  <div class="given-names">K.H.M.</div></div></div> <div class="year">2010</div>, <div class="article-title">Mapping a major gene for growth habit and QTLs for <em>Ascochyta</em> blight resistance and flowering time in a population between chickpea and <em>Cicer reticulatum</em></div>, <div class="source">Euphytica</div> , <div class="volume">173</div>, <div class="fpage">307</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20a%20major%20gene%20for%20growth%20habit%20and%20QTLs%20for%20Ascochyta%20blight%20resistance%20and%20flowering%20time%20in%20a%20population%20between%20chickpea%20and%20Cicer%20reticulatum&amp;author=N.%20Aryamanesh&amp;author=M.N.%20Nelson&amp;author=G.%20Yan&amp;author=H.J.%20Clarke&amp;author=K.H.M.%20Siddique&amp;publication_year=2010&amp;journal=Euphytica&amp;volume=173&amp;pages=307-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-009-0086-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-009-0086-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-009-0086-2"> </span></div> </div></div></div></div></div><div content-id="DSV020C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jamalabadi</div>  <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Saidi</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Karami</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kharkesh</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Talebi</div>  <div class="given-names">R.</div></div></div> <div class="year">2013</div>, <div class="article-title">Molecular mapping and characterization of genes governing time to flowering, seed weight, and plant height in an intraspecific genetic linkage map of chickpea (<em>Cicer arietinum</em>)</div>, <div class="source">Biochem. Genet.</div> , <div class="volume">51</div>, <div class="fpage">387</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20and%20characterization%20of%20genes%20governing%20time%20to%20flowering%2C%20seed%20weight%2C%20and%20plant%20height%20in%20an%20intraspecific%20genetic%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum)&amp;author=J.G.%20Jamalabadi&amp;author=A.%20Saidi&amp;author=E.%20Karami&amp;author=M.%20Kharkesh&amp;author=R.%20Talebi&amp;publication_year=2013&amp;journal=Biochem.%20Genet.&amp;volume=51&amp;pages=387-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10528-013-9571-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10528-013-9571-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10528-013-9571-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23371372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laurie</div>  <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Diwadkar</div>  <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jaudal</div>  <div class="given-names">M.</div></div> et al.  </div>. <div class="year">2011</div>, <div class="article-title">The <em>Medicago</em> <em>FLOWERING LOCUS T</em> homolog, <em>MtFTa1</em>, is a key regulator of flowering time</div>, <div class="source">Plant Physiol.</div> , <div class="volume">156</div>, <div class="fpage">2207</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20FLOWERING%20LOCUS%20T%20homolog%2C%20MtFTa1%2C%20is%20a%20key%20regulator%20of%20flowering%20time&amp;author=R.E.%20Laurie&amp;author=P.%20Diwadkar&amp;author=M.%20Jaudal&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=2207-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.180182" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.180182" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.180182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andres</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Coupland</div>  <div class="given-names">G.</div></div></div> <div class="year">2012</div>, <div class="article-title">The genetic basis of ﬂowering responses to seasonal cues</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">13</div>, <div class="fpage">627</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20basis%20of%20%EF%AC%82owering%20responses%20to%20seasonal%20cues&amp;author=F.%20Andres&amp;author=G.%20Coupland&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=627-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22898651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhai</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lu</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liang</div>  <div class="given-names">S.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title"><em>GmFT4</em>, a homolog of <em>FLOWERING LOCUS T</em>, is positively regulated by <em>E1</em> and functions as a flowering repressor in soybean</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e89030</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GmFT4%2C%20a%20homolog%20of%20FLOWERING%20LOCUS%20T%2C%20is%20positively%20regulated%20by%20E1%20and%20functions%20as%20a%20flowering%20repressor%20in%20soybean&amp;author=H.%20Zhai&amp;author=S.%20Lu&amp;author=S.%20Liang&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e89030" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0089030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0089030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0089030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24586488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weller</div>  <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Martínez</div>  <div class="given-names">R.O.</div></div></div> <div class="year">2015</div>, <div class="article-title">Genetic control of flowering time in legumes</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">6</div>, <div class="fpage">207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20flowering%20time%20in%20legumes&amp;author=J.L.%20Weller&amp;author=R.O.%20Mart%C3%ADnez&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci.&amp;volume=6&amp;pages=207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25914700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smaczniak</div>  <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Immink</div>  <div class="given-names">R.G.</div></div>, <div class="name string-name"><div class="surname">Muiño</div>  <div class="given-names">J.M.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">Characterization of MADS-domain transcription factor complexes in <em>Arabidopsis</em> flower development</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">1560</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20MADS-domain%20transcription%20factor%20complexes%20in%20Arabidopsis%20flower%20development&amp;author=C.%20Smaczniak&amp;author=R.G.%20Immink&amp;author=J.M.%20Mui%C3%B1o&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=1560-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1112871109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1112871109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1112871109"> </span></div> </div></div></div></div></div><div content-id="DSV020C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noguero</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Atif</div>  <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ochatt</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Thompson</div>  <div class="given-names">R.D.</div></div></div> <div class="year">2013</div>, <div class="article-title">The role of the DNA-binding One Zinc Finger (<em>DOF</em>) transcription factor family in plants</div>, <div class="source">Plant Sci.</div> , <div class="volume">209</div>, <div class="fpage">32</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20the%20DNA-binding%20One%20Zinc%20Finger%20(DOF)%20transcription%20factor%20family%20in%20plants&amp;author=M.%20Noguero&amp;author=R.M.%20Atif&amp;author=S.%20Ochatt&amp;author=R.D.%20Thompson&amp;publication_year=2013&amp;journal=Plant%20Sci.&amp;volume=209&amp;pages=32-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2013.03.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2013.03.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2013.03.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23759101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dreni</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kater</div>  <div class="given-names">M.M.</div></div></div> <div class="year">2014</div>, <div class="article-title">MADS reloaded: evolution of the <em>AGAMOUS</em> subfamily genes</div>, <div class="source">New Phytol.</div> , <div class="volume">201</div>, <div class="fpage">717</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MADS%20reloaded%3A%20evolution%20of%20the%20AGAMOUS%20subfamily%20genes&amp;author=L.%20Dreni&amp;author=M.M.%20Kater&amp;publication_year=2014&amp;journal=New%20Phytol.&amp;volume=201&amp;pages=717-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24164649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV020C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV020C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lopez</div>  <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Sun</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Blair</div>  <div class="given-names">P.B.</div></div>, <div class="name string-name"><div class="surname">Mukhtar</div>  <div class="given-names">M.S.</div></div></div> <div class="year">2015</div>, <div class="article-title"><em>TCP</em> three-way handshake: linking developmental processes with plant immunity</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">20</div>, <div class="fpage">238</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TCP%20three-way%20handshake%3A%20linking%20developmental%20processes%20with%20plant%20immunity&amp;author=J.A.%20Lopez&amp;author=Y.%20Sun&amp;author=P.B.%20Blair&amp;author=M.S.%20Mukhtar&amp;publication_year=2015&amp;journal=Trends%20Plant%20Sci.&amp;volume=20&amp;pages=238-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2015.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2015.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2015.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25655280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348317&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_table4.pdf?Expires=2147483647&amp;Signature=uD-2iXj09LlYCC~VPb5eaowUZcmJh0DTX92BcqOPBH6hMc0a-yZ-Ba-s9W4i3p6Z5yoVaHVS9jGGXniQf3~lILuIsEKUsZQXd~DwEDX1ipHF3H9a0UENkXeVvUoDts~TlR6wbekjvz23NB4tbp5fzO0ZllfrVU9mPEDukHhW3fRzrjtLARCBEN8uuSuKpyFtmQqDSJK-Jcm5Y91VRNqmwxToZ4dH47I29nPMgFMJq079CJ5t8uC3~dku9r1Lupm2aQAp3xs0uJLWHCsQrwOM-OnIUaehrbFDOjJ75MikNCCbfMesaQdZvk4iTgh2SfQl5UCHFuTpiyR~qKM2LofB8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_table3.xlsx?Expires=2147483647&amp;Signature=vD8kuZnwSFdbeLyrs-jeSnIE5ePtGBf49195xqmBOkoa2RS4mmKBArtLVWVUUuI~--z9z99pijiuhtfuu3Y7IvnSOWSYgaE09-QZkQmJ9p-Rs5Dd791Syf~561po7GO-wLrui8k2hug2eO9ISDJGqQ~6cXHunfqjw3LA453Yy8cN4iX5LM4KiNjpa5pFMYuMdk0fs4a6LZfA1j~gfT1I~QE0NYmDcZBEAEywvCwDs1uvIdPUewQ3i-vy0oo7dwJLdNiDXxNbull2IiM2WqCcwPiacaRfcY6L~h-6QHBPifWlRxtU9Zgdy6ZWrkjxwaMSg~AyYFS6UG35H3FVU4A5mQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_table2.pdf?Expires=2147483647&amp;Signature=h9hxGUOWEogJYgKkiKiiIecC84jYg3Bym~VzY5KfzkhHyv6MTUqM5K6zR59rRfdneiLlnaRr8rHFoKIMbeHbsvZgrImU-HvM7D7Y0WtKSqqoMBBUEZxQB6wL~KLr08zyDNIvAZ3MrWQ3rgvlj-bUXmJFUtBTdAG7YHQmohL3ezt3BDnWWssrvJXBuT3TI3sTYvVmG4EBl8FVZhiGI1u5ikn4MScrODf98VGgGf~d~NwvkJtkpcHn2kPl15jBKDO6rtkGIPoubVfzXBxDoUGKGE6pfZp~-9NhM2zwuChk4TPHqwnzNWoN5hdJZl7ro6EAHyaGT2GA3kuHJIQUFAsZiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_table1.pdf?Expires=2147483647&amp;Signature=XDPCVyzyQIzbyG7i56crh9kjYIoOI1zFFju1jGp~LUSWNR5ODWb9Ct6MhsSnUBNyaiSsk~IX5wSkuOmTeZUb9fy20w3T5DWtnWjjJszVI66sodg7sM~Se899GmmRwV03yD0bEn8e2YQ3-txeXjnhrViexDdYt6is2Hj4YxKPade9bmBICnVev4BkQ3iJOVr6~5LcgcTZd~9OPMOak-28gYWtkZyIVb2bIpmxV5HGhSk6iGtOElfLeudz~Cot2gd7Qg-kjtYjrnDgr4mPseTg2bpui5C0DxFknsR32~3AaMLuREaf1aZgVyc0EE3QbKqvOG836Cb7hZ49YfF81rdpSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_fig5.pdf?Expires=2147483647&amp;Signature=vFujd854ea96MvllAo3lbz0RZVsBlnJfJVacB0gJP5qCG9EhYYUX1D5yKs7KceYOW0kR9ztjSdI8qp2rS1p2bf-EodKpii8vqFO8I8enoXEKR8GrSYr529uEKc5fQQk82Xe4DBm42B5jjOJcrf0fS1BOY2tOXd-sZpGh1LCDd~e9w9XELE1qYeKIhZAKfH0zae3uag7QkFFkTTCm2wX9Ya4BI-iFUizXUvyAPCBjjlQ3kT2UU09f~AhjvSYSw0WGg6me2Ijm2FUILLYsaQdP3CpraQ6Yv~23~Ot96hkal0WL9pxyBkvqfksIRsB0JmmVhIeaKdw7J~xjFkTYK~dkWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_fig4.pdf?Expires=2147483647&amp;Signature=tRhYMtda29C36Cs-hbOsf4czxJXDvtnpA1ep7~WhjeoBWwnBU5UDCTpQ-XqTmnKGlICvCEXDYNjl2XkQEQGeLvoOvOgLtz08zOvVz5YHIife~skA2OXPiRr0Q3RjHdR5Wl2L71fjIaAjznRxFfTU7YNFwWnROzObau5NAchLr3ewZXVzi0Tu3S~D5sqJXd1F5hC12TrfmytPu8MRJs6KSp-OrS5gjkyAGkxctgq5Vo0NPZzKDe8a2C2bYboKZ1g2BPp-9wwYvsZTWpjCsl8FbVvyuAsnj-E8lwb8bHMLOEy07po7pchwMzjmTz~eC9X8Nnz6gFosyrmmW62Z~YXGaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_fig3.pdf?Expires=2147483647&amp;Signature=SjRLzaP4CtmbFJ9wFVwgDKFYJLnEyd8aMnXBeY4MFzVKvixH5a0yXHCU9qjukoZvsNNX7mnES5PhS~ZHapcwqFNF4-etXw3J0N~--h4fvupRGKrBUiwk9g2cGYG5xRiZ3~dvF49tJLoaWqE6Y6ochJfMoD5cAZvRAPe3IczioMGcN7CMH0-8RNKw7gVyJeFtweTeMGhR7jwXShTt4Z9A-J7sbYGb8maPQIq6TKKns5zQUqy316WxhPJhNVKfrZNuFWxrx69Is~5sU8yK02vAkhBlkikCPjvhi4ttkPYaZezHD9ap6SRAOzgPcG0MzIQcNjFnTdNVJ2nk7v-U6LGb6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_fig2.pdf?Expires=2147483647&amp;Signature=Yf3BM9GHTe4IxQZZOBMXW44D0iwgvk829QfmpaZcaV2RujSlNPqRj3Rb40VNhcdAdfJX58wPBUKn7VLPKMb-Kt7MPXgqt1i3YN4OfPVJ3ImjDqQxOKZFmCY0XqL0U~oL52yQ9uoB~tFSsi5qrgUwRWg8eBj4vN1Fcsu043hV3~Owjyvc5P9Rys1RDxft~6rrEX6vOuN75oHvMA3LkutzDY64amQ~b3sCs-KnzmWDpgIDqss7tdr25NgT9~1l538e7sV71CSOUgbGPme-jC5xNZMpoz06aeoaDcrZbHO24e27zd-FlQaqci~N0uWq5d4AyYp6XK2QI9UkEwVnFU70eQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/5/10.1093/dnares/dsv020/2/dsv020-dsv020supp_fig1.pdf?Expires=2147483647&amp;Signature=eyPocbTtcD9tkkoXLa3U~aFVo3bwipqKE5nXE2a~T2FaMJyer1XintxoNx7RKye-uKvCt1q7L97E8cbCavvoLP6Ie~vMqqR7QXT2Az-x49~kk6210SzcVRrl3K45xuCIhqC4tk23qLp7EgrYUEAZp6ZHhPJsQrp1JrB2YreadM3ag54Ff1S1tN7UX23sNKxCesfgYan2AMidj6WFUZ5Noo4PXjzPLpbyXYMY48GAjiyZNichnTygs~mxg1GKEmrlIzcBS-ESs0ldSH2URcwDmg~wpLEMKPC9pJX7~25BpEXg3nCXJbXLfz8HYRTWHVqiosB2Kf0slpJzvkPOJrx2EA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">678</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348317"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362820100007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f5%2f377%2f348317">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362820100007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F5%2F377%2F348317" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348317'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide insertion–deletion (InDel) marker discovery and genotyping for genomics-assisted breeding applications in chickpea" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f5%2f377%2f348317">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

