

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/2/147/332826","@type":"ScholarlyArticle","name":"Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing","datePublished":"2015-02-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2015-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu047","keywords":["<italic>Gossypium hirsutum</italic>, RAD sequencing","genetic map","QTL mapping","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/2/147/332826","author":[{"name":"Wang, Hantao","affiliation":"National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University ,  Wuhan 430070, Hubei ,  China","@type":"Person"},{"name":"Jin, Xin","affiliation":"National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University ,  Wuhan 430070, Hubei ,  China","@type":"Person"},{"name":"Zhang, Beibei","affiliation":"National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University ,  Wuhan 430070, Hubei ,  China","@type":"Person"},{"name":"Shen, Chao","affiliation":"National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University ,  Wuhan 430070, Hubei ,  China","@type":"Person"},{"name":"Lin, Zhongxu","affiliation":"National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University ,  Wuhan 430070, Hubei ,  China","@type":"Person"}],"description":"Abstract.  RAD sequencing was performed using DH962 and Jimian5 as upland cotton mapping parents. Sequencing data for DH962 and Jimian5 were assembled into the ","pageStart":"147","pageEnd":"160","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing" />
    <meta property="og:description" content="Abstract.  RAD sequencing was performed using DH962 and Jimian5 as upland cotton mapping parents. Sequencing data for DH962 and Jimian5 were assembled into the " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/2/147/332826" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-02-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wang, Hantao" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, Hubei, China" /><meta name="citation_author" content="Jin, Xin" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, Hubei, China" /><meta name="citation_author" content="Zhang, Beibei" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, Hubei, China" /><meta name="citation_author" content="Shen, Chao" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, Hubei, China" /><meta name="citation_author" content="Lin, Zhongxu" /><meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, Hubei, China" /><meta name="citation_title" content="Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing" /><meta name="citation_firstpage" content="147" /><meta name="citation_lastpage" content="160" /><meta name="citation_doi" content="10.1093/dnares/dsu047" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2015/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Toward sequencing cotton (Gossypium) genomes; citation_author=Chen  Z.; citation_author=Scheffler  B.E.; citation_author=Dennis  E.;  citation_journal_title=Plant Physiol.;  citation_year=2007;  citation_volume=145;  citation_pages=1303-10; " /><meta name="citation_reference" content="citation_title=Addition of 455 microsatellite marker loci to the high-density Gossypium hirsutum TM-1×G. barbadense 3–79 genetic map; citation_author=Fang  D.D.; citation_author=Yu  J.Z.;  citation_journal_title=Cotton Sci.;  citation_year=2012;  citation_volume=16;  citation_pages=229-48; " /><meta name="citation_reference" content="citation_title=A high-density integrative linkage map for Gossypium hirsutum; citation_author=Lin  Z.; citation_author=Zhang  Y.; citation_author=Zhang  X.; citation_author=Guo  X.;  citation_journal_title=Euphytica;  citation_year=2009;  citation_volume=166;  citation_pages=35-45; " /><meta name="citation_reference" content="citation_title=Genetic mapping and quantitative trait locus analysis of fiber quality traits using a three-parent composite population in upland cotton (Gossypium hirsutum L.); citation_author=Zhang  K.; citation_author=Zhang  J.; citation_author=Ma  J.;  citation_journal_title=Mol. Breeding;  citation_year=2012;  citation_volume=29;  citation_pages=335-48; " /><meta name="citation_reference" content="citation_title=Construction of genetic map and QTL analysis of fiber quality traits for Upland cotton (Gossypium hirsutum L.); citation_author=Tang  S.; citation_author=Teng  Z.; citation_author=Zhai  T.;  citation_journal_title=Euphytica;  citation_year=2015;  citation_volume=201;  citation_pages=195-213; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L; citation_author=Liu  R.; citation_author=Wang  B.; citation_author=Guo  W.;  citation_journal_title=Mol. Breeding;  citation_year=2012;  citation_volume=29;  citation_pages=297-311; " /><meta name="citation_reference" content="citation_title=Applications of single nucleotide polymorphisms in crop genetics; citation_author=Rafalski  A.;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2002;  citation_volume=5;  citation_pages=94-100; " /><meta name="citation_reference" content="citation_title=SNP marker diversity in common bean (Phaseolus vulgaris L.); citation_author=Cortés  A.J.; citation_author=Chavarro  M.C.; citation_author=Blair  M.W.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=827-45; " /><meta name="citation_reference" content="citation_title=Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms; citation_author=Yamamoto  T.; citation_author=Nagasaki  H.; citation_author=Yonemaru  J.;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=267" /><meta name="citation_reference" content="citation_title=Single-nucleotide polymorphism discovery by high-throughput sequencing in sorghum; citation_author=Nelson  J.C.; citation_author=Wang  S.; citation_author=Wu  Y.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=352" /><meta name="citation_reference" content="citation_title=Development and mapping of SNP assays in allotetraploid cotton; citation_author=Byers  R.L.; citation_author=Harker  D.B.; citation_author=Yourstone  S.M.; citation_author=Maughan  P.J.; citation_author=Udall  J.A.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1201-14; " /><meta name="citation_reference" content="citation_title=Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in Lupinus angustifolius L; citation_author=Yang  H.; citation_author=Tao  Y.; citation_author=Zheng  Z.; citation_author=Li  C.; citation_author=Sweetingham  M.W.; citation_author=Howieson  J.G.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=318" /><meta name="citation_reference" content="citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers; citation_author=Miller  M.R.; citation_author=Dunham  J.P.; citation_author=Amores  A.; citation_author=Cresko  W.A.; citation_author=Johnson  E.A.;  citation_journal_title=Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=240-8; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers; citation_author=Baird  N.A.; citation_author=Etter  P.D.; citation_author=Atwood  T.S.;  citation_journal_title=PloS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=High-throughput polymorphism detection and genotyping in Brassica napus using next-generation RAD sequencing; citation_author=Bus  A.; citation_author=Hecht  J.; citation_author=Huettel  B.; citation_author=Reinhardt  R.; citation_author=Stich  B.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=281" /><meta name="citation_reference" content="citation_title=Construction of a high-density genetic map for grape using next generation restriction-site associated DNA sequencing; citation_author=Wang  N.; citation_author=Fang  L.; citation_author=Xin  H.; citation_author=Wang  L.; citation_author=Li  S.;  citation_journal_title=BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=148" /><meta name="citation_reference" content="citation_title=De novo sequencing of sunflower genome for SNP discovery using RAD (Restriction site Associated DNA) approach; citation_author=Pegadaraju  V.; citation_author=Nipper  R.; citation_author=Hulke  B.; citation_author=Qi  L.; citation_author=Schultz  Q.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=556" /><meta name="citation_reference" content="citation_title=Genetic analysis of major QTLs and epistasis interaction for yield and fiber quality in upland cotton; citation_author=Lin  Z.; citation_author=Feng  C.; citation_author=Guo  X.; citation_author=Zhang  X.;  citation_journal_title=Sci. Agric. Sin.;  citation_year=2009;  citation_volume=42;  citation_pages=3036-47; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; citation_author=Zerbino  D.R.; citation_author=Birney  E.;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; citation_author=Huang  X.;  citation_journal_title=Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=AgBase: a functional genomics resource for agriculture; citation_author=McCarthy  F.; citation_author=Wang  N.; citation_author=Magee  G.B.;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=229" /><meta name="citation_reference" content="citation_title=The Gene Ontology (GO) database and informatics resource; citation_author=Harris  M.A.; citation_author=Clark  J.; citation_author=Ireland  A.;  citation_journal_title=Nucl. Acid Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D258-61; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools; citation_author=Li  H.; citation_author=Handsaker  B.; citation_author=Wysoker  A.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=The genome analysis toolkit: a mapreduce framework for analyzing next-generation DNA sequencing data; citation_author=McKenna  A.; citation_author=Hanna  M.; citation_author=Banks  E.;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=SNP detection for massively parallel whole-genome resequencing; citation_author=Li  R.; citation_author=Li  Y.; citation_author=Fang  X.;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1124-32; " /><meta name="citation_reference" content="citation_title=Development and evaluation of intron and insertion–deletion markers for Gossypium barbadense; citation_author=Wang  X.; citation_author=Ren  G.; citation_author=Li  X.; citation_author=Tu  J.; citation_author=Lin  Z.; citation_author=Zhang  X.;  citation_journal_title=Plant Mol. Biol. Rep.;  citation_year=2012;  citation_volume=30;  citation_pages=605-13; " /><meta name="citation_reference" content="citation_title=Construction of introgression lines, development of new markers, and construction of high-density genetic linkage map in cotton; citation_author=Li  X.; citation_publisher=Wuhan, Huazhong Agricultural University, ;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Genetic evaluation of EST-SSRs derived from Gossypium herbaceum; citation_author=Yu  Y.; citation_author=Wang  Z.; citation_author=Feng  C.; citation_author=Zhang  Y.; citation_author=Lin  Z.; citation_author=Zhang  X.;  citation_journal_title=Acta Agron. Sin.;  citation_year=2008;  citation_volume=34;  citation_pages=2085-94; " /><meta name="citation_reference" content="citation_title=Isolation, characterization, and mapping of genomic microsatellite markers for the first time in sea-island cotton (Gossypium barbadense); citation_author=Zhang  P.; citation_author=Wang  X.; citation_author=Yu  Y.; citation_author=Yu  Y.; citation_author=Lin  Z.; citation_author=Zhang  X.;  citation_journal_title=Acta Agron. Sin.;  citation_year=2009;  citation_volume=35;  citation_pages=1013-20; " /><meta name="citation_reference" content="citation_title=Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC1 population between gossypium hirsutum and G. barbadense; citation_author=Yu  Y.; citation_author=Yuan  D.; citation_author=Liang  S.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=15" /><meta name="citation_reference" content="citation_title=Unbiased genomic distribution of genes related to cell morphogenesis in cotton by chromosome mapping; citation_author=Liu  C.; citation_author=Lin  Z.; citation_author=Zhang  X.;  citation_journal_title=Plant Cell Tissue Organ Cult.;  citation_year=2011;  citation_volume=108;  citation_pages=529-34; " /><meta name="citation_reference" content="citation_title=A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in Gossypium; citation_author=Guo  W.; citation_author=Cai  C.; citation_author=Wang  C.;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=527-41; " /><meta name="citation_reference" content="citation_title=A preliminary analysis of genome structure and composition in Gossypium hirsutum; citation_author=Guo  W.; citation_author=Cai  C.; citation_author=Wang  C.; citation_author=Zhao  L.; citation_author=Wang  L.; citation_author=Zhang  T.;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=314" /><meta name="citation_reference" content="citation_title=Identification of 700 new microsatellite loci from cotton (G. hirsutum L.); citation_author=Hoffman  S.M.; citation_author=Kohel  R.J.; citation_author=Pepper  A.E.; citation_author=Xiao  J.; citation_author=Yu  J.Z.; citation_author=Grum  D.S.;  citation_journal_title=Cotton Sci.;  citation_year=2007;  citation_volume=11;  citation_pages=208-41; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map; citation_author=Stam  P.;  citation_journal_title=Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; citation_author=Voorrips  R.E.;  citation_journal_title=Heredity;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Report on QTL nomenclature; citation_author=McCouch  S.R.; citation_author=Cho  Y.G.; citation_author=Yano  M.; citation_author=Paul  E.; citation_author=Blinstrub  M.;  citation_journal_title=Rice Genet. Newsl.;  citation_year=1997;  citation_volume=14;  citation_pages=11-131; " /><meta name="citation_reference" content="citation_title=Identification of SNP and SSR markers in eggplant using RAD tag sequencing; citation_author=Barchi  L.; citation_author=Lanteri  S.; citation_author=Portis  E.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=304" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The genome of Theobroma cacao; citation_author=Argout  X.; citation_author=Salse  J.; citation_author=Aury  J.M.;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Comparison and development of EST–SSRs from two 454 sequencing libraries of Gossypium barbadense; citation_author=Wang  H.; citation_author=Li  X.; citation_author=Gao  W.; citation_author=Jin  X.; citation_author=Zhang  X.; citation_author=Lin  Z.;  citation_journal_title=Euphytica;  citation_year=2014;  citation_volume=198;  citation_pages=277-88; " /><meta name="citation_reference" content="citation_title=RAD tag sequencing as a source of SNP markers in Cynara cardunculus L; citation_author=Scaglione  D.; citation_author=Acquadro  A.; citation_author=Portis  E.; citation_author=Tirone  M.; citation_author=Knapp  S.J.; citation_author=Lanteri  S.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=3" /><meta name="citation_reference" content="citation_title=Local de novo assembly of RAD paired-end contigs using short sequencing reads; citation_author=Etter  P.D.; citation_author=Preston  J.L.; citation_author=Bassham  S.; citation_author=Cresko  W.A.; citation_author=Johnson  E.A.;  citation_journal_title=PloS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e18561" /><meta name="citation_reference" content="citation_title=Control of plant trichome development by a cotton fiber MYB gene; citation_author=Wang  S.; citation_author=Wang  J.; citation_author=Yu  N.;  citation_journal_title=Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=2323-34; " /><meta name="citation_reference" content="citation_title=The control of single-celled cotton fiber elongation by developmentally reversible gating of plasmodesmata and coordinated expression of sucrose and K+ transporters and expansin; citation_author=Ruan  Y.; citation_author=Llewellyn  D.J.; citation_author=Furbank  R.T.;  citation_journal_title=Plant Cell;  citation_year=2001;  citation_volume=13;  citation_pages=47-60; " /><meta name="citation_reference" content="citation_title=Comparative evolutionary and developmental dynamics of the cotton (Gossypium hirsutum) fiber transcriptome; citation_author=Yoo  M.J.; citation_author=Wendel  J.F.;  citation_journal_title=PLoS Genet.;  citation_year=2014;  citation_volume=10;  citation_pages=e1004073" /><meta name="citation_reference" content="citation_title=Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres; citation_author=Paterson  A.H.; citation_author=Wendel  J.F.; citation_author=Gundlach  H.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=492;  citation_pages=423-7; " /><meta name="citation_reference" content="citation_title=Genome sequence of the cultivated cotton Gossypium arboreum; citation_author=Li  F.; citation_author=Fan  G.; citation_author=Wang  K.;  citation_journal_title=Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=567-72; " /><meta name="citation_reference" content="citation_title=Development of molecular markers from Indian genotypes of two Gossypium L. species; citation_author=Srivastava  A.; citation_author=Jena  S.N.; citation_author=Ranjan  A.;  citation_journal_title=Plant Breed;  citation_year=2013;  citation_volume=132;  citation_pages=506-13; " /><meta name="citation_reference" content="citation_title=QTL mapping for fiber quality traits across multiple generations and environments in upland cotton; citation_author=Sun  F.; citation_author=Zhang  J.; citation_author=Wang  S.;  citation_journal_title=Mol. Breeding;  citation_year=2012;  citation_volume=30;  citation_pages=569-82; " /><meta name="citation_reference" content="citation_title=A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (Gossypium); citation_author=Rong  J.K.; citation_author=Abbey  C.; citation_author=Bowers  J.E.;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=389-417; " /><meta name="citation_reference" content="citation_title=Detection and genotyping of restriction fragment associated polymorphisms in polyploid crops with a pseudo-reference sequence: a case study in allotetraploid Brassica napus; citation_author=Chen  X.; citation_author=Li  X.; citation_author=Zhang  B.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=346" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/2/147/951618/dsu047.pdf" /><meta name="description" content="Abstract.  RAD sequencing was performed using DH962 and Jimian5 as upland cotton mapping parents. Sequencing data for DH962 and Jimian5 were assembled into the " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/2/147/332826" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/2/147/332826" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/2/147/332826");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu047';
    </script>
<title>
	Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./332826" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbugATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgSGFudGFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50LCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBIdWJlaSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaW4sIFhpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgSHViZWksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIEJlaWJlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgSHViZWksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hlbiwgQ2hhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEtleSBMYWJvcmF0b3J5IG9mIENyb3AgR2VuZXRpYyBJbXByb3ZlbWVudCwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgSHViZWksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGluLCBaaG9uZ3h1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgS2V5IExhYm9yYXRvcnkgb2YgQ3JvcCBHZW5ldGljIEltcHJvdmVtZW50LCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBIdWJlaSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkVucmljaG1lbnQgb2YgYW4gaW50cmFzcGVjaWZpYyBnZW5ldGljIG1hcCBvZiB1cGxhbmQgY290dG9uIGJ5IGRldmVsb3BpbmcgbWFya2VycyB1c2luZyBwYXJlbnRhbCBSQUQgc2VxdWVuY2luZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE0NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgc2VxdWVuY2luZyBjb3R0b24gKEdvc3N5cGl1bSkgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFouOyBjaXRhdGlvbl9hdXRob3I9U2NoZWZmbGVyICBCLkUuOyBjaXRhdGlvbl9hdXRob3I9RGVubmlzICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ1OyAgY2l0YXRpb25fcGFnZXM9MTMwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRkaXRpb24gb2YgNDU1IG1pY3Jvc2F0ZWxsaXRlIG1hcmtlciBsb2NpIHRvIHRoZSBoaWdoLWRlbnNpdHkgR29zc3lwaXVtIGhpcnN1dHVtIFRNLTHDl0cuIGJhcmJhZGVuc2UgM+KAkzc5IGdlbmV0aWMgbWFwOyBjaXRhdGlvbl9hdXRob3I9RmFuZyAgRC5ELjsgY2l0YXRpb25fYXV0aG9yPVl1ICBKLlouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db3R0b24gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIyOS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgaW50ZWdyYXRpdmUgbGlua2FnZSBtYXAgZm9yIEdvc3N5cGl1bSBoaXJzdXR1bTsgY2l0YXRpb25fYXV0aG9yPUxpbiAgWi47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWC47IGNpdGF0aW9uX2F1dGhvcj1HdW8gIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9MzUtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwcGluZyBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIGFuYWx5c2lzIG9mIGZpYmVyIHF1YWxpdHkgdHJhaXRzIHVzaW5nIGEgdGhyZWUtcGFyZW50IGNvbXBvc2l0ZSBwb3B1bGF0aW9uIGluIHVwbGFuZCBjb3R0b24gKEdvc3N5cGl1bSBoaXJzdXR1bSBMLik7IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgSy47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgSi47IGNpdGF0aW9uX2F1dGhvcj1NYSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zMzUtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBnZW5ldGljIG1hcCBhbmQgUVRMIGFuYWx5c2lzIG9mIGZpYmVyIHF1YWxpdHkgdHJhaXRzIGZvciBVcGxhbmQgY290dG9uIChHb3NzeXBpdW0gaGlyc3V0dW0gTC4pOyBjaXRhdGlvbl9hdXRob3I9VGFuZyAgUy47IGNpdGF0aW9uX2F1dGhvcj1UZW5nICBaLjsgY2l0YXRpb25fYXV0aG9yPVpoYWkgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjAxOyAgY2l0YXRpb25fcGFnZXM9MTk1LTIxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgbWFwcGluZyBmb3IgeWllbGQgYW5kIGl0cyBjb21wb25lbnRzIGJ5IHVzaW5nIHR3byBpbW1vcnRhbGl6ZWQgcG9wdWxhdGlvbnMgb2YgYSBoZXRlcm90aWMgaHlicmlkIGluIEdvc3N5cGl1bSBoaXJzdXR1bSBMOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBSLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEIuOyBjaXRhdGlvbl9hdXRob3I9R3VvICBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI5Ny0zMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9ucyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIGNyb3AgZ2VuZXRpY3M7IGNpdGF0aW9uX2F1dGhvcj1SYWZhbHNraSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05NC0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBtYXJrZXIgZGl2ZXJzaXR5IGluIGNvbW1vbiBiZWFuIChQaGFzZW9sdXMgdnVsZ2FyaXMgTC4pOyBjaXRhdGlvbl9hdXRob3I9Q29ydMOpcyAgQS5KLjsgY2l0YXRpb25fYXV0aG9yPUNoYXZhcnJvICBNLkMuOyBjaXRhdGlvbl9hdXRob3I9QmxhaXIgIE0uVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9ODI3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lIGRlZmluaXRpb24gb2YgdGhlIHBlZGlncmVlIGhhcGxvdHlwZXMgb2YgY2xvc2VseSByZWxhdGVkIHJpY2UgY3VsdGl2YXJzIGJ5IG1lYW5zIG9mIGdlbm9tZS13aWRlIGRpc2NvdmVyeSBvZiBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zOyBjaXRhdGlvbl9hdXRob3I9WWFtYW1vdG8gIFQuOyBjaXRhdGlvbl9hdXRob3I9TmFnYXNha2kgIEguOyBjaXRhdGlvbl9hdXRob3I9WW9uZW1hcnUgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGJ5IGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIGluIHNvcmdodW07IGNpdGF0aW9uX2F1dGhvcj1OZWxzb24gIEouQy47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBTLjsgY2l0YXRpb25fYXV0aG9yPVd1ICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIG1hcHBpbmcgb2YgU05QIGFzc2F5cyBpbiBhbGxvdGV0cmFwbG9pZCBjb3R0b247IGNpdGF0aW9uX2F1dGhvcj1CeWVycyAgUi5MLjsgY2l0YXRpb25fYXV0aG9yPUhhcmtlciAgRC5CLjsgY2l0YXRpb25fYXV0aG9yPVlvdXJzdG9uZSAgUy5NLjsgY2l0YXRpb25fYXV0aG9yPU1hdWdoYW4gIFAuSi47IGNpdGF0aW9uX2F1dGhvcj1VZGFsbCAgSi5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz0xMjAxLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbiBvZiBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBmb3IgcmFwaWQgbWFya2VyIGRldmVsb3BtZW50IGluIG1vbGVjdWxhciBwbGFudCBicmVlZGluZzogYSBjYXNlIHN0dWR5IG9uIGFudGhyYWNub3NlIGRpc2Vhc2UgcmVzaXN0YW5jZSBpbiBMdXBpbnVzIGFuZ3VzdGlmb2xpdXMgTDsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIEguOyBjaXRhdGlvbl9hdXRob3I9VGFvICBZLjsgY2l0YXRpb25fYXV0aG9yPVpoZW5nICBaLjsgY2l0YXRpb25fYXV0aG9yPUxpICBDLjsgY2l0YXRpb25fYXV0aG9yPVN3ZWV0aW5naGFtICBNLlcuOyBjaXRhdGlvbl9hdXRob3I9SG93aWVzb24gIEouRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTMxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGFuZCBjb3N0LWVmZmVjdGl2ZSBwb2x5bW9ycGhpc20gaWRlbnRpZmljYXRpb24gYW5kIGdlbm90eXBpbmcgdXNpbmcgcmVzdHJpY3Rpb24gc2l0ZSBhc3NvY2lhdGVkIEROQSAoUkFEKSBtYXJrZXJzOyBjaXRhdGlvbl9hdXRob3I9TWlsbGVyICBNLlIuOyBjaXRhdGlvbl9hdXRob3I9RHVuaGFtICBKLlAuOyBjaXRhdGlvbl9hdXRob3I9QW1vcmVzICBBLjsgY2l0YXRpb25fYXV0aG9yPUNyZXNrbyAgVy5BLjsgY2l0YXRpb25fYXV0aG9yPUpvaG5zb24gIEUuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjQwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7IGNpdGF0aW9uX2F1dGhvcj1CYWlyZCAgTi5BLjsgY2l0YXRpb25fYXV0aG9yPUV0dGVyICBQLkQuOyBjaXRhdGlvbl9hdXRob3I9QXR3b29kICBULlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTMzNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgcG9seW1vcnBoaXNtIGRldGVjdGlvbiBhbmQgZ2Vub3R5cGluZyBpbiBCcmFzc2ljYSBuYXB1cyB1c2luZyBuZXh0LWdlbmVyYXRpb24gUkFEIHNlcXVlbmNpbmc7IGNpdGF0aW9uX2F1dGhvcj1CdXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9SGVjaHQgIEouOyBjaXRhdGlvbl9hdXRob3I9SHVldHRlbCAgQi47IGNpdGF0aW9uX2F1dGhvcj1SZWluaGFyZHQgIFIuOyBjaXRhdGlvbl9hdXRob3I9U3RpY2ggIEIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXAgZm9yIGdyYXBlIHVzaW5nIG5leHQgZ2VuZXJhdGlvbiByZXN0cmljdGlvbi1zaXRlIGFzc29jaWF0ZWQgRE5BIHNlcXVlbmNpbmc7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBOLjsgY2l0YXRpb25fYXV0aG9yPUZhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9WGluICBILjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9TGkgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIHNlcXVlbmNpbmcgb2Ygc3VuZmxvd2VyIGdlbm9tZSBmb3IgU05QIGRpc2NvdmVyeSB1c2luZyBSQUQgKFJlc3RyaWN0aW9uIHNpdGUgQXNzb2NpYXRlZCBETkEpIGFwcHJvYWNoOyBjaXRhdGlvbl9hdXRob3I9UGVnYWRhcmFqdSAgVi47IGNpdGF0aW9uX2F1dGhvcj1OaXBwZXIgIFIuOyBjaXRhdGlvbl9hdXRob3I9SHVsa2UgIEIuOyBjaXRhdGlvbl9hdXRob3I9UWkgIEwuOyBjaXRhdGlvbl9hdXRob3I9U2NodWx0eiAgUS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTU1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgb2YgbWFqb3IgUVRMcyBhbmQgZXBpc3Rhc2lzIGludGVyYWN0aW9uIGZvciB5aWVsZCBhbmQgZmliZXIgcXVhbGl0eSBpbiB1cGxhbmQgY290dG9uOyBjaXRhdGlvbl9hdXRob3I9TGluICBaLjsgY2l0YXRpb25fYXV0aG9yPUZlbmcgIEMuOyBjaXRhdGlvbl9hdXRob3I9R3VvICBYLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBBZ3JpYy4gU2luLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTMwMzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlbHZldDogYWxnb3JpdGhtcyBmb3IgZGUgbm92byBzaG9ydCByZWFkIGFzc2VtYmx5IHVzaW5nIGRlIEJydWlqbiBncmFwaHM7IGNpdGF0aW9uX2F1dGhvcj1aZXJiaW5vICBELlIuOyBjaXRhdGlvbl9hdXRob3I9QmlybmV5ICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgY2l0YXRpb25fYXV0aG9yPUh1YW5nICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWdCYXNlOiBhIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzb3VyY2UgZm9yIGFncmljdWx0dXJlOyBjaXRhdGlvbl9hdXRob3I9TWNDYXJ0aHkgIEYuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgTi47IGNpdGF0aW9uX2F1dGhvcj1NYWdlZSAgRy5CLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2VuZSBPbnRvbG9neSAoR08pIGRhdGFiYXNlIGFuZCBpbmZvcm1hdGljcyByZXNvdXJjZTsgY2l0YXRpb25fYXV0aG9yPUhhcnJpcyAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPUNsYXJrICBKLjsgY2l0YXRpb25fYXV0aG9yPUlyZWxhbmQgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1EMjU4LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYWxpZ25tZW50L21hcCBmb3JtYXQgYW5kIFNBTXRvb2xzOyBjaXRhdGlvbl9hdXRob3I9TGkgIEguOyBjaXRhdGlvbl9hdXRob3I9SGFuZHNha2VyICBCLjsgY2l0YXRpb25fYXV0aG9yPVd5c29rZXIgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBhbmFseXNpcyB0b29sa2l0OiBhIG1hcHJlZHVjZSBmcmFtZXdvcmsgZm9yIGFuYWx5emluZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgY2l0YXRpb25fYXV0aG9yPU1jS2VubmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9SGFubmEgIE0uOyBjaXRhdGlvbl9hdXRob3I9QmFua3MgIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAgZGV0ZWN0aW9uIGZvciBtYXNzaXZlbHkgcGFyYWxsZWwgd2hvbGUtZ2Vub21lIHJlc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPUxpICBSLjsgY2l0YXRpb25fYXV0aG9yPUxpICBZLjsgY2l0YXRpb25fYXV0aG9yPUZhbmcgIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTExMjQtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBldmFsdWF0aW9uIG9mIGludHJvbiBhbmQgaW5zZXJ0aW9u4oCTZGVsZXRpb24gbWFya2VycyBmb3IgR29zc3lwaXVtIGJhcmJhZGVuc2U7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBYLjsgY2l0YXRpb25fYXV0aG9yPVJlbiAgRy47IGNpdGF0aW9uX2F1dGhvcj1MaSAgWC47IGNpdGF0aW9uX2F1dGhvcj1UdSAgSi47IGNpdGF0aW9uX2F1dGhvcj1MaW4gIFouOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02MDUtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBpbnRyb2dyZXNzaW9uIGxpbmVzLCBkZXZlbG9wbWVudCBvZiBuZXcgbWFya2VycywgYW5kIGNvbnN0cnVjdGlvbiBvZiBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBsaW5rYWdlIG1hcCBpbiBjb3R0b247IGNpdGF0aW9uX2F1dGhvcj1MaSAgWC47IGNpdGF0aW9uX3B1Ymxpc2hlcj1XdWhhbiwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBldmFsdWF0aW9uIG9mIEVTVC1TU1JzIGRlcml2ZWQgZnJvbSBHb3NzeXBpdW0gaGVyYmFjZXVtOyBjaXRhdGlvbl9hdXRob3I9WXUgIFkuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1GZW5nICBDLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPUxpbiAgWi47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEgQWdyb24uIFNpbi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0yMDg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24sIGNoYXJhY3Rlcml6YXRpb24sIGFuZCBtYXBwaW5nIG9mIGdlbm9taWMgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmb3IgdGhlIGZpcnN0IHRpbWUgaW4gc2VhLWlzbGFuZCBjb3R0b24gKEdvc3N5cGl1bSBiYXJiYWRlbnNlKTsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBQLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFguOyBjaXRhdGlvbl9hdXRob3I9WXUgIFkuOyBjaXRhdGlvbl9hdXRob3I9WXUgIFkuOyBjaXRhdGlvbl9hdXRob3I9TGluICBaLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBBZ3Jvbi4gU2luLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTMtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzdHJ1Y3R1cmUgb2YgY290dG9uIHJldmVhbGVkIGJ5IGEgZ2Vub21lLXdpZGUgU1NSIGdlbmV0aWMgbWFwIGNvbnN0cnVjdGVkIGZyb20gYSBCQzEgcG9wdWxhdGlvbiBiZXR3ZWVuIGdvc3N5cGl1bSBoaXJzdXR1bSBhbmQgRy4gYmFyYmFkZW5zZTsgY2l0YXRpb25fYXV0aG9yPVl1ICBZLjsgY2l0YXRpb25fYXV0aG9yPVl1YW4gIEQuOyBjaXRhdGlvbl9hdXRob3I9TGlhbmcgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuYmlhc2VkIGdlbm9taWMgZGlzdHJpYnV0aW9uIG9mIGdlbmVzIHJlbGF0ZWQgdG8gY2VsbCBtb3JwaG9nZW5lc2lzIGluIGNvdHRvbiBieSBjaHJvbW9zb21lIG1hcHBpbmc7IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEMuOyBjaXRhdGlvbl9hdXRob3I9TGluICBaLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBUaXNzdWUgT3JnYW4gQ3VsdC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9NTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pY3Jvc2F0ZWxsaXRlLWJhc2VkLCBnZW5lLXJpY2ggbGlua2FnZSBtYXAgcmV2ZWFscyBnZW5vbWUgc3RydWN0dXJlLCBmdW5jdGlvbiBhbmQgZXZvbHV0aW9uIGluIEdvc3N5cGl1bTsgY2l0YXRpb25fYXV0aG9yPUd1byAgVy47IGNpdGF0aW9uX2F1dGhvcj1DYWkgIEMuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTUyNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmVsaW1pbmFyeSBhbmFseXNpcyBvZiBnZW5vbWUgc3RydWN0dXJlIGFuZCBjb21wb3NpdGlvbiBpbiBHb3NzeXBpdW0gaGlyc3V0dW07IGNpdGF0aW9uX2F1dGhvcj1HdW8gIFcuOyBjaXRhdGlvbl9hdXRob3I9Q2FpICBDLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEMuOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgTC47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBMLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiA3MDAgbmV3IG1pY3Jvc2F0ZWxsaXRlIGxvY2kgZnJvbSBjb3R0b24gKEcuIGhpcnN1dHVtIEwuKTsgY2l0YXRpb25fYXV0aG9yPUhvZmZtYW4gIFMuTS47IGNpdGF0aW9uX2F1dGhvcj1Lb2hlbCAgUi5KLjsgY2l0YXRpb25fYXV0aG9yPVBlcHBlciAgQS5FLjsgY2l0YXRpb25fYXV0aG9yPVhpYW8gIEouOyBjaXRhdGlvbl9hdXRob3I9WXUgIEouWi47IGNpdGF0aW9uX2F1dGhvcj1HcnVtICBELlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db3R0b24gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIwOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGludGVncmF0ZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgYnkgbWVhbnMgb2YgYSBuZXcgY29tcHV0ZXIgcGFja2FnZTogSm9pbiBNYXA7IGNpdGF0aW9uX2F1dGhvcj1TdGFtICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTczOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwQ2hhcnQ6IHNvZnR3YXJlIGZvciB0aGUgZ3JhcGhpY2FsIHByZXNlbnRhdGlvbiBvZiBsaW5rYWdlIG1hcHMgYW5kIFFUTHM7IGNpdGF0aW9uX2F1dGhvcj1Wb29ycmlwcyAgUi5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBvcnQgb24gUVRMIG5vbWVuY2xhdHVyZTsgY2l0YXRpb25fYXV0aG9yPU1jQ291Y2ggIFMuUi47IGNpdGF0aW9uX2F1dGhvcj1DaG8gIFkuRy47IGNpdGF0aW9uX2F1dGhvcj1ZYW5vICBNLjsgY2l0YXRpb25fYXV0aG9yPVBhdWwgIEUuOyBjaXRhdGlvbl9hdXRob3I9QmxpbnN0cnViICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UmljZSBHZW5ldC4gTmV3c2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTEtMTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBTTlAgYW5kIFNTUiBtYXJrZXJzIGluIGVnZ3BsYW50IHVzaW5nIFJBRCB0YWcgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPUJhcmNoaSAgTC47IGNpdGF0aW9uX2F1dGhvcj1MYW50ZXJpICBTLjsgY2l0YXRpb25fYXV0aG9yPVBvcnRpcyAgRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZsb3dlcmluZyBwbGFudCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPUFyYWJpZG9wc2lzIEdlbm9tZSBJbml0aWF0aXZlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBUaGVvYnJvbWEgY2FjYW87IGNpdGF0aW9uX2F1dGhvcj1BcmdvdXQgIFguOyBjaXRhdGlvbl9hdXRob3I9U2Fsc2UgIEouOyBjaXRhdGlvbl9hdXRob3I9QXVyeSAgSi5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBhbmQgZGV2ZWxvcG1lbnQgb2YgRVNU4oCTU1NScyBmcm9tIHR3byA0NTQgc2VxdWVuY2luZyBsaWJyYXJpZXMgb2YgR29zc3lwaXVtIGJhcmJhZGVuc2U7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBILjsgY2l0YXRpb25fYXV0aG9yPUxpICBYLjsgY2l0YXRpb25fYXV0aG9yPUdhbyAgVy47IGNpdGF0aW9uX2F1dGhvcj1KaW4gIFguOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFguOyBjaXRhdGlvbl9hdXRob3I9TGluICBaLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE5ODsgIGNpdGF0aW9uX3BhZ2VzPTI3Ny04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFEIHRhZyBzZXF1ZW5jaW5nIGFzIGEgc291cmNlIG9mIFNOUCBtYXJrZXJzIGluIEN5bmFyYSBjYXJkdW5jdWx1cyBMOyBjaXRhdGlvbl9hdXRob3I9U2NhZ2xpb25lICBELjsgY2l0YXRpb25fYXV0aG9yPUFjcXVhZHJvICBBLjsgY2l0YXRpb25fYXV0aG9yPVBvcnRpcyAgRS47IGNpdGF0aW9uX2F1dGhvcj1UaXJvbmUgIE0uOyBjaXRhdGlvbl9hdXRob3I9S25hcHAgIFMuSi47IGNpdGF0aW9uX2F1dGhvcj1MYW50ZXJpICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsIGRlIG5vdm8gYXNzZW1ibHkgb2YgUkFEIHBhaXJlZC1lbmQgY29udGlncyB1c2luZyBzaG9ydCBzZXF1ZW5jaW5nIHJlYWRzOyBjaXRhdGlvbl9hdXRob3I9RXR0ZXIgIFAuRC47IGNpdGF0aW9uX2F1dGhvcj1QcmVzdG9uICBKLkwuOyBjaXRhdGlvbl9hdXRob3I9QmFzc2hhbSAgUy47IGNpdGF0aW9uX2F1dGhvcj1DcmVza28gIFcuQS47IGNpdGF0aW9uX2F1dGhvcj1Kb2huc29uICBFLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE4NTYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbCBvZiBwbGFudCB0cmljaG9tZSBkZXZlbG9wbWVudCBieSBhIGNvdHRvbiBmaWJlciBNWUIgZ2VuZTsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFMuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgSi47IGNpdGF0aW9uX2F1dGhvcj1ZdSAgTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yMzIzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29udHJvbCBvZiBzaW5nbGUtY2VsbGVkIGNvdHRvbiBmaWJlciBlbG9uZ2F0aW9uIGJ5IGRldmVsb3BtZW50YWxseSByZXZlcnNpYmxlIGdhdGluZyBvZiBwbGFzbW9kZXNtYXRhIGFuZCBjb29yZGluYXRlZCBleHByZXNzaW9uIG9mIHN1Y3Jvc2UgYW5kIEsrIHRyYW5zcG9ydGVycyBhbmQgZXhwYW5zaW47IGNpdGF0aW9uX2F1dGhvcj1SdWFuICBZLjsgY2l0YXRpb25fYXV0aG9yPUxsZXdlbGx5biAgRC5KLjsgY2l0YXRpb25fYXV0aG9yPUZ1cmJhbmsgIFIuVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz00Ny02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZXZvbHV0aW9uYXJ5IGFuZCBkZXZlbG9wbWVudGFsIGR5bmFtaWNzIG9mIHRoZSBjb3R0b24gKEdvc3N5cGl1bSBoaXJzdXR1bSkgZmliZXIgdHJhbnNjcmlwdG9tZTsgY2l0YXRpb25fYXV0aG9yPVlvbyAgTS5KLjsgY2l0YXRpb25fYXV0aG9yPVdlbmRlbCAgSi5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDA3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGVhdGVkIHBvbHlwbG9pZGl6YXRpb24gb2YgR29zc3lwaXVtIGdlbm9tZXMgYW5kIHRoZSBldm9sdXRpb24gb2Ygc3Bpbm5hYmxlIGNvdHRvbiBmaWJyZXM7IGNpdGF0aW9uX2F1dGhvcj1QYXRlcnNvbiAgQS5ILjsgY2l0YXRpb25fYXV0aG9yPVdlbmRlbCAgSi5GLjsgY2l0YXRpb25fYXV0aG9yPUd1bmRsYWNoICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MjsgIGNpdGF0aW9uX3BhZ2VzPTQyMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGN1bHRpdmF0ZWQgY290dG9uIEdvc3N5cGl1bSBhcmJvcmV1bTsgY2l0YXRpb25fYXV0aG9yPUxpICBGLjsgY2l0YXRpb25fYXV0aG9yPUZhbiAgRy47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz01NjctNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIG1vbGVjdWxhciBtYXJrZXJzIGZyb20gSW5kaWFuIGdlbm90eXBlcyBvZiB0d28gR29zc3lwaXVtIEwuIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1Tcml2YXN0YXZhICBBLjsgY2l0YXRpb25fYXV0aG9yPUplbmEgIFMuTi47IGNpdGF0aW9uX2F1dGhvcj1SYW5qYW4gIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCcmVlZDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz01MDYtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBtYXBwaW5nIGZvciBmaWJlciBxdWFsaXR5IHRyYWl0cyBhY3Jvc3MgbXVsdGlwbGUgZ2VuZXJhdGlvbnMgYW5kIGVudmlyb25tZW50cyBpbiB1cGxhbmQgY290dG9uOyBjaXRhdGlvbl9hdXRob3I9U3VuICBGLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBKLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NTY5LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIDMzNDctbG9jdXMgZ2VuZXRpYyByZWNvbWJpbmF0aW9uIG1hcCBvZiBzZXF1ZW5jZS10YWdnZWQgc2l0ZXMgcmV2ZWFscyBmZWF0dXJlcyBvZiBnZW5vbWUgb3JnYW5pemF0aW9uLCB0cmFuc21pc3Npb24gYW5kIGV2b2x1dGlvbiBvZiBjb3R0b24gKEdvc3N5cGl1bSk7IGNpdGF0aW9uX2F1dGhvcj1Sb25nICBKLksuOyBjaXRhdGlvbl9hdXRob3I9QWJiZXkgIEMuOyBjaXRhdGlvbl9hdXRob3I9Qm93ZXJzICBKLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0zODktNDE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIGdlbm90eXBpbmcgb2YgcmVzdHJpY3Rpb24gZnJhZ21lbnQgYXNzb2NpYXRlZCBwb2x5bW9ycGhpc21zIGluIHBvbHlwbG9pZCBjcm9wcyB3aXRoIGEgcHNldWRvLXJlZmVyZW5jZSBzZXF1ZW5jZTogYSBjYXNlIHN0dWR5IGluIGFsbG90ZXRyYXBsb2lkIEJyYXNzaWNhIG5hcHVzOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgWC47IGNpdGF0aW9uX2F1dGhvcj1MaSAgWC47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8yLzE0Ny85NTE2MTgvZHN1MDQ3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBSQUQgc2VxdWVuY2luZyB3YXMgcGVyZm9ybWVkIHVzaW5nIERIOTYyIGFuZCBKaW1pYW41IGFzIHVwbGFuZCBjb3R0b24gbWFwcGluZyBwYXJlbnRzLiBTZXF1ZW5jaW5nIGRhdGEgZm9yIERIOTYyIGFuZCBKaW1pYW41IHdlcmUgYXNzZW1ibGVkIGludG8gdGhlICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMyODI2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZE7wlT9vbjjHQE54acfzrd35okqI" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f147%2f332826" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f147%2f332826" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12675" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&amp;Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/2">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4593499">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593499">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593503">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593503">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593514">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593514">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593580">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593580">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593617">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593627">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593627">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593629">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593629">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593631">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593631">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593633">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593633">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4593635">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4593635">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/2/133/332691">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/2/161/332908">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hantao Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hantao+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hantao Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xin Jin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xin+Jin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jin X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jin X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xin Jin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beibei Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Beibei+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Beibei Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chao Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chao+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shen C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chao Shen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhongxu Lin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070, Hubei</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-87283955. Fax. +86 27-87280196. E-mail: <a href="mailto:linzhongxu@mail.hzau.edu.cn" target="_blank">linzhongxu@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhongxu+Lin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lin Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lin Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhongxu Lin</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 147–160, <a href='https://doi.org/10.1093/dnares/dsu047'>https://doi.org/10.1093/dnares/dsu047</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 February 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 October 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 December 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="332826" href="/dnaresearch/article-pdf/22/2/147/951618/dsu047.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hantao Wang, Xin Jin, Beibei Zhang, Chao Shen, Zhongxu Lin;  Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing, <em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 147–160, <a href='https://doi.org/10.1093/dnares/dsu047'>https://doi.org/10.1093/dnares/dsu047</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/332826?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/332826?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Enrichment%20of%20an%20intraspecific%20genetic%20map%20of%20upland%20cotton%20by%20developing%20markers%20using%20parental%20RAD%20sequencing&amp;publicationDate=2015-02-05&amp;volumeNum=22&amp;issueNum=2&amp;author=Wang%2C%20Hantao%3B%20Jin%2C%20Xin&amp;startPage=147&amp;endPage=160&amp;contentId=10.1093%2Fdnares%2Fdsu047&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f147%2f332826" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4593499 id="4593499" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>RAD sequencing was performed using DH962 and Jimian5 as upland cotton mapping parents. Sequencing data for DH962 and Jimian5 were assembled into the genome sequences of ≈55.27 and ≈57.06 Mb, respectively. Analysing genome sequences of the two parents, 1,323 SSR, 3,838 insertion/deletion (InDel), and 9,366 single-nucleotide polymorphism (SNP) primer pairs were developed. All of the SSRs, 121 InDels, 441 SNPs, and other 6,747 primer pairs were screened in the two parents, and a total of 535 new polymorphic loci were identified. A genetic map including 1,013 loci was constructed using these results and 506 loci previously published for this population. Twenty-seven new QTLs for yield and fibre quality were identified, indicating that the efficiency of QTL detection was greatly improved by the increase in map density. Comparative genomics showed there to be considerable homology and collinearity between the A<sub>T</sub> and A<sub>2</sub> genomes and between the D<sub>T</sub> and D<sub>5</sub> genomes, although there were a few exchanges and introgressions among the chromosomes of the A<sub>2</sub> genome. Here, the development of markers using parental RAD sequencing was effective, and a high-density intraspecific genetic map was constructed. This map can be used for molecular marker-assisted selection in cotton.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Gossypium hirsutum, RAD sequencing&quot;"><em>Gossypium hirsutum</em>, RAD sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL mapping&quot;">QTL mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=4593503 id="4593503" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Among the four cultivated species (two diploids, <em>Gossypium herbaceum</em> and <em>Gossypium arboreum</em>, and two allotetraploids, <em>Gossypium hirsutum</em> and <em>Gossypium barbadense</em>) of cotton, upland cotton (<em>G. hirsutum</em>, 2<em>n</em> = 4<em>x</em> = 52, genome size ≈2.5 Gb) is widely cultivated in the world and accounts for ∼95% of worldwide cotton production.<sup><a reveal-id="DSU047C1" data-open="DSU047C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, the narrow genetic background of upland cotton has resulted in inbreeding depression and reduced genetic variability, and the improvement of cotton yield was slow using conventional cultivar breeding programmes. The development of molecular marker technology provides an opportunity for the construction of a genetic linkage map. A high-density genetic map creates a better platform for researching the structure of an organism's genome, for dissecting traits of interest, for fine mapping of QTLs, and for map-based cloning.<sup><a reveal-id="DSU047C2" data-open="DSU047C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>There are 19,074 publicly available SSRs (genomic SSRs and EST-SSRs), 3,541 RFLPs, 2,146 AFLPs, and 1,018 single-nucleotide polymorphisms (SNPs) on the CottonGen database (<a class="link link-uri" href="http://www.cottongen.org/">http://www.cottongen.org/</a>), and most of them have been used to construct interspecific and intraspecific genetic maps. However, due to the narrow genetic diversity of upland cotton, the polymorphism of molecular markers between intraspecific hybrids of upland cotton is low. To date, due to the number of markers and the genome coverage, upland cotton genetic maps are inferior to interspecific maps.<sup><a reveal-id="DSU047C3  DSU047C6" data-open="DSU047C3  DSU047C6" class="link link-ref link-reveal xref-bibr">3–6</a></sup> The development of more polymorphic markers is thus needed.</p><p>SNPs are single-base variations, including transitions, transversions, and insertions/deletions (InDels). SNPs are the most abundant type of molecular genetic markers in the genome, can be found in coding and non-coding regions,<sup><a reveal-id="DSU047C7" data-open="DSU047C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and are used for genetic map construction, genetic diversity analysis, QTL mapping, and marker-assisted selection breeding.<sup><a reveal-id="DSU047C8" data-open="DSU047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The rapid development of next-generation sequencing (NGS) technology has facilitated genome-wide SNP discovery, and studies of NGS-derived SNPs have been reported in diploid and complex polyploid plants such as common bean,<sup><a reveal-id="DSU047C8" data-open="DSU047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> rice,<sup><a reveal-id="DSU047C9" data-open="DSU047C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and sorghum.<sup><a reveal-id="DSU047C10" data-open="DSU047C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The application of NGS-derived SNPs in cotton was rare. Byers <em>et al.</em><sup><a reveal-id="DSU047C11" data-open="DSU047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> detected a larger number of SNPs generated from the GR-RSC libraries by using the Roche 454 pyrosequencing platform, in which 11,834 SNPs were found in 6,469 contigs between two <em>G. hirsutum</em>, Acala Maxxa and TX2094. As one of several methods that are based on NGS platforms to develop markers,<sup><a reveal-id="DSU047C12" data-open="DSU047C12" class="link link-ref link-reveal xref-bibr">12</a></sup> restriction site-associated DNA sequencing (RAD-Seq) obtains sequences of restriction enzyme digestion tags using Illumina sequencing.<sup><a reveal-id="DSU047C13" data-open="DSU047C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This technology can greatly reduce the complexity of genomes, can identify abundant genetic markers quickly in an entire genome of some species with and without a reference genome, and also combines the advantages of low cost and high throughput.<sup><a reveal-id="DSU047C14" data-open="DSU047C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Bus <em>et al.</em><sup><a reveal-id="DSU047C15" data-open="DSU047C15" class="link link-ref link-reveal xref-bibr">15</a></sup> detected &gt;20,000 SNPs and 125 InDels from &gt;113,000 RAD clusters of <em>Brassica napus</em>, and about one-third of the RAD clusters were mapped on the reference sequence of <em>Brassica rapa</em>. The RAD-seq has been used to generate large numbers of SNPs for many species recently.<sup><a reveal-id="DSU047C15  DSU047C17" data-open="DSU047C15  DSU047C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> This method can be applied in upland cotton to explore and develop more genetic markers to promote genome research in this species.</p><p>In this study, two RAD libraries using two mapping parents of upland cotton were constructed. The objectives were to (i) develop molecular markers based on parental RAD sequencing, (ii) enrich a high-density <em>G. hirsutum</em> genetic map using these molecular markers, (iii) detect new QTLs associated with yield components and fibre quality traits, and (iv) assess the homology of <em>G. hirsutum</em> to the A<sub>2</sub> and D<sub>5</sub> genomes.</p>                            <h2 scrollto-destination=4593514 id="4593514" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4593517 id="4593517" class="section-title" data-legacy-id=s2a>2.1. Plant materials and field trait data collection</h3>
<p><em>Gossypium hirsutum</em> acc. DH962 and <em>G. hirsutum</em> cv. Jimian5 served as the parents of an F<sub>2</sub> mapping population.<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> This mapping population was used to screen for polymorphisms and to construct a genetic map.</p><p>The traits of F<sub>2</sub> individuals were represented by the average values of F<sub>2:3</sub> lines.<sup><a reveal-id="DSU047C18" data-open="DSU047C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The traits included the number of bolls per plant (BN), seed cotton weight per boll (SCW), lint weight per boll (LW), lint percentage (LP), lint index (LI), seed index (SI), fibre length (FL, mm), fibre strength (FS, cN/tex), fibre length uniformity ratio (FU), fibre elongation (FE), and fibre micronaire (MV).</p>                            <h3 scrollto-destination=4593524 id="4593524" class="section-title" data-legacy-id=s2b>2.2. RAD library preparation and sequencing</h3>
<p>Fresh young leaf tissue from the two parents was used to extract the genomic DNA using a Plant Genomic DNA Kit (TIANGEN Biotech, Beijing, China) in accordance with the manufacturer's instructions. Each of the two DNA samples was processed into RAD libraries in a manner similar to that reported by Baird <em>et al.</em><sup><a reveal-id="DSU047C14" data-open="DSU047C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Briefly, genomic DNA was digested for 60 min at 37°C in a 50 μl reaction with 20 units (U) of <em>Eco</em>RI (New England Biolabs, NEB), and then the samples were heat inactivated at 65°C for 20 min. Then 2.5 μl of 100 nM P1 adapter, a modified Illumina adapter (Illumina, Inc.), was added to each sample along with 1 μl of 10 mM ATP (Promega), 1 μl of 10× NEB Buffer 4, 1 μl of 1,000 U of T4 DNA ligase (Enzymatics, Inc.), and 5 μl H<sub>2</sub>O, and the reaction was incubated at room temperature for 20 min. Samples were again heat inactivated for 20 min at 65°C, pooled, and randomly sheared with a Bioruptor (Diagenode) to an average size of 500 bp. Samples were then separated by electrophoresis through a 1.5% agarose, 0.5× TBE gel, and DNA fragments from 300 to 700 bp were isolated using a MinElute Gel Extraction Kit (Qiagen). End blunting enzymes (Enzymatics, Inc.) were used to polish the dsDNA ends. The samples were repurified using a MinElute column (Qiagen). Then 15 U of Exo-Klenow (Enzymatics, Inc.) was added, and the sample was incubated at 37°C to generate 3′ adenine overhangs. The samples were purified, and 1 μl of 10 μM P2 adapter, a divergent modified Solexa adapter (Illumina, Inc.), was ligated to these DNA fragments at 18°C. The samples were again purified as above and eluted in 50 μl. The eluate was quantified using a Qubit fluorimeter, and 20 ng of this product was used in PCR amplification with 20 μl Phusion MasterMix (NEB), 5 μl of 10 μM modified Solexa Amplification primer mix (Illumina, Inc.), and up to 100 μl H<sub>2</sub>O. Phusion PCR settings were copied from the product guidelines, and the samples were processed for a total of 18 cycles. Samples were gel purified, and excised DNA ranged from 300 to 700 bp in size. It was diluted to 1 nM.</p><p>The two RAD libraries were run on an Illumina Genome Analyzer II at Beijing Genomics Institute (BGI) in Shenzhen. Illumina/Solexa protocols were followed for a 2 × 50 base paired-end (PE) sequencing run.</p>                            <h3 scrollto-destination=4593531 id="4593531" class="section-title" data-legacy-id=s2c>2.3. Sequence analysis and <em>de novo</em> assembly</h3>
<p>Raw reads of the two materials were filtered to remove reads with the following conditions: &gt;2% <em>N</em> calls, polyA structures, adapter contamination, base (quality scores ≤ 5) number accounts for 50% of the reads in PE libraries. A FASTX Toolkit Updated (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>) was used to filter the PE reads further, and then the sequences were assembled using a Velvet sequence assembler (version 1.0.18).<sup><a reveal-id="DSU047C19" data-open="DSU047C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This system was with a hash length of 31 bp and a minimum contig size of 200 bp, with other parameters set to default values.</p><p>CAP3<sup><a reveal-id="DSU047C20" data-open="DSU047C20" class="link link-ref link-reveal xref-bibr">20</a></sup> was used to identify sequences in common between the mapping parents, with the overlap length cut-off set at 80 bp and overlap percent cut-off set at 95. The resulting non-redundant data set (Gh-D-J; <em>Gossypium hirsutum</em>-DH962-Jimian5) included singletons from ‘DH962’ and ‘Jimian5’ and common contigs derived from both parents' assembled contigs.</p>                            <h3 scrollto-destination=4593538 id="4593538" class="section-title" data-legacy-id=s2d>2.4. Sequence annotation</h3>
<p>A BLASTX search was performed against the TAIR10 protein databases (<a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) with an <em>E</em>-value cut-off of 1e−15 for the sequences of the Gh-D-J data set. The annotated sequences were assigned functions based on Arabidopsis GO SLIM (<a class="link link-uri" href="http://ftp://ftp.arabidopsis.org/home/tair/Ontologies/Gene_Ontology/">ftp://ftp.arabidopsis.org/home/tair/Ontologies/Gene_Ontology/</a>) and then mapped to higher level categories (plant GO slim) using GOSlim Viewer<sup><a reveal-id="DSU047C21" data-open="DSU047C21" class="link link-ref link-reveal xref-bibr">21</a></sup> according to the three principal GO categories: molecular function, cellular component, and biological process.<sup><a reveal-id="DSU047C22" data-open="DSU047C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=4593543 id="4593543" class="section-title" data-legacy-id=s2e>2.5. Mining of SSRs, InDels, and SNPs and primer design</h3>
<p>An overall workflow of marker discovery and analysis performed in this study is shown in Fig. <a reveal-id="DSU047F1" data-open="DSU047F1" class="link link-reveal link-table xref-fig">1</a>. </p>                        <a id="4593548" scrollto-destination="4593548"></a>
<div id="DSU047F1" data-id="DSU047F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04701.png?Expires=2147483647&Signature=gmXM0uGe9BhmvqRL~S8zG8NpBwLL6uxyixuPxStY5LzXbVvg6bGmZGBPpXVm9R37MYjwMcVdMfm~xcVANa~ywgjZcBfLMimCKW7Ao~294zEqFl6eKYZYlTq7msWGNn8wEwuZAmi1NmVF9WkenNzmK55ic5eQK8ehsABPeP-mfyt~zPFKie483NaemqR6fHJBQkhhYBAOavz3K~547S-vVh7Mdjtjp20klfxHzvEE~5RbpkdEr-R43rIqRAVwYHlCBb8K25KK0EBiSCFjj~YxNDRDWp3viKqBfdc9BrnwhCEKN7l8vn4sEtEXo3EM6VP8KTVVWbLxYRIPf7QFneqgng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall workflow of marker discovery and analysis." path-from-xml="dsu04701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593548/dsu04701.tiff" path-from-xml="dsu04701.tiff" target="_blank">View large</a><a section="4593548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04701.png?Expires=2147483647&Signature=FMlRfz4TXmFKCiSUctSozmHt3sSXMZ3Nl-cURMNrOkYHDAOHI1azmiUmIzrjR3Zrh7~9MkXb4KLqG4LypYRwerSe03CEbceNez7h3JxeH6irLVJDiwNyQ0plu-RB-CgVSWRVgGfVZ43dsXjp7kknGJAf5I3g-GoUYwlWkAxSf-jOYs-hSPdL5FlN1s36N5ibqaX-egVfWvcEvY46dxIkzTdJXQhwyTRpgkSKC6nU-WkD2vJzCYKKpLX9KFfKl7m~LUq39nQdR1pjWKUMUduU00qeO9uRRGxTM4Ut44gzxgHOdmsKliYvEiIMc0sEvcUEXh3N-MpJY1hrxCVyUSpuTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593548&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overall workflow of marker discovery and analysis.</p></div></div><div content-id="DSU047F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04701.png?Expires=2147483647&Signature=gmXM0uGe9BhmvqRL~S8zG8NpBwLL6uxyixuPxStY5LzXbVvg6bGmZGBPpXVm9R37MYjwMcVdMfm~xcVANa~ywgjZcBfLMimCKW7Ao~294zEqFl6eKYZYlTq7msWGNn8wEwuZAmi1NmVF9WkenNzmK55ic5eQK8ehsABPeP-mfyt~zPFKie483NaemqR6fHJBQkhhYBAOavz3K~547S-vVh7Mdjtjp20klfxHzvEE~5RbpkdEr-R43rIqRAVwYHlCBb8K25KK0EBiSCFjj~YxNDRDWp3viKqBfdc9BrnwhCEKN7l8vn4sEtEXo3EM6VP8KTVVWbLxYRIPf7QFneqgng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall workflow of marker discovery and analysis." path-from-xml="dsu04701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593548/dsu04701.tiff" path-from-xml="dsu04701.tiff" target="_blank">View large</a><a section="4593548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04701.png?Expires=2147483647&Signature=FMlRfz4TXmFKCiSUctSozmHt3sSXMZ3Nl-cURMNrOkYHDAOHI1azmiUmIzrjR3Zrh7~9MkXb4KLqG4LypYRwerSe03CEbceNez7h3JxeH6irLVJDiwNyQ0plu-RB-CgVSWRVgGfVZ43dsXjp7kknGJAf5I3g-GoUYwlWkAxSf-jOYs-hSPdL5FlN1s36N5ibqaX-egVfWvcEvY46dxIkzTdJXQhwyTRpgkSKC6nU-WkD2vJzCYKKpLX9KFfKl7m~LUq39nQdR1pjWKUMUduU00qeO9uRRGxTM4Ut44gzxgHOdmsKliYvEiIMc0sEvcUEXh3N-MpJY1hrxCVyUSpuTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593548&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overall workflow of marker discovery and analysis.</p></div></div>                            <h4 scrollto-destination=4593552 id="4593552" class="section-title" data-legacy-id=s2e1>2.5.1. SSR</h4>
<p>The contigs of two parents were screened for SSR motifs using the microsatellite (MISA) searching tool (<a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>) implemented in PERL. The parameter settings were as follows: the minimum repeat unit was defined as seven repeats for dinucleotide motifs, five repeats for tri-motifs, four repeats for tetra-motifs, and three repeats for penta-, hexa-, hepta-, and octa-motifs. The maximum number of interrupting bases in a compound microsatellite was set to 500.</p>                            <h4 scrollto-destination=4593556 id="4593556" class="section-title" data-legacy-id=s2e2>2.5.2. InDel</h4>
<p>BWA (<a class="link link-uri" href="http://sourceforge.net/projects/bio-bwa/files/">http://sourceforge.net/projects/bio-bwa/files/</a>) was used to map the RAD sequencing reads of DH962 sample to the assembled contigs of Jimian5 and map the RAD sequencing reads of Jimian5 sample to the assembled contigs of DH962. The mapping results were transformed into bam format with SAMtool v0.1.19.<sup><a reveal-id="DSU047C23" data-open="DSU047C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Then the bam files were sorted, and duplicates were removed using SAMtool v0.1.19. Finally, GATK software was used to call InDels.<sup><a reveal-id="DSU047C24" data-open="DSU047C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h4 scrollto-destination=4593560 id="4593560" class="section-title" data-legacy-id=s2e3>2.5.3. SNP</h4>
<p>SOAPaligner v2.21 (<a class="link link-uri" href="http://soap.genomics.org.cn/soapaligner.html">http://soap.genomics.org.cn/soapaligner.html</a>) software was used to map the RAD sequencing reads of DH962 to the assembled contigs of Jimian5 and map the RAD sequencing reads of Jimian5 to the assembled contigs of DH962, run with a maximum mismatch of 2 bp. The mapping results, shown as RAD SE reads, were used to identify SNPs using SOAPsnp.<sup><a reveal-id="DSU047C25" data-open="DSU047C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>For all SNPs and InDels mined, sites were considered as homozygous when the minor allele frequency was below 0.10, heterozygous when the minor allele frequency was above 0.25, and unknown when the minor allele frequency was between 0.10 and 0.25.</p><p>The contigs containing markers were used to design primers, employing the Primer3 program (<a class="link link-uri" href="http://bioinfo.ut.ee/primer3/">http://bioinfo.ut.ee/primer3/</a>). The primer length was set between 18 and 24 nucleotides with an optimum size of 20 bp; the optimum annealing temperature was 57°C; the GC content was set between 40 and 60% with an optimum GC content of 50%; and the predicted PCR products ranged from 100 to 500 bp. If the distance between two or more markers was &lt;500 bp in a region, primers were for both ends of that region. However, the predicted PCR product was shorter than 800 bp in this study.</p>                            <h3 scrollto-destination=4593567 id="4593567" class="section-title" data-legacy-id=s2f>2.6. Remove the redundancy of SSRs</h3>
<p>A total of 13 Mb of sequence data (genomic SSRs and EST-SSRs) were collected from all the SSR markers of the CottonGen database and used to assess the novelty of these SSR sequences. Among the collected markers, some were assembled by several ESTs using the Phrap program (<a class="link link-uri" href="http://www.phrap.org/index.html">http://www.phrap.org/index.html</a>). Because most of the collected markers were from ESTs, and RAD sequences were genomic, it was necessary to remove the redundant sequences accurately. This was accomplished as follows: (i) the sequences between the SSR primers were extracted from RAD sequences; (ii) the SSRs in extracted sequences were shielded using RepeatMask (<a class="link link-uri" href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>) program; (iii) the remaining of sequences, which lacked SSRs, were BLAST against the collected markers; if the matched bases numbered ≥50, the SSR primers were considered to be redundant.</p>                            <h3 scrollto-destination=4593569 id="4593569" class="section-title" data-legacy-id=s2g>2.7. Marker redundancy check between DH962 and Jimian5</h3>
<p>Among markers between DH962 and Jimian5, some were located on the same location or homologous location. We used the immediate flanking sequences (≥20 bp on both sides) of Jimian5 markers to BLAST the immediate flanking sequences (≥20 bp on both sides) of DH962 markers with an <em>E</em>-value cut-off of 1e−5 (Fig. <a reveal-id="DSU047F2" data-open="DSU047F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="4593571" scrollto-destination="4593571"></a>
<div id="DSU047F2" data-id="DSU047F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04702.png?Expires=2147483647&Signature=icTBvz-aDmTALwK6C3aOl3DXN00cTwYK5IA844aIhqGPveV0LSh7NmeMKeigWLQOe0tE-wbX~03iGJiWw2Go~DdmOGM06--bgxLcBMy4KD~TVCwAjYg6KVrTyQWPgzECdsyxQx7WFeH8ExMXN7pNsCuegu-Fc7yFj8fUHy-Zk~HWomHp7XOpQaLMnwGGMHThO0Cpx7IkNU2dWeh7CUyMijqFAVv-YNyAWHi6~Kvu-Z8w21fU~BUDTgXq3-D8xx-o3aRxNf-b2iuMRu0NZAIH50XnvtjRxXmyYVc1ntarpWUmuKOB3pFfwj7b1xdMSbYnei4B7th6vZckau-0skYaEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analysis of the flanking sequences of markers of DH962 and Jimian5." path-from-xml="dsu04702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593571/dsu04702.tiff" path-from-xml="dsu04702.tiff" target="_blank">View large</a><a section="4593571" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04702.png?Expires=2147483647&Signature=zdgd3kCQ-HTIasocTUfUpuK8wfwHncdt4cuZSKVXirsiwJmGIyGn-nFnONMyHPlUSQToAfW0G1vemTipHerFcawUv4BC3AHb0-SaFhhO4Vz5DjiFLre9ylKawzLTQGKOn1zajnFImMunl9K2rcE~RMQ3H~iApRHDfzkTYfLC4X6r8spONkc0MtrfBnpbTALW0qrOrn5ADkfK54rZEV6q4tTO54xkQho2eDzJZlgWHPDVh5I0OOYT-CA45aabep3PFEnLCl2AjpDaD5h5QnHZksDd5T-dGD5eUyJ1aq4xvMOZ0doJtmJ3HPcf~AUS0NM~Xsw8X5Hydzk0mCR7dQAUEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593571&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analysis of the flanking sequences of markers of DH962 and Jimian5.</p></div></div><div content-id="DSU047F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04702.png?Expires=2147483647&Signature=icTBvz-aDmTALwK6C3aOl3DXN00cTwYK5IA844aIhqGPveV0LSh7NmeMKeigWLQOe0tE-wbX~03iGJiWw2Go~DdmOGM06--bgxLcBMy4KD~TVCwAjYg6KVrTyQWPgzECdsyxQx7WFeH8ExMXN7pNsCuegu-Fc7yFj8fUHy-Zk~HWomHp7XOpQaLMnwGGMHThO0Cpx7IkNU2dWeh7CUyMijqFAVv-YNyAWHi6~Kvu-Z8w21fU~BUDTgXq3-D8xx-o3aRxNf-b2iuMRu0NZAIH50XnvtjRxXmyYVc1ntarpWUmuKOB3pFfwj7b1xdMSbYnei4B7th6vZckau-0skYaEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analysis of the flanking sequences of markers of DH962 and Jimian5." path-from-xml="dsu04702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593571/dsu04702.tiff" path-from-xml="dsu04702.tiff" target="_blank">View large</a><a section="4593571" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04702.png?Expires=2147483647&Signature=zdgd3kCQ-HTIasocTUfUpuK8wfwHncdt4cuZSKVXirsiwJmGIyGn-nFnONMyHPlUSQToAfW0G1vemTipHerFcawUv4BC3AHb0-SaFhhO4Vz5DjiFLre9ylKawzLTQGKOn1zajnFImMunl9K2rcE~RMQ3H~iApRHDfzkTYfLC4X6r8spONkc0MtrfBnpbTALW0qrOrn5ADkfK54rZEV6q4tTO54xkQho2eDzJZlgWHPDVh5I0OOYT-CA45aabep3PFEnLCl2AjpDaD5h5QnHZksDd5T-dGD5eUyJ1aq4xvMOZ0doJtmJ3HPcf~AUS0NM~Xsw8X5Hydzk0mCR7dQAUEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593571&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analysis of the flanking sequences of markers of DH962 and Jimian5.</p></div></div><p>Finally, the remaining markers with primers were given the prefix HAU (indicating Huazhong Agricultural University) and synthesized by Beijing Tianyi Huiyuan Life Science and Technology, Inc. (Wuhan, China).</p>                            <h3 scrollto-destination=4593573 id="4593573" class="section-title" data-legacy-id=s2h>2.8. Homology of the upland cotton to the A<sub>2</sub> and D<sub>5</sub> genomes</h3>
<p>For further analysis of the homology of the two sub-genomes of upland cotton to the A<sub>2</sub> and D<sub>5</sub> genomes, a BLASTN search was performed against the diploid A<sub>2</sub> genome of <em>Gossypium arboretum</em> and D<sub>5</sub> genome of <em>Gossypium raimondii</em> (<a class="link link-uri" href="http://www.phytozome.net/cotton.php">http://www.phytozome.net/cotton.php</a>) with an <em>E</em>-value cut-off of 1e−10 for the sequences of primers on the upland cotton genetic map in this study.</p>                            <h3 scrollto-destination=4593575 id="4593575" class="section-title" data-legacy-id=s2i>2.9. Marker genotyping</h3>
<p>In addition to the primers developed in this study, a total of 1,869 markers were also chosen according to the criteria published by Wang<sup><a reveal-id="DSU047C26" data-open="DSU047C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and an interspecific BC<sub>1</sub> genetic map from this lab.<sup><a reveal-id="DSU047C27" data-open="DSU047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A total of 4,878 SSRs (3,479 HAU, 699 NAU, 700 Gh) were obtained from previous studies.<sup><a reveal-id="DSU047C28  DSU047C34" data-open="DSU047C28  DSU047C34" class="link link-ref link-reveal xref-bibr">28–34</a></sup> PCR amplifications of primers and silver staining were performed as described by Lin <em>et al.</em><sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> PCR products of all the primer pairs were separated on 6% denaturing polyacrylamide gel<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> or 8% native polyacrylamide gels using single-strand conformation polymorphism (SSCP) technology.<sup><a reveal-id="DSU047C26" data-open="DSU047C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The SSCP technology workflow was as follows: PCR products were run on 8% native polyacrylamide gels (1:29, bis to acrylamide). The gels were exposed to electrophoresis at a constant 15 W at 25°C for 3.5–4 h. After electrophoresis, the DNA fragments were visualized by silver staining. The silver staining protocol was the same as the SSR protocol.</p>                            <h3 scrollto-destination=4593577 id="4593577" class="section-title" data-legacy-id=s2j>2.10. Linkage map construction and QTL analysis</h3>
<p>A <em>χ</em><sup>2</sup> test was performed to determine whether the genotypic frequencies at each locus deviated from the expected 1 : 2 : 1 or 3 : 1 segregation ratio in the F<sub>2</sub> population. The polymorphic loci were integrated into the F<sub>2</sub> linkage map using JoinMap V3.0.<sup><a reveal-id="DSU047C35" data-open="DSU047C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The logarithm of odds (LOD) threshold was 4.0, and the maximum recombination fraction was 0.4. Genetic map distances in centiMorgans (cM) were calculated using the Kosambi mapping function. The resulting linkage map was drawn using MapChart V2.2 software.<sup><a reveal-id="DSU047C36" data-open="DSU047C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Linkage groups were assigned to chromosomes on the basis of a BC<sub>1</sub> linkage map,<sup><a reveal-id="DSU047C27" data-open="DSU047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> an F<sub>2</sub> linkage map,<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and marker mapping information on the CottonGen database.</p><p>QTLs were identified using Windows QTL Cartographer 2.5 (<a class="link link-uri" href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a>) by composite interval mapping (CIM). The statistical significance of the LOD threshold value was determined by running a permutation procedure 1,000 times for all traits. QTL nomenclature was adapted using the method proposed by McCouch <em>et al.</em><sup><a reveal-id="DSU047C37" data-open="DSU047C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h2 scrollto-destination=4593580 id="4593580" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4593581 id="4593581" class="section-title" data-legacy-id=s3a>3.1. RAD sequencing and <em>de novo</em> contig assembly</h3>
<p>A total of 62.46 million and 61.27 million raw reads were produced from the DH962 and Jimian5 RAD libraries, respectively. After quality filtering, 5.15 and 5.18 Gb of clean reads were obtained from DH962 and Jimian5 RAD libraries, the GC contents were 34.00 and 34.17%, and the Q scores &gt;20 were 97.84 and 97.94%, respectively.</p><p>Initial <em>de nov</em>o assembly produced ≈55.27 Mb of DH962 genome sequence distributed over 178,157 individual contigs. Contig lengths ranged from 200 to 3,195 bp with an average length of 310 bp, and the lengths of most contigs were between 200 and 800 bp. The clean reads of Jimian5 were assembled into a ≈57.06 Mb genome sequence distributed over 181,422 individual contigs. Contig lengths ranged from 200 to 2,355 bp with an average length of 316 bp, and the lengths of most contigs were between 200 and 800 bp.</p><p>The Gh-D-J data set consisted of 251,816 sequences totalling ≈85.76 Mb (mean length 340 bp, Fig. <a reveal-id="DSU047F3" data-open="DSU047F3" class="link link-reveal link-table xref-fig">3</a>), of which 105,264 (38.14 Mb) sequences were shared (Table <a reveal-id="DSU047TB1" data-open="DSU047TB1" class="link link-reveal link-table xref-fig">1</a>). The repetitive elements in the sequences, Gh-D-J, were evaluated using the RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>). The relative number of repetitive elements on the contigs was 4.76%, which is similar to results from PE RAD-seq studies in other plant genomes.<sup><a reveal-id="DSU047C17" data-open="DSU047C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU047C38" data-open="DSU047C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The major classes of repetitive DNA elements belonged to Gypsy/DIRS1 and Ty1/Copia long-terminal repeat (LTR) retroelement families and simple repeats (Fig. <a reveal-id="DSU047F4" data-open="DSU047F4" class="link link-reveal link-table xref-fig">4</a>). The GC dinucleotide content for Gh-D-J was ∼34.72%, which is similar to that of both <em>Arabidopsis thaliana</em><sup><a reveal-id="DSU047C39" data-open="DSU047C39" class="link link-ref link-reveal xref-bibr">39</a></sup>and <em>Theobroma cacao</em>.<sup><a reveal-id="DSU047C39" data-open="DSU047C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSU047C40" data-open="DSU047C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p>                        <a id="4593585" scrollto-destination="4593585"></a>
<div content-id="DSU047TB1" class="table-modal"><div class="table-wrap"><div id="DSU047TB1" class="table-wrap-title" data-id="DSU047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics of the RAD tag sequencing via illumina (DH962, Jimian5)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Illumina reads (million) after sequence editing </td><td>62.46 </td><td>61.27 </td></tr><tr><td>Sequences bases (Gb) after sequence editing  </td><td>5.15 </td><td>5.18 </td></tr><tr><td>Number of contigs </td><td>178,157 (55.27 Mb) </td><td>181,422 (57.06 Mb) </td></tr><tr><td>Average contig length (bp) </td><td>310 </td><td>314 </td></tr><tr><td>Common contigs </td><td colspan="2">105,264 (38.14 Mb) </td></tr><tr><td>Contig number of <br />Gh-D-J data set </td><td colspan="2">251,816 (85.76 Mb) </td></tr><tr><td>Number of SSR primer pairs </td><td colspan="2">1,323 </td></tr><tr><td>Number of InDel primer pairs </td><td colspan="2">3,838 </td></tr><tr><td>Number of SNP primer pairs </td><td colspan="2">9,366 </td></tr><tr><td>Number of sequences with markers </td><td colspan="2">14,433 </td></tr></tbody></table></div><div content-id="DSU047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Illumina reads (million) after sequence editing </td><td>62.46 </td><td>61.27 </td></tr><tr><td>Sequences bases (Gb) after sequence editing  </td><td>5.15 </td><td>5.18 </td></tr><tr><td>Number of contigs </td><td>178,157 (55.27 Mb) </td><td>181,422 (57.06 Mb) </td></tr><tr><td>Average contig length (bp) </td><td>310 </td><td>314 </td></tr><tr><td>Common contigs </td><td colspan="2">105,264 (38.14 Mb) </td></tr><tr><td>Contig number of <br />Gh-D-J data set </td><td colspan="2">251,816 (85.76 Mb) </td></tr><tr><td>Number of SSR primer pairs </td><td colspan="2">1,323 </td></tr><tr><td>Number of InDel primer pairs </td><td colspan="2">3,838 </td></tr><tr><td>Number of SNP primer pairs </td><td colspan="2">9,366 </td></tr><tr><td>Number of sequences with markers </td><td colspan="2">14,433 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4593585" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU047TB1" class="table-wrap-title" data-id="DSU047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics of the RAD tag sequencing via illumina (DH962, Jimian5)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Illumina reads (million) after sequence editing </td><td>62.46 </td><td>61.27 </td></tr><tr><td>Sequences bases (Gb) after sequence editing  </td><td>5.15 </td><td>5.18 </td></tr><tr><td>Number of contigs </td><td>178,157 (55.27 Mb) </td><td>181,422 (57.06 Mb) </td></tr><tr><td>Average contig length (bp) </td><td>310 </td><td>314 </td></tr><tr><td>Common contigs </td><td colspan="2">105,264 (38.14 Mb) </td></tr><tr><td>Contig number of <br />Gh-D-J data set </td><td colspan="2">251,816 (85.76 Mb) </td></tr><tr><td>Number of SSR primer pairs </td><td colspan="2">1,323 </td></tr><tr><td>Number of InDel primer pairs </td><td colspan="2">3,838 </td></tr><tr><td>Number of SNP primer pairs </td><td colspan="2">9,366 </td></tr><tr><td>Number of sequences with markers </td><td colspan="2">14,433 </td></tr></tbody></table></div><div content-id="DSU047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Illumina reads (million) after sequence editing </td><td>62.46 </td><td>61.27 </td></tr><tr><td>Sequences bases (Gb) after sequence editing  </td><td>5.15 </td><td>5.18 </td></tr><tr><td>Number of contigs </td><td>178,157 (55.27 Mb) </td><td>181,422 (57.06 Mb) </td></tr><tr><td>Average contig length (bp) </td><td>310 </td><td>314 </td></tr><tr><td>Common contigs </td><td colspan="2">105,264 (38.14 Mb) </td></tr><tr><td>Contig number of <br />Gh-D-J data set </td><td colspan="2">251,816 (85.76 Mb) </td></tr><tr><td>Number of SSR primer pairs </td><td colspan="2">1,323 </td></tr><tr><td>Number of InDel primer pairs </td><td colspan="2">3,838 </td></tr><tr><td>Number of SNP primer pairs </td><td colspan="2">9,366 </td></tr><tr><td>Number of sequences with markers </td><td colspan="2">14,433 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4593585" target="_blank">View Large</a></div></div>                        <a id="4593586" scrollto-destination="4593586"></a>
<div id="DSU047F3" data-id="DSU047F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04703.png?Expires=2147483647&Signature=LN9KGVVIVF3EUSGQTBgGEx8IsNhbCQ49DUWdaKR42Rikxwao4REAIS2hinPhgktR9ITDwP9xa41hHNd9ajl8mQwhIlR9ynD4ElpOBl9EGHcjPI0WekOedIZfUVgPk2ndUZ2b6IqSZ67E8yKqWLaNf~0dCCMkbIyM7ZGnF8IGDVc3vqwP8UjBgdKz6jlFh1czjYf3tqzHqOU2moN5k-DcntzaQSdpLU6ChIUshsDPVEgoyPcUQT1rR~-LV89ChX4VJEg6oKjIauyUzekmcSb8MGQJu~XJDdZOLNTA3g6pP3SYbbbRm94PoSqEYVdN7lMdslyT0WWYc~QrWrrP3EhUlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence length distribution of the Gh-D-J data set." path-from-xml="dsu04703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593586/dsu04703.tiff" path-from-xml="dsu04703.tiff" target="_blank">View large</a><a section="4593586" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04703.png?Expires=2147483647&Signature=OZo7cozhXEvIzsOwu7SctzdjiIFiOL619-7pgWpd97i3JP3zMqiGVAUxNy7iSBbugMkamE~A6EWpfYiX5eFcnqFc2BZNK8v2Nx9DNwJeTFVvAxz0zvX7E3rGqsmWg~kO~ndoC1r7m-Tjcytb7ZUXYN6N1VlnELtg4FG0LNFEpa1q9kcWOWUmrEaMHEQSGVOMpMp4C7YbMSd0kUdpQki9F2HZw7B0LYNjvVCDJyELX86zyxvl8kcXxhi7dmlYJG77LQF0Hd0MqQ9kldK3W9GzXCpsGmShxAULrAHLJ9urntIOo~1OS8tAMKkWFjNiXZnu~04BidzwEAkxWH59b-uU0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593586&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence length distribution of the Gh-D-J data set.</p></div></div><div content-id="DSU047F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04703.png?Expires=2147483647&Signature=LN9KGVVIVF3EUSGQTBgGEx8IsNhbCQ49DUWdaKR42Rikxwao4REAIS2hinPhgktR9ITDwP9xa41hHNd9ajl8mQwhIlR9ynD4ElpOBl9EGHcjPI0WekOedIZfUVgPk2ndUZ2b6IqSZ67E8yKqWLaNf~0dCCMkbIyM7ZGnF8IGDVc3vqwP8UjBgdKz6jlFh1czjYf3tqzHqOU2moN5k-DcntzaQSdpLU6ChIUshsDPVEgoyPcUQT1rR~-LV89ChX4VJEg6oKjIauyUzekmcSb8MGQJu~XJDdZOLNTA3g6pP3SYbbbRm94PoSqEYVdN7lMdslyT0WWYc~QrWrrP3EhUlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence length distribution of the Gh-D-J data set." path-from-xml="dsu04703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593586/dsu04703.tiff" path-from-xml="dsu04703.tiff" target="_blank">View large</a><a section="4593586" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04703.png?Expires=2147483647&Signature=OZo7cozhXEvIzsOwu7SctzdjiIFiOL619-7pgWpd97i3JP3zMqiGVAUxNy7iSBbugMkamE~A6EWpfYiX5eFcnqFc2BZNK8v2Nx9DNwJeTFVvAxz0zvX7E3rGqsmWg~kO~ndoC1r7m-Tjcytb7ZUXYN6N1VlnELtg4FG0LNFEpa1q9kcWOWUmrEaMHEQSGVOMpMp4C7YbMSd0kUdpQki9F2HZw7B0LYNjvVCDJyELX86zyxvl8kcXxhi7dmlYJG77LQF0Hd0MqQ9kldK3W9GzXCpsGmShxAULrAHLJ9urntIOo~1OS8tAMKkWFjNiXZnu~04BidzwEAkxWH59b-uU0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593586&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence length distribution of the Gh-D-J data set.</p></div></div>                        <a id="4593587" scrollto-destination="4593587"></a>
<div id="DSU047F4" data-id="DSU047F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04704.png?Expires=2147483647&Signature=bpdjgE-yrVPB3N605kSeRFX4OuL3~tX~RFmCMbQMMVFEn-M4TApg1XxeibWajHYyjw8XoczERAEdf3-eASK~3YTgFggWVe8fq-HzO9KbmnfeSUrk5ztZNyLlQFQMVXjlQ32UyRDy0T3mqo~wTwhuCKzN57NEHi1ptWrLZG0mE~Rc6ZipNMhsNOyoVdvK3ld70NB-f-FFO8As5bb7Gdpwj8BbfMRG3YJoeWj4vTV23zLyAfioFSHYriXa5mzb2GolnPX9J05OQpwmzlfiaTpqlHRX~G49IPrr3iGp1H44CIWKkTqhjdKdmdIcc2kEp6jy5m-b23IHmJjV8G3HoH6iUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The representation of known repetitive elements in Gossypium hirsutum RAD sequences." path-from-xml="dsu04704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593587/dsu04704.tiff" path-from-xml="dsu04704.tiff" target="_blank">View large</a><a section="4593587" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04704.png?Expires=2147483647&Signature=kgnmxkWPo7jskTVE7zNJmTmAtncpgN~kgMGZAIhpQCAm46h2ECa7kGjHeMOAUPmZsqMrMlsheYtipoRPIoeXSMNNu9A3YfghS3lLvNifJNvsz9e1rMRAfV8Qez0OhXTc8vQnJoBllB19KTvDzxkIeEYx96dHHq1NAQLDI~Yw9~ejFsNNhNW-QSsKHqWyQ4J2sNf5Rs7xzU4vXeTTzMx7ffMZ6mBlT00j56~p45LsZMm~Nbb7tqAfnVM4zumpteXrr-dbTYcUQQaWhpMZe57xfH9eU0OV~HT7DswUVy2uvsw~o0m5S6wO3700-4o9IvLtuawIkTXGwaG-P59bQF1NgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593587&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The representation of known repetitive elements in <em>Gossypium hirsutum</em> RAD sequences.</p></div></div><div content-id="DSU047F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04704.png?Expires=2147483647&Signature=bpdjgE-yrVPB3N605kSeRFX4OuL3~tX~RFmCMbQMMVFEn-M4TApg1XxeibWajHYyjw8XoczERAEdf3-eASK~3YTgFggWVe8fq-HzO9KbmnfeSUrk5ztZNyLlQFQMVXjlQ32UyRDy0T3mqo~wTwhuCKzN57NEHi1ptWrLZG0mE~Rc6ZipNMhsNOyoVdvK3ld70NB-f-FFO8As5bb7Gdpwj8BbfMRG3YJoeWj4vTV23zLyAfioFSHYriXa5mzb2GolnPX9J05OQpwmzlfiaTpqlHRX~G49IPrr3iGp1H44CIWKkTqhjdKdmdIcc2kEp6jy5m-b23IHmJjV8G3HoH6iUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The representation of known repetitive elements in Gossypium hirsutum RAD sequences." path-from-xml="dsu04704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593587/dsu04704.tiff" path-from-xml="dsu04704.tiff" target="_blank">View large</a><a section="4593587" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04704.png?Expires=2147483647&Signature=kgnmxkWPo7jskTVE7zNJmTmAtncpgN~kgMGZAIhpQCAm46h2ECa7kGjHeMOAUPmZsqMrMlsheYtipoRPIoeXSMNNu9A3YfghS3lLvNifJNvsz9e1rMRAfV8Qez0OhXTc8vQnJoBllB19KTvDzxkIeEYx96dHHq1NAQLDI~Yw9~ejFsNNhNW-QSsKHqWyQ4J2sNf5Rs7xzU4vXeTTzMx7ffMZ6mBlT00j56~p45LsZMm~Nbb7tqAfnVM4zumpteXrr-dbTYcUQQaWhpMZe57xfH9eU0OV~HT7DswUVy2uvsw~o0m5S6wO3700-4o9IvLtuawIkTXGwaG-P59bQF1NgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593587&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The representation of known repetitive elements in <em>Gossypium hirsutum</em> RAD sequences.</p></div></div>                            <h3 scrollto-destination=4593588 id="4593588" class="section-title" data-legacy-id=s3b>3.2. GO categorization</h3>
<p>In all, 15,535 (6.17%) sequences of the Gh-D-J data set were significantly matched to 7,719 <em>A. thaliana</em> genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The annotated Gh-D-J sequences were further functionally classified using a plant-specific GO slim (Fig. <a reveal-id="DSU047F5" data-open="DSU047F5" class="link link-reveal link-table xref-fig">5</a>). Of the Gh-D-J sequences, 41,222 GO terms were categorized under biological process, 19,406 under cellular component, and 13,899 under molecular function. Fibre development is a very important process in cotton. Then 58 contigs were identified as being related to the MYB family, 10 contigs were found to be related to expansin biosynthesis, and 27 contigs were found to be related to the ethylene response. A total of 60 contigs were related to cell wall and cellulose biogenesis, 17 of which were related to cellulose synthase (<em>CESA</em>) genes (16 were required in expanding primary walls, and 1 was required in the thickening of secondary walls), 11 to the cellulose synthase protein family, and 32 to cellulose synthase-like (<em>CSL</em>) genes. </p>                        <a id="4593590" scrollto-destination="4593590"></a>
<div id="DSU047F5" data-id="DSU047F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04705.png?Expires=2147483647&Signature=IkRq-BQ9xzS6KiCpbRE1bqlJRde7ccyK81kKQjHD9ZwNAqBr3WJATa28zJnnS7hBnBYlh5WvBHeKT1Jj8TdoNGLmlZ3Apo9~lWauyteIYQPEfUB1GvAX3P~9nMjtAOP-WfcSVRb0wT05QrBEWWAsIqRtkpeyBi2Fuxs7MiKf~xrlZ7nM2LxkkhM~e-O88xG9IY1tJK6QI9g408bbLUnJBAXdt9eqHnv-t6TWh-9izcPkQtoI5HT3le0ln0i-9eqarCBlaYJGWTNyBjBa9Tz4KunhvOpJmDmPHzlqdQ8ZegdyPA9pISj4O9MnEiVrkYTV-mS8B~X6G3n5~UgFqA9V3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene ontology classification of the Gh-D-J data set. The three GO categories are cellular component, molecular function, and biological process." path-from-xml="dsu04705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593590/dsu04705.tiff" path-from-xml="dsu04705.tiff" target="_blank">View large</a><a section="4593590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04705.png?Expires=2147483647&Signature=LuZRtS-AZBgR8Jca5Dw7H3yh69hsFuP35Gp0jTNC81vIDneT6AIRnAbW8Gvm1Zdnr5-Gy7B8Y8AJL3UOmYo3PnhPhvnNQuk8HdvpvUDfNE7YRVcBoA7hDmR20xzTzDokfn86ka-69GPHObd-4IdVIfCPGFM8ob~a3JN8gD723zMyuHS9148Zi6YGbmnrmEPT~CGVym-IWJ8pgvs0waJuF4GJTplvTGpXiv6gFj2fxp9Ex1vqJl6r1ylf4BnLmZd1J47lQKdija~UNlgWOOdTSwZki-f6csZ0Ja9-fEV5iFP4MAe-QMwKdnt5oUZKIpQHwQMsTHj1JvDNg4ubWrvJTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593590&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene ontology classification of the Gh-D-J data set. The three GO categories are cellular component, molecular function, and biological process.</p></div></div><div content-id="DSU047F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04705.png?Expires=2147483647&Signature=IkRq-BQ9xzS6KiCpbRE1bqlJRde7ccyK81kKQjHD9ZwNAqBr3WJATa28zJnnS7hBnBYlh5WvBHeKT1Jj8TdoNGLmlZ3Apo9~lWauyteIYQPEfUB1GvAX3P~9nMjtAOP-WfcSVRb0wT05QrBEWWAsIqRtkpeyBi2Fuxs7MiKf~xrlZ7nM2LxkkhM~e-O88xG9IY1tJK6QI9g408bbLUnJBAXdt9eqHnv-t6TWh-9izcPkQtoI5HT3le0ln0i-9eqarCBlaYJGWTNyBjBa9Tz4KunhvOpJmDmPHzlqdQ8ZegdyPA9pISj4O9MnEiVrkYTV-mS8B~X6G3n5~UgFqA9V3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene ontology classification of the Gh-D-J data set. The three GO categories are cellular component, molecular function, and biological process." path-from-xml="dsu04705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593590/dsu04705.tiff" path-from-xml="dsu04705.tiff" target="_blank">View large</a><a section="4593590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04705.png?Expires=2147483647&Signature=LuZRtS-AZBgR8Jca5Dw7H3yh69hsFuP35Gp0jTNC81vIDneT6AIRnAbW8Gvm1Zdnr5-Gy7B8Y8AJL3UOmYo3PnhPhvnNQuk8HdvpvUDfNE7YRVcBoA7hDmR20xzTzDokfn86ka-69GPHObd-4IdVIfCPGFM8ob~a3JN8gD723zMyuHS9148Zi6YGbmnrmEPT~CGVym-IWJ8pgvs0waJuF4GJTplvTGpXiv6gFj2fxp9Ex1vqJl6r1ylf4BnLmZd1J47lQKdija~UNlgWOOdTSwZki-f6csZ0Ja9-fEV5iFP4MAe-QMwKdnt5oUZKIpQHwQMsTHj1JvDNg4ubWrvJTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593590&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene ontology classification of the Gh-D-J data set. The three GO categories are cellular component, molecular function, and biological process.</p></div></div><p>The research and discovery of pest- and disease-resistant genes is also important. A total of 46 contigs were found to be related to nucleotide-binding site (NBS) domains using GO analysis, 15 contigs were found to be related to the coiled-coil (CC)-NBS-leucine-rich repeat (LRR) family, 12 contigs were related to the toll/interleukin-1 receptor (TIR)-NBS family, and 19 contigs were related to the TIR-NBS-LRR family.</p>                            <h3 scrollto-destination=4593592 id="4593592" class="section-title" data-legacy-id=s3c>3.3. Characterization and identification of SSRs in the two RAD libraries</h3>
<p>A total of 5,158 SSRs (containing 244 compound SSRs) and 5,461 SSRs (containing 246 compound SSRs) were identified within contig sequences of DH962 and Jimian5, respectively, with an average frequency of 1/10.58 kb (Table <a reveal-id="DSU047TB2" data-open="DSU047TB2" class="link link-reveal link-table xref-fig">2</a>). In the two RAD libraries, the most abundant type of repeat motifs was pentanucleotide repeat (Table <a reveal-id="DSU047TB2" data-open="DSU047TB2" class="link link-reveal link-table xref-fig">2</a>). The frequencies of motif types were similar (Table <a reveal-id="DSU047TB2" data-open="DSU047TB2" class="link link-reveal link-table xref-fig">2</a>), with AT/TA as the most frequent repeat, followed by AAAAT/ATTTT, AAT/ATT, AAAT/ATTT, AAG/CTT, and AAAAG/CTTTT in descending order (Table <a reveal-id="DSU047TB2" data-open="DSU047TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="4593594" scrollto-destination="4593594"></a>
<div content-id="DSU047TB2" class="table-modal"><div class="table-wrap"><div id="DSU047TB2" class="table-wrap-title" data-id="DSU047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SSRs identified from two libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Research items </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Total number of identified SSRs </td><td>5,158 </td><td>5,461 </td></tr><tr><td>Number of SSR-containing sequences </td><td>4,911 </td><td>5,214 </td></tr><tr><td>Number of SSRs present in compound formation </td><td>244 </td><td>246 </td></tr><tr><td>Frequency of SSRs </td><td>1/10.72 kb </td><td>1/10.44 kb </td></tr><tr><td rowspan="7">Frequency of motif size </td><td>Dinucleotide (18.96%) </td><td>Dinucleotide (18.57%) </td></tr><tr><td>Trinucleotide (17.82%) </td><td>Trinucleotide (18.49%) </td></tr><tr><td>Tetranucleotide (12.39%) </td><td>Tetranucleotide (12.54%) </td></tr><tr><td>Pentanucleotide (33.29%) </td><td>Pentanucleotide (32.72%) </td></tr><tr><td>Hexanucleotide (13.90%) </td><td>Hexanucleotide (13.94%) </td></tr><tr><td>Heptanucleotide (3.16%) </td><td>Heptanucleotide (3.19%) </td></tr><tr><td>Octanucleotide (0.48%) </td><td>Octanucleotide (0.55%) </td></tr><tr><td rowspan="18">Frequency of major motif type </td><td>AT/AT (12.62%) </td><td>AT/AT (12.43%) </td></tr><tr><td>AAAAT/ATTTT (12.04%) </td><td>AAAAT/ATTTT (11.79%) </td></tr><tr><td>AAT/ATT (7.06%) </td><td>AAT/ATT (7.34%) </td></tr><tr><td>AAAT/ATTT (6.07%) </td><td>AAAT/ATTT (6.08%) </td></tr><tr><td>AAG/CTT (5.16%) </td><td>AAAAG/CTTTT (5.44%) </td></tr><tr><td>AAAAG/CTTTT (4.94%) </td><td>AAG/CTT (5.13%) </td></tr><tr><td>AG/CT (3.78%) </td><td>AG/CT (3.77%) </td></tr><tr><td>AAATT/AATTT (3.70%) </td><td>AAATT/AATTT (3.70%) </td></tr><tr><td>AAAAAT/ATTTTT (2.77%) </td><td>AAAAAT/ATTTTT (3.19%) </td></tr><tr><td>AC/GT (2.56%) </td><td>ATC/ATG (2.44%) </td></tr><tr><td>ATC/ATG (2.27%) </td><td>AC/GT (2.36%) </td></tr><tr><td>AATAT/ATATT (1.78%) </td><td>AAAAAG/CTTTTT (1.50%) </td></tr><tr><td>AATT/AATT (1.43%) </td><td>AATAT/ATATT (1.46%) </td></tr><tr><td>ACAT/ATGT (1.40%) </td><td>AATT/AATT (1.39%) </td></tr><tr><td>AAAAAG/CTTTTT (1.40%) </td><td>ACAT/ATGT (1.37%) </td></tr><tr><td>AAAAC/GTTTT (1.28%) </td><td>AAAG/CTTT (1.28%) </td></tr><tr><td>AAAG/CTTT (1.22%) </td><td>AAAAC/GTTTT (1.26%) </td></tr><tr><td>AAC/GTT (1.07%) </td><td>AAC/GTT (0.97%) </td></tr></tbody></table></div><div content-id="DSU047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Research items </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Total number of identified SSRs </td><td>5,158 </td><td>5,461 </td></tr><tr><td>Number of SSR-containing sequences </td><td>4,911 </td><td>5,214 </td></tr><tr><td>Number of SSRs present in compound formation </td><td>244 </td><td>246 </td></tr><tr><td>Frequency of SSRs </td><td>1/10.72 kb </td><td>1/10.44 kb </td></tr><tr><td rowspan="7">Frequency of motif size </td><td>Dinucleotide (18.96%) </td><td>Dinucleotide (18.57%) </td></tr><tr><td>Trinucleotide (17.82%) </td><td>Trinucleotide (18.49%) </td></tr><tr><td>Tetranucleotide (12.39%) </td><td>Tetranucleotide (12.54%) </td></tr><tr><td>Pentanucleotide (33.29%) </td><td>Pentanucleotide (32.72%) </td></tr><tr><td>Hexanucleotide (13.90%) </td><td>Hexanucleotide (13.94%) </td></tr><tr><td>Heptanucleotide (3.16%) </td><td>Heptanucleotide (3.19%) </td></tr><tr><td>Octanucleotide (0.48%) </td><td>Octanucleotide (0.55%) </td></tr><tr><td rowspan="18">Frequency of major motif type </td><td>AT/AT (12.62%) </td><td>AT/AT (12.43%) </td></tr><tr><td>AAAAT/ATTTT (12.04%) </td><td>AAAAT/ATTTT (11.79%) </td></tr><tr><td>AAT/ATT (7.06%) </td><td>AAT/ATT (7.34%) </td></tr><tr><td>AAAT/ATTT (6.07%) </td><td>AAAT/ATTT (6.08%) </td></tr><tr><td>AAG/CTT (5.16%) </td><td>AAAAG/CTTTT (5.44%) </td></tr><tr><td>AAAAG/CTTTT (4.94%) </td><td>AAG/CTT (5.13%) </td></tr><tr><td>AG/CT (3.78%) </td><td>AG/CT (3.77%) </td></tr><tr><td>AAATT/AATTT (3.70%) </td><td>AAATT/AATTT (3.70%) </td></tr><tr><td>AAAAAT/ATTTTT (2.77%) </td><td>AAAAAT/ATTTTT (3.19%) </td></tr><tr><td>AC/GT (2.56%) </td><td>ATC/ATG (2.44%) </td></tr><tr><td>ATC/ATG (2.27%) </td><td>AC/GT (2.36%) </td></tr><tr><td>AATAT/ATATT (1.78%) </td><td>AAAAAG/CTTTTT (1.50%) </td></tr><tr><td>AATT/AATT (1.43%) </td><td>AATAT/ATATT (1.46%) </td></tr><tr><td>ACAT/ATGT (1.40%) </td><td>AATT/AATT (1.39%) </td></tr><tr><td>AAAAAG/CTTTTT (1.40%) </td><td>ACAT/ATGT (1.37%) </td></tr><tr><td>AAAAC/GTTTT (1.28%) </td><td>AAAG/CTTT (1.28%) </td></tr><tr><td>AAAG/CTTT (1.22%) </td><td>AAAAC/GTTTT (1.26%) </td></tr><tr><td>AAC/GTT (1.07%) </td><td>AAC/GTT (0.97%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4593594" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU047TB2" class="table-wrap-title" data-id="DSU047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SSRs identified from two libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Research items </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Total number of identified SSRs </td><td>5,158 </td><td>5,461 </td></tr><tr><td>Number of SSR-containing sequences </td><td>4,911 </td><td>5,214 </td></tr><tr><td>Number of SSRs present in compound formation </td><td>244 </td><td>246 </td></tr><tr><td>Frequency of SSRs </td><td>1/10.72 kb </td><td>1/10.44 kb </td></tr><tr><td rowspan="7">Frequency of motif size </td><td>Dinucleotide (18.96%) </td><td>Dinucleotide (18.57%) </td></tr><tr><td>Trinucleotide (17.82%) </td><td>Trinucleotide (18.49%) </td></tr><tr><td>Tetranucleotide (12.39%) </td><td>Tetranucleotide (12.54%) </td></tr><tr><td>Pentanucleotide (33.29%) </td><td>Pentanucleotide (32.72%) </td></tr><tr><td>Hexanucleotide (13.90%) </td><td>Hexanucleotide (13.94%) </td></tr><tr><td>Heptanucleotide (3.16%) </td><td>Heptanucleotide (3.19%) </td></tr><tr><td>Octanucleotide (0.48%) </td><td>Octanucleotide (0.55%) </td></tr><tr><td rowspan="18">Frequency of major motif type </td><td>AT/AT (12.62%) </td><td>AT/AT (12.43%) </td></tr><tr><td>AAAAT/ATTTT (12.04%) </td><td>AAAAT/ATTTT (11.79%) </td></tr><tr><td>AAT/ATT (7.06%) </td><td>AAT/ATT (7.34%) </td></tr><tr><td>AAAT/ATTT (6.07%) </td><td>AAAT/ATTT (6.08%) </td></tr><tr><td>AAG/CTT (5.16%) </td><td>AAAAG/CTTTT (5.44%) </td></tr><tr><td>AAAAG/CTTTT (4.94%) </td><td>AAG/CTT (5.13%) </td></tr><tr><td>AG/CT (3.78%) </td><td>AG/CT (3.77%) </td></tr><tr><td>AAATT/AATTT (3.70%) </td><td>AAATT/AATTT (3.70%) </td></tr><tr><td>AAAAAT/ATTTTT (2.77%) </td><td>AAAAAT/ATTTTT (3.19%) </td></tr><tr><td>AC/GT (2.56%) </td><td>ATC/ATG (2.44%) </td></tr><tr><td>ATC/ATG (2.27%) </td><td>AC/GT (2.36%) </td></tr><tr><td>AATAT/ATATT (1.78%) </td><td>AAAAAG/CTTTTT (1.50%) </td></tr><tr><td>AATT/AATT (1.43%) </td><td>AATAT/ATATT (1.46%) </td></tr><tr><td>ACAT/ATGT (1.40%) </td><td>AATT/AATT (1.39%) </td></tr><tr><td>AAAAAG/CTTTTT (1.40%) </td><td>ACAT/ATGT (1.37%) </td></tr><tr><td>AAAAC/GTTTT (1.28%) </td><td>AAAG/CTTT (1.28%) </td></tr><tr><td>AAAG/CTTT (1.22%) </td><td>AAAAC/GTTTT (1.26%) </td></tr><tr><td>AAC/GTT (1.07%) </td><td>AAC/GTT (0.97%) </td></tr></tbody></table></div><div content-id="DSU047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Research items </th><th>DH962 </th><th>Jimian5 </th></tr></thead><tbody><tr><td>Total number of identified SSRs </td><td>5,158 </td><td>5,461 </td></tr><tr><td>Number of SSR-containing sequences </td><td>4,911 </td><td>5,214 </td></tr><tr><td>Number of SSRs present in compound formation </td><td>244 </td><td>246 </td></tr><tr><td>Frequency of SSRs </td><td>1/10.72 kb </td><td>1/10.44 kb </td></tr><tr><td rowspan="7">Frequency of motif size </td><td>Dinucleotide (18.96%) </td><td>Dinucleotide (18.57%) </td></tr><tr><td>Trinucleotide (17.82%) </td><td>Trinucleotide (18.49%) </td></tr><tr><td>Tetranucleotide (12.39%) </td><td>Tetranucleotide (12.54%) </td></tr><tr><td>Pentanucleotide (33.29%) </td><td>Pentanucleotide (32.72%) </td></tr><tr><td>Hexanucleotide (13.90%) </td><td>Hexanucleotide (13.94%) </td></tr><tr><td>Heptanucleotide (3.16%) </td><td>Heptanucleotide (3.19%) </td></tr><tr><td>Octanucleotide (0.48%) </td><td>Octanucleotide (0.55%) </td></tr><tr><td rowspan="18">Frequency of major motif type </td><td>AT/AT (12.62%) </td><td>AT/AT (12.43%) </td></tr><tr><td>AAAAT/ATTTT (12.04%) </td><td>AAAAT/ATTTT (11.79%) </td></tr><tr><td>AAT/ATT (7.06%) </td><td>AAT/ATT (7.34%) </td></tr><tr><td>AAAT/ATTT (6.07%) </td><td>AAAT/ATTT (6.08%) </td></tr><tr><td>AAG/CTT (5.16%) </td><td>AAAAG/CTTTT (5.44%) </td></tr><tr><td>AAAAG/CTTTT (4.94%) </td><td>AAG/CTT (5.13%) </td></tr><tr><td>AG/CT (3.78%) </td><td>AG/CT (3.77%) </td></tr><tr><td>AAATT/AATTT (3.70%) </td><td>AAATT/AATTT (3.70%) </td></tr><tr><td>AAAAAT/ATTTTT (2.77%) </td><td>AAAAAT/ATTTTT (3.19%) </td></tr><tr><td>AC/GT (2.56%) </td><td>ATC/ATG (2.44%) </td></tr><tr><td>ATC/ATG (2.27%) </td><td>AC/GT (2.36%) </td></tr><tr><td>AATAT/ATATT (1.78%) </td><td>AAAAAG/CTTTTT (1.50%) </td></tr><tr><td>AATT/AATT (1.43%) </td><td>AATAT/ATATT (1.46%) </td></tr><tr><td>ACAT/ATGT (1.40%) </td><td>AATT/AATT (1.39%) </td></tr><tr><td>AAAAAG/CTTTTT (1.40%) </td><td>ACAT/ATGT (1.37%) </td></tr><tr><td>AAAAC/GTTTT (1.28%) </td><td>AAAG/CTTT (1.28%) </td></tr><tr><td>AAAG/CTTT (1.22%) </td><td>AAAAC/GTTTT (1.26%) </td></tr><tr><td>AAC/GTT (1.07%) </td><td>AAC/GTT (0.97%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4593594" target="_blank">View Large</a></div></div><p>After redundancies and duplications were removed, 1,399 non-redundant SSRs remained in both DH962 and Jimian5. These were located on the same location or homologous location. Among the 1,399 SSRs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2a</a>), there were 67 SSRs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2b</a>) that were different in the number of motifs between DH962 and Jimian5, which were <em>in silico</em> polymorphic. A comparison of the SSR sequences of DH962 and Jimian5 showed 598 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2c</a>) non-redundant SSRs in DH962 only and 658 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2d</a>) non-redundant SSR in Jimian5 only. Finally, 1,323 SSR primer pairs (67 + 598 + 658) were used to screen for polymorphisms (Fig. <a reveal-id="DSU047F1" data-open="DSU047F1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h3 scrollto-destination=4593596 id="4593596" class="section-title" data-legacy-id=s3d>3.4. Characterization and identification of InDels and SNPs in the two RAD libraries</h3>
<p>InDel calling showed there to be 3,558 InDels after mapping DH962 reads to Jimian5 assembled contigs (D/J-I) and 3,240 InDels after mapping Jimian5 reads to DH962 assembled contigs (J/D-I). The called results that conformed to any one of the conditions {MQ0 ≥ 4 [[MQ0/(1.0 * DP)] &gt; 0.1]; MQ &lt; 30.0; QUAL &lt; 50; DP &lt; 5} were removed. After filtering, the mapping of D/J-I included 3,119 InDels, and the mapping of J/D-I included 2,869 InDels. The frequency of InDels was 1/18.46 kb. Among the 3,119 InDels of D/J-I, 1,355 were insertions and 1,764 were deletions, with the average length of the InDels being 1.4 bp (Fig. <a reveal-id="DSU047F6" data-open="DSU047F6" class="link link-reveal link-table xref-fig">6</a>a). Among the 2,869 InDels of J/D-I, 1,301 were insertions, and 1,568 were deletions. The average length of the InDels was 1.3 bp (Fig. <a reveal-id="DSU047F6" data-open="DSU047F6" class="link link-reveal link-table xref-fig">6</a>a). After BLAST of the flanking sequences of InDels, 1,686 InDels were found to appear only in D/J-I (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3a</a>), and 1,477 InDels appeared only in J/D-I (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3b</a>), with 675 InDels appearing in both (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3c</a>). Among the 675 sites, there were 121 sites between DH962 and Jimian5 that were homozygous and polymorphic (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3d</a>). A total of 3,838 InDel primer pairs were obtained (Fig. <a reveal-id="DSU047F1" data-open="DSU047F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4593598" scrollto-destination="4593598"></a>
<div id="DSU047F6" data-id="DSU047F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04706.png?Expires=2147483647&Signature=rD3Fk6bd5h8hFaya0XDmO1PmvCnHT~xam0MOFhLzh-IxiRucXy5oBNtPxUVwVgmd0-7z52TtgjhGk0Ld8UTKyRWI6Bx2cOtfiZh0eRqHFXBYTbKd6WDLi~35TicdpE0wVtZUxU9YE0x9qp6C6gnUGbGYOVeYgdtGrveI55GsfFXu996yxWEKLNCvZ0qvp8i-Po3gB5w1FoOqRxzQC0rDsGCvf~3koKH8~kUp9x8la9KFhB69Sqf1FSY0e0818gjQXZTgKMvFStDLymQ2GpgLLn5qXS~-u9Zh~16JHK6Goz8nmmi~PYN0mJrZOsNz8iOj8P68zLd7wjA~hHGtR5dRgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics and distribution of InDels and SNPs in two RAD libraries. (a) Distribution of insertions (+) and deletions (−). (b) Transitions and transversions occurring within the mined SNPs." path-from-xml="dsu04706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593598/dsu04706.tiff" path-from-xml="dsu04706.tiff" target="_blank">View large</a><a section="4593598" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04706.png?Expires=2147483647&Signature=ewVS1f~v3M4S8U765KBfyC-fousV27lBDAcSiZn66aa-~DN2h8CP75hgrMknDVGcIr7q2Rx9thvrJFDX7C5tGcVsi8caQoEaolRArrdXGvSDRx2ccWTys9rfz~KpCrh9owLVyuV1qif6gIUij6hFjF1GZl9ppCqcjoUHQiNrIhjQvBqJgwas3ARBHNfs6JUxeChg9jh6tcsQNCIr~xLObSrd9HdmjAgWScvOUtiNNX5TDxO~WD-0snNpH11CpHen9KgUoPQBg48ew3zUlMa1DiFt3~139tNtfoUn4wvRX5xkYRYthd5bUFoyillxHd1xknzTxj5G5XQfiOkmXKBwmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593598&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics and distribution of InDels and SNPs in two RAD libraries. (a) Distribution of insertions (+) and deletions (−). (b) Transitions and transversions occurring within the mined SNPs.</p></div></div><div content-id="DSU047F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04706.png?Expires=2147483647&Signature=rD3Fk6bd5h8hFaya0XDmO1PmvCnHT~xam0MOFhLzh-IxiRucXy5oBNtPxUVwVgmd0-7z52TtgjhGk0Ld8UTKyRWI6Bx2cOtfiZh0eRqHFXBYTbKd6WDLi~35TicdpE0wVtZUxU9YE0x9qp6C6gnUGbGYOVeYgdtGrveI55GsfFXu996yxWEKLNCvZ0qvp8i-Po3gB5w1FoOqRxzQC0rDsGCvf~3koKH8~kUp9x8la9KFhB69Sqf1FSY0e0818gjQXZTgKMvFStDLymQ2GpgLLn5qXS~-u9Zh~16JHK6Goz8nmmi~PYN0mJrZOsNz8iOj8P68zLd7wjA~hHGtR5dRgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics and distribution of InDels and SNPs in two RAD libraries. (a) Distribution of insertions (+) and deletions (−). (b) Transitions and transversions occurring within the mined SNPs." path-from-xml="dsu04706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593598/dsu04706.tiff" path-from-xml="dsu04706.tiff" target="_blank">View large</a><a section="4593598" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04706.png?Expires=2147483647&Signature=ewVS1f~v3M4S8U765KBfyC-fousV27lBDAcSiZn66aa-~DN2h8CP75hgrMknDVGcIr7q2Rx9thvrJFDX7C5tGcVsi8caQoEaolRArrdXGvSDRx2ccWTys9rfz~KpCrh9owLVyuV1qif6gIUij6hFjF1GZl9ppCqcjoUHQiNrIhjQvBqJgwas3ARBHNfs6JUxeChg9jh6tcsQNCIr~xLObSrd9HdmjAgWScvOUtiNNX5TDxO~WD-0snNpH11CpHen9KgUoPQBg48ew3zUlMa1DiFt3~139tNtfoUn4wvRX5xkYRYthd5bUFoyillxHd1xknzTxj5G5XQfiOkmXKBwmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593598&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics and distribution of InDels and SNPs in two RAD libraries. (a) Distribution of insertions (+) and deletions (−). (b) Transitions and transversions occurring within the mined SNPs.</p></div></div><p>SNP calling identified 26,757 SNPs in D/J-S and 25,682 SNPs in J/D-S. When screened with a coverage depth ≥ 8, 9,048 SNPs remained in D/J-S, and 8,090 SNPs remained in J/D-S. The frequency of SNPs was 1/6.55 Kb, and the transition/transversion ratio of all the SNPs was 1.76. Of the 9,048 SNPs in D/J-S, the observed SNP transition/transversion ratio was 1.71 (Fig. <a reveal-id="DSU047F6" data-open="DSU047F6" class="link link-reveal link-table xref-fig">6</a>b), and most SNPs were identified between 50 and 350 bp of the start of each contig. Of the 8,090 SNPs in J/D-S, the observed SNP transition/transversion ratio was 1.82, and most SNPs were identified between 50 and 300 bp from the start of each contig.</p><p>After BLAST the flanking sequences of SNPs, 4,314 SNPs were found to appear only in D/J-S (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4a</a>), and 3,836 SNPs appeared only in J/D-S (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4b</a>), with 1,216 SNPs being found in both (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4c</a>). Of the 1,216 sites, there were 441 sites that were homozygous and polymorphic between DH962 and Jimian5 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4d</a>). Finally, a total of 9,366 SNP primer pairs were obtained (Fig. <a reveal-id="DSU047F1" data-open="DSU047F1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h3 scrollto-destination=4593601 id="4593601" class="section-title" data-legacy-id=s3e>3.5. <em>In silico</em> mapping of the marker contigs</h3>
<p>A total of 14,433 contigs containing markers were used to analyse the distribution of the markers on the A<sub>2</sub> and D<sub>5</sub> genomes. We used all of the 14,433 contigs to BLAST with the sequences of the A<sub>2</sub> and D<sub>5</sub> genomes, and 14,103 (97.71%) contigs were matched (Fig. <a reveal-id="DSU047F7" data-open="DSU047F7" class="link link-reveal link-table xref-fig">7</a>). Of the 14,103 contigs, 6,995 were matched on the 13 chromosomes of the A<sub>2</sub> genome, and 7,108 were matched on the 13 chromosomes of the D<sub>5</sub> genome. The hits were found in an essentially uniform distribution on every chromosome, except there were slightly fewer on the Ga2 and Ga5 of the A<sub>2</sub> genome and on Gr3 and Gr12 of the D<sub>5</sub> genome. </p>                        <a id="4593603" scrollto-destination="4593603"></a>
<div id="DSU047F7" data-id="DSU047F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04707.png?Expires=2147483647&Signature=4AIe-gUl4trbmOaTvmC9hbyrIOmli5g9NCvksv5LqUYMUpKQ837fMHub4jVa4cQiqAOZKsysX8RJbaxby4LP24FMcWqb1YXopGpgI708RNitETyDAzStA2-oxl89Qz16w6rhsGLoLV2Pkm8iZ~RexthNxmIrGMtTzPki~4Agn6aWeHEolukHOnHWu~O8eQwx0P~dJgPBWx8GXdG8t8QSm6EEJ-8Dkv4DDZ-c7I6S7UeviyjKIQRaWniAh3qGRz2fylA4enPoaFkKkVBkoVrNmq0xfyTDUq9-bVE8hecn8Wojofg0Su3tzHPnPPgbMmjWuO9qe52jq1Zkgu8DbDIiIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of the 14,103 contigs on the A2 and D5 genomes. (a) The distribution of contigs on the A2 genome; (b) the distribution of contigs on the D5 genome." path-from-xml="dsu04707.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593603/dsu04707.tiff" path-from-xml="dsu04707.tiff" target="_blank">View large</a><a section="4593603" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04707.png?Expires=2147483647&Signature=nCfYERdLPcdcAxeKdz8vlANimKp7AgDtsm4epYPHeJKfh6ebnDScM6ANu-D2xvxO~3NMJojjN1pWGm7Eq3HA62xFalQZp5QP~X2YPgj~ixWxwP-PVSZ7p8nmqRIuAdqVsccZZinvJSW62TZ0~TzjJBSvzBGjJnHul5F-bdHytm0~F-BCGfBc~nb0GO2ZpP4p2opXncoUCXG36MRAgoxxp8kgYOGaZJxLKVhhhzDIAJOLj3sPX1tG6-zJHHzSl9lebU-fFRJTqcj8oIZ7LOVCn7O~4MINrnKtG8Q3Ewd~Gb30NhrQ0DltIZNykZtEuf8-DpyeycRX0yKRubfwr22vYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593603&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04707.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of the 14,103 contigs on the A<sub>2</sub> and D<sub>5</sub> genomes. (a) The distribution of contigs on the A<sub>2</sub> genome; (b) the distribution of contigs on the D<sub>5</sub> genome.</p></div></div><div content-id="DSU047F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04707.png?Expires=2147483647&Signature=4AIe-gUl4trbmOaTvmC9hbyrIOmli5g9NCvksv5LqUYMUpKQ837fMHub4jVa4cQiqAOZKsysX8RJbaxby4LP24FMcWqb1YXopGpgI708RNitETyDAzStA2-oxl89Qz16w6rhsGLoLV2Pkm8iZ~RexthNxmIrGMtTzPki~4Agn6aWeHEolukHOnHWu~O8eQwx0P~dJgPBWx8GXdG8t8QSm6EEJ-8Dkv4DDZ-c7I6S7UeviyjKIQRaWniAh3qGRz2fylA4enPoaFkKkVBkoVrNmq0xfyTDUq9-bVE8hecn8Wojofg0Su3tzHPnPPgbMmjWuO9qe52jq1Zkgu8DbDIiIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of the 14,103 contigs on the A2 and D5 genomes. (a) The distribution of contigs on the A2 genome; (b) the distribution of contigs on the D5 genome." path-from-xml="dsu04707.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593603/dsu04707.tiff" path-from-xml="dsu04707.tiff" target="_blank">View large</a><a section="4593603" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04707.png?Expires=2147483647&Signature=nCfYERdLPcdcAxeKdz8vlANimKp7AgDtsm4epYPHeJKfh6ebnDScM6ANu-D2xvxO~3NMJojjN1pWGm7Eq3HA62xFalQZp5QP~X2YPgj~ixWxwP-PVSZ7p8nmqRIuAdqVsccZZinvJSW62TZ0~TzjJBSvzBGjJnHul5F-bdHytm0~F-BCGfBc~nb0GO2ZpP4p2opXncoUCXG36MRAgoxxp8kgYOGaZJxLKVhhhzDIAJOLj3sPX1tG6-zJHHzSl9lebU-fFRJTqcj8oIZ7LOVCn7O~4MINrnKtG8Q3Ewd~Gb30NhrQ0DltIZNykZtEuf8-DpyeycRX0yKRubfwr22vYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593603&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04707.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of the 14,103 contigs on the A<sub>2</sub> and D<sub>5</sub> genomes. (a) The distribution of contigs on the A<sub>2</sub> genome; (b) the distribution of contigs on the D<sub>5</sub> genome.</p></div></div>                            <h3 scrollto-destination=4593604 id="4593604" class="section-title" data-legacy-id=s3f>3.6. Validation of the SSR, InDel, and SNP markers</h3>
<p>All of the 1,323 SSRs were screened for polymorphisms between DH962 and Jimian5. After being genotyped on 6% denaturing polyacrylamide gels, 1,304 (98.56%) were amplified successfully, and 66 SSRs showed polymorphism. The remaining 1,257 SSRs were subjected to SSCP analysis on 8% native polyacrylamide gels, and 17 SSRs showed polymorphisms. A total of 83 SSRs showed polymorphisms (6.27%). Among the 67 SSRs that were polymorphic between DH962 and Jimian5 by bioinformatic analysis, 12 (17.91%) showed polymorphisms.</p><p>Of the 3,838 InDels and 9,366 SNPs, 121 InDels and 441 SNPs, which were <em>in silico</em> polymorphic between two parents, were selected to detect polymorphisms between DH962 and Jimian5 by SSCP analysis. Thirty-one InDels (25.62%) were polymorphic and revealed 33 polymorphic loci, and 49 SNPs (11.11%) were polymorphic.</p><p>The polymorphisms of the 1,323 SSRs, 121 InDels, and 441 SNPs between <em>G. hirsutum</em> cv. Emian22 and <em>G. barbadense</em> acc. 3–79, the mapping parents of our interspecific population,<sup><a reveal-id="DSU047C41" data-open="DSU047C41" class="link link-ref link-reveal xref-bibr">41</a></sup> were also screened. A total of 610 SSRs (46.11%) (437 were genotyped on 6% denaturing polyacrylamide gels, and 173 were analysed by SSCP), 46 InDels (38.02%), and 127 SNPs (28.80%) were found to be polymorphic.</p>                            <h3 scrollto-destination=4593608 id="4593608" class="section-title" data-legacy-id=s3g>3.7. Genetic linkage map construction</h3>
<p>A total of 3,479 HAU, 699 NAU, and 700 Gh SSR primer pairs obtained from previous studies<sup><a reveal-id="DSU047C28  DSU047C34" data-open="DSU047C28  DSU047C34" class="link link-ref link-reveal xref-bibr">28–34</a></sup> were screened for polymorphism in DH962 and Jimian5, and 173 (3.55%) showed polymorphisms, revealing 178 polymorphic loci. Additionally, 1,869 markers from the current interspecific BC<sub>1</sub> genetic maps<sup><a reveal-id="DSU047C27" data-open="DSU047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and Wang <em>et al.</em><sup><a reveal-id="DSU047C26" data-open="DSU047C26" class="link link-ref link-reveal xref-bibr">26</a></sup> were screened for polymorphism in DH962 and Jimian5, and 187 showed polymorphisms, revealing 192 polymorphic loci. Adding the 165 polymorphic loci obtained from this study and the previously published 506 loci,<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> a total of 1,041 loci were used for linkage analysis. Finally, 1,013 loci were mapped on 50 linkage groups with 41 linkage groups assigned to 23 chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The total length of the linkage map was 3,004.71 cM, with a mean distance of 2.97 cM between adjacent markers.</p><p>Among the 535 loci obtained in this study, 99 loci deviated from an expected 1 : 2 : 1 or 3 : 1 segregation ratio (<em>P</em> ≤ 0.05), and 8 loci were found to not be mapped. Seven segregation distortion regions were found on four chromosomes, with many segregation distortion loci found on LG1/Chr9 or 23, LG5/Chr7, and LG6/Chr25.</p>                            <h3 scrollto-destination=4593611 id="4593611" class="section-title" data-legacy-id=s3h>3.8. QTLs for yield components and fibre quality traits</h3>
<p>For yield components, 21 QTLs were detected and mapped on seven chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), and explained from 9.80 to 16.62% of the phenotypic variation (PV), with LOD scores ranging from 4.22 to 7.83 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). There were 2 QTLs for BN, 2 for SCW, 6 for LW, 5 for SI, 2 for LP, 4 for LI, and 18 were newly found QTLs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p><p>A total of 12 QTLs for fibre quality were distributed on six chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), and explained 7.59–37.09% of the PV, with LOD scores ranging from 4.04 to 9.57 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Of the 12 QTLs, 3 QTLs were for FL, 2 QTLs for FS, 2 QTLs for FE, 5 QTLs for MV, and 9 QTLs were novel (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p>                            <h3 scrollto-destination=4593614 id="4593614" class="section-title" data-legacy-id=s3i>3.9. Homology analysis between upland cotton and the A<sub>2</sub> and D<sub>5</sub> genomes</h3>
<p>The sequences of 562 markers (the sequences of SRAPs were not available) mapped on chromosomes of the upland cotton genetic map in this study were used to BLASTN with the diploid A<sub>2</sub> genome of <em>G. arboretum</em> and the D<sub>5</sub> genome of <em>G. raimondii</em>, using an <em>E</em>-value cut-off of 1e−10. After analysis, the homology and collinearity between the A<sub>T</sub> genome and the A<sub>2</sub> genome were high, except for Chr2 and Ga2, Chr5 and Ga10, and Chr10 and Ga9 (Fig. <a reveal-id="DSU047F8" data-open="DSU047F8" class="link link-reveal link-table xref-fig">8</a>a; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6a</a>). In the current study, four unassembled scaffolds of the A<sub>2</sub> genome were anchored by our map. HAU-DJ-S078 matched on scaffold7300, NAU2687 on scaffold3678, HAU-DJ-S168 on scaffold1365, and NBRI_HQ527767 on scaffold4507. Results showed that the homology and collinearity between the D<sub>T</sub> genome and the D<sub>5</sub> genome were high on every chromosome (Fig. <a reveal-id="DSU047F8" data-open="DSU047F8" class="link link-reveal link-table xref-fig">8</a>b; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6b</a>). </p>                        <a id="4593616" scrollto-destination="4593616"></a>
<div id="DSU047F8" data-id="DSU047F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04708.png?Expires=2147483647&Signature=WThsK-KAdx7rYJsUnVxcuUAzqsD0f7Fv6moDGpcqiGxUpucuttTb5I2r1UYO37HsfUm5w1DxSsH4R2wwXLVJTVRBsROfjj0~gJCwt1HOp09bG5bL~WGQEcN-AgRD-PxNYim2IeCT1z4dZQ-rPyIsDSX-ZnC6ZZiX7OBsrQw-wYorFotwWBPP1zWL-v5nAaO6w6QM9RKMq~WbkvBiyneBd-uZZ5mJCRsVcGS9BZrg3nJupMMLafB47Z2LttIxsZpMdYWyopQWpdZnHvhwDze6ORPRlWaw8ozky3KsQngjc7PAnLyngIVQ~FRDGX-ecTNwD0e9JtLex41LpCmW90Z2Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of 562 markers on the A2 and D5 genomes. (a) Analysis of homology between the AT and A2 genomes; (b) analysis of homology between the DT and D5 genomes." path-from-xml="dsu04708.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593616/dsu04708.tiff" path-from-xml="dsu04708.tiff" target="_blank">View large</a><a section="4593616" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04708.png?Expires=2147483647&Signature=T~pvTmFBAJewRerm3DUQ9NXyZfE7csICdBqUKPLIUha7g3rQ3tKiVvm7SR8Yrvjix-wYKwScAK0BknsnlVwm3yEny1M7eb3swdB1dO5WA4LDzgIkzJffQd0OL097af4exHBkYwP~XShUJF65fTelkIdWkXjK5eF8nBDrrLCArjiv8spZsECLZ7wHJ8KjN4mDKTpPCub-0Jb8HLQOc8RhESy-zPQ2EYsN~44kzOQvYfQkBR6dg21IetY3kvAdcf~~uWaczx6cPujBZVbHIdanhW~V8mweaol8rgzDX3mlqhM9SqOh8HsgHJAURA7rmHeCwLX5Vc8KkcqiHMka73RcWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593616&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04708.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of 562 markers on the A<sub>2</sub> and D<sub>5</sub> genomes. (a) Analysis of homology between the A<sub>T</sub> and A<sub>2</sub> genomes; (b) analysis of homology between the D<sub>T</sub> and D<sub>5</sub> genomes.</p></div></div><div content-id="DSU047F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/m_dsu04708.png?Expires=2147483647&Signature=WThsK-KAdx7rYJsUnVxcuUAzqsD0f7Fv6moDGpcqiGxUpucuttTb5I2r1UYO37HsfUm5w1DxSsH4R2wwXLVJTVRBsROfjj0~gJCwt1HOp09bG5bL~WGQEcN-AgRD-PxNYim2IeCT1z4dZQ-rPyIsDSX-ZnC6ZZiX7OBsrQw-wYorFotwWBPP1zWL-v5nAaO6w6QM9RKMq~WbkvBiyneBd-uZZ5mJCRsVcGS9BZrg3nJupMMLafB47Z2LttIxsZpMdYWyopQWpdZnHvhwDze6ORPRlWaw8ozky3KsQngjc7PAnLyngIVQ~FRDGX-ecTNwD0e9JtLex41LpCmW90Z2Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of 562 markers on the A2 and D5 genomes. (a) Analysis of homology between the AT and A2 genomes; (b) analysis of homology between the DT and D5 genomes." path-from-xml="dsu04708.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4593616/dsu04708.tiff" path-from-xml="dsu04708.tiff" target="_blank">View large</a><a section="4593616" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu04708.png?Expires=2147483647&Signature=T~pvTmFBAJewRerm3DUQ9NXyZfE7csICdBqUKPLIUha7g3rQ3tKiVvm7SR8Yrvjix-wYKwScAK0BknsnlVwm3yEny1M7eb3swdB1dO5WA4LDzgIkzJffQd0OL097af4exHBkYwP~XShUJF65fTelkIdWkXjK5eF8nBDrrLCArjiv8spZsECLZ7wHJ8KjN4mDKTpPCub-0Jb8HLQOc8RhESy-zPQ2EYsN~44kzOQvYfQkBR6dg21IetY3kvAdcf~~uWaczx6cPujBZVbHIdanhW~V8mweaol8rgzDX3mlqhM9SqOh8HsgHJAURA7rmHeCwLX5Vc8KkcqiHMka73RcWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4593616&ar=332826&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04708.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of 562 markers on the A<sub>2</sub> and D<sub>5</sub> genomes. (a) Analysis of homology between the A<sub>T</sub> and A<sub>2</sub> genomes; (b) analysis of homology between the D<sub>T</sub> and D<sub>5</sub> genomes.</p></div></div>                            <h2 scrollto-destination=4593617 id="4593617" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Until this study, there was no reference genome available for the allotetraploid cotton. The lack of genome sequences resulted in slow genome research progress in allotetraploid cotton, especially <em>G. hirsutum</em>. As the most important cultivated cotton, genome research of <em>G. hirsutum</em> is particularly urgent. In this study, the assembly produced 251,816 contigs, an amount that was too high, indicating that <em>Eco</em>RI restriction sites were abundant on the genome of <em>G. hirsutum</em>. There were more contigs for <em>G. hirsutum</em> than for other plants in an assembly analysis of PE RAD-seq studies.<sup><a reveal-id="DSU047C17" data-open="DSU047C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU047C38" data-open="DSU047C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSU047C42" data-open="DSU047C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The sequencing depth, the assembly method, the high ploidy, and the heterozygosity of <em>G. hirsutum</em>, the large genome, and the high homology between the A<sub>T</sub> genome and the D<sub>T</sub> genome may all have contributed to larger numbers of assembled contigs.</p><p>There are many challenges for RAD-seq read assembly. Repetitive sequences are nearly indistinguishable in the context of a short sequence read. Short overlaps are easily pieced together in whole-genome assembly, even though short reads have a high error rate.<sup><a reveal-id="DSU047C43" data-open="DSU047C43" class="link link-ref link-reveal xref-bibr">43</a></sup> When assembling the reads with Velvet,<sup><a reveal-id="DSU047C19" data-open="DSU047C19" class="link link-ref link-reveal xref-bibr">19</a></sup> the user should choose parameters such as hash length and expected coverage. During assembly, a median value for each parameter is chosen, and the assembly therefore is less optimal in those regions that differ from that median. Velvet can also be reconfigured to use longer hash length than the default maximum of 31, but this requires hundreds of gigabytes of memory, which is a problem for whole-genome shotgun assembly of complex genomes such as allotetraploid cotton.</p><p>In this study, 10.33 Gb of clean reads of <em>G. hirsutum</em> were obtained from a pair of mapping parents using RAD-seq, and the GC contents of the two RAD libraries for DH962 and Jimian5 were 34.00 and 34.17%, respectively. The data sizes from the two materials were similar, and GC content was consistent with results from PE RAD-seq studies in other plant genomes.<sup><a reveal-id="DSU047C17" data-open="DSU047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> After identifying the sequences in common between the mapping parents using the CAP3 program<sup><a reveal-id="DSU047C20" data-open="DSU047C20" class="link link-ref link-reveal xref-bibr">20</a></sup> (using the parameters of an overlap length cut-off of 80 bp and an overlap percent cut-off of 95), a Gh-D-J data set of ≈85.76 Mb was obtained. This indicated that the sequences from two genotypes represented ∼3.43% of the tetraploid cotton genome. So far, comparative sequencing of such a large part of the <em>G. hirsutum</em> genome has not been performed in two genotypes. These genome sequences will provide significant information for the molecular breeding project and genome research of <em>G. hirsutum</em>.</p><p>Fibre development has always been a major focus for cotton genome studies and breeders. In this study, 58 contigs were associated with the MYB family, 10 contigs with the expansin family, and 27 contigs with the ethylene response. The MYB family, expansin family, and ethylene biosynthesis are highly related to the initiation and elongation of cotton fibre cells.<sup><a reveal-id="DSU047C44" data-open="DSU047C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSU047C45" data-open="DSU047C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Another 17 contigs were found to be related to cellulose synthase (<em>CESA</em>) genes and 32 to cellulose synthase-like (<em>CSL</em>) genes. In the study by Yoo and Wendel,<sup><a reveal-id="DSU047C46" data-open="DSU047C46" class="link link-ref link-reveal xref-bibr">46</a></sup><em>CESA</em> and <em>CSL</em> genes were shown to be expressed specifically during primary and secondary cell wall biosynthesis of <em>G. hirsutum</em>. The genes found in the current study would be very useful for research of cotton fibre cell development. We also found 46 contigs that were related to NBS domains. In studies of the <em>G. arboretum</em> (A<sub>2</sub>) genome and the <em>G. raimondii</em> (D<sub>5</sub>) genome,<sup><a reveal-id="DSU047C47" data-open="DSU047C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSU047C48" data-open="DSU047C48" class="link link-ref link-reveal xref-bibr">48</a></sup> NBS domains may be related with a Verticillium wilt resistance gene, so the discovery of NBS domains in the <em>G. hirsutum</em> genome is significant for research into Verticillium wilt resistance of <em>G. hirsutum</em>.</p><p>The two plant materials, DH962 and Jimian5, had quite different characteristics in yield and fibre quality traits,<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> which were suitable for mapping population establishment. To develop genetic markers efficiently, the genomes of the two parents were sequenced. The number of hits was slightly less on Ga2 and Ga5 of the A<sub>2</sub> genome (two shorter chromosomes)<sup><a reveal-id="DSU047C48" data-open="DSU047C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and on Gr3 and Gr12 of the D<sub>5</sub> genome than on other chromosomes (two shorter chromosomes)<sup><a reveal-id="DSU047C47" data-open="DSU047C47" class="link link-ref link-reveal xref-bibr">47</a></sup> (Fig. <a reveal-id="DSU047F6" data-open="DSU047F6" class="link link-reveal link-table xref-fig">6</a>). The markers obtained here were uniformly distributed on every chromosome, and the quality of data from the RAD-seq was high and representative. The conditions were suitable for effective marker development.</p><p>Comparing the genome sequences of the two parents, a total of 17,138 SNPs were found in the two RAD libraries, and the transition/transversion ratio of all of the SNPs was 1.76. This result was consistent with other studies using RAD-seq.<sup><a reveal-id="DSU047C17" data-open="DSU047C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU047C38" data-open="DSU047C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Byers <em>et al.</em><sup><a reveal-id="DSU047C11" data-open="DSU047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> found 11,834 SNPs between two <em>G. hirsutum</em> varieties (Acala Maxxa and TX2094) from the GR-RSC libraries using the Roche 454 pyrosequencing platform. In the study by Srivastava <em>et al.</em>,<sup><a reveal-id="DSU047C49" data-open="DSU047C49" class="link link-ref link-reveal xref-bibr">49</a></sup> a total of 1,440 expressed SSRs and 2,608 SNPs were identified from the transcriptome of two genotypes of <em>G. hirsutum</em> using the Roche 454 pyrosequencing platform. Because of the low levels of genetic diversity in upland cotton, there were very few polymorphisms between intraspecific strains of upland cotton. Analysis of upland cotton data involved some challenges: (i) upland cotton is an allotetraploid, which leads to the existence of lots of homologous sequences, (ii) a big complex genome structure (≈2.5 Gb), with too many repeat sequences, and (iii) lack of a reference genome. Here, 9,366 SNPs and 3,838 InDels suitable for further analysis were observed. Another 1,323 SSRs were also identified. All of the 14,527 markers were used for genome research, construction of a genetic map, and to explain the origin and evolution of cotton further. In this research, the number of detected SNPs was more than that found by Byers <em>et al.</em><sup><a reveal-id="DSU047C11" data-open="DSU047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and Srivastava <em>et al</em><em>.</em><sup><a reveal-id="DSU047C49" data-open="DSU047C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Until now, research related to SNP identification and genome structure in cotton has been far more superficial than in other major crops. Next-generation RAD sequencing was used here to detect SNPs in <em>G. hirsutum</em>. Results demonstrated that RAD-seq is an effective and economic method for SNP detection in <em>G. hirsutum</em>.</p><p>After validation by the parents, DH962 and Jimian5, the polymorphism rate of SSRs was found to be 6.27%. In previous studies,<sup><a reveal-id="DSU047C3" data-open="DSU047C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU047C6" data-open="DSU047C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU047C50" data-open="DSU047C50" class="link link-ref link-reveal xref-bibr">50</a></sup> polymorphism rates of SSRs were ∼2.5–3.5%, and the rate of polymorphism markers from previous studies<sup><a reveal-id="DSU047C28  DSU047C34" data-open="DSU047C28  DSU047C34" class="link link-ref link-reveal xref-bibr">28–34</a></sup> was 3.55% in the current study. In this study, markers were developed based on parental RAD-seq methodology, and the rate of polymorphism of SSRs was double that of previous studies. The rate of polymorphism of InDels and SNPs developed using the same method was 25.62 and 11.11%, respectively. These results indicated that this method of analysis could effectively and cost-effectively improve the detection of polymorphisms, and that these polymorphisms could be used as references for later marker development. In this study, 2× sequencing depth and 2 genotypes were used to develop markers. As the amount of sequencing material and depth increased, more data and more markers became available. At the same time, the limitations of bioinformatic tools and of genotyping technology can influence these results. To confirm universality and compatibility, all of the primer pairs were also screened between <em>G. hirsutum</em> cv. Emian22 and <em>G. barbadense</em> acc. 3-79, and revealed additional polymorphisms. The current approach was found to be effective with SSRs, InDels, and SNPs, and the polymorphic markers could be used to enrich the genetic map for further analysis.<sup><a reveal-id="DSU047C41" data-open="DSU047C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The calculated polymorphism rate in this study indicated that the application of InDels and SNPs could be more effective than SSRs in intraspecific population studies of upland cotton, and the use of markers found to be more effective in interspecific populations than in intraspecific populations. Owing to the narrow genetic base in upland cotton, the differences among upland cotton genome sequences may be mainly single-base deletions, insertions, and mutations. This phenomenon led to high polymorphism, so larger scale development of InDels and SNPs will be very informative in studying the molecular and quantitative genetics of cotton.</p><p>In this study, 535 polymorphic loci were identified, and a genetic map of upland cotton containing 1,013 loci was constructed. This map was one of two maps composed of &gt;1,000 markers in upland cotton.<sup><a reveal-id="DSU047C5" data-open="DSU047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The more saturated map provided a better platform to provide insight into and to analyse the complex cotton genome. We used the new map for mapping QTLs for yield components, 21 QTLs were detected and mapped on seven chromosomes, and 12 QTLs related to fibre quality were distributed on six chromosomes. In a previous work by Lin <em>et al.</em>,<sup><a reveal-id="DSU047C18" data-open="DSU047C18" class="link link-ref link-reveal xref-bibr">18</a></sup> nine QTLs were detected for yield traits and five for fibre quality traits. Six QTLs were found on the adjacent area of the previous study, and another eight QTLs were detected, but the LOD value was decreased. Another 27 new QTLs were detected. The QTL <em>qFL-c10</em> was detected in the Lin <em>et al.</em><sup><a reveal-id="DSU047C18" data-open="DSU047C18" class="link link-ref link-reveal xref-bibr">18</a></sup> study and in the present study, and had a high LOD value, with &gt;35% of the PV explained. This QTL may be a major loci controlling fibre length. Eight QTLs for LW and SI were found from 47.7 to 56.88 cM on LG17/Chr8 and explained 9.80–14.09% of the PV (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). As this region may be a hotspot for LW and SI, we will structure a permanent intraspecific population to verify the stability of these QTLs. As map density increased, the efficiency of QTL detection also increased considerably. The more saturated map can facilitate whole-genome sequencing, fine mapping of QTLs, and map-based cloning, and therefore a better understanding of cotton genome structure and improvement of cotton breeding.</p><p>The current results revealed that the homology and collinearity between A<sub>T</sub> and A<sub>2</sub> and D<sub>T</sub> and D<sub>5</sub> were high on every chromosome except for Chr2 and Ga2, Chr5 and Ga10, and Chr10 and Ga9. In previous reports,<sup><a reveal-id="DSU047C41" data-open="DSU047C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSU047C51" data-open="DSU047C51" class="link link-ref link-reveal xref-bibr">51</a></sup> there was considerable homology and collinearity between the D<sub>T</sub> of the allotetraploid cotton and the D<sub>5</sub> genome. The current study is consistent with this result. In the A<sub>T</sub> genome, some exchanges occurred between Ga10 and Ga12 and this phenomenon as observed in previous studies.<sup><a reveal-id="DSU047C41" data-open="DSU047C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSU047C51" data-open="DSU047C51" class="link link-ref link-reveal xref-bibr">51</a></sup> At the same time, some fragments of Ga13 had introgressed into Gh10, and some fragments of Ga1 and Ga5 had introgressed into Gh2. This information provides an important reference for the sequence assembly of upland cotton and helps to better understand the origin and evolution of polyploidization and genomic integration studies in cotton. Because the genome data size of A<sub>2</sub> is double that of D<sub>5</sub>, more errors would occur during assembly. High-density genetic linkage maps are often used to anchor scaffolds or contigs generated by whole-genome sequencing to chromosomes.<sup><a reveal-id="DSU047C47" data-open="DSU047C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSU047C48" data-open="DSU047C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Chen <em>et al.</em><sup><a reveal-id="DSU047C52" data-open="DSU047C52" class="link link-ref link-reveal xref-bibr">52</a></sup> assigned 44 unassembled scaffolds to diploid <em>B. rapa</em> chromosomes using the high-density genetic map of the allotetraploid <em>B. napus</em>. In this study, four unassembled scaffolds were anchored by the map produced here. The high homology and collinearity on these corresponding chromosomes between the A<sub>T</sub> and A<sub>2</sub> genomes indicated that the current results were appropriate for the refinement of A<sub>2</sub> genome sequences. To prevent problems in the future, more markers are needed to enrich the genetic map to improve research into cotton genome structure.</p>                            <h2 scrollto-destination=4593627 id="4593627" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>The genome sequences of DH962 and Jimian5 developed from RAD-seq were available at NCBI Sequence Read Archive under SRA Project number SRP050345.</p>                            <h2 scrollto-destination=4593629 id="4593629" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4593631 id="4593631" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was financially supported by the National Basic Research Program (Grant No. 2011CB109303) and the Fundamental Research Funds for the Central Universities (Grant No. 2014PY015). Funding to pay the Open Access publication charges for this article was provided by the National Basic Research Program.</p>                            <h2 scrollto-destination=4593633 id="4593633" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to BGI for next-generation sequencing services.</p>                            <h2 scrollto-destination=4593635 id="4593635" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU047C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Scheffler</div> <div class="given-names">B.E.</div></div> <div class="name"><div class="surname">Dennis</div> <div class="given-names">E.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Toward sequencing cotton (<em>Gossypium</em>) genomes</div>, <div class="source">Plant Physiol.</div> , <div class="volume">145</div>, <div class="fpage">1303</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20sequencing%20cotton%20(Gossypium)%20genomes&amp;author=Z.%20Chen&amp;author=B.E.%20Scheffler&amp;author=E.%20Dennis&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=145&amp;pages=1303-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.107672" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.107672" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.107672"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18056866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fang</div> <div class="given-names">D.D.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">J.Z.</div></div></div> <div class="year">2012</div>, <div class="article-title">Addition of 455 microsatellite marker loci to the high-density <em>Gossypium hirsutum</em> TM-1×<em>G. barbadense</em> 3–79 genetic map</div>, <div class="source">Cotton Sci.</div> , <div class="volume">16</div>, <div class="fpage">229</div>–<div class="lpage">48</div>.</div></div></div></div><div content-id="DSU047C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div></div> <div class="year">2009</div>, <div class="article-title">A high-density integrative linkage map for <em>Gossypium hirsutum</em></div>, <div class="source">Euphytica</div> , <div class="volume">166</div>, <div class="fpage">35</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20integrative%20linkage%20map%20for%20Gossypium%20hirsutum&amp;author=Z.%20Lin&amp;author=Y.%20Zhang&amp;author=X.%20Zhang&amp;author=X.%20Guo&amp;publication_year=2009&amp;journal=Euphytica&amp;volume=166&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-008-9822-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-008-9822-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-008-9822-2"> </span></div> </div></div></div></div></div><div content-id="DSU047C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Genetic mapping and quantitative trait locus analysis of fiber quality traits using a three-parent composite population in upland cotton (<em>Gossypium hirsutum</em> L.)</div>, <div class="source">Mol. Breeding</div> , <div class="volume">29</div>, <div class="fpage">335</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20quantitative%20trait%20locus%20analysis%20of%20fiber%20quality%20traits%20using%20a%20three-parent%20composite%20population%20in%20upland%20cotton%20(Gossypium%20hirsutum%20L.)&amp;author=K.%20Zhang&amp;author=J.%20Zhang&amp;author=J.%20Ma&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=29&amp;pages=335-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9549-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9549-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9549-y"> </span></div> </div></div></div></div></div><div content-id="DSU047C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Teng</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhai</div> <div class="given-names">T.</div></div>et al.  </div> <div class="year">2015</div>, <div class="article-title">Construction of genetic map and QTL analysis of fiber quality traits for Upland cotton (<em>Gossypium hirsutum</em> L.)</div>, <div class="source">Euphytica</div> , <div class="volume">201</div>, <div class="fpage">195</div>–<div class="lpage">213</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20genetic%20map%20and%20QTL%20analysis%20of%20fiber%20quality%20traits%20for%20Upland%20cotton%20(Gossypium%20hirsutum%20L.)&amp;author=S.%20Tang&amp;author=Z.%20Teng&amp;author=T.%20Zhai&amp;publication_year=2015&amp;journal=Euphytica&amp;volume=201&amp;pages=195-213" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-014-1189-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-014-1189-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-014-1189-y"> </span></div> </div></div></div></div></div><div content-id="DSU047C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Guo</div> <div class="given-names">W.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in <em>Gossypium hirsutum</em> L</div>, <div class="source">Mol. Breeding</div> , <div class="volume">29</div>, <div class="fpage">297</div>–<div class="lpage">311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20mapping%20for%20yield%20and%20its%20components%20by%20using%20two%20immortalized%20populations%20of%20a%20heterotic%20hybrid%20in%20Gossypium%20hirsutum%20L&amp;author=R.%20Liu&amp;author=B.%20Wang&amp;author=W.%20Guo&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=29&amp;pages=297-311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9547-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9547-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9547-0"> </span></div> </div></div></div></div></div><div content-id="DSU047C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div></div> <div class="year">2002</div>, <div class="article-title">Applications of single nucleotide polymorphisms in crop genetics</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">5</div>, <div class="fpage">94</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20single%20nucleotide%20polymorphisms%20in%20crop%20genetics&amp;author=A.%20Rafalski&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=5&amp;pages=94-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(02)00240-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(02)00240-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(02)00240-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11856602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cortés</div> <div class="given-names">A.J.</div></div> <div class="name"><div class="surname">Chavarro</div> <div class="given-names">M.C.</div></div> <div class="name"><div class="surname">Blair</div> <div class="given-names">M.W.</div></div></div> <div class="year">2011</div>, <div class="article-title">SNP marker diversity in common bean (<em>Phaseolus vulgaris</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">123</div>, <div class="fpage">827</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20marker%20diversity%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=A.J.%20Cort%C3%A9s&amp;author=M.C.%20Chavarro&amp;author=M.W.%20Blair&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=827-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1630-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1630-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1630-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21785951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Yonemaru</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Fine definition of the pedigree haplotypes of closely related rice cultivars by means of genome-wide discovery of single-nucleotide polymorphisms</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">267</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20definition%20of%20the%20pedigree%20haplotypes%20of%20closely%20related%20rice%20cultivars%20by%20means%20of%20genome-wide%20discovery%20of%20single-nucleotide%20polymorphisms&amp;author=T.%20Yamamoto&amp;author=H.%20Nagasaki&amp;author=J.%20Yonemaru&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20423466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Single-nucleotide polymorphism discovery by high-throughput sequencing in sorghum</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">352</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-nucleotide%20polymorphism%20discovery%20by%20high-throughput%20sequencing%20in%20sorghum&amp;author=J.C.%20Nelson&amp;author=S.%20Wang&amp;author=Y.%20Wu&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=352" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21736744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Byers</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Harker</div> <div class="given-names">D.B.</div></div> <div class="name"><div class="surname">Yourstone</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Maughan</div> <div class="given-names">P.J.</div></div> <div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div></div> <div class="year">2012</div>, <div class="article-title">Development and mapping of SNP assays in allotetraploid cotton</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">124</div>, <div class="fpage">1201</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20mapping%20of%20SNP%20assays%20in%20allotetraploid%20cotton&amp;author=R.L.%20Byers&amp;author=D.B.%20Harker&amp;author=S.M.%20Yourstone&amp;author=P.J.%20Maughan&amp;author=J.A.%20Udall&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1201-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1780-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1780-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1780-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22252442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zheng</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Sweetingham</div> <div class="given-names">M.W.</div></div> <div class="name"><div class="surname">Howieson</div> <div class="given-names">J.G.</div></div></div> <div class="year">2012</div>, <div class="article-title">Application of next-generation sequencing for rapid marker development in molecular plant breeding: a case study on anthracnose disease resistance in <em>Lupinus angustifolius</em> L</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">318</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20next-generation%20sequencing%20for%20rapid%20marker%20development%20in%20molecular%20plant%20breeding%3A%20a%20case%20study%20on%20anthracnose%20disease%20resistance%20in%20Lupinus%20angustifolius%20L&amp;author=H.%20Yang&amp;author=Y.%20Tao&amp;author=Z.%20Zheng&amp;author=C.%20Li&amp;author=M.W.%20Sweetingham&amp;author=J.G.%20Howieson&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-318" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-318" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-318"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22805587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">M.R.</div></div> <div class="name"><div class="surname">Dunham</div> <div class="given-names">J.P.</div></div> <div class="name"><div class="surname">Amores</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div> <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.A.</div></div></div> <div class="year">2007</div>, <div class="article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers</div>, <div class="source">Genome Res.</div> , <div class="volume">17</div>, <div class="fpage">240</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20cost-effective%20polymorphism%20identification%20and%20genotyping%20using%20restriction%20site%20associated%20DNA%20(RAD)%20markers&amp;author=M.R.%20Miller&amp;author=J.P.%20Dunham&amp;author=A.%20Amores&amp;author=W.A.%20Cresko&amp;author=E.A.%20Johnson&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=240-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5681207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5681207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5681207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17189378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Baird</div> <div class="given-names">N.A.</div></div> <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Atwood</div> <div class="given-names">T.S.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PloS ONE</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.A.%20Baird&amp;author=P.D.%20Etter&amp;author=T.S.%20Atwood&amp;publication_year=2008&amp;journal=PloS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bus</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hecht</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Huettel</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Reinhardt</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Stich</div> <div class="given-names">B.</div></div></div> <div class="year">2012</div>, <div class="article-title">High-throughput polymorphism detection and genotyping in <em>Brassica napus</em> using next-generation RAD sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">281</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20polymorphism%20detection%20and%20genotyping%20in%20Brassica%20napus%20using%20next-generation%20RAD%20sequencing&amp;author=A.%20Bus&amp;author=J.%20Hecht&amp;author=B.%20Huettel&amp;author=R.%20Reinhardt&amp;author=B.%20Stich&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=281" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22726880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Xin</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, <div class="article-title">Construction of a high-density genetic map for grape using next generation restriction-site associated DNA sequencing</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">12</div>, <div class="fpage">148</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20for%20grape%20using%20next%20generation%20restriction-site%20associated%20DNA%20sequencing&amp;author=N.%20Wang&amp;author=L.%20Fang&amp;author=H.%20Xin&amp;author=L.%20Wang&amp;author=S.%20Li&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22908993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Pegadaraju</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Nipper</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Hulke</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Qi</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Schultz</div> <div class="given-names">Q.</div></div></div> <div class="year">2013</div>, <div class="article-title">De novo sequencing of sunflower genome for SNP discovery using RAD (Restriction site Associated DNA) approach</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">556</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20sequencing%20of%20sunflower%20genome%20for%20SNP%20discovery%20using%20RAD%20(Restriction%20site%20Associated%20DNA)%20approach&amp;author=V.%20Pegadaraju&amp;author=R.%20Nipper&amp;author=B.%20Hulke&amp;author=L.%20Qi&amp;author=Q.%20Schultz&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=556" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23947483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Feng</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2009</div>, <div class="article-title">Genetic analysis of major QTLs and epistasis interaction for yield and fiber quality in upland cotton</div>, <div class="source">Sci. Agric. Sin.</div> , <div class="volume">42</div>, <div class="fpage">3036</div>–<div class="lpage">47</div>.</div></div></div></div><div content-id="DSU047C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div> <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2008</div>, <div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">821</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div></div> <div class="year">1999</div>, <div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="volume">9</div>, <div class="fpage">868</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">McCarthy</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Magee</div> <div class="given-names">G.B.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">AgBase: a functional genomics resource for agriculture</div>, <div class="source">BMC Genomics</div> , <div class="volume">7</div>, <div class="fpage">229</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AgBase%3A%20a%20functional%20genomics%20resource%20for%20agriculture&amp;author=F.%20McCarthy&amp;author=N.%20Wang&amp;author=G.B.%20Magee&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-229"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16961921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Harris</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Clark</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Ireland</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">The Gene Ontology (GO) database and informatics resource</div>, <div class="source">Nucl. Acid Res.</div> , <div class="volume">32</div>, <div class="fpage">D258</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gene%20Ontology%20(GO)%20database%20and%20informatics%20resource&amp;author=M.A.%20Harris&amp;author=J.%20Clark&amp;author=A.%20Ireland&amp;publication_year=2004&amp;journal=Nucl.%20Acid%20Res.&amp;volume=32&amp;pages=D258-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh066"> </span></div> </div></div></div></div></div><div content-id="DSU047C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">McKenna</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">The genome analysis toolkit: a mapreduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20analysis%20toolkit%3A%20a%20mapreduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Fang</div> <div class="given-names">X.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">SNP detection for massively parallel whole-genome resequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">1124</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20detection%20for%20massively%20parallel%20whole-genome%20resequencing&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=X.%20Fang&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1124-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.088013.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.088013.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.088013.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Ren</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Tu</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2012</div>, <div class="article-title">Development and evaluation of intron and insertion–deletion markers for <em>Gossypium barbadense</em></div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="volume">30</div>, <div class="fpage">605</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20evaluation%20of%20intron%20and%20insertion%E2%80%93deletion%20markers%20for%20Gossypium%20barbadense&amp;author=X.%20Wang&amp;author=G.%20Ren&amp;author=X.%20Li&amp;author=J.%20Tu&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2012&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=30&amp;pages=605-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-011-0369-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-011-0369-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-011-0369-3"> </span></div> </div></div></div></div></div><div content-id="DSU047C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div></div> <div class="year">2013</div>, <div class="article-title">Construction of introgression lines, development of new markers, and construction of high-density genetic linkage map in cotton</div>. <div class="comment">Doctor Dissertation</div>, <div class="publisher-name">Wuhan, Huazhong Agricultural University</div>.</div></div></div></div><div content-id="DSU047C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Feng</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2008</div>, <div class="article-title">Genetic evaluation of EST-SSRs derived from <em>Gossypium herbaceum</em></div>, <div class="source">Acta Agron. Sin.</div> , <div class="volume">34</div>, <div class="fpage">2085</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20evaluation%20of%20EST-SSRs%20derived%20from%20Gossypium%20herbaceum&amp;author=Y.%20Yu&amp;author=Z.%20Wang&amp;author=C.%20Feng&amp;author=Y.%20Zhang&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2008&amp;journal=Acta%20Agron.%20Sin.&amp;volume=34&amp;pages=2085-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1875-2780(09)60020-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1875-2780(09)60020-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1875-2780(09)60020-1"> </span></div> </div></div></div></div></div><div content-id="DSU047C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2009</div>, <div class="article-title">Isolation, characterization, and mapping of genomic microsatellite markers for the first time in sea-island cotton (<em>Gossypium barbadense</em>)</div>, <div class="source">Acta Agron. Sin.</div> , <div class="volume">35</div>, <div class="fpage">1013</div>–<div class="lpage">20</div>.</div></div></div></div><div content-id="DSU047C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Yuan</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Liang</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Genome structure of cotton revealed by a genome-wide SSR genetic map constructed from a BC<sub>1</sub> population between <em>gossypium hirsutum</em> and <em>G. barbadense</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20cotton%20revealed%20by%20a%20genome-wide%20SSR%20genetic%20map%20constructed%20from%20a%20BC1%20population%20between%20gossypium%20hirsutum%20and%20G.%20barbadense&amp;author=Y.%20Yu&amp;author=D.%20Yuan&amp;author=S.%20Liang&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21214949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div> <div class="year">2011</div>, <div class="article-title">Unbiased genomic distribution of genes related to cell morphogenesis in cotton by chromosome mapping</div>, <div class="source">Plant Cell Tissue Organ Cult.</div> , <div class="volume">108</div>, <div class="fpage">529</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unbiased%20genomic%20distribution%20of%20genes%20related%20to%20cell%20morphogenesis%20in%20cotton%20by%20chromosome%20mapping&amp;author=C.%20Liu&amp;author=Z.%20Lin&amp;author=X.%20Zhang&amp;publication_year=2011&amp;journal=Plant%20Cell%20Tissue%20Organ%20Cult.&amp;volume=108&amp;pages=529-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11240-011-0059-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11240-011-0059-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11240-011-0059-8"> </span></div> </div></div></div></div></div><div content-id="DSU047C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Cai</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">A microsatellite-based, gene-rich linkage map reveals genome structure, function and evolution in <em>Gossypium</em></div>, <div class="source">Genetics</div> , <div class="volume">176</div>, <div class="fpage">527</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%2C%20gene-rich%20linkage%20map%20reveals%20genome%20structure%2C%20function%20and%20evolution%20in%20Gossypium&amp;author=W.%20Guo&amp;author=C.%20Cai&amp;author=C.%20Wang&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=527-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.070375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.070375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.070375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17409069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Cai</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhao</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div></div> <div class="year">2008</div>, <div class="article-title">A preliminary analysis of genome structure and composition in <em>Gossypium hirsutum</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">314</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20preliminary%20analysis%20of%20genome%20structure%20and%20composition%20in%20Gossypium%20hirsutum&amp;author=W.%20Guo&amp;author=C.%20Cai&amp;author=C.%20Wang&amp;author=L.%20Zhao&amp;author=L.%20Wang&amp;author=T.%20Zhang&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18590573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hoffman</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Kohel</div> <div class="given-names">R.J.</div></div> <div class="name"><div class="surname">Pepper</div> <div class="given-names">A.E.</div></div> <div class="name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">J.Z.</div></div> <div class="name"><div class="surname">Grum</div> <div class="given-names">D.S.</div></div></div> <div class="year">2007</div>, <div class="article-title">Identification of 700 new microsatellite loci from cotton (<em>G. hirsutum</em> L.)</div>, <div class="source">Cotton Sci.</div> , <div class="volume">11</div>, <div class="fpage">208</div>–<div class="lpage">41</div>.</div></div></div></div><div content-id="DSU047C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div></div> <div class="year">1993</div>, <div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map</div>, <div class="source">Plant J.</div> , <div class="volume">3</div>, <div class="fpage">739</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20Join%20Map&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSU047C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div></div> <div class="year">2002</div>, <div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">Heredity</div> , <div class="volume">93</div>, <div class="fpage">77</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=Heredity&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div> </div></div></div></div></div><div content-id="DSU047C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div> <div class="name"><div class="surname">Cho</div> <div class="given-names">Y.G.</div></div> <div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Paul</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Blinstrub</div> <div class="given-names">M.</div></div></div> <div class="year">1997</div>, <div class="article-title">Report on QTL nomenclature</div>, <div class="source">Rice Genet. Newsl.</div> , <div class="volume">14</div>, <div class="fpage">11</div>–<div class="lpage">131</div>.</div></div></div></div><div content-id="DSU047C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Barchi</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Lanteri</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Portis</div> <div class="given-names">E.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Identification of SNP and SSR markers in eggplant using RAD tag sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">304</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20SNP%20and%20SSR%20markers%20in%20eggplant%20using%20RAD%20tag%20sequencing&amp;author=L.%20Barchi&amp;author=S.%20Lanteri&amp;author=E.%20Portis&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21663628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="collab">Arabidopsis Genome Initiative</div>. <div class="year">2000</div>, <div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>–<div class="lpage">815</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div><div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Arabidopsis+Genome+Initiative.+2000,+Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana,+Nature,+408,+796–815." target="_blank">Google Scholar</a></div></div></div></div></div></div><div content-id="DSU047C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">The genome of <em>Theobroma cacao</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">101</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.M.%20Aury&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.736" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Gao</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Jin</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div></div> <div class="year">2014</div>, <div class="article-title">Comparison and development of EST–SSRs from two 454 sequencing libraries of <em>Gossypium barbadense</em></div>, <div class="source">Euphytica</div> , <div class="volume">198</div>, <div class="fpage">277</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20and%20development%20of%20EST%E2%80%93SSRs%20from%20two%20454%20sequencing%20libraries%20of%20Gossypium%20barbadense&amp;author=H.%20Wang&amp;author=X.%20Li&amp;author=W.%20Gao&amp;author=X.%20Jin&amp;author=X.%20Zhang&amp;author=Z.%20Lin&amp;publication_year=2014&amp;journal=Euphytica&amp;volume=198&amp;pages=277-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-014-1104-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-014-1104-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-014-1104-6"> </span></div> </div></div></div></div></div><div content-id="DSU047C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Scaglione</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Acquadro</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Portis</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Tirone</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Knapp</div> <div class="given-names">S.J.</div></div> <div class="name"><div class="surname">Lanteri</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, <div class="article-title">RAD tag sequencing as a source of SNP markers in <em>Cynara cardunculus</em> L</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAD%20tag%20sequencing%20as%20a%20source%20of%20SNP%20markers%20in%20Cynara%20cardunculus%20L&amp;author=D.%20Scaglione&amp;author=A.%20Acquadro&amp;author=E.%20Portis&amp;author=M.%20Tirone&amp;author=S.J.%20Knapp&amp;author=S.%20Lanteri&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Preston</div> <div class="given-names">J.L.</div></div> <div class="name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div> <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.A.</div></div></div> <div class="year">2011</div>, <div class="article-title">Local de novo assembly of RAD paired-end contigs using short sequencing reads</div>, <div class="source">PloS ONE</div> , <div class="volume">6</div>, <div class="fpage">e18561</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Local%20de%20novo%20assembly%20of%20RAD%20paired-end%20contigs%20using%20short%20sequencing%20reads&amp;author=P.D.%20Etter&amp;author=J.L.%20Preston&amp;author=S.%20Bassham&amp;author=W.A.%20Cresko&amp;author=E.A.%20Johnson&amp;publication_year=2011&amp;journal=PloS%20ONE&amp;volume=6&amp;pages=e18561" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0018561" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0018561" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0018561"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21541009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">N.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">Control of plant trichome development by a cotton fiber MYB gene</div>, <div class="source">Plant Cell</div> , <div class="volume">16</div>, <div class="fpage">2323</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20plant%20trichome%20development%20by%20a%20cotton%20fiber%20MYB%20gene&amp;author=S.%20Wang&amp;author=J.%20Wang&amp;author=N.%20Yu&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=2323-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.024844" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.024844" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.024844"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15316114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ruan</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Llewellyn</div> <div class="given-names">D.J.</div></div> <div class="name"><div class="surname">Furbank</div> <div class="given-names">R.T.</div></div></div> <div class="year">2001</div>, <div class="article-title">The control of single-celled cotton fiber elongation by developmentally reversible gating of plasmodesmata and coordinated expression of sucrose and K+ transporters and expansin</div>, <div class="source">Plant Cell</div> , <div class="volume">13</div>, <div class="fpage">47</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20control%20of%20single-celled%20cotton%20fiber%20elongation%20by%20developmentally%20reversible%20gating%20of%20plasmodesmata%20and%20coordinated%20expression%20of%20sucrose%20and%20K%2B%20transporters%20and%20expansin&amp;author=Y.%20Ruan&amp;author=D.J.%20Llewellyn&amp;author=R.T.%20Furbank&amp;publication_year=2001&amp;journal=Plant%20Cell&amp;volume=13&amp;pages=47-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11158528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yoo</div> <div class="given-names">M.J.</div></div> <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div></div> <div class="year">2014</div>, <div class="article-title">Comparative evolutionary and developmental dynamics of the cotton (<em>Gossypium hirsutum</em>) fiber transcriptome</div>, <div class="source">PLoS Genet.</div> , <div class="volume">10</div>, <div class="fpage">e1004073</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20evolutionary%20and%20developmental%20dynamics%20of%20the%20cotton%20(Gossypium%20hirsutum)%20fiber%20transcriptome&amp;author=M.J.%20Yoo&amp;author=J.F.%20Wendel&amp;publication_year=2014&amp;journal=PLoS%20Genet.&amp;volume=10&amp;pages=e1004073" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24391525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div> <div class="name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div> <div class="name"><div class="surname">Gundlach</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Repeated polyploidization of <em>Gossypium</em> genomes and the evolution of spinnable cotton fibres</div>, <div class="source">Nature</div> , <div class="volume">492</div>, <div class="fpage">423</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeated%20polyploidization%20of%20Gossypium%20genomes%20and%20the%20evolution%20of%20spinnable%20cotton%20fibres&amp;author=A.H.%20Paterson&amp;author=J.F.%20Wendel&amp;author=H.%20Gundlach&amp;publication_year=2012&amp;journal=Nature&amp;volume=492&amp;pages=423-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11798" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11798" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11798"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23257886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Fan</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Genome sequence of the cultivated cotton <em>Gossypium arboreum</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">46</div>, <div class="fpage">567</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20cultivated%20cotton%20Gossypium%20arboreum&amp;author=F.%20Li&amp;author=G.%20Fan&amp;author=K.%20Wang&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=567-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24836287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Srivastava</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Jena</div> <div class="given-names">S.N.</div></div> <div class="name"><div class="surname">Ranjan</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Development of molecular markers from Indian genotypes of two <em>Gossypium</em> L. species</div>, <div class="source">Plant Breed</div> , <div class="volume">132</div>, <div class="fpage">506</div>–<div class="lpage">13</div>.</div></div></div></div><div content-id="DSU047C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">QTL mapping for fiber quality traits across multiple generations and environments in upland cotton</div>, <div class="source">Mol. Breeding</div> , <div class="volume">30</div>, <div class="fpage">569</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20for%20fiber%20quality%20traits%20across%20multiple%20generations%20and%20environments%20in%20upland%20cotton&amp;author=F.%20Sun&amp;author=J.%20Zhang&amp;author=S.%20Wang&amp;publication_year=2012&amp;journal=Mol.%20Breeding&amp;volume=30&amp;pages=569-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9645-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9645-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9645-z"> </span></div> </div></div></div></div></div><div content-id="DSU047C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rong</div> <div class="given-names">J.K.</div></div> <div class="name"><div class="surname">Abbey</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>et al.  </div> <div class="year">2004</div>, <div class="article-title">A 3347-locus genetic recombination map of sequence-tagged sites reveals features of genome organization, transmission and evolution of cotton (<em>Gossypium</em>)</div>, <div class="source">Genetics</div> , <div class="volume">166</div>, <div class="fpage">389</div>–<div class="lpage">417</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%203347-locus%20genetic%20recombination%20map%20of%20sequence-tagged%20sites%20reveals%20features%20of%20genome%20organization%2C%20transmission%20and%20evolution%20of%20cotton%20(Gossypium)&amp;author=J.K.%20Rong&amp;author=C.%20Abbey&amp;author=J.E.%20Bowers&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=389-417" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.1.389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.1.389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.1.389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15020432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU047C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU047C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Detection and genotyping of restriction fragment associated polymorphisms in polyploid crops with a pseudo-reference sequence: a case study in allotetraploid <em>Brassica napus</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">346</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20genotyping%20of%20restriction%20fragment%20associated%20polymorphisms%20in%20polyploid%20crops%20with%20a%20pseudo-reference%20sequence%3A%20a%20case%20study%20in%20allotetraploid%20Brassica%20napus&amp;author=X.%20Chen&amp;author=X.%20Li&amp;author=B.%20Zhang&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23706002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=332826&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table6.xls?Expires=2147483647&amp;Signature=AQx0kh6YogqTfRJwpRIdOiUY3cBFDIyTjPClGyQ1zmhsciPmkFqZKk~BlEq1CBNlCcfJa61XMclxYyqDWMIsVyAxX1qTEpMCsXX70zRs9g1ADGOQfs~s5Njrh3dAG76aGZuM4bNEpjOOu4aEUHvt0NxwkqTlFn5JUMWiEKqQXu3JIFpvAukiuSkYMR6E--X~tBsIr2-sSiy0EoM~8NY1CxN5qZpUS5pTSNOhf1Y6T8u4SC6srGWLLP4UihCjZJDsKDU-ZQ6GVk-MbKpgfyusalK5ynfFdo6YzWzmNwnRiCa7ItBL7BCs2ypJnj3pwbk~sx~qjyWA8uapTUTd35tEmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table5.pdf?Expires=2147483647&amp;Signature=gfxoB89sU74dxETMqPZkA8o8Zuh5hU1oanQ~tiRJCzTyi0-TWjL2eryfZa5julOhDq7Mk-LHGhMvYuTA8D6eDkyfl2giWsML57v1fyTEDE9~ts8PcQOAI7hQe4xGJ1eVDO9Yz4TDuQvdqy1H42tEEbxfL4GxKrUza6jC-QhccIu93X~IEx6w9HpuaRZpG1JmJSS~9NNBwgFNkWCL5j00r4EQkQPmdBp9Eajqq5vOeCUTs-~coJssVq2obCYfVVbMussyCZkH4E3JH718jZX9gPGCYvQelIgfTWfipI9vvWX7D2GmvyFdzZTdoaQHJ46LAC2arvEmXhftIs-7Qm730w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table4.xls?Expires=2147483647&amp;Signature=G07S3R0WoabzBl4AJuy~LssV-3ovs1J4-HCbRkoFXPh7FNOzq4TSu-VuYT2Ltr-ACLkCGtnhqjT9DOMC2lz~9M0XfDqQDLYN8SyF3ZMoHnbEGABiqQRMlwv4Gu8PcacRh2uJj6q4CptlMQE~D6R5aMyIRAgUCg6EYHUns92Y1p0qNqQ6bmtlptpaWgrUGO9BPqAQRztzpyq1hF0myrMv3MClyFtpXyJj1hvFoudCYDfb2M~4ljJ~VhXrcB7P8FCBzuwPXkJdCD1HAthQTXUrlDdvoyP5gL-Ws6-8LRyZ0BWm14cIk9SMtYalH03eA4H3qiY9J8Umyz~-pig299NySw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table3.xls?Expires=2147483647&amp;Signature=VaZhNXuOC2wxYJyzXnTS-ztnUAEqRBVnHwWZbnbppbrwQjE7iCBSOi4-VNZruuODWhLTGrMvVhjijK9Ug0Iwj1YJf-fpwREvxY~CRX5aP2LdBAsevAYDDxYiMBF9PmA3OH~SRR7Dl44PdGcdUsAmeszWJgoaewB9jl1XQm36WRmuvOzdh~6SPEYdWGvHy3UccehXN89Uv8PEFVKfCeBVfU084U1oKFaGfg9ytdemAxiceWZzwo9UOWHtc08fh2csF1QL0k0QEQeF-7tQQKfUWKBryXaUwxb5BwMYZU8IYSROOoXyWA3hUqyizPcgNgDK41-8x-IyfvT~4ssPoCGRqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table2.xls?Expires=2147483647&amp;Signature=pOU5D9L5FMR-tBcRnvdMHYc9Y5ajoEHbSZI~2nAL~fZvRdU9SjDF-buKB1LQn9NSOqrtO~J-QWnAxgl9o4pTBg10x9EbsrFCqjYvRq2rUrUnzwt7gQfsAzkerm~eWLWxC7WvMkUPOzo22E6pEi8H9IgK1UvEE7kAWd47A1hwWCUDam~wUKeF9hG3V6U8vYSbiSbRAXFqOBC694OxcwbSWxXg2s5FTPQ4m68I04S0pav7hZNV6JVPN-M7-gquTIyWRrLHLAOw11XBaZc8r0rHJDWnQH2t-rVCwLZSDO1FblxqBKNXk103D29zCmsF9dmqX7ivZtT-FSvYzSk-4~T9Cw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_table1.xls?Expires=2147483647&amp;Signature=eu4pFOTeJSSnE9BIAn4N8hE4RUaJa-hnlB8cfsjLdmEyR0t8I-96YCUJOXNhzjNEO3oQwSci0V-L59jPxwbs1~530naYv4hXErKmhoGkEm9~OkJon5FaTAsa-QTMN-jMXlKIyfaABrxRX58~B1prbwJqiC2Eylw7QlEeFS7gdqDyrs0j7nPuvlg3pULhRXCHqadMLURBKdLIRKuKcuftvzu0aJtno8sO8A9PaSW8CA7eGmgJf~Cq9SE6JedzyNQq9Ea~QMBVdPkgA58dv7E-no4hxg83yqEWag8-~GMzCuEV4seOI3velkrsoKgItK1h7A-hqNfCn7Nkq79UIp6HxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu047/2/dsu047-dsu047supp_fig1.pdf?Expires=2147483647&amp;Signature=emMK0O1ZJepAX-v1m7ZLNQGjfrG1VxDsAmiDEJr0EUZ62xHLYF0OLXLm-np6Iyz85NDHd9Wj73QXhUThKJvNzITZBdX5cpS~dnj8o9J847gaBSfLS8SI~5IOJXrLGyZ7qL229rOOj4nQsIZ8ILZMgQgNQda6qLOdBSzh~jqmC2oLRzpr2oFo~C4Is~cHeBLWz7of8-hg2DUl8mCPuT8jQHRxqp4rjhliwSeGV61doaHrkheuEgrVbFRwQcckkDmnzIr4ned~lIh31Y89M-HW3fZiSyNz0u6cK2M4xrWDSH7KZ2~Ns5T8jIsi3WIHqXjtZoYFz88zb-0FtZ4pSRIN1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">559</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">22</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu047"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="332826"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f2%2f147%2f332826">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000353644600003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (22)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F2%2F147%2F332826" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/332826'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Enrichment of an intraspecific genetic map of upland cotton by developing markers using parental RAD sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu047");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f147%2f332826">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

