

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/6/549/419528","@type":"ScholarlyArticle","name":"Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome","datePublished":"2013-07-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2013-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst030","keywords":["DNA methylation","primordial germ cell","epigenome","reprogramming","cancer testis antigen"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/6/549/419528","author":[{"name":"Ikeda, Rieko","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan    2  Graduate School of Life and Environmental Sciences ,  University of Tsukuba ,  Ibaraki 305-8572 ,  Japan","@type":"Person"},{"name":"Shiura, Hirosuke","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Numata, Koji","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Sugimoto, Michihiko","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Kondo, Masayo","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan","@type":"Person"},{"name":"Mise, Nathan","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan    3  Department of Environmental Preventive Medicine ,  Jichi Medical University ,  Tochigi 329-0431 ,  Japan","@type":"Person"},{"name":"Suzuki, Masako","affiliation":"4  Department of Genetics (Computational Genetics) ,  Albert Einstein College of Medicine ,  Bronx, NY 10461 ,  USA","@type":"Person"},{"name":"Greally, John M.","affiliation":"4  Department of Genetics (Computational Genetics) ,  Albert Einstein College of Medicine ,  Bronx, NY 10461 ,  USA","@type":"Person"},{"name":"Abe, Kuniya","affiliation":"1  Technology and Development Team for Mammalian Genome Dynamics ,  RIKEN BioResource Center ,  3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074 ,  Japan    2  Graduate School of Life and Environmental Sciences ,  University of Tsukuba ,  Ibaraki 305-8572 ,  Japan","@type":"Person"}],"description":"Abstract.  To understand the epigenetic regulation required for germ cell-specific gene expression in the mouse, we analysed DNA methylation profiles of develop","pageStart":"549","pageEnd":"565","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome" />
    <meta property="og:description" content="Abstract.  To understand the epigenetic regulation required for germ cell-specific gene expression in the mouse, we analysed DNA methylation profiles of develop" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/6/549/419528" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-07-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ikeda, Rieko" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan" /><meta name="citation_author" content="Shiura, Hirosuke" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Numata, Koji" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Sugimoto, Michihiko" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Kondo, Masayo" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author" content="Mise, Nathan" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author_institution" content="Department of Environmental Preventive Medicine, Jichi Medical University, Tochigi 329-0431, Japan" /><meta name="citation_author" content="Suzuki, Masako" /><meta name="citation_author_institution" content="Department of Genetics (Computational Genetics), Albert Einstein College of Medicine, Bronx, NY 10461, USA" /><meta name="citation_author" content="Greally, John M." /><meta name="citation_author_institution" content="Department of Genetics (Computational Genetics), Albert Einstein College of Medicine, Bronx, NY 10461, USA" /><meta name="citation_author" content="Abe, Kuniya" /><meta name="citation_author_institution" content="Technology and Development Team for Mammalian Genome Dynamics, RIKEN BioResource Center, 3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074, Japan" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan" /><meta name="citation_title" content="Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome" /><meta name="citation_firstpage" content="549" /><meta name="citation_lastpage" content="565" /><meta name="citation_doi" content="10.1093/dnares/dst030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2013/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Epigenetic reprogramming in mouse primordial germ cells; Mech. Dev.;  citation_year=2002;  citation_volume=117;  citation_pages=15-23; " /><meta name="citation_reference" content="citation_title=Chromatin dynamics during epigenetic reprogramming in the mouse germ line; Nature;  citation_year=2008;  citation_volume=452;  citation_pages=877-81; " /><meta name="citation_reference" content="citation_title=Primordial germ cells in the mouse; Dev. Biol.;  citation_year=2003;  citation_volume=262;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Targets and dynamics of promoter DNA methylation during early mouse development; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1093-100; " /><meta name="citation_reference" content="citation_title=Genome-scale DNA methylation maps of pluripotent and differentiated cells; Nature;  citation_year=2008;  citation_volume=454;  citation_pages=766-70; " /><meta name="citation_reference" content="citation_title=DNA methylation landscapes: provocative insights from epigenomics; Nat. Rev. Genet.;  citation_year=2008;  citation_volume=9;  citation_pages=465-76; " /><meta name="citation_reference" content="citation_title=Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=40-6; " /><meta name="citation_reference" content="citation_title=Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=1101-05; " /><meta name="citation_reference" content="citation_title=Global profiling of DNA methylation erasure in mouse primordial germ cells; Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=633-41; " /><meta name="citation_reference" content="citation_title=The dynamics of genome-wide DNA methylation reprogramming in mouse primordial germ cells; Mol. Cell;  citation_year=2012;  citation_volume=48;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=Germline DNA demethylation dynamics and imprint erasure through 5-hydroxymethylcytosine; Science;  citation_year=2013;  citation_volume=339;  citation_pages=448-52; " /><meta name="citation_reference" content="citation_title=Comparative isoschizomer profiling of cytosine methylation: the HELP assay; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1046-55; " /><meta name="citation_reference" content="citation_title=Comprehensive high-throughput arrays for relative methylation (CHARM); Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=780-90; " /><meta name="citation_reference" content="citation_title=High-resolution genome-wide cytosine methylation profiling with simultaneous copy number analysis and optimization for limited cell numbers; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=3829-39; " /><meta name="citation_reference" content="citation_title=An abundance of X-linked genes expressed in spermatogonia; Nat. Genet.;  citation_year=2001;  citation_volume=27;  citation_pages=422-6; " /><meta name="citation_reference" content="citation_title=The mouse X chromosome is enriched for multicopy testis genes showing postmeiotic expression; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=794-9; " /><meta name="citation_reference" content="citation_title=X chromosome reactivation initiates in nascent primordial germ cells in mice; PLoS Genet.;  citation_year=2007;  citation_volume=3;  citation_pages=e116" /><meta name="citation_reference" content="citation_title=Cancer/testis (CT) antigens: potential targets for immunotherapy; Cancer Sci.;  citation_year=2009;  citation_volume=100;  citation_pages=2014-21; " /><meta name="citation_reference" content="citation_title=Epigenotype switching of imprintable loci in embryonic germ cells; Dev. Genes Evol.;  citation_year=1998;  citation_volume=207;  citation_pages=551-61; " /><meta name="citation_reference" content="citation_title=Differences and similarities in the developmental status of embryo-derived stem cells and primordial revealed by global expression profiling; Genes Cells;  citation_year=2008;  citation_volume=13;  citation_pages=863-77; " /><meta name="citation_reference" content="citation_title=Identifi-cation and characterization of stem cells in prepubertal spermatogenesis in mice; Dev. Biol.;  citation_year=2003;  citation_volume=258;  citation_pages=209-25; " /><meta name="citation_reference" content="citation_title=Long-term proliferation in culture and germline transmission of mouse male germline stem cells; Biol. Reprod.;  citation_year=2003;  citation_volume=69;  citation_pages=612-6; " /><meta name="citation_reference" content="citation_title=A high-speed congenic strategy using first-wave male germ cells; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e4943" /><meta name="citation_reference" content="citation_title=The X-linked imprinted gene family Fthl17 shows predominantly female expression following the two-cell stage in mouse embryos; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=3672-81; " /><meta name="citation_reference" content="citation_title=Unbiased amplification of a highly complex mixture of DNA fragments by ‘lone linker’-tagged PCR; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=4293-4; " /><meta name="citation_reference" content="citation_title=Impeding Xist expression from the active X chromosome improves mouse somatic cell cloning; Science;  citation_year=2010;  citation_volume=330;  citation_pages=496-9; " /><meta name="citation_reference" content="citation_title=Threshold selection method from gray-level histograms; IEEE Trans. Syst. Man. Cybern.;  citation_year=1979;  citation_volume=9;  citation_pages=62-6; " /><meta name="citation_reference" content="citation_title=On the generalized distance in statistics; Proc. Natl Inst. Sci. India;  citation_year=1936;  citation_volume=2;  citation_pages=49" /><meta name="citation_reference" content="citation_title=Expression analysis of the mouse multi-copy X-linked gene, Xlr-related, meiosis-regulated (Xmr), reveals that Xmr encodes a spermatid-expressed cytoplasmic protein, SLX/XMR; Biol. Reprod.;  citation_year=2007;  citation_volume=77;  citation_pages=329-35; " /><meta name="citation_reference" content="citation_title=A genetic basis for a postmeiotic X versus Y chromosome intragenomic conflict in the mouse; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002900" /><meta name="citation_reference" content="citation_title=Cell- and stage-specific expression of mage genes during mouse spermatogenesis; Mammal. Genome;  citation_year=2000;  citation_volume=11;  citation_pages=696-9; " /><meta name="citation_reference" content="citation_title=Circular binary segmentation for the analysis of array-based DNA copy number data; Biostatistics;  citation_year=2004;  citation_volume=5;  citation_pages=557-72; " /><meta name="citation_reference" content="citation_title=CpG islands in vertebrate genomes; J. Mol. Biol.;  citation_year=1987;  citation_volume=196;  citation_pages=261-82; " /><meta name="citation_reference" content="citation_title=Segmental duplications: organization and impact within the current human genome project assembly; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1005-17; " /><meta name="citation_reference" content="citation_title=Isolation of a DEAD-family protein gene that encodes a murine homolog of Drosophila vasa and its specific expression in germ cell lineage; Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=12258-62; " /><meta name="citation_reference" content="citation_title=H3K27me3 forms BLOCs over silent genes and intergenic regions and specifies a histone banding pattern on a mouse autosomal chromosome; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=221-33; " /><meta name="citation_reference" content="citation_title=Large histone H3 lysine 9 dimethylated chromatin blocks distinguish differentiated from embryonic stem cells; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=246-50; " /><meta name="citation_reference" content="citation_title=DNA methylation regulates long-range gene silencing of an X-linked homeobox gene cluster in a lineage-specific manner; Genes Dev.;  citation_year=2006;  citation_volume=20;  citation_pages=3382-94; " /><meta name="citation_reference" content="citation_title=Distinct roles for histone methyltransferases G9a and GLP in cancer germ-line antigen gene regulation in human cancer cells and murine embryonic stem cells; Mol. Cancer Res.;  citation_year=2009;  citation_volume=7;  citation_pages=851-62; " /><meta name="citation_reference" content="citation_title=Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1861-9; " /><meta name="citation_reference" content="citation_title=DNA methylation and SETDB1/H3K9me3 regulate predominantly distinct sets of genes, retroelements, and chimeric transcripts in mESCs; Cell Stem Cell;  citation_year=2011;  citation_volume=8;  citation_pages=676-87; " /><meta name="citation_reference" content="citation_title=Cancer/testis antigens, gametogenesis and cancer; Nat. Rev. Cancer;  citation_year=2005;  citation_volume=5;  citation_pages=615-25; " /><meta name="citation_reference" content="citation_title=Affy exon tissues: exon levels in normal tissues in human, mouse and rat; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2442-3; " /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/6/549/1163351/dst030.pdf" /><meta name="description" content="Abstract.  To understand the epigenetic regulation required for germ cell-specific gene expression in the mouse, we analysed DNA methylation profiles of develop" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/6/549/419528" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/6/549/419528" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/6/549/419528");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst030';
    </script>
<title>
	Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./419528" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYptPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJa2VkYSwgUmlla28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZWNobm9sb2d5IGFuZCBEZXZlbG9wbWVudCBUZWFtIGZvciBNYW1tYWxpYW4gR2Vub21lIER5bmFtaWNzLCBSSUtFTiBCaW9SZXNvdXJjZSBDZW50ZXIsIDMtMS0xIEtveWFkYWksIFRzdWt1YmEgQ2l0eSwgSWJhcmFraSAzMDUtMDA3NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBhbmQgRW52aXJvbm1lbnRhbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTcyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaXVyYSwgSGlyb3N1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZWNobm9sb2d5IGFuZCBEZXZlbG9wbWVudCBUZWFtIGZvciBNYW1tYWxpYW4gR2Vub21lIER5bmFtaWNzLCBSSUtFTiBCaW9SZXNvdXJjZSBDZW50ZXIsIDMtMS0xIEtveWFkYWksIFRzdWt1YmEgQ2l0eSwgSWJhcmFraSAzMDUtMDA3NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOdW1hdGEsIEtvamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUZWNobm9sb2d5IGFuZCBEZXZlbG9wbWVudCBUZWFtIGZvciBNYW1tYWxpYW4gR2Vub21lIER5bmFtaWNzLCBSSUtFTiBCaW9SZXNvdXJjZSBDZW50ZXIsIDMtMS0xIEtveWFkYWksIFRzdWt1YmEgQ2l0eSwgSWJhcmFraSAzMDUtMDA3NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdWdpbW90bywgTWljaGloaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGVjaG5vbG9neSBhbmQgRGV2ZWxvcG1lbnQgVGVhbSBmb3IgTWFtbWFsaWFuIEdlbm9tZSBEeW5hbWljcywgUklLRU4gQmlvUmVzb3VyY2UgQ2VudGVyLCAzLTEtMSBLb3lhZGFpLCBUc3VrdWJhIENpdHksIEliYXJha2kgMzA1LTAwNzQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29uZG8sIE1hc2F5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRlY2hub2xvZ3kgYW5kIERldmVsb3BtZW50IFRlYW0gZm9yIE1hbW1hbGlhbiBHZW5vbWUgRHluYW1pY3MsIFJJS0VOIEJpb1Jlc291cmNlIENlbnRlciwgMy0xLTEgS295YWRhaSwgVHN1a3ViYSBDaXR5LCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pc2UsIE5hdGhhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRlY2hub2xvZ3kgYW5kIERldmVsb3BtZW50IFRlYW0gZm9yIE1hbW1hbGlhbiBHZW5vbWUgRHluYW1pY3MsIFJJS0VOIEJpb1Jlc291cmNlIENlbnRlciwgMy0xLTEgS295YWRhaSwgVHN1a3ViYSBDaXR5LCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRW52aXJvbm1lbnRhbCBQcmV2ZW50aXZlIE1lZGljaW5lLCBKaWNoaSBNZWRpY2FsIFVuaXZlcnNpdHksIFRvY2hpZ2kgMzI5LTA0MzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3V6dWtpLCBNYXNha28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEdlbmV0aWNzIChDb21wdXRhdGlvbmFsIEdlbmV0aWNzKSwgQWxiZXJ0IEVpbnN0ZWluIENvbGxlZ2Ugb2YgTWVkaWNpbmUsIEJyb254LCBOWSAxMDQ2MSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3JlYWxseSwgSm9obiBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2VuZXRpY3MgKENvbXB1dGF0aW9uYWwgR2VuZXRpY3MpLCBBbGJlcnQgRWluc3RlaW4gQ29sbGVnZSBvZiBNZWRpY2luZSwgQnJvbngsIE5ZIDEwNDYxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIEt1bml5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRlY2hub2xvZ3kgYW5kIERldmVsb3BtZW50IFRlYW0gZm9yIE1hbW1hbGlhbiBHZW5vbWUgRHluYW1pY3MsIFJJS0VOIEJpb1Jlc291cmNlIENlbnRlciwgMy0xLTEgS295YWRhaSwgVHN1a3ViYSBDaXR5LCBJYmFyYWtpIDMwNS0wMDc0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIGFuZCBFbnZpcm9ubWVudGFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1NzIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJMYXJnZSwgTWFsZSBHZXJtIENlbGwtU3BlY2lmaWMgSHlwb21ldGh5bGF0ZWQgRE5BIERvbWFpbnMgV2l0aCBVbmlxdWUgR2Vub21pYyBhbmQgRXBpZ2Vub21pYyBGZWF0dXJlcyBvbiB0aGUgTW91c2UgWCBDaHJvbW9zb21lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1NjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVwcm9ncmFtbWluZyBpbiBtb3VzZSBwcmltb3JkaWFsIGdlcm0gY2VsbHM7IE1lY2guIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9MTUtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tYXRpbiBkeW5hbWljcyBkdXJpbmcgZXBpZ2VuZXRpYyByZXByb2dyYW1taW5nIGluIHRoZSBtb3VzZSBnZXJtIGxpbmU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTI7ICBjaXRhdGlvbl9wYWdlcz04NzctODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1vcmRpYWwgZ2VybSBjZWxscyBpbiB0aGUgbW91c2U7IERldi4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0cyBhbmQgZHluYW1pY3Mgb2YgcHJvbW90ZXIgRE5BIG1ldGh5bGF0aW9uIGR1cmluZyBlYXJseSBtb3VzZSBkZXZlbG9wbWVudDsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDkzLTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIEROQSBtZXRoeWxhdGlvbiBtYXBzIG9mIHBsdXJpcG90ZW50IGFuZCBkaWZmZXJlbnRpYXRlZCBjZWxsczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1NDsgIGNpdGF0aW9uX3BhZ2VzPTc2Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIGxhbmRzY2FwZXM6IHByb3ZvY2F0aXZlIGluc2lnaHRzIGZyb20gZXBpZ2Vub21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQ2NS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVnaW9ucyBvZiBmb2NhbCBETkEgaHlwZXJtZXRoeWxhdGlvbiBhbmQgbG9uZy1yYW5nZSBoeXBvbWV0aHlsYXRpb24gaW4gY29sb3JlY3RhbCBjYW5jZXIgY29pbmNpZGUgd2l0aCBudWNsZWFyIGxhbWluYS1hc3NvY2lhdGVkIGRvbWFpbnM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9NDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZXJhc3VyZSBvZiBETkEgbWV0aHlsYXRpb24gaW4gbW91c2UgcHJpbW9yZGlhbCBnZXJtIGNlbGxzIGlzIGFmZmVjdGVkIGJ5IEFJRCBkZWZpY2llbmN5OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9MTEwMS0wNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHByb2ZpbGluZyBvZiBETkEgbWV0aHlsYXRpb24gZXJhc3VyZSBpbiBtb3VzZSBwcmltb3JkaWFsIGdlcm0gY2VsbHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NjMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHluYW1pY3Mgb2YgZ2Vub21lLXdpZGUgRE5BIG1ldGh5bGF0aW9uIHJlcHJvZ3JhbW1pbmcgaW4gbW91c2UgcHJpbW9yZGlhbCBnZXJtIGNlbGxzOyBNb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZXJtbGluZSBETkEgZGVtZXRoeWxhdGlvbiBkeW5hbWljcyBhbmQgaW1wcmludCBlcmFzdXJlIHRocm91Z2ggNS1oeWRyb3h5bWV0aHlsY3l0b3NpbmU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzM5OyAgY2l0YXRpb25fcGFnZXM9NDQ4LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBpc29zY2hpem9tZXIgcHJvZmlsaW5nIG9mIGN5dG9zaW5lIG1ldGh5bGF0aW9uOiB0aGUgSEVMUCBhc3NheTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDQ2LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGhpZ2gtdGhyb3VnaHB1dCBhcnJheXMgZm9yIHJlbGF0aXZlIG1ldGh5bGF0aW9uIChDSEFSTSk7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NzgwLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gZ2Vub21lLXdpZGUgY3l0b3NpbmUgbWV0aHlsYXRpb24gcHJvZmlsaW5nIHdpdGggc2ltdWx0YW5lb3VzIGNvcHkgbnVtYmVyIGFuYWx5c2lzIGFuZCBvcHRpbWl6YXRpb24gZm9yIGxpbWl0ZWQgY2VsbCBudW1iZXJzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0zODI5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhYnVuZGFuY2Ugb2YgWC1saW5rZWQgZ2VuZXMgZXhwcmVzc2VkIGluIHNwZXJtYXRvZ29uaWE7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb3VzZSBYIGNocm9tb3NvbWUgaXMgZW5yaWNoZWQgZm9yIG11bHRpY29weSB0ZXN0aXMgZ2VuZXMgc2hvd2luZyBwb3N0bWVpb3RpYyBleHByZXNzaW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTc5NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1YIGNocm9tb3NvbWUgcmVhY3RpdmF0aW9uIGluaXRpYXRlcyBpbiBuYXNjZW50IHByaW1vcmRpYWwgZ2VybSBjZWxscyBpbiBtaWNlOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTExNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhbmNlci90ZXN0aXMgKENUKSBhbnRpZ2VuczogcG90ZW50aWFsIHRhcmdldHMgZm9yIGltbXVub3RoZXJhcHk7IENhbmNlciBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTIwMTQtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbm90eXBlIHN3aXRjaGluZyBvZiBpbXByaW50YWJsZSBsb2NpIGluIGVtYnJ5b25pYyBnZXJtIGNlbGxzOyBEZXYuIEdlbmVzIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTIwNzsgIGNpdGF0aW9uX3BhZ2VzPTU1MS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW5jZXMgYW5kIHNpbWlsYXJpdGllcyBpbiB0aGUgZGV2ZWxvcG1lbnRhbCBzdGF0dXMgb2YgZW1icnlvLWRlcml2ZWQgc3RlbSBjZWxscyBhbmQgcHJpbW9yZGlhbCByZXZlYWxlZCBieSBnbG9iYWwgZXhwcmVzc2lvbiBwcm9maWxpbmc7IEdlbmVzIENlbGxzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9ODYzLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaS1jYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygc3RlbSBjZWxscyBpbiBwcmVwdWJlcnRhbCBzcGVybWF0b2dlbmVzaXMgaW4gbWljZTsgRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTg7ICBjaXRhdGlvbl9wYWdlcz0yMDktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmctdGVybSBwcm9saWZlcmF0aW9uIGluIGN1bHR1cmUgYW5kIGdlcm1saW5lIHRyYW5zbWlzc2lvbiBvZiBtb3VzZSBtYWxlIGdlcm1saW5lIHN0ZW0gY2VsbHM7IEJpb2wuIFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz02MTItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLXNwZWVkIGNvbmdlbmljIHN0cmF0ZWd5IHVzaW5nIGZpcnN0LXdhdmUgbWFsZSBnZXJtIGNlbGxzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTQ5NDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgWC1saW5rZWQgaW1wcmludGVkIGdlbmUgZmFtaWx5IEZ0aGwxNyBzaG93cyBwcmVkb21pbmFudGx5IGZlbWFsZSBleHByZXNzaW9uIGZvbGxvd2luZyB0aGUgdHdvLWNlbGwgc3RhZ2UgaW4gbW91c2UgZW1icnlvczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MzY3Mi04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5iaWFzZWQgYW1wbGlmaWNhdGlvbiBvZiBhIGhpZ2hseSBjb21wbGV4IG1peHR1cmUgb2YgRE5BIGZyYWdtZW50cyBieSDigJhsb25lIGxpbmtlcuKAmS10YWdnZWQgUENSOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00MjkzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGVkaW5nIFhpc3QgZXhwcmVzc2lvbiBmcm9tIHRoZSBhY3RpdmUgWCBjaHJvbW9zb21lIGltcHJvdmVzIG1vdXNlIHNvbWF0aWMgY2VsbCBjbG9uaW5nOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMzMDsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaHJlc2hvbGQgc2VsZWN0aW9uIG1ldGhvZCBmcm9tIGdyYXktbGV2ZWwgaGlzdG9ncmFtczsgSUVFRSBUcmFucy4gU3lzdC4gTWFuLiBDeWJlcm4uOyAgY2l0YXRpb25feWVhcj0xOTc5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgZ2VuZXJhbGl6ZWQgZGlzdGFuY2UgaW4gc3RhdGlzdGljczsgUHJvYy4gTmF0bCBJbnN0LiBTY2kuIEluZGlhOyAgY2l0YXRpb25feWVhcj0xOTM2OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGhlIG1vdXNlIG11bHRpLWNvcHkgWC1saW5rZWQgZ2VuZSwgWGxyLXJlbGF0ZWQsIG1laW9zaXMtcmVndWxhdGVkIChYbXIpLCByZXZlYWxzIHRoYXQgWG1yIGVuY29kZXMgYSBzcGVybWF0aWQtZXhwcmVzc2VkIGN5dG9wbGFzbWljIHByb3RlaW4sIFNMWC9YTVI7IEJpb2wuIFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBiYXNpcyBmb3IgYSBwb3N0bWVpb3RpYyBYIHZlcnN1cyBZIGNocm9tb3NvbWUgaW50cmFnZW5vbWljIGNvbmZsaWN0IGluIHRoZSBtb3VzZTsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyOTAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbC0gYW5kIHN0YWdlLXNwZWNpZmljIGV4cHJlc3Npb24gb2YgbWFnZSBnZW5lcyBkdXJpbmcgbW91c2Ugc3Blcm1hdG9nZW5lc2lzOyBNYW1tYWwuIEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTY5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjdWxhciBiaW5hcnkgc2VnbWVudGF0aW9uIGZvciB0aGUgYW5hbHlzaXMgb2YgYXJyYXktYmFzZWQgRE5BIGNvcHkgbnVtYmVyIGRhdGE7IEJpb3N0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTU1Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3BHIGlzbGFuZHMgaW4gdmVydGVicmF0ZSBnZW5vbWVzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE5NjsgIGNpdGF0aW9uX3BhZ2VzPTI2MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VnbWVudGFsIGR1cGxpY2F0aW9uczogb3JnYW5pemF0aW9uIGFuZCBpbXBhY3Qgd2l0aGluIHRoZSBjdXJyZW50IGh1bWFuIGdlbm9tZSBwcm9qZWN0IGFzc2VtYmx5OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDUtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiBhIERFQUQtZmFtaWx5IHByb3RlaW4gZ2VuZSB0aGF0IGVuY29kZXMgYSBtdXJpbmUgaG9tb2xvZyBvZiBEcm9zb3BoaWxhIHZhc2EgYW5kIGl0cyBzcGVjaWZpYyBleHByZXNzaW9uIGluIGdlcm0gY2VsbCBsaW5lYWdlOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTIyNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUgzSzI3bWUzIGZvcm1zIEJMT0NzIG92ZXIgc2lsZW50IGdlbmVzIGFuZCBpbnRlcmdlbmljIHJlZ2lvbnMgYW5kIHNwZWNpZmllcyBhIGhpc3RvbmUgYmFuZGluZyBwYXR0ZXJuIG9uIGEgbW91c2UgYXV0b3NvbWFsIGNocm9tb3NvbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjIxLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBoaXN0b25lIEgzIGx5c2luZSA5IGRpbWV0aHlsYXRlZCBjaHJvbWF0aW4gYmxvY2tzIGRpc3Rpbmd1aXNoIGRpZmZlcmVudGlhdGVkIGZyb20gZW1icnlvbmljIHN0ZW0gY2VsbHM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MjQ2LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gcmVndWxhdGVzIGxvbmctcmFuZ2UgZ2VuZSBzaWxlbmNpbmcgb2YgYW4gWC1saW5rZWQgaG9tZW9ib3ggZ2VuZSBjbHVzdGVyIGluIGEgbGluZWFnZS1zcGVjaWZpYyBtYW5uZXI7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0zMzgyLTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdCByb2xlcyBmb3IgaGlzdG9uZSBtZXRoeWx0cmFuc2ZlcmFzZXMgRzlhIGFuZCBHTFAgaW4gY2FuY2VyIGdlcm0tbGluZSBhbnRpZ2VuIGdlbmUgcmVndWxhdGlvbiBpbiBodW1hbiBjYW5jZXIgY2VsbHMgYW5kIG11cmluZSBlbWJyeW9uaWMgc3RlbSBjZWxsczsgTW9sLiBDYW5jZXIgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODUxLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnZlcnRlZCByZXBlYXQgc3RydWN0dXJlIG9mIHRoZSBodW1hbiBnZW5vbWU6IHRoZSBYLWNocm9tb3NvbWUgY29udGFpbnMgYSBwcmVwb25kZXJhbmNlIG9mIGxhcmdlLCBoaWdobHkgaG9tb2xvZ291cyBpbnZlcnRlZCByZXBlYXRzIHRoYXQgY29udGFpbiB0ZXN0ZXMgZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTg2MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gYW5kIFNFVERCMS9IM0s5bWUzIHJlZ3VsYXRlIHByZWRvbWluYW50bHkgZGlzdGluY3Qgc2V0cyBvZiBnZW5lcywgcmV0cm9lbGVtZW50cywgYW5kIGNoaW1lcmljIHRyYW5zY3JpcHRzIGluIG1FU0NzOyBDZWxsIFN0ZW0gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Njc2LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYW5jZXIvdGVzdGlzIGFudGlnZW5zLCBnYW1ldG9nZW5lc2lzIGFuZCBjYW5jZXI7IE5hdC4gUmV2LiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTYxNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWZmeSBleG9uIHRpc3N1ZXM6IGV4b24gbGV2ZWxzIGluIG5vcm1hbCB0aXNzdWVzIGluIGh1bWFuLCBtb3VzZSBhbmQgcmF0OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI0NDItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lTWF0Y2hlcjogYSBncmFwaGljYWwgdXNlciBpbnRlcmZhY2UgZm9yIEROQSBzZXF1ZW5jZSBjb21wYXJpc29uOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC82LzU0OS8xMTYzMzUxL2RzdDAzMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVG8gdW5kZXJzdGFuZCB0aGUgZXBpZ2VuZXRpYyByZWd1bGF0aW9uIHJlcXVpcmVkIGZvciBnZXJtIGNlbGwtc3BlY2lmaWMgZ2VuZSBleHByZXNzaW9uIGluIHRoZSBtb3VzZSwgd2UgYW5hbHlzZWQgRE5BIG1ldGh5bGF0aW9uIHByb2ZpbGVzIG9mIGRldmVsb3AiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQxOTUyOCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTqnEakHkxcFo6E+dYS2x+1R1w0wA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f549%2f419528" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f549%2f419528" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16367" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&amp;Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/6">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6212921">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6212921">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6212923">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6212923">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6212926">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6212926">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6212941">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6212941">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6212992">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6212992">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6213001">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6213001">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6213003">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6213003">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6213005">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6213005">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6213007">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6213007">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/6/537/419314">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/6/567/419919">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rieko Ikeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life and Environmental Sciences</div>, <div class="insititution">University of Tsukuba</div>, <div class="addr-line">Ibaraki 305-8572</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rieko+Ikeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikeda R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikeda R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rieko Ikeda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirosuke Shiura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirosuke+Shiura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiura H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiura H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirosuke Shiura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Numata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Numata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Numata K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Numata K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Koji Numata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michihiko Sugimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michihiko+Sugimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugimoto M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugimoto M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michihiko Sugimoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayo Kondo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayo+Kondo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kondo M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kondo M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masayo Kondo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nathan Mise</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Environmental Preventive Medicine</div>, <div class="insititution">Jichi Medical University</div>, <div class="addr-line">Tochigi 329-0431</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nathan+Mise">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mise N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mise N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nathan Mise</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masako Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Genetics (Computational Genetics)</div>, <div class="insititution">Albert Einstein College of Medicine</div>, <div class="addr-line">Bronx, NY 10461</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masako+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masako Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John M. Greally</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Genetics (Computational Genetics)</div>, <div class="insititution">Albert Einstein College of Medicine</div>, <div class="addr-line">Bronx, NY 10461</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+M.+Greally">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Greally J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Greally J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">John M. Greally</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kuniya Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Technology and Development Team for Mammalian Genome Dynamics</div>, <div class="insititution">RIKEN BioResource Center</div>, <div class="addr-line">3-1-1 Koyadai, Tsukuba City, Ibaraki 305-0074</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life and Environmental Sciences</div>, <div class="insititution">University of Tsukuba</div>, <div class="addr-line">Ibaraki 305-8572</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 29-836-9198. Fax. +81 29-836-9199. Email: <a href="mailto:abe@rtc.riken.jp" target="_blank">abe@rtc.riken.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kuniya+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kuniya Abe</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 549–565, <a href='https://doi.org/10.1093/dnares/dst030'>https://doi.org/10.1093/dnares/dst030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 July 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 April 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 June 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="419528" href="/dnaresearch/article-pdf/20/6/549/1163351/dst030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rieko Ikeda, Hirosuke Shiura, Koji Numata, Michihiko Sugimoto, Masayo Kondo, Nathan Mise, Masako Suzuki, John M. Greally, Kuniya Abe;  Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome, <em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 549–565, <a href='https://doi.org/10.1093/dnares/dst030'>https://doi.org/10.1093/dnares/dst030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/419528?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/419528?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Large%2C%20Male%20Germ%20Cell-Specific%20Hypomethylated%20DNA%20Domains%20With%20Unique%20Genomic%20and%20Epigenomic%20Features%20on%20the%20Mouse%20X%20Chromosome&amp;publicationDate=2013-07-15&amp;volumeNum=20&amp;issueNum=6&amp;author=Ikeda%2C%20Rieko%3B%20Shiura%2C%20Hirosuke&amp;startPage=549&amp;endPage=565&amp;contentId=10.1093%2Fdnares%2Fdst030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f549%2f419528" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6212921 id="6212921" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>To understand the epigenetic regulation required for germ cell-specific gene expression in the mouse, we analysed DNA methylation profiles of developing germ cells using a microarray-based assay adapted for a small number of cells. The analysis revealed differentially methylated sites between cell types tested. Here, we focused on a group of genomic sequences hypomethylated specifically in germline cells as candidate regions involved in the epigenetic regulation of germline gene expression. These hypomethylated sequences tend to be clustered, forming large (10 kb to ∼9 Mb) genomic domains, particularly on the X chromosome of male germ cells. Most of these regions, designated here as large hypomethylated domains (LoDs), correspond to segmentally duplicated regions that contain gene families showing germ cell- or testis-specific expression, including cancer testis antigen genes. We found an inverse correlation between DNA methylation level and expression of genes in these domains. Most LoDs appear to be enriched with H3 lysine 9 dimethylation, usually regarded as a repressive histone modification, although some LoD genes can be expressed in male germ cells. It thus appears that such a unique epigenomic state associated with the LoDs may constitute a basis for the specific expression of genes contained in these genomic domains.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;primordial germ cell&quot;">primordial germ cell</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenome">epigenome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="reprogramming">reprogramming</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cancer testis antigen&quot;">cancer testis antigen</a></div>                            <h2 scrollto-destination=6212923 id="6212923" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Among the many cells that constitute an organism, only the germ cell can transmit its genetic information to the next generation. To achieve this vital function, germ cells must possess a distinctive gene transcription programme and epigenomic features unique to this cell lineage. It is accepted that, during development, germ cells undergo marked changes in their global epigenetic status, termed ‘epigenetic reprogramming’.<sup><a reveal-id="DST030C1" data-open="DST030C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST030C2" data-open="DST030C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, details of the epigenetic changes and their relationship to the establishment of germ cell-specific gene expression have been poorly characterized, possibly for technical reasons. The number of developing primordial germ cells (PGCs) in embryos is limited,<sup><a reveal-id="DST030C3" data-open="DST030C3" class="link link-ref link-reveal xref-bibr">3</a></sup> restricting the conventional analysis of epigenetic status. Most of the genome-scale studies on DNA methylation have focused on CpG islands (CGIs) or gene promoters (e.g. Borgel <em>et al.</em><sup><a reveal-id="DST030C4" data-open="DST030C4" class="link link-ref link-reveal xref-bibr">4</a></sup>). Currently, the importance of DNA methylation of other parts of genes, non-genic or intergenic regions for establishing gene expression and the nuclear organization of chromatin is increasingly recognized.<sup><a reveal-id="DST030C5  DST030C7" data-open="DST030C5  DST030C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> Although recent research has advanced our understanding of the PGC epigenome,<sup><a reveal-id="DST030C8  DST030C11" data-open="DST030C8  DST030C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> further studies are still required to gain more detailed information on epigenomic features of germline cells and their involvement in defining germ cell-specific gene expression.</p><p>In this study, we used a proven method of DNA methylation analysis called the HELP assay (<em>Hpa</em>II tiny fragment enrichment by ligation-mediated polymerase chain reaction (PCR)).<sup><a reveal-id="DST030C12  DST030C14" data-open="DST030C12  DST030C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> Because this method uses linker-mediated PCR, it can be adapted for the small-scale analysis of developing germ cells. Oda <em>et al.</em><sup><a reveal-id="DST030C14" data-open="DST030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> developed an improved version of the method, nanoHELP. Here, we have fine-tuned the protocol further. This modified nanoHELP method provides a global analysis of the DNA methylation status of CCGG sites using only a subnanogram (≥0.5 ng) quantity of genomic DNA. The custom-made genomic microarray used in this study is unusual in that the CCGG sites of the intergenic regions as well as the promoters and gene bodies of the RefSeq genes could be tested. This HELP microarray may provide new information about previously unexplored parts of the germ cell epigenome. We applied this method to analyse DNA methylation in the mouse X chromosome. We reasoned that epigenomic features specific to germ cells can be found by focusing on the X chromosome, because the X chromosome carries many germ cell-expressed genes<sup><a reveal-id="DST030C15" data-open="DST030C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST030C16" data-open="DST030C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and undergoes major epigenetic changes (e.g. X chromosome reactivation) during germ cell development.<sup><a reveal-id="DST030C17" data-open="DST030C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In this analysis, we found for the first time a group of sequences that are specifically hypomethylated on the X chromosome of male germline cells. These sequences form relatively large genomic domains that harbour gene families displaying specific expression in germ cells. We term these regions as large hypomethylated domains (LoDs). LoDs have not been detected in previous studies, including recent whole-genome bisulphite sequence analyses,<sup><a reveal-id="DST030C10" data-open="DST030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> probably because mapping of bisulphite-converted short sequence reads onto locally duplicated regions such as LoDs is technically challenging. In contrast, the experimental design of the HELP assay, which involved removal of the potentially confounding effect of copy number difference<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and inclusion of a probe design that selects unique sequences for hybridization, was effective in finding LoDs. Interestingly, many genes with homology to human cancer testis antigen (CTA) genes are contained within the LoDs. CTA genes are normally expressed only in the germline, and are also expressed in some tumour cell types.<sup><a reveal-id="DST030C18" data-open="DST030C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The results presented in this study may shed light on the epigenetic basis for the germline gene expression programme and its relationship with oncogenesis.</p>                            <h2 scrollto-destination=6212926 id="6212926" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6212927 id="6212927" class="section-title" data-legacy-id=s2a>2.1. Sample preparation and purification of DNA and RNA</h3>
<p>TMA5 cells are male embryonic stem (ES) cells derived from the 129/Sv mouse.<sup><a reveal-id="DST030C19" data-open="DST030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The female ES#5 line was from F1 hybrid mice between TgN(deGFP)20<sup>Imeg</sup> (RBRC No. 00822) and MSM/Ms (RBRC No. 00209). The embryonic germ (EG) cell lines used in this study were TMA55G (male) and TMA58G (female).<sup><a reveal-id="DST030C19" data-open="DST030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> These ES and EG cell lines were cultured, as described previously.<sup><a reveal-id="DST030C20" data-open="DST030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The <em>Oct3/4</em>–<em>GFP</em> transgenic mouse line TgN(deGFP)18<sup>Imeg</sup> (RBRC No. 00821)<sup><a reveal-id="DST030C21" data-open="DST030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> was used to collect PGCs from developing mouse embryos, as described previously.<sup><a reveal-id="DST030C20" data-open="DST030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Germline stem (GS) cells were obtained from the RIKEN Cell Bank (RCB1968) and were cultured on a feeder layer as described.<sup><a reveal-id="DST030C22" data-open="DST030C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Germ cells expressing the <em>Venus</em> reporter were purified by fluorescence-activated cell sorting (FACS) from adult testis cells of the <em>Mvh-Venus</em> bacterial artificial chromosome transgenic mouse line, Tg(Mvh-Venus)1Rbrc (Mise and Abe, unpublished results).<sup><a reveal-id="DST030C23" data-open="DST030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> All animal experiments were approved by the Institutional Animal Experiment Committee of the RIKEN BioResource Center. DNA and RNA were extracted simultaneously from the same samples using an AllPrep DNA/RNA Micro Kit (Qiagen, Hilden, Germany). The quality of RNA samples was checked using an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA).</p>                            <h3 scrollto-destination=6212929 id="6212929" class="section-title" data-legacy-id=s2b>2.2. Gene expression profiling</h3>
<p>A 44K custom microarray<sup><a reveal-id="DST030C24" data-open="DST030C24" class="link link-ref link-reveal xref-bibr">24</a></sup> was used for gene expression profiling throughout this study. This custom array covers all the known protein-coding genes as well as expression sequence tags derived from PGC cDNA libraries (Abe, unpublished) and was manufactured by Agilent Technologies. Total RNA was labelled with Cy3-CTP with a Quick Amp Labeling Kit (Agilent Technologies). Hybridization was performed according to the protocol suggested by the supplier. Hybridized slides were scanned using a microarray scanner (Agilent Technologies), and the signals were processed with the Feature Extraction software ver. 10.5.1.1 (Agilent Technologies). The processed signal data were normalized and analysed by the Gene Spring GX11.5 software (Agilent Technologies). The microarray experiments were conducted using biologically duplicated samples.</p>                            <h3 scrollto-destination=6212931 id="6212931" class="section-title" data-legacy-id=s2c>2.3. Modified nanoHELP: linker-mediated amplification and hybridization</h3>
<p>The nanoHELP assay, a microarray-based DNA methylation analysis, was performed according to our previous reports<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST030C14" data-open="DST030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> with modifications. Briefly, genomic DNA (0.5–2 ng) was digested by <em>Hpa</em>II or <em>Msp</em>I in 100 μl of reaction mixture at 37°C overnight. This was followed by DNA purification with the MinElute Reaction Cleanup Kit (Qiagen), and the digested DNA was ligated to linker adapters, NHpaII12/NhpaII24 and JHpaII12/JhpaII24<sup><a reveal-id="DST030C14" data-open="DST030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> overnight at 16°C. After removing the linker adapters, the ligated DNA was added to a total of 50-μl PCR reaction mixture containing 1.5 μl each of 20 μM primer (NHpaII24, 5′-GCAACTGTGCTATCCGAGGGAAGC-3′; JHpaII24, 5′-CGACGTCGACTATCCATGAACAGC-3′), 10 μl of 5 M betaine (Sigma-Aldrich), 200 μM of dNTPs and 2.5 units of ExTaq DNA polymerase (TaKaRa Bio, Inc., Otsu, Japan) in a buffer supplied by the manufacturer. The mixture was heated at 72°C for 10 min and subjected to PCR amplification with the following parameters: 15 cycles at 95°C for 30 s and 72°C for 3 min, with a final extension at 72°C for 10 min. After the first round of amplification, one-tenth of the volume of the reaction was added to a fresh PCR reaction mix containing the same primers, and amplified for an additional 10–15 cycles<sup><a reveal-id="DST030C14" data-open="DST030C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST030C25" data-open="DST030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> with the same PCR parameters as described above. The PCR products were purified using the MinElute Kit (Qiagen). An additional column washing step with 750 μl of 35% guanidine hydrochloride (Nacalai Tesque, Inc., Kyoto, Japan) solution was performed to remove the residual primer–adapters. The amplified DNA originally digested with <em>Hpa</em>II was labelled with Cy5-labelled Random 9-mers (TriLink Biotechnologies, San Diego, CA, USA), and the <em>Msp</em>I-digested DNA was labelled with Cy3-Random 9-mers (TriLink Biotechnologies). The labelled DNAs were mixed and hybridized with a custom microarray (Roche NimbleGen, Madison, WI, USA) using a NimbleGen Array Hybridization Kit (<a class="link link-uri" href="http://www.nimblegen.com/products/lit/lit.html">http://www.nimblegen.com/products/lit/lit.html</a>). After washing with the NimbleGen Array wash kit, the microarrays were scanned on an Agilent Technologies Scanner G2505C with a setting of 5-μm resolution. The HELP array experiments were performed using biological replicates. The raw data were processed using the NimbleScan 2.4 data extraction software (NimbleGen) to obtain the processed log<sub>2</sub> (Cy5/Cy3) ratio data.</p><p>Our previously published ChIP-on-chip data for cumulus cells<sup><a reveal-id="DST030C26" data-open="DST030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> were reanalysed and used. The ChIP-on-chip experiment using GS cells was performed as described in the same paper.</p>                            <h3 scrollto-destination=6212935 id="6212935" class="section-title" data-legacy-id=s2d>2.4. Microarray design</h3>
<p>The microarrays were designed to represent restriction fragments with 5′-CCGG restriction sites in a size range of 200–2000 bp (=CCGG segments) on mouse Chromosome 7 and the X chromosome. Ten 50-mer oligonucleotide probes were designed from unique sequences in each CCGG segment, avoiding repeat-masked regions and sequence ambiguities. Probe sequences were selected using a score-based selection algorithm, as described.<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Detailed information for the coverage of genomic regions on each chromosome and annotations of the CCGG segments are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Information about the positions of probes, <em>M</em>-values obtained from different samples and k-means cluster number are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>, and gff files of the HELP array data are available at our web site (<a class="link link-uri" href="http://www.brc.riken.go.jp/lab/mcd/mcd2/protocol/nanoHELP.html">http://www.brc.riken.go.jp/lab/mcd/mcd2/protocol/nanoHELP.html</a>).</p>                            <h3 scrollto-destination=6212937 id="6212937" class="section-title" data-legacy-id=s2e>2.5. Data analysis</h3>
<p>The steps in the HELP data analysis are shown schematically in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. Briefly, hybridization signal noise is first removed from the processed data by cutting off the values in the range of random sequence probes. In our microarray, 10 oligonucleotide probes are normally assigned to each CCGG segment. The median signal intensity of the 10 probes is calculated and used to define the segment's signal intensity. Using the median signal values, the <em>Hpa</em>II/<em>Msp</em>I ratio is then calculated for each CCGG segment and converted to a log<sub>2</sub> value to obtain the <em>M</em>-value. After normalization of the microarray ratio data, hypomethylated and hypermethylated segments are distinguished using an R script (<a class="link link-uri" href="http://www.r-project.org/">http://www.r-project.org/</a>) that determines the threshold values based on a binarization method.<sup><a reveal-id="DST030C27" data-open="DST030C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The marginal width of the threshold is calculated using the Mahalanobis distance.<sup><a reveal-id="DST030C28" data-open="DST030C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The log<sub>2</sub> value at the threshold is set as 0, so that unmethylated segments have a value of ≥0 and methylated segments have a negative value (&lt;0). For interarray normalization of the <em>Hpa</em>II/<em>Msp</em>I ratio, the threshold value of each array data set is scaled to 0. All the bioinformatic analysis described here is based on the UCSC mm8 genome assembly (<a class="link link-uri" href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>).</p>                            <h3 scrollto-destination=6212939 id="6212939" class="section-title" data-legacy-id=s2f>2.6. Accession number</h3>
<p>Gene expression microarray data and the HELP array data are available at the Gene Expression Omnibus database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) (Accession number GSE39895).</p>                            <h2 scrollto-destination=6212941 id="6212941" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6212942 id="6212942" class="section-title" data-legacy-id=s3a>3.1. DNA methylation analysis with subnanogram amounts of genomic DNA</h3>
<p>For epigenomic analyses of developing cells, materials can be very limited in quantity, precluding conventional analytical techniques. One of our goals was to describe comprehensively the epigenomic changes during the development of early embryos and germ cells in the mouse. Towards this goal, we use the HELP assay,<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> a proven, microarray-based method, for the analysis of DNA methylation.<sup><a reveal-id="DST030C13" data-open="DST030C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The original HELP protocol requires 10 μg of genomic DNA as the starting material,<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> but Oda <em>et al.</em><sup><a reveal-id="DST030C14" data-open="DST030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> established an improved version of the method, nanoHELP, for the analysis of a limited amount of starting DNA. Here, we have fine-tuned the protocol further for the analysis of 0.5–2 ng of starting material. The details of this method are described in Section 2, and the flow of the data analysis is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>.</p><p>The HELP assay is a microarray-based method that detects the subset of unmethylated <em>Hpa</em>II fragments in the genome and uses the corresponding, methylation-insensitive <em>Msp</em>I representations as a control. The <em>M</em>-value, an index of the methylation level, is calculated as log<sub>2</sub>(<em>Hpa</em>II signal/<em>Msp</em>I signal) as described in Section 2: unmethylated segment has a value of ≥0 and methylated segment has a negative value of &lt;0. As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2A–D</a>, the modified nanoHELP assay generated data having good correlations with the data obtained by the original protocol. To validate these results, bisulphite pyrosequencing analysis of six CpG sites was performed as described.<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The results showed that the modified nanoHELP assay could generate reliable data (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2E</a>).</p>                            <h3 scrollto-destination=6212945 id="6212945" class="section-title" data-legacy-id=s3b>3.2. Custom HELP microarray used in this study</h3>
<p>The number of restriction sites for <em>Hpa</em>II, 5′-CCGG, in the mouse genome (UCSC mm8) is 1 588 546, covering ∼7.5% of the total CpG dinucleotides in the genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1A</a>). The CCGG sites are distributed almost evenly over the mouse genome and do not show an apparent bias to a particular genomic context. Thus, the use of CCGG sites is suitable for obtaining a chromosome-wide view of CpG methylation profiles.</p><p>In this experiment, we designed a custom microarray harbouring 382 018 oligoprobes. We first selected <em>Hpa</em>II or <em>Msp</em>I fragments (designated here as CCGG segments) with a size range of 200–2000 bp, mostly from mouse Chromosomes 7 and X, and designed 10 unique sequence probes of 50 nucleotides per CCGG segment. The custom microarray can detect 22 128 CCGG segments on Chromosome 7 and 14 472 on the X chromosome, which correspond to 46 and 39% of the total segments on each chromosome, respectively. The CCGG segments were selected in an unbiased fashion except for a part of the Chromosome 7 and a small number of segments associated with some germ cell-related genes on Chromosomes 6, 11 and 12 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1B</a>). <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1C and D</a> describes the categorization of the CCGG segments based on the genome annotations. About 47% of the segments map to intergenic regions, 5% to promoter regions and 46% to bodies of RefSeq genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1C</a>). About 75% of the X-linked RefSeq genes are covered by this HELP microarray (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1D</a>). Because most of the CCGG segments in CGIs are &lt;200 bp, ∼1% of CGIs annotated in the UCSC mm8 genome assembly can be assayed by this array (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1E</a>).</p><p>CGIs and gene promoter regions have been the main targets in most DNA methylation studies. However, the importance of DNA methylation in genomic regions outside the promoters is becoming increasingly apparent.<sup><a reveal-id="DST030C5" data-open="DST030C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST030C6" data-open="DST030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> It is expected that this microarray method should be appropriate for the analysis of previously unexplored and potentially informative parts of the genome.</p>                            <h3 scrollto-destination=6212952 id="6212952" class="section-title" data-legacy-id=s3c>3.3. Analysis of DNA methylation profiles of stem cells and germline cells</h3>
<p>We performed DNA methylation profiling of the following samples: ES cells from male and female blastocysts, male and female EG cells established from PGCs at embryonic day 12.5 (E12.5), GS cells derived from spermatogonia<sup><a reveal-id="DST030C22" data-open="DST030C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and male and female PGCs purified from <em>Oct3/4–GFP</em> transgenic embryos<sup><a reveal-id="DST030C20" data-open="DST030C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST030C21" data-open="DST030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> in various stages by FACS. PGCs were isolated from male and female E10.5, E13.5 and E17.5 embryos. PGCs have not entered the gonads at E10.5, and that are colonized within the gonads in E13.5 embryos. At E17.5, PGCs are subjected to mitotic arrest in male gonads, and female PGCs are arrested in the early phase of meiosis.<sup><a reveal-id="DST030C3" data-open="DST030C3" class="link link-ref link-reveal xref-bibr">3</a></sup> We isolated germ cells from newborn ovary and testis. Whole adult testis, thymus and brain were isolated from male mice and used for the analysis. Germ cells in the adult testis were purified by FACS from <em>Mvh</em> (<em>mouse Vasa homolog</em>)-<em>Venus</em> transgenic mouse (Mise and Abe, unpublished results).<sup><a reveal-id="DST030C23" data-open="DST030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Gene expression profiling of all samples was conducted using our custom 44K microarray.</p><p>Figure <a reveal-id="DST030F1" data-open="DST030F1" class="link link-reveal link-table xref-fig">1</a> shows the results of principal component analysis (PCA) and hierarchical cluster analysis of the DNA methylation profiles and the gene expression profiles. In this comparison, pluripotent stem cells (i.e. ES and EG cells) and PGCs from various stages show similar but distinct expression profiles; ES and EG cells are positioned more closely (blue circle) relative to PGCs (red circle) (Fig. <a reveal-id="DST030F1" data-open="DST030F1" class="link link-reveal link-table xref-fig">1</a>A and C). This result confirms our previous findings that PGCs possess a distinct transcription programme from ES cells, although both share the expression of common ‘signature genes’.<sup><a reveal-id="DST030C20" data-open="DST030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In contrast, analysis of the DNA methylation profiles showed the differences between samples more clearly. PGC samples could be classified into two groups: one comprising female PGCs and early male PGCs (i.e. E10.5 and E13.5 in the red circle) and E17.5 and P0.5 male germ cells that formed a cluster together with GS cells and testis (green circle) (Fig. <a reveal-id="DST030F1" data-open="DST030F1" class="link link-reveal link-table xref-fig">1</a>B and D). Male PGCs in different stages appeared to be more distantly related to each other than to female PGCs, suggesting that the DNA methylation profiles change more drastically during male PGC development. These results suggest that cell types can be classified by their DNA methylation profiles and that, in some cases, DNA methylation profiling can display differences in the cellular state more effectively. </p>                        <a id="6212958" scrollto-destination="6212958"></a>
<div id="DST030F1" data-id="DST030F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03001.png?Expires=2147483647&Signature=YzgYIdRvaDxUVmSYU9dbtn~og3HwweJKysoTyZEEFDRPzq1OWGuVXP9zLE3XC-pfcY1XFhUrrCindYeCILjxlKNM0eC9G2upDzM0Hhdyp2s6rGl2Kz~NCAVmNqfJ2EG2UjeAHy2m-V~du1~TILg4YvvuwnioomS2d1KRvsh3oloYhngTTG4klRN2QVGfijOnue~pXTWs561vk04Y9TlCNG~SZQpg-8wUiR7U3m02faa-a6UH0kIfosTrG~jHkEEjMRClNM7NnfXV0WLPv-0i6YOjARpUrEswdpGZcCUESoJ4xgRXSbC0E4ig-jl8w-FJQp5EoKkdTwf6uNlDKXrDyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Profiling of gene expression and DNA methylation in germ cells and stem cells. (A) PCA of the expression profiles of germ cells, stem cells and adult organs. ES_m (male ES cells), ES_f (female ES cells), EG_m (male EG cells), EG_f (female EG cells), 10.5m (PGCs from male E10.5 embryos), 10.5f (female E10.5 PGCs), 13.5m (male E13.5 PGCs), 13.5f (female E13.5 PGCs), 17.5m (male E17.5 PGCs), 17.5f (female E17.5 PGCs), P0.5m (spermatogonia from P0.5 neonates), P0.5f (oocytes from P0.5 neonates) and GS cells. Testis, thymus and brain were isolated from male adult mice. (B) PCA analysis of DNA methylation profiles of germ cells, stem cells and adult organs. (C) Hierarchical clustering of gene expression profiles. (D) Hierarchical clustering of DNA methylation profiles." path-from-xml="dst03001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212958/dst03001.tiff" path-from-xml="dst03001.tiff" target="_blank">View large</a><a section="6212958" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03001.png?Expires=2147483647&Signature=LMpMINjfQ57rsekDFjUSuZUMHyAfmD2HZMKFmagYBe-xEZhKKysNbtJB8qI4DNtQjrQazrSsaNPYkw1dqwdISBpjQm5ycGFf9I5GftXwYh7Xec0skbeNQLSCLs--hWgZO~dhq68etXwDIsIIumMtoOvt~~kNAkp35zYiYhfqqtbvlIqGl2v0DPtan8Xs7zUNyLVc66CbenK-0gq6uyulr5ZuIYUDPwzOVLyaLBTYWbhFUoutaa8q~R9z0kC8rcHw6ZCfg~j~3uxjwUaw5bfD-vrhptX~ThLqVVjAUohdaSmvWjvC5w1tH1hxDd34Me3pkAap~kTNu60tURB3ah54WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212958&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Profiling of gene expression and DNA methylation in germ cells and stem cells. (A) PCA of the expression profiles of germ cells, stem cells and adult organs. ES_m (male ES cells), ES_f (female ES cells), EG_m (male EG cells), EG_f (female EG cells), 10.5m (PGCs from male E10.5 embryos), 10.5f (female E10.5 PGCs), 13.5m (male E13.5 PGCs), 13.5f (female E13.5 PGCs), 17.5m (male E17.5 PGCs), 17.5f (female E17.5 PGCs), P0.5m (spermatogonia from P0.5 neonates), P0.5f (oocytes from P0.5 neonates) and GS cells. Testis, thymus and brain were isolated from male adult mice. (B) PCA analysis of DNA methylation profiles of germ cells, stem cells and adult organs. (C) Hierarchical clustering of gene expression profiles. (D) Hierarchical clustering of DNA methylation profiles.</p></div></div><div content-id="DST030F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03001.png?Expires=2147483647&Signature=YzgYIdRvaDxUVmSYU9dbtn~og3HwweJKysoTyZEEFDRPzq1OWGuVXP9zLE3XC-pfcY1XFhUrrCindYeCILjxlKNM0eC9G2upDzM0Hhdyp2s6rGl2Kz~NCAVmNqfJ2EG2UjeAHy2m-V~du1~TILg4YvvuwnioomS2d1KRvsh3oloYhngTTG4klRN2QVGfijOnue~pXTWs561vk04Y9TlCNG~SZQpg-8wUiR7U3m02faa-a6UH0kIfosTrG~jHkEEjMRClNM7NnfXV0WLPv-0i6YOjARpUrEswdpGZcCUESoJ4xgRXSbC0E4ig-jl8w-FJQp5EoKkdTwf6uNlDKXrDyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Profiling of gene expression and DNA methylation in germ cells and stem cells. (A) PCA of the expression profiles of germ cells, stem cells and adult organs. ES_m (male ES cells), ES_f (female ES cells), EG_m (male EG cells), EG_f (female EG cells), 10.5m (PGCs from male E10.5 embryos), 10.5f (female E10.5 PGCs), 13.5m (male E13.5 PGCs), 13.5f (female E13.5 PGCs), 17.5m (male E17.5 PGCs), 17.5f (female E17.5 PGCs), P0.5m (spermatogonia from P0.5 neonates), P0.5f (oocytes from P0.5 neonates) and GS cells. Testis, thymus and brain were isolated from male adult mice. (B) PCA analysis of DNA methylation profiles of germ cells, stem cells and adult organs. (C) Hierarchical clustering of gene expression profiles. (D) Hierarchical clustering of DNA methylation profiles." path-from-xml="dst03001.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212958/dst03001.tiff" path-from-xml="dst03001.tiff" target="_blank">View large</a><a section="6212958" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03001.png?Expires=2147483647&Signature=LMpMINjfQ57rsekDFjUSuZUMHyAfmD2HZMKFmagYBe-xEZhKKysNbtJB8qI4DNtQjrQazrSsaNPYkw1dqwdISBpjQm5ycGFf9I5GftXwYh7Xec0skbeNQLSCLs--hWgZO~dhq68etXwDIsIIumMtoOvt~~kNAkp35zYiYhfqqtbvlIqGl2v0DPtan8Xs7zUNyLVc66CbenK-0gq6uyulr5ZuIYUDPwzOVLyaLBTYWbhFUoutaa8q~R9z0kC8rcHw6ZCfg~j~3uxjwUaw5bfD-vrhptX~ThLqVVjAUohdaSmvWjvC5w1tH1hxDd34Me3pkAap~kTNu60tURB3ah54WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212958&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03001.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Profiling of gene expression and DNA methylation in germ cells and stem cells. (A) PCA of the expression profiles of germ cells, stem cells and adult organs. ES_m (male ES cells), ES_f (female ES cells), EG_m (male EG cells), EG_f (female EG cells), 10.5m (PGCs from male E10.5 embryos), 10.5f (female E10.5 PGCs), 13.5m (male E13.5 PGCs), 13.5f (female E13.5 PGCs), 17.5m (male E17.5 PGCs), 17.5f (female E17.5 PGCs), P0.5m (spermatogonia from P0.5 neonates), P0.5f (oocytes from P0.5 neonates) and GS cells. Testis, thymus and brain were isolated from male adult mice. (B) PCA analysis of DNA methylation profiles of germ cells, stem cells and adult organs. (C) Hierarchical clustering of gene expression profiles. (D) Hierarchical clustering of DNA methylation profiles.</p></div></div>                            <h3 scrollto-destination=6212959 id="6212959" class="section-title" data-legacy-id=s3d>3.4. k-means cluster analysis of DNA methylation profiles to visualize cell type-specific differentially methylated regions</h3>
<p>To visualize differences in the DNA methylation profiles of the samples tested, non-hierarchical k-means analysis (<em>k</em> = 12) was performed using the data obtained from the 28 217 informative CCGG segments; the result is shown as a heat map in Fig. <a reveal-id="DST030F2" data-open="DST030F2" class="link link-reveal link-table xref-fig">2</a>A. One of the most conspicuous trends was that the genomes of the PGCs examined are mostly hypomethylated except for male E17.5 PGCs. A box plot of the <em>M</em>-value for each sample is shown in Fig. <a reveal-id="DST030F2" data-open="DST030F2" class="link link-reveal link-table xref-fig">2</a>B. Global levels of DNA methylation are lower in the E10.5 PGC genome than in ES and EG cells, and the levels are even lower at E13.5. At E17.5, the methylation level of male PGCs is increased, whereas female PGCs maintain a hypomethylated status similar to that at E10.5 or E13.5. The difference in the methylation level between male and female germ cells is most prominent in neonates: male spermatogonia have a highly methylated genome. GS cells possess a similar DNA methylation profile to that of P0.5 spermatogonia (<em>R</em> = 0.80). Epigenetic features of GS cells have not been reported to date, and this result suggests that GS cells should provide a valuable <em>in vitro</em> model for epigenetic studies of spermatogonial cells. Adult testis, which comprises both germ and somatic cells, has a slightly lower DNA methylation level relative to spermatogonia (P0.5 male) and GS cells (Fig. <a reveal-id="DST030F2" data-open="DST030F2" class="link link-reveal link-table xref-fig">2</a>B). </p>                        <a id="6212961" scrollto-destination="6212961"></a>
<div id="DST030F2" data-id="DST030F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03002.png?Expires=2147483647&Signature=GsnWJegC-Cfa9Iol4NZkwfiKYa7vXXyCkCrbt7d6c9jzbyHP9ekhHZajlIAl7VMuUzNfPPFcPaNEY89zxyRrBaLVCpXE1nbC7lHYkVQ8J-68qfnGwkkaRImCwotWZc80VEVnVUHTw3MdJYvm1Byl543PNNFu~wyq~fAqfzmSeL585ZUTgI8nj3KQeq0W0nixCCtIIAaNi02o3D63dG7yV~aj2RJj6whdtmzLuxbAuUQsBKraij06zAZ1kEy19fXePjW25BFt4yCVfvHSDqDYjA5h4bEcSYOR4Gc7SIvfWNr3HsENnr9vtZzXv7OqPy944boW1Gj6joeJAXzwibrL5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation dynamics during germ cell development. (A) k-means clustering of DNA methylation profiles of germ cells, stem cells and adult organs. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, M-value = 6.00; highly methylated segments in dark red, M-value = –6.00). (B) Changes in global DNA methylation levels during PGC development. The M-value was calculated for each sample and is shown as a box plot. The bottom and top of the boxes are the 25th and 75th percentile, respectively." path-from-xml="dst03002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212961/dst03002.tiff" path-from-xml="dst03002.tiff" target="_blank">View large</a><a section="6212961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03002.png?Expires=2147483647&Signature=TdmZ9LFbfemCxIFFztGExouR4kuXrPRzurmFRA3KkXIGnI6OO8zAPRzPPgND-6J5y9GqiXfVAzXRdxij~VoH4yKgqIY5XBXtImHAWAEqXhgjk8U7KXGjO0FME1JGL-nDCL9HSW-x0tcrr9g1vgLYbmvu-h~ESXIH1Aeg~DhsPbp31IMQJu0Z74bEUNGMtRiyiLFMdsk6rsc2agC2ut40DGZ0ajDHKNLBlEXCpNK-4RFH-CW~jXQ~7bP~Cm~CngZi2yFUHppjvJrtuvR9m3IRlTgv1DeFRDePmRugF2f2tkSkUCyzBNlbiRWWnHolPbeYUpX2EpqMYsVkfUyDglasIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212961&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation dynamics during germ cell development. (A) k-means clustering of DNA methylation profiles of germ cells, stem cells and adult organs. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, <em>M</em>-value = 6.00; highly methylated segments in dark red, <em>M</em>-value = –6.00). (B) Changes in global DNA methylation levels during PGC development. The <em>M</em>-value was calculated for each sample and is shown as a box plot. The bottom and top of the boxes are the 25th and 75th percentile, respectively.</p></div></div><div content-id="DST030F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03002.png?Expires=2147483647&Signature=GsnWJegC-Cfa9Iol4NZkwfiKYa7vXXyCkCrbt7d6c9jzbyHP9ekhHZajlIAl7VMuUzNfPPFcPaNEY89zxyRrBaLVCpXE1nbC7lHYkVQ8J-68qfnGwkkaRImCwotWZc80VEVnVUHTw3MdJYvm1Byl543PNNFu~wyq~fAqfzmSeL585ZUTgI8nj3KQeq0W0nixCCtIIAaNi02o3D63dG7yV~aj2RJj6whdtmzLuxbAuUQsBKraij06zAZ1kEy19fXePjW25BFt4yCVfvHSDqDYjA5h4bEcSYOR4Gc7SIvfWNr3HsENnr9vtZzXv7OqPy944boW1Gj6joeJAXzwibrL5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation dynamics during germ cell development. (A) k-means clustering of DNA methylation profiles of germ cells, stem cells and adult organs. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, M-value = 6.00; highly methylated segments in dark red, M-value = –6.00). (B) Changes in global DNA methylation levels during PGC development. The M-value was calculated for each sample and is shown as a box plot. The bottom and top of the boxes are the 25th and 75th percentile, respectively." path-from-xml="dst03002.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212961/dst03002.tiff" path-from-xml="dst03002.tiff" target="_blank">View large</a><a section="6212961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03002.png?Expires=2147483647&Signature=TdmZ9LFbfemCxIFFztGExouR4kuXrPRzurmFRA3KkXIGnI6OO8zAPRzPPgND-6J5y9GqiXfVAzXRdxij~VoH4yKgqIY5XBXtImHAWAEqXhgjk8U7KXGjO0FME1JGL-nDCL9HSW-x0tcrr9g1vgLYbmvu-h~ESXIH1Aeg~DhsPbp31IMQJu0Z74bEUNGMtRiyiLFMdsk6rsc2agC2ut40DGZ0ajDHKNLBlEXCpNK-4RFH-CW~jXQ~7bP~Cm~CngZi2yFUHppjvJrtuvR9m3IRlTgv1DeFRDePmRugF2f2tkSkUCyzBNlbiRWWnHolPbeYUpX2EpqMYsVkfUyDglasIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212961&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03002.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation dynamics during germ cell development. (A) k-means clustering of DNA methylation profiles of germ cells, stem cells and adult organs. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, <em>M</em>-value = 6.00; highly methylated segments in dark red, <em>M</em>-value = –6.00). (B) Changes in global DNA methylation levels during PGC development. The <em>M</em>-value was calculated for each sample and is shown as a box plot. The bottom and top of the boxes are the 25th and 75th percentile, respectively.</p></div></div><p>Although most of the PGC genomes are hypomethylated, genomic regions classified as Cluster 12 remain methylated at a level similar to that in the other cell types examined. The rest of the clusters showed some cell or tissue specificities in DNA methylation.</p>                            <h3 scrollto-destination=6212963 id="6212963" class="section-title" data-legacy-id=s3e>3.5. Characterization of germline-specific hypomethylated CCGG segments on the X chromosome</h3>
<p>The clusters were then characterized by examining their tissue specificities in DNA methylation patterns. We noticed that Cluster 4 comprises the segments hypomethylated only in PGCs, GS cells and the testis, whereas these segments are hypermethylated in ES and EG cells, and somatic organs. These putative germline-specific hypomethylated segments were characterized further. Although no particular GO terms are enriched, Cluster 4 contains genes expressed in the germ cells of testis such as <em>Xmr</em> (<em>Xlr</em>-related, meiosis regulated)<sup><a reveal-id="DST030C29" data-open="DST030C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST030C30" data-open="DST030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> or CTA genes; e.g. the <em>Mage</em> (melanoma antigen) gene family.<sup><a reveal-id="DST030C31" data-open="DST030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The number of Cluster 4 CCGG segments mapped onto the X chromosome is disproportionately high. There are 1004 segments in Cluster 4, and 715 (71.2%) are on the X chromosome, which has 14 472 CCGG segments in total. In contrast, 219 of Cluster 4 segments (21.8%) are on Chromosome 7, which carries 22 128 segments. Because genes expressed in germ cells or testis are known to be enriched on the X chromosome,<sup><a reveal-id="DST030C15" data-open="DST030C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST030C16" data-open="DST030C16" class="link link-ref link-reveal xref-bibr">16</a></sup> we focused on the Cluster 4 segments of the X chromosome as candidates involved in the epigenetic regulation of germ cell-specific gene expression.</p><p>As shown in Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3A–C</a>, we plotted the <em>M</em>-values of all the CCGG segments along the X chromosome using the data obtained from each sample (grey dots). To translate the <em>M</em>-value measurements into regions of equal <em>M</em>-value, we used a circular binary segmentation programme, which is used normally for comparative genomic hybridization analysis.<sup><a reveal-id="DST030C32" data-open="DST030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Using this programme, we drew lines (black horizontal lines) to show regions of equal <em>M</em>-value. By tracing the line, we could identify the genomic regions in which the <em>M</em>-value changes significantly from the flanking regions. The <em>M</em>-values of the segments belonging to Cluster 4 are overlaid as red dots. The distributions of the <em>M</em>-values in the DNA of somatic cells (i.e. brain and thymus) along the entire X chromosome are similar to each other: the average <em>M</em>-value is less than –1, with some local exceptions. The Cluster 4 dots are mapped even below the average line, indicating that, as expected, Cluster 4 segments are hypermethylated in both brain and thymus (Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a>). In ES and EG cells (Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3A</a>), the average <em>M</em>-values of the Cluster 4 segments do not change significantly along the X chromosome and are positioned below –1, suggesting that Cluster 4 segments are largely hypermethylated in the genomes of ES or EG cells. In sharp contrast, in E17.5 male PGC DNA, it appears that the average <em>M</em>-value line is often discontinuous, and that hypomethylated CCGG segments exist over relatively large, contiguous genomic regions (Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a>). For example, the average <em>M</em>-value of the segments within the ∼9 Mb genomic region harbouring the <em>Xmr</em> gene cluster (double-headed arrows) is close to 0, and Cluster 4 segments are enriched in this region. </p>                        <a id="6212966" scrollto-destination="6212966"></a>
<div id="DST030F3" data-id="DST030F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03003.png?Expires=2147483647&Signature=2UJFRpRnOBIqJzODpMVR8nO~oDstoTlAZtCBxO2c7bjW5M~lgKpwzw1z-bnUFhWsXbEeneIDGODQ8iqiGvy0sM2MpgJ~Sh~NKz-v98fF7~j7lNfD5giGgUiprJfCrFzFdF68FY3wUu167ByD45AchbsVhCq6VuHkj9CIuKNy4BcffK2ia7lbrq3DsoSn0zuVJuJRbymnUo5sl8l6ADd-7bDO3Z16vHLnNQPGpWuCZjBepSdvqxltd~U3qupzfOOOmTc2WbLAtCInoTRB8DN4p4dQ-LFo7kgrl5qin5oKBMM1E9BGdNm9iubTLIlix3uLuZkZb457v14aUbzDLEJkRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Discovery of large, contiguous genomic regions with low DNA methylation modifications in male germ cells. The methylation profiles of genomic DNA along the mouse X chromosome. Samples used for the analysis are indicated in each figure. The M-value of each CCGG segment obtained from the analysis of each sample DNA is plotted on the mouse X chromosome (grey dots). The y-axis represents the M-value; log2(HpaII/MspI). The black line was drawn using DNAcopy, a circular binary segmentation programme obtained from http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html.32 Red dots represent CCGG segments belonging to Cluster 4. Double-headed arrows indicate the position of the Xmr gene cluster." path-from-xml="dst03003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212966/dst03003.tiff" path-from-xml="dst03003.tiff" target="_blank">View large</a><a section="6212966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03003.png?Expires=2147483647&Signature=VIEjErEGRr2wZGNzgwt-SIxtLxr7K2Q5TFHkjZajow9TleJ3CfJ12xpya7itD2wJjjn0OUpZ4VZ76eOpzh0MoMefU50eSS~sTEXw-PHdL-53XcbkUoIGTp1RjGaRnxrayH68TnY34OwzYCwv4rZ7Zl9LKHCoEGLmFKpRKS~pb58dwvXjisiuh0eFcqNoA1Bp844Qb1V7UOuGS6Tslp1Vw7pW1rbzSeBIj4pUVfTb-y9XqRxjPEHIGQcW6wxPTPowx390sv8296htU0eouxyIE6sDy399cJNbWluc9LPC6R2Sn06TFlYMwDpbk879kWnJj8JyEbfHbosjUDuUUWX~XA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212966&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Discovery of large, contiguous genomic regions with low DNA methylation modifications in male germ cells. The methylation profiles of genomic DNA along the mouse X chromosome. Samples used for the analysis are indicated in each figure. The <em>M</em>-value of each CCGG segment obtained from the analysis of each sample DNA is plotted on the mouse X chromosome (grey dots). The <em>y</em>-axis represents the <em>M</em>-value; log<sub>2</sub>(<em>Hpa</em>II/<em>Msp</em>I). The black line was drawn using DNAcopy, a circular binary segmentation programme obtained from <a class="link link-uri" href="http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html">http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html</a>.<sup><a reveal-id="DST030C32" data-open="DST030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Red dots represent CCGG segments belonging to Cluster 4. Double-headed arrows indicate the position of the <em>Xmr</em> gene cluster.</p></div></div><div content-id="DST030F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03003.png?Expires=2147483647&Signature=2UJFRpRnOBIqJzODpMVR8nO~oDstoTlAZtCBxO2c7bjW5M~lgKpwzw1z-bnUFhWsXbEeneIDGODQ8iqiGvy0sM2MpgJ~Sh~NKz-v98fF7~j7lNfD5giGgUiprJfCrFzFdF68FY3wUu167ByD45AchbsVhCq6VuHkj9CIuKNy4BcffK2ia7lbrq3DsoSn0zuVJuJRbymnUo5sl8l6ADd-7bDO3Z16vHLnNQPGpWuCZjBepSdvqxltd~U3qupzfOOOmTc2WbLAtCInoTRB8DN4p4dQ-LFo7kgrl5qin5oKBMM1E9BGdNm9iubTLIlix3uLuZkZb457v14aUbzDLEJkRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Discovery of large, contiguous genomic regions with low DNA methylation modifications in male germ cells. The methylation profiles of genomic DNA along the mouse X chromosome. Samples used for the analysis are indicated in each figure. The M-value of each CCGG segment obtained from the analysis of each sample DNA is plotted on the mouse X chromosome (grey dots). The y-axis represents the M-value; log2(HpaII/MspI). The black line was drawn using DNAcopy, a circular binary segmentation programme obtained from http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html.32 Red dots represent CCGG segments belonging to Cluster 4. Double-headed arrows indicate the position of the Xmr gene cluster." path-from-xml="dst03003.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212966/dst03003.tiff" path-from-xml="dst03003.tiff" target="_blank">View large</a><a section="6212966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03003.png?Expires=2147483647&Signature=VIEjErEGRr2wZGNzgwt-SIxtLxr7K2Q5TFHkjZajow9TleJ3CfJ12xpya7itD2wJjjn0OUpZ4VZ76eOpzh0MoMefU50eSS~sTEXw-PHdL-53XcbkUoIGTp1RjGaRnxrayH68TnY34OwzYCwv4rZ7Zl9LKHCoEGLmFKpRKS~pb58dwvXjisiuh0eFcqNoA1Bp844Qb1V7UOuGS6Tslp1Vw7pW1rbzSeBIj4pUVfTb-y9XqRxjPEHIGQcW6wxPTPowx390sv8296htU0eouxyIE6sDy399cJNbWluc9LPC6R2Sn06TFlYMwDpbk879kWnJj8JyEbfHbosjUDuUUWX~XA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212966&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03003.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Discovery of large, contiguous genomic regions with low DNA methylation modifications in male germ cells. The methylation profiles of genomic DNA along the mouse X chromosome. Samples used for the analysis are indicated in each figure. The <em>M</em>-value of each CCGG segment obtained from the analysis of each sample DNA is plotted on the mouse X chromosome (grey dots). The <em>y</em>-axis represents the <em>M</em>-value; log<sub>2</sub>(<em>Hpa</em>II/<em>Msp</em>I). The black line was drawn using DNAcopy, a circular binary segmentation programme obtained from <a class="link link-uri" href="http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html">http://www.bioconductor.org/packages/2.3/bioc/html/DNAcopy.html</a>.<sup><a reveal-id="DST030C32" data-open="DST030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Red dots represent CCGG segments belonging to Cluster 4. Double-headed arrows indicate the position of the <em>Xmr</em> gene cluster.</p></div></div><p>It is obvious that the distribution of the Cluster 4 segments is not uniform, and that these Cluster 4 segments form ‘hypomethylated domains’ compared with their flanking regions. These trends persist in P0.5 spermatogonia and GS cells derived from spermatogonia (Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a>), with a few cell type-specific differences. In testis DNA, the overall methylation pattern of the Cluster 4 segments is essentially similar to that found in the male germline cells, described above (Fig. <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a>). We also examined earlier stages of male PGCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3B</a>). In E10.5 male PGCs, formation of hypomethylated domains, e.g. <em>Xmr</em> region, is not as obvious as seen in E17.5 male PGCs. In E13.5 male PGC DNA, the distribution of the Cluster 4 segments is similar to that found in E17.5 male PGCs. It thus appears that clustering of hypomethylated DNA segments become increasingly evident on the X chromosome during the development of male germ cells.</p>                            <h3 scrollto-destination=6212968 id="6212968" class="section-title" data-legacy-id=s3f>3.6. Discovery of large genomic regions hypomethylated specifically in male germline cells</h3>
<p>Although it is clear that segment DNAs possess generally lower methylation levels in female PGCs than in cells of somatic organs, the formation of hypomethylated DNA regions as seen in male PGCs is not evident in female PGCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3B and C</a>). We therefore decided to focus on the male germline-specific hypomethylated DNA regions comprised of Cluster 4 segments. To visualize the hypomethylated DNA regions in the male germline from a different viewpoint, we plotted fold differences in the methylation level between somatic and male germ cell DNA along the X chromosome (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>A). Because methylation patterns of Cluster 4 segments are essentially similar in testis, E17.5 and P0.5 male PGCs, the testis was chosen for this analysis. Brain was also used as somatic tissue for this analysis. As the data were plotted with a log<sub>2</sub> scale, a negative value indicates the lower level of DNA methylation in the testis than in the brain. The plot revealed broad domains with lower methylation levels in the testis and therefore, in male germ cells of late stages (coloured light blue in Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>A). These broad domains of hypomethylated DNA described above are distinct from CGIs, which are generally located within or near a promoter and have a typical length of 300–3000 bp.<sup><a reveal-id="DST030C33" data-open="DST030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The broad and hypomethylated domains identified here are often much larger than CGIs and do not show preferential localization at promoter regions. Thus, these broad domains do not correspond to the known hypomethylated regions and may represent a hitherto unknown epigenomic entity. For convenience of discussion, we here designate such a broad and hypomethylated domain as an LoD. By definition, a LoD has a size of &gt;10 kb and shows more than a 2-fold difference in <em>M</em>-value between germline and somatic cells (testis and brain in this case). Each LoD should also have at least one Cluster 4 segment. </p>                        <a id="6212970" scrollto-destination="6212970"></a>
<div id="DST030F4" data-id="DST030F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03004.png?Expires=2147483647&Signature=iIOTVdzN5VdGL1SRaHySlMczFYIkwRB~RTo4drbRtxQZUYd1SiriLKVpRESIMIx~thwrSDDXQbUTAidm-qta5agTpidXhpJDdG4~5j-rz9UORRiHbdoQrKTPTxWTV543M4Os4D515G6hiXcOq~EqZkpGDTawMEvs-QeJ5QJNtyxPTkt0mbLG-NMhz0vtDRx~Dn8CeFS0hIVb3JlRitHubddfSNQl5vWU2tJbf1-5apw1VDwwxTlcwuU9a~zbAn9k2J~4Q-yBtUhUbXTXiB6sgcPTOPp2G0eyO3nlmeNCx5lQF~npfkn16RcbBhNLDsGVQaXgPK4iOOMDE32Sr1e~1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Demonstration of LoDs on the mouse X chromosome. (A) Differences in methylation levels between brain and testis genomic DNA along the mouse X chromosome. Fold changes in the methylation level, i.e. brain M-value versus testis M-value, were calculated for each CCGG segment, and plotted using the log2 scale along the X chromosome. The blue lines indicate genomic regions showing more than a 2-fold difference between the brain M-value and testis M-value. Light blue represents hypomethylated regions in the testis relative to the brain. Green dots at the bottom represent the positions of CGIs. (B) The positions of segmentally duplicated regions along the mouse X chromosome. Segmentally duplicated regions &gt;1000 bases with &gt;98% similarity are counted, and the frequencies of duplications (y-axis) are shown. (The data were from UCSC Genome Browser.) Grey bars represent duplications occurring on the X and other chromosomes, and yellow bars represent the frequencies of duplications mapped only on the X chromosome. (C) Methylation analysis of LoDs 10 and 12 by Southern blot hybridization. Genomic DNAs of the male thymus, male brain and testis were digested by either methylation-sensitive HpaII (H) or methylation-insensitive isoschizomer, MspI (M). The Southern blot was hybridized with a probe targeted to LoDs 10 and 12. A primer pair (FW: 5′-GCTGGGTCCAGCTTCCCTGG-3′, RV: 5′-TGGCACCCCTCCTGCCTGAT-3′) was used to amplify a 807-bp sequence using testis cDNA for generation of the probe. The 807-bp probe contains locally repeated sequences and corresponds to both LoDs 10 and 12 located upstream of Mageb1/b2 genes. (D) Methylation analysis of LoDs 10 and 12 in purified germ cells. Germ cells expressing the Mvh-Venus reporter were purified from adult testis by FACS. DNAs from purified germ cells and whole testis were digested with MspI plus BamHI (M + B), HpaII plus BamHI (H + B) or BamHI only (B). A Southern blot was made using these DNAs and hybridized with the same probe as used in Fig. 4C." path-from-xml="dst03004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212970/dst03004.tiff" path-from-xml="dst03004.tiff" target="_blank">View large</a><a section="6212970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03004.png?Expires=2147483647&Signature=OOwrsrVk1dUyeNte~1YgSC--t1QrWDrwLnWmIjDCtnjGU83ns-LZSo2MnAwaT2OX19D3NEaOFqqE8tX9SA1lkL3oHssEXREj01rMJvREuwls1Z9FbSiDRGONrEqr0wONbBamY2OYaKRzloOS6tq1wW4VfNJLjZtVPyCyfb4bFNG6zhOjw9wNZMqaZS8ZaZAmrXv0-JFNL-yEyxybBFlwxr~B0teR0toB2hPvfZa33v3Z5fLf385oCF8uH7VElS0FbyqO4Fip9LnXMgpmRVoBDoS2Idjensq9D~wI14igVT7yWuV4aR9aEQ8E0FdECnOCjHm-icq7ss-WyS1LvDY8qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212970&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Demonstration of LoDs on the mouse X chromosome. (A) Differences in methylation levels between brain and testis genomic DNA along the mouse X chromosome. Fold changes in the methylation level, i.e. brain <em>M</em>-value versus testis <em>M</em>-value, were calculated for each CCGG segment, and plotted using the log<sub>2</sub> scale along the X chromosome. The blue lines indicate genomic regions showing more than a 2-fold difference between the brain <em>M</em>-value and testis <em>M</em>-value. Light blue represents hypomethylated regions in the testis relative to the brain. Green dots at the bottom represent the positions of CGIs. (B) The positions of segmentally duplicated regions along the mouse X chromosome. Segmentally duplicated regions &gt;1000 bases with &gt;98% similarity are counted, and the frequencies of duplications (<em>y</em>-axis) are shown. (The data were from UCSC Genome Browser.) Grey bars represent duplications occurring on the X and other chromosomes, and yellow bars represent the frequencies of duplications mapped only on the X chromosome. (C) Methylation analysis of LoDs 10 and 12 by Southern blot hybridization. Genomic DNAs of the male thymus, male brain and testis were digested by either methylation-sensitive <em>Hpa</em>II (H) or methylation-insensitive isoschizomer, <em>Msp</em>I (M). The Southern blot was hybridized with a probe targeted to LoDs 10 and 12. A primer pair (FW: 5′-GCTGGGTCCAGCTTCCCTGG-3′, RV: 5′-TGGCACCCCTCCTGCCTGAT-3′) was used to amplify a 807-bp sequence using testis cDNA for generation of the probe. The 807-bp probe contains locally repeated sequences and corresponds to both LoDs 10 and 12 located upstream of <em>Mageb1/b2</em> genes. (D) Methylation analysis of LoDs 10 and 12 in purified germ cells. Germ cells expressing the <em>Mvh-Venus</em> reporter were purified from adult testis by FACS. DNAs from purified germ cells and whole testis were digested with <em>Msp</em>I plus <em>Bam</em>HI (M + B), <em>Hpa</em>II plus <em>Bam</em>HI (H + B) or <em>Bam</em>HI only (B). A Southern blot was made using these DNAs and hybridized with the same probe as used in Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C.</p></div></div><div content-id="DST030F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03004.png?Expires=2147483647&Signature=iIOTVdzN5VdGL1SRaHySlMczFYIkwRB~RTo4drbRtxQZUYd1SiriLKVpRESIMIx~thwrSDDXQbUTAidm-qta5agTpidXhpJDdG4~5j-rz9UORRiHbdoQrKTPTxWTV543M4Os4D515G6hiXcOq~EqZkpGDTawMEvs-QeJ5QJNtyxPTkt0mbLG-NMhz0vtDRx~Dn8CeFS0hIVb3JlRitHubddfSNQl5vWU2tJbf1-5apw1VDwwxTlcwuU9a~zbAn9k2J~4Q-yBtUhUbXTXiB6sgcPTOPp2G0eyO3nlmeNCx5lQF~npfkn16RcbBhNLDsGVQaXgPK4iOOMDE32Sr1e~1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Demonstration of LoDs on the mouse X chromosome. (A) Differences in methylation levels between brain and testis genomic DNA along the mouse X chromosome. Fold changes in the methylation level, i.e. brain M-value versus testis M-value, were calculated for each CCGG segment, and plotted using the log2 scale along the X chromosome. The blue lines indicate genomic regions showing more than a 2-fold difference between the brain M-value and testis M-value. Light blue represents hypomethylated regions in the testis relative to the brain. Green dots at the bottom represent the positions of CGIs. (B) The positions of segmentally duplicated regions along the mouse X chromosome. Segmentally duplicated regions &gt;1000 bases with &gt;98% similarity are counted, and the frequencies of duplications (y-axis) are shown. (The data were from UCSC Genome Browser.) Grey bars represent duplications occurring on the X and other chromosomes, and yellow bars represent the frequencies of duplications mapped only on the X chromosome. (C) Methylation analysis of LoDs 10 and 12 by Southern blot hybridization. Genomic DNAs of the male thymus, male brain and testis were digested by either methylation-sensitive HpaII (H) or methylation-insensitive isoschizomer, MspI (M). The Southern blot was hybridized with a probe targeted to LoDs 10 and 12. A primer pair (FW: 5′-GCTGGGTCCAGCTTCCCTGG-3′, RV: 5′-TGGCACCCCTCCTGCCTGAT-3′) was used to amplify a 807-bp sequence using testis cDNA for generation of the probe. The 807-bp probe contains locally repeated sequences and corresponds to both LoDs 10 and 12 located upstream of Mageb1/b2 genes. (D) Methylation analysis of LoDs 10 and 12 in purified germ cells. Germ cells expressing the Mvh-Venus reporter were purified from adult testis by FACS. DNAs from purified germ cells and whole testis were digested with MspI plus BamHI (M + B), HpaII plus BamHI (H + B) or BamHI only (B). A Southern blot was made using these DNAs and hybridized with the same probe as used in Fig. 4C." path-from-xml="dst03004.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212970/dst03004.tiff" path-from-xml="dst03004.tiff" target="_blank">View large</a><a section="6212970" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03004.png?Expires=2147483647&Signature=OOwrsrVk1dUyeNte~1YgSC--t1QrWDrwLnWmIjDCtnjGU83ns-LZSo2MnAwaT2OX19D3NEaOFqqE8tX9SA1lkL3oHssEXREj01rMJvREuwls1Z9FbSiDRGONrEqr0wONbBamY2OYaKRzloOS6tq1wW4VfNJLjZtVPyCyfb4bFNG6zhOjw9wNZMqaZS8ZaZAmrXv0-JFNL-yEyxybBFlwxr~B0teR0toB2hPvfZa33v3Z5fLf385oCF8uH7VElS0FbyqO4Fip9LnXMgpmRVoBDoS2Idjensq9D~wI14igVT7yWuV4aR9aEQ8E0FdECnOCjHm-icq7ss-WyS1LvDY8qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212970&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03004.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Demonstration of LoDs on the mouse X chromosome. (A) Differences in methylation levels between brain and testis genomic DNA along the mouse X chromosome. Fold changes in the methylation level, i.e. brain <em>M</em>-value versus testis <em>M</em>-value, were calculated for each CCGG segment, and plotted using the log<sub>2</sub> scale along the X chromosome. The blue lines indicate genomic regions showing more than a 2-fold difference between the brain <em>M</em>-value and testis <em>M</em>-value. Light blue represents hypomethylated regions in the testis relative to the brain. Green dots at the bottom represent the positions of CGIs. (B) The positions of segmentally duplicated regions along the mouse X chromosome. Segmentally duplicated regions &gt;1000 bases with &gt;98% similarity are counted, and the frequencies of duplications (<em>y</em>-axis) are shown. (The data were from UCSC Genome Browser.) Grey bars represent duplications occurring on the X and other chromosomes, and yellow bars represent the frequencies of duplications mapped only on the X chromosome. (C) Methylation analysis of LoDs 10 and 12 by Southern blot hybridization. Genomic DNAs of the male thymus, male brain and testis were digested by either methylation-sensitive <em>Hpa</em>II (H) or methylation-insensitive isoschizomer, <em>Msp</em>I (M). The Southern blot was hybridized with a probe targeted to LoDs 10 and 12. A primer pair (FW: 5′-GCTGGGTCCAGCTTCCCTGG-3′, RV: 5′-TGGCACCCCTCCTGCCTGAT-3′) was used to amplify a 807-bp sequence using testis cDNA for generation of the probe. The 807-bp probe contains locally repeated sequences and corresponds to both LoDs 10 and 12 located upstream of <em>Mageb1/b2</em> genes. (D) Methylation analysis of LoDs 10 and 12 in purified germ cells. Germ cells expressing the <em>Mvh-Venus</em> reporter were purified from adult testis by FACS. DNAs from purified germ cells and whole testis were digested with <em>Msp</em>I plus <em>Bam</em>HI (M + B), <em>Hpa</em>II plus <em>Bam</em>HI (H + B) or <em>Bam</em>HI only (B). A Southern blot was made using these DNAs and hybridized with the same probe as used in Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C.</p></div></div>                            <h3 scrollto-destination=6212971 id="6212971" class="section-title" data-legacy-id=s3g>3.7. Overlap of LoDs with segmentally duplicated regions</h3>
<p>Using the definition described above, we list the LoDs of the X chromosome in Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a>. There are 16 LoDs on the X chromosome (Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a>), and their sizes are generally large: 11 of the 16 LoDs are &gt;100 kb (mean: 1 219 252 bp), and six of the large LoDs are &gt;1 Mb. The mammalian genome is replete with segmentally duplicated regions.<sup><a reveal-id="DST030C34" data-open="DST030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Although segmental duplications can be found on every chromosome, they are particularly abundant on the sex chromosomes. Because LoDs are generally large and contain gene families such as <em>Xmr</em>, we asked whether LoDs overlap with segmentally duplicated regions. As shown in Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>A and B and Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a>, all LoDs on the X chromosome are found to contain segmentally duplicated regions. The use of the <em>Msp</em>I control represents an unusual strength of the HELP assay to remove the potentially confounding effect of copy number variation.<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Combined with a probe design that selects only unique sequences for hybridization, these aspects ensure that the DNA methylation readout from regions of constitutive segmental duplication accurately reflects the underlying DNA methylation and is not influenced by DNA copy number. </p>                        <a id="6212973" scrollto-destination="6212973"></a>
<div content-id="DST030TB1" class="table-modal"><div class="table-wrap"><div id="DST030TB1" class="table-wrap-title" data-id="DST030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of LoDs on the mouse X chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LoD no. </th><th rowspan="2">Position </th><th rowspan="2">Length (bp) </th><th rowspan="2">Segment number on HELP array </th><th colspan="3">Methylation<hr /> </th><th rowspan="2">Gene families </th><th rowspan="2">Description </th><th colspan="3">Number of duplications<hr /> </th><th rowspan="2">Human CTA genes </th></tr><tr><th>Brain <em>M</em>-value/testis <em>M</em>-value </th><th>CGI </th><th>Seg Dups </th><th>Total </th><th>Within LoD </th><th>Outside LoD on chrX </th></tr></thead><tbody><tr><td>1 </td><td>chrX:3035387-4561626 </td><td>1 526 239 </td><td>12 </td><td>−2.52 </td><td> </td><td>1344 </td><td>Gmcl1l </td><td>Germ cell-less protein-like 1 </td><td>11 </td><td>8 </td><td>3 </td><td> </td></tr><tr><td>2 </td><td>chrX:7857031-7924410 </td><td>67 379 </td><td>5 </td><td>−1.924 </td><td> </td><td>88 </td><td>Ssx9 </td><td>Synovial sarcoma, X breakpoint 9 </td><td>11 </td><td>4 </td><td>7 </td><td>CTA </td></tr><tr><td>3 </td><td>chrX:8116622-8236784 </td><td>120 162 </td><td>19 </td><td>−1.82 </td><td> </td><td>45 </td><td>Fthl17 </td><td>Ferritin, heavy polypeptide-like 17 </td><td>7 </td><td>6 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="3">4 </td><td rowspan="3">chrX:22991291-32117922 </td><td rowspan="3">9 126 631 </td><td rowspan="3">100 </td><td rowspan="3">−1.4745 </td><td rowspan="3">  </td><td rowspan="3">10 697 </td><td>Xmr </td><td>XMR protein (Xlr-related, meiosis regulated) </td><td>32 </td><td>28 </td><td>4 </td><td> </td></tr><tr><td>Gmcl1l </td><td>Germ cell-less homologue 1 (Drosophila)-like </td><td>11 </td><td>2 </td><td>9 </td><td> </td></tr><tr><td>LOC236749 </td><td>Hypothetical protein LOC236749 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>5 </td><td>chrX:50380769-52184422 </td><td>1 803 653 </td><td>48 </td><td>−1.75 </td><td>1 </td><td>1971 </td><td>Similar to Xmr protein </td><td>Adult male testis cDNA, RIKEN full-length enriched library, clone: 4930527E24 product: weakly similar to XMR PROTEIN </td><td>30 </td><td>16 </td><td>14 </td><td> </td></tr><tr><td>6 </td><td>chrX:57970833-58090999 </td><td>120 166 </td><td>5 </td><td>−2.14 </td><td> </td><td>23 </td><td>Ldoc1 </td><td>Leucine zipper, down-regulated in cancer 1 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>7 </td><td>chrX:58775958-58815172 </td><td>39 214 </td><td>13 </td><td>−3.45 </td><td> </td><td>10 </td><td><sup>a</sup>1700019B21Rik </td><td><em>Mus musculus</em> adult male testis cDNA, <em>M. musculus</em> RIKEN cDNA 1700019B21 gene (1700019B21Rik), transcript variant 1, non-coding RNA </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>8 </td><td>chrX:72448732-72544743 </td><td>96 011 </td><td>4 </td><td>−1.59 </td><td> </td><td>67 </td><td>LOC238829 </td><td>Hypothetical protein LOC238829 (AK133378—<em>M. musculus</em> adult male testis cDNA, RIKEN full-length enriched library, clone: 4933402E19 product: hypothetical protein, full insert sequence) </td><td>4 </td><td>1 </td><td>3 </td><td> </td></tr><tr><td rowspan="2">9 </td><td rowspan="2">chrX:84663998-86056286 </td><td rowspan="2">1 392 288 </td><td rowspan="2">23 </td><td rowspan="2">−1.2207 </td><td rowspan="2">  </td><td rowspan="2">334 </td><td>Pet2 </td><td>Plasmacytoma expressed transcript 2 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>4932429P05Rik </td><td><em>M. musculus</em> RIKEN cDNA 4932429P05 gene (4932429P05Rik), mRNA. (<em>M. musculus</em> adult male testis cDNA) SMEK homologue 3, putative </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>10 </td><td>chrX:87564706-87579103 </td><td>14 397 </td><td>20 </td><td>−3.70 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1, Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="2">11 </td><td rowspan="2">chrX:87598220-88271067 </td><td rowspan="2">672 847 </td><td rowspan="2">15 </td><td rowspan="2">−1.0483 </td><td rowspan="2">  </td><td rowspan="2">43 </td><td>Mageb5 </td><td>Melanoma antigen, family B, 5 </td><td>2 </td><td>2 </td><td>0 </td><td>CTA </td></tr><tr><td>Mageb1 </td><td>Melanoma antigen, family B, 1 </td><td>2 </td><td>1 </td><td>1 </td><td>CTA </td></tr><tr><td>12 </td><td>chrX:88271564-88285701 </td><td>14 137 </td><td>21 </td><td>−3.76 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1,Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="4">13 </td><td rowspan="4">chrX:103040719-103447968 </td><td rowspan="4">407 249 </td><td rowspan="4">10 </td><td rowspan="4">−1.186 </td><td rowspan="4">  </td><td rowspan="4">158 </td><td>A630033H20Rik </td><td>Hypothetical protein LOC213438 (product: similar to putative purinergic receptor P2Y10) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Gpr23 </td><td>G protein-coupled receptor 23 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>P2ry10 </td><td>Purinergic receptor P2Y, G-protein coupled 10 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Zcchc5 </td><td>Zinc finger, CCHC domain containing 5 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>14 </td><td>chrX:142924085-145405975 </td><td>2 481 890 </td><td>112 </td><td>−1.0089 </td><td>2 </td><td>1666 </td><td>Ott </td><td>Ovary testis transcribed </td><td>17 </td><td>16 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="2">15 </td><td rowspan="2">chrX:149333243-150403389 </td><td rowspan="2">1 070 146 </td><td rowspan="2">41 </td><td rowspan="2">−1.4149 </td><td rowspan="2">  </td><td rowspan="2">211 </td><td>Magea </td><td>Melanoma antigen family A 2, 3, 5, 6, 8 </td><td>7 </td><td>5 </td><td>2 </td><td>CTA </td></tr><tr><td>Samt4 </td><td><em>M. musculus</em> spermatogenesis associated multi-pass transmembrane protein 4 (Samt4), mRNA (hypothetical protein LOC75185) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>16 </td><td>chrX:160673416-161229042 </td><td>555 626 </td><td>13 </td><td>−1.13 </td><td> </td><td>274 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DST030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LoD no. </th><th rowspan="2">Position </th><th rowspan="2">Length (bp) </th><th rowspan="2">Segment number on HELP array </th><th colspan="3">Methylation<hr /> </th><th rowspan="2">Gene families </th><th rowspan="2">Description </th><th colspan="3">Number of duplications<hr /> </th><th rowspan="2">Human CTA genes </th></tr><tr><th>Brain <em>M</em>-value/testis <em>M</em>-value </th><th>CGI </th><th>Seg Dups </th><th>Total </th><th>Within LoD </th><th>Outside LoD on chrX </th></tr></thead><tbody><tr><td>1 </td><td>chrX:3035387-4561626 </td><td>1 526 239 </td><td>12 </td><td>−2.52 </td><td> </td><td>1344 </td><td>Gmcl1l </td><td>Germ cell-less protein-like 1 </td><td>11 </td><td>8 </td><td>3 </td><td> </td></tr><tr><td>2 </td><td>chrX:7857031-7924410 </td><td>67 379 </td><td>5 </td><td>−1.924 </td><td> </td><td>88 </td><td>Ssx9 </td><td>Synovial sarcoma, X breakpoint 9 </td><td>11 </td><td>4 </td><td>7 </td><td>CTA </td></tr><tr><td>3 </td><td>chrX:8116622-8236784 </td><td>120 162 </td><td>19 </td><td>−1.82 </td><td> </td><td>45 </td><td>Fthl17 </td><td>Ferritin, heavy polypeptide-like 17 </td><td>7 </td><td>6 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="3">4 </td><td rowspan="3">chrX:22991291-32117922 </td><td rowspan="3">9 126 631 </td><td rowspan="3">100 </td><td rowspan="3">−1.4745 </td><td rowspan="3">  </td><td rowspan="3">10 697 </td><td>Xmr </td><td>XMR protein (Xlr-related, meiosis regulated) </td><td>32 </td><td>28 </td><td>4 </td><td> </td></tr><tr><td>Gmcl1l </td><td>Germ cell-less homologue 1 (Drosophila)-like </td><td>11 </td><td>2 </td><td>9 </td><td> </td></tr><tr><td>LOC236749 </td><td>Hypothetical protein LOC236749 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>5 </td><td>chrX:50380769-52184422 </td><td>1 803 653 </td><td>48 </td><td>−1.75 </td><td>1 </td><td>1971 </td><td>Similar to Xmr protein </td><td>Adult male testis cDNA, RIKEN full-length enriched library, clone: 4930527E24 product: weakly similar to XMR PROTEIN </td><td>30 </td><td>16 </td><td>14 </td><td> </td></tr><tr><td>6 </td><td>chrX:57970833-58090999 </td><td>120 166 </td><td>5 </td><td>−2.14 </td><td> </td><td>23 </td><td>Ldoc1 </td><td>Leucine zipper, down-regulated in cancer 1 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>7 </td><td>chrX:58775958-58815172 </td><td>39 214 </td><td>13 </td><td>−3.45 </td><td> </td><td>10 </td><td><sup>a</sup>1700019B21Rik </td><td><em>Mus musculus</em> adult male testis cDNA, <em>M. musculus</em> RIKEN cDNA 1700019B21 gene (1700019B21Rik), transcript variant 1, non-coding RNA </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>8 </td><td>chrX:72448732-72544743 </td><td>96 011 </td><td>4 </td><td>−1.59 </td><td> </td><td>67 </td><td>LOC238829 </td><td>Hypothetical protein LOC238829 (AK133378—<em>M. musculus</em> adult male testis cDNA, RIKEN full-length enriched library, clone: 4933402E19 product: hypothetical protein, full insert sequence) </td><td>4 </td><td>1 </td><td>3 </td><td> </td></tr><tr><td rowspan="2">9 </td><td rowspan="2">chrX:84663998-86056286 </td><td rowspan="2">1 392 288 </td><td rowspan="2">23 </td><td rowspan="2">−1.2207 </td><td rowspan="2">  </td><td rowspan="2">334 </td><td>Pet2 </td><td>Plasmacytoma expressed transcript 2 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>4932429P05Rik </td><td><em>M. musculus</em> RIKEN cDNA 4932429P05 gene (4932429P05Rik), mRNA. (<em>M. musculus</em> adult male testis cDNA) SMEK homologue 3, putative </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>10 </td><td>chrX:87564706-87579103 </td><td>14 397 </td><td>20 </td><td>−3.70 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1, Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="2">11 </td><td rowspan="2">chrX:87598220-88271067 </td><td rowspan="2">672 847 </td><td rowspan="2">15 </td><td rowspan="2">−1.0483 </td><td rowspan="2">  </td><td rowspan="2">43 </td><td>Mageb5 </td><td>Melanoma antigen, family B, 5 </td><td>2 </td><td>2 </td><td>0 </td><td>CTA </td></tr><tr><td>Mageb1 </td><td>Melanoma antigen, family B, 1 </td><td>2 </td><td>1 </td><td>1 </td><td>CTA </td></tr><tr><td>12 </td><td>chrX:88271564-88285701 </td><td>14 137 </td><td>21 </td><td>−3.76 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1,Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="4">13 </td><td rowspan="4">chrX:103040719-103447968 </td><td rowspan="4">407 249 </td><td rowspan="4">10 </td><td rowspan="4">−1.186 </td><td rowspan="4">  </td><td rowspan="4">158 </td><td>A630033H20Rik </td><td>Hypothetical protein LOC213438 (product: similar to putative purinergic receptor P2Y10) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Gpr23 </td><td>G protein-coupled receptor 23 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>P2ry10 </td><td>Purinergic receptor P2Y, G-protein coupled 10 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Zcchc5 </td><td>Zinc finger, CCHC domain containing 5 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>14 </td><td>chrX:142924085-145405975 </td><td>2 481 890 </td><td>112 </td><td>−1.0089 </td><td>2 </td><td>1666 </td><td>Ott </td><td>Ovary testis transcribed </td><td>17 </td><td>16 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="2">15 </td><td rowspan="2">chrX:149333243-150403389 </td><td rowspan="2">1 070 146 </td><td rowspan="2">41 </td><td rowspan="2">−1.4149 </td><td rowspan="2">  </td><td rowspan="2">211 </td><td>Magea </td><td>Melanoma antigen family A 2, 3, 5, 6, 8 </td><td>7 </td><td>5 </td><td>2 </td><td>CTA </td></tr><tr><td>Samt4 </td><td><em>M. musculus</em> spermatogenesis associated multi-pass transmembrane protein 4 (Samt4), mRNA (hypothetical protein LOC75185) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>16 </td><td>chrX:160673416-161229042 </td><td>555 626 </td><td>13 </td><td>−1.13 </td><td> </td><td>274 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Non-coding RNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6212973" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST030TB1" class="table-wrap-title" data-id="DST030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of LoDs on the mouse X chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LoD no. </th><th rowspan="2">Position </th><th rowspan="2">Length (bp) </th><th rowspan="2">Segment number on HELP array </th><th colspan="3">Methylation<hr /> </th><th rowspan="2">Gene families </th><th rowspan="2">Description </th><th colspan="3">Number of duplications<hr /> </th><th rowspan="2">Human CTA genes </th></tr><tr><th>Brain <em>M</em>-value/testis <em>M</em>-value </th><th>CGI </th><th>Seg Dups </th><th>Total </th><th>Within LoD </th><th>Outside LoD on chrX </th></tr></thead><tbody><tr><td>1 </td><td>chrX:3035387-4561626 </td><td>1 526 239 </td><td>12 </td><td>−2.52 </td><td> </td><td>1344 </td><td>Gmcl1l </td><td>Germ cell-less protein-like 1 </td><td>11 </td><td>8 </td><td>3 </td><td> </td></tr><tr><td>2 </td><td>chrX:7857031-7924410 </td><td>67 379 </td><td>5 </td><td>−1.924 </td><td> </td><td>88 </td><td>Ssx9 </td><td>Synovial sarcoma, X breakpoint 9 </td><td>11 </td><td>4 </td><td>7 </td><td>CTA </td></tr><tr><td>3 </td><td>chrX:8116622-8236784 </td><td>120 162 </td><td>19 </td><td>−1.82 </td><td> </td><td>45 </td><td>Fthl17 </td><td>Ferritin, heavy polypeptide-like 17 </td><td>7 </td><td>6 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="3">4 </td><td rowspan="3">chrX:22991291-32117922 </td><td rowspan="3">9 126 631 </td><td rowspan="3">100 </td><td rowspan="3">−1.4745 </td><td rowspan="3">  </td><td rowspan="3">10 697 </td><td>Xmr </td><td>XMR protein (Xlr-related, meiosis regulated) </td><td>32 </td><td>28 </td><td>4 </td><td> </td></tr><tr><td>Gmcl1l </td><td>Germ cell-less homologue 1 (Drosophila)-like </td><td>11 </td><td>2 </td><td>9 </td><td> </td></tr><tr><td>LOC236749 </td><td>Hypothetical protein LOC236749 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>5 </td><td>chrX:50380769-52184422 </td><td>1 803 653 </td><td>48 </td><td>−1.75 </td><td>1 </td><td>1971 </td><td>Similar to Xmr protein </td><td>Adult male testis cDNA, RIKEN full-length enriched library, clone: 4930527E24 product: weakly similar to XMR PROTEIN </td><td>30 </td><td>16 </td><td>14 </td><td> </td></tr><tr><td>6 </td><td>chrX:57970833-58090999 </td><td>120 166 </td><td>5 </td><td>−2.14 </td><td> </td><td>23 </td><td>Ldoc1 </td><td>Leucine zipper, down-regulated in cancer 1 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>7 </td><td>chrX:58775958-58815172 </td><td>39 214 </td><td>13 </td><td>−3.45 </td><td> </td><td>10 </td><td><sup>a</sup>1700019B21Rik </td><td><em>Mus musculus</em> adult male testis cDNA, <em>M. musculus</em> RIKEN cDNA 1700019B21 gene (1700019B21Rik), transcript variant 1, non-coding RNA </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>8 </td><td>chrX:72448732-72544743 </td><td>96 011 </td><td>4 </td><td>−1.59 </td><td> </td><td>67 </td><td>LOC238829 </td><td>Hypothetical protein LOC238829 (AK133378—<em>M. musculus</em> adult male testis cDNA, RIKEN full-length enriched library, clone: 4933402E19 product: hypothetical protein, full insert sequence) </td><td>4 </td><td>1 </td><td>3 </td><td> </td></tr><tr><td rowspan="2">9 </td><td rowspan="2">chrX:84663998-86056286 </td><td rowspan="2">1 392 288 </td><td rowspan="2">23 </td><td rowspan="2">−1.2207 </td><td rowspan="2">  </td><td rowspan="2">334 </td><td>Pet2 </td><td>Plasmacytoma expressed transcript 2 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>4932429P05Rik </td><td><em>M. musculus</em> RIKEN cDNA 4932429P05 gene (4932429P05Rik), mRNA. (<em>M. musculus</em> adult male testis cDNA) SMEK homologue 3, putative </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>10 </td><td>chrX:87564706-87579103 </td><td>14 397 </td><td>20 </td><td>−3.70 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1, Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="2">11 </td><td rowspan="2">chrX:87598220-88271067 </td><td rowspan="2">672 847 </td><td rowspan="2">15 </td><td rowspan="2">−1.0483 </td><td rowspan="2">  </td><td rowspan="2">43 </td><td>Mageb5 </td><td>Melanoma antigen, family B, 5 </td><td>2 </td><td>2 </td><td>0 </td><td>CTA </td></tr><tr><td>Mageb1 </td><td>Melanoma antigen, family B, 1 </td><td>2 </td><td>1 </td><td>1 </td><td>CTA </td></tr><tr><td>12 </td><td>chrX:88271564-88285701 </td><td>14 137 </td><td>21 </td><td>−3.76 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1,Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="4">13 </td><td rowspan="4">chrX:103040719-103447968 </td><td rowspan="4">407 249 </td><td rowspan="4">10 </td><td rowspan="4">−1.186 </td><td rowspan="4">  </td><td rowspan="4">158 </td><td>A630033H20Rik </td><td>Hypothetical protein LOC213438 (product: similar to putative purinergic receptor P2Y10) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Gpr23 </td><td>G protein-coupled receptor 23 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>P2ry10 </td><td>Purinergic receptor P2Y, G-protein coupled 10 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Zcchc5 </td><td>Zinc finger, CCHC domain containing 5 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>14 </td><td>chrX:142924085-145405975 </td><td>2 481 890 </td><td>112 </td><td>−1.0089 </td><td>2 </td><td>1666 </td><td>Ott </td><td>Ovary testis transcribed </td><td>17 </td><td>16 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="2">15 </td><td rowspan="2">chrX:149333243-150403389 </td><td rowspan="2">1 070 146 </td><td rowspan="2">41 </td><td rowspan="2">−1.4149 </td><td rowspan="2">  </td><td rowspan="2">211 </td><td>Magea </td><td>Melanoma antigen family A 2, 3, 5, 6, 8 </td><td>7 </td><td>5 </td><td>2 </td><td>CTA </td></tr><tr><td>Samt4 </td><td><em>M. musculus</em> spermatogenesis associated multi-pass transmembrane protein 4 (Samt4), mRNA (hypothetical protein LOC75185) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>16 </td><td>chrX:160673416-161229042 </td><td>555 626 </td><td>13 </td><td>−1.13 </td><td> </td><td>274 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DST030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LoD no. </th><th rowspan="2">Position </th><th rowspan="2">Length (bp) </th><th rowspan="2">Segment number on HELP array </th><th colspan="3">Methylation<hr /> </th><th rowspan="2">Gene families </th><th rowspan="2">Description </th><th colspan="3">Number of duplications<hr /> </th><th rowspan="2">Human CTA genes </th></tr><tr><th>Brain <em>M</em>-value/testis <em>M</em>-value </th><th>CGI </th><th>Seg Dups </th><th>Total </th><th>Within LoD </th><th>Outside LoD on chrX </th></tr></thead><tbody><tr><td>1 </td><td>chrX:3035387-4561626 </td><td>1 526 239 </td><td>12 </td><td>−2.52 </td><td> </td><td>1344 </td><td>Gmcl1l </td><td>Germ cell-less protein-like 1 </td><td>11 </td><td>8 </td><td>3 </td><td> </td></tr><tr><td>2 </td><td>chrX:7857031-7924410 </td><td>67 379 </td><td>5 </td><td>−1.924 </td><td> </td><td>88 </td><td>Ssx9 </td><td>Synovial sarcoma, X breakpoint 9 </td><td>11 </td><td>4 </td><td>7 </td><td>CTA </td></tr><tr><td>3 </td><td>chrX:8116622-8236784 </td><td>120 162 </td><td>19 </td><td>−1.82 </td><td> </td><td>45 </td><td>Fthl17 </td><td>Ferritin, heavy polypeptide-like 17 </td><td>7 </td><td>6 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="3">4 </td><td rowspan="3">chrX:22991291-32117922 </td><td rowspan="3">9 126 631 </td><td rowspan="3">100 </td><td rowspan="3">−1.4745 </td><td rowspan="3">  </td><td rowspan="3">10 697 </td><td>Xmr </td><td>XMR protein (Xlr-related, meiosis regulated) </td><td>32 </td><td>28 </td><td>4 </td><td> </td></tr><tr><td>Gmcl1l </td><td>Germ cell-less homologue 1 (Drosophila)-like </td><td>11 </td><td>2 </td><td>9 </td><td> </td></tr><tr><td>LOC236749 </td><td>Hypothetical protein LOC236749 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>5 </td><td>chrX:50380769-52184422 </td><td>1 803 653 </td><td>48 </td><td>−1.75 </td><td>1 </td><td>1971 </td><td>Similar to Xmr protein </td><td>Adult male testis cDNA, RIKEN full-length enriched library, clone: 4930527E24 product: weakly similar to XMR PROTEIN </td><td>30 </td><td>16 </td><td>14 </td><td> </td></tr><tr><td>6 </td><td>chrX:57970833-58090999 </td><td>120 166 </td><td>5 </td><td>−2.14 </td><td> </td><td>23 </td><td>Ldoc1 </td><td>Leucine zipper, down-regulated in cancer 1 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>7 </td><td>chrX:58775958-58815172 </td><td>39 214 </td><td>13 </td><td>−3.45 </td><td> </td><td>10 </td><td><sup>a</sup>1700019B21Rik </td><td><em>Mus musculus</em> adult male testis cDNA, <em>M. musculus</em> RIKEN cDNA 1700019B21 gene (1700019B21Rik), transcript variant 1, non-coding RNA </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>8 </td><td>chrX:72448732-72544743 </td><td>96 011 </td><td>4 </td><td>−1.59 </td><td> </td><td>67 </td><td>LOC238829 </td><td>Hypothetical protein LOC238829 (AK133378—<em>M. musculus</em> adult male testis cDNA, RIKEN full-length enriched library, clone: 4933402E19 product: hypothetical protein, full insert sequence) </td><td>4 </td><td>1 </td><td>3 </td><td> </td></tr><tr><td rowspan="2">9 </td><td rowspan="2">chrX:84663998-86056286 </td><td rowspan="2">1 392 288 </td><td rowspan="2">23 </td><td rowspan="2">−1.2207 </td><td rowspan="2">  </td><td rowspan="2">334 </td><td>Pet2 </td><td>Plasmacytoma expressed transcript 2 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>4932429P05Rik </td><td><em>M. musculus</em> RIKEN cDNA 4932429P05 gene (4932429P05Rik), mRNA. (<em>M. musculus</em> adult male testis cDNA) SMEK homologue 3, putative </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>10 </td><td>chrX:87564706-87579103 </td><td>14 397 </td><td>20 </td><td>−3.70 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1, Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="2">11 </td><td rowspan="2">chrX:87598220-88271067 </td><td rowspan="2">672 847 </td><td rowspan="2">15 </td><td rowspan="2">−1.0483 </td><td rowspan="2">  </td><td rowspan="2">43 </td><td>Mageb5 </td><td>Melanoma antigen, family B, 5 </td><td>2 </td><td>2 </td><td>0 </td><td>CTA </td></tr><tr><td>Mageb1 </td><td>Melanoma antigen, family B, 1 </td><td>2 </td><td>1 </td><td>1 </td><td>CTA </td></tr><tr><td>12 </td><td>chrX:88271564-88285701 </td><td>14 137 </td><td>21 </td><td>−3.76 </td><td> </td><td>1 </td><td> </td><td>(up site of Mageb1,Mageb2) </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td rowspan="4">13 </td><td rowspan="4">chrX:103040719-103447968 </td><td rowspan="4">407 249 </td><td rowspan="4">10 </td><td rowspan="4">−1.186 </td><td rowspan="4">  </td><td rowspan="4">158 </td><td>A630033H20Rik </td><td>Hypothetical protein LOC213438 (product: similar to putative purinergic receptor P2Y10) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Gpr23 </td><td>G protein-coupled receptor 23 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>P2ry10 </td><td>Purinergic receptor P2Y, G-protein coupled 10 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>Zcchc5 </td><td>Zinc finger, CCHC domain containing 5 </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>14 </td><td>chrX:142924085-145405975 </td><td>2 481 890 </td><td>112 </td><td>−1.0089 </td><td>2 </td><td>1666 </td><td>Ott </td><td>Ovary testis transcribed </td><td>17 </td><td>16 </td><td>1 </td><td>CTA </td></tr><tr><td rowspan="2">15 </td><td rowspan="2">chrX:149333243-150403389 </td><td rowspan="2">1 070 146 </td><td rowspan="2">41 </td><td rowspan="2">−1.4149 </td><td rowspan="2">  </td><td rowspan="2">211 </td><td>Magea </td><td>Melanoma antigen family A 2, 3, 5, 6, 8 </td><td>7 </td><td>5 </td><td>2 </td><td>CTA </td></tr><tr><td>Samt4 </td><td><em>M. musculus</em> spermatogenesis associated multi-pass transmembrane protein 4 (Samt4), mRNA (hypothetical protein LOC75185) </td><td>1 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>16 </td><td>chrX:160673416-161229042 </td><td>555 626 </td><td>13 </td><td>−1.13 </td><td> </td><td>274 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Non-coding RNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6212973" target="_blank">View Large</a></div></div><p>Hypomethylation of two such domains, LoD 10 and 12, was confirmed by Southern blot analysis (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C and D). Since LoD 10 and 12 contain homologous, locally repeated sequences, a hybridization probe can be used to assess the methylation status of both regions. The genomic DNAs of the thymus, brain and testis were digested by either methylation-sensitive <em>Hpa</em>II or the methylation-insensitive <em>Msp</em>I. In the <em>Hpa</em>II digests of thymus and brain DNA, no bands were detected except for a hybridization signal in the unresolved part of the lanes, indicating that the genomic region is hypermethylated in somatic organs. In the <em>Hpa</em>II digest of testis DNA, many bands were detected, and the band pattern was essentially the same as that found in the <em>Msp</em>I digest, clearly indicating that this region is largely unmethylated in the testis. Given that the testis comprises both germ and somatic cells, we asked whether LoDs are hypomethylated in germ cells. We used an <em>Mvh</em><sup><a reveal-id="DST030C35" data-open="DST030C35" class="link link-ref link-reveal xref-bibr">35</a></sup>-<em>Venus</em> reporter transgenic mouse line, in which germ cells are marked by <em>Venus</em> fluorescence protein (Mise and Abe, unpublished results). We also used FACS to purify the <em>Venus</em>-positive germ cells from the adult testis and performed Southern analysis. The results indicated that the genomic regions in the purified germ cells are indeed hypomethylated (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C and D).</p>                            <h3 scrollto-destination=6212975 id="6212975" class="section-title" data-legacy-id=s3h>3.8. Predominance of genes expressed in male germ cells or in the testis in LoDs</h3>
<p>We noticed that most LoDs contain genes that are expressed in the testis. For example, <em>Gmcl1l</em> (germ cell-less protein-like 1-like), <em>Ssx9</em>, <em>Fthl17</em>, <em>Xmr</em>, <em>Mageb</em>, <em>Ott</em>, <em>Samt4</em> and <em>Magea</em> are expressed in the testis and are included in LoDs 1, 2, 3, 4, 11, 14 and 15, respectively (Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a>). Expression of these genes are also detected in germ cells purified from adult testes (data not shown). If we omit LoDs 10, 12 and 16, which do not carry known genes, only LoDs 6 and 13 do not contain genes predominantly expressed in germ cells (Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The mean expression levels of genes contained in LoDs are shown in Fig. <a reveal-id="DST030F5" data-open="DST030F5" class="link link-reveal link-table xref-fig">5</a>A. Genes within LoDs show significantly higher expression in the testis than in the brain. Figure <a reveal-id="DST030F5" data-open="DST030F5" class="link link-reveal link-table xref-fig">5</a>B shows the mean levels of DNA methylation within and outside LoDs on the mouse X chromosome. Figure <a reveal-id="DST030F5" data-open="DST030F5" class="link link-reveal link-table xref-fig">5</a> indicates that there is an inverse correlation between the level of DNA methylation and the expression of genes in LoDs. </p>                        <a id="6212977" scrollto-destination="6212977"></a>
<div id="DST030F5" data-id="DST030F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03005.png?Expires=2147483647&Signature=gsZP5VQRJOjbMb6jpqawsxOu4DchPD88KF70-D6HQjd8j4PQoJhXxYsXA0Ac0dq3D8NvJATmjNxjzkzGnYiGdmiGBpyTifVvhDD1PSQ5Rl~9~kGdxj0ys8Vy2atVCWefdGqxvG8r30etfPVodFmDrj5vbPH-BFmR2DxRdqV00-vHQOEs9NNX8lT01D16t8TzvIPKiWmfTnBvIDiVZkuwXqhEVKRSQjznQ~kSmsLIBlF8sTl0hq4AGeJFIvUnCTpRnWNuERb6oPxjBlr3TAKao3sDMaK9FSwPipyz21lONIc0AISizBS9zPafFFL5hpIlgaNQJMxNurnNZEpNQQpeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inverse relationship between DNA methylation and expression of genes within the LoDs. (A) A plot of the expression levels of genes contained in LoDs, and regions outside LoDs on the mouse X chromosome. Gene expression data were obtained from the Affymetrix Exon array data set.43 The expression values of exons contained in LoDs were averaged and plotted, and the data points from regions outside LoDs were similarly averaged. For statistical analysis of the data from regions outside LoDs, the same number of data points as those used to analyse within LoDs were randomly selected and used. (B) Mean methylation levels of genomic DNA within LoDs. The mean M-values from the CCGG segments contained in LoDs or in regions outside LoDs are shown. Statistical significance was tested by Wilcoxon t-test, and P-values are shown within the figures." path-from-xml="dst03005.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212977/dst03005.tiff" path-from-xml="dst03005.tiff" target="_blank">View large</a><a section="6212977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03005.png?Expires=2147483647&Signature=hzA~8jvSGrbZy9GUgZWV8D~0IKNpZivtoPpyu9sr6O0xfSt-wMsgQzMu1rVqhFp~RpttOu2ReMonL2f2A9oHg12BGRzKdAguH7LIbEYYib2-CaHtacNneaKmmJyXKyomSejAVBV0iGrdTji-mdZfBcmRVvzK~FQxR4JfSN0ULpEr~a-caUv7FPu0JlP-58A1bIax58SUzchuPVvqLo83-ngsekPGYZaOOfRrIMedxLAozNejSNewb6K-G4dW28vTSbZHI9eecjvZOSHX03ht16Qxg~DY8mMjzRBH0DCeH-vmLDj6TAmaG4rctkG7GMm4MNjDV-lcqo2jzKwP-mr1RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212977&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03005.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inverse relationship between DNA methylation and expression of genes within the LoDs. (A) A plot of the expression levels of genes contained in LoDs, and regions outside LoDs on the mouse X chromosome. Gene expression data were obtained from the Affymetrix Exon array data set.<sup><a reveal-id="DST030C43" data-open="DST030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The expression values of exons contained in LoDs were averaged and plotted, and the data points from regions outside LoDs were similarly averaged. For statistical analysis of the data from regions outside LoDs, the same number of data points as those used to analyse within LoDs were randomly selected and used. (B) Mean methylation levels of genomic DNA within LoDs. The mean <em>M</em>-values from the CCGG segments contained in LoDs or in regions outside LoDs are shown. Statistical significance was tested by Wilcoxon <em>t-</em>test, and <em>P</em>-values are shown within the figures.</p></div></div><div content-id="DST030F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03005.png?Expires=2147483647&Signature=gsZP5VQRJOjbMb6jpqawsxOu4DchPD88KF70-D6HQjd8j4PQoJhXxYsXA0Ac0dq3D8NvJATmjNxjzkzGnYiGdmiGBpyTifVvhDD1PSQ5Rl~9~kGdxj0ys8Vy2atVCWefdGqxvG8r30etfPVodFmDrj5vbPH-BFmR2DxRdqV00-vHQOEs9NNX8lT01D16t8TzvIPKiWmfTnBvIDiVZkuwXqhEVKRSQjznQ~kSmsLIBlF8sTl0hq4AGeJFIvUnCTpRnWNuERb6oPxjBlr3TAKao3sDMaK9FSwPipyz21lONIc0AISizBS9zPafFFL5hpIlgaNQJMxNurnNZEpNQQpeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Inverse relationship between DNA methylation and expression of genes within the LoDs. (A) A plot of the expression levels of genes contained in LoDs, and regions outside LoDs on the mouse X chromosome. Gene expression data were obtained from the Affymetrix Exon array data set.43 The expression values of exons contained in LoDs were averaged and plotted, and the data points from regions outside LoDs were similarly averaged. For statistical analysis of the data from regions outside LoDs, the same number of data points as those used to analyse within LoDs were randomly selected and used. (B) Mean methylation levels of genomic DNA within LoDs. The mean M-values from the CCGG segments contained in LoDs or in regions outside LoDs are shown. Statistical significance was tested by Wilcoxon t-test, and P-values are shown within the figures." path-from-xml="dst03005.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212977/dst03005.tiff" path-from-xml="dst03005.tiff" target="_blank">View large</a><a section="6212977" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03005.png?Expires=2147483647&Signature=hzA~8jvSGrbZy9GUgZWV8D~0IKNpZivtoPpyu9sr6O0xfSt-wMsgQzMu1rVqhFp~RpttOu2ReMonL2f2A9oHg12BGRzKdAguH7LIbEYYib2-CaHtacNneaKmmJyXKyomSejAVBV0iGrdTji-mdZfBcmRVvzK~FQxR4JfSN0ULpEr~a-caUv7FPu0JlP-58A1bIax58SUzchuPVvqLo83-ngsekPGYZaOOfRrIMedxLAozNejSNewb6K-G4dW28vTSbZHI9eecjvZOSHX03ht16Qxg~DY8mMjzRBH0DCeH-vmLDj6TAmaG4rctkG7GMm4MNjDV-lcqo2jzKwP-mr1RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212977&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03005.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Inverse relationship between DNA methylation and expression of genes within the LoDs. (A) A plot of the expression levels of genes contained in LoDs, and regions outside LoDs on the mouse X chromosome. Gene expression data were obtained from the Affymetrix Exon array data set.<sup><a reveal-id="DST030C43" data-open="DST030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The expression values of exons contained in LoDs were averaged and plotted, and the data points from regions outside LoDs were similarly averaged. For statistical analysis of the data from regions outside LoDs, the same number of data points as those used to analyse within LoDs were randomly selected and used. (B) Mean methylation levels of genomic DNA within LoDs. The mean <em>M</em>-values from the CCGG segments contained in LoDs or in regions outside LoDs are shown. Statistical significance was tested by Wilcoxon <em>t-</em>test, and <em>P</em>-values are shown within the figures.</p></div></div>                            <h3 scrollto-destination=6212978 id="6212978" class="section-title" data-legacy-id=s3i>3.9. Genomic structures of LoDs: <em>Xmr</em>/<em>Slx</em> and <em>Mageb</em> regions</h3>
<p>In addition, we demonstrate the detailed structures of two LoD regions (Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>). LoD 4 is ∼9.1 Mb in size (chrX: 22 991 291–32 117 922) and contains three distinct genes/gene families, all of which are expressed specifically in the testis. Because <em>Xmr</em> is a synonymous gene with <em>Slx</em>,<sup><a reveal-id="DST030C29" data-open="DST030C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST030C30" data-open="DST030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> we call this gene/gene family either <em>Xmr</em> or <em>Xmr/Slx</em> in this study. <em>Xmr/Slx</em> is known to be expressed in spermatids, where it encodes a protein, SLX/XMR, normally localized in cytoplasm.<sup><a reveal-id="DST030C29" data-open="DST030C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST030C30" data-open="DST030C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>Xmr/Slx</em> represents a locally duplicated multigene family, whose copy number is at least 28 in LoD 4. <em>Gmcl1l</em> and <em>LOC236749</em> are included in the same LoD, and both are expressed in the testis and in purified male germ cells (Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>A; data not shown). </p>                        <a id="6212980" scrollto-destination="6212980"></a>
<div id="DST030F6" data-id="DST030F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03006.png?Expires=2147483647&Signature=iifjIXEouOwd7IlCKhsvRAyOgXozi9dHsFfn2JbSfiYzTxyWeuxxRNHrKDYtm0~6Z-De~Lirv28SYqgjqTQX0REBmFVgsgdDuRKTgyhtI7yDjnMBUY6Be6GxQyKzeVYrjIRjFppFVYFMK-lif-ijBuXMkRvAnIO5ZM6WMLueNXhyFEYBoEaAi3WQPwlYYhus-XcXK64GnD9AU5dVuBbPVsKpRMJYcG3m7ULdMyKmzMH123AfNOxNJiJb-BAKBzouffPvabfH4Oasya50m6iUIx3HEfvu7BjGLtUAlgm4kMU4Kyt7UXkBwAo~Pms~Mhh4MqyIT8mmuWmPRpSckuU6XQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structures of LoDs: Xmr and Mageb. (A) Genomic structure of LoD 4 (Xmr). The top section represents a similarity dot plot44 of LoD 4 and flanking regions (chrX: 21 000 000–32 940 000; UCSC mm8). Similarities of the sequences are colour-coded as shown by the colour bar on the right (high similarity in dark red, 100% similarity; low similarity in blue, 70% similarity). The horizontal lines represent direct repeats, and the vertical lines indicate IRs. The middle section depicts the locations of genes contained in LoD 4 (coloured rectangle). Exon array expression data for the brain and testis43 are shown at the bottom (high gene expression in red and low expression in green). The positions of sequences with homology to the Xmr cDNA probe are also shown by grey vertical bars (Southern probe). (B) Southern blot probed with the Xmr cDNA. Genomic DNAs from three organs were digested with either MspI or HpaII. Xmr cDNA probe (708 bp) was amplified from testis cDNA using primers, Xmr FW: 5′-AAGGGTGCAGTTGTGAAGGT-3′, Xmr Rv: 5′-TGTTGGTCTCCATGTTCATCA-3′. The hybridization signals in the unresolved part of the testis DNA blot are likely to reflect non-specific cross-hybridization. Southern blot analysis data using DNA doubly digested by BamHI plus either HpaII or MspI confirmed this notion (data not shown). (C) Genomic structure of LoDs 10, 11 and 12 (Mageb1/b2). Top: similarity dot plot. The vertical lines and coloured arrows indicate positions of IRs. The colouring of the similarities is the same as found in Fig. 6A. Homologous repeats are represented by the same colour. A magnified view of the IRs contained in LoDs 10 and 12 is presented on the left. Gene expression data for the brain and testis are shown at the bottom. The positions of sequences with homology to a probe used for Southern blot analysis (Fig. 4C and 4D) are shown by grey bars (Southern probe). (D) A DNA methylation heat map of LoDs 10 and 12. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, M-value = 6.00; highly methylated segments in dark red, M-value = –6.00)." path-from-xml="dst03006.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212980/dst03006.tiff" path-from-xml="dst03006.tiff" target="_blank">View large</a><a section="6212980" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03006.png?Expires=2147483647&Signature=4tgRu5ZiJcVo57TFHicaPrlA~ezjnduwdo-Pts7QHLZrhnRfwcs88oOhGspCon2oj3jWwz2igxyGq9co~BYRdc7Kr9dvIqJ4y3N7fSGMFPrvOOJhWGVEOO4eH4oWr3d2pUPPfI-TuG~HvLv4JOkMjgN~1Yby8KT~7TuUd8gR86DRC7z1EoGBAkNTuSTiYlBC63EIQQ38Fb83I8wzftglFhtO5GswyvMKwn5iqAvJ0rrARZYnvvV09pl5jLN8EGz-3LlWUcAPV4jZDzPddhQSj7tkmK0owAVuWmZ-pry5keYfDiqAJXtMdp00SD1OxaPzLaPXcs9zswHUED06T-KOuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212980&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03006.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structures of LoDs: <em>Xmr</em> and <em>Mageb</em>. (A) Genomic structure of LoD 4 (<em>Xmr</em>). The top section represents a similarity dot plot<sup><a reveal-id="DST030C44" data-open="DST030C44" class="link link-ref link-reveal xref-bibr">44</a></sup> of LoD 4 and flanking regions (chrX: 21 000 000–32 940 000; UCSC mm8). Similarities of the sequences are colour-coded as shown by the colour bar on the right (high similarity in dark red, 100% similarity; low similarity in blue, 70% similarity). The horizontal lines represent direct repeats, and the vertical lines indicate IRs. The middle section depicts the locations of genes contained in LoD 4 (coloured rectangle). Exon array expression data for the brain and testis<sup><a reveal-id="DST030C43" data-open="DST030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> are shown at the bottom (high gene expression in red and low expression in green). The positions of sequences with homology to the <em>Xmr</em> cDNA probe are also shown by grey vertical bars (Southern probe). (B) Southern blot probed with the <em>Xmr</em> cDNA. Genomic DNAs from three organs were digested with either <em>Msp</em>I or <em>Hpa</em>II. <em>Xmr</em> cDNA probe (708 bp) was amplified from testis cDNA using primers, Xmr FW: 5′-AAGGGTGCAGTTGTGAAGGT-3′, Xmr Rv: 5′-TGTTGGTCTCCATGTTCATCA-3′. The hybridization signals in the unresolved part of the testis DNA blot are likely to reflect non-specific cross-hybridization. Southern blot analysis data using DNA doubly digested by <em>Bam</em>HI plus either <em>Hpa</em>II or <em>Msp</em>I confirmed this notion (data not shown). (C) Genomic structure of LoDs 10, 11 and 12 (<em>Mageb1/b2</em>). Top: similarity dot plot. The vertical lines and coloured arrows indicate positions of IRs. The colouring of the similarities is the same as found in Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>A. Homologous repeats are represented by the same colour. A magnified view of the IRs contained in LoDs 10 and 12 is presented on the left. Gene expression data for the brain and testis are shown at the bottom. The positions of sequences with homology to a probe used for Southern blot analysis (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C and 4D) are shown by grey bars (Southern probe). (D) A DNA methylation heat map of LoDs 10 and 12. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, <em>M</em>-value = 6.00; highly methylated segments in dark red, <em>M</em>-value = –6.00).</p></div></div><div content-id="DST030F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03006.png?Expires=2147483647&Signature=iifjIXEouOwd7IlCKhsvRAyOgXozi9dHsFfn2JbSfiYzTxyWeuxxRNHrKDYtm0~6Z-De~Lirv28SYqgjqTQX0REBmFVgsgdDuRKTgyhtI7yDjnMBUY6Be6GxQyKzeVYrjIRjFppFVYFMK-lif-ijBuXMkRvAnIO5ZM6WMLueNXhyFEYBoEaAi3WQPwlYYhus-XcXK64GnD9AU5dVuBbPVsKpRMJYcG3m7ULdMyKmzMH123AfNOxNJiJb-BAKBzouffPvabfH4Oasya50m6iUIx3HEfvu7BjGLtUAlgm4kMU4Kyt7UXkBwAo~Pms~Mhh4MqyIT8mmuWmPRpSckuU6XQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structures of LoDs: Xmr and Mageb. (A) Genomic structure of LoD 4 (Xmr). The top section represents a similarity dot plot44 of LoD 4 and flanking regions (chrX: 21 000 000–32 940 000; UCSC mm8). Similarities of the sequences are colour-coded as shown by the colour bar on the right (high similarity in dark red, 100% similarity; low similarity in blue, 70% similarity). The horizontal lines represent direct repeats, and the vertical lines indicate IRs. The middle section depicts the locations of genes contained in LoD 4 (coloured rectangle). Exon array expression data for the brain and testis43 are shown at the bottom (high gene expression in red and low expression in green). The positions of sequences with homology to the Xmr cDNA probe are also shown by grey vertical bars (Southern probe). (B) Southern blot probed with the Xmr cDNA. Genomic DNAs from three organs were digested with either MspI or HpaII. Xmr cDNA probe (708 bp) was amplified from testis cDNA using primers, Xmr FW: 5′-AAGGGTGCAGTTGTGAAGGT-3′, Xmr Rv: 5′-TGTTGGTCTCCATGTTCATCA-3′. The hybridization signals in the unresolved part of the testis DNA blot are likely to reflect non-specific cross-hybridization. Southern blot analysis data using DNA doubly digested by BamHI plus either HpaII or MspI confirmed this notion (data not shown). (C) Genomic structure of LoDs 10, 11 and 12 (Mageb1/b2). Top: similarity dot plot. The vertical lines and coloured arrows indicate positions of IRs. The colouring of the similarities is the same as found in Fig. 6A. Homologous repeats are represented by the same colour. A magnified view of the IRs contained in LoDs 10 and 12 is presented on the left. Gene expression data for the brain and testis are shown at the bottom. The positions of sequences with homology to a probe used for Southern blot analysis (Fig. 4C and 4D) are shown by grey bars (Southern probe). (D) A DNA methylation heat map of LoDs 10 and 12. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, M-value = 6.00; highly methylated segments in dark red, M-value = –6.00)." path-from-xml="dst03006.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212980/dst03006.tiff" path-from-xml="dst03006.tiff" target="_blank">View large</a><a section="6212980" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03006.png?Expires=2147483647&Signature=4tgRu5ZiJcVo57TFHicaPrlA~ezjnduwdo-Pts7QHLZrhnRfwcs88oOhGspCon2oj3jWwz2igxyGq9co~BYRdc7Kr9dvIqJ4y3N7fSGMFPrvOOJhWGVEOO4eH4oWr3d2pUPPfI-TuG~HvLv4JOkMjgN~1Yby8KT~7TuUd8gR86DRC7z1EoGBAkNTuSTiYlBC63EIQQ38Fb83I8wzftglFhtO5GswyvMKwn5iqAvJ0rrARZYnvvV09pl5jLN8EGz-3LlWUcAPV4jZDzPddhQSj7tkmK0owAVuWmZ-pry5keYfDiqAJXtMdp00SD1OxaPzLaPXcs9zswHUED06T-KOuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212980&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03006.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structures of LoDs: <em>Xmr</em> and <em>Mageb</em>. (A) Genomic structure of LoD 4 (<em>Xmr</em>). The top section represents a similarity dot plot<sup><a reveal-id="DST030C44" data-open="DST030C44" class="link link-ref link-reveal xref-bibr">44</a></sup> of LoD 4 and flanking regions (chrX: 21 000 000–32 940 000; UCSC mm8). Similarities of the sequences are colour-coded as shown by the colour bar on the right (high similarity in dark red, 100% similarity; low similarity in blue, 70% similarity). The horizontal lines represent direct repeats, and the vertical lines indicate IRs. The middle section depicts the locations of genes contained in LoD 4 (coloured rectangle). Exon array expression data for the brain and testis<sup><a reveal-id="DST030C43" data-open="DST030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> are shown at the bottom (high gene expression in red and low expression in green). The positions of sequences with homology to the <em>Xmr</em> cDNA probe are also shown by grey vertical bars (Southern probe). (B) Southern blot probed with the <em>Xmr</em> cDNA. Genomic DNAs from three organs were digested with either <em>Msp</em>I or <em>Hpa</em>II. <em>Xmr</em> cDNA probe (708 bp) was amplified from testis cDNA using primers, Xmr FW: 5′-AAGGGTGCAGTTGTGAAGGT-3′, Xmr Rv: 5′-TGTTGGTCTCCATGTTCATCA-3′. The hybridization signals in the unresolved part of the testis DNA blot are likely to reflect non-specific cross-hybridization. Southern blot analysis data using DNA doubly digested by <em>Bam</em>HI plus either <em>Hpa</em>II or <em>Msp</em>I confirmed this notion (data not shown). (C) Genomic structure of LoDs 10, 11 and 12 (<em>Mageb1/b2</em>). Top: similarity dot plot. The vertical lines and coloured arrows indicate positions of IRs. The colouring of the similarities is the same as found in Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>A. Homologous repeats are represented by the same colour. A magnified view of the IRs contained in LoDs 10 and 12 is presented on the left. Gene expression data for the brain and testis are shown at the bottom. The positions of sequences with homology to a probe used for Southern blot analysis (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C and 4D) are shown by grey bars (Southern probe). (D) A DNA methylation heat map of LoDs 10 and 12. DNA methylation levels of the CCGG segments are represented as a heat map (unmethylated segments in dark blue, <em>M</em>-value = 6.00; highly methylated segments in dark red, <em>M</em>-value = –6.00).</p></div></div><p>The LoD 4 region represents one of the largest segmentally duplicated regions on the mouse X chromosome (Katsura and Satta, personal communication) and can be divided into four subregions (<a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst030/-/DC1" target="_blank">Supple-mentary Fig. S4</a>). Subregion I spans ∼3 Mb and harbours tandemly repeated <em>Xmr</em> genes. Subregion II spans ∼3.8 Mb and comprises both tandem and inverted repeats (IRs) of <em>Xmr</em> genes. Subregion III contains tandem and IRs of <em>Gmcl1l</em> genes, which are duplicated on two distant sites on the X chromosome; the other site is also classified as LoD 1 (Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a>). Subregion IV is &lt;1 Mb and contains tandem repeats of <em>Xmr</em> genes. One hundred and forty-one CCGG segments are mapped within LoD 4, and the fold difference in methylation level (brain versus testis) of these segments were calculated as described in Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>A. The mean value is –1.4745, suggesting that the CCGG segments in this region are generally hypomethylated in the testis genome (Table <a reveal-id="DST030TB1" data-open="DST030TB1" class="link link-reveal link-table xref-fig">1</a> and Figs <a reveal-id="DST030F3" data-open="DST030F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>Because of the repetitive nature of the LoD region, HELP probes cannot be assigned for most of the subregions II and IV. To examine DNA methylation in these regions, we performed Southern analysis of the testis and brain DNA digested with either <em>Hpa</em>II or <em>Msp</em>I, and hybridized with an <em>Xmr</em> cDNA probe. As shown in Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>, the <em>Xmr</em> cDNA probe should be able to assess the methylation status of 161 restriction fragments. These fragments are distributed evenly within subregions I, II and IV, and fill the gaps of information provided by the nanoHELP assay, which tests only unique sequences. The results of the Southern blot analysis shown in Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>B demonstrate that the <em>Xmr</em> region is highly methylated in the brain and liver, whereas a considerable proportion of the restriction fragments appear unmethylated in the testis.</p><p>It has been suggested that transcriptionally active genes are hypomethylated in their promoter region, while their gene bodies tend to be hypermethylated.<sup><a reveal-id="DST030C6" data-open="DST030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, a magnified view of the LoD 4 region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>) indicates that all CCGG segments in this region are hypomethylated in the testis and male PGCs regardless of their positions with respect to the <em>Xmr</em> genes. Both the probes positioned near the transcription start sites and the probes positioned at introns or even at intergenic regions are unmethylated in the testis, GS and male PGCs. The Southern blot analysis data suggest that the CCGG segments containing exons of the <em>Xmr</em> genes appear to be relatively hypomethylated in the testis (Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>B). These results imply that methylation of the whole LoD 4 is subjected to a region-wide regulation. This feature is shared by other LoDs not described here.</p><p><em>Mageb</em> belongs to the <em>Mage</em> (melanoma antigen) gene family, which is expressed in spermatogenic cells and in some cancer cells.<sup><a reveal-id="DST030C31" data-open="DST030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Figure <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>C shows a genomic region spanning ∼1 Mb that contains <em>Mageb1</em> and <em>Mageb2</em> genes. This region represents a large IR with arms of ∼400 kb in length. At the ends of both arms, LoDs 10 and 12 are located 4–2 kb upstream of the transcription start sites of <em>Mageb1</em> and <em>Mageb2</em>, respectively. These two LoDs do not contain the <em>Mageb</em> locus itself (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Both LoDs are highly homologous and ∼14 kb long, and comprise repeat sequences with a unit size of ∼3 kb. These sequences are both tandem and IRs (Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>C; magnified part), are found only in these LoD regions and are clearly hypomethylated only in germ cells (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Hypomethylation of LoDs 10 and 12 was confirmed by Southern blot analysis as described (Fig. <a reveal-id="DST030F4" data-open="DST030F4" class="link link-reveal link-table xref-fig">4</a>C and D).</p>                            <h3 scrollto-destination=6212985 id="6212985" class="section-title" data-legacy-id=s3j>3.10. Developmental changes in the methylation levels of LoDs</h3>
<p>LoDs are hypomethylated in the testis, GS cells and male PGCs. The methylation heat maps of LoDs 10 and 12 shown in Fig. <a reveal-id="DST030F6" data-open="DST030F6" class="link link-reveal link-table xref-fig">6</a>D illustrate how the DNA methylation of LoDs changes during germ cell development. During development, the PGC genome undergoes global DNA demethylation, which is known to be completed between E11.5 and E13.5.<sup><a reveal-id="DST030C2" data-open="DST030C2" class="link link-ref link-reveal xref-bibr">2</a></sup> In E10.5 PGCs, the LoDs tested here are not unmethylated completely, whereas demethylation of LoD DNA progresses in PGCs by E13.5. At E17.5, the LoD regions are largely unmethylated in both male and female PGCs. This trend persists in later stages of male germ cells, whereas the methylation levels of the LoDs appear to increase in newborn oocytes. The results together with the results shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a> suggest that, in general, LoDs begin to form between E10.5 and E13.5, and distinct hypomethylated domains are established around E13.5 in the male germline. Despite the global increase in DNA methylation at later stages of male germline development (Fig. <a reveal-id="DST030F2" data-open="DST030F2" class="link link-reveal link-table xref-fig">2</a>B), hypomethylation of LoD DNAs is maintained in male germ cells. Although LoDs 10 and 12 are shared by male and female PGCs, the overall DNA methylation patterns are not identical, suggesting that a distinct epigenomic status is generated in male and female germlines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>).</p>                            <h3 scrollto-destination=6212988 id="6212988" class="section-title" data-legacy-id=s3k>3.11. Coincidence of most LoDs with broad domains of the repressive histone mark, H3K9 dimethylation</h3>
<p>We have shown that most LoDs are broad genomic domains with low DNA methylation levels that form boundaries between the LoDs and other methylated parts of the genome. The mammalian genome can be divided into broad domains of distinct histone modifications.<sup><a reveal-id="DST030C36" data-open="DST030C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> For example, LOCKs (large organized chromatin K9 modifications) are genomic domains with histone H3 lysine 9 dimethylation (H3K9me2) modification thought to be involved in region-wide gene repression.<sup><a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> To investigate the relationship between LoDs and the repressive histone mark, we performed ChIP-on-chip analysis to detect H3K9me2 enrichment in GS cells as a representative of germ cells in this test and in cumulus (somatic cells in the ovary) cells as a somatic cell control.<sup><a reveal-id="DST030C26" data-open="DST030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Figure <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a> shows the H3K9me2 modification patterns on the X chromosome in both GS and cumulus cells. The overall pattern of H3K9me2 modifications along the X chromosome in GS cells is essentially similar to that in cumulus cells (Fig. <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>A and C). Enrichment of the modifications along the LoD regions (coloured light blue) is seen in both GS and cumulus cells (Fig. <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>A and C). In contrast, as expected, DNA methylation levels in the LoD regions are high in cumulus cells and low in GS cells (Fig. <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>B and D). Figure <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>E shows a magnified view of LoD 12, indicating that the hypomethylated region has the H3K9me2 mark. A significant enrichment of H3K9me2 is found in most (11 of 16) LoDs in GS cells (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). </p>                        <a id="6212990" scrollto-destination="6212990"></a>
<div id="DST030F7" data-id="DST030F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03007.png?Expires=2147483647&Signature=PYQG6PF2ZA8RPK9KSN-s47~XBHHqacOCSlEdSxI7n8AzebNuoR4o~4iZr1fmJYu4sNRrKdJ7x4Cv5gpKenX7SLgc~Q~Zj4m1qLfDNmpOYFK~Rn5-FvRa-u05ese5zhSDKGbHVsC5-WgoLwgNrSgrAlAk6fMGaLZmybDMLb7XgywwUew12DcI~KWmjktGWQeUy~r9BNZovGgLYB1~XJMvTwHKc4hIDjpIxJ7VBMegLJw-IpCxSuqTk8M1o-ShWmg9YEggt9lXA6IDuH9rMC1vCTdTjWm3ggDt4O7VMV54BsWgci2aHZImzHzWhT3jKYbOatk6N2Yb0EjoXtvQGOzF9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Large stretches of H3K9 dimethyl modifications overlapping LoDs. (A) ChIP-on-chip data for the H3K9me2 modification along the X chromosome in GS cells. The green dots represent individual probe data. The positions of LoDs are shown in light blue. (B) DNA methylation profile of GS cells within the X chromosome. The blue line denotes the average M-values. The positions of LoDs are shown in light blue. (C) ChIP-on-chip data for H3K9me2 modification along the X chromosome in cumulus cells. (D) The DNA methylation profile of cumulus cells within the X chromosome. (E) A magnified view of the LoD 12 region. Top, the H3K9me2 modification in GS cells (blue) and in cumulus cells (pink). Middle, DNA methylation data for E10.5 male PGCs, E13.5 male PGCs, E17.5 male PGCs, P0.5 spermatogonia, GS cells, testis, brain, thymus and cumulus cells. Bottom, positions of LoD and Mageb1/b2 genes. (F) Quantitative reverse transcription polymerase chain reaction (RT-PCR) expression analysis of genes in LoD regions. The Ssx9 (Ssx), Fthl17 (Fthl), Xmr, Mageb1, Ott and Magea8 genes are located within LoDs on the X chromosome. Beta-actin (ACTB) and Gapdh (GAPD) genes were also examined. The expression level in the testis is set at 1.0, and the expression levels in other cells and tissues (brain, cumulus cells, GS cells and liver) relative to that in the testis are shown. Primers used for this RT-PCR analysis are listed in Supplementary Table S4A." path-from-xml="dst03007.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212990/dst03007.tiff" path-from-xml="dst03007.tiff" target="_blank">View large</a><a section="6212990" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03007.png?Expires=2147483647&Signature=tVbTtTlGhyb05V3u0N~5lIYzwZCGco9HlZa3GEnKl8lcV9NAaxCG4HkmkAhXNPTlUp8xV3nzqQKwQC4b-1AfIk~nWG13D2RfyNuw2V4xgBxpj4q4oJemlgdESoDlVeF5im2v1YhDzFntYfIONprTcOAoCUxLivcWE6pcxKTZj6dWp0U6ce-SNn-fMby0mt813eJcCTBc6CKinF3~xelwQe8a6d8F8g-B7CshGh2mXytVC1vVBlyz6e7O4q74~tdlC7AxbwKltRtw2xmle-bBWOnUsDfnQAn3pT-VN3tl4zqbRDIkmPJsnWNzYdWiicjR-WIfXEf2YP7gptWfFV~ahA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212990&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03007.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Large stretches of H3K9 dimethyl modifications overlapping LoDs. (A) ChIP-on-chip data for the H3K9me2 modification along the X chromosome in GS cells. The green dots represent individual probe data. The positions of LoDs are shown in light blue. (B) DNA methylation profile of GS cells within the X chromosome. The blue line denotes the average <em>M</em>-values. The positions of LoDs are shown in light blue. (C) ChIP-on-chip data for H3K9me2 modification along the X chromosome in cumulus cells. (D) The DNA methylation profile of cumulus cells within the X chromosome. (E) A magnified view of the LoD 12 region. Top, the H3K9me2 modification in GS cells (blue) and in cumulus cells (pink). Middle, DNA methylation data for E10.5 male PGCs, E13.5 male PGCs, E17.5 male PGCs, P0.5 spermatogonia, GS cells, testis, brain, thymus and cumulus cells. Bottom, positions of LoD and <em>Mageb1/b2</em> genes. (F) Quantitative reverse transcription polymerase chain reaction (RT-PCR) expression analysis of genes in LoD regions. The <em>Ssx9</em> (Ssx), <em>Fthl17</em> (Fthl), <em>Xmr</em>, <em>Mageb1</em>, <em>Ott</em> and <em>Magea8</em> genes are located within LoDs on the X chromosome. <em>Beta-actin</em> (ACTB) and <em>Gapdh</em> (GAPD) genes were also examined. The expression level in the testis is set at 1.0, and the expression levels in other cells and tissues (brain, cumulus cells, GS cells and liver) relative to that in the testis are shown. Primers used for this RT-PCR analysis are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4A</a>.</p></div></div><div content-id="DST030F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/m_dst03007.png?Expires=2147483647&Signature=PYQG6PF2ZA8RPK9KSN-s47~XBHHqacOCSlEdSxI7n8AzebNuoR4o~4iZr1fmJYu4sNRrKdJ7x4Cv5gpKenX7SLgc~Q~Zj4m1qLfDNmpOYFK~Rn5-FvRa-u05ese5zhSDKGbHVsC5-WgoLwgNrSgrAlAk6fMGaLZmybDMLb7XgywwUew12DcI~KWmjktGWQeUy~r9BNZovGgLYB1~XJMvTwHKc4hIDjpIxJ7VBMegLJw-IpCxSuqTk8M1o-ShWmg9YEggt9lXA6IDuH9rMC1vCTdTjWm3ggDt4O7VMV54BsWgci2aHZImzHzWhT3jKYbOatk6N2Yb0EjoXtvQGOzF9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Large stretches of H3K9 dimethyl modifications overlapping LoDs. (A) ChIP-on-chip data for the H3K9me2 modification along the X chromosome in GS cells. The green dots represent individual probe data. The positions of LoDs are shown in light blue. (B) DNA methylation profile of GS cells within the X chromosome. The blue line denotes the average M-values. The positions of LoDs are shown in light blue. (C) ChIP-on-chip data for H3K9me2 modification along the X chromosome in cumulus cells. (D) The DNA methylation profile of cumulus cells within the X chromosome. (E) A magnified view of the LoD 12 region. Top, the H3K9me2 modification in GS cells (blue) and in cumulus cells (pink). Middle, DNA methylation data for E10.5 male PGCs, E13.5 male PGCs, E17.5 male PGCs, P0.5 spermatogonia, GS cells, testis, brain, thymus and cumulus cells. Bottom, positions of LoD and Mageb1/b2 genes. (F) Quantitative reverse transcription polymerase chain reaction (RT-PCR) expression analysis of genes in LoD regions. The Ssx9 (Ssx), Fthl17 (Fthl), Xmr, Mageb1, Ott and Magea8 genes are located within LoDs on the X chromosome. Beta-actin (ACTB) and Gapdh (GAPD) genes were also examined. The expression level in the testis is set at 1.0, and the expression levels in other cells and tissues (brain, cumulus cells, GS cells and liver) relative to that in the testis are shown. Primers used for this RT-PCR analysis are listed in Supplementary Table S4A." path-from-xml="dst03007.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6212990/dst03007.tiff" path-from-xml="dst03007.tiff" target="_blank">View large</a><a section="6212990" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst03007.png?Expires=2147483647&Signature=tVbTtTlGhyb05V3u0N~5lIYzwZCGco9HlZa3GEnKl8lcV9NAaxCG4HkmkAhXNPTlUp8xV3nzqQKwQC4b-1AfIk~nWG13D2RfyNuw2V4xgBxpj4q4oJemlgdESoDlVeF5im2v1YhDzFntYfIONprTcOAoCUxLivcWE6pcxKTZj6dWp0U6ce-SNn-fMby0mt813eJcCTBc6CKinF3~xelwQe8a6d8F8g-B7CshGh2mXytVC1vVBlyz6e7O4q74~tdlC7AxbwKltRtw2xmle-bBWOnUsDfnQAn3pT-VN3tl4zqbRDIkmPJsnWNzYdWiicjR-WIfXEf2YP7gptWfFV~ahA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6212990&ar=419528&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03007.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Large stretches of H3K9 dimethyl modifications overlapping LoDs. (A) ChIP-on-chip data for the H3K9me2 modification along the X chromosome in GS cells. The green dots represent individual probe data. The positions of LoDs are shown in light blue. (B) DNA methylation profile of GS cells within the X chromosome. The blue line denotes the average <em>M</em>-values. The positions of LoDs are shown in light blue. (C) ChIP-on-chip data for H3K9me2 modification along the X chromosome in cumulus cells. (D) The DNA methylation profile of cumulus cells within the X chromosome. (E) A magnified view of the LoD 12 region. Top, the H3K9me2 modification in GS cells (blue) and in cumulus cells (pink). Middle, DNA methylation data for E10.5 male PGCs, E13.5 male PGCs, E17.5 male PGCs, P0.5 spermatogonia, GS cells, testis, brain, thymus and cumulus cells. Bottom, positions of LoD and <em>Mageb1/b2</em> genes. (F) Quantitative reverse transcription polymerase chain reaction (RT-PCR) expression analysis of genes in LoD regions. The <em>Ssx9</em> (Ssx), <em>Fthl17</em> (Fthl), <em>Xmr</em>, <em>Mageb1</em>, <em>Ott</em> and <em>Magea8</em> genes are located within LoDs on the X chromosome. <em>Beta-actin</em> (ACTB) and <em>Gapdh</em> (GAPD) genes were also examined. The expression level in the testis is set at 1.0, and the expression levels in other cells and tissues (brain, cumulus cells, GS cells and liver) relative to that in the testis are shown. Primers used for this RT-PCR analysis are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4A</a>.</p></div></div><p>Expression of six genes contained in the LoDs was examined in cumulus, GS, testis and two other somatic cell types by quantitative reverse transcription polymerase chain reaction (RT-PCR) analysis (Fig. <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>F). <em>Fthl17</em>, <em>Ott</em>, <em>Mageb</em> and <em>Magea</em> are included in the LoDs and are expressed in the testis; the expression level of these genes is much higher in GS cells, but only negligible expression is detected in somatic cells. This result indicates that genes in hypomethylated LoDs can be expressed even though the same region has continuous H3K9me2 modifications (Fig. <a reveal-id="DST030F7" data-open="DST030F7" class="link link-reveal link-table xref-fig">7</a>A and C and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>), demonstrating peculiar epigenomic features of LoD regions. It is reasonable to expect that <em>Ssx</em> and <em>Xmr</em> are barely detectable in GS cells, because these genes become active in post-meiotic stages,<sup><a reveal-id="DST030C16" data-open="DST030C16" class="link link-ref link-reveal xref-bibr">16</a></sup> whereas GS cells are derived from pre-meiotic spermatogonia. It is probable that, in GS cells, other factors required for the expression of post-meiotic genes (e.g. transcription factors) are lacking. These results suggest that DNA hypomethylation in LoDs may not be sufficient by itself, but is a prerequisite for the expression of LoD genes.</p>                            <h2 scrollto-destination=6212992 id="6212992" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In the present study, we have analysed the DNA methylation profiles of developing germ cells using the modified nanoHELP method, which requires only a limited amount of DNA. Recent studies by Guibert <em>et al.</em><sup><a reveal-id="DST030C9" data-open="DST030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> using the methylated DNA immunoprecipitation analysis of a promoter array and Seisenberger <em>et al.</em><sup><a reveal-id="DST030C10" data-open="DST030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> using a whole-genome bisulphite sequencing suggest that DNA demethylation of the PGC genome is initiated earlier than previously thought.<sup><a reveal-id="DST030C1" data-open="DST030C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST030C2" data-open="DST030C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Our finding that the PGC genome is substantially hypomethylated already at E10.5 is consistent with the result of Seisenberger <em>et al.</em>,<sup><a reveal-id="DST030C10" data-open="DST030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> confirming the technical reliability of our method. Our data from developing germ cells revealed for the first time the presence of large, hypomethylated DNA domains on the X chromosome of male germline cells in mice.</p>                            <h3 scrollto-destination=6212994 id="6212994" class="section-title" data-legacy-id=s4a>4.1. Discovery of large hypomethylated domains of epigenomic organization</h3>
<p>Traditionally, epigenetic studies have focused on modifications of genes or elements adjacent to genes. However, with the development of genome-wide assays, recent studies have revealed marked clustering of particular histone modifications over relatively large genomic regions; e.g. LOCKs and BLOCs (broad local enrichments) enriched with the histone marks H3K9me2 and H3K27me3, respectively.<sup><a reveal-id="DST030C36" data-open="DST030C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> These large epigenetic marks, LOCKs in particular, are thought to be involved in gene silencing. DNA methylation is found throughout the mammalian genome except for short unmethylated regions, CGIs, which typically occur around the transcription start sites of genes.<sup><a reveal-id="DST030C6" data-open="DST030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The LoDs described in this work are also hypomethylated genomic regions, but are distinct from CGIs in terms of their size, tissue specificity and genomic structure. To our knowledge, large differentially methylated DNA regions showing germ cell specificities, such as LoDs, have not been previously reported. This may be because previous studies have focused only on methylation of gene promoters and not broader genomic contexts in germ cell samples. In contrast, our custom HELP chip method could assess the DNA methylation status of both genic and intergenic regions using the meager amounts of DNA that could be sampled from germ cell genomes in this study.</p><p>Seisenberger <em>et al.</em><sup><a reveal-id="DST030C10" data-open="DST030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> recently reported the results of whole-genome bisulphite sequencing analysis of the mouse PGC genome. We analysed their data on E16.5 male PGCs to determine whether LoDs could be found at the single-nucleotide level and found that the number of sequence reads mapped to LoDs were significantly lower than that mapped to the flanking regions (data not shown). Given that mapping of bisulphite-converted short sequence reads onto locally duplicated regions is technically challenging, the probability of finding LoDs using the bisulphite sequencing data currently available seems low. In contrast, the HELP assay uses an <em>Msp</em>I control to remove the potentially confounding effect of copy number variation<sup><a reveal-id="DST030C12" data-open="DST030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> along with a probe design that selects unique sequences for hybridization. These ensure that the DNA methylation readout from regions of segmental duplication is genuinely reflective of the underlying DNA methylation. Oda <em>et al.</em><sup><a reveal-id="DST030C38" data-open="DST030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> reported that CGI methylation of an X-linked homeobox gene cluster spanning ∼1 Mb is under long-range regulation in a tissue-specific manner. Therefore, widespread changes in DNA methylation could occur depending on the cellular phenotype or differentiation status.</p>                            <h3 scrollto-destination=6212997 id="6212997" class="section-title" data-legacy-id=s4b>4.2. Peculiar epigenomic features of LoDs</h3>
<p>LoDs have been detected based on arbitrary criteria, but most share common features. Most LoDs represent segmental duplications that harbour germline-expressed genes and overlap with large H3K9me2-enriched domains. Wen <em>et al.</em><sup><a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> described large H3K9me2-enriched chromatin blocks, LOCKs, in the human and mouse. The occurrence of LOCKs is differentiation specific: there are more LOCKs in differentiated cells, and genes contained in the LOCKs tend to be repressed during differentiation. Because LOCKs substantially overlap with lamin B-associated domains, a gene-silencing mechanism based on three-dimensional subnuclear organization has been proposed.<sup><a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> We found that most LoDs are enriched with H3K9me2 modifications, and that at least four LoDs—1, 2, 3 and 4—correspond to the LOCKs described by Wen <em>et al.</em><sup><a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (data not shown). Overlaps of other LoDs with LOCKs cannot be checked because LOCKs data are not available for the rest of the mouse X chromosome. Overlap of LoDs with LOCKs is counterintuitive because LOCKs are supposed to repress gene expression, whereas genes can be highly expressed within LoDs. This may be reconciled if we assume that gene silencing in LoDs is complete when both DNA methylation and H3K9me2 marks are established, but is derepressed in the absence of DNA methylation. Consistent with this idea, somatic cells such as cumulus cells, which have both marks, do not express the LoD genes, although germ cell genes can be active in DNA-hypomethylated but H3K9-dimethylated LoDs. The H3K9me2 histone methyltransferases, G9a and GLP, are required for DNA methylation in ES cells, but not in cancer cells.<sup><a reveal-id="DST030C39" data-open="DST030C39" class="link link-ref link-reveal xref-bibr">39</a></sup> It is thus likely that DNA methylation and the H3K9me2 modification are not always interdependent, and that they can be regulated independently in the LoD regions of male germ cells and cancer cells.</p>                            <h3 scrollto-destination=6212999 id="6212999" class="section-title" data-legacy-id=s4c>4.3. Segmental duplication, hypomethylation and gene expression in germ cells and cancer cells</h3>
<p>Through the analysis of germ cell-specific hypomethylated regions, we found that LoDs overlap with large segmentally duplicated regions, within which germ cell-expressed genes are commonly found. Some of these genes, such as <em>Xmr</em>, are found only in rodents. In contrast, the <em>Mage</em> gene family genes, <em>Ssx</em> and <em>Fthl17</em>, are conserved in the human genome and are known as CTA genes, which are expressed specifically in germ cells and in some tumour cell types. More than 260 CTA genes have been detected in the human (<a class="link link-uri" href="http://www.cta.lncc.br/">http://www.cta.lncc.br/</a>), and half of them are on the X chromosome. Most of the X-linked CTA genes are organized as multicopy gene families.<sup><a reveal-id="DST030C18" data-open="DST030C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Warburton <em>et al.</em><sup><a reveal-id="DST030C40" data-open="DST030C40" class="link link-ref link-reveal xref-bibr">40</a></sup> searched the IR structures in the human genome and found that the X chromosome is replete with large IRs harbouring testis genes, most of which encode CTA genes. More than 40% of large IRs found in the mouse genome are on the X chromosome, and <em>Ssx</em>, <em>Fthl17</em> and the <em>Xmr</em> loci are contained in such regions. Thus, three kinds of studies with different starting points reached the same conclusion: the X chromosome is abundant with duplicated regions containing germ cell-expressed genes, including CTA genes. To this, we add the new observation that these regions also have unique epigenomic features, i.e. widespread DNA hypomethylation and H3K9me2 enrichment. The epigenomic features of the LoDs could account for the finding that CTA genes can be activated by inhibition of DNA methylation but not by a reduction in H3K9 dimethylation,<sup><a reveal-id="DST030C39" data-open="DST030C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST030C41" data-open="DST030C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and suggest that DNA methylation is the key epigenetic mechanism involved with the regulation of LoD–CTA genes. It is not fully understood how DNA methylation regulates the coordinated expression of CTA genes in a cell type-specific manner. It is also necessary to clarify whether CTA gene expression contributes directly to oncogenesis or whether it simply reflects global chromatin changes that occur during tumour formation. Simpson <em>et al.</em><sup><a reveal-id="DST030C42" data-open="DST030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> postulated an intriguing hypothesis that the aberrant expression of germline genes in cancer reflects the activation of the gametogenic programme, which is normally silenced in somatic cells. The gametogenic programme is normally repressed because germline-specific products would be harmful for normal somatic cells, whereas they would be advantageous for cancer cells. To test this hypothesis, it will be essential to elucidate the activation mechanism for the germline gene expression programme, as well as the epigenetic and chromatin status required for the operation of this programme. As shown in this study, widespread DNA hypomethylation may be a prerequisite for the activation of LoD genes, including CTA genes. In addition to DNA methylation, the nuclear chromatin environments within germ cells and/or tumour cells may also be important for long-range transcriptional control over large genomic regions, because LOCKs,<sup><a reveal-id="DST030C37" data-open="DST030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> LoDs and the partially methylated domains found in colorectal cancer<sup><a reveal-id="DST030C7" data-open="DST030C7" class="link link-ref link-reveal xref-bibr">7</a></sup> are correlated with nuclear lamina-associated domains. Therefore, further studies of epigenomic features and the nuclear architecture of LoDs may shed light on the germline gene expression programme and its relationship to oncogenesis.<sup><a reveal-id="DST030C42" data-open="DST030C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h2 scrollto-destination=6213001 id="6213001" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6213003 id="6213003" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported, in part, by Grants-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology of Japan to K.A.</p>                            <h2 scrollto-destination=6213005 id="6213005" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank Drs T. Suzuki for help with bisulphite pyrosequencing, S. Matoba and A. Ogura for cumulus samples, and Y. Katsura and Y. Satta for information and discussion on segmentally duplicated regions of the X chromosome. We also thank Ms Y. Koga for help with the experiments.</p>                            <h2 scrollto-destination=6213007 id="6213007" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST030C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hajkova</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Erhardt</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lane</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Epigenetic reprogramming in mouse primordial germ cells</div>, <div class="source">Mech. Dev.</div> , <div class="year">2002</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">15</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20reprogramming%20in%20mouse%20primordial%20germ%20cells&amp;author=P.%20Hajkova&amp;author=S.%20Erhardt&amp;author=N.%20Lane&amp;publication_year=2002&amp;journal=Mech.%20Dev.&amp;volume=117&amp;pages=15-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0925-4773(02)00181-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0925-4773(02)00181-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0925-4773(02)00181-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12204247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hajkova</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ancelin</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Waldmann</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Chromatin dynamics during epigenetic reprogramming in the mouse germ line</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">877</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20dynamics%20during%20epigenetic%20reprogramming%20in%20the%20mouse%20germ%20line&amp;author=P.%20Hajkova&amp;author=K.%20Ancelin&amp;author=T.%20Waldmann&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=877-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18354397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLaren</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Primordial germ cells in the mouse</div>, <div class="source">Dev. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">1</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primordial%20germ%20cells%20in%20the%20mouse&amp;author=A.%20McLaren&amp;publication_year=2003&amp;journal=Dev.%20Biol.&amp;volume=262&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0012-1606(03)00214-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0012-1606(03)00214-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0012-1606(03)00214-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14512014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borgel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Guibert</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Targets and dynamics of promoter DNA methylation during early mouse development</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">1093</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targets%20and%20dynamics%20of%20promoter%20DNA%20methylation%20during%20early%20mouse%20development&amp;author=J.%20Borgel&amp;author=S.%20Guibert&amp;author=Y.%20Li&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1093-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21057502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meissner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mikkelsen</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genome-scale DNA methylation maps of pluripotent and differentiated cells</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">454</div> (pg. <div class="fpage">766</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20DNA%20methylation%20maps%20of%20pluripotent%20and%20differentiated%20cells&amp;author=A.%20Meissner&amp;author=T.S.%20Mikkelsen&amp;author=H.%20Gu&amp;publication_year=2008&amp;journal=Nature&amp;volume=454&amp;pages=766-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18600261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Bird</div> <div class="given-names">A.</div></div>. </div><div class="article-title">DNA methylation landscapes: provocative insights from epigenomics</div>, <div class="source"><em>Nat. Rev. Genet</em>.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">465</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20landscapes%3A%20provocative%20insights%20from%20epigenomics&amp;author=M.M.%20Suzuki&amp;author=A.%20Bird&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=9&amp;pages=465-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berman</div> <div class="given-names">B.P.</div></div>,  <div class="name"><div class="surname">Weisenberger</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Aman</div> <div class="given-names">J.F.</div></div>, et al. </div><div class="article-title">Regions of focal DNA hypermethylation and long-range hypomethylation in colorectal cancer coincide with nuclear lamina-associated domains</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">40</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regions%20of%20focal%20DNA%20hypermethylation%20and%20long-range%20hypomethylation%20in%20colorectal%20cancer%20coincide%20with%20nuclear%20lamina-associated%20domains&amp;author=B.P.%20Berman&amp;author=D.J.%20Weisenberger&amp;author=J.F.%20Aman&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=40-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.969" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.969" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.969"> </span></div> </div></div></div></div></div><div content-id="DST030C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Popp</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dean</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome-wide erasure of DNA methylation in mouse primordial germ cells is affected by AID deficiency</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">1101</div>-<div class="lpage">05</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20erasure%20of%20DNA%20methylation%20in%20mouse%20primordial%20germ%20cells%20is%20affected%20by%20AID%20deficiency&amp;author=C.%20Popp&amp;author=W.%20Dean&amp;author=S.%20Feng&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=1101-05" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08829" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08829" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08829"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guibert</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Forné</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Global profiling of DNA methylation erasure in mouse primordial germ cells</div>, <div class="source">Genome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">633</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20profiling%20of%20DNA%20methylation%20erasure%20in%20mouse%20primordial%20germ%20cells&amp;author=S.%20Guibert&amp;author=T.%20Forn%C3%A9&amp;author=M.%20Weber&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=633-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.130997.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.130997.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.130997.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seisenberger</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Andrews</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Krueger</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">The dynamics of genome-wide DNA methylation reprogramming in mouse primordial germ cells</div>, <div class="source">Mol. Cell</div> , <div class="year">2012</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20dynamics%20of%20genome-wide%20DNA%20methylation%20reprogramming%20in%20mouse%20primordial%20germ%20cells&amp;author=S.%20Seisenberger&amp;author=S.%20Andrews&amp;author=F.%20Krueger&amp;publication_year=2012&amp;journal=Mol.%20Cell&amp;volume=48&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2012.11.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2012.11.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2012.11.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23062951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hackett</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Sengupta</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zylicz</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">Germline DNA demethylation dynamics and imprint erasure through 5-hydroxymethylcytosine</div>, <div class="source">Science</div> , <div class="year">2013</div>, vol. <div class="volume">339</div> (pg. <div class="fpage">448</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Germline%20DNA%20demethylation%20dynamics%20and%20imprint%20erasure%20through%205-hydroxymethylcytosine&amp;author=J.A.%20Hackett&amp;author=R.%20Sengupta&amp;author=J.J.%20Zylicz&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=448-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1229277" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1229277" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1229277"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23223451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khulan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Ye</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fazzari</div> <div class="given-names">M.J.</div></div>, et al. </div><div class="article-title">Comparative isoschizomer profiling of cytosine methylation: the HELP assay</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1046</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20isoschizomer%20profiling%20of%20cytosine%20methylation%3A%20the%20HELP%20assay&amp;author=B.%20Khulan&amp;author=R.F.%20Thompson&amp;author=K.%20Ye&amp;author=M.J.%20Fazzari&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1046-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5273806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5273806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5273806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16809668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Ladd-Acosta</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Carvalho</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Brandenburg</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Wen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Feinberg</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Comprehensive high-throughput arrays for relative methylation (CHARM)</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">780</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20high-throughput%20arrays%20for%20relative%20methylation%20(CHARM)&amp;author=R.A.%20Irizarry&amp;author=C.%20Ladd-Acosta&amp;author=B.%20Carvalho&amp;author=H.%20Wu&amp;author=S.A.%20Brandenburg&amp;author=B.%20Wen&amp;author=A.P.%20Feinberg&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=780-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7301508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7301508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7301508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18316654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Glass</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">R.F.</div></div>, et al. </div><div class="article-title">High-resolution genome-wide cytosine methylation profiling with simultaneous copy number analysis and optimization for limited cell numbers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">3829</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20genome-wide%20cytosine%20methylation%20profiling%20with%20simultaneous%20copy%20number%20analysis%20and%20optimization%20for%20limited%20cell%20numbers&amp;author=M.%20Oda&amp;author=J.L.%20Glass&amp;author=R.F.%20Thompson&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=3829-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp260" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp260" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp260"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19386619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">McCarrey</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">An abundance of X-linked genes expressed in spermatogonia</div>, <div class="source">Nat. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">422</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20abundance%20of%20X-linked%20genes%20expressed%20in%20spermatogonia&amp;author=P.J.%20Wang&amp;author=J.R.%20McCarrey&amp;author=F.%20Yang&amp;author=D.C.%20Page&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=27&amp;pages=422-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/86927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F86927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F86927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11279525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mueller</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Mahadevaiah</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Warburton</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Page</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">J.M.A.</div></div>. </div><div class="article-title">The mouse X chromosome is enriched for multicopy testis genes showing postmeiotic expression</div>, <div class="source">Nat. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">794</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mouse%20X%20chromosome%20is%20enriched%20for%20multicopy%20testis%20genes%20showing%20postmeiotic%20expression&amp;author=J.L.%20Mueller&amp;author=S.K.%20Mahadevaiah&amp;author=P.J.%20Park&amp;author=P.E.%20Warburton&amp;author=D.C.%20Page&amp;author=J.M.A.%20Turner&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=794-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugimoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">K.</div></div>. </div><div class="article-title">X chromosome reactivation initiates in nascent primordial germ cells in mice</div>, <div class="source">PLoS Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e116</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=X%20chromosome%20reactivation%20initiates%20in%20nascent%20primordial%20germ%20cells%20in%20mice&amp;author=M.%20Sugimoto&amp;author=K.%20Abe&amp;publication_year=2007&amp;journal=PLoS%20Genet.&amp;volume=3&amp;pages=e116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17676999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caballero</div> <div class="given-names">O.L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.-T.</div></div>. </div><div class="article-title">Cancer/testis (CT) antigens: potential targets for immunotherapy</div>, <div class="source">Cancer Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">2014</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%2Ftestis%20(CT)%20antigens%3A%20potential%20targets%20for%20immunotherapy&amp;author=O.L.%20Caballero&amp;author=Y.-T.%20Chen&amp;publication_year=2009&amp;journal=Cancer%20Sci.&amp;volume=100&amp;pages=2014-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cas.2009.100.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcas.2009.100.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcas.2009.100.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19719775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hilton</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Epigenotype switching of imprintable loci in embryonic germ cells</div>, <div class="source">Dev. Genes Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">207</div> (pg. <div class="fpage">551</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenotype%20switching%20of%20imprintable%20loci%20in%20embryonic%20germ%20cells&amp;author=T.%20Tada&amp;author=M.%20Tada&amp;author=K.%20Hilton&amp;publication_year=1998&amp;journal=Dev.%20Genes%20Evol.&amp;volume=207&amp;pages=551-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004270050146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004270050146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004270050146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9510550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mise</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fuchikami</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugimoto</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Differences and similarities in the developmental status of embryo-derived stem cells and primordial revealed by global expression profiling</div>, <div class="source">Genes Cells</div> , <div class="year">2008</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">863</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20and%20similarities%20in%20the%20developmental%20status%20of%20embryo-derived%20stem%20cells%20and%20primordial%20revealed%20by%20global%20expression%20profiling&amp;author=N.%20Mise&amp;author=T.%20Fuchikami&amp;author=M.%20Sugimoto&amp;publication_year=2008&amp;journal=Genes%20Cells&amp;volume=13&amp;pages=863-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/gtc.2008.13.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgtc.2008.13.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgtc.2008.13.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18782224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohbo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohmura</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Identifi-cation and characterization of stem cells in prepubertal spermatogenesis in mice</div>, <div class="source">Dev. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">258</div> (pg. <div class="fpage">209</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifi-cation%20and%20characterization%20of%20stem%20cells%20in%20prepubertal%20spermatogenesis%20in%20mice&amp;author=K.%20Ohbo&amp;author=S.%20Yoshida&amp;author=M.%20Ohmura&amp;publication_year=2003&amp;journal=Dev.%20Biol.&amp;volume=258&amp;pages=209-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0012-1606(03)00111-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0012-1606(03)00111-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0012-1606(03)00111-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12781694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanatsu-Shinohara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ogonuki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Miki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogura</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Toyokuni</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shinohara</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Long-term proliferation in culture and germline transmission of mouse male germline stem cells</div>, <div class="source">Biol. Reprod.</div> , <div class="year">2003</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">612</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-term%20proliferation%20in%20culture%20and%20germline%20transmission%20of%20mouse%20male%20germline%20stem%20cells&amp;author=M.%20Kanatsu-Shinohara&amp;author=N.%20Ogonuki&amp;author=K.%20Inoue&amp;author=H.%20Miki&amp;author=A.%20Ogura&amp;author=S.%20Toyokuni&amp;author=T.%20Shinohara&amp;publication_year=2003&amp;journal=Biol.%20Reprod.&amp;volume=69&amp;pages=612-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1095/biolreprod.103.017012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1095%2Fbiolreprod.103.017012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1095%2Fbiolreprod.103.017012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12700182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogonuki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirose</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high-speed congenic strategy using first-wave male germ cells</div>, <div class="source">PLoS One</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4943</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-speed%20congenic%20strategy%20using%20first-wave%20male%20germ%20cells&amp;author=N.%20Ogonuki&amp;author=K.%20Inoue&amp;author=M.%20Hirose&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e4943" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0004943" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0004943" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0004943"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19333383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fujihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mise</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">The X-linked imprinted gene family <em>Fthl17</em> shows predominantly female expression following the two-cell stage in mouse embryos</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">3672</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20X-linked%20imprinted%20gene%20family%20Fthl17%20shows%20predominantly%20female%20expression%20following%20the%20two-cell%20stage%20in%20mouse%20embryos&amp;author=S.%20Kobayashi&amp;author=Y.%20Fujihara&amp;author=N.%20Mise&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=3672-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20185572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ko</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ko</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nishiguchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Unbiased amplification of a highly complex mixture of DNA fragments by ‘lone linker’-tagged PCR</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">4293</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unbiased%20amplification%20of%20a%20highly%20complex%20mixture%20of%20DNA%20fragments%20by%20%E2%80%98lone%20linker%E2%80%99-tagged%20PCR&amp;author=M.%20Ko&amp;author=S.%20Ko&amp;author=N.%20Takahashi&amp;author=K.%20Nishiguchi&amp;author=K.%20Abe&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=4293-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.14.4293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.14.4293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.14.4293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2377489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inoue</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kohda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugimoto</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Impeding <em>Xist</em> expression from the active X chromosome improves mouse somatic cell cloning</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">330</div> (pg. <div class="fpage">496</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impeding%20Xist%20expression%20from%20the%20active%20X%20chromosome%20improves%20mouse%20somatic%20cell%20cloning&amp;author=K.%20Inoue&amp;author=T.%20Kohda&amp;author=M.%20Sugimoto&amp;publication_year=2010&amp;journal=Science&amp;volume=330&amp;pages=496-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1194174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1194174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1194174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20847234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Otsu</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Threshold selection method from gray-level histograms</div>, <div class="source">IEEE Trans. Syst. Man. Cybern.</div> , <div class="year">1979</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">62</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Threshold%20selection%20method%20from%20gray-level%20histograms&amp;author=N.%20Otsu&amp;publication_year=1979&amp;journal=IEEE%20Trans.%20Syst.%20Man.%20Cybern.&amp;volume=9&amp;pages=62-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TSMC.1979.4310076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTSMC.1979.4310076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTSMC.1979.4310076"> </span></div> </div></div></div></div></div><div content-id="DST030C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mahalanobis</div> <div class="given-names">P.C.</div></div>. </div><div class="article-title">On the generalized distance in statistics</div>, <div class="source">Proc. Natl Inst. Sci. India</div> , <div class="year">1936</div>, vol. <div class="volume">2</div> pg. <div class="fpage">49</div> </div></div></div></div><div content-id="DST030C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reynard</div> <div class="given-names">L.N.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">J.M.A.</div></div>,  <div class="name"><div class="surname">Cocquet</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Expression analysis of the mouse multi-copy X-linked gene, <em>Xlr</em>-related, meiosis-regulated (<em>Xmr</em>), reveals that <em>Xmr</em> encodes a spermatid-expressed cytoplasmic protein, <em>SLX/XMR</em></div>, <div class="source">Biol. Reprod.</div> , <div class="year">2007</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">329</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20analysis%20of%20the%20mouse%20multi-copy%20X-linked%20gene%2C%20Xlr-related%2C%20meiosis-regulated%20(Xmr)%2C%20reveals%20that%20Xmr%20encodes%20a%20spermatid-expressed%20cytoplasmic%20protein%2C%20SLX%2FXMR&amp;author=L.N.%20Reynard&amp;author=J.M.A.%20Turner&amp;author=J.%20Cocquet&amp;publication_year=2007&amp;journal=Biol.%20Reprod.&amp;volume=77&amp;pages=329-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1095/biolreprod.107.061101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1095%2Fbiolreprod.107.061101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1095%2Fbiolreprod.107.061101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17475928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cocquet</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ellis</div> <div class="given-names">P.J.I.</div></div>,  <div class="name"><div class="surname">Mahadevaiah</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Affara</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Vaiman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Burgoyne</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">A genetic basis for a postmeiotic X versus Y chromosome intragenomic conflict in the mouse</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002900</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20basis%20for%20a%20postmeiotic%20X%20versus%20Y%20chromosome%20intragenomic%20conflict%20in%20the%20mouse&amp;author=J.%20Cocquet&amp;author=P.J.I.%20Ellis&amp;author=S.K.%20Mahadevaiah&amp;author=N.A.%20Affara&amp;author=D.%20Vaiman&amp;author=P.S.%20Burgoyne&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23028340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clotman</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">De Backer</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">De Plaen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Boon</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Picard</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Cell- and stage-specific expression of mage genes during mouse spermatogenesis</div>, <div class="source">Mammal. Genome</div> , <div class="year">2000</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">696</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell-%20and%20stage-specific%20expression%20of%20mage%20genes%20during%20mouse%20spermatogenesis&amp;author=F.%20Clotman&amp;author=O.%20De%20Backer&amp;author=E.%20De%20Plaen&amp;author=T.%20Boon&amp;author=J.%20Picard&amp;publication_year=2000&amp;journal=Mammal.%20Genome&amp;volume=11&amp;pages=696-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s003350010116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs003350010116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs003350010116"> </span></div> </div></div></div></div></div><div content-id="DST030C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olshen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Venkatraman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lucito</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wigler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Circular binary segmentation for the analysis of array-based DNA copy number data</div>, <div class="source">Biostatistics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">557</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circular%20binary%20segmentation%20for%20the%20analysis%20of%20array-based%20DNA%20copy%20number%20data&amp;author=A.%20Olshen&amp;author=E.%20Venkatraman&amp;author=R.%20Lucito&amp;author=M.%20Wigler&amp;publication_year=2004&amp;journal=Biostatistics&amp;volume=5&amp;pages=557-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/biostatistics/kxh008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbiostatistics%2Fkxh008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbiostatistics%2Fkxh008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15475419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardiner-Garden</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Frommer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">CpG islands in vertebrate genomes</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1987</div>, vol. <div class="volume">196</div> (pg. <div class="fpage">261</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG%20islands%20in%20vertebrate%20genomes&amp;author=M.%20Gardiner-Garden&amp;author=M.%20Frommer&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol.&amp;volume=196&amp;pages=261-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90689-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90689-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90689-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3656447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yavor</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Massa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Trask</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Eichler</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Segmental duplications: organization and impact within the current human genome project assembly</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1005</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20duplications%3A%20organization%20and%20impact%20within%20the%20current%20human%20genome%20project%20assembly&amp;author=J.%20Bailey&amp;author=A.%20Yavor&amp;author=H.%20Massa&amp;author=B.%20Trask&amp;author=E.%20Eichler&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1005-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.GR-1871R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR-1871R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.GR-1871R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujiwara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Komiya</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawabata</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Isolation of a DEAD-family protein gene that encodes a murine homolog of <em>Drosophila vasa</em> and its specific expression in germ cell lineage</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">12258</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20a%20DEAD-family%20protein%20gene%20that%20encodes%20a%20murine%20homolog%20of%20Drosophila%20vasa%20and%20its%20specific%20expression%20in%20germ%20cell%20lineage&amp;author=Y.%20Fujiwara&amp;author=T.%20Komiya&amp;author=H.%20Kawabata&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=12258-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.25.12258" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.25.12258" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.25.12258"> </span></div> </div></div></div></div></div><div content-id="DST030C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pauler</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Sloane</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">H3K27me3 forms BLOCs over silent genes and intergenic regions and specifies a histone banding pattern on a mouse autosomal chromosome</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">221</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H3K27me3%20forms%20BLOCs%20over%20silent%20genes%20and%20intergenic%20regions%20and%20specifies%20a%20histone%20banding%20pattern%20on%20a%20mouse%20autosomal%20chromosome&amp;author=F.M.%20Pauler&amp;author=M.A.%20Sloane&amp;author=R.%20Huang&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=221-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.080861.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.080861.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.080861.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19047520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shinkai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Irizarry</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Feinberg</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Large histone H3 lysine 9 dimethylated chromatin blocks distinguish differentiated from embryonic stem cells</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">246</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20histone%20H3%20lysine%209%20dimethylated%20chromatin%20blocks%20distinguish%20differentiated%20from%20embryonic%20stem%20cells&amp;author=B.%20Wen&amp;author=H.%20Wu&amp;author=Y.%20Shinkai&amp;author=R.A.%20Irizarry&amp;author=A.P.%20Feinberg&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=246-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamagiwa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">DNA methylation regulates long-range gene silencing of an X-linked homeobox gene cluster in a lineage-specific manner</div>, <div class="source">Genes Dev.</div> , <div class="year">2006</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3382</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20regulates%20long-range%20gene%20silencing%20of%20an%20X-linked%20homeobox%20gene%20cluster%20in%20a%20lineage-specific%20manner&amp;author=M.%20Oda&amp;author=A.%20Yamagiwa&amp;author=S.%20Yamamoto&amp;publication_year=2006&amp;journal=Genes%20Dev.&amp;volume=20&amp;pages=3382-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1470906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1470906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1470906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17182866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Link</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Gangisetty</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">James</div> <div class="given-names">S.R.</div></div>, et al. </div><div class="article-title">Distinct roles for histone methyltransferases G9a and GLP in cancer germ-line antigen gene regulation in human cancer cells and murine embryonic stem cells</div>, <div class="source">Mol. Cancer Res.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">851</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20roles%20for%20histone%20methyltransferases%20G9a%20and%20GLP%20in%20cancer%20germ-line%20antigen%20gene%20regulation%20in%20human%20cancer%20cells%20and%20murine%20embryonic%20stem%20cells&amp;author=P.A.%20Link&amp;author=O.%20Gangisetty&amp;author=S.R.%20James&amp;publication_year=2009&amp;journal=Mol.%20Cancer%20Res.&amp;volume=7&amp;pages=851-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1541-7786.MCR-08-0497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1541-7786.MCR-08-0497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1541-7786.MCR-08-0497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19531572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warburton</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Giordano</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1861</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inverted%20repeat%20structure%20of%20the%20human%20genome%3A%20the%20X-chromosome%20contains%20a%20preponderance%20of%20large%2C%20highly%20homologous%20inverted%20repeats%20that%20contain%20testes%20genes&amp;author=P.%20Warburton&amp;author=J.%20Giordano&amp;author=F.%20Cheung&amp;author=Y.%20Gelfand&amp;author=G.%20Benson&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1861-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2542904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2542904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2542904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karimi</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Goyal</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Maksakova</div> <div class="given-names">I.A.</div></div>, et al. </div><div class="article-title">DNA methylation and SETDB1/H3K9me3 regulate predominantly distinct sets of genes, retroelements, and chimeric transcripts in mESCs</div>, <div class="source">Cell Stem Cell</div> , <div class="year">2011</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">676</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20SETDB1%2FH3K9me3%20regulate%20predominantly%20distinct%20sets%20of%20genes%2C%20retroelements%2C%20and%20chimeric%20transcripts%20in%20mESCs&amp;author=M.M.%20Karimi&amp;author=P.%20Goyal&amp;author=I.A.%20Maksakova&amp;publication_year=2011&amp;journal=Cell%20Stem%20Cell&amp;volume=8&amp;pages=676-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2011.04.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2011.04.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2011.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21624812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">A.J.G.</div></div>,  <div class="name"><div class="surname">Caballero</div> <div class="given-names">O.L.</div></div>,  <div class="name"><div class="surname">Jungbluth</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.-T.</div></div>,  <div class="name"><div class="surname">Old</div> <div class="given-names">L.J.</div></div>. </div><div class="article-title">Cancer/testis antigens, gametogenesis and cancer</div>, <div class="source">Nat. Rev. Cancer</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">615</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%2Ftestis%20antigens%2C%20gametogenesis%20and%20cancer&amp;author=A.J.G.%20Simpson&amp;author=O.L.%20Caballero&amp;author=A.%20Jungbluth&amp;author=Y.-T.%20Chen&amp;author=L.J.%20Old&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=5&amp;pages=615-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc1669" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc1669" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc1669"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16034368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pohl</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sugnet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cline</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Affy exon tissues: exon levels in normal tissues in human, mouse and rat</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2442</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Affy%20exon%20tissues%3A%20exon%20levels%20in%20normal%20tissues%20in%20human%2C%20mouse%20and%20rat&amp;author=A.%20Pohl&amp;author=C.%20Sugnet&amp;author=T.%20Clark&amp;author=K.%20Smith&amp;author=P.%20Fujita&amp;author=M.S.%20Cline&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2442-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19570805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST030C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST030C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=419528&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_table4.xls?Expires=2147483647&amp;Signature=g59cqtwdgD1hOQzcga4GxUf6WaeIxW1gV8Yj6GyXP0z9DhZLvxOSSxj2TJ2KE27seTCX5ZIzV6RN6K0J5nX7yWQdYjNiWRP4DgsGTsK0pcMVVWORajiCKfrbA0T5-WdJFL5dWmJVHkms9Saq6-PjHowqgm0L5FWeaiKgdQ-ROUxQ804rCQZ2Q7bKNEhm~o1Bw9V6FYwegIsXN6TG-6B-q7ELlkjMYHLLpsyCSFRACCQRXWeVHk39~freT81~1XCMbUrI3KqN2RxeRt7zsuiuwSvO88cAlgkGYJt-ZhcGKmLI1xFacwuihspxOUaIHoQoU2vVd4cWVARAxLwBEGvz9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_table3.xls?Expires=2147483647&amp;Signature=aXfOevr~4wwbugKZgulN8WMtdiP36SAzcq8x9Tp2EoYGWogSTZqbu4BAG1oDPHDEDTx7OwiU232w67BAvrwGVUNSt5PAvaD4hcdMsb9kTMiLYmYstfqbBV9ivJ0t~-NWyUrSBSVgJS29NnxUUhQN7Lf8XdFUKDP-Xng3WDJNbnx7UVEaR560fx-51bL4MNewTernG5brpEOM9TbnaxRWgKsWUIsNe9CkGJ0-OOZa5OVsup5~haZhTEpTw9MtWiJMoziELXOeZb6RppMB6kI-lyNL51yXPhNIHaEGd9AzWKhrU3F~UpAp2heR-wQ-xq5-Dr2LTBbayfKIoeu2fY03CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_table2.xls?Expires=2147483647&amp;Signature=mhmVniNrqFDCZyAjriq8rD8FOXpzbtjFB~QaSFpw8IlOJ-QpEKYRe56unMctxjDdfAib0GY586NFi4U8lpukgyiKuVQhcp9Bvgacm0e4I~EpLQHG-wNvFLdkEbtEEK9rXwGKCwpPYxDTK5hM8mMhyrGEr7LkJczMZXjEGr9RP~btyw6IeRU6KY0tUSiuSFNB4SIRTyLabSs~sxo0t33GpZv9DjhNxbFe6XkzdhfrTpJbPwYQpppSXqbpPT~BWPlHFyyfHCEFfVQ68qqi4n6eaZprzYqkZCC3L2w0zJ0zXpGoWBtIUj3YBW9m1W~TvEr35nD8C-QPEkou11DsJJ~hXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_table1.xls?Expires=2147483647&amp;Signature=gl2lfT7cYo5nAcBt1f0EkqlC8yvdHYTjTeVCpTrIvDM224HeIo9Is2HaigDAt3K-mmmSpeJb8Z~dtuCgD2GiQkY7dBURN5el3xELEzp29cNrOnsnMZ6b6t14ZVa1o3fdHC~GBdTeicsokFH3lLK1jEF9PAz91XuhlHilSuPgvkeM0GQWFv08M6Oi0bu15bNMGW7i-o0ZPm5SmrULkfc07IXAk2XxsvAbT4vf-8fy7ry0Y2LIGAO4RV~~OXU-6ALFDKicIJjSIIoonRvreOTP1f4dE76Q~NNSs1pLJAoIm5-6MAz-FWc3iodXfLcaCt3neaRubfQqtzrUAbDbeVNcaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_figS3B.png?Expires=2147483647&amp;Signature=f8HHUMDpMonJ-rQA0Q12g4SJxgND1dMZ4B31F7fjA7vJmN0-TWkSoQl~wiTIRkKOCeFtnpvjvSRkxm14knwFYgLxltyRtT3lwxKgk8gJTolt0XZIjuKaPzdfivEhatottSwmf6twGylcK6DIdkAsoLWjzxtO2TkDDRsUTkdRo0ZUUamEmATLZ0-IBuTDNc~GFU-YHKmh3gnDpzDXIn7PGK1spanudL5PxByNwiziBWXkXfUUv~H6Pz8ACZspF1V9iJ8axm81-M9YKd1hMKSex9NsQMdXJIDN~DkS7wHCOeXOVG-GY7U9w9LblOey4arTqNo13LeNZWHlquABLIlvPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig7.png?Expires=2147483647&amp;Signature=rrhxRzCvEQHZdfi8y5LpPbCd1fezEmIhuKxdniwqX-aSGJC0xpvWoY2QXbPxNKr0T8m~sO~5A8rj8oA6vRL9XPYcbi7EKPv7QejactIsIr~qBeXzvdgj~~YRwZAtvXhbMOBvLfWh-QlSApa0FCFdrYDaAzyeCcEj2-hFCiPljQHhZv1Oc8UX~1v1MbDkJbldtVSoDXonok4mwGzpZle7PmSZFEYzPTWoqlzaDSIJ1DwcY1TE7LzwSqaoUJ-paN-h-OeZ~zd3P73W3gCQwqMK70qV6wMgRGYPayP3UHCXsK1jA40jYgE40AlOOEorBN0WAqFjH0xwx0J4M2dLO~-sHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig6.png?Expires=2147483647&amp;Signature=NGsPsJHNqt2ke0MCNKnEtsvXSsgQNuJOWVq0Iz0SA2x-oi-0bK4eoCQ-lmOWJLWCfPl0E1hjOUxwlAlLPR~Ahaq6o9ZzK5UN1Yi5Y5UAGw~3wWLQo7xAIIi4WPpqmd8Yj2fuifvlBSG0L1JGJtZOXvY~f3nGV~gDYR8eR81kc-wEYIP9EHHPv1FGWhMRVqOlLi~HvnTJdzJH0ho3nb53YUhJGRxHEmTSlsmnS3SyA0fZYgP2fmaAyYPlwa5DmV4gc9Pzs0MazYkOQh8DeYYkR3pXuBMGylW-mnF1roMTIFsvnYBQ8ZJ50khudezU1TThzW8IdLyNoU3O75nL~KVsvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig5.png?Expires=2147483647&amp;Signature=zG406Hx-q5xGEnQ6eqAIryJ3l8V~Ao6RbpSWCBUIl8ZID3q~qZh7wKyC7Ml17v6bPdq54tl0HxsPN0NPp33VuF7sRHgb~S33IQmqivKcixH5g1kf1U3hLUxTlLkm3NEI~ZZRM6otZvpiMSwMENYFBSUVUJsY3gBPoJpO2B~11GJpvMRFRYsTnbsWjgZ1UZbvleiSzTwXNClM7UJisKZ~bDWBwW2EFqDV4coA5ujqUlBh15meWHoDSxoqs0IqkrX4XN0qP27MsVkVI24sscoamVEGFlJSf2MVDEUOwypiUuNoTuhbs-1n~buJWcwVjigAjKIKAGFOXUdnFjjew84g8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig4.png?Expires=2147483647&amp;Signature=esAC7CdF-Que59Sk~Kq~Z~IIQB8ua1saA36jHg4mfVlw~9CIBgc6Q5330MroUqOzvhKnvMymmnITrLVj2wOJIC0mEx1xLNC7XnON4iOaMY6V9fwARIrRKTtYEy7cYtdGYeRQcZJfacDwvxsyO0MtQ858usOmIgLG-zmqkgBIpxK4Yil8FoqehrwzyCXqcc3jnWrWbitRFXWfrYvcttJSeE5cpkm-coK~~sEHShp~40UlXRga3HEnqi72dHSmLNmvqY5XiYZJCCScongan2F8XMNERcfmlbUqhOC~INKRFsAohS9jJJbIrSRQjX2fpDYhglW9rP0pSwEfMGfSF5VUEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig3C.png?Expires=2147483647&amp;Signature=iIeuVYyKmz0Tf8calo7TKbUEiLz3BPIdFutV5EQB0Nfbw1yhgAOyuxRk08xa~Owwr51rQCgDlH0e8B6nOVE8N67MNSVoR4TmSTZF-TyAbZ~cZ60axKGnsOJhI0oUFZa3I8NJr4ZwP11CsfdqsYfMbWOt-KMDezYOWrd5M3xNEq-MlaJXempUqZpQvqXLysHbozSjDXizWwE4DFsGjAabeV8d-D2KnKKbqrEiiYWw2JB56OWroQQVbxaHnomu8BjPAgw3z8u1mQaGYHlZbHAjMh6l0~XdCZ3NLcz95FcV2WUl9SzQiP5naJm4eXJeatW573VnBukPsMliPuZm3-YtVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig3A.png?Expires=2147483647&amp;Signature=dZDC~la-xNZzZ7tYUI2ZyYHMH-esTJtUitH9aEWBsDVyyeIfH5tC58my2X5g17a7dJyV9rlm7XzhZJjX-TN1OpeysTcqPUmw5~JW3DRWGqf9NTDGalqREXiIiMc1OhT2O0b4iOpf8BOJBPKQo0OvjiFlOROTVr~12hlCgrKIAoyw2MEVQ~KablN11ZDh2NXV1XVRAVNQqvGIyA4siyx~qbGs0-mGfFZPNeyY6wPqTKJA1uuz4YDVVYx63L5Gm0I5sPoFk2C5ihxbmGBo7oyqGAeFWdltqtr4sVQ8LiBZWlDpO0UzFF9jC0bQjEwRUaR~uiVtYuaT1Uh63IxSnHNAAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig2.png?Expires=2147483647&amp;Signature=QEAopq92eyKJEKHzpR10og6UBxKnlpE4NMi0YVhaX~iiNapGCo9sowOhvOvSy8vIFNy80thRk3U3HQ63tOq4F2ob2iJhYtTC3d0VQC5VYHUHsS6BfCCBlIs0UptnCWp~oWP2QJV7WtIjPUfQZIhN~N0kU5I1pjfEin3218fyQJ94irhBztDgVjMq9X~CdrTFeSUqUa61X0TCXSyESGX3twovN9TmOaRuIGlO7LsaG9ZBET11cfgH9yVu6XfHvEM6tDkDJypXFDSOqA1QHul6mVMmC-5dno~YoEJzDqp8qdxwCsDAVpNsScIbzj0qHJ7jIHN4a89OjE4rC5wraI0mXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp_fig1.png?Expires=2147483647&amp;Signature=ekBvgElcSjSP7Vs41OReNFS77B4IqEuaQiBsOfwWWzMGTSmIqfk0KwrZqVbkwAAKV2UZI6Ea7yEwN~g674x-EcvPAt5Tn7KK-ozF2aUMweyUNw7D-1WBi~08fJrXJfL74KIRppVfKVU771Hg4dLqC62Etwr5yvfCG3e1l3jyKjHEJfqjGuuDfOBPfPkL8JHGojz06PxwvGEMkBaZ09Ee1EDwVTiG3kUp242cB6OblukZbcErhMxgerQzimilEigkswI7GkEOkJh4JaDMsjmWTLHzOjG-Aa8QDaR-02I1wJ97289Ne9b5jKzEN~i7agSNbO~~mtyHgolo-z~U~nnEuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst030/2/dst030-dst030supp.doc?Expires=2147483647&amp;Signature=5KWFhVih-xMEnuSTSbs3EY53kmNGAWhbHJ7S6KSFopKTFRQGJjfuliPQuZKY2i2lfjih9GWMTc-~DcwRb2HDupBU3Q1tKlRKyjBSR2U0fzzlE-1UsF3Yi0JEGXuk~2IxfPB6tBRBCuuR9Q3oqiKA880R2BnozYDbtuTVcdz914dLpcRsMXLPd4zZF1ANDjwimRbj804uQfvxaAypI76wDt77~E-JeSeP~xq5wo6UPL3~ryPR~Xg62GaVHOGy6Oa-r07Xpns9JmrhbAcRJ6GsxXsSTrcACwPtMLJaR7cDaVvOkhInwG5o1nQTBWmXbWvUUpKy6x1PB3jz2OsN9HhISw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">340</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="419528"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f6%2f549%2f419528">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330168100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F6%2F549%2F419528" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/419528'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Large, Male Germ Cell-Specific Hypomethylated DNA Domains With Unique Genomic and Epigenomic Features on the Mouse X Chromosome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f549%2f419528">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

