

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/1/27/347182","@type":"ScholarlyArticle","name":"Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study","datePublished":"2013-10-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2014-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst038","keywords":["alternative splicing","fungal genomes","transcriptome analysis","multi-cellular complexity","retained intron"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/1/27/347182","author":[{"name":"Grützmann, Konrad","affiliation":"1  Department of Bioinformatics ,  Friedrich Schiller University Jena ,  Ernst-Abbe-Platz 2, Jena D-07743 ,  Germany","@type":"Person"},{"name":"Szafranski, Karol","affiliation":"2  Genome Analysis ,  Leibniz Institute for Age Research – Fritz Lipmann Institute ,  Jena ,  Germany","@type":"Person"},{"name":"Pohl, Martin","affiliation":"1  Department of Bioinformatics ,  Friedrich Schiller University Jena ,  Ernst-Abbe-Platz 2, Jena D-07743 ,  Germany","@type":"Person"},{"name":"Voigt, Kerstin","affiliation":"3  Jena Microbial Resource Collection, Leibnitz Institute for Natural Product Research and Infection Biology - Hans-Knöll-Institute, Jena ,  Germany    4  Leibniz Institute for Natural Product Research and Infection Biology – Hans-Knöll-Institute, Jena ,  Germany","@type":"Person"},{"name":"Petzold, Andreas","affiliation":"2  Genome Analysis ,  Leibniz Institute for Age Research – Fritz Lipmann Institute ,  Jena ,  Germany","@type":"Person"},{"name":"Schuster, Stefan","affiliation":"1  Department of Bioinformatics ,  Friedrich Schiller University Jena ,  Ernst-Abbe-Platz 2, Jena D-07743 ,  Germany","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is a cellular process that increases a cell's coding capacity from a limited set of genes. Although AS is common in higher ","pageStart":"27","pageEnd":"39","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is a cellular process that increases a cell&#39;s coding capacity from a limited set of genes. Although AS is common in higher " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/1/27/347182" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-10-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Grützmann, Konrad" /><meta name="citation_author_institution" content="Department of Bioinformatics, Friedrich Schiller University Jena, Ernst-Abbe-Platz 2, Jena D-07743, Germany" /><meta name="citation_author" content="Szafranski, Karol" /><meta name="citation_author_institution" content="Genome Analysis, Leibniz Institute for Age Research – Fritz Lipmann Institute, Jena, Germany" /><meta name="citation_author" content="Pohl, Martin" /><meta name="citation_author_institution" content="Department of Bioinformatics, Friedrich Schiller University Jena, Ernst-Abbe-Platz 2, Jena D-07743, Germany" /><meta name="citation_author" content="Voigt, Kerstin" /><meta name="citation_author_institution" content="Jena Microbial Resource Collection, Leibnitz Institute for Natural Product Research and Infection Biology - Hans-Knöll-Institute, Jena, Germany" /><meta name="citation_author_institution" content="Leibniz Institute for Natural Product Research and Infection Biology – Hans-Knöll-Institute, Jena, Germany" /><meta name="citation_author" content="Petzold, Andreas" /><meta name="citation_author_institution" content="Genome Analysis, Leibniz Institute for Age Research – Fritz Lipmann Institute, Jena, Germany" /><meta name="citation_author" content="Schuster, Stefan" /><meta name="citation_author_institution" content="Department of Bioinformatics, Friedrich Schiller University Jena, Ernst-Abbe-Platz 2, Jena D-07743, Germany" /><meta name="citation_title" content="Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study" /><meta name="citation_firstpage" content="27" /><meta name="citation_lastpage" content="39" /><meta name="citation_doi" content="10.1093/dnares/dst038" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2014/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Function of alternative splicing; Gene;  citation_year=2005;  citation_volume=344;  citation_pages=1-20; " /><meta name="citation_reference" content="citation_title=Mechanisms of alternative pre-messenger RNA splicing; Annu. Rev. Biochem.;  citation_year=2003;  citation_volume=72;  citation_pages=291-336; " /><meta name="citation_reference" content="citation_title=Alternative splicing of neuroligin regulates the rate of presynaptic differentiation; J. Neurosci.;  citation_year=2010;  citation_volume=30;  citation_pages=11435-46; " /><meta name="citation_reference" content="citation_title=Regulation of alternative splicing by reversible protein phosphorylation; J. Biol. Chem.;  citation_year=2008;  citation_volume=283;  citation_pages=1223-7; " /><meta name="citation_reference" content="citation_title=Mouse tissues express multiple splice variants of prominin-1; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e12325" /><meta name="citation_reference" content="citation_title=ASAP II database: analysis and comparative genomics of alternative splicing in 15 animal species; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D93-8; " /><meta name="citation_reference" content="citation_title=Different levels of alternative splicing among eukaryotes; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=125-31; " /><meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=1413-5; " /><meta name="citation_reference" content="citation_title=Splice-site pairing is an intrinsically high fidelity process; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=1766-71; " /><meta name="citation_reference" content="citation_title=Alternative splicing: new insights from global analyses; Cell;  citation_year=2006;  citation_volume=126;  citation_pages=37-47; " /><meta name="citation_reference" content="citation_title=Phylogenetically widespread alternative splicing at unusual GYNGYN donors; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R65" /><meta name="citation_reference" content="citation_title=Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=1255-7; " /><meta name="citation_reference" content="citation_title=Tassdb2—a comprehensive database of subtle alternative splicing events; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=216" /><meta name="citation_reference" content="citation_title=Sequence features responsible for intron retention in human; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=59" /><meta name="citation_reference" content="citation_title=Cross-kingdom patterns of alternative splicing and splice recognition; Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R50" /><meta name="citation_reference" content="citation_title=Introns and splicing elements of five diverse fungi; Eukaryot. Cell;  citation_year=2004;  citation_volume=3;  citation_pages=1088-100; " /><meta name="citation_reference" content="citation_title=Functional and evolutionary analysis of alternatively spliced genes is consistent with an early eukaryotic origin of alternative splicing; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=188" /><meta name="citation_reference" content="citation_title=Gene discovery and gene expression in the rice blast fungus, Magnaporthe grisea: analysis of expressed sequence tags; Mol. Plant Microbe. Interact.;  citation_year=2004;  citation_volume=17;  citation_pages=1337-47; " /><meta name="citation_reference" content="citation_title=Survey of the transcriptome of Aspergillus oryzae via massively parallel mRNA sequencing; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=5075-87; " /><meta name="citation_reference" content="citation_title=The genome of the basidiomycetous yeast and human pathogen Cryptococcus neoformans; Science;  citation_year=2005;  citation_volume=307;  citation_pages=1321-4; " /><meta name="citation_reference" content="citation_title=Gene discovery and transcript analyses in the corn smut pathogen Ustilago maydis: expressed sequence tag and genome sequence comparison; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=334" /><meta name="citation_reference" content="NCBI (National Center of Biotechnology Information) 2011 http://www.ncbi.nlm.nih.gov (18 January 2012, date last accessed) " /><meta name="citation_reference" content="citation_author=DOE Joint Genome Institute;  citation_url=http://www.jgi.doe.gov/;  citation_URL_Access_Date=(1 February 2012, date last accessed); A DOE Office of Science User Facility of Lawrence Berkeley National Laboratory;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Saccharomyces genome database: underlying principles and organisation; Brief Bioinform.;  citation_year=2004;  citation_volume=5;  citation_pages=9-22; " /><meta name="citation_reference" content="2011 Broad Institute of MIT and Harvard Fusarium oxysporum , Paracoccidioides brasiliensis , and Rhizopus oryzae Sequencing Projects http://www.broadinstitute.org (2 February 2012, date last accessed) " /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Improved spliced alignment from an information theoretic approach; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=13-20; " /><meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6097-100; " /><meta name="citation_reference" content="citation_author=R Development Core Team; citation_text=ISBN 3-900051-07-0; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria; R: A Language and Environment for Statistical Computing;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=A probabilistic model of local sequence alignment that simplifies statistical significance estimation; PLoS Comput. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000069" /><meta name="citation_reference" content="citation_title=Reconstructing the early evolution of fungi using a six-gene phylogeny; Nature;  citation_year=2006;  citation_volume=443;  citation_pages=818-22; " /><meta name="citation_reference" content="citation_title=What makes Aspergillus fumigatus a successful pathogen? Genes and molecules involved in invasive aspergillosis; Rev. Iberoam Micol.;  citation_year=2010;  citation_volume=27;  citation_pages=155-82; " /><meta name="citation_reference" content="citation_title=Phenotypic switching of Cryptococcus neoformans and Cryptococcus gattii; Mycopathologia;  citation_year=2008;  citation_volume=166;  citation_pages=181-8; " /><meta name="citation_reference" content="citation_title=Genes potentially relevant in the parasitic phase of the fungal pathogen Paracoccidioides brasiliensis; Mycopathologia;  citation_year=2011;  citation_volume=171;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Comparative functional genomics of the fission yeasts; Science;  citation_year=2011;  citation_volume=332;  citation_pages=930-6; " /><meta name="citation_reference" content="citation_title=Intron–exon structures of eukaryotic model organisms; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=3219-28; " /><meta name="citation_reference" content="citation_title=Alternative splicing: current perspectives; Bioessays;  citation_year=2008;  citation_volume=30;  citation_pages=38-47; " /><meta name="citation_reference" content="citation_title=Transcriptional elongation and alternative splicing; Biochim Biophys Acta;  citation_year=2013;  citation_volume=1829;  citation_pages=134-40; " /><meta name="citation_reference" content="citation_title=The in vivo kinetics of RNA polymerase II elongation during co-transcriptional splicing; PLoS Biol.;  citation_year=2011;  citation_volume=9;  citation_pages=e1000573" /><meta name="citation_reference" content="citation_title=Prevalence of alternative splicing choices in Arabidopsis thaliana; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=102" /><meta name="citation_reference" content="citation_title=NMD: RNA biology meets human genetic medicine; Biochem. J.;  citation_year=2010;  citation_volume=430;  citation_pages=365-77; " /><meta name="citation_reference" content="citation_title=Detection and analysis of alternative splicing in Yarrowia lipolytica reveal structural constraints facilitating nonsense-mediated decay of intron-retaining transcripts; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R65" /><meta name="citation_reference" content="citation_title=Evolutionary roles of upstream open reading frames in mediating gene regulation in fungi; Annu. Rev. Microbiol.;  citation_year=2009;  citation_volume=63;  citation_pages=385-409; " /><meta name="citation_reference" content=" citation_edition=2nd edition; citation_publisher=Academic Press, London, UK; The Fungi;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Comparative genomic analysis of fungal genomes reveals intron-rich ancestors; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R223" /><meta name="citation_reference" content="citation_title=Evolution of SR protein and hnRNP splicing regulatory factors; Wiley Interdiscip. Rev. RNA;  citation_year=2012;  citation_volume=3;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=The evolutionarily conserved core design of the catalytic activation step of the yeast spliceosome; Mol. Cell;  citation_year=2009;  citation_volume=36;  citation_pages=593-608; " /><meta name="citation_reference" content="citation_title=Interactions between two fission yeast serine/arginine-rich proteins and their modulation by phosphorylation; Biochem. J.;  citation_year=2002;  citation_volume=368;  citation_pages=527-34; " /><meta name="citation_reference" content="citation_title=A single ancient origin for prototypical serine/arginine-rich splicing factors; Plant Physiol.;  citation_year=2012;  citation_volume=158;  citation_pages=546-60; " /><meta name="citation_reference" content="citation_title=Alternative splicing and RNA selection pressure—evolutionary consequences for eukaryotic genomes; Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=499-509; " /><meta name="citation_reference" content="citation_title=Isolation of UmRrm75, a gene involved in dimorphism and virulence of Ustilago maydis; Microbiol. Res.;  citation_year=2012;  citation_volume=167;  citation_pages=270-82; " /><meta name="citation_reference" content="citation_title=Pleiotropic function of intersectin homologue Cin1 in Cryptococcus neoformans; Mol. Microbiol.;  citation_year=2010;  citation_volume=76;  citation_pages=662-76; " /><meta name="citation_reference" content="citation_title=Ste12 and ste12-like proteins, fungal transcription factors regulating development and pathogenicity; Eukaryot. Cell;  citation_year=2010;  citation_volume=9;  citation_pages=480-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/1/27/993873/dst038.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is a cellular process that increases a cell's coding capacity from a limited set of genes. Although AS is common in higher " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/1/27/347182" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/1/27/347182" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/1/27/347182");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst038';
    </script>
<title>
	Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347182" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf9vPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHcsO8dHptYW5uLCBLb25yYWQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBGcmllZHJpY2ggU2NoaWxsZXIgVW5pdmVyc2l0eSBKZW5hLCBFcm5zdC1BYmJlLVBsYXR6IDIsIEplbmEgRC0wNzc0MywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN6YWZyYW5za2ksIEthcm9sIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2Vub21lIEFuYWx5c2lzLCBMZWlibml6IEluc3RpdHV0ZSBmb3IgQWdlIFJlc2VhcmNoIOKAkyBGcml0eiBMaXBtYW5uIEluc3RpdHV0ZSwgSmVuYSwgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBvaGwsIE1hcnRpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIEZyaWVkcmljaCBTY2hpbGxlciBVbml2ZXJzaXR5IEplbmEsIEVybnN0LUFiYmUtUGxhdHogMiwgSmVuYSBELTA3NzQzLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVm9pZ3QsIEtlcnN0aW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKZW5hIE1pY3JvYmlhbCBSZXNvdXJjZSBDb2xsZWN0aW9uLCBMZWlibml0eiBJbnN0aXR1dGUgZm9yIE5hdHVyYWwgUHJvZHVjdCBSZXNlYXJjaCBhbmQgSW5mZWN0aW9uIEJpb2xvZ3kgLSBIYW5zLUtuw7ZsbC1JbnN0aXR1dGUsIEplbmEsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMZWlibml6IEluc3RpdHV0ZSBmb3IgTmF0dXJhbCBQcm9kdWN0IFJlc2VhcmNoIGFuZCBJbmZlY3Rpb24gQmlvbG9neSDigJMgSGFucy1LbsO2bGwtSW5zdGl0dXRlLCBKZW5hLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGV0em9sZCwgQW5kcmVhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlbm9tZSBBbmFseXNpcywgTGVpYm5peiBJbnN0aXR1dGUgZm9yIEFnZSBSZXNlYXJjaCDigJMgRnJpdHogTGlwbWFubiBJbnN0aXR1dGUsIEplbmEsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2h1c3RlciwgU3RlZmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcywgRnJpZWRyaWNoIFNjaGlsbGVyIFVuaXZlcnNpdHkgSmVuYSwgRXJuc3QtQWJiZS1QbGF0eiAyLCBKZW5hIEQtMDc3NDMsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkZ1bmdhbCBBbHRlcm5hdGl2ZSBTcGxpY2luZyBpcyBBc3NvY2lhdGVkIHdpdGggTXVsdGljZWxsdWxhciBDb21wbGV4aXR5IGFuZCBWaXJ1bGVuY2U6IEEgR2Vub21lLVdpZGUgTXVsdGktU3BlY2llcyBTdHVkeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb24gb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzQ0OyAgY2l0YXRpb25fcGFnZXM9MS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBhbHRlcm5hdGl2ZSBwcmUtbWVzc2VuZ2VyIFJOQSBzcGxpY2luZzsgQW5udS4gUmV2LiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIG9mIG5ldXJvbGlnaW4gcmVndWxhdGVzIHRoZSByYXRlIG9mIHByZXN5bmFwdGljIGRpZmZlcmVudGlhdGlvbjsgSi4gTmV1cm9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTE0MzUtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYnkgcmV2ZXJzaWJsZSBwcm90ZWluIHBob3NwaG9yeWxhdGlvbjsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjgzOyAgY2l0YXRpb25fcGFnZXM9MTIyMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3VzZSB0aXNzdWVzIGV4cHJlc3MgbXVsdGlwbGUgc3BsaWNlIHZhcmlhbnRzIG9mIHByb21pbmluLTE7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTIzMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BU0FQIElJIGRhdGFiYXNlOiBhbmFseXNpcyBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3Mgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gMTUgYW5pbWFsIHNwZWNpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQ5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnQgbGV2ZWxzIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGFtb25nIGV1a2FyeW90ZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBzdXJ2ZXlpbmcgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgY29tcGxleGl0eSBpbiB0aGUgaHVtYW4gdHJhbnNjcmlwdG9tZSBieSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0xNDEzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljZS1zaXRlIHBhaXJpbmcgaXMgYW4gaW50cmluc2ljYWxseSBoaWdoIGZpZGVsaXR5IHByb2Nlc3M7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTc2Ni03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmc6IG5ldyBpbnNpZ2h0cyBmcm9tIGdsb2JhbCBhbmFseXNlczsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz0zNy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljYWxseSB3aWRlc3ByZWFkIGFsdGVybmF0aXZlIHNwbGljaW5nIGF0IHVudXN1YWwgR1lOR1lOIGRvbm9yczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1SNjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIG9jY3VycmVuY2Ugb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYXQgTkFHTkFHIGFjY2VwdG9ycyBjb250cmlidXRlcyB0byBwcm90ZW9tZSBwbGFzdGljaXR5OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTEyNTUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFzc2RiMuKAlGEgY29tcHJlaGVuc2l2ZSBkYXRhYmFzZSBvZiBzdWJ0bGUgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBmZWF0dXJlcyByZXNwb25zaWJsZSBmb3IgaW50cm9uIHJldGVudGlvbiBpbiBodW1hbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz01OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3NzLWtpbmdkb20gcGF0dGVybnMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIHNwbGljZSByZWNvZ25pdGlvbjsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb25zIGFuZCBzcGxpY2luZyBlbGVtZW50cyBvZiBmaXZlIGRpdmVyc2UgZnVuZ2k7IEV1a2FyeW90LiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMDg4LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbmQgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGFsdGVybmF0aXZlbHkgc3BsaWNlZCBnZW5lcyBpcyBjb25zaXN0ZW50IHdpdGggYW4gZWFybHkgZXVrYXJ5b3RpYyBvcmlnaW4gb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7IEJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkaXNjb3ZlcnkgYW5kIGdlbmUgZXhwcmVzc2lvbiBpbiB0aGUgcmljZSBibGFzdCBmdW5ndXMsIE1hZ25hcG9ydGhlIGdyaXNlYTogYW5hbHlzaXMgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IE1vbC4gUGxhbnQgTWljcm9iZS4gSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTMzNy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VydmV5IG9mIHRoZSB0cmFuc2NyaXB0b21lIG9mIEFzcGVyZ2lsbHVzIG9yeXphZSB2aWEgbWFzc2l2ZWx5IHBhcmFsbGVsIG1STkEgc2VxdWVuY2luZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NTA3NS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgYmFzaWRpb215Y2V0b3VzIHllYXN0IGFuZCBodW1hbiBwYXRob2dlbiBDcnlwdG9jb2NjdXMgbmVvZm9ybWFuczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDc7ICBjaXRhdGlvbl9wYWdlcz0xMzIxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZGlzY292ZXJ5IGFuZCB0cmFuc2NyaXB0IGFuYWx5c2VzIGluIHRoZSBjb3JuIHNtdXQgcGF0aG9nZW4gVXN0aWxhZ28gbWF5ZGlzOiBleHByZXNzZWQgc2VxdWVuY2UgdGFnIGFuZCBnZW5vbWUgc2VxdWVuY2UgY29tcGFyaXNvbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOQ0JJIChOYXRpb25hbCBDZW50ZXIgb2YgQmlvdGVjaG5vbG9neSBJbmZvcm1hdGlvbikgMjAxMSBodHRwOi8vd3d3Lm5jYmkubmxtLm5paC5nb3YgKDE4IEphbnVhcnkgMjAxMiwgZGF0ZSBsYXN0IGFjY2Vzc2VkKSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9RE9FIEpvaW50IEdlbm9tZSBJbnN0aXR1dGU7ICBjaXRhdGlvbl91cmw9aHR0cDovL3d3dy5qZ2kuZG9lLmdvdi87ICBjaXRhdGlvbl9VUkxfQWNjZXNzX0RhdGU9KDEgRmVicnVhcnkgMjAxMiwgZGF0ZSBsYXN0IGFjY2Vzc2VkKTsgQSBET0UgT2ZmaWNlIG9mIFNjaWVuY2UgVXNlciBGYWNpbGl0eSBvZiBMYXdyZW5jZSBCZXJrZWxleSBOYXRpb25hbCBMYWJvcmF0b3J5OyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWNjaGFyb215Y2VzIGdlbm9tZSBkYXRhYmFzZTogdW5kZXJseWluZyBwcmluY2lwbGVzIGFuZCBvcmdhbmlzYXRpb247IEJyaWVmIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IjIwMTEgQnJvYWQgSW5zdGl0dXRlIG9mIE1JVCBhbmQgSGFydmFyZCBGdXNhcml1bSBveHlzcG9ydW0gLCBQYXJhY29jY2lkaW9pZGVzIGJyYXNpbGllbnNpcyAsIGFuZCBSaGl6b3B1cyBvcnl6YWUgU2VxdWVuY2luZyBQcm9qZWN0cyBodHRwOi8vd3d3LmJyb2FkaW5zdGl0dXRlLm9yZyAoMiBGZWJydWFyeSAyMDEyLCBkYXRlIGxhc3QgYWNjZXNzZWQpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIHNwbGljZWQgYWxpZ25tZW50IGZyb20gYW4gaW5mb3JtYXRpb24gdGhlb3JldGljIGFwcHJvYWNoOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBsb2dvczogYSBuZXcgd2F5IHRvIGRpc3BsYXkgY29uc2Vuc3VzIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjA5Ny0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1SIERldmVsb3BtZW50IENvcmUgVGVhbTsgY2l0YXRpb25fdGV4dD1JU0JOIDMtOTAwMDUxLTA3LTA7IGNpdGF0aW9uX3B1Ymxpc2hlcj1SIEZvdW5kYXRpb24gZm9yIFN0YXRpc3RpY2FsIENvbXB1dGluZywgVmllbm5hLCBBdXN0cmlhOyBSOiBBIExhbmd1YWdlIGFuZCBFbnZpcm9ubWVudCBmb3IgU3RhdGlzdGljYWwgQ29tcHV0aW5nOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb2JhYmlsaXN0aWMgbW9kZWwgb2YgbG9jYWwgc2VxdWVuY2UgYWxpZ25tZW50IHRoYXQgc2ltcGxpZmllcyBzdGF0aXN0aWNhbCBzaWduaWZpY2FuY2UgZXN0aW1hdGlvbjsgUExvUyBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDA2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29uc3RydWN0aW5nIHRoZSBlYXJseSBldm9sdXRpb24gb2YgZnVuZ2kgdXNpbmcgYSBzaXgtZ2VuZSBwaHlsb2dlbnk7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDM7ICBjaXRhdGlvbl9wYWdlcz04MTgtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdoYXQgbWFrZXMgQXNwZXJnaWxsdXMgZnVtaWdhdHVzIGEgc3VjY2Vzc2Z1bCBwYXRob2dlbj8gR2VuZXMgYW5kIG1vbGVjdWxlcyBpbnZvbHZlZCBpbiBpbnZhc2l2ZSBhc3BlcmdpbGxvc2lzOyBSZXYuIEliZXJvYW0gTWljb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTU1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGVub3R5cGljIHN3aXRjaGluZyBvZiBDcnlwdG9jb2NjdXMgbmVvZm9ybWFucyBhbmQgQ3J5cHRvY29jY3VzIGdhdHRpaTsgTXljb3BhdGhvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVzIHBvdGVudGlhbGx5IHJlbGV2YW50IGluIHRoZSBwYXJhc2l0aWMgcGhhc2Ugb2YgdGhlIGZ1bmdhbCBwYXRob2dlbiBQYXJhY29jY2lkaW9pZGVzIGJyYXNpbGllbnNpczsgTXljb3BhdGhvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTcxOyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBmdW5jdGlvbmFsIGdlbm9taWNzIG9mIHRoZSBmaXNzaW9uIHllYXN0czsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzI7ICBjaXRhdGlvbl9wYWdlcz05MzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9u4oCTZXhvbiBzdHJ1Y3R1cmVzIG9mIGV1a2FyeW90aWMgbW9kZWwgb3JnYW5pc21zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMjE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZzogY3VycmVudCBwZXJzcGVjdGl2ZXM7IEJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTM4LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgZWxvbmdhdGlvbiBhbmQgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7IEJpb2NoaW0gQmlvcGh5cyBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE4Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMzQtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbiB2aXZvIGtpbmV0aWNzIG9mIFJOQSBwb2x5bWVyYXNlIElJIGVsb25nYXRpb24gZHVyaW5nIGNvLXRyYW5zY3JpcHRpb25hbCBzcGxpY2luZzsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA1NzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmV2YWxlbmNlIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGNob2ljZXMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5NRDogUk5BIGJpb2xvZ3kgbWVldHMgaHVtYW4gZ2VuZXRpYyBtZWRpY2luZTsgQmlvY2hlbS4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDMwOyAgY2l0YXRpb25fcGFnZXM9MzY1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIGFuYWx5c2lzIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIFlhcnJvd2lhIGxpcG9seXRpY2EgcmV2ZWFsIHN0cnVjdHVyYWwgY29uc3RyYWludHMgZmFjaWxpdGF0aW5nIG5vbnNlbnNlLW1lZGlhdGVkIGRlY2F5IG9mIGludHJvbi1yZXRhaW5pbmcgdHJhbnNjcmlwdHM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPVI2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSByb2xlcyBvZiB1cHN0cmVhbSBvcGVuIHJlYWRpbmcgZnJhbWVzIGluIG1lZGlhdGluZyBnZW5lIHJlZ3VsYXRpb24gaW4gZnVuZ2k7IEFubnUuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTM4NS00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl9lZGl0aW9uPTJuZCBlZGl0aW9uOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIExvbmRvbiwgVUs7IFRoZSBGdW5naTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyBvZiBmdW5nYWwgZ2Vub21lcyByZXZlYWxzIGludHJvbi1yaWNoIGFuY2VzdG9yczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1SMjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIFNSIHByb3RlaW4gYW5kIGhuUk5QIHNwbGljaW5nIHJlZ3VsYXRvcnkgZmFjdG9yczsgV2lsZXkgSW50ZXJkaXNjaXAuIFJldi4gUk5BOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJpbHkgY29uc2VydmVkIGNvcmUgZGVzaWduIG9mIHRoZSBjYXRhbHl0aWMgYWN0aXZhdGlvbiBzdGVwIG9mIHRoZSB5ZWFzdCBzcGxpY2Vvc29tZTsgTW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NTkzLTYwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJhY3Rpb25zIGJldHdlZW4gdHdvIGZpc3Npb24geWVhc3Qgc2VyaW5lL2FyZ2luaW5lLXJpY2ggcHJvdGVpbnMgYW5kIHRoZWlyIG1vZHVsYXRpb24gYnkgcGhvc3Bob3J5bGF0aW9uOyBCaW9jaGVtLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zNjg7ICBjaXRhdGlvbl9wYWdlcz01MjctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2luZ2xlIGFuY2llbnQgb3JpZ2luIGZvciBwcm90b3R5cGljYWwgc2VyaW5lL2FyZ2luaW5lLXJpY2ggc3BsaWNpbmcgZmFjdG9yczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTU4OyAgY2l0YXRpb25fcGFnZXM9NTQ2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBhbmQgUk5BIHNlbGVjdGlvbiBwcmVzc3VyZeKAlGV2b2x1dGlvbmFyeSBjb25zZXF1ZW5jZXMgZm9yIGV1a2FyeW90aWMgZ2Vub21lczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDk5LTUwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIFVtUnJtNzUsIGEgZ2VuZSBpbnZvbHZlZCBpbiBkaW1vcnBoaXNtIGFuZCB2aXJ1bGVuY2Ugb2YgVXN0aWxhZ28gbWF5ZGlzOyBNaWNyb2Jpb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9MjcwLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGVpb3Ryb3BpYyBmdW5jdGlvbiBvZiBpbnRlcnNlY3RpbiBob21vbG9ndWUgQ2luMSBpbiBDcnlwdG9jb2NjdXMgbmVvZm9ybWFuczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc2OyAgY2l0YXRpb25fcGFnZXM9NjYyLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGUxMiBhbmQgc3RlMTItbGlrZSBwcm90ZWlucywgZnVuZ2FsIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZWd1bGF0aW5nIGRldmVsb3BtZW50IGFuZCBwYXRob2dlbmljaXR5OyBFdWthcnlvdC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDgwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8xLzI3Lzk5Mzg3My9kc3QwMzgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGVybmF0aXZlIHNwbGljaW5nIChBUykgaXMgYSBjZWxsdWxhciBwcm9jZXNzIHRoYXQgaW5jcmVhc2VzIGEgY2VsbCdzIGNvZGluZyBjYXBhY2l0eSBmcm9tIGEgbGltaXRlZCBzZXQgb2YgZ2VuZXMuIEFsdGhvdWdoIEFTIGlzIGNvbW1vbiBpbiBoaWdoZXIgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDcxODImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkKYaCw8vN/JPlxw9K2TQlHykrfLU=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f27%2f347182" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f27%2f347182" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13381" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/1/1/m_cover.gif?Expires=2147483647&amp;Signature=O6KdUwcvHbd2-JMT7ataLxhbwOeCBuxRp98Pq5MMbgeFNh0CYTmpXkkm0SI3qUm3MJ0vHXWxOm2kPBu5Rgo0fKHtDJDfZESs9V~0tNUUm6kyGHNICw1kvqdJFcMATuPRllJYdqzXipnnf-Skkbo14bB23qukbSJgkTF8pGT99rZf8sbETeNndWrSEgQxZdETKMev~I0NCAvT0v~xsxmWuK-UxMPFC5n4zZ5VTdx1iGP0wcMBg32Rgk2MsAMBA~xEVLTIrTKFiOSHU6O-J6MxiLzFiPOJsgoGHQ67KOEexoif2dgVWS4VER03BHFw9~dH0~yNMz5hZOC1eS-fW6NcsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/1">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4909483">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909483">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909487">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909487">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909501">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909501">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909533">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909533">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909590">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909590">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909637">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909637">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909640">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909640">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4909645">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4909645">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/1/15/347005">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/1/41/347328">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Konrad Gr&#252;tzmann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Friedrich Schiller University Jena</div>, <div class="addr-line">Ernst-Abbe-Platz 2, Jena D-07743</div>, <div class="country">Germany</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +49-3641-949581. Fax. +49-3641-946452. Email: <a href="mailto:konrad.g@uni-jena.de" target="_blank">konrad.g@uni-jena.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Konrad+Gr%c3%bctzmann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gr&#252;tzmann K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gr&#252;tzmann K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Konrad Grützmann</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karol Szafranski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Genome Analysis</div>, <div class="insititution">Leibniz Institute for Age Research – Fritz Lipmann Institute</div>, <div class="addr-line">Jena</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karol+Szafranski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Szafranski K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Szafranski K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Karol Szafranski</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Pohl</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Friedrich Schiller University Jena</div>, <div class="addr-line">Ernst-Abbe-Platz 2, Jena D-07743</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Pohl">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pohl M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pohl M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin Pohl</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kerstin Voigt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Jena Microbial Resource Collection, Leibnitz Institute for Natural Product Research and Infection Biology - Hans-Knöll-Institute, Jena</div>, <div class="country">Germany</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Leibniz Institute for Natural Product Research and Infection Biology – Hans-Knöll-Institute, Jena</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kerstin+Voigt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Voigt K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Voigt K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kerstin Voigt</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andreas Petzold</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Genome Analysis</div>, <div class="insititution">Leibniz Institute for Age Research – Fritz Lipmann Institute</div>, <div class="addr-line">Jena</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andreas+Petzold">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Petzold A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Petzold A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andreas Petzold</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stefan Schuster</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Friedrich Schiller University Jena</div>, <div class="addr-line">Ernst-Abbe-Platz 2, Jena D-07743</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stefan+Schuster">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schuster S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schuster S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stefan Schuster</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 27–39, <a href='https://doi.org/10.1093/dnares/dst038'>https://doi.org/10.1093/dnares/dst038</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 October 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 November 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 September 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347182" href="/dnaresearch/article-pdf/21/1/27/993873/dst038.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Konrad Grützmann, Karol Szafranski, Martin Pohl, Kerstin Voigt, Andreas Petzold, Stefan Schuster;  Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study, <em>DNA Research</em>, Volume 21, Issue 1, 1 February 2014, Pages 27–39, <a href='https://doi.org/10.1093/dnares/dst038'>https://doi.org/10.1093/dnares/dst038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347182?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347182?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Fungal%20Alternative%20Splicing%20is%20Associated%20with%20Multicellular%20Complexity%20and%20Virulence%3A%20A%20Genome-Wide%20Multi-Species%20Study&amp;publicationDate=2013-10-11&amp;volumeNum=21&amp;issueNum=1&amp;author=Gr%C3%BCtzmann%2C%20Konrad%3B%20Szafranski%2C%20Karol&amp;startPage=27&amp;endPage=39&amp;contentId=10.1093%2Fdnares%2Fdst038&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f27%2f347182" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4909483 id="4909483" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is a cellular process that increases a cell's coding capacity from a limited set of genes. Although AS is common in higher plants and animals, its prevalence in other eukaryotes is mostly unknown. In fungi the involvement of AS in gene expression and its effect on multi-cellularity and virulence is of great medical and economic interest. We present a genome-wide comparative study of AS in 23 informative fungi of different taxa, based on alignments of public transcript sequences. Random sampling of expressed sequence tags allows for robust and comparable estimations of AS rates. We find that a greater fraction of fungal genes than previously expected is associated with AS. We estimate that on average, 6.4% of the annotated genes are affected by AS, with <em>Cryptococcus neoformans</em> showing an extraordinary rate of 18%. The investigated Basidiomycota show higher average AS rates (8.6%) than the Ascomycota (6.0%), although not significant. We find that multi-cellular complexity and younger evolutionary age associate with higher AS rates. Furthermore, AS affects genes involved in pathogenic lifestyle, particularly in functions of stress response and dimorphic switching. Together, our analysis strongly supports the view that AS is a rather common phenomenon in fungi and associates with higher multi-cellular complexity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;fungal genomes&quot;">fungal genomes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome analysis&quot;">transcriptome analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multi-cellular complexity&quot;">multi-cellular complexity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;retained intron&quot;">retained intron</a></div>                            <h2 scrollto-destination=4909487 id="4909487" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Via alternative splicing (AS) different mRNA isoforms are produced from one single gene. This diversification is one explanation for the discrepancy between the relatively low gene numbers of higher eukaryotes on the one hand and their cellular complexity on the other hand. AS affects binding properties, intracellular localization, enzymatic activity and many more properties of proteins.<sup><a reveal-id="DST038C1" data-open="DST038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Examples of regulated pathways are sex determination in <em>Drosophila melanogaster</em>,<sup><a reveal-id="DST038C2" data-open="DST038C2" class="link link-ref link-reveal xref-bibr">2</a></sup> neuronal differentiation in rat<sup><a reveal-id="DST038C3" data-open="DST038C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and auto-regulation of LAMMER kinases, which take part in splicing factor activation.<sup><a reveal-id="DST038C4" data-open="DST038C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Not only the mere presence of an isoform, but also the exact splice isoform ratio can influence the phenotype of cells and can be regulated in a tissue-dependent manner.<sup><a reveal-id="DST038C5" data-open="DST038C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>Expressed sequence tags (ESTs) have widely been used to detect AS and quantify the transcript diversity arising from AS. For example, AS estimates for animals range from 53% of the multi-exon genes in human, 53% in mouse, 24% in rat, 22% in chicken, 19% in fruit fly to 6% in roundworm.<sup><a reveal-id="DST038C6" data-open="DST038C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Interestingly, despite their relatedness, the estimates for mouse and rat differ remarkably. A possible reason for this are too few transcript data that limit the detection of AS events. Therefore, an approach was suggested that corrects for the amounts of transcripts and yields similar AS rates of ∼31% for mouse and rat.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Finally, from deep transcriptome sequencing an AS rate of &gt;90% was estimated for humans.<sup><a reveal-id="DST038C8" data-open="DST038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> These findings support the view that sensitive methods will ultimately detect splicing variants for every multi-exon gene.<sup><a reveal-id="DST038C9" data-open="DST038C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>The basic AS types are the following: in exon skipping (SE, cassette exon), the exon can be spliced out of the transcript together with its flanking introns.<sup><a reveal-id="DST038C10" data-open="DST038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Alternative 5′ splice site (A5′SS, alternative donor) selection<sup><a reveal-id="DST038C11" data-open="DST038C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and alternative 3′ splice site (A3′SS, alternative acceptor) selection<sup><a reveal-id="DST038C12" data-open="DST038C12" class="link link-ref link-reveal xref-bibr">12</a></sup> result in longer exons and corresponding isoforms.<sup><a reveal-id="DST038C13" data-open="DST038C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Intron retention describes a mechanism where an intron can remain in the mature mRNA.<sup><a reveal-id="DST038C14" data-open="DST038C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Previous studies showed that eukaryote species do not have equal distributions of these AS types. Cassette exons predominantly occur in animals, whereas intron retention is more frequent in other taxa.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Fungi, especially <em>Saccharomyces cerevisiae</em>, have been used extensively as a reduced and easily manageable model system in biological research. There are many fungi that cause human and plant diseases (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), which provoke worldwide costs of several billion dollars a year. Other fungi are used for industrial fermentation and production of food and feed additives or are crucial in the degradation of xenobiotics and in the conversion of cellulose into biofuels (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Fungi have compact genomes (the majority 10–90 Mbp) and genes with small introns. They also show extended consensus sequences for the 5′SS and the branchpoint region.<sup><a reveal-id="DST038C16" data-open="DST038C16" class="link link-ref link-reveal xref-bibr">16</a></sup> These features facilitate a structural interpretation of intron sequences, and they suggest low-complex AS patterns, both of which make fungi attractive models for mechanistic studies of (alternative) splicing.</p><p>A few studies estimated fungal AS rates on a genome-wide scale in comparative manner. Varying but relatively low AS frequencies were discovered in fungi and microsporidia (0–5% of genes in <em>S. cerevisiae</em>, <em>Schizosaccharo-myces pombe</em>, <em>Encephalitozoon cuniculi</em> and <em>Cryptococcus neoformans</em>).<sup><a reveal-id="DST038C17" data-open="DST038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> A correspondence between intron numbers per gene and AS numbers was found for the studied species. Also, AS seems to affect genes of different functions. Genes associated with regulation have higher AS levels. Evolutionarily old genes were found to be affected more often.<sup><a reveal-id="DST038C17" data-open="DST038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In another comparative study of 14 fungi among other eukaryotes, also varying amounts of AS were observed. Yeasts showed nearly no events, and around 1000 AS instances were found for <em>C. neoformans</em> and <em>Coccidioides posadasii</em>, each.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Studies of single fungal species show results from only a few AS events in <em>Magnaporthe grisea</em><sup><a reveal-id="DST038C18" data-open="DST038C18" class="link link-ref link-reveal xref-bibr">18</a></sup> to rates of 8.6% in <em>Aspergillus oryzae</em><sup><a reveal-id="DST038C19" data-open="DST038C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and 4.2% in <em>C. neoformans</em>.<sup><a reveal-id="DST038C20" data-open="DST038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Remarkably, Ho <em>et al</em>.<sup><a reveal-id="DST038C21" data-open="DST038C21" class="link link-ref link-reveal xref-bibr">21</a></sup> estimate an AS rate for <em>Ustilago maydis</em> of 26% in a subset of multi-exon genes that have support by at least two ESTs.</p><p>So far, AS research was mainly focused on animals and plants. With this study, we give a comprehensive report on fungi as the third eukaryote crown group. The comparability of the previous results on fungal AS is hampered due to the application of different biochemical and computational strategies. Thus, we undertook a systematic genome-wide comparative analysis of AS in 23 informative fungal species. The basis of our analysis are alignments of transcript sequences to genome sequences, and an AS rate estimation similar to that of Kim <em>et al</em>.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                            <h2 scrollto-destination=4909501 id="4909501" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4909504 id="4909504" class="section-title" data-legacy-id=s2a>2.1. Data sources and preparation</h3>
<p>We downloaded chromosomal sequences, reference transcripts and gene annotations of 25 species (26 different strains) from NCBI's GenBank, RefSeq and EntrezGene databases, respectively.<sup><a reveal-id="DST038C22" data-open="DST038C22" class="link link-ref link-reveal xref-bibr">22</a></sup> These data were complemented with most up-to-date sequences and annotations of <em>Pichia stipitis</em> (Joint Genome Institute<sup><a reveal-id="DST038C23" data-open="DST038C23" class="link link-ref link-reveal xref-bibr">23</a></sup>) and <em>S. cerevisiae</em> (<em>Saccharomyces</em> genome database<sup><a reveal-id="DST038C24" data-open="DST038C24" class="link link-ref link-reveal xref-bibr">24</a></sup>). Genome sequences and annotations of further three species (five strains) were from the Broad Institute (<em>Fusarium oxysporum</em>, <em>Paracoccidioides brasiliensis</em> Pb01, Pb03 and Pb18, <em>Rhizopus oryzae</em><sup><a reveal-id="DST038C25" data-open="DST038C25" class="link link-ref link-reveal xref-bibr">25</a></sup>) and for further three species from the Joint Genome Institute (<em>Phanerochaete chrysosporium</em>, <em>Trichoderma reesei</em>, <em>Mycosphaerella graminicola</em><sup><a reveal-id="DST038C23" data-open="DST038C23" class="link link-ref link-reveal xref-bibr">23</a></sup>). ESTs for all species were downloaded from NCBI's dbEST database, except for <em>Arthroderma benhamiae</em>, where Roche 454 data are from NCBI's SRA.<sup><a reveal-id="DST038C22" data-open="DST038C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Four species were excluded from the analysis because there were &lt;200 ESTs. This yielded 27 species (30 strains) with sufficient data (Table <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">1</a>). We masked low-complexity repeats from the genome sequences using the program RepeatMasker (Smit <em>et al</em>., unpublished). We removed sequence contamination, low-quality and low-complexity sequences from the ESTs using SeqClean (unpublished, ‘The Gene Index Project’ of Harvard University). Roche 454 reads were additionally cleaned for adapter stretches using in-house software. </p>                        <a id="4909509" scrollto-destination="4909509"></a>
<div content-id="DST038TB1" class="table-modal"><div class="table-wrap"><div id="DST038TB1" class="table-wrap-title" data-id="DST038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Annotation, EST mapping and AS data of the studied species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Taxon<sup>a</sup> </th><th>Species </th><th>Lifestyle<sup>b</sup> </th><th>Annotated genes </th><th>Annotated introns </th><th>Number of available reads </th><th>% Filtered and mapped reads </th><th>% Genes covered with ≥2 reads </th><th>RIs </th><th>Skipped exons </th><th>Alternative 5′ intron ends </th><th>Alternative 3′ intron ends </th><th>% Genes w. any type of AS </th></tr></thead><tbody><tr><td>A </td><td><em>Ajellomyces capsulatus</em> </td><td>HP </td><td>9314 </td><td>16 275 </td><td>26 389 </td><td>55 </td><td>11 </td><td>51 </td><td>2 </td><td>5 </td><td>15 </td><td>6.5 </td></tr><tr><td>A </td><td><em>Arthroderma benhamiae</em> </td><td>HP </td><td>7984 </td><td>10 332 </td><td>1 040 774 </td><td>86 </td><td>86 </td><td>1381 </td><td>68 </td><td>292 </td><td>445 </td><td>8.2 </td></tr><tr><td>A </td><td><em>Chaetomium globosum</em> </td><td>HP </td><td>11 048 </td><td>17 396 </td><td>1557 </td><td>34 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td> </td></tr><tr><td>A </td><td><em>Coccidioides immitis</em> </td><td>HP </td><td>10 440 </td><td>17 815 </td><td>62 729 </td><td>93 </td><td>49 </td><td>664 </td><td>18 </td><td>152 </td><td>225 </td><td>13.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb01 </td><td>HP </td><td>9132 </td><td>28 179 </td><td>41 463 </td><td>75 </td><td>33 </td><td>235 </td><td>23 </td><td>67 </td><td>110 </td><td>15.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb03 </td><td>HP </td><td>7875 </td><td>19 575 </td><td>41 463 </td><td>71 </td><td>35 </td><td>134 </td><td>16 </td><td>31 </td><td>52 </td><td>10 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb18 </td><td>HP </td><td>8741 </td><td>24 498 </td><td>41 463 </td><td>71 </td><td>32 </td><td>134 </td><td>15 </td><td>31 </td><td>52 </td><td>10.5 </td></tr><tr><td>A </td><td><em>Aspergillus nidulans</em> </td><td>NP </td><td>9541 </td><td>16 797 </td><td>16 848 </td><td>89 </td><td>15 </td><td>81 </td><td>1 </td><td>11 </td><td>14 </td><td>7.3 </td></tr><tr><td>A </td><td><em>Aspergillus niger</em> </td><td>NP </td><td>10 597 </td><td>17 668 </td><td>46 938 </td><td>91 </td><td>28 </td><td>323 </td><td>7 </td><td>37 </td><td>43 </td><td>9.5 </td></tr><tr><td>A </td><td><em>Aspergillus oryzae</em> </td><td>NP </td><td>12 823 </td><td>20 916 </td><td>9051 </td><td>94 </td><td>9 </td><td>70 </td><td>2 </td><td>5 </td><td>12 </td><td> </td></tr><tr><td>A </td><td><em>Neurospora crassa</em> </td><td>NP </td><td>9841 </td><td>14 323 </td><td>277 147 </td><td>83 </td><td>52 </td><td>511 </td><td>57 </td><td>128 </td><td>164 </td><td>8.8 </td></tr><tr><td>A </td><td><em><span class="underline">Pichia stipitis</span></em> </td><td>NP </td><td>5807 </td><td>2580 </td><td>19 621 </td><td>95 </td><td>21 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>A </td><td><em>Podospora anserina</em> </td><td>NP </td><td>10 257 </td><td>11 261 </td><td>51 862 </td><td>92 </td><td>30 </td><td>194 </td><td>5 </td><td>43 </td><td>83 </td><td>4.8 </td></tr><tr><td>A </td><td><em><span class="underline">Saccharomyces cerevisiae</span></em> </td><td>NP </td><td>5781 </td><td>332 </td><td>34 915 </td><td>97 </td><td>41 </td><td>2 </td><td>0 </td><td>2 </td><td>7 </td><td>0.18 </td></tr><tr><td>A </td><td><em><span class="underline">Schizosaccharomyces pombe</span></em> </td><td>NP </td><td>5073 </td><td>3878 </td><td>8123 </td><td>78 </td><td>10 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0.6 </td></tr><tr><td>A </td><td><em>Trichoderma reesei</em> </td><td>NP </td><td>9143 </td><td>18 802 </td><td>44 964 </td><td>76 </td><td>40 </td><td>66 </td><td>2 </td><td>18 </td><td>22 </td><td>2.5 </td></tr><tr><td>A </td><td><em>Botryotinia fuckeliana</em> </td><td>PP </td><td>16 389 </td><td>22 334 </td><td>10 982 </td><td>58 </td><td>5 </td><td>19 </td><td>2 </td><td>5 </td><td>3 </td><td>2.7 </td></tr><tr><td>A </td><td><em>Fusarium oxysporum</em> </td><td>PP </td><td>17 735 </td><td>30 161 </td><td>9248 </td><td>67 </td><td>3 </td><td>33 </td><td>0 </td><td>4 </td><td>5 </td><td> </td></tr><tr><td>A </td><td><em>Gibberella zeae</em> </td><td>PP </td><td>23 218 </td><td>38 261 </td><td>21 355 </td><td>91 </td><td>14 </td><td>75 </td><td>1 </td><td>9 </td><td>16 </td><td>5.9 </td></tr><tr><td>A </td><td><em>Magnaporthe grisea</em> </td><td>PP </td><td>14 010 </td><td>18 795 </td><td>88 292 </td><td>86 </td><td>35 </td><td>222 </td><td>31 </td><td>62 </td><td>128 </td><td>7.9 </td></tr><tr><td>A </td><td><em>Mycosphaerella graminicola</em> </td><td>PP </td><td>10 952 </td><td>17 661 </td><td>32 194 </td><td>83 </td><td>33 </td><td>140 </td><td>9 </td><td>29 </td><td>55 </td><td>6.1 </td></tr><tr><td>A </td><td><em>Phaeosphaeria nodorum</em> </td><td>PP </td><td>15 983 </td><td>21 371 </td><td>15 973 </td><td>79 </td><td>9 </td><td>20 </td><td>1 </td><td>2 </td><td>7 </td><td>2.4 </td></tr><tr><td>A </td><td><em>Sclerotinia sclerotiorum</em> </td><td>PP </td><td>14 446 </td><td>20 240 </td><td>1844 </td><td>74 </td><td>1 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> B-3501A </td><td>HP </td><td>6583 </td><td>15 244 </td><td>74 724 </td><td>92 </td><td>69 </td><td>900 </td><td>31 </td><td>106 </td><td>229 </td><td>18.2 </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> JEC21 </td><td>HP </td><td>6604 </td><td>15 554 </td><td>74 724 </td><td>92 </td><td>70 </td><td>945 </td><td>31 </td><td>120 </td><td>244 </td><td>19.9 </td></tr><tr><td>B </td><td><em>Coprinopsis cinerea</em> </td><td>NP </td><td>13 544 </td><td>30 180 </td><td>15 777 </td><td>84 </td><td>15 </td><td>173 </td><td>4 </td><td>15 </td><td>36 </td><td>8.6 </td></tr><tr><td>B </td><td><em>Laccaria bicolor</em> </td><td>NP </td><td>18 216 </td><td>36 757 </td><td>34 345 </td><td>87 </td><td>21 </td><td>253 </td><td>18 </td><td>35 </td><td>74 </td><td>5.9 </td></tr><tr><td>B </td><td><em>Phanerochaete chrysosporium</em> </td><td>NP </td><td>10 048 </td><td>48 688 </td><td>12 869 </td><td>97 </td><td>18 </td><td>186 </td><td>5 </td><td>21 </td><td>51 </td><td>7.7 </td></tr><tr><td>B </td><td><em>Ustilago maydis</em> </td><td>PP </td><td>6522 </td><td>4279 </td><td>39 308 </td><td>88 </td><td>50 </td><td>34 </td><td>13 </td><td>14 </td><td>36 </td><td>2.3 </td></tr><tr><td>M </td><td><em>Rhizopus oryzae</em> </td><td>HP </td><td>17 459 </td><td>40 515 </td><td>13 313 </td><td>85 </td><td>9 </td><td>26 </td><td>0 </td><td>4 </td><td>11 </td><td>2.3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>Mean </td><td>6.4 </td></tr></tbody></table></div><div content-id="DST038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Taxon<sup>a</sup> </th><th>Species </th><th>Lifestyle<sup>b</sup> </th><th>Annotated genes </th><th>Annotated introns </th><th>Number of available reads </th><th>% Filtered and mapped reads </th><th>% Genes covered with ≥2 reads </th><th>RIs </th><th>Skipped exons </th><th>Alternative 5′ intron ends </th><th>Alternative 3′ intron ends </th><th>% Genes w. any type of AS </th></tr></thead><tbody><tr><td>A </td><td><em>Ajellomyces capsulatus</em> </td><td>HP </td><td>9314 </td><td>16 275 </td><td>26 389 </td><td>55 </td><td>11 </td><td>51 </td><td>2 </td><td>5 </td><td>15 </td><td>6.5 </td></tr><tr><td>A </td><td><em>Arthroderma benhamiae</em> </td><td>HP </td><td>7984 </td><td>10 332 </td><td>1 040 774 </td><td>86 </td><td>86 </td><td>1381 </td><td>68 </td><td>292 </td><td>445 </td><td>8.2 </td></tr><tr><td>A </td><td><em>Chaetomium globosum</em> </td><td>HP </td><td>11 048 </td><td>17 396 </td><td>1557 </td><td>34 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td> </td></tr><tr><td>A </td><td><em>Coccidioides immitis</em> </td><td>HP </td><td>10 440 </td><td>17 815 </td><td>62 729 </td><td>93 </td><td>49 </td><td>664 </td><td>18 </td><td>152 </td><td>225 </td><td>13.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb01 </td><td>HP </td><td>9132 </td><td>28 179 </td><td>41 463 </td><td>75 </td><td>33 </td><td>235 </td><td>23 </td><td>67 </td><td>110 </td><td>15.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb03 </td><td>HP </td><td>7875 </td><td>19 575 </td><td>41 463 </td><td>71 </td><td>35 </td><td>134 </td><td>16 </td><td>31 </td><td>52 </td><td>10 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb18 </td><td>HP </td><td>8741 </td><td>24 498 </td><td>41 463 </td><td>71 </td><td>32 </td><td>134 </td><td>15 </td><td>31 </td><td>52 </td><td>10.5 </td></tr><tr><td>A </td><td><em>Aspergillus nidulans</em> </td><td>NP </td><td>9541 </td><td>16 797 </td><td>16 848 </td><td>89 </td><td>15 </td><td>81 </td><td>1 </td><td>11 </td><td>14 </td><td>7.3 </td></tr><tr><td>A </td><td><em>Aspergillus niger</em> </td><td>NP </td><td>10 597 </td><td>17 668 </td><td>46 938 </td><td>91 </td><td>28 </td><td>323 </td><td>7 </td><td>37 </td><td>43 </td><td>9.5 </td></tr><tr><td>A </td><td><em>Aspergillus oryzae</em> </td><td>NP </td><td>12 823 </td><td>20 916 </td><td>9051 </td><td>94 </td><td>9 </td><td>70 </td><td>2 </td><td>5 </td><td>12 </td><td> </td></tr><tr><td>A </td><td><em>Neurospora crassa</em> </td><td>NP </td><td>9841 </td><td>14 323 </td><td>277 147 </td><td>83 </td><td>52 </td><td>511 </td><td>57 </td><td>128 </td><td>164 </td><td>8.8 </td></tr><tr><td>A </td><td><em><span class="underline">Pichia stipitis</span></em> </td><td>NP </td><td>5807 </td><td>2580 </td><td>19 621 </td><td>95 </td><td>21 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>A </td><td><em>Podospora anserina</em> </td><td>NP </td><td>10 257 </td><td>11 261 </td><td>51 862 </td><td>92 </td><td>30 </td><td>194 </td><td>5 </td><td>43 </td><td>83 </td><td>4.8 </td></tr><tr><td>A </td><td><em><span class="underline">Saccharomyces cerevisiae</span></em> </td><td>NP </td><td>5781 </td><td>332 </td><td>34 915 </td><td>97 </td><td>41 </td><td>2 </td><td>0 </td><td>2 </td><td>7 </td><td>0.18 </td></tr><tr><td>A </td><td><em><span class="underline">Schizosaccharomyces pombe</span></em> </td><td>NP </td><td>5073 </td><td>3878 </td><td>8123 </td><td>78 </td><td>10 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0.6 </td></tr><tr><td>A </td><td><em>Trichoderma reesei</em> </td><td>NP </td><td>9143 </td><td>18 802 </td><td>44 964 </td><td>76 </td><td>40 </td><td>66 </td><td>2 </td><td>18 </td><td>22 </td><td>2.5 </td></tr><tr><td>A </td><td><em>Botryotinia fuckeliana</em> </td><td>PP </td><td>16 389 </td><td>22 334 </td><td>10 982 </td><td>58 </td><td>5 </td><td>19 </td><td>2 </td><td>5 </td><td>3 </td><td>2.7 </td></tr><tr><td>A </td><td><em>Fusarium oxysporum</em> </td><td>PP </td><td>17 735 </td><td>30 161 </td><td>9248 </td><td>67 </td><td>3 </td><td>33 </td><td>0 </td><td>4 </td><td>5 </td><td> </td></tr><tr><td>A </td><td><em>Gibberella zeae</em> </td><td>PP </td><td>23 218 </td><td>38 261 </td><td>21 355 </td><td>91 </td><td>14 </td><td>75 </td><td>1 </td><td>9 </td><td>16 </td><td>5.9 </td></tr><tr><td>A </td><td><em>Magnaporthe grisea</em> </td><td>PP </td><td>14 010 </td><td>18 795 </td><td>88 292 </td><td>86 </td><td>35 </td><td>222 </td><td>31 </td><td>62 </td><td>128 </td><td>7.9 </td></tr><tr><td>A </td><td><em>Mycosphaerella graminicola</em> </td><td>PP </td><td>10 952 </td><td>17 661 </td><td>32 194 </td><td>83 </td><td>33 </td><td>140 </td><td>9 </td><td>29 </td><td>55 </td><td>6.1 </td></tr><tr><td>A </td><td><em>Phaeosphaeria nodorum</em> </td><td>PP </td><td>15 983 </td><td>21 371 </td><td>15 973 </td><td>79 </td><td>9 </td><td>20 </td><td>1 </td><td>2 </td><td>7 </td><td>2.4 </td></tr><tr><td>A </td><td><em>Sclerotinia sclerotiorum</em> </td><td>PP </td><td>14 446 </td><td>20 240 </td><td>1844 </td><td>74 </td><td>1 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> B-3501A </td><td>HP </td><td>6583 </td><td>15 244 </td><td>74 724 </td><td>92 </td><td>69 </td><td>900 </td><td>31 </td><td>106 </td><td>229 </td><td>18.2 </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> JEC21 </td><td>HP </td><td>6604 </td><td>15 554 </td><td>74 724 </td><td>92 </td><td>70 </td><td>945 </td><td>31 </td><td>120 </td><td>244 </td><td>19.9 </td></tr><tr><td>B </td><td><em>Coprinopsis cinerea</em> </td><td>NP </td><td>13 544 </td><td>30 180 </td><td>15 777 </td><td>84 </td><td>15 </td><td>173 </td><td>4 </td><td>15 </td><td>36 </td><td>8.6 </td></tr><tr><td>B </td><td><em>Laccaria bicolor</em> </td><td>NP </td><td>18 216 </td><td>36 757 </td><td>34 345 </td><td>87 </td><td>21 </td><td>253 </td><td>18 </td><td>35 </td><td>74 </td><td>5.9 </td></tr><tr><td>B </td><td><em>Phanerochaete chrysosporium</em> </td><td>NP </td><td>10 048 </td><td>48 688 </td><td>12 869 </td><td>97 </td><td>18 </td><td>186 </td><td>5 </td><td>21 </td><td>51 </td><td>7.7 </td></tr><tr><td>B </td><td><em>Ustilago maydis</em> </td><td>PP </td><td>6522 </td><td>4279 </td><td>39 308 </td><td>88 </td><td>50 </td><td>34 </td><td>13 </td><td>14 </td><td>36 </td><td>2.3 </td></tr><tr><td>M </td><td><em>Rhizopus oryzae</em> </td><td>HP </td><td>17 459 </td><td>40 515 </td><td>13 313 </td><td>85 </td><td>9 </td><td>26 </td><td>0 </td><td>4 </td><td>11 </td><td>2.3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>Mean </td><td>6.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note, for <em>P. brasiliensis</em> and <em>C. neoformans</em> the same EST data were used for all strains, and hence, the same EST statistics come about. Roche 454 transcript sequences are used for <em>A. benhamiae</em>, and classical EST data for all other species. AS rates in the last column are from random sampling.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Taxa are Ascomycota (A), Basidiomycota (B) and Mucoromycotina (M). Yeasts are underlined.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Lifestyle: non-pathogenic (NP), plant pathogenic (PP), human pathogenic (HP).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4909509" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST038TB1" class="table-wrap-title" data-id="DST038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Annotation, EST mapping and AS data of the studied species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Taxon<sup>a</sup> </th><th>Species </th><th>Lifestyle<sup>b</sup> </th><th>Annotated genes </th><th>Annotated introns </th><th>Number of available reads </th><th>% Filtered and mapped reads </th><th>% Genes covered with ≥2 reads </th><th>RIs </th><th>Skipped exons </th><th>Alternative 5′ intron ends </th><th>Alternative 3′ intron ends </th><th>% Genes w. any type of AS </th></tr></thead><tbody><tr><td>A </td><td><em>Ajellomyces capsulatus</em> </td><td>HP </td><td>9314 </td><td>16 275 </td><td>26 389 </td><td>55 </td><td>11 </td><td>51 </td><td>2 </td><td>5 </td><td>15 </td><td>6.5 </td></tr><tr><td>A </td><td><em>Arthroderma benhamiae</em> </td><td>HP </td><td>7984 </td><td>10 332 </td><td>1 040 774 </td><td>86 </td><td>86 </td><td>1381 </td><td>68 </td><td>292 </td><td>445 </td><td>8.2 </td></tr><tr><td>A </td><td><em>Chaetomium globosum</em> </td><td>HP </td><td>11 048 </td><td>17 396 </td><td>1557 </td><td>34 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td> </td></tr><tr><td>A </td><td><em>Coccidioides immitis</em> </td><td>HP </td><td>10 440 </td><td>17 815 </td><td>62 729 </td><td>93 </td><td>49 </td><td>664 </td><td>18 </td><td>152 </td><td>225 </td><td>13.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb01 </td><td>HP </td><td>9132 </td><td>28 179 </td><td>41 463 </td><td>75 </td><td>33 </td><td>235 </td><td>23 </td><td>67 </td><td>110 </td><td>15.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb03 </td><td>HP </td><td>7875 </td><td>19 575 </td><td>41 463 </td><td>71 </td><td>35 </td><td>134 </td><td>16 </td><td>31 </td><td>52 </td><td>10 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb18 </td><td>HP </td><td>8741 </td><td>24 498 </td><td>41 463 </td><td>71 </td><td>32 </td><td>134 </td><td>15 </td><td>31 </td><td>52 </td><td>10.5 </td></tr><tr><td>A </td><td><em>Aspergillus nidulans</em> </td><td>NP </td><td>9541 </td><td>16 797 </td><td>16 848 </td><td>89 </td><td>15 </td><td>81 </td><td>1 </td><td>11 </td><td>14 </td><td>7.3 </td></tr><tr><td>A </td><td><em>Aspergillus niger</em> </td><td>NP </td><td>10 597 </td><td>17 668 </td><td>46 938 </td><td>91 </td><td>28 </td><td>323 </td><td>7 </td><td>37 </td><td>43 </td><td>9.5 </td></tr><tr><td>A </td><td><em>Aspergillus oryzae</em> </td><td>NP </td><td>12 823 </td><td>20 916 </td><td>9051 </td><td>94 </td><td>9 </td><td>70 </td><td>2 </td><td>5 </td><td>12 </td><td> </td></tr><tr><td>A </td><td><em>Neurospora crassa</em> </td><td>NP </td><td>9841 </td><td>14 323 </td><td>277 147 </td><td>83 </td><td>52 </td><td>511 </td><td>57 </td><td>128 </td><td>164 </td><td>8.8 </td></tr><tr><td>A </td><td><em><span class="underline">Pichia stipitis</span></em> </td><td>NP </td><td>5807 </td><td>2580 </td><td>19 621 </td><td>95 </td><td>21 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>A </td><td><em>Podospora anserina</em> </td><td>NP </td><td>10 257 </td><td>11 261 </td><td>51 862 </td><td>92 </td><td>30 </td><td>194 </td><td>5 </td><td>43 </td><td>83 </td><td>4.8 </td></tr><tr><td>A </td><td><em><span class="underline">Saccharomyces cerevisiae</span></em> </td><td>NP </td><td>5781 </td><td>332 </td><td>34 915 </td><td>97 </td><td>41 </td><td>2 </td><td>0 </td><td>2 </td><td>7 </td><td>0.18 </td></tr><tr><td>A </td><td><em><span class="underline">Schizosaccharomyces pombe</span></em> </td><td>NP </td><td>5073 </td><td>3878 </td><td>8123 </td><td>78 </td><td>10 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0.6 </td></tr><tr><td>A </td><td><em>Trichoderma reesei</em> </td><td>NP </td><td>9143 </td><td>18 802 </td><td>44 964 </td><td>76 </td><td>40 </td><td>66 </td><td>2 </td><td>18 </td><td>22 </td><td>2.5 </td></tr><tr><td>A </td><td><em>Botryotinia fuckeliana</em> </td><td>PP </td><td>16 389 </td><td>22 334 </td><td>10 982 </td><td>58 </td><td>5 </td><td>19 </td><td>2 </td><td>5 </td><td>3 </td><td>2.7 </td></tr><tr><td>A </td><td><em>Fusarium oxysporum</em> </td><td>PP </td><td>17 735 </td><td>30 161 </td><td>9248 </td><td>67 </td><td>3 </td><td>33 </td><td>0 </td><td>4 </td><td>5 </td><td> </td></tr><tr><td>A </td><td><em>Gibberella zeae</em> </td><td>PP </td><td>23 218 </td><td>38 261 </td><td>21 355 </td><td>91 </td><td>14 </td><td>75 </td><td>1 </td><td>9 </td><td>16 </td><td>5.9 </td></tr><tr><td>A </td><td><em>Magnaporthe grisea</em> </td><td>PP </td><td>14 010 </td><td>18 795 </td><td>88 292 </td><td>86 </td><td>35 </td><td>222 </td><td>31 </td><td>62 </td><td>128 </td><td>7.9 </td></tr><tr><td>A </td><td><em>Mycosphaerella graminicola</em> </td><td>PP </td><td>10 952 </td><td>17 661 </td><td>32 194 </td><td>83 </td><td>33 </td><td>140 </td><td>9 </td><td>29 </td><td>55 </td><td>6.1 </td></tr><tr><td>A </td><td><em>Phaeosphaeria nodorum</em> </td><td>PP </td><td>15 983 </td><td>21 371 </td><td>15 973 </td><td>79 </td><td>9 </td><td>20 </td><td>1 </td><td>2 </td><td>7 </td><td>2.4 </td></tr><tr><td>A </td><td><em>Sclerotinia sclerotiorum</em> </td><td>PP </td><td>14 446 </td><td>20 240 </td><td>1844 </td><td>74 </td><td>1 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> B-3501A </td><td>HP </td><td>6583 </td><td>15 244 </td><td>74 724 </td><td>92 </td><td>69 </td><td>900 </td><td>31 </td><td>106 </td><td>229 </td><td>18.2 </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> JEC21 </td><td>HP </td><td>6604 </td><td>15 554 </td><td>74 724 </td><td>92 </td><td>70 </td><td>945 </td><td>31 </td><td>120 </td><td>244 </td><td>19.9 </td></tr><tr><td>B </td><td><em>Coprinopsis cinerea</em> </td><td>NP </td><td>13 544 </td><td>30 180 </td><td>15 777 </td><td>84 </td><td>15 </td><td>173 </td><td>4 </td><td>15 </td><td>36 </td><td>8.6 </td></tr><tr><td>B </td><td><em>Laccaria bicolor</em> </td><td>NP </td><td>18 216 </td><td>36 757 </td><td>34 345 </td><td>87 </td><td>21 </td><td>253 </td><td>18 </td><td>35 </td><td>74 </td><td>5.9 </td></tr><tr><td>B </td><td><em>Phanerochaete chrysosporium</em> </td><td>NP </td><td>10 048 </td><td>48 688 </td><td>12 869 </td><td>97 </td><td>18 </td><td>186 </td><td>5 </td><td>21 </td><td>51 </td><td>7.7 </td></tr><tr><td>B </td><td><em>Ustilago maydis</em> </td><td>PP </td><td>6522 </td><td>4279 </td><td>39 308 </td><td>88 </td><td>50 </td><td>34 </td><td>13 </td><td>14 </td><td>36 </td><td>2.3 </td></tr><tr><td>M </td><td><em>Rhizopus oryzae</em> </td><td>HP </td><td>17 459 </td><td>40 515 </td><td>13 313 </td><td>85 </td><td>9 </td><td>26 </td><td>0 </td><td>4 </td><td>11 </td><td>2.3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>Mean </td><td>6.4 </td></tr></tbody></table></div><div content-id="DST038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Taxon<sup>a</sup> </th><th>Species </th><th>Lifestyle<sup>b</sup> </th><th>Annotated genes </th><th>Annotated introns </th><th>Number of available reads </th><th>% Filtered and mapped reads </th><th>% Genes covered with ≥2 reads </th><th>RIs </th><th>Skipped exons </th><th>Alternative 5′ intron ends </th><th>Alternative 3′ intron ends </th><th>% Genes w. any type of AS </th></tr></thead><tbody><tr><td>A </td><td><em>Ajellomyces capsulatus</em> </td><td>HP </td><td>9314 </td><td>16 275 </td><td>26 389 </td><td>55 </td><td>11 </td><td>51 </td><td>2 </td><td>5 </td><td>15 </td><td>6.5 </td></tr><tr><td>A </td><td><em>Arthroderma benhamiae</em> </td><td>HP </td><td>7984 </td><td>10 332 </td><td>1 040 774 </td><td>86 </td><td>86 </td><td>1381 </td><td>68 </td><td>292 </td><td>445 </td><td>8.2 </td></tr><tr><td>A </td><td><em>Chaetomium globosum</em> </td><td>HP </td><td>11 048 </td><td>17 396 </td><td>1557 </td><td>34 </td><td>1 </td><td>1 </td><td>0 </td><td>0 </td><td>0 </td><td> </td></tr><tr><td>A </td><td><em>Coccidioides immitis</em> </td><td>HP </td><td>10 440 </td><td>17 815 </td><td>62 729 </td><td>93 </td><td>49 </td><td>664 </td><td>18 </td><td>152 </td><td>225 </td><td>13.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb01 </td><td>HP </td><td>9132 </td><td>28 179 </td><td>41 463 </td><td>75 </td><td>33 </td><td>235 </td><td>23 </td><td>67 </td><td>110 </td><td>15.4 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb03 </td><td>HP </td><td>7875 </td><td>19 575 </td><td>41 463 </td><td>71 </td><td>35 </td><td>134 </td><td>16 </td><td>31 </td><td>52 </td><td>10 </td></tr><tr><td>A </td><td><em>Paracoccidioides brasiliensis</em> Pb18 </td><td>HP </td><td>8741 </td><td>24 498 </td><td>41 463 </td><td>71 </td><td>32 </td><td>134 </td><td>15 </td><td>31 </td><td>52 </td><td>10.5 </td></tr><tr><td>A </td><td><em>Aspergillus nidulans</em> </td><td>NP </td><td>9541 </td><td>16 797 </td><td>16 848 </td><td>89 </td><td>15 </td><td>81 </td><td>1 </td><td>11 </td><td>14 </td><td>7.3 </td></tr><tr><td>A </td><td><em>Aspergillus niger</em> </td><td>NP </td><td>10 597 </td><td>17 668 </td><td>46 938 </td><td>91 </td><td>28 </td><td>323 </td><td>7 </td><td>37 </td><td>43 </td><td>9.5 </td></tr><tr><td>A </td><td><em>Aspergillus oryzae</em> </td><td>NP </td><td>12 823 </td><td>20 916 </td><td>9051 </td><td>94 </td><td>9 </td><td>70 </td><td>2 </td><td>5 </td><td>12 </td><td> </td></tr><tr><td>A </td><td><em>Neurospora crassa</em> </td><td>NP </td><td>9841 </td><td>14 323 </td><td>277 147 </td><td>83 </td><td>52 </td><td>511 </td><td>57 </td><td>128 </td><td>164 </td><td>8.8 </td></tr><tr><td>A </td><td><em><span class="underline">Pichia stipitis</span></em> </td><td>NP </td><td>5807 </td><td>2580 </td><td>19 621 </td><td>95 </td><td>21 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>A </td><td><em>Podospora anserina</em> </td><td>NP </td><td>10 257 </td><td>11 261 </td><td>51 862 </td><td>92 </td><td>30 </td><td>194 </td><td>5 </td><td>43 </td><td>83 </td><td>4.8 </td></tr><tr><td>A </td><td><em><span class="underline">Saccharomyces cerevisiae</span></em> </td><td>NP </td><td>5781 </td><td>332 </td><td>34 915 </td><td>97 </td><td>41 </td><td>2 </td><td>0 </td><td>2 </td><td>7 </td><td>0.18 </td></tr><tr><td>A </td><td><em><span class="underline">Schizosaccharomyces pombe</span></em> </td><td>NP </td><td>5073 </td><td>3878 </td><td>8123 </td><td>78 </td><td>10 </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>0.6 </td></tr><tr><td>A </td><td><em>Trichoderma reesei</em> </td><td>NP </td><td>9143 </td><td>18 802 </td><td>44 964 </td><td>76 </td><td>40 </td><td>66 </td><td>2 </td><td>18 </td><td>22 </td><td>2.5 </td></tr><tr><td>A </td><td><em>Botryotinia fuckeliana</em> </td><td>PP </td><td>16 389 </td><td>22 334 </td><td>10 982 </td><td>58 </td><td>5 </td><td>19 </td><td>2 </td><td>5 </td><td>3 </td><td>2.7 </td></tr><tr><td>A </td><td><em>Fusarium oxysporum</em> </td><td>PP </td><td>17 735 </td><td>30 161 </td><td>9248 </td><td>67 </td><td>3 </td><td>33 </td><td>0 </td><td>4 </td><td>5 </td><td> </td></tr><tr><td>A </td><td><em>Gibberella zeae</em> </td><td>PP </td><td>23 218 </td><td>38 261 </td><td>21 355 </td><td>91 </td><td>14 </td><td>75 </td><td>1 </td><td>9 </td><td>16 </td><td>5.9 </td></tr><tr><td>A </td><td><em>Magnaporthe grisea</em> </td><td>PP </td><td>14 010 </td><td>18 795 </td><td>88 292 </td><td>86 </td><td>35 </td><td>222 </td><td>31 </td><td>62 </td><td>128 </td><td>7.9 </td></tr><tr><td>A </td><td><em>Mycosphaerella graminicola</em> </td><td>PP </td><td>10 952 </td><td>17 661 </td><td>32 194 </td><td>83 </td><td>33 </td><td>140 </td><td>9 </td><td>29 </td><td>55 </td><td>6.1 </td></tr><tr><td>A </td><td><em>Phaeosphaeria nodorum</em> </td><td>PP </td><td>15 983 </td><td>21 371 </td><td>15 973 </td><td>79 </td><td>9 </td><td>20 </td><td>1 </td><td>2 </td><td>7 </td><td>2.4 </td></tr><tr><td>A </td><td><em>Sclerotinia sclerotiorum</em> </td><td>PP </td><td>14 446 </td><td>20 240 </td><td>1844 </td><td>74 </td><td>1 </td><td>2 </td><td>0 </td><td>1 </td><td>0 </td><td> </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> B-3501A </td><td>HP </td><td>6583 </td><td>15 244 </td><td>74 724 </td><td>92 </td><td>69 </td><td>900 </td><td>31 </td><td>106 </td><td>229 </td><td>18.2 </td></tr><tr><td>B </td><td><em>Cryptococcus neoformans</em> JEC21 </td><td>HP </td><td>6604 </td><td>15 554 </td><td>74 724 </td><td>92 </td><td>70 </td><td>945 </td><td>31 </td><td>120 </td><td>244 </td><td>19.9 </td></tr><tr><td>B </td><td><em>Coprinopsis cinerea</em> </td><td>NP </td><td>13 544 </td><td>30 180 </td><td>15 777 </td><td>84 </td><td>15 </td><td>173 </td><td>4 </td><td>15 </td><td>36 </td><td>8.6 </td></tr><tr><td>B </td><td><em>Laccaria bicolor</em> </td><td>NP </td><td>18 216 </td><td>36 757 </td><td>34 345 </td><td>87 </td><td>21 </td><td>253 </td><td>18 </td><td>35 </td><td>74 </td><td>5.9 </td></tr><tr><td>B </td><td><em>Phanerochaete chrysosporium</em> </td><td>NP </td><td>10 048 </td><td>48 688 </td><td>12 869 </td><td>97 </td><td>18 </td><td>186 </td><td>5 </td><td>21 </td><td>51 </td><td>7.7 </td></tr><tr><td>B </td><td><em>Ustilago maydis</em> </td><td>PP </td><td>6522 </td><td>4279 </td><td>39 308 </td><td>88 </td><td>50 </td><td>34 </td><td>13 </td><td>14 </td><td>36 </td><td>2.3 </td></tr><tr><td>M </td><td><em>Rhizopus oryzae</em> </td><td>HP </td><td>17 459 </td><td>40 515 </td><td>13 313 </td><td>85 </td><td>9 </td><td>26 </td><td>0 </td><td>4 </td><td>11 </td><td>2.3 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>Mean </td><td>6.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note, for <em>P. brasiliensis</em> and <em>C. neoformans</em> the same EST data were used for all strains, and hence, the same EST statistics come about. Roche 454 transcript sequences are used for <em>A. benhamiae</em>, and classical EST data for all other species. AS rates in the last column are from random sampling.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Taxa are Ascomycota (A), Basidiomycota (B) and Mucoromycotina (M). Yeasts are underlined.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Lifestyle: non-pathogenic (NP), plant pathogenic (PP), human pathogenic (HP).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4909509" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4909511 id="4909511" class="section-title" data-legacy-id=s2b>2.2. Transcriptome-genome alignments and splice site conservation</h3>
<p>Spliced transcript-genome alignments were built in two steps: ESTs were mapped with Blat<sup><a reveal-id="DST038C26" data-open="DST038C26" class="link link-ref link-reveal xref-bibr">26</a></sup> to obtain first rough guide alignments. The best Blat hits were further splice aligned with exalin.<sup><a reveal-id="DST038C27" data-open="DST038C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To use SS information as additional input for exalin, we prepared a scoring matrix based on SS consensi from <em>Neurospora crassa</em> as suggested by Zhang <em>et al</em>. Since SSs are conserved among fungi, this model was used for the analysis of all species. Alignments were filtered for minimal score (20 bits), mismatches (≤10%, no mismatches in 5 nt region of SSs) and minimum length of exons and introns (6 and 40 nt, respectively). Only alignments with SSs from canonical (GT|AG) or well-accepted non-canonical (GC|AG, AT|AC) classes<sup><a reveal-id="DST038C16" data-open="DST038C16" class="link link-ref link-reveal xref-bibr">16</a></sup> were considered for further analysis.</p><p>SS sequence conservation was calculated as information content per position.<sup><a reveal-id="DST038C28" data-open="DST038C28" class="link link-ref link-reveal xref-bibr">28</a></sup> To this end, we extracted the sequence from −4 to +7 nt from the exon–intron boundary, and the region from −4 to +4 nt from the intron–exon boundary. We used the upstream boundary of alternative 3′ SSs and the downstream boundary of alternative 5′ SSs.</p>                            <h3 scrollto-destination=4909516 id="4909516" class="section-title" data-legacy-id=s2c>2.3. Detection of AS</h3>
<p>Custom Perl scripts were used to analyse filtered transcript-genome alignments for four AS events: exon skipping (cassette exon), alternative 5′SS and alternative 3′SS selection and intron retention. Using splice positions (genomic starts and ends of exons and introns) we compared the positions between all exons and introns to find overlaps and identify the basic AS types. AS events were predicted based on EST discrepancies only, not on discrepancies between ESTs and annotations. To account for the limited sequence data used in our analysis, one EST was considered sufficient to support an mRNA isoform. Constitutively spliced exons and introns were defined as not having support of AS at a minimum coverage of 10 ESTs.</p>                            <h3 scrollto-destination=4909520 id="4909520" class="section-title" data-legacy-id=s2d>2.4. Random sampling of transcripts and per-gene AS rates</h3>
<p>Random sampling was done for each genomic location with <em>n</em> ≥ 2 aligned transcripts. We randomly drew a defined number of transcripts and estimated the AS rate. To do so, AS events were assigned to genes based on mapping coordinates, and the number of AS affected genes was divided by the overall number of genes detected by random sampling. Then, we multiplied the AS rates with the number of genes having introns (potential AS candidates) divided by the number of all genes. This yielded whole genome AS rate estimations. This procedure was repeated 20 times to calculate a mean AS rate estimation. The procedure was done with different sampling depths, drawing 2–10 ESTs per locus. Due to a low EST coverage, loci with a higher coverage than 10 ESTs are rare for most analysed species. Thus, to avoid a bias towards highly expressed genes, results from lower sampling depths were kept in sampling repeats with higher sampling depths. That is, sampling depth <em>i</em> means to draw at most <em>i</em> ESTs from a locus. Pearson's product moment and its corresponding significance test was used to assess the correlation between AS rates and number of mapped ESTs (based on Student's <em>t</em>-test, assuming normal distribution of the data, R version 2.12.1<sup><a reveal-id="DST038C29" data-open="DST038C29" class="link link-ref link-reveal xref-bibr">29</a></sup>). Four species were excluded from correlation analysis because &lt;5% of their annotated genes were covered by the sampled ESTs (Table <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>): <em>Aspergillus oryzae</em>, <em>Chaeto-mium globosum</em>, <em>Fusarium oxysporum</em>, <em>Sclerotinia sclerotiorum</em>.</p>                            <h3 scrollto-destination=4909525 id="4909525" class="section-title" data-legacy-id=s2e>2.5. Functional gene annotations and enrichment statistics</h3>
<p>Genes were searched for protein domain motifs using HMMER3<sup><a reveal-id="DST038C30" data-open="DST038C30" class="link link-ref link-reveal xref-bibr">30</a></sup> (<em>e</em>-value &lt; 0.01) together with the Pfam database (release 24). Alternatively, Pfam domain annotations were downloaded from the Broad Institute (<em>Fusarium</em> sp., <em>Paracoccidioides</em> sp., <em>R. oryzae</em>). Associations between Pfam domains and AS were tested using the following model: per species, all genes that have at least two EST hits are taken into consideration with their Pfam assignment and number of introns. Thereof, all introns are assumed to have an equal, species-specific probability <em>p</em> to be alternatively spliced, as averaged from the empirical data. The probability <em>P</em>(<em>g</em>∈AS) that a gene with <em>n</em> introns is alternatively spliced is calculated as <em>P</em>(<em>g</em>∈ AS) = 1 − (1 − <em>p</em>)<em><sup>n</sup></em>.</p><p>Then, the expected number of alternatively spliced genes coding a certain Pfam is calculated by cumulation: Exp(<em>n</em><sub>AS,Pfam</sub>) = Σ<em>P</em>(<em>g<sub>i</sub></em> ∈ AS).</p><p>The distribution of <em>n</em><sub>AS,Pfam</sub> was obtained from Monte Carlo simulation of the cumulation terms (<em>n</em> = 10<sup>6</sup>). Binomial rather than hypergeometric simulation of <em>P</em>(<em>g<sub>i</sub></em> ∈ AS) simplified the calculations and yielded a slightly wider distribution, resulting in conservative estimates of the distribution quantiles. Correction for multiple testing was done using the Bonferroni method.</p>                            <h2 scrollto-destination=4909533 id="4909533" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4909536 id="4909536" class="section-title" data-legacy-id=s3a>3.1. Mining of introns and splicing signals</h3>
<p>The number of available ESTs per species varies in a wide range (1557–1 040 774). To detect AS, at least two transcripts per locus are needed, i.e. one for each of at least two splicing isoforms. We find that, depending on the species, 0–100% of the annotated introns are overlapped by at least two ESTs (25% on average over all fungi; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), and 1–86% of genes are overlapped by at least two ESTs (28% on average; Table <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Per species, 98–100% of the detected introns per species harbour typical SSs (GT|AG), whereas non-canonical SSs (GC|AG, AT|AC) are rare (0–2%), in accordance with a previous study on fungi.<sup><a reveal-id="DST038C16" data-open="DST038C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The sets of reliable genomic intron and exon coordinates were subsequently examined for AS events.</p>                            <h3 scrollto-destination=4909541 id="4909541" class="section-title" data-legacy-id=s3b>3.2. Whole genome AS rates</h3>
<p>The numbers of detected AS events strongly depend on the numbers of available ESTs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1a</a>; Pearson correlation coefficient <em>r</em> = 0.82, <em>P</em>-value 1.8 × 10<sup>−6</sup>). A very high coverage of introns with ESTs, especially when using next generation transcriptome sequencing, can reveal even very rare events that may partly represent splicing noise of the cell. This can lead to overestimation of AS propensity of a species. On the other hand, an uneven genomic distribution of transcripts leads to an under-sampling of the genome-wide splice isoforms. To circumvent these pitfalls, we applied a random sampling strategy, similar to the one of Kim <em>et al</em>.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> to obtain AS rate estimations that are independent of EST amounts and distributions. We left out species where &lt;5% of multi-exon genes were covered by the sampled ESTs for estimation of whole genome AS rates (last column <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). For them we do not expect the estimations to be reliable enough. We mapped the AS events that were recovered by random sampling to genomic locations of annotated genes to calculate AS rates per gene. We found that the correlation between these AS rates and the EST numbers is clearly reduced (<em>r</em> = 0.16, <em>P</em>-value = 0.46, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1b</a>). Thus, random sampling gives AS rate estimates that are comparable between species.</p><p>The more ESTs were sampled from a genomic location (sampling depth) the higher is the chance of finding AS events (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). We decided to sample up to 10 ESTs per locus to reduce the chance of sampling rare events and, thus, overestimation of AS capacities. The reduced gains of AS rates with higher sampling depth support this decision (decreasing slopes of curves in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Thus, the following results refer to a sampling depth of 10 ESTs, if not stated differently.</p><p>6.4% of fungal genes are affected by AS when averaging on species level (Table <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Excluding ascomycetous yeasts (<em>P. stipitis</em>, <em>S. cerevisiae</em> and <em>S. pombe</em> 0.26% AS affected genes), the rate is 7.3%. <em>Coccidioides immitis</em> and <em>C. neoformans</em> show outstanding AS rates of 13 and 18/20%, respectively (strains JEC21 and B-3501A). The relative proportions of the AS types averaged over all species, in the order of frequency are: intron retention 61%, alternative 3′ SSs 23%, alternative 5′ SSs 13% and skipped exons 3% (Fig. <a reveal-id="DST038F1" data-open="DST038F1" class="link link-reveal link-table xref-fig">1</a>a). We only took into account strains B-3501A and Pb01 from <em>C. neoformans</em> and <em>P. brasiliensis</em>, respectively, for mean value calculation. </p>                        <a id="4909550" scrollto-destination="4909550"></a>
<div id="DST038F1" data-id="DST038F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/m_dst03801.png?Expires=2147483647&Signature=aoLOxAFXPkga2uNkbRL4w70ApHBNmPEq2EoW0cu-bO5E~5x3v3r035M73vPMl-75YVBa9AIC~k6wZuIuClazifGuTVPpVWmtFt3WMPBLJFjfm7NfRmpmBgx7mrQpKDYCG~aA~HJqFXPYPyVpSFO5jTxnT22tR7cWMDRmj4dKETkm-tj1EaLzJ6drwvpcPUHrvyuiVsxkvRrPSYqVrQtCC5bRhlOQ2EZV6~KkG0RilwBo37b9dBNYWr6AUT6LR6Y2RTD~Hbmh7KINKHvpvmfluBcQmHGhnFG9kWS69-LMIXGJw7-oDrTm9hxIkB22Q692rjdeBwHd8IH~kQq~KCJb~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alternative splice type distribution per taxon from random sampling approach. Pie portions: intron retention (IR), skipped exons (SE), alternative 5′ splice sites (A5′SS) and 3′ splice sites (A3′SS). Only the 23 informative fungi are considered, i.e. those where AS rates could be estimated (cf. Table 1). Only non-yeasts are considered in chart 1B (17 ascomycetes − 3 yeasts = 14)." path-from-xml="dst03801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4909550/dst03801.tiff" path-from-xml="dst03801.tiff" target="_blank">View large</a><a section="4909550" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/dst03801.png?Expires=2147483647&Signature=KFQ~jvmiCLTh3hppabN7KPxTaZ0CdwCOrNwqaLch1sHbLpzTnKArQ4XEoSmTDIk9cTUtIEBoN0aAnhINDjZPNc9zjkGaNLhROlAfsOjURYy2IMl6M1qF-KdRBvP5EBA4xNahM2yN~GRoO4aXHThGwKmDJdgZKcRh3Qa7n1v5z1EjHx-tZ83JlwRg4tR2IWnf2OGwD1Z40UufwARBSHgV2tOEEWlfaOoN5LKBkD4jlQbS-~xdypdIIRLPBLE3Ela0V3Q3CWdvcb6dJ08WIo1zuqqy~qYtqCmkyRdClOgtvfIuukm0Be0jv44Y2xwDqxiE63zO4paL8Vutqmj3fy0j-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4909550&ar=347182&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alternative splice type distribution per taxon from random sampling approach. Pie portions: intron retention (IR), skipped exons (SE), alternative 5′ splice sites (A5′SS) and 3′ splice sites (A3′SS). Only the 23 informative fungi are considered, i.e. those where AS rates could be estimated (cf. <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">Table 1</a>). Only non-yeasts are considered in chart 1B (17 ascomycetes − 3 yeasts = 14).</p></div></div><div content-id="DST038F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/m_dst03801.png?Expires=2147483647&Signature=aoLOxAFXPkga2uNkbRL4w70ApHBNmPEq2EoW0cu-bO5E~5x3v3r035M73vPMl-75YVBa9AIC~k6wZuIuClazifGuTVPpVWmtFt3WMPBLJFjfm7NfRmpmBgx7mrQpKDYCG~aA~HJqFXPYPyVpSFO5jTxnT22tR7cWMDRmj4dKETkm-tj1EaLzJ6drwvpcPUHrvyuiVsxkvRrPSYqVrQtCC5bRhlOQ2EZV6~KkG0RilwBo37b9dBNYWr6AUT6LR6Y2RTD~Hbmh7KINKHvpvmfluBcQmHGhnFG9kWS69-LMIXGJw7-oDrTm9hxIkB22Q692rjdeBwHd8IH~kQq~KCJb~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alternative splice type distribution per taxon from random sampling approach. Pie portions: intron retention (IR), skipped exons (SE), alternative 5′ splice sites (A5′SS) and 3′ splice sites (A3′SS). Only the 23 informative fungi are considered, i.e. those where AS rates could be estimated (cf. Table 1). Only non-yeasts are considered in chart 1B (17 ascomycetes − 3 yeasts = 14)." path-from-xml="dst03801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4909550/dst03801.tiff" path-from-xml="dst03801.tiff" target="_blank">View large</a><a section="4909550" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/dst03801.png?Expires=2147483647&Signature=KFQ~jvmiCLTh3hppabN7KPxTaZ0CdwCOrNwqaLch1sHbLpzTnKArQ4XEoSmTDIk9cTUtIEBoN0aAnhINDjZPNc9zjkGaNLhROlAfsOjURYy2IMl6M1qF-KdRBvP5EBA4xNahM2yN~GRoO4aXHThGwKmDJdgZKcRh3Qa7n1v5z1EjHx-tZ83JlwRg4tR2IWnf2OGwD1Z40UufwARBSHgV2tOEEWlfaOoN5LKBkD4jlQbS-~xdypdIIRLPBLE3Ela0V3Q3CWdvcb6dJ08WIo1zuqqy~qYtqCmkyRdClOgtvfIuukm0Be0jv44Y2xwDqxiE63zO4paL8Vutqmj3fy0j-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4909550&ar=347182&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alternative splice type distribution per taxon from random sampling approach. Pie portions: intron retention (IR), skipped exons (SE), alternative 5′ splice sites (A5′SS) and 3′ splice sites (A3′SS). Only the 23 informative fungi are considered, i.e. those where AS rates could be estimated (cf. <a reveal-id="DST038TB1" data-open="DST038TB1" class="link link-reveal link-table xref-fig">Table 1</a>). Only non-yeasts are considered in chart 1B (17 ascomycetes − 3 yeasts = 14).</p></div></div>                            <h3 scrollto-destination=4909554 id="4909554" class="section-title" data-legacy-id=s3c>3.3. Validation of retained introns</h3>
<p>An alternative explanation for detected retained introns (RIs) would be the presence of unprocessed pre-mRNA in the sequenced samples or a contamination with DNA. First of all, the EST libraries used for this study were all prepared from total RNA and enriched for poly(A)-mRNA. This makes DNA contamination very unlikely. To further validate the detected RIs, we assessed the number of RI-supporting ESTs that have been already processed in the following way. For each species, we counted the number of RIs where at least one EST of the isoform that harbours the RI supports a spliced intron at another EST position. For all species with RIs, between 74 and 100% (average 96%) of those isoforms contain a processed intron (details see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). This clearly indicates that most RIs are authentic RNA events.</p>                            <h3 scrollto-destination=4909559 id="4909559" class="section-title" data-legacy-id=s3d>3.4. Correlations of AS rates and genomic features</h3>
<p>We calculated the correlations between genome and splicing quantities. We find a strong correlation (<em>r</em> = 0.73, <em>P</em>-value 8.8 × 10<sup>−5</sup>) between the number of EST-covered introns and the number of RIs across the species. This hints at fungal introns to have a certain chance <em>per se</em> to be retained in an alternative manner. In contrast, there is only a slight and barely significant correlation of the extrapolated genome-wide AS numbers with gene numbers (<em>r</em> = 0.41, <em>P</em> = 0.0502) and with genome size in nucleotides (<em>r</em> = 0.53, <em>P</em> = 0.009). Further, there is no correlation of the AS rate per gene with gene numbers (<em>r</em> = −0.05, <em>P</em> = 0.82) nor with genome size (<em>r</em> = 0.12, <em>P</em> = 0.57). Nonetheless, the small genome sizes (in base pairs and gene numbers) of the yeasts <em>S. cerevisiae</em>, <em>S. pombe</em> and <em>P. stipitis</em> coincide with their clearly reduced AS propensity.</p>                            <h3 scrollto-destination=4909563 id="4909563" class="section-title" data-legacy-id=s3e>3.5. Intron retention is the major AS type in fungi</h3>
<p>Intron retention makes up two-thirds of the AS events in the investigated fungi. This also holds for each fungal group separately. We investigated the properties of affected introns and their aberration from constitutively spliced ones. We find that RIs are shorter (89 nt) than constitutively spliced introns (93 nt), on average across all species, though not significant (Mann–Whitney U-test, <em>P</em> = 0.211, <em>n</em> = 5665/23 268). Neither constitutively spliced nor RIs tend to preserve the reading frame. That is, in both sets, intron lengths are distributed evenly over the three possible remainders of division by three. Constitutively spliced introns: remainder zero, 32%; remainder one, 34%; remainder two, 34%; RIs: 33, 34 and 33%, respectively.</p>                            <h3 scrollto-destination=4909567 id="4909567" class="section-title" data-legacy-id=s3f>3.6. Varying alternative splice propensity is taxon-dependent</h3>
<p>We summarized and averaged the resampled AS rates into two different fungal taxa (see a species tree in Fig. <a reveal-id="DST038F2" data-open="DST038F2" class="link link-reveal link-table xref-fig">2</a>). On average in Basidiomycota more genes are affected by AS (8.6%) than in Ascomycota (7.2% w/o ascomycetous yeasts, Mann–Whitney U-test, not significant, <em>n</em> = 5/14). Without the species showing outlying AS rates (<em>C. immitis</em>, <em>P. brasiliensis</em>, <em>C. neoformans</em>), the rates for Basidiomycota (6.1%) and Ascomycota (4.9%) are still different. Basidio-mycota and Ascomycota have very similar AS type proportions, with Basidiomycota showing slightly more RIs and less alternative 5′ SSs (Fig. <a reveal-id="DST038F1" data-open="DST038F1" class="link link-reveal link-table xref-fig">1</a>). In both cases RIs make up around two-thirds of all AS events while skipped exons are only marginally present. The ascomycetous yeasts of our study (<em>P. stipitis</em>, <em>S. cerevisiae</em> and <em>S. pombe</em>) show an AS rate of 0.26% on average, which is significantly lower than the rate of the other Ascomycota (Mann–Whitney U-test, <em>P</em>-value 0.003, <em>n</em> = 3/14). An explanation for this difference may be deviations in structural gene properties that influence splicing. We find that Basidiomycota have on average shorter constitutively spliced introns (86 nt) than Ascomycota (96 nt, Mann–Whitney U-test, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <em>n</em> = 5205/17 936), and also shorter RIs (72 nt vs. 95 nt, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <em>n</em> = 1545/4093). Considering the ascomycetous yeasts separately, they show on average 326-nt long constitutively spliced introns and 132-nt long RIs, though it should be noted that yeast RI data are only based on five introns. In contrast, the one Mucoromycotina (formerly Zygomycota) <em>R. oryzae</em> has very short constitutively spliced introns (61 nt) and RIs (54 nt). </p>                        <a id="4909572" scrollto-destination="4909572"></a>
<div id="DST038F2" data-id="DST038F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/m_dst03802.png?Expires=2147483647&Signature=EGcOqaeLxd4Gmf~gNxJPhMeB6zxO~7lOV8xiXGMVdA5YAYxHZPRXWlt8vZneZ~0ywMt2jmrMkaS-QHUtFmeHVmGXPQWKa-w8TwtkwBfrJNr1Q0oY4W~I24I2z6K6j917m8DaEtOM4NgcsdQchCEsj37E6YiqdOKx1kT30DLso25uJvi67aN1UQAXM2nUOEXHhm~o-5FEUQecG7yUeTUzxGL-r-GGGH29TqAwA4Rfgp6jHGJwXqc7~zZlOghl-~xcdawt1IY44oRj2pxM01MZV9b6JL4-smQOCuiOqaLP8OFE92ggcBirA0R~wwjQOTJiAQmhddCk~JWbjEEA1u9Vjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Species tree. This phylogenetic tree shows the evolutionary relationship between the analysed species, based on James et al.31 Percentages and bars next to the species represent the estimated AS rates per gene. AS rates for each strain are shown in case of species with more than one analysed strain. Species' lifestyles are colour coded: human pathogens, black; plant pathogens, dark gray; non-pathogenic fungi, light gray. Yeasts are underlined." path-from-xml="dst03802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4909572/dst03802.tiff" path-from-xml="dst03802.tiff" target="_blank">View large</a><a section="4909572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/dst03802.png?Expires=2147483647&Signature=NAxXUoY6wdi2v-y1o2sZMYgkUT8roSwRFIyXNrXl2ED0Kr7P8tLeheCVP4ygW-8HGBDmCjhMLMBlhTvSQhkdpI~UsbRMXLMtXiruxIw4CLV4tq-3v6S7mjxXAkmKlzjY86UZ9MBeRTATOiNjtv2ZH7XO0Np9b-zteuPbUiv5uc8cOzD3c~kBqTVIOLYFQPvWOZPc6E5q7sRcA3VUc7uvdks5LUQwNyQb2WKeZIavJ6raCW15GH7MJ5W0uCKGK2yXQcmASLCob3vK6TVjClvnxA2rgP7XG8mdWYNnaSRNmMuPWYFPHZt0S-OipvZRPOSrAcFW7dCT3v53Zd5iIqh6KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4909572&ar=347182&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Species tree. This phylogenetic tree shows the evolutionary relationship between the analysed species, based on James <em>et al</em>.<sup><a reveal-id="DST038C31" data-open="DST038C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Percentages and bars next to the species represent the estimated AS rates per gene. AS rates for each strain are shown in case of species with more than one analysed strain. Species' lifestyles are colour coded: human pathogens, black; plant pathogens, dark gray; non-pathogenic fungi, light gray. Yeasts are underlined.</p></div></div><div content-id="DST038F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/m_dst03802.png?Expires=2147483647&Signature=EGcOqaeLxd4Gmf~gNxJPhMeB6zxO~7lOV8xiXGMVdA5YAYxHZPRXWlt8vZneZ~0ywMt2jmrMkaS-QHUtFmeHVmGXPQWKa-w8TwtkwBfrJNr1Q0oY4W~I24I2z6K6j917m8DaEtOM4NgcsdQchCEsj37E6YiqdOKx1kT30DLso25uJvi67aN1UQAXM2nUOEXHhm~o-5FEUQecG7yUeTUzxGL-r-GGGH29TqAwA4Rfgp6jHGJwXqc7~zZlOghl-~xcdawt1IY44oRj2pxM01MZV9b6JL4-smQOCuiOqaLP8OFE92ggcBirA0R~wwjQOTJiAQmhddCk~JWbjEEA1u9Vjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Species tree. This phylogenetic tree shows the evolutionary relationship between the analysed species, based on James et al.31 Percentages and bars next to the species represent the estimated AS rates per gene. AS rates for each strain are shown in case of species with more than one analysed strain. Species' lifestyles are colour coded: human pathogens, black; plant pathogens, dark gray; non-pathogenic fungi, light gray. Yeasts are underlined." path-from-xml="dst03802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4909572/dst03802.tiff" path-from-xml="dst03802.tiff" target="_blank">View large</a><a section="4909572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/dst03802.png?Expires=2147483647&Signature=NAxXUoY6wdi2v-y1o2sZMYgkUT8roSwRFIyXNrXl2ED0Kr7P8tLeheCVP4ygW-8HGBDmCjhMLMBlhTvSQhkdpI~UsbRMXLMtXiruxIw4CLV4tq-3v6S7mjxXAkmKlzjY86UZ9MBeRTATOiNjtv2ZH7XO0Np9b-zteuPbUiv5uc8cOzD3c~kBqTVIOLYFQPvWOZPc6E5q7sRcA3VUc7uvdks5LUQwNyQb2WKeZIavJ6raCW15GH7MJ5W0uCKGK2yXQcmASLCob3vK6TVjClvnxA2rgP7XG8mdWYNnaSRNmMuPWYFPHZt0S-OipvZRPOSrAcFW7dCT3v53Zd5iIqh6KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4909572&ar=347182&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Species tree. This phylogenetic tree shows the evolutionary relationship between the analysed species, based on James <em>et al</em>.<sup><a reveal-id="DST038C31" data-open="DST038C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Percentages and bars next to the species represent the estimated AS rates per gene. AS rates for each strain are shown in case of species with more than one analysed strain. Species' lifestyles are colour coded: human pathogens, black; plant pathogens, dark gray; non-pathogenic fungi, light gray. Yeasts are underlined.</p></div></div><p>To further support the idea of the influence of gene properties on taxon-dependent AS frequencies, we compared the average conservation of SS motifs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3a and b</a>). Sequence conservation in terms of information content can be considered as a proxy for SS fidelity. We find that ascomycetous retained as well as constitutively spliced introns show higher SS conservation than the corresponding basidiomycetous ones (not significant, Mann–Whitney U-test, all <em>P</em> &gt; 0.08). The one Mucoromycotina, <em>R. oryzae</em>, has higher SS conservation in both types of introns than the Basidiomycota, yet cannot clearly be distinguished from Ascomycota in this respect. Yeasts show the highest SS conservation. However, the number of sampled yeasts and Basidiomycota are very small so that only 5′SSs of yeast RIs are significantly more highly conserved than 5′SSs of basidiomycetous RIs (<em>P</em> = 0.036, <em>n</em> = 2 yeasts (yeast <em>P. stipitis</em> contributes no RIs), 5 Basidiomycota).</p>                            <h3 scrollto-destination=4909578 id="4909578" class="section-title" data-legacy-id=s3g>3.7. Functional characterization of AS</h3>
<p>To study the function of fungal AS we analysed annotated and predicted Pfam domains for all genes and their relations to the AS rate of the gene families. We pooled all data and asked if particular Pfam domains are associated with higher AS rates. In a neutral model, AS is homogeneously distributed over all introns. Based on this model, we calculated the expected fraction of AS-associated genes per Pfam domain and compared it with the observed AS fraction. Together, six significantly AS-enriched Pfam gene families were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Two are ribosomal genes (PF01479, PF01599) and two are genes involved in thiamine biosynthesis (PF09084, PF01946). We note that these gene families show particularly high expression rates (on average, EST coverage is 34, compared with 0.6 for a non-AS-enriched control group). Since ESTs are the primary evidence for AS, and the detection rate of AS increases with EST coverage, it is well possible that the high expression rates alone account for the Pfam-AS association in these gene groups.</p><p>Apart from these, the other two significantly AS-enriched Pfam gene families are fungi specific (PF08520, PF12586) with unknown domain function. Remarkably, domain PF12586 occurs only in Cryptococcus. The next Pfam gene family with a known function, though below global significance (<em>P</em> = 0.35 with Bonferroni correction), is PF03073 and comprises integral membrane proteins that act as negative regulators of gene expression in response to oxygen or light (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p>                            <h3 scrollto-destination=4909584 id="4909584" class="section-title" data-legacy-id=s3h>3.8. AS is associated with dimorphic switch and pathogenicity</h3>
<p>Comparing AS rates of pathogenic and non-pathogenic fungi, we found interesting aspects: the rate in pathogenic species is higher (7.6%) than in non-pathogenic species (5.1%). Considering only human pathogens, the rate of 10.7% is even more striking, yet the differences are not significant (Mann–Whitney U-test, all <em>P</em>-values &gt;0.09, <em>n</em> = 11 non-pathogenic, 6 human pathogenic).</p><p>The Pfam domain descriptions of the AS affected genes pointed to an involvement in stress response to an altering environment as it occurs during host infection: heat shock proteins, chaperone/chaperonin. These proteins mediate stress response, for example thermo-tolerance in mammalian hosts.<sup><a reveal-id="DST038C32" data-open="DST038C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Furthermore, AS-affected genes are often related to availability of copper, which is typical when penetrating human host tissue: multi-copper oxidase and CTR copper transporter family. Glucuronoxylomannan, the predominant capsular polysaccharide in <em>C. neoformans</em>, experiences a structural change during dimorphic switching. Thus, the capsule surface changes, which results in a reduced recognition by the host's immune system.<sup><a reveal-id="DST038C33" data-open="DST038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We identified homologues of the proteins involved in the production and modification of glucuronoxylomannan for all investigated fungi via sequence similarity using BLASTP. Four of these proteins do show AS association, namely RIs, two in <em>C. neoformans</em> JEC21 and two in <em>C. neoformans</em> B-3501A. Three are hypothetical proteins harbouring a glycosyltransferase GTB or CAP59 mtransfer region. Two are annotated as mannosyltransferase 1 (<a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst038/-/DC1" target="_blank">Supple-mentary Table S6</a>). However, the predicted homologues are not significantly enriched in AS association (hypergeometric test, <em>P</em> &gt; 0.1).</p><p>Another virulence factor is the adaptation of a fungus to the altered environment of the host tissue. Up-regulation of oxidative and heat shock stress associated genes as <em>tps1</em>, <em>hsp30</em> and <em>ddr48</em> in <em>P. brasiliensis</em> P01 likely convey to cope with this micro-niche climate.<sup><a reveal-id="DST038C34" data-open="DST038C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The identified homologues of these 3 genes are frequently affected by AS in pathogenic fungi (15 cases) and 5 times in non-pathogenic fungi (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Among the Tps1 homologues are genes from <em>C. neoformans</em> B-3501A and JEC21, one of <em>P. anserina</em> and one of <em>T. reesei</em>, an alpha,alpha-trehalose-phosphate synthase Tps1 subunit of <em>L. bicolor</em> and a hypothetical protein similar to alpha,alpha-trehalose-phosphate synthase subunit TPS3 of <em>N. crassa</em>. Some of the genes are affected by multiple AS events. The AS-associated Hsp30 homologues are three chaperones/small heat shock proteins from <em>C. immitis</em>, <em>L. bicolor</em> and <em>U. maydis</em>. Finally, Ddr48-homologues with AS association are hypothetical/predicted proteins of <em>C. immitis</em>, <em>A. capsulatus</em>, <em>C. neoformans</em> and <em>M. graminicola</em>, two of which have a predicted function: ‘similar to potential stress response protein’, and ‘Glycosyltrans-ferase GTB type’. These stress response-related proteins are significantly enriched in AS association (hypergeometric test, <em>P</em> = 0.00022).</p>                            <h2 scrollto-destination=4909590 id="4909590" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=4909593 id="4909593" class="section-title" data-legacy-id=s4a>4.1. AS rate estimation</h3>
<p>We here presented a comparative genome-wide survey of AS in the fungal kingdom. We based our survey mainly on Sanger-sequenced EST data (one species' ESTs are from 454 sequencing) and corresponding annotated genomes. In current AS studies, often next generation transcriptome sequence data with millions of short ESTs are used. However, though for some fungi, these data are available, the other prerequisite of having a well-annotated genome is rarely fulfilled. According to our results, next generation sequencing technologies with EST lengths of &gt;200 nt (met by Roche 454 as well as Illumina/Solexa platforms) should be well feasible for the detection of the basic AS types in fungi. This is because the read length is clearly longer than the average fungal intron and exon lengths (constitutively spliced introns 93 nt and exons 132 nt).</p><p>The alignment of transcript sequences to genomes is currently the most effective way to detect alterations of mature mRNA at a large scale. However, Fox-Walsh and Hertel<sup><a reveal-id="DST038C9" data-open="DST038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> argued that every multi-exon gene has a certain AS frequency, and the detection of an alternative isoform is a matter of sensitivity of the method applied. Thus, we here use a random sampling approach similar to the one by Kim <em>et al</em>.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> This universal normalization approach led to AS rate estimates that are independent of the number and distribution of the ESTs, and thus, are more comparable across species. We found AS events in every of the 27 studied fungi except one (<em>P. stipitis</em>), with an average rate of 6.4% of genes. Thus, we suppose that AS is a common phenomenon in the fungal kingdom. <em>Coccidioides imitis</em> and <em>C. neoformans</em> show outstanding AS rates of 13 and 18%, the latter being about three times more than anticipated in earlier studies.<sup><a reveal-id="DST038C17" data-open="DST038C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST038C20" data-open="DST038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> While successively increasing the sampling depth from 2–10, we found that the relative proportions of the AS rates between most species remain constant (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). This underpins the reliability of our normalization method. Because many loci have a lower EST coverage than the sampling depth of 10, our analysis yielded rather conservative estimates. It is likely that with deep transcriptome sequencing more fungal AS events will be found. Even when excluding very rare events, this may elevate the AS rates. This trend was seen for human and other mammals already,<sup><a reveal-id="DST038C8" data-open="DST038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and can be supported by the finding that for <em>A. benhamiae</em> and <em>N. crassa</em>, both of which have high EST coverage, the AS rates clearly kept rising at higher sampling depths (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), opposed to most of the other species.</p><p>Finally, in a recent study on fission yeasts, 433 AS events in overall 5144 genes were found in <em>S. pombe</em>.<sup><a reveal-id="DST038C35" data-open="DST038C35" class="link link-ref link-reveal xref-bibr">35</a></sup> While considering scaling effects due to sequencing depth, our results agree well with these findings in that the AS rate is very low compared with that in non-yeast Ascomycota (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Calculation S1</a>). This validates the comparability of our normalized AS rate results.</p>                            <h3 scrollto-destination=4909601 id="4909601" class="section-title" data-legacy-id=s4b>4.2. Fungal introns have an innate propensity to be retained</h3>
<p>We found that the trend of relative AS type distribution was the same in all the investigated fungal species. Intron retention made up the most prevalent of the investigated types (61% of the events). Contrarily, skipped exons were very rare (3%) and alternative 3′ (23%) and 5′ SSs (13%) comprised a third of the events. These results are in general agreement with previous findings on fungal AS<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and are similar to trends in plants.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In contrast, skipped exons are more common than RIs in invertebrates and even more frequent in vertebrates.<sup><a reveal-id="DST038C7" data-open="DST038C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>The more introns a species genome harbours the more splicing needs to take place. The question is whether this also increases the chance to have alternatively spliced introns <em>per se</em>. Indeed, we found a strong correlation of genome-wide intron numbers and numbers of RIs. Thus, fungal introns seem to have an innate chance to be alternatively spliced. Similarly, Irimia <em>et al</em>.<sup><a reveal-id="DST038C17" data-open="DST038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> found a correspondence between AS and intron number per gene in 12 eukaryotes.</p><p>We found that fungal RIs are shorter than constitutively spliced introns. Also, on the species level, there is a correspondence between intron lengths and their propensity to be alternatively spliced. Together, this hints at an involvement of the intron length in the recognition of introns. The intron definition mechanism is a model proposed to explain this same effect in plants. Splicing factors bind to the recognition sites on the RNA, and ‘bridge’ across the intron by mutual binding. Thus, failed recognition of one SS typically results in intron retention.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> This is in contrast to metazoan splicing, where splicing factors are assumed to form stable complexes across exons (exon definition mechanism) and where failed SS recognition typically results in exon skipping. It explains why metazoan introns tend to be much longer (e.g. 3413 nt in human<sup><a reveal-id="DST038C36" data-open="DST038C36" class="link link-ref link-reveal xref-bibr">36</a></sup>) but are rarely retained. Thus, we propose that the intron definition mechanism is prevalent in fungi similar to plants.</p><p>Finally, there is a hypothesis that connects SS conservation with splicing propensity, saying that strict adherence to the SS motif promotes the splicing machinery to bind more reliably to the SS and thus decreases the chance of AS.<sup><a reveal-id="DST038C37" data-open="DST038C37" class="link link-ref link-reveal xref-bibr">37</a></sup> McGuire <em>et al</em>.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> find weaker (i.e., less conserved) SSs at RIs compared with constitutively spliced introns in all their investigated species. Here, when comparing introns (both retained and normally spliced ones) between the taxa, we find that higher SS conservation correlates with lower AS rates, which supports the hypothesis.</p>                            <h3 scrollto-destination=4909611 id="4909611" class="section-title" data-legacy-id=s4c>4.3. Fungal RIs are authentic and likely trigger nonsense-mediated mRNA decay</h3>
<p>There is a debate if RIs are authentic AS events or represent incompletely spliced pre-mRNA. Contamination with genomic DNA is very unlikely since the construction of EST libraries relies on affinity-based poly(A)+ mRNA enrichment. From the analysis of fungal RIs, we found no tendency to preserve the reading frame, similar to results on non-fungal species in a previous study.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> This may support the hypothesis of spurious intron retention. However, we have several arguments against it. For the majority of EST libraries analysed here, cDNA was produced by poly(A)-tail capture, ensuring that ESTs derive from fully transcribed mRNAs. The current consensus is that intron splicing occurs predominantly cotranscriptionally,<sup><a reveal-id="DST038C38" data-open="DST038C38" class="link link-ref link-reveal xref-bibr">38</a></sup> corroborated by findings that the nascent mRNA can recruit multiple spliceosomes simultaneously.<sup><a reveal-id="DST038C39" data-open="DST038C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Though the exact kinetics of RNA processing and export are unknwon, intron splicing is likely finished shortly after transcription. This supports the hypothesis that if a detected multi-intron mRNA was spliced at one intron, it has already been spliced at the other introns, too. In fact, averaged over all species, 96% of the transcript isoforms that support an RI contain a processed intron at another position, as was similarly reported for RIs in <em>Arabidopsis thaliana</em>.<sup><a reveal-id="DST038C40" data-open="DST038C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In these cases, the completed splicing of co-transcribed introns indicates that the molecules have passed spliceosomal processing and that RIs likely represent authentic events on mRNA. However, it is possible that RI-containing mRNAs had not left the nucleus, awaiting a later processing cycle or degradation. Nevertheless, even if this is true, these cases illustrate inherent differences in splicing efficiency.</p><p>It was argued that despite a weak selection for coding potential, splice variants having RIs unlikely yield functional proteins.<sup><a reveal-id="DST038C15" data-open="DST038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> While we suppose that most RIs are authentic AS events, the isoforms with a frame-shifting RI unlikely yield productive, protein-coding mRNAs. However, we hypothesize that fungal RIs may in part be a means for post-transcriptional regulation via nonsense-mediated mRNA decay (NMD), in which transcripts containing premature termination codons (PTCs) are degraded.<sup><a reveal-id="DST038C41" data-open="DST038C41" class="link link-ref link-reveal xref-bibr">41</a></sup> This is because RI sequences with frame shifts probably introduce PTCs (15 randomly drawn triplets pose a chance of &gt;50% to contain a stop codon). Most of the NMD-related components<sup><a reveal-id="DST038C41" data-open="DST038C41" class="link link-ref link-reveal xref-bibr">41</a></sup> are conserved in most of the fungi present in NCBI's HomoloGene database (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). <em>Saccharomyces cerevisiae</em> has an NMD machinery which is, however, not essential. Most RIs of the yeast <em>Yarrowia lipolytica</em>, contain PTCs and there is evidence that corresponding RNA is degraded by NMD.<sup><a reveal-id="DST038C42" data-open="DST038C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Finally, first evidence for functional NMD were found in <em>N. crassa</em>.<sup><a reveal-id="DST038C43" data-open="DST038C43" class="link link-ref link-reveal xref-bibr">43</a></sup> As long as experimental data for a functional relevance of RIs are missing, we note that RIs qualify as mediators for a splicing-dependent mechanism of gene expression regulation, based on structure as well as on statistical association with functional categories (see below).</p>                            <h3 scrollto-destination=4909616 id="4909616" class="section-title" data-legacy-id=s4d>4.4. Does AS facilitate multi-cellular complexity?</h3>
<p>The complexity of the (multi-)cellular structure has long since been an important feature to classify fungi into sub-taxa.<sup><a reveal-id="DST038C44" data-open="DST038C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Typical instances of diverse complexity are, being yeast or mold, and characteristics of sexual structures. There are predominantly single-celled yeasts, namely <em>S. pombe</em>, <em>S. cerevisiae</em> and <em>P. stipitis</em>, within the phylum of the Ascomycota, whose most complex yeast form is a four-spore ascus. The Mucoromycotina <em>R. oryzae</em> forms simple zygospores during sexual reproduction, but differentiated multi-cellular sporangia for asexual reproduction. Filamentous Ascomycetes produce more complex thalli, as, e.g. ascocarps (apothecium, cleistothecium, perithecium). Finally, Basidio-mycota, probably the most recent ‘crown group’ of fungi, develop complex fruiting bodies.<sup><a reveal-id="DST038C44" data-open="DST038C44" class="link link-ref link-reveal xref-bibr">44</a></sup> We here find that the average AS rate of the mentioned taxa correlates with this order of complexity: <em>Saccharomycotina</em> and <em>Taphrinomycotina</em> (0.26% per-gene AS rate), Mucoro-mycotina (2.3%), Pezizomycotina (7.2%, Ascomycota excluding yeasts) and Basidiomycota (8.6%). We speculate that AS contributes to multi-cellular complexity of the fungi.</p><p>We find that the fungi with the smallest genomes show nearly no AS. These are the ascomycetous yeasts <em>S. cerevisiae</em>, <em>S. pombe</em> and <em>P. stipitis</em>. This is consistent with an earlier study on <em>S. cerevisiae</em> and <em>S. pombe</em>.<sup><a reveal-id="DST038C17" data-open="DST038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> A major reason for this is probably the reduced proportion of intron-containing genes, e.g. 5% of <em>S. cerevisiae</em> genes vs. 86% in <em>C. immitis</em>, since Hemiascomycetes (Saccharomycetes) experienced intron loss during the course of evolution.<sup><a reveal-id="DST038C45" data-open="DST038C45" class="link link-ref link-reveal xref-bibr">45</a></sup> However, from a certain genome size on, neither the AS rate nor the absolute AS number show any correlation. And, to an extreme, <em>C. neoformans</em> has only ca. 6600 genes but the highest found AS rate (18%).</p><p>The composition of the splicing machinery can give another perspective in understanding the differences in AS capability. The core components of the spliceosome, i.e. the five snRNPs and essential dynamic factors like Prp8 or Slu7, are generally conserved in eukaryotes. However, the small subunit of U2AF (U2AF35 in human), involved in recognition of the 3′SS, is absent in <em>S. cerevisiae</em>. The family of serine/arginine-rich (SR) proteins comprise many known splicing regulators, and it was proposed that a higher SR protein diversity increases the AS complexity.<sup><a reveal-id="DST038C46" data-open="DST038C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Our results do support this hypothesis: among yeasts, which have the lowest AS rates, <em>S. cerevisiae</em> has no SR proteins, only an SR-like homologue Npl3,<sup><a reveal-id="DST038C47" data-open="DST038C47" class="link link-ref link-reveal xref-bibr">47</a></sup> and <em>S. pombe</em> has only two SR proteins.<sup><a reveal-id="DST038C48" data-open="DST038C48" class="link link-ref link-reveal xref-bibr">48</a></sup> On the other hand, many of the other species of our study were found to have many SR and SR-related proteins,<sup><a reveal-id="DST038C49" data-open="DST038C49" class="link link-ref link-reveal xref-bibr">49</a></sup> in accordance with their higher AS rates.</p><p>We used Pfam domain annotations to analyse the possible functional associations of AS. The most significantly AS-enriched Pfam-coding gene families are ribosomal or do function in thiamine biosynthesis. However, these findings should be taken with caution since the expression level (i.e. EST coverage) is ∼50-fold higher than average. Other AS-enriched gene families with moderate gene expression levels do code for fungi-specific protein domains of unknown function. This may indicate that AS is associated with enhanced evolutionary dynamics in these gene families, consistent with a supportive role of AS in gene evolution.<sup><a reveal-id="DST038C50" data-open="DST038C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p><p>Taking together the relatively low fraction of AS-associated gene families and the gene expression bias among the few candidates, we conclude that a homogenous distribution model is currently a sufficient explanation for the occurrence of AS among the EST-covered genes. However, we anticipate that increasing EST sequencing depths, and a saturation of a major fraction of genes, will allow more detailed insights into the functional association of AS in fungi.</p><p>Both, the elevated AS rates and the greater amount of splicing regulators of the more complex fungi suggest the hypothesis that AS may facilitate multi-cellular complexity. Furthermore, we found that AS is involved in another elaborate trait of certain fungi, namely virulence.</p>                            <h3 scrollto-destination=4909629 id="4909629" class="section-title" data-legacy-id=s4e>4.5. AS likely regulates virulence of pathogenic fungi</h3>
<p>A first hint of AS involvement in pathogenicity was given by mere comparison of average AS rates. Human pathogenic fungi show on average a twice as high AS rate (10.7%) than non-pathogenic fungi (5.1%, neither plant nor human pathogenic). This is corroborated by the keywords of AS-associated Pfam domains which indicate enrichment for stress response functions. Moreover, a direct search for homologues of <em>P. brasiliensis</em>' <em>tps1</em>, <em>hsp30</em> and <em>ddr48</em> genes that convey cell rescue of this fungus while facing oxidative and heat shock stress in the human body,<sup><a reveal-id="DST038C34" data-open="DST038C34" class="link link-ref link-reveal xref-bibr">34</a></sup> yielded many AS-associated genes in human and plant pathogenic fungi. Hence, it is likely that AS is involved in gene expression regulation during the adaptation to the environmental conditions in the host.</p><p>The dimorphic switch is another virulence factor, and a key of persistent virulence.<sup><a reveal-id="DST038C33" data-open="DST038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> During host penetration, a fungus can either switch to filamentous growth (e.g. <em>C. albicans</em>, <em>A. fumigatus</em>), or switch from filamentous to uni-cellular growth (e.g. <em>P. brasiliensis</em> Pb01, <em>C. immitis</em>). The dimorphic switch is only poorly understood. However, several contributing compounds have been identified. <em>Cryptococcus neoformans</em>' glucuronoxylomannan (GMX), a capsular polysaccharide, is crucial for switching, as it alters the capsule surface. This increases the resistance against host immune system by hampering antibody and complement mediated phagocytosis.<sup><a reveal-id="DST038C33" data-open="DST038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We found two homologues of GMX production and modification proteins in <em>C. neoformans</em> (in B-3501A and JEC21), each containing an RI. Of the 19 predicted homologues of <em>tps1</em>, <em>hsp30</em>, <em>ddr48</em> and GMX-related genes, 5 have AS association in four non-pathogenic fungi (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S6 and S7</a>).</p><p>An association of AS with pathogenicity has been found in former studies already. The <em>UrRm75</em> gene in <em>U. maydis</em>, involved in dimorphism and virulence, contains four introns and has an alternative 3′SS.<sup><a reveal-id="DST038C51" data-open="DST038C51" class="link link-ref link-reveal xref-bibr">51</a></sup> A putative heat shock protein and a putative alpha, alpha-trehalose-phosphate synthase (both stress response-associated) were predicted to be affected by AS in <em>C. neoformans</em>.<sup><a reveal-id="DST038C20" data-open="DST038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Transcripts of cryptococcal intersectin 1 undergo AS and its disruption affects the production of several virulence factors in <em>C. neoformans</em>.<sup><a reveal-id="DST038C52" data-open="DST038C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In many fungi, Ste12-like transcription factors play essential roles in invasive growth and pseudohyphal development, and their gene transcripts are affected by AS within a conserved exon–intron structure.<sup><a reveal-id="DST038C53" data-open="DST038C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Summar-izing, gene regulation via AS likely facilitates virulence of pathogenic fungi on various levels.</p>                            <h2 scrollto-destination=4909637 id="4909637" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was funded by the Friedrich-Schiller-University Jena and the Jena School for Microbial Communication (JSMC).</p>                            <h2 scrollto-destination=4909640 id="4909640" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Matthias Platzer from the Fritz Lipmann Institute (Jena), Ina Weiß from the chair of bioinformatics Jena, Michael Hiller from MPI of Molecular Cell Biology and Genetics (Dresden) and Kerstin Hoffmann from the Friedrich-Schiller-University Jena, Germany for helpful discussions. Further, we thank Igor Grigoriev from the Department of Energy's Joint Genome Institute and Lucia Alvarado-Balderrama from the Broad Institute of MIT and Harvard for the permission to use and publish data of these institutes.</p>                            <h2 scrollto-destination=4909645 id="4909645" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST038C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ben-Ari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rafalska</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Function of alternative splicing</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">344</div> (pg. <div class="fpage">1</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2004.10.022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2004.10.022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Function%20of%20alternative%20splicing&amp;author=S.%20Stamm&amp;author=S.%20Ben-Ari&amp;author=I.%20Rafalska&amp;publication_year=2005&amp;journal=Gene&amp;volume=344&amp;pages=1-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.10.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.10.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.10.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15656968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Black</div> <div class="given-names">D.L.</div></div>. </div><div class="article-title">Mechanisms of alternative pre-messenger RNA splicing</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="year">2003</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">291</div>-<div class="lpage">336</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.biochem.72.121801.161720" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.biochem.72.121801.161720</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20alternative%20pre-messenger%20RNA%20splicing&amp;author=D.L.%20Black&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=72&amp;pages=291-336" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.72.121801.161720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.72.121801.161720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.72.121801.161720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12626338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dean</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Isacoff</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Alternative splicing of neuroligin regulates the rate of presynaptic differentiation</div>, <div class="source">J. Neurosci.</div> , <div class="year">2010</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">11435</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1523/JNEUROSCI.2946-10.2010" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1523/JNEUROSCI.2946-10.2010</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20neuroligin%20regulates%20the%20rate%20of%20presynaptic%20differentiation&amp;author=H.%20Lee&amp;author=C.%20Dean&amp;author=E.%20Isacoff&amp;publication_year=2010&amp;journal=J.%20Neurosci.&amp;volume=30&amp;pages=11435-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1523/JNEUROSCI.2946-10.2010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1523%2FJNEUROSCI.2946-10.2010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1523%2FJNEUROSCI.2946-10.2010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20739565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Regulation of alternative splicing by reversible protein phosphorylation</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2008</div>, vol. <div class="volume">283</div> (pg. <div class="fpage">1223</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.R700034200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.R700034200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20alternative%20splicing%20by%20reversible%20protein%20phosphorylation&amp;author=S.%20Stamm&amp;publication_year=2008&amp;journal=J.%20Biol.%20Chem.&amp;volume=283&amp;pages=1223-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.R700034200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.R700034200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.R700034200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18024427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kemper</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tol</div> <div class="given-names">M.J.P.M.</div></div>,  <div class="name"><div class="surname">Medema</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Mouse tissues express multiple splice variants of prominin-1</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e12325</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0012325" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0012325</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20tissues%20express%20multiple%20splice%20variants%20of%20prominin-1&amp;author=K.%20Kemper&amp;author=M.J.P.M.%20Tol&amp;author=J.P.%20Medema&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e12325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0012325" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0012325" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0012325"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Alekseyenko</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Roy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">ASAP II database: analysis and comparative genomics of alternative splicing in 15 animal species</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D93</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl884" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl884</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ASAP%20II%20database%3A%20analysis%20and%20comparative%20genomics%20of%20alternative%20splicing%20in%2015%20animal%20species&amp;author=N.%20Kim&amp;author=A.V.%20Alekseyenko&amp;author=M.%20Roy&amp;author=C.%20Lee&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D93-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl884" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl884" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl884"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17108355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Magen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ast</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Different levels of alternative splicing among eukaryotes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">125</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl924" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl924</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20levels%20of%20alternative%20splicing%20among%20eukaryotes&amp;author=E.%20Kim&amp;author=A.%20Magen&amp;author=G.%20Ast&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl924" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl924" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl924"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Shai</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Frey</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1413</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.259" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.259</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.J.%20Lee&amp;author=B.J.%20Frey&amp;author=B.J.%20Blencowe&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=1413-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fox-Walsh</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Hertel</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">Splice-site pairing is an intrinsically high fidelity process</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1766</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0813128106" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0813128106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splice-site%20pairing%20is%20an%20intrinsically%20high%20fidelity%20process&amp;author=K.L.%20Fox-Walsh&amp;author=K.J.%20Hertel&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=1766-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0813128106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0813128106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0813128106"> </span></div> </div></div></div></div></div><div content-id="DST038C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Alternative splicing: new insights from global analyses</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">37</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2006.06.023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2006.06.023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20new%20insights%20from%20global%20analyses&amp;author=B.J.%20Blencowe&amp;publication_year=2006&amp;journal=Cell&amp;volume=126&amp;pages=37-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.06.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.06.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.06.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16839875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiller</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Huse</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Szafranski</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Rosenstiel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Schreiber</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Backofen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Platzer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Phylogenetically widespread alternative splicing at unusual GYNGYN donors</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R65</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2006-7-7-r65" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2006-7-7-r65</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetically%20widespread%20alternative%20splicing%20at%20unusual%20GYNGYN%20donors&amp;author=M.%20Hiller&amp;author=K.%20Huse&amp;author=K.%20Szafranski&amp;author=P.%20Rosenstiel&amp;author=S.%20Schreiber&amp;author=R.%20Backofen&amp;author=M.%20Platzer&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-7-r65" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-7-r65" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-7-r65"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16869967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiller</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Huse</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Szafranski</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Widespread occurrence of alternative splicing at NAGNAG acceptors contributes to proteome plasticity</div>, <div class="source">Nat. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">1255</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng1469" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng1469</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20alternative%20splicing%20at%20NAGNAG%20acceptors%20contributes%20to%20proteome%20plasticity&amp;author=M.%20Hiller&amp;author=K.%20Huse&amp;author=K.%20Szafranski&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=1255-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15516930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sinha</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Lenser</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Jahn</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Tassdb2—a comprehensive database of subtle alternative splicing events</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">216</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-11-216" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-11-216</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tassdb2%E2%80%94a%20comprehensive%20database%20of%20subtle%20alternative%20splicing%20events&amp;author=R.%20Sinha&amp;author=T.%20Lenser&amp;author=N.%20Jahn&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20429909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakabe</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">de Souza</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Sequence features responsible for intron retention in human</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">59</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-8-59" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-8-59</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20features%20responsible%20for%20intron%20retention%20in%20human&amp;author=N.J.%20Sakabe&amp;author=S.J.%20de%20Souza&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17324281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McGuire</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Pearson</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Neafsey</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Galagan</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Cross-kingdom patterns of alternative splicing and splice recognition</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R50</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2008-9-3-r50" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2008-9-3-r50</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-kingdom%20patterns%20of%20alternative%20splicing%20and%20splice%20recognition&amp;author=A.M.%20McGuire&amp;author=M.D.%20Pearson&amp;author=D.E.%20Neafsey&amp;author=J.E.%20Galagan&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-3-r50" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-3-r50" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-3-r50"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18321378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kupfer</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Drabenstot</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Buchanan</div> <div class="given-names">K.L.</div></div>, et al. </div><div class="article-title">Introns and splicing elements of five diverse fungi</div>, <div class="source">Eukaryot. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1088</div>-<div class="lpage">100</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/EC.3.5.1088-1100.2004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/EC.3.5.1088-1100.2004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20and%20splicing%20elements%20of%20five%20diverse%20fungi&amp;author=D.M.%20Kupfer&amp;author=S.D.%20Drabenstot&amp;author=K.L.%20Buchanan&amp;publication_year=2004&amp;journal=Eukaryot.%20Cell&amp;volume=3&amp;pages=1088-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.3.5.1088-1100.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.3.5.1088-1100.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.3.5.1088-1100.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15470237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irimia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rukov</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Penny</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Roy</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">Functional and evolutionary analysis of alternatively spliced genes is consistent with an early eukaryotic origin of alternative splicing</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> pg. <div class="fpage">188</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-7-188" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2148-7-188</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20evolutionary%20analysis%20of%20alternatively%20spliced%20genes%20is%20consistent%20with%20an%20early%20eukaryotic%20origin%20of%20alternative%20splicing&amp;author=M.%20Irimia&amp;author=J.L.%20Rukov&amp;author=D.%20Penny&amp;author=S.W.%20Roy&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17916237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ebbole</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Thon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bhattarai</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dean</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Gene discovery and gene expression in the rice blast fungus, <em>Magnaporthe grisea</em>: analysis of expressed sequence tags</div>, <div class="source">Mol. Plant Microbe. Interact.</div> , <div class="year">2004</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1337</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1094/MPMI.2004.17.12.1337" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1094/MPMI.2004.17.12.1337</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20gene%20expression%20in%20the%20rice%20blast%20fungus%2C%20Magnaporthe%20grisea%3A%20analysis%20of%20expressed%20sequence%20tags&amp;author=D.J.%20Ebbole&amp;author=Y.%20Jin&amp;author=M.%20Thon&amp;author=H.%20Pan&amp;author=E.%20Bhattarai&amp;author=T.%20Thomas&amp;author=R.%20Dean&amp;publication_year=2004&amp;journal=Mol.%20Plant%20Microbe.%20Interact.&amp;volume=17&amp;pages=1337-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2004.17.12.1337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2004.17.12.1337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2004.17.12.1337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15597739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Survey of the transcriptome of <em>Aspergillus oryzae</em> via massively parallel mRNA sequencing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">5075</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkq256" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkq256</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20of%20the%20transcriptome%20of%20Aspergillus%20oryzae%20via%20massively%20parallel%20mRNA%20sequencing&amp;author=B.%20Wang&amp;author=G.%20Guo&amp;author=C.%20Wang&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=5075-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq256"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20392818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loftus</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Fung</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Roncaglia</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The genome of the basidiomycetous yeast and human pathogen <em>Cryptococcus neoformans</em></div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1321</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1103773" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1103773</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20basidiomycetous%20yeast%20and%20human%20pathogen%20Cryptococcus%20neoformans&amp;author=B.J.%20Loftus&amp;author=E.%20Fung&amp;author=P.%20Roncaglia&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1321-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1103773" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1103773" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1103773"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15653466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ho</div> <div class="given-names">E.C.H.</div></div>,  <div class="name"><div class="surname">Cahill</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Saville</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Gene discovery and transcript analyses in the corn smut pathogen <em>Ustilago maydis</em>: expressed sequence tag and genome sequence comparison</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">334</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-8-334" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-8-334</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20and%20transcript%20analyses%20in%20the%20corn%20smut%20pathogen%20Ustilago%20maydis%3A%20expressed%20sequence%20tag%20and%20genome%20sequence%20comparison&amp;author=E.C.H.%20Ho&amp;author=M.J.%20Cahill&amp;author=B.J.%20Saville&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=334" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-334" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-334" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-334"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17892571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="collab">NCBI (National Center of Biotechnology Information)</div>, <div class="year">2011</div> <div class="comment"><a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a> (18 January 2012, date last accessed)</div></div></div></div></div><div content-id="DST038C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="collab">DOE Joint Genome Institute</div>, <div class="source">A DOE Office of Science User Facility of Lawrence Berkeley National Laboratory</div> , <div class="year">2011</div> <div class="comment"><a class="link link-uri" href="http://www.jgi.doe.gov/">http://www.jgi.doe.gov/</a> (1 February 2012, date last accessed)</div></div></div></div></div><div content-id="DST038C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dwight</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Balakrishnan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Christie</div> <div class="given-names">K.R.</div></div>, et al. </div><div class="article-title">Saccharomyces genome database: underlying principles and organisation</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">9</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bib/5.1.9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bib/5.1.9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20genome%20database%3A%20underlying%20principles%20and%20organisation&amp;author=S.S.%20Dwight&amp;author=R.%20Balakrishnan&amp;author=K.R.%20Christie&amp;publication_year=2004&amp;journal=Brief%20Bioinform.&amp;volume=5&amp;pages=9-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/5.1.9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F5.1.9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2F5.1.9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation">, <div class="year">2011</div> <div class="comment">Broad Institute of MIT and Harvard <em>Fusarium oxysporum</em>, <em>Paracoccidioides brasiliensis</em>, and <em>Rhizopus oryzae</em> Sequencing Projects <a class="link link-uri" href="http://www.broadinstitute.org">http://www.broadinstitute.org</a> (2 February 2012, date last accessed)</div></div></div></div></div><div content-id="DST038C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT–the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Improved spliced alignment from an information theoretic approach</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">13</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti748" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti748</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20spliced%20alignment%20from%20an%20information%20theoretic%20approach&amp;author=M.%20Zhang&amp;author=W.%20Gish&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=13-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti748"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16267086" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneider</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Sequence logos: a new way to display consensus sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6097</div>-<div class="lpage">100</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/18.20.6097" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/18.20.6097</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;author=T.D.%20Schneider&amp;author=R.M.%20Stephens&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6097-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.20.6097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.20.6097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2172928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="collab">R Development Core Team</div>, <div class="source">R: A Language and Environment for Statistical Computing</div> , <div class="year">2010</div><div class="publisher-loc">Vienna, Austria</div><div class="publisher-name">R Foundation for Statistical Computing</div> <div class="comment">ISBN 3-900051-07-0</div></div></div></div></div><div content-id="DST038C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">A probabilistic model of local sequence alignment that simplifies statistical significance estimation</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000069</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pcbi.1000069" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pcbi.1000069</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20probabilistic%20model%20of%20local%20sequence%20alignment%20that%20simplifies%20statistical%20significance%20estimation&amp;author=S.R.%20Eddy&amp;publication_year=2008&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=4&amp;pages=e1000069" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516236" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">James</div> <div class="given-names">T.Y.</div></div>,  <div class="name"><div class="surname">Kauff</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Schoch</div> <div class="given-names">C.L.</div></div>, et al. </div><div class="article-title">Reconstructing the early evolution of fungi using a six-gene phylogeny</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">443</div> (pg. <div class="fpage">818</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature05110" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature05110</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20the%20early%20evolution%20of%20fungi%20using%20a%20six-gene%20phylogeny&amp;author=T.Y.%20James&amp;author=F.%20Kauff&amp;author=C.L.%20Schoch&amp;publication_year=2006&amp;journal=Nature&amp;volume=443&amp;pages=818-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17051209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abad</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Fernández-Molina</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Bikandi</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">What makes <em>Aspergillus fumigatus</em> a successful pathogen? Genes and molecules involved in invasive aspergillosis</div>, <div class="source">Rev. Iberoam Micol.</div> , <div class="year">2010</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">155</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.riam.2010.10.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.riam.2010.10.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20makes%20Aspergillus%20fumigatus%20a%20successful%20pathogen%3F%20Genes%20and%20molecules%20involved%20in%20invasive%20aspergillosis&amp;author=A.%20Abad&amp;author=J.V.%20Fern%C3%A1ndez-Molina&amp;author=J.%20Bikandi&amp;publication_year=2010&amp;journal=Rev.%20Iberoam%20Micol.&amp;volume=27&amp;pages=155-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.riam.2010.10.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.riam.2010.10.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.riam.2010.10.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20974273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fries</div> <div class="given-names">B.C.</div></div>. </div><div class="article-title">Phenotypic switching of <em>Cryptococcus neoformans</em> and <em>Cryptococcus gattii</em></div>, <div class="source">Mycopathologia</div> , <div class="year">2008</div>, vol. <div class="volume">166</div> (pg. <div class="fpage">181</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11046-008-9137-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11046-008-9137-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20switching%20of%20Cryptococcus%20neoformans%20and%20Cryptococcus%20gattii&amp;author=N.%20Jain&amp;author=B.C.%20Fries&amp;publication_year=2008&amp;journal=Mycopathologia&amp;volume=166&amp;pages=181-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11046-008-9137-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11046-008-9137-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11046-008-9137-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18568418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borges</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Bailão</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Báo</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Pereira</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Parente</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">de Almeida Soares</div> <div class="given-names">C.M.</div></div>. </div><div class="article-title">Genes potentially relevant in the parasitic phase of the fungal pathogen <em>Paracoccidioides brasiliensis</em></div>, <div class="source">Mycopathologia</div> , <div class="year">2011</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">1</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11046-010-9349-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11046-010-9349-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20potentially%20relevant%20in%20the%20parasitic%20phase%20of%20the%20fungal%20pathogen%20Paracoccidioides%20brasiliensis&amp;author=C.L.%20Borges&amp;author=A.M.%20Bail%C3%A3o&amp;author=S.N.%20B%C3%A1o&amp;author=M.%20Pereira&amp;author=J.A.%20Parente&amp;author=C.M.%20de%20Almeida%20Soares&amp;publication_year=2011&amp;journal=Mycopathologia&amp;volume=171&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11046-010-9349-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11046-010-9349-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11046-010-9349-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20669049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rhind</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Comparative functional genomics of the fission yeasts</div>, <div class="source">Science</div> , <div class="year">2011</div>, vol. <div class="volume">332</div> (pg. <div class="fpage">930</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1203357" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1203357</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20functional%20genomics%20of%20the%20fission%20yeasts&amp;author=N.%20Rhind&amp;author=Z.%20Chen&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Science&amp;volume=332&amp;pages=930-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1203357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1203357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1203357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deutsch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Intron–exon structures of eukaryotic model organisms</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">3219</div>-<div class="lpage">28</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/27.15.3219" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/27.15.3219</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%E2%80%93exon%20structures%20of%20eukaryotic%20model%20organisms&amp;author=M.%20Deutsch&amp;author=M.%20Long&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=3219-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.15.3219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.15.3219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.15.3219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10454621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Goren</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ast</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Alternative splicing: current perspectives</div>, <div class="source">Bioessays</div> , <div class="year">2008</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">38</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/bies.20692" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/bies.20692</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20current%20perspectives&amp;author=E.%20Kim&amp;author=A.%20Goren&amp;author=G.%20Ast&amp;publication_year=2008&amp;journal=Bioessays&amp;volume=30&amp;pages=38-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18081010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dujardin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lafaille</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Petrillo</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Transcriptional elongation and alternative splicing</div>, <div class="source">Biochim Biophys Acta</div> , <div class="year">2013</div>, vol. <div class="volume">1829</div> (pg. <div class="fpage">134</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20elongation%20and%20alternative%20splicing&amp;author=G.%20Dujardin&amp;author=C.%20Lafaille&amp;author=E.%20Petrillo&amp;publication_year=2013&amp;journal=Biochim%20Biophys%20Acta&amp;volume=1829&amp;pages=134-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagrm.2012.08.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2012.08.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagrm.2012.08.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22975042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brody</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Neufeld</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bieberstein</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">The in vivo kinetics of RNA polymerase II elongation during co-transcriptional splicing</div>, <div class="source">PLoS Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> pg. <div class="fpage">e1000573</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pbio.1000573" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pbio.1000573</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20in%20vivo%20kinetics%20of%20RNA%20polymerase%20II%20elongation%20during%20co-transcriptional%20splicing&amp;author=Y.%20Brody&amp;author=N.%20Neufeld&amp;author=N.%20Bieberstein&amp;publication_year=2011&amp;journal=PLoS%20Biol.&amp;volume=9&amp;pages=e1000573" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1000573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1000573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1000573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21264352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">English</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Loraine</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">Prevalence of alternative splicing choices in <em>Arabidopsis thaliana</em></div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">102</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-102" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-102</a></div><div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-102" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-102</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prevalence%20of%20alternative%20splicing%20choices%20in%20Arabidopsis%20thaliana&amp;author=A.C.%20English&amp;author=K.S.%20Patel&amp;author=A.E.%20Loraine&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhuvanagiri</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schlitter</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Hentze</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Kulozik</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">NMD: RNA biology meets human genetic medicine</div>, <div class="source">Biochem. J.</div> , <div class="year">2010</div>, vol. <div class="volume">430</div> (pg. <div class="fpage">365</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1042/BJ20100699" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1042/BJ20100699</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NMD%3A%20RNA%20biology%20meets%20human%20genetic%20medicine&amp;author=M.%20Bhuvanagiri&amp;author=A.M.%20Schlitter&amp;author=M.W.%20Hentze&amp;author=A.E.%20Kulozik&amp;publication_year=2010&amp;journal=Biochem.%20J.&amp;volume=430&amp;pages=365-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20100699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20100699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20100699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20795950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mekouar</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Blanc-Lenfle</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ozanne</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Detection and analysis of alternative splicing in <em>Yarrowia lipolytica</em> reveal structural constraints facilitating nonsense-mediated decay of intron-retaining transcripts</div>, <div class="source">Genome Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">R65</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2010-11-6-r65" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2010-11-6-r65</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20analysis%20of%20alternative%20splicing%20in%20Yarrowia%20lipolytica%20reveal%20structural%20constraints%20facilitating%20nonsense-mediated%20decay%20of%20intron-retaining%20transcripts&amp;author=M.%20Mekouar&amp;author=I.%20Blanc-Lenfle&amp;author=C.%20Ozanne&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-6-r65" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-6-r65" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-6-r65"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20573210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hood</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Neafsey</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Galagan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sachs</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Evolutionary roles of upstream open reading frames in mediating gene regulation in fungi</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">385</div>-<div class="lpage">409</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.micro.62.081307.162835" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.micro.62.081307.162835</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20roles%20of%20upstream%20open%20reading%20frames%20in%20mediating%20gene%20regulation%20in%20fungi&amp;author=H.M.%20Hood&amp;author=D.E.%20Neafsey&amp;author=J.%20Galagan&amp;author=M.S.%20Sachs&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=63&amp;pages=385-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.62.081307.162835" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.62.081307.162835" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.62.081307.162835"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19514854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carlile</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Watkinson</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Gooday</div> <div class="given-names">G.W.</div></div>. </div>, <div class="source">The Fungi</div> , <div class="year">2001</div><div class="edition">2nd edition</div><div class="publisher-loc">London, UK</div><div class="publisher-name">Academic Press</div></div></div></div></div><div content-id="DST038C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stajich</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Dietrich</div> <div class="given-names">F.S.</div></div>,  <div class="name"><div class="surname">Roy</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">Comparative genomic analysis of fungal genomes reveals intron-rich ancestors</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R223</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2007-8-10-r223" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2007-8-10-r223</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20analysis%20of%20fungal%20genomes%20reveals%20intron-rich%20ancestors&amp;author=J.E.%20Stajich&amp;author=F.S.%20Dietrich&amp;author=S.W.%20Roy&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-10-r223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-10-r223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-10-r223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17949488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Busch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hertel</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">Evolution of SR protein and hnRNP splicing regulatory factors</div>, <div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="year">2012</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/wrna.100" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/wrna.100</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20SR%20protein%20and%20hnRNP%20splicing%20regulatory%20factors&amp;author=A.%20Busch&amp;author=K.J.%20Hertel&amp;publication_year=2012&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=3&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21898828" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fabrizio</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Dannenberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dube</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kastner</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Stark</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Urlaub</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lührmann</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The evolutionarily conserved core design of the catalytic activation step of the yeast spliceosome</div>, <div class="source">Mol. Cell</div> , <div class="year">2009</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">593</div>-<div class="lpage">608</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.molcel.2009.09.040" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.molcel.2009.09.040</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionarily%20conserved%20core%20design%20of%20the%20catalytic%20activation%20step%20of%20the%20yeast%20spliceosome&amp;author=P.%20Fabrizio&amp;author=J.%20Dannenberg&amp;author=P.%20Dube&amp;author=B.%20Kastner&amp;author=H.%20Stark&amp;author=H.%20Urlaub&amp;author=R.%20L%C3%BChrmann&amp;publication_year=2009&amp;journal=Mol.%20Cell&amp;volume=36&amp;pages=593-608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2009.09.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2009.09.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2009.09.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19941820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Käufer</div> <div class="given-names">N.F.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Interactions between two fission yeast serine/arginine-rich proteins and their modulation by phosphorylation</div>, <div class="source">Biochem. J.</div> , <div class="year">2002</div>, vol. <div class="volume">368</div> (pg. <div class="fpage">527</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1042/BJ20021133" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1042/BJ20021133</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactions%20between%20two%20fission%20yeast%20serine%2Farginine-rich%20proteins%20and%20their%20modulation%20by%20phosphorylation&amp;author=Z.%20Tang&amp;author=N.F.%20K%C3%A4ufer&amp;author=R.J.%20Lin&amp;publication_year=2002&amp;journal=Biochem.%20J.&amp;volume=368&amp;pages=527-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj20021133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj20021133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj20021133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12186627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Califice</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Baurain</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hanikenne</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Motte</div> <div class="given-names">P.</div></div>. </div><div class="article-title">A single ancient origin for prototypical serine/arginine-rich splicing factors</div>, <div class="source">Plant Physiol.</div> , <div class="year">2012</div>, vol. <div class="volume">158</div> (pg. <div class="fpage">546</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.111.189019" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.111.189019</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single%20ancient%20origin%20for%20prototypical%20serine%2Farginine-rich%20splicing%20factors&amp;author=S.%20Califice&amp;author=D.%20Baurain&amp;author=M.%20Hanikenne&amp;author=P.%20Motte&amp;publication_year=2012&amp;journal=Plant%20Physiol.&amp;volume=158&amp;pages=546-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.111.189019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.189019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.111.189019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22158759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Alternative splicing and RNA selection pressure—evolutionary consequences for eukaryotic genomes</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">499</div>-<div class="lpage">509</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1896" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1896</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20RNA%20selection%20pressure%E2%80%94evolutionary%20consequences%20for%20eukaryotic%20genomes&amp;author=Y.%20Xing&amp;author=C.%20Lee&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=499-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1896" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1896" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1896"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16770337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodríguez-Kessler</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Baeza-Montañez</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">García-Pedrajas</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Tapia-Moreno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gold</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jiméneze Bremont</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Ruiz-Herrera</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Isolation of <em>UmRrm75</em>, a gene involved in dimorphism and virulence of <em>Ustilago maydis</em></div>, <div class="source">Microbiol. Res.</div> , <div class="year">2012</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">270</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.micres.2011.10.007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.micres.2011.10.007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20UmRrm75%2C%20a%20gene%20involved%20in%20dimorphism%20and%20virulence%20of%20Ustilago%20maydis&amp;author=M.%20Rodr%C3%ADguez-Kessler&amp;author=L.%20Baeza-Monta%C3%B1ez&amp;author=M.D.%20Garc%C3%ADa-Pedrajas&amp;author=A.%20Tapia-Moreno&amp;author=S.%20Gold&amp;author=J.F.%20Jim%C3%A9neze%20Bremont&amp;author=J.%20Ruiz-Herrera&amp;publication_year=2012&amp;journal=Microbiol.%20Res.&amp;volume=167&amp;pages=270-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.micres.2011.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micres.2011.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.micres.2011.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22154329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Whittington</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Pleiotropic function of intersectin homologue Cin1 in <em>Cryptococcus neoformans</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">662</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-2958.2010.07121.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-2958.2010.07121.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pleiotropic%20function%20of%20intersectin%20homologue%20Cin1%20in%20Cryptococcus%20neoformans&amp;author=G.%20Shen&amp;author=A.%20Whittington&amp;author=K.%20Song&amp;author=P.%20Wang&amp;publication_year=2010&amp;journal=Mol.%20Microbiol.&amp;volume=76&amp;pages=662-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2010.76.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2010.76.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2010.76.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20345666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST038C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST038C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoi</div> <div class="given-names">J.W.S.</div></div>,  <div class="name"><div class="surname">Dumas</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Ste12 and ste12-like proteins, fungal transcription factors regulating development and pathogenicity</div>, <div class="source">Eukaryot. Cell</div> , <div class="year">2010</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">480</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/EC.00333-09" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/EC.00333-09</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ste12%20and%20ste12-like%20proteins%2C%20fungal%20transcription%20factors%20regulating%20development%20and%20pathogenicity&amp;author=J.W.S.%20Hoi&amp;author=B.%20Dumas&amp;publication_year=2010&amp;journal=Eukaryot.%20Cell&amp;volume=9&amp;pages=480-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00333-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00333-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00333-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20139240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by: Dr Takashi Ito</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347182&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/1/10.1093/dnares/dst038/2/dst038-dst038supp.pdf?Expires=2147483647&amp;Signature=Aqi5jaWhIeD6-7Myr3spUN9mLUiHY1m~Vsocaa67G-qGj-EK2lf8ShAA99TlFC01K75rAKdgWNHOLA9oPZPK26H2g255GJ~Zz4ETSCSTDnxVonaheFS8zypg8kQflbOCGxH1yBdPS-gDW8cqJZASwsgBj6b3w5F6F9ayOvnA7feCFlqEUXD8Cg6w8mIe-UrqlSjnqZHCCV5utC0V0zfkc~RS9nYeKAAkyRURBOlihBNa8V2Z3MDiOP9hJwnO08G~nSLMWrJdJ8r56BkhH9R872ro8895QXNL5LZ4uAogW~5plBpPc7-a69C65aNI7NRE1GO06LPLvrQ~5XxhqQV46Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">345</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">24</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst038"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347182"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897000003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f1%2f27%2f347182">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897000003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (24)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F1%2F27%2F347182" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347182'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Fungal Alternative Splicing is Associated with Multicellular Complexity and Virulence: A Genome-Wide Multi-Species Study" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst038");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f1%2f27%2f347182">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

